{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n",
    "import scipy \n",
    "import matplotlib.patches as patches\n",
    "import glob\n",
    "import random "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing 54 CDRs\n"
     ]
    }
   ],
   "source": [
    "#Out directory for generated plots\n",
    "plotDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/Axelsson_CaganBlass/GenotypeMatrices/GenomePlots/'\n",
    "\n",
    "#Non-overlap, Total callset\n",
    "noTotal = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/Total_54callset_MxbRemoved/NoSliding/AutoXPar/village_wolf_mxb_rm_54callset_autoXpar_Hudson_Fst_200kbWindow.txt_filtered_windows_RofA_Fst_Zscores.txt'\n",
    "#Overlap, Total callset\n",
    "overTotal = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/Total_54callset_MxbRemoved/Sliding/AutoXPar/village_wolf_mxb_rm_54callset_autoXpar_Hudson_Fst_200kbWindow_50kbSlide.txt_filtered_windows_RofA_Fst_Zscores.txt'\n",
    "#Per Site, Total callset\n",
    "perTotal = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/Total_54callset_MxbRemoved/PerSite/Merged_VillageWolf_mxb_rm_PERSITE_54callset_AutoXParXNonPar_Hudson_Fst_PerSite_Zscores.txt'\n",
    "\n",
    "#Reading into file array\n",
    "fileArray = []\n",
    "fileArray = [noTotal,overTotal,perTotal]\n",
    "\n",
    "#Reading in CDR regions\n",
    "cdrfile = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/Axelsson_CaganBlass/GenotypeMatrices/input/Merged_Axelsson_CaganBlass_canFam3.1.bed'\n",
    "\n",
    "ROIarray = []\n",
    "cdrFile = open(cdrfile, 'r')\n",
    "for line in cdrFile:\n",
    "    line = line.rstrip().split('\\t')\n",
    "    coord = line[0] + ':' + line[1] + '-' + line[2]\n",
    "    ROIarray.append([coord, line[3]])\n",
    "\n",
    "print('Processing %i CDRs' % (len(ROIarray)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'CDR34': ['chrX', '77050000', '77550000', 'CDR34', ['CB_7', 'CB_8', 'CB_9']], 'CDR35': ['chrX', '78250000', '78650000', 'CDR35', ['CB_7', 'CB_8', 'CB_9']], 'CDR36': ['chrX', '105500000', '105800000', 'CDR36', ['CB_13', 'CB_16', 'CB_18', 'CB_15', 'CB_17']], 'CDR1': ['chr1', '2300000', '3250000', 'CDR1', ['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']], 'CDR14': ['chr10', '3950000', '4150000', 'CDR14', ['AX_17']], 'CDR3': ['chr3', '18800000', '19050000', 'CDR3', ['AX_7']], 'CDR25': ['chr18', '2550000', '3750000', 'CDR25', ['AX_26']], 'CDR8': ['chr6', '46800000', '47350000', 'CDR8', ['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']], 'CDR19': ['chr14', '7200000', '7550000', 'CDR19', ['AX_20']], 'CDR2': ['chr1', '79800000', '80150000', 'CDR2', ['AX_4']], 'CDR32': ['chrX', '65900000', '66200000', 'CDR32', ['CB_6']], 'CDR37': ['chrX', '107800000', '110050000', 'CDR37', ['CB_13', 'CB_16', 'CB_18', 'CB_15', 'CB_17']], 'CDR4': ['chr4', '40800000', '41000000', 'CDR4', ['AX_10']], 'CDR20': ['chr16', '7050000', '7400000', 'CDR20', ['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']], 'CDR31': ['chrX', '42850000', '43100000', 'CDR31', ['AX_37']], 'CDR23': ['chr18', '350000', '1100000', 'CDR23', ['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']], 'CDR27': ['chr25', '1050000', '1350000', 'CDR27', ['AX_30']]}\n"
     ]
    }
   ],
   "source": [
    "#reading in intersections with our CDRs\n",
    "intFile = open('/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/Axelsson_CaganBlass/results/Merged_IntersectKiddCDRs_with_Axelsson_AND_CaganBlass.txt','r')\n",
    "\n",
    "intDict = {}\n",
    "ints = []\n",
    "\n",
    "for line in intFile:\n",
    "    line=line.rstrip().split('\\t')\n",
    "    CDRchrom, CDRstart, CDRend, CDRid = line[0:4]\n",
    "    intersect = line[7]\n",
    "    if CDRid not in ints:\n",
    "        ints.append(CDRid)\n",
    "        intDict[CDRid] = []\n",
    "        intersectList = []\n",
    "        intDict[CDRid] = [CDRchrom, CDRstart, CDRend, CDRid,intersectList]\n",
    "    intersectList.append(intersect)\n",
    "    intDict[CDRid][4] = intersectList\n",
    "\n",
    "print(intDict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##############################################################################################\n",
    "def find_gene_pos():\n",
    "    #GTF file with genes of interest:\n",
    "    gtfFile = open('/home/ampend/kidd-lab/ampend-projects/BLAST2GO/ensembl_input/Canis_familiaris.CanFam3.1.81.gtf','r')\n",
    "\n",
    "    #YOU NEED TO DO THIS BY HAND BY GREP'ING THE GTF FILE:\n",
    "    #genesOfInterest = ['ENSCAFG00000028653','RNPC3','COL11A1']\n",
    "    genesOfInterest = []\n",
    "    genePos = []\n",
    "\n",
    "    for line in gtfFile:\n",
    "        if '#' in line:\n",
    "            continue\n",
    "        line=line.rstrip().split('\\t')\n",
    "        if 'exon' not in line[2]:\n",
    "            continue\n",
    "        chrom = 'chr' + line[0]\n",
    "        start = int(line[3])\n",
    "        end = int(line[4])\n",
    "        if chrom == ROIchrom:\n",
    "            if start >= ROIstart and end <= ROIend:\n",
    "                #Finding gene ID in line\n",
    "                geneInfo = line[8]\n",
    "                if 'gene_name' in geneInfo:\n",
    "                    geneInfo = geneInfo.split(';')\n",
    "                    geneInfo = geneInfo[5].split(' ')\n",
    "                    geneID = geneInfo[2].replace('\"','')\n",
    "                else:\n",
    "                    geneInfo = geneInfo.split(';')\n",
    "                    geneInfo = geneInfo[0].split(' ')\n",
    "                    geneID = geneInfo[1].replace('\"','')\n",
    "                \"\"\"for i in genesOfInterest:\n",
    "                    if geneID == i:\n",
    "                        genePos.append([geneID,start,end])\"\"\"\n",
    "                if geneID not in genesOfInterest:\n",
    "                    genesOfInterest.append(geneID)\n",
    "                genePos.append([geneID,start,end])\n",
    "            else:\n",
    "                continue\n",
    "        else:\n",
    "            continue\n",
    "    gtfFile.close()\n",
    "    return genesOfInterest,genePos\n",
    "###################################################################\n",
    "##LONG## \n",
    "def get_data():\n",
    "    index=-1\n",
    "    for file in fileArray:\n",
    "        index += 1\n",
    "        inFile = open(file,'r')\n",
    "        for line in inFile:\n",
    "            if 'Chrom' in line or 'CHROM' in line:\n",
    "                continue\n",
    "            line = line.rstrip().split('\\t')\n",
    "            if 'PerSite' in file:\n",
    "                chrom,start,end,Z = line[0],int(line[1]),int(line[1]),float(line[3])\n",
    "            else:\n",
    "                chrom,start,end, Z = line[0],int(line[1]),int(line[2]),float(line[9])\n",
    "            #Now check if the call is within the ROI\n",
    "            if chrom == ROIchrom and start >= ROIstart and end <= ROIend:\n",
    "                if '50kbSlide' in file: #OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    SlideArray.append([chrom,midpoint,Z])\n",
    "                    continue\n",
    "                if 'PerSite' in file: #PER SITE WINDOW FILE\n",
    "                    perSiteArray.append([chrom,start,Z])\n",
    "                    continue\n",
    "                else: #NO OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    NoSlideArray.append([chrom,midpoint,Z])\n",
    "                    continue                \n",
    "                #outFile.write('%s\\t%s\\t%s\\t%s\\n' % (chrom,start,end,Z)) \n",
    "        inFile.close()\n",
    "    return SlideArray,perSiteArray,NoSlideArray\n",
    "####################################################################################\n",
    "def get_fst():\n",
    "    ##LONG## \n",
    "    index=-1\n",
    "    perSiteArray, SlideArray, NoSlideArray = [] , [], []\n",
    "    for file in fileArray:\n",
    "        index += 1\n",
    "        inFile = open(file,'r')\n",
    "        for line in inFile:\n",
    "            if 'Chrom' in line or 'CHROM' in line:\n",
    "                continue\n",
    "            line = line.rstrip().split('\\t')\n",
    "            if 'PerSite' in file:\n",
    "                chrom,start,end,Z = line[0],int(line[1]),int(line[1]),float(line[3])\n",
    "            else:\n",
    "                chrom,start,end, Z = line[0],int(line[1]),int(line[2]),float(line[9])\n",
    "            #Now check if the call is within the ROI\n",
    "            if chrom == ROIchrom and start >= ROIstart and end <= ROIend:\n",
    "                if '50kbSlide' in file: #OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    SlideArray.append([chrom,midpoint,Z])\n",
    "                    continue\n",
    "                if 'PerSite' in file: #PER SITE WINDOW FILE\n",
    "                    perSiteArray.append([chrom,start,Z])\n",
    "                    continue\n",
    "                else: #NO OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    NoSlideArray.append([chrom,midpoint,Z])\n",
    "                    continue                \n",
    "    return perSiteArray, SlideArray, NoSlideArray\n",
    "##############################################################################\n",
    "def thin_snps():\n",
    "    #THIS THINS THE MASSIVE LIST OF PER SITE ZFST VALUES\n",
    "    percentage = float(50)\n",
    "    k = int(len(perSiteArray) * percentage // 100)\n",
    "    indices = random.sample(xrange(len(perSiteArray)), k)\n",
    "    thinned_perSiteArray = [perSiteArray[i] for i in indices]\n",
    "    return thinned_perSiteArray\n",
    "#############################################################################\n",
    "def determine_sig(NoSlideArray,SlideArray):\n",
    "    passes = False\n",
    "    plotArray = []\n",
    "    for i in range(0,len(SlideArray)):\n",
    "        x, y, width = SlideArray[i][1], SlideArray[i][2], 15000\n",
    "        if y > 5:\n",
    "            passes = True\n",
    "    if passes == False:\n",
    "        plotArray = NoSlideArray\n",
    "        barColor = \"green\"\n",
    "    else: \n",
    "        plotArray = SlideArray\n",
    "        barColor = \"red\"\n",
    "    return plotArray, barColor\n",
    "#############################################################################\n",
    "def draw_plot(ROIchrom,ROI_CDR, SlideArray, NoSlideArray,perSiteArray):\n",
    "    plt.figure(figsize=(14,6))\n",
    "    #plotArray, barColor = determine_sig(NoSlideArray,SlideArray)\n",
    "    \n",
    "    #sliding window\n",
    "    for i in range(0,len(SlideArray)):\n",
    "        x, y, width = SlideArray[i][1], SlideArray[i][2], 15000\n",
    "        if y > 5:\n",
    "            plt.bar(x, y, width, color=\"red\",alpha=0.6,linewidth=1)\n",
    "        else:\n",
    "            plt.bar(x, y, width, fill=False,alpha=0.6,linewidth=1)\n",
    "    #NO sliding window\n",
    "    for i in range(0,len(NoSlideArray)):\n",
    "        x, y, width = NoSlideArray[i][1]-15000, NoSlideArray[i][2], 15000\n",
    "        if y > 5:\n",
    "            plt.bar(x, y, width, color=\"green\",alpha=0.6,linewidth=1)\n",
    "        else:\n",
    "            plt.bar(x, y, width, fill=False,alpha=0.6,linewidth=1)\n",
    "    #SNPs\n",
    "    for i in range(0,len(perSiteArray)):\n",
    "        x,y =perSiteArray[i][1],perSiteArray[i][2]\n",
    "        if y > 5:\n",
    "            plt.scatter(x,y,facecolor='red',edgecolor='k',s=20)\n",
    "        else:\n",
    "            plt.scatter(x,y,color='black',alpha=0.8,s=10)\n",
    "    #DEFINING AXIS LIMS\n",
    "    plt.ylim(-9,8.5)\n",
    "    plt.xlim(ROIstart,ROIend)\n",
    "    ax = plt.axes()\n",
    "    #DRAwING GENE MODELS\n",
    "    ###mp = []\n",
    "    IDheights=[-3.5,-4.5,-5.5,-6.5,-7.5,-8.5]\n",
    "    geneheights=[-3,-4,-5,-6,-7,-8]\n",
    "    index = -1\n",
    "    for gene in genesOfInterest:\n",
    "        xRange = []\n",
    "        #determines height to print the genes and gene IDs\n",
    "        if index == 5:\n",
    "            index = 0\n",
    "        else:\n",
    "            index += 1\n",
    "        for i in range(0,len(genePos)):\n",
    "            if gene == genePos[i][0]:\n",
    "                x = genePos[i][1]\n",
    "                length = genePos[i][2]-genePos[i][1]\n",
    "                #ax.add_patch(patches.Rectangle((x, -3),length,0.75,facecolor=\"black\"))\n",
    "                ax.add_patch(patches.Rectangle((x,geneheights[index]),length,0.75,facecolor=\"black\"))\n",
    "                xRange.append(x)\n",
    "        xMin,xMax = min(xRange),max(xRange)\n",
    "        plt.hlines(y=geneheights[index]+0.4, xmin=xMin, xmax=xMax,lw=1,color=\"black\")\n",
    "        #ADDING GENE TEXT\n",
    "        if 'ENSCAFG' in gene:\n",
    "            gene = gene.replace('ENSCAFG','ENS')\n",
    "        if xMax < ROIend-10000: #Make sure the gene ID doesn't get printed off the plot\n",
    "            ax.text(xMax+2500,geneheights[index]+0.3,gene,fontsize=7)\n",
    "    #ADDING TITLES\n",
    "    plt.xlabel('Chromosome %s Position' % ROIchrom,fontsize=20)\n",
    "    plt.ylabel(r'$ZF_{ST}$',fontsize=20)\n",
    "    #Add CDR window\n",
    "    CDR = [ROIstart+210000,ROIend-210000]\n",
    "    ax.add_patch(patches.Rectangle((CDR[0], -1.75),(CDR[1]-CDR[0]),0.5,fill=None))\n",
    "    ax.text(((ROIstart+ROIend)/2-10000), -1.7,str(ROI_CDR))\n",
    "    ax.ticklabel_format(useOffset=False,style='plain')#turn off sci-notation in  xaxis labelling\n",
    "    plt.savefig(plotDir + ROI_CDR + 'Locus_ZFstPlotWithGeneModels.png')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr1:2515608-3315787\n",
      "Looking for the Fst information for these coordinates: chr1 2305608 3525787\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['SNORA70', 'GALR1', 'MBP', 'ZNF236', 'U6', 'ZNF516']\n",
      "chr1:46572800-46874590\n",
      "Looking for the Fst information for these coordinates: chr1 46362800 47084590\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ARID1B', 'TMEM242', 'ENSCAFG00000028896', 'ZDHHC14']\n",
      "chr1:63560426-63760428\n",
      "Looking for the Fst information for these coordinates: chr1 63350426 63970428\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['NKAIN2', 'RNF217']\n",
      "chr1:79948273-80148387\n",
      "Looking for the Fst information for these coordinates: chr1 79738273 80358387\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000030357', 'U7']\n",
      "chr3:15326157-15626159\n",
      "Looking for the Fst information for these coordinates: chr3 15116157 15836159\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['KIAA0825', 'FAM172A', '7SK', 'ENSCAFG00000008096']\n",
      "chr3:18623348-18823350\n",
      "Looking for the Fst information for these coordinates: chr3 18413348 19033350\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['LYSMD3', 'POLR3G', 'ENSCAFG00000028450', 'MBLAC2', 'CETN3', 'ENSCAFG00000031095', '7SK']\n",
      "chr3:32064282-32261994\n",
      "Looking for the Fst information for these coordinates: chr3 31854282 32471994\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['HERC2', '7SK', 'ENSCAFG00000009769', 'NIPA1', 'NIPA2', 'CYFIP1', 'TUBGCP5', 'OCA2']\n",
      "chr4:14494336-14694347\n",
      "Looking for the Fst information for these coordinates: chr4 14284336 14904347\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ZNF365', 'ADO', 'EGR2']\n",
      "chr4:40803258-41001116\n",
      "Looking for the Fst information for these coordinates: chr4 40593258 41211116\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['FGF18', 'ENSCAFG00000016912', 'U6', 'TLX3', 'RANBP17', 'GABRP']\n",
      "chr6:24980112-25280253\n",
      "Looking for the Fst information for these coordinates: chr6 24770112 25490253\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000024109', 'ACSM5', 'PDILT', 'UMOD', 'GP2', 'ENSCAFG00000027918', 'GPR139', 'GPRC5B', 'IQCK']\n",
      "chr6:46854108-47454177\n",
      "Looking for the Fst information for these coordinates: chr6 46644108 47664177\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000028653', 'RNPC3', 'COL11A1']\n",
      "chr6:53253173-53453184\n",
      "Looking for the Fst information for these coordinates: chr6 53043173 53663184\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000029795']\n",
      "chr7:24632211-25033464\n",
      "Looking for the Fst information for these coordinates: chr7 24422211 25243464\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000014399', 'ENSCAFG00000031419', 'ENSCAFG00000028184', 'ENSCAFG00000025140', 'U6', '7SK']\n",
      "chr8:27696698-27896700\n",
      "Looking for the Fst information for these coordinates: chr8 27486698 28106700\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['TMX1', 'U6', 'FRMD6']\n",
      "chr10:2714703-2914705\n",
      "Looking for the Fst information for these coordinates: chr10 2504703 3124705\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['LRIG3', 'U6']\n",
      "chr10:3615191-4015195\n",
      "Looking for the Fst information for these coordinates: chr10 3405191 4225195\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['SLC16A7', 'SNORA31']\n",
      "chr11:47269654-47669725\n",
      "Looking for the Fst information for these coordinates: chr11 47059654 47879725\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000001731', 'ENSCAFG00000001734', 'ENSCAFG00000001737', '5S_rRNA']\n",
      "chr14:7244539-7543124\n",
      "Looking for the Fst information for these coordinates: chr14 7034539 7753124\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000020442', 'cfa-mir-96', 'cfa-mir-182', 'NRF1', 'STRIP2', 'AHCYL2', 'ENSCAFG00000001530', 'SMO', 'TSPAN33', 'TNPO3', 'ENSCAFG00000029750', 'IRF5', 'KCP']\n",
      "chr15:5093546-5393613\n",
      "Looking for the Fst information for these coordinates: chr15 4883546 5603613\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['MANEAL', 'GNL2', 'DNALI1', 'SNIP1', 'MEAF6', 'ZC3H12A', 'GRIK3']\n",
      "chr15:35192647-35392649\n",
      "Looking for the Fst information for these coordinates: chr15 34982647 35602649\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['NDUFA12', 'NR2C1', 'FGD6', 'VEZT', 'cfa-mir-331', 'METAP2', 'USP44', 'NTN4']\n",
      "chr16:6828778-7342805\n",
      "Looking for the Fst information for these coordinates: chr16 6618778 7552805\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['KEL', 'C7orf34', 'TRPV5', 'TRPV6', 'ENSCAFG00000003745', 'TRBV30', 'ENSCAFG00000014478', 'ENSCAFG00000028951', 'ENSCAFG00000014481', 'TRBV27', 'ENSCAFG00000031359', 'TRBV16', 'TRBV15', 'ENSCAFG00000024810', 'ENSCAFG00000003812', 'ENSCAFG00000003811', 'ENSCAFG00000030524', 'TRBV3-1', 'ENSCAFG00000003818', 'ENSCAFG00000003820', 'PRSS58', 'ENSCAFG00000003827', 'ENSCAFG00000029181', 'ENSCAFG00000030973', 'ENSCAFG00000030631', 'MGAM2', 'MGAM', 'TAS2R38', 'CLEC5A', 'ENSCAFG00000003655', 'ENSCAFG00000003876', 'PRSS37', 'ENSCAFG00000003879', 'U6', 'TAS2R5', 'TAS2R3', 'SSBP1', 'WEE2', 'KIAA1147']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/sw/lsa/centos7/python-anaconda-arc-connect/latest/lib/python3.5/site-packages/matplotlib/pyplot.py:524: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).\n",
      "  max_open_warning, RuntimeWarning)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr17:38657284-38857286\n",
      "Looking for the Fst information for these coordinates: chr17 38447284 39067286\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['RMND5A', 'RNF103', 'ENSCAFG00000007479', 'KDM3A', 'REEP1', 'MRPL35', 'IMMT', 'SNORA51', 'PTCD3', 'SNORD94', 'POLR1A']\n",
      "chr18:414502-1414626\n",
      "Looking for the Fst information for these coordinates: chr18 204502 1624626\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ABCA13', 'ENSCAFG00000031496', 'ENSCAFG00000024474', 'ENSCAFG00000030239', 'VWC2', 'ZPBP', 'ENSCAFG00000003358', 'SNORA7', 'ENSCAFG00000029400']\n",
      "chr18:3217603-4718037\n",
      "Looking for the Fst information for these coordinates: chr18 3007603 4928037\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['7SK', 'U1', 'ENSCAFG00000003455']\n",
      "chr19:37683163-37883157\n",
      "Looking for the Fst information for these coordinates: chr19 37473163 38093157\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['TMEM163', 'ACMSD', 'ENSCAFG00000031430', 'CCNT2', 'MAP3K19', 'RAB3GAP1', 'ZRANB3']\n",
      "chr22:19995751-20192912\n",
      "Looking for the Fst information for these coordinates: chr22 19785751 20402912\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " []\n",
      "chr25:1015697-1515729\n",
      "Looking for the Fst information for these coordinates: chr25 805697 1725729\n",
      "\n",
      "Genes of Interest in Region of Interest:\n",
      " ['ENSCAFG00000005968', 'COG6', 'LHFP']\n"
     ]
    }
   ],
   "source": [
    "for i in range(0,len(ROIarray)):\n",
    "    ROI =  ROIarray[i][0]\n",
    "    print(ROI)\n",
    "    ROIchrom,pos = ROI.split(':')\n",
    "    ROI_CDR = ROIarray[i][1]\n",
    "    ROIstart,ROIend = pos.split('-')\n",
    "    ROIstart,ROIend = int(ROIstart)-210000,int(ROIend)+210000\n",
    "\n",
    "    print('Looking for the Fst information for these coordinates:',ROIchrom,ROIstart,ROIend)\n",
    "    #Finding genes in the region\n",
    "    genesOfInterest = []\n",
    "    genePos = []\n",
    "    genesOfInterest,genePos = find_gene_pos()\n",
    "\n",
    "    print('\\nGenes of Interest in Region of Interest:\\n', genesOfInterest)\n",
    "    \n",
    "    perSiteArray, SlideArray, NoSlideArray = get_fst()\n",
    "    #thinned_perSiteArray = thin_snps()\n",
    "    #perSiteArray = thinned_perSiteArray\n",
    "    #length = len(thinned_perSiteArray)\n",
    "    #print '%i SNPs went are found in this region' % length\n",
    "    draw_plot(ROIchrom,ROI_CDR, SlideArray, NoSlideArray,perSiteArray)\n",
    "\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['CB_7', 'CB_8', 'CB_9']\n",
      "['CB_7', 'CB_8', 'CB_9']\n",
      "['CB_13', 'CB_16', 'CB_18', 'CB_15', 'CB_17']\n",
      "['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']\n",
      "['AX_17']\n",
      "['AX_7']\n",
      "['AX_26']\n",
      "['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']\n",
      "['AX_20']\n",
      "['AX_4']\n",
      "['CB_6']\n",
      "['CB_13', 'CB_16', 'CB_18', 'CB_15', 'CB_17']\n",
      "['AX_10']\n",
      "['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']\n",
      "['AX_37']\n",
      "['AX_40', 'AX_41', 'AX_39', 'CB_1', 'CB_2', 'CB_3', 'CB_4', 'CB_5']\n",
      "['AX_30']\n"
     ]
    }
   ],
   "source": [
    "for i in intDict:\n",
    "    print (intDict[i][4])\n",
    "    if ROI_CDR in intDict[i][4]:\n",
    "        print (intDict[i][3],intDict[i][4])\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "AX_1\n"
     ]
    }
   ],
   "source": [
    "print (ROI_CDR)"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
