{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#2017-05-09\n",
    "#A. Pendleton\n",
    "#This script is used to make an AGP file for Zoey. Can be built upon as new forms\n",
    "#   of contigs become assembled and incorporated into the AGP.\n",
    "\n",
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import subprocess\n",
    "\n",
    "#import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#This script is used to make an AGP file for Zoey. Can be built upon as new contigs become assembled and incorporated into the available set for assembly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "CHROM = 'chr18'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Current working directory is:\n",
      " /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/\n"
     ]
    }
   ],
   "source": [
    "wkDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/'\n",
    "print('Current working directory is:\\n', wkDir)\n",
    "\n",
    "logFile = open(wkDir + 'temp/iPythonNotebook_LogFile_' + CHROM + '_v1.txt','w')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Identified coordinates for 77 primary contigs on chr18\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "54"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#####READING IN PRIMARY CONTIG COORDINATES FROM MUMMER ALIGNMENT#######\n",
    "#Primary contig alignments (no canu processing)\n",
    "primaryContigCoord = open(wkDir + 'input/primary.2017-04-10.txt', 'r')\n",
    "#primaryContigCoordBed = open(wkDir + 'input/primary.2017-04-10_' + CHROM +'.bed', 'w')\n",
    "\n",
    "contigDict,index = {}, 0\n",
    "\n",
    "for line in primaryContigCoord:\n",
    "    if line.startswith(\"#\") is True: #skips header line\n",
    "        continue\n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID, length, Dir, chrom, start, end = line[0],line[1],line[2],line[3],int(line[4]),int(line[5])    \n",
    "    if CHROM not in chrom: #FILTERING ONLY FOR CHROM 18 FOR NOW!!! REMOVE LATER \n",
    "        continue\n",
    "    index += 1\n",
    "    contigDict[index] = [chrom,start,end,contigID,Dir,length]\n",
    "    #primaryContigCoordBed.write('%s\\t%i\\t%i\\t%s\\n' % (chrom,int(start-1),end,contigID))\n",
    "    #if index > 15: #FOR TESTING--- REMOVE LATER\n",
    "    #    break\n",
    "    \n",
    "primaryContigCoord.close()\n",
    "#primaryContigCoordBed.close()\n",
    "print ('Identified coordinates for %i primary contigs on %s' % (len(contigDict), CHROM))\n",
    "logFile.write('Identified coordinates for %i primary contigs on %s' % (len(contigDict), CHROM))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step 1. Read in coordinates from alignment file based on Mummer of primary contigs (per Jeff)\n",
    "Step 2. BLAT each contig against the contig proximal to that proximal to it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_line(Dict,curr_index,i):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[curr_index][0],int(Dict[curr_index][1]),int(Dict[curr_index][2]), Dict[curr_index][3], Dict[curr_index][4], int(Dict[curr_index][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i+1][0],int(Dict[i+1][1]),int(Dict[i+1][2]), Dict[i+1][3], Dict[i+1][4], int(Dict[i+1][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def read_last_line(Dict):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[i-1][0],int(Dict[i-1][1]),int(Dict[i-1][2]), Dict[i-1][3], Dict[i-1][4], int(Dict[i-1][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i][0],int(Dict[i][1]),int(Dict[i][2]), Dict[i][3], Dict[i][4], int(Dict[i][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def process_last_contig(contigDict,curr_index,i):\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_last_line(contigDict)\n",
    "    #print ('\\n#%s (%i) -- %s (%i)' % (contigID1,i-1,contigID2,i))\n",
    "    logFile.write(('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,i-1,contigID2,i)))\n",
    "    if start1 == start2 or end1 == end2:\n",
    "        return\n",
    "    if end2 > end1:\n",
    "        curr_index+=1 \n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        inList.append(contigID2)\n",
    "        #print('PASS - Last contig on chromosome overlap or is spaced correctly')\n",
    "        logFile.write('PASS - Last contig on chromosome overlap or is spaced correctly\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_first_contig(contigDict,curr_index,i):\n",
    "    curr_index+=1\n",
    "    #Always save first and contig in the dictionary/dataset\n",
    "    curated_contigDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "    curated_contigDict[i+1] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    inList.append(contigID2)\n",
    "    #print('PASS - First call in dataset')\n",
    "    logFile.write('PASS - First call in dataset\\n')\n",
    "    return curr_index \n",
    "###################################################################################################\n",
    "def process_same_starts(contigDict,curr_index,i):\n",
    "    if start1 == start2 and end1 > end2:\n",
    "        #print('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "    if start1 == start2 and end1 < end2: #overwrite previous position if contig 2 is longer than contig 1\n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        #print('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_same_ends(contigDict,curr_index,i):    \n",
    "    if start1 < start2 and end1 == end2:\n",
    "        #print('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return \n",
    "    if start2 < start1 and end1 == end2: \n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DONE!\n"
     ]
    }
   ],
   "source": [
    "curated_contigDict, curr_index, inList = {}, 1, []\n",
    "chrom1,chrom2 = '0', '0' #initializing values\n",
    "\n",
    "\n",
    "for key, value in contigDict.items(): \n",
    "    i = int(key)\n",
    "    #1. If last contig in dataset \n",
    "    if i == len(contigDict):\n",
    "        continue\n",
    "    \n",
    "    #2. If contigs are on different chromosomes:\n",
    "    if chrom1 != chrom2:\n",
    "        process_last_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(contigDict,curr_index,i)\n",
    "    #print ('\\n#%s (%i) -- %s (%i)' % (contigID1,curr_index,contigID2,i+1))\n",
    "    logFile.write('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,curr_index,contigID2,i+1))\n",
    "\n",
    "    #3. Automatically saves first contig in dataset\n",
    "    if curr_index == 1 and len(curated_contigDict) == 0:\n",
    "        curr_index = process_first_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "    \n",
    "    #4. If contigs have the same start site, choose the longest contig\n",
    "    if start1 == start2:\n",
    "        process_same_starts(contigDict,curr_index,i)\n",
    "        continue\n",
    "\n",
    "    #5. If contigs have same end coordinate, choose longest contig (i.e. contig #1 if sorted)\n",
    "    if end1 == end2: \n",
    "        process_same_ends(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    #6. If contigs #1 and #2 have same start AND end, keep contig #1\n",
    "    if start1 == start2 and end1 == end2:\n",
    "        #print('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        continue #because contig1 is already in the dictionary\n",
    "        \n",
    "    #7. If contig #2 is fully within contig #1 -- continue\n",
    "    if start1 < start2 and end1 > end2: # contig2 is fully within contig1\n",
    "        #curated_contigDict[curr_index] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "        #print('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) +'\\n')\n",
    "        continue\n",
    "    if start1 > start2 and end1 < end2:\n",
    "        #print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)\n",
    "    \n",
    "    #8. If contig passes all these - then automatically saves\n",
    "    curr_index=i+1\n",
    "    #if contigID2 not in inList:\n",
    "    curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    #print('PASS - contigs overlap or are spaced correctly')\n",
    "    logFile.write('PASS - contigs overlap or are spaced correctly\\n')\n",
    "\n",
    "#print ('Identified CURATED coordinates for %i primary contigs' % len(curated_contigDict))\n",
    "logFile.write('\\n##Identified CURATED coordinates for %i primary contigs\\n\\n########\\n\\n' % len(curated_contigDict))\n",
    "\n",
    "\n",
    "#Re-naming dictionary keys\n",
    "count=0\n",
    "for keys in curated_contigDict:\n",
    "    count+=1\n",
    "    curated_contigDict[count]=curated_contigDict.pop(keys)\n",
    "\n",
    "print('DONE!')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"print('Non-curated:')\\nfor i in contigDict:\\n    print(i,contigDict[i])\\nprint( '\\nCurated:')\\nfor i in curated_contigDict:\\n    print(i, curated_contigDict[i])\""
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"print('Non-curated:')\n",
    "for i in contigDict:\n",
    "    print(i,contigDict[i])\n",
    "print( '\\nCurated:')\n",
    "for i in curated_contigDict:\n",
    "    print(i, curated_contigDict[i])\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1):\n",
    "    chrom2 = chrom1\n",
    "    start2 = start1 - 10000\n",
    "    end2 = start1 + 10000\n",
    "    contigID2 = 'canFam3'\n",
    "    Dir2 = 'fwd'\n",
    "    length1 = end2-start2\n",
    "    \n",
    "    if start2 < 0:\n",
    "        start2 = 0\n",
    "    \n",
    "    #Determine overlap \n",
    "    determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    " \n",
    "    #Find coordinates to extract\n",
    "    find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    \n",
    "    #Extract FASTA\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/genomes/canFam3.1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "###################################################################################################\n",
    "def determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    #How much do the contigs overlap?\n",
    "    global overlap\n",
    "    overlap = end1 - start2\n",
    "    #print('\\nOverlap = ', overlap)\n",
    "    #OVERLAPPING CONTIGS\n",
    "    if overlap > 0:\n",
    "        find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    #DIRECTLY ADJACENT CONTIGS\n",
    "    if overlap == 0:\n",
    "        #print('Contigs are directly adjacent to one another')\n",
    "        logFile.write('Contigs are directly adjacent to one another\\n')\n",
    "        #WRITE FUNCTION FOR THESE\n",
    "    #CONTIGS WITH GAP BETWEEN THEM\n",
    "    if overlap < 0:\n",
    "        #print('Gap between contigs')\n",
    "        logFile.write('Gap between contigs\\n')\n",
    "\n",
    "###################################################################################################\n",
    "def find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    ###Determine coordinates to extract for BLAT\n",
    "    global extract_coord1\n",
    "    global extract_coord2\n",
    "    #Contig #1\n",
    "    if 'fwd' in Dir1:\n",
    "        extract_coord1 = [length1 - overlap - 10000, length1]\n",
    "    else:\n",
    "        extract_coord1 = [0, overlap + 10000]\n",
    "    #Contig #2\n",
    "    if 'fwd' in Dir2:\n",
    "        extract_coord2 = [0, overlap + 10000] \n",
    "    else:\n",
    "        extract_coord2 = [length2 - overlap - 10000, length2]\n",
    "    \n",
    "    #print ('Coordinates to extract for BLAT: ',extract_coord1,extract_coord2,'\\n')\n",
    "    logFile.write('Coordinates to extract for BLAT: ' + str(extract_coord1) + str(extract_coord2) + '\\n')\n",
    "\n",
    "    #safety check\n",
    "    for i in range(0,1): #CHECKS \n",
    "        #= if the region to extract extends beyond the length of the contig, if so then the coordinate changes to the length of contig\n",
    "        if extract_coord1[i] > length1:\n",
    "            extract_coord1[i] = length1\n",
    "        if extract_coord2[i] > length2:\n",
    "            extract_coord2[i] = length2\n",
    "        # If the region extended too far, and the value is negative -- then make starting extraction coordinate = 0\n",
    "        if extract_coord1[i] < 0: \n",
    "            extract_coord1[i] = 0\n",
    "        if extract_coord2[i] < 0:\n",
    "            extract_coord2[i] = 0\n",
    "            \n",
    "    if overlap < 0: #### SEND TO DIFFERENT FUNCTION LATER -- FOLLOW OVERLAPPING CONTIGS FOR NOW\n",
    "        #print('Contigs do not overlap')\n",
    "        logFile.write('Contigs do not overlap\\n')\n",
    "    #Where to find fasta of contigs\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1, contigID2,Dir2,extract_coord2)\n",
    "    \n",
    "###################################################################################################\n",
    "def extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID1 + '/' + contigID1 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID1,extract_coord1[0],extract_coord1[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    #Contig 2\n",
    "    fasta_path2 = fastaRoot + contigID2 + '/' + contigID2 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %stemp/contig2.fa' % (fasta_path2,contigID2,extract_coord2[0],extract_coord2[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "###################################################################################################\n",
    "def run_blat(wkDir):\n",
    "    blatcmd = 'blat %stemp/contig1.fa %stemp/contig2.fa %stemp/temp.blat' % (wkDir,wkDir,wkDir)\n",
    "    logFile.write(blatcmd + '\\n')\n",
    "    subprocess.call(blatcmd,shell=True)\n",
    "    \n",
    "    \n",
    "###################################################################################################\n",
    "def parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    inFile = open('%stemp/temp.blat' % (wkDir),'r')\n",
    "    lineCount, cleanAlignment, offset = 0, False, 0\n",
    "    for line in inFile:\n",
    "        line=line.rstrip().split('\\t')\n",
    "        lineCount+=1\n",
    "        if lineCount > 5:#skips the BLAT results headers\n",
    "            score,strand = int(line[0]),line[8]\n",
    "            percent_of_overlap = float(score)/overlap\n",
    "            if percent_of_overlap < float(0.75):\n",
    "                continue\n",
    "            #print('\\n#Parsing BLAT results','Percent of overlap matched in BLAT: ',format(percent_of_overlap, '.3f'))\n",
    "            logFile.write('\\n#Parsing BLAT results\\n' + 'Percent of overlap matched in BLAT: ' + format(percent_of_overlap, '.3f') + ' \\n')\n",
    "            #CHECKS TO MAKE SURE WE DID THIS RIGHT - correct end vs correct end\n",
    "            #if '+' not in strand:\n",
    "            #    print('ERROR: Top hit not in proper orientation.... SKIPPING -- PLEASE CHECK')\n",
    "            #    return\n",
    "            #parse contig #1 (left contig)\n",
    "            blat_length1, blat_start1, blat_end1 = int(line[14]),int(line[15]),int(line[16])\n",
    "            #print(blat_length1, blat_start1, blat_end1)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length1, blat_start1, blat_end1))\n",
    "            #parse contig #2 (right contig)\n",
    "            blat_length2, blat_start2, blat_end2 = int(line[10]),int(line[11]),int(line[12])\n",
    "            #print(blat_length2, blat_start2, blat_end2)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length2, blat_start2, blat_end2))\n",
    "            #Calculate offset  - to compensate for non-clean alignments (extension of left contig that does not overlap with adjacent contig)\n",
    "            offset = calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2)\n",
    "            return offset\n",
    "###################################################################################################\n",
    "def calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2):\n",
    "    global offset\n",
    "    if 'rc' in Dir1:\n",
    "        offset = blat_start1 - 0\n",
    "    else:\n",
    "        offset = blat_length1 - blat_end1\n",
    "    if offset < 3:\n",
    "        offset = 0\n",
    "    #print('\\nBLAT offset = ', offset)\n",
    "    logFile.write('\\nBLAT offset = ' + str(offset) + '\\n')\n",
    "    return offset\n",
    "\n",
    "###################################################################################################\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "50 elements in position dictionary from Zoey PacBio contigs....\n"
     ]
    }
   ],
   "source": [
    "#write_AGP_header(agpFile)\n",
    "posDict, offset_BLAT = {}, []\n",
    "\n",
    "for i in range(1,len(curated_contigDict)+1): \n",
    "    curr_index = i\n",
    "    offset = 0 #set equal to zero at beginning, will change if there is one from parsing the blat hit(s)\n",
    "    #If last entry, automatically goes in\n",
    "    if i == len(curated_contigDict):\n",
    "        chrom1,start1,end1,contigID1,Dir1,length1 = curated_contigDict[curr_index][0],int(curated_contigDict[curr_index][1]),int(curated_contigDict[curr_index][2]), curated_contigDict[curr_index][3], curated_contigDict[curr_index][4], int(curated_contigDict[curr_index][5])\n",
    "        posDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1,int('0'),int('0'),'na']\n",
    "        continue\n",
    "        \n",
    "    #1. Reading in coordinates from contigList\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(curated_contigDict, curr_index, i)\n",
    "    \n",
    "    #2. Checking contigs are on same chromosome\n",
    "    if chrom1 != chrom2:# or i == 1: #This is first contig on chromosome, need to process it first\n",
    "        #continue\n",
    "        process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1) \n",
    "        continue\n",
    "    #print ('\\n#',contigID1, contigID2,'\\n',chrom1,start1,end1,contigID1,Dir1,length1,'\\n',chrom2,start2,end2,contigID2,Dir2,length2)\n",
    "    \n",
    "    #3. Determines the overlap/orientation of the two contigs\n",
    "    determine_contig_overlap(end1,contigID1,Dir1,length1,end2,contigID2,Dir2,length2)\n",
    "\n",
    "    #4. BLAT the properly oriented contig ends against one another\n",
    "    #     and parse the results\n",
    "    run_blat(wkDir)\n",
    "    parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "    if offset > 0:\n",
    "        offset_BLAT.append([contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2,overlap])\n",
    "    \n",
    "    #save contig1 information to dictionary\n",
    "    posDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1,overlap,offset,contigID2]\n",
    "\n",
    "\n",
    "    #print('###################\\n')\n",
    "    #if i > 2:\n",
    "    #    break\n",
    "print('%i elements in position dictionary from Zoey PacBio contigs....' % (len(posDict)))        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'for i in posDict:\\n    print (posDict[i][0:9])'"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"for i in posDict:\n",
    "    print (posDict[i][0:9])\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\"\"\"def process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = agp_chrom, 0, 0\n",
    "    \n",
    "    if direction == 'fwd':\n",
    "        direction = '+' #changes notation of the direction\n",
    "    if direction == 'rc':\n",
    "        direction = '-'  #changes notation of the direction\n",
    "    \n",
    "    #Specially process those with offsets > 0\n",
    "    if offset > 0 or prev_offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "    #Those with BLAT offsets = 0\n",
    "    else:\n",
    "        agp_start, agp_end = 1, length\n",
    "        contig_start, contig_end = 1, length\n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "########################################################################\n",
    "def process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction):\n",
    "    if agp_prev_overlap < 5 and agp_prev_overlap > 0:\n",
    "        agp_prev_overlap = 0\n",
    "        \n",
    "    if prev_offset ==0  and offset > 0: #contig left of offset\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "\n",
    "    if prev_offset > 0 and offset == 0: #contig right of offset\n",
    "        agp_start = agp_end + 500 + 2 #compensate for added gap between the two which has length = 500\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap  \n",
    "            \n",
    "    if prev_offset > 0 and offset > 0:\n",
    "        print('This type of alignment needs to be readdressed...\\n')\n",
    "        sys.exit()\n",
    "        \n",
    "        \n",
    "    info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "\n",
    "########################################################################\n",
    "def process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = posDict[i-1][0], int(posDict[i-1][6]),int(posDict[i-1][7])\n",
    "    \n",
    "    #Specially address those with BLAT offsets (unaligned sequence at the junctions)\n",
    "    if prev_offset > 0 or offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "        return info\n",
    "    \n",
    "    #Junctions without unaligned sequence(s):\n",
    "    else:\n",
    "        if agp_prev_overlap < 5:\n",
    "            agp_prev_overlap = 0\n",
    "\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "\n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "        return info       \n",
    "        \n",
    "########################################################################  \n",
    "def process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    gap_length = 10000\n",
    "    agp_start = agp_end + 1\n",
    "    agp_end = agp_start + gap_length\n",
    "    if 'CTG' in contigID:\n",
    "        gap_type = 'contig'\n",
    "    else:#To change once we add more types of contigs in here\n",
    "        gap_type = 'OTHER'\n",
    "    agp_prev_overlap = 0\n",
    "    \n",
    "    info = [agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap]\n",
    "    return info  \n",
    "\n",
    "########################################################################  \n",
    "def write_AGP_header(agpFile): #alter for each assembly\n",
    "    agpFile.write('##agp-version 1.0\\n# ORGANISM: Canis lupus familiaris\\n# TAX_ID: 9615\\n')\n",
    "    agpFile.write('# ASSEMBLY NAME: Zoey_v1\\n# ASSEMBLY DATE: 19-April-2017\\n')\n",
    "    agpFile.write('# GENOME CENTER: University of Michigan - J.M. Kidd Lab\\n')\n",
    "    agpFile.write('# DESCRIPTION: AGP specifying the assembly of chromosome 18 from primary PacBio contigs from FALCON assembly\\n')      \n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = agp_chrom, 0, 0\n",
    "    \n",
    "    if direction == 'fwd':\n",
    "        direction = '+' #changes notation of the direction\n",
    "    if direction == 'rc':\n",
    "        direction = '-'  #changes notation of the direction\n",
    "    \n",
    "    #Specially process those with offsets > 0\n",
    "    if offset > 0 or prev_offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "    #Those with BLAT offsets = 0\n",
    "    else:\n",
    "        agp_start, agp_end = 1, length\n",
    "        contig_start, contig_end = 1, length\n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "\n",
    "########################################################################\n",
    "def process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction):\n",
    "    if agp_prev_overlap < 5 and agp_prev_overlap > 0:\n",
    "        agp_prev_overlap = 0\n",
    "        \n",
    "    if prev_offset ==0  and offset > 0: #contig left of offset\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "\n",
    "    if prev_offset > 0 and offset == 0: #contig right of offset\n",
    "        agp_start = agp_end + 500 + 1 #compensate for added gap between the two which has length = 500\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap  \n",
    "            \n",
    "    if prev_offset > 0 and offset > 0:\n",
    "        print('This type of alignment needs to be readdressed...\\n')\n",
    "        sys.exit()\n",
    "        \n",
    "        \n",
    "    info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "\n",
    "########################################################################\n",
    "def process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = posDict[i-1][0], int(posDict[i-1][6]),int(posDict[i-1][7])\n",
    "    \n",
    "    #Specially address those with BLAT offsets (unaligned sequence at the junctions)\n",
    "    if prev_offset > 0 or offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "        return info\n",
    "    \n",
    "    #Junctions without unaligned sequence(s):\n",
    "    else:\n",
    "        if agp_prev_overlap < 5:\n",
    "            agp_prev_overlap = 0\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "        \n",
    "        shift = contig_end - contig_start\n",
    "\n",
    "        #now calculate the AGP coordinates of where the contig goes\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + shift #+ 1\n",
    "        \n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "        return info       \n",
    "        \n",
    "########################################################################  \n",
    "def process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    gap_length = 10000\n",
    "    agp_start = agp_end + 1\n",
    "    agp_end = agp_start + gap_length - 1\n",
    "    if 'CTG' in contigID:\n",
    "        gap_type = 'contig'\n",
    "    else:#To change once we add more types of contigs in here\n",
    "        gap_type = 'OTHER'\n",
    "    agp_prev_overlap = 0\n",
    "    \n",
    "    info = [agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap]\n",
    "    return info  \n",
    "\n",
    "########################################################################  \n",
    "def write_AGP_header(agpFile): #alter for each assembly\n",
    "    agpFile.write('##agp-version 1.0\\n# ORGANISM: Canis lupus familiaris\\n# TAX_ID: 9615\\n')\n",
    "    agpFile.write('# ASSEMBLY NAME: Zoey_v1\\n# ASSEMBLY DATE: 19-April-2017\\n')\n",
    "    agpFile.write('# GENOME CENTER: University of Michigan - J.M. Kidd Lab\\n')\n",
    "    agpFile.write('# DESCRIPTION: AGP specifying the assembly of chromosome 18 from primary PacBio contigs from FALCON assembly\\n')      \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now putting the information in the curated contig set (contigs constituting the golden path) into the coordinates of Zoey, and adding gap positions, if present. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DONE!!\n"
     ]
    }
   ],
   "source": [
    "#Defining AGP outfile\n",
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.txt', 'w')\n",
    "\n",
    "#Now putting them in the coordinates of the Zoey genome!! :) \n",
    "count, agp = 0, [] \n",
    "\n",
    "for i in posDict:\n",
    "    count += 1\n",
    "    agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig = posDict[i][0:9]\n",
    "    #print('\\n#',contigID)\n",
    "    \n",
    "    #Processing first contig in dataset -OR- on a new chromosome\n",
    "    if count == 1 or agp_chrom != agp_prev_chrom: \n",
    "        count = 1\n",
    "        info = process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)\n",
    "        agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction, offset = info[0:10]\n",
    "        agp.append([agp_prev_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "        #print(agp_prev_chrom,agp_start, agp_end, contig_start,contig_end,direction,agp_prev_overlap)\n",
    "    \n",
    "    #Process next contig (not first contig on chrom)\n",
    "    else:\n",
    "        if overlap >= 0: #if contigs overlap --> NO GAP!\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            if offset > 0:\n",
    "                #Add contig to the left of the gap\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "                #print(agp_prev_chrom,agp_start, agp_end, contig_start,contig_end,direction,agp_prev_overlap)\n",
    "                #Add introduced gap from unaligned sequences from contig-contig junction BLAT\n",
    "                count += 1\n",
    "                agp.append([agp_chrom, agp_end+1, agp_end+499, count, 'U', '500','BLAT_gap','no','na'])\n",
    "                #print(agp_chrom, agp_end+1, agp_end+501, count, 'U', '500','BLAT_gap','no','na')\n",
    "            else:\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])         \n",
    "                #print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "        else: #if contigs do not overlap --> GAP!\n",
    "            #process contig to the left of the gap\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "            #print(agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction)\n",
    "            \n",
    "            #processing gap\n",
    "            count += 1\n",
    "            gap_info = process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap = gap_info[0:7]\n",
    "            agp.append([agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na'])\n",
    "            #print(agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na')\n",
    "    \"\"\"if count > 5:\n",
    "        break\"\"\"\n",
    "\n",
    "#Write AGP header lines\n",
    "write_AGP_header(agpFile)\n",
    "\n",
    "#Write out to agp File the information for contigs/gaps\n",
    "for i in range(0,len(agp)):\n",
    "    agpFile.write(\"\\t\".join(map(str,agp[i])) + '\\n')\n",
    "agpFile.close()\n",
    "logFile.close()\n",
    "\n",
    "print('DONE!!')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\"\"\"#Defining AGP outfile\n",
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.txt', 'w')\n",
    "\n",
    "#Now putting them in the coordinates of the Zoey genome!! :) \n",
    "count, agp = 0, [] \n",
    "\n",
    "for i in posDict:\n",
    "    count += 1\n",
    "    agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig = posDict[i][0:9]\n",
    "    #print('\\n#',contigID)\n",
    "    #print(posDict[i])\n",
    "    \n",
    "    #print(count,agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)    \n",
    "    \n",
    "    #Processing first contig in dataset -OR- on a new chromosome\n",
    "    if count == 1 or agp_chrom != agp_prev_chrom: \n",
    "        count = 1\n",
    "        info = process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)\n",
    "        agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction, offset = info[0:10]\n",
    "        agp.append([agp_prev_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "        #print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "    \n",
    "    #Process next contig (not first contig on chrom)\n",
    "    else:\n",
    "        if overlap > 0: #if contigs overlap --> NO GAP!\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            if offset > 0:\n",
    "                #Add contig to the left of the gap\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "                #print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "                #Add introduced gap from unaligned sequences from contig-contig junction BLAT\n",
    "                count += 1\n",
    "                agp.append([agp_chrom, agp_end+1, agp_end+501, count, 'U', '500','BLAT_gap','no','na'])\n",
    "                #print(agp_chrom, agp_end+1, agp_end+501, count, 'U', '500','BLAT_gap','no','na')\n",
    "            else:\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])         \n",
    "                #print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "        else: #if contigs do not overlap --> GAP!\n",
    "            #process contig to the left of the gap\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "            #print(agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction)\n",
    "            \n",
    "            #processing gap\n",
    "            count += 1\n",
    "            gap_info = process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap = gap_info[0:7]\n",
    "            agp.append([agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na'])\n",
    "            #print(agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na')\n",
    "    #if count > 15:\n",
    "    #    break\n",
    "\n",
    "#Write AGP header lines\n",
    "write_AGP_header(agpFile)\n",
    "\n",
    "#Write out to agp File the information for contigs/gaps\n",
    "for i in range(0,len(agp)):\n",
    "    agpFile.write(\"\\t\".join(map(str,agp[i])) + '\\n')\n",
    "agpFile.close()\n",
    "logFile.close()\n",
    "\n",
    "print('DONE!!')\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['chr18', 1, 24712, 1, 'D', 'CTG-1462', 1, 24712, '-']\n",
      "['chr18', 24713, 3988291, 2, 'D', 'CTG-0201', 1, 3963579, '-']\n",
      "['chr18', 3988292, 3999716, 3, 'D', 'CTG-1861', 1730, 13154, '+']\n",
      "['chr18', 3999717, 4678554, 4, 'D', 'CTG-0743', 2638, 681475, '+']\n",
      "['chr18', 4678555, 5292060, 5, 'D', 'CTG-0770', 1, 613506, '-']\n",
      "['chr18', 5292061, 9634081, 6, 'D', 'CTG-0177', 1, 4342021, '-']\n",
      "['chr18', 9634082, 10274654, 7, 'D', 'CTG-0767', 1, 640573, '-']\n",
      "['chr18', 10274655, 11350765, 8, 'D', 'CTG-0615', 1, 1076111, '-']\n",
      "['chr18', 11350766, 11351251, 9, 'D', 'CTG-1278', 37188, 37673, '+']\n",
      "['chr18', 11351252, 13500930, 10, 'D', 'CTG-0400', 1, 2149679, '-']\n",
      "['chr18', 13500931, 13510930, 11, 'U', 10000, 'contig', 'no', 'na']\n",
      "['chr18', 13510931, 14860277, 12, 'D', 'CTG-0564', 1, 1349347, '-']\n",
      "['chr18', 14860278, 14870277, 13, 'U', 10000, 'contig', 'no', 'na']\n",
      "['chr18', 14870278, 17952979, 14, 'D', 'CTG-0274', 1, 3082702, '-']\n",
      "['chr18', 17952980, 17962979, 15, 'U', 10000, 'contig', 'no', 'na']\n",
      "['chr18', 17962980, 18358008, 16, 'D', 'CTG-0878', 1, 395029, '-']\n",
      "['chr18', 18358009, 18368008, 17, 'U', 10000, 'contig', 'no', 'na']\n"
     ]
    }
   ],
   "source": [
    "for i in range(0,len(agp)):\n",
    "    print(agp[i])\n",
    "    if i>15:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###################################################################################################\n",
    "def extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID + '/' + contigID + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID,extract_coord1,extract_coord2,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "###################################################################################################\n",
    "def extract_filledGap_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = gap_fastaRoot + '/' + contigID + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID,extract_coord1,extract_coord2,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "\n",
    "###################################################################################################    \n",
    "def reverse_comp_contig_fasta(wkDir):\n",
    "    cmd = 'fastarevcomp %stemp/contig1.fa > %stemp/contig1.fa.rc' % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    cmd = 'mv %stemp/contig1.fa.rc %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True) \n",
    "###################################################################################################    \n",
    "def read_in_fasta(wkDir):    \n",
    "    inFile = open(wkDir + 'temp/seq.fa', 'r')\n",
    "    for line in inFile:\n",
    "        line=line.rstrip()\n",
    "        if '>' in line:\n",
    "            continue\n",
    "        else:\n",
    "            seq.append(line)\n",
    "    inFile.close()\n",
    "    return seq\n",
    "\n",
    "###################################################################################################    \n",
    "def reformat_fasta_file(wkDir,CHROM):\n",
    "    agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v2_' + CHROM + '.fa', 'r')\n",
    "    tmpFile = open(wkDir + 'results/tmp.fa', 'w')\n",
    "\n",
    "    for line in agp_Fasta:\n",
    "        if '>' in line:\n",
    "            tmpFile.write(line)#Writes chrom ID\n",
    "            continue\n",
    "        seq=line.rstrip()\n",
    "        \n",
    "        start = 0\n",
    "        fasta_length = float(len(seq))\n",
    "        fasta_line_length = 80\n",
    "        Max = int(fasta_length/fasta_line_length)\n",
    "\n",
    "        for i in range(0,Max+1):\n",
    "            Line = seq[start:start+fasta_line_length]\n",
    "            start = start + fasta_line_length\n",
    "            tmpFile.write(Line + '\\n')\n",
    "            \"\"\"if i > 10:\n",
    "                break\"\"\"    \n",
    "\n",
    "    agp_Fasta.close()\n",
    "    tmpFile.close()\n",
    "    \n",
    "    cmd = 'mv %sresults/tmp.fa %sresults/AGP_Zoey_Assembly_v2_%s.fa'  % (wkDir,wkDir,CHROM)\n",
    "    subprocess.call(cmd,shell=True) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\"\"\"###################################################################################################\n",
    "def extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID + '/' + contigID + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID,extract_coord1,extract_coord2,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "###################################################################################################    \n",
    "def reverse_comp_contig_fasta(wkDir):\n",
    "    cmd = 'fastarevcomp %stemp/contig1.fa > %stemp/contig1.fa.rc' % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    cmd = 'mv %stemp/contig1.fa.rc %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True) \n",
    "###################################################################################################    \n",
    "def read_in_fasta(wkDir):    \n",
    "    inFile = open(wkDir + 'temp/seq.fa', 'r')\n",
    "    for line in inFile:\n",
    "        line=line.rstrip()\n",
    "        if '>' in line:\n",
    "            continue\n",
    "        else:\n",
    "            seq.append(line)\n",
    "    inFile.close()\n",
    "    return seq\n",
    "\n",
    "###################################################################################################    \n",
    "def reformat_fasta_file(wkDir,CHROM):\n",
    "    agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.fa', 'r')\n",
    "    tmpFile = open(wkDir + 'results/tmp.fa', 'w')\n",
    "\n",
    "    for line in agp_Fasta:\n",
    "        if '>' in line:\n",
    "            tmpFile.write(line)#Writes chrom ID\n",
    "            continue\n",
    "        seq=line.rstrip()\n",
    "\n",
    "        start = 0\n",
    "        fasta_length = float(len(seq))\n",
    "        fasta_line_length = 80\n",
    "        Max = int(fasta_length/fasta_line_length)\n",
    "\n",
    "        for i in range(0,Max+1):\n",
    "            Line = seq[start:start+fasta_line_length]\n",
    "            start = start + fasta_line_length\n",
    "            tmpFile.write(Line + '\\n')\n",
    "            #if i > 10:\n",
    "                #break   \n",
    "\n",
    "    agp_Fasta.close()\n",
    "    tmpFile.close()\n",
    "\n",
    "    cmd = 'cp %sresults/tmp.fa %sresults/AGP_Zoey_Assembly_v1_%s.fa'  % (wkDir,wkDir,CHROM)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True) \"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished writing out the sequence for 74 contigs and gaps to Zoey fasta file\n"
     ]
    }
   ],
   "source": [
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.txt', 'r')\n",
    "fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.fa', 'w')\n",
    "\n",
    "gap_fastaRoot = wkDir + 'input/filled_gaps/' + CHROM + '/'\n",
    "\n",
    "contigID, count, prev_chrom = '', 0, ''\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line:\n",
    "        continue\n",
    "    count+= 1\n",
    "    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID = line[5]\n",
    "    \n",
    "    seq = []\n",
    "    \n",
    "    if 'CTG' in contigID or 'chr' in contigID:\n",
    "        chrom, contigID, extract_coord1, extract_coord2, orient = line[0],line[5],int(line[6]),int(line[7]), line[8]\n",
    "        if 'CTG' in contigID:\n",
    "            #Extract fasta\n",
    "            extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2)\n",
    "        if 'chr' in contigID:\n",
    "            extract_filledGap_fasta(gap_fastaRoot,contigID,orient,extract_coord1,extract_coord2)\n",
    "        #Reverse complement extracted FASTA if needed\n",
    "        if '-' == orient:\n",
    "            reverse_comp_contig_fasta(wkDir)\n",
    "        else:\n",
    "            cmd = 'mv %stemp/contig1.fa %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "            #print(cmd)\n",
    "            subprocess.call(cmd,shell=True) \n",
    "        \n",
    "        seq = read_in_fasta(wkDir)\n",
    "    else: #These are gaps   \n",
    "        gap_length = int(line[5])\n",
    "        seq = 'N' * gap_length\n",
    "    \n",
    "    #Writes '>' chromosome to the fasta file\n",
    "    if count == 1 or prev_chrom != chrom:\n",
    "        if count > 1:\n",
    "            agp_Fasta.write('\\n')\n",
    "        agp_Fasta.write('>' + chrom + '\\n')\n",
    "    \n",
    "    for i in range(0,len(seq)):\n",
    "        Seq = seq[i].upper()#.upper()print(seq[i])\n",
    "        agp_Fasta.write(Seq)\n",
    "\n",
    "    prev_chrom = chrom #reset prev_chrom identity\n",
    "\n",
    "\n",
    "print('Finished writing out the sequence for %i contigs and gaps to Zoey fasta file' % count)\n",
    "agpFile.close()\n",
    "agp_Fasta.close()\n",
    "\n",
    "#Reformat the fasta file to only have certain # of nucleotides per line\n",
    "reformat_fasta_file(wkDir,CHROM)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "\"\"\"\"agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.txt', 'r')\n",
    "fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.fa', 'w')\n",
    "\n",
    "contigID, count, prev_chrom = '', 0, ''\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line:\n",
    "        continue\n",
    "    count+= 1\n",
    "    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID = line[5]\n",
    "    \n",
    "    seq = []\n",
    "    \n",
    "    if 'CTG' in contigID:\n",
    "        chrom, contigID, extract_coord1, extract_coord2, orient = line[0],line[5],int(line[6]),int(line[7]), line[8]\n",
    "        #Extract fasta\n",
    "        extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2)\n",
    "    \n",
    "        #Reverse complement extracted FASTA if needed\n",
    "        if '-' == orient:\n",
    "            reverse_comp_contig_fasta(wkDir)\n",
    "        else:\n",
    "            cmd = 'mv %stemp/contig1.fa %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "            #print(cmd)\n",
    "            subprocess.call(cmd,shell=True) \n",
    "        \n",
    "        seq = read_in_fasta(wkDir)\n",
    "    else: #These are gaps   \n",
    "        gap_length = int(line[5])\n",
    "        seq = 'N' * gap_length\n",
    "    \n",
    "    #Writes '>' chromosome to the fasta file\n",
    "    if count == 1 or prev_chrom != chrom:\n",
    "        if count > 1:\n",
    "            agp_Fasta.write('\\n')\n",
    "        agp_Fasta.write('>' + chrom + '\\n')\n",
    "    \n",
    "    for i in range(0,len(seq)):\n",
    "        Seq = seq[i].upper()#.upper()print(seq[i])\n",
    "        agp_Fasta.write(Seq)\n",
    "        #agp_Fasta.write(seq[i])\n",
    "\n",
    "    prev_chrom = chrom #reset prev_chrom identity\n",
    "\n",
    "\n",
    "print('Finished writing out the sequence for %i contigs and gaps to Zoey fasta file' % count)\n",
    "agpFile.close()\n",
    "agp_Fasta.close()\n",
    "\n",
    "#Reformat the fasta file to only have certain # of nucleotides per line\n",
    "reformat_fasta_file(wkDir,CHROM)\"\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Have generated the golden path file. Below, I'm calculating statistics on the AGP. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "The below cell calculates the continuity on the given chromosome from Zoey's AGP"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##Counts\n",
      "50 contigs are in the Golden Path for Zoey\n",
      "24 gaps remain in the Golden Path for Zoey\n",
      "Of these 24 gaps, 1 gap(s) correspond(s) to BLAT gaps (unaligned sequences where gaps were introduced)\n",
      "\n",
      "##Statistics\n",
      "Average continuity (bp):  2230659.96\n"
     ]
    }
   ],
   "source": [
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1_' + CHROM + '.txt', 'r')\n",
    "\n",
    "contig_count,gap_count,line_count,blat_gap_count = 0,0,0,0\n",
    "prev_Type,start_stop_list,contig_start = '',[],1\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line: #skips header\n",
    "        continue    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    line_count += 1\n",
    "    chrom,start,end,num,Type = line[0], int(line[1]),int(line[2]),int(line[3]),line[4]\n",
    "    if 'D' in Type:\n",
    "        #this is a contig\n",
    "        contig_count += 1\n",
    "        if 'U' in prev_Type:\n",
    "            contig_start = start\n",
    "    if 'U' in Type:\n",
    "        #this is a gap\n",
    "        gap_count += 1\n",
    "        if 'BLAT' in line[6]:\n",
    "            blat_gap_count+=1\n",
    "        #print(start,end)\n",
    "        if 'D' in prev_Type:\n",
    "            contig_end = start - 1\n",
    "            start_stop_list.append([contig_start,contig_end])\n",
    "    prev_Type = Type\n",
    "    \n",
    "print('##Counts\\n%i contigs are in the Golden Path for Zoey' % contig_count)\n",
    "print('%i gaps remain in the Golden Path for Zoey' % gap_count)\n",
    "print('Of these %i gaps, %i gap(s) correspond(s) to BLAT gaps (unaligned sequences where gaps were introduced)' %(gap_count,blat_gap_count))\n",
    "\n",
    "continuous_contig_array = []\n",
    "for i in range(0,len(start_stop_list)):\n",
    "    start,end = int(start_stop_list[i][0]),int(start_stop_list[i][1])\n",
    "    length = end-start\n",
    "    continuous_contig_array.append(length)\n",
    "\n",
    "Average_continuity = format(np.mean(continuous_contig_array), '.2f') \n",
    "print('\\n##Statistics\\nAverage continuity (bp): ', Average_continuity)  \n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "In order to calculate continuity in CanFam3.1, we'll need to know the chromosome IDs and the chromosome lengths.\n",
    "\n",
    "The next cell stores the chromosome identifiers with the lengths of each chromosome."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'chr17': 64289059, 'chr4': 88276631, 'chr3': 91889043, 'chr26': 38964690, 'chr1': 122678785, 'chr28': 41182112, 'chr18': 55844845, 'chr6': 77573801, 'chr31': 39895921, 'chr38': 23914537, 'chr9': 61074082, 'chr29': 41845238, 'chr11': 74389097, 'chr34': 42124431, 'chr12': 72498081, 'chr25': 51628933, 'chr20': 58134056, 'chrM': 16727, 'chr16': 59632846, 'chr10': 69331447, 'chr8': 74330416, 'chr13': 63241923, 'chr21': 50858623, 'chr15': 64190966, 'chrX': 123869142, 'chr23': 52294480, 'chr7': 80974532, 'chr27': 45876710, 'chr2': 85426708, 'chr19': 53741614, 'chr36': 30810995, 'chr24': 47698779, 'chr33': 31377067, 'chr37': 30902991, 'chr22': 61439934, 'chr35': 26524999, 'chr30': 40214260, 'chr5': 88915250, 'chr32': 38810281, 'chr14': 60966679}\n"
     ]
    }
   ],
   "source": [
    "chrom_length_file = open(wkDir + 'input/cyto_band.bed','r')\n",
    "chrom_lengths = {}\n",
    "\n",
    "for line in chrom_length_file:\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,length = line[0],int(line[2])\n",
    "    if 'chrUn' in chrom:\n",
    "        continue\n",
    "    chrom_lengths[chrom] = length\n",
    "    \n",
    "print (chrom_lengths)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The following cell will store the gaps separately on each chromosome so that we can calculate continuity."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 19553 gaps in the canFam3 assembly (not including chrUn)\n",
      "Average continuity (bp):  118532.19\n"
     ]
    }
   ],
   "source": [
    "gapFile = open(wkDir + 'input/gaps.bed.sorted','r')\n",
    "#to keep track of the previous call\n",
    "prev_chrom, prev_start, prev_end = '', '', ''\n",
    "#save continuous stretches to array\n",
    "cont_pos_list, per_chrom, count =  [], {}, 0 \n",
    "\n",
    "\n",
    "for line in gapFile:\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,start,end=line[0],int(line[1]),int(line[2])\n",
    "    if 'chrUn' in chrom:\n",
    "        continue\n",
    "    #FOR TESTING\n",
    "    \"\"\"if 'chr18' not in chrom:\n",
    "        continue\"\"\"\n",
    "    count+=1\n",
    "\n",
    "    #If first line in file\n",
    "    if count == 1:\n",
    "        per_chrom[chrom] = []\n",
    "        if start == 0: #gap is at the beginning of the chromosome\n",
    "            #Save for next\n",
    "            cont_start = end\n",
    "            prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "            continue\n",
    "        #first continuous stretch on chromosome\n",
    "        cont_start = 0\n",
    "        cont_end = start - 1\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        \n",
    "        #next continuous stretch, define start\n",
    "        cont_start = end\n",
    "        #Save for next\n",
    "        prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "        continue\n",
    "    #If the next call is on a different chromosome, must save the previous gap's info\n",
    "    if chrom not in prev_chrom:\n",
    "        per_chrom[chrom] = []\n",
    "        #previous continuous stretch\n",
    "        cont_end = chrom_lengths[prev_chrom]\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        #first continuous stretch on new chromosome\n",
    "        cont_start = 0\n",
    "        cont_end = start - 1\n",
    "        #next adjacent stretch, define start\n",
    "        cont_start = end      \n",
    "    else:\n",
    "        cont_end = start - 1\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        #next adjacent strech, define start\n",
    "        cont_start = end\n",
    "        \n",
    "    prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "    \n",
    "\n",
    "print('There are %i gaps in the canFam3 assembly (not including chrUn)' % count)\n",
    "gapFile.close()\n",
    "\n",
    "\n",
    "#Calculating the continuity statistics\n",
    "continuous_contig_array = []\n",
    "for i in range(0,len(cont_pos_list)):\n",
    "    start,end = int(cont_pos_list[i][1]),int(cont_pos_list[i][2])\n",
    "    length = end-start\n",
    "    continuous_contig_array.append(length)    \n",
    "Average_continuity = format(np.mean(continuous_contig_array), '.2f')\n",
    "print('Average continuity (bp): ', Average_continuity)  \n",
    "\n",
    "\n",
    "    \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The following cell will calculate the number of gaps per chromosome in the CanFam3.1 assembly as well as the average continuity value per chromosome."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Chromosome\tGap_Count\tAverageContinuity\n",
      "chr17 513 124429.61\n",
      "chr38 308 79114.00\n",
      "chr3 707 129768.72\n",
      "chr29 264 157243.09\n",
      "chr1 1121 108669.12\n",
      "chr28 395 103803.70\n",
      "chr18 589 94481.08\n",
      "chr5 892 98934.37\n",
      "chr6 786 98439.18\n",
      "chr4 622 141570.59\n",
      "chr36 254 120487.31\n",
      "chr11 533 139018.32\n",
      "chr34 297 140893.48\n",
      "chr12 482 150218.18\n",
      "chr24 455 104445.93\n",
      "chr31 288 138703.69\n",
      "chr20 687 83932.28\n",
      "chr16 442 133897.94\n",
      "chr10 744 92985.91\n",
      "chr33 264 118442.50\n",
      "chr13 488 129005.97\n",
      "chr21 374 135844.89\n",
      "chr15 637 100647.08\n",
      "chrX 1032 119702.28\n",
      "chr23 308 169341.04\n",
      "chr7 607 132802.75\n",
      "chr8 583 126900.71\n",
      "chr27 393 116399.34\n",
      "chr2 828 102654.87\n",
      "chr19 304 177014.63\n",
      "chr25 384 133718.28\n",
      "chr37 327 92451.86\n",
      "chr22 371 165163.49\n",
      "chr35 211 126727.09\n",
      "chr30 372 107145.10\n",
      "chr9 764 79047.59\n",
      "chr26 391 99154.57\n",
      "chr32 190 202505.99\n",
      "chr14 345 175115.68\n"
     ]
    }
   ],
   "source": [
    "#Continuity values per chromosome\n",
    "print('Chromosome\\tGap_Count\\tAverageContinuity')\n",
    "for i in per_chrom:\n",
    "    chrom = i\n",
    "    continuous_contig_array = []\n",
    "    for j in per_chrom[chrom]:\n",
    "        start = int(j[1])\n",
    "        end = int(j[2])\n",
    "        length = end - start\n",
    "        continuous_contig_array.append(length)\n",
    "    Average_continuity = format(np.mean(continuous_contig_array), '.2f')\n",
    "    print(chrom, len(per_chrom[chrom]),Average_continuity )"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
