{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#2017-04-25\n",
    "#A. Pendleton\n",
    "#This script is used to make an AGP file for Zoey. Can be built upon as new forms\n",
    "#   of contigs become assembled and incorporated into the AGP.\n",
    "\n",
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "\n",
    "#import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n",
    "import scipy \n",
    "import matplotlib.patches as patches\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#This script is used to make an AGP file for Zoey. Can be built upon as new contigs become assembled and incorporated into the available set for assembly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Current working directory is:\n",
      " /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/\n",
      "Writing out AGP to file:\n",
      " <_io.TextIOWrapper name='/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/results/AGP_Zoey_Assembly_v1.txt' mode='w' encoding='UTF-8'>\n"
     ]
    }
   ],
   "source": [
    "wkDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/'\n",
    "print('Current working directory is:\\n', wkDir)\n",
    "\n",
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'w')\n",
    "print('Writing out AGP to file:\\n', agpFile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Identified coordinates for 11 primary contigs\n"
     ]
    }
   ],
   "source": [
    "#####READING IN PRIMARY CONTIG COORDINATES FROM MUMMER ALIGNMENT#######\n",
    "#Primary contig alignments (no canu processing)\n",
    "primaryContigCoord = open(wkDir + 'input/primary.2017-04-10.txt', 'r')\n",
    "primaryContigCoordBed = open(wkDir + 'input/primary.2017-04-10.bed', 'w')\n",
    "\n",
    "contigDict,index = {}, 0\n",
    "\n",
    "for line in primaryContigCoord:\n",
    "    if line.startswith(\"#\") is True: #skips header line\n",
    "        continue\n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID, length, Dir, chrom, start, end = line[0],line[1],line[2],line[3],int(line[4]),int(line[5])    \n",
    "    if 'chr18' not in chrom: #FILTERING ONLY FOR CHROM 18 FOR NOW!!! REMOVE LATER \n",
    "        continue\n",
    "    index += 1\n",
    "    contigDict[index] = [chrom,start,end,contigID,Dir,length]\n",
    "    primaryContigCoordBed.write('%s\\t%i\\t%i\\t%s\\n' % (chrom,int(start-1),end,contigID))\n",
    "    if index > 10: #FOR TESTING--- REMOVE LATER\n",
    "        break\n",
    "    \n",
    "primaryContigCoord.close()\n",
    "primaryContigCoordBed.close()\n",
    "print ('Identified coordinates for %i primary contigs' % len(contigDict))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step 1. Read in coordinates from alignment file based on Mummer of primary contigs (per Jeff)\n",
    "Step 2. BLAT each contig against the contig proximal to that proximal to it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_line(Dict,curr_index,i):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[curr_index][0],int(Dict[curr_index][1]),int(Dict[curr_index][2]), Dict[curr_index][3], Dict[curr_index][4], int(Dict[curr_index][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i+1][0],int(Dict[i+1][1]),int(Dict[i+1][2]), Dict[i+1][3], Dict[i+1][4], int(Dict[i+1][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def read_last_line(Dict):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[i-1][0],int(Dict[i-1][1]),int(Dict[i-1][2]), Dict[i-1][3], Dict[i-1][4], int(Dict[i-1][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i][0],int(Dict[i][1]),int(Dict[i][2]), Dict[i][3], Dict[i][4], int(Dict[i][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def process_last_contig(contigDict,curr_index,i):\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_last_line(contigDict)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,i-1,contigID2,i))\n",
    "    if start1 == start2 or end1 == end2:\n",
    "        return\n",
    "    if end2 > end1:\n",
    "        curr_index+=1 \n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        print('PASS - Last contig on chromosome overlap or is spaced correctly')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_first_contig(contigDict,curr_index,i):\n",
    "    curr_index+=1\n",
    "    #Always save first contig in the dictionary/dataset\n",
    "    curated_contigDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "    print('PASS - First call in dataset')\n",
    "    return curr_index \n",
    "###################################################################################################\n",
    "def process_same_starts(contigDict,curr_index,i):\n",
    "    if start1 == start2 and end1 > end2:\n",
    "        print('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        return\n",
    "    if start1 == start2 and end1 < end2: #overwrite previous position if contig 2 is longer than contig 1\n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        print('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_same_ends(contigDict,curr_index,i):    \n",
    "    if start1 < start2 and end1 == end2:\n",
    "        print('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        return \n",
    "    if start2 < start1 and end1 == end2: \n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#CTG-1462 (1) -- CTG-0201 (2)\n",
      "PASS - First call in dataset\n",
      "\n",
      "#CTG-0201 (2) -- CTG-2401 (3)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0201\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', '3968428']\n",
      "['chr18', 28961, 33381, 'CTG-2401', 'fwd', '4387']\n",
      "\n",
      "#CTG-0201 (2) -- CTG-1986 (4)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0201\n",
      "['chr18', 28961, 33381, 'CTG-2401', 'fwd', '4387']\n",
      "['chr18', 1781533, 1791763, 'CTG-1986', 'rc', '10221']\n",
      "\n",
      "#CTG-0201 (2) -- CTG-1861 (5)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1861 (5) -- CTG-0743 (6)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0743 (6) -- CTG-0770 (7)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0770 (7) -- CTG-0177 (8)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0177 (8) -- CTG-0767 (9)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0767 (9) -- CTG-0615 (10)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0615 (10) -- CTG-1278 (11)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "Identified CURATED coordinates for 9 primary contigs\n"
     ]
    }
   ],
   "source": [
    "curated_contigDict, curr_index = {}, 1\n",
    "chrom1,chrom2 = '0', '0' #initializing values\n",
    "\n",
    "\n",
    "for key, value in contigDict.items(): \n",
    "    i = int(key)\n",
    "    #1. If last contig in dataset \n",
    "    if i == len(contigDict):\n",
    "        continue\n",
    "    \n",
    "    #2. If contigs are on different chromosomes:\n",
    "    if chrom1 != chrom2:\n",
    "        process_last_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(contigDict,curr_index,i)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,curr_index,contigID2,i+1))\n",
    "\n",
    "    #3. Automatically saves first contig in dataset\n",
    "    if curr_index == 1 and len(curated_contigDict) == 0:\n",
    "        curr_index = process_first_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "    \n",
    "    #4. If contigs have the same start site, choose the longest contig\n",
    "    if start1 == start2:\n",
    "        process_same_starts(contigDict,curr_index,i)\n",
    "        continue\n",
    "\n",
    "    #5. If contigs have same end coordinate, choose longest contig (i.e. contig #1 if sorted)\n",
    "    if end1 == end2: \n",
    "        process_same_ends(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    #6. If contigs #1 and #2 have same start AND end, keep contig #1\n",
    "    if start1 == start2 and end1 == end2:\n",
    "        print('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        continue #because contig1 is already in the dictionary\n",
    "        \n",
    "    #7. If contig #2 is fully within contig #1 -- continue\n",
    "    if start1 < start2 and end1 > end2: # contig2 is fully within contig1\n",
    "        curated_contigDict[curr_index] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "        print('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        continue\n",
    "    if start1 > start2 and end1 < end2:\n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        sys.exit(1)\n",
    "        \n",
    "    #8. If contig passes all these - then automatically saves\n",
    "    curr_index=i+1\n",
    "    curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    #curated_contigDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "    print('PASS - contigs overlap or are spaced correctly')\n",
    "\n",
    "print ('Identified CURATED coordinates for %i primary contigs' % len(curated_contigDict))\n",
    "   \n",
    "#Re-naming dictionary keys\n",
    "count=0\n",
    "for keys in curated_contigDict:\n",
    "    count+=1\n",
    "    curated_contigDict[count]=curated_contigDict.pop(keys)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Non-curated:\n",
      "1 ['chr18', 2619, 26992, 'CTG-1462', 'rc', '24712']\n",
      "2 ['chr18', 22143, 3966426, 'CTG-0201', 'rc', '3968428']\n",
      "3 ['chr18', 28961, 33381, 'CTG-2401', 'fwd', '4387']\n",
      "4 ['chr18', 1781533, 1791763, 'CTG-1986', 'rc', '10221']\n",
      "5 ['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', '13154']\n",
      "6 ['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', '681475']\n",
      "7 ['chr18', 4634379, 5281622, 'CTG-0770', 'rc', '639396']\n",
      "8 ['chr18', 5260259, 9645181, 'CTG-0177', 'rc', '4363384']\n",
      "9 ['chr18', 9638686, 10286737, 'CTG-0767', 'rc', '647068']\n",
      "10 ['chr18', 10284678, 11357644, 'CTG-0615', 'rc', '1078170']\n",
      "11 ['chr18', 11320457, 11367174, 'CTG-1278', 'fwd', '37673']\n",
      "\n",
      "Curated:\n",
      "1 ['chr18', 2619, 26992, 'CTG-1462', 'rc', 24712]\n",
      "2 ['chr18', 22143, 3966426, 'CTG-0201', 'rc', 3968428]\n",
      "3 ['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', 13154]\n",
      "4 ['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', 681475]\n",
      "5 ['chr18', 4634379, 5281622, 'CTG-0770', 'rc', 639396]\n",
      "6 ['chr18', 5260259, 9645181, 'CTG-0177', 'rc', 4363384]\n",
      "7 ['chr18', 9638686, 10286737, 'CTG-0767', 'rc', 647068]\n",
      "8 ['chr18', 10284678, 11357644, 'CTG-0615', 'rc', 1078170]\n",
      "9 ['chr18', 11320457, 11367174, 'CTG-1278', 'fwd', 37673]\n"
     ]
    }
   ],
   "source": [
    "print('Non-curated:')\n",
    "for i in contigDict:\n",
    "    print(i,contigDict[i])\n",
    "print( '\\nCurated:')\n",
    "for i in curated_contigDict:\n",
    "    print(i, curated_contigDict[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "metadata": {},
   "outputs": [],
   "source": [
    "def process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1):\n",
    "    chrom2 = chrom1\n",
    "    start2 = start1 - 10000\n",
    "    end2 = start1 + 10000\n",
    "    contigID2 = 'canFam3'\n",
    "    Dir2 = 'fwd'\n",
    "    length1 = end2-start2\n",
    "    \n",
    "    if start2 < 0:\n",
    "        start2 = 0\n",
    "    \n",
    "    #Determine overlap \n",
    "    determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    \n",
    "    #Find coordinates to extract\n",
    "    find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    \n",
    "    #Extract FASTA\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/genomes/canFam3.1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "###################################################################################################\n",
    "def determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    #How much do the contigs overlap?\n",
    "    global overlap\n",
    "    overlap = end1 - start2\n",
    "    print('\\nOverlap = ', overlap)\n",
    "    #OVERLAPPING CONTIGS\n",
    "    if overlap > 0:\n",
    "        find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    #DIRECTLY ADJACENT CONTIGS\n",
    "    if overlap == 0:\n",
    "        print('Contigs are directly adjacent to one another')\n",
    "        #WRITE FUNCTION FOR THESE\n",
    "    #CONTIGS WITH GAP BETWEEN THEM\n",
    "    if overlap < 0:\n",
    "        print('Gap between contigs')\n",
    "        #WRITE FUNCTION FOR THESE\n",
    "###################################################################################################\n",
    "def find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    ###Determine coordinates to extract for BLAT\n",
    "    global extract_coord1\n",
    "    global extract_coord2\n",
    "    #Contig #1\n",
    "    if 'fwd' in Dir1:\n",
    "        extract_coord1 = [length1 - overlap - 10000, length1]\n",
    "    else:\n",
    "        extract_coord1 = [0, overlap + 10000]\n",
    "    #Contig #2\n",
    "    if 'fwd' in Dir2:\n",
    "        extract_coord2 = [0, overlap + 10000] \n",
    "    else:\n",
    "        extract_coord2 = [length2 - overlap - 10000, length2]\n",
    "    \n",
    "    print ('Coordinates to extract for BLAT: ',extract_coord1,extract_coord2,'\\n')\n",
    "    #safety check\n",
    "    for i in range(0,1): #CHECKS \n",
    "        #= if the region to extract extends beyond the length of the contig, if so then the coordinate changes to the length of contig\n",
    "        if extract_coord1[i] > length1:\n",
    "            extract_coord1[i] = length1\n",
    "        if extract_coord2[i] > length2:\n",
    "            extract_coord2[i] = length2\n",
    "        # If the region extended too far, and the value is negative -- then make starting extraction coordinate = 0\n",
    "        if extract_coord1[i] < 0: \n",
    "            extract_coord1[i] = 0\n",
    "        if extract_coord2[i] < 0:\n",
    "            extract_coord2[i] = 0\n",
    "            \n",
    "    if overlap < 0: #### SEND TO DIFFERENT FUNCTION LATER -- FOLLOW OVERLAPPING CONTIGS FOR NOW\n",
    "        print('Contigs do not overlap')\n",
    "    \n",
    "    data = []\n",
    "    \n",
    "    #Where to find fasta of contigs\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1, contigID2,Dir2,extract_coord2)\n",
    "    \n",
    "###################################################################################################\n",
    "def extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID1 + '/' + contigID1 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID1,extract_coord1[0],extract_coord1[1],wkDir)\n",
    "    #print (cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    #Contig 2\n",
    "    fasta_path2 = fastaRoot + contigID2 + '/' + contigID2 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %stemp/contig2.fa' % (fasta_path2,contigID2,extract_coord2[0],extract_coord2[1],wkDir)\n",
    "    #print (cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "###################################################################################################\n",
    "def run_blat(wkDir):\n",
    "    blatcmd = 'blat %stemp/contig1.fa %stemp/contig2.fa %stemp/temp.blat' % (wkDir,wkDir,wkDir)\n",
    "    #print(blatcmd)\n",
    "    subprocess.call(blatcmd,shell=True)\n",
    "    \n",
    "    \n",
    "###################################################################################################\n",
    "def parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    inFile = open('%stemp/temp.blat' % (wkDir),'r')\n",
    "    lineCount, cleanAlignment, offset = 0, False, 0\n",
    "    for line in inFile:\n",
    "        line=line.rstrip().split('\\t')\n",
    "        lineCount+=1\n",
    "        if lineCount > 5:#skips the BLAT results headers\n",
    "            score,strand = int(line[0]),line[8]\n",
    "            percent_of_overlap = float(score)/overlap\n",
    "            if percent_of_overlap < float(0.75):\n",
    "                continue\n",
    "            print('\\n#Parsing BLAT results','Percent of overlap matched in BLAT: ',format(percent_of_overlap, '.3f'))\n",
    "            #CHECKS TO MAKE SURE WE DID THIS RIGHT - correct end vs correct end\n",
    "            if '+' not in strand:\n",
    "                print('ERROR: Top hit not in proper orientation.... SKIPPING -- PLEASE CHECK')\n",
    "                return\n",
    "            #parse contig #1 (left contig)\n",
    "            blat_length1, blat_start1, blat_end1 = int(line[14]),int(line[15]),int(line[16])\n",
    "            print(blat_length1, blat_start1, blat_end1)\n",
    "            #parse contig #2 (right contig)\n",
    "            blat_length2, blat_start2, blat_end2 = int(line[10]),int(line[11]),int(line[12])\n",
    "            print(blat_length2, blat_start2, blat_end2)\n",
    "            #Calculate offset  - to compensate for non-clean alignments (extension of left contig that does not overlap with adjacent contig)\n",
    "            calculate_offset(overlap,blat_length1, blat_start1, blat_end1,blat_length2, blat_start2, blat_end2)\n",
    "\n",
    "###################################################################################################\n",
    "def calculate_offset(overlap,blat_length1, blat_start1, blat_end1,blat_length2, blat_start2, blat_end2):\n",
    "    global offset\n",
    "    offset = blat_start1 - 0\n",
    "    print('\\nBLAT offset = ', offset)\n",
    "\n",
    "###################################################################################################\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "# CTG-1462 CTG-0201 \n",
      " chr18 2619 26992 CTG-1462 rc 24712 \n",
      " chr18 22143 3966426 CTG-0201 rc 3968428\n",
      "\n",
      "Overlap =  4849\n",
      "Coordinates to extract for BLAT:  [0, 14849] [3953579, 3968428] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.982\n",
      "14849 0 4806\n",
      "14850 10026 14849\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0201 CTG-1861 \n",
      " chr18 22143 3966426 CTG-0201 rc 3968428 \n",
      " chr18 3964697 3978047 CTG-1861 fwd 13154\n",
      "\n",
      "Overlap =  1729\n",
      "Coordinates to extract for BLAT:  [0, 11729] [0, 11729] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.995\n",
      "ERROR: Top hit not in proper orientation.... SKIPPING -- PLEASE CHECK\n"
     ]
    }
   ],
   "source": [
    "#write_AGP_header(agpFile)\n",
    "posDict = {}\n",
    "\n",
    "for i in range(1,len(curated_contigDict)-1): \n",
    "    curr_index=i\n",
    "    #1. Reading in coordinates from contigList\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(curated_contigDict, curr_index, i)\n",
    "    #2. Checking contigs are on same chromosome\n",
    "    if chrom1 != chrom2:# or i == 1: #This is first contig on chromosome, need to process it first\n",
    "        #continue\n",
    "        process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1) \n",
    "        continue\n",
    "    print ('\\n#',contigID1, contigID2,'\\n',chrom1,start1,end1,contigID1,Dir1,length1,'\\n',chrom2,start2,end2,contigID2,Dir2,length2)\n",
    "    \n",
    "    #3. Determines the overlap/orientation of the two contigs\n",
    "    determine_contig_overlap(end1,contigID1,Dir1,length1,end2,contigID2,Dir2,length2)\n",
    "    \n",
    "    #4. BLAT the properly oriented contig ends against one another\n",
    "    #     and parse the results\n",
    "    run_blat(wkDir)\n",
    "    parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "    \n",
    "    #save contig1 information to dictionary\n",
    "    posDict[i] = [chrom1,start1,end1,contigID1,Dir1,overlap,offset,contigID2]\n",
    "\n",
    "\n",
    "    if i > 1:\n",
    "        break\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['chr18', 2619, 26992, 'CTG-1462', 'rc', 4849, 0, 'CTG-0201']\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', 1729, 0, 'CTG-1861']\n",
      "['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', 2637, 10004, 'CTG-0743']\n",
      "['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', 25890, 10004, 'CTG-0770']\n",
      "['chr18', 4634379, 5281622, 'CTG-0770', 'rc', 21363, 0, 'CTG-0177']\n",
      "['chr18', 5260259, 9645181, 'CTG-0177', 'rc', 6495, 0, 'CTG-0767']\n",
      "['chr18', 9638686, 10286737, 'CTG-0767', 'rc', 2059, 0, 'CTG-0615']\n"
     ]
    }
   ],
   "source": [
    "for i in posDict:\n",
    "    print (posDict[i])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def write_AGP_header(agpFile): #alter for each assembly\n",
    "    agpFile.write('##agp-version 1.0\\n# ORGANISM: Canis lupus familiaris\\n# TAX_ID: 9615\\n')\n",
    "    agpFile.write('# ASSEMBLY NAME: Zoey_v1\\n# ASSEMBLY DATE: 19-April-2017\\n')\n",
    "    agpFile.write('# GENOME CENTER: U. Michigan - Kidd Lab\\n')\n",
    "    agpFile.write('# DESCRIPTION: AGP specifying the assembly of chromosome 18 from primary PacBio contigs from FALCON assembly\\n')"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
