{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#2017-05-05\n",
    "#A. Pendleton\n",
    "#This script is used to make an AGP file for Zoey. Can be built upon as new forms\n",
    "#   of contigs become assembled and incorporated into the AGP.\n",
    "\n",
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "\n",
    "#import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n",
    "import scipy \n",
    "import matplotlib.patches as patches\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#This script is used to make an AGP file for Zoey. Can be built upon as new contigs become assembled and incorporated into the available set for assembly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Current working directory is:\n",
      " /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/\n"
     ]
    }
   ],
   "source": [
    "wkDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/'\n",
    "print('Current working directory is:\\n', wkDir)\n",
    "\n",
    "logFile = open(wkDir + 'temp/iPythonNotebook_LogFile.txt','w')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Identified coordinates for 2688 primary contigs\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "49"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#####READING IN PRIMARY CONTIG COORDINATES FROM MUMMER ALIGNMENT#######\n",
    "#Primary contig alignments (no canu processing)\n",
    "primaryContigCoord = open(wkDir + 'input/primary.2017-04-10.txt', 'r')\n",
    "primaryContigCoordBed = open(wkDir + 'input/primary.2017-04-10.bed', 'w')\n",
    "\n",
    "contigDict,index = {}, 0\n",
    "\n",
    "for line in primaryContigCoord:\n",
    "    if line.startswith(\"#\") is True: #skips header line\n",
    "        continue\n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID, length, Dir, chrom, start, end = line[0],line[1],line[2],line[3],int(line[4]),int(line[5])    \n",
    "    \"\"\"if 'chr18' not in chrom: #FILTERING ONLY FOR CHROM 18 FOR NOW!!! REMOVE LATER \n",
    "        continue\"\"\"\n",
    "    index += 1\n",
    "    contigDict[index] = [chrom,start,end,contigID,Dir,length]\n",
    "    primaryContigCoordBed.write('%s\\t%i\\t%i\\t%s\\n' % (chrom,int(start-1),end,contigID))\n",
    "    #if index > 15: #FOR TESTING--- REMOVE LATER\n",
    "    #    break\n",
    "    \n",
    "primaryContigCoord.close()\n",
    "primaryContigCoordBed.close()\n",
    "print ('Identified coordinates for %i primary contigs' % len(contigDict))\n",
    "logFile.write('Identified coordinates for %i primary contigs\\n\\n' % len(contigDict))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step 1. Read in coordinates from alignment file based on Mummer of primary contigs (per Jeff)\n",
    "Step 2. BLAT each contig against the contig proximal to that proximal to it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_line(Dict,curr_index,i):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[curr_index][0],int(Dict[curr_index][1]),int(Dict[curr_index][2]), Dict[curr_index][3], Dict[curr_index][4], int(Dict[curr_index][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i+1][0],int(Dict[i+1][1]),int(Dict[i+1][2]), Dict[i+1][3], Dict[i+1][4], int(Dict[i+1][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def read_last_line(Dict):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[i-1][0],int(Dict[i-1][1]),int(Dict[i-1][2]), Dict[i-1][3], Dict[i-1][4], int(Dict[i-1][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i][0],int(Dict[i][1]),int(Dict[i][2]), Dict[i][3], Dict[i][4], int(Dict[i][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def process_last_contig(contigDict,curr_index,i):\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_last_line(contigDict)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,i-1,contigID2,i))\n",
    "    logFile.write(('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,i-1,contigID2,i)))\n",
    "    if start1 == start2 or end1 == end2:\n",
    "        return\n",
    "    if end2 > end1:\n",
    "        curr_index+=1 \n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        inList.append(contigID2)\n",
    "        print('PASS - Last contig on chromosome overlap or is spaced correctly')\n",
    "        logFile.write('PASS - Last contig on chromosome overlap or is spaced correctly\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_first_contig(contigDict,curr_index,i):\n",
    "    curr_index+=1\n",
    "    #Always save first and contig in the dictionary/dataset\n",
    "    curated_contigDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "    curated_contigDict[i+1] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    inList.append(contigID2)\n",
    "    print('PASS - First call in dataset')\n",
    "    logFile.write('PASS - First call in dataset\\n')\n",
    "    return curr_index \n",
    "###################################################################################################\n",
    "def process_same_starts(contigDict,curr_index,i):\n",
    "    if start1 == start2 and end1 > end2:\n",
    "        print('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "    if start1 == start2 and end1 < end2: #overwrite previous position if contig 2 is longer than contig 1\n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        print('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_same_ends(contigDict,curr_index,i):    \n",
    "    if start1 < start2 and end1 == end2:\n",
    "        print('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return \n",
    "    if start2 < start1 and end1 == end2: \n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#CTG-0076 (1) -- CTG-1722 (2)\n",
      "PASS - First call in dataset\n",
      "\n",
      "#CTG-1722 (2) -- CTG-0353 (3)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0353 (3) -- CTG-0052 (4)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0052 (4) -- CTG-0231 (5)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0231 (5) -- CTG-0230 (6)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0231\n",
      "['chr1', 17343875, 20909826, 'CTG-0231', 'rc', '3565775']\n",
      "['chr1', 17356763, 17361463, 'CTG-0230', 'fwd', '4861']\n",
      "\n",
      "#CTG-0231 (5) -- CTG-0143 (7)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0143 (7) -- CTG-2241 (8)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0143\n",
      "['chr1', 20906456, 25699815, 'CTG-0143', 'fwd', '4864237']\n",
      "['chr1', 24634452, 24637932, 'CTG-2241', 'rc', '6265']\n",
      "\n",
      "#CTG-0143 (7) -- CTG-1991 (9)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0143\n",
      "['chr1', 24634452, 24637932, 'CTG-2241', 'rc', '6265']\n",
      "['chr1', 24635586, 24645896, 'CTG-1991', 'fwd', '10214']\n",
      "\n",
      "#CTG-0143 (7) -- CTG-1396 (10)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1396 (10) -- CTG-1379 (11)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1379 (11) -- CTG-0024 (12)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0024 (12) -- CTG-1946 (13)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0024\n",
      "['chr1', 25755738, 36467787, 'CTG-0024', 'fwd', '10717598']\n",
      "['chr1', 27217144, 27227978, 'CTG-1946', 'fwd', '10965']\n",
      "\n",
      "#CTG-0024 (12) -- CTG-1311 (14)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0024\n",
      "['chr1', 27217144, 27227978, 'CTG-1946', 'fwd', '10965']\n",
      "['chr1', 29866215, 29900291, 'CTG-1311', 'rc', '33827']\n",
      "\n",
      "#CTG-0024 (12) -- CTG-2188 (15)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0024\n",
      "['chr1', 29866215, 29900291, 'CTG-1311', 'rc', '33827']\n",
      "['chr1', 32851876, 32861333, 'CTG-2188', 'fwd', '6995']\n",
      "\n",
      "#CTG-0024 (12) -- CTG-0385 (16)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0385 (16) -- CTG-0004 (17)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0004 (17) -- CTG-2528 (18)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0004\n",
      "['chr1', 38720302, 59110378, 'CTG-0004', 'fwd', '20419135']\n",
      "['chr1', 40651459, 40653818, 'CTG-2528', 'rc', '2348']\n",
      "\n",
      "#CTG-0004 (17) -- CTG-0407 (19)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0407 (19) -- CTG-0618 (20)\n",
      "Ends same, kept contig 1 only\n",
      "['chr1', 59096260, 61209851, 'CTG-0407', 'fwd', '2126156']\n",
      "['chr1', 61158168, 61209851, 'CTG-0618', 'fwd', '52102']\n",
      "\n",
      "#CTG-0407 (19) -- CTG-0130 (21)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0130 (21) -- CTG-1758 (22)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0130\n",
      "['chr1', 61188275, 66351705, 'CTG-0130', 'rc', '5154938']\n",
      "['chr1', 62229523, 62244989, 'CTG-1758', 'fwd', '15078']\n",
      "\n",
      "#CTG-0130 (21) -- CTG-2498 (23)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0130\n",
      "['chr1', 62229523, 62244989, 'CTG-1758', 'fwd', '15078']\n",
      "['chr1', 62761375, 62764135, 'CTG-2498', 'fwd', '2761']\n",
      "\n",
      "#CTG-0130 (21) -- CTG-2332 (24)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0130\n",
      "['chr1', 62761375, 62764135, 'CTG-2498', 'fwd', '2761']\n",
      "['chr1', 62935360, 62940483, 'CTG-2332', 'fwd', '5097']\n",
      "\n",
      "#CTG-0130 (21) -- CTG-0993 (25)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0993 (25) -- CTG-0996 (26)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0993\n",
      "['chr1', 66351706, 66556441, 'CTG-0993', 'rc', '211149']\n",
      "['chr1', 66362653, 66365421, 'CTG-0996', 'fwd', '2765']\n",
      "\n",
      "#CTG-0993 (25) -- CTG-0862 (27)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0862 (27) -- CTG-0436 (28)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0436 (28) -- CTG-1592 (29)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0436\n",
      "['chr1', 66965446, 68897138, 'CTG-0436', 'rc', '1936162']\n",
      "['chr1', 66987432, 66998191, 'CTG-1592', 'fwd', '10703']\n",
      "\n",
      "#CTG-0436 (28) -- CTG-0765 (30)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0765 (30) -- CTG-1087 (31)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1087 (31) -- CTG-0660 (32)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0660 (32) -- CTG-0666 (33)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0660\n",
      "['chr1', 69711636, 70626323, 'CTG-0660', 'fwd', '913263']\n",
      "['chr1', 70605601, 70618035, 'CTG-0666', 'rc', '12222']\n",
      "\n",
      "#CTG-0660 (32) -- CTG-1836 (34)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0660\n",
      "['chr1', 70605601, 70618035, 'CTG-0666', 'rc', '12222']\n",
      "['chr1', 70605601, 70619332, 'CTG-1836', 'rc', '13515']\n",
      "\n",
      "#CTG-0660 (32) -- CTG-0097 (35)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0097 (35) -- CTG-1556 (36)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0097\n",
      "['chr1', 70627236, 76622361, 'CTG-0097', 'fwd', '6008365']\n",
      "['chr1', 75657275, 75678103, 'CTG-1556', 'fwd', '20715']\n",
      "\n",
      "#CTG-0097 (35) -- CTG-0111 (37)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0111 (37) -- CTG-2162 (38)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0111\n",
      "['chr1', 76599640, 81949499, 'CTG-0111', 'rc', '5393392']\n",
      "['chr1', 79470081, 79477228, 'CTG-2162', 'fwd', '7322']\n",
      "\n",
      "#CTG-0111 (37) -- CTG-0369 (39)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0369 (39) -- CTG-0859 (40)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0859 (40) -- CTG-0020 (41)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0020 (41) -- CTG-1621 (42)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0020\n",
      "['chr1', 84718798, 95730658, 'CTG-0020', 'rc', '11023292']\n",
      "['chr1', 89803822, 89843086, 'CTG-1621', 'rc', '18592']\n",
      "\n",
      "#CTG-0020 (41) -- CTG-1963 (43)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0020\n",
      "['chr1', 89803822, 89843086, 'CTG-1621', 'rc', '18592']\n",
      "['chr1', 94198447, 94209113, 'CTG-1963', 'rc', '10614']\n",
      "\n",
      "#CTG-0020 (41) -- CTG-0454 (44)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0454 (44) -- CTG-1337 (45)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0454\n",
      "['chr1', 95743978, 97604415, 'CTG-0454', 'fwd', '1866490']\n",
      "['chr1', 96553808, 96597189, 'CTG-1337', 'rc', '31705']\n",
      "\n",
      "#CTG-0454 (44) -- CTG-1064 (46)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1064 (46) -- CTG-1207 (47)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-1064\n",
      "['chr1', 97594676, 97736654, 'CTG-1064', 'fwd', '143852']\n",
      "['chr1', 97712549, 97708013, 'CTG-1207', 'fwd', '56066']\n",
      "\n",
      "#CTG-1064 (46) -- CTG-0535 (48)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0535 (48) -- CTG-1061 (49)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1061 (49) -- CTG-0608 (50)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0608 (50) -- CTG-1208 (51)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1208 (51) -- CTG-0706 (52)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0706 (52) -- CTG-1296 (53)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1296 (53) -- CTG-0529 (54)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0529 (54) -- CTG-2392 (55)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0529\n",
      "['chr1', 101301041, 102769755, 'CTG-0529', 'rc', '1472751']\n",
      "['chr1', 102427714, 102432218, 'CTG-2392', 'rc', '4479']\n",
      "\n",
      "#CTG-0529 (54) -- CTG-0791 (56)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0791 (56) -- CTG-1513 (57)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0791\n",
      "['chr1', 102773034, 103333006, 'CTG-0791', 'rc', '560087']\n",
      "['chr1', 103021999, 103043832, 'CTG-1513', 'rc', '22325']\n",
      "\n",
      "#CTG-0791 (56) -- CTG-0826 (58)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0826 (58) -- CTG-1671 (59)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0826\n",
      "['chr1', 103346881, 104014811, 'CTG-0826', 'fwd', '468543']\n",
      "['chr1', 103645388, 103663295, 'CTG-1671', 'rc', '17498']\n",
      "\n",
      "#CTG-0826 (58) -- CTG-0915 (60)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0915 (60) -- CTG-2083 (61)\n",
      "FAIL -- Share start, contig1 longer\n",
      "['chr1', 103995997, 104545757, 'CTG-0915', 'rc', '315752']\n",
      "['chr1', 103995997, 104004946, 'CTG-2083', 'rc', '8637']\n",
      "\n",
      "#CTG-0915 (60) -- CTG-1279 (62)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0915\n",
      "['chr1', 103995997, 104004946, 'CTG-2083', 'rc', '8637']\n",
      "['chr1', 104275592, 104313849, 'CTG-1279', 'fwd', '37802']\n",
      "\n",
      "#CTG-0915 (60) -- CTG-0978 (63)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0915\n",
      "['chr1', 104275592, 104313849, 'CTG-1279', 'fwd', '37802']\n",
      "['chr1', 104302821, 104540459, 'CTG-0978', 'fwd', '237040']\n",
      "\n",
      "#CTG-0915 (60) -- CTG-2132 (64)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0915\n",
      "['chr1', 104302821, 104540459, 'CTG-0978', 'fwd', '237040']\n",
      "['chr1', 104306596, 104543140, 'CTG-2132', 'fwd', '7717']\n",
      "\n",
      "#CTG-0915 (60) -- CTG-1923 (65)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0915\n",
      "['chr1', 104306596, 104543140, 'CTG-2132', 'fwd', '7717']\n",
      "['chr1', 104525421, 104538450, 'CTG-1923', 'rc', '11558']\n",
      "\n",
      "#CTG-0915 (60) -- CTG-1354 (66)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1354 (66) -- CTG-1941 (67)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-1354\n",
      "['chr1', 104528420, 104558658, 'CTG-1354', 'fwd', '29968']\n",
      "['chr1', 104528590, 104538450, 'CTG-1941', 'rc', '11092']\n",
      "\n",
      "#CTG-1354 (66) -- CTG-1183 (68)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1183 (68) -- CTG-1411 (69)\n",
      "FAIL -- Share start, contig1 longer\n",
      "['chr1', 104542988, 104606793, 'CTG-1183', 'rc', '63633']\n",
      "['chr1', 104542988, 104555453, 'CTG-1411', 'rc', '12411']\n",
      "\n",
      "#CTG-1183 (68) -- CTG-0366 (70)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0366 (70) -- CTG-0626 (71)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0626 (71) -- CTG-1009 (72)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1009 (72) -- CTG-1229 (73)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1229 (73) -- CTG-0724 (74)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0724 (74) -- CTG-0121 (75)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0121 (75) -- CTG-2490 (76)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 108979413, 114321508, 'CTG-0121', 'fwd', '5297349']\n",
      "['chr1', 109555803, 109558693, 'CTG-2490', 'rc', '2841']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1834 (77)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 109555803, 109558693, 'CTG-2490', 'rc', '2841']\n",
      "['chr1', 111777933, 111813889, 'CTG-1834', 'rc', '13552']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-2180 (78)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 111777933, 111813889, 'CTG-1834', 'rc', '13552']\n",
      "['chr1', 111815492, 111822411, 'CTG-2180', 'fwd', '7081']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1871 (79)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 111815492, 111822411, 'CTG-2180', 'fwd', '7081']\n",
      "['chr1', 111920481, 111933239, 'CTG-1871', 'rc', '12892']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1439 (80)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 111920481, 111933239, 'CTG-1871', 'rc', '12892']\n",
      "['chr1', 112512391, 112552522, 'CTG-1439', 'fwd', '25943']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1964 (81)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112512391, 112552522, 'CTG-1439', 'fwd', '25943']\n",
      "['chr1', 112521715, 112546730, 'CTG-1964', 'fwd', '10575']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-2512 (82)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112521715, 112546730, 'CTG-1964', 'fwd', '10575']\n",
      "['chr1', 112533490, 112546238, 'CTG-2512', 'rc', '2555']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1426 (83)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112533490, 112546238, 'CTG-2512', 'rc', '2555']\n",
      "['chr1', 112537980, 112532372, 'CTG-1426', 'rc', '26641']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-2413 (84)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112537980, 112532372, 'CTG-1426', 'rc', '26641']\n",
      "['chr1', 112539301, 112557527, 'CTG-2413', 'rc', '4141']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-2291 (85)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112539301, 112557527, 'CTG-2413', 'rc', '4141']\n",
      "['chr1', 112544015, 112549645, 'CTG-2291', 'rc', '5619']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1496 (86)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0121\n",
      "['chr1', 112544015, 112549645, 'CTG-2291', 'rc', '5619']\n",
      "['chr1', 112915760, 112939208, 'CTG-1496', 'rc', '23222']\n",
      "\n",
      "#CTG-0121 (75) -- CTG-1223 (87)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1223 (87) -- CTG-0435 (88)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0435 (88) -- CTG-0920 (89)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0920 (89) -- CTG-0259 (90)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0259 (90) -- CTG-0820 (91)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0259\n",
      "['chr1', 116551203, 119906706, 'CTG-0259', 'rc', '3349177']\n",
      "['chr1', 116554749, 116564076, 'CTG-0820', 'fwd', '9336']\n",
      "\n",
      "#CTG-0259 (90) -- CTG-1325 (92)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0259\n",
      "['chr1', 116554749, 116564076, 'CTG-0820', 'fwd', '9336']\n",
      "['chr1', 117019768, 117053072, 'CTG-1325', 'rc', '32610']\n",
      "\n",
      "#CTG-0259 (90) -- CTG-0422 (93)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0422 (93) -- CTG-1962 (94)\n",
      "FAIL -- Share start, contig1 longer\n",
      "['chr1', 119891324, 121923628, 'CTG-0422', 'rc', '2059298']\n",
      "['chr1', 119891324, 119902058, 'CTG-1962', 'rc', '10690']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-1799 (95)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0422\n",
      "['chr1', 119891324, 119902058, 'CTG-1962', 'rc', '10690']\n",
      "['chr1', 119903723, 119917968, 'CTG-1799', 'fwd', '14081']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-1803 (96)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0422\n",
      "['chr1', 119903723, 119917968, 'CTG-1799', 'fwd', '14081']\n",
      "['chr1', 119903723, 119917968, 'CTG-1803', 'fwd', '14040']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-2060 (97)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0422\n",
      "['chr1', 119903723, 119917968, 'CTG-1803', 'fwd', '14040']\n",
      "['chr1', 119903723, 119913038, 'CTG-2060', 'fwd', '9156']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-1528 (98)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0422\n",
      "['chr1', 119903723, 119913038, 'CTG-2060', 'fwd', '9156']\n",
      "['chr1', 120695727, 120717415, 'CTG-1528', 'fwd', '21642']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-2064 (99)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0422\n",
      "['chr1', 120695727, 120717415, 'CTG-1528', 'fwd', '21642']\n",
      "['chr1', 121229516, 121238569, 'CTG-2064', 'rc', '9018']\n",
      "\n",
      "#CTG-0422 (93) -- CTG-1030 (100)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1030 (100) -- CTG-0793 (101)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0793 (101) -- CTG-1944 (102)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0793\n",
      "['chr1', 122113446, 122678785, 'CTG-0793', 'fwd', '564775']\n",
      "['chr1', 122141358, 122152178, 'CTG-1944', 'fwd', '11139']\n",
      "\n",
      "#CTG-0793 (101) -- CTG-0119 (103)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1944 (102) -- CTG-0119 (103)\n",
      "\n",
      "#CTG-0119 (103) -- CTG-1678 (104)\n",
      "\n",
      "#CTG-1678 (104) -- CTG-2081 (105)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2081 (105) -- CTG-1696 (106)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1696 (106) -- CTG-1981 (107)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1981 (107) -- CTG-0828 (108)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0828 (108) -- CTG-1638 (109)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1638 (109) -- CTG-0463 (110)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0463 (110) -- CTG-0462 (111)\n",
      "\n",
      "#CTG-0462 (111) -- CTG-0320 (112)\n",
      "\n",
      "#CTG-0320 (112) -- CTG-2208 (113)\n",
      "\n",
      "#CTG-2208 (113) -- CTG-0876 (114)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0876 (114) -- CTG-0633 (115)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0633 (115) -- CTG-0632 (116)\n",
      "\n",
      "#CTG-0632 (116) -- CTG-0654 (117)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0654 (117) -- CTG-2126 (118)\n",
      "\n",
      "#CTG-2126 (118) -- CTG-1418 (119)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1418 (119) -- CTG-0650 (120)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0650 (120) -- CTG-0545 (121)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0545 (121) -- CTG-0689 (122)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0689 (122) -- CTG-1376 (123)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1376 (123) -- CTG-1035 (124)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1035 (124) -- CTG-1217 (125)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1217 (125) -- CTG-1043 (126)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1043 (126) -- CTG-1004 (127)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1004 (127) -- CTG-1887 (128)\n",
      "\n",
      "#CTG-1887 (128) -- CTG-2042 (129)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2042 (129) -- CTG-0039 (130)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0039 (130) -- CTG-1936 (131)\n",
      "\n",
      "#CTG-1936 (131) -- CTG-1542 (132)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1542 (132) -- CTG-0838 (133)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0838 (133) -- CTG-0701 (134)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0701 (134) -- CTG-0037 (135)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0037 (135) -- CTG-0118 (136)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0118 (136) -- CTG-1683 (137)\n",
      "\n",
      "#CTG-1683 (137) -- CTG-1802 (138)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1802 (138) -- CTG-0587 (139)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0587 (139) -- CTG-0566 (140)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0566 (140) -- CTG-2024 (141)\n",
      "\n",
      "#CTG-2024 (141) -- CTG-0572 (142)\n",
      "\n",
      "#CTG-0572 (142) -- CTG-0494 (143)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0494 (143) -- CTG-0094 (144)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0094 (144) -- CTG-1800 (145)\n",
      "\n",
      "#CTG-1800 (145) -- CTG-1875 (146)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1875 (146) -- CTG-0172 (147)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0172 (147) -- CTG-0103 (148)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0103 (148) -- CTG-0173 (149)\n",
      "\n",
      "#CTG-0173 (149) -- CTG-0162 (150)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0162 (150) -- CTG-2035 (151)\n",
      "\n",
      "#CTG-2035 (151) -- CTG-2246 (152)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2246 (152) -- CTG-0691 (153)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0691 (153) -- CTG-0528 (154)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0528 (154) -- CTG-2260 (155)\n",
      "\n",
      "#CTG-2260 (155) -- CTG-0634 (156)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0634 (156) -- CTG-1104 (157)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1104 (157) -- CTG-1746 (158)\n",
      "\n",
      "#CTG-1746 (158) -- CTG-0992 (159)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0992 (159) -- CTG-0867 (160)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0867 (160) -- CTG-0692 (161)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0692 (161) -- CTG-0464 (162)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0464 (162) -- CTG-2581 (163)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2581 (163) -- CTG-0333 (164)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0333 (164) -- CTG-2513 (165)\n",
      "\n",
      "#CTG-2513 (165) -- CTG-0399 (166)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0399 (166) -- CTG-2557 (167)\n",
      "\n",
      "#CTG-2557 (167) -- CTG-1363 (168)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1363 (168) -- CTG-2356 (169)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2356 (169) -- CTG-0761 (170)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0761 (170) -- CTG-2108 (171)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2108 (171) -- CTG-1796 (172)\n",
      "\n",
      "#CTG-1796 (172) -- CTG-1787 (173)\n",
      "\n",
      "#CTG-1787 (173) -- CTG-1773 (174)\n",
      "\n",
      "#CTG-1773 (174) -- CTG-1859 (175)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1859 (175) -- CTG-1876 (176)\n",
      "\n",
      "#CTG-1876 (176) -- CTG-1331 (177)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1331 (177) -- CTG-1461 (178)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1461 (178) -- CTG-1978 (179)\n",
      "\n",
      "#CTG-1978 (179) -- CTG-2197 (180)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2197 (180) -- CTG-1736 (181)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1736 (181) -- CTG-1029 (182)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1029 (182) -- CTG-0737 (183)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0737 (183) -- CTG-0983 (184)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0983 (184) -- CTG-2105 (185)\n",
      "\n",
      "#CTG-2105 (185) -- CTG-1138 (186)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1138 (186) -- CTG-1110 (187)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1110 (187) -- CTG-1552 (188)\n",
      "\n",
      "#CTG-1552 (188) -- CTG-1994 (189)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1994 (189) -- CTG-1323 (190)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1323 (190) -- CTG-0902 (191)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0902 (191) -- CTG-1417 (192)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1417 (192) -- CTG-0774 (193)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0774 (193) -- CTG-0896 (194)\n",
      "\n",
      "#CTG-0896 (194) -- CTG-1786 (195)\n",
      "\n",
      "#CTG-1786 (195) -- CTG-2543 (196)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2543 (196) -- CTG-0760 (197)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0760 (197) -- CTG-1974 (198)\n",
      "\n",
      "#CTG-1974 (198) -- CTG-1422 (199)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1422 (199) -- CTG-1792 (200)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1792 (200) -- CTG-2122 (201)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2122 (201) -- CTG-0702 (202)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0702 (202) -- CTG-1177 (203)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1177 (203) -- CTG-0113 (204)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0113 (204) -- CTG-1601 (205)\n",
      "\n",
      "#CTG-1601 (205) -- CTG-0123 (206)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0123 (206) -- CTG-2491 (207)\n",
      "\n",
      "#CTG-2491 (207) -- CTG-1437 (208)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1437 (208) -- CTG-1820 (209)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1820 (209) -- CTG-1091 (210)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1091 (210) -- CTG-1063 (211)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1063 (211) -- CTG-0224 (212)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0224 (212) -- CTG-2172 (213)\n",
      "\n",
      "#CTG-2172 (213) -- CTG-0169 (214)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0169 (214) -- CTG-2339 (215)\n",
      "\n",
      "#CTG-2339 (215) -- CTG-0840 (216)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0840 (216) -- CTG-0472 (217)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0472 (217) -- CTG-0308 (218)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0308 (218) -- CTG-1438 (219)\n",
      "\n",
      "#CTG-1438 (219) -- CTG-0108 (220)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0108 (220) -- CTG-1397 (221)\n",
      "\n",
      "#CTG-1397 (221) -- CTG-0140 (222)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0140 (222) -- CTG-1660 (223)\n",
      "\n",
      "#CTG-1660 (223) -- CTG-2417 (224)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2417 (224) -- CTG-1649 (225)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1649 (225) -- CTG-0611 (226)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0611 (226) -- CTG-1934 (227)\n",
      "\n",
      "#CTG-1934 (227) -- CTG-0358 (228)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0358 (228) -- CTG-2411 (229)\n",
      "\n",
      "#CTG-2411 (229) -- CTG-1926 (230)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1926 (230) -- CTG-1639 (231)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1639 (231) -- CTG-0784 (232)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0784 (232) -- CTG-0903 (233)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0903 (233) -- CTG-1233 (234)\n",
      "\n",
      "#CTG-1233 (234) -- CTG-0182 (235)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0182 (235) -- CTG-2032 (236)\n",
      "\n",
      "#CTG-2032 (236) -- CTG-0258 (237)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0258 (237) -- CTG-1321 (238)\n",
      "\n",
      "#CTG-1321 (238) -- CTG-0033 (239)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0033 (239) -- CTG-1497 (240)\n",
      "\n",
      "#CTG-1497 (240) -- CTG-2306 (241)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2306 (241) -- CTG-1373 (242)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1373 (242) -- CTG-0313 (243)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0313 (243) -- CTG-0912 (244)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0912 (244) -- CTG-0712 (245)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0712 (245) -- CTG-2016 (246)\n",
      "\n",
      "#CTG-2016 (246) -- CTG-1493 (247)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1493 (247) -- CTG-1074 (248)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1074 (248) -- CTG-0639 (249)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0639 (249) -- CTG-0795 (250)\n",
      "\n",
      "#CTG-0795 (250) -- CTG-2059 (251)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2059 (251) -- CTG-2412 (252)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2412 (252) -- CTG-0594 (253)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0594 (253) -- CTG-1092 (254)\n",
      "\n",
      "#CTG-1092 (254) -- CTG-0465 (255)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0465 (255) -- CTG-1485 (256)\n",
      "\n",
      "#CTG-1485 (256) -- CTG-2555 (257)\n",
      "\n",
      "#CTG-2555 (257) -- CTG-2625 (258)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2625 (258) -- CTG-0354 (259)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0354 (259) -- CTG-1545 (260)\n",
      "\n",
      "#CTG-1545 (260) -- CTG-1872 (261)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1872 (261) -- CTG-1766 (262)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1766 (262) -- CTG-0016 (263)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0016 (263) -- CTG-0484 (264)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0484 (264) -- CTG-1539 (265)\n",
      "\n",
      "#CTG-1539 (265) -- CTG-1305 (266)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1305 (266) -- CTG-1584 (267)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1584 (267) -- CTG-0758 (268)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0758 (268) -- CTG-0739 (269)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0739 (269) -- CTG-0062 (270)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0062 (270) -- CTG-1221 (271)\n",
      "\n",
      "#CTG-1221 (271) -- CTG-1995 (272)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1995 (272) -- CTG-1563 (273)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1563 (273) -- CTG-0361 (274)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0361 (274) -- CTG-0500 (275)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0500 (275) -- CTG-0493 (276)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0493 (276) -- CTG-1730 (277)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1730 (277) -- CTG-0178 (278)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0178 (278) -- CTG-0339 (279)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0339 (279) -- CTG-0752 (280)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0752 (280) -- CTG-1707 (281)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1707 (281) -- CTG-0748 (282)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0748 (282) -- CTG-0346 (283)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0346 (283) -- CTG-0174 (284)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0174 (284) -- CTG-1785 (285)\n",
      "\n",
      "#CTG-1785 (285) -- CTG-2574 (286)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2574 (286) -- CTG-2590 (287)\n",
      "\n",
      "#CTG-2590 (287) -- CTG-1499 (288)\n",
      "\n",
      "#CTG-1499 (288) -- CTG-2340 (289)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2340 (289) -- CTG-2519 (290)\n",
      "\n",
      "#CTG-2519 (290) -- CTG-2515 (291)\n",
      "\n",
      "#CTG-2515 (291) -- CTG-1862 (292)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1862 (292) -- CTG-2214 (293)\n",
      "\n",
      "#CTG-2214 (293) -- CTG-1914 (294)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1914 (294) -- CTG-2420 (295)\n",
      "\n",
      "#CTG-2420 (295) -- CTG-2585 (296)\n",
      "\n",
      "#CTG-2585 (296) -- CTG-1677 (297)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1677 (297) -- CTG-0693 (298)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0693 (298) -- CTG-0323 (299)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0323 (299) -- CTG-0751 (300)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0751 (300) -- CTG-0184 (301)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0184 (301) -- CTG-1793 (302)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1793 (302) -- CTG-1050 (303)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1050 (303) -- CTG-1429 (304)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1429 (304) -- CTG-0519 (305)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0519 (305) -- CTG-0521 (306)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0521 (306) -- CTG-1316 (307)\n",
      "\n",
      "#CTG-1316 (307) -- CTG-0368 (308)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0368 (308) -- CTG-0605 (309)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0605 (309) -- CTG-1475 (310)\n",
      "\n",
      "#CTG-1475 (310) -- CTG-0330 (311)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0330 (311) -- CTG-1775 (312)\n",
      "\n",
      "#CTG-1775 (312) -- CTG-0164 (313)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0164 (313) -- CTG-2176 (314)\n",
      "\n",
      "#CTG-2176 (314) -- CTG-1870 (315)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1870 (315) -- CTG-1687 (316)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1687 (316) -- CTG-0264 (317)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0264 (317) -- CTG-2443 (318)\n",
      "\n",
      "#CTG-2443 (318) -- CTG-0844 (319)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0844 (319) -- CTG-2014 (320)\n",
      "\n",
      "#CTG-2014 (320) -- CTG-1017 (321)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1017 (321) -- CTG-0947 (322)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0947 (322) -- CTG-0057 (323)\n",
      "\n",
      "#CTG-0057 (323) -- CTG-0612 (324)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0612 (324) -- CTG-0384 (325)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0384 (325) -- CTG-1364 (326)\n",
      "\n",
      "#CTG-1364 (326) -- CTG-1729 (327)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1729 (327) -- CTG-0694 (328)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0694 (328) -- CTG-0461 (329)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0461 (329) -- CTG-0674 (330)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0674 (330) -- CTG-1935 (331)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1935 (331) -- CTG-0966 (332)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0966 (332) -- CTG-1637 (333)\n",
      "\n",
      "#CTG-1637 (333) -- CTG-2212 (334)\n",
      "\n",
      "#CTG-2212 (334) -- CTG-0280 (335)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0280 (335) -- CTG-0185 (336)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0185 (336) -- CTG-0281 (337)\n",
      "\n",
      "#CTG-0281 (337) -- CTG-1711 (338)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1711 (338) -- CTG-1999 (339)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1999 (339) -- CTG-0011 (340)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0011 (340) -- CTG-1989 (341)\n",
      "\n",
      "#CTG-1989 (341) -- CTG-1931 (342)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1931 (342) -- CTG-2068 (343)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2068 (343) -- CTG-0678 (344)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0678 (344) -- CTG-0382 (345)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0382 (345) -- CTG-2117 (346)\n",
      "\n",
      "#CTG-2117 (346) -- CTG-1003 (347)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1003 (347) -- CTG-1609 (348)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1609 (348) -- CTG-0302 (349)\n",
      "\n",
      "#CTG-0302 (349) -- CTG-0424 (350)\n",
      "\n",
      "#CTG-0424 (350) -- CTG-0051 (351)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0051 (351) -- CTG-2360 (352)\n",
      "\n",
      "#CTG-2360 (352) -- CTG-0677 (353)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0677 (353) -- CTG-2167 (354)\n",
      "\n",
      "#CTG-2167 (354) -- CTG-0112 (355)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0112 (355) -- CTG-2174 (356)\n",
      "\n",
      "#CTG-2174 (356) -- CTG-2175 (357)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2175 (357) -- CTG-2171 (358)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2171 (358) -- CTG-1105 (359)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1105 (359) -- CTG-1477 (360)\n",
      "\n",
      "#CTG-1477 (360) -- CTG-0804 (361)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0804 (361) -- CTG-0910 (362)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0910 (362) -- CTG-1591 (363)\n",
      "\n",
      "#CTG-1591 (363) -- CTG-0675 (364)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0675 (364) -- CTG-0319 (365)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0319 (365) -- CTG-2527 (366)\n",
      "\n",
      "#CTG-2527 (366) -- CTG-1465 (367)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1465 (367) -- CTG-0895 (368)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0895 (368) -- CTG-1079 (369)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1079 (369) -- CTG-1930 (370)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1930 (370) -- CTG-1145 (371)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1145 (371) -- CTG-2336 (372)\n",
      "\n",
      "#CTG-2336 (372) -- CTG-1153 (373)\n",
      "\n",
      "#CTG-1153 (373) -- CTG-1432 (374)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1432 (374) -- CTG-1295 (375)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1295 (375) -- CTG-1287 (376)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1287 (376) -- CTG-1620 (377)\n",
      "\n",
      "#CTG-1620 (377) -- CTG-2219 (378)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2219 (378) -- CTG-2561 (379)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2561 (379) -- CTG-1969 (380)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1969 (380) -- CTG-0942 (381)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0942 (381) -- CTG-0911 (382)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0911 (382) -- CTG-1172 (383)\n",
      "\n",
      "#CTG-1172 (383) -- CTG-0877 (384)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0877 (384) -- CTG-0682 (385)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0682 (385) -- CTG-1568 (386)\n",
      "\n",
      "#CTG-1568 (386) -- CTG-0824 (387)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0824 (387) -- CTG-1112 (388)\n",
      "\n",
      "#CTG-1112 (388) -- CTG-2502 (389)\n",
      "\n",
      "#CTG-2502 (389) -- CTG-0193 (390)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0193 (390) -- CTG-0010 (391)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0010 (391) -- CTG-2487 (392)\n",
      "\n",
      "#CTG-2487 (392) -- CTG-0268 (393)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0268 (393) -- CTG-0945 (394)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0945 (394) -- CTG-1976 (395)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1976 (395) -- CTG-0372 (396)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0372 (396) -- CTG-1526 (397)\n",
      "\n",
      "#CTG-1526 (397) -- CTG-2403 (398)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2403 (398) -- CTG-2266 (399)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2266 (399) -- CTG-0099 (400)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0099 (400) -- CTG-1044 (401)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1044 (401) -- CTG-0044 (402)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0044 (402) -- CTG-2446 (403)\n",
      "\n",
      "#CTG-2446 (403) -- CTG-2569 (404)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2569 (404) -- CTG-1608 (405)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1608 (405) -- CTG-0571 (406)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0571 (406) -- CTG-1781 (407)\n",
      "\n",
      "#CTG-1781 (407) -- CTG-0649 (408)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0649 (408) -- CTG-1258 (409)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1258 (409) -- CTG-1518 (410)\n",
      "\n",
      "#CTG-1518 (410) -- CTG-1739 (411)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1739 (411) -- CTG-0526 (412)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0526 (412) -- CTG-0962 (413)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0962 (413) -- CTG-0733 (414)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0733 (414) -- CTG-1027 (415)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1027 (415) -- CTG-1028 (416)\n",
      "\n",
      "#CTG-1028 (416) -- CTG-0683 (417)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0683 (417) -- CTG-1351 (418)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1351 (418) -- CTG-0090 (419)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0090 (419) -- CTG-2633 (420)\n",
      "\n",
      "#CTG-2633 (420) -- CTG-2317 (421)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2317 (421) -- CTG-0287 (422)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0287 (422) -- CTG-0216 (423)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0216 (423) -- CTG-2012 (424)\n",
      "\n",
      "#CTG-2012 (424) -- CTG-1055 (425)\n",
      "\n",
      "#CTG-1055 (425) -- CTG-0556 (426)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0556 (426) -- CTG-0559 (427)\n",
      "\n",
      "#CTG-0559 (427) -- CTG-0460 (428)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0460 (428) -- CTG-1847 (429)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1847 (429) -- CTG-0365 (430)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0365 (430) -- CTG-1377 (431)\n",
      "\n",
      "#CTG-1377 (431) -- CTG-2098 (432)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2098 (432) -- CTG-0614 (433)\n",
      "\n",
      "#CTG-0614 (433) -- CTG-0636 (434)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0636 (434) -- CTG-2477 (435)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2477 (435) -- CTG-0786 (436)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0786 (436) -- CTG-0495 (437)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0495 (437) -- CTG-2320 (438)\n",
      "\n",
      "#CTG-2320 (438) -- CTG-1777 (439)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1777 (439) -- CTG-2232 (440)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2232 (440) -- CTG-0124 (441)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0124 (441) -- CTG-2573 (442)\n",
      "\n",
      "#CTG-2573 (442) -- CTG-1572 (443)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1572 (443) -- CTG-2086 (444)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2086 (444) -- CTG-2072 (445)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2072 (445) -- CTG-1559 (446)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1559 (446) -- CTG-1827 (447)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1827 (447) -- CTG-2041 (448)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2041 (448) -- CTG-0087 (449)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0087 (449) -- CTG-1523 (450)\n",
      "\n",
      "#CTG-1523 (450) -- CTG-1826 (451)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1826 (451) -- CTG-0008 (452)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0008 (452) -- CTG-0007 (453)\n",
      "\n",
      "#CTG-0007 (453) -- CTG-0420 (454)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0420 (454) -- CTG-0421 (455)\n",
      "\n",
      "#CTG-0421 (455) -- CTG-1852 (456)\n",
      "\n",
      "#CTG-1852 (456) -- CTG-2369 (457)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2369 (457) -- CTG-2394 (458)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2394 (458) -- CTG-2169 (459)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2169 (459) -- CTG-0056 (460)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0056 (460) -- CTG-0217 (461)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0217 (461) -- CTG-2224 (462)\n",
      "\n",
      "#CTG-2224 (462) -- CTG-0312 (463)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0312 (463) -- CTG-0744 (464)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0744 (464) -- CTG-0370 (465)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0370 (465) -- CTG-2069 (466)\n",
      "\n",
      "#CTG-2069 (466) -- CTG-2479 (467)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2479 (467) -- CTG-0843 (468)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0843 (468) -- CTG-0851 (469)\n",
      "\n",
      "#CTG-0851 (469) -- CTG-0880 (470)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0880 (470) -- CTG-1012 (471)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1012 (471) -- CTG-0977 (472)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0977 (472) -- CTG-1023 (473)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1023 (473) -- CTG-0627 (474)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0627 (474) -- CTG-1535 (475)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1535 (475) -- CTG-1014 (476)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1014 (476) -- CTG-1013 (477)\n",
      "\n",
      "#CTG-1013 (477) -- CTG-0832 (478)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0832 (478) -- CTG-1360 (479)\n",
      "\n",
      "#CTG-1360 (479) -- CTG-1196 (480)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1196 (480) -- CTG-1054 (481)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1054 (481) -- CTG-0782 (482)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0782 (482) -- CTG-2537 (483)\n",
      "\n",
      "#CTG-2537 (483) -- CTG-1520 (484)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1520 (484) -- CTG-2028 (485)\n",
      "\n",
      "#CTG-2028 (485) -- CTG-1910 (486)\n",
      "\n",
      "#CTG-1910 (486) -- CTG-1670 (487)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1670 (487) -- CTG-0522 (488)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0522 (488) -- CTG-2279 (489)\n",
      "\n",
      "#CTG-2279 (489) -- CTG-1750 (490)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1750 (490) -- CTG-1900 (491)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1900 (491) -- CTG-2082 (492)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2082 (492) -- CTG-1124 (493)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1124 (493) -- CTG-2379 (494)\n",
      "\n",
      "#CTG-2379 (494) -- CTG-0948 (495)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0948 (495) -- CTG-0145 (496)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0145 (496) -- CTG-0200 (497)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0200 (497) -- CTG-0146 (498)\n",
      "\n",
      "#CTG-0146 (498) -- CTG-1967 (499)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1967 (499) -- CTG-2207 (500)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2207 (500) -- CTG-2031 (501)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2031 (501) -- CTG-2111 (502)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2111 (502) -- CTG-1447 (503)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1447 (503) -- CTG-2071 (504)\n",
      "\n",
      "#CTG-2071 (504) -- CTG-1148 (505)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1148 (505) -- CTG-1024 (506)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1024 (506) -- CTG-1710 (507)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1710 (507) -- CTG-0640 (508)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0640 (508) -- CTG-0492 (509)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0492 (509) -- CTG-2200 (510)\n",
      "\n",
      "#CTG-2200 (510) -- CTG-0850 (511)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0850 (511) -- CTG-1413 (512)\n",
      "\n",
      "#CTG-1413 (512) -- CTG-0981 (513)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0981 (513) -- CTG-1674 (514)\n",
      "\n",
      "#CTG-1674 (514) -- CTG-1765 (515)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1765 (515) -- CTG-0066 (516)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0066 (516) -- CTG-1805 (517)\n",
      "\n",
      "#CTG-1805 (517) -- CTG-2138 (518)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2138 (518) -- CTG-0609 (519)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0609 (519) -- CTG-0250 (520)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0250 (520) -- CTG-2025 (521)\n",
      "\n",
      "#CTG-2025 (521) -- CTG-0012 (522)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0012 (522) -- CTG-2685 (523)\n",
      "\n",
      "#CTG-2685 (523) -- CTG-0984 (524)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0984 (524) -- CTG-0703 (525)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0703 (525) -- CTG-0542 (526)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0542 (526) -- CTG-0861 (527)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0861 (527) -- CTG-0518 (528)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0518 (528) -- CTG-0360 (529)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0360 (529) -- CTG-1492 (530)\n",
      "\n",
      "#CTG-1492 (530) -- CTG-0503 (531)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0503 (531) -- CTG-0554 (532)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0554 (532) -- CTG-1382 (533)\n",
      "\n",
      "#CTG-1382 (533) -- CTG-1734 (534)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1734 (534) -- CTG-1090 (535)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1090 (535) -- CTG-0916 (536)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0916 (536) -- CTG-0883 (537)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0883 (537) -- CTG-1975 (538)\n",
      "\n",
      "#CTG-1975 (538) -- CTG-1019 (539)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1019 (539) -- CTG-1721 (540)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1721 (540) -- CTG-1726 (541)\n",
      "\n",
      "#CTG-1726 (541) -- CTG-1741 (542)\n",
      "\n",
      "#CTG-1741 (542) -- CTG-1753 (543)\n",
      "\n",
      "#CTG-1753 (543) -- CTG-1780 (544)\n",
      "\n",
      "#CTG-1780 (544) -- CTG-1784 (545)\n",
      "\n",
      "#CTG-1784 (545) -- CTG-2209 (546)\n",
      "\n",
      "#CTG-2209 (546) -- CTG-2128 (547)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2128 (547) -- CTG-0565 (548)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0565 (548) -- CTG-1368 (549)\n",
      "\n",
      "#CTG-1368 (549) -- CTG-1582 (550)\n",
      "\n",
      "#CTG-1582 (550) -- CTG-2124 (551)\n",
      "\n",
      "#CTG-2124 (551) -- CTG-1101 (552)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1101 (552) -- CTG-2070 (553)\n",
      "\n",
      "#CTG-2070 (553) -- CTG-0964 (554)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0964 (554) -- CTG-1128 (555)\n",
      "\n",
      "#CTG-1128 (555) -- CTG-1199 (556)\n",
      "\n",
      "#CTG-1199 (556) -- CTG-1534 (557)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1534 (557) -- CTG-1473 (558)\n",
      "\n",
      "#CTG-1473 (558) -- CTG-0772 (559)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0772 (559) -- CTG-0635 (560)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0635 (560) -- CTG-1306 (561)\n",
      "\n",
      "#CTG-1306 (561) -- CTG-0708 (562)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0708 (562) -- CTG-0711 (563)\n",
      "\n",
      "#CTG-0711 (563) -- CTG-1157 (564)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1157 (564) -- CTG-1235 (565)\n",
      "\n",
      "#CTG-1235 (565) -- CTG-1276 (566)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1276 (566) -- CTG-1238 (567)\n",
      "\n",
      "#CTG-1238 (567) -- CTG-1515 (568)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1515 (568) -- CTG-2203 (569)\n",
      "\n",
      "#CTG-2203 (569) -- CTG-2373 (570)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2373 (570) -- CTG-1575 (571)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1575 (571) -- CTG-1669 (572)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1669 (572) -- CTG-1239 (573)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1239 (573) -- CTG-1899 (574)\n",
      "\n",
      "#CTG-1899 (574) -- CTG-1647 (575)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1647 (575) -- CTG-1942 (576)\n",
      "\n",
      "#CTG-1942 (576) -- CTG-2372 (577)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2372 (577) -- CTG-1191 (578)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1191 (578) -- CTG-2258 (579)\n",
      "\n",
      "#CTG-2258 (579) -- CTG-1719 (580)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1719 (580) -- CTG-2299 (581)\n",
      "\n",
      "#CTG-2299 (581) -- CTG-1281 (582)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1281 (582) -- CTG-0180 (583)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0180 (583) -- CTG-1298 (584)\n",
      "\n",
      "#CTG-1298 (584) -- CTG-2298 (585)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2298 (585) -- CTG-0787 (586)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0787 (586) -- CTG-1659 (587)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1659 (587) -- CTG-1685 (588)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1685 (588) -- CTG-1384 (589)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1384 (589) -- CTG-1393 (590)\n",
      "\n",
      "#CTG-1393 (590) -- CTG-1210 (591)\n",
      "\n",
      "#CTG-1210 (591) -- CTG-1928 (592)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1928 (592) -- CTG-1927 (593)\n",
      "\n",
      "#CTG-1927 (593) -- CTG-1789 (594)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1789 (594) -- CTG-1274 (595)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1274 (595) -- CTG-2109 (596)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2109 (596) -- CTG-1484 (597)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1484 (597) -- CTG-1665 (598)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1665 (598) -- CTG-2627 (599)\n",
      "\n",
      "#CTG-2627 (599) -- CTG-0342 (600)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0342 (600) -- CTG-2099 (601)\n",
      "\n",
      "#CTG-2099 (601) -- CTG-0005 (602)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0005 (602) -- CTG-2166 (603)\n",
      "\n",
      "#CTG-2166 (603) -- CTG-2315 (604)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2315 (604) -- CTG-1162 (605)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1162 (605) -- CTG-1606 (606)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1606 (606) -- CTG-0886 (607)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0886 (607) -- CTG-1524 (608)\n",
      "\n",
      "#CTG-1524 (608) -- CTG-2567 (609)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2567 (609) -- CTG-1897 (610)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1897 (610) -- CTG-2407 (611)\n",
      "\n",
      "#CTG-2407 (611) -- CTG-1506 (612)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1506 (612) -- CTG-0077 (613)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0077 (613) -- CTG-0383 (614)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0383 (614) -- CTG-0741 (615)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0741 (615) -- CTG-0504 (616)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0504 (616) -- CTG-0730 (617)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0730 (617) -- CTG-0736 (618)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0736 (618) -- CTG-0402 (619)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0402 (619) -- CTG-0551 (620)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0551 (620) -- CTG-2383 (621)\n",
      "\n",
      "#CTG-2383 (621) -- CTG-0241 (622)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0241 (622) -- CTG-1500 (623)\n",
      "\n",
      "#CTG-1500 (623) -- CTG-1841 (624)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1841 (624) -- CTG-2000 (625)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2000 (625) -- CTG-0026 (626)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0026 (626) -- CTG-2552 (627)\n",
      "\n",
      "#CTG-2552 (627) -- CTG-1650 (628)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1650 (628) -- CTG-0735 (629)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0735 (629) -- CTG-0734 (630)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0734 (630) -- CTG-0779 (631)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0779 (631) -- CTG-0371 (632)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0371 (632) -- CTG-0477 (633)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0477 (633) -- CTG-1081 (634)\n",
      "\n",
      "#CTG-1081 (634) -- CTG-2533 (635)\n",
      "\n",
      "#CTG-2533 (635) -- CTG-1261 (636)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1261 (636) -- CTG-0794 (637)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0794 (637) -- CTG-0796 (638)\n",
      "\n",
      "#CTG-0796 (638) -- CTG-1066 (639)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1066 (639) -- CTG-1838 (640)\n",
      "\n",
      "#CTG-1838 (640) -- CTG-2514 (641)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2514 (641) -- CTG-1174 (642)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1174 (642) -- CTG-0591 (643)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0591 (643) -- CTG-1348 (644)\n",
      "\n",
      "#CTG-1348 (644) -- CTG-2221 (645)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2221 (645) -- CTG-0986 (646)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0986 (646) -- CTG-0356 (647)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0356 (647) -- CTG-1508 (648)\n",
      "\n",
      "#CTG-1508 (648) -- CTG-1514 (649)\n",
      "\n",
      "#CTG-1514 (649) -- CTG-0926 (650)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0926 (650) -- CTG-1308 (651)\n",
      "\n",
      "#CTG-1308 (651) -- CTG-1924 (652)\n",
      "\n",
      "#CTG-1924 (652) -- CTG-0401 (653)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0401 (653) -- CTG-2466 (654)\n",
      "\n",
      "#CTG-2466 (654) -- CTG-0997 (655)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0997 (655) -- CTG-1548 (656)\n",
      "\n",
      "#CTG-1548 (656) -- CTG-1406 (657)\n",
      "\n",
      "#CTG-1406 (657) -- CTG-1462 (658)\n",
      "\n",
      "#CTG-1462 (658) -- CTG-0201 (659)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0201 (659) -- CTG-2401 (660)\n",
      "\n",
      "#CTG-2401 (660) -- CTG-1986 (661)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1986 (661) -- CTG-1861 (662)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1861 (662) -- CTG-0743 (663)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0743 (663) -- CTG-0770 (664)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0770 (664) -- CTG-0177 (665)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0177 (665) -- CTG-0767 (666)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0767 (666) -- CTG-0615 (667)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0615 (667) -- CTG-1278 (668)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1278 (668) -- CTG-0400 (669)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0400 (669) -- CTG-0564 (670)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0564 (670) -- CTG-1619 (671)\n",
      "\n",
      "#CTG-1619 (671) -- CTG-0274 (672)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0274 (672) -- CTG-1531 (673)\n",
      "\n",
      "#CTG-1531 (673) -- CTG-2242 (674)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2242 (674) -- CTG-0878 (675)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0878 (675) -- CTG-1943 (676)\n",
      "\n",
      "#CTG-1943 (676) -- CTG-1134 (677)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1134 (677) -- CTG-1314 (678)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1314 (678) -- CTG-1776 (679)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1776 (679) -- CTG-1294 (680)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1294 (680) -- CTG-0810 (681)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0810 (681) -- CTG-1846 (682)\n",
      "\n",
      "#CTG-1846 (682) -- CTG-0953 (683)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0953 (683) -- CTG-0581 (684)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0581 (684) -- CTG-1471 (685)\n",
      "\n",
      "#CTG-1471 (685) -- CTG-0901 (686)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0901 (686) -- CTG-0483 (687)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0483 (687) -- CTG-0776 (688)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0776 (688) -- CTG-2368 (689)\n",
      "\n",
      "#CTG-2368 (689) -- CTG-0412 (690)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0412 (690) -- CTG-0630 (691)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0630 (691) -- CTG-1329 (692)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1329 (692) -- CTG-1342 (693)\n",
      "\n",
      "#CTG-1342 (693) -- CTG-0458 (694)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0458 (694) -- CTG-1682 (695)\n",
      "\n",
      "#CTG-1682 (695) -- CTG-0416 (696)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0416 (696) -- CTG-1241 (697)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1241 (697) -- CTG-0907 (698)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0907 (698) -- CTG-0235 (699)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0235 (699) -- CTG-0352 (700)\n",
      "\n",
      "#CTG-0352 (700) -- CTG-0351 (701)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0351 (701) -- CTG-0725 (702)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0725 (702) -- CTG-0432 (703)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0432 (703) -- CTG-1491 (704)\n",
      "\n",
      "#CTG-1491 (704) -- CTG-1297 (705)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1297 (705) -- CTG-0756 (706)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0756 (706) -- CTG-2303 (707)\n",
      "\n",
      "#CTG-2303 (707) -- CTG-0835 (708)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#CTG-0835 (708) -- CTG-2129 (709)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2129 (709) -- CTG-0720 (710)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0720 (710) -- CTG-1189 (711)\n",
      "\n",
      "#CTG-1189 (711) -- CTG-1048 (712)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1048 (712) -- CTG-0817 (713)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0817 (713) -- CTG-0816 (714)\n",
      "\n",
      "#CTG-0816 (714) -- CTG-0272 (715)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0272 (715) -- CTG-0989 (716)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0989 (716) -- CTG-0213 (717)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0213 (717) -- CTG-2170 (718)\n",
      "\n",
      "#CTG-2170 (718) -- CTG-1441 (719)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1441 (719) -- CTG-1240 (720)\n",
      "\n",
      "#CTG-1240 (720) -- CTG-0854 (721)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0854 (721) -- CTG-1000 (722)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1000 (722) -- CTG-1006 (723)\n",
      "\n",
      "#CTG-1006 (723) -- CTG-0282 (724)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0282 (724) -- CTG-1948 (725)\n",
      "\n",
      "#CTG-1948 (725) -- CTG-0620 (726)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0620 (726) -- CTG-0619 (727)\n",
      "\n",
      "#CTG-0619 (727) -- CTG-0459 (728)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0459 (728) -- CTG-1277 (729)\n",
      "\n",
      "#CTG-1277 (729) -- CTG-2416 (730)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2416 (730) -- CTG-1173 (731)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1173 (731) -- CTG-2278 (732)\n",
      "\n",
      "#CTG-2278 (732) -- CTG-1290 (733)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1290 (733) -- CTG-1414 (734)\n",
      "\n",
      "#CTG-1414 (734) -- CTG-0035 (735)\n",
      "\n",
      "#CTG-0035 (735) -- CTG-1735 (736)\n",
      "\n",
      "#CTG-1735 (736) -- CTG-0647 (737)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0647 (737) -- CTG-2596 (738)\n",
      "\n",
      "#CTG-2596 (738) -- CTG-0871 (739)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0871 (739) -- CTG-1041 (740)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1041 (740) -- CTG-1007 (741)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1007 (741) -- CTG-0357 (742)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0357 (742) -- CTG-0122 (743)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0122 (743) -- CTG-1458 (744)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1458 (744) -- CTG-1538 (745)\n",
      "\n",
      "#CTG-1538 (745) -- CTG-0625 (746)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0625 (746) -- CTG-2034 (747)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2034 (747) -- CTG-1117 (748)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1117 (748) -- CTG-0940 (749)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0940 (749) -- CTG-1925 (750)\n",
      "\n",
      "#CTG-1925 (750) -- CTG-1610 (751)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1610 (751) -- CTG-1113 (752)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1113 (752) -- CTG-1011 (753)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1011 (753) -- CTG-2454 (754)\n",
      "\n",
      "#CTG-2454 (754) -- CTG-1176 (755)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1176 (755) -- CTG-0869 (756)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0869 (756) -- CTG-0413 (757)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0413 (757) -- CTG-0684 (758)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0684 (758) -- CTG-1016 (759)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1016 (759) -- CTG-2008 (760)\n",
      "\n",
      "#CTG-2008 (760) -- CTG-0602 (761)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0602 (761) -- CTG-0444 (762)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0444 (762) -- CTG-0803 (763)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0803 (763) -- CTG-0387 (764)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0387 (764) -- CTG-2029 (765)\n",
      "\n",
      "#CTG-2029 (765) -- CTG-0388 (766)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0388 (766) -- CTG-0839 (767)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0839 (767) -- CTG-0120 (768)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0120 (768) -- CTG-1271 (769)\n",
      "\n",
      "#CTG-1271 (769) -- CTG-2144 (770)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2144 (770) -- CTG-0289 (771)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0289 (771) -- CTG-0797 (772)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0797 (772) -- CTG-0181 (773)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0181 (773) -- CTG-2361 (774)\n",
      "\n",
      "#CTG-2361 (774) -- CTG-2419 (775)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2419 (775) -- CTG-1860 (776)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1860 (776) -- CTG-2294 (777)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2294 (777) -- CTG-0852 (778)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0852 (778) -- CTG-0063 (779)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0063 (779) -- CTG-1537 (780)\n",
      "\n",
      "#CTG-1537 (780) -- CTG-1551 (781)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1551 (781) -- CTG-1502 (782)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1502 (782) -- CTG-1801 (783)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1801 (783) -- CTG-0263 (784)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0263 (784) -- CTG-0658 (785)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0658 (785) -- CTG-0408 (786)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0408 (786) -- CTG-0663 (787)\n",
      "\n",
      "#CTG-0663 (787) -- CTG-0698 (788)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0698 (788) -- CTG-0757 (789)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0757 (789) -- CTG-1056 (790)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1056 (790) -- CTG-0638 (791)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0638 (791) -- CTG-1018 (792)\n",
      "\n",
      "#CTG-1018 (792) -- CTG-1139 (793)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1139 (793) -- CTG-1881 (794)\n",
      "\n",
      "#CTG-1881 (794) -- CTG-1830 (795)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1830 (795) -- CTG-2057 (796)\n",
      "\n",
      "#CTG-2057 (796) -- CTG-0815 (797)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0815 (797) -- CTG-0822 (798)\n",
      "\n",
      "#CTG-0822 (798) -- CTG-0381 (799)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0381 (799) -- CTG-1740 (800)\n",
      "\n",
      "#CTG-1740 (800) -- CTG-1839 (801)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1839 (801) -- CTG-1158 (802)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1158 (802) -- CTG-0603 (803)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0603 (803) -- CTG-1435 (804)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1435 (804) -- CTG-1248 (805)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1248 (805) -- CTG-1905 (806)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1905 (806) -- CTG-2276 (807)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2276 (807) -- CTG-2325 (808)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2325 (808) -- CTG-1752 (809)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1752 (809) -- CTG-1873 (810)\n",
      "\n",
      "#CTG-1873 (810) -- CTG-1822 (811)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1822 (811) -- CTG-1929 (812)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1929 (812) -- CTG-1326 (813)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1326 (813) -- CTG-1879 (814)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1879 (814) -- CTG-1811 (815)\n",
      "\n",
      "#CTG-1811 (815) -- CTG-2257 (816)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2257 (816) -- CTG-2525 (817)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2525 (817) -- CTG-2040 (818)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2040 (818) -- CTG-1596 (819)\n",
      "\n",
      "#CTG-1596 (819) -- CTG-2139 (820)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2139 (820) -- CTG-1469 (821)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1469 (821) -- CTG-1403 (822)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1403 (822) -- CTG-2194 (823)\n",
      "\n",
      "#CTG-2194 (823) -- CTG-1341 (824)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1341 (824) -- CTG-2078 (825)\n",
      "\n",
      "#CTG-2078 (825) -- CTG-2277 (826)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2277 (826) -- CTG-2485 (827)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2485 (827) -- CTG-1760 (828)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1760 (828) -- CTG-1844 (829)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1844 (829) -- CTG-2123 (830)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2123 (830) -- CTG-1129 (831)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1129 (831) -- CTG-1420 (832)\n",
      "\n",
      "#CTG-1420 (832) -- CTG-1386 (833)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1386 (833) -- CTG-1864 (834)\n",
      "\n",
      "#CTG-1864 (834) -- CTG-1578 (835)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1578 (835) -- CTG-1913 (836)\n",
      "\n",
      "#CTG-1913 (836) -- CTG-2435 (837)\n",
      "\n",
      "#CTG-2435 (837) -- CTG-2509 (838)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2509 (838) -- CTG-2316 (839)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2316 (839) -- CTG-2523 (840)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2523 (840) -- CTG-1624 (841)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1624 (841) -- CTG-2293 (842)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2293 (842) -- CTG-2508 (843)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2508 (843) -- CTG-2418 (844)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2418 (844) -- CTG-1629 (845)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1629 (845) -- CTG-1327 (846)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1327 (846) -- CTG-0914 (847)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0914 (847) -- CTG-1284 (848)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1284 (848) -- CTG-2233 (849)\n",
      "\n",
      "#CTG-2233 (849) -- CTG-1302 (850)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1302 (850) -- CTG-1880 (851)\n",
      "\n",
      "#CTG-1880 (851) -- CTG-1404 (852)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1404 (852) -- CTG-1367 (853)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1367 (853) -- CTG-1059 (854)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1059 (854) -- CTG-1330 (855)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1330 (855) -- CTG-2423 (856)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2423 (856) -- CTG-1617 (857)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1617 (857) -- CTG-2007 (858)\n",
      "\n",
      "#CTG-2007 (858) -- CTG-1681 (859)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1681 (859) -- CTG-1641 (860)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1641 (860) -- CTG-2613 (861)\n",
      "\n",
      "#CTG-2613 (861) -- CTG-1273 (862)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1273 (862) -- CTG-1806 (863)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1806 (863) -- CTG-0411 (864)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0411 (864) -- CTG-2521 (865)\n",
      "\n",
      "#CTG-2521 (865) -- CTG-2085 (866)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2085 (866) -- CTG-1782 (867)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1782 (867) -- CTG-0769 (868)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0769 (868) -- CTG-2437 (869)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2437 (869) -- CTG-1142 (870)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1142 (870) -- CTG-0419 (871)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0419 (871) -- CTG-2087 (872)\n",
      "\n",
      "#CTG-2087 (872) -- CTG-1186 (873)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1186 (873) -- CTG-0466 (874)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0466 (874) -- CTG-0659 (875)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0659 (875) -- CTG-0125 (876)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0125 (876) -- CTG-0988 (877)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0988 (877) -- CTG-1369 (878)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1369 (878) -- CTG-0754 (879)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0754 (879) -- CTG-1319 (880)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1319 (880) -- CTG-2518 (881)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2518 (881) -- CTG-1313 (882)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1313 (882) -- CTG-0156 (883)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0156 (883) -- CTG-2322 (884)\n",
      "\n",
      "#CTG-2322 (884) -- CTG-1907 (885)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1907 (885) -- CTG-1902 (886)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1902 (886) -- CTG-1188 (887)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1188 (887) -- CTG-0095 (888)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0095 (888) -- CTG-1672 (889)\n",
      "\n",
      "#CTG-1672 (889) -- CTG-2270 (890)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2270 (890) -- CTG-0439 (891)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0439 (891) -- CTG-0438 (892)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0438 (892) -- CTG-0240 (893)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0240 (893) -- CTG-1626 (894)\n",
      "\n",
      "#CTG-1626 (894) -- CTG-0337 (895)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0337 (895) -- CTG-0127 (896)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0127 (896) -- CTG-1767 (897)\n",
      "\n",
      "#CTG-1767 (897) -- CTG-1185 (898)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1185 (898) -- CTG-0809 (899)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0809 (899) -- CTG-0296 (900)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0296 (900) -- CTG-1334 (901)\n",
      "\n",
      "#CTG-1334 (901) -- CTG-0613 (902)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0613 (902) -- CTG-1309 (903)\n",
      "\n",
      "#CTG-1309 (903) -- CTG-0410 (904)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0410 (904) -- CTG-0071 (905)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0071 (905) -- CTG-2101 (906)\n",
      "\n",
      "#CTG-2101 (906) -- CTG-2501 (907)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2501 (907) -- CTG-0427 (908)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0427 (908) -- CTG-0624 (909)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0624 (909) -- CTG-1851 (910)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1851 (910) -- CTG-1170 (911)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1170 (911) -- CTG-1204 (912)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1204 (912) -- CTG-1252 (913)\n",
      "\n",
      "#CTG-1252 (913) -- CTG-1179 (914)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1179 (914) -- CTG-1970 (915)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1970 (915) -- CTG-1122 (916)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1122 (916) -- CTG-1100 (917)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1100 (917) -- CTG-1108 (918)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1108 (918) -- CTG-1813 (919)\n",
      "\n",
      "#CTG-1813 (919) -- CTG-0054 (920)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0054 (920) -- CTG-2463 (921)\n",
      "\n",
      "#CTG-2463 (921) -- CTG-1483 (922)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1483 (922) -- CTG-2223 (923)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2223 (923) -- CTG-2333 (924)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2333 (924) -- CTG-0292 (925)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0292 (925) -- CTG-0425 (926)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0425 (926) -- CTG-1226 (927)\n",
      "\n",
      "#CTG-1226 (927) -- CTG-0013 (928)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0013 (928) -- CTG-0858 (929)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0858 (929) -- CTG-1220 (930)\n",
      "\n",
      "#CTG-1220 (930) -- CTG-0544 (931)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0544 (931) -- CTG-1689 (932)\n",
      "\n",
      "#CTG-1689 (932) -- CTG-0543 (933)\n",
      "\n",
      "#CTG-0543 (933) -- CTG-2027 (934)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2027 (934) -- CTG-1073 (935)\n",
      "\n",
      "#CTG-1073 (935) -- CTG-0299 (936)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0299 (936) -- CTG-1109 (937)\n",
      "\n",
      "#CTG-1109 (937) -- CTG-0015 (938)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0015 (938) -- CTG-2220 (939)\n",
      "\n",
      "#CTG-2220 (939) -- CTG-0040 (940)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0040 (940) -- CTG-0394 (941)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0394 (941) -- CTG-0003 (942)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0003 (942) -- CTG-2272 (943)\n",
      "\n",
      "#CTG-2272 (943) -- CTG-1690 (944)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1690 (944) -- CTG-2102 (945)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2102 (945) -- CTG-2686 (946)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2686 (946) -- CTG-0009 (947)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0009 (947) -- CTG-0418 (948)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0418 (948) -- CTG-0079 (949)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0079 (949) -- CTG-2602 (950)\n",
      "\n",
      "#CTG-2602 (950) -- CTG-2147 (951)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2147 (951) -- CTG-0277 (952)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0277 (952) -- CTG-1532 (953)\n",
      "\n",
      "#CTG-1532 (953) -- CTG-2235 (954)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2235 (954) -- CTG-0897 (955)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0897 (955) -- CTG-0648 (956)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0648 (956) -- CTG-2549 (957)\n",
      "\n",
      "#CTG-2549 (957) -- CTG-2564 (958)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2564 (958) -- CTG-0588 (959)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0588 (959) -- CTG-1656 (960)\n",
      "\n",
      "#CTG-1656 (960) -- CTG-0205 (961)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0205 (961) -- CTG-1812 (962)\n",
      "\n",
      "#CTG-1812 (962) -- CTG-1509 (963)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1509 (963) -- CTG-0768 (964)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0768 (964) -- CTG-2362 (965)\n",
      "\n",
      "#CTG-2362 (965) -- CTG-0781 (966)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0781 (966) -- CTG-2638 (967)\n",
      "\n",
      "#CTG-2638 (967) -- CTG-1317 (968)\n",
      "\n",
      "#CTG-1317 (968) -- CTG-0350 (969)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0350 (969) -- CTG-0641 (970)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0641 (970) -- CTG-1794 (971)\n",
      "\n",
      "#CTG-1794 (971) -- CTG-0569 (972)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0569 (972) -- CTG-0645 (973)\n",
      "\n",
      "#CTG-0645 (973) -- CTG-0089 (974)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0089 (974) -- CTG-2619 (975)\n",
      "\n",
      "#CTG-2619 (975) -- CTG-1772 (976)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1772 (976) -- CTG-2092 (977)\n",
      "\n",
      "#CTG-2092 (977) -- CTG-0882 (978)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0882 (978) -- CTG-0128 (979)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0128 (979) -- CTG-1468 (980)\n",
      "\n",
      "#CTG-1468 (980) -- CTG-2393 (981)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2393 (981) -- CTG-0955 (982)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0955 (982) -- CTG-0116 (983)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0116 (983) -- CTG-2324 (984)\n",
      "\n",
      "#CTG-2324 (984) -- CTG-1340 (985)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1340 (985) -- CTG-2598 (986)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2598 (986) -- CTG-2095 (987)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2095 (987) -- CTG-2354 (988)\n",
      "\n",
      "#CTG-2354 (988) -- CTG-0909 (989)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0909 (989) -- CTG-0578 (990)\n",
      "\n",
      "#CTG-0578 (990) -- CTG-0570 (991)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0570 (991) -- CTG-0331 (992)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0331 (992) -- CTG-1633 (993)\n",
      "\n",
      "#CTG-1633 (993) -- CTG-1051 (994)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1051 (994) -- CTG-1312 (995)\n",
      "\n",
      "#CTG-1312 (995) -- CTG-0929 (996)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0929 (996) -- CTG-2430 (997)\n",
      "\n",
      "#CTG-2430 (997) -- CTG-0990 (998)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0990 (998) -- CTG-2367 (999)\n",
      "\n",
      "#CTG-2367 (999) -- CTG-1904 (1000)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1904 (1000) -- CTG-0846 (1001)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0846 (1001) -- CTG-1778 (1002)\n",
      "\n",
      "#CTG-1778 (1002) -- CTG-1178 (1003)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1178 (1003) -- CTG-0657 (1004)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0657 (1004) -- CTG-2001 (1005)\n",
      "\n",
      "#CTG-2001 (1005) -- CTG-1449 (1006)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1449 (1006) -- CTG-0818 (1007)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0818 (1007) -- CTG-1755 (1008)\n",
      "\n",
      "#CTG-1755 (1008) -- CTG-0552 (1009)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0552 (1009) -- CTG-2121 (1010)\n",
      "\n",
      "#CTG-2121 (1010) -- CTG-2141 (1011)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2141 (1011) -- CTG-1940 (1012)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1940 (1012) -- CTG-1504 (1013)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1504 (1013) -- CTG-2548 (1014)\n",
      "\n",
      "#CTG-2548 (1014) -- CTG-1098 (1015)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1098 (1015) -- CTG-0068 (1016)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0068 (1016) -- CTG-0069 (1017)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0069 (1017) -- CTG-0271 (1018)\n",
      "\n",
      "#CTG-0271 (1018) -- CTG-1560 (1019)\n",
      "\n",
      "#CTG-1560 (1019) -- CTG-0204 (1020)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0204 (1020) -- CTG-2204 (1021)\n",
      "\n",
      "#CTG-2204 (1021) -- CTG-2135 (1022)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2135 (1022) -- CTG-2493 (1023)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2493 (1023) -- CTG-0532 (1024)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0532 (1024) -- CTG-1409 (1025)\n",
      "\n",
      "#CTG-1409 (1025) -- CTG-0959 (1026)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0959 (1026) -- CTG-2334 (1027)\n",
      "\n",
      "#CTG-2334 (1027) -- CTG-1869 (1028)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1869 (1028) -- CTG-0497 (1029)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0497 (1029) -- CTG-2568 (1030)\n",
      "\n",
      "#CTG-2568 (1030) -- CTG-0498 (1031)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0498 (1031) -- CTG-0502 (1032)\n",
      "\n",
      "#CTG-0502 (1032) -- CTG-2269 (1033)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2269 (1033) -- CTG-0860 (1034)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0860 (1034) -- CTG-0763 (1035)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0763 (1035) -- CTG-1405 (1036)\n",
      "\n",
      "#CTG-1405 (1036) -- CTG-1444 (1037)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1444 (1037) -- CTG-1725 (1038)\n",
      "\n",
      "#CTG-1725 (1038) -- CTG-1854 (1039)\n",
      "\n",
      "#CTG-1854 (1039) -- CTG-2106 (1040)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2106 (1040) -- CTG-2345 (1041)\n",
      "\n",
      "#CTG-2345 (1041) -- CTG-0055 (1042)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0055 (1042) -- CTG-0100 (1043)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0100 (1043) -- CTG-0220 (1044)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0220 (1044) -- CTG-2047 (1045)\n",
      "\n",
      "#CTG-2047 (1045) -- CTG-2329 (1046)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2329 (1046) -- CTG-0150 (1047)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0150 (1047) -- CTG-0249 (1048)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0249 (1048) -- CTG-0780 (1049)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0780 (1049) -- CTG-0045 (1050)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0045 (1050) -- CTG-2033 (1051)\n",
      "\n",
      "#CTG-2033 (1051) -- CTG-2455 (1052)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2455 (1052) -- CTG-0168 (1053)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0168 (1053) -- CTG-0661 (1054)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0661 (1054) -- CTG-1973 (1055)\n",
      "\n",
      "#CTG-1973 (1055) -- CTG-1865 (1056)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1865 (1056) -- CTG-0017 (1057)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0017 (1057) -- CTG-2177 (1058)\n",
      "\n",
      "#CTG-2177 (1058) -- CTG-1267 (1059)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1267 (1059) -- CTG-2323 (1060)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2323 (1060) -- CTG-2222 (1061)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2222 (1061) -- CTG-0960 (1062)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0960 (1062) -- CTG-1453 (1063)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1453 (1063) -- CTG-0393 (1064)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0393 (1064) -- CTG-0469 (1065)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0469 (1065) -- CTG-0451 (1066)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0451 (1066) -- CTG-0539 (1067)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0539 (1067) -- CTG-0550 (1068)\n",
      "\n",
      "#CTG-0550 (1068) -- CTG-0298 (1069)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0298 (1069) -- CTG-1243 (1070)\n",
      "\n",
      "#CTG-1243 (1070) -- CTG-0721 (1071)\n",
      "\n",
      "#CTG-0721 (1071) -- CTG-1353 (1072)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1353 (1072) -- CTG-1490 (1073)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1490 (1073) -- CTG-0590 (1074)\n",
      "\n",
      "#CTG-0590 (1074) -- CTG-1307 (1075)\n",
      "\n",
      "#CTG-1307 (1075) -- CTG-0428 (1076)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0428 (1076) -- CTG-1517 (1077)\n",
      "\n",
      "#CTG-1517 (1077) -- CTG-1213 (1078)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1213 (1078) -- CTG-1371 (1079)\n",
      "\n",
      "#CTG-1371 (1079) -- CTG-1253 (1080)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1253 (1080) -- CTG-0391 (1081)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0391 (1081) -- CTG-0036 (1082)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0036 (1082) -- CTG-1430 (1083)\n",
      "\n",
      "#CTG-1430 (1083) -- CTG-1365 (1084)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1365 (1084) -- CTG-2363 (1085)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2363 (1085) -- CTG-0800 (1086)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0800 (1086) -- CTG-1245 (1087)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1245 (1087) -- CTG-1084 (1088)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1084 (1088) -- CTG-1663 (1089)\n",
      "\n",
      "#CTG-1663 (1089) -- CTG-0951 (1090)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0951 (1090) -- CTG-2440 (1091)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2440 (1091) -- CTG-0126 (1092)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0126 (1092) -- CTG-1395 (1093)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1395 (1093) -- CTG-1289 (1094)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1289 (1094) -- CTG-2009 (1095)\n",
      "\n",
      "#CTG-2009 (1095) -- CTG-0343 (1096)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0343 (1096) -- CTG-1625 (1097)\n",
      "\n",
      "#CTG-1625 (1097) -- CTG-0995 (1098)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0995 (1098) -- CTG-1343 (1099)\n",
      "\n",
      "#CTG-1343 (1099) -- CTG-1727 (1100)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1727 (1100) -- CTG-1282 (1101)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1282 (1101) -- CTG-0001 (1102)\n",
      "\n",
      "#CTG-0001 (1102) -- CTG-2517 (1103)\n",
      "\n",
      "#CTG-2517 (1103) -- CTG-1889 (1104)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1889 (1104) -- CTG-2600 (1105)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2600 (1105) -- CTG-2688 (1106)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2688 (1106) -- CTG-0309 (1107)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0309 (1107) -- CTG-0596 (1108)\n",
      "\n",
      "#CTG-0596 (1108) -- CTG-0261 (1109)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0261 (1109) -- CTG-1627 (1110)\n",
      "\n",
      "#CTG-1627 (1110) -- CTG-0836 (1111)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0836 (1111) -- CTG-0248 (1112)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0248 (1112) -- CTG-1519 (1113)\n",
      "\n",
      "#CTG-1519 (1113) -- CTG-1731 (1114)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1731 (1114) -- CTG-0913 (1115)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0913 (1115) -- CTG-0561 (1116)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0561 (1116) -- CTG-1821 (1117)\n",
      "\n",
      "#CTG-1821 (1117) -- CTG-0042 (1118)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0042 (1118) -- CTG-1372 (1119)\n",
      "\n",
      "#CTG-1372 (1119) -- CTG-2288 (1120)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2288 (1120) -- CTG-0773 (1121)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0773 (1121) -- CTG-0775 (1122)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0775 (1122) -- CTG-0688 (1123)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0688 (1123) -- CTG-0690 (1124)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0690 (1124) -- CTG-1892 (1125)\n",
      "\n",
      "#CTG-1892 (1125) -- CTG-1666 (1126)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1666 (1126) -- CTG-1135 (1127)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1135 (1127) -- CTG-1577 (1128)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1577 (1128) -- CTG-0380 (1129)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0380 (1129) -- CTG-1642 (1130)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1642 (1130) -- CTG-2285 (1131)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2285 (1131) -- CTG-0695 (1132)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0695 (1132) -- CTG-2161 (1133)\n",
      "\n",
      "#CTG-2161 (1133) -- CTG-0102 (1134)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0102 (1134) -- CTG-2565 (1135)\n",
      "\n",
      "#CTG-2565 (1135) -- CTG-0046 (1136)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0046 (1136) -- CTG-1808 (1137)\n",
      "\n",
      "#CTG-1808 (1137) -- CTG-2626 (1138)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2626 (1138) -- CTG-1708 (1139)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1708 (1139) -- CTG-2465 (1140)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2465 (1140) -- CTG-2541 (1141)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2541 (1141) -- CTG-2397 (1142)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2397 (1142) -- CTG-1156 (1143)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1156 (1143) -- CTG-0245 (1144)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0245 (1144) -- CTG-0239 (1145)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0239 (1145) -- CTG-1858 (1146)\n",
      "\n",
      "#CTG-1858 (1146) -- CTG-1256 (1147)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1256 (1147) -- CTG-1550 (1148)\n",
      "\n",
      "#CTG-1550 (1148) -- CTG-1724 (1149)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1724 (1149) -- CTG-1744 (1150)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1744 (1150) -- CTG-0081 (1151)\n",
      "\n",
      "#CTG-0081 (1151) -- CTG-1645 (1152)\n",
      "\n",
      "#CTG-1645 (1152) -- CTG-0573 (1153)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0573 (1153) -- CTG-0131 (1154)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0131 (1154) -- CTG-2505 (1155)\n",
      "\n",
      "#CTG-2505 (1155) -- CTG-2263 (1156)\n",
      "\n",
      "#CTG-2263 (1156) -- CTG-0930 (1157)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0930 (1157) -- CTG-1715 (1158)\n",
      "\n",
      "#CTG-1715 (1158) -- CTG-1713 (1159)\n",
      "\n",
      "#CTG-1713 (1159) -- CTG-1214 (1160)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1214 (1160) -- CTG-0134 (1161)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0134 (1161) -- CTG-2023 (1162)\n",
      "\n",
      "#CTG-2023 (1162) -- CTG-0814 (1163)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0814 (1163) -- CTG-0221 (1164)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0221 (1164) -- CTG-0478 (1165)\n",
      "\n",
      "#CTG-0478 (1165) -- CTG-1850 (1166)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1850 (1166) -- CTG-1698 (1167)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1698 (1167) -- CTG-0583 (1168)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0583 (1168) -- CTG-0101 (1169)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0101 (1169) -- CTG-0584 (1170)\n",
      "\n",
      "#CTG-0584 (1170) -- CTG-2002 (1171)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2002 (1171) -- CTG-2153 (1172)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2153 (1172) -- CTG-1585 (1173)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1585 (1173) -- CTG-0270 (1174)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0270 (1174) -- CTG-1718 (1175)\n",
      "\n",
      "#CTG-1718 (1175) -- CTG-1115 (1176)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1115 (1176) -- CTG-1728 (1177)\n",
      "\n",
      "#CTG-1728 (1177) -- CTG-0807 (1178)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0807 (1178) -- CTG-2307 (1179)\n",
      "\n",
      "#CTG-2307 (1179) -- CTG-0671 (1180)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0671 (1180) -- CTG-2261 (1181)\n",
      "\n",
      "#CTG-2261 (1181) -- CTG-0866 (1182)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0866 (1182) -- CTG-1768 (1183)\n",
      "\n",
      "#CTG-1768 (1183) -- CTG-0490 (1184)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0490 (1184) -- CTG-0473 (1185)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0473 (1185) -- CTG-1922 (1186)\n",
      "\n",
      "#CTG-1922 (1186) -- CTG-1742 (1187)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1742 (1187) -- CTG-0667 (1188)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0667 (1188) -- CTG-0811 (1189)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0811 (1189) -- CTG-0159 (1190)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0159 (1190) -- CTG-2189 (1191)\n",
      "\n",
      "#CTG-2189 (1191) -- CTG-0105 (1192)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0105 (1192) -- CTG-0091 (1193)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0091 (1193) -- CTG-1694 (1194)\n",
      "\n",
      "#CTG-1694 (1194) -- CTG-2049 (1195)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2049 (1195) -- CTG-0890 (1196)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0890 (1196) -- CTG-2355 (1197)\n",
      "\n",
      "#CTG-2355 (1197) -- CTG-0479 (1198)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0479 (1198) -- CTG-1908 (1199)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1908 (1199) -- CTG-0390 (1200)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0390 (1200) -- CTG-1082 (1201)\n",
      "\n",
      "#CTG-1082 (1201) -- CTG-1075 (1202)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1075 (1202) -- CTG-0255 (1203)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0255 (1203) -- CTG-1622 (1204)\n",
      "\n",
      "#CTG-1622 (1204) -- CTG-1618 (1205)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1618 (1205) -- CTG-0717 (1206)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0717 (1206) -- CTG-2089 (1207)\n",
      "\n",
      "#CTG-2089 (1207) -- CTG-0485 (1208)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0485 (1208) -- CTG-1945 (1209)\n",
      "\n",
      "#CTG-1945 (1209) -- CTG-0334 (1210)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0334 (1210) -- CTG-1333 (1211)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1333 (1211) -- CTG-0777 (1212)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0777 (1212) -- CTG-0764 (1213)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0764 (1213) -- CTG-0047 (1214)\n",
      "\n",
      "#CTG-0047 (1214) -- CTG-2287 (1215)\n",
      "\n",
      "#CTG-2287 (1215) -- CTG-1389 (1216)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1389 (1216) -- CTG-0187 (1217)\n",
      "\n",
      "#CTG-0187 (1217) -- CTG-0139 (1218)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0139 (1218) -- CTG-0707 (1219)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0707 (1219) -- CTG-1155 (1220)\n",
      "\n",
      "#CTG-1155 (1220) -- CTG-1184 (1221)\n",
      "\n",
      "#CTG-1184 (1221) -- CTG-0534 (1222)\n",
      "\n",
      "#CTG-0534 (1222) -- CTG-1096 (1223)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1096 (1223) -- CTG-0825 (1224)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0825 (1224) -- CTG-0924 (1225)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0924 (1225) -- CTG-2453 (1226)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2453 (1226) -- CTG-0114 (1227)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0114 (1227) -- CTG-1917 (1228)\n",
      "\n",
      "#CTG-1917 (1228) -- CTG-1359 (1229)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1359 (1229) -- CTG-0899 (1230)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0899 (1230) -- CTG-0894 (1231)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0894 (1231) -- CTG-0957 (1232)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0957 (1232) -- CTG-1071 (1233)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1071 (1233) -- CTG-1270 (1234)\n",
      "\n",
      "#CTG-1270 (1234) -- CTG-1130 (1235)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1130 (1235) -- CTG-2536 (1236)\n",
      "\n",
      "#CTG-2536 (1236) -- CTG-0546 (1237)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0546 (1237) -- CTG-0373 (1238)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0373 (1238) -- CTG-1111 (1239)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1111 (1239) -- CTG-0207 (1240)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0207 (1240) -- CTG-1292 (1241)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1292 (1241) -- CTG-0377 (1242)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0377 (1242) -- CTG-0731 (1243)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0731 (1243) -- CTG-2384 (1244)\n",
      "\n",
      "#CTG-2384 (1244) -- CTG-1058 (1245)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1058 (1245) -- CTG-1025 (1246)\n",
      "\n",
      "#CTG-1025 (1246) -- CTG-1010 (1247)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1010 (1247) -- CTG-0889 (1248)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0889 (1248) -- CTG-1720 (1249)\n",
      "\n",
      "#CTG-1720 (1249) -- CTG-0653 (1250)\n",
      "\n",
      "#CTG-0653 (1250) -- CTG-1980 (1251)\n",
      "\n",
      "#CTG-1980 (1251) -- CTG-2254 (1252)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2254 (1252) -- CTG-0290 (1253)\n",
      "\n",
      "#CTG-0290 (1253) -- CTG-1522 (1254)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1522 (1254) -- CTG-0058 (1255)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0058 (1255) -- CTG-1400 (1256)\n",
      "\n",
      "#CTG-1400 (1256) -- CTG-2206 (1257)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2206 (1257) -- CTG-1709 (1258)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1709 (1258) -- CTG-1285 (1259)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1285 (1259) -- CTG-0509 (1260)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0509 (1260) -- CTG-1454 (1261)\n",
      "\n",
      "#CTG-1454 (1261) -- CTG-0837 (1262)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0837 (1262) -- CTG-0415 (1263)\n",
      "\n",
      "#CTG-0415 (1263) -- CTG-0414 (1264)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0414 (1264) -- CTG-0304 (1265)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0304 (1265) -- CTG-0223 (1266)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0223 (1266) -- CTG-0236 (1267)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0236 (1267) -- CTG-1751 (1268)\n",
      "\n",
      "#CTG-1751 (1268) -- CTG-1039 (1269)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1039 (1269) -- CTG-1595 (1270)\n",
      "\n",
      "#CTG-1595 (1270) -- CTG-1060 (1271)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1060 (1271) -- CTG-1958 (1272)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1958 (1272) -- CTG-1494 (1273)\n",
      "\n",
      "#CTG-1494 (1273) -- CTG-2015 (1274)\n",
      "\n",
      "#CTG-2015 (1274) -- CTG-2193 (1275)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2193 (1275) -- CTG-1262 (1276)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1262 (1276) -- CTG-1114 (1277)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1114 (1277) -- CTG-2150 (1278)\n",
      "\n",
      "#CTG-2150 (1278) -- CTG-0175 (1279)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0175 (1279) -- CTG-2482 (1280)\n",
      "\n",
      "#CTG-2482 (1280) -- CTG-0141 (1281)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0141 (1281) -- CTG-1344 (1282)\n",
      "\n",
      "#CTG-1344 (1282) -- CTG-1692 (1283)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1692 (1283) -- CTG-0823 (1284)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0823 (1284) -- CTG-0879 (1285)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0879 (1285) -- CTG-0243 (1286)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0243 (1286) -- CTG-1328 (1287)\n",
      "\n",
      "#CTG-1328 (1287) -- CTG-1714 (1288)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1714 (1288) -- CTG-1425 (1289)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1425 (1289) -- CTG-2037 (1290)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2037 (1290) -- CTG-0753 (1291)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0753 (1291) -- CTG-0577 (1292)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0577 (1292) -- CTG-0197 (1293)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0197 (1293) -- CTG-2205 (1294)\n",
      "\n",
      "#CTG-2205 (1294) -- CTG-2215 (1295)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2215 (1295) -- CTG-1856 (1296)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1856 (1296) -- CTG-0389 (1297)\n",
      "\n",
      "#CTG-0389 (1297) -- CTG-1573 (1298)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1573 (1298) -- CTG-0453 (1299)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0453 (1299) -- CTG-0345 (1300)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0345 (1300) -- CTG-0344 (1301)\n",
      "\n",
      "#CTG-0344 (1301) -- CTG-0297 (1302)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0297 (1302) -- CTG-1676 (1303)\n",
      "\n",
      "#CTG-1676 (1303) -- CTG-0293 (1304)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0293 (1304) -- CTG-2216 (1305)\n",
      "\n",
      "#CTG-2216 (1305) -- CTG-0592 (1306)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0592 (1306) -- CTG-0032 (1307)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0032 (1307) -- CTG-1498 (1308)\n",
      "\n",
      "#CTG-1498 (1308) -- CTG-2114 (1309)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2114 (1309) -- CTG-1463 (1310)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1463 (1310) -- CTG-0209 (1311)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0209 (1311) -- CTG-0813 (1312)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0813 (1312) -- CTG-2409 (1313)\n",
      "\n",
      "#CTG-2409 (1313) -- CTG-0023 (1314)\n",
      "\n",
      "#CTG-0023 (1314) -- CTG-0656 (1315)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0656 (1315) -- CTG-2439 (1316)\n",
      "\n",
      "#CTG-2439 (1316) -- CTG-2542 (1317)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2542 (1317) -- CTG-0147 (1318)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0147 (1318) -- CTG-0148 (1319)\n",
      "\n",
      "#CTG-0148 (1319) -- CTG-1002 (1320)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1002 (1320) -- CTG-0214 (1321)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0214 (1321) -- CTG-1380 (1322)\n",
      "\n",
      "#CTG-1380 (1322) -- CTG-0954 (1323)\n",
      "\n",
      "#CTG-0954 (1323) -- CTG-0585 (1324)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0585 (1324) -- CTG-0208 (1325)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0208 (1325) -- CTG-0628 (1326)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0628 (1326) -- CTG-2156 (1327)\n",
      "\n",
      "#CTG-2156 (1327) -- CTG-0696 (1328)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0696 (1328) -- CTG-0538 (1329)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0538 (1329) -- CTG-0188 (1330)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0188 (1330) -- CTG-1489 (1331)\n",
      "\n",
      "#CTG-1489 (1331) -- CTG-1587 (1332)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1587 (1332) -- CTG-0329 (1333)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0329 (1333) -- CTG-0423 (1334)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0423 (1334) -- CTG-2592 (1335)\n",
      "\n",
      "#CTG-2592 (1335) -- CTG-0623 (1336)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0623 (1336) -- CTG-1954 (1337)\n",
      "\n",
      "#CTG-1954 (1337) -- CTG-0295 (1338)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0295 (1338) -- CTG-1320 (1339)\n",
      "\n",
      "#CTG-1320 (1339) -- CTG-1553 (1340)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1553 (1340) -- CTG-1141 (1341)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1141 (1341) -- CTG-1603 (1342)\n",
      "\n",
      "#CTG-1603 (1342) -- CTG-2255 (1343)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2255 (1343) -- CTG-1576 (1344)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1576 (1344) -- CTG-0347 (1345)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0347 (1345) -- CTG-1877 (1346)\n",
      "\n",
      "#CTG-1877 (1346) -- CTG-0560 (1347)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0560 (1347) -- CTG-1442 (1348)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1442 (1348) -- CTG-1600 (1349)\n",
      "\n",
      "#CTG-1600 (1349) -- CTG-0452 (1350)\n",
      "\n",
      "#CTG-0452 (1350) -- CTG-1586 (1351)\n",
      "\n",
      "#CTG-1586 (1351) -- CTG-0586 (1352)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0586 (1352) -- CTG-1644 (1353)\n",
      "\n",
      "#CTG-1644 (1353) -- CTG-0065 (1354)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0065 (1354) -- CTG-0749 (1355)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0749 (1355) -- CTG-0266 (1356)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0266 (1356) -- CTG-1554 (1357)\n",
      "\n",
      "#CTG-1554 (1357) -- CTG-1906 (1358)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1906 (1358) -- CTG-0976 (1359)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0976 (1359) -- CTG-0801 (1360)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0801 (1360) -- CTG-0931 (1361)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0931 (1361) -- CTG-1614 (1362)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1614 (1362) -- CTG-2290 (1363)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2290 (1363) -- CTG-0599 (1364)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0599 (1364) -- CTG-1222 (1365)\n",
      "\n",
      "#CTG-1222 (1365) -- CTG-0406 (1366)\n",
      "\n",
      "#CTG-0406 (1366) -- CTG-2253 (1367)\n",
      "\n",
      "#CTG-2253 (1367) -- CTG-1052 (1368)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1052 (1368) -- CTG-0189 (1369)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0189 (1369) -- CTG-2054 (1370)\n",
      "\n",
      "#CTG-2054 (1370) -- CTG-0144 (1371)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0144 (1371) -- CTG-1288 (1372)\n",
      "\n",
      "#CTG-1288 (1372) -- CTG-2382 (1373)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2382 (1373) -- CTG-0064 (1374)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0064 (1374) -- CTG-0093 (1375)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0093 (1375) -- CTG-1769 (1376)\n",
      "\n",
      "#CTG-1769 (1376) -- CTG-2010 (1377)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2010 (1377) -- CTG-1829 (1378)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1829 (1378) -- CTG-2127 (1379)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2127 (1379) -- CTG-0714 (1380)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0714 (1380) -- CTG-0396 (1381)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0396 (1381) -- CTG-1700 (1382)\n",
      "\n",
      "#CTG-1700 (1382) -- CTG-0138 (1383)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0138 (1383) -- CTG-1392 (1384)\n",
      "\n",
      "#CTG-1392 (1384) -- CTG-1350 (1385)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1350 (1385) -- CTG-1078 (1386)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1078 (1386) -- CTG-0994 (1387)\n",
      "\n",
      "#CTG-0994 (1387) -- CTG-1088 (1388)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1088 (1388) -- CTG-0728 (1389)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0728 (1389) -- CTG-2043 (1390)\n",
      "\n",
      "#CTG-2043 (1390) -- CTG-1549 (1391)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1549 (1391) -- CTG-0363 (1392)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0363 (1392) -- CTG-1280 (1393)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1280 (1393) -- CTG-1684 (1394)\n",
      "\n",
      "#CTG-1684 (1394) -- CTG-2143 (1395)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2143 (1395) -- CTG-0501 (1396)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0501 (1396) -- CTG-2607 (1397)\n",
      "\n",
      "#CTG-2607 (1397) -- CTG-2238 (1398)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2238 (1398) -- CTG-0759 (1399)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0759 (1399) -- CTG-2341 (1400)\n",
      "\n",
      "#CTG-2341 (1400) -- CTG-0517 (1401)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0517 (1401) -- CTG-0222 (1402)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0222 (1402) -- CTG-0520 (1403)\n",
      "\n",
      "#CTG-0520 (1403) -- CTG-0050 (1404)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0050 (1404) -- CTG-1664 (1405)\n",
      "\n",
      "#CTG-1664 (1405) -- CTG-0932 (1406)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0932 (1406) -- CTG-0937 (1407)\n",
      "\n",
      "#CTG-0937 (1407) -- CTG-1651 (1408)\n",
      "\n",
      "#CTG-1651 (1408) -- CTG-0378 (1409)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0378 (1409) -- CTG-0386 (1410)\n",
      "\n",
      "#CTG-0386 (1410) -- CTG-0863 (1411)\n",
      "\n",
      "#CTG-0863 (1411) -- CTG-0637 (1412)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0637 (1412) -- CTG-0048 (1413)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0048 (1413) -- CTG-1816 (1414)\n",
      "\n",
      "#CTG-1816 (1414) -- CTG-0133 (1415)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0133 (1415) -- CTG-0709 (1416)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0709 (1416) -- CTG-1408 (1417)\n",
      "\n",
      "#CTG-1408 (1417) -- CTG-0679 (1418)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0679 (1418) -- CTG-0027 (1419)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0027 (1419) -- CTG-2522 (1420)\n",
      "\n",
      "#CTG-2522 (1420) -- CTG-1536 (1421)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1536 (1421) -- CTG-2608 (1422)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2608 (1422) -- CTG-0314 (1423)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0314 (1423) -- CTG-2131 (1424)\n",
      "\n",
      "#CTG-2131 (1424) -- CTG-2067 (1425)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2067 (1425) -- CTG-0376 (1426)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0376 (1426) -- CTG-2376 (1427)\n",
      "\n",
      "#CTG-2376 (1427) -- CTG-0664 (1428)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0664 (1428) -- CTG-1874 (1429)\n",
      "\n",
      "#CTG-1874 (1429) -- CTG-2562 (1430)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2562 (1430) -- CTG-1774 (1431)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1774 (1431) -- CTG-0470 (1432)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0470 (1432) -- CTG-0508 (1433)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0508 (1433) -- CTG-0523 (1434)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0523 (1434) -- CTG-0788 (1435)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0788 (1435) -- CTG-0558 (1436)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0558 (1436) -- CTG-0872 (1437)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0872 (1437) -- CTG-2187 (1438)\n",
      "\n",
      "#CTG-2187 (1438) -- CTG-2196 (1439)\n",
      "\n",
      "#CTG-2196 (1439) -- CTG-0475 (1440)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0475 (1440) -- CTG-1021 (1441)\n",
      "\n",
      "#CTG-1021 (1441) -- CTG-1086 (1442)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1086 (1442) -- CTG-1143 (1443)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1143 (1443) -- CTG-1265 (1444)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1265 (1444) -- CTG-0971 (1445)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0971 (1445) -- CTG-0448 (1446)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0448 (1446) -- CTG-0233 (1447)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0233 (1447) -- CTG-1886 (1448)\n",
      "\n",
      "#CTG-1886 (1448) -- CTG-2265 (1449)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2265 (1449) -- CTG-2225 (1450)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2225 (1450) -- CTG-0446 (1451)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0446 (1451) -- CTG-1779 (1452)\n",
      "\n",
      "#CTG-1779 (1452) -- CTG-1166 (1453)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1166 (1453) -- CTG-0227 (1454)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0227 (1454) -- CTG-1594 (1455)\n",
      "\n",
      "#CTG-1594 (1455) -- CTG-2056 (1456)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2056 (1456) -- CTG-1357 (1457)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1357 (1457) -- CTG-2201 (1458)\n",
      "\n",
      "#CTG-2201 (1458) -- CTG-2357 (1459)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2357 (1459) -- CTG-0713 (1460)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0713 (1460) -- CTG-0670 (1461)\n",
      "\n",
      "#CTG-0670 (1461) -- CTG-0137 (1462)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0137 (1462) -- CTG-0136 (1463)\n",
      "\n",
      "#CTG-0136 (1463) -- CTG-0049 (1464)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0049 (1464) -- CTG-2163 (1465)\n",
      "\n",
      "#CTG-2163 (1465) -- CTG-2415 (1466)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2415 (1466) -- CTG-2459 (1467)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2459 (1467) -- CTG-0034 (1468)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0034 (1468) -- CTG-2282 (1469)\n",
      "\n",
      "#CTG-2282 (1469) -- CTG-1661 (1470)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1661 (1470) -- CTG-1855 (1471)\n",
      "\n",
      "#CTG-1855 (1471) -- CTG-1481 (1472)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1481 (1472) -- CTG-0514 (1473)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0514 (1473) -- CTG-0510 (1474)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0510 (1474) -- CTG-0218 (1475)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0218 (1475) -- CTG-1952 (1476)\n",
      "\n",
      "#CTG-1952 (1476) -- CTG-1303 (1477)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1303 (1477) -- CTG-0855 (1478)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0855 (1478) -- CTG-2510 (1479)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2510 (1479) -- CTG-1470 (1480)\n",
      "\n",
      "#CTG-1470 (1480) -- CTG-1431 (1481)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1431 (1481) -- CTG-0440 (1482)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0440 (1482) -- CTG-1566 (1483)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1566 (1483) -- CTG-0215 (1484)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0215 (1484) -- CTG-1832 (1485)\n",
      "\n",
      "#CTG-1832 (1485) -- CTG-0195 (1486)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0195 (1486) -- CTG-1272 (1487)\n",
      "\n",
      "#CTG-1272 (1487) -- CTG-0176 (1488)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0176 (1488) -- CTG-1487 (1489)\n",
      "\n",
      "#CTG-1487 (1489) -- CTG-0098 (1490)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0098 (1490) -- CTG-2296 (1491)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2296 (1491) -- CTG-0031 (1492)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0031 (1492) -- CTG-2687 (1493)\n",
      "\n",
      "#CTG-2687 (1493) -- CTG-1580 (1494)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1580 (1494) -- CTG-1072 (1495)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1072 (1495) -- CTG-0598 (1496)\n",
      "\n",
      "#CTG-0598 (1496) -- CTG-0442 (1497)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0442 (1497) -- CTG-0622 (1498)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0622 (1498) -- CTG-0166 (1499)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0166 (1499) -- CTG-1192 (1500)\n",
      "\n",
      "#CTG-1192 (1500) -- CTG-1763 (1501)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1763 (1501) -- CTG-2229 (1502)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2229 (1502) -- CTG-0864 (1503)\n",
      "\n",
      "#CTG-0864 (1503) -- CTG-0306 (1504)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0306 (1504) -- CTG-0303 (1505)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0303 (1505) -- CTG-1456 (1506)\n",
      "\n",
      "#CTG-1456 (1506) -- CTG-0398 (1507)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0398 (1507) -- CTG-1175 (1508)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1175 (1508) -- CTG-1564 (1509)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1564 (1509) -- CTG-1733 (1510)\n",
      "\n",
      "#CTG-1733 (1510) -- CTG-0827 (1511)\n",
      "\n",
      "#CTG-0827 (1511) -- CTG-0906 (1512)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0906 (1512) -- CTG-0600 (1513)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0600 (1513) -- CTG-0904 (1514)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0904 (1514) -- CTG-2107 (1515)\n",
      "\n",
      "#CTG-2107 (1515) -- CTG-0433 (1516)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0433 (1516) -- CTG-0662 (1517)\n",
      "\n",
      "#CTG-0662 (1517) -- CTG-2365 (1518)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2365 (1518) -- CTG-1818 (1519)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1818 (1519) -- CTG-0574 (1520)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0574 (1520) -- CTG-2237 (1521)\n",
      "\n",
      "#CTG-2237 (1521) -- CTG-0830 (1522)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0830 (1522) -- CTG-0933 (1523)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0933 (1523) -- CTG-0014 (1524)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0014 (1524) -- CTG-1410 (1525)\n",
      "\n",
      "#CTG-1410 (1525) -- CTG-1938 (1526)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1938 (1526) -- CTG-0028 (1527)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0028 (1527) -- CTG-1761 (1528)\n",
      "\n",
      "#CTG-1761 (1528) -- CTG-2046 (1529)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2046 (1529) -- CTG-1324 (1530)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1324 (1530) -- CTG-0676 (1531)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0676 (1531) -- CTG-1988 (1532)\n",
      "\n",
      "#CTG-1988 (1532) -- CTG-1443 (1533)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1443 (1533) -- CTG-1388 (1534)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1388 (1534) -- CTG-1062 (1535)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1062 (1535) -- CTG-2326 (1536)\n",
      "\n",
      "#CTG-2326 (1536) -- CTG-0228 (1537)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0228 (1537) -- CTG-1702 (1538)\n",
      "\n",
      "#CTG-1702 (1538) -- CTG-2063 (1539)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2063 (1539) -- CTG-1338 (1540)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1338 (1540) -- CTG-1510 (1541)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1510 (1541) -- CTG-2061 (1542)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2061 (1542) -- CTG-0441 (1543)\n",
      "\n",
      "#CTG-0441 (1543) -- CTG-2400 (1544)\n",
      "\n",
      "#CTG-2400 (1544) -- CTG-0905 (1545)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0905 (1545) -- CTG-0256 (1546)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0256 (1546) -- CTG-1703 (1547)\n",
      "\n",
      "#CTG-1703 (1547) -- CTG-0191 (1548)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0191 (1548) -- CTG-1495 (1549)\n",
      "\n",
      "#CTG-1495 (1549) -- CTG-1269 (1550)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1269 (1550) -- CTG-0107 (1551)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0107 (1551) -- CTG-2050 (1552)\n",
      "\n",
      "#CTG-2050 (1552) -- CTG-1036 (1553)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1036 (1553) -- CTG-0273 (1554)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0273 (1554) -- CTG-1593 (1555)\n",
      "\n",
      "#CTG-1593 (1555) -- CTG-0082 (1556)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0082 (1556) -- CTG-0359 (1557)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0359 (1557) -- CTG-1299 (1558)\n",
      "\n",
      "#CTG-1299 (1558) -- CTG-0512 (1559)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0512 (1559) -- CTG-2545 (1560)\n",
      "\n",
      "#CTG-2545 (1560) -- CTG-0740 (1561)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0740 (1561) -- CTG-2489 (1562)\n",
      "\n",
      "#CTG-2489 (1562) -- CTG-0555 (1563)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0555 (1563) -- CTG-1912 (1564)\n",
      "\n",
      "#CTG-1912 (1564) -- CTG-0965 (1565)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0965 (1565) -- CTG-1546 (1566)\n",
      "\n",
      "#CTG-1546 (1566) -- CTG-2017 (1567)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2017 (1567) -- CTG-0616 (1568)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0616 (1568) -- CTG-0154 (1569)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0154 (1569) -- CTG-1358 (1570)\n",
      "\n",
      "#CTG-1358 (1570) -- CTG-0999 (1571)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0999 (1571) -- CTG-1950 (1572)\n",
      "\n",
      "#CTG-1950 (1572) -- CTG-0229 (1573)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0229 (1573) -- CTG-0242 (1574)\n",
      "\n",
      "#CTG-0242 (1574) -- CTG-0829 (1575)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0829 (1575) -- CTG-2044 (1576)\n",
      "\n",
      "#CTG-2044 (1576) -- CTG-2438 (1577)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2438 (1577) -- CTG-0445 (1578)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0445 (1578) -- CTG-1446 (1579)\n",
      "\n",
      "#CTG-1446 (1579) -- CTG-1704 (1580)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1704 (1580) -- CTG-0468 (1581)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0468 (1581) -- CTG-1749 (1582)\n",
      "\n",
      "#CTG-1749 (1582) -- CTG-0888 (1583)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0888 (1583) -- CTG-0887 (1584)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0887 (1584) -- CTG-0785 (1585)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0785 (1585) -- CTG-1375 (1586)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1375 (1586) -- CTG-0061 (1587)\n",
      "\n",
      "#CTG-0061 (1587) -- CTG-0059 (1588)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0059 (1588) -- CTG-1436 (1589)\n",
      "\n",
      "#CTG-1436 (1589) -- CTG-1450 (1590)\n",
      "\n",
      "#CTG-1450 (1590) -- CTG-1543 (1591)\n",
      "\n",
      "#CTG-1543 (1591) -- CTG-1589 (1592)\n",
      "\n",
      "#CTG-1589 (1592) -- CTG-1630 (1593)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1630 (1593) -- CTG-2090 (1594)\n",
      "\n",
      "#CTG-2090 (1594) -- CTG-0610 (1595)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0610 (1595) -- CTG-1581 (1596)\n",
      "\n",
      "#CTG-1581 (1596) -- CTG-1828 (1597)\n",
      "\n",
      "#CTG-1828 (1597) -- CTG-1390 (1598)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1390 (1598) -- CTG-0642 (1599)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0642 (1599) -- CTG-0431 (1600)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0431 (1600) -- CTG-0262 (1601)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0262 (1601) -- CTG-2292 (1602)\n",
      "\n",
      "#CTG-2292 (1602) -- CTG-0379 (1603)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0379 (1603) -- CTG-0088 (1604)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0088 (1604) -- CTG-1745 (1605)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1745 (1605) -- CTG-0582 (1606)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0582 (1606) -- CTG-2391 (1607)\n",
      "\n",
      "#CTG-2391 (1607) -- CTG-1361 (1608)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1361 (1608) -- CTG-0367 (1609)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0367 (1609) -- CTG-1049 (1610)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1049 (1610) -- CTG-0704 (1611)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0704 (1611) -- CTG-0745 (1612)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0745 (1612) -- CTG-0746 (1613)\n",
      "\n",
      "#CTG-0746 (1613) -- CTG-0575 (1614)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0575 (1614) -- CTG-0053 (1615)\n",
      "\n",
      "#CTG-0053 (1615) -- CTG-1590 (1616)\n",
      "\n",
      "#CTG-1590 (1616) -- CTG-0668 (1617)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0668 (1617) -- CTG-0170 (1618)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0170 (1618) -- CTG-2136 (1619)\n",
      "\n",
      "#CTG-2136 (1619) -- CTG-0018 (1620)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0018 (1620) -- CTG-2058 (1621)\n",
      "\n",
      "#CTG-2058 (1621) -- CTG-2084 (1622)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2084 (1622) -- CTG-2318 (1623)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2318 (1623) -- CTG-0171 (1624)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0171 (1624) -- CTG-2484 (1625)\n",
      "\n",
      "#CTG-2484 (1625) -- CTG-2464 (1626)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2464 (1626) -- CTG-0579 (1627)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0579 (1627) -- CTG-1699 (1628)\n",
      "\n",
      "#CTG-1699 (1628) -- CTG-1121 (1629)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1121 (1629) -- CTG-1264 (1630)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1264 (1630) -- CTG-2304 (1631)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2304 (1631) -- CTG-1182 (1632)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1182 (1632) -- CTG-0939 (1633)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0939 (1633) -- CTG-0153 (1634)\n",
      "\n",
      "#CTG-0153 (1634) -- CTG-2145 (1635)\n",
      "\n",
      "#CTG-2145 (1635) -- CTG-0199 (1636)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0199 (1636) -- CTG-1561 (1637)\n",
      "\n",
      "#CTG-1561 (1637) -- CTG-0340 (1638)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0340 (1638) -- CTG-0513 (1639)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0513 (1639) -- CTG-0891 (1640)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0891 (1640) -- CTG-0799 (1641)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0799 (1641) -- CTG-0802 (1642)\n",
      "\n",
      "#CTG-0802 (1642) -- CTG-0335 (1643)\n",
      "\n",
      "#CTG-0335 (1643) -- CTG-0549 (1644)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0549 (1644) -- CTG-0078 (1645)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0078 (1645) -- CTG-0285 (1646)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0285 (1646) -- CTG-0898 (1647)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0898 (1647) -- CTG-0918 (1648)\n",
      "\n",
      "#CTG-0918 (1648) -- CTG-0434 (1649)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0434 (1649) -- CTG-0210 (1650)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0210 (1650) -- CTG-1209 (1651)\n",
      "\n",
      "#CTG-1209 (1651) -- CTG-2471 (1652)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2471 (1652) -- CTG-1402 (1653)\n",
      "\n",
      "#CTG-1402 (1653) -- CTG-0025 (1654)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0025 (1654) -- CTG-1996 (1655)\n",
      "\n",
      "#CTG-1996 (1655) -- CTG-1387 (1656)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1387 (1656) -- CTG-0219 (1657)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0219 (1657) -- CTG-1503 (1658)\n",
      "\n",
      "#CTG-1503 (1658) -- CTG-0747 (1659)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0747 (1659) -- CTG-0536 (1660)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0536 (1660) -- CTG-0505 (1661)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0505 (1661) -- CTG-0104 (1662)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0104 (1662) -- CTG-1476 (1663)\n",
      "\n",
      "#CTG-1476 (1663) -- CTG-1089 (1664)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1089 (1664) -- CTG-0115 (1665)\n",
      "\n",
      "#CTG-0115 (1665) -- CTG-0395 (1666)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0395 (1666) -- CTG-1399 (1667)\n",
      "\n",
      "#CTG-1399 (1667) -- CTG-1046 (1668)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1046 (1668) -- CTG-0300 (1669)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0300 (1669) -- CTG-0487 (1670)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0487 (1670) -- CTG-0900 (1671)\n",
      "\n",
      "#CTG-0900 (1671) -- CTG-0474 (1672)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0474 (1672) -- CTG-0680 (1673)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0680 (1673) -- CTG-0157 (1674)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0157 (1674) -- CTG-1817 (1675)\n",
      "\n",
      "#CTG-1817 (1675) -- CTG-2424 (1676)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2424 (1676) -- CTG-1903 (1677)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1903 (1677) -- CTG-1748 (1678)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1748 (1678) -- CTG-0833 (1679)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0833 (1679) -- CTG-0437 (1680)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0437 (1680) -- CTG-1479 (1681)\n",
      "\n",
      "#CTG-1479 (1681) -- CTG-2546 (1682)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2546 (1682) -- CTG-0149 (1683)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0149 (1683) -- CTG-2148 (1684)\n",
      "\n",
      "#CTG-2148 (1684) -- CTG-0155 (1685)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0155 (1685) -- CTG-1612 (1686)\n",
      "\n",
      "#CTG-1612 (1686) -- CTG-2227 (1687)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2227 (1687) -- CTG-0527 (1688)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0527 (1688) -- CTG-2158 (1689)\n",
      "\n",
      "#CTG-2158 (1689) -- CTG-1737 (1690)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1737 (1690) -- CTG-1093 (1691)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1093 (1691) -- CTG-0778 (1692)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0778 (1692) -- CTG-0893 (1693)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0893 (1693) -- CTG-1065 (1694)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1065 (1694) -- CTG-0967 (1695)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0967 (1695) -- CTG-0562 (1696)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0562 (1696) -- CTG-2444 (1697)\n",
      "\n",
      "#CTG-2444 (1697) -- CTG-2313 (1698)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2313 (1698) -- CTG-2165 (1699)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2165 (1699) -- CTG-0917 (1700)\n",
      "\n",
      "#CTG-0917 (1700) -- CTG-0316 (1701)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0316 (1701) -- CTG-2093 (1702)\n",
      "\n",
      "#CTG-2093 (1702) -- CTG-0936 (1703)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0936 (1703) -- CTG-0067 (1704)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0067 (1704) -- CTG-2154 (1705)\n",
      "\n",
      "#CTG-2154 (1705) -- CTG-0355 (1706)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0355 (1706) -- CTG-0790 (1707)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0790 (1707) -- CTG-0284 (1708)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0284 (1708) -- CTG-2240 (1709)\n",
      "\n",
      "#CTG-2240 (1709) -- CTG-1085 (1710)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1085 (1710) -- CTG-0604 (1711)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0604 (1711) -- CTG-2520 (1712)\n",
      "\n",
      "#CTG-2520 (1712) -- CTG-1015 (1713)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1015 (1713) -- CTG-0426 (1714)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0426 (1714) -- CTG-1474 (1715)\n",
      "\n",
      "#CTG-1474 (1715) -- CTG-0533 (1716)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0533 (1716) -- CTG-1909 (1717)\n",
      "\n",
      "#CTG-1909 (1717) -- CTG-1987 (1718)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1987 (1718) -- CTG-1797 (1719)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1797 (1719) -- CTG-0315 (1720)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0315 (1720) -- CTG-1529 (1721)\n",
      "\n",
      "#CTG-1529 (1721) -- CTG-0755 (1722)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0755 (1722) -- CTG-0819 (1723)\n",
      "\n",
      "#CTG-0819 (1723) -- CTG-0597 (1724)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0597 (1724) -- CTG-0167 (1725)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0167 (1725) -- CTG-2488 (1726)\n",
      "\n",
      "#CTG-2488 (1726) -- CTG-1771 (1727)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1771 (1727) -- CTG-2245 (1728)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2245 (1728) -- CTG-0206 (1729)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0206 (1729) -- CTG-1878 (1730)\n",
      "\n",
      "#CTG-1878 (1730) -- CTG-0968 (1731)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0968 (1731) -- CTG-0727 (1732)\n",
      "\n",
      "#CTG-0727 (1732) -- CTG-0726 (1733)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0726 (1733) -- CTG-0925 (1734)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0925 (1734) -- CTG-1562 (1735)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1562 (1735) -- CTG-0338 (1736)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0338 (1736) -- CTG-1459 (1737)\n",
      "\n",
      "#CTG-1459 (1737) -- CTG-1675 (1738)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1675 (1738) -- CTG-0251 (1739)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0251 (1739) -- CTG-0151 (1740)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0151 (1740) -- CTG-1788 (1741)\n",
      "\n",
      "#CTG-1788 (1741) -- CTG-0326 (1742)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0326 (1742) -- CTG-2004 (1743)\n",
      "\n",
      "#CTG-2004 (1743) -- CTG-1094 (1744)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1094 (1744) -- CTG-1119 (1745)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1119 (1745) -- CTG-0480 (1746)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0480 (1746) -- CTG-0482 (1747)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0482 (1747) -- CTG-0961 (1748)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0961 (1748) -- CTG-0085 (1749)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0085 (1749) -- CTG-0086 (1750)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0086 (1750) -- CTG-0117 (1751)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0117 (1751) -- CTG-1557 (1752)\n",
      "\n",
      "#CTG-1557 (1752) -- CTG-0132 (1753)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0132 (1753) -- CTG-2492 (1754)\n",
      "\n",
      "#CTG-2492 (1754) -- CTG-0030 (1755)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0030 (1755) -- CTG-1759 (1756)\n",
      "\n",
      "#CTG-1759 (1756) -- CTG-1571 (1757)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1571 (1757) -- CTG-1648 (1758)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1648 (1758) -- CTG-1893 (1759)\n",
      "\n",
      "#CTG-1893 (1759) -- CTG-2311 (1760)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2311 (1760) -- CTG-0646 (1761)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0646 (1761) -- CTG-2113 (1762)\n",
      "\n",
      "#CTG-2113 (1762) -- CTG-0729 (1763)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0729 (1763) -- CTG-0723 (1764)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0723 (1764) -- CTG-1346 (1765)\n",
      "\n",
      "#CTG-1346 (1765) -- CTG-0946 (1766)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0946 (1766) -- CTG-0673 (1767)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0673 (1767) -- CTG-1366 (1768)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1366 (1768) -- CTG-1200 (1769)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1200 (1769) -- CTG-1031 (1770)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1031 (1770) -- CTG-1831 (1771)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1831 (1771) -- CTG-1895 (1772)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1895 (1772) -- CTG-0537 (1773)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0537 (1773) -- CTG-2297 (1774)\n",
      "\n",
      "#CTG-2297 (1774) -- CTG-2390 (1775)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2390 (1775) -- CTG-1756 (1776)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1756 (1776) -- CTG-0260 (1777)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0260 (1777) -- CTG-2452 (1778)\n",
      "\n",
      "#CTG-2452 (1778) -- CTG-1352 (1779)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1352 (1779) -- CTG-0938 (1780)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0938 (1780) -- CTG-0450 (1781)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0450 (1781) -- CTG-2319 (1782)\n",
      "\n",
      "#CTG-2319 (1782) -- CTG-0849 (1783)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0849 (1783) -- CTG-1378 (1784)\n",
      "\n",
      "#CTG-1378 (1784) -- CTG-1251 (1785)\n",
      "\n",
      "#CTG-1251 (1785) -- CTG-0161 (1786)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0161 (1786) -- CTG-2577 (1787)\n",
      "\n",
      "#CTG-2577 (1787) -- CTG-2639 (1788)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2639 (1788) -- CTG-0307 (1789)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0307 (1789) -- CTG-2198 (1790)\n",
      "\n",
      "#CTG-2198 (1790) -- CTG-0288 (1791)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0288 (1791) -- CTG-2314 (1792)\n",
      "\n",
      "#CTG-2314 (1792) -- CTG-2410 (1793)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2410 (1793) -- CTG-1598 (1794)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1598 (1794) -- CTG-0392 (1795)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0392 (1795) -- CTG-0732 (1796)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0732 (1796) -- CTG-0019 (1797)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0019 (1797) -- CTG-2507 (1798)\n",
      "\n",
      "#CTG-2507 (1798) -- CTG-1883 (1799)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1883 (1799) -- CTG-0524 (1800)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0524 (1800) -- CTG-0265 (1801)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0265 (1801) -- CTG-1466 (1802)\n",
      "\n",
      "#CTG-1466 (1802) -- CTG-0738 (1803)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0738 (1803) -- CTG-2116 (1804)\n",
      "\n",
      "#CTG-2116 (1804) -- CTG-2079 (1805)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2079 (1805) -- CTG-0716 (1806)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0716 (1806) -- CTG-0183 (1807)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0183 (1807) -- CTG-0006 (1808)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0006 (1808) -- CTG-1349 (1809)\n",
      "\n",
      "#CTG-1349 (1809) -- CTG-1332 (1810)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1332 (1810) -- CTG-1679 (1811)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1679 (1811) -- CTG-0318 (1812)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0318 (1812) -- CTG-2133 (1813)\n",
      "\n",
      "#CTG-2133 (1813) -- CTG-0321 (1814)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0321 (1814) -- CTG-0332 (1815)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0332 (1815) -- CTG-2612 (1816)\n",
      "\n",
      "#CTG-2612 (1816) -- CTG-2226 (1817)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2226 (1817) -- CTG-0198 (1818)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0198 (1818) -- CTG-0203 (1819)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0203 (1819) -- CTG-0874 (1820)\n",
      "\n",
      "#CTG-0874 (1820) -- CTG-0247 (1821)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0247 (1821) -- CTG-0234 (1822)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0234 (1822) -- CTG-0665 (1823)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0665 (1823) -- CTG-0135 (1824)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0135 (1824) -- CTG-1635 (1825)\n",
      "\n",
      "#CTG-1635 (1825) -- CTG-0409 (1826)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0409 (1826) -- CTG-1478 (1827)\n",
      "\n",
      "#CTG-1478 (1827) -- CTG-0211 (1828)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0211 (1828) -- CTG-0530 (1829)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0530 (1829) -- CTG-0952 (1830)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0952 (1830) -- CTG-2308 (1831)\n",
      "\n",
      "#CTG-2308 (1831) -- CTG-0328 (1832)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0328 (1832) -- CTG-0327 (1833)\n",
      "\n",
      "#CTG-0327 (1833) -- CTG-1868 (1834)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1868 (1834) -- CTG-2402 (1835)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2402 (1835) -- CTG-1250 (1836)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1250 (1836) -- CTG-2539 (1837)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2539 (1837) -- CTG-1236 (1838)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1236 (1838) -- CTG-2553 (1839)\n",
      "\n",
      "#CTG-2553 (1839) -- CTG-1965 (1840)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1965 (1840) -- CTG-1197 (1841)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1197 (1841) -- CTG-0196 (1842)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0196 (1842) -- CTG-1933 (1843)\n",
      "\n",
      "#CTG-1933 (1843) -- CTG-1224 (1844)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1224 (1844) -- CTG-0324 (1845)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0324 (1845) -- CTG-0655 (1846)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0655 (1846) -- CTG-2018 (1847)\n",
      "\n",
      "#CTG-2018 (1847) -- CTG-0771 (1848)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0771 (1848) -- CTG-0589 (1849)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0589 (1849) -- CTG-0928 (1850)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0928 (1850) -- CTG-0742 (1851)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0742 (1851) -- CTG-0317 (1852)\n",
      "\n",
      "#CTG-0317 (1852) -- CTG-1840 (1853)\n",
      "\n",
      "#CTG-1840 (1853) -- CTG-0341 (1854)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0341 (1854) -- CTG-0621 (1855)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0621 (1855) -- CTG-0631 (1856)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0631 (1856) -- CTG-0687 (1857)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0687 (1857) -- CTG-1033 (1858)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1033 (1858) -- CTG-2677 (1859)\n",
      "\n",
      "#CTG-2677 (1859) -- CTG-0681 (1860)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0681 (1860) -- CTG-0672 (1861)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0672 (1861) -- CTG-2597 (1862)\n",
      "\n",
      "#CTG-2597 (1862) -- CTG-1304 (1863)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1304 (1863) -- CTG-1159 (1864)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1159 (1864) -- CTG-2344 (1865)\n",
      "\n",
      "#CTG-2344 (1865) -- CTG-1237 (1866)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1237 (1866) -- CTG-1640 (1867)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1640 (1867) -- CTG-1646 (1868)\n",
      "\n",
      "#CTG-1646 (1868) -- CTG-1863 (1869)\n",
      "\n",
      "#CTG-1863 (1869) -- CTG-2387 (1870)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2387 (1870) -- CTG-0974 (1871)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0974 (1871) -- CTG-1613 (1872)\n",
      "\n",
      "#CTG-1613 (1872) -- CTG-0593 (1873)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0593 (1873) -- CTG-0686 (1874)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0686 (1874) -- CTG-1888 (1875)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1888 (1875) -- CTG-0927 (1876)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0927 (1876) -- CTG-1381 (1877)\n",
      "\n",
      "#CTG-1381 (1877) -- CTG-0041 (1878)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0041 (1878) -- CTG-2534 (1879)\n",
      "\n",
      "#CTG-2534 (1879) -- CTG-0043 (1880)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0043 (1880) -- CTG-0029 (1881)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0029 (1881) -- CTG-1982 (1882)\n",
      "\n",
      "#CTG-1982 (1882) -- CTG-2281 (1883)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2281 (1883) -- CTG-1383 (1884)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1383 (1884) -- CTG-1247 (1885)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1247 (1885) -- CTG-0190 (1886)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0190 (1886) -- CTG-0404 (1887)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0404 (1887) -- CTG-1599 (1888)\n",
      "\n",
      "#CTG-1599 (1888) -- CTG-1701 (1889)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1701 (1889) -- CTG-0212 (1890)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0212 (1890) -- CTG-1770 (1891)\n",
      "\n",
      "#CTG-1770 (1891) -- CTG-0617 (1892)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0617 (1892) -- CTG-0705 (1893)\n",
      "\n",
      "#CTG-0705 (1893) -- CTG-0982 (1894)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0982 (1894) -- CTG-1293 (1895)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1293 (1895) -- CTG-2202 (1896)\n",
      "\n",
      "#CTG-2202 (1896) -- CTG-0232 (1897)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0232 (1897) -- CTG-2475 (1898)\n",
      "\n",
      "#CTG-2475 (1898) -- CTG-1602 (1899)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1602 (1899) -- CTG-0179 (1900)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0179 (1900) -- CTG-0038 (1901)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0038 (1901) -- CTG-2097 (1902)\n",
      "\n",
      "#CTG-2097 (1902) -- CTG-2104 (1903)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2104 (1903) -- CTG-0403 (1904)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0403 (1904) -- CTG-1993 (1905)\n",
      "\n",
      "#CTG-1993 (1905) -- CTG-2149 (1906)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2149 (1906) -- CTG-1632 (1907)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1632 (1907) -- CTG-2210 (1908)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2210 (1908) -- CTG-0644 (1909)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0644 (1909) -- CTG-2249 (1910)\n",
      "\n",
      "#CTG-2249 (1910) -- CTG-2006 (1911)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2006 (1911) -- CTG-0294 (1912)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0294 (1912) -- CTG-0301 (1913)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0301 (1913) -- CTG-0349 (1914)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0349 (1914) -- CTG-2589 (1915)\n",
      "\n",
      "#CTG-2589 (1915) -- CTG-1118 (1916)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1118 (1916) -- CTG-0710 (1917)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0710 (1917) -- CTG-0417 (1918)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0417 (1918) -- CTG-2364 (1919)\n",
      "\n",
      "#CTG-2364 (1919) -- CTG-1032 (1920)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1032 (1920) -- CTG-0202 (1921)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0202 (1921) -- CTG-2011 (1922)\n",
      "\n",
      "#CTG-2011 (1922) -- CTG-0923 (1923)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0923 (1923) -- CTG-0467 (1924)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0467 (1924) -- CTG-0257 (1925)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0257 (1925) -- CTG-0908 (1926)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0908 (1926) -- CTG-0975 (1927)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0975 (1927) -- CTG-2250 (1928)\n",
      "\n",
      "#CTG-2250 (1928) -- CTG-0080 (1929)\n",
      "\n",
      "#CTG-0080 (1929) -- CTG-0969 (1930)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0969 (1930) -- CTG-0507 (1931)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0507 (1931) -- CTG-2005 (1932)\n",
      "\n",
      "#CTG-2005 (1932) -- CTG-1611 (1933)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1611 (1933) -- CTG-0834 (1934)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0834 (1934) -- CTG-0841 (1935)\n",
      "\n",
      "#CTG-0841 (1935) -- CTG-2309 (1936)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2309 (1936) -- CTG-0084 (1937)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0084 (1937) -- CTG-1957 (1938)\n",
      "\n",
      "#CTG-1957 (1938) -- CTG-1723 (1939)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1723 (1939) -- CTG-0022 (1940)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0022 (1940) -- CTG-1423 (1941)\n",
      "\n",
      "#CTG-1423 (1941) -- CTG-1527 (1942)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1527 (1942) -- CTG-1680 (1943)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1680 (1943) -- CTG-2284 (1944)\n",
      "\n",
      "#CTG-2284 (1944) -- CTG-0848 (1945)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0848 (1945) -- CTG-0283 (1946)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0283 (1946) -- CTG-0002 (1947)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0002 (1947) -- CTG-1738 (1948)\n",
      "\n",
      "#CTG-1738 (1948) -- CTG-2088 (1949)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2088 (1949) -- CTG-2076 (1950)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2076 (1950) -- CTG-2606 (1951)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2606 (1951) -- CTG-1259 (1952)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1259 (1952) -- CTG-1998 (1953)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1998 (1953) -- CTG-0499 (1954)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0499 (1954) -- CTG-0553 (1955)\n",
      "\n",
      "#CTG-0553 (1955) -- CTG-1833 (1956)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1833 (1956) -- CTG-0606 (1957)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0606 (1957) -- CTG-1885 (1958)\n",
      "\n",
      "#CTG-1885 (1958) -- CTG-1144 (1959)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1144 (1959) -- CTG-0158 (1960)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0158 (1960) -- CTG-1891 (1961)\n",
      "\n",
      "#CTG-1891 (1961) -- CTG-0348 (1962)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0348 (1962) -- CTG-1511 (1963)\n",
      "\n",
      "#CTG-1511 (1963) -- CTG-0072 (1964)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0072 (1964) -- CTG-0073 (1965)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0073 (1965) -- CTG-0845 (1966)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0845 (1966) -- CTG-0806 (1967)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0806 (1967) -- CTG-1712 (1968)\n",
      "\n",
      "#CTG-1712 (1968) -- CTG-0950 (1969)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0950 (1969) -- CTG-0325 (1970)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0325 (1970) -- CTG-2256 (1971)\n",
      "\n",
      "#CTG-2256 (1971) -- CTG-0311 (1972)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0311 (1972) -- CTG-0310 (1973)\n",
      "\n",
      "#CTG-0310 (1973) -- CTG-2604 (1974)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2604 (1974) -- CTG-1190 (1975)\n",
      "\n",
      "#CTG-1190 (1975) -- CTG-2020 (1976)\n",
      "\n",
      "#CTG-2020 (1976) -- CTG-1795 (1977)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1795 (1977) -- CTG-2183 (1978)\n",
      "\n",
      "#CTG-2183 (1978) -- CTG-0789 (1979)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0789 (1979) -- CTG-1040 (1980)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1040 (1980) -- CTG-0540 (1981)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0540 (1981) -- CTG-1047 (1982)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1047 (1982) -- CTG-1126 (1983)\n",
      "\n",
      "#CTG-1126 (1983) -- CTG-2048 (1984)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2048 (1984) -- CTG-1857 (1985)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1857 (1985) -- CTG-1120 (1986)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1120 (1986) -- CTG-0160 (1987)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0160 (1987) -- CTG-2184 (1988)\n",
      "\n",
      "#CTG-2184 (1988) -- CTG-2348 (1989)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2348 (1989) -- CTG-0092 (1990)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0092 (1990) -- CTG-1525 (1991)\n",
      "\n",
      "#CTG-1525 (1991) -- CTG-1921 (1992)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1921 (1992) -- CTG-1460 (1993)\n",
      "\n",
      "#CTG-1460 (1993) -- CTG-1419 (1994)\n",
      "\n",
      "#CTG-1419 (1994) -- CTG-1605 (1995)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1605 (1995) -- CTG-0253 (1996)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0253 (1996) -- CTG-1716 (1997)\n",
      "\n",
      "#CTG-1716 (1997) -- CTG-0652 (1998)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0652 (1998) -- CTG-0798 (1999)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0798 (1999) -- CTG-0429 (2000)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0429 (2000) -- CTG-2676 (2001)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2676 (2001) -- CTG-0375 (2002)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0375 (2002) -- CTG-0142 (2003)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0142 (2003) -- CTG-0364 (2004)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0364 (2004) -- CTG-0083 (2005)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0083 (2005) -- CTG-2395 (2006)\n",
      "\n",
      "#CTG-2395 (2006) -- CTG-2302 (2007)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2302 (2007) -- CTG-0060 (2008)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0060 (2008) -- CTG-2244 (2009)\n",
      "\n",
      "#CTG-2244 (2009) -- CTG-1230 (2010)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1230 (2010) -- CTG-0699 (2011)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0699 (2011) -- CTG-0110 (2012)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0110 (2012) -- CTG-1757 (2013)\n",
      "\n",
      "#CTG-1757 (2013) -- CTG-2274 (2014)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2274 (2014) -- CTG-0106 (2015)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0106 (2015) -- CTG-1673 (2016)\n",
      "\n",
      "#CTG-1673 (2016) -- CTG-0563 (2017)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0563 (2017) -- CTG-1339 (2018)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1339 (2018) -- CTG-2036 (2019)\n",
      "\n",
      "#CTG-2036 (2019) -- CTG-2682 (2020)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2682 (2020) -- CTG-1234 (2021)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1234 (2021) -- CTG-0246 (2022)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0246 (2022) -- CTG-2211 (2023)\n",
      "\n",
      "#CTG-2211 (2023) -- CTG-0457 (2024)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0457 (2024) -- CTG-0808 (2025)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0808 (2025) -- CTG-0109 (2026)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0109 (2026) -- CTG-0129 (2027)\n",
      "\n",
      "#CTG-0129 (2027) -- CTG-1634 (2028)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1634 (2028) -- CTG-1512 (2029)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1512 (2029) -- CTG-2094 (2030)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2094 (2030) -- CTG-0225 (2031)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0225 (2031) -- CTG-0547 (2032)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0547 (2032) -- CTG-0831 (2033)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0831 (2033) -- CTG-0944 (2034)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0944 (2034) -- CTG-1455 (2035)\n",
      "\n",
      "#CTG-1455 (2035) -- CTG-1362 (2036)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1362 (2036) -- CTG-2399 (2037)\n",
      "\n",
      "#CTG-2399 (2037) -- CTG-1249 (2038)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1249 (2038) -- CTG-1588 (2039)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1588 (2039) -- CTG-0980 (2040)\n",
      "\n",
      "#CTG-0980 (2040) -- CTG-2228 (2041)\n",
      "\n",
      "#CTG-2228 (2041) -- CTG-1823 (2042)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1823 (2042) -- CTG-1275 (2043)\n",
      "\n",
      "#CTG-1275 (2043) -- CTG-1884 (2044)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1884 (2044) -- CTG-2427 (2045)\n",
      "\n",
      "#CTG-2427 (2045) -- CTG-1662 (2046)\n",
      "\n",
      "#CTG-1662 (2046) -- CTG-1615 (2047)\n",
      "\n",
      "#CTG-1615 (2047) -- CTG-1607 (2048)\n",
      "\n",
      "#CTG-1607 (2048) -- CTG-1544 (2049)\n",
      "\n",
      "#CTG-1544 (2049) -- CTG-1164 (2050)\n",
      "\n",
      "#CTG-1164 (2050) -- CTG-2280 (2051)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2280 (2051) -- CTG-1979 (2052)\n",
      "\n",
      "#CTG-1979 (2052) -- CTG-1300 (2053)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1300 (2053) -- CTG-1853 (2054)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1853 (2054) -- CTG-1433 (2055)\n",
      "\n",
      "#CTG-1433 (2055) -- CTG-1136 (2056)\n",
      "\n",
      "#CTG-1136 (2056) -- CTG-2370 (2057)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2370 (2057) -- CTG-1732 (2058)\n",
      "\n",
      "#CTG-1732 (2058) -- CTG-1901 (2059)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1901 (2059) -- CTG-1336 (2060)\n",
      "\n",
      "#CTG-1336 (2060) -- CTG-2146 (2061)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2146 (2061) -- CTG-1205 (2062)\n",
      "\n",
      "#CTG-1205 (2062) -- CTG-1565 (2063)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1565 (2063) -- CTG-1286 (2064)\n",
      "\n",
      "#CTG-1286 (2064) -- CTG-1555 (2065)\n",
      "\n",
      "#CTG-1555 (2065) -- CTG-1202 (2066)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1202 (2066) -- CTG-1394 (2067)\n",
      "\n",
      "#CTG-1394 (2067) -- CTG-2039 (2068)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2039 (2068) -- CTG-2243 (2069)\n",
      "\n",
      "#CTG-2243 (2069) -- CTG-1762 (2070)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1762 (2070) -- CTG-0958 (2071)\n",
      "\n",
      "#CTG-0958 (2071) -- CTG-1283 (2072)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1283 (2072) -- CTG-1849 (2073)\n",
      "\n",
      "#CTG-1849 (2073) -- CTG-1653 (2074)\n",
      "\n",
      "#CTG-1653 (2074) -- CTG-1448 (2075)\n",
      "\n",
      "#CTG-1448 (2075) -- CTG-1215 (2076)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1215 (2076) -- CTG-1628 (2077)\n",
      "\n",
      "#CTG-1628 (2077) -- CTG-1604 (2078)\n",
      "\n",
      "#CTG-1604 (2078) -- CTG-2247 (2079)\n",
      "\n",
      "#CTG-2247 (2079) -- CTG-1149 (2080)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1149 (2080) -- CTG-1150 (2081)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1150 (2081) -- CTG-1984 (2082)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1984 (2082) -- CTG-1263 (2083)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1263 (2083) -- CTG-2021 (2084)\n",
      "\n",
      "#CTG-2021 (2084) -- CTG-2168 (2085)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2168 (2085) -- CTG-1218 (2086)\n",
      "\n",
      "#CTG-1218 (2086) -- CTG-2385 (2087)\n",
      "\n",
      "#CTG-2385 (2087) -- CTG-1167 (2088)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1167 (2088) -- CTG-2474 (2089)\n",
      "\n",
      "#CTG-2474 (2089) -- CTG-1037 (2090)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1037 (2090) -- CTG-1790 (2091)\n",
      "\n",
      "#CTG-1790 (2091) -- CTG-1693 (2092)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1693 (2092) -- CTG-1467 (2093)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1467 (2093) -- CTG-1125 (2094)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1125 (2094) -- CTG-1127 (2095)\n",
      "\n",
      "#CTG-1127 (2095) -- CTG-1133 (2096)\n",
      "\n",
      "#CTG-1133 (2096) -- CTG-1147 (2097)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1147 (2097) -- CTG-0973 (2098)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0973 (2098) -- CTG-1842 (2099)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1842 (2099) -- CTG-1421 (2100)\n",
      "\n",
      "#CTG-1421 (2100) -- CTG-2389 (2101)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2389 (2101) -- CTG-1203 (2102)\n",
      "\n",
      "#CTG-1203 (2102) -- CTG-2137 (2103)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2137 (2103) -- CTG-1688 (2104)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1688 (2104) -- CTG-1370 (2105)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1370 (2105) -- CTG-1246 (2106)\n",
      "\n",
      "#CTG-1246 (2106) -- CTG-1231 (2107)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1231 (2107) -- CTG-1116 (2108)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1116 (2108) -- CTG-0750 (2109)\n",
      "\n",
      "#CTG-0750 (2109) -- CTG-0226 (2110)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0226 (2110) -- CTG-0238 (2111)\n",
      "\n",
      "#CTG-0238 (2111) -- CTG-0719 (2112)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0719 (2112) -- CTG-0276 (2113)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0276 (2113) -- CTG-2398 (2114)\n",
      "\n",
      "#CTG-2398 (2114) -- CTG-2432 (2115)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2432 (2115) -- CTG-2560 (2116)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2560 (2116) -- CTG-2051 (2117)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2051 (2117) -- CTG-1335 (2118)\n",
      "\n",
      "#CTG-1335 (2118) -- CTG-2506 (2119)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2506 (2119) -- CTG-1254 (2120)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1254 (2120) -- CTG-0766 (2121)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0766 (2121) -- CTG-2295 (2122)\n",
      "\n",
      "#CTG-2295 (2122) -- CTG-1242 (2123)\n",
      "\n",
      "#CTG-1242 (2123) -- CTG-1541 (2124)\n",
      "\n",
      "#CTG-1541 (2124) -- CTG-1260 (2125)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1260 (2125) -- CTG-0275 (2126)\n",
      "\n",
      "#CTG-0275 (2126) -- CTG-1668 (2127)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1668 (2127) -- CTG-1616 (2128)\n",
      "\n",
      "#CTG-1616 (2128) -- CTG-1717 (2129)\n",
      "\n",
      "#CTG-1717 (2129) -- CTG-1216 (2130)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1216 (2130) -- CTG-2273 (2131)\n",
      "\n",
      "#CTG-2273 (2131) -- CTG-1654 (2132)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1654 (2132) -- CTG-1623 (2133)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1623 (2133) -- CTG-2195 (2134)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2195 (2134) -- CTG-1815 (2135)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1815 (2135) -- CTG-1391 (2136)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1391 (2136) -- CTG-1412 (2137)\n",
      "\n",
      "#CTG-1412 (2137) -- CTG-0567 (2138)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0567 (2138) -- CTG-2431 (2139)\n",
      "\n",
      "#CTG-2431 (2139) -- CTG-1916 (2140)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1916 (2140) -- CTG-1211 (2141)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1211 (2141) -- CTG-1569 (2142)\n",
      "\n",
      "#CTG-1569 (2142) -- CTG-0885 (2143)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0885 (2143) -- CTG-1747 (2144)\n",
      "\n",
      "#CTG-1747 (2144) -- CTG-0922 (2145)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0922 (2145) -- CTG-2451 (2146)\n",
      "\n",
      "#CTG-2451 (2146) -- CTG-0489 (2147)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0489 (2147) -- CTG-2343 (2148)\n",
      "\n",
      "#CTG-2343 (2148) -- CTG-2030 (2149)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2030 (2149) -- CTG-1374 (2150)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1374 (2150) -- CTG-0362 (2151)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0362 (2151) -- CTG-2462 (2152)\n",
      "\n",
      "#CTG-2462 (2152) -- CTG-1452 (2153)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1452 (2153) -- CTG-0531 (2154)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0531 (2154) -- CTG-1355 (2155)\n",
      "\n",
      "#CTG-1355 (2155) -- CTG-1997 (2156)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1997 (2156) -- CTG-1266 (2157)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1266 (2157) -- CTG-1168 (2158)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1168 (2158) -- CTG-1983 (2159)\n",
      "\n",
      "#CTG-1983 (2159) -- CTG-2077 (2160)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2077 (2160) -- CTG-1001 (2161)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1001 (2161) -- CTG-2075 (2162)\n",
      "\n",
      "#CTG-2075 (2162) -- CTG-1939 (2163)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1939 (2163) -- CTG-1232 (2164)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1232 (2164) -- CTG-1804 (2165)\n",
      "\n",
      "#CTG-1804 (2165) -- CTG-1440 (2166)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1440 (2166) -- CTG-2155 (2167)\n",
      "\n",
      "#CTG-2155 (2167) -- CTG-1824 (2168)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1824 (2168) -- CTG-1825 (2169)\n",
      "\n",
      "#CTG-1825 (2169) -- CTG-1482 (2170)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1482 (2170) -- CTG-1505 (2171)\n",
      "\n",
      "#CTG-1505 (2171) -- CTG-1507 (2172)\n",
      "\n",
      "#CTG-1507 (2172) -- CTG-1990 (2173)\n",
      "\n",
      "#CTG-1990 (2173) -- CTG-2330 (2174)\n",
      "\n",
      "#CTG-2330 (2174) -- CTG-1268 (2175)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1268 (2175) -- CTG-1516 (2176)\n",
      "\n",
      "#CTG-1516 (2176) -- CTG-1194 (2177)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1194 (2177) -- CTG-1045 (2178)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1045 (2178) -- CTG-0881 (2179)\n",
      "\n",
      "#CTG-0881 (2179) -- CTG-0762 (2180)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0762 (2180) -- CTG-1161 (2181)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1161 (2181) -- CTG-1212 (2182)\n",
      "\n",
      "#CTG-1212 (2182) -- CTG-1464 (2183)\n",
      "\n",
      "#CTG-1464 (2183) -- CTG-1322 (2184)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1322 (2184) -- CTG-1547 (2185)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1547 (2185) -- CTG-1356 (2186)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1356 (2186) -- CTG-2615 (2187)\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#CTG-2615 (2187) -- CTG-0821 (2188)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0821 (2188) -- CTG-2408 (2189)\n",
      "\n",
      "#CTG-2408 (2189) -- CTG-1068 (2190)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1068 (2190) -- CTG-0629 (2191)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0629 (2191) -- CTG-1697 (2192)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1697 (2192) -- CTG-0491 (2193)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0491 (2193) -- CTG-1667 (2194)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1667 (2194) -- CTG-1140 (2195)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1140 (2195) -- CTG-0405 (2196)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0405 (2196) -- CTG-0074 (2197)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0074 (2197) -- CTG-1783 (2198)\n",
      "\n",
      "#CTG-1783 (2198) -- CTG-1583 (2199)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1583 (2199) -- CTG-2445 (2200)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2445 (2200) -- CTG-0152 (2201)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0152 (2201) -- CTG-2152 (2202)\n",
      "\n",
      "#CTG-2152 (2202) -- CTG-0192 (2203)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0192 (2203) -- CTG-1764 (2204)\n",
      "\n",
      "#CTG-1764 (2204) -- CTG-0430 (2205)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0430 (2205) -- CTG-2472 (2206)\n",
      "\n",
      "#CTG-2472 (2206) -- CTG-1567 (2207)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1567 (2207) -- CTG-1424 (2208)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1424 (2208) -- CTG-0163 (2209)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0163 (2209) -- CTG-2559 (2210)\n",
      "\n",
      "#CTG-2559 (2210) -- CTG-2442 (2211)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2442 (2211) -- CTG-1956 (2212)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1956 (2212) -- CTG-2599 (2213)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2599 (2213) -- CTG-0685 (2214)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0685 (2214) -- CTG-0305 (2215)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0305 (2215) -- CTG-0856 (2216)\n",
      "\n",
      "#CTG-0856 (2216) -- CTG-1318 (2217)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1318 (2217) -- CTG-0322 (2218)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0322 (2218) -- CTG-0783 (2219)\n",
      "\n",
      "#CTG-0783 (2219) -- CTG-1894 (2220)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1894 (2220) -- CTG-2351 (2221)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2351 (2221) -- CTG-0700 (2222)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0700 (2222) -- CTG-0548 (2223)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0548 (2223) -- CTG-0506 (2224)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0506 (2224) -- CTG-0481 (2225)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0481 (2225) -- CTG-2458 (2226)\n",
      "\n",
      "#CTG-2458 (2226) -- CTG-0075 (2227)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0075 (2227) -- CTG-1427 (2228)\n",
      "\n",
      "#CTG-1427 (2228) -- CTG-1451 (2229)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1451 (2229) -- CTG-1472 (2230)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1472 (2230) -- CTG-2267 (2231)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2267 (2231) -- CTG-1743 (2232)\n",
      "\n",
      "#CTG-1743 (2232) -- CTG-2683 (2233)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2683 (2233) -- CTG-2199 (2234)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2199 (2234) -- CTG-2179 (2235)\n",
      "\n",
      "#CTG-2179 (2235) -- CTG-1154 (2236)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1154 (2236) -- CTG-1132 (2237)\n",
      "\n",
      "#CTG-1132 (2237) -- CTG-2062 (2238)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2062 (2238) -- CTG-1070 (2239)\n",
      "\n",
      "#CTG-1070 (2239) -- CTG-1181 (2240)\n",
      "\n",
      "#CTG-1181 (2240) -- CTG-2378 (2241)\n",
      "\n",
      "#CTG-2378 (2241) -- CTG-2178 (2242)\n",
      "\n",
      "#CTG-2178 (2242) -- CTG-1347 (2243)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1347 (2243) -- CTG-2404 (2244)\n",
      "\n",
      "#CTG-2404 (2244) -- CTG-2157 (2245)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2157 (2245) -- CTG-2134 (2246)\n",
      "\n",
      "#CTG-2134 (2246) -- CTG-1310 (2247)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1310 (2247) -- CTG-1791 (2248)\n",
      "\n",
      "#CTG-1791 (2248) -- CTG-2091 (2249)\n",
      "\n",
      "#CTG-2091 (2249) -- CTG-2191 (2250)\n",
      "\n",
      "#CTG-2191 (2250) -- CTG-2436 (2251)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2436 (2251) -- CTG-1932 (2252)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1932 (2252) -- CTG-2380 (2253)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2380 (2253) -- CTG-1814 (2254)\n",
      "\n",
      "#CTG-1814 (2254) -- CTG-2100 (2255)\n",
      "\n",
      "#CTG-2100 (2255) -- CTG-2375 (2256)\n",
      "\n",
      "#CTG-2375 (2256) -- CTG-2480 (2257)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2480 (2257) -- CTG-1652 (2258)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1652 (2258) -- CTG-1798 (2259)\n",
      "\n",
      "#CTG-1798 (2259) -- CTG-2125 (2260)\n",
      "\n",
      "#CTG-2125 (2260) -- CTG-1953 (2261)\n",
      "\n",
      "#CTG-1953 (2261) -- CTG-0651 (2262)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0651 (2262) -- CTG-1407 (2263)\n",
      "\n",
      "#CTG-1407 (2263) -- CTG-2580 (2264)\n",
      "\n",
      "#CTG-2580 (2264) -- CTG-2268 (2265)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2268 (2265) -- CTG-1972 (2266)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1972 (2266) -- CTG-2045 (2267)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2045 (2267) -- CTG-2386 (2268)\n",
      "\n",
      "#CTG-2386 (2268) -- CTG-2486 (2269)\n",
      "\n",
      "#CTG-2486 (2269) -- CTG-1896 (2270)\n",
      "\n",
      "#CTG-1896 (2270) -- CTG-2055 (2271)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2055 (2271) -- CTG-2115 (2272)\n",
      "\n",
      "#CTG-2115 (2272) -- CTG-2338 (2273)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2338 (2273) -- CTG-2575 (2274)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2575 (2274) -- CTG-2349 (2275)\n",
      "\n",
      "#CTG-2349 (2275) -- CTG-2526 (2276)\n",
      "\n",
      "#CTG-2526 (2276) -- CTG-2601 (2277)\n",
      "\n",
      "#CTG-2601 (2277) -- CTG-2570 (2278)\n",
      "\n",
      "#CTG-2570 (2278) -- CTG-2271 (2279)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2271 (2279) -- CTG-2425 (2280)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2425 (2280) -- CTG-2346 (2281)\n",
      "\n",
      "#CTG-2346 (2281) -- CTG-2118 (2282)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2118 (2282) -- CTG-2457 (2283)\n",
      "\n",
      "#CTG-2457 (2283) -- CTG-2328 (2284)\n",
      "\n",
      "#CTG-2328 (2284) -- CTG-2262 (2285)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2262 (2285) -- CTG-2429 (2286)\n",
      "\n",
      "#CTG-2429 (2286) -- CTG-2321 (2287)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2321 (2287) -- CTG-2516 (2288)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2516 (2288) -- CTG-2673 (2289)\n",
      "\n",
      "#CTG-2673 (2289) -- CTG-2181 (2290)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2181 (2290) -- CTG-2251 (2291)\n",
      "\n",
      "#CTG-2251 (2291) -- CTG-2588 (2292)\n",
      "\n",
      "#CTG-2588 (2292) -- CTG-1631 (2293)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1631 (2293) -- CTG-2151 (2294)\n",
      "\n",
      "#CTG-2151 (2294) -- CTG-2468 (2295)\n",
      "\n",
      "#CTG-2468 (2295) -- CTG-2236 (2296)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2236 (2296) -- CTG-2160 (2297)\n",
      "\n",
      "#CTG-2160 (2297) -- CTG-1915 (2298)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1915 (2298) -- CTG-2259 (2299)\n",
      "\n",
      "#CTG-2259 (2299) -- CTG-2231 (2300)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2231 (2300) -- CTG-2594 (2301)\n",
      "\n",
      "#CTG-2594 (2301) -- CTG-2337 (2302)\n",
      "\n",
      "#CTG-2337 (2302) -- CTG-0987 (2303)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0987 (2303) -- CTG-1501 (2304)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1501 (2304) -- CTG-1533 (2305)\n",
      "\n",
      "#CTG-1533 (2305) -- CTG-1837 (2306)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1837 (2306) -- CTG-2483 (2307)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2483 (2307) -- CTG-2013 (2308)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2013 (2308) -- CTG-2616 (2309)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2616 (2309) -- CTG-2586 (2310)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2586 (2310) -- CTG-1754 (2311)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1754 (2311) -- CTG-1480 (2312)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1480 (2312) -- CTG-2358 (2313)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2358 (2313) -- CTG-2192 (2314)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2192 (2314) -- CTG-1937 (2315)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1937 (2315) -- CTG-2556 (2316)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2556 (2316) -- CTG-1655 (2317)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1655 (2317) -- CTG-2678 (2318)\n",
      "\n",
      "#CTG-2678 (2318) -- CTG-2538 (2319)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2538 (2319) -- CTG-2301 (2320)\n",
      "\n",
      "#CTG-2301 (2320) -- CTG-1960 (2321)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1960 (2321) -- CTG-2679 (2322)\n",
      "\n",
      "#CTG-2679 (2322) -- CTG-2130 (2323)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2130 (2323) -- CTG-2310 (2324)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2310 (2324) -- CTG-1992 (2325)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1992 (2325) -- CTG-2026 (2326)\n",
      "\n",
      "#CTG-2026 (2326) -- CTG-2478 (2327)\n",
      "\n",
      "#CTG-2478 (2327) -- CTG-2388 (2328)\n",
      "\n",
      "#CTG-2388 (2328) -- CTG-1845 (2329)\n",
      "\n",
      "#CTG-1845 (2329) -- CTG-1171 (2330)\n",
      "\n",
      "#CTG-1171 (2330) -- CTG-1244 (2331)\n",
      "\n",
      "#CTG-1244 (2331) -- CTG-1225 (2332)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1225 (2332) -- CTG-1227 (2333)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1227 (2333) -- CTG-1843 (2334)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1843 (2334) -- CTG-2405 (2335)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2405 (2335) -- CTG-2587 (2336)\n",
      "\n",
      "#CTG-2587 (2336) -- CTG-2234 (2337)\n",
      "\n",
      "#CTG-2234 (2337) -- CTG-2350 (2338)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2350 (2338) -- CTG-1705 (2339)\n",
      "\n",
      "#CTG-1705 (2339) -- CTG-2066 (2340)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2066 (2340) -- CTG-1961 (2341)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1961 (2341) -- CTG-2396 (2342)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2396 (2342) -- CTG-2289 (2343)\n",
      "\n",
      "#CTG-2289 (2343) -- CTG-2065 (2344)\n",
      "\n",
      "#CTG-2065 (2344) -- CTG-2347 (2345)\n",
      "\n",
      "#CTG-2347 (2345) -- CTG-1810 (2346)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1810 (2346) -- CTG-2441 (2347)\n",
      "\n",
      "#CTG-2441 (2347) -- CTG-2312 (2348)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2312 (2348) -- CTG-2448 (2349)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2448 (2349) -- CTG-2264 (2350)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2264 (2350) -- CTG-2140 (2351)\n",
      "\n",
      "#CTG-2140 (2351) -- CTG-2620 (2352)\n",
      "\n",
      "#CTG-2620 (2352) -- CTG-1597 (2353)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1597 (2353) -- CTG-2635 (2354)\n",
      "\n",
      "#CTG-2635 (2354) -- CTG-2080 (2355)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2080 (2355) -- CTG-1695 (2356)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1695 (2356) -- CTG-1005 (2357)\n",
      "\n",
      "#CTG-1005 (2357) -- CTG-1398 (2358)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1398 (2358) -- CTG-1530 (2359)\n",
      "\n",
      "#CTG-1530 (2359) -- CTG-1911 (2360)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1911 (2360) -- CTG-2217 (2361)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2217 (2361) -- CTG-2327 (2362)\n",
      "\n",
      "#CTG-2327 (2362) -- CTG-1658 (2363)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1658 (2363) -- CTG-2331 (2364)\n",
      "\n",
      "#CTG-2331 (2364) -- CTG-2414 (2365)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2414 (2365) -- CTG-2112 (2366)\n",
      "\n",
      "#CTG-2112 (2366) -- CTG-1807 (2367)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1807 (2367) -- CTG-1195 (2368)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1195 (2368) -- CTG-2618 (2369)\n",
      "\n",
      "#CTG-2618 (2369) -- CTG-2582 (2370)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2582 (2370) -- CTG-2524 (2371)\n",
      "\n",
      "#CTG-2524 (2371) -- CTG-1706 (2372)\n",
      "\n",
      "#CTG-1706 (2372) -- CTG-1919 (2373)\n",
      "\n",
      "#CTG-1919 (2373) -- CTG-1574 (2374)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1574 (2374) -- CTG-1558 (2375)\n",
      "\n",
      "#CTG-1558 (2375) -- CTG-2052 (2376)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2052 (2376) -- CTG-1401 (2377)\n",
      "\n",
      "#CTG-1401 (2377) -- CTG-2504 (2378)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2504 (2378) -- CTG-1819 (2379)\n",
      "\n",
      "#CTG-1819 (2379) -- CTG-2450 (2380)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2450 (2380) -- CTG-2022 (2381)\n",
      "\n",
      "#CTG-2022 (2381) -- CTG-2300 (2382)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2300 (2382) -- CTG-1099 (2383)\n",
      "\n",
      "#CTG-1099 (2383) -- CTG-2428 (2384)\n",
      "\n",
      "#CTG-2428 (2384) -- CTG-1890 (2385)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1890 (2385) -- CTG-2614 (2386)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2614 (2386) -- CTG-1959 (2387)\n",
      "\n",
      "#CTG-1959 (2387) -- CTG-2377 (2388)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2377 (2388) -- CTG-2248 (2389)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2248 (2389) -- CTG-2421 (2390)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2421 (2390) -- CTG-2681 (2391)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2681 (2391) -- CTG-2119 (2392)\n",
      "\n",
      "#CTG-2119 (2392) -- CTG-2073 (2393)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2073 (2393) -- CTG-2103 (2394)\n",
      "\n",
      "#CTG-2103 (2394) -- CTG-1898 (2395)\n",
      "\n",
      "#CTG-1898 (2395) -- CTG-2674 (2396)\n",
      "\n",
      "#CTG-2674 (2396) -- CTG-2530 (2397)\n",
      "\n",
      "#CTG-2530 (2397) -- CTG-1809 (2398)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1809 (2398) -- CTG-2675 (2399)\n",
      "\n",
      "#CTG-2675 (2399) -- CTG-1977 (2400)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1977 (2400) -- CTG-1540 (2401)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1540 (2401) -- CTG-1920 (2402)\n",
      "\n",
      "#CTG-1920 (2402) -- CTG-2558 (2403)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2558 (2403) -- CTG-1416 (2404)\n",
      "\n",
      "#CTG-1416 (2404) -- CTG-2352 (2405)\n",
      "\n",
      "#CTG-2352 (2405) -- CTG-2275 (2406)\n",
      "\n",
      "#CTG-2275 (2406) -- CTG-2422 (2407)\n",
      "\n",
      "#CTG-2422 (2407) -- CTG-2218 (2408)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2218 (2408) -- CTG-1968 (2409)\n",
      "\n",
      "#CTG-1968 (2409) -- CTG-2621 (2410)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2621 (2410) -- CTG-0286 (2411)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0286 (2411) -- CTG-0486 (2412)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0486 (2412) -- CTG-0488 (2413)\n",
      "\n",
      "#CTG-0488 (2413) -- CTG-0455 (2414)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0455 (2414) -- CTG-0397 (2415)\n",
      "\n",
      "#CTG-0397 (2415) -- CTG-0269 (2416)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0269 (2416) -- CTG-1985 (2417)\n",
      "\n",
      "#CTG-1985 (2417) -- CTG-0998 (2418)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0998 (2418) -- CTG-2096 (2419)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2096 (2419) -- CTG-0070 (2420)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0070 (2420) -- CTG-2623 (2421)\n",
      "\n",
      "#CTG-2623 (2421) -- CTG-1434 (2422)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1434 (2422) -- CTG-0449 (2423)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0449 (2423) -- CTG-0252 (2424)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0252 (2424) -- CTG-1848 (2425)\n",
      "\n",
      "#CTG-1848 (2425) -- CTG-1579 (2426)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1579 (2426) -- CTG-0165 (2427)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0165 (2427) -- CTG-0643 (2428)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0643 (2428) -- CTG-0607 (2429)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0607 (2429) -- CTG-0291 (2430)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0291 (2430) -- CTG-0868 (2431)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0868 (2431) -- CTG-0021 (2432)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0021 (2432) -- CTG-2305 (2433)\n",
      "\n",
      "#CTG-2305 (2433) -- CTG-1488 (2434)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1488 (2434) -- CTG-1103 (2435)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1103 (2435) -- CTG-0972 (2436)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0972 (2436) -- CTG-0985 (2437)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0985 (2437) -- CTG-1193 (2438)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1193 (2438) -- CTG-2609 (2439)\n",
      "\n",
      "#CTG-2609 (2439) -- CTG-1163 (2440)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1163 (2440) -- CTG-1457 (2441)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1457 (2441) -- CTG-1083 (2442)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1083 (2442) -- CTG-1955 (2443)\n",
      "\n",
      "#CTG-1955 (2443) -- CTG-1042 (2444)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1042 (2444) -- CTG-1169 (2445)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1169 (2445) -- CTG-1291 (2446)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1291 (2446) -- CTG-0949 (2447)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0949 (2447) -- CTG-1180 (2448)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1180 (2448) -- CTG-1095 (2449)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1095 (2449) -- CTG-0476 (2450)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0476 (2450) -- CTG-2434 (2451)\n",
      "\n",
      "#CTG-2434 (2451) -- CTG-2213 (2452)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2213 (2452) -- CTG-1228 (2453)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1228 (2453) -- CTG-1026 (2454)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1026 (2454) -- CTG-1521 (2455)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1521 (2455) -- CTG-1206 (2456)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1206 (2456) -- CTG-1069 (2457)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1069 (2457) -- CTG-1947 (2458)\n",
      "\n",
      "#CTG-1947 (2458) -- CTG-1131 (2459)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1131 (2459) -- CTG-0805 (2460)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0805 (2460) -- CTG-1067 (2461)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1067 (2461) -- CTG-1020 (2462)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1020 (2462) -- CTG-1187 (2463)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1187 (2463) -- CTG-1160 (2464)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1160 (2464) -- CTG-1077 (2465)\n",
      "\n",
      "#CTG-1077 (2465) -- CTG-1053 (2466)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1053 (2466) -- CTG-1038 (2467)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1038 (2467) -- CTG-0511 (2468)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0511 (2468) -- CTG-1106 (2469)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1106 (2469) -- CTG-0919 (2470)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0919 (2470) -- CTG-0875 (2471)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0875 (2471) -- CTG-0884 (2472)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0884 (2472) -- CTG-0186 (2473)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0186 (2473) -- CTG-0718 (2474)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0718 (2474) -- CTG-0970 (2475)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0970 (2475) -- CTG-1198 (2476)\n",
      "\n",
      "#CTG-1198 (2476) -- CTG-0576 (2477)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0576 (2477) -- CTG-1867 (2478)\n",
      "\n",
      "#CTG-1867 (2478) -- CTG-1415 (2479)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1415 (2479) -- CTG-0963 (2480)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0963 (2480) -- CTG-0847 (2481)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0847 (2481) -- CTG-1034 (2482)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1034 (2482) -- CTG-0853 (2483)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0853 (2483) -- CTG-0669 (2484)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0669 (2484) -- CTG-1445 (2485)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1445 (2485) -- CTG-1146 (2486)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1146 (2486) -- CTG-0956 (2487)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0956 (2487) -- CTG-0943 (2488)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0943 (2488) -- CTG-2003 (2489)\n",
      "\n",
      "#CTG-2003 (2489) -- CTG-0541 (2490)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0541 (2490) -- CTG-2239 (2491)\n",
      "\n",
      "#CTG-2239 (2491) -- CTG-0941 (2492)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0941 (2492) -- CTG-0870 (2493)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0870 (2493) -- CTG-1151 (2494)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1151 (2494) -- CTG-2335 (2495)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2335 (2495) -- CTG-0934 (2496)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0934 (2496) -- CTG-0267 (2497)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0267 (2497) -- CTG-0374 (2498)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0374 (2498) -- CTG-0456 (2499)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0456 (2499) -- CTG-0812 (2500)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0812 (2500) -- CTG-0979 (2501)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0979 (2501) -- CTG-1866 (2502)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1866 (2502) -- CTG-1076 (2503)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1076 (2503) -- CTG-0935 (2504)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0935 (2504) -- CTG-2182 (2505)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2182 (2505) -- CTG-2186 (2506)\n",
      "\n",
      "#CTG-2186 (2506) -- CTG-1137 (2507)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1137 (2507) -- CTG-0580 (2508)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0580 (2508) -- CTG-0443 (2509)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0443 (2509) -- CTG-0857 (2510)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0857 (2510) -- CTG-0842 (2511)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0842 (2511) -- CTG-0715 (2512)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0715 (2512) -- CTG-1201 (2513)\n",
      "\n",
      "#CTG-1201 (2513) -- CTG-0595 (2514)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0595 (2514) -- CTG-0568 (2515)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0568 (2515) -- CTG-1301 (2516)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1301 (2516) -- CTG-2622 (2517)\n",
      "\n",
      "#CTG-2622 (2517) -- CTG-0722 (2518)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0722 (2518) -- CTG-0447 (2519)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0447 (2519) -- CTG-0096 (2520)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0096 (2520) -- CTG-1123 (2521)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1123 (2521) -- CTG-0237 (2522)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0237 (2522) -- CTG-1570 (2523)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1570 (2523) -- CTG-2038 (2524)\n",
      "\n",
      "#CTG-2038 (2524) -- CTG-1255 (2525)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1255 (2525) -- CTG-1951 (2526)\n",
      "\n",
      "#CTG-1951 (2526) -- CTG-0471 (2527)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0471 (2527) -- CTG-1918 (2528)\n",
      "\n",
      "#CTG-1918 (2528) -- CTG-0194 (2529)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0194 (2529) -- CTG-1643 (2530)\n",
      "\n",
      "#CTG-1643 (2530) -- CTG-0892 (2531)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0892 (2531) -- CTG-1686 (2532)\n",
      "\n",
      "#CTG-1686 (2532) -- CTG-1080 (2533)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1080 (2533) -- CTG-0991 (2534)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0991 (2534) -- CTG-0873 (2535)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0873 (2535) -- CTG-1097 (2536)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1097 (2536) -- CTG-0525 (2537)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0525 (2537) -- CTG-0516 (2538)\n",
      "\n",
      "#CTG-0516 (2538) -- CTG-0515 (2539)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0515 (2539) -- CTG-0244 (2540)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0244 (2540) -- CTG-1008 (2541)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1008 (2541) -- CTG-0921 (2542)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0921 (2542) -- CTG-1428 (2543)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1428 (2543) -- CTG-1107 (2544)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1107 (2544) -- CTG-0279 (2545)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0279 (2545) -- CTG-0254 (2546)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0254 (2546) -- CTG-0557 (2547)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0557 (2547) -- CTG-1971 (2548)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1971 (2548) -- CTG-0865 (2549)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0865 (2549) -- CTG-0278 (2550)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0278 (2550) -- CTG-2371 (2551)\n",
      "\n",
      "#CTG-2371 (2551) -- CTG-1165 (2552)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1165 (2552) -- CTG-1102 (2553)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1102 (2553) -- CTG-1152 (2554)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1152 (2554) -- CTG-1486 (2555)\n",
      "\n",
      "#CTG-1486 (2555) -- CTG-1057 (2556)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1057 (2556) -- CTG-0336 (2557)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0336 (2557) -- CTG-1022 (2558)\n",
      "\n",
      "#CTG-1022 (2558) -- CTG-1345 (2559)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1345 (2559) -- CTG-1385 (2560)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1385 (2560) -- CTG-1691 (2561)\n",
      "\n",
      "#CTG-1691 (2561) -- CTG-2019 (2562)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-2019 (2562) -- CTG-1219 (2563)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1219 (2563) -- CTG-0496 (2564)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0496 (2564) -- CTG-2426 (2565)\n",
      "\n",
      "#CTG-2426 (2565) -- CTG-1257 (2566)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-1257 (2566) -- CTG-1835 (2567)\n",
      "\n",
      "#CTG-1835 (2567) -- CTG-0601 (2568)\n",
      "PASS - Last contig on chromosome overlap or is spaced correctly\n",
      "\n",
      "#CTG-0601 (2568) -- CTG-0697 (2569)\n",
      "\n",
      "#CTG-0697 (2569) -- CTG-0792 (2570)\n",
      "\n",
      "#CTG-0792 (2570) -- CTG-1315 (2571)\n",
      "\n",
      "#CTG-1315 (2571) -- CTG-1636 (2572)\n",
      "\n",
      "#CTG-1636 (2572) -- CTG-1657 (2573)\n",
      "\n",
      "#CTG-1657 (2573) -- CTG-1882 (2574)\n",
      "\n",
      "#CTG-1882 (2574) -- CTG-1949 (2575)\n",
      "\n",
      "#CTG-1949 (2575) -- CTG-1966 (2576)\n",
      "\n",
      "#CTG-1966 (2576) -- CTG-2053 (2577)\n",
      "\n",
      "#CTG-2053 (2577) -- CTG-2074 (2578)\n",
      "\n",
      "#CTG-2074 (2578) -- CTG-2110 (2579)\n",
      "\n",
      "#CTG-2110 (2579) -- CTG-2120 (2580)\n",
      "\n",
      "#CTG-2120 (2580) -- CTG-2142 (2581)\n",
      "\n",
      "#CTG-2142 (2581) -- CTG-2159 (2582)\n",
      "\n",
      "#CTG-2159 (2582) -- CTG-2164 (2583)\n",
      "\n",
      "#CTG-2164 (2583) -- CTG-2173 (2584)\n",
      "\n",
      "#CTG-2173 (2584) -- CTG-2185 (2585)\n",
      "\n",
      "#CTG-2185 (2585) -- CTG-2190 (2586)\n",
      "\n",
      "#CTG-2190 (2586) -- CTG-2230 (2587)\n",
      "\n",
      "#CTG-2230 (2587) -- CTG-2252 (2588)\n",
      "\n",
      "#CTG-2252 (2588) -- CTG-2283 (2589)\n",
      "\n",
      "#CTG-2283 (2589) -- CTG-2286 (2590)\n",
      "\n",
      "#CTG-2286 (2590) -- CTG-2342 (2591)\n",
      "\n",
      "#CTG-2342 (2591) -- CTG-2353 (2592)\n",
      "\n",
      "#CTG-2353 (2592) -- CTG-2359 (2593)\n",
      "\n",
      "#CTG-2359 (2593) -- CTG-2366 (2594)\n",
      "\n",
      "#CTG-2366 (2594) -- CTG-2374 (2595)\n",
      "\n",
      "#CTG-2374 (2595) -- CTG-2381 (2596)\n",
      "\n",
      "#CTG-2381 (2596) -- CTG-2406 (2597)\n",
      "\n",
      "#CTG-2406 (2597) -- CTG-2433 (2598)\n",
      "\n",
      "#CTG-2433 (2598) -- CTG-2447 (2599)\n",
      "\n",
      "#CTG-2447 (2599) -- CTG-2449 (2600)\n",
      "\n",
      "#CTG-2449 (2600) -- CTG-2456 (2601)\n",
      "\n",
      "#CTG-2456 (2601) -- CTG-2460 (2602)\n",
      "\n",
      "#CTG-2460 (2602) -- CTG-2461 (2603)\n",
      "\n",
      "#CTG-2461 (2603) -- CTG-2467 (2604)\n",
      "\n",
      "#CTG-2467 (2604) -- CTG-2469 (2605)\n",
      "\n",
      "#CTG-2469 (2605) -- CTG-2470 (2606)\n",
      "\n",
      "#CTG-2470 (2606) -- CTG-2473 (2607)\n",
      "\n",
      "#CTG-2473 (2607) -- CTG-2476 (2608)\n",
      "\n",
      "#CTG-2476 (2608) -- CTG-2481 (2609)\n",
      "\n",
      "#CTG-2481 (2609) -- CTG-2494 (2610)\n",
      "\n",
      "#CTG-2494 (2610) -- CTG-2495 (2611)\n",
      "\n",
      "#CTG-2495 (2611) -- CTG-2496 (2612)\n",
      "\n",
      "#CTG-2496 (2612) -- CTG-2497 (2613)\n",
      "\n",
      "#CTG-2497 (2613) -- CTG-2499 (2614)\n",
      "\n",
      "#CTG-2499 (2614) -- CTG-2500 (2615)\n",
      "\n",
      "#CTG-2500 (2615) -- CTG-2503 (2616)\n",
      "\n",
      "#CTG-2503 (2616) -- CTG-2511 (2617)\n",
      "\n",
      "#CTG-2511 (2617) -- CTG-2529 (2618)\n",
      "\n",
      "#CTG-2529 (2618) -- CTG-2531 (2619)\n",
      "\n",
      "#CTG-2531 (2619) -- CTG-2532 (2620)\n",
      "\n",
      "#CTG-2532 (2620) -- CTG-2535 (2621)\n",
      "\n",
      "#CTG-2535 (2621) -- CTG-2540 (2622)\n",
      "\n",
      "#CTG-2540 (2622) -- CTG-2544 (2623)\n",
      "\n",
      "#CTG-2544 (2623) -- CTG-2547 (2624)\n",
      "\n",
      "#CTG-2547 (2624) -- CTG-2550 (2625)\n",
      "\n",
      "#CTG-2550 (2625) -- CTG-2551 (2626)\n",
      "\n",
      "#CTG-2551 (2626) -- CTG-2554 (2627)\n",
      "\n",
      "#CTG-2554 (2627) -- CTG-2563 (2628)\n",
      "\n",
      "#CTG-2563 (2628) -- CTG-2566 (2629)\n",
      "\n",
      "#CTG-2566 (2629) -- CTG-2571 (2630)\n",
      "\n",
      "#CTG-2571 (2630) -- CTG-2572 (2631)\n",
      "\n",
      "#CTG-2572 (2631) -- CTG-2576 (2632)\n",
      "\n",
      "#CTG-2576 (2632) -- CTG-2578 (2633)\n",
      "\n",
      "#CTG-2578 (2633) -- CTG-2579 (2634)\n",
      "\n",
      "#CTG-2579 (2634) -- CTG-2583 (2635)\n",
      "\n",
      "#CTG-2583 (2635) -- CTG-2584 (2636)\n",
      "\n",
      "#CTG-2584 (2636) -- CTG-2591 (2637)\n",
      "\n",
      "#CTG-2591 (2637) -- CTG-2593 (2638)\n",
      "\n",
      "#CTG-2593 (2638) -- CTG-2595 (2639)\n",
      "\n",
      "#CTG-2595 (2639) -- CTG-2603 (2640)\n",
      "\n",
      "#CTG-2603 (2640) -- CTG-2605 (2641)\n",
      "\n",
      "#CTG-2605 (2641) -- CTG-2610 (2642)\n",
      "\n",
      "#CTG-2610 (2642) -- CTG-2611 (2643)\n",
      "\n",
      "#CTG-2611 (2643) -- CTG-2617 (2644)\n",
      "\n",
      "#CTG-2617 (2644) -- CTG-2624 (2645)\n",
      "\n",
      "#CTG-2624 (2645) -- CTG-2628 (2646)\n",
      "\n",
      "#CTG-2628 (2646) -- CTG-2629 (2647)\n",
      "\n",
      "#CTG-2629 (2647) -- CTG-2630 (2648)\n",
      "\n",
      "#CTG-2630 (2648) -- CTG-2631 (2649)\n",
      "\n",
      "#CTG-2631 (2649) -- CTG-2632 (2650)\n",
      "\n",
      "#CTG-2632 (2650) -- CTG-2634 (2651)\n",
      "\n",
      "#CTG-2634 (2651) -- CTG-2636 (2652)\n",
      "\n",
      "#CTG-2636 (2652) -- CTG-2637 (2653)\n",
      "\n",
      "#CTG-2637 (2653) -- CTG-2640 (2654)\n",
      "\n",
      "#CTG-2640 (2654) -- CTG-2641 (2655)\n",
      "\n",
      "#CTG-2641 (2655) -- CTG-2642 (2656)\n",
      "\n",
      "#CTG-2642 (2656) -- CTG-2643 (2657)\n",
      "\n",
      "#CTG-2643 (2657) -- CTG-2644 (2658)\n",
      "\n",
      "#CTG-2644 (2658) -- CTG-2645 (2659)\n",
      "\n",
      "#CTG-2645 (2659) -- CTG-2646 (2660)\n",
      "\n",
      "#CTG-2646 (2660) -- CTG-2647 (2661)\n",
      "\n",
      "#CTG-2647 (2661) -- CTG-2648 (2662)\n",
      "\n",
      "#CTG-2648 (2662) -- CTG-2649 (2663)\n",
      "\n",
      "#CTG-2649 (2663) -- CTG-2650 (2664)\n",
      "\n",
      "#CTG-2650 (2664) -- CTG-2651 (2665)\n",
      "\n",
      "#CTG-2651 (2665) -- CTG-2652 (2666)\n",
      "\n",
      "#CTG-2652 (2666) -- CTG-2653 (2667)\n",
      "\n",
      "#CTG-2653 (2667) -- CTG-2654 (2668)\n",
      "\n",
      "#CTG-2654 (2668) -- CTG-2655 (2669)\n",
      "\n",
      "#CTG-2655 (2669) -- CTG-2656 (2670)\n",
      "\n",
      "#CTG-2656 (2670) -- CTG-2657 (2671)\n",
      "\n",
      "#CTG-2657 (2671) -- CTG-2658 (2672)\n",
      "\n",
      "#CTG-2658 (2672) -- CTG-2659 (2673)\n",
      "\n",
      "#CTG-2659 (2673) -- CTG-2660 (2674)\n",
      "\n",
      "#CTG-2660 (2674) -- CTG-2661 (2675)\n",
      "\n",
      "#CTG-2661 (2675) -- CTG-2662 (2676)\n",
      "\n",
      "#CTG-2662 (2676) -- CTG-2663 (2677)\n",
      "\n",
      "#CTG-2663 (2677) -- CTG-2664 (2678)\n",
      "\n",
      "#CTG-2664 (2678) -- CTG-2665 (2679)\n",
      "\n",
      "#CTG-2665 (2679) -- CTG-2666 (2680)\n",
      "\n",
      "#CTG-2666 (2680) -- CTG-2667 (2681)\n",
      "\n",
      "#CTG-2667 (2681) -- CTG-2668 (2682)\n",
      "\n",
      "#CTG-2668 (2682) -- CTG-2669 (2683)\n",
      "\n",
      "#CTG-2669 (2683) -- CTG-2670 (2684)\n",
      "\n",
      "#CTG-2670 (2684) -- CTG-2671 (2685)\n",
      "\n",
      "#CTG-2671 (2685) -- CTG-2672 (2686)\n",
      "\n",
      "#CTG-2672 (2686) -- CTG-2680 (2687)\n",
      "Identified CURATED coordinates for 1729 primary contigs\n"
     ]
    }
   ],
   "source": [
    "curated_contigDict, curr_index, inList = {}, 1, []\n",
    "chrom1,chrom2 = '0', '0' #initializing values\n",
    "\n",
    "\n",
    "for key, value in contigDict.items(): \n",
    "    i = int(key)\n",
    "    #1. If last contig in dataset \n",
    "    if i == len(contigDict):\n",
    "        continue\n",
    "    \n",
    "    #2. If contigs are on different chromosomes:\n",
    "    if chrom1 != chrom2:\n",
    "        process_last_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(contigDict,curr_index,i)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,curr_index,contigID2,i+1))\n",
    "    logFile.write('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,curr_index,contigID2,i+1))\n",
    "\n",
    "    #3. Automatically saves first contig in dataset\n",
    "    if curr_index == 1 and len(curated_contigDict) == 0:\n",
    "        curr_index = process_first_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "    \n",
    "    #4. If contigs have the same start site, choose the longest contig\n",
    "    if start1 == start2:\n",
    "        process_same_starts(contigDict,curr_index,i)\n",
    "        continue\n",
    "\n",
    "    #5. If contigs have same end coordinate, choose longest contig (i.e. contig #1 if sorted)\n",
    "    if end1 == end2: \n",
    "        process_same_ends(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    #6. If contigs #1 and #2 have same start AND end, keep contig #1\n",
    "    if start1 == start2 and end1 == end2:\n",
    "        print('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        continue #because contig1 is already in the dictionary\n",
    "        \n",
    "    #7. If contig #2 is fully within contig #1 -- continue\n",
    "    if start1 < start2 and end1 > end2: # contig2 is fully within contig1\n",
    "        #curated_contigDict[curr_index] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "        print('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) +'\\n')\n",
    "        continue\n",
    "    if start1 > start2 and end1 < end2:\n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)\n",
    "    \n",
    "    #8. If contig passes all these - then automatically saves\n",
    "    curr_index=i+1\n",
    "    #if contigID2 not in inList:\n",
    "    curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    print('PASS - contigs overlap or are spaced correctly')\n",
    "    logFile.write('PASS - contigs overlap or are spaced correctly\\n')\n",
    "\n",
    "print ('Identified CURATED coordinates for %i primary contigs' % len(curated_contigDict))\n",
    "logFile.write('\\n##Identified CURATED coordinates for %i primary contigs\\n\\n########\\n\\n' % len(curated_contigDict))\n",
    "\n",
    "\n",
    "#Re-naming dictionary keys\n",
    "count=0\n",
    "for keys in curated_contigDict:\n",
    "    count+=1\n",
    "    curated_contigDict[count]=curated_contigDict.pop(keys)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"print('Non-curated:')\\nfor i in contigDict:\\n    print(i,contigDict[i])\\nprint( '\\nCurated:')\\nfor i in curated_contigDict:\\n    print(i, curated_contigDict[i])\""
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"print('Non-curated:')\n",
    "for i in contigDict:\n",
    "    print(i,contigDict[i])\n",
    "print( '\\nCurated:')\n",
    "for i in curated_contigDict:\n",
    "    print(i, curated_contigDict[i])\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1):\n",
    "    chrom2 = chrom1\n",
    "    start2 = start1 - 10000\n",
    "    end2 = start1 + 10000\n",
    "    contigID2 = 'canFam3'\n",
    "    Dir2 = 'fwd'\n",
    "    length1 = end2-start2\n",
    "    \n",
    "    if start2 < 0:\n",
    "        start2 = 0\n",
    "    \n",
    "    #Determine overlap \n",
    "    determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    " \n",
    "    #Find coordinates to extract\n",
    "    find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    \n",
    "    #Extract FASTA\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/genomes/canFam3.1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "###################################################################################################\n",
    "def determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    #How much do the contigs overlap?\n",
    "    global overlap\n",
    "    overlap = end1 - start2\n",
    "    print('\\nOverlap = ', overlap)\n",
    "    #OVERLAPPING CONTIGS\n",
    "    if overlap > 0:\n",
    "        find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    #DIRECTLY ADJACENT CONTIGS\n",
    "    if overlap == 0:\n",
    "        print('Contigs are directly adjacent to one another')\n",
    "        logFile.write('Contigs are directly adjacent to one another\\n')\n",
    "        #WRITE FUNCTION FOR THESE\n",
    "    #CONTIGS WITH GAP BETWEEN THEM\n",
    "    if overlap < 0:\n",
    "        print('Gap between contigs')\n",
    "        logFile.write('Gap between contigs\\n')\n",
    "\n",
    "###################################################################################################\n",
    "def find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    ###Determine coordinates to extract for BLAT\n",
    "    global extract_coord1\n",
    "    global extract_coord2\n",
    "    #Contig #1\n",
    "    if 'fwd' in Dir1:\n",
    "        extract_coord1 = [length1 - overlap - 10000, length1]\n",
    "    else:\n",
    "        extract_coord1 = [0, overlap + 10000]\n",
    "    #Contig #2\n",
    "    if 'fwd' in Dir2:\n",
    "        extract_coord2 = [0, overlap + 10000] \n",
    "    else:\n",
    "        extract_coord2 = [length2 - overlap - 10000, length2]\n",
    "    \n",
    "    print ('Coordinates to extract for BLAT: ',extract_coord1,extract_coord2,'\\n')\n",
    "    logFile.write('Coordinates to extract for BLAT: ' + str(extract_coord1) + str(extract_coord2) + '\\n')\n",
    "\n",
    "    #safety check\n",
    "    for i in range(0,1): #CHECKS \n",
    "        #= if the region to extract extends beyond the length of the contig, if so then the coordinate changes to the length of contig\n",
    "        if extract_coord1[i] > length1:\n",
    "            extract_coord1[i] = length1\n",
    "        if extract_coord2[i] > length2:\n",
    "            extract_coord2[i] = length2\n",
    "        # If the region extended too far, and the value is negative -- then make starting extraction coordinate = 0\n",
    "        if extract_coord1[i] < 0: \n",
    "            extract_coord1[i] = 0\n",
    "        if extract_coord2[i] < 0:\n",
    "            extract_coord2[i] = 0\n",
    "            \n",
    "    if overlap < 0: #### SEND TO DIFFERENT FUNCTION LATER -- FOLLOW OVERLAPPING CONTIGS FOR NOW\n",
    "        print('Contigs do not overlap')\n",
    "        logFile.write('Contigs do not overlap\\n')\n",
    "    #Where to find fasta of contigs\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1, contigID2,Dir2,extract_coord2)\n",
    "    \n",
    "###################################################################################################\n",
    "def extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID1 + '/' + contigID1 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID1,extract_coord1[0],extract_coord1[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    #Contig 2\n",
    "    fasta_path2 = fastaRoot + contigID2 + '/' + contigID2 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %stemp/contig2.fa' % (fasta_path2,contigID2,extract_coord2[0],extract_coord2[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "###################################################################################################\n",
    "def run_blat(wkDir):\n",
    "    blatcmd = 'blat %stemp/contig1.fa %stemp/contig2.fa %stemp/temp.blat' % (wkDir,wkDir,wkDir)\n",
    "    logFile.write(blatcmd + '\\n')\n",
    "    subprocess.call(blatcmd,shell=True)\n",
    "    \n",
    "    \n",
    "###################################################################################################\n",
    "def parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    inFile = open('%stemp/temp.blat' % (wkDir),'r')\n",
    "    lineCount, cleanAlignment, offset = 0, False, 0\n",
    "    for line in inFile:\n",
    "        line=line.rstrip().split('\\t')\n",
    "        lineCount+=1\n",
    "        if lineCount > 5:#skips the BLAT results headers\n",
    "            score,strand = int(line[0]),line[8]\n",
    "            percent_of_overlap = float(score)/overlap\n",
    "            if percent_of_overlap < float(0.75):\n",
    "                continue\n",
    "            print('\\n#Parsing BLAT results','Percent of overlap matched in BLAT: ',format(percent_of_overlap, '.3f'))\n",
    "            logFile.write('\\n#Parsing BLAT results\\n' + 'Percent of overlap matched in BLAT: ' + format(percent_of_overlap, '.3f') + ' \\n')\n",
    "            #CHECKS TO MAKE SURE WE DID THIS RIGHT - correct end vs correct end\n",
    "            #if '+' not in strand:\n",
    "            #    print('ERROR: Top hit not in proper orientation.... SKIPPING -- PLEASE CHECK')\n",
    "            #    return\n",
    "            #parse contig #1 (left contig)\n",
    "            blat_length1, blat_start1, blat_end1 = int(line[14]),int(line[15]),int(line[16])\n",
    "            print(blat_length1, blat_start1, blat_end1)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length1, blat_start1, blat_end1))\n",
    "            #parse contig #2 (right contig)\n",
    "            blat_length2, blat_start2, blat_end2 = int(line[10]),int(line[11]),int(line[12])\n",
    "            print(blat_length2, blat_start2, blat_end2)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length2, blat_start2, blat_end2))\n",
    "            #Calculate offset  - to compensate for non-clean alignments (extension of left contig that does not overlap with adjacent contig)\n",
    "            offset = calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2)\n",
    "            return offset\n",
    "###################################################################################################\n",
    "def calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2):\n",
    "    global offset\n",
    "    if 'rc' in Dir1:\n",
    "        offset = blat_start1 - 0\n",
    "    else:\n",
    "        offset = blat_length1 - blat_end1\n",
    "    if offset < 3:\n",
    "        offset = 0\n",
    "    print('\\nBLAT offset = ', offset)\n",
    "    logFile.write('\\nBLAT offset = ' + str(offset) + '\\n')\n",
    "    return offset\n",
    "\n",
    "###################################################################################################\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "# CTG-0076 CTG-0353 \n",
      " chr1 202580 7069329 CTG-0076 rc 6872240 \n",
      " chr1 7064688 9594107 CTG-0353 rc 2527416\n",
      "\n",
      "Overlap =  4641\n",
      "Coordinates to extract for BLAT:  [0, 14641] [2512775, 2527416] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.966\n",
      "14641 0 4568\n",
      "14642 10112 14642\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0353 CTG-0052 \n",
      " chr1 7064688 9594107 CTG-0353 rc 2527416 \n",
      " chr1 9575524 17361463 CTG-0052 fwd 7832328\n",
      "\n",
      "Overlap =  18583\n",
      "Coordinates to extract for BLAT:  [0, 28583] [0, 28583] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.996\n",
      "28583 0 18559\n",
      "28583 65 18675\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0052 CTG-0231 \n",
      " chr1 9575524 17361463 CTG-0052 fwd 7832328 \n",
      " chr1 17343875 20909826 CTG-0231 rc 3565775\n",
      "\n",
      "Overlap =  17588\n",
      "Coordinates to extract for BLAT:  [7804740, 7832328] [3538187, 3565775] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.986\n",
      "27589 10057 27589\n",
      "27589 10168 27589\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0231 CTG-0143 \n",
      " chr1 17343875 20909826 CTG-0231 rc 3565775 \n",
      " chr1 20906456 25699815 CTG-0143 fwd 4864237\n",
      "\n",
      "Overlap =  3370\n",
      "Coordinates to extract for BLAT:  [0, 13370] [0, 13370] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.999\n",
      "13370 0 3368\n",
      "13370 0 3371\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0143 CTG-1396 \n",
      " chr1 20906456 25699815 CTG-0143 fwd 4864237 \n",
      " chr1 25699816 25725901 CTG-1396 fwd 25604\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1396 CTG-1379 \n",
      " chr1 25699816 25725901 CTG-1396 fwd 25604 \n",
      " chr1 25711171 25739889 CTG-1379 rc 28991\n",
      "\n",
      "Overlap =  14730\n",
      "Coordinates to extract for BLAT:  [874, 25604] [4261, 28991] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.012\n",
      "24731 9677 24731\n",
      "24731 9693 24731\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1379 CTG-0024 \n",
      " chr1 25711171 25739889 CTG-1379 rc 28991 \n",
      " chr1 25755738 36467787 CTG-0024 fwd 10717598\n",
      "\n",
      "Overlap =  -15849\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0024 CTG-0385 \n",
      " chr1 25755738 36467787 CTG-0024 fwd 10717598 \n",
      " chr1 36458508 38724234 CTG-0385 rc 2268506\n",
      "\n",
      "Overlap =  9279\n",
      "Coordinates to extract for BLAT:  [10698319, 10717598] [2249227, 2268506] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.990\n",
      "19280 10024 19277\n",
      "19280 10084 19280\n",
      "\n",
      "BLAT offset =  3\n",
      "\n",
      "# CTG-0385 CTG-0004 \n",
      " chr1 36458508 38724234 CTG-0385 rc 2268506 \n",
      " chr1 38720302 59110378 CTG-0004 fwd 20419135\n",
      "\n",
      "Overlap =  3932\n",
      "Coordinates to extract for BLAT:  [0, 13932] [0, 13932] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.999\n",
      "13932 0 4791\n",
      "13932 0 3930\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0004 CTG-0407 \n",
      " chr1 38720302 59110378 CTG-0004 fwd 20419135 \n",
      " chr1 59096260 61209851 CTG-0407 fwd 2126156\n",
      "\n",
      "Overlap =  14118\n",
      "Coordinates to extract for BLAT:  [20395017, 20419135] [0, 24118] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.976\n",
      "24119 10268 24119\n",
      "24118 0 14074\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0407 CTG-0130 \n",
      " chr1 59096260 61209851 CTG-0407 fwd 2126156 \n",
      " chr1 61188275 66351705 CTG-0130 rc 5154938\n",
      "\n",
      "Overlap =  21576\n",
      "Coordinates to extract for BLAT:  [2094580, 2126156] [5123362, 5154938] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.011\n",
      "31577 9655 31577\n",
      "31577 9684 31577\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0130 CTG-0993 \n",
      " chr1 61188275 66351705 CTG-0130 rc 5154938 \n",
      " chr1 66351706 66556441 CTG-0993 rc 211149\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0993 CTG-0862 \n",
      " chr1 66351706 66556441 CTG-0993 rc 211149 \n",
      " chr1 66556429 66965443 CTG-0862 rc 401947\n",
      "\n",
      "Overlap =  12\n",
      "Coordinates to extract for BLAT:  [0, 10012] [391935, 401947] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  9.250\n",
      "10012 4713 4874\n",
      "10013 5652 5783\n",
      "\n",
      "BLAT offset =  4713\n",
      "\n",
      "# CTG-0862 CTG-0436 \n",
      " chr1 66556429 66965443 CTG-0862 rc 401947 \n",
      " chr1 66965446 68897138 CTG-0436 rc 1936162\n",
      "\n",
      "Overlap =  -3\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0436 CTG-0765 \n",
      " chr1 66965446 68897138 CTG-0436 rc 1936162 \n",
      " chr1 68905887 69550157 CTG-0765 fwd 654711\n",
      "\n",
      "Overlap =  -8749\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0765 CTG-1087 \n",
      " chr1 68905887 69550157 CTG-0765 fwd 654711 \n",
      " chr1 69587225 69718599 CTG-1087 rc 125331\n",
      "\n",
      "Overlap =  -37068\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1087 CTG-0660 \n",
      " chr1 69587225 69718599 CTG-1087 rc 125331 \n",
      " chr1 69711636 70626323 CTG-0660 fwd 913263\n",
      "\n",
      "Overlap =  6963\n",
      "Coordinates to extract for BLAT:  [0, 16963] [0, 16963] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.993\n",
      "16963 0 6975\n",
      "16963 0 7032\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0660 CTG-0097 \n",
      " chr1 69711636 70626323 CTG-0660 fwd 913263 \n",
      " chr1 70627236 76622361 CTG-0097 fwd 6008365\n",
      "\n",
      "Overlap =  -913\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0097 CTG-0111 \n",
      " chr1 70627236 76622361 CTG-0097 fwd 6008365 \n",
      " chr1 76599640 81949499 CTG-0111 rc 5393392\n",
      "\n",
      "Overlap =  22721\n",
      "Coordinates to extract for BLAT:  [5975644, 6008365] [5360671, 5393392] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.076\n",
      "32722 7967 32722\n",
      "32722 8181 32722\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0111 CTG-0369 \n",
      " chr1 76599640 81949499 CTG-0111 rc 5393392 \n",
      " chr1 81942528 84296154 CTG-0369 fwd 2357637\n",
      "\n",
      "Overlap =  6971\n",
      "Coordinates to extract for BLAT:  [0, 16971] [0, 16971] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.995\n",
      "16971 0 7004\n",
      "16971 0 6961\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0369 CTG-0859 \n",
      " chr1 81942528 84296154 CTG-0369 fwd 2357637 \n",
      " chr1 84305675 84729161 CTG-0859 fwd 422460\n",
      "\n",
      "Overlap =  -9521\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0859 CTG-0020 \n",
      " chr1 84305675 84729161 CTG-0859 fwd 422460 \n",
      " chr1 84718798 95730658 CTG-0020 rc 11023292\n",
      "\n",
      "Overlap =  10363\n",
      "Coordinates to extract for BLAT:  [402097, 422460] [11002929, 11023292] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.968\n",
      "20364 10255 20364\n",
      "20364 10063 20364\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0020 CTG-0454 \n",
      " chr1 84718798 95730658 CTG-0020 rc 11023292 \n",
      " chr1 95743978 97604415 CTG-0454 fwd 1866490\n",
      "\n",
      "Overlap =  -13320\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0454 CTG-1064 \n",
      " chr1 95743978 97604415 CTG-0454 fwd 1866490 \n",
      " chr1 97594676 97736654 CTG-1064 fwd 143852\n",
      "\n",
      "Overlap =  9739\n",
      "Coordinates to extract for BLAT:  [1846751, 1866490] [0, 19739] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.997\n",
      "19740 10004 19740\n",
      "19739 72 9805\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1064 CTG-0535 \n",
      " chr1 97594676 97736654 CTG-1064 fwd 143852 \n",
      " chr1 97781455 99248611 CTG-0535 fwd 1442346\n",
      "\n",
      "Overlap =  -44801\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0535 CTG-1061 \n",
      " chr1 97781455 99248611 CTG-0535 fwd 1442346 \n",
      " chr1 99211891 99356231 CTG-1061 rc 132265\n",
      "\n",
      "Overlap =  36720\n",
      "Coordinates to extract for BLAT:  [1395626, 1442346] [85545, 132265] \n",
      "\n",
      "\n",
      "# CTG-1061 CTG-0608 \n",
      " chr1 99211891 99356231 CTG-1061 rc 132265 \n",
      " chr1 99334953 100405792 CTG-0608 rc 1122011\n",
      "\n",
      "Overlap =  21278\n",
      "Coordinates to extract for BLAT:  [0, 31278] [1090733, 1122011] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.987\n",
      "31278 0 21269\n",
      "31279 9329 31279\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0608 CTG-1208 \n",
      " chr1 99334953 100405792 CTG-0608 rc 1122011 \n",
      " chr1 100405793 100459886 CTG-1208 rc 55152\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1208 CTG-0706 \n",
      " chr1 100405793 100459886 CTG-1208 rc 55152 \n",
      " chr1 100459887 101266045 CTG-0706 rc 805962\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0706 CTG-1296 \n",
      " chr1 100459887 101266045 CTG-0706 rc 805962 \n",
      " chr1 101261688 101312301 CTG-1296 fwd 35875\n",
      "\n",
      "Overlap =  4357\n",
      "Coordinates to extract for BLAT:  [0, 14357] [0, 14357] \n",
      "\n",
      "\n",
      "# CTG-1296 CTG-0529 \n",
      " chr1 101261688 101312301 CTG-1296 fwd 35875 \n",
      " chr1 101301041 102769755 CTG-0529 rc 1472751\n",
      "\n",
      "Overlap =  11260\n",
      "Coordinates to extract for BLAT:  [14615, 35875] [1451491, 1472751] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.003\n",
      "21261 9960 21261\n",
      "21261 9962 21261\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0529 CTG-0791 \n",
      " chr1 101301041 102769755 CTG-0529 rc 1472751 \n",
      " chr1 102773034 103333006 CTG-0791 rc 560087\n",
      "\n",
      "Overlap =  -3279\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0791 CTG-0826 \n",
      " chr1 102773034 103333006 CTG-0791 rc 560087 \n",
      " chr1 103346881 104014811 CTG-0826 fwd 468543\n",
      "\n",
      "Overlap =  -13875\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0826 CTG-0915 \n",
      " chr1 103346881 104014811 CTG-0826 fwd 468543 \n",
      " chr1 103995997 104545757 CTG-0915 rc 315752\n",
      "\n",
      "Overlap =  18814\n",
      "Coordinates to extract for BLAT:  [439729, 468543] [286938, 315752] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.979\n",
      "28815 10325 28815\n",
      "28815 10064 28815\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0915 CTG-1354 \n",
      " chr1 103995997 104545757 CTG-0915 rc 315752 \n",
      " chr1 104528420 104558658 CTG-1354 fwd 29968\n",
      "\n",
      "Overlap =  17337\n",
      "Coordinates to extract for BLAT:  [0, 27337] [0, 27337] \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "# CTG-1354 CTG-1183 \n",
      " chr1 104528420 104558658 CTG-1354 fwd 29968 \n",
      " chr1 104542988 104606793 CTG-1183 rc 63633\n",
      "\n",
      "Overlap =  15670\n",
      "Coordinates to extract for BLAT:  [4298, 29968] [37963, 63633] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.955\n",
      "25671 10248 25671\n",
      "25671 10021 25671\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1183 CTG-0366 \n",
      " chr1 104542988 104606793 CTG-1183 rc 63633 \n",
      " chr1 104568678 106945658 CTG-0366 fwd 2388817\n",
      "\n",
      "Overlap =  38115\n",
      "Coordinates to extract for BLAT:  [0, 48115] [0, 48115] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.964\n",
      "48115 0 38000\n",
      "48115 0 37981\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0366 CTG-0626 \n",
      " chr1 104568678 106945658 CTG-0366 fwd 2388817 \n",
      " chr1 106947491 107986272 CTG-0626 rc 1036859\n",
      "\n",
      "Overlap =  -1833\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0626 CTG-1009 \n",
      " chr1 106947491 107986272 CTG-0626 rc 1036859 \n",
      " chr1 107971823 108201839 CTG-1009 rc 194951\n",
      "\n",
      "Overlap =  14449\n",
      "Coordinates to extract for BLAT:  [0, 24449] [170502, 194951] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.988\n",
      "24449 0 14666\n",
      "24450 10047 24450\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1009 CTG-1229 \n",
      " chr1 107971823 108201839 CTG-1009 rc 194951 \n",
      " chr1 108177315 108227703 CTG-1229 rc 50417\n",
      "\n",
      "Overlap =  24524\n",
      "Coordinates to extract for BLAT:  [0, 34524] [15893, 50417] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.983\n",
      "34524 0 24504\n",
      "34525 10297 34525\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1229 CTG-0724 \n",
      " chr1 108177315 108227703 CTG-1229 rc 50417 \n",
      " chr1 108229326 108990375 CTG-0724 fwd 758922\n",
      "\n",
      "Overlap =  -1623\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0724 CTG-0121 \n",
      " chr1 108229326 108990375 CTG-0724 fwd 758922 \n",
      " chr1 108979413 114321508 CTG-0121 fwd 5297349\n",
      "\n",
      "Overlap =  10962\n",
      "Coordinates to extract for BLAT:  [737960, 758922] [0, 20962] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.989\n",
      "20963 9791 20963\n",
      "20962 0 10911\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0121 CTG-1223 \n",
      " chr1 108979413 114321508 CTG-0121 fwd 5297349 \n",
      " chr1 114300900 114347957 CTG-1223 rc 47449\n",
      "\n",
      "Overlap =  20608\n",
      "Coordinates to extract for BLAT:  [5266741, 5297349] [16841, 47449] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.795\n",
      "30609 10080 30609\n",
      "30609 10167 30609\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1223 CTG-0435 \n",
      " chr1 114300900 114347957 CTG-1223 rc 47449 \n",
      " chr1 114310791 116236272 CTG-0435 rc 1922262\n",
      "\n",
      "Overlap =  37166\n",
      "Coordinates to extract for BLAT:  [0, 47166] [1875096, 1922262] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.903\n",
      "47166 0 37719\n",
      "47167 9103 47167\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0435 CTG-0920 \n",
      " chr1 114310791 116236272 CTG-0435 rc 1922262 \n",
      " chr1 116254178 116564076 CTG-0920 fwd 308056\n",
      "\n",
      "Overlap =  -17906\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0920 CTG-0259 \n",
      " chr1 116254178 116564076 CTG-0920 fwd 308056 \n",
      " chr1 116551203 119906706 CTG-0259 rc 3349177\n",
      "\n",
      "Overlap =  12873\n",
      "Coordinates to extract for BLAT:  [285183, 308056] [3326304, 3349177] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.000\n",
      "22874 9992 22874\n",
      "22874 9980 22874\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0259 CTG-0422 \n",
      " chr1 116551203 119906706 CTG-0259 rc 3349177 \n",
      " chr1 119891324 121923628 CTG-0422 rc 2059298\n",
      "\n",
      "Overlap =  15382\n",
      "Coordinates to extract for BLAT:  [0, 25382] [2033916, 2059298] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.986\n",
      "25382 0 15310\n",
      "25383 10013 25383\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0422 CTG-1030 \n",
      " chr1 119891324 121923628 CTG-0422 rc 2059298 \n",
      " chr1 121933605 122105081 CTG-1030 rc 171326\n",
      "\n",
      "Overlap =  -9977\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1030 CTG-0793 \n",
      " chr1 121933605 122105081 CTG-1030 rc 171326 \n",
      " chr1 122113446 122678785 CTG-0793 fwd 564775\n",
      "\n",
      "Overlap =  -8365\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  122663758\n",
      "Coordinates to extract for BLAT:  [-122653758, 20000] [0, 122673758] \n",
      "\n",
      "Coordinates to extract for BLAT:  [-122653758, 20000] [0, 122673758] \n",
      "\n",
      "\n",
      "# CTG-0119 CTG-2081 \n",
      " chr10 15027 5357940 CTG-0119 rc 5332249 \n",
      " chr10 2164504 2173224 CTG-2081 fwd 8719\n",
      "\n",
      "Overlap =  3193436\n",
      "Coordinates to extract for BLAT:  [0, 3203436] [0, 3203436] \n",
      "\n",
      "\n",
      "# CTG-2081 CTG-1696 \n",
      " chr10 2164504 2173224 CTG-2081 fwd 8719 \n",
      " chr10 5357941 5374672 CTG-1696 rc 16701\n",
      "\n",
      "Overlap =  -3184717\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1696 CTG-1981 \n",
      " chr10 5357941 5374672 CTG-1696 rc 16701 \n",
      " chr10 5374589 5377417 CTG-1981 fwd 2819\n",
      "\n",
      "Overlap =  83\n",
      "Coordinates to extract for BLAT:  [0, 10083] [0, 10083] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.012\n",
      "10083 0 84\n",
      "2819 0 84\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1981 CTG-0828 \n",
      " chr10 5374589 5377417 CTG-1981 fwd 2819 \n",
      " chr10 5376493 5851406 CTG-0828 rc 479898\n",
      "\n",
      "Overlap =  924\n",
      "Coordinates to extract for BLAT:  [-8105, 2819] [468974, 479898] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.994\n",
      "2819 1897 2819\n",
      "10925 7437 8362\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0828 CTG-1638 \n",
      " chr10 5376493 5851406 CTG-0828 rc 479898 \n",
      " chr10 5851411 5869780 CTG-1638 rc 18473\n",
      "\n",
      "Overlap =  -5\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1638 CTG-0463 \n",
      " chr10 5851411 5869780 CTG-1638 rc 18473 \n",
      " chr10 5865187 7677964 CTG-0463 fwd 1808680\n",
      "\n",
      "Overlap =  4593\n",
      "Coordinates to extract for BLAT:  [0, 14593] [0, 14593] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.017\n",
      "14593 0 4703\n",
      "14593 0 4948\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0463 CTG-0876 \n",
      " chr10 5865187 7677964 CTG-0463 fwd 1808680 \n",
      " chr10 10448133 10831424 CTG-0876 rc 382392\n",
      "\n",
      "Overlap =  -2770169\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0876 CTG-0633 \n",
      " chr10 10448133 10831424 CTG-0876 rc 382392 \n",
      " chr10 10831425 11857519 CTG-0633 rc 1018080\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0633 CTG-0654 \n",
      " chr10 10831425 11857519 CTG-0633 rc 1018080 \n",
      " chr10 11861231 12810286 CTG-0654 rc 951678\n",
      "\n",
      "Overlap =  -3712\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0654 CTG-1418 \n",
      " chr10 11861231 12810286 CTG-0654 rc 951678 \n",
      " chr10 12270253 12297198 CTG-1418 fwd 27066\n",
      "\n",
      "Overlap =  540033\n",
      "Coordinates to extract for BLAT:  [0, 550033] [0, 550033] \n",
      "\n",
      "\n",
      "# CTG-1418 CTG-0650 \n",
      " chr10 12270253 12297198 CTG-1418 fwd 27066 \n",
      " chr10 12814209 13775761 CTG-0650 fwd 960528\n",
      "\n",
      "Overlap =  -517011\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0650 CTG-0545 \n",
      " chr10 12814209 13775761 CTG-0650 fwd 960528 \n",
      " chr10 13746681 15140472 CTG-0545 fwd 1390529\n",
      "\n",
      "Overlap =  29080\n",
      "Coordinates to extract for BLAT:  [921448, 960528] [0, 39080] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.029\n",
      "39081 9123 39081\n",
      "39080 0 29948\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0545 CTG-0689 \n",
      " chr10 13746681 15140472 CTG-0545 fwd 1390529 \n",
      " chr10 15137022 15993756 CTG-0689 rc 860982\n",
      "\n",
      "Overlap =  3450\n",
      "Coordinates to extract for BLAT:  [1377079, 1390529] [847532, 860982] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.997\n",
      "13451 10005 13451\n",
      "13451 10004 13451\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0689 CTG-1376 \n",
      " chr10 15137022 15993756 CTG-0689 rc 860982 \n",
      " chr10 16017240 16046284 CTG-1376 fwd 29018\n",
      "\n",
      "Overlap =  -23484\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1376 CTG-1035 \n",
      " chr10 16017240 16046284 CTG-1376 fwd 29018 \n",
      " chr10 16033440 16164457 CTG-1035 rc 173590\n",
      "\n",
      "Overlap =  12844\n",
      "Coordinates to extract for BLAT:  [6174, 29018] [150746, 173590] \n",
      "\n",
      "\n",
      "# CTG-1035 CTG-1217 \n",
      " chr10 16033440 16164457 CTG-1035 rc 173590 \n",
      " chr10 16177621 16229510 CTG-1217 fwd 51867\n",
      "\n",
      "Overlap =  -13164\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1217 CTG-1043 \n",
      " chr10 16177621 16229510 CTG-1217 fwd 51867 \n",
      " chr10 16227491 16393169 CTG-1043 rc 164648\n",
      "\n",
      "Overlap =  2019\n",
      "Coordinates to extract for BLAT:  [39848, 51867] [152629, 164648] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.482\n",
      "12020 1833 7734\n",
      "12020 2182 11171\n",
      "\n",
      "BLAT offset =  4286\n",
      "\n",
      "# CTG-1043 CTG-1004 \n",
      " chr10 16227491 16393169 CTG-1043 rc 164648 \n",
      " chr10 16408407 16557742 CTG-1004 rc 194250\n",
      "\n",
      "Overlap =  -15238\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-1004 CTG-2042 \n",
      " chr10 16408407 16557742 CTG-1004 rc 194250 \n",
      " chr10 16554326 16557433 CTG-2042 fwd 9522\n",
      "\n",
      "Overlap =  3416\n",
      "Coordinates to extract for BLAT:  [0, 13416] [0, 13416] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.758\n",
      "13416 7432 10338\n",
      "9522 4677 7616\n",
      "\n",
      "BLAT offset =  7432\n",
      "\n",
      "# CTG-2042 CTG-0039 \n",
      " chr10 16554326 16557433 CTG-2042 fwd 9522 \n",
      " chr10 16557255 25491258 CTG-0039 rc 8950220\n",
      "\n",
      "Overlap =  178\n",
      "Coordinates to extract for BLAT:  [-656, 9522] [8940042, 8950220] \n",
      "\n",
      "\n",
      "# CTG-0039 CTG-1542 \n",
      " chr10 16557255 25491258 CTG-0039 rc 8950220 \n",
      " chr10 25484567 25506164 CTG-1542 rc 21472\n",
      "\n",
      "Overlap =  6691\n",
      "Coordinates to extract for BLAT:  [0, 16691] [4781, 21472] \n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.992\n",
      "16691 0 6705\n",
      "16692 10028 16689\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-1542 CTG-0838 \n",
      " chr10 25484567 25506164 CTG-1542 rc 21472 \n",
      " chr10 25488111 25926063 CTG-0838 fwd 460067\n",
      "\n",
      "Overlap =  18053\n",
      "Coordinates to extract for BLAT:  [0, 28053] [0, 28053] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.968\n",
      "21472 0 17947\n",
      "28053 0 17635\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0838 CTG-0701 \n",
      " chr10 25488111 25926063 CTG-0838 fwd 460067 \n",
      " chr10 25889999 26731033 CTG-0701 rc 826784\n",
      "\n",
      "Overlap =  36064\n",
      "Coordinates to extract for BLAT:  [414003, 460067] [780720, 826784] \n",
      "\n",
      "\n",
      "# CTG-0701 CTG-0037 \n",
      " chr10 25889999 26731033 CTG-0701 rc 826784 \n",
      " chr10 26727734 35678751 CTG-0037 rc 9019875\n",
      "\n",
      "Overlap =  3299\n",
      "Coordinates to extract for BLAT:  [0, 13299] [9006576, 9019875] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.997\n",
      "13299 0 3297\n",
      "13300 10000 13300\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0037 CTG-0118 \n",
      " chr10 26727734 35678751 CTG-0037 rc 9019875 \n",
      " chr10 35674529 41019048 CTG-0118 rc 5335242\n",
      "\n",
      "Overlap =  4222\n",
      "Coordinates to extract for BLAT:  [0, 14222] [5321020, 5335242] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.995\n",
      "14222 0 4202\n",
      "14223 9999 14222\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0118 CTG-1802 \n",
      " chr10 35674529 41019048 CTG-0118 rc 5335242 \n",
      " chr10 40294563 40308624 CTG-1802 rc 13999\n",
      "\n",
      "Overlap =  724485\n",
      "Coordinates to extract for BLAT:  [0, 734485] [-720486, 13999] \n",
      "\n",
      "\n",
      "# CTG-1802 CTG-0587 \n",
      " chr10 40294563 40308624 CTG-1802 rc 13999 \n",
      " chr10 41017140 42277784 CTG-0587 rc 1256094\n",
      "\n",
      "Overlap =  -708516\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0587 CTG-0566 \n",
      " chr10 41017140 42277784 CTG-0587 rc 1256094 \n",
      " chr10 42265116 43604437 CTG-0566 rc 1330183\n",
      "\n",
      "Overlap =  12668\n",
      "Coordinates to extract for BLAT:  [0, 22668] [1307515, 1330183] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.955\n",
      "22668 0 12357\n",
      "22669 10499 22669\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "# CTG-0566 CTG-0494 \n",
      " chr10 42265116 43604437 CTG-0566 rc 1330183 \n",
      " chr10 43604234 45235939 CTG-0494 fwd 1637300\n",
      "\n",
      "Overlap =  203\n",
      "Coordinates to extract for BLAT:  [0, 10203] [0, 10203] \n",
      "\n",
      "\n",
      "# CTG-0494 CTG-0094 \n",
      " chr10 43604234 45235939 CTG-0494 fwd 1637300 \n",
      " chr10 45244757 51364851 CTG-0094 fwd 6104459\n",
      "\n",
      "Overlap =  -8818\n",
      "Gap between contigs\n",
      "\n",
      "# CTG-0094 CTG-1875 \n",
      " chr10 45244757 51364851 CTG-0094 fwd 6104459 \n",
      " chr10 48115729 48128481 CTG-1875 fwd 12727\n",
      "\n",
      "Overlap =  3249122\n",
      "Coordinates to extract for BLAT:  [2845337, 6104459] [0, 3259122] \n",
      "\n"
     ]
    }
   ],
   "source": [
    "#write_AGP_header(agpFile)\n",
    "posDict, offset_BLAT = {}, []\n",
    "\n",
    "for i in range(1,len(curated_contigDict)): \n",
    "    curr_index = i\n",
    "    offset = 0 #set equal to zero at beginning, will change if there is one from parsing the blat hit(s)\n",
    "    #1. Reading in coordinates from contigList\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(curated_contigDict, curr_index, i)\n",
    "    \n",
    "    #2. Checking contigs are on same chromosome\n",
    "    if chrom1 != chrom2:# or i == 1: #This is first contig on chromosome, need to process it first\n",
    "        #continue\n",
    "        process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1) \n",
    "        continue\n",
    "    print ('\\n#',contigID1, contigID2,'\\n',chrom1,start1,end1,contigID1,Dir1,length1,'\\n',chrom2,start2,end2,contigID2,Dir2,length2)\n",
    "    \n",
    "    #3. Determines the overlap/orientation of the two contigs\n",
    "    determine_contig_overlap(end1,contigID1,Dir1,length1,end2,contigID2,Dir2,length2)\n",
    "\n",
    "    #4. BLAT the properly oriented contig ends against one another\n",
    "    #     and parse the results\n",
    "    run_blat(wkDir)\n",
    "    parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "    if offset > 0:\n",
    "        offset_BLAT.append([contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2,overlap])\n",
    "    \n",
    "    #save contig1 information to dictionary\n",
    "    posDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1,overlap,offset,contigID2]\n",
    "\n",
    "\n",
    "    #print('###################\\n')\n",
    "    #if i > 2:\n",
    "    #    break\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['chr18', 2619, 26992, 'CTG-1462', 'rc', 24712, 4849, 0, 'CTG-0201']\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', 3968428, 1729, 0, 'CTG-1861']\n",
      "['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', 13154, 2637, 0, 'CTG-0743']\n",
      "['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', 681475, 25890, 0, 'CTG-0770']\n",
      "['chr18', 4634379, 5281622, 'CTG-0770', 'rc', 639396, 21363, 0, 'CTG-0177']\n",
      "['chr18', 5260259, 9645181, 'CTG-0177', 'rc', 4363384, 6495, 0, 'CTG-0767']\n",
      "['chr18', 9638686, 10286737, 'CTG-0767', 'rc', 647068, 2059, 0, 'CTG-0615']\n",
      "['chr18', 10284678, 11357644, 'CTG-0615', 'rc', 1078170, 37187, 0, 'CTG-1278']\n",
      "['chr18', 11320457, 11367174, 'CTG-1278', 'fwd', 37673, 40908, 0, 'CTG-0400']\n",
      "['chr18', 11326266, 13501863, 'CTG-0400', 'rc', 2190587, -4852, 0, 'CTG-0564']\n",
      "['chr18', 13506715, 14852995, 'CTG-0564', 'rc', 1349347, -1291, 0, 'CTG-0274']\n",
      "['chr18', 14854286, 17956069, 'CTG-0274', 'rc', 3082702, -8964, 0, 'CTG-0878']\n",
      "['chr18', 17965033, 18359454, 'CTG-0878', 'rc', 395029, -30419, 0, 'CTG-1134']\n",
      "['chr18', 18389873, 18474563, 'CTG-1134', 'fwd', 85495, 4029, 0, 'CTG-1314']\n",
      "['chr18', 18470534, 18504760, 'CTG-1314', 'fwd', 34113, 13268, 0, 'CTG-1776']\n",
      "['chr18', 18491492, 18506121, 'CTG-1776', 'rc', 14596, 1360, 0, 'CTG-1294']\n",
      "['chr18', 18504761, 18543987, 'CTG-1294', 'fwd', 38857, -44329, 0, 'CTG-0810']\n",
      "['chr18', 18588316, 19084754, 'CTG-0810', 'rc', 490363, -7170, 0, 'CTG-0953']\n",
      "['chr18', 19091924, 19366433, 'CTG-0953', 'rc', 273581, -1515, 0, 'CTG-0581']\n",
      "['chr18', 19367948, 20648758, 'CTG-0581', 'fwd', 1280510, 13620, 0, 'CTG-0901']\n",
      "['chr18', 20635138, 20993149, 'CTG-0901', 'rc', 355818, -7656, 0, 'CTG-0483']\n",
      "['chr18', 21000805, 22654130, 'CTG-0483', 'rc', 1655346, 24646, 0, 'CTG-0776']\n",
      "['chr18', 22629484, 23228962, 'CTG-0776', 'rc', 597680, 14898, 0, 'CTG-0412']\n",
      "['chr18', 23214064, 25317843, 'CTG-0412', 'rc', 2106211, -14267, 0, 'CTG-0630']\n",
      "['chr18', 25332110, 26328164, 'CTG-0630', 'fwd', 1034698, 20307, 0, 'CTG-1329']\n",
      "['chr18', 26307857, 26340773, 'CTG-1329', 'rc', 32757, -8228, 0, 'CTG-0458']\n",
      "['chr18', 26349001, 28160518, 'CTG-0458', 'fwd', 1814553, -2158, 0, 'CTG-0416']\n",
      "['chr18', 28162676, 30223054, 'CTG-0416', 'rc', 2045472, 5188, 0, 'CTG-1241']\n",
      "['chr18', 30217866, 30259696, 'CTG-1241', 'rc', 47540, 11141, 5040, 'CTG-0907']\n",
      "['chr18', 30248555, 30591231, 'CTG-0907', 'rc', 349004, -18271, 0, 'CTG-0235']\n",
      "['chr18', 30609502, 34128003, 'CTG-0235', 'fwd', 3508078, 10472, 0, 'CTG-0351']\n",
      "['chr18', 34117531, 36616051, 'CTG-0351', 'rc', 2534292, -13551, 0, 'CTG-0725']\n",
      "['chr18', 36629602, 37366888, 'CTG-0725', 'fwd', 736753, -51643, 0, 'CTG-0432']\n",
      "['chr18', 37418531, 39290284, 'CTG-0432', 'fwd', 1950806, 13916, 0, 'CTG-1297']\n",
      "['chr18', 39276368, 39312643, 'CTG-1297', 'rc', 35966, -13658, 0, 'CTG-0756']\n",
      "['chr18', 39326301, 39981535, 'CTG-0756', 'fwd', 668698, 16254, 0, 'CTG-0835']\n",
      "['chr18', 39965281, 40427697, 'CTG-0835', 'fwd', 450689, -2585, 0, 'CTG-2129']\n",
      "['chr18', 40430282, 40438128, 'CTG-2129', 'rc', 7821, -91411, 0, 'CTG-0720']\n",
      "['chr18', 40529539, 41359396, 'CTG-0720', 'fwd', 738619, 17872, 0, 'CTG-1048']\n",
      "['chr18', 41341524, 41503975, 'CTG-1048', 'fwd', 164381, 10209, 0, 'CTG-0817']\n",
      "['chr18', 41493766, 41997929, 'CTG-0817', 'rc', 504439, -6219, 0, 'CTG-0272']\n",
      "['chr18', 42004148, 45129980, 'CTG-0272', 'fwd', 3124127, -15243, 0, 'CTG-0989']\n",
      "['chr18', 45145223, 45362298, 'CTG-0989', 'fwd', 216468, -5494, 0, 'CTG-0213']\n",
      "['chr18', 45367792, 49109540, 'CTG-0213', 'fwd', 3736794, 25995, 0, 'CTG-1240']\n",
      "['chr18', 49083545, 49130813, 'CTG-1240', 'rc', 47251, 34303, 0, 'CTG-0854']\n",
      "['chr18', 49096510, 49548067, 'CTG-0854', 'fwd', 454291, -4039, 0, 'CTG-1000']\n",
      "['chr18', 49552106, 49746838, 'CTG-1000', 'fwd', 194595, -1, 0, 'CTG-0282']\n",
      "['chr18', 49746839, 52779109, 'CTG-0282', 'fwd', 3034721, 10361, 0, 'CTG-0620']\n",
      "['chr18', 52768748, 53825339, 'CTG-0620', 'rc', 1057651, -9050, 0, 'CTG-0459']\n"
     ]
    }
   ],
   "source": [
    "for i in posDict:\n",
    "    print (posDict[i][0:9])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = agp_chrom, 0, 0\n",
    "    \n",
    "    if direction == 'fwd':\n",
    "        direction = '+' #changes notation of the direction\n",
    "    if direction == 'rc':\n",
    "        direction = '-'  #changes notation of the direction\n",
    "    \n",
    "    #Specially process those with offsets > 0\n",
    "    if offset > 0 or prev_offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "    #Those with BLAT offsets = 0\n",
    "    else:\n",
    "        agp_start, agp_end = 1, length\n",
    "        contig_start, contig_end = 1, length\n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "########################################################################\n",
    "def process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction):\n",
    "    if agp_prev_overlap < 5 and agp_prev_overlap > 0:\n",
    "        agp_prev_overlap = 0\n",
    "        \n",
    "    if prev_offset ==0  and offset > 0: #contig left of offset\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "\n",
    "    if prev_offset > 0 and offset == 0: #contig right of offset\n",
    "        agp_start = agp_end + 500 + 2 #compensate for added gap between the two which has length = 500\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap  \n",
    "            \n",
    "    if prev_offset > 0 and offset > 0:\n",
    "        print('This type of alignment needs to be readdressed...\\n')\n",
    "        sys.exit()\n",
    "        \n",
    "        \n",
    "    info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "    \n",
    "    return info\n",
    "\n",
    "########################################################################\n",
    "def process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    agp_prev_chrom, agp_prev_overlap, prev_offset = posDict[i-1][0], int(posDict[i-1][6]),int(posDict[i-1][7])\n",
    "    \n",
    "    #Specially address those with BLAT offsets (unaligned sequence at the junctions)\n",
    "    if prev_offset > 0 or offset > 0:\n",
    "        info = process_blat_offsets(agp_prev_chrom, agp_prev_overlap, prev_offset, agp_start, agp_end, overlap, offset,direction)\n",
    "        return info\n",
    "    \n",
    "    #Junctions without unaligned sequence(s):\n",
    "    else:\n",
    "        if agp_prev_overlap < 5:\n",
    "            agp_prev_overlap = 0\n",
    "\n",
    "        agp_start = agp_end + 1\n",
    "        agp_end = agp_start + length + agp_prev_overlap\n",
    "\n",
    "        #Determine contig coordinates that align\n",
    "        if direction == 'fwd':\n",
    "            direction = '+' #changes notation of the direction\n",
    "            contig_start = 1 + agp_prev_overlap\n",
    "            contig_end = length\n",
    "        if direction == 'rc':\n",
    "            direction = '-'  #changes notation of the direction\n",
    "            contig_start = 1\n",
    "            contig_end = length - agp_prev_overlap\n",
    "\n",
    "        info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction,offset]\n",
    "        return info       \n",
    "        \n",
    "########################################################################  \n",
    "def process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    gap_length = 10000\n",
    "    agp_start = agp_end + 1\n",
    "    agp_end = agp_start + gap_length\n",
    "    if 'CTG' in contigID:\n",
    "        gap_type = 'contig'\n",
    "    else:#To change once we add more types of contigs in here\n",
    "        gap_type = 'OTHER'\n",
    "    agp_prev_overlap = 0\n",
    "    \n",
    "    info = [agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap]\n",
    "    return info  \n",
    "\n",
    "########################################################################  \n",
    "def write_AGP_header(agpFile): #alter for each assembly\n",
    "    agpFile.write('##agp-version 1.0\\n# ORGANISM: Canis lupus familiaris\\n# TAX_ID: 9615\\n')\n",
    "    agpFile.write('# ASSEMBLY NAME: Zoey_v1\\n# ASSEMBLY DATE: 19-April-2017\\n')\n",
    "    agpFile.write('# GENOME CENTER: University of Michigan - J.M. Kidd Lab\\n')\n",
    "    agpFile.write('# DESCRIPTION: AGP specifying the assembly of chromosome 18 from primary PacBio contigs from FALCON assembly\\n')      \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now putting the information in the curated contig set (contigs constituting the golden path) into the coordinates of Zoey, and adding gap positions, if present. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction\n",
      "\n",
      "# CTG-1462\n",
      "['chr18', 2619, 26992, 'CTG-1462', 'rc', 24712, 4849, 0, 'CTG-0201']\n",
      "....\n",
      "chr18 0 1 24712 1 24712 -\n",
      "\n",
      "# CTG-0201\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', 3968428, 1729, 0, 'CTG-1861']\n",
      "....\n",
      "chr18 4849 24713 3997990 1 3963579 -\n",
      "\n",
      "# CTG-1861\n",
      "['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', 13154, 2637, 0, 'CTG-0743']\n",
      "....\n",
      "chr18 1729 3997991 4012874 1730 13154 +\n",
      "\n",
      "# CTG-0743\n",
      "['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', 681475, 25890, 0, 'CTG-0770']\n",
      "....\n",
      "chr18 2637 4012875 4696987 2638 681475 +\n",
      "\n",
      "# CTG-0770\n",
      "['chr18', 4634379, 5281622, 'CTG-0770', 'rc', 639396, 21363, 0, 'CTG-0177']\n",
      "....\n",
      "chr18 25890 4696988 5362274 1 613506 -\n",
      "\n",
      "# CTG-0177\n",
      "['chr18', 5260259, 9645181, 'CTG-0177', 'rc', 4363384, 6495, 0, 'CTG-0767']\n",
      "....\n",
      "chr18 21363 5362275 9747022 1 4342021 -\n",
      "\n",
      "# CTG-0767\n",
      "['chr18', 9638686, 10286737, 'CTG-0767', 'rc', 647068, 2059, 0, 'CTG-0615']\n",
      "....\n",
      "chr18 6495 9747023 10400586 1 640573 -\n",
      "\n",
      "# CTG-0615\n",
      "['chr18', 10284678, 11357644, 'CTG-0615', 'rc', 1078170, 37187, 0, 'CTG-1278']\n",
      "....\n",
      "chr18 2059 10400587 11480816 1 1076111 -\n",
      "\n",
      "# CTG-1278\n",
      "['chr18', 11320457, 11367174, 'CTG-1278', 'fwd', 37673, 40908, 0, 'CTG-0400']\n",
      "....\n",
      "chr18 37187 11480817 11555677 37188 37673 +\n",
      "\n",
      "# CTG-0400\n",
      "['chr18', 11326266, 13501863, 'CTG-0400', 'rc', 2190587, -4852, 0, 'CTG-0564']\n",
      "....\n",
      "chr18 11555678 13787173 10 D CTG-0400 1 2149679 -\n",
      "chr18 13787174 13797174 11 U 10000 contig no na\n",
      "\n",
      "# CTG-0564\n",
      "['chr18', 13506715, 14852995, 'CTG-0564', 'rc', 1349347, -1291, 0, 'CTG-0274']\n",
      "....\n",
      "chr18 13797175 15146522 12 D CTG-0564 1 1349347 -\n",
      "chr18 15146523 15156523 13 U 10000 contig no na\n",
      "\n",
      "# CTG-0274\n",
      "['chr18', 14854286, 17956069, 'CTG-0274', 'rc', 3082702, -8964, 0, 'CTG-0878']\n",
      "....\n",
      "chr18 15156524 18239226 14 D CTG-0274 1 3082702 -\n",
      "chr18 18239227 18249227 15 U 10000 contig no na\n",
      "\n",
      "# CTG-0878\n",
      "['chr18', 17965033, 18359454, 'CTG-0878', 'rc', 395029, -30419, 0, 'CTG-1134']\n",
      "....\n",
      "chr18 18249228 18644257 16 D CTG-0878 1 395029 -\n",
      "chr18 18644258 18654258 17 U 10000 contig no na\n",
      "\n",
      "# CTG-1134\n",
      "['chr18', 18389873, 18474563, 'CTG-1134', 'fwd', 85495, 4029, 0, 'CTG-1314']\n",
      "....\n",
      "chr18 0 18654259 18739754 1 85495 +\n",
      "\n",
      "# CTG-1314\n",
      "['chr18', 18470534, 18504760, 'CTG-1314', 'fwd', 34113, 13268, 0, 'CTG-1776']\n",
      "....\n",
      "chr18 4029 18739755 18777897 4030 34113 +\n",
      "\n",
      "# CTG-1776\n",
      "['chr18', 18491492, 18506121, 'CTG-1776', 'rc', 14596, 1360, 0, 'CTG-1294']\n",
      "....\n",
      "chr18 13268 18777898 18805762 1 1328 -\n",
      "\n",
      "# CTG-1294\n",
      "['chr18', 18504761, 18543987, 'CTG-1294', 'fwd', 38857, -44329, 0, 'CTG-0810']\n",
      "....\n",
      "chr18 18805763 18845980 21 D CTG-1294 1361 38857 +\n",
      "chr18 18845981 18855981 22 U 10000 contig no na\n",
      "\n",
      "# CTG-0810\n",
      "['chr18', 18588316, 19084754, 'CTG-0810', 'rc', 490363, -7170, 0, 'CTG-0953']\n",
      "....\n",
      "chr18 18855982 19346345 23 D CTG-0810 1 490363 -\n",
      "chr18 19346346 19356346 24 U 10000 contig no na\n",
      "\n",
      "# CTG-0953\n",
      "['chr18', 19091924, 19366433, 'CTG-0953', 'rc', 273581, -1515, 0, 'CTG-0581']\n",
      "....\n",
      "chr18 19356347 19629928 25 D CTG-0953 1 273581 -\n",
      "chr18 19629929 19639929 26 U 10000 contig no na\n",
      "\n",
      "# CTG-0581\n",
      "['chr18', 19367948, 20648758, 'CTG-0581', 'fwd', 1280510, 13620, 0, 'CTG-0901']\n",
      "....\n",
      "chr18 0 19639930 20920440 1 1280510 +\n",
      "\n",
      "# CTG-0901\n",
      "['chr18', 20635138, 20993149, 'CTG-0901', 'rc', 355818, -7656, 0, 'CTG-0483']\n",
      "....\n",
      "chr18 20920441 21289879 28 D CTG-0901 1 342198 -\n",
      "chr18 21289880 21299880 29 U 10000 contig no na\n",
      "\n",
      "# CTG-0483\n",
      "['chr18', 21000805, 22654130, 'CTG-0483', 'rc', 1655346, 24646, 0, 'CTG-0776']\n",
      "....\n",
      "chr18 0 21299881 22955227 1 1655346 -\n",
      "\n",
      "# CTG-0776\n",
      "['chr18', 22629484, 23228962, 'CTG-0776', 'rc', 597680, 14898, 0, 'CTG-0412']\n",
      "....\n",
      "chr18 24646 22955228 23577554 1 573034 -\n",
      "\n",
      "# CTG-0412\n",
      "['chr18', 23214064, 25317843, 'CTG-0412', 'rc', 2106211, -14267, 0, 'CTG-0630']\n",
      "....\n",
      "chr18 23577555 25698664 32 D CTG-0412 1 2091313 -\n",
      "chr18 25698665 25708665 33 U 10000 contig no na\n",
      "\n",
      "# CTG-0630\n",
      "['chr18', 25332110, 26328164, 'CTG-0630', 'fwd', 1034698, 20307, 0, 'CTG-1329']\n",
      "....\n",
      "chr18 0 25708666 26743364 1 1034698 +\n",
      "\n",
      "# CTG-1329\n",
      "['chr18', 26307857, 26340773, 'CTG-1329', 'rc', 32757, -8228, 0, 'CTG-0458']\n",
      "....\n",
      "chr18 26743365 26796429 35 D CTG-1329 1 12450 -\n",
      "chr18 26796430 26806430 36 U 10000 contig no na\n",
      "\n",
      "# CTG-0458\n",
      "['chr18', 26349001, 28160518, 'CTG-0458', 'fwd', 1814553, -2158, 0, 'CTG-0416']\n",
      "....\n",
      "chr18 26806431 28620984 37 D CTG-0458 1 1814553 +\n",
      "chr18 28620985 28630985 38 U 10000 contig no na\n",
      "\n",
      "# CTG-0416\n",
      "['chr18', 28162676, 30223054, 'CTG-0416', 'rc', 2045472, 5188, 0, 'CTG-1241']\n",
      "....\n",
      "chr18 0 28630986 30676458 1 2045472 -\n",
      "\n",
      "# CTG-1241\n",
      "['chr18', 30217866, 30259696, 'CTG-1241', 'rc', 47540, 11141, 5040, 'CTG-0907']\n",
      "....\n",
      "chr18 5188 30676459 30729187 1 42352 -\n",
      "chr18 30729188 30729688 41 U 500 BLAT_gap no na\n",
      "\n",
      "# CTG-0907\n",
      "['chr18', 30248555, 30591231, 'CTG-0907', 'rc', 349004, -18271, 0, 'CTG-0235']\n",
      "....\n",
      "chr18 30729689 31089834 42 D CTG-0907 1 337863 -\n",
      "chr18 31089835 31099835 43 U 10000 contig no na\n",
      "\n",
      "# CTG-0235\n",
      "['chr18', 30609502, 34128003, 'CTG-0235', 'fwd', 3508078, 10472, 0, 'CTG-0351']\n",
      "....\n",
      "chr18 0 31099836 34607914 1 3508078 +\n",
      "\n",
      "# CTG-0351\n",
      "['chr18', 34117531, 36616051, 'CTG-0351', 'rc', 2534292, -13551, 0, 'CTG-0725']\n",
      "....\n",
      "chr18 34607915 37152679 45 D CTG-0351 1 2523820 -\n",
      "chr18 37152680 37162680 46 U 10000 contig no na\n",
      "\n",
      "# CTG-0725\n",
      "['chr18', 36629602, 37366888, 'CTG-0725', 'fwd', 736753, -51643, 0, 'CTG-0432']\n",
      "....\n",
      "chr18 37162681 37899434 47 D CTG-0725 1 736753 +\n",
      "chr18 37899435 37909435 48 U 10000 contig no na\n",
      "\n",
      "# CTG-0432\n",
      "['chr18', 37418531, 39290284, 'CTG-0432', 'fwd', 1950806, 13916, 0, 'CTG-1297']\n",
      "....\n",
      "chr18 0 37909436 39860242 1 1950806 +\n",
      "\n",
      "# CTG-1297\n",
      "['chr18', 39276368, 39312643, 'CTG-1297', 'rc', 35966, -13658, 0, 'CTG-0756']\n",
      "....\n",
      "chr18 39860243 39910125 50 D CTG-1297 1 22050 -\n",
      "chr18 39910126 39920126 51 U 10000 contig no na\n",
      "\n",
      "# CTG-0756\n",
      "['chr18', 39326301, 39981535, 'CTG-0756', 'fwd', 668698, 16254, 0, 'CTG-0835']\n",
      "....\n",
      "chr18 0 39920127 40588825 1 668698 +\n",
      "\n",
      "# CTG-0835\n",
      "['chr18', 39965281, 40427697, 'CTG-0835', 'fwd', 450689, -2585, 0, 'CTG-2129']\n",
      "....\n",
      "chr18 40588826 41055769 53 D CTG-0835 16255 450689 +\n",
      "chr18 41055770 41065770 54 U 10000 contig no na\n",
      "\n",
      "# CTG-2129\n",
      "['chr18', 40430282, 40438128, 'CTG-2129', 'rc', 7821, -91411, 0, 'CTG-0720']\n",
      "....\n",
      "chr18 41065771 41073592 55 D CTG-2129 1 7821 -\n",
      "chr18 41073593 41083593 56 U 10000 contig no na\n",
      "\n",
      "# CTG-0720\n",
      "['chr18', 40529539, 41359396, 'CTG-0720', 'fwd', 738619, 17872, 0, 'CTG-1048']\n",
      "....\n",
      "chr18 0 41083594 41822213 1 738619 +\n",
      "\n",
      "# CTG-1048\n",
      "['chr18', 41341524, 41503975, 'CTG-1048', 'fwd', 164381, 10209, 0, 'CTG-0817']\n",
      "....\n",
      "chr18 17872 41822214 42004467 17873 164381 +\n",
      "\n",
      "# CTG-0817\n",
      "['chr18', 41493766, 41997929, 'CTG-0817', 'rc', 504439, -6219, 0, 'CTG-0272']\n",
      "....\n",
      "chr18 42004468 42519116 59 D CTG-0817 1 494230 -\n",
      "chr18 42519117 42529117 60 U 10000 contig no na\n",
      "\n",
      "# CTG-0272\n",
      "['chr18', 42004148, 45129980, 'CTG-0272', 'fwd', 3124127, -15243, 0, 'CTG-0989']\n",
      "....\n",
      "chr18 42529118 45653245 61 D CTG-0272 1 3124127 +\n",
      "chr18 45653246 45663246 62 U 10000 contig no na\n",
      "\n",
      "# CTG-0989\n",
      "['chr18', 45145223, 45362298, 'CTG-0989', 'fwd', 216468, -5494, 0, 'CTG-0213']\n",
      "....\n",
      "chr18 45663247 45879715 63 D CTG-0989 1 216468 +\n",
      "chr18 45879716 45889716 64 U 10000 contig no na\n",
      "\n",
      "# CTG-0213\n",
      "['chr18', 45367792, 49109540, 'CTG-0213', 'fwd', 3736794, 25995, 0, 'CTG-1240']\n",
      "....\n",
      "chr18 0 45889717 49626511 1 3736794 +\n",
      "\n",
      "# CTG-1240\n",
      "['chr18', 49083545, 49130813, 'CTG-1240', 'rc', 47251, 34303, 0, 'CTG-0854']\n",
      "....\n",
      "chr18 25995 49626512 49699758 1 21256 -\n",
      "\n",
      "# CTG-0854\n",
      "['chr18', 49096510, 49548067, 'CTG-0854', 'fwd', 454291, -4039, 0, 'CTG-1000']\n",
      "....\n",
      "chr18 49699759 50188353 67 D CTG-0854 34304 454291 +\n",
      "chr18 50188354 50198354 68 U 10000 contig no na\n",
      "\n",
      "# CTG-1000\n",
      "['chr18', 49552106, 49746838, 'CTG-1000', 'fwd', 194595, -1, 0, 'CTG-0282']\n",
      "....\n",
      "chr18 50198355 50392950 69 D CTG-1000 1 194595 +\n",
      "chr18 50392951 50402951 70 U 10000 contig no na\n",
      "\n",
      "# CTG-0282\n",
      "['chr18', 49746839, 52779109, 'CTG-0282', 'fwd', 3034721, 10361, 0, 'CTG-0620']\n",
      "....\n",
      "chr18 0 50402952 53437673 1 3034721 +\n",
      "\n",
      "# CTG-0620\n",
      "['chr18', 52768748, 53825339, 'CTG-0620', 'rc', 1057651, -9050, 0, 'CTG-0459']\n",
      "....\n",
      "chr18 53437674 54505686 72 D CTG-0620 1 1047290 -\n",
      "chr18 54505687 54515687 73 U 10000 contig no na\n"
     ]
    }
   ],
   "source": [
    "#Defining AGP outfile\n",
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'w')\n",
    "\n",
    "#Now putting them in the coordinates of the Zoey genome!! :) \n",
    "count, agp = 0, [] \n",
    "print('agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction')\n",
    "\n",
    "for i in posDict:\n",
    "    count += 1\n",
    "    agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig = posDict[i][0:9]\n",
    "    print('\\n#',contigID)\n",
    "    print(posDict[i])\n",
    "    print('....')\n",
    "    #print(count,agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)    \n",
    "    \n",
    "    #Processing first contig in dataset -OR- on a new chromosome\n",
    "    if count == 1 or agp_chrom != agp_prev_chrom: \n",
    "        count = 1\n",
    "        info = process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)\n",
    "        agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction, offset = info[0:10]\n",
    "        agp.append([agp_prev_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "        print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "    \n",
    "    #Process next contig (not first contig on chrom)\n",
    "    else:\n",
    "        if overlap > 0: #if contigs overlap --> NO GAP!\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            if offset > 0:\n",
    "                #Add contig to the left of the gap\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "                print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "                #Add introduced gap from unaligned sequences from contig-contig junction BLAT\n",
    "                count += 1\n",
    "                agp.append([agp_chrom, agp_end+1, agp_end+501, count, 'U', '500','BLAT_gap','no','na'])\n",
    "                print(agp_chrom, agp_end+1, agp_end+501, count, 'U', '500','BLAT_gap','no','na')\n",
    "            else:\n",
    "                agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])         \n",
    "                print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "        else: #if contigs do not overlap --> GAP!\n",
    "            #process contig to the left of the gap\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction,offset = info[0:10]\n",
    "            agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction])\n",
    "            print(agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction)\n",
    "            \n",
    "            #processing gap\n",
    "            count += 1\n",
    "            gap_info = process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap = gap_info[0:7]\n",
    "            agp.append([agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na'])\n",
    "            print(agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na')\n",
    "    #if count > 15:\n",
    "    #    break\n",
    "\n",
    "#Write AGP header lines\n",
    "write_AGP_header(agpFile)\n",
    "\n",
    "#Write out to agp File the information for contigs/gaps\n",
    "for i in range(0,len(agp)):\n",
    "    agpFile.write(\"\\t\".join(map(str,agp[i])) + '\\n')\n",
    "agpFile.close()\n",
    "logFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###################################################################################################\n",
    "def extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID + '/' + contigID + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID,extract_coord1,extract_coord2,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "###################################################################################################    \n",
    "def reverse_comp_contig_fasta(wkDir):\n",
    "    cmd = 'fastarevcomp %stemp/contig1.fa > %stemp/contig1.fa.rc' % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    cmd = 'mv %stemp/contig1.fa.rc %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True) \n",
    "###################################################################################################    \n",
    "def read_in_fasta(wkDir):    \n",
    "    inFile = open(wkDir + 'temp/seq.fa', 'r')\n",
    "    for line in inFile:\n",
    "        line=line.rstrip()\n",
    "        if '>' in line:\n",
    "            continue\n",
    "        else:\n",
    "            seq.append(line)\n",
    "    inFile.close()\n",
    "    return seq\n",
    "\n",
    "#Reformat the fasta file\n",
    "agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1.fa', 'r')\n",
    "tmpFile = open(wkDir + 'results/tmp.fa', 'w')\n",
    "\n",
    "###################################################################################################    \n",
    "def reformat_fasta_file(wkDir):\n",
    "    agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1.fa', 'r')\n",
    "    tmpFile = open(wkDir + 'results/tmp.fa', 'w')\n",
    "\n",
    "    for line in agp_Fasta:\n",
    "        if '>' in line:\n",
    "            tmpFile.write(line)#Writes chrom ID\n",
    "            continue\n",
    "        seq=line.rstrip()\n",
    "\n",
    "        start = 0\n",
    "        fasta_length = float(len(seq))\n",
    "        fasta_line_length = 80\n",
    "        Max = int(fasta_length/fasta_line_length)\n",
    "\n",
    "        for i in range(0,Max+1):\n",
    "            Line = seq[start:start+fasta_line_length]\n",
    "            start = start + fasta_line_length\n",
    "            tmpFile.write(Line + '\\n')\n",
    "            \"\"\"if i > 10:\n",
    "                break\"\"\"    \n",
    "\n",
    "    agp_Fasta.close()\n",
    "    tmpFile.close()\n",
    "\n",
    "    cmd = 'mv %sresults/tmp.fa %sresults/AGP_Zoey_Assembly_v1.fa'  % (wkDir,wkDir)\n",
    "    #print(cmd)\n",
    "    subprocess.call(cmd,shell=True) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished writing out the sequence for 73 contigs and gaps to Zoey fasta file\n"
     ]
    }
   ],
   "source": [
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'r')\n",
    "fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "agp_Fasta = open(wkDir + 'results/AGP_Zoey_Assembly_v1.fa2', 'w')\n",
    "\n",
    "contigID = ''\n",
    "count, prev_chrom = 0, ''\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line:\n",
    "        continue\n",
    "    count+= 1\n",
    "    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID = line[5]\n",
    "    \n",
    "    seq = []\n",
    "    \n",
    "    if 'CTG' in contigID:\n",
    "        chrom, contigID, extract_coord1, extract_coord2, orient = line[0],line[5],int(line[6]),int(line[7]), line[8]\n",
    "        #Extract fasta\n",
    "        extract_contig_fasta(fastaRoot,contigID,orient,extract_coord1,extract_coord2)\n",
    "    \n",
    "        #Reverse complement extracted FASTA if needed\n",
    "        if '-' == orient:\n",
    "            reverse_comp_contig_fasta(wkDir)\n",
    "        else:\n",
    "            cmd = 'mv %stemp/contig1.fa %stemp/seq.fa'  % (wkDir,wkDir)\n",
    "            #print(cmd)\n",
    "            subprocess.call(cmd,shell=True) \n",
    "        \n",
    "        seq = read_in_fasta(wkDir)\n",
    "    else: #These are gaps   \n",
    "        gap_length = int(line[5])\n",
    "        seq = 'N' * gap_length\n",
    "    \n",
    "    #Writes '>' chromosome to the fasta file\n",
    "    if count == 1 or prev_chrom != chrom:\n",
    "        if count > 1:\n",
    "            agp_Fasta.write('\\n')\n",
    "        agp_Fasta.write('>' + chrom + '\\n')\n",
    "    \n",
    "    for i in range(0,len(seq)):\n",
    "        Seq = seq[i].upper()#.upper()print(seq[i])\n",
    "        agp_Fasta.write(Seq)\n",
    "        #agp_Fasta.write(seq[i])\n",
    "\n",
    "    prev_chrom = chrom #reset prev_chrom identity\n",
    "\n",
    "\n",
    "print('Finished writing out the sequence for %i contigs and gaps to Zoey fasta file' % count)\n",
    "agpFile.close()\n",
    "agp_Fasta.close()\n",
    "\n",
    "\n",
    "#Reformat the fasta file to only have certain # of nucleotides per line\n",
    "reformat_fasta_file(wkDir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 69,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Have generated the golden path file. Below, I'm calculating statistics on the AGP. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "##Counts\n",
      "49 contigs are in the Golden Path for Zoey\n",
      "24 gaps remain in the Golden Path for Zoey\n",
      "Of these 24 gaps, 1 gap(s) correspond(s) to BLAT gaps (unaligned sequences where gaps were introduced)\n",
      "\n",
      "##Statistics\n",
      "Average continuity (bp):  2261880.79\n"
     ]
    }
   ],
   "source": [
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'r')\n",
    "\n",
    "contig_count,gap_count,line_count,blat_gap_count = 0,0,0,0\n",
    "prev_Type,start_stop_list,contig_start = '',[],1\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line: #skips header\n",
    "        continue    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    line_count += 1\n",
    "    chrom,start,end,num,Type = line[0], int(line[1]),int(line[2]),int(line[3]),line[4]\n",
    "    if 'D' in Type:\n",
    "        #this is a contig\n",
    "        contig_count += 1\n",
    "        if 'U' in prev_Type:\n",
    "            contig_start = start\n",
    "    if 'U' in Type:\n",
    "        #this is a gap\n",
    "        gap_count += 1\n",
    "        if 'BLAT' in line[6]:\n",
    "            blat_gap_count+=1\n",
    "        #print(start,end)\n",
    "        if 'D' in prev_Type:\n",
    "            contig_end = start - 1\n",
    "            start_stop_list.append([contig_start,contig_end])\n",
    "    prev_Type = Type\n",
    "    \n",
    "print('##Counts\\n%i contigs are in the Golden Path for Zoey' % contig_count)\n",
    "print('%i gaps remain in the Golden Path for Zoey' % gap_count)\n",
    "print('Of these %i gaps, %i gap(s) correspond(s) to BLAT gaps (unaligned sequences where gaps were introduced)' %(gap_count,blat_gap_count))\n",
    "\n",
    "continuous_contig_array = []\n",
    "for i in range(0,len(start_stop_list)):\n",
    "    start,end = int(start_stop_list[i][0]),int(start_stop_list[i][1])\n",
    "    length = end-start\n",
    "    continuous_contig_array.append(length)\n",
    "\n",
    "Average_continuity = format(np.mean(continuous_contig_array), '.2f') \n",
    "print('\\n##Statistics\\nAverage continuity (bp): ', Average_continuity)  \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'chr5': 88915250, 'chr6': 77573801, 'chr16': 59632846, 'chr34': 42124431, 'chr14': 60966679, 'chr33': 31377067, 'chr4': 88276631, 'chr32': 38810281, 'chr31': 39895921, 'chr13': 63241923, 'chr28': 41182112, 'chr38': 23914537, 'chr36': 30810995, 'chr12': 72498081, 'chr10': 69331447, 'chr21': 50858623, 'chr22': 61439934, 'chrM': 16727, 'chr24': 47698779, 'chr25': 51628933, 'chrX': 123869142, 'chr3': 91889043, 'chr27': 45876710, 'chr17': 64289059, 'chr2': 85426708, 'chr35': 26524999, 'chr8': 74330416, 'chr30': 40214260, 'chr18': 55844845, 'chr19': 53741614, 'chr29': 41845238, 'chr11': 74389097, 'chr23': 52294480, 'chr15': 64190966, 'chr37': 30902991, 'chr7': 80974532, 'chr9': 61074082, 'chr20': 58134056, 'chr26': 38964690, 'chr1': 122678785}\n"
     ]
    }
   ],
   "source": [
    "chrom_length_file = open(wkDir + 'input/cyto_band.bed','r')\n",
    "chrom_lengths = {}\n",
    "\n",
    "for line in chrom_length_file:\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,length = line[0],int(line[2])\n",
    "    if 'chrUn' in chrom:\n",
    "        continue\n",
    "    chrom_lengths[chrom] = length\n",
    "    \n",
    "print (chrom_lengths)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 19553 gaps in the canFam3 assembly (not including chrUn)\n",
      "Average continuity (bp):  118532.19\n"
     ]
    }
   ],
   "source": [
    "gapFile = open(wkDir + 'input/gaps.bed.sorted','r')\n",
    "#to keep track of the previous call\n",
    "prev_chrom, prev_start, prev_end = '', '', ''\n",
    "#save continuous stretches to array\n",
    "cont_pos_list, per_chrom, count =  [], {}, 0 \n",
    "\n",
    "\n",
    "for line in gapFile:\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,start,end=line[0],int(line[1]),int(line[2])\n",
    "    if 'chrUn' in chrom:\n",
    "        continue\n",
    "    #FOR TESTING\n",
    "    \"\"\"if 'chr18' not in chrom:\n",
    "        continue\"\"\"\n",
    "    count+=1\n",
    "\n",
    "    #If first line in file\n",
    "    if count == 1:\n",
    "        per_chrom[chrom] = []\n",
    "        if start == 0: #gap is at the beginning of the chromosome\n",
    "            #Save for next\n",
    "            cont_start = end\n",
    "            prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "            continue\n",
    "        #first continuous stretch on chromosome\n",
    "        cont_start = 0\n",
    "        cont_end = start - 1\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        \n",
    "        #next continuous stretch, define start\n",
    "        cont_start = end\n",
    "        #Save for next\n",
    "        prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "        continue\n",
    "    #If the next call is on a different chromosome, must save the previous gap's info\n",
    "    if chrom not in prev_chrom:\n",
    "        per_chrom[chrom] = []\n",
    "        #previous continuous stretch\n",
    "        cont_end = chrom_lengths[prev_chrom]\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        #first continuous stretch on new chromosome\n",
    "        cont_start = 0\n",
    "        cont_end = start - 1\n",
    "        #next adjacent stretch, define start\n",
    "        cont_start = end      \n",
    "    else:\n",
    "        cont_end = start - 1\n",
    "        cont_pos_list.append([count-1,cont_start,cont_end])\n",
    "        per_chrom[chrom].append([count-1,cont_start,cont_end])\n",
    "        #next adjacent strech, define start\n",
    "        cont_start = end\n",
    "        \n",
    "    prev_chrom, prev_start, prev_end = chrom, start, end\n",
    "    \n",
    "\n",
    "print('There are %i gaps in the canFam3 assembly (not including chrUn)' % count)\n",
    "gapFile.close()\n",
    "\n",
    "\n",
    "#Calculating the continuity statistics\n",
    "continuous_contig_array = []\n",
    "for i in range(0,len(cont_pos_list)):\n",
    "    start,end = int(cont_pos_list[i][1]),int(cont_pos_list[i][2])\n",
    "    length = end-start\n",
    "    continuous_contig_array.append(length)    \n",
    "Average_continuity = format(np.mean(continuous_contig_array), '.2f')\n",
    "print('Average continuity (bp): ', Average_continuity)  \n",
    "\n",
    "\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "# chr32\n",
      "190 gaps on chr32\n",
      "Average continuity (bp):  202505.99\n",
      "\n",
      "# chr26\n",
      "391 gaps on chr26\n",
      "Average continuity (bp):  99154.57\n",
      "\n",
      "# chr16\n",
      "442 gaps on chr16\n",
      "Average continuity (bp):  133897.94\n",
      "\n",
      "# chr34\n",
      "297 gaps on chr34\n",
      "Average continuity (bp):  140893.48\n",
      "\n",
      "# chr14\n",
      "345 gaps on chr14\n",
      "Average continuity (bp):  175115.68\n",
      "\n",
      "# chr33\n",
      "264 gaps on chr33\n",
      "Average continuity (bp):  118442.50\n",
      "\n",
      "# chr1\n",
      "1121 gaps on chr1\n",
      "Average continuity (bp):  108669.12\n",
      "\n",
      "# chrX\n",
      "1032 gaps on chrX\n",
      "Average continuity (bp):  119702.28\n",
      "\n",
      "# chr13\n",
      "488 gaps on chr13\n",
      "Average continuity (bp):  129005.97\n",
      "\n",
      "# chr9\n",
      "764 gaps on chr9\n",
      "Average continuity (bp):  79047.59\n",
      "\n",
      "# chr28\n",
      "395 gaps on chr28\n",
      "Average continuity (bp):  103803.70\n",
      "\n",
      "# chr7\n",
      "607 gaps on chr7\n",
      "Average continuity (bp):  132802.75\n",
      "\n",
      "# chr12\n",
      "482 gaps on chr12\n",
      "Average continuity (bp):  150218.18\n",
      "\n",
      "# chr10\n",
      "744 gaps on chr10\n",
      "Average continuity (bp):  92985.91\n",
      "\n",
      "# chr21\n",
      "374 gaps on chr21\n",
      "Average continuity (bp):  135844.89\n",
      "\n",
      "# chr22\n",
      "371 gaps on chr22\n",
      "Average continuity (bp):  165163.49\n",
      "\n",
      "# chr4\n",
      "622 gaps on chr4\n",
      "Average continuity (bp):  141570.59\n",
      "\n",
      "# chr25\n",
      "384 gaps on chr25\n",
      "Average continuity (bp):  133718.28\n",
      "\n",
      "# chr29\n",
      "264 gaps on chr29\n",
      "Average continuity (bp):  157243.09\n",
      "\n",
      "# chr3\n",
      "707 gaps on chr3\n",
      "Average continuity (bp):  129768.72\n",
      "\n",
      "# chr17\n",
      "513 gaps on chr17\n",
      "Average continuity (bp):  124429.61\n",
      "\n",
      "# chr2\n",
      "828 gaps on chr2\n",
      "Average continuity (bp):  102654.87\n",
      "\n",
      "# chr35\n",
      "211 gaps on chr35\n",
      "Average continuity (bp):  126727.09\n",
      "\n",
      "# chr8\n",
      "583 gaps on chr8\n",
      "Average continuity (bp):  126900.71\n",
      "\n",
      "# chr38\n",
      "308 gaps on chr38\n",
      "Average continuity (bp):  79114.00\n",
      "\n",
      "# chr18\n",
      "589 gaps on chr18\n",
      "Average continuity (bp):  94481.08\n",
      "\n",
      "# chr19\n",
      "304 gaps on chr19\n",
      "Average continuity (bp):  177014.63\n",
      "\n",
      "# chr30\n",
      "372 gaps on chr30\n",
      "Average continuity (bp):  107145.10\n",
      "\n",
      "# chr11\n",
      "533 gaps on chr11\n",
      "Average continuity (bp):  139018.32\n",
      "\n",
      "# chr23\n",
      "308 gaps on chr23\n",
      "Average continuity (bp):  169341.04\n",
      "\n",
      "# chr27\n",
      "393 gaps on chr27\n",
      "Average continuity (bp):  116399.34\n",
      "\n",
      "# chr37\n",
      "327 gaps on chr37\n",
      "Average continuity (bp):  92451.86\n",
      "\n",
      "# chr36\n",
      "254 gaps on chr36\n",
      "Average continuity (bp):  120487.31\n",
      "\n",
      "# chr24\n",
      "455 gaps on chr24\n",
      "Average continuity (bp):  104445.93\n",
      "\n",
      "# chr20\n",
      "687 gaps on chr20\n",
      "Average continuity (bp):  83932.28\n",
      "\n",
      "# chr5\n",
      "892 gaps on chr5\n",
      "Average continuity (bp):  98934.37\n",
      "\n",
      "# chr31\n",
      "288 gaps on chr31\n",
      "Average continuity (bp):  138703.69\n",
      "\n",
      "# chr6\n",
      "786 gaps on chr6\n",
      "Average continuity (bp):  98439.18\n",
      "\n",
      "# chr15\n",
      "637 gaps on chr15\n",
      "Average continuity (bp):  100647.08\n"
     ]
    }
   ],
   "source": [
    "#Continuity values per chromosome\n",
    "for i in per_chrom:\n",
    "    chrom = i\n",
    "    print('\\n#',chrom)\n",
    "    print('%i gaps on %s' % (len(per_chrom[chrom]),chrom))\n",
    "    continuous_contig_array = []\n",
    "    for j in per_chrom[chrom]:\n",
    "        start = int(j[1])\n",
    "        end = int(j[2])\n",
    "        length = end - start\n",
    "        continuous_contig_array.append(length)\n",
    "    Average_continuity = format(np.mean(continuous_contig_array), '.2f')\n",
    "    print('Average continuity (bp): ', Average_continuity)"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
