{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#2017-05-02\n",
    "#A. Pendleton\n",
    "#This script is used to make an AGP file for Zoey. Can be built upon as new forms\n",
    "#   of contigs become assembled and incorporated into the AGP.\n",
    "\n",
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "\n",
    "#import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n",
    "import scipy \n",
    "import matplotlib.patches as patches\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#This script is used to make an AGP file for Zoey. Can be built upon as new contigs become assembled and incorporated into the available set for assembly."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Current working directory is:\n",
      " /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/\n"
     ]
    }
   ],
   "source": [
    "wkDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/AGP/'\n",
    "print('Current working directory is:\\n', wkDir)\n",
    "\n",
    "logFile = open(wkDir + 'temp/iPythonNotebook_LogFile.txt','w')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Identified coordinates for 77 primary contigs\n"
     ]
    }
   ],
   "source": [
    "#####READING IN PRIMARY CONTIG COORDINATES FROM MUMMER ALIGNMENT#######\n",
    "#Primary contig alignments (no canu processing)\n",
    "primaryContigCoord = open(wkDir + 'input/primary.2017-04-10.txt', 'r')\n",
    "primaryContigCoordBed = open(wkDir + 'input/primary.2017-04-10.bed', 'w')\n",
    "\n",
    "contigDict,index = {}, 0\n",
    "\n",
    "for line in primaryContigCoord:\n",
    "    if line.startswith(\"#\") is True: #skips header line\n",
    "        continue\n",
    "    line = line.rstrip().split('\\t')\n",
    "    contigID, length, Dir, chrom, start, end = line[0],line[1],line[2],line[3],int(line[4]),int(line[5])    \n",
    "    if 'chr18' not in chrom: #FILTERING ONLY FOR CHROM 18 FOR NOW!!! REMOVE LATER \n",
    "        continue\n",
    "    index += 1\n",
    "    contigDict[index] = [chrom,start,end,contigID,Dir,length]\n",
    "    primaryContigCoordBed.write('%s\\t%i\\t%i\\t%s\\n' % (chrom,int(start-1),end,contigID))\n",
    "    #if index > 15: #FOR TESTING--- REMOVE LATER\n",
    "    #    break\n",
    "    \n",
    "primaryContigCoord.close()\n",
    "primaryContigCoordBed.close()\n",
    "print ('Identified coordinates for %i primary contigs' % len(contigDict))\n",
    "logFile.write('Identified coordinates for %i primary contigs\\n\\n' % len(contigDict))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Step 1. Read in coordinates from alignment file based on Mummer of primary contigs (per Jeff)\n",
    "Step 2. BLAT each contig against the contig proximal to that proximal to it"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_line(Dict,curr_index,i):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[curr_index][0],int(Dict[curr_index][1]),int(Dict[curr_index][2]), Dict[curr_index][3], Dict[curr_index][4], int(Dict[curr_index][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i+1][0],int(Dict[i+1][1]),int(Dict[i+1][2]), Dict[i+1][3], Dict[i+1][4], int(Dict[i+1][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def read_last_line(Dict):\n",
    "    #Contig 1\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1 = Dict[i-1][0],int(Dict[i-1][1]),int(Dict[i-1][2]), Dict[i-1][3], Dict[i-1][4], int(Dict[i-1][5])\n",
    "    #Contig 2\n",
    "    chrom2,start2,end2,contigID2,Dir2,length2 = Dict[i][0],int(Dict[i][1]),int(Dict[i][2]), Dict[i][3], Dict[i][4], int(Dict[i][5])\n",
    "    return chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2\n",
    "###################################################################################################\n",
    "def process_last_contig(contigDict,curr_index,i):\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_last_line(contigDict)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,i-1,contigID2,i))\n",
    "    logFile.write(('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,i-1,contigID2,i)))\n",
    "    if start1 == start2 or end1 == end2:\n",
    "        return\n",
    "    if end2 > end1:\n",
    "        curr_index+=1 \n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        inList.append(contigID2)\n",
    "        print('PASS - Last contig on chromosome overlap or is spaced correctly')\n",
    "        logFile.write('PASS - Last contig on chromosome overlap or is spaced correctly\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_first_contig(contigDict,curr_index,i):\n",
    "    curr_index+=1\n",
    "    #Always save first and contig in the dictionary/dataset\n",
    "    curated_contigDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "    curated_contigDict[i+1] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    inList.append(contigID2)\n",
    "    print('PASS - First call in dataset')\n",
    "    logFile.write('PASS - First call in dataset\\n')\n",
    "    return curr_index \n",
    "###################################################################################################\n",
    "def process_same_starts(contigDict,curr_index,i):\n",
    "    if start1 == start2 and end1 > end2:\n",
    "        print('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig1 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "    if start1 == start2 and end1 < end2: #overwrite previous position if contig 2 is longer than contig 1\n",
    "        curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "        print('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Share start, contig2 longer' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return\n",
    "###################################################################################################\n",
    "def process_same_ends(contigDict,curr_index,i):    \n",
    "    if start1 < start2 and end1 == end2:\n",
    "        print('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('Ends same, kept contig 1 only' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        return \n",
    "    if start2 < start1 and end1 == end2: \n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#CTG-1462 (1) -- CTG-0201 (2)\n",
      "PASS - First call in dataset\n",
      "\n",
      "#CTG-0201 (2) -- CTG-2401 (3)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0201\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', '3968428']\n",
      "['chr18', 28961, 33381, 'CTG-2401', 'fwd', '4387']\n",
      "\n",
      "#CTG-0201 (2) -- CTG-1986 (4)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0201\n",
      "['chr18', 28961, 33381, 'CTG-2401', 'fwd', '4387']\n",
      "['chr18', 1781533, 1791763, 'CTG-1986', 'rc', '10221']\n",
      "\n",
      "#CTG-0201 (2) -- CTG-1861 (5)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1861 (5) -- CTG-0743 (6)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0743 (6) -- CTG-0770 (7)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0770 (7) -- CTG-0177 (8)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0177 (8) -- CTG-0767 (9)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0767 (9) -- CTG-0615 (10)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0615 (10) -- CTG-1278 (11)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1278 (11) -- CTG-0400 (12)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0400 (12) -- CTG-0564 (13)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0564 (13) -- CTG-1619 (14)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0564\n",
      "['chr18', 13506715, 14852995, 'CTG-0564', 'rc', '1349347']\n",
      "['chr18', 13622684, 13641303, 'CTG-1619', 'rc', '18385']\n",
      "\n",
      "#CTG-0564 (13) -- CTG-0274 (15)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0274 (15) -- CTG-1531 (16)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0274\n",
      "['chr18', 14854286, 17956069, 'CTG-0274', 'rc', '3082702']\n",
      "['chr18', 15450107, 15471804, 'CTG-1531', 'rc', '21481']\n",
      "\n",
      "#CTG-0274 (15) -- CTG-2242 (17)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0274\n",
      "['chr18', 15450107, 15471804, 'CTG-1531', 'rc', '21481']\n",
      "['chr18', 16180032, 16186206, 'CTG-2242', 'fwd', '6227']\n",
      "\n",
      "#CTG-0274 (15) -- CTG-0878 (18)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0878 (18) -- CTG-1943 (19)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0878\n",
      "['chr18', 17965033, 18359454, 'CTG-0878', 'rc', '395029']\n",
      "['chr18', 17968013, 17978681, 'CTG-1943', 'fwd', '11140']\n",
      "\n",
      "#CTG-0878 (18) -- CTG-1134 (20)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1134 (20) -- CTG-1314 (21)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1314 (21) -- CTG-1776 (22)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1776 (22) -- CTG-1294 (23)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1294 (23) -- CTG-0810 (24)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0810 (24) -- CTG-1846 (25)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0810\n",
      "['chr18', 18588316, 19084754, 'CTG-0810', 'rc', '490363']\n",
      "['chr18', 18843238, 18856597, 'CTG-1846', 'fwd', '13283']\n",
      "\n",
      "#CTG-0810 (24) -- CTG-0953 (26)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0953 (26) -- CTG-0581 (27)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0581 (27) -- CTG-1471 (28)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0581\n",
      "['chr18', 19367948, 20648758, 'CTG-0581', 'fwd', '1280510']\n",
      "['chr18', 19378074, 19402333, 'CTG-1471', 'fwd', '24190']\n",
      "\n",
      "#CTG-0581 (27) -- CTG-0901 (29)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0901 (29) -- CTG-0483 (30)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0483 (30) -- CTG-0776 (31)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0776 (31) -- CTG-2368 (32)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0776\n",
      "['chr18', 22629484, 23228962, 'CTG-0776', 'rc', '597680']\n",
      "['chr18', 22649616, 22654209, 'CTG-2368', 'fwd', '4758']\n",
      "\n",
      "#CTG-0776 (31) -- CTG-0412 (33)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0412 (33) -- CTG-0630 (34)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0630 (34) -- CTG-1329 (35)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1329 (35) -- CTG-1342 (36)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-1329\n",
      "['chr18', 26307857, 26340773, 'CTG-1329', 'rc', '32757']\n",
      "['chr18', 26318097, 26328164, 'CTG-1342', 'fwd', '9905']\n",
      "\n",
      "#CTG-1329 (35) -- CTG-0458 (37)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0458 (37) -- CTG-1682 (38)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0458\n",
      "['chr18', 26349001, 28160518, 'CTG-0458', 'fwd', '1814553']\n",
      "['chr18', 27488589, 27505855, 'CTG-1682', 'fwd', '17221']\n",
      "\n",
      "#CTG-0458 (37) -- CTG-0416 (39)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0416 (39) -- CTG-1241 (40)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1241 (40) -- CTG-0907 (41)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0907 (41) -- CTG-0235 (42)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0235 (42) -- CTG-0352 (43)\n",
      "Ends same, kept contig 1 only\n",
      "['chr18', 30609502, 34128003, 'CTG-0235', 'fwd', '3508078']\n",
      "['chr18', 34115809, 34128003, 'CTG-0352', 'fwd', '11912']\n",
      "\n",
      "#CTG-0235 (42) -- CTG-0351 (44)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0351 (44) -- CTG-0725 (45)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0725 (45) -- CTG-0432 (46)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0432 (46) -- CTG-1491 (47)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0432\n",
      "['chr18', 37418531, 39290284, 'CTG-0432', 'fwd', '1950806']\n",
      "['chr18', 39099709, 39121533, 'CTG-1491', 'rc', '23352']\n",
      "\n",
      "#CTG-0432 (46) -- CTG-1297 (48)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1297 (48) -- CTG-0756 (49)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0756 (49) -- CTG-2303 (50)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0756\n",
      "['chr18', 39326301, 39981535, 'CTG-0756', 'fwd', '668698']\n",
      "['chr18', 39958497, 39963239, 'CTG-2303', 'rc', '5448']\n",
      "\n",
      "#CTG-0756 (49) -- CTG-0835 (51)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0835 (51) -- CTG-2129 (52)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-2129 (52) -- CTG-0720 (53)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0720 (53) -- CTG-1189 (54)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0720\n",
      "['chr18', 40529539, 41359396, 'CTG-0720', 'fwd', '738619']\n",
      "['chr18', 40771772, 40831157, 'CTG-1189', 'rc', '61834']\n",
      "\n",
      "#CTG-0720 (53) -- CTG-1048 (55)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1048 (55) -- CTG-0817 (56)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0817 (56) -- CTG-0816 (57)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0817\n",
      "['chr18', 41493766, 41997929, 'CTG-0817', 'rc', '504439']\n",
      "['chr18', 41502348, 41504115, 'CTG-0816', 'fwd', '4435']\n",
      "\n",
      "#CTG-0817 (56) -- CTG-0272 (58)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0272 (58) -- CTG-0989 (59)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0989 (59) -- CTG-0213 (60)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0213 (60) -- CTG-2170 (61)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0213\n",
      "['chr18', 45367792, 49109540, 'CTG-0213', 'fwd', '3736794']\n",
      "['chr18', 45972968, 45980117, 'CTG-2170', 'fwd', '7107']\n",
      "\n",
      "#CTG-0213 (60) -- CTG-1441 (62)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0213\n",
      "['chr18', 45972968, 45980117, 'CTG-2170', 'fwd', '7107']\n",
      "['chr18', 49083545, 49093953, 'CTG-1441', 'rc', '26456']\n",
      "\n",
      "#CTG-0213 (60) -- CTG-1240 (63)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1240 (63) -- CTG-0854 (64)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0854 (64) -- CTG-1000 (65)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-1000 (65) -- CTG-1006 (66)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-1000\n",
      "['chr18', 49552106, 49746838, 'CTG-1000', 'fwd', '194595']\n",
      "['chr18', 49736239, 49740270, 'CTG-1006', 'rc', '4020']\n",
      "\n",
      "#CTG-1000 (65) -- CTG-0282 (67)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0282 (67) -- CTG-1948 (68)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0282\n",
      "['chr18', 49746839, 52779109, 'CTG-0282', 'fwd', '3034721']\n",
      "['chr18', 52200427, 52211517, 'CTG-1948', 'rc', '10948']\n",
      "\n",
      "#CTG-0282 (67) -- CTG-0620 (69)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0620 (69) -- CTG-0619 (70)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0620\n",
      "['chr18', 52768748, 53825339, 'CTG-0620', 'rc', '1057651']\n",
      "['chr18', 52777903, 52779109, 'CTG-0619', 'fwd', '1203']\n",
      "\n",
      "#CTG-0620 (69) -- CTG-0459 (71)\n",
      "PASS - contigs overlap or are spaced correctly\n",
      "\n",
      "#CTG-0459 (71) -- CTG-1277 (72)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 53834389, 55739311, 'CTG-0459', 'rc', '1823704']\n",
      "['chr18', 54280562, 54313992, 'CTG-1277', 'fwd', '34690']\n",
      "\n",
      "#CTG-0459 (71) -- CTG-2416 (73)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 54280562, 54313992, 'CTG-1277', 'fwd', '34690']\n",
      "['chr18', 54454560, 54458649, 'CTG-2416', 'rc', '4031']\n",
      "\n",
      "#CTG-0459 (71) -- CTG-1173 (74)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 54454560, 54458649, 'CTG-2416', 'rc', '4031']\n",
      "['chr18', 55623887, 55702128, 'CTG-1173', 'rc', '80210']\n",
      "\n",
      "#CTG-0459 (71) -- CTG-2278 (75)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 55623887, 55702128, 'CTG-1173', 'rc', '80210']\n",
      "['chr18', 55658963, 55664717, 'CTG-2278', 'fwd', '5770']\n",
      "\n",
      "#CTG-0459 (71) -- CTG-1290 (76)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 55658963, 55664717, 'CTG-2278', 'fwd', '5770']\n",
      "['chr18', 55704253, 55738170, 'CTG-1290', 'rc', '36820']\n",
      "\n",
      "#CTG-0459 (71) -- CTG-1414 (77)\n",
      "FAIL -- Contig2 fully within contig1\n",
      "Keeping contig: CTG-0459\n",
      "['chr18', 55704253, 55738170, 'CTG-1290', 'rc', '36820']\n",
      "['chr18', 55714781, 55722996, 'CTG-1414', 'fwd', '9390']\n",
      "Identified CURATED coordinates for 50 primary contigs\n"
     ]
    }
   ],
   "source": [
    "curated_contigDict, curr_index, inList = {}, 1, []\n",
    "chrom1,chrom2 = '0', '0' #initializing values\n",
    "\n",
    "\n",
    "for key, value in contigDict.items(): \n",
    "    i = int(key)\n",
    "    #1. If last contig in dataset \n",
    "    if i == len(contigDict):\n",
    "        continue\n",
    "    \n",
    "    #2. If contigs are on different chromosomes:\n",
    "    if chrom1 != chrom2:\n",
    "        process_last_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(contigDict,curr_index,i)\n",
    "    print ('\\n#%s (%i) -- %s (%i)' % (contigID1,curr_index,contigID2,i+1))\n",
    "    logFile.write('\\n#%s (%i) -- %s (%i)\\n' % (contigID1,curr_index,contigID2,i+1))\n",
    "\n",
    "    #3. Automatically saves first contig in dataset\n",
    "    if curr_index == 1 and len(curated_contigDict) == 0:\n",
    "        curr_index = process_first_contig(contigDict,curr_index,i)\n",
    "        continue\n",
    "    \n",
    "    #4. If contigs have the same start site, choose the longest contig\n",
    "    if start1 == start2:\n",
    "        process_same_starts(contigDict,curr_index,i)\n",
    "        continue\n",
    "\n",
    "    #5. If contigs have same end coordinate, choose longest contig (i.e. contig #1 if sorted)\n",
    "    if end1 == end2: \n",
    "        process_same_ends(contigDict,curr_index,i)\n",
    "        continue\n",
    "        \n",
    "    #6. If contigs #1 and #2 have same start AND end, keep contig #1\n",
    "    if start1 == start2 and end1 == end2:\n",
    "        print('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Same start and end coordinate' + '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) + '\\n')\n",
    "        continue #because contig1 is already in the dictionary\n",
    "        \n",
    "    #7. If contig #2 is fully within contig #1 -- continue\n",
    "    if start1 < start2 and end1 > end2: # contig2 is fully within contig1\n",
    "        #curated_contigDict[curr_index] = [chrom1,start1,end1,contigID1,Dir1,length1]\n",
    "        print('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]))\n",
    "        logFile.write('FAIL -- Contig2 fully within contig1' + '\\nKeeping contig: ' + contigID1 +  '\\n' + str(contigDict[i]) + '\\n' + str(contigDict[i+1]) +'\\n')\n",
    "        continue\n",
    "    if start1 > start2 and end1 < end2:\n",
    "        print('ERROR - is this contig list sorted??!!')\n",
    "        logFile.write('ERROR - is this contig list sorted??!!\\n\\n\\n')\n",
    "        sys.exit(1)\n",
    "    \n",
    "    #8. If contig passes all these - then automatically saves\n",
    "    curr_index=i+1\n",
    "    #if contigID2 not in inList:\n",
    "    curated_contigDict[i] = [chrom2,start2,end2,contigID2,Dir2,length2]\n",
    "    print('PASS - contigs overlap or are spaced correctly')\n",
    "    logFile.write('PASS - contigs overlap or are spaced correctly\\n')\n",
    "\n",
    "print ('Identified CURATED coordinates for %i primary contigs' % len(curated_contigDict))\n",
    "logFile.write('\\n##Identified CURATED coordinates for %i primary contigs\\n\\n########\\n\\n' % len(curated_contigDict))\n",
    "\n",
    "\n",
    "#Re-naming dictionary keys\n",
    "count=0\n",
    "for keys in curated_contigDict:\n",
    "    count+=1\n",
    "    curated_contigDict[count]=curated_contigDict.pop(keys)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"print('Non-curated:')\\nfor i in contigDict:\\n    print(i,contigDict[i])\\nprint( '\\nCurated:')\\nfor i in curated_contigDict:\\n    print(i, curated_contigDict[i])\""
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"print('Non-curated:')\n",
    "for i in contigDict:\n",
    "    print(i,contigDict[i])\n",
    "print( '\\nCurated:')\n",
    "for i in curated_contigDict:\n",
    "    print(i, curated_contigDict[i])\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1):\n",
    "    chrom2 = chrom1\n",
    "    start2 = start1 - 10000\n",
    "    end2 = start1 + 10000\n",
    "    contigID2 = 'canFam3'\n",
    "    Dir2 = 'fwd'\n",
    "    length1 = end2-start2\n",
    "    \n",
    "    if start2 < 0:\n",
    "        start2 = 0\n",
    "    \n",
    "    #Determine overlap \n",
    "    determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    " \n",
    "    #Find coordinates to extract\n",
    "    find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    \n",
    "    #Extract FASTA\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/genomes/canFam3.1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "###################################################################################################\n",
    "def determine_contig_overlap(end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    #How much do the contigs overlap?\n",
    "    global overlap\n",
    "    overlap = end1 - start2\n",
    "    print('\\nOverlap = ', overlap)\n",
    "    #OVERLAPPING CONTIGS\n",
    "    if overlap > 0:\n",
    "        find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2)\n",
    "    #DIRECTLY ADJACENT CONTIGS\n",
    "    if overlap == 0:\n",
    "        print('Contigs are directly adjacent to one another')\n",
    "        logFile.write('Contigs are directly adjacent to one another\\n')\n",
    "        #WRITE FUNCTION FOR THESE\n",
    "    #CONTIGS WITH GAP BETWEEN THEM\n",
    "    if overlap < 0:\n",
    "        print('Gap between contigs')\n",
    "        logFile.write('Gap between contigs\\n')\n",
    "\n",
    "###################################################################################################\n",
    "def find_overlapping_coordinates(overlap,end1, contigID1, Dir1, length1, end2, contigID2, Dir2, length2):\n",
    "    ###Determine coordinates to extract for BLAT\n",
    "    global extract_coord1\n",
    "    global extract_coord2\n",
    "    #Contig #1\n",
    "    if 'fwd' in Dir1:\n",
    "        extract_coord1 = [length1 - overlap - 10000, length1]\n",
    "    else:\n",
    "        extract_coord1 = [0, overlap + 10000]\n",
    "    #Contig #2\n",
    "    if 'fwd' in Dir2:\n",
    "        extract_coord2 = [0, overlap + 10000] \n",
    "    else:\n",
    "        extract_coord2 = [length2 - overlap - 10000, length2]\n",
    "    \n",
    "    print ('Coordinates to extract for BLAT: ',extract_coord1,extract_coord2,'\\n')\n",
    "    logFile.write('Coordinates to extract for BLAT: ' + str(extract_coord1) + str(extract_coord2) + '\\n')\n",
    "\n",
    "    #safety check\n",
    "    for i in range(0,1): #CHECKS \n",
    "        #= if the region to extract extends beyond the length of the contig, if so then the coordinate changes to the length of contig\n",
    "        if extract_coord1[i] > length1:\n",
    "            extract_coord1[i] = length1\n",
    "        if extract_coord2[i] > length2:\n",
    "            extract_coord2[i] = length2\n",
    "        # If the region extended too far, and the value is negative -- then make starting extraction coordinate = 0\n",
    "        if extract_coord1[i] < 0: \n",
    "            extract_coord1[i] = 0\n",
    "        if extract_coord2[i] < 0:\n",
    "            extract_coord2[i] = 0\n",
    "            \n",
    "    if overlap < 0: #### SEND TO DIFFERENT FUNCTION LATER -- FOLLOW OVERLAPPING CONTIGS FOR NOW\n",
    "        print('Contigs do not overlap')\n",
    "        logFile.write('Contigs do not overlap\\n')\n",
    "    #Where to find fasta of contigs\n",
    "    fastaRoot = '/home/ampend/links/kidd-lab/jmkidd-projects/zoey/contig-assignment/kmer-matches/eval1/'\n",
    "    extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1, contigID2,Dir2,extract_coord2)\n",
    "    \n",
    "###################################################################################################\n",
    "def extract_fasta(fastaRoot,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    #Contig 1\n",
    "    fasta_path1 = fastaRoot + contigID1 + '/' + contigID1 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i  > %stemp/contig1.fa' % (fasta_path1,contigID1,extract_coord1[0],extract_coord1[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "    #Contig 2\n",
    "    fasta_path2 = fastaRoot + contigID2 + '/' + contigID2 + '.fa'\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %stemp/contig2.fa' % (fasta_path2,contigID2,extract_coord2[0],extract_coord2[1],wkDir)\n",
    "    logFile.write(cmd + '\\n')\n",
    "    subprocess.call(cmd,shell=True)\n",
    "    \n",
    "###################################################################################################\n",
    "def run_blat(wkDir):\n",
    "    blatcmd = 'blat %stemp/contig1.fa %stemp/contig2.fa %stemp/temp.blat' % (wkDir,wkDir,wkDir)\n",
    "    logFile.write(blatcmd + '\\n')\n",
    "    subprocess.call(blatcmd,shell=True)\n",
    "    \n",
    "    \n",
    "###################################################################################################\n",
    "def parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2):\n",
    "    inFile = open('%stemp/temp.blat' % (wkDir),'r')\n",
    "    lineCount, cleanAlignment, offset = 0, False, 0\n",
    "    for line in inFile:\n",
    "        line=line.rstrip().split('\\t')\n",
    "        lineCount+=1\n",
    "        if lineCount > 5:#skips the BLAT results headers\n",
    "            score,strand = int(line[0]),line[8]\n",
    "            percent_of_overlap = float(score)/overlap\n",
    "            if percent_of_overlap < float(0.75):\n",
    "                continue\n",
    "            print('\\n#Parsing BLAT results','Percent of overlap matched in BLAT: ',format(percent_of_overlap, '.3f'))\n",
    "            logFile.write('\\n#Parsing BLAT results\\n' + 'Percent of overlap matched in BLAT: ' + format(percent_of_overlap, '.3f') + ' \\n')\n",
    "            #CHECKS TO MAKE SURE WE DID THIS RIGHT - correct end vs correct end\n",
    "            #if '+' not in strand:\n",
    "            #    print('ERROR: Top hit not in proper orientation.... SKIPPING -- PLEASE CHECK')\n",
    "            #    return\n",
    "            #parse contig #1 (left contig)\n",
    "            blat_length1, blat_start1, blat_end1 = int(line[14]),int(line[15]),int(line[16])\n",
    "            print(blat_length1, blat_start1, blat_end1)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length1, blat_start1, blat_end1))\n",
    "            #parse contig #2 (right contig)\n",
    "            blat_length2, blat_start2, blat_end2 = int(line[10]),int(line[11]),int(line[12])\n",
    "            print(blat_length2, blat_start2, blat_end2)\n",
    "            logFile.write('length = %i, start = %i, end = %i' % (blat_length2, blat_start2, blat_end2))\n",
    "            #Calculate offset  - to compensate for non-clean alignments (extension of left contig that does not overlap with adjacent contig)\n",
    "            offset = calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2)\n",
    "            return offset\n",
    "###################################################################################################\n",
    "def calculate_offset(overlap,blat_length1, blat_start1, blat_end1,Dir1,blat_length2, blat_start2, blat_end2,Dir2):\n",
    "    global offset\n",
    "    if 'rc' in Dir1:\n",
    "        offset = blat_start1 - 0\n",
    "    else:\n",
    "        offset = blat_length1 - blat_end1\n",
    "    if offset < 3:\n",
    "        offset = 0\n",
    "    print('\\nBLAT offset = ', offset)\n",
    "    logFile.write('\\nBLAT offset = ' + str(offset) + '\\n')\n",
    "    return offset\n",
    "\n",
    "###################################################################################################\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Overlap =  4849\n",
      "Coordinates to extract for BLAT:  [0, 14849] [3953579, 3968428] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.982\n",
      "14849 0 4806\n",
      "14850 10026 14849\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  1729\n",
      "Coordinates to extract for BLAT:  [0, 11729] [0, 11729] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.995\n",
      "11729 0 1720\n",
      "11729 0 1725\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  2637\n",
      "Coordinates to extract for BLAT:  [517, 13154] [0, 12637] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.997\n",
      "12638 10004 12638\n",
      "12637 0 2633\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  25890\n",
      "Coordinates to extract for BLAT:  [645585, 681475] [603506, 639396] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.993\n",
      "35891 10079 35891\n",
      "35891 10087 35890\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  21363\n",
      "Coordinates to extract for BLAT:  [0, 31363] [4332021, 4363384] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.994\n",
      "31363 0 21380\n",
      "31364 10063 31364\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  6495\n",
      "Coordinates to extract for BLAT:  [0, 16495] [630573, 647068] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.022\n",
      "16495 0 6645\n",
      "16496 9854 16496\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  2059\n",
      "Coordinates to extract for BLAT:  [0, 12059] [1066111, 1078170] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.000\n",
      "12059 0 2060\n",
      "12060 10000 12060\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  37187\n",
      "Coordinates to extract for BLAT:  [0, 47187] [0, 47187] \n",
      "\n",
      "\n",
      "Overlap =  40908\n",
      "Coordinates to extract for BLAT:  [-13235, 37673] [2139679, 2190587] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.777\n",
      "37673 5570 37673\n",
      "50909 9584 50909\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -4852\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -1291\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -8964\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -30419\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  4029\n",
      "Coordinates to extract for BLAT:  [71466, 85495] [0, 14029] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.994\n",
      "14030 10004 14030\n",
      "14029 0 4018\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  13268\n",
      "Coordinates to extract for BLAT:  [10845, 34113] [-8672, 14596] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.990\n",
      "23269 10055 23269\n",
      "14596 1351 14596\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  1360\n",
      "Coordinates to extract for BLAT:  [0, 11360] [0, 11360] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.989\n",
      "11360 0 1351\n",
      "11360 0 1351\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -44329\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -7170\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -1515\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  13620\n",
      "Coordinates to extract for BLAT:  [1256890, 1280510] [332198, 355818] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.020\n",
      "23621 9495 23621\n",
      "23621 9651 23621\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -7656\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  24646\n",
      "Coordinates to extract for BLAT:  [0, 34646] [563034, 597680] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.996\n",
      "34646 0 24635\n",
      "34647 10011 34647\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  14898\n",
      "Coordinates to extract for BLAT:  [0, 24898] [2081313, 2106211] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.002\n",
      "24898 0 15175\n",
      "24899 9911 24899\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -14267\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  20307\n",
      "Coordinates to extract for BLAT:  [1004391, 1034698] [2450, 32757] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.975\n",
      "30308 10199 30308\n",
      "30308 10150 30308\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -8228\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -2158\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  5188\n",
      "Coordinates to extract for BLAT:  [0, 15188] [32352, 47540] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.998\n",
      "15188 0 7033\n",
      "15189 9402 14628\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  11141\n",
      "Coordinates to extract for BLAT:  [0, 21141] [327863, 349004] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.001\n",
      "21141 5040 16212\n",
      "21142 1789 21142\n",
      "\n",
      "BLAT offset =  5040\n",
      "\n",
      "Overlap =  -18271\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  10472\n",
      "Coordinates to extract for BLAT:  [3487606, 3508078] [2513820, 2534292] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.988\n",
      "20473 10032 20473\n",
      "20473 10074 20473\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -13551\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -51643\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  13916\n",
      "Coordinates to extract for BLAT:  [1926890, 1950806] [12050, 35966] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.965\n",
      "23917 10401 23917\n",
      "23917 10282 23914\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -13658\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  16254\n",
      "Coordinates to extract for BLAT:  [642444, 668698] [0, 26254] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.986\n",
      "26255 10057 26255\n",
      "26254 0 16186\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -2585\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -91411\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  17872\n",
      "Coordinates to extract for BLAT:  [710747, 738619] [0, 27872] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.960\n",
      "27873 10449 27873\n",
      "27872 0 17505\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  10209\n",
      "Coordinates to extract for BLAT:  [144172, 164381] [484230, 504439] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  1.026\n",
      "20210 9692 20210\n",
      "20210 9653 20210\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -6219\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -15243\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -5494\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  25995\n",
      "Coordinates to extract for BLAT:  [3700799, 3736794] [11256, 47251] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.957\n",
      "35996 10383 35996\n",
      "35996 9569 35214\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  34303\n",
      "Coordinates to extract for BLAT:  [0, 44303] [0, 44303] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.951\n",
      "44303 0 33634\n",
      "44303 0 33832\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -4039\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  -1\n",
      "Gap between contigs\n",
      "\n",
      "Overlap =  10361\n",
      "Coordinates to extract for BLAT:  [3014360, 3034721] [1037290, 1057651] \n",
      "\n",
      "\n",
      "#Parsing BLAT results Percent of overlap matched in BLAT:  0.995\n",
      "20362 10034 20362\n",
      "20362 10015 20362\n",
      "\n",
      "BLAT offset =  0\n",
      "\n",
      "Overlap =  -9050\n",
      "Gap between contigs\n"
     ]
    }
   ],
   "source": [
    "#write_AGP_header(agpFile)\n",
    "posDict, offset_BLAT = {}, []\n",
    "\n",
    "for i in range(1,len(curated_contigDict)): \n",
    "    curr_index=i\n",
    "    #1. Reading in coordinates from contigList\n",
    "    chrom1,start1,end1,contigID1,Dir1,length1,chrom2,start2,end2,contigID2,Dir2,length2 = read_line(curated_contigDict, curr_index, i)\n",
    "    \n",
    "    #2. Checking contigs are on same chromosome\n",
    "    if chrom1 != chrom2:# or i == 1: #This is first contig on chromosome, need to process it first\n",
    "        #continue\n",
    "        process_first_contig_on_chrom(chrom1,start1,end1,contigID1,Dir1,length1) \n",
    "        continue\n",
    "    #print ('\\n#',contigID1, contigID2,'\\n',chrom1,start1,end1,contigID1,Dir1,length1,'\\n',chrom2,start2,end2,contigID2,Dir2,length2)\n",
    "    \n",
    "    #3. Determines the overlap/orientation of the two contigs\n",
    "    determine_contig_overlap(end1,contigID1,Dir1,length1,end2,contigID2,Dir2,length2)\n",
    "\n",
    "    #4. BLAT the properly oriented contig ends against one another\n",
    "    #     and parse the results\n",
    "    run_blat(wkDir)\n",
    "    parse_blat(wkDir,overlap,contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2)\n",
    "    if offset > 0:\n",
    "        offset_BLAT.append([contigID1,Dir1,extract_coord1,contigID2,Dir2,extract_coord2,overlap])\n",
    "    \n",
    "    #save contig1 information to dictionary\n",
    "    posDict[i] = [chrom1,start1,end1,contigID1,Dir1,length1,overlap,offset,contigID2]\n",
    "\n",
    "\n",
    "    #print('###################\\n')\n",
    "    #if i > 2:\n",
    "    #    break\n",
    "        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'for i in posDict:\\n    print (posDict[i][0:9])'"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\"\"\"for i in posDict:\n",
    "    print (posDict[i][0:9])\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig):\n",
    "    agp_prev_chrom, agp_prev_overlap = agp_chrom, 0\n",
    "    \n",
    "    if direction == 'fwd':\n",
    "        direction = '+' #changes notation of the direction\n",
    "    if direction == 'rc':\n",
    "        direction = '-'  #changes notation of the direction\n",
    "    agp_start, agp_end = 1, length\n",
    "    contig_start, contig_end = 1, length\n",
    "    \n",
    "    info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction]\n",
    "    return info\n",
    "########################################################################\n",
    "def process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    agp_prev_chrom, agp_prev_overlap = posDict[i-1][0], int(posDict[i-1][6])\n",
    "\n",
    "    if agp_prev_overlap < 0:\n",
    "        agp_prev_overlap = 0\n",
    "    \n",
    "    agp_start = agp_end + 1\n",
    "    agp_end = agp_start + length + agp_prev_overlap\n",
    "    \n",
    "    #Determine contig coordinates that align\n",
    "    if direction == 'fwd':\n",
    "        direction = '+' #changes notation of the direction\n",
    "        contig_start = 1 + agp_prev_overlap\n",
    "        contig_end = length\n",
    "    if direction == 'rc':\n",
    "        direction = '-'  #changes notation of the direction\n",
    "        contig_start = 1\n",
    "        contig_end = length - agp_prev_overlap\n",
    "        \n",
    "    info = [agp_prev_chrom,agp_prev_overlap,agp_start, agp_end,contig_start,contig_end,direction]\n",
    "    return info       \n",
    "        \n",
    "########################################################################  \n",
    "def process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end):\n",
    "    gap_length = 10000\n",
    "    agp_start = agp_end + 1\n",
    "    agp_end = agp_start + gap_length\n",
    "    if 'CTG' in contigID:\n",
    "        gap_type = 'contig'\n",
    "    else:#To change once we add more types of contigs in here\n",
    "        gap_type = 'OTHER'\n",
    "    agp_prev_overlap = 0\n",
    "    \n",
    "    info = [agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap]\n",
    "    return info  \n",
    "\n",
    "########################################################################  \n",
    "      \n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now putting the information in the curated contig set (contigs constituting the golden path) into the coordinates of Zoey, and adding gap positions, if present. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def write_AGP_header(agpFile): #alter for each assembly\n",
    "    agpFile.write('##agp-version 1.0\\n# ORGANISM: Canis lupus familiaris\\n# TAX_ID: 9615\\n')\n",
    "    agpFile.write('# ASSEMBLY NAME: Zoey_v1\\n# ASSEMBLY DATE: 19-April-2017\\n')\n",
    "    agpFile.write('# GENOME CENTER: University of Michigan - J.M. Kidd Lab\\n')\n",
    "    agpFile.write('# DESCRIPTION: AGP specifying the assembly of chromosome 18 from primary PacBio contigs from FALCON assembly\\n')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction\n",
      "\n",
      "# CTG-1462\n",
      "['chr18', 2619, 26992, 'CTG-1462', 'rc', 24712, 4849, 0, 'CTG-0201']\n",
      "....\n",
      "chr18 0 1 24712 1 24712 -\n",
      "\n",
      "# CTG-0201\n",
      "['chr18', 22143, 3966426, 'CTG-0201', 'rc', 3968428, 1729, 0, 'CTG-1861']\n",
      "....\n",
      "chr18 4849 24713 3997990 1 3963579 -\n",
      "\n",
      "# CTG-1861\n",
      "['chr18', 3964697, 3978047, 'CTG-1861', 'fwd', 13154, 2637, 0, 'CTG-0743']\n",
      "....\n",
      "chr18 1729 3997991 4012874 1730 13154 +\n",
      "\n",
      "# CTG-0743\n",
      "['chr18', 3975410, 4660269, 'CTG-0743', 'fwd', 681475, 25890, 0, 'CTG-0770']\n",
      "....\n",
      "chr18 2637 4012875 4696987 2638 681475 +\n",
      "\n",
      "# CTG-0770\n",
      "['chr18', 4634379, 5281622, 'CTG-0770', 'rc', 639396, 21363, 0, 'CTG-0177']\n",
      "....\n",
      "chr18 25890 4696988 5362274 1 613506 -\n",
      "\n",
      "# CTG-0177\n",
      "['chr18', 5260259, 9645181, 'CTG-0177', 'rc', 4363384, 6495, 0, 'CTG-0767']\n",
      "....\n",
      "chr18 21363 5362275 9747022 1 4342021 -\n",
      "\n",
      "# CTG-0767\n",
      "['chr18', 9638686, 10286737, 'CTG-0767', 'rc', 647068, 2059, 0, 'CTG-0615']\n",
      "....\n",
      "chr18 6495 9747023 10400586 1 640573 -\n",
      "\n",
      "# CTG-0615\n",
      "['chr18', 10284678, 11357644, 'CTG-0615', 'rc', 1078170, 37187, 0, 'CTG-1278']\n",
      "....\n",
      "chr18 2059 10400587 11480816 1 1076111 -\n",
      "\n",
      "# CTG-1278\n",
      "['chr18', 11320457, 11367174, 'CTG-1278', 'fwd', 37673, 40908, 0, 'CTG-0400']\n",
      "....\n",
      "chr18 37187 11480817 11555677 37188 37673 +\n",
      "\n",
      "# CTG-0400\n",
      "['chr18', 11326266, 13501863, 'CTG-0400', 'rc', 2190587, -4852, 0, 'CTG-0564']\n",
      "....\n",
      "\n",
      "# CTG-0564\n",
      "['chr18', 13506715, 14852995, 'CTG-0564', 'rc', 1349347, -1291, 0, 'CTG-0274']\n",
      "....\n",
      "\n",
      "# CTG-0274\n",
      "['chr18', 14854286, 17956069, 'CTG-0274', 'rc', 3082702, -8964, 0, 'CTG-0878']\n",
      "....\n",
      "\n",
      "# CTG-0878\n",
      "['chr18', 17965033, 18359454, 'CTG-0878', 'rc', 395029, -30419, 0, 'CTG-1134']\n",
      "....\n",
      "\n",
      "# CTG-1134\n",
      "['chr18', 18389873, 18474563, 'CTG-1134', 'fwd', 85495, 4029, 0, 'CTG-1314']\n",
      "....\n",
      "chr18 0 18654259 18739754 1 85495 +\n",
      "\n",
      "# CTG-1314\n",
      "['chr18', 18470534, 18504760, 'CTG-1314', 'fwd', 34113, 13268, 0, 'CTG-1776']\n",
      "....\n",
      "chr18 4029 18739755 18777897 4030 34113 +\n",
      "\n",
      "# CTG-1776\n",
      "['chr18', 18491492, 18506121, 'CTG-1776', 'rc', 14596, 1360, 0, 'CTG-1294']\n",
      "....\n",
      "chr18 13268 18777898 18805762 1 1328 -\n",
      "\n",
      "# CTG-1294\n",
      "['chr18', 18504761, 18543987, 'CTG-1294', 'fwd', 38857, -44329, 0, 'CTG-0810']\n",
      "....\n",
      "\n",
      "# CTG-0810\n",
      "['chr18', 18588316, 19084754, 'CTG-0810', 'rc', 490363, -7170, 0, 'CTG-0953']\n",
      "....\n",
      "\n",
      "# CTG-0953\n",
      "['chr18', 19091924, 19366433, 'CTG-0953', 'rc', 273581, -1515, 0, 'CTG-0581']\n",
      "....\n",
      "\n",
      "# CTG-0581\n",
      "['chr18', 19367948, 20648758, 'CTG-0581', 'fwd', 1280510, 13620, 0, 'CTG-0901']\n",
      "....\n",
      "chr18 0 19639930 20920440 1 1280510 +\n",
      "\n",
      "# CTG-0901\n",
      "['chr18', 20635138, 20993149, 'CTG-0901', 'rc', 355818, -7656, 0, 'CTG-0483']\n",
      "....\n",
      "\n",
      "# CTG-0483\n",
      "['chr18', 21000805, 22654130, 'CTG-0483', 'rc', 1655346, 24646, 0, 'CTG-0776']\n",
      "....\n",
      "chr18 0 21299881 22955227 1 1655346 -\n",
      "\n",
      "# CTG-0776\n",
      "['chr18', 22629484, 23228962, 'CTG-0776', 'rc', 597680, 14898, 0, 'CTG-0412']\n",
      "....\n",
      "chr18 24646 22955228 23577554 1 573034 -\n",
      "\n",
      "# CTG-0412\n",
      "['chr18', 23214064, 25317843, 'CTG-0412', 'rc', 2106211, -14267, 0, 'CTG-0630']\n",
      "....\n",
      "\n",
      "# CTG-0630\n",
      "['chr18', 25332110, 26328164, 'CTG-0630', 'fwd', 1034698, 20307, 0, 'CTG-1329']\n",
      "....\n",
      "chr18 0 25708666 26743364 1 1034698 +\n",
      "\n",
      "# CTG-1329\n",
      "['chr18', 26307857, 26340773, 'CTG-1329', 'rc', 32757, -8228, 0, 'CTG-0458']\n",
      "....\n",
      "\n",
      "# CTG-0458\n",
      "['chr18', 26349001, 28160518, 'CTG-0458', 'fwd', 1814553, -2158, 0, 'CTG-0416']\n",
      "....\n",
      "\n",
      "# CTG-0416\n",
      "['chr18', 28162676, 30223054, 'CTG-0416', 'rc', 2045472, 5188, 0, 'CTG-1241']\n",
      "....\n",
      "chr18 0 28630986 30676458 1 2045472 -\n",
      "\n",
      "# CTG-1241\n",
      "['chr18', 30217866, 30259696, 'CTG-1241', 'rc', 47540, 11141, 5040, 'CTG-0907']\n",
      "....\n",
      "chr18 5188 30676459 30729187 1 42352 -\n",
      "\n",
      "# CTG-0907\n",
      "['chr18', 30248555, 30591231, 'CTG-0907', 'rc', 349004, -18271, 5040, 'CTG-0235']\n",
      "....\n",
      "\n",
      "# CTG-0235\n",
      "['chr18', 30609502, 34128003, 'CTG-0235', 'fwd', 3508078, 10472, 0, 'CTG-0351']\n",
      "....\n",
      "chr18 0 31099335 34607413 1 3508078 +\n",
      "\n",
      "# CTG-0351\n",
      "['chr18', 34117531, 36616051, 'CTG-0351', 'rc', 2534292, -13551, 0, 'CTG-0725']\n",
      "....\n",
      "\n",
      "# CTG-0725\n",
      "['chr18', 36629602, 37366888, 'CTG-0725', 'fwd', 736753, -51643, 0, 'CTG-0432']\n",
      "....\n",
      "\n",
      "# CTG-0432\n",
      "['chr18', 37418531, 39290284, 'CTG-0432', 'fwd', 1950806, 13916, 0, 'CTG-1297']\n",
      "....\n",
      "chr18 0 37908935 39859741 1 1950806 +\n",
      "\n",
      "# CTG-1297\n",
      "['chr18', 39276368, 39312643, 'CTG-1297', 'rc', 35966, -13658, 0, 'CTG-0756']\n",
      "....\n",
      "\n",
      "# CTG-0756\n",
      "['chr18', 39326301, 39981535, 'CTG-0756', 'fwd', 668698, 16254, 0, 'CTG-0835']\n",
      "....\n",
      "chr18 0 39919626 40588324 1 668698 +\n",
      "\n",
      "# CTG-0835\n",
      "['chr18', 39965281, 40427697, 'CTG-0835', 'fwd', 450689, -2585, 0, 'CTG-2129']\n",
      "....\n",
      "\n",
      "# CTG-2129\n",
      "['chr18', 40430282, 40438128, 'CTG-2129', 'rc', 7821, -91411, 0, 'CTG-0720']\n",
      "....\n",
      "\n",
      "# CTG-0720\n",
      "['chr18', 40529539, 41359396, 'CTG-0720', 'fwd', 738619, 17872, 0, 'CTG-1048']\n",
      "....\n",
      "chr18 0 41083093 41821712 1 738619 +\n",
      "\n",
      "# CTG-1048\n",
      "['chr18', 41341524, 41503975, 'CTG-1048', 'fwd', 164381, 10209, 0, 'CTG-0817']\n",
      "....\n",
      "chr18 17872 41821713 42003966 17873 164381 +\n",
      "\n",
      "# CTG-0817\n",
      "['chr18', 41493766, 41997929, 'CTG-0817', 'rc', 504439, -6219, 0, 'CTG-0272']\n",
      "....\n",
      "\n",
      "# CTG-0272\n",
      "['chr18', 42004148, 45129980, 'CTG-0272', 'fwd', 3124127, -15243, 0, 'CTG-0989']\n",
      "....\n",
      "\n",
      "# CTG-0989\n",
      "['chr18', 45145223, 45362298, 'CTG-0989', 'fwd', 216468, -5494, 0, 'CTG-0213']\n",
      "....\n",
      "\n",
      "# CTG-0213\n",
      "['chr18', 45367792, 49109540, 'CTG-0213', 'fwd', 3736794, 25995, 0, 'CTG-1240']\n",
      "....\n",
      "chr18 0 45889216 49626010 1 3736794 +\n",
      "\n",
      "# CTG-1240\n",
      "['chr18', 49083545, 49130813, 'CTG-1240', 'rc', 47251, 34303, 0, 'CTG-0854']\n",
      "....\n",
      "chr18 25995 49626011 49699257 1 21256 -\n",
      "\n",
      "# CTG-0854\n",
      "['chr18', 49096510, 49548067, 'CTG-0854', 'fwd', 454291, -4039, 0, 'CTG-1000']\n",
      "....\n",
      "\n",
      "# CTG-1000\n",
      "['chr18', 49552106, 49746838, 'CTG-1000', 'fwd', 194595, -1, 0, 'CTG-0282']\n",
      "....\n",
      "\n",
      "# CTG-0282\n",
      "['chr18', 49746839, 52779109, 'CTG-0282', 'fwd', 3034721, 10361, 0, 'CTG-0620']\n",
      "....\n",
      "chr18 0 50402451 53437172 1 3034721 +\n",
      "\n",
      "# CTG-0620\n",
      "['chr18', 52768748, 53825339, 'CTG-0620', 'rc', 1057651, -9050, 0, 'CTG-0459']\n",
      "....\n"
     ]
    }
   ],
   "source": [
    "#Defining AGP outfile\n",
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'w')\n",
    "\n",
    "#Now putting them in the coordinates of the Zoey genome!! :) \n",
    "count, agp = 0, [] \n",
    "print('agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction')\n",
    "\n",
    "for i in posDict:\n",
    "    count += 1\n",
    "    agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig = posDict[i][0:9]\n",
    "    print('\\n#',contigID)\n",
    "    print(posDict[i])\n",
    "    print('....')\n",
    "    #print(count,agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)    \n",
    "    \n",
    "    #Processing first contig in dataset -OR- on a new chromosome\n",
    "    if count == 1 or agp_chrom != agp_prev_chrom: \n",
    "        count = 1\n",
    "        info = process_first_agp_contig(agp_chrom, start, end, contigID, direction, length, overlap, offset, pairedContig)\n",
    "        agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction = info[0:9]\n",
    "        agp.append([chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "        print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "    #Process next contig (not first contig on chrom)\n",
    "    else:\n",
    "        if overlap > 0: #if contigs overlap --> NO GAP!\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction = info[0:9]\n",
    "            agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "            print(agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction)\n",
    "        else: #if contigs do not overlap --> GAP!\n",
    "            #process contig to the left of the gap\n",
    "            info = process_next_agp_contig(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_prev_chrom,agp_prev_overlap,agp_start, agp_end, contig_start,contig_end,direction = info[0:9]\n",
    "            agp.append([agp_chrom, agp_start, agp_end, count, 'D', contigID, contig_start, contig_end, direction ])\n",
    "\n",
    "            #processing gap\n",
    "            count += 1\n",
    "            gap_info = process_agp_GAP(agp_chrom, direction, length, overlap, offset, pairedContig, agp_start, agp_end)\n",
    "            agp_chrom,agp_start,agp_end,gap_type,gap_length,agp_prev_overlap = gap_info[0:7]\n",
    "            agp.append([agp_chrom,agp_start,agp_end,count,'U',gap_length,gap_type,'no','na'])\n",
    "    #if count > 15:\n",
    "    #    break\n",
    "\n",
    "#Write AGP header lines\n",
    "write_AGP_header(agpFile)\n",
    "\n",
    "#Write out to agp File the information for contigs/gaps\n",
    "for i in range(0,len(agp)):\n",
    "    agpFile.write(\"\\t\".join(map(str,agp[i])) + '\\n')\n",
    "agpFile.close()\n",
    "logFile.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Have generated the golden path file. Below, I'm calculating statistics on the AGP. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "49 contigs are in the Golden Path for Zoey\n",
      "23 gaps remain in the Golden Path for Zoey\n",
      "Average continuity (bp):  2360223.48\n"
     ]
    }
   ],
   "source": [
    "agpFile = open(wkDir + 'results/AGP_Zoey_Assembly_v1.txt', 'r')\n",
    "\n",
    "contig_count,gap_count,line_count = 0,0,0\n",
    "prev_Type,start_stop_list,contig_start = '',[],1\n",
    "\n",
    "for line in agpFile:\n",
    "    if '#' in line: #skips header\n",
    "        continue    \n",
    "    line = line.rstrip().split('\\t')\n",
    "    line_count += 1\n",
    "    chrom,start,end,num,Type = line[0], int(line[1]),int(line[2]),int(line[3]),line[4]\n",
    "    if 'D' in Type:\n",
    "        #this is a contig\n",
    "        contig_count += 1\n",
    "        if 'U' in prev_Type:\n",
    "            contig_start = start\n",
    "    if 'U' in Type:\n",
    "        #this is a gap\n",
    "        gap_count += 1\n",
    "        #print(start,end)\n",
    "        if 'D' in prev_Type:\n",
    "            contig_end = start - 1\n",
    "            start_stop_list.append([contig_start,contig_end])\n",
    "    prev_Type = Type\n",
    "    \n",
    "print('%i contigs are in the Golden Path for Zoey' % contig_count)\n",
    "print('%i gaps remain in the Golden Path for Zoey' % gap_count)\n",
    "\n",
    "\n",
    "continuous_contig_array = []\n",
    "for i in range(0,len(start_stop_list)):\n",
    "    start,end = int(start_stop_list[i][0]),int(start_stop_list[i][1])\n",
    "    length = end-start\n",
    "    continuous_contig_array.append(length)\n",
    "\n",
    "Average_continuity = format(np.mean(continuous_contig_array), '.2f') \n",
    "print('Average continuity (bp): ', Average_continuity)  \n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
