{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import glob\n",
    "import gzip\n",
    "import fileinput\n",
    "import time\n",
    "import datetime\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_chrom_sizes():\n",
    "    chromSizefile = '/home/ampend/links/kidd-lab/www-mirror/www/track-hub/canFam3/canFam3.1-browser-chrom-sizes.fai'\n",
    "    print ('Reading in chromosome lengths from: ', chromSizefile)\n",
    "    chromSizeFile = open(chromSizefile, 'r')\n",
    "\n",
    "    chromSizeList,chromlist = {},[]\n",
    "    #I also need a list of chr and position for finding the index for windows \n",
    "    for line in chromSizeFile:\n",
    "        if \"chrUn\" in line: #ignore chromsome unknowns \n",
    "            continue\n",
    "        if \"chrM\" in line: #ignore mito\n",
    "            continue\n",
    "\n",
    "        line = line.rstrip()\n",
    "        line = line.split()\n",
    "\n",
    "        chromID = line[0] #chromosome ID\n",
    "        chromlist.append(chromID)\n",
    "        chromLength = int(line[1]) #length of chromosome\n",
    "        chromSizeList[chromID] = chromLength\n",
    "\n",
    "    return chromSizeList, chromlist\n",
    "#####################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading in chromosome lengths from:  /home/ampend/links/kidd-lab/www-mirror/www/track-hub/canFam3/canFam3.1-browser-chrom-sizes.fai\n"
     ]
    }
   ],
   "source": [
    "wkDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/'\n",
    "\n",
    "#Getting chromosome sizes\n",
    "chromSizeList,chromlist = get_chrom_sizes()\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.0.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_0.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.0.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_0.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.1.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_1.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.1.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_1.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.2.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_2.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.2.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_2.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.3.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_3.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.3.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_3.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.4.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_4.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.4.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_4.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.5.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_5.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.5.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_5.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.6.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_6.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.6.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_6.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.7.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_7.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.7.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_7.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.8.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_8.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.8.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_8.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_chr*.9.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/wolf_9.simulation.recode.frq.count\n",
      "cat /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_chr*.9.simulation.recode.frq.count > /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/input/dog_9.simulation.recode.frq.count\n"
     ]
    }
   ],
   "source": [
    "#Combining the VCF frq.count files into whole genome files for simulations 0-9\n",
    "for sim in range(0,10): #for simulations 0-9\n",
    "    #for wolves\n",
    "    cmd = 'cat %sinput/wolf_chr*.%i.simulation.recode.frq.count > %sinput/wolf_%i.simulation.recode.frq.count' % (wkDir,sim,wkDir,sim)\n",
    "    runCMD(cmd)\n",
    "    #for village dogs\n",
    "    cmd = 'cat %sinput/dog_chr*.%i.simulation.recode.frq.count > %sinput/dog_%i.simulation.recode.frq.count' % (wkDir,sim,wkDir,sim)\n",
    "    runCMD(cmd)\n",
    "print('DONE merging the dog and wolf frq.count files within simulation')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def get_chr_pos(infile):\n",
    "    chr_pos = []\n",
    "    for line in open(infile,'r'):\n",
    "        if line[0] == 'C':\n",
    "            continue \n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        if len(chromlist) == 0 or line[0] != chromlist[-1]:\n",
    "            chromlist.append(line[0])\n",
    "        chr_pos.append(line) #I dont want to append the whole line I just need chr and pos...\n",
    "    return chr_pos\n",
    "#####################################################################################################\n",
    "def process_inputlines(line):\n",
    "    line = line.rstrip() #removing extraneous whitespace characters\n",
    "    line = line.split() #delimiting \"columns\" in the file based on tabs\n",
    "    line[4] = float(line[4].split(':')[-1])\n",
    "    line[5] = float(line[5].split(':')[-1])\n",
    "    return line[4],line[5] \n",
    "#####################################################################################################\n",
    "#####################################################################\n",
    "# PER JEFF:\n",
    "# Calculates estimate of 2 pop Fst using Hudson's estimator\n",
    "# as described in Bhatia et al 2013\n",
    "# input is [a1_count,a2_count][a1_counts,a2_count]\n",
    "# returns (fst,numerator,denominator) for use in taking ratio of averages for multiple markers\n",
    "def fst_hudson_twopop(pop1,pop2):\n",
    "    n1 = pop1[0] + pop1[1]\n",
    "    n2 = pop2[0] + pop2[1]\n",
    "    p1 = float(pop1[0]) / float((pop1[0]+pop1[1]))  # allele freq in pop 1\n",
    "    p2 = float(pop2[0]) / float((pop2[0]+pop2[1]))  # allele freq in pop 2\n",
    "\n",
    "    num = (p1-p2)*(p1-p2) -  ((p1*(1.0-p1))/(n1-1)) -  ((p2*(1.0-p2))/(n2-1.0))\n",
    "    denom = p1*(1.0-p2) + p2*(1.0-p1)\n",
    "    f = float(num)/float(denom)\n",
    "    \n",
    "    return (f,num,denom)\n",
    "#####################################################################################################\n",
    "def calc_windows_fst_data(fstOutFile,chrom):\n",
    "    head_idx, tail_idx, last_line = 0, 0, 0\n",
    "    chrom = 'chr' + str(chrom)\n",
    "    #for chrom in chromlist:\n",
    "    window_start, window_end, windowCount, lastWindow = 0, 0, 0, False\n",
    "    for i in range(1,130000000,50000):  #figure out the head/tail index\n",
    "        window_start = i\n",
    "        window_end = i + 200000 - 1\n",
    "        if window_start > chromSizeList[chrom]:\n",
    "            break\n",
    "        if int(window_end) > int(chromSizeList[chrom]): #if the end of the window extends further than the length of the chromsome, then the window end is the length of the chromosome\n",
    "            window_end = int(chromSizeList[chrom])\n",
    "        while int(chr_pos[head_idx][1]) < i and chr_pos[head_idx][0] == chrom:\n",
    "            head_idx += 1\n",
    "        if head_idx >= len(chr_pos):\n",
    "            break\n",
    "        if tail_idx >= len(chr_pos):\n",
    "            break\n",
    "        while (int(chr_pos[tail_idx][1]) <= (i+200000) and chr_pos[tail_idx][0] == chrom):\n",
    "            tail_idx += 1\n",
    "            if tail_idx == len(chr_pos): \n",
    "                last_line = 1\n",
    "                break\n",
    "        #print tail_idx\n",
    "        tail_ind_new = tail_idx - 1\n",
    "        # Columns for output\n",
    "        window_chr = chr_pos[head_idx][0]\n",
    "        N_varients = tail_ind_new - head_idx\n",
    "        windowCount += 1 \n",
    "        N_varients_fixed,N_varients_tot,N_varients_forFst = 0,0,0 \n",
    "\n",
    "        if N_varients < 1: #This excludes windows with no variants in them. These will not go to calculating Fst\n",
    "            continue\n",
    "        #Writing out chrom positions of windows with variant counts for the FST outfile\n",
    "        fList, numList, denomList = [], [], []\n",
    "\n",
    "        #Here is where you'll read in the new subroutine from Jeff (at the top)\n",
    "        for k in range(head_idx, tail_ind_new+1): #This is going to calculate the F,num,denom at each site in the window\n",
    "            N_varients_tot += 1\n",
    "            if pop1[k][0] == 0 and pop2[k][0] == 0 or pop1[k][1] == 0 and pop2[k][1] == 0:\n",
    "                print ('pop1[k][0] and pop2[k][0]', pop1[k][0], pop2[k][0])\n",
    "                print ('pop1[k][1] and pop2[k][1]', pop1[k][1], pop2[k][1])\n",
    "                N_varients_fixed+=1\n",
    "                continue\n",
    "            temp = fst_hudson_twopop(pop1[k],pop2[k]) #This function writes out three values: 1) f, 2) numerator, and 3) denominator\n",
    "            f = temp[0] #defining variables for comprehension, not utility, here that were output from the fst_hudson_twopop function\n",
    "            num = temp[1]\n",
    "            denom = temp[2]\n",
    "            #N_varients_tot += 1\n",
    "            #counting fixed SNPs:(if fst = 0, then it's a fixed snp in both dogs and wolves)\n",
    "            #if f == 0:\n",
    "                #N_varients_fixed_OLD +=1\n",
    "                #continue #This is throwing out all sites that are fixed in the wolves AND dogs at the same site. These aren't informative and shouldn't be added to the lists of f/num/denom\n",
    "            N_varients_forFst += 1 \n",
    "            fList.append(f) #now we are saving those to a list here that can be used to calculate the average later\n",
    "            numList.append(num)\n",
    "            denomList.append(denom)    \n",
    "\n",
    "        ########################\n",
    "        #Only until all the values for f,num,and denom are calculated can you find the average of ratios, etc.\n",
    "        #Here is where you would calculate the AofR and RofA from the lists we generated above (f,num,denom)\n",
    "        #AofR = averaging all of the Fst ratios \n",
    "        if len(fList) == 0:\n",
    "            continue\n",
    "        AofR = float(sum(fList))/len(fList)\n",
    "\n",
    "        #RofA = averaging all the numerators and dividing it by the average of denominators:\n",
    "        meanNum = float(sum(numList))/len(numList) #check where to put the float\n",
    "        meanDenom = float(sum(denomList))/len(denomList)\n",
    "        RofA = float(meanNum)/meanDenom\n",
    "        ########################\n",
    "        fstOutFile.write('%s\\t%s\\t%s\\t%s_%s\\t' % (window_chr, window_start, window_end, window_chr, window_start)) #change this to the new outfile for FST data\n",
    "        #here is where you want to write out the windowID, RofA, AofR\n",
    "        fstOutFile.write('%i\\t%i\\t%i\\t%f\\t%f\\n' % (N_varients_tot, N_varients_forFst, N_varients_fixed, RofA, AofR)) #(remember that you can use %.3f (example) to limit the decimal places for these if you want)\n",
    "\n",
    "        if last_line == 1:\n",
    "            break\n",
    "        if chr_pos[tail_idx][0] != chrom:\n",
    "            head_idx = tail_idx\n",
    "            break\n",
    "\n",
    "    #This prints out the chromosome stats when everything is done running and this corresponds to stats for the last chromosome only\n",
    "    #print ('Chrom: ', chrom)\n",
    "    #print ('Window count', windowCount)\n",
    "    #break\n",
    "    fstOutFile.close()\n",
    "#####################################################################################################\n",
    "def filter_windows(fstOutfile):\n",
    "    inFile = open(fstOutfile, 'r')\n",
    "    outFile = open(fstOutfile + '_filtered_windows', 'w')\n",
    "    print('Writing filtered windows (at least 10 SNPs) to output file: ', fstOutfile + '_filtered_windows' )\n",
    "    count = 0 #number of windows with more than 10 variants\n",
    "    for line in inFile: #processing the outfile generated from fst calculations\n",
    "        LINE=line.rstrip().split('\\t')\n",
    "        if 'CHROM' in LINE[0]: #skip header\n",
    "            outFile.write(line) #writes header to the outfile\n",
    "            continue\n",
    "        var_count = int(LINE[4]) #counts the variants\n",
    "        if var_count >= 10:\n",
    "            count+=1\n",
    "            outFile.write(line)    \n",
    "    print('%i windows with at least ten variants' % count)\n",
    "    inFile.close()\n",
    "    outFile.close()\n",
    "#####################################################################################################\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "START=  2017-08-16 15:03:52\n",
      "Processing Simulation 0\n",
      "Processing Simulation 1\n",
      "Processing Simulation 2\n",
      "Processing Simulation 3\n",
      "Chr1\n",
      "Chrom:  chr1\n",
      "Window count 2451\n",
      "Chr2\n",
      "Chrom:  chr2\n",
      "Window count 1706\n",
      "Chr3\n",
      "Chrom:  chr3\n",
      "Window count 1835\n",
      "Chr4\n",
      "Chrom:  chr4\n",
      "Window count 1763\n",
      "Chr5\n",
      "Chrom:  chr5\n",
      "Window count 1776\n",
      "Chr6\n",
      "Chrom:  chr6\n",
      "Window count 1549\n",
      "Chr7\n",
      "Chrom:  chr7\n",
      "Window count 1617\n",
      "Chr8\n",
      "Chrom:  chr8\n",
      "Window count 1484\n",
      "Chr9\n",
      "Chrom:  chr9\n",
      "Window count 1219\n",
      "Chr10\n",
      "Chrom:  chr10\n",
      "Window count 1384\n",
      "Chr11\n",
      "Chrom:  chr11\n",
      "Window count 1485\n",
      "Chr12\n",
      "Chrom:  chr12\n",
      "Window count 1447\n",
      "Chr13\n",
      "Chrom:  chr13\n",
      "Window count 1262\n",
      "Chr14\n",
      "Chrom:  chr14\n",
      "Window count 1217\n",
      "Chr15\n",
      "Chrom:  chr15\n",
      "Window count 1281\n",
      "Chr16\n",
      "Chrom:  chr16\n",
      "Window count 1190\n",
      "Chr17\n",
      "Chrom:  chr17\n",
      "Window count 1283\n",
      "Chr18\n",
      "Chrom:  chr18\n",
      "Window count 1114\n",
      "Chr19\n",
      "Chrom:  chr19\n",
      "Window count 1072\n",
      "Chr20\n",
      "Chrom:  chr20\n",
      "Window count 1160\n",
      "Chr21\n",
      "Chrom:  chr21\n",
      "Window count 1015\n",
      "Chr22\n",
      "Chrom:  chr22\n",
      "Window count 1226\n",
      "Chr23\n",
      "Chrom:  chr23\n",
      "Window count 1043\n",
      "Chr24\n",
      "Chrom:  chr24\n",
      "Window count 951\n",
      "Chr25\n",
      "Chrom:  chr25\n",
      "Window count 1030\n",
      "Chr26\n",
      "Chrom:  chr26\n",
      "Window count 777\n",
      "Chr27\n",
      "Chrom:  chr27\n",
      "Window count 915\n",
      "Chr28\n",
      "Chrom:  chr28\n",
      "Window count 821\n",
      "Chr29\n",
      "Chrom:  chr29\n",
      "Window count 834\n",
      "Chr30\n",
      "Chrom:  chr30\n",
      "Window count 802\n",
      "Chr31\n",
      "Chrom:  chr31\n",
      "Window count 795\n",
      "Chr32\n",
      "Chrom:  chr32\n",
      "Window count 774\n",
      "Chr33\n",
      "Chrom:  chr33\n",
      "Window count 625\n",
      "Chr34\n",
      "Chrom:  chr34\n",
      "Window count 840\n",
      "Chr35\n",
      "Chrom:  chr35\n",
      "Window count 528\n",
      "Chr36\n",
      "Chrom:  chr36\n",
      "Window count 614\n",
      "Chr37\n",
      "Chrom:  chr37\n",
      "Window count 616\n",
      "Chr38\n",
      "Chrom:  chr38\n",
      "Window count 476\n",
      "Writing filtered windows (at least 10 SNPs) to output file:  /home/ampend/links/kidd-lab/ampend-projects/Angela/Simulations/results/Simulation.3_Hudson_Fst_200kbWindow_50kbSlide.txt_filtered_windows\n",
      "43977 windows with at least ten variants\n",
      "Processing Simulation 4\n",
      "Processing Simulation 5\n",
      "Processing Simulation 6\n",
      "Processing Simulation 7\n",
      "Processing Simulation 8\n",
      "Processing Simulation 9\n",
      "15:09:48\n"
     ]
    }
   ],
   "source": [
    "ts1 = time.time()\n",
    "st = datetime.datetime.fromtimestamp(ts1).strftime('%Y-%m-%d %H:%M:%S')\n",
    "print('START= ',st)\n",
    "\n",
    "for i in range(0,10):\n",
    "    print('Processing Simulation %i' % i)\n",
    "    if i != 3:\n",
    "        continue\n",
    "    for chrom in range(1,39):\n",
    "        print('Chr%i' % chrom)\n",
    "        \"\"\"dogCountFile = wkDir + 'input/' + 'dog_chr1.0.simulation.recode.frq.count'\n",
    "        wolfCountFile = wkDir + 'input/' + 'wolf_chr1.0.simulation.recode.frq.count'\"\"\"\n",
    "        #IN FILES\n",
    "        dogCountFile = wkDir + 'input/' + 'dog_chr%i.%i.simulation.recode.frq.count' % (chrom,i)\n",
    "        wolfCountFile = wkDir + 'input/' + 'wolf_chr%i.%i.simulation.recode.frq.count' % (chrom,i)\n",
    "        #OUT FILES\n",
    "        fstOutfile = wkDir + 'results/' + 'Simulation.chr%i.%i_Hudson_Fst_200kbWindow_50kbSlide.txt' % (chrom,i)#Write out the new FST data here\n",
    "        fstOutFile = open(fstOutfile, 'w')\n",
    "        #write header to outfile:\n",
    "        fstOutFile.write('CHROM\\tSTART_POS\\tEND_POS\\tWINDOW_ID\\tN_variants_tot\\tN_variants_forFst\\tN_variants_fixed\\tRofA_Fst\\tAofR_Fst\\n')\n",
    "\n",
    "        #1. Get SNP positions\n",
    "        chr_pos = get_chr_pos(dogCountFile)\n",
    "\n",
    "        #2. Process dog SNP frequency file\n",
    "        pop1 = [] #dog allele counts\n",
    "        for line in fileinput.input([dogCountFile]):  #change count file for dog and count file for wolf here\n",
    "            if line[0] == 'C':\n",
    "                continue \n",
    "            temp = process_inputlines(line) #temp's structure is allele1, allele2\n",
    "            pop1.append(temp) #this is adding the two alleles to pop1\n",
    "\n",
    "        #3. Process WOLF SNP frequency file\n",
    "        pop2 = [] #wolf allele counts\n",
    "        for line in fileinput.input([wolfCountFile]):\t\n",
    "            if line[0] == 'C':\n",
    "                continue \n",
    "            temp = process_inputlines(line)\n",
    "            pop2.append(temp)\n",
    "\n",
    "        #4. Make sliding windows and calculate Fst within them\n",
    "        calc_windows_fst_data(fstOutFile,chrom)\n",
    "        \n",
    "    #5. Merge the individual chromosome FST data together per simulation\n",
    "    simulation = i\n",
    "    mergedOutfile = wkDir + 'results/' + 'Simulation.%i_Hudson_Fst_200kbWindow_50kbSlide.txt' % simulation\n",
    "    cmd = 'cat %sresults/Simulation.chr*.%i_Hudson_Fst_200kbWindow_50kbSlide.txt | grep -v \"CHROM\" > %s' % (wkDir,simulation,mergedOutfile)\n",
    "    runCMD(cmd)\n",
    "\n",
    "    header = 'CHROM\\tSTART_POS\\tEND_POS\\tWINDOW_ID\\tN_variants_tot\\tN_variants_forFst\\tN_variants_fixed\\tRofA_Fst\\tAofR_Fst'\n",
    "    cmd = 'echo \\'%s\\' | cat - %s >  temp && mv temp %s' % (header, mergedOutfile,mergedOutfile)\n",
    "    runCMD(cmd)    \n",
    "    \n",
    "    #6. Filter windows == must have at least ten SNPs within it\n",
    "    filter_windows(mergedOutfile)\n",
    "    \n",
    "    #6. Z-transform the data\n",
    "    cmd = 'Rscript %sscripts/Ztransform.R %s_filtered_windows' % (wkDir,mergedOutfile)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    #break\n",
    "fstOutFile.close()\n",
    "\n",
    "#Check how long it took\n",
    "print(datetime.datetime.fromtimestamp(time.time()).strftime('%H:%M:%S'))\n",
    " "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "for i in range(0,10):\n",
    "    simulation = i\n",
    "    inFile = wkDir + 'results/' + 'Simulation.%i_Hudson_Fst_200kbWindow_50kbSlide.txt_filtered_windows_RofA_Fst_Zscores.txt' % (simulation)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
