{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import random \n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import glob\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Making list of chromosomes to process\n",
    "chromToDo = []\n",
    "for i in range(1,39):\n",
    "    chrom = 'chr' + str(i)\n",
    "    chromToDo.append(chrom)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Root results directory\n",
    "resultsDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/XP-CLR/results/'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading in chromosome lengths from:  /home/ampend/links/kidd-lab/www-mirror/www/track-hub/canFam3/canFam3.1-browser-chrom-sizes.fai\n"
     ]
    }
   ],
   "source": [
    "#Reading in chromosome sizes to determine boundaries of sliding windows per chromosome\n",
    "chromSizefile = '/home/ampend/links/kidd-lab/www-mirror/www/track-hub/canFam3/canFam3.1-browser-chrom-sizes.fai'\n",
    "print ('Reading in chromosome lengths from: ', chromSizefile)\n",
    "chromSizeFile = open(chromSizefile, 'r')\n",
    "\n",
    "chromSizeList, chromlist = {}, []\n",
    "#I also need a list of chr and position for finding the index for windows \n",
    "for line in chromSizeFile:\n",
    "    if \"chrM\" in line or \"chrUn\" in line or \"chrX\" in line: #ignore mito\n",
    "        continue\n",
    "    line = line.rstrip().split()\n",
    "\n",
    "    chromID,chromLength = line[0],int(line[1]) #chromosome ID and length\n",
    "    chromlist.append(chromID)\n",
    "    chromSizeList[chromID] = chromLength"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def read_xpclr_file(inFile):\n",
    "    #For keeping track of data\n",
    "    totalData, chroms = {}, []\n",
    "    scores, passCount = [], 0\n",
    "    totalGrids,infCounts,failed_SNPZero,failed_SNPMin = 0, 0, 0, 0\n",
    "\n",
    "    for line in inFile:\n",
    "        line=line.rstrip().split(' ')\n",
    "        if 'chrom' in line[0]:#skips header\n",
    "            continue\n",
    "        chrom = 'chr' + line[0]\n",
    "        if chrom not in chroms:\n",
    "            chroms.append(chrom)\n",
    "            totalData[chrom] = []\n",
    "        #processing line for more data\n",
    "        gridNum, snpCount = int(line[1]),int(line[2])\n",
    "        physical_pos,genetic_pos,max_s = round(float(line[3])),float(line[4]),float(line[6])\n",
    "        XPCLR_score = line[5]\n",
    "        totalGrids+=1\n",
    "        #keeping track of certain data points\n",
    "        if 'inf' in XPCLR_score:\n",
    "            infCounts+=1\n",
    "            continue\n",
    "        if snpCount < 1:\n",
    "            failed_SNPZero += 1\n",
    "            continue\n",
    "        if snpCount < 5:\n",
    "            failed_SNPMin += 1\n",
    "            continue\n",
    "        XPCLR_score=float(line[5])\n",
    "        scores.append(XPCLR_score)\n",
    "        totalData[chrom].append([chrom,gridNum,snpCount,physical_pos,genetic_pos,XPCLR_score,max_s])\n",
    "        passCount += 1\n",
    "        #Writing this information to a bedfile\n",
    "        bedFile.write('%s\\t%s\\t%s\\t%s_%s\\n' % (chrom,physical_pos-1,physical_pos,chrom,passCount))\n",
    "    inFile.close()\n",
    "    ################################################################\n",
    "    print('%s grids were processed by XP-CLR' % ((totalGrids)))\n",
    "    print('##EXCLUSIONS')\n",
    "    print('%s of the grids had scores equal to infinity and were excluded until we determine what that means (failure??)' % infCounts)\n",
    "    print('%s of the grids had no SNPs and were removed' % failed_SNPZero)\n",
    "    print('%s of the grids had fewer than 5 SNPs and were removed' % failed_SNPMin)\n",
    "    print('##FOR PROCESSING')\n",
    "    print('%s of the grids remained for further processing after passing filtrations' % passCount)\n",
    "    \n",
    "    return totalData\n",
    "#########################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "### Wolves_v_Dogs ####\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#In/Out files\n",
    "#subDir = 'Dogs_v_Wolves'\n",
    "subDir = 'Wolves_v_Dogs'\n",
    "gridType = 'window50kb_gridSize2kb'\n",
    "print('\\n### %s ####\\n' % subDir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "chr5\n"
     ]
    }
   ],
   "source": [
    "for i in totalData:\n",
    "    print(i)\n",
    "    break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1101423 grids were processed by XP-CLR\n",
      "##EXCLUSIONS\n",
      "627668 of the grids had scores equal to infinity and were excluded until we determine what that means (failure??)\n",
      "51309 of the grids had no SNPs and were removed\n",
      "72387 of the grids had fewer than 5 SNPs and were removed\n",
      "##FOR PROCESSING\n",
      "350059 of the grids remained for further processing after passing filtrations\n",
      "350059 windows read into chr_pos array for processing\n"
     ]
    }
   ],
   "source": [
    "#Saving the positions along each chromosome of the 50kb windows \n",
    "chr_pos = []\n",
    "posFile = open(resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/Merged_' + subDir + '_XPCLR_CM_' + gridType + '.xpclr.txt','r')\n",
    "bedFile = open(resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/Merged_' + subDir + '_XPCLR_CM_' + gridType + '.xpclr.bed','w')\n",
    "\n",
    "totalData = read_xpclr_file(posFile)\n",
    "for chrom in totalData:\n",
    "    for i in range(0,len(totalData[chrom])):\n",
    "        chrom,pos = totalData[chrom][i][0],totalData[chrom][i][3]\n",
    "        chr_pos.append([chrom, pos])\n",
    "posFile.close()\n",
    "print('%i windows read into chr_pos array for processing' % len(chr_pos))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Processing chr27 with size of 45876710\n",
      "Done with chromosome chr27 with size 45876710 bp\n",
      "Processing chr2 with size of 85426708\n",
      "Done with chromosome chr2 with size 85426708 bp\n",
      "Processing chr7 with size of 80974532\n",
      "Done with chromosome chr7 with size 80974532 bp\n",
      "Processing chr35 with size of 26524999\n",
      "Done with chromosome chr35 with size 26524999 bp\n",
      "Processing chr10 with size of 69331447\n",
      "Done with chromosome chr10 with size 69331447 bp\n",
      "Processing chr25 with size of 51628933\n",
      "Done with chromosome chr25 with size 51628933 bp\n",
      "Processing chr9 with size of 61074082\n",
      "Done with chromosome chr9 with size 61074082 bp\n",
      "Processing chr32 with size of 38810281\n",
      "Done with chromosome chr32 with size 38810281 bp\n",
      "Processing chr15 with size of 64190966\n",
      "Done with chromosome chr15 with size 64190966 bp\n",
      "Processing chr8 with size of 74330416\n",
      "Done with chromosome chr8 with size 74330416 bp\n",
      "Processing chr1 with size of 122678785\n",
      "Done with chromosome chr1 with size 122678785 bp\n",
      "Processing chr36 with size of 30810995\n",
      "Done with chromosome chr36 with size 30810995 bp\n",
      "Processing chr26 with size of 38964690\n",
      "Done with chromosome chr26 with size 38964690 bp\n",
      "Processing chr13 with size of 63241923\n",
      "Done with chromosome chr13 with size 63241923 bp\n",
      "Processing chr37 with size of 30902991\n",
      "Done with chromosome chr37 with size 30902991 bp\n",
      "Processing chr31 with size of 39895921\n",
      "Done with chromosome chr31 with size 39895921 bp\n",
      "Processing chr18 with size of 55844845\n",
      "Done with chromosome chr18 with size 55844845 bp\n",
      "Processing chr16 with size of 59632846\n",
      "Done with chromosome chr16 with size 59632846 bp\n",
      "Processing chr38 with size of 23914537\n",
      "Done with chromosome chr38 with size 23914537 bp\n",
      "Processing chr11 with size of 74389097\n",
      "Done with chromosome chr11 with size 74389097 bp\n",
      "Processing chr28 with size of 41182112\n",
      "Done with chromosome chr28 with size 41182112 bp\n",
      "Processing chr17 with size of 64289059\n",
      "Done with chromosome chr17 with size 64289059 bp\n",
      "Processing chr20 with size of 58134056\n",
      "Done with chromosome chr20 with size 58134056 bp\n",
      "Processing chr3 with size of 91889043\n",
      "Done with chromosome chr3 with size 91889043 bp\n",
      "Processing chr12 with size of 72498081\n",
      "Done with chromosome chr12 with size 72498081 bp\n",
      "Processing chr22 with size of 61439934\n",
      "Done with chromosome chr22 with size 61439934 bp\n",
      "Processing chr5 with size of 88915250\n",
      "Done with chromosome chr5 with size 88915250 bp\n",
      "Processing chr33 with size of 31377067\n",
      "Done with chromosome chr33 with size 31377067 bp\n",
      "Processing chr19 with size of 53741614\n",
      "Done with chromosome chr19 with size 53741614 bp\n",
      "Processing chr34 with size of 42124431\n",
      "Done with chromosome chr34 with size 42124431 bp\n",
      "Processing chr30 with size of 40214260\n",
      "Done with chromosome chr30 with size 40214260 bp\n",
      "Processing chr29 with size of 41845238\n",
      "Done with chromosome chr29 with size 41845238 bp\n",
      "Processing chr4 with size of 88276631\n",
      "Done with chromosome chr4 with size 88276631 bp\n",
      "Processing chr21 with size of 50858623\n",
      "Done with chromosome chr21 with size 50858623 bp\n",
      "Processing chr14 with size of 60966679\n",
      "Done with chromosome chr14 with size 60966679 bp\n",
      "Processing chr6 with size of 77573801\n",
      "Done with chromosome chr6 with size 77573801 bp\n",
      "Processing chr23 with size of 52294480\n",
      "Done with chromosome chr23 with size 52294480 bp\n",
      "Processing chr24 with size of 47698779\n",
      "Done with chromosome chr24 with size 47698779 bp\n",
      "220053 sliding window coordinates are found\n"
     ]
    }
   ],
   "source": [
    "\"\"\"#Generating the sliding window coordinates\n",
    "head_idx, tail_idx, last_line = 0, 0, 0\n",
    "sliding_window_coord_dict = {}\n",
    "sliding_window_averages = {}\n",
    "\n",
    "for chrom in chromSizeList:\n",
    "    print('Processing %s with size of %i' % (chrom,chromSizeList[chrom]))\n",
    "    sliding_window_coord_dict[chrom] = []\n",
    "    window_start, window_end, totalWindowCount, lastWindow  = 0, 0, 0, False\n",
    "    for i in range(1,130000000,10000):  #sliding windows along each chromosome 100kb long sliding 10kb at a time\n",
    "        if i > 1 and window_end ==  int(chromSizeList[chrom]):\n",
    "            break\n",
    "        window_start = i\n",
    "        window_end = i + 100000 - 1\n",
    "        \n",
    "        if int(window_start) > int(chromSizeList[chrom]):\n",
    "            break\n",
    "        if int(window_end) > int(chromSizeList[chrom]): #if the end of the window extends further than the length of the chromsome, then the window end is the length of the chromosome\n",
    "            window_end = int(chromSizeList[chrom])\n",
    "        \n",
    "        while int(chr_pos[head_idx][1]) < i and chr_pos[head_idx][0] == chrom:\n",
    "            head_idx += 1\n",
    "        if head_idx >= len(chr_pos): #if head index is greater than the total number of SNPs, cancel\n",
    "            break\n",
    "        if tail_idx >= len(chr_pos):#if head index is greater than the total number of SNPs, cancel\n",
    "            break\n",
    "        \n",
    "        while (int(chr_pos[tail_idx][1]) <= (i+100000) and chr_pos[tail_idx][0] == chrom):\n",
    "            tail_idx += 1\n",
    "            if tail_idx == len(chr_pos): \n",
    "                last_line = 1\n",
    "                break\n",
    "        ##################\n",
    "        windowScores = []\n",
    "        for p in range(0,len(totalData[chrom])):\n",
    "            pos_chrom,pos_pos,pos_score = totalData[chrom][p][0],int(totalData[chrom][p][3]), float(totalData[chrom][p][5])\n",
    "            if pos_chrom != chrom:\n",
    "                continue\n",
    "            if pos_pos >= window_start and pos_pos <= window_end:\n",
    "                windowScores.append(pos_score)\n",
    "            if pos_pos > window_end:\n",
    "                break\n",
    "        if len(windowScores) > 0:\n",
    "            avg_score, median_score = round(np.mean(windowScores),3), round(np.median(windowScores),3)     \n",
    "        else:\n",
    "            avg_score, median_score = 'na', 'na'\n",
    "        sliding_window_coord_dict[chrom].append([chrom,window_start,window_end,len(windowScores),avg_score,median_score])\n",
    "    print('Done with chromosome %s with size %i bp' % (chrom, chromSizeList[chrom]))\n",
    "    \n",
    "    \n",
    "#Writing window averages to outfile\n",
    "count = 0\n",
    "outFile = open(resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/WindowAverages_' + subDir + '_XP-CLR_50kbWindow_2kbGrid.txt','w')\n",
    "outFile.write('Chromosome\\tWindow_Start\\tWindow_End\\tNumber_of_XP-CLR_Regions\\tAverage_XP-CLR_Score\\tMedian_XP-CLR_Score\\n')\n",
    "for i in sliding_window_coord_dict:\n",
    "    chrom = i\n",
    "    count = count + len(sliding_window_coord_dict[chrom])\n",
    "    for window in range(0,len(sliding_window_coord_dict[chrom])):\n",
    "        outFile.write('\\t'.join(map(str,sliding_window_coord_dict[chrom][window])) + '\\n')\n",
    "outFile.close()\n",
    "#How many windows are spanning the genome?\n",
    "print('%i sliding window coordinates are found' % count)\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def slide_basic_stats(slideData):\n",
    "    #Save as pandas dataframe\n",
    "    print('#Calculating basic stats')\n",
    "    df = pd.DataFrame(slideData, columns=['Chromosome','Window_Start','Window_End','XP-CLR_Window_Count','Average_XP-CLR_Score','Median_XP-CLR_Score'])\n",
    "    scoreMed = df['Average_XP-CLR_Score'].median()\n",
    "    scoreMean = df['Average_XP-CLR_Score'].mean()\n",
    "    scoreMax = df['Average_XP-CLR_Score'].max()\n",
    "    print ('Median XP-CLR Score is', scoreMed)\n",
    "    print ('Mean XP-CLR Score is ', scoreMean)\n",
    "    print ('Max XP-CLR Score is ', scoreMax)\n",
    "\n",
    "    pct99, pct995, pct999 = np.percentile(df['Average_XP-CLR_Score'], 99), np.percentile(df['Average_XP-CLR_Score'], 99.5), np.percentile(df['Average_XP-CLR_Score'], 99.9)\n",
    "    print('##SIGNIFICANCE CUT-OFFS\\n99th percentile of XP-CLR scores == ', pct99)\n",
    "    print('99.5th percentile of XP-CLR scores == ', pct995)\n",
    "    print('99.9th percentile of XP-CLR scores == ', pct999)\n",
    "    \n",
    "    return pct99,pct995,pct999\n",
    "################################################################################################\n",
    "def draw_slide_hist(plotFile, pct99):\n",
    "    fig, axs = plt.subplots(1,2, figsize=(15, 5)) #Set up subplots for below distributions\n",
    "\n",
    "    #Graph 1 = Score Distribution\n",
    "    df['Average_XP-CLR_Score'].plot(ax=axs[0],kind='hist', bins=350,xlim=(0,50),alpha=0.75, normed = True)\n",
    "    ax = axs[0]\n",
    "    ax.set_xlabel('Average XP-CLR Score')\n",
    "    ax.set_title('All Chromosomes - XP-CLR Average Sliding Window Score Distribution')\n",
    "    ax.vlines(x=pct99,ymin=0,ymax=0.4, lw=1,color=\"red\")\n",
    "\n",
    "    #Graph 2 = SNP Count Distribution\n",
    "    df['XP-CLR_Window_Count'].plot(ax=axs[1],kind='hist', bins=75,alpha=0.75, normed = True, color=\"darkorange\")\n",
    "    ax = axs[1]\n",
    "    ax.set_xlabel('XP-CLR Windows per Sliding Window')\n",
    "    ax.set_title('All Chromosomes - Per Sliding Window XP-CLR Window Counts')\n",
    "    avg_SNPCount = df['XP-CLR_Window_Count'].mean()\n",
    "    ax.vlines(x=avg_SNPCount,ymin=0,ymax=0.1, lw=1,color=\"red\")\n",
    "\n",
    "    plt.tight_layout()\n",
    "    plt.savefig(plotFile) #Saving plots\n",
    "################################################################################################\n",
    "def slide_draw_Manhattan_Plot(slideData,plotFile):\n",
    "    df = pd.DataFrame(slideData, columns=['Chromosome','Window_Start','Window_End','XP-CLR_Window_Count','Average_XP-CLR_Score','Median_XP-CLR_Score'])\n",
    "    df.Chromosome = df.Chromosome.astype('category')\n",
    "    df.Chromosome = df.Chromosome.cat.set_categories(chromToDo, ordered=True)\n",
    "    df = df.sort_values('Chromosome')\n",
    "\n",
    "    df.head()\n",
    "    # How to plot gene vs. -log10(pvalue) and colour it by chromosome?\n",
    "    df['ind'] = range(len(df))\n",
    "    df_grouped = df.groupby(('Chromosome'))\n",
    "\n",
    "    ###PLOTTING MANHATTAN PLOT \n",
    "    #plotDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/XP-CLR/results/plots/'\n",
    "    fig = plt.figure(figsize=(24,16))\n",
    "    ax = fig.add_subplot(111)\n",
    "    x_labels, x_labels_pos, colors = [], [], ['black','gray']\n",
    "    for num, (name, group) in enumerate(df_grouped):\n",
    "        group.plot(kind='scatter', x='ind', y='Average_XP-CLR_Score',color=colors[num % len(colors)], ax=ax)\n",
    "        x_labels.append(name)\n",
    "        x_labels_pos.append((group['ind'].iloc[-1] - (group['ind'].iloc[-1] - group['ind'].iloc[0])/2))\n",
    "    #Xaxis\n",
    "    ax.set_xlim([0, len(df)])\n",
    "    ax.set_xticklabels(ax.xaxis.get_majorticklabels(), rotation=45)\n",
    "    ax.set_xticks(x_labels_pos)\n",
    "    ax.set_xticklabels(x_labels, fontsize=16)\n",
    "    ax.set_xlabel('Chromosome',fontsize=20)\n",
    "    #Yaxis\n",
    "    #ax.set_ylim([0, scoreMax+10])\n",
    "    ax.set_ylabel('Average XP-CLR Score',fontsize=20)\n",
    "    #Tick label sizes\n",
    "    plt.tick_params(axis='both', which='major', labelsize=16)\n",
    "    fig.tight_layout(rect=[0,0,.8,1]) \n",
    "    plt.hlines(y=pct995,xmin=0,xmax=len(df), lw=1,color=\"red\", label=\"99.5th Sig Level\")\n",
    "    legend = ax.legend(loc='upper right', shadow=False,prop={'size': 20})\n",
    "    plt.title(subDir.replace('_',' ') + ' XP-CLR Scores',fontsize=30)\n",
    "    plt.tight_layout()\n",
    "    plt.savefig(plotFile)\n",
    "##############################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def slide_find_sig_windows(sigThreshold, slideData):\n",
    "    print('\\n#SIGNIFICANT WINDOWS\\nSignificance threshold is XP-CLR = ', sigThreshold)\n",
    "    sigHits = []\n",
    "\n",
    "    for i in range(0,len(slideData)):\n",
    "        chrom, score = slideData[i][0], round(float(slideData[i][4]),3)\n",
    "        if score > sigThreshold:\n",
    "            sigHits.append(slideData[i])\n",
    "    print('%i significant windows found from XP-CLR'%(len(sigHits)))\n",
    "    return sigHits\n",
    "##########################################################################################\n",
    "def slide_merge_sig_windows(sigHits):\n",
    "    sigCount = 0\n",
    "    print('\\n#MERGED WINDOWS')\n",
    "    chromList = []\n",
    "    for i in range(0,len(sigHits)):\n",
    "        chrom, start, end, score = sigHits[i][0], int(sigHits[i][1]), int(sigHits[i][2]),float(sigHits[i][4])\n",
    "        winID  = chrom + '_' + str(num)\n",
    "\n",
    "        #keeping track of chromosomes analyzed, helps save information automatically for first call on a chromosome\n",
    "        #   for further comparisons with the next call\n",
    "        if chrom not in chromList:\n",
    "            chromList.append(chrom)\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            sigCount += 1\n",
    "            mergedWindows[sigCount] = []\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "            #print('FIRST',chrom, num, pos, start, end, score)\n",
    "            continue\n",
    "        #processing calls\n",
    "        if CHROM == chrom and start - START < 100001: #can only have one window between them\n",
    "            #print('YES',chrom, num, pos, start, end, score)\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "            CHROM, NUM, START,END = chrom, num, start, end\n",
    "\n",
    "        else:\n",
    "            #print('NO',chrom, num, pos, start, end, score)\n",
    "            sigCount += 1\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            mergedWindows[sigCount] = []\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "\n",
    "        CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "    print('%i merged windows found to be significant' % sigCount)\n",
    "    return mergedWindows\n",
    "##########################################################################################\n",
    "def slide_filter_merged_windows(mergedWindows):\n",
    "    print('\\n#FILTER MERGED WINDOWS')\n",
    "    merged3Windows, count = [], 0\n",
    "\n",
    "    for i in mergedWindows:\n",
    "        if len(mergedWindows[i]) < 3:\n",
    "            continue\n",
    "        count += 1 #keeping track of how many merged windows with at least 3 sig windows are present\n",
    "        chrom, start = mergedWindows[i][0][0], mergedWindows[i][0][2] #chrom and start can be defined by the first indexed window in the list\n",
    "        scores = []\n",
    "        for j in mergedWindows[i]:\n",
    "            end = j[3] #end will always be the last window read through the loop, constantly overwriting\n",
    "            scores.append(j[4]) #to keep track of for calculating the average score\n",
    "        meanScore = format(round(np.mean(scores),3)) #formats and rounds the average scores per merged window\n",
    "        winLength = end - start + 1\n",
    "        ID = 'XP-CLR_' + str(count) \n",
    "        merged3Windows.append([chrom,start,end,ID,winLength,meanScore]) #now save all the information\n",
    "\n",
    "    print('%i windows contained three or more adjacent windows and were merged' % count) \n",
    "    return merged3Windows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "220053 average scores in windows added back to sliding window coord dict\n"
     ]
    }
   ],
   "source": [
    "#read in chromosome average files === saves time\n",
    "inFile = open(resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/WindowAverages_' + subDir + '_XP-CLR_50kbWindow_2kbGrid.txt','r')\n",
    "#reinitiating the dictionary\n",
    "sliding_window_coord_dict, chroms, count = {}, [], 0\n",
    "\n",
    "for line in inFile:\n",
    "    if 'Average' in line:\n",
    "        continue\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom = line[0]\n",
    "    if chrom not in chroms:\n",
    "        chroms.append(chrom)\n",
    "        sliding_window_coord_dict[chrom] = []\n",
    "    sliding_window_coord_dict[chrom].append(line)\n",
    "    count += 1\n",
    "print('%i average scores in windows added back to sliding window coord dict' % count)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#Calculating basic stats\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    }
   ],
   "source": [
    "print('#Calculating basic stats')\n",
    "df = pd.DataFrame(slideData, columns=['Chromosome','Window_Start','Window_End','XP-CLR_Window_Count','Average_XP-CLR_Score','Median_XP-CLR_Score'])\n",
    "scoreMed = df['Average_XP-CLR_Score'].median()\n",
    "scoreMean = df['Average_XP-CLR_Score'].mean()\n",
    "scoreMax = df['Average_XP-CLR_Score'].max()\n",
    "print ('Median XP-CLR Score is', scoreMed)\n",
    "print ('Mean XP-CLR Score is ', scoreMean)\n",
    "print ('Max XP-CLR Score is ', scoreMax)\n",
    "\n",
    "pct99, pct995, pct999 = np.percentile(df['Average_XP-CLR_Score'], 99), np.percentile(df['Average_XP-CLR_Score'], 99.5), np.percentile(df['Average_XP-CLR_Score'], 99.9)\n",
    "print('##SIGNIFICANCE CUT-OFFS\\n99th percentile of XP-CLR scores == ', pct99)\n",
    "print('99.5th percentile of XP-CLR scores == ', pct995)\n",
    "print('99.9th percentile of XP-CLR scores == ', pct999)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21280 sliding windows had no data and were therefore ignored...\n",
      "198773 windows remain for analysis and determination of significance thresholds\n",
      "\n"
     ]
    },
    {
     "ename": "TypeError",
     "evalue": "can't multiply sequence by non-int of type 'float'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-69-f9840bee5bfc>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m     25\u001b[0m \u001b[0mprint\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m'Mean XP-CLR Score is '\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mscoreMean\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     26\u001b[0m \u001b[0;31m#print ('Max XP-CLR Score is ', scoreMax)\"\"\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 27\u001b[0;31m \u001b[0mpct99\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpct995\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpct999\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'Average_XP-CLR_Score'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m99\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'Average_XP-CLR_Score'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m99.5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpercentile\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdf\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'Average_XP-CLR_Score'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m99.9\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     28\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'##SIGNIFICANCE CUT-OFFS\\n99th percentile of XP-CLR scores == '\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpct99\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/numpy/lib/function_base.py\u001b[0m in \u001b[0;36mpercentile\u001b[0;34m(a, q, axis, out, overwrite_input, interpolation, keepdims)\u001b[0m\n\u001b[1;32m   3697\u001b[0m     r, k = _ureduce(a, func=_percentile, q=q, axis=axis, out=out,\n\u001b[1;32m   3698\u001b[0m                     \u001b[0moverwrite_input\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0moverwrite_input\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 3699\u001b[0;31m                     interpolation=interpolation)\n\u001b[0m\u001b[1;32m   3700\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mkeepdims\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   3701\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mq\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mndim\u001b[0m \u001b[0;34m==\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/numpy/lib/function_base.py\u001b[0m in \u001b[0;36m_ureduce\u001b[0;34m(a, func, **kwargs)\u001b[0m\n\u001b[1;32m   3423\u001b[0m         \u001b[0mkeepdim\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0ma\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mndim\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   3424\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 3425\u001b[0;31m     \u001b[0mr\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mfunc\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0ma\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   3426\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mr\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkeepdim\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   3427\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/numpy/lib/function_base.py\u001b[0m in \u001b[0;36m_percentile\u001b[0;34m(a, q, axis, out, overwrite_input, interpolation, keepdims)\u001b[0m\n\u001b[1;32m   3814\u001b[0m             \u001b[0mn\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0misnan\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0map\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m...\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   3815\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 3816\u001b[0;31m         \u001b[0mx1\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtake\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0map\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindices_below\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mweights_below\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   3817\u001b[0m         \u001b[0mx2\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtake\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0map\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mindices_above\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mweights_above\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   3818\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mTypeError\u001b[0m: can't multiply sequence by non-int of type 'float'"
     ]
    }
   ],
   "source": [
    "#Processing the sliding window data with sliding window XP-CLR score averages\n",
    "slideData, noDataCount = [], 0\n",
    "for chrom in sliding_window_coord_dict:\n",
    "    for window in range(0,len(sliding_window_coord_dict[chrom])):\n",
    "        if 'na' in sliding_window_coord_dict[chrom][window]: #ignore sliding windows with no data\n",
    "            noDataCount += 1\n",
    "            continue\n",
    "        chrom = sliding_window_coord_dict[chrom][window][0]\n",
    "        start = int(sliding_window_coord_dict[chrom][window][1])\n",
    "        end = int(sliding_window_coord_dict[chrom][window][2])\n",
    "        windowcount = int(sliding_window_coord_dict[chrom][window][3])\n",
    "        mean = float(sliding_window_coord_dict[chrom][window][4])\n",
    "        #median = float(sliding_window_coord_dict[chrom][window][5])\n",
    "        slideData.append([chrom,start,end,windowcount,mean])\n",
    "print('%i sliding windows had no data and were therefore ignored...' % noDataCount)\n",
    "print('%i windows remain for analysis and determination of significance thresholds\\n' % len(slideData))\n",
    "\n",
    "\n",
    "df = pd.DataFrame(test, columns=['Chromosome','Window_Start','Window_End','XP-CLR_Window_Count','Average_XP-CLR_Score','Median_XP-CLR_Score'])\n",
    "df.head()\n",
    "\"\"\"scoreMed = df['Average_XP-CLR_Score'].median()\n",
    "scoreMean = df['Average_XP-CLR_Score'].mean()\n",
    "#scoreMax = df['Average_XP-CLR_Score'].max()\n",
    "print ('Median XP-CLR Score is', scoreMed)\n",
    "print ('Mean XP-CLR Score is ', scoreMean)\n",
    "#print ('Max XP-CLR Score is ', scoreMax)\"\"\"\n",
    "pct99, pct995, pct999 = np.percentile(df['Average_XP-CLR_Score'], 99), np.percentile(df['Average_XP-CLR_Score'], 99.5), np.percentile(df['Average_XP-CLR_Score'], 99.9)\n",
    "print('##SIGNIFICANCE CUT-OFFS\\n99th percentile of XP-CLR scores == ', pct99)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "21280 sliding windows had no data and were therefore ignored...\n",
      "198773 windows remain for analysis and determination of significance thresholds\n",
      "\n",
      "#Calculating basic stats\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    }
   ],
   "source": [
    "#Processing the sliding window data with sliding window XP-CLR score averages\n",
    "slideData, noDataCount = [], 0\n",
    "for chrom in sliding_window_coord_dict:\n",
    "    for window in range(0,len(sliding_window_coord_dict[chrom])):\n",
    "        if 'na' in sliding_window_coord_dict[chrom][window]: #ignore sliding windows with no data\n",
    "            noDataCount += 1\n",
    "            continue\n",
    "        slideData.append(sliding_window_coord_dict[chrom][window])\n",
    "print('%i sliding windows had no data and were therefore ignored...' % noDataCount)\n",
    "print('%i windows remain for analysis and determination of significance thresholds\\n' % len(slideData))\n",
    "\n",
    "pct99,pct995,pct999 = slide_basic_stats(slideData)\n",
    "\n",
    "#plot histogram of sliding window averages\n",
    "plotFile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/plots/' + 'WindowAverages_' + subDir + '_XP-CLR_50kbWindow_2kbGrid.png'\n",
    "#draw_slide_hist(plotFile, pct99)\n",
    "#Draw Manhattan plot of sliding window averages\n",
    "plotFile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/plots/' + 'ManhattanPlot_WindowAverages_' + subDir + '_XP-CLR_50kbWindow_2kbGrid.png'\n",
    "#slide_draw_Manhattan_Plot(slideData,plotFile)\n",
    "\n",
    "#Find significant windows above 99.5th percentile \n",
    "sigHits = slide_find_sig_windows(pct995, slideData)\n",
    "#Merge significanit windows above 99.5th percentile\n",
    "mergedWindows = slide_merge_sig_windows(sigHits)\n",
    "#Now identifying significant windows that are at least 3 windows wide\n",
    "outFile = open(resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/SignificantWindows/' + 'Significant_3Windows_pct995_AverageXP-CLRScore.txt', 'w')\n",
    "merged3Windows = slide_filter_merged_windows(mergedWindows)\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    outFile.write('\\t'.join(map(str,merged3Windows[i]))+'\\n')\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 170,
   "metadata": {},
   "outputs": [
    {
     "ename": "AttributeError",
     "evalue": "'dict_keys' object has no attribute 'sort'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-170-f1568558f580>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0moperator\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0ma\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mwinDict\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mkeys\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 8\u001b[0;31m \u001b[0ma\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msort\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mkey\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0moperator\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitemgetter\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m: 'dict_keys' object has no attribute 'sort'"
     ]
    }
   ],
   "source": [
    "winDict = {}\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    winID = merged3Windows[i][0] + ' ' + str(merged3Windows[i][1])\n",
    "    winDict[winID] = merged3Windows[i]\n",
    "\n",
    "import operator\n",
    "a = winDict.keys()\n",
    "a.sort(key = operator.itemgetter(1))"
   ]
  },
  {
   "cell_type": "raw",
   "metadata": {},
   "source": [
    "################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def basic_stats(totalData):\n",
    "    #Save full MAF array into a pandas dataframe\n",
    "    df = pd.DataFrame(totalData,columns=['Chromosome','Grid','SNPCount','GenPos','cMPos','Score','MaxScore'])\n",
    "    df.head()\n",
    "\n",
    "    scoreMed = df['Score'].median()\n",
    "    scoreMean = df['Score'].mean()\n",
    "    scoreMax = df['Score'].max()\n",
    "    print ('Median XP-CLR Score is', scoreMed)\n",
    "    print ('Mean XP-CLR Score is ', scoreMean)\n",
    "    print ('Max XP-CLR Score is ', scoreMax)\n",
    "\n",
    "    pct99, pct995, pct999 = np.percentile(df['Score'], 99), np.percentile(df['Score'], 99.5), np.percentile(df['Score'], 99.9)\n",
    "    print('##SIGNIFICANCE CUT-OFFS\\n99th percentile of XP-CLR scores == ', pct99)\n",
    "    print('99.5th percentile of XP-CLR scores == ', pct995)\n",
    "    print('99.9th percentile of XP-CLR scores == ', pct999)\n",
    "    return df, pct99, pct995, scoreMax\n",
    "######################################################################################################\n",
    "def draw_hist(plotFile, pct99):\n",
    "    fig, axs = plt.subplots(1,2, figsize=(15, 5)) #Set up subplots for below distributions\n",
    "\n",
    "    #Graph 1 = Score Distribution\n",
    "    df['Score'].plot(ax=axs[0],kind='hist', bins=350,xlim=(0,50),alpha=0.75, normed = True)\n",
    "    ax = axs[0]\n",
    "    ax.set_xlabel('XP-CLR Score')\n",
    "    ax.set_title('All Chromosomes - XP-CLR Score Distribution')\n",
    "    ax.vlines(x=pct99,ymin=0,ymax=0.4, lw=1,color=\"red\")\n",
    "\n",
    "    #Graph 2 = SNP Count Distribution\n",
    "    df['SNPCount'].plot(ax=axs[1],kind='hist', bins=75,alpha=0.75, normed = True, color=\"darkorange\")\n",
    "    ax = axs[1]\n",
    "    ax.set_xlabel('XP-CLR SNP Count')\n",
    "    ax.set_title('All Chromosomes - Per Grid SNP Counts')\n",
    "    avg_SNPCount = df['SNPCount'].mean()\n",
    "    ax.vlines(x=avg_SNPCount,ymin=0,ymax=0.1, lw=1,color=\"red\")\n",
    "\n",
    "    plt.savefig(plotFile) #Saving plots\n",
    "##############################################################################################################\n",
    "def draw_Manhattan_Plot(totalData,scoreMax):\n",
    "    df = pd.DataFrame(totalData,columns=['Chromosome','Grid','SNPCount','GenPos','cMPos','Score','MaxScore'])\n",
    "    df.Chromosome = df.Chromosome.astype('category')\n",
    "    df.Chromosome = df.Chromosome.cat.set_categories(chromToDo, ordered=True)\n",
    "    df = df.sort_values('Chromosome')\n",
    "\n",
    "    df.head()\n",
    "    # How to plot gene vs. -log10(pvalue) and colour it by chromosome?\n",
    "    df['ind'] = range(len(df))\n",
    "    df_grouped = df.groupby(('Chromosome'))\n",
    "\n",
    "    ###PLOTTING MANHATTAN PLOT \n",
    "    #plotDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/XP-CLR/results/plots/'\n",
    "    fig = plt.figure(figsize=(24,12))\n",
    "    ax = fig.add_subplot(111)\n",
    "    x_labels, x_labels_pos, colors = [], [], ['black','gray']\n",
    "    for num, (name, group) in enumerate(df_grouped):\n",
    "        group.plot(kind='scatter', x='ind', y='Score',color=colors[num % len(colors)], ax=ax)\n",
    "        x_labels.append(name)\n",
    "        x_labels_pos.append((group['ind'].iloc[-1] - (group['ind'].iloc[-1] - group['ind'].iloc[0])/2))\n",
    "    #Xaxis\n",
    "    ax.set_xlim([0, len(df)])\n",
    "    ax.set_xticklabels(ax.xaxis.get_majorticklabels(), rotation=45)\n",
    "    ax.set_xticks(x_labels_pos)\n",
    "    ax.set_xticklabels(x_labels, fontsize=16)\n",
    "    ax.set_xlabel('Chromosome',fontsize=20)\n",
    "    #Yaxis\n",
    "    ax.set_ylim([0, scoreMax+10])\n",
    "    ax.set_ylabel('XP-CLR Score',fontsize=20)\n",
    "    #Tick label sizes\n",
    "    plt.tick_params(axis='both', which='major', labelsize=16)\n",
    "    fig.tight_layout(rect=[0,0,.8,1]) \n",
    "    plt.hlines(y=pct995,xmin=0,xmax=len(df), lw=1,color=\"red\", label=\"99.5th Sig Level\")\n",
    "    legend = ax.legend(loc='upper right', shadow=False,prop={'size': 20})\n",
    "    plt.title(subDir.replace('_',' ') + ' XP-CLR Scores',fontsize=30)\n",
    "    plt.savefig(plotDir + subDir + '_50kbWindow_2kbGridSize' + '_XP-CLR.png')\n",
    "##############################################################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def find_sig_windows(sigThreshold, totalData):\n",
    "    print('\\n#SIGNIFICANT WINDOWS\\nSignificance threshold is XP-CLR = ', sigThreshold)\n",
    "    sigHits = []\n",
    "\n",
    "    for i in range(0,len(totalData)):\n",
    "        chrom, score = totalData[i][0], round(float(totalData[i][5]),3)\n",
    "        if score > sigThreshold:\n",
    "            sigHits.append(totalData[i])\n",
    "    print('%i significant windows found from XP-CLR'%(len(sigHits)))\n",
    "    return sigHits\n",
    "\n",
    "chromList, count, sigCount = [], 0, -1\n",
    "CHROM, WINDOW, SCORE = 0, 0, 0\n",
    "mergedWindows = {}\n",
    "##############################################################################################################\n",
    "def merge_sig_windows(sigHits):\n",
    "    sigCount = 0\n",
    "    print('\\n#MERGED WINDOWS')\n",
    "    chromList = []\n",
    "    for i in range(0,len(sigHits)):\n",
    "        chrom, num, pos, score = sigHits[i][0], sigHits[i][1], round(sigHits[i][3]),float(sigHits[i][5])\n",
    "        start, end = pos-25000, pos+25000 #start/end of the sliding window is 25kb from either side of this grid point\n",
    "        winID  = chrom + '_' + str(num)\n",
    "\n",
    "        #keeping track of chromosomes analyzed, helps save information automatically for first call on a chromosome\n",
    "        #   for further comparisons with the next call\n",
    "        if chrom not in chromList:\n",
    "            chromList.append(chrom)\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            sigCount += 1\n",
    "            mergedWindows[sigCount] = []\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "            #print('FIRST',chrom, num, pos, start, end, score)\n",
    "            continue\n",
    "        #processing calls\n",
    "        if CHROM == chrom and num - NUM < 2: #can only have one window between them\n",
    "            #print('YES',chrom, num, pos, start, end, score)\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "            CHROM, NUM, START,END = chrom, num, start, end\n",
    "\n",
    "        else:\n",
    "            #print('NO',chrom, num, pos, start, end, score)\n",
    "            sigCount += 1\n",
    "            CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "            mergedWindows[sigCount] = []\n",
    "            mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "\n",
    "        CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "\n",
    "    print('%i merged windows found to be significant' % sigCount)\n",
    "    return mergedWindows\n",
    "##############################################################################################################\n",
    "def filter_merged_windows(mergedWindows):\n",
    "    print('\\n#FILTER MERGED WINDOWS')\n",
    "    merged3Windows, count = [], 0\n",
    "\n",
    "    for i in mergedWindows:\n",
    "        if len(mergedWindows[i]) < 3:\n",
    "            continue\n",
    "        count += 1 #keeping track of how many merged windows with at least 3 sig windows are present\n",
    "        chrom, start = mergedWindows[i][0][0], mergedWindows[i][0][2] #chrom and start can be defined by the first indexed window in the list\n",
    "        scores = []\n",
    "        for j in mergedWindows[i]:\n",
    "            end = j[3] #end will always be the last window read through the loop, constantly overwriting\n",
    "            scores.append(j[4]) #to keep track of for calculating the average score\n",
    "        meanScore = format(round(np.mean(scores),3)) #formats and rounds the average scores per merged window\n",
    "        winLength = end - start\n",
    "        ID = 'XP-CLR_' + str(count) \n",
    "        merged3Windows.append([chrom,start,end,ID,winLength,meanScore]) #now save all the information\n",
    "\n",
    "    print('%i windows contained three or more adjacent windows and were merged' % count) \n",
    "    return merged3Windows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##############################################################################################\n",
    "def find_gene_pos():\n",
    "    #GTF file with genes of interest:\n",
    "    gtfFile = open('/home/ampend/links/kidd-lab/ampend-projects/BLAST2GO/ensembl_input/Canis_familiaris.CanFam3.1.81.gtf','r')\n",
    "    #YOU NEED TO DO THIS BY HAND BY GREP'ING THE GTF FILE:\n",
    "    genesOfInterest, genePos = [], []\n",
    "    for line in gtfFile:\n",
    "        if '#' in line:\n",
    "            continue\n",
    "        line=line.rstrip().split('\\t')\n",
    "        if 'exon' not in line[2]:\n",
    "            continue\n",
    "        chrom, start, end, direction = 'chr' + line[0], int(line[3]), int(line[4]), str(line[6])\n",
    "        if chrom == ROIchrom:\n",
    "            if start >= ROIstart and end <= ROIend:\n",
    "                geneInfo = line[8]  #Finding gene ID in line\n",
    "                if 'gene_name' in geneInfo:\n",
    "                    geneInfo = geneInfo.split(';')\n",
    "                    geneInfo = geneInfo[5].split(' ')\n",
    "                    geneID = geneInfo[2].replace('\"','')\n",
    "                else:\n",
    "                    geneInfo = geneInfo.split(';')\n",
    "                    geneInfo = geneInfo[0].split(' ')\n",
    "                    geneID = geneInfo[1].replace('\"','')\n",
    "                if 'ENSCAFG00000028653' in geneID:\n",
    "                    geneID = 'AMY2B'\n",
    "                if geneID not in genesOfInterest:\n",
    "                    genesOfInterest.append(geneID)\n",
    "                genePos.append([geneID,start,end,direction])\n",
    "            else:\n",
    "                continue\n",
    "        else:\n",
    "            continue\n",
    "    gtfFile.close()\n",
    "    return genesOfInterest,genePos\n",
    "###################################################################\n",
    "def get_xpclr_score(totalData,ROIchrom,ROIstart,ROIend):\n",
    "    ##LONG## \n",
    "    dataArray, index = [], -1\n",
    "    for window in totalData:\n",
    "        chrom,midpoint,start, end, score = window[0], int(window[3]), int(window[3])-25000, int(window[3])+25000, float(window[5])\n",
    "        #Now check if the call is within the ROI\n",
    "        if chrom == ROIchrom and start >= ROIstart and end <= ROIend:\n",
    "            dataArray.append([chrom,midpoint,score])\n",
    "    return dataArray\n",
    "###################################################################\n",
    "def get_fst():\n",
    "    #Per Site, Total callset\n",
    "    perTotal = '/home/ampend/links/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/Total_54callset_MxbRemoved/PerSite/Merged_VillageWolf_mxb_rm_PERSITE_54callset_AutoXParXNonPar_Hudson_Fst_PerSite_Zscores.txt'\n",
    "    fileArray = []\n",
    "    fileArray.append(perTotal)\n",
    "\n",
    "    index=-1\n",
    "    perSiteArray, SlideArray, NoSlideArray = [] , [], []\n",
    "    for file in fileArray:\n",
    "        index += 1\n",
    "        inFile = open(file,'r')\n",
    "        for line in inFile:\n",
    "            if 'Chrom' in line or 'CHROM' in line:\n",
    "                continue\n",
    "            line = line.rstrip().split('\\t')\n",
    "            if 'PerSite' in file:\n",
    "                chrom,start,end,Z = line[0],int(line[1]),int(line[1]),float(line[3])\n",
    "            else:\n",
    "                chrom,start,end, Z = line[0],int(line[1]),int(line[2]),float(line[9])\n",
    "            #Now check if the call is within the ROI\n",
    "            if chrom == ROIchrom and start >= ROIstart and end <= ROIend:\n",
    "                if '50kbSlide' in file: #OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    SlideArray.append([chrom,midpoint,Z])\n",
    "                    continue\n",
    "                if 'PerSite' in file: #PER SITE WINDOW FILE\n",
    "                    perSiteArray.append([chrom,start,Z])\n",
    "                    continue\n",
    "                else: #NO OVERLAP WINDOW FILE\n",
    "                    midpoint = float(start + end)/2\n",
    "                    NoSlideArray.append([chrom,midpoint,Z])\n",
    "                    continue                \n",
    "    return perSiteArray, SlideArray, NoSlideArray\n",
    "###################################################################\n",
    "def simpleaxis(ax): #takes away the right and top borders/axes of a plot or subplot\n",
    "    ax.spines['top'].set_visible(False)\n",
    "    ax.spines['right'].set_visible(False)\n",
    "    ax.get_xaxis().tick_bottom()\n",
    "    ax.get_yaxis().tick_left()\n",
    "###################################################################\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "### Wolves_v_Dogs ####\n",
      "\n"
     ]
    },
    {
     "ename": "AttributeError",
     "evalue": "'dict' object has no attribute 'append'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mAttributeError\u001b[0m                            Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-19-d89a3737c7ec>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[1;32m      8\u001b[0m \u001b[0mbedFile\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mbedfile\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m'w'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 10\u001b[0;31m \u001b[0mtotalData\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mread_xpclr_file\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     11\u001b[0m \"\"\"df, pct99, pct995, scoreMax = basic_stats(totalData)\n\u001b[1;32m     12\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m<ipython-input-18-eed74f91dfe2>\u001b[0m in \u001b[0;36mread_xpclr_file\u001b[0;34m()\u001b[0m\n\u001b[1;32m     30\u001b[0m         \u001b[0mXPCLR_score\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mfloat\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     31\u001b[0m         \u001b[0mscores\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mXPCLR_score\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 32\u001b[0;31m         \u001b[0mtotalData\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mchrom\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mchrom\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mgridNum\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0msnpCount\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mphysical_pos\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mgenetic_pos\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mXPCLR_score\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mmax_s\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     33\u001b[0m         \u001b[0;31m#Writing this information to a bedfile\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     34\u001b[0m         \u001b[0mbedFile\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwrite\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'%s\\t%s\\t%s\\t%s_%s\\n'\u001b[0m \u001b[0;34m%\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mchrom\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mphysical_pos\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mphysical_pos\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mchrom\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0mcount\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mAttributeError\u001b[0m: 'dict' object has no attribute 'append'"
     ]
    }
   ],
   "source": [
    "#In/Out files\n",
    "#subDir = 'Dogs_v_Wolves'\n",
    "subDir = 'Wolves_v_Dogs'\n",
    "print('\\n### %s ####\\n' % subDir)\n",
    "infile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/Merged_VillageWolf_XPCLR_CM_window50kb_gridSize2kb.xpclr.txt'\n",
    "inFile = open(infile,'r')\n",
    "bedfile = infile.replace('.txt','.bed')\n",
    "bedFile = open(bedfile,'w')\n",
    "\n",
    "totalData = read_xpclr_file()\n",
    "\"\"\"df, pct99, pct995, scoreMax = basic_stats(totalData)\n",
    "\n",
    "#Draw score distributions\n",
    "plotDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/XP-CLR/results/No_MAF_Filter/plots/'\n",
    "plotFile = plotDir + subDir + '_Score_SNPCount_Distributions_50kbWindow_2kbGridSize' + '_XP-CLR.png'\n",
    "draw_hist(plotFile, pct99)\n",
    "draw_Manhattan_Plot(totalData,scoreMax)\n",
    "\n",
    "#Find significant hits\n",
    "sigThreshold = round(float(pct995),3)\n",
    "sigHits = find_sig_windows(sigThreshold, totalData)\n",
    "mergedWindows = merge_sig_windows(sigHits)\n",
    "merged3Windows = filter_merged_windows(mergedWindows)\n",
    "###############################\n",
    "#Writing out\n",
    "outfile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/SignificantWindows/' +  'Merged_SignificantWindows_' + subDir + '_XP-CLR_50kbWindow_2kbGrid.txt'\n",
    "outFile = open(outfile,'w')\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    #print('\\t'.join(map(str,merged3Windows[i])) + '\\n')\n",
    "    outFile.write('\\t'.join(map(str,merged3Windows[i])) + '\\n')\n",
    "outFile.close()\n",
    "###############################\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def draw_triple_plot(ROIchrom,ROI_CDR,dataArray, coord, plotFile):\n",
    "    #Three subplots, the axes array is 1-d\n",
    "    f, axarr = plt.subplots(3, sharex=True, figsize=(18,12))\n",
    "    ########################################\n",
    "    ####PLOT 1 - Per Site SNP Fst\n",
    "    x, y = [], []\n",
    "    for i in range(0,len(perSiteArray)):\n",
    "        x.append(perSiteArray[i][1])\n",
    "        y.append(perSiteArray[i][2])\n",
    "    colors=y\n",
    "    sc = axarr[0].scatter(x, y,c=colors, cmap=plt.cm.jet, vmin=0, vmax=10, alpha=0.8)\n",
    "    axarr[0].set_title('XP-CLR Window Scores and Per Site FST scores in region of interest %s\\n%s' % (ROI_CDR,coord))\n",
    "    simpleaxis(axarr[0])\n",
    "    axarr[0].set_ylabel('Per Site Z-FST', fontsize =18)\n",
    "    axarr[0].hlines(y=5, xmin=np.min(x), xmax=np.max(x),lw=1,color=\"red\",linestyles='dashed')\n",
    "    #y-axis\n",
    "    axarr[0].set_ylim(0,10)\n",
    "    #Add color bar\n",
    "    f.subplots_adjust(right=1.8)\n",
    "    cbar_ax = f.add_axes([1.0, 0.71, 0.02, 0.23])\n",
    "    f.colorbar(sc, cax=cbar_ax)\n",
    "    ########################################\n",
    "    ####PLOT 2- XP-CLR Scores\n",
    "    scoreList = []\n",
    "    for i in range(0,len(dataArray)):\n",
    "        x, y, width = dataArray[i][1], dataArray[i][2], 1500\n",
    "        scoreList.append(y)\n",
    "        if y > pct995:\n",
    "            axarr[1].bar(x, y, width, color=\"red\",linewidth=1)\n",
    "        else:\n",
    "            axarr[1].bar(x, y, width, color=\"black\",fill=False,linewidth=1)\n",
    "    simpleaxis(axarr[1])\n",
    "    axarr[1].set_ylabel('XP-CLR Score', fontsize=18)\n",
    "    axarr[1].set_ylim(0,np.max(scoreList)+10)\n",
    "    ########################################\n",
    "    ####PLOT 3 - GENE MODELS\n",
    "    #Add CDR  ROI window\n",
    "    CDR = [ROIstart+210000,ROIend-210000]\n",
    "    axarr[2].add_patch(patches.Rectangle((CDR[0], 9.4),(CDR[1]-CDR[0]),12,fill=True,color=\"grey\"))\n",
    "    axarr[2].text(((ROIstart+ROIend)/2)-13000, 9.6,str(ROI_CDR),fontsize=14)\n",
    "\n",
    "    IDheights, geneheights, index, heights =[2.5,3.5,4.5,5.5,6.5,7.5], [3,4,5,6,7,8], -1, []\n",
    "    for gene in genesOfInterest:\n",
    "        xRange = []\n",
    "        #determines height to print the genes and gene IDs\n",
    "        if index == 5:\n",
    "            index = 0\n",
    "        else:\n",
    "            index += 1\n",
    "        for i in range(0,len(genePos)):\n",
    "            if gene == genePos[i][0]:\n",
    "                x, direction, length = genePos[i][1], genePos[i][3], genePos[i][2]-genePos[i][1]\n",
    "                axarr[2].add_patch(patches.Rectangle((x,geneheights[index]),length,0.5,facecolor=\"black\"))\n",
    "                xRange.append(x)\n",
    "        xMin, xMax = min(xRange), max(xRange)\n",
    "        if direction == '-':\n",
    "            axarr[2].plot(xMin-2000, geneheights[index]+0.25, marker='<',color=\"blue\",markersize=10)\n",
    "        if direction == '+':\n",
    "            axarr[2].plot(xMax+2000, geneheights[index]+0.25, marker='>',color=\"blue\",markersize=10)\n",
    "        axarr[2].hlines(y=geneheights[index]+0.25, xmin=xMin, xmax=xMax,lw=1,color=\"black\")\n",
    "        if 'ENSCAFG' in gene:#ADDING GENE TEXT\n",
    "            gene = gene.replace('ENSCAFG','ENS')\n",
    "        geneMidpoint = ((xMax+xMin)/2)-10000\n",
    "        if geneMidpoint < ROIend-10000: #Make sure the gene ID doesn't get printed off the plot\n",
    "            axarr[2].text(geneMidpoint,geneheights[index]-0.4,gene,fontsize=14)\n",
    "        heights.append(geneheights[index])\n",
    "    axarr[2].axes.get_yaxis().set_ticks([])\n",
    "    axarr[2].set_ylim(+1.75,+10)\n",
    "    simpleaxis(axarr[2])\n",
    "    axarr[2].set_ylabel('Ensembl Gene Models', fontsize=18, labelpad=22)\n",
    "    axarr[2].set_xlabel('Chromosome %s (bp)' % (ROIchrom.replace('chr','')) , fontsize=20) #rotate yaxis label and move over from axis ticks\n",
    "    axarr[2].ticklabel_format(useOffset=False,style='plain',fontsize=18)#turn off sci-notation in  xaxis labelling\n",
    "\n",
    "    #Draw plot\n",
    "    f.tight_layout()\n",
    "    f.savefig(plotFile)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Plotting individual merged, significant windows (with at least 3 windows)\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    ROIchrom, ROIstart, ROIend, ROI_CDR = merged3Windows[i][0],int(merged3Windows[i][1]),int(merged3Windows[i][2]),merged3Windows[i][3]\n",
    "    print('#%s' % ROI_CDR)\n",
    "    coord = ROIchrom + ':' + str(ROIstart) + '-' + str(ROIend)\n",
    "    ROIstart, ROIend = ROIstart - 200000, ROIend + 200000\n",
    "    print ('Looking for the Fst information for these coordinates:',ROIchrom,ROIstart,ROIend)\n",
    "    #Finding genes in the region\n",
    "    genesOfInterest,genePos = find_gene_pos()\n",
    "\n",
    "    print ('\\nGenes of Interest in Region of Interest:\\n', genesOfInterest)\n",
    "\n",
    "    dataArray = get_xpclr_score(totalData,ROIchrom,ROIstart,ROIend)\n",
    "    print('%i windows found within the region of interest (%s)' % (len(dataArray),ROI_CDR))\n",
    "    perSiteArray, SlideArray, NoSlideArray = get_fst()\n",
    "    print('%i per site FST values in the region of interest (%s)' % (len(perSiteArray),ROI_CDR) )\n",
    "\n",
    "    #Draw per window plots\n",
    "    plotFile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/SignificantWindows/' + ROI_CDR + '_LocusPlot.png'\n",
    "    draw_triple_plot(ROIchrom, ROI_CDR, dataArray, coord, plotFile)\n",
    "print('DONE!')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Plotting individual merged, significant windows (with at least 3 windows)\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    ROIchrom, ROIstart, ROIend, ROI_CDR = merged3Windows[i][0],int(merged3Windows[i][1]),int(merged3Windows[i][2]),merged3Windows[i][3]\n",
    "    print('#%s' % ROI_CDR)\n",
    "    coord = ROIchrom + ':' + str(ROIstart) + '-' + str(ROIend)\n",
    "    ROIstart, ROIend = ROIstart - 200000, ROIend + 200000\n",
    "    print ('Looking for the Fst information for these coordinates:',ROIchrom,ROIstart,ROIend)\n",
    "    #Finding genes in the region\n",
    "    genesOfInterest,genePos = find_gene_pos()\n",
    "\n",
    "    print ('\\nGenes of Interest in Region of Interest:\\n', genesOfInterest)\n",
    "\n",
    "    dataArray = get_xpclr_score(totalData,ROIchrom,ROIstart,ROIend)\n",
    "    print('%i windows found within the region of interest (%s)' % (len(dataArray),ROI_CDR))\n",
    "    perSiteArray, SlideArray, NoSlideArray = get_fst()\n",
    "    print('%i per site FST values in the region of interest (%s)' % (len(perSiteArray),ROI_CDR) )\n",
    "    \n",
    "    #Draw per window plots\n",
    "    plotFile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/SignificantWindows/' + ROI_CDR + '_LocusPlot.png'\n",
    "    draw_triple_plot(ROIchrom, ROI_CDR, dataArray, coord, plotFile)\n",
    "    break\n",
    "print('DONE!')  \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#############################################################################\n",
    "#def draw_plot(ROIchrom,ROI_CDR,dataArray,pct995, coord):\n",
    "    fig, ax1 = plt.subplots(1,1,figsize=(14,8))\n",
    "\n",
    "    #plotArray, barColor = determine_sig(NoSlideArray,SlideArray)\n",
    "    #PLOT LIMITS\n",
    "    yMaximum = 9\n",
    "    yMinimum = -9\n",
    "    ax1.ticklabel_format(useOffset=False,style='plain')#turn off sci-notation in  xaxis labelling\n",
    "\n",
    "    #PLOTTING PER SITE Fst SNPs\n",
    "    x, y = [], []\n",
    "    for i in range(0,len(perSiteArray)):\n",
    "        x.append(perSiteArray[i][1])\n",
    "        y.append(perSiteArray[i][2])\n",
    "    colors=y\n",
    "    ax1.scatter(x, y ,c=colors, cmap=plt.cm.gist_earth, alpha=0.3)\n",
    "    ax1.set_ylabel('Fst Z-Score', fontsize=18)\n",
    "    #ax2.set_ylim(-4,10)\n",
    "\n",
    "    scoreList = []\n",
    "    ax2 = ax1.twinx()\n",
    "    #PLOTTING XP-CLR SLIDING WINDOWS\n",
    "    for i in range(0,len(dataArray)):\n",
    "        x, y, width = dataArray[i][1], dataArray[i][2], 1500\n",
    "        scoreList.append(dataArray[i][2])\n",
    "        if y > pct995:\n",
    "            ax2.bar(x, y, width, color=\"red\",linewidth=1)\n",
    "        else:\n",
    "            ax2.bar(x, y, width, color=\"black\",fill=False,linewidth=1)\n",
    "    maxScore = np.max(scoreList)\n",
    "    ax2.set_ylim(-4,maxScore + (.1*maxScore)) \n",
    "    ax2.set_ylabel('XP-CLR Score', fontsize=18, rotation=270, labelpad=20) #rotate yaxis label and move over from axis ticks\n",
    "    ax2.ticklabel_format(useOffset=False,style='plain')#turn off sci-notation in  xaxis labelling\n",
    "\n",
    "\n",
    "    plt.title('XP-CLR Window Scores and Per Site FST scores in region of interest %s (%s)' % (ROI_CDR, coord))\n",
    "    \n",
    "    \n",
    "    \n",
    "    plt.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#############################################################################\n",
    "#def draw_plot(ROIchrom,ROI_CDR,dataArray,pct995):\n",
    "fig, ax1 = plt.subplots(1,1,figsize=(14,6))\n",
    "\n",
    "#plotArray, barColor = determine_sig(NoSlideArray,SlideArray)\n",
    "#PLOT LIMITS\n",
    "yMaximum = 9\n",
    "yMinimum = -9\n",
    "ax1.ticklabel_format(useOffset=False,style='plain')#turn off sci-notation in  xaxis labelling\n",
    "\n",
    "scoreList = []\n",
    "#PLOTTING XP-CLR SLIDING WINDOWS\n",
    "for i in range(0,len(dataArray)):\n",
    "    x, y, width = dataArray[i][1], dataArray[i][2], 1500\n",
    "    scoreList.append(dataArray[i][2])\n",
    "    if y > pct995:\n",
    "        ax1.bar(x, y, width, color=\"blue\",alpha=0.9,linewidth=1)\n",
    "    else:\n",
    "        plt.bar(x, y, width, fill=False,alpha=0.6,linewidth=1)\n",
    "maxScore = np.max(scoreList)\n",
    "ax1.set_ylim(-(.1*maxScore),maxScore + (.1*maxScore))\n",
    "\n",
    "#PLOTTING PER SITE Fst SNPs\n",
    "ax2 = ax1.twinx()\n",
    "for i in range(0,len(perSiteArray)):\n",
    "    x,y =perSiteArray[i][1],perSiteArray[i][2]\n",
    "    if y > 5:\n",
    "        ax2.scatter(x,y,facecolor='red',edgecolor='k',s=20)\n",
    "    if y < 5 and y > 0:\n",
    "        ax2.scatter(x,y,color='black',alpha=0.8,s=10)\n",
    "ax2.set_ylim(-(10*.1),10)\n",
    "#DEFINING AXIS LIMS\n",
    "#plt.ylim(yMinimum,yMaximum+0.5)\n",
    "#plt.xlim(ROIstart,ROIend)\n",
    "#ax = plt.axes()\n",
    "\n",
    "\"\"\"\n",
    "ax3 = ax1.twinx()\n",
    "\n",
    "#DRAwING GENE MODELS\n",
    "###mp = []\n",
    "IDheights=[-3.5,-4.5,-5.5,-6.5,-7.5,-8.5]\n",
    "geneheights=[-3,-4,-5,-6,-7,-8]\n",
    "index = -1\n",
    "for gene in genesOfInterest:\n",
    "    xRange = []\n",
    "    #determines height to print the genes and gene IDs\n",
    "    if index == 5:\n",
    "        index = 0\n",
    "    else:\n",
    "        index += 1\n",
    "    for i in range(0,len(genePos)):\n",
    "        if gene == genePos[i][0]:\n",
    "            x = genePos[i][1]\n",
    "            direction = genePos[i][3]\n",
    "            length = genePos[i][2]-genePos[i][1]\n",
    "            #ax.add_patch(patches.Rectangle((x, -3),length,0.75,facecolor=\"black\"))\n",
    "            ax3.add_patch(patches.Rectangle((x,geneheights[index]),length,0.75,facecolor=\"black\"))\n",
    "            xRange.append(x)\n",
    "    xMin,xMax = min(xRange),max(xRange)\n",
    "    if direction == '-':\n",
    "        ax3.plot(xMin-0.25, geneheights[index]+0.4, marker='<',color=\"blue\")\n",
    "    if direction == '+':\n",
    "        ax3.plot(xMax+0.25, geneheights[index]+0.4, marker='>',color=\"blue\")\n",
    "    ax3.hlines(y=geneheights[index]+0.4, xmin=xMin, xmax=xMax,lw=1,color=\"black\")\n",
    "    #ADDING GENE TEXT\n",
    "    if 'ENSCAFG' in gene:\n",
    "        gene = gene.replace('ENSCAFG','ENS')\n",
    "    geneMidpoint = ((xMax+xMin)/2)-10000\n",
    "    if geneMidpoint < ROIend-10000: #Make sure the gene ID doesn't get printed off the plot\n",
    "        #ax.text(xMax+2500,geneheights[index]+0.3,gene,fontsize=7)\n",
    "        print (geneMidpoint,geneheights[index]-0.4)\n",
    "        ax3.text(geneMidpoint,geneheights[index]-0.4,gene,fontsize=7)\"\"\"\n",
    "\n",
    "        \n",
    "\"\"\"#ADDING TITLES\n",
    "plt.xlabel('Chromosome %s Position' % (ROIchrom.replace(\"chr\",\"\")),fontsize=20)\n",
    "plt.ylabel(r'$ZF_{ST}$',fontsize=20)\n",
    "plt.title('Per Site and Window ZFst for %s' % ROI_CDR)\n",
    "#Add CDR window\n",
    "CDR = [ROIstart+210000,ROIend-210000]\n",
    "if 'CDR20' in ROI_CDR:\n",
    "    CDR = [ROIstart+100000,ROIend-100000]\n",
    "ax.add_patch(patches.Rectangle((CDR[0], -1.75),(CDR[1]-CDR[0]),0.5,fill=None))\n",
    "ax.text(((ROIstart+ROIend)/2-10000), -1.7,str(ROI_CDR))\n",
    "ax.ticklabel_format(useOffset=False,style='plain')#turn off sci-notation in  xaxis labelling\n",
    "#plt.savefig(plotDir + ROI_CDR + 'Locus_ZFstPlotWithGeneModels.png')\"\"\"\n",
    "#############################################################################"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#In/Out files\n",
    "subDir = 'Wolves_v_Dogs'\n",
    "print('\\n### %s ####\\n' % subDir)\n",
    "infile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/Merged_WolfVillage_XPCLR_CM_window50kb_gridSize2kb.xpclr.txt'\n",
    "inFile = open(infile,'r')\n",
    "bedfile = infile.replace('.txt','.bed')\n",
    "bedFile = open(bedfile,'w')\n",
    "\n",
    "#For keeping track of data\n",
    "totalData = []\n",
    "scores,chromProcessed = [], []\n",
    "totalGrids,infCounts,failed_SNPZero,failed_SNPMin = 0, 0, 0, 0\n",
    "\n",
    "for line in inFile:\n",
    "    line=line.rstrip().split(' ')\n",
    "    if 'chrom' in line[0]:#skips header\n",
    "        continue\n",
    "    chrom = 'chr' + line[0]\n",
    "    #Naming the sweep is based on chromosome and the window number on that chromosome\n",
    "    if chrom in chromProcessed:\n",
    "        count += 1\n",
    "    else:\n",
    "        chromProcessed.append(chrom)\n",
    "        count = 1\n",
    "    #processing line for more data\n",
    "    gridNum, snpCount = int(line[1]),int(line[2])\n",
    "    physical_pos,genetic_pos,max_s = round(float(line[3])),float(line[4]),float(line[6])\n",
    "    XPCLR_score = line[5]\n",
    "    totalGrids+=1\n",
    "    #keeping track of certain data points\n",
    "    if 'inf' in XPCLR_score:\n",
    "        infCounts+=1\n",
    "        continue\n",
    "    if snpCount < 1:\n",
    "        failed_SNPZero += 1\n",
    "        continue\n",
    "    if snpCount < 5:\n",
    "        failed_SNPMin += 1\n",
    "        continue\n",
    "    XPCLR_score=float(line[5])\n",
    "    scores.append(XPCLR_score)\n",
    "    totalData.append([chrom,gridNum,snpCount,physical_pos,genetic_pos,XPCLR_score,max_s])\n",
    "    #Writing this information to a bedfile\n",
    "    bedFile.write('%s\\t%s\\t%s\\t%s_%s\\n' % (chrom,physical_pos-1,physical_pos,chrom,count))\n",
    "inFile.close()\n",
    "################################################################\n",
    "print('%s grids were processed by XP-CLR' % ((totalGrids)))\n",
    "print('##EXCLUSIONS')\n",
    "print('%s of the grids had scores equal to infinity and were excluded until we determine what that means (failure??)' % infCounts)\n",
    "print('%s of the grids had no SNPs and were removed' % failed_SNPZero)\n",
    "print('%s of the grids had fewer than 5 SNPs and were removed' % failed_SNPMin)\n",
    "print('##FOR PROCESSING')\n",
    "print('%s of the grids remained for further processing after passing filtrations' % len(totalData))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###COMPARE WITH FST/VST sweeps\n",
    "print('\\n### %s ####\\n' % subDir)\n",
    "\n",
    "cdrfile = '/home/ampend/links/kidd-lab/ampend-projects/Angela/Merged_Final_CDRs_VCDRs/Total_CDR_VCDR.bed'\n",
    "bedOut = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/Intersect_With_CDR_VCDR.txt'\n",
    "\n",
    "cmd = 'bedtools intersect -a %s -b %s > %s' % (cdrfile,bedfile,bedOut)\n",
    "runCMD(cmd)\n",
    "print(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#Save full MAF array into a pandas dataframe\n",
    "df = pd.DataFrame(totalData,columns=['Chromosome','Grid','SNPCount','GenPos','cMPos','Score','MaxScore'])\n",
    "df.head()\n",
    "\n",
    "################################################\n",
    "myPlot = df['Score'].plot(kind='hist', bins=350,xlim=(0,50),alpha=0.75, normed = True)\n",
    "myPlot.set_xlabel('XP-CLR Score')\n",
    "myPlot.set_title('All Chromosomes - XP-CLR Score Distribution')\n",
    "plt.vlines(x=pct99,ymin=0,ymax=0.4, lw=1,color=\"red\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\n",
    "################################################\n",
    "myPlot = df['SNPCount'].plot(kind='hist', bins=10,alpha=0.75, normed = True)\n",
    "myPlot.set_xlabel('XP-CLR SNP Count')\n",
    "myPlot.set_title('All Chromosomes - Per Grid SNP Counts')\n",
    "avg_SNPCount = df['SNPCount'].mean()\n",
    "plt.vlines(x=avg_SNPCount,ymin=0,ymax=0.1, lw=1,color=\"red\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "df = pd.DataFrame(totalData,columns=['Chromosome','Grid','SNPCount','GenPos','cMPos','Score','MaxScore'])\n",
    "df.Chromosome = df.Chromosome.astype('category')\n",
    "df.Chromosome = df.Chromosome.cat.set_categories(chromToDo, ordered=True)\n",
    "df = df.sort_values('Chromosome')\n",
    "\n",
    "df.head()\n",
    "# How to plot gene vs. -log10(pvalue) and colour it by chromosome?\n",
    "df['ind'] = range(len(df))\n",
    "df_grouped = df.groupby(('Chromosome'))\n",
    "\n",
    "###PLOTTING MANHATTAN PLOT \n",
    "plotDir = '/home/ampend/links/kidd-lab/ampend-projects/Angela/XP-CLR/results/No_MAF_Filter/Window50kb_GridSize2kb/plots'\n",
    "fig = plt.figure(figsize=(24,12))\n",
    "ax = fig.add_subplot(111)\n",
    "colors = ['black','gray']\n",
    "x_labels = []\n",
    "x_labels_pos = []\n",
    "for num, (name, group) in enumerate(df_grouped):\n",
    "    group.plot(kind='scatter', x='ind', y='Score',color=colors[num % len(colors)], ax=ax)\n",
    "    x_labels.append(name)\n",
    "    x_labels_pos.append((group['ind'].iloc[-1] - (group['ind'].iloc[-1] - group['ind'].iloc[0])/2))\n",
    "ax.set_xticklabels(ax.xaxis.get_majorticklabels(), rotation=45)\n",
    "ax.set_xticks(x_labels_pos)\n",
    "ax.set_xticklabels(x_labels)\n",
    "plt.hlines(y=pct995,xmin=0,xmax=len(df), lw=1,color=\"red\")\n",
    "plt.hlines(y=pct999,xmin=0,xmax=len(df), lw=1,color=\"red\")\n",
    "ax.set_xlim([0, len(df)])\n",
    "ax.set_ylim([0, np.max(scores)+10])\n",
    "ax.set_xlabel('Chromosome')\n",
    "#plt.savefig(plotDir + 'Wolves_v_Dogs_50kbWindow_2kbGridSize' + '_XP-CLR.png')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "print('######  Wolves versus Dogs ######')\n",
    "sigThreshold = round(float(pct995),3)\n",
    "print(sigThreshold)\n",
    "\n",
    "sigHits = []\n",
    "\n",
    "for i in range(0,len(totalData)):\n",
    "    chrom, score = totalData[i][0], round(float(totalData[i][5]),3)\n",
    "    if score > sigThreshold:\n",
    "        sigHits.append(totalData[i])\n",
    "print('%i significant windows found from XP-CLR'%(len(sigHits)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "chromList, count, sigCount = [], 0, -1\n",
    "CHROM, WINDOW, SCORE = 0, 0, 0\n",
    "mergedWindows = {}\n",
    "\n",
    "for i in range(0,len(sigHits)):\n",
    "    chrom, num, pos, score = sigHits[i][0], sigHits[i][1], round(sigHits[i][3]),float(sigHits[i][5])\n",
    "    start, end = pos-25000, pos+25000 #start/end of the sliding window is 25kb from either side of this grid point\n",
    "    winID, count = chrom + '_' + str(num), count +1\n",
    "    \n",
    "    #keeping track of chromosomes analyzed, helps save information automatically for first call on a chromosome\n",
    "    #   for further comparisons with the next call\n",
    "    if chrom not in chromList:\n",
    "        chromList.append(chrom)\n",
    "        CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "        sigCount += 1\n",
    "        mergedWindows[sigCount] = []\n",
    "        mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "        #print('FIRST',chrom, num, pos, start, end, score)\n",
    "        continue\n",
    "    #processing calls\n",
    "    if CHROM == chrom and num - NUM < 2: #can only have one window between them\n",
    "        #print('YES',chrom, num, pos, start, end, score)\n",
    "        CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "        mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "        CHROM, NUM, START,END = chrom, num, start, end\n",
    "\n",
    "    else:\n",
    "        #print('NO',chrom, num, pos, start, end, score)\n",
    "        sigCount += 1\n",
    "        CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "        mergedWindows[sigCount] = []\n",
    "        mergedWindows[sigCount].append([CHROM, NUM, START, END, SCORE])\n",
    "        \n",
    "    CHROM, NUM, START,END,SCORE = chrom, num, start, end, score\n",
    "\n",
    "print('%i merged windows found to be significant' % sigCount)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#def filter_merged_windows(mergedWindows):\n",
    "merged3Windows, count = [], 0\n",
    "\n",
    "for i in mergedWindows:\n",
    "    if len(mergedWindows[i]) < 3:\n",
    "        continue\n",
    "    count += 1 #keeping track of how many merged windows with at least 3 sig windows are present\n",
    "    chrom, start = mergedWindows[i][0][0], mergedWindows[i][0][2] #chrom and start can be defined by the first indexed window in the list\n",
    "    scores = []\n",
    "    for j in mergedWindows[i]:\n",
    "        end = j[3] #end will always be the last window read through the loop, constantly overwriting\n",
    "        scores.append(j[4]) #to keep track of for calculating the average score\n",
    "    meanScore = format(round(np.mean(scores),3)) #formats and rounds the average scores per merged window\n",
    "    winLength = end - start\n",
    "    ID = \n",
    "    merged3Windows.append([chrom,start,end,ID,winLength,meanScore]) #now save all the information\n",
    "\n",
    "\n",
    "print('%i windows contained three or more adjacent windows and were merged' % count)  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "outfile = resultsDir + 'No_MAF_Filter/' + 'Window50kb_GridSize2kb/' + subDir + '/SignificantWindows/' +  'Merged_SignificantWindows_WolfvsDog_XP-CLR_50kbWindow_2kbGrid.txt'\n",
    "outFile = open(outfile,'w')\n",
    "\n",
    "\n",
    "for i in range(0,len(merged3Windows)):\n",
    "    #print('\\t'.join(map(str,merged3Windows[i])) + '\\n')\n",
    "    outFile.write('\\t'.join(map(str,merged3Windows[i])) + '\\n')\n",
    "\n",
    "outFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
