{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "\n",
    "import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#Working directory for the Vst analyses is the following:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/\n"
     ]
    }
   ],
   "source": [
    "vstTypes = ['FastCN', 'QuicKmer']\n",
    "vstDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/'\n",
    "print '#Working directory for the Vst analyses is the following:\\n', vstDir\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#Reading in final FST CDR regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed\n"
     ]
    }
   ],
   "source": [
    "##############\n",
    "# INPUT FST FILE\n",
    "##############\n",
    "fstfile = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed'\n",
    "print '\\n#Reading in final FST CDR regions from:\\n', fstfile\n",
    "fstFile = open(fstfile, 'r')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def make_new_Vst_files(vstType, vstDir):\n",
    "    vstfile = vstDir + 'input/' + vstType + '_candidates_5.bed'\n",
    "    print '#Reading in candidate VST regions from:\\n', vstfile\n",
    "    vstFile = open(vstfile, 'r')\n",
    "    #open new bed file to store information and rename the windows\n",
    "    newVstbedfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.bed'\n",
    "    print 'Writing out new window IDs and modifying file structure for the following BED file:\\n', newVstbedfile\n",
    "    newVstbedFile = open(newVstbedfile, 'w')\n",
    "    #open new text file to store information and rename the windows that matches the bed file\n",
    "    newVstoutfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.txt'\n",
    "    print 'Writing out new window IDs and modifying file structure for the following TEXT file:\\n', newVstoutfile\n",
    "    newVstoutFile = open(newVstoutfile, 'w')\n",
    "\n",
    "    vstCount = 0\n",
    "    vstDict = {}\n",
    "\n",
    "    for line in vstFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        vstCount += 1\n",
    "\n",
    "        chrom = line[0]\n",
    "        start = line[1]\n",
    "        end = line[2]\n",
    "        zScore = line[3]\n",
    "        winID = vstType + '_' + str(vstCount)\n",
    "\n",
    "        newVstbedFile.write('%s\\t%s\\t%s\\t%s\\n' % (chrom, start,end, winID))\n",
    "        #text file needs a 1-based start coordinate\n",
    "        newStart = int(start) + 1\n",
    "        newVstoutFile.write('%s\\t%s\\t%s\\t%s\\t%s\\n' % (chrom, newStart,end, winID, zScore))\n",
    "\n",
    "    print 'Wrote new information for %i VST windows to outfile' % (vstCount)\n",
    "    vstFile.close()\n",
    "    newVstbedFile.close()\n",
    "    newVstoutFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Created final table of the 139 CDRs based on QuicKmer and fastCN VST analysis\n"
     ]
    }
   ],
   "source": [
    "def chrom_loc_compare(item1, item2):\n",
    "    if item1[0] == item2[0]:\n",
    "        if int(item1[1]) > int(item2[1]):\n",
    "            return 1\n",
    "        else:\n",
    "            return -1\n",
    "    else:\n",
    "        try:\n",
    "            chrom_num1 = int(item1[0][3:])\n",
    "        except Exception:\n",
    "            return 1\n",
    "        try:\n",
    "            chrom_num2 = int(item2[0][3:])\n",
    "        except Exception:\n",
    "            return -1\n",
    "        if chrom_num1 > chrom_num2:\n",
    "            return 1\n",
    "        else:\n",
    "            return -1\n",
    "#Sorting the array based on the subroutine outlined above\n",
    "cdrarr = sorted(cdrarr, cmp=chrom_loc_compare)\n",
    "\n",
    "#Now re-naming the windows to CDR + integer based on the ordered array\n",
    "for i,item in enumerate(cdrarr):\n",
    "    cdrarr[i][3]='CDR_'+str(i+1)\n",
    "    \n",
    "print 'Created final table of the %i CDRs based on QuicKmer and fastCN VST analysis' % len(cdrarr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_with_FST_CDRs(vstType):\n",
    "    for i in vstTypes:\n",
    "        vstType = i\n",
    "        print '###Vst Type: ', vstType\n",
    "        #New VST bedfile for intersecting\n",
    "        newVstbedfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.bed'\n",
    "\n",
    "        #Intersecting the vst BED file with the FST results\n",
    "        outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "        print '#Intersecting the FST and VST windows directly...'\n",
    "        cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "        print cmd\n",
    "        genutils.runCMD(cmd)\n",
    "        #checking window counts\n",
    "        f = outfile\n",
    "        count = count_lines(f)\n",
    "        print '-->%i Vst windows found to intersect FST regions' % count  \n",
    "\n",
    "        #Intersecting the vst BED file with the FST results using window approach\n",
    "        #50kb windows\n",
    "        outfile = vstDir + 'results/' + '50kbWindowIntersect_'+ vstType + '_Vst_with_FstCDRs.txt'\n",
    "        print '\\n#Intersecting the FST and VST windows with 50kb buffering windows...'\n",
    "        cmd = 'bedtools window -w 50000 -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "        print cmd\n",
    "        genutils.runCMD(cmd)\n",
    "        #checking window counts\n",
    "        f = outfile\n",
    "        count = count_lines(f)\n",
    "        print '-->%i Vst windows found to intersect FST regions with window size = 50kb' % count\n",
    "\n",
    "        #200kb windows\n",
    "        outfile = vstDir + 'results/' + '200kbWindowIntersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "        print '\\n#Intersecting the FST and VST windows with 200kb buffering windows...'\n",
    "        cmd = 'bedtools window -w 50000 -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "        print cmd\n",
    "        genutils.runCMD(cmd)\n",
    "        #checking window counts\n",
    "        f = outfile\n",
    "        count = count_lines(f)\n",
    "        print '-->%i Vst windows found to intersect FST regions with window size = 200kb' % count\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_fastCN_with_QuicKmer(vstDir):\n",
    "    #Now intersect VST candidates from QuicKmer with FASTCN\n",
    "    fastcnfile = vstDir + 'input/' + 'Mod_FastCN_candidates_5.bed'\n",
    "    quickmerfile = vstDir + 'input/' + 'Mod_QuicKmer_candidates_5.bed'\n",
    "\n",
    "    #Intersecting command:\n",
    "    outfile = vstDir + 'results/Intersect_FastCN_with_QuicKmer_Regions.txt'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (fastcnfile, quickmerfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd) \n",
    "\n",
    "    inFile = open(outfile, 'r')\n",
    "    #checking window counts\n",
    "    f = outfile\n",
    "    count = count_lines(f)\n",
    "    print '-->%i FastCN Vst windows were found to precisely intersect QuicKmer FST regions (i.e. no buffering window of overlap)' % count\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def find_CDRs(vstDir):\n",
    "    ###Read FastCN results into dictionary\n",
    "    fastcnfile = vstDir + 'input/' + 'Mod_FastCN_candidates_5.txt'\n",
    "    fastcnFile = open(fastcnfile,'r')\n",
    "\n",
    "    fastDict = {}\n",
    "\n",
    "    for line in fastcnFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        chrom = line[0]\n",
    "        start = int(line[1]) #1-based coordinate\n",
    "        end = int(line[2])\n",
    "        ID = line[3]\n",
    "        Z = float(line[4])\n",
    "        fastDict[ID] = [chrom,start,end,ID,'NA', Z,'fastCN']\n",
    "    print '#Finished reading fastCN window coordinates for %i windows' % (len(fastDict))\n",
    "\n",
    "    ###Read QuicKmer results into dictionary\n",
    "    quickmerfile = vstDir + 'input/' + 'Mod_QuicKmer_candidates_5.txt'\n",
    "    quickmerFile = open(quickmerfile,'r')\n",
    "    quickmerDict = {}\n",
    "\n",
    "    for line in quickmerFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        chrom = line[0]\n",
    "        start = int(line[1]) #1-based coordinate\n",
    "        end = int(line[2])\n",
    "        ID = line[3]\n",
    "        Z = float(line[4])\n",
    "        quickmerDict[ID] = [chrom,start,end,ID,'NA', Z,'Quic-Kmer']\n",
    "    print '\\n#Finished reading quickmer window coordinates for %i windows' % (len(quickmerDict))\n",
    "\n",
    "    ###Processing the intersection file to create list of unique copy-number CDRs\n",
    "    intfile = vstDir + 'results/Intersect_FastCN_with_QuicKmer_Regions.txt'\n",
    "    print '\\n#Reading in intersection results from:\\n', intfile\n",
    "    intFile = open(intfile, 'r')\n",
    "\n",
    "    intDict = {}\n",
    "    noList = []\n",
    "\n",
    "    winCount = 0\n",
    "\n",
    "    for line in intFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        winCount += 1 \n",
    "\n",
    "        chrom = line[0]\n",
    "        start1 = int(line[1])\n",
    "        end1 = int(line[2])\n",
    "        ID1 = line[3]\n",
    "        start2 = int(line[5])\n",
    "        end2 = int(line[6])\n",
    "        ID2 = line[7]\n",
    "\n",
    "        minStart = min([start1,start2])\n",
    "        maxEnd = max([end1, end2])\n",
    "\n",
    "        window = chrom + ':' + str(minStart) + '-' + str(maxEnd)\n",
    "        if window in intDict:\n",
    "            continue\n",
    "        else:\n",
    "            intDict[window] = [chrom, minStart, maxEnd, 'NA', 'NA', 'Both']\n",
    "        #Add the IDs of the intersecting windows to the \"no list\", meaning that we want to use the coordinates\n",
    "        #   we just determined based on the intersecting coordinates (minStart and maxStart)\n",
    "        #Note: these MAY be the same coordinates as the original fastCN or QuicKmer results, but we just want to use\n",
    "        #   what we just calculated instead.\n",
    "        noList.append(ID1) #Will not be using this window ID in our final set, will be using the minStarts and maxEnds\n",
    "        noList.append(ID2) #Will not be using this window ID in our final set, will be using the minStarts and maxEnds\n",
    "\n",
    "    print '\\n#Finished reading in the %i intersecting windows between QuicKmer and fastCN' % (winCount)\n",
    "    print 'There are now %i entries in the intersect dictionary' % (len(intDict))\n",
    "\n",
    "    #Now we need to determine the final coordinates of the CDRs\n",
    "    cdrDict = {}\n",
    "    cdrarr = []\n",
    "    #Add the fastCN window information for all windows NOT in the \"No list\" AKA those windows that did not intersect\n",
    "    #    with QuicKmer windows\n",
    "    for i in fastDict:\n",
    "        ID = i\n",
    "        if ID not in noList:\n",
    "            cdrDict[ID] = fastDict[ID]\n",
    "            cdrarr.append(fastDict[ID])\n",
    "    #Add the QuicKmer window information for all windows NOT in the \"No list\" AKA those windows that did not intersect\n",
    "    #    with fastCN windows\n",
    "    for i in quickmerDict:\n",
    "        ID = i\n",
    "        if ID not in noList:\n",
    "            cdrDict[ID] = quickmerDict[ID]\n",
    "            cdrarr.append(quickmerDict[ID])\n",
    "    #Now add the remaining windows which are the windows that resulted from the intersect analysis:\n",
    "    for i in intDict:\n",
    "        ID = i\n",
    "        cdrDict[ID] = intDict[ID]\n",
    "        cdrarr.append(intDict[ID])\n",
    "\n",
    "    print '\\n#Renaming the CDR regions from sorted CDR array'\n",
    "    \n",
    "    for i in cdrarr:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[['chr1', 98731163, 98734162, 'CDR_1', 'NA', 'NA', 'fastCN'], ['chr2', 4744801, 4749424, 'CDR_2', 2.7085449528200001, 'NA', 'fastCN'], ['chr2', 4753379, 4757311, 'CDR_3', 1.11281065981, 'NA', 'fastCN'], ['chr2', 5893665, 5897044, 'CDR_4', 'NA', 'NA', 'fastCN'], ['chr2', 5901157, 5905516, 'CDR_5', 'NA', 'NA', 'fastCN'], ['chr2', 69825432, 69838692, 'CDR_6', 'NA', 'NA'], ['chr2', 82411420, 82416607, 'CDR_7', 'NA', 'NA', 'fastCN'], ['chr3', 58219595, 58223305, 'CDR_8', 'NA', 'NA', 'fastCN'], ['chr5', 3352778, 3357464, 'CDR_9', 'NA', 'NA'], ['chr5', 28078011, 28081577, 'CDR_10', 'NA', 'NA', 'fastCN'], ['chr5', 41925909, 41930008, 'CDR_11', 2.7845478076200001, 'NA', 'fastCN'], ['chr5', 42899318, 42903237, 'CDR_12', 'NA', 'NA', 'fastCN'], ['chr5', 43169600, 43172693, 'CDR_13', 'NA', 'NA', 'fastCN'], ['chr5', 78144490, 78166450, 'CDR_14', 'NA', 'NA', 'fastCN'], ['chr5', 78182013, 78360039, 'CDR_15', 'NA', 'NA', 'fastCN'], ['chr5', 78374146, 78390852, 'CDR_16', 'NA', 'NA', 'fastCN'], ['chr6', 15410542, 15413725, 'CDR_17', 'NA', 'NA', 'fastCN'], ['chr6', 46945639, 46957719, 'CDR_18', 'NA', 'NA', 'fastCN'], ['chr6', 47182974, 47193718, 'CDR_19', 'NA', 'NA', 'fastCN'], ['chr7', 10032909, 10036813, 'CDR_20', 'NA', 'NA', 'fastCN'], ['chr8', 1966353, 1970437, 'CDR_21', 'NA', 'NA', 'fastCN'], ['chr8', 7599605, 7622135, 'CDR_22', 'NA', 'NA', 'fastCN'], ['chr8', 53758374, 53763681, 'CDR_23', 'NA', 'NA', 'fastCN'], ['chr9', 7321283, 7384855, 'CDR_24', 'NA', 'NA', 'fastCN'], ['chr9', 7516162, 7641414, 'CDR_25', 'NA', 'NA', 'fastCN'], ['chr9', 7661854, 7737525, 'CDR_26', 'NA', 'NA', 'fastCN'], ['chr9', 7754408, 7938737, 'CDR_27', 'NA', 'NA'], ['chr9', 8544167, 8561524, 'CDR_28', 'NA', 'NA'], ['chr9', 8544167, 8562714, 'CDR_29', 'NA', 'NA'], ['chr9', 8600560, 8951324, 'CDR_30', 'NA', 'NA'], ['chr9', 8631901, 8951324, 'CDR_31', 'NA', 'NA'], ['chr9', 8964753, 8984145, 'CDR_32', 'NA', 'NA', 'fastCN'], ['chr9', 10330211, 10337545, 'CDR_33', 1.04378489319, 'NA', 'fastCN'], ['chr9', 10342434, 10371497, 'CDR_34', 'NA', 'NA'], ['chr9', 16641260, 16680137, 'CDR_35', 'NA', 'NA', 'fastCN'], ['chr9', 16744383, 16859364, 'CDR_36', 'NA', 'NA', 'fastCN'], ['chr9', 16877305, 16985787, 'CDR_37', 'NA', 'NA'], ['chr9', 16882744, 16893682, 'CDR_38', 'NA', 'NA', 'Quic-Kmer'], ['chr9', 16939109, 16948988, 'CDR_39', 'NA', 'NA', 'Quic-Kmer'], ['chr9', 16952546, 16955720, 'CDR_40', 5.2557528768599999, 'NA', 'Quic-Kmer'], ['chr9', 16959107, 16962209, 'CDR_41', 5.0998676338899998, 'NA', 'Quic-Kmer'], ['chr9', 16972400, 16975654, 'CDR_42', 5.1382104795799997, 'NA', 'Quic-Kmer'], ['chr9', 16991374, 16995520, 'CDR_43', 'NA', 'NA', 'fastCN'], ['chr9', 17224289, 17227519, 'CDR_44', 'NA', 'NA', 'fastCN'], ['chr9', 17381439, 17393534, 'CDR_45', 1.1447108835699999, 'NA', 'fastCN'], ['chr9', 17416057, 17506784, 'CDR_46', 'NA', 'NA', 'fastCN'], ['chr9', 17566298, 17569992, 'CDR_47', 'NA', 'NA', 'fastCN'], ['chr9', 17582186, 17586099, 'CDR_48', 'NA', 'NA', 'fastCN'], ['chr9', 17603233, 17623006, 'CDR_49', 'NA', 'NA', 'fastCN'], ['chr9', 17637687, 17642940, 'CDR_50', 'NA', 'NA', 'fastCN'], ['chr9', 17652788, 17664805, 'CDR_51', 'NA', 'NA', 'fastCN'], ['chr9', 17686241, 17690525, 'CDR_52', 'NA', 'NA', 'fastCN'], ['chr9', 17694058, 17702422, 'CDR_53', 'NA', 'NA', 'fastCN'], ['chr9', 17721679, 17730002, 'CDR_54', 'NA', 'NA', 'fastCN'], ['chr9', 17733647, 17798515, 'CDR_55', 'NA', 'NA'], ['chr9', 17738690, 17748463, 'CDR_56', 'NA', 'NA', 'Quic-Kmer'], ['chr9', 17762280, 17768691, 'CDR_57', 'NA', 'NA', 'Quic-Kmer'], ['chr9', 17786017, 17789448, 'CDR_58', 5.0191296839800001, 'NA', 'Quic-Kmer'], ['chr9', 17868732, 17872488, 'CDR_59', 'NA', 'NA', 'fastCN'], ['chr9', 17893050, 17897179, 'CDR_60', 'NA', 'NA', 'fastCN'], ['chr9', 17914249, 17922786, 'CDR_61', 'NA', 'NA', 'fastCN'], ['chr9', 17954513, 17958303, 'CDR_62', 'NA', 'NA', 'fastCN'], ['chr9', 18004900, 18009743, 'CDR_63', 4.9829251320300001, 'NA', 'fastCN'], ['chr9', 18148901, 18214141, 'CDR_64', 'NA', 'NA', 'fastCN'], ['chr9', 18233653, 18239704, 'CDR_65', 'NA', 'NA', 'fastCN'], ['chr9', 20183024, 20186298, 'CDR_66', 'NA', 'NA', 'fastCN'], ['chr9', 38177034, 38185546, 'CDR_67', 'NA', 'NA', 'fastCN'], ['chr9', 51883569, 51900568, 'CDR_68', 'NA', 'NA', 'fastCN'], ['chr9', 55851423, 55854521, 'CDR_69', 'NA', 'NA', 'fastCN'], ['chr10', 18989872, 18993304, 'CDR_70', 'NA', 'NA', 'fastCN'], ['chr11', 74183981, 74188359, 'CDR_71', 'NA', 'NA', 'fastCN'], ['chr12', 45750870, 45769332, 'CDR_72', 'NA', 'NA', 'fastCN'], ['chr14', 6844795, 6850381, 'CDR_73', 'NA', 'NA', 'fastCN'], ['chr15', 42715924, 42719898, 'CDR_74', 'NA', 'NA', 'fastCN'], ['chr16', 1118089, 1124557, 'CDR_75', 'NA', 'NA', 'fastCN'], ['chr16', 1127558, 1134421, 'CDR_76', 'NA', 'NA', 'fastCN'], ['chr16', 1155946, 1162957, 'CDR_77', 'NA', 'NA', 'fastCN'], ['chr16', 10456224, 10462283, 'CDR_78', -0.60011255221500004, 'NA', 'fastCN'], ['chr16', 10465736, 10476308, 'CDR_79', 3.1483526240900002, 'NA', 'fastCN'], ['chr16', 10483609, 10487277, 'CDR_80', 0.27451187245100001, 'NA', 'fastCN'], ['chr16', 13902206, 13905521, 'CDR_81', -0.37789686668799999, 'NA', 'fastCN'], ['chr16', 13908392, 13928460, 'CDR_82', 'NA', 'NA'], ['chr16', 13932155, 13938219, 'CDR_83', 0.0151558026152, 'NA', 'fastCN'], ['chr17', 37832526, 37836601, 'CDR_84', 'NA', 'NA', 'fastCN'], ['chr18', 18393092, 18406430, 'CDR_85', 'NA', 'NA', 'fastCN'], ['chr18', 18446639, 18588928, 'CDR_86', 'NA', 'NA'], ['chr18', 18601849, 18844948, 'CDR_87', 'NA', 'NA', 'fastCN'], ['chr18', 25757830, 25762065, 'CDR_88', 0.39877720020700003, 'NA', 'fastCN'], ['chr18', 51296069, 51299256, 'CDR_89', 'NA', 'NA', 'fastCN'], ['chr19', 52801473, 52813734, 'CDR_90', 'NA', 'NA', 'fastCN'], ['chr19', 52868686, 52900889, 'CDR_91', 'NA', 'NA', 'fastCN'], ['chr19', 52905749, 52916566, 'CDR_92', 'NA', 'NA', 'fastCN'], ['chr19', 52919567, 52930961, 'CDR_93', 'NA', 'NA', 'fastCN'], ['chr21', 29926277, 29929486, 'CDR_94', 'NA', 'NA', 'fastCN'], ['chr22', 19365422, 19368634, 'CDR_95', 5.67274213439, 'NA', 'Quic-Kmer'], ['chr23', 20500362, 20734562, 'CDR_96', 'NA', 'NA'], ['chr23', 20558128, 20723036, 'CDR_97', 'NA', 'NA', 'Quic-Kmer'], ['chr24', 8966714, 8978459, 'CDR_98', 'NA', 'NA'], ['chr25', 48153141, 48159951, 'CDR_99', 'NA', 'NA', 'fastCN'], ['chr25', 48536876, 48540269, 'CDR_100', 'NA', 'NA', 'fastCN'], ['chr26', 28834087, 28918835, 'CDR_101', 'NA', 'NA'], ['chr26', 31206104, 31573989, 'CDR_102', 'NA', 'NA'], ['chr26', 31582014, 31610728, 'CDR_103', 'NA', 'NA', 'fastCN'], ['chr26', 31618477, 31622704, 'CDR_104', 'NA', 'NA', 'fastCN'], ['chr26', 37622598, 37627666, 'CDR_105', 'NA', 'NA', 'fastCN'], ['chr27', 2410260, 2414244, 'CDR_106', 'NA', 'NA', 'fastCN'], ['chr29', 7180204, 7191587, 'CDR_107', 'NA', 'NA', 'fastCN'], ['chr31', 28414018, 28417651, 'CDR_108', 'NA', 'NA', 'fastCN'], ['chr31', 39634858, 39638122, 'CDR_109', 'NA', 'NA', 'fastCN'], ['chr34', 10735324, 10739057, 'CDR_110', 'NA', 'NA', 'fastCN'], ['chr34', 10844974, 10848546, 'CDR_111', 'NA', 'NA', 'fastCN'], ['chr35', 23973122, 23976474, 'CDR_112', 'NA', 'NA', 'fastCN'], ['chr37', 5454011, 5459208, 'CDR_113', 'NA', 'NA', 'fastCN'], ['chrX', 2891287, 2927525, 'CDR_114', 'NA', 'NA'], ['chrX', 12470380, 12482733, 'CDR_115', 'NA', 'NA', 'fastCN'], ['chrX', 28809568, 28813047, 'CDR_116', 8.6781797234700004, 'NA', 'Quic-Kmer'], ['chrX', 32368102, 32375742, 'CDR_117', 'NA', 'NA', 'fastCN'], ['chrX', 43942451, 43947588, 'CDR_118', 'NA', 'NA', 'fastCN'], ['chrX', 48242879, 48311358, 'CDR_119', 6.6414745566400004, 'NA', 'Quic-Kmer'], ['chrX', 49446634, 49449649, 'CDR_120', 5.1554660962999996, 'NA', 'Quic-Kmer'], ['chrX', 51967338, 51970985, 'CDR_121', 'NA', 'NA', 'fastCN'], ['chrX', 54072270, 54081502, 'CDR_122', 'NA', 'NA'], ['chrX', 66534390, 66539880, 'CDR_123', 7.6138505188599996, 'NA', 'Quic-Kmer'], ['chrX', 67611784, 67615658, 'CDR_124', 5.5205169560499998, 'NA', 'Quic-Kmer'], ['chrX', 88850331, 88854576, 'CDR_125', 7.1058219902199999, 'NA', 'Quic-Kmer'], ['chrX', 105578453, 105581452, 'CDR_126', 'NA', 'NA', 'fastCN'], ['chrX', 105800970, 105803969, 'CDR_127', 1.50461848526, 'NA', 'fastCN'], ['chrX', 106787627, 106795003, 'CDR_128', 'NA', 'NA'], ['chrX', 117556917, 117562916, 'CDR_129', 'NA', 'NA', 'fastCN'], ['chrX', 117604750, 117608511, 'CDR_130', 2.5003152152800001, 'NA', 'fastCN'], ['chrX', 117819397, 117825644, 'CDR_131', -0.36980075027100001, 'NA', 'fastCN'], ['chrX', 118084885, 118095678, 'CDR_132', -0.22918752657399999, 'NA', 'fastCN'], ['chrX', 118304084, 118307626, 'CDR_133', 1.5930231072700001, 'NA', 'fastCN'], ['chrX', 120884790, 120888063, 'CDR_134', 'NA', 'NA', 'fastCN'], ['chrX', 121317116, 121320162, 'CDR_135', 5.1442224424600003, 'NA', 'Quic-Kmer'], ['chrX', 121582250, 121585504, 'CDR_136', 'NA', 'NA', 'fastCN'], ['chrX', 121712350, 121715349, 'CDR_137', 'NA', 'NA', 'fastCN'], ['chrX', 122489610, 122499387, 'CDR_138', 5.7318685838299999, 'NA'], ['chrX', 123857035, 123862384, 'CDR_139', 5.5088691253300004, 'NA', 'Quic-Kmer']]\n"
     ]
    }
   ],
   "source": [
    "print cdrarr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#1. Changing the input file types for fastCN and QuicKmer\n",
      "#Reading in candidate VST regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following BED file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following TEXT file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.txt\n",
      "Wrote new information for 37 VST windows to outfile\n",
      "\n",
      "#2. Now intersecting fastCN and QuicKmer regions to help in finding CDRs\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_FastCN_with_QuicKmer_Regions.txt\n",
      "-->29 FastCN Vst windows were found to precisely intersect QuicKmer FST regions (i.e. no buffering window of overlap)\n",
      "\n",
      "#3. Parsing fastCN/QuicKmer intersect to find CDRs\n",
      "#Finished reading fastCN window coordinates for 120 windows\n",
      "\n",
      "#Finished reading quickmer window coordinates for 37 windows\n",
      "\n",
      "#Reading in intersection results from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_FastCN_with_QuicKmer_Regions.txt\n",
      "\n",
      "#Finished reading in the 29 intersecting windows between QuicKmer and fastCN\n",
      "There are now 20 entries in the intersect dictionary\n",
      "\n",
      "#Renaming the CDR regions\n"
     ]
    }
   ],
   "source": [
    "####################################################\n",
    "#1. Generating new VST files with proper unique IDs \n",
    "####################################################\n",
    "print '\\n#1. Changing the input file types for fastCN and QuicKmer'\n",
    "make_new_Vst_files(vstType, vstDir)\n",
    "\n",
    "####################################################\n",
    "#2. Intersecting fastCN and QuicKmer regions to help identify VST CDRs\n",
    "####################################################\n",
    "print '\\n#2. Now intersecting fastCN and QuicKmer regions to help in finding CDRs'\n",
    "intersect_fastCN_with_QuicKmer(vstDir)\n",
    "\n",
    "####################################################\n",
    "#3. Identify VST CDRs\n",
    "####################################################\n",
    "print '\\n#3. Parsing fastCN/QuicKmer intersect to find CDRs'\n",
    "find_CDRs(vstDir)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_with_Ensembl_Genes(newVstbedfile):\n",
    "    print 'Now intersecting bedfiles...\\n'\n",
    "    #Determining variables\n",
    "    inBedFile = newVstbedfile\n",
    "    genebedfile = '/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed'\n",
    "    print 'Reading in the Ensembl 81 genes from the following file:\\n', genebedfile\n",
    "    geneBedFile = open(genebedfile, 'r')\n",
    "    outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt'\n",
    "\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (inBedFile, genebedfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_gene_intersect_file(intfile, vstType):\n",
    "    b2goDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/BLAST2GO/'\n",
    "    b2goList = open(b2goDir + vstType + '_GeneList.txt', 'w')\n",
    "    #Reading the intersect file results\n",
    "    intersectFile = open(intfile,'r') #outfile from the intersect step with ensembl 81 genes\n",
    "\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    ensGDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    ensID = ''\n",
    "    sigGenes = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        winID = line[3]\n",
    "        hit = line[7].split('_')\n",
    "        Protein = hit[0]\n",
    "        Gene = hit[1]\n",
    "        b2goID = Gene + '|' + Protein\n",
    "        ensGID = hit[2]\n",
    "        \n",
    "        if winID in ensGDict:\n",
    "            ensGDict[winID].append(ensGID)\n",
    "        else:\n",
    "            ensGDict[winID] = []\n",
    "            ensGDict[winID].append(ensGID) \n",
    "            sigGenes.append(b2goID)\n",
    "    intersectFile.close()\n",
    "    for i in sigGenes:\n",
    "        gene = i\n",
    "        b2goList.write('%s\\n' % gene)\n",
    "    \n",
    "    return ensGDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_FST_CDR_intersect_file():\n",
    "    #Reading the intersect file results to FST CDRs\n",
    "    outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "    intersectFile = open(outfile,'r') #outfile from the intersect step above\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    fstDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    fstID = ''\n",
    "    fstWindows = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        vstID = line[3]\n",
    "        fstID = line[7]\n",
    "\n",
    "        if vstID in fstDict:\n",
    "            fstDict[vstID].append(fstID)\n",
    "        else:\n",
    "            fstDict[vstID] = []\n",
    "            fstDict[vstID].append(fstID) \n",
    "    intersectFile.close()\n",
    "    return fstDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def making_summary_text_files(vstType,fstDict,ensGDict):\n",
    "    newVstoutfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.txt'\n",
    "    vstFile = open(newVstoutfile, 'r')\n",
    "\n",
    "    ############################################################################\n",
    "    # SUMMARIZING VST HITS WITH ENSEMBL GENES AND FST REGIONS\n",
    "    ##########################################################################\n",
    "    Outfile = vstDir + 'results/' + vstType + '_CDRs_WithFSTIntersects_GeneTable.txt' #Write out the VST Summary data table here\n",
    "    print '\\nWriting VST summary data table with FST intersects and gene IDs here: \\n', Outfile\n",
    "    outFile = open(Outfile, 'w')\n",
    "    #Defining header based on the structure of the dictionary you created:\n",
    "    #Add to this if you're adding more to your dictionary \n",
    "    headerLine = 'Chrom\\tStart\\tEnd\\tWindowID\\tWindowLength(bp)AverageZ-score\\tIntersecting_FST_CDR\\tIntersectingGenes\\n'\n",
    "    outFile.write(headerLine)\n",
    "\n",
    "    vstCount = 0\n",
    "    #Change this if you're working with the QuicKmer VST calls or the FastCN VST calls\n",
    "\n",
    "    WinID = '' #Setting equal to nothing\n",
    "    allDict = {} # defining the dictionary that will have all the results of bedtool intersect compiled by windows\n",
    "\n",
    "    for line in vstFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        vstCount += 1 #To use when naming the VST regions with unique identifiers\n",
    "\n",
    "        chrom = line[0]\n",
    "        start_pos = int(line[1])\n",
    "        end_pos = int(line[2])\n",
    "        WinID = line[3]\n",
    "        winLength = end_pos - start_pos\n",
    "        zscore = line[4] #TO ADD ONCE YOU GET FILE BACK FROM FEICHEN\n",
    "        winCoord = chrom + \":\" + str(start_pos) + \"-\" + str(end_pos)\n",
    "\n",
    "        #Open up dictionary for WinID\n",
    "        #0=WinID, 1=chr, 2=start_pos, 3=end_pos, 4=Window length, #5=Zscore\n",
    "        #6=Intersects with an FST CDR (default = False), \n",
    "        #7=Intersects with Ensembl Gene IDs (default = False)\n",
    "        #8=window coordinates\n",
    "        allDict[WinID] = [chrom,start_pos,end_pos,WinID,winLength,zscore,False,False,winCoord] #setting up empty dictionary\n",
    "\n",
    "        #Did this VST window intersect with a FST CDR?\n",
    "        if WinID in fstDict: #If this window is in the ensembl list you generated, then...:\n",
    "            y = \", \".join(map(str, fstDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "            allDict[WinID][6] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "        #Did this VST window intersect with a gene?\n",
    "        if WinID in ensGDict: #If this window is in the ensembl list you generated, then...:\n",
    "            y = \", \".join(map(str, ensGDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "            allDict[WinID][7] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "    print 'There are now %i VST regions saved to the allDict dictionary for further processing' % (len(allDict))\n",
    "\n",
    "    #Writing out results\n",
    "    for keys in sorted(allDict.keys()): #Now loops through each \n",
    "        #print \"\\t\".join(map(str,allDict[keys]))\n",
    "        outFile.write(\"\\t\".join(map(str,allDict[keys])))\n",
    "        outFile.write(\"\\n\")\n",
    "    outFile.close()\n",
    "    vstFile.close()\n",
    "    return allDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def gene_fst_intersect_summary(allDict):\n",
    "    geneCount = 0\n",
    "    fstCount = 0\n",
    "    for keys in sorted(allDict.keys()):\n",
    "        if allDict[keys][6] is not False:\n",
    "            fstCount += 1\n",
    "        if allDict[keys][7] is not False:\n",
    "            geneCount += 1\n",
    "\n",
    "    print '%i windows intersect with FST CDRs' % (fstCount)\n",
    "    print '%i windows intersect with Ensembl genes' % (geneCount)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def make_gene_enrichment_list():\n",
    "    for winID in allDict.keys():\n",
    "        print "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "############\n",
      "##FastCN\n",
      "############\n",
      "\n",
      "Now intersecting VST regions with FST CDRs\n",
      "#Reading in candidate VST regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/FastCN_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following BED file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following TEXT file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.txt\n",
      "Wrote new information for 120 VST windows to outfile\n",
      "Now intersecting VST regions with FST CDRs\n",
      "#Intersecting the FST and VST windows directly...\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_FastCN_Vst_with_FstCDRs.txt\n",
      "-->6 Vst windows found to intersect FST regions\n",
      "\n",
      "#Intersecting the FST and VST windows with 50kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/50kbWindowIntersect_FastCN_Vst_with_FstCDRs.txt\n",
      "-->8 Vst windows found to intersect FST regions with window size = 50kb\n",
      "\n",
      "#Intersecting the FST and VST windows with 200kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/200kbWindowIntersect_FastCN_Vst_with_FstCDRs.txt\n",
      "-->8 Vst windows found to intersect FST regions with window size = 200kb\n",
      "Now intersecting VST regions with Ensembl 81 genes\n",
      "Now intersecting bedfiles...\n",
      "\n",
      "Reading in the Ensembl 81 genes from the following file:\n",
      "/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_FastCN_VSTWindows_Ensembl81Genes.txt\n",
      "Now parsing Ensembl gene intersect file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_FastCN_VSTWindows_Ensembl81Genes.txt\n",
      "{'FastCN_18': ['MAGI2'], 'FastCN_35': ['MLPH'], 'FastCN_36': ['ENSCAFG00000012363'], 'FastCN_37': ['ENSCAFG00000014142'], 'FastCN_31': ['KAZN'], 'FastCN_32': ['TPP1'], 'FastCN_19': ['MAGI2'], 'FastCN_17': ['MAGI2'], 'FastCN_73': ['ENSCAFG00000015160'], 'FastCN_38': ['ENSCAFG00000028972', 'TOP3B'], 'FastCN_75': ['ENSCAFG00000032099'], 'FastCN_11': ['ENSCAFG00000030664'], 'FastCN_52': ['PDGFD'], 'FastCN_53': ['ENSCAFG00000018534', 'PEMT'], 'FastCN_50': ['ENSCAFG00000029887'], 'FastCN_55': ['SERBP1'], 'FastCN_59': ['TMEM184A'], 'FastCN_96': ['ENSCAFG00000029973'], 'FastCN_97': ['ENSCAFG00000013770'], 'FastCN_107': ['AR'], 'FastCN_106': ['CXorf67'], 'FastCN_105': ['LANCL3'], 'FastCN_77': ['ENSCAFG00000032512'], 'FastCN_103': ['ENSCAFG00000032113', 'ENSCAFG00000028745', 'ENSCAFG00000031750'], 'FastCN_102': ['FAM129B'], 'FastCN_99': ['AOC2', 'ENSCAFG00000014638'], 'FastCN_4': ['ENSCAFG00000003032'], 'FastCN_5': ['ENSCAFG00000001499'], 'FastCN_7': ['ADCY1'], 'FastCN_1': ['BICD2'], 'FastCN_8': ['ADCY1'], 'FastCN_9': ['ADCY1'], 'FastCN_23': ['ENSCAFG00000032415', 'ENSCAFG00000005717'], 'FastCN_22': ['ENSCAFG00000005717'], 'FastCN_21': ['SART1'], 'FastCN_26': ['ENSCAFG00000023337'], 'FastCN_24': ['ENSCAFG00000032415'], 'FastCN_63': ['CHGA'], 'FastCN_29': ['ENSCAFG00000003859'], 'FastCN_41': ['MINPP1'], 'FastCN_43': ['ENSCAFG00000007005'], 'FastCN_42': ['KRT4'], 'FastCN_45': ['ENSCAFG00000032318'], 'FastCN_44': ['ADAMTS7'], 'FastCN_46': ['PCNT'], 'FastCN_49': ['HIST1H1A'], 'FastCN_60': ['ENSCAFG00000028653'], 'FastCN_120': ['ENSCAFG00000019588'], 'FastCN_118': ['BCAP31', 'PLXNB3'], 'FastCN_119': ['L1CAM'], 'FastCN_81': ['ENSCAFG00000031850'], 'FastCN_112': ['ENSCAFG00000029639', 'ENSCAFG00000031344'], 'FastCN_115': ['ENSCAFG00000028611', 'ENSCAFG00000023067'], 'FastCN_116': ['ENSCAFG00000030952'], 'FastCN_117': ['ENSCAFG00000019178']}\n",
      "Now parsing the FST CDR intersect file...\n",
      "Now generating summary text files for FastCN results...\n",
      "\n",
      "Writing VST summary data table with FST intersects and gene IDs here: \n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/FastCN_CDRs_WithFSTIntersects_GeneTable.txt\n",
      "There are now 120 VST regions saved to the allDict dictionary for further processing\n",
      "Now finding how many genes and CDR regions intersect with the FastCN results...\n",
      "6 windows intersect with FST CDRs\n",
      "55 windows intersect with Ensembl genes\n",
      "\n",
      "############\n",
      "##QuicKmer\n",
      "############\n",
      "\n",
      "Now intersecting VST regions with FST CDRs\n",
      "#Reading in candidate VST regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following BED file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following TEXT file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.txt\n",
      "Wrote new information for 37 VST windows to outfile\n",
      "Now intersecting VST regions with FST CDRs\n",
      "#Intersecting the FST and VST windows directly...\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_QuicKmer_Vst_with_FstCDRs.txt\n",
      "-->2 Vst windows found to intersect FST regions\n",
      "\n",
      "#Intersecting the FST and VST windows with 50kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/50kbWindowIntersect_QuicKmer_Vst_with_FstCDRs.txt\n",
      "-->2 Vst windows found to intersect FST regions with window size = 50kb\n",
      "\n",
      "#Intersecting the FST and VST windows with 200kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/200kbWindowIntersect_QuicKmer_Vst_with_FstCDRs.txt\n",
      "-->2 Vst windows found to intersect FST regions with window size = 200kb\n",
      "Now intersecting VST regions with Ensembl 81 genes\n",
      "Now intersecting bedfiles...\n",
      "\n",
      "Reading in the Ensembl 81 genes from the following file:\n",
      "/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed -b /home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_QuicKmer_VSTWindows_Ensembl81Genes.txt\n",
      "Now parsing Ensembl gene intersect file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_QuicKmer_VSTWindows_Ensembl81Genes.txt\n",
      "{'QuicKmer_8': ['VPREB1', 'ENSCAFG00000014142', 'ENSCAFG00000014144'], 'QuicKmer_9': ['ENSCAFG00000028972'], 'QuicKmer_1': ['ENSCAFG00000028561'], 'QuicKmer_2': ['MAGI2'], 'QuicKmer_28': ['ENSCAFG00000031570', 'ENSCAFG00000016517'], 'QuicKmer_15': ['ENSCAFG00000032099'], 'QuicKmer_32': ['DACH2'], 'QuicKmer_26': ['ENSCAFG00000032113', 'ENSCAFG00000028745'], 'QuicKmer_36': ['ENSCAFG00000019588']}\n",
      "Now parsing the FST CDR intersect file...\n",
      "Now generating summary text files for QuicKmer results...\n",
      "\n",
      "Writing VST summary data table with FST intersects and gene IDs here: \n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/QuicKmer_CDRs_WithFSTIntersects_GeneTable.txt\n",
      "There are now 37 VST regions saved to the allDict dictionary for further processing\n",
      "Now finding how many genes and CDR regions intersect with the QuicKmer results...\n",
      "2 windows intersect with FST CDRs\n",
      "9 windows intersect with Ensembl genes\n"
     ]
    }
   ],
   "source": [
    "for i in vstTypes:\n",
    "    vstType = i\n",
    "    print '\\n############\\n##%s\\n############\\n' % (vstType)\n",
    "    intfile = vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt'\n",
    "    \n",
    "    ###############################\n",
    "    #2. Intersecting with FST CDRs\n",
    "    ###############################\n",
    "    print 'Now intersecting VST regions with FST CDRs'\n",
    "    intersect_with_FST_CDRs(vstType)\n",
    "    \n",
    "    ####################################\n",
    "    #3. Intersecting with Ensembl Genes\n",
    "    ####################################\n",
    "    print 'Now intersecting VST regions with Ensembl 81 genes'\n",
    "    newVstbedfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.bed'\n",
    "    intersect_with_Ensembl_Genes(newVstbedfile)\n",
    "\n",
    "    ########################################################\n",
    "    #4. Parse bedtools intersect file against Ensembl genes\n",
    "    ########################################################\n",
    "    print 'Now parsing Ensembl gene intersect file:\\n', intfile\n",
    "    ensGDict = parse_gene_intersect_file(intfile, vstType)\n",
    "    print ensGDict\n",
    "\n",
    "    ###################################################\n",
    "    #5. Parse bedtools intersect file against FST CDRs\n",
    "    ###################################################\n",
    "    print 'Now parsing the FST CDR intersect file...'\n",
    "    fstDict = parse_FST_CDR_intersect_file()\n",
    "    \n",
    "    ###########################################################\n",
    "    #6. Summarizing VST and Ensembl hits to output text files\n",
    "    ###########################################################\n",
    "    print 'Now generating summary text files for %s results...' % (vstType)\n",
    "    allDict = making_summary_text_files(vstType,fstDict,ensGDict)    \n",
    "    \n",
    "    ##################################################\n",
    "    #7. How many genes and FST CDRs are intersecting?\n",
    "    ##################################################\n",
    "    print 'Now finding how many genes and CDR regions intersect with the %s results...' % (vstType)\n",
    "    gene_fst_intersect_summary(allDict)\n",
    "    \n",
    "    ##################################################\n",
    "    #8. Make gene list for GO gene enrichment steps\n",
    "    ##################################################\n",
    "    #print 'Now writing outfile with the gene IDs in these windows to search for enriched GO categories...'\n",
    "    #make_gene_enrichment_list()\n",
    "    \n",
    "    \n",
    "    \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "CDR_1 98731163 98734162\n"
     ]
    }
   ],
   "source": [
    "#################\n",
    "#Finding Zscores for CDRs\n",
    "#################\n",
    "\n",
    "#VST bedfiles that have Z-scores for all windows with at least 1.5 CN range (filtered VST window set)\n",
    "fastcnfile = vstDir + 'input/' + 'FastCN_zscore_15CN_regions.bed' #'Mod_FastCN_candidates_5.txt'\n",
    "quickmerfile = vstDir + 'input/' + 'QuicKmer_zscore_15CN_regions.bed' #'Mod_QuicKmer_candidates_5.txt'\n",
    "\n",
    "#Saving these bedfiles to an array\n",
    "fileArray = [fastcnfile, quickmerfile]\n",
    "\n",
    "#Now we need to assign the Z-scores to the windows\n",
    "for i,item in enumerate(cdrarr):\n",
    "    chrom = cdrarr[i][0]\n",
    "    start = int(cdrarr[i][1])\n",
    "    end = int(cdrarr[i][2])\n",
    "    WinID = cdrarr[i][3]\n",
    "    #print 'Analyzing this window now...:\\n', WinID\n",
    "    #Reads through each file\n",
    "    index = 0\n",
    "    zscore1 = []\n",
    "    zscore2 = []\n",
    "    print WinID, start, end\n",
    "    break\n",
    "    for file in fileArray:\n",
    "        #SET 1 SNPS ARE INDICES 1 & 2\n",
    "        #SET 2 SNPS ARE INDICES 3 & 4\n",
    "        index += 1 \n",
    "        #open the file\n",
    "        inFile = open(file, 'r')\n",
    "        #reading through the Zscore file\n",
    "        for line in inFile:\n",
    "            line=line.rstrip()\n",
    "            line=line.split('\\t')\n",
    "            if 'CHROM' in line:\n",
    "                continue\n",
    "            CHROM = line[0]\n",
    "            START = int(line[1])\n",
    "            END = int(line[2])\n",
    "            ROA_Z = float(line[3])\n",
    "            #Check: Does this chromosome match the CDR window chrom?\n",
    "            if CHROM != chrom:\n",
    "                continue #skip if the chromosomes dont match to the chr in CDR regions\n",
    "            #Else, it's skipped\n",
    "            #Now check start and end coordinates\n",
    "            if start >= START and end <= END:\n",
    "                if index == 1 or index == 2:\n",
    "                    zscore1.append(ROA_Z)\n",
    "                if index == 3 or index == 4:\n",
    "                    zscore2.append(ROA_Z)\n",
    "                print WinID,start,START,end,END,ROA_Z\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "    #If no SNPs were found in these windows then the lengths will \n",
    "    #\tstill be equal to zero. Just set the values in the dictionary to 'NA'\n",
    "    #\tand continue through the loop without calculating their means\n",
    "    if len(zscore1) == 0: #SNP set 1\n",
    "        cdrarr[i][4] = 'NA'\n",
    "        #allDict[WinID][] = 'NA'\n",
    "    if len(zscore2) == 0: #SNP set 2less \n",
    "        cdrarr[i][5] = 'NA'\n",
    "        #allDict[WinID][11] = 'NA'\n",
    "    #PROCESS SNP SET 1\n",
    "    if len(zscore1) > 0:\n",
    "        #Calculate the MEAN Zscores for the SNP1 and SNP2 sets\n",
    "        #mean_Z1 = np.mean(zscore1) \n",
    "        #Now save these means to the dictionary in their respective indices \n",
    "        #cdrarr[i][4] = mean_Z1\n",
    "        #Calculate the MAXIMUM Zscores for the SNP1 and SNP2 sets\n",
    "        max_Z1 = np.max(zscore1) \n",
    "        #Now save these means to the dictionary in their respective indices \n",
    "        cdrarr[i][4] = max_Z1\n",
    "    #DO THE SAME AS ABOVE FOR SNP SET 2\n",
    "    if len(zscore2) > 0:\n",
    "        #mean_Z2 = np.mean(zscore2)\n",
    "        #cdrarr[i][4] = mean_Z2\n",
    "        max_Z2 = np.max(zscore2)\n",
    "        cdrarr[i][5] = max_Z2\n",
    "    break\n",
    "print cdrarr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###########################\n",
    "#Creating bedfile\n",
    "##########################\n",
    "#INFILE\n",
    "infile = options.input\n",
    "inFile = open(infile, 'r')\n",
    "print '\\nInfile of significant windows is:\\n ', options.input\n",
    "#OUTFILE\n",
    "newName = infile.replace('.txt','')\n",
    "outfile = newName + '.bed'\n",
    "print '\\nBedfile is being written here:\\n', outfile\n",
    "outFile = open(outfile, 'w')\n",
    "\n",
    "for line in inFile:\n",
    "    #Skip header line\t\n",
    "    if 'Chrom' in line:\n",
    "        continue\n",
    "\n",
    "    line = line.rstrip()\n",
    "    line = line.split()\n",
    "\n",
    "    chrom = line[0]\n",
    "    start = int(line[1]) - 1\n",
    "    end = int(line[2])\n",
    "    windowID = line[3]\n",
    "\n",
    "    outFile.write('%s\\t%i\\t%i\\t%s\\n'%(chrom,start,end,windowID))\n",
    "outFile.close()\n",
    "inFile.close()\n",
    "\n",
    "\n",
    "\n",
    "###########################\n",
    "#Intersecting \n",
    "###########################\n",
    "print '\\nNow intersecting bedfiles...\\n'\n",
    "#Determining variables\n",
    "inBedFile = outfile\n",
    "geneBedFile = open(options.geneBed, 'r')\n",
    "outfile = options.outdir + 'Intersect_' + 'SIGNonOverlappingWindows_' + 'FST_Ensembl81Genes.txt'\n",
    "\n",
    "cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (inBedFile, options.geneBed, outfile)\n",
    "print cmd\n",
    "genutils.runCMD(cmd)       \n",
    "\n",
    "###############################\n",
    "#Parse bedtools intersect file \n",
    "###############################\n",
    "#Reading the intersect file results\n",
    "intersectFile = open(outfile,'r') #outfile from the intersect step above\n",
    "# going through Ensembl bedtool intersect results file\n",
    "ensGDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "ensID = ''\n",
    "sigGenes = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "for line in intersectFile:\n",
    "    line = line.rstrip() #removing extraneous whitespace characters\n",
    "    line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "    winID = line[3]\n",
    "    hit = line[7].split('_')\n",
    "    Gene = hit[1]\n",
    "    ensGID = hit[2]\n",
    "    sigGenes.append(Gene)\n",
    "    if winID in ensGDict:\n",
    "        ensGDict[winID].append(ensGID)\n",
    "    else:\n",
    "        ensGDict[winID] = []\n",
    "        ensGDict[winID].append(ensGID) \n",
    "intersectFile.close()\n",
    "\n",
    "############################################################################\n",
    "# Input the bed file to make the dictionary\n",
    "# The dictionary will have all outlier Fst windows, regardless if there are genes in those\n",
    "#\twindows\n",
    "newName = options.input.replace('.txt', '')\n",
    "Outfile = newName + '_GeneTable.txt' #Write out the new FST data here\n",
    "print '\\nWriting new FST data table with gene IDs here: \\n', Outfile\n",
    "outFile = open(Outfile, 'w')\n",
    "\n",
    "#Defining header based on the structure of the dictionary you created:\n",
    "headerLine = 'Chrom\\tCDR_Start\\tCDR_End\\tCDR_ID\\tCDR_Length\\tSet\\tGenes\\tCDR_Coordinates\\tSNP_Set1_OverlappingWindows_MeanZscore\\t\\tSNP_Set1_OverlappingWindows_MaxZscore\\tSNP_Set2_OverlappingWindows_MeanZscore\\tSNP_Set2_OverlappingWindows_MaxZscore\\n'\n",
    "outFile.write(headerLine)\n",
    "\n",
    "WinID = '' #Setting equal to nothing\n",
    "allDict = {} # defining the dictionary that will have all the results of bedtool intersect compiled by windows\n",
    "\n",
    "windowCount = 0\n",
    "inBedFile = open(options.input, 'r')\n",
    "\n",
    "for line in inBedFile:\n",
    "    if 'Chrom' in line: #skips header, if present\n",
    "        continue\n",
    "    line = line.rstrip() #removing extraneous whitespace characters\n",
    "    line = line.split('\\t') #delimiting \"columns\" in the file based on tabs\t\t\n",
    "    windowCount += 1 \n",
    "\n",
    "    chr = line[0]\n",
    "    start_pos = line[1]\n",
    "    end_pos = line[2]\n",
    "    WinID = line[3]\n",
    "    winLength = line[4]\n",
    "    callset = line[5]\n",
    "    winCoord = chr + \":\" + start_pos + \"-\" + end_pos\n",
    "    #Open up dictionary for WinID\n",
    "    #0=WinID, 1=chr, 2=start_pos, 3=end_pos, 4=zscore,\n",
    "    #5 = CDR length\n",
    "    #6=Ensembl Gene IDs (default = False)\n",
    "    #7 = Window coordinates in one string (for ease of display in tables for manuscript)\n",
    "    #8 = TOTAL SNPs (SET 1) - Mean ROA Zscore - Overlapping(Sliding) Window\n",
    "    #9 = TOTAL SNPs (SET 1) - Maximum ROA Zscore - Overlapping(Sliding) Window\n",
    "    #10 = CALLSET 3 SNPs (SET 2) - Mean ROA Zscore - Overlapping(Sliding) Window\n",
    "    #11 = CALLSET 3 SNPs (SET 2) - Maximum ROA Zscore - Overlapping(Sliding) Window\n",
    "    #once I get the script working I will add more to the dictionary. For example,\n",
    "    # Broad bedtools intersect results\n",
    "    allDict[WinID] = [chr,start_pos,end_pos,WinID,winLength,callset,False,winCoord,'NA', 'NA', 'NA', 'NA'] #setting up empty dictionary\n",
    "\n",
    "    if WinID in ensGDict: #If this window is in the ensembl list you generated, then...:\n",
    "        y = \", \".join(map(str, ensGDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "        allDict[WinID][6] = y\n",
    "    #For when you add Broad for instance\n",
    "    #if WinID in broadGDict:\n",
    "    #\tsame thing as above\n",
    "    #\tnow you'd add to allDict[WindID][6] or [7]\n",
    "\n",
    "print allDict\n",
    "# Sort dictionary\n",
    "#sorted(allDict.keys())\t\n",
    "\n",
    "############################################################################\n",
    "#Writing the geneID + proteinID of all ensembl 81 genes that intersect\n",
    "#\twith significant windows\n",
    "####Input files:\n",
    "##GeneList that went into BLAST2GO pipeline\n",
    "#Format: [0] = ENSCAFP ID and [1] = ENSCAFG ID\n",
    "inFile = open('/home/jmkidd/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_GeneTables_WithEnscafIDsAndChrom.txt', 'r')\n",
    "\n",
    "#Saving gene-protein links that were processed with BLAST2GO to array\n",
    "protGene = {}\n",
    "\n",
    "for line in inFile:\n",
    "    line = line.rstrip()\n",
    "    line = line.split()\n",
    "    geneID = line[1]\n",
    "    protID = line[0]\n",
    "    protGene[geneID] = []\n",
    "    protGene[geneID] = protID\n",
    "print '\\n%i genes read into gene-protein ID array' % len(protGene)\n",
    "\n",
    "#FOR BLAST2GO ENRICHMENT FILES\n",
    "blast2goDir = '/home/jmkidd/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/BLAST2GO/'\n",
    "enrFile = open(blast2goDir + options.root + 'EnrichmentGeneSet.txt', 'w')\n",
    "\n",
    "#Goes through sig gene array from above and links the proteins and gene IDs together\n",
    "#\tto then write to the enrichment file that can be inputted into BLAST2GO\n",
    "print 'Writing significant genes to test for enrichment in BLAST2GO to this file: ', enrFile\n",
    "for i in sigGenes:\n",
    "    geneID = i\n",
    "    protID = protGene[i]\n",
    "    enrFile.write('%s|%s\\n' % (geneID, protID))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading in Z-scores from the following file\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/QuicKmer_zscore_15CN_regions.bed\n",
      "chr1\t81318\t183299\t-0.655461080375\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#MAKING UCSC TRACKS\n",
    "zfile = vstDir + 'input/' + vstType + '_zscore_15CN_regions.bed'\n",
    "print 'Reading in Z-scores from the following file\\n', zfile\n",
    "zFile = open(zfile, 'r')\n",
    "\n",
    "for line in zFile:\n",
    "    line = line.rstrip()\n",
    "    line = line.split('\\t')\n",
    "    chrom = line[0]\n",
    "    start = line[1]\n",
    "    end = line[2]\n",
    "    zScore = \n",
    "    print line\n",
    "    break"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [conda root]",
   "language": "python",
   "name": "conda-root-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
