{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import scipy.stats as stats\n",
    "import pandas as pd\n",
    "\n",
    "import genutils\n",
    "import os\n",
    "import sys\n",
    "import numpy as np\n",
    "import re\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#Working directory for the Vst analyses is the following:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/\n"
     ]
    }
   ],
   "source": [
    "vstTypes = ['FastCN', 'QuicKmer']\n",
    "vstDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/'\n",
    "print '#Working directory for the Vst analyses is the following:\\n', vstDir\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def make_new_Vst_files(vstDir):\n",
    "    for i in vstTypes:\n",
    "        vstType = i\n",
    "        print '###%s###' % vstType\n",
    "        \n",
    "        vstfile = vstDir + 'input/' + vstType + '_candidates_5.bed'\n",
    "        print '#Reading in candidate VST regions from:\\n', vstfile\n",
    "        vstFile = open(vstfile, 'r')\n",
    "        #open new bed file to store information and rename the windows\n",
    "        newVstbedfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.bed'\n",
    "        print 'Writing out new window IDs and modifying file structure for the following BED file:\\n', newVstbedfile\n",
    "        newVstbedFile = open(newVstbedfile, 'w')\n",
    "        #open new text file to store information and rename the windows that matches the bed file\n",
    "        newVstoutfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.txt'\n",
    "        print 'Writing out new window IDs and modifying file structure for the following TEXT file:\\n', newVstoutfile\n",
    "        newVstoutFile = open(newVstoutfile, 'w')\n",
    "\n",
    "        vstCount = 0\n",
    "        vstDict = {}\n",
    "\n",
    "        for line in vstFile:\n",
    "            line = line.rstrip()\n",
    "            line = line.split('\\t')\n",
    "            vstCount += 1\n",
    "\n",
    "            chrom = line[0]\n",
    "            start = line[1]\n",
    "            end = line[2]\n",
    "            zScore = line[3]\n",
    "            winID = vstType + '_' + str(vstCount)\n",
    "\n",
    "            newVstbedFile.write('%s\\t%s\\t%s\\t%s\\n' % (chrom, start,end, winID))\n",
    "            #text file needs a 1-based start coordinate\n",
    "            newStart = int(start) + 1\n",
    "            newVstoutFile.write('%s\\t%s\\t%s\\t%s\\t%s\\n' % (chrom, newStart,end, winID, zScore))\n",
    "\n",
    "        print 'Wrote new information for %i VST windows to outfile' % (vstCount)\n",
    "        vstFile.close()\n",
    "        newVstbedFile.close()\n",
    "        newVstoutFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_fastCN_with_QuicKmer(vstDir):\n",
    "    #Now intersect VST candidates from QuicKmer with FASTCN\n",
    "    fastcnfile = vstDir + 'input/' + 'Mod_FastCN_candidates_5.bed'\n",
    "    quickmerfile = vstDir + 'input/' + 'Mod_QuicKmer_candidates_5.bed'\n",
    "\n",
    "    #Intersecting command:\n",
    "    outfile = vstDir + 'results/Intersect_FastCN_with_QuicKmer_Regions.txt'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (fastcnfile, quickmerfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd) \n",
    "\n",
    "    inFile = open(outfile, 'r')\n",
    "    #checking window counts\n",
    "    f = outfile\n",
    "    count = count_lines(f)\n",
    "    print '-->%i FastCN Vst windows were found to precisely intersect QuicKmer FST regions (i.e. no buffering window of overlap)' % count\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def chrom_loc_compare(item1, item2):\n",
    "    if item1[0] == item2[0]:\n",
    "        if int(item1[1]) > int(item2[1]):\n",
    "            return 1\n",
    "        else:\n",
    "            return -1\n",
    "    else:\n",
    "        try:\n",
    "            chrom_num1 = int(item1[0][3:])\n",
    "        except Exception:\n",
    "            return 1\n",
    "        try:\n",
    "            chrom_num2 = int(item2[0][3:])\n",
    "        except Exception:\n",
    "            return -1\n",
    "        if chrom_num1 > chrom_num2:\n",
    "            return 1\n",
    "        else:\n",
    "            return -1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def find_CDRs(vstDir):\n",
    "    fastcnbed = vstDir + 'input/' + 'Mod_FastCN_candidates_5.bed'\n",
    "    quickmerbed = vstDir + 'input/' + 'Mod_QuicKmer_candidates_5.bed'\n",
    "    outfile = vstDir + 'input/' + 'Merged_AllVstWindows.bed'\n",
    "\n",
    "    print '#Writing merged bed file'\n",
    "    cmd = 'cat %s %s > %s' % (fastcnbed, quickmerbed, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "\n",
    "    print '#Sorting merged bed file'\n",
    "    cmd = 'bedtools sort -i %s > %s.sorted' % (outfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "\n",
    "    print '#Merging merged bed file'\n",
    "    cmd = 'bedtools merge -i %s.sorted > %s.sorted.merged' % (outfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "\n",
    "    cdrarr = []\n",
    "    inFile = open(outfile + '.sorted.merged', 'r')\n",
    "    for line in inFile:\n",
    "        line=line.rstrip()\n",
    "        line=line.split('\\t')\n",
    "        chrom, start, end = line[0:3]\n",
    "        cdrarr.append([chrom,start,end,'ID','NA','NA','NA'])\n",
    "\n",
    "    #Sorting the array based on the subroutine outlined above\n",
    "    cdrarr = sorted(cdrarr, cmp=chrom_loc_compare)\n",
    "\n",
    "    #Now re-naming the windows to CDR + integer based on the ordered array\n",
    "    for i,item in enumerate(cdrarr):\n",
    "        cdrarr[i][3]='VCDR'+str(i+1)\n",
    "\n",
    "    print 'Created final table of the %i CDRs based on QuicKmer and fastCN VST analysis' % len(cdrarr)\n",
    "\n",
    "    cdrFile = open(vstDir + 'results/' + 'TotalVstCDR_' + 'Coordinates.bed', 'w')\n",
    "    print 'Writing the table to:\\n', cdrFile\n",
    "\n",
    "    for i,item in enumerate(cdrarr):\n",
    "        cdrFile.write('\\t'.join(map(str,cdrarr[i][0:4])) + '\\n')\n",
    "    cdrFile.close()\n",
    "    \n",
    "    return cdrarr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_VstCDRs_WithTotalCalls(vstDir):\n",
    "    #Intersecting bed coordinates of cdr file with the total z-scores from fastCN and quickmer\n",
    "    cdrfile = vstDir + 'results/' + 'TotalVstCDR_' + 'Coordinates.bed'\n",
    "    #VST bedfiles that have Z-scores for all windows with at least 1.5 CN range (filtered VST window set)\n",
    "    fastcnfile = vstDir + 'input/' + 'FastCN_zscore_15CN_regions.bed' \n",
    "    quickmerfile = vstDir + 'input/' + 'QuicKmer_zscore_15CN_regions.bed' \n",
    "\n",
    "    #Intersecting with fastCN\n",
    "    outfile = vstDir + 'results/' + 'Intersect_VstCDRs_with_TotalFastCN.txt'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (cdrfile, fastcnfile, outfile)\n",
    "    #print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "\n",
    "    #Intersecting with quickmer\n",
    "    outfile = vstDir + 'results/' + 'Intersect_VstCDRs_with_TotalQuicKmer.txt'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (cdrfile, quickmerfile, outfile)\n",
    "    #print cmd\n",
    "    genutils.runCMD(cmd)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def assign_Zscores(vstDir):\n",
    "    fastcnfile = vstDir + 'results/' + 'Intersect_VstCDRs_with_TotalFastCN.txt'\n",
    "    quickmerfile = vstDir + 'results/' + 'Intersect_VstCDRs_with_TotalQuicKmer.txt'\n",
    "    fileArray = [fastcnfile, quickmerfile]\n",
    "\n",
    "    index = 0\n",
    "    zArray = []\n",
    "\n",
    "    for i in fileArray:\n",
    "        infile = open(i, 'r')\n",
    "        index+=1 \n",
    "        zDict = {} #clear the dictionary with each new index\n",
    "        for line in infile:\n",
    "            line=line.rstrip()\n",
    "            line=line.split('\\t')\n",
    "            if index == 1:\n",
    "                Type = 'FastCN'\n",
    "            if index == 2:\n",
    "                Type = 'QuicKmer'\n",
    "            chrom,start,end,ID,Z = line[0],line[1],line[2],line[3],float(line[7])\n",
    "            zArray.append([chrom,start,end,ID,Z,Type])\n",
    "            currentZ = float(Z)\n",
    "            if ID in zDict:\n",
    "                dictZ = float(zDict[ID][4])\n",
    "                if currentZ > dictZ:\n",
    "                    zDict[ID][4] = currentZ\n",
    "                else:\n",
    "                    continue\n",
    "            else:\n",
    "                zDict[ID] = []\n",
    "                zDict[ID] = [chrom,start,end,ID,Z,Type]\n",
    "        for i in zDict:\n",
    "            zArray.append(zDict[i])\n",
    "\n",
    "    for i,items in enumerate(zArray):\n",
    "        zWin = zArray[i][3]\n",
    "        zZ = zArray[i][4]\n",
    "        vSet = zArray[i][5]\n",
    "        for i,items in enumerate(cdrarr):\n",
    "            cWin = cdrarr[i][3]\n",
    "            if vSet == 'FastCN':\n",
    "                cZ = cdrarr[i][4]\n",
    "            if vSet == 'QuicKmer':\n",
    "                cZ = cdrarr[i][5]\n",
    "            if zWin == cWin: #if it's the same CDR region being analyzed (same window ID)\n",
    "                if vSet == 'FastCN':\n",
    "                    cdrarr[i][4] = zZ\n",
    "                if vSet == 'QuicKmer':\n",
    "                    cdrarr[i][5] = zZ\n",
    "            else:\n",
    "                continue\n",
    "    return cdrarr"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def summarize_CDRs(cdr):\n",
    "    for i,items in enumerate(cdrarr):\n",
    "        zFastcn = cdrarr[i][4]\n",
    "        zQuickmer = cdrarr[i][5]\n",
    "        #Set = cdrarr[i][6]\n",
    "        if zQuickmer != 'NA' and zFastcn != 'NA' and float(zQuickmer) > 5.0 and float(zFastcn) > 5.0:\n",
    "            cdrarr[i][6] = 'Both'\n",
    "            continue\n",
    "        if zQuickmer != 'NA' and zFastcn != 'NA' and float(zQuickmer) < 5.0 and float(zFastcn) > 5.0:\n",
    "            cdrarr[i][6] = 'fastCN'\n",
    "            continue\n",
    "        if zQuickmer != 'NA' and zFastcn != 'NA' and float(zQuickmer) > 5.0 and float(zFastcn) < 5.0:\n",
    "            cdrarr[i][6] = 'Quic-Kmer'\n",
    "            continue\n",
    "        if zQuickmer == 'NA' and zFastcn != 'NA':\n",
    "            cdrarr[i][6] = 'fastCN'\n",
    "        if zQuickmer != 'NA' and zFastcn == 'NA':\n",
    "            cdrarr[i][6] = 'Quic-Kmer'\n",
    "\n",
    "    outFile = open(vstDir + 'results/Merged/' + 'Final_VST_CDRs_ZscoreTable.txt', 'w')\n",
    "    fastCount,quickCount,bothCount = 0,0,0\n",
    "    bedFile = open(vstDir + 'results/Merged/' + 'Final_VST_CDRs.bed', 'w')\n",
    "    for i, items in enumerate(cdrarr):\n",
    "        #Writing out final table\n",
    "        y = \"\\t\".join(map(str, cdrarr[i])) + '\\n'\n",
    "        outFile.write(y)\n",
    "        #How many are significant in FastCN only\n",
    "        if cdrarr[i][6] == 'fastCN':\n",
    "            fastCount += 1\n",
    "        if cdrarr[i][6] == 'Quic-Kmer':\n",
    "            quickCount += 1\n",
    "        if cdrarr[i][6] == 'Both':\n",
    "            bothCount += 1\n",
    "        y = \"\\t\".join(map(str, cdrarr[i][0:4])) + '\\n'\n",
    "        bedFile.write(y)\n",
    "    print '%i windows are significant in fastCN only' % fastCount\n",
    "    print '%i windows are significant in Quic-Kmer only' % quickCount\n",
    "    print '%i windows are significant in BOTH fastCN and Quic-Kmer' % bothCount\n",
    "    print 'TOTAL!'\n",
    "    print '%i CDRs from the VST analysis' % len(cdrarr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "invalid syntax (<ipython-input-51-36065305524d>, line 1)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-51-36065305524d>\"\u001b[0;36m, line \u001b[0;32m1\u001b[0m\n\u001b[0;31m    def make_bedfile()\u001b[0m\n\u001b[0m                      ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid syntax\n"
     ]
    }
   ],
   "source": [
    "def make_bedfile(infile):\n",
    "    for i in infile:\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_with_FST_CDRs(vstType):\n",
    "    ##############\n",
    "    # INPUT FST FILE\n",
    "    ##############\n",
    "    fstfile = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed'\n",
    "    print '\\n#Reading in final FST CDR regions from:\\n', fstfile\n",
    "    fstFile = open(fstfile, 'r')\n",
    "    print '###Vst Type: ', vstType\n",
    "    #New VST bedfile for intersecting\n",
    "    newVstbedfile = vstDir + 'results/Merged/' + 'Final_VST_CDRs.bed'\n",
    "\n",
    "    #Intersecting the vst BED file with the FST results\n",
    "    outfile = vstDir + 'results/Merged/' + 'Intersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "    print '#Intersecting the FST and VST windows directly...'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "    #checking window counts\n",
    "    f = outfile\n",
    "    count = count_lines(f)\n",
    "    print '-->%i Vst windows found to intersect FST regions' % count  \n",
    "\n",
    "    #Intersecting the vst BED file with the FST results using window approach\n",
    "    #50kb windows\n",
    "    outfile = vstDir + 'results/Merged/' + '50kbWindowIntersect_'+ vstType + '_Vst_with_FstCDRs.txt'\n",
    "    print '\\n#Intersecting the FST and VST windows with 50kb buffering windows...'\n",
    "    cmd = 'bedtools window -w 50000 -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "    #checking window counts\n",
    "    f = outfile\n",
    "    count = count_lines(f)\n",
    "    print '-->%i Vst windows found to intersect FST regions with window size = 50kb' % count\n",
    "\n",
    "    #200kb windows\n",
    "    outfile = vstDir + 'results/Merged/' + '200kbWindowIntersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "    print '\\n#Intersecting the FST and VST windows with 200kb buffering windows...'\n",
    "    cmd = 'bedtools window -w 50000 -a %s -b %s > %s' % (newVstbedfile, fstfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd)\n",
    "    #checking window counts\n",
    "    f = outfile\n",
    "    count = count_lines(f)\n",
    "    print '-->%i Vst windows found to intersect FST regions with window size = 200kb' % count\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_FST_CDR_intersect_file():\n",
    "    #Reading the intersect file results to FST CDRs\n",
    "    outfile = vstDir + 'results/Merged/' + 'Intersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "    intersectFile = open(outfile,'r') #outfile from the intersect step above\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    fstDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    fstID = ''\n",
    "    fstWindows = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        vstID = line[3]\n",
    "        fstID = line[7]\n",
    "\n",
    "        if vstID in fstDict:\n",
    "            fstDict[vstID].append(fstID)\n",
    "        else:\n",
    "            fstDict[vstID] = []\n",
    "            fstDict[vstID].append(fstID) \n",
    "    intersectFile.close()\n",
    "    return fstDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_with_Ensembl_Genes():\n",
    "    print 'Now intersecting bedfiles...\\n'\n",
    "    #Determining variables\n",
    "    inBedFile = vstDir + 'results/Merged/' + 'Final_VST_CDRs.bed'\n",
    "    genebedfile = '/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed'\n",
    "    print 'Reading in the Ensembl 81 genes from the following file:\\n', genebedfile\n",
    "    geneBedFile = open(genebedfile, 'r')\n",
    "    outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt'\n",
    "\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (inBedFile, genebedfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_gene_intersect_file(vstType):\n",
    "    b2goDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/BLAST2GO/'\n",
    "    b2goList = open(vstDir + 'results/Merged/' + vstType + '_GeneList.txt', 'w')\n",
    "    #Reading the intersect file results\n",
    "    intersectFile = open(vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt','r') #outfile from the intersect step with ensembl 81 genes\n",
    "\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    ensGDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    ensID = ''\n",
    "    sigGenes = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        winID = line[3]\n",
    "        hit = line[7].split('_')\n",
    "        Protein = hit[0]\n",
    "        Gene = hit[1]\n",
    "        b2goID = Gene + '|' + Protein\n",
    "        ensGID = hit[2]\n",
    "        \n",
    "        if winID in ensGDict:\n",
    "            ensGDict[winID].append(ensGID)\n",
    "        else:\n",
    "            ensGDict[winID] = []\n",
    "            ensGDict[winID].append(ensGID) \n",
    "            sigGenes.append(b2goID)\n",
    "    intersectFile.close()\n",
    "    for i in sigGenes:\n",
    "        gene = i\n",
    "        b2goList.write('%s\\n' % gene)\n",
    "    \n",
    "    return ensGDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Writing VST summary data table with FST intersects and gene IDs here: \n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Total_VST_WithFSTIntersects_GeneTable.txt\n",
      "There are now 128 VST regions saved to the allDict dictionary for further processing\n"
     ]
    }
   ],
   "source": [
    "#def making_summary_text_files(vstType,fstDict,ensGDict):\n",
    "newVstoutfile = vstDir + 'results/Merged/' + 'Final_VST_CDRs_ZscoreTable.txt'\n",
    "vstFile = open(newVstoutfile, 'r')\n",
    "\n",
    "############################################################################\n",
    "# SUMMARIZING VST HITS WITH ENSEMBL GENES AND FST REGIONS\n",
    "##########################################################################\n",
    "Outfile = vstDir + 'results/Merged/' + vstType + '_WithFSTIntersects_GeneTable.txt' #Write out the VST Summary data table here\n",
    "print '\\nWriting VST summary data table with FST intersects and gene IDs here: \\n', Outfile\n",
    "outFile = open(Outfile, 'w')\n",
    "#Defining header based on the structure of the dictionary you created:\n",
    "#Add to this if you're adding more to your dictionary \n",
    "headerLine = 'Chrom\\tStart\\tEnd\\tWindowID\\tWindowLength(bp)AverageZ-score\\tIntersecting_FST_CDR\\tIntersectingGenes\\n'\n",
    "outFile.write(headerLine)\n",
    "\n",
    "vstCount = 0\n",
    "#Change this if you're working with the QuicKmer VST calls or the FastCN VST calls\n",
    "\n",
    "WinID = '' #Setting equal to nothing\n",
    "allDict = {} # defining the dictionary that will have all the results of bedtool intersect compiled by windows\n",
    "\n",
    "for line in vstFile:\n",
    "    line = line.rstrip()\n",
    "    line = line.split('\\t')\n",
    "    vstCount += 1 #To use when naming the VST regions with unique identifiers\n",
    "\n",
    "    chrom, start_pos, end_pos, WinID, zscoreFastCN,zscoreQuicKmer,Set = line[0:7]\n",
    "    \"\"\"start_pos = int(line[1])\n",
    "    end_pos = int(line[2])\n",
    "    WinID = line[3]\"\"\"\n",
    "    winLength = int(end_pos) - int(start_pos)\n",
    "    \"\"\"zscoreFastCN = line[4]\n",
    "    zscoreQuicKmer = line[5]\n",
    "    Set = line[6]\"\"\"\n",
    "    winCoord = chrom + \":\" + str(start_pos) + \"-\" + str(end_pos)\n",
    "\n",
    "    #Open up dictionary for WinID\n",
    "    #0=WinID, 1=chr, 2=start_pos, 3=end_pos, 4=Window length, #5=Zscore\n",
    "    #6=Intersects with an FST CDR (default = False), \n",
    "    #7=Intersects with Ensembl Gene IDs (default = False)\n",
    "    #8=window coordinates\n",
    "    allDict[WinID] = [chrom,start_pos,end_pos,WinID,winLength,zscoreFastCN,zscoreQuicKmer,Set,False,False,winCoord] #setting up empty dictionary\n",
    "\n",
    "    #Did this VST window intersect with a FST CDR?\n",
    "    if WinID in fstDict: #If this window is in the ensembl list you generated, then...:\n",
    "        y = \", \".join(map(str, fstDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "        allDict[WinID][8] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "    #Did this VST window intersect with a gene?\n",
    "    if WinID in ensGDict: #If this window is in the ensembl list you generated, then...:\n",
    "        y = \", \".join(map(str, ensGDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "        allDict[WinID][8] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "print 'There are now %i VST regions saved to the allDict dictionary for further processing' % (len(allDict))\n",
    "\n",
    "#Writing out results\n",
    "for keys in sorted(allDict.keys()): #Now loops through each \n",
    "    #print \"\\t\".join(map(str,allDict[keys]))\n",
    "    outFile.write(\"\\t\".join(map(str,allDict[keys])))\n",
    "    outFile.write(\"\\n\")\n",
    "outFile.close()\n",
    "vstFile.close()\n",
    "#return allDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "##1. Changing the input file types for fastCN and QuicKmer\n",
      "###FastCN###\n",
      "#Reading in candidate VST regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/FastCN_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following BED file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following TEXT file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.txt\n",
      "Wrote new information for 120 VST windows to outfile\n",
      "###QuicKmer###\n",
      "#Reading in candidate VST regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following BED file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed\n",
      "Writing out new window IDs and modifying file structure for the following TEXT file:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.txt\n",
      "Wrote new information for 37 VST windows to outfile\n",
      "\n",
      "##3. Parsing fastCN/QuicKmer intersect to find CDRs and assigning Zscores\n",
      "#Writing merged bed file\n",
      "cat /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_FastCN_candidates_5.bed /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Mod_QuicKmer_candidates_5.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Merged_AllVstWindows.bed\n",
      "#Sorting merged bed file\n",
      "bedtools sort -i /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Merged_AllVstWindows.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Merged_AllVstWindows.bed.sorted\n",
      "#Merging merged bed file\n",
      "bedtools merge -i /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Merged_AllVstWindows.bed.sorted > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/input/Merged_AllVstWindows.bed.sorted.merged\n",
      "Created final table of the 128 CDRs based on QuicKmer and fastCN VST analysis\n",
      "Writing the table to:\n",
      "<open file '/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/TotalVstCDR_Coordinates.bed', mode 'w' at 0x2af883cd6f60>\n",
      "\n",
      "##4. Making final Zscore table for the merged, final VST CDRs...\n",
      "101 windows are significant in fastCN only\n",
      "9 windows are significant in Quic-Kmer only\n",
      "18 windows are significant in BOTH fastCN and Quic-Kmer\n",
      "TOTAL!\n",
      "128 CDRs from the VST analysis\n",
      "\n",
      "##5. Intersecting final VST CDRs with FST CDRs...\n",
      "\n",
      "#Reading in final FST CDR regions from:\n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed\n",
      "###Vst Type:  Total_VST\n",
      "#Intersecting the FST and VST windows directly...\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Final_VST_CDRs.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Intersect_Total_VST_Vst_with_FstCDRs.txt\n",
      "-->7 Vst windows found to intersect FST regions\n",
      "\n",
      "#Intersecting the FST and VST windows with 50kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Final_VST_CDRs.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/50kbWindowIntersect_Total_VST_Vst_with_FstCDRs.txt\n",
      "-->8 Vst windows found to intersect FST regions with window size = 50kb\n",
      "\n",
      "#Intersecting the FST and VST windows with 200kb buffering windows...\n",
      "bedtools window -w 50000 -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Final_VST_CDRs.bed -b /home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/MergedWindows/FinalMergedCDRs/Final_CDRCoordinates_AllSNPSets_Table.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/200kbWindowIntersect_Total_VST_Vst_with_FstCDRs.txt\n",
      "-->8 Vst windows found to intersect FST regions with window size = 200kb\n",
      "\n",
      "##6. Intersecting final VST CDRs with Ensembl 81 genes...\n",
      "Now intersecting bedfiles...\n",
      "\n",
      "Reading in the Ensembl 81 genes from the following file:\n",
      "/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed\n",
      "bedtools intersect -wo -a /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Merged/Final_VST_CDRs.bed -b /home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed > /home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Intersect_Total_VST_VSTWindows_Ensembl81Genes.txt\n",
      "Now generating summary text files for Total_VST results...\n",
      "\n",
      "Writing VST summary data table with FST intersects and gene IDs here: \n",
      "/home/ampend/kidd-lab/ampend-projects/Angela/Feichen_VST/results/Total_VST_CDRs_WithFSTIntersects_GeneTable.txt\n",
      "There are now 128 VST regions saved to the allDict dictionary for further processing\n"
     ]
    }
   ],
   "source": [
    "\n",
    "####################################################\n",
    "#1. Generating new VST files with proper unique IDs \n",
    "####################################################\n",
    "print '\\n##1. Changing the input file types for fastCN and QuicKmer'\n",
    "make_new_Vst_files(vstDir)\n",
    "\n",
    "####################################################\n",
    "#2. Intersecting fastCN and QuicKmer regions to help identify VST CDRs\n",
    "####################################################\n",
    "#print '\\n##2. Now intersecting fastCN and QuicKmer regions to help in finding CDRs'\n",
    "#intersect_fastCN_with_QuicKmer(vstDir)\n",
    "\n",
    "####################################################\n",
    "#3. Identify VST CDRs\n",
    "####################################################\n",
    "print '\\n##3. Parsing fastCN/QuicKmer intersect to find CDRs and assigning Zscores'\n",
    "cdrarr = find_CDRs(vstDir)\n",
    "\n",
    "####################################################\n",
    "#4. Finding CDR Z-scores\n",
    "####################################################\n",
    "print '\\n##4. Making final Zscore table for the merged, final VST CDRs...'\n",
    "intersect_VstCDRs_WithTotalCalls(vstDir)\n",
    "cdrarr = assign_Zscores(vstDir)\n",
    "summarize_CDRs(cdrarr)\n",
    "\n",
    "####################################################\n",
    "#5. Finding intersecting final VST CDRs with FST CDRs\n",
    "####################################################\n",
    "print '\\n##5. Intersecting final VST CDRs with FST CDRs...'\n",
    "vstType = 'Total_VST'\n",
    "intersect_with_FST_CDRs(vstType)\n",
    "fstDict = parse_FST_CDR_intersect_file()\n",
    "\n",
    "####################################################\n",
    "#6. Finding intersecting genes with final VST CDRs \n",
    "####################################################\n",
    "print '\\n##6. Intersecting final VST CDRs with Ensembl 81 genes...'\n",
    "intersect_with_Ensembl_Genes()\n",
    "ensGDict = parse_gene_intersect_file(vstType)\n",
    "\n",
    "###########################################################\n",
    "#7. Summarizing VST and Ensembl hits to output text files\n",
    "###########################################################\n",
    "print 'Now generating summary text files for %s results...' % (vstType)\n",
    "allDict = making_summary_text_files(vstType,fstDict,ensGDict) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def intersect_with_Ensembl_Genes(newVstbedfile):\n",
    "    print 'Now intersecting bedfiles...\\n'\n",
    "    #Determining variables\n",
    "    inBedFile = newVstbedfile\n",
    "    genebedfile = '/home/ampend/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_Ensembl81_GeneTables_WithEnscafIDsAndChrom.bed'\n",
    "    print 'Reading in the Ensembl 81 genes from the following file:\\n', genebedfile\n",
    "    geneBedFile = open(genebedfile, 'r')\n",
    "    outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt'\n",
    "\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (inBedFile, genebedfile, outfile)\n",
    "    print cmd\n",
    "    genutils.runCMD(cmd) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_gene_intersect_file(intfile, vstType):\n",
    "    b2goDir = '/home/ampend/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/BLAST2GO/'\n",
    "    b2goList = open(b2goDir + vstType + '_GeneList.txt', 'w')\n",
    "    #Reading the intersect file results\n",
    "    intersectFile = open(intfile,'r') #outfile from the intersect step with ensembl 81 genes\n",
    "\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    ensGDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    ensID = ''\n",
    "    sigGenes = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        winID = line[3]\n",
    "        hit = line[7].split('_')\n",
    "        Protein = hit[0]\n",
    "        Gene = hit[1]\n",
    "        b2goID = Gene + '|' + Protein\n",
    "        ensGID = hit[2]\n",
    "        \n",
    "        if winID in ensGDict:\n",
    "            ensGDict[winID].append(ensGID)\n",
    "        else:\n",
    "            ensGDict[winID] = []\n",
    "            ensGDict[winID].append(ensGID) \n",
    "            sigGenes.append(b2goID)\n",
    "    intersectFile.close()\n",
    "    for i in sigGenes:\n",
    "        gene = i\n",
    "        b2goList.write('%s\\n' % gene)\n",
    "    \n",
    "    return ensGDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def parse_FST_CDR_intersect_file():\n",
    "    #Reading the intersect file results to FST CDRs\n",
    "    outfile = vstDir + 'results/' + 'Intersect_' + vstType + '_Vst_with_FstCDRs.txt'\n",
    "    intersectFile = open(outfile,'r') #outfile from the intersect step above\n",
    "    # going through Ensembl bedtool intersect results file\n",
    "    fstDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "    fstID = ''\n",
    "    fstWindows = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "    for line in intersectFile:\n",
    "        line = line.rstrip() #removing extraneous whitespace characters\n",
    "        line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "        vstID = line[3]\n",
    "        fstID = line[7]\n",
    "\n",
    "        if vstID in fstDict:\n",
    "            fstDict[vstID].append(fstID)\n",
    "        else:\n",
    "            fstDict[vstID] = []\n",
    "            fstDict[vstID].append(fstID) \n",
    "    intersectFile.close()\n",
    "    return fstDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def making_summary_text_files(vstType,fstDict,ensGDict):\n",
    "    newVstoutfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.txt'\n",
    "    vstFile = open(newVstoutfile, 'r')\n",
    "\n",
    "    ############################################################################\n",
    "    # SUMMARIZING VST HITS WITH ENSEMBL GENES AND FST REGIONS\n",
    "    ##########################################################################\n",
    "    Outfile = vstDir + 'results/' + vstType + '_CDRs_WithFSTIntersects_GeneTable.txt' #Write out the VST Summary data table here\n",
    "    print '\\nWriting VST summary data table with FST intersects and gene IDs here: \\n', Outfile\n",
    "    outFile = open(Outfile, 'w')\n",
    "    #Defining header based on the structure of the dictionary you created:\n",
    "    #Add to this if you're adding more to your dictionary \n",
    "    headerLine = 'Chrom\\tStart\\tEnd\\tWindowID\\tWindowLength(bp)AverageZ-score\\tIntersecting_FST_CDR\\tIntersectingGenes\\n'\n",
    "    outFile.write(headerLine)\n",
    "\n",
    "    vstCount = 0\n",
    "    #Change this if you're working with the QuicKmer VST calls or the FastCN VST calls\n",
    "\n",
    "    WinID = '' #Setting equal to nothing\n",
    "    allDict = {} # defining the dictionary that will have all the results of bedtool intersect compiled by windows\n",
    "\n",
    "    for line in vstFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split('\\t')\n",
    "        vstCount += 1 #To use when naming the VST regions with unique identifiers\n",
    "\n",
    "        chrom = line[0]\n",
    "        start_pos = int(line[1])\n",
    "        end_pos = int(line[2])\n",
    "        WinID = line[3]\n",
    "        winLength = end_pos - start_pos\n",
    "        zscore = line[4] #TO ADD ONCE YOU GET FILE BACK FROM FEICHEN\n",
    "        winCoord = chrom + \":\" + str(start_pos) + \"-\" + str(end_pos)\n",
    "\n",
    "        #Open up dictionary for WinID\n",
    "        #0=WinID, 1=chr, 2=start_pos, 3=end_pos, 4=Window length, #5=Zscore\n",
    "        #6=Intersects with an FST CDR (default = False), \n",
    "        #7=Intersects with Ensembl Gene IDs (default = False)\n",
    "        #8=window coordinates\n",
    "        allDict[WinID] = [chrom,start_pos,end_pos,WinID,winLength,zscore,False,False,winCoord] #setting up empty dictionary\n",
    "\n",
    "        #Did this VST window intersect with a FST CDR?\n",
    "        if WinID in fstDict: #If this window is in the ensembl list you generated, then...:\n",
    "            y = \", \".join(map(str, fstDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "            allDict[WinID][6] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "        #Did this VST window intersect with a gene?\n",
    "        if WinID in ensGDict: #If this window is in the ensembl list you generated, then...:\n",
    "            y = \", \".join(map(str, ensGDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "            allDict[WinID][7] = y #Now it adds 'y' you generated in the line above to your dictionary\n",
    "\n",
    "    print 'There are now %i VST regions saved to the allDict dictionary for further processing' % (len(allDict))\n",
    "\n",
    "    #Writing out results\n",
    "    for keys in sorted(allDict.keys()): #Now loops through each \n",
    "        #print \"\\t\".join(map(str,allDict[keys]))\n",
    "        outFile.write(\"\\t\".join(map(str,allDict[keys])))\n",
    "        outFile.write(\"\\n\")\n",
    "    outFile.close()\n",
    "    vstFile.close()\n",
    "    return allDict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "def gene_fst_intersect_summary(allDict):\n",
    "    geneCount = 0\n",
    "    fstCount = 0\n",
    "    for keys in sorted(allDict.keys()):\n",
    "        if allDict[keys][6] is not False:\n",
    "            fstCount += 1\n",
    "        if allDict[keys][7] is not False:\n",
    "            geneCount += 1\n",
    "\n",
    "    print '%i windows intersect with FST CDRs' % (fstCount)\n",
    "    print '%i windows intersect with Ensembl genes' % (geneCount)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def make_gene_enrichment_list():\n",
    "    for winID in allDict.keys():\n",
    "        print "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "for i in vstTypes:\n",
    "    vstType = i\n",
    "    print '\\n############\\n##%s\\n############\\n' % (vstType)\n",
    "    intfile = vstDir + 'results/' + 'Intersect_' + vstType + '_VSTWindows_Ensembl81Genes.txt'\n",
    "    \n",
    "    ###############################\n",
    "    #2. Intersecting with FST CDRs\n",
    "    ###############################\n",
    "    print 'Now intersecting VST regions with FST CDRs'\n",
    "    intersect_with_FST_CDRs(vstType)\n",
    "    \n",
    "    ####################################\n",
    "    #3. Intersecting with Ensembl Genes\n",
    "    ####################################\n",
    "    print 'Now intersecting VST regions with Ensembl 81 genes'\n",
    "    newVstbedfile = vstDir + 'input/' + 'Mod_' + vstType + '_candidates_5.bed'\n",
    "    intersect_with_Ensembl_Genes(newVstbedfile)\n",
    "\n",
    "    ########################################################\n",
    "    #4. Parse bedtools intersect file against Ensembl genes\n",
    "    ########################################################\n",
    "    print 'Now parsing Ensembl gene intersect file:\\n', intfile\n",
    "    ensGDict = parse_gene_intersect_file(intfile, vstType)\n",
    "    print ensGDict\n",
    "\n",
    "    ###################################################\n",
    "    #5. Parse bedtools intersect file against FST CDRs\n",
    "    ###################################################\n",
    "    print 'Now parsing the FST CDR intersect file...'\n",
    "    fstDict = parse_FST_CDR_intersect_file()\n",
    "    \n",
    "    ###########################################################\n",
    "    #6. Summarizing VST and Ensembl hits to output text files\n",
    "    ###########################################################\n",
    "    print 'Now generating summary text files for %s results...' % (vstType)\n",
    "    allDict = making_summary_text_files(vstType,fstDict,ensGDict)    \n",
    "    \n",
    "    ##################################################\n",
    "    #7. How many genes and FST CDRs are intersecting?\n",
    "    ##################################################\n",
    "    print 'Now finding how many genes and CDR regions intersect with the %s results...' % (vstType)\n",
    "    gene_fst_intersect_summary(allDict)\n",
    "    \n",
    "    ##################################################\n",
    "    #8. Make gene list for GO gene enrichment steps\n",
    "    ##################################################\n",
    "    #print 'Now writing outfile with the gene IDs in these windows to search for enriched GO categories...'\n",
    "    #make_gene_enrichment_list()\n",
    "    \n",
    "    \n",
    "    \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "###########################\n",
    "#Creating bedfile\n",
    "##########################\n",
    "#INFILE\n",
    "infile = options.input\n",
    "inFile = open(infile, 'r')\n",
    "print '\\nInfile of significant windows is:\\n ', options.input\n",
    "#OUTFILE\n",
    "newName = infile.replace('.txt','')\n",
    "outfile = newName + '.bed'\n",
    "print '\\nBedfile is being written here:\\n', outfile\n",
    "outFile = open(outfile, 'w')\n",
    "\n",
    "for line in inFile:\n",
    "    #Skip header line\t\n",
    "    if 'Chrom' in line:\n",
    "        continue\n",
    "\n",
    "    line = line.rstrip()\n",
    "    line = line.split()\n",
    "\n",
    "    chrom = line[0]\n",
    "    start = int(line[1]) - 1\n",
    "    end = int(line[2])\n",
    "    windowID = line[3]\n",
    "\n",
    "    outFile.write('%s\\t%i\\t%i\\t%s\\n'%(chrom,start,end,windowID))\n",
    "outFile.close()\n",
    "inFile.close()\n",
    "\n",
    "\n",
    "\n",
    "###########################\n",
    "#Intersecting \n",
    "###########################\n",
    "print '\\nNow intersecting bedfiles...\\n'\n",
    "#Determining variables\n",
    "inBedFile = outfile\n",
    "geneBedFile = open(options.geneBed, 'r')\n",
    "outfile = options.outdir + 'Intersect_' + 'SIGNonOverlappingWindows_' + 'FST_Ensembl81Genes.txt'\n",
    "\n",
    "cmd = 'bedtools intersect -wo -a %s -b %s > %s' % (inBedFile, options.geneBed, outfile)\n",
    "print cmd\n",
    "genutils.runCMD(cmd)       \n",
    "\n",
    "###############################\n",
    "#Parse bedtools intersect file \n",
    "###############################\n",
    "#Reading the intersect file results\n",
    "intersectFile = open(outfile,'r') #outfile from the intersect step above\n",
    "# going through Ensembl bedtool intersect results file\n",
    "ensGDict = {} #ensGDict[windowID] = [gene1,gene2,gene3...]\n",
    "ensID = ''\n",
    "sigGenes = [] #To keep track of all gene IDs (ENSCAFG) that are in sig windows\n",
    "\n",
    "for line in intersectFile:\n",
    "    line = line.rstrip() #removing extraneous whitespace characters\n",
    "    line = line.split() #delimiting \"columns\" in the file based on tabs\t\t\n",
    "    winID = line[3]\n",
    "    hit = line[7].split('_')\n",
    "    Gene = hit[1]\n",
    "    ensGID = hit[2]\n",
    "    sigGenes.append(Gene)\n",
    "    if winID in ensGDict:\n",
    "        ensGDict[winID].append(ensGID)\n",
    "    else:\n",
    "        ensGDict[winID] = []\n",
    "        ensGDict[winID].append(ensGID) \n",
    "intersectFile.close()\n",
    "\n",
    "############################################################################\n",
    "# Input the bed file to make the dictionary\n",
    "# The dictionary will have all outlier Fst windows, regardless if there are genes in those\n",
    "#\twindows\n",
    "newName = options.input.replace('.txt', '')\n",
    "Outfile = newName + '_GeneTable.txt' #Write out the new FST data here\n",
    "print '\\nWriting new FST data table with gene IDs here: \\n', Outfile\n",
    "outFile = open(Outfile, 'w')\n",
    "\n",
    "#Defining header based on the structure of the dictionary you created:\n",
    "headerLine = 'Chrom\\tCDR_Start\\tCDR_End\\tCDR_ID\\tCDR_Length\\tSet\\tGenes\\tCDR_Coordinates\\tSNP_Set1_OverlappingWindows_MeanZscore\\t\\tSNP_Set1_OverlappingWindows_MaxZscore\\tSNP_Set2_OverlappingWindows_MeanZscore\\tSNP_Set2_OverlappingWindows_MaxZscore\\n'\n",
    "outFile.write(headerLine)\n",
    "\n",
    "WinID = '' #Setting equal to nothing\n",
    "allDict = {} # defining the dictionary that will have all the results of bedtool intersect compiled by windows\n",
    "\n",
    "windowCount = 0\n",
    "inBedFile = open(options.input, 'r')\n",
    "\n",
    "for line in inBedFile:\n",
    "    if 'Chrom' in line: #skips header, if present\n",
    "        continue\n",
    "    line = line.rstrip() #removing extraneous whitespace characters\n",
    "    line = line.split('\\t') #delimiting \"columns\" in the file based on tabs\t\t\n",
    "    windowCount += 1 \n",
    "\n",
    "    chr = line[0]\n",
    "    start_pos = line[1]\n",
    "    end_pos = line[2]\n",
    "    WinID = line[3]\n",
    "    winLength = line[4]\n",
    "    callset = line[5]\n",
    "    winCoord = chr + \":\" + start_pos + \"-\" + end_pos\n",
    "    #Open up dictionary for WinID\n",
    "    #0=WinID, 1=chr, 2=start_pos, 3=end_pos, 4=zscore,\n",
    "    #5 = CDR length\n",
    "    #6=Ensembl Gene IDs (default = False)\n",
    "    #7 = Window coordinates in one string (for ease of display in tables for manuscript)\n",
    "    #8 = TOTAL SNPs (SET 1) - Mean ROA Zscore - Overlapping(Sliding) Window\n",
    "    #9 = TOTAL SNPs (SET 1) - Maximum ROA Zscore - Overlapping(Sliding) Window\n",
    "    #10 = CALLSET 3 SNPs (SET 2) - Mean ROA Zscore - Overlapping(Sliding) Window\n",
    "    #11 = CALLSET 3 SNPs (SET 2) - Maximum ROA Zscore - Overlapping(Sliding) Window\n",
    "    #once I get the script working I will add more to the dictionary. For example,\n",
    "    # Broad bedtools intersect results\n",
    "    allDict[WinID] = [chr,start_pos,end_pos,WinID,winLength,callset,False,winCoord,'NA', 'NA', 'NA', 'NA'] #setting up empty dictionary\n",
    "\n",
    "    if WinID in ensGDict: #If this window is in the ensembl list you generated, then...:\n",
    "        y = \", \".join(map(str, ensGDict[WinID])) #This takes the list you generated from the Ensembl file and joins each entry in the list with a comma\n",
    "        allDict[WinID][6] = y\n",
    "    #For when you add Broad for instance\n",
    "    #if WinID in broadGDict:\n",
    "    #\tsame thing as above\n",
    "    #\tnow you'd add to allDict[WindID][6] or [7]\n",
    "\n",
    "print allDict\n",
    "# Sort dictionary\n",
    "#sorted(allDict.keys())\t\n",
    "\n",
    "############################################################################\n",
    "#Writing the geneID + proteinID of all ensembl 81 genes that intersect\n",
    "#\twith significant windows\n",
    "####Input files:\n",
    "##GeneList that went into BLAST2GO pipeline\n",
    "#Format: [0] = ENSCAFP ID and [1] = ENSCAFG ID\n",
    "inFile = open('/home/jmkidd/kidd-lab/ampend-projects/BLAST2GO/results/BLAST2GO_GeneTables_WithEnscafIDsAndChrom.txt', 'r')\n",
    "\n",
    "#Saving gene-protein links that were processed with BLAST2GO to array\n",
    "protGene = {}\n",
    "\n",
    "for line in inFile:\n",
    "    line = line.rstrip()\n",
    "    line = line.split()\n",
    "    geneID = line[1]\n",
    "    protID = line[0]\n",
    "    protGene[geneID] = []\n",
    "    protGene[geneID] = protID\n",
    "print '\\n%i genes read into gene-protein ID array' % len(protGene)\n",
    "\n",
    "#FOR BLAST2GO ENRICHMENT FILES\n",
    "blast2goDir = '/home/jmkidd/kidd-lab/ampend-projects/Angela/Re-RunningAnalysis_NewScripts/new_results/results/BLAST2GO/'\n",
    "enrFile = open(blast2goDir + options.root + 'EnrichmentGeneSet.txt', 'w')\n",
    "\n",
    "#Goes through sig gene array from above and links the proteins and gene IDs together\n",
    "#\tto then write to the enrichment file that can be inputted into BLAST2GO\n",
    "print 'Writing significant genes to test for enrichment in BLAST2GO to this file: ', enrFile\n",
    "for i in sigGenes:\n",
    "    geneID = i\n",
    "    protID = protGene[i]\n",
    "    enrFile.write('%s|%s\\n' % (geneID, protID))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "#MAKING UCSC TRACKS\n",
    "zfile = vstDir + 'input/' + vstType + '_zscore_15CN_regions.bed'\n",
    "print 'Reading in Z-scores from the following file\\n', zfile\n",
    "zFile = open(zfile, 'r')\n",
    "\n",
    "for line in zFile:\n",
    "    line = line.rstrip()\n",
    "    line = line.split('\\t')\n",
    "    chrom = line[0]\n",
    "    start = line[1]\n",
    "    end = line[2]\n",
    "    zScore = \n",
    "    print line\n",
    "    break"
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [conda root]",
   "language": "python",
   "name": "conda-root-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}
