{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2018-07-10\n",
    "# A. Pendleton\n",
    "# Calculating gene model statistics at various stages\n",
    "# Stats include intron/exon lengths, GC content, etc."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import gzip\n",
    "import fileinput\n",
    "import glob\n",
    "from scipy import stats\n",
    "import re\n",
    "from matplotlib_venn import venn3, venn3_circles\n",
    "from collections import OrderedDict\n",
    "import seaborn as sns\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)\n",
    "# TO REMOVE TOP AND RIGHT AXIS OF PLOTS\n",
    "def simpleaxis(ax):\n",
    "    ax.spines['top'].set_visible(False)\n",
    "    ax.spines['right'].set_visible(False)\n",
    "    ax.get_xaxis().tick_bottom()\n",
    "    ax.get_yaxis().tick_left()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PASA OUTPUTS\n",
    "## Calculating Stats using 2 thresholds of maximum intron lengths during PASA processing: \n",
    "## Length 1 = 10,000 bp, Length 2 = 100,000 bp (default for PASA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr25', 'PASA', 'exon', '6837', '6895', '.', '-', '.', 'gene_id \"chr25.g3\"; transcript_id \"chr25.g3.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30529', '30799', '.', '-', '.', 'gene_id \"chr3.g5\"; transcript_id \"chr3.g5.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26072', '26294', '.', '-', '.', 'gene_id \"chr3.g11\"; transcript_id \"chr3.g11.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30091', '30541', '.', '-', '.', 'gene_id \"chr3.g18\"; transcript_id \"chr3.g18.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '17287', '18285', '.', '-', '.', 'gene_id \"chr3.g19\"; transcript_id \"chr3.g19.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '68638', '68996', '.', '-', '.', 'gene_id \"chr25.g77\"; transcript_id \"chr25.g77.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '70120', '71164', '.', '-', '.', 'gene_id \"chr25.g80\"; transcript_id \"chr25.g80.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '78756', '78908', '.', '+', '.', 'gene_id \"chr3.g102\"; transcript_id \"chr3.g102.i3\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '77355', '78908', '.', '+', '.', 'gene_id \"chr3.g102\"; transcript_id \"chr3.g102.i4\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '183805', '183873', '.', '+', '.', 'gene_id \"chr3.g109\"; transcript_id \"chr3.g109.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '183806', '183873', '.', '+', '.', 'gene_id \"chr3.g109\"; transcript_id \"chr3.g109.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '105959', '106128', '.', '-', '.', 'gene_id \"chr3.g115\"; transcript_id \"chr3.g115.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '216887', '217267', '.', '+', '.', 'gene_id \"chr3.g117\"; transcript_id \"chr3.g117.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '108941', '109164', '.', '+', '.', 'gene_id \"chr3.g141\"; transcript_id \"chr3.g141.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '131412', '131566', '.', '+', '.', 'gene_id \"chr3.g165\"; transcript_id \"chr3.g165.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '136950', '137230', '.', '-', '.', 'gene_id \"chr3.g168\"; transcript_id \"chr3.g168.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '278043', '278112', '.', '+', '.', 'gene_id \"chr25.g206\"; transcript_id \"chr25.g206.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '633164', '633327', '.', '+', '.', 'gene_id \"chr25.g394\"; transcript_id \"chr25.g394.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '758586', '758649', '.', '-', '.', 'gene_id \"chr25.g465\"; transcript_id \"chr25.g465.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '853133', '853324', '.', '-', '.', 'gene_id \"chr25.g498\"; transcript_id \"chr25.g498.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '853331', '853491', '.', '+', '.', 'gene_id \"chr25.g499\"; transcript_id \"chr25.g499.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '935208', '935239', '.', '+', '.', 'gene_id \"chr25.g530\"; transcript_id \"chr25.g530.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1010059', '1010541', '.', '+', '.', 'gene_id \"chr25.g538\"; transcript_id \"chr25.g538.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1087685', '1087940', '.', '+', '.', 'gene_id \"chr25.g563\"; transcript_id \"chr25.g563.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '1436134', '1436173', '.', '-', '.', 'gene_id \"chr3.g646\"; transcript_id \"chr3.g646.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '2303009', '2303223', '.', '+', '.', 'gene_id \"chr3.g949\"; transcript_id \"chr3.g949.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '2380944', '2380983', '.', '-', '.', 'gene_id \"chr3.g1025\"; transcript_id \"chr3.g1025.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1905378', '1905567', '.', '+', '.', 'gene_id \"chr25.g972\"; transcript_id \"chr25.g972.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1905699', '1905864', '.', '-', '.', 'gene_id \"chr25.g973\"; transcript_id \"chr25.g973.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1926213', '1926278', '.', '+', '.', 'gene_id \"chr25.g984\"; transcript_id \"chr25.g984.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1990173', '1990340', '.', '+', '.', 'gene_id \"chr25.g1018\"; transcript_id \"chr25.g1018.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '2228092', '2228347', '.', '+', '.', 'gene_id \"chr25.g1162\"; transcript_id \"chr25.g1162.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '2716857', '2716890', '.', '+', '.', 'gene_id \"chr3.g1235\"; transcript_id \"chr3.g1235.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '2792079', '2792262', '.', '-', '.', 'gene_id \"chr3.g1280\"; transcript_id \"chr3.g1280.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3094708', '3094757', '.', '+', '.', 'gene_id \"chr3.g1365\"; transcript_id \"chr3.g1365.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '2832812', '2832944', '.', '+', '.', 'gene_id \"chr25.g1323\"; transcript_id \"chr25.g1323.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3509204', '3509425', '.', '+', '.', 'gene_id \"chr3.g1466\"; transcript_id \"chr3.g1466.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '3397774', '3397815', '.', '+', '.', 'gene_id \"chr25.g1437\"; transcript_id \"chr25.g1437.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '3485736', '3485795', '.', '-', '.', 'gene_id \"chr25.g1455\"; transcript_id \"chr25.g1455.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '3647986', '3648439', '.', '+', '.', 'gene_id \"chr25.g1575\"; transcript_id \"chr25.g1575.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3818553', '3818812', '.', '+', '.', 'gene_id \"chr3.g1647\"; transcript_id \"chr3.g1647.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3986085', '3986136', '.', '-', '.', 'gene_id \"chr3.g1733\"; transcript_id \"chr3.g1733.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '4131037', '4131084', '.', '-', '.', 'gene_id \"chr3.g1782\"; transcript_id \"chr3.g1782.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '4198990', '4199358', '.', '+', '.', 'gene_id \"chr25.g1878\"; transcript_id \"chr25.g1878.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '4275663', '4276090', '.', '-', '.', 'gene_id \"chr25.g1921\"; transcript_id \"chr25.g1921.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '4605615', '4606832', '.', '+', '.', 'gene_id \"chr3.g1942\"; transcript_id \"chr3.g1942.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '4691130', '4691187', '.', '+', '.', 'gene_id \"chr3.g1961\"; transcript_id \"chr3.g1961.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '5547826', '5547888', '.', '-', '.', 'gene_id \"chr3.g2027\"; transcript_id \"chr3.g2027.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '7256362', '7256667', '.', '+', '.', 'gene_id \"chr3.g2109\"; transcript_id \"chr3.g2109.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '5054828', '5055022', '.', '+', '.', 'gene_id \"chr25.g2205\"; transcript_id \"chr25.g2205.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '5532616', '5532724', '.', '-', '.', 'gene_id \"chr25.g2372\"; transcript_id \"chr25.g2372.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8013123', '8013287', '.', '-', '.', 'gene_id \"chr3.g2424\"; transcript_id \"chr3.g2424.i21\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8056566', '8056716', '.', '-', '.', 'gene_id \"chr3.g2507\"; transcript_id \"chr3.g2507.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8056754', '8057118', '.', '-', '.', 'gene_id \"chr3.g2508\"; transcript_id \"chr3.g2508.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8126861', '8127515', '.', '-', '.', 'gene_id \"chr3.g2535\"; transcript_id \"chr3.g2535.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8146896', '8147069', '.', '+', '.', 'gene_id \"chr3.g2544\"; transcript_id \"chr3.g2544.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8221105', '8221341', '.', '+', '.', 'gene_id \"chr3.g2589\"; transcript_id \"chr3.g2589.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '8957674', '8957735', '.', '-', '.', 'gene_id \"chr3.g2696\"; transcript_id \"chr3.g2696.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9840122', '9840781', '.', '+', '.', 'gene_id \"chr3.g2739\"; transcript_id \"chr3.g2739.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9818587', '9818720', '.', '+', '.', 'gene_id \"chr3.g2741\"; transcript_id \"chr3.g2741.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9804438', '9806102', '.', '+', '.', 'gene_id \"chr3.g2744\"; transcript_id \"chr3.g2744.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '6999655', '6999929', '.', '-', '.', 'gene_id \"chr25.g2784\"; transcript_id \"chr25.g2784.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9608859', '9609093', '.', '-', '.', 'gene_id \"chr3.g2824\"; transcript_id \"chr3.g2824.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9609074', '9609384', '.', '+', '.', 'gene_id \"chr3.g2825\"; transcript_id \"chr3.g2825.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9610399', '9610657', '.', '+', '.', 'gene_id \"chr3.g2827\"; transcript_id \"chr3.g2827.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9612603', '9613606', '.', '+', '.', 'gene_id \"chr3.g2828\"; transcript_id \"chr3.g2828.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9610805', '9611294', '.', '-', '.', 'gene_id \"chr3.g2830\"; transcript_id \"chr3.g2830.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9614099', '9614482', '.', '-', '.', 'gene_id \"chr3.g2835\"; transcript_id \"chr3.g2835.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '7075097', '7075157', '.', '+', '.', 'gene_id \"chr25.g2860\"; transcript_id \"chr25.g2860.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9834249', '9834532', '.', '+', '.', 'gene_id \"chr3.g2928\"; transcript_id \"chr3.g2928.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '10039503', '10039542', '.', '-', '.', 'gene_id \"chr3.g2963\"; transcript_id \"chr3.g2963.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '10333386', '10333421', '.', '+', '.', 'gene_id \"chr3.g2980\"; transcript_id \"chr3.g2980.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '11061691', '11061722', '.', '+', '.', 'gene_id \"chr3.g3016\"; transcript_id \"chr3.g3016.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '11240150', '11240332', '.', '+', '.', 'gene_id \"chr3.g3138\"; transcript_id \"chr3.g3138.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12375126', '12375377', '.', '+', '.', 'gene_id \"chr3.g3245\"; transcript_id \"chr3.g3245.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '7732647', '7732860', '.', '+', '.', 'gene_id \"chr25.g3270\"; transcript_id \"chr25.g3270.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '7786549', '7786593', '.', '-', '.', 'gene_id \"chr25.g3296\"; transcript_id \"chr25.g3296.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12489272', '12489582', '.', '-', '.', 'gene_id \"chr3.g3326\"; transcript_id \"chr3.g3326.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12521147', '12521180', '.', '+', '.', 'gene_id \"chr3.g3343\"; transcript_id \"chr3.g3343.i3\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12518466', '12518787', '.', '+', '.', 'gene_id \"chr3.g3345\"; transcript_id \"chr3.g3345.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12521573', '12521725', '.', '-', '.', 'gene_id \"chr3.g3346\"; transcript_id \"chr3.g3346.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12521801', '12522854', '.', '+', '.', 'gene_id \"chr3.g3347\"; transcript_id \"chr3.g3347.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12823870', '12824163', '.', '-', '.', 'gene_id \"chr3.g3598\"; transcript_id \"chr3.g3598.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13268391', '13268458', '.', '-', '.', 'gene_id \"chr3.g3728\"; transcript_id \"chr3.g3728.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13386969', '13387172', '.', '-', '.', 'gene_id \"chr3.g3735\"; transcript_id \"chr3.g3735.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13670631', '13670836', '.', '-', '.', 'gene_id \"chr3.g3910\"; transcript_id \"chr3.g3910.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13751062', '13751268', '.', '+', '.', 'gene_id \"chr3.g3990\"; transcript_id \"chr3.g3990.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13754490', '13754970', '.', '+', '.', 'gene_id \"chr3.g3994\"; transcript_id \"chr3.g3994.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13754978', '13755161', '.', '-', '.', 'gene_id \"chr3.g3995\"; transcript_id \"chr3.g3995.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13772056', '13772255', '.', '+', '.', 'gene_id \"chr3.g4010\"; transcript_id \"chr3.g4010.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13772258', '13772466', '.', '+', '.', 'gene_id \"chr3.g4011\"; transcript_id \"chr3.g4011.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13965300', '13965335', '.', '+', '.', 'gene_id \"chr3.g4174\"; transcript_id \"chr3.g4174.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '9444506', '9444661', '.', '-', '.', 'gene_id \"chr25.g4110\"; transcript_id \"chr25.g4110.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15039589', '15039787', '.', '-', '.', 'gene_id \"chr3.g4592\"; transcript_id \"chr3.g4592.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '10365412', '10365597', '.', '-', '.', 'gene_id \"chr25.g4543\"; transcript_id \"chr25.g4543.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15160758', '15160814', '.', '-', '.', 'gene_id \"chr3.g4654\"; transcript_id \"chr3.g4654.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15220398', '15220550', '.', '-', '.', 'gene_id \"chr3.g4683\"; transcript_id \"chr3.g4683.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '10491264', '10491454', '.', '+', '.', 'gene_id \"chr25.g4626\"; transcript_id \"chr25.g4626.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '10534578', '10534769', '.', '-', '.', 'gene_id \"chr25.g4643\"; transcript_id \"chr25.g4643.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11046513', '11046702', '.', '+', '.', 'gene_id \"chr25.g4737\"; transcript_id \"chr25.g4737.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11086371', '11086967', '.', '-', '.', 'gene_id \"chr25.g4786\"; transcript_id \"chr25.g4786.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11088265', '11088462', '.', '-', '.', 'gene_id \"chr25.g4789\"; transcript_id \"chr25.g4789.i3\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11088265', '11088462', '.', '-', '.', 'gene_id \"chr25.g4789\"; transcript_id \"chr25.g4789.i4\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15479084', '15479357', '.', '+', '.', 'gene_id \"chr3.g4895\"; transcript_id \"chr3.g4895.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15480680', '15481627', '.', '-', '.', 'gene_id \"chr3.g4897\"; transcript_id \"chr3.g4897.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15717220', '15717417', '.', '+', '.', 'gene_id \"chr3.g5006\"; transcript_id \"chr3.g5006.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15724358', '15724729', '.', '-', '.', 'gene_id \"chr3.g5013\"; transcript_id \"chr3.g5013.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15739747', '15739989', '.', '-', '.', 'gene_id \"chr3.g5034\"; transcript_id \"chr3.g5034.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11396376', '11396557', '.', '+', '.', 'gene_id \"chr25.g4984\"; transcript_id \"chr25.g4984.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11396844', '11397010', '.', '-', '.', 'gene_id \"chr25.g4985\"; transcript_id \"chr25.g4985.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15825078', '15825283', '.', '-', '.', 'gene_id \"chr3.g5072\"; transcript_id \"chr3.g5072.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11521311', '11521358', '.', '+', '.', 'gene_id \"chr25.g5137\"; transcript_id \"chr25.g5137.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '17093070', '17093221', '.', '+', '.', 'gene_id \"chr3.g5239\"; transcript_id \"chr3.g5239.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11597329', '11597390', '.', '-', '.', 'gene_id \"chr25.g5235\"; transcript_id \"chr25.g5235.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11925519', '11926572', '.', '+', '.', 'gene_id \"chr25.g5309\"; transcript_id \"chr25.g5309.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11923424', '11923603', '.', '+', '.', 'gene_id \"chr25.g5310\"; transcript_id \"chr25.g5310.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11994053', '11994127', '.', '+', '.', 'gene_id \"chr25.g5333\"; transcript_id \"chr25.g5333.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12020873', '12022228', '.', '+', '.', 'gene_id \"chr25.g5335\"; transcript_id \"chr25.g5335.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12022398', '12023508', '.', '+', '.', 'gene_id \"chr25.g5337\"; transcript_id \"chr25.g5337.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12054085', '12054920', '.', '+', '.', 'gene_id \"chr25.g5406\"; transcript_id \"chr25.g5406.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12054944', '12055177', '.', '+', '.', 'gene_id \"chr25.g5407\"; transcript_id \"chr25.g5407.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12056329', '12056556', '.', '-', '.', 'gene_id \"chr25.g5410\"; transcript_id \"chr25.g5410.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12076314', '12076555', '.', '-', '.', 'gene_id \"chr25.g5431\"; transcript_id \"chr25.g5431.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12084220', '12084460', '.', '-', '.', 'gene_id \"chr25.g5439\"; transcript_id \"chr25.g5439.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12349233', '12349401', '.', '+', '.', 'gene_id \"chr25.g5554\"; transcript_id \"chr25.g5554.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '18392641', '18393217', '.', '+', '.', 'gene_id \"chr3.g5662\"; transcript_id \"chr3.g5662.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '12855768', '12856059', '.', '-', '.', 'gene_id \"chr25.g5729\"; transcript_id \"chr25.g5729.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '18973671', '18974307', '.', '-', '.', 'gene_id \"chr3.g5789\"; transcript_id \"chr3.g5789.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19119049', '19119081', '.', '+', '.', 'gene_id \"chr3.g5798\"; transcript_id \"chr3.g5798.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13023761', '13023913', '.', '+', '.', 'gene_id \"chr25.g5783\"; transcript_id \"chr25.g5783.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13170113', '13170629', '.', '-', '.', 'gene_id \"chr25.g5869\"; transcript_id \"chr25.g5869.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13232910', '13233097', '.', '-', '.', 'gene_id \"chr25.g5910\"; transcript_id \"chr25.g5910.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19812417', '19812693', '.', '+', '.', 'gene_id \"chr3.g5997\"; transcript_id \"chr3.g5997.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19813481', '19814825', '.', '-', '.', 'gene_id \"chr3.g5998\"; transcript_id \"chr3.g5998.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19814803', '19815243', '.', '+', '.', 'gene_id \"chr3.g5999\"; transcript_id \"chr3.g5999.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '20927777', '20928295', '.', '-', '.', 'gene_id \"chr3.g6277\"; transcript_id \"chr3.g6277.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '20989544', '20989636', '.', '+', '.', 'gene_id \"chr3.g6310\"; transcript_id \"chr3.g6310.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '21511028', '21511058', '.', '+', '.', 'gene_id \"chr3.g6435\"; transcript_id \"chr3.g6435.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '21569357', '21569399', '.', '+', '.', 'gene_id \"chr3.g6459\"; transcript_id \"chr3.g6459.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '14904746', '14904825', '.', '-', '.', 'gene_id \"chr25.g6427\"; transcript_id \"chr25.g6427.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '22328475', '22328736', '.', '-', '.', 'gene_id \"chr3.g6514\"; transcript_id \"chr3.g6514.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr25', 'PASA', 'exon', '14976300', '14976351', '.', '+', '.', 'gene_id \"chr25.g6462\"; transcript_id \"chr25.g6462.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '15081840', '15081883', '.', '-', '.', 'gene_id \"chr25.g6515\"; transcript_id \"chr25.g6515.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '15184532', '15184565', '.', '+', '.', 'gene_id \"chr25.g6534\"; transcript_id \"chr25.g6534.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '15571442', '15571491', '.', '-', '.', 'gene_id \"chr25.g6638\"; transcript_id \"chr25.g6638.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '24108059', '24108234', '.', '+', '.', 'gene_id \"chr3.g6737\"; transcript_id \"chr3.g6737.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '24290648', '24290684', '.', '-', '.', 'gene_id \"chr3.g6821\"; transcript_id \"chr3.g6821.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '16800689', '16800936', '.', '+', '.', 'gene_id \"chr25.g6835\"; transcript_id \"chr25.g6835.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '16800917', '16801205', '.', '-', '.', 'gene_id \"chr25.g6836\"; transcript_id \"chr25.g6836.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '16854082', '16854604', '.', '-', '.', 'gene_id \"chr25.g6916\"; transcript_id \"chr25.g6916.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25198380', '25198417', '.', '+', '.', 'gene_id \"chr3.g6997\"; transcript_id \"chr3.g6997.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25389752', '25390014', '.', '-', '.', 'gene_id \"chr3.g7052\"; transcript_id \"chr3.g7052.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '16943912', '16944181', '.', '+', '.', 'gene_id \"chr25.g7001\"; transcript_id \"chr25.g7001.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25734173', '25734719', '.', '+', '.', 'gene_id \"chr3.g7243\"; transcript_id \"chr3.g7243.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25759904', '25760099', '.', '+', '.', 'gene_id \"chr3.g7257\"; transcript_id \"chr3.g7257.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25761525', '25761842', '.', '+', '.', 'gene_id \"chr3.g7259\"; transcript_id \"chr3.g7259.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25778494', '25778691', '.', '+', '.', 'gene_id \"chr3.g7278\"; transcript_id \"chr3.g7278.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25778754', '25778999', '.', '+', '.', 'gene_id \"chr3.g7279\"; transcript_id \"chr3.g7279.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25779004', '25779432', '.', '+', '.', 'gene_id \"chr3.g7281\"; transcript_id \"chr3.g7281.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25783063', '25783221', '.', '+', '.', 'gene_id \"chr3.g7290\"; transcript_id \"chr3.g7290.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26274258', '26274294', '.', '-', '.', 'gene_id \"chr3.g7615\"; transcript_id \"chr3.g7615.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26334250', '26334788', '.', '-', '.', 'gene_id \"chr3.g7632\"; transcript_id \"chr3.g7632.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '17876033', '17876068', '.', '+', '.', 'gene_id \"chr25.g7557\"; transcript_id \"chr25.g7557.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18127173', '18127223', '.', '+', '.', 'gene_id \"chr25.g7682\"; transcript_id \"chr25.g7682.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18127187', '18127223', '.', '-', '.', 'gene_id \"chr25.g7683\"; transcript_id \"chr25.g7683.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26735713', '26735760', '.', '-', '.', 'gene_id \"chr3.g7822\"; transcript_id \"chr3.g7822.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26991988', '26992032', '.', '-', '.', 'gene_id \"chr3.g7946\"; transcript_id \"chr3.g7946.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18395190', '18395268', '.', '-', '.', 'gene_id \"chr25.g7912\"; transcript_id \"chr25.g7912.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '27221382', '27221419', '.', '-', '.', 'gene_id \"chr3.g8089\"; transcript_id \"chr3.g8089.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18497749', '18497789', '.', '+', '.', 'gene_id \"chr25.g8046\"; transcript_id \"chr25.g8046.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18526297', '18526344', '.', '+', '.', 'gene_id \"chr25.g8055\"; transcript_id \"chr25.g8055.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18569077', '18569221', '.', '+', '.', 'gene_id \"chr25.g8078\"; transcript_id \"chr25.g8078.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18793301', '18793424', '.', '+', '.', 'gene_id \"chr25.g8139\"; transcript_id \"chr25.g8139.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '18936928', '18936962', '.', '+', '.', 'gene_id \"chr25.g8153\"; transcript_id \"chr25.g8153.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19018594', '19018654', '.', '+', '.', 'gene_id \"chr25.g8184\"; transcript_id \"chr25.g8184.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '27430338', '27430385', '.', '+', '.', 'gene_id \"chr3.g8252\"; transcript_id \"chr3.g8252.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19020172', '19021372', '.', '-', '.', 'gene_id \"chr25.g8188\"; transcript_id \"chr25.g8188.i4\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19022746', '19022893', '.', '-', '.', 'gene_id \"chr25.g8188\"; transcript_id \"chr25.g8188.i7\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '27719324', '27719389', '.', '+', '.', 'gene_id \"chr3.g8433\"; transcript_id \"chr3.g8433.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '27884209', '27884657', '.', '+', '.', 'gene_id \"chr3.g8503\"; transcript_id \"chr3.g8503.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '27901806', '27901847', '.', '+', '.', 'gene_id \"chr3.g8507\"; transcript_id \"chr3.g8507.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19566001', '19566171', '.', '+', '.', 'gene_id \"chr25.g8505\"; transcript_id \"chr25.g8505.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19571267', '19571531', '.', '+', '.', 'gene_id \"chr25.g8507\"; transcript_id \"chr25.g8507.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19739838', '19739874', '.', '+', '.', 'gene_id \"chr25.g8576\"; transcript_id \"chr25.g8576.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19851429', '19851594', '.', '+', '.', 'gene_id \"chr25.g8592\"; transcript_id \"chr25.g8592.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '19851429', '19851611', '.', '+', '.', 'gene_id \"chr25.g8592\"; transcript_id \"chr25.g8592.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28518372', '28519506', '.', '-', '.', 'gene_id \"chr3.g8786\"; transcript_id \"chr3.g8786.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '20092015', '20092235', '.', '+', '.', 'gene_id \"chr25.g8752\"; transcript_id \"chr25.g8752.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '20271524', '20271585', '.', '-', '.', 'gene_id \"chr25.g8843\"; transcript_id \"chr25.g8843.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28612330', '28612503', '.', '-', '.', 'gene_id \"chr3.g8898\"; transcript_id \"chr3.g8898.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28811989', '28812195', '.', '-', '.', 'gene_id \"chr3.g8945\"; transcript_id \"chr3.g8945.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28812280', '28812555', '.', '-', '.', 'gene_id \"chr3.g8946\"; transcript_id \"chr3.g8946.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28927303', '28927547', '.', '-', '.', 'gene_id \"chr3.g9004\"; transcript_id \"chr3.g9004.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '20669121', '20669282', '.', '+', '.', 'gene_id \"chr25.g9009\"; transcript_id \"chr25.g9009.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29132716', '29132873', '.', '+', '.', 'gene_id \"chr3.g9079\"; transcript_id \"chr3.g9079.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '20861295', '20861454', '.', '-', '.', 'gene_id \"chr25.g9071\"; transcript_id \"chr25.g9071.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29201739', '29202082', '.', '+', '.', 'gene_id \"chr3.g9131\"; transcript_id \"chr3.g9131.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29500241', '29500297', '.', '-', '.', 'gene_id \"chr3.g9178\"; transcript_id \"chr3.g9178.i9\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29445447', '29445567', '.', '+', '.', 'gene_id \"chr3.g9181\"; transcript_id \"chr3.g9181.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '21552987', '21553024', '.', '-', '.', 'gene_id \"chr25.g9222\"; transcript_id \"chr25.g9222.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '21621853', '21622044', '.', '+', '.', 'gene_id \"chr25.g9228\"; transcript_id \"chr25.g9228.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '21810630', '21810669', '.', '-', '.', 'gene_id \"chr25.g9245\"; transcript_id \"chr25.g9245.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '22615763', '22616114', '.', '-', '.', 'gene_id \"chr25.g9290\"; transcript_id \"chr25.g9290.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29788535', '29788710', '.', '-', '.', 'gene_id \"chr3.g9380\"; transcript_id \"chr3.g9380.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '22965879', '22966205', '.', '-', '.', 'gene_id \"chr25.g9346\"; transcript_id \"chr25.g9346.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29879983', '29880138', '.', '-', '.', 'gene_id \"chr3.g9451\"; transcript_id \"chr3.g9451.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '23940002', '23940176', '.', '+', '.', 'gene_id \"chr25.g9483\"; transcript_id \"chr25.g9483.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30502231', '30502266', '.', '+', '.', 'gene_id \"chr3.g9589\"; transcript_id \"chr3.g9589.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30553942', '30554096', '.', '-', '.', 'gene_id \"chr3.g9598\"; transcript_id \"chr3.g9598.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30807908', '30808149', '.', '-', '.', 'gene_id \"chr3.g9877\"; transcript_id \"chr3.g9877.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30854803', '30854848', '.', '-', '.', 'gene_id \"chr3.g9926\"; transcript_id \"chr3.g9926.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31052010', '31052175', '.', '+', '.', 'gene_id \"chr3.g9961\"; transcript_id \"chr3.g9961.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31052659', '31052989', '.', '-', '.', 'gene_id \"chr3.g9962\"; transcript_id \"chr3.g9962.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31111399', '31111621', '.', '+', '.', 'gene_id \"chr3.g9998\"; transcript_id \"chr3.g9998.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31496349', '31496552', '.', '-', '.', 'gene_id \"chr3.g10034\"; transcript_id \"chr3.g10034.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31593359', '31593402', '.', '+', '.', 'gene_id \"chr3.g10104\"; transcript_id \"chr3.g10104.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31599529', '31599583', '.', '+', '.', 'gene_id \"chr3.g10116\"; transcript_id \"chr3.g10116.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '25346792', '25346825', '.', '+', '.', 'gene_id \"chr25.g10001\"; transcript_id \"chr25.g10001.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31794555', '31794741', '.', '-', '.', 'gene_id \"chr3.g10235\"; transcript_id \"chr3.g10235.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26137535', '26137815', '.', '+', '.', 'gene_id \"chr25.g10116\"; transcript_id \"chr25.g10116.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26138091', '26138535', '.', '-', '.', 'gene_id \"chr25.g10117\"; transcript_id \"chr25.g10117.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26211178', '26211316', '.', '+', '.', 'gene_id \"chr25.g10178\"; transcript_id \"chr25.g10178.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26230988', '26231689', '.', '-', '.', 'gene_id \"chr25.g10192\"; transcript_id \"chr25.g10192.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26232236', '26232391', '.', '-', '.', 'gene_id \"chr25.g10194\"; transcript_id \"chr25.g10194.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '32706436', '32706487', '.', '+', '.', 'gene_id \"chr3.g10460\"; transcript_id \"chr3.g10460.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '32718764', '32718828', '.', '+', '.', 'gene_id \"chr3.g10461\"; transcript_id \"chr3.g10461.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '32995852', '32995893', '.', '+', '.', 'gene_id \"chr3.g10478\"; transcript_id \"chr3.g10478.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33336137', '33336240', '.', '-', '.', 'gene_id \"chr3.g10532\"; transcript_id \"chr3.g10532.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26911627', '26911864', '.', '+', '.', 'gene_id \"chr25.g10453\"; transcript_id \"chr25.g10453.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26912093', '26912249', '.', '-', '.', 'gene_id \"chr25.g10455\"; transcript_id \"chr25.g10455.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '27077807', '27077859', '.', '-', '.', 'gene_id \"chr25.g10466\"; transcript_id \"chr25.g10466.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '27077810', '27077861', '.', '-', '.', 'gene_id \"chr25.g10466\"; transcript_id \"chr25.g10466.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33794435', '33794599', '.', '-', '.', 'gene_id \"chr3.g10630\"; transcript_id \"chr3.g10630.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33979066', '33979239', '.', '-', '.', 'gene_id \"chr3.g10675\"; transcript_id \"chr3.g10675.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33990688', '33990778', '.', '-', '.', 'gene_id \"chr3.g10680\"; transcript_id \"chr3.g10680.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '34082140', '34082192', '.', '-', '.', 'gene_id \"chr3.g10686\"; transcript_id \"chr3.g10686.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35329911', '35329956', '.', '+', '.', 'gene_id \"chr3.g10886\"; transcript_id \"chr3.g10886.i13\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '27875603', '27875740', '.', '-', '.', 'gene_id \"chr25.g10773\"; transcript_id \"chr25.g10773.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '27876151', '27876329', '.', '-', '.', 'gene_id \"chr25.g10774\"; transcript_id \"chr25.g10774.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '27876405', '27876978', '.', '+', '.', 'gene_id \"chr25.g10775\"; transcript_id \"chr25.g10775.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i4\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i5\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i6\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i7\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i8\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i9\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35545371', '35546015', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i10\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35580537', '35580646', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i20\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35580537', '35580758', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i21\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35580497', '35580650', '.', '-', '.', 'gene_id \"chr3.g11073\"; transcript_id \"chr3.g11073.i22\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35635504', '35635660', '.', '-', '.', 'gene_id \"chr3.g11076\"; transcript_id \"chr3.g11076.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35579790', '35579968', '.', '+', '.', 'gene_id \"chr3.g11085\"; transcript_id \"chr3.g11085.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35602819', '35603046', '.', '-', '.', 'gene_id \"chr3.g11098\"; transcript_id \"chr3.g11098.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35605228', '35605308', '.', '-', '.', 'gene_id \"chr3.g11102\"; transcript_id \"chr3.g11102.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28481380', '28481622', '.', '+', '.', 'gene_id \"chr25.g10950\"; transcript_id \"chr25.g10950.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35645486', '35645696', '.', '+', '.', 'gene_id \"chr3.g11123\"; transcript_id \"chr3.g11123.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35659817', '35660058', '.', '-', '.', 'gene_id \"chr3.g11132\"; transcript_id \"chr3.g11132.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35666869', '35667024', '.', '+', '.', 'gene_id \"chr3.g11143\"; transcript_id \"chr3.g11143.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35693416', '35693600', '.', '+', '.', 'gene_id \"chr3.g11154\"; transcript_id \"chr3.g11154.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35767473', '35767568', '.', '+', '.', 'gene_id \"chr3.g11194\"; transcript_id \"chr3.g11194.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35767500', '35767566', '.', '+', '.', 'gene_id \"chr3.g11194\"; transcript_id \"chr3.g11194.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28654170', '28654386', '.', '-', '.', 'gene_id \"chr25.g11071\"; transcript_id \"chr25.g11071.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '36333935', '36334091', '.', '+', '.', 'gene_id \"chr3.g11250\"; transcript_id \"chr3.g11250.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28654429', '28654767', '.', '-', '.', 'gene_id \"chr25.g11072\"; transcript_id \"chr25.g11072.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28654429', '28654767', '.', '-', '.', 'gene_id \"chr25.g11072\"; transcript_id \"chr25.g11072.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28676946', '28676985', '.', '-', '.', 'gene_id \"chr25.g11085\"; transcript_id \"chr25.g11085.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '29176868', '29176898', '.', '+', '.', 'gene_id \"chr25.g11341\"; transcript_id \"chr25.g11341.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '29243819', '29243983', '.', '+', '.', 'gene_id \"chr25.g11356\"; transcript_id \"chr25.g11356.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '38187350', '38187482', '.', '-', '.', 'gene_id \"chr3.g11817\"; transcript_id \"chr3.g11817.i3\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '29998607', '29998644', '.', '-', '.', 'gene_id \"chr25.g11717\"; transcript_id \"chr25.g11717.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '38831044', '38831089', '.', '+', '.', 'gene_id \"chr3.g11959\"; transcript_id \"chr3.g11959.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '38858382', '38858581', '.', '+', '.', 'gene_id \"chr3.g11963\"; transcript_id \"chr3.g11963.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30079661', '30079958', '.', '+', '.', 'gene_id \"chr25.g11786\"; transcript_id \"chr25.g11786.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39000991', '39001143', '.', '+', '.', 'gene_id \"chr3.g12028\"; transcript_id \"chr3.g12028.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30394236', '30394276', '.', '+', '.', 'gene_id \"chr25.g12015\"; transcript_id \"chr25.g12015.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39509104', '39509133', '.', '-', '.', 'gene_id \"chr3.g12190\"; transcript_id \"chr3.g12190.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39515129', '39515343', '.', '-', '.', 'gene_id \"chr3.g12198\"; transcript_id \"chr3.g12198.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39561971', '39562173', '.', '+', '.', 'gene_id \"chr3.g12221\"; transcript_id \"chr3.g12221.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39562299', '39562460', '.', '-', '.', 'gene_id \"chr3.g12222\"; transcript_id \"chr3.g12222.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30435955', '30435988', '.', '-', '.', 'gene_id \"chr25.g12073\"; transcript_id \"chr25.g12073.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30448874', '30449186', '.', '-', '.', 'gene_id \"chr25.g12081\"; transcript_id \"chr25.g12081.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30449257', '30449530', '.', '+', '.', 'gene_id \"chr25.g12082\"; transcript_id \"chr25.g12082.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39662000', '39662040', '.', '+', '.', 'gene_id \"chr3.g12286\"; transcript_id \"chr3.g12286.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30891584', '30891849', '.', '-', '.', 'gene_id \"chr25.g12186\"; transcript_id \"chr25.g12186.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30984838', '30984880', '.', '+', '.', 'gene_id \"chr25.g12207\"; transcript_id \"chr25.g12207.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39868776', '39869037', '.', '-', '.', 'gene_id \"chr3.g12442\"; transcript_id \"chr3.g12442.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40027051', '40027854', '.', '+', '.', 'gene_id \"chr3.g12523\"; transcript_id \"chr3.g12523.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40029949', '40030254', '.', '-', '.', 'gene_id \"chr3.g12526\"; transcript_id \"chr3.g12526.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40030418', '40030568', '.', '+', '.', 'gene_id \"chr3.g12527\"; transcript_id \"chr3.g12527.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40036180', '40036340', '.', '-', '.', 'gene_id \"chr3.g12529\"; transcript_id \"chr3.g12529.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40036398', '40036580', '.', '+', '.', 'gene_id \"chr3.g12530\"; transcript_id \"chr3.g12530.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31351766', '31351968', '.', '+', '.', 'gene_id \"chr25.g12371\"; transcript_id \"chr25.g12371.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40115758', '40115825', '.', '-', '.', 'gene_id \"chr3.g12566\"; transcript_id \"chr3.g12566.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40199304', '40199526', '.', '+', '.', 'gene_id \"chr3.g12578\"; transcript_id \"chr3.g12578.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31413983', '31414051', '.', '-', '.', 'gene_id \"chr25.g12415\"; transcript_id \"chr25.g12415.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31629378', '31629418', '.', '+', '.', 'gene_id \"chr25.g12454\"; transcript_id \"chr25.g12454.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40497543', '40497580', '.', '+', '.', 'gene_id \"chr3.g12715\"; transcript_id \"chr3.g12715.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40540128', '40540158', '.', '-', '.', 'gene_id \"chr3.g12723\"; transcript_id \"chr3.g12723.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '32142825', '32142877', '.', '-', '.', 'gene_id \"chr25.g12527\"; transcript_id \"chr25.g12527.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '32142826', '32142877', '.', '-', '.', 'gene_id \"chr25.g12527\"; transcript_id \"chr25.g12527.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '32412577', '32412607', '.', '+', '.', 'gene_id \"chr25.g12641\"; transcript_id \"chr25.g12641.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '32578230', '32578278', '.', '-', '.', 'gene_id \"chr25.g12655\"; transcript_id \"chr25.g12655.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '32578234', '32578282', '.', '-', '.', 'gene_id \"chr25.g12655\"; transcript_id \"chr25.g12655.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40795693', '40796000', '.', '+', '.', 'gene_id \"chr3.g12853\"; transcript_id \"chr3.g12853.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40849830', '40849865', '.', '+', '.', 'gene_id \"chr3.g12887\"; transcript_id \"chr3.g12887.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr3', 'PASA', 'exon', '40969760', '40969946', '.', '+', '.', 'gene_id \"chr3.g12948\"; transcript_id \"chr3.g12948.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '34062900', '34063158', '.', '-', '.', 'gene_id \"chr25.g12845\"; transcript_id \"chr25.g12845.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41298262', '41298504', '.', '+', '.', 'gene_id \"chr3.g13116\"; transcript_id \"chr3.g13116.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41300763', '41300985', '.', '-', '.', 'gene_id \"chr3.g13120\"; transcript_id \"chr3.g13120.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '34464058', '34464258', '.', '-', '.', 'gene_id \"chr25.g13056\"; transcript_id \"chr25.g13056.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41841780', '41842357', '.', '+', '.', 'gene_id \"chr3.g13404\"; transcript_id \"chr3.g13404.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41882912', '41883519', '.', '-', '.', 'gene_id \"chr3.g13452\"; transcript_id \"chr3.g13452.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '34900848', '34901438', '.', '-', '.', 'gene_id \"chr25.g13310\"; transcript_id \"chr25.g13310.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '34900848', '34901516', '.', '-', '.', 'gene_id \"chr25.g13310\"; transcript_id \"chr25.g13310.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41898196', '41898394', '.', '+', '.', 'gene_id \"chr3.g13473\"; transcript_id \"chr3.g13473.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35202666', '35203417', '.', '+', '.', 'gene_id \"chr25.g13474\"; transcript_id \"chr25.g13474.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35202656', '35203417', '.', '+', '.', 'gene_id \"chr25.g13474\"; transcript_id \"chr25.g13474.i3\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35361606', '35362678', '.', '+', '.', 'gene_id \"chr25.g13608\"; transcript_id \"chr25.g13608.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35361439', '35361608', '.', '+', '.', 'gene_id \"chr25.g13609\"; transcript_id \"chr25.g13609.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43059596', '43059633', '.', '+', '.', 'gene_id \"chr3.g13822\"; transcript_id \"chr3.g13822.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43068597', '43068631', '.', '+', '.', 'gene_id \"chr3.g13824\"; transcript_id \"chr3.g13824.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43098815', '43098850', '.', '-', '.', 'gene_id \"chr3.g13831\"; transcript_id \"chr3.g13831.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43499785', '43500051', '.', '-', '.', 'gene_id \"chr3.g13869\"; transcript_id \"chr3.g13869.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35936562', '35936599', '.', '+', '.', 'gene_id \"chr25.g13838\"; transcript_id \"chr25.g13838.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43662656', '43662719', '.', '+', '.', 'gene_id \"chr3.g13931\"; transcript_id \"chr3.g13931.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '36028952', '36029402', '.', '-', '.', 'gene_id \"chr25.g13887\"; transcript_id \"chr25.g13887.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '36055553', '36055584', '.', '+', '.', 'gene_id \"chr25.g13913\"; transcript_id \"chr25.g13913.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '36113462', '36113666', '.', '-', '.', 'gene_id \"chr25.g13963\"; transcript_id \"chr25.g13963.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '44163863', '44164214', '.', '+', '.', 'gene_id \"chr3.g14160\"; transcript_id \"chr3.g14160.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '36398458', '36398493', '.', '-', '.', 'gene_id \"chr25.g14068\"; transcript_id \"chr25.g14068.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '44191859', '44192283', '.', '+', '.', 'gene_id \"chr3.g14195\"; transcript_id \"chr3.g14195.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '44500163', '44500478', '.', '+', '.', 'gene_id \"chr3.g14249\"; transcript_id \"chr3.g14249.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37001666', '37001851', '.', '-', '.', 'gene_id \"chr25.g14146\"; transcript_id \"chr25.g14146.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '44722052', '44722139', '.', '+', '.', 'gene_id \"chr3.g14260\"; transcript_id \"chr3.g14260.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37160497', '37160819', '.', '+', '.', 'gene_id \"chr25.g14197\"; transcript_id \"chr25.g14197.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37283998', '37284203', '.', '+', '.', 'gene_id \"chr25.g14244\"; transcript_id \"chr25.g14244.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37329355', '37329417', '.', '+', '.', 'gene_id \"chr25.g14249\"; transcript_id \"chr25.g14249.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37549023', '37549205', '.', '+', '.', 'gene_id \"chr25.g14379\"; transcript_id \"chr25.g14379.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37621103', '37621172', '.', '-', '.', 'gene_id \"chr25.g14414\"; transcript_id \"chr25.g14414.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46415204', '46415397', '.', '+', '.', 'gene_id \"chr3.g14512\"; transcript_id \"chr3.g14512.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37942109', '37942320', '.', '+', '.', 'gene_id \"chr25.g14563\"; transcript_id \"chr25.g14563.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37941838', '37942127', '.', '-', '.', 'gene_id \"chr25.g14564\"; transcript_id \"chr25.g14564.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46832469', '46832513', '.', '+', '.', 'gene_id \"chr3.g14621\"; transcript_id \"chr3.g14621.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46984941', '46985111', '.', '-', '.', 'gene_id \"chr3.g14667\"; transcript_id \"chr3.g14667.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '38086678', '38086722', '.', '-', '.', 'gene_id \"chr25.g14666\"; transcript_id \"chr25.g14666.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '47105426', '47105489', '.', '-', '.', 'gene_id \"chr3.g14826\"; transcript_id \"chr3.g14826.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '47125375', '47125852', '.', '+', '.', 'gene_id \"chr3.g14854\"; transcript_id \"chr3.g14854.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '47230823', '47231048', '.', '+', '.', 'gene_id \"chr3.g14924\"; transcript_id \"chr3.g14924.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '39397115', '39397151', '.', '+', '.', 'gene_id \"chr25.g14961\"; transcript_id \"chr25.g14961.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '39446315', '39446363', '.', '+', '.', 'gene_id \"chr25.g14964\"; transcript_id \"chr25.g14964.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48089795', '48090055', '.', '+', '.', 'gene_id \"chr3.g15238\"; transcript_id \"chr3.g15238.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48096078', '48096117', '.', '+', '.', 'gene_id \"chr3.g15242\"; transcript_id \"chr3.g15242.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48151941', '48152335', '.', '-', '.', 'gene_id \"chr3.g15275\"; transcript_id \"chr3.g15275.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48303723', '48303767', '.', '-', '.', 'gene_id \"chr3.g15309\"; transcript_id \"chr3.g15309.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '40160565', '40160601', '.', '+', '.', 'gene_id \"chr25.g15221\"; transcript_id \"chr25.g15221.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48479597', '48479761', '.', '+', '.', 'gene_id \"chr3.g15352\"; transcript_id \"chr3.g15352.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '40167723', '40168002', '.', '-', '.', 'gene_id \"chr25.g15342\"; transcript_id \"chr25.g15342.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '40436582', '40436775', '.', '-', '.', 'gene_id \"chr25.g15532\"; transcript_id \"chr25.g15532.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '40692372', '40692543', '.', '+', '.', 'gene_id \"chr25.g15672\"; transcript_id \"chr25.g15672.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '49627829', '49628320', '.', '-', '.', 'gene_id \"chr3.g15832\"; transcript_id \"chr3.g15832.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '49708140', '49708170', '.', '-', '.', 'gene_id \"chr3.g15849\"; transcript_id \"chr3.g15849.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '41034412', '41034463', '.', '+', '.', 'gene_id \"chr25.g15798\"; transcript_id \"chr25.g15798.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51179729', '51179887', '.', '-', '.', 'gene_id \"chr3.g15999\"; transcript_id \"chr3.g15999.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51180201', '51180543', '.', '+', '.', 'gene_id \"chr3.g16000\"; transcript_id \"chr3.g16000.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51180536', '51181099', '.', '-', '.', 'gene_id \"chr3.g16001\"; transcript_id \"chr3.g16001.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51181107', '51181263', '.', '-', '.', 'gene_id \"chr3.g16002\"; transcript_id \"chr3.g16002.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51660371', '51661128', '.', '-', '.', 'gene_id \"chr3.g16134\"; transcript_id \"chr3.g16134.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51661110', '51661281', '.', '+', '.', 'gene_id \"chr3.g16135\"; transcript_id \"chr3.g16135.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51730038', '51730074', '.', '-', '.', 'gene_id \"chr3.g16202\"; transcript_id \"chr3.g16202.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51783364', '51783455', '.', '-', '.', 'gene_id \"chr3.g16238\"; transcript_id \"chr3.g16238.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52064769', '52064819', '.', '-', '.', 'gene_id \"chr3.g16354\"; transcript_id \"chr3.g16354.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52137744', '52138002', '.', '-', '.', 'gene_id \"chr3.g16415\"; transcript_id \"chr3.g16415.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '42669035', '42669277', '.', '+', '.', 'gene_id \"chr25.g16410\"; transcript_id \"chr25.g16410.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '42935100', '42935394', '.', '+', '.', 'gene_id \"chr25.g16600\"; transcript_id \"chr25.g16600.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '42935692', '42936029', '.', '+', '.', 'gene_id \"chr25.g16602\"; transcript_id \"chr25.g16602.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52599668', '52599712', '.', '+', '.', 'gene_id \"chr3.g16670\"; transcript_id \"chr3.g16670.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52597030', '52597682', '.', '-', '.', 'gene_id \"chr3.g16671\"; transcript_id \"chr3.g16671.i9\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52609485', '52609635', '.', '+', '.', 'gene_id \"chr3.g16677\"; transcript_id \"chr3.g16677.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52608178', '52608419', '.', '-', '.', 'gene_id \"chr3.g16679\"; transcript_id \"chr3.g16679.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52622803', '52623036', '.', '-', '.', 'gene_id \"chr3.g16683\"; transcript_id \"chr3.g16683.i6\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52612291', '52612502', '.', '+', '.', 'gene_id \"chr3.g16692\"; transcript_id \"chr3.g16692.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52612919', '52613953', '.', '+', '.', 'gene_id \"chr3.g16694\"; transcript_id \"chr3.g16694.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52612501', '52612806', '.', '+', '.', 'gene_id \"chr3.g16695\"; transcript_id \"chr3.g16695.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52737452', '52737701', '.', '-', '.', 'gene_id \"chr3.g16761\"; transcript_id \"chr3.g16761.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43120814', '43120853', '.', '+', '.', 'gene_id \"chr25.g16727\"; transcript_id \"chr25.g16727.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43233878', '43234105', '.', '-', '.', 'gene_id \"chr25.g16794\"; transcript_id \"chr25.g16794.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43234168', '43234348', '.', '-', '.', 'gene_id \"chr25.g16795\"; transcript_id \"chr25.g16795.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43319259', '43319298', '.', '+', '.', 'gene_id \"chr25.g16859\"; transcript_id \"chr25.g16859.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53349944', '53350094', '.', '+', '.', 'gene_id \"chr3.g17173\"; transcript_id \"chr3.g17173.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53350203', '53350474', '.', '-', '.', 'gene_id \"chr3.g17174\"; transcript_id \"chr3.g17174.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53361751', '53361911', '.', '-', '.', 'gene_id \"chr3.g17185\"; transcript_id \"chr3.g17185.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53361988', '53362164', '.', '+', '.', 'gene_id \"chr3.g17186\"; transcript_id \"chr3.g17186.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53362819', '53363042', '.', '+', '.', 'gene_id \"chr3.g17189\"; transcript_id \"chr3.g17189.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43690917', '43690962', '.', '+', '.', 'gene_id \"chr25.g17117\"; transcript_id \"chr25.g17117.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43774937', '43775153', '.', '+', '.', 'gene_id \"chr25.g17207\"; transcript_id \"chr25.g17207.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53538511', '53538723', '.', '-', '.', 'gene_id \"chr3.g17354\"; transcript_id \"chr3.g17354.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43800263', '43800334', '.', '+', '.', 'gene_id \"chr25.g17233\"; transcript_id \"chr25.g17233.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53595689', '53595728', '.', '+', '.', 'gene_id \"chr3.g17413\"; transcript_id \"chr3.g17413.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44009596', '44010008', '.', '+', '.', 'gene_id \"chr25.g17364\"; transcript_id \"chr25.g17364.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44009567', '44010016', '.', '-', '.', 'gene_id \"chr25.g17365\"; transcript_id \"chr25.g17365.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44016229', '44016405', '.', '+', '.', 'gene_id \"chr25.g17377\"; transcript_id \"chr25.g17377.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53842919', '53843018', '.', '-', '.', 'gene_id \"chr3.g17595\"; transcript_id \"chr3.g17595.i2\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44190392', '44190545', '.', '-', '.', 'gene_id \"chr25.g17494\"; transcript_id \"chr25.g17494.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44521773', '44521933', '.', '-', '.', 'gene_id \"chr25.g17770\"; transcript_id \"chr25.g17770.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44521973', '44522188', '.', '+', '.', 'gene_id \"chr25.g17771\"; transcript_id \"chr25.g17771.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44675004', '44675232', '.', '+', '.', 'gene_id \"chr25.g17874\"; transcript_id \"chr25.g17874.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44675241', '44675495', '.', '-', '.', 'gene_id \"chr25.g17875\"; transcript_id \"chr25.g17875.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54457519', '54457739', '.', '-', '.', 'gene_id \"chr3.g18113\"; transcript_id \"chr3.g18113.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54458088', '54458308', '.', '-', '.', 'gene_id \"chr3.g18121\"; transcript_id \"chr3.g18121.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54497764', '54498030', '.', '+', '.', 'gene_id \"chr3.g18158\"; transcript_id \"chr3.g18158.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54498465', '54498686', '.', '-', '.', 'gene_id \"chr3.g18160\"; transcript_id \"chr3.g18160.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44938873', '44939219', '.', '+', '.', 'gene_id \"chr25.g18095\"; transcript_id \"chr25.g18095.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44942038', '44942220', '.', '+', '.', 'gene_id \"chr25.g18097\"; transcript_id \"chr25.g18097.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44989594', '44989754', '.', '-', '.', 'gene_id \"chr25.g18161\"; transcript_id \"chr25.g18161.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44989966', '44990159', '.', '-', '.', 'gene_id \"chr25.g18163\"; transcript_id \"chr25.g18163.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '45130071', '45130306', '.', '+', '.', 'gene_id \"chr25.g18225\"; transcript_id \"chr25.g18225.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54855970', '54856083', '.', '+', '.', 'gene_id \"chr3.g18465\"; transcript_id \"chr3.g18465.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54859187', '54859357', '.', '-', '.', 'gene_id \"chr3.g18471\"; transcript_id \"chr3.g18471.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '45178598', '45178632', '.', '-', '.', 'gene_id \"chr25.g18286\"; transcript_id \"chr25.g18286.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54887738', '54887797', '.', '-', '.', 'gene_id \"chr3.g18509\"; transcript_id \"chr3.g18509.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '45250536', '45250686', '.', '+', '.', 'gene_id \"chr25.g18320\"; transcript_id \"chr25.g18320.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '55156799', '55156951', '.', '-', '.', 'gene_id \"chr3.g18614\"; transcript_id \"chr3.g18614.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '46180172', '46180338', '.', '+', '.', 'gene_id \"chr25.g18506\"; transcript_id \"chr25.g18506.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '46261488', '46261527', '.', '-', '.', 'gene_id \"chr25.g18514\"; transcript_id \"chr25.g18514.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '46535531', '46535571', '.', '-', '.', 'gene_id \"chr25.g18578\"; transcript_id \"chr25.g18578.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '55843148', '55843305', '.', '-', '.', 'gene_id \"chr3.g18864\"; transcript_id \"chr3.g18864.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56094659', '56094844', '.', '-', '.', 'gene_id \"chr3.g18949\"; transcript_id \"chr3.g18949.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56095145', '56095423', '.', '+', '.', 'gene_id \"chr3.g18950\"; transcript_id \"chr3.g18950.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56170104', '56170355', '.', '+', '.', 'gene_id \"chr3.g18966\"; transcript_id \"chr3.g18966.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56389260', '56389773', '.', '-', '.', 'gene_id \"chr3.g19014\"; transcript_id \"chr3.g19014.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56411025', '56412931', '.', '-', '.', 'gene_id \"chr3.g19024\"; transcript_id \"chr3.g19024.i20\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '47151256', '47151646', '.', '-', '.', 'gene_id \"chr25.g18799\"; transcript_id \"chr25.g18799.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '47198557', '47198596', '.', '-', '.', 'gene_id \"chr25.g18825\"; transcript_id \"chr25.g18825.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56624470', '56624518', '.', '-', '.', 'gene_id \"chr3.g19119\"; transcript_id \"chr3.g19119.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr3', 'PASA', 'exon', '57298365', '57298661', '.', '+', '.', 'gene_id \"chr3.g19430\"; transcript_id \"chr3.g19430.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '57298734', '57298914', '.', '+', '.', 'gene_id \"chr3.g19431\"; transcript_id \"chr3.g19431.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '57337388', '57337543', '.', '-', '.', 'gene_id \"chr3.g19483\"; transcript_id \"chr3.g19483.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48468701', '48468761', '.', '-', '.', 'gene_id \"chr25.g19326\"; transcript_id \"chr25.g19326.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48518438', '48518629', '.', '-', '.', 'gene_id \"chr25.g19368\"; transcript_id \"chr25.g19368.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48536575', '48536628', '.', '-', '.', 'gene_id \"chr25.g19382\"; transcript_id \"chr25.g19382.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48645245', '48645283', '.', '+', '.', 'gene_id \"chr25.g19460\"; transcript_id \"chr25.g19460.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48734599', '48734767', '.', '-', '.', 'gene_id \"chr25.g19552\"; transcript_id \"chr25.g19552.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48825621', '48825888', '.', '+', '.', 'gene_id \"chr25.g19572\"; transcript_id \"chr25.g19572.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '58735336', '58735649', '.', '+', '.', 'gene_id \"chr3.g20054\"; transcript_id \"chr3.g20054.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '58737220', '58737272', '.', '-', '.', 'gene_id \"chr3.g20058\"; transcript_id \"chr3.g20058.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '49370513', '49370574', '.', '-', '.', 'gene_id \"chr25.g19919\"; transcript_id \"chr25.g19919.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '49380711', '49380924', '.', '-', '.', 'gene_id \"chr25.g19927\"; transcript_id \"chr25.g19927.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '58910529', '58910691', '.', '-', '.', 'gene_id \"chr3.g20176\"; transcript_id \"chr3.g20176.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '58978018', '58978325', '.', '+', '.', 'gene_id \"chr3.g20232\"; transcript_id \"chr3.g20232.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '59080979', '59081194', '.', '-', '.', 'gene_id \"chr3.g20309\"; transcript_id \"chr3.g20309.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '59094761', '59094828', '.', '-', '.', 'gene_id \"chr3.g20316\"; transcript_id \"chr3.g20316.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '59661756', '59661941', '.', '-', '.', 'gene_id \"chr3.g20422\"; transcript_id \"chr3.g20422.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '50517423', '50517534', '.', '+', '.', 'gene_id \"chr25.g20221\"; transcript_id \"chr25.g20221.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '50760894', '50760932', '.', '+', '.', 'gene_id \"chr25.g20316\"; transcript_id \"chr25.g20316.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60120492', '60123743', '.', '-', '.', 'gene_id \"chr3.g20633\"; transcript_id \"chr3.g20633.i4\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60175005', '60175695', '.', '+', '.', 'gene_id \"chr3.g20634\"; transcript_id \"chr3.g20634.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60175005', '60175402', '.', '+', '.', 'gene_id \"chr3.g20634\"; transcript_id \"chr3.g20634.i5\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60146984', '60149555', '.', '+', '.', 'gene_id \"chr3.g20635\"; transcript_id \"chr3.g20635.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60172899', '60173028', '.', '+', '.', 'gene_id \"chr3.g20636\"; transcript_id \"chr3.g20636.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60161786', '60161946', '.', '-', '.', 'gene_id \"chr3.g20650\"; transcript_id \"chr3.g20650.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60130155', '60130367', '.', '-', '.', 'gene_id \"chr3.g20654\"; transcript_id \"chr3.g20654.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60179020', '60179103', '.', '-', '.', 'gene_id \"chr3.g20690\"; transcript_id \"chr3.g20690.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60179020', '60179085', '.', '-', '.', 'gene_id \"chr3.g20690\"; transcript_id \"chr3.g20690.i3\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '60615963', '60617019', '.', '-', '.', 'gene_id \"chr3.g20780\"; transcript_id \"chr3.g20780.i3\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51196388', '51196586', '.', '+', '.', 'gene_id \"chr25.g20584\"; transcript_id \"chr25.g20584.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51196612', '51196924', '.', '-', '.', 'gene_id \"chr25.g20585\"; transcript_id \"chr25.g20585.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51231877', '51232210', '.', '-', '.', 'gene_id \"chr25.g20606\"; transcript_id \"chr25.g20606.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51275893', '51276586', '.', '-', '.', 'gene_id \"chr25.g20650\"; transcript_id \"chr25.g20650.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51295305', '51295346', '.', '-', '.', 'gene_id \"chr25.g20671\"; transcript_id \"chr25.g20671.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61108187', '61108223', '.', '-', '.', 'gene_id \"chr3.g21041\"; transcript_id \"chr3.g21041.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61215771', '61215991', '.', '+', '.', 'gene_id \"chr3.g21098\"; transcript_id \"chr3.g21098.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51491755', '51491872', '.', '+', '.', 'gene_id \"chr25.g20836\"; transcript_id \"chr25.g20836.i3\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '51512700', '51512994', '.', '-', '.', 'gene_id \"chr25.g20866\"; transcript_id \"chr25.g20866.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61449373', '61449532', '.', '-', '.', 'gene_id \"chr3.g21307\"; transcript_id \"chr3.g21307.i1\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '49177271', '49177411', '100', '+', '.', 'gene_id \"chr25.g21069\"; transcript_id \"blat.proc28144.chain_16443\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '36025164', '36025570', '100', '+', '.', 'gene_id \"chr25.g21187\"; transcript_id \"blat.proc18128.chain_19947\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '24365774', '24365803', '100', '+', '.', 'gene_id \"chr25.g21193\"; transcript_id \"blat.proc28144.chain_4531\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '42447835', '42447875', '100', '+', '.', 'gene_id \"chr25.g21421\"; transcript_id \"blat.proc18128.chain_26060\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30000782', '30000857', '100', '+', '.', 'gene_id \"chr25.g21431\"; transcript_id \"blat.proc28144.chain_12888\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '447567', '447858', '100', '-', '.', 'gene_id \"chr25.g21433\"; transcript_id \"blat.proc4587.chain_3119\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26214719', '26214770', '100', '+', '.', 'gene_id \"chr25.g21484\"; transcript_id \"blat.proc4587.chain_11485\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '22201978', '22202018', '100', '-', '.', 'gene_id \"chr25.g21487\"; transcript_id \"blat.proc18128.chain_22693\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '17231444', '17231489', '100', '+', '.', 'gene_id \"chr25.g21584\"; transcript_id \"blat.proc28144.chain_11247\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '49292429', '49292478', '100', '+', '.', 'gene_id \"chr25.g21652\"; transcript_id \"blat.proc4005.chain_13232\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '9165313', '9165352', '100', '-', '.', 'gene_id \"chr25.g21710\"; transcript_id \"blat.proc18128.chain_26010\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '11807418', '11807450', '100', '+', '.', 'gene_id \"chr25.g21777\"; transcript_id \"blat.proc28144.chain_13227\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31242648', '31242699', '100', '-', '.', 'gene_id \"chr25.g21923\"; transcript_id \"blat.proc28144.chain_11046\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31244297', '31244355', '100', '+', '.', 'gene_id \"chr25.g22107\"; transcript_id \"blat.proc4587.chain_8417\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '46227345', '46227379', '100', '-', '.', 'gene_id \"chr25.g22645\"; transcript_id \"blat.proc18128.chain_23301\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '24935582', '24935633', '100', '-', '.', 'gene_id \"chr25.g22654\"; transcript_id \"blat.proc28144.chain_11047\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13154417', '13154454', '100', '+', '.', 'gene_id \"chr25.g22687\"; transcript_id \"blat.proc18128.chain_1582\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13823242', '13823272', '100', '+', '.', 'gene_id \"chr25.g22851\"; transcript_id \"blat.proc18128.chain_22457\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '31422259', '31422289', '100', '-', '.', 'gene_id \"chr25.g22884\"; transcript_id \"blat.proc18128.chain_16149\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '48208686', '48209210', '100', '-', '.', 'gene_id \"chr25.g23000\"; transcript_id \"blat.proc28144.chain_8116\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '2233908', '2233956', '100', '-', '.', 'gene_id \"chr25.g23139\"; transcript_id \"blat.proc4587.chain_11321\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '25591907', '25591953', '100', '-', '.', 'gene_id \"chr25.g23169\"; transcript_id \"blat.proc28144.chain_5869\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '13208022', '13208079', '100', '-', '.', 'gene_id \"chr25.g23433\"; transcript_id \"blat.proc18128.chain_24225\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '50237082', '50237282', '100', '+', '.', 'gene_id \"chr25.g23746\"; transcript_id \"blat.proc18128.chain_19733\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '29876197', '29876316', '100', '+', '.', 'gene_id \"chr25.g23781\"; transcript_id \"blat.proc18128.chain_25242\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '20496436', '20496522', '100', '+', '.', 'gene_id \"chr25.g23876\"; transcript_id \"blat.proc28144.chain_3420\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '21084446', '21084716', '100', '-', '.', 'gene_id \"chr25.g23920\"; transcript_id \"blat.proc18128.chain_7133\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '45520420', '45520476', '100', '+', '.', 'gene_id \"chr25.g24143\"; transcript_id \"blat.proc28144.chain_12401\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '1463236', '1463718', '100', '-', '.', 'gene_id \"chr25.g24557\"; transcript_id \"blat.proc18128.chain_7145\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '21401228', '21401276', '100', '-', '.', 'gene_id \"chr25.g24665\"; transcript_id \"blat.proc4587.chain_7402\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43401885', '43401954', '100', '+', '.', 'gene_id \"chr25.g24912\"; transcript_id \"blat.proc4587.chain_5190\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '5210094', '5210261', '100', '+', '.', 'gene_id \"chr25.g24999\"; transcript_id \"blat.proc18128.chain_22764\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '35890856', '35890890', '100', '+', '.', 'gene_id \"chr25.g25205\"; transcript_id \"blat.proc4587.chain_2507\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '25583034', '25583088', '100', '-', '.', 'gene_id \"chr25.g25210\"; transcript_id \"blat.proc18128.chain_23975\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '43609206', '43609247', '100', '+', '.', 'gene_id \"chr25.g25216\"; transcript_id \"blat.proc28144.chain_20722\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '37088584', '37088641', '100', '+', '.', 'gene_id \"chr25.g25232\"; transcript_id \"blat.proc18128.chain_24683\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '15982542', '15982596', '100', '+', '.', 'gene_id \"chr25.g25285\"; transcript_id \"blat.proc4005.chain_21313\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '28096869', '28096899', '100', '+', '.', 'gene_id \"chr25.g25456\"; transcript_id \"blat.proc28144.chain_3222\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '29353367', '29353429', '100', '+', '.', 'gene_id \"chr25.g25474\"; transcript_id \"blat.proc18128.chain_26016\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '25665368', '25665483', '100', '+', '.', 'gene_id \"chr25.g25642\"; transcript_id \"blat.proc4587.chain_1349\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '26675889', '26675935', '100', '-', '.', 'gene_id \"chr25.g25745\"; transcript_id \"blat.proc18128.chain_8650\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '41694191', '41694289', '100', '-', '.', 'gene_id \"chr25.g25865\"; transcript_id \"blat.proc4587.chain_1235\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '34288237', '34288288', '100', '+', '.', 'gene_id \"chr25.g26026\"; transcript_id \"blat.proc28144.chain_8442\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '447923', '448099', '100', '-', '.', 'gene_id \"chr25.g26104\"; transcript_id \"blat.proc4587.chain_3114\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '39338199', '39338261', '100', '+', '.', 'gene_id \"chr25.g26207\"; transcript_id \"blat.proc18128.chain_22958\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '30367030', '30367062', '100', '+', '.', 'gene_id \"chr25.g26237\"; transcript_id \"blat.proc28144.chain_18806\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '44688028', '44688065', '100', '-', '.', 'gene_id \"chr25.g26255\"; transcript_id \"blat.proc28144.chain_4644\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '3854719', '3854881', '100', '+', '.', 'gene_id \"chr25.g26416\"; transcript_id \"blat.proc18128.chain_13601\";']\n",
      "negative:  ['chr25', 'PASA', 'exon', '39375944', '39376087', '100', '+', '.', 'gene_id \"chr25.g26457\"; transcript_id \"blat.proc18128.chain_19570\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61541922', '61541953', '.', '+', '.', 'gene_id \"chr3.g21406\"; transcript_id \"chr3.g21406.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61722580', '61722690', '.', '+', '.', 'gene_id \"chr3.g21548\"; transcript_id \"chr3.g21548.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61758952', '61759300', '.', '+', '.', 'gene_id \"chr3.g21588\"; transcript_id \"chr3.g21588.i1\";']\n",
      "negative:  ['scaffold_1100', 'PASA', 'exon', '1201', '1422', '.', '+', '.', 'gene_id \"scaffold_1100.g4\"; transcript_id \"scaffold_1100.g4.i1\";']\n",
      "negative:  ['scaffold_1100', 'PASA', 'exon', '1411', '2154', '.', '+', '.', 'gene_id \"scaffold_1100.g5\"; transcript_id \"scaffold_1100.g5.i1\";']\n",
      "negative:  ['scaffold_1100', 'PASA', 'exon', '878', '1145', '100', '+', '.', 'gene_id \"scaffold_1100.g9\"; transcript_id \"blat.proc18128.chain_13221\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62098956', '62099086', '.', '-', '.', 'gene_id \"chr3.g21731\"; transcript_id \"chr3.g21731.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62114714', '62115204', '.', '-', '.', 'gene_id \"chr3.g21751\"; transcript_id \"chr3.g21751.i1\";']\n",
      "negative:  ['scaffold_1504', 'PASA', 'exon', '395', '462', '.', '-', '.', 'gene_id \"scaffold_1504.g1\"; transcript_id \"scaffold_1504.g1.i7\";']\n",
      "negative:  ['scaffold_1504', 'PASA', 'exon', '1', '64', '.', '-', '.', 'gene_id \"scaffold_1504.g1\"; transcript_id \"scaffold_1504.g1.i8\";']\n",
      "negative:  ['scaffold_1504', 'PASA', 'exon', '22422', '22462', '100', '-', '.', 'gene_id \"scaffold_1504.g17\"; transcript_id \"blat.proc28144.chain_4319\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62337600', '62337749', '.', '-', '.', 'gene_id \"chr3.g21893\"; transcript_id \"chr3.g21893.i5\";']\n",
      "negative:  ['scaffold_1584', 'PASA', 'exon', '13349', '13394', '100', '+', '.', 'gene_id \"scaffold_1584.g6\"; transcript_id \"blat.proc4005.chain_23415\";']\n",
      "negative:  ['scaffold_1584', 'PASA', 'exon', '10265', '10341', '100', '-', '.', 'gene_id \"scaffold_1584.g7\"; transcript_id \"blat.proc28144.chain_13169\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62612743', '62612962', '.', '+', '.', 'gene_id \"chr3.g21988\"; transcript_id \"chr3.g21988.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62612928', '62613202', '.', '-', '.', 'gene_id \"chr3.g21989\"; transcript_id \"chr3.g21989.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62709230', '62709277', '.', '+', '.', 'gene_id \"chr3.g22068\"; transcript_id \"chr3.g22068.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62996495', '62996644', '.', '+', '.', 'gene_id \"chr3.g22200\"; transcript_id \"chr3.g22200.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62995211', '62995383', '.', '+', '.', 'gene_id \"chr3.g22200\"; transcript_id \"chr3.g22200.i3\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '63189610', '63189654', '.', '+', '.', 'gene_id \"chr3.g22220\"; transcript_id \"chr3.g22220.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '199387', '199651', '.', '+', '.', 'gene_id \"chr11.g154\"; transcript_id \"chr11.g154.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '455130', '455344', '.', '-', '.', 'gene_id \"chr11.g318\"; transcript_id \"chr11.g318.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64141840', '64142071', '.', '-', '.', 'gene_id \"chr3.g22570\"; transcript_id \"chr3.g22570.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64608280', '64608572', '.', '+', '.', 'gene_id \"chr3.g22721\"; transcript_id \"chr3.g22721.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64608600', '64608980', '.', '-', '.', 'gene_id \"chr3.g22723\"; transcript_id \"chr3.g22723.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64823752', '64824059', '.', '-', '.', 'gene_id \"chr3.g22921\"; transcript_id \"chr3.g22921.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '65284193', '65284349', '.', '-', '.', 'gene_id \"chr3.g23144\"; transcript_id \"chr3.g23144.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '66129821', '66129923', '.', '-', '.', 'gene_id \"chr3.g23203\"; transcript_id \"chr3.g23203.i5\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '1736796', '1737016', '.', '+', '.', 'gene_id \"chr11.g1067\"; transcript_id \"chr11.g1067.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '1821085', '1821306', '.', '+', '.', 'gene_id \"chr11.g1151\"; transcript_id \"chr11.g1151.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '66727045', '66727075', '.', '-', '.', 'gene_id \"chr3.g23408\"; transcript_id \"chr3.g23408.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '67822767', '67822851', '.', '-', '.', 'gene_id \"chr3.g23462\"; transcript_id \"chr3.g23462.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '68053851', '68053917', '.', '+', '.', 'gene_id \"chr3.g23475\"; transcript_id \"chr3.g23475.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '69608931', '69609138', '.', '-', '.', 'gene_id \"chr3.g23790\"; transcript_id \"chr3.g23790.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '69881291', '69881728', '.', '-', '.', 'gene_id \"chr3.g23944\"; transcript_id \"chr3.g23944.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '70034899', '70034932', '.', '+', '.', 'gene_id \"chr3.g24078\"; transcript_id \"chr3.g24078.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71434898', '71435144', '.', '-', '.', 'gene_id \"chr3.g24364\"; transcript_id \"chr3.g24364.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71434173', '71434348', '.', '-', '.', 'gene_id \"chr3.g24374\"; transcript_id \"chr3.g24374.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr3', 'PASA', 'exon', '71515566', '71516043', '.', '-', '.', 'gene_id \"chr3.g24439\"; transcript_id \"chr3.g24439.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71648995', '71649216', '.', '-', '.', 'gene_id \"chr3.g24527\"; transcript_id \"chr3.g24527.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71768417', '71768467', '.', '-', '.', 'gene_id \"chr3.g24576\"; transcript_id \"chr3.g24576.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71837050', '71837082', '.', '+', '.', 'gene_id \"chr3.g24645\"; transcript_id \"chr3.g24645.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71842956', '71843175', '.', '-', '.', 'gene_id \"chr3.g24658\"; transcript_id \"chr3.g24658.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72076202', '72076242', '.', '+', '.', 'gene_id \"chr3.g24878\"; transcript_id \"chr3.g24878.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72092560', '72092732', '.', '-', '.', 'gene_id \"chr3.g24907\"; transcript_id \"chr3.g24907.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72198485', '72198770', '.', '+', '.', 'gene_id \"chr3.g25005\"; transcript_id \"chr3.g25005.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '5849197', '5849482', '.', '+', '.', 'gene_id \"chr11.g2865\"; transcript_id \"chr11.g2865.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72445545', '72445574', '.', '-', '.', 'gene_id \"chr3.g25083\"; transcript_id \"chr3.g25083.i3\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72750855', '72751067', '.', '-', '.', 'gene_id \"chr3.g25290\"; transcript_id \"chr3.g25290.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72778743', '72779142', '.', '+', '.', 'gene_id \"chr3.g25315\"; transcript_id \"chr3.g25315.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72788724', '72789050', '.', '+', '.', 'gene_id \"chr3.g25329\"; transcript_id \"chr3.g25329.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72790119', '72790388', '.', '+', '.', 'gene_id \"chr3.g25331\"; transcript_id \"chr3.g25331.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '6401768', '6401815', '.', '+', '.', 'gene_id \"chr11.g3144\"; transcript_id \"chr11.g3144.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72819020', '72819174', '.', '+', '.', 'gene_id \"chr3.g25370\"; transcript_id \"chr3.g25370.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72819223', '72819516', '.', '-', '.', 'gene_id \"chr3.g25372\"; transcript_id \"chr3.g25372.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '7459339', '7459370', '.', '+', '.', 'gene_id \"chr11.g3199\"; transcript_id \"chr11.g3199.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72920480', '72920661', '.', '-', '.', 'gene_id \"chr3.g25461\"; transcript_id \"chr3.g25461.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72920767', '72921061', '.', '-', '.', 'gene_id \"chr3.g25462\"; transcript_id \"chr3.g25462.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '8150777', '8151056', '.', '+', '.', 'gene_id \"chr11.g3278\"; transcript_id \"chr11.g3278.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '8151497', '8151791', '.', '+', '.', 'gene_id \"chr11.g3279\"; transcript_id \"chr11.g3279.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73000595', '73000642', '.', '-', '.', 'gene_id \"chr3.g25526\"; transcript_id \"chr3.g25526.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73021609', '73022527', '.', '+', '.', 'gene_id \"chr3.g25572\"; transcript_id \"chr3.g25572.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73022679', '73022835', '.', '-', '.', 'gene_id \"chr3.g25573\"; transcript_id \"chr3.g25573.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73083401', '73083430', '.', '+', '.', 'gene_id \"chr3.g25586\"; transcript_id \"chr3.g25586.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73101579', '73101613', '.', '-', '.', 'gene_id \"chr3.g25588\"; transcript_id \"chr3.g25588.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '8512560', '8512746', '.', '+', '.', 'gene_id \"chr11.g3419\"; transcript_id \"chr11.g3419.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73361180', '73361501', '.', '-', '.', 'gene_id \"chr3.g25810\"; transcript_id \"chr3.g25810.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73389721', '73389883', '.', '+', '.', 'gene_id \"chr3.g25841\"; transcript_id \"chr3.g25841.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '8743359', '8743606', '.', '+', '.', 'gene_id \"chr11.g3655\"; transcript_id \"chr11.g3655.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73682955', '73683338', '.', '+', '.', 'gene_id \"chr3.g26034\"; transcript_id \"chr3.g26034.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '73683394', '73683605', '.', '+', '.', 'gene_id \"chr3.g26036\"; transcript_id \"chr3.g26036.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '9397427', '9397606', '.', '-', '.', 'gene_id \"chr11.g3876\"; transcript_id \"chr11.g3876.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '9397632', '9397869', '.', '-', '.', 'gene_id \"chr11.g3877\"; transcript_id \"chr11.g3877.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74019528', '74019898', '.', '+', '.', 'gene_id \"chr3.g26147\"; transcript_id \"chr3.g26147.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '9802469', '9802518', '.', '+', '.', 'gene_id \"chr11.g3918\"; transcript_id \"chr11.g3918.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74113908', '74114159', '.', '-', '.', 'gene_id \"chr3.g26178\"; transcript_id \"chr3.g26178.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74114612', '74115014', '.', '-', '.', 'gene_id \"chr3.g26179\"; transcript_id \"chr3.g26179.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '9989691', '9989957', '.', '-', '.', 'gene_id \"chr11.g3954\"; transcript_id \"chr11.g3954.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '9993543', '9993915', '.', '+', '.', 'gene_id \"chr11.g3964\"; transcript_id \"chr11.g3964.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '10158118', '10158181', '.', '+', '.', 'gene_id \"chr11.g3987\"; transcript_id \"chr11.g3987.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '11004494', '11004705', '.', '-', '.', 'gene_id \"chr11.g4115\"; transcript_id \"chr11.g4115.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74415614', '74416046', '.', '+', '.', 'gene_id \"chr3.g26394\"; transcript_id \"chr3.g26394.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74553674', '74553835', '.', '-', '.', 'gene_id \"chr3.g26427\"; transcript_id \"chr3.g26427.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74666422', '74666609', '.', '+', '.', 'gene_id \"chr3.g26553\"; transcript_id \"chr3.g26553.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '11797268', '11797456', '.', '+', '.', 'gene_id \"chr11.g4374\"; transcript_id \"chr11.g4374.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '75635277', '75635385', '.', '+', '.', 'gene_id \"chr3.g26732\"; transcript_id \"chr3.g26732.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '75762999', '75763034', '.', '+', '.', 'gene_id \"chr3.g26788\"; transcript_id \"chr3.g26788.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '76342733', '76342764', '.', '+', '.', 'gene_id \"chr3.g26889\"; transcript_id \"chr3.g26889.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '12604529', '12604946', '.', '+', '.', 'gene_id \"chr11.g4669\"; transcript_id \"chr11.g4669.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '12605363', '12605647', '.', '-', '.', 'gene_id \"chr11.g4670\"; transcript_id \"chr11.g4670.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '77548479', '77548514', '.', '+', '.', 'gene_id \"chr3.g26953\"; transcript_id \"chr3.g26953.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '78036897', '78037094', '.', '-', '.', 'gene_id \"chr3.g26980\"; transcript_id \"chr3.g26980.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '79158994', '79159560', '.', '+', '.', 'gene_id \"chr3.g27040\"; transcript_id \"chr3.g27040.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '79158994', '79159476', '.', '+', '.', 'gene_id \"chr3.g27040\"; transcript_id \"chr3.g27040.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80150242', '80150435', '.', '-', '.', 'gene_id \"chr3.g27182\"; transcript_id \"chr3.g27182.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80166208', '80166240', '.', '+', '.', 'gene_id \"chr3.g27195\"; transcript_id \"chr3.g27195.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80192755', '80193018', '.', '-', '.', 'gene_id \"chr3.g27214\"; transcript_id \"chr3.g27214.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80192182', '80192776', '.', '+', '.', 'gene_id \"chr3.g27217\"; transcript_id \"chr3.g27217.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '13801001', '13801370', '.', '-', '.', 'gene_id \"chr11.g5048\"; transcript_id \"chr11.g5048.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80393257', '80393859', '.', '+', '.', 'gene_id \"chr3.g27342\"; transcript_id \"chr3.g27342.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80393871', '80394097', '.', '-', '.', 'gene_id \"chr3.g27343\"; transcript_id \"chr3.g27343.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80394148', '80394472', '.', '-', '.', 'gene_id \"chr3.g27344\"; transcript_id \"chr3.g27344.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80395029', '80395414', '.', '-', '.', 'gene_id \"chr3.g27345\"; transcript_id \"chr3.g27345.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '80416281', '80416319', '.', '-', '.', 'gene_id \"chr3.g27362\"; transcript_id \"chr3.g27362.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '82090992', '82091037', '.', '-', '.', 'gene_id \"chr3.g27533\"; transcript_id \"chr3.g27533.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '15194281', '15194510', '.', '-', '.', 'gene_id \"chr11.g5338\"; transcript_id \"chr11.g5338.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '83077751', '83077811', '.', '-', '.', 'gene_id \"chr3.g27631\"; transcript_id \"chr3.g27631.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '83302392', '83302874', '.', '-', '.', 'gene_id \"chr3.g27642\"; transcript_id \"chr3.g27642.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '83676328', '83676492', '.', '+', '.', 'gene_id \"chr3.g27702\"; transcript_id \"chr3.g27702.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '83676487', '83676654', '.', '-', '.', 'gene_id \"chr3.g27703\"; transcript_id \"chr3.g27703.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '84166007', '84166262', '.', '-', '.', 'gene_id \"chr3.g27910\"; transcript_id \"chr3.g27910.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '15699163', '15699385', '.', '+', '.', 'gene_id \"chr11.g5807\"; transcript_id \"chr11.g5807.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '15699405', '15699601', '.', '-', '.', 'gene_id \"chr11.g5808\"; transcript_id \"chr11.g5808.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '84367956', '84368425', '.', '+', '.', 'gene_id \"chr3.g28074\"; transcript_id \"chr3.g28074.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '16728377', '16728728', '.', '-', '.', 'gene_id \"chr11.g6067\"; transcript_id \"chr11.g6067.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '85112132', '85112632', '.', '+', '.', 'gene_id \"chr3.g28370\"; transcript_id \"chr3.g28370.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '85110807', '85110963', '.', '-', '.', 'gene_id \"chr3.g28371\"; transcript_id \"chr3.g28371.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '85112691', '85112881', '.', '-', '.', 'gene_id \"chr3.g28372\"; transcript_id \"chr3.g28372.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '85207491', '85207642', '.', '-', '.', 'gene_id \"chr3.g28439\"; transcript_id \"chr3.g28439.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '85413403', '85413571', '.', '-', '.', 'gene_id \"chr3.g28512\"; transcript_id \"chr3.g28512.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '17896156', '17896546', '.', '+', '.', 'gene_id \"chr11.g6264\"; transcript_id \"chr11.g6264.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '18677572', '18677623', '.', '+', '.', 'gene_id \"chr11.g6413\"; transcript_id \"chr11.g6413.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '19032972', '19035602', '.', '-', '.', 'gene_id \"chr11.g6450\"; transcript_id \"chr11.g6450.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '86389811', '86389864', '.', '+', '.', 'gene_id \"chr3.g28761\"; transcript_id \"chr3.g28761.i5\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '19251745', '19252018', '.', '-', '.', 'gene_id \"chr11.g6611\"; transcript_id \"chr11.g6611.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '19271611', '19271784', '.', '+', '.', 'gene_id \"chr11.g6631\"; transcript_id \"chr11.g6631.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87509893', '87510220', '.', '+', '.', 'gene_id \"chr3.g29054\"; transcript_id \"chr3.g29054.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87510301', '87510544', '.', '-', '.', 'gene_id \"chr3.g29055\"; transcript_id \"chr3.g29055.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87510531', '87510740', '.', '+', '.', 'gene_id \"chr3.g29056\"; transcript_id \"chr3.g29056.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87568879', '87569747', '.', '+', '.', 'gene_id \"chr3.g29086\"; transcript_id \"chr3.g29086.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87619005', '87619041', '.', '+', '.', 'gene_id \"chr3.g29101\"; transcript_id \"chr3.g29101.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87619012', '87619048', '.', '+', '.', 'gene_id \"chr3.g29101\"; transcript_id \"chr3.g29101.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87934475', '87934683', '.', '-', '.', 'gene_id \"chr3.g29130\"; transcript_id \"chr3.g29130.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '87962623', '87962670', '.', '+', '.', 'gene_id \"chr3.g29135\"; transcript_id \"chr3.g29135.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '88268846', '88269073', '.', '+', '.', 'gene_id \"chr3.g29188\"; transcript_id \"chr3.g29188.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '88395169', '88395209', '.', '+', '.', 'gene_id \"chr3.g29216\"; transcript_id \"chr3.g29216.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '88992852', '88993188', '.', '+', '.', 'gene_id \"chr3.g29311\"; transcript_id \"chr3.g29311.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '89118975', '89119223', '.', '-', '.', 'gene_id \"chr3.g29405\"; transcript_id \"chr3.g29405.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '89456012', '89456222', '.', '+', '.', 'gene_id \"chr3.g29626\"; transcript_id \"chr3.g29626.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '20415325', '20415616', '.', '+', '.', 'gene_id \"chr11.g7372\"; transcript_id \"chr11.g7372.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '89619100', '89619135', '.', '+', '.', 'gene_id \"chr3.g29635\"; transcript_id \"chr3.g29635.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '89955998', '89956232', '.', '-', '.', 'gene_id \"chr3.g29657\"; transcript_id \"chr3.g29657.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr3', 'PASA', 'exon', '90935946', '90935980', '.', '+', '.', 'gene_id \"chr3.g29751\"; transcript_id \"chr3.g29751.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '90972651', '90972753', '.', '+', '.', 'gene_id \"chr3.g29758\"; transcript_id \"chr3.g29758.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91353120', '91353536', '.', '+', '.', 'gene_id \"chr3.g29858\"; transcript_id \"chr3.g29858.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91363463', '91363750', '.', '-', '.', 'gene_id \"chr3.g29867\"; transcript_id \"chr3.g29867.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91363178', '91363478', '.', '-', '.', 'gene_id \"chr3.g29869\"; transcript_id \"chr3.g29869.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91364119', '91364362', '.', '+', '.', 'gene_id \"chr3.g29870\"; transcript_id \"chr3.g29870.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91809753', '91809908', '.', '-', '.', 'gene_id \"chr3.g30207\"; transcript_id \"chr3.g30207.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91809917', '91810104', '.', '-', '.', 'gene_id \"chr3.g30208\"; transcript_id \"chr3.g30208.i1\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91874424', '91874650', '.', '-', '.', 'gene_id \"chr3.g30243\"; transcript_id \"chr3.g30243.i2\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '76783515', '76783547', '100', '-', '.', 'gene_id \"chr3.g30613\"; transcript_id \"blat.proc18128.chain_9125\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9692081', '9692148', '100', '+', '.', 'gene_id \"chr3.g30750\"; transcript_id \"blat.proc18128.chain_7070\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '49432641', '49432910', '100', '-', '.', 'gene_id \"chr3.g30782\"; transcript_id \"blat.proc28144.chain_1650\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '1583764', '1583921', '100', '+', '.', 'gene_id \"chr3.g30979\"; transcript_id \"blat.proc28144.chain_16763\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '9892304', '9892333', '100', '+', '.', 'gene_id \"chr3.g30983\"; transcript_id \"blat.proc28144.chain_4530\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74228826', '74228940', '100', '+', '.', 'gene_id \"chr3.g31098\"; transcript_id \"blat.proc18128.chain_14421\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31873382', '31873428', '100', '+', '.', 'gene_id \"chr3.g31186\"; transcript_id \"blat.proc18128.chain_5302\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '47542292', '47542332', '100', '+', '.', 'gene_id \"chr3.g31195\"; transcript_id \"blat.proc18128.chain_8666\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46579369', '46579586', '100', '+', '.', 'gene_id \"chr3.g31214\"; transcript_id \"blat.proc18128.chain_14699\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '15478146', '15478952', '100', '-', '.', 'gene_id \"chr3.g31334\"; transcript_id \"blat.proc18128.chain_13164\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30162111', '30162148', '100', '+', '.', 'gene_id \"chr3.g31409\"; transcript_id \"blat.proc28144.chain_13519\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '51729864', '51729907', '100', '+', '.', 'gene_id \"chr3.g31598\"; transcript_id \"blat.proc18128.chain_18728\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26656740', '26656782', '100', '+', '.', 'gene_id \"chr3.g31810\"; transcript_id \"blat.proc4005.chain_1532\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '77403623', '77403700', '100', '-', '.', 'gene_id \"chr3.g31850\"; transcript_id \"blat.proc18128.chain_14873\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33212514', '33212590', '100', '+', '.', 'gene_id \"chr3.g31887\"; transcript_id \"blat.proc28144.chain_13165\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '17605876', '17605937', '100', '-', '.', 'gene_id \"chr3.g31928\"; transcript_id \"blat.proc18128.chain_13818\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '23313393', '23313434', '100', '-', '.', 'gene_id \"chr3.g31965\"; transcript_id \"blat.proc18128.chain_18196\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '67025120', '67025156', '100', '+', '.', 'gene_id \"chr3.g32025\"; transcript_id \"blat.proc18128.chain_7052\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91360527', '91360681', '100', '-', '.', 'gene_id \"chr3.g32065\"; transcript_id \"blat.proc18128.chain_21747\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '43931562', '43931658', '100', '-', '.', 'gene_id \"chr3.g32171\"; transcript_id \"blat.proc4587.chain_3081\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '55300795', '55300843', '100', '-', '.', 'gene_id \"chr3.g32190\"; transcript_id \"blat.proc4587.chain_9613\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28870751', '28870795', '100', '+', '.', 'gene_id \"chr3.g32319\"; transcript_id \"blat.proc4005.chain_1313\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '88021448', '88021480', '100', '-', '.', 'gene_id \"chr3.g32411\"; transcript_id \"blat.proc28144.chain_133\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '57486142', '57486193', '100', '+', '.', 'gene_id \"chr3.g32417\"; transcript_id \"blat.proc18128.chain_1948\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '75233155', '75233253', '100', '-', '.', 'gene_id \"chr3.g32422\"; transcript_id \"blat.proc4587.chain_3118\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13339197', '13339254', '100', '+', '.', 'gene_id \"chr3.g32491\"; transcript_id \"blat.proc18128.chain_5758\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '36016241', '36016272', '100', '-', '.', 'gene_id \"chr3.g32554\"; transcript_id \"blat.proc28144.chain_23472\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '32057979', '32058020', '100', '+', '.', 'gene_id \"chr3.g32592\"; transcript_id \"blat.proc18128.chain_8865\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '71323823', '71323857', '100', '+', '.', 'gene_id \"chr3.g32632\"; transcript_id \"blat.proc18128.chain_22171\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61920140', '61920199', '100', '-', '.', 'gene_id \"chr3.g32647\"; transcript_id \"blat.proc18128.chain_6853\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '68781223', '68781796', '100', '+', '.', 'gene_id \"chr3.g32666\"; transcript_id \"blat.proc18128.chain_19929\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26595845', '26595882', '100', '+', '.', 'gene_id \"chr3.g32681\"; transcript_id \"blat.proc4587.chain_4365\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '81724619', '81724654', '100', '-', '.', 'gene_id \"chr3.g32753\"; transcript_id \"blat.proc18128.chain_14356\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40490998', '40491034', '100', '+', '.', 'gene_id \"chr3.g32816\"; transcript_id \"blat.proc28144.chain_7140\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '32560569', '32560616', '100', '+', '.', 'gene_id \"chr3.g32868\"; transcript_id \"blat.proc18128.chain_17205\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '66748997', '66749056', '100', '-', '.', 'gene_id \"chr3.g32923\"; transcript_id \"blat.proc18128.chain_23042\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '79800256', '79800316', '100', '-', '.', 'gene_id \"chr3.g32938\"; transcript_id \"blat.proc4005.chain_1739\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '33558059', '33558174', '100', '-', '.', 'gene_id \"chr3.g32976\"; transcript_id \"blat.proc4587.chain_268\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29258848', '29259028', '100', '+', '.', 'gene_id \"chr3.g33147\"; transcript_id \"blat.proc28144.chain_12996\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '45703185', '45703238', '100', '+', '.', 'gene_id \"chr3.g33167\"; transcript_id \"blat.proc4587.chain_13566\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '30446932', '30447223', '100', '-', '.', 'gene_id \"chr3.g33204\"; transcript_id \"blat.proc28144.chain_4365\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31670430', '31671004', '100', '-', '.', 'gene_id \"chr3.g33249\"; transcript_id \"blat.proc4005.chain_5200\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46740080', '46740191', '100', '+', '.', 'gene_id \"chr3.g33368\"; transcript_id \"blat.proc18128.chain_7237\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '81390207', '81390252', '100', '+', '.', 'gene_id \"chr3.g33395\"; transcript_id \"blat.proc18128.chain_16694\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '25122119', '25122205', '100', '+', '.', 'gene_id \"chr3.g33396\"; transcript_id \"blat.proc4005.chain_2969\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41965677', '41965744', '100', '+', '.', 'gene_id \"chr3.g33476\"; transcript_id \"blat.proc4005.chain_23397\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '42052639', '42052670', '100', '-', '.', 'gene_id \"chr3.g33528\"; transcript_id \"blat.proc4005.chain_20574\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3155736', '3155769', '100', '-', '.', 'gene_id \"chr3.g33535\"; transcript_id \"blat.proc28144.chain_9812\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31583577', '31583757', '100', '-', '.', 'gene_id \"chr3.g33644\"; transcript_id \"blat.proc18128.chain_19911\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '79508725', '79508773', '100', '-', '.', 'gene_id \"chr3.g33666\"; transcript_id \"blat.proc28144.chain_17416\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '88044628', '88044660', '100', '+', '.', 'gene_id \"chr3.g33773\"; transcript_id \"blat.proc18128.chain_643\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19345832', '19345867', '100', '+', '.', 'gene_id \"chr3.g33797\"; transcript_id \"blat.proc4005.chain_2843\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52885603', '52885640', '100', '-', '.', 'gene_id \"chr3.g33873\"; transcript_id \"blat.proc28144.chain_13269\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91070660', '91070740', '100', '-', '.', 'gene_id \"chr3.g34019\"; transcript_id \"blat.proc18128.chain_15845\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35481862', '35481904', '100', '+', '.', 'gene_id \"chr3.g34245\"; transcript_id \"blat.proc4005.chain_11150\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26613665', '26613699', '100', '-', '.', 'gene_id \"chr3.g34259\"; transcript_id \"blat.proc28144.chain_12407\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '68166987', '68167020', '100', '+', '.', 'gene_id \"chr3.g34356\"; transcript_id \"blat.proc18128.chain_7837\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '84982898', '84982927', '100', '-', '.', 'gene_id \"chr3.g34357\"; transcript_id \"blat.proc4587.chain_600\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '45702195', '45702230', '100', '-', '.', 'gene_id \"chr3.g34391\"; transcript_id \"blat.proc18128.chain_10799\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '5694391', '5694426', '100', '+', '.', 'gene_id \"chr3.g34396\"; transcript_id \"blat.proc4587.chain_8140\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '12341940', '12341978', '100', '-', '.', 'gene_id \"chr3.g34443\"; transcript_id \"blat.proc4587.chain_3960\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62834727', '62834763', '100', '-', '.', 'gene_id \"chr3.g34448\"; transcript_id \"blat.proc28144.chain_3960\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '83728701', '83728754', '100', '+', '.', 'gene_id \"chr3.g34457\"; transcript_id \"blat.proc4005.chain_20914\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3404966', '3405012', '100', '+', '.', 'gene_id \"chr3.g34494\"; transcript_id \"blat.proc28144.chain_20721\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13873631', '13873693', '100', '+', '.', 'gene_id \"chr3.g34507\"; transcript_id \"blat.proc28144.chain_14609\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '18521063', '18521100', '100', '-', '.', 'gene_id \"chr3.g34519\"; transcript_id \"blat.proc18128.chain_22578\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53643830', '53644572', '100', '+', '.', 'gene_id \"chr3.g34641\"; transcript_id \"blat.proc4587.chain_7562\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13771287', '13771343', '100', '-', '.', 'gene_id \"chr3.g34776\"; transcript_id \"blat.proc28144.chain_10962\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '59220201', '59220230', '100', '+', '.', 'gene_id \"chr3.g34805\"; transcript_id \"blat.proc28144.chain_12524\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72320035', '72320103', '100', '+', '.', 'gene_id \"chr3.g34894\"; transcript_id \"blat.proc18128.chain_6675\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '59081385', '59081558', '100', '+', '.', 'gene_id \"chr3.g34954\"; transcript_id \"blat.proc4587.chain_7239\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '3452722', '3452752', '100', '+', '.', 'gene_id \"chr3.g35034\"; transcript_id \"blat.proc4005.chain_22421\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '75704980', '75705013', '100', '+', '.', 'gene_id \"chr3.g35108\"; transcript_id \"blat.proc18128.chain_7298\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '61225234', '61225279', '100', '-', '.', 'gene_id \"chr3.g35174\"; transcript_id \"blat.proc18128.chain_2292\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64710233', '64710360', '100', '-', '.', 'gene_id \"chr3.g35229\"; transcript_id \"blat.proc18128.chain_17037\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '36916369', '36916508', '100', '-', '.', 'gene_id \"chr3.g35232\"; transcript_id \"blat.proc28144.chain_16078\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '6965468', '6965643', '100', '-', '.', 'gene_id \"chr3.g35427\"; transcript_id \"blat.proc28144.chain_5192\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '90617059', '90617099', '100', '+', '.', 'gene_id \"chr3.g35479\"; transcript_id \"blat.proc4005.chain_10830\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '46769148', '46769179', '100', '+', '.', 'gene_id \"chr3.g35527\"; transcript_id \"blat.proc28144.chain_23473\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '75233318', '75233494', '100', '-', '.', 'gene_id \"chr3.g35718\"; transcript_id \"blat.proc4587.chain_3113\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29830830', '29830866', '100', '-', '.', 'gene_id \"chr3.g35727\"; transcript_id \"blat.proc28144.chain_20354\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '58993650', '58993691', '100', '-', '.', 'gene_id \"chr3.g35745\"; transcript_id \"blat.proc4587.chain_6025\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '64077843', '64077907', '100', '+', '.', 'gene_id \"chr3.g36220\"; transcript_id \"blat.proc4587.chain_9084\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '72505653', '72505742', '100', '+', '.', 'gene_id \"chr3.g36304\"; transcript_id \"blat.proc28144.chain_7414\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '45748098', '45748141', '100', '-', '.', 'gene_id \"chr3.g36309\"; transcript_id \"blat.proc18128.chain_25107\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '13098128', '13098189', '100', '+', '.', 'gene_id \"chr3.g36340\"; transcript_id \"blat.proc18128.chain_26947\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '34926732', '34926771', '100', '+', '.', 'gene_id \"chr3.g36349\"; transcript_id \"blat.proc18128.chain_2840\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '81488494', '81488535', '100', '+', '.', 'gene_id \"chr3.g36378\"; transcript_id \"blat.proc4587.chain_1486\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28773426', '28773474', '100', '+', '.', 'gene_id \"chr3.g36440\"; transcript_id \"blat.proc18128.chain_19857\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '34812953', '34812987', '100', '+', '.', 'gene_id \"chr3.g36513\"; transcript_id \"blat.proc4587.chain_9297\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '56977786', '56977905', '100', '-', '.', 'gene_id \"chr3.g36549\"; transcript_id \"blat.proc18128.chain_16457\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '90045850', '90045879', '100', '+', '.', 'gene_id \"chr3.g36642\"; transcript_id \"blat.proc28144.chain_4529\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '23102547', '23102592', '100', '-', '.', 'gene_id \"chr3.g36656\"; transcript_id \"blat.proc4587.chain_741\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '10255080', '10255129', '100', '+', '.', 'gene_id \"chr3.g36661\"; transcript_id \"blat.proc28144.chain_18367\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '40247668', '40247856', '100', '-', '.', 'gene_id \"chr3.g36726\"; transcript_id \"blat.proc4005.chain_23065\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '35882152', '35882301', '100', '+', '.', 'gene_id \"chr3.g36831\"; transcript_id \"blat.proc4587.chain_11371\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '1716019', '1716065', '100', '-', '.', 'gene_id \"chr3.g36854\"; transcript_id \"blat.proc4005.chain_16553\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '66389450', '66389491', '100', '-', '.', 'gene_id \"chr3.g36883\"; transcript_id \"blat.proc18128.chain_22451\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '24557405', '24557467', '100', '-', '.', 'gene_id \"chr3.g36900\"; transcript_id \"blat.proc4587.chain_6557\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '74862855', '74862896', '100', '+', '.', 'gene_id \"chr3.g36901\"; transcript_id \"blat.proc18128.chain_25680\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '79031039', '79031076', '100', '+', '.', 'gene_id \"chr3.g36914\"; transcript_id \"blat.proc18128.chain_16636\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '4416067', '4416112', '100', '-', '.', 'gene_id \"chr3.g36927\"; transcript_id \"blat.proc4005.chain_11011\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '31580156', '31580211', '100', '-', '.', 'gene_id \"chr3.g36993\"; transcript_id \"blat.proc18128.chain_17613\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '10656096', '10656151', '100', '-', '.', 'gene_id \"chr3.g37138\"; transcript_id \"blat.proc4587.chain_4467\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '89111679', '89111727', '100', '-', '.', 'gene_id \"chr3.g37178\"; transcript_id \"blat.proc28144.chain_18608\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '52331002', '52331033', '100', '-', '.', 'gene_id \"chr3.g37255\"; transcript_id \"blat.proc18128.chain_22620\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62733353', '62733401', '100', '-', '.', 'gene_id \"chr3.g37261\"; transcript_id \"blat.proc4005.chain_14153\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29598344', '29598399', '100', '+', '.', 'gene_id \"chr3.g37437\"; transcript_id \"blat.proc28144.chain_6816\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '26336281', '26336504', '100', '+', '.', 'gene_id \"chr3.g37518\"; transcript_id \"blat.proc18128.chain_19919\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '50917901', '50917932', '100', '-', '.', 'gene_id \"chr3.g37549\"; transcript_id \"blat.proc4587.chain_4457\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '19936873', '19936905', '100', '+', '.', 'gene_id \"chr3.g37572\"; transcript_id \"blat.proc4587.chain_12517\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '36365738', '36365782', '100', '-', '.', 'gene_id \"chr3.g37591\"; transcript_id \"blat.proc28144.chain_3559\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '18368730', '18368774', '100', '-', '.', 'gene_id \"chr3.g37638\"; transcript_id \"blat.proc18128.chain_19961\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '28306718', '28306763', '100', '-', '.', 'gene_id \"chr3.g37747\"; transcript_id \"blat.proc18128.chain_24610\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '39113115', '39113174', '100', '+', '.', 'gene_id \"chr3.g37839\"; transcript_id \"blat.proc28144.chain_3149\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '91334311', '91334355', '100', '-', '.', 'gene_id \"chr3.g37910\"; transcript_id \"blat.proc18128.chain_12174\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '48169919', '48169960', '100', '+', '.', 'gene_id \"chr3.g37923\"; transcript_id \"blat.proc28144.chain_16420\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '22267636', '22267853', '100', '+', '.', 'gene_id \"chr3.g37942\"; transcript_id \"blat.proc18128.chain_14698\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '53073725', '53073758', '100', '+', '.', 'gene_id \"chr3.g38104\"; transcript_id \"blat.proc28144.chain_2748\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '37825967', '37826052', '100', '+', '.', 'gene_id \"chr3.g38195\"; transcript_id \"blat.proc28144.chain_8157\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '70639048', '70639129', '100', '-', '.', 'gene_id \"chr3.g38440\"; transcript_id \"blat.proc18128.chain_14195\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '20928563', '20928789', '100', '+', '.', 'gene_id \"chr3.g38442\"; transcript_id \"blat.proc18128.chain_5013\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '76351246', '76351425', '100', '-', '.', 'gene_id \"chr3.g38526\"; transcript_id \"blat.proc28144.chain_20981\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '41846144', '41846180', '100', '+', '.', 'gene_id \"chr3.g38529\"; transcript_id \"blat.proc18128.chain_1580\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '62006863', '62006897', '100', '+', '.', 'gene_id \"chr3.g38578\"; transcript_id \"blat.proc18128.chain_17127\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '66296934', '66296991', '100', '+', '.', 'gene_id \"chr3.g38580\"; transcript_id \"blat.proc4587.chain_5520\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '54297033', '54297071', '100', '+', '.', 'gene_id \"chr3.g38591\"; transcript_id \"blat.proc18128.chain_22654\";']\n",
      "negative:  ['chr3', 'PASA', 'exon', '29540018', '29540064', '100', '-', '.', 'gene_id \"chr3.g38718\"; transcript_id \"blat.proc28144.chain_3777\";']\n",
      "negative:  ['scaffold_917', 'PASA', 'exon', '8055', '8118', '.', '-', '.', 'gene_id \"scaffold_917.g1\"; transcript_id \"scaffold_917.g1.i1\";']\n",
      "negative:  ['scaffold_1696', 'PASA', 'exon', '123574', '123618', '.', '+', '.', 'gene_id \"scaffold_1696.g14\"; transcript_id \"scaffold_1696.g14.i1\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '22406', '22634', '.', '-', '.', 'gene_id \"scaffold_1564.g15\"; transcript_id \"scaffold_1564.g15.i2\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '22965', '23420', '.', '-', '.', 'gene_id \"scaffold_1564.g15\"; transcript_id \"scaffold_1564.g15.i3\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '23405', '23916', '.', '+', '.', 'gene_id \"scaffold_1564.g16\"; transcript_id \"scaffold_1564.g16.i1\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '23405', '23907', '.', '+', '.', 'gene_id \"scaffold_1564.g16\"; transcript_id \"scaffold_1564.g16.i2\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '27637', '27821', '.', '-', '.', 'gene_id \"scaffold_1564.g24\"; transcript_id \"scaffold_1564.g24.i1\";']\n",
      "negative:  ['scaffold_1564', 'PASA', 'exon', '27420', '27622', '.', '-', '.', 'gene_id \"scaffold_1564.g26\"; transcript_id \"scaffold_1564.g26.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '23221698', '23221961', '.', '+', '.', 'gene_id \"chr11.g8728\"; transcript_id \"chr11.g8728.i1\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '3541', '3575', '.', '+', '.', 'gene_id \"scaffold_1511.g1\"; transcript_id \"scaffold_1511.g1.i1\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '6827', '6986', '.', '-', '.', 'gene_id \"scaffold_1511.g2\"; transcript_id \"scaffold_1511.g2.i1\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '15347', '16859', '.', '-', '.', 'gene_id \"scaffold_1511.g14\"; transcript_id \"scaffold_1511.g14.i1\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '15960', '16032', '.', '-', '.', 'gene_id \"scaffold_1511.g14\"; transcript_id \"scaffold_1511.g14.i2\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '15179', '15347', '.', '-', '.', 'gene_id \"scaffold_1511.g15\"; transcript_id \"scaffold_1511.g15.i1\";']\n",
      "negative:  ['scaffold_1511', 'PASA', 'exon', '6109', '6602', '100', '-', '.', 'gene_id \"scaffold_1511.g44\"; transcript_id \"blat.proc4587.chain_3182\";']\n",
      "negative:  ['scaffold_1484', 'PASA', 'exon', '11708', '11792', '.', '+', '.', 'gene_id \"scaffold_1484.g16\"; transcript_id \"scaffold_1484.g16.i1\";']\n",
      "negative:  ['scaffold_1484', 'PASA', 'exon', '14064', '14471', '.', '-', '.', 'gene_id \"scaffold_1484.g18\"; transcript_id \"scaffold_1484.g18.i2\";']\n",
      "negative:  ['scaffold_1484', 'PASA', 'exon', '18116', '18260', '.', '-', '.', 'gene_id \"scaffold_1484.g18\"; transcript_id \"scaffold_1484.g18.i2\";']\n",
      "negative:  ['scaffold_1720', 'PASA', 'exon', '5269', '5311', '.', '+', '.', 'gene_id \"scaffold_1720.g1\"; transcript_id \"scaffold_1720.g1.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '24906723', '24906929', '.', '-', '.', 'gene_id \"chr11.g9074\"; transcript_id \"chr11.g9074.i1\";']\n",
      "negative:  ['scaffold_1825', 'PASA', 'exon', '5369', '5541', '.', '-', '.', 'gene_id \"scaffold_1825.g6\"; transcript_id \"scaffold_1825.g6.i1\";']\n",
      "negative:  ['scaffold_1825', 'PASA', 'exon', '5990', '6367', '.', '-', '.', 'gene_id \"scaffold_1825.g7\"; transcript_id \"scaffold_1825.g7.i1\";']\n",
      "negative:  ['scaffold_1825', 'PASA', 'exon', '6162', '6381', '.', '-', '.', 'gene_id \"scaffold_1825.g7\"; transcript_id \"scaffold_1825.g7.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25575005', '25575161', '.', '-', '.', 'gene_id \"chr11.g9521\"; transcript_id \"chr11.g9521.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25640815', '25640992', '.', '-', '.', 'gene_id \"chr11.g9553\"; transcript_id \"chr11.g9553.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25640978', '25641155', '.', '+', '.', 'gene_id \"chr11.g9554\"; transcript_id \"chr11.g9554.i1\";']\n",
      "negative:  ['scaffold_1348', 'PASA', 'exon', '16513', '16569', '.', '+', '.', 'gene_id \"scaffold_1348.g16\"; transcript_id \"scaffold_1348.g16.i1\";']\n",
      "negative:  ['scaffold_1348', 'PASA', 'exon', '5995', '6270', '100', '+', '.', 'gene_id \"scaffold_1348.g31\"; transcript_id \"blat.proc4587.chain_5017\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25823444', '25823483', '.', '-', '.', 'gene_id \"chr11.g9684\"; transcript_id \"chr11.g9684.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25984296', '25984491', '.', '-', '.', 'gene_id \"chr11.g9830\"; transcript_id \"chr11.g9830.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '25984632', '25984814', '.', '-', '.', 'gene_id \"chr11.g9831\"; transcript_id \"chr11.g9831.i1\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '699', '1085', '.', '-', '.', 'gene_id \"scaffold_688.g2\"; transcript_id \"scaffold_688.g2.i1\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '665', '977', '.', '-', '.', 'gene_id \"scaffold_688.g2\"; transcript_id \"scaffold_688.g2.i2\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '1465', '2861', '.', '-', '.', 'gene_id \"scaffold_688.g3\"; transcript_id \"scaffold_688.g3.i2\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '1284', '1421', '.', '-', '.', 'gene_id \"scaffold_688.g4\"; transcript_id \"scaffold_688.g4.i1\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '1252', '1421', '.', '-', '.', 'gene_id \"scaffold_688.g4\"; transcript_id \"scaffold_688.g4.i2\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '1', '665', '.', '-', '.', 'gene_id \"scaffold_688.g5\"; transcript_id \"scaffold_688.g5.i1\";']\n",
      "negative:  ['scaffold_688', 'PASA', 'exon', '2', '631', '.', '-', '.', 'gene_id \"scaffold_688.g5\"; transcript_id \"scaffold_688.g5.i2\";']\n",
      "negative:  ['scaffold_1267', 'PASA', 'exon', '16703', '16903', '.', '+', '.', 'gene_id \"scaffold_1267.g4\"; transcript_id \"scaffold_1267.g4.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '26532091', '26532276', '.', '+', '.', 'gene_id \"chr11.g10278\"; transcript_id \"chr11.g10278.i1\";']\n",
      "negative:  ['scaffold_1284', 'PASA', 'exon', '9590', '9698', '100', '+', '.', 'gene_id \"scaffold_1284.g11\"; transcript_id \"blat.proc28144.chain_9276\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '34728', '34879', '.', '-', '.', 'gene_id \"scaffold_218.g42\"; transcript_id \"scaffold_218.g42.i1\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '120019', '120315', '.', '+', '.', 'gene_id \"scaffold_218.g99\"; transcript_id \"scaffold_218.g99.i1\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '122815', '122919', '.', '+', '.', 'gene_id \"scaffold_218.g102\"; transcript_id \"scaffold_218.g102.i2\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '158651', '158808', '.', '-', '.', 'gene_id \"scaffold_218.g124\"; transcript_id \"scaffold_218.g124.i1\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '187872', '187903', '.', '-', '.', 'gene_id \"scaffold_218.g140\"; transcript_id \"scaffold_218.g140.i1\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '106827', '107039', '100', '-', '.', 'gene_id \"scaffold_218.g155\"; transcript_id \"blat.proc4587.chain_8208\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '201551', '201693', '100', '+', '.', 'gene_id \"scaffold_218.g178\"; transcript_id \"blat.proc4005.chain_3237\";']\n",
      "negative:  ['scaffold_218', 'PASA', 'exon', '136583', '136624', '100', '-', '.', 'gene_id \"scaffold_218.g197\"; transcript_id \"blat.proc28144.chain_6120\";']\n",
      "negative:  ['scaffold_1674', 'PASA', 'exon', '45660', '45919', '100', '+', '.', 'gene_id \"scaffold_1674.g16\"; transcript_id \"blat.proc18128.chain_21873\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '28747813', '28748077', '.', '+', '.', 'gene_id \"chr11.g10950\"; transcript_id \"chr11.g10950.i1\";']\n",
      "negative:  ['scaffold_1583', 'PASA', 'exon', '31624', '31666', '.', '+', '.', 'gene_id \"scaffold_1583.g9\"; transcript_id \"scaffold_1583.g9.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '32775098', '32775676', '.', '+', '.', 'gene_id \"chr11.g11480\"; transcript_id \"chr11.g11480.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '250965', '251106', '.', '-', '.', 'gene_id \"chr14.g25\"; transcript_id \"chr14.g25.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '33532362', '33532551', '.', '-', '.', 'gene_id \"chr11.g11661\"; transcript_id \"chr11.g11661.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '33532666', '33533219', '.', '+', '.', 'gene_id \"chr11.g11662\"; transcript_id \"chr11.g11662.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '1438050', '1438089', '.', '-', '.', 'gene_id \"chr14.g435\"; transcript_id \"chr14.g435.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '34465395', '34465590', '.', '-', '.', 'gene_id \"chr11.g12024\"; transcript_id \"chr11.g12024.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '35203104', '35203325', '.', '+', '.', 'gene_id \"chr11.g12330\"; transcript_id \"chr11.g12330.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '36113449', '36115463', '.', '-', '.', 'gene_id \"chr11.g12689\"; transcript_id \"chr11.g12689.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '4351966', '4352177', '.', '-', '.', 'gene_id \"chr14.g1404\"; transcript_id \"chr14.g1404.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '36631423', '36631658', '.', '-', '.', 'gene_id \"chr11.g13036\"; transcript_id \"chr11.g13036.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '36631660', '36632108', '.', '-', '.', 'gene_id \"chr11.g13037\"; transcript_id \"chr11.g13037.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '5228518', '5228768', '.', '+', '.', 'gene_id \"chr14.g1608\"; transcript_id \"chr14.g1608.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '38132819', '38132853', '.', '+', '.', 'gene_id \"chr11.g13330\"; transcript_id \"chr11.g13330.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '5911059', '5911261', '.', '-', '.', 'gene_id \"chr14.g1995\"; transcript_id \"chr14.g1995.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '5989227', '5989606', '.', '-', '.', 'gene_id \"chr14.g2126\"; transcript_id \"chr14.g2126.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr11', 'PASA', 'exon', '40206355', '40206493', '.', '+', '.', 'gene_id \"chr11.g13949\"; transcript_id \"chr11.g13949.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '40028046', '40028354', '.', '-', '.', 'gene_id \"chr11.g14024\"; transcript_id \"chr11.g14024.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '40510575', '40510814', '.', '+', '.', 'gene_id \"chr11.g14202\"; transcript_id \"chr11.g14202.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '40510837', '40511044', '.', '+', '.', 'gene_id \"chr11.g14203\"; transcript_id \"chr11.g14203.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '6778411', '6778464', '.', '-', '.', 'gene_id \"chr14.g2748\"; transcript_id \"chr14.g2748.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '6817004', '6818772', '.', '-', '.', 'gene_id \"chr14.g2763\"; transcript_id \"chr14.g2763.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '6881425', '6881759', '.', '-', '.', 'gene_id \"chr14.g2817\"; transcript_id \"chr14.g2817.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '6881764', '6882676', '.', '-', '.', 'gene_id \"chr14.g2818\"; transcript_id \"chr14.g2818.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '6908819', '6909110', '.', '-', '.', 'gene_id \"chr14.g2851\"; transcript_id \"chr14.g2851.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '41013153', '41013433', '.', '+', '.', 'gene_id \"chr11.g14457\"; transcript_id \"chr11.g14457.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '41013499', '41013777', '.', '-', '.', 'gene_id \"chr11.g14458\"; transcript_id \"chr11.g14458.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '44381303', '44381351', '.', '-', '.', 'gene_id \"chr11.g14750\"; transcript_id \"chr11.g14750.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '44654634', '44654802', '.', '+', '.', 'gene_id \"chr11.g14848\"; transcript_id \"chr11.g14848.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '44736652', '44737079', '.', '-', '.', 'gene_id \"chr11.g14918\"; transcript_id \"chr11.g14918.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '44737123', '44737712', '.', '+', '.', 'gene_id \"chr11.g14919\"; transcript_id \"chr11.g14919.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '44896955', '44897135', '.', '+', '.', 'gene_id \"chr11.g15047\"; transcript_id \"chr11.g15047.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '8101013', '8101197', '.', '+', '.', 'gene_id \"chr14.g3774\"; transcript_id \"chr14.g3774.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '49465703', '49465978', '.', '-', '.', 'gene_id \"chr11.g15914\"; transcript_id \"chr11.g15914.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '11436125', '11436161', '.', '+', '.', 'gene_id \"chr14.g4458\"; transcript_id \"chr14.g4458.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '49979127', '49979514', '.', '-', '.', 'gene_id \"chr11.g16272\"; transcript_id \"chr11.g16272.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '50179857', '50180110', '.', '-', '.', 'gene_id \"chr11.g16479\"; transcript_id \"chr11.g16479.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '13620112', '13620161', '.', '+', '.', 'gene_id \"chr14.g5174\"; transcript_id \"chr14.g5174.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '13848761', '13848998', '.', '+', '.', 'gene_id \"chr14.g5280\"; transcript_id \"chr14.g5280.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '50843675', '50844082', '.', '-', '.', 'gene_id \"chr11.g17002\"; transcript_id \"chr11.g17002.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16116772', '16116807', '.', '-', '.', 'gene_id \"chr14.g5713\"; transcript_id \"chr14.g5713.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16192153', '16192356', '.', '+', '.', 'gene_id \"chr14.g5739\"; transcript_id \"chr14.g5739.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16300081', '16300619', '.', '-', '.', 'gene_id \"chr14.g5826\"; transcript_id \"chr14.g5826.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16542544', '16542789', '.', '-', '.', 'gene_id \"chr14.g5856\"; transcript_id \"chr14.g5856.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16542543', '16542789', '.', '-', '.', 'gene_id \"chr14.g5856\"; transcript_id \"chr14.g5856.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16690077', '16690138', '.', '-', '.', 'gene_id \"chr14.g5858\"; transcript_id \"chr14.g5858.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16556601', '16556696', '.', '-', '.', 'gene_id \"chr14.g5859\"; transcript_id \"chr14.g5859.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16599394', '16599471', '.', '-', '.', 'gene_id \"chr14.g5860\"; transcript_id \"chr14.g5860.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16658839', '16658934', '.', '-', '.', 'gene_id \"chr14.g5863\"; transcript_id \"chr14.g5863.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16517923', '16518074', '.', '-', '.', 'gene_id \"chr14.g5866\"; transcript_id \"chr14.g5866.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16517597', '16517680', '.', '-', '.', 'gene_id \"chr14.g5867\"; transcript_id \"chr14.g5867.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '16935527', '16936040', '.', '-', '.', 'gene_id \"chr14.g6087\"; transcript_id \"chr14.g6087.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '17016758', '17017030', '.', '+', '.', 'gene_id \"chr14.g6124\"; transcript_id \"chr14.g6124.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '52320628', '52320827', '.', '+', '.', 'gene_id \"chr11.g17996\"; transcript_id \"chr11.g17996.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '52414963', '52415112', '.', '-', '.', 'gene_id \"chr11.g18035\"; transcript_id \"chr11.g18035.i8\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '52388461', '52388755', '.', '+', '.', 'gene_id \"chr11.g18067\"; transcript_id \"chr11.g18067.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '19851027', '19852192', '.', '+', '.', 'gene_id \"chr14.g7006\"; transcript_id \"chr14.g7006.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '19965307', '19965522', '.', '+', '.', 'gene_id \"chr14.g7075\"; transcript_id \"chr14.g7075.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '20001420', '20001575', '.', '+', '.', 'gene_id \"chr14.g7104\"; transcript_id \"chr14.g7104.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '20001829', '20002190', '.', '-', '.', 'gene_id \"chr14.g7105\"; transcript_id \"chr14.g7105.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '53613253', '53613290', '.', '+', '.', 'gene_id \"chr11.g18790\"; transcript_id \"chr11.g18790.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '20385523', '20385718', '.', '+', '.', 'gene_id \"chr14.g7231\"; transcript_id \"chr14.g7231.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '53726897', '53728252', '.', '-', '.', 'gene_id \"chr11.g18860\"; transcript_id \"chr11.g18860.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '20725488', '20725849', '.', '-', '.', 'gene_id \"chr14.g7418\"; transcript_id \"chr14.g7418.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21176928', '21177079', '.', '+', '.', 'gene_id \"chr14.g7560\"; transcript_id \"chr14.g7560.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '54330407', '54330649', '.', '+', '.', 'gene_id \"chr11.g19160\"; transcript_id \"chr11.g19160.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21219151', '21219349', '.', '+', '.', 'gene_id \"chr14.g7602\"; transcript_id \"chr14.g7602.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21227343', '21227556', '.', '-', '.', 'gene_id \"chr14.g7610\"; transcript_id \"chr14.g7610.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21227560', '21227726', '.', '+', '.', 'gene_id \"chr14.g7611\"; transcript_id \"chr14.g7611.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21268471', '21268636', '.', '-', '.', 'gene_id \"chr14.g7657\"; transcript_id \"chr14.g7657.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21291648', '21291873', '.', '-', '.', 'gene_id \"chr14.g7683\"; transcript_id \"chr14.g7683.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21292014', '21292289', '.', '-', '.', 'gene_id \"chr14.g7684\"; transcript_id \"chr14.g7684.i2\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '54486703', '54486902', '.', '-', '.', 'gene_id \"chr11.g19303\"; transcript_id \"chr11.g19303.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '21693518', '21693560', '.', '+', '.', 'gene_id \"chr14.g7808\"; transcript_id \"chr14.g7808.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '54868605', '54869991', '.', '+', '.', 'gene_id \"chr11.g19550\"; transcript_id \"chr11.g19550.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '54868270', '54868543', '.', '-', '.', 'gene_id \"chr11.g19551\"; transcript_id \"chr11.g19551.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '54866460', '54867333', '.', '-', '.', 'gene_id \"chr11.g19553\"; transcript_id \"chr11.g19553.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '23354426', '23355091', '.', '+', '.', 'gene_id \"chr14.g8352\"; transcript_id \"chr14.g8352.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '55811905', '55812063', '.', '+', '.', 'gene_id \"chr11.g19978\"; transcript_id \"chr11.g19978.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '23576661', '23576837', '.', '-', '.', 'gene_id \"chr14.g8555\"; transcript_id \"chr14.g8555.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '24845085', '24846156', '.', '-', '.', 'gene_id \"chr14.g8775\"; transcript_id \"chr14.g8775.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '57171809', '57171842', '.', '-', '.', 'gene_id \"chr11.g20530\"; transcript_id \"chr11.g20530.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '26904505', '26904551', '.', '+', '.', 'gene_id \"chr14.g9145\"; transcript_id \"chr14.g9145.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '28693685', '28693842', '.', '+', '.', 'gene_id \"chr14.g9388\"; transcript_id \"chr14.g9388.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '60447905', '60448383', '.', '+', '.', 'gene_id \"chr11.g21081\"; transcript_id \"chr11.g21081.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '61018164', '61018647', '.', '-', '.', 'gene_id \"chr11.g21357\"; transcript_id \"chr11.g21357.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '61221558', '61221835', '.', '+', '.', 'gene_id \"chr11.g21473\"; transcript_id \"chr11.g21473.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '61221904', '61222599', '.', '+', '.', 'gene_id \"chr11.g21474\"; transcript_id \"chr11.g21474.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '61222583', '61222907', '.', '-', '.', 'gene_id \"chr11.g21475\"; transcript_id \"chr11.g21475.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '62074504', '62074748', '.', '+', '.', 'gene_id \"chr11.g21568\"; transcript_id \"chr11.g21568.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '62428959', '62428993', '.', '+', '.', 'gene_id \"chr11.g21813\"; transcript_id \"chr11.g21813.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '32625804', '32625858', '.', '-', '.', 'gene_id \"chr14.g10147\"; transcript_id \"chr14.g10147.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '63077179', '63077214', '.', '+', '.', 'gene_id \"chr11.g21945\"; transcript_id \"chr11.g21945.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '33430475', '33430678', '.', '-', '.', 'gene_id \"chr14.g10356\"; transcript_id \"chr14.g10356.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '34273000', '34273207', '.', '-', '.', 'gene_id \"chr14.g10454\"; transcript_id \"chr14.g10454.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '34273000', '34273207', '.', '-', '.', 'gene_id \"chr14.g10454\"; transcript_id \"chr14.g10454.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '34657357', '34658200', '.', '-', '.', 'gene_id \"chr14.g10571\"; transcript_id \"chr14.g10571.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '35451294', '35451381', '.', '+', '.', 'gene_id \"chr14.g10670\"; transcript_id \"chr14.g10670.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '64501398', '64501743', '.', '+', '.', 'gene_id \"chr11.g22524\"; transcript_id \"chr11.g22524.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '36075164', '36075344', '.', '+', '.', 'gene_id \"chr14.g10876\"; transcript_id \"chr14.g10876.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '36075337', '36075813', '.', '-', '.', 'gene_id \"chr14.g10877\"; transcript_id \"chr14.g10877.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr14', 'PASA', 'exon', '36785981', '36786112', '.', '-', '.', 'gene_id \"chr14.g11156\"; transcript_id \"chr14.g11156.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '65797699', '65797733', '.', '+', '.', 'gene_id \"chr11.g23079\"; transcript_id \"chr11.g23079.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '65925494', '65926056', '.', '-', '.', 'gene_id \"chr11.g23209\"; transcript_id \"chr11.g23209.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '66196513', '66196699', '.', '+', '.', 'gene_id \"chr11.g23341\"; transcript_id \"chr11.g23341.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '38535150', '38535336', '.', '-', '.', 'gene_id \"chr14.g11734\"; transcript_id \"chr14.g11734.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '38535493', '38535759', '.', '-', '.', 'gene_id \"chr14.g11735\"; transcript_id \"chr14.g11735.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '66924534', '66924737', '.', '-', '.', 'gene_id \"chr11.g23829\"; transcript_id \"chr11.g23829.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '39541683', '39541852', '.', '+', '.', 'gene_id \"chr14.g12025\"; transcript_id \"chr14.g12025.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '40123183', '40123356', '.', '+', '.', 'gene_id \"chr14.g12277\"; transcript_id \"chr14.g12277.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41007667', '41007846', '.', '+', '.', 'gene_id \"chr14.g12644\"; transcript_id \"chr14.g12644.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41007895', '41008167', '.', '-', '.', 'gene_id \"chr14.g12646\"; transcript_id \"chr14.g12646.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41175701', '41176021', '.', '-', '.', 'gene_id \"chr14.g12753\"; transcript_id \"chr14.g12753.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41176185', '41176366', '.', '-', '.', 'gene_id \"chr14.g12754\"; transcript_id \"chr14.g12754.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41286322', '41286500', '.', '+', '.', 'gene_id \"chr14.g12866\"; transcript_id \"chr14.g12866.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41286514', '41286777', '.', '-', '.', 'gene_id \"chr14.g12867\"; transcript_id \"chr14.g12867.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '41484057', '41484091', '.', '+', '.', 'gene_id \"chr14.g12940\"; transcript_id \"chr14.g12940.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '73414184', '73414635', '.', '-', '.', 'gene_id \"chr11.g25467\"; transcript_id \"chr11.g25467.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '73717512', '73717543', '.', '-', '.', 'gene_id \"chr11.g25736\"; transcript_id \"chr11.g25736.i1\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '58437853', '58437896', '100', '+', '.', 'gene_id \"chr11.g27016\"; transcript_id \"blat.proc28144.chain_12471\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '18681963', '18682137', '100', '-', '.', 'gene_id \"chr11.g27156\"; transcript_id \"blat.proc4587.chain_13327\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '50862571', '50862633', '100', '+', '.', 'gene_id \"chr11.g27405\"; transcript_id \"blat.proc4005.chain_14658\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '36249983', '36250039', '100', '+', '.', 'gene_id \"chr11.g27664\"; transcript_id \"blat.proc4587.chain_14112\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '62832199', '62832232', '100', '-', '.', 'gene_id \"chr11.g28332\"; transcript_id \"blat.proc18128.chain_13508\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '32160519', '32160579', '100', '-', '.', 'gene_id \"chr11.g28514\"; transcript_id \"blat.proc18128.chain_6909\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '67337228', '67337614', '100', '+', '.', 'gene_id \"chr11.g28752\"; transcript_id \"blat.proc4587.chain_10049\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '27228492', '27228596', '100', '-', '.', 'gene_id \"chr11.g29122\"; transcript_id \"blat.proc4587.chain_13247\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '39453929', '39454141', '100', '-', '.', 'gene_id \"chr11.g29703\"; transcript_id \"blat.proc18128.chain_8356\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '3349902', '3349975', '100', '-', '.', 'gene_id \"chr11.g30200\"; transcript_id \"blat.proc4587.chain_13210\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '60357095', '60357143', '100', '-', '.', 'gene_id \"chr11.g30464\"; transcript_id \"blat.proc4587.chain_25\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '53497613', '53497673', '100', '-', '.', 'gene_id \"chr11.g30523\"; transcript_id \"blat.proc4587.chain_13821\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '480896', '480965', '100', '-', '.', 'gene_id \"chr11.g30702\"; transcript_id \"blat.proc18128.chain_5792\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '18976659', '18976692', '100', '+', '.', 'gene_id \"chr11.g31736\"; transcript_id \"blat.proc18128.chain_15574\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '10574851', '10574884', '100', '-', '.', 'gene_id \"chr11.g32901\"; transcript_id \"blat.proc18128.chain_6144\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '50006861', '50006902', '100', '+', '.', 'gene_id \"chr11.g32903\"; transcript_id \"blat.proc28144.chain_18181\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '39031907', '39031990', '100', '+', '.', 'gene_id \"chr11.g33240\"; transcript_id \"blat.proc4587.chain_9974\";']\n",
      "negative:  ['chr11', 'PASA', 'exon', '56857126', '56857423', '100', '+', '.', 'gene_id \"chr11.g33261\"; transcript_id \"blat.proc4587.chain_10079\";']\n",
      "negative:  ['scaffold_1218', 'PASA', 'exon', '6675', '6843', '100', '+', '.', 'gene_id \"scaffold_1218.g16\"; transcript_id \"blat.proc18128.chain_17504\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '44170462', '44170856', '.', '+', '.', 'gene_id \"chr14.g14087\"; transcript_id \"chr14.g14087.i1\";']\n",
      "negative:  ['scaffold_1083', 'PASA', 'exon', '6719', '7212', '100', '-', '.', 'gene_id \"scaffold_1083.g13\"; transcript_id \"blat.proc4587.chain_3184\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '46046855', '46046898', '.', '+', '.', 'gene_id \"chr14.g14765\"; transcript_id \"chr14.g14765.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '47297640', '47297844', '.', '-', '.', 'gene_id \"chr14.g14996\"; transcript_id \"chr14.g14996.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '47297479', '47297638', '.', '-', '.', 'gene_id \"chr14.g14997\"; transcript_id \"chr14.g14997.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '48450418', '48450653', '.', '+', '.', 'gene_id \"chr14.g15679\"; transcript_id \"chr14.g15679.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '48884495', '48884722', '.', '+', '.', 'gene_id \"chr14.g15979\"; transcript_id \"chr14.g15979.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '51132474', '51132549', '.', '+', '.', 'gene_id \"chr14.g16281\"; transcript_id \"chr14.g16281.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '51675642', '51675801', '.', '-', '.', 'gene_id \"chr14.g16499\"; transcript_id \"chr14.g16499.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '52488009', '52488283', '.', '+', '.', 'gene_id \"chr14.g16850\"; transcript_id \"chr14.g16850.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '53419407', '53419583', '.', '+', '.', 'gene_id \"chr14.g17154\"; transcript_id \"chr14.g17154.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '13993416', '13993616', '.', '+', '.', 'gene_id \"chr1.g3039\"; transcript_id \"chr1.g3039.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '14304869', '14305118', '.', '-', '.', 'gene_id \"chr1.g3104\"; transcript_id \"chr1.g3104.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '14305318', '14305483', '.', '+', '.', 'gene_id \"chr1.g3106\"; transcript_id \"chr1.g3106.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '14893808', '14894053', '.', '-', '.', 'gene_id \"chr1.g3262\"; transcript_id \"chr1.g3262.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '54588098', '54588139', '.', '-', '.', 'gene_id \"chr14.g17460\"; transcript_id \"chr14.g17460.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '15083058', '15083231', '.', '+', '.', 'gene_id \"chr1.g3298\"; transcript_id \"chr1.g3298.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '55555428', '55555467', '.', '-', '.', 'gene_id \"chr14.g17642\"; transcript_id \"chr14.g17642.i2\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '55607294', '55607348', '.', '-', '.', 'gene_id \"chr14.g17665\"; transcript_id \"chr14.g17665.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '55691305', '55691521', '.', '-', '.', 'gene_id \"chr14.g17721\"; transcript_id \"chr14.g17721.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '55691559', '55691766', '.', '-', '.', 'gene_id \"chr14.g17722\"; transcript_id \"chr14.g17722.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '17280868', '17281017', '.', '-', '.', 'gene_id \"chr1.g3596\"; transcript_id \"chr1.g3596.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '17345061', '17345640', '.', '+', '.', 'gene_id \"chr1.g3617\"; transcript_id \"chr1.g3617.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '17680272', '17680516', '.', '-', '.', 'gene_id \"chr1.g3709\"; transcript_id \"chr1.g3709.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '17702501', '17702666', '.', '+', '.', 'gene_id \"chr1.g3717\"; transcript_id \"chr1.g3717.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr14', 'PASA', 'exon', '55936751', '55937073', '.', '-', '.', 'gene_id \"chr14.g17881\"; transcript_id \"chr14.g17881.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '56627600', '56627749', '.', '+', '.', 'gene_id \"chr14.g18123\"; transcript_id \"chr14.g18123.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '56627754', '56628091', '.', '-', '.', 'gene_id \"chr14.g18124\"; transcript_id \"chr14.g18124.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '57034809', '57034857', '.', '-', '.', 'gene_id \"chr14.g18241\"; transcript_id \"chr14.g18241.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '59209358', '59209882', '.', '+', '.', 'gene_id \"chr14.g18485\"; transcript_id \"chr14.g18485.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '59390574', '59390758', '.', '-', '.', 'gene_id \"chr14.g18672\"; transcript_id \"chr14.g18672.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '59547872', '59548114', '.', '+', '.', 'gene_id \"chr14.g18781\"; transcript_id \"chr14.g18781.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '60050823', '60051242', '.', '-', '.', 'gene_id \"chr14.g18875\"; transcript_id \"chr14.g18875.i1\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '17909093', '17909344', '100', '+', '.', 'gene_id \"chr14.g19621\"; transcript_id \"blat.proc28144.chain_11564\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '50404777', '50405223', '100', '-', '.', 'gene_id \"chr14.g19718\"; transcript_id \"blat.proc4587.chain_12972\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '36831996', '36832031', '100', '-', '.', 'gene_id \"chr14.g19826\"; transcript_id \"blat.proc18128.chain_6143\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '56440573', '56440611', '100', '-', '.', 'gene_id \"chr14.g19905\"; transcript_id \"blat.proc18128.chain_9691\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '50280727', '50280758', '100', '-', '.', 'gene_id \"chr14.g20155\"; transcript_id \"blat.proc4587.chain_10965\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '14257875', '14257924', '100', '+', '.', 'gene_id \"chr14.g20271\"; transcript_id \"blat.proc4587.chain_3636\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '59967913', '59967962', '100', '-', '.', 'gene_id \"chr14.g20621\"; transcript_id \"blat.proc28144.chain_17254\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '37387018', '37387071', '100', '-', '.', 'gene_id \"chr14.g21075\"; transcript_id \"blat.proc4587.chain_3903\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '48761893', '48761932', '100', '-', '.', 'gene_id \"chr14.g21593\"; transcript_id \"blat.proc4005.chain_8962\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '44740343', '44740412', '100', '-', '.', 'gene_id \"chr14.g21710\"; transcript_id \"blat.proc18128.chain_25237\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '11807224', '11807476', '100', '+', '.', 'gene_id \"chr14.g21994\"; transcript_id \"blat.proc4587.chain_558\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '11132723', '11132762', '100', '+', '.', 'gene_id \"chr14.g22083\"; transcript_id \"blat.proc4005.chain_6120\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '45980823', '45980887', '100', '-', '.', 'gene_id \"chr14.g22530\"; transcript_id \"blat.proc4587.chain_11110\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '5730582', '5730896', '100', '+', '.', 'gene_id \"chr14.g22781\"; transcript_id \"blat.proc4587.chain_9607\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '44553533', '44553632', '100', '+', '.', 'gene_id \"chr14.g23135\"; transcript_id \"blat.proc4587.chain_2734\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '30038853', '30038888', '100', '-', '.', 'gene_id \"chr14.g23284\"; transcript_id \"blat.proc4587.chain_2660\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '18183780', '18183998', '100', '-', '.', 'gene_id \"chr14.g23736\"; transcript_id \"blat.proc4587.chain_10020\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '56461558', '56461688', '100', '+', '.', 'gene_id \"chr14.g23842\"; transcript_id \"blat.proc28144.chain_10830\";']\n",
      "negative:  ['chr14', 'PASA', 'exon', '5545608', '5545661', '100', '-', '.', 'gene_id \"chr14.g24298\"; transcript_id \"blat.proc4005.chain_6121\";']\n",
      "negative:  ['scaffold_1300', 'PASA', 'exon', '7043', '7210', '100', '+', '.', 'gene_id \"scaffold_1300.g8\"; transcript_id \"blat.proc18128.chain_25500\";']\n",
      "negative:  ['scaffold_1300', 'PASA', 'exon', '15518', '15555', '100', '+', '.', 'gene_id \"scaffold_1300.g10\"; transcript_id \"blat.proc18128.chain_6976\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '28126643', '28126891', '.', '-', '.', 'gene_id \"chr1.g5674\"; transcript_id \"chr1.g5674.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '929', '999', '.', '+', '.', 'gene_id \"scaffold_219.g2\"; transcript_id \"scaffold_219.g2.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '137418', '137734', '.', '+', '.', 'gene_id \"scaffold_219.g81\"; transcript_id \"scaffold_219.g81.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '137771', '138071', '.', '-', '.', 'gene_id \"scaffold_219.g82\"; transcript_id \"scaffold_219.g82.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '137878', '138032', '.', '-', '.', 'gene_id \"scaffold_219.g82\"; transcript_id \"scaffold_219.g82.i2\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '138069', '138362', '.', '+', '.', 'gene_id \"scaffold_219.g83\"; transcript_id \"scaffold_219.g83.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '142314', '142731', '.', '-', '.', 'gene_id \"scaffold_219.g89\"; transcript_id \"scaffold_219.g89.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '142052', '142317', '.', '+', '.', 'gene_id \"scaffold_219.g90\"; transcript_id \"scaffold_219.g90.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '142705', '143107', '.', '-', '.', 'gene_id \"scaffold_219.g91\"; transcript_id \"scaffold_219.g91.i1\";']\n",
      "negative:  ['scaffold_219', 'PASA', 'exon', '145995', '146176', '100', '+', '.', 'gene_id \"scaffold_219.g123\"; transcript_id \"blat.proc4587.chain_12745\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '580052', '580818', '.', '-', '.', 'gene_id \"chr10.g499\"; transcript_id \"chr10.g499.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '1573354', '1573590', '.', '-', '.', 'gene_id \"chr10.g1273\"; transcript_id \"chr10.g1273.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '2354885', '2354935', '.', '+', '.', 'gene_id \"chr10.g1636\"; transcript_id \"chr10.g1636.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '3314391', '3314615', '.', '-', '.', 'gene_id \"chr10.g1859\"; transcript_id \"chr10.g1859.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '5607081', '5607280', '.', '-', '.', 'gene_id \"chr10.g2304\"; transcript_id \"chr10.g2304.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '5657800', '5657976', '.', '-', '.', 'gene_id \"chr10.g2368\"; transcript_id \"chr10.g2368.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '5657998', '5658173', '.', '+', '.', 'gene_id \"chr10.g2369\"; transcript_id \"chr10.g2369.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '5804065', '5804749', '.', '-', '.', 'gene_id \"chr10.g2509\"; transcript_id \"chr10.g2509.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '6046248', '6046555', '.', '-', '.', 'gene_id \"chr10.g2664\"; transcript_id \"chr10.g2664.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '6046809', '6046947', '.', '+', '.', 'gene_id \"chr10.g2665\"; transcript_id \"chr10.g2665.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '6304288', '6304329', '.', '+', '.', 'gene_id \"chr10.g2689\"; transcript_id \"chr10.g2689.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7144424', '7144583', '.', '+', '.', 'gene_id \"chr10.g3143\"; transcript_id \"chr10.g3143.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7486103', '7486266', '.', '+', '.', 'gene_id \"chr10.g3364\"; transcript_id \"chr10.g3364.i10\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7485061', '7485271', '.', '+', '.', 'gene_id \"chr10.g3398\"; transcript_id \"chr10.g3398.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7485389', '7485820', '.', '+', '.', 'gene_id \"chr10.g3399\"; transcript_id \"chr10.g3399.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7815725', '7816255', '.', '-', '.', 'gene_id \"chr10.g3501\"; transcript_id \"chr10.g3501.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '7816256', '7817454', '.', '-', '.', 'gene_id \"chr10.g3502\"; transcript_id \"chr10.g3502.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '11041281', '11041472', '.', '-', '.', 'gene_id \"chr10.g4687\"; transcript_id \"chr10.g4687.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '11052112', '11052163', '.', '-', '.', 'gene_id \"chr10.g4693\"; transcript_id \"chr10.g4693.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '11614508', '11614862', '.', '-', '.', 'gene_id \"chr10.g5025\"; transcript_id \"chr10.g5025.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '12189040', '12189529', '.', '-', '.', 'gene_id \"chr10.g5275\"; transcript_id \"chr10.g5275.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '12202524', '12202836', '.', '+', '.', 'gene_id \"chr10.g5296\"; transcript_id \"chr10.g5296.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '12202841', '12203081', '.', '-', '.', 'gene_id \"chr10.g5297\"; transcript_id \"chr10.g5297.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '13288189', '13288421', '.', '-', '.', 'gene_id \"chr10.g5764\"; transcript_id \"chr10.g5764.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '13288485', '13288740', '.', '-', '.', 'gene_id \"chr10.g5765\"; transcript_id \"chr10.g5765.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '13290424', '13290786', '.', '-', '.', 'gene_id \"chr10.g5768\"; transcript_id \"chr10.g5768.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '13424775', '13425006', '.', '-', '.', 'gene_id \"chr10.g5908\"; transcript_id \"chr10.g5908.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr10', 'PASA', 'exon', '17578439', '17578498', '.', '-', '.', 'gene_id \"chr10.g6872\"; transcript_id \"chr10.g6872.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '18700536', '18700696', '.', '+', '.', 'gene_id \"chr10.g7099\"; transcript_id \"chr10.g7099.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20225500', '20225701', '.', '-', '.', 'gene_id \"chr10.g7533\"; transcript_id \"chr10.g7533.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20226661', '20226938', '.', '+', '.', 'gene_id \"chr10.g7534\"; transcript_id \"chr10.g7534.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20434616', '20435417', '.', '+', '.', 'gene_id \"chr10.g7671\"; transcript_id \"chr10.g7671.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20436669', '20436921', '.', '+', '.', 'gene_id \"chr10.g7674\"; transcript_id \"chr10.g7674.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20586297', '20586890', '.', '-', '.', 'gene_id \"chr10.g7816\"; transcript_id \"chr10.g7816.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '20743048', '20743347', '.', '-', '.', 'gene_id \"chr10.g7888\"; transcript_id \"chr10.g7888.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '23324624', '23324770', '.', '+', '.', 'gene_id \"chr10.g9451\"; transcript_id \"chr10.g9451.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '23713700', '23713902', '.', '+', '.', 'gene_id \"chr10.g9666\"; transcript_id \"chr10.g9666.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '25212177', '25212351', '.', '-', '.', 'gene_id \"chr10.g10997\"; transcript_id \"chr10.g10997.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '25212369', '25212699', '.', '+', '.', 'gene_id \"chr10.g10998\"; transcript_id \"chr10.g10998.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '25556614', '25556851', '.', '-', '.', 'gene_id \"chr10.g11346\"; transcript_id \"chr10.g11346.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '25556866', '25557102', '.', '+', '.', 'gene_id \"chr10.g11347\"; transcript_id \"chr10.g11347.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '26156465', '26157827', '.', '+', '.', 'gene_id \"chr10.g11644\"; transcript_id \"chr10.g11644.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '26286141', '26286640', '.', '+', '.', 'gene_id \"chr10.g11746\"; transcript_id \"chr10.g11746.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '26285678', '26285913', '.', '-', '.', 'gene_id \"chr10.g11747\"; transcript_id \"chr10.g11747.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '26649469', '26649521', '.', '-', '.', 'gene_id \"chr10.g11998\"; transcript_id \"chr10.g11998.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '27174340', '27174636', '.', '-', '.', 'gene_id \"chr10.g12405\"; transcript_id \"chr10.g12405.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '28855953', '28856457', '.', '+', '.', 'gene_id \"chr10.g13444\"; transcript_id \"chr10.g13444.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '30658968', '30659379', '.', '+', '.', 'gene_id \"chr10.g13973\"; transcript_id \"chr10.g13973.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '31513868', '31514019', '.', '+', '.', 'gene_id \"chr10.g14390\"; transcript_id \"chr10.g14390.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '31514158', '31514313', '.', '-', '.', 'gene_id \"chr10.g14391\"; transcript_id \"chr10.g14391.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '33990599', '33991122', '.', '-', '.', 'gene_id \"chr10.g15457\"; transcript_id \"chr10.g15457.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '33991078', '33991357', '.', '+', '.', 'gene_id \"chr10.g15458\"; transcript_id \"chr10.g15458.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '38288787', '38289003', '.', '+', '.', 'gene_id \"chr10.g16743\"; transcript_id \"chr10.g16743.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '38289010', '38289239', '.', '-', '.', 'gene_id \"chr10.g16744\"; transcript_id \"chr10.g16744.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '41024595', '41024761', '.', '-', '.', 'gene_id \"chr10.g17286\"; transcript_id \"chr10.g17286.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '43454328', '43454488', '.', '-', '.', 'gene_id \"chr10.g18447\"; transcript_id \"chr10.g18447.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '44057357', '44057398', '.', '+', '.', 'gene_id \"chr10.g18752\"; transcript_id \"chr10.g18752.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '44938064', '44938300', '.', '+', '.', 'gene_id \"chr10.g19242\"; transcript_id \"chr10.g19242.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '45457517', '45457560', '.', '-', '.', 'gene_id \"chr10.g19449\"; transcript_id \"chr10.g19449.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '45583747', '45583784', '.', '+', '.', 'gene_id \"chr10.g19532\"; transcript_id \"chr10.g19532.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '46385011', '46385131', '.', '+', '.', 'gene_id \"chr10.g19960\"; transcript_id \"chr10.g19960.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '46386218', '46386562', '.', '+', '.', 'gene_id \"chr10.g19961\"; transcript_id \"chr10.g19961.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '46749667', '46749838', '.', '+', '.', 'gene_id \"chr10.g20186\"; transcript_id \"chr10.g20186.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '47233538', '47233962', '.', '-', '.', 'gene_id \"chr10.g20504\"; transcript_id \"chr10.g20504.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '48331506', '48331689', '.', '+', '.', 'gene_id \"chr10.g20997\"; transcript_id \"chr10.g20997.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '48688749', '48688951', '.', '+', '.', 'gene_id \"chr10.g21219\"; transcript_id \"chr10.g21219.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '48809579', '48809743', '.', '+', '.', 'gene_id \"chr10.g21319\"; transcript_id \"chr10.g21319.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '49161924', '49162242', '.', '-', '.', 'gene_id \"chr10.g21570\"; transcript_id \"chr10.g21570.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '49717005', '49717246', '.', '-', '.', 'gene_id \"chr10.g21952\"; transcript_id \"chr10.g21952.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '49916112', '49916154', '.', '+', '.', 'gene_id \"chr10.g22109\"; transcript_id \"chr10.g22109.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '50353280', '50353314', '.', '-', '.', 'gene_id \"chr10.g22262\"; transcript_id \"chr10.g22262.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '50451804', '50451843', '.', '+', '.', 'gene_id \"chr10.g22345\"; transcript_id \"chr10.g22345.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '50829505', '50829689', '.', '-', '.', 'gene_id \"chr10.g22564\"; transcript_id \"chr10.g22564.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '50831228', '50831382', '.', '+', '.', 'gene_id \"chr10.g22567\"; transcript_id \"chr10.g22567.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '51143011', '51143547', '.', '+', '.', 'gene_id \"chr10.g22767\"; transcript_id \"chr10.g22767.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '51484312', '51484465', '.', '-', '.', 'gene_id \"chr10.g22955\"; transcript_id \"chr10.g22955.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '53275294', '53275420', '.', '+', '.', 'gene_id \"chr10.g23180\"; transcript_id \"chr10.g23180.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '55431733', '55432341', '.', '+', '.', 'gene_id \"chr10.g23402\"; transcript_id \"chr10.g23402.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '55499078', '55499650', '.', '-', '.', 'gene_id \"chr10.g23461\"; transcript_id \"chr10.g23461.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '55513440', '55513471', '.', '+', '.', 'gene_id \"chr10.g23475\"; transcript_id \"chr10.g23475.i2\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '56179436', '56179793', '.', '+', '.', 'gene_id \"chr10.g24082\"; transcript_id \"chr10.g24082.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '57172830', '57173001', '.', '+', '.', 'gene_id \"chr10.g24521\"; transcript_id \"chr10.g24521.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '57133531', '57133836', '.', '-', '.', 'gene_id \"chr10.g24565\"; transcript_id \"chr10.g24565.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '57849879', '57849914', '.', '+', '.', 'gene_id \"chr10.g24888\"; transcript_id \"chr10.g24888.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '62153145', '62153430', '.', '-', '.', 'gene_id \"chr10.g25831\"; transcript_id \"chr10.g25831.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '63494052', '63494259', '.', '+', '.', 'gene_id \"chr10.g26654\"; transcript_id \"chr10.g26654.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '64014425', '64014458', '.', '+', '.', 'gene_id \"chr10.g26864\"; transcript_id \"chr10.g26864.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '64967594', '64967852', '.', '-', '.', 'gene_id \"chr10.g27449\"; transcript_id \"chr10.g27449.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '64967958', '64968193', '.', '-', '.', 'gene_id \"chr10.g27450\"; transcript_id \"chr10.g27450.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '65178636', '65178832', '.', '+', '.', 'gene_id \"chr10.g27504\"; transcript_id \"chr10.g27504.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '66250322', '66250371', '.', '-', '.', 'gene_id \"chr10.g27902\"; transcript_id \"chr10.g27902.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '66522803', '66522946', '.', '-', '.', 'gene_id \"chr10.g27956\"; transcript_id \"chr10.g27956.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '66597775', '66597942', '.', '-', '.', 'gene_id \"chr10.g28033\"; transcript_id \"chr10.g28033.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '66873496', '66873543', '.', '-', '.', 'gene_id \"chr10.g28076\"; transcript_id \"chr10.g28076.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '68045142', '68045404', '.', '-', '.', 'gene_id \"chr10.g28367\"; transcript_id \"chr10.g28367.i3\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '68314616', '68314677', '.', '-', '.', 'gene_id \"chr10.g28525\"; transcript_id \"chr10.g28525.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '68539371', '68539785', '.', '+', '.', 'gene_id \"chr10.g28647\"; transcript_id \"chr10.g28647.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '68680687', '68681023', '.', '-', '.', 'gene_id \"chr10.g28771\"; transcript_id \"chr10.g28771.i1\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '31608226', '31608277', '100', '-', '.', 'gene_id \"chr10.g30316\"; transcript_id \"blat.proc4587.chain_7296\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '22410158', '22410192', '100', '+', '.', 'gene_id \"chr10.g30646\"; transcript_id \"blat.proc18128.chain_3289\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '31011699', '31011761', '100', '-', '.', 'gene_id \"chr10.g31848\"; transcript_id \"blat.proc4587.chain_373\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '58662009', '58662054', '100', '-', '.', 'gene_id \"chr10.g32017\"; transcript_id \"blat.proc4005.chain_18330\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '47417760', '47418266', '100', '+', '.', 'gene_id \"chr10.g33038\"; transcript_id \"blat.proc4587.chain_14042\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '57543256', '57543312', '100', '-', '.', 'gene_id \"chr10.g33260\"; transcript_id \"blat.proc18128.chain_42\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '21104185', '21104228', '100', '+', '.', 'gene_id \"chr10.g33650\"; transcript_id \"blat.proc28144.chain_5142\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '3223278', '3223309', '100', '+', '.', 'gene_id \"chr10.g34116\"; transcript_id \"blat.proc4587.chain_9300\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '31385582', '31385628', '100', '-', '.', 'gene_id \"chr10.g36034\"; transcript_id \"blat.proc4587.chain_1113\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '63388764', '63388904', '100', '-', '.', 'gene_id \"chr10.g36193\"; transcript_id \"blat.proc28144.chain_15462\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '21333493', '21333540', '100', '+', '.', 'gene_id \"chr10.g36235\"; transcript_id \"blat.proc4587.chain_1826\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '63381823', '63381854', '100', '-', '.', 'gene_id \"chr10.g36360\"; transcript_id \"blat.proc4587.chain_12951\";']\n",
      "negative:  ['chr10', 'PASA', 'exon', '13281026', '13281102', '100', '-', '.', 'gene_id \"chr10.g36635\"; transcript_id \"blat.proc4587.chain_1455\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '497655', '497850', '.', '+', '.', 'gene_id \"chr23.g87\"; transcript_id \"chr23.g87.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '956722', '956755', '.', '+', '.', 'gene_id \"chr23.g211\"; transcript_id \"chr23.g211.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1075929', '1076235', '.', '-', '.', 'gene_id \"chr23.g279\"; transcript_id \"chr23.g279.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1246834', '1247130', '.', '+', '.', 'gene_id \"chr23.g410\"; transcript_id \"chr23.g410.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1247334', '1247727', '.', '+', '.', 'gene_id \"chr23.g413\"; transcript_id \"chr23.g413.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1497070', '1497247', '.', '-', '.', 'gene_id \"chr23.g519\"; transcript_id \"chr23.g519.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1597809', '1598049', '.', '-', '.', 'gene_id \"chr23.g643\"; transcript_id \"chr23.g643.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1598032', '1598614', '.', '+', '.', 'gene_id \"chr23.g644\"; transcript_id \"chr23.g644.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1802242', '1802410', '.', '-', '.', 'gene_id \"chr23.g810\"; transcript_id \"chr23.g810.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1802242', '1802351', '.', '-', '.', 'gene_id \"chr23.g810\"; transcript_id \"chr23.g810.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2858919', '2859070', '.', '-', '.', 'gene_id \"chr23.g1326\"; transcript_id \"chr23.g1326.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2920358', '2920390', '.', '-', '.', 'gene_id \"chr23.g1340\"; transcript_id \"chr23.g1340.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2964167', '2964217', '.', '-', '.', 'gene_id \"chr23.g1376\"; transcript_id \"chr23.g1376.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3330810', '3330857', '.', '-', '.', 'gene_id \"chr23.g1517\"; transcript_id \"chr23.g1517.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3419732', '3420083', '.', '+', '.', 'gene_id \"chr23.g1558\"; transcript_id \"chr23.g1558.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3443486', '3443704', '.', '+', '.', 'gene_id \"chr23.g1574\"; transcript_id \"chr23.g1574.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3472851', '3473041', '.', '-', '.', 'gene_id \"chr23.g1597\"; transcript_id \"chr23.g1597.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr23', 'PASA', 'exon', '3563602', '3563653', '.', '-', '.', 'gene_id \"chr23.g1712\"; transcript_id \"chr23.g1712.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3598336', '3598480', '.', '+', '.', 'gene_id \"chr23.g1740\"; transcript_id \"chr23.g1740.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3638093', '3638375', '.', '-', '.', 'gene_id \"chr23.g1787\"; transcript_id \"chr23.g1787.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3781177', '3781404', '.', '+', '.', 'gene_id \"chr23.g1885\"; transcript_id \"chr23.g1885.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3781411', '3781443', '.', '-', '.', 'gene_id \"chr23.g1887\"; transcript_id \"chr23.g1887.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3781411', '3782005', '.', '-', '.', 'gene_id \"chr23.g1887\"; transcript_id \"chr23.g1887.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3883597', '3883832', '.', '-', '.', 'gene_id \"chr23.g1971\"; transcript_id \"chr23.g1971.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3886744', '3887009', '.', '+', '.', 'gene_id \"chr23.g1973\"; transcript_id \"chr23.g1973.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3934283', '3934326', '.', '+', '.', 'gene_id \"chr23.g2008\"; transcript_id \"chr23.g2008.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '4229843', '4230115', '.', '-', '.', 'gene_id \"chr23.g2166\"; transcript_id \"chr23.g2166.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '4667113', '4667244', '.', '-', '.', 'gene_id \"chr23.g2221\"; transcript_id \"chr23.g2221.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '5402823', '5402859', '.', '+', '.', 'gene_id \"chr23.g2282\"; transcript_id \"chr23.g2282.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '5594088', '5594221', '.', '-', '.', 'gene_id \"chr23.g2307\"; transcript_id \"chr23.g2307.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '5822299', '5822365', '.', '+', '.', 'gene_id \"chr23.g2324\"; transcript_id \"chr23.g2324.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '5899199', '5899383', '.', '+', '.', 'gene_id \"chr23.g2388\"; transcript_id \"chr23.g2388.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6037010', '6037046', '.', '-', '.', 'gene_id \"chr23.g2423\"; transcript_id \"chr23.g2423.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6671153', '6671217', '.', '-', '.', 'gene_id \"chr23.g2503\"; transcript_id \"chr23.g2503.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6671153', '6671221', '.', '-', '.', 'gene_id \"chr23.g2503\"; transcript_id \"chr23.g2503.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6986204', '6989868', '.', '-', '.', 'gene_id \"chr23.g2598\"; transcript_id \"chr23.g2598.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6987796', '6987900', '.', '-', '.', 'gene_id \"chr23.g2598\"; transcript_id \"chr23.g2598.i3\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '7044807', '7045183', '.', '+', '.', 'gene_id \"chr23.g2639\"; transcript_id \"chr23.g2639.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '7716735', '7716964', '.', '+', '.', 'gene_id \"chr23.g2924\"; transcript_id \"chr23.g2924.i3\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '7657591', '7657649', '.', '+', '.', 'gene_id \"chr23.g3004\"; transcript_id \"chr23.g3004.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '7801840', '7801947', '.', '+', '.', 'gene_id \"chr23.g3149\"; transcript_id \"chr23.g3149.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '8146972', '8147023', '.', '-', '.', 'gene_id \"chr23.g3427\"; transcript_id \"chr23.g3427.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '8388399', '8388594', '.', '+', '.', 'gene_id \"chr23.g3580\"; transcript_id \"chr23.g3580.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '8388030', '8388417', '.', '+', '.', 'gene_id \"chr23.g3581\"; transcript_id \"chr23.g3581.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '8865583', '8865692', '.', '+', '.', 'gene_id \"chr23.g3656\"; transcript_id \"chr23.g3656.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '9294598', '9294890', '.', '+', '.', 'gene_id \"chr23.g3949\"; transcript_id \"chr23.g3949.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '9772842', '9773076', '.', '+', '.', 'gene_id \"chr23.g4092\"; transcript_id \"chr23.g4092.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '9802135', '9802312', '.', '+', '.', 'gene_id \"chr23.g4102\"; transcript_id \"chr23.g4102.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '10343941', '10344017', '.', '+', '.', 'gene_id \"chr23.g4396\"; transcript_id \"chr23.g4396.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '10841416', '10841457', '.', '+', '.', 'gene_id \"chr23.g4497\"; transcript_id \"chr23.g4497.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '11447921', '11447978', '.', '-', '.', 'gene_id \"chr23.g4581\"; transcript_id \"chr23.g4581.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '11625635', '11625736', '.', '-', '.', 'gene_id \"chr23.g4719\"; transcript_id \"chr23.g4719.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '11694621', '11694657', '.', '+', '.', 'gene_id \"chr23.g4724\"; transcript_id \"chr23.g4724.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '11964276', '11964349', '.', '-', '.', 'gene_id \"chr23.g4899\"; transcript_id \"chr23.g4899.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12187054', '12187688', '.', '-', '.', 'gene_id \"chr23.g5078\"; transcript_id \"chr23.g5078.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12192539', '12192734', '.', '-', '.', 'gene_id \"chr23.g5085\"; transcript_id \"chr23.g5085.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12241569', '12241615', '.', '+', '.', 'gene_id \"chr23.g5119\"; transcript_id \"chr23.g5119.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12308005', '12308184', '.', '-', '.', 'gene_id \"chr23.g5144\"; transcript_id \"chr23.g5144.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12691102', '12691310', '.', '+', '.', 'gene_id \"chr23.g5458\"; transcript_id \"chr23.g5458.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12864781', '12865148', '.', '+', '.', 'gene_id \"chr23.g5521\"; transcript_id \"chr23.g5521.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '13094693', '13094939', '.', '+', '.', 'gene_id \"chr23.g5557\"; transcript_id \"chr23.g5557.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '13657107', '13657268', '.', '+', '.', 'gene_id \"chr23.g5712\"; transcript_id \"chr23.g5712.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '14035909', '14036063', '.', '+', '.', 'gene_id \"chr23.g5807\"; transcript_id \"chr23.g5807.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '14865459', '14865628', '.', '+', '.', 'gene_id \"chr23.g6030\"; transcript_id \"chr23.g6030.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15243469', '15243629', '.', '+', '.', 'gene_id \"chr23.g6267\"; transcript_id \"chr23.g6267.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15256523', '15257193', '.', '-', '.', 'gene_id \"chr23.g6271\"; transcript_id \"chr23.g6271.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15255618', '15256541', '.', '-', '.', 'gene_id \"chr23.g6272\"; transcript_id \"chr23.g6272.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15698725', '15698938', '.', '-', '.', 'gene_id \"chr23.g6332\"; transcript_id \"chr23.g6332.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15859245', '15859774', '.', '-', '.', 'gene_id \"chr23.g6373\"; transcript_id \"chr23.g6373.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '16827757', '16827955', '.', '-', '.', 'gene_id \"chr23.g6655\"; transcript_id \"chr23.g6655.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '17444376', '17444665', '.', '+', '.', 'gene_id \"chr23.g6804\"; transcript_id \"chr23.g6804.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '18206643', '18206911', '.', '-', '.', 'gene_id \"chr23.g6941\"; transcript_id \"chr23.g6941.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '18719768', '18719813', '.', '-', '.', 'gene_id \"chr23.g7122\"; transcript_id \"chr23.g7122.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19594157', '19594409', '.', '-', '.', 'gene_id \"chr23.g7382\"; transcript_id \"chr23.g7382.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19599438', '19599596', '.', '-', '.', 'gene_id \"chr23.g7387\"; transcript_id \"chr23.g7387.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19776094', '19776369', '.', '-', '.', 'gene_id \"chr23.g7529\"; transcript_id \"chr23.g7529.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19950841', '19951175', '.', '+', '.', 'gene_id \"chr23.g7597\"; transcript_id \"chr23.g7597.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19950193', '19950449', '.', '-', '.', 'gene_id \"chr23.g7607\"; transcript_id \"chr23.g7607.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19978369', '19978620', '.', '+', '.', 'gene_id \"chr23.g7629\"; transcript_id \"chr23.g7629.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19978737', '19979001', '.', '-', '.', 'gene_id \"chr23.g7630\"; transcript_id \"chr23.g7630.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20277158', '20277384', '.', '+', '.', 'gene_id \"chr23.g7833\"; transcript_id \"chr23.g7833.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20277879', '20278443', '.', '-', '.', 'gene_id \"chr23.g7836\"; transcript_id \"chr23.g7836.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20289805', '20290075', '.', '-', '.', 'gene_id \"chr23.g7850\"; transcript_id \"chr23.g7850.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20295586', '20295800', '.', '+', '.', 'gene_id \"chr23.g7853\"; transcript_id \"chr23.g7853.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20296083', '20297179', '.', '-', '.', 'gene_id \"chr23.g7855\"; transcript_id \"chr23.g7855.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20341690', '20341861', '.', '-', '.', 'gene_id \"chr23.g7878\"; transcript_id \"chr23.g7878.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20375077', '20375239', '.', '+', '.', 'gene_id \"chr23.g7890\"; transcript_id \"chr23.g7890.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20396553', '20396761', '.', '-', '.', 'gene_id \"chr23.g7899\"; transcript_id \"chr23.g7899.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20423007', '20423078', '.', '-', '.', 'gene_id \"chr23.g7922\"; transcript_id \"chr23.g7922.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20453273', '20453431', '.', '+', '.', 'gene_id \"chr23.g7935\"; transcript_id \"chr23.g7935.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr23', 'PASA', 'exon', '20676803', '20676838', '.', '+', '.', 'gene_id \"chr23.g8093\"; transcript_id \"chr23.g8093.i1\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '23960041', '23960239', '100', '+', '.', 'gene_id \"chr1.g48340\"; transcript_id \"blat.proc4587.chain_14008\";']\n",
      "negative:  ['chr1', 'PASA', 'exon', '23925899', '23926160', '100', '-', '.', 'gene_id \"chr1.g52686\"; transcript_id \"blat.proc4587.chain_14009\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '21362723', '21362784', '.', '+', '.', 'gene_id \"chr23.g8166\"; transcript_id \"chr23.g8166.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '21855400', '21855852', '.', '-', '.', 'gene_id \"chr23.g8217\"; transcript_id \"chr23.g8217.i1\";']\n",
      "negative:  ['scaffold_1549', 'PASA', 'exon', '19888', '19920', '.', '-', '.', 'gene_id \"scaffold_1549.g1\"; transcript_id \"scaffold_1549.g1.i1\";']\n",
      "negative:  ['scaffold_1549', 'PASA', 'exon', '2078', '2163', '100', '+', '.', 'gene_id \"scaffold_1549.g2\"; transcript_id \"blat.proc4587.chain_9011\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '22319569', '22319602', '.', '-', '.', 'gene_id \"chr23.g8308\"; transcript_id \"chr23.g8308.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '22485321', '22485352', '.', '-', '.', 'gene_id \"chr23.g8322\"; transcript_id \"chr23.g8322.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '62271', '62435', '.', '-', '.', 'gene_id \"chr37.g55\"; transcript_id \"chr37.g55.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '23280665', '23280892', '.', '+', '.', 'gene_id \"chr23.g8450\"; transcript_id \"chr23.g8450.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '266865', '267154', '.', '-', '.', 'gene_id \"chr37.g141\"; transcript_id \"chr37.g141.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '267631', '267784', '.', '+', '.', 'gene_id \"chr37.g144\"; transcript_id \"chr37.g144.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '331964', '332195', '.', '-', '.', 'gene_id \"chr37.g206\"; transcript_id \"chr37.g206.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '332419', '332876', '.', '+', '.', 'gene_id \"chr37.g214\"; transcript_id \"chr37.g214.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '24066700', '24066850', '.', '-', '.', 'gene_id \"chr23.g8694\"; transcript_id \"chr23.g8694.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '648141', '648293', '.', '-', '.', 'gene_id \"chr37.g405\"; transcript_id \"chr37.g405.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '909209', '909485', '.', '-', '.', 'gene_id \"chr37.g564\"; transcript_id \"chr37.g564.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '910075', '910201', '.', '-', '.', 'gene_id \"chr37.g566\"; transcript_id \"chr37.g566.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '911064', '911276', '.', '-', '.', 'gene_id \"chr37.g567\"; transcript_id \"chr37.g567.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '930589', '932186', '.', '-', '.', 'gene_id \"chr37.g576\"; transcript_id \"chr37.g576.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '956013', '956110', '.', '-', '.', 'gene_id \"chr37.g577\"; transcript_id \"chr37.g577.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '955762', '956110', '.', '-', '.', 'gene_id \"chr37.g577\"; transcript_id \"chr37.g577.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '972773', '975040', '.', '-', '.', 'gene_id \"chr37.g587\"; transcript_id \"chr37.g587.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '959815', '960613', '.', '+', '.', 'gene_id \"chr37.g624\"; transcript_id \"chr37.g624.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '960834', '961573', '.', '+', '.', 'gene_id \"chr37.g626\"; transcript_id \"chr37.g626.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '25596309', '25596486', '.', '+', '.', 'gene_id \"chr23.g9009\"; transcript_id \"chr23.g9009.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '1287604', '1287640', '.', '-', '.', 'gene_id \"chr37.g752\"; transcript_id \"chr37.g752.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '25895895', '25896178', '.', '+', '.', 'gene_id \"chr23.g9221\"; transcript_id \"chr23.g9221.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '25945372', '25945733', '.', '+', '.', 'gene_id \"chr23.g9256\"; transcript_id \"chr23.g9256.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '26044120', '26044319', '.', '+', '.', 'gene_id \"chr23.g9344\"; transcript_id \"chr23.g9344.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '1726375', '1726553', '.', '-', '.', 'gene_id \"chr37.g1057\"; transcript_id \"chr37.g1057.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '1726569', '1726747', '.', '+', '.', 'gene_id \"chr37.g1058\"; transcript_id \"chr37.g1058.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '1775329', '1775494', '.', '-', '.', 'gene_id \"chr37.g1073\"; transcript_id \"chr37.g1073.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2189191', '2189397', '.', '-', '.', 'gene_id \"chr37.g1157\"; transcript_id \"chr37.g1157.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2347237', '2347394', '.', '-', '.', 'gene_id \"chr37.g1230\"; transcript_id \"chr37.g1230.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2347871', '2348237', '.', '-', '.', 'gene_id \"chr37.g1232\"; transcript_id \"chr37.g1232.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2348288', '2348620', '.', '-', '.', 'gene_id \"chr37.g1233\"; transcript_id \"chr37.g1233.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2348663', '2348852', '.', '+', '.', 'gene_id \"chr37.g1234\"; transcript_id \"chr37.g1234.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2349474', '2349818', '.', '-', '.', 'gene_id \"chr37.g1238\"; transcript_id \"chr37.g1238.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2351356', '2351529', '.', '-', '.', 'gene_id \"chr37.g1241\"; transcript_id \"chr37.g1241.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2691379', '2691469', '.', '-', '.', 'gene_id \"chr37.g1341\"; transcript_id \"chr37.g1341.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2691463', '2691650', '.', '-', '.', 'gene_id \"chr37.g1341\"; transcript_id \"chr37.g1341.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '2691379', '2691854', '.', '+', '.', 'gene_id \"chr37.g1342\"; transcript_id \"chr37.g1342.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '3929801', '3929860', '.', '-', '.', 'gene_id \"chr37.g1395\"; transcript_id \"chr37.g1395.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '26921905', '26921948', '.', '+', '.', 'gene_id \"chr23.g9725\"; transcript_id \"chr23.g9725.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '26939609', '26939978', '.', '+', '.', 'gene_id \"chr23.g9732\"; transcript_id \"chr23.g9732.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '26940340', '26940539', '.', '-', '.', 'gene_id \"chr23.g9733\"; transcript_id \"chr23.g9733.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '4646993', '4647038', '.', '-', '.', 'gene_id \"chr37.g1436\"; transcript_id \"chr37.g1436.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5685552', '5685969', '.', '+', '.', 'gene_id \"chr37.g1623\"; transcript_id \"chr37.g1623.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5687480', '5687794', '.', '+', '.', 'gene_id \"chr37.g1625\"; transcript_id \"chr37.g1625.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5691467', '5691681', '.', '-', '.', 'gene_id \"chr37.g1628\"; transcript_id \"chr37.g1628.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5691717', '5691878', '.', '-', '.', 'gene_id \"chr37.g1629\"; transcript_id \"chr37.g1629.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5696040', '5696202', '.', '+', '.', 'gene_id \"chr37.g1632\"; transcript_id \"chr37.g1632.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5699509', '5699809', '.', '+', '.', 'gene_id \"chr37.g1633\"; transcript_id \"chr37.g1633.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5700096', '5700635', '.', '+', '.', 'gene_id \"chr37.g1635\"; transcript_id \"chr37.g1635.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '27270702', '27270947', '.', '+', '.', 'gene_id \"chr23.g9985\"; transcript_id \"chr23.g9985.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5706213', '5706375', '.', '+', '.', 'gene_id \"chr37.g1638\"; transcript_id \"chr37.g1638.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5706480', '5706678', '.', '+', '.', 'gene_id \"chr37.g1639\"; transcript_id \"chr37.g1639.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5707406', '5707438', '.', '+', '.', 'gene_id \"chr37.g1641\"; transcript_id \"chr37.g1641.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5710082', '5710384', '.', '-', '.', 'gene_id \"chr37.g1643\"; transcript_id \"chr37.g1643.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5710552', '5710951', '.', '+', '.', 'gene_id \"chr37.g1644\"; transcript_id \"chr37.g1644.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5710930', '5711076', '.', '-', '.', 'gene_id \"chr37.g1645\"; transcript_id \"chr37.g1645.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5711079', '5711305', '.', '+', '.', 'gene_id \"chr37.g1647\"; transcript_id \"chr37.g1647.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5712271', '5712467', '.', '-', '.', 'gene_id \"chr37.g1649\"; transcript_id \"chr37.g1649.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5712490', '5712784', '.', '+', '.', 'gene_id \"chr37.g1650\"; transcript_id \"chr37.g1650.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5712596', '5712784', '.', '-', '.', 'gene_id \"chr37.g1651\"; transcript_id \"chr37.g1651.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5713228', '5713548', '.', '+', '.', 'gene_id \"chr37.g1652\"; transcript_id \"chr37.g1652.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5741003', '5745797', '.', '-', '.', 'gene_id \"chr37.g1662\"; transcript_id \"chr37.g1662.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5762398', '5762807', '.', '-', '.', 'gene_id \"chr37.g1663\"; transcript_id \"chr37.g1663.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5763004', '5763301', '.', '+', '.', 'gene_id \"chr37.g1665\"; transcript_id \"chr37.g1665.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5783426', '5783619', '.', '+', '.', 'gene_id \"chr37.g1668\"; transcript_id \"chr37.g1668.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5798070', '5798412', '.', '-', '.', 'gene_id \"chr37.g1670\"; transcript_id \"chr37.g1670.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5783071', '5783437', '.', '+', '.', 'gene_id \"chr37.g1672\"; transcript_id \"chr37.g1672.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5765457', '5765628', '.', '-', '.', 'gene_id \"chr37.g1673\"; transcript_id \"chr37.g1673.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5746118', '5746549', '.', '-', '.', 'gene_id \"chr37.g1675\"; transcript_id \"chr37.g1675.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5746718', '5746868', '.', '+', '.', 'gene_id \"chr37.g1676\"; transcript_id \"chr37.g1676.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5746952', '5747257', '.', '-', '.', 'gene_id \"chr37.g1677\"; transcript_id \"chr37.g1677.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5748644', '5748797', '.', '-', '.', 'gene_id \"chr37.g1678\"; transcript_id \"chr37.g1678.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5750798', '5750997', '.', '-', '.', 'gene_id \"chr37.g1682\"; transcript_id \"chr37.g1682.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5753090', '5753249', '.', '+', '.', 'gene_id \"chr37.g1686\"; transcript_id \"chr37.g1686.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5753262', '5753471', '.', '+', '.', 'gene_id \"chr37.g1687\"; transcript_id \"chr37.g1687.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5753468', '5753671', '.', '-', '.', 'gene_id \"chr37.g1688\"; transcript_id \"chr37.g1688.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5754041', '5754362', '.', '-', '.', 'gene_id \"chr37.g1689\"; transcript_id \"chr37.g1689.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5754589', '5754844', '.', '-', '.', 'gene_id \"chr37.g1691\"; transcript_id \"chr37.g1691.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5759651', '5759817', '.', '+', '.', 'gene_id \"chr37.g1696\"; transcript_id \"chr37.g1696.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5760857', '5761009', '.', '-', '.', 'gene_id \"chr37.g1697\"; transcript_id \"chr37.g1697.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5761244', '5761859', '.', '-', '.', 'gene_id \"chr37.g1698\"; transcript_id \"chr37.g1698.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5764562', '5764729', '.', '+', '.', 'gene_id \"chr37.g1700\"; transcript_id \"chr37.g1700.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '27321894', '27322292', '.', '-', '.', 'gene_id \"chr23.g10039\"; transcript_id \"chr23.g10039.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5779486', '5779668', '.', '-', '.', 'gene_id \"chr37.g1702\"; transcript_id \"chr37.g1702.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5780717', '5780938', '.', '+', '.', 'gene_id \"chr37.g1704\"; transcript_id \"chr37.g1704.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5784741', '5784979', '.', '+', '.', 'gene_id \"chr37.g1705\"; transcript_id \"chr37.g1705.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5785489', '5785744', '.', '-', '.', 'gene_id \"chr37.g1706\"; transcript_id \"chr37.g1706.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5787036', '5787236', '.', '-', '.', 'gene_id \"chr37.g1708\"; transcript_id \"chr37.g1708.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5789188', '5789404', '.', '+', '.', 'gene_id \"chr37.g1711\"; transcript_id \"chr37.g1711.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5790686', '5791086', '.', '+', '.', 'gene_id \"chr37.g1712\"; transcript_id \"chr37.g1712.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5793132', '5793352', '.', '+', '.', 'gene_id \"chr37.g1714\"; transcript_id \"chr37.g1714.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5794416', '5794670', '.', '+', '.', 'gene_id \"chr37.g1715\"; transcript_id \"chr37.g1715.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5796274', '5796464', '.', '-', '.', 'gene_id \"chr37.g1716\"; transcript_id \"chr37.g1716.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5799045', '5799231', '.', '+', '.', 'gene_id \"chr37.g1722\"; transcript_id \"chr37.g1722.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5820146', '5820300', '.', '+', '.', 'gene_id \"chr37.g1728\"; transcript_id \"chr37.g1728.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5824298', '5824471', '.', '-', '.', 'gene_id \"chr37.g1729\"; transcript_id \"chr37.g1729.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5826234', '5826396', '.', '+', '.', 'gene_id \"chr37.g1730\"; transcript_id \"chr37.g1730.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5852388', '5852561', '.', '-', '.', 'gene_id \"chr37.g1736\"; transcript_id \"chr37.g1736.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5853648', '5854018', '.', '+', '.', 'gene_id \"chr37.g1737\"; transcript_id \"chr37.g1737.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5854021', '5854251', '.', '+', '.', 'gene_id \"chr37.g1738\"; transcript_id \"chr37.g1738.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5855705', '5856055', '.', '-', '.', 'gene_id \"chr37.g1741\"; transcript_id \"chr37.g1741.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5856056', '5856500', '.', '+', '.', 'gene_id \"chr37.g1742\"; transcript_id \"chr37.g1742.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5858150', '5858333', '.', '+', '.', 'gene_id \"chr37.g1745\"; transcript_id \"chr37.g1745.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5867056', '5867313', '.', '-', '.', 'gene_id \"chr37.g1749\"; transcript_id \"chr37.g1749.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5867619', '5867874', '.', '+', '.', 'gene_id \"chr37.g1751\"; transcript_id \"chr37.g1751.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5868611', '5868879', '.', '+', '.', 'gene_id \"chr37.g1753\"; transcript_id \"chr37.g1753.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5869311', '5869725', '.', '-', '.', 'gene_id \"chr37.g1755\"; transcript_id \"chr37.g1755.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5870982', '5871268', '.', '-', '.', 'gene_id \"chr37.g1756\"; transcript_id \"chr37.g1756.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5927339', '5927513', '.', '+', '.', 'gene_id \"chr37.g1770\"; transcript_id \"chr37.g1770.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5935772', '5935949', '.', '+', '.', 'gene_id \"chr37.g1776\"; transcript_id \"chr37.g1776.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5942790', '5942960', '.', '-', '.', 'gene_id \"chr37.g1777\"; transcript_id \"chr37.g1777.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5943207', '5943427', '.', '-', '.', 'gene_id \"chr37.g1778\"; transcript_id \"chr37.g1778.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5956135', '5956293', '.', '+', '.', 'gene_id \"chr37.g1779\"; transcript_id \"chr37.g1779.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '5966181', '5966506', '.', '-', '.', 'gene_id \"chr37.g1783\"; transcript_id \"chr37.g1783.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6030233', '6030445', '.', '-', '.', 'gene_id \"chr37.g1791\"; transcript_id \"chr37.g1791.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6034563', '6034735', '.', '+', '.', 'gene_id \"chr37.g1794\"; transcript_id \"chr37.g1794.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6041003', '6041073', '.', '-', '.', 'gene_id \"chr37.g1799\"; transcript_id \"chr37.g1799.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6072546', '6072706', '.', '+', '.', 'gene_id \"chr37.g1800\"; transcript_id \"chr37.g1800.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6095360', '6095714', '.', '-', '.', 'gene_id \"chr37.g1805\"; transcript_id \"chr37.g1805.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6102731', '6103228', '.', '-', '.', 'gene_id \"chr37.g1808\"; transcript_id \"chr37.g1808.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6104742', '6105003', '.', '+', '.', 'gene_id \"chr37.g1811\"; transcript_id \"chr37.g1811.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6167259', '6167838', '.', '+', '.', 'gene_id \"chr37.g1819\"; transcript_id \"chr37.g1819.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6165013', '6165631', '.', '-', '.', 'gene_id \"chr37.g1821\"; transcript_id \"chr37.g1821.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6151255', '6151420', '.', '+', '.', 'gene_id \"chr37.g1823\"; transcript_id \"chr37.g1823.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6152226', '6152379', '.', '+', '.', 'gene_id \"chr37.g1824\"; transcript_id \"chr37.g1824.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6156926', '6157107', '.', '-', '.', 'gene_id \"chr37.g1828\"; transcript_id \"chr37.g1828.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6160448', '6160640', '.', '+', '.', 'gene_id \"chr37.g1830\"; transcript_id \"chr37.g1830.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6162909', '6163522', '.', '+', '.', 'gene_id \"chr37.g1834\"; transcript_id \"chr37.g1834.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6167873', '6168123', '.', '+', '.', 'gene_id \"chr37.g1837\"; transcript_id \"chr37.g1837.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6168312', '6168473', '.', '-', '.', 'gene_id \"chr37.g1838\"; transcript_id \"chr37.g1838.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6225824', '6226778', '.', '+', '.', 'gene_id \"chr37.g1843\"; transcript_id \"chr37.g1843.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6208421', '6208976', '.', '+', '.', 'gene_id \"chr37.g1843\"; transcript_id \"chr37.g1843.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6190816', '6190931', '.', '+', '.', 'gene_id \"chr37.g1843\"; transcript_id \"chr37.g1843.i13\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6203741', '6204433', '.', '+', '.', 'gene_id \"chr37.g1843\"; transcript_id \"chr37.g1843.i14\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6213656', '6213842', '.', '+', '.', 'gene_id \"chr37.g1845\"; transcript_id \"chr37.g1845.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6232139', '6233064', '.', '+', '.', 'gene_id \"chr37.g1850\"; transcript_id \"chr37.g1850.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6202962', '6203140', '.', '+', '.', 'gene_id \"chr37.g1853\"; transcript_id \"chr37.g1853.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6206698', '6207009', '.', '+', '.', 'gene_id \"chr37.g1857\"; transcript_id \"chr37.g1857.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6212736', '6212923', '.', '-', '.', 'gene_id \"chr37.g1860\"; transcript_id \"chr37.g1860.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6218508', '6218730', '.', '-', '.', 'gene_id \"chr37.g1866\"; transcript_id \"chr37.g1866.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6219021', '6219300', '.', '-', '.', 'gene_id \"chr37.g1868\"; transcript_id \"chr37.g1868.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6220164', '6220317', '.', '-', '.', 'gene_id \"chr37.g1869\"; transcript_id \"chr37.g1869.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6220477', '6220676', '.', '+', '.', 'gene_id \"chr37.g1870\"; transcript_id \"chr37.g1870.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6229975', '6230256', '.', '-', '.', 'gene_id \"chr37.g1872\"; transcript_id \"chr37.g1872.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6234172', '6234565', '.', '-', '.', 'gene_id \"chr37.g1877\"; transcript_id \"chr37.g1877.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6234983', '6235200', '.', '+', '.', 'gene_id \"chr37.g1879\"; transcript_id \"chr37.g1879.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6235807', '6235972', '.', '+', '.', 'gene_id \"chr37.g1880\"; transcript_id \"chr37.g1880.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6236045', '6236446', '.', '-', '.', 'gene_id \"chr37.g1881\"; transcript_id \"chr37.g1881.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6238855', '6239015', '.', '+', '.', 'gene_id \"chr37.g1885\"; transcript_id \"chr37.g1885.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6239127', '6239281', '.', '+', '.', 'gene_id \"chr37.g1886\"; transcript_id \"chr37.g1886.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6239314', '6239466', '.', '+', '.', 'gene_id \"chr37.g1887\"; transcript_id \"chr37.g1887.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6240081', '6240244', '.', '-', '.', 'gene_id \"chr37.g1888\"; transcript_id \"chr37.g1888.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6242095', '6242481', '.', '+', '.', 'gene_id \"chr37.g1889\"; transcript_id \"chr37.g1889.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6244691', '6245667', '.', '+', '.', 'gene_id \"chr37.g1892\"; transcript_id \"chr37.g1892.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6245313', '6245667', '.', '+', '.', 'gene_id \"chr37.g1892\"; transcript_id \"chr37.g1892.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6246275', '6246546', '.', '+', '.', 'gene_id \"chr37.g1894\"; transcript_id \"chr37.g1894.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6268374', '6268661', '.', '+', '.', 'gene_id \"chr37.g1905\"; transcript_id \"chr37.g1905.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6256201', '6256592', '.', '+', '.', 'gene_id \"chr37.g1906\"; transcript_id \"chr37.g1906.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6251281', '6252230', '.', '-', '.', 'gene_id \"chr37.g1907\"; transcript_id \"chr37.g1907.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6252234', '6252637', '.', '-', '.', 'gene_id \"chr37.g1908\"; transcript_id \"chr37.g1908.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6253819', '6254325', '.', '+', '.', 'gene_id \"chr37.g1910\"; transcript_id \"chr37.g1910.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6254303', '6254537', '.', '-', '.', 'gene_id \"chr37.g1911\"; transcript_id \"chr37.g1911.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6254535', '6254958', '.', '+', '.', 'gene_id \"chr37.g1912\"; transcript_id \"chr37.g1912.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6259235', '6259441', '.', '+', '.', 'gene_id \"chr37.g1917\"; transcript_id \"chr37.g1917.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6260984', '6261151', '.', '-', '.', 'gene_id \"chr37.g1919\"; transcript_id \"chr37.g1919.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6261993', '6262146', '.', '-', '.', 'gene_id \"chr37.g1922\"; transcript_id \"chr37.g1922.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6263841', '6263993', '.', '+', '.', 'gene_id \"chr37.g1924\"; transcript_id \"chr37.g1924.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6267462', '6267838', '.', '-', '.', 'gene_id \"chr37.g1925\"; transcript_id \"chr37.g1925.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6284012', '6284182', '.', '+', '.', 'gene_id \"chr37.g1930\"; transcript_id \"chr37.g1930.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6284222', '6284866', '.', '-', '.', 'gene_id \"chr37.g1931\"; transcript_id \"chr37.g1931.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6286153', '6286366', '.', '+', '.', 'gene_id \"chr37.g1933\"; transcript_id \"chr37.g1933.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6293108', '6293261', '.', '-', '.', 'gene_id \"chr37.g1935\"; transcript_id \"chr37.g1935.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6296982', '6297187', '.', '+', '.', 'gene_id \"chr37.g1936\"; transcript_id \"chr37.g1936.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6304224', '6304399', '.', '+', '.', 'gene_id \"chr37.g1939\"; transcript_id \"chr37.g1939.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '27609387', '27609454', '.', '+', '.', 'gene_id \"chr23.g10302\"; transcript_id \"chr23.g10302.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6310164', '6310487', '.', '+', '.', 'gene_id \"chr37.g1942\"; transcript_id \"chr37.g1942.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6317992', '6318030', '.', '+', '.', 'gene_id \"chr37.g1945\"; transcript_id \"chr37.g1945.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6322521', '6322688', '.', '-', '.', 'gene_id \"chr37.g1946\"; transcript_id \"chr37.g1946.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6323529', '6323684', '.', '+', '.', 'gene_id \"chr37.g1948\"; transcript_id \"chr37.g1948.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6330126', '6330319', '.', '+', '.', 'gene_id \"chr37.g1953\"; transcript_id \"chr37.g1953.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6330919', '6331144', '.', '-', '.', 'gene_id \"chr37.g1955\"; transcript_id \"chr37.g1955.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6331122', '6331376', '.', '+', '.', 'gene_id \"chr37.g1956\"; transcript_id \"chr37.g1956.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6345733', '6345891', '.', '-', '.', 'gene_id \"chr37.g1958\"; transcript_id \"chr37.g1958.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6354857', '6355181', '.', '-', '.', 'gene_id \"chr37.g1960\"; transcript_id \"chr37.g1960.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6356033', '6356384', '.', '+', '.', 'gene_id \"chr37.g1961\"; transcript_id \"chr37.g1961.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6360804', '6361549', '.', '+', '.', 'gene_id \"chr37.g1965\"; transcript_id \"chr37.g1965.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6362206', '6362382', '.', '+', '.', 'gene_id \"chr37.g1966\"; transcript_id \"chr37.g1966.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6362767', '6363043', '.', '+', '.', 'gene_id \"chr37.g1967\"; transcript_id \"chr37.g1967.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6364866', '6365244', '.', '+', '.', 'gene_id \"chr37.g1971\"; transcript_id \"chr37.g1971.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6365226', '6365619', '.', '+', '.', 'gene_id \"chr37.g1972\"; transcript_id \"chr37.g1972.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6374689', '6375049', '.', '+', '.', 'gene_id \"chr37.g1976\"; transcript_id \"chr37.g1976.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6375154', '6375484', '.', '-', '.', 'gene_id \"chr37.g1977\"; transcript_id \"chr37.g1977.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6376870', '6377078', '.', '-', '.', 'gene_id \"chr37.g1979\"; transcript_id \"chr37.g1979.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6377839', '6377994', '.', '-', '.', 'gene_id \"chr37.g1980\"; transcript_id \"chr37.g1980.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6390588', '6390861', '.', '-', '.', 'gene_id \"chr37.g1982\"; transcript_id \"chr37.g1982.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6391308', '6391705', '.', '-', '.', 'gene_id \"chr37.g1984\"; transcript_id \"chr37.g1984.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6392672', '6392989', '.', '+', '.', 'gene_id \"chr37.g1986\"; transcript_id \"chr37.g1986.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6396485', '6396656', '.', '-', '.', 'gene_id \"chr37.g1987\"; transcript_id \"chr37.g1987.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6420812', '6421040', '.', '-', '.', 'gene_id \"chr37.g1990\"; transcript_id \"chr37.g1990.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6439177', '6442177', '.', '-', '.', 'gene_id \"chr37.g1991\"; transcript_id \"chr37.g1991.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6454841', '6455040', '.', '-', '.', 'gene_id \"chr37.g1994\"; transcript_id \"chr37.g1994.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6470388', '6471083', '.', '-', '.', 'gene_id \"chr37.g1995\"; transcript_id \"chr37.g1995.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6444623', '6444941', '.', '+', '.', 'gene_id \"chr37.g1998\"; transcript_id \"chr37.g1998.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6459652', '6459889', '.', '-', '.', 'gene_id \"chr37.g2002\"; transcript_id \"chr37.g2002.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6460976', '6461253', '.', '+', '.', 'gene_id \"chr37.g2003\"; transcript_id \"chr37.g2003.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6464139', '6464365', '.', '-', '.', 'gene_id \"chr37.g2009\"; transcript_id \"chr37.g2009.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6463869', '6464151', '.', '+', '.', 'gene_id \"chr37.g2010\"; transcript_id \"chr37.g2010.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6469250', '6469981', '.', '-', '.', 'gene_id \"chr37.g2011\"; transcript_id \"chr37.g2011.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6469250', '6469961', '.', '-', '.', 'gene_id \"chr37.g2011\"; transcript_id \"chr37.g2011.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6468840', '6469259', '.', '+', '.', 'gene_id \"chr37.g2012\"; transcript_id \"chr37.g2012.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6472019', '6472175', '.', '-', '.', 'gene_id \"chr37.g2014\"; transcript_id \"chr37.g2014.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6497131', '6497288', '.', '-', '.', 'gene_id \"chr37.g2016\"; transcript_id \"chr37.g2016.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6500013', '6500229', '.', '+', '.', 'gene_id \"chr37.g2018\"; transcript_id \"chr37.g2018.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6500880', '6501127', '.', '+', '.', 'gene_id \"chr37.g2020\"; transcript_id \"chr37.g2020.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6518992', '6519201', '.', '-', '.', 'gene_id \"chr37.g2028\"; transcript_id \"chr37.g2028.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6519222', '6519415', '.', '+', '.', 'gene_id \"chr37.g2029\"; transcript_id \"chr37.g2029.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6522020', '6522598', '.', '+', '.', 'gene_id \"chr37.g2033\"; transcript_id \"chr37.g2033.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6525433', '6526345', '.', '+', '.', 'gene_id \"chr37.g2038\"; transcript_id \"chr37.g2038.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6527304', '6527877', '.', '+', '.', 'gene_id \"chr37.g2039\"; transcript_id \"chr37.g2039.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6528402', '6528688', '.', '-', '.', 'gene_id \"chr37.g2041\"; transcript_id \"chr37.g2041.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6528187', '6528415', '.', '-', '.', 'gene_id \"chr37.g2043\"; transcript_id \"chr37.g2043.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6530812', '6531051', '.', '-', '.', 'gene_id \"chr37.g2046\"; transcript_id \"chr37.g2046.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6531500', '6531669', '.', '+', '.', 'gene_id \"chr37.g2048\"; transcript_id \"chr37.g2048.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6531677', '6531966', '.', '-', '.', 'gene_id \"chr37.g2049\"; transcript_id \"chr37.g2049.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6532367', '6532552', '.', '+', '.', 'gene_id \"chr37.g2050\"; transcript_id \"chr37.g2050.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6534313', '6534455', '.', '+', '.', 'gene_id \"chr37.g2051\"; transcript_id \"chr37.g2051.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6534584', '6534908', '.', '-', '.', 'gene_id \"chr37.g2053\"; transcript_id \"chr37.g2053.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6535325', '6535516', '.', '-', '.', 'gene_id \"chr37.g2055\"; transcript_id \"chr37.g2055.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6537694', '6537855', '.', '+', '.', 'gene_id \"chr37.g2057\"; transcript_id \"chr37.g2057.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6537842', '6538202', '.', '+', '.', 'gene_id \"chr37.g2058\"; transcript_id \"chr37.g2058.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6545671', '6545843', '.', '+', '.', 'gene_id \"chr37.g2061\"; transcript_id \"chr37.g2061.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6545671', '6545844', '.', '+', '.', 'gene_id \"chr37.g2061\"; transcript_id \"chr37.g2061.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6545465', '6545679', '.', '-', '.', 'gene_id \"chr37.g2062\"; transcript_id \"chr37.g2062.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6547585', '6547749', '.', '-', '.', 'gene_id \"chr37.g2063\"; transcript_id \"chr37.g2063.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6548656', '6548811', '.', '+', '.', 'gene_id \"chr37.g2065\"; transcript_id \"chr37.g2065.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6549878', '6550033', '.', '-', '.', 'gene_id \"chr37.g2066\"; transcript_id \"chr37.g2066.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6550076', '6550368', '.', '-', '.', 'gene_id \"chr37.g2067\"; transcript_id \"chr37.g2067.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6550353', '6550511', '.', '-', '.', 'gene_id \"chr37.g2067\"; transcript_id \"chr37.g2067.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6553205', '6553481', '.', '+', '.', 'gene_id \"chr37.g2069\"; transcript_id \"chr37.g2069.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6557395', '6557537', '.', '+', '.', 'gene_id \"chr37.g2070\"; transcript_id \"chr37.g2070.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6557534', '6557710', '.', '+', '.', 'gene_id \"chr37.g2071\"; transcript_id \"chr37.g2071.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6563895', '6564090', '.', '-', '.', 'gene_id \"chr37.g2074\"; transcript_id \"chr37.g2074.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6565090', '6565256', '.', '+', '.', 'gene_id \"chr37.g2075\"; transcript_id \"chr37.g2075.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6566282', '6566524', '.', '-', '.', 'gene_id \"chr37.g2080\"; transcript_id \"chr37.g2080.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6566663', '6566907', '.', '-', '.', 'gene_id \"chr37.g2081\"; transcript_id \"chr37.g2081.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6568687', '6569035', '.', '-', '.', 'gene_id \"chr37.g2083\"; transcript_id \"chr37.g2083.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6569276', '6569446', '.', '-', '.', 'gene_id \"chr37.g2085\"; transcript_id \"chr37.g2085.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6570429', '6570754', '.', '-', '.', 'gene_id \"chr37.g2086\"; transcript_id \"chr37.g2086.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6571150', '6571538', '.', '-', '.', 'gene_id \"chr37.g2087\"; transcript_id \"chr37.g2087.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6575271', '6575624', '.', '-', '.', 'gene_id \"chr37.g2092\"; transcript_id \"chr37.g2092.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6589864', '6589934', '.', '-', '.', 'gene_id \"chr37.g2096\"; transcript_id \"chr37.g2096.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6590691', '6590940', '.', '+', '.', 'gene_id \"chr37.g2097\"; transcript_id \"chr37.g2097.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6591053', '6591307', '.', '+', '.', 'gene_id \"chr37.g2098\"; transcript_id \"chr37.g2098.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6599768', '6599886', '.', '+', '.', 'gene_id \"chr37.g2102\"; transcript_id \"chr37.g2102.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6601081', '6601159', '.', '-', '.', 'gene_id \"chr37.g2104\"; transcript_id \"chr37.g2104.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6620958', '6621412', '.', '-', '.', 'gene_id \"chr37.g2105\"; transcript_id \"chr37.g2105.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28208609', '28208643', '.', '-', '.', 'gene_id \"chr23.g10473\"; transcript_id \"chr23.g10473.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6662335', '6662906', '.', '-', '.', 'gene_id \"chr37.g2115\"; transcript_id \"chr37.g2115.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6660340', '6660785', '.', '+', '.', 'gene_id \"chr37.g2116\"; transcript_id \"chr37.g2116.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6664125', '6664365', '.', '+', '.', 'gene_id \"chr37.g2118\"; transcript_id \"chr37.g2118.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6674184', '6674506', '.', '-', '.', 'gene_id \"chr37.g2122\"; transcript_id \"chr37.g2122.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6675664', '6675844', '.', '+', '.', 'gene_id \"chr37.g2125\"; transcript_id \"chr37.g2125.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6692805', '6693000', '.', '-', '.', 'gene_id \"chr37.g2127\"; transcript_id \"chr37.g2127.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6694786', '6694937', '.', '+', '.', 'gene_id \"chr37.g2128\"; transcript_id \"chr37.g2128.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6695636', '6696067', '.', '+', '.', 'gene_id \"chr37.g2129\"; transcript_id \"chr37.g2129.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6696922', '6697112', '.', '+', '.', 'gene_id \"chr37.g2131\"; transcript_id \"chr37.g2131.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6699283', '6699663', '.', '-', '.', 'gene_id \"chr37.g2132\"; transcript_id \"chr37.g2132.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6701518', '6701691', '.', '-', '.', 'gene_id \"chr37.g2133\"; transcript_id \"chr37.g2133.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6703541', '6703777', '.', '+', '.', 'gene_id \"chr37.g2134\"; transcript_id \"chr37.g2134.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6703942', '6704451', '.', '+', '.', 'gene_id \"chr37.g2135\"; transcript_id \"chr37.g2135.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6711721', '6711895', '.', '+', '.', 'gene_id \"chr37.g2138\"; transcript_id \"chr37.g2138.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6712231', '6712411', '.', '-', '.', 'gene_id \"chr37.g2139\"; transcript_id \"chr37.g2139.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6712926', '6713085', '.', '-', '.', 'gene_id \"chr37.g2140\"; transcript_id \"chr37.g2140.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6713687', '6713930', '.', '+', '.', 'gene_id \"chr37.g2141\"; transcript_id \"chr37.g2141.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6720239', '6720442', '.', '+', '.', 'gene_id \"chr37.g2144\"; transcript_id \"chr37.g2144.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6721286', '6721493', '.', '-', '.', 'gene_id \"chr37.g2145\"; transcript_id \"chr37.g2145.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6721786', '6722013', '.', '+', '.', 'gene_id \"chr37.g2146\"; transcript_id \"chr37.g2146.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6736016', '6736203', '.', '+', '.', 'gene_id \"chr37.g2151\"; transcript_id \"chr37.g2151.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6736185', '6736249', '.', '-', '.', 'gene_id \"chr37.g2152\"; transcript_id \"chr37.g2152.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28279515', '28279559', '.', '-', '.', 'gene_id \"chr23.g10516\"; transcript_id \"chr23.g10516.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6736395', '6736718', '.', '-', '.', 'gene_id \"chr37.g2153\"; transcript_id \"chr37.g2153.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6736845', '6737105', '.', '-', '.', 'gene_id \"chr37.g2155\"; transcript_id \"chr37.g2155.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6738910', '6739073', '.', '-', '.', 'gene_id \"chr37.g2157\"; transcript_id \"chr37.g2157.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6740019', '6740397', '.', '+', '.', 'gene_id \"chr37.g2158\"; transcript_id \"chr37.g2158.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6741620', '6741949', '.', '-', '.', 'gene_id \"chr37.g2162\"; transcript_id \"chr37.g2162.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6744482', '6744646', '.', '-', '.', 'gene_id \"chr37.g2163\"; transcript_id \"chr37.g2163.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6745549', '6745727', '.', '+', '.', 'gene_id \"chr37.g2164\"; transcript_id \"chr37.g2164.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6748659', '6748895', '.', '-', '.', 'gene_id \"chr37.g2168\"; transcript_id \"chr37.g2168.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6750148', '6750342', '.', '-', '.', 'gene_id \"chr37.g2169\"; transcript_id \"chr37.g2169.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6751465', '6751747', '.', '-', '.', 'gene_id \"chr37.g2170\"; transcript_id \"chr37.g2170.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6751824', '6752071', '.', '-', '.', 'gene_id \"chr37.g2173\"; transcript_id \"chr37.g2173.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6752354', '6752681', '.', '+', '.', 'gene_id \"chr37.g2174\"; transcript_id \"chr37.g2174.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6754307', '6754478', '.', '-', '.', 'gene_id \"chr37.g2175\"; transcript_id \"chr37.g2175.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6756014', '6756400', '.', '-', '.', 'gene_id \"chr37.g2178\"; transcript_id \"chr37.g2178.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6811089', '6811459', '.', '+', '.', 'gene_id \"chr37.g2189\"; transcript_id \"chr37.g2189.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6825491', '6826836', '.', '-', '.', 'gene_id \"chr37.g2192\"; transcript_id \"chr37.g2192.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6823023', '6823173', '.', '-', '.', 'gene_id \"chr37.g2196\"; transcript_id \"chr37.g2196.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6822885', '6823038', '.', '+', '.', 'gene_id \"chr37.g2197\"; transcript_id \"chr37.g2197.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6826917', '6827160', '.', '-', '.', 'gene_id \"chr37.g2198\"; transcript_id \"chr37.g2198.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6832791', '6833050', '.', '-', '.', 'gene_id \"chr37.g2199\"; transcript_id \"chr37.g2199.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6818477', '6818802', '.', '+', '.', 'gene_id \"chr37.g2200\"; transcript_id \"chr37.g2200.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6813810', '6814032', '.', '-', '.', 'gene_id \"chr37.g2201\"; transcript_id \"chr37.g2201.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6836214', '6836401', '.', '+', '.', 'gene_id \"chr37.g2202\"; transcript_id \"chr37.g2202.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6816577', '6816828', '.', '-', '.', 'gene_id \"chr37.g2203\"; transcript_id \"chr37.g2203.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6817268', '6817621', '.', '+', '.', 'gene_id \"chr37.g2206\"; transcript_id \"chr37.g2206.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr37', 'PASA', 'exon', '6823681', '6823831', '.', '+', '.', 'gene_id \"chr37.g2210\"; transcript_id \"chr37.g2210.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6830400', '6830742', '.', '+', '.', 'gene_id \"chr37.g2211\"; transcript_id \"chr37.g2211.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6833450', '6833702', '.', '-', '.', 'gene_id \"chr37.g2214\"; transcript_id \"chr37.g2214.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6843953', '6844112', '.', '-', '.', 'gene_id \"chr37.g2217\"; transcript_id \"chr37.g2217.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6848026', '6848221', '.', '-', '.', 'gene_id \"chr37.g2223\"; transcript_id \"chr37.g2223.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6848775', '6849027', '.', '-', '.', 'gene_id \"chr37.g2227\"; transcript_id \"chr37.g2227.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6849019', '6849202', '.', '-', '.', 'gene_id \"chr37.g2228\"; transcript_id \"chr37.g2228.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6852642', '6853193', '.', '+', '.', 'gene_id \"chr37.g2229\"; transcript_id \"chr37.g2229.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6860928', '6861093', '.', '+', '.', 'gene_id \"chr37.g2229\"; transcript_id \"chr37.g2229.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6860726', '6860929', '.', '-', '.', 'gene_id \"chr37.g2230\"; transcript_id \"chr37.g2230.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6853254', '6854018', '.', '-', '.', 'gene_id \"chr37.g2231\"; transcript_id \"chr37.g2231.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6854114', '6854524', '.', '-', '.', 'gene_id \"chr37.g2234\"; transcript_id \"chr37.g2234.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6856263', '6856563', '.', '-', '.', 'gene_id \"chr37.g2235\"; transcript_id \"chr37.g2235.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6856263', '6856620', '.', '-', '.', 'gene_id \"chr37.g2235\"; transcript_id \"chr37.g2235.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6856534', '6857086', '.', '+', '.', 'gene_id \"chr37.g2236\"; transcript_id \"chr37.g2236.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6856602', '6857086', '.', '+', '.', 'gene_id \"chr37.g2236\"; transcript_id \"chr37.g2236.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6857123', '6857908', '.', '-', '.', 'gene_id \"chr37.g2238\"; transcript_id \"chr37.g2238.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6858080', '6858295', '.', '-', '.', 'gene_id \"chr37.g2241\"; transcript_id \"chr37.g2241.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6860220', '6860614', '.', '+', '.', 'gene_id \"chr37.g2243\"; transcript_id \"chr37.g2243.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6863458', '6863933', '.', '+', '.', 'gene_id \"chr37.g2244\"; transcript_id \"chr37.g2244.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6864025', '6864511', '.', '+', '.', 'gene_id \"chr37.g2246\"; transcript_id \"chr37.g2246.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6864830', '6865206', '.', '-', '.', 'gene_id \"chr37.g2248\"; transcript_id \"chr37.g2248.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6865190', '6865408', '.', '+', '.', 'gene_id \"chr37.g2249\"; transcript_id \"chr37.g2249.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6868279', '6869622', '.', '-', '.', 'gene_id \"chr37.g2253\"; transcript_id \"chr37.g2253.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6867688', '6867856', '.', '+', '.', 'gene_id \"chr37.g2254\"; transcript_id \"chr37.g2254.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6872354', '6872822', '.', '+', '.', 'gene_id \"chr37.g2261\"; transcript_id \"chr37.g2261.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6889541', '6889755', '.', '+', '.', 'gene_id \"chr37.g2262\"; transcript_id \"chr37.g2262.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6885974', '6886840', '.', '-', '.', 'gene_id \"chr37.g2265\"; transcript_id \"chr37.g2265.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6886841', '6887842', '.', '-', '.', 'gene_id \"chr37.g2266\"; transcript_id \"chr37.g2266.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6878242', '6878728', '.', '+', '.', 'gene_id \"chr37.g2270\"; transcript_id \"chr37.g2270.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6878812', '6878992', '.', '-', '.', 'gene_id \"chr37.g2271\"; transcript_id \"chr37.g2271.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6883029', '6883156', '.', '+', '.', 'gene_id \"chr37.g2275\"; transcript_id \"chr37.g2275.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6883337', '6883532', '.', '-', '.', 'gene_id \"chr37.g2276\"; transcript_id \"chr37.g2276.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6883885', '6884114', '.', '+', '.', 'gene_id \"chr37.g2277\"; transcript_id \"chr37.g2277.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6887907', '6888401', '.', '-', '.', 'gene_id \"chr37.g2281\"; transcript_id \"chr37.g2281.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6888402', '6888607', '.', '+', '.', 'gene_id \"chr37.g2283\"; transcript_id \"chr37.g2283.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6888443', '6888607', '.', '+', '.', 'gene_id \"chr37.g2283\"; transcript_id \"chr37.g2283.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6888693', '6888905', '.', '+', '.', 'gene_id \"chr37.g2284\"; transcript_id \"chr37.g2284.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6889150', '6889428', '.', '-', '.', 'gene_id \"chr37.g2285\"; transcript_id \"chr37.g2285.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6892365', '6892590', '.', '-', '.', 'gene_id \"chr37.g2287\"; transcript_id \"chr37.g2287.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6898590', '6898761', '.', '+', '.', 'gene_id \"chr37.g2289\"; transcript_id \"chr37.g2289.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6899840', '6900011', '.', '-', '.', 'gene_id \"chr37.g2290\"; transcript_id \"chr37.g2290.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6906640', '6906814', '.', '-', '.', 'gene_id \"chr37.g2290\"; transcript_id \"chr37.g2290.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6904689', '6905009', '.', '+', '.', 'gene_id \"chr37.g2291\"; transcript_id \"chr37.g2291.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6904839', '6905002', '.', '+', '.', 'gene_id \"chr37.g2291\"; transcript_id \"chr37.g2291.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6906856', '6907014', '.', '-', '.', 'gene_id \"chr37.g2292\"; transcript_id \"chr37.g2292.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6909511', '6909662', '.', '+', '.', 'gene_id \"chr37.g2294\"; transcript_id \"chr37.g2294.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6905496', '6905776', '.', '+', '.', 'gene_id \"chr37.g2297\"; transcript_id \"chr37.g2297.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6901229', '6901608', '.', '+', '.', 'gene_id \"chr37.g2298\"; transcript_id \"chr37.g2298.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6899773', '6899967', '.', '+', '.', 'gene_id \"chr37.g2299\"; transcript_id \"chr37.g2299.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6900632', '6901095', '.', '+', '.', 'gene_id \"chr37.g2300\"; transcript_id \"chr37.g2300.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6907294', '6907652', '.', '+', '.', 'gene_id \"chr37.g2303\"; transcript_id \"chr37.g2303.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6911880', '6912370', '.', '-', '.', 'gene_id \"chr37.g2306\"; transcript_id \"chr37.g2306.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6912432', '6913023', '.', '-', '.', 'gene_id \"chr37.g2307\"; transcript_id \"chr37.g2307.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6914712', '6914874', '.', '+', '.', 'gene_id \"chr37.g2308\"; transcript_id \"chr37.g2308.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6913905', '6914731', '.', '+', '.', 'gene_id \"chr37.g2308\"; transcript_id \"chr37.g2308.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6913702', '6913927', '.', '-', '.', 'gene_id \"chr37.g2309\"; transcript_id \"chr37.g2309.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28443726', '28443767', '.', '-', '.', 'gene_id \"chr23.g10650\"; transcript_id \"chr23.g10650.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6923989', '6924084', '.', '-', '.', 'gene_id \"chr37.g2316\"; transcript_id \"chr37.g2316.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6948045', '6948952', '.', '+', '.', 'gene_id \"chr37.g2319\"; transcript_id \"chr37.g2319.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6928244', '6928643', '.', '-', '.', 'gene_id \"chr37.g2320\"; transcript_id \"chr37.g2320.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6950293', '6950626', '.', '-', '.', 'gene_id \"chr37.g2322\"; transcript_id \"chr37.g2322.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6949498', '6950245', '.', '-', '.', 'gene_id \"chr37.g2323\"; transcript_id \"chr37.g2323.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6948943', '6949460', '.', '-', '.', 'gene_id \"chr37.g2324\"; transcript_id \"chr37.g2324.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6927992', '6928253', '.', '-', '.', 'gene_id \"chr37.g2326\"; transcript_id \"chr37.g2326.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6927288', '6927960', '.', '-', '.', 'gene_id \"chr37.g2329\"; transcript_id \"chr37.g2329.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6929023', '6929388', '.', '+', '.', 'gene_id \"chr37.g2330\"; transcript_id \"chr37.g2330.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6930024', '6930403', '.', '+', '.', 'gene_id \"chr37.g2331\"; transcript_id \"chr37.g2331.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6931094', '6931292', '.', '+', '.', 'gene_id \"chr37.g2333\"; transcript_id \"chr37.g2333.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6933092', '6933288', '.', '+', '.', 'gene_id \"chr37.g2337\"; transcript_id \"chr37.g2337.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6935278', '6935713', '.', '+', '.', 'gene_id \"chr37.g2340\"; transcript_id \"chr37.g2340.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6939174', '6939613', '.', '+', '.', 'gene_id \"chr37.g2343\"; transcript_id \"chr37.g2343.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6939729', '6939964', '.', '+', '.', 'gene_id \"chr37.g2344\"; transcript_id \"chr37.g2344.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6939961', '6940138', '.', '-', '.', 'gene_id \"chr37.g2345\"; transcript_id \"chr37.g2345.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6940160', '6940264', '.', '-', '.', 'gene_id \"chr37.g2346\"; transcript_id \"chr37.g2346.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6940242', '6940469', '.', '-', '.', 'gene_id \"chr37.g2346\"; transcript_id \"chr37.g2346.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6941253', '6941460', '.', '-', '.', 'gene_id \"chr37.g2348\"; transcript_id \"chr37.g2348.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6942812', '6943035', '.', '-', '.', 'gene_id \"chr37.g2350\"; transcript_id \"chr37.g2350.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6946342', '6946913', '.', '+', '.', 'gene_id \"chr37.g2353\"; transcript_id \"chr37.g2353.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6954049', '6954550', '.', '+', '.', 'gene_id \"chr37.g2356\"; transcript_id \"chr37.g2356.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6955608', '6955717', '.', '-', '.', 'gene_id \"chr37.g2357\"; transcript_id \"chr37.g2357.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6957344', '6957773', '.', '+', '.', 'gene_id \"chr37.g2359\"; transcript_id \"chr37.g2359.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6956729', '6957354', '.', '-', '.', 'gene_id \"chr37.g2360\"; transcript_id \"chr37.g2360.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28469273', '28469457', '.', '+', '.', 'gene_id \"chr23.g10690\"; transcript_id \"chr23.g10690.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6958865', '6959264', '.', '+', '.', 'gene_id \"chr37.g2363\"; transcript_id \"chr37.g2363.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28469968', '28470132', '.', '-', '.', 'gene_id \"chr23.g10692\"; transcript_id \"chr23.g10692.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7028776', '7029617', '.', '+', '.', 'gene_id \"chr37.g2366\"; transcript_id \"chr37.g2366.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6965500', '6965935', '.', '+', '.', 'gene_id \"chr37.g2368\"; transcript_id \"chr37.g2368.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6966107', '6966257', '.', '-', '.', 'gene_id \"chr37.g2370\"; transcript_id \"chr37.g2370.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6966311', '6966936', '.', '-', '.', 'gene_id \"chr37.g2371\"; transcript_id \"chr37.g2371.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6967180', '6967440', '.', '-', '.', 'gene_id \"chr37.g2372\"; transcript_id \"chr37.g2372.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6969185', '6969506', '.', '+', '.', 'gene_id \"chr37.g2376\"; transcript_id \"chr37.g2376.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6971989', '6972148', '.', '+', '.', 'gene_id \"chr37.g2381\"; transcript_id \"chr37.g2381.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6976121', '6976313', '.', '+', '.', 'gene_id \"chr37.g2382\"; transcript_id \"chr37.g2382.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6976549', '6976800', '.', '+', '.', 'gene_id \"chr37.g2383\"; transcript_id \"chr37.g2383.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6987754', '6987930', '.', '+', '.', 'gene_id \"chr37.g2386\"; transcript_id \"chr37.g2386.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6988004', '6988210', '.', '+', '.', 'gene_id \"chr37.g2387\"; transcript_id \"chr37.g2387.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6992128', '6992164', '.', '+', '.', 'gene_id \"chr37.g2390\"; transcript_id \"chr37.g2390.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '6992215', '6992567', '.', '-', '.', 'gene_id \"chr37.g2391\"; transcript_id \"chr37.g2391.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7000159', '7000428', '.', '-', '.', 'gene_id \"chr37.g2392\"; transcript_id \"chr37.g2392.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7001563', '7001769', '.', '-', '.', 'gene_id \"chr37.g2393\"; transcript_id \"chr37.g2393.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7003435', '7003590', '.', '+', '.', 'gene_id \"chr37.g2395\"; transcript_id \"chr37.g2395.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7008661', '7008900', '.', '+', '.', 'gene_id \"chr37.g2399\"; transcript_id \"chr37.g2399.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7010913', '7011065', '.', '-', '.', 'gene_id \"chr37.g2401\"; transcript_id \"chr37.g2401.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7011068', '7011311', '.', '-', '.', 'gene_id \"chr37.g2402\"; transcript_id \"chr37.g2402.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7018914', '7019393', '.', '+', '.', 'gene_id \"chr37.g2405\"; transcript_id \"chr37.g2405.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7022305', '7022467', '.', '-', '.', 'gene_id \"chr37.g2408\"; transcript_id \"chr37.g2408.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7023330', '7023516', '.', '-', '.', 'gene_id \"chr37.g2411\"; transcript_id \"chr37.g2411.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7023356', '7023516', '.', '+', '.', 'gene_id \"chr37.g2412\"; transcript_id \"chr37.g2412.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7023666', '7023904', '.', '-', '.', 'gene_id \"chr37.g2413\"; transcript_id \"chr37.g2413.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7023690', '7023904', '.', '-', '.', 'gene_id \"chr37.g2413\"; transcript_id \"chr37.g2413.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7024407', '7024619', '.', '-', '.', 'gene_id \"chr37.g2414\"; transcript_id \"chr37.g2414.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7026485', '7026649', '.', '+', '.', 'gene_id \"chr37.g2415\"; transcript_id \"chr37.g2415.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7027216', '7027577', '.', '-', '.', 'gene_id \"chr37.g2417\"; transcript_id \"chr37.g2417.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7027703', '7027862', '.', '+', '.', 'gene_id \"chr37.g2418\"; transcript_id \"chr37.g2418.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7034487', '7034878', '.', '+', '.', 'gene_id \"chr37.g2422\"; transcript_id \"chr37.g2422.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7036090', '7036493', '.', '+', '.', 'gene_id \"chr37.g2426\"; transcript_id \"chr37.g2426.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7037184', '7037408', '.', '+', '.', 'gene_id \"chr37.g2427\"; transcript_id \"chr37.g2427.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7037391', '7037469', '.', '+', '.', 'gene_id \"chr37.g2427\"; transcript_id \"chr37.g2427.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7036682', '7036972', '.', '-', '.', 'gene_id \"chr37.g2428\"; transcript_id \"chr37.g2428.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7056887', '7058625', '.', '+', '.', 'gene_id \"chr37.g2433\"; transcript_id \"chr37.g2433.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7051466', '7051739', '.', '+', '.', 'gene_id \"chr37.g2435\"; transcript_id \"chr37.g2435.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '28695412', '28695456', '.', '-', '.', 'gene_id \"chr23.g10760\"; transcript_id \"chr23.g10760.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7054053', '7054374', '.', '+', '.', 'gene_id \"chr37.g2437\"; transcript_id \"chr37.g2437.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7054861', '7055027', '.', '+', '.', 'gene_id \"chr37.g2441\"; transcript_id \"chr37.g2441.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7060518', '7060754', '.', '+', '.', 'gene_id \"chr37.g2443\"; transcript_id \"chr37.g2443.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7060818', '7060982', '.', '-', '.', 'gene_id \"chr37.g2444\"; transcript_id \"chr37.g2444.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7061015', '7061575', '.', '+', '.', 'gene_id \"chr37.g2445\"; transcript_id \"chr37.g2445.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7063109', '7063290', '.', '-', '.', 'gene_id \"chr37.g2447\"; transcript_id \"chr37.g2447.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7066384', '7066577', '.', '+', '.', 'gene_id \"chr37.g2450\"; transcript_id \"chr37.g2450.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7070765', '7070936', '.', '+', '.', 'gene_id \"chr37.g2451\"; transcript_id \"chr37.g2451.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7071004', '7071230', '.', '-', '.', 'gene_id \"chr37.g2452\"; transcript_id \"chr37.g2452.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7071573', '7071765', '.', '+', '.', 'gene_id \"chr37.g2454\"; transcript_id \"chr37.g2454.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7076829', '7077579', '.', '+', '.', 'gene_id \"chr37.g2459\"; transcript_id \"chr37.g2459.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7084155', '7084306', '.', '+', '.', 'gene_id \"chr37.g2462\"; transcript_id \"chr37.g2462.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7080900', '7081088', '.', '-', '.', 'gene_id \"chr37.g2464\"; transcript_id \"chr37.g2464.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7081726', '7081978', '.', '+', '.', 'gene_id \"chr37.g2465\"; transcript_id \"chr37.g2465.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7118984', '7119143', '.', '-', '.', 'gene_id \"chr37.g2474\"; transcript_id \"chr37.g2474.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7124175', '7124631', '.', '-', '.', 'gene_id \"chr37.g2478\"; transcript_id \"chr37.g2478.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7125390', '7125626', '.', '-', '.', 'gene_id \"chr37.g2480\"; transcript_id \"chr37.g2480.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7125675', '7126012', '.', '-', '.', 'gene_id \"chr37.g2481\"; transcript_id \"chr37.g2481.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7127375', '7127615', '.', '+', '.', 'gene_id \"chr37.g2484\"; transcript_id \"chr37.g2484.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7133569', '7133765', '.', '+', '.', 'gene_id \"chr37.g2487\"; transcript_id \"chr37.g2487.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7134884', '7135154', '.', '-', '.', 'gene_id \"chr37.g2490\"; transcript_id \"chr37.g2490.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7135334', '7135564', '.', '-', '.', 'gene_id \"chr37.g2491\"; transcript_id \"chr37.g2491.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7135603', '7135806', '.', '+', '.', 'gene_id \"chr37.g2492\"; transcript_id \"chr37.g2492.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7135849', '7137411', '.', '-', '.', 'gene_id \"chr37.g2494\"; transcript_id \"chr37.g2494.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7137813', '7138198', '.', '+', '.', 'gene_id \"chr37.g2496\"; transcript_id \"chr37.g2496.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7143063', '7143261', '.', '+', '.', 'gene_id \"chr37.g2500\"; transcript_id \"chr37.g2500.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7143472', '7143694', '.', '+', '.', 'gene_id \"chr37.g2501\"; transcript_id \"chr37.g2501.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7145710', '7145978', '.', '+', '.', 'gene_id \"chr37.g2502\"; transcript_id \"chr37.g2502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7158451', '7158655', '.', '+', '.', 'gene_id \"chr37.g2505\"; transcript_id \"chr37.g2505.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7165147', '7165824', '.', '+', '.', 'gene_id \"chr37.g2510\"; transcript_id \"chr37.g2510.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7168843', '7169374', '.', '+', '.', 'gene_id \"chr37.g2511\"; transcript_id \"chr37.g2511.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7169946', '7170105', '.', '-', '.', 'gene_id \"chr37.g2512\"; transcript_id \"chr37.g2512.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7171004', '7171154', '.', '+', '.', 'gene_id \"chr37.g2513\"; transcript_id \"chr37.g2513.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7173241', '7173426', '.', '+', '.', 'gene_id \"chr37.g2515\"; transcript_id \"chr37.g2515.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7173845', '7174358', '.', '-', '.', 'gene_id \"chr37.g2516\"; transcript_id \"chr37.g2516.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7174674', '7175641', '.', '-', '.', 'gene_id \"chr37.g2520\"; transcript_id \"chr37.g2520.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7175796', '7176135', '.', '-', '.', 'gene_id \"chr37.g2523\"; transcript_id \"chr37.g2523.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7176181', '7177160', '.', '-', '.', 'gene_id \"chr37.g2524\"; transcript_id \"chr37.g2524.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7177484', '7178259', '.', '-', '.', 'gene_id \"chr37.g2527\"; transcript_id \"chr37.g2527.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7178638', '7179023', '.', '-', '.', 'gene_id \"chr37.g2529\"; transcript_id \"chr37.g2529.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7178507', '7178660', '.', '+', '.', 'gene_id \"chr37.g2530\"; transcript_id \"chr37.g2530.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7180814', '7181225', '.', '-', '.', 'gene_id \"chr37.g2531\"; transcript_id \"chr37.g2531.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7187701', '7187854', '.', '+', '.', 'gene_id \"chr37.g2535\"; transcript_id \"chr37.g2535.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7195227', '7195388', '.', '+', '.', 'gene_id \"chr37.g2538\"; transcript_id \"chr37.g2538.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7200783', '7200942', '.', '-', '.', 'gene_id \"chr37.g2540\"; transcript_id \"chr37.g2540.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7201233', '7201397', '.', '-', '.', 'gene_id \"chr37.g2541\"; transcript_id \"chr37.g2541.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7202506', '7202770', '.', '+', '.', 'gene_id \"chr37.g2542\"; transcript_id \"chr37.g2542.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7202798', '7203049', '.', '-', '.', 'gene_id \"chr37.g2543\"; transcript_id \"chr37.g2543.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7204933', '7205185', '.', '-', '.', 'gene_id \"chr37.g2544\"; transcript_id \"chr37.g2544.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7213597', '7213763', '.', '-', '.', 'gene_id \"chr37.g2546\"; transcript_id \"chr37.g2546.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7214405', '7214615', '.', '-', '.', 'gene_id \"chr37.g2549\"; transcript_id \"chr37.g2549.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7219219', '7219527', '.', '-', '.', 'gene_id \"chr37.g2552\"; transcript_id \"chr37.g2552.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7221263', '7221659', '.', '-', '.', 'gene_id \"chr37.g2554\"; transcript_id \"chr37.g2554.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7230122', '7230272', '.', '-', '.', 'gene_id \"chr37.g2557\"; transcript_id \"chr37.g2557.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7243472', '7243760', '.', '+', '.', 'gene_id \"chr37.g2560\"; transcript_id \"chr37.g2560.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29205242', '29205286', '.', '+', '.', 'gene_id \"chr23.g10900\"; transcript_id \"chr23.g10900.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7244078', '7244338', '.', '-', '.', 'gene_id \"chr37.g2562\"; transcript_id \"chr37.g2562.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7247661', '7247790', '.', '+', '.', 'gene_id \"chr37.g2565\"; transcript_id \"chr37.g2565.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7250637', '7250787', '.', '+', '.', 'gene_id \"chr37.g2566\"; transcript_id \"chr37.g2566.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7253509', '7253729', '.', '+', '.', 'gene_id \"chr37.g2567\"; transcript_id \"chr37.g2567.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7255188', '7255388', '.', '-', '.', 'gene_id \"chr37.g2569\"; transcript_id \"chr37.g2569.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7255710', '7255914', '.', '-', '.', 'gene_id \"chr37.g2572\"; transcript_id \"chr37.g2572.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7256591', '7257317', '.', '+', '.', 'gene_id \"chr37.g2573\"; transcript_id \"chr37.g2573.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7257792', '7258013', '.', '+', '.', 'gene_id \"chr37.g2575\"; transcript_id \"chr37.g2575.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7260122', '7260338', '.', '+', '.', 'gene_id \"chr37.g2577\"; transcript_id \"chr37.g2577.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7260343', '7260500', '.', '+', '.', 'gene_id \"chr37.g2578\"; transcript_id \"chr37.g2578.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7261416', '7261800', '.', '-', '.', 'gene_id \"chr37.g2580\"; transcript_id \"chr37.g2580.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7263953', '7264118', '.', '+', '.', 'gene_id \"chr37.g2583\"; transcript_id \"chr37.g2583.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7267635', '7267813', '.', '+', '.', 'gene_id \"chr37.g2584\"; transcript_id \"chr37.g2584.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7276926', '7277341', '.', '+', '.', 'gene_id \"chr37.g2587\"; transcript_id \"chr37.g2587.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7279431', '7279590', '.', '-', '.', 'gene_id \"chr37.g2591\"; transcript_id \"chr37.g2591.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7279880', '7280312', '.', '+', '.', 'gene_id \"chr37.g2592\"; transcript_id \"chr37.g2592.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7283453', '7283990', '.', '+', '.', 'gene_id \"chr37.g2594\"; transcript_id \"chr37.g2594.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7307737', '7308002', '.', '-', '.', 'gene_id \"chr37.g2600\"; transcript_id \"chr37.g2600.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7308373', '7308532', '.', '+', '.', 'gene_id \"chr37.g2602\"; transcript_id \"chr37.g2602.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7311785', '7312002', '.', '+', '.', 'gene_id \"chr37.g2604\"; transcript_id \"chr37.g2604.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7312667', '7312916', '.', '+', '.', 'gene_id \"chr37.g2605\"; transcript_id \"chr37.g2605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7322334', '7322516', '.', '+', '.', 'gene_id \"chr37.g2611\"; transcript_id \"chr37.g2611.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7324210', '7324643', '.', '+', '.', 'gene_id \"chr37.g2612\"; transcript_id \"chr37.g2612.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7325330', '7325574', '.', '+', '.', 'gene_id \"chr37.g2614\"; transcript_id \"chr37.g2614.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7327132', '7327201', '.', '+', '.', 'gene_id \"chr37.g2615\"; transcript_id \"chr37.g2615.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7327182', '7327404', '.', '-', '.', 'gene_id \"chr37.g2616\"; transcript_id \"chr37.g2616.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7340585', '7341138', '.', '+', '.', 'gene_id \"chr37.g2618\"; transcript_id \"chr37.g2618.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7340590', '7341138', '.', '+', '.', 'gene_id \"chr37.g2618\"; transcript_id \"chr37.g2618.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7339378', '7340335', '.', '+', '.', 'gene_id \"chr37.g2619\"; transcript_id \"chr37.g2619.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7341689', '7341925', '.', '+', '.', 'gene_id \"chr37.g2622\"; transcript_id \"chr37.g2622.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7342277', '7342569', '.', '-', '.', 'gene_id \"chr37.g2624\"; transcript_id \"chr37.g2624.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7363031', '7363202', '.', '+', '.', 'gene_id \"chr37.g2627\"; transcript_id \"chr37.g2627.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7362840', '7363101', '.', '+', '.', 'gene_id \"chr37.g2627\"; transcript_id \"chr37.g2627.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7451232', '7453282', '.', '-', '.', 'gene_id \"chr37.g2631\"; transcript_id \"chr37.g2631.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7386787', '7387022', '.', '+', '.', 'gene_id \"chr37.g2632\"; transcript_id \"chr37.g2632.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7388115', '7388324', '.', '-', '.', 'gene_id \"chr37.g2635\"; transcript_id \"chr37.g2635.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7388420', '7388764', '.', '-', '.', 'gene_id \"chr37.g2636\"; transcript_id \"chr37.g2636.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7389520', '7389772', '.', '+', '.', 'gene_id \"chr37.g2638\"; transcript_id \"chr37.g2638.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7396147', '7396308', '.', '+', '.', 'gene_id \"chr37.g2640\"; transcript_id \"chr37.g2640.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7445530', '7445683', '.', '-', '.', 'gene_id \"chr37.g2644\"; transcript_id \"chr37.g2644.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7445813', '7445995', '.', '-', '.', 'gene_id \"chr37.g2645\"; transcript_id \"chr37.g2645.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7447828', '7447981', '.', '-', '.', 'gene_id \"chr37.g2646\"; transcript_id \"chr37.g2646.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29614795', '29614877', '.', '-', '.', 'gene_id \"chr23.g10992\"; transcript_id \"chr23.g10992.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29614796', '29614877', '.', '-', '.', 'gene_id \"chr23.g10992\"; transcript_id \"chr23.g10992.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29649863', '29650167', '.', '+', '.', 'gene_id \"chr23.g10993\"; transcript_id \"chr23.g10993.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29595581', '29595853', '.', '-', '.', 'gene_id \"chr23.g10995\"; transcript_id \"chr23.g10995.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29710956', '29711320', '.', '-', '.', 'gene_id \"chr23.g11007\"; transcript_id \"chr23.g11007.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29711299', '29712075', '.', '+', '.', 'gene_id \"chr23.g11008\"; transcript_id \"chr23.g11008.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7449315', '7449591', '.', '-', '.', 'gene_id \"chr37.g2649\"; transcript_id \"chr37.g2649.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7453315', '7453929', '.', '+', '.', 'gene_id \"chr37.g2653\"; transcript_id \"chr37.g2653.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7455617', '7455932', '.', '-', '.', 'gene_id \"chr37.g2655\"; transcript_id \"chr37.g2655.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7459031', '7459462', '.', '+', '.', 'gene_id \"chr37.g2659\"; transcript_id \"chr37.g2659.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7462778', '7462946', '.', '+', '.', 'gene_id \"chr37.g2661\"; transcript_id \"chr37.g2661.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7610364', '7610515', '.', '+', '.', 'gene_id \"chr37.g2670\"; transcript_id \"chr37.g2670.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7610562', '7610756', '.', '+', '.', 'gene_id \"chr37.g2671\"; transcript_id \"chr37.g2671.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7638663', '7639067', '.', '+', '.', 'gene_id \"chr37.g2672\"; transcript_id \"chr37.g2672.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '7849588', '7849622', '.', '+', '.', 'gene_id \"chr37.g2684\"; transcript_id \"chr37.g2684.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8020962', '8021028', '.', '+', '.', 'gene_id \"chr37.g2703\"; transcript_id \"chr37.g2703.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8036412', '8036515', '.', '+', '.', 'gene_id \"chr37.g2705\"; transcript_id \"chr37.g2705.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8050877', '8051042', '.', '-', '.', 'gene_id \"chr37.g2707\"; transcript_id \"chr37.g2707.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8089057', '8089244', '.', '+', '.', 'gene_id \"chr37.g2712\"; transcript_id \"chr37.g2712.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8092192', '8092437', '.', '+', '.', 'gene_id \"chr37.g2713\"; transcript_id \"chr37.g2713.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8091596', '8091772', '.', '-', '.', 'gene_id \"chr37.g2714\"; transcript_id \"chr37.g2714.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8091792', '8091970', '.', '-', '.', 'gene_id \"chr37.g2715\"; transcript_id \"chr37.g2715.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8092790', '8092937', '.', '-', '.', 'gene_id \"chr37.g2717\"; transcript_id \"chr37.g2717.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29728536', '29728712', '.', '-', '.', 'gene_id \"chr23.g11084\"; transcript_id \"chr23.g11084.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29754384', '29754609', '.', '+', '.', 'gene_id \"chr23.g11114\"; transcript_id \"chr23.g11114.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8592834', '8592877', '.', '-', '.', 'gene_id \"chr37.g2762\"; transcript_id \"chr37.g2762.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '8991200', '8991236', '.', '-', '.', 'gene_id \"chr37.g2850\"; transcript_id \"chr37.g2850.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9002088', '9002259', '.', '+', '.', 'gene_id \"chr37.g2860\"; transcript_id \"chr37.g2860.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '30147738', '30147907', '.', '-', '.', 'gene_id \"chr23.g11271\"; transcript_id \"chr23.g11271.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '30147738', '30147911', '.', '-', '.', 'gene_id \"chr23.g11271\"; transcript_id \"chr23.g11271.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9286968', '9287147', '.', '-', '.', 'gene_id \"chr37.g2908\"; transcript_id \"chr37.g2908.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9292105', '9292429', '.', '+', '.', 'gene_id \"chr37.g2911\"; transcript_id \"chr37.g2911.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9292105', '9292429', '.', '+', '.', 'gene_id \"chr37.g2911\"; transcript_id \"chr37.g2911.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9447506', '9448175', '.', '+', '.', 'gene_id \"chr37.g2911\"; transcript_id \"chr37.g2911.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9448351', '9448953', '.', '+', '.', 'gene_id \"chr37.g2915\"; transcript_id \"chr37.g2915.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9302004', '9302177', '.', '-', '.', 'gene_id \"chr37.g2922\"; transcript_id \"chr37.g2922.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9302528', '9302726', '.', '-', '.', 'gene_id \"chr37.g2934\"; transcript_id \"chr37.g2934.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9310133', '9310755', '.', '+', '.', 'gene_id \"chr37.g2939\"; transcript_id \"chr37.g2939.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9310133', '9310500', '.', '+', '.', 'gene_id \"chr37.g2939\"; transcript_id \"chr37.g2939.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9323630', '9323784', '.', '-', '.', 'gene_id \"chr37.g2952\"; transcript_id \"chr37.g2952.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9325921', '9326095', '.', '-', '.', 'gene_id \"chr37.g2954\"; transcript_id \"chr37.g2954.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9326820', '9327310', '.', '+', '.', 'gene_id \"chr37.g2955\"; transcript_id \"chr37.g2955.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9327449', '9327824', '.', '+', '.', 'gene_id \"chr37.g2956\"; transcript_id \"chr37.g2956.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9330810', '9330869', '.', '+', '.', 'gene_id \"chr37.g2957\"; transcript_id \"chr37.g2957.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '30407730', '30407888', '.', '+', '.', 'gene_id \"chr23.g11321\"; transcript_id \"chr23.g11321.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9337401', '9337559', '.', '+', '.', 'gene_id \"chr37.g2963\"; transcript_id \"chr37.g2963.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9353340', '9353689', '.', '-', '.', 'gene_id \"chr37.g2974\"; transcript_id \"chr37.g2974.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9355763', '9355992', '.', '-', '.', 'gene_id \"chr37.g2975\"; transcript_id \"chr37.g2975.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9361397', '9362015', '.', '+', '.', 'gene_id \"chr37.g2986\"; transcript_id \"chr37.g2986.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9361397', '9361760', '.', '+', '.', 'gene_id \"chr37.g2986\"; transcript_id \"chr37.g2986.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9387129', '9387401', '.', '+', '.', 'gene_id \"chr37.g2995\"; transcript_id \"chr37.g2995.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9386585', '9387129', '.', '+', '.', 'gene_id \"chr37.g2998\"; transcript_id \"chr37.g2998.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9400104', '9400541', '.', '+', '.', 'gene_id \"chr37.g3001\"; transcript_id \"chr37.g3001.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9405842', '9405936', '.', '-', '.', 'gene_id \"chr37.g3008\"; transcript_id \"chr37.g3008.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9405999', '9406274', '.', '+', '.', 'gene_id \"chr37.g3009\"; transcript_id \"chr37.g3009.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '30613297', '30613381', '.', '+', '.', 'gene_id \"chr23.g11385\"; transcript_id \"chr23.g11385.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9444385', '9444916', '.', '+', '.', 'gene_id \"chr37.g3048\"; transcript_id \"chr37.g3048.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9446160', '9447281', '.', '-', '.', 'gene_id \"chr37.g3049\"; transcript_id \"chr37.g3049.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9456328', '9456836', '.', '+', '.', 'gene_id \"chr37.g3052\"; transcript_id \"chr37.g3052.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9457508', '9458080', '.', '+', '.', 'gene_id \"chr37.g3054\"; transcript_id \"chr37.g3054.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9458087', '9458451', '.', '-', '.', 'gene_id \"chr37.g3055\"; transcript_id \"chr37.g3055.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9458490', '9459875', '.', '-', '.', 'gene_id \"chr37.g3056\"; transcript_id \"chr37.g3056.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9459955', '9460209', '.', '-', '.', 'gene_id \"chr37.g3057\"; transcript_id \"chr37.g3057.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9460560', '9460819', '.', '-', '.', 'gene_id \"chr37.g3059\"; transcript_id \"chr37.g3059.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9460826', '9461008', '.', '+', '.', 'gene_id \"chr37.g3060\"; transcript_id \"chr37.g3060.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9461002', '9461499', '.', '-', '.', 'gene_id \"chr37.g3061\"; transcript_id \"chr37.g3061.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9461986', '9462034', '.', '-', '.', 'gene_id \"chr37.g3063\"; transcript_id \"chr37.g3063.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9462271', '9462401', '.', '-', '.', 'gene_id \"chr37.g3065\"; transcript_id \"chr37.g3065.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9462454', '9462946', '.', '-', '.', 'gene_id \"chr37.g3067\"; transcript_id \"chr37.g3067.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9462454', '9462946', '.', '-', '.', 'gene_id \"chr37.g3067\"; transcript_id \"chr37.g3067.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9483503', '9483737', '.', '-', '.', 'gene_id \"chr37.g3067\"; transcript_id \"chr37.g3067.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9464103', '9467420', '.', '-', '.', 'gene_id \"chr37.g3069\"; transcript_id \"chr37.g3069.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9474011', '9474388', '.', '-', '.', 'gene_id \"chr37.g3077\"; transcript_id \"chr37.g3077.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9482081', '9482849', '.', '-', '.', 'gene_id \"chr37.g3085\"; transcript_id \"chr37.g3085.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9486897', '9487113', '.', '+', '.', 'gene_id \"chr37.g3090\"; transcript_id \"chr37.g3090.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9487477', '9488246', '.', '-', '.', 'gene_id \"chr37.g3092\"; transcript_id \"chr37.g3092.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9488254', '9488588', '.', '+', '.', 'gene_id \"chr37.g3093\"; transcript_id \"chr37.g3093.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9492641', '9492855', '.', '-', '.', 'gene_id \"chr37.g3100\"; transcript_id \"chr37.g3100.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9537749', '9538502', '.', '+', '.', 'gene_id \"chr37.g3109\"; transcript_id \"chr37.g3109.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9622176', '9622405', '.', '+', '.', 'gene_id \"chr37.g3135\"; transcript_id \"chr37.g3135.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9612862', '9613015', '.', '+', '.', 'gene_id \"chr37.g3137\"; transcript_id \"chr37.g3137.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9620678', '9622170', '.', '-', '.', 'gene_id \"chr37.g3138\"; transcript_id \"chr37.g3138.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9596037', '9596192', '.', '-', '.', 'gene_id \"chr37.g3142\"; transcript_id \"chr37.g3142.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9608860', '9609131', '.', '+', '.', 'gene_id \"chr37.g3145\"; transcript_id \"chr37.g3145.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9609884', '9610425', '.', '-', '.', 'gene_id \"chr37.g3146\"; transcript_id \"chr37.g3146.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9623428', '9623603', '.', '-', '.', 'gene_id \"chr37.g3156\"; transcript_id \"chr37.g3156.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9623780', '9624202', '.', '-', '.', 'gene_id \"chr37.g3158\"; transcript_id \"chr37.g3158.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9623780', '9623813', '.', '+', '.', 'gene_id \"chr37.g3159\"; transcript_id \"chr37.g3159.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9624658', '9625396', '.', '+', '.', 'gene_id \"chr37.g3161\"; transcript_id \"chr37.g3161.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9626971', '9627163', '.', '-', '.', 'gene_id \"chr37.g3163\"; transcript_id \"chr37.g3163.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9630200', '9630590', '.', '+', '.', 'gene_id \"chr37.g3165\"; transcript_id \"chr37.g3165.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9669278', '9669446', '.', '+', '.', 'gene_id \"chr37.g3170\"; transcript_id \"chr37.g3170.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9669854', '9670008', '.', '+', '.', 'gene_id \"chr37.g3170\"; transcript_id \"chr37.g3170.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9734167', '9734320', '.', '+', '.', 'gene_id \"chr37.g3185\"; transcript_id \"chr37.g3185.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9734626', '9734977', '.', '+', '.', 'gene_id \"chr37.g3187\"; transcript_id \"chr37.g3187.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9778677', '9781118', '.', '-', '.', 'gene_id \"chr37.g3188\"; transcript_id \"chr37.g3188.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9778677', '9779088', '.', '-', '.', 'gene_id \"chr37.g3188\"; transcript_id \"chr37.g3188.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9752934', '9753252', '.', '+', '.', 'gene_id \"chr37.g3189\"; transcript_id \"chr37.g3189.i11\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9735985', '9736450', '.', '+', '.', 'gene_id \"chr37.g3189\"; transcript_id \"chr37.g3189.i15\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9741816', '9742243', '.', '-', '.', 'gene_id \"chr37.g3190\"; transcript_id \"chr37.g3190.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9741803', '9742243', '.', '-', '.', 'gene_id \"chr37.g3190\"; transcript_id \"chr37.g3190.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9770216', '9770557', '.', '+', '.', 'gene_id \"chr37.g3197\"; transcript_id \"chr37.g3197.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9741109', '9741706', '.', '+', '.', 'gene_id \"chr37.g3200\"; transcript_id \"chr37.g3200.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9767514', '9767696', '.', '-', '.', 'gene_id \"chr37.g3208\"; transcript_id \"chr37.g3208.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9773255', '9773475', '.', '-', '.', 'gene_id \"chr37.g3209\"; transcript_id \"chr37.g3209.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9774228', '9774526', '.', '+', '.', 'gene_id \"chr37.g3211\"; transcript_id \"chr37.g3211.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9774614', '9774865', '.', '-', '.', 'gene_id \"chr37.g3212\"; transcript_id \"chr37.g3212.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9784280', '9784702', '.', '+', '.', 'gene_id \"chr37.g3218\"; transcript_id \"chr37.g3218.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9785727', '9785834', '.', '-', '.', 'gene_id \"chr37.g3220\"; transcript_id \"chr37.g3220.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9856523', '9856701', '.', '+', '.', 'gene_id \"chr37.g3229\"; transcript_id \"chr37.g3229.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9856253', '9856533', '.', '+', '.', 'gene_id \"chr37.g3232\"; transcript_id \"chr37.g3232.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9854376', '9854619', '.', '+', '.', 'gene_id \"chr37.g3240\"; transcript_id \"chr37.g3240.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9865492', '9865707', '.', '+', '.', 'gene_id \"chr37.g3245\"; transcript_id \"chr37.g3245.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9868818', '9869021', '.', '+', '.', 'gene_id \"chr37.g3249\"; transcript_id \"chr37.g3249.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9874680', '9874862', '.', '-', '.', 'gene_id \"chr37.g3252\"; transcript_id \"chr37.g3252.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9892840', '9893381', '.', '+', '.', 'gene_id \"chr37.g3262\"; transcript_id \"chr37.g3262.i6\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9901288', '9901878', '.', '-', '.', 'gene_id \"chr37.g3266\"; transcript_id \"chr37.g3266.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9896417', '9896598', '.', '-', '.', 'gene_id \"chr37.g3270\"; transcript_id \"chr37.g3270.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9893948', '9894182', '.', '+', '.', 'gene_id \"chr37.g3274\"; transcript_id \"chr37.g3274.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9896923', '9897414', '.', '+', '.', 'gene_id \"chr37.g3278\"; transcript_id \"chr37.g3278.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9896788', '9896941', '.', '-', '.', 'gene_id \"chr37.g3280\"; transcript_id \"chr37.g3280.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9923968', '9924450', '.', '+', '.', 'gene_id \"chr37.g3289\"; transcript_id \"chr37.g3289.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9943534', '9943899', '.', '-', '.', 'gene_id \"chr37.g3295\"; transcript_id \"chr37.g3295.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9941686', '9941942', '.', '-', '.', 'gene_id \"chr37.g3295\"; transcript_id \"chr37.g3295.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9949944', '9950113', '.', '+', '.', 'gene_id \"chr37.g3296\"; transcript_id \"chr37.g3296.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9942489', '9943422', '.', '+', '.', 'gene_id \"chr37.g3297\"; transcript_id \"chr37.g3297.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9947463', '9947730', '.', '+', '.', 'gene_id \"chr37.g3302\"; transcript_id \"chr37.g3302.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9941480', '9941638', '.', '+', '.', 'gene_id \"chr37.g3304\"; transcript_id \"chr37.g3304.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9945592', '9946110', '.', '-', '.', 'gene_id \"chr37.g3305\"; transcript_id \"chr37.g3305.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9946092', '9946208', '.', '-', '.', 'gene_id \"chr37.g3305\"; transcript_id \"chr37.g3305.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9946246', '9946593', '.', '-', '.', 'gene_id \"chr37.g3307\"; transcript_id \"chr37.g3307.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9950494', '9951040', '.', '+', '.', 'gene_id \"chr37.g3308\"; transcript_id \"chr37.g3308.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9950494', '9951051', '.', '-', '.', 'gene_id \"chr37.g3309\"; transcript_id \"chr37.g3309.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9952609', '9952701', '.', '-', '.', 'gene_id \"chr37.g3312\"; transcript_id \"chr37.g3312.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9959766', '9960195', '.', '+', '.', 'gene_id \"chr37.g3316\"; transcript_id \"chr37.g3316.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9961542', '9961841', '.', '+', '.', 'gene_id \"chr37.g3320\"; transcript_id \"chr37.g3320.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9960913', '9961431', '.', '+', '.', 'gene_id \"chr37.g3323\"; transcript_id \"chr37.g3323.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9963053', '9963268', '.', '-', '.', 'gene_id \"chr37.g3329\"; transcript_id \"chr37.g3329.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9995111', '9995328', '.', '+', '.', 'gene_id \"chr37.g3337\"; transcript_id \"chr37.g3337.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10008011', '10009139', '.', '+', '.', 'gene_id \"chr37.g3339\"; transcript_id \"chr37.g3339.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10012239', '10012569', '.', '-', '.', 'gene_id \"chr37.g3342\"; transcript_id \"chr37.g3342.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9970330', '9970533', '.', '+', '.', 'gene_id \"chr37.g3346\"; transcript_id \"chr37.g3346.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10022341', '10022680', '.', '+', '.', 'gene_id \"chr37.g3349\"; transcript_id \"chr37.g3349.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10015929', '10016726', '.', '-', '.', 'gene_id \"chr37.g3351\"; transcript_id \"chr37.g3351.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10019392', '10019549', '.', '-', '.', 'gene_id \"chr37.g3352\"; transcript_id \"chr37.g3352.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10022042', '10022354', '.', '+', '.', 'gene_id \"chr37.g3353\"; transcript_id \"chr37.g3353.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9970090', '9970347', '.', '-', '.', 'gene_id \"chr37.g3354\"; transcript_id \"chr37.g3354.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9977696', '9977782', '.', '-', '.', 'gene_id \"chr37.g3360\"; transcript_id \"chr37.g3360.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9977764', '9978224', '.', '+', '.', 'gene_id \"chr37.g3361\"; transcript_id \"chr37.g3361.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9978627', '9978902', '.', '-', '.', 'gene_id \"chr37.g3362\"; transcript_id \"chr37.g3362.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9978981', '9979380', '.', '-', '.', 'gene_id \"chr37.g3364\"; transcript_id \"chr37.g3364.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9979407', '9979535', '.', '+', '.', 'gene_id \"chr37.g3366\"; transcript_id \"chr37.g3366.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9979514', '9979692', '.', '-', '.', 'gene_id \"chr37.g3367\"; transcript_id \"chr37.g3367.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9979673', '9979821', '.', '+', '.', 'gene_id \"chr37.g3368\"; transcript_id \"chr37.g3368.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9979912', '9980204', '.', '+', '.', 'gene_id \"chr37.g3369\"; transcript_id \"chr37.g3369.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '9986063', '9986284', '.', '+', '.', 'gene_id \"chr37.g3379\"; transcript_id \"chr37.g3379.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10006190', '10006663', '.', '+', '.', 'gene_id \"chr37.g3386\"; transcript_id \"chr37.g3386.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10006655', '10007396', '.', '-', '.', 'gene_id \"chr37.g3387\"; transcript_id \"chr37.g3387.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10013423', '10013583', '.', '+', '.', 'gene_id \"chr37.g3388\"; transcript_id \"chr37.g3388.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10018567', '10018931', '.', '-', '.', 'gene_id \"chr37.g3390\"; transcript_id \"chr37.g3390.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10023213', '10023641', '.', '+', '.', 'gene_id \"chr37.g3394\"; transcript_id \"chr37.g3394.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '31217840', '31217997', '.', '+', '.', 'gene_id \"chr23.g11760\"; transcript_id \"chr23.g11760.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10040735', '10041370', '.', '-', '.', 'gene_id \"chr37.g3416\"; transcript_id \"chr37.g3416.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10047316', '10047872', '.', '-', '.', 'gene_id \"chr37.g3422\"; transcript_id \"chr37.g3422.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10048059', '10048347', '.', '-', '.', 'gene_id \"chr37.g3423\"; transcript_id \"chr37.g3423.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10048445', '10048671', '.', '-', '.', 'gene_id \"chr37.g3425\"; transcript_id \"chr37.g3425.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10048813', '10049159', '.', '+', '.', 'gene_id \"chr37.g3427\"; transcript_id \"chr37.g3427.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10129964', '10130649', '.', '+', '.', 'gene_id \"chr37.g3429\"; transcript_id \"chr37.g3429.i5\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10060127', '10062767', '.', '+', '.', 'gene_id \"chr37.g3431\"; transcript_id \"chr37.g3431.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10060135', '10062767', '.', '+', '.', 'gene_id \"chr37.g3431\"; transcript_id \"chr37.g3431.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10061824', '10061995', '.', '+', '.', 'gene_id \"chr37.g3431\"; transcript_id \"chr37.g3431.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10130771', '10131041', '.', '-', '.', 'gene_id \"chr37.g3435\"; transcript_id \"chr37.g3435.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10063074', '10063196', '.', '+', '.', 'gene_id \"chr37.g3436\"; transcript_id \"chr37.g3436.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10068285', '10068314', '.', '-', '.', 'gene_id \"chr37.g3440\"; transcript_id \"chr37.g3440.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10076625', '10076800', '.', '-', '.', 'gene_id \"chr37.g3446\"; transcript_id \"chr37.g3446.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10084838', '10085029', '.', '+', '.', 'gene_id \"chr37.g3461\"; transcript_id \"chr37.g3461.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10093555', '10093960', '.', '+', '.', 'gene_id \"chr37.g3468\"; transcript_id \"chr37.g3468.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '31351798', '31351832', '.', '-', '.', 'gene_id \"chr23.g11811\"; transcript_id \"chr23.g11811.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10099138', '10099223', '.', '+', '.', 'gene_id \"chr37.g3474\"; transcript_id \"chr37.g3474.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10109053', '10109240', '.', '+', '.', 'gene_id \"chr37.g3478\"; transcript_id \"chr37.g3478.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10112290', '10112789', '.', '+', '.', 'gene_id \"chr37.g3483\"; transcript_id \"chr37.g3483.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10115060', '10115269', '.', '-', '.', 'gene_id \"chr37.g3491\"; transcript_id \"chr37.g3491.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10118001', '10118798', '.', '-', '.', 'gene_id \"chr37.g3496\"; transcript_id \"chr37.g3496.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10117783', '10118010', '.', '+', '.', 'gene_id \"chr37.g3497\"; transcript_id \"chr37.g3497.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10119310', '10119569', '.', '-', '.', 'gene_id \"chr37.g3499\"; transcript_id \"chr37.g3499.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10120384', '10120599', '.', '+', '.', 'gene_id \"chr37.g3502\"; transcript_id \"chr37.g3502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10120697', '10121109', '.', '+', '.', 'gene_id \"chr37.g3503\"; transcript_id \"chr37.g3503.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10128063', '10128256', '.', '+', '.', 'gene_id \"chr37.g3508\"; transcript_id \"chr37.g3508.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10131179', '10131418', '.', '-', '.', 'gene_id \"chr37.g3511\"; transcript_id \"chr37.g3511.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10135707', '10135968', '.', '+', '.', 'gene_id \"chr37.g3519\"; transcript_id \"chr37.g3519.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10167872', '10168683', '.', '+', '.', 'gene_id \"chr37.g3532\"; transcript_id \"chr37.g3532.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10155533', '10155574', '.', '+', '.', 'gene_id \"chr37.g3533\"; transcript_id \"chr37.g3533.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10155548', '10156470', '.', '+', '.', 'gene_id \"chr37.g3533\"; transcript_id \"chr37.g3533.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10159927', '10160078', '.', '+', '.', 'gene_id \"chr37.g3537\"; transcript_id \"chr37.g3537.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10164683', '10167142', '.', '-', '.', 'gene_id \"chr37.g3540\"; transcript_id \"chr37.g3540.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10182072', '10182451', '.', '+', '.', 'gene_id \"chr37.g3547\"; transcript_id \"chr37.g3547.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10180925', '10181121', '.', '+', '.', 'gene_id \"chr37.g3548\"; transcript_id \"chr37.g3548.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10177310', '10177526', '.', '-', '.', 'gene_id \"chr37.g3554\"; transcript_id \"chr37.g3554.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10191010', '10191519', '.', '+', '.', 'gene_id \"chr37.g3555\"; transcript_id \"chr37.g3555.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10187200', '10187539', '.', '-', '.', 'gene_id \"chr37.g3564\"; transcript_id \"chr37.g3564.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10188586', '10188776', '.', '+', '.', 'gene_id \"chr37.g3568\"; transcript_id \"chr37.g3568.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10189716', '10189945', '.', '+', '.', 'gene_id \"chr37.g3571\"; transcript_id \"chr37.g3571.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10232505', '10233414', '.', '-', '.', 'gene_id \"chr37.g3576\"; transcript_id \"chr37.g3576.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10227120', '10227449', '.', '+', '.', 'gene_id \"chr37.g3580\"; transcript_id \"chr37.g3580.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10227428', '10228220', '.', '-', '.', 'gene_id \"chr37.g3582\"; transcript_id \"chr37.g3582.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10211763', '10212140', '.', '-', '.', 'gene_id \"chr37.g3583\"; transcript_id \"chr37.g3583.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10212132', '10212299', '.', '+', '.', 'gene_id \"chr37.g3585\"; transcript_id \"chr37.g3585.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10235244', '10235378', '.', '-', '.', 'gene_id \"chr37.g3589\"; transcript_id \"chr37.g3589.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10204481', '10204641', '.', '-', '.', 'gene_id \"chr37.g3590\"; transcript_id \"chr37.g3590.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10233395', '10234353', '.', '+', '.', 'gene_id \"chr37.g3592\"; transcript_id \"chr37.g3592.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10204718', '10204887', '.', '-', '.', 'gene_id \"chr37.g3595\"; transcript_id \"chr37.g3595.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10206091', '10206387', '.', '+', '.', 'gene_id \"chr37.g3598\"; transcript_id \"chr37.g3598.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10207743', '10208485', '.', '-', '.', 'gene_id \"chr37.g3600\"; transcript_id \"chr37.g3600.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10210039', '10210576', '.', '-', '.', 'gene_id \"chr37.g3605\"; transcript_id \"chr37.g3605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10216474', '10216507', '.', '-', '.', 'gene_id \"chr37.g3609\"; transcript_id \"chr37.g3609.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10216427', '10216602', '.', '+', '.', 'gene_id \"chr37.g3610\"; transcript_id \"chr37.g3610.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10223619', '10224066', '.', '-', '.', 'gene_id \"chr37.g3617\"; transcript_id \"chr37.g3617.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10224213', '10224384', '.', '+', '.', 'gene_id \"chr37.g3618\"; transcript_id \"chr37.g3618.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10224362', '10224721', '.', '+', '.', 'gene_id \"chr37.g3618\"; transcript_id \"chr37.g3618.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10229256', '10229506', '.', '-', '.', 'gene_id \"chr37.g3623\"; transcript_id \"chr37.g3623.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10235471', '10235771', '.', '-', '.', 'gene_id \"chr37.g3625\"; transcript_id \"chr37.g3625.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10236480', '10236649', '.', '-', '.', 'gene_id \"chr37.g3629\"; transcript_id \"chr37.g3629.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10284995', '10285966', '.', '+', '.', 'gene_id \"chr37.g3631\"; transcript_id \"chr37.g3631.i11\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10282607', '10282784', '.', '+', '.', 'gene_id \"chr37.g3631\"; transcript_id \"chr37.g3631.i14\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10283969', '10284006', '.', '+', '.', 'gene_id \"chr37.g3631\"; transcript_id \"chr37.g3631.i15\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10311822', '10312348', '.', '-', '.', 'gene_id \"chr37.g3632\"; transcript_id \"chr37.g3632.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10308041', '10308247', '.', '-', '.', 'gene_id \"chr37.g3638\"; transcript_id \"chr37.g3638.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10278950', '10279343', '.', '-', '.', 'gene_id \"chr37.g3642\"; transcript_id \"chr37.g3642.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10248451', '10248891', '.', '+', '.', 'gene_id \"chr37.g3646\"; transcript_id \"chr37.g3646.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10249359', '10249891', '.', '-', '.', 'gene_id \"chr37.g3648\"; transcript_id \"chr37.g3648.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10249981', '10250367', '.', '-', '.', 'gene_id \"chr37.g3649\"; transcript_id \"chr37.g3649.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10253079', '10253261', '.', '+', '.', 'gene_id \"chr37.g3651\"; transcript_id \"chr37.g3651.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10270787', '10271085', '.', '+', '.', 'gene_id \"chr37.g3656\"; transcript_id \"chr37.g3656.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10271045', '10271174', '.', '-', '.', 'gene_id \"chr37.g3657\"; transcript_id \"chr37.g3657.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10270787', '10271003', '.', '-', '.', 'gene_id \"chr37.g3658\"; transcript_id \"chr37.g3658.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10276109', '10276264', '.', '+', '.', 'gene_id \"chr37.g3659\"; transcript_id \"chr37.g3659.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10278065', '10278784', '.', '+', '.', 'gene_id \"chr37.g3664\"; transcript_id \"chr37.g3664.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10290399', '10291002', '.', '+', '.', 'gene_id \"chr37.g3669\"; transcript_id \"chr37.g3669.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10292029', '10292543', '.', '+', '.', 'gene_id \"chr37.g3670\"; transcript_id \"chr37.g3670.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10299849', '10300018', '.', '-', '.', 'gene_id \"chr37.g3678\"; transcript_id \"chr37.g3678.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '31862655', '31862731', '.', '+', '.', 'gene_id \"chr23.g11989\"; transcript_id \"chr23.g11989.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10314672', '10314880', '.', '-', '.', 'gene_id \"chr37.g3686\"; transcript_id \"chr37.g3686.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10321071', '10321237', '.', '-', '.', 'gene_id \"chr37.g3691\"; transcript_id \"chr37.g3691.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10321887', '10321928', '.', '+', '.', 'gene_id \"chr37.g3692\"; transcript_id \"chr37.g3692.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10322413', '10322591', '.', '+', '.', 'gene_id \"chr37.g3693\"; transcript_id \"chr37.g3693.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10331098', '10331248', '.', '+', '.', 'gene_id \"chr37.g3698\"; transcript_id \"chr37.g3698.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10337382', '10337547', '.', '-', '.', 'gene_id \"chr37.g3699\"; transcript_id \"chr37.g3699.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10337197', '10337419', '.', '-', '.', 'gene_id \"chr37.g3699\"; transcript_id \"chr37.g3699.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10338168', '10338403', '.', '-', '.', 'gene_id \"chr37.g3701\"; transcript_id \"chr37.g3701.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32026018', '32026588', '.', '-', '.', 'gene_id \"chr23.g12008\"; transcript_id \"chr23.g12008.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10351543', '10351885', '.', '+', '.', 'gene_id \"chr37.g3706\"; transcript_id \"chr37.g3706.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10351139', '10351540', '.', '+', '.', 'gene_id \"chr37.g3712\"; transcript_id \"chr37.g3712.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10350018', '10350309', '.', '+', '.', 'gene_id \"chr37.g3715\"; transcript_id \"chr37.g3715.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10451781', '10451982', '.', '+', '.', 'gene_id \"chr37.g3753\"; transcript_id \"chr37.g3753.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10482973', '10483593', '.', '-', '.', 'gene_id \"chr37.g3768\"; transcript_id \"chr37.g3768.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10485640', '10485763', '.', '-', '.', 'gene_id \"chr37.g3769\"; transcript_id \"chr37.g3769.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10500599', '10500832', '.', '-', '.', 'gene_id \"chr37.g3777\"; transcript_id \"chr37.g3777.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10502921', '10503083', '.', '+', '.', 'gene_id \"chr37.g3778\"; transcript_id \"chr37.g3778.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10520703', '10520810', '.', '+', '.', 'gene_id \"chr37.g3799\"; transcript_id \"chr37.g3799.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10548980', '10549570', '.', '+', '.', 'gene_id \"chr37.g3803\"; transcript_id \"chr37.g3803.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10527951', '10528390', '.', '+', '.', 'gene_id \"chr37.g3805\"; transcript_id \"chr37.g3805.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10534527', '10534755', '.', '+', '.', 'gene_id \"chr37.g3814\"; transcript_id \"chr37.g3814.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10539429', '10539647', '.', '+', '.', 'gene_id \"chr37.g3820\"; transcript_id \"chr37.g3820.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10593051', '10593615', '.', '+', '.', 'gene_id \"chr37.g3833\"; transcript_id \"chr37.g3833.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10586551', '10586702', '.', '+', '.', 'gene_id \"chr37.g3834\"; transcript_id \"chr37.g3834.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10583917', '10584127', '.', '-', '.', 'gene_id \"chr37.g3835\"; transcript_id \"chr37.g3835.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10591426', '10591643', '.', '-', '.', 'gene_id \"chr37.g3847\"; transcript_id \"chr37.g3847.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10591640', '10591919', '.', '+', '.', 'gene_id \"chr37.g3848\"; transcript_id \"chr37.g3848.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10594167', '10594712', '.', '-', '.', 'gene_id \"chr37.g3849\"; transcript_id \"chr37.g3849.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10594933', '10595454', '.', '-', '.', 'gene_id \"chr37.g3852\"; transcript_id \"chr37.g3852.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10609980', '10610254', '.', '-', '.', 'gene_id \"chr37.g3862\"; transcript_id \"chr37.g3862.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10619115', '10619474', '.', '-', '.', 'gene_id \"chr37.g3865\"; transcript_id \"chr37.g3865.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10645497', '10645559', '.', '-', '.', 'gene_id \"chr37.g3870\"; transcript_id \"chr37.g3870.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10710989', '10711217', '.', '+', '.', 'gene_id \"chr37.g3872\"; transcript_id \"chr37.g3872.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10677335', '10677715', '.', '+', '.', 'gene_id \"chr37.g3873\"; transcript_id \"chr37.g3873.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10663557', '10663746', '.', '-', '.', 'gene_id \"chr37.g3876\"; transcript_id \"chr37.g3876.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10648688', '10648920', '.', '+', '.', 'gene_id \"chr37.g3878\"; transcript_id \"chr37.g3878.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10645641', '10645965', '.', '-', '.', 'gene_id \"chr37.g3883\"; transcript_id \"chr37.g3883.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10664663', '10664838', '.', '+', '.', 'gene_id \"chr37.g3903\"; transcript_id \"chr37.g3903.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10703572', '10703726', '.', '+', '.', 'gene_id \"chr37.g3920\"; transcript_id \"chr37.g3920.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10703190', '10703592', '.', '+', '.', 'gene_id \"chr37.g3920\"; transcript_id \"chr37.g3920.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10818369', '10818583', '.', '+', '.', 'gene_id \"chr37.g3942\"; transcript_id \"chr37.g3942.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10818669', '10818821', '.', '-', '.', 'gene_id \"chr37.g3943\"; transcript_id \"chr37.g3943.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10857060', '10857223', '.', '-', '.', 'gene_id \"chr37.g3950\"; transcript_id \"chr37.g3950.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10862714', '10863385', '.', '+', '.', 'gene_id \"chr37.g3955\"; transcript_id \"chr37.g3955.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10862165', '10862371', '.', '-', '.', 'gene_id \"chr37.g3956\"; transcript_id \"chr37.g3956.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32422616', '32422822', '.', '-', '.', 'gene_id \"chr23.g12243\"; transcript_id \"chr23.g12243.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10932058', '10932151', '.', '+', '.', 'gene_id \"chr37.g3984\"; transcript_id \"chr37.g3984.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10941566', '10946383', '.', '+', '.', 'gene_id \"chr37.g3991\"; transcript_id \"chr37.g3991.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10941566', '10943655', '.', '+', '.', 'gene_id \"chr37.g3991\"; transcript_id \"chr37.g3991.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10946403', '10946610', '.', '-', '.', 'gene_id \"chr37.g3994\"; transcript_id \"chr37.g3994.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10946687', '10946881', '.', '-', '.', 'gene_id \"chr37.g3995\"; transcript_id \"chr37.g3995.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10946937', '10947209', '.', '-', '.', 'gene_id \"chr37.g3996\"; transcript_id \"chr37.g3996.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10949417', '10949731', '.', '+', '.', 'gene_id \"chr37.g3997\"; transcript_id \"chr37.g3997.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10949759', '10949947', '.', '+', '.', 'gene_id \"chr37.g3999\"; transcript_id \"chr37.g3999.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10965635', '10966008', '.', '+', '.', 'gene_id \"chr37.g4004\"; transcript_id \"chr37.g4004.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10985315', '10985361', '.', '-', '.', 'gene_id \"chr37.g4011\"; transcript_id \"chr37.g4011.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10986016', '10986193', '.', '+', '.', 'gene_id \"chr37.g4013\"; transcript_id \"chr37.g4013.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '10995449', '10995610', '.', '+', '.', 'gene_id \"chr37.g4016\"; transcript_id \"chr37.g4016.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11016575', '11016776', '.', '-', '.', 'gene_id \"chr37.g4018\"; transcript_id \"chr37.g4018.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11019705', '11019892', '.', '-', '.', 'gene_id \"chr37.g4020\"; transcript_id \"chr37.g4020.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11034422', '11034688', '.', '-', '.', 'gene_id \"chr37.g4027\"; transcript_id \"chr37.g4027.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11034689', '11035425', '.', '-', '.', 'gene_id \"chr37.g4029\"; transcript_id \"chr37.g4029.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11034689', '11034993', '.', '-', '.', 'gene_id \"chr37.g4029\"; transcript_id \"chr37.g4029.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11074035', '11074260', '.', '+', '.', 'gene_id \"chr37.g4035\"; transcript_id \"chr37.g4035.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11093733', '11094007', '.', '-', '.', 'gene_id \"chr37.g4041\"; transcript_id \"chr37.g4041.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11093733', '11094007', '.', '-', '.', 'gene_id \"chr37.g4041\"; transcript_id \"chr37.g4041.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11102731', '11102932', '.', '+', '.', 'gene_id \"chr37.g4042\"; transcript_id \"chr37.g4042.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11087923', '11088104', '.', '-', '.', 'gene_id \"chr37.g4046\"; transcript_id \"chr37.g4046.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11094146', '11094450', '.', '+', '.', 'gene_id \"chr37.g4047\"; transcript_id \"chr37.g4047.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11116629', '11116832', '.', '+', '.', 'gene_id \"chr37.g4048\"; transcript_id \"chr37.g4048.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11088138', '11088553', '.', '+', '.', 'gene_id \"chr37.g4049\"; transcript_id \"chr37.g4049.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11088563', '11088839', '.', '+', '.', 'gene_id \"chr37.g4051\"; transcript_id \"chr37.g4051.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11090108', '11090205', '.', '+', '.', 'gene_id \"chr37.g4052\"; transcript_id \"chr37.g4052.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11090691', '11090902', '.', '+', '.', 'gene_id \"chr37.g4055\"; transcript_id \"chr37.g4055.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr37', 'PASA', 'exon', '11097741', '11097931', '.', '+', '.', 'gene_id \"chr37.g4063\"; transcript_id \"chr37.g4063.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11103309', '11103470', '.', '+', '.', 'gene_id \"chr37.g4067\"; transcript_id \"chr37.g4067.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11103655', '11104090', '.', '-', '.', 'gene_id \"chr37.g4068\"; transcript_id \"chr37.g4068.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11103616', '11103674', '.', '-', '.', 'gene_id \"chr37.g4068\"; transcript_id \"chr37.g4068.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11106935', '11107916', '.', '-', '.', 'gene_id \"chr37.g4071\"; transcript_id \"chr37.g4071.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11116834', '11117306', '.', '-', '.', 'gene_id \"chr37.g4085\"; transcript_id \"chr37.g4085.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11116834', '11117689', '.', '-', '.', 'gene_id \"chr37.g4085\"; transcript_id \"chr37.g4085.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11117288', '11117689', '.', '-', '.', 'gene_id \"chr37.g4085\"; transcript_id \"chr37.g4085.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11117285', '11117689', '.', '+', '.', 'gene_id \"chr37.g4086\"; transcript_id \"chr37.g4086.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11118836', '11119032', '.', '-', '.', 'gene_id \"chr37.g4089\"; transcript_id \"chr37.g4089.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11166287', '11166591', '.', '+', '.', 'gene_id \"chr37.g4092\"; transcript_id \"chr37.g4092.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11166984', '11167387', '.', '-', '.', 'gene_id \"chr37.g4093\"; transcript_id \"chr37.g4093.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11173310', '11173612', '.', '+', '.', 'gene_id \"chr37.g4095\"; transcript_id \"chr37.g4095.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11152253', '11152486', '.', '+', '.', 'gene_id \"chr37.g4099\"; transcript_id \"chr37.g4099.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11153657', '11153830', '.', '-', '.', 'gene_id \"chr37.g4106\"; transcript_id \"chr37.g4106.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11153341', '11153626', '.', '-', '.', 'gene_id \"chr37.g4108\"; transcript_id \"chr37.g4108.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11155813', '11155986', '.', '+', '.', 'gene_id \"chr37.g4111\"; transcript_id \"chr37.g4111.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11170789', '11170920', '.', '-', '.', 'gene_id \"chr37.g4120\"; transcript_id \"chr37.g4120.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11227275', '11227525', '.', '+', '.', 'gene_id \"chr37.g4143\"; transcript_id \"chr37.g4143.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11227537', '11227732', '.', '-', '.', 'gene_id \"chr37.g4145\"; transcript_id \"chr37.g4145.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11228579', '11228755', '.', '-', '.', 'gene_id \"chr37.g4151\"; transcript_id \"chr37.g4151.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11231313', '11231778', '.', '-', '.', 'gene_id \"chr37.g4158\"; transcript_id \"chr37.g4158.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11231889', '11232199', '.', '-', '.', 'gene_id \"chr37.g4159\"; transcript_id \"chr37.g4159.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11232197', '11232360', '.', '-', '.', 'gene_id \"chr37.g4160\"; transcript_id \"chr37.g4160.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11233654', '11234018', '.', '-', '.', 'gene_id \"chr37.g4161\"; transcript_id \"chr37.g4161.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11242316', '11242801', '.', '-', '.', 'gene_id \"chr37.g4164\"; transcript_id \"chr37.g4164.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11246874', '11247204', '.', '+', '.', 'gene_id \"chr37.g4166\"; transcript_id \"chr37.g4166.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11249995', '11250232', '.', '-', '.', 'gene_id \"chr37.g4168\"; transcript_id \"chr37.g4168.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11254454', '11254842', '.', '+', '.', 'gene_id \"chr37.g4172\"; transcript_id \"chr37.g4172.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11255212', '11255250', '.', '-', '.', 'gene_id \"chr37.g4174\"; transcript_id \"chr37.g4174.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11257463', '11257613', '.', '-', '.', 'gene_id \"chr37.g4179\"; transcript_id \"chr37.g4179.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11258380', '11258510', '.', '-', '.', 'gene_id \"chr37.g4181\"; transcript_id \"chr37.g4181.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11258380', '11258547', '.', '-', '.', 'gene_id \"chr37.g4181\"; transcript_id \"chr37.g4181.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11258695', '11259075', '.', '-', '.', 'gene_id \"chr37.g4183\"; transcript_id \"chr37.g4183.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11259012', '11259181', '.', '-', '.', 'gene_id \"chr37.g4183\"; transcript_id \"chr37.g4183.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11260612', '11260802', '.', '-', '.', 'gene_id \"chr37.g4186\"; transcript_id \"chr37.g4186.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11265055', '11265364', '.', '-', '.', 'gene_id \"chr37.g4189\"; transcript_id \"chr37.g4189.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11265374', '11265956', '.', '+', '.', 'gene_id \"chr37.g4190\"; transcript_id \"chr37.g4190.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11266683', '11266988', '.', '+', '.', 'gene_id \"chr37.g4193\"; transcript_id \"chr37.g4193.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11267110', '11267367', '.', '-', '.', 'gene_id \"chr37.g4194\"; transcript_id \"chr37.g4194.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11272249', '11272721', '.', '+', '.', 'gene_id \"chr37.g4199\"; transcript_id \"chr37.g4199.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11274010', '11274272', '.', '-', '.', 'gene_id \"chr37.g4205\"; transcript_id \"chr37.g4205.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32660173', '32660222', '.', '-', '.', 'gene_id \"chr23.g12471\"; transcript_id \"chr23.g12471.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11291420', '11291624', '.', '+', '.', 'gene_id \"chr37.g4209\"; transcript_id \"chr37.g4209.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11321190', '11321454', '.', '+', '.', 'gene_id \"chr37.g4222\"; transcript_id \"chr37.g4222.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11321706', '11321904', '.', '+', '.', 'gene_id \"chr37.g4223\"; transcript_id \"chr37.g4223.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11321915', '11322123', '.', '-', '.', 'gene_id \"chr37.g4224\"; transcript_id \"chr37.g4224.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11332049', '11332252', '.', '-', '.', 'gene_id \"chr37.g4233\"; transcript_id \"chr37.g4233.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11337116', '11337387', '.', '+', '.', 'gene_id \"chr37.g4238\"; transcript_id \"chr37.g4238.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32707187', '32707343', '.', '-', '.', 'gene_id \"chr23.g12502\"; transcript_id \"chr23.g12502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11337589', '11338897', '.', '+', '.', 'gene_id \"chr37.g4240\"; transcript_id \"chr37.g4240.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11341824', '11342103', '.', '+', '.', 'gene_id \"chr37.g4247\"; transcript_id \"chr37.g4247.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11347033', '11347261', '.', '+', '.', 'gene_id \"chr37.g4253\"; transcript_id \"chr37.g4253.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11349136', '11349378', '.', '-', '.', 'gene_id \"chr37.g4255\"; transcript_id \"chr37.g4255.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11349437', '11349703', '.', '+', '.', 'gene_id \"chr37.g4257\"; transcript_id \"chr37.g4257.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11350362', '11350568', '.', '+', '.', 'gene_id \"chr37.g4258\"; transcript_id \"chr37.g4258.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11351554', '11351710', '.', '+', '.', 'gene_id \"chr37.g4262\"; transcript_id \"chr37.g4262.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11360389', '11360577', '.', '-', '.', 'gene_id \"chr37.g4268\"; transcript_id \"chr37.g4268.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11391120', '11391342', '.', '+', '.', 'gene_id \"chr37.g4276\"; transcript_id \"chr37.g4276.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11396202', '11396513', '.', '-', '.', 'gene_id \"chr37.g4283\"; transcript_id \"chr37.g4283.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11403841', '11404399', '.', '-', '.', 'gene_id \"chr37.g4295\"; transcript_id \"chr37.g4295.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11424460', '11424812', '.', '-', '.', 'gene_id \"chr37.g4310\"; transcript_id \"chr37.g4310.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11424573', '11424812', '.', '-', '.', 'gene_id \"chr37.g4310\"; transcript_id \"chr37.g4310.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11520693', '11520852', '.', '+', '.', 'gene_id \"chr37.g4334\"; transcript_id \"chr37.g4334.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11526896', '11527137', '.', '+', '.', 'gene_id \"chr37.g4336\"; transcript_id \"chr37.g4336.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11533505', '11533639', '.', '+', '.', 'gene_id \"chr37.g4341\"; transcript_id \"chr37.g4341.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11542400', '11542596', '.', '+', '.', 'gene_id \"chr37.g4347\"; transcript_id \"chr37.g4347.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11545977', '11546170', '.', '-', '.', 'gene_id \"chr37.g4351\"; transcript_id \"chr37.g4351.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11546301', '11546619', '.', '+', '.', 'gene_id \"chr37.g4352\"; transcript_id \"chr37.g4352.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11568042', '11571492', '.', '+', '.', 'gene_id \"chr37.g4374\"; transcript_id \"chr37.g4374.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11568042', '11571441', '.', '+', '.', 'gene_id \"chr37.g4374\"; transcript_id \"chr37.g4374.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11596419', '11596558', '.', '+', '.', 'gene_id \"chr37.g4379\"; transcript_id \"chr37.g4379.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11588137', '11588347', '.', '+', '.', 'gene_id \"chr37.g4380\"; transcript_id \"chr37.g4380.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11597385', '11597622', '.', '-', '.', 'gene_id \"chr37.g4382\"; transcript_id \"chr37.g4382.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11635049', '11635198', '.', '+', '.', 'gene_id \"chr37.g4394\"; transcript_id \"chr37.g4394.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11642575', '11642898', '.', '-', '.', 'gene_id \"chr37.g4402\"; transcript_id \"chr37.g4402.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11646258', '11646487', '.', '-', '.', 'gene_id \"chr37.g4404\"; transcript_id \"chr37.g4404.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11646540', '11646700', '.', '-', '.', 'gene_id \"chr37.g4405\"; transcript_id \"chr37.g4405.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11647517', '11647629', '.', '+', '.', 'gene_id \"chr37.g4406\"; transcript_id \"chr37.g4406.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11647517', '11647629', '.', '+', '.', 'gene_id \"chr37.g4406\"; transcript_id \"chr37.g4406.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11649054', '11649244', '.', '+', '.', 'gene_id \"chr37.g4406\"; transcript_id \"chr37.g4406.i6\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11649054', '11649244', '.', '+', '.', 'gene_id \"chr37.g4406\"; transcript_id \"chr37.g4406.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11666854', '11667156', '.', '-', '.', 'gene_id \"chr37.g4410\"; transcript_id \"chr37.g4410.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11651289', '11651844', '.', '+', '.', 'gene_id \"chr37.g4411\"; transcript_id \"chr37.g4411.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11648882', '11649054', '.', '-', '.', 'gene_id \"chr37.g4413\"; transcript_id \"chr37.g4413.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11648270', '11648520', '.', '-', '.', 'gene_id \"chr37.g4415\"; transcript_id \"chr37.g4415.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11650040', '11650199', '.', '+', '.', 'gene_id \"chr37.g4416\"; transcript_id \"chr37.g4416.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11652000', '11652635', '.', '+', '.', 'gene_id \"chr37.g4418\"; transcript_id \"chr37.g4418.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11658974', '11659127', '.', '+', '.', 'gene_id \"chr37.g4423\"; transcript_id \"chr37.g4423.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11661290', '11661583', '.', '+', '.', 'gene_id \"chr37.g4424\"; transcript_id \"chr37.g4424.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11662041', '11662242', '.', '-', '.', 'gene_id \"chr37.g4425\"; transcript_id \"chr37.g4425.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11679614', '11679817', '.', '+', '.', 'gene_id \"chr37.g4438\"; transcript_id \"chr37.g4438.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11679913', '11680483', '.', '-', '.', 'gene_id \"chr37.g4440\"; transcript_id \"chr37.g4440.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11680003', '11680483', '.', '-', '.', 'gene_id \"chr37.g4440\"; transcript_id \"chr37.g4440.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32914271', '32914326', '.', '+', '.', 'gene_id \"chr23.g12705\"; transcript_id \"chr23.g12705.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11686784', '11687149', '.', '+', '.', 'gene_id \"chr37.g4449\"; transcript_id \"chr37.g4449.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11687302', '11687627', '.', '+', '.', 'gene_id \"chr37.g4450\"; transcript_id \"chr37.g4450.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11737176', '11737570', '.', '+', '.', 'gene_id \"chr37.g4455\"; transcript_id \"chr37.g4455.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11694527', '11694795', '.', '-', '.', 'gene_id \"chr37.g4459\"; transcript_id \"chr37.g4459.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11694399', '11694523', '.', '-', '.', 'gene_id \"chr37.g4460\"; transcript_id \"chr37.g4460.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11694999', '11695327', '.', '-', '.', 'gene_id \"chr37.g4465\"; transcript_id \"chr37.g4465.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11695879', '11695926', '.', '-', '.', 'gene_id \"chr37.g4466\"; transcript_id \"chr37.g4466.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11695426', '11695901', '.', '-', '.', 'gene_id \"chr37.g4466\"; transcript_id \"chr37.g4466.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11707123', '11707537', '.', '+', '.', 'gene_id \"chr37.g4473\"; transcript_id \"chr37.g4473.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11732295', '11732465', '.', '-', '.', 'gene_id \"chr37.g4480\"; transcript_id \"chr37.g4480.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11741162', '11741400', '.', '+', '.', 'gene_id \"chr37.g4487\"; transcript_id \"chr37.g4487.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11741432', '11742016', '.', '-', '.', 'gene_id \"chr37.g4488\"; transcript_id \"chr37.g4488.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11745471', '11745605', '.', '+', '.', 'gene_id \"chr37.g4495\"; transcript_id \"chr37.g4495.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11755053', '11755330', '.', '-', '.', 'gene_id \"chr37.g4502\"; transcript_id \"chr37.g4502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11755095', '11755330', '.', '-', '.', 'gene_id \"chr37.g4502\"; transcript_id \"chr37.g4502.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11755315', '11755485', '.', '+', '.', 'gene_id \"chr37.g4503\"; transcript_id \"chr37.g4503.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11755487', '11755837', '.', '+', '.', 'gene_id \"chr37.g4504\"; transcript_id \"chr37.g4504.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11758346', '11758593', '.', '+', '.', 'gene_id \"chr37.g4508\"; transcript_id \"chr37.g4508.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11780008', '11780224', '.', '-', '.', 'gene_id \"chr37.g4525\"; transcript_id \"chr37.g4525.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11781446', '11781622', '.', '+', '.', 'gene_id \"chr37.g4526\"; transcript_id \"chr37.g4526.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11781739', '11781912', '.', '-', '.', 'gene_id \"chr37.g4529\"; transcript_id \"chr37.g4529.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11888547', '11888780', '.', '+', '.', 'gene_id \"chr37.g4532\"; transcript_id \"chr37.g4532.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11966508', '11967448', '.', '-', '.', 'gene_id \"chr37.g4533\"; transcript_id \"chr37.g4533.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11955628', '11955920', '.', '-', '.', 'gene_id \"chr37.g4540\"; transcript_id \"chr37.g4540.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11788982', '11789251', '.', '-', '.', 'gene_id \"chr37.g4542\"; transcript_id \"chr37.g4542.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11789253', '11789577', '.', '-', '.', 'gene_id \"chr37.g4543\"; transcript_id \"chr37.g4543.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11802308', '11802482', '.', '-', '.', 'gene_id \"chr37.g4551\"; transcript_id \"chr37.g4551.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11803877', '11804204', '.', '+', '.', 'gene_id \"chr37.g4554\"; transcript_id \"chr37.g4554.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11824229', '11824398', '.', '+', '.', 'gene_id \"chr37.g4560\"; transcript_id \"chr37.g4560.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11859863', '11860104', '.', '-', '.', 'gene_id \"chr37.g4583\"; transcript_id \"chr37.g4583.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11878818', '11879048', '.', '+', '.', 'gene_id \"chr37.g4590\"; transcript_id \"chr37.g4590.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11890430', '11890751', '.', '+', '.', 'gene_id \"chr37.g4598\"; transcript_id \"chr37.g4598.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11892634', '11892866', '.', '+', '.', 'gene_id \"chr37.g4605\"; transcript_id \"chr37.g4605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11892967', '11893413', '.', '-', '.', 'gene_id \"chr37.g4606\"; transcript_id \"chr37.g4606.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11892974', '11893412', '.', '+', '.', 'gene_id \"chr37.g4607\"; transcript_id \"chr37.g4607.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11895692', '11896112', '.', '+', '.', 'gene_id \"chr37.g4609\"; transcript_id \"chr37.g4609.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11905938', '11906088', '.', '-', '.', 'gene_id \"chr37.g4612\"; transcript_id \"chr37.g4612.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11906119', '11906300', '.', '+', '.', 'gene_id \"chr37.g4613\"; transcript_id \"chr37.g4613.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11906525', '11906722', '.', '-', '.', 'gene_id \"chr37.g4615\"; transcript_id \"chr37.g4615.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11906701', '11906854', '.', '-', '.', 'gene_id \"chr37.g4615\"; transcript_id \"chr37.g4615.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11911088', '11911244', '.', '-', '.', 'gene_id \"chr37.g4621\"; transcript_id \"chr37.g4621.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11921973', '11922256', '.', '-', '.', 'gene_id \"chr37.g4630\"; transcript_id \"chr37.g4630.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11922528', '11922702', '.', '-', '.', 'gene_id \"chr37.g4631\"; transcript_id \"chr37.g4631.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11922968', '11923137', '.', '-', '.', 'gene_id \"chr37.g4632\"; transcript_id \"chr37.g4632.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11926014', '11926265', '.', '+', '.', 'gene_id \"chr37.g4636\"; transcript_id \"chr37.g4636.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11929427', '11929615', '.', '+', '.', 'gene_id \"chr37.g4638\"; transcript_id \"chr37.g4638.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11932601', '11932744', '.', '-', '.', 'gene_id \"chr37.g4641\"; transcript_id \"chr37.g4641.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11936066', '11936229', '.', '+', '.', 'gene_id \"chr37.g4646\"; transcript_id \"chr37.g4646.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11936230', '11936383', '.', '-', '.', 'gene_id \"chr37.g4647\"; transcript_id \"chr37.g4647.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11945733', '11946051', '.', '-', '.', 'gene_id \"chr37.g4659\"; transcript_id \"chr37.g4659.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11945733', '11946051', '.', '+', '.', 'gene_id \"chr37.g4660\"; transcript_id \"chr37.g4660.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11947494', '11947646', '.', '+', '.', 'gene_id \"chr37.g4662\"; transcript_id \"chr37.g4662.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11948834', '11949007', '.', '-', '.', 'gene_id \"chr37.g4665\"; transcript_id \"chr37.g4665.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11950938', '11950978', '.', '+', '.', 'gene_id \"chr37.g4668\"; transcript_id \"chr37.g4668.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11950708', '11950845', '.', '+', '.', 'gene_id \"chr37.g4669\"; transcript_id \"chr37.g4669.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11954388', '11954558', '.', '+', '.', 'gene_id \"chr37.g4670\"; transcript_id \"chr37.g4670.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11954540', '11954727', '.', '-', '.', 'gene_id \"chr37.g4671\"; transcript_id \"chr37.g4671.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11954792', '11955369', '.', '+', '.', 'gene_id \"chr37.g4672\"; transcript_id \"chr37.g4672.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11967449', '11967712', '.', '-', '.', 'gene_id \"chr37.g4675\"; transcript_id \"chr37.g4675.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11969936', '11970346', '.', '-', '.', 'gene_id \"chr37.g4677\"; transcript_id \"chr37.g4677.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11971435', '11972243', '.', '-', '.', 'gene_id \"chr37.g4680\"; transcript_id \"chr37.g4680.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11973648', '11974121', '.', '-', '.', 'gene_id \"chr37.g4683\"; transcript_id \"chr37.g4683.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11974199', '11976518', '.', '-', '.', 'gene_id \"chr37.g4685\"; transcript_id \"chr37.g4685.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11976519', '11977170', '.', '+', '.', 'gene_id \"chr37.g4688\"; transcript_id \"chr37.g4688.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11977176', '11977385', '.', '+', '.', 'gene_id \"chr37.g4689\"; transcript_id \"chr37.g4689.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11977176', '11977311', '.', '+', '.', 'gene_id \"chr37.g4689\"; transcript_id \"chr37.g4689.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12024344', '12024523', '.', '+', '.', 'gene_id \"chr37.g4693\"; transcript_id \"chr37.g4693.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11978584', '11978801', '.', '-', '.', 'gene_id \"chr37.g4695\"; transcript_id \"chr37.g4695.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11986589', '11987098', '.', '+', '.', 'gene_id \"chr37.g4705\"; transcript_id \"chr37.g4705.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11992271', '11992428', '.', '+', '.', 'gene_id \"chr37.g4712\"; transcript_id \"chr37.g4712.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11998466', '11999095', '.', '+', '.', 'gene_id \"chr37.g4719\"; transcript_id \"chr37.g4719.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '11998466', '11998955', '.', '+', '.', 'gene_id \"chr37.g4719\"; transcript_id \"chr37.g4719.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12016837', '12017229', '.', '+', '.', 'gene_id \"chr37.g4727\"; transcript_id \"chr37.g4727.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12024663', '12025038', '.', '+', '.', 'gene_id \"chr37.g4734\"; transcript_id \"chr37.g4734.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12026508', '12026718', '.', '+', '.', 'gene_id \"chr37.g4739\"; transcript_id \"chr37.g4739.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12027353', '12027646', '.', '+', '.', 'gene_id \"chr37.g4741\"; transcript_id \"chr37.g4741.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12027940', '12028394', '.', '+', '.', 'gene_id \"chr37.g4742\"; transcript_id \"chr37.g4742.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12031518', '12031660', '.', '-', '.', 'gene_id \"chr37.g4746\"; transcript_id \"chr37.g4746.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12039678', '12042348', '.', '-', '.', 'gene_id \"chr37.g4754\"; transcript_id \"chr37.g4754.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12047842', '12048355', '.', '+', '.', 'gene_id \"chr37.g4759\"; transcript_id \"chr37.g4759.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12054430', '12054806', '.', '-', '.', 'gene_id \"chr37.g4765\"; transcript_id \"chr37.g4765.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12115982', '12116151', '.', '+', '.', 'gene_id \"chr37.g4766\"; transcript_id \"chr37.g4766.i6\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12054902', '12055205', '.', '+', '.', 'gene_id \"chr37.g4769\"; transcript_id \"chr37.g4769.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12060373', '12060640', '.', '-', '.', 'gene_id \"chr37.g4770\"; transcript_id \"chr37.g4770.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12086953', '12087132', '.', '+', '.', 'gene_id \"chr37.g4782\"; transcript_id \"chr37.g4782.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12091983', '12092160', '.', '+', '.', 'gene_id \"chr37.g4785\"; transcript_id \"chr37.g4785.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12103714', '12103968', '.', '+', '.', 'gene_id \"chr37.g4787\"; transcript_id \"chr37.g4787.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12147632', '12147805', '.', '-', '.', 'gene_id \"chr37.g4806\"; transcript_id \"chr37.g4806.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12148900', '12149172', '.', '-', '.', 'gene_id \"chr37.g4809\"; transcript_id \"chr37.g4809.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12149183', '12149423', '.', '+', '.', 'gene_id \"chr37.g4811\"; transcript_id \"chr37.g4811.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12163857', '12164331', '.', '-', '.', 'gene_id \"chr37.g4819\"; transcript_id \"chr37.g4819.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12182753', '12183568', '.', '+', '.', 'gene_id \"chr37.g4826\"; transcript_id \"chr37.g4826.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12183746', '12183906', '.', '-', '.', 'gene_id \"chr37.g4828\"; transcript_id \"chr37.g4828.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12185585', '12185864', '.', '+', '.', 'gene_id \"chr37.g4832\"; transcript_id \"chr37.g4832.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12192109', '12192405', '.', '-', '.', 'gene_id \"chr37.g4838\"; transcript_id \"chr37.g4838.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12193931', '12194203', '.', '+', '.', 'gene_id \"chr37.g4844\"; transcript_id \"chr37.g4844.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12193869', '12194221', '.', '-', '.', 'gene_id \"chr37.g4845\"; transcript_id \"chr37.g4845.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12205028', '12205354', '.', '-', '.', 'gene_id \"chr37.g4851\"; transcript_id \"chr37.g4851.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12202858', '12203357', '.', '+', '.', 'gene_id \"chr37.g4853\"; transcript_id \"chr37.g4853.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12223602', '12223836', '.', '-', '.', 'gene_id \"chr37.g4881\"; transcript_id \"chr37.g4881.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12228348', '12228805', '.', '+', '.', 'gene_id \"chr37.g4887\"; transcript_id \"chr37.g4887.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12263663', '12263826', '.', '-', '.', 'gene_id \"chr37.g4897\"; transcript_id \"chr37.g4897.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12266562', '12266615', '.', '-', '.', 'gene_id \"chr37.g4898\"; transcript_id \"chr37.g4898.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12267263', '12267455', '.', '+', '.', 'gene_id \"chr37.g4900\"; transcript_id \"chr37.g4900.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '33586862', '33588225', '.', '-', '.', 'gene_id \"chr23.g13153\"; transcript_id \"chr23.g13153.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12354341', '12357332', '.', '-', '.', 'gene_id \"chr37.g4922\"; transcript_id \"chr37.g4922.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12353196', '12353800', '.', '-', '.', 'gene_id \"chr37.g4923\"; transcript_id \"chr37.g4923.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12350716', '12351358', '.', '-', '.', 'gene_id \"chr37.g4924\"; transcript_id \"chr37.g4924.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12350716', '12351325', '.', '-', '.', 'gene_id \"chr37.g4924\"; transcript_id \"chr37.g4924.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12350716', '12350872', '.', '-', '.', 'gene_id \"chr37.g4924\"; transcript_id \"chr37.g4924.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12351419', '12351626', '.', '+', '.', 'gene_id \"chr37.g4925\"; transcript_id \"chr37.g4925.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12364588', '12367100', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12364424', '12364608', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12363871', '12364445', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i5\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12363871', '12364402', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i6\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12362651', '12364445', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12363878', '12364402', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i8\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12363878', '12364445', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i9\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12362694', '12364402', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12364588', '12364736', '.', '-', '.', 'gene_id \"chr37.g4931\"; transcript_id \"chr37.g4931.i11\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12367090', '12368533', '.', '+', '.', 'gene_id \"chr37.g4932\"; transcript_id \"chr37.g4932.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12368535', '12369828', '.', '+', '.', 'gene_id \"chr37.g4934\"; transcript_id \"chr37.g4934.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12384447', '12384609', '.', '+', '.', 'gene_id \"chr37.g4937\"; transcript_id \"chr37.g4937.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12385802', '12385975', '.', '+', '.', 'gene_id \"chr37.g4938\"; transcript_id \"chr37.g4938.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12407230', '12407384', '.', '-', '.', 'gene_id \"chr37.g4943\"; transcript_id \"chr37.g4943.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12437625', '12437818', '.', '+', '.', 'gene_id \"chr37.g4965\"; transcript_id \"chr37.g4965.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '33932743', '33932784', '.', '+', '.', 'gene_id \"chr23.g13216\"; transcript_id \"chr23.g13216.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '33950968', '33951363', '.', '+', '.', 'gene_id \"chr23.g13217\"; transcript_id \"chr23.g13217.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12555491', '12555641', '.', '+', '.', 'gene_id \"chr37.g4974\"; transcript_id \"chr37.g4974.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12563574', '12563863', '.', '-', '.', 'gene_id \"chr37.g4975\"; transcript_id \"chr37.g4975.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12574001', '12574034', '.', '-', '.', 'gene_id \"chr37.g4977\"; transcript_id \"chr37.g4977.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12606423', '12606572', '.', '+', '.', 'gene_id \"chr37.g4980\"; transcript_id \"chr37.g4980.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12620145', '12620297', '.', '-', '.', 'gene_id \"chr37.g4982\"; transcript_id \"chr37.g4982.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12630818', '12631042', '.', '-', '.', 'gene_id \"chr37.g4985\"; transcript_id \"chr37.g4985.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12877484', '12877675', '.', '+', '.', 'gene_id \"chr37.g5003\"; transcript_id \"chr37.g5003.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '12890731', '12890887', '.', '+', '.', 'gene_id \"chr37.g5004\"; transcript_id \"chr37.g5004.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34453147', '34454226', '.', '+', '.', 'gene_id \"chr23.g13265\"; transcript_id \"chr23.g13265.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34454276', '34455465', '.', '+', '.', 'gene_id \"chr23.g13266\"; transcript_id \"chr23.g13266.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34510854', '34511041', '.', '+', '.', 'gene_id \"chr23.g13270\"; transcript_id \"chr23.g13270.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34510854', '34510924', '.', '+', '.', 'gene_id \"chr23.g13270\"; transcript_id \"chr23.g13270.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34503234', '34503338', '.', '+', '.', 'gene_id \"chr23.g13271\"; transcript_id \"chr23.g13271.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34503234', '34503304', '.', '+', '.', 'gene_id \"chr23.g13271\"; transcript_id \"chr23.g13271.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34507191', '34507233', '.', '+', '.', 'gene_id \"chr23.g13277\"; transcript_id \"chr23.g13277.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34500506', '34500747', '.', '+', '.', 'gene_id \"chr23.g13281\"; transcript_id \"chr23.g13281.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34516780', '34517342', '.', '+', '.', 'gene_id \"chr23.g13300\"; transcript_id \"chr23.g13300.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34532315', '34532382', '.', '+', '.', 'gene_id \"chr23.g13310\"; transcript_id \"chr23.g13310.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '13374190', '13374534', '.', '+', '.', 'gene_id \"chr37.g5093\"; transcript_id \"chr37.g5093.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34698095', '34698400', '.', '-', '.', 'gene_id \"chr23.g13396\"; transcript_id \"chr23.g13396.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34709515', '34709544', '.', '-', '.', 'gene_id \"chr23.g13410\"; transcript_id \"chr23.g13410.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '13940310', '13940342', '.', '+', '.', 'gene_id \"chr37.g5310\"; transcript_id \"chr37.g5310.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '14077141', '14077202', '.', '-', '.', 'gene_id \"chr37.g5374\"; transcript_id \"chr37.g5374.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35218117', '35218164', '.', '+', '.', 'gene_id \"chr23.g13777\"; transcript_id \"chr23.g13777.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35485904', '35486033', '.', '-', '.', 'gene_id \"chr23.g13883\"; transcript_id \"chr23.g13883.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '14528109', '14528519', '.', '-', '.', 'gene_id \"chr37.g5585\"; transcript_id \"chr37.g5585.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35765497', '35765704', '.', '+', '.', 'gene_id \"chr23.g13961\"; transcript_id \"chr23.g13961.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '14575563', '14575609', '.', '+', '.', 'gene_id \"chr37.g5669\"; transcript_id \"chr37.g5669.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '14803685', '14804689', '.', '+', '.', 'gene_id \"chr37.g5750\"; transcript_id \"chr37.g5750.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35987648', '35987809', '.', '+', '.', 'gene_id \"chr23.g14105\"; transcript_id \"chr23.g14105.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35987932', '35988045', '.', '-', '.', 'gene_id \"chr23.g14107\"; transcript_id \"chr23.g14107.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15033742', '15033921', '.', '+', '.', 'gene_id \"chr37.g5816\"; transcript_id \"chr37.g5816.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15033925', '15034129', '.', '+', '.', 'gene_id \"chr37.g5817\"; transcript_id \"chr37.g5817.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15034161', '15034466', '.', '-', '.', 'gene_id \"chr37.g5818\"; transcript_id \"chr37.g5818.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15034484', '15034946', '.', '+', '.', 'gene_id \"chr37.g5819\"; transcript_id \"chr37.g5819.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15035395', '15035785', '.', '+', '.', 'gene_id \"chr37.g5820\"; transcript_id \"chr37.g5820.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '36200105', '36200136', '.', '-', '.', 'gene_id \"chr23.g14178\"; transcript_id \"chr23.g14178.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15296696', '15296875', '.', '-', '.', 'gene_id \"chr37.g5885\"; transcript_id \"chr37.g5885.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '36905455', '36905520', '.', '-', '.', 'gene_id \"chr23.g14251\"; transcript_id \"chr23.g14251.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37006265', '37006313', '.', '+', '.', 'gene_id \"chr23.g14267\"; transcript_id \"chr23.g14267.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37094476', '37094513', '.', '+', '.', 'gene_id \"chr23.g14272\"; transcript_id \"chr23.g14272.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37129345', '37129951', '.', '-', '.', 'gene_id \"chr23.g14290\"; transcript_id \"chr23.g14290.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37130257', '37130868', '.', '+', '.', 'gene_id \"chr23.g14291\"; transcript_id \"chr23.g14291.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37202624', '37203009', '.', '+', '.', 'gene_id \"chr23.g14315\"; transcript_id \"chr23.g14315.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '15876859', '15876901', '.', '-', '.', 'gene_id \"chr37.g6086\"; transcript_id \"chr37.g6086.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '16143903', '16143955', '.', '+', '.', 'gene_id \"chr37.g6346\"; transcript_id \"chr37.g6346.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37763898', '37764130', '.', '+', '.', 'gene_id \"chr23.g14720\"; transcript_id \"chr23.g14720.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '16232406', '16232437', '.', '-', '.', 'gene_id \"chr37.g6430\"; transcript_id \"chr37.g6430.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '16338700', '16338884', '.', '+', '.', 'gene_id \"chr37.g6457\"; transcript_id \"chr37.g6457.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37938720', '37939892', '.', '-', '.', 'gene_id \"chr23.g14784\"; transcript_id \"chr23.g14784.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37938380', '37938558', '.', '+', '.', 'gene_id \"chr23.g14785\"; transcript_id \"chr23.g14785.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '38004781', '38004822', '.', '-', '.', 'gene_id \"chr23.g14865\"; transcript_id \"chr23.g14865.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '16694404', '16694468', '.', '+', '.', 'gene_id \"chr37.g6596\"; transcript_id \"chr37.g6596.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '16750360', '16750396', '.', '+', '.', 'gene_id \"chr37.g6598\"; transcript_id \"chr37.g6598.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '38088669', '38089010', '.', '+', '.', 'gene_id \"chr23.g14936\"; transcript_id \"chr23.g14936.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '38089423', '38089920', '.', '+', '.', 'gene_id \"chr23.g14937\"; transcript_id \"chr23.g14937.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17170368', '17170414', '.', '-', '.', 'gene_id \"chr37.g6650\"; transcript_id \"chr37.g6650.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17215211', '17215242', '.', '-', '.', 'gene_id \"chr37.g6663\"; transcript_id \"chr37.g6663.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17483540', '17483926', '.', '-', '.', 'gene_id \"chr37.g6769\"; transcript_id \"chr37.g6769.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17484129', '17484506', '.', '+', '.', 'gene_id \"chr37.g6770\"; transcript_id \"chr37.g6770.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17485675', '17486221', '.', '-', '.', 'gene_id \"chr37.g6772\"; transcript_id \"chr37.g6772.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17593625', '17593706', '.', '-', '.', 'gene_id \"chr37.g6798\"; transcript_id \"chr37.g6798.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '17841634', '17841895', '.', '-', '.', 'gene_id \"chr37.g6928\"; transcript_id \"chr37.g6928.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '38501419', '38501452', '.', '-', '.', 'gene_id \"chr23.g15223\"; transcript_id \"chr23.g15223.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '38830492', '38830679', '.', '-', '.', 'gene_id \"chr23.g15406\"; transcript_id \"chr23.g15406.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '39059586', '39059840', '.', '-', '.', 'gene_id \"chr23.g15489\"; transcript_id \"chr23.g15489.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '18302882', '18303151', '.', '+', '.', 'gene_id \"chr37.g7334\"; transcript_id \"chr37.g7334.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '39172349', '39172397', '.', '+', '.', 'gene_id \"chr23.g15576\"; transcript_id \"chr23.g15576.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '18618941', '18619112', '.', '-', '.', 'gene_id \"chr37.g7354\"; transcript_id \"chr37.g7354.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '18619788', '18620052', '.', '-', '.', 'gene_id \"chr37.g7356\"; transcript_id \"chr37.g7356.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '18794802', '18794878', '.', '+', '.', 'gene_id \"chr37.g7365\"; transcript_id \"chr37.g7365.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '18951935', '18952010', '.', '+', '.', 'gene_id \"chr37.g7375\"; transcript_id \"chr37.g7375.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19096568', '19096753', '.', '+', '.', 'gene_id \"chr37.g7378\"; transcript_id \"chr37.g7378.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19034255', '19034428', '.', '+', '.', 'gene_id \"chr37.g7384\"; transcript_id \"chr37.g7384.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19045864', '19046069', '.', '-', '.', 'gene_id \"chr37.g7388\"; transcript_id \"chr37.g7388.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19297816', '19297953', '.', '+', '.', 'gene_id \"chr37.g7403\"; transcript_id \"chr37.g7403.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19378395', '19378631', '.', '-', '.', 'gene_id \"chr37.g7410\"; transcript_id \"chr37.g7410.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19442539', '19442725', '.', '+', '.', 'gene_id \"chr37.g7412\"; transcript_id \"chr37.g7412.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19527591', '19527793', '.', '-', '.', 'gene_id \"chr37.g7419\"; transcript_id \"chr37.g7419.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19528140', '19528475', '.', '+', '.', 'gene_id \"chr37.g7422\"; transcript_id \"chr37.g7422.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19529217', '19529376', '.', '+', '.', 'gene_id \"chr37.g7423\"; transcript_id \"chr37.g7423.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19530187', '19531115', '.', '+', '.', 'gene_id \"chr37.g7425\"; transcript_id \"chr37.g7425.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19532591', '19532746', '.', '-', '.', 'gene_id \"chr37.g7426\"; transcript_id \"chr37.g7426.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '39296918', '39296979', '.', '-', '.', 'gene_id \"chr23.g15661\"; transcript_id \"chr23.g15661.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19600603', '19600950', '.', '-', '.', 'gene_id \"chr37.g7431\"; transcript_id \"chr37.g7431.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19602928', '19603109', '.', '-', '.', 'gene_id \"chr37.g7432\"; transcript_id \"chr37.g7432.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19603099', '19603381', '.', '-', '.', 'gene_id \"chr37.g7433\"; transcript_id \"chr37.g7433.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19603388', '19603713', '.', '+', '.', 'gene_id \"chr37.g7434\"; transcript_id \"chr37.g7434.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '39300522', '39300742', '.', '-', '.', 'gene_id \"chr23.g15668\"; transcript_id \"chr23.g15668.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '39301074', '39301317', '.', '-', '.', 'gene_id \"chr23.g15669\"; transcript_id \"chr23.g15669.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19605910', '19606258', '.', '+', '.', 'gene_id \"chr37.g7439\"; transcript_id \"chr37.g7439.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19606788', '19607719', '.', '+', '.', 'gene_id \"chr37.g7440\"; transcript_id \"chr37.g7440.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19608589', '19609463', '.', '-', '.', 'gene_id \"chr37.g7442\"; transcript_id \"chr37.g7442.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19609442', '19609670', '.', '+', '.', 'gene_id \"chr37.g7443\"; transcript_id \"chr37.g7443.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19612326', '19612484', '.', '-', '.', 'gene_id \"chr37.g7444\"; transcript_id \"chr37.g7444.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19612496', '19612709', '.', '-', '.', 'gene_id \"chr37.g7445\"; transcript_id \"chr37.g7445.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19615977', '19616162', '.', '+', '.', 'gene_id \"chr37.g7446\"; transcript_id \"chr37.g7446.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19617004', '19617159', '.', '-', '.', 'gene_id \"chr37.g7447\"; transcript_id \"chr37.g7447.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19619179', '19619707', '.', '+', '.', 'gene_id \"chr37.g7449\"; transcript_id \"chr37.g7449.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19620631', '19620784', '.', '+', '.', 'gene_id \"chr37.g7450\"; transcript_id \"chr37.g7450.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19622710', '19622861', '.', '+', '.', 'gene_id \"chr37.g7452\"; transcript_id \"chr37.g7452.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19667948', '19668113', '.', '-', '.', 'gene_id \"chr37.g7459\"; transcript_id \"chr37.g7459.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19709091', '19709482', '.', '-', '.', 'gene_id \"chr37.g7463\"; transcript_id \"chr37.g7463.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19736536', '19736797', '.', '-', '.', 'gene_id \"chr37.g7467\"; transcript_id \"chr37.g7467.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19830190', '19830437', '.', '+', '.', 'gene_id \"chr37.g7475\"; transcript_id \"chr37.g7475.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19830190', '19830447', '.', '-', '.', 'gene_id \"chr37.g7476\"; transcript_id \"chr37.g7476.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19839435', '19839586', '.', '-', '.', 'gene_id \"chr37.g7481\"; transcript_id \"chr37.g7481.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19839692', '19839898', '.', '-', '.', 'gene_id \"chr37.g7482\"; transcript_id \"chr37.g7482.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19855089', '19855594', '.', '-', '.', 'gene_id \"chr37.g7483\"; transcript_id \"chr37.g7483.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19855613', '19855786', '.', '-', '.', 'gene_id \"chr37.g7484\"; transcript_id \"chr37.g7484.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19869886', '19870136', '.', '-', '.', 'gene_id \"chr37.g7487\"; transcript_id \"chr37.g7487.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19871077', '19871294', '.', '-', '.', 'gene_id \"chr37.g7489\"; transcript_id \"chr37.g7489.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19870904', '19871081', '.', '-', '.', 'gene_id \"chr37.g7490\"; transcript_id \"chr37.g7490.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19874647', '19874881', '.', '-', '.', 'gene_id \"chr37.g7494\"; transcript_id \"chr37.g7494.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19879836', '19879994', '.', '-', '.', 'gene_id \"chr37.g7495\"; transcript_id \"chr37.g7495.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19880848', '19881039', '.', '+', '.', 'gene_id \"chr37.g7498\"; transcript_id \"chr37.g7498.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19889373', '19889683', '.', '+', '.', 'gene_id \"chr37.g7501\"; transcript_id \"chr37.g7501.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19890122', '19890815', '.', '-', '.', 'gene_id \"chr37.g7504\"; transcript_id \"chr37.g7504.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19890819', '19891201', '.', '+', '.', 'gene_id \"chr37.g7505\"; transcript_id \"chr37.g7505.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19894231', '19894436', '.', '-', '.', 'gene_id \"chr37.g7507\"; transcript_id \"chr37.g7507.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19923597', '19923775', '.', '+', '.', 'gene_id \"chr37.g7508\"; transcript_id \"chr37.g7508.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19924906', '19925523', '.', '-', '.', 'gene_id \"chr37.g7509\"; transcript_id \"chr37.g7509.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19925923', '19926220', '.', '-', '.', 'gene_id \"chr37.g7511\"; transcript_id \"chr37.g7511.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19927881', '19928033', '.', '-', '.', 'gene_id \"chr37.g7514\"; transcript_id \"chr37.g7514.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19928477', '19928653', '.', '-', '.', 'gene_id \"chr37.g7515\"; transcript_id \"chr37.g7515.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19929516', '19930071', '.', '+', '.', 'gene_id \"chr37.g7516\"; transcript_id \"chr37.g7516.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19930282', '19930482', '.', '+', '.', 'gene_id \"chr37.g7518\"; transcript_id \"chr37.g7518.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19944529', '19944961', '.', '-', '.', 'gene_id \"chr37.g7527\"; transcript_id \"chr37.g7527.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19945960', '19946168', '.', '-', '.', 'gene_id \"chr37.g7528\"; transcript_id \"chr37.g7528.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19946163', '19946431', '.', '-', '.', 'gene_id \"chr37.g7529\"; transcript_id \"chr37.g7529.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19947961', '19948118', '.', '-', '.', 'gene_id \"chr37.g7530\"; transcript_id \"chr37.g7530.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19956573', '19956756', '.', '-', '.', 'gene_id \"chr37.g7534\"; transcript_id \"chr37.g7534.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19959620', '19959855', '.', '+', '.', 'gene_id \"chr37.g7537\"; transcript_id \"chr37.g7537.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19961824', '19962018', '.', '+', '.', 'gene_id \"chr37.g7538\"; transcript_id \"chr37.g7538.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19970603', '19970782', '.', '-', '.', 'gene_id \"chr37.g7543\"; transcript_id \"chr37.g7543.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19983897', '19984067', '.', '+', '.', 'gene_id \"chr37.g7546\"; transcript_id \"chr37.g7546.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '19999586', '19999764', '.', '-', '.', 'gene_id \"chr37.g7550\"; transcript_id \"chr37.g7550.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20000170', '20000428', '.', '-', '.', 'gene_id \"chr37.g7551\"; transcript_id \"chr37.g7551.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20021621', '20022222', '.', '+', '.', 'gene_id \"chr37.g7556\"; transcript_id \"chr37.g7556.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20024298', '20024669', '.', '+', '.', 'gene_id \"chr37.g7557\"; transcript_id \"chr37.g7557.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20024785', '20024969', '.', '-', '.', 'gene_id \"chr37.g7558\"; transcript_id \"chr37.g7558.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20031416', '20031604', '.', '+', '.', 'gene_id \"chr37.g7560\"; transcript_id \"chr37.g7560.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20033184', '20033468', '.', '+', '.', 'gene_id \"chr37.g7563\"; transcript_id \"chr37.g7563.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20072112', '20072270', '.', '-', '.', 'gene_id \"chr37.g7565\"; transcript_id \"chr37.g7565.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20153988', '20154146', '.', '-', '.', 'gene_id \"chr37.g7576\"; transcript_id \"chr37.g7576.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20154502', '20154538', '.', '+', '.', 'gene_id \"chr37.g7577\"; transcript_id \"chr37.g7577.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20248634', '20248692', '.', '+', '.', 'gene_id \"chr37.g7588\"; transcript_id \"chr37.g7588.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20248670', '20248885', '.', '-', '.', 'gene_id \"chr37.g7589\"; transcript_id \"chr37.g7589.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20274024', '20274446', '.', '-', '.', 'gene_id \"chr37.g7595\"; transcript_id \"chr37.g7595.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20319156', '20319332', '.', '-', '.', 'gene_id \"chr37.g7601\"; transcript_id \"chr37.g7601.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20335983', '20336150', '.', '-', '.', 'gene_id \"chr37.g7602\"; transcript_id \"chr37.g7602.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20337132', '20337331', '.', '+', '.', 'gene_id \"chr37.g7603\"; transcript_id \"chr37.g7603.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20337464', '20337620', '.', '+', '.', 'gene_id \"chr37.g7604\"; transcript_id \"chr37.g7604.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20342054', '20342241', '.', '+', '.', 'gene_id \"chr37.g7605\"; transcript_id \"chr37.g7605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20344788', '20344967', '.', '-', '.', 'gene_id \"chr37.g7606\"; transcript_id \"chr37.g7606.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20411003', '20412912', '.', '-', '.', 'gene_id \"chr37.g7610\"; transcript_id \"chr37.g7610.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20407340', '20411004', '.', '+', '.', 'gene_id \"chr37.g7611\"; transcript_id \"chr37.g7611.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20422506', '20422686', '.', '+', '.', 'gene_id \"chr37.g7614\"; transcript_id \"chr37.g7614.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20424545', '20424986', '.', '+', '.', 'gene_id \"chr37.g7615\"; transcript_id \"chr37.g7615.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20431350', '20431568', '.', '+', '.', 'gene_id \"chr37.g7618\"; transcript_id \"chr37.g7618.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20432062', '20432257', '.', '+', '.', 'gene_id \"chr37.g7619\"; transcript_id \"chr37.g7619.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20439984', '20440364', '.', '-', '.', 'gene_id \"chr37.g7630\"; transcript_id \"chr37.g7630.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20440604', '20440893', '.', '-', '.', 'gene_id \"chr37.g7632\"; transcript_id \"chr37.g7632.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20441168', '20441689', '.', '-', '.', 'gene_id \"chr37.g7633\"; transcript_id \"chr37.g7633.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20443191', '20443669', '.', '-', '.', 'gene_id \"chr37.g7634\"; transcript_id \"chr37.g7634.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20443716', '20444166', '.', '+', '.', 'gene_id \"chr37.g7635\"; transcript_id \"chr37.g7635.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20444150', '20444331', '.', '-', '.', 'gene_id \"chr37.g7636\"; transcript_id \"chr37.g7636.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20444650', '20444902', '.', '-', '.', 'gene_id \"chr37.g7637\"; transcript_id \"chr37.g7637.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20448937', '20449106', '.', '-', '.', 'gene_id \"chr37.g7642\"; transcript_id \"chr37.g7642.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20453446', '20453622', '.', '-', '.', 'gene_id \"chr37.g7644\"; transcript_id \"chr37.g7644.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20453622', '20453823', '.', '+', '.', 'gene_id \"chr37.g7645\"; transcript_id \"chr37.g7645.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20453804', '20454025', '.', '-', '.', 'gene_id \"chr37.g7646\"; transcript_id \"chr37.g7646.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20455791', '20456061', '.', '-', '.', 'gene_id \"chr37.g7648\"; transcript_id \"chr37.g7648.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20456051', '20456336', '.', '+', '.', 'gene_id \"chr37.g7649\"; transcript_id \"chr37.g7649.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20461191', '20461381', '.', '-', '.', 'gene_id \"chr37.g7651\"; transcript_id \"chr37.g7651.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20461377', '20461733', '.', '-', '.', 'gene_id \"chr37.g7652\"; transcript_id \"chr37.g7652.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20471626', '20471751', '.', '+', '.', 'gene_id \"chr37.g7655\"; transcript_id \"chr37.g7655.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20482734', '20483048', '.', '+', '.', 'gene_id \"chr37.g7658\"; transcript_id \"chr37.g7658.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20484939', '20485484', '.', '-', '.', 'gene_id \"chr37.g7660\"; transcript_id \"chr37.g7660.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20516220', '20516541', '.', '-', '.', 'gene_id \"chr37.g7668\"; transcript_id \"chr37.g7668.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20518140', '20518469', '.', '+', '.', 'gene_id \"chr37.g7670\"; transcript_id \"chr37.g7670.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20528967', '20529198', '.', '+', '.', 'gene_id \"chr37.g7673\"; transcript_id \"chr37.g7673.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20529758', '20530124', '.', '+', '.', 'gene_id \"chr37.g7674\"; transcript_id \"chr37.g7674.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20530702', '20531007', '.', '+', '.', 'gene_id \"chr37.g7676\"; transcript_id \"chr37.g7676.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20531025', '20531343', '.', '+', '.', 'gene_id \"chr37.g7678\"; transcript_id \"chr37.g7678.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20532645', '20533116', '.', '-', '.', 'gene_id \"chr37.g7679\"; transcript_id \"chr37.g7679.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20533942', '20534291', '.', '-', '.', 'gene_id \"chr37.g7681\"; transcript_id \"chr37.g7681.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20534278', '20534511', '.', '+', '.', 'gene_id \"chr37.g7682\"; transcript_id \"chr37.g7682.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20534652', '20534808', '.', '+', '.', 'gene_id \"chr37.g7683\"; transcript_id \"chr37.g7683.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20534923', '20535175', '.', '-', '.', 'gene_id \"chr37.g7684\"; transcript_id \"chr37.g7684.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20535167', '20535340', '.', '-', '.', 'gene_id \"chr37.g7685\"; transcript_id \"chr37.g7685.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20545839', '20546006', '.', '-', '.', 'gene_id \"chr37.g7688\"; transcript_id \"chr37.g7688.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20552660', '20553096', '.', '-', '.', 'gene_id \"chr37.g7690\"; transcript_id \"chr37.g7690.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20558499', '20558804', '.', '-', '.', 'gene_id \"chr37.g7694\"; transcript_id \"chr37.g7694.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20558819', '20559040', '.', '-', '.', 'gene_id \"chr37.g7695\"; transcript_id \"chr37.g7695.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20560349', '20560528', '.', '+', '.', 'gene_id \"chr37.g7696\"; transcript_id \"chr37.g7696.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20562489', '20562654', '.', '-', '.', 'gene_id \"chr37.g7700\"; transcript_id \"chr37.g7700.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20653324', '20653360', '.', '-', '.', 'gene_id \"chr37.g7712\"; transcript_id \"chr37.g7712.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20660709', '20661035', '.', '-', '.', 'gene_id \"chr37.g7713\"; transcript_id \"chr37.g7713.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20661038', '20661342', '.', '-', '.', 'gene_id \"chr37.g7714\"; transcript_id \"chr37.g7714.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20821080', '20821708', '.', '+', '.', 'gene_id \"chr37.g7716\"; transcript_id \"chr37.g7716.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20821080', '20821208', '.', '+', '.', 'gene_id \"chr37.g7716\"; transcript_id \"chr37.g7716.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20684012', '20684226', '.', '-', '.', 'gene_id \"chr37.g7720\"; transcript_id \"chr37.g7720.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20684822', '20685193', '.', '-', '.', 'gene_id \"chr37.g7722\"; transcript_id \"chr37.g7722.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20685226', '20685869', '.', '+', '.', 'gene_id \"chr37.g7723\"; transcript_id \"chr37.g7723.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20685873', '20686111', '.', '-', '.', 'gene_id \"chr37.g7724\"; transcript_id \"chr37.g7724.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20686121', '20686572', '.', '+', '.', 'gene_id \"chr37.g7725\"; transcript_id \"chr37.g7725.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20686730', '20686942', '.', '-', '.', 'gene_id \"chr37.g7726\"; transcript_id \"chr37.g7726.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20687080', '20687257', '.', '-', '.', 'gene_id \"chr37.g7727\"; transcript_id \"chr37.g7727.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20687680', '20687994', '.', '-', '.', 'gene_id \"chr37.g7729\"; transcript_id \"chr37.g7729.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20689067', '20689389', '.', '+', '.', 'gene_id \"chr37.g7731\"; transcript_id \"chr37.g7731.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20688057', '20688333', '.', '+', '.', 'gene_id \"chr37.g7732\"; transcript_id \"chr37.g7732.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20690046', '20690478', '.', '-', '.', 'gene_id \"chr37.g7734\"; transcript_id \"chr37.g7734.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20690480', '20690739', '.', '-', '.', 'gene_id \"chr37.g7735\"; transcript_id \"chr37.g7735.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20696851', '20697130', '.', '+', '.', 'gene_id \"chr37.g7740\"; transcript_id \"chr37.g7740.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20697973', '20698341', '.', '-', '.', 'gene_id \"chr37.g7743\"; transcript_id \"chr37.g7743.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20700441', '20700658', '.', '+', '.', 'gene_id \"chr37.g7746\"; transcript_id \"chr37.g7746.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20701725', '20702022', '.', '-', '.', 'gene_id \"chr37.g7747\"; transcript_id \"chr37.g7747.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20702687', '20703196', '.', '+', '.', 'gene_id \"chr37.g7749\"; transcript_id \"chr37.g7749.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20703476', '20703667', '.', '+', '.', 'gene_id \"chr37.g7751\"; transcript_id \"chr37.g7751.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '41072249', '41072292', '.', '-', '.', 'gene_id \"chr23.g15979\"; transcript_id \"chr23.g15979.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20748227', '20748568', '.', '-', '.', 'gene_id \"chr37.g7755\"; transcript_id \"chr37.g7755.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20749895', '20750049', '.', '-', '.', 'gene_id \"chr37.g7756\"; transcript_id \"chr37.g7756.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20763153', '20763309', '.', '-', '.', 'gene_id \"chr37.g7760\"; transcript_id \"chr37.g7760.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20770449', '20770754', '.', '-', '.', 'gene_id \"chr37.g7761\"; transcript_id \"chr37.g7761.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20835292', '20835527', '.', '+', '.', 'gene_id \"chr37.g7767\"; transcript_id \"chr37.g7767.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20836393', '20836597', '.', '-', '.', 'gene_id \"chr37.g7768\"; transcript_id \"chr37.g7768.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20837388', '20837660', '.', '+', '.', 'gene_id \"chr37.g7769\"; transcript_id \"chr37.g7769.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20837648', '20837843', '.', '+', '.', 'gene_id \"chr37.g7770\"; transcript_id \"chr37.g7770.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20840408', '20840669', '.', '-', '.', 'gene_id \"chr37.g7771\"; transcript_id \"chr37.g7771.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20849027', '20849118', '.', '+', '.', 'gene_id \"chr37.g7774\"; transcript_id \"chr37.g7774.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20858349', '20858517', '.', '-', '.', 'gene_id \"chr37.g7779\"; transcript_id \"chr37.g7779.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20859464', '20859661', '.', '+', '.', 'gene_id \"chr37.g7780\"; transcript_id \"chr37.g7780.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20862979', '20863217', '.', '+', '.', 'gene_id \"chr37.g7783\"; transcript_id \"chr37.g7783.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20862698', '20863000', '.', '-', '.', 'gene_id \"chr37.g7784\"; transcript_id \"chr37.g7784.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20866660', '20866903', '.', '-', '.', 'gene_id \"chr37.g7787\"; transcript_id \"chr37.g7787.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20867227', '20867452', '.', '-', '.', 'gene_id \"chr37.g7788\"; transcript_id \"chr37.g7788.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20867823', '20869006', '.', '+', '.', 'gene_id \"chr37.g7789\"; transcript_id \"chr37.g7789.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20869260', '20870376', '.', '-', '.', 'gene_id \"chr37.g7790\"; transcript_id \"chr37.g7790.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20870896', '20872085', '.', '+', '.', 'gene_id \"chr37.g7792\"; transcript_id \"chr37.g7792.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20870896', '20871440', '.', '+', '.', 'gene_id \"chr37.g7792\"; transcript_id \"chr37.g7792.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '41270526', '41270574', '.', '+', '.', 'gene_id \"chr23.g16015\"; transcript_id \"chr23.g16015.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20908233', '20908400', '.', '-', '.', 'gene_id \"chr37.g7799\"; transcript_id \"chr37.g7799.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20912003', '20912053', '.', '+', '.', 'gene_id \"chr37.g7801\"; transcript_id \"chr37.g7801.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20931952', '20932106', '.', '+', '.', 'gene_id \"chr37.g7807\"; transcript_id \"chr37.g7807.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20931973', '20932263', '.', '+', '.', 'gene_id \"chr37.g7807\"; transcript_id \"chr37.g7807.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20932415', '20932569', '.', '+', '.', 'gene_id \"chr37.g7809\"; transcript_id \"chr37.g7809.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20941827', '20941981', '.', '+', '.', 'gene_id \"chr37.g7813\"; transcript_id \"chr37.g7813.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '20942689', '20942852', '.', '+', '.', 'gene_id \"chr37.g7815\"; transcript_id \"chr37.g7815.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21154661', '21154844', '.', '+', '.', 'gene_id \"chr37.g7829\"; transcript_id \"chr37.g7829.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21417236', '21417440', '.', '+', '.', 'gene_id \"chr37.g7845\"; transcript_id \"chr37.g7845.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21465411', '21465616', '.', '-', '.', 'gene_id \"chr37.g7848\"; transcript_id \"chr37.g7848.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21589193', '21589368', '.', '-', '.', 'gene_id \"chr37.g7858\"; transcript_id \"chr37.g7858.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21593156', '21593385', '.', '-', '.', 'gene_id \"chr37.g7860\"; transcript_id \"chr37.g7860.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21596079', '21596256', '.', '-', '.', 'gene_id \"chr37.g7862\"; transcript_id \"chr37.g7862.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21596474', '21596649', '.', '-', '.', 'gene_id \"chr37.g7864\"; transcript_id \"chr37.g7864.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21740223', '21743439', '.', '+', '.', 'gene_id \"chr37.g7872\"; transcript_id \"chr37.g7872.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21785300', '21785493', '.', '-', '.', 'gene_id \"chr37.g7875\"; transcript_id \"chr37.g7875.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21856669', '21856899', '.', '-', '.', 'gene_id \"chr37.g7880\"; transcript_id \"chr37.g7880.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21858166', '21858396', '.', '-', '.', 'gene_id \"chr37.g7882\"; transcript_id \"chr37.g7882.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21858405', '21858602', '.', '-', '.', 'gene_id \"chr37.g7883\"; transcript_id \"chr37.g7883.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21863202', '21863546', '.', '+', '.', 'gene_id \"chr37.g7886\"; transcript_id \"chr37.g7886.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21869706', '21869888', '.', '+', '.', 'gene_id \"chr37.g7890\"; transcript_id \"chr37.g7890.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21874028', '21874246', '.', '+', '.', 'gene_id \"chr37.g7892\"; transcript_id \"chr37.g7892.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21897626', '21897803', '.', '-', '.', 'gene_id \"chr37.g7895\"; transcript_id \"chr37.g7895.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21906060', '21906260', '.', '-', '.', 'gene_id \"chr37.g7897\"; transcript_id \"chr37.g7897.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21910013', '21910172', '.', '-', '.', 'gene_id \"chr37.g7898\"; transcript_id \"chr37.g7898.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21919594', '21919778', '.', '+', '.', 'gene_id \"chr37.g7904\"; transcript_id \"chr37.g7904.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21922115', '21922309', '.', '+', '.', 'gene_id \"chr37.g7906\"; transcript_id \"chr37.g7906.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21922324', '21922712', '.', '+', '.', 'gene_id \"chr37.g7907\"; transcript_id \"chr37.g7907.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21923217', '21923821', '.', '-', '.', 'gene_id \"chr37.g7908\"; transcript_id \"chr37.g7908.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21924865', '21925188', '.', '+', '.', 'gene_id \"chr37.g7910\"; transcript_id \"chr37.g7910.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21925995', '21926279', '.', '+', '.', 'gene_id \"chr37.g7912\"; transcript_id \"chr37.g7912.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21929730', '21929919', '.', '+', '.', 'gene_id \"chr37.g7915\"; transcript_id \"chr37.g7915.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21934812', '21935016', '.', '-', '.', 'gene_id \"chr37.g7916\"; transcript_id \"chr37.g7916.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21936868', '21937091', '.', '+', '.', 'gene_id \"chr37.g7918\"; transcript_id \"chr37.g7918.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '21937895', '21938090', '.', '-', '.', 'gene_id \"chr37.g7920\"; transcript_id \"chr37.g7920.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22051693', '22051858', '.', '+', '.', 'gene_id \"chr37.g7926\"; transcript_id \"chr37.g7926.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22144591', '22144909', '.', '+', '.', 'gene_id \"chr37.g7939\"; transcript_id \"chr37.g7939.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22144914', '22145253', '.', '-', '.', 'gene_id \"chr37.g7940\"; transcript_id \"chr37.g7940.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22154861', '22155032', '.', '-', '.', 'gene_id \"chr37.g7943\"; transcript_id \"chr37.g7943.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22155140', '22155507', '.', '-', '.', 'gene_id \"chr37.g7944\"; transcript_id \"chr37.g7944.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22155489', '22155657', '.', '-', '.', 'gene_id \"chr37.g7944\"; transcript_id \"chr37.g7944.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22155489', '22155624', '.', '-', '.', 'gene_id \"chr37.g7944\"; transcript_id \"chr37.g7944.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22155489', '22155624', '.', '+', '.', 'gene_id \"chr37.g7945\"; transcript_id \"chr37.g7945.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22166464', '22166573', '.', '+', '.', 'gene_id \"chr37.g7946\"; transcript_id \"chr37.g7946.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '41821550', '41821586', '.', '+', '.', 'gene_id \"chr23.g16188\"; transcript_id \"chr23.g16188.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22189346', '22189652', '.', '+', '.', 'gene_id \"chr37.g7949\"; transcript_id \"chr37.g7949.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22189346', '22189527', '.', '+', '.', 'gene_id \"chr37.g7949\"; transcript_id \"chr37.g7949.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22191280', '22191446', '.', '+', '.', 'gene_id \"chr37.g7952\"; transcript_id \"chr37.g7952.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22219092', '22219220', '.', '+', '.', 'gene_id \"chr37.g7956\"; transcript_id \"chr37.g7956.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22219201', '22219374', '.', '-', '.', 'gene_id \"chr37.g7957\"; transcript_id \"chr37.g7957.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22329419', '22329608', '.', '+', '.', 'gene_id \"chr37.g7968\"; transcript_id \"chr37.g7968.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22329584', '22329894', '.', '+', '.', 'gene_id \"chr37.g7968\"; transcript_id \"chr37.g7968.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22356688', '22357586', '.', '+', '.', 'gene_id \"chr37.g7970\"; transcript_id \"chr37.g7970.i5\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22335573', '22335806', '.', '-', '.', 'gene_id \"chr37.g7971\"; transcript_id \"chr37.g7971.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22339545', '22340294', '.', '-', '.', 'gene_id \"chr37.g7972\"; transcript_id \"chr37.g7972.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22341477', '22341903', '.', '-', '.', 'gene_id \"chr37.g7973\"; transcript_id \"chr37.g7973.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22333542', '22333710', '.', '+', '.', 'gene_id \"chr37.g7977\"; transcript_id \"chr37.g7977.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22337080', '22337563', '.', '-', '.', 'gene_id \"chr37.g7980\"; transcript_id \"chr37.g7980.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22338017', '22338308', '.', '-', '.', 'gene_id \"chr37.g7981\"; transcript_id \"chr37.g7981.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '41996214', '41996270', '.', '+', '.', 'gene_id \"chr23.g16219\"; transcript_id \"chr23.g16219.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22338379', '22338694', '.', '-', '.', 'gene_id \"chr37.g7982\"; transcript_id \"chr37.g7982.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22339026', '22339262', '.', '-', '.', 'gene_id \"chr37.g7983\"; transcript_id \"chr37.g7983.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22342228', '22342658', '.', '-', '.', 'gene_id \"chr37.g7985\"; transcript_id \"chr37.g7985.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22354351', '22354490', '.', '+', '.', 'gene_id \"chr37.g7992\"; transcript_id \"chr37.g7992.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22355660', '22355898', '.', '+', '.', 'gene_id \"chr37.g7995\"; transcript_id \"chr37.g7995.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22374302', '22375122', '.', '-', '.', 'gene_id \"chr37.g7997\"; transcript_id \"chr37.g7997.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22377558', '22378732', '.', '-', '.', 'gene_id \"chr37.g7997\"; transcript_id \"chr37.g7997.i11\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22397308', '22398158', '.', '-', '.', 'gene_id \"chr37.g7997\"; transcript_id \"chr37.g7997.i17\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22370703', '22372344', '.', '-', '.', 'gene_id \"chr37.g7997\"; transcript_id \"chr37.g7997.i20\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22395425', '22397242', '.', '-', '.', 'gene_id \"chr37.g7997\"; transcript_id \"chr37.g7997.i21\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22394790', '22395283', '.', '+', '.', 'gene_id \"chr37.g7998\"; transcript_id \"chr37.g7998.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22393173', '22394399', '.', '+', '.', 'gene_id \"chr37.g8001\"; transcript_id \"chr37.g8001.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22389855', '22390171', '.', '-', '.', 'gene_id \"chr37.g8002\"; transcript_id \"chr37.g8002.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22419841', '22420379', '.', '-', '.', 'gene_id \"chr37.g8006\"; transcript_id \"chr37.g8006.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr37', 'PASA', 'exon', '22380883', '22381408', '.', '+', '.', 'gene_id \"chr37.g8008\"; transcript_id \"chr37.g8008.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22388823', '22389013', '.', '-', '.', 'gene_id \"chr37.g8011\"; transcript_id \"chr37.g8011.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22381545', '22381713', '.', '-', '.', 'gene_id \"chr37.g8014\"; transcript_id \"chr37.g8014.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22422966', '22423162', '.', '-', '.', 'gene_id \"chr37.g8016\"; transcript_id \"chr37.g8016.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22429316', '22429539', '.', '+', '.', 'gene_id \"chr37.g8018\"; transcript_id \"chr37.g8018.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22372626', '22372960', '.', '-', '.', 'gene_id \"chr37.g8020\"; transcript_id \"chr37.g8020.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22375242', '22377463', '.', '+', '.', 'gene_id \"chr37.g8021\"; transcript_id \"chr37.g8021.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22369070', '22369490', '.', '-', '.', 'gene_id \"chr37.g8022\"; transcript_id \"chr37.g8022.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22384434', '22384809', '.', '-', '.', 'gene_id \"chr37.g8024\"; transcript_id \"chr37.g8024.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22398547', '22398660', '.', '-', '.', 'gene_id \"chr37.g8025\"; transcript_id \"chr37.g8025.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22398547', '22398666', '.', '-', '.', 'gene_id \"chr37.g8025\"; transcript_id \"chr37.g8025.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22398943', '22399378', '.', '-', '.', 'gene_id \"chr37.g8027\"; transcript_id \"chr37.g8027.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22400299', '22400594', '.', '+', '.', 'gene_id \"chr37.g8028\"; transcript_id \"chr37.g8028.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22401133', '22401331', '.', '+', '.', 'gene_id \"chr37.g8029\"; transcript_id \"chr37.g8029.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22402199', '22402369', '.', '-', '.', 'gene_id \"chr37.g8030\"; transcript_id \"chr37.g8030.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22404224', '22404483', '.', '+', '.', 'gene_id \"chr37.g8032\"; transcript_id \"chr37.g8032.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22420713', '22420954', '.', '-', '.', 'gene_id \"chr37.g8033\"; transcript_id \"chr37.g8033.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22431648', '22431853', '.', '+', '.', 'gene_id \"chr37.g8037\"; transcript_id \"chr37.g8037.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22433052', '22433305', '.', '-', '.', 'gene_id \"chr37.g8038\"; transcript_id \"chr37.g8038.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22565773', '22565825', '.', '+', '.', 'gene_id \"chr37.g8051\"; transcript_id \"chr37.g8051.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22568751', '22568904', '.', '-', '.', 'gene_id \"chr37.g8053\"; transcript_id \"chr37.g8053.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22569491', '22569941', '.', '+', '.', 'gene_id \"chr37.g8055\"; transcript_id \"chr37.g8055.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22570764', '22571027', '.', '-', '.', 'gene_id \"chr37.g8056\"; transcript_id \"chr37.g8056.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22575248', '22575494', '.', '+', '.', 'gene_id \"chr37.g8058\"; transcript_id \"chr37.g8058.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22585290', '22585573', '.', '-', '.', 'gene_id \"chr37.g8059\"; transcript_id \"chr37.g8059.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22583898', '22584089', '.', '+', '.', 'gene_id \"chr37.g8062\"; transcript_id \"chr37.g8062.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22584325', '22584605', '.', '+', '.', 'gene_id \"chr37.g8063\"; transcript_id \"chr37.g8063.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22585878', '22586060', '.', '+', '.', 'gene_id \"chr37.g8065\"; transcript_id \"chr37.g8065.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22586126', '22586440', '.', '-', '.', 'gene_id \"chr37.g8066\"; transcript_id \"chr37.g8066.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22586652', '22586837', '.', '-', '.', 'gene_id \"chr37.g8067\"; transcript_id \"chr37.g8067.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22589450', '22589852', '.', '-', '.', 'gene_id \"chr37.g8070\"; transcript_id \"chr37.g8070.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22591090', '22591260', '.', '-', '.', 'gene_id \"chr37.g8072\"; transcript_id \"chr37.g8072.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '42750408', '42750583', '.', '+', '.', 'gene_id \"chr23.g16286\"; transcript_id \"chr23.g16286.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22592642', '22592911', '.', '-', '.', 'gene_id \"chr37.g8073\"; transcript_id \"chr37.g8073.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22594480', '22594706', '.', '-', '.', 'gene_id \"chr37.g8075\"; transcript_id \"chr37.g8075.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22594706', '22595332', '.', '+', '.', 'gene_id \"chr37.g8076\"; transcript_id \"chr37.g8076.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '42755616', '42755996', '.', '-', '.', 'gene_id \"chr23.g16291\"; transcript_id \"chr23.g16291.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22596833', '22596988', '.', '+', '.', 'gene_id \"chr37.g8080\"; transcript_id \"chr37.g8080.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22596976', '22597483', '.', '+', '.', 'gene_id \"chr37.g8081\"; transcript_id \"chr37.g8081.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22597482', '22597649', '.', '-', '.', 'gene_id \"chr37.g8082\"; transcript_id \"chr37.g8082.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22599316', '22599531', '.', '-', '.', 'gene_id \"chr37.g8084\"; transcript_id \"chr37.g8084.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22599745', '22599956', '.', '-', '.', 'gene_id \"chr37.g8085\"; transcript_id \"chr37.g8085.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22600957', '22601127', '.', '+', '.', 'gene_id \"chr37.g8086\"; transcript_id \"chr37.g8086.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22601459', '22601818', '.', '+', '.', 'gene_id \"chr37.g8088\"; transcript_id \"chr37.g8088.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22602105', '22602295', '.', '+', '.', 'gene_id \"chr37.g8089\"; transcript_id \"chr37.g8089.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22602355', '22602525', '.', '-', '.', 'gene_id \"chr37.g8090\"; transcript_id \"chr37.g8090.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22602583', '22602833', '.', '-', '.', 'gene_id \"chr37.g8091\"; transcript_id \"chr37.g8091.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22606447', '22607051', '.', '-', '.', 'gene_id \"chr37.g8093\"; transcript_id \"chr37.g8093.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22619852', '22620096', '.', '+', '.', 'gene_id \"chr37.g8097\"; transcript_id \"chr37.g8097.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22623285', '22623456', '.', '-', '.', 'gene_id \"chr37.g8098\"; transcript_id \"chr37.g8098.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22642201', '22642383', '.', '-', '.', 'gene_id \"chr37.g8099\"; transcript_id \"chr37.g8099.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22642642', '22642850', '.', '-', '.', 'gene_id \"chr37.g8100\"; transcript_id \"chr37.g8100.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22642998', '22643170', '.', '-', '.', 'gene_id \"chr37.g8101\"; transcript_id \"chr37.g8101.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22643449', '22643602', '.', '+', '.', 'gene_id \"chr37.g8103\"; transcript_id \"chr37.g8103.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22673287', '22673453', '.', '+', '.', 'gene_id \"chr37.g8106\"; transcript_id \"chr37.g8106.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22695266', '22695440', '.', '-', '.', 'gene_id \"chr37.g8113\"; transcript_id \"chr37.g8113.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22737899', '22738278', '.', '-', '.', 'gene_id \"chr37.g8120\"; transcript_id \"chr37.g8120.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22739966', '22740146', '.', '+', '.', 'gene_id \"chr37.g8121\"; transcript_id \"chr37.g8121.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22818550', '22818701', '.', '-', '.', 'gene_id \"chr37.g8126\"; transcript_id \"chr37.g8126.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22851537', '22851796', '.', '-', '.', 'gene_id \"chr37.g8127\"; transcript_id \"chr37.g8127.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22866611', '22866797', '.', '-', '.', 'gene_id \"chr37.g8128\"; transcript_id \"chr37.g8128.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22866851', '22867047', '.', '-', '.', 'gene_id \"chr37.g8129\"; transcript_id \"chr37.g8129.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22871971', '22872287', '.', '+', '.', 'gene_id \"chr37.g8131\"; transcript_id \"chr37.g8131.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22870978', '22871079', '.', '+', '.', 'gene_id \"chr37.g8133\"; transcript_id \"chr37.g8133.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22875292', '22875584', '.', '+', '.', 'gene_id \"chr37.g8136\"; transcript_id \"chr37.g8136.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22876713', '22877065', '.', '+', '.', 'gene_id \"chr37.g8138\"; transcript_id \"chr37.g8138.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22878076', '22878229', '.', '-', '.', 'gene_id \"chr37.g8140\"; transcript_id \"chr37.g8140.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22883180', '22883356', '.', '-', '.', 'gene_id \"chr37.g8141\"; transcript_id \"chr37.g8141.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '43503247', '43503285', '.', '+', '.', 'gene_id \"chr23.g16353\"; transcript_id \"chr23.g16353.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22900014', '22900342', '.', '+', '.', 'gene_id \"chr37.g8147\"; transcript_id \"chr37.g8147.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22929289', '22929512', '.', '+', '.', 'gene_id \"chr37.g8151\"; transcript_id \"chr37.g8151.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22929123', '22929303', '.', '-', '.', 'gene_id \"chr37.g8152\"; transcript_id \"chr37.g8152.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '43761697', '43761730', '.', '+', '.', 'gene_id \"chr23.g16364\"; transcript_id \"chr23.g16364.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22944408', '22944611', '.', '+', '.', 'gene_id \"chr37.g8157\"; transcript_id \"chr37.g8157.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22944605', '22944755', '.', '-', '.', 'gene_id \"chr37.g8159\"; transcript_id \"chr37.g8159.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22946370', '22946547', '.', '+', '.', 'gene_id \"chr37.g8164\"; transcript_id \"chr37.g8164.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22947575', '22947743', '.', '-', '.', 'gene_id \"chr37.g8165\"; transcript_id \"chr37.g8165.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22957831', '22958277', '.', '-', '.', 'gene_id \"chr37.g8166\"; transcript_id \"chr37.g8166.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22966585', '22966912', '.', '-', '.', 'gene_id \"chr37.g8167\"; transcript_id \"chr37.g8167.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22966894', '22967399', '.', '+', '.', 'gene_id \"chr37.g8168\"; transcript_id \"chr37.g8168.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22949204', '22950409', '.', '-', '.', 'gene_id \"chr37.g8169\"; transcript_id \"chr37.g8169.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22950538', '22950682', '.', '-', '.', 'gene_id \"chr37.g8171\"; transcript_id \"chr37.g8171.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22951762', '22952096', '.', '+', '.', 'gene_id \"chr37.g8174\"; transcript_id \"chr37.g8174.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22952740', '22953009', '.', '-', '.', 'gene_id \"chr37.g8175\"; transcript_id \"chr37.g8175.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22956592', '22957135', '.', '+', '.', 'gene_id \"chr37.g8179\"; transcript_id \"chr37.g8179.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22957213', '22957391', '.', '-', '.', 'gene_id \"chr37.g8180\"; transcript_id \"chr37.g8180.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22959916', '22960121', '.', '-', '.', 'gene_id \"chr37.g8182\"; transcript_id \"chr37.g8182.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22960155', '22960353', '.', '+', '.', 'gene_id \"chr37.g8183\"; transcript_id \"chr37.g8183.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22969881', '22970194', '.', '+', '.', 'gene_id \"chr37.g8188\"; transcript_id \"chr37.g8188.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22969881', '22970065', '.', '+', '.', 'gene_id \"chr37.g8188\"; transcript_id \"chr37.g8188.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22969753', '22969908', '.', '+', '.', 'gene_id \"chr37.g8188\"; transcript_id \"chr37.g8188.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22976671', '22976809', '.', '-', '.', 'gene_id \"chr37.g8193\"; transcript_id \"chr37.g8193.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22978928', '22979263', '.', '-', '.', 'gene_id \"chr37.g8196\"; transcript_id \"chr37.g8196.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22979688', '22980304', '.', '+', '.', 'gene_id \"chr37.g8197\"; transcript_id \"chr37.g8197.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22980296', '22980534', '.', '-', '.', 'gene_id \"chr37.g8198\"; transcript_id \"chr37.g8198.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22982584', '22982772', '.', '+', '.', 'gene_id \"chr37.g8201\"; transcript_id \"chr37.g8201.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22982979', '22983372', '.', '+', '.', 'gene_id \"chr37.g8202\"; transcript_id \"chr37.g8202.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22984612', '22985309', '.', '-', '.', 'gene_id \"chr37.g8206\"; transcript_id \"chr37.g8206.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22987281', '22987502', '.', '+', '.', 'gene_id \"chr37.g8208\"; transcript_id \"chr37.g8208.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22988885', '22989043', '.', '+', '.', 'gene_id \"chr37.g8211\"; transcript_id \"chr37.g8211.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22988885', '22988973', '.', '+', '.', 'gene_id \"chr37.g8211\"; transcript_id \"chr37.g8211.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22991330', '22991511', '.', '+', '.', 'gene_id \"chr37.g8214\"; transcript_id \"chr37.g8214.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22990702', '22990856', '.', '-', '.', 'gene_id \"chr37.g8215\"; transcript_id \"chr37.g8215.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22992346', '22992634', '.', '-', '.', 'gene_id \"chr37.g8217\"; transcript_id \"chr37.g8217.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22991938', '22992365', '.', '+', '.', 'gene_id \"chr37.g8218\"; transcript_id \"chr37.g8218.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22993578', '22993761', '.', '+', '.', 'gene_id \"chr37.g8219\"; transcript_id \"chr37.g8219.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '22997326', '22997499', '.', '+', '.', 'gene_id \"chr37.g8224\"; transcript_id \"chr37.g8224.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23004760', '23005935', '.', '-', '.', 'gene_id \"chr37.g8228\"; transcript_id \"chr37.g8228.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23006746', '23007506', '.', '-', '.', 'gene_id \"chr37.g8230\"; transcript_id \"chr37.g8230.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23087433', '23088331', '.', '+', '.', 'gene_id \"chr37.g8235\"; transcript_id \"chr37.g8235.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23092861', '23093305', '.', '+', '.', 'gene_id \"chr37.g8237\"; transcript_id \"chr37.g8237.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23088528', '23088799', '.', '-', '.', 'gene_id \"chr37.g8238\"; transcript_id \"chr37.g8238.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23057400', '23058003', '.', '-', '.', 'gene_id \"chr37.g8239\"; transcript_id \"chr37.g8239.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23047545', '23047731', '.', '-', '.', 'gene_id \"chr37.g8241\"; transcript_id \"chr37.g8241.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23041516', '23041801', '.', '+', '.', 'gene_id \"chr37.g8243\"; transcript_id \"chr37.g8243.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23008332', '23008612', '.', '+', '.', 'gene_id \"chr37.g8244\"; transcript_id \"chr37.g8244.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23012742', '23012991', '.', '+', '.', 'gene_id \"chr37.g8246\"; transcript_id \"chr37.g8246.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23013035', '23013340', '.', '-', '.', 'gene_id \"chr37.g8247\"; transcript_id \"chr37.g8247.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23013657', '23013851', '.', '+', '.', 'gene_id \"chr37.g8249\"; transcript_id \"chr37.g8249.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23017935', '23018092', '.', '-', '.', 'gene_id \"chr37.g8251\"; transcript_id \"chr37.g8251.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23018997', '23019541', '.', '+', '.', 'gene_id \"chr37.g8253\"; transcript_id \"chr37.g8253.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23026296', '23026449', '.', '+', '.', 'gene_id \"chr37.g8256\"; transcript_id \"chr37.g8256.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23027348', '23027563', '.', '+', '.', 'gene_id \"chr37.g8257\"; transcript_id \"chr37.g8257.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23034247', '23034599', '.', '+', '.', 'gene_id \"chr37.g8262\"; transcript_id \"chr37.g8262.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23036754', '23036983', '.', '+', '.', 'gene_id \"chr37.g8264\"; transcript_id \"chr37.g8264.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23040507', '23040688', '.', '-', '.', 'gene_id \"chr37.g8265\"; transcript_id \"chr37.g8265.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23041968', '23042152', '.', '-', '.', 'gene_id \"chr37.g8266\"; transcript_id \"chr37.g8266.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23042163', '23042444', '.', '-', '.', 'gene_id \"chr37.g8268\"; transcript_id \"chr37.g8268.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23044311', '23044685', '.', '+', '.', 'gene_id \"chr37.g8270\"; transcript_id \"chr37.g8270.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23052208', '23052473', '.', '+', '.', 'gene_id \"chr37.g8272\"; transcript_id \"chr37.g8272.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23053530', '23053730', '.', '+', '.', 'gene_id \"chr37.g8273\"; transcript_id \"chr37.g8273.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23061762', '23061982', '.', '+', '.', 'gene_id \"chr37.g8280\"; transcript_id \"chr37.g8280.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23075610', '23075862', '.', '+', '.', 'gene_id \"chr37.g8286\"; transcript_id \"chr37.g8286.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23077848', '23078015', '.', '-', '.', 'gene_id \"chr37.g8288\"; transcript_id \"chr37.g8288.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23082634', '23082882', '.', '-', '.', 'gene_id \"chr37.g8294\"; transcript_id \"chr37.g8294.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23085834', '23086398', '.', '-', '.', 'gene_id \"chr37.g8297\"; transcript_id \"chr37.g8297.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23089367', '23089522', '.', '-', '.', 'gene_id \"chr37.g8300\"; transcript_id \"chr37.g8300.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23132060', '23132582', '.', '+', '.', 'gene_id \"chr37.g8306\"; transcript_id \"chr37.g8306.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23133160', '23133360', '.', '-', '.', 'gene_id \"chr37.g8307\"; transcript_id \"chr37.g8307.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23153950', '23154300', '.', '-', '.', 'gene_id \"chr37.g8313\"; transcript_id \"chr37.g8313.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23154489', '23154681', '.', '+', '.', 'gene_id \"chr37.g8314\"; transcript_id \"chr37.g8314.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23154289', '23154497', '.', '+', '.', 'gene_id \"chr37.g8315\"; transcript_id \"chr37.g8315.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23155104', '23155267', '.', '+', '.', 'gene_id \"chr37.g8317\"; transcript_id \"chr37.g8317.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23159411', '23159622', '.', '-', '.', 'gene_id \"chr37.g8319\"; transcript_id \"chr37.g8319.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23160234', '23160385', '.', '-', '.', 'gene_id \"chr37.g8320\"; transcript_id \"chr37.g8320.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23162812', '23162972', '.', '+', '.', 'gene_id \"chr37.g8321\"; transcript_id \"chr37.g8321.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23162966', '23163197', '.', '-', '.', 'gene_id \"chr37.g8322\"; transcript_id \"chr37.g8322.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23236847', '23237541', '.', '-', '.', 'gene_id \"chr37.g8330\"; transcript_id \"chr37.g8330.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23244481', '23244673', '.', '-', '.', 'gene_id \"chr37.g8333\"; transcript_id \"chr37.g8333.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23244979', '23245472', '.', '+', '.', 'gene_id \"chr37.g8334\"; transcript_id \"chr37.g8334.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23248458', '23248663', '.', '+', '.', 'gene_id \"chr37.g8339\"; transcript_id \"chr37.g8339.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23249706', '23249997', '.', '-', '.', 'gene_id \"chr37.g8340\"; transcript_id \"chr37.g8340.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23250256', '23250480', '.', '+', '.', 'gene_id \"chr37.g8342\"; transcript_id \"chr37.g8342.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23250318', '23250480', '.', '+', '.', 'gene_id \"chr37.g8342\"; transcript_id \"chr37.g8342.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23250816', '23251026', '.', '+', '.', 'gene_id \"chr37.g8343\"; transcript_id \"chr37.g8343.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23251314', '23251519', '.', '+', '.', 'gene_id \"chr37.g8344\"; transcript_id \"chr37.g8344.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23251505', '23251724', '.', '-', '.', 'gene_id \"chr37.g8345\"; transcript_id \"chr37.g8345.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23251739', '23251925', '.', '-', '.', 'gene_id \"chr37.g8346\"; transcript_id \"chr37.g8346.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23251921', '23252127', '.', '+', '.', 'gene_id \"chr37.g8347\"; transcript_id \"chr37.g8347.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23252163', '23252731', '.', '-', '.', 'gene_id \"chr37.g8348\"; transcript_id \"chr37.g8348.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23255659', '23255726', '.', '-', '.', 'gene_id \"chr37.g8349\"; transcript_id \"chr37.g8349.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23270210', '23270380', '.', '-', '.', 'gene_id \"chr37.g8351\"; transcript_id \"chr37.g8351.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23272115', '23272348', '.', '-', '.', 'gene_id \"chr37.g8352\"; transcript_id \"chr37.g8352.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23298080', '23298285', '.', '+', '.', 'gene_id \"chr37.g8356\"; transcript_id \"chr37.g8356.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23327356', '23327573', '.', '-', '.', 'gene_id \"chr37.g8358\"; transcript_id \"chr37.g8358.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23322187', '23322398', '.', '-', '.', 'gene_id \"chr37.g8359\"; transcript_id \"chr37.g8359.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23324786', '23325438', '.', '-', '.', 'gene_id \"chr37.g8360\"; transcript_id \"chr37.g8360.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23322028', '23322193', '.', '-', '.', 'gene_id \"chr37.g8361\"; transcript_id \"chr37.g8361.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23320852', '23321662', '.', '+', '.', 'gene_id \"chr37.g8363\"; transcript_id \"chr37.g8363.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23326814', '23327052', '.', '+', '.', 'gene_id \"chr37.g8365\"; transcript_id \"chr37.g8365.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23297707', '23297959', '.', '-', '.', 'gene_id \"chr37.g8367\"; transcript_id \"chr37.g8367.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23279305', '23279509', '.', '-', '.', 'gene_id \"chr37.g8368\"; transcript_id \"chr37.g8368.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23327565', '23327794', '.', '+', '.', 'gene_id \"chr37.g8369\"; transcript_id \"chr37.g8369.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23280559', '23280788', '.', '+', '.', 'gene_id \"chr37.g8372\"; transcript_id \"chr37.g8372.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23280826', '23280981', '.', '+', '.', 'gene_id \"chr37.g8374\"; transcript_id \"chr37.g8374.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23288706', '23288900', '.', '+', '.', 'gene_id \"chr37.g8378\"; transcript_id \"chr37.g8378.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23289733', '23290050', '.', '+', '.', 'gene_id \"chr37.g8379\"; transcript_id \"chr37.g8379.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23292106', '23292318', '.', '-', '.', 'gene_id \"chr37.g8382\"; transcript_id \"chr37.g8382.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23292340', '23292701', '.', '-', '.', 'gene_id \"chr37.g8383\"; transcript_id \"chr37.g8383.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23298404', '23298672', '.', '+', '.', 'gene_id \"chr37.g8388\"; transcript_id \"chr37.g8388.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23300441', '23300654', '.', '+', '.', 'gene_id \"chr37.g8389\"; transcript_id \"chr37.g8389.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23303123', '23303274', '.', '-', '.', 'gene_id \"chr37.g8390\"; transcript_id \"chr37.g8390.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23302833', '23303124', '.', '-', '.', 'gene_id \"chr37.g8391\"; transcript_id \"chr37.g8391.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23304803', '23304971', '.', '-', '.', 'gene_id \"chr37.g8394\"; transcript_id \"chr37.g8394.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23306444', '23306717', '.', '+', '.', 'gene_id \"chr37.g8396\"; transcript_id \"chr37.g8396.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23307700', '23308154', '.', '-', '.', 'gene_id \"chr37.g8398\"; transcript_id \"chr37.g8398.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23313008', '23313510', '.', '-', '.', 'gene_id \"chr37.g8399\"; transcript_id \"chr37.g8399.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23313550', '23313887', '.', '+', '.', 'gene_id \"chr37.g8400\"; transcript_id \"chr37.g8400.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23314078', '23314248', '.', '-', '.', 'gene_id \"chr37.g8401\"; transcript_id \"chr37.g8401.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23314768', '23315005', '.', '-', '.', 'gene_id \"chr37.g8402\"; transcript_id \"chr37.g8402.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23315221', '23315769', '.', '-', '.', 'gene_id \"chr37.g8403\"; transcript_id \"chr37.g8403.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23316296', '23316827', '.', '+', '.', 'gene_id \"chr37.g8405\"; transcript_id \"chr37.g8405.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '44238649', '44239131', '.', '+', '.', 'gene_id \"chr23.g16665\"; transcript_id \"chr23.g16665.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23318927', '23319792', '.', '+', '.', 'gene_id \"chr37.g8407\"; transcript_id \"chr37.g8407.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23319841', '23320028', '.', '-', '.', 'gene_id \"chr37.g8408\"; transcript_id \"chr37.g8408.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23322693', '23323519', '.', '-', '.', 'gene_id \"chr37.g8409\"; transcript_id \"chr37.g8409.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23322534', '23322695', '.', '+', '.', 'gene_id \"chr37.g8410\"; transcript_id \"chr37.g8410.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23324254', '23324671', '.', '+', '.', 'gene_id \"chr37.g8411\"; transcript_id \"chr37.g8411.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23326060', '23326434', '.', '+', '.', 'gene_id \"chr37.g8412\"; transcript_id \"chr37.g8412.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23327083', '23327339', '.', '+', '.', 'gene_id \"chr37.g8413\"; transcript_id \"chr37.g8413.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23334068', '23336162', '.', '+', '.', 'gene_id \"chr37.g8414\"; transcript_id \"chr37.g8414.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23337218', '23337478', '.', '+', '.', 'gene_id \"chr37.g8415\"; transcript_id \"chr37.g8415.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23339501', '23339655', '.', '+', '.', 'gene_id \"chr37.g8422\"; transcript_id \"chr37.g8422.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23341353', '23341493', '.', '-', '.', 'gene_id \"chr37.g8424\"; transcript_id \"chr37.g8424.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23341520', '23342504', '.', '-', '.', 'gene_id \"chr37.g8425\"; transcript_id \"chr37.g8425.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23343169', '23343651', '.', '+', '.', 'gene_id \"chr37.g8426\"; transcript_id \"chr37.g8426.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23345565', '23345824', '.', '+', '.', 'gene_id \"chr37.g8428\"; transcript_id \"chr37.g8428.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23345853', '23346135', '.', '+', '.', 'gene_id \"chr37.g8429\"; transcript_id \"chr37.g8429.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23347764', '23348289', '.', '+', '.', 'gene_id \"chr37.g8431\"; transcript_id \"chr37.g8431.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23348310', '23348556', '.', '+', '.', 'gene_id \"chr37.g8432\"; transcript_id \"chr37.g8432.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23352871', '23353086', '.', '-', '.', 'gene_id \"chr37.g8435\"; transcript_id \"chr37.g8435.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23354616', '23354807', '.', '-', '.', 'gene_id \"chr37.g8436\"; transcript_id \"chr37.g8436.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23373039', '23373877', '.', '-', '.', 'gene_id \"chr37.g8442\"; transcript_id \"chr37.g8442.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23456879', '23457443', '.', '+', '.', 'gene_id \"chr37.g8453\"; transcript_id \"chr37.g8453.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23457151', '23457189', '.', '+', '.', 'gene_id \"chr37.g8453\"; transcript_id \"chr37.g8453.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23458639', '23458856', '.', '-', '.', 'gene_id \"chr37.g8454\"; transcript_id \"chr37.g8454.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23458991', '23459650', '.', '-', '.', 'gene_id \"chr37.g8455\"; transcript_id \"chr37.g8455.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23460361', '23460876', '.', '+', '.', 'gene_id \"chr37.g8458\"; transcript_id \"chr37.g8458.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23462272', '23463096', '.', '-', '.', 'gene_id \"chr37.g8462\"; transcript_id \"chr37.g8462.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23463287', '23464471', '.', '+', '.', 'gene_id \"chr37.g8463\"; transcript_id \"chr37.g8463.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23467404', '23467469', '.', '+', '.', 'gene_id \"chr37.g8464\"; transcript_id \"chr37.g8464.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23477708', '23478441', '.', '+', '.', 'gene_id \"chr37.g8469\"; transcript_id \"chr37.g8469.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23474688', '23477518', '.', '+', '.', 'gene_id \"chr37.g8471\"; transcript_id \"chr37.g8471.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23487094', '23487314', '.', '+', '.', 'gene_id \"chr37.g8476\"; transcript_id \"chr37.g8476.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23506909', '23507108', '.', '+', '.', 'gene_id \"chr37.g8478\"; transcript_id \"chr37.g8478.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23491340', '23492394', '.', '-', '.', 'gene_id \"chr37.g8479\"; transcript_id \"chr37.g8479.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23496149', '23496358', '.', '-', '.', 'gene_id \"chr37.g8486\"; transcript_id \"chr37.g8486.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23499219', '23499478', '.', '+', '.', 'gene_id \"chr37.g8489\"; transcript_id \"chr37.g8489.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23499963', '23500331', '.', '+', '.', 'gene_id \"chr37.g8491\"; transcript_id \"chr37.g8491.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23501455', '23501702', '.', '-', '.', 'gene_id \"chr37.g8493\"; transcript_id \"chr37.g8493.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23501764', '23501951', '.', '-', '.', 'gene_id \"chr37.g8494\"; transcript_id \"chr37.g8494.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23502121', '23502299', '.', '-', '.', 'gene_id \"chr37.g8495\"; transcript_id \"chr37.g8495.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23503694', '23504118', '.', '+', '.', 'gene_id \"chr37.g8498\"; transcript_id \"chr37.g8498.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23512840', '23513044', '.', '+', '.', 'gene_id \"chr37.g8500\"; transcript_id \"chr37.g8500.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23526448', '23526680', '.', '-', '.', 'gene_id \"chr37.g8502\"; transcript_id \"chr37.g8502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23533077', '23533244', '.', '+', '.', 'gene_id \"chr37.g8507\"; transcript_id \"chr37.g8507.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23536387', '23536660', '.', '-', '.', 'gene_id \"chr37.g8508\"; transcript_id \"chr37.g8508.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23547702', '23547886', '.', '+', '.', 'gene_id \"chr37.g8515\"; transcript_id \"chr37.g8515.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23559063', '23559278', '.', '+', '.', 'gene_id \"chr37.g8518\"; transcript_id \"chr37.g8518.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23570524', '23570697', '.', '+', '.', 'gene_id \"chr37.g8521\"; transcript_id \"chr37.g8521.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23598185', '23600077', '.', '+', '.', 'gene_id \"chr37.g8522\"; transcript_id \"chr37.g8522.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23600064', '23600394', '.', '-', '.', 'gene_id \"chr37.g8523\"; transcript_id \"chr37.g8523.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23602558', '23602926', '.', '+', '.', 'gene_id \"chr37.g8524\"; transcript_id \"chr37.g8524.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23647485', '23647768', '.', '-', '.', 'gene_id \"chr37.g8529\"; transcript_id \"chr37.g8529.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23653742', '23654331', '.', '+', '.', 'gene_id \"chr37.g8530\"; transcript_id \"chr37.g8530.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23753522', '23753941', '.', '+', '.', 'gene_id \"chr37.g8542\"; transcript_id \"chr37.g8542.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23866380', '23866553', '.', '+', '.', 'gene_id \"chr37.g8550\"; transcript_id \"chr37.g8550.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '23868057', '23868244', '.', '+', '.', 'gene_id \"chr37.g8552\"; transcript_id \"chr37.g8552.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24071067', '24071361', '.', '-', '.', 'gene_id \"chr37.g8573\"; transcript_id \"chr37.g8573.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24095594', '24095706', '.', '-', '.', 'gene_id \"chr37.g8580\"; transcript_id \"chr37.g8580.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24095594', '24095751', '.', '-', '.', 'gene_id \"chr37.g8580\"; transcript_id \"chr37.g8580.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24095594', '24095706', '.', '+', '.', 'gene_id \"chr37.g8581\"; transcript_id \"chr37.g8581.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24108553', '24108712', '.', '-', '.', 'gene_id \"chr37.g8583\"; transcript_id \"chr37.g8583.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24110099', '24110211', '.', '-', '.', 'gene_id \"chr37.g8584\"; transcript_id \"chr37.g8584.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24168329', '24168527', '.', '+', '.', 'gene_id \"chr37.g8592\"; transcript_id \"chr37.g8592.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24188881', '24189288', '.', '+', '.', 'gene_id \"chr37.g8594\"; transcript_id \"chr37.g8594.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24278780', '24278976', '.', '-', '.', 'gene_id \"chr37.g8601\"; transcript_id \"chr37.g8601.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24316764', '24316922', '.', '-', '.', 'gene_id \"chr37.g8603\"; transcript_id \"chr37.g8603.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24322742', '24323026', '.', '-', '.', 'gene_id \"chr37.g8605\"; transcript_id \"chr37.g8605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24372097', '24372452', '.', '-', '.', 'gene_id \"chr37.g8611\"; transcript_id \"chr37.g8611.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24468739', '24469073', '.', '-', '.', 'gene_id \"chr37.g8621\"; transcript_id \"chr37.g8621.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24470985', '24471236', '.', '-', '.', 'gene_id \"chr37.g8623\"; transcript_id \"chr37.g8623.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24494247', '24494459', '.', '+', '.', 'gene_id \"chr37.g8627\"; transcript_id \"chr37.g8627.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24495144', '24495194', '.', '+', '.', 'gene_id \"chr37.g8629\"; transcript_id \"chr37.g8629.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24495312', '24495776', '.', '-', '.', 'gene_id \"chr37.g8630\"; transcript_id \"chr37.g8630.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24495908', '24497145', '.', '+', '.', 'gene_id \"chr37.g8632\"; transcript_id \"chr37.g8632.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24497235', '24497659', '.', '-', '.', 'gene_id \"chr37.g8633\"; transcript_id \"chr37.g8633.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24498306', '24498474', '.', '-', '.', 'gene_id \"chr37.g8635\"; transcript_id \"chr37.g8635.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24499183', '24499441', '.', '-', '.', 'gene_id \"chr37.g8636\"; transcript_id \"chr37.g8636.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24529476', '24529996', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24510428', '24510696', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i16\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24548119', '24548254', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i17\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24614085', '24614124', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i18\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24614085', '24614123', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i20\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24637413', '24637489', '.', '-', '.', 'gene_id \"chr37.g8637\"; transcript_id \"chr37.g8637.i24\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24515704', '24517426', '.', '+', '.', 'gene_id \"chr37.g8641\"; transcript_id \"chr37.g8641.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24515704', '24517942', '.', '+', '.', 'gene_id \"chr37.g8641\"; transcript_id \"chr37.g8641.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24499453', '24500490', '.', '+', '.', 'gene_id \"chr37.g8643\"; transcript_id \"chr37.g8643.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24582712', '24582892', '.', '+', '.', 'gene_id \"chr37.g8649\"; transcript_id \"chr37.g8649.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24513446', '24513812', '.', '+', '.', 'gene_id \"chr37.g8650\"; transcript_id \"chr37.g8650.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24500745', '24501103', '.', '+', '.', 'gene_id \"chr37.g8652\"; transcript_id \"chr37.g8652.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24599839', '24599991', '.', '+', '.', 'gene_id \"chr37.g8657\"; transcript_id \"chr37.g8657.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24518890', '24519146', '.', '+', '.', 'gene_id \"chr37.g8659\"; transcript_id \"chr37.g8659.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24535455', '24536050', '.', '+', '.', 'gene_id \"chr37.g8660\"; transcript_id \"chr37.g8660.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24529974', '24530640', '.', '-', '.', 'gene_id \"chr37.g8661\"; transcript_id \"chr37.g8661.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24588769', '24589886', '.', '-', '.', 'gene_id \"chr37.g8663\"; transcript_id \"chr37.g8663.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24582439', '24582629', '.', '+', '.', 'gene_id \"chr37.g8664\"; transcript_id \"chr37.g8664.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24509603', '24509952', '.', '-', '.', 'gene_id \"chr37.g8665\"; transcript_id \"chr37.g8665.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24531486', '24531646', '.', '-', '.', 'gene_id \"chr37.g8671\"; transcript_id \"chr37.g8671.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24531831', '24532202', '.', '-', '.', 'gene_id \"chr37.g8672\"; transcript_id \"chr37.g8672.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24533551', '24534211', '.', '+', '.', 'gene_id \"chr37.g8673\"; transcript_id \"chr37.g8673.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24536346', '24536576', '.', '+', '.', 'gene_id \"chr37.g8674\"; transcript_id \"chr37.g8674.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24537276', '24537450', '.', '-', '.', 'gene_id \"chr37.g8676\"; transcript_id \"chr37.g8676.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24538123', '24538285', '.', '-', '.', 'gene_id \"chr37.g8678\"; transcript_id \"chr37.g8678.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24539127', '24539324', '.', '-', '.', 'gene_id \"chr37.g8679\"; transcript_id \"chr37.g8679.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24538757', '24539049', '.', '-', '.', 'gene_id \"chr37.g8681\"; transcript_id \"chr37.g8681.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24539326', '24539727', '.', '+', '.', 'gene_id \"chr37.g8682\"; transcript_id \"chr37.g8682.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24539839', '24540091', '.', '-', '.', 'gene_id \"chr37.g8683\"; transcript_id \"chr37.g8683.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24540105', '24540350', '.', '+', '.', 'gene_id \"chr37.g8684\"; transcript_id \"chr37.g8684.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24540906', '24541291', '.', '-', '.', 'gene_id \"chr37.g8685\"; transcript_id \"chr37.g8685.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24543047', '24543775', '.', '+', '.', 'gene_id \"chr37.g8687\"; transcript_id \"chr37.g8687.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24543753', '24545450', '.', '-', '.', 'gene_id \"chr37.g8688\"; transcript_id \"chr37.g8688.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24545497', '24545952', '.', '-', '.', 'gene_id \"chr37.g8690\"; transcript_id \"chr37.g8690.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24549122', '24549284', '.', '+', '.', 'gene_id \"chr37.g8693\"; transcript_id \"chr37.g8693.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24550394', '24550560', '.', '-', '.', 'gene_id \"chr37.g8696\"; transcript_id \"chr37.g8696.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24552547', '24552863', '.', '-', '.', 'gene_id \"chr37.g8698\"; transcript_id \"chr37.g8698.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24553490', '24554046', '.', '+', '.', 'gene_id \"chr37.g8700\"; transcript_id \"chr37.g8700.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24554655', '24554840', '.', '+', '.', 'gene_id \"chr37.g8701\"; transcript_id \"chr37.g8701.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24560457', '24560525', '.', '-', '.', 'gene_id \"chr37.g8708\"; transcript_id \"chr37.g8708.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24562083', '24562827', '.', '+', '.', 'gene_id \"chr37.g8709\"; transcript_id \"chr37.g8709.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24564471', '24564716', '.', '+', '.', 'gene_id \"chr37.g8713\"; transcript_id \"chr37.g8713.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24564953', '24565647', '.', '-', '.', 'gene_id \"chr37.g8714\"; transcript_id \"chr37.g8714.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24565632', '24566307', '.', '-', '.', 'gene_id \"chr37.g8715\"; transcript_id \"chr37.g8715.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24566300', '24566571', '.', '-', '.', 'gene_id \"chr37.g8716\"; transcript_id \"chr37.g8716.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24568964', '24569128', '.', '-', '.', 'gene_id \"chr37.g8719\"; transcript_id \"chr37.g8719.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24570167', '24570522', '.', '+', '.', 'gene_id \"chr37.g8720\"; transcript_id \"chr37.g8720.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24570598', '24570808', '.', '+', '.', 'gene_id \"chr37.g8721\"; transcript_id \"chr37.g8721.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24571843', '24572094', '.', '-', '.', 'gene_id \"chr37.g8722\"; transcript_id \"chr37.g8722.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24572776', '24573120', '.', '+', '.', 'gene_id \"chr37.g8723\"; transcript_id \"chr37.g8723.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24575186', '24575415', '.', '+', '.', 'gene_id \"chr37.g8728\"; transcript_id \"chr37.g8728.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24577564', '24577878', '.', '+', '.', 'gene_id \"chr37.g8733\"; transcript_id \"chr37.g8733.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24577271', '24577582', '.', '+', '.', 'gene_id \"chr37.g8734\"; transcript_id \"chr37.g8734.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24579460', '24579663', '.', '+', '.', 'gene_id \"chr37.g8736\"; transcript_id \"chr37.g8736.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24579647', '24579855', '.', '+', '.', 'gene_id \"chr37.g8737\"; transcript_id \"chr37.g8737.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24583852', '24584181', '.', '+', '.', 'gene_id \"chr37.g8738\"; transcript_id \"chr37.g8738.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24584223', '24584399', '.', '-', '.', 'gene_id \"chr37.g8739\"; transcript_id \"chr37.g8739.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24585054', '24585399', '.', '+', '.', 'gene_id \"chr37.g8740\"; transcript_id \"chr37.g8740.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24592554', '24592727', '.', '+', '.', 'gene_id \"chr37.g8744\"; transcript_id \"chr37.g8744.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24595296', '24595452', '.', '+', '.', 'gene_id \"chr37.g8748\"; transcript_id \"chr37.g8748.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24598877', '24599044', '.', '-', '.', 'gene_id \"chr37.g8753\"; transcript_id \"chr37.g8753.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24605057', '24605442', '.', '+', '.', 'gene_id \"chr37.g8758\"; transcript_id \"chr37.g8758.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24607139', '24607317', '.', '+', '.', 'gene_id \"chr37.g8759\"; transcript_id \"chr37.g8759.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24608580', '24608745', '.', '+', '.', 'gene_id \"chr37.g8761\"; transcript_id \"chr37.g8761.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24608755', '24608859', '.', '+', '.', 'gene_id \"chr37.g8763\"; transcript_id \"chr37.g8763.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24609249', '24609504', '.', '+', '.', 'gene_id \"chr37.g8764\"; transcript_id \"chr37.g8764.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24610291', '24610522', '.', '-', '.', 'gene_id \"chr37.g8766\"; transcript_id \"chr37.g8766.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24610068', '24610300', '.', '-', '.', 'gene_id \"chr37.g8767\"; transcript_id \"chr37.g8767.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24611062', '24611741', '.', '-', '.', 'gene_id \"chr37.g8768\"; transcript_id \"chr37.g8768.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24611890', '24612069', '.', '-', '.', 'gene_id \"chr37.g8770\"; transcript_id \"chr37.g8770.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24612293', '24612455', '.', '-', '.', 'gene_id \"chr37.g8771\"; transcript_id \"chr37.g8771.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24612613', '24612764', '.', '-', '.', 'gene_id \"chr37.g8772\"; transcript_id \"chr37.g8772.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24612849', '24613005', '.', '+', '.', 'gene_id \"chr37.g8773\"; transcript_id \"chr37.g8773.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24612994', '24613252', '.', '+', '.', 'gene_id \"chr37.g8774\"; transcript_id \"chr37.g8774.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24616746', '24617053', '.', '-', '.', 'gene_id \"chr37.g8776\"; transcript_id \"chr37.g8776.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24618795', '24619020', '.', '-', '.', 'gene_id \"chr37.g8777\"; transcript_id \"chr37.g8777.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24619269', '24619434', '.', '-', '.', 'gene_id \"chr37.g8779\"; transcript_id \"chr37.g8779.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24621481', '24621662', '.', '-', '.', 'gene_id \"chr37.g8780\"; transcript_id \"chr37.g8780.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24622627', '24623166', '.', '+', '.', 'gene_id \"chr37.g8783\"; transcript_id \"chr37.g8783.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24625510', '24625736', '.', '-', '.', 'gene_id \"chr37.g8787\"; transcript_id \"chr37.g8787.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24627547', '24627729', '.', '-', '.', 'gene_id \"chr37.g8790\"; transcript_id \"chr37.g8790.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24627547', '24627777', '.', '-', '.', 'gene_id \"chr37.g8790\"; transcript_id \"chr37.g8790.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24627868', '24628135', '.', '-', '.', 'gene_id \"chr37.g8791\"; transcript_id \"chr37.g8791.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24628607', '24628801', '.', '+', '.', 'gene_id \"chr37.g8792\"; transcript_id \"chr37.g8792.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24628115', '24628628', '.', '-', '.', 'gene_id \"chr37.g8793\"; transcript_id \"chr37.g8793.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24632649', '24632813', '.', '-', '.', 'gene_id \"chr37.g8798\"; transcript_id \"chr37.g8798.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24633542', '24633759', '.', '+', '.', 'gene_id \"chr37.g8801\"; transcript_id \"chr37.g8801.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24634856', '24635023', '.', '-', '.', 'gene_id \"chr37.g8802\"; transcript_id \"chr37.g8802.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24643154', '24643353', '.', '+', '.', 'gene_id \"chr37.g8808\"; transcript_id \"chr37.g8808.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24643908', '24644063', '.', '-', '.', 'gene_id \"chr37.g8809\"; transcript_id \"chr37.g8809.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24646333', '24646488', '.', '-', '.', 'gene_id \"chr37.g8811\"; transcript_id \"chr37.g8811.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24647054', '24647218', '.', '-', '.', 'gene_id \"chr37.g8813\"; transcript_id \"chr37.g8813.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24647497', '24647653', '.', '+', '.', 'gene_id \"chr37.g8814\"; transcript_id \"chr37.g8814.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24647986', '24648156', '.', '-', '.', 'gene_id \"chr37.g8817\"; transcript_id \"chr37.g8817.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24651646', '24651858', '.', '-', '.', 'gene_id \"chr37.g8819\"; transcript_id \"chr37.g8819.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24651842', '24652077', '.', '+', '.', 'gene_id \"chr37.g8820\"; transcript_id \"chr37.g8820.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24654446', '24654926', '.', '-', '.', 'gene_id \"chr37.g8823\"; transcript_id \"chr37.g8823.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24655766', '24656070', '.', '+', '.', 'gene_id \"chr37.g8825\"; transcript_id \"chr37.g8825.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24657693', '24657933', '.', '-', '.', 'gene_id \"chr37.g8829\"; transcript_id \"chr37.g8829.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24658274', '24658616', '.', '+', '.', 'gene_id \"chr37.g8830\"; transcript_id \"chr37.g8830.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24661712', '24661926', '.', '+', '.', 'gene_id \"chr37.g8831\"; transcript_id \"chr37.g8831.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24666672', '24666937', '.', '-', '.', 'gene_id \"chr37.g8837\"; transcript_id \"chr37.g8837.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24668331', '24668528', '.', '+', '.', 'gene_id \"chr37.g8839\"; transcript_id \"chr37.g8839.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24670824', '24671074', '.', '+', '.', 'gene_id \"chr37.g8841\"; transcript_id \"chr37.g8841.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24671067', '24671232', '.', '-', '.', 'gene_id \"chr37.g8842\"; transcript_id \"chr37.g8842.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24674730', '24674769', '.', '-', '.', 'gene_id \"chr37.g8845\"; transcript_id \"chr37.g8845.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24679607', '24679765', '.', '-', '.', 'gene_id \"chr37.g8848\"; transcript_id \"chr37.g8848.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24686020', '24686134', '.', '-', '.', 'gene_id \"chr37.g8852\"; transcript_id \"chr37.g8852.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24686725', '24687087', '.', '+', '.', 'gene_id \"chr37.g8853\"; transcript_id \"chr37.g8853.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24687763', '24688285', '.', '+', '.', 'gene_id \"chr37.g8855\"; transcript_id \"chr37.g8855.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24692374', '24692655', '.', '-', '.', 'gene_id \"chr37.g8856\"; transcript_id \"chr37.g8856.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24698456', '24698923', '.', '-', '.', 'gene_id \"chr37.g8859\"; transcript_id \"chr37.g8859.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24708518', '24708717', '.', '-', '.', 'gene_id \"chr37.g8864\"; transcript_id \"chr37.g8864.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24709490', '24709728', '.', '-', '.', 'gene_id \"chr37.g8867\"; transcript_id \"chr37.g8867.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24711621', '24712271', '.', '+', '.', 'gene_id \"chr37.g8869\"; transcript_id \"chr37.g8869.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24717477', '24717639', '.', '-', '.', 'gene_id \"chr37.g8872\"; transcript_id \"chr37.g8872.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24725475', '24725659', '.', '+', '.', 'gene_id \"chr37.g8874\"; transcript_id \"chr37.g8874.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24727039', '24727200', '.', '-', '.', 'gene_id \"chr37.g8876\"; transcript_id \"chr37.g8876.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24728547', '24728864', '.', '-', '.', 'gene_id \"chr37.g8879\"; transcript_id \"chr37.g8879.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24731577', '24732471', '.', '+', '.', 'gene_id \"chr37.g8884\"; transcript_id \"chr37.g8884.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24747067', '24747757', '.', '+', '.', 'gene_id \"chr37.g8890\"; transcript_id \"chr37.g8890.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24747067', '24747757', '.', '+', '.', 'gene_id \"chr37.g8890\"; transcript_id \"chr37.g8890.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24749576', '24749774', '.', '+', '.', 'gene_id \"chr37.g8892\"; transcript_id \"chr37.g8892.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24753138', '24753388', '.', '+', '.', 'gene_id \"chr37.g8894\"; transcript_id \"chr37.g8894.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24754029', '24754303', '.', '+', '.', 'gene_id \"chr37.g8896\"; transcript_id \"chr37.g8896.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24766521', '24768010', '.', '-', '.', 'gene_id \"chr37.g8899\"; transcript_id \"chr37.g8899.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24766483', '24766544', '.', '-', '.', 'gene_id \"chr37.g8899\"; transcript_id \"chr37.g8899.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24763780', '24764488', '.', '-', '.', 'gene_id \"chr37.g8900\"; transcript_id \"chr37.g8900.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24764466', '24764684', '.', '-', '.', 'gene_id \"chr37.g8900\"; transcript_id \"chr37.g8900.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24761645', '24761937', '.', '-', '.', 'gene_id \"chr37.g8904\"; transcript_id \"chr37.g8904.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24762002', '24762221', '.', '+', '.', 'gene_id \"chr37.g8905\"; transcript_id \"chr37.g8905.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24768485', '24768802', '.', '-', '.', 'gene_id \"chr37.g8907\"; transcript_id \"chr37.g8907.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24769695', '24769887', '.', '-', '.', 'gene_id \"chr37.g8909\"; transcript_id \"chr37.g8909.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24769913', '24770126', '.', '-', '.', 'gene_id \"chr37.g8911\"; transcript_id \"chr37.g8911.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24771283', '24771495', '.', '+', '.', 'gene_id \"chr37.g8912\"; transcript_id \"chr37.g8912.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24771498', '24771867', '.', '-', '.', 'gene_id \"chr37.g8913\"; transcript_id \"chr37.g8913.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24772427', '24772628', '.', '+', '.', 'gene_id \"chr37.g8917\"; transcript_id \"chr37.g8917.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24774571', '24774734', '.', '+', '.', 'gene_id \"chr37.g8918\"; transcript_id \"chr37.g8918.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24775599', '24776031', '.', '+', '.', 'gene_id \"chr37.g8919\"; transcript_id \"chr37.g8919.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24776064', '24776232', '.', '+', '.', 'gene_id \"chr37.g8920\"; transcript_id \"chr37.g8920.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24776537', '24777063', '.', '-', '.', 'gene_id \"chr37.g8921\"; transcript_id \"chr37.g8921.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24781548', '24782533', '.', '-', '.', 'gene_id \"chr37.g8924\"; transcript_id \"chr37.g8924.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24782891', '24786399', '.', '+', '.', 'gene_id \"chr37.g8925\"; transcript_id \"chr37.g8925.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24784375', '24784697', '.', '+', '.', 'gene_id \"chr37.g8925\"; transcript_id \"chr37.g8925.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24784375', '24786399', '.', '+', '.', 'gene_id \"chr37.g8925\"; transcript_id \"chr37.g8925.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24817503', '24817532', '.', '-', '.', 'gene_id \"chr37.g8928\"; transcript_id \"chr37.g8928.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24827144', '24827780', '.', '-', '.', 'gene_id \"chr37.g8930\"; transcript_id \"chr37.g8930.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24851339', '24852098', '.', '+', '.', 'gene_id \"chr37.g8934\"; transcript_id \"chr37.g8934.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24841047', '24842499', '.', '-', '.', 'gene_id \"chr37.g8935\"; transcript_id \"chr37.g8935.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24857586', '24857795', '.', '-', '.', 'gene_id \"chr37.g8939\"; transcript_id \"chr37.g8939.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24843077', '24845413', '.', '+', '.', 'gene_id \"chr37.g8940\"; transcript_id \"chr37.g8940.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24854487', '24857160', '.', '-', '.', 'gene_id \"chr37.g8941\"; transcript_id \"chr37.g8941.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24853925', '24854429', '.', '+', '.', 'gene_id \"chr37.g8942\"; transcript_id \"chr37.g8942.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24853209', '24853868', '.', '-', '.', 'gene_id \"chr37.g8944\"; transcript_id \"chr37.g8944.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24845560', '24846228', '.', '+', '.', 'gene_id \"chr37.g8948\"; transcript_id \"chr37.g8948.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24837918', '24838216', '.', '-', '.', 'gene_id \"chr37.g8949\"; transcript_id \"chr37.g8949.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24829116', '24829278', '.', '-', '.', 'gene_id \"chr37.g8950\"; transcript_id \"chr37.g8950.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24832357', '24832561', '.', '-', '.', 'gene_id \"chr37.g8956\"; transcript_id \"chr37.g8956.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24833592', '24833753', '.', '+', '.', 'gene_id \"chr37.g8957\"; transcript_id \"chr37.g8957.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24847418', '24848036', '.', '+', '.', 'gene_id \"chr37.g8959\"; transcript_id \"chr37.g8959.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24848298', '24849106', '.', '-', '.', 'gene_id \"chr37.g8961\"; transcript_id \"chr37.g8961.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24860780', '24860969', '.', '+', '.', 'gene_id \"chr37.g8968\"; transcript_id \"chr37.g8968.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24860397', '24860799', '.', '-', '.', 'gene_id \"chr37.g8969\"; transcript_id \"chr37.g8969.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24862458', '24862658', '.', '+', '.', 'gene_id \"chr37.g8972\"; transcript_id \"chr37.g8972.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24862696', '24862951', '.', '+', '.', 'gene_id \"chr37.g8973\"; transcript_id \"chr37.g8973.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24863603', '24864051', '.', '+', '.', 'gene_id \"chr37.g8975\"; transcript_id \"chr37.g8975.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24864064', '24864332', '.', '-', '.', 'gene_id \"chr37.g8976\"; transcript_id \"chr37.g8976.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24865480', '24865989', '.', '-', '.', 'gene_id \"chr37.g8978\"; transcript_id \"chr37.g8978.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24867059', '24867340', '.', '-', '.', 'gene_id \"chr37.g8980\"; transcript_id \"chr37.g8980.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24867565', '24867797', '.', '-', '.', 'gene_id \"chr37.g8981\"; transcript_id \"chr37.g8981.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24868820', '24868978', '.', '+', '.', 'gene_id \"chr37.g8982\"; transcript_id \"chr37.g8982.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24870375', '24871152', '.', '-', '.', 'gene_id \"chr37.g8984\"; transcript_id \"chr37.g8984.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24874963', '24875396', '.', '-', '.', 'gene_id \"chr37.g8985\"; transcript_id \"chr37.g8985.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24878665', '24878845', '.', '-', '.', 'gene_id \"chr37.g8985\"; transcript_id \"chr37.g8985.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24878573', '24878656', '.', '-', '.', 'gene_id \"chr37.g8985\"; transcript_id \"chr37.g8985.i9\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24873107', '24873352', '.', '-', '.', 'gene_id \"chr37.g8987\"; transcript_id \"chr37.g8987.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24876882', '24877104', '.', '-', '.', 'gene_id \"chr37.g8988\"; transcript_id \"chr37.g8988.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24878868', '24879142', '.', '+', '.', 'gene_id \"chr37.g8989\"; transcript_id \"chr37.g8989.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24878098', '24878593', '.', '+', '.', 'gene_id \"chr37.g8991\"; transcript_id \"chr37.g8991.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24879126', '24879696', '.', '+', '.', 'gene_id \"chr37.g8992\"; transcript_id \"chr37.g8992.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24884848', '24885323', '.', '+', '.', 'gene_id \"chr37.g8994\"; transcript_id \"chr37.g8994.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24881830', '24882410', '.', '+', '.', 'gene_id \"chr37.g8995\"; transcript_id \"chr37.g8995.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24882392', '24883515', '.', '-', '.', 'gene_id \"chr37.g8996\"; transcript_id \"chr37.g8996.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24885907', '24886108', '.', '-', '.', 'gene_id \"chr37.g8998\"; transcript_id \"chr37.g8998.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24886826', '24886990', '.', '+', '.', 'gene_id \"chr37.g9001\"; transcript_id \"chr37.g9001.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24889363', '24889540', '.', '-', '.', 'gene_id \"chr37.g9005\"; transcript_id \"chr37.g9005.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24893668', '24893960', '.', '-', '.', 'gene_id \"chr37.g9007\"; transcript_id \"chr37.g9007.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24910256', '24910544', '.', '-', '.', 'gene_id \"chr37.g9015\"; transcript_id \"chr37.g9015.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24911584', '24912207', '.', '+', '.', 'gene_id \"chr37.g9017\"; transcript_id \"chr37.g9017.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24912221', '24913516', '.', '+', '.', 'gene_id \"chr37.g9019\"; transcript_id \"chr37.g9019.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24913804', '24914614', '.', '-', '.', 'gene_id \"chr37.g9020\"; transcript_id \"chr37.g9020.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24915108', '24915626', '.', '-', '.', 'gene_id \"chr37.g9021\"; transcript_id \"chr37.g9021.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24915666', '24915834', '.', '+', '.', 'gene_id \"chr37.g9022\"; transcript_id \"chr37.g9022.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24915860', '24916138', '.', '+', '.', 'gene_id \"chr37.g9023\"; transcript_id \"chr37.g9023.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24917025', '24917399', '.', '+', '.', 'gene_id \"chr37.g9024\"; transcript_id \"chr37.g9024.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24918130', '24919108', '.', '-', '.', 'gene_id \"chr37.g9025\"; transcript_id \"chr37.g9025.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24921313', '24921604', '.', '+', '.', 'gene_id \"chr37.g9029\"; transcript_id \"chr37.g9029.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24926250', '24926858', '.', '-', '.', 'gene_id \"chr37.g9031\"; transcript_id \"chr37.g9031.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24923775', '24924107', '.', '-', '.', 'gene_id \"chr37.g9032\"; transcript_id \"chr37.g9032.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24950306', '24951848', '.', '+', '.', 'gene_id \"chr37.g9037\"; transcript_id \"chr37.g9037.i7\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24942479', '24942667', '.', '-', '.', 'gene_id \"chr37.g9038\"; transcript_id \"chr37.g9038.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24945016', '24945392', '.', '+', '.', 'gene_id \"chr37.g9040\"; transcript_id \"chr37.g9040.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24951895', '24951982', '.', '+', '.', 'gene_id \"chr37.g9042\"; transcript_id \"chr37.g9042.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24956190', '24956905', '.', '-', '.', 'gene_id \"chr37.g9046\"; transcript_id \"chr37.g9046.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24956997', '24957551', '.', '+', '.', 'gene_id \"chr37.g9049\"; transcript_id \"chr37.g9049.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24962994', '24963214', '.', '-', '.', 'gene_id \"chr37.g9053\"; transcript_id \"chr37.g9053.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24960210', '24960363', '.', '+', '.', 'gene_id \"chr37.g9054\"; transcript_id \"chr37.g9054.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24962004', '24962154', '.', '-', '.', 'gene_id \"chr37.g9056\"; transcript_id \"chr37.g9056.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24970678', '24970944', '.', '+', '.', 'gene_id \"chr37.g9059\"; transcript_id \"chr37.g9059.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24971530', '24971683', '.', '+', '.', 'gene_id \"chr37.g9060\"; transcript_id \"chr37.g9060.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24973788', '24973945', '.', '-', '.', 'gene_id \"chr37.g9062\"; transcript_id \"chr37.g9062.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24976235', '24976838', '.', '-', '.', 'gene_id \"chr37.g9063\"; transcript_id \"chr37.g9063.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24973128', '24973326', '.', '-', '.', 'gene_id \"chr37.g9064\"; transcript_id \"chr37.g9064.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24973445', '24973713', '.', '-', '.', 'gene_id \"chr37.g9066\"; transcript_id \"chr37.g9066.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24980882', '24981050', '.', '-', '.', 'gene_id \"chr37.g9067\"; transcript_id \"chr37.g9067.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '24997826', '24998156', '.', '+', '.', 'gene_id \"chr37.g9068\"; transcript_id \"chr37.g9068.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25003618', '25003850', '.', '+', '.', 'gene_id \"chr37.g9070\"; transcript_id \"chr37.g9070.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25010801', '25011009', '.', '-', '.', 'gene_id \"chr37.g9071\"; transcript_id \"chr37.g9071.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25021386', '25021916', '.', '-', '.', 'gene_id \"chr37.g9077\"; transcript_id \"chr37.g9077.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25023378', '25023653', '.', '-', '.', 'gene_id \"chr37.g9081\"; transcript_id \"chr37.g9081.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25043466', '25043617', '.', '-', '.', 'gene_id \"chr37.g9088\"; transcript_id \"chr37.g9088.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25044400', '25044692', '.', '-', '.', 'gene_id \"chr37.g9089\"; transcript_id \"chr37.g9089.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25045100', '25045280', '.', '+', '.', 'gene_id \"chr37.g9091\"; transcript_id \"chr37.g9091.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25046873', '25047028', '.', '+', '.', 'gene_id \"chr37.g9093\"; transcript_id \"chr37.g9093.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25063539', '25063747', '.', '-', '.', 'gene_id \"chr37.g9095\"; transcript_id \"chr37.g9095.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25065002', '25065340', '.', '-', '.', 'gene_id \"chr37.g9096\"; transcript_id \"chr37.g9096.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25065398', '25066246', '.', '+', '.', 'gene_id \"chr37.g9097\"; transcript_id \"chr37.g9097.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25093436', '25093639', '.', '-', '.', 'gene_id \"chr37.g9100\"; transcript_id \"chr37.g9100.i1\";']\n",
      "long:  ['chr37', 'PASA', 'exon', '25076848', '25076985', '.', '-', '.', 'gene_id \"chr37.g9104\"; transcript_id \"chr37.g9104.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25077509', '25077756', '.', '+', '.', 'gene_id \"chr37.g9107\"; transcript_id \"chr37.g9107.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25088408', '25088692', '.', '+', '.', 'gene_id \"chr37.g9111\"; transcript_id \"chr37.g9111.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25090728', '25090915', '.', '-', '.', 'gene_id \"chr37.g9113\"; transcript_id \"chr37.g9113.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25091902', '25092200', '.', '+', '.', 'gene_id \"chr37.g9116\"; transcript_id \"chr37.g9116.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25096280', '25096464', '.', '-', '.', 'gene_id \"chr37.g9120\"; transcript_id \"chr37.g9120.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25097609', '25097929', '.', '+', '.', 'gene_id \"chr37.g9121\"; transcript_id \"chr37.g9121.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25098748', '25099032', '.', '-', '.', 'gene_id \"chr37.g9122\"; transcript_id \"chr37.g9122.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25102712', '25102897', '.', '-', '.', 'gene_id \"chr37.g9126\"; transcript_id \"chr37.g9126.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25103174', '25103664', '.', '+', '.', 'gene_id \"chr37.g9127\"; transcript_id \"chr37.g9127.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25103915', '25104099', '.', '-', '.', 'gene_id \"chr37.g9129\"; transcript_id \"chr37.g9129.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25105046', '25105273', '.', '-', '.', 'gene_id \"chr37.g9130\"; transcript_id \"chr37.g9130.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25132847', '25135774', '.', '+', '.', 'gene_id \"chr37.g9134\"; transcript_id \"chr37.g9134.i3\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25119781', '25120065', '.', '-', '.', 'gene_id \"chr37.g9135\"; transcript_id \"chr37.g9135.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25119124', '25119532', '.', '+', '.', 'gene_id \"chr37.g9136\"; transcript_id \"chr37.g9136.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25108166', '25108404', '.', '+', '.', 'gene_id \"chr37.g9143\"; transcript_id \"chr37.g9143.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25109362', '25109543', '.', '-', '.', 'gene_id \"chr37.g9146\"; transcript_id \"chr37.g9146.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25110019', '25110256', '.', '+', '.', 'gene_id \"chr37.g9147\"; transcript_id \"chr37.g9147.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25110854', '25111311', '.', '-', '.', 'gene_id \"chr37.g9148\"; transcript_id \"chr37.g9148.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25110483', '25110866', '.', '+', '.', 'gene_id \"chr37.g9149\"; transcript_id \"chr37.g9149.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25113499', '25113874', '.', '-', '.', 'gene_id \"chr37.g9154\"; transcript_id \"chr37.g9154.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25114054', '25114231', '.', '+', '.', 'gene_id \"chr37.g9155\"; transcript_id \"chr37.g9155.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25114470', '25114838', '.', '+', '.', 'gene_id \"chr37.g9156\"; transcript_id \"chr37.g9156.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25114922', '25115096', '.', '+', '.', 'gene_id \"chr37.g9157\"; transcript_id \"chr37.g9157.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25116741', '25116926', '.', '+', '.', 'gene_id \"chr37.g9159\"; transcript_id \"chr37.g9159.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25117539', '25117922', '.', '+', '.', 'gene_id \"chr37.g9161\"; transcript_id \"chr37.g9161.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25121578', '25121768', '.', '+', '.', 'gene_id \"chr37.g9162\"; transcript_id \"chr37.g9162.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25137671', '25137964', '.', '-', '.', 'gene_id \"chr37.g9166\"; transcript_id \"chr37.g9166.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25137508', '25137674', '.', '+', '.', 'gene_id \"chr37.g9167\"; transcript_id \"chr37.g9167.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25139116', '25139276', '.', '+', '.', 'gene_id \"chr37.g9168\"; transcript_id \"chr37.g9168.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25142194', '25142404', '.', '+', '.', 'gene_id \"chr37.g9173\"; transcript_id \"chr37.g9173.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25149243', '25149672', '.', '-', '.', 'gene_id \"chr37.g9177\"; transcript_id \"chr37.g9177.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25160436', '25162217', '.', '+', '.', 'gene_id \"chr37.g9179\"; transcript_id \"chr37.g9179.i9\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25160334', '25160456', '.', '+', '.', 'gene_id \"chr37.g9179\"; transcript_id \"chr37.g9179.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25164082', '25164400', '.', '+', '.', 'gene_id \"chr37.g9180\"; transcript_id \"chr37.g9180.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25150881', '25151062', '.', '-', '.', 'gene_id \"chr37.g9183\"; transcript_id \"chr37.g9183.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25152917', '25153289', '.', '+', '.', 'gene_id \"chr37.g9185\"; transcript_id \"chr37.g9185.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25153789', '25153954', '.', '+', '.', 'gene_id \"chr37.g9186\"; transcript_id \"chr37.g9186.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25154031', '25154237', '.', '-', '.', 'gene_id \"chr37.g9187\"; transcript_id \"chr37.g9187.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25155823', '25155977', '.', '+', '.', 'gene_id \"chr37.g9189\"; transcript_id \"chr37.g9189.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25158324', '25158498', '.', '-', '.', 'gene_id \"chr37.g9190\"; transcript_id \"chr37.g9190.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25159025', '25159674', '.', '+', '.', 'gene_id \"chr37.g9191\"; transcript_id \"chr37.g9191.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25159695', '25159925', '.', '+', '.', 'gene_id \"chr37.g9192\"; transcript_id \"chr37.g9192.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25172448', '25172874', '.', '+', '.', 'gene_id \"chr37.g9196\"; transcript_id \"chr37.g9196.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25182199', '25182848', '.', '+', '.', 'gene_id \"chr37.g9198\"; transcript_id \"chr37.g9198.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25166896', '25167061', '.', '+', '.', 'gene_id \"chr37.g9199\"; transcript_id \"chr37.g9199.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25167425', '25167664', '.', '-', '.', 'gene_id \"chr37.g9200\"; transcript_id \"chr37.g9200.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25171732', '25172128', '.', '-', '.', 'gene_id \"chr37.g9201\"; transcript_id \"chr37.g9201.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25174576', '25174833', '.', '+', '.', 'gene_id \"chr37.g9206\"; transcript_id \"chr37.g9206.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25187298', '25187508', '.', '-', '.', 'gene_id \"chr37.g9208\"; transcript_id \"chr37.g9208.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25185384', '25185615', '.', '+', '.', 'gene_id \"chr37.g9209\"; transcript_id \"chr37.g9209.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25186109', '25186333', '.', '-', '.', 'gene_id \"chr37.g9210\"; transcript_id \"chr37.g9210.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25196689', '25197100', '.', '+', '.', 'gene_id \"chr37.g9213\"; transcript_id \"chr37.g9213.i15\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25218027', '25218217', '.', '-', '.', 'gene_id \"chr37.g9217\"; transcript_id \"chr37.g9217.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25203458', '25203675', '.', '+', '.', 'gene_id \"chr37.g9225\"; transcript_id \"chr37.g9225.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25208672', '25209015', '.', '+', '.', 'gene_id \"chr37.g9227\"; transcript_id \"chr37.g9227.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25212697', '25212851', '.', '-', '.', 'gene_id \"chr37.g9229\"; transcript_id \"chr37.g9229.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25213282', '25213593', '.', '-', '.', 'gene_id \"chr37.g9231\"; transcript_id \"chr37.g9231.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25222514', '25222683', '.', '-', '.', 'gene_id \"chr37.g9233\"; transcript_id \"chr37.g9233.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25224407', '25224750', '.', '+', '.', 'gene_id \"chr37.g9235\"; transcript_id \"chr37.g9235.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25224790', '25224827', '.', '-', '.', 'gene_id \"chr37.g9237\"; transcript_id \"chr37.g9237.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25232267', '25232524', '.', '+', '.', 'gene_id \"chr37.g9239\"; transcript_id \"chr37.g9239.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25250898', '25251124', '.', '-', '.', 'gene_id \"chr37.g9241\"; transcript_id \"chr37.g9241.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25258923', '25259136', '.', '-', '.', 'gene_id \"chr37.g9242\"; transcript_id \"chr37.g9242.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25258734', '25258945', '.', '+', '.', 'gene_id \"chr37.g9243\"; transcript_id \"chr37.g9243.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25257839', '25258094', '.', '-', '.', 'gene_id \"chr37.g9244\"; transcript_id \"chr37.g9244.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25254941', '25255253', '.', '+', '.', 'gene_id \"chr37.g9245\"; transcript_id \"chr37.g9245.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25253212', '25253365', '.', '-', '.', 'gene_id \"chr37.g9250\"; transcript_id \"chr37.g9250.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25252767', '25253085', '.', '-', '.', 'gene_id \"chr37.g9251\"; transcript_id \"chr37.g9251.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25258499', '25258752', '.', '+', '.', 'gene_id \"chr37.g9252\"; transcript_id \"chr37.g9252.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25260183', '25260490', '.', '+', '.', 'gene_id \"chr37.g9254\"; transcript_id \"chr37.g9254.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25242387', '25242662', '.', '-', '.', 'gene_id \"chr37.g9260\"; transcript_id \"chr37.g9260.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25242664', '25242887', '.', '+', '.', 'gene_id \"chr37.g9261\"; transcript_id \"chr37.g9261.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25242915', '25243145', '.', '+', '.', 'gene_id \"chr37.g9262\"; transcript_id \"chr37.g9262.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25243273', '25243429', '.', '+', '.', 'gene_id \"chr37.g9263\"; transcript_id \"chr37.g9263.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25256673', '25256899', '.', '-', '.', 'gene_id \"chr37.g9266\"; transcript_id \"chr37.g9266.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25261805', '25262000', '.', '+', '.', 'gene_id \"chr37.g9268\"; transcript_id \"chr37.g9268.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25262326', '25262773', '.', '-', '.', 'gene_id \"chr37.g9269\"; transcript_id \"chr37.g9269.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25262818', '25263028', '.', '+', '.', 'gene_id \"chr37.g9270\"; transcript_id \"chr37.g9270.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25268890', '25269067', '.', '+', '.', 'gene_id \"chr37.g9277\"; transcript_id \"chr37.g9277.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25271810', '25272040', '.', '-', '.', 'gene_id \"chr37.g9279\"; transcript_id \"chr37.g9279.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25272041', '25272309', '.', '+', '.', 'gene_id \"chr37.g9281\"; transcript_id \"chr37.g9281.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25286618', '25287068', '.', '+', '.', 'gene_id \"chr37.g9286\"; transcript_id \"chr37.g9286.i9\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25286394', '25286633', '.', '-', '.', 'gene_id \"chr37.g9287\"; transcript_id \"chr37.g9287.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25324225', '25324680', '.', '-', '.', 'gene_id \"chr37.g9288\"; transcript_id \"chr37.g9288.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25289218', '25289643', '.', '+', '.', 'gene_id \"chr37.g9294\"; transcript_id \"chr37.g9294.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25289896', '25290388', '.', '-', '.', 'gene_id \"chr37.g9296\"; transcript_id \"chr37.g9296.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25290364', '25290654', '.', '-', '.', 'gene_id \"chr37.g9297\"; transcript_id \"chr37.g9297.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25292063', '25292343', '.', '-', '.', 'gene_id \"chr37.g9299\"; transcript_id \"chr37.g9299.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25292908', '25293029', '.', '+', '.', 'gene_id \"chr37.g9301\"; transcript_id \"chr37.g9301.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25293114', '25293307', '.', '-', '.', 'gene_id \"chr37.g9302\"; transcript_id \"chr37.g9302.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45686020', '45686244', '.', '+', '.', 'gene_id \"chr23.g17517\"; transcript_id \"chr23.g17517.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25293430', '25294815', '.', '-', '.', 'gene_id \"chr37.g9303\"; transcript_id \"chr37.g9303.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25293430', '25294024', '.', '-', '.', 'gene_id \"chr37.g9303\"; transcript_id \"chr37.g9303.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25296701', '25297071', '.', '+', '.', 'gene_id \"chr37.g9306\"; transcript_id \"chr37.g9306.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25297960', '25298409', '.', '-', '.', 'gene_id \"chr37.g9309\"; transcript_id \"chr37.g9309.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25297793', '25297965', '.', '+', '.', 'gene_id \"chr37.g9310\"; transcript_id \"chr37.g9310.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25298141', '25298182', '.', '+', '.', 'gene_id \"chr37.g9311\"; transcript_id \"chr37.g9311.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25298454', '25298794', '.', '+', '.', 'gene_id \"chr37.g9313\"; transcript_id \"chr37.g9313.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25299871', '25300261', '.', '+', '.', 'gene_id \"chr37.g9315\"; transcript_id \"chr37.g9315.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25300241', '25300547', '.', '-', '.', 'gene_id \"chr37.g9316\"; transcript_id \"chr37.g9316.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25301493', '25302064', '.', '+', '.', 'gene_id \"chr37.g9322\"; transcript_id \"chr37.g9322.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25302807', '25303224', '.', '+', '.', 'gene_id \"chr37.g9323\"; transcript_id \"chr37.g9323.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45732199', '45732388', '.', '+', '.', 'gene_id \"chr23.g17530\"; transcript_id \"chr23.g17530.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25304731', '25305051', '.', '+', '.', 'gene_id \"chr37.g9328\"; transcript_id \"chr37.g9328.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45745373', '45745553', '.', '+', '.', 'gene_id \"chr23.g17533\"; transcript_id \"chr23.g17533.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25306694', '25307118', '.', '+', '.', 'gene_id \"chr37.g9330\"; transcript_id \"chr37.g9330.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45768356', '45768561', '.', '-', '.', 'gene_id \"chr23.g17537\"; transcript_id \"chr23.g17537.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25309464', '25309644', '.', '-', '.', 'gene_id \"chr37.g9334\"; transcript_id \"chr37.g9334.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25310160', '25310487', '.', '-', '.', 'gene_id \"chr37.g9336\"; transcript_id \"chr37.g9336.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25330266', '25330464', '.', '-', '.', 'gene_id \"chr37.g9344\"; transcript_id \"chr37.g9344.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25331999', '25332168', '.', '-', '.', 'gene_id \"chr37.g9346\"; transcript_id \"chr37.g9346.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25332166', '25332384', '.', '+', '.', 'gene_id \"chr37.g9347\"; transcript_id \"chr37.g9347.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25353194', '25353371', '.', '-', '.', 'gene_id \"chr37.g9351\"; transcript_id \"chr37.g9351.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25378054', '25378991', '.', '+', '.', 'gene_id \"chr37.g9353\"; transcript_id \"chr37.g9353.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25377875', '25378071', '.', '-', '.', 'gene_id \"chr37.g9355\"; transcript_id \"chr37.g9355.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25363842', '25364016', '.', '-', '.', 'gene_id \"chr37.g9356\"; transcript_id \"chr37.g9356.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25365951', '25366118', '.', '-', '.', 'gene_id \"chr37.g9359\"; transcript_id \"chr37.g9359.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25368540', '25368727', '.', '-', '.', 'gene_id \"chr37.g9360\"; transcript_id \"chr37.g9360.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25369886', '25370185', '.', '+', '.', 'gene_id \"chr37.g9362\"; transcript_id \"chr37.g9362.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25370589', '25370891', '.', '-', '.', 'gene_id \"chr37.g9363\"; transcript_id \"chr37.g9363.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25372640', '25372794', '.', '-', '.', 'gene_id \"chr37.g9366\"; transcript_id \"chr37.g9366.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25373724', '25373882', '.', '+', '.', 'gene_id \"chr37.g9370\"; transcript_id \"chr37.g9370.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25374969', '25375316', '.', '+', '.', 'gene_id \"chr37.g9371\"; transcript_id \"chr37.g9371.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25380310', '25380387', '.', '-', '.', 'gene_id \"chr37.g9373\"; transcript_id \"chr37.g9373.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25381975', '25382061', '.', '-', '.', 'gene_id \"chr37.g9374\"; transcript_id \"chr37.g9374.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25386900', '25387059', '.', '-', '.', 'gene_id \"chr37.g9377\"; transcript_id \"chr37.g9377.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25392805', '25393016', '.', '+', '.', 'gene_id \"chr37.g9379\"; transcript_id \"chr37.g9379.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25396816', '25396998', '.', '-', '.', 'gene_id \"chr37.g9382\"; transcript_id \"chr37.g9382.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25401607', '25401653', '.', '-', '.', 'gene_id \"chr37.g9386\"; transcript_id \"chr37.g9386.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25411716', '25411892', '.', '+', '.', 'gene_id \"chr37.g9389\"; transcript_id \"chr37.g9389.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25412014', '25412204', '.', '-', '.', 'gene_id \"chr37.g9390\"; transcript_id \"chr37.g9390.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25431908', '25432093', '.', '-', '.', 'gene_id \"chr37.g9397\"; transcript_id \"chr37.g9397.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25453848', '25455876', '.', '+', '.', 'gene_id \"chr37.g9400\"; transcript_id \"chr37.g9400.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25448994', '25453051', '.', '+', '.', 'gene_id \"chr37.g9401\"; transcript_id \"chr37.g9401.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25453043', '25453868', '.', '+', '.', 'gene_id \"chr37.g9402\"; transcript_id \"chr37.g9402.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25486968', '25487133', '.', '-', '.', 'gene_id \"chr37.g9405\"; transcript_id \"chr37.g9405.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25511521', '25511990', '.', '-', '.', 'gene_id \"chr37.g9407\"; transcript_id \"chr37.g9407.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25495640', '25495818', '.', '-', '.', 'gene_id \"chr37.g9410\"; transcript_id \"chr37.g9410.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25500301', '25500470', '.', '-', '.', 'gene_id \"chr37.g9413\"; transcript_id \"chr37.g9413.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25500799', '25501239', '.', '-', '.', 'gene_id \"chr37.g9415\"; transcript_id \"chr37.g9415.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25501675', '25501844', '.', '-', '.', 'gene_id \"chr37.g9416\"; transcript_id \"chr37.g9416.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25502050', '25502245', '.', '+', '.', 'gene_id \"chr37.g9417\"; transcript_id \"chr37.g9417.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25502269', '25502503', '.', '-', '.', 'gene_id \"chr37.g9418\"; transcript_id \"chr37.g9418.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25533998', '25534155', '.', '+', '.', 'gene_id \"chr37.g9424\"; transcript_id \"chr37.g9424.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25540282', '25540959', '.', '-', '.', 'gene_id \"chr37.g9427\"; transcript_id \"chr37.g9427.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25541179', '25541350', '.', '-', '.', 'gene_id \"chr37.g9430\"; transcript_id \"chr37.g9430.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25545401', '25545575', '.', '-', '.', 'gene_id \"chr37.g9434\"; transcript_id \"chr37.g9434.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25546607', '25546834', '.', '-', '.', 'gene_id \"chr37.g9438\"; transcript_id \"chr37.g9438.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25559716', '25559895', '.', '+', '.', 'gene_id \"chr37.g9450\"; transcript_id \"chr37.g9450.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25560216', '25560438', '.', '+', '.', 'gene_id \"chr37.g9452\"; transcript_id \"chr37.g9452.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25563712', '25563950', '.', '+', '.', 'gene_id \"chr37.g9455\"; transcript_id \"chr37.g9455.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25564108', '25564360', '.', '-', '.', 'gene_id \"chr37.g9456\"; transcript_id \"chr37.g9456.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25570335', '25570520', '.', '+', '.', 'gene_id \"chr37.g9458\"; transcript_id \"chr37.g9458.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25580060', '25580522', '.', '-', '.', 'gene_id \"chr37.g9461\"; transcript_id \"chr37.g9461.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25580707', '25580905', '.', '-', '.', 'gene_id \"chr37.g9462\"; transcript_id \"chr37.g9462.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25582230', '25582438', '.', '+', '.', 'gene_id \"chr37.g9464\"; transcript_id \"chr37.g9464.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25591074', '25591220', '.', '+', '.', 'gene_id \"chr37.g9467\"; transcript_id \"chr37.g9467.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25592582', '25592878', '.', '+', '.', 'gene_id \"chr37.g9468\"; transcript_id \"chr37.g9468.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25599961', '25600345', '.', '-', '.', 'gene_id \"chr37.g9469\"; transcript_id \"chr37.g9469.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25618176', '25618369', '.', '-', '.', 'gene_id \"chr37.g9474\"; transcript_id \"chr37.g9474.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25620773', '25621072', '.', '-', '.', 'gene_id \"chr37.g9480\"; transcript_id \"chr37.g9480.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25640531', '25640709', '.', '-', '.', 'gene_id \"chr37.g9488\"; transcript_id \"chr37.g9488.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25663021', '25663530', '.', '-', '.', 'gene_id \"chr37.g9493\"; transcript_id \"chr37.g9493.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25650260', '25650483', '.', '-', '.', 'gene_id \"chr37.g9495\"; transcript_id \"chr37.g9495.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25656619', '25657007', '.', '-', '.', 'gene_id \"chr37.g9497\"; transcript_id \"chr37.g9497.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25657101', '25657552', '.', '-', '.', 'gene_id \"chr37.g9498\"; transcript_id \"chr37.g9498.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25657595', '25657968', '.', '-', '.', 'gene_id \"chr37.g9499\"; transcript_id \"chr37.g9499.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25658067', '25658251', '.', '-', '.', 'gene_id \"chr37.g9500\"; transcript_id \"chr37.g9500.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25658601', '25658887', '.', '-', '.', 'gene_id \"chr37.g9501\"; transcript_id \"chr37.g9501.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25655099', '25655380', '.', '-', '.', 'gene_id \"chr37.g9502\"; transcript_id \"chr37.g9502.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25652814', '25653153', '.', '+', '.', 'gene_id \"chr37.g9504\"; transcript_id \"chr37.g9504.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25664019', '25664883', '.', '+', '.', 'gene_id \"chr37.g9505\"; transcript_id \"chr37.g9505.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25665463', '25665651', '.', '+', '.', 'gene_id \"chr37.g9507\"; transcript_id \"chr37.g9507.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25665815', '25665988', '.', '-', '.', 'gene_id \"chr37.g9508\"; transcript_id \"chr37.g9508.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25669470', '25669647', '.', '-', '.', 'gene_id \"chr37.g9512\"; transcript_id \"chr37.g9512.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25673709', '25673926', '.', '+', '.', 'gene_id \"chr37.g9515\"; transcript_id \"chr37.g9515.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25675808', '25676182', '.', '+', '.', 'gene_id \"chr37.g9518\"; transcript_id \"chr37.g9518.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25679007', '25679366', '.', '+', '.', 'gene_id \"chr37.g9520\"; transcript_id \"chr37.g9520.i13\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25685589', '25685826', '.', '+', '.', 'gene_id \"chr37.g9522\"; transcript_id \"chr37.g9522.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25683944', '25684408', '.', '+', '.', 'gene_id \"chr37.g9523\"; transcript_id \"chr37.g9523.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25686360', '25686707', '.', '-', '.', 'gene_id \"chr37.g9524\"; transcript_id \"chr37.g9524.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25680811', '25681090', '.', '+', '.', 'gene_id \"chr37.g9525\"; transcript_id \"chr37.g9525.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25681967', '25682224', '.', '+', '.', 'gene_id \"chr37.g9527\"; transcript_id \"chr37.g9527.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25682241', '25682515', '.', '+', '.', 'gene_id \"chr37.g9528\"; transcript_id \"chr37.g9528.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25683212', '25683391', '.', '+', '.', 'gene_id \"chr37.g9529\"; transcript_id \"chr37.g9529.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25684495', '25685177', '.', '+', '.', 'gene_id \"chr37.g9530\"; transcript_id \"chr37.g9530.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25693822', '25694680', '.', '-', '.', 'gene_id \"chr37.g9531\"; transcript_id \"chr37.g9531.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25695544', '25695824', '.', '+', '.', 'gene_id \"chr37.g9533\"; transcript_id \"chr37.g9533.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25694701', '25694905', '.', '-', '.', 'gene_id \"chr37.g9534\"; transcript_id \"chr37.g9534.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25693477', '25693709', '.', '-', '.', 'gene_id \"chr37.g9536\"; transcript_id \"chr37.g9536.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25699186', '25699366', '.', '+', '.', 'gene_id \"chr37.g9539\"; transcript_id \"chr37.g9539.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25698951', '25699101', '.', '+', '.', 'gene_id \"chr37.g9539\"; transcript_id \"chr37.g9539.i5\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25708996', '25709116', '.', '+', '.', 'gene_id \"chr37.g9543\"; transcript_id \"chr37.g9543.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25713663', '25714215', '.', '+', '.', 'gene_id \"chr37.g9544\"; transcript_id \"chr37.g9544.i11\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25713663', '25714176', '.', '+', '.', 'gene_id \"chr37.g9544\"; transcript_id \"chr37.g9544.i12\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25715875', '25716119', '.', '+', '.', 'gene_id \"chr37.g9544\"; transcript_id \"chr37.g9544.i13\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25716540', '25716575', '.', '+', '.', 'gene_id \"chr37.g9544\"; transcript_id \"chr37.g9544.i13\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25714415', '25714552', '.', '-', '.', 'gene_id \"chr37.g9545\"; transcript_id \"chr37.g9545.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25717717', '25718112', '.', '-', '.', 'gene_id \"chr37.g9548\"; transcript_id \"chr37.g9548.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25719727', '25719880', '.', '-', '.', 'gene_id \"chr37.g9551\"; transcript_id \"chr37.g9551.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25720635', '25720860', '.', '+', '.', 'gene_id \"chr37.g9552\"; transcript_id \"chr37.g9552.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25720945', '25721279', '.', '+', '.', 'gene_id \"chr37.g9554\"; transcript_id \"chr37.g9554.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25721277', '25721734', '.', '-', '.', 'gene_id \"chr37.g9555\"; transcript_id \"chr37.g9555.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25722046', '25722314', '.', '-', '.', 'gene_id \"chr37.g9557\"; transcript_id \"chr37.g9557.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25724442', '25724697', '.', '+', '.', 'gene_id \"chr37.g9558\"; transcript_id \"chr37.g9558.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25724708', '25724865', '.', '+', '.', 'gene_id \"chr37.g9559\"; transcript_id \"chr37.g9559.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25725060', '25725180', '.', '+', '.', 'gene_id \"chr37.g9560\"; transcript_id \"chr37.g9560.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25725804', '25726194', '.', '+', '.', 'gene_id \"chr37.g9563\"; transcript_id \"chr37.g9563.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25726246', '25726558', '.', '-', '.', 'gene_id \"chr37.g9564\"; transcript_id \"chr37.g9564.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25726544', '25726776', '.', '+', '.', 'gene_id \"chr37.g9565\"; transcript_id \"chr37.g9565.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25726847', '25727016', '.', '+', '.', 'gene_id \"chr37.g9566\"; transcript_id \"chr37.g9566.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25730280', '25730831', '.', '+', '.', 'gene_id \"chr37.g9569\"; transcript_id \"chr37.g9569.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25730071', '25730258', '.', '+', '.', 'gene_id \"chr37.g9572\"; transcript_id \"chr37.g9572.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25730880', '25731191', '.', '+', '.', 'gene_id \"chr37.g9573\"; transcript_id \"chr37.g9573.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25732748', '25733050', '.', '-', '.', 'gene_id \"chr37.g9579\"; transcript_id \"chr37.g9579.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25734135', '25735259', '.', '-', '.', 'gene_id \"chr37.g9580\"; transcript_id \"chr37.g9580.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25745264', '25745459', '.', '+', '.', 'gene_id \"chr37.g9586\"; transcript_id \"chr37.g9586.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25745463', '25745738', '.', '+', '.', 'gene_id \"chr37.g9588\"; transcript_id \"chr37.g9588.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25747565', '25748781', '.', '+', '.', 'gene_id \"chr37.g9590\"; transcript_id \"chr37.g9590.i4\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25747375', '25748781', '.', '+', '.', 'gene_id \"chr37.g9590\"; transcript_id \"chr37.g9590.i9\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25749078', '25749818', '.', '+', '.', 'gene_id \"chr37.g9590\"; transcript_id \"chr37.g9590.i10\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25746801', '25746997', '.', '+', '.', 'gene_id \"chr37.g9591\"; transcript_id \"chr37.g9591.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25750265', '25750419', '.', '+', '.', 'gene_id \"chr37.g9593\"; transcript_id \"chr37.g9593.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25749867', '25750144', '.', '-', '.', 'gene_id \"chr37.g9594\"; transcript_id \"chr37.g9594.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25748790', '25748999', '.', '-', '.', 'gene_id \"chr37.g9595\"; transcript_id \"chr37.g9595.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25753845', '25754103', '.', '+', '.', 'gene_id \"chr37.g9599\"; transcript_id \"chr37.g9599.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25753621', '25753859', '.', '+', '.', 'gene_id \"chr37.g9600\"; transcript_id \"chr37.g9600.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25754148', '25754322', '.', '-', '.', 'gene_id \"chr37.g9601\"; transcript_id \"chr37.g9601.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25754465', '25754713', '.', '-', '.', 'gene_id \"chr37.g9602\"; transcript_id \"chr37.g9602.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25756409', '25756818', '.', '+', '.', 'gene_id \"chr37.g9603\"; transcript_id \"chr37.g9603.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25756945', '25758963', '.', '+', '.', 'gene_id \"chr37.g9605\"; transcript_id \"chr37.g9605.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25760422', '25760628', '.', '+', '.', 'gene_id \"chr37.g9612\"; transcript_id \"chr37.g9612.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25769294', '25769536', '.', '-', '.', 'gene_id \"chr37.g9613\"; transcript_id \"chr37.g9613.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25771897', '25772282', '.', '+', '.', 'gene_id \"chr37.g9616\"; transcript_id \"chr37.g9616.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25788642', '25789092', '.', '+', '.', 'gene_id \"chr37.g9619\"; transcript_id \"chr37.g9619.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25818192', '25818845', '.', '-', '.', 'gene_id \"chr37.g9627\"; transcript_id \"chr37.g9627.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25827781', '25828411', '.', '-', '.', 'gene_id \"chr37.g9629\"; transcript_id \"chr37.g9629.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25821491', '25822143', '.', '-', '.', 'gene_id \"chr37.g9630\"; transcript_id \"chr37.g9630.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25822921', '25823127', '.', '+', '.', 'gene_id \"chr37.g9637\"; transcript_id \"chr37.g9637.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25824928', '25825109', '.', '+', '.', 'gene_id \"chr37.g9638\"; transcript_id \"chr37.g9638.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25849003', '25849440', '.', '+', '.', 'gene_id \"chr37.g9640\"; transcript_id \"chr37.g9640.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25849979', '25850199', '.', '+', '.', 'gene_id \"chr37.g9643\"; transcript_id \"chr37.g9643.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25850218', '25850370', '.', '-', '.', 'gene_id \"chr37.g9644\"; transcript_id \"chr37.g9644.i2\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25850359', '25850558', '.', '+', '.', 'gene_id \"chr37.g9645\"; transcript_id \"chr37.g9645.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25852009', '25852188', '.', '-', '.', 'gene_id \"chr37.g9648\"; transcript_id \"chr37.g9648.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25856713', '25856958', '.', '+', '.', 'gene_id \"chr37.g9651\"; transcript_id \"chr37.g9651.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25854205', '25854356', '.', '-', '.', 'gene_id \"chr37.g9653\"; transcript_id \"chr37.g9653.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25856961', '25858245', '.', '+', '.', 'gene_id \"chr37.g9656\"; transcript_id \"chr37.g9656.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25855008', '25855452', '.', '+', '.', 'gene_id \"chr37.g9657\"; transcript_id \"chr37.g9657.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25864030', '25864235', '.', '+', '.', 'gene_id \"chr37.g9660\"; transcript_id \"chr37.g9660.i1\";']\n",
      "negative:  ['chr37', 'PASA', 'exon', '25864263', '25864393', '.', '+', '.', 'gene_id \"chr37.g9663\"; transcript_id \"chr37.g9663.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '46985959', '46986962', '.', '+', '.', 'gene_id \"chr23.g17939\"; transcript_id \"chr23.g17939.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47039539', '47040420', '.', '-', '.', 'gene_id \"chr23.g18015\"; transcript_id \"chr23.g18015.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47039539', '47039968', '.', '-', '.', 'gene_id \"chr23.g18015\"; transcript_id \"chr23.g18015.i2\";']\n",
      "negative:  ['scaffold_1605', 'PASA', 'exon', '23703', '23858', '100', '+', '.', 'gene_id \"scaffold_1605.g3\"; transcript_id \"blat.proc4587.chain_6479\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47723978', '47726110', '.', '-', '.', 'gene_id \"chr23.g18193\"; transcript_id \"chr23.g18193.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47723978', '47724211', '.', '-', '.', 'gene_id \"chr23.g18193\"; transcript_id \"chr23.g18193.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47723977', '47724211', '.', '+', '.', 'gene_id \"chr23.g18194\"; transcript_id \"chr23.g18194.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16631', '16662', '.', '-', '.', 'gene_id \"chr9.g15\"; transcript_id \"chr9.g15.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '48527993', '48528029', '.', '+', '.', 'gene_id \"chr23.g18300\"; transcript_id \"chr23.g18300.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '140914', '140956', '.', '+', '.', 'gene_id \"chr9.g96\"; transcript_id \"chr9.g96.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '237211', '237241', '.', '+', '.', 'gene_id \"chr9.g176\"; transcript_id \"chr9.g176.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '283313', '283525', '.', '+', '.', 'gene_id \"chr9.g204\"; transcript_id \"chr9.g204.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '298575', '298863', '.', '+', '.', 'gene_id \"chr9.g213\"; transcript_id \"chr9.g213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '299214', '299408', '.', '-', '.', 'gene_id \"chr9.g215\"; transcript_id \"chr9.g215.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '299622', '299828', '.', '-', '.', 'gene_id \"chr9.g216\"; transcript_id \"chr9.g216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '326456', '326521', '.', '-', '.', 'gene_id \"chr9.g233\"; transcript_id \"chr9.g233.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '327985', '328683', '.', '-', '.', 'gene_id \"chr9.g233\"; transcript_id \"chr9.g233.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '319734', '319886', '.', '-', '.', 'gene_id \"chr9.g233\"; transcript_id \"chr9.g233.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '327438', '328683', '.', '+', '.', 'gene_id \"chr9.g234\"; transcript_id \"chr9.g234.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '322233', '325477', '.', '+', '.', 'gene_id \"chr9.g236\"; transcript_id \"chr9.g236.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49291105', '49291336', '.', '+', '.', 'gene_id \"chr23.g18562\"; transcript_id \"chr23.g18562.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49505590', '49505838', '.', '+', '.', 'gene_id \"chr23.g18615\"; transcript_id \"chr23.g18615.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49706288', '49707810', '.', '-', '.', 'gene_id \"chr23.g18663\"; transcript_id \"chr23.g18663.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49707937', '49708152', '.', '+', '.', 'gene_id \"chr23.g18665\"; transcript_id \"chr23.g18665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '562188', '562366', '.', '+', '.', 'gene_id \"chr9.g515\"; transcript_id \"chr9.g515.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '563005', '563214', '.', '-', '.', 'gene_id \"chr9.g517\"; transcript_id \"chr9.g517.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '587489', '587527', '.', '-', '.', 'gene_id \"chr9.g520\"; transcript_id \"chr9.g520.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '945598', '945750', '.', '+', '.', 'gene_id \"chr9.g794\"; transcript_id \"chr9.g794.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '958887', '959070', '.', '+', '.', 'gene_id \"chr9.g801\"; transcript_id \"chr9.g801.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1049179', '1049431', '.', '+', '.', 'gene_id \"chr9.g880\"; transcript_id \"chr9.g880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1062103', '1062865', '.', '+', '.', 'gene_id \"chr9.g893\"; transcript_id \"chr9.g893.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1252837', '1253101', '.', '+', '.', 'gene_id \"chr9.g1065\"; transcript_id \"chr9.g1065.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '50868699', '50869100', '.', '+', '.', 'gene_id \"chr23.g19174\"; transcript_id \"chr23.g19174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1509045', '1509197', '.', '+', '.', 'gene_id \"chr9.g1238\"; transcript_id \"chr9.g1238.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1508833', '1508997', '.', '+', '.', 'gene_id \"chr9.g1239\"; transcript_id \"chr9.g1239.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '51638980', '51639138', '.', '-', '.', 'gene_id \"chr23.g19324\"; transcript_id \"chr23.g19324.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '51831425', '51831694', '.', '+', '.', 'gene_id \"chr23.g19370\"; transcript_id \"chr23.g19370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1708494', '1708534', '.', '+', '.', 'gene_id \"chr9.g1457\"; transcript_id \"chr9.g1457.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '1768019', '1768184', '.', '+', '.', 'gene_id \"chr9.g1500\"; transcript_id \"chr9.g1500.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52223318', '52223591', '.', '-', '.', 'gene_id \"chr23.g19575\"; transcript_id \"chr23.g19575.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52245177', '52245220', '.', '+', '.', 'gene_id \"chr23.g19585\"; transcript_id \"chr23.g19585.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52245182', '52245220', '.', '+', '.', 'gene_id \"chr23.g19585\"; transcript_id \"chr23.g19585.i2\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52245187', '52245220', '.', '+', '.', 'gene_id \"chr23.g19585\"; transcript_id \"chr23.g19585.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '2020915', '2020945', '.', '+', '.', 'gene_id \"chr9.g1593\"; transcript_id \"chr9.g1593.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '2188248', '2188401', '.', '+', '.', 'gene_id \"chr9.g1720\"; transcript_id \"chr9.g1720.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52570302', '52570491', '.', '+', '.', 'gene_id \"chr23.g19764\"; transcript_id \"chr23.g19764.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52579019', '52579062', '.', '+', '.', 'gene_id \"chr23.g19766\"; transcript_id \"chr23.g19766.i1\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49850122', '49850187', '100', '-', '.', 'gene_id \"chr23.g20046\"; transcript_id \"blat.proc4005.chain_19659\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '31469632', '31469725', '100', '+', '.', 'gene_id \"chr23.g20526\"; transcript_id \"blat.proc18128.chain_19970\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2292308', '2292365', '100', '+', '.', 'gene_id \"chr23.g20609\"; transcript_id \"blat.proc4587.chain_1078\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49851433', '49851495', '100', '+', '.', 'gene_id \"chr23.g20678\"; transcript_id \"blat.proc18128.chain_2891\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '32527674', '32527772', '100', '+', '.', 'gene_id \"chr23.g20746\"; transcript_id \"blat.proc4587.chain_13842\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34756567', '34756637', '100', '-', '.', 'gene_id \"chr23.g20802\"; transcript_id \"blat.proc18128.chain_9730\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2166820', '2166911', '100', '-', '.', 'gene_id \"chr23.g20812\"; transcript_id \"blat.proc18128.chain_18745\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '35653319', '35653369', '100', '+', '.', 'gene_id \"chr23.g20869\"; transcript_id \"blat.proc4587.chain_1675\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '10180509', '10180580', '100', '-', '.', 'gene_id \"chr23.g20916\"; transcript_id \"blat.proc4587.chain_3685\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '37597383', '37597429', '100', '-', '.', 'gene_id \"chr23.g21073\"; transcript_id \"blat.proc4005.chain_9076\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '1131937', '1131974', '100', '-', '.', 'gene_id \"chr23.g21198\"; transcript_id \"blat.proc4587.chain_5662\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '25754512', '25754707', '100', '-', '.', 'gene_id \"chr23.g21537\"; transcript_id \"blat.proc4587.chain_1500\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2833694', '2833857', '100', '+', '.', 'gene_id \"chr23.g21728\"; transcript_id \"blat.proc4587.chain_753\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45817011', '45817074', '100', '+', '.', 'gene_id \"chr23.g21861\"; transcript_id \"blat.proc28144.chain_157\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '41069546', '41069595', '100', '-', '.', 'gene_id \"chr23.g22069\"; transcript_id \"blat.proc4005.chain_10633\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '13729370', '13729425', '100', '+', '.', 'gene_id \"chr23.g22127\"; transcript_id \"blat.proc18128.chain_10564\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '5150997', '5151099', '100', '-', '.', 'gene_id \"chr23.g22139\"; transcript_id \"blat.proc18128.chain_15832\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29610824', '29610858', '100', '+', '.', 'gene_id \"chr23.g22209\"; transcript_id \"blat.proc28144.chain_19345\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '7341421', '7341509', '100', '-', '.', 'gene_id \"chr23.g22237\"; transcript_id \"blat.proc18128.chain_24123\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2858627', '2858851', '100', '+', '.', 'gene_id \"chr23.g22245\"; transcript_id \"blat.proc4587.chain_754\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '52420090', '52420158', '100', '-', '.', 'gene_id \"chr23.g22608\"; transcript_id \"blat.proc18128.chain_8694\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '34941786', '34942099', '100', '+', '.', 'gene_id \"chr23.g22620\"; transcript_id \"blat.proc18128.chain_8488\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '10595737', '10595766', '100', '-', '.', 'gene_id \"chr23.g22633\"; transcript_id \"blat.proc28144.chain_22909\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '42229968', '42230003', '100', '-', '.', 'gene_id \"chr23.g22662\"; transcript_id \"blat.proc28144.chain_22618\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '13012785', '13012886', '100', '+', '.', 'gene_id \"chr23.g22740\"; transcript_id \"blat.proc4587.chain_8331\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '42960320', '42960372', '100', '+', '.', 'gene_id \"chr23.g22749\"; transcript_id \"blat.proc4005.chain_7482\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '22350173', '22350213', '100', '-', '.', 'gene_id \"chr23.g22802\"; transcript_id \"blat.proc18128.chain_3954\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '3056846', '3056895', '100', '+', '.', 'gene_id \"chr23.g22913\"; transcript_id \"blat.proc18128.chain_26283\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '47584419', '47584466', '100', '-', '.', 'gene_id \"chr23.g22923\"; transcript_id \"blat.proc28144.chain_8710\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20090055', '20090093', '100', '-', '.', 'gene_id \"chr23.g22948\"; transcript_id \"blat.proc18128.chain_24613\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '655742', '655853', '100', '-', '.', 'gene_id \"chr23.g22957\"; transcript_id \"blat.proc18128.chain_23174\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6817190', '6817401', '100', '+', '.', 'gene_id \"chr23.g23129\"; transcript_id \"blat.proc4587.chain_4371\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '51569452', '51569529', '100', '+', '.', 'gene_id \"chr23.g23131\"; transcript_id \"blat.proc18128.chain_15780\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2518120', '2518226', '100', '+', '.', 'gene_id \"chr23.g23494\"; transcript_id \"blat.proc18128.chain_22296\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '43104301', '43104344', '100', '-', '.', 'gene_id \"chr23.g23545\"; transcript_id \"blat.proc18128.chain_21604\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '15482650', '15482693', '100', '+', '.', 'gene_id \"chr23.g23600\"; transcript_id \"blat.proc18128.chain_20713\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '24643592', '24643643', '100', '+', '.', 'gene_id \"chr23.g23605\"; transcript_id \"blat.proc28144.chain_4738\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '4318355', '4318397', '100', '+', '.', 'gene_id \"chr23.g23849\"; transcript_id \"blat.proc28144.chain_11869\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6228638', '6228710', '100', '-', '.', 'gene_id \"chr23.g24263\"; transcript_id \"blat.proc18128.chain_6300\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '45870999', '45871081', '100', '-', '.', 'gene_id \"chr23.g24312\"; transcript_id \"blat.proc18128.chain_9499\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6702458', '6702608', '100', '-', '.', 'gene_id \"chr23.g24318\"; transcript_id \"blat.proc4587.chain_1899\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '14563647', '14563696', '100', '-', '.', 'gene_id \"chr23.g24539\"; transcript_id \"blat.proc18128.chain_7350\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '12273918', '12273960', '100', '-', '.', 'gene_id \"chr23.g24620\"; transcript_id \"blat.proc4587.chain_3834\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '51791134', '51791163', '100', '-', '.', 'gene_id \"chr23.g24684\"; transcript_id \"blat.proc18128.chain_7559\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '9321324', '9321356', '100', '-', '.', 'gene_id \"chr23.g24732\"; transcript_id \"blat.proc18128.chain_17787\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '30598187', '30598232', '100', '+', '.', 'gene_id \"chr23.g24761\"; transcript_id \"blat.proc4005.chain_11062\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '20108852', '20108909', '100', '-', '.', 'gene_id \"chr23.g24828\"; transcript_id \"blat.proc18128.chain_12245\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '16954851', '16954893', '100', '-', '.', 'gene_id \"chr23.g24934\"; transcript_id \"blat.proc18128.chain_5330\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '6227660', '6228193', '100', '+', '.', 'gene_id \"chr23.g25049\"; transcript_id \"blat.proc18128.chain_3530\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '49850497', '49850532', '100', '+', '.', 'gene_id \"chr23.g25104\"; transcript_id \"blat.proc28144.chain_13592\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '29293010', '29293046', '100', '-', '.', 'gene_id \"chr23.g25126\"; transcript_id \"blat.proc4005.chain_5250\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '19848192', '19848295', '100', '-', '.', 'gene_id \"chr23.g25261\"; transcript_id \"blat.proc18128.chain_25961\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '2783291', '2783328', '100', '+', '.', 'gene_id \"chr23.g25283\"; transcript_id \"blat.proc28144.chain_18995\";']\n",
      "negative:  ['chr23', 'PASA', 'exon', '17862593', '17862629', '100', '-', '.', 'gene_id \"chr23.g25433\"; transcript_id \"blat.proc18128.chain_5491\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '2346785', '2346969', '.', '-', '.', 'gene_id \"chr9.g1814\"; transcript_id \"chr9.g1814.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '2351334', '2351500', '.', '+', '.', 'gene_id \"chr9.g1818\"; transcript_id \"chr9.g1818.i1\";']\n",
      "negative:  ['scaffold_940', 'PASA', 'exon', '4484', '4516', '.', '+', '.', 'gene_id \"scaffold_940.g4\"; transcript_id \"scaffold_940.g4.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3285099', '3285291', '.', '+', '.', 'gene_id \"chr9.g2289\"; transcript_id \"chr9.g2289.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3320531', '3320804', '.', '-', '.', 'gene_id \"chr9.g2331\"; transcript_id \"chr9.g2331.i1\";']\n",
      "negative:  ['scaffold_1589', 'PASA', 'exon', '24316', '24381', '.', '+', '.', 'gene_id \"scaffold_1589.g1\"; transcript_id \"scaffold_1589.g1.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3518690', '3518908', '.', '-', '.', 'gene_id \"chr9.g2443\"; transcript_id \"chr9.g2443.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3537422', '3537602', '.', '-', '.', 'gene_id \"chr9.g2448\"; transcript_id \"chr9.g2448.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3574642', '3574674', '.', '+', '.', 'gene_id \"chr9.g2458\"; transcript_id \"chr9.g2458.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3710946', '3711002', '.', '+', '.', 'gene_id \"chr9.g2576\"; transcript_id \"chr9.g2576.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3731437', '3731519', '.', '+', '.', 'gene_id \"chr9.g2596\"; transcript_id \"chr9.g2596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3805853', '3805923', '.', '+', '.', 'gene_id \"chr9.g2687\"; transcript_id \"chr9.g2687.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3808656', '3809081', '.', '-', '.', 'gene_id \"chr9.g2689\"; transcript_id \"chr9.g2689.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '468833', '469012', '.', '+', '.', 'gene_id \"chr17.g110\"; transcript_id \"chr17.g110.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4001726', '4001756', '.', '-', '.', 'gene_id \"chr9.g2817\"; transcript_id \"chr9.g2817.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '848509', '848698', '.', '+', '.', 'gene_id \"chr17.g254\"; transcript_id \"chr17.g254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4416385', '4416687', '.', '-', '.', 'gene_id \"chr9.g3034\"; transcript_id \"chr9.g3034.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4459670', '4460435', '.', '+', '.', 'gene_id \"chr9.g3088\"; transcript_id \"chr9.g3088.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '1893570', '1893737', '.', '-', '.', 'gene_id \"chr17.g475\"; transcript_id \"chr17.g475.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '1923931', '1924257', '.', '+', '.', 'gene_id \"chr17.g483\"; transcript_id \"chr17.g483.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4572781', '4572901', '.', '+', '.', 'gene_id \"chr9.g3160\"; transcript_id \"chr9.g3160.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '2001812', '2001993', '.', '-', '.', 'gene_id \"chr17.g533\"; transcript_id \"chr17.g533.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '2002496', '2002732', '.', '-', '.', 'gene_id \"chr17.g534\"; transcript_id \"chr17.g534.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4638933', '4639093', '.', '+', '.', 'gene_id \"chr9.g3204\"; transcript_id \"chr9.g3204.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '2076901', '2077064', '.', '-', '.', 'gene_id \"chr17.g606\"; transcript_id \"chr17.g606.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4861268', '4862795', '.', '+', '.', 'gene_id \"chr9.g3344\"; transcript_id \"chr9.g3344.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4850986', '4851583', '.', '+', '.', 'gene_id \"chr9.g3344\"; transcript_id \"chr9.g3344.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4863475', '4864087', '.', '+', '.', 'gene_id \"chr9.g3344\"; transcript_id \"chr9.g3344.i21\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4862813', '4863496', '.', '-', '.', 'gene_id \"chr9.g3354\"; transcript_id \"chr9.g3354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4855242', '4855553', '.', '-', '.', 'gene_id \"chr9.g3355\"; transcript_id \"chr9.g3355.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '2616819', '2616976', '.', '-', '.', 'gene_id \"chr17.g710\"; transcript_id \"chr17.g710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4963249', '4963406', '.', '-', '.', 'gene_id \"chr9.g3467\"; transcript_id \"chr9.g3467.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4988797', '4989082', '.', '+', '.', 'gene_id \"chr9.g3515\"; transcript_id \"chr9.g3515.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4989077', '4989519', '.', '-', '.', 'gene_id \"chr9.g3516\"; transcript_id \"chr9.g3516.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '4605607', '4605829', '.', '+', '.', 'gene_id \"chr17.g955\"; transcript_id \"chr17.g955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5224773', '5224992', '.', '+', '.', 'gene_id \"chr9.g3684\"; transcript_id \"chr9.g3684.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '5688370', '5688507', '.', '+', '.', 'gene_id \"chr17.g1149\"; transcript_id \"chr17.g1149.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5614281', '5614312', '.', '+', '.', 'gene_id \"chr9.g4041\"; transcript_id \"chr9.g4041.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5670309', '5670379', '.', '-', '.', 'gene_id \"chr9.g4084\"; transcript_id \"chr9.g4084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5715375', '5715615', '.', '+', '.', 'gene_id \"chr9.g4122\"; transcript_id \"chr9.g4122.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5715375', '5715811', '.', '+', '.', 'gene_id \"chr9.g4122\"; transcript_id \"chr9.g4122.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5757075', '5757241', '.', '-', '.', 'gene_id \"chr9.g4163\"; transcript_id \"chr9.g4163.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5879786', '5880858', '.', '-', '.', 'gene_id \"chr9.g4318\"; transcript_id \"chr9.g4318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5880910', '5881343', '.', '-', '.', 'gene_id \"chr9.g4320\"; transcript_id \"chr9.g4320.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '6356210', '6356503', '.', '-', '.', 'gene_id \"chr17.g1481\"; transcript_id \"chr17.g1481.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5894287', '5894388', '.', '+', '.', 'gene_id \"chr9.g4329\"; transcript_id \"chr9.g4329.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5950053', '5950296', '.', '+', '.', 'gene_id \"chr9.g4392\"; transcript_id \"chr9.g4392.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '6531256', '6531896', '.', '-', '.', 'gene_id \"chr17.g1647\"; transcript_id \"chr17.g1647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6072602', '6072646', '.', '-', '.', 'gene_id \"chr9.g4538\"; transcript_id \"chr9.g4538.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6072605', '6072644', '.', '-', '.', 'gene_id \"chr9.g4538\"; transcript_id \"chr9.g4538.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6072704', '6072907', '.', '+', '.', 'gene_id \"chr9.g4540\"; transcript_id \"chr9.g4540.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6128507', '6128754', '.', '+', '.', 'gene_id \"chr9.g4585\"; transcript_id \"chr9.g4585.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6198304', '6198506', '.', '+', '.', 'gene_id \"chr9.g4660\"; transcript_id \"chr9.g4660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6198507', '6198951', '.', '+', '.', 'gene_id \"chr9.g4661\"; transcript_id \"chr9.g4661.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6198949', '6199452', '.', '-', '.', 'gene_id \"chr9.g4662\"; transcript_id \"chr9.g4662.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '6902945', '6903127', '.', '+', '.', 'gene_id \"chr17.g1864\"; transcript_id \"chr17.g1864.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '6995686', '6997603', '.', '+', '.', 'gene_id \"chr17.g1898\"; transcript_id \"chr17.g1898.i3\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '6985776', '6986129', '.', '+', '.', 'gene_id \"chr17.g1898\"; transcript_id \"chr17.g1898.i5\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '7015578', '7015613', '.', '+', '.', 'gene_id \"chr17.g1917\"; transcript_id \"chr17.g1917.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '7243366', '7243535', '.', '-', '.', 'gene_id \"chr17.g1999\"; transcript_id \"chr17.g1999.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '7243588', '7243746', '.', '-', '.', 'gene_id \"chr17.g2001\"; transcript_id \"chr17.g2001.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7336392', '7336627', '.', '+', '.', 'gene_id \"chr9.g5048\"; transcript_id \"chr9.g5048.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7356241', '7356423', '.', '-', '.', 'gene_id \"chr9.g5049\"; transcript_id \"chr9.g5049.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7425275', '7425904', '.', '+', '.', 'gene_id \"chr9.g5054\"; transcript_id \"chr9.g5054.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7426500', '7426826', '.', '-', '.', 'gene_id \"chr9.g5056\"; transcript_id \"chr9.g5056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7429143', '7429575', '.', '-', '.', 'gene_id \"chr9.g5058\"; transcript_id \"chr9.g5058.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7429595', '7429756', '.', '-', '.', 'gene_id \"chr9.g5059\"; transcript_id \"chr9.g5059.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7474089', '7474280', '.', '-', '.', 'gene_id \"chr9.g5060\"; transcript_id \"chr9.g5060.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7548583', '7548814', '.', '-', '.', 'gene_id \"chr9.g5073\"; transcript_id \"chr9.g5073.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7554873', '7555033', '.', '-', '.', 'gene_id \"chr9.g5077\"; transcript_id \"chr9.g5077.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7560798', '7560973', '.', '-', '.', 'gene_id \"chr9.g5083\"; transcript_id \"chr9.g5083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7560997', '7561154', '.', '-', '.', 'gene_id \"chr9.g5084\"; transcript_id \"chr9.g5084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7569379', '7569562', '.', '+', '.', 'gene_id \"chr9.g5086\"; transcript_id \"chr9.g5086.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7571112', '7571386', '.', '+', '.', 'gene_id \"chr9.g5091\"; transcript_id \"chr9.g5091.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7571424', '7571584', '.', '+', '.', 'gene_id \"chr9.g5092\"; transcript_id \"chr9.g5092.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7572145', '7572299', '.', '+', '.', 'gene_id \"chr9.g5094\"; transcript_id \"chr9.g5094.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7575289', '7575654', '.', '-', '.', 'gene_id \"chr9.g5098\"; transcript_id \"chr9.g5098.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7576318', '7576514', '.', '+', '.', 'gene_id \"chr9.g5099\"; transcript_id \"chr9.g5099.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7576600', '7576772', '.', '+', '.', 'gene_id \"chr9.g5100\"; transcript_id \"chr9.g5100.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7576874', '7577164', '.', '-', '.', 'gene_id \"chr9.g5101\"; transcript_id \"chr9.g5101.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7577185', '7577531', '.', '+', '.', 'gene_id \"chr9.g5102\"; transcript_id \"chr9.g5102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7577522', '7577682', '.', '-', '.', 'gene_id \"chr9.g5103\"; transcript_id \"chr9.g5103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7618525', '7620890', '.', '-', '.', 'gene_id \"chr9.g5106\"; transcript_id \"chr9.g5106.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7620495', '7620890', '.', '-', '.', 'gene_id \"chr9.g5106\"; transcript_id \"chr9.g5106.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7599494', '7599653', '.', '-', '.', 'gene_id \"chr9.g5109\"; transcript_id \"chr9.g5109.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7620881', '7621294', '.', '-', '.', 'gene_id \"chr9.g5110\"; transcript_id \"chr9.g5110.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7601076', '7601248', '.', '-', '.', 'gene_id \"chr9.g5111\"; transcript_id \"chr9.g5111.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7601557', '7601718', '.', '-', '.', 'gene_id \"chr9.g5112\"; transcript_id \"chr9.g5112.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7602898', '7603068', '.', '+', '.', 'gene_id \"chr9.g5115\"; transcript_id \"chr9.g5115.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7606494', '7606824', '.', '-', '.', 'gene_id \"chr9.g5121\"; transcript_id \"chr9.g5121.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7606651', '7606824', '.', '-', '.', 'gene_id \"chr9.g5121\"; transcript_id \"chr9.g5121.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7606896', '7607466', '.', '-', '.', 'gene_id \"chr9.g5122\"; transcript_id \"chr9.g5122.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7606913', '7607466', '.', '-', '.', 'gene_id \"chr9.g5122\"; transcript_id \"chr9.g5122.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7607470', '7607958', '.', '-', '.', 'gene_id \"chr9.g5123\"; transcript_id \"chr9.g5123.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7608176', '7608522', '.', '+', '.', 'gene_id \"chr9.g5126\"; transcript_id \"chr9.g5126.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7609380', '7609563', '.', '-', '.', 'gene_id \"chr9.g5127\"; transcript_id \"chr9.g5127.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7610217', '7610385', '.', '+', '.', 'gene_id \"chr9.g5131\"; transcript_id \"chr9.g5131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7610370', '7610831', '.', '-', '.', 'gene_id \"chr9.g5132\"; transcript_id \"chr9.g5132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7611015', '7611330', '.', '+', '.', 'gene_id \"chr9.g5133\"; transcript_id \"chr9.g5133.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7612023', '7612229', '.', '-', '.', 'gene_id \"chr9.g5134\"; transcript_id \"chr9.g5134.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7612408', '7612450', '.', '-', '.', 'gene_id \"chr9.g5135\"; transcript_id \"chr9.g5135.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7615156', '7615473', '.', '-', '.', 'gene_id \"chr9.g5139\"; transcript_id \"chr9.g5139.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7615955', '7616119', '.', '+', '.', 'gene_id \"chr9.g5141\"; transcript_id \"chr9.g5141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7616332', '7616526', '.', '-', '.', 'gene_id \"chr9.g5142\"; transcript_id \"chr9.g5142.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7617059', '7617220', '.', '-', '.', 'gene_id \"chr9.g5144\"; transcript_id \"chr9.g5144.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7621697', '7621937', '.', '-', '.', 'gene_id \"chr9.g5145\"; transcript_id \"chr9.g5145.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7650138', '7650491', '.', '+', '.', 'gene_id \"chr9.g5158\"; transcript_id \"chr9.g5158.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7662562', '7663089', '.', '-', '.', 'gene_id \"chr9.g5162\"; transcript_id \"chr9.g5162.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7653636', '7653939', '.', '+', '.', 'gene_id \"chr9.g5163\"; transcript_id \"chr9.g5163.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7653098', '7653939', '.', '+', '.', 'gene_id \"chr9.g5163\"; transcript_id \"chr9.g5163.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7680934', '7681444', '.', '+', '.', 'gene_id \"chr9.g5167\"; transcript_id \"chr9.g5167.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7685203', '7685378', '.', '+', '.', 'gene_id \"chr9.g5174\"; transcript_id \"chr9.g5174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7684445', '7684857', '.', '+', '.', 'gene_id \"chr9.g5175\"; transcript_id \"chr9.g5175.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7682496', '7682651', '.', '-', '.', 'gene_id \"chr9.g5176\"; transcript_id \"chr9.g5176.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7680390', '7680865', '.', '+', '.', 'gene_id \"chr9.g5178\"; transcript_id \"chr9.g5178.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7667061', '7667215', '.', '-', '.', 'gene_id \"chr9.g5188\"; transcript_id \"chr9.g5188.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7667232', '7667443', '.', '+', '.', 'gene_id \"chr9.g5189\"; transcript_id \"chr9.g5189.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7667232', '7667879', '.', '-', '.', 'gene_id \"chr9.g5190\"; transcript_id \"chr9.g5190.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7667442', '7667879', '.', '+', '.', 'gene_id \"chr9.g5191\"; transcript_id \"chr9.g5191.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7668518', '7668744', '.', '+', '.', 'gene_id \"chr9.g5193\"; transcript_id \"chr9.g5193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7671165', '7671562', '.', '+', '.', 'gene_id \"chr9.g5196\"; transcript_id \"chr9.g5196.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7671740', '7672169', '.', '+', '.', 'gene_id \"chr9.g5199\"; transcript_id \"chr9.g5199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7674694', '7674994', '.', '+', '.', 'gene_id \"chr9.g5203\"; transcript_id \"chr9.g5203.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7675957', '7676114', '.', '+', '.', 'gene_id \"chr9.g5205\"; transcript_id \"chr9.g5205.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7690754', '7690858', '.', '+', '.', 'gene_id \"chr9.g5210\"; transcript_id \"chr9.g5210.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7690990', '7691248', '.', '-', '.', 'gene_id \"chr9.g5212\"; transcript_id \"chr9.g5212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7693043', '7693408', '.', '+', '.', 'gene_id \"chr9.g5214\"; transcript_id \"chr9.g5214.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7694957', '7695199', '.', '-', '.', 'gene_id \"chr9.g5218\"; transcript_id \"chr9.g5218.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7699085', '7699322', '.', '+', '.', 'gene_id \"chr9.g5222\"; transcript_id \"chr9.g5222.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7701334', '7701656', '.', '-', '.', 'gene_id \"chr9.g5227\"; transcript_id \"chr9.g5227.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7872060', '7872179', '.', '+', '.', 'gene_id \"chr9.g5243\"; transcript_id \"chr9.g5243.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7959169', '7959241', '.', '+', '.', 'gene_id \"chr9.g5243\"; transcript_id \"chr9.g5243.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8082464', '8082734', '.', '+', '.', 'gene_id \"chr9.g5243\"; transcript_id \"chr9.g5243.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8175039', '8175231', '.', '+', '.', 'gene_id \"chr9.g5248\"; transcript_id \"chr9.g5248.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7776981', '7777195', '.', '+', '.', 'gene_id \"chr9.g5255\"; transcript_id \"chr9.g5255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7778072', '7778407', '.', '+', '.', 'gene_id \"chr9.g5258\"; transcript_id \"chr9.g5258.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7778458', '7778772', '.', '+', '.', 'gene_id \"chr9.g5259\"; transcript_id \"chr9.g5259.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7786987', '7787368', '.', '+', '.', 'gene_id \"chr9.g5268\"; transcript_id \"chr9.g5268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7787380', '7787626', '.', '+', '.', 'gene_id \"chr9.g5270\"; transcript_id \"chr9.g5270.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7791319', '7791533', '.', '-', '.', 'gene_id \"chr9.g5274\"; transcript_id \"chr9.g5274.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7792262', '7792653', '.', '-', '.', 'gene_id \"chr9.g5275\"; transcript_id \"chr9.g5275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7792646', '7792873', '.', '+', '.', 'gene_id \"chr9.g5276\"; transcript_id \"chr9.g5276.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7793315', '7793518', '.', '-', '.', 'gene_id \"chr9.g5277\"; transcript_id \"chr9.g5277.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7795287', '7795530', '.', '-', '.', 'gene_id \"chr9.g5279\"; transcript_id \"chr9.g5279.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7795691', '7795957', '.', '-', '.', 'gene_id \"chr9.g5280\"; transcript_id \"chr9.g5280.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7797222', '7797620', '.', '-', '.', 'gene_id \"chr9.g5284\"; transcript_id \"chr9.g5284.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '7990111', '7990178', '.', '-', '.', 'gene_id \"chr17.g2426\"; transcript_id \"chr17.g2426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7813494', '7813653', '.', '+', '.', 'gene_id \"chr9.g5291\"; transcript_id \"chr9.g5291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7829753', '7829926', '.', '+', '.', 'gene_id \"chr9.g5297\"; transcript_id \"chr9.g5297.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7842750', '7843061', '.', '-', '.', 'gene_id \"chr9.g5299\"; transcript_id \"chr9.g5299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7844074', '7844298', '.', '-', '.', 'gene_id \"chr9.g5303\"; transcript_id \"chr9.g5303.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7844570', '7844762', '.', '+', '.', 'gene_id \"chr9.g5304\"; transcript_id \"chr9.g5304.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7846300', '7846487', '.', '-', '.', 'gene_id \"chr9.g5305\"; transcript_id \"chr9.g5305.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7846884', '7847940', '.', '-', '.', 'gene_id \"chr9.g5306\"; transcript_id \"chr9.g5306.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7846884', '7847508', '.', '-', '.', 'gene_id \"chr9.g5306\"; transcript_id \"chr9.g5306.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7848009', '7848160', '.', '+', '.', 'gene_id \"chr9.g5308\"; transcript_id \"chr9.g5308.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7860023', '7860186', '.', '-', '.', 'gene_id \"chr9.g5312\"; transcript_id \"chr9.g5312.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7860256', '7860535', '.', '-', '.', 'gene_id \"chr9.g5313\"; transcript_id \"chr9.g5313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7860536', '7860704', '.', '-', '.', 'gene_id \"chr9.g5314\"; transcript_id \"chr9.g5314.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7862890', '7863050', '.', '+', '.', 'gene_id \"chr9.g5316\"; transcript_id \"chr9.g5316.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7869892', '7870085', '.', '+', '.', 'gene_id \"chr9.g5320\"; transcript_id \"chr9.g5320.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7872761', '7872967', '.', '-', '.', 'gene_id \"chr9.g5321\"; transcript_id \"chr9.g5321.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7881601', '7881633', '.', '-', '.', 'gene_id \"chr9.g5325\"; transcript_id \"chr9.g5325.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7883724', '7883920', '.', '+', '.', 'gene_id \"chr9.g5328\"; transcript_id \"chr9.g5328.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7884804', '7885104', '.', '-', '.', 'gene_id \"chr9.g5331\"; transcript_id \"chr9.g5331.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7885099', '7885625', '.', '-', '.', 'gene_id \"chr9.g5332\"; transcript_id \"chr9.g5332.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7888518', '7888684', '.', '+', '.', 'gene_id \"chr9.g5335\"; transcript_id \"chr9.g5335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7890370', '7890527', '.', '-', '.', 'gene_id \"chr9.g5336\"; transcript_id \"chr9.g5336.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7890894', '7891117', '.', '-', '.', 'gene_id \"chr9.g5338\"; transcript_id \"chr9.g5338.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7891314', '7891519', '.', '+', '.', 'gene_id \"chr9.g5339\"; transcript_id \"chr9.g5339.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7900257', '7900777', '.', '-', '.', 'gene_id \"chr9.g5343\"; transcript_id \"chr9.g5343.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7910356', '7910677', '.', '-', '.', 'gene_id \"chr9.g5348\"; transcript_id \"chr9.g5348.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7967832', '7967994', '.', '+', '.', 'gene_id \"chr9.g5364\"; transcript_id \"chr9.g5364.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7969666', '7969873', '.', '-', '.', 'gene_id \"chr9.g5366\"; transcript_id \"chr9.g5366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7987715', '7987889', '.', '+', '.', 'gene_id \"chr9.g5376\"; transcript_id \"chr9.g5376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7988442', '7988695', '.', '+', '.', 'gene_id \"chr9.g5377\"; transcript_id \"chr9.g5377.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7992425', '7992582', '.', '+', '.', 'gene_id \"chr9.g5379\"; transcript_id \"chr9.g5379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7992623', '7992954', '.', '-', '.', 'gene_id \"chr9.g5380\"; transcript_id \"chr9.g5380.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7993852', '7994069', '.', '+', '.', 'gene_id \"chr9.g5382\"; transcript_id \"chr9.g5382.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7995290', '7995516', '.', '+', '.', 'gene_id \"chr9.g5387\"; transcript_id \"chr9.g5387.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7998025', '7998295', '.', '+', '.', 'gene_id \"chr9.g5390\"; transcript_id \"chr9.g5390.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8001183', '8001354', '.', '+', '.', 'gene_id \"chr9.g5392\"; transcript_id \"chr9.g5392.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8001552', '8001944', '.', '-', '.', 'gene_id \"chr9.g5395\"; transcript_id \"chr9.g5395.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8001928', '8002220', '.', '+', '.', 'gene_id \"chr9.g5396\"; transcript_id \"chr9.g5396.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8002524', '8002853', '.', '-', '.', 'gene_id \"chr9.g5398\"; transcript_id \"chr9.g5398.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8002851', '8003030', '.', '+', '.', 'gene_id \"chr9.g5399\"; transcript_id \"chr9.g5399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8003404', '8003585', '.', '-', '.', 'gene_id \"chr9.g5400\"; transcript_id \"chr9.g5400.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '8163895', '8164601', '.', '+', '.', 'gene_id \"chr17.g2544\"; transcript_id \"chr17.g2544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8007300', '8007592', '.', '-', '.', 'gene_id \"chr9.g5402\"; transcript_id \"chr9.g5402.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8007054', '8007592', '.', '-', '.', 'gene_id \"chr9.g5402\"; transcript_id \"chr9.g5402.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8009559', '8009857', '.', '-', '.', 'gene_id \"chr9.g5405\"; transcript_id \"chr9.g5405.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8009847', '8009999', '.', '-', '.', 'gene_id \"chr9.g5406\"; transcript_id \"chr9.g5406.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8015634', '8015824', '.', '+', '.', 'gene_id \"chr9.g5408\"; transcript_id \"chr9.g5408.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8015807', '8015980', '.', '-', '.', 'gene_id \"chr9.g5409\"; transcript_id \"chr9.g5409.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8016215', '8016372', '.', '-', '.', 'gene_id \"chr9.g5411\"; transcript_id \"chr9.g5411.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8018529', '8018745', '.', '+', '.', 'gene_id \"chr9.g5414\"; transcript_id \"chr9.g5414.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8018753', '8018958', '.', '-', '.', 'gene_id \"chr9.g5415\"; transcript_id \"chr9.g5415.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8035827', '8036006', '.', '+', '.', 'gene_id \"chr9.g5419\"; transcript_id \"chr9.g5419.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8038035', '8038242', '.', '+', '.', 'gene_id \"chr9.g5421\"; transcript_id \"chr9.g5421.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8038440', '8038595', '.', '+', '.', 'gene_id \"chr9.g5423\"; transcript_id \"chr9.g5423.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8038666', '8038823', '.', '-', '.', 'gene_id \"chr9.g5425\"; transcript_id \"chr9.g5425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8040026', '8040280', '.', '-', '.', 'gene_id \"chr9.g5427\"; transcript_id \"chr9.g5427.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8040352', '8040753', '.', '-', '.', 'gene_id \"chr9.g5429\"; transcript_id \"chr9.g5429.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8041228', '8041419', '.', '-', '.', 'gene_id \"chr9.g5431\"; transcript_id \"chr9.g5431.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8044458', '8044610', '.', '-', '.', 'gene_id \"chr9.g5434\"; transcript_id \"chr9.g5434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8044957', '8045257', '.', '-', '.', 'gene_id \"chr9.g5435\"; transcript_id \"chr9.g5435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8044613', '8044981', '.', '-', '.', 'gene_id \"chr9.g5436\"; transcript_id \"chr9.g5436.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8050590', '8050741', '.', '-', '.', 'gene_id \"chr9.g5439\"; transcript_id \"chr9.g5439.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8051701', '8052242', '.', '-', '.', 'gene_id \"chr9.g5442\"; transcript_id \"chr9.g5442.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8058220', '8058546', '.', '+', '.', 'gene_id \"chr9.g5451\"; transcript_id \"chr9.g5451.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8058577', '8058750', '.', '+', '.', 'gene_id \"chr9.g5452\"; transcript_id \"chr9.g5452.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8059320', '8059833', '.', '-', '.', 'gene_id \"chr9.g5453\"; transcript_id \"chr9.g5453.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8064857', '8065013', '.', '+', '.', 'gene_id \"chr9.g5455\"; transcript_id \"chr9.g5455.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8068208', '8068395', '.', '-', '.', 'gene_id \"chr9.g5458\"; transcript_id \"chr9.g5458.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8071228', '8071430', '.', '+', '.', 'gene_id \"chr9.g5460\"; transcript_id \"chr9.g5460.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8072294', '8072459', '.', '+', '.', 'gene_id \"chr9.g5462\"; transcript_id \"chr9.g5462.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8072799', '8072961', '.', '-', '.', 'gene_id \"chr9.g5463\"; transcript_id \"chr9.g5463.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8073156', '8073525', '.', '-', '.', 'gene_id \"chr9.g5464\"; transcript_id \"chr9.g5464.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8073156', '8073363', '.', '-', '.', 'gene_id \"chr9.g5464\"; transcript_id \"chr9.g5464.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8075649', '8075932', '.', '-', '.', 'gene_id \"chr9.g5468\"; transcript_id \"chr9.g5468.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8075942', '8076194', '.', '+', '.', 'gene_id \"chr9.g5469\"; transcript_id \"chr9.g5469.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8076672', '8077058', '.', '-', '.', 'gene_id \"chr9.g5471\"; transcript_id \"chr9.g5471.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8077860', '8078139', '.', '-', '.', 'gene_id \"chr9.g5476\"; transcript_id \"chr9.g5476.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8079915', '8080353', '.', '-', '.', 'gene_id \"chr9.g5477\"; transcript_id \"chr9.g5477.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8080555', '8081179', '.', '+', '.', 'gene_id \"chr9.g5479\"; transcript_id \"chr9.g5479.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8081176', '8081347', '.', '-', '.', 'gene_id \"chr9.g5480\"; transcript_id \"chr9.g5480.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8083064', '8083499', '.', '-', '.', 'gene_id \"chr9.g5482\"; transcript_id \"chr9.g5482.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8085899', '8086048', '.', '+', '.', 'gene_id \"chr9.g5488\"; transcript_id \"chr9.g5488.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8087879', '8088071', '.', '-', '.', 'gene_id \"chr9.g5493\"; transcript_id \"chr9.g5493.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8088990', '8089637', '.', '+', '.', 'gene_id \"chr9.g5495\"; transcript_id \"chr9.g5495.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8089791', '8090162', '.', '+', '.', 'gene_id \"chr9.g5496\"; transcript_id \"chr9.g5496.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8090182', '8090639', '.', '+', '.', 'gene_id \"chr9.g5497\"; transcript_id \"chr9.g5497.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8313976', '8314385', '.', '-', '.', 'gene_id \"chr9.g5503\"; transcript_id \"chr9.g5503.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8154574', '8154802', '.', '-', '.', 'gene_id \"chr9.g5515\"; transcript_id \"chr9.g5515.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8155134', '8155589', '.', '-', '.', 'gene_id \"chr9.g5515\"; transcript_id \"chr9.g5515.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8155576', '8156079', '.', '+', '.', 'gene_id \"chr9.g5516\"; transcript_id \"chr9.g5516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8155576', '8156078', '.', '+', '.', 'gene_id \"chr9.g5516\"; transcript_id \"chr9.g5516.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8155576', '8156089', '.', '+', '.', 'gene_id \"chr9.g5516\"; transcript_id \"chr9.g5516.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8159571', '8159772', '.', '-', '.', 'gene_id \"chr9.g5525\"; transcript_id \"chr9.g5525.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8166982', '8167519', '.', '+', '.', 'gene_id \"chr9.g5527\"; transcript_id \"chr9.g5527.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8167259', '8167394', '.', '+', '.', 'gene_id \"chr9.g5527\"; transcript_id \"chr9.g5527.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8166984', '8167266', '.', '-', '.', 'gene_id \"chr9.g5528\"; transcript_id \"chr9.g5528.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8177007', '8177455', '.', '-', '.', 'gene_id \"chr9.g5529\"; transcript_id \"chr9.g5529.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8181985', '8182376', '.', '-', '.', 'gene_id \"chr9.g5531\"; transcript_id \"chr9.g5531.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8184125', '8184278', '.', '+', '.', 'gene_id \"chr9.g5533\"; transcript_id \"chr9.g5533.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8186912', '8187303', '.', '-', '.', 'gene_id \"chr9.g5538\"; transcript_id \"chr9.g5538.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8187399', '8187624', '.', '+', '.', 'gene_id \"chr9.g5539\"; transcript_id \"chr9.g5539.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8190728', '8191160', '.', '+', '.', 'gene_id \"chr9.g5544\"; transcript_id \"chr9.g5544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8202651', '8202790', '.', '-', '.', 'gene_id \"chr9.g5547\"; transcript_id \"chr9.g5547.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8204908', '8205155', '.', '+', '.', 'gene_id \"chr9.g5552\"; transcript_id \"chr9.g5552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8207680', '8207855', '.', '+', '.', 'gene_id \"chr9.g5557\"; transcript_id \"chr9.g5557.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8217389', '8217661', '.', '+', '.', 'gene_id \"chr9.g5569\"; transcript_id \"chr9.g5569.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8225222', '8225421', '.', '+', '.', 'gene_id \"chr9.g5572\"; transcript_id \"chr9.g5572.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8228477', '8228707', '.', '+', '.', 'gene_id \"chr9.g5574\"; transcript_id \"chr9.g5574.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8232720', '8232904', '.', '+', '.', 'gene_id \"chr9.g5576\"; transcript_id \"chr9.g5576.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8235245', '8235925', '.', '+', '.', 'gene_id \"chr9.g5579\"; transcript_id \"chr9.g5579.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8235098', '8235249', '.', '+', '.', 'gene_id \"chr9.g5581\"; transcript_id \"chr9.g5581.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8235932', '8236233', '.', '+', '.', 'gene_id \"chr9.g5583\"; transcript_id \"chr9.g5583.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8263443', '8263661', '.', '+', '.', 'gene_id \"chr9.g5586\"; transcript_id \"chr9.g5586.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8272660', '8273295', '.', '-', '.', 'gene_id \"chr9.g5589\"; transcript_id \"chr9.g5589.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8273280', '8273463', '.', '+', '.', 'gene_id \"chr9.g5590\"; transcript_id \"chr9.g5590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8285008', '8285244', '.', '+', '.', 'gene_id \"chr9.g5594\"; transcript_id \"chr9.g5594.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8285885', '8286199', '.', '+', '.', 'gene_id \"chr9.g5595\"; transcript_id \"chr9.g5595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8305303', '8305581', '.', '+', '.', 'gene_id \"chr9.g5613\"; transcript_id \"chr9.g5613.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8308072', '8308250', '.', '+', '.', 'gene_id \"chr9.g5618\"; transcript_id \"chr9.g5618.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8307835', '8308081', '.', '+', '.', 'gene_id \"chr9.g5620\"; transcript_id \"chr9.g5620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8308629', '8308847', '.', '-', '.', 'gene_id \"chr9.g5621\"; transcript_id \"chr9.g5621.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8314416', '8314584', '.', '+', '.', 'gene_id \"chr9.g5623\"; transcript_id \"chr9.g5623.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8318286', '8318474', '.', '+', '.', 'gene_id \"chr9.g5626\"; transcript_id \"chr9.g5626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8318571', '8318828', '.', '-', '.', 'gene_id \"chr9.g5627\"; transcript_id \"chr9.g5627.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8318982', '8319449', '.', '-', '.', 'gene_id \"chr9.g5628\"; transcript_id \"chr9.g5628.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8321491', '8321715', '.', '-', '.', 'gene_id \"chr9.g5631\"; transcript_id \"chr9.g5631.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8321864', '8322531', '.', '+', '.', 'gene_id \"chr9.g5632\"; transcript_id \"chr9.g5632.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8322524', '8322765', '.', '-', '.', 'gene_id \"chr9.g5633\"; transcript_id \"chr9.g5633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8357227', '8357268', '.', '+', '.', 'gene_id \"chr9.g5634\"; transcript_id \"chr9.g5634.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8355766', '8357243', '.', '-', '.', 'gene_id \"chr9.g5635\"; transcript_id \"chr9.g5635.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8344970', '8345146', '.', '-', '.', 'gene_id \"chr9.g5637\"; transcript_id \"chr9.g5637.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8345275', '8345554', '.', '+', '.', 'gene_id \"chr9.g5638\"; transcript_id \"chr9.g5638.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8346415', '8346575', '.', '-', '.', 'gene_id \"chr9.g5639\"; transcript_id \"chr9.g5639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8411977', '8412165', '.', '+', '.', 'gene_id \"chr9.g5644\"; transcript_id \"chr9.g5644.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '9240494', '9240696', '.', '-', '.', 'gene_id \"chr17.g2782\"; transcript_id \"chr17.g2782.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8427767', '8427981', '.', '+', '.', 'gene_id \"chr9.g5655\"; transcript_id \"chr9.g5655.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8436998', '8437184', '.', '+', '.', 'gene_id \"chr9.g5658\"; transcript_id \"chr9.g5658.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8429583', '8429879', '.', '-', '.', 'gene_id \"chr9.g5660\"; transcript_id \"chr9.g5660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8438604', '8438835', '.', '-', '.', 'gene_id \"chr9.g5663\"; transcript_id \"chr9.g5663.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8451564', '8451721', '.', '+', '.', 'gene_id \"chr9.g5665\"; transcript_id \"chr9.g5665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8464934', '8464964', '.', '-', '.', 'gene_id \"chr9.g5666\"; transcript_id \"chr9.g5666.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8481188', '8481398', '.', '-', '.', 'gene_id \"chr9.g5667\"; transcript_id \"chr9.g5667.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8522240', '8522817', '.', '-', '.', 'gene_id \"chr9.g5668\"; transcript_id \"chr9.g5668.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8534096', '8534302', '.', '-', '.', 'gene_id \"chr9.g5671\"; transcript_id \"chr9.g5671.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8538238', '8538607', '.', '+', '.', 'gene_id \"chr9.g5673\"; transcript_id \"chr9.g5673.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8538623', '8538786', '.', '+', '.', 'gene_id \"chr9.g5674\"; transcript_id \"chr9.g5674.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8544910', '8545688', '.', '-', '.', 'gene_id \"chr9.g5676\"; transcript_id \"chr9.g5676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8546111', '8546393', '.', '-', '.', 'gene_id \"chr9.g5678\"; transcript_id \"chr9.g5678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8552555', '8552709', '.', '-', '.', 'gene_id \"chr9.g5685\"; transcript_id \"chr9.g5685.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8552705', '8552922', '.', '-', '.', 'gene_id \"chr9.g5686\"; transcript_id \"chr9.g5686.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8556732', '8556928', '.', '+', '.', 'gene_id \"chr9.g5687\"; transcript_id \"chr9.g5687.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8575569', '8575734', '.', '+', '.', 'gene_id \"chr9.g5691\"; transcript_id \"chr9.g5691.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8576800', '8577026', '.', '-', '.', 'gene_id \"chr9.g5694\"; transcript_id \"chr9.g5694.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8580957', '8581242', '.', '-', '.', 'gene_id \"chr9.g5696\"; transcript_id \"chr9.g5696.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8580736', '8580948', '.', '-', '.', 'gene_id \"chr9.g5697\"; transcript_id \"chr9.g5697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8585209', '8585251', '.', '+', '.', 'gene_id \"chr9.g5701\"; transcript_id \"chr9.g5701.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8604116', '8604239', '.', '+', '.', 'gene_id \"chr9.g5702\"; transcript_id \"chr9.g5702.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8604278', '8604720', '.', '-', '.', 'gene_id \"chr9.g5703\"; transcript_id \"chr9.g5703.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8604278', '8604763', '.', '-', '.', 'gene_id \"chr9.g5703\"; transcript_id \"chr9.g5703.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8604822', '8605205', '.', '-', '.', 'gene_id \"chr9.g5705\"; transcript_id \"chr9.g5705.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8605206', '8605363', '.', '-', '.', 'gene_id \"chr9.g5706\"; transcript_id \"chr9.g5706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8605357', '8605548', '.', '+', '.', 'gene_id \"chr9.g5707\"; transcript_id \"chr9.g5707.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8605800', '8606335', '.', '+', '.', 'gene_id \"chr9.g5708\"; transcript_id \"chr9.g5708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8883212', '8883574', '.', '+', '.', 'gene_id \"chr9.g5733\"; transcript_id \"chr9.g5733.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8884648', '8884789', '.', '+', '.', 'gene_id \"chr9.g5735\"; transcript_id \"chr9.g5735.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '9887731', '9887771', '.', '-', '.', 'gene_id \"chr17.g2863\"; transcript_id \"chr17.g2863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9126346', '9126646', '.', '+', '.', 'gene_id \"chr9.g5754\"; transcript_id \"chr9.g5754.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9607218', '9607416', '.', '-', '.', 'gene_id \"chr9.g5784\"; transcript_id \"chr9.g5784.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '9646517', '9646802', '.', '-', '.', 'gene_id \"chr9.g5791\"; transcript_id \"chr9.g5791.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9681237', '9681273', '.', '+', '.', 'gene_id \"chr9.g5799\"; transcript_id \"chr9.g5799.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9708601', '9708647', '.', '-', '.', 'gene_id \"chr9.g5807\"; transcript_id \"chr9.g5807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9784105', '9784390', '.', '+', '.', 'gene_id \"chr9.g5823\"; transcript_id \"chr9.g5823.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9808968', '9809156', '.', '+', '.', 'gene_id \"chr9.g5825\"; transcript_id \"chr9.g5825.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9783953', '9784108', '.', '-', '.', 'gene_id \"chr9.g5826\"; transcript_id \"chr9.g5826.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9789422', '9789691', '.', '+', '.', 'gene_id \"chr9.g5827\"; transcript_id \"chr9.g5827.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9787497', '9787649', '.', '-', '.', 'gene_id \"chr9.g5828\"; transcript_id \"chr9.g5828.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9788303', '9788633', '.', '-', '.', 'gene_id \"chr9.g5831\"; transcript_id \"chr9.g5831.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9788636', '9788807', '.', '-', '.', 'gene_id \"chr9.g5832\"; transcript_id \"chr9.g5832.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9790775', '9790972', '.', '+', '.', 'gene_id \"chr9.g5833\"; transcript_id \"chr9.g5833.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9795197', '9795400', '.', '-', '.', 'gene_id \"chr9.g5837\"; transcript_id \"chr9.g5837.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9796195', '9796357', '.', '+', '.', 'gene_id \"chr9.g5839\"; transcript_id \"chr9.g5839.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9796360', '9796519', '.', '-', '.', 'gene_id \"chr9.g5840\"; transcript_id \"chr9.g5840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9799583', '9800018', '.', '-', '.', 'gene_id \"chr9.g5843\"; transcript_id \"chr9.g5843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9799996', '9800187', '.', '+', '.', 'gene_id \"chr9.g5844\"; transcript_id \"chr9.g5844.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9800727', '9801071', '.', '+', '.', 'gene_id \"chr9.g5846\"; transcript_id \"chr9.g5846.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9809446', '9809740', '.', '-', '.', 'gene_id \"chr9.g5848\"; transcript_id \"chr9.g5848.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9858102', '9858458', '.', '-', '.', 'gene_id \"chr9.g5851\"; transcript_id \"chr9.g5851.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9948544', '9948905', '.', '-', '.', 'gene_id \"chr9.g5855\"; transcript_id \"chr9.g5855.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9945888', '9946817', '.', '-', '.', 'gene_id \"chr9.g5857\"; transcript_id \"chr9.g5857.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9886275', '9886465', '.', '+', '.', 'gene_id \"chr9.g5866\"; transcript_id \"chr9.g5866.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9896987', '9897162', '.', '-', '.', 'gene_id \"chr9.g5870\"; transcript_id \"chr9.g5870.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9903446', '9903630', '.', '+', '.', 'gene_id \"chr9.g5875\"; transcript_id \"chr9.g5875.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9905237', '9905411', '.', '-', '.', 'gene_id \"chr9.g5876\"; transcript_id \"chr9.g5876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9907375', '9907731', '.', '+', '.', 'gene_id \"chr9.g5879\"; transcript_id \"chr9.g5879.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9908633', '9908811', '.', '+', '.', 'gene_id \"chr9.g5881\"; transcript_id \"chr9.g5881.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9910329', '9910597', '.', '+', '.', 'gene_id \"chr9.g5882\"; transcript_id \"chr9.g5882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9911873', '9912074', '.', '+', '.', 'gene_id \"chr9.g5884\"; transcript_id \"chr9.g5884.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9915892', '9916220', '.', '-', '.', 'gene_id \"chr9.g5887\"; transcript_id \"chr9.g5887.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9916197', '9916241', '.', '-', '.', 'gene_id \"chr9.g5887\"; transcript_id \"chr9.g5887.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9916218', '9916677', '.', '+', '.', 'gene_id \"chr9.g5889\"; transcript_id \"chr9.g5889.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9922586', '9923067', '.', '+', '.', 'gene_id \"chr9.g5895\"; transcript_id \"chr9.g5895.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9927053', '9927206', '.', '-', '.', 'gene_id \"chr9.g5897\"; transcript_id \"chr9.g5897.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9932182', '9932620', '.', '+', '.', 'gene_id \"chr9.g5902\"; transcript_id \"chr9.g5902.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9931969', '9932182', '.', '-', '.', 'gene_id \"chr9.g5903\"; transcript_id \"chr9.g5903.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9933226', '9933592', '.', '+', '.', 'gene_id \"chr9.g5905\"; transcript_id \"chr9.g5905.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9934730', '9934966', '.', '+', '.', 'gene_id \"chr9.g5906\"; transcript_id \"chr9.g5906.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9943560', '9943766', '.', '+', '.', 'gene_id \"chr9.g5909\"; transcript_id \"chr9.g5909.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9943767', '9944056', '.', '+', '.', 'gene_id \"chr9.g5910\"; transcript_id \"chr9.g5910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10014582', '10015163', '.', '+', '.', 'gene_id \"chr9.g5912\"; transcript_id \"chr9.g5912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10155808', '10156342', '.', '+', '.', 'gene_id \"chr9.g5913\"; transcript_id \"chr9.g5913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10161369', '10161567', '.', '+', '.', 'gene_id \"chr9.g5918\"; transcript_id \"chr9.g5918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10162864', '10164454', '.', '-', '.', 'gene_id \"chr9.g5919\"; transcript_id \"chr9.g5919.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9987802', '9988084', '.', '+', '.', 'gene_id \"chr9.g5921\"; transcript_id \"chr9.g5921.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9994273', '9994497', '.', '+', '.', 'gene_id \"chr9.g5928\"; transcript_id \"chr9.g5928.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10000885', '10001259', '.', '+', '.', 'gene_id \"chr9.g5936\"; transcript_id \"chr9.g5936.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10004307', '10004487', '.', '-', '.', 'gene_id \"chr9.g5940\"; transcript_id \"chr9.g5940.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10004695', '10004917', '.', '-', '.', 'gene_id \"chr9.g5941\"; transcript_id \"chr9.g5941.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10004943', '10005232', '.', '-', '.', 'gene_id \"chr9.g5942\"; transcript_id \"chr9.g5942.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10007226', '10007401', '.', '-', '.', 'gene_id \"chr9.g5946\"; transcript_id \"chr9.g5946.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10008676', '10008968', '.', '-', '.', 'gene_id \"chr9.g5948\"; transcript_id \"chr9.g5948.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10009685', '10009995', '.', '-', '.', 'gene_id \"chr9.g5950\"; transcript_id \"chr9.g5950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10011183', '10011385', '.', '+', '.', 'gene_id \"chr9.g5951\"; transcript_id \"chr9.g5951.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10012236', '10012557', '.', '+', '.', 'gene_id \"chr9.g5954\"; transcript_id \"chr9.g5954.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10012804', '10013066', '.', '-', '.', 'gene_id \"chr9.g5955\"; transcript_id \"chr9.g5955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10014210', '10014358', '.', '+', '.', 'gene_id \"chr9.g5958\"; transcript_id \"chr9.g5958.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10015400', '10015669', '.', '-', '.', 'gene_id \"chr9.g5959\"; transcript_id \"chr9.g5959.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10026376', '10027046', '.', '+', '.', 'gene_id \"chr9.g5965\"; transcript_id \"chr9.g5965.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10026717', '10027046', '.', '+', '.', 'gene_id \"chr9.g5965\"; transcript_id \"chr9.g5965.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10027636', '10027815', '.', '+', '.', 'gene_id \"chr9.g5968\"; transcript_id \"chr9.g5968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10027809', '10028059', '.', '-', '.', 'gene_id \"chr9.g5969\"; transcript_id \"chr9.g5969.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10034677', '10034937', '.', '-', '.', 'gene_id \"chr9.g5974\"; transcript_id \"chr9.g5974.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10034933', '10035243', '.', '+', '.', 'gene_id \"chr9.g5975\"; transcript_id \"chr9.g5975.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10035686', '10035927', '.', '+', '.', 'gene_id \"chr9.g5977\"; transcript_id \"chr9.g5977.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10036585', '10036970', '.', '+', '.', 'gene_id \"chr9.g5979\"; transcript_id \"chr9.g5979.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10037976', '10038217', '.', '+', '.', 'gene_id \"chr9.g5982\"; transcript_id \"chr9.g5982.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10038527', '10038807', '.', '+', '.', 'gene_id \"chr9.g5983\"; transcript_id \"chr9.g5983.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10039039', '10039415', '.', '-', '.', 'gene_id \"chr9.g5985\"; transcript_id \"chr9.g5985.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10040204', '10040683', '.', '+', '.', 'gene_id \"chr9.g5990\"; transcript_id \"chr9.g5990.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10042251', '10042536', '.', '+', '.', 'gene_id \"chr9.g5993\"; transcript_id \"chr9.g5993.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10042969', '10043368', '.', '-', '.', 'gene_id \"chr9.g5994\"; transcript_id \"chr9.g5994.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10044992', '10045220', '.', '-', '.', 'gene_id \"chr9.g5998\"; transcript_id \"chr9.g5998.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10046567', '10046768', '.', '+', '.', 'gene_id \"chr9.g6001\"; transcript_id \"chr9.g6001.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10058056', '10058635', '.', '+', '.', 'gene_id \"chr9.g6006\"; transcript_id \"chr9.g6006.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10058621', '10058651', '.', '-', '.', 'gene_id \"chr9.g6007\"; transcript_id \"chr9.g6007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10063135', '10063449', '.', '+', '.', 'gene_id \"chr9.g6012\"; transcript_id \"chr9.g6012.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10063445', '10063629', '.', '-', '.', 'gene_id \"chr9.g6013\"; transcript_id \"chr9.g6013.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10069661', '10069987', '.', '+', '.', 'gene_id \"chr9.g6015\"; transcript_id \"chr9.g6015.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '11585375', '11585614', '.', '+', '.', 'gene_id \"chr17.g3137\"; transcript_id \"chr17.g3137.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10081932', '10082199', '.', '+', '.', 'gene_id \"chr9.g6027\"; transcript_id \"chr9.g6027.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10086562', '10086829', '.', '+', '.', 'gene_id \"chr9.g6029\"; transcript_id \"chr9.g6029.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10089864', '10090084', '.', '-', '.', 'gene_id \"chr9.g6032\"; transcript_id \"chr9.g6032.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10090239', '10090452', '.', '+', '.', 'gene_id \"chr9.g6033\"; transcript_id \"chr9.g6033.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10095940', '10096172', '.', '-', '.', 'gene_id \"chr9.g6035\"; transcript_id \"chr9.g6035.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10096610', '10096805', '.', '+', '.', 'gene_id \"chr9.g6036\"; transcript_id \"chr9.g6036.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10096810', '10097032', '.', '-', '.', 'gene_id \"chr9.g6038\"; transcript_id \"chr9.g6038.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10097316', '10098219', '.', '+', '.', 'gene_id \"chr9.g6039\"; transcript_id \"chr9.g6039.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10098737', '10098904', '.', '+', '.', 'gene_id \"chr9.g6041\"; transcript_id \"chr9.g6041.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10099398', '10099629', '.', '+', '.', 'gene_id \"chr9.g6042\"; transcript_id \"chr9.g6042.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10101811', '10102293', '.', '+', '.', 'gene_id \"chr9.g6045\"; transcript_id \"chr9.g6045.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10127298', '10127563', '.', '+', '.', 'gene_id \"chr9.g6060\"; transcript_id \"chr9.g6060.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10128914', '10129088', '.', '+', '.', 'gene_id \"chr9.g6061\"; transcript_id \"chr9.g6061.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10129066', '10129355', '.', '+', '.', 'gene_id \"chr9.g6061\"; transcript_id \"chr9.g6061.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10129348', '10129618', '.', '+', '.', 'gene_id \"chr9.g6062\"; transcript_id \"chr9.g6062.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10130357', '10130534', '.', '-', '.', 'gene_id \"chr9.g6064\"; transcript_id \"chr9.g6064.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10131813', '10132039', '.', '+', '.', 'gene_id \"chr9.g6066\"; transcript_id \"chr9.g6066.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10132935', '10133092', '.', '-', '.', 'gene_id \"chr9.g6067\"; transcript_id \"chr9.g6067.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10133096', '10133428', '.', '-', '.', 'gene_id \"chr9.g6068\"; transcript_id \"chr9.g6068.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10135115', '10135490', '.', '-', '.', 'gene_id \"chr9.g6069\"; transcript_id \"chr9.g6069.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10135677', '10135916', '.', '+', '.', 'gene_id \"chr9.g6070\"; transcript_id \"chr9.g6070.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10147185', '10147434', '.', '-', '.', 'gene_id \"chr9.g6078\"; transcript_id \"chr9.g6078.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10176931', '10177046', '.', '-', '.', 'gene_id \"chr9.g6087\"; transcript_id \"chr9.g6087.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10177105', '10179176', '.', '+', '.', 'gene_id \"chr9.g6088\"; transcript_id \"chr9.g6088.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10179020', '10179176', '.', '+', '.', 'gene_id \"chr9.g6088\"; transcript_id \"chr9.g6088.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10191146', '10192913', '.', '+', '.', 'gene_id \"chr9.g6089\"; transcript_id \"chr9.g6089.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10191927', '10192913', '.', '+', '.', 'gene_id \"chr9.g6089\"; transcript_id \"chr9.g6089.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10276658', '10276878', '.', '+', '.', 'gene_id \"chr9.g6090\"; transcript_id \"chr9.g6090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10198222', '10199120', '.', '+', '.', 'gene_id \"chr9.g6091\"; transcript_id \"chr9.g6091.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10175606', '10176960', '.', '+', '.', 'gene_id \"chr9.g6093\"; transcript_id \"chr9.g6093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10210865', '10211240', '.', '+', '.', 'gene_id \"chr9.g6096\"; transcript_id \"chr9.g6096.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10192944', '10195371', '.', '+', '.', 'gene_id \"chr9.g6097\"; transcript_id \"chr9.g6097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10210627', '10210842', '.', '+', '.', 'gene_id \"chr9.g6098\"; transcript_id \"chr9.g6098.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10197842', '10198226', '.', '+', '.', 'gene_id \"chr9.g6100\"; transcript_id \"chr9.g6100.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10180350', '10180533', '.', '-', '.', 'gene_id \"chr9.g6101\"; transcript_id \"chr9.g6101.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10186159', '10187097', '.', '-', '.', 'gene_id \"chr9.g6103\"; transcript_id \"chr9.g6103.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10189707', '10190409', '.', '+', '.', 'gene_id \"chr9.g6105\"; transcript_id \"chr9.g6105.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10189707', '10190409', '.', '+', '.', 'gene_id \"chr9.g6105\"; transcript_id \"chr9.g6105.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10206968', '10207477', '.', '+', '.', 'gene_id \"chr9.g6112\"; transcript_id \"chr9.g6112.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10207501', '10207773', '.', '+', '.', 'gene_id \"chr9.g6114\"; transcript_id \"chr9.g6114.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '12035152', '12035228', '.', '+', '.', 'gene_id \"chr17.g3221\"; transcript_id \"chr17.g3221.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10208739', '10208971', '.', '-', '.', 'gene_id \"chr9.g6116\"; transcript_id \"chr9.g6116.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10211396', '10211678', '.', '-', '.', 'gene_id \"chr9.g6117\"; transcript_id \"chr9.g6117.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10211686', '10212088', '.', '-', '.', 'gene_id \"chr9.g6118\"; transcript_id \"chr9.g6118.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10212759', '10213161', '.', '+', '.', 'gene_id \"chr9.g6119\"; transcript_id \"chr9.g6119.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10212458', '10212772', '.', '+', '.', 'gene_id \"chr9.g6121\"; transcript_id \"chr9.g6121.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10213310', '10213462', '.', '+', '.', 'gene_id \"chr9.g6122\"; transcript_id \"chr9.g6122.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10214407', '10214591', '.', '+', '.', 'gene_id \"chr9.g6124\"; transcript_id \"chr9.g6124.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10216913', '10217208', '.', '+', '.', 'gene_id \"chr9.g6127\"; transcript_id \"chr9.g6127.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10220530', '10220755', '.', '-', '.', 'gene_id \"chr9.g6131\"; transcript_id \"chr9.g6131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10220863', '10221270', '.', '+', '.', 'gene_id \"chr9.g6132\"; transcript_id \"chr9.g6132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10238341', '10238530', '.', '+', '.', 'gene_id \"chr9.g6137\"; transcript_id \"chr9.g6137.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10248700', '10248865', '.', '+', '.', 'gene_id \"chr9.g6145\"; transcript_id \"chr9.g6145.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10248909', '10249286', '.', '-', '.', 'gene_id \"chr9.g6146\"; transcript_id \"chr9.g6146.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10310212', '10310415', '.', '+', '.', 'gene_id \"chr9.g6164\"; transcript_id \"chr9.g6164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10319277', '10319459', '.', '-', '.', 'gene_id \"chr9.g6168\"; transcript_id \"chr9.g6168.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10323851', '10324186', '.', '-', '.', 'gene_id \"chr9.g6171\"; transcript_id \"chr9.g6171.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10329039', '10329225', '.', '+', '.', 'gene_id \"chr9.g6177\"; transcript_id \"chr9.g6177.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10442951', '10443290', '.', '+', '.', 'gene_id \"chr9.g6192\"; transcript_id \"chr9.g6192.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10444232', '10444546', '.', '-', '.', 'gene_id \"chr9.g6193\"; transcript_id \"chr9.g6193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10482133', '10482315', '.', '-', '.', 'gene_id \"chr9.g6197\"; transcript_id \"chr9.g6197.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '12282268', '12282744', '.', '+', '.', 'gene_id \"chr17.g3312\"; transcript_id \"chr17.g3312.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10518180', '10518310', '.', '-', '.', 'gene_id \"chr9.g6215\"; transcript_id \"chr9.g6215.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10500250', '10500540', '.', '-', '.', 'gene_id \"chr9.g6216\"; transcript_id \"chr9.g6216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10500004', '10500268', '.', '+', '.', 'gene_id \"chr9.g6217\"; transcript_id \"chr9.g6217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10502112', '10502415', '.', '+', '.', 'gene_id \"chr9.g6218\"; transcript_id \"chr9.g6218.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10528948', '10529063', '.', '+', '.', 'gene_id \"chr9.g6228\"; transcript_id \"chr9.g6228.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10599963', '10600162', '.', '+', '.', 'gene_id \"chr9.g6228\"; transcript_id \"chr9.g6228.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10671376', '10672154', '.', '-', '.', 'gene_id \"chr9.g6232\"; transcript_id \"chr9.g6232.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10671044', '10671389', '.', '-', '.', 'gene_id \"chr9.g6234\"; transcript_id \"chr9.g6234.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10570638', '10570794', '.', '-', '.', 'gene_id \"chr9.g6241\"; transcript_id \"chr9.g6241.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10588594', '10588768', '.', '+', '.', 'gene_id \"chr9.g6254\"; transcript_id \"chr9.g6254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10591841', '10592159', '.', '+', '.', 'gene_id \"chr9.g6256\"; transcript_id \"chr9.g6256.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10594909', '10595062', '.', '+', '.', 'gene_id \"chr9.g6259\"; transcript_id \"chr9.g6259.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10595890', '10596096', '.', '+', '.', 'gene_id \"chr9.g6260\"; transcript_id \"chr9.g6260.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10596501', '10596828', '.', '-', '.', 'gene_id \"chr9.g6262\"; transcript_id \"chr9.g6262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10600178', '10600544', '.', '-', '.', 'gene_id \"chr9.g6264\"; transcript_id \"chr9.g6264.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10601431', '10601728', '.', '+', '.', 'gene_id \"chr9.g6265\"; transcript_id \"chr9.g6265.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10603918', '10604195', '.', '-', '.', 'gene_id \"chr9.g6268\"; transcript_id \"chr9.g6268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10605340', '10605710', '.', '+', '.', 'gene_id \"chr9.g6269\"; transcript_id \"chr9.g6269.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10605758', '10605920', '.', '-', '.', 'gene_id \"chr9.g6270\"; transcript_id \"chr9.g6270.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10606184', '10606349', '.', '+', '.', 'gene_id \"chr9.g6271\"; transcript_id \"chr9.g6271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10616990', '10617247', '.', '-', '.', 'gene_id \"chr9.g6278\"; transcript_id \"chr9.g6278.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10648851', '10649090', '.', '+', '.', 'gene_id \"chr9.g6291\"; transcript_id \"chr9.g6291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10652786', '10653054', '.', '-', '.', 'gene_id \"chr9.g6296\"; transcript_id \"chr9.g6296.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10653830', '10653980', '.', '+', '.', 'gene_id \"chr9.g6297\"; transcript_id \"chr9.g6297.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10654046', '10654223', '.', '-', '.', 'gene_id \"chr9.g6298\"; transcript_id \"chr9.g6298.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10654442', '10654629', '.', '+', '.', 'gene_id \"chr9.g6299\"; transcript_id \"chr9.g6299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10654725', '10654998', '.', '+', '.', 'gene_id \"chr9.g6300\"; transcript_id \"chr9.g6300.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10654987', '10655260', '.', '-', '.', 'gene_id \"chr9.g6301\"; transcript_id \"chr9.g6301.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10656563', '10656806', '.', '-', '.', 'gene_id \"chr9.g6305\"; transcript_id \"chr9.g6305.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10656600', '10656806', '.', '-', '.', 'gene_id \"chr9.g6305\"; transcript_id \"chr9.g6305.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10657951', '10658310', '.', '-', '.', 'gene_id \"chr9.g6306\"; transcript_id \"chr9.g6306.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10659191', '10659400', '.', '-', '.', 'gene_id \"chr9.g6308\"; transcript_id \"chr9.g6308.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10659569', '10659775', '.', '-', '.', 'gene_id \"chr9.g6311\"; transcript_id \"chr9.g6311.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10660181', '10660696', '.', '-', '.', 'gene_id \"chr9.g6313\"; transcript_id \"chr9.g6313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10660705', '10660960', '.', '-', '.', 'gene_id \"chr9.g6314\"; transcript_id \"chr9.g6314.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10661397', '10661690', '.', '+', '.', 'gene_id \"chr9.g6316\"; transcript_id \"chr9.g6316.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10661817', '10662834', '.', '-', '.', 'gene_id \"chr9.g6318\"; transcript_id \"chr9.g6318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10662914', '10663112', '.', '-', '.', 'gene_id \"chr9.g6319\"; transcript_id \"chr9.g6319.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10663223', '10663417', '.', '-', '.', 'gene_id \"chr9.g6320\"; transcript_id \"chr9.g6320.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10663427', '10663606', '.', '+', '.', 'gene_id \"chr9.g6321\"; transcript_id \"chr9.g6321.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10664099', '10664730', '.', '-', '.', 'gene_id \"chr9.g6322\"; transcript_id \"chr9.g6322.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10666026', '10666384', '.', '+', '.', 'gene_id \"chr9.g6328\"; transcript_id \"chr9.g6328.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10668847', '10669548', '.', '+', '.', 'gene_id \"chr9.g6332\"; transcript_id \"chr9.g6332.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10669825', '10670110', '.', '+', '.', 'gene_id \"chr9.g6333\"; transcript_id \"chr9.g6333.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10670152', '10670342', '.', '+', '.', 'gene_id \"chr9.g6334\"; transcript_id \"chr9.g6334.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10675586', '10675816', '.', '-', '.', 'gene_id \"chr9.g6335\"; transcript_id \"chr9.g6335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10675895', '10676334', '.', '+', '.', 'gene_id \"chr9.g6336\"; transcript_id \"chr9.g6336.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10676365', '10676686', '.', '+', '.', 'gene_id \"chr9.g6337\"; transcript_id \"chr9.g6337.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10682225', '10682439', '.', '-', '.', 'gene_id \"chr9.g6339\"; transcript_id \"chr9.g6339.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10682225', '10682439', '.', '-', '.', 'gene_id \"chr9.g6339\"; transcript_id \"chr9.g6339.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10679317', '10679562', '.', '-', '.', 'gene_id \"chr9.g6339\"; transcript_id \"chr9.g6339.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10689093', '10689247', '.', '+', '.', 'gene_id \"chr9.g6340\"; transcript_id \"chr9.g6340.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10681894', '10682231', '.', '+', '.', 'gene_id \"chr9.g6341\"; transcript_id \"chr9.g6341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10679571', '10679833', '.', '+', '.', 'gene_id \"chr9.g6345\"; transcript_id \"chr9.g6345.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10680187', '10680831', '.', '-', '.', 'gene_id \"chr9.g6348\"; transcript_id \"chr9.g6348.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10684925', '10685208', '.', '-', '.', 'gene_id \"chr9.g6349\"; transcript_id \"chr9.g6349.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10686155', '10686741', '.', '-', '.', 'gene_id \"chr9.g6350\"; transcript_id \"chr9.g6350.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10691329', '10691478', '.', '+', '.', 'gene_id \"chr9.g6351\"; transcript_id \"chr9.g6351.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10703759', '10703907', '.', '-', '.', 'gene_id \"chr9.g6354\"; transcript_id \"chr9.g6354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10725456', '10725736', '.', '-', '.', 'gene_id \"chr9.g6358\"; transcript_id \"chr9.g6358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10771163', '10771387', '.', '-', '.', 'gene_id \"chr9.g6367\"; transcript_id \"chr9.g6367.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '13108069', '13108372', '.', '+', '.', 'gene_id \"chr17.g3463\"; transcript_id \"chr17.g3463.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10773004', '10773873', '.', '+', '.', 'gene_id \"chr9.g6370\"; transcript_id \"chr9.g6370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10817816', '10818227', '.', '-', '.', 'gene_id \"chr9.g6374\"; transcript_id \"chr9.g6374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10823781', '10824142', '.', '-', '.', 'gene_id \"chr9.g6375\"; transcript_id \"chr9.g6375.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10817524', '10817826', '.', '+', '.', 'gene_id \"chr9.g6376\"; transcript_id \"chr9.g6376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10817263', '10817546', '.', '-', '.', 'gene_id \"chr9.g6377\"; transcript_id \"chr9.g6377.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10810066', '10810261', '.', '-', '.', 'gene_id \"chr9.g6379\"; transcript_id \"chr9.g6379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10816432', '10816738', '.', '+', '.', 'gene_id \"chr9.g6381\"; transcript_id \"chr9.g6381.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10813999', '10814694', '.', '-', '.', 'gene_id \"chr9.g6383\"; transcript_id \"chr9.g6383.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10814952', '10815238', '.', '-', '.', 'gene_id \"chr9.g6385\"; transcript_id \"chr9.g6385.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10824144', '10824384', '.', '+', '.', 'gene_id \"chr9.g6386\"; transcript_id \"chr9.g6386.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10849979', '10850196', '.', '+', '.', 'gene_id \"chr9.g6392\"; transcript_id \"chr9.g6392.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10850999', '10851054', '.', '+', '.', 'gene_id \"chr9.g6392\"; transcript_id \"chr9.g6392.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10857504', '10857787', '.', '-', '.', 'gene_id \"chr9.g6396\"; transcript_id \"chr9.g6396.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10857817', '10858051', '.', '+', '.', 'gene_id \"chr9.g6397\"; transcript_id \"chr9.g6397.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10873797', '10873856', '.', '+', '.', 'gene_id \"chr9.g6400\"; transcript_id \"chr9.g6400.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10875653', '10877081', '.', '-', '.', 'gene_id \"chr9.g6401\"; transcript_id \"chr9.g6401.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10874872', '10875185', '.', '-', '.', 'gene_id \"chr9.g6402\"; transcript_id \"chr9.g6402.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10875438', '10875608', '.', '-', '.', 'gene_id \"chr9.g6403\"; transcript_id \"chr9.g6403.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10881902', '10882061', '.', '+', '.', 'gene_id \"chr9.g6413\"; transcript_id \"chr9.g6413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10885426', '10885712', '.', '+', '.', 'gene_id \"chr9.g6414\"; transcript_id \"chr9.g6414.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10884052', '10884222', '.', '+', '.', 'gene_id \"chr9.g6414\"; transcript_id \"chr9.g6414.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10883044', '10883400', '.', '-', '.', 'gene_id \"chr9.g6416\"; transcript_id \"chr9.g6416.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10890806', '10890841', '.', '+', '.', 'gene_id \"chr9.g6418\"; transcript_id \"chr9.g6418.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10886448', '10886617', '.', '-', '.', 'gene_id \"chr9.g6419\"; transcript_id \"chr9.g6419.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10888609', '10889951', '.', '-', '.', 'gene_id \"chr9.g6420\"; transcript_id \"chr9.g6420.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10896626', '10896895', '.', '-', '.', 'gene_id \"chr9.g6424\"; transcript_id \"chr9.g6424.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10897886', '10897949', '.', '-', '.', 'gene_id \"chr9.g6428\"; transcript_id \"chr9.g6428.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10897748', '10897909', '.', '-', '.', 'gene_id \"chr9.g6428\"; transcript_id \"chr9.g6428.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10897677', '10897720', '.', '-', '.', 'gene_id \"chr9.g6431\"; transcript_id \"chr9.g6431.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10902227', '10902467', '.', '-', '.', 'gene_id \"chr9.g6434\"; transcript_id \"chr9.g6434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913582', '10913633', '.', '+', '.', 'gene_id \"chr9.g6435\"; transcript_id \"chr9.g6435.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10914175', '10914216', '.', '+', '.', 'gene_id \"chr9.g6435\"; transcript_id \"chr9.g6435.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10914216', '.', '+', '.', 'gene_id \"chr9.g6435\"; transcript_id \"chr9.g6435.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10914573', '.', '+', '.', 'gene_id \"chr9.g6435\"; transcript_id \"chr9.g6435.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10915353', '.', '+', '.', 'gene_id \"chr9.g6435\"; transcript_id \"chr9.g6435.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10915353', '.', '-', '.', 'gene_id \"chr9.g6437\"; transcript_id \"chr9.g6437.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10914573', '.', '-', '.', 'gene_id \"chr9.g6437\"; transcript_id \"chr9.g6437.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10913767', '.', '-', '.', 'gene_id \"chr9.g6437\"; transcript_id \"chr9.g6437.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913653', '10914216', '.', '-', '.', 'gene_id \"chr9.g6437\"; transcript_id \"chr9.g6437.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10953827', '10954066', '.', '-', '.', 'gene_id \"chr9.g6438\"; transcript_id \"chr9.g6438.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10913427', '10913633', '.', '-', '.', 'gene_id \"chr9.g6440\"; transcript_id \"chr9.g6440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10948753', '10948992', '.', '-', '.', 'gene_id \"chr9.g6441\"; transcript_id \"chr9.g6441.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10919534', '10919777', '.', '-', '.', 'gene_id \"chr9.g6447\"; transcript_id \"chr9.g6447.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10958038', '10958263', '.', '+', '.', 'gene_id \"chr9.g6448\"; transcript_id \"chr9.g6448.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10915490', '10915710', '.', '-', '.', 'gene_id \"chr9.g6450\"; transcript_id \"chr9.g6450.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10915490', '10915731', '.', '-', '.', 'gene_id \"chr9.g6450\"; transcript_id \"chr9.g6450.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10916049', '10916313', '.', '-', '.', 'gene_id \"chr9.g6451\"; transcript_id \"chr9.g6451.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10919868', '10920268', '.', '+', '.', 'gene_id \"chr9.g6454\"; transcript_id \"chr9.g6454.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10937379', '10937545', '.', '+', '.', 'gene_id \"chr9.g6466\"; transcript_id \"chr9.g6466.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10941815', '10942040', '.', '+', '.', 'gene_id \"chr9.g6469\"; transcript_id \"chr9.g6469.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10943192', '10943372', '.', '+', '.', 'gene_id \"chr9.g6471\"; transcript_id \"chr9.g6471.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10942891', '10943203', '.', '+', '.', 'gene_id \"chr9.g6472\"; transcript_id \"chr9.g6472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10946436', '10946632', '.', '-', '.', 'gene_id \"chr9.g6477\"; transcript_id \"chr9.g6477.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10948141', '10948293', '.', '-', '.', 'gene_id \"chr9.g6481\"; transcript_id \"chr9.g6481.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10951987', '10952200', '.', '-', '.', 'gene_id \"chr9.g6484\"; transcript_id \"chr9.g6484.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10952296', '10953024', '.', '+', '.', 'gene_id \"chr9.g6485\"; transcript_id \"chr9.g6485.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10953004', '10953210', '.', '-', '.', 'gene_id \"chr9.g6486\"; transcript_id \"chr9.g6486.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10953650', '10953819', '.', '+', '.', 'gene_id \"chr9.g6487\"; transcript_id \"chr9.g6487.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10955480', '10955660', '.', '+', '.', 'gene_id \"chr9.g6489\"; transcript_id \"chr9.g6489.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10961323', '10961537', '.', '-', '.', 'gene_id \"chr9.g6492\"; transcript_id \"chr9.g6492.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10986963', '10987163', '.', '+', '.', 'gene_id \"chr9.g6497\"; transcript_id \"chr9.g6497.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10990702', '10990875', '.', '+', '.', 'gene_id \"chr9.g6497\"; transcript_id \"chr9.g6497.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10982498', '10982547', '.', '-', '.', 'gene_id \"chr9.g6498\"; transcript_id \"chr9.g6498.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10970183', '10970494', '.', '+', '.', 'gene_id \"chr9.g6516\"; transcript_id \"chr9.g6516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10983711', '10983842', '.', '+', '.', 'gene_id \"chr9.g6519\"; transcript_id \"chr9.g6519.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10995365', '10995522', '.', '-', '.', 'gene_id \"chr9.g6529\"; transcript_id \"chr9.g6529.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10995513', '10995861', '.', '+', '.', 'gene_id \"chr9.g6530\"; transcript_id \"chr9.g6530.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11034833', '11035059', '.', '+', '.', 'gene_id \"chr9.g6532\"; transcript_id \"chr9.g6532.i67\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11213671', '11214048', '.', '+', '.', 'gene_id \"chr9.g6533\"; transcript_id \"chr9.g6533.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11213671', '11214048', '.', '+', '.', 'gene_id \"chr9.g6533\"; transcript_id \"chr9.g6533.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11213671', '11214048', '.', '+', '.', 'gene_id \"chr9.g6533\"; transcript_id \"chr9.g6533.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11189819', '11190345', '.', '+', '.', 'gene_id \"chr9.g6533\"; transcript_id \"chr9.g6533.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11169639', '11169989', '.', '+', '.', 'gene_id \"chr9.g6533\"; transcript_id \"chr9.g6533.i17\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11024551', '11024817', '.', '-', '.', 'gene_id \"chr9.g6535\"; transcript_id \"chr9.g6535.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11092671', '11092895', '.', '-', '.', 'gene_id \"chr9.g6536\"; transcript_id \"chr9.g6536.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11015720', '11016563', '.', '-', '.', 'gene_id \"chr9.g6537\"; transcript_id \"chr9.g6537.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11023692', '11023923', '.', '+', '.', 'gene_id \"chr9.g6538\"; transcript_id \"chr9.g6538.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11023185', '11023667', '.', '-', '.', 'gene_id \"chr9.g6539\"; transcript_id \"chr9.g6539.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11188413', '11188806', '.', '-', '.', 'gene_id \"chr9.g6541\"; transcript_id \"chr9.g6541.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11097085', '11097539', '.', '+', '.', 'gene_id \"chr9.g6547\"; transcript_id \"chr9.g6547.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11014933', '11015157', '.', '-', '.', 'gene_id \"chr9.g6554\"; transcript_id \"chr9.g6554.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11015151', '11015427', '.', '+', '.', 'gene_id \"chr9.g6555\"; transcript_id \"chr9.g6555.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11138604', '11139050', '.', '+', '.', 'gene_id \"chr9.g6558\"; transcript_id \"chr9.g6558.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11014227', '11014496', '.', '+', '.', 'gene_id \"chr9.g6559\"; transcript_id \"chr9.g6559.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11115923', '11116200', '.', '-', '.', 'gene_id \"chr9.g6562\"; transcript_id \"chr9.g6562.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11138109', '11138359', '.', '-', '.', 'gene_id \"chr9.g6563\"; transcript_id \"chr9.g6563.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11138340', '11138577', '.', '-', '.', 'gene_id \"chr9.g6564\"; transcript_id \"chr9.g6564.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11006975', '11007282', '.', '+', '.', 'gene_id \"chr9.g6580\"; transcript_id \"chr9.g6580.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11010460', '11010652', '.', '+', '.', 'gene_id \"chr9.g6584\"; transcript_id \"chr9.g6584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11011804', '11012204', '.', '-', '.', 'gene_id \"chr9.g6586\"; transcript_id \"chr9.g6586.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11012284', '11012446', '.', '-', '.', 'gene_id \"chr9.g6587\"; transcript_id \"chr9.g6587.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11014595', '11014891', '.', '-', '.', 'gene_id \"chr9.g6589\"; transcript_id \"chr9.g6589.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11015439', '11015674', '.', '-', '.', 'gene_id \"chr9.g6590\"; transcript_id \"chr9.g6590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11018609', '11019018', '.', '+', '.', 'gene_id \"chr9.g6591\"; transcript_id \"chr9.g6591.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11018609', '11019030', '.', '+', '.', 'gene_id \"chr9.g6591\"; transcript_id \"chr9.g6591.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11018628', '11018669', '.', '-', '.', 'gene_id \"chr9.g6592\"; transcript_id \"chr9.g6592.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11019075', '11019400', '.', '+', '.', 'gene_id \"chr9.g6593\"; transcript_id \"chr9.g6593.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11019854', '11020483', '.', '+', '.', 'gene_id \"chr9.g6594\"; transcript_id \"chr9.g6594.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11020099', '11020483', '.', '+', '.', 'gene_id \"chr9.g6594\"; transcript_id \"chr9.g6594.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11021235', '11021583', '.', '-', '.', 'gene_id \"chr9.g6595\"; transcript_id \"chr9.g6595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11021808', '11022174', '.', '+', '.', 'gene_id \"chr9.g6597\"; transcript_id \"chr9.g6597.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11029667', '11029982', '.', '-', '.', 'gene_id \"chr9.g6601\"; transcript_id \"chr9.g6601.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11031531', '11031844', '.', '+', '.', 'gene_id \"chr9.g6603\"; transcript_id \"chr9.g6603.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11032378', '11032637', '.', '+', '.', 'gene_id \"chr9.g6605\"; transcript_id \"chr9.g6605.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11034188', '11034413', '.', '-', '.', 'gene_id \"chr9.g6609\"; transcript_id \"chr9.g6609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11035092', '11035270', '.', '-', '.', 'gene_id \"chr9.g6611\"; transcript_id \"chr9.g6611.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11035390', '11035620', '.', '-', '.', 'gene_id \"chr9.g6613\"; transcript_id \"chr9.g6613.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11036174', '11036336', '.', '+', '.', 'gene_id \"chr9.g6614\"; transcript_id \"chr9.g6614.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11042138', '11042375', '.', '-', '.', 'gene_id \"chr9.g6616\"; transcript_id \"chr9.g6616.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11042372', '11042530', '.', '+', '.', 'gene_id \"chr9.g6617\"; transcript_id \"chr9.g6617.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11047355', '11047522', '.', '-', '.', 'gene_id \"chr9.g6624\"; transcript_id \"chr9.g6624.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11069335', '11070073', '.', '-', '.', 'gene_id \"chr9.g6633\"; transcript_id \"chr9.g6633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11070062', '11070213', '.', '-', '.', 'gene_id \"chr9.g6634\"; transcript_id \"chr9.g6634.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11072340', '11072581', '.', '+', '.', 'gene_id \"chr9.g6635\"; transcript_id \"chr9.g6635.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11072814', '11072946', '.', '+', '.', 'gene_id \"chr9.g6638\"; transcript_id \"chr9.g6638.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11075151', '11075446', '.', '+', '.', 'gene_id \"chr9.g6641\"; transcript_id \"chr9.g6641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11083751', '11083998', '.', '+', '.', 'gene_id \"chr9.g6646\"; transcript_id \"chr9.g6646.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11094413', '11094518', '.', '+', '.', 'gene_id \"chr9.g6651\"; transcript_id \"chr9.g6651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11096602', '11097020', '.', '+', '.', 'gene_id \"chr9.g6653\"; transcript_id \"chr9.g6653.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11098406', '11098600', '.', '+', '.', 'gene_id \"chr9.g6654\"; transcript_id \"chr9.g6654.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11100096', '11100521', '.', '-', '.', 'gene_id \"chr9.g6656\"; transcript_id \"chr9.g6656.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11100538', '11101068', '.', '-', '.', 'gene_id \"chr9.g6657\"; transcript_id \"chr9.g6657.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11101276', '11101479', '.', '-', '.', 'gene_id \"chr9.g6659\"; transcript_id \"chr9.g6659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11103583', '11103735', '.', '+', '.', 'gene_id \"chr9.g6664\"; transcript_id \"chr9.g6664.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11111538', '11111917', '.', '-', '.', 'gene_id \"chr9.g6667\"; transcript_id \"chr9.g6667.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11112535', '11112750', '.', '+', '.', 'gene_id \"chr9.g6668\"; transcript_id \"chr9.g6668.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11112775', '11112932', '.', '+', '.', 'gene_id \"chr9.g6669\"; transcript_id \"chr9.g6669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11117253', '11117442', '.', '+', '.', 'gene_id \"chr9.g6674\"; transcript_id \"chr9.g6674.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11117424', '11117679', '.', '-', '.', 'gene_id \"chr9.g6675\"; transcript_id \"chr9.g6675.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11117662', '11118060', '.', '-', '.', 'gene_id \"chr9.g6676\"; transcript_id \"chr9.g6676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11118383', '11118967', '.', '+', '.', 'gene_id \"chr9.g6678\"; transcript_id \"chr9.g6678.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11135278', '11136034', '.', '-', '.', 'gene_id \"chr9.g6693\"; transcript_id \"chr9.g6693.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11139999', '11140225', '.', '-', '.', 'gene_id \"chr9.g6697\"; transcript_id \"chr9.g6697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11140205', '11140399', '.', '+', '.', 'gene_id \"chr9.g6698\"; transcript_id \"chr9.g6698.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11141650', '11141816', '.', '+', '.', 'gene_id \"chr9.g6701\"; transcript_id \"chr9.g6701.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11142360', '11142576', '.', '-', '.', 'gene_id \"chr9.g6702\"; transcript_id \"chr9.g6702.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11143456', '11143619', '.', '+', '.', 'gene_id \"chr9.g6703\"; transcript_id \"chr9.g6703.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11146120', '11146627', '.', '+', '.', 'gene_id \"chr9.g6704\"; transcript_id \"chr9.g6704.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11149710', '11149927', '.', '-', '.', 'gene_id \"chr9.g6709\"; transcript_id \"chr9.g6709.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11150059', '11150613', '.', '+', '.', 'gene_id \"chr9.g6710\"; transcript_id \"chr9.g6710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11151099', '11151347', '.', '-', '.', 'gene_id \"chr9.g6712\"; transcript_id \"chr9.g6712.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11151736', '11151919', '.', '-', '.', 'gene_id \"chr9.g6713\"; transcript_id \"chr9.g6713.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11152502', '11152713', '.', '-', '.', 'gene_id \"chr9.g6715\"; transcript_id \"chr9.g6715.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11152852', '11153505', '.', '+', '.', 'gene_id \"chr9.g6716\"; transcript_id \"chr9.g6716.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11155436', '11156538', '.', '+', '.', 'gene_id \"chr9.g6717\"; transcript_id \"chr9.g6717.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11156103', '11156538', '.', '+', '.', 'gene_id \"chr9.g6717\"; transcript_id \"chr9.g6717.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11161838', '11162040', '.', '+', '.', 'gene_id \"chr9.g6725\"; transcript_id \"chr9.g6725.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11181661', '11182020', '.', '-', '.', 'gene_id \"chr9.g6731\"; transcript_id \"chr9.g6731.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11182028', '11182408', '.', '+', '.', 'gene_id \"chr9.g6732\"; transcript_id \"chr9.g6732.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11191163', '11191686', '.', '+', '.', 'gene_id \"chr9.g6738\"; transcript_id \"chr9.g6738.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11191163', '11191356', '.', '+', '.', 'gene_id \"chr9.g6738\"; transcript_id \"chr9.g6738.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11192037', '11193079', '.', '+', '.', 'gene_id \"chr9.g6740\"; transcript_id \"chr9.g6740.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11192547', '11193066', '.', '-', '.', 'gene_id \"chr9.g6741\"; transcript_id \"chr9.g6741.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11198234', '11198403', '.', '+', '.', 'gene_id \"chr9.g6747\"; transcript_id \"chr9.g6747.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11199891', '11200177', '.', '+', '.', 'gene_id \"chr9.g6751\"; transcript_id \"chr9.g6751.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11200273', '11201020', '.', '-', '.', 'gene_id \"chr9.g6752\"; transcript_id \"chr9.g6752.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11200273', '11201011', '.', '+', '.', 'gene_id \"chr9.g6753\"; transcript_id \"chr9.g6753.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11202101', '11202337', '.', '+', '.', 'gene_id \"chr9.g6755\"; transcript_id \"chr9.g6755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11203405', '11203575', '.', '+', '.', 'gene_id \"chr9.g6758\"; transcript_id \"chr9.g6758.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11204803', '11205029', '.', '-', '.', 'gene_id \"chr9.g6760\"; transcript_id \"chr9.g6760.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11206067', '11206272', '.', '+', '.', 'gene_id \"chr9.g6761\"; transcript_id \"chr9.g6761.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11208584', '11208755', '.', '+', '.', 'gene_id \"chr9.g6765\"; transcript_id \"chr9.g6765.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11209212', '11209373', '.', '-', '.', 'gene_id \"chr9.g6767\"; transcript_id \"chr9.g6767.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11223093', '11223246', '.', '-', '.', 'gene_id \"chr9.g6777\"; transcript_id \"chr9.g6777.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11232928', '11233352', '.', '-', '.', 'gene_id \"chr9.g6788\"; transcript_id \"chr9.g6788.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11237452', '11237616', '.', '+', '.', 'gene_id \"chr9.g6793\"; transcript_id \"chr9.g6793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11236992', '11237473', '.', '+', '.', 'gene_id \"chr9.g6793\"; transcript_id \"chr9.g6793.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11245116', '11245274', '.', '-', '.', 'gene_id \"chr9.g6795\"; transcript_id \"chr9.g6795.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11256951', '11257256', '.', '-', '.', 'gene_id \"chr9.g6800\"; transcript_id \"chr9.g6800.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11256666', '11256943', '.', '-', '.', 'gene_id \"chr9.g6804\"; transcript_id \"chr9.g6804.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11257597', '11257888', '.', '-', '.', 'gene_id \"chr9.g6805\"; transcript_id \"chr9.g6805.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11258882', '11259201', '.', '-', '.', 'gene_id \"chr9.g6807\"; transcript_id \"chr9.g6807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11259198', '11259543', '.', '+', '.', 'gene_id \"chr9.g6808\"; transcript_id \"chr9.g6808.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11261236', '11261385', '.', '+', '.', 'gene_id \"chr9.g6812\"; transcript_id \"chr9.g6812.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11263602', '11264034', '.', '+', '.', 'gene_id \"chr9.g6814\"; transcript_id \"chr9.g6814.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11264016', '11264186', '.', '-', '.', 'gene_id \"chr9.g6815\"; transcript_id \"chr9.g6815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11293587', '11296165', '.', '-', '.', 'gene_id \"chr9.g6829\"; transcript_id \"chr9.g6829.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11292708', '11293581', '.', '-', '.', 'gene_id \"chr9.g6831\"; transcript_id \"chr9.g6831.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11411492', '11411792', '.', '+', '.', 'gene_id \"chr9.g6832\"; transcript_id \"chr9.g6832.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11401988', '11402365', '.', '+', '.', 'gene_id \"chr9.g6837\"; transcript_id \"chr9.g6837.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11299549', '11299836', '.', '-', '.', 'gene_id \"chr9.g6839\"; transcript_id \"chr9.g6839.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11305061', '11305220', '.', '+', '.', 'gene_id \"chr9.g6842\"; transcript_id \"chr9.g6842.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11310640', '11310761', '.', '-', '.', 'gene_id \"chr9.g6843\"; transcript_id \"chr9.g6843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11315408', '11315535', '.', '-', '.', 'gene_id \"chr9.g6852\"; transcript_id \"chr9.g6852.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11315408', '11315441', '.', '-', '.', 'gene_id \"chr9.g6852\"; transcript_id \"chr9.g6852.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11316077', '11316369', '.', '-', '.', 'gene_id \"chr9.g6853\"; transcript_id \"chr9.g6853.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11316758', '11317066', '.', '-', '.', 'gene_id \"chr9.g6855\"; transcript_id \"chr9.g6855.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11316758', '11316981', '.', '-', '.', 'gene_id \"chr9.g6855\"; transcript_id \"chr9.g6855.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11320054', '11320600', '.', '+', '.', 'gene_id \"chr9.g6857\"; transcript_id \"chr9.g6857.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11321155', '11321383', '.', '-', '.', 'gene_id \"chr9.g6858\"; transcript_id \"chr9.g6858.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11321170', '11321397', '.', '-', '.', 'gene_id \"chr9.g6858\"; transcript_id \"chr9.g6858.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11322088', '11322353', '.', '-', '.', 'gene_id \"chr9.g6859\"; transcript_id \"chr9.g6859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11322700', '11322857', '.', '-', '.', 'gene_id \"chr9.g6860\"; transcript_id \"chr9.g6860.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11324558', '11325054', '.', '+', '.', 'gene_id \"chr9.g6862\"; transcript_id \"chr9.g6862.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11327146', '11327458', '.', '+', '.', 'gene_id \"chr9.g6865\"; transcript_id \"chr9.g6865.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11327445', '11327605', '.', '+', '.', 'gene_id \"chr9.g6866\"; transcript_id \"chr9.g6866.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11332935', '11333119', '.', '-', '.', 'gene_id \"chr9.g6870\"; transcript_id \"chr9.g6870.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11333203', '11333471', '.', '-', '.', 'gene_id \"chr9.g6872\"; transcript_id \"chr9.g6872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11348719', '11348749', '.', '+', '.', 'gene_id \"chr9.g6878\"; transcript_id \"chr9.g6878.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11355339', '11355493', '.', '-', '.', 'gene_id \"chr9.g6880\"; transcript_id \"chr9.g6880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11361873', '11362021', '.', '+', '.', 'gene_id \"chr9.g6884\"; transcript_id \"chr9.g6884.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11361373', '11361586', '.', '-', '.', 'gene_id \"chr9.g6885\"; transcript_id \"chr9.g6885.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11361530', '11361860', '.', '-', '.', 'gene_id \"chr9.g6885\"; transcript_id \"chr9.g6885.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11362001', '11362232', '.', '-', '.', 'gene_id \"chr9.g6886\"; transcript_id \"chr9.g6886.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11367575', '11367833', '.', '+', '.', 'gene_id \"chr9.g6887\"; transcript_id \"chr9.g6887.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11367993', '11368152', '.', '+', '.', 'gene_id \"chr9.g6888\"; transcript_id \"chr9.g6888.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11373974', '11374184', '.', '+', '.', 'gene_id \"chr9.g6893\"; transcript_id \"chr9.g6893.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11374652', '11375004', '.', '-', '.', 'gene_id \"chr9.g6895\"; transcript_id \"chr9.g6895.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11376575', '11376959', '.', '+', '.', 'gene_id \"chr9.g6896\"; transcript_id \"chr9.g6896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11386673', '11386707', '.', '-', '.', 'gene_id \"chr9.g6899\"; transcript_id \"chr9.g6899.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11395326', '11395498', '.', '+', '.', 'gene_id \"chr9.g6903\"; transcript_id \"chr9.g6903.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11395509', '11395678', '.', '-', '.', 'gene_id \"chr9.g6904\"; transcript_id \"chr9.g6904.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11396133', '11396299', '.', '-', '.', 'gene_id \"chr9.g6906\"; transcript_id \"chr9.g6906.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11396375', '11396564', '.', '-', '.', 'gene_id \"chr9.g6907\"; transcript_id \"chr9.g6907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11398531', '11398998', '.', '-', '.', 'gene_id \"chr9.g6910\"; transcript_id \"chr9.g6910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11399335', '11399507', '.', '+', '.', 'gene_id \"chr9.g6912\"; transcript_id \"chr9.g6912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11399494', '11399686', '.', '-', '.', 'gene_id \"chr9.g6913\"; transcript_id \"chr9.g6913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11399674', '11400302', '.', '+', '.', 'gene_id \"chr9.g6914\"; transcript_id \"chr9.g6914.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11404388', '11404625', '.', '+', '.', 'gene_id \"chr9.g6918\"; transcript_id \"chr9.g6918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11404960', '11405174', '.', '+', '.', 'gene_id \"chr9.g6920\"; transcript_id \"chr9.g6920.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11410078', '11411306', '.', '+', '.', 'gene_id \"chr9.g6927\"; transcript_id \"chr9.g6927.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11414715', '11414880', '.', '+', '.', 'gene_id \"chr9.g6929\"; transcript_id \"chr9.g6929.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11418268', '11418618', '.', '+', '.', 'gene_id \"chr9.g6930\"; transcript_id \"chr9.g6930.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11418806', '11419017', '.', '+', '.', 'gene_id \"chr9.g6931\"; transcript_id \"chr9.g6931.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11419336', '11419737', '.', '+', '.', 'gene_id \"chr9.g6933\"; transcript_id \"chr9.g6933.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11421113', '11421662', '.', '+', '.', 'gene_id \"chr9.g6936\"; transcript_id \"chr9.g6936.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11421672', '11421842', '.', '-', '.', 'gene_id \"chr9.g6937\"; transcript_id \"chr9.g6937.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11421851', '11422023', '.', '-', '.', 'gene_id \"chr9.g6938\"; transcript_id \"chr9.g6938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11422941', '11423188', '.', '+', '.', 'gene_id \"chr9.g6940\"; transcript_id \"chr9.g6940.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11423393', '11423722', '.', '-', '.', 'gene_id \"chr9.g6942\"; transcript_id \"chr9.g6942.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11423939', '11424259', '.', '+', '.', 'gene_id \"chr9.g6943\"; transcript_id \"chr9.g6943.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '15321689', '15321932', '.', '-', '.', 'gene_id \"chr17.g3957\"; transcript_id \"chr17.g3957.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11473516', '11473707', '.', '+', '.', 'gene_id \"chr9.g6948\"; transcript_id \"chr9.g6948.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11428599', '11429231', '.', '-', '.', 'gene_id \"chr9.g6959\"; transcript_id \"chr9.g6959.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11429598', '11429790', '.', '-', '.', 'gene_id \"chr9.g6961\"; transcript_id \"chr9.g6961.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11430617', '11430867', '.', '+', '.', 'gene_id \"chr9.g6963\"; transcript_id \"chr9.g6963.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11431409', '11432974', '.', '-', '.', 'gene_id \"chr9.g6965\"; transcript_id \"chr9.g6965.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11436260', '11436332', '.', '-', '.', 'gene_id \"chr9.g6971\"; transcript_id \"chr9.g6971.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11439322', '11439495', '.', '-', '.', 'gene_id \"chr9.g6976\"; transcript_id \"chr9.g6976.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11441193', '11441266', '.', '+', '.', 'gene_id \"chr9.g6981\"; transcript_id \"chr9.g6981.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11442421', '11443082', '.', '+', '.', 'gene_id \"chr9.g6983\"; transcript_id \"chr9.g6983.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11444490', '11444701', '.', '-', '.', 'gene_id \"chr9.g6988\"; transcript_id \"chr9.g6988.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11445653', '11446566', '.', '-', '.', 'gene_id \"chr9.g6991\"; transcript_id \"chr9.g6991.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11446544', '11446841', '.', '+', '.', 'gene_id \"chr9.g6992\"; transcript_id \"chr9.g6992.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11450972', '11451189', '.', '-', '.', 'gene_id \"chr9.g7004\"; transcript_id \"chr9.g7004.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11451349', '11451511', '.', '-', '.', 'gene_id \"chr9.g7005\"; transcript_id \"chr9.g7005.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11452717', '11452852', '.', '-', '.', 'gene_id \"chr9.g7007\"; transcript_id \"chr9.g7007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11453034', '11454510', '.', '-', '.', 'gene_id \"chr9.g7008\"; transcript_id \"chr9.g7008.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11454543', '11454722', '.', '-', '.', 'gene_id \"chr9.g7010\"; transcript_id \"chr9.g7010.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11455845', '11456188', '.', '+', '.', 'gene_id \"chr9.g7012\"; transcript_id \"chr9.g7012.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11457146', '11457467', '.', '-', '.', 'gene_id \"chr9.g7014\"; transcript_id \"chr9.g7014.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11457477', '11457869', '.', '+', '.', 'gene_id \"chr9.g7015\"; transcript_id \"chr9.g7015.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11458707', '11458884', '.', '+', '.', 'gene_id \"chr9.g7017\"; transcript_id \"chr9.g7017.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11462790', '11463350', '.', '-', '.', 'gene_id \"chr9.g7022\"; transcript_id \"chr9.g7022.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11465305', '11465455', '.', '+', '.', 'gene_id \"chr9.g7025\"; transcript_id \"chr9.g7025.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11466494', '11466662', '.', '+', '.', 'gene_id \"chr9.g7028\"; transcript_id \"chr9.g7028.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11494771', '11494932', '.', '+', '.', 'gene_id \"chr9.g7045\"; transcript_id \"chr9.g7045.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11495782', '11495934', '.', '+', '.', 'gene_id \"chr9.g7047\"; transcript_id \"chr9.g7047.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11496023', '11496319', '.', '+', '.', 'gene_id \"chr9.g7048\"; transcript_id \"chr9.g7048.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11496604', '11497015', '.', '+', '.', 'gene_id \"chr9.g7050\"; transcript_id \"chr9.g7050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11497369', '11497945', '.', '+', '.', 'gene_id \"chr9.g7051\"; transcript_id \"chr9.g7051.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11497948', '11498527', '.', '-', '.', 'gene_id \"chr9.g7052\"; transcript_id \"chr9.g7052.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11499204', '11499537', '.', '+', '.', 'gene_id \"chr9.g7054\"; transcript_id \"chr9.g7054.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11502386', '11502667', '.', '-', '.', 'gene_id \"chr9.g7057\"; transcript_id \"chr9.g7057.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11509247', '11509505', '.', '-', '.', 'gene_id \"chr9.g7062\"; transcript_id \"chr9.g7062.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11510611', '11510782', '.', '-', '.', 'gene_id \"chr9.g7063\"; transcript_id \"chr9.g7063.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11510838', '11511255', '.', '-', '.', 'gene_id \"chr9.g7064\"; transcript_id \"chr9.g7064.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11512422', '11512742', '.', '+', '.', 'gene_id \"chr9.g7066\"; transcript_id \"chr9.g7066.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11520732', '11521421', '.', '-', '.', 'gene_id \"chr9.g7074\"; transcript_id \"chr9.g7074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11521522', '11521711', '.', '-', '.', 'gene_id \"chr9.g7075\"; transcript_id \"chr9.g7075.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '15382454', '15382533', '.', '-', '.', 'gene_id \"chr17.g4093\"; transcript_id \"chr17.g4093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11536839', '11537035', '.', '-', '.', 'gene_id \"chr9.g7081\"; transcript_id \"chr9.g7081.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11530460', '11530681', '.', '+', '.', 'gene_id \"chr9.g7083\"; transcript_id \"chr9.g7083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11577741', '11578110', '.', '+', '.', 'gene_id \"chr9.g7084\"; transcript_id \"chr9.g7084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11530790', '11531115', '.', '+', '.', 'gene_id \"chr9.g7086\"; transcript_id \"chr9.g7086.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11531503', '11531698', '.', '-', '.', 'gene_id \"chr9.g7087\"; transcript_id \"chr9.g7087.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11532935', '11533082', '.', '+', '.', 'gene_id \"chr9.g7091\"; transcript_id \"chr9.g7091.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11533588', '11533747', '.', '+', '.', 'gene_id \"chr9.g7093\"; transcript_id \"chr9.g7093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11535263', '11535420', '.', '-', '.', 'gene_id \"chr9.g7095\"; transcript_id \"chr9.g7095.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11536277', '11536769', '.', '+', '.', 'gene_id \"chr9.g7096\"; transcript_id \"chr9.g7096.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11542438', '11542648', '.', '+', '.', 'gene_id \"chr9.g7101\"; transcript_id \"chr9.g7101.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11564505', '11564687', '.', '+', '.', 'gene_id \"chr9.g7103\"; transcript_id \"chr9.g7103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11566284', '11566463', '.', '+', '.', 'gene_id \"chr9.g7105\"; transcript_id \"chr9.g7105.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11566443', '11566641', '.', '-', '.', 'gene_id \"chr9.g7106\"; transcript_id \"chr9.g7106.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11567266', '11567614', '.', '+', '.', 'gene_id \"chr9.g7107\"; transcript_id \"chr9.g7107.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11568959', '11569173', '.', '-', '.', 'gene_id \"chr9.g7108\"; transcript_id \"chr9.g7108.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11593650', '11593917', '.', '-', '.', 'gene_id \"chr9.g7117\"; transcript_id \"chr9.g7117.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11593894', '11594067', '.', '+', '.', 'gene_id \"chr9.g7118\"; transcript_id \"chr9.g7118.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11601054', '11601204', '.', '+', '.', 'gene_id \"chr9.g7124\"; transcript_id \"chr9.g7124.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11601223', '11601507', '.', '-', '.', 'gene_id \"chr9.g7125\"; transcript_id \"chr9.g7125.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11605976', '11606154', '.', '-', '.', 'gene_id \"chr9.g7127\"; transcript_id \"chr9.g7127.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11606480', '11606742', '.', '-', '.', 'gene_id \"chr9.g7128\"; transcript_id \"chr9.g7128.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11608403', '11608649', '.', '-', '.', 'gene_id \"chr9.g7131\"; transcript_id \"chr9.g7131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11612524', '11612783', '.', '-', '.', 'gene_id \"chr9.g7136\"; transcript_id \"chr9.g7136.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11614782', '11615836', '.', '-', '.', 'gene_id \"chr9.g7141\"; transcript_id \"chr9.g7141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11623215', '11623381', '.', '-', '.', 'gene_id \"chr9.g7145\"; transcript_id \"chr9.g7145.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11623672', '11623848', '.', '-', '.', 'gene_id \"chr9.g7146\"; transcript_id \"chr9.g7146.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11690649', '11690765', '.', '-', '.', 'gene_id \"chr9.g7161\"; transcript_id \"chr9.g7161.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11693660', '11693941', '.', '+', '.', 'gene_id \"chr9.g7162\"; transcript_id \"chr9.g7162.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11693948', '11694321', '.', '+', '.', 'gene_id \"chr9.g7164\"; transcript_id \"chr9.g7164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11694592', '11694788', '.', '-', '.', 'gene_id \"chr9.g7165\"; transcript_id \"chr9.g7165.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11697761', '11697970', '.', '-', '.', 'gene_id \"chr9.g7169\"; transcript_id \"chr9.g7169.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11698031', '11698144', '.', '-', '.', 'gene_id \"chr9.g7170\"; transcript_id \"chr9.g7170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11703157', '11703281', '.', '+', '.', 'gene_id \"chr9.g7180\"; transcript_id \"chr9.g7180.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11702964', '11703048', '.', '-', '.', 'gene_id \"chr9.g7181\"; transcript_id \"chr9.g7181.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11702964', '11703143', '.', '-', '.', 'gene_id \"chr9.g7181\"; transcript_id \"chr9.g7181.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11702964', '11703150', '.', '-', '.', 'gene_id \"chr9.g7181\"; transcript_id \"chr9.g7181.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11750169', '11750405', '.', '+', '.', 'gene_id \"chr9.g7187\"; transcript_id \"chr9.g7187.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11788315', '11788501', '.', '+', '.', 'gene_id \"chr9.g7192\"; transcript_id \"chr9.g7192.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11815204', '11815368', '.', '+', '.', 'gene_id \"chr9.g7196\"; transcript_id \"chr9.g7196.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11815413', '11815656', '.', '-', '.', 'gene_id \"chr9.g7197\"; transcript_id \"chr9.g7197.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11840884', '11841091', '.', '-', '.', 'gene_id \"chr9.g7206\"; transcript_id \"chr9.g7206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11843128', '11843405', '.', '+', '.', 'gene_id \"chr9.g7208\"; transcript_id \"chr9.g7208.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11843504', '11843671', '.', '+', '.', 'gene_id \"chr9.g7209\"; transcript_id \"chr9.g7209.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11843525', '11843671', '.', '+', '.', 'gene_id \"chr9.g7209\"; transcript_id \"chr9.g7209.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11845516', '11845982', '.', '+', '.', 'gene_id \"chr9.g7211\"; transcript_id \"chr9.g7211.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11845971', '11846327', '.', '+', '.', 'gene_id \"chr9.g7212\"; transcript_id \"chr9.g7212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11846833', '11847097', '.', '+', '.', 'gene_id \"chr9.g7216\"; transcript_id \"chr9.g7216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11849473', '11849632', '.', '+', '.', 'gene_id \"chr9.g7220\"; transcript_id \"chr9.g7220.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11849628', '11849870', '.', '-', '.', 'gene_id \"chr9.g7221\"; transcript_id \"chr9.g7221.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11860127', '11860349', '.', '-', '.', 'gene_id \"chr9.g7226\"; transcript_id \"chr9.g7226.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11860650', '11860906', '.', '-', '.', 'gene_id \"chr9.g7228\"; transcript_id \"chr9.g7228.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11860985', '11861335', '.', '+', '.', 'gene_id \"chr9.g7229\"; transcript_id \"chr9.g7229.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11861349', '11861688', '.', '+', '.', 'gene_id \"chr9.g7230\"; transcript_id \"chr9.g7230.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11868949', '11869099', '.', '-', '.', 'gene_id \"chr9.g7237\"; transcript_id \"chr9.g7237.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11876801', '11876972', '.', '+', '.', 'gene_id \"chr9.g7242\"; transcript_id \"chr9.g7242.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11883549', '11883920', '.', '+', '.', 'gene_id \"chr9.g7243\"; transcript_id \"chr9.g7243.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11893321', '11893493', '.', '+', '.', 'gene_id \"chr9.g7245\"; transcript_id \"chr9.g7245.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11906370', '11906594', '.', '+', '.', 'gene_id \"chr9.g7252\"; transcript_id \"chr9.g7252.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11910413', '11910589', '.', '-', '.', 'gene_id \"chr9.g7255\"; transcript_id \"chr9.g7255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11912800', '11913483', '.', '-', '.', 'gene_id \"chr9.g7256\"; transcript_id \"chr9.g7256.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11918530', '11918696', '.', '+', '.', 'gene_id \"chr9.g7262\"; transcript_id \"chr9.g7262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11925185', '11925354', '.', '+', '.', 'gene_id \"chr9.g7264\"; transcript_id \"chr9.g7264.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11952105', '11952427', '.', '-', '.', 'gene_id \"chr9.g7268\"; transcript_id \"chr9.g7268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11954908', '11955103', '.', '-', '.', 'gene_id \"chr9.g7271\"; transcript_id \"chr9.g7271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11955584', '11956128', '.', '+', '.', 'gene_id \"chr9.g7272\"; transcript_id \"chr9.g7272.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11955360', '11955591', '.', '+', '.', 'gene_id \"chr9.g7273\"; transcript_id \"chr9.g7273.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11957974', '11958124', '.', '-', '.', 'gene_id \"chr9.g7275\"; transcript_id \"chr9.g7275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11967069', '11967303', '.', '+', '.', 'gene_id \"chr9.g7280\"; transcript_id \"chr9.g7280.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11973929', '11974031', '.', '-', '.', 'gene_id \"chr9.g7286\"; transcript_id \"chr9.g7286.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11974899', '11975082', '.', '-', '.', 'gene_id \"chr9.g7287\"; transcript_id \"chr9.g7287.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11987994', '11988063', '.', '+', '.', 'gene_id \"chr9.g7291\"; transcript_id \"chr9.g7291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12001961', '12002168', '.', '+', '.', 'gene_id \"chr9.g7306\"; transcript_id \"chr9.g7306.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12007536', '12007979', '.', '-', '.', 'gene_id \"chr9.g7311\"; transcript_id \"chr9.g7311.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12009046', '12009219', '.', '+', '.', 'gene_id \"chr9.g7313\"; transcript_id \"chr9.g7313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12009200', '12009438', '.', '-', '.', 'gene_id \"chr9.g7314\"; transcript_id \"chr9.g7314.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12009439', '12009659', '.', '+', '.', 'gene_id \"chr9.g7315\"; transcript_id \"chr9.g7315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12009680', '12009968', '.', '+', '.', 'gene_id \"chr9.g7317\"; transcript_id \"chr9.g7317.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12010148', '12010306', '.', '+', '.', 'gene_id \"chr9.g7318\"; transcript_id \"chr9.g7318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12010960', '12011145', '.', '+', '.', 'gene_id \"chr9.g7321\"; transcript_id \"chr9.g7321.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12011213', '12011371', '.', '+', '.', 'gene_id \"chr9.g7322\"; transcript_id \"chr9.g7322.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12017300', '12017449', '.', '-', '.', 'gene_id \"chr9.g7325\"; transcript_id \"chr9.g7325.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12027188', '12028073', '.', '+', '.', 'gene_id \"chr9.g7328\"; transcript_id \"chr9.g7328.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12029581', '12029786', '.', '-', '.', 'gene_id \"chr9.g7330\"; transcript_id \"chr9.g7330.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12030425', '12030656', '.', '+', '.', 'gene_id \"chr9.g7332\"; transcript_id \"chr9.g7332.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12032747', '12032925', '.', '+', '.', 'gene_id \"chr9.g7334\"; transcript_id \"chr9.g7334.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12040301', '12040551', '.', '+', '.', 'gene_id \"chr9.g7336\"; transcript_id \"chr9.g7336.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12041615', '12042354', '.', '+', '.', 'gene_id \"chr9.g7339\"; transcript_id \"chr9.g7339.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12042869', '12043122', '.', '+', '.', 'gene_id \"chr9.g7340\"; transcript_id \"chr9.g7340.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12043234', '12043620', '.', '-', '.', 'gene_id \"chr9.g7341\"; transcript_id \"chr9.g7341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12043616', '12044046', '.', '+', '.', 'gene_id \"chr9.g7342\"; transcript_id \"chr9.g7342.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12054650', '12054861', '.', '-', '.', 'gene_id \"chr9.g7354\"; transcript_id \"chr9.g7354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12057281', '12057920', '.', '-', '.', 'gene_id \"chr9.g7357\"; transcript_id \"chr9.g7357.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12068268', '12068426', '.', '-', '.', 'gene_id \"chr9.g7362\"; transcript_id \"chr9.g7362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12081872', '12082196', '.', '+', '.', 'gene_id \"chr9.g7365\"; transcript_id \"chr9.g7365.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12074502', '12074707', '.', '-', '.', 'gene_id \"chr9.g7368\"; transcript_id \"chr9.g7368.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12073323', '12073501', '.', '+', '.', 'gene_id \"chr9.g7369\"; transcript_id \"chr9.g7369.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12075087', '12075526', '.', '-', '.', 'gene_id \"chr9.g7372\"; transcript_id \"chr9.g7372.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12075531', '12076590', '.', '-', '.', 'gene_id \"chr9.g7373\"; transcript_id \"chr9.g7373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12076621', '12076792', '.', '+', '.', 'gene_id \"chr9.g7374\"; transcript_id \"chr9.g7374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12076979', '12077644', '.', '-', '.', 'gene_id \"chr9.g7375\"; transcript_id \"chr9.g7375.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12077855', '12078008', '.', '-', '.', 'gene_id \"chr9.g7377\"; transcript_id \"chr9.g7377.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12079270', '12079932', '.', '-', '.', 'gene_id \"chr9.g7379\"; transcript_id \"chr9.g7379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12080907', '12081264', '.', '-', '.', 'gene_id \"chr9.g7380\"; transcript_id \"chr9.g7380.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12081327', '12081483', '.', '+', '.', 'gene_id \"chr9.g7381\"; transcript_id \"chr9.g7381.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12084296', '12084740', '.', '-', '.', 'gene_id \"chr9.g7383\"; transcript_id \"chr9.g7383.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12085453', '12085628', '.', '-', '.', 'gene_id \"chr9.g7384\"; transcript_id \"chr9.g7384.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12087917', '12088367', '.', '+', '.', 'gene_id \"chr9.g7388\"; transcript_id \"chr9.g7388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12090421', '12091035', '.', '+', '.', 'gene_id \"chr9.g7389\"; transcript_id \"chr9.g7389.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12094376', '12094558', '.', '+', '.', 'gene_id \"chr9.g7391\"; transcript_id \"chr9.g7391.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12116656', '12116817', '.', '+', '.', 'gene_id \"chr9.g7401\"; transcript_id \"chr9.g7401.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12120511', '12120665', '.', '-', '.', 'gene_id \"chr9.g7402\"; transcript_id \"chr9.g7402.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12106514', '12106643', '.', '+', '.', 'gene_id \"chr9.g7404\"; transcript_id \"chr9.g7404.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12107761', '12107929', '.', '+', '.', 'gene_id \"chr9.g7406\"; transcript_id \"chr9.g7406.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12115506', '12115660', '.', '+', '.', 'gene_id \"chr9.g7408\"; transcript_id \"chr9.g7408.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12115929', '12116081', '.', '+', '.', 'gene_id \"chr9.g7409\"; transcript_id \"chr9.g7409.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12117687', '12117999', '.', '+', '.', 'gene_id \"chr9.g7410\"; transcript_id \"chr9.g7410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12118638', '12119090', '.', '-', '.', 'gene_id \"chr9.g7412\"; transcript_id \"chr9.g7412.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12119381', '12119689', '.', '+', '.', 'gene_id \"chr9.g7413\"; transcript_id \"chr9.g7413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12119748', '12119940', '.', '+', '.', 'gene_id \"chr9.g7414\"; transcript_id \"chr9.g7414.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12122069', '12122232', '.', '-', '.', 'gene_id \"chr9.g7416\"; transcript_id \"chr9.g7416.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12122796', '12122964', '.', '-', '.', 'gene_id \"chr9.g7417\"; transcript_id \"chr9.g7417.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12122796', '12123070', '.', '-', '.', 'gene_id \"chr9.g7417\"; transcript_id \"chr9.g7417.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12122475', '12122808', '.', '-', '.', 'gene_id \"chr9.g7418\"; transcript_id \"chr9.g7418.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12132100', '12132307', '.', '+', '.', 'gene_id \"chr9.g7424\"; transcript_id \"chr9.g7424.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12166718', '12166982', '.', '-', '.', 'gene_id \"chr9.g7428\"; transcript_id \"chr9.g7428.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12170704', '12171079', '.', '-', '.', 'gene_id \"chr9.g7430\"; transcript_id \"chr9.g7430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12174778', '12174930', '.', '+', '.', 'gene_id \"chr9.g7438\"; transcript_id \"chr9.g7438.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12177296', '12177564', '.', '-', '.', 'gene_id \"chr9.g7440\"; transcript_id \"chr9.g7440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12177604', '12177714', '.', '+', '.', 'gene_id \"chr9.g7443\"; transcript_id \"chr9.g7443.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12180826', '12181291', '.', '+', '.', 'gene_id \"chr9.g7446\"; transcript_id \"chr9.g7446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12181277', '12181479', '.', '-', '.', 'gene_id \"chr9.g7447\"; transcript_id \"chr9.g7447.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12180338', '12180830', '.', '+', '.', 'gene_id \"chr9.g7448\"; transcript_id \"chr9.g7448.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12180369', '12180830', '.', '-', '.', 'gene_id \"chr9.g7449\"; transcript_id \"chr9.g7449.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12181653', '12181816', '.', '+', '.', 'gene_id \"chr9.g7450\"; transcript_id \"chr9.g7450.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12181794', '12182037', '.', '-', '.', 'gene_id \"chr9.g7451\"; transcript_id \"chr9.g7451.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12181819', '12182037', '.', '+', '.', 'gene_id \"chr9.g7452\"; transcript_id \"chr9.g7452.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12183550', '12183856', '.', '+', '.', 'gene_id \"chr9.g7457\"; transcript_id \"chr9.g7457.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12185603', '12185840', '.', '+', '.', 'gene_id \"chr9.g7458\"; transcript_id \"chr9.g7458.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12193185', '12193411', '.', '-', '.', 'gene_id \"chr9.g7464\"; transcript_id \"chr9.g7464.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12194186', '12195360', '.', '+', '.', 'gene_id \"chr9.g7466\"; transcript_id \"chr9.g7466.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12195924', '12196238', '.', '-', '.', 'gene_id \"chr9.g7467\"; transcript_id \"chr9.g7467.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12201748', '12202113', '.', '+', '.', 'gene_id \"chr9.g7469\"; transcript_id \"chr9.g7469.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12202216', '12202384', '.', '-', '.', 'gene_id \"chr9.g7470\"; transcript_id \"chr9.g7470.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12202424', '12202457', '.', '-', '.', 'gene_id \"chr9.g7472\"; transcript_id \"chr9.g7472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12202764', '12203261', '.', '-', '.', 'gene_id \"chr9.g7473\"; transcript_id \"chr9.g7473.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12202569', '12202774', '.', '-', '.', 'gene_id \"chr9.g7474\"; transcript_id \"chr9.g7474.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12203377', '12203718', '.', '+', '.', 'gene_id \"chr9.g7475\"; transcript_id \"chr9.g7475.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12205294', '12205479', '.', '-', '.', 'gene_id \"chr9.g7478\"; transcript_id \"chr9.g7478.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12212015', '12212499', '.', '+', '.', 'gene_id \"chr9.g7479\"; transcript_id \"chr9.g7479.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12208406', '12208540', '.', '+', '.', 'gene_id \"chr9.g7480\"; transcript_id \"chr9.g7480.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12273139', '12273595', '.', '+', '.', 'gene_id \"chr9.g7492\"; transcript_id \"chr9.g7492.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12275833', '12276402', '.', '-', '.', 'gene_id \"chr9.g7494\"; transcript_id \"chr9.g7494.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12265627', '12265779', '.', '+', '.', 'gene_id \"chr9.g7498\"; transcript_id \"chr9.g7498.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12286822', '12287497', '.', '+', '.', 'gene_id \"chr9.g7499\"; transcript_id \"chr9.g7499.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12222949', '12223121', '.', '-', '.', 'gene_id \"chr9.g7504\"; transcript_id \"chr9.g7504.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12226092', '12226296', '.', '-', '.', 'gene_id \"chr9.g7506\"; transcript_id \"chr9.g7506.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12228285', '12228620', '.', '-', '.', 'gene_id \"chr9.g7509\"; transcript_id \"chr9.g7509.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12233955', '12234132', '.', '+', '.', 'gene_id \"chr9.g7519\"; transcript_id \"chr9.g7519.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12236377', '12236558', '.', '+', '.', 'gene_id \"chr9.g7524\"; transcript_id \"chr9.g7524.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12236182', '12236393', '.', '-', '.', 'gene_id \"chr9.g7526\"; transcript_id \"chr9.g7526.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12238409', '12238631', '.', '+', '.', 'gene_id \"chr9.g7528\"; transcript_id \"chr9.g7528.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12238649', '12239191', '.', '+', '.', 'gene_id \"chr9.g7529\"; transcript_id \"chr9.g7529.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12250007', '12250141', '.', '-', '.', 'gene_id \"chr9.g7537\"; transcript_id \"chr9.g7537.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12250007', '12250133', '.', '-', '.', 'gene_id \"chr9.g7537\"; transcript_id \"chr9.g7537.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12250007', '12250195', '.', '-', '.', 'gene_id \"chr9.g7537\"; transcript_id \"chr9.g7537.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12257936', '12258327', '.', '-', '.', 'gene_id \"chr9.g7544\"; transcript_id \"chr9.g7544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12261218', '12261376', '.', '+', '.', 'gene_id \"chr9.g7548\"; transcript_id \"chr9.g7548.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12261408', '12261685', '.', '-', '.', 'gene_id \"chr9.g7549\"; transcript_id \"chr9.g7549.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12264258', '12264831', '.', '+', '.', 'gene_id \"chr9.g7552\"; transcript_id \"chr9.g7552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12265841', '12266156', '.', '-', '.', 'gene_id \"chr9.g7555\"; transcript_id \"chr9.g7555.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12266618', '12266884', '.', '-', '.', 'gene_id \"chr9.g7557\"; transcript_id \"chr9.g7557.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12267665', '12268087', '.', '+', '.', 'gene_id \"chr9.g7560\"; transcript_id \"chr9.g7560.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12269038', '12269519', '.', '-', '.', 'gene_id \"chr9.g7562\"; transcript_id \"chr9.g7562.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12270190', '12270427', '.', '+', '.', 'gene_id \"chr9.g7564\"; transcript_id \"chr9.g7564.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12271132', '12271313', '.', '-', '.', 'gene_id \"chr9.g7567\"; transcript_id \"chr9.g7567.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12271473', '12271652', '.', '+', '.', 'gene_id \"chr9.g7568\"; transcript_id \"chr9.g7568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12271643', '12272079', '.', '-', '.', 'gene_id \"chr9.g7569\"; transcript_id \"chr9.g7569.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12272204', '12272375', '.', '+', '.', 'gene_id \"chr9.g7570\"; transcript_id \"chr9.g7570.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12273968', '12274169', '.', '+', '.', 'gene_id \"chr9.g7572\"; transcript_id \"chr9.g7572.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12274434', '12274524', '.', '+', '.', 'gene_id \"chr9.g7574\"; transcript_id \"chr9.g7574.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12274535', '12275443', '.', '-', '.', 'gene_id \"chr9.g7575\"; transcript_id \"chr9.g7575.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12277488', '12277655', '.', '+', '.', 'gene_id \"chr9.g7576\"; transcript_id \"chr9.g7576.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12277820', '12278008', '.', '+', '.', 'gene_id \"chr9.g7577\"; transcript_id \"chr9.g7577.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12282676', '12283033', '.', '+', '.', 'gene_id \"chr9.g7578\"; transcript_id \"chr9.g7578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12290450', '12290667', '.', '+', '.', 'gene_id \"chr9.g7583\"; transcript_id \"chr9.g7583.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12309573', '12311199', '.', '-', '.', 'gene_id \"chr9.g7588\"; transcript_id \"chr9.g7588.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12301083', '12301341', '.', '+', '.', 'gene_id \"chr9.g7590\"; transcript_id \"chr9.g7590.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12309255', '12309469', '.', '+', '.', 'gene_id \"chr9.g7591\"; transcript_id \"chr9.g7591.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12291656', '12291831', '.', '+', '.', 'gene_id \"chr9.g7595\"; transcript_id \"chr9.g7595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12299509', '12300295', '.', '+', '.', 'gene_id \"chr9.g7596\"; transcript_id \"chr9.g7596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12312315', '12312490', '.', '-', '.', 'gene_id \"chr9.g7597\"; transcript_id \"chr9.g7597.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12296819', '12297355', '.', '+', '.', 'gene_id \"chr9.g7598\"; transcript_id \"chr9.g7598.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12294607', '12294759', '.', '+', '.', 'gene_id \"chr9.g7599\"; transcript_id \"chr9.g7599.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12293629', '12293948', '.', '-', '.', 'gene_id \"chr9.g7600\"; transcript_id \"chr9.g7600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12301808', '12307975', '.', '+', '.', 'gene_id \"chr9.g7601\"; transcript_id \"chr9.g7601.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12300638', '12300996', '.', '+', '.', 'gene_id \"chr9.g7602\"; transcript_id \"chr9.g7602.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12296508', '12296727', '.', '-', '.', 'gene_id \"chr9.g7605\"; transcript_id \"chr9.g7605.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12294483', '12294635', '.', '-', '.', 'gene_id \"chr9.g7606\"; transcript_id \"chr9.g7606.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12293986', '12294245', '.', '+', '.', 'gene_id \"chr9.g7608\"; transcript_id \"chr9.g7608.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12294787', '12296289', '.', '+', '.', 'gene_id \"chr9.g7609\"; transcript_id \"chr9.g7609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12312847', '12314039', '.', '+', '.', 'gene_id \"chr9.g7614\"; transcript_id \"chr9.g7614.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12314852', '12315280', '.', '+', '.', 'gene_id \"chr9.g7620\"; transcript_id \"chr9.g7620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12315271', '12315680', '.', '+', '.', 'gene_id \"chr9.g7621\"; transcript_id \"chr9.g7621.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12315819', '12316629', '.', '+', '.', 'gene_id \"chr9.g7622\"; transcript_id \"chr9.g7622.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12316633', '12317220', '.', '+', '.', 'gene_id \"chr9.g7623\"; transcript_id \"chr9.g7623.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12317373', '12318741', '.', '+', '.', 'gene_id \"chr9.g7626\"; transcript_id \"chr9.g7626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12319853', '12320134', '.', '-', '.', 'gene_id \"chr9.g7627\"; transcript_id \"chr9.g7627.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12321071', '12321384', '.', '+', '.', 'gene_id \"chr9.g7629\"; transcript_id \"chr9.g7629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12378756', '12379950', '.', '+', '.', 'gene_id \"chr9.g7633\"; transcript_id \"chr9.g7633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12321889', '12322298', '.', '+', '.', 'gene_id \"chr9.g7637\"; transcript_id \"chr9.g7637.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12373683', '12374116', '.', '-', '.', 'gene_id \"chr9.g7638\"; transcript_id \"chr9.g7638.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12336007', '12336337', '.', '+', '.', 'gene_id \"chr9.g7641\"; transcript_id \"chr9.g7641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12333088', '12333312', '.', '-', '.', 'gene_id \"chr9.g7642\"; transcript_id \"chr9.g7642.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12373452', '12373672', '.', '+', '.', 'gene_id \"chr9.g7645\"; transcript_id \"chr9.g7645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12374737', '12374979', '.', '+', '.', 'gene_id \"chr9.g7646\"; transcript_id \"chr9.g7646.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12374985', '12375641', '.', '+', '.', 'gene_id \"chr9.g7647\"; transcript_id \"chr9.g7647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12376994', '12377162', '.', '+', '.', 'gene_id \"chr9.g7648\"; transcript_id \"chr9.g7648.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12377151', '12377316', '.', '-', '.', 'gene_id \"chr9.g7649\"; transcript_id \"chr9.g7649.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12326638', '12326790', '.', '+', '.', 'gene_id \"chr9.g7657\"; transcript_id \"chr9.g7657.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12330720', '12331291', '.', '+', '.', 'gene_id \"chr9.g7658\"; transcript_id \"chr9.g7658.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12331272', '12331489', '.', '-', '.', 'gene_id \"chr9.g7659\"; transcript_id \"chr9.g7659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12331889', '12332402', '.', '+', '.', 'gene_id \"chr9.g7660\"; transcript_id \"chr9.g7660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12332564', '12333076', '.', '+', '.', 'gene_id \"chr9.g7665\"; transcript_id \"chr9.g7665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12333739', '12333967', '.', '+', '.', 'gene_id \"chr9.g7667\"; transcript_id \"chr9.g7667.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12334110', '12334527', '.', '+', '.', 'gene_id \"chr9.g7668\"; transcript_id \"chr9.g7668.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12335036', '12335363', '.', '-', '.', 'gene_id \"chr9.g7669\"; transcript_id \"chr9.g7669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12335358', '12335543', '.', '+', '.', 'gene_id \"chr9.g7670\"; transcript_id \"chr9.g7670.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12338481', '12338815', '.', '+', '.', 'gene_id \"chr9.g7671\"; transcript_id \"chr9.g7671.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12346022', '12346204', '.', '-', '.', 'gene_id \"chr9.g7676\"; transcript_id \"chr9.g7676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12348842', '12349027', '.', '+', '.', 'gene_id \"chr9.g7678\"; transcript_id \"chr9.g7678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12352013', '12352180', '.', '-', '.', 'gene_id \"chr9.g7679\"; transcript_id \"chr9.g7679.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12353418', '12353751', '.', '-', '.', 'gene_id \"chr9.g7681\"; transcript_id \"chr9.g7681.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12397281', '12397414', '.', '-', '.', 'gene_id \"chr9.g7699\"; transcript_id \"chr9.g7699.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12397268', '12397414', '.', '-', '.', 'gene_id \"chr9.g7699\"; transcript_id \"chr9.g7699.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12398513', '12398723', '.', '-', '.', 'gene_id \"chr9.g7704\"; transcript_id \"chr9.g7704.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12398258', '12398522', '.', '+', '.', 'gene_id \"chr9.g7705\"; transcript_id \"chr9.g7705.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12397054', '12397313', '.', '+', '.', 'gene_id \"chr9.g7708\"; transcript_id \"chr9.g7708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12396742', '12396959', '.', '-', '.', 'gene_id \"chr9.g7709\"; transcript_id \"chr9.g7709.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12396503', '12396659', '.', '-', '.', 'gene_id \"chr9.g7710\"; transcript_id \"chr9.g7710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12391763', '12392846', '.', '-', '.', 'gene_id \"chr9.g7714\"; transcript_id \"chr9.g7714.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12401714', '12401968', '.', '+', '.', 'gene_id \"chr9.g7717\"; transcript_id \"chr9.g7717.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12402073', '12403013', '.', '+', '.', 'gene_id \"chr9.g7718\"; transcript_id \"chr9.g7718.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12402073', '12403016', '.', '+', '.', 'gene_id \"chr9.g7718\"; transcript_id \"chr9.g7718.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12403118', '12404146', '.', '-', '.', 'gene_id \"chr9.g7719\"; transcript_id \"chr9.g7719.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12403171', '12404146', '.', '-', '.', 'gene_id \"chr9.g7719\"; transcript_id \"chr9.g7719.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12404132', '12404769', '.', '+', '.', 'gene_id \"chr9.g7720\"; transcript_id \"chr9.g7720.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12404395', '12404769', '.', '-', '.', 'gene_id \"chr9.g7721\"; transcript_id \"chr9.g7721.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12418279', '12418592', '.', '-', '.', 'gene_id \"chr9.g7729\"; transcript_id \"chr9.g7729.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12424099', '12424465', '.', '+', '.', 'gene_id \"chr9.g7733\"; transcript_id \"chr9.g7733.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12467979', '12468244', '.', '-', '.', 'gene_id \"chr9.g7752\"; transcript_id \"chr9.g7752.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12494541', '12494621', '.', '-', '.', 'gene_id \"chr9.g7768\"; transcript_id \"chr9.g7768.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12494541', '12494613', '.', '-', '.', 'gene_id \"chr9.g7768\"; transcript_id \"chr9.g7768.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12495257', '12496553', '.', '+', '.', 'gene_id \"chr9.g7770\"; transcript_id \"chr9.g7770.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12496530', '12496891', '.', '-', '.', 'gene_id \"chr9.g7771\"; transcript_id \"chr9.g7771.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12496929', '12497154', '.', '-', '.', 'gene_id \"chr9.g7772\"; transcript_id \"chr9.g7772.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12497166', '12497356', '.', '-', '.', 'gene_id \"chr9.g7773\"; transcript_id \"chr9.g7773.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12498275', '12498812', '.', '+', '.', 'gene_id \"chr9.g7775\"; transcript_id \"chr9.g7775.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12503417', '12503926', '.', '-', '.', 'gene_id \"chr9.g7779\"; transcript_id \"chr9.g7779.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12504802', '12504989', '.', '-', '.', 'gene_id \"chr9.g7781\"; transcript_id \"chr9.g7781.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12505228', '12505905', '.', '-', '.', 'gene_id \"chr9.g7784\"; transcript_id \"chr9.g7784.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12517975', '12518196', '.', '-', '.', 'gene_id \"chr9.g7791\"; transcript_id \"chr9.g7791.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12518313', '12518541', '.', '-', '.', 'gene_id \"chr9.g7792\"; transcript_id \"chr9.g7792.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12518568', '12519140', '.', '+', '.', 'gene_id \"chr9.g7793\"; transcript_id \"chr9.g7793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12520338', '12520519', '.', '+', '.', 'gene_id \"chr9.g7799\"; transcript_id \"chr9.g7799.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12524334', '12524545', '.', '-', '.', 'gene_id \"chr9.g7804\"; transcript_id \"chr9.g7804.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12570498', '12570724', '.', '+', '.', 'gene_id \"chr9.g7815\"; transcript_id \"chr9.g7815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12548834', '12549007', '.', '-', '.', 'gene_id \"chr9.g7817\"; transcript_id \"chr9.g7817.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12553085', '12553470', '.', '-', '.', 'gene_id \"chr9.g7819\"; transcript_id \"chr9.g7819.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12556912', '12557252', '.', '+', '.', 'gene_id \"chr9.g7823\"; transcript_id \"chr9.g7823.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12557484', '12557924', '.', '+', '.', 'gene_id \"chr9.g7824\"; transcript_id \"chr9.g7824.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12567970', '12568147', '.', '-', '.', 'gene_id \"chr9.g7833\"; transcript_id \"chr9.g7833.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12568427', '12568933', '.', '+', '.', 'gene_id \"chr9.g7835\"; transcript_id \"chr9.g7835.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12569070', '12569325', '.', '+', '.', 'gene_id \"chr9.g7837\"; transcript_id \"chr9.g7837.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12571015', '12571377', '.', '-', '.', 'gene_id \"chr9.g7838\"; transcript_id \"chr9.g7838.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12574484', '12574722', '.', '+', '.', 'gene_id \"chr9.g7843\"; transcript_id \"chr9.g7843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12574806', '12575160', '.', '-', '.', 'gene_id \"chr9.g7844\"; transcript_id \"chr9.g7844.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12576982', '12577148', '.', '-', '.', 'gene_id \"chr9.g7848\"; transcript_id \"chr9.g7848.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12577533', '12577989', '.', '-', '.', 'gene_id \"chr9.g7849\"; transcript_id \"chr9.g7849.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12579100', '12579255', '.', '+', '.', 'gene_id \"chr9.g7853\"; transcript_id \"chr9.g7853.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12579256', '12579658', '.', '+', '.', 'gene_id \"chr9.g7854\"; transcript_id \"chr9.g7854.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12581103', '12581140', '.', '+', '.', 'gene_id \"chr9.g7856\"; transcript_id \"chr9.g7856.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12582628', '12582866', '.', '+', '.', 'gene_id \"chr9.g7859\"; transcript_id \"chr9.g7859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12583482', '12583735', '.', '-', '.', 'gene_id \"chr9.g7861\"; transcript_id \"chr9.g7861.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12584126', '12584375', '.', '+', '.', 'gene_id \"chr9.g7863\"; transcript_id \"chr9.g7863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12594422', '12594589', '.', '+', '.', 'gene_id \"chr9.g7871\"; transcript_id \"chr9.g7871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12595093', '12595249', '.', '-', '.', 'gene_id \"chr9.g7872\"; transcript_id \"chr9.g7872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12597213', '12597427', '.', '-', '.', 'gene_id \"chr9.g7876\"; transcript_id \"chr9.g7876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12602192', '12602362', '.', '-', '.', 'gene_id \"chr9.g7882\"; transcript_id \"chr9.g7882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12608459', '12608667', '.', '-', '.', 'gene_id \"chr9.g7887\"; transcript_id \"chr9.g7887.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12630425', '12631237', '.', '+', '.', 'gene_id \"chr9.g7899\"; transcript_id \"chr9.g7899.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12631785', '12632762', '.', '+', '.', 'gene_id \"chr9.g7899\"; transcript_id \"chr9.g7899.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12628002', '12628414', '.', '+', '.', 'gene_id \"chr9.g7904\"; transcript_id \"chr9.g7904.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12628956', '12629197', '.', '+', '.', 'gene_id \"chr9.g7907\"; transcript_id \"chr9.g7907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12646294', '12646702', '.', '-', '.', 'gene_id \"chr9.g7909\"; transcript_id \"chr9.g7909.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12735227', '12735314', '.', '+', '.', 'gene_id \"chr9.g7910\"; transcript_id \"chr9.g7910.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12639169', '12639471', '.', '+', '.', 'gene_id \"chr9.g7913\"; transcript_id \"chr9.g7913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12734551', '12734982', '.', '-', '.', 'gene_id \"chr9.g7916\"; transcript_id \"chr9.g7916.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12635450', '12636155', '.', '-', '.', 'gene_id \"chr9.g7918\"; transcript_id \"chr9.g7918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12638000', '12638168', '.', '-', '.', 'gene_id \"chr9.g7919\"; transcript_id \"chr9.g7919.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12668754', '12669252', '.', '-', '.', 'gene_id \"chr9.g7920\"; transcript_id \"chr9.g7920.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12654623', '12655208', '.', '+', '.', 'gene_id \"chr9.g7921\"; transcript_id \"chr9.g7921.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12646793', '12647157', '.', '-', '.', 'gene_id \"chr9.g7923\"; transcript_id \"chr9.g7923.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12637648', '12637983', '.', '+', '.', 'gene_id \"chr9.g7929\"; transcript_id \"chr9.g7929.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12653548', '12654483', '.', '-', '.', 'gene_id \"chr9.g7934\"; transcript_id \"chr9.g7934.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12655714', '12655922', '.', '-', '.', 'gene_id \"chr9.g7935\"; transcript_id \"chr9.g7935.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12655940', '12656183', '.', '-', '.', 'gene_id \"chr9.g7936\"; transcript_id \"chr9.g7936.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12657002', '12657166', '.', '-', '.', 'gene_id \"chr9.g7938\"; transcript_id \"chr9.g7938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12657491', '12657661', '.', '+', '.', 'gene_id \"chr9.g7940\"; transcript_id \"chr9.g7940.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12657676', '12657941', '.', '+', '.', 'gene_id \"chr9.g7942\"; transcript_id \"chr9.g7942.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12658476', '12658866', '.', '+', '.', 'gene_id \"chr9.g7943\"; transcript_id \"chr9.g7943.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12659065', '12659304', '.', '+', '.', 'gene_id \"chr9.g7945\"; transcript_id \"chr9.g7945.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12660161', '12660753', '.', '-', '.', 'gene_id \"chr9.g7946\"; transcript_id \"chr9.g7946.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12660623', '12660753', '.', '-', '.', 'gene_id \"chr9.g7946\"; transcript_id \"chr9.g7946.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12659974', '12660162', '.', '-', '.', 'gene_id \"chr9.g7947\"; transcript_id \"chr9.g7947.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12660875', '12661046', '.', '-', '.', 'gene_id \"chr9.g7949\"; transcript_id \"chr9.g7949.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12661301', '12661807', '.', '-', '.', 'gene_id \"chr9.g7950\"; transcript_id \"chr9.g7950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12662966', '12663374', '.', '-', '.', 'gene_id \"chr9.g7953\"; transcript_id \"chr9.g7953.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12663848', '12664112', '.', '-', '.', 'gene_id \"chr9.g7954\"; transcript_id \"chr9.g7954.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12663682', '12663855', '.', '-', '.', 'gene_id \"chr9.g7955\"; transcript_id \"chr9.g7955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12672727', '12672949', '.', '+', '.', 'gene_id \"chr9.g7963\"; transcript_id \"chr9.g7963.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12679217', '12679420', '.', '-', '.', 'gene_id \"chr9.g7970\"; transcript_id \"chr9.g7970.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12701335', '12701495', '.', '-', '.', 'gene_id \"chr9.g7980\"; transcript_id \"chr9.g7980.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '18207610', '18207820', '.', '-', '.', 'gene_id \"chr17.g4892\"; transcript_id \"chr17.g4892.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12705229', '12705538', '.', '-', '.', 'gene_id \"chr9.g7985\"; transcript_id \"chr9.g7985.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12706328', '12706655', '.', '-', '.', 'gene_id \"chr9.g7987\"; transcript_id \"chr9.g7987.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12707101', '12707526', '.', '-', '.', 'gene_id \"chr9.g7989\"; transcript_id \"chr9.g7989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12725444', '12725927', '.', '-', '.', 'gene_id \"chr9.g8000\"; transcript_id \"chr9.g8000.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12768206', '12768487', '.', '-', '.', 'gene_id \"chr9.g8005\"; transcript_id \"chr9.g8005.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12768880', '12769630', '.', '-', '.', 'gene_id \"chr9.g8006\"; transcript_id \"chr9.g8006.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12770218', '12770553', '.', '-', '.', 'gene_id \"chr9.g8008\"; transcript_id \"chr9.g8008.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12775997', '12780656', '.', '-', '.', 'gene_id \"chr9.g8017\"; transcript_id \"chr9.g8017.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12774919', '12776302', '.', '-', '.', 'gene_id \"chr9.g8017\"; transcript_id \"chr9.g8017.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12789506', '12790321', '.', '-', '.', 'gene_id \"chr9.g8017\"; transcript_id \"chr9.g8017.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12797124', '12797536', '.', '-', '.', 'gene_id \"chr9.g8018\"; transcript_id \"chr9.g8018.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12817782', '12818650', '.', '-', '.', 'gene_id \"chr9.g8021\"; transcript_id \"chr9.g8021.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12813003', '12813932', '.', '+', '.', 'gene_id \"chr9.g8030\"; transcript_id \"chr9.g8030.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12815014', '12815285', '.', '-', '.', 'gene_id \"chr9.g8031\"; transcript_id \"chr9.g8031.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12781173', '12782356', '.', '+', '.', 'gene_id \"chr9.g8034\"; transcript_id \"chr9.g8034.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12782446', '12782802', '.', '+', '.', 'gene_id \"chr9.g8035\"; transcript_id \"chr9.g8035.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12784087', '12784915', '.', '-', '.', 'gene_id \"chr9.g8036\"; transcript_id \"chr9.g8036.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12785022', '12785629', '.', '-', '.', 'gene_id \"chr9.g8039\"; transcript_id \"chr9.g8039.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12788405', '12788947', '.', '-', '.', 'gene_id \"chr9.g8043\"; transcript_id \"chr9.g8043.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12788962', '12789385', '.', '+', '.', 'gene_id \"chr9.g8045\"; transcript_id \"chr9.g8045.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12802871', '12803382', '.', '-', '.', 'gene_id \"chr9.g8048\"; transcript_id \"chr9.g8048.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12803433', '12803812', '.', '+', '.', 'gene_id \"chr9.g8049\"; transcript_id \"chr9.g8049.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12803849', '12804573', '.', '-', '.', 'gene_id \"chr9.g8051\"; transcript_id \"chr9.g8051.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12805390', '12805694', '.', '-', '.', 'gene_id \"chr9.g8053\"; transcript_id \"chr9.g8053.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12806890', '12807163', '.', '-', '.', 'gene_id \"chr9.g8055\"; transcript_id \"chr9.g8055.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12807222', '12807427', '.', '+', '.', 'gene_id \"chr9.g8056\"; transcript_id \"chr9.g8056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12808918', '12809394', '.', '-', '.', 'gene_id \"chr9.g8058\"; transcript_id \"chr9.g8058.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12809376', '12809574', '.', '+', '.', 'gene_id \"chr9.g8059\"; transcript_id \"chr9.g8059.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12816661', '12817063', '.', '-', '.', 'gene_id \"chr9.g8065\"; transcript_id \"chr9.g8065.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12817402', '12817599', '.', '-', '.', 'gene_id \"chr9.g8066\"; transcript_id \"chr9.g8066.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12819027', '12819398', '.', '-', '.', 'gene_id \"chr9.g8067\"; transcript_id \"chr9.g8067.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12819772', '12820039', '.', '-', '.', 'gene_id \"chr9.g8070\"; transcript_id \"chr9.g8070.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12820881', '12821180', '.', '-', '.', 'gene_id \"chr9.g8071\"; transcript_id \"chr9.g8071.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12875073', '12875484', '.', '-', '.', 'gene_id \"chr9.g8090\"; transcript_id \"chr9.g8090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12867107', '12868897', '.', '-', '.', 'gene_id \"chr9.g8093\"; transcript_id \"chr9.g8093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12873673', '12873986', '.', '+', '.', 'gene_id \"chr9.g8098\"; transcript_id \"chr9.g8098.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12874543', '12874747', '.', '+', '.', 'gene_id \"chr9.g8099\"; transcript_id \"chr9.g8099.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12877347', '12877510', '.', '+', '.', 'gene_id \"chr9.g8102\"; transcript_id \"chr9.g8102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12878543', '12881204', '.', '-', '.', 'gene_id \"chr9.g8104\"; transcript_id \"chr9.g8104.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12879026', '12881204', '.', '-', '.', 'gene_id \"chr9.g8104\"; transcript_id \"chr9.g8104.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12881828', '12882880', '.', '+', '.', 'gene_id \"chr9.g8106\"; transcript_id \"chr9.g8106.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12884839', '12885076', '.', '-', '.', 'gene_id \"chr9.g8108\"; transcript_id \"chr9.g8108.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12885089', '12885459', '.', '-', '.', 'gene_id \"chr9.g8111\"; transcript_id \"chr9.g8111.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12886198', '12886471', '.', '+', '.', 'gene_id \"chr9.g8112\"; transcript_id \"chr9.g8112.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12886470', '12886594', '.', '-', '.', 'gene_id \"chr9.g8114\"; transcript_id \"chr9.g8114.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12886632', '12886832', '.', '-', '.', 'gene_id \"chr9.g8115\"; transcript_id \"chr9.g8115.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12887806', '12887988', '.', '+', '.', 'gene_id \"chr9.g8118\"; transcript_id \"chr9.g8118.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12895366', '12895623', '.', '+', '.', 'gene_id \"chr9.g8123\"; transcript_id \"chr9.g8123.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12895085', '12895397', '.', '+', '.', 'gene_id \"chr9.g8123\"; transcript_id \"chr9.g8123.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12905005', '12905759', '.', '-', '.', 'gene_id \"chr9.g8124\"; transcript_id \"chr9.g8124.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12892058', '12892224', '.', '+', '.', 'gene_id \"chr9.g8126\"; transcript_id \"chr9.g8126.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12905737', '12907013', '.', '+', '.', 'gene_id \"chr9.g8128\"; transcript_id \"chr9.g8128.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12895783', '12896257', '.', '+', '.', 'gene_id \"chr9.g8133\"; transcript_id \"chr9.g8133.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12896948', '12897166', '.', '-', '.', 'gene_id \"chr9.g8136\"; transcript_id \"chr9.g8136.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12897165', '12897576', '.', '+', '.', 'gene_id \"chr9.g8137\"; transcript_id \"chr9.g8137.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12897582', '12898047', '.', '+', '.', 'gene_id \"chr9.g8139\"; transcript_id \"chr9.g8139.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12899327', '12899663', '.', '-', '.', 'gene_id \"chr9.g8142\"; transcript_id \"chr9.g8142.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12904321', '12904734', '.', '-', '.', 'gene_id \"chr9.g8144\"; transcript_id \"chr9.g8144.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12907789', '12907993', '.', '+', '.', 'gene_id \"chr9.g8148\"; transcript_id \"chr9.g8148.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12913561', '12913727', '.', '-', '.', 'gene_id \"chr9.g8150\"; transcript_id \"chr9.g8150.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12911922', '12912177', '.', '-', '.', 'gene_id \"chr9.g8150\"; transcript_id \"chr9.g8150.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12912245', '12912616', '.', '-', '.', 'gene_id \"chr9.g8150\"; transcript_id \"chr9.g8150.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12913225', '12913540', '.', '-', '.', 'gene_id \"chr9.g8150\"; transcript_id \"chr9.g8150.i15\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12914011', '12914388', '.', '-', '.', 'gene_id \"chr9.g8150\"; transcript_id \"chr9.g8150.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12948347', '12949422', '.', '-', '.', 'gene_id \"chr9.g8151\"; transcript_id \"chr9.g8151.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12948150', '12948369', '.', '-', '.', 'gene_id \"chr9.g8151\"; transcript_id \"chr9.g8151.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12928439', '12929097', '.', '-', '.', 'gene_id \"chr9.g8153\"; transcript_id \"chr9.g8153.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12935383', '12935589', '.', '+', '.', 'gene_id \"chr9.g8155\"; transcript_id \"chr9.g8155.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12929306', '12929785', '.', '+', '.', 'gene_id \"chr9.g8156\"; transcript_id \"chr9.g8156.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12943937', '12944427', '.', '+', '.', 'gene_id \"chr9.g8159\"; transcript_id \"chr9.g8159.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12915471', '12915710', '.', '-', '.', 'gene_id \"chr9.g8161\"; transcript_id \"chr9.g8161.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12935964', '12936595', '.', '-', '.', 'gene_id \"chr9.g8163\"; transcript_id \"chr9.g8163.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12945192', '12945681', '.', '-', '.', 'gene_id \"chr9.g8164\"; transcript_id \"chr9.g8164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12944833', '12945208', '.', '-', '.', 'gene_id \"chr9.g8165\"; transcript_id \"chr9.g8165.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12931978', '12932509', '.', '-', '.', 'gene_id \"chr9.g8166\"; transcript_id \"chr9.g8166.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12919123', '12919559', '.', '-', '.', 'gene_id \"chr9.g8171\"; transcript_id \"chr9.g8171.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12920191', '12920434', '.', '+', '.', 'gene_id \"chr9.g8172\"; transcript_id \"chr9.g8172.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12920191', '12920294', '.', '+', '.', 'gene_id \"chr9.g8172\"; transcript_id \"chr9.g8172.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12924007', '12924259', '.', '-', '.', 'gene_id \"chr9.g8174\"; transcript_id \"chr9.g8174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12925590', '12925738', '.', '+', '.', 'gene_id \"chr9.g8180\"; transcript_id \"chr9.g8180.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12931040', '12931624', '.', '+', '.', 'gene_id \"chr9.g8183\"; transcript_id \"chr9.g8183.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12934240', '12934465', '.', '-', '.', 'gene_id \"chr9.g8184\"; transcript_id \"chr9.g8184.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12933547', '12934244', '.', '+', '.', 'gene_id \"chr9.g8185\"; transcript_id \"chr9.g8185.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12937541', '12937924', '.', '+', '.', 'gene_id \"chr9.g8189\"; transcript_id \"chr9.g8189.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12938257', '12938433', '.', '-', '.', 'gene_id \"chr9.g8190\"; transcript_id \"chr9.g8190.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12938447', '12938850', '.', '+', '.', 'gene_id \"chr9.g8191\"; transcript_id \"chr9.g8191.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12939216', '12939425', '.', '+', '.', 'gene_id \"chr9.g8192\"; transcript_id \"chr9.g8192.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12942549', '12943139', '.', '-', '.', 'gene_id \"chr9.g8198\"; transcript_id \"chr9.g8198.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12942973', '12943139', '.', '-', '.', 'gene_id \"chr9.g8198\"; transcript_id \"chr9.g8198.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12943116', '12943386', '.', '+', '.', 'gene_id \"chr9.g8199\"; transcript_id \"chr9.g8199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12984063', '12985240', '.', '-', '.', 'gene_id \"chr9.g8205\"; transcript_id \"chr9.g8205.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12986129', '12986648', '.', '-', '.', 'gene_id \"chr9.g8207\"; transcript_id \"chr9.g8207.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12985764', '12986144', '.', '+', '.', 'gene_id \"chr9.g8208\"; transcript_id \"chr9.g8208.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12985399', '12985777', '.', '-', '.', 'gene_id \"chr9.g8211\"; transcript_id \"chr9.g8211.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12962362', '12963308', '.', '+', '.', 'gene_id \"chr9.g8212\"; transcript_id \"chr9.g8212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12963315', '12963493', '.', '+', '.', 'gene_id \"chr9.g8213\"; transcript_id \"chr9.g8213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12963505', '12963731', '.', '+', '.', 'gene_id \"chr9.g8214\"; transcript_id \"chr9.g8214.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12964255', '12964633', '.', '-', '.', 'gene_id \"chr9.g8215\"; transcript_id \"chr9.g8215.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12971446', '12971604', '.', '+', '.', 'gene_id \"chr9.g8219\"; transcript_id \"chr9.g8219.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12971626', '12971911', '.', '-', '.', 'gene_id \"chr9.g8220\"; transcript_id \"chr9.g8220.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12978740', '12978937', '.', '-', '.', 'gene_id \"chr9.g8222\"; transcript_id \"chr9.g8222.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12979929', '12980212', '.', '+', '.', 'gene_id \"chr9.g8224\"; transcript_id \"chr9.g8224.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12980537', '12980688', '.', '+', '.', 'gene_id \"chr9.g8225\"; transcript_id \"chr9.g8225.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12981626', '12981789', '.', '-', '.', 'gene_id \"chr9.g8228\"; transcript_id \"chr9.g8228.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12982555', '12982720', '.', '-', '.', 'gene_id \"chr9.g8231\"; transcript_id \"chr9.g8231.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12983670', '12983985', '.', '-', '.', 'gene_id \"chr9.g8233\"; transcript_id \"chr9.g8233.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13008251', '13008347', '.', '-', '.', 'gene_id \"chr9.g8242\"; transcript_id \"chr9.g8242.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13022429', '13022630', '.', '-', '.', 'gene_id \"chr9.g8249\"; transcript_id \"chr9.g8249.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13022688', '13022911', '.', '+', '.', 'gene_id \"chr9.g8251\"; transcript_id \"chr9.g8251.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13030794', '13030988', '.', '-', '.', 'gene_id \"chr9.g8255\"; transcript_id \"chr9.g8255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13035344', '13036133', '.', '-', '.', 'gene_id \"chr9.g8259\"; transcript_id \"chr9.g8259.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13043890', '13044140', '.', '-', '.', 'gene_id \"chr9.g8264\"; transcript_id \"chr9.g8264.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13047696', '13048008', '.', '-', '.', 'gene_id \"chr9.g8267\"; transcript_id \"chr9.g8267.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13063757', '13064379', '.', '+', '.', 'gene_id \"chr9.g8275\"; transcript_id \"chr9.g8275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13063757', '13064000', '.', '+', '.', 'gene_id \"chr9.g8275\"; transcript_id \"chr9.g8275.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13068126', '13068384', '.', '-', '.', 'gene_id \"chr9.g8281\"; transcript_id \"chr9.g8281.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13074826', '13075112', '.', '+', '.', 'gene_id \"chr9.g8288\"; transcript_id \"chr9.g8288.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13157260', '13157674', '.', '-', '.', 'gene_id \"chr9.g8301\"; transcript_id \"chr9.g8301.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13157260', '13157506', '.', '-', '.', 'gene_id \"chr9.g8301\"; transcript_id \"chr9.g8301.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13156969', '13157126', '.', '+', '.', 'gene_id \"chr9.g8302\"; transcript_id \"chr9.g8302.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13156969', '13157136', '.', '+', '.', 'gene_id \"chr9.g8302\"; transcript_id \"chr9.g8302.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13157129', '13157293', '.', '+', '.', 'gene_id \"chr9.g8305\"; transcript_id \"chr9.g8305.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13160693', '13161185', '.', '-', '.', 'gene_id \"chr9.g8310\"; transcript_id \"chr9.g8310.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13161204', '13161409', '.', '+', '.', 'gene_id \"chr9.g8312\"; transcript_id \"chr9.g8312.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13161415', '13161699', '.', '+', '.', 'gene_id \"chr9.g8313\"; transcript_id \"chr9.g8313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13237931', '13238470', '.', '-', '.', 'gene_id \"chr9.g8317\"; transcript_id \"chr9.g8317.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13197930', '13198820', '.', '-', '.', 'gene_id \"chr9.g8317\"; transcript_id \"chr9.g8317.i22\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13292605', '13292693', '.', '-', '.', 'gene_id \"chr9.g8317\"; transcript_id \"chr9.g8317.i24\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13198799', '13198977', '.', '+', '.', 'gene_id \"chr9.g8320\"; transcript_id \"chr9.g8320.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13236918', '13237372', '.', '+', '.', 'gene_id \"chr9.g8323\"; transcript_id \"chr9.g8323.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13210947', '13211147', '.', '+', '.', 'gene_id \"chr9.g8326\"; transcript_id \"chr9.g8326.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13200386', '13200819', '.', '+', '.', 'gene_id \"chr9.g8331\"; transcript_id \"chr9.g8331.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13212560', '13212800', '.', '+', '.', 'gene_id \"chr9.g8335\"; transcript_id \"chr9.g8335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13253827', '13254030', '.', '+', '.', 'gene_id \"chr9.g8336\"; transcript_id \"chr9.g8336.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13236329', '13236818', '.', '-', '.', 'gene_id \"chr9.g8337\"; transcript_id \"chr9.g8337.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13223535', '13223832', '.', '-', '.', 'gene_id \"chr9.g8338\"; transcript_id \"chr9.g8338.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13192007', '13192242', '.', '+', '.', 'gene_id \"chr9.g8339\"; transcript_id \"chr9.g8339.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13202872', '13204269', '.', '-', '.', 'gene_id \"chr9.g8344\"; transcript_id \"chr9.g8344.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13204713', '13204876', '.', '-', '.', 'gene_id \"chr9.g8345\"; transcript_id \"chr9.g8345.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13208186', '13208338', '.', '-', '.', 'gene_id \"chr9.g8349\"; transcript_id \"chr9.g8349.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13212039', '13212375', '.', '+', '.', 'gene_id \"chr9.g8352\"; transcript_id \"chr9.g8352.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13212385', '13212540', '.', '-', '.', 'gene_id \"chr9.g8354\"; transcript_id \"chr9.g8354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13213252', '13213466', '.', '-', '.', 'gene_id \"chr9.g8355\"; transcript_id \"chr9.g8355.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13215016', '13215170', '.', '-', '.', 'gene_id \"chr9.g8358\"; transcript_id \"chr9.g8358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13215505', '13215769', '.', '+', '.', 'gene_id \"chr9.g8359\"; transcript_id \"chr9.g8359.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13218966', '13219171', '.', '+', '.', 'gene_id \"chr9.g8360\"; transcript_id \"chr9.g8360.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13219165', '13219329', '.', '-', '.', 'gene_id \"chr9.g8361\"; transcript_id \"chr9.g8361.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13219352', '13219518', '.', '-', '.', 'gene_id \"chr9.g8362\"; transcript_id \"chr9.g8362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13220712', '13220813', '.', '-', '.', 'gene_id \"chr9.g8364\"; transcript_id \"chr9.g8364.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13220813', '13221208', '.', '-', '.', 'gene_id \"chr9.g8365\"; transcript_id \"chr9.g8365.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13222387', '13222732', '.', '+', '.', 'gene_id \"chr9.g8367\"; transcript_id \"chr9.g8367.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13222878', '13223196', '.', '-', '.', 'gene_id \"chr9.g8368\"; transcript_id \"chr9.g8368.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13225850', '13226014', '.', '+', '.', 'gene_id \"chr9.g8370\"; transcript_id \"chr9.g8370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13228298', '13228664', '.', '+', '.', 'gene_id \"chr9.g8373\"; transcript_id \"chr9.g8373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13228636', '13229347', '.', '-', '.', 'gene_id \"chr9.g8374\"; transcript_id \"chr9.g8374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13230696', '13230857', '.', '+', '.', 'gene_id \"chr9.g8375\"; transcript_id \"chr9.g8375.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13230930', '13231143', '.', '-', '.', 'gene_id \"chr9.g8376\"; transcript_id \"chr9.g8376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13231834', '13232047', '.', '-', '.', 'gene_id \"chr9.g8380\"; transcript_id \"chr9.g8380.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13234113', '13234302', '.', '+', '.', 'gene_id \"chr9.g8384\"; transcript_id \"chr9.g8384.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13239424', '13239785', '.', '-', '.', 'gene_id \"chr9.g8389\"; transcript_id \"chr9.g8389.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13260701', '13260946', '.', '+', '.', 'gene_id \"chr9.g8399\"; transcript_id \"chr9.g8399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13260957', '13261419', '.', '-', '.', 'gene_id \"chr9.g8400\"; transcript_id \"chr9.g8400.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13261405', '13262114', '.', '+', '.', 'gene_id \"chr9.g8401\"; transcript_id \"chr9.g8401.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13262149', '13262402', '.', '+', '.', 'gene_id \"chr9.g8404\"; transcript_id \"chr9.g8404.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13262761', '13262941', '.', '+', '.', 'gene_id \"chr9.g8405\"; transcript_id \"chr9.g8405.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13264452', '13264802', '.', '-', '.', 'gene_id \"chr9.g8410\"; transcript_id \"chr9.g8410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13264922', '13265157', '.', '+', '.', 'gene_id \"chr9.g8411\"; transcript_id \"chr9.g8411.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13265315', '13265555', '.', '+', '.', 'gene_id \"chr9.g8412\"; transcript_id \"chr9.g8412.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13265652', '13265819', '.', '-', '.', 'gene_id \"chr9.g8413\"; transcript_id \"chr9.g8413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13265652', '13265819', '.', '-', '.', 'gene_id \"chr9.g8413\"; transcript_id \"chr9.g8413.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13273958', '13274114', '.', '-', '.', 'gene_id \"chr9.g8419\"; transcript_id \"chr9.g8419.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13274102', '13274248', '.', '-', '.', 'gene_id \"chr9.g8420\"; transcript_id \"chr9.g8420.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13274347', '13274645', '.', '-', '.', 'gene_id \"chr9.g8421\"; transcript_id \"chr9.g8421.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13276492', '13276697', '.', '+', '.', 'gene_id \"chr9.g8422\"; transcript_id \"chr9.g8422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13284029', '13284312', '.', '+', '.', 'gene_id \"chr9.g8430\"; transcript_id \"chr9.g8430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13285096', '13285264', '.', '+', '.', 'gene_id \"chr9.g8431\"; transcript_id \"chr9.g8431.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13286696', '13287271', '.', '+', '.', 'gene_id \"chr9.g8434\"; transcript_id \"chr9.g8434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13291328', '13291540', '.', '-', '.', 'gene_id \"chr9.g8435\"; transcript_id \"chr9.g8435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13303872', '13304059', '.', '+', '.', 'gene_id \"chr9.g8450\"; transcript_id \"chr9.g8450.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13340382', '13340542', '.', '-', '.', 'gene_id \"chr9.g8465\"; transcript_id \"chr9.g8465.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13347617', '13347910', '.', '-', '.', 'gene_id \"chr9.g8468\"; transcript_id \"chr9.g8468.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13347617', '13347686', '.', '-', '.', 'gene_id \"chr9.g8468\"; transcript_id \"chr9.g8468.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13347887', '13348043', '.', '+', '.', 'gene_id \"chr9.g8470\"; transcript_id \"chr9.g8470.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13348526', '13348716', '.', '+', '.', 'gene_id \"chr9.g8472\"; transcript_id \"chr9.g8472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13350855', '13351107', '.', '+', '.', 'gene_id \"chr9.g8477\"; transcript_id \"chr9.g8477.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13353724', '13353879', '.', '+', '.', 'gene_id \"chr9.g8481\"; transcript_id \"chr9.g8481.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13355877', '13356056', '.', '+', '.', 'gene_id \"chr9.g8484\"; transcript_id \"chr9.g8484.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13355877', '13356090', '.', '+', '.', 'gene_id \"chr9.g8484\"; transcript_id \"chr9.g8484.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13356075', '13356284', '.', '+', '.', 'gene_id \"chr9.g8485\"; transcript_id \"chr9.g8485.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13356397', '13356803', '.', '-', '.', 'gene_id \"chr9.g8486\"; transcript_id \"chr9.g8486.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13375282', '13375623', '.', '+', '.', 'gene_id \"chr9.g8491\"; transcript_id \"chr9.g8491.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13375638', '13376212', '.', '-', '.', 'gene_id \"chr9.g8493\"; transcript_id \"chr9.g8493.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13395961', '13396136', '.', '+', '.', 'gene_id \"chr9.g8498\"; transcript_id \"chr9.g8498.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13396621', '13396788', '.', '-', '.', 'gene_id \"chr9.g8500\"; transcript_id \"chr9.g8500.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13401001', '13401085', '.', '-', '.', 'gene_id \"chr9.g8506\"; transcript_id \"chr9.g8506.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13401852', '13402059', '.', '+', '.', 'gene_id \"chr9.g8507\"; transcript_id \"chr9.g8507.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13405035', '13405234', '.', '+', '.', 'gene_id \"chr9.g8509\"; transcript_id \"chr9.g8509.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13423735', '13423900', '.', '+', '.', 'gene_id \"chr9.g8513\"; transcript_id \"chr9.g8513.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13426176', '13426472', '.', '-', '.', 'gene_id \"chr9.g8517\"; transcript_id \"chr9.g8517.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13456684', '13456765', '.', '+', '.', 'gene_id \"chr9.g8519\"; transcript_id \"chr9.g8519.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13456994', '13457258', '.', '+', '.', 'gene_id \"chr9.g8519\"; transcript_id \"chr9.g8519.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13438756', '13439038', '.', '+', '.', 'gene_id \"chr9.g8521\"; transcript_id \"chr9.g8521.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13428128', '13428378', '.', '+', '.', 'gene_id \"chr9.g8522\"; transcript_id \"chr9.g8522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13444720', '13444966', '.', '+', '.', 'gene_id \"chr9.g8525\"; transcript_id \"chr9.g8525.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13439041', '13439250', '.', '-', '.', 'gene_id \"chr9.g8531\"; transcript_id \"chr9.g8531.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13439527', '13439620', '.', '-', '.', 'gene_id \"chr9.g8532\"; transcript_id \"chr9.g8532.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13449258', '13449475', '.', '-', '.', 'gene_id \"chr9.g8536\"; transcript_id \"chr9.g8536.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13449451', '13449616', '.', '-', '.', 'gene_id \"chr9.g8536\"; transcript_id \"chr9.g8536.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13449833', '13450091', '.', '-', '.', 'gene_id \"chr9.g8537\"; transcript_id \"chr9.g8537.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13450193', '13450381', '.', '+', '.', 'gene_id \"chr9.g8538\"; transcript_id \"chr9.g8538.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13482697', '13483309', '.', '+', '.', 'gene_id \"chr9.g8547\"; transcript_id \"chr9.g8547.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13474326', '13478981', '.', '+', '.', 'gene_id \"chr9.g8548\"; transcript_id \"chr9.g8548.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13479674', '13479871', '.', '+', '.', 'gene_id \"chr9.g8552\"; transcript_id \"chr9.g8552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13479222', '13479695', '.', '-', '.', 'gene_id \"chr9.g8553\"; transcript_id \"chr9.g8553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13479034', '13479223', '.', '-', '.', 'gene_id \"chr9.g8554\"; transcript_id \"chr9.g8554.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13480585', '13480771', '.', '-', '.', 'gene_id \"chr9.g8555\"; transcript_id \"chr9.g8555.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13479979', '13480626', '.', '-', '.', 'gene_id \"chr9.g8555\"; transcript_id \"chr9.g8555.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13482130', '13482389', '.', '+', '.', 'gene_id \"chr9.g8561\"; transcript_id \"chr9.g8561.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13486985', '13487167', '.', '-', '.', 'gene_id \"chr9.g8563\"; transcript_id \"chr9.g8563.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13499206', '13499372', '.', '-', '.', 'gene_id \"chr9.g8568\"; transcript_id \"chr9.g8568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13499435', '13499584', '.', '+', '.', 'gene_id \"chr9.g8569\"; transcript_id \"chr9.g8569.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13499583', '13499785', '.', '-', '.', 'gene_id \"chr9.g8570\"; transcript_id \"chr9.g8570.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13499838', '13500094', '.', '+', '.', 'gene_id \"chr9.g8571\"; transcript_id \"chr9.g8571.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13502401', '13502826', '.', '+', '.', 'gene_id \"chr9.g8574\"; transcript_id \"chr9.g8574.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13504290', '13504478', '.', '-', '.', 'gene_id \"chr9.g8577\"; transcript_id \"chr9.g8577.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13510949', '13511271', '.', '-', '.', 'gene_id \"chr9.g8584\"; transcript_id \"chr9.g8584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13511362', '13511758', '.', '-', '.', 'gene_id \"chr9.g8585\"; transcript_id \"chr9.g8585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13513282', '13513449', '.', '-', '.', 'gene_id \"chr9.g8586\"; transcript_id \"chr9.g8586.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13515717', '13516024', '.', '-', '.', 'gene_id \"chr9.g8590\"; transcript_id \"chr9.g8590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13517579', '13517743', '.', '-', '.', 'gene_id \"chr9.g8595\"; transcript_id \"chr9.g8595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13519618', '13520013', '.', '+', '.', 'gene_id \"chr9.g8598\"; transcript_id \"chr9.g8598.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13520341', '13520445', '.', '+', '.', 'gene_id \"chr9.g8599\"; transcript_id \"chr9.g8599.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13522966', '13523250', '.', '+', '.', 'gene_id \"chr9.g8602\"; transcript_id \"chr9.g8602.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13522843', '13522983', '.', '+', '.', 'gene_id \"chr9.g8602\"; transcript_id \"chr9.g8602.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13525869', '13526046', '.', '+', '.', 'gene_id \"chr9.g8607\"; transcript_id \"chr9.g8607.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13528994', '13529149', '.', '+', '.', 'gene_id \"chr9.g8611\"; transcript_id \"chr9.g8611.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13529390', '13529556', '.', '+', '.', 'gene_id \"chr9.g8612\"; transcript_id \"chr9.g8612.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13532339', '13532600', '.', '-', '.', 'gene_id \"chr9.g8617\"; transcript_id \"chr9.g8617.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13537927', '13538209', '.', '+', '.', 'gene_id \"chr9.g8619\"; transcript_id \"chr9.g8619.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13538282', '13538442', '.', '+', '.', 'gene_id \"chr9.g8620\"; transcript_id \"chr9.g8620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13539379', '13539565', '.', '+', '.', 'gene_id \"chr9.g8624\"; transcript_id \"chr9.g8624.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13539895', '13540120', '.', '+', '.', 'gene_id \"chr9.g8626\"; transcript_id \"chr9.g8626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13540847', '13541176', '.', '+', '.', 'gene_id \"chr9.g8629\"; transcript_id \"chr9.g8629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13541720', '13541912', '.', '+', '.', 'gene_id \"chr9.g8630\"; transcript_id \"chr9.g8630.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13541926', '13542197', '.', '-', '.', 'gene_id \"chr9.g8631\"; transcript_id \"chr9.g8631.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13543177', '13543481', '.', '-', '.', 'gene_id \"chr9.g8632\"; transcript_id \"chr9.g8632.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13544939', '13545346', '.', '+', '.', 'gene_id \"chr9.g8635\"; transcript_id \"chr9.g8635.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19036993', '19037158', '.', '+', '.', 'gene_id \"chr17.g5478\"; transcript_id \"chr17.g5478.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13549112', '13549375', '.', '+', '.', 'gene_id \"chr9.g8640\"; transcript_id \"chr9.g8640.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13549363', '13549696', '.', '-', '.', 'gene_id \"chr9.g8641\"; transcript_id \"chr9.g8641.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19037159', '19037354', '.', '-', '.', 'gene_id \"chr17.g5480\"; transcript_id \"chr17.g5480.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13550266', '13550411', '.', '+', '.', 'gene_id \"chr9.g8643\"; transcript_id \"chr9.g8643.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13552097', '13552273', '.', '-', '.', 'gene_id \"chr9.g8645\"; transcript_id \"chr9.g8645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13552799', '13553384', '.', '-', '.', 'gene_id \"chr9.g8647\"; transcript_id \"chr9.g8647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13554291', '13554445', '.', '+', '.', 'gene_id \"chr9.g8650\"; transcript_id \"chr9.g8650.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13555005', '13555167', '.', '-', '.', 'gene_id \"chr9.g8652\"; transcript_id \"chr9.g8652.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13561387', '13561540', '.', '+', '.', 'gene_id \"chr9.g8661\"; transcript_id \"chr9.g8661.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13566925', '13567179', '.', '+', '.', 'gene_id \"chr9.g8663\"; transcript_id \"chr9.g8663.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13567628', '13567808', '.', '-', '.', 'gene_id \"chr9.g8665\"; transcript_id \"chr9.g8665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13570027', '13570285', '.', '-', '.', 'gene_id \"chr9.g8668\"; transcript_id \"chr9.g8668.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13583035', '13583577', '.', '-', '.', 'gene_id \"chr9.g8676\"; transcript_id \"chr9.g8676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13584451', '13584621', '.', '+', '.', 'gene_id \"chr9.g8678\"; transcript_id \"chr9.g8678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13591430', '13591635', '.', '+', '.', 'gene_id \"chr9.g8683\"; transcript_id \"chr9.g8683.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13592274', '13592511', '.', '-', '.', 'gene_id \"chr9.g8684\"; transcript_id \"chr9.g8684.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13592513', '13592876', '.', '+', '.', 'gene_id \"chr9.g8685\"; transcript_id \"chr9.g8685.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13592684', '13592876', '.', '+', '.', 'gene_id \"chr9.g8685\"; transcript_id \"chr9.g8685.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13595815', '13596104', '.', '-', '.', 'gene_id \"chr9.g8688\"; transcript_id \"chr9.g8688.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13596121', '13596428', '.', '-', '.', 'gene_id \"chr9.g8691\"; transcript_id \"chr9.g8691.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13598885', '13599134', '.', '-', '.', 'gene_id \"chr9.g8694\"; transcript_id \"chr9.g8694.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13602064', '13602212', '.', '+', '.', 'gene_id \"chr9.g8697\"; transcript_id \"chr9.g8697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13614126', '13614288', '.', '-', '.', 'gene_id \"chr9.g8704\"; transcript_id \"chr9.g8704.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19102306', '19102479', '.', '-', '.', 'gene_id \"chr17.g5542\"; transcript_id \"chr17.g5542.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13632799', '13632952', '.', '-', '.', 'gene_id \"chr9.g8707\"; transcript_id \"chr9.g8707.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13632991', '13633197', '.', '-', '.', 'gene_id \"chr9.g8708\"; transcript_id \"chr9.g8708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13633997', '13634239', '.', '+', '.', 'gene_id \"chr9.g8710\"; transcript_id \"chr9.g8710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13635152', '13635382', '.', '+', '.', 'gene_id \"chr9.g8711\"; transcript_id \"chr9.g8711.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13652862', '13653018', '.', '-', '.', 'gene_id \"chr9.g8718\"; transcript_id \"chr9.g8718.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13657455', '13657646', '.', '-', '.', 'gene_id \"chr9.g8722\"; transcript_id \"chr9.g8722.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13657795', '13658000', '.', '+', '.', 'gene_id \"chr9.g8723\"; transcript_id \"chr9.g8723.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13658229', '13658483', '.', '-', '.', 'gene_id \"chr9.g8724\"; transcript_id \"chr9.g8724.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13661404', '13661823', '.', '-', '.', 'gene_id \"chr9.g8732\"; transcript_id \"chr9.g8732.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13664101', '13664288', '.', '+', '.', 'gene_id \"chr9.g8736\"; transcript_id \"chr9.g8736.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13669577', '13669876', '.', '-', '.', 'gene_id \"chr9.g8743\"; transcript_id \"chr9.g8743.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13674107', '13674277', '.', '+', '.', 'gene_id \"chr9.g8745\"; transcript_id \"chr9.g8745.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13684641', '13684953', '.', '-', '.', 'gene_id \"chr9.g8751\"; transcript_id \"chr9.g8751.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13685005', '13685361', '.', '+', '.', 'gene_id \"chr9.g8752\"; transcript_id \"chr9.g8752.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13685465', '13685711', '.', '-', '.', 'gene_id \"chr9.g8753\"; transcript_id \"chr9.g8753.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13692833', '13693025', '.', '-', '.', 'gene_id \"chr9.g8757\"; transcript_id \"chr9.g8757.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13693195', '13693368', '.', '+', '.', 'gene_id \"chr9.g8758\"; transcript_id \"chr9.g8758.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13694607', '13694771', '.', '+', '.', 'gene_id \"chr9.g8759\"; transcript_id \"chr9.g8759.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13694751', '13695105', '.', '-', '.', 'gene_id \"chr9.g8760\"; transcript_id \"chr9.g8760.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13695363', '13695553', '.', '-', '.', 'gene_id \"chr9.g8762\"; transcript_id \"chr9.g8762.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13699146', '13699344', '.', '+', '.', 'gene_id \"chr9.g8768\"; transcript_id \"chr9.g8768.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13699378', '13699535', '.', '-', '.', 'gene_id \"chr9.g8769\"; transcript_id \"chr9.g8769.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13703754', '13703931', '.', '+', '.', 'gene_id \"chr9.g8773\"; transcript_id \"chr9.g8773.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13717383', '13717559', '.', '-', '.', 'gene_id \"chr9.g8779\"; transcript_id \"chr9.g8779.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13717867', '13718029', '.', '+', '.', 'gene_id \"chr9.g8781\"; transcript_id \"chr9.g8781.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13718041', '13718275', '.', '+', '.', 'gene_id \"chr9.g8782\"; transcript_id \"chr9.g8782.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13720780', '13721151', '.', '+', '.', 'gene_id \"chr9.g8785\"; transcript_id \"chr9.g8785.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13721154', '13721332', '.', '-', '.', 'gene_id \"chr9.g8786\"; transcript_id \"chr9.g8786.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19216288', '19216502', '.', '-', '.', 'gene_id \"chr17.g5640\"; transcript_id \"chr17.g5640.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13725891', '13726716', '.', '+', '.', 'gene_id \"chr9.g8791\"; transcript_id \"chr9.g8791.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13725891', '13726554', '.', '+', '.', 'gene_id \"chr9.g8791\"; transcript_id \"chr9.g8791.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13731897', '13732817', '.', '+', '.', 'gene_id \"chr9.g8800\"; transcript_id \"chr9.g8800.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13733344', '13733978', '.', '-', '.', 'gene_id \"chr9.g8801\"; transcript_id \"chr9.g8801.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13734643', '13734851', '.', '+', '.', 'gene_id \"chr9.g8802\"; transcript_id \"chr9.g8802.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13735647', '13735879', '.', '-', '.', 'gene_id \"chr9.g8805\"; transcript_id \"chr9.g8805.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13779559', '13779730', '.', '+', '.', 'gene_id \"chr9.g8809\"; transcript_id \"chr9.g8809.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13746671', '13746899', '.', '-', '.', 'gene_id \"chr9.g8811\"; transcript_id \"chr9.g8811.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13778730', '13779549', '.', '-', '.', 'gene_id \"chr9.g8813\"; transcript_id \"chr9.g8813.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13778405', '13778679', '.', '-', '.', 'gene_id \"chr9.g8815\"; transcript_id \"chr9.g8815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13774381', '13774605', '.', '-', '.', 'gene_id \"chr9.g8816\"; transcript_id \"chr9.g8816.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13749550', '13749866', '.', '-', '.', 'gene_id \"chr9.g8820\"; transcript_id \"chr9.g8820.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13750666', '13750944', '.', '-', '.', 'gene_id \"chr9.g8821\"; transcript_id \"chr9.g8821.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13752838', '13753061', '.', '-', '.', 'gene_id \"chr9.g8824\"; transcript_id \"chr9.g8824.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13758566', '13758793', '.', '+', '.', 'gene_id \"chr9.g8829\"; transcript_id \"chr9.g8829.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13758659', '13758793', '.', '+', '.', 'gene_id \"chr9.g8829\"; transcript_id \"chr9.g8829.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13760364', '13760487', '.', '+', '.', 'gene_id \"chr9.g8834\"; transcript_id \"chr9.g8834.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13769542', '13769784', '.', '+', '.', 'gene_id \"chr9.g8840\"; transcript_id \"chr9.g8840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13769833', '13770043', '.', '-', '.', 'gene_id \"chr9.g8841\"; transcript_id \"chr9.g8841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13771240', '13771392', '.', '+', '.', 'gene_id \"chr9.g8842\"; transcript_id \"chr9.g8842.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13772592', '13772931', '.', '-', '.', 'gene_id \"chr9.g8843\"; transcript_id \"chr9.g8843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13773258', '13773466', '.', '-', '.', 'gene_id \"chr9.g8845\"; transcript_id \"chr9.g8845.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13777055', '13777105', '.', '-', '.', 'gene_id \"chr9.g8848\"; transcript_id \"chr9.g8848.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13820571', '13820882', '.', '+', '.', 'gene_id \"chr9.g8859\"; transcript_id \"chr9.g8859.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13857524', '13857714', '.', '+', '.', 'gene_id \"chr9.g8865\"; transcript_id \"chr9.g8865.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13886921', '13887150', '.', '+', '.', 'gene_id \"chr9.g8866\"; transcript_id \"chr9.g8866.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13890234', '13890269', '.', '+', '.', 'gene_id \"chr9.g8866\"; transcript_id \"chr9.g8866.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14023128', '14026058', '.', '+', '.', 'gene_id \"chr9.g8866\"; transcript_id \"chr9.g8866.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14023128', '14026058', '.', '+', '.', 'gene_id \"chr9.g8866\"; transcript_id \"chr9.g8866.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13890633', '13891312', '.', '-', '.', 'gene_id \"chr9.g8868\"; transcript_id \"chr9.g8868.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13915152', '13915455', '.', '+', '.', 'gene_id \"chr9.g8869\"; transcript_id \"chr9.g8869.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14018625', '14018833', '.', '-', '.', 'gene_id \"chr9.g8871\"; transcript_id \"chr9.g8871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13996003', '13996175', '.', '+', '.', 'gene_id \"chr9.g8872\"; transcript_id \"chr9.g8872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13947875', '13948025', '.', '+', '.', 'gene_id \"chr9.g8874\"; transcript_id \"chr9.g8874.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13957247', '13957716', '.', '-', '.', 'gene_id \"chr9.g8875\"; transcript_id \"chr9.g8875.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13959106', '13959502', '.', '+', '.', 'gene_id \"chr9.g8876\"; transcript_id \"chr9.g8876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13883147', '13883413', '.', '-', '.', 'gene_id \"chr9.g8889\"; transcript_id \"chr9.g8889.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13892227', '13892424', '.', '+', '.', 'gene_id \"chr9.g8895\"; transcript_id \"chr9.g8895.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13893778', '13893923', '.', '-', '.', 'gene_id \"chr9.g8896\"; transcript_id \"chr9.g8896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13893941', '13894105', '.', '+', '.', 'gene_id \"chr9.g8897\"; transcript_id \"chr9.g8897.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13899710', '13899979', '.', '+', '.', 'gene_id \"chr9.g8898\"; transcript_id \"chr9.g8898.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13911028', '13911202', '.', '+', '.', 'gene_id \"chr9.g8906\"; transcript_id \"chr9.g8906.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13911591', '13911987', '.', '+', '.', 'gene_id \"chr9.g8908\"; transcript_id \"chr9.g8908.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13913280', '13913502', '.', '+', '.', 'gene_id \"chr9.g8910\"; transcript_id \"chr9.g8910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13913499', '13913669', '.', '-', '.', 'gene_id \"chr9.g8911\"; transcript_id \"chr9.g8911.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13914652', '13915040', '.', '+', '.', 'gene_id \"chr9.g8913\"; transcript_id \"chr9.g8913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13919400', '13919532', '.', '-', '.', 'gene_id \"chr9.g8916\"; transcript_id \"chr9.g8916.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13927603', '13927760', '.', '+', '.', 'gene_id \"chr9.g8921\"; transcript_id \"chr9.g8921.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13941700', '13942027', '.', '-', '.', 'gene_id \"chr9.g8926\"; transcript_id \"chr9.g8926.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13942353', '13942659', '.', '-', '.', 'gene_id \"chr9.g8928\"; transcript_id \"chr9.g8928.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13943609', '13944402', '.', '-', '.', 'gene_id \"chr9.g8931\"; transcript_id \"chr9.g8931.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13948053', '13948322', '.', '+', '.', 'gene_id \"chr9.g8934\"; transcript_id \"chr9.g8934.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13948388', '13948542', '.', '+', '.', 'gene_id \"chr9.g8936\"; transcript_id \"chr9.g8936.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13953547', '13953974', '.', '-', '.', 'gene_id \"chr9.g8940\"; transcript_id \"chr9.g8940.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13954063', '13954289', '.', '+', '.', 'gene_id \"chr9.g8941\"; transcript_id \"chr9.g8941.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13955489', '13955814', '.', '+', '.', 'gene_id \"chr9.g8942\"; transcript_id \"chr9.g8942.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19573571', '19573832', '.', '-', '.', 'gene_id \"chr17.g5795\"; transcript_id \"chr17.g5795.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '19573842', '19573993', '.', '-', '.', 'gene_id \"chr17.g5796\"; transcript_id \"chr17.g5796.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13961994', '13962460', '.', '+', '.', 'gene_id \"chr9.g8946\"; transcript_id \"chr9.g8946.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13962441', '13962609', '.', '+', '.', 'gene_id \"chr9.g8946\"; transcript_id \"chr9.g8946.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13971224', '13971445', '.', '+', '.', 'gene_id \"chr9.g8950\"; transcript_id \"chr9.g8950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13971430', '13971623', '.', '-', '.', 'gene_id \"chr9.g8951\"; transcript_id \"chr9.g8951.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13979150', '13979416', '.', '+', '.', 'gene_id \"chr9.g8955\"; transcript_id \"chr9.g8955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13980050', '13980421', '.', '-', '.', 'gene_id \"chr9.g8958\"; transcript_id \"chr9.g8958.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13980429', '13980630', '.', '-', '.', 'gene_id \"chr9.g8959\"; transcript_id \"chr9.g8959.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13991494', '13991717', '.', '-', '.', 'gene_id \"chr9.g8970\"; transcript_id \"chr9.g8970.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13995430', '13995740', '.', '+', '.', 'gene_id \"chr9.g8973\"; transcript_id \"chr9.g8973.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14017119', '14017286', '.', '-', '.', 'gene_id \"chr9.g8982\"; transcript_id \"chr9.g8982.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14029984', '14030172', '.', '+', '.', 'gene_id \"chr9.g8987\"; transcript_id \"chr9.g8987.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14029984', '14030172', '.', '+', '.', 'gene_id \"chr9.g8987\"; transcript_id \"chr9.g8987.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14034134', '14034673', '.', '+', '.', 'gene_id \"chr9.g8990\"; transcript_id \"chr9.g8990.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14035755', '14035937', '.', '-', '.', 'gene_id \"chr9.g8993\"; transcript_id \"chr9.g8993.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14035919', '14036414', '.', '+', '.', 'gene_id \"chr9.g8994\"; transcript_id \"chr9.g8994.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14037548', '14038276', '.', '+', '.', 'gene_id \"chr9.g8995\"; transcript_id \"chr9.g8995.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14038704', '14039196', '.', '-', '.', 'gene_id \"chr9.g8996\"; transcript_id \"chr9.g8996.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14041460', '14041657', '.', '-', '.', 'gene_id \"chr9.g8999\"; transcript_id \"chr9.g8999.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14042165', '14044233', '.', '-', '.', 'gene_id \"chr9.g9001\"; transcript_id \"chr9.g9001.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14041678', '14042181', '.', '-', '.', 'gene_id \"chr9.g9004\"; transcript_id \"chr9.g9004.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14044212', '14044490', '.', '-', '.', 'gene_id \"chr9.g9005\"; transcript_id \"chr9.g9005.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14045146', '14045555', '.', '-', '.', 'gene_id \"chr9.g9008\"; transcript_id \"chr9.g9008.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14045618', '14045791', '.', '-', '.', 'gene_id \"chr9.g9009\"; transcript_id \"chr9.g9009.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14045776', '14045877', '.', '-', '.', 'gene_id \"chr9.g9009\"; transcript_id \"chr9.g9009.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14045776', '14045914', '.', '-', '.', 'gene_id \"chr9.g9009\"; transcript_id \"chr9.g9009.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14045905', '14046193', '.', '-', '.', 'gene_id \"chr9.g9011\"; transcript_id \"chr9.g9011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14047542', '14047701', '.', '-', '.', 'gene_id \"chr9.g9013\"; transcript_id \"chr9.g9013.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14047983', '14048139', '.', '-', '.', 'gene_id \"chr9.g9014\"; transcript_id \"chr9.g9014.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14048458', '14048895', '.', '+', '.', 'gene_id \"chr9.g9015\"; transcript_id \"chr9.g9015.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14120077', '14120334', '.', '+', '.', 'gene_id \"chr9.g9018\"; transcript_id \"chr9.g9018.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14083067', '14083221', '.', '-', '.', 'gene_id \"chr9.g9024\"; transcript_id \"chr9.g9024.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14089968', '14090129', '.', '+', '.', 'gene_id \"chr9.g9027\"; transcript_id \"chr9.g9027.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14091006', '14091494', '.', '+', '.', 'gene_id \"chr9.g9028\"; transcript_id \"chr9.g9028.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14091488', '14091995', '.', '-', '.', 'gene_id \"chr9.g9029\"; transcript_id \"chr9.g9029.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14092212', '14092587', '.', '+', '.', 'gene_id \"chr9.g9030\"; transcript_id \"chr9.g9030.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14186161', '14187395', '.', '-', '.', 'gene_id \"chr9.g9040\"; transcript_id \"chr9.g9040.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14277370', '14277539', '.', '+', '.', 'gene_id \"chr9.g9058\"; transcript_id \"chr9.g9058.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14268350', '14268592', '.', '+', '.', 'gene_id \"chr9.g9059\"; transcript_id \"chr9.g9059.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14321238', '14326957', '.', '-', '.', 'gene_id \"chr9.g9060\"; transcript_id \"chr9.g9060.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14254523', '14254813', '.', '+', '.', 'gene_id \"chr9.g9064\"; transcript_id \"chr9.g9064.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14255629', '14256908', '.', '-', '.', 'gene_id \"chr9.g9067\"; transcript_id \"chr9.g9067.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14257348', '14257743', '.', '-', '.', 'gene_id \"chr9.g9069\"; transcript_id \"chr9.g9069.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14257754', '14257954', '.', '-', '.', 'gene_id \"chr9.g9070\"; transcript_id \"chr9.g9070.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14257960', '14258264', '.', '-', '.', 'gene_id \"chr9.g9071\"; transcript_id \"chr9.g9071.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14258586', '14258781', '.', '+', '.', 'gene_id \"chr9.g9073\"; transcript_id \"chr9.g9073.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14258767', '14259101', '.', '-', '.', 'gene_id \"chr9.g9074\"; transcript_id \"chr9.g9074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14259103', '14259247', '.', '+', '.', 'gene_id \"chr9.g9075\"; transcript_id \"chr9.g9075.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14259224', '14259378', '.', '+', '.', 'gene_id \"chr9.g9075\"; transcript_id \"chr9.g9075.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14260514', '14260750', '.', '-', '.', 'gene_id \"chr9.g9078\"; transcript_id \"chr9.g9078.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14261140', '14261319', '.', '-', '.', 'gene_id \"chr9.g9079\"; transcript_id \"chr9.g9079.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14261448', '14261688', '.', '+', '.', 'gene_id \"chr9.g9081\"; transcript_id \"chr9.g9081.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14261718', '14262095', '.', '-', '.', 'gene_id \"chr9.g9082\"; transcript_id \"chr9.g9082.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14262392', '14262703', '.', '+', '.', 'gene_id \"chr9.g9083\"; transcript_id \"chr9.g9083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14264605', '14264847', '.', '+', '.', 'gene_id \"chr9.g9084\"; transcript_id \"chr9.g9084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14264866', '14265161', '.', '-', '.', 'gene_id \"chr9.g9085\"; transcript_id \"chr9.g9085.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14265457', '14265786', '.', '-', '.', 'gene_id \"chr9.g9086\"; transcript_id \"chr9.g9086.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14269499', '14269814', '.', '+', '.', 'gene_id \"chr9.g9089\"; transcript_id \"chr9.g9089.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14269853', '14270108', '.', '-', '.', 'gene_id \"chr9.g9090\"; transcript_id \"chr9.g9090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14270371', '14270541', '.', '-', '.', 'gene_id \"chr9.g9091\"; transcript_id \"chr9.g9091.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14271949', '14272440', '.', '+', '.', 'gene_id \"chr9.g9092\"; transcript_id \"chr9.g9092.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14272705', '14272918', '.', '+', '.', 'gene_id \"chr9.g9094\"; transcript_id \"chr9.g9094.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14273291', '14273479', '.', '-', '.', 'gene_id \"chr9.g9096\"; transcript_id \"chr9.g9096.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14276740', '14276911', '.', '+', '.', 'gene_id \"chr9.g9102\"; transcript_id \"chr9.g9102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14279361', '14279637', '.', '+', '.', 'gene_id \"chr9.g9104\"; transcript_id \"chr9.g9104.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14286834', '14286984', '.', '+', '.', 'gene_id \"chr9.g9110\"; transcript_id \"chr9.g9110.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14293338', '14293836', '.', '+', '.', 'gene_id \"chr9.g9116\"; transcript_id \"chr9.g9116.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14347933', '14348127', '.', '-', '.', 'gene_id \"chr9.g9130\"; transcript_id \"chr9.g9130.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14362363', '14362515', '.', '+', '.', 'gene_id \"chr9.g9140\"; transcript_id \"chr9.g9140.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14363535', '14363712', '.', '+', '.', 'gene_id \"chr9.g9142\"; transcript_id \"chr9.g9142.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14367249', '14367451', '.', '-', '.', 'gene_id \"chr9.g9148\"; transcript_id \"chr9.g9148.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14384190', '14384345', '.', '-', '.', 'gene_id \"chr9.g9151\"; transcript_id \"chr9.g9151.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14384743', '14384890', '.', '+', '.', 'gene_id \"chr9.g9153\"; transcript_id \"chr9.g9153.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14387714', '14387951', '.', '+', '.', 'gene_id \"chr9.g9157\"; transcript_id \"chr9.g9157.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14388030', '14388218', '.', '+', '.', 'gene_id \"chr9.g9158\"; transcript_id \"chr9.g9158.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14395917', '14396079', '.', '+', '.', 'gene_id \"chr9.g9164\"; transcript_id \"chr9.g9164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14396235', '14397188', '.', '-', '.', 'gene_id \"chr9.g9165\"; transcript_id \"chr9.g9165.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14401739', '14402259', '.', '+', '.', 'gene_id \"chr9.g9168\"; transcript_id \"chr9.g9168.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14402272', '14402862', '.', '-', '.', 'gene_id \"chr9.g9171\"; transcript_id \"chr9.g9171.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14403266', '14403621', '.', '-', '.', 'gene_id \"chr9.g9172\"; transcript_id \"chr9.g9172.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14406857', '14407246', '.', '-', '.', 'gene_id \"chr9.g9178\"; transcript_id \"chr9.g9178.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14409334', '14409485', '.', '+', '.', 'gene_id \"chr9.g9181\"; transcript_id \"chr9.g9181.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14413019', '14413197', '.', '-', '.', 'gene_id \"chr9.g9182\"; transcript_id \"chr9.g9182.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14413320', '14413492', '.', '+', '.', 'gene_id \"chr9.g9184\"; transcript_id \"chr9.g9184.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14421952', '14422105', '.', '+', '.', 'gene_id \"chr9.g9187\"; transcript_id \"chr9.g9187.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14428635', '14428931', '.', '+', '.', 'gene_id \"chr9.g9192\"; transcript_id \"chr9.g9192.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14430700', '14431086', '.', '-', '.', 'gene_id \"chr9.g9195\"; transcript_id \"chr9.g9195.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14432416', '14432656', '.', '+', '.', 'gene_id \"chr9.g9197\"; transcript_id \"chr9.g9197.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14434533', '14434780', '.', '-', '.', 'gene_id \"chr9.g9199\"; transcript_id \"chr9.g9199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14441916', '14442081', '.', '-', '.', 'gene_id \"chr9.g9207\"; transcript_id \"chr9.g9207.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14443943', '14444464', '.', '-', '.', 'gene_id \"chr9.g9210\"; transcript_id \"chr9.g9210.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14444912', '14446147', '.', '-', '.', 'gene_id \"chr9.g9211\"; transcript_id \"chr9.g9211.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14446127', '14446363', '.', '+', '.', 'gene_id \"chr9.g9212\"; transcript_id \"chr9.g9212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14446892', '14447069', '.', '-', '.', 'gene_id \"chr9.g9214\"; transcript_id \"chr9.g9214.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14447771', '14447937', '.', '-', '.', 'gene_id \"chr9.g9216\"; transcript_id \"chr9.g9216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14450232', '14450614', '.', '+', '.', 'gene_id \"chr9.g9218\"; transcript_id \"chr9.g9218.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14451160', '14451345', '.', '+', '.', 'gene_id \"chr9.g9220\"; transcript_id \"chr9.g9220.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14451333', '14451867', '.', '+', '.', 'gene_id \"chr9.g9221\"; transcript_id \"chr9.g9221.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14453353', '14453760', '.', '+', '.', 'gene_id \"chr9.g9225\"; transcript_id \"chr9.g9225.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14453775', '14454094', '.', '-', '.', 'gene_id \"chr9.g9227\"; transcript_id \"chr9.g9227.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14455064', '14455250', '.', '+', '.', 'gene_id \"chr9.g9228\"; transcript_id \"chr9.g9228.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14456538', '14456713', '.', '-', '.', 'gene_id \"chr9.g9230\"; transcript_id \"chr9.g9230.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14457067', '14457222', '.', '-', '.', 'gene_id \"chr9.g9232\"; transcript_id \"chr9.g9232.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14463906', '14464087', '.', '+', '.', 'gene_id \"chr9.g9240\"; transcript_id \"chr9.g9240.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14466213', '14466501', '.', '+', '.', 'gene_id \"chr9.g9241\"; transcript_id \"chr9.g9241.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14469680', '14470046', '.', '+', '.', 'gene_id \"chr9.g9244\"; transcript_id \"chr9.g9244.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14471038', '14471192', '.', '-', '.', 'gene_id \"chr9.g9245\"; transcript_id \"chr9.g9245.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14477503', '14477711', '.', '+', '.', 'gene_id \"chr9.g9253\"; transcript_id \"chr9.g9253.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14478933', '14479108', '.', '+', '.', 'gene_id \"chr9.g9257\"; transcript_id \"chr9.g9257.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14482372', '14482549', '.', '-', '.', 'gene_id \"chr9.g9262\"; transcript_id \"chr9.g9262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14484091', '14484254', '.', '+', '.', 'gene_id \"chr9.g9266\"; transcript_id \"chr9.g9266.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14485780', '14486226', '.', '+', '.', 'gene_id \"chr9.g9268\"; transcript_id \"chr9.g9268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14496288', '14496561', '.', '+', '.', 'gene_id \"chr9.g9272\"; transcript_id \"chr9.g9272.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14499246', '14499591', '.', '+', '.', 'gene_id \"chr9.g9276\"; transcript_id \"chr9.g9276.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14499571', '14499722', '.', '-', '.', 'gene_id \"chr9.g9277\"; transcript_id \"chr9.g9277.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14500334', '14500551', '.', '-', '.', 'gene_id \"chr9.g9279\"; transcript_id \"chr9.g9279.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14518049', '14518306', '.', '+', '.', 'gene_id \"chr9.g9290\"; transcript_id \"chr9.g9290.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14526145', '14526304', '.', '-', '.', 'gene_id \"chr9.g9298\"; transcript_id \"chr9.g9298.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14526285', '14526474', '.', '+', '.', 'gene_id \"chr9.g9299\"; transcript_id \"chr9.g9299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14534669', '14534822', '.', '-', '.', 'gene_id \"chr9.g9303\"; transcript_id \"chr9.g9303.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14539475', '14539899', '.', '-', '.', 'gene_id \"chr9.g9309\"; transcript_id \"chr9.g9309.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14542931', '14543436', '.', '+', '.', 'gene_id \"chr9.g9313\"; transcript_id \"chr9.g9313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14543539', '14544020', '.', '-', '.', 'gene_id \"chr9.g9315\"; transcript_id \"chr9.g9315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14544598', '14544775', '.', '-', '.', 'gene_id \"chr9.g9317\"; transcript_id \"chr9.g9317.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14544839', '14545062', '.', '+', '.', 'gene_id \"chr9.g9318\"; transcript_id \"chr9.g9318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14545077', '14545348', '.', '-', '.', 'gene_id \"chr9.g9319\"; transcript_id \"chr9.g9319.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14546624', '14546875', '.', '-', '.', 'gene_id \"chr9.g9320\"; transcript_id \"chr9.g9320.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14547735', '14547917', '.', '-', '.', 'gene_id \"chr9.g9322\"; transcript_id \"chr9.g9322.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14549077', '14549252', '.', '-', '.', 'gene_id \"chr9.g9324\"; transcript_id \"chr9.g9324.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14549452', '14549635', '.', '+', '.', 'gene_id \"chr9.g9325\"; transcript_id \"chr9.g9325.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14550820', '14551035', '.', '-', '.', 'gene_id \"chr9.g9327\"; transcript_id \"chr9.g9327.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14551578', '14552424', '.', '-', '.', 'gene_id \"chr9.g9328\"; transcript_id \"chr9.g9328.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14552411', '14552606', '.', '+', '.', 'gene_id \"chr9.g9329\"; transcript_id \"chr9.g9329.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14552609', '14552931', '.', '-', '.', 'gene_id \"chr9.g9330\"; transcript_id \"chr9.g9330.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14556184', '14556383', '.', '-', '.', 'gene_id \"chr9.g9334\"; transcript_id \"chr9.g9334.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14557072', '14557228', '.', '+', '.', 'gene_id \"chr9.g9335\"; transcript_id \"chr9.g9335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14558415', '14558570', '.', '-', '.', 'gene_id \"chr9.g9339\"; transcript_id \"chr9.g9339.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '20207239', '20207736', '.', '-', '.', 'gene_id \"chr17.g6206\"; transcript_id \"chr17.g6206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14561488', '14561772', '.', '-', '.', 'gene_id \"chr9.g9343\"; transcript_id \"chr9.g9343.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14563333', '14563559', '.', '+', '.', 'gene_id \"chr9.g9346\"; transcript_id \"chr9.g9346.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14563112', '14563351', '.', '-', '.', 'gene_id \"chr9.g9347\"; transcript_id \"chr9.g9347.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14564009', '14564494', '.', '-', '.', 'gene_id \"chr9.g9348\"; transcript_id \"chr9.g9348.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14566554', '14567504', '.', '+', '.', 'gene_id \"chr9.g9353\"; transcript_id \"chr9.g9353.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14569302', '14569475', '.', '+', '.', 'gene_id \"chr9.g9355\"; transcript_id \"chr9.g9355.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14569852', '14570035', '.', '+', '.', 'gene_id \"chr9.g9356\"; transcript_id \"chr9.g9356.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14575785', '14575973', '.', '-', '.', 'gene_id \"chr9.g9367\"; transcript_id \"chr9.g9367.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14576500', '14576728', '.', '+', '.', 'gene_id \"chr9.g9368\"; transcript_id \"chr9.g9368.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14578820', '14579129', '.', '+', '.', 'gene_id \"chr9.g9375\"; transcript_id \"chr9.g9375.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14579180', '14579424', '.', '+', '.', 'gene_id \"chr9.g9376\"; transcript_id \"chr9.g9376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14580487', '14580684', '.', '-', '.', 'gene_id \"chr9.g9378\"; transcript_id \"chr9.g9378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14580663', '14581293', '.', '+', '.', 'gene_id \"chr9.g9379\"; transcript_id \"chr9.g9379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14581302', '14581451', '.', '-', '.', 'gene_id \"chr9.g9380\"; transcript_id \"chr9.g9380.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14581302', '14581790', '.', '-', '.', 'gene_id \"chr9.g9380\"; transcript_id \"chr9.g9380.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14583401', '14584344', '.', '-', '.', 'gene_id \"chr9.g9386\"; transcript_id \"chr9.g9386.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14585107', '14585351', '.', '-', '.', 'gene_id \"chr9.g9388\"; transcript_id \"chr9.g9388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14585741', '14585971', '.', '+', '.', 'gene_id \"chr9.g9389\"; transcript_id \"chr9.g9389.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14586605', '14586710', '.', '+', '.', 'gene_id \"chr9.g9391\"; transcript_id \"chr9.g9391.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14587139', '14587568', '.', '-', '.', 'gene_id \"chr9.g9394\"; transcript_id \"chr9.g9394.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14587657', '14588002', '.', '-', '.', 'gene_id \"chr9.g9395\"; transcript_id \"chr9.g9395.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14588994', '14589361', '.', '-', '.', 'gene_id \"chr9.g9399\"; transcript_id \"chr9.g9399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14589323', '14589514', '.', '+', '.', 'gene_id \"chr9.g9400\"; transcript_id \"chr9.g9400.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14589502', '14589719', '.', '-', '.', 'gene_id \"chr9.g9401\"; transcript_id \"chr9.g9401.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14589993', '14590473', '.', '+', '.', 'gene_id \"chr9.g9402\"; transcript_id \"chr9.g9402.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14591246', '14591795', '.', '-', '.', 'gene_id \"chr9.g9405\"; transcript_id \"chr9.g9405.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14591956', '14592402', '.', '-', '.', 'gene_id \"chr9.g9407\"; transcript_id \"chr9.g9407.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14593343', '14593953', '.', '+', '.', 'gene_id \"chr9.g9408\"; transcript_id \"chr9.g9408.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14595053', '14595415', '.', '-', '.', 'gene_id \"chr9.g9411\"; transcript_id \"chr9.g9411.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14596711', '14596993', '.', '+', '.', 'gene_id \"chr9.g9413\"; transcript_id \"chr9.g9413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14596994', '14597327', '.', '-', '.', 'gene_id \"chr9.g9415\"; transcript_id \"chr9.g9415.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14600011', '14600231', '.', '-', '.', 'gene_id \"chr9.g9420\"; transcript_id \"chr9.g9420.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14601007', '14601420', '.', '+', '.', 'gene_id \"chr9.g9421\"; transcript_id \"chr9.g9421.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14601781', '14602022', '.', '-', '.', 'gene_id \"chr9.g9423\"; transcript_id \"chr9.g9423.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14602393', '14603575', '.', '-', '.', 'gene_id \"chr9.g9424\"; transcript_id \"chr9.g9424.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14603970', '14604202', '.', '+', '.', 'gene_id \"chr9.g9425\"; transcript_id \"chr9.g9425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14604213', '14604727', '.', '+', '.', 'gene_id \"chr9.g9426\"; transcript_id \"chr9.g9426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14607298', '14607683', '.', '-', '.', 'gene_id \"chr9.g9428\"; transcript_id \"chr9.g9428.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14611942', '14612410', '.', '+', '.', 'gene_id \"chr9.g9437\"; transcript_id \"chr9.g9437.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14614817', '14615347', '.', '-', '.', 'gene_id \"chr9.g9439\"; transcript_id \"chr9.g9439.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14615529', '14615935', '.', '+', '.', 'gene_id \"chr9.g9440\"; transcript_id \"chr9.g9440.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '14615955', '14616129', '.', '+', '.', 'gene_id \"chr9.g9441\"; transcript_id \"chr9.g9441.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14616473', '14616695', '.', '-', '.', 'gene_id \"chr9.g9443\"; transcript_id \"chr9.g9443.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14617698', '14618139', '.', '-', '.', 'gene_id \"chr9.g9445\"; transcript_id \"chr9.g9445.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14618350', '14618686', '.', '-', '.', 'gene_id \"chr9.g9446\"; transcript_id \"chr9.g9446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14619140', '14619314', '.', '-', '.', 'gene_id \"chr9.g9447\"; transcript_id \"chr9.g9447.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14619721', '14619871', '.', '+', '.', 'gene_id \"chr9.g9449\"; transcript_id \"chr9.g9449.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14623458', '14623862', '.', '+', '.', 'gene_id \"chr9.g9454\"; transcript_id \"chr9.g9454.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14623834', '14624013', '.', '-', '.', 'gene_id \"chr9.g9455\"; transcript_id \"chr9.g9455.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14624092', '14624731', '.', '-', '.', 'gene_id \"chr9.g9456\"; transcript_id \"chr9.g9456.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14624711', '14624887', '.', '+', '.', 'gene_id \"chr9.g9457\"; transcript_id \"chr9.g9457.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14628000', '14628392', '.', '+', '.', 'gene_id \"chr9.g9465\"; transcript_id \"chr9.g9465.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14628434', '14628619', '.', '+', '.', 'gene_id \"chr9.g9466\"; transcript_id \"chr9.g9466.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14628612', '14629015', '.', '-', '.', 'gene_id \"chr9.g9467\"; transcript_id \"chr9.g9467.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14629687', '14630140', '.', '-', '.', 'gene_id \"chr9.g9470\"; transcript_id \"chr9.g9470.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14632185', '14632351', '.', '-', '.', 'gene_id \"chr9.g9475\"; transcript_id \"chr9.g9475.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14633887', '14634714', '.', '-', '.', 'gene_id \"chr9.g9479\"; transcript_id \"chr9.g9479.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14636689', '14636917', '.', '+', '.', 'gene_id \"chr9.g9480\"; transcript_id \"chr9.g9480.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14636906', '14637082', '.', '-', '.', 'gene_id \"chr9.g9482\"; transcript_id \"chr9.g9482.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14636079', '14636538', '.', '-', '.', 'gene_id \"chr9.g9483\"; transcript_id \"chr9.g9483.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14697386', '14698041', '.', '+', '.', 'gene_id \"chr9.g9494\"; transcript_id \"chr9.g9494.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14709048', '14710632', '.', '-', '.', 'gene_id \"chr9.g9495\"; transcript_id \"chr9.g9495.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14691792', '14692096', '.', '-', '.', 'gene_id \"chr9.g9497\"; transcript_id \"chr9.g9497.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14696793', '14697243', '.', '-', '.', 'gene_id \"chr9.g9500\"; transcript_id \"chr9.g9500.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14708806', '14709065', '.', '+', '.', 'gene_id \"chr9.g9501\"; transcript_id \"chr9.g9501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14696383', '14696757', '.', '+', '.', 'gene_id \"chr9.g9507\"; transcript_id \"chr9.g9507.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14714968', '14715178', '.', '+', '.', 'gene_id \"chr9.g9511\"; transcript_id \"chr9.g9511.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14715173', '14715333', '.', '-', '.', 'gene_id \"chr9.g9512\"; transcript_id \"chr9.g9512.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14888811', '14888917', '.', '-', '.', 'gene_id \"chr9.g9514\"; transcript_id \"chr9.g9514.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14730304', '14732250', '.', '-', '.', 'gene_id \"chr9.g9516\"; transcript_id \"chr9.g9516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14716877', '14717356', '.', '+', '.', 'gene_id \"chr9.g9517\"; transcript_id \"chr9.g9517.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14717979', '14718275', '.', '-', '.', 'gene_id \"chr9.g9519\"; transcript_id \"chr9.g9519.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14718255', '14718639', '.', '+', '.', 'gene_id \"chr9.g9520\"; transcript_id \"chr9.g9520.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14718941', '14719120', '.', '-', '.', 'gene_id \"chr9.g9521\"; transcript_id \"chr9.g9521.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14721551', '14721747', '.', '-', '.', 'gene_id \"chr9.g9525\"; transcript_id \"chr9.g9525.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14721783', '14722065', '.', '+', '.', 'gene_id \"chr9.g9526\"; transcript_id \"chr9.g9526.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14726752', '14727111', '.', '-', '.', 'gene_id \"chr9.g9530\"; transcript_id \"chr9.g9530.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14727378', '14727553', '.', '+', '.', 'gene_id \"chr9.g9531\"; transcript_id \"chr9.g9531.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14727691', '14727910', '.', '+', '.', 'gene_id \"chr9.g9532\"; transcript_id \"chr9.g9532.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14729339', '14729520', '.', '-', '.', 'gene_id \"chr9.g9534\"; transcript_id \"chr9.g9534.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14730067', '14730301', '.', '+', '.', 'gene_id \"chr9.g9535\"; transcript_id \"chr9.g9535.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14743725', '14743915', '.', '+', '.', 'gene_id \"chr9.g9546\"; transcript_id \"chr9.g9546.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14743725', '14743903', '.', '+', '.', 'gene_id \"chr9.g9546\"; transcript_id \"chr9.g9546.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14750337', '14750769', '.', '+', '.', 'gene_id \"chr9.g9552\"; transcript_id \"chr9.g9552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14775824', '14775976', '.', '+', '.', 'gene_id \"chr9.g9554\"; transcript_id \"chr9.g9554.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14781047', '14781302', '.', '-', '.', 'gene_id \"chr9.g9556\"; transcript_id \"chr9.g9556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14839720', '14839770', '.', '+', '.', 'gene_id \"chr9.g9563\"; transcript_id \"chr9.g9563.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14839723', '14839774', '.', '+', '.', 'gene_id \"chr9.g9563\"; transcript_id \"chr9.g9563.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14844898', '14845052', '.', '-', '.', 'gene_id \"chr9.g9567\"; transcript_id \"chr9.g9567.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14844568', '14844916', '.', '-', '.', 'gene_id \"chr9.g9567\"; transcript_id \"chr9.g9567.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14846599', '14846753', '.', '+', '.', 'gene_id \"chr9.g9570\"; transcript_id \"chr9.g9570.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14847408', '14847612', '.', '+', '.', 'gene_id \"chr9.g9571\"; transcript_id \"chr9.g9571.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14877872', '14878069', '.', '+', '.', 'gene_id \"chr9.g9578\"; transcript_id \"chr9.g9578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14878114', '14878287', '.', '-', '.', 'gene_id \"chr9.g9579\"; transcript_id \"chr9.g9579.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14878550', '14878653', '.', '+', '.', 'gene_id \"chr9.g9580\"; transcript_id \"chr9.g9580.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14881216', '14881401', '.', '-', '.', 'gene_id \"chr9.g9583\"; transcript_id \"chr9.g9583.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14883388', '14883593', '.', '+', '.', 'gene_id \"chr9.g9585\"; transcript_id \"chr9.g9585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14883598', '14883845', '.', '+', '.', 'gene_id \"chr9.g9587\"; transcript_id \"chr9.g9587.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15072180', '15073195', '.', '-', '.', 'gene_id \"chr9.g9592\"; transcript_id \"chr9.g9592.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14908682', '14908865', '.', '-', '.', 'gene_id \"chr9.g9594\"; transcript_id \"chr9.g9594.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14976949', '14977305', '.', '-', '.', 'gene_id \"chr9.g9595\"; transcript_id \"chr9.g9595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15030660', '15030809', '.', '-', '.', 'gene_id \"chr9.g9596\"; transcript_id \"chr9.g9596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14922081', '14922245', '.', '+', '.', 'gene_id \"chr9.g9597\"; transcript_id \"chr9.g9597.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14955035', '14955221', '.', '-', '.', 'gene_id \"chr9.g9602\"; transcript_id \"chr9.g9602.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14956083', '14956114', '.', '-', '.', 'gene_id \"chr9.g9603\"; transcript_id \"chr9.g9603.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14962066', '14962216', '.', '+', '.', 'gene_id \"chr9.g9610\"; transcript_id \"chr9.g9610.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14968367', '14968746', '.', '-', '.', 'gene_id \"chr9.g9614\"; transcript_id \"chr9.g9614.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14969576', '14969738', '.', '+', '.', 'gene_id \"chr9.g9615\"; transcript_id \"chr9.g9615.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14970932', '14971142', '.', '+', '.', 'gene_id \"chr9.g9616\"; transcript_id \"chr9.g9616.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14986893', '14987049', '.', '-', '.', 'gene_id \"chr9.g9626\"; transcript_id \"chr9.g9626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14991055', '14991209', '.', '-', '.', 'gene_id \"chr9.g9629\"; transcript_id \"chr9.g9629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14994746', '14995022', '.', '-', '.', 'gene_id \"chr9.g9631\"; transcript_id \"chr9.g9631.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15001908', '15002173', '.', '+', '.', 'gene_id \"chr9.g9633\"; transcript_id \"chr9.g9633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15002298', '15002458', '.', '+', '.', 'gene_id \"chr9.g9634\"; transcript_id \"chr9.g9634.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15022061', '15022342', '.', '-', '.', 'gene_id \"chr9.g9639\"; transcript_id \"chr9.g9639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15023191', '15023577', '.', '+', '.', 'gene_id \"chr9.g9641\"; transcript_id \"chr9.g9641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15031032', '15031232', '.', '-', '.', 'gene_id \"chr9.g9645\"; transcript_id \"chr9.g9645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15043544', '15043763', '.', '+', '.', 'gene_id \"chr9.g9650\"; transcript_id \"chr9.g9650.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15048244', '15048384', '.', '-', '.', 'gene_id \"chr9.g9651\"; transcript_id \"chr9.g9651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15048446', '15048666', '.', '-', '.', 'gene_id \"chr9.g9652\"; transcript_id \"chr9.g9652.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15063437', '15063586', '.', '-', '.', 'gene_id \"chr9.g9655\"; transcript_id \"chr9.g9655.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15066598', '15066802', '.', '-', '.', 'gene_id \"chr9.g9658\"; transcript_id \"chr9.g9658.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15068177', '15068358', '.', '-', '.', 'gene_id \"chr9.g9659\"; transcript_id \"chr9.g9659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15071050', '15071224', '.', '+', '.', 'gene_id \"chr9.g9660\"; transcript_id \"chr9.g9660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15075094', '15075269', '.', '+', '.', 'gene_id \"chr9.g9665\"; transcript_id \"chr9.g9665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15109897', '15110169', '.', '+', '.', 'gene_id \"chr9.g9675\"; transcript_id \"chr9.g9675.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15110325', '15110750', '.', '-', '.', 'gene_id \"chr9.g9676\"; transcript_id \"chr9.g9676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15168976', '15169214', '.', '+', '.', 'gene_id \"chr9.g9684\"; transcript_id \"chr9.g9684.i17\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15196669', '15197718', '.', '+', '.', 'gene_id \"chr9.g9684\"; transcript_id \"chr9.g9684.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15173408', '15173720', '.', '-', '.', 'gene_id \"chr9.g9689\"; transcript_id \"chr9.g9689.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15174367', '15174934', '.', '-', '.', 'gene_id \"chr9.g9691\"; transcript_id \"chr9.g9691.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15179015', '15179248', '.', '+', '.', 'gene_id \"chr9.g9697\"; transcript_id \"chr9.g9697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15179142', '15179248', '.', '+', '.', 'gene_id \"chr9.g9697\"; transcript_id \"chr9.g9697.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15179859', '15180191', '.', '-', '.', 'gene_id \"chr9.g9700\"; transcript_id \"chr9.g9700.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15179887', '15180191', '.', '-', '.', 'gene_id \"chr9.g9700\"; transcript_id \"chr9.g9700.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15181040', '15181223', '.', '+', '.', 'gene_id \"chr9.g9701\"; transcript_id \"chr9.g9701.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15181270', '15181485', '.', '+', '.', 'gene_id \"chr9.g9702\"; transcript_id \"chr9.g9702.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15183162', '15183461', '.', '+', '.', 'gene_id \"chr9.g9704\"; transcript_id \"chr9.g9704.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15183696', '15183847', '.', '+', '.', 'gene_id \"chr9.g9706\"; transcript_id \"chr9.g9706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15185415', '15185704', '.', '+', '.', 'gene_id \"chr9.g9707\"; transcript_id \"chr9.g9707.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15185694', '15186041', '.', '-', '.', 'gene_id \"chr9.g9708\"; transcript_id \"chr9.g9708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15186322', '15186474', '.', '-', '.', 'gene_id \"chr9.g9709\"; transcript_id \"chr9.g9709.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15186162', '15186340', '.', '+', '.', 'gene_id \"chr9.g9710\"; transcript_id \"chr9.g9710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15188046', '15188367', '.', '+', '.', 'gene_id \"chr9.g9713\"; transcript_id \"chr9.g9713.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15189082', '15189402', '.', '-', '.', 'gene_id \"chr9.g9714\"; transcript_id \"chr9.g9714.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15334425', '15334585', '.', '-', '.', 'gene_id \"chr9.g9725\"; transcript_id \"chr9.g9725.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15427129', '15428699', '.', '+', '.', 'gene_id \"chr9.g9745\"; transcript_id \"chr9.g9745.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15418114', '15418954', '.', '+', '.', 'gene_id \"chr9.g9747\"; transcript_id \"chr9.g9747.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15422320', '15422526', '.', '+', '.', 'gene_id \"chr9.g9748\"; transcript_id \"chr9.g9748.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15419075', '15419511', '.', '+', '.', 'gene_id \"chr9.g9750\"; transcript_id \"chr9.g9750.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15448609', '15448845', '.', '+', '.', 'gene_id \"chr9.g9755\"; transcript_id \"chr9.g9755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15499641', '15499824', '.', '-', '.', 'gene_id \"chr9.g9769\"; transcript_id \"chr9.g9769.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15499034', '15499269', '.', '-', '.', 'gene_id \"chr9.g9770\"; transcript_id \"chr9.g9770.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15506397', '15506973', '.', '-', '.', 'gene_id \"chr9.g9777\"; transcript_id \"chr9.g9777.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15507004', '15507257', '.', '-', '.', 'gene_id \"chr9.g9778\"; transcript_id \"chr9.g9778.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15507292', '15507594', '.', '-', '.', 'gene_id \"chr9.g9779\"; transcript_id \"chr9.g9779.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15508182', '15508357', '.', '+', '.', 'gene_id \"chr9.g9781\"; transcript_id \"chr9.g9781.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15508444', '15508653', '.', '-', '.', 'gene_id \"chr9.g9784\"; transcript_id \"chr9.g9784.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15512568', '15512802', '.', '+', '.', 'gene_id \"chr9.g9790\"; transcript_id \"chr9.g9790.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15525133', '15525242', '.', '-', '.', 'gene_id \"chr9.g9796\"; transcript_id \"chr9.g9796.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15517394', '15517641', '.', '-', '.', 'gene_id \"chr9.g9802\"; transcript_id \"chr9.g9802.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15517394', '15517551', '.', '-', '.', 'gene_id \"chr9.g9802\"; transcript_id \"chr9.g9802.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15518646', '15518816', '.', '+', '.', 'gene_id \"chr9.g9803\"; transcript_id \"chr9.g9803.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15519325', '15520159', '.', '+', '.', 'gene_id \"chr9.g9804\"; transcript_id \"chr9.g9804.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15520143', '15520378', '.', '-', '.', 'gene_id \"chr9.g9805\"; transcript_id \"chr9.g9805.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15520377', '15520554', '.', '+', '.', 'gene_id \"chr9.g9806\"; transcript_id \"chr9.g9806.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15521487', '15521707', '.', '-', '.', 'gene_id \"chr9.g9807\"; transcript_id \"chr9.g9807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15523813', '15523987', '.', '+', '.', 'gene_id \"chr9.g9811\"; transcript_id \"chr9.g9811.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15573102', '15574805', '.', '-', '.', 'gene_id \"chr9.g9824\"; transcript_id \"chr9.g9824.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15568385', '15568595', '.', '-', '.', 'gene_id \"chr9.g9827\"; transcript_id \"chr9.g9827.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15540646', '15541100', '.', '+', '.', 'gene_id \"chr9.g9834\"; transcript_id \"chr9.g9834.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15541459', '15542028', '.', '-', '.', 'gene_id \"chr9.g9835\"; transcript_id \"chr9.g9835.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15542261', '15542418', '.', '+', '.', 'gene_id \"chr9.g9837\"; transcript_id \"chr9.g9837.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15544045', '15544588', '.', '-', '.', 'gene_id \"chr9.g9838\"; transcript_id \"chr9.g9838.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15544561', '15544811', '.', '+', '.', 'gene_id \"chr9.g9839\"; transcript_id \"chr9.g9839.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15544906', '15545156', '.', '-', '.', 'gene_id \"chr9.g9840\"; transcript_id \"chr9.g9840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15545803', '15546089', '.', '-', '.', 'gene_id \"chr9.g9841\"; transcript_id \"chr9.g9841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15553833', '15554486', '.', '-', '.', 'gene_id \"chr9.g9848\"; transcript_id \"chr9.g9848.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15552204', '15552536', '.', '+', '.', 'gene_id \"chr9.g9849\"; transcript_id \"chr9.g9849.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15554469', '15554818', '.', '+', '.', 'gene_id \"chr9.g9850\"; transcript_id \"chr9.g9850.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15554934', '15555131', '.', '-', '.', 'gene_id \"chr9.g9851\"; transcript_id \"chr9.g9851.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15556101', '15556286', '.', '-', '.', 'gene_id \"chr9.g9854\"; transcript_id \"chr9.g9854.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15564905', '15565125', '.', '-', '.', 'gene_id \"chr9.g9861\"; transcript_id \"chr9.g9861.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15568884', '15569488', '.', '+', '.', 'gene_id \"chr9.g9867\"; transcript_id \"chr9.g9867.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15571945', '15572105', '.', '+', '.', 'gene_id \"chr9.g9869\"; transcript_id \"chr9.g9869.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15571185', '15571957', '.', '-', '.', 'gene_id \"chr9.g9871\"; transcript_id \"chr9.g9871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15598917', '15599293', '.', '+', '.', 'gene_id \"chr9.g9874\"; transcript_id \"chr9.g9874.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15599929', '15600211', '.', '+', '.', 'gene_id \"chr9.g9875\"; transcript_id \"chr9.g9875.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15600408', '15600588', '.', '+', '.', 'gene_id \"chr9.g9876\"; transcript_id \"chr9.g9876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15601680', '15602531', '.', '+', '.', 'gene_id \"chr9.g9880\"; transcript_id \"chr9.g9880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15609897', '15611057', '.', '+', '.', 'gene_id \"chr9.g9882\"; transcript_id \"chr9.g9882.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15605637', '15606009', '.', '+', '.', 'gene_id \"chr9.g9882\"; transcript_id \"chr9.g9882.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15606501', '15607197', '.', '-', '.', 'gene_id \"chr9.g9886\"; transcript_id \"chr9.g9886.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15720892', '15722184', '.', '+', '.', 'gene_id \"chr9.g9890\"; transcript_id \"chr9.g9890.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15722377', '15724466', '.', '+', '.', 'gene_id \"chr9.g9890\"; transcript_id \"chr9.g9890.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15723461', '15723643', '.', '+', '.', 'gene_id \"chr9.g9890\"; transcript_id \"chr9.g9890.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15667664', '15667781', '.', '+', '.', 'gene_id \"chr9.g9890\"; transcript_id \"chr9.g9890.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15748164', '15748374', '.', '+', '.', 'gene_id \"chr9.g9892\"; transcript_id \"chr9.g9892.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15716356', '15716989', '.', '-', '.', 'gene_id \"chr9.g9893\"; transcript_id \"chr9.g9893.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15713377', '15713536', '.', '-', '.', 'gene_id \"chr9.g9894\"; transcript_id \"chr9.g9894.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15728481', '15729601', '.', '+', '.', 'gene_id \"chr9.g9895\"; transcript_id \"chr9.g9895.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15712683', '15712836', '.', '-', '.', 'gene_id \"chr9.g9896\"; transcript_id \"chr9.g9896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15706422', '15706594', '.', '+', '.', 'gene_id \"chr9.g9897\"; transcript_id \"chr9.g9897.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15728204', '15728498', '.', '+', '.', 'gene_id \"chr9.g9900\"; transcript_id \"chr9.g9900.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15731742', '15732223', '.', '+', '.', 'gene_id \"chr9.g9902\"; transcript_id \"chr9.g9902.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15733208', '15733407', '.', '+', '.', 'gene_id \"chr9.g9905\"; transcript_id \"chr9.g9905.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15623676', '15623910', '.', '+', '.', 'gene_id \"chr9.g9912\"; transcript_id \"chr9.g9912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15627123', '15627325', '.', '-', '.', 'gene_id \"chr9.g9914\"; transcript_id \"chr9.g9914.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15627846', '15627931', '.', '+', '.', 'gene_id \"chr9.g9915\"; transcript_id \"chr9.g9915.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15634967', '15635311', '.', '-', '.', 'gene_id \"chr9.g9926\"; transcript_id \"chr9.g9926.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15635473', '15635757', '.', '+', '.', 'gene_id \"chr9.g9928\"; transcript_id \"chr9.g9928.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15641136', '15641678', '.', '+', '.', 'gene_id \"chr9.g9931\"; transcript_id \"chr9.g9931.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15698919', '15699104', '.', '-', '.', 'gene_id \"chr9.g9938\"; transcript_id \"chr9.g9938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15700173', '15700369', '.', '+', '.', 'gene_id \"chr9.g9939\"; transcript_id \"chr9.g9939.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15710803', '15710954', '.', '+', '.', 'gene_id \"chr9.g9948\"; transcript_id \"chr9.g9948.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15711090', '15712296', '.', '+', '.', 'gene_id \"chr9.g9949\"; transcript_id \"chr9.g9949.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15718851', '15719855', '.', '+', '.', 'gene_id \"chr9.g9955\"; transcript_id \"chr9.g9955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15726540', '15726759', '.', '+', '.', 'gene_id \"chr9.g9958\"; transcript_id \"chr9.g9958.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15729603', '15729845', '.', '-', '.', 'gene_id \"chr9.g9960\"; transcript_id \"chr9.g9960.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15729860', '15731021', '.', '-', '.', 'gene_id \"chr9.g9961\"; transcript_id \"chr9.g9961.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15734048', '15734304', '.', '-', '.', 'gene_id \"chr9.g9962\"; transcript_id \"chr9.g9962.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15735430', '15735853', '.', '-', '.', 'gene_id \"chr9.g9966\"; transcript_id \"chr9.g9966.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15738589', '15739295', '.', '+', '.', 'gene_id \"chr9.g9972\"; transcript_id \"chr9.g9972.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15739841', '15740007', '.', '+', '.', 'gene_id \"chr9.g9974\"; transcript_id \"chr9.g9974.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15745778', '15745973', '.', '-', '.', 'gene_id \"chr9.g9980\"; transcript_id \"chr9.g9980.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15746957', '15747164', '.', '-', '.', 'gene_id \"chr9.g9984\"; transcript_id \"chr9.g9984.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15747207', '15747404', '.', '+', '.', 'gene_id \"chr9.g9985\"; transcript_id \"chr9.g9985.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15749189', '15749373', '.', '+', '.', 'gene_id \"chr9.g9986\"; transcript_id \"chr9.g9986.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15750196', '15750440', '.', '+', '.', 'gene_id \"chr9.g9987\"; transcript_id \"chr9.g9987.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15750464', '15750695', '.', '+', '.', 'gene_id \"chr9.g9989\"; transcript_id \"chr9.g9989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15753330', '15753691', '.', '+', '.', 'gene_id \"chr9.g9994\"; transcript_id \"chr9.g9994.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15786265', '15786318', '.', '+', '.', 'gene_id \"chr9.g9999\"; transcript_id \"chr9.g9999.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15802561', '15802690', '.', '-', '.', 'gene_id \"chr9.g10001\"; transcript_id \"chr9.g10001.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15802668', '15803259', '.', '-', '.', 'gene_id \"chr9.g10001\"; transcript_id \"chr9.g10001.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15802329', '15802556', '.', '-', '.', 'gene_id \"chr9.g10004\"; transcript_id \"chr9.g10004.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15803373', '15804670', '.', '+', '.', 'gene_id \"chr9.g10007\"; transcript_id \"chr9.g10007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15795719', '15795832', '.', '+', '.', 'gene_id \"chr9.g10014\"; transcript_id \"chr9.g10014.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15798471', '15798920', '.', '+', '.', 'gene_id \"chr9.g10017\"; transcript_id \"chr9.g10017.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15799233', '15799489', '.', '-', '.', 'gene_id \"chr9.g10018\"; transcript_id \"chr9.g10018.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15862594', '15862835', '.', '-', '.', 'gene_id \"chr9.g10031\"; transcript_id \"chr9.g10031.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15812975', '15813638', '.', '-', '.', 'gene_id \"chr9.g10036\"; transcript_id \"chr9.g10036.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15812743', '15812979', '.', '-', '.', 'gene_id \"chr9.g10037\"; transcript_id \"chr9.g10037.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15816191', '15816350', '.', '+', '.', 'gene_id \"chr9.g10041\"; transcript_id \"chr9.g10041.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15832318', '15832481', '.', '+', '.', 'gene_id \"chr9.g10053\"; transcript_id \"chr9.g10053.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15833252', '15833411', '.', '-', '.', 'gene_id \"chr9.g10055\"; transcript_id \"chr9.g10055.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15833434', '15833939', '.', '+', '.', 'gene_id \"chr9.g10056\"; transcript_id \"chr9.g10056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15841564', '15842035', '.', '+', '.', 'gene_id \"chr9.g10063\"; transcript_id \"chr9.g10063.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15842561', '15843006', '.', '-', '.', 'gene_id \"chr9.g10064\"; transcript_id \"chr9.g10064.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15856148', '15856505', '.', '-', '.', 'gene_id \"chr9.g10087\"; transcript_id \"chr9.g10087.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15857474', '15857724', '.', '-', '.', 'gene_id \"chr9.g10090\"; transcript_id \"chr9.g10090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15859807', '15860007', '.', '-', '.', 'gene_id \"chr9.g10093\"; transcript_id \"chr9.g10093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15865416', '15865574', '.', '+', '.', 'gene_id \"chr9.g10097\"; transcript_id \"chr9.g10097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15979132', '15979286', '.', '+', '.', 'gene_id \"chr9.g10119\"; transcript_id \"chr9.g10119.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15999147', '15999303', '.', '-', '.', 'gene_id \"chr9.g10120\"; transcript_id \"chr9.g10120.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16039135', '16039631', '.', '-', '.', 'gene_id \"chr9.g10125\"; transcript_id \"chr9.g10125.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16027754', '16027925', '.', '-', '.', 'gene_id \"chr9.g10125\"; transcript_id \"chr9.g10125.i19\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16050531', '16050594', '.', '-', '.', 'gene_id \"chr9.g10125\"; transcript_id \"chr9.g10125.i22\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16050596', '16050631', '.', '-', '.', 'gene_id \"chr9.g10125\"; transcript_id \"chr9.g10125.i23\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16066872', '16067028', '.', '-', '.', 'gene_id \"chr9.g10126\"; transcript_id \"chr9.g10126.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16029464', '16029553', '.', '+', '.', 'gene_id \"chr9.g10127\"; transcript_id \"chr9.g10127.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16029464', '16029680', '.', '+', '.', 'gene_id \"chr9.g10127\"; transcript_id \"chr9.g10127.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16059966', '16060116', '.', '-', '.', 'gene_id \"chr9.g10138\"; transcript_id \"chr9.g10138.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16066565', '16066882', '.', '+', '.', 'gene_id \"chr9.g10140\"; transcript_id \"chr9.g10140.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16048204', '16049590', '.', '+', '.', 'gene_id \"chr9.g10141\"; transcript_id \"chr9.g10141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16044113', '16044270', '.', '-', '.', 'gene_id \"chr9.g10142\"; transcript_id \"chr9.g10142.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16035063', '16035698', '.', '+', '.', 'gene_id \"chr9.g10146\"; transcript_id \"chr9.g10146.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16035934', '16036090', '.', '+', '.', 'gene_id \"chr9.g10149\"; transcript_id \"chr9.g10149.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16036252', '16036510', '.', '+', '.', 'gene_id \"chr9.g10151\"; transcript_id \"chr9.g10151.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16041590', '16041820', '.', '+', '.', 'gene_id \"chr9.g10153\"; transcript_id \"chr9.g10153.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16041334', '16041611', '.', '+', '.', 'gene_id \"chr9.g10154\"; transcript_id \"chr9.g10154.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16052230', '16052480', '.', '+', '.', 'gene_id \"chr9.g10161\"; transcript_id \"chr9.g10161.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16061110', '16061423', '.', '+', '.', 'gene_id \"chr9.g10168\"; transcript_id \"chr9.g10168.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16061578', '16061798', '.', '+', '.', 'gene_id \"chr9.g10169\"; transcript_id \"chr9.g10169.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16067634', '16067849', '.', '+', '.', 'gene_id \"chr9.g10172\"; transcript_id \"chr9.g10172.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16075864', '16076382', '.', '-', '.', 'gene_id \"chr9.g10174\"; transcript_id \"chr9.g10174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16088505', '16088683', '.', '+', '.', 'gene_id \"chr9.g10180\"; transcript_id \"chr9.g10180.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16092288', '16093005', '.', '+', '.', 'gene_id \"chr9.g10185\"; transcript_id \"chr9.g10185.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16096584', '16096787', '.', '-', '.', 'gene_id \"chr9.g10186\"; transcript_id \"chr9.g10186.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16130143', '16130870', '.', '-', '.', 'gene_id \"chr9.g10189\"; transcript_id \"chr9.g10189.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16145414', '16145685', '.', '-', '.', 'gene_id \"chr9.g10190\"; transcript_id \"chr9.g10190.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16172698', '16172923', '.', '+', '.', 'gene_id \"chr9.g10198\"; transcript_id \"chr9.g10198.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16137425', '16137621', '.', '+', '.', 'gene_id \"chr9.g10199\"; transcript_id \"chr9.g10199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16123998', '16124593', '.', '-', '.', 'gene_id \"chr9.g10201\"; transcript_id \"chr9.g10201.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16124582', '16124848', '.', '+', '.', 'gene_id \"chr9.g10202\"; transcript_id \"chr9.g10202.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16124858', '16126444', '.', '+', '.', 'gene_id \"chr9.g10203\"; transcript_id \"chr9.g10203.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16129203', '16129421', '.', '-', '.', 'gene_id \"chr9.g10206\"; transcript_id \"chr9.g10206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16132973', '16133226', '.', '-', '.', 'gene_id \"chr9.g10212\"; transcript_id \"chr9.g10212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16134296', '16134589', '.', '+', '.', 'gene_id \"chr9.g10213\"; transcript_id \"chr9.g10213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16168454', '16168932', '.', '-', '.', 'gene_id \"chr9.g10236\"; transcript_id \"chr9.g10236.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16172924', '16173131', '.', '+', '.', 'gene_id \"chr9.g10240\"; transcript_id \"chr9.g10240.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16173435', '16173606', '.', '-', '.', 'gene_id \"chr9.g10241\"; transcript_id \"chr9.g10241.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16192201', '16192466', '.', '+', '.', 'gene_id \"chr9.g10256\"; transcript_id \"chr9.g10256.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16247451', '16247490', '.', '-', '.', 'gene_id \"chr9.g10257\"; transcript_id \"chr9.g10257.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16249908', '16250021', '.', '-', '.', 'gene_id \"chr9.g10257\"; transcript_id \"chr9.g10257.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16197665', '16198447', '.', '-', '.', 'gene_id \"chr9.g10257\"; transcript_id \"chr9.g10257.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16195298', '16195594', '.', '-', '.', 'gene_id \"chr9.g10258\"; transcript_id \"chr9.g10258.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16249999', '16250170', '.', '+', '.', 'gene_id \"chr9.g10259\"; transcript_id \"chr9.g10259.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16192564', '16193909', '.', '+', '.', 'gene_id \"chr9.g10260\"; transcript_id \"chr9.g10260.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16193759', '16193909', '.', '+', '.', 'gene_id \"chr9.g10260\"; transcript_id \"chr9.g10260.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16214958', '16215042', '.', '+', '.', 'gene_id \"chr9.g10263\"; transcript_id \"chr9.g10263.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16208785', '16209293', '.', '+', '.', 'gene_id \"chr9.g10272\"; transcript_id \"chr9.g10272.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16217693', '16219179', '.', '-', '.', 'gene_id \"chr9.g10273\"; transcript_id \"chr9.g10273.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16196767', '16197615', '.', '+', '.', 'gene_id \"chr9.g10275\"; transcript_id \"chr9.g10275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16200593', '16200825', '.', '-', '.', 'gene_id \"chr9.g10276\"; transcript_id \"chr9.g10276.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16202143', '16202380', '.', '-', '.', 'gene_id \"chr9.g10277\"; transcript_id \"chr9.g10277.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16195608', '16196637', '.', '+', '.', 'gene_id \"chr9.g10279\"; transcript_id \"chr9.g10279.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16217693', '16219179', '.', '+', '.', 'gene_id \"chr9.g10280\"; transcript_id \"chr9.g10280.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16208513', '16208787', '.', '-', '.', 'gene_id \"chr9.g10282\"; transcript_id \"chr9.g10282.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16212059', '16212913', '.', '-', '.', 'gene_id \"chr9.g10283\"; transcript_id \"chr9.g10283.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16235963', '16236205', '.', '+', '.', 'gene_id \"chr9.g10289\"; transcript_id \"chr9.g10289.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16238380', '16238583', '.', '+', '.', 'gene_id \"chr9.g10290\"; transcript_id \"chr9.g10290.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16242000', '16242471', '.', '+', '.', 'gene_id \"chr9.g10291\"; transcript_id \"chr9.g10291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16199848', '16200079', '.', '+', '.', 'gene_id \"chr9.g10292\"; transcript_id \"chr9.g10292.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16202791', '16203005', '.', '-', '.', 'gene_id \"chr9.g10293\"; transcript_id \"chr9.g10293.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16215740', '16215938', '.', '+', '.', 'gene_id \"chr9.g10297\"; transcript_id \"chr9.g10297.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16215860', '16215987', '.', '+', '.', 'gene_id \"chr9.g10297\"; transcript_id \"chr9.g10297.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16216913', '16217402', '.', '-', '.', 'gene_id \"chr9.g10300\"; transcript_id \"chr9.g10300.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16222711', '16222913', '.', '+', '.', 'gene_id \"chr9.g10305\"; transcript_id \"chr9.g10305.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16224089', '16224320', '.', '-', '.', 'gene_id \"chr9.g10308\"; transcript_id \"chr9.g10308.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16231605', '16231850', '.', '-', '.', 'gene_id \"chr9.g10314\"; transcript_id \"chr9.g10314.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16237214', '16237368', '.', '+', '.', 'gene_id \"chr9.g10323\"; transcript_id \"chr9.g10323.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16240129', '16240376', '.', '+', '.', 'gene_id \"chr9.g10326\"; transcript_id \"chr9.g10326.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16240596', '16240807', '.', '-', '.', 'gene_id \"chr9.g10327\"; transcript_id \"chr9.g10327.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16240983', '16241183', '.', '+', '.', 'gene_id \"chr9.g10329\"; transcript_id \"chr9.g10329.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16243118', '16243453', '.', '-', '.', 'gene_id \"chr9.g10333\"; transcript_id \"chr9.g10333.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16244060', '16244453', '.', '-', '.', 'gene_id \"chr9.g10334\"; transcript_id \"chr9.g10334.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16250812', '16251018', '.', '-', '.', 'gene_id \"chr9.g10337\"; transcript_id \"chr9.g10337.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16263779', '16264595', '.', '+', '.', 'gene_id \"chr9.g10341\"; transcript_id \"chr9.g10341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16265197', '16265483', '.', '+', '.', 'gene_id \"chr9.g10343\"; transcript_id \"chr9.g10343.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16264597', '16265012', '.', '-', '.', 'gene_id \"chr9.g10344\"; transcript_id \"chr9.g10344.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16264597', '16265477', '.', '-', '.', 'gene_id \"chr9.g10344\"; transcript_id \"chr9.g10344.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16373650', '16373922', '.', '-', '.', 'gene_id \"chr9.g10353\"; transcript_id \"chr9.g10353.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16373332', '16373659', '.', '+', '.', 'gene_id \"chr9.g10354\"; transcript_id \"chr9.g10354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16330078', '16330437', '.', '+', '.', 'gene_id \"chr9.g10358\"; transcript_id \"chr9.g10358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16316861', '16317131', '.', '-', '.', 'gene_id \"chr9.g10359\"; transcript_id \"chr9.g10359.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16374105', '16374153', '.', '+', '.', 'gene_id \"chr9.g10362\"; transcript_id \"chr9.g10362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16319426', '16319652', '.', '-', '.', 'gene_id \"chr9.g10366\"; transcript_id \"chr9.g10366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16329537', '16329699', '.', '+', '.', 'gene_id \"chr9.g10373\"; transcript_id \"chr9.g10373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16335867', '16336122', '.', '+', '.', 'gene_id \"chr9.g10374\"; transcript_id \"chr9.g10374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16363454', '16363611', '.', '+', '.', 'gene_id \"chr9.g10390\"; transcript_id \"chr9.g10390.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16366438', '16366589', '.', '+', '.', 'gene_id \"chr9.g10394\"; transcript_id \"chr9.g10394.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16370732', '16371016', '.', '-', '.', 'gene_id \"chr9.g10397\"; transcript_id \"chr9.g10397.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16375426', '16375591', '.', '+', '.', 'gene_id \"chr9.g10402\"; transcript_id \"chr9.g10402.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16375898', '16376199', '.', '-', '.', 'gene_id \"chr9.g10404\"; transcript_id \"chr9.g10404.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16379833', '16379991', '.', '+', '.', 'gene_id \"chr9.g10408\"; transcript_id \"chr9.g10408.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16380748', '16381217', '.', '+', '.', 'gene_id \"chr9.g10410\"; transcript_id \"chr9.g10410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16382040', '16382486', '.', '-', '.', 'gene_id \"chr9.g10413\"; transcript_id \"chr9.g10413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16394101', '16394259', '.', '+', '.', 'gene_id \"chr9.g10422\"; transcript_id \"chr9.g10422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16396543', '16396896', '.', '+', '.', 'gene_id \"chr9.g10424\"; transcript_id \"chr9.g10424.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16400519', '16401031', '.', '-', '.', 'gene_id \"chr9.g10426\"; transcript_id \"chr9.g10426.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16401176', '16401464', '.', '+', '.', 'gene_id \"chr9.g10427\"; transcript_id \"chr9.g10427.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16401441', '16401626', '.', '+', '.', 'gene_id \"chr9.g10427\"; transcript_id \"chr9.g10427.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16401633', '16401917', '.', '-', '.', 'gene_id \"chr9.g10430\"; transcript_id \"chr9.g10430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16402656', '16403108', '.', '+', '.', 'gene_id \"chr9.g10432\"; transcript_id \"chr9.g10432.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16443744', '16443999', '.', '+', '.', 'gene_id \"chr9.g10441\"; transcript_id \"chr9.g10441.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16559179', '16560449', '.', '+', '.', 'gene_id \"chr9.g10444\"; transcript_id \"chr9.g10444.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16556961', '16559078', '.', '-', '.', 'gene_id \"chr9.g10448\"; transcript_id \"chr9.g10448.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16470373', '16470503', '.', '+', '.', 'gene_id \"chr9.g10455\"; transcript_id \"chr9.g10455.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16470463', '16470514', '.', '-', '.', 'gene_id \"chr9.g10456\"; transcript_id \"chr9.g10456.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16487388', '16487611', '.', '+', '.', 'gene_id \"chr9.g10464\"; transcript_id \"chr9.g10464.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16488006', '16488220', '.', '-', '.', 'gene_id \"chr9.g10465\"; transcript_id \"chr9.g10465.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16488221', '16488513', '.', '+', '.', 'gene_id \"chr9.g10466\"; transcript_id \"chr9.g10466.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16530099', '16530285', '.', '-', '.', 'gene_id \"chr9.g10471\"; transcript_id \"chr9.g10471.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16530132', '16530285', '.', '-', '.', 'gene_id \"chr9.g10471\"; transcript_id \"chr9.g10471.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16534540', '16534672', '.', '-', '.', 'gene_id \"chr9.g10472\"; transcript_id \"chr9.g10472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16539700', '16539920', '.', '-', '.', 'gene_id \"chr9.g10476\"; transcript_id \"chr9.g10476.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16547209', '16547367', '.', '+', '.', 'gene_id \"chr9.g10479\"; transcript_id \"chr9.g10479.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16547620', '16547769', '.', '+', '.', 'gene_id \"chr9.g10480\"; transcript_id \"chr9.g10480.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16550905', '16551243', '.', '+', '.', 'gene_id \"chr9.g10481\"; transcript_id \"chr9.g10481.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16551351', '16551627', '.', '-', '.', 'gene_id \"chr9.g10483\"; transcript_id \"chr9.g10483.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16551643', '16551802', '.', '+', '.', 'gene_id \"chr9.g10484\"; transcript_id \"chr9.g10484.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16763202', '16763697', '.', '+', '.', 'gene_id \"chr9.g10510\"; transcript_id \"chr9.g10510.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16763205', '16763697', '.', '+', '.', 'gene_id \"chr9.g10510\"; transcript_id \"chr9.g10510.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16813005', '16813158', '.', '-', '.', 'gene_id \"chr9.g10519\"; transcript_id \"chr9.g10519.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16817630', '16817784', '.', '-', '.', 'gene_id \"chr9.g10522\"; transcript_id \"chr9.g10522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16883565', '16883775', '.', '-', '.', 'gene_id \"chr9.g10538\"; transcript_id \"chr9.g10538.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16886901', '16888177', '.', '-', '.', 'gene_id \"chr9.g10541\"; transcript_id \"chr9.g10541.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16888257', '16888526', '.', '-', '.', 'gene_id \"chr9.g10543\"; transcript_id \"chr9.g10543.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16888654', '16888899', '.', '+', '.', 'gene_id \"chr9.g10544\"; transcript_id \"chr9.g10544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16892387', '16892548', '.', '+', '.', 'gene_id \"chr9.g10550\"; transcript_id \"chr9.g10550.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16902305', '16902635', '.', '-', '.', 'gene_id \"chr9.g10551\"; transcript_id \"chr9.g10551.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16892782', '16893660', '.', '-', '.', 'gene_id \"chr9.g10551\"; transcript_id \"chr9.g10551.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16901402', '16901554', '.', '-', '.', 'gene_id \"chr9.g10553\"; transcript_id \"chr9.g10553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16908858', '16909014', '.', '-', '.', 'gene_id \"chr9.g10555\"; transcript_id \"chr9.g10555.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16911916', '16912087', '.', '+', '.', 'gene_id \"chr9.g10556\"; transcript_id \"chr9.g10556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16919984', '16920140', '.', '+', '.', 'gene_id \"chr9.g10561\"; transcript_id \"chr9.g10561.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16920803', '16920985', '.', '+', '.', 'gene_id \"chr9.g10562\"; transcript_id \"chr9.g10562.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16936716', '16936894', '.', '+', '.', 'gene_id \"chr9.g10571\"; transcript_id \"chr9.g10571.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16993336', '16993500', '.', '+', '.', 'gene_id \"chr9.g10585\"; transcript_id \"chr9.g10585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16998362', '16998495', '.', '+', '.', 'gene_id \"chr9.g10592\"; transcript_id \"chr9.g10592.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17005944', '17006107', '.', '-', '.', 'gene_id \"chr9.g10603\"; transcript_id \"chr9.g10603.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17006938', '17007085', '.', '+', '.', 'gene_id \"chr9.g10606\"; transcript_id \"chr9.g10606.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17011475', '17011753', '.', '+', '.', 'gene_id \"chr9.g10609\"; transcript_id \"chr9.g10609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17024539', '17024820', '.', '-', '.', 'gene_id \"chr9.g10620\"; transcript_id \"chr9.g10620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17029157', '17029291', '.', '-', '.', 'gene_id \"chr9.g10628\"; transcript_id \"chr9.g10628.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17030628', '17030910', '.', '+', '.', 'gene_id \"chr9.g10632\"; transcript_id \"chr9.g10632.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17033960', '17034116', '.', '+', '.', 'gene_id \"chr9.g10640\"; transcript_id \"chr9.g10640.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17034119', '17034345', '.', '+', '.', 'gene_id \"chr9.g10641\"; transcript_id \"chr9.g10641.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17034762', '17038979', '.', '-', '.', 'gene_id \"chr9.g10642\"; transcript_id \"chr9.g10642.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17034762', '17038386', '.', '-', '.', 'gene_id \"chr9.g10642\"; transcript_id \"chr9.g10642.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17039002', '17040228', '.', '+', '.', 'gene_id \"chr9.g10646\"; transcript_id \"chr9.g10646.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17041131', '17041550', '.', '-', '.', 'gene_id \"chr9.g10647\"; transcript_id \"chr9.g10647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17041801', '17042304', '.', '-', '.', 'gene_id \"chr9.g10648\"; transcript_id \"chr9.g10648.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17042577', '17042741', '.', '+', '.', 'gene_id \"chr9.g10650\"; transcript_id \"chr9.g10650.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '22073271', '22073342', '.', '-', '.', 'gene_id \"chr17.g7600\"; transcript_id \"chr17.g7600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17043491', '17044106', '.', '-', '.', 'gene_id \"chr9.g10651\"; transcript_id \"chr9.g10651.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17048567', '17048898', '.', '-', '.', 'gene_id \"chr9.g10651\"; transcript_id \"chr9.g10651.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17046626', '17047003', '.', '+', '.', 'gene_id \"chr9.g10654\"; transcript_id \"chr9.g10654.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17047053', '17047211', '.', '-', '.', 'gene_id \"chr9.g10657\"; transcript_id \"chr9.g10657.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17049589', '17050035', '.', '+', '.', 'gene_id \"chr9.g10660\"; transcript_id \"chr9.g10660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17051061', '17051916', '.', '+', '.', 'gene_id \"chr9.g10661\"; transcript_id \"chr9.g10661.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17054711', '17054948', '.', '-', '.', 'gene_id \"chr9.g10664\"; transcript_id \"chr9.g10664.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17055074', '17055312', '.', '+', '.', 'gene_id \"chr9.g10665\"; transcript_id \"chr9.g10665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17056308', '17056567', '.', '+', '.', 'gene_id \"chr9.g10669\"; transcript_id \"chr9.g10669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17056558', '17056754', '.', '+', '.', 'gene_id \"chr9.g10670\"; transcript_id \"chr9.g10670.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17057004', '17057332', '.', '+', '.', 'gene_id \"chr9.g10671\"; transcript_id \"chr9.g10671.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17057335', '17057522', '.', '+', '.', 'gene_id \"chr9.g10672\"; transcript_id \"chr9.g10672.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17057853', '17057965', '.', '-', '.', 'gene_id \"chr9.g10673\"; transcript_id \"chr9.g10673.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17060009', '17061207', '.', '+', '.', 'gene_id \"chr9.g10678\"; transcript_id \"chr9.g10678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17066155', '17066395', '.', '-', '.', 'gene_id \"chr9.g10686\"; transcript_id \"chr9.g10686.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17072130', '17072395', '.', '-', '.', 'gene_id \"chr9.g10689\"; transcript_id \"chr9.g10689.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17074500', '17075620', '.', '-', '.', 'gene_id \"chr9.g10690\"; transcript_id \"chr9.g10690.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17075622', '17076712', '.', '-', '.', 'gene_id \"chr9.g10691\"; transcript_id \"chr9.g10691.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17083222', '17083737', '.', '-', '.', 'gene_id \"chr9.g10700\"; transcript_id \"chr9.g10700.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17084062', '17084807', '.', '+', '.', 'gene_id \"chr9.g10704\"; transcript_id \"chr9.g10704.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17084656', '17084807', '.', '+', '.', 'gene_id \"chr9.g10704\"; transcript_id \"chr9.g10704.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17086316', '17086804', '.', '+', '.', 'gene_id \"chr9.g10708\"; transcript_id \"chr9.g10708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17092538', '17092742', '.', '+', '.', 'gene_id \"chr9.g10710\"; transcript_id \"chr9.g10710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17183045', '17183199', '.', '-', '.', 'gene_id \"chr9.g10721\"; transcript_id \"chr9.g10721.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17329062', '17329228', '.', '-', '.', 'gene_id \"chr9.g10730\"; transcript_id \"chr9.g10730.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17632784', '17632819', '.', '+', '.', 'gene_id \"chr9.g10750\"; transcript_id \"chr9.g10750.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17637711', '17639499', '.', '+', '.', 'gene_id \"chr9.g10755\"; transcript_id \"chr9.g10755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17640030', '17640252', '.', '-', '.', 'gene_id \"chr9.g10756\"; transcript_id \"chr9.g10756.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17639786', '17640009', '.', '-', '.', 'gene_id \"chr9.g10758\"; transcript_id \"chr9.g10758.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17642586', '17645928', '.', '+', '.', 'gene_id \"chr9.g10759\"; transcript_id \"chr9.g10759.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17641074', '17641697', '.', '+', '.', 'gene_id \"chr9.g10760\"; transcript_id \"chr9.g10760.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17641675', '17641875', '.', '+', '.', 'gene_id \"chr9.g10760\"; transcript_id \"chr9.g10760.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17666358', '17666612', '.', '+', '.', 'gene_id \"chr9.g10767\"; transcript_id \"chr9.g10767.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17654822', '17654989', '.', '+', '.', 'gene_id \"chr9.g10768\"; transcript_id \"chr9.g10768.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17658926', '17661001', '.', '-', '.', 'gene_id \"chr9.g10769\"; transcript_id \"chr9.g10769.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17655039', '17655862', '.', '-', '.', 'gene_id \"chr9.g10770\"; transcript_id \"chr9.g10770.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17654975', '17655093', '.', '-', '.', 'gene_id \"chr9.g10770\"; transcript_id \"chr9.g10770.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17660991', '17662811', '.', '+', '.', 'gene_id \"chr9.g10771\"; transcript_id \"chr9.g10771.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17662836', '17665394', '.', '+', '.', 'gene_id \"chr9.g10772\"; transcript_id \"chr9.g10772.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17649247', '17650218', '.', '-', '.', 'gene_id \"chr9.g10780\"; transcript_id \"chr9.g10780.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17650206', '17652303', '.', '-', '.', 'gene_id \"chr9.g10781\"; transcript_id \"chr9.g10781.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17667625', '17667815', '.', '+', '.', 'gene_id \"chr9.g10787\"; transcript_id \"chr9.g10787.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17668317', '17668492', '.', '-', '.', 'gene_id \"chr9.g10788\"; transcript_id \"chr9.g10788.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17671819', '17672318', '.', '+', '.', 'gene_id \"chr9.g10791\"; transcript_id \"chr9.g10791.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17672326', '17672898', '.', '-', '.', 'gene_id \"chr9.g10792\"; transcript_id \"chr9.g10792.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17672876', '17673317', '.', '-', '.', 'gene_id \"chr9.g10793\"; transcript_id \"chr9.g10793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17680470', '17680722', '.', '+', '.', 'gene_id \"chr9.g10797\"; transcript_id \"chr9.g10797.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17675653', '17677118', '.', '-', '.', 'gene_id \"chr9.g10799\"; transcript_id \"chr9.g10799.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17677317', '17678375', '.', '-', '.', 'gene_id \"chr9.g10800\"; transcript_id \"chr9.g10800.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17678437', '17678662', '.', '-', '.', 'gene_id \"chr9.g10801\"; transcript_id \"chr9.g10801.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17674366', '17675650', '.', '-', '.', 'gene_id \"chr9.g10802\"; transcript_id \"chr9.g10802.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17686289', '17686706', '.', '+', '.', 'gene_id \"chr9.g10804\"; transcript_id \"chr9.g10804.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17685914', '17686315', '.', '+', '.', 'gene_id \"chr9.g10806\"; transcript_id \"chr9.g10806.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17686703', '17686969', '.', '-', '.', 'gene_id \"chr9.g10807\"; transcript_id \"chr9.g10807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17690954', '17691255', '.', '+', '.', 'gene_id \"chr9.g10815\"; transcript_id \"chr9.g10815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17690993', '17691147', '.', '+', '.', 'gene_id \"chr9.g10815\"; transcript_id \"chr9.g10815.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17690663', '17690956', '.', '-', '.', 'gene_id \"chr9.g10816\"; transcript_id \"chr9.g10816.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17695708', '17695922', '.', '+', '.', 'gene_id \"chr9.g10824\"; transcript_id \"chr9.g10824.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '22290695', '22290731', '.', '+', '.', 'gene_id \"chr17.g7751\"; transcript_id \"chr17.g7751.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17731895', '17732030', '.', '-', '.', 'gene_id \"chr9.g10855\"; transcript_id \"chr9.g10855.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17732023', '17732305', '.', '+', '.', 'gene_id \"chr9.g10856\"; transcript_id \"chr9.g10856.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17749013', '17749425', '.', '+', '.', 'gene_id \"chr9.g10862\"; transcript_id \"chr9.g10862.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17750119', '17750383', '.', '+', '.', 'gene_id \"chr9.g10863\"; transcript_id \"chr9.g10863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17751882', '17752174', '.', '-', '.', 'gene_id \"chr9.g10866\"; transcript_id \"chr9.g10866.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17752247', '17752504', '.', '-', '.', 'gene_id \"chr9.g10867\"; transcript_id \"chr9.g10867.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17774309', '17774476', '.', '-', '.', 'gene_id \"chr9.g10872\"; transcript_id \"chr9.g10872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17807411', '17807791', '.', '-', '.', 'gene_id \"chr9.g10879\"; transcript_id \"chr9.g10879.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17807503', '17807791', '.', '-', '.', 'gene_id \"chr9.g10879\"; transcript_id \"chr9.g10879.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17790212', '17790376', '.', '-', '.', 'gene_id \"chr9.g10882\"; transcript_id \"chr9.g10882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17790513', '17790779', '.', '+', '.', 'gene_id \"chr9.g10883\"; transcript_id \"chr9.g10883.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17791792', '17792241', '.', '-', '.', 'gene_id \"chr9.g10884\"; transcript_id \"chr9.g10884.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17792785', '17793011', '.', '-', '.', 'gene_id \"chr9.g10886\"; transcript_id \"chr9.g10886.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17796815', '17797200', '.', '+', '.', 'gene_id \"chr9.g10893\"; transcript_id \"chr9.g10893.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17797604', '17797875', '.', '+', '.', 'gene_id \"chr9.g10894\"; transcript_id \"chr9.g10894.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17800136', '17800178', '.', '-', '.', 'gene_id \"chr9.g10898\"; transcript_id \"chr9.g10898.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17801970', '17802360', '.', '-', '.', 'gene_id \"chr9.g10900\"; transcript_id \"chr9.g10900.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17854739', '17858752', '.', '-', '.', 'gene_id \"chr9.g10905\"; transcript_id \"chr9.g10905.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17851733', '17851924', '.', '+', '.', 'gene_id \"chr9.g10907\"; transcript_id \"chr9.g10907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17844561', '17845411', '.', '-', '.', 'gene_id \"chr9.g10909\"; transcript_id \"chr9.g10909.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17842379', '17843022', '.', '-', '.', 'gene_id \"chr9.g10911\"; transcript_id \"chr9.g10911.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17837334', '17837518', '.', '-', '.', 'gene_id \"chr9.g10912\"; transcript_id \"chr9.g10912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17849972', '17850147', '.', '+', '.', 'gene_id \"chr9.g10913\"; transcript_id \"chr9.g10913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17851939', '17852105', '.', '-', '.', 'gene_id \"chr9.g10914\"; transcript_id \"chr9.g10914.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17812346', '17813049', '.', '-', '.', 'gene_id \"chr9.g10921\"; transcript_id \"chr9.g10921.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17812052', '17812338', '.', '-', '.', 'gene_id \"chr9.g10923\"; transcript_id \"chr9.g10923.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17818646', '17818835', '.', '-', '.', 'gene_id \"chr9.g10928\"; transcript_id \"chr9.g10928.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17826342', '17826402', '.', '+', '.', 'gene_id \"chr9.g10937\"; transcript_id \"chr9.g10937.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17827270', '17827434', '.', '-', '.', 'gene_id \"chr9.g10939\"; transcript_id \"chr9.g10939.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17831861', '17832209', '.', '-', '.', 'gene_id \"chr9.g10946\"; transcript_id \"chr9.g10946.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17833323', '17833490', '.', '-', '.', 'gene_id \"chr9.g10948\"; transcript_id \"chr9.g10948.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17833500', '17833839', '.', '-', '.', 'gene_id \"chr9.g10949\"; transcript_id \"chr9.g10949.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17833986', '17834476', '.', '-', '.', 'gene_id \"chr9.g10951\"; transcript_id \"chr9.g10951.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17836049', '17836306', '.', '-', '.', 'gene_id \"chr9.g10952\"; transcript_id \"chr9.g10952.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17836538', '17836879', '.', '+', '.', 'gene_id \"chr9.g10954\"; transcript_id \"chr9.g10954.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17837526', '17837816', '.', '-', '.', 'gene_id \"chr9.g10955\"; transcript_id \"chr9.g10955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17839312', '17839477', '.', '-', '.', 'gene_id \"chr9.g10959\"; transcript_id \"chr9.g10959.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17839894', '17840482', '.', '+', '.', 'gene_id \"chr9.g10960\"; transcript_id \"chr9.g10960.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17841606', '17841762', '.', '+', '.', 'gene_id \"chr9.g10961\"; transcript_id \"chr9.g10961.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17849346', '17849513', '.', '+', '.', 'gene_id \"chr9.g10965\"; transcript_id \"chr9.g10965.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17850883', '17851376', '.', '-', '.', 'gene_id \"chr9.g10968\"; transcript_id \"chr9.g10968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17851354', '17851508', '.', '+', '.', 'gene_id \"chr9.g10969\"; transcript_id \"chr9.g10969.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17870860', '17871119', '.', '-', '.', 'gene_id \"chr9.g10983\"; transcript_id \"chr9.g10983.i28\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17870609', '17870770', '.', '-', '.', 'gene_id \"chr9.g10983\"; transcript_id \"chr9.g10983.i31\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17879305', '17879605', '.', '+', '.', 'gene_id \"chr9.g10984\"; transcript_id \"chr9.g10984.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17879682', '17879874', '.', '+', '.', 'gene_id \"chr9.g10984\"; transcript_id \"chr9.g10984.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17871877', '17872389', '.', '-', '.', 'gene_id \"chr9.g10985\"; transcript_id \"chr9.g10985.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17876466', '17877335', '.', '-', '.', 'gene_id \"chr9.g10988\"; transcript_id \"chr9.g10988.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17877615', '17877805', '.', '-', '.', 'gene_id \"chr9.g10989\"; transcript_id \"chr9.g10989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17888359', '17888613', '.', '+', '.', 'gene_id \"chr9.g10991\"; transcript_id \"chr9.g10991.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17870246', '17870462', '.', '-', '.', 'gene_id \"chr9.g10993\"; transcript_id \"chr9.g10993.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17871651', '17871863', '.', '+', '.', 'gene_id \"chr9.g10995\"; transcript_id \"chr9.g10995.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17879031', '17879241', '.', '-', '.', 'gene_id \"chr9.g10996\"; transcript_id \"chr9.g10996.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17894267', '17894509', '.', '+', '.', 'gene_id \"chr9.g10997\"; transcript_id \"chr9.g10997.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17881856', '17882214', '.', '+', '.', 'gene_id \"chr9.g10999\"; transcript_id \"chr9.g10999.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17880119', '17880535', '.', '+', '.', 'gene_id \"chr9.g11000\"; transcript_id \"chr9.g11000.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17878020', '17878255', '.', '+', '.', 'gene_id \"chr9.g11002\"; transcript_id \"chr9.g11002.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17885549', '17885918', '.', '+', '.', 'gene_id \"chr9.g11009\"; transcript_id \"chr9.g11009.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17886045', '17886339', '.', '+', '.', 'gene_id \"chr9.g11010\"; transcript_id \"chr9.g11010.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17886320', '17886656', '.', '-', '.', 'gene_id \"chr9.g11011\"; transcript_id \"chr9.g11011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17901796', '17901966', '.', '+', '.', 'gene_id \"chr9.g11023\"; transcript_id \"chr9.g11023.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17904767', '17904799', '.', '-', '.', 'gene_id \"chr9.g11025\"; transcript_id \"chr9.g11025.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17920016', '17920167', '.', '-', '.', 'gene_id \"chr9.g11031\"; transcript_id \"chr9.g11031.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17934270', '17934378', '.', '-', '.', 'gene_id \"chr9.g11035\"; transcript_id \"chr9.g11035.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17931618', '17934295', '.', '-', '.', 'gene_id \"chr9.g11035\"; transcript_id \"chr9.g11035.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17929879', '17930053', '.', '+', '.', 'gene_id \"chr9.g11037\"; transcript_id \"chr9.g11037.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17926072', '17926502', '.', '+', '.', 'gene_id \"chr9.g11039\"; transcript_id \"chr9.g11039.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17928305', '17928525', '.', '+', '.', 'gene_id \"chr9.g11040\"; transcript_id \"chr9.g11040.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17927708', '17927988', '.', '-', '.', 'gene_id \"chr9.g11042\"; transcript_id \"chr9.g11042.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17940490', '17940851', '.', '-', '.', 'gene_id \"chr9.g11046\"; transcript_id \"chr9.g11046.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17943113', '17945799', '.', '+', '.', 'gene_id \"chr9.g11048\"; transcript_id \"chr9.g11048.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17947469', '17948040', '.', '+', '.', 'gene_id \"chr9.g11050\"; transcript_id \"chr9.g11050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17946777', '17947039', '.', '-', '.', 'gene_id \"chr9.g11051\"; transcript_id \"chr9.g11051.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17968989', '17969256', '.', '+', '.', 'gene_id \"chr9.g11058\"; transcript_id \"chr9.g11058.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17962967', '17963121', '.', '-', '.', 'gene_id \"chr9.g11061\"; transcript_id \"chr9.g11061.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17964549', '17964758', '.', '+', '.', 'gene_id \"chr9.g11062\"; transcript_id \"chr9.g11062.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17968350', '17968634', '.', '+', '.', 'gene_id \"chr9.g11063\"; transcript_id \"chr9.g11063.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17968744', '17968894', '.', '+', '.', 'gene_id \"chr9.g11074\"; transcript_id \"chr9.g11074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18007139', '18007711', '.', '-', '.', 'gene_id \"chr9.g11078\"; transcript_id \"chr9.g11078.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17979361', '17979616', '.', '-', '.', 'gene_id \"chr9.g11080\"; transcript_id \"chr9.g11080.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18024502', '18024803', '.', '+', '.', 'gene_id \"chr9.g11081\"; transcript_id \"chr9.g11081.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18024429', '18024531', '.', '+', '.', 'gene_id \"chr9.g11081\"; transcript_id \"chr9.g11081.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17976567', '17977158', '.', '-', '.', 'gene_id \"chr9.g11084\"; transcript_id \"chr9.g11084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18009244', '18009534', '.', '-', '.', 'gene_id \"chr9.g11087\"; transcript_id \"chr9.g11087.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17979754', '17979968', '.', '-', '.', 'gene_id \"chr9.g11090\"; transcript_id \"chr9.g11090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17983724', '17984551', '.', '-', '.', 'gene_id \"chr9.g11092\"; transcript_id \"chr9.g11092.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17985580', '17985863', '.', '+', '.', 'gene_id \"chr9.g11094\"; transcript_id \"chr9.g11094.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17986368', '17986587', '.', '-', '.', 'gene_id \"chr9.g11097\"; transcript_id \"chr9.g11097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17982733', '17983555', '.', '+', '.', 'gene_id \"chr9.g11099\"; transcript_id \"chr9.g11099.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17991685', '17992048', '.', '+', '.', 'gene_id \"chr9.g11100\"; transcript_id \"chr9.g11100.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18025892', '18026213', '.', '-', '.', 'gene_id \"chr9.g11101\"; transcript_id \"chr9.g11101.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18029953', '18030164', '.', '+', '.', 'gene_id \"chr9.g11102\"; transcript_id \"chr9.g11102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17977142', '17977294', '.', '+', '.', 'gene_id \"chr9.g11103\"; transcript_id \"chr9.g11103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18033307', '18033363', '.', '-', '.', 'gene_id \"chr9.g11104\"; transcript_id \"chr9.g11104.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17981611', '17982207', '.', '-', '.', 'gene_id \"chr9.g11108\"; transcript_id \"chr9.g11108.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17986611', '17987248', '.', '+', '.', 'gene_id \"chr9.g11109\"; transcript_id \"chr9.g11109.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17987521', '17987758', '.', '+', '.', 'gene_id \"chr9.g11111\"; transcript_id \"chr9.g11111.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17989519', '17989841', '.', '+', '.', 'gene_id \"chr9.g11116\"; transcript_id \"chr9.g11116.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17998236', '17998454', '.', '+', '.', 'gene_id \"chr9.g11120\"; transcript_id \"chr9.g11120.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17999196', '17999332', '.', '-', '.', 'gene_id \"chr9.g11121\"; transcript_id \"chr9.g11121.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17998982', '17999202', '.', '+', '.', 'gene_id \"chr9.g11122\"; transcript_id \"chr9.g11122.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18000024', '18000284', '.', '+', '.', 'gene_id \"chr9.g11124\"; transcript_id \"chr9.g11124.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18020027', '18020283', '.', '-', '.', 'gene_id \"chr9.g11145\"; transcript_id \"chr9.g11145.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18024155', '18024390', '.', '-', '.', 'gene_id \"chr9.g11153\"; transcript_id \"chr9.g11153.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18027995', '18028184', '.', '+', '.', 'gene_id \"chr9.g11157\"; transcript_id \"chr9.g11157.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18031176', '18031464', '.', '-', '.', 'gene_id \"chr9.g11162\"; transcript_id \"chr9.g11162.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18093886', '18094051', '.', '-', '.', 'gene_id \"chr9.g11180\"; transcript_id \"chr9.g11180.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18112649', '18113315', '.', '-', '.', 'gene_id \"chr9.g11186\"; transcript_id \"chr9.g11186.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18128261', '18128973', '.', '+', '.', 'gene_id \"chr9.g11187\"; transcript_id \"chr9.g11187.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18123310', '18123465', '.', '-', '.', 'gene_id \"chr9.g11190\"; transcript_id \"chr9.g11190.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18125794', '18126409', '.', '-', '.', 'gene_id \"chr9.g11192\"; transcript_id \"chr9.g11192.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18125375', '18125618', '.', '-', '.', 'gene_id \"chr9.g11193\"; transcript_id \"chr9.g11193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18123445', '18123758', '.', '+', '.', 'gene_id \"chr9.g11195\"; transcript_id \"chr9.g11195.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18121406', '18122076', '.', '-', '.', 'gene_id \"chr9.g11197\"; transcript_id \"chr9.g11197.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18174746', '18174874', '.', '-', '.', 'gene_id \"chr9.g11202\"; transcript_id \"chr9.g11202.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18179658', '18180319', '.', '-', '.', 'gene_id \"chr9.g11206\"; transcript_id \"chr9.g11206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18180457', '18180628', '.', '-', '.', 'gene_id \"chr9.g11208\"; transcript_id \"chr9.g11208.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18178814', '18178908', '.', '-', '.', 'gene_id \"chr9.g11209\"; transcript_id \"chr9.g11209.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18175077', '18175306', '.', '-', '.', 'gene_id \"chr9.g11210\"; transcript_id \"chr9.g11210.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18170747', '18170902', '.', '-', '.', 'gene_id \"chr9.g11212\"; transcript_id \"chr9.g11212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18156964', '18157127', '.', '+', '.', 'gene_id \"chr9.g11213\"; transcript_id \"chr9.g11213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18145044', '18145220', '.', '-', '.', 'gene_id \"chr9.g11217\"; transcript_id \"chr9.g11217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18156648', '18156847', '.', '+', '.', 'gene_id \"chr9.g11222\"; transcript_id \"chr9.g11222.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18160415', '18160842', '.', '+', '.', 'gene_id \"chr9.g11225\"; transcript_id \"chr9.g11225.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18160415', '18160836', '.', '+', '.', 'gene_id \"chr9.g11225\"; transcript_id \"chr9.g11225.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18164846', '18165125', '.', '-', '.', 'gene_id \"chr9.g11231\"; transcript_id \"chr9.g11231.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18166058', '18166408', '.', '-', '.', 'gene_id \"chr9.g11235\"; transcript_id \"chr9.g11235.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18176748', '18176853', '.', '+', '.', 'gene_id \"chr9.g11243\"; transcript_id \"chr9.g11243.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18178351', '18178478', '.', '+', '.', 'gene_id \"chr9.g11246\"; transcript_id \"chr9.g11246.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18181718', '18181934', '.', '-', '.', 'gene_id \"chr9.g11249\"; transcript_id \"chr9.g11249.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18185825', '18186035', '.', '-', '.', 'gene_id \"chr9.g11252\"; transcript_id \"chr9.g11252.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18186065', '18186234', '.', '-', '.', 'gene_id \"chr9.g11253\"; transcript_id \"chr9.g11253.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18186900', '18187094', '.', '+', '.', 'gene_id \"chr9.g11254\"; transcript_id \"chr9.g11254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18188730', '18189786', '.', '+', '.', 'gene_id \"chr9.g11255\"; transcript_id \"chr9.g11255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18193495', '18193649', '.', '-', '.', 'gene_id \"chr9.g11260\"; transcript_id \"chr9.g11260.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18195947', '18196097', '.', '-', '.', 'gene_id \"chr9.g11262\"; transcript_id \"chr9.g11262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18206027', '18206200', '.', '+', '.', 'gene_id \"chr9.g11265\"; transcript_id \"chr9.g11265.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18222733', '18223057', '.', '+', '.', 'gene_id \"chr9.g11268\"; transcript_id \"chr9.g11268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18258755', '18258936', '.', '-', '.', 'gene_id \"chr9.g11279\"; transcript_id \"chr9.g11279.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18263525', '18263678', '.', '+', '.', 'gene_id \"chr9.g11281\"; transcript_id \"chr9.g11281.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18279485', '18279641', '.', '-', '.', 'gene_id \"chr9.g11284\"; transcript_id \"chr9.g11284.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18280078', '18280280', '.', '+', '.', 'gene_id \"chr9.g11286\"; transcript_id \"chr9.g11286.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18280263', '18280419', '.', '-', '.', 'gene_id \"chr9.g11287\"; transcript_id \"chr9.g11287.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18280893', '18281095', '.', '+', '.', 'gene_id \"chr9.g11290\"; transcript_id \"chr9.g11290.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18290411', '18290985', '.', '-', '.', 'gene_id \"chr9.g11292\"; transcript_id \"chr9.g11292.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18297787', '18298789', '.', '+', '.', 'gene_id \"chr9.g11294\"; transcript_id \"chr9.g11294.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18286762', '18286981', '.', '+', '.', 'gene_id \"chr9.g11297\"; transcript_id \"chr9.g11297.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18286986', '18287279', '.', '+', '.', 'gene_id \"chr9.g11298\"; transcript_id \"chr9.g11298.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18287384', '18287582', '.', '+', '.', 'gene_id \"chr9.g11299\"; transcript_id \"chr9.g11299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18287583', '18287825', '.', '-', '.', 'gene_id \"chr9.g11300\"; transcript_id \"chr9.g11300.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18287876', '18288043', '.', '+', '.', 'gene_id \"chr9.g11301\"; transcript_id \"chr9.g11301.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18298997', '18299660', '.', '+', '.', 'gene_id \"chr9.g11304\"; transcript_id \"chr9.g11304.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18299967', '18300102', '.', '+', '.', 'gene_id \"chr9.g11306\"; transcript_id \"chr9.g11306.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18301304', '18301570', '.', '-', '.', 'gene_id \"chr9.g11309\"; transcript_id \"chr9.g11309.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18315583', '18315766', '.', '-', '.', 'gene_id \"chr9.g11313\"; transcript_id \"chr9.g11313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18334001', '18334177', '.', '+', '.', 'gene_id \"chr9.g11320\"; transcript_id \"chr9.g11320.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18333138', '18333796', '.', '-', '.', 'gene_id \"chr9.g11324\"; transcript_id \"chr9.g11324.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '23677897', '23677958', '.', '-', '.', 'gene_id \"chr17.g8172\"; transcript_id \"chr17.g8172.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18409578', '18409646', '.', '+', '.', 'gene_id \"chr9.g11332\"; transcript_id \"chr9.g11332.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18445920', '18446228', '.', '+', '.', 'gene_id \"chr9.g11344\"; transcript_id \"chr9.g11344.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18452985', '18453017', '.', '-', '.', 'gene_id \"chr9.g11347\"; transcript_id \"chr9.g11347.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18475590', '18475907', '.', '+', '.', 'gene_id \"chr9.g11352\"; transcript_id \"chr9.g11352.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18568553', '18568695', '.', '+', '.', 'gene_id \"chr9.g11352\"; transcript_id \"chr9.g11352.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18540070', '18540302', '.', '-', '.', 'gene_id \"chr9.g11358\"; transcript_id \"chr9.g11358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18540541', '18541675', '.', '+', '.', 'gene_id \"chr9.g11362\"; transcript_id \"chr9.g11362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18476959', '18477180', '.', '-', '.', 'gene_id \"chr9.g11364\"; transcript_id \"chr9.g11364.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18480429', '18480831', '.', '-', '.', 'gene_id \"chr9.g11365\"; transcript_id \"chr9.g11365.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18486341', '18486945', '.', '+', '.', 'gene_id \"chr9.g11370\"; transcript_id \"chr9.g11370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18487092', '18487334', '.', '+', '.', 'gene_id \"chr9.g11371\"; transcript_id \"chr9.g11371.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18493960', '18494145', '.', '-', '.', 'gene_id \"chr9.g11373\"; transcript_id \"chr9.g11373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18505649', '18505872', '.', '+', '.', 'gene_id \"chr9.g11376\"; transcript_id \"chr9.g11376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18505955', '18506280', '.', '+', '.', 'gene_id \"chr9.g11378\"; transcript_id \"chr9.g11378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18507094', '18507321', '.', '+', '.', 'gene_id \"chr9.g11379\"; transcript_id \"chr9.g11379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18507374', '18507528', '.', '+', '.', 'gene_id \"chr9.g11380\"; transcript_id \"chr9.g11380.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18513921', '18514053', '.', '-', '.', 'gene_id \"chr9.g11395\"; transcript_id \"chr9.g11395.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18514770', '18514942', '.', '+', '.', 'gene_id \"chr9.g11397\"; transcript_id \"chr9.g11397.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18515987', '18516219', '.', '+', '.', 'gene_id \"chr9.g11399\"; transcript_id \"chr9.g11399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18518288', '18518489', '.', '-', '.', 'gene_id \"chr9.g11403\"; transcript_id \"chr9.g11403.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18518675', '18518889', '.', '-', '.', 'gene_id \"chr9.g11405\"; transcript_id \"chr9.g11405.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18518936', '18519122', '.', '+', '.', 'gene_id \"chr9.g11407\"; transcript_id \"chr9.g11407.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18521606', '18522367', '.', '-', '.', 'gene_id \"chr9.g11413\"; transcript_id \"chr9.g11413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18525031', '18525375', '.', '+', '.', 'gene_id \"chr9.g11422\"; transcript_id \"chr9.g11422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18525801', '18526022', '.', '+', '.', 'gene_id \"chr9.g11423\"; transcript_id \"chr9.g11423.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18526235', '18526635', '.', '+', '.', 'gene_id \"chr9.g11425\"; transcript_id \"chr9.g11425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18529620', '18529756', '.', '-', '.', 'gene_id \"chr9.g11430\"; transcript_id \"chr9.g11430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18529936', '18530377', '.', '+', '.', 'gene_id \"chr9.g11432\"; transcript_id \"chr9.g11432.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18530419', '18530493', '.', '-', '.', 'gene_id \"chr9.g11433\"; transcript_id \"chr9.g11433.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18530692', '18531021', '.', '-', '.', 'gene_id \"chr9.g11434\"; transcript_id \"chr9.g11434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18531671', '18532154', '.', '+', '.', 'gene_id \"chr9.g11437\"; transcript_id \"chr9.g11437.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18532121', '18532494', '.', '-', '.', 'gene_id \"chr9.g11438\"; transcript_id \"chr9.g11438.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '23995893', '23996152', '.', '+', '.', 'gene_id \"chr17.g8286\"; transcript_id \"chr17.g8286.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18535947', '18536032', '.', '+', '.', 'gene_id \"chr9.g11445\"; transcript_id \"chr9.g11445.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18536041', '18536469', '.', '-', '.', 'gene_id \"chr9.g11446\"; transcript_id \"chr9.g11446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18538479', '18538702', '.', '+', '.', 'gene_id \"chr9.g11448\"; transcript_id \"chr9.g11448.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18537837', '18538436', '.', '-', '.', 'gene_id \"chr9.g11449\"; transcript_id \"chr9.g11449.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '24007581', '24007780', '.', '+', '.', 'gene_id \"chr17.g8294\"; transcript_id \"chr17.g8294.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18541783', '18542278', '.', '+', '.', 'gene_id \"chr9.g11450\"; transcript_id \"chr9.g11450.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18545881', '18546270', '.', '+', '.', 'gene_id \"chr9.g11454\"; transcript_id \"chr9.g11454.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18546416', '18546699', '.', '+', '.', 'gene_id \"chr9.g11456\"; transcript_id \"chr9.g11456.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18548600', '18548809', '.', '-', '.', 'gene_id \"chr9.g11458\"; transcript_id \"chr9.g11458.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18558156', '18558330', '.', '+', '.', 'gene_id \"chr9.g11468\"; transcript_id \"chr9.g11468.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18557968', '18558176', '.', '+', '.', 'gene_id \"chr9.g11468\"; transcript_id \"chr9.g11468.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18558476', '18558650', '.', '+', '.', 'gene_id \"chr9.g11470\"; transcript_id \"chr9.g11470.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18561890', '18561931', '.', '-', '.', 'gene_id \"chr9.g11474\"; transcript_id \"chr9.g11474.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18572390', '18572839', '.', '+', '.', 'gene_id \"chr9.g11484\"; transcript_id \"chr9.g11484.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18574418', '18574613', '.', '-', '.', 'gene_id \"chr9.g11487\"; transcript_id \"chr9.g11487.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18579890', '18580425', '.', '+', '.', 'gene_id \"chr9.g11495\"; transcript_id \"chr9.g11495.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18581351', '18581404', '.', '+', '.', 'gene_id \"chr9.g11500\"; transcript_id \"chr9.g11500.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18581223', '18581353', '.', '+', '.', 'gene_id \"chr9.g11502\"; transcript_id \"chr9.g11502.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18597225', '18597469', '.', '-', '.', 'gene_id \"chr9.g11510\"; transcript_id \"chr9.g11510.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18599857', '18600008', '.', '-', '.', 'gene_id \"chr9.g11512\"; transcript_id \"chr9.g11512.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18598501', '18598700', '.', '-', '.', 'gene_id \"chr9.g11513\"; transcript_id \"chr9.g11513.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18615500', '18615688', '.', '+', '.', 'gene_id \"chr9.g11520\"; transcript_id \"chr9.g11520.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18617274', '18617702', '.', '-', '.', 'gene_id \"chr9.g11522\"; transcript_id \"chr9.g11522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18619684', '18619849', '.', '-', '.', 'gene_id \"chr9.g11523\"; transcript_id \"chr9.g11523.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18620414', '18620601', '.', '+', '.', 'gene_id \"chr9.g11526\"; transcript_id \"chr9.g11526.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18654181', '18654402', '.', '-', '.', 'gene_id \"chr9.g11543\"; transcript_id \"chr9.g11543.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18652310', '18652580', '.', '-', '.', 'gene_id \"chr9.g11545\"; transcript_id \"chr9.g11545.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18660790', '18660967', '.', '-', '.', 'gene_id \"chr9.g11551\"; transcript_id \"chr9.g11551.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18664881', '18665169', '.', '-', '.', 'gene_id \"chr9.g11553\"; transcript_id \"chr9.g11553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18710179', '18710448', '.', '+', '.', 'gene_id \"chr9.g11563\"; transcript_id \"chr9.g11563.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18734064', '18734264', '.', '+', '.', 'gene_id \"chr9.g11575\"; transcript_id \"chr9.g11575.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18723249', '18723721', '.', '-', '.', 'gene_id \"chr9.g11577\"; transcript_id \"chr9.g11577.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18729553', '18729966', '.', '+', '.', 'gene_id \"chr9.g11578\"; transcript_id \"chr9.g11578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18723039', '18723262', '.', '+', '.', 'gene_id \"chr9.g11581\"; transcript_id \"chr9.g11581.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18736909', '18737020', '.', '-', '.', 'gene_id \"chr9.g11582\"; transcript_id \"chr9.g11582.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18735871', '18736085', '.', '+', '.', 'gene_id \"chr9.g11583\"; transcript_id \"chr9.g11583.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18735191', '18735408', '.', '-', '.', 'gene_id \"chr9.g11584\"; transcript_id \"chr9.g11584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18732522', '18732688', '.', '-', '.', 'gene_id \"chr9.g11586\"; transcript_id \"chr9.g11586.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18756384', '18756440', '.', '-', '.', 'gene_id \"chr9.g11594\"; transcript_id \"chr9.g11594.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18754848', '18755868', '.', '-', '.', 'gene_id \"chr9.g11594\"; transcript_id \"chr9.g11594.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18745867', '18746175', '.', '-', '.', 'gene_id \"chr9.g11601\"; transcript_id \"chr9.g11601.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18750365', '18750625', '.', '+', '.', 'gene_id \"chr9.g11602\"; transcript_id \"chr9.g11602.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18748772', '18749003', '.', '+', '.', 'gene_id \"chr9.g11603\"; transcript_id \"chr9.g11603.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18763861', '18764125', '.', '+', '.', 'gene_id \"chr9.g11607\"; transcript_id \"chr9.g11607.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18766251', '18767029', '.', '+', '.', 'gene_id \"chr9.g11609\"; transcript_id \"chr9.g11609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18767030', '18767140', '.', '-', '.', 'gene_id \"chr9.g11610\"; transcript_id \"chr9.g11610.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18770055', '18770264', '.', '-', '.', 'gene_id \"chr9.g11615\"; transcript_id \"chr9.g11615.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18773836', '18774028', '.', '-', '.', 'gene_id \"chr9.g11618\"; transcript_id \"chr9.g11618.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18788315', '18788498', '.', '-', '.', 'gene_id \"chr9.g11626\"; transcript_id \"chr9.g11626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18845873', '18846042', '.', '+', '.', 'gene_id \"chr9.g11630\"; transcript_id \"chr9.g11630.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18845575', '18845751', '.', '+', '.', 'gene_id \"chr9.g11630\"; transcript_id \"chr9.g11630.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18822500', '18823039', '.', '+', '.', 'gene_id \"chr9.g11630\"; transcript_id \"chr9.g11630.i15\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18823021', '18823220', '.', '-', '.', 'gene_id \"chr9.g11635\"; transcript_id \"chr9.g11635.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18844764', '18845147', '.', '+', '.', 'gene_id \"chr9.g11637\"; transcript_id \"chr9.g11637.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18834406', '18834706', '.', '+', '.', 'gene_id \"chr9.g11642\"; transcript_id \"chr9.g11642.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18841778', '18841928', '.', '+', '.', 'gene_id \"chr9.g11643\"; transcript_id \"chr9.g11643.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18843160', '18843437', '.', '+', '.', 'gene_id \"chr9.g11644\"; transcript_id \"chr9.g11644.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18843418', '18843893', '.', '-', '.', 'gene_id \"chr9.g11645\"; transcript_id \"chr9.g11645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18845328', '18845506', '.', '-', '.', 'gene_id \"chr9.g11647\"; transcript_id \"chr9.g11647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18845267', '18845532', '.', '+', '.', 'gene_id \"chr9.g11648\"; transcript_id \"chr9.g11648.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18815880', '18816213', '.', '+', '.', 'gene_id \"chr9.g11651\"; transcript_id \"chr9.g11651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18816226', '18816543', '.', '+', '.', 'gene_id \"chr9.g11652\"; transcript_id \"chr9.g11652.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18818090', '18818346', '.', '-', '.', 'gene_id \"chr9.g11653\"; transcript_id \"chr9.g11653.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18819887', '18820095', '.', '-', '.', 'gene_id \"chr9.g11656\"; transcript_id \"chr9.g11656.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18821805', '18822003', '.', '+', '.', 'gene_id \"chr9.g11659\"; transcript_id \"chr9.g11659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18823964', '18824178', '.', '-', '.', 'gene_id \"chr9.g11662\"; transcript_id \"chr9.g11662.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18824897', '18825207', '.', '+', '.', 'gene_id \"chr9.g11663\"; transcript_id \"chr9.g11663.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18826130', '18827044', '.', '+', '.', 'gene_id \"chr9.g11665\"; transcript_id \"chr9.g11665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18836220', '18836548', '.', '+', '.', 'gene_id \"chr9.g11666\"; transcript_id \"chr9.g11666.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18837413', '18837611', '.', '+', '.', 'gene_id \"chr9.g11669\"; transcript_id \"chr9.g11669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18837740', '18837906', '.', '-', '.', 'gene_id \"chr9.g11670\"; transcript_id \"chr9.g11670.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18837907', '18838134', '.', '+', '.', 'gene_id \"chr9.g11671\"; transcript_id \"chr9.g11671.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18841365', '18841515', '.', '+', '.', 'gene_id \"chr9.g11673\"; transcript_id \"chr9.g11673.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18874784', '18875328', '.', '+', '.', 'gene_id \"chr9.g11674\"; transcript_id \"chr9.g11674.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18850174', '18850372', '.', '-', '.', 'gene_id \"chr9.g11677\"; transcript_id \"chr9.g11677.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18856617', '18856891', '.', '-', '.', 'gene_id \"chr9.g11678\"; transcript_id \"chr9.g11678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18848690', '18848896', '.', '-', '.', 'gene_id \"chr9.g11679\"; transcript_id \"chr9.g11679.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18873646', '18874607', '.', '+', '.', 'gene_id \"chr9.g11684\"; transcript_id \"chr9.g11684.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18855836', '18856120', '.', '+', '.', 'gene_id \"chr9.g11688\"; transcript_id \"chr9.g11688.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18859768', '18860184', '.', '+', '.', 'gene_id \"chr9.g11692\"; transcript_id \"chr9.g11692.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18859777', '18860184', '.', '+', '.', 'gene_id \"chr9.g11692\"; transcript_id \"chr9.g11692.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18860165', '18860347', '.', '-', '.', 'gene_id \"chr9.g11693\"; transcript_id \"chr9.g11693.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18864732', '18864921', '.', '-', '.', 'gene_id \"chr9.g11697\"; transcript_id \"chr9.g11697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18869492', '18870087', '.', '-', '.', 'gene_id \"chr9.g11702\"; transcript_id \"chr9.g11702.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18870123', '18870524', '.', '-', '.', 'gene_id \"chr9.g11703\"; transcript_id \"chr9.g11703.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18871950', '18872367', '.', '-', '.', 'gene_id \"chr9.g11709\"; transcript_id \"chr9.g11709.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18888985', '18889218', '.', '+', '.', 'gene_id \"chr9.g11720\"; transcript_id \"chr9.g11720.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18980554', '18980715', '.', '+', '.', 'gene_id \"chr9.g11743\"; transcript_id \"chr9.g11743.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18978139', '18978437', '.', '+', '.', 'gene_id \"chr9.g11767\"; transcript_id \"chr9.g11767.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18996002', '18996184', '.', '-', '.', 'gene_id \"chr9.g11777\"; transcript_id \"chr9.g11777.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18998776', '18999633', '.', '-', '.', 'gene_id \"chr9.g11780\"; transcript_id \"chr9.g11780.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18998340', '18998769', '.', '-', '.', 'gene_id \"chr9.g11780\"; transcript_id \"chr9.g11780.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18996977', '18997376', '.', '+', '.', 'gene_id \"chr9.g11783\"; transcript_id \"chr9.g11783.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19009902', '19010054', '.', '-', '.', 'gene_id \"chr9.g11784\"; transcript_id \"chr9.g11784.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19000720', '19001227', '.', '+', '.', 'gene_id \"chr9.g11786\"; transcript_id \"chr9.g11786.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19001905', '19002227', '.', '+', '.', 'gene_id \"chr9.g11787\"; transcript_id \"chr9.g11787.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19002205', '19002961', '.', '-', '.', 'gene_id \"chr9.g11788\"; transcript_id \"chr9.g11788.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19024903', '19025108', '.', '+', '.', 'gene_id \"chr9.g11794\"; transcript_id \"chr9.g11794.i22\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19042514', '19042824', '.', '+', '.', 'gene_id \"chr9.g11795\"; transcript_id \"chr9.g11795.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19044807', '19045007', '.', '-', '.', 'gene_id \"chr9.g11797\"; transcript_id \"chr9.g11797.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19025585', '19025652', '.', '-', '.', 'gene_id \"chr9.g11798\"; transcript_id \"chr9.g11798.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19044329', '19044501', '.', '-', '.', 'gene_id \"chr9.g11800\"; transcript_id \"chr9.g11800.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19029600', '19029731', '.', '-', '.', 'gene_id \"chr9.g11802\"; transcript_id \"chr9.g11802.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19046054', '19046229', '.', '+', '.', 'gene_id \"chr9.g11815\"; transcript_id \"chr9.g11815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19046245', '19046592', '.', '+', '.', 'gene_id \"chr9.g11816\"; transcript_id \"chr9.g11816.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19050102', '19050348', '.', '-', '.', 'gene_id \"chr9.g11818\"; transcript_id \"chr9.g11818.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19053917', '19054171', '.', '-', '.', 'gene_id \"chr9.g11824\"; transcript_id \"chr9.g11824.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19060563', '19060718', '.', '+', '.', 'gene_id \"chr9.g11827\"; transcript_id \"chr9.g11827.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19067023', '19067880', '.', '+', '.', 'gene_id \"chr9.g11835\"; transcript_id \"chr9.g11835.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19069669', '19070186', '.', '-', '.', 'gene_id \"chr9.g11841\"; transcript_id \"chr9.g11841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19070734', '19071843', '.', '+', '.', 'gene_id \"chr9.g11843\"; transcript_id \"chr9.g11843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19071827', '19072568', '.', '-', '.', 'gene_id \"chr9.g11844\"; transcript_id \"chr9.g11844.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19110414', '19110607', '.', '+', '.', 'gene_id \"chr9.g11852\"; transcript_id \"chr9.g11852.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19168468', '19168824', '.', '+', '.', 'gene_id \"chr9.g11857\"; transcript_id \"chr9.g11857.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19168825', '19169367', '.', '+', '.', 'gene_id \"chr9.g11859\"; transcript_id \"chr9.g11859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19188571', '19189081', '.', '-', '.', 'gene_id \"chr9.g11861\"; transcript_id \"chr9.g11861.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19169982', '19170149', '.', '-', '.', 'gene_id \"chr9.g11863\"; transcript_id \"chr9.g11863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19181988', '19182242', '.', '+', '.', 'gene_id \"chr9.g11868\"; transcript_id \"chr9.g11868.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19230493', '19230635', '.', '+', '.', 'gene_id \"chr9.g11870\"; transcript_id \"chr9.g11870.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19257974', '19258003', '.', '+', '.', 'gene_id \"chr9.g11873\"; transcript_id \"chr9.g11873.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19276422', '19276579', '.', '-', '.', 'gene_id \"chr9.g11875\"; transcript_id \"chr9.g11875.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19285333', '19285561', '.', '-', '.', 'gene_id \"chr9.g11885\"; transcript_id \"chr9.g11885.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19288390', '19288756', '.', '+', '.', 'gene_id \"chr9.g11886\"; transcript_id \"chr9.g11886.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19294189', '19294366', '.', '+', '.', 'gene_id \"chr9.g11888\"; transcript_id \"chr9.g11888.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19286615', '19286735', '.', '+', '.', 'gene_id \"chr9.g11892\"; transcript_id \"chr9.g11892.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19286772', '19288049', '.', '+', '.', 'gene_id \"chr9.g11894\"; transcript_id \"chr9.g11894.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19302143', '19302312', '.', '+', '.', 'gene_id \"chr9.g11906\"; transcript_id \"chr9.g11906.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19304072', '19304291', '.', '-', '.', 'gene_id \"chr9.g11907\"; transcript_id \"chr9.g11907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19304693', '19304802', '.', '+', '.', 'gene_id \"chr9.g11908\"; transcript_id \"chr9.g11908.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19304694', '19304802', '.', '+', '.', 'gene_id \"chr9.g11908\"; transcript_id \"chr9.g11908.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19304297', '19304713', '.', '-', '.', 'gene_id \"chr9.g11909\"; transcript_id \"chr9.g11909.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19310208', '19310397', '.', '-', '.', 'gene_id \"chr9.g11911\"; transcript_id \"chr9.g11911.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19380729', '19382512', '.', '-', '.', 'gene_id \"chr9.g11932\"; transcript_id \"chr9.g11932.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19395477', '19395840', '.', '-', '.', 'gene_id \"chr9.g11940\"; transcript_id \"chr9.g11940.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19420310', '19420564', '.', '-', '.', 'gene_id \"chr9.g11950\"; transcript_id \"chr9.g11950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19440583', '19440825', '.', '-', '.', 'gene_id \"chr9.g11961\"; transcript_id \"chr9.g11961.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19440671', '19440825', '.', '-', '.', 'gene_id \"chr9.g11961\"; transcript_id \"chr9.g11961.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19440892', '19441566', '.', '-', '.', 'gene_id \"chr9.g11961\"; transcript_id \"chr9.g11961.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19444931', '19445611', '.', '+', '.', 'gene_id \"chr9.g11962\"; transcript_id \"chr9.g11962.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19431900', '19432972', '.', '-', '.', 'gene_id \"chr9.g11963\"; transcript_id \"chr9.g11963.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19435374', '19435999', '.', '-', '.', 'gene_id \"chr9.g11964\"; transcript_id \"chr9.g11964.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19435943', '19436255', '.', '-', '.', 'gene_id \"chr9.g11964\"; transcript_id \"chr9.g11964.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19439755', '19440241', '.', '+', '.', 'gene_id \"chr9.g11965\"; transcript_id \"chr9.g11965.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19439633', '19439786', '.', '+', '.', 'gene_id \"chr9.g11965\"; transcript_id \"chr9.g11965.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19458372', '19458459', '.', '+', '.', 'gene_id \"chr9.g11968\"; transcript_id \"chr9.g11968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19452125', '19452794', '.', '-', '.', 'gene_id \"chr9.g11971\"; transcript_id \"chr9.g11971.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19458137', '19458308', '.', '+', '.', 'gene_id \"chr9.g11973\"; transcript_id \"chr9.g11973.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19515334', '19515520', '.', '-', '.', 'gene_id \"chr9.g11974\"; transcript_id \"chr9.g11974.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19442345', '19442857', '.', '+', '.', 'gene_id \"chr9.g11978\"; transcript_id \"chr9.g11978.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19436654', '19436953', '.', '+', '.', 'gene_id \"chr9.g11979\"; transcript_id \"chr9.g11979.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19443364', '19443745', '.', '+', '.', 'gene_id \"chr9.g11980\"; transcript_id \"chr9.g11980.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19455455', '19455798', '.', '-', '.', 'gene_id \"chr9.g11985\"; transcript_id \"chr9.g11985.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19449248', '19449414', '.', '+', '.', 'gene_id \"chr9.g11988\"; transcript_id \"chr9.g11988.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19441599', '19442127', '.', '-', '.', 'gene_id \"chr9.g11989\"; transcript_id \"chr9.g11989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19436271', '19436675', '.', '+', '.', 'gene_id \"chr9.g11996\"; transcript_id \"chr9.g11996.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19438268', '19438962', '.', '+', '.', 'gene_id \"chr9.g11997\"; transcript_id \"chr9.g11997.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19432998', '19433354', '.', '+', '.', 'gene_id \"chr9.g12000\"; transcript_id \"chr9.g12000.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19433432', '19434462', '.', '-', '.', 'gene_id \"chr9.g12001\"; transcript_id \"chr9.g12001.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19434441', '19434559', '.', '+', '.', 'gene_id \"chr9.g12002\"; transcript_id \"chr9.g12002.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19447947', '19448143', '.', '+', '.', 'gene_id \"chr9.g12003\"; transcript_id \"chr9.g12003.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19451103', '19451571', '.', '-', '.', 'gene_id \"chr9.g12006\"; transcript_id \"chr9.g12006.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19451575', '19451812', '.', '-', '.', 'gene_id \"chr9.g12007\"; transcript_id \"chr9.g12007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19452875', '19453286', '.', '-', '.', 'gene_id \"chr9.g12008\"; transcript_id \"chr9.g12008.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19454906', '19455103', '.', '-', '.', 'gene_id \"chr9.g12012\"; transcript_id \"chr9.g12012.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19486587', '19486764', '.', '-', '.', 'gene_id \"chr9.g12022\"; transcript_id \"chr9.g12022.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19487636', '19487917', '.', '-', '.', 'gene_id \"chr9.g12025\"; transcript_id \"chr9.g12025.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19514275', '19514435', '.', '-', '.', 'gene_id \"chr9.g12029\"; transcript_id \"chr9.g12029.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19517715', '19517899', '.', '+', '.', 'gene_id \"chr9.g12032\"; transcript_id \"chr9.g12032.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19529548', '19529699', '.', '+', '.', 'gene_id \"chr9.g12033\"; transcript_id \"chr9.g12033.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19538591', '19539901', '.', '+', '.', 'gene_id \"chr9.g12034\"; transcript_id \"chr9.g12034.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19546013', '19546188', '.', '+', '.', 'gene_id \"chr9.g12044\"; transcript_id \"chr9.g12044.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19547666', '19549182', '.', '+', '.', 'gene_id \"chr9.g12046\"; transcript_id \"chr9.g12046.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19551280', '19551463', '.', '-', '.', 'gene_id \"chr9.g12049\"; transcript_id \"chr9.g12049.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19550571', '19550953', '.', '-', '.', 'gene_id \"chr9.g12050\"; transcript_id \"chr9.g12050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19558849', '19558973', '.', '+', '.', 'gene_id \"chr9.g12059\"; transcript_id \"chr9.g12059.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19562728', '19562938', '.', '-', '.', 'gene_id \"chr9.g12063\"; transcript_id \"chr9.g12063.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '25291373', '25291586', '.', '+', '.', 'gene_id \"chr17.g8825\"; transcript_id \"chr17.g8825.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19570946', '19571374', '.', '-', '.', 'gene_id \"chr9.g12069\"; transcript_id \"chr9.g12069.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19578153', '19578362', '.', '-', '.', 'gene_id \"chr9.g12070\"; transcript_id \"chr9.g12070.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19567802', '19567857', '.', '+', '.', 'gene_id \"chr9.g12071\"; transcript_id \"chr9.g12071.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19567330', '19567809', '.', '+', '.', 'gene_id \"chr9.g12071\"; transcript_id \"chr9.g12071.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19570132', '19570410', '.', '+', '.', 'gene_id \"chr9.g12072\"; transcript_id \"chr9.g12072.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19568555', '19568824', '.', '-', '.', 'gene_id \"chr9.g12074\"; transcript_id \"chr9.g12074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19573923', '19574151', '.', '+', '.', 'gene_id \"chr9.g12076\"; transcript_id \"chr9.g12076.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19568954', '19570093', '.', '-', '.', 'gene_id \"chr9.g12077\"; transcript_id \"chr9.g12077.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19573671', '19573853', '.', '-', '.', 'gene_id \"chr9.g12084\"; transcript_id \"chr9.g12084.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19582368', '19582634', '.', '-', '.', 'gene_id \"chr9.g12093\"; transcript_id \"chr9.g12093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19584624', '19584808', '.', '-', '.', 'gene_id \"chr9.g12096\"; transcript_id \"chr9.g12096.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19604170', '19604449', '.', '+', '.', 'gene_id \"chr9.g12102\"; transcript_id \"chr9.g12102.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19594661', '19594853', '.', '+', '.', 'gene_id \"chr9.g12102\"; transcript_id \"chr9.g12102.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19604092', '19604191', '.', '-', '.', 'gene_id \"chr9.g12103\"; transcript_id \"chr9.g12103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19602696', '19603399', '.', '+', '.', 'gene_id \"chr9.g12105\"; transcript_id \"chr9.g12105.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19600201', '19600460', '.', '+', '.', 'gene_id \"chr9.g12107\"; transcript_id \"chr9.g12107.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19597236', '19597451', '.', '+', '.', 'gene_id \"chr9.g12108\"; transcript_id \"chr9.g12108.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19600953', '19601217', '.', '-', '.', 'gene_id \"chr9.g12110\"; transcript_id \"chr9.g12110.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19603781', '19603970', '.', '-', '.', 'gene_id \"chr9.g12113\"; transcript_id \"chr9.g12113.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19607444', '19607637', '.', '+', '.', 'gene_id \"chr9.g12114\"; transcript_id \"chr9.g12114.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19611860', '19612102', '.', '+', '.', 'gene_id \"chr9.g12117\"; transcript_id \"chr9.g12117.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19627843', '19627994', '.', '-', '.', 'gene_id \"chr9.g12120\"; transcript_id \"chr9.g12120.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19628002', '19628222', '.', '-', '.', 'gene_id \"chr9.g12121\"; transcript_id \"chr9.g12121.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19633816', '19633853', '.', '+', '.', 'gene_id \"chr9.g12131\"; transcript_id \"chr9.g12131.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19633169', '19633551', '.', '-', '.', 'gene_id \"chr9.g12136\"; transcript_id \"chr9.g12136.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19639721', '19639841', '.', '+', '.', 'gene_id \"chr9.g12139\"; transcript_id \"chr9.g12139.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19644444', '19644783', '.', '-', '.', 'gene_id \"chr9.g12140\"; transcript_id \"chr9.g12140.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19644760', '19644865', '.', '-', '.', 'gene_id \"chr9.g12140\"; transcript_id \"chr9.g12140.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19658725', '19658938', '.', '+', '.', 'gene_id \"chr9.g12144\"; transcript_id \"chr9.g12144.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19656660', '19656885', '.', '+', '.', 'gene_id \"chr9.g12151\"; transcript_id \"chr9.g12151.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19671586', '19671670', '.', '-', '.', 'gene_id \"chr9.g12161\"; transcript_id \"chr9.g12161.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19678201', '19678666', '.', '-', '.', 'gene_id \"chr9.g12171\"; transcript_id \"chr9.g12171.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19675688', '19675988', '.', '+', '.', 'gene_id \"chr9.g12172\"; transcript_id \"chr9.g12172.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19679870', '19680038', '.', '+', '.', 'gene_id \"chr9.g12175\"; transcript_id \"chr9.g12175.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19683215', '19683530', '.', '-', '.', 'gene_id \"chr9.g12176\"; transcript_id \"chr9.g12176.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19687884', '19688293', '.', '+', '.', 'gene_id \"chr9.g12178\"; transcript_id \"chr9.g12178.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19720860', '19721018', '.', '-', '.', 'gene_id \"chr9.g12192\"; transcript_id \"chr9.g12192.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19709719', '19710005', '.', '-', '.', 'gene_id \"chr9.g12193\"; transcript_id \"chr9.g12193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19722838', '19724442', '.', '+', '.', 'gene_id \"chr9.g12196\"; transcript_id \"chr9.g12196.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19716535', '19717314', '.', '+', '.', 'gene_id \"chr9.g12197\"; transcript_id \"chr9.g12197.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19715364', '19716434', '.', '+', '.', 'gene_id \"chr9.g12198\"; transcript_id \"chr9.g12198.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19713799', '19715255', '.', '+', '.', 'gene_id \"chr9.g12199\"; transcript_id \"chr9.g12199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19713249', '19713696', '.', '+', '.', 'gene_id \"chr9.g12200\"; transcript_id \"chr9.g12200.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19712833', '19713253', '.', '-', '.', 'gene_id \"chr9.g12202\"; transcript_id \"chr9.g12202.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19735308', '19736105', '.', '-', '.', 'gene_id \"chr9.g12205\"; transcript_id \"chr9.g12205.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19729433', '19729725', '.', '-', '.', 'gene_id \"chr9.g12205\"; transcript_id \"chr9.g12205.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19737094', '19737376', '.', '-', '.', 'gene_id \"chr9.g12208\"; transcript_id \"chr9.g12208.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19726729', '19726980', '.', '-', '.', 'gene_id \"chr9.g12209\"; transcript_id \"chr9.g12209.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19732067', '19732268', '.', '+', '.', 'gene_id \"chr9.g12211\"; transcript_id \"chr9.g12211.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19732315', '19732858', '.', '+', '.', 'gene_id \"chr9.g12212\"; transcript_id \"chr9.g12212.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19733189', '19733736', '.', '+', '.', 'gene_id \"chr9.g12215\"; transcript_id \"chr9.g12215.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19733848', '19735208', '.', '+', '.', 'gene_id \"chr9.g12217\"; transcript_id \"chr9.g12217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19750279', '19750536', '.', '+', '.', 'gene_id \"chr9.g12222\"; transcript_id \"chr9.g12222.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19751342', '19752342', '.', '-', '.', 'gene_id \"chr9.g12226\"; transcript_id \"chr9.g12226.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19751342', '19752342', '.', '-', '.', 'gene_id \"chr9.g12226\"; transcript_id \"chr9.g12226.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19751342', '19752342', '.', '-', '.', 'gene_id \"chr9.g12226\"; transcript_id \"chr9.g12226.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19751342', '19752342', '.', '-', '.', 'gene_id \"chr9.g12226\"; transcript_id \"chr9.g12226.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19754115', '19754399', '.', '+', '.', 'gene_id \"chr9.g12229\"; transcript_id \"chr9.g12229.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19756537', '19757127', '.', '-', '.', 'gene_id \"chr9.g12232\"; transcript_id \"chr9.g12232.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19757106', '19757376', '.', '+', '.', 'gene_id \"chr9.g12233\"; transcript_id \"chr9.g12233.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19795241', '19795568', '.', '-', '.', 'gene_id \"chr9.g12235\"; transcript_id \"chr9.g12235.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19797978', '19798129', '.', '-', '.', 'gene_id \"chr9.g12235\"; transcript_id \"chr9.g12235.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19798156', '19798634', '.', '+', '.', 'gene_id \"chr9.g12237\"; transcript_id \"chr9.g12237.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19794507', '19794619', '.', '+', '.', 'gene_id \"chr9.g12238\"; transcript_id \"chr9.g12238.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19794507', '19794614', '.', '+', '.', 'gene_id \"chr9.g12238\"; transcript_id \"chr9.g12238.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19774175', '19774559', '.', '-', '.', 'gene_id \"chr9.g12239\"; transcript_id \"chr9.g12239.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19792515', '19792760', '.', '-', '.', 'gene_id \"chr9.g12243\"; transcript_id \"chr9.g12243.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19790608', '19790900', '.', '+', '.', 'gene_id \"chr9.g12244\"; transcript_id \"chr9.g12244.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19782592', '19783043', '.', '-', '.', 'gene_id \"chr9.g12246\"; transcript_id \"chr9.g12246.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19792894', '19793457', '.', '-', '.', 'gene_id \"chr9.g12247\"; transcript_id \"chr9.g12247.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19793510', '19793752', '.', '-', '.', 'gene_id \"chr9.g12248\"; transcript_id \"chr9.g12248.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19802403', '19803070', '.', '+', '.', 'gene_id \"chr9.g12249\"; transcript_id \"chr9.g12249.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19801028', '19801272', '.', '-', '.', 'gene_id \"chr9.g12251\"; transcript_id \"chr9.g12251.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19800881', '19801047', '.', '+', '.', 'gene_id \"chr9.g12252\"; transcript_id \"chr9.g12252.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19800653', '19800826', '.', '+', '.', 'gene_id \"chr9.g12253\"; transcript_id \"chr9.g12253.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19803353', '19803591', '.', '+', '.', 'gene_id \"chr9.g12254\"; transcript_id \"chr9.g12254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19806377', '19806671', '.', '+', '.', 'gene_id \"chr9.g12255\"; transcript_id \"chr9.g12255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19812154', '19812304', '.', '-', '.', 'gene_id \"chr9.g12275\"; transcript_id \"chr9.g12275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19825186', '19827313', '.', '+', '.', 'gene_id \"chr9.g12280\"; transcript_id \"chr9.g12280.i32\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19827900', '19828091', '.', '+', '.', 'gene_id \"chr9.g12282\"; transcript_id \"chr9.g12282.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19829386', '19829811', '.', '+', '.', 'gene_id \"chr9.g12287\"; transcript_id \"chr9.g12287.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19863880', '19864100', '.', '+', '.', 'gene_id \"chr9.g12297\"; transcript_id \"chr9.g12297.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19865552', '19866005', '.', '+', '.', 'gene_id \"chr9.g12298\"; transcript_id \"chr9.g12298.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19866066', '19866382', '.', '+', '.', 'gene_id \"chr9.g12299\"; transcript_id \"chr9.g12299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19865213', '19865362', '.', '+', '.', 'gene_id \"chr9.g12302\"; transcript_id \"chr9.g12302.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19868307', '19868877', '.', '-', '.', 'gene_id \"chr9.g12308\"; transcript_id \"chr9.g12308.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19887743', '19890372', '.', '-', '.', 'gene_id \"chr9.g12309\"; transcript_id \"chr9.g12309.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19885701', '19886228', '.', '-', '.', 'gene_id \"chr9.g12311\"; transcript_id \"chr9.g12311.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19871420', '19871501', '.', '+', '.', 'gene_id \"chr9.g12313\"; transcript_id \"chr9.g12313.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19871702', '19871953', '.', '+', '.', 'gene_id \"chr9.g12315\"; transcript_id \"chr9.g12315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19873371', '19873810', '.', '-', '.', 'gene_id \"chr9.g12318\"; transcript_id \"chr9.g12318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19873371', '19873784', '.', '-', '.', 'gene_id \"chr9.g12318\"; transcript_id \"chr9.g12318.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19873856', '19874210', '.', '+', '.', 'gene_id \"chr9.g12319\"; transcript_id \"chr9.g12319.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19876457', '19876613', '.', '-', '.', 'gene_id \"chr9.g12323\"; transcript_id \"chr9.g12323.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19876497', '19876613', '.', '-', '.', 'gene_id \"chr9.g12323\"; transcript_id \"chr9.g12323.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19876643', '19876830', '.', '-', '.', 'gene_id \"chr9.g12324\"; transcript_id \"chr9.g12324.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19876950', '19877113', '.', '-', '.', 'gene_id \"chr9.g12325\"; transcript_id \"chr9.g12325.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19878496', '19878650', '.', '-', '.', 'gene_id \"chr9.g12326\"; transcript_id \"chr9.g12326.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19878642', '19878832', '.', '+', '.', 'gene_id \"chr9.g12327\"; transcript_id \"chr9.g12327.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19901922', '19902131', '.', '+', '.', 'gene_id \"chr9.g12333\"; transcript_id \"chr9.g12333.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19898663', '19898894', '.', '-', '.', 'gene_id \"chr9.g12334\"; transcript_id \"chr9.g12334.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19897906', '19898302', '.', '+', '.', 'gene_id \"chr9.g12335\"; transcript_id \"chr9.g12335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19896898', '19897064', '.', '+', '.', 'gene_id \"chr9.g12337\"; transcript_id \"chr9.g12337.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19897071', '19897292', '.', '-', '.', 'gene_id \"chr9.g12338\"; transcript_id \"chr9.g12338.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19907742', '19907939', '.', '+', '.', 'gene_id \"chr9.g12342\"; transcript_id \"chr9.g12342.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19913512', '19914471', '.', '+', '.', 'gene_id \"chr9.g12346\"; transcript_id \"chr9.g12346.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19916957', '19917548', '.', '+', '.', 'gene_id \"chr9.g12347\"; transcript_id \"chr9.g12347.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19916366', '19916557', '.', '+', '.', 'gene_id \"chr9.g12348\"; transcript_id \"chr9.g12348.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19923286', '19923463', '.', '-', '.', 'gene_id \"chr9.g12353\"; transcript_id \"chr9.g12353.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19923543', '19923693', '.', '+', '.', 'gene_id \"chr9.g12354\"; transcript_id \"chr9.g12354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19934080', '19934370', '.', '-', '.', 'gene_id \"chr9.g12359\"; transcript_id \"chr9.g12359.i21\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19961420', '19961691', '.', '+', '.', 'gene_id \"chr9.g12362\"; transcript_id \"chr9.g12362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19934498', '19934673', '.', '+', '.', 'gene_id \"chr9.g12363\"; transcript_id \"chr9.g12363.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19945920', '19946248', '.', '-', '.', 'gene_id \"chr9.g12366\"; transcript_id \"chr9.g12366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19928813', '19931937', '.', '+', '.', 'gene_id \"chr9.g12372\"; transcript_id \"chr9.g12372.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19933664', '19934025', '.', '+', '.', 'gene_id \"chr9.g12374\"; transcript_id \"chr9.g12374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19941076', '19941752', '.', '-', '.', 'gene_id \"chr9.g12376\"; transcript_id \"chr9.g12376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19941419', '19941752', '.', '-', '.', 'gene_id \"chr9.g12376\"; transcript_id \"chr9.g12376.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19941076', '19941416', '.', '+', '.', 'gene_id \"chr9.g12377\"; transcript_id \"chr9.g12377.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19941731', '19942278', '.', '+', '.', 'gene_id \"chr9.g12378\"; transcript_id \"chr9.g12378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19942340', '19942496', '.', '-', '.', 'gene_id \"chr9.g12379\"; transcript_id \"chr9.g12379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19942526', '19942683', '.', '-', '.', 'gene_id \"chr9.g12381\"; transcript_id \"chr9.g12381.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '25578961', '25579175', '.', '-', '.', 'gene_id \"chr17.g9043\"; transcript_id \"chr17.g9043.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19957767', '19958091', '.', '+', '.', 'gene_id \"chr9.g12387\"; transcript_id \"chr9.g12387.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19958091', '19958337', '.', '-', '.', 'gene_id \"chr9.g12388\"; transcript_id \"chr9.g12388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19960239', '19960457', '.', '+', '.', 'gene_id \"chr9.g12391\"; transcript_id \"chr9.g12391.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19962150', '19962455', '.', '-', '.', 'gene_id \"chr9.g12394\"; transcript_id \"chr9.g12394.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19967107', '19967266', '.', '-', '.', 'gene_id \"chr9.g12396\"; transcript_id \"chr9.g12396.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19984624', '19984777', '.', '-', '.', 'gene_id \"chr9.g12407\"; transcript_id \"chr9.g12407.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19989704', '19989875', '.', '+', '.', 'gene_id \"chr9.g12409\"; transcript_id \"chr9.g12409.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20015908', '20016584', '.', '-', '.', 'gene_id \"chr9.g12423\"; transcript_id \"chr9.g12423.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20018904', '20019350', '.', '+', '.', 'gene_id \"chr9.g12428\"; transcript_id \"chr9.g12428.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20018686', '20018918', '.', '+', '.', 'gene_id \"chr9.g12430\"; transcript_id \"chr9.g12430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20020107', '20020257', '.', '+', '.', 'gene_id \"chr9.g12432\"; transcript_id \"chr9.g12432.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20019915', '20020107', '.', '-', '.', 'gene_id \"chr9.g12433\"; transcript_id \"chr9.g12433.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20020678', '20021078', '.', '+', '.', 'gene_id \"chr9.g12434\"; transcript_id \"chr9.g12434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20021342', '20021742', '.', '+', '.', 'gene_id \"chr9.g12435\"; transcript_id \"chr9.g12435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20024803', '20025134', '.', '-', '.', 'gene_id \"chr9.g12437\"; transcript_id \"chr9.g12437.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20027506', '20027682', '.', '+', '.', 'gene_id \"chr9.g12440\"; transcript_id \"chr9.g12440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20029113', '20030826', '.', '-', '.', 'gene_id \"chr9.g12447\"; transcript_id \"chr9.g12447.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20031602', '20033006', '.', '-', '.', 'gene_id \"chr9.g12448\"; transcript_id \"chr9.g12448.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '20031602', '20032769', '.', '-', '.', 'gene_id \"chr9.g12448\"; transcript_id \"chr9.g12448.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20044219', '20044465', '.', '+', '.', 'gene_id \"chr9.g12451\"; transcript_id \"chr9.g12451.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20109353', '20109661', '.', '+', '.', 'gene_id \"chr9.g12451\"; transcript_id \"chr9.g12451.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20105042', '20105091', '.', '+', '.', 'gene_id \"chr9.g12453\"; transcript_id \"chr9.g12453.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20113211', '20115565', '.', '-', '.', 'gene_id \"chr9.g12454\"; transcript_id \"chr9.g12454.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20115497', '20115565', '.', '-', '.', 'gene_id \"chr9.g12454\"; transcript_id \"chr9.g12454.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20115527', '20116414', '.', '-', '.', 'gene_id \"chr9.g12454\"; transcript_id \"chr9.g12454.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20117137', '20117513', '.', '-', '.', 'gene_id \"chr9.g12455\"; transcript_id \"chr9.g12455.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20112501', '20113041', '.', '+', '.', 'gene_id \"chr9.g12457\"; transcript_id \"chr9.g12457.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20099248', '20099471', '.', '+', '.', 'gene_id \"chr9.g12459\"; transcript_id \"chr9.g12459.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20110040', '20110783', '.', '-', '.', 'gene_id \"chr9.g12460\"; transcript_id \"chr9.g12460.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20105465', '20106244', '.', '-', '.', 'gene_id \"chr9.g12463\"; transcript_id \"chr9.g12463.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20045225', '20045576', '.', '+', '.', 'gene_id \"chr9.g12470\"; transcript_id \"chr9.g12470.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20045634', '20046198', '.', '+', '.', 'gene_id \"chr9.g12471\"; transcript_id \"chr9.g12471.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20046646', '20046828', '.', '+', '.', 'gene_id \"chr9.g12472\"; transcript_id \"chr9.g12472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20047592', '20047878', '.', '-', '.', 'gene_id \"chr9.g12475\"; transcript_id \"chr9.g12475.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20048823', '20049098', '.', '+', '.', 'gene_id \"chr9.g12478\"; transcript_id \"chr9.g12478.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20049079', '20049520', '.', '-', '.', 'gene_id \"chr9.g12479\"; transcript_id \"chr9.g12479.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20050916', '20051029', '.', '+', '.', 'gene_id \"chr9.g12482\"; transcript_id \"chr9.g12482.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20052328', '20052820', '.', '+', '.', 'gene_id \"chr9.g12484\"; transcript_id \"chr9.g12484.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20052836', '20053140', '.', '+', '.', 'gene_id \"chr9.g12486\"; transcript_id \"chr9.g12486.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20054723', '20055021', '.', '-', '.', 'gene_id \"chr9.g12489\"; transcript_id \"chr9.g12489.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20055001', '20055206', '.', '-', '.', 'gene_id \"chr9.g12489\"; transcript_id \"chr9.g12489.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20054723', '20055206', '.', '+', '.', 'gene_id \"chr9.g12490\"; transcript_id \"chr9.g12490.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20055317', '20055486', '.', '-', '.', 'gene_id \"chr9.g12491\"; transcript_id \"chr9.g12491.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '25756533', '25756568', '.', '+', '.', 'gene_id \"chr17.g9134\"; transcript_id \"chr17.g9134.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20055873', '20056095', '.', '+', '.', 'gene_id \"chr9.g12493\"; transcript_id \"chr9.g12493.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20059218', '20059301', '.', '+', '.', 'gene_id \"chr9.g12496\"; transcript_id \"chr9.g12496.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20060309', '20060550', '.', '+', '.', 'gene_id \"chr9.g12497\"; transcript_id \"chr9.g12497.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20060136', '20060319', '.', '-', '.', 'gene_id \"chr9.g12498\"; transcript_id \"chr9.g12498.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20061326', '20061556', '.', '+', '.', 'gene_id \"chr9.g12500\"; transcript_id \"chr9.g12500.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20061546', '20061745', '.', '-', '.', 'gene_id \"chr9.g12501\"; transcript_id \"chr9.g12501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20061813', '20062110', '.', '+', '.', 'gene_id \"chr9.g12502\"; transcript_id \"chr9.g12502.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20061854', '20062110', '.', '+', '.', 'gene_id \"chr9.g12502\"; transcript_id \"chr9.g12502.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20063941', '20064027', '.', '+', '.', 'gene_id \"chr9.g12509\"; transcript_id \"chr9.g12509.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20063919', '20064449', '.', '+', '.', 'gene_id \"chr9.g12509\"; transcript_id \"chr9.g12509.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20065353', '20065666', '.', '-', '.', 'gene_id \"chr9.g12511\"; transcript_id \"chr9.g12511.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20073912', '20074079', '.', '+', '.', 'gene_id \"chr9.g12522\"; transcript_id \"chr9.g12522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20075397', '20075600', '.', '+', '.', 'gene_id \"chr9.g12524\"; transcript_id \"chr9.g12524.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20078902', '20079121', '.', '-', '.', 'gene_id \"chr9.g12530\"; transcript_id \"chr9.g12530.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20079159', '20079422', '.', '-', '.', 'gene_id \"chr9.g12531\"; transcript_id \"chr9.g12531.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20079557', '20079701', '.', '+', '.', 'gene_id \"chr9.g12533\"; transcript_id \"chr9.g12533.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20079790', '20080049', '.', '+', '.', 'gene_id \"chr9.g12534\"; transcript_id \"chr9.g12534.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20081556', '20081772', '.', '+', '.', 'gene_id \"chr9.g12536\"; transcript_id \"chr9.g12536.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20085184', '20085597', '.', '-', '.', 'gene_id \"chr9.g12540\"; transcript_id \"chr9.g12540.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20091794', '20092160', '.', '-', '.', 'gene_id \"chr9.g12550\"; transcript_id \"chr9.g12550.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20092765', '20092982', '.', '-', '.', 'gene_id \"chr9.g12552\"; transcript_id \"chr9.g12552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20093013', '20093710', '.', '+', '.', 'gene_id \"chr9.g12553\"; transcript_id \"chr9.g12553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20094813', '20095337', '.', '+', '.', 'gene_id \"chr9.g12555\"; transcript_id \"chr9.g12555.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20094016', '20094191', '.', '+', '.', 'gene_id \"chr9.g12556\"; transcript_id \"chr9.g12556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20094173', '20094819', '.', '-', '.', 'gene_id \"chr9.g12557\"; transcript_id \"chr9.g12557.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20095342', '20095529', '.', '-', '.', 'gene_id \"chr9.g12558\"; transcript_id \"chr9.g12558.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20107331', '20108112', '.', '-', '.', 'gene_id \"chr9.g12564\"; transcript_id \"chr9.g12564.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20123079', '20123235', '.', '+', '.', 'gene_id \"chr9.g12568\"; transcript_id \"chr9.g12568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20122755', '20123235', '.', '+', '.', 'gene_id \"chr9.g12568\"; transcript_id \"chr9.g12568.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20130074', '20130223', '.', '-', '.', 'gene_id \"chr9.g12570\"; transcript_id \"chr9.g12570.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20124490', '20124984', '.', '-', '.', 'gene_id \"chr9.g12573\"; transcript_id \"chr9.g12573.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20130731', '20131237', '.', '+', '.', 'gene_id \"chr9.g12575\"; transcript_id \"chr9.g12575.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20130335', '20130636', '.', '+', '.', 'gene_id \"chr9.g12577\"; transcript_id \"chr9.g12577.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20123662', '20123713', '.', '+', '.', 'gene_id \"chr9.g12582\"; transcript_id \"chr9.g12582.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20123298', '20123617', '.', '+', '.', 'gene_id \"chr9.g12583\"; transcript_id \"chr9.g12583.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20122312', '20122475', '.', '+', '.', 'gene_id \"chr9.g12584\"; transcript_id \"chr9.g12584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20133094', '20133448', '.', '-', '.', 'gene_id \"chr9.g12593\"; transcript_id \"chr9.g12593.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20134987', '20135801', '.', '-', '.', 'gene_id \"chr9.g12596\"; transcript_id \"chr9.g12596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20151951', '20152592', '.', '+', '.', 'gene_id \"chr9.g12604\"; transcript_id \"chr9.g12604.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20152805', '20153150', '.', '+', '.', 'gene_id \"chr9.g12604\"; transcript_id \"chr9.g12604.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20219837', '20220131', '.', '+', '.', 'gene_id \"chr9.g12604\"; transcript_id \"chr9.g12604.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20205303', '20205792', '.', '+', '.', 'gene_id \"chr9.g12605\"; transcript_id \"chr9.g12605.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20205303', '20205449', '.', '+', '.', 'gene_id \"chr9.g12605\"; transcript_id \"chr9.g12605.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20205303', '20205447', '.', '+', '.', 'gene_id \"chr9.g12605\"; transcript_id \"chr9.g12605.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20219312', '20219694', '.', '-', '.', 'gene_id \"chr9.g12606\"; transcript_id \"chr9.g12606.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20219312', '20219478', '.', '-', '.', 'gene_id \"chr9.g12606\"; transcript_id \"chr9.g12606.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20211616', '20212257', '.', '-', '.', 'gene_id \"chr9.g12609\"; transcript_id \"chr9.g12609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20213063', '20213231', '.', '+', '.', 'gene_id \"chr9.g12610\"; transcript_id \"chr9.g12610.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20204267', '20204795', '.', '-', '.', 'gene_id \"chr9.g12614\"; transcript_id \"chr9.g12614.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20204890', '20205190', '.', '-', '.', 'gene_id \"chr9.g12616\"; transcript_id \"chr9.g12616.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20208190', '20208385', '.', '+', '.', 'gene_id \"chr9.g12617\"; transcript_id \"chr9.g12617.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20218931', '20219249', '.', '-', '.', 'gene_id \"chr9.g12620\"; transcript_id \"chr9.g12620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20217936', '20218886', '.', '-', '.', 'gene_id \"chr9.g12621\"; transcript_id \"chr9.g12621.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20167320', '20167628', '.', '-', '.', 'gene_id \"chr9.g12633\"; transcript_id \"chr9.g12633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20167616', '20167714', '.', '+', '.', 'gene_id \"chr9.g12634\"; transcript_id \"chr9.g12634.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20169172', '20169339', '.', '-', '.', 'gene_id \"chr9.g12638\"; transcript_id \"chr9.g12638.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20169846', '20169996', '.', '+', '.', 'gene_id \"chr9.g12640\"; transcript_id \"chr9.g12640.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20171088', '20171723', '.', '+', '.', 'gene_id \"chr9.g12642\"; transcript_id \"chr9.g12642.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20170517', '20171103', '.', '+', '.', 'gene_id \"chr9.g12643\"; transcript_id \"chr9.g12643.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20173139', '20173364', '.', '-', '.', 'gene_id \"chr9.g12644\"; transcript_id \"chr9.g12644.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20176999', '20177150', '.', '-', '.', 'gene_id \"chr9.g12652\"; transcript_id \"chr9.g12652.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20182584', '20183327', '.', '-', '.', 'gene_id \"chr9.g12659\"; transcript_id \"chr9.g12659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20183462', '20183616', '.', '-', '.', 'gene_id \"chr9.g12661\"; transcript_id \"chr9.g12661.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20183597', '20183850', '.', '+', '.', 'gene_id \"chr9.g12662\"; transcript_id \"chr9.g12662.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20202086', '20202262', '.', '-', '.', 'gene_id \"chr9.g12681\"; transcript_id \"chr9.g12681.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20209708', '20210043', '.', '+', '.', 'gene_id \"chr9.g12686\"; transcript_id \"chr9.g12686.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20210730', '20211166', '.', '+', '.', 'gene_id \"chr9.g12687\"; transcript_id \"chr9.g12687.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20215059', '20215236', '.', '-', '.', 'gene_id \"chr9.g12689\"; transcript_id \"chr9.g12689.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20215349', '20216645', '.', '-', '.', 'gene_id \"chr9.g12690\"; transcript_id \"chr9.g12690.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20216661', '20217052', '.', '+', '.', 'gene_id \"chr9.g12693\"; transcript_id \"chr9.g12693.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20223368', '20223744', '.', '+', '.', 'gene_id \"chr9.g12694\"; transcript_id \"chr9.g12694.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20224020', '20224191', '.', '+', '.', 'gene_id \"chr9.g12697\"; transcript_id \"chr9.g12697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20224731', '20224880', '.', '+', '.', 'gene_id \"chr9.g12700\"; transcript_id \"chr9.g12700.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20225998', '20226042', '.', '-', '.', 'gene_id \"chr9.g12706\"; transcript_id \"chr9.g12706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20229787', '20229876', '.', '-', '.', 'gene_id \"chr9.g12707\"; transcript_id \"chr9.g12707.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20249618', '20251018', '.', '-', '.', 'gene_id \"chr9.g12713\"; transcript_id \"chr9.g12713.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20256218', '20256687', '.', '-', '.', 'gene_id \"chr9.g12717\"; transcript_id \"chr9.g12717.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20238691', '20239220', '.', '-', '.', 'gene_id \"chr9.g12718\"; transcript_id \"chr9.g12718.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20241434', '20241617', '.', '-', '.', 'gene_id \"chr9.g12719\"; transcript_id \"chr9.g12719.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20240090', '20240286', '.', '+', '.', 'gene_id \"chr9.g12722\"; transcript_id \"chr9.g12722.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20242097', '20242316', '.', '+', '.', 'gene_id \"chr9.g12723\"; transcript_id \"chr9.g12723.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20246171', '20246332', '.', '-', '.', 'gene_id \"chr9.g12725\"; transcript_id \"chr9.g12725.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20246653', '20246812', '.', '+', '.', 'gene_id \"chr9.g12726\"; transcript_id \"chr9.g12726.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20258464', '20258603', '.', '+', '.', 'gene_id \"chr9.g12728\"; transcript_id \"chr9.g12728.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20278275', '20279232', '.', '+', '.', 'gene_id \"chr9.g12728\"; transcript_id \"chr9.g12728.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20277510', '20277840', '.', '+', '.', 'gene_id \"chr9.g12729\"; transcript_id \"chr9.g12729.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20279284', '20279638', '.', '+', '.', 'gene_id \"chr9.g12732\"; transcript_id \"chr9.g12732.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20265116', '20265406', '.', '+', '.', 'gene_id \"chr9.g12739\"; transcript_id \"chr9.g12739.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20269767', '20269929', '.', '-', '.', 'gene_id \"chr9.g12744\"; transcript_id \"chr9.g12744.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20276602', '20276984', '.', '+', '.', 'gene_id \"chr9.g12755\"; transcript_id \"chr9.g12755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20290170', '20290364', '.', '+', '.', 'gene_id \"chr9.g12763\"; transcript_id \"chr9.g12763.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20287220', '20287396', '.', '+', '.', 'gene_id \"chr9.g12763\"; transcript_id \"chr9.g12763.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20289941', '20290129', '.', '-', '.', 'gene_id \"chr9.g12764\"; transcript_id \"chr9.g12764.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20288408', '20288571', '.', '+', '.', 'gene_id \"chr9.g12765\"; transcript_id \"chr9.g12765.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20296141', '20296495', '.', '+', '.', 'gene_id \"chr9.g12767\"; transcript_id \"chr9.g12767.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20295947', '20296158', '.', '+', '.', 'gene_id \"chr9.g12769\"; transcript_id \"chr9.g12769.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20342143', '20343902', '.', '+', '.', 'gene_id \"chr9.g12773\"; transcript_id \"chr9.g12773.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20340286', '20341831', '.', '+', '.', 'gene_id \"chr9.g12773\"; transcript_id \"chr9.g12773.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20339312', '20340062', '.', '-', '.', 'gene_id \"chr9.g12774\"; transcript_id \"chr9.g12774.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20340055', '20340298', '.', '+', '.', 'gene_id \"chr9.g12776\"; transcript_id \"chr9.g12776.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20337781', '20338961', '.', '+', '.', 'gene_id \"chr9.g12779\"; transcript_id \"chr9.g12779.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20337790', '20338961', '.', '-', '.', 'gene_id \"chr9.g12780\"; transcript_id \"chr9.g12780.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20343957', '20345467', '.', '+', '.', 'gene_id \"chr9.g12781\"; transcript_id \"chr9.g12781.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20351954', '20352142', '.', '+', '.', 'gene_id \"chr9.g12785\"; transcript_id \"chr9.g12785.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20359238', '20359425', '.', '+', '.', 'gene_id \"chr9.g12793\"; transcript_id \"chr9.g12793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20359744', '20359896', '.', '-', '.', 'gene_id \"chr9.g12795\"; transcript_id \"chr9.g12795.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20360789', '20361107', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20360789', '20361107', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20383145', '20384028', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20372013', '20372623', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i17\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20372013', '20372278', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20386385', '20387054', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i19\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20360789', '20361107', '.', '+', '.', 'gene_id \"chr9.g12798\"; transcript_id \"chr9.g12798.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20380908', '20381264', '.', '+', '.', 'gene_id \"chr9.g12801\"; transcript_id \"chr9.g12801.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20380699', '20380918', '.', '+', '.', 'gene_id \"chr9.g12802\"; transcript_id \"chr9.g12802.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20376653', '20376841', '.', '+', '.', 'gene_id \"chr9.g12804\"; transcript_id \"chr9.g12804.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20362120', '20362568', '.', '+', '.', 'gene_id \"chr9.g12808\"; transcript_id \"chr9.g12808.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20367460', '20367699', '.', '+', '.', 'gene_id \"chr9.g12811\"; transcript_id \"chr9.g12811.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20367460', '20367699', '.', '+', '.', 'gene_id \"chr9.g12811\"; transcript_id \"chr9.g12811.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20367980', '20369076', '.', '-', '.', 'gene_id \"chr9.g12813\"; transcript_id \"chr9.g12813.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20372767', '20373713', '.', '+', '.', 'gene_id \"chr9.g12815\"; transcript_id \"chr9.g12815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20378002', '20378167', '.', '-', '.', 'gene_id \"chr9.g12819\"; transcript_id \"chr9.g12819.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20380233', '20380384', '.', '+', '.', 'gene_id \"chr9.g12822\"; transcript_id \"chr9.g12822.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20380371', '20380574', '.', '+', '.', 'gene_id \"chr9.g12823\"; transcript_id \"chr9.g12823.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20397798', '20398281', '.', '+', '.', 'gene_id \"chr9.g12828\"; transcript_id \"chr9.g12828.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20396951', '20397581', '.', '+', '.', 'gene_id \"chr9.g12831\"; transcript_id \"chr9.g12831.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20400919', '20401232', '.', '+', '.', 'gene_id \"chr9.g12833\"; transcript_id \"chr9.g12833.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20424980', '20425480', '.', '-', '.', 'gene_id \"chr9.g12834\"; transcript_id \"chr9.g12834.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20402889', '20403065', '.', '-', '.', 'gene_id \"chr9.g12841\"; transcript_id \"chr9.g12841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20406565', '20406828', '.', '+', '.', 'gene_id \"chr9.g12845\"; transcript_id \"chr9.g12845.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20407095', '20407263', '.', '+', '.', 'gene_id \"chr9.g12846\"; transcript_id \"chr9.g12846.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20410252', '20410449', '.', '+', '.', 'gene_id \"chr9.g12852\"; transcript_id \"chr9.g12852.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20413858', '20414063', '.', '+', '.', 'gene_id \"chr9.g12855\"; transcript_id \"chr9.g12855.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20414900', '20415108', '.', '+', '.', 'gene_id \"chr9.g12859\"; transcript_id \"chr9.g12859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20415495', '20415661', '.', '-', '.', 'gene_id \"chr9.g12860\"; transcript_id \"chr9.g12860.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20415696', '20415926', '.', '-', '.', 'gene_id \"chr9.g12862\"; transcript_id \"chr9.g12862.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20417785', '20418813', '.', '+', '.', 'gene_id \"chr9.g12866\"; transcript_id \"chr9.g12866.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20424184', '20424419', '.', '+', '.', 'gene_id \"chr9.g12871\"; transcript_id \"chr9.g12871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20424584', '20424979', '.', '+', '.', 'gene_id \"chr9.g12872\"; transcript_id \"chr9.g12872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20452806', '20455381', '.', '+', '.', 'gene_id \"chr9.g12876\"; transcript_id \"chr9.g12876.i15\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20451660', '20452818', '.', '+', '.', 'gene_id \"chr9.g12876\"; transcript_id \"chr9.g12876.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20444647', '20445494', '.', '+', '.', 'gene_id \"chr9.g12876\"; transcript_id \"chr9.g12876.i19\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20451461', '20451612', '.', '+', '.', 'gene_id \"chr9.g12876\"; transcript_id \"chr9.g12876.i21\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20462808', '20462934', '.', '+', '.', 'gene_id \"chr9.g12877\"; transcript_id \"chr9.g12877.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20475269', '20475622', '.', '-', '.', 'gene_id \"chr9.g12878\"; transcript_id \"chr9.g12878.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20477087', '20477208', '.', '+', '.', 'gene_id \"chr9.g12879\"; transcript_id \"chr9.g12879.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20477188', '20477422', '.', '+', '.', 'gene_id \"chr9.g12879\"; transcript_id \"chr9.g12879.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20464264', '20465046', '.', '-', '.', 'gene_id \"chr9.g12885\"; transcript_id \"chr9.g12885.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20470221', '20470412', '.', '+', '.', 'gene_id \"chr9.g12886\"; transcript_id \"chr9.g12886.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20451074', '20451283', '.', '-', '.', 'gene_id \"chr9.g12887\"; transcript_id \"chr9.g12887.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20476322', '20476512', '.', '+', '.', 'gene_id \"chr9.g12889\"; transcript_id \"chr9.g12889.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20455480', '20455728', '.', '-', '.', 'gene_id \"chr9.g12893\"; transcript_id \"chr9.g12893.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20451300', '20451477', '.', '-', '.', 'gene_id \"chr9.g12894\"; transcript_id \"chr9.g12894.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20470405', '20470580', '.', '-', '.', 'gene_id \"chr9.g12895\"; transcript_id \"chr9.g12895.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20459747', '20460040', '.', '+', '.', 'gene_id \"chr9.g12896\"; transcript_id \"chr9.g12896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20475604', '20476141', '.', '+', '.', 'gene_id \"chr9.g12897\"; transcript_id \"chr9.g12897.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20464018', '20464278', '.', '+', '.', 'gene_id \"chr9.g12899\"; transcript_id \"chr9.g12899.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20476563', '20476897', '.', '-', '.', 'gene_id \"chr9.g12901\"; transcript_id \"chr9.g12901.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20463648', '20463840', '.', '+', '.', 'gene_id \"chr9.g12902\"; transcript_id \"chr9.g12902.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20436907', '20437093', '.', '-', '.', 'gene_id \"chr9.g12905\"; transcript_id \"chr9.g12905.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20443891', '20444011', '.', '+', '.', 'gene_id \"chr9.g12910\"; transcript_id \"chr9.g12910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20447238', '20447488', '.', '-', '.', 'gene_id \"chr9.g12912\"; transcript_id \"chr9.g12912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20456678', '20456877', '.', '+', '.', 'gene_id \"chr9.g12918\"; transcript_id \"chr9.g12918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20457062', '20457217', '.', '-', '.', 'gene_id \"chr9.g12920\"; transcript_id \"chr9.g12920.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '27342046', '27342091', '.', '+', '.', 'gene_id \"chr17.g9462\"; transcript_id \"chr17.g9462.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20459092', '20459307', '.', '-', '.', 'gene_id \"chr9.g12922\"; transcript_id \"chr9.g12922.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20461677', '20461891', '.', '-', '.', 'gene_id \"chr9.g12928\"; transcript_id \"chr9.g12928.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20461897', '20462068', '.', '-', '.', 'gene_id \"chr9.g12929\"; transcript_id \"chr9.g12929.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20467334', '20467778', '.', '-', '.', 'gene_id \"chr9.g12933\"; transcript_id \"chr9.g12933.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20468431', '20469176', '.', '+', '.', 'gene_id \"chr9.g12935\"; transcript_id \"chr9.g12935.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20471653', '20471763', '.', '-', '.', 'gene_id \"chr9.g12936\"; transcript_id \"chr9.g12936.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20471554', '20471667', '.', '+', '.', 'gene_id \"chr9.g12937\"; transcript_id \"chr9.g12937.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20471769', '20471862', '.', '+', '.', 'gene_id \"chr9.g12938\"; transcript_id \"chr9.g12938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20505621', '20506123', '.', '+', '.', 'gene_id \"chr9.g12955\"; transcript_id \"chr9.g12955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20504505', '20504865', '.', '+', '.', 'gene_id \"chr9.g12956\"; transcript_id \"chr9.g12956.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20503938', '20504518', '.', '-', '.', 'gene_id \"chr9.g12957\"; transcript_id \"chr9.g12957.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20497397', '20497901', '.', '-', '.', 'gene_id \"chr9.g12958\"; transcript_id \"chr9.g12958.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20498979', '20499236', '.', '-', '.', 'gene_id \"chr9.g12962\"; transcript_id \"chr9.g12962.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20501369', '20501563', '.', '+', '.', 'gene_id \"chr9.g12966\"; transcript_id \"chr9.g12966.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20501562', '20501759', '.', '-', '.', 'gene_id \"chr9.g12967\"; transcript_id \"chr9.g12967.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20502020', '20502876', '.', '-', '.', 'gene_id \"chr9.g12968\"; transcript_id \"chr9.g12968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20511644', '20511858', '.', '-', '.', 'gene_id \"chr9.g12971\"; transcript_id \"chr9.g12971.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20519637', '20520129', '.', '+', '.', 'gene_id \"chr9.g12977\"; transcript_id \"chr9.g12977.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20523032', '20523359', '.', '-', '.', 'gene_id \"chr9.g12980\"; transcript_id \"chr9.g12980.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20535625', '20535805', '.', '+', '.', 'gene_id \"chr9.g12983\"; transcript_id \"chr9.g12983.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20554592', '20555361', '.', '+', '.', 'gene_id \"chr9.g12983\"; transcript_id \"chr9.g12983.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20562768', '20563708', '.', '-', '.', 'gene_id \"chr9.g12984\"; transcript_id \"chr9.g12984.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20563686', '20563844', '.', '-', '.', 'gene_id \"chr9.g12984\"; transcript_id \"chr9.g12984.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20563730', '20563844', '.', '-', '.', 'gene_id \"chr9.g12984\"; transcript_id \"chr9.g12984.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20548863', '20549103', '.', '-', '.', 'gene_id \"chr9.g12989\"; transcript_id \"chr9.g12989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20550085', '20550246', '.', '-', '.', 'gene_id \"chr9.g12991\"; transcript_id \"chr9.g12991.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20543865', '20544027', '.', '-', '.', 'gene_id \"chr9.g13009\"; transcript_id \"chr9.g13009.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20547911', '20548134', '.', '-', '.', 'gene_id \"chr9.g13010\"; transcript_id \"chr9.g13010.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20546943', '20547928', '.', '-', '.', 'gene_id \"chr9.g13011\"; transcript_id \"chr9.g13011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20548122', '20548282', '.', '+', '.', 'gene_id \"chr9.g13013\"; transcript_id \"chr9.g13013.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20556602', '20556984', '.', '+', '.', 'gene_id \"chr9.g13016\"; transcript_id \"chr9.g13016.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20582124', '20582328', '.', '-', '.', 'gene_id \"chr9.g13022\"; transcript_id \"chr9.g13022.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20582351', '20583002', '.', '+', '.', 'gene_id \"chr9.g13023\"; transcript_id \"chr9.g13023.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20583353', '20583497', '.', '-', '.', 'gene_id \"chr9.g13026\"; transcript_id \"chr9.g13026.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20607565', '20608339', '.', '+', '.', 'gene_id \"chr9.g13034\"; transcript_id \"chr9.g13034.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20606589', '20607199', '.', '+', '.', 'gene_id \"chr9.g13035\"; transcript_id \"chr9.g13035.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20607257', '20607446', '.', '+', '.', 'gene_id \"chr9.g13036\"; transcript_id \"chr9.g13036.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20608793', '20609025', '.', '-', '.', 'gene_id \"chr9.g13037\"; transcript_id \"chr9.g13037.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '28739780', '28740303', '.', '-', '.', 'gene_id \"chr17.g9567\"; transcript_id \"chr17.g9567.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20665022', '20665092', '.', '+', '.', 'gene_id \"chr9.g13048\"; transcript_id \"chr9.g13048.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20666909', '20666962', '.', '-', '.', 'gene_id \"chr9.g13054\"; transcript_id \"chr9.g13054.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20659511', '20659565', '.', '-', '.', 'gene_id \"chr9.g13056\"; transcript_id \"chr9.g13056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20667056', '20667880', '.', '-', '.', 'gene_id \"chr9.g13057\"; transcript_id \"chr9.g13057.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20699822', '20699874', '.', '+', '.', 'gene_id \"chr9.g13065\"; transcript_id \"chr9.g13065.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20700895', '20701444', '.', '-', '.', 'gene_id \"chr9.g13067\"; transcript_id \"chr9.g13067.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20700728', '20700894', '.', '-', '.', 'gene_id \"chr9.g13068\"; transcript_id \"chr9.g13068.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20699907', '20700101', '.', '-', '.', 'gene_id \"chr9.g13070\"; transcript_id \"chr9.g13070.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20699198', '20699795', '.', '-', '.', 'gene_id \"chr9.g13071\"; transcript_id \"chr9.g13071.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20720964', '20720993', '.', '-', '.', 'gene_id \"chr9.g13072\"; transcript_id \"chr9.g13072.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20744819', '20745767', '.', '-', '.', 'gene_id \"chr9.g13074\"; transcript_id \"chr9.g13074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20746702', '20747353', '.', '+', '.', 'gene_id \"chr9.g13075\"; transcript_id \"chr9.g13075.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20747606', '20747998', '.', '+', '.', 'gene_id \"chr9.g13076\"; transcript_id \"chr9.g13076.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20756751', '20756934', '.', '+', '.', 'gene_id \"chr9.g13077\"; transcript_id \"chr9.g13077.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20760175', '20760760', '.', '+', '.', 'gene_id \"chr9.g13077\"; transcript_id \"chr9.g13077.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20759347', '20759464', '.', '-', '.', 'gene_id \"chr9.g13078\"; transcript_id \"chr9.g13078.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20759447', '20759651', '.', '-', '.', 'gene_id \"chr9.g13078\"; transcript_id \"chr9.g13078.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20749164', '20749331', '.', '+', '.', 'gene_id \"chr9.g13083\"; transcript_id \"chr9.g13083.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20784574', '20784757', '.', '+', '.', 'gene_id \"chr9.g13097\"; transcript_id \"chr9.g13097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20784754', '20784970', '.', '-', '.', 'gene_id \"chr9.g13098\"; transcript_id \"chr9.g13098.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20799748', '20799916', '.', '+', '.', 'gene_id \"chr9.g13099\"; transcript_id \"chr9.g13099.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20796436', '20796468', '.', '+', '.', 'gene_id \"chr9.g13099\"; transcript_id \"chr9.g13099.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20785372', '20785684', '.', '-', '.', 'gene_id \"chr9.g13101\"; transcript_id \"chr9.g13101.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20785691', '20786149', '.', '-', '.', 'gene_id \"chr9.g13101\"; transcript_id \"chr9.g13101.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20795486', '20795669', '.', '-', '.', 'gene_id \"chr9.g13102\"; transcript_id \"chr9.g13102.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20784978', '20785176', '.', '+', '.', 'gene_id \"chr9.g13103\"; transcript_id \"chr9.g13103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20795266', '20795505', '.', '+', '.', 'gene_id \"chr9.g13104\"; transcript_id \"chr9.g13104.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20798920', '20799157', '.', '+', '.', 'gene_id \"chr9.g13109\"; transcript_id \"chr9.g13109.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20795674', '20796177', '.', '+', '.', 'gene_id \"chr9.g13110\"; transcript_id \"chr9.g13110.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20807422', '20807478', '.', '+', '.', 'gene_id \"chr9.g13118\"; transcript_id \"chr9.g13118.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20828265', '20828692', '.', '+', '.', 'gene_id \"chr9.g13131\"; transcript_id \"chr9.g13131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20834694', '20835398', '.', '+', '.', 'gene_id \"chr9.g13134\"; transcript_id \"chr9.g13134.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20847541', '20847969', '.', '+', '.', 'gene_id \"chr9.g13141\"; transcript_id \"chr9.g13141.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20847734', '20847969', '.', '+', '.', 'gene_id \"chr9.g13141\"; transcript_id \"chr9.g13141.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20855632', '20855904', '.', '-', '.', 'gene_id \"chr9.g13143\"; transcript_id \"chr9.g13143.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20858547', '20858919', '.', '+', '.', 'gene_id \"chr9.g13145\"; transcript_id \"chr9.g13145.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20858684', '20858919', '.', '+', '.', 'gene_id \"chr9.g13145\"; transcript_id \"chr9.g13145.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20860608', '20860642', '.', '+', '.', 'gene_id \"chr9.g13147\"; transcript_id \"chr9.g13147.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20869689', '20870123', '.', '+', '.', 'gene_id \"chr9.g13153\"; transcript_id \"chr9.g13153.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20871962', '20872234', '.', '+', '.', 'gene_id \"chr9.g13154\"; transcript_id \"chr9.g13154.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20871939', '20872171', '.', '+', '.', 'gene_id \"chr9.g13154\"; transcript_id \"chr9.g13154.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20871939', '20872054', '.', '+', '.', 'gene_id \"chr9.g13154\"; transcript_id \"chr9.g13154.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20875089', '20875270', '.', '-', '.', 'gene_id \"chr9.g13157\"; transcript_id \"chr9.g13157.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20874933', '20875090', '.', '-', '.', 'gene_id \"chr9.g13158\"; transcript_id \"chr9.g13158.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20871720', '20871957', '.', '-', '.', 'gene_id \"chr9.g13159\"; transcript_id \"chr9.g13159.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20886562', '20886792', '.', '+', '.', 'gene_id \"chr9.g13160\"; transcript_id \"chr9.g13160.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20884166', '20884338', '.', '+', '.', 'gene_id \"chr9.g13160\"; transcript_id \"chr9.g13160.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20885590', '20885839', '.', '+', '.', 'gene_id \"chr9.g13163\"; transcript_id \"chr9.g13163.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20891274', '20891468', '.', '-', '.', 'gene_id \"chr9.g13164\"; transcript_id \"chr9.g13164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20905648', '20905878', '.', '+', '.', 'gene_id \"chr9.g13166\"; transcript_id \"chr9.g13166.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20901300', '20901342', '.', '+', '.', 'gene_id \"chr9.g13166\"; transcript_id \"chr9.g13166.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20900981', '20901156', '.', '+', '.', 'gene_id \"chr9.g13166\"; transcript_id \"chr9.g13166.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20903295', '20903460', '.', '+', '.', 'gene_id \"chr9.g13170\"; transcript_id \"chr9.g13170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20924180', '20924410', '.', '+', '.', 'gene_id \"chr9.g13172\"; transcript_id \"chr9.g13172.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20920389', '20920598', '.', '+', '.', 'gene_id \"chr9.g13172\"; transcript_id \"chr9.g13172.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20922199', '20922369', '.', '+', '.', 'gene_id \"chr9.g13174\"; transcript_id \"chr9.g13174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20958388', '20958853', '.', '+', '.', 'gene_id \"chr9.g13182\"; transcript_id \"chr9.g13182.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20958661', '20958973', '.', '+', '.', 'gene_id \"chr9.g13182\"; transcript_id \"chr9.g13182.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20958896', '20960505', '.', '+', '.', 'gene_id \"chr9.g13182\"; transcript_id \"chr9.g13182.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20967488', '20967552', '.', '-', '.', 'gene_id \"chr9.g13187\"; transcript_id \"chr9.g13187.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20992441', '20993524', '.', '+', '.', 'gene_id \"chr9.g13193\"; transcript_id \"chr9.g13193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20992236', '20992479', '.', '+', '.', 'gene_id \"chr9.g13193\"; transcript_id \"chr9.g13193.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20992988', '20993143', '.', '+', '.', 'gene_id \"chr9.g13193\"; transcript_id \"chr9.g13193.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20996527', '20997163', '.', '-', '.', 'gene_id \"chr9.g13198\"; transcript_id \"chr9.g13198.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20997935', '20998107', '.', '-', '.', 'gene_id \"chr9.g13199\"; transcript_id \"chr9.g13199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20999377', '20999539', '.', '+', '.', 'gene_id \"chr9.g13200\"; transcript_id \"chr9.g13200.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '28895093', '28895529', '.', '-', '.', 'gene_id \"chr17.g9688\"; transcript_id \"chr17.g9688.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21003961', '21004318', '.', '+', '.', 'gene_id \"chr9.g13202\"; transcript_id \"chr9.g13202.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21004753', '21004858', '.', '+', '.', 'gene_id \"chr9.g13203\"; transcript_id \"chr9.g13203.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21006193', '21006394', '.', '-', '.', 'gene_id \"chr9.g13204\"; transcript_id \"chr9.g13204.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '28898928', '28899132', '.', '+', '.', 'gene_id \"chr17.g9693\"; transcript_id \"chr17.g9693.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21009579', '21010149', '.', '+', '.', 'gene_id \"chr9.g13206\"; transcript_id \"chr9.g13206.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21016104', '21016176', '.', '+', '.', 'gene_id \"chr9.g13206\"; transcript_id \"chr9.g13206.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21015241', '21015422', '.', '+', '.', 'gene_id \"chr9.g13206\"; transcript_id \"chr9.g13206.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21011894', '21012092', '.', '-', '.', 'gene_id \"chr9.g13209\"; transcript_id \"chr9.g13209.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21017689', '21017889', '.', '-', '.', 'gene_id \"chr9.g13211\"; transcript_id \"chr9.g13211.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21033851', '21034465', '.', '+', '.', 'gene_id \"chr9.g13213\"; transcript_id \"chr9.g13213.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21036299', '21036500', '.', '-', '.', 'gene_id \"chr9.g13215\"; transcript_id \"chr9.g13215.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21039640', '21040254', '.', '+', '.', 'gene_id \"chr9.g13217\"; transcript_id \"chr9.g13217.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21042184', '21042385', '.', '-', '.', 'gene_id \"chr9.g13219\"; transcript_id \"chr9.g13219.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21045529', '21046143', '.', '+', '.', 'gene_id \"chr9.g13222\"; transcript_id \"chr9.g13222.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21048073', '21048274', '.', '-', '.', 'gene_id \"chr9.g13224\"; transcript_id \"chr9.g13224.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21061072', '21061687', '.', '+', '.', 'gene_id \"chr9.g13226\"; transcript_id \"chr9.g13226.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21063523', '21063724', '.', '-', '.', 'gene_id \"chr9.g13228\"; transcript_id \"chr9.g13228.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21066855', '21067469', '.', '+', '.', 'gene_id \"chr9.g13231\"; transcript_id \"chr9.g13231.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21069399', '21069600', '.', '-', '.', 'gene_id \"chr9.g13233\"; transcript_id \"chr9.g13233.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21070986', '21071399', '.', '-', '.', 'gene_id \"chr9.g13235\"; transcript_id \"chr9.g13235.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21115759', '21115789', '.', '-', '.', 'gene_id \"chr9.g13247\"; transcript_id \"chr9.g13247.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21115737', '21115781', '.', '+', '.', 'gene_id \"chr9.g13248\"; transcript_id \"chr9.g13248.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21115887', '21116324', '.', '+', '.', 'gene_id \"chr9.g13249\"; transcript_id \"chr9.g13249.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21127214', '21127650', '.', '+', '.', 'gene_id \"chr9.g13254\"; transcript_id \"chr9.g13254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21144768', '21145184', '.', '-', '.', 'gene_id \"chr9.g13256\"; transcript_id \"chr9.g13256.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21144768', '21145184', '.', '-', '.', 'gene_id \"chr9.g13256\"; transcript_id \"chr9.g13256.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21144768', '21145125', '.', '-', '.', 'gene_id \"chr9.g13256\"; transcript_id \"chr9.g13256.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21146117', '21146604', '.', '-', '.', 'gene_id \"chr9.g13256\"; transcript_id \"chr9.g13256.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21136433', '21136477', '.', '+', '.', 'gene_id \"chr9.g13257\"; transcript_id \"chr9.g13257.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21160596', '21160640', '.', '+', '.', 'gene_id \"chr9.g13261\"; transcript_id \"chr9.g13261.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21153112', '21153250', '.', '+', '.', 'gene_id \"chr9.g13262\"; transcript_id \"chr9.g13262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21183753', '21184643', '.', '-', '.', 'gene_id \"chr9.g13268\"; transcript_id \"chr9.g13268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21185131', '21185278', '.', '-', '.', 'gene_id \"chr9.g13270\"; transcript_id \"chr9.g13270.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21185629', '21185867', '.', '-', '.', 'gene_id \"chr9.g13271\"; transcript_id \"chr9.g13271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21200700', '21201193', '.', '-', '.', 'gene_id \"chr9.g13274\"; transcript_id \"chr9.g13274.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21200700', '21201449', '.', '-', '.', 'gene_id \"chr9.g13274\"; transcript_id \"chr9.g13274.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21194443', '21194953', '.', '-', '.', 'gene_id \"chr9.g13274\"; transcript_id \"chr9.g13274.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21194443', '21195199', '.', '-', '.', 'gene_id \"chr9.g13274\"; transcript_id \"chr9.g13274.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21189248', '21190146', '.', '-', '.', 'gene_id \"chr9.g13275\"; transcript_id \"chr9.g13275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21189248', '21189759', '.', '-', '.', 'gene_id \"chr9.g13275\"; transcript_id \"chr9.g13275.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21189248', '21189435', '.', '-', '.', 'gene_id \"chr9.g13275\"; transcript_id \"chr9.g13275.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21202077', '21202263', '.', '-', '.', 'gene_id \"chr9.g13284\"; transcript_id \"chr9.g13284.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21203118', '21203324', '.', '-', '.', 'gene_id \"chr9.g13285\"; transcript_id \"chr9.g13285.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21204590', '21205417', '.', '-', '.', 'gene_id \"chr9.g13287\"; transcript_id \"chr9.g13287.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21205058', '21205099', '.', '-', '.', 'gene_id \"chr9.g13287\"; transcript_id \"chr9.g13287.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21210457', '21211410', '.', '-', '.', 'gene_id \"chr9.g13290\"; transcript_id \"chr9.g13290.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21210520', '21210987', '.', '-', '.', 'gene_id \"chr9.g13290\"; transcript_id \"chr9.g13290.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21210946', '21211018', '.', '-', '.', 'gene_id \"chr9.g13290\"; transcript_id \"chr9.g13290.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21218579', '21218642', '.', '-', '.', 'gene_id \"chr9.g13297\"; transcript_id \"chr9.g13297.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21221441', '21222114', '.', '-', '.', 'gene_id \"chr9.g13299\"; transcript_id \"chr9.g13299.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21223196', '21223408', '.', '+', '.', 'gene_id \"chr9.g13300\"; transcript_id \"chr9.g13300.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21236839', '21237036', '.', '+', '.', 'gene_id \"chr9.g13304\"; transcript_id \"chr9.g13304.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21245816', '21246455', '.', '+', '.', 'gene_id \"chr9.g13306\"; transcript_id \"chr9.g13306.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21255572', '21256384', '.', '-', '.', 'gene_id \"chr9.g13310\"; transcript_id \"chr9.g13310.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21255710', '21256072', '.', '-', '.', 'gene_id \"chr9.g13310\"; transcript_id \"chr9.g13310.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21262786', '21263505', '.', '-', '.', 'gene_id \"chr9.g13312\"; transcript_id \"chr9.g13312.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21263483', '21263615', '.', '-', '.', 'gene_id \"chr9.g13312\"; transcript_id \"chr9.g13312.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21263622', '21263901', '.', '+', '.', 'gene_id \"chr9.g13314\"; transcript_id \"chr9.g13314.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21264117', '21264344', '.', '+', '.', 'gene_id \"chr9.g13315\"; transcript_id \"chr9.g13315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21265753', '21266007', '.', '+', '.', 'gene_id \"chr9.g13316\"; transcript_id \"chr9.g13316.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21277151', '21277329', '.', '+', '.', 'gene_id \"chr9.g13318\"; transcript_id \"chr9.g13318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21282189', '21282422', '.', '+', '.', 'gene_id \"chr9.g13319\"; transcript_id \"chr9.g13319.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21277358', '21278447', '.', '-', '.', 'gene_id \"chr9.g13322\"; transcript_id \"chr9.g13322.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21286207', '21286471', '.', '-', '.', 'gene_id \"chr9.g13327\"; transcript_id \"chr9.g13327.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21290113', '21290710', '.', '-', '.', 'gene_id \"chr9.g13329\"; transcript_id \"chr9.g13329.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21302289', '21302561', '.', '-', '.', 'gene_id \"chr9.g13333\"; transcript_id \"chr9.g13333.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21302396', '21302696', '.', '-', '.', 'gene_id \"chr9.g13333\"; transcript_id \"chr9.g13333.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21319238', '21319302', '.', '+', '.', 'gene_id \"chr9.g13334\"; transcript_id \"chr9.g13334.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21319238', '21319303', '.', '+', '.', 'gene_id \"chr9.g13334\"; transcript_id \"chr9.g13334.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21324689', '21324872', '.', '+', '.', 'gene_id \"chr9.g13337\"; transcript_id \"chr9.g13337.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21332214', '21332364', '.', '+', '.', 'gene_id \"chr9.g13339\"; transcript_id \"chr9.g13339.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21333628', '21333877', '.', '-', '.', 'gene_id \"chr9.g13341\"; transcript_id \"chr9.g13341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21340467', '21340622', '.', '+', '.', 'gene_id \"chr9.g13344\"; transcript_id \"chr9.g13344.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21342189', '21342343', '.', '+', '.', 'gene_id \"chr9.g13345\"; transcript_id \"chr9.g13345.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21342323', '21342538', '.', '-', '.', 'gene_id \"chr9.g13346\"; transcript_id \"chr9.g13346.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21385242', '21385469', '.', '+', '.', 'gene_id \"chr9.g13354\"; transcript_id \"chr9.g13354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21396601', '21396781', '.', '-', '.', 'gene_id \"chr9.g13360\"; transcript_id \"chr9.g13360.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21396992', '21397244', '.', '-', '.', 'gene_id \"chr9.g13363\"; transcript_id \"chr9.g13363.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21397228', '21397459', '.', '-', '.', 'gene_id \"chr9.g13364\"; transcript_id \"chr9.g13364.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21402893', '21403124', '.', '-', '.', 'gene_id \"chr9.g13368\"; transcript_id \"chr9.g13368.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21403107', '21403369', '.', '+', '.', 'gene_id \"chr9.g13369\"; transcript_id \"chr9.g13369.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21406930', '21407116', '.', '-', '.', 'gene_id \"chr9.g13374\"; transcript_id \"chr9.g13374.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '29370581', '29372161', '.', '-', '.', 'gene_id \"chr17.g9863\"; transcript_id \"chr17.g9863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21419843', '21420048', '.', '-', '.', 'gene_id \"chr9.g13384\"; transcript_id \"chr9.g13384.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21420128', '21420680', '.', '-', '.', 'gene_id \"chr9.g13385\"; transcript_id \"chr9.g13385.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21425210', '21425463', '.', '-', '.', 'gene_id \"chr9.g13387\"; transcript_id \"chr9.g13387.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21427399', '21427561', '.', '+', '.', 'gene_id \"chr9.g13388\"; transcript_id \"chr9.g13388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21428416', '21429358', '.', '+', '.', 'gene_id \"chr9.g13390\"; transcript_id \"chr9.g13390.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21434005', '21434140', '.', '+', '.', 'gene_id \"chr9.g13394\"; transcript_id \"chr9.g13394.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21435572', '21435731', '.', '-', '.', 'gene_id \"chr9.g13395\"; transcript_id \"chr9.g13395.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21431909', '21432747', '.', '-', '.', 'gene_id \"chr9.g13396\"; transcript_id \"chr9.g13396.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21433783', '21434027', '.', '-', '.', 'gene_id \"chr9.g13397\"; transcript_id \"chr9.g13397.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21433311', '21433450', '.', '-', '.', 'gene_id \"chr9.g13398\"; transcript_id \"chr9.g13398.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21432790', '21433198', '.', '-', '.', 'gene_id \"chr9.g13399\"; transcript_id \"chr9.g13399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21436869', '21437147', '.', '-', '.', 'gene_id \"chr9.g13401\"; transcript_id \"chr9.g13401.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21448749', '21449068', '.', '-', '.', 'gene_id \"chr9.g13406\"; transcript_id \"chr9.g13406.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21449406', '21449655', '.', '+', '.', 'gene_id \"chr9.g13408\"; transcript_id \"chr9.g13408.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21451003', '21451161', '.', '+', '.', 'gene_id \"chr9.g13410\"; transcript_id \"chr9.g13410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21451450', '21451783', '.', '-', '.', 'gene_id \"chr9.g13411\"; transcript_id \"chr9.g13411.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21451197', '21451450', '.', '+', '.', 'gene_id \"chr9.g13413\"; transcript_id \"chr9.g13413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21467283', '21467588', '.', '-', '.', 'gene_id \"chr9.g13416\"; transcript_id \"chr9.g13416.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21471667', '21471863', '.', '+', '.', 'gene_id \"chr9.g13417\"; transcript_id \"chr9.g13417.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21471667', '21471841', '.', '+', '.', 'gene_id \"chr9.g13417\"; transcript_id \"chr9.g13417.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21471863', '21472122', '.', '+', '.', 'gene_id \"chr9.g13419\"; transcript_id \"chr9.g13419.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21467737', '21467949', '.', '+', '.', 'gene_id \"chr9.g13420\"; transcript_id \"chr9.g13420.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21468883', '21469203', '.', '+', '.', 'gene_id \"chr9.g13422\"; transcript_id \"chr9.g13422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21470778', '21471018', '.', '-', '.', 'gene_id \"chr9.g13425\"; transcript_id \"chr9.g13425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21471296', '21471563', '.', '-', '.', 'gene_id \"chr9.g13426\"; transcript_id \"chr9.g13426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21479328', '21479389', '.', '-', '.', 'gene_id \"chr9.g13428\"; transcript_id \"chr9.g13428.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21492170', '21492487', '.', '-', '.', 'gene_id \"chr9.g13431\"; transcript_id \"chr9.g13431.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21508092', '21508511', '.', '+', '.', 'gene_id \"chr9.g13434\"; transcript_id \"chr9.g13434.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21512049', '21512343', '.', '-', '.', 'gene_id \"chr9.g13435\"; transcript_id \"chr9.g13435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21509370', '21509695', '.', '-', '.', 'gene_id \"chr9.g13443\"; transcript_id \"chr9.g13443.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21508792', '21509206', '.', '-', '.', 'gene_id \"chr9.g13445\"; transcript_id \"chr9.g13445.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21510369', '21510719', '.', '-', '.', 'gene_id \"chr9.g13446\"; transcript_id \"chr9.g13446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21511357', '21511625', '.', '+', '.', 'gene_id \"chr9.g13449\"; transcript_id \"chr9.g13449.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21570548', '21570708', '.', '+', '.', 'gene_id \"chr9.g13459\"; transcript_id \"chr9.g13459.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21571117', '21571312', '.', '-', '.', 'gene_id \"chr9.g13463\"; transcript_id \"chr9.g13463.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21581334', '21581535', '.', '-', '.', 'gene_id \"chr9.g13468\"; transcript_id \"chr9.g13468.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21601755', '21602928', '.', '+', '.', 'gene_id \"chr9.g13470\"; transcript_id \"chr9.g13470.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21583642', '21583798', '.', '+', '.', 'gene_id \"chr9.g13476\"; transcript_id \"chr9.g13476.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21595257', '21596235', '.', '-', '.', 'gene_id \"chr9.g13483\"; transcript_id \"chr9.g13483.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21596735', '21597245', '.', '+', '.', 'gene_id \"chr9.g13485\"; transcript_id \"chr9.g13485.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21636193', '21636456', '.', '+', '.', 'gene_id \"chr9.g13493\"; transcript_id \"chr9.g13493.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21638288', '21638441', '.', '-', '.', 'gene_id \"chr9.g13494\"; transcript_id \"chr9.g13494.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21646121', '21648453', '.', '+', '.', 'gene_id \"chr9.g13499\"; transcript_id \"chr9.g13499.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21649998', '21650300', '.', '-', '.', 'gene_id \"chr9.g13505\"; transcript_id \"chr9.g13505.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21650282', '21650772', '.', '+', '.', 'gene_id \"chr9.g13506\"; transcript_id \"chr9.g13506.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21654831', '21654986', '.', '-', '.', 'gene_id \"chr9.g13509\"; transcript_id \"chr9.g13509.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21655005', '21655226', '.', '-', '.', 'gene_id \"chr9.g13510\"; transcript_id \"chr9.g13510.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21660128', '21660622', '.', '+', '.', 'gene_id \"chr9.g13513\"; transcript_id \"chr9.g13513.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21673163', '21675282', '.', '+', '.', 'gene_id \"chr9.g13515\"; transcript_id \"chr9.g13515.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21673018', '21673178', '.', '-', '.', 'gene_id \"chr9.g13516\"; transcript_id \"chr9.g13516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21724012', '21724294', '.', '-', '.', 'gene_id \"chr9.g13520\"; transcript_id \"chr9.g13520.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21724012', '21724294', '.', '-', '.', 'gene_id \"chr9.g13520\"; transcript_id \"chr9.g13520.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21722979', '21723133', '.', '+', '.', 'gene_id \"chr9.g13522\"; transcript_id \"chr9.g13522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21707865', '21708322', '.', '+', '.', 'gene_id \"chr9.g13524\"; transcript_id \"chr9.g13524.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21708306', '21708607', '.', '-', '.', 'gene_id \"chr9.g13525\"; transcript_id \"chr9.g13525.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21709527', '21709878', '.', '-', '.', 'gene_id \"chr9.g13526\"; transcript_id \"chr9.g13526.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21710776', '21710935', '.', '-', '.', 'gene_id \"chr9.g13527\"; transcript_id \"chr9.g13527.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21715163', '21715474', '.', '-', '.', 'gene_id \"chr9.g13529\"; transcript_id \"chr9.g13529.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21718507', '21718672', '.', '+', '.', 'gene_id \"chr9.g13532\"; transcript_id \"chr9.g13532.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21726415', '21726749', '.', '+', '.', 'gene_id \"chr9.g13535\"; transcript_id \"chr9.g13535.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21729534', '21729830', '.', '+', '.', 'gene_id \"chr9.g13538\"; transcript_id \"chr9.g13538.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21729894', '21731130', '.', '+', '.', 'gene_id \"chr9.g13539\"; transcript_id \"chr9.g13539.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21731478', '21731737', '.', '-', '.', 'gene_id \"chr9.g13540\"; transcript_id \"chr9.g13540.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21731748', '21732197', '.', '+', '.', 'gene_id \"chr9.g13541\"; transcript_id \"chr9.g13541.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21732225', '21732435', '.', '-', '.', 'gene_id \"chr9.g13542\"; transcript_id \"chr9.g13542.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21752639', '21753022', '.', '-', '.', 'gene_id \"chr9.g13545\"; transcript_id \"chr9.g13545.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21737892', '21738448', '.', '-', '.', 'gene_id \"chr9.g13546\"; transcript_id \"chr9.g13546.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21742740', '21743590', '.', '-', '.', 'gene_id \"chr9.g13548\"; transcript_id \"chr9.g13548.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21751124', '21751683', '.', '-', '.', 'gene_id \"chr9.g13551\"; transcript_id \"chr9.g13551.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21753358', '21753781', '.', '+', '.', 'gene_id \"chr9.g13552\"; transcript_id \"chr9.g13552.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21749308', '21749984', '.', '+', '.', 'gene_id \"chr9.g13553\"; transcript_id \"chr9.g13553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21753820', '21754035', '.', '+', '.', 'gene_id \"chr9.g13556\"; transcript_id \"chr9.g13556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21773350', '21773429', '.', '-', '.', 'gene_id \"chr9.g13564\"; transcript_id \"chr9.g13564.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21773410', '21774020', '.', '-', '.', 'gene_id \"chr9.g13564\"; transcript_id \"chr9.g13564.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21800966', '21801114', '.', '+', '.', 'gene_id \"chr9.g13567\"; transcript_id \"chr9.g13567.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21774465', '21774687', '.', '+', '.', 'gene_id \"chr9.g13568\"; transcript_id \"chr9.g13568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21808248', '21808294', '.', '+', '.', 'gene_id \"chr9.g13571\"; transcript_id \"chr9.g13571.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21813023', '21813159', '.', '-', '.', 'gene_id \"chr9.g13572\"; transcript_id \"chr9.g13572.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21830205', '21830519', '.', '+', '.', 'gene_id \"chr9.g13581\"; transcript_id \"chr9.g13581.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21825699', '21825983', '.', '-', '.', 'gene_id \"chr9.g13584\"; transcript_id \"chr9.g13584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21833642', '21834479', '.', '+', '.', 'gene_id \"chr9.g13594\"; transcript_id \"chr9.g13594.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21834472', '21835005', '.', '-', '.', 'gene_id \"chr9.g13595\"; transcript_id \"chr9.g13595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21836587', '21837676', '.', '-', '.', 'gene_id \"chr9.g13596\"; transcript_id \"chr9.g13596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21838314', '21838901', '.', '+', '.', 'gene_id \"chr9.g13598\"; transcript_id \"chr9.g13598.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21838938', '21839150', '.', '+', '.', 'gene_id \"chr9.g13599\"; transcript_id \"chr9.g13599.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21839145', '21839305', '.', '-', '.', 'gene_id \"chr9.g13600\"; transcript_id \"chr9.g13600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21841958', '21842392', '.', '+', '.', 'gene_id \"chr9.g13602\"; transcript_id \"chr9.g13602.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21841650', '21841813', '.', '+', '.', 'gene_id \"chr9.g13604\"; transcript_id \"chr9.g13604.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21844676', '21845245', '.', '-', '.', 'gene_id \"chr9.g13608\"; transcript_id \"chr9.g13608.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21846679', '21846829', '.', '+', '.', 'gene_id \"chr9.g13612\"; transcript_id \"chr9.g13612.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21850211', '21850411', '.', '+', '.', 'gene_id \"chr9.g13620\"; transcript_id \"chr9.g13620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21853461', '21853791', '.', '+', '.', 'gene_id \"chr9.g13625\"; transcript_id \"chr9.g13625.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21870194', '21870776', '.', '-', '.', 'gene_id \"chr9.g13632\"; transcript_id \"chr9.g13632.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21870986', '21871172', '.', '-', '.', 'gene_id \"chr9.g13634\"; transcript_id \"chr9.g13634.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21883885', '21884061', '.', '-', '.', 'gene_id \"chr9.g13635\"; transcript_id \"chr9.g13635.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21884042', '21884366', '.', '+', '.', 'gene_id \"chr9.g13636\"; transcript_id \"chr9.g13636.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21912759', '21912949', '.', '+', '.', 'gene_id \"chr9.g13639\"; transcript_id \"chr9.g13639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21939603', '21939839', '.', '+', '.', 'gene_id \"chr9.g13641\"; transcript_id \"chr9.g13641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21893424', '21893584', '.', '+', '.', 'gene_id \"chr9.g13645\"; transcript_id \"chr9.g13645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21895042', '21895348', '.', '-', '.', 'gene_id \"chr9.g13647\"; transcript_id \"chr9.g13647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21895676', '21895831', '.', '+', '.', 'gene_id \"chr9.g13648\"; transcript_id \"chr9.g13648.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21896544', '21896712', '.', '-', '.', 'gene_id \"chr9.g13650\"; transcript_id \"chr9.g13650.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21900905', '21901237', '.', '+', '.', 'gene_id \"chr9.g13659\"; transcript_id \"chr9.g13659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21901609', '21901764', '.', '-', '.', 'gene_id \"chr9.g13661\"; transcript_id \"chr9.g13661.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21913979', '21914015', '.', '-', '.', 'gene_id \"chr9.g13665\"; transcript_id \"chr9.g13665.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21919249', '21919462', '.', '-', '.', 'gene_id \"chr9.g13668\"; transcript_id \"chr9.g13668.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21920244', '21920397', '.', '-', '.', 'gene_id \"chr9.g13669\"; transcript_id \"chr9.g13669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21920482', '21920541', '.', '+', '.', 'gene_id \"chr9.g13673\"; transcript_id \"chr9.g13673.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21922347', '21922527', '.', '-', '.', 'gene_id \"chr9.g13675\"; transcript_id \"chr9.g13675.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21922552', '21923133', '.', '+', '.', 'gene_id \"chr9.g13676\"; transcript_id \"chr9.g13676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21923116', '21923184', '.', '+', '.', 'gene_id \"chr9.g13676\"; transcript_id \"chr9.g13676.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21934136', '21934309', '.', '+', '.', 'gene_id \"chr9.g13685\"; transcript_id \"chr9.g13685.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21934156', '21934309', '.', '+', '.', 'gene_id \"chr9.g13685\"; transcript_id \"chr9.g13685.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21935504', '21935934', '.', '-', '.', 'gene_id \"chr9.g13688\"; transcript_id \"chr9.g13688.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21938595', '21938813', '.', '-', '.', 'gene_id \"chr9.g13697\"; transcript_id \"chr9.g13697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21940816', '21941040', '.', '-', '.', 'gene_id \"chr9.g13698\"; transcript_id \"chr9.g13698.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21940816', '21941167', '.', '-', '.', 'gene_id \"chr9.g13698\"; transcript_id \"chr9.g13698.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21952558', '21955213', '.', '+', '.', 'gene_id \"chr9.g13702\"; transcript_id \"chr9.g13702.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21951540', '21952191', '.', '+', '.', 'gene_id \"chr9.g13703\"; transcript_id \"chr9.g13703.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21959210', '21959457', '.', '+', '.', 'gene_id \"chr9.g13706\"; transcript_id \"chr9.g13706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21943816', '21944119', '.', '+', '.', 'gene_id \"chr9.g13708\"; transcript_id \"chr9.g13708.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21949007', '21949356', '.', '+', '.', 'gene_id \"chr9.g13712\"; transcript_id \"chr9.g13712.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21962693', '21962883', '.', '+', '.', 'gene_id \"chr9.g13714\"; transcript_id \"chr9.g13714.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21963002', '21963853', '.', '+', '.', 'gene_id \"chr9.g13715\"; transcript_id \"chr9.g13715.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21964285', '21964651', '.', '-', '.', 'gene_id \"chr9.g13716\"; transcript_id \"chr9.g13716.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21969714', '21971142', '.', '+', '.', 'gene_id \"chr9.g13719\"; transcript_id \"chr9.g13719.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21993388', '21993706', '.', '-', '.', 'gene_id \"chr9.g13721\"; transcript_id \"chr9.g13721.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21976414', '21976809', '.', '+', '.', 'gene_id \"chr9.g13723\"; transcript_id \"chr9.g13723.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21974804', '21975478', '.', '-', '.', 'gene_id \"chr9.g13724\"; transcript_id \"chr9.g13724.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21972930', '21973202', '.', '-', '.', 'gene_id \"chr9.g13726\"; transcript_id \"chr9.g13726.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21979926', '21980609', '.', '+', '.', 'gene_id \"chr9.g13731\"; transcript_id \"chr9.g13731.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21981013', '21981176', '.', '-', '.', 'gene_id \"chr9.g13732\"; transcript_id \"chr9.g13732.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21982062', '21982398', '.', '-', '.', 'gene_id \"chr9.g13736\"; transcript_id \"chr9.g13736.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '29820346', '29823143', '.', '+', '.', 'gene_id \"chr17.g10234\"; transcript_id \"chr17.g10234.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '29820346', '29823143', '.', '+', '.', 'gene_id \"chr17.g10234\"; transcript_id \"chr17.g10234.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21991165', '21991427', '.', '-', '.', 'gene_id \"chr9.g13752\"; transcript_id \"chr9.g13752.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21991430', '21992454', '.', '+', '.', 'gene_id \"chr9.g13753\"; transcript_id \"chr9.g13753.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21999612', '22000079', '.', '-', '.', 'gene_id \"chr9.g13754\"; transcript_id \"chr9.g13754.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22011633', '22011861', '.', '-', '.', 'gene_id \"chr9.g13760\"; transcript_id \"chr9.g13760.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22014261', '22014440', '.', '+', '.', 'gene_id \"chr9.g13761\"; transcript_id \"chr9.g13761.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22047234', '22048458', '.', '+', '.', 'gene_id \"chr9.g13767\"; transcript_id \"chr9.g13767.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22032472', '22032765', '.', '+', '.', 'gene_id \"chr9.g13769\"; transcript_id \"chr9.g13769.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22033384', '22033735', '.', '-', '.', 'gene_id \"chr9.g13770\"; transcript_id \"chr9.g13770.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22033880', '22034475', '.', '-', '.', 'gene_id \"chr9.g13772\"; transcript_id \"chr9.g13772.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22029026', '22029300', '.', '-', '.', 'gene_id \"chr9.g13773\"; transcript_id \"chr9.g13773.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22054095', '22054270', '.', '-', '.', 'gene_id \"chr9.g13789\"; transcript_id \"chr9.g13789.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22082884', '22083067', '.', '-', '.', 'gene_id \"chr9.g13801\"; transcript_id \"chr9.g13801.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22086591', '22087071', '.', '-', '.', 'gene_id \"chr9.g13806\"; transcript_id \"chr9.g13806.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22087845', '22088196', '.', '-', '.', 'gene_id \"chr9.g13807\"; transcript_id \"chr9.g13807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22091209', '22091416', '.', '+', '.', 'gene_id \"chr9.g13808\"; transcript_id \"chr9.g13808.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22094835', '22094957', '.', '-', '.', 'gene_id \"chr9.g13812\"; transcript_id \"chr9.g13812.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22079648', '22080041', '.', '+', '.', 'gene_id \"chr9.g13813\"; transcript_id \"chr9.g13813.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22072066', '22072304', '.', '+', '.', 'gene_id \"chr9.g13818\"; transcript_id \"chr9.g13818.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22074278', '22074462', '.', '+', '.', 'gene_id \"chr9.g13819\"; transcript_id \"chr9.g13819.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22091505', '22091791', '.', '-', '.', 'gene_id \"chr9.g13828\"; transcript_id \"chr9.g13828.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22091960', '22092528', '.', '+', '.', 'gene_id \"chr9.g13830\"; transcript_id \"chr9.g13830.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22111485', '22112248', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22110358', '22111059', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22106490', '22106670', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22106126', '22106427', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22105036', '22106012', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22104355', '22105026', '.', '-', '.', 'gene_id \"chr9.g13831\"; transcript_id \"chr9.g13831.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22113557', '22113973', '.', '+', '.', 'gene_id \"chr9.g13832\"; transcript_id \"chr9.g13832.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22112487', '22112913', '.', '+', '.', 'gene_id \"chr9.g13838\"; transcript_id \"chr9.g13838.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22113164', '22113309', '.', '+', '.', 'gene_id \"chr9.g13840\"; transcript_id \"chr9.g13840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22114289', '22114445', '.', '+', '.', 'gene_id \"chr9.g13841\"; transcript_id \"chr9.g13841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22114486', '22114649', '.', '-', '.', 'gene_id \"chr9.g13842\"; transcript_id \"chr9.g13842.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22114798', '22114963', '.', '-', '.', 'gene_id \"chr9.g13843\"; transcript_id \"chr9.g13843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22124039', '22124206', '.', '-', '.', 'gene_id \"chr9.g13852\"; transcript_id \"chr9.g13852.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22120945', '22121205', '.', '-', '.', 'gene_id \"chr9.g13852\"; transcript_id \"chr9.g13852.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22125264', '22125581', '.', '+', '.', 'gene_id \"chr9.g13854\"; transcript_id \"chr9.g13854.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22124316', '22124470', '.', '+', '.', 'gene_id \"chr9.g13858\"; transcript_id \"chr9.g13858.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22124949', '22125150', '.', '+', '.', 'gene_id \"chr9.g13859\"; transcript_id \"chr9.g13859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22126565', '22126758', '.', '-', '.', 'gene_id \"chr9.g13860\"; transcript_id \"chr9.g13860.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22127347', '22127624', '.', '+', '.', 'gene_id \"chr9.g13861\"; transcript_id \"chr9.g13861.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22128303', '22128484', '.', '-', '.', 'gene_id \"chr9.g13862\"; transcript_id \"chr9.g13862.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22128630', '22129062', '.', '-', '.', 'gene_id \"chr9.g13863\"; transcript_id \"chr9.g13863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22129068', '22129315', '.', '+', '.', 'gene_id \"chr9.g13864\"; transcript_id \"chr9.g13864.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22129512', '22129672', '.', '+', '.', 'gene_id \"chr9.g13865\"; transcript_id \"chr9.g13865.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22129512', '22129696', '.', '+', '.', 'gene_id \"chr9.g13865\"; transcript_id \"chr9.g13865.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22142840', '22143000', '.', '+', '.', 'gene_id \"chr9.g13874\"; transcript_id \"chr9.g13874.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22145845', '22146011', '.', '-', '.', 'gene_id \"chr9.g13876\"; transcript_id \"chr9.g13876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22165032', '22165497', '.', '-', '.', 'gene_id \"chr9.g13880\"; transcript_id \"chr9.g13880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22161853', '22162073', '.', '-', '.', 'gene_id \"chr9.g13882\"; transcript_id \"chr9.g13882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22186102', '22186277', '.', '-', '.', 'gene_id \"chr9.g13896\"; transcript_id \"chr9.g13896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22204961', '22205112', '.', '+', '.', 'gene_id \"chr9.g13910\"; transcript_id \"chr9.g13910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22200171', '22200478', '.', '+', '.', 'gene_id \"chr9.g13911\"; transcript_id \"chr9.g13911.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22212494', '22212653', '.', '+', '.', 'gene_id \"chr9.g13912\"; transcript_id \"chr9.g13912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22231905', '22232100', '.', '+', '.', 'gene_id \"chr9.g13913\"; transcript_id \"chr9.g13913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22238564', '22238723', '.', '+', '.', 'gene_id \"chr9.g13916\"; transcript_id \"chr9.g13916.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22239160', '22239319', '.', '+', '.', 'gene_id \"chr9.g13918\"; transcript_id \"chr9.g13918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22239360', '22239810', '.', '-', '.', 'gene_id \"chr9.g13919\"; transcript_id \"chr9.g13919.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22240657', '22240809', '.', '+', '.', 'gene_id \"chr9.g13921\"; transcript_id \"chr9.g13921.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22241023', '22241149', '.', '-', '.', 'gene_id \"chr9.g13924\"; transcript_id \"chr9.g13924.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22293990', '22296376', '.', '-', '.', 'gene_id \"chr9.g13933\"; transcript_id \"chr9.g13933.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22272722', '22273076', '.', '+', '.', 'gene_id \"chr9.g13937\"; transcript_id \"chr9.g13937.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22269667', '22269821', '.', '+', '.', 'gene_id \"chr9.g13938\"; transcript_id \"chr9.g13938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22310992', '22311070', '.', '+', '.', 'gene_id \"chr9.g13955\"; transcript_id \"chr9.g13955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22310965', '22311025', '.', '+', '.', 'gene_id \"chr9.g13955\"; transcript_id \"chr9.g13955.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22309657', '22310314', '.', '+', '.', 'gene_id \"chr9.g13957\"; transcript_id \"chr9.g13957.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22315328', '22315489', '.', '+', '.', 'gene_id \"chr9.g13959\"; transcript_id \"chr9.g13959.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22315804', '22316235', '.', '+', '.', 'gene_id \"chr9.g13961\"; transcript_id \"chr9.g13961.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22321390', '22321772', '.', '+', '.', 'gene_id \"chr9.g13963\"; transcript_id \"chr9.g13963.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22323454', '22323629', '.', '-', '.', 'gene_id \"chr9.g13966\"; transcript_id \"chr9.g13966.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22330091', '22330266', '.', '-', '.', 'gene_id \"chr9.g13970\"; transcript_id \"chr9.g13970.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22339410', '22339694', '.', '-', '.', 'gene_id \"chr9.g13970\"; transcript_id \"chr9.g13970.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22329420', '22329772', '.', '-', '.', 'gene_id \"chr9.g13970\"; transcript_id \"chr9.g13970.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22331755', '22333148', '.', '+', '.', 'gene_id \"chr9.g13977\"; transcript_id \"chr9.g13977.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22329119', '22329301', '.', '-', '.', 'gene_id \"chr9.g13979\"; transcript_id \"chr9.g13979.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22328865', '22329141', '.', '+', '.', 'gene_id \"chr9.g13981\"; transcript_id \"chr9.g13981.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22328573', '22328777', '.', '-', '.', 'gene_id \"chr9.g13982\"; transcript_id \"chr9.g13982.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22334060', '22334248', '.', '-', '.', 'gene_id \"chr9.g13987\"; transcript_id \"chr9.g13987.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22338114', '22338413', '.', '-', '.', 'gene_id \"chr9.g13992\"; transcript_id \"chr9.g13992.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22341202', '22341361', '.', '-', '.', 'gene_id \"chr9.g13993\"; transcript_id \"chr9.g13993.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30330896', '30331111', '.', '-', '.', 'gene_id \"chr17.g10449\"; transcript_id \"chr17.g10449.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22349481', '22349688', '.', '-', '.', 'gene_id \"chr9.g14005\"; transcript_id \"chr9.g14005.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22350017', '22350243', '.', '+', '.', 'gene_id \"chr9.g14007\"; transcript_id \"chr9.g14007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22372340', '22374260', '.', '-', '.', 'gene_id \"chr9.g14011\"; transcript_id \"chr9.g14011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22365377', '22365618', '.', '+', '.', 'gene_id \"chr9.g14015\"; transcript_id \"chr9.g14015.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22365631', '22366004', '.', '+', '.', 'gene_id \"chr9.g14016\"; transcript_id \"chr9.g14016.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22382837', '22383124', '.', '-', '.', 'gene_id \"chr9.g14021\"; transcript_id \"chr9.g14021.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22382837', '22383124', '.', '-', '.', 'gene_id \"chr9.g14021\"; transcript_id \"chr9.g14021.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22386541', '22386711', '.', '-', '.', 'gene_id \"chr9.g14022\"; transcript_id \"chr9.g14022.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22382359', '22382550', '.', '-', '.', 'gene_id \"chr9.g14026\"; transcript_id \"chr9.g14026.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22382599', '22382852', '.', '-', '.', 'gene_id \"chr9.g14029\"; transcript_id \"chr9.g14029.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22378910', '22379176', '.', '-', '.', 'gene_id \"chr9.g14031\"; transcript_id \"chr9.g14031.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22400390', '22400618', '.', '-', '.', 'gene_id \"chr9.g14044\"; transcript_id \"chr9.g14044.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22400865', '22401120', '.', '+', '.', 'gene_id \"chr9.g14045\"; transcript_id \"chr9.g14045.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22401705', '22402044', '.', '+', '.', 'gene_id \"chr9.g14047\"; transcript_id \"chr9.g14047.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22402051', '22402306', '.', '+', '.', 'gene_id \"chr9.g14048\"; transcript_id \"chr9.g14048.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22408378', '22410572', '.', '-', '.', 'gene_id \"chr9.g14049\"; transcript_id \"chr9.g14049.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22407171', '22407379', '.', '+', '.', 'gene_id \"chr9.g14050\"; transcript_id \"chr9.g14050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22406988', '22407234', '.', '+', '.', 'gene_id \"chr9.g14050\"; transcript_id \"chr9.g14050.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22411954', '22412087', '.', '+', '.', 'gene_id \"chr9.g14051\"; transcript_id \"chr9.g14051.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22411415', '22411970', '.', '+', '.', 'gene_id \"chr9.g14051\"; transcript_id \"chr9.g14051.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22412007', '22412137', '.', '+', '.', 'gene_id \"chr9.g14051\"; transcript_id \"chr9.g14051.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22405733', '22406826', '.', '+', '.', 'gene_id \"chr9.g14055\"; transcript_id \"chr9.g14055.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22407431', '22408357', '.', '+', '.', 'gene_id \"chr9.g14056\"; transcript_id \"chr9.g14056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22431852', '22433851', '.', '+', '.', 'gene_id \"chr9.g14061\"; transcript_id \"chr9.g14061.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22430472', '22430621', '.', '-', '.', 'gene_id \"chr9.g14063\"; transcript_id \"chr9.g14063.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22431488', '22431847', '.', '+', '.', 'gene_id \"chr9.g14065\"; transcript_id \"chr9.g14065.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22449073', '22449259', '.', '+', '.', 'gene_id \"chr9.g14066\"; transcript_id \"chr9.g14066.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22462079', '22462337', '.', '+', '.', 'gene_id \"chr9.g14071\"; transcript_id \"chr9.g14071.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22462505', '22462803', '.', '+', '.', 'gene_id \"chr9.g14072\"; transcript_id \"chr9.g14072.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22463283', '22463511', '.', '-', '.', 'gene_id \"chr9.g14074\"; transcript_id \"chr9.g14074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22463102', '22463283', '.', '+', '.', 'gene_id \"chr9.g14075\"; transcript_id \"chr9.g14075.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22478354', '22480159', '.', '+', '.', 'gene_id \"chr9.g14082\"; transcript_id \"chr9.g14082.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22479299', '22480159', '.', '+', '.', 'gene_id \"chr9.g14082\"; transcript_id \"chr9.g14082.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22465923', '22466057', '.', '+', '.', 'gene_id \"chr9.g14083\"; transcript_id \"chr9.g14083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22465559', '22467386', '.', '+', '.', 'gene_id \"chr9.g14083\"; transcript_id \"chr9.g14083.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22467718', '22467874', '.', '-', '.', 'gene_id \"chr9.g14085\"; transcript_id \"chr9.g14085.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22470192', '22470746', '.', '-', '.', 'gene_id \"chr9.g14086\"; transcript_id \"chr9.g14086.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22470730', '22472728', '.', '+', '.', 'gene_id \"chr9.g14087\"; transcript_id \"chr9.g14087.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22472950', '22473107', '.', '+', '.', 'gene_id \"chr9.g14089\"; transcript_id \"chr9.g14089.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22521099', '22521249', '.', '+', '.', 'gene_id \"chr9.g14090\"; transcript_id \"chr9.g14090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22475582', '22478207', '.', '+', '.', 'gene_id \"chr9.g14092\"; transcript_id \"chr9.g14092.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22469771', '22469984', '.', '-', '.', 'gene_id \"chr9.g14093\"; transcript_id \"chr9.g14093.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22485266', '22485465', '.', '-', '.', 'gene_id \"chr9.g14102\"; transcript_id \"chr9.g14102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22532336', '22532584', '.', '-', '.', 'gene_id \"chr9.g14115\"; transcript_id \"chr9.g14115.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22533921', '22534148', '.', '+', '.', 'gene_id \"chr9.g14116\"; transcript_id \"chr9.g14116.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30531106', '30531275', '.', '+', '.', 'gene_id \"chr17.g10547\"; transcript_id \"chr17.g10547.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22574087', '22574404', '.', '-', '.', 'gene_id \"chr9.g14127\"; transcript_id \"chr9.g14127.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22575969', '22576122', '.', '-', '.', 'gene_id \"chr9.g14128\"; transcript_id \"chr9.g14128.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22575969', '22576110', '.', '-', '.', 'gene_id \"chr9.g14128\"; transcript_id \"chr9.g14128.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22573116', '22573479', '.', '-', '.', 'gene_id \"chr9.g14132\"; transcript_id \"chr9.g14132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22567054', '22567413', '.', '-', '.', 'gene_id \"chr9.g14133\"; transcript_id \"chr9.g14133.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22575332', '22575526', '.', '+', '.', 'gene_id \"chr9.g14134\"; transcript_id \"chr9.g14134.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22566323', '22566461', '.', '+', '.', 'gene_id \"chr9.g14141\"; transcript_id \"chr9.g14141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22566133', '22566293', '.', '-', '.', 'gene_id \"chr9.g14142\"; transcript_id \"chr9.g14142.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30531983', '30532330', '.', '-', '.', 'gene_id \"chr17.g10550\"; transcript_id \"chr17.g10550.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22561058', '22561284', '.', '-', '.', 'gene_id \"chr9.g14146\"; transcript_id \"chr9.g14146.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22561285', '22561730', '.', '-', '.', 'gene_id \"chr9.g14147\"; transcript_id \"chr9.g14147.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22561723', '22561984', '.', '-', '.', 'gene_id \"chr9.g14149\"; transcript_id \"chr9.g14149.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22561980', '22562131', '.', '-', '.', 'gene_id \"chr9.g14151\"; transcript_id \"chr9.g14151.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22562171', '22562386', '.', '-', '.', 'gene_id \"chr9.g14152\"; transcript_id \"chr9.g14152.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22572002', '22572161', '.', '-', '.', 'gene_id \"chr9.g14161\"; transcript_id \"chr9.g14161.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22574419', '22574648', '.', '+', '.', 'gene_id \"chr9.g14162\"; transcript_id \"chr9.g14162.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22574419', '22574620', '.', '+', '.', 'gene_id \"chr9.g14162\"; transcript_id \"chr9.g14162.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22576470', '22576660', '.', '+', '.', 'gene_id \"chr9.g14165\"; transcript_id \"chr9.g14165.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22576424', '22576518', '.', '+', '.', 'gene_id \"chr9.g14165\"; transcript_id \"chr9.g14165.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22576793', '22577070', '.', '-', '.', 'gene_id \"chr9.g14167\"; transcript_id \"chr9.g14167.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22578201', '22578331', '.', '+', '.', 'gene_id \"chr9.g14168\"; transcript_id \"chr9.g14168.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22586236', '22586446', '.', '-', '.', 'gene_id \"chr9.g14173\"; transcript_id \"chr9.g14173.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22586660', '22586824', '.', '+', '.', 'gene_id \"chr9.g14174\"; transcript_id \"chr9.g14174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22644763', '22644823', '.', '+', '.', 'gene_id \"chr9.g14175\"; transcript_id \"chr9.g14175.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22644756', '22644823', '.', '+', '.', 'gene_id \"chr9.g14175\"; transcript_id \"chr9.g14175.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22587969', '22588261', '.', '+', '.', 'gene_id \"chr9.g14175\"; transcript_id \"chr9.g14175.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22690834', '22691837', '.', '-', '.', 'gene_id \"chr9.g14176\"; transcript_id \"chr9.g14176.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22586853', '22587021', '.', '+', '.', 'gene_id \"chr9.g14177\"; transcript_id \"chr9.g14177.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22663387', '22663560', '.', '+', '.', 'gene_id \"chr9.g14179\"; transcript_id \"chr9.g14179.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22685830', '22686227', '.', '+', '.', 'gene_id \"chr9.g14181\"; transcript_id \"chr9.g14181.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22686213', '22687378', '.', '-', '.', 'gene_id \"chr9.g14183\"; transcript_id \"chr9.g14183.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22673577', '22673940', '.', '+', '.', 'gene_id \"chr9.g14189\"; transcript_id \"chr9.g14189.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22600755', '22601042', '.', '+', '.', 'gene_id \"chr9.g14201\"; transcript_id \"chr9.g14201.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22605519', '22605970', '.', '+', '.', 'gene_id \"chr9.g14205\"; transcript_id \"chr9.g14205.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22605568', '22605652', '.', '+', '.', 'gene_id \"chr9.g14205\"; transcript_id \"chr9.g14205.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22613655', '22613757', '.', '-', '.', 'gene_id \"chr9.g14214\"; transcript_id \"chr9.g14214.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22622271', '22622423', '.', '+', '.', 'gene_id \"chr9.g14217\"; transcript_id \"chr9.g14217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22622692', '22623005', '.', '+', '.', 'gene_id \"chr9.g14219\"; transcript_id \"chr9.g14219.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22624227', '22624427', '.', '-', '.', 'gene_id \"chr9.g14220\"; transcript_id \"chr9.g14220.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22624406', '22624615', '.', '-', '.', 'gene_id \"chr9.g14220\"; transcript_id \"chr9.g14220.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22624849', '22625041', '.', '-', '.', 'gene_id \"chr9.g14221\"; transcript_id \"chr9.g14221.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22625623', '22626134', '.', '+', '.', 'gene_id \"chr9.g14224\"; transcript_id \"chr9.g14224.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22626976', '22627152', '.', '+', '.', 'gene_id \"chr9.g14225\"; transcript_id \"chr9.g14225.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22628810', '22629086', '.', '+', '.', 'gene_id \"chr9.g14229\"; transcript_id \"chr9.g14229.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22629687', '22629984', '.', '+', '.', 'gene_id \"chr9.g14230\"; transcript_id \"chr9.g14230.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22629311', '22629591', '.', '+', '.', 'gene_id \"chr9.g14232\"; transcript_id \"chr9.g14232.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22629967', '22630118', '.', '-', '.', 'gene_id \"chr9.g14233\"; transcript_id \"chr9.g14233.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22632138', '22632700', '.', '-', '.', 'gene_id \"chr9.g14236\"; transcript_id \"chr9.g14236.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22637277', '22637504', '.', '+', '.', 'gene_id \"chr9.g14242\"; transcript_id \"chr9.g14242.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22652965', '22653238', '.', '-', '.', 'gene_id \"chr9.g14248\"; transcript_id \"chr9.g14248.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22653380', '22653455', '.', '+', '.', 'gene_id \"chr9.g14250\"; transcript_id \"chr9.g14250.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30598442', '30598683', '.', '-', '.', 'gene_id \"chr17.g10651\"; transcript_id \"chr17.g10651.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30598675', '30599187', '.', '+', '.', 'gene_id \"chr17.g10652\"; transcript_id \"chr17.g10652.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30599168', '30599651', '.', '+', '.', 'gene_id \"chr17.g10653\"; transcript_id \"chr17.g10653.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22661730', '22661958', '.', '-', '.', 'gene_id \"chr9.g14262\"; transcript_id \"chr9.g14262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22664968', '22665640', '.', '-', '.', 'gene_id \"chr9.g14265\"; transcript_id \"chr9.g14265.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22665656', '22665999', '.', '+', '.', 'gene_id \"chr9.g14268\"; transcript_id \"chr9.g14268.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22666111', '22666616', '.', '+', '.', 'gene_id \"chr9.g14270\"; transcript_id \"chr9.g14270.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22666519', '22666616', '.', '+', '.', 'gene_id \"chr9.g14270\"; transcript_id \"chr9.g14270.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22668122', '22669537', '.', '+', '.', 'gene_id \"chr9.g14272\"; transcript_id \"chr9.g14272.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22669640', '22669936', '.', '+', '.', 'gene_id \"chr9.g14274\"; transcript_id \"chr9.g14274.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22670693', '22670956', '.', '-', '.', 'gene_id \"chr9.g14276\"; transcript_id \"chr9.g14276.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22672465', '22672788', '.', '-', '.', 'gene_id \"chr9.g14279\"; transcript_id \"chr9.g14279.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22672773', '22673087', '.', '+', '.', 'gene_id \"chr9.g14280\"; transcript_id \"chr9.g14280.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22673193', '22673575', '.', '+', '.', 'gene_id \"chr9.g14281\"; transcript_id \"chr9.g14281.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22674524', '22674706', '.', '-', '.', 'gene_id \"chr9.g14282\"; transcript_id \"chr9.g14282.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22676868', '22677021', '.', '-', '.', 'gene_id \"chr9.g14284\"; transcript_id \"chr9.g14284.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22680688', '22680928', '.', '-', '.', 'gene_id \"chr9.g14289\"; transcript_id \"chr9.g14289.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22683272', '22683724', '.', '-', '.', 'gene_id \"chr9.g14291\"; transcript_id \"chr9.g14291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22685228', '22685408', '.', '-', '.', 'gene_id \"chr9.g14295\"; transcript_id \"chr9.g14295.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22696531', '22697626', '.', '-', '.', 'gene_id \"chr9.g14297\"; transcript_id \"chr9.g14297.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22705658', '22705877', '.', '+', '.', 'gene_id \"chr9.g14298\"; transcript_id \"chr9.g14298.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22696090', '22696191', '.', '-', '.', 'gene_id \"chr9.g14301\"; transcript_id \"chr9.g14301.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22696172', '22696349', '.', '-', '.', 'gene_id \"chr9.g14301\"; transcript_id \"chr9.g14301.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22701768', '22702044', '.', '-', '.', 'gene_id \"chr9.g14302\"; transcript_id \"chr9.g14302.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22702607', '22703006', '.', '+', '.', 'gene_id \"chr9.g14304\"; transcript_id \"chr9.g14304.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22713327', '22713504', '.', '-', '.', 'gene_id \"chr9.g14311\"; transcript_id \"chr9.g14311.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22716522', '22716741', '.', '-', '.', 'gene_id \"chr9.g14315\"; transcript_id \"chr9.g14315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22717784', '22717962', '.', '+', '.', 'gene_id \"chr9.g14318\"; transcript_id \"chr9.g14318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22727193', '22727349', '.', '-', '.', 'gene_id \"chr9.g14320\"; transcript_id \"chr9.g14320.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22725991', '22726192', '.', '-', '.', 'gene_id \"chr9.g14322\"; transcript_id \"chr9.g14322.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22738030', '22738438', '.', '-', '.', 'gene_id \"chr9.g14331\"; transcript_id \"chr9.g14331.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22738517', '22739171', '.', '-', '.', 'gene_id \"chr9.g14331\"; transcript_id \"chr9.g14331.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22737591', '22737964', '.', '+', '.', 'gene_id \"chr9.g14333\"; transcript_id \"chr9.g14333.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22751835', '22752425', '.', '+', '.', 'gene_id \"chr9.g14334\"; transcript_id \"chr9.g14334.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22759013', '22759372', '.', '+', '.', 'gene_id \"chr9.g14335\"; transcript_id \"chr9.g14335.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22759774', '22760069', '.', '-', '.', 'gene_id \"chr9.g14336\"; transcript_id \"chr9.g14336.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22752599', '22752773', '.', '+', '.', 'gene_id \"chr9.g14340\"; transcript_id \"chr9.g14340.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22747600', '22747766', '.', '-', '.', 'gene_id \"chr9.g14341\"; transcript_id \"chr9.g14341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22747783', '22747953', '.', '+', '.', 'gene_id \"chr9.g14342\"; transcript_id \"chr9.g14342.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22747934', '22748345', '.', '-', '.', 'gene_id \"chr9.g14343\"; transcript_id \"chr9.g14343.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22754681', '22754864', '.', '+', '.', 'gene_id \"chr9.g14346\"; transcript_id \"chr9.g14346.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22756000', '22756319', '.', '+', '.', 'gene_id \"chr9.g14347\"; transcript_id \"chr9.g14347.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22756988', '22757285', '.', '+', '.', 'gene_id \"chr9.g14351\"; transcript_id \"chr9.g14351.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22756733', '22757004', '.', '+', '.', 'gene_id \"chr9.g14353\"; transcript_id \"chr9.g14353.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22771301', '22771471', '.', '-', '.', 'gene_id \"chr9.g14355\"; transcript_id \"chr9.g14355.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22772103', '22772322', '.', '+', '.', 'gene_id \"chr9.g14356\"; transcript_id \"chr9.g14356.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22772612', '22772959', '.', '-', '.', 'gene_id \"chr9.g14358\"; transcript_id \"chr9.g14358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22820012', '22820138', '.', '-', '.', 'gene_id \"chr9.g14365\"; transcript_id \"chr9.g14365.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22806062', '22806402', '.', '+', '.', 'gene_id \"chr9.g14366\"; transcript_id \"chr9.g14366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22777519', '22777930', '.', '+', '.', 'gene_id \"chr9.g14370\"; transcript_id \"chr9.g14370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22777940', '22778345', '.', '+', '.', 'gene_id \"chr9.g14371\"; transcript_id \"chr9.g14371.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22779572', '22779775', '.', '+', '.', 'gene_id \"chr9.g14373\"; transcript_id \"chr9.g14373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22780576', '22780837', '.', '+', '.', 'gene_id \"chr9.g14377\"; transcript_id \"chr9.g14377.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22786220', '22786409', '.', '+', '.', 'gene_id \"chr9.g14378\"; transcript_id \"chr9.g14378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22788000', '22788170', '.', '-', '.', 'gene_id \"chr9.g14379\"; transcript_id \"chr9.g14379.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22788063', '22788171', '.', '-', '.', 'gene_id \"chr9.g14379\"; transcript_id \"chr9.g14379.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22789177', '22789342', '.', '-', '.', 'gene_id \"chr9.g14382\"; transcript_id \"chr9.g14382.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22790752', '22790906', '.', '+', '.', 'gene_id \"chr9.g14383\"; transcript_id \"chr9.g14383.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22793547', '22793738', '.', '+', '.', 'gene_id \"chr9.g14388\"; transcript_id \"chr9.g14388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22799505', '22799655', '.', '+', '.', 'gene_id \"chr9.g14392\"; transcript_id \"chr9.g14392.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22803423', '22803601', '.', '+', '.', 'gene_id \"chr9.g14397\"; transcript_id \"chr9.g14397.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22822677', '22822837', '.', '-', '.', 'gene_id \"chr9.g14403\"; transcript_id \"chr9.g14403.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22832578', '22832742', '.', '+', '.', 'gene_id \"chr9.g14410\"; transcript_id \"chr9.g14410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22833263', '22833544', '.', '+', '.', 'gene_id \"chr9.g14413\"; transcript_id \"chr9.g14413.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22834221', '22834464', '.', '+', '.', 'gene_id \"chr9.g14414\"; transcript_id \"chr9.g14414.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22901291', '22901468', '.', '-', '.', 'gene_id \"chr9.g14433\"; transcript_id \"chr9.g14433.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22901479', '22902093', '.', '-', '.', 'gene_id \"chr9.g14434\"; transcript_id \"chr9.g14434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22902104', '22902681', '.', '+', '.', 'gene_id \"chr9.g14435\"; transcript_id \"chr9.g14435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22903917', '22904090', '.', '+', '.', 'gene_id \"chr9.g14437\"; transcript_id \"chr9.g14437.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22904207', '22904949', '.', '-', '.', 'gene_id \"chr9.g14439\"; transcript_id \"chr9.g14439.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22906611', '22906749', '.', '+', '.', 'gene_id \"chr9.g14440\"; transcript_id \"chr9.g14440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22911903', '22912120', '.', '+', '.', 'gene_id \"chr9.g14442\"; transcript_id \"chr9.g14442.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22925680', '22926089', '.', '-', '.', 'gene_id \"chr9.g14445\"; transcript_id \"chr9.g14445.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22926084', '22927171', '.', '+', '.', 'gene_id \"chr9.g14446\"; transcript_id \"chr9.g14446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22916304', '22916480', '.', '-', '.', 'gene_id \"chr9.g14452\"; transcript_id \"chr9.g14452.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22916304', '22916437', '.', '-', '.', 'gene_id \"chr9.g14452\"; transcript_id \"chr9.g14452.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22915802', '22916026', '.', '-', '.', 'gene_id \"chr9.g14453\"; transcript_id \"chr9.g14453.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22928077', '22928263', '.', '-', '.', 'gene_id \"chr9.g14456\"; transcript_id \"chr9.g14456.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22938274', '22938524', '.', '+', '.', 'gene_id \"chr9.g14469\"; transcript_id \"chr9.g14469.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22940188', '22940688', '.', '+', '.', 'gene_id \"chr9.g14473\"; transcript_id \"chr9.g14473.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22945065', '22945365', '.', '+', '.', 'gene_id \"chr9.g14479\"; transcript_id \"chr9.g14479.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22968489', '22968641', '.', '-', '.', 'gene_id \"chr9.g14496\"; transcript_id \"chr9.g14496.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22968992', '22969716', '.', '+', '.', 'gene_id \"chr9.g14497\"; transcript_id \"chr9.g14497.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22963518', '22963716', '.', '-', '.', 'gene_id \"chr9.g14499\"; transcript_id \"chr9.g14499.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22965794', '22966177', '.', '+', '.', 'gene_id \"chr9.g14501\"; transcript_id \"chr9.g14501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22973742', '22973860', '.', '+', '.', 'gene_id \"chr9.g14508\"; transcript_id \"chr9.g14508.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22972807', '22973309', '.', '+', '.', 'gene_id \"chr9.g14509\"; transcript_id \"chr9.g14509.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22975674', '22976212', '.', '-', '.', 'gene_id \"chr9.g14513\"; transcript_id \"chr9.g14513.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22977887', '22977927', '.', '+', '.', 'gene_id \"chr9.g14515\"; transcript_id \"chr9.g14515.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23004284', '23004539', '.', '+', '.', 'gene_id \"chr9.g14519\"; transcript_id \"chr9.g14519.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23009446', '23009891', '.', '-', '.', 'gene_id \"chr9.g14520\"; transcript_id \"chr9.g14520.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23005023', '23005254', '.', '-', '.', 'gene_id \"chr9.g14521\"; transcript_id \"chr9.g14521.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23005415', '23005838', '.', '-', '.', 'gene_id \"chr9.g14524\"; transcript_id \"chr9.g14524.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23004579', '23005045', '.', '-', '.', 'gene_id \"chr9.g14525\"; transcript_id \"chr9.g14525.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23002823', '23002998', '.', '-', '.', 'gene_id \"chr9.g14526\"; transcript_id \"chr9.g14526.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23009971', '23010185', '.', '+', '.', 'gene_id \"chr9.g14527\"; transcript_id \"chr9.g14527.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23012565', '23012787', '.', '+', '.', 'gene_id \"chr9.g14539\"; transcript_id \"chr9.g14539.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23030199', '23031266', '.', '-', '.', 'gene_id \"chr9.g14540\"; transcript_id \"chr9.g14540.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23031121', '23031266', '.', '-', '.', 'gene_id \"chr9.g14540\"; transcript_id \"chr9.g14540.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23034689', '23035188', '.', '-', '.', 'gene_id \"chr9.g14544\"; transcript_id \"chr9.g14544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23028743', '23029133', '.', '-', '.', 'gene_id \"chr9.g14547\"; transcript_id \"chr9.g14547.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23012941', '23013100', '.', '+', '.', 'gene_id \"chr9.g14549\"; transcript_id \"chr9.g14549.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23035413', '23035891', '.', '+', '.', 'gene_id \"chr9.g14551\"; transcript_id \"chr9.g14551.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23013313', '23013619', '.', '-', '.', 'gene_id \"chr9.g14554\"; transcript_id \"chr9.g14554.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23013675', '23013867', '.', '-', '.', 'gene_id \"chr9.g14556\"; transcript_id \"chr9.g14556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23015981', '23016136', '.', '+', '.', 'gene_id \"chr9.g14560\"; transcript_id \"chr9.g14560.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23016819', '23017028', '.', '+', '.', 'gene_id \"chr9.g14562\"; transcript_id \"chr9.g14562.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23020939', '23021093', '.', '-', '.', 'gene_id \"chr9.g14568\"; transcript_id \"chr9.g14568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23027011', '23028003', '.', '+', '.', 'gene_id \"chr9.g14574\"; transcript_id \"chr9.g14574.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23032920', '23033205', '.', '-', '.', 'gene_id \"chr9.g14578\"; transcript_id \"chr9.g14578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23034407', '23034615', '.', '+', '.', 'gene_id \"chr9.g14582\"; transcript_id \"chr9.g14582.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23047028', '23047199', '.', '+', '.', 'gene_id \"chr9.g14585\"; transcript_id \"chr9.g14585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23042895', '23044340', '.', '-', '.', 'gene_id \"chr9.g14587\"; transcript_id \"chr9.g14587.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23051357', '23051579', '.', '-', '.', 'gene_id \"chr9.g14589\"; transcript_id \"chr9.g14589.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23044475', '23044671', '.', '+', '.', 'gene_id \"chr9.g14591\"; transcript_id \"chr9.g14591.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23045658', '23046081', '.', '-', '.', 'gene_id \"chr9.g14592\"; transcript_id \"chr9.g14592.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23045036', '23045607', '.', '-', '.', 'gene_id \"chr9.g14593\"; transcript_id \"chr9.g14593.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23046411', '23046892', '.', '-', '.', 'gene_id \"chr9.g14595\"; transcript_id \"chr9.g14595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23052492', '23052736', '.', '-', '.', 'gene_id \"chr9.g14600\"; transcript_id \"chr9.g14600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23059410', '23059922', '.', '+', '.', 'gene_id \"chr9.g14607\"; transcript_id \"chr9.g14607.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23062840', '23063441', '.', '-', '.', 'gene_id \"chr9.g14608\"; transcript_id \"chr9.g14608.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23063523', '23063876', '.', '-', '.', 'gene_id \"chr9.g14609\"; transcript_id \"chr9.g14609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23065874', '23066051', '.', '-', '.', 'gene_id \"chr9.g14611\"; transcript_id \"chr9.g14611.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23070335', '23070665', '.', '+', '.', 'gene_id \"chr9.g14620\"; transcript_id \"chr9.g14620.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23071490', '23071651', '.', '-', '.', 'gene_id \"chr9.g14621\"; transcript_id \"chr9.g14621.i36\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23081330', '23082586', '.', '+', '.', 'gene_id \"chr9.g14626\"; transcript_id \"chr9.g14626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23080505', '23080754', '.', '+', '.', 'gene_id \"chr9.g14627\"; transcript_id \"chr9.g14627.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23079355', '23079507', '.', '+', '.', 'gene_id \"chr9.g14628\"; transcript_id \"chr9.g14628.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23077583', '23078058', '.', '+', '.', 'gene_id \"chr9.g14629\"; transcript_id \"chr9.g14629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23072717', '23074765', '.', '+', '.', 'gene_id \"chr9.g14630\"; transcript_id \"chr9.g14630.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23097230', '23099705', '.', '+', '.', 'gene_id \"chr9.g14639\"; transcript_id \"chr9.g14639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23121733', '23121839', '.', '-', '.', 'gene_id \"chr9.g14647\"; transcript_id \"chr9.g14647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23157868', '23158166', '.', '+', '.', 'gene_id \"chr9.g14654\"; transcript_id \"chr9.g14654.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23222075', '23222199', '.', '+', '.', 'gene_id \"chr9.g14659\"; transcript_id \"chr9.g14659.i17\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23221493', '23221976', '.', '-', '.', 'gene_id \"chr9.g14664\"; transcript_id \"chr9.g14664.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23189215', '23189436', '.', '+', '.', 'gene_id \"chr9.g14668\"; transcript_id \"chr9.g14668.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23200069', '23200255', '.', '-', '.', 'gene_id \"chr9.g14669\"; transcript_id \"chr9.g14669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23203073', '23203751', '.', '-', '.', 'gene_id \"chr9.g14673\"; transcript_id \"chr9.g14673.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23204299', '23204598', '.', '-', '.', 'gene_id \"chr9.g14675\"; transcript_id \"chr9.g14675.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23205262', '23205467', '.', '+', '.', 'gene_id \"chr9.g14678\"; transcript_id \"chr9.g14678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23207195', '23207395', '.', '-', '.', 'gene_id \"chr9.g14679\"; transcript_id \"chr9.g14679.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23211990', '23212155', '.', '+', '.', 'gene_id \"chr9.g14681\"; transcript_id \"chr9.g14681.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23212631', '23213221', '.', '-', '.', 'gene_id \"chr9.g14682\"; transcript_id \"chr9.g14682.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23213740', '23214123', '.', '+', '.', 'gene_id \"chr9.g14684\"; transcript_id \"chr9.g14684.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23215542', '23215964', '.', '+', '.', 'gene_id \"chr9.g14686\"; transcript_id \"chr9.g14686.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23220073', '23220255', '.', '+', '.', 'gene_id \"chr9.g14691\"; transcript_id \"chr9.g14691.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23220369', '23220677', '.', '-', '.', 'gene_id \"chr9.g14692\"; transcript_id \"chr9.g14692.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23226318', '23226511', '.', '+', '.', 'gene_id \"chr9.g14693\"; transcript_id \"chr9.g14693.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23226639', '23226929', '.', '+', '.', 'gene_id \"chr9.g14694\"; transcript_id \"chr9.g14694.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23227187', '23227764', '.', '+', '.', 'gene_id \"chr9.g14697\"; transcript_id \"chr9.g14697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23227766', '23228416', '.', '-', '.', 'gene_id \"chr9.g14699\"; transcript_id \"chr9.g14699.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23229288', '23229565', '.', '+', '.', 'gene_id \"chr9.g14700\"; transcript_id \"chr9.g14700.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23261152', '23261331', '.', '-', '.', 'gene_id \"chr9.g14711\"; transcript_id \"chr9.g14711.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23260739', '23261172', '.', '-', '.', 'gene_id \"chr9.g14711\"; transcript_id \"chr9.g14711.i17\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23272395', '23274164', '.', '-', '.', 'gene_id \"chr9.g14711\"; transcript_id \"chr9.g14711.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23244891', '23246254', '.', '+', '.', 'gene_id \"chr9.g14712\"; transcript_id \"chr9.g14712.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23277683', '23278390', '.', '-', '.', 'gene_id \"chr9.g14714\"; transcript_id \"chr9.g14714.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23249801', '23250314', '.', '+', '.', 'gene_id \"chr9.g14715\"; transcript_id \"chr9.g14715.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23244393', '23244906', '.', '+', '.', 'gene_id \"chr9.g14717\"; transcript_id \"chr9.g14717.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23254571', '23254806', '.', '-', '.', 'gene_id \"chr9.g14725\"; transcript_id \"chr9.g14725.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23254815', '23255096', '.', '-', '.', 'gene_id \"chr9.g14726\"; transcript_id \"chr9.g14726.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23257233', '23257565', '.', '-', '.', 'gene_id \"chr9.g14730\"; transcript_id \"chr9.g14730.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '31059899', '31060054', '.', '+', '.', 'gene_id \"chr17.g11072\"; transcript_id \"chr17.g11072.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '31060338', '31060530', '.', '+', '.', 'gene_id \"chr17.g11073\"; transcript_id \"chr17.g11073.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23257799', '23257986', '.', '-', '.', 'gene_id \"chr9.g14735\"; transcript_id \"chr9.g14735.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23259386', '23259931', '.', '-', '.', 'gene_id \"chr9.g14738\"; transcript_id \"chr9.g14738.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23260084', '23260261', '.', '-', '.', 'gene_id \"chr9.g14739\"; transcript_id \"chr9.g14739.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23280116', '23280287', '.', '-', '.', 'gene_id \"chr9.g14749\"; transcript_id \"chr9.g14749.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23285192', '23285576', '.', '-', '.', 'gene_id \"chr9.g14753\"; transcript_id \"chr9.g14753.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23284381', '23284639', '.', '+', '.', 'gene_id \"chr9.g14754\"; transcript_id \"chr9.g14754.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23311827', '23312077', '.', '-', '.', 'gene_id \"chr9.g14770\"; transcript_id \"chr9.g14770.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23314216', '23314360', '.', '+', '.', 'gene_id \"chr9.g14771\"; transcript_id \"chr9.g14771.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23317835', '23317867', '.', '+', '.', 'gene_id \"chr9.g14773\"; transcript_id \"chr9.g14773.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23324902', '23325063', '.', '-', '.', 'gene_id \"chr9.g14774\"; transcript_id \"chr9.g14774.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23333887', '23335479', '.', '-', '.', 'gene_id \"chr9.g14776\"; transcript_id \"chr9.g14776.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23343619', '23343889', '.', '-', '.', 'gene_id \"chr9.g14780\"; transcript_id \"chr9.g14780.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23355840', '23356003', '.', '+', '.', 'gene_id \"chr9.g14783\"; transcript_id \"chr9.g14783.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23350529', '23350921', '.', '-', '.', 'gene_id \"chr9.g14785\"; transcript_id \"chr9.g14785.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23340697', '23340849', '.', '-', '.', 'gene_id \"chr9.g14787\"; transcript_id \"chr9.g14787.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23340857', '23341357', '.', '+', '.', 'gene_id \"chr9.g14788\"; transcript_id \"chr9.g14788.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23344212', '23344388', '.', '-', '.', 'gene_id \"chr9.g14792\"; transcript_id \"chr9.g14792.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23345013', '23345235', '.', '-', '.', 'gene_id \"chr9.g14793\"; transcript_id \"chr9.g14793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23348470', '23348844', '.', '-', '.', 'gene_id \"chr9.g14796\"; transcript_id \"chr9.g14796.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23349349', '23349594', '.', '+', '.', 'gene_id \"chr9.g14799\"; transcript_id \"chr9.g14799.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23356492', '23356694', '.', '+', '.', 'gene_id \"chr9.g14806\"; transcript_id \"chr9.g14806.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23357101', '23357284', '.', '-', '.', 'gene_id \"chr9.g14807\"; transcript_id \"chr9.g14807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23363578', '23363752', '.', '-', '.', 'gene_id \"chr9.g14811\"; transcript_id \"chr9.g14811.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23375169', '23375329', '.', '+', '.', 'gene_id \"chr9.g14815\"; transcript_id \"chr9.g14815.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23396608', '23397066', '.', '-', '.', 'gene_id \"chr9.g14816\"; transcript_id \"chr9.g14816.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23392712', '23392983', '.', '+', '.', 'gene_id \"chr9.g14819\"; transcript_id \"chr9.g14819.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23429141', '23429260', '.', '+', '.', 'gene_id \"chr9.g14828\"; transcript_id \"chr9.g14828.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23433906', '23434075', '.', '+', '.', 'gene_id \"chr9.g14828\"; transcript_id \"chr9.g14828.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23525724', '23525901', '.', '+', '.', 'gene_id \"chr9.g14828\"; transcript_id \"chr9.g14828.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23510570', '23510879', '.', '+', '.', 'gene_id \"chr9.g14829\"; transcript_id \"chr9.g14829.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23433688', '23434075', '.', '-', '.', 'gene_id \"chr9.g14831\"; transcript_id \"chr9.g14831.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23429141', '23429258', '.', '-', '.', 'gene_id \"chr9.g14833\"; transcript_id \"chr9.g14833.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23508650', '23508963', '.', '-', '.', 'gene_id \"chr9.g14835\"; transcript_id \"chr9.g14835.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23508173', '23508587', '.', '+', '.', 'gene_id \"chr9.g14841\"; transcript_id \"chr9.g14841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23530443', '23530631', '.', '-', '.', 'gene_id \"chr9.g14845\"; transcript_id \"chr9.g14845.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23530637', '23531410', '.', '-', '.', 'gene_id \"chr9.g14846\"; transcript_id \"chr9.g14846.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23533376', '23533531', '.', '+', '.', 'gene_id \"chr9.g14847\"; transcript_id \"chr9.g14847.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23442809', '23443129', '.', '-', '.', 'gene_id \"chr9.g14851\"; transcript_id \"chr9.g14851.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23442809', '23443082', '.', '-', '.', 'gene_id \"chr9.g14851\"; transcript_id \"chr9.g14851.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23445537', '23445967', '.', '-', '.', 'gene_id \"chr9.g14853\"; transcript_id \"chr9.g14853.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23447419', '23447538', '.', '-', '.', 'gene_id \"chr9.g14854\"; transcript_id \"chr9.g14854.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23448578', '23448883', '.', '+', '.', 'gene_id \"chr9.g14855\"; transcript_id \"chr9.g14855.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23465253', '23465424', '.', '-', '.', 'gene_id \"chr9.g14863\"; transcript_id \"chr9.g14863.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23474147', '23474377', '.', '-', '.', 'gene_id \"chr9.g14871\"; transcript_id \"chr9.g14871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23474472', '23474741', '.', '-', '.', 'gene_id \"chr9.g14872\"; transcript_id \"chr9.g14872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23482123', '23482360', '.', '-', '.', 'gene_id \"chr9.g14873\"; transcript_id \"chr9.g14873.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23490953', '23491274', '.', '+', '.', 'gene_id \"chr9.g14878\"; transcript_id \"chr9.g14878.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23491910', '23491949', '.', '-', '.', 'gene_id \"chr9.g14879\"; transcript_id \"chr9.g14879.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23492321', '23492480', '.', '+', '.', 'gene_id \"chr9.g14880\"; transcript_id \"chr9.g14880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23493137', '23493307', '.', '+', '.', 'gene_id \"chr9.g14881\"; transcript_id \"chr9.g14881.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23493342', '23493683', '.', '+', '.', 'gene_id \"chr9.g14882\"; transcript_id \"chr9.g14882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23498245', '23498414', '.', '+', '.', 'gene_id \"chr9.g14884\"; transcript_id \"chr9.g14884.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23511024', '23511236', '.', '+', '.', 'gene_id \"chr9.g14888\"; transcript_id \"chr9.g14888.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23511599', '23511634', '.', '-', '.', 'gene_id \"chr9.g14890\"; transcript_id \"chr9.g14890.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23519460', '23519657', '.', '-', '.', 'gene_id \"chr9.g14896\"; transcript_id \"chr9.g14896.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23522044', '23522428', '.', '-', '.', 'gene_id \"chr9.g14900\"; transcript_id \"chr9.g14900.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23525957', '23526129', '.', '-', '.', 'gene_id \"chr9.g14905\"; transcript_id \"chr9.g14905.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23527267', '23527643', '.', '-', '.', 'gene_id \"chr9.g14907\"; transcript_id \"chr9.g14907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23527329', '23527643', '.', '-', '.', 'gene_id \"chr9.g14907\"; transcript_id \"chr9.g14907.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23527917', '23528088', '.', '+', '.', 'gene_id \"chr9.g14909\"; transcript_id \"chr9.g14909.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23527932', '23528088', '.', '+', '.', 'gene_id \"chr9.g14909\"; transcript_id \"chr9.g14909.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23531556', '23531805', '.', '+', '.', 'gene_id \"chr9.g14912\"; transcript_id \"chr9.g14912.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23531795', '23531856', '.', '-', '.', 'gene_id \"chr9.g14913\"; transcript_id \"chr9.g14913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23531947', '23532528', '.', '+', '.', 'gene_id \"chr9.g14914\"; transcript_id \"chr9.g14914.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23532677', '23533234', '.', '-', '.', 'gene_id \"chr9.g14915\"; transcript_id \"chr9.g14915.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23537370', '23537679', '.', '-', '.', 'gene_id \"chr9.g14919\"; transcript_id \"chr9.g14919.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23562009', '23562597', '.', '+', '.', 'gene_id \"chr9.g14924\"; transcript_id \"chr9.g14924.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23562579', '23562733', '.', '+', '.', 'gene_id \"chr9.g14924\"; transcript_id \"chr9.g14924.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23564339', '23564940', '.', '+', '.', 'gene_id \"chr9.g14927\"; transcript_id \"chr9.g14927.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23565180', '23566089', '.', '+', '.', 'gene_id \"chr9.g14929\"; transcript_id \"chr9.g14929.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23588544', '23589254', '.', '+', '.', 'gene_id \"chr9.g14930\"; transcript_id \"chr9.g14930.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23581703', '23582417', '.', '-', '.', 'gene_id \"chr9.g14935\"; transcript_id \"chr9.g14935.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23584074', '23584433', '.', '-', '.', 'gene_id \"chr9.g14938\"; transcript_id \"chr9.g14938.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23583350', '23583968', '.', '-', '.', 'gene_id \"chr9.g14939\"; transcript_id \"chr9.g14939.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23589325', '23589957', '.', '+', '.', 'gene_id \"chr9.g14942\"; transcript_id \"chr9.g14942.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23572553', '23572714', '.', '+', '.', 'gene_id \"chr9.g14945\"; transcript_id \"chr9.g14945.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23567887', '23568056', '.', '-', '.', 'gene_id \"chr9.g14947\"; transcript_id \"chr9.g14947.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23570759', '23571214', '.', '+', '.', 'gene_id \"chr9.g14951\"; transcript_id \"chr9.g14951.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23578490', '23578658', '.', '-', '.', 'gene_id \"chr9.g14952\"; transcript_id \"chr9.g14952.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23584523', '23585082', '.', '+', '.', 'gene_id \"chr9.g14954\"; transcript_id \"chr9.g14954.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23585086', '23585551', '.', '-', '.', 'gene_id \"chr9.g14955\"; transcript_id \"chr9.g14955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23594451', '23594610', '.', '+', '.', 'gene_id \"chr9.g14957\"; transcript_id \"chr9.g14957.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23601691', '23601889', '.', '+', '.', 'gene_id \"chr9.g14960\"; transcript_id \"chr9.g14960.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23601691', '23601889', '.', '+', '.', 'gene_id \"chr9.g14960\"; transcript_id \"chr9.g14960.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23601691', '23601889', '.', '+', '.', 'gene_id \"chr9.g14960\"; transcript_id \"chr9.g14960.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23606766', '23607055', '.', '+', '.', 'gene_id \"chr9.g14964\"; transcript_id \"chr9.g14964.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23607429', '23607772', '.', '-', '.', 'gene_id \"chr9.g14966\"; transcript_id \"chr9.g14966.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23607804', '23607983', '.', '-', '.', 'gene_id \"chr9.g14968\"; transcript_id \"chr9.g14968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23609678', '23609880', '.', '-', '.', 'gene_id \"chr9.g14971\"; transcript_id \"chr9.g14971.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23611005', '23611368', '.', '+', '.', 'gene_id \"chr9.g14972\"; transcript_id \"chr9.g14972.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23611537', '23612156', '.', '-', '.', 'gene_id \"chr9.g14973\"; transcript_id \"chr9.g14973.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23612340', '23612673', '.', '+', '.', 'gene_id \"chr9.g14974\"; transcript_id \"chr9.g14974.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23612913', '23613276', '.', '+', '.', 'gene_id \"chr9.g14975\"; transcript_id \"chr9.g14975.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23613277', '23613444', '.', '-', '.', 'gene_id \"chr9.g14976\"; transcript_id \"chr9.g14976.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23613755', '23614129', '.', '+', '.', 'gene_id \"chr9.g14977\"; transcript_id \"chr9.g14977.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23614162', '23614319', '.', '-', '.', 'gene_id \"chr9.g14978\"; transcript_id \"chr9.g14978.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23635672', '23635831', '.', '-', '.', 'gene_id \"chr9.g14983\"; transcript_id \"chr9.g14983.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23641295', '23641447', '.', '-', '.', 'gene_id \"chr9.g14987\"; transcript_id \"chr9.g14987.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23645627', '23645839', '.', '+', '.', 'gene_id \"chr9.g14990\"; transcript_id \"chr9.g14990.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23653841', '23654078', '.', '-', '.', 'gene_id \"chr9.g14992\"; transcript_id \"chr9.g14992.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23712895', '23713643', '.', '+', '.', 'gene_id \"chr9.g15002\"; transcript_id \"chr9.g15002.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23715920', '23716070', '.', '-', '.', 'gene_id \"chr9.g15003\"; transcript_id \"chr9.g15003.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23706915', '23707108', '.', '-', '.', 'gene_id \"chr9.g15005\"; transcript_id \"chr9.g15005.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23707575', '23707915', '.', '-', '.', 'gene_id \"chr9.g15007\"; transcript_id \"chr9.g15007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23709390', '23709762', '.', '+', '.', 'gene_id \"chr9.g15010\"; transcript_id \"chr9.g15010.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23710291', '23710549', '.', '-', '.', 'gene_id \"chr9.g15011\"; transcript_id \"chr9.g15011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23710577', '23711386', '.', '+', '.', 'gene_id \"chr9.g15012\"; transcript_id \"chr9.g15012.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23714386', '23714551', '.', '-', '.', 'gene_id \"chr9.g15014\"; transcript_id \"chr9.g15014.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23732596', '23732856', '.', '+', '.', 'gene_id \"chr9.g15017\"; transcript_id \"chr9.g15017.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23731770', '23731992', '.', '-', '.', 'gene_id \"chr9.g15018\"; transcript_id \"chr9.g15018.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23731389', '23731705', '.', '-', '.', 'gene_id \"chr9.g15019\"; transcript_id \"chr9.g15019.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23727626', '23727816', '.', '-', '.', 'gene_id \"chr9.g15021\"; transcript_id \"chr9.g15021.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23725506', '23725824', '.', '+', '.', 'gene_id \"chr9.g15023\"; transcript_id \"chr9.g15023.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23720246', '23720858', '.', '-', '.', 'gene_id \"chr9.g15024\"; transcript_id \"chr9.g15024.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23723368', '23723520', '.', '+', '.', 'gene_id \"chr9.g15026\"; transcript_id \"chr9.g15026.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23726822', '23726958', '.', '+', '.', 'gene_id \"chr9.g15028\"; transcript_id \"chr9.g15028.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23729850', '23730035', '.', '+', '.', 'gene_id \"chr9.g15032\"; transcript_id \"chr9.g15032.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23730123', '23730287', '.', '+', '.', 'gene_id \"chr9.g15033\"; transcript_id \"chr9.g15033.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23736999', '23737377', '.', '-', '.', 'gene_id \"chr9.g15034\"; transcript_id \"chr9.g15034.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23735838', '23736457', '.', '-', '.', 'gene_id \"chr9.g15035\"; transcript_id \"chr9.g15035.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23769336', '23769496', '.', '-', '.', 'gene_id \"chr9.g15039\"; transcript_id \"chr9.g15039.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23783222', '23783589', '.', '+', '.', 'gene_id \"chr9.g15041\"; transcript_id \"chr9.g15041.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23783732', '23783974', '.', '+', '.', 'gene_id \"chr9.g15042\"; transcript_id \"chr9.g15042.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23784122', '23784343', '.', '-', '.', 'gene_id \"chr9.g15043\"; transcript_id \"chr9.g15043.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23784302', '23784680', '.', '-', '.', 'gene_id \"chr9.g15043\"; transcript_id \"chr9.g15043.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23785112', '23785180', '.', '-', '.', 'gene_id \"chr9.g15047\"; transcript_id \"chr9.g15047.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23806739', '23806932', '.', '-', '.', 'gene_id \"chr9.g15049\"; transcript_id \"chr9.g15049.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23804576', '23805480', '.', '-', '.', 'gene_id \"chr9.g15050\"; transcript_id \"chr9.g15050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23800332', '23801450', '.', '+', '.', 'gene_id \"chr9.g15052\"; transcript_id \"chr9.g15052.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23788204', '23789260', '.', '+', '.', 'gene_id \"chr9.g15055\"; transcript_id \"chr9.g15055.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23786515', '23786720', '.', '+', '.', 'gene_id \"chr9.g15056\"; transcript_id \"chr9.g15056.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23787088', '23788061', '.', '+', '.', 'gene_id \"chr9.g15058\"; transcript_id \"chr9.g15058.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23789891', '23790123', '.', '-', '.', 'gene_id \"chr9.g15062\"; transcript_id \"chr9.g15062.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23790833', '23791209', '.', '+', '.', 'gene_id \"chr9.g15063\"; transcript_id \"chr9.g15063.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23795540', '23795704', '.', '-', '.', 'gene_id \"chr9.g15072\"; transcript_id \"chr9.g15072.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23799000', '23799227', '.', '-', '.', 'gene_id \"chr9.g15077\"; transcript_id \"chr9.g15077.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23802797', '23803805', '.', '+', '.', 'gene_id \"chr9.g15083\"; transcript_id \"chr9.g15083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23814173', '23814689', '.', '-', '.', 'gene_id \"chr9.g15090\"; transcript_id \"chr9.g15090.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23820771', '23821023', '.', '+', '.', 'gene_id \"chr9.g15092\"; transcript_id \"chr9.g15092.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23820397', '23820749', '.', '+', '.', 'gene_id \"chr9.g15092\"; transcript_id \"chr9.g15092.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23818847', '23819117', '.', '+', '.', 'gene_id \"chr9.g15092\"; transcript_id \"chr9.g15092.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23823738', '23823937', '.', '-', '.', 'gene_id \"chr9.g15093\"; transcript_id \"chr9.g15093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23823726', '23823937', '.', '-', '.', 'gene_id \"chr9.g15093\"; transcript_id \"chr9.g15093.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23817194', '23817476', '.', '+', '.', 'gene_id \"chr9.g15097\"; transcript_id \"chr9.g15097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23817557', '23818057', '.', '-', '.', 'gene_id \"chr9.g15098\"; transcript_id \"chr9.g15098.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23818217', '23818575', '.', '+', '.', 'gene_id \"chr9.g15100\"; transcript_id \"chr9.g15100.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23824039', '23824217', '.', '-', '.', 'gene_id \"chr9.g15101\"; transcript_id \"chr9.g15101.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23824196', '23825066', '.', '-', '.', 'gene_id \"chr9.g15101\"; transcript_id \"chr9.g15101.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23825016', '23825399', '.', '+', '.', 'gene_id \"chr9.g15102\"; transcript_id \"chr9.g15102.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23827124', '23827554', '.', '-', '.', 'gene_id \"chr9.g15103\"; transcript_id \"chr9.g15103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23827124', '23827554', '.', '-', '.', 'gene_id \"chr9.g15103\"; transcript_id \"chr9.g15103.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23827838', '23828471', '.', '+', '.', 'gene_id \"chr9.g15104\"; transcript_id \"chr9.g15104.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23849149', '23849300', '.', '+', '.', 'gene_id \"chr9.g15116\"; transcript_id \"chr9.g15116.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23841750', '23841935', '.', '-', '.', 'gene_id \"chr9.g15117\"; transcript_id \"chr9.g15117.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23843655', '23843817', '.', '+', '.', 'gene_id \"chr9.g15122\"; transcript_id \"chr9.g15122.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23840422', '23840842', '.', '+', '.', 'gene_id \"chr9.g15123\"; transcript_id \"chr9.g15123.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23845483', '23845696', '.', '+', '.', 'gene_id \"chr9.g15124\"; transcript_id \"chr9.g15124.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23840950', '23841291', '.', '+', '.', 'gene_id \"chr9.g15126\"; transcript_id \"chr9.g15126.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23848659', '23848996', '.', '-', '.', 'gene_id \"chr9.g15128\"; transcript_id \"chr9.g15128.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23850003', '23850230', '.', '-', '.', 'gene_id \"chr9.g15129\"; transcript_id \"chr9.g15129.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23850299', '23850634', '.', '+', '.', 'gene_id \"chr9.g15130\"; transcript_id \"chr9.g15130.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23850732', '23851040', '.', '-', '.', 'gene_id \"chr9.g15131\"; transcript_id \"chr9.g15131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23851082', '23851304', '.', '-', '.', 'gene_id \"chr9.g15132\"; transcript_id \"chr9.g15132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23875491', '23875522', '.', '+', '.', 'gene_id \"chr9.g15137\"; transcript_id \"chr9.g15137.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23891645', '23891907', '.', '+', '.', 'gene_id \"chr9.g15149\"; transcript_id \"chr9.g15149.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23922439', '23926037', '.', '-', '.', 'gene_id \"chr9.g15157\"; transcript_id \"chr9.g15157.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23926587', '23926661', '.', '-', '.', 'gene_id \"chr9.g15157\"; transcript_id \"chr9.g15157.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23926587', '23926661', '.', '-', '.', 'gene_id \"chr9.g15157\"; transcript_id \"chr9.g15157.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23930623', '23931064', '.', '-', '.', 'gene_id \"chr9.g15157\"; transcript_id \"chr9.g15157.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23926297', '23926612', '.', '+', '.', 'gene_id \"chr9.g15158\"; transcript_id \"chr9.g15158.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23913753', '23914404', '.', '+', '.', 'gene_id \"chr9.g15160\"; transcript_id \"chr9.g15160.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23913753', '23915622', '.', '+', '.', 'gene_id \"chr9.g15160\"; transcript_id \"chr9.g15160.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23931909', '23932116', '.', '+', '.', 'gene_id \"chr9.g15163\"; transcript_id \"chr9.g15163.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23916299', '23916566', '.', '+', '.', 'gene_id \"chr9.g15164\"; transcript_id \"chr9.g15164.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23931588', '23931883', '.', '+', '.', 'gene_id \"chr9.g15166\"; transcript_id \"chr9.g15166.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23910874', '23913165', '.', '+', '.', 'gene_id \"chr9.g15168\"; transcript_id \"chr9.g15168.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23913265', '23913524', '.', '-', '.', 'gene_id \"chr9.g15169\"; transcript_id \"chr9.g15169.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23913508', '23913745', '.', '+', '.', 'gene_id \"chr9.g15170\"; transcript_id \"chr9.g15170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23932282', '23932449', '.', '+', '.', 'gene_id \"chr9.g15171\"; transcript_id \"chr9.g15171.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23932428', '23932816', '.', '-', '.', 'gene_id \"chr9.g15172\"; transcript_id \"chr9.g15172.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23933638', '23933852', '.', '+', '.', 'gene_id \"chr9.g15173\"; transcript_id \"chr9.g15173.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23933882', '23934052', '.', '+', '.', 'gene_id \"chr9.g15174\"; transcript_id \"chr9.g15174.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23934070', '23934325', '.', '+', '.', 'gene_id \"chr9.g15175\"; transcript_id \"chr9.g15175.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23934349', '23934561', '.', '+', '.', 'gene_id \"chr9.g15176\"; transcript_id \"chr9.g15176.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23934349', '23934552', '.', '+', '.', 'gene_id \"chr9.g15176\"; transcript_id \"chr9.g15176.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23935963', '23936337', '.', '+', '.', 'gene_id \"chr9.g15179\"; transcript_id \"chr9.g15179.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23936480', '23936822', '.', '-', '.', 'gene_id \"chr9.g15181\"; transcript_id \"chr9.g15181.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '31903321', '31903532', '.', '-', '.', 'gene_id \"chr17.g11459\"; transcript_id \"chr17.g11459.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23937132', '23937292', '.', '-', '.', 'gene_id \"chr9.g15184\"; transcript_id \"chr9.g15184.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23942801', '23942999', '.', '+', '.', 'gene_id \"chr9.g15188\"; transcript_id \"chr9.g15188.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23945618', '23946219', '.', '-', '.', 'gene_id \"chr9.g15192\"; transcript_id \"chr9.g15192.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23946226', '23946386', '.', '-', '.', 'gene_id \"chr9.g15193\"; transcript_id \"chr9.g15193.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23958945', '23959508', '.', '+', '.', 'gene_id \"chr9.g15199\"; transcript_id \"chr9.g15199.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23966492', '23966723', '.', '-', '.', 'gene_id \"chr9.g15204\"; transcript_id \"chr9.g15204.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23963318', '23963628', '.', '-', '.', 'gene_id \"chr9.g15206\"; transcript_id \"chr9.g15206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23957140', '23957660', '.', '+', '.', 'gene_id \"chr9.g15208\"; transcript_id \"chr9.g15208.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23957736', '23958382', '.', '+', '.', 'gene_id \"chr9.g15209\"; transcript_id \"chr9.g15209.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23962169', '23962328', '.', '-', '.', 'gene_id \"chr9.g15213\"; transcript_id \"chr9.g15213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23963086', '23963235', '.', '-', '.', 'gene_id \"chr9.g15215\"; transcript_id \"chr9.g15215.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23964492', '23964894', '.', '-', '.', 'gene_id \"chr9.g15217\"; transcript_id \"chr9.g15217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23967729', '23968128', '.', '+', '.', 'gene_id \"chr9.g15219\"; transcript_id \"chr9.g15219.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23968278', '23968346', '.', '+', '.', 'gene_id \"chr9.g15220\"; transcript_id \"chr9.g15220.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23968391', '23968489', '.', '+', '.', 'gene_id \"chr9.g15220\"; transcript_id \"chr9.g15220.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23968659', '23969126', '.', '-', '.', 'gene_id \"chr9.g15222\"; transcript_id \"chr9.g15222.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23970523', '23971071', '.', '+', '.', 'gene_id \"chr9.g15224\"; transcript_id \"chr9.g15224.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23976751', '23977350', '.', '+', '.', 'gene_id \"chr9.g15227\"; transcript_id \"chr9.g15227.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24263199', '24263548', '.', '+', '.', 'gene_id \"chr9.g15228\"; transcript_id \"chr9.g15228.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24229862', '24229969', '.', '+', '.', 'gene_id \"chr9.g15229\"; transcript_id \"chr9.g15229.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23989558', '23989774', '.', '+', '.', 'gene_id \"chr9.g15240\"; transcript_id \"chr9.g15240.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24008572', '24008752', '.', '+', '.', 'gene_id \"chr9.g15249\"; transcript_id \"chr9.g15249.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24013615', '24013789', '.', '+', '.', 'gene_id \"chr9.g15256\"; transcript_id \"chr9.g15256.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24019520', '24019757', '.', '+', '.', 'gene_id \"chr9.g15259\"; transcript_id \"chr9.g15259.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '24019844', '24020215', '.', '-', '.', 'gene_id \"chr9.g15260\"; transcript_id \"chr9.g15260.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24058105', '24058148', '.', '+', '.', 'gene_id \"chr9.g15266\"; transcript_id \"chr9.g15266.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24082960', '24083218', '.', '+', '.', 'gene_id \"chr9.g15271\"; transcript_id \"chr9.g15271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24122131', '24122679', '.', '+', '.', 'gene_id \"chr9.g15273\"; transcript_id \"chr9.g15273.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24128856', '24129008', '.', '+', '.', 'gene_id \"chr9.g15275\"; transcript_id \"chr9.g15275.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24134658', '24134843', '.', '+', '.', 'gene_id \"chr9.g15277\"; transcript_id \"chr9.g15277.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24139343', '24139511', '.', '-', '.', 'gene_id \"chr9.g15282\"; transcript_id \"chr9.g15282.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24320403', '24320862', '.', '+', '.', 'gene_id \"chr9.g15316\"; transcript_id \"chr9.g15316.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '32081934', '32082315', '.', '-', '.', 'gene_id \"chr17.g11585\"; transcript_id \"chr17.g11585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24402698', '24403447', '.', '-', '.', 'gene_id \"chr9.g15325\"; transcript_id \"chr9.g15325.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24374622', '24375348', '.', '-', '.', 'gene_id \"chr9.g15326\"; transcript_id \"chr9.g15326.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24374622', '24375348', '.', '-', '.', 'gene_id \"chr9.g15326\"; transcript_id \"chr9.g15326.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24401699', '24401731', '.', '+', '.', 'gene_id \"chr9.g15329\"; transcript_id \"chr9.g15329.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24375813', '24376005', '.', '-', '.', 'gene_id \"chr9.g15338\"; transcript_id \"chr9.g15338.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24379764', '24379946', '.', '+', '.', 'gene_id \"chr9.g15339\"; transcript_id \"chr9.g15339.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24381090', '24381563', '.', '+', '.', 'gene_id \"chr9.g15340\"; transcript_id \"chr9.g15340.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24380595', '24380827', '.', '+', '.', 'gene_id \"chr9.g15341\"; transcript_id \"chr9.g15341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24382122', '24382556', '.', '-', '.', 'gene_id \"chr9.g15344\"; transcript_id \"chr9.g15344.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24385897', '24386102', '.', '-', '.', 'gene_id \"chr9.g15347\"; transcript_id \"chr9.g15347.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24386233', '24386576', '.', '-', '.', 'gene_id \"chr9.g15348\"; transcript_id \"chr9.g15348.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24387419', '24387735', '.', '-', '.', 'gene_id \"chr9.g15350\"; transcript_id \"chr9.g15350.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24394768', '24394920', '.', '-', '.', 'gene_id \"chr9.g15352\"; transcript_id \"chr9.g15352.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24395122', '24395345', '.', '-', '.', 'gene_id \"chr9.g15353\"; transcript_id \"chr9.g15353.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24397633', '24397835', '.', '+', '.', 'gene_id \"chr9.g15354\"; transcript_id \"chr9.g15354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24415449', '24415640', '.', '+', '.', 'gene_id \"chr9.g15366\"; transcript_id \"chr9.g15366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24416324', '24416795', '.', '-', '.', 'gene_id \"chr9.g15367\"; transcript_id \"chr9.g15367.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24416802', '24417484', '.', '-', '.', 'gene_id \"chr9.g15367\"; transcript_id \"chr9.g15367.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24419078', '24419339', '.', '+', '.', 'gene_id \"chr9.g15370\"; transcript_id \"chr9.g15370.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24428782', '24429066', '.', '+', '.', 'gene_id \"chr9.g15373\"; transcript_id \"chr9.g15373.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24419378', '24421002', '.', '-', '.', 'gene_id \"chr9.g15375\"; transcript_id \"chr9.g15375.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24433329', '24433400', '.', '+', '.', 'gene_id \"chr9.g15376\"; transcript_id \"chr9.g15376.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24432601', '24433281', '.', '+', '.', 'gene_id \"chr9.g15378\"; transcript_id \"chr9.g15378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24437474', '24437848', '.', '+', '.', 'gene_id \"chr9.g15381\"; transcript_id \"chr9.g15381.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24477112', '24477145', '.', '-', '.', 'gene_id \"chr9.g15387\"; transcript_id \"chr9.g15387.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24553801', '24554212', '.', '+', '.', 'gene_id \"chr9.g15399\"; transcript_id \"chr9.g15399.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24566618', '24566771', '.', '-', '.', 'gene_id \"chr9.g15410\"; transcript_id \"chr9.g15410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24570283', '24570375', '.', '-', '.', 'gene_id \"chr9.g15411\"; transcript_id \"chr9.g15411.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24576521', '24576677', '.', '+', '.', 'gene_id \"chr9.g15412\"; transcript_id \"chr9.g15412.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24578565', '24578793', '.', '-', '.', 'gene_id \"chr9.g15419\"; transcript_id \"chr9.g15419.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24595074', '24595341', '.', '+', '.', 'gene_id \"chr9.g15422\"; transcript_id \"chr9.g15422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24596679', '24596860', '.', '+', '.', 'gene_id \"chr9.g15423\"; transcript_id \"chr9.g15423.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24602878', '24603071', '.', '+', '.', 'gene_id \"chr9.g15427\"; transcript_id \"chr9.g15427.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24639374', '24639942', '.', '-', '.', 'gene_id \"chr9.g15431\"; transcript_id \"chr9.g15431.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24613551', '24613913', '.', '-', '.', 'gene_id \"chr9.g15433\"; transcript_id \"chr9.g15433.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24635737', '24635881', '.', '+', '.', 'gene_id \"chr9.g15435\"; transcript_id \"chr9.g15435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24632861', '24633155', '.', '-', '.', 'gene_id \"chr9.g15436\"; transcript_id \"chr9.g15436.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24632433', '24632766', '.', '+', '.', 'gene_id \"chr9.g15438\"; transcript_id \"chr9.g15438.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24626983', '24627166', '.', '+', '.', 'gene_id \"chr9.g15440\"; transcript_id \"chr9.g15440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24626710', '24626917', '.', '-', '.', 'gene_id \"chr9.g15441\"; transcript_id \"chr9.g15441.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24616883', '24617177', '.', '+', '.', 'gene_id \"chr9.g15447\"; transcript_id \"chr9.g15447.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24617665', '24617772', '.', '-', '.', 'gene_id \"chr9.g15448\"; transcript_id \"chr9.g15448.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24617753', '24617901', '.', '-', '.', 'gene_id \"chr9.g15448\"; transcript_id \"chr9.g15448.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24617800', '24617901', '.', '-', '.', 'gene_id \"chr9.g15448\"; transcript_id \"chr9.g15448.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24617998', '24618192', '.', '-', '.', 'gene_id \"chr9.g15451\"; transcript_id \"chr9.g15451.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24623040', '24623193', '.', '+', '.', 'gene_id \"chr9.g15456\"; transcript_id \"chr9.g15456.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24622882', '24623049', '.', '-', '.', 'gene_id \"chr9.g15457\"; transcript_id \"chr9.g15457.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24637787', '24637951', '.', '-', '.', 'gene_id \"chr9.g15468\"; transcript_id \"chr9.g15468.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24643622', '24643957', '.', '+', '.', 'gene_id \"chr9.g15469\"; transcript_id \"chr9.g15469.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24692898', '24693154', '.', '-', '.', 'gene_id \"chr9.g15494\"; transcript_id \"chr9.g15494.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24721069', '24721461', '.', '+', '.', 'gene_id \"chr9.g15501\"; transcript_id \"chr9.g15501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24721069', '24721320', '.', '+', '.', 'gene_id \"chr9.g15501\"; transcript_id \"chr9.g15501.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24717800', '24718431', '.', '+', '.', 'gene_id \"chr9.g15503\"; transcript_id \"chr9.g15503.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24720603', '24720844', '.', '+', '.', 'gene_id \"chr9.g15508\"; transcript_id \"chr9.g15508.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24718461', '24719122', '.', '+', '.', 'gene_id \"chr9.g15510\"; transcript_id \"chr9.g15510.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24724603', '24724851', '.', '+', '.', 'gene_id \"chr9.g15511\"; transcript_id \"chr9.g15511.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24706435', '24707596', '.', '+', '.', 'gene_id \"chr9.g15512\"; transcript_id \"chr9.g15512.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24705638', '24706452', '.', '+', '.', 'gene_id \"chr9.g15514\"; transcript_id \"chr9.g15514.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24705113', '24705660', '.', '+', '.', 'gene_id \"chr9.g15515\"; transcript_id \"chr9.g15515.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24703706', '24705044', '.', '+', '.', 'gene_id \"chr9.g15516\"; transcript_id \"chr9.g15516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24702620', '24703660', '.', '+', '.', 'gene_id \"chr9.g15518\"; transcript_id \"chr9.g15518.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24701757', '24702442', '.', '+', '.', 'gene_id \"chr9.g15523\"; transcript_id \"chr9.g15523.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24709918', '24710182', '.', '+', '.', 'gene_id \"chr9.g15527\"; transcript_id \"chr9.g15527.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24709107', '24709663', '.', '-', '.', 'gene_id \"chr9.g15529\"; transcript_id \"chr9.g15529.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '32191015', '32191230', '.', '+', '.', 'gene_id \"chr17.g11738\"; transcript_id \"chr17.g11738.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24710456', '24710686', '.', '+', '.', 'gene_id \"chr9.g15531\"; transcript_id \"chr9.g15531.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24719625', '24719975', '.', '-', '.', 'gene_id \"chr9.g15532\"; transcript_id \"chr9.g15532.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '32207924', '32208085', '.', '-', '.', 'gene_id \"chr17.g11748\"; transcript_id \"chr17.g11748.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24874811', '24874993', '.', '+', '.', 'gene_id \"chr9.g15555\"; transcript_id \"chr9.g15555.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '32251175', '32251213', '.', '+', '.', 'gene_id \"chr17.g11767\"; transcript_id \"chr17.g11767.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24918466', '24918643', '.', '+', '.', 'gene_id \"chr9.g15572\"; transcript_id \"chr9.g15572.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24923833', '24924014', '.', '+', '.', 'gene_id \"chr9.g15577\"; transcript_id \"chr9.g15577.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24938826', '24939402', '.', '+', '.', 'gene_id \"chr9.g15584\"; transcript_id \"chr9.g15584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24939938', '24940400', '.', '-', '.', 'gene_id \"chr9.g15590\"; transcript_id \"chr9.g15590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24942689', '24942853', '.', '-', '.', 'gene_id \"chr9.g15592\"; transcript_id \"chr9.g15592.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24940790', '24941840', '.', '-', '.', 'gene_id \"chr9.g15593\"; transcript_id \"chr9.g15593.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24946582', '24947049', '.', '+', '.', 'gene_id \"chr9.g15594\"; transcript_id \"chr9.g15594.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24948536', '24948678', '.', '-', '.', 'gene_id \"chr9.g15596\"; transcript_id \"chr9.g15596.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24952446', '24952637', '.', '+', '.', 'gene_id \"chr9.g15600\"; transcript_id \"chr9.g15600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24954467', '24955166', '.', '+', '.', 'gene_id \"chr9.g15603\"; transcript_id \"chr9.g15603.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24952207', '24952338', '.', '+', '.', 'gene_id \"chr9.g15604\"; transcript_id \"chr9.g15604.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24954130', '24954441', '.', '-', '.', 'gene_id \"chr9.g15605\"; transcript_id \"chr9.g15605.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24957948', '24958296', '.', '+', '.', 'gene_id \"chr9.g15609\"; transcript_id \"chr9.g15609.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24981963', '24982138', '.', '-', '.', 'gene_id \"chr9.g15613\"; transcript_id \"chr9.g15613.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24990922', '24995317', '.', '+', '.', 'gene_id \"chr9.g15616\"; transcript_id \"chr9.g15616.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25012444', '25012600', '.', '+', '.', 'gene_id \"chr9.g15617\"; transcript_id \"chr9.g15617.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25000014', '25000165', '.', '-', '.', 'gene_id \"chr9.g15618\"; transcript_id \"chr9.g15618.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25000873', '25001440', '.', '+', '.', 'gene_id \"chr9.g15621\"; transcript_id \"chr9.g15621.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25001581', '25001760', '.', '-', '.', 'gene_id \"chr9.g15623\"; transcript_id \"chr9.g15623.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25002708', '25003005', '.', '-', '.', 'gene_id \"chr9.g15625\"; transcript_id \"chr9.g15625.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25004886', '25005040', '.', '+', '.', 'gene_id \"chr9.g15626\"; transcript_id \"chr9.g15626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25018924', '25019124', '.', '-', '.', 'gene_id \"chr9.g15638\"; transcript_id \"chr9.g15638.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25019260', '25019530', '.', '-', '.', 'gene_id \"chr9.g15639\"; transcript_id \"chr9.g15639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25027568', '25027703', '.', '-', '.', 'gene_id \"chr9.g15643\"; transcript_id \"chr9.g15643.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25027705', '25027825', '.', '-', '.', 'gene_id \"chr9.g15644\"; transcript_id \"chr9.g15644.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25030626', '25031037', '.', '+', '.', 'gene_id \"chr9.g15647\"; transcript_id \"chr9.g15647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25051243', '25051418', '.', '-', '.', 'gene_id \"chr9.g15659\"; transcript_id \"chr9.g15659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25056891', '25057009', '.', '+', '.', 'gene_id \"chr9.g15660\"; transcript_id \"chr9.g15660.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25090956', '25092319', '.', '-', '.', 'gene_id \"chr9.g15668\"; transcript_id \"chr9.g15668.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25092372', '25094653', '.', '-', '.', 'gene_id \"chr9.g15668\"; transcript_id \"chr9.g15668.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25097686', '25097900', '.', '+', '.', 'gene_id \"chr9.g15669\"; transcript_id \"chr9.g15669.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25096716', '25096812', '.', '+', '.', 'gene_id \"chr9.g15672\"; transcript_id \"chr9.g15672.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25094730', '25094912', '.', '-', '.', 'gene_id \"chr9.g15674\"; transcript_id \"chr9.g15674.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25098527', '25098785', '.', '+', '.', 'gene_id \"chr9.g15676\"; transcript_id \"chr9.g15676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25183491', '25184252', '.', '-', '.', 'gene_id \"chr9.g15679\"; transcript_id \"chr9.g15679.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25183256', '25183501', '.', '-', '.', 'gene_id \"chr9.g15680\"; transcript_id \"chr9.g15680.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25165266', '25165435', '.', '-', '.', 'gene_id \"chr9.g15681\"; transcript_id \"chr9.g15681.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25185831', '25186310', '.', '+', '.', 'gene_id \"chr9.g15687\"; transcript_id \"chr9.g15687.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25246253', '25246478', '.', '+', '.', 'gene_id \"chr9.g15699\"; transcript_id \"chr9.g15699.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25248287', '25249384', '.', '-', '.', 'gene_id \"chr9.g15700\"; transcript_id \"chr9.g15700.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25282065', '25282117', '.', '-', '.', 'gene_id \"chr9.g15700\"; transcript_id \"chr9.g15700.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25255404', '25255588', '.', '+', '.', 'gene_id \"chr9.g15705\"; transcript_id \"chr9.g15705.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25253961', '25254177', '.', '+', '.', 'gene_id \"chr9.g15706\"; transcript_id \"chr9.g15706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25258576', '25258758', '.', '-', '.', 'gene_id \"chr9.g15710\"; transcript_id \"chr9.g15710.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25251353', '25253693', '.', '-', '.', 'gene_id \"chr9.g15711\"; transcript_id \"chr9.g15711.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25254182', '25254551', '.', '+', '.', 'gene_id \"chr9.g15714\"; transcript_id \"chr9.g15714.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '33455290', '33455566', '.', '+', '.', 'gene_id \"chr17.g11909\"; transcript_id \"chr17.g11909.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25280019', '25280194', '.', '-', '.', 'gene_id \"chr9.g15731\"; transcript_id \"chr9.g15731.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25286591', '25286756', '.', '+', '.', 'gene_id \"chr9.g15738\"; transcript_id \"chr9.g15738.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25297118', '25297271', '.', '-', '.', 'gene_id \"chr9.g15747\"; transcript_id \"chr9.g15747.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25298883', '25299049', '.', '+', '.', 'gene_id \"chr9.g15749\"; transcript_id \"chr9.g15749.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25299949', '25300188', '.', '+', '.', 'gene_id \"chr9.g15751\"; transcript_id \"chr9.g15751.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25301595', '25301818', '.', '+', '.', 'gene_id \"chr9.g15754\"; transcript_id \"chr9.g15754.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25309162', '25309339', '.', '+', '.', 'gene_id \"chr9.g15762\"; transcript_id \"chr9.g15762.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25328797', '25329074', '.', '+', '.', 'gene_id \"chr9.g15768\"; transcript_id \"chr9.g15768.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25329741', '25330461', '.', '+', '.', 'gene_id \"chr9.g15772\"; transcript_id \"chr9.g15772.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25329187', '25329757', '.', '+', '.', 'gene_id \"chr9.g15773\"; transcript_id \"chr9.g15773.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25337035', '25337407', '.', '-', '.', 'gene_id \"chr9.g15775\"; transcript_id \"chr9.g15775.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25336633', '25336941', '.', '+', '.', 'gene_id \"chr9.g15776\"; transcript_id \"chr9.g15776.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25333945', '25334186', '.', '+', '.', 'gene_id \"chr9.g15777\"; transcript_id \"chr9.g15777.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25333683', '25333874', '.', '-', '.', 'gene_id \"chr9.g15778\"; transcript_id \"chr9.g15778.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25343282', '25345325', '.', '+', '.', 'gene_id \"chr9.g15783\"; transcript_id \"chr9.g15783.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25349927', '25350915', '.', '-', '.', 'gene_id \"chr9.g15784\"; transcript_id \"chr9.g15784.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25350967', '25353991', '.', '+', '.', 'gene_id \"chr9.g15786\"; transcript_id \"chr9.g15786.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25359255', '25359765', '.', '+', '.', 'gene_id \"chr9.g15788\"; transcript_id \"chr9.g15788.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25359930', '25360120', '.', '-', '.', 'gene_id \"chr9.g15789\"; transcript_id \"chr9.g15789.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25349268', '25349959', '.', '+', '.', 'gene_id \"chr9.g15793\"; transcript_id \"chr9.g15793.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25358389', '25359129', '.', '+', '.', 'gene_id \"chr9.g15794\"; transcript_id \"chr9.g15794.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25345307', '25345711', '.', '-', '.', 'gene_id \"chr9.g15795\"; transcript_id \"chr9.g15795.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25346563', '25347095', '.', '-', '.', 'gene_id \"chr9.g15798\"; transcript_id \"chr9.g15798.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25369147', '25369730', '.', '+', '.', 'gene_id \"chr9.g15808\"; transcript_id \"chr9.g15808.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25378357', '25378647', '.', '+', '.', 'gene_id \"chr9.g15823\"; transcript_id \"chr9.g15823.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25378671', '25379422', '.', '-', '.', 'gene_id \"chr9.g15825\"; transcript_id \"chr9.g15825.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25380165', '25380357', '.', '-', '.', 'gene_id \"chr9.g15827\"; transcript_id \"chr9.g15827.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25388652', '25388969', '.', '-', '.', 'gene_id \"chr9.g15838\"; transcript_id \"chr9.g15838.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25390244', '25390457', '.', '-', '.', 'gene_id \"chr9.g15840\"; transcript_id \"chr9.g15840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25421224', '25421389', '.', '+', '.', 'gene_id \"chr9.g15855\"; transcript_id \"chr9.g15855.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25433594', '25434205', '.', '-', '.', 'gene_id \"chr9.g15859\"; transcript_id \"chr9.g15859.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25413364', '25413514', '.', '-', '.', 'gene_id \"chr9.g15867\"; transcript_id \"chr9.g15867.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25420138', '25420662', '.', '-', '.', 'gene_id \"chr9.g15871\"; transcript_id \"chr9.g15871.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25425342', '25425499', '.', '-', '.', 'gene_id \"chr9.g15872\"; transcript_id \"chr9.g15872.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25427533', '25428533', '.', '+', '.', 'gene_id \"chr9.g15876\"; transcript_id \"chr9.g15876.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25432187', '25432683', '.', '+', '.', 'gene_id \"chr9.g15882\"; transcript_id \"chr9.g15882.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25437952', '25438112', '.', '-', '.', 'gene_id \"chr9.g15883\"; transcript_id \"chr9.g15883.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25448048', '25448334', '.', '+', '.', 'gene_id \"chr9.g15897\"; transcript_id \"chr9.g15897.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25450730', '25451113', '.', '-', '.', 'gene_id \"chr9.g15898\"; transcript_id \"chr9.g15898.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25580031', '25580438', '.', '-', '.', 'gene_id \"chr9.g15905\"; transcript_id \"chr9.g15905.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25583840', '25584247', '.', '+', '.', 'gene_id \"chr9.g15908\"; transcript_id \"chr9.g15908.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25582031', '25582432', '.', '+', '.', 'gene_id \"chr9.g15910\"; transcript_id \"chr9.g15910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25582559', '25582776', '.', '+', '.', 'gene_id \"chr9.g15911\"; transcript_id \"chr9.g15911.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25583050', '25583473', '.', '-', '.', 'gene_id \"chr9.g15913\"; transcript_id \"chr9.g15913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25640087', '25640470', '.', '+', '.', 'gene_id \"chr9.g15927\"; transcript_id \"chr9.g15927.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25649575', '25650182', '.', '+', '.', 'gene_id \"chr9.g15927\"; transcript_id \"chr9.g15927.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25649310', '25649497', '.', '+', '.', 'gene_id \"chr9.g15927\"; transcript_id \"chr9.g15927.i11\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25647409', '25647577', '.', '+', '.', 'gene_id \"chr9.g15928\"; transcript_id \"chr9.g15928.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25647409', '25647532', '.', '+', '.', 'gene_id \"chr9.g15928\"; transcript_id \"chr9.g15928.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25661196', '25661359', '.', '-', '.', 'gene_id \"chr9.g15929\"; transcript_id \"chr9.g15929.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25651332', '25651978', '.', '+', '.', 'gene_id \"chr9.g15930\"; transcript_id \"chr9.g15930.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25642854', '25643259', '.', '+', '.', 'gene_id \"chr9.g15931\"; transcript_id \"chr9.g15931.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25648686', '25648885', '.', '+', '.', 'gene_id \"chr9.g15933\"; transcript_id \"chr9.g15933.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25639455', '25639711', '.', '-', '.', 'gene_id \"chr9.g15935\"; transcript_id \"chr9.g15935.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25650260', '25651277', '.', '+', '.', 'gene_id \"chr9.g15939\"; transcript_id \"chr9.g15939.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25646248', '25646566', '.', '+', '.', 'gene_id \"chr9.g15942\"; transcript_id \"chr9.g15942.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25638844', '25639191', '.', '+', '.', 'gene_id \"chr9.g15949\"; transcript_id \"chr9.g15949.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25628653', '25628870', '.', '+', '.', 'gene_id \"chr9.g15950\"; transcript_id \"chr9.g15950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25629358', '25629514', '.', '+', '.', 'gene_id \"chr9.g15951\"; transcript_id \"chr9.g15951.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25630070', '25630310', '.', '+', '.', 'gene_id \"chr9.g15952\"; transcript_id \"chr9.g15952.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25630373', '25631056', '.', '+', '.', 'gene_id \"chr9.g15953\"; transcript_id \"chr9.g15953.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25631997', '25632150', '.', '-', '.', 'gene_id \"chr9.g15955\"; transcript_id \"chr9.g15955.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25632457', '25632847', '.', '+', '.', 'gene_id \"chr9.g15956\"; transcript_id \"chr9.g15956.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25634734', '25635003', '.', '+', '.', 'gene_id \"chr9.g15960\"; transcript_id \"chr9.g15960.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25634451', '25634755', '.', '-', '.', 'gene_id \"chr9.g15961\"; transcript_id \"chr9.g15961.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25637488', '25637693', '.', '-', '.', 'gene_id \"chr9.g15964\"; transcript_id \"chr9.g15964.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25642538', '25642816', '.', '+', '.', 'gene_id \"chr9.g15967\"; transcript_id \"chr9.g15967.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25659957', '25660146', '.', '-', '.', 'gene_id \"chr9.g15971\"; transcript_id \"chr9.g15971.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25673010', '25673241', '.', '+', '.', 'gene_id \"chr9.g15977\"; transcript_id \"chr9.g15977.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25676372', '25676684', '.', '-', '.', 'gene_id \"chr9.g15980\"; transcript_id \"chr9.g15980.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25676753', '25677279', '.', '-', '.', 'gene_id \"chr9.g15982\"; transcript_id \"chr9.g15982.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25663796', '25664144', '.', '+', '.', 'gene_id \"chr9.g15984\"; transcript_id \"chr9.g15984.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25667432', '25667644', '.', '+', '.', 'gene_id \"chr9.g15989\"; transcript_id \"chr9.g15989.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25667752', '25668408', '.', '-', '.', 'gene_id \"chr9.g15990\"; transcript_id \"chr9.g15990.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25668480', '25669247', '.', '+', '.', 'gene_id \"chr9.g15991\"; transcript_id \"chr9.g15991.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25669255', '25669994', '.', '-', '.', 'gene_id \"chr9.g15992\"; transcript_id \"chr9.g15992.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25670929', '25671141', '.', '+', '.', 'gene_id \"chr9.g15994\"; transcript_id \"chr9.g15994.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25682534', '25682701', '.', '+', '.', 'gene_id \"chr9.g15999\"; transcript_id \"chr9.g15999.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25694394', '25694575', '.', '+', '.', 'gene_id \"chr9.g16000\"; transcript_id \"chr9.g16000.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25695379', '25695704', '.', '+', '.', 'gene_id \"chr9.g16002\"; transcript_id \"chr9.g16002.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25703228', '25703384', '.', '-', '.', 'gene_id \"chr9.g16004\"; transcript_id \"chr9.g16004.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25700931', '25703217', '.', '-', '.', 'gene_id \"chr9.g16004\"; transcript_id \"chr9.g16004.i13\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25705429', '25707325', '.', '-', '.', 'gene_id \"chr9.g16004\"; transcript_id \"chr9.g16004.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25707032', '25707325', '.', '-', '.', 'gene_id \"chr9.g16004\"; transcript_id \"chr9.g16004.i15\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25697072', '25697285', '.', '-', '.', 'gene_id \"chr9.g16010\"; transcript_id \"chr9.g16010.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25697564', '25697965', '.', '-', '.', 'gene_id \"chr9.g16011\"; transcript_id \"chr9.g16011.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25697948', '25698194', '.', '+', '.', 'gene_id \"chr9.g16012\"; transcript_id \"chr9.g16012.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25704273', '25704641', '.', '-', '.', 'gene_id \"chr9.g16017\"; transcript_id \"chr9.g16017.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25715010', '25715163', '.', '+', '.', 'gene_id \"chr9.g16023\"; transcript_id \"chr9.g16023.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25730157', '25730677', '.', '+', '.', 'gene_id \"chr9.g16027\"; transcript_id \"chr9.g16027.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25731368', '25731521', '.', '+', '.', 'gene_id \"chr9.g16027\"; transcript_id \"chr9.g16027.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25733888', '25734083', '.', '+', '.', 'gene_id \"chr9.g16027\"; transcript_id \"chr9.g16027.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25741465', '25741665', '.', '-', '.', 'gene_id \"chr9.g16029\"; transcript_id \"chr9.g16029.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25731972', '25733110', '.', '-', '.', 'gene_id \"chr9.g16030\"; transcript_id \"chr9.g16030.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25733134', '25733292', '.', '+', '.', 'gene_id \"chr9.g16033\"; transcript_id \"chr9.g16033.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25748601', '25748758', '.', '-', '.', 'gene_id \"chr9.g16034\"; transcript_id \"chr9.g16034.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25753437', '25753657', '.', '-', '.', 'gene_id \"chr9.g16034\"; transcript_id \"chr9.g16034.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25744930', '25746093', '.', '-', '.', 'gene_id \"chr9.g16034\"; transcript_id \"chr9.g16034.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25760442', '25760814', '.', '+', '.', 'gene_id \"chr9.g16039\"; transcript_id \"chr9.g16039.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25751566', '25751837', '.', '+', '.', 'gene_id \"chr9.g16041\"; transcript_id \"chr9.g16041.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25750755', '25751025', '.', '+', '.', 'gene_id \"chr9.g16042\"; transcript_id \"chr9.g16042.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25749654', '25749858', '.', '+', '.', 'gene_id \"chr9.g16043\"; transcript_id \"chr9.g16043.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25748768', '25749209', '.', '+', '.', 'gene_id \"chr9.g16044\"; transcript_id \"chr9.g16044.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25747871', '25748205', '.', '+', '.', 'gene_id \"chr9.g16045\"; transcript_id \"chr9.g16045.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25833889', '25834207', '.', '-', '.', 'gene_id \"chr9.g16065\"; transcript_id \"chr9.g16065.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25873200', '25873250', '.', '-', '.', 'gene_id \"chr9.g16069\"; transcript_id \"chr9.g16069.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25889105', '25889431', '.', '-', '.', 'gene_id \"chr9.g16074\"; transcript_id \"chr9.g16074.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25879275', '25879493', '.', '+', '.', 'gene_id \"chr9.g16076\"; transcript_id \"chr9.g16076.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25880240', '25880519', '.', '+', '.', 'gene_id \"chr9.g16080\"; transcript_id \"chr9.g16080.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25881585', '25882059', '.', '-', '.', 'gene_id \"chr9.g16081\"; transcript_id \"chr9.g16081.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25881432', '25881595', '.', '-', '.', 'gene_id \"chr9.g16083\"; transcript_id \"chr9.g16083.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25884227', '25884545', '.', '+', '.', 'gene_id \"chr9.g16085\"; transcript_id \"chr9.g16085.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25884733', '25884959', '.', '-', '.', 'gene_id \"chr9.g16086\"; transcript_id \"chr9.g16086.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25890121', '25890454', '.', '+', '.', 'gene_id \"chr9.g16088\"; transcript_id \"chr9.g16088.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25901294', '25901941', '.', '-', '.', 'gene_id \"chr9.g16090\"; transcript_id \"chr9.g16090.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25900976', '25901308', '.', '+', '.', 'gene_id \"chr9.g16092\"; transcript_id \"chr9.g16092.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25899754', '25900665', '.', '+', '.', 'gene_id \"chr9.g16093\"; transcript_id \"chr9.g16093.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25900681', '25900902', '.', '+', '.', 'gene_id \"chr9.g16095\"; transcript_id \"chr9.g16095.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25904777', '25904928', '.', '-', '.', 'gene_id \"chr9.g16096\"; transcript_id \"chr9.g16096.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25922325', '25922620', '.', '-', '.', 'gene_id \"chr9.g16103\"; transcript_id \"chr9.g16103.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25916248', '25916703', '.', '-', '.', 'gene_id \"chr9.g16105\"; transcript_id \"chr9.g16105.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25922690', '25922866', '.', '+', '.', 'gene_id \"chr9.g16107\"; transcript_id \"chr9.g16107.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25917218', '25917336', '.', '+', '.', 'gene_id \"chr9.g16109\"; transcript_id \"chr9.g16109.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25918013', '25918358', '.', '+', '.', 'gene_id \"chr9.g16111\"; transcript_id \"chr9.g16111.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '34169253', '34169503', '.', '-', '.', 'gene_id \"chr17.g12216\"; transcript_id \"chr17.g12216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25974948', '25975401', '.', '+', '.', 'gene_id \"chr9.g16124\"; transcript_id \"chr9.g16124.i16\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25968632', '25968823', '.', '+', '.', 'gene_id \"chr9.g16124\"; transcript_id \"chr9.g16124.i20\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25959247', '25959569', '.', '-', '.', 'gene_id \"chr9.g16125\"; transcript_id \"chr9.g16125.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25958883', '25959195', '.', '-', '.', 'gene_id \"chr9.g16125\"; transcript_id \"chr9.g16125.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25958610', '25958768', '.', '-', '.', 'gene_id \"chr9.g16125\"; transcript_id \"chr9.g16125.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25940085', '25940402', '.', '-', '.', 'gene_id \"chr9.g16126\"; transcript_id \"chr9.g16126.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25966018', '25967325', '.', '-', '.', 'gene_id \"chr9.g16130\"; transcript_id \"chr9.g16130.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25965107', '25965953', '.', '+', '.', 'gene_id \"chr9.g16131\"; transcript_id \"chr9.g16131.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25968905', '25969176', '.', '+', '.', 'gene_id \"chr9.g16132\"; transcript_id \"chr9.g16132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25957946', '25958558', '.', '+', '.', 'gene_id \"chr9.g16134\"; transcript_id \"chr9.g16134.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25974814', '25974987', '.', '-', '.', 'gene_id \"chr9.g16136\"; transcript_id \"chr9.g16136.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25942675', '25942844', '.', '+', '.', 'gene_id \"chr9.g16137\"; transcript_id \"chr9.g16137.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25944946', '25945197', '.', '+', '.', 'gene_id \"chr9.g16140\"; transcript_id \"chr9.g16140.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25945332', '25945559', '.', '-', '.', 'gene_id \"chr9.g16141\"; transcript_id \"chr9.g16141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25964307', '25964817', '.', '+', '.', 'gene_id \"chr9.g16149\"; transcript_id \"chr9.g16149.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25980886', '25981069', '.', '-', '.', 'gene_id \"chr9.g16155\"; transcript_id \"chr9.g16155.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25981161', '25984707', '.', '-', '.', 'gene_id \"chr9.g16157\"; transcript_id \"chr9.g16157.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25987089', '25987306', '.', '+', '.', 'gene_id \"chr9.g16158\"; transcript_id \"chr9.g16158.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26006502', '26007416', '.', '+', '.', 'gene_id \"chr9.g16161\"; transcript_id \"chr9.g16161.i18\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25994379', '25994584', '.', '+', '.', 'gene_id \"chr9.g16161\"; transcript_id \"chr9.g16161.i21\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26008390', '26008569', '.', '+', '.', 'gene_id \"chr9.g16168\"; transcript_id \"chr9.g16168.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26013171', '26013337', '.', '+', '.', 'gene_id \"chr9.g16170\"; transcript_id \"chr9.g16170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26009400', '26009529', '.', '-', '.', 'gene_id \"chr9.g16179\"; transcript_id \"chr9.g16179.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26027409', '26027593', '.', '-', '.', 'gene_id \"chr9.g16187\"; transcript_id \"chr9.g16187.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26027652', '26027893', '.', '+', '.', 'gene_id \"chr9.g16188\"; transcript_id \"chr9.g16188.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26028229', '26028465', '.', '-', '.', 'gene_id \"chr9.g16189\"; transcript_id \"chr9.g16189.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26028817', '26029114', '.', '-', '.', 'gene_id \"chr9.g16191\"; transcript_id \"chr9.g16191.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26036198', '26036354', '.', '+', '.', 'gene_id \"chr9.g16195\"; transcript_id \"chr9.g16195.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26034536', '26034773', '.', '-', '.', 'gene_id \"chr9.g16196\"; transcript_id \"chr9.g16196.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26033977', '26034241', '.', '-', '.', 'gene_id \"chr9.g16198\"; transcript_id \"chr9.g16198.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26032358', '26032803', '.', '-', '.', 'gene_id \"chr9.g16199\"; transcript_id \"chr9.g16199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26031540', '26031712', '.', '-', '.', 'gene_id \"chr9.g16200\"; transcript_id \"chr9.g16200.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26040986', '26041169', '.', '-', '.', 'gene_id \"chr9.g16206\"; transcript_id \"chr9.g16206.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26100170', '26100769', '.', '+', '.', 'gene_id \"chr9.g16216\"; transcript_id \"chr9.g16216.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26090177', '26090486', '.', '-', '.', 'gene_id \"chr9.g16217\"; transcript_id \"chr9.g16217.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26099788', '26100190', '.', '-', '.', 'gene_id \"chr9.g16219\"; transcript_id \"chr9.g16219.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26088669', '26088904', '.', '+', '.', 'gene_id \"chr9.g16223\"; transcript_id \"chr9.g16223.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26089135', '26089677', '.', '+', '.', 'gene_id \"chr9.g16224\"; transcript_id \"chr9.g16224.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26096842', '26096978', '.', '+', '.', 'gene_id \"chr9.g16228\"; transcript_id \"chr9.g16228.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26104124', '26104333', '.', '+', '.', 'gene_id \"chr9.g16230\"; transcript_id \"chr9.g16230.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26104578', '26104729', '.', '-', '.', 'gene_id \"chr9.g16231\"; transcript_id \"chr9.g16231.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26113721', '26113846', '.', '+', '.', 'gene_id \"chr9.g16232\"; transcript_id \"chr9.g16232.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26151858', '26152070', '.', '-', '.', 'gene_id \"chr9.g16235\"; transcript_id \"chr9.g16235.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26150489', '26150651', '.', '+', '.', 'gene_id \"chr9.g16236\"; transcript_id \"chr9.g16236.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26142485', '26142685', '.', '+', '.', 'gene_id \"chr9.g16240\"; transcript_id \"chr9.g16240.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26156908', '26157123', '.', '+', '.', 'gene_id \"chr9.g16241\"; transcript_id \"chr9.g16241.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26157113', '26157271', '.', '+', '.', 'gene_id \"chr9.g16243\"; transcript_id \"chr9.g16243.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26128506', '26128656', '.', '+', '.', 'gene_id \"chr9.g16250\"; transcript_id \"chr9.g16250.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26131537', '26131898', '.', '-', '.', 'gene_id \"chr9.g16253\"; transcript_id \"chr9.g16253.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26131989', '26132254', '.', '+', '.', 'gene_id \"chr9.g16255\"; transcript_id \"chr9.g16255.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26145493', '26145647', '.', '+', '.', 'gene_id \"chr9.g16258\"; transcript_id \"chr9.g16258.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26146253', '26146589', '.', '+', '.', 'gene_id \"chr9.g16259\"; transcript_id \"chr9.g16259.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26146895', '26147050', '.', '-', '.', 'gene_id \"chr9.g16262\"; transcript_id \"chr9.g16262.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26149902', '26150071', '.', '-', '.', 'gene_id \"chr9.g16265\"; transcript_id \"chr9.g16265.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26152135', '26152688', '.', '-', '.', 'gene_id \"chr9.g16267\"; transcript_id \"chr9.g16267.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26152843', '26153072', '.', '+', '.', 'gene_id \"chr9.g16271\"; transcript_id \"chr9.g16271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26159119', '26159287', '.', '-', '.', 'gene_id \"chr9.g16277\"; transcript_id \"chr9.g16277.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26165097', '26165528', '.', '+', '.', 'gene_id \"chr9.g16281\"; transcript_id \"chr9.g16281.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26164987', '26165111', '.', '-', '.', 'gene_id \"chr9.g16282\"; transcript_id \"chr9.g16282.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26173967', '26174152', '.', '+', '.', 'gene_id \"chr9.g16284\"; transcript_id \"chr9.g16284.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26193910', '26196204', '.', '-', '.', 'gene_id \"chr9.g16291\"; transcript_id \"chr9.g16291.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26214751', '26215087', '.', '+', '.', 'gene_id \"chr9.g16292\"; transcript_id \"chr9.g16292.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26201421', '26201803', '.', '+', '.', 'gene_id \"chr9.g16296\"; transcript_id \"chr9.g16296.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26185979', '26186233', '.', '-', '.', 'gene_id \"chr9.g16305\"; transcript_id \"chr9.g16305.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26188174', '26188624', '.', '-', '.', 'gene_id \"chr9.g16308\"; transcript_id \"chr9.g16308.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26189892', '26190128', '.', '-', '.', 'gene_id \"chr9.g16312\"; transcript_id \"chr9.g16312.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26191738', '26192024', '.', '-', '.', 'gene_id \"chr9.g16316\"; transcript_id \"chr9.g16316.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26192243', '26192598', '.', '+', '.', 'gene_id \"chr9.g16318\"; transcript_id \"chr9.g16318.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26200873', '26201045', '.', '+', '.', 'gene_id \"chr9.g16324\"; transcript_id \"chr9.g16324.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26220851', '26221036', '.', '-', '.', 'gene_id \"chr9.g16331\"; transcript_id \"chr9.g16331.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26255201', '26255390', '.', '-', '.', 'gene_id \"chr9.g16340\"; transcript_id \"chr9.g16340.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26255815', '26255964', '.', '+', '.', 'gene_id \"chr9.g16341\"; transcript_id \"chr9.g16341.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26266802', '26267325', '.', '-', '.', 'gene_id \"chr9.g16350\"; transcript_id \"chr9.g16350.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26293366', '26293582', '.', '-', '.', 'gene_id \"chr9.g16362\"; transcript_id \"chr9.g16362.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26295118', '26295278', '.', '-', '.', 'gene_id \"chr9.g16364\"; transcript_id \"chr9.g16364.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26295903', '26296177', '.', '-', '.', 'gene_id \"chr9.g16366\"; transcript_id \"chr9.g16366.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26296187', '26296363', '.', '-', '.', 'gene_id \"chr9.g16368\"; transcript_id \"chr9.g16368.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26296538', '26300414', '.', '-', '.', 'gene_id \"chr9.g16369\"; transcript_id \"chr9.g16369.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26296538', '26300328', '.', '-', '.', 'gene_id \"chr9.g16369\"; transcript_id \"chr9.g16369.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26296934', '26300414', '.', '-', '.', 'gene_id \"chr9.g16369\"; transcript_id \"chr9.g16369.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26296934', '26300328', '.', '-', '.', 'gene_id \"chr9.g16369\"; transcript_id \"chr9.g16369.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26297407', '26297597', '.', '-', '.', 'gene_id \"chr9.g16369\"; transcript_id \"chr9.g16369.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26311046', '26311217', '.', '+', '.', 'gene_id \"chr9.g16374\"; transcript_id \"chr9.g16374.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26311801', '26312056', '.', '+', '.', 'gene_id \"chr9.g16378\"; transcript_id \"chr9.g16378.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26316537', '26316601', '.', '+', '.', 'gene_id \"chr9.g16384\"; transcript_id \"chr9.g16384.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26326686', '26326972', '.', '+', '.', 'gene_id \"chr9.g16388\"; transcript_id \"chr9.g16388.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26332113', '26332517', '.', '-', '.', 'gene_id \"chr9.g16394\"; transcript_id \"chr9.g16394.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26336025', '26336196', '.', '+', '.', 'gene_id \"chr9.g16395\"; transcript_id \"chr9.g16395.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26337829', '26338073', '.', '-', '.', 'gene_id \"chr9.g16396\"; transcript_id \"chr9.g16396.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26343114', '26343170', '.', '-', '.', 'gene_id \"chr9.g16399\"; transcript_id \"chr9.g16399.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26364224', '26364438', '.', '+', '.', 'gene_id \"chr9.g16409\"; transcript_id \"chr9.g16409.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26366899', '26367058', '.', '+', '.', 'gene_id \"chr9.g16410\"; transcript_id \"chr9.g16410.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26393521', '26393572', '.', '-', '.', 'gene_id \"chr9.g16421\"; transcript_id \"chr9.g16421.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26489307', '26489427', '.', '-', '.', 'gene_id \"chr9.g16421\"; transcript_id \"chr9.g16421.i12\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26394783', '26395125', '.', '-', '.', 'gene_id \"chr9.g16422\"; transcript_id \"chr9.g16422.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26394792', '26395125', '.', '-', '.', 'gene_id \"chr9.g16422\"; transcript_id \"chr9.g16422.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26394800', '26395125', '.', '-', '.', 'gene_id \"chr9.g16422\"; transcript_id \"chr9.g16422.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26413109', '26413616', '.', '+', '.', 'gene_id \"chr9.g16425\"; transcript_id \"chr9.g16425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26421793', '26422222', '.', '+', '.', 'gene_id \"chr9.g16426\"; transcript_id \"chr9.g16426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26404540', '26404729', '.', '+', '.', 'gene_id \"chr9.g16429\"; transcript_id \"chr9.g16429.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26404890', '26405205', '.', '+', '.', 'gene_id \"chr9.g16430\"; transcript_id \"chr9.g16430.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26412408', '26412720', '.', '+', '.', 'gene_id \"chr9.g16432\"; transcript_id \"chr9.g16432.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26398830', '26398983', '.', '+', '.', 'gene_id \"chr9.g16434\"; transcript_id \"chr9.g16434.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26380606', '26380748', '.', '-', '.', 'gene_id \"chr9.g16435\"; transcript_id \"chr9.g16435.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26385620', '26386128', '.', '+', '.', 'gene_id \"chr9.g16438\"; transcript_id \"chr9.g16438.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26414731', '26414879', '.', '+', '.', 'gene_id \"chr9.g16459\"; transcript_id \"chr9.g16459.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26423292', '26423720', '.', '-', '.', 'gene_id \"chr9.g16465\"; transcript_id \"chr9.g16465.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26432315', '26432576', '.', '+', '.', 'gene_id \"chr9.g16478\"; transcript_id \"chr9.g16478.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26448425', '26448573', '.', '+', '.', 'gene_id \"chr9.g16483\"; transcript_id \"chr9.g16483.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26451243', '26451312', '.', '+', '.', 'gene_id \"chr9.g16490\"; transcript_id \"chr9.g16490.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26490474', '26490641', '.', '-', '.', 'gene_id \"chr9.g16499\"; transcript_id \"chr9.g16499.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26519627', '26519822', '.', '+', '.', 'gene_id \"chr9.g16510\"; transcript_id \"chr9.g16510.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26532931', '26533154', '.', '+', '.', 'gene_id \"chr9.g16522\"; transcript_id \"chr9.g16522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26536123', '26536254', '.', '+', '.', 'gene_id \"chr9.g16523\"; transcript_id \"chr9.g16523.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26538999', '26539350', '.', '+', '.', 'gene_id \"chr9.g16524\"; transcript_id \"chr9.g16524.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26539050', '26539350', '.', '+', '.', 'gene_id \"chr9.g16524\"; transcript_id \"chr9.g16524.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26555242', '26555939', '.', '+', '.', 'gene_id \"chr9.g16530\"; transcript_id \"chr9.g16530.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26565048', '26565266', '.', '+', '.', 'gene_id \"chr9.g16532\"; transcript_id \"chr9.g16532.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26563691', '26563936', '.', '-', '.', 'gene_id \"chr9.g16533\"; transcript_id \"chr9.g16533.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26563704', '26564325', '.', '-', '.', 'gene_id \"chr9.g16533\"; transcript_id \"chr9.g16533.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26552390', '26552538', '.', '-', '.', 'gene_id \"chr9.g16534\"; transcript_id \"chr9.g16534.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26556411', '26556892', '.', '-', '.', 'gene_id \"chr9.g16537\"; transcript_id \"chr9.g16537.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26563512', '26563665', '.', '+', '.', 'gene_id \"chr9.g16539\"; transcript_id \"chr9.g16539.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26551322', '26551463', '.', '+', '.', 'gene_id \"chr9.g16544\"; transcript_id \"chr9.g16544.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26550474', '26550630', '.', '-', '.', 'gene_id \"chr9.g16545\"; transcript_id \"chr9.g16545.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26558318', '26558597', '.', '+', '.', 'gene_id \"chr9.g16547\"; transcript_id \"chr9.g16547.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26551949', '26552372', '.', '+', '.', 'gene_id \"chr9.g16550\"; transcript_id \"chr9.g16550.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26553484', '26553881', '.', '+', '.', 'gene_id \"chr9.g16551\"; transcript_id \"chr9.g16551.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26553859', '26553932', '.', '+', '.', 'gene_id \"chr9.g16551\"; transcript_id \"chr9.g16551.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26554310', '26554530', '.', '-', '.', 'gene_id \"chr9.g16553\"; transcript_id \"chr9.g16553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26554747', '26554925', '.', '+', '.', 'gene_id \"chr9.g16555\"; transcript_id \"chr9.g16555.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26554933', '26555126', '.', '-', '.', 'gene_id \"chr9.g16556\"; transcript_id \"chr9.g16556.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26559823', '26560046', '.', '+', '.', 'gene_id \"chr9.g16557\"; transcript_id \"chr9.g16557.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26560051', '26560734', '.', '+', '.', 'gene_id \"chr9.g16558\"; transcript_id \"chr9.g16558.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26574178', '26574551', '.', '+', '.', 'gene_id \"chr9.g16560\"; transcript_id \"chr9.g16560.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26607657', '26607819', '.', '-', '.', 'gene_id \"chr9.g16562\"; transcript_id \"chr9.g16562.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26585940', '26586607', '.', '+', '.', 'gene_id \"chr9.g16567\"; transcript_id \"chr9.g16567.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26583412', '26583625', '.', '-', '.', 'gene_id \"chr9.g16568\"; transcript_id \"chr9.g16568.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26583227', '26583434', '.', '+', '.', 'gene_id \"chr9.g16569\"; transcript_id \"chr9.g16569.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26592389', '26592566', '.', '+', '.', 'gene_id \"chr9.g16572\"; transcript_id \"chr9.g16572.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26576136', '26576466', '.', '+', '.', 'gene_id \"chr9.g16575\"; transcript_id \"chr9.g16575.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26577069', '26577299', '.', '+', '.', 'gene_id \"chr9.g16578\"; transcript_id \"chr9.g16578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26582806', '26582977', '.', '+', '.', 'gene_id \"chr9.g16585\"; transcript_id \"chr9.g16585.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26587780', '26588515', '.', '-', '.', 'gene_id \"chr9.g16586\"; transcript_id \"chr9.g16586.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26589064', '26589446', '.', '+', '.', 'gene_id \"chr9.g16587\"; transcript_id \"chr9.g16587.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26589064', '26589195', '.', '+', '.', 'gene_id \"chr9.g16587\"; transcript_id \"chr9.g16587.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26588662', '26589022', '.', '+', '.', 'gene_id \"chr9.g16588\"; transcript_id \"chr9.g16588.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26589479', '26590106', '.', '-', '.', 'gene_id \"chr9.g16590\"; transcript_id \"chr9.g16590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26594231', '26594407', '.', '+', '.', 'gene_id \"chr9.g16594\"; transcript_id \"chr9.g16594.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26595019', '26595311', '.', '+', '.', 'gene_id \"chr9.g16595\"; transcript_id \"chr9.g16595.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26599469', '26599634', '.', '+', '.', 'gene_id \"chr9.g16600\"; transcript_id \"chr9.g16600.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26602205', '26602611', '.', '+', '.', 'gene_id \"chr9.g16604\"; transcript_id \"chr9.g16604.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26606783', '26607024', '.', '-', '.', 'gene_id \"chr9.g16610\"; transcript_id \"chr9.g16610.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26607935', '26608088', '.', '-', '.', 'gene_id \"chr9.g16611\"; transcript_id \"chr9.g16611.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26608396', '26608707', '.', '+', '.', 'gene_id \"chr9.g16612\"; transcript_id \"chr9.g16612.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26608991', '26609352', '.', '-', '.', 'gene_id \"chr9.g16613\"; transcript_id \"chr9.g16613.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26614779', '26615105', '.', '+', '.', 'gene_id \"chr9.g16622\"; transcript_id \"chr9.g16622.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26617768', '26617944', '.', '+', '.', 'gene_id \"chr9.g16623\"; transcript_id \"chr9.g16623.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26621013', '26621232', '.', '-', '.', 'gene_id \"chr9.g16626\"; transcript_id \"chr9.g16626.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26622155', '26622349', '.', '-', '.', 'gene_id \"chr9.g16629\"; transcript_id \"chr9.g16629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26624432', '26624582', '.', '-', '.', 'gene_id \"chr9.g16633\"; transcript_id \"chr9.g16633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26624597', '26625349', '.', '+', '.', 'gene_id \"chr9.g16634\"; transcript_id \"chr9.g16634.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26625391', '26625589', '.', '-', '.', 'gene_id \"chr9.g16636\"; transcript_id \"chr9.g16636.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26626535', '26627420', '.', '-', '.', 'gene_id \"chr9.g16639\"; transcript_id \"chr9.g16639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26627698', '26628638', '.', '-', '.', 'gene_id \"chr9.g16641\"; transcript_id \"chr9.g16641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26631437', '26631652', '.', '-', '.', 'gene_id \"chr9.g16645\"; transcript_id \"chr9.g16645.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26634601', '26634807', '.', '+', '.', 'gene_id \"chr9.g16651\"; transcript_id \"chr9.g16651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26640344', '26640566', '.', '+', '.', 'gene_id \"chr9.g16659\"; transcript_id \"chr9.g16659.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26673355', '26675748', '.', '-', '.', 'gene_id \"chr9.g16663\"; transcript_id \"chr9.g16663.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26678231', '26678578', '.', '-', '.', 'gene_id \"chr9.g16666\"; transcript_id \"chr9.g16666.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26672872', '26673221', '.', '-', '.', 'gene_id \"chr9.g16667\"; transcript_id \"chr9.g16667.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '35024269', '35024581', '.', '-', '.', 'gene_id \"chr17.g12762\"; transcript_id \"chr17.g12762.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26665785', '26666343', '.', '-', '.', 'gene_id \"chr9.g16683\"; transcript_id \"chr9.g16683.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '35037895', '35038143', '.', '-', '.', 'gene_id \"chr17.g12772\"; transcript_id \"chr17.g12772.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26668450', '26669247', '.', '-', '.', 'gene_id \"chr9.g16688\"; transcript_id \"chr9.g16688.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26681728', '26681989', '.', '-', '.', 'gene_id \"chr9.g16698\"; transcript_id \"chr9.g16698.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26684117', '26684405', '.', '-', '.', 'gene_id \"chr9.g16699\"; transcript_id \"chr9.g16699.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26698003', '26698665', '.', '-', '.', 'gene_id \"chr9.g16700\"; transcript_id \"chr9.g16700.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26699137', '26699299', '.', '+', '.', 'gene_id \"chr9.g16701\"; transcript_id \"chr9.g16701.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26789977', '26790065', '.', '-', '.', 'gene_id \"chr9.g16714\"; transcript_id \"chr9.g16714.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27415092', '27416071', '.', '+', '.', 'gene_id \"chr9.g16732\"; transcript_id \"chr9.g16732.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27217036', '27217180', '.', '+', '.', 'gene_id \"chr9.g16733\"; transcript_id \"chr9.g16733.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27335308', '27335382', '.', '+', '.', 'gene_id \"chr9.g16735\"; transcript_id \"chr9.g16735.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27080073', '27080251', '.', '+', '.', 'gene_id \"chr9.g16763\"; transcript_id \"chr9.g16763.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27247044', '27247323', '.', '-', '.', 'gene_id \"chr9.g16777\"; transcript_id \"chr9.g16777.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27247337', '27247517', '.', '+', '.', 'gene_id \"chr9.g16778\"; transcript_id \"chr9.g16778.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27285757', '27285963', '.', '-', '.', 'gene_id \"chr9.g16780\"; transcript_id \"chr9.g16780.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27317332', '27317531', '.', '-', '.', 'gene_id \"chr9.g16786\"; transcript_id \"chr9.g16786.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27330183', '27330363', '.', '-', '.', 'gene_id \"chr9.g16787\"; transcript_id \"chr9.g16787.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27383895', '27384234', '.', '-', '.', 'gene_id \"chr9.g16790\"; transcript_id \"chr9.g16790.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27417496', '27417975', '.', '+', '.', 'gene_id \"chr9.g16794\"; transcript_id \"chr9.g16794.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '35340442', '35340657', '.', '-', '.', 'gene_id \"chr17.g12893\"; transcript_id \"chr17.g12893.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27605499', '27605539', '.', '+', '.', 'gene_id \"chr9.g16817\"; transcript_id \"chr9.g16817.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27660167', '27660323', '.', '+', '.', 'gene_id \"chr9.g16821\"; transcript_id \"chr9.g16821.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27660167', '27660447', '.', '+', '.', 'gene_id \"chr9.g16821\"; transcript_id \"chr9.g16821.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27724999', '27725083', '.', '+', '.', 'gene_id \"chr9.g16827\"; transcript_id \"chr9.g16827.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27726101', '27726268', '.', '-', '.', 'gene_id \"chr9.g16829\"; transcript_id \"chr9.g16829.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27786066', '27786278', '.', '-', '.', 'gene_id \"chr9.g16840\"; transcript_id \"chr9.g16840.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27803158', '27803313', '.', '+', '.', 'gene_id \"chr9.g16841\"; transcript_id \"chr9.g16841.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27818078', '27818247', '.', '-', '.', 'gene_id \"chr9.g16844\"; transcript_id \"chr9.g16844.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27919755', '27920002', '.', '+', '.', 'gene_id \"chr9.g16848\"; transcript_id \"chr9.g16848.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27919758', '27920002', '.', '+', '.', 'gene_id \"chr9.g16848\"; transcript_id \"chr9.g16848.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '28168743', '28168978', '.', '-', '.', 'gene_id \"chr9.g16881\"; transcript_id \"chr9.g16881.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '28533129', '28533180', '.', '-', '.', 'gene_id \"chr9.g16915\"; transcript_id \"chr9.g16915.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '28941928', '28941957', '.', '-', '.', 'gene_id \"chr9.g16960\"; transcript_id \"chr9.g16960.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29128954', '29128995', '.', '+', '.', 'gene_id \"chr9.g16973\"; transcript_id \"chr9.g16973.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29548824', '29548860', '.', '+', '.', 'gene_id \"chr9.g17003\"; transcript_id \"chr9.g17003.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29694506', '29695069', '.', '+', '.', 'gene_id \"chr9.g17087\"; transcript_id \"chr9.g17087.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29744339', '29744583', '.', '-', '.', 'gene_id \"chr9.g17123\"; transcript_id \"chr9.g17123.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29906816', '29907192', '.', '-', '.', 'gene_id \"chr9.g17221\"; transcript_id \"chr9.g17221.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '36378327', '36378501', '.', '+', '.', 'gene_id \"chr17.g13329\"; transcript_id \"chr17.g13329.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '36381897', '36382372', '.', '-', '.', 'gene_id \"chr17.g13333\"; transcript_id \"chr17.g13333.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '36382372', '36382761', '.', '+', '.', 'gene_id \"chr17.g13334\"; transcript_id \"chr17.g13334.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29932793', '29933088', '.', '+', '.', 'gene_id \"chr9.g17266\"; transcript_id \"chr9.g17266.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '30436826', '30437106', '.', '-', '.', 'gene_id \"chr9.g17492\"; transcript_id \"chr9.g17492.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '30466386', '30466575', '.', '+', '.', 'gene_id \"chr9.g17515\"; transcript_id \"chr9.g17515.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '36999105', '36999383', '.', '-', '.', 'gene_id \"chr17.g13648\"; transcript_id \"chr17.g13648.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31673487', '31673553', '.', '+', '.', 'gene_id \"chr9.g17935\"; transcript_id \"chr9.g17935.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31782164', '31782388', '.', '-', '.', 'gene_id \"chr9.g18014\"; transcript_id \"chr9.g18014.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31782750', '31782962', '.', '+', '.', 'gene_id \"chr9.g18015\"; transcript_id \"chr9.g18015.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31783172', '31783343', '.', '-', '.', 'gene_id \"chr9.g18016\"; transcript_id \"chr9.g18016.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '38236045', '38236129', '.', '-', '.', 'gene_id \"chr17.g14133\"; transcript_id \"chr17.g14133.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '38223264', '38223497', '.', '-', '.', 'gene_id \"chr17.g14170\"; transcript_id \"chr17.g14170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31927123', '31927379', '.', '-', '.', 'gene_id \"chr9.g18113\"; transcript_id \"chr9.g18113.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '31955254', '31955455', '.', '-', '.', 'gene_id \"chr9.g18141\"; transcript_id \"chr9.g18141.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32064085', '32064138', '.', '+', '.', 'gene_id \"chr9.g18199\"; transcript_id \"chr9.g18199.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32201191', '32201277', '.', '+', '.', 'gene_id \"chr9.g18259\"; transcript_id \"chr9.g18259.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32201789', '32201822', '.', '+', '.', 'gene_id \"chr9.g18260\"; transcript_id \"chr9.g18260.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32238473', '32238510', '.', '-', '.', 'gene_id \"chr9.g18288\"; transcript_id \"chr9.g18288.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32333876', '32333917', '.', '-', '.', 'gene_id \"chr9.g18349\"; transcript_id \"chr9.g18349.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '38572263', '38572455', '.', '-', '.', 'gene_id \"chr17.g14461\"; transcript_id \"chr17.g14461.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32536870', '32537159', '.', '-', '.', 'gene_id \"chr9.g18430\"; transcript_id \"chr9.g18430.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '38830900', '38831100', '.', '-', '.', 'gene_id \"chr17.g14618\"; transcript_id \"chr17.g14618.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32783551', '32783593', '.', '-', '.', 'gene_id \"chr9.g18578\"; transcript_id \"chr9.g18578.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32809139', '32809172', '.', '+', '.', 'gene_id \"chr9.g18584\"; transcript_id \"chr9.g18584.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32898574', '32898613', '.', '-', '.', 'gene_id \"chr9.g18621\"; transcript_id \"chr9.g18621.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32912068', '32912320', '.', '+', '.', 'gene_id \"chr9.g18632\"; transcript_id \"chr9.g18632.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32924443', '32924670', '.', '+', '.', 'gene_id \"chr9.g18651\"; transcript_id \"chr9.g18651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32949516', '32949545', '.', '-', '.', 'gene_id \"chr9.g18686\"; transcript_id \"chr9.g18686.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32951751', '32951940', '.', '-', '.', 'gene_id \"chr9.g18687\"; transcript_id \"chr9.g18687.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '33311986', '33312220', '.', '-', '.', 'gene_id \"chr9.g18842\"; transcript_id \"chr9.g18842.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '39473484', '39473534', '.', '-', '.', 'gene_id \"chr17.g15089\"; transcript_id \"chr17.g15089.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '33687875', '33688129', '.', '+', '.', 'gene_id \"chr9.g19069\"; transcript_id \"chr9.g19069.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34013382', '34013430', '.', '-', '.', 'gene_id \"chr9.g19337\"; transcript_id \"chr9.g19337.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34064910', '34065070', '.', '-', '.', 'gene_id \"chr9.g19394\"; transcript_id \"chr9.g19394.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34079617', '34079646', '.', '+', '.', 'gene_id \"chr9.g19400\"; transcript_id \"chr9.g19400.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34080581', '34080687', '.', '+', '.', 'gene_id \"chr9.g19403\"; transcript_id \"chr9.g19403.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '39911627', '39911682', '.', '+', '.', 'gene_id \"chr17.g15524\"; transcript_id \"chr17.g15524.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '40054851', '40055458', '.', '+', '.', 'gene_id \"chr17.g15586\"; transcript_id \"chr17.g15586.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34303081', '34303165', '.', '+', '.', 'gene_id \"chr9.g19535\"; transcript_id \"chr9.g19535.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34515419', '34515519', '.', '+', '.', 'gene_id \"chr9.g19605\"; transcript_id \"chr9.g19605.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34663958', '34663997', '.', '+', '.', 'gene_id \"chr9.g19635\"; transcript_id \"chr9.g19635.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34760347', '34760717', '.', '-', '.', 'gene_id \"chr9.g19653\"; transcript_id \"chr9.g19653.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '42461245', '42461280', '.', '-', '.', 'gene_id \"chr17.g15842\"; transcript_id \"chr17.g15842.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34899920', '34900186', '.', '+', '.', 'gene_id \"chr9.g19761\"; transcript_id \"chr9.g19761.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34953743', '34954086', '.', '-', '.', 'gene_id \"chr9.g19823\"; transcript_id \"chr9.g19823.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34985208', '34985363', '.', '+', '.', 'gene_id \"chr9.g19867\"; transcript_id \"chr9.g19867.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '43883440', '43883667', '.', '-', '.', 'gene_id \"chr17.g16017\"; transcript_id \"chr17.g16017.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35095679', '35095721', '.', '-', '.', 'gene_id \"chr9.g19993\"; transcript_id \"chr9.g19993.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35230073', '35230148', '.', '-', '.', 'gene_id \"chr9.g20065\"; transcript_id \"chr9.g20065.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35230073', '35230147', '.', '-', '.', 'gene_id \"chr9.g20065\"; transcript_id \"chr9.g20065.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35414655', '35414764', '.', '-', '.', 'gene_id \"chr9.g20113\"; transcript_id \"chr9.g20113.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35468308', '35468526', '.', '-', '.', 'gene_id \"chr9.g20140\"; transcript_id \"chr9.g20140.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35505981', '35506205', '.', '+', '.', 'gene_id \"chr9.g20165\"; transcript_id \"chr9.g20165.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35507698', '35507911', '.', '+', '.', 'gene_id \"chr9.g20170\"; transcript_id \"chr9.g20170.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35510736', '35510951', '.', '-', '.', 'gene_id \"chr9.g20175\"; transcript_id \"chr9.g20175.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '47378114', '47378227', '.', '-', '.', 'gene_id \"chr17.g16391\"; transcript_id \"chr17.g16391.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr17', 'PASA', 'exon', '47510533', '47510699', '.', '+', '.', 'gene_id \"chr17.g16423\"; transcript_id \"chr17.g16423.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35863597', '35863831', '.', '+', '.', 'gene_id \"chr9.g20447\"; transcript_id \"chr9.g20447.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35875719', '35875912', '.', '+', '.', 'gene_id \"chr9.g20459\"; transcript_id \"chr9.g20459.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36073137', '36073170', '.', '-', '.', 'gene_id \"chr9.g20614\"; transcript_id \"chr9.g20614.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36081523', '36081553', '.', '-', '.', 'gene_id \"chr9.g20622\"; transcript_id \"chr9.g20622.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36337964', '36338153', '.', '+', '.', 'gene_id \"chr9.g20666\"; transcript_id \"chr9.g20666.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36374394', '36374606', '.', '+', '.', 'gene_id \"chr9.g20689\"; transcript_id \"chr9.g20689.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36532739', '36532899', '.', '+', '.', 'gene_id \"chr9.g20849\"; transcript_id \"chr9.g20849.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '48558510', '48558544', '.', '+', '.', 'gene_id \"chr17.g16900\"; transcript_id \"chr17.g16900.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36623573', '36624094', '.', '+', '.', 'gene_id \"chr9.g20918\"; transcript_id \"chr9.g20918.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36625195', '36625661', '.', '-', '.', 'gene_id \"chr9.g20924\"; transcript_id \"chr9.g20924.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36728875', '36728947', '.', '-', '.', 'gene_id \"chr9.g21003\"; transcript_id \"chr9.g21003.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37008985', '37009105', '.', '+', '.', 'gene_id \"chr9.g21271\"; transcript_id \"chr9.g21271.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37008847', '37009105', '.', '+', '.', 'gene_id \"chr9.g21271\"; transcript_id \"chr9.g21271.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37008985', '37009050', '.', '+', '.', 'gene_id \"chr9.g21271\"; transcript_id \"chr9.g21271.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37433614', '37433646', '.', '+', '.', 'gene_id \"chr9.g21522\"; transcript_id \"chr9.g21522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37551669', '37552636', '.', '+', '.', 'gene_id \"chr9.g21639\"; transcript_id \"chr9.g21639.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37552637', '37553100', '.', '-', '.', 'gene_id \"chr9.g21641\"; transcript_id \"chr9.g21641.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37636714', '37636782', '.', '+', '.', 'gene_id \"chr9.g21705\"; transcript_id \"chr9.g21705.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37655945', '37657028', '.', '-', '.', 'gene_id \"chr9.g21720\"; transcript_id \"chr9.g21720.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37656916', '37657028', '.', '-', '.', 'gene_id \"chr9.g21720\"; transcript_id \"chr9.g21720.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37718065', '37718107', '.', '-', '.', 'gene_id \"chr9.g21755\"; transcript_id \"chr9.g21755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37825361', '37826312', '.', '+', '.', 'gene_id \"chr9.g21781\"; transcript_id \"chr9.g21781.i7\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37819759', '37820187', '.', '+', '.', 'gene_id \"chr9.g21781\"; transcript_id \"chr9.g21781.i8\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37823535', '37823760', '.', '+', '.', 'gene_id \"chr9.g21781\"; transcript_id \"chr9.g21781.i10\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37821616', '37822077', '.', '+', '.', 'gene_id \"chr9.g21782\"; transcript_id \"chr9.g21782.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37840532', '37840773', '.', '+', '.', 'gene_id \"chr9.g21814\"; transcript_id \"chr9.g21814.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37840390', '37840545', '.', '-', '.', 'gene_id \"chr9.g21815\"; transcript_id \"chr9.g21815.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '49874693', '49874865', '.', '+', '.', 'gene_id \"chr17.g17853\"; transcript_id \"chr17.g17853.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37900933', '37901007', '.', '+', '.', 'gene_id \"chr9.g21926\"; transcript_id \"chr9.g21926.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37908390', '37908642', '.', '+', '.', 'gene_id \"chr9.g21939\"; transcript_id \"chr9.g21939.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '38058647', '38058676', '.', '+', '.', 'gene_id \"chr9.g22032\"; transcript_id \"chr9.g22032.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '38170295', '38170610', '.', '-', '.', 'gene_id \"chr9.g22046\"; transcript_id \"chr9.g22046.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '50354809', '50355030', '.', '+', '.', 'gene_id \"chr17.g18085\"; transcript_id \"chr17.g18085.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '50377761', '50378164', '.', '-', '.', 'gene_id \"chr17.g18097\"; transcript_id \"chr17.g18097.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39010337', '39010761', '.', '-', '.', 'gene_id \"chr9.g22142\"; transcript_id \"chr9.g22142.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39241495', '39241534', '.', '+', '.', 'gene_id \"chr9.g22161\"; transcript_id \"chr9.g22161.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '50613229', '50613383', '.', '+', '.', 'gene_id \"chr17.g18195\"; transcript_id \"chr17.g18195.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39420706', '39420754', '.', '-', '.', 'gene_id \"chr9.g22207\"; transcript_id \"chr9.g22207.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39474693', '39474873', '.', '-', '.', 'gene_id \"chr9.g22224\"; transcript_id \"chr9.g22224.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39535103', '39535366', '.', '-', '.', 'gene_id \"chr9.g22232\"; transcript_id \"chr9.g22232.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39547615', '39547822', '.', '-', '.', 'gene_id \"chr9.g22235\"; transcript_id \"chr9.g22235.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39744167', '39744388', '.', '+', '.', 'gene_id \"chr9.g22265\"; transcript_id \"chr9.g22265.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '50930131', '50930312', '.', '-', '.', 'gene_id \"chr17.g18291\"; transcript_id \"chr17.g18291.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '51050872', '51051120', '.', '+', '.', 'gene_id \"chr17.g18358\"; transcript_id \"chr17.g18358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40048199', '40048263', '.', '+', '.', 'gene_id \"chr9.g22454\"; transcript_id \"chr9.g22454.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '51317652', '51317849', '.', '-', '.', 'gene_id \"chr17.g18553\"; transcript_id \"chr17.g18553.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40184798', '40185016', '.', '+', '.', 'gene_id \"chr9.g22590\"; transcript_id \"chr9.g22590.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40245740', '40245929', '.', '-', '.', 'gene_id \"chr9.g22647\"; transcript_id \"chr9.g22647.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40418322', '40420709', '.', '+', '.', 'gene_id \"chr9.g22751\"; transcript_id \"chr9.g22751.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '51798989', '51799209', '.', '+', '.', 'gene_id \"chr17.g18851\"; transcript_id \"chr17.g18851.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40878321', '40878425', '.', '+', '.', 'gene_id \"chr9.g23002\"; transcript_id \"chr9.g23002.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41048964', '41049163', '.', '-', '.', 'gene_id \"chr9.g23124\"; transcript_id \"chr9.g23124.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '52510845', '52511058', '.', '+', '.', 'gene_id \"chr17.g19326\"; transcript_id \"chr17.g19326.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41476146', '41476661', '.', '+', '.', 'gene_id \"chr9.g23354\"; transcript_id \"chr9.g23354.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41533834', '41533899', '.', '-', '.', 'gene_id \"chr9.g23396\"; transcript_id \"chr9.g23396.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '53451579', '53451875', '.', '-', '.', 'gene_id \"chr17.g19611\"; transcript_id \"chr17.g19611.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42046716', '42046775', '.', '-', '.', 'gene_id \"chr9.g23633\"; transcript_id \"chr9.g23633.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42380637', '42380793', '.', '+', '.', 'gene_id \"chr9.g23889\"; transcript_id \"chr9.g23889.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42450807', '42450984', '.', '-', '.', 'gene_id \"chr9.g23950\"; transcript_id \"chr9.g23950.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42455921', '42456247', '.', '-', '.', 'gene_id \"chr9.g23952\"; transcript_id \"chr9.g23952.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42508404', '42508609', '.', '-', '.', 'gene_id \"chr9.g24008\"; transcript_id \"chr9.g24008.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42516413', '42517090', '.', '-', '.', 'gene_id \"chr9.g24019\"; transcript_id \"chr9.g24019.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42610876', '42611053', '.', '-', '.', 'gene_id \"chr9.g24127\"; transcript_id \"chr9.g24127.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42621264', '42621457', '.', '-', '.', 'gene_id \"chr9.g24144\"; transcript_id \"chr9.g24144.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42955731', '42955782', '.', '-', '.', 'gene_id \"chr9.g24388\"; transcript_id \"chr9.g24388.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '55081339', '55081497', '.', '-', '.', 'gene_id \"chr17.g20394\"; transcript_id \"chr17.g20394.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43194874', '43195138', '.', '-', '.', 'gene_id \"chr9.g24527\"; transcript_id \"chr9.g24527.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43195143', '43195391', '.', '+', '.', 'gene_id \"chr9.g24528\"; transcript_id \"chr9.g24528.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43196909', '43196961', '.', '-', '.', 'gene_id \"chr9.g24533\"; transcript_id \"chr9.g24533.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43204716', '43205039', '.', '-', '.', 'gene_id \"chr9.g24542\"; transcript_id \"chr9.g24542.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43237559', '43237912', '.', '+', '.', 'gene_id \"chr9.g24575\"; transcript_id \"chr9.g24575.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43545586', '43545720', '.', '+', '.', 'gene_id \"chr9.g24861\"; transcript_id \"chr9.g24861.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43766279', '43766602', '.', '-', '.', 'gene_id \"chr9.g24997\"; transcript_id \"chr9.g24997.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43831758', '43831821', '.', '+', '.', 'gene_id \"chr9.g25065\"; transcript_id \"chr9.g25065.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43842380', '43842425', '.', '-', '.', 'gene_id \"chr9.g25068\"; transcript_id \"chr9.g25068.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44035647', '44036042', '.', '+', '.', 'gene_id \"chr9.g25183\"; transcript_id \"chr9.g25183.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44069990', '44070355', '.', '-', '.', 'gene_id \"chr9.g25198\"; transcript_id \"chr9.g25198.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44069990', '44070173', '.', '-', '.', 'gene_id \"chr9.g25198\"; transcript_id \"chr9.g25198.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44070066', '44070330', '.', '-', '.', 'gene_id \"chr9.g25198\"; transcript_id \"chr9.g25198.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44070066', '44070355', '.', '-', '.', 'gene_id \"chr9.g25198\"; transcript_id \"chr9.g25198.i5\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44091245', '44091290', '.', '-', '.', 'gene_id \"chr9.g25231\"; transcript_id \"chr9.g25231.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44141662', '44142310', '.', '-', '.', 'gene_id \"chr9.g25274\"; transcript_id \"chr9.g25274.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '57401644', '57401788', '.', '-', '.', 'gene_id \"chr17.g21283\"; transcript_id \"chr17.g21283.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44243805', '44243835', '.', '-', '.', 'gene_id \"chr9.g25349\"; transcript_id \"chr9.g25349.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '57752744', '57752794', '.', '+', '.', 'gene_id \"chr17.g21486\"; transcript_id \"chr17.g21486.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '57809186', '57809226', '.', '+', '.', 'gene_id \"chr17.g21491\"; transcript_id \"chr17.g21491.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44404394', '44404576', '.', '-', '.', 'gene_id \"chr9.g25502\"; transcript_id \"chr9.g25502.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44438365', '44438425', '.', '+', '.', 'gene_id \"chr9.g25516\"; transcript_id \"chr9.g25516.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44467869', '44468028', '.', '+', '.', 'gene_id \"chr9.g25548\"; transcript_id \"chr9.g25548.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44486462', '44486612', '.', '+', '.', 'gene_id \"chr9.g25562\"; transcript_id \"chr9.g25562.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58068535', '58068755', '.', '+', '.', 'gene_id \"chr17.g21583\"; transcript_id \"chr17.g21583.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58152267', '58152445', '.', '+', '.', 'gene_id \"chr17.g21624\"; transcript_id \"chr17.g21624.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58368185', '58368230', '.', '-', '.', 'gene_id \"chr17.g21697\"; transcript_id \"chr17.g21697.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44649312', '44649594', '.', '-', '.', 'gene_id \"chr9.g25715\"; transcript_id \"chr9.g25715.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58461996', '58462198', '.', '-', '.', 'gene_id \"chr17.g21814\"; transcript_id \"chr17.g21814.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44740848', '44741302', '.', '-', '.', 'gene_id \"chr9.g25800\"; transcript_id \"chr9.g25800.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44751676', '44751856', '.', '-', '.', 'gene_id \"chr9.g25807\"; transcript_id \"chr9.g25807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44769246', '44769399', '.', '-', '.', 'gene_id \"chr9.g25818\"; transcript_id \"chr9.g25818.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44820116', '44820179', '.', '-', '.', 'gene_id \"chr9.g25887\"; transcript_id \"chr9.g25887.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '44950819', '44950849', '.', '+', '.', 'gene_id \"chr9.g25955\"; transcript_id \"chr9.g25955.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58841105', '58841177', '.', '+', '.', 'gene_id \"chr17.g22009\"; transcript_id \"chr17.g22009.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45049064', '45049237', '.', '+', '.', 'gene_id \"chr9.g26036\"; transcript_id \"chr9.g26036.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58932060', '58933201', '.', '-', '.', 'gene_id \"chr17.g22069\"; transcript_id \"chr17.g22069.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45057080', '45057340', '.', '+', '.', 'gene_id \"chr9.g26050\"; transcript_id \"chr9.g26050.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45057459', '45057748', '.', '+', '.', 'gene_id \"chr9.g26051\"; transcript_id \"chr9.g26051.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45057728', '45057937', '.', '-', '.', 'gene_id \"chr9.g26052\"; transcript_id \"chr9.g26052.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45058087', '45058406', '.', '-', '.', 'gene_id \"chr9.g26053\"; transcript_id \"chr9.g26053.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45103445', '45103512', '.', '-', '.', 'gene_id \"chr9.g26089\"; transcript_id \"chr9.g26089.i6\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45116217', '45116258', '.', '+', '.', 'gene_id \"chr9.g26104\"; transcript_id \"chr9.g26104.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45120360', '45120396', '.', '-', '.', 'gene_id \"chr9.g26113\"; transcript_id \"chr9.g26113.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '59038557', '59038767', '.', '-', '.', 'gene_id \"chr17.g22180\"; transcript_id \"chr17.g22180.i8\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '59037778', '59038406', '.', '-', '.', 'gene_id \"chr17.g22185\"; transcript_id \"chr17.g22185.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45185086', '45185121', '.', '+', '.', 'gene_id \"chr9.g26169\"; transcript_id \"chr9.g26169.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '59053857', '59054198', '.', '+', '.', 'gene_id \"chr17.g22196\"; transcript_id \"chr17.g22196.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45412311', '45412614', '.', '-', '.', 'gene_id \"chr9.g26440\"; transcript_id \"chr9.g26440.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45412680', '45412867', '.', '-', '.', 'gene_id \"chr9.g26441\"; transcript_id \"chr9.g26441.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45413191', '45413351', '.', '+', '.', 'gene_id \"chr9.g26444\"; transcript_id \"chr9.g26444.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '59392946', '59393169', '.', '-', '.', 'gene_id \"chr17.g22488\"; transcript_id \"chr17.g22488.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45488313', '45488481', '.', '+', '.', 'gene_id \"chr9.g26498\"; transcript_id \"chr9.g26498.i9\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45679741', '45679804', '.', '-', '.', 'gene_id \"chr9.g26663\"; transcript_id \"chr9.g26663.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr17', 'PASA', 'exon', '60026681', '60026724', '.', '+', '.', 'gene_id \"chr17.g23112\"; transcript_id \"chr17.g23112.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46218402', '46218459', '.', '+', '.', 'gene_id \"chr9.g27137\"; transcript_id \"chr9.g27137.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '60170990', '60171042', '.', '+', '.', 'gene_id \"chr17.g23254\"; transcript_id \"chr17.g23254.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46299416', '46299470', '.', '-', '.', 'gene_id \"chr9.g27226\"; transcript_id \"chr9.g27226.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46352255', '46352285', '.', '-', '.', 'gene_id \"chr9.g27276\"; transcript_id \"chr9.g27276.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46525952', '46525987', '.', '-', '.', 'gene_id \"chr9.g27358\"; transcript_id \"chr9.g27358.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46628388', '46628467', '.', '+', '.', 'gene_id \"chr9.g27410\"; transcript_id \"chr9.g27410.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46725845', '46726328', '.', '-', '.', 'gene_id \"chr9.g27426\"; transcript_id \"chr9.g27426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46962123', '46962301', '.', '-', '.', 'gene_id \"chr9.g27466\"; transcript_id \"chr9.g27466.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46962468', '46962698', '.', '-', '.', 'gene_id \"chr9.g27467\"; transcript_id \"chr9.g27467.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '60542602', '60542764', '.', '-', '.', 'gene_id \"chr17.g23622\"; transcript_id \"chr17.g23622.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '47403692', '47403762', '.', '-', '.', 'gene_id \"chr9.g27746\"; transcript_id \"chr9.g27746.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '47594511', '47594541', '.', '-', '.', 'gene_id \"chr9.g27807\"; transcript_id \"chr9.g27807.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '47650789', '47650831', '.', '-', '.', 'gene_id \"chr9.g27816\"; transcript_id \"chr9.g27816.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48010216', '48010261', '.', '+', '.', 'gene_id \"chr9.g28134\"; transcript_id \"chr9.g28134.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '61963969', '61964210', '.', '+', '.', 'gene_id \"chr17.g24229\"; transcript_id \"chr17.g24229.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '61983668', '61984049', '.', '-', '.', 'gene_id \"chr17.g24245\"; transcript_id \"chr17.g24245.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '61984057', '61984190', '.', '-', '.', 'gene_id \"chr17.g24246\"; transcript_id \"chr17.g24246.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48069679', '48069829', '.', '-', '.', 'gene_id \"chr9.g28196\"; transcript_id \"chr9.g28196.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48230031', '48231338', '.', '-', '.', 'gene_id \"chr9.g28315\"; transcript_id \"chr9.g28315.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48231674', '48232894', '.', '-', '.', 'gene_id \"chr9.g28316\"; transcript_id \"chr9.g28316.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '62241472', '62241824', '.', '+', '.', 'gene_id \"chr17.g24389\"; transcript_id \"chr17.g24389.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48524430', '48524661', '.', '+', '.', 'gene_id \"chr9.g28541\"; transcript_id \"chr9.g28541.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48524643', '48525708', '.', '-', '.', 'gene_id \"chr9.g28542\"; transcript_id \"chr9.g28542.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48526136', '48526640', '.', '-', '.', 'gene_id \"chr9.g28545\"; transcript_id \"chr9.g28545.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '62592960', '62593019', '.', '+', '.', 'gene_id \"chr17.g24617\"; transcript_id \"chr17.g24617.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '62657084', '62657167', '.', '+', '.', 'gene_id \"chr17.g24679\"; transcript_id \"chr17.g24679.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '62837051', '62837093', '.', '+', '.', 'gene_id \"chr17.g24787\"; transcript_id \"chr17.g24787.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49105169', '49105338', '.', '-', '.', 'gene_id \"chr9.g28829\"; transcript_id \"chr9.g28829.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '62896589', '62896771', '.', '-', '.', 'gene_id \"chr17.g24844\"; transcript_id \"chr17.g24844.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49195002', '49195186', '.', '-', '.', 'gene_id \"chr9.g28901\"; transcript_id \"chr9.g28901.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49223054', '49223207', '.', '+', '.', 'gene_id \"chr9.g28910\"; transcript_id \"chr9.g28910.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49623142', '49623173', '.', '+', '.', 'gene_id \"chr9.g29088\"; transcript_id \"chr9.g29088.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63409260', '63409318', '.', '-', '.', 'gene_id \"chr17.g25078\"; transcript_id \"chr17.g25078.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49783479', '49783795', '.', '-', '.', 'gene_id \"chr9.g29235\"; transcript_id \"chr9.g29235.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49783777', '49783943', '.', '+', '.', 'gene_id \"chr9.g29236\"; transcript_id \"chr9.g29236.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49814235', '49814418', '.', '-', '.', 'gene_id \"chr9.g29252\"; transcript_id \"chr9.g29252.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63531263', '63531446', '.', '+', '.', 'gene_id \"chr17.g25130\"; transcript_id \"chr17.g25130.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63532074', '63532286', '.', '+', '.', 'gene_id \"chr17.g25132\"; transcript_id \"chr17.g25132.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50016897', '50016956', '.', '+', '.', 'gene_id \"chr9.g29360\"; transcript_id \"chr9.g29360.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50262569', '50262628', '.', '-', '.', 'gene_id \"chr9.g29471\"; transcript_id \"chr9.g29471.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50284290', '50284325', '.', '-', '.', 'gene_id \"chr9.g29475\"; transcript_id \"chr9.g29475.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50444369', '50444553', '.', '-', '.', 'gene_id \"chr9.g29499\"; transcript_id \"chr9.g29499.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50444185', '50444387', '.', '+', '.', 'gene_id \"chr9.g29500\"; transcript_id \"chr9.g29500.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50444594', '50444812', '.', '-', '.', 'gene_id \"chr9.g29501\"; transcript_id \"chr9.g29501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50551438', '50551469', '.', '+', '.', 'gene_id \"chr9.g29525\"; transcript_id \"chr9.g29525.i1\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63996044', '63996204', '.', '+', '.', 'gene_id \"chr17.g25429\"; transcript_id \"chr17.g25429.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '50736872', '50736918', '.', '+', '.', 'gene_id \"chr9.g29567\"; transcript_id \"chr9.g29567.i2\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '5121008', '5121169', '100', '+', '.', 'gene_id \"chr17.g25535\"; transcript_id \"blat.proc18128.chain_20787\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '34298441', '34298475', '100', '-', '.', 'gene_id \"chr17.g25543\"; transcript_id \"blat.proc28144.chain_18050\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '46196634', '46196681', '100', '+', '.', 'gene_id \"chr17.g25681\"; transcript_id \"blat.proc18128.chain_8007\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '52916012', '52916050', '100', '-', '.', 'gene_id \"chr17.g25973\"; transcript_id \"blat.proc4587.chain_10413\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '37815131', '37815172', '100', '+', '.', 'gene_id \"chr17.g26157\"; transcript_id \"blat.proc18128.chain_15034\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '23018044', '23018077', '100', '+', '.', 'gene_id \"chr17.g26939\"; transcript_id \"blat.proc4587.chain_4157\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '59553847', '59553882', '100', '+', '.', 'gene_id \"chr17.g26962\"; transcript_id \"blat.proc18128.chain_4805\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '48148943', '48148987', '100', '+', '.', 'gene_id \"chr17.g26968\"; transcript_id \"blat.proc18128.chain_7901\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '30583597', '30583645', '100', '+', '.', 'gene_id \"chr17.g27051\"; transcript_id \"blat.proc18128.chain_2349\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '39246552', '39246662', '100', '-', '.', 'gene_id \"chr17.g27064\"; transcript_id \"blat.proc18128.chain_7273\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '20809669', '20809710', '100', '-', '.', 'gene_id \"chr17.g27110\"; transcript_id \"blat.proc4005.chain_624\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '33799601', '33799655', '100', '-', '.', 'gene_id \"chr17.g27265\"; transcript_id \"blat.proc4005.chain_2277\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '33773722', '33773768', '100', '-', '.', 'gene_id \"chr17.g28064\"; transcript_id \"blat.proc28144.chain_10296\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '26285451', '26285495', '100', '-', '.', 'gene_id \"chr17.g28100\"; transcript_id \"blat.proc28144.chain_1751\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63927808', '63928037', '100', '+', '.', 'gene_id \"chr17.g28214\"; transcript_id \"blat.proc18128.chain_21192\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '34034481', '34034517', '100', '+', '.', 'gene_id \"chr17.g28379\"; transcript_id \"blat.proc4005.chain_16169\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '12500747', '12501193', '100', '-', '.', 'gene_id \"chr17.g28560\"; transcript_id \"blat.proc4587.chain_12971\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '52763891', '52764203', '100', '-', '.', 'gene_id \"chr17.g29059\"; transcript_id \"blat.proc18128.chain_11924\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63688414', '63688468', '100', '+', '.', 'gene_id \"chr17.g29091\"; transcript_id \"blat.proc28144.chain_22004\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '52022774', '52022841', '100', '-', '.', 'gene_id \"chr17.g29188\"; transcript_id \"blat.proc4587.chain_11766\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58467635', '58467672', '100', '-', '.', 'gene_id \"chr17.g29265\"; transcript_id \"blat.proc4587.chain_2183\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '15268889', '15269091', '100', '+', '.', 'gene_id \"chr17.g29272\"; transcript_id \"blat.proc18128.chain_23375\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '49370608', '49370641', '100', '+', '.', 'gene_id \"chr17.g29489\"; transcript_id \"blat.proc4587.chain_86\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63218984', '63219013', '100', '-', '.', 'gene_id \"chr17.g29641\"; transcript_id \"blat.proc28144.chain_21364\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '60693806', '60693839', '100', '+', '.', 'gene_id \"chr17.g29699\"; transcript_id \"blat.proc28144.chain_24168\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '49128660', '49128776', '100', '+', '.', 'gene_id \"chr17.g30216\"; transcript_id \"blat.proc4587.chain_8495\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '44938161', '44938322', '100', '+', '.', 'gene_id \"chr17.g31054\"; transcript_id \"blat.proc18128.chain_20788\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '41904012', '41904100', '100', '-', '.', 'gene_id \"chr17.g31164\"; transcript_id \"blat.proc28144.chain_7643\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '58627687', '58627716', '100', '+', '.', 'gene_id \"chr17.g31213\"; transcript_id \"blat.proc4005.chain_11979\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '41497760', '41497821', '100', '+', '.', 'gene_id \"chr17.g31305\"; transcript_id \"blat.proc28144.chain_4844\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '26031739', '26031826', '100', '+', '.', 'gene_id \"chr17.g31925\"; transcript_id \"blat.proc18128.chain_2809\";']\n",
      "negative:  ['chr17', 'PASA', 'exon', '63413099', '63413281', '100', '-', '.', 'gene_id \"chr17.g32125\"; transcript_id \"blat.proc4587.chain_2083\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51135802', '51135977', '.', '-', '.', 'gene_id \"chr9.g29805\"; transcript_id \"chr9.g29805.i14\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51135282', '51135496', '.', '+', '.', 'gene_id \"chr9.g29811\"; transcript_id \"chr9.g29811.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51304822', '51304861', '.', '-', '.', 'gene_id \"chr9.g29899\"; transcript_id \"chr9.g29899.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51343109', '51343340', '.', '-', '.', 'gene_id \"chr9.g29900\"; transcript_id \"chr9.g29900.i1\";']\n",
      "negative:  ['scaffold_1216', 'PASA', 'exon', '6687', '6854', '100', '+', '.', 'gene_id \"scaffold_1216.g15\"; transcript_id \"blat.proc18128.chain_17505\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51558688', '51558803', '.', '+', '.', 'gene_id \"chr9.g30030\"; transcript_id \"chr9.g30030.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51575316', '51575519', '.', '+', '.', 'gene_id \"chr9.g30060\"; transcript_id \"chr9.g30060.i1\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '24628', '25079', '.', '+', '.', 'gene_id \"scaffold_126.g5\"; transcript_id \"scaffold_126.g5.i2\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '165628', '165660', '.', '-', '.', 'gene_id \"scaffold_126.g118\"; transcript_id \"scaffold_126.g118.i1\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '309171', '309327', '.', '+', '.', 'gene_id \"scaffold_126.g189\"; transcript_id \"scaffold_126.g189.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52009776', '52010190', '.', '-', '.', 'gene_id \"chr9.g30283\"; transcript_id \"chr9.g30283.i1\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '604928', '605146', '.', '+', '.', 'gene_id \"scaffold_126.g429\"; transcript_id \"scaffold_126.g429.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52213441', '52213611', '.', '-', '.', 'gene_id \"chr9.g30492\"; transcript_id \"chr9.g30492.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52213984', '52214200', '.', '-', '.', 'gene_id \"chr9.g30493\"; transcript_id \"chr9.g30493.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52214365', '52214787', '.', '+', '.', 'gene_id \"chr9.g30496\"; transcript_id \"chr9.g30496.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52215256', '52216275', '.', '-', '.', 'gene_id \"chr9.g30497\"; transcript_id \"chr9.g30497.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52217143', '52217331', '.', '+', '.', 'gene_id \"chr9.g30501\"; transcript_id \"chr9.g30501.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52217691', '52217943', '.', '+', '.', 'gene_id \"chr9.g30503\"; transcript_id \"chr9.g30503.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52217968', '52218199', '.', '-', '.', 'gene_id \"chr9.g30504\"; transcript_id \"chr9.g30504.i1\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '814936', '815099', '.', '+', '.', 'gene_id \"scaffold_126.g541\"; transcript_id \"scaffold_126.g541.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52465964', '52466007', '.', '-', '.', 'gene_id \"chr9.g30714\"; transcript_id \"chr9.g30714.i1\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '7874', '7911', '100', '-', '.', 'gene_id \"scaffold_126.g783\"; transcript_id \"blat.proc18128.chain_6410\";']\n",
      "negative:  ['scaffold_126', 'PASA', 'exon', '690543', '690630', '100', '-', '.', 'gene_id \"scaffold_126.g827\"; transcript_id \"blat.proc28144.chain_5372\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52530236', '52530306', '.', '-', '.', 'gene_id \"chr9.g30770\"; transcript_id \"chr9.g30770.i1\";']\n",
      "negative:  ['scaffold_1644', 'PASA', 'exon', '48988', '49031', '100', '+', '.', 'gene_id \"scaffold_1644.g15\"; transcript_id \"blat.proc28144.chain_6558\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52679087', '52679270', '.', '+', '.', 'gene_id \"chr9.g30880\"; transcript_id \"chr9.g30880.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52838631', '52838663', '.', '-', '.', 'gene_id \"chr9.g30968\"; transcript_id \"chr9.g30968.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52839881', '52839920', '.', '+', '.', 'gene_id \"chr9.g30970\"; transcript_id \"chr9.g30970.i1\";']\n",
      "negative:  ['scaffold_1490', 'PASA', 'exon', '1755', '1809', '100', '+', '.', 'gene_id \"scaffold_1490.g1\"; transcript_id \"blat.proc18128.chain_12872\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52896994', '52897230', '.', '-', '.', 'gene_id \"chr9.g31006\"; transcript_id \"chr9.g31006.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52897208', '52897440', '.', '+', '.', 'gene_id \"chr9.g31007\"; transcript_id \"chr9.g31007.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53031751', '53031781', '.', '+', '.', 'gene_id \"chr9.g31071\"; transcript_id \"chr9.g31071.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53232638', '53232767', '.', '+', '.', 'gene_id \"chr9.g31218\"; transcript_id \"chr9.g31218.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53427585', '53427938', '.', '+', '.', 'gene_id \"chr9.g31328\"; transcript_id \"chr9.g31328.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53528649', '53528876', '.', '-', '.', 'gene_id \"chr9.g31438\"; transcript_id \"chr9.g31438.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '53570205', '53570250', '.', '+', '.', 'gene_id \"chr9.g31469\"; transcript_id \"chr9.g31469.i1\";']\n",
      "negative:  ['scaffold_1236', 'PASA', 'exon', '16242', '16280', '100', '+', '.', 'gene_id \"scaffold_1236.g2\"; transcript_id \"blat.proc28144.chain_4665\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53604462', '53604626', '.', '+', '.', 'gene_id \"chr9.g31522\"; transcript_id \"chr9.g31522.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53604628', '53604983', '.', '-', '.', 'gene_id \"chr9.g31523\"; transcript_id \"chr9.g31523.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53683116', '53683314', '.', '+', '.', 'gene_id \"chr9.g31547\"; transcript_id \"chr9.g31547.i1\";']\n",
      "negative:  ['scaffold_1575', 'PASA', 'exon', '8834', '9048', '.', '-', '.', 'gene_id \"scaffold_1575.g9\"; transcript_id \"scaffold_1575.g9.i1\";']\n",
      "negative:  ['scaffold_1575', 'PASA', 'exon', '5064', '5323', '100', '+', '.', 'gene_id \"scaffold_1575.g18\"; transcript_id \"blat.proc18128.chain_21877\";']\n",
      "negative:  ['scaffold_1687', 'PASA', 'exon', '16629', '16763', '.', '-', '.', 'gene_id \"scaffold_1687.g8\"; transcript_id \"scaffold_1687.g8.i1\";']\n",
      "negative:  ['scaffold_1687', 'PASA', 'exon', '70724', '70775', '100', '+', '.', 'gene_id \"scaffold_1687.g37\"; transcript_id \"blat.proc4587.chain_2463\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53985476', '53985659', '.', '-', '.', 'gene_id \"chr9.g31769\"; transcript_id \"chr9.g31769.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54078511', '54078641', '.', '+', '.', 'gene_id \"chr9.g31809\"; transcript_id \"chr9.g31809.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54287337', '54287487', '.', '-', '.', 'gene_id \"chr9.g31913\"; transcript_id \"chr9.g31913.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54669297', '54669569', '.', '-', '.', 'gene_id \"chr9.g32213\"; transcript_id \"chr9.g32213.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54821523', '54824190', '.', '-', '.', 'gene_id \"chr9.g32410\"; transcript_id \"chr9.g32410.i3\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54837805', '54838671', '.', '+', '.', 'gene_id \"chr9.g32412\"; transcript_id \"chr9.g32412.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54833616', '54833792', '.', '+', '.', 'gene_id \"chr9.g32418\"; transcript_id \"chr9.g32418.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54836905', '54837079', '.', '+', '.', 'gene_id \"chr9.g32425\"; transcript_id \"chr9.g32425.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54837059', '54837305', '.', '-', '.', 'gene_id \"chr9.g32426\"; transcript_id \"chr9.g32426.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54837461', '54837662', '.', '+', '.', 'gene_id \"chr9.g32428\"; transcript_id \"chr9.g32428.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54864274', '54864391', '.', '-', '.', 'gene_id \"chr9.g32454\"; transcript_id \"chr9.g32454.i9\";']\n",
      "negative:  ['scaffold_1699', 'PASA', 'exon', '99614', '99659', '.', '+', '.', 'gene_id \"scaffold_1699.g19\"; transcript_id \"scaffold_1699.g19.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55066251', '55066436', '.', '+', '.', 'gene_id \"chr9.g32628\"; transcript_id \"chr9.g32628.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55066586', '55066769', '.', '+', '.', 'gene_id \"chr9.g32629\"; transcript_id \"chr9.g32629.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55083991', '55084373', '.', '-', '.', 'gene_id \"chr9.g32653\"; transcript_id \"chr9.g32653.i1\";']\n",
      "negative:  ['scaffold_1505', 'PASA', 'exon', '14501', '14572', '100', '+', '.', 'gene_id \"scaffold_1505.g10\"; transcript_id \"blat.proc18128.chain_6293\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55185002', '55185061', '.', '-', '.', 'gene_id \"chr9.g32723\"; transcript_id \"chr9.g32723.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55275150', '55275836', '.', '+', '.', 'gene_id \"chr9.g32784\"; transcript_id \"chr9.g32784.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55287869', '55288328', '.', '-', '.', 'gene_id \"chr9.g32843\"; transcript_id \"chr9.g32843.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55424137', '55424331', '.', '+', '.', 'gene_id \"chr9.g32987\"; transcript_id \"chr9.g32987.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1373632', '1373700', '.', '+', '.', 'gene_id \"chr2.g106\"; transcript_id \"chr2.g106.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55558543', '55558738', '.', '+', '.', 'gene_id \"chr9.g33117\"; transcript_id \"chr9.g33117.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1543368', '1543603', '.', '-', '.', 'gene_id \"chr2.g248\"; transcript_id \"chr2.g248.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1553882', '1554075', '.', '-', '.', 'gene_id \"chr2.g265\"; transcript_id \"chr2.g265.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55676813', '55676923', '.', '-', '.', 'gene_id \"chr9.g33239\"; transcript_id \"chr9.g33239.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1700698', '1700958', '.', '+', '.', 'gene_id \"chr2.g455\"; transcript_id \"chr2.g455.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '56107320', '56107588', '.', '-', '.', 'gene_id \"chr9.g33444\"; transcript_id \"chr9.g33444.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1778671', '1778715', '.', '-', '.', 'gene_id \"chr2.g473\"; transcript_id \"chr2.g473.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '56140855', '56140901', '.', '-', '.', 'gene_id \"chr9.g33472\"; transcript_id \"chr9.g33472.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '56141163', '56141386', '.', '-', '.', 'gene_id \"chr9.g33473\"; transcript_id \"chr9.g33473.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '56371423', '56371643', '.', '-', '.', 'gene_id \"chr9.g33534\"; transcript_id \"chr9.g33534.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '2085794', '2085947', '.', '+', '.', 'gene_id \"chr2.g706\"; transcript_id \"chr2.g706.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57346465', '57346615', '.', '+', '.', 'gene_id \"chr9.g33952\"; transcript_id \"chr9.g33952.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '2793523', '2793556', '.', '-', '.', 'gene_id \"chr2.g1022\"; transcript_id \"chr2.g1022.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '2822126', '2822331', '.', '+', '.', 'gene_id \"chr2.g1046\"; transcript_id \"chr2.g1046.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57504623', '57504683', '.', '+', '.', 'gene_id \"chr9.g34100\"; transcript_id \"chr9.g34100.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '3496344', '3496447', '.', '+', '.', 'gene_id \"chr2.g1178\"; transcript_id \"chr2.g1178.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57558198', '57558272', '.', '-', '.', 'gene_id \"chr9.g34159\"; transcript_id \"chr9.g34159.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57560597', '57560771', '.', '+', '.', 'gene_id \"chr9.g34162\"; transcript_id \"chr9.g34162.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '3592932', '3593131', '.', '+', '.', 'gene_id \"chr2.g1282\"; transcript_id \"chr2.g1282.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '3598408', '3599266', '.', '-', '.', 'gene_id \"chr2.g1288\"; transcript_id \"chr2.g1288.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '3727768', '3728021', '.', '+', '.', 'gene_id \"chr2.g1309\"; transcript_id \"chr2.g1309.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57787650', '57788099', '.', '-', '.', 'gene_id \"chr9.g34327\"; transcript_id \"chr9.g34327.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57788137', '57788317', '.', '-', '.', 'gene_id \"chr9.g34329\"; transcript_id \"chr9.g34329.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57878119', '57878310', '.', '+', '.', 'gene_id \"chr9.g34423\"; transcript_id \"chr9.g34423.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4302470', '4302884', '.', '-', '.', 'gene_id \"chr2.g1459\"; transcript_id \"chr2.g1459.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4307692', '4307883', '.', '-', '.', 'gene_id \"chr2.g1465\"; transcript_id \"chr2.g1465.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57951717', '57951919', '.', '+', '.', 'gene_id \"chr9.g34476\"; transcript_id \"chr9.g34476.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57951926', '57952148', '.', '+', '.', 'gene_id \"chr9.g34478\"; transcript_id \"chr9.g34478.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4351205', '4351257', '.', '-', '.', 'gene_id \"chr2.g1501\"; transcript_id \"chr2.g1501.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4434155', '4434199', '.', '-', '.', 'gene_id \"chr2.g1558\"; transcript_id \"chr2.g1558.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4434177', '4434211', '.', '-', '.', 'gene_id \"chr2.g1558\"; transcript_id \"chr2.g1558.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58036897', '58037066', '.', '+', '.', 'gene_id \"chr9.g34612\"; transcript_id \"chr9.g34612.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58138395', '58139141', '.', '+', '.', 'gene_id \"chr9.g34648\"; transcript_id \"chr9.g34648.i4\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58164622', '58166475', '.', '+', '.', 'gene_id \"chr9.g34651\"; transcript_id \"chr9.g34651.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58159265', '58159938', '.', '+', '.', 'gene_id \"chr9.g34653\"; transcript_id \"chr9.g34653.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58139840', '58139987', '.', '+', '.', 'gene_id \"chr9.g34656\"; transcript_id \"chr9.g34656.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58164116', '58164571', '.', '+', '.', 'gene_id \"chr9.g34658\"; transcript_id \"chr9.g34658.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58152526', '58152910', '.', '+', '.', 'gene_id \"chr9.g34666\"; transcript_id \"chr9.g34666.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58152927', '58153091', '.', '-', '.', 'gene_id \"chr9.g34667\"; transcript_id \"chr9.g34667.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58153312', '58153562', '.', '+', '.', 'gene_id \"chr9.g34670\"; transcript_id \"chr9.g34670.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58157814', '58158050', '.', '+', '.', 'gene_id \"chr9.g34675\"; transcript_id \"chr9.g34675.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58158107', '58158815', '.', '+', '.', 'gene_id \"chr9.g34676\"; transcript_id \"chr9.g34676.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58167689', '58169714', '.', '-', '.', 'gene_id \"chr9.g34678\"; transcript_id \"chr9.g34678.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58171498', '58171709', '.', '-', '.', 'gene_id \"chr9.g34681\"; transcript_id \"chr9.g34681.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58174152', '58174385', '.', '+', '.', 'gene_id \"chr9.g34683\"; transcript_id \"chr9.g34683.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58173487', '58174155', '.', '-', '.', 'gene_id \"chr9.g34684\"; transcript_id \"chr9.g34684.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58176567', '58176891', '.', '+', '.', 'gene_id \"chr9.g34688\"; transcript_id \"chr9.g34688.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4661166', '4661402', '.', '+', '.', 'gene_id \"chr2.g1735\"; transcript_id \"chr2.g1735.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58209390', '58209560', '.', '-', '.', 'gene_id \"chr9.g34717\"; transcript_id \"chr9.g34717.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58235188', '58235368', '.', '+', '.', 'gene_id \"chr9.g34736\"; transcript_id \"chr9.g34736.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4784873', '4785127', '.', '+', '.', 'gene_id \"chr2.g1845\"; transcript_id \"chr2.g1845.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58278170', '58279130', '.', '+', '.', 'gene_id \"chr9.g34791\"; transcript_id \"chr9.g34791.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58282175', '58282498', '.', '+', '.', 'gene_id \"chr9.g34798\"; transcript_id \"chr9.g34798.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58282779', '58283448', '.', '+', '.', 'gene_id \"chr9.g34800\"; transcript_id \"chr9.g34800.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '6170573', '6170724', '.', '+', '.', 'gene_id \"chr2.g2116\"; transcript_id \"chr2.g2116.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '6436913', '6436971', '.', '-', '.', 'gene_id \"chr2.g2242\"; transcript_id \"chr2.g2242.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58978827', '58979343', '.', '-', '.', 'gene_id \"chr9.g35236\"; transcript_id \"chr9.g35236.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '6615944', '6616180', '.', '+', '.', 'gene_id \"chr2.g2356\"; transcript_id \"chr2.g2356.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59194841', '59194883', '.', '-', '.', 'gene_id \"chr9.g35375\"; transcript_id \"chr9.g35375.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59320372', '59320407', '.', '+', '.', 'gene_id \"chr9.g35472\"; transcript_id \"chr9.g35472.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '7456044', '7456080', '.', '+', '.', 'gene_id \"chr2.g2548\"; transcript_id \"chr2.g2548.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59387031', '59387063', '.', '+', '.', 'gene_id \"chr9.g35531\"; transcript_id \"chr9.g35531.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '7463977', '7464029', '.', '+', '.', 'gene_id \"chr2.g2552\"; transcript_id \"chr2.g2552.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '7777643', '7777830', '.', '+', '.', 'gene_id \"chr2.g2648\"; transcript_id \"chr2.g2648.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59560486', '59561106', '.', '-', '.', 'gene_id \"chr9.g35711\"; transcript_id \"chr9.g35711.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '8035944', '8036098', '.', '-', '.', 'gene_id \"chr2.g2755\"; transcript_id \"chr2.g2755.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59600662', '59600699', '.', '-', '.', 'gene_id \"chr9.g35751\"; transcript_id \"chr9.g35751.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59605661', '59606047', '.', '+', '.', 'gene_id \"chr9.g35753\"; transcript_id \"chr9.g35753.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59650596', '59650769', '.', '+', '.', 'gene_id \"chr9.g35784\"; transcript_id \"chr9.g35784.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59650600', '59650769', '.', '+', '.', 'gene_id \"chr9.g35784\"; transcript_id \"chr9.g35784.i2\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59816503', '59816683', '.', '-', '.', 'gene_id \"chr9.g35814\"; transcript_id \"chr9.g35814.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59987476', '59987653', '.', '-', '.', 'gene_id \"chr9.g35902\"; transcript_id \"chr9.g35902.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59993841', '59993995', '.', '+', '.', 'gene_id \"chr9.g35907\"; transcript_id \"chr9.g35907.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '60013819', '60013989', '.', '+', '.', 'gene_id \"chr9.g35922\"; transcript_id \"chr9.g35922.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '8530319', '8530362', '.', '+', '.', 'gene_id \"chr2.g3030\"; transcript_id \"chr2.g3030.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '9022515', '9022736', '.', '+', '.', 'gene_id \"chr2.g3366\"; transcript_id \"chr2.g3366.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '9022820', '9023020', '.', '-', '.', 'gene_id \"chr2.g3367\"; transcript_id \"chr2.g3367.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '9152032', '9152221', '.', '+', '.', 'gene_id \"chr2.g3446\"; transcript_id \"chr2.g3446.i1\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23538099', '23538195', '100', '-', '.', 'gene_id \"chr9.g36518\"; transcript_id \"blat.proc4005.chain_13030\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18771077', '18771234', '100', '-', '.', 'gene_id \"chr9.g36519\"; transcript_id \"blat.proc4005.chain_14211\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9783609', '9783837', '100', '+', '.', 'gene_id \"chr9.g36522\"; transcript_id \"blat.proc4005.chain_15640\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25943868', '25944031', '100', '-', '.', 'gene_id \"chr9.g36552\"; transcript_id \"blat.proc4005.chain_1085\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16204057', '16204394', '100', '+', '.', 'gene_id \"chr9.g36561\"; transcript_id \"blat.proc4005.chain_14254\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19455183', '19455375', '100', '+', '.', 'gene_id \"chr9.g36564\"; transcript_id \"blat.proc4005.chain_14159\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14346407', '14346587', '100', '-', '.', 'gene_id \"chr9.g36565\"; transcript_id \"blat.proc4005.chain_16294\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13945277', '13945459', '100', '+', '.', 'gene_id \"chr9.g36578\"; transcript_id \"blat.proc4005.chain_16182\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12689863', '12690067', '100', '+', '.', 'gene_id \"chr9.g36582\"; transcript_id \"blat.proc4005.chain_15970\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58055158', '58055241', '100', '-', '.', 'gene_id \"chr9.g36589\"; transcript_id \"blat.proc18128.chain_7271\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17882316', '17882568', '100', '+', '.', 'gene_id \"chr9.g36598\"; transcript_id \"blat.proc4005.chain_4359\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9901790', '9901867', '100', '+', '.', 'gene_id \"chr9.g36607\"; transcript_id \"blat.proc4005.chain_21092\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12068026', '12068184', '100', '+', '.', 'gene_id \"chr9.g36647\"; transcript_id \"blat.proc4005.chain_17401\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22316415', '22316655', '100', '-', '.', 'gene_id \"chr9.g36678\"; transcript_id \"blat.proc4005.chain_15006\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12263736', '12263935', '100', '-', '.', 'gene_id \"chr9.g36692\"; transcript_id \"blat.proc4005.chain_16589\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23394763', '23395136', '100', '-', '.', 'gene_id \"chr9.g36693\"; transcript_id \"blat.proc4005.chain_3272\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12939633', '12939814', '100', '-', '.', 'gene_id \"chr9.g36697\"; transcript_id \"blat.proc4005.chain_18740\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21384947', '21385120', '100', '+', '.', 'gene_id \"chr9.g36708\"; transcript_id \"blat.proc4005.chain_14935\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19032997', '19033276', '100', '+', '.', 'gene_id \"chr9.g36719\"; transcript_id \"blat.proc4005.chain_14782\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20437949', '20438109', '100', '-', '.', 'gene_id \"chr9.g36720\"; transcript_id \"blat.proc4005.chain_14909\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11033701', '11033859', '100', '+', '.', 'gene_id \"chr9.g36722\"; transcript_id \"blat.proc4005.chain_23835\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18004095', '18004316', '100', '-', '.', 'gene_id \"chr9.g36730\"; transcript_id \"blat.proc4005.chain_14736\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11868484', '11868632', '100', '+', '.', 'gene_id \"chr9.g36735\"; transcript_id \"blat.proc4005.chain_18967\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11035832', '11036036', '100', '-', '.', 'gene_id \"chr9.g36736\"; transcript_id \"blat.proc4005.chain_15550\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6966423', '6966469', '100', '-', '.', 'gene_id \"chr9.g36739\"; transcript_id \"blat.proc18128.chain_7827\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21198216', '21198377', '100', '+', '.', 'gene_id \"chr9.g36741\"; transcript_id \"blat.proc4005.chain_8649\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22577071', '22577287', '100', '-', '.', 'gene_id \"chr9.g36745\"; transcript_id \"blat.proc4005.chain_2892\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21958490', '21958647', '100', '-', '.', 'gene_id \"chr9.g36747\"; transcript_id \"blat.proc4005.chain_13738\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20291133', '20291305', '100', '-', '.', 'gene_id \"chr9.g36757\"; transcript_id \"blat.proc4005.chain_15205\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23734196', '23734406', '100', '-', '.', 'gene_id \"chr9.g36768\"; transcript_id \"blat.proc4005.chain_902\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13281349', '13281528', '100', '+', '.', 'gene_id \"chr9.g36783\"; transcript_id \"blat.proc4005.chain_16199\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21126682', '21126753', '100', '+', '.', 'gene_id \"chr9.g36788\"; transcript_id \"blat.proc4005.chain_13869\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17963338', '17963635', '100', '+', '.', 'gene_id \"chr9.g36793\"; transcript_id \"blat.proc4005.chain_14464\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22204185', '22204314', '100', '+', '.', 'gene_id \"chr9.g36827\"; transcript_id \"blat.proc4005.chain_5097\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25475220', '25475292', '100', '-', '.', 'gene_id \"chr9.g36841\"; transcript_id \"blat.proc4005.chain_12439\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10041289', '10041461', '100', '-', '.', 'gene_id \"chr9.g36842\"; transcript_id \"blat.proc4005.chain_18669\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11482612', '11482840', '100', '-', '.', 'gene_id \"chr9.g36844\"; transcript_id \"blat.proc4005.chain_15563\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19875771', '19875949', '100', '-', '.', 'gene_id \"chr9.g36873\"; transcript_id \"blat.proc4005.chain_14918\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14363801', '14363976', '100', '-', '.', 'gene_id \"chr9.g36875\"; transcript_id \"blat.proc4005.chain_15683\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12385438', '12385583', '100', '+', '.', 'gene_id \"chr9.g36881\"; transcript_id \"blat.proc4005.chain_15512\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13254714', '13254962', '100', '+', '.', 'gene_id \"chr9.g36882\"; transcript_id \"blat.proc4005.chain_15994\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22937799', '22937986', '100', '+', '.', 'gene_id \"chr9.g36917\"; transcript_id \"blat.proc4005.chain_14944\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12922064', '12922281', '100', '+', '.', 'gene_id \"chr9.g36921\"; transcript_id \"blat.proc4005.chain_17175\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23520212', '23520317', '100', '-', '.', 'gene_id \"chr9.g36940\"; transcript_id \"blat.proc4005.chain_14638\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20989923', '20990081', '100', '-', '.', 'gene_id \"chr9.g36989\"; transcript_id \"blat.proc4005.chain_14438\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22280524', '22280587', '100', '-', '.', 'gene_id \"chr9.g37014\"; transcript_id \"blat.proc18128.chain_16513\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12882943', '12884838', '100', '-', '.', 'gene_id \"chr9.g37069\"; transcript_id \"blat.proc4005.chain_19637\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11288379', '11288510', '100', '-', '.', 'gene_id \"chr9.g37074\"; transcript_id \"blat.proc4005.chain_21323\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13280672', '13280914', '100', '+', '.', 'gene_id \"chr9.g37123\"; transcript_id \"blat.proc4005.chain_15810\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8568279', '8568431', '100', '-', '.', 'gene_id \"chr9.g37127\"; transcript_id \"blat.proc4005.chain_16350\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13399338', '13399499', '100', '-', '.', 'gene_id \"chr9.g37137\"; transcript_id \"blat.proc4005.chain_15929\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17067267', '17067587', '100', '-', '.', 'gene_id \"chr9.g37163\"; transcript_id \"blat.proc4005.chain_11706\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16347313', '16347568', '100', '-', '.', 'gene_id \"chr9.g37164\"; transcript_id \"blat.proc4005.chain_15175\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11501186', '11501395', '100', '-', '.', 'gene_id \"chr9.g37166\"; transcript_id \"blat.proc4005.chain_17436\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18772849', '18773019', '100', '+', '.', 'gene_id \"chr9.g37177\"; transcript_id \"blat.proc4005.chain_6435\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26670829', '26670953', '100', '+', '.', 'gene_id \"chr9.g37178\"; transcript_id \"blat.proc4005.chain_15110\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17671522', '17671811', '100', '+', '.', 'gene_id \"chr9.g37198\"; transcript_id \"blat.proc4005.chain_3881\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12404773', '12404950', '100', '+', '.', 'gene_id \"chr9.g37200\"; transcript_id \"blat.proc4005.chain_20982\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24719979', '24720134', '100', '-', '.', 'gene_id \"chr9.g37226\"; transcript_id \"blat.proc4005.chain_5479\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17012599', '17012805', '100', '-', '.', 'gene_id \"chr9.g37231\"; transcript_id \"blat.proc4005.chain_13803\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13729349', '13729630', '100', '-', '.', 'gene_id \"chr9.g37237\"; transcript_id \"blat.proc4005.chain_22842\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21135994', '21136037', '100', '-', '.', 'gene_id \"chr9.g37264\"; transcript_id \"blat.proc4005.chain_15173\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11047983', '11048048', '100', '+', '.', 'gene_id \"chr9.g37270\"; transcript_id \"blat.proc18128.chain_2172\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16378241', '16378285', '100', '+', '.', 'gene_id \"chr9.g37287\"; transcript_id \"blat.proc28144.chain_13124\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14466013', '14466166', '100', '-', '.', 'gene_id \"chr9.g37289\"; transcript_id \"blat.proc4005.chain_16828\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26166203', '26166370', '100', '+', '.', 'gene_id \"chr9.g37290\"; transcript_id \"blat.proc4005.chain_14166\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16810027', '16810057', '100', '-', '.', 'gene_id \"chr9.g37307\"; transcript_id \"blat.proc18128.chain_2582\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13343047', '13343203', '100', '+', '.', 'gene_id \"chr9.g37315\"; transcript_id \"blat.proc4005.chain_19696\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19506945', '19507027', '100', '-', '.', 'gene_id \"chr9.g37337\"; transcript_id \"blat.proc4005.chain_2345\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26536301', '26536416', '100', '+', '.', 'gene_id \"chr9.g37342\"; transcript_id \"blat.proc4005.chain_13664\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12243096', '12243284', '100', '+', '.', 'gene_id \"chr9.g37343\"; transcript_id \"blat.proc4005.chain_15682\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21803053', '21803214', '100', '-', '.', 'gene_id \"chr9.g37351\"; transcript_id \"blat.proc4005.chain_15305\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13941407', '13941686', '100', '+', '.', 'gene_id \"chr9.g37352\"; transcript_id \"blat.proc4005.chain_17865\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21988394', '21988562', '100', '-', '.', 'gene_id \"chr9.g37363\"; transcript_id \"blat.proc4005.chain_15064\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13248815', '13248966', '100', '-', '.', 'gene_id \"chr9.g37371\"; transcript_id \"blat.proc4005.chain_23240\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23153056', '23153146', '100', '-', '.', 'gene_id \"chr9.g37410\"; transcript_id \"blat.proc4005.chain_3890\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25910516', '25910704', '100', '-', '.', 'gene_id \"chr9.g37432\"; transcript_id \"blat.proc4005.chain_11833\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15644965', '15645005', '100', '-', '.', 'gene_id \"chr9.g37459\"; transcript_id \"blat.proc28144.chain_7862\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42034626', '42034669', '100', '-', '.', 'gene_id \"chr9.g37467\"; transcript_id \"blat.proc18128.chain_18717\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8344110', '8344176', '100', '+', '.', 'gene_id \"chr9.g37489\"; transcript_id \"blat.proc4005.chain_22049\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23804099', '23804352', '100', '-', '.', 'gene_id \"chr9.g37492\"; transcript_id \"blat.proc4005.chain_14759\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13009234', '13009313', '100', '+', '.', 'gene_id \"chr9.g37509\"; transcript_id \"blat.proc4005.chain_21764\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7550980', '7551143', '100', '+', '.', 'gene_id \"chr9.g37516\"; transcript_id \"blat.proc4005.chain_21843\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21214927', '21215107', '100', '-', '.', 'gene_id \"chr9.g37542\"; transcript_id \"blat.proc4005.chain_10302\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13036499', '13036806', '100', '+', '.', 'gene_id \"chr9.g37545\"; transcript_id \"blat.proc4005.chain_16349\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18970606', '18970797', '100', '+', '.', 'gene_id \"chr9.g37597\"; transcript_id \"blat.proc4005.chain_15161\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19051772', '19051931', '100', '-', '.', 'gene_id \"chr9.g37599\"; transcript_id \"blat.proc4005.chain_6205\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12685044', '12685211', '100', '-', '.', 'gene_id \"chr9.g37610\"; transcript_id \"blat.proc4005.chain_15823\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22603147', '22603342', '100', '-', '.', 'gene_id \"chr9.g37614\"; transcript_id \"blat.proc4005.chain_14307\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13687084', '13687340', '100', '+', '.', 'gene_id \"chr9.g37622\"; transcript_id \"blat.proc4005.chain_16337\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34622924', '34622973', '100', '-', '.', 'gene_id \"chr9.g37641\"; transcript_id \"blat.proc18128.chain_23346\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12921764', '12921988', '100', '+', '.', 'gene_id \"chr9.g37660\"; transcript_id \"blat.proc4005.chain_17184\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '33914961', '33915010', '100', '+', '.', 'gene_id \"chr9.g37663\"; transcript_id \"blat.proc4587.chain_6551\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10047125', '10047262', '100', '-', '.', 'gene_id \"chr9.g37664\"; transcript_id \"blat.proc4005.chain_15734\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21906890', '21906972', '100', '+', '.', 'gene_id \"chr9.g37666\"; transcript_id \"blat.proc4005.chain_13821\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11883927', '11884261', '100', '-', '.', 'gene_id \"chr9.g37683\"; transcript_id \"blat.proc4005.chain_19638\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11262386', '11262555', '100', '-', '.', 'gene_id \"chr9.g37689\"; transcript_id \"blat.proc4005.chain_19660\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12464955', '12465062', '100', '-', '.', 'gene_id \"chr9.g37711\"; transcript_id \"blat.proc4005.chain_17119\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23732085', '23732251', '100', '-', '.', 'gene_id \"chr9.g37720\"; transcript_id \"blat.proc4005.chain_14429\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12571643', '12571778', '100', '+', '.', 'gene_id \"chr9.g37733\"; transcript_id \"blat.proc4005.chain_16598\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8580244', '8580545', '100', '+', '.', 'gene_id \"chr9.g37735\"; transcript_id \"blat.proc4005.chain_15431\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8475200', '8475260', '100', '+', '.', 'gene_id \"chr9.g37742\"; transcript_id \"blat.proc4005.chain_3186\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18006845', '18007125', '100', '-', '.', 'gene_id \"chr9.g37754\"; transcript_id \"blat.proc4005.chain_14431\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '30887624', '30887700', '100', '+', '.', 'gene_id \"chr9.g37765\"; transcript_id \"blat.proc4587.chain_3679\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7654889', '7655135', '100', '+', '.', 'gene_id \"chr9.g37774\"; transcript_id \"blat.proc4005.chain_21699\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5196984', '5197014', '100', '+', '.', 'gene_id \"chr9.g37777\"; transcript_id \"blat.proc28144.chain_13622\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21115205', '21115276', '100', '+', '.', 'gene_id \"chr9.g37791\"; transcript_id \"blat.proc4005.chain_13871\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8037428', '8037665', '100', '+', '.', 'gene_id \"chr9.g37801\"; transcript_id \"blat.proc4005.chain_18610\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8902762', '8902797', '100', '+', '.', 'gene_id \"chr9.g37816\"; transcript_id \"blat.proc18128.chain_18191\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27864643', '27864813', '100', '-', '.', 'gene_id \"chr9.g37842\"; transcript_id \"blat.proc4005.chain_5442\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16530316', '16530592', '100', '+', '.', 'gene_id \"chr9.g37847\"; transcript_id \"blat.proc4005.chain_14937\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13856321', '13856587', '100', '+', '.', 'gene_id \"chr9.g37865\"; transcript_id \"blat.proc4005.chain_16214\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '40813371', '40813409', '100', '-', '.', 'gene_id \"chr9.g37873\"; transcript_id \"blat.proc28144.chain_21756\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16331865', '16332069', '100', '+', '.', 'gene_id \"chr9.g37882\"; transcript_id \"blat.proc4005.chain_14797\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16353943', '16354109', '100', '+', '.', 'gene_id \"chr9.g37886\"; transcript_id \"blat.proc4005.chain_15207\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10320019', '10320255', '100', '+', '.', 'gene_id \"chr9.g37887\"; transcript_id \"blat.proc4005.chain_15656\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23281275', '23281404', '100', '+', '.', 'gene_id \"chr9.g37892\"; transcript_id \"blat.proc4005.chain_252\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18341501', '18341654', '100', '-', '.', 'gene_id \"chr9.g37902\"; transcript_id \"blat.proc4005.chain_14911\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10062777', '10062960', '100', '+', '.', 'gene_id \"chr9.g37903\"; transcript_id \"blat.proc4005.chain_16297\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7811111', '7811301', '100', '+', '.', 'gene_id \"chr9.g37914\"; transcript_id \"blat.proc4005.chain_16098\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19587221', '19587401', '100', '-', '.', 'gene_id \"chr9.g37915\"; transcript_id \"blat.proc4005.chain_14590\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10361593', '10361664', '100', '+', '.', 'gene_id \"chr9.g37931\"; transcript_id \"blat.proc4005.chain_20227\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19382652', '19382683', '100', '+', '.', 'gene_id \"chr9.g37936\"; transcript_id \"blat.proc4005.chain_732\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11422578', '11422845', '100', '+', '.', 'gene_id \"chr9.g37942\"; transcript_id \"blat.proc4005.chain_18515\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18961757', '18961907', '100', '+', '.', 'gene_id \"chr9.g37947\"; transcript_id \"blat.proc4005.chain_5068\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8039528', '8039666', '100', '+', '.', 'gene_id \"chr9.g37964\"; transcript_id \"blat.proc4005.chain_21662\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11630130', '11630264', '100', '-', '.', 'gene_id \"chr9.g37989\"; transcript_id \"blat.proc4005.chain_19257\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26601837', '26602003', '100', '-', '.', 'gene_id \"chr9.g38004\"; transcript_id \"blat.proc4005.chain_14267\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18960629', '18960789', '100', '-', '.', 'gene_id \"chr9.g38027\"; transcript_id \"blat.proc4005.chain_5126\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10025787', '10025971', '100', '+', '.', 'gene_id \"chr9.g38028\"; transcript_id \"blat.proc4005.chain_16211\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14485239', '14485421', '100', '+', '.', 'gene_id \"chr9.g38034\"; transcript_id \"blat.proc4005.chain_16116\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8300553', '8300662', '100', '-', '.', 'gene_id \"chr9.g38042\"; transcript_id \"blat.proc4005.chain_21437\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26396199', '26396279', '100', '-', '.', 'gene_id \"chr9.g38059\"; transcript_id \"blat.proc4005.chain_9383\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13227994', '13228166', '100', '-', '.', 'gene_id \"chr9.g38063\"; transcript_id \"blat.proc4005.chain_18632\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18835350', '18835568', '100', '+', '.', 'gene_id \"chr9.g38073\"; transcript_id \"blat.proc4005.chain_14563\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24303979', '24304215', '100', '+', '.', 'gene_id \"chr9.g38081\"; transcript_id \"blat.proc4005.chain_9087\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24442766', '24442922', '100', '+', '.', 'gene_id \"chr9.g38095\"; transcript_id \"blat.proc4005.chain_5090\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '6672129', '6672244', '100', '-', '.', 'gene_id \"chr9.g38097\"; transcript_id \"blat.proc18128.chain_26313\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24890754', '24890898', '100', '-', '.', 'gene_id \"chr9.g38101\"; transcript_id \"blat.proc4005.chain_14248\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15751821', '15751998', '100', '+', '.', 'gene_id \"chr9.g38106\"; transcript_id \"blat.proc4005.chain_14864\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27318044', '27318253', '100', '+', '.', 'gene_id \"chr9.g38109\"; transcript_id \"blat.proc4005.chain_15180\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43599413', '43599556', '100', '+', '.', 'gene_id \"chr9.g38148\"; transcript_id \"blat.proc4005.chain_3239\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13277886', '13278063', '100', '-', '.', 'gene_id \"chr9.g38151\"; transcript_id \"blat.proc4005.chain_15885\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19813973', '19814145', '100', '+', '.', 'gene_id \"chr9.g38162\"; transcript_id \"blat.proc4005.chain_704\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22564884', '22565048', '100', '-', '.', 'gene_id \"chr9.g38166\"; transcript_id \"blat.proc4005.chain_14332\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22563577', '22563717', '100', '+', '.', 'gene_id \"chr9.g38180\"; transcript_id \"blat.proc4005.chain_2570\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26549848', '26550044', '100', '-', '.', 'gene_id \"chr9.g38191\"; transcript_id \"blat.proc4005.chain_15038\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19030857', '19031059', '100', '+', '.', 'gene_id \"chr9.g38195\"; transcript_id \"blat.proc4005.chain_14597\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24695501', '24695682', '100', '+', '.', 'gene_id \"chr9.g38212\"; transcript_id \"blat.proc4005.chain_14216\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34045205', '34045323', '100', '+', '.', 'gene_id \"chr9.g38216\"; transcript_id \"blat.proc18128.chain_6669\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20415120', '20415308', '100', '+', '.', 'gene_id \"chr9.g38219\"; transcript_id \"blat.proc4005.chain_5115\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13610677', '13610743', '100', '+', '.', 'gene_id \"chr9.g38259\"; transcript_id \"blat.proc4005.chain_23872\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20172908', '20173037', '100', '-', '.', 'gene_id \"chr9.g38270\"; transcript_id \"blat.proc4005.chain_3971\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11616095', '11616176', '100', '-', '.', 'gene_id \"chr9.g38279\"; transcript_id \"blat.proc18128.chain_7141\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18171600', '18171766', '100', '-', '.', 'gene_id \"chr9.g38284\"; transcript_id \"blat.proc4005.chain_14379\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19595650', '19595835', '100', '+', '.', 'gene_id \"chr9.g38299\"; transcript_id \"blat.proc4005.chain_14763\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25895802', '25895997', '100', '-', '.', 'gene_id \"chr9.g38307\"; transcript_id \"blat.proc4005.chain_15070\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17028337', '17028517', '100', '-', '.', 'gene_id \"chr9.g38309\"; transcript_id \"blat.proc4005.chain_9616\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13420988', '13421179', '100', '-', '.', 'gene_id \"chr9.g38314\"; transcript_id \"blat.proc4005.chain_19927\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23443472', '23443630', '100', '+', '.', 'gene_id \"chr9.g38327\"; transcript_id \"blat.proc4005.chain_14391\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19776758', '19776803', '100', '-', '.', 'gene_id \"chr9.g38333\"; transcript_id \"blat.proc4587.chain_12245\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10812674', '10812913', '100', '+', '.', 'gene_id \"chr9.g38338\"; transcript_id \"blat.proc4005.chain_15997\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12704209', '12704367', '100', '+', '.', 'gene_id \"chr9.g38340\"; transcript_id \"blat.proc4005.chain_17830\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13204537', '13204701', '100', '+', '.', 'gene_id \"chr9.g38359\"; transcript_id \"blat.proc4005.chain_22876\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8558608', '8558758', '100', '-', '.', 'gene_id \"chr9.g38373\"; transcript_id \"blat.proc4005.chain_16360\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14204840', '14205032', '100', '+', '.', 'gene_id \"chr9.g38394\"; transcript_id \"blat.proc4005.chain_16140\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13506106', '13506479', '100', '+', '.', 'gene_id \"chr9.g38402\"; transcript_id \"blat.proc4005.chain_16056\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9093493', '9093596', '100', '+', '.', 'gene_id \"chr9.g38409\"; transcript_id \"blat.proc4005.chain_15594\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26456389', '26456555', '100', '-', '.', 'gene_id \"chr9.g38410\"; transcript_id \"blat.proc4005.chain_15086\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21198023', '21198127', '100', '-', '.', 'gene_id \"chr9.g38426\"; transcript_id \"blat.proc4005.chain_15258\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23466026', '23466139', '100', '-', '.', 'gene_id \"chr9.g38445\"; transcript_id \"blat.proc4005.chain_11110\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14488497', '14488953', '100', '+', '.', 'gene_id \"chr9.g38504\"; transcript_id \"blat.proc4005.chain_15450\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26590709', '26590866', '100', '-', '.', 'gene_id \"chr9.g38516\"; transcript_id \"blat.proc4005.chain_10956\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12627111', '12627348', '100', '+', '.', 'gene_id \"chr9.g38547\"; transcript_id \"blat.proc4005.chain_16023\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20841067', '20841247', '100', '+', '.', 'gene_id \"chr9.g38563\"; transcript_id \"blat.proc4005.chain_15209\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22649778', '22650098', '100', '-', '.', 'gene_id \"chr9.g38565\"; transcript_id \"blat.proc4005.chain_14338\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36644795', '36644901', '100', '+', '.', 'gene_id \"chr9.g38568\"; transcript_id \"blat.proc4005.chain_1973\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '54737505', '54737561', '100', '+', '.', 'gene_id \"chr9.g38582\"; transcript_id \"blat.proc28144.chain_7882\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25360669', '25360772', '100', '+', '.', 'gene_id \"chr9.g38583\"; transcript_id \"blat.proc4005.chain_63\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14938088', '14938143', '100', '-', '.', 'gene_id \"chr9.g38618\"; transcript_id \"blat.proc4005.chain_21954\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57072800', '57072832', '100', '+', '.', 'gene_id \"chr9.g38643\"; transcript_id \"blat.proc28144.chain_3219\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13008532', '13008745', '100', '-', '.', 'gene_id \"chr9.g38647\"; transcript_id \"blat.proc4005.chain_17024\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '43855633', '43855685', '100', '+', '.', 'gene_id \"chr9.g38668\"; transcript_id \"blat.proc4587.chain_10566\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17081137', '17081528', '100', '-', '.', 'gene_id \"chr9.g38671\"; transcript_id \"blat.proc4005.chain_5161\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10318630', '10318919', '100', '-', '.', 'gene_id \"chr9.g38672\"; transcript_id \"blat.proc4005.chain_18949\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26150129', '26150297', '100', '-', '.', 'gene_id \"chr9.g38693\"; transcript_id \"blat.proc4005.chain_15026\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20920969', '20921141', '100', '-', '.', 'gene_id \"chr9.g38699\"; transcript_id \"blat.proc4005.chain_14671\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12584700', '12584901', '100', '+', '.', 'gene_id \"chr9.g38713\"; transcript_id \"blat.proc4005.chain_16039\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18631293', '18631355', '100', '+', '.', 'gene_id \"chr9.g38714\"; transcript_id \"blat.proc4005.chain_5506\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15558571', '15558737', '100', '+', '.', 'gene_id \"chr9.g38731\"; transcript_id \"blat.proc18128.chain_7080\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23587363', '23587614', '100', '+', '.', 'gene_id \"chr9.g38736\"; transcript_id \"blat.proc4005.chain_9428\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42020521', '42020671', '100', '+', '.', 'gene_id \"chr9.g38763\"; transcript_id \"blat.proc18128.chain_26800\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18092032', '18092803', '100', '-', '.', 'gene_id \"chr9.g38765\"; transcript_id \"blat.proc4005.chain_13770\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10204138', '10204372', '100', '+', '.', 'gene_id \"chr9.g38795\"; transcript_id \"blat.proc4005.chain_15606\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11676753', '11677025', '100', '-', '.', 'gene_id \"chr9.g38809\"; transcript_id \"blat.proc4005.chain_17419\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18171071', '18171415', '100', '+', '.', 'gene_id \"chr9.g38836\"; transcript_id \"blat.proc4005.chain_14629\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '47010922', '47010987', '100', '+', '.', 'gene_id \"chr9.g38840\"; transcript_id \"blat.proc4005.chain_5735\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22320465', '22320631', '100', '-', '.', 'gene_id \"chr9.g38858\"; transcript_id \"blat.proc4005.chain_14313\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16339086', '16339253', '100', '-', '.', 'gene_id \"chr9.g38861\"; transcript_id \"blat.proc4005.chain_14296\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '28663336', '28663382', '100', '+', '.', 'gene_id \"chr9.g38877\"; transcript_id \"blat.proc18128.chain_5611\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19418536', '19418627', '100', '+', '.', 'gene_id \"chr9.g38900\"; transcript_id \"blat.proc4005.chain_4466\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13793630', '13793681', '100', '+', '.', 'gene_id \"chr9.g38905\"; transcript_id \"blat.proc28144.chain_22014\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18781359', '18781577', '100', '-', '.', 'gene_id \"chr9.g38910\"; transcript_id \"blat.proc4005.chain_14479\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25984797', '25985053', '100', '+', '.', 'gene_id \"chr9.g38912\"; transcript_id \"blat.proc4005.chain_13768\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27787031', '27787115', '100', '-', '.', 'gene_id \"chr9.g38925\"; transcript_id \"blat.proc4005.chain_12764\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15744229', '15744526', '100', '-', '.', 'gene_id \"chr9.g38937\"; transcript_id \"blat.proc4005.chain_14859\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13591642', '13591894', '100', '-', '.', 'gene_id \"chr9.g38947\"; transcript_id \"blat.proc4005.chain_16045\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14508840', '14509017', '100', '+', '.', 'gene_id \"chr9.g38950\"; transcript_id \"blat.proc4005.chain_15850\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23466217', '23466276', '100', '-', '.', 'gene_id \"chr9.g38957\"; transcript_id \"blat.proc4005.chain_11111\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46236689', '46236728', '100', '-', '.', 'gene_id \"chr9.g38971\"; transcript_id \"blat.proc28144.chain_5591\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19720652', '19720836', '100', '+', '.', 'gene_id \"chr9.g38976\"; transcript_id \"blat.proc4005.chain_15051\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10934896', '10935045', '100', '-', '.', 'gene_id \"chr9.g38977\"; transcript_id \"blat.proc4005.chain_17349\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18515427', '18515710', '100', '+', '.', 'gene_id \"chr9.g38993\"; transcript_id \"blat.proc4005.chain_3781\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22391828', '22392300', '100', '-', '.', 'gene_id \"chr9.g38998\"; transcript_id \"blat.proc4005.chain_9599\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21008737', '21008778', '100', '-', '.', 'gene_id \"chr9.g39004\"; transcript_id \"blat.proc18128.chain_25052\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26639362', '26639394', '100', '-', '.', 'gene_id \"chr9.g39020\"; transcript_id \"blat.proc28144.chain_22964\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12674026', '12674133', '100', '+', '.', 'gene_id \"chr9.g39029\"; transcript_id \"blat.proc4005.chain_23422\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11963336', '11963497', '100', '-', '.', 'gene_id \"chr9.g39038\"; transcript_id \"blat.proc4005.chain_15818\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24308066', '24308146', '100', '-', '.', 'gene_id \"chr9.g39042\"; transcript_id \"blat.proc4005.chain_7959\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9938211', '9938530', '100', '+', '.', 'gene_id \"chr9.g39080\"; transcript_id \"blat.proc4005.chain_16102\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24178978', '24179013', '100', '-', '.', 'gene_id \"chr9.g39081\"; transcript_id \"blat.proc4587.chain_5078\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18851790', '18851930', '100', '+', '.', 'gene_id \"chr9.g39095\"; transcript_id \"blat.proc4005.chain_452\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13226342', '13226773', '100', '+', '.', 'gene_id \"chr9.g39098\"; transcript_id \"blat.proc4005.chain_18005\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10617849', '10618125', '100', '+', '.', 'gene_id \"chr9.g39103\"; transcript_id \"blat.proc4005.chain_16109\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13762779', '13762845', '100', '+', '.', 'gene_id \"chr9.g39108\"; transcript_id \"blat.proc4005.chain_15660\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41445593', '41445664', '100', '-', '.', 'gene_id \"chr9.g39113\"; transcript_id \"blat.proc4587.chain_6586\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8239651', '8239943', '100', '-', '.', 'gene_id \"chr9.g39134\"; transcript_id \"blat.proc4587.chain_9281\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25112286', '25112340', '100', '+', '.', 'gene_id \"chr9.g39143\"; transcript_id \"blat.proc4587.chain_3391\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46560280', '46560330', '100', '+', '.', 'gene_id \"chr9.g39159\"; transcript_id \"blat.proc18128.chain_11756\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10604736', '10604894', '100', '+', '.', 'gene_id \"chr9.g39160\"; transcript_id \"blat.proc4005.chain_21163\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11352967', '11353166', '100', '-', '.', 'gene_id \"chr9.g39163\"; transcript_id \"blat.proc4005.chain_17759\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26423097', '26423291', '100', '-', '.', 'gene_id \"chr9.g39165\"; transcript_id \"blat.proc4005.chain_1614\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21575303', '21575461', '100', '-', '.', 'gene_id \"chr9.g39166\"; transcript_id \"blat.proc4005.chain_4796\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23392992', '23393029', '100', '-', '.', 'gene_id \"chr9.g39167\"; transcript_id \"blat.proc4005.chain_14580\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16621384', '16621428', '100', '-', '.', 'gene_id \"chr9.g39173\"; transcript_id \"blat.proc4005.chain_15058\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3578377', '3578410', '100', '+', '.', 'gene_id \"chr9.g39188\"; transcript_id \"blat.proc4005.chain_5999\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3428590', '3428625', '100', '-', '.', 'gene_id \"chr9.g39196\"; transcript_id \"blat.proc4587.chain_1631\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23806543', '23806718', '100', '-', '.', 'gene_id \"chr9.g39215\"; transcript_id \"blat.proc4005.chain_872\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22071358', '22071441', '100', '-', '.', 'gene_id \"chr9.g39218\"; transcript_id \"blat.proc4005.chain_11804\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26380372', '26380544', '100', '-', '.', 'gene_id \"chr9.g39231\"; transcript_id \"blat.proc4005.chain_14693\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18694833', '18695002', '100', '-', '.', 'gene_id \"chr9.g39256\"; transcript_id \"blat.proc4005.chain_15167\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21076136', '21076282', '100', '+', '.', 'gene_id \"chr9.g39266\"; transcript_id \"blat.proc4005.chain_6073\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15619770', '15619930', '100', '-', '.', 'gene_id \"chr9.g39275\"; transcript_id \"blat.proc4005.chain_15138\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9942354', '9942606', '100', '+', '.', 'gene_id \"chr9.g39321\"; transcript_id \"blat.proc4005.chain_15526\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14035608', '14035745', '100', '-', '.', 'gene_id \"chr9.g39336\"; transcript_id \"blat.proc4005.chain_17597\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35526185', '35526221', '100', '+', '.', 'gene_id \"chr9.g39355\"; transcript_id \"blat.proc28144.chain_989\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42623843', '42623909', '100', '+', '.', 'gene_id \"chr9.g39363\"; transcript_id \"blat.proc4005.chain_19653\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13843665', '13843746', '100', '-', '.', 'gene_id \"chr9.g39378\"; transcript_id \"blat.proc28144.chain_22441\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22789444', '22789604', '100', '-', '.', 'gene_id \"chr9.g39385\"; transcript_id \"blat.proc4005.chain_15206\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23977808', '23978521', '100', '-', '.', 'gene_id \"chr9.g39388\"; transcript_id \"blat.proc4005.chain_4086\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23454301', '23454422', '100', '-', '.', 'gene_id \"chr9.g39400\"; transcript_id \"blat.proc4005.chain_14273\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23654240', '23654523', '100', '-', '.', 'gene_id \"chr9.g39419\"; transcript_id \"blat.proc4005.chain_14651\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21979563', '21979656', '100', '+', '.', 'gene_id \"chr9.g39421\"; transcript_id \"blat.proc4005.chain_13868\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7786704', '7786867', '100', '+', '.', 'gene_id \"chr9.g39427\"; transcript_id \"blat.proc4005.chain_15735\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35910323', '35910365', '100', '+', '.', 'gene_id \"chr9.g39434\"; transcript_id \"blat.proc18128.chain_22521\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22574841', '22575035', '100', '+', '.', 'gene_id \"chr9.g39490\"; transcript_id \"blat.proc4005.chain_14544\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23345597', '23345927', '100', '-', '.', 'gene_id \"chr9.g39494\"; transcript_id \"blat.proc4005.chain_3072\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26294443', '26294586', '100', '-', '.', 'gene_id \"chr9.g39496\"; transcript_id \"blat.proc4005.chain_10280\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7989965', '7990149', '100', '-', '.', 'gene_id \"chr9.g39500\"; transcript_id \"blat.proc4005.chain_16241\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12570259', '12570428', '100', '-', '.', 'gene_id \"chr9.g39511\"; transcript_id \"blat.proc4005.chain_15504\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8077215', '8077430', '100', '+', '.', 'gene_id \"chr9.g39520\"; transcript_id \"blat.proc4005.chain_17455\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20090735', '20091606', '100', '+', '.', 'gene_id \"chr9.g39540\"; transcript_id \"blat.proc4005.chain_7481\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12278110', '12278276', '100', '+', '.', 'gene_id \"chr9.g39551\"; transcript_id \"blat.proc4005.chain_15944\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32515882', '32515980', '100', '-', '.', 'gene_id \"chr9.g39557\"; transcript_id \"blat.proc4587.chain_5009\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11211505', '11211664', '100', '+', '.', 'gene_id \"chr9.g39559\"; transcript_id \"blat.proc4005.chain_16464\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21568721', '21568987', '100', '+', '.', 'gene_id \"chr9.g39566\"; transcript_id \"blat.proc4005.chain_4832\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23976349', '23976524', '100', '-', '.', 'gene_id \"chr9.g39583\"; transcript_id \"blat.proc4005.chain_14173\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15854270', '15854427', '100', '+', '.', 'gene_id \"chr9.g39585\"; transcript_id \"blat.proc4005.chain_14560\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18190444', '18190594', '100', '-', '.', 'gene_id \"chr9.g39610\"; transcript_id \"blat.proc4005.chain_14199\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20167761', '20167969', '100', '+', '.', 'gene_id \"chr9.g39612\"; transcript_id \"blat.proc4005.chain_6867\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '28184848', '28184882', '100', '-', '.', 'gene_id \"chr9.g39617\"; transcript_id \"blat.proc4587.chain_6082\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22612794', '22612998', '100', '-', '.', 'gene_id \"chr9.g39656\"; transcript_id \"blat.proc4005.chain_4980\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19439282', '19439436', '100', '-', '.', 'gene_id \"chr9.g39658\"; transcript_id \"blat.proc4005.chain_7141\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21947872', '21947974', '100', '+', '.', 'gene_id \"chr9.g39679\"; transcript_id \"blat.proc4005.chain_4011\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15546092', '15546404', '100', '+', '.', 'gene_id \"chr9.g39733\"; transcript_id \"blat.proc4005.chain_6841\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10603229', '10603358', '100', '-', '.', 'gene_id \"chr9.g39737\"; transcript_id \"blat.proc4005.chain_18443\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18966478', '18966671', '100', '+', '.', 'gene_id \"chr9.g39755\"; transcript_id \"blat.proc4005.chain_4979\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15180764', '15181039', '100', '-', '.', 'gene_id \"chr9.g39791\"; transcript_id \"blat.proc4005.chain_18805\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13599779', '13599911', '100', '+', '.', 'gene_id \"chr9.g39795\"; transcript_id \"blat.proc4005.chain_19522\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12525339', '12525492', '100', '-', '.', 'gene_id \"chr9.g39797\"; transcript_id \"blat.proc4005.chain_15923\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18538953', '18539313', '100', '-', '.', 'gene_id \"chr9.g39804\"; transcript_id \"blat.proc4005.chain_13954\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27409100', '27409179', '100', '+', '.', 'gene_id \"chr9.g39816\"; transcript_id \"blat.proc4587.chain_5201\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14452984', '14453107', '100', '-', '.', 'gene_id \"chr9.g39819\"; transcript_id \"blat.proc4005.chain_20297\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22434816', '22435022', '100', '-', '.', 'gene_id \"chr9.g39822\"; transcript_id \"blat.proc4005.chain_14857\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20332537', '20332576', '100', '+', '.', 'gene_id \"chr9.g39829\"; transcript_id \"blat.proc4005.chain_6922\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1785030', '1785059', '100', '+', '.', 'gene_id \"chr9.g39859\"; transcript_id \"blat.proc18128.chain_24166\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23828813', '23829031', '100', '-', '.', 'gene_id \"chr9.g39882\"; transcript_id \"blat.proc4005.chain_6197\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24700787', '24700892', '100', '-', '.', 'gene_id \"chr9.g39895\"; transcript_id \"blat.proc4005.chain_4496\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '38214733', '38214774', '100', '+', '.', 'gene_id \"chr9.g39913\"; transcript_id \"blat.proc28144.chain_4936\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23214619', '23214992', '100', '-', '.', 'gene_id \"chr9.g39920\"; transcript_id \"blat.proc4005.chain_10039\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51258364', '51258404', '100', '-', '.', 'gene_id \"chr9.g39936\"; transcript_id \"blat.proc28144.chain_4501\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20830924', '20831117', '100', '+', '.', 'gene_id \"chr9.g39944\"; transcript_id \"blat.proc4005.chain_14593\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16827086', '16827276', '100', '+', '.', 'gene_id \"chr9.g39947\"; transcript_id \"blat.proc4005.chain_15200\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12348538', '12348837', '100', '-', '.', 'gene_id \"chr9.g39951\"; transcript_id \"blat.proc4005.chain_18178\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16059037', '16059320', '100', '+', '.', 'gene_id \"chr9.g39964\"; transcript_id \"blat.proc4005.chain_15037\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr9', 'PASA', 'exon', '31443215', '31443247', '100', '-', '.', 'gene_id \"chr9.g39968\"; transcript_id \"blat.proc4587.chain_10031\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23033512', '23033732', '100', '+', '.', 'gene_id \"chr9.g39977\"; transcript_id \"blat.proc4005.chain_1370\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42205780', '42205944', '100', '-', '.', 'gene_id \"chr9.g39980\"; transcript_id \"blat.proc4587.chain_6588\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9999548', '9999788', '100', '-', '.', 'gene_id \"chr9.g39993\"; transcript_id \"blat.proc4005.chain_17607\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57471274', '57471320', '100', '+', '.', 'gene_id \"chr9.g39995\"; transcript_id \"blat.proc4005.chain_22479\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8211885', '8212077', '100', '-', '.', 'gene_id \"chr9.g39997\"; transcript_id \"blat.proc4005.chain_16123\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13252733', '13252900', '100', '+', '.', 'gene_id \"chr9.g40005\"; transcript_id \"blat.proc4005.chain_15797\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17081540', '17081690', '100', '-', '.', 'gene_id \"chr9.g40009\"; transcript_id \"blat.proc4005.chain_5181\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14483103', '14483261', '100', '-', '.', 'gene_id \"chr9.g40010\"; transcript_id \"blat.proc4005.chain_16320\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7986484', '7986663', '100', '-', '.', 'gene_id \"chr9.g40017\"; transcript_id \"blat.proc4005.chain_16147\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12881583', '12881758', '100', '+', '.', 'gene_id \"chr9.g40019\"; transcript_id \"blat.proc4005.chain_21091\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8558256', '8558527', '100', '-', '.', 'gene_id \"chr9.g40020\"; transcript_id \"blat.proc4005.chain_16361\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15620041', '15620296', '100', '-', '.', 'gene_id \"chr9.g40037\"; transcript_id \"blat.proc4005.chain_14781\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23981546', '23981713', '100', '+', '.', 'gene_id \"chr9.g40040\"; transcript_id \"blat.proc4005.chain_10238\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23028084', '23028738', '100', '-', '.', 'gene_id \"chr9.g40045\"; transcript_id \"blat.proc4005.chain_4616\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '4427793', '4427885', '100', '-', '.', 'gene_id \"chr9.g40080\"; transcript_id \"blat.proc18128.chain_14853\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19919007', '19919376', '100', '+', '.', 'gene_id \"chr9.g40125\"; transcript_id \"blat.proc4005.chain_13895\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13205620', '13205777', '100', '+', '.', 'gene_id \"chr9.g40140\"; transcript_id \"blat.proc4005.chain_16087\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15613297', '15613459', '100', '+', '.', 'gene_id \"chr9.g40146\"; transcript_id \"blat.proc4005.chain_15114\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '30792810', '30792843', '100', '+', '.', 'gene_id \"chr9.g40147\"; transcript_id \"blat.proc28144.chain_1177\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14988021', '14988393', '100', '+', '.', 'gene_id \"chr9.g40150\"; transcript_id \"blat.proc4005.chain_15491\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19049850', '19049957', '100', '+', '.', 'gene_id \"chr9.g40152\"; transcript_id \"blat.proc4005.chain_11868\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '58570818', '58570847', '100', '+', '.', 'gene_id \"chr9.g40172\"; transcript_id \"blat.proc18128.chain_885\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11071012', '11071231', '100', '-', '.', 'gene_id \"chr9.g40190\"; transcript_id \"blat.proc4005.chain_20059\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7999238', '7999418', '100', '-', '.', 'gene_id \"chr9.g40201\"; transcript_id \"blat.proc4005.chain_15793\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16260992', '16261039', '100', '-', '.', 'gene_id \"chr9.g40206\"; transcript_id \"blat.proc4005.chain_21265\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18122980', '18123214', '100', '+', '.', 'gene_id \"chr9.g40214\"; transcript_id \"blat.proc4005.chain_2841\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '46431489', '46431522', '100', '-', '.', 'gene_id \"chr9.g40233\"; transcript_id \"blat.proc4005.chain_23301\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17902029', '17902184', '100', '-', '.', 'gene_id \"chr9.g40246\"; transcript_id \"blat.proc4005.chain_15330\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22557930', '22558188', '100', '+', '.', 'gene_id \"chr9.g40247\"; transcript_id \"blat.proc4005.chain_15045\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26302677', '26303018', '100', '+', '.', 'gene_id \"chr9.g40263\"; transcript_id \"blat.proc4005.chain_827\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19709141', '19709301', '100', '-', '.', 'gene_id \"chr9.g40289\"; transcript_id \"blat.proc4005.chain_14886\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18655624', '18655899', '100', '-', '.', 'gene_id \"chr9.g40294\"; transcript_id \"blat.proc4005.chain_14238\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24319184', '24319345', '100', '+', '.', 'gene_id \"chr9.g40305\"; transcript_id \"blat.proc4005.chain_3763\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12659575', '12659780', '100', '+', '.', 'gene_id \"chr9.g40319\"; transcript_id \"blat.proc4005.chain_15900\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '39803743', '39803786', '100', '-', '.', 'gene_id \"chr9.g40321\"; transcript_id \"blat.proc28144.chain_21217\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12525507', '12525661', '100', '-', '.', 'gene_id \"chr9.g40328\"; transcript_id \"blat.proc4005.chain_15924\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '52429802', '52429843', '100', '+', '.', 'gene_id \"chr9.g40338\"; transcript_id \"blat.proc4005.chain_16844\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9996280', '9996440', '100', '-', '.', 'gene_id \"chr9.g40341\"; transcript_id \"blat.proc4005.chain_16137\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17050562', '17051057', '100', '+', '.', 'gene_id \"chr9.g40342\"; transcript_id \"blat.proc4005.chain_2144\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23833754', '23834122', '100', '+', '.', 'gene_id \"chr9.g40344\"; transcript_id \"blat.proc4005.chain_389\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12005951', '12006230', '100', '+', '.', 'gene_id \"chr9.g40346\"; transcript_id \"blat.proc4005.chain_16605\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26558119', '26558285', '100', '-', '.', 'gene_id \"chr9.g40349\"; transcript_id \"blat.proc4005.chain_2481\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8461199', '8461244', '100', '-', '.', 'gene_id \"chr9.g40355\"; transcript_id \"blat.proc4005.chain_22843\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23481660', '23481949', '100', '+', '.', 'gene_id \"chr9.g40364\"; transcript_id \"blat.proc4005.chain_14858\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14518309', '14518469', '100', '-', '.', 'gene_id \"chr9.g40374\"; transcript_id \"blat.proc4005.chain_17233\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15046080', '15046263', '100', '-', '.', 'gene_id \"chr9.g40379\"; transcript_id \"blat.proc4005.chain_16328\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12350198', '12350254', '100', '-', '.', 'gene_id \"chr9.g40385\"; transcript_id \"blat.proc4005.chain_15614\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20334636', '20334851', '100', '+', '.', 'gene_id \"chr9.g40390\"; transcript_id \"blat.proc4005.chain_14798\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9896551', '9896785', '100', '-', '.', 'gene_id \"chr9.g40410\"; transcript_id \"blat.proc4005.chain_15787\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20275721', '20276001', '100', '+', '.', 'gene_id \"chr9.g40413\"; transcript_id \"blat.proc4005.chain_13907\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15599364', '15599735', '100', '-', '.', 'gene_id \"chr9.g40432\"; transcript_id \"blat.proc4005.chain_408\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25663521', '25663653', '100', '+', '.', 'gene_id \"chr9.g40436\"; transcript_id \"blat.proc4005.chain_3222\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23907644', '23907945', '100', '+', '.', 'gene_id \"chr9.g40447\"; transcript_id \"blat.proc4005.chain_1056\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12714930', '12715119', '100', '+', '.', 'gene_id \"chr9.g40481\"; transcript_id \"blat.proc4005.chain_21788\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10311159', '10311327', '100', '-', '.', 'gene_id \"chr9.g40485\"; transcript_id \"blat.proc4005.chain_19745\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24931641', '24931680', '100', '+', '.', 'gene_id \"chr9.g40490\"; transcript_id \"blat.proc4005.chain_14195\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12426499', '12426717', '100', '+', '.', 'gene_id \"chr9.g40524\"; transcript_id \"blat.proc4005.chain_15717\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12630219', '12630374', '100', '-', '.', 'gene_id \"chr9.g40538\"; transcript_id \"blat.proc4005.chain_20061\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20862681', '20862873', '100', '+', '.', 'gene_id \"chr9.g40551\"; transcript_id \"blat.proc4005.chain_14591\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19693790', '19693942', '100', '-', '.', 'gene_id \"chr9.g40552\"; transcript_id \"blat.proc4005.chain_5164\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18142947', '18143042', '100', '-', '.', 'gene_id \"chr9.g40574\"; transcript_id \"blat.proc4005.chain_4427\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23048212', '23048429', '100', '+', '.', 'gene_id \"chr9.g40592\"; transcript_id \"blat.proc4005.chain_8536\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17970881', '17971032', '100', '-', '.', 'gene_id \"chr9.g40597\"; transcript_id \"blat.proc4005.chain_14824\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22593662', '22593813', '100', '+', '.', 'gene_id \"chr9.g40632\"; transcript_id \"blat.proc4005.chain_14536\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '29329167', '29329240', '100', '-', '.', 'gene_id \"chr9.g40638\"; transcript_id \"blat.proc4587.chain_3682\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11675540', '11675641', '100', '-', '.', 'gene_id \"chr9.g40648\"; transcript_id \"blat.proc4005.chain_20694\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12028699', '12028899', '100', '-', '.', 'gene_id \"chr9.g40689\"; transcript_id \"blat.proc4005.chain_16979\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21937803', '21938018', '100', '-', '.', 'gene_id \"chr9.g40707\"; transcript_id \"blat.proc4005.chain_184\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9922393', '9922538', '100', '-', '.', 'gene_id \"chr9.g40709\"; transcript_id \"blat.proc4005.chain_16677\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20762690', '20762939', '100', '-', '.', 'gene_id \"chr9.g40714\"; transcript_id \"blat.proc4005.chain_14985\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13436285', '13436408', '100', '-', '.', 'gene_id \"chr9.g40725\"; transcript_id \"blat.proc4005.chain_19662\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15175143', '15175293', '100', '-', '.', 'gene_id \"chr9.g40748\"; transcript_id \"blat.proc4005.chain_17115\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19648867', '19648919', '100', '-', '.', 'gene_id \"chr9.g40763\"; transcript_id \"blat.proc28144.chain_2167\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '59986296', '59986345', '100', '-', '.', 'gene_id \"chr9.g40764\"; transcript_id \"blat.proc18128.chain_17920\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17845952', '17846192', '100', '-', '.', 'gene_id \"chr9.g40784\"; transcript_id \"blat.proc4005.chain_1766\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9920147', '9920328', '100', '-', '.', 'gene_id \"chr9.g40792\"; transcript_id \"blat.proc4005.chain_16203\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21109599', '21109678', '100', '-', '.', 'gene_id \"chr9.g40807\"; transcript_id \"blat.proc4005.chain_8126\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11129843', '11130102', '100', '+', '.', 'gene_id \"chr9.g40812\"; transcript_id \"blat.proc4005.chain_15769\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23339650', '23339918', '100', '-', '.', 'gene_id \"chr9.g40817\"; transcript_id \"blat.proc4005.chain_3099\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16395973', '16396144', '100', '-', '.', 'gene_id \"chr9.g40830\"; transcript_id \"blat.proc4005.chain_14995\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14511208', '14511379', '100', '-', '.', 'gene_id \"chr9.g40837\"; transcript_id \"blat.proc4005.chain_16121\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25640548', '25640797', '100', '+', '.', 'gene_id \"chr9.g40838\"; transcript_id \"blat.proc4005.chain_3223\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22048833', '22049051', '100', '+', '.', 'gene_id \"chr9.g40839\"; transcript_id \"blat.proc4005.chain_14194\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24719258', '24719435', '100', '+', '.', 'gene_id \"chr9.g40842\"; transcript_id \"blat.proc4005.chain_10070\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26120224', '26120367', '100', '+', '.', 'gene_id \"chr9.g40857\"; transcript_id \"blat.proc4005.chain_14721\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20057505', '20057736', '100', '-', '.', 'gene_id \"chr9.g40863\"; transcript_id \"blat.proc4005.chain_14085\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22703140', '22703320', '100', '+', '.', 'gene_id \"chr9.g40864\"; transcript_id \"blat.proc4005.chain_8028\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13626963', '13627033', '100', '-', '.', 'gene_id \"chr9.g40866\"; transcript_id \"blat.proc4005.chain_23516\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26387482', '26387811', '100', '-', '.', 'gene_id \"chr9.g40867\"; transcript_id \"blat.proc4005.chain_13801\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18568137', '18568252', '100', '-', '.', 'gene_id \"chr9.g40870\"; transcript_id \"blat.proc4005.chain_1791\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23141884', '23141915', '100', '-', '.', 'gene_id \"chr9.g40893\"; transcript_id \"blat.proc28144.chain_23313\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20403488', '20403674', '100', '-', '.', 'gene_id \"chr9.g40909\"; transcript_id \"blat.proc4005.chain_5299\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9505837', '9505871', '100', '-', '.', 'gene_id \"chr9.g40918\"; transcript_id \"blat.proc28144.chain_7593\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10920437', '10920541', '100', '-', '.', 'gene_id \"chr9.g40935\"; transcript_id \"blat.proc4005.chain_16937\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20990581', '20990861', '100', '-', '.', 'gene_id \"chr9.g40949\"; transcript_id \"blat.proc4005.chain_14287\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15819665', '15819822', '100', '+', '.', 'gene_id \"chr9.g40964\"; transcript_id \"blat.proc4005.chain_15155\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '5196871', '5196910', '100', '-', '.', 'gene_id \"chr9.g40977\"; transcript_id \"blat.proc18128.chain_13684\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13691171', '13691330', '100', '+', '.', 'gene_id \"chr9.g40983\"; transcript_id \"blat.proc4005.chain_16179\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13397547', '13397710', '100', '+', '.', 'gene_id \"chr9.g40988\"; transcript_id \"blat.proc4005.chain_15979\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21988114', '21988375', '100', '-', '.', 'gene_id \"chr9.g41034\"; transcript_id \"blat.proc4005.chain_4608\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12944466', '12944652', '100', '+', '.', 'gene_id \"chr9.g41040\"; transcript_id \"blat.proc4005.chain_19054\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20051642', '20051794', '100', '-', '.', 'gene_id \"chr9.g41045\"; transcript_id \"blat.proc4005.chain_12119\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3859949', '3860023', '100', '+', '.', 'gene_id \"chr9.g41052\"; transcript_id \"blat.proc18128.chain_8663\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14014329', '14014493', '100', '-', '.', 'gene_id \"chr9.g41065\"; transcript_id \"blat.proc4005.chain_15957\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16064094', '16064299', '100', '+', '.', 'gene_id \"chr9.g41068\"; transcript_id \"blat.proc4005.chain_497\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20091687', '20091776', '100', '-', '.', 'gene_id \"chr9.g41114\"; transcript_id \"blat.proc4005.chain_5188\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8055216', '8055439', '100', '-', '.', 'gene_id \"chr9.g41121\"; transcript_id \"blat.proc4005.chain_16014\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12125195', '12125433', '100', '-', '.', 'gene_id \"chr9.g41124\"; transcript_id \"blat.proc4005.chain_15716\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22654735', '22654912', '100', '-', '.', 'gene_id \"chr9.g41126\"; transcript_id \"blat.proc4005.chain_15313\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17785811', '17785963', '100', '+', '.', 'gene_id \"chr9.g41137\"; transcript_id \"blat.proc4005.chain_14218\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18863276', '18863381', '100', '-', '.', 'gene_id \"chr9.g41150\"; transcript_id \"blat.proc4005.chain_1771\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19609310', '19609400', '100', '-', '.', 'gene_id \"chr9.g41152\"; transcript_id \"blat.proc4005.chain_1441\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20850270', '20850463', '100', '+', '.', 'gene_id \"chr9.g41154\"; transcript_id \"blat.proc4005.chain_14592\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22523187', '22523220', '100', '+', '.', 'gene_id \"chr9.g41163\"; transcript_id \"blat.proc4005.chain_2901\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11409651', '11409806', '100', '+', '.', 'gene_id \"chr9.g41174\"; transcript_id \"blat.proc18128.chain_20660\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20039393', '20039539', '100', '-', '.', 'gene_id \"chr9.g41180\"; transcript_id \"blat.proc4005.chain_14245\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18827188', '18829675', '100', '-', '.', 'gene_id \"chr9.g41192\"; transcript_id \"blat.proc4005.chain_5337\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19942704', '19942884', '100', '-', '.', 'gene_id \"chr9.g41199\"; transcript_id \"blat.proc4005.chain_1316\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11792514', '11792653', '100', '-', '.', 'gene_id \"chr9.g41219\"; transcript_id \"blat.proc4005.chain_16510\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19444144', '19444216', '100', '+', '.', 'gene_id \"chr9.g41221\"; transcript_id \"blat.proc4005.chain_9368\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20588239', '20588283', '100', '-', '.', 'gene_id \"chr9.g41226\"; transcript_id \"blat.proc4005.chain_7972\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22649234', '22649456', '100', '-', '.', 'gene_id \"chr9.g41227\"; transcript_id \"blat.proc4005.chain_5015\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10025993', '10026066', '100', '+', '.', 'gene_id \"chr9.g41234\"; transcript_id \"blat.proc4005.chain_15704\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11020507', '11020566', '100', '-', '.', 'gene_id \"chr9.g41273\"; transcript_id \"blat.proc4005.chain_23783\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19581172', '19581252', '100', '+', '.', 'gene_id \"chr9.g41277\"; transcript_id \"blat.proc4005.chain_6570\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17906526', '17906714', '100', '+', '.', 'gene_id \"chr9.g41288\"; transcript_id \"blat.proc4005.chain_14322\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11120822', '11121016', '100', '-', '.', 'gene_id \"chr9.g41292\"; transcript_id \"blat.proc4005.chain_15999\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17972695', '17972978', '100', '-', '.', 'gene_id \"chr9.g41294\"; transcript_id \"blat.proc4005.chain_14672\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24726675', '24726913', '100', '+', '.', 'gene_id \"chr9.g41297\"; transcript_id \"blat.proc4005.chain_15116\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10653501', '10653798', '100', '+', '.', 'gene_id \"chr9.g41306\"; transcript_id \"blat.proc4005.chain_15796\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24626471', '24626634', '100', '-', '.', 'gene_id \"chr9.g41312\"; transcript_id \"blat.proc4005.chain_14957\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22183301', '22183476', '100', '-', '.', 'gene_id \"chr9.g41313\"; transcript_id \"blat.proc4005.chain_15007\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23059082', '23059234', '100', '-', '.', 'gene_id \"chr9.g41337\"; transcript_id \"blat.proc4005.chain_3948\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17962638', '17962720', '100', '-', '.', 'gene_id \"chr9.g41346\"; transcript_id \"blat.proc4005.chain_14631\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10203840', '10204032', '100', '-', '.', 'gene_id \"chr9.g41368\"; transcript_id \"blat.proc4005.chain_15605\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9316003', '9316032', '100', '+', '.', 'gene_id \"chr9.g41372\"; transcript_id \"blat.proc28144.chain_4525\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '49687011', '49687121', '100', '+', '.', 'gene_id \"chr9.g41382\"; transcript_id \"blat.proc18128.chain_24165\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15069656', '15070872', '100', '-', '.', 'gene_id \"chr9.g41393\"; transcript_id \"blat.proc4005.chain_21941\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12159546', '12159783', '100', '+', '.', 'gene_id \"chr9.g41395\"; transcript_id \"blat.proc4005.chain_15464\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26462442', '26462644', '100', '-', '.', 'gene_id \"chr9.g41396\"; transcript_id \"blat.proc4005.chain_9536\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21914584', '21914749', '100', '-', '.', 'gene_id \"chr9.g41404\"; transcript_id \"blat.proc4005.chain_6084\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21109469', '21109540', '100', '+', '.', 'gene_id \"chr9.g41412\"; transcript_id \"blat.proc4005.chain_13872\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22572803', '22572995', '100', '+', '.', 'gene_id \"chr9.g41414\"; transcript_id \"blat.proc4005.chain_12044\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13278137', '13278361', '100', '-', '.', 'gene_id \"chr9.g41420\"; transcript_id \"blat.proc4005.chain_15884\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22344936', '22345100', '100', '+', '.', 'gene_id \"chr9.g41432\"; transcript_id \"blat.proc4005.chain_14242\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13027871', '13028102', '100', '+', '.', 'gene_id \"chr9.g41449\"; transcript_id \"blat.proc4005.chain_16265\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24262704', '24262785', '100', '+', '.', 'gene_id \"chr9.g41452\"; transcript_id \"blat.proc4005.chain_1438\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12721907', '12722081', '100', '+', '.', 'gene_id \"chr9.g41458\"; transcript_id \"blat.proc4005.chain_15387\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19450994', '19451096', '100', '-', '.', 'gene_id \"chr9.g41465\"; transcript_id \"blat.proc4005.chain_8422\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10954805', '10954975', '100', '+', '.', 'gene_id \"chr9.g41468\"; transcript_id \"blat.proc4005.chain_16375\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12537173', '12537341', '100', '+', '.', 'gene_id \"chr9.g41469\"; transcript_id \"blat.proc4005.chain_16139\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21895433', '21895594', '100', '-', '.', 'gene_id \"chr9.g41477\"; transcript_id \"blat.proc4005.chain_9290\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18983578', '18983793', '100', '+', '.', 'gene_id \"chr9.g41484\"; transcript_id \"blat.proc4005.chain_15143\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20514355', '20514574', '100', '+', '.', 'gene_id \"chr9.g41499\"; transcript_id \"blat.proc4005.chain_14123\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23046151', '23046256', '100', '-', '.', 'gene_id \"chr9.g41505\"; transcript_id \"blat.proc4005.chain_2255\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16550408', '16550577', '100', '-', '.', 'gene_id \"chr9.g41539\"; transcript_id \"blat.proc4005.chain_14457\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '35465816', '35465875', '100', '+', '.', 'gene_id \"chr9.g41544\"; transcript_id \"blat.proc28144.chain_15021\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13280101', '13280257', '100', '-', '.', 'gene_id \"chr9.g41546\"; transcript_id \"blat.proc4005.chain_16317\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34429665', '34429706', '100', '-', '.', 'gene_id \"chr9.g41575\"; transcript_id \"blat.proc28144.chain_19723\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20574304', '20574455', '100', '-', '.', 'gene_id \"chr9.g41585\"; transcript_id \"blat.proc4005.chain_6926\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15603266', '15603413', '100', '-', '.', 'gene_id \"chr9.g41587\"; transcript_id \"blat.proc4005.chain_12558\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10015230', '10015381', '100', '+', '.', 'gene_id \"chr9.g41597\"; transcript_id \"blat.proc4005.chain_15770\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13809252', '13809302', '100', '+', '.', 'gene_id \"chr9.g41606\"; transcript_id \"blat.proc28144.chain_11039\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17980971', '17981155', '100', '-', '.', 'gene_id \"chr9.g41612\"; transcript_id \"blat.proc4005.chain_6010\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20694628', '20694799', '100', '+', '.', 'gene_id \"chr9.g41616\"; transcript_id \"blat.proc4005.chain_14257\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20748175', '20748337', '100', '-', '.', 'gene_id \"chr9.g41631\"; transcript_id \"blat.proc4005.chain_13979\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11988542', '11988792', '100', '+', '.', 'gene_id \"chr9.g41634\"; transcript_id \"blat.proc4005.chain_16373\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7612246', '7612401', '100', '+', '.', 'gene_id \"chr9.g41635\"; transcript_id \"blat.proc4005.chain_18388\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11134962', '11135211', '100', '-', '.', 'gene_id \"chr9.g41639\"; transcript_id \"blat.proc4005.chain_15895\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23736646', '23736811', '100', '+', '.', 'gene_id \"chr9.g41656\"; transcript_id \"blat.proc4005.chain_14627\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12831597', '12831842', '100', '+', '.', 'gene_id \"chr9.g41661\"; transcript_id \"blat.proc4005.chain_15713\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26202530', '26202800', '100', '-', '.', 'gene_id \"chr9.g41671\"; transcript_id \"blat.proc4005.chain_2959\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13026449', '13026526', '100', '-', '.', 'gene_id \"chr9.g41674\"; transcript_id \"blat.proc4005.chain_20639\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10935085', '10935490', '100', '+', '.', 'gene_id \"chr9.g41676\"; transcript_id \"blat.proc4005.chain_17352\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16263075', '16263167', '100', '+', '.', 'gene_id \"chr9.g41687\"; transcript_id \"blat.proc4005.chain_2490\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20266602', '20266763', '100', '+', '.', 'gene_id \"chr9.g41696\"; transcript_id \"blat.proc4005.chain_13953\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13327606', '13327685', '100', '+', '.', 'gene_id \"chr9.g41698\"; transcript_id \"blat.proc4005.chain_20192\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19439550', '19439585', '100', '+', '.', 'gene_id \"chr9.g41723\"; transcript_id \"blat.proc4005.chain_7708\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21424078', '21424344', '100', '+', '.', 'gene_id \"chr9.g41753\"; transcript_id \"blat.proc4005.chain_15004\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17841394', '17841549', '100', '-', '.', 'gene_id \"chr9.g41765\"; transcript_id \"blat.proc4005.chain_10191\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10210279', '10210618', '100', '-', '.', 'gene_id \"chr9.g41775\"; transcript_id \"blat.proc4005.chain_16716\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26433477', '26433642', '100', '+', '.', 'gene_id \"chr9.g41786\"; transcript_id \"blat.proc4005.chain_14564\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13447476', '13447610', '100', '+', '.', 'gene_id \"chr9.g41788\"; transcript_id \"blat.proc4005.chain_16641\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9665290', '9665476', '100', '-', '.', 'gene_id \"chr9.g41802\"; transcript_id \"blat.proc4005.chain_16010\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41374967', '41375050', '100', '-', '.', 'gene_id \"chr9.g41829\"; transcript_id \"blat.proc18128.chain_18476\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19447124', '19447470', '100', '-', '.', 'gene_id \"chr9.g41838\"; transcript_id \"blat.proc4005.chain_14775\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25914722', '25914918', '100', '-', '.', 'gene_id \"chr9.g41841\"; transcript_id \"blat.proc4005.chain_11809\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14723933', '14723994', '100', '-', '.', 'gene_id \"chr9.g41887\"; transcript_id \"blat.proc4005.chain_20561\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10007407', '10007627', '100', '-', '.', 'gene_id \"chr9.g41888\"; transcript_id \"blat.proc4005.chain_17715\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13550501', '13550717', '100', '+', '.', 'gene_id \"chr9.g41889\"; transcript_id \"blat.proc4005.chain_15604\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22819931', '22820005', '100', '-', '.', 'gene_id \"chr9.g41890\"; transcript_id \"blat.proc4005.chain_5397\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21714012', '21714164', '100', '-', '.', 'gene_id \"chr9.g41901\"; transcript_id \"blat.proc4005.chain_7777\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24722326', '24722506', '100', '-', '.', 'gene_id \"chr9.g41913\"; transcript_id \"blat.proc4005.chain_5291\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12831335', '12831533', '100', '+', '.', 'gene_id \"chr9.g41918\"; transcript_id \"blat.proc4005.chain_15714\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12272597', '12272973', '100', '-', '.', 'gene_id \"chr9.g41924\"; transcript_id \"blat.proc4005.chain_17784\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19596497', '19596776', '100', '-', '.', 'gene_id \"chr9.g41926\"; transcript_id \"blat.proc4005.chain_14247\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26386435', '26386656', '100', '+', '.', 'gene_id \"chr9.g41927\"; transcript_id \"blat.proc4005.chain_965\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15731331', '15731593', '100', '-', '.', 'gene_id \"chr9.g41942\"; transcript_id \"blat.proc4005.chain_14013\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '42710433', '42710508', '100', '+', '.', 'gene_id \"chr9.g41945\"; transcript_id \"blat.proc18128.chain_7585\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11354054', '11354127', '100', '+', '.', 'gene_id \"chr9.g41953\"; transcript_id \"blat.proc4005.chain_15659\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13207825', '13208009', '100', '+', '.', 'gene_id \"chr9.g41963\"; transcript_id \"blat.proc4005.chain_18105\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26782814', '26782855', '100', '+', '.', 'gene_id \"chr9.g41968\"; transcript_id \"blat.proc18128.chain_17851\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23947010', '23947172', '100', '+', '.', 'gene_id \"chr9.g41969\"; transcript_id \"blat.proc4005.chain_14716\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12631381', '12631459', '100', '-', '.', 'gene_id \"chr9.g41979\"; transcript_id \"blat.proc4005.chain_19258\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15608253', '15608294', '100', '+', '.', 'gene_id \"chr9.g41982\"; transcript_id \"blat.proc4005.chain_2539\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13715238', '13715509', '100', '+', '.', 'gene_id \"chr9.g41985\"; transcript_id \"blat.proc4005.chain_15887\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '48516653', '48516682', '100', '+', '.', 'gene_id \"chr9.g41986\"; transcript_id \"blat.proc4587.chain_2034\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25005361', '25005518', '100', '+', '.', 'gene_id \"chr9.g41990\"; transcript_id \"blat.proc4005.chain_14669\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21340073', '21340258', '100', '+', '.', 'gene_id \"chr9.g41994\"; transcript_id \"blat.proc4005.chain_10176\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20583678', '20583775', '100', '-', '.', 'gene_id \"chr9.g41996\"; transcript_id \"blat.proc4005.chain_3293\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19302542', '19302718', '100', '+', '.', 'gene_id \"chr9.g42001\"; transcript_id \"blat.proc4005.chain_14148\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14411637', '14411791', '100', '-', '.', 'gene_id \"chr9.g42019\"; transcript_id \"blat.proc4005.chain_15344\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19808203', '19808360', '100', '+', '.', 'gene_id \"chr9.g42020\"; transcript_id \"blat.proc4005.chain_14078\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3975343', '3975411', '100', '-', '.', 'gene_id \"chr9.g42024\"; transcript_id \"blat.proc4587.chain_4838\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7701157', '7701307', '100', '+', '.', 'gene_id \"chr9.g42028\"; transcript_id \"blat.proc4005.chain_20445\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51928179', '51928228', '100', '-', '.', 'gene_id \"chr9.g42042\"; transcript_id \"blat.proc18128.chain_1944\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25291141', '25291213', '100', '-', '.', 'gene_id \"chr9.g42074\"; transcript_id \"blat.proc4005.chain_3668\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8239236', '8239431', '100', '-', '.', 'gene_id \"chr9.g42076\"; transcript_id \"blat.proc4587.chain_9279\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20610398', '20610556', '100', '-', '.', 'gene_id \"chr9.g42103\"; transcript_id \"blat.proc4005.chain_14947\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '33331770', '33331808', '100', '-', '.', 'gene_id \"chr9.g42104\"; transcript_id \"blat.proc4005.chain_2908\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '57301564', '57301632', '100', '-', '.', 'gene_id \"chr9.g42107\"; transcript_id \"blat.proc4587.chain_10310\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18527013', '18527087', '100', '-', '.', 'gene_id \"chr9.g42117\"; transcript_id \"blat.proc4005.chain_6625\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20197467', '20197635', '100', '+', '.', 'gene_id \"chr9.g42118\"; transcript_id \"blat.proc4005.chain_15293\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21984346', '21984516', '100', '-', '.', 'gene_id \"chr9.g42124\"; transcript_id \"blat.proc4005.chain_2565\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14751053', '14751277', '100', '+', '.', 'gene_id \"chr9.g42125\"; transcript_id \"blat.proc4005.chain_15721\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12221531', '12221918', '100', '-', '.', 'gene_id \"chr9.g42132\"; transcript_id \"blat.proc4005.chain_15920\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12595346', '12595551', '100', '-', '.', 'gene_id \"chr9.g42138\"; transcript_id \"blat.proc4005.chain_17011\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13853432', '13853561', '100', '-', '.', 'gene_id \"chr9.g42142\"; transcript_id \"blat.proc4005.chain_15456\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16244748', '16245224', '100', '-', '.', 'gene_id \"chr9.g42144\"; transcript_id \"blat.proc4005.chain_14064\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15830267', '15830402', '100', '+', '.', 'gene_id \"chr9.g42152\"; transcript_id \"blat.proc4005.chain_4983\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23157653', '23157718', '100', '+', '.', 'gene_id \"chr9.g42154\"; transcript_id \"blat.proc4005.chain_14960\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8182637', '8182819', '100', '+', '.', 'gene_id \"chr9.g42157\"; transcript_id \"blat.proc4005.chain_15985\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11325719', '11325780', '100', '-', '.', 'gene_id \"chr9.g42162\"; transcript_id \"blat.proc4005.chain_20209\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22755735', '22755990', '100', '-', '.', 'gene_id \"chr9.g42165\"; transcript_id \"blat.proc4005.chain_2668\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15819090', '15819274', '100', '-', '.', 'gene_id \"chr9.g42173\"; transcript_id \"blat.proc4005.chain_14812\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16532219', '16532289', '100', '+', '.', 'gene_id \"chr9.g42202\"; transcript_id \"blat.proc4005.chain_10647\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16240407', '16240512', '100', '-', '.', 'gene_id \"chr9.g42222\"; transcript_id \"blat.proc4005.chain_9374\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14507070', '14507304', '100', '+', '.', 'gene_id \"chr9.g42232\"; transcript_id \"blat.proc4005.chain_16348\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23494820', '23494996', '100', '-', '.', 'gene_id \"chr9.g42233\"; transcript_id \"blat.proc4005.chain_15264\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10935528', '10935685', '100', '+', '.', 'gene_id \"chr9.g42238\"; transcript_id \"blat.proc4005.chain_17358\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10740435', '10740467', '100', '-', '.', 'gene_id \"chr9.g42245\"; transcript_id \"blat.proc4587.chain_3122\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13509137', '13509219', '100', '+', '.', 'gene_id \"chr9.g42269\"; transcript_id \"blat.proc4005.chain_18502\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '37359298', '37359338', '100', '+', '.', 'gene_id \"chr9.g42270\"; transcript_id \"blat.proc18128.chain_7937\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7382835', '7383100', '100', '+', '.', 'gene_id \"chr9.g42294\"; transcript_id \"blat.proc4005.chain_16773\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10257940', '10258075', '100', '+', '.', 'gene_id \"chr9.g42340\"; transcript_id \"blat.proc4005.chain_23502\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20161717', '20161989', '100', '+', '.', 'gene_id \"chr9.g42345\"; transcript_id \"blat.proc4005.chain_13850\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9950828', '9951130', '100', '+', '.', 'gene_id \"chr9.g42368\"; transcript_id \"blat.proc4005.chain_20896\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8319696', '8319952', '100', '+', '.', 'gene_id \"chr9.g42371\"; transcript_id \"blat.proc4005.chain_15390\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '24708070', '24708330', '100', '+', '.', 'gene_id \"chr9.g42387\"; transcript_id \"blat.proc4005.chain_14582\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14391847', '14392057', '100', '-', '.', 'gene_id \"chr9.g42397\"; transcript_id \"blat.proc4005.chain_16318\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34114700', '34114761', '100', '+', '.', 'gene_id \"chr9.g42402\"; transcript_id \"blat.proc28144.chain_8835\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21319721', '21320399', '100', '-', '.', 'gene_id \"chr9.g42416\"; transcript_id \"blat.proc4005.chain_13964\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8248997', '8249044', '100', '+', '.', 'gene_id \"chr9.g42426\"; transcript_id \"blat.proc18128.chain_5575\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20382841', '20382997', '100', '-', '.', 'gene_id \"chr9.g42430\"; transcript_id \"blat.proc4005.chain_15281\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19617232', '19617337', '100', '-', '.', 'gene_id \"chr9.g42440\"; transcript_id \"blat.proc4005.chain_1680\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18153822', '18153974', '100', '-', '.', 'gene_id \"chr9.g42445\"; transcript_id \"blat.proc4005.chain_14624\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10068292', '10068658', '100', '+', '.', 'gene_id \"chr9.g42460\"; transcript_id \"blat.proc4005.chain_15837\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21430000', '21430155', '100', '+', '.', 'gene_id \"chr9.g42466\"; transcript_id \"blat.proc4005.chain_14219\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11455072', '11455141', '100', '-', '.', 'gene_id \"chr9.g42473\"; transcript_id \"blat.proc4005.chain_21069\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20512899', '20513151', '100', '-', '.', 'gene_id \"chr9.g42474\"; transcript_id \"blat.proc4005.chain_793\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '36240048', '36240081', '100', '-', '.', 'gene_id \"chr9.g42487\"; transcript_id \"blat.proc28144.chain_18815\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11522103', '11522213', '100', '-', '.', 'gene_id \"chr9.g42494\"; transcript_id \"blat.proc4005.chain_22494\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7755037', '7755117', '100', '+', '.', 'gene_id \"chr9.g42498\"; transcript_id \"blat.proc4005.chain_19059\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12424469', '12424623', '100', '-', '.', 'gene_id \"chr9.g42505\"; transcript_id \"blat.proc4005.chain_16581\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14278412', '14278631', '100', '-', '.', 'gene_id \"chr9.g42517\"; transcript_id \"blat.proc4005.chain_15829\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26292439', '26293027', '100', '+', '.', 'gene_id \"chr9.g42524\"; transcript_id \"blat.proc4005.chain_14069\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16224569', '16224762', '100', '+', '.', 'gene_id \"chr9.g42534\"; transcript_id \"blat.proc4005.chain_1566\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13221329', '13221559', '100', '+', '.', 'gene_id \"chr9.g42536\"; transcript_id \"blat.proc4005.chain_19939\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12192544', '12192708', '100', '+', '.', 'gene_id \"chr9.g42538\"; transcript_id \"blat.proc4005.chain_19020\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19527389', '19527496', '100', '+', '.', 'gene_id \"chr9.g42563\"; transcript_id \"blat.proc4005.chain_5949\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20491951', '20492132', '100', '+', '.', 'gene_id \"chr9.g42579\"; transcript_id \"blat.proc4005.chain_14991\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23025296', '23026338', '100', '+', '.', 'gene_id \"chr9.g42593\"; transcript_id \"blat.proc4005.chain_10566\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15180501', '15180762', '100', '-', '.', 'gene_id \"chr9.g42599\"; transcript_id \"blat.proc4005.chain_18807\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18222567', '18222641', '100', '+', '.', 'gene_id \"chr9.g42623\"; transcript_id \"blat.proc4005.chain_10983\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15602624', '15602783', '100', '+', '.', 'gene_id \"chr9.g42643\"; transcript_id \"blat.proc4005.chain_6109\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25025183', '25025378', '100', '-', '.', 'gene_id \"chr9.g42658\"; transcript_id \"blat.proc4005.chain_15159\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27114870', '27114949', '100', '+', '.', 'gene_id \"chr9.g42667\"; transcript_id \"blat.proc18128.chain_21117\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14528429', '14528604', '100', '-', '.', 'gene_id \"chr9.g42675\"; transcript_id \"blat.proc4005.chain_15998\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23709966', '23710130', '100', '-', '.', 'gene_id \"chr9.g42686\"; transcript_id \"blat.proc4005.chain_10223\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11214759', '11214910', '100', '-', '.', 'gene_id \"chr9.g42689\"; transcript_id \"blat.proc4005.chain_17720\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16487097', '16487248', '100', '-', '.', 'gene_id \"chr9.g42698\"; transcript_id \"blat.proc4005.chain_9611\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21938341', '21938547', '100', '-', '.', 'gene_id \"chr9.g42722\"; transcript_id \"blat.proc4005.chain_803\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22389029', '22389259', '100', '-', '.', 'gene_id \"chr9.g42725\"; transcript_id \"blat.proc4005.chain_15202\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7507100', '7507139', '100', '+', '.', 'gene_id \"chr9.g42739\"; transcript_id \"blat.proc28144.chain_23063\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13773777', '13773901', '100', '-', '.', 'gene_id \"chr9.g42755\"; transcript_id \"blat.proc4005.chain_23036\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23707111', '23707266', '100', '-', '.', 'gene_id \"chr9.g42758\"; transcript_id \"blat.proc4005.chain_14129\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '3677356', '3677390', '100', '+', '.', 'gene_id \"chr9.g42760\"; transcript_id \"blat.proc4587.chain_7528\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10085815', '10086007', '100', '-', '.', 'gene_id \"chr9.g42778\"; transcript_id \"blat.proc4005.chain_15626\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20070365', '20070427', '100', '+', '.', 'gene_id \"chr9.g42788\"; transcript_id \"blat.proc4005.chain_15111\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19746938', '19747304', '100', '+', '.', 'gene_id \"chr9.g42789\"; transcript_id \"blat.proc4005.chain_13730\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '33422966', '33423013', '100', '+', '.', 'gene_id \"chr9.g42807\"; transcript_id \"blat.proc18128.chain_7344\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20574491', '20574653', '100', '+', '.', 'gene_id \"chr9.g42809\"; transcript_id \"blat.proc4005.chain_6929\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11207800', '11207906', '100', '+', '.', 'gene_id \"chr9.g42815\"; transcript_id \"blat.proc4005.chain_16417\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10617685', '10617839', '100', '+', '.', 'gene_id \"chr9.g42818\"; transcript_id \"blat.proc4005.chain_16110\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '45409433', '45409466', '100', '-', '.', 'gene_id \"chr9.g42829\"; transcript_id \"blat.proc18128.chain_3290\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '51237704', '51237739', '100', '+', '.', 'gene_id \"chr9.g42833\"; transcript_id \"blat.proc18128.chain_1786\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16396275', '16396414', '100', '+', '.', 'gene_id \"chr9.g42852\"; transcript_id \"blat.proc4005.chain_6050\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11500943', '11501123', '100', '-', '.', 'gene_id \"chr9.g42860\"; transcript_id \"blat.proc4005.chain_17435\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18172860', '18173033', '100', '+', '.', 'gene_id \"chr9.g42868\"; transcript_id \"blat.proc4005.chain_15275\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11319056', '11319099', '100', '-', '.', 'gene_id \"chr9.g42877\"; transcript_id \"blat.proc4005.chain_15674\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26407451', '26407681', '100', '-', '.', 'gene_id \"chr9.g42884\"; transcript_id \"blat.proc4005.chain_14511\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13974467', '13974544', '100', '+', '.', 'gene_id \"chr9.g42950\"; transcript_id \"blat.proc4005.chain_19588\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22134040', '22134210', '100', '-', '.', 'gene_id \"chr9.g42970\"; transcript_id \"blat.proc4005.chain_15148\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19596938', '19597015', '100', '+', '.', 'gene_id \"chr9.g42993\"; transcript_id \"blat.proc4005.chain_2358\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20491813', '20491890', '100', '+', '.', 'gene_id \"chr9.g43003\"; transcript_id \"blat.proc4005.chain_4589\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15732493', '15732653', '100', '+', '.', 'gene_id \"chr9.g43006\"; transcript_id \"blat.proc4005.chain_14654\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9643370', '9643526', '100', '-', '.', 'gene_id \"chr9.g43025\"; transcript_id \"blat.proc4005.chain_16113\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12276634', '12276941', '100', '-', '.', 'gene_id \"chr9.g43029\"; transcript_id \"blat.proc4005.chain_16699\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27456272', '27456318', '100', '-', '.', 'gene_id \"chr9.g43065\"; transcript_id \"blat.proc28144.chain_3558\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18179413', '18179564', '100', '-', '.', 'gene_id \"chr9.g43107\"; transcript_id \"blat.proc4005.chain_14118\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13883680', '13883842', '100', '+', '.', 'gene_id \"chr9.g43124\"; transcript_id \"blat.proc4005.chain_16047\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8210146', '8210290', '100', '+', '.', 'gene_id \"chr9.g43125\"; transcript_id \"blat.proc4005.chain_16061\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16071624', '16071697', '100', '+', '.', 'gene_id \"chr9.g43137\"; transcript_id \"blat.proc4005.chain_11242\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23933399', '23933572', '100', '-', '.', 'gene_id \"chr9.g43170\"; transcript_id \"blat.proc4005.chain_8898\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15704067', '15704273', '100', '+', '.', 'gene_id \"chr9.g43190\"; transcript_id \"blat.proc4005.chain_15291\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '32660343', '32660386', '100', '-', '.', 'gene_id \"chr9.g43196\"; transcript_id \"blat.proc4587.chain_5636\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9942152', '9942318', '100', '-', '.', 'gene_id \"chr9.g43197\"; transcript_id \"blat.proc4005.chain_15525\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18570511', '18570689', '100', '-', '.', 'gene_id \"chr9.g43230\"; transcript_id \"blat.proc4005.chain_5549\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21324354', '21324437', '100', '+', '.', 'gene_id \"chr9.g43244\"; transcript_id \"blat.proc4005.chain_2876\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11545055', '11545274', '100', '-', '.', 'gene_id \"chr9.g43251\"; transcript_id \"blat.proc4005.chain_16784\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20058667', '20058819', '100', '-', '.', 'gene_id \"chr9.g43262\"; transcript_id \"blat.proc4005.chain_14495\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20071767', '20071838', '100', '+', '.', 'gene_id \"chr9.g43272\"; transcript_id \"blat.proc4005.chain_13254\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25041612', '25041755', '100', '+', '.', 'gene_id \"chr9.g43276\"; transcript_id \"blat.proc4005.chain_14339\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12049967', '12050135', '100', '+', '.', 'gene_id \"chr9.g43292\"; transcript_id \"blat.proc4005.chain_18941\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22784586', '22784751', '100', '+', '.', 'gene_id \"chr9.g43312\"; transcript_id \"blat.proc4005.chain_14263\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13590117', '13590149', '100', '-', '.', 'gene_id \"chr9.g43340\"; transcript_id \"blat.proc28144.chain_14287\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18022029', '18022114', '100', '+', '.', 'gene_id \"chr9.g43343\"; transcript_id \"blat.proc18128.chain_13344\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10925798', '10925971', '100', '-', '.', 'gene_id \"chr9.g43350\"; transcript_id \"blat.proc4005.chain_15625\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10967039', '10967219', '100', '+', '.', 'gene_id \"chr9.g43365\"; transcript_id \"blat.proc4005.chain_21537\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11364399', '11364482', '100', '+', '.', 'gene_id \"chr9.g43367\"; transcript_id \"blat.proc4005.chain_23205\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10094297', '10094506', '100', '+', '.', 'gene_id \"chr9.g43380\"; transcript_id \"blat.proc4005.chain_20798\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15708413', '15708525', '100', '+', '.', 'gene_id \"chr9.g43395\"; transcript_id \"blat.proc4005.chain_6024\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26438435', '26438597', '100', '+', '.', 'gene_id \"chr9.g43405\"; transcript_id \"blat.proc4005.chain_10543\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16547966', '16548103', '100', '+', '.', 'gene_id \"chr9.g43430\"; transcript_id \"blat.proc4005.chain_7983\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23841370', '23841674', '100', '+', '.', 'gene_id \"chr9.g43458\"; transcript_id \"blat.proc4005.chain_3470\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9692796', '9692948', '100', '-', '.', 'gene_id \"chr9.g43459\"; transcript_id \"blat.proc4005.chain_16470\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14347456', '14347609', '100', '-', '.', 'gene_id \"chr9.g43480\"; transcript_id \"blat.proc4005.chain_15357\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10013890', '10014119', '100', '-', '.', 'gene_id \"chr9.g43507\"; transcript_id \"blat.proc4005.chain_22611\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21773177', '21773329', '100', '-', '.', 'gene_id \"chr9.g43510\"; transcript_id \"blat.proc4005.chain_14933\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15096531', '15096601', '100', '+', '.', 'gene_id \"chr9.g43543\"; transcript_id \"blat.proc4005.chain_15370\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19879293', '19879462', '100', '+', '.', 'gene_id \"chr9.g43544\"; transcript_id \"blat.proc4005.chain_5491\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14776645', '14776823', '100', '+', '.', 'gene_id \"chr9.g43548\"; transcript_id \"blat.proc4005.chain_15791\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '55505269', '55505301', '100', '-', '.', 'gene_id \"chr9.g43559\"; transcript_id \"blat.proc28144.chain_12050\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12011775', '12011962', '100', '-', '.', 'gene_id \"chr9.g43570\"; transcript_id \"blat.proc4005.chain_16122\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18006589', '18006826', '100', '-', '.', 'gene_id \"chr9.g43575\"; transcript_id \"blat.proc4005.chain_15017\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8318076', '8318166', '100', '+', '.', 'gene_id \"chr9.g43597\"; transcript_id \"blat.proc4005.chain_20620\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '9132633', '9132776', '100', '+', '.', 'gene_id \"chr9.g43652\"; transcript_id \"blat.proc4005.chain_19811\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19041890', '19042166', '100', '+', '.', 'gene_id \"chr9.g43655\"; transcript_id \"blat.proc4005.chain_6275\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22928632', '22929069', '100', '+', '.', 'gene_id \"chr9.g43667\"; transcript_id \"blat.proc4005.chain_7285\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13990865', '13991355', '100', '-', '.', 'gene_id \"chr9.g43679\"; transcript_id \"blat.proc4005.chain_16980\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22040816', '22040968', '100', '-', '.', 'gene_id \"chr9.g43680\"; transcript_id \"blat.proc4005.chain_15084\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19769626', '19769897', '100', '-', '.', 'gene_id \"chr9.g43685\"; transcript_id \"blat.proc4005.chain_4376\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22535993', '22536314', '100', '-', '.', 'gene_id \"chr9.g43700\"; transcript_id \"blat.proc4005.chain_14381\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10646130', '10646193', '100', '-', '.', 'gene_id \"chr9.g43713\"; transcript_id \"blat.proc4005.chain_21324\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14629234', '14629396', '100', '+', '.', 'gene_id \"chr9.g43736\"; transcript_id \"blat.proc4005.chain_20524\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10120691', '10120858', '100', '-', '.', 'gene_id \"chr9.g43737\"; transcript_id \"blat.proc4005.chain_16315\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21892382', '21892548', '100', '-', '.', 'gene_id \"chr9.g43750\"; transcript_id \"blat.proc4005.chain_14192\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '17827613', '17827785', '100', '+', '.', 'gene_id \"chr9.g43762\"; transcript_id \"blat.proc4005.chain_11925\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21348573', '21348753', '100', '+', '.', 'gene_id \"chr9.g43764\"; transcript_id \"blat.proc4005.chain_12318\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10536766', '10536907', '100', '+', '.', 'gene_id \"chr9.g43801\"; transcript_id \"blat.proc4005.chain_20435\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14388237', '14388452', '100', '+', '.', 'gene_id \"chr9.g43806\"; transcript_id \"blat.proc4005.chain_20700\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23454158', '23454273', '100', '-', '.', 'gene_id \"chr9.g43818\"; transcript_id \"blat.proc4005.chain_3408\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8209915', '8210081', '100', '-', '.', 'gene_id \"chr9.g43822\"; transcript_id \"blat.proc4005.chain_16062\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25346171', '25346554', '100', '-', '.', 'gene_id \"chr9.g43831\"; transcript_id \"blat.proc4005.chain_11325\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22638047', '22638164', '100', '+', '.', 'gene_id \"chr9.g43842\"; transcript_id \"blat.proc4005.chain_7203\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13808974', '13809156', '100', '-', '.', 'gene_id \"chr9.g43850\"; transcript_id \"blat.proc4005.chain_20681\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13776648', '13776727', '100', '+', '.', 'gene_id \"chr9.g43865\"; transcript_id \"blat.proc4005.chain_19827\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13571425', '13571522', '100', '+', '.', 'gene_id \"chr9.g43870\"; transcript_id \"blat.proc4005.chain_22878\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18035327', '18035458', '100', '-', '.', 'gene_id \"chr9.g43874\"; transcript_id \"blat.proc4005.chain_14965\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8528885', '8529041', '100', '+', '.', 'gene_id \"chr9.g43875\"; transcript_id \"blat.proc4005.chain_16128\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22747329', '22747494', '100', '+', '.', 'gene_id \"chr9.g43878\"; transcript_id \"blat.proc4005.chain_14695\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23894846', '23894997', '100', '-', '.', 'gene_id \"chr9.g43888\"; transcript_id \"blat.proc4005.chain_15326\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26631759', '26631865', '100', '+', '.', 'gene_id \"chr9.g43907\"; transcript_id \"blat.proc4005.chain_13691\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13273317', '13273409', '100', '-', '.', 'gene_id \"chr9.g43921\"; transcript_id \"blat.proc4005.chain_16992\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15807099', '15807284', '100', '-', '.', 'gene_id \"chr9.g43933\"; transcript_id \"blat.proc4005.chain_14643\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26554601', '26554744', '100', '-', '.', 'gene_id \"chr9.g43950\"; transcript_id \"blat.proc4005.chain_7341\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11456313', '11456473', '100', '+', '.', 'gene_id \"chr9.g43957\"; transcript_id \"blat.proc4005.chain_21782\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10055173', '10055365', '100', '-', '.', 'gene_id \"chr9.g43967\"; transcript_id \"blat.proc4005.chain_15790\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19446323', '19446716', '100', '-', '.', 'gene_id \"chr9.g43972\"; transcript_id \"blat.proc4005.chain_14444\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21294747', '21294933', '100', '+', '.', 'gene_id \"chr9.g43981\"; transcript_id \"blat.proc4005.chain_15224\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '27450751', '27450875', '100', '+', '.', 'gene_id \"chr9.g43997\"; transcript_id \"blat.proc28144.chain_17974\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '30234073', '30234107', '100', '+', '.', 'gene_id \"chr9.g44013\"; transcript_id \"blat.proc4005.chain_18396\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26553233', '26553334', '100', '-', '.', 'gene_id \"chr9.g44018\"; transcript_id \"blat.proc4005.chain_1681\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26200355', '26200414', '100', '+', '.', 'gene_id \"chr9.g44022\"; transcript_id \"blat.proc4005.chain_4927\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18525503', '18525653', '100', '-', '.', 'gene_id \"chr9.g44026\"; transcript_id \"blat.proc4005.chain_14027\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14205118', '14205365', '100', '+', '.', 'gene_id \"chr9.g44029\"; transcript_id \"blat.proc4005.chain_16141\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13867929', '13868144', '100', '+', '.', 'gene_id \"chr9.g44036\"; transcript_id \"blat.proc4005.chain_16043\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11038102', '11038349', '100', '+', '.', 'gene_id \"chr9.g44052\"; transcript_id \"blat.proc4005.chain_23597\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23086173', '23086329', '100', '+', '.', 'gene_id \"chr9.g44073\"; transcript_id \"blat.proc4005.chain_15284\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '14351635', '14351825', '100', '+', '.', 'gene_id \"chr9.g44095\"; transcript_id \"blat.proc4005.chain_16885\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '8037746', '8037909', '100', '+', '.', 'gene_id \"chr9.g44113\"; transcript_id \"blat.proc4005.chain_18605\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19419146', '19419216', '100', '-', '.', 'gene_id \"chr9.g44142\"; transcript_id \"blat.proc4005.chain_14033\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '53493202', '53493480', '100', '+', '.', 'gene_id \"chr9.g44146\"; transcript_id \"blat.proc18128.chain_10351\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19497886', '19497940', '100', '-', '.', 'gene_id \"chr9.g44167\"; transcript_id \"blat.proc4005.chain_6854\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '23996324', '23996478', '100', '+', '.', 'gene_id \"chr9.g44177\"; transcript_id \"blat.proc4005.chain_14589\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19269469', '19269619', '100', '-', '.', 'gene_id \"chr9.g44179\"; transcript_id \"blat.proc4005.chain_15151\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12290271', '12290426', '100', '-', '.', 'gene_id \"chr9.g44180\"; transcript_id \"blat.proc4005.chain_18039\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '1818659', '1818808', '100', '+', '.', 'gene_id \"chr9.g44189\"; transcript_id \"blat.proc4587.chain_11370\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13254297', '13254645', '100', '+', '.', 'gene_id \"chr9.g44223\"; transcript_id \"blat.proc4005.chain_15993\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18619061', '18619235', '100', '-', '.', 'gene_id \"chr9.g44236\"; transcript_id \"blat.proc4005.chain_9863\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13908798', '13908987', '100', '-', '.', 'gene_id \"chr9.g44239\"; transcript_id \"blat.proc4005.chain_19689\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '15007395', '15007565', '100', '-', '.', 'gene_id \"chr9.g44263\"; transcript_id \"blat.proc4005.chain_22059\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '7474514', '7474605', '100', '-', '.', 'gene_id \"chr9.g44312\"; transcript_id \"blat.proc4005.chain_17903\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '11104541', '11105031', '100', '+', '.', 'gene_id \"chr9.g44319\"; transcript_id \"blat.proc4005.chain_23551\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25680443', '25680620', '100', '-', '.', 'gene_id \"chr9.g44321\"; transcript_id \"blat.proc4005.chain_783\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '16815723', '16815890', '100', '-', '.', 'gene_id \"chr9.g44332\"; transcript_id \"blat.proc4005.chain_3406\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22899229', '22899273', '100', '+', '.', 'gene_id \"chr9.g44334\"; transcript_id \"blat.proc28144.chain_10333\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25660998', '25661185', '100', '+', '.', 'gene_id \"chr9.g44349\"; transcript_id \"blat.proc4005.chain_3236\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20420983', '20421158', '100', '+', '.', 'gene_id \"chr9.g44362\"; transcript_id \"blat.proc4005.chain_13802\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20211321', '20211492', '100', '+', '.', 'gene_id \"chr9.g44396\"; transcript_id \"blat.proc4005.chain_5890\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20255884', '20256036', '100', '-', '.', 'gene_id \"chr9.g44399\"; transcript_id \"blat.proc4005.chain_11627\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19448675', '19448797', '100', '-', '.', 'gene_id \"chr9.g44411\"; transcript_id \"blat.proc4005.chain_10768\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19493765', '19493795', '100', '-', '.', 'gene_id \"chr9.g44412\"; transcript_id \"blat.proc28144.chain_3225\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13466941', '13467009', '100', '-', '.', 'gene_id \"chr9.g44418\"; transcript_id \"blat.proc28144.chain_2208\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '10016457', '10016657', '100', '+', '.', 'gene_id \"chr9.g44419\"; transcript_id \"blat.proc4005.chain_15846\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26135261', '26135427', '100', '+', '.', 'gene_id \"chr9.g44437\"; transcript_id \"blat.proc4005.chain_9431\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '41368703', '41368735', '100', '-', '.', 'gene_id \"chr9.g44447\"; transcript_id \"blat.proc28144.chain_18026\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '34073374', '34073424', '100', '-', '.', 'gene_id \"chr9.g44452\"; transcript_id \"blat.proc28144.chain_20183\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '22108922', '22110260', '100', '-', '.', 'gene_id \"chr9.g44466\"; transcript_id \"blat.proc4005.chain_3725\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19172104', '19172389', '100', '-', '.', 'gene_id \"chr9.g44489\"; transcript_id \"blat.proc4005.chain_14974\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '25427022', '25427175', '100', '+', '.', 'gene_id \"chr9.g44497\"; transcript_id \"blat.proc4005.chain_14802\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19593985', '19594200', '100', '-', '.', 'gene_id \"chr9.g44510\"; transcript_id \"blat.proc4005.chain_14951\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21922257', '21922327', '100', '+', '.', 'gene_id \"chr9.g44522\"; transcript_id \"blat.proc4005.chain_4786\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '20923631', '20923786', '100', '+', '.', 'gene_id \"chr9.g44531\"; transcript_id \"blat.proc4005.chain_15104\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '13846127', '13846441', '100', '-', '.', 'gene_id \"chr9.g44534\"; transcript_id \"blat.proc4005.chain_17716\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '12920933', '12921109', '100', '+', '.', 'gene_id \"chr9.g44543\"; transcript_id \"blat.proc4005.chain_16308\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '21651545', '21651693', '100', '+', '.', 'gene_id \"chr9.g44545\"; transcript_id \"blat.proc4005.chain_2039\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19958470', '19958669', '100', '+', '.', 'gene_id \"chr9.g44562\"; transcript_id \"blat.proc4005.chain_11496\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '18026561', '18026717', '100', '+', '.', 'gene_id \"chr9.g44575\"; transcript_id \"blat.proc4005.chain_15249\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '26433123', '26433335', '100', '-', '.', 'gene_id \"chr9.g44576\"; transcript_id \"blat.proc4005.chain_14653\";']\n",
      "negative:  ['chr9', 'PASA', 'exon', '19922538', '19922615', '100', '-', '.', 'gene_id \"chr9.g44578\"; transcript_id \"blat.proc4005.chain_9877\";']\n",
      "negative:  ['scaffold_1246', 'PASA', 'exon', '14706', '16224', '.', '-', '.', 'gene_id \"scaffold_1246.g15\"; transcript_id \"scaffold_1246.g15.i1\";']\n",
      "negative:  ['scaffold_1246', 'PASA', 'exon', '15322', '15393', '.', '-', '.', 'gene_id \"scaffold_1246.g15\"; transcript_id \"scaffold_1246.g15.i2\";']\n",
      "negative:  ['scaffold_1246', 'PASA', 'exon', '14538', '14706', '.', '-', '.', 'gene_id \"scaffold_1246.g16\"; transcript_id \"scaffold_1246.g16.i1\";']\n",
      "negative:  ['scaffold_1246', 'PASA', 'exon', '5285', '5777', '100', '-', '.', 'gene_id \"scaffold_1246.g24\"; transcript_id \"blat.proc4587.chain_3183\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '9503660', '9503979', '.', '+', '.', 'gene_id \"chr2.g3549\"; transcript_id \"chr2.g3549.i1\";']\n",
      "negative:  ['scaffold_1582', 'PASA', 'exon', '1816', '1856', '.', '+', '.', 'gene_id \"scaffold_1582.g4\"; transcript_id \"scaffold_1582.g4.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '9606969', '9607093', '.', '+', '.', 'gene_id \"chr2.g3578\"; transcript_id \"chr2.g3578.i2\";']\n",
      "negative:  ['scaffold_1238', 'PASA', 'exon', '10090', '10330', '100', '-', '.', 'gene_id \"scaffold_1238.g22\"; transcript_id \"blat.proc4587.chain_308\";']\n",
      "negative:  ['scaffold_1629', 'PASA', 'exon', '24135', '24209', '.', '+', '.', 'gene_id \"scaffold_1629.g3\"; transcript_id \"scaffold_1629.g3.i1\";']\n",
      "negative:  ['scaffold_1629', 'PASA', 'exon', '29382', '29641', '.', '+', '.', 'gene_id \"scaffold_1629.g6\"; transcript_id \"scaffold_1629.g6.i2\";']\n",
      "negative:  ['scaffold_1395', 'PASA', 'exon', '10831', '11017', '.', '+', '.', 'gene_id \"scaffold_1395.g2\"; transcript_id \"scaffold_1395.g2.i2\";']\n",
      "negative:  ['scaffold_1395', 'PASA', 'exon', '17533', '17641', '100', '+', '.', 'gene_id \"scaffold_1395.g9\"; transcript_id \"blat.proc28144.chain_9275\";']\n",
      "negative:  ['scaffold_1392', 'PASA', 'exon', '10553', '10894', '.', '-', '.', 'gene_id \"scaffold_1392.g13\"; transcript_id \"scaffold_1392.g13.i1\";']\n",
      "negative:  ['scaffold_1392', 'PASA', 'exon', '10319', '10573', '.', '+', '.', 'gene_id \"scaffold_1392.g14\"; transcript_id \"scaffold_1392.g14.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10529516', '10529678', '.', '-', '.', 'gene_id \"chr2.g3949\"; transcript_id \"chr2.g3949.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10560092', '10560281', '.', '+', '.', 'gene_id \"chr2.g3973\"; transcript_id \"chr2.g3973.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10560465', '10560698', '.', '+', '.', 'gene_id \"chr2.g3974\"; transcript_id \"chr2.g3974.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10580237', '10580549', '.', '+', '.', 'gene_id \"chr2.g4002\"; transcript_id \"chr2.g4002.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10859822', '10859862', '.', '-', '.', 'gene_id \"chr2.g4066\"; transcript_id \"chr2.g4066.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '10984688', '10984735', '.', '-', '.', 'gene_id \"chr2.g4079\"; transcript_id \"chr2.g4079.i1\";']\n",
      "negative:  ['scaffold_732', 'PASA', 'exon', '2937', '3094', '.', '-', '.', 'gene_id \"scaffold_732.g1\"; transcript_id \"scaffold_732.g1.i1\";']\n",
      "negative:  ['scaffold_732', 'PASA', 'exon', '2219', '2712', '100', '-', '.', 'gene_id \"scaffold_732.g11\"; transcript_id \"blat.proc4587.chain_3181\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '11189585', '11189626', '.', '+', '.', 'gene_id \"chr2.g4153\"; transcript_id \"chr2.g4153.i1\";']\n",
      "negative:  ['scaffold_871', 'PASA', 'exon', '4004', '5467', '.', '-', '.', 'gene_id \"scaffold_871.g2\"; transcript_id \"scaffold_871.g2.i1\";']\n",
      "negative:  ['scaffold_871', 'PASA', 'exon', '2494', '2533', '100', '+', '.', 'gene_id \"scaffold_871.g4\"; transcript_id \"blat.proc28144.chain_10057\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '11397013', '11397171', '.', '-', '.', 'gene_id \"chr2.g4194\"; transcript_id \"chr2.g4194.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '11406196', '11406232', '.', '+', '.', 'gene_id \"chr2.g4203\"; transcript_id \"chr2.g4203.i1\";']\n",
      "negative:  ['scaffold_1606', 'PASA', 'exon', '800', '845', '100', '+', '.', 'gene_id \"scaffold_1606.g10\"; transcript_id \"blat.proc4005.chain_23414\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '12293877', '12294332', '.', '+', '.', 'gene_id \"chr2.g4397\"; transcript_id \"chr2.g4397.i1\";']\n",
      "negative:  ['scaffold_1668', 'PASA', 'exon', '16733', '16898', '.', '+', '.', 'gene_id \"scaffold_1668.g2\"; transcript_id \"scaffold_1668.g2.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '13069475', '13069659', '.', '+', '.', 'gene_id \"chr2.g4745\"; transcript_id \"chr2.g4745.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '13233759', '13233804', '.', '-', '.', 'gene_id \"chr2.g4911\"; transcript_id \"chr2.g4911.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '13499926', '13500097', '.', '-', '.', 'gene_id \"chr2.g5060\"; transcript_id \"chr2.g5060.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '270159', '270308', '.', '+', '.', 'gene_id \"chr8.g104\"; transcript_id \"chr8.g104.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '270589', '270815', '.', '-', '.', 'gene_id \"chr8.g107\"; transcript_id \"chr8.g107.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '550093', '550181', '.', '+', '.', 'gene_id \"chr8.g135\"; transcript_id \"chr8.g135.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '617014', '617241', '.', '-', '.', 'gene_id \"chr8.g142\"; transcript_id \"chr8.g142.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1117898', '1117927', '.', '+', '.', 'gene_id \"chr8.g233\"; transcript_id \"chr8.g233.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1219029', '1219069', '.', '-', '.', 'gene_id \"chr8.g254\"; transcript_id \"chr8.g254.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1218887', '1218937', '.', '-', '.', 'gene_id \"chr8.g254\"; transcript_id \"chr8.g254.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1336931', '1337154', '.', '-', '.', 'gene_id \"chr8.g402\"; transcript_id \"chr8.g402.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1423899', '1424068', '.', '+', '.', 'gene_id \"chr8.g445\"; transcript_id \"chr8.g445.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '14199623', '14201767', '.', '-', '.', 'gene_id \"chr2.g5400\"; transcript_id \"chr2.g5400.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1494423', '1494626', '.', '-', '.', 'gene_id \"chr8.g453\"; transcript_id \"chr8.g453.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1557463', '1557641', '.', '+', '.', 'gene_id \"chr8.g457\"; transcript_id \"chr8.g457.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1619165', '1619318', '.', '-', '.', 'gene_id \"chr8.g472\"; transcript_id \"chr8.g472.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1619788', '1620016', '.', '-', '.', 'gene_id \"chr8.g473\"; transcript_id \"chr8.g473.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1619996', '1620430', '.', '+', '.', 'gene_id \"chr8.g474\"; transcript_id \"chr8.g474.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '14364403', '14364547', '.', '+', '.', 'gene_id \"chr2.g5483\"; transcript_id \"chr2.g5483.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '14469273', '14469524', '.', '+', '.', 'gene_id \"chr2.g5538\"; transcript_id \"chr2.g5538.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1824121', '1825395', '.', '-', '.', 'gene_id \"chr8.g640\"; transcript_id \"chr8.g640.i10\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1978341', '1978428', '.', '-', '.', 'gene_id \"chr8.g804\"; transcript_id \"chr8.g804.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1978342', '1978428', '.', '-', '.', 'gene_id \"chr8.g804\"; transcript_id \"chr8.g804.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2093446', '2093650', '.', '+', '.', 'gene_id \"chr8.g885\"; transcript_id \"chr8.g885.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2093772', '2094022', '.', '-', '.', 'gene_id \"chr8.g886\"; transcript_id \"chr8.g886.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2217291', '2217324', '.', '-', '.', 'gene_id \"chr8.g923\"; transcript_id \"chr8.g923.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2251555', '2253634', '.', '-', '.', 'gene_id \"chr8.g932\"; transcript_id \"chr8.g932.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2251555', '2253614', '.', '-', '.', 'gene_id \"chr8.g932\"; transcript_id \"chr8.g932.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2412228', '2412309', '.', '+', '.', 'gene_id \"chr8.g960\"; transcript_id \"chr8.g960.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2412233', '2412309', '.', '+', '.', 'gene_id \"chr8.g960\"; transcript_id \"chr8.g960.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2431839', '2432001', '.', '+', '.', 'gene_id \"chr8.g968\"; transcript_id \"chr8.g968.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2466251', '2466293', '.', '-', '.', 'gene_id \"chr8.g980\"; transcript_id \"chr8.g980.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2716181', '2716460', '.', '+', '.', 'gene_id \"chr8.g1084\"; transcript_id \"chr8.g1084.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2716481', '2716699', '.', '+', '.', 'gene_id \"chr8.g1085\"; transcript_id \"chr8.g1085.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '16056022', '16056254', '.', '+', '.', 'gene_id \"chr2.g6083\"; transcript_id \"chr2.g6083.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2766700', '2766749', '.', '-', '.', 'gene_id \"chr8.g1138\"; transcript_id \"chr8.g1138.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2789278', '2789478', '.', '+', '.', 'gene_id \"chr8.g1173\"; transcript_id \"chr8.g1173.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '2902140', '2902177', '.', '-', '.', 'gene_id \"chr8.g1208\"; transcript_id \"chr8.g1208.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '16347482', '16347941', '.', '-', '.', 'gene_id \"chr2.g6262\"; transcript_id \"chr2.g6262.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3103440', '3103609', '.', '-', '.', 'gene_id \"chr8.g1411\"; transcript_id \"chr8.g1411.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3206862', '3207345', '.', '-', '.', 'gene_id \"chr8.g1468\"; transcript_id \"chr8.g1468.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3207576', '3207847', '.', '+', '.', 'gene_id \"chr8.g1469\"; transcript_id \"chr8.g1469.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3305621', '3305652', '.', '-', '.', 'gene_id \"chr8.g1579\"; transcript_id \"chr8.g1579.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '16803937', '16803979', '.', '+', '.', 'gene_id \"chr2.g6492\"; transcript_id \"chr2.g6492.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3427686', '3427889', '.', '-', '.', 'gene_id \"chr8.g1654\"; transcript_id \"chr8.g1654.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '17246617', '17246923', '.', '-', '.', 'gene_id \"chr2.g6630\"; transcript_id \"chr2.g6630.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '17573890', '17574299', '.', '+', '.', 'gene_id \"chr2.g6816\"; transcript_id \"chr2.g6816.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4199651', '4199824', '.', '-', '.', 'gene_id \"chr8.g2143\"; transcript_id \"chr8.g2143.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4199313', '4199533', '.', '+', '.', 'gene_id \"chr8.g2144\"; transcript_id \"chr8.g2144.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4218930', '4218979', '.', '+', '.', 'gene_id \"chr8.g2155\"; transcript_id \"chr8.g2155.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '17791111', '17791561', '.', '-', '.', 'gene_id \"chr2.g6988\"; transcript_id \"chr2.g6988.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4551468', '4551517', '.', '+', '.', 'gene_id \"chr8.g2256\"; transcript_id \"chr8.g2256.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '18276303', '18276335', '.', '-', '.', 'gene_id \"chr2.g7113\"; transcript_id \"chr2.g7113.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4933000', '4933288', '.', '-', '.', 'gene_id \"chr8.g2378\"; transcript_id \"chr8.g2378.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5126649', '5126694', '.', '-', '.', 'gene_id \"chr8.g2401\"; transcript_id \"chr8.g2401.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5150496', '5150538', '.', '-', '.', 'gene_id \"chr8.g2413\"; transcript_id \"chr8.g2413.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5618437', '5618596', '.', '+', '.', 'gene_id \"chr8.g2453\"; transcript_id \"chr8.g2453.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5618538', '5619020', '.', '+', '.', 'gene_id \"chr8.g2453\"; transcript_id \"chr8.g2453.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5618738', '5618965', '.', '+', '.', 'gene_id \"chr8.g2453\"; transcript_id \"chr8.g2453.i3\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr8', 'PASA', 'exon', '6101165', '6101198', '.', '-', '.', 'gene_id \"chr8.g2555\"; transcript_id \"chr8.g2555.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6424171', '6424252', '.', '+', '.', 'gene_id \"chr8.g2580\"; transcript_id \"chr8.g2580.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6488682', '6488713', '.', '+', '.', 'gene_id \"chr8.g2584\"; transcript_id \"chr8.g2584.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6702721', '6702767', '.', '-', '.', 'gene_id \"chr8.g2601\"; transcript_id \"chr8.g2601.i6\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '18991123', '18991303', '.', '+', '.', 'gene_id \"chr2.g7352\"; transcript_id \"chr2.g7352.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6742781', '6743074', '.', '-', '.', 'gene_id \"chr8.g2647\"; transcript_id \"chr8.g2647.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6981305', '6981335', '.', '+', '.', 'gene_id \"chr8.g2663\"; transcript_id \"chr8.g2663.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6981305', '6981335', '.', '-', '.', 'gene_id \"chr8.g2664\"; transcript_id \"chr8.g2664.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '7069700', '7069756', '.', '-', '.', 'gene_id \"chr8.g2673\"; transcript_id \"chr8.g2673.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '7153512', '7153691', '.', '-', '.', 'gene_id \"chr8.g2683\"; transcript_id \"chr8.g2683.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '7605185', '7605243', '.', '-', '.', 'gene_id \"chr8.g2719\"; transcript_id \"chr8.g2719.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8032375', '8032427', '.', '-', '.', 'gene_id \"chr8.g2758\"; transcript_id \"chr8.g2758.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8054876', '8055017', '.', '+', '.', 'gene_id \"chr8.g2772\"; transcript_id \"chr8.g2772.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '19291598', '19291631', '.', '-', '.', 'gene_id \"chr2.g7561\"; transcript_id \"chr2.g7561.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8510229', '8510483', '.', '+', '.', 'gene_id \"chr8.g2851\"; transcript_id \"chr8.g2851.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8611772', '8611819', '.', '-', '.', 'gene_id \"chr8.g2857\"; transcript_id \"chr8.g2857.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8833923', '8834146', '.', '-', '.', 'gene_id \"chr8.g2871\"; transcript_id \"chr8.g2871.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8834165', '8835121', '.', '+', '.', 'gene_id \"chr8.g2878\"; transcript_id \"chr8.g2878.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8831316', '8831466', '.', '-', '.', 'gene_id \"chr8.g2905\"; transcript_id \"chr8.g2905.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9327153', '9327182', '.', '+', '.', 'gene_id \"chr8.g2989\"; transcript_id \"chr8.g2989.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9732895', '9733110', '.', '+', '.', 'gene_id \"chr8.g3053\"; transcript_id \"chr8.g3053.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9788536', '9788799', '.', '+', '.', 'gene_id \"chr8.g3095\"; transcript_id \"chr8.g3095.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9788836', '9789184', '.', '+', '.', 'gene_id \"chr8.g3096\"; transcript_id \"chr8.g3096.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9931675', '9931840', '.', '+', '.', 'gene_id \"chr8.g3160\"; transcript_id \"chr8.g3160.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '20340719', '20340913', '.', '-', '.', 'gene_id \"chr2.g8021\"; transcript_id \"chr2.g8021.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10200711', '10200911', '.', '-', '.', 'gene_id \"chr8.g3421\"; transcript_id \"chr8.g3421.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10200916', '10201087', '.', '-', '.', 'gene_id \"chr8.g3422\"; transcript_id \"chr8.g3422.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10256667', '10257609', '.', '-', '.', 'gene_id \"chr8.g3446\"; transcript_id \"chr8.g3446.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10293400', '10293429', '.', '-', '.', 'gene_id \"chr8.g3470\"; transcript_id \"chr8.g3470.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10356756', '10356842', '.', '-', '.', 'gene_id \"chr8.g3513\"; transcript_id \"chr8.g3513.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10433729', '10433886', '.', '-', '.', 'gene_id \"chr8.g3579\"; transcript_id \"chr8.g3579.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10541772', '10542049', '.', '-', '.', 'gene_id \"chr8.g3620\"; transcript_id \"chr8.g3620.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10807781', '10807838', '.', '+', '.', 'gene_id \"chr8.g3659\"; transcript_id \"chr8.g3659.i6\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10698983', '10699173', '.', '-', '.', 'gene_id \"chr8.g3692\"; transcript_id \"chr8.g3692.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10765592', '10765720', '.', '+', '.', 'gene_id \"chr8.g3715\"; transcript_id \"chr8.g3715.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10833959', '10834004', '.', '+', '.', 'gene_id \"chr8.g3757\"; transcript_id \"chr8.g3757.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10895203', '10895241', '.', '+', '.', 'gene_id \"chr8.g3769\"; transcript_id \"chr8.g3769.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10988147', '10988312', '.', '-', '.', 'gene_id \"chr8.g3795\"; transcript_id \"chr8.g3795.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '11026579', '11026753', '.', '-', '.', 'gene_id \"chr8.g3831\"; transcript_id \"chr8.g3831.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '11071444', '11071779', '.', '-', '.', 'gene_id \"chr8.g3867\"; transcript_id \"chr8.g3867.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '21733130', '21733197', '.', '-', '.', 'gene_id \"chr2.g8683\"; transcript_id \"chr2.g8683.i5\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '11715747', '11715800', '.', '-', '.', 'gene_id \"chr8.g3972\"; transcript_id \"chr8.g3972.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '11750125', '11750158', '.', '-', '.', 'gene_id \"chr8.g3974\"; transcript_id \"chr8.g3974.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '12233667', '12233972', '.', '+', '.', 'gene_id \"chr8.g4023\"; transcript_id \"chr8.g4023.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '21823383', '21823556', '.', '+', '.', 'gene_id \"chr2.g8821\"; transcript_id \"chr2.g8821.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '12668755', '12668964', '.', '+', '.', 'gene_id \"chr8.g4098\"; transcript_id \"chr8.g4098.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '12715811', '12715844', '.', '-', '.', 'gene_id \"chr8.g4129\"; transcript_id \"chr8.g4129.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '21984517', '21984673', '.', '-', '.', 'gene_id \"chr2.g8926\"; transcript_id \"chr2.g8926.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '21986079', '21986948', '.', '-', '.', 'gene_id \"chr2.g8930\"; transcript_id \"chr2.g8930.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '22022776', '22022985', '.', '-', '.', 'gene_id \"chr2.g8963\"; transcript_id \"chr2.g8963.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '22043070', '22043232', '.', '+', '.', 'gene_id \"chr2.g8997\"; transcript_id \"chr2.g8997.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13050700', '13050996', '.', '+', '.', 'gene_id \"chr8.g4243\"; transcript_id \"chr8.g4243.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13072519', '13072678', '.', '-', '.', 'gene_id \"chr8.g4255\"; transcript_id \"chr8.g4255.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13097201', '13097375', '.', '-', '.', 'gene_id \"chr8.g4267\"; transcript_id \"chr8.g4267.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '22090344', '22090395', '.', '+', '.', 'gene_id \"chr2.g9028\"; transcript_id \"chr2.g9028.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13151849', '13152057', '.', '+', '.', 'gene_id \"chr8.g4303\"; transcript_id \"chr8.g4303.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '22122340', '22122379', '.', '+', '.', 'gene_id \"chr2.g9072\"; transcript_id \"chr2.g9072.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13404627', '13404666', '.', '+', '.', 'gene_id \"chr8.g4471\"; transcript_id \"chr8.g4471.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '22563237', '22563459', '.', '-', '.', 'gene_id \"chr2.g9243\"; transcript_id \"chr2.g9243.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13633818', '13634083', '.', '+', '.', 'gene_id \"chr8.g4675\"; transcript_id \"chr8.g4675.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13773336', '13773505', '.', '+', '.', 'gene_id \"chr8.g4793\"; transcript_id \"chr8.g4793.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13775540', '13775755', '.', '-', '.', 'gene_id \"chr8.g4797\"; transcript_id \"chr8.g4797.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13836564', '13837110', '.', '-', '.', 'gene_id \"chr8.g4814\"; transcript_id \"chr8.g4814.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13845536', '13845978', '.', '-', '.', 'gene_id \"chr8.g4816\"; transcript_id \"chr8.g4816.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13844092', '13845547', '.', '+', '.', 'gene_id \"chr8.g4817\"; transcript_id \"chr8.g4817.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13831128', '13831285', '.', '-', '.', 'gene_id \"chr8.g4820\"; transcript_id \"chr8.g4820.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13831465', '13832545', '.', '-', '.', 'gene_id \"chr8.g4822\"; transcript_id \"chr8.g4822.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13869023', '13869071', '.', '+', '.', 'gene_id \"chr8.g4859\"; transcript_id \"chr8.g4859.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14351298', '14351442', '.', '+', '.', 'gene_id \"chr8.g5070\"; transcript_id \"chr8.g5070.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14450681', '14450730', '.', '-', '.', 'gene_id \"chr8.g5111\"; transcript_id \"chr8.g5111.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14512887', '14512928', '.', '-', '.', 'gene_id \"chr8.g5125\"; transcript_id \"chr8.g5125.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '25558359', '25558628', '.', '-', '.', 'gene_id \"chr2.g9822\"; transcript_id \"chr2.g9822.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14939712', '14939742', '.', '+', '.', 'gene_id \"chr8.g5260\"; transcript_id \"chr8.g5260.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15517439', '15517780', '.', '+', '.', 'gene_id \"chr8.g5544\"; transcript_id \"chr8.g5544.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15517771', '15517929', '.', '-', '.', 'gene_id \"chr8.g5545\"; transcript_id \"chr8.g5545.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15654295', '15654628', '.', '+', '.', 'gene_id \"chr8.g5611\"; transcript_id \"chr8.g5611.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15654679', '15654893', '.', '+', '.', 'gene_id \"chr8.g5612\"; transcript_id \"chr8.g5612.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15763005', '15763046', '.', '-', '.', 'gene_id \"chr8.g5663\"; transcript_id \"chr8.g5663.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15892509', '15893378', '.', '-', '.', 'gene_id \"chr8.g5725\"; transcript_id \"chr8.g5725.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15893750', '15893799', '.', '-', '.', 'gene_id \"chr8.g5730\"; transcript_id \"chr8.g5730.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15894883', '15895607', '.', '+', '.', 'gene_id \"chr8.g5733\"; transcript_id \"chr8.g5733.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27218553', '27218905', '.', '-', '.', 'gene_id \"chr2.g10442\"; transcript_id \"chr2.g10442.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27221722', '27222099', '.', '-', '.', 'gene_id \"chr2.g10444\"; transcript_id \"chr2.g10444.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27226779', '27227306', '.', '-', '.', 'gene_id \"chr2.g10450\"; transcript_id \"chr2.g10450.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27226537', '27226809', '.', '+', '.', 'gene_id \"chr2.g10451\"; transcript_id \"chr2.g10451.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27221316', '27221449', '.', '+', '.', 'gene_id \"chr2.g10453\"; transcript_id \"chr2.g10453.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27262424', '27262907', '.', '+', '.', 'gene_id \"chr2.g10503\"; transcript_id \"chr2.g10503.i4\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27270173', '27270349', '.', '+', '.', 'gene_id \"chr2.g10514\"; transcript_id \"chr2.g10514.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27279369', '27279529', '.', '+', '.', 'gene_id \"chr2.g10522\"; transcript_id \"chr2.g10522.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '16401382', '16401587', '.', '+', '.', 'gene_id \"chr8.g5843\"; transcript_id \"chr8.g5843.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27307139', '27307626', '.', '-', '.', 'gene_id \"chr2.g10550\"; transcript_id \"chr2.g10550.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '16924811', '16924847', '.', '+', '.', 'gene_id \"chr8.g5925\"; transcript_id \"chr8.g5925.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17038363', '17038395', '.', '-', '.', 'gene_id \"chr8.g5948\"; transcript_id \"chr8.g5948.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17275339', '17275553', '.', '+', '.', 'gene_id \"chr8.g6111\"; transcript_id \"chr8.g6111.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17275543', '17275697', '.', '-', '.', 'gene_id \"chr8.g6123\"; transcript_id \"chr8.g6123.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17274636', '17274839', '.', '-', '.', 'gene_id \"chr8.g6126\"; transcript_id \"chr8.g6126.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17303837', '17305242', '.', '+', '.', 'gene_id \"chr8.g6203\"; transcript_id \"chr8.g6203.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17374746', '17374902', '.', '+', '.', 'gene_id \"chr8.g6270\"; transcript_id \"chr8.g6270.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17487357', '17487557', '.', '+', '.', 'gene_id \"chr8.g6301\"; transcript_id \"chr8.g6301.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '17487357', '17487557', '.', '+', '.', 'gene_id \"chr8.g6301\"; transcript_id \"chr8.g6301.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '29254422', '29254623', '.', '-', '.', 'gene_id \"chr2.g11212\"; transcript_id \"chr2.g11212.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19361981', '19362258', '.', '-', '.', 'gene_id \"chr8.g6516\"; transcript_id \"chr8.g6516.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '29363685', '29363835', '.', '+', '.', 'gene_id \"chr2.g11238\"; transcript_id \"chr2.g11238.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19378456', '19378939', '.', '+', '.', 'gene_id \"chr8.g6519\"; transcript_id \"chr8.g6519.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19475549', '19475602', '.', '+', '.', 'gene_id \"chr8.g6537\"; transcript_id \"chr8.g6537.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19689350', '19689479', '.', '-', '.', 'gene_id \"chr8.g6559\"; transcript_id \"chr8.g6559.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '20684514', '20684767', '.', '-', '.', 'gene_id \"chr8.g6619\"; transcript_id \"chr8.g6619.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '20819862', '20820029', '.', '+', '.', 'gene_id \"chr8.g6638\"; transcript_id \"chr8.g6638.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '20949503', '20949539', '.', '-', '.', 'gene_id \"chr8.g6651\"; transcript_id \"chr8.g6651.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31241786', '31241948', '.', '-', '.', 'gene_id \"chr2.g11434\"; transcript_id \"chr2.g11434.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '21738054', '21738565', '.', '+', '.', 'gene_id \"chr8.g6697\"; transcript_id \"chr8.g6697.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '21866317', '21866356', '.', '+', '.', 'gene_id \"chr8.g6724\"; transcript_id \"chr8.g6724.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '21991421', '21991474', '.', '-', '.', 'gene_id \"chr8.g6737\"; transcript_id \"chr8.g6737.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31322686', '31322733', '.', '+', '.', 'gene_id \"chr2.g11477\"; transcript_id \"chr2.g11477.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '22229180', '22229384', '.', '+', '.', 'gene_id \"chr8.g6811\"; transcript_id \"chr8.g6811.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '22262885', '22262939', '.', '+', '.', 'gene_id \"chr8.g6842\"; transcript_id \"chr8.g6842.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31467228', '31467259', '.', '-', '.', 'gene_id \"chr2.g11621\"; transcript_id \"chr2.g11621.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '22509513', '22509624', '.', '+', '.', 'gene_id \"chr8.g6998\"; transcript_id \"chr8.g6998.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31580203', '31580816', '.', '+', '.', 'gene_id \"chr2.g11734\"; transcript_id \"chr2.g11734.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31678562', '31678924', '.', '-', '.', 'gene_id \"chr2.g11764\"; transcript_id \"chr2.g11764.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '23350375', '23350691', '.', '-', '.', 'gene_id \"chr8.g7050\"; transcript_id \"chr8.g7050.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '23427732', '23427789', '.', '+', '.', 'gene_id \"chr8.g7056\"; transcript_id \"chr8.g7056.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '23850475', '23850523', '.', '-', '.', 'gene_id \"chr8.g7105\"; transcript_id \"chr8.g7105.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31832484', '31832516', '.', '-', '.', 'gene_id \"chr2.g11849\"; transcript_id \"chr2.g11849.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '24471853', '24472182', '.', '-', '.', 'gene_id \"chr8.g7205\"; transcript_id \"chr8.g7205.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '24472478', '24472633', '.', '+', '.', 'gene_id \"chr8.g7207\"; transcript_id \"chr8.g7207.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '31979244', '31979508', '.', '+', '.', 'gene_id \"chr2.g11960\"; transcript_id \"chr2.g11960.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '24783579', '24783618', '.', '-', '.', 'gene_id \"chr8.g7232\"; transcript_id \"chr8.g7232.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '25266863', '25266921', '.', '-', '.', 'gene_id \"chr8.g7268\"; transcript_id \"chr8.g7268.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '25430082', '25430311', '.', '+', '.', 'gene_id \"chr8.g7289\"; transcript_id \"chr8.g7289.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '25681845', '25681907', '.', '-', '.', 'gene_id \"chr8.g7305\"; transcript_id \"chr8.g7305.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '25690814', '25690844', '.', '+', '.', 'gene_id \"chr8.g7308\"; transcript_id \"chr8.g7308.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32062532', '32062752', '.', '-', '.', 'gene_id \"chr2.g12053\"; transcript_id \"chr2.g12053.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32063058', '32063305', '.', '-', '.', 'gene_id \"chr2.g12054\"; transcript_id \"chr2.g12054.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32064044', '32064204', '.', '-', '.', 'gene_id \"chr2.g12057\"; transcript_id \"chr2.g12057.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26057373', '26057604', '.', '-', '.', 'gene_id \"chr8.g7347\"; transcript_id \"chr8.g7347.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26057373', '26057429', '.', '-', '.', 'gene_id \"chr8.g7347\"; transcript_id \"chr8.g7347.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26156063', '26156109', '.', '-', '.', 'gene_id \"chr8.g7442\"; transcript_id \"chr8.g7442.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26201371', '26201572', '.', '-', '.', 'gene_id \"chr8.g7463\"; transcript_id \"chr8.g7463.i14\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26209326', '26209564', '.', '-', '.', 'gene_id \"chr8.g7475\"; transcript_id \"chr8.g7475.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26377006', '26377286', '.', '-', '.', 'gene_id \"chr8.g7554\"; transcript_id \"chr8.g7554.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32413357', '32413511', '.', '+', '.', 'gene_id \"chr2.g12330\"; transcript_id \"chr2.g12330.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32413605', '32413904', '.', '+', '.', 'gene_id \"chr2.g12331\"; transcript_id \"chr2.g12331.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26406013', '26406156', '.', '-', '.', 'gene_id \"chr8.g7602\"; transcript_id \"chr8.g7602.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26406214', '26407322', '.', '+', '.', 'gene_id \"chr8.g7603\"; transcript_id \"chr8.g7603.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32455429', '32455807', '.', '-', '.', 'gene_id \"chr2.g12385\"; transcript_id \"chr2.g12385.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26534044', '26534074', '.', '-', '.', 'gene_id \"chr8.g7760\"; transcript_id \"chr8.g7760.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '32843746', '32849063', '.', '+', '.', 'gene_id \"chr2.g12528\"; transcript_id \"chr2.g12528.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26648315', '26648357', '.', '+', '.', 'gene_id \"chr8.g7847\"; transcript_id \"chr8.g7847.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '33126688', '33126847', '.', '-', '.', 'gene_id \"chr2.g12769\"; transcript_id \"chr2.g12769.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '26996663', '26996715', '.', '+', '.', 'gene_id \"chr8.g8116\"; transcript_id \"chr8.g8116.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27163168', '27163324', '.', '-', '.', 'gene_id \"chr8.g8250\"; transcript_id \"chr8.g8250.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '33372250', '33372422', '.', '-', '.', 'gene_id \"chr2.g12958\"; transcript_id \"chr2.g12958.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27334349', '27334399', '.', '-', '.', 'gene_id \"chr8.g8307\"; transcript_id \"chr8.g8307.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '33659952', '33659988', '.', '-', '.', 'gene_id \"chr2.g13047\"; transcript_id \"chr2.g13047.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27529054', '27529123', '.', '-', '.', 'gene_id \"chr8.g8353\"; transcript_id \"chr8.g8353.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27749545', '27749750', '.', '-', '.', 'gene_id \"chr8.g8390\"; transcript_id \"chr8.g8390.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28182143', '28182198', '.', '-', '.', 'gene_id \"chr8.g8567\"; transcript_id \"chr8.g8567.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28185533', '28185592', '.', '-', '.', 'gene_id \"chr8.g8569\"; transcript_id \"chr8.g8569.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '33958635', '33958664', '.', '+', '.', 'gene_id \"chr2.g13330\"; transcript_id \"chr2.g13330.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28588463', '28588680', '.', '-', '.', 'gene_id \"chr8.g8724\"; transcript_id \"chr8.g8724.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28588687', '28588851', '.', '+', '.', 'gene_id \"chr8.g8725\"; transcript_id \"chr8.g8725.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28609143', '28609189', '.', '-', '.', 'gene_id \"chr8.g8736\"; transcript_id \"chr8.g8736.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '34222145', '34222179', '.', '-', '.', 'gene_id \"chr2.g13533\"; transcript_id \"chr2.g13533.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28740207', '28740328', '.', '-', '.', 'gene_id \"chr8.g8801\"; transcript_id \"chr8.g8801.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28775492', '28775652', '.', '-', '.', 'gene_id \"chr8.g8803\"; transcript_id \"chr8.g8803.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '34560685', '34560978', '.', '-', '.', 'gene_id \"chr2.g13719\"; transcript_id \"chr2.g13719.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '34589423', '34589738', '.', '-', '.', 'gene_id \"chr2.g13736\"; transcript_id \"chr2.g13736.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28982762', '28983216', '.', '+', '.', 'gene_id \"chr8.g9016\"; transcript_id \"chr8.g9016.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28982862', '28982951', '.', '+', '.', 'gene_id \"chr8.g9016\"; transcript_id \"chr8.g9016.i6\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29051484', '29051533', '.', '+', '.', 'gene_id \"chr8.g9056\"; transcript_id \"chr8.g9056.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '34943365', '34943665', '.', '-', '.', 'gene_id \"chr2.g13854\"; transcript_id \"chr2.g13854.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '35164920', '35165100', '.', '-', '.', 'gene_id \"chr2.g13990\"; transcript_id \"chr2.g13990.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29537881', '29537923', '.', '+', '.', 'gene_id \"chr8.g9234\"; transcript_id \"chr8.g9234.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29537925', '29538147', '.', '+', '.', 'gene_id \"chr8.g9235\"; transcript_id \"chr8.g9235.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '35370516', '35370614', '.', '+', '.', 'gene_id \"chr2.g14062\"; transcript_id \"chr2.g14062.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29954560', '29954875', '.', '-', '.', 'gene_id \"chr8.g9324\"; transcript_id \"chr8.g9324.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30247346', '30247528', '.', '+', '.', 'gene_id \"chr8.g9399\"; transcript_id \"chr8.g9399.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30247520', '30247722', '.', '-', '.', 'gene_id \"chr8.g9400\"; transcript_id \"chr8.g9400.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '35706394', '35706690', '.', '-', '.', 'gene_id \"chr2.g14271\"; transcript_id \"chr2.g14271.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30427939', '30427972', '.', '-', '.', 'gene_id \"chr8.g9557\"; transcript_id \"chr8.g9557.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr8', 'PASA', 'exon', '30616329', '30616364', '.', '-', '.', 'gene_id \"chr8.g9705\"; transcript_id \"chr8.g9705.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30683904', '30684061', '.', '-', '.', 'gene_id \"chr8.g9727\"; transcript_id \"chr8.g9727.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '36731169', '36731446', '.', '-', '.', 'gene_id \"chr2.g14496\"; transcript_id \"chr2.g14496.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30788743', '30789505', '.', '+', '.', 'gene_id \"chr8.g9808\"; transcript_id \"chr8.g9808.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30802257', '30802413', '.', '-', '.', 'gene_id \"chr8.g9829\"; transcript_id \"chr8.g9829.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '37750623', '37750812', '.', '-', '.', 'gene_id \"chr2.g14584\"; transcript_id \"chr2.g14584.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '37751950', '37752205', '.', '-', '.', 'gene_id \"chr2.g14585\"; transcript_id \"chr2.g14585.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30850821', '30850953', '.', '+', '.', 'gene_id \"chr8.g9888\"; transcript_id \"chr8.g9888.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30939685', '30939870', '.', '+', '.', 'gene_id \"chr8.g9930\"; transcript_id \"chr8.g9930.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30970712', '30971191', '.', '+', '.', 'gene_id \"chr8.g9937\"; transcript_id \"chr8.g9937.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '38448698', '38448866', '.', '-', '.', 'gene_id \"chr2.g14822\"; transcript_id \"chr2.g14822.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31156555', '31156732', '.', '+', '.', 'gene_id \"chr8.g10073\"; transcript_id \"chr8.g10073.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '38374879', '38375166', '.', '+', '.', 'gene_id \"chr2.g14851\"; transcript_id \"chr2.g14851.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31205237', '31205279', '.', '-', '.', 'gene_id \"chr8.g10087\"; transcript_id \"chr8.g10087.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31333858', '31334023', '.', '-', '.', 'gene_id \"chr8.g10172\"; transcript_id \"chr8.g10172.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31334012', '31334298', '.', '-', '.', 'gene_id \"chr8.g10173\"; transcript_id \"chr8.g10173.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31410607', '31410698', '.', '+', '.', 'gene_id \"chr8.g10184\"; transcript_id \"chr8.g10184.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31631732', '31631992', '.', '+', '.', 'gene_id \"chr8.g10200\"; transcript_id \"chr8.g10200.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '38807265', '38807443', '.', '-', '.', 'gene_id \"chr2.g15004\"; transcript_id \"chr2.g15004.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '32670730', '32670914', '.', '+', '.', 'gene_id \"chr8.g10471\"; transcript_id \"chr8.g10471.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33211514', '33211554', '.', '-', '.', 'gene_id \"chr8.g10659\"; transcript_id \"chr8.g10659.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33345547', '33346677', '.', '+', '.', 'gene_id \"chr8.g10696\"; transcript_id \"chr8.g10696.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33345482', '33346679', '.', '+', '.', 'gene_id \"chr8.g10696\"; transcript_id \"chr8.g10696.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33552332', '33552572', '.', '+', '.', 'gene_id \"chr8.g10878\"; transcript_id \"chr8.g10878.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33603845', '33603922', '.', '+', '.', 'gene_id \"chr8.g10921\"; transcript_id \"chr8.g10921.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '40752685', '40752728', '.', '+', '.', 'gene_id \"chr2.g15702\"; transcript_id \"chr2.g15702.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '40902024', '40902059', '.', '+', '.', 'gene_id \"chr2.g15813\"; transcript_id \"chr2.g15813.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '34337594', '34337852', '.', '+', '.', 'gene_id \"chr8.g11112\"; transcript_id \"chr8.g11112.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41835980', '41836197', '.', '-', '.', 'gene_id \"chr2.g16125\"; transcript_id \"chr2.g16125.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41836270', '41836643', '.', '-', '.', 'gene_id \"chr2.g16126\"; transcript_id \"chr2.g16126.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41836644', '41836895', '.', '+', '.', 'gene_id \"chr2.g16127\"; transcript_id \"chr2.g16127.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '34712328', '34712377', '.', '-', '.', 'gene_id \"chr8.g11363\"; transcript_id \"chr8.g11363.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41901958', '41902070', '.', '+', '.', 'gene_id \"chr2.g16161\"; transcript_id \"chr2.g16161.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41901958', '41902112', '.', '+', '.', 'gene_id \"chr2.g16161\"; transcript_id \"chr2.g16161.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '42629085', '42629125', '.', '+', '.', 'gene_id \"chr2.g16225\"; transcript_id \"chr2.g16225.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35044723', '35044760', '.', '-', '.', 'gene_id \"chr8.g11459\"; transcript_id \"chr8.g11459.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '42929388', '42929427', '.', '+', '.', 'gene_id \"chr2.g16254\"; transcript_id \"chr2.g16254.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35198467', '35198516', '.', '+', '.', 'gene_id \"chr8.g11571\"; transcript_id \"chr8.g11571.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35199880', '35200187', '.', '-', '.', 'gene_id \"chr8.g11572\"; transcript_id \"chr8.g11572.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35497516', '35497583', '.', '+', '.', 'gene_id \"chr8.g11627\"; transcript_id \"chr8.g11627.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35809637', '35810163', '.', '+', '.', 'gene_id \"chr8.g11782\"; transcript_id \"chr8.g11782.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43580815', '43581185', '.', '+', '.', 'gene_id \"chr2.g16505\"; transcript_id \"chr2.g16505.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43580652', '43580841', '.', '-', '.', 'gene_id \"chr2.g16506\"; transcript_id \"chr2.g16506.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36064382', '36064636', '.', '+', '.', 'gene_id \"chr8.g11891\"; transcript_id \"chr8.g11891.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36066766', '36067100', '.', '-', '.', 'gene_id \"chr8.g11896\"; transcript_id \"chr8.g11896.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36107100', '36107132', '.', '+', '.', 'gene_id \"chr8.g11917\"; transcript_id \"chr8.g11917.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43739447', '43739640', '.', '-', '.', 'gene_id \"chr2.g16625\"; transcript_id \"chr2.g16625.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36321518', '36321722', '.', '-', '.', 'gene_id \"chr8.g11993\"; transcript_id \"chr8.g11993.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36414774', '36415193', '.', '-', '.', 'gene_id \"chr8.g12001\"; transcript_id \"chr8.g12001.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36486946', '36487165', '.', '-', '.', 'gene_id \"chr8.g12048\"; transcript_id \"chr8.g12048.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43930631', '43930939', '.', '+', '.', 'gene_id \"chr2.g16749\"; transcript_id \"chr2.g16749.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36495226', '36495405', '.', '+', '.', 'gene_id \"chr8.g12057\"; transcript_id \"chr8.g12057.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36495397', '36495787', '.', '+', '.', 'gene_id \"chr8.g12058\"; transcript_id \"chr8.g12058.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36495822', '36496483', '.', '-', '.', 'gene_id \"chr8.g12059\"; transcript_id \"chr8.g12059.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43959351', '43959537', '.', '+', '.', 'gene_id \"chr2.g16761\"; transcript_id \"chr2.g16761.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37484231', '37484271', '.', '+', '.', 'gene_id \"chr8.g12206\"; transcript_id \"chr8.g12206.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37530450', '37530491', '.', '-', '.', 'gene_id \"chr8.g12213\"; transcript_id \"chr8.g12213.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37667116', '37667337', '.', '+', '.', 'gene_id \"chr8.g12265\"; transcript_id \"chr8.g12265.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '45042361', '45042617', '.', '-', '.', 'gene_id \"chr2.g17127\"; transcript_id \"chr2.g17127.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38177738', '38178042', '.', '+', '.', 'gene_id \"chr8.g12620\"; transcript_id \"chr8.g12620.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38178054', '38178381', '.', '-', '.', 'gene_id \"chr8.g12622\"; transcript_id \"chr8.g12622.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38462485', '38462557', '.', '+', '.', 'gene_id \"chr8.g12791\"; transcript_id \"chr8.g12791.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38511879', '38511919', '.', '+', '.', 'gene_id \"chr8.g12828\"; transcript_id \"chr8.g12828.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38659665', '38660334', '.', '-', '.', 'gene_id \"chr8.g12863\"; transcript_id \"chr8.g12863.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38609273', '38609314', '.', '+', '.', 'gene_id \"chr8.g12864\"; transcript_id \"chr8.g12864.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38624961', '38625121', '.', '-', '.', 'gene_id \"chr8.g12888\"; transcript_id \"chr8.g12888.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39003012', '39003132', '.', '+', '.', 'gene_id \"chr8.g13142\"; transcript_id \"chr8.g13142.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '38997645', '38998607', '.', '-', '.', 'gene_id \"chr8.g13160\"; transcript_id \"chr8.g13160.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39267128', '39267335', '.', '+', '.', 'gene_id \"chr8.g13348\"; transcript_id \"chr8.g13348.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39267349', '39267725', '.', '-', '.', 'gene_id \"chr8.g13350\"; transcript_id \"chr8.g13350.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39282167', '39282337', '.', '-', '.', 'gene_id \"chr8.g13357\"; transcript_id \"chr8.g13357.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39512084', '39512257', '.', '-', '.', 'gene_id \"chr8.g13474\"; transcript_id \"chr8.g13474.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39515629', '39515979', '.', '+', '.', 'gene_id \"chr8.g13479\"; transcript_id \"chr8.g13479.i9\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39515022', '39515228', '.', '-', '.', 'gene_id \"chr8.g13486\"; transcript_id \"chr8.g13486.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39807354', '39807527', '.', '+', '.', 'gene_id \"chr8.g13617\"; transcript_id \"chr8.g13617.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40533129', '40533820', '.', '-', '.', 'gene_id \"chr8.g13829\"; transcript_id \"chr8.g13829.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40625464', '40625647', '.', '+', '.', 'gene_id \"chr8.g13900\"; transcript_id \"chr8.g13900.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40696615', '40696928', '.', '+', '.', 'gene_id \"chr8.g13937\"; transcript_id \"chr8.g13937.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40696615', '40696897', '.', '+', '.', 'gene_id \"chr8.g13937\"; transcript_id \"chr8.g13937.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '49223497', '49223724', '.', '-', '.', 'gene_id \"chr2.g18730\"; transcript_id \"chr2.g18730.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '49223920', '49224088', '.', '-', '.', 'gene_id \"chr2.g18731\"; transcript_id \"chr2.g18731.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40919018', '40919187', '.', '-', '.', 'gene_id \"chr8.g14111\"; transcript_id \"chr8.g14111.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '49384044', '49384108', '.', '+', '.', 'gene_id \"chr2.g18850\"; transcript_id \"chr2.g18850.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41094144', '41094204', '.', '+', '.', 'gene_id \"chr8.g14212\"; transcript_id \"chr8.g14212.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41095580', '41095846', '.', '+', '.', 'gene_id \"chr8.g14213\"; transcript_id \"chr8.g14213.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41171204', '41171379', '.', '-', '.', 'gene_id \"chr8.g14230\"; transcript_id \"chr8.g14230.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '50941599', '50941712', '.', '-', '.', 'gene_id \"chr2.g19216\"; transcript_id \"chr2.g19216.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41577268', '41577422', '.', '+', '.', 'gene_id \"chr8.g14694\"; transcript_id \"chr8.g14694.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41633299', '41633361', '.', '+', '.', 'gene_id \"chr8.g14731\"; transcript_id \"chr8.g14731.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41776514', '41776582', '.', '-', '.', 'gene_id \"chr8.g14817\"; transcript_id \"chr8.g14817.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '51508096', '51508268', '.', '-', '.', 'gene_id \"chr2.g19385\"; transcript_id \"chr2.g19385.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '51508318', '51508593', '.', '-', '.', 'gene_id \"chr2.g19386\"; transcript_id \"chr2.g19386.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '51882003', '51882208', '.', '+', '.', 'gene_id \"chr2.g19502\"; transcript_id \"chr2.g19502.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '51956232', '51956411', '.', '-', '.', 'gene_id \"chr2.g19576\"; transcript_id \"chr2.g19576.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42365866', '42365912', '.', '+', '.', 'gene_id \"chr8.g15037\"; transcript_id \"chr8.g15037.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42516230', '42516659', '.', '+', '.', 'gene_id \"chr8.g15101\"; transcript_id \"chr8.g15101.i5\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '52139336', '52139687', '.', '+', '.', 'gene_id \"chr2.g19759\"; transcript_id \"chr2.g19759.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '52139769', '52139954', '.', '+', '.', 'gene_id \"chr2.g19760\"; transcript_id \"chr2.g19760.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42605916', '42606067', '.', '+', '.', 'gene_id \"chr8.g15178\"; transcript_id \"chr8.g15178.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42735046', '42735284', '.', '+', '.', 'gene_id \"chr8.g15242\"; transcript_id \"chr8.g15242.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42690656', '42690828', '.', '+', '.', 'gene_id \"chr8.g15277\"; transcript_id \"chr8.g15277.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42721505', '42721659', '.', '-', '.', 'gene_id \"chr8.g15318\"; transcript_id \"chr8.g15318.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '52620502', '52620888', '.', '+', '.', 'gene_id \"chr2.g19982\"; transcript_id \"chr2.g19982.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '52620502', '52620653', '.', '+', '.', 'gene_id \"chr2.g19982\"; transcript_id \"chr2.g19982.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42876586', '42876643', '.', '-', '.', 'gene_id \"chr8.g15462\"; transcript_id \"chr8.g15462.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43011163', '43011206', '.', '+', '.', 'gene_id \"chr8.g15520\"; transcript_id \"chr8.g15520.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43084434', '43084600', '.', '+', '.', 'gene_id \"chr8.g15596\"; transcript_id \"chr8.g15596.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43204473', '43204695', '.', '+', '.', 'gene_id \"chr8.g15650\"; transcript_id \"chr8.g15650.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '53083526', '53083880', '.', '+', '.', 'gene_id \"chr2.g20270\"; transcript_id \"chr2.g20270.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '53226282', '53226327', '.', '+', '.', 'gene_id \"chr2.g20339\"; transcript_id \"chr2.g20339.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43282961', '43283558', '.', '+', '.', 'gene_id \"chr8.g15771\"; transcript_id \"chr8.g15771.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43365313', '43365380', '.', '-', '.', 'gene_id \"chr8.g15830\"; transcript_id \"chr8.g15830.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43564555', '43566359', '.', '-', '.', 'gene_id \"chr8.g15916\"; transcript_id \"chr8.g15916.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43649646', '43649861', '.', '-', '.', 'gene_id \"chr8.g15985\"; transcript_id \"chr8.g15985.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '54176824', '54177188', '.', '+', '.', 'gene_id \"chr2.g20789\"; transcript_id \"chr2.g20789.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43995972', '43996929', '.', '+', '.', 'gene_id \"chr8.g16218\"; transcript_id \"chr8.g16218.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44007151', '44007254', '.', '+', '.', 'gene_id \"chr8.g16223\"; transcript_id \"chr8.g16223.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44012150', '44012574', '.', '-', '.', 'gene_id \"chr8.g16227\"; transcript_id \"chr8.g16227.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44067131', '44067265', '.', '+', '.', 'gene_id \"chr8.g16252\"; transcript_id \"chr8.g16252.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44254523', '44254727', '.', '+', '.', 'gene_id \"chr8.g16397\"; transcript_id \"chr8.g16397.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44255944', '44256246', '.', '+', '.', 'gene_id \"chr8.g16398\"; transcript_id \"chr8.g16398.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44408393', '44408454', '.', '+', '.', 'gene_id \"chr8.g16469\"; transcript_id \"chr8.g16469.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '55133416', '55133510', '.', '-', '.', 'gene_id \"chr2.g21183\"; transcript_id \"chr2.g21183.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44462476', '44462508', '.', '+', '.', 'gene_id \"chr8.g16545\"; transcript_id \"chr8.g16545.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '55345514', '55345675', '.', '+', '.', 'gene_id \"chr2.g21227\"; transcript_id \"chr2.g21227.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '55497592', '55497864', '.', '-', '.', 'gene_id \"chr2.g21349\"; transcript_id \"chr2.g21349.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44741224', '44741270', '.', '+', '.', 'gene_id \"chr8.g16668\"; transcript_id \"chr8.g16668.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44823488', '44823541', '.', '+', '.', 'gene_id \"chr8.g16706\"; transcript_id \"chr8.g16706.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44840321', '44840565', '.', '-', '.', 'gene_id \"chr8.g16714\"; transcript_id \"chr8.g16714.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44873469', '44873630', '.', '-', '.', 'gene_id \"chr8.g16736\"; transcript_id \"chr8.g16736.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44995930', '44996213', '.', '+', '.', 'gene_id \"chr8.g16821\"; transcript_id \"chr8.g16821.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45110868', '45111115', '.', '+', '.', 'gene_id \"chr8.g16900\"; transcript_id \"chr8.g16900.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45248631', '45248666', '.', '-', '.', 'gene_id \"chr8.g16934\"; transcript_id \"chr8.g16934.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45287308', '45287509', '.', '-', '.', 'gene_id \"chr8.g16944\"; transcript_id \"chr8.g16944.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45371642', '45371679', '.', '+', '.', 'gene_id \"chr8.g16973\"; transcript_id \"chr8.g16973.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45556060', '45556100', '.', '+', '.', 'gene_id \"chr8.g16995\"; transcript_id \"chr8.g16995.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45712759', '45712903', '.', '-', '.', 'gene_id \"chr8.g17036\"; transcript_id \"chr8.g17036.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46254878', '46255243', '.', '+', '.', 'gene_id \"chr8.g17262\"; transcript_id \"chr8.g17262.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46296358', '46296421', '.', '-', '.', 'gene_id \"chr8.g17280\"; transcript_id \"chr8.g17280.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46300216', '46300245', '.', '+', '.', 'gene_id \"chr8.g17284\"; transcript_id \"chr8.g17284.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46359984', '46360156', '.', '-', '.', 'gene_id \"chr8.g17294\"; transcript_id \"chr8.g17294.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46409649', '46409680', '.', '+', '.', 'gene_id \"chr8.g17362\"; transcript_id \"chr8.g17362.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr8', 'PASA', 'exon', '46538070', '46538106', '.', '+', '.', 'gene_id \"chr8.g17473\"; transcript_id \"chr8.g17473.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46605775', '46605827', '.', '-', '.', 'gene_id \"chr8.g17544\"; transcript_id \"chr8.g17544.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '56965574', '56965613', '.', '-', '.', 'gene_id \"chr2.g22396\"; transcript_id \"chr2.g22396.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46732830', '46733673', '.', '-', '.', 'gene_id \"chr8.g17625\"; transcript_id \"chr8.g17625.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46772341', '46772521', '.', '+', '.', 'gene_id \"chr8.g17688\"; transcript_id \"chr8.g17688.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46793603', '46793747', '.', '+', '.', 'gene_id \"chr8.g17719\"; transcript_id \"chr8.g17719.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46800751', '46800788', '.', '+', '.', 'gene_id \"chr8.g17726\"; transcript_id \"chr8.g17726.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46917542', '46917692', '.', '+', '.', 'gene_id \"chr8.g17823\"; transcript_id \"chr8.g17823.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47016224', '47016324', '.', '+', '.', 'gene_id \"chr8.g17864\"; transcript_id \"chr8.g17864.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47084199', '47086058', '.', '+', '.', 'gene_id \"chr8.g17866\"; transcript_id \"chr8.g17866.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47073417', '47073580', '.', '-', '.', 'gene_id \"chr8.g17875\"; transcript_id \"chr8.g17875.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47075639', '47075846', '.', '-', '.', 'gene_id \"chr8.g17877\"; transcript_id \"chr8.g17877.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47001140', '47001191', '.', '+', '.', 'gene_id \"chr8.g17915\"; transcript_id \"chr8.g17915.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47004888', '47004920', '.', '-', '.', 'gene_id \"chr8.g17917\"; transcript_id \"chr8.g17917.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47122693', '47122727', '.', '+', '.', 'gene_id \"chr8.g18013\"; transcript_id \"chr8.g18013.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47283748', '47283850', '.', '-', '.', 'gene_id \"chr8.g18168\"; transcript_id \"chr8.g18168.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '58536972', '58537196', '.', '-', '.', 'gene_id \"chr2.g23008\"; transcript_id \"chr2.g23008.i13\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47476842', '47476874', '.', '+', '.', 'gene_id \"chr8.g18238\"; transcript_id \"chr8.g18238.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47516780', '47516864', '.', '+', '.', 'gene_id \"chr8.g18253\"; transcript_id \"chr8.g18253.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47529552', '47529771', '.', '+', '.', 'gene_id \"chr8.g18264\"; transcript_id \"chr8.g18264.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47752801', '47753186', '.', '+', '.', 'gene_id \"chr8.g18400\"; transcript_id \"chr8.g18400.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '59201788', '59201942', '.', '+', '.', 'gene_id \"chr2.g23218\"; transcript_id \"chr2.g23218.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '47764701', '47764751', '.', '+', '.', 'gene_id \"chr8.g18428\"; transcript_id \"chr8.g18428.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '59442449', '59442486', '.', '-', '.', 'gene_id \"chr2.g23310\"; transcript_id \"chr2.g23310.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48032489', '48032716', '.', '+', '.', 'gene_id \"chr8.g18656\"; transcript_id \"chr8.g18656.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '59682577', '59682730', '.', '+', '.', 'gene_id \"chr2.g23445\"; transcript_id \"chr2.g23445.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48054488', '48054835', '.', '-', '.', 'gene_id \"chr8.g18693\"; transcript_id \"chr8.g18693.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48054940', '48055185', '.', '+', '.', 'gene_id \"chr8.g18694\"; transcript_id \"chr8.g18694.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '59723332', '59723487', '.', '+', '.', 'gene_id \"chr2.g23489\"; transcript_id \"chr2.g23489.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '59738424', '59738691', '.', '-', '.', 'gene_id \"chr2.g23501\"; transcript_id \"chr2.g23501.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48416233', '48416402', '.', '+', '.', 'gene_id \"chr8.g18955\"; transcript_id \"chr8.g18955.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48421296', '48421341', '.', '-', '.', 'gene_id \"chr8.g18958\"; transcript_id \"chr8.g18958.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48468477', '48468512', '.', '-', '.', 'gene_id \"chr8.g18974\"; transcript_id \"chr8.g18974.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48826851', '48827043', '.', '-', '.', 'gene_id \"chr8.g19050\"; transcript_id \"chr8.g19050.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48765241', '48765416', '.', '+', '.', 'gene_id \"chr8.g19144\"; transcript_id \"chr8.g19144.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '61499384', '61499536', '.', '-', '.', 'gene_id \"chr2.g23873\"; transcript_id \"chr2.g23873.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48793382', '48793710', '.', '-', '.', 'gene_id \"chr8.g19175\"; transcript_id \"chr8.g19175.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48793726', '48793939', '.', '-', '.', 'gene_id \"chr8.g19176\"; transcript_id \"chr8.g19176.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48793962', '48794238', '.', '+', '.', 'gene_id \"chr8.g19177\"; transcript_id \"chr8.g19177.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48827566', '48827764', '.', '-', '.', 'gene_id \"chr8.g19217\"; transcript_id \"chr8.g19217.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48962285', '48962365', '.', '+', '.', 'gene_id \"chr8.g19315\"; transcript_id \"chr8.g19315.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49028229', '49028404', '.', '+', '.', 'gene_id \"chr8.g19356\"; transcript_id \"chr8.g19356.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49027961', '49028251', '.', '-', '.', 'gene_id \"chr8.g19357\"; transcript_id \"chr8.g19357.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49055476', '49055920', '.', '+', '.', 'gene_id \"chr8.g19366\"; transcript_id \"chr8.g19366.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49057391', '49057974', '.', '+', '.', 'gene_id \"chr8.g19370\"; transcript_id \"chr8.g19370.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '62159816', '62160148', '.', '-', '.', 'gene_id \"chr2.g24149\"; transcript_id \"chr2.g24149.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '62160201', '62160376', '.', '-', '.', 'gene_id \"chr2.g24150\"; transcript_id \"chr2.g24150.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '62237411', '62237679', '.', '-', '.', 'gene_id \"chr2.g24167\"; transcript_id \"chr2.g24167.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '62237666', '62237864', '.', '+', '.', 'gene_id \"chr2.g24168\"; transcript_id \"chr2.g24168.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49591227', '49591267', '.', '+', '.', 'gene_id \"chr8.g19492\"; transcript_id \"chr8.g19492.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '49993402', '49993812', '.', '+', '.', 'gene_id \"chr8.g19769\"; transcript_id \"chr8.g19769.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50053011', '50053058', '.', '+', '.', 'gene_id \"chr8.g19802\"; transcript_id \"chr8.g19802.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50057371', '50057565', '.', '+', '.', 'gene_id \"chr8.g19811\"; transcript_id \"chr8.g19811.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50123195', '50123284', '.', '-', '.', 'gene_id \"chr8.g19856\"; transcript_id \"chr8.g19856.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '63555218', '63555262', '.', '+', '.', 'gene_id \"chr2.g24604\"; transcript_id \"chr2.g24604.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '63772713', '63772744', '.', '-', '.', 'gene_id \"chr2.g24646\"; transcript_id \"chr2.g24646.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50310104', '50310227', '.', '+', '.', 'gene_id \"chr8.g19990\"; transcript_id \"chr8.g19990.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50357544', '50357737', '.', '+', '.', 'gene_id \"chr8.g20044\"; transcript_id \"chr8.g20044.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50385397', '50385433', '.', '-', '.', 'gene_id \"chr8.g20069\"; transcript_id \"chr8.g20069.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50520885', '50520918', '.', '+', '.', 'gene_id \"chr8.g20125\"; transcript_id \"chr8.g20125.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '64178279', '64178493', '.', '-', '.', 'gene_id \"chr2.g24870\"; transcript_id \"chr2.g24870.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50611254', '50611290', '.', '-', '.', 'gene_id \"chr8.g20156\"; transcript_id \"chr8.g20156.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '51332483', '51332540', '.', '+', '.', 'gene_id \"chr8.g20196\"; transcript_id \"chr8.g20196.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '51517125', '51517169', '.', '-', '.', 'gene_id \"chr8.g20215\"; transcript_id \"chr8.g20215.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '64646611', '64646681', '.', '-', '.', 'gene_id \"chr2.g25007\"; transcript_id \"chr2.g25007.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '64683639', '64683822', '.', '-', '.', 'gene_id \"chr2.g25033\"; transcript_id \"chr2.g25033.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '52718601', '52718642', '.', '+', '.', 'gene_id \"chr8.g20333\"; transcript_id \"chr8.g20333.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '64815494', '64815764', '.', '-', '.', 'gene_id \"chr2.g25101\"; transcript_id \"chr2.g25101.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '52982412', '52982539', '.', '-', '.', 'gene_id \"chr8.g20414\"; transcript_id \"chr8.g20414.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '52982413', '52982461', '.', '-', '.', 'gene_id \"chr8.g20414\"; transcript_id \"chr8.g20414.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53230674', '53230853', '.', '-', '.', 'gene_id \"chr8.g20452\"; transcript_id \"chr8.g20452.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53320591', '53320816', '.', '+', '.', 'gene_id \"chr8.g20463\"; transcript_id \"chr8.g20463.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65100627', '65100783', '.', '-', '.', 'gene_id \"chr2.g25274\"; transcript_id \"chr2.g25274.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53544005', '53544262', '.', '+', '.', 'gene_id \"chr8.g20574\"; transcript_id \"chr8.g20574.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53636225', '53636437', '.', '+', '.', 'gene_id \"chr8.g20620\"; transcript_id \"chr8.g20620.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65264794', '65265091', '.', '-', '.', 'gene_id \"chr2.g25397\"; transcript_id \"chr2.g25397.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53718323', '53718473', '.', '+', '.', 'gene_id \"chr8.g20700\"; transcript_id \"chr8.g20700.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65611055', '65612105', '.', '-', '.', 'gene_id \"chr2.g25666\"; transcript_id \"chr2.g25666.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '55716941', '55716984', '.', '+', '.', 'gene_id \"chr8.g20908\"; transcript_id \"chr8.g20908.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '55758631', '55758695', '.', '+', '.', 'gene_id \"chr8.g20913\"; transcript_id \"chr8.g20913.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65628921', '65629102', '.', '-', '.', 'gene_id \"chr2.g25699\"; transcript_id \"chr2.g25699.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65629089', '65629428', '.', '+', '.', 'gene_id \"chr2.g25700\"; transcript_id \"chr2.g25700.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65631367', '65631819', '.', '-', '.', 'gene_id \"chr2.g25703\"; transcript_id \"chr2.g25703.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65631940', '65632138', '.', '+', '.', 'gene_id \"chr2.g25704\"; transcript_id \"chr2.g25704.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '56952080', '56952137', '.', '+', '.', 'gene_id \"chr8.g20967\"; transcript_id \"chr8.g20967.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65743789', '65744391', '.', '+', '.', 'gene_id \"chr2.g25771\"; transcript_id \"chr2.g25771.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65743461', '65743716', '.', '+', '.', 'gene_id \"chr2.g25772\"; transcript_id \"chr2.g25772.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57202735', '57203232', '.', '-', '.', 'gene_id \"chr8.g21008\"; transcript_id \"chr8.g21008.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '65739816', '65740492', '.', '-', '.', 'gene_id \"chr2.g25801\"; transcript_id \"chr2.g25801.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57235992', '57236198', '.', '+', '.', 'gene_id \"chr8.g21050\"; transcript_id \"chr8.g21050.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57857413', '57857442', '.', '-', '.', 'gene_id \"chr8.g21145\"; transcript_id \"chr8.g21145.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '58551332', '58551755', '.', '+', '.', 'gene_id \"chr8.g21180\"; transcript_id \"chr8.g21180.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66051161', '66051333', '.', '-', '.', 'gene_id \"chr2.g26066\"; transcript_id \"chr2.g26066.i7\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66070903', '66071125', '.', '-', '.', 'gene_id \"chr2.g26115\"; transcript_id \"chr2.g26115.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66070903', '66071014', '.', '-', '.', 'gene_id \"chr2.g26115\"; transcript_id \"chr2.g26115.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59287421', '59287680', '.', '-', '.', 'gene_id \"chr8.g21292\"; transcript_id \"chr8.g21292.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59439654', '59439697', '.', '+', '.', 'gene_id \"chr8.g21409\"; transcript_id \"chr8.g21409.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59595075', '59595120', '.', '+', '.', 'gene_id \"chr8.g21432\"; transcript_id \"chr8.g21432.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59638777', '59638879', '.', '-', '.', 'gene_id \"chr8.g21475\"; transcript_id \"chr8.g21475.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59676053', '59676159', '.', '-', '.', 'gene_id \"chr8.g21505\"; transcript_id \"chr8.g21505.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66479948', '66479980', '.', '-', '.', 'gene_id \"chr2.g26423\"; transcript_id \"chr2.g26423.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66611667', '66611852', '.', '+', '.', 'gene_id \"chr2.g26513\"; transcript_id \"chr2.g26513.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66611893', '66612160', '.', '+', '.', 'gene_id \"chr2.g26514\"; transcript_id \"chr2.g26514.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59789057', '59789375', '.', '+', '.', 'gene_id \"chr8.g21609\"; transcript_id \"chr8.g21609.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59793943', '59794075', '.', '+', '.', 'gene_id \"chr8.g21615\"; transcript_id \"chr8.g21615.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59831773', '59831975', '.', '+', '.', 'gene_id \"chr8.g21640\"; transcript_id \"chr8.g21640.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '67187486', '67187656', '.', '-', '.', 'gene_id \"chr2.g26820\"; transcript_id \"chr2.g26820.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60476681', '60476838', '.', '+', '.', 'gene_id \"chr8.g21987\"; transcript_id \"chr8.g21987.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60487839', '60488170', '.', '+', '.', 'gene_id \"chr8.g22003\"; transcript_id \"chr8.g22003.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '67991890', '67991938', '.', '-', '.', 'gene_id \"chr2.g27089\"; transcript_id \"chr2.g27089.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60633990', '60634211', '.', '+', '.', 'gene_id \"chr8.g22112\"; transcript_id \"chr8.g22112.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68250628', '68250799', '.', '+', '.', 'gene_id \"chr2.g27108\"; transcript_id \"chr2.g27108.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68261175', '68261338', '.', '+', '.', 'gene_id \"chr2.g27109\"; transcript_id \"chr2.g27109.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68515332', '68515372', '.', '-', '.', 'gene_id \"chr2.g27140\"; transcript_id \"chr2.g27140.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60830565', '60830768', '.', '-', '.', 'gene_id \"chr8.g22159\"; transcript_id \"chr8.g22159.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60847522', '60847558', '.', '-', '.', 'gene_id \"chr8.g22186\"; transcript_id \"chr8.g22186.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68681369', '68681607', '.', '+', '.', 'gene_id \"chr2.g27189\"; transcript_id \"chr2.g27189.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68730303', '68730339', '.', '-', '.', 'gene_id \"chr2.g27211\"; transcript_id \"chr2.g27211.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68773189', '68773470', '.', '+', '.', 'gene_id \"chr2.g27276\"; transcript_id \"chr2.g27276.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68833948', '68834189', '.', '-', '.', 'gene_id \"chr2.g27334\"; transcript_id \"chr2.g27334.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61200692', '61200878', '.', '+', '.', 'gene_id \"chr8.g22333\"; transcript_id \"chr8.g22333.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61215571', '61215763', '.', '+', '.', 'gene_id \"chr8.g22335\"; transcript_id \"chr8.g22335.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61261315', '61261530', '.', '+', '.', 'gene_id \"chr8.g22375\"; transcript_id \"chr8.g22375.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61261688', '61261852', '.', '-', '.', 'gene_id \"chr8.g22376\"; transcript_id \"chr8.g22376.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61362611', '61364539', '.', '-', '.', 'gene_id \"chr8.g22414\"; transcript_id \"chr8.g22414.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61380311', '61380360', '.', '-', '.', 'gene_id \"chr8.g22422\"; transcript_id \"chr8.g22422.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61462080', '61462168', '.', '+', '.', 'gene_id \"chr8.g22438\"; transcript_id \"chr8.g22438.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68960706', '68960896', '.', '+', '.', 'gene_id \"chr2.g27469\"; transcript_id \"chr2.g27469.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61639933', '61640469', '.', '-', '.', 'gene_id \"chr8.g22558\"; transcript_id \"chr8.g22558.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '69349757', '69350028', '.', '-', '.', 'gene_id \"chr2.g27764\"; transcript_id \"chr2.g27764.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '69355930', '69356010', '.', '+', '.', 'gene_id \"chr2.g27770\"; transcript_id \"chr2.g27770.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61966263', '61966315', '.', '+', '.', 'gene_id \"chr8.g22761\"; transcript_id \"chr8.g22761.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62043276', '62043312', '.', '+', '.', 'gene_id \"chr8.g22768\"; transcript_id \"chr8.g22768.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62080593', '62081035', '.', '-', '.', 'gene_id \"chr8.g22810\"; transcript_id \"chr8.g22810.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62366866', '62366915', '.', '-', '.', 'gene_id \"chr8.g22961\"; transcript_id \"chr8.g22961.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '69924355', '69924614', '.', '+', '.', 'gene_id \"chr2.g28194\"; transcript_id \"chr2.g28194.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62677189', '62677220', '.', '+', '.', 'gene_id \"chr8.g23162\"; transcript_id \"chr8.g23162.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62893369', '62893402', '.', '-', '.', 'gene_id \"chr8.g23231\"; transcript_id \"chr8.g23231.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '62932185', '62932250', '.', '-', '.', 'gene_id \"chr8.g23248\"; transcript_id \"chr8.g23248.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '70197300', '70197527', '.', '+', '.', 'gene_id \"chr2.g28470\"; transcript_id \"chr2.g28470.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '63962677', '63962872', '.', '-', '.', 'gene_id \"chr8.g23633\"; transcript_id \"chr8.g23633.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '63963240', '63963515', '.', '-', '.', 'gene_id \"chr8.g23635\"; transcript_id \"chr8.g23635.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '70604891', '70605203', '.', '+', '.', 'gene_id \"chr2.g28750\"; transcript_id \"chr2.g28750.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '70605203', '70605381', '.', '+', '.', 'gene_id \"chr2.g28751\"; transcript_id \"chr2.g28751.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64043311', '64043382', '.', '+', '.', 'gene_id \"chr8.g23678\"; transcript_id \"chr8.g23678.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64231657', '64231840', '.', '-', '.', 'gene_id \"chr8.g23745\"; transcript_id \"chr8.g23745.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64297435', '64297614', '.', '+', '.', 'gene_id \"chr8.g23809\"; transcript_id \"chr8.g23809.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64328270', '64328441', '.', '+', '.', 'gene_id \"chr8.g23830\"; transcript_id \"chr8.g23830.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64328421', '64328833', '.', '-', '.', 'gene_id \"chr8.g23831\"; transcript_id \"chr8.g23831.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64331828', '64331858', '.', '-', '.', 'gene_id \"chr8.g23838\"; transcript_id \"chr8.g23838.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '70784606', '70785020', '.', '+', '.', 'gene_id \"chr2.g28971\"; transcript_id \"chr2.g28971.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64333976', '64334979', '.', '-', '.', 'gene_id \"chr8.g23841\"; transcript_id \"chr8.g23841.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64335064', '64335334', '.', '+', '.', 'gene_id \"chr8.g23843\"; transcript_id \"chr8.g23843.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64335331', '64335982', '.', '+', '.', 'gene_id \"chr8.g23844\"; transcript_id \"chr8.g23844.i2\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64335961', '64336865', '.', '-', '.', 'gene_id \"chr8.g23845\"; transcript_id \"chr8.g23845.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64521722', '64521754', '.', '-', '.', 'gene_id \"chr8.g23884\"; transcript_id \"chr8.g23884.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '65003587', '65003622', '.', '-', '.', 'gene_id \"chr8.g24030\"; transcript_id \"chr8.g24030.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr8', 'PASA', 'exon', '65241069', '65241291', '.', '-', '.', 'gene_id \"chr8.g24214\"; transcript_id \"chr8.g24214.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '65474028', '65474069', '.', '+', '.', 'gene_id \"chr8.g24254\"; transcript_id \"chr8.g24254.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '71471585', '71471755', '.', '-', '.', 'gene_id \"chr2.g29463\"; transcript_id \"chr2.g29463.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '71471978', '71472266', '.', '-', '.', 'gene_id \"chr2.g29464\"; transcript_id \"chr2.g29464.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67217880', '67218064', '.', '-', '.', 'gene_id \"chr8.g24420\"; transcript_id \"chr8.g24420.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67375184', '67375349', '.', '-', '.', 'gene_id \"chr8.g24446\"; transcript_id \"chr8.g24446.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67592348', '67592418', '.', '+', '.', 'gene_id \"chr8.g24480\"; transcript_id \"chr8.g24480.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67839278', '67839457', '.', '+', '.', 'gene_id \"chr8.g24609\"; transcript_id \"chr8.g24609.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67839501', '67839684', '.', '-', '.', 'gene_id \"chr8.g24610\"; transcript_id \"chr8.g24610.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67966988', '67967379', '.', '-', '.', 'gene_id \"chr8.g24695\"; transcript_id \"chr8.g24695.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67970703', '67971131', '.', '-', '.', 'gene_id \"chr8.g24698\"; transcript_id \"chr8.g24698.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67968809', '67968971', '.', '-', '.', 'gene_id \"chr8.g24700\"; transcript_id \"chr8.g24700.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67987840', '67989461', '.', '+', '.', 'gene_id \"chr8.g24703\"; transcript_id \"chr8.g24703.i10\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '67980609', '67980898', '.', '+', '.', 'gene_id \"chr8.g24703\"; transcript_id \"chr8.g24703.i16\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68002684', '68003105', '.', '-', '.', 'gene_id \"chr8.g24721\"; transcript_id \"chr8.g24721.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68003095', '68003262', '.', '+', '.', 'gene_id \"chr8.g24722\"; transcript_id \"chr8.g24722.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68003466', '68003672', '.', '-', '.', 'gene_id \"chr8.g24724\"; transcript_id \"chr8.g24724.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72375129', '72375190', '.', '+', '.', 'gene_id \"chr2.g30023\"; transcript_id \"chr2.g30023.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72528523', '72528701', '.', '+', '.', 'gene_id \"chr2.g30154\"; transcript_id \"chr2.g30154.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68482789', '68482912', '.', '+', '.', 'gene_id \"chr8.g24999\"; transcript_id \"chr8.g24999.i4\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72600090', '72600239', '.', '-', '.', 'gene_id \"chr2.g30228\"; transcript_id \"chr2.g30228.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72673727', '72674881', '.', '-', '.', 'gene_id \"chr2.g30268\"; transcript_id \"chr2.g30268.i4\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72678882', '72679162', '.', '+', '.', 'gene_id \"chr2.g30274\"; transcript_id \"chr2.g30274.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '72743675', '72743705', '.', '-', '.', 'gene_id \"chr2.g30300\"; transcript_id \"chr2.g30300.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68947902', '68947938', '.', '-', '.', 'gene_id \"chr8.g25306\"; transcript_id \"chr8.g25306.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '69123628', '69123771', '.', '+', '.', 'gene_id \"chr8.g25361\"; transcript_id \"chr8.g25361.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '69243290', '69243551', '.', '+', '.', 'gene_id \"chr8.g25383\"; transcript_id \"chr8.g25383.i4\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '69242952', '69243324', '.', '-', '.', 'gene_id \"chr8.g25384\"; transcript_id \"chr8.g25384.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '73361531', '73362342', '.', '+', '.', 'gene_id \"chr2.g30609\"; transcript_id \"chr2.g30609.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '73396871', '73396933', '.', '-', '.', 'gene_id \"chr2.g30656\"; transcript_id \"chr2.g30656.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '73414605', '73415812', '.', '+', '.', 'gene_id \"chr2.g30682\"; transcript_id \"chr2.g30682.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '73470958', '73471129', '.', '+', '.', 'gene_id \"chr2.g30719\"; transcript_id \"chr2.g30719.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '73573052', '73573132', '.', '+', '.', 'gene_id \"chr2.g30776\"; transcript_id \"chr2.g30776.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '69856761', '69856913', '.', '-', '.', 'gene_id \"chr8.g25576\"; transcript_id \"chr8.g25576.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70026422', '70026730', '.', '+', '.', 'gene_id \"chr8.g25737\"; transcript_id \"chr8.g25737.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70067088', '70068231', '.', '+', '.', 'gene_id \"chr8.g25781\"; transcript_id \"chr8.g25781.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74171408', '74171736', '.', '-', '.', 'gene_id \"chr2.g30994\"; transcript_id \"chr2.g30994.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74358229', '74358559', '.', '-', '.', 'gene_id \"chr2.g31076\"; transcript_id \"chr2.g31076.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74360481', '74360642', '.', '+', '.', 'gene_id \"chr2.g31081\"; transcript_id \"chr2.g31081.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70223083', '70223122', '.', '+', '.', 'gene_id \"chr8.g25932\"; transcript_id \"chr8.g25932.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70286726', '70286772', '.', '-', '.', 'gene_id \"chr8.g26036\"; transcript_id \"chr8.g26036.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70330644', '70330838', '.', '-', '.', 'gene_id \"chr8.g26069\"; transcript_id \"chr8.g26069.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74575714', '74576051', '.', '+', '.', 'gene_id \"chr2.g31250\"; transcript_id \"chr2.g31250.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74656113', '74656507', '.', '-', '.', 'gene_id \"chr2.g31312\"; transcript_id \"chr2.g31312.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70474893', '70474929', '.', '+', '.', 'gene_id \"chr8.g26175\"; transcript_id \"chr8.g26175.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70606964', '70607115', '.', '-', '.', 'gene_id \"chr8.g26270\"; transcript_id \"chr8.g26270.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '74893028', '74893062', '.', '-', '.', 'gene_id \"chr2.g31471\"; transcript_id \"chr2.g31471.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70652565', '70652717', '.', '+', '.', 'gene_id \"chr8.g26309\"; transcript_id \"chr8.g26309.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '75037905', '75037937', '.', '+', '.', 'gene_id \"chr2.g31528\"; transcript_id \"chr2.g31528.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '75172996', '75173956', '.', '-', '.', 'gene_id \"chr2.g31545\"; transcript_id \"chr2.g31545.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70839216', '70839247', '.', '-', '.', 'gene_id \"chr8.g26448\"; transcript_id \"chr8.g26448.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71085096', '71085253', '.', '-', '.', 'gene_id \"chr8.g26568\"; transcript_id \"chr8.g26568.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71116221', '71116659', '.', '+', '.', 'gene_id \"chr8.g26626\"; transcript_id \"chr8.g26626.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '75479448', '75479852', '.', '+', '.', 'gene_id \"chr2.g31920\"; transcript_id \"chr2.g31920.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '75479887', '75480135', '.', '-', '.', 'gene_id \"chr2.g31921\"; transcript_id \"chr2.g31921.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71220695', '71221354', '.', '-', '.', 'gene_id \"chr8.g26750\"; transcript_id \"chr8.g26750.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '75533578', '75533974', '.', '+', '.', 'gene_id \"chr2.g31986\"; transcript_id \"chr2.g31986.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71354415', '71354608', '.', '-', '.', 'gene_id \"chr8.g26904\"; transcript_id \"chr8.g26904.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71387306', '71388316', '.', '-', '.', 'gene_id \"chr8.g26930\"; transcript_id \"chr8.g26930.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71454998', '71455060', '.', '-', '.', 'gene_id \"chr8.g26981\"; transcript_id \"chr8.g26981.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71462733', '71463054', '.', '-', '.', 'gene_id \"chr8.g26994\"; transcript_id \"chr8.g26994.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71509943', '71510331', '.', '+', '.', 'gene_id \"chr8.g27050\"; transcript_id \"chr8.g27050.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71548992', '71549064', '.', '+', '.', 'gene_id \"chr8.g27062\"; transcript_id \"chr8.g27062.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71631873', '71632072', '.', '+', '.', 'gene_id \"chr8.g27105\"; transcript_id \"chr8.g27105.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76043790', '76044007', '.', '-', '.', 'gene_id \"chr2.g32295\"; transcript_id \"chr2.g32295.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71728304', '71729032', '.', '+', '.', 'gene_id \"chr8.g27116\"; transcript_id \"chr8.g27116.i13\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71727815', '71728120', '.', '-', '.', 'gene_id \"chr8.g27119\"; transcript_id \"chr8.g27119.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71725176', '71725622', '.', '-', '.', 'gene_id \"chr8.g27120\"; transcript_id \"chr8.g27120.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71719156', '71719565', '.', '-', '.', 'gene_id \"chr8.g27121\"; transcript_id \"chr8.g27121.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71726644', '71726879', '.', '+', '.', 'gene_id \"chr8.g27130\"; transcript_id \"chr8.g27130.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71729651', '71729850', '.', '+', '.', 'gene_id \"chr8.g27131\"; transcript_id \"chr8.g27131.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71731472', '71731704', '.', '-', '.', 'gene_id \"chr8.g27133\"; transcript_id \"chr8.g27133.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71731846', '71732145', '.', '-', '.', 'gene_id \"chr8.g27134\"; transcript_id \"chr8.g27134.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76229391', '76235151', '.', '-', '.', 'gene_id \"chr2.g32380\"; transcript_id \"chr2.g32380.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76251981', '76253283', '.', '-', '.', 'gene_id \"chr2.g32380\"; transcript_id \"chr2.g32380.i7\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76237340', '76237580', '.', '-', '.', 'gene_id \"chr2.g32381\"; transcript_id \"chr2.g32381.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76248643', '76248818', '.', '+', '.', 'gene_id \"chr2.g32393\"; transcript_id \"chr2.g32393.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76249511', '76249833', '.', '+', '.', 'gene_id \"chr2.g32395\"; transcript_id \"chr2.g32395.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76253311', '76253655', '.', '+', '.', 'gene_id \"chr2.g32399\"; transcript_id \"chr2.g32399.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '72150031', '72150185', '.', '-', '.', 'gene_id \"chr8.g27259\"; transcript_id \"chr8.g27259.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76410860', '76411337', '.', '+', '.', 'gene_id \"chr2.g32492\"; transcript_id \"chr2.g32492.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76626544', '76626633', '.', '-', '.', 'gene_id \"chr2.g32632\"; transcript_id \"chr2.g32632.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76626544', '76627268', '.', '-', '.', 'gene_id \"chr2.g32632\"; transcript_id \"chr2.g32632.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76626542', '76626633', '.', '-', '.', 'gene_id \"chr2.g32632\"; transcript_id \"chr2.g32632.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76626542', '76627268', '.', '-', '.', 'gene_id \"chr2.g32632\"; transcript_id \"chr2.g32632.i4\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76565522', '76565655', '.', '-', '.', 'gene_id \"chr2.g32634\"; transcript_id \"chr2.g32634.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76696229', '76696449', '.', '-', '.', 'gene_id \"chr2.g32637\"; transcript_id \"chr2.g32637.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76566545', '76566767', '.', '-', '.', 'gene_id \"chr2.g32644\"; transcript_id \"chr2.g32644.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '72551203', '72551433', '.', '-', '.', 'gene_id \"chr8.g27493\"; transcript_id \"chr8.g27493.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76620131', '76620282', '.', '-', '.', 'gene_id \"chr2.g32718\"; transcript_id \"chr2.g32718.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76672452', '76672876', '.', '-', '.', 'gene_id \"chr2.g32789\"; transcript_id \"chr2.g32789.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76672872', '76673088', '.', '+', '.', 'gene_id \"chr2.g32790\"; transcript_id \"chr2.g32790.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76673458', '76674160', '.', '-', '.', 'gene_id \"chr2.g32792\"; transcript_id \"chr2.g32792.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76720685', '76722192', '.', '+', '.', 'gene_id \"chr2.g32824\"; transcript_id \"chr2.g32824.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73390440', '73390525', '.', '+', '.', 'gene_id \"chr8.g27793\"; transcript_id \"chr8.g27793.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73498453', '73498646', '.', '+', '.', 'gene_id \"chr8.g27805\"; transcript_id \"chr8.g27805.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73562489', '73562555', '.', '-', '.', 'gene_id \"chr8.g27811\"; transcript_id \"chr8.g27811.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73805403', '73805434', '.', '+', '.', 'gene_id \"chr8.g27839\"; transcript_id \"chr8.g27839.i1\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73805403', '73805434', '.', '-', '.', 'gene_id \"chr8.g27840\"; transcript_id \"chr8.g27840.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76993505', '76993954', '.', '-', '.', 'gene_id \"chr2.g33015\"; transcript_id \"chr2.g33015.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '76993506', '76993660', '.', '-', '.', 'gene_id \"chr2.g33015\"; transcript_id \"chr2.g33015.i3\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68545346', '68545389', '100', '+', '.', 'gene_id \"chr8.g27926\"; transcript_id \"blat.proc18128.chain_23520\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '59969877', '59969950', '100', '+', '.', 'gene_id \"chr8.g27961\"; transcript_id \"blat.proc18128.chain_25513\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1576640', '1576867', '100', '+', '.', 'gene_id \"chr8.g27985\"; transcript_id \"blat.proc28144.chain_1779\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '55494603', '55494638', '100', '-', '.', 'gene_id \"chr8.g28030\"; transcript_id \"blat.proc18128.chain_4806\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44006788', '44007098', '100', '+', '.', 'gene_id \"chr8.g28055\"; transcript_id \"blat.proc18128.chain_268\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35622770', '35622802', '100', '-', '.', 'gene_id \"chr8.g28124\"; transcript_id \"blat.proc4587.chain_12121\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45293546', '45293650', '100', '-', '.', 'gene_id \"chr8.g28176\"; transcript_id \"blat.proc18128.chain_20761\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '7662366', '7662417', '100', '+', '.', 'gene_id \"chr8.g28230\"; transcript_id \"blat.proc18128.chain_23538\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43664287', '43664327', '100', '-', '.', 'gene_id \"chr8.g28232\"; transcript_id \"blat.proc18128.chain_13851\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64873772', '64873820', '100', '+', '.', 'gene_id \"chr8.g28256\"; transcript_id \"blat.proc28144.chain_4710\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70449764', '70449812', '100', '-', '.', 'gene_id \"chr8.g28320\"; transcript_id \"blat.proc4587.chain_9610\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39348798', '39348862', '100', '-', '.', 'gene_id \"chr8.g28457\"; transcript_id \"blat.proc18128.chain_19318\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70696821', '70696877', '100', '+', '.', 'gene_id \"chr8.g28479\"; transcript_id \"blat.proc4587.chain_10231\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40372584', '40372620', '100', '-', '.', 'gene_id \"chr8.g28662\"; transcript_id \"blat.proc18128.chain_743\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '33835474', '33835523', '100', '-', '.', 'gene_id \"chr8.g28799\"; transcript_id \"blat.proc28144.chain_21897\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '18486691', '18486891', '100', '-', '.', 'gene_id \"chr8.g29003\"; transcript_id \"blat.proc4005.chain_5991\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31506660', '31506690', '100', '-', '.', 'gene_id \"chr8.g29005\"; transcript_id \"blat.proc28144.chain_372\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6891237', '6891523', '100', '-', '.', 'gene_id \"chr8.g29029\"; transcript_id \"blat.proc4005.chain_648\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '18935541', '18935578', '100', '-', '.', 'gene_id \"chr8.g29044\"; transcript_id \"blat.proc28144.chain_5554\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28678016', '28678092', '100', '-', '.', 'gene_id \"chr8.g29065\"; transcript_id \"blat.proc18128.chain_5283\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '61124424', '61124463', '100', '+', '.', 'gene_id \"chr8.g29102\"; transcript_id \"blat.proc4587.chain_7397\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73372885', '73372997', '100', '-', '.', 'gene_id \"chr8.g29122\"; transcript_id \"blat.proc28144.chain_21295\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3906191', '3906447', '100', '-', '.', 'gene_id \"chr8.g29209\"; transcript_id \"blat.proc18128.chain_744\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29152544', '29152677', '100', '-', '.', 'gene_id \"chr8.g29240\"; transcript_id \"blat.proc28144.chain_17864\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '4325419', '4325479', '100', '-', '.', 'gene_id \"chr8.g29241\"; transcript_id \"blat.proc18128.chain_865\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '964693', '964843', '100', '+', '.', 'gene_id \"chr8.g29426\"; transcript_id \"blat.proc18128.chain_25379\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '9352175', '9352239', '100', '+', '.', 'gene_id \"chr8.g29444\"; transcript_id \"blat.proc4005.chain_4297\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '21442807', '21442860', '100', '-', '.', 'gene_id \"chr8.g29528\"; transcript_id \"blat.proc18128.chain_3771\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53067250', '53067282', '100', '-', '.', 'gene_id \"chr8.g29599\"; transcript_id \"blat.proc18128.chain_6980\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '43987534', '43987642', '100', '-', '.', 'gene_id \"chr8.g29604\"; transcript_id \"blat.proc18128.chain_8238\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40336768', '40336813', '100', '+', '.', 'gene_id \"chr8.g29633\"; transcript_id \"blat.proc28144.chain_10201\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '63002374', '63002424', '100', '+', '.', 'gene_id \"chr8.g29777\"; transcript_id \"blat.proc4005.chain_1025\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19231649', '19231680', '100', '+', '.', 'gene_id \"chr8.g29784\"; transcript_id \"blat.proc4587.chain_1348\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '34701139', '34701212', '100', '-', '.', 'gene_id \"chr8.g29799\"; transcript_id \"blat.proc4587.chain_3683\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44834089', '44834127', '100', '-', '.', 'gene_id \"chr8.g29924\"; transcript_id \"blat.proc18128.chain_17949\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3535396', '3535465', '100', '-', '.', 'gene_id \"chr8.g30045\"; transcript_id \"blat.proc18128.chain_964\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '19251387', '19251472', '100', '+', '.', 'gene_id \"chr8.g30132\"; transcript_id \"blat.proc18128.chain_25171\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39533667', '39533725', '100', '+', '.', 'gene_id \"chr8.g30281\"; transcript_id \"blat.proc18128.chain_13812\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10589897', '10590412', '100', '-', '.', 'gene_id \"chr8.g30389\"; transcript_id \"blat.proc4587.chain_13467\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '15675285', '15675322', '100', '-', '.', 'gene_id \"chr8.g30446\"; transcript_id \"blat.proc18128.chain_7900\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27757907', '27757970', '100', '+', '.', 'gene_id \"chr8.g30470\"; transcript_id \"blat.proc4005.chain_10042\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73245644', '73248070', '100', '-', '.', 'gene_id \"chr8.g30473\"; transcript_id \"blat.proc28144.chain_11172\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73456798', '73456843', '100', '+', '.', 'gene_id \"chr8.g30636\"; transcript_id \"blat.proc4587.chain_6591\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60593184', '60593256', '100', '-', '.', 'gene_id \"chr8.g30658\"; transcript_id \"blat.proc18128.chain_13173\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '16772849', '16772913', '100', '+', '.', 'gene_id \"chr8.g30741\"; transcript_id \"blat.proc28144.chain_7534\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '69314465', '69314515', '100', '-', '.', 'gene_id \"chr8.g30814\"; transcript_id \"blat.proc18128.chain_19236\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14857712', '14857743', '100', '+', '.', 'gene_id \"chr8.g30823\"; transcript_id \"blat.proc28144.chain_15905\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48166843', '48166890', '100', '-', '.', 'gene_id \"chr8.g30989\"; transcript_id \"blat.proc18128.chain_12689\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '31437179', '31437214', '100', '-', '.', 'gene_id \"chr8.g31039\"; transcript_id \"blat.proc4005.chain_19746\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '13135136', '13135166', '100', '+', '.', 'gene_id \"chr8.g31077\"; transcript_id \"blat.proc4005.chain_11007\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27720374', '27720416', '100', '+', '.', 'gene_id \"chr8.g31079\"; transcript_id \"blat.proc4005.chain_883\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68927683', '68927731', '100', '-', '.', 'gene_id \"chr8.g31081\"; transcript_id \"blat.proc28144.chain_13903\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37998236', '37998268', '100', '+', '.', 'gene_id \"chr8.g31100\"; transcript_id \"blat.proc28144.chain_13030\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14797800', '14797844', '100', '-', '.', 'gene_id \"chr8.g31107\"; transcript_id \"blat.proc4005.chain_7890\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39371327', '39371467', '100', '-', '.', 'gene_id \"chr8.g31156\"; transcript_id \"blat.proc4587.chain_4718\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39213611', '39213642', '100', '+', '.', 'gene_id \"chr8.g31185\"; transcript_id \"blat.proc28144.chain_1760\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1691820', '1692000', '100', '-', '.', 'gene_id \"chr8.g31193\"; transcript_id \"blat.proc18128.chain_8276\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '20982070', '20982109', '100', '-', '.', 'gene_id \"chr8.g31257\"; transcript_id \"blat.proc4005.chain_19285\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '68779932', '68779977', '100', '-', '.', 'gene_id \"chr8.g31321\"; transcript_id \"blat.proc18128.chain_13774\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '70654739', '70654774', '100', '+', '.', 'gene_id \"chr8.g31377\"; transcript_id \"blat.proc4005.chain_276\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '74014972', '74015013', '100', '-', '.', 'gene_id \"chr8.g31427\"; transcript_id \"blat.proc4005.chain_22294\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '74125557', '74125602', '100', '+', '.', 'gene_id \"chr8.g31510\"; transcript_id \"blat.proc4005.chain_22839\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '66961295', '66961334', '100', '+', '.', 'gene_id \"chr8.g31590\"; transcript_id \"blat.proc4005.chain_5382\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30486180', '30486223', '100', '-', '.', 'gene_id \"chr8.g31701\"; transcript_id \"blat.proc18128.chain_16729\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '22914074', '22914104', '100', '-', '.', 'gene_id \"chr8.g31718\"; transcript_id \"blat.proc28144.chain_19153\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '44685355', '44685422', '100', '-', '.', 'gene_id \"chr8.g31734\"; transcript_id \"blat.proc28144.chain_5700\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '72972973', '72973013', '100', '-', '.', 'gene_id \"chr8.g31796\"; transcript_id \"blat.proc28144.chain_7488\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6234732', '6234767', '100', '+', '.', 'gene_id \"chr8.g31932\"; transcript_id \"blat.proc28144.chain_3947\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '11822627', '11822670', '100', '+', '.', 'gene_id \"chr8.g31953\"; transcript_id \"blat.proc18128.chain_16086\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '50655182', '50655222', '100', '+', '.', 'gene_id \"chr8.g32007\"; transcript_id \"blat.proc28144.chain_5496\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '28763855', '28763892', '100', '+', '.', 'gene_id \"chr8.g32038\"; transcript_id \"blat.proc28144.chain_890\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29310978', '29311050', '100', '+', '.', 'gene_id \"chr8.g32111\"; transcript_id \"blat.proc4005.chain_13586\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48159096', '48159165', '100', '-', '.', 'gene_id \"chr8.g32184\"; transcript_id \"blat.proc4587.chain_13133\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '74184864', '74184980', '100', '-', '.', 'gene_id \"chr8.g32222\"; transcript_id \"blat.proc28144.chain_21296\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5141664', '5141700', '100', '+', '.', 'gene_id \"chr8.g32549\"; transcript_id \"blat.proc4587.chain_13991\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '65108673', '65108794', '100', '-', '.', 'gene_id \"chr8.g32622\"; transcript_id \"blat.proc18128.chain_20101\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '74142521', '74142571', '100', '+', '.', 'gene_id \"chr8.g32694\"; transcript_id \"blat.proc18128.chain_4988\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46662137', '46662226', '100', '-', '.', 'gene_id \"chr8.g32898\"; transcript_id \"blat.proc18128.chain_20306\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '63891728', '63891822', '100', '+', '.', 'gene_id \"chr8.g32908\"; transcript_id \"blat.proc4587.chain_10618\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '21218032', '21218236', '100', '+', '.', 'gene_id \"chr8.g32946\"; transcript_id \"blat.proc18128.chain_8423\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '34181434', '34181577', '100', '+', '.', 'gene_id \"chr8.g32953\"; transcript_id \"blat.proc18128.chain_3587\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46260637', '46260786', '100', '+', '.', 'gene_id \"chr8.g33022\"; transcript_id \"blat.proc4587.chain_7909\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6303671', '6303733', '100', '-', '.', 'gene_id \"chr8.g33044\"; transcript_id \"blat.proc4587.chain_3764\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '46660349', '46660396', '100', '-', '.', 'gene_id \"chr8.g33076\"; transcript_id \"blat.proc4005.chain_2447\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40361451', '40361490', '100', '+', '.', 'gene_id \"chr8.g33116\"; transcript_id \"blat.proc4005.chain_14196\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37349126', '37349172', '100', '-', '.', 'gene_id \"chr8.g33122\"; transcript_id \"blat.proc18128.chain_17415\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '29243077', '29243120', '100', '+', '.', 'gene_id \"chr8.g33174\"; transcript_id \"blat.proc4587.chain_5071\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '24575713', '24575824', '100', '+', '.', 'gene_id \"chr8.g33197\"; transcript_id \"blat.proc4005.chain_22700\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71583331', '71583373', '100', '+', '.', 'gene_id \"chr8.g33344\"; transcript_id \"blat.proc28144.chain_7406\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '48447773', '48447825', '100', '+', '.', 'gene_id \"chr8.g33516\"; transcript_id \"blat.proc28144.chain_19518\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57693688', '57693757', '100', '+', '.', 'gene_id \"chr8.g33560\"; transcript_id \"blat.proc28144.chain_3504\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40828826', '40828888', '100', '-', '.', 'gene_id \"chr8.g33578\"; transcript_id \"blat.proc18128.chain_1256\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '36470973', '36471014', '100', '-', '.', 'gene_id \"chr8.g33635\"; transcript_id \"blat.proc18128.chain_15846\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '3132287', '3132336', '100', '+', '.', 'gene_id \"chr8.g33716\"; transcript_id \"blat.proc18128.chain_14823\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '37850416', '37850460', '100', '-', '.', 'gene_id \"chr8.g33756\"; transcript_id \"blat.proc4587.chain_3331\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '18783332', '18783507', '100', '+', '.', 'gene_id \"chr8.g33881\"; transcript_id \"blat.proc28144.chain_5187\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '30716606', '30716638', '100', '+', '.', 'gene_id \"chr8.g33882\"; transcript_id \"blat.proc4005.chain_13524\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '45244814', '45244851', '100', '-', '.', 'gene_id \"chr8.g33913\"; transcript_id \"blat.proc4587.chain_12120\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '39970525', '39970575', '100', '+', '.', 'gene_id \"chr8.g34034\"; transcript_id \"blat.proc4587.chain_2733\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '27098303', '27098358', '100', '+', '.', 'gene_id \"chr8.g34038\"; transcript_id \"blat.proc18128.chain_6990\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10831301', '10831354', '100', '+', '.', 'gene_id \"chr8.g34130\"; transcript_id \"blat.proc18128.chain_3775\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71303519', '71303551', '100', '-', '.', 'gene_id \"chr8.g34224\"; transcript_id \"blat.proc18128.chain_16218\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '42335632', '42335674', '100', '-', '.', 'gene_id \"chr8.g34303\"; transcript_id \"blat.proc4587.chain_4482\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '5094850', '5094883', '100', '+', '.', 'gene_id \"chr8.g34345\"; transcript_id \"blat.proc18128.chain_17185\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14097179', '14097260', '100', '+', '.', 'gene_id \"chr8.g34410\"; transcript_id \"blat.proc18128.chain_6430\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '71474660', '71474712', '100', '-', '.', 'gene_id \"chr8.g34444\"; transcript_id \"blat.proc28144.chain_4687\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '8282199', '8282252', '100', '-', '.', 'gene_id \"chr8.g34494\"; transcript_id \"blat.proc28144.chain_1590\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '34673960', '34674002', '100', '+', '.', 'gene_id \"chr8.g34569\"; transcript_id \"blat.proc28144.chain_11801\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '65796292', '65796358', '100', '-', '.', 'gene_id \"chr8.g34585\"; transcript_id \"blat.proc28144.chain_8084\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '53397969', '53398020', '100', '+', '.', 'gene_id \"chr8.g34608\"; transcript_id \"blat.proc28144.chain_1392\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr8', 'PASA', 'exon', '28250655', '28250795', '100', '-', '.', 'gene_id \"chr8.g34620\"; transcript_id \"blat.proc18128.chain_13870\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '41528490', '41528547', '100', '-', '.', 'gene_id \"chr8.g34873\"; transcript_id \"blat.proc4587.chain_11193\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57950294', '57950325', '100', '+', '.', 'gene_id \"chr8.g34960\"; transcript_id \"blat.proc28144.chain_4097\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '6507642', '6507797', '100', '-', '.', 'gene_id \"chr8.g34967\"; transcript_id \"blat.proc18128.chain_21756\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '10169975', '10170018', '100', '-', '.', 'gene_id \"chr8.g35004\"; transcript_id \"blat.proc4587.chain_12459\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '14558285', '14558323', '100', '-', '.', 'gene_id \"chr8.g35090\"; transcript_id \"blat.proc18128.chain_15852\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '1025720', '1025949', '100', '+', '.', 'gene_id \"chr8.g35126\"; transcript_id \"blat.proc28144.chain_1778\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '73349102', '73349151', '100', '+', '.', 'gene_id \"chr8.g35167\"; transcript_id \"blat.proc18128.chain_4989\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '40715132', '40715203', '100', '-', '.', 'gene_id \"chr8.g35218\"; transcript_id \"blat.proc4587.chain_7691\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '55340825', '55340873', '100', '-', '.', 'gene_id \"chr8.g35260\"; transcript_id \"blat.proc4005.chain_1385\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '35606460', '35606584', '100', '-', '.', 'gene_id \"chr8.g35294\"; transcript_id \"blat.proc28144.chain_17978\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '57585552', '57585664', '100', '+', '.', 'gene_id \"chr8.g35308\"; transcript_id \"blat.proc18128.chain_25569\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '12232369', '12232528', '100', '+', '.', 'gene_id \"chr8.g35317\"; transcript_id \"blat.proc18128.chain_6952\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '12986018', '12986052', '100', '-', '.', 'gene_id \"chr8.g35419\"; transcript_id \"blat.proc18128.chain_21504\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '74040234', '74040263', '100', '+', '.', 'gene_id \"chr8.g35461\"; transcript_id \"blat.proc28144.chain_4526\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '64684214', '64684388', '100', '+', '.', 'gene_id \"chr8.g35472\"; transcript_id \"blat.proc28144.chain_16404\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '60351537', '60351693', '100', '-', '.', 'gene_id \"chr8.g35510\"; transcript_id \"blat.proc4587.chain_8191\";']\n",
      "negative:  ['chr8', 'PASA', 'exon', '72317188', '72317227', '100', '-', '.', 'gene_id \"chr8.g35608\"; transcript_id \"blat.proc18128.chain_22348\";']\n",
      "negative:  ['scaffold_1665', 'PASA', 'exon', '2358', '2455', '100', '+', '.', 'gene_id \"scaffold_1665.g17\"; transcript_id \"blat.proc18128.chain_15659\";']\n",
      "negative:  ['scaffold_1631', 'PASA', 'exon', '33811', '33868', '100', '-', '.', 'gene_id \"scaffold_1631.g8\"; transcript_id \"blat.proc4005.chain_6278\";']\n",
      "negative:  ['scaffold_1517', 'PASA', 'exon', '1', '64', '.', '-', '.', 'gene_id \"scaffold_1517.g1\"; transcript_id \"scaffold_1517.g1.i7\";']\n",
      "negative:  ['scaffold_1517', 'PASA', 'exon', '342', '409', '.', '-', '.', 'gene_id \"scaffold_1517.g1\"; transcript_id \"scaffold_1517.g1.i8\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78354218', '78354442', '.', '+', '.', 'gene_id \"chr2.g33387\"; transcript_id \"chr2.g33387.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78520681', '78520831', '.', '-', '.', 'gene_id \"chr2.g33450\"; transcript_id \"chr2.g33450.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78712458', '78712636', '.', '-', '.', 'gene_id \"chr2.g33624\"; transcript_id \"chr2.g33624.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78742680', '78742927', '.', '-', '.', 'gene_id \"chr2.g33660\"; transcript_id \"chr2.g33660.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78756608', '78756771', '.', '-', '.', 'gene_id \"chr2.g33671\"; transcript_id \"chr2.g33671.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78818490', '78818746', '.', '+', '.', 'gene_id \"chr2.g33731\"; transcript_id \"chr2.g33731.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78818755', '78818917', '.', '-', '.', 'gene_id \"chr2.g33732\"; transcript_id \"chr2.g33732.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '982419', '982698', '.', '-', '.', 'gene_id \"chr16.g265\"; transcript_id \"chr16.g265.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '982974', '983154', '.', '+', '.', 'gene_id \"chr16.g267\"; transcript_id \"chr16.g267.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78854586', '78854616', '.', '+', '.', 'gene_id \"chr2.g33792\"; transcript_id \"chr2.g33792.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78891431', '78891461', '.', '+', '.', 'gene_id \"chr2.g33819\"; transcript_id \"chr2.g33819.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '1348568', '1349105', '.', '-', '.', 'gene_id \"chr16.g522\"; transcript_id \"chr16.g522.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '1704862', '1704923', '.', '-', '.', 'gene_id \"chr16.g672\"; transcript_id \"chr16.g672.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79244096', '79244133', '.', '+', '.', 'gene_id \"chr2.g34142\"; transcript_id \"chr2.g34142.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79338578', '79338823', '.', '-', '.', 'gene_id \"chr2.g34255\"; transcript_id \"chr2.g34255.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '1858751', '1858927', '.', '-', '.', 'gene_id \"chr16.g845\"; transcript_id \"chr16.g845.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '1858964', '1859139', '.', '-', '.', 'gene_id \"chr16.g846\"; transcript_id \"chr16.g846.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79467578', '79467708', '.', '-', '.', 'gene_id \"chr2.g34390\"; transcript_id \"chr2.g34390.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79554314', '79554490', '.', '-', '.', 'gene_id \"chr2.g34459\"; transcript_id \"chr2.g34459.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79554547', '79554707', '.', '-', '.', 'gene_id \"chr2.g34460\"; transcript_id \"chr2.g34460.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79650390', '79650488', '.', '+', '.', 'gene_id \"chr2.g34492\"; transcript_id \"chr2.g34492.i6\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79650390', '79650586', '.', '+', '.', 'gene_id \"chr2.g34492\"; transcript_id \"chr2.g34492.i7\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '4750135', '4750244', '.', '-', '.', 'gene_id \"chr16.g1113\"; transcript_id \"chr16.g1113.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '4943299', '4943560', '.', '-', '.', 'gene_id \"chr16.g1136\"; transcript_id \"chr16.g1136.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '4943571', '4943793', '.', '-', '.', 'gene_id \"chr16.g1137\"; transcript_id \"chr16.g1137.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '5043360', '5043704', '.', '+', '.', 'gene_id \"chr16.g1195\"; transcript_id \"chr16.g1195.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '5132411', '5132464', '.', '-', '.', 'gene_id \"chr16.g1266\"; transcript_id \"chr16.g1266.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '80700567', '80700728', '.', '+', '.', 'gene_id \"chr2.g34872\"; transcript_id \"chr2.g34872.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '5449653', '5449808', '.', '-', '.', 'gene_id \"chr16.g1470\"; transcript_id \"chr16.g1470.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '80847233', '80847759', '.', '-', '.', 'gene_id \"chr2.g34989\"; transcript_id \"chr2.g34989.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '80877794', '80878172', '.', '+', '.', 'gene_id \"chr2.g35004\"; transcript_id \"chr2.g35004.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '5868446', '5868483', '.', '-', '.', 'gene_id \"chr16.g1537\"; transcript_id \"chr16.g1537.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '80959915', '80960093', '.', '-', '.', 'gene_id \"chr2.g35049\"; transcript_id \"chr2.g35049.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81342321', '81343786', '.', '-', '.', 'gene_id \"chr2.g35264\"; transcript_id \"chr2.g35264.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81347117', '81347369', '.', '+', '.', 'gene_id \"chr2.g35265\"; transcript_id \"chr2.g35265.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81430714', '81431007', '.', '+', '.', 'gene_id \"chr2.g35432\"; transcript_id \"chr2.g35432.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81430989', '81431160', '.', '-', '.', 'gene_id \"chr2.g35433\"; transcript_id \"chr2.g35433.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81437263', '81437524', '.', '-', '.', 'gene_id \"chr2.g35438\"; transcript_id \"chr2.g35438.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81437541', '81437939', '.', '-', '.', 'gene_id \"chr2.g35439\"; transcript_id \"chr2.g35439.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '7210086', '7210274', '.', '-', '.', 'gene_id \"chr16.g2096\"; transcript_id \"chr16.g2096.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82082291', '82082654', '.', '-', '.', 'gene_id \"chr2.g35852\"; transcript_id \"chr2.g35852.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82364670', '82365412', '.', '+', '.', 'gene_id \"chr2.g36096\"; transcript_id \"chr2.g36096.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82389664', '82389979', '.', '-', '.', 'gene_id \"chr2.g36107\"; transcript_id \"chr2.g36107.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82389886', '82389979', '.', '-', '.', 'gene_id \"chr2.g36107\"; transcript_id \"chr2.g36107.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82389889', '82389979', '.', '-', '.', 'gene_id \"chr2.g36107\"; transcript_id \"chr2.g36107.i3\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82420641', '82420829', '.', '-', '.', 'gene_id \"chr2.g36120\"; transcript_id \"chr2.g36120.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82569508', '82569732', '.', '+', '.', 'gene_id \"chr2.g36169\"; transcript_id \"chr2.g36169.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '8221750', '8221788', '.', '+', '.', 'gene_id \"chr16.g2638\"; transcript_id \"chr16.g2638.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82747652', '82747848', '.', '-', '.', 'gene_id \"chr2.g36338\"; transcript_id \"chr2.g36338.i1\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '82748003', '82748296', '.', '-', '.', 'gene_id \"chr2.g36339\"; transcript_id \"chr2.g36339.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '8259117', '8259199', '.', '+', '.', 'gene_id \"chr16.g2676\"; transcript_id \"chr16.g2676.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '8259165', '8259218', '.', '+', '.', 'gene_id \"chr16.g2676\"; transcript_id \"chr16.g2676.i2\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '21966540', '21966882', '100', '+', '.', 'gene_id \"chr2.g36673\"; transcript_id \"blat.proc4587.chain_9415\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43549786', '43550153', '100', '+', '.', 'gene_id \"chr2.g36722\"; transcript_id \"blat.proc4587.chain_9137\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '13334802', '13334873', '100', '+', '.', 'gene_id \"chr2.g36968\"; transcript_id \"blat.proc18128.chain_18760\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '7038380', '7038946', '100', '+', '.', 'gene_id \"chr2.g37100\"; transcript_id \"blat.proc18128.chain_4175\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81450193', '81450350', '100', '+', '.', 'gene_id \"chr2.g37171\"; transcript_id \"blat.proc18128.chain_18829\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '37079219', '37079253', '100', '-', '.', 'gene_id \"chr2.g37258\"; transcript_id \"blat.proc4005.chain_9978\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '26950036', '26950170', '100', '-', '.', 'gene_id \"chr2.g37260\"; transcript_id \"blat.proc4587.chain_8682\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '57388123', '57388152', '100', '-', '.', 'gene_id \"chr2.g37502\"; transcript_id \"blat.proc18128.chain_5003\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43550190', '43550393', '100', '-', '.', 'gene_id \"chr2.g37624\"; transcript_id \"blat.proc4587.chain_9141\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '3590787', '3591691', '100', '+', '.', 'gene_id \"chr2.g37637\"; transcript_id \"blat.proc18128.chain_5914\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '60601960', '60602005', '100', '-', '.', 'gene_id \"chr2.g37886\"; transcript_id \"blat.proc4005.chain_5077\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '15353221', '15353447', '100', '+', '.', 'gene_id \"chr2.g38185\"; transcript_id \"blat.proc18128.chain_21900\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79622939', '79622972', '100', '+', '.', 'gene_id \"chr2.g38371\"; transcript_id \"blat.proc18128.chain_7940\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '53632622', '53632677', '100', '+', '.', 'gene_id \"chr2.g38455\"; transcript_id \"blat.proc4587.chain_2717\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '42463239', '42463763', '100', '+', '.', 'gene_id \"chr2.g38496\"; transcript_id \"blat.proc28144.chain_7994\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '69473217', '69473261', '100', '-', '.', 'gene_id \"chr2.g38579\"; transcript_id \"blat.proc4587.chain_9112\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '52211213', '52211244', '100', '-', '.', 'gene_id \"chr2.g38587\"; transcript_id \"blat.proc4005.chain_17453\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '38265665', '38265880', '100', '-', '.', 'gene_id \"chr2.g38624\"; transcript_id \"blat.proc18128.chain_22321\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '69977736', '69977776', '100', '-', '.', 'gene_id \"chr2.g38907\"; transcript_id \"blat.proc18128.chain_21605\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '62592568', '62592600', '100', '-', '.', 'gene_id \"chr2.g39026\"; transcript_id \"blat.proc18128.chain_7331\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '2292044', '2292088', '100', '+', '.', 'gene_id \"chr2.g39191\"; transcript_id \"blat.proc4005.chain_22434\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43532977', '43533159', '100', '+', '.', 'gene_id \"chr2.g39258\"; transcript_id \"blat.proc4587.chain_9140\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '68919705', '68919812', '100', '-', '.', 'gene_id \"chr2.g39317\"; transcript_id \"blat.proc28144.chain_14238\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '67552640', '67552716', '100', '-', '.', 'gene_id \"chr2.g39701\"; transcript_id \"blat.proc4587.chain_11253\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '56017385', '56017447', '100', '+', '.', 'gene_id \"chr2.g39894\"; transcript_id \"blat.proc4587.chain_7940\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1393102', '1393187', '100', '+', '.', 'gene_id \"chr2.g39910\"; transcript_id \"blat.proc18128.chain_17142\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27477442', '27477491', '100', '+', '.', 'gene_id \"chr2.g39965\"; transcript_id \"blat.proc18128.chain_24072\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '11327088', '11327118', '100', '-', '.', 'gene_id \"chr2.g40256\"; transcript_id \"blat.proc18128.chain_21016\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '24347616', '24347661', '100', '-', '.', 'gene_id \"chr2.g40277\"; transcript_id \"blat.proc4587.chain_7404\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '730610', '730643', '100', '+', '.', 'gene_id \"chr2.g40837\"; transcript_id \"blat.proc4587.chain_8143\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79885584', '79885615', '100', '-', '.', 'gene_id \"chr2.g40935\"; transcript_id \"blat.proc4005.chain_9844\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '56383202', '56383496', '100', '+', '.', 'gene_id \"chr2.g41207\"; transcript_id \"blat.proc18128.chain_10\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '41468996', '41469047', '100', '+', '.', 'gene_id \"chr2.g41562\"; transcript_id \"blat.proc18128.chain_10952\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '27361304', '27361335', '100', '+', '.', 'gene_id \"chr2.g41591\"; transcript_id \"blat.proc18128.chain_7550\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4379411', '4379454', '100', '+', '.', 'gene_id \"chr2.g41830\"; transcript_id \"blat.proc18128.chain_23298\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '23977813', '23977890', '100', '-', '.', 'gene_id \"chr2.g41877\"; transcript_id \"blat.proc18128.chain_11206\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '71231343', '71231377', '100', '-', '.', 'gene_id \"chr2.g41882\"; transcript_id \"blat.proc4587.chain_38\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '11902313', '11902438', '100', '+', '.', 'gene_id \"chr2.g42201\"; transcript_id \"blat.proc18128.chain_1027\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '43533541', '43533744', '100', '-', '.', 'gene_id \"chr2.g42582\"; transcript_id \"blat.proc4587.chain_9142\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '79168591', '79168667', '100', '+', '.', 'gene_id \"chr2.g42642\"; transcript_id \"blat.proc4005.chain_1247\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '81506509', '81506749', '100', '-', '.', 'gene_id \"chr2.g42693\"; transcript_id \"blat.proc4587.chain_309\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '48662085', '48662116', '100', '+', '.', 'gene_id \"chr2.g42840\"; transcript_id \"blat.proc28144.chain_1752\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '15622509', '15622569', '100', '-', '.', 'gene_id \"chr2.g42964\"; transcript_id \"blat.proc18128.chain_17017\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '910117', '910184', '100', '-', '.', 'gene_id \"chr2.g43139\"; transcript_id \"blat.proc18128.chain_13339\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '4302069', '4302291', '100', '-', '.', 'gene_id \"chr2.g43342\"; transcript_id \"blat.proc4587.chain_1754\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '1930874', '1930907', '100', '-', '.', 'gene_id \"chr2.g43617\"; transcript_id \"blat.proc18128.chain_26862\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '70839020', '70839049', '100', '-', '.', 'gene_id \"chr2.g43656\"; transcript_id \"blat.proc18128.chain_12830\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '39297323', '39297352', '100', '+', '.', 'gene_id \"chr2.g43995\"; transcript_id \"blat.proc4587.chain_13545\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '15829724', '15829890', '100', '-', '.', 'gene_id \"chr2.g44183\"; transcript_id \"blat.proc18128.chain_19725\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '66212735', '66212785', '100', '+', '.', 'gene_id \"chr2.g44316\"; transcript_id \"blat.proc18128.chain_10355\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '6148954', '6149008', '100', '-', '.', 'gene_id \"chr2.g44391\"; transcript_id \"blat.proc18128.chain_10288\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '33469863', '33469972', '100', '-', '.', 'gene_id \"chr2.g44916\"; transcript_id \"blat.proc18128.chain_11140\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '60241340', '60241379', '100', '-', '.', 'gene_id \"chr2.g45168\"; transcript_id \"blat.proc18128.chain_18549\";']\n",
      "negative:  ['chr2', 'PASA', 'exon', '78617501', '78617541', '100', '-', '.', 'gene_id \"chr2.g45177\"; transcript_id \"blat.proc28144.chain_15628\";']\n",
      "negative:  ['scaffold_1581', 'PASA', 'exon', '5759', '5936', '.', '-', '.', 'gene_id \"scaffold_1581.g8\"; transcript_id \"scaffold_1581.g8.i1\";']\n",
      "negative:  ['scaffold_1581', 'PASA', 'exon', '3621', '3663', '100', '+', '.', 'gene_id \"scaffold_1581.g51\"; transcript_id \"blat.proc4005.chain_1847\";']\n",
      "negative:  ['scaffold_1109', 'PASA', 'exon', '2583', '3092', '.', '+', '.', 'gene_id \"scaffold_1109.g1\"; transcript_id \"scaffold_1109.g1.i1\";']\n",
      "negative:  ['scaffold_1109', 'PASA', 'exon', '2224', '2508', '100', '-', '.', 'gene_id \"scaffold_1109.g13\"; transcript_id \"blat.proc4587.chain_13073\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '33191', '33542', '.', '+', '.', 'gene_id \"chr19.g5\"; transcript_id \"chr19.g5.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23255', '23751', '.', '+', '.', 'gene_id \"chr19.g7\"; transcript_id \"chr19.g7.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52140', '52434', '.', '+', '.', 'gene_id \"chr19.g8\"; transcript_id \"chr19.g8.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38458', '41857', '.', '+', '.', 'gene_id \"chr19.g9\"; transcript_id \"chr19.g9.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18750', '18969', '.', '-', '.', 'gene_id \"chr19.g16\"; transcript_id \"chr19.g16.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19437', '19911', '.', '-', '.', 'gene_id \"chr19.g18\"; transcript_id \"chr19.g18.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22688', '22867', '.', '+', '.', 'gene_id \"chr19.g20\"; transcript_id \"chr19.g20.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22890', '23136', '.', '+', '.', 'gene_id \"chr19.g21\"; transcript_id \"chr19.g21.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '44589', '45144', '.', '+', '.', 'gene_id \"chr19.g25\"; transcript_id \"chr19.g25.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45427', '46513', '.', '+', '.', 'gene_id \"chr19.g26\"; transcript_id \"chr19.g26.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '49476', '50301', '.', '-', '.', 'gene_id \"chr19.g29\"; transcript_id \"chr19.g29.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '50786', '51197', '.', '+', '.', 'gene_id \"chr19.g30\"; transcript_id \"chr19.g30.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '58845', '59003', '.', '-', '.', 'gene_id \"chr19.g32\"; transcript_id \"chr19.g32.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '92307', '92479', '.', '-', '.', 'gene_id \"chr19.g41\"; transcript_id \"chr19.g41.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '94462', '94813', '.', '-', '.', 'gene_id \"chr19.g44\"; transcript_id \"chr19.g44.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '95144', '95350', '.', '-', '.', 'gene_id \"chr19.g46\"; transcript_id \"chr19.g46.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '95409', '95573', '.', '+', '.', 'gene_id \"chr19.g47\"; transcript_id \"chr19.g47.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '97787', '98017', '.', '+', '.', 'gene_id \"chr19.g49\"; transcript_id \"chr19.g49.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '99682', '99994', '.', '+', '.', 'gene_id \"chr19.g50\"; transcript_id \"chr19.g50.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '99983', '100174', '.', '-', '.', 'gene_id \"chr19.g51\"; transcript_id \"chr19.g51.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '112117', '112287', '.', '+', '.', 'gene_id \"chr19.g54\"; transcript_id \"chr19.g54.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '112325', '112476', '.', '+', '.', 'gene_id \"chr19.g55\"; transcript_id \"chr19.g55.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '116287', '116891', '.', '-', '.', 'gene_id \"chr19.g57\"; transcript_id \"chr19.g57.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '118142', '118669', '.', '-', '.', 'gene_id \"chr19.g60\"; transcript_id \"chr19.g60.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '119007', '119161', '.', '-', '.', 'gene_id \"chr19.g61\"; transcript_id \"chr19.g61.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '126927', '127529', '.', '-', '.', 'gene_id \"chr19.g65\"; transcript_id \"chr19.g65.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '148255', '148543', '.', '-', '.', 'gene_id \"chr19.g75\"; transcript_id \"chr19.g75.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '214413', '215822', '.', '+', '.', 'gene_id \"chr19.g87\"; transcript_id \"chr19.g87.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '218774', '219349', '.', '+', '.', 'gene_id \"chr19.g88\"; transcript_id \"chr19.g88.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '232917', '233091', '.', '+', '.', 'gene_id \"chr19.g93\"; transcript_id \"chr19.g93.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '235338', '235519', '.', '-', '.', 'gene_id \"chr19.g95\"; transcript_id \"chr19.g95.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '236884', '237044', '.', '+', '.', 'gene_id \"chr19.g97\"; transcript_id \"chr19.g97.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '239113', '239389', '.', '-', '.', 'gene_id \"chr19.g99\"; transcript_id \"chr19.g99.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '238958', '239120', '.', '+', '.', 'gene_id \"chr19.g100\"; transcript_id \"chr19.g100.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '244015', '244497', '.', '-', '.', 'gene_id \"chr19.g104\"; transcript_id \"chr19.g104.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '247288', '247560', '.', '-', '.', 'gene_id \"chr19.g106\"; transcript_id \"chr19.g106.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '253756', '254479', '.', '-', '.', 'gene_id \"chr19.g113\"; transcript_id \"chr19.g113.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '258179', '258376', '.', '+', '.', 'gene_id \"chr19.g118\"; transcript_id \"chr19.g118.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '261226', '262024', '.', '+', '.', 'gene_id \"chr19.g121\"; transcript_id \"chr19.g121.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '262933', '263176', '.', '-', '.', 'gene_id \"chr19.g124\"; transcript_id \"chr19.g124.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr19', 'PASA', 'exon', '307382', '307616', '.', '+', '.', 'gene_id \"chr19.g130\"; transcript_id \"chr19.g130.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1067049', '1067226', '.', '+', '.', 'gene_id \"chr19.g152\"; transcript_id \"chr19.g152.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1079090', '1079946', '.', '+', '.', 'gene_id \"chr19.g152\"; transcript_id \"chr19.g152.i6\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1079090', '1079540', '.', '+', '.', 'gene_id \"chr19.g152\"; transcript_id \"chr19.g152.i7\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1095618', '1095777', '.', '+', '.', 'gene_id \"chr19.g152\"; transcript_id \"chr19.g152.i8\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1095756', '1096435', '.', '-', '.', 'gene_id \"chr19.g155\"; transcript_id \"chr19.g155.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '892465', '892781', '.', '+', '.', 'gene_id \"chr19.g168\"; transcript_id \"chr19.g168.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '900382', '900545', '.', '+', '.', 'gene_id \"chr19.g173\"; transcript_id \"chr19.g173.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '903831', '904084', '.', '-', '.', 'gene_id \"chr19.g174\"; transcript_id \"chr19.g174.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '908507', '908966', '.', '-', '.', 'gene_id \"chr19.g177\"; transcript_id \"chr19.g177.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '996357', '996618', '.', '-', '.', 'gene_id \"chr19.g194\"; transcript_id \"chr19.g194.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '997502', '997699', '.', '+', '.', 'gene_id \"chr19.g195\"; transcript_id \"chr19.g195.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1000657', '1000828', '.', '-', '.', 'gene_id \"chr19.g196\"; transcript_id \"chr19.g196.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1013352', '1013503', '.', '-', '.', 'gene_id \"chr19.g201\"; transcript_id \"chr19.g201.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1049978', '1050273', '.', '-', '.', 'gene_id \"chr19.g204\"; transcript_id \"chr19.g204.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1058430', '1059246', '.', '+', '.', 'gene_id \"chr19.g212\"; transcript_id \"chr19.g212.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1072644', '1072887', '.', '-', '.', 'gene_id \"chr19.g217\"; transcript_id \"chr19.g217.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1081214', '1081420', '.', '+', '.', 'gene_id \"chr19.g222\"; transcript_id \"chr19.g222.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1085082', '1085232', '.', '+', '.', 'gene_id \"chr19.g225\"; transcript_id \"chr19.g225.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1103788', '1103968', '.', '+', '.', 'gene_id \"chr19.g236\"; transcript_id \"chr19.g236.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1104876', '1105790', '.', '+', '.', 'gene_id \"chr19.g237\"; transcript_id \"chr19.g237.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1108827', '1109006', '.', '-', '.', 'gene_id \"chr19.g241\"; transcript_id \"chr19.g241.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1108985', '1109152', '.', '+', '.', 'gene_id \"chr19.g242\"; transcript_id \"chr19.g242.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1111204', '1111512', '.', '-', '.', 'gene_id \"chr19.g245\"; transcript_id \"chr19.g245.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1118830', '1119411', '.', '+', '.', 'gene_id \"chr19.g248\"; transcript_id \"chr19.g248.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1121295', '1121493', '.', '-', '.', 'gene_id \"chr19.g254\"; transcript_id \"chr19.g254.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1142582', '1142814', '.', '+', '.', 'gene_id \"chr19.g263\"; transcript_id \"chr19.g263.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1156704', '1156898', '.', '+', '.', 'gene_id \"chr19.g269\"; transcript_id \"chr19.g269.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1161189', '1161393', '.', '+', '.', 'gene_id \"chr19.g272\"; transcript_id \"chr19.g272.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1161356', '1161544', '.', '+', '.', 'gene_id \"chr19.g272\"; transcript_id \"chr19.g272.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1161618', '1161801', '.', '-', '.', 'gene_id \"chr19.g273\"; transcript_id \"chr19.g273.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1164964', '1165153', '.', '-', '.', 'gene_id \"chr19.g275\"; transcript_id \"chr19.g275.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1168916', '1169147', '.', '+', '.', 'gene_id \"chr19.g280\"; transcript_id \"chr19.g280.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1171450', '1171853', '.', '+', '.', 'gene_id \"chr19.g285\"; transcript_id \"chr19.g285.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1172505', '1172854', '.', '+', '.', 'gene_id \"chr19.g289\"; transcript_id \"chr19.g289.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1174343', '1174675', '.', '+', '.', 'gene_id \"chr19.g290\"; transcript_id \"chr19.g290.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1177962', '1178281', '.', '-', '.', 'gene_id \"chr19.g292\"; transcript_id \"chr19.g292.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1178763', '1178932', '.', '-', '.', 'gene_id \"chr19.g294\"; transcript_id \"chr19.g294.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1183042', '1183262', '.', '+', '.', 'gene_id \"chr19.g299\"; transcript_id \"chr19.g299.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1351078', '1351275', '.', '-', '.', 'gene_id \"chr19.g316\"; transcript_id \"chr19.g316.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1342646', '1342951', '.', '-', '.', 'gene_id \"chr19.g318\"; transcript_id \"chr19.g318.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1344354', '1344435', '.', '+', '.', 'gene_id \"chr19.g322\"; transcript_id \"chr19.g322.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1344396', '1344435', '.', '-', '.', 'gene_id \"chr19.g323\"; transcript_id \"chr19.g323.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1346020', '1346230', '.', '+', '.', 'gene_id \"chr19.g325\"; transcript_id \"chr19.g325.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1356432', '1356600', '.', '+', '.', 'gene_id \"chr19.g327\"; transcript_id \"chr19.g327.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1361182', '1361358', '.', '+', '.', 'gene_id \"chr19.g333\"; transcript_id \"chr19.g333.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1362199', '1362407', '.', '-', '.', 'gene_id \"chr19.g334\"; transcript_id \"chr19.g334.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1362678', '1362871', '.', '-', '.', 'gene_id \"chr19.g335\"; transcript_id \"chr19.g335.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1363121', '1363309', '.', '+', '.', 'gene_id \"chr19.g337\"; transcript_id \"chr19.g337.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1368049', '1368265', '.', '-', '.', 'gene_id \"chr19.g339\"; transcript_id \"chr19.g339.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1375797', '1376012', '.', '-', '.', 'gene_id \"chr19.g340\"; transcript_id \"chr19.g340.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1376549', '1376724', '.', '+', '.', 'gene_id \"chr19.g341\"; transcript_id \"chr19.g341.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1400572', '1400828', '.', '+', '.', 'gene_id \"chr19.g349\"; transcript_id \"chr19.g349.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1577248', '1577411', '.', '+', '.', 'gene_id \"chr19.g367\"; transcript_id \"chr19.g367.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1657842', '1658005', '.', '-', '.', 'gene_id \"chr19.g374\"; transcript_id \"chr19.g374.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1658130', '1658450', '.', '-', '.', 'gene_id \"chr19.g375\"; transcript_id \"chr19.g375.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1691526', '1691680', '.', '-', '.', 'gene_id \"chr19.g382\"; transcript_id \"chr19.g382.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1713594', '1713752', '.', '-', '.', 'gene_id \"chr19.g386\"; transcript_id \"chr19.g386.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1720943', '1721157', '.', '-', '.', 'gene_id \"chr19.g387\"; transcript_id \"chr19.g387.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1724161', '1724328', '.', '+', '.', 'gene_id \"chr19.g388\"; transcript_id \"chr19.g388.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1743072', '1743311', '.', '-', '.', 'gene_id \"chr19.g395\"; transcript_id \"chr19.g395.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1747333', '1747490', '.', '+', '.', 'gene_id \"chr19.g396\"; transcript_id \"chr19.g396.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1760412', '1760545', '.', '+', '.', 'gene_id \"chr19.g400\"; transcript_id \"chr19.g400.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1769888', '1770153', '.', '-', '.', 'gene_id \"chr19.g401\"; transcript_id \"chr19.g401.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1822572', '1822866', '.', '-', '.', 'gene_id \"chr19.g410\"; transcript_id \"chr19.g410.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1827829', '1827998', '.', '-', '.', 'gene_id \"chr19.g412\"; transcript_id \"chr19.g412.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1828194', '1828408', '.', '-', '.', 'gene_id \"chr19.g413\"; transcript_id \"chr19.g413.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1831625', '1831830', '.', '+', '.', 'gene_id \"chr19.g414\"; transcript_id \"chr19.g414.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1842258', '1842466', '.', '+', '.', 'gene_id \"chr19.g423\"; transcript_id \"chr19.g423.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1841915', '1842274', '.', '+', '.', 'gene_id \"chr19.g425\"; transcript_id \"chr19.g425.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1846289', '1846457', '.', '-', '.', 'gene_id \"chr19.g426\"; transcript_id \"chr19.g426.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1869791', '1869998', '.', '+', '.', 'gene_id \"chr19.g431\"; transcript_id \"chr19.g431.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1873739', '1873993', '.', '-', '.', 'gene_id \"chr19.g432\"; transcript_id \"chr19.g432.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1922612', '1922845', '.', '+', '.', 'gene_id \"chr19.g438\"; transcript_id \"chr19.g438.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1922462', '1922614', '.', '+', '.', 'gene_id \"chr19.g439\"; transcript_id \"chr19.g439.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1959446', '1959902', '.', '+', '.', 'gene_id \"chr19.g444\"; transcript_id \"chr19.g444.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1967815', '1967981', '.', '-', '.', 'gene_id \"chr19.g445\"; transcript_id \"chr19.g445.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2020999', '2021410', '.', '+', '.', 'gene_id \"chr19.g452\"; transcript_id \"chr19.g452.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2031777', '2031994', '.', '+', '.', 'gene_id \"chr19.g460\"; transcript_id \"chr19.g460.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2041037', '2041234', '.', '+', '.', 'gene_id \"chr19.g461\"; transcript_id \"chr19.g461.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2048139', '2048705', '.', '+', '.', 'gene_id \"chr19.g463\"; transcript_id \"chr19.g463.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2056144', '2056735', '.', '+', '.', 'gene_id \"chr19.g468\"; transcript_id \"chr19.g468.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2096011', '2096263', '.', '-', '.', 'gene_id \"chr19.g474\"; transcript_id \"chr19.g474.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2143184', '2143337', '.', '+', '.', 'gene_id \"chr19.g479\"; transcript_id \"chr19.g479.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2144522', '2144927', '.', '-', '.', 'gene_id \"chr19.g482\"; transcript_id \"chr19.g482.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2152686', '2152829', '.', '-', '.', 'gene_id \"chr19.g488\"; transcript_id \"chr19.g488.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2155148', '2155361', '.', '-', '.', 'gene_id \"chr19.g489\"; transcript_id \"chr19.g489.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2161891', '2162084', '.', '-', '.', 'gene_id \"chr19.g500\"; transcript_id \"chr19.g500.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '9889275', '9889578', '.', '+', '.', 'gene_id \"chr16.g3984\"; transcript_id \"chr16.g3984.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2172426', '2172755', '.', '-', '.', 'gene_id \"chr19.g509\"; transcript_id \"chr19.g509.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2174140', '2174439', '.', '+', '.', 'gene_id \"chr19.g511\"; transcript_id \"chr19.g511.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2183517', '2183735', '.', '+', '.', 'gene_id \"chr19.g514\"; transcript_id \"chr19.g514.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2252785', '2253681', '.', '+', '.', 'gene_id \"chr19.g522\"; transcript_id \"chr19.g522.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2186754', '2186952', '.', '-', '.', 'gene_id \"chr19.g526\"; transcript_id \"chr19.g526.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2186975', '2187231', '.', '-', '.', 'gene_id \"chr19.g527\"; transcript_id \"chr19.g527.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2189623', '2190035', '.', '-', '.', 'gene_id \"chr19.g528\"; transcript_id \"chr19.g528.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2192600', '2193424', '.', '+', '.', 'gene_id \"chr19.g532\"; transcript_id \"chr19.g532.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2193700', '2194351', '.', '-', '.', 'gene_id \"chr19.g533\"; transcript_id \"chr19.g533.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2198526', '2198789', '.', '-', '.', 'gene_id \"chr19.g535\"; transcript_id \"chr19.g535.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2218630', '2218817', '.', '+', '.', 'gene_id \"chr19.g538\"; transcript_id \"chr19.g538.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2227696', '2227870', '.', '-', '.', 'gene_id \"chr19.g547\"; transcript_id \"chr19.g547.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2228153', '2228283', '.', '+', '.', 'gene_id \"chr19.g548\"; transcript_id \"chr19.g548.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2229268', '2229433', '.', '+', '.', 'gene_id \"chr19.g550\"; transcript_id \"chr19.g550.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2229671', '2230056', '.', '-', '.', 'gene_id \"chr19.g553\"; transcript_id \"chr19.g553.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2234423', '2234750', '.', '-', '.', 'gene_id \"chr19.g556\"; transcript_id \"chr19.g556.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2237150', '2237340', '.', '+', '.', 'gene_id \"chr19.g557\"; transcript_id \"chr19.g557.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2241168', '2241335', '.', '+', '.', 'gene_id \"chr19.g559\"; transcript_id \"chr19.g559.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2241623', '2241824', '.', '+', '.', 'gene_id \"chr19.g560\"; transcript_id \"chr19.g560.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2241331', '2241635', '.', '-', '.', 'gene_id \"chr19.g561\"; transcript_id \"chr19.g561.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2247435', '2247740', '.', '-', '.', 'gene_id \"chr19.g566\"; transcript_id \"chr19.g566.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2249252', '2249702', '.', '+', '.', 'gene_id \"chr19.g570\"; transcript_id \"chr19.g570.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2293725', '2293962', '.', '+', '.', 'gene_id \"chr19.g586\"; transcript_id \"chr19.g586.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2314604', '2314792', '.', '+', '.', 'gene_id \"chr19.g588\"; transcript_id \"chr19.g588.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2302140', '2302392', '.', '+', '.', 'gene_id \"chr19.g591\"; transcript_id \"chr19.g591.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2321036', '2321190', '.', '+', '.', 'gene_id \"chr19.g599\"; transcript_id \"chr19.g599.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2322530', '2322734', '.', '+', '.', 'gene_id \"chr19.g602\"; transcript_id \"chr19.g602.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2340948', '2341124', '.', '-', '.', 'gene_id \"chr19.g609\"; transcript_id \"chr19.g609.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2389862', '2390000', '.', '-', '.', 'gene_id \"chr19.g612\"; transcript_id \"chr19.g612.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2357924', '2358085', '.', '-', '.', 'gene_id \"chr19.g613\"; transcript_id \"chr19.g613.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2389368', '2389523', '.', '-', '.', 'gene_id \"chr19.g618\"; transcript_id \"chr19.g618.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2398721', '2398904', '.', '-', '.', 'gene_id \"chr19.g621\"; transcript_id \"chr19.g621.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2402194', '2402347', '.', '-', '.', 'gene_id \"chr19.g622\"; transcript_id \"chr19.g622.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2423312', '2423522', '.', '+', '.', 'gene_id \"chr19.g623\"; transcript_id \"chr19.g623.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2443710', '2443973', '.', '-', '.', 'gene_id \"chr19.g628\"; transcript_id \"chr19.g628.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2445410', '2445639', '.', '+', '.', 'gene_id \"chr19.g629\"; transcript_id \"chr19.g629.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2449137', '2449304', '.', '+', '.', 'gene_id \"chr19.g630\"; transcript_id \"chr19.g630.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2458543', '2458864', '.', '+', '.', 'gene_id \"chr19.g637\"; transcript_id \"chr19.g637.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2528120', '2528463', '.', '-', '.', 'gene_id \"chr19.g657\"; transcript_id \"chr19.g657.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2544450', '2544603', '.', '-', '.', 'gene_id \"chr19.g661\"; transcript_id \"chr19.g661.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2544638', '2544809', '.', '-', '.', 'gene_id \"chr19.g662\"; transcript_id \"chr19.g662.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2557515', '2557738', '.', '-', '.', 'gene_id \"chr19.g670\"; transcript_id \"chr19.g670.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2557744', '2558321', '.', '-', '.', 'gene_id \"chr19.g672\"; transcript_id \"chr19.g672.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2558748', '2558941', '.', '-', '.', 'gene_id \"chr19.g674\"; transcript_id \"chr19.g674.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2567672', '2568331', '.', '-', '.', 'gene_id \"chr19.g684\"; transcript_id \"chr19.g684.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2568379', '2569269', '.', '+', '.', 'gene_id \"chr19.g685\"; transcript_id \"chr19.g685.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2580238', '2581435', '.', '+', '.', 'gene_id \"chr19.g695\"; transcript_id \"chr19.g695.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2579997', '2580260', '.', '+', '.', 'gene_id \"chr19.g697\"; transcript_id \"chr19.g697.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2588732', '2588914', '.', '-', '.', 'gene_id \"chr19.g700\"; transcript_id \"chr19.g700.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2592620', '2592841', '.', '+', '.', 'gene_id \"chr19.g710\"; transcript_id \"chr19.g710.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2601499', '2601880', '.', '-', '.', 'gene_id \"chr19.g713\"; transcript_id \"chr19.g713.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2603416', '2603750', '.', '+', '.', 'gene_id \"chr19.g716\"; transcript_id \"chr19.g716.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2616368', '2616550', '.', '+', '.', 'gene_id \"chr19.g724\"; transcript_id \"chr19.g724.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2617124', '2617269', '.', '+', '.', 'gene_id \"chr19.g726\"; transcript_id \"chr19.g726.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2651702', '2651855', '.', '+', '.', 'gene_id \"chr19.g731\"; transcript_id \"chr19.g731.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2651554', '2651706', '.', '-', '.', 'gene_id \"chr19.g732\"; transcript_id \"chr19.g732.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2656378', '2657144', '.', '-', '.', 'gene_id \"chr19.g734\"; transcript_id \"chr19.g734.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2662532', '2662870', '.', '-', '.', 'gene_id \"chr19.g740\"; transcript_id \"chr19.g740.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2679370', '2679673', '.', '+', '.', 'gene_id \"chr19.g748\"; transcript_id \"chr19.g748.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2681328', '2681662', '.', '+', '.', 'gene_id \"chr19.g751\"; transcript_id \"chr19.g751.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2696495', '2696831', '.', '+', '.', 'gene_id \"chr19.g758\"; transcript_id \"chr19.g758.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2696117', '2696507', '.', '+', '.', 'gene_id \"chr19.g760\"; transcript_id \"chr19.g760.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2701612', '2701890', '.', '+', '.', 'gene_id \"chr19.g761\"; transcript_id \"chr19.g761.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2707702', '2708469', '.', '+', '.', 'gene_id \"chr19.g767\"; transcript_id \"chr19.g767.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2709272', '2709627', '.', '-', '.', 'gene_id \"chr19.g768\"; transcript_id \"chr19.g768.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2710506', '2710689', '.', '+', '.', 'gene_id \"chr19.g770\"; transcript_id \"chr19.g770.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2724759', '2725035', '.', '+', '.', 'gene_id \"chr19.g784\"; transcript_id \"chr19.g784.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2732238', '2732460', '.', '+', '.', 'gene_id \"chr19.g788\"; transcript_id \"chr19.g788.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2737923', '2738168', '.', '+', '.', 'gene_id \"chr19.g790\"; transcript_id \"chr19.g790.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2738225', '2738408', '.', '+', '.', 'gene_id \"chr19.g791\"; transcript_id \"chr19.g791.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2747448', '2747622', '.', '+', '.', 'gene_id \"chr19.g797\"; transcript_id \"chr19.g797.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2755575', '2755781', '.', '-', '.', 'gene_id \"chr19.g800\"; transcript_id \"chr19.g800.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2758141', '2758315', '.', '+', '.', 'gene_id \"chr19.g802\"; transcript_id \"chr19.g802.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2763772', '2763960', '.', '+', '.', 'gene_id \"chr19.g808\"; transcript_id \"chr19.g808.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2769701', '2769878', '.', '-', '.', 'gene_id \"chr19.g809\"; transcript_id \"chr19.g809.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2770396', '2770576', '.', '-', '.', 'gene_id \"chr19.g810\"; transcript_id \"chr19.g810.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2772534', '2773121', '.', '-', '.', 'gene_id \"chr19.g813\"; transcript_id \"chr19.g813.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2783545', '2783725', '.', '+', '.', 'gene_id \"chr19.g819\"; transcript_id \"chr19.g819.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2794360', '2794532', '.', '-', '.', 'gene_id \"chr19.g820\"; transcript_id \"chr19.g820.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2819169', '2819538', '.', '-', '.', 'gene_id \"chr19.g825\"; transcript_id \"chr19.g825.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2819618', '2819899', '.', '-', '.', 'gene_id \"chr19.g826\"; transcript_id \"chr19.g826.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2821446', '2821634', '.', '+', '.', 'gene_id \"chr19.g827\"; transcript_id \"chr19.g827.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2822091', '2824364', '.', '-', '.', 'gene_id \"chr19.g829\"; transcript_id \"chr19.g829.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2827327', '2827495', '.', '-', '.', 'gene_id \"chr19.g832\"; transcript_id \"chr19.g832.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2831329', '2831561', '.', '+', '.', 'gene_id \"chr19.g835\"; transcript_id \"chr19.g835.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2833355', '2833587', '.', '+', '.', 'gene_id \"chr19.g837\"; transcript_id \"chr19.g837.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2837365', '2837533', '.', '-', '.', 'gene_id \"chr19.g838\"; transcript_id \"chr19.g838.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2844289', '2844478', '.', '+', '.', 'gene_id \"chr19.g840\"; transcript_id \"chr19.g840.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2847767', '2847988', '.', '+', '.', 'gene_id \"chr19.g844\"; transcript_id \"chr19.g844.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2866873', '2867131', '.', '-', '.', 'gene_id \"chr19.g848\"; transcript_id \"chr19.g848.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2884004', '2884258', '.', '-', '.', 'gene_id \"chr19.g853\"; transcript_id \"chr19.g853.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2889514', '2889699', '.', '-', '.', 'gene_id \"chr19.g854\"; transcript_id \"chr19.g854.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2892009', '2893506', '.', '-', '.', 'gene_id \"chr19.g857\"; transcript_id \"chr19.g857.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2896572', '2896936', '.', '-', '.', 'gene_id \"chr19.g858\"; transcript_id \"chr19.g858.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2898352', '2898642', '.', '-', '.', 'gene_id \"chr19.g859\"; transcript_id \"chr19.g859.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2900614', '2900821', '.', '+', '.', 'gene_id \"chr19.g860\"; transcript_id \"chr19.g860.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2900910', '2901307', '.', '+', '.', 'gene_id \"chr19.g861\"; transcript_id \"chr19.g861.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2902057', '2902232', '.', '+', '.', 'gene_id \"chr19.g863\"; transcript_id \"chr19.g863.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2902517', '2902694', '.', '+', '.', 'gene_id \"chr19.g864\"; transcript_id \"chr19.g864.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2907124', '2907351', '.', '-', '.', 'gene_id \"chr19.g867\"; transcript_id \"chr19.g867.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2911221', '2911340', '.', '+', '.', 'gene_id \"chr19.g869\"; transcript_id \"chr19.g869.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2910776', '2911244', '.', '+', '.', 'gene_id \"chr19.g869\"; transcript_id \"chr19.g869.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2916691', '2917120', '.', '+', '.', 'gene_id \"chr19.g875\"; transcript_id \"chr19.g875.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2918748', '2918861', '.', '+', '.', 'gene_id \"chr19.g877\"; transcript_id \"chr19.g877.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2922592', '2923480', '.', '+', '.', 'gene_id \"chr19.g884\"; transcript_id \"chr19.g884.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2923914', '2924266', '.', '-', '.', 'gene_id \"chr19.g885\"; transcript_id \"chr19.g885.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2925252', '2926057', '.', '+', '.', 'gene_id \"chr19.g886\"; transcript_id \"chr19.g886.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '10587020', '10587233', '.', '+', '.', 'gene_id \"chr16.g4424\"; transcript_id \"chr16.g4424.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2926785', '2927072', '.', '+', '.', 'gene_id \"chr19.g889\"; transcript_id \"chr19.g889.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2932145', '2932718', '.', '+', '.', 'gene_id \"chr19.g895\"; transcript_id \"chr19.g895.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2938157', '2938379', '.', '-', '.', 'gene_id \"chr19.g902\"; transcript_id \"chr19.g902.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2944773', '2945047', '.', '-', '.', 'gene_id \"chr19.g906\"; transcript_id \"chr19.g906.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2945878', '2946080', '.', '-', '.', 'gene_id \"chr19.g909\"; transcript_id \"chr19.g909.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '10617999', '10618219', '.', '-', '.', 'gene_id \"chr16.g4454\"; transcript_id \"chr16.g4454.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2946194', '2946367', '.', '-', '.', 'gene_id \"chr19.g910\"; transcript_id \"chr19.g910.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '10618447', '10618695', '.', '-', '.', 'gene_id \"chr16.g4455\"; transcript_id \"chr16.g4455.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2948967', '2949290', '.', '-', '.', 'gene_id \"chr19.g913\"; transcript_id \"chr19.g913.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2951568', '2951736', '.', '+', '.', 'gene_id \"chr19.g917\"; transcript_id \"chr19.g917.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2951785', '2952004', '.', '-', '.', 'gene_id \"chr19.g918\"; transcript_id \"chr19.g918.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2957060', '2957123', '.', '-', '.', 'gene_id \"chr19.g923\"; transcript_id \"chr19.g923.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '10628976', '10629157', '.', '+', '.', 'gene_id \"chr16.g4468\"; transcript_id \"chr16.g4468.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2957985', '2958208', '.', '-', '.', 'gene_id \"chr19.g924\"; transcript_id \"chr19.g924.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2968934', '2969098', '.', '+', '.', 'gene_id \"chr19.g931\"; transcript_id \"chr19.g931.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2969892', '2970031', '.', '-', '.', 'gene_id \"chr19.g934\"; transcript_id \"chr19.g934.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2978970', '2979156', '.', '-', '.', 'gene_id \"chr19.g936\"; transcript_id \"chr19.g936.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2979882', '2980359', '.', '+', '.', 'gene_id \"chr19.g937\"; transcript_id \"chr19.g937.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2983458', '2983661', '.', '+', '.', 'gene_id \"chr19.g938\"; transcript_id \"chr19.g938.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2991246', '2991620', '.', '+', '.', 'gene_id \"chr19.g941\"; transcript_id \"chr19.g941.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2992330', '2992495', '.', '-', '.', 'gene_id \"chr19.g942\"; transcript_id \"chr19.g942.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2996669', '2997680', '.', '-', '.', 'gene_id \"chr19.g944\"; transcript_id \"chr19.g944.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3007390', '3007827', '.', '+', '.', 'gene_id \"chr19.g947\"; transcript_id \"chr19.g947.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3012621', '3012849', '.', '+', '.', 'gene_id \"chr19.g948\"; transcript_id \"chr19.g948.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3020189', '3020454', '.', '+', '.', 'gene_id \"chr19.g953\"; transcript_id \"chr19.g953.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3022516', '3022707', '.', '-', '.', 'gene_id \"chr19.g956\"; transcript_id \"chr19.g956.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3024923', '3025361', '.', '+', '.', 'gene_id \"chr19.g959\"; transcript_id \"chr19.g959.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3031135', '3031300', '.', '+', '.', 'gene_id \"chr19.g965\"; transcript_id \"chr19.g965.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3032428', '3032581', '.', '-', '.', 'gene_id \"chr19.g968\"; transcript_id \"chr19.g968.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3034874', '3035021', '.', '-', '.', 'gene_id \"chr19.g972\"; transcript_id \"chr19.g972.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3129846', '3130174', '.', '+', '.', 'gene_id \"chr19.g996\"; transcript_id \"chr19.g996.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3208743', '3208908', '.', '+', '.', 'gene_id \"chr19.g1010\"; transcript_id \"chr19.g1010.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3215107', '3215308', '.', '-', '.', 'gene_id \"chr19.g1011\"; transcript_id \"chr19.g1011.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3192843', '3193076', '.', '-', '.', 'gene_id \"chr19.g1016\"; transcript_id \"chr19.g1016.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3204923', '3205211', '.', '-', '.', 'gene_id \"chr19.g1020\"; transcript_id \"chr19.g1020.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3210769', '3211506', '.', '+', '.', 'gene_id \"chr19.g1024\"; transcript_id \"chr19.g1024.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3220872', '3221030', '.', '-', '.', 'gene_id \"chr19.g1029\"; transcript_id \"chr19.g1029.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3226162', '3226315', '.', '+', '.', 'gene_id \"chr19.g1030\"; transcript_id \"chr19.g1030.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3239765', '3239948', '.', '+', '.', 'gene_id \"chr19.g1031\"; transcript_id \"chr19.g1031.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3252250', '3252426', '.', '-', '.', 'gene_id \"chr19.g1034\"; transcript_id \"chr19.g1034.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3275021', '3275454', '.', '-', '.', 'gene_id \"chr19.g1041\"; transcript_id \"chr19.g1041.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3301043', '3301343', '.', '-', '.', 'gene_id \"chr19.g1047\"; transcript_id \"chr19.g1047.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3306311', '3306812', '.', '+', '.', 'gene_id \"chr19.g1048\"; transcript_id \"chr19.g1048.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3308389', '3309190', '.', '-', '.', 'gene_id \"chr19.g1050\"; transcript_id \"chr19.g1050.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3349727', '3351039', '.', '+', '.', 'gene_id \"chr19.g1054\"; transcript_id \"chr19.g1054.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3394526', '3394973', '.', '+', '.', 'gene_id \"chr19.g1057\"; transcript_id \"chr19.g1057.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3318903', '3319143', '.', '+', '.', 'gene_id \"chr19.g1064\"; transcript_id \"chr19.g1064.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3319503', '3319721', '.', '-', '.', 'gene_id \"chr19.g1065\"; transcript_id \"chr19.g1065.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3322012', '3322188', '.', '-', '.', 'gene_id \"chr19.g1067\"; transcript_id \"chr19.g1067.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3344871', '3345054', '.', '+', '.', 'gene_id \"chr19.g1069\"; transcript_id \"chr19.g1069.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3346117', '3346294', '.', '+', '.', 'gene_id \"chr19.g1070\"; transcript_id \"chr19.g1070.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3353227', '3353802', '.', '-', '.', 'gene_id \"chr19.g1073\"; transcript_id \"chr19.g1073.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3354270', '3354422', '.', '-', '.', 'gene_id \"chr19.g1074\"; transcript_id \"chr19.g1074.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3354655', '3354862', '.', '+', '.', 'gene_id \"chr19.g1076\"; transcript_id \"chr19.g1076.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3362672', '3362830', '.', '-', '.', 'gene_id \"chr19.g1079\"; transcript_id \"chr19.g1079.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3390605', '3390780', '.', '-', '.', 'gene_id \"chr19.g1084\"; transcript_id \"chr19.g1084.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3402142', '3402344', '.', '+', '.', 'gene_id \"chr19.g1089\"; transcript_id \"chr19.g1089.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3404117', '3404140', '.', '+', '.', 'gene_id \"chr19.g1089\"; transcript_id \"chr19.g1089.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3403913', '3404139', '.', '+', '.', 'gene_id \"chr19.g1089\"; transcript_id \"chr19.g1089.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3405547', '3405897', '.', '+', '.', 'gene_id \"chr19.g1091\"; transcript_id \"chr19.g1091.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3404478', '3404971', '.', '-', '.', 'gene_id \"chr19.g1096\"; transcript_id \"chr19.g1096.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3405030', '3405333', '.', '-', '.', 'gene_id \"chr19.g1097\"; transcript_id \"chr19.g1097.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3424084', '3424242', '.', '+', '.', 'gene_id \"chr19.g1104\"; transcript_id \"chr19.g1104.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3438012', '3438269', '.', '-', '.', 'gene_id \"chr19.g1115\"; transcript_id \"chr19.g1115.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3445254', '3445417', '.', '-', '.', 'gene_id \"chr19.g1116\"; transcript_id \"chr19.g1116.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3447743', '3447878', '.', '+', '.', 'gene_id \"chr19.g1118\"; transcript_id \"chr19.g1118.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3478319', '3478501', '.', '+', '.', 'gene_id \"chr19.g1129\"; transcript_id \"chr19.g1129.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3484190', '3484381', '.', '+', '.', 'gene_id \"chr19.g1131\"; transcript_id \"chr19.g1131.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3486514', '3486743', '.', '-', '.', 'gene_id \"chr19.g1137\"; transcript_id \"chr19.g1137.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3586620', '3587421', '.', '-', '.', 'gene_id \"chr19.g1138\"; transcript_id \"chr19.g1138.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3534314', '3535080', '.', '-', '.', 'gene_id \"chr19.g1141\"; transcript_id \"chr19.g1141.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3493867', '3494036', '.', '-', '.', 'gene_id \"chr19.g1145\"; transcript_id \"chr19.g1145.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3497312', '3497342', '.', '+', '.', 'gene_id \"chr19.g1147\"; transcript_id \"chr19.g1147.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '11112656', '11112903', '.', '+', '.', 'gene_id \"chr16.g4742\"; transcript_id \"chr16.g4742.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3505486', '3505652', '.', '-', '.', 'gene_id \"chr19.g1153\"; transcript_id \"chr19.g1153.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '11113131', '11113352', '.', '+', '.', 'gene_id \"chr16.g4743\"; transcript_id \"chr16.g4743.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3506331', '3506497', '.', '-', '.', 'gene_id \"chr19.g1154\"; transcript_id \"chr19.g1154.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3509980', '3510803', '.', '+', '.', 'gene_id \"chr19.g1159\"; transcript_id \"chr19.g1159.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3524325', '3524689', '.', '-', '.', 'gene_id \"chr19.g1165\"; transcript_id \"chr19.g1165.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3544220', '3544542', '.', '+', '.', 'gene_id \"chr19.g1173\"; transcript_id \"chr19.g1173.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3551676', '3551939', '.', '+', '.', 'gene_id \"chr19.g1181\"; transcript_id \"chr19.g1181.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3551388', '3551686', '.', '+', '.', 'gene_id \"chr19.g1183\"; transcript_id \"chr19.g1183.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3552697', '3554464', '.', '-', '.', 'gene_id \"chr19.g1185\"; transcript_id \"chr19.g1185.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3576715', '3576924', '.', '+', '.', 'gene_id \"chr19.g1191\"; transcript_id \"chr19.g1191.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3590542', '3590828', '.', '-', '.', 'gene_id \"chr19.g1196\"; transcript_id \"chr19.g1196.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3590542', '3590828', '.', '-', '.', 'gene_id \"chr19.g1196\"; transcript_id \"chr19.g1196.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3599587', '3600008', '.', '-', '.', 'gene_id \"chr19.g1199\"; transcript_id \"chr19.g1199.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3597783', '3598081', '.', '-', '.', 'gene_id \"chr19.g1203\"; transcript_id \"chr19.g1203.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3600500', '3600991', '.', '-', '.', 'gene_id \"chr19.g1205\"; transcript_id \"chr19.g1205.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3603562', '3604396', '.', '-', '.', 'gene_id \"chr19.g1208\"; transcript_id \"chr19.g1208.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3615008', '3615385', '.', '-', '.', 'gene_id \"chr19.g1214\"; transcript_id \"chr19.g1214.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3617984', '3618263', '.', '-', '.', 'gene_id \"chr19.g1216\"; transcript_id \"chr19.g1216.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3619975', '3620218', '.', '+', '.', 'gene_id \"chr19.g1217\"; transcript_id \"chr19.g1217.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3621910', '3622039', '.', '-', '.', 'gene_id \"chr19.g1219\"; transcript_id \"chr19.g1219.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3623834', '3625749', '.', '+', '.', 'gene_id \"chr19.g1220\"; transcript_id \"chr19.g1220.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3623000', '3623838', '.', '-', '.', 'gene_id \"chr19.g1222\"; transcript_id \"chr19.g1222.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3761948', '3762351', '.', '+', '.', 'gene_id \"chr19.g1239\"; transcript_id \"chr19.g1239.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3762344', '3762575', '.', '-', '.', 'gene_id \"chr19.g1240\"; transcript_id \"chr19.g1240.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3927306', '3927437', '.', '+', '.', 'gene_id \"chr19.g1263\"; transcript_id \"chr19.g1263.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4056092', '4056276', '.', '+', '.', 'gene_id \"chr19.g1273\"; transcript_id \"chr19.g1273.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4232076', '4232107', '.', '+', '.', 'gene_id \"chr19.g1283\"; transcript_id \"chr19.g1283.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4936501', '4936909', '.', '-', '.', 'gene_id \"chr19.g1341\"; transcript_id \"chr19.g1341.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4965588', '4965745', '.', '+', '.', 'gene_id \"chr19.g1347\"; transcript_id \"chr19.g1347.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4970511', '4970858', '.', '-', '.', 'gene_id \"chr19.g1351\"; transcript_id \"chr19.g1351.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4975774', '4975939', '.', '-', '.', 'gene_id \"chr19.g1352\"; transcript_id \"chr19.g1352.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5271446', '5271600', '.', '-', '.', 'gene_id \"chr19.g1364\"; transcript_id \"chr19.g1364.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5306535', '5306969', '.', '+', '.', 'gene_id \"chr19.g1369\"; transcript_id \"chr19.g1369.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5322885', '5323014', '.', '+', '.', 'gene_id \"chr19.g1371\"; transcript_id \"chr19.g1371.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5451888', '5452000', '.', '-', '.', 'gene_id \"chr19.g1377\"; transcript_id \"chr19.g1377.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '12490709', '12490749', '.', '-', '.', 'gene_id \"chr16.g4959\"; transcript_id \"chr16.g4959.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5583898', '5583999', '.', '+', '.', 'gene_id \"chr19.g1388\"; transcript_id \"chr19.g1388.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '5585024', '5585105', '.', '+', '.', 'gene_id \"chr19.g1388\"; transcript_id \"chr19.g1388.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '6023681', '6023887', '.', '-', '.', 'gene_id \"chr19.g1413\"; transcript_id \"chr19.g1413.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '6059857', '6060322', '.', '-', '.', 'gene_id \"chr19.g1416\"; transcript_id \"chr19.g1416.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7515028', '7515361', '.', '-', '.', 'gene_id \"chr19.g1484\"; transcript_id \"chr19.g1484.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7706909', '7707175', '.', '-', '.', 'gene_id \"chr19.g1494\"; transcript_id \"chr19.g1494.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7863835', '7864013', '.', '+', '.', 'gene_id \"chr19.g1509\"; transcript_id \"chr19.g1509.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7865728', '7865908', '.', '+', '.', 'gene_id \"chr19.g1511\"; transcript_id \"chr19.g1511.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7866093', '7866420', '.', '+', '.', 'gene_id \"chr19.g1513\"; transcript_id \"chr19.g1513.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7868900', '7868939', '.', '-', '.', 'gene_id \"chr19.g1516\"; transcript_id \"chr19.g1516.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7876854', '7877146', '.', '+', '.', 'gene_id \"chr19.g1519\"; transcript_id \"chr19.g1519.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7877723', '7877901', '.', '+', '.', 'gene_id \"chr19.g1520\"; transcript_id \"chr19.g1520.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7878141', '7878353', '.', '-', '.', 'gene_id \"chr19.g1522\"; transcript_id \"chr19.g1522.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8701045', '8701301', '.', '+', '.', 'gene_id \"chr19.g1563\"; transcript_id \"chr19.g1563.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8714229', '8714815', '.', '-', '.', 'gene_id \"chr19.g1564\"; transcript_id \"chr19.g1564.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8725342', '8725601', '.', '-', '.', 'gene_id \"chr19.g1566\"; transcript_id \"chr19.g1566.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8833604', '8833849', '.', '-', '.', 'gene_id \"chr19.g1573\"; transcript_id \"chr19.g1573.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '13152322', '13152502', '.', '+', '.', 'gene_id \"chr16.g5181\"; transcript_id \"chr16.g5181.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '13179102', '13179133', '.', '-', '.', 'gene_id \"chr16.g5197\"; transcript_id \"chr16.g5197.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11754635', '11754947', '.', '-', '.', 'gene_id \"chr19.g1699\"; transcript_id \"chr19.g1699.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11757265', '11757425', '.', '-', '.', 'gene_id \"chr19.g1701\"; transcript_id \"chr19.g1701.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11758699', '11759258', '.', '-', '.', 'gene_id \"chr19.g1703\"; transcript_id \"chr19.g1703.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11759253', '11759568', '.', '+', '.', 'gene_id \"chr19.g1704\"; transcript_id \"chr19.g1704.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11832516', '11832671', '.', '+', '.', 'gene_id \"chr19.g1716\"; transcript_id \"chr19.g1716.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11834807', '11835045', '.', '+', '.', 'gene_id \"chr19.g1717\"; transcript_id \"chr19.g1717.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11846623', '11846981', '.', '-', '.', 'gene_id \"chr19.g1722\"; transcript_id \"chr19.g1722.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11863965', '11864137', '.', '-', '.', 'gene_id \"chr19.g1730\"; transcript_id \"chr19.g1730.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11868836', '11869035', '.', '-', '.', 'gene_id \"chr19.g1734\"; transcript_id \"chr19.g1734.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11918345', '11918802', '.', '-', '.', 'gene_id \"chr19.g1743\"; transcript_id \"chr19.g1743.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11984789', '11985003', '.', '+', '.', 'gene_id \"chr19.g1747\"; transcript_id \"chr19.g1747.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12000047', '12000277', '.', '+', '.', 'gene_id \"chr19.g1756\"; transcript_id \"chr19.g1756.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12060681', '12060786', '.', '-', '.', 'gene_id \"chr19.g1763\"; transcript_id \"chr19.g1763.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12076436', '12076707', '.', '-', '.', 'gene_id \"chr19.g1771\"; transcript_id \"chr19.g1771.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12078794', '12078946', '.', '+', '.', 'gene_id \"chr19.g1775\"; transcript_id \"chr19.g1775.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12117899', '12118127', '.', '+', '.', 'gene_id \"chr19.g1782\"; transcript_id \"chr19.g1782.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12118656', '12118821', '.', '+', '.', 'gene_id \"chr19.g1785\"; transcript_id \"chr19.g1785.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12152222', '12152280', '.', '-', '.', 'gene_id \"chr19.g1795\"; transcript_id \"chr19.g1795.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12185465', '12185668', '.', '-', '.', 'gene_id \"chr19.g1801\"; transcript_id \"chr19.g1801.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12187804', '12188302', '.', '-', '.', 'gene_id \"chr19.g1803\"; transcript_id \"chr19.g1803.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12189713', '12190330', '.', '+', '.', 'gene_id \"chr19.g1804\"; transcript_id \"chr19.g1804.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12192883', '12193142', '.', '+', '.', 'gene_id \"chr19.g1805\"; transcript_id \"chr19.g1805.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12192700', '12192905', '.', '-', '.', 'gene_id \"chr19.g1806\"; transcript_id \"chr19.g1806.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12202037', '12202668', '.', '+', '.', 'gene_id \"chr19.g1810\"; transcript_id \"chr19.g1810.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12204280', '12204585', '.', '+', '.', 'gene_id \"chr19.g1812\"; transcript_id \"chr19.g1812.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12222240', '12222488', '.', '+', '.', 'gene_id \"chr19.g1814\"; transcript_id \"chr19.g1814.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12231116', '12231327', '.', '-', '.', 'gene_id \"chr19.g1818\"; transcript_id \"chr19.g1818.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12231721', '12231899', '.', '+', '.', 'gene_id \"chr19.g1820\"; transcript_id \"chr19.g1820.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12234699', '12234962', '.', '-', '.', 'gene_id \"chr19.g1823\"; transcript_id \"chr19.g1823.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12252488', '12252833', '.', '-', '.', 'gene_id \"chr19.g1832\"; transcript_id \"chr19.g1832.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12256238', '12256675', '.', '-', '.', 'gene_id \"chr19.g1836\"; transcript_id \"chr19.g1836.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12260578', '12260903', '.', '-', '.', 'gene_id \"chr19.g1837\"; transcript_id \"chr19.g1837.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12266157', '12266478', '.', '+', '.', 'gene_id \"chr19.g1843\"; transcript_id \"chr19.g1843.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12274593', '12274764', '.', '-', '.', 'gene_id \"chr19.g1846\"; transcript_id \"chr19.g1846.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12280913', '12281178', '.', '-', '.', 'gene_id \"chr19.g1850\"; transcript_id \"chr19.g1850.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12283484', '12283689', '.', '+', '.', 'gene_id \"chr19.g1853\"; transcript_id \"chr19.g1853.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12285735', '12285954', '.', '+', '.', 'gene_id \"chr19.g1856\"; transcript_id \"chr19.g1856.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12291814', '12292091', '.', '-', '.', 'gene_id \"chr19.g1860\"; transcript_id \"chr19.g1860.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12292703', '12292921', '.', '+', '.', 'gene_id \"chr19.g1862\"; transcript_id \"chr19.g1862.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12301389', '12301582', '.', '-', '.', 'gene_id \"chr19.g1866\"; transcript_id \"chr19.g1866.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12303456', '12303714', '.', '-', '.', 'gene_id \"chr19.g1868\"; transcript_id \"chr19.g1868.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12304504', '12304685', '.', '+', '.', 'gene_id \"chr19.g1869\"; transcript_id \"chr19.g1869.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12306002', '12306177', '.', '-', '.', 'gene_id \"chr19.g1870\"; transcript_id \"chr19.g1870.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '13738967', '13739015', '.', '+', '.', 'gene_id \"chr16.g5503\"; transcript_id \"chr16.g5503.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12522721', '12522819', '.', '+', '.', 'gene_id \"chr19.g1899\"; transcript_id \"chr19.g1899.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12777924', '12778436', '.', '+', '.', 'gene_id \"chr19.g1941\"; transcript_id \"chr19.g1941.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12791297', '12792602', '.', '+', '.', 'gene_id \"chr19.g1942\"; transcript_id \"chr19.g1942.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12779227', '12779405', '.', '-', '.', 'gene_id \"chr19.g1946\"; transcript_id \"chr19.g1946.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12804014', '12804219', '.', '-', '.', 'gene_id \"chr19.g1956\"; transcript_id \"chr19.g1956.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12805637', '12805834', '.', '-', '.', 'gene_id \"chr19.g1960\"; transcript_id \"chr19.g1960.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12826183', '12826357', '.', '-', '.', 'gene_id \"chr19.g1968\"; transcript_id \"chr19.g1968.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12829880', '12830256', '.', '+', '.', 'gene_id \"chr19.g1972\"; transcript_id \"chr19.g1972.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12847026', '12847498', '.', '+', '.', 'gene_id \"chr19.g1975\"; transcript_id \"chr19.g1975.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12967738', '12967872', '.', '+', '.', 'gene_id \"chr19.g1978\"; transcript_id \"chr19.g1978.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12862627', '12863198', '.', '+', '.', 'gene_id \"chr19.g1980\"; transcript_id \"chr19.g1980.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12847518', '12847809', '.', '-', '.', 'gene_id \"chr19.g1988\"; transcript_id \"chr19.g1988.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12849586', '12849926', '.', '-', '.', 'gene_id \"chr19.g1992\"; transcript_id \"chr19.g1992.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12860169', '12860447', '.', '+', '.', 'gene_id \"chr19.g2002\"; transcript_id \"chr19.g2002.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12860673', '12860831', '.', '-', '.', 'gene_id \"chr19.g2003\"; transcript_id \"chr19.g2003.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12862017', '12862202', '.', '+', '.', 'gene_id \"chr19.g2004\"; transcript_id \"chr19.g2004.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12890450', '12890862', '.', '-', '.', 'gene_id \"chr19.g2016\"; transcript_id \"chr19.g2016.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12892432', '12892622', '.', '+', '.', 'gene_id \"chr19.g2017\"; transcript_id \"chr19.g2017.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12901957', '12902212', '.', '+', '.', 'gene_id \"chr19.g2025\"; transcript_id \"chr19.g2025.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12903534', '12904021', '.', '+', '.', 'gene_id \"chr19.g2028\"; transcript_id \"chr19.g2028.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12904927', '12905090', '.', '+', '.', 'gene_id \"chr19.g2030\"; transcript_id \"chr19.g2030.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12914286', '12914721', '.', '-', '.', 'gene_id \"chr19.g2037\"; transcript_id \"chr19.g2037.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12916936', '12917684', '.', '-', '.', 'gene_id \"chr19.g2038\"; transcript_id \"chr19.g2038.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12924837', '12925123', '.', '-', '.', 'gene_id \"chr19.g2047\"; transcript_id \"chr19.g2047.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12956248', '12956418', '.', '+', '.', 'gene_id \"chr19.g2058\"; transcript_id \"chr19.g2058.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12963360', '12964531', '.', '-', '.', 'gene_id \"chr19.g2064\"; transcript_id \"chr19.g2064.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12968022', '12968414', '.', '-', '.', 'gene_id \"chr19.g2067\"; transcript_id \"chr19.g2067.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12970618', '12970846', '.', '+', '.', 'gene_id \"chr19.g2071\"; transcript_id \"chr19.g2071.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12980790', '12980947', '.', '+', '.', 'gene_id \"chr19.g2076\"; transcript_id \"chr19.g2076.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12988806', '12988969', '.', '-', '.', 'gene_id \"chr19.g2078\"; transcript_id \"chr19.g2078.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12993466', '12993787', '.', '+', '.', 'gene_id \"chr19.g2081\"; transcript_id \"chr19.g2081.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13004728', '13004906', '.', '-', '.', 'gene_id \"chr19.g2092\"; transcript_id \"chr19.g2092.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12998280', '12998448', '.', '-', '.', 'gene_id \"chr19.g2093\"; transcript_id \"chr19.g2093.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13012864', '13013882', '.', '+', '.', 'gene_id \"chr19.g2100\"; transcript_id \"chr19.g2100.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13014699', '13015100', '.', '-', '.', 'gene_id \"chr19.g2104\"; transcript_id \"chr19.g2104.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13024186', '13024458', '.', '+', '.', 'gene_id \"chr19.g2112\"; transcript_id \"chr19.g2112.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13024966', '13025131', '.', '+', '.', 'gene_id \"chr19.g2113\"; transcript_id \"chr19.g2113.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13025281', '13025466', '.', '-', '.', 'gene_id \"chr19.g2115\"; transcript_id \"chr19.g2115.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13030317', '13030804', '.', '-', '.', 'gene_id \"chr19.g2117\"; transcript_id \"chr19.g2117.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13033067', '13033282', '.', '+', '.', 'gene_id \"chr19.g2119\"; transcript_id \"chr19.g2119.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13033948', '13034098', '.', '+', '.', 'gene_id \"chr19.g2121\"; transcript_id \"chr19.g2121.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13036027', '13036219', '.', '+', '.', 'gene_id \"chr19.g2124\"; transcript_id \"chr19.g2124.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13044327', '13044502', '.', '-', '.', 'gene_id \"chr19.g2129\"; transcript_id \"chr19.g2129.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13062253', '13062474', '.', '+', '.', 'gene_id \"chr19.g2143\"; transcript_id \"chr19.g2143.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13077203', '13077487', '.', '-', '.', 'gene_id \"chr19.g2153\"; transcript_id \"chr19.g2153.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13103319', '13103475', '.', '-', '.', 'gene_id \"chr19.g2166\"; transcript_id \"chr19.g2166.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13134562', '13134749', '.', '+', '.', 'gene_id \"chr19.g2170\"; transcript_id \"chr19.g2170.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13156518', '13156670', '.', '-', '.', 'gene_id \"chr19.g2176\"; transcript_id \"chr19.g2176.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13160555', '13160732', '.', '+', '.', 'gene_id \"chr19.g2179\"; transcript_id \"chr19.g2179.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13175144', '13175327', '.', '-', '.', 'gene_id \"chr19.g2187\"; transcript_id \"chr19.g2187.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13179193', '13179458', '.', '+', '.', 'gene_id \"chr19.g2194\"; transcript_id \"chr19.g2194.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13179461', '13179776', '.', '-', '.', 'gene_id \"chr19.g2195\"; transcript_id \"chr19.g2195.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13179461', '13179760', '.', '+', '.', 'gene_id \"chr19.g2196\"; transcript_id \"chr19.g2196.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13179754', '13179938', '.', '+', '.', 'gene_id \"chr19.g2197\"; transcript_id \"chr19.g2197.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13228116', '13228456', '.', '+', '.', 'gene_id \"chr19.g2207\"; transcript_id \"chr19.g2207.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13260833', '13261047', '.', '+', '.', 'gene_id \"chr19.g2217\"; transcript_id \"chr19.g2217.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13267165', '13267579', '.', '-', '.', 'gene_id \"chr19.g2222\"; transcript_id \"chr19.g2222.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '14352881', '14353224', '.', '+', '.', 'gene_id \"chr16.g5863\"; transcript_id \"chr16.g5863.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13333925', '13334046', '.', '+', '.', 'gene_id \"chr19.g2237\"; transcript_id \"chr19.g2237.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13344363', '13344478', '.', '+', '.', 'gene_id \"chr19.g2239\"; transcript_id \"chr19.g2239.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13539848', '13540018', '.', '+', '.', 'gene_id \"chr19.g2254\"; transcript_id \"chr19.g2254.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13569590', '13569752', '.', '-', '.', 'gene_id \"chr19.g2256\"; transcript_id \"chr19.g2256.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13574692', '13574923', '.', '+', '.', 'gene_id \"chr19.g2259\"; transcript_id \"chr19.g2259.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13573544', '13573713', '.', '+', '.', 'gene_id \"chr19.g2259\"; transcript_id \"chr19.g2259.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14571107', '14571249', '.', '+', '.', 'gene_id \"chr19.g2331\"; transcript_id \"chr19.g2331.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14575695', '14575854', '.', '+', '.', 'gene_id \"chr19.g2333\"; transcript_id \"chr19.g2333.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14579977', '14580178', '.', '+', '.', 'gene_id \"chr19.g2335\"; transcript_id \"chr19.g2335.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14722953', '14723184', '.', '-', '.', 'gene_id \"chr19.g2349\"; transcript_id \"chr19.g2349.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14912192', '14912671', '.', '-', '.', 'gene_id \"chr19.g2355\"; transcript_id \"chr19.g2355.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15221479', '15222535', '.', '-', '.', 'gene_id \"chr19.g2373\"; transcript_id \"chr19.g2373.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15318056', '15318444', '.', '+', '.', 'gene_id \"chr19.g2388\"; transcript_id \"chr19.g2388.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15391611', '15391762', '.', '+', '.', 'gene_id \"chr19.g2392\"; transcript_id \"chr19.g2392.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15392122', '15392364', '.', '+', '.', 'gene_id \"chr19.g2393\"; transcript_id \"chr19.g2393.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15392652', '15396964', '.', '-', '.', 'gene_id \"chr19.g2394\"; transcript_id \"chr19.g2394.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15971583', '15971758', '.', '+', '.', 'gene_id \"chr19.g2427\"; transcript_id \"chr19.g2427.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16144687', '16144731', '.', '+', '.', 'gene_id \"chr19.g2440\"; transcript_id \"chr19.g2440.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16144692', '16144767', '.', '-', '.', 'gene_id \"chr19.g2442\"; transcript_id \"chr19.g2442.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16219283', '16219460', '.', '-', '.', 'gene_id \"chr19.g2445\"; transcript_id \"chr19.g2445.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16219455', '16219751', '.', '-', '.', 'gene_id \"chr19.g2446\"; transcript_id \"chr19.g2446.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16512257', '16512434', '.', '+', '.', 'gene_id \"chr19.g2467\"; transcript_id \"chr19.g2467.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16614406', '16614769', '.', '+', '.', 'gene_id \"chr19.g2485\"; transcript_id \"chr19.g2485.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16620230', '16620547', '.', '-', '.', 'gene_id \"chr19.g2488\"; transcript_id \"chr19.g2488.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16620890', '16621485', '.', '+', '.', 'gene_id \"chr19.g2489\"; transcript_id \"chr19.g2489.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16625113', '16625432', '.', '-', '.', 'gene_id \"chr19.g2493\"; transcript_id \"chr19.g2493.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16625964', '16626241', '.', '+', '.', 'gene_id \"chr19.g2495\"; transcript_id \"chr19.g2495.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16631185', '16631441', '.', '-', '.', 'gene_id \"chr19.g2498\"; transcript_id \"chr19.g2498.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16643567', '16643768', '.', '+', '.', 'gene_id \"chr19.g2503\"; transcript_id \"chr19.g2503.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16650208', '16650360', '.', '-', '.', 'gene_id \"chr19.g2505\"; transcript_id \"chr19.g2505.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16667419', '16667641', '.', '-', '.', 'gene_id \"chr19.g2511\"; transcript_id \"chr19.g2511.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16694564', '16694755', '.', '+', '.', 'gene_id \"chr19.g2514\"; transcript_id \"chr19.g2514.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16695459', '16695631', '.', '+', '.', 'gene_id \"chr19.g2515\"; transcript_id \"chr19.g2515.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16739962', '16740185', '.', '+', '.', 'gene_id \"chr19.g2519\"; transcript_id \"chr19.g2519.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16776152', '16776393', '.', '+', '.', 'gene_id \"chr19.g2523\"; transcript_id \"chr19.g2523.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16800527', '16801302', '.', '-', '.', 'gene_id \"chr19.g2525\"; transcript_id \"chr19.g2525.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16834769', '16834815', '.', '-', '.', 'gene_id \"chr19.g2527\"; transcript_id \"chr19.g2527.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16887255', '16887481', '.', '-', '.', 'gene_id \"chr19.g2543\"; transcript_id \"chr19.g2543.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16887572', '16887746', '.', '-', '.', 'gene_id \"chr19.g2544\"; transcript_id \"chr19.g2544.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16888919', '16889536', '.', '-', '.', 'gene_id \"chr19.g2546\"; transcript_id \"chr19.g2546.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16890217', '16890377', '.', '+', '.', 'gene_id \"chr19.g2549\"; transcript_id \"chr19.g2549.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16892109', '16892361', '.', '+', '.', 'gene_id \"chr19.g2551\"; transcript_id \"chr19.g2551.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16894205', '16894487', '.', '+', '.', 'gene_id \"chr19.g2554\"; transcript_id \"chr19.g2554.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16909082', '16909470', '.', '+', '.', 'gene_id \"chr19.g2563\"; transcript_id \"chr19.g2563.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16919307', '16919913', '.', '-', '.', 'gene_id \"chr19.g2565\"; transcript_id \"chr19.g2565.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16911301', '16911547', '.', '+', '.', 'gene_id \"chr19.g2569\"; transcript_id \"chr19.g2569.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16912074', '16912295', '.', '+', '.', 'gene_id \"chr19.g2571\"; transcript_id \"chr19.g2571.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16912285', '16912524', '.', '-', '.', 'gene_id \"chr19.g2572\"; transcript_id \"chr19.g2572.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16913573', '16913764', '.', '+', '.', 'gene_id \"chr19.g2573\"; transcript_id \"chr19.g2573.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16914017', '16914172', '.', '-', '.', 'gene_id \"chr19.g2574\"; transcript_id \"chr19.g2574.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16915330', '16915489', '.', '-', '.', 'gene_id \"chr19.g2576\"; transcript_id \"chr19.g2576.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16936436', '16936563', '.', '-', '.', 'gene_id \"chr19.g2580\"; transcript_id \"chr19.g2580.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16971001', '16971157', '.', '-', '.', 'gene_id \"chr19.g2584\"; transcript_id \"chr19.g2584.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16996602', '16996708', '.', '-', '.', 'gene_id \"chr19.g2589\"; transcript_id \"chr19.g2589.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17033961', '17034326', '.', '-', '.', 'gene_id \"chr19.g2599\"; transcript_id \"chr19.g2599.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17035558', '17035772', '.', '+', '.', 'gene_id \"chr19.g2600\"; transcript_id \"chr19.g2600.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17038415', '17038592', '.', '+', '.', 'gene_id \"chr19.g2601\"; transcript_id \"chr19.g2601.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17045542', '17045900', '.', '-', '.', 'gene_id \"chr19.g2609\"; transcript_id \"chr19.g2609.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17117867', '17118028', '.', '-', '.', 'gene_id \"chr19.g2621\"; transcript_id \"chr19.g2621.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17118709', '17118875', '.', '-', '.', 'gene_id \"chr19.g2624\"; transcript_id \"chr19.g2624.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17182874', '17183116', '.', '-', '.', 'gene_id \"chr19.g2625\"; transcript_id \"chr19.g2625.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17128758', '17128922', '.', '-', '.', 'gene_id \"chr19.g2628\"; transcript_id \"chr19.g2628.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17130295', '17130526', '.', '-', '.', 'gene_id \"chr19.g2629\"; transcript_id \"chr19.g2629.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17157007', '17157168', '.', '-', '.', 'gene_id \"chr19.g2645\"; transcript_id \"chr19.g2645.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17160614', '17160957', '.', '+', '.', 'gene_id \"chr19.g2648\"; transcript_id \"chr19.g2648.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17177194', '17177505', '.', '+', '.', 'gene_id \"chr19.g2652\"; transcript_id \"chr19.g2652.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17227231', '17227395', '.', '-', '.', 'gene_id \"chr19.g2656\"; transcript_id \"chr19.g2656.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17239049', '17239208', '.', '+', '.', 'gene_id \"chr19.g2661\"; transcript_id \"chr19.g2661.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17239734', '17239946', '.', '-', '.', 'gene_id \"chr19.g2663\"; transcript_id \"chr19.g2663.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17243379', '17243593', '.', '-', '.', 'gene_id \"chr19.g2667\"; transcript_id \"chr19.g2667.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17248476', '17248655', '.', '+', '.', 'gene_id \"chr19.g2671\"; transcript_id \"chr19.g2671.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17249308', '17249556', '.', '-', '.', 'gene_id \"chr19.g2672\"; transcript_id \"chr19.g2672.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17250029', '17250405', '.', '-', '.', 'gene_id \"chr19.g2673\"; transcript_id \"chr19.g2673.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17253412', '17253565', '.', '-', '.', 'gene_id \"chr19.g2674\"; transcript_id \"chr19.g2674.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17254997', '17255162', '.', '-', '.', 'gene_id \"chr19.g2675\"; transcript_id \"chr19.g2675.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17265018', '17265637', '.', '+', '.', 'gene_id \"chr19.g2676\"; transcript_id \"chr19.g2676.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17295557', '17295784', '.', '-', '.', 'gene_id \"chr19.g2685\"; transcript_id \"chr19.g2685.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17307301', '17307444', '.', '+', '.', 'gene_id \"chr19.g2692\"; transcript_id \"chr19.g2692.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17322089', '17322257', '.', '-', '.', 'gene_id \"chr19.g2697\"; transcript_id \"chr19.g2697.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17343144', '17343356', '.', '+', '.', 'gene_id \"chr19.g2698\"; transcript_id \"chr19.g2698.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17350163', '17350316', '.', '+', '.', 'gene_id \"chr19.g2699\"; transcript_id \"chr19.g2699.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17361861', '17362083', '.', '+', '.', 'gene_id \"chr19.g2700\"; transcript_id \"chr19.g2700.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17368031', '17368192', '.', '+', '.', 'gene_id \"chr19.g2702\"; transcript_id \"chr19.g2702.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17398153', '17398404', '.', '-', '.', 'gene_id \"chr19.g2704\"; transcript_id \"chr19.g2704.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17442964', '17443116', '.', '+', '.', 'gene_id \"chr19.g2717\"; transcript_id \"chr19.g2717.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17455640', '17455807', '.', '-', '.', 'gene_id \"chr19.g2719\"; transcript_id \"chr19.g2719.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17464840', '17465109', '.', '+', '.', 'gene_id \"chr19.g2722\"; transcript_id \"chr19.g2722.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17484184', '17484347', '.', '+', '.', 'gene_id \"chr19.g2730\"; transcript_id \"chr19.g2730.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17486520', '17487279', '.', '+', '.', 'gene_id \"chr19.g2733\"; transcript_id \"chr19.g2733.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17494179', '17494416', '.', '+', '.', 'gene_id \"chr19.g2739\"; transcript_id \"chr19.g2739.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17498313', '17498685', '.', '+', '.', 'gene_id \"chr19.g2742\"; transcript_id \"chr19.g2742.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17501957', '17502143', '.', '-', '.', 'gene_id \"chr19.g2743\"; transcript_id \"chr19.g2743.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17521202', '17521402', '.', '+', '.', 'gene_id \"chr19.g2745\"; transcript_id \"chr19.g2745.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17621065', '17621258', '.', '+', '.', 'gene_id \"chr19.g2762\"; transcript_id \"chr19.g2762.i5\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17616839', '17617102', '.', '-', '.', 'gene_id \"chr19.g2768\"; transcript_id \"chr19.g2768.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17685858', '17686010', '.', '+', '.', 'gene_id \"chr19.g2773\"; transcript_id \"chr19.g2773.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17694710', '17694959', '.', '-', '.', 'gene_id \"chr19.g2777\"; transcript_id \"chr19.g2777.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17703537', '17703711', '.', '-', '.', 'gene_id \"chr19.g2780\"; transcript_id \"chr19.g2780.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17788854', '17789032', '.', '+', '.', 'gene_id \"chr19.g2787\"; transcript_id \"chr19.g2787.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17896994', '17897166', '.', '+', '.', 'gene_id \"chr19.g2795\"; transcript_id \"chr19.g2795.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17947337', '17947479', '.', '-', '.', 'gene_id \"chr19.g2797\"; transcript_id \"chr19.g2797.i31\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17913027', '17913428', '.', '-', '.', 'gene_id \"chr19.g2800\"; transcript_id \"chr19.g2800.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18077450', '18077611', '.', '+', '.', 'gene_id \"chr19.g2801\"; transcript_id \"chr19.g2801.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17918612', '17918767', '.', '-', '.', 'gene_id \"chr19.g2806\"; transcript_id \"chr19.g2806.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17913496', '17913846', '.', '+', '.', 'gene_id \"chr19.g2810\"; transcript_id \"chr19.g2810.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17914989', '17915370', '.', '-', '.', 'gene_id \"chr19.g2811\"; transcript_id \"chr19.g2811.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17921751', '17922028', '.', '-', '.', 'gene_id \"chr19.g2816\"; transcript_id \"chr19.g2816.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17922251', '17922540', '.', '-', '.', 'gene_id \"chr19.g2817\"; transcript_id \"chr19.g2817.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17923803', '17923980', '.', '+', '.', 'gene_id \"chr19.g2819\"; transcript_id \"chr19.g2819.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17933043', '17933196', '.', '+', '.', 'gene_id \"chr19.g2820\"; transcript_id \"chr19.g2820.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17937762', '17937999', '.', '+', '.', 'gene_id \"chr19.g2822\"; transcript_id \"chr19.g2822.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17940131', '17940455', '.', '+', '.', 'gene_id \"chr19.g2825\"; transcript_id \"chr19.g2825.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17957270', '17957426', '.', '-', '.', 'gene_id \"chr19.g2832\"; transcript_id \"chr19.g2832.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17978654', '17979087', '.', '-', '.', 'gene_id \"chr19.g2836\"; transcript_id \"chr19.g2836.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '15408975', '15409015', '.', '+', '.', 'gene_id \"chr16.g6596\"; transcript_id \"chr16.g6596.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17979483', '17980183', '.', '+', '.', 'gene_id \"chr19.g2838\"; transcript_id \"chr19.g2838.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17992694', '17992825', '.', '+', '.', 'gene_id \"chr19.g2842\"; transcript_id \"chr19.g2842.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17995941', '17996182', '.', '+', '.', 'gene_id \"chr19.g2847\"; transcript_id \"chr19.g2847.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18020977', '18021153', '.', '+', '.', 'gene_id \"chr19.g2849\"; transcript_id \"chr19.g2849.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18047974', '18048241', '.', '-', '.', 'gene_id \"chr19.g2858\"; transcript_id \"chr19.g2858.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18062100', '18062298', '.', '+', '.', 'gene_id \"chr19.g2859\"; transcript_id \"chr19.g2859.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18118563', '18118718', '.', '+', '.', 'gene_id \"chr19.g2867\"; transcript_id \"chr19.g2867.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18193601', '18193644', '.', '-', '.', 'gene_id \"chr19.g2874\"; transcript_id \"chr19.g2874.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18280293', '18280451', '.', '+', '.', 'gene_id \"chr19.g2883\"; transcript_id \"chr19.g2883.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18325022', '18325279', '.', '+', '.', 'gene_id \"chr19.g2889\"; transcript_id \"chr19.g2889.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18336087', '18336271', '.', '-', '.', 'gene_id \"chr19.g2892\"; transcript_id \"chr19.g2892.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18368898', '18369163', '.', '+', '.', 'gene_id \"chr19.g2901\"; transcript_id \"chr19.g2901.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18360880', '18361471', '.', '-', '.', 'gene_id \"chr19.g2902\"; transcript_id \"chr19.g2902.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18363598', '18364696', '.', '-', '.', 'gene_id \"chr19.g2903\"; transcript_id \"chr19.g2903.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18376972', '18377181', '.', '-', '.', 'gene_id \"chr19.g2905\"; transcript_id \"chr19.g2905.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18371956', '18372188', '.', '+', '.', 'gene_id \"chr19.g2910\"; transcript_id \"chr19.g2910.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18377943', '18378165', '.', '+', '.', 'gene_id \"chr19.g2911\"; transcript_id \"chr19.g2911.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18378259', '18378552', '.', '+', '.', 'gene_id \"chr19.g2912\"; transcript_id \"chr19.g2912.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18379045', '18379250', '.', '-', '.', 'gene_id \"chr19.g2913\"; transcript_id \"chr19.g2913.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18382920', '18383135', '.', '-', '.', 'gene_id \"chr19.g2914\"; transcript_id \"chr19.g2914.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18395579', '18396510', '.', '-', '.', 'gene_id \"chr19.g2921\"; transcript_id \"chr19.g2921.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18398869', '18399027', '.', '-', '.', 'gene_id \"chr19.g2925\"; transcript_id \"chr19.g2925.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18399045', '18399257', '.', '+', '.', 'gene_id \"chr19.g2926\"; transcript_id \"chr19.g2926.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18439059', '18439255', '.', '+', '.', 'gene_id \"chr19.g2940\"; transcript_id \"chr19.g2940.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18440037', '18440247', '.', '+', '.', 'gene_id \"chr19.g2941\"; transcript_id \"chr19.g2941.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18440238', '18440644', '.', '-', '.', 'gene_id \"chr19.g2942\"; transcript_id \"chr19.g2942.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18491870', '18492325', '.', '+', '.', 'gene_id \"chr19.g2950\"; transcript_id \"chr19.g2950.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18498737', '18499130', '.', '+', '.', 'gene_id \"chr19.g2953\"; transcript_id \"chr19.g2953.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18518171', '18519326', '.', '-', '.', 'gene_id \"chr19.g2958\"; transcript_id \"chr19.g2958.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18503205', '18503473', '.', '+', '.', 'gene_id \"chr19.g2960\"; transcript_id \"chr19.g2960.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18511843', '18512209', '.', '-', '.', 'gene_id \"chr19.g2965\"; transcript_id \"chr19.g2965.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18512397', '18512900', '.', '-', '.', 'gene_id \"chr19.g2968\"; transcript_id \"chr19.g2968.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18516369', '18516574', '.', '-', '.', 'gene_id \"chr19.g2970\"; transcript_id \"chr19.g2970.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18534675', '18534998', '.', '+', '.', 'gene_id \"chr19.g2972\"; transcript_id \"chr19.g2972.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18609056', '18609195', '.', '+', '.', 'gene_id \"chr19.g2987\"; transcript_id \"chr19.g2987.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18627085', '18627237', '.', '-', '.', 'gene_id \"chr19.g2989\"; transcript_id \"chr19.g2989.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18732173', '18732283', '.', '+', '.', 'gene_id \"chr19.g2997\"; transcript_id \"chr19.g2997.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18849617', '18852827', '.', '+', '.', 'gene_id \"chr19.g3014\"; transcript_id \"chr19.g3014.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18859655', '18859857', '.', '+', '.', 'gene_id \"chr19.g3021\"; transcript_id \"chr19.g3021.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18870804', '18871074', '.', '-', '.', 'gene_id \"chr19.g3039\"; transcript_id \"chr19.g3039.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18880323', '18880528', '.', '-', '.', 'gene_id \"chr19.g3047\"; transcript_id \"chr19.g3047.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18897995', '18898263', '.', '+', '.', 'gene_id \"chr19.g3052\"; transcript_id \"chr19.g3052.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18908460', '18909454', '.', '-', '.', 'gene_id \"chr19.g3057\"; transcript_id \"chr19.g3057.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18912067', '18913187', '.', '+', '.', 'gene_id \"chr19.g3064\"; transcript_id \"chr19.g3064.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18943697', '18943814', '.', '+', '.', 'gene_id \"chr19.g3068\"; transcript_id \"chr19.g3068.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19024514', '19024685', '.', '-', '.', 'gene_id \"chr19.g3078\"; transcript_id \"chr19.g3078.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19037469', '19037668', '.', '-', '.', 'gene_id \"chr19.g3083\"; transcript_id \"chr19.g3083.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19046941', '19047086', '.', '+', '.', 'gene_id \"chr19.g3086\"; transcript_id \"chr19.g3086.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19051223', '19051401', '.', '+', '.', 'gene_id \"chr19.g3092\"; transcript_id \"chr19.g3092.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19062117', '19062408', '.', '-', '.', 'gene_id \"chr19.g3095\"; transcript_id \"chr19.g3095.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19214320', '19215303', '.', '-', '.', 'gene_id \"chr19.g3115\"; transcript_id \"chr19.g3115.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19280056', '19280180', '.', '-', '.', 'gene_id \"chr19.g3118\"; transcript_id \"chr19.g3118.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19350162', '19350331', '.', '+', '.', 'gene_id \"chr19.g3143\"; transcript_id \"chr19.g3143.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19363800', '19363956', '.', '+', '.', 'gene_id \"chr19.g3146\"; transcript_id \"chr19.g3146.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19363978', '19364147', '.', '+', '.', 'gene_id \"chr19.g3147\"; transcript_id \"chr19.g3147.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19366275', '19366449', '.', '+', '.', 'gene_id \"chr19.g3149\"; transcript_id \"chr19.g3149.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19393701', '19393868', '.', '-', '.', 'gene_id \"chr19.g3154\"; transcript_id \"chr19.g3154.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19394557', '19394632', '.', '-', '.', 'gene_id \"chr19.g3155\"; transcript_id \"chr19.g3155.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19414962', '19415146', '.', '-', '.', 'gene_id \"chr19.g3166\"; transcript_id \"chr19.g3166.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19417518', '19417728', '.', '-', '.', 'gene_id \"chr19.g3169\"; transcript_id \"chr19.g3169.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19417724', '19417908', '.', '-', '.', 'gene_id \"chr19.g3170\"; transcript_id \"chr19.g3170.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19418442', '19418742', '.', '-', '.', 'gene_id \"chr19.g3171\"; transcript_id \"chr19.g3171.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19421992', '19422343', '.', '-', '.', 'gene_id \"chr19.g3172\"; transcript_id \"chr19.g3172.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19430754', '19431478', '.', '-', '.', 'gene_id \"chr19.g3174\"; transcript_id \"chr19.g3174.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19883774', '19884085', '.', '+', '.', 'gene_id \"chr19.g3209\"; transcript_id \"chr19.g3209.i8\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19894742', '19895054', '.', '-', '.', 'gene_id \"chr19.g3215\"; transcript_id \"chr19.g3215.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19883621', '19883779', '.', '-', '.', 'gene_id \"chr19.g3216\"; transcript_id \"chr19.g3216.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19904478', '19904816', '.', '+', '.', 'gene_id \"chr19.g3221\"; transcript_id \"chr19.g3221.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19904679', '19904816', '.', '+', '.', 'gene_id \"chr19.g3221\"; transcript_id \"chr19.g3221.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '15794109', '15794260', '.', '+', '.', 'gene_id \"chr16.g6976\"; transcript_id \"chr16.g6976.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '15794275', '15794442', '.', '-', '.', 'gene_id \"chr16.g6977\"; transcript_id \"chr16.g6977.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19939788', '19939988', '.', '+', '.', 'gene_id \"chr19.g3241\"; transcript_id \"chr19.g3241.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20191828', '20192030', '.', '-', '.', 'gene_id \"chr19.g3260\"; transcript_id \"chr19.g3260.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20207316', '20207497', '.', '-', '.', 'gene_id \"chr19.g3263\"; transcript_id \"chr19.g3263.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20206395', '20206771', '.', '+', '.', 'gene_id \"chr19.g3264\"; transcript_id \"chr19.g3264.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20208327', '20208431', '.', '+', '.', 'gene_id \"chr19.g3264\"; transcript_id \"chr19.g3264.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20328419', '20328688', '.', '+', '.', 'gene_id \"chr19.g3283\"; transcript_id \"chr19.g3283.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20396671', '20397059', '.', '+', '.', 'gene_id \"chr19.g3288\"; transcript_id \"chr19.g3288.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '15993701', '15993937', '.', '-', '.', 'gene_id \"chr16.g7036\"; transcript_id \"chr16.g7036.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20424089', '20424270', '.', '-', '.', 'gene_id \"chr19.g3299\"; transcript_id \"chr19.g3299.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20609045', '20609353', '.', '+', '.', 'gene_id \"chr19.g3324\"; transcript_id \"chr19.g3324.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20614160', '20614334', '.', '-', '.', 'gene_id \"chr19.g3325\"; transcript_id \"chr19.g3325.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20624698', '20624844', '.', '-', '.', 'gene_id \"chr19.g3327\"; transcript_id \"chr19.g3327.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20655204', '20655598', '.', '+', '.', 'gene_id \"chr19.g3333\"; transcript_id \"chr19.g3333.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20675979', '20676313', '.', '-', '.', 'gene_id \"chr19.g3336\"; transcript_id \"chr19.g3336.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20682199', '20682360', '.', '-', '.', 'gene_id \"chr19.g3343\"; transcript_id \"chr19.g3343.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20684369', '20684653', '.', '-', '.', 'gene_id \"chr19.g3344\"; transcript_id \"chr19.g3344.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20688116', '20688272', '.', '+', '.', 'gene_id \"chr19.g3350\"; transcript_id \"chr19.g3350.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20722025', '20722110', '.', '+', '.', 'gene_id \"chr19.g3355\"; transcript_id \"chr19.g3355.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20688971', '20689272', '.', '+', '.', 'gene_id \"chr19.g3356\"; transcript_id \"chr19.g3356.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20707001', '20707632', '.', '+', '.', 'gene_id \"chr19.g3361\"; transcript_id \"chr19.g3361.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20708410', '20708620', '.', '-', '.', 'gene_id \"chr19.g3364\"; transcript_id \"chr19.g3364.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20710660', '20710852', '.', '+', '.', 'gene_id \"chr19.g3365\"; transcript_id \"chr19.g3365.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20740812', '20741381', '.', '+', '.', 'gene_id \"chr19.g3372\"; transcript_id \"chr19.g3372.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20742972', '20743509', '.', '-', '.', 'gene_id \"chr19.g3374\"; transcript_id \"chr19.g3374.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20743528', '20743695', '.', '-', '.', 'gene_id \"chr19.g3375\"; transcript_id \"chr19.g3375.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20743533', '20743695', '.', '-', '.', 'gene_id \"chr19.g3375\"; transcript_id \"chr19.g3375.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20745399', '20745553', '.', '-', '.', 'gene_id \"chr19.g3380\"; transcript_id \"chr19.g3380.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20753175', '20754248', '.', '-', '.', 'gene_id \"chr19.g3385\"; transcript_id \"chr19.g3385.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20764395', '20764579', '.', '-', '.', 'gene_id \"chr19.g3398\"; transcript_id \"chr19.g3398.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20764196', '20764410', '.', '-', '.', 'gene_id \"chr19.g3399\"; transcript_id \"chr19.g3399.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20771528', '20772004', '.', '-', '.', 'gene_id \"chr19.g3403\"; transcript_id \"chr19.g3403.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20774997', '20775241', '.', '+', '.', 'gene_id \"chr19.g3408\"; transcript_id \"chr19.g3408.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20783230', '20783525', '.', '-', '.', 'gene_id \"chr19.g3412\"; transcript_id \"chr19.g3412.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20784315', '20784529', '.', '-', '.', 'gene_id \"chr19.g3413\"; transcript_id \"chr19.g3413.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20784514', '20784698', '.', '-', '.', 'gene_id \"chr19.g3414\"; transcript_id \"chr19.g3414.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20795576', '20795824', '.', '-', '.', 'gene_id \"chr19.g3417\"; transcript_id \"chr19.g3417.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20810130', '20810356', '.', '-', '.', 'gene_id \"chr19.g3428\"; transcript_id \"chr19.g3428.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20813430', '20813490', '.', '-', '.', 'gene_id \"chr19.g3429\"; transcript_id \"chr19.g3429.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20814366', '20814528', '.', '+', '.', 'gene_id \"chr19.g3431\"; transcript_id \"chr19.g3431.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20815345', '20815897', '.', '-', '.', 'gene_id \"chr19.g3432\"; transcript_id \"chr19.g3432.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20817644', '20817909', '.', '-', '.', 'gene_id \"chr19.g3433\"; transcript_id \"chr19.g3433.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20819365', '20819919', '.', '+', '.', 'gene_id \"chr19.g3436\"; transcript_id \"chr19.g3436.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20821482', '20822731', '.', '+', '.', 'gene_id \"chr19.g3439\"; transcript_id \"chr19.g3439.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20824844', '20825160', '.', '+', '.', 'gene_id \"chr19.g3442\"; transcript_id \"chr19.g3442.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20844080', '20844273', '.', '+', '.', 'gene_id \"chr19.g3446\"; transcript_id \"chr19.g3446.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20867255', '20867444', '.', '-', '.', 'gene_id \"chr19.g3450\"; transcript_id \"chr19.g3450.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '21103577', '21103964', '.', '+', '.', 'gene_id \"chr19.g3464\"; transcript_id \"chr19.g3464.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '21492137', '21492202', '.', '-', '.', 'gene_id \"chr19.g3499\"; transcript_id \"chr19.g3499.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '21637383', '21637519', '.', '+', '.', 'gene_id \"chr19.g3515\"; transcript_id \"chr19.g3515.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22145783', '22145950', '.', '-', '.', 'gene_id \"chr19.g3570\"; transcript_id \"chr19.g3570.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22184846', '22185009', '.', '-', '.', 'gene_id \"chr19.g3577\"; transcript_id \"chr19.g3577.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22186249', '22186510', '.', '-', '.', 'gene_id \"chr19.g3578\"; transcript_id \"chr19.g3578.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22255554', '22256163', '.', '+', '.', 'gene_id \"chr19.g3592\"; transcript_id \"chr19.g3592.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22309644', '22309852', '.', '+', '.', 'gene_id \"chr19.g3598\"; transcript_id \"chr19.g3598.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22323335', '22323589', '.', '-', '.', 'gene_id \"chr19.g3612\"; transcript_id \"chr19.g3612.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22325198', '22325491', '.', '-', '.', 'gene_id \"chr19.g3614\"; transcript_id \"chr19.g3614.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22327589', '22327937', '.', '-', '.', 'gene_id \"chr19.g3617\"; transcript_id \"chr19.g3617.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22344160', '22344713', '.', '-', '.', 'gene_id \"chr19.g3635\"; transcript_id \"chr19.g3635.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22348708', '22349759', '.', '+', '.', 'gene_id \"chr19.g3639\"; transcript_id \"chr19.g3639.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22357019', '22357395', '.', '-', '.', 'gene_id \"chr19.g3644\"; transcript_id \"chr19.g3644.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22357384', '22357759', '.', '-', '.', 'gene_id \"chr19.g3645\"; transcript_id \"chr19.g3645.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '16450625', '16450662', '.', '+', '.', 'gene_id \"chr16.g7412\"; transcript_id \"chr16.g7412.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22409205', '22410248', '.', '+', '.', 'gene_id \"chr19.g3653\"; transcript_id \"chr19.g3653.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22416202', '22416681', '.', '-', '.', 'gene_id \"chr19.g3655\"; transcript_id \"chr19.g3655.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22441220', '22441503', '.', '-', '.', 'gene_id \"chr19.g3656\"; transcript_id \"chr19.g3656.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22447240', '22447544', '.', '+', '.', 'gene_id \"chr19.g3658\"; transcript_id \"chr19.g3658.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22418024', '22418183', '.', '+', '.', 'gene_id \"chr19.g3672\"; transcript_id \"chr19.g3672.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22441980', '22442172', '.', '-', '.', 'gene_id \"chr19.g3677\"; transcript_id \"chr19.g3677.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22442913', '22443074', '.', '-', '.', 'gene_id \"chr19.g3678\"; transcript_id \"chr19.g3678.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22446902', '22447134', '.', '+', '.', 'gene_id \"chr19.g3680\"; transcript_id \"chr19.g3680.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22465578', '22465729', '.', '-', '.', 'gene_id \"chr19.g3683\"; transcript_id \"chr19.g3683.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22477460', '22477948', '.', '-', '.', 'gene_id \"chr19.g3686\"; transcript_id \"chr19.g3686.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22496125', '22496362', '.', '+', '.', 'gene_id \"chr19.g3689\"; transcript_id \"chr19.g3689.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22502944', '22503161', '.', '-', '.', 'gene_id \"chr19.g3693\"; transcript_id \"chr19.g3693.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22509215', '22509403', '.', '-', '.', 'gene_id \"chr19.g3698\"; transcript_id \"chr19.g3698.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22514903', '22515190', '.', '-', '.', 'gene_id \"chr19.g3700\"; transcript_id \"chr19.g3700.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22517787', '22518410', '.', '-', '.', 'gene_id \"chr19.g3702\"; transcript_id \"chr19.g3702.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22521180', '22521939', '.', '-', '.', 'gene_id \"chr19.g3705\"; transcript_id \"chr19.g3705.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22653120', '22653644', '.', '-', '.', 'gene_id \"chr19.g3726\"; transcript_id \"chr19.g3726.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22596306', '22596719', '.', '-', '.', 'gene_id \"chr19.g3731\"; transcript_id \"chr19.g3731.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22616472', '22616812', '.', '-', '.', 'gene_id \"chr19.g3741\"; transcript_id \"chr19.g3741.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22621085', '22621354', '.', '-', '.', 'gene_id \"chr19.g3746\"; transcript_id \"chr19.g3746.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22621450', '22621692', '.', '-', '.', 'gene_id \"chr19.g3747\"; transcript_id \"chr19.g3747.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22647944', '22648158', '.', '-', '.', 'gene_id \"chr19.g3758\"; transcript_id \"chr19.g3758.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22658913', '22659156', '.', '+', '.', 'gene_id \"chr19.g3765\"; transcript_id \"chr19.g3765.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22703652', '22704053', '.', '+', '.', 'gene_id \"chr19.g3780\"; transcript_id \"chr19.g3780.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22706105', '22706259', '.', '-', '.', 'gene_id \"chr19.g3784\"; transcript_id \"chr19.g3784.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22721200', '22721566', '.', '+', '.', 'gene_id \"chr19.g3786\"; transcript_id \"chr19.g3786.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22710199', '22710430', '.', '+', '.', 'gene_id \"chr19.g3789\"; transcript_id \"chr19.g3789.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22710458', '22710669', '.', '+', '.', 'gene_id \"chr19.g3790\"; transcript_id \"chr19.g3790.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22712948', '22713210', '.', '+', '.', 'gene_id \"chr19.g3793\"; transcript_id \"chr19.g3793.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22713313', '22714024', '.', '+', '.', 'gene_id \"chr19.g3795\"; transcript_id \"chr19.g3795.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22720030', '22720332', '.', '-', '.', 'gene_id \"chr19.g3800\"; transcript_id \"chr19.g3800.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22748936', '22749265', '.', '+', '.', 'gene_id \"chr19.g3807\"; transcript_id \"chr19.g3807.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22736208', '22736425', '.', '+', '.', 'gene_id \"chr19.g3809\"; transcript_id \"chr19.g3809.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22739761', '22739933', '.', '+', '.', 'gene_id \"chr19.g3811\"; transcript_id \"chr19.g3811.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22739898', '22740001', '.', '+', '.', 'gene_id \"chr19.g3811\"; transcript_id \"chr19.g3811.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22741567', '22741753', '.', '+', '.', 'gene_id \"chr19.g3812\"; transcript_id \"chr19.g3812.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22742469', '22742776', '.', '+', '.', 'gene_id \"chr19.g3813\"; transcript_id \"chr19.g3813.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22746079', '22746234', '.', '+', '.', 'gene_id \"chr19.g3815\"; transcript_id \"chr19.g3815.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22748341', '22748587', '.', '-', '.', 'gene_id \"chr19.g3817\"; transcript_id \"chr19.g3817.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22753075', '22753648', '.', '+', '.', 'gene_id \"chr19.g3822\"; transcript_id \"chr19.g3822.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22873292', '22874127', '.', '+', '.', 'gene_id \"chr19.g3827\"; transcript_id \"chr19.g3827.i22\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22933819', '22934009', '.', '+', '.', 'gene_id \"chr19.g3836\"; transcript_id \"chr19.g3836.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22926142', '22926489', '.', '+', '.', 'gene_id \"chr19.g3837\"; transcript_id \"chr19.g3837.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22770840', '22770991', '.', '-', '.', 'gene_id \"chr19.g3840\"; transcript_id \"chr19.g3840.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22791649', '22791889', '.', '+', '.', 'gene_id \"chr19.g3849\"; transcript_id \"chr19.g3849.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22796512', '22796749', '.', '+', '.', 'gene_id \"chr19.g3854\"; transcript_id \"chr19.g3854.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22800787', '22801779', '.', '+', '.', 'gene_id \"chr19.g3860\"; transcript_id \"chr19.g3860.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22802859', '22803049', '.', '-', '.', 'gene_id \"chr19.g3862\"; transcript_id \"chr19.g3862.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22828186', '22828376', '.', '-', '.', 'gene_id \"chr19.g3866\"; transcript_id \"chr19.g3866.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22828808', '22829444', '.', '-', '.', 'gene_id \"chr19.g3867\"; transcript_id \"chr19.g3867.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22830395', '22830529', '.', '-', '.', 'gene_id \"chr19.g3869\"; transcript_id \"chr19.g3869.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22834011', '22834188', '.', '-', '.', 'gene_id \"chr19.g3872\"; transcript_id \"chr19.g3872.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22835302', '22835470', '.', '-', '.', 'gene_id \"chr19.g3873\"; transcript_id \"chr19.g3873.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22839184', '22839829', '.', '-', '.', 'gene_id \"chr19.g3877\"; transcript_id \"chr19.g3877.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22846546', '22846777', '.', '+', '.', 'gene_id \"chr19.g3882\"; transcript_id \"chr19.g3882.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22850131', '22850563', '.', '+', '.', 'gene_id \"chr19.g3889\"; transcript_id \"chr19.g3889.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22851490', '22851676', '.', '-', '.', 'gene_id \"chr19.g3890\"; transcript_id \"chr19.g3890.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22851654', '22852406', '.', '-', '.', 'gene_id \"chr19.g3890\"; transcript_id \"chr19.g3890.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22852807', '22853619', '.', '-', '.', 'gene_id \"chr19.g3892\"; transcript_id \"chr19.g3892.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22883119', '22883297', '.', '-', '.', 'gene_id \"chr19.g3903\"; transcript_id \"chr19.g3903.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22888570', '22889213', '.', '-', '.', 'gene_id \"chr19.g3905\"; transcript_id \"chr19.g3905.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22889855', '22890033', '.', '-', '.', 'gene_id \"chr19.g3906\"; transcript_id \"chr19.g3906.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22899725', '22900155', '.', '+', '.', 'gene_id \"chr19.g3916\"; transcript_id \"chr19.g3916.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22902260', '22902414', '.', '-', '.', 'gene_id \"chr19.g3917\"; transcript_id \"chr19.g3917.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22904279', '22904435', '.', '-', '.', 'gene_id \"chr19.g3917\"; transcript_id \"chr19.g3917.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22920842', '22921048', '.', '+', '.', 'gene_id \"chr19.g3926\"; transcript_id \"chr19.g3926.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22921027', '22921338', '.', '+', '.', 'gene_id \"chr19.g3926\"; transcript_id \"chr19.g3926.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22930764', '22930996', '.', '+', '.', 'gene_id \"chr19.g3930\"; transcript_id \"chr19.g3930.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22945951', '22946190', '.', '+', '.', 'gene_id \"chr19.g3934\"; transcript_id \"chr19.g3934.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23015439', '23015682', '.', '-', '.', 'gene_id \"chr19.g3936\"; transcript_id \"chr19.g3936.i9\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22988185', '22990261', '.', '-', '.', 'gene_id \"chr19.g3938\"; transcript_id \"chr19.g3938.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22967408', '22967564', '.', '-', '.', 'gene_id \"chr19.g3940\"; transcript_id \"chr19.g3940.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr19', 'PASA', 'exon', '23002836', '23003680', '.', '-', '.', 'gene_id \"chr19.g3941\"; transcript_id \"chr19.g3941.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23016133', '23016303', '.', '-', '.', 'gene_id \"chr19.g3942\"; transcript_id \"chr19.g3942.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23005292', '23006206', '.', '-', '.', 'gene_id \"chr19.g3943\"; transcript_id \"chr19.g3943.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23000689', '23000846', '.', '+', '.', 'gene_id \"chr19.g3948\"; transcript_id \"chr19.g3948.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23016378', '23016790', '.', '-', '.', 'gene_id \"chr19.g3953\"; transcript_id \"chr19.g3953.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23024521', '23024675', '.', '+', '.', 'gene_id \"chr19.g3955\"; transcript_id \"chr19.g3955.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23036458', '23036644', '.', '-', '.', 'gene_id \"chr19.g3958\"; transcript_id \"chr19.g3958.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23077437', '23077723', '.', '+', '.', 'gene_id \"chr19.g3964\"; transcript_id \"chr19.g3964.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '17442682', '17442748', '.', '-', '.', 'gene_id \"chr16.g7701\"; transcript_id \"chr16.g7701.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23083165', '23083347', '.', '-', '.', 'gene_id \"chr19.g3967\"; transcript_id \"chr19.g3967.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23094365', '23094543', '.', '-', '.', 'gene_id \"chr19.g3976\"; transcript_id \"chr19.g3976.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23095178', '23095358', '.', '+', '.', 'gene_id \"chr19.g3978\"; transcript_id \"chr19.g3978.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23097385', '23097562', '.', '-', '.', 'gene_id \"chr19.g3980\"; transcript_id \"chr19.g3980.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23113411', '23113792', '.', '-', '.', 'gene_id \"chr19.g3992\"; transcript_id \"chr19.g3992.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23132619', '23132808', '.', '-', '.', 'gene_id \"chr19.g3996\"; transcript_id \"chr19.g3996.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23125667', '23125821', '.', '+', '.', 'gene_id \"chr19.g4005\"; transcript_id \"chr19.g4005.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23140957', '23141104', '.', '-', '.', 'gene_id \"chr19.g4008\"; transcript_id \"chr19.g4008.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23154254', '23154463', '.', '-', '.', 'gene_id \"chr19.g4017\"; transcript_id \"chr19.g4017.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23154442', '23154598', '.', '+', '.', 'gene_id \"chr19.g4018\"; transcript_id \"chr19.g4018.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23154677', '23154842', '.', '-', '.', 'gene_id \"chr19.g4019\"; transcript_id \"chr19.g4019.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23222306', '23222337', '.', '-', '.', 'gene_id \"chr19.g4036\"; transcript_id \"chr19.g4036.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23234139', '23235547', '.', '-', '.', 'gene_id \"chr19.g4043\"; transcript_id \"chr19.g4043.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23239635', '23240264', '.', '+', '.', 'gene_id \"chr19.g4045\"; transcript_id \"chr19.g4045.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23243060', '23243283', '.', '+', '.', 'gene_id \"chr19.g4048\"; transcript_id \"chr19.g4048.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23243272', '23243704', '.', '+', '.', 'gene_id \"chr19.g4049\"; transcript_id \"chr19.g4049.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23244578', '23244864', '.', '+', '.', 'gene_id \"chr19.g4052\"; transcript_id \"chr19.g4052.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23245968', '23246123', '.', '-', '.', 'gene_id \"chr19.g4053\"; transcript_id \"chr19.g4053.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23257615', '23257880', '.', '+', '.', 'gene_id \"chr19.g4058\"; transcript_id \"chr19.g4058.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23271321', '23271495', '.', '-', '.', 'gene_id \"chr19.g4063\"; transcript_id \"chr19.g4063.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23271483', '23271645', '.', '-', '.', 'gene_id \"chr19.g4064\"; transcript_id \"chr19.g4064.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23275678', '23276066', '.', '-', '.', 'gene_id \"chr19.g4066\"; transcript_id \"chr19.g4066.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23278008', '23278174', '.', '-', '.', 'gene_id \"chr19.g4071\"; transcript_id \"chr19.g4071.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23290256', '23290447', '.', '+', '.', 'gene_id \"chr19.g4076\"; transcript_id \"chr19.g4076.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23317787', '23318356', '.', '-', '.', 'gene_id \"chr19.g4092\"; transcript_id \"chr19.g4092.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23357730', '23357967', '.', '+', '.', 'gene_id \"chr19.g4105\"; transcript_id \"chr19.g4105.i5\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23344150', '23344543', '.', '+', '.', 'gene_id \"chr19.g4110\"; transcript_id \"chr19.g4110.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23350400', '23350704', '.', '+', '.', 'gene_id \"chr19.g4111\"; transcript_id \"chr19.g4111.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23351208', '23351510', '.', '+', '.', 'gene_id \"chr19.g4112\"; transcript_id \"chr19.g4112.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23351496', '23351698', '.', '-', '.', 'gene_id \"chr19.g4113\"; transcript_id \"chr19.g4113.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23366515', '23366713', '.', '+', '.', 'gene_id \"chr19.g4121\"; transcript_id \"chr19.g4121.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23369492', '23369638', '.', '+', '.', 'gene_id \"chr19.g4123\"; transcript_id \"chr19.g4123.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23371283', '23371475', '.', '-', '.', 'gene_id \"chr19.g4124\"; transcript_id \"chr19.g4124.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23371461', '23372319', '.', '-', '.', 'gene_id \"chr19.g4125\"; transcript_id \"chr19.g4125.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23384153', '23384348', '.', '+', '.', 'gene_id \"chr19.g4131\"; transcript_id \"chr19.g4131.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23391253', '23391484', '.', '-', '.', 'gene_id \"chr19.g4136\"; transcript_id \"chr19.g4136.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23395905', '23396129', '.', '-', '.', 'gene_id \"chr19.g4140\"; transcript_id \"chr19.g4140.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23400786', '23400947', '.', '-', '.', 'gene_id \"chr19.g4148\"; transcript_id \"chr19.g4148.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23403062', '23403239', '.', '+', '.', 'gene_id \"chr19.g4150\"; transcript_id \"chr19.g4150.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23406709', '23406845', '.', '-', '.', 'gene_id \"chr19.g4153\"; transcript_id \"chr19.g4153.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23406903', '23407075', '.', '+', '.', 'gene_id \"chr19.g4154\"; transcript_id \"chr19.g4154.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23420932', '23421099', '.', '-', '.', 'gene_id \"chr19.g4160\"; transcript_id \"chr19.g4160.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23421667', '23421817', '.', '-', '.', 'gene_id \"chr19.g4161\"; transcript_id \"chr19.g4161.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23430953', '23431690', '.', '+', '.', 'gene_id \"chr19.g4166\"; transcript_id \"chr19.g4166.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23432719', '23433341', '.', '-', '.', 'gene_id \"chr19.g4167\"; transcript_id \"chr19.g4167.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23433283', '23433442', '.', '-', '.', 'gene_id \"chr19.g4167\"; transcript_id \"chr19.g4167.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23511270', '23511577', '.', '+', '.', 'gene_id \"chr19.g4182\"; transcript_id \"chr19.g4182.i8\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23543218', '23543461', '.', '-', '.', 'gene_id \"chr19.g4186\"; transcript_id \"chr19.g4186.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23544711', '23544965', '.', '-', '.', 'gene_id \"chr19.g4187\"; transcript_id \"chr19.g4187.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23532227', '23532382', '.', '-', '.', 'gene_id \"chr19.g4192\"; transcript_id \"chr19.g4192.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23538569', '23538744', '.', '+', '.', 'gene_id \"chr19.g4198\"; transcript_id \"chr19.g4198.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23539013', '23539167', '.', '-', '.', 'gene_id \"chr19.g4199\"; transcript_id \"chr19.g4199.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23572485', '23572656', '.', '+', '.', 'gene_id \"chr19.g4203\"; transcript_id \"chr19.g4203.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23686591', '23686938', '.', '-', '.', 'gene_id \"chr19.g4217\"; transcript_id \"chr19.g4217.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23861424', '23861924', '.', '-', '.', 'gene_id \"chr19.g4238\"; transcript_id \"chr19.g4238.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23861923', '23862179', '.', '+', '.', 'gene_id \"chr19.g4239\"; transcript_id \"chr19.g4239.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23863605', '23863805', '.', '+', '.', 'gene_id \"chr19.g4241\"; transcript_id \"chr19.g4241.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23863944', '23864122', '.', '+', '.', 'gene_id \"chr19.g4242\"; transcript_id \"chr19.g4242.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23869807', '23869958', '.', '+', '.', 'gene_id \"chr19.g4243\"; transcript_id \"chr19.g4243.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23954515', '23954665', '.', '-', '.', 'gene_id \"chr19.g4254\"; transcript_id \"chr19.g4254.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23965028', '23965352', '.', '+', '.', 'gene_id \"chr19.g4255\"; transcript_id \"chr19.g4255.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23973679', '23974501', '.', '+', '.', 'gene_id \"chr19.g4262\"; transcript_id \"chr19.g4262.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24019071', '24019275', '.', '-', '.', 'gene_id \"chr19.g4266\"; transcript_id \"chr19.g4266.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23977421', '23977631', '.', '+', '.', 'gene_id \"chr19.g4267\"; transcript_id \"chr19.g4267.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23986516', '23987161', '.', '-', '.', 'gene_id \"chr19.g4271\"; transcript_id \"chr19.g4271.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23988662', '23989009', '.', '+', '.', 'gene_id \"chr19.g4274\"; transcript_id \"chr19.g4274.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23992329', '23992533', '.', '+', '.', 'gene_id \"chr19.g4279\"; transcript_id \"chr19.g4279.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23993441', '23993646', '.', '+', '.', 'gene_id \"chr19.g4282\"; transcript_id \"chr19.g4282.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23997841', '23998184', '.', '+', '.', 'gene_id \"chr19.g4285\"; transcript_id \"chr19.g4285.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24001637', '24001876', '.', '-', '.', 'gene_id \"chr19.g4288\"; transcript_id \"chr19.g4288.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24005660', '24005923', '.', '+', '.', 'gene_id \"chr19.g4292\"; transcript_id \"chr19.g4292.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24006924', '24007293', '.', '-', '.', 'gene_id \"chr19.g4293\"; transcript_id \"chr19.g4293.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24017966', '24018165', '.', '+', '.', 'gene_id \"chr19.g4305\"; transcript_id \"chr19.g4305.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24326636', '24326766', '.', '-', '.', 'gene_id \"chr19.g4321\"; transcript_id \"chr19.g4321.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '25603711', '25604105', '.', '-', '.', 'gene_id \"chr19.g4396\"; transcript_id \"chr19.g4396.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '25623838', '25624309', '.', '+', '.', 'gene_id \"chr19.g4400\"; transcript_id \"chr19.g4400.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '18878883', '18878912', '.', '-', '.', 'gene_id \"chr16.g8126\"; transcript_id \"chr16.g8126.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '25635221', '25635412', '.', '+', '.', 'gene_id \"chr19.g4405\"; transcript_id \"chr19.g4405.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26014643', '26014907', '.', '+', '.', 'gene_id \"chr19.g4435\"; transcript_id \"chr19.g4435.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26062660', '26062844', '.', '+', '.', 'gene_id \"chr19.g4437\"; transcript_id \"chr19.g4437.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26229708', '26229858', '.', '+', '.', 'gene_id \"chr19.g4448\"; transcript_id \"chr19.g4448.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26311359', '26311518', '.', '+', '.', 'gene_id \"chr19.g4458\"; transcript_id \"chr19.g4458.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26409431', '26409695', '.', '+', '.', 'gene_id \"chr19.g4462\"; transcript_id \"chr19.g4462.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26437174', '26437342', '.', '+', '.', 'gene_id \"chr19.g4466\"; transcript_id \"chr19.g4466.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26437500', '26437678', '.', '+', '.', 'gene_id \"chr19.g4467\"; transcript_id \"chr19.g4467.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '19388444', '19388669', '.', '-', '.', 'gene_id \"chr16.g8227\"; transcript_id \"chr16.g8227.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '19389538', '19389860', '.', '+', '.', 'gene_id \"chr16.g8230\"; transcript_id \"chr16.g8230.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '27343753', '27343797', '.', '+', '.', 'gene_id \"chr19.g4534\"; transcript_id \"chr19.g4534.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28113918', '28113979', '.', '+', '.', 'gene_id \"chr19.g4587\"; transcript_id \"chr19.g4587.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28471267', '28471488', '.', '-', '.', 'gene_id \"chr19.g4608\"; transcript_id \"chr19.g4608.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28474920', '28475071', '.', '+', '.', 'gene_id \"chr19.g4609\"; transcript_id \"chr19.g4609.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28483265', '28483542', '.', '-', '.', 'gene_id \"chr19.g4611\"; transcript_id \"chr19.g4611.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28484904', '28485079', '.', '-', '.', 'gene_id \"chr19.g4613\"; transcript_id \"chr19.g4613.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28489697', '28489856', '.', '-', '.', 'gene_id \"chr19.g4616\"; transcript_id \"chr19.g4616.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28490176', '28490424', '.', '-', '.', 'gene_id \"chr19.g4618\"; transcript_id \"chr19.g4618.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28492043', '28492195', '.', '-', '.', 'gene_id \"chr19.g4619\"; transcript_id \"chr19.g4619.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28494800', '28494984', '.', '+', '.', 'gene_id \"chr19.g4620\"; transcript_id \"chr19.g4620.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28504641', '28504795', '.', '-', '.', 'gene_id \"chr19.g4627\"; transcript_id \"chr19.g4627.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28510041', '28510303', '.', '+', '.', 'gene_id \"chr19.g4628\"; transcript_id \"chr19.g4628.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28516831', '28517008', '.', '+', '.', 'gene_id \"chr19.g4631\"; transcript_id \"chr19.g4631.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28520896', '28521057', '.', '-', '.', 'gene_id \"chr19.g4635\"; transcript_id \"chr19.g4635.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28520478', '28520922', '.', '-', '.', 'gene_id \"chr19.g4635\"; transcript_id \"chr19.g4635.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28529097', '28529309', '.', '-', '.', 'gene_id \"chr19.g4646\"; transcript_id \"chr19.g4646.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28538754', '28539437', '.', '-', '.', 'gene_id \"chr19.g4651\"; transcript_id \"chr19.g4651.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28541535', '28541838', '.', '+', '.', 'gene_id \"chr19.g4659\"; transcript_id \"chr19.g4659.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28547477', '28547934', '.', '+', '.', 'gene_id \"chr19.g4664\"; transcript_id \"chr19.g4664.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28557749', '28558008', '.', '-', '.', 'gene_id \"chr19.g4667\"; transcript_id \"chr19.g4667.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28559537', '28559829', '.', '-', '.', 'gene_id \"chr19.g4668\"; transcript_id \"chr19.g4668.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28554329', '28554501', '.', '-', '.', 'gene_id \"chr19.g4669\"; transcript_id \"chr19.g4669.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28560639', '28561042', '.', '-', '.', 'gene_id \"chr19.g4670\"; transcript_id \"chr19.g4670.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28567275', '28567510', '.', '-', '.', 'gene_id \"chr19.g4673\"; transcript_id \"chr19.g4673.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28584531', '28584858', '.', '+', '.', 'gene_id \"chr19.g4678\"; transcript_id \"chr19.g4678.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28586349', '28586500', '.', '-', '.', 'gene_id \"chr19.g4680\"; transcript_id \"chr19.g4680.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28588114', '28588378', '.', '-', '.', 'gene_id \"chr19.g4681\"; transcript_id \"chr19.g4681.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28588933', '28589145', '.', '-', '.', 'gene_id \"chr19.g4682\"; transcript_id \"chr19.g4682.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28590509', '28590874', '.', '+', '.', 'gene_id \"chr19.g4685\"; transcript_id \"chr19.g4685.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28591517', '28592330', '.', '-', '.', 'gene_id \"chr19.g4686\"; transcript_id \"chr19.g4686.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28599978', '28600988', '.', '+', '.', 'gene_id \"chr19.g4694\"; transcript_id \"chr19.g4694.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28603782', '28604743', '.', '-', '.', 'gene_id \"chr19.g4697\"; transcript_id \"chr19.g4697.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28612443', '28613470', '.', '+', '.', 'gene_id \"chr19.g4704\"; transcript_id \"chr19.g4704.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28613457', '28615055', '.', '-', '.', 'gene_id \"chr19.g4705\"; transcript_id \"chr19.g4705.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28621340', '28623348', '.', '-', '.', 'gene_id \"chr19.g4713\"; transcript_id \"chr19.g4713.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28624368', '28624608', '.', '+', '.', 'gene_id \"chr19.g4715\"; transcript_id \"chr19.g4715.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28626489', '28626575', '.', '-', '.', 'gene_id \"chr19.g4720\"; transcript_id \"chr19.g4720.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28778871', '28779070', '.', '+', '.', 'gene_id \"chr19.g4722\"; transcript_id \"chr19.g4722.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28881372', '28881550', '.', '+', '.', 'gene_id \"chr19.g4729\"; transcript_id \"chr19.g4729.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28643296', '28643604', '.', '-', '.', 'gene_id \"chr19.g4730\"; transcript_id \"chr19.g4730.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28643854', '28644481', '.', '+', '.', 'gene_id \"chr19.g4731\"; transcript_id \"chr19.g4731.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '19751034', '19751278', '.', '-', '.', 'gene_id \"chr16.g8469\"; transcript_id \"chr16.g8469.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28668624', '28668940', '.', '-', '.', 'gene_id \"chr19.g4743\"; transcript_id \"chr19.g4743.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28668981', '28669256', '.', '-', '.', 'gene_id \"chr19.g4744\"; transcript_id \"chr19.g4744.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28672289', '28672494', '.', '-', '.', 'gene_id \"chr19.g4745\"; transcript_id \"chr19.g4745.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28678210', '28678385', '.', '-', '.', 'gene_id \"chr19.g4748\"; transcript_id \"chr19.g4748.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28680466', '28680918', '.', '-', '.', 'gene_id \"chr19.g4751\"; transcript_id \"chr19.g4751.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28696419', '28696605', '.', '-', '.', 'gene_id \"chr19.g4754\"; transcript_id \"chr19.g4754.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28696637', '28696902', '.', '-', '.', 'gene_id \"chr19.g4755\"; transcript_id \"chr19.g4755.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28702068', '28702319', '.', '-', '.', 'gene_id \"chr19.g4756\"; transcript_id \"chr19.g4756.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28715591', '28716426', '.', '+', '.', 'gene_id \"chr19.g4758\"; transcript_id \"chr19.g4758.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28742721', '28742873', '.', '-', '.', 'gene_id \"chr19.g4759\"; transcript_id \"chr19.g4759.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28744123', '28744288', '.', '+', '.', 'gene_id \"chr19.g4761\"; transcript_id \"chr19.g4761.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28751339', '28751671', '.', '+', '.', 'gene_id \"chr19.g4763\"; transcript_id \"chr19.g4763.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28759523', '28759696', '.', '+', '.', 'gene_id \"chr19.g4766\"; transcript_id \"chr19.g4766.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28764870', '28765047', '.', '+', '.', 'gene_id \"chr19.g4767\"; transcript_id \"chr19.g4767.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28794234', '28794630', '.', '+', '.', 'gene_id \"chr19.g4773\"; transcript_id \"chr19.g4773.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28809155', '28809314', '.', '+', '.', 'gene_id \"chr19.g4777\"; transcript_id \"chr19.g4777.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28814803', '28815292', '.', '-', '.', 'gene_id \"chr19.g4780\"; transcript_id \"chr19.g4780.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28823365', '28823606', '.', '+', '.', 'gene_id \"chr19.g4788\"; transcript_id \"chr19.g4788.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28836553', '28836799', '.', '+', '.', 'gene_id \"chr19.g4793\"; transcript_id \"chr19.g4793.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28846845', '28847262', '.', '-', '.', 'gene_id \"chr19.g4796\"; transcript_id \"chr19.g4796.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28847259', '28847575', '.', '-', '.', 'gene_id \"chr19.g4797\"; transcript_id \"chr19.g4797.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28883003', '28883444', '.', '-', '.', 'gene_id \"chr19.g4799\"; transcript_id \"chr19.g4799.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28885432', '28885629', '.', '-', '.', 'gene_id \"chr19.g4802\"; transcript_id \"chr19.g4802.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28984071', '28984246', '.', '+', '.', 'gene_id \"chr19.g4819\"; transcript_id \"chr19.g4819.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28934184', '28934378', '.', '-', '.', 'gene_id \"chr19.g4823\"; transcript_id \"chr19.g4823.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28969017', '28969377', '.', '-', '.', 'gene_id \"chr19.g4835\"; transcript_id \"chr19.g4835.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28971482', '28971651', '.', '-', '.', 'gene_id \"chr19.g4838\"; transcript_id \"chr19.g4838.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29013933', '29014128', '.', '-', '.', 'gene_id \"chr19.g4851\"; transcript_id \"chr19.g4851.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29021055', '29021246', '.', '+', '.', 'gene_id \"chr19.g4855\"; transcript_id \"chr19.g4855.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29200398', '29200602', '.', '+', '.', 'gene_id \"chr19.g4875\"; transcript_id \"chr19.g4875.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29207296', '29207504', '.', '+', '.', 'gene_id \"chr19.g4877\"; transcript_id \"chr19.g4877.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29207551', '29207834', '.', '+', '.', 'gene_id \"chr19.g4878\"; transcript_id \"chr19.g4878.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29235809', '29235980', '.', '-', '.', 'gene_id \"chr19.g4879\"; transcript_id \"chr19.g4879.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29253008', '29253371', '.', '-', '.', 'gene_id \"chr19.g4882\"; transcript_id \"chr19.g4882.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29273630', '29273662', '.', '-', '.', 'gene_id \"chr19.g4886\"; transcript_id \"chr19.g4886.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29312294', '29312501', '.', '-', '.', 'gene_id \"chr19.g4890\"; transcript_id \"chr19.g4890.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '19922984', '19923619', '.', '+', '.', 'gene_id \"chr16.g8634\"; transcript_id \"chr16.g8634.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29412440', '29413151', '.', '-', '.', 'gene_id \"chr19.g4901\"; transcript_id \"chr19.g4901.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29436825', '29437041', '.', '+', '.', 'gene_id \"chr19.g4903\"; transcript_id \"chr19.g4903.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29440274', '29440443', '.', '+', '.', 'gene_id \"chr19.g4905\"; transcript_id \"chr19.g4905.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29449235', '29449517', '.', '-', '.', 'gene_id \"chr19.g4909\"; transcript_id \"chr19.g4909.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29446696', '29447011', '.', '+', '.', 'gene_id \"chr19.g4912\"; transcript_id \"chr19.g4912.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29451582', '29451750', '.', '+', '.', 'gene_id \"chr19.g4914\"; transcript_id \"chr19.g4914.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29534094', '29534301', '.', '-', '.', 'gene_id \"chr19.g4926\"; transcript_id \"chr19.g4926.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29534611', '29534903', '.', '+', '.', 'gene_id \"chr19.g4927\"; transcript_id \"chr19.g4927.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29617366', '29617532', '.', '-', '.', 'gene_id \"chr19.g4936\"; transcript_id \"chr19.g4936.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29681249', '29681528', '.', '+', '.', 'gene_id \"chr19.g4940\"; transcript_id \"chr19.g4940.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29686565', '29686747', '.', '+', '.', 'gene_id \"chr19.g4946\"; transcript_id \"chr19.g4946.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29692996', '29693153', '.', '+', '.', 'gene_id \"chr19.g4948\"; transcript_id \"chr19.g4948.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29696209', '29696391', '.', '-', '.', 'gene_id \"chr19.g4949\"; transcript_id \"chr19.g4949.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29705552', '29705738', '.', '-', '.', 'gene_id \"chr19.g4950\"; transcript_id \"chr19.g4950.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29707705', '29707962', '.', '+', '.', 'gene_id \"chr19.g4952\"; transcript_id \"chr19.g4952.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29711276', '29712653', '.', '-', '.', 'gene_id \"chr19.g4957\"; transcript_id \"chr19.g4957.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29723930', '29724180', '.', '+', '.', 'gene_id \"chr19.g4968\"; transcript_id \"chr19.g4968.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29818463', '29818628', '.', '+', '.', 'gene_id \"chr19.g4975\"; transcript_id \"chr19.g4975.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29831462', '29832812', '.', '-', '.', 'gene_id \"chr19.g4981\"; transcript_id \"chr19.g4981.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '20047780', '20048136', '.', '+', '.', 'gene_id \"chr16.g8719\"; transcript_id \"chr16.g8719.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '20048788', '20049131', '.', '+', '.', 'gene_id \"chr16.g8721\"; transcript_id \"chr16.g8721.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '20049245', '20049397', '.', '-', '.', 'gene_id \"chr16.g8722\"; transcript_id \"chr16.g8722.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29834462', '29834817', '.', '+', '.', 'gene_id \"chr19.g4985\"; transcript_id \"chr19.g4985.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29877008', '29877221', '.', '-', '.', 'gene_id \"chr19.g4990\"; transcript_id \"chr19.g4990.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29880167', '29880436', '.', '+', '.', 'gene_id \"chr19.g4992\"; transcript_id \"chr19.g4992.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29882171', '29882370', '.', '-', '.', 'gene_id \"chr19.g4993\"; transcript_id \"chr19.g4993.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29890391', '29890584', '.', '-', '.', 'gene_id \"chr19.g4999\"; transcript_id \"chr19.g4999.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29890580', '29890896', '.', '+', '.', 'gene_id \"chr19.g5000\"; transcript_id \"chr19.g5000.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29895891', '29896750', '.', '+', '.', 'gene_id \"chr19.g5004\"; transcript_id \"chr19.g5004.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29898765', '29899742', '.', '-', '.', 'gene_id \"chr19.g5006\"; transcript_id \"chr19.g5006.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29901718', '29901976', '.', '+', '.', 'gene_id \"chr19.g5009\"; transcript_id \"chr19.g5009.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29904298', '29904470', '.', '-', '.', 'gene_id \"chr19.g5011\"; transcript_id \"chr19.g5011.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29906993', '29907191', '.', '+', '.', 'gene_id \"chr19.g5013\"; transcript_id \"chr19.g5013.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29907429', '29907862', '.', '+', '.', 'gene_id \"chr19.g5014\"; transcript_id \"chr19.g5014.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29909451', '29909602', '.', '+', '.', 'gene_id \"chr19.g5017\"; transcript_id \"chr19.g5017.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29914134', '29914470', '.', '+', '.', 'gene_id \"chr19.g5018\"; transcript_id \"chr19.g5018.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29919977', '29920541', '.', '+', '.', 'gene_id \"chr19.g5019\"; transcript_id \"chr19.g5019.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29920691', '29920947', '.', '+', '.', 'gene_id \"chr19.g5020\"; transcript_id \"chr19.g5020.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29923850', '29924006', '.', '-', '.', 'gene_id \"chr19.g5024\"; transcript_id \"chr19.g5024.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29924580', '29924996', '.', '-', '.', 'gene_id \"chr19.g5026\"; transcript_id \"chr19.g5026.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29949580', '29949760', '.', '+', '.', 'gene_id \"chr19.g5035\"; transcript_id \"chr19.g5035.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29949788', '29949946', '.', '+', '.', 'gene_id \"chr19.g5036\"; transcript_id \"chr19.g5036.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29952545', '29952797', '.', '+', '.', 'gene_id \"chr19.g5038\"; transcript_id \"chr19.g5038.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29952856', '29953028', '.', '+', '.', 'gene_id \"chr19.g5039\"; transcript_id \"chr19.g5039.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29953006', '29953692', '.', '-', '.', 'gene_id \"chr19.g5040\"; transcript_id \"chr19.g5040.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29986172', '29986450', '.', '-', '.', 'gene_id \"chr19.g5048\"; transcript_id \"chr19.g5048.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29987446', '29987858', '.', '-', '.', 'gene_id \"chr19.g5049\"; transcript_id \"chr19.g5049.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29988392', '29988563', '.', '+', '.', 'gene_id \"chr19.g5051\"; transcript_id \"chr19.g5051.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29993386', '29993734', '.', '+', '.', 'gene_id \"chr19.g5057\"; transcript_id \"chr19.g5057.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30009625', '30009784', '.', '+', '.', 'gene_id \"chr19.g5060\"; transcript_id \"chr19.g5060.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30033266', '30033540', '.', '+', '.', 'gene_id \"chr19.g5071\"; transcript_id \"chr19.g5071.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30101226', '30101330', '.', '-', '.', 'gene_id \"chr19.g5078\"; transcript_id \"chr19.g5078.i3\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30053392', '30053707', '.', '-', '.', 'gene_id \"chr19.g5086\"; transcript_id \"chr19.g5086.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30064394', '30064559', '.', '+', '.', 'gene_id \"chr19.g5094\"; transcript_id \"chr19.g5094.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30090845', '30091003', '.', '+', '.', 'gene_id \"chr19.g5099\"; transcript_id \"chr19.g5099.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30114489', '30114680', '.', '-', '.', 'gene_id \"chr19.g5104\"; transcript_id \"chr19.g5104.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30118956', '30119119', '.', '-', '.', 'gene_id \"chr19.g5105\"; transcript_id \"chr19.g5105.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30152131', '30152478', '.', '+', '.', 'gene_id \"chr19.g5112\"; transcript_id \"chr19.g5112.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30171975', '30172177', '.', '+', '.', 'gene_id \"chr19.g5118\"; transcript_id \"chr19.g5118.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30236796', '30237037', '.', '+', '.', 'gene_id \"chr19.g5129\"; transcript_id \"chr19.g5129.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30252820', '30253388', '.', '-', '.', 'gene_id \"chr19.g5135\"; transcript_id \"chr19.g5135.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30259711', '30259865', '.', '-', '.', 'gene_id \"chr19.g5142\"; transcript_id \"chr19.g5142.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30291932', '30292196', '.', '+', '.', 'gene_id \"chr19.g5147\"; transcript_id \"chr19.g5147.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30293693', '30293866', '.', '-', '.', 'gene_id \"chr19.g5150\"; transcript_id \"chr19.g5150.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30300353', '30300550', '.', '+', '.', 'gene_id \"chr19.g5156\"; transcript_id \"chr19.g5156.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30301479', '30301641', '.', '+', '.', 'gene_id \"chr19.g5157\"; transcript_id \"chr19.g5157.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30308159', '30308387', '.', '-', '.', 'gene_id \"chr19.g5159\"; transcript_id \"chr19.g5159.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30310239', '30310474', '.', '-', '.', 'gene_id \"chr19.g5160\"; transcript_id \"chr19.g5160.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30327081', '30327304', '.', '+', '.', 'gene_id \"chr19.g5167\"; transcript_id \"chr19.g5167.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30329888', '30330192', '.', '-', '.', 'gene_id \"chr19.g5168\"; transcript_id \"chr19.g5168.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30331741', '30332075', '.', '+', '.', 'gene_id \"chr19.g5170\"; transcript_id \"chr19.g5170.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30435502', '30435995', '.', '+', '.', 'gene_id \"chr19.g5193\"; transcript_id \"chr19.g5193.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30441520', '30441794', '.', '+', '.', 'gene_id \"chr19.g5198\"; transcript_id \"chr19.g5198.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30465175', '30465435', '.', '+', '.', 'gene_id \"chr19.g5201\"; transcript_id \"chr19.g5201.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30636493', '30636739', '.', '-', '.', 'gene_id \"chr19.g5221\"; transcript_id \"chr19.g5221.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30613727', '30613891', '.', '-', '.', 'gene_id \"chr19.g5224\"; transcript_id \"chr19.g5224.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30627736', '30627948', '.', '-', '.', 'gene_id \"chr19.g5228\"; transcript_id \"chr19.g5228.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30629029', '30629343', '.', '+', '.', 'gene_id \"chr19.g5229\"; transcript_id \"chr19.g5229.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30629928', '30630095', '.', '+', '.', 'gene_id \"chr19.g5230\"; transcript_id \"chr19.g5230.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30632903', '30633200', '.', '-', '.', 'gene_id \"chr19.g5233\"; transcript_id \"chr19.g5233.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30641702', '30642197', '.', '-', '.', 'gene_id \"chr19.g5234\"; transcript_id \"chr19.g5234.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30715307', '30715523', '.', '+', '.', 'gene_id \"chr19.g5239\"; transcript_id \"chr19.g5239.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30713594', '30713746', '.', '+', '.', 'gene_id \"chr19.g5244\"; transcript_id \"chr19.g5244.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30739994', '30740257', '.', '-', '.', 'gene_id \"chr19.g5256\"; transcript_id \"chr19.g5256.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30768289', '30768450', '.', '+', '.', 'gene_id \"chr19.g5269\"; transcript_id \"chr19.g5269.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30769531', '30769680', '.', '+', '.', 'gene_id \"chr19.g5270\"; transcript_id \"chr19.g5270.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30776245', '30776423', '.', '+', '.', 'gene_id \"chr19.g5273\"; transcript_id \"chr19.g5273.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30779754', '30780027', '.', '-', '.', 'gene_id \"chr19.g5277\"; transcript_id \"chr19.g5277.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30780022', '30780184', '.', '-', '.', 'gene_id \"chr19.g5278\"; transcript_id \"chr19.g5278.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30788178', '30788580', '.', '+', '.', 'gene_id \"chr19.g5282\"; transcript_id \"chr19.g5282.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30826192', '30826365', '.', '-', '.', 'gene_id \"chr19.g5300\"; transcript_id \"chr19.g5300.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30831941', '30832134', '.', '+', '.', 'gene_id \"chr19.g5304\"; transcript_id \"chr19.g5304.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30847265', '30847653', '.', '-', '.', 'gene_id \"chr19.g5314\"; transcript_id \"chr19.g5314.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30862123', '30862413', '.', '+', '.', 'gene_id \"chr19.g5317\"; transcript_id \"chr19.g5317.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31081970', '31082158', '.', '+', '.', 'gene_id \"chr19.g5337\"; transcript_id \"chr19.g5337.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31173322', '31173381', '.', '-', '.', 'gene_id \"chr19.g5340\"; transcript_id \"chr19.g5340.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31188678', '31189001', '.', '+', '.', 'gene_id \"chr19.g5341\"; transcript_id \"chr19.g5341.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31590897', '31591066', '.', '+', '.', 'gene_id \"chr19.g5364\"; transcript_id \"chr19.g5364.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31598402', '31598669', '.', '-', '.', 'gene_id \"chr19.g5365\"; transcript_id \"chr19.g5365.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31614363', '31614558', '.', '-', '.', 'gene_id \"chr19.g5371\"; transcript_id \"chr19.g5371.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31628412', '31628580', '.', '-', '.', 'gene_id \"chr19.g5379\"; transcript_id \"chr19.g5379.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31653134', '31653310', '.', '-', '.', 'gene_id \"chr19.g5382\"; transcript_id \"chr19.g5382.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31689154', '31689424', '.', '+', '.', 'gene_id \"chr19.g5389\"; transcript_id \"chr19.g5389.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31696853', '31697090', '.', '+', '.', 'gene_id \"chr19.g5390\"; transcript_id \"chr19.g5390.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31742596', '31742851', '.', '-', '.', 'gene_id \"chr19.g5402\"; transcript_id \"chr19.g5402.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31744445', '31744616', '.', '+', '.', 'gene_id \"chr19.g5405\"; transcript_id \"chr19.g5405.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31744880', '31745054', '.', '+', '.', 'gene_id \"chr19.g5406\"; transcript_id \"chr19.g5406.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31746275', '31746481', '.', '-', '.', 'gene_id \"chr19.g5408\"; transcript_id \"chr19.g5408.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31752131', '31752292', '.', '-', '.', 'gene_id \"chr19.g5409\"; transcript_id \"chr19.g5409.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31761282', '31761437', '.', '+', '.', 'gene_id \"chr19.g5413\"; transcript_id \"chr19.g5413.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31788105', '31788252', '.', '+', '.', 'gene_id \"chr19.g5423\"; transcript_id \"chr19.g5423.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31783119', '31783334', '.', '-', '.', 'gene_id \"chr19.g5429\"; transcript_id \"chr19.g5429.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31783372', '31783637', '.', '-', '.', 'gene_id \"chr19.g5430\"; transcript_id \"chr19.g5430.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31785853', '31786027', '.', '-', '.', 'gene_id \"chr19.g5431\"; transcript_id \"chr19.g5431.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31798404', '31798645', '.', '-', '.', 'gene_id \"chr19.g5436\"; transcript_id \"chr19.g5436.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31839756', '31839926', '.', '+', '.', 'gene_id \"chr19.g5448\"; transcript_id \"chr19.g5448.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31853786', '31854332', '.', '-', '.', 'gene_id \"chr19.g5450\"; transcript_id \"chr19.g5450.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31854948', '31855120', '.', '+', '.', 'gene_id \"chr19.g5452\"; transcript_id \"chr19.g5452.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31855101', '31855585', '.', '+', '.', 'gene_id \"chr19.g5452\"; transcript_id \"chr19.g5452.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31855876', '31856043', '.', '-', '.', 'gene_id \"chr19.g5453\"; transcript_id \"chr19.g5453.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31858539', '31858849', '.', '-', '.', 'gene_id \"chr19.g5456\"; transcript_id \"chr19.g5456.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31860916', '31861211', '.', '-', '.', 'gene_id \"chr19.g5460\"; transcript_id \"chr19.g5460.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31861196', '31861366', '.', '+', '.', 'gene_id \"chr19.g5461\"; transcript_id \"chr19.g5461.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31878887', '31879210', '.', '+', '.', 'gene_id \"chr19.g5469\"; transcript_id \"chr19.g5469.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31882229', '31882381', '.', '-', '.', 'gene_id \"chr19.g5476\"; transcript_id \"chr19.g5476.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '21558045', '21558144', '.', '+', '.', 'gene_id \"chr16.g9229\"; transcript_id \"chr16.g9229.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31947009', '31947190', '.', '+', '.', 'gene_id \"chr19.g5482\"; transcript_id \"chr19.g5482.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32032672', '32033340', '.', '+', '.', 'gene_id \"chr19.g5486\"; transcript_id \"chr19.g5486.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31963542', '31963761', '.', '-', '.', 'gene_id \"chr19.g5490\"; transcript_id \"chr19.g5490.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31968307', '31968464', '.', '+', '.', 'gene_id \"chr19.g5496\"; transcript_id \"chr19.g5496.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31979919', '31980150', '.', '-', '.', 'gene_id \"chr19.g5497\"; transcript_id \"chr19.g5497.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31982090', '31982310', '.', '+', '.', 'gene_id \"chr19.g5498\"; transcript_id \"chr19.g5498.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31992171', '31992682', '.', '+', '.', 'gene_id \"chr19.g5503\"; transcript_id \"chr19.g5503.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31997096', '31997420', '.', '-', '.', 'gene_id \"chr19.g5506\"; transcript_id \"chr19.g5506.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32002926', '32003126', '.', '-', '.', 'gene_id \"chr19.g5508\"; transcript_id \"chr19.g5508.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32013760', '32013946', '.', '+', '.', 'gene_id \"chr19.g5509\"; transcript_id \"chr19.g5509.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32026406', '32026666', '.', '-', '.', 'gene_id \"chr19.g5512\"; transcript_id \"chr19.g5512.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32028967', '32029125', '.', '+', '.', 'gene_id \"chr19.g5515\"; transcript_id \"chr19.g5515.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32033627', '32033816', '.', '-', '.', 'gene_id \"chr19.g5520\"; transcript_id \"chr19.g5520.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32064247', '32064477', '.', '-', '.', 'gene_id \"chr19.g5536\"; transcript_id \"chr19.g5536.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32066104', '32066496', '.', '+', '.', 'gene_id \"chr19.g5540\"; transcript_id \"chr19.g5540.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32091341', '32091760', '.', '+', '.', 'gene_id \"chr19.g5547\"; transcript_id \"chr19.g5547.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32091862', '32092443', '.', '+', '.', 'gene_id \"chr19.g5548\"; transcript_id \"chr19.g5548.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32085641', '32085823', '.', '-', '.', 'gene_id \"chr19.g5554\"; transcript_id \"chr19.g5554.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32094410', '32094734', '.', '-', '.', 'gene_id \"chr19.g5558\"; transcript_id \"chr19.g5558.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32378303', '32378641', '.', '+', '.', 'gene_id \"chr19.g5583\"; transcript_id \"chr19.g5583.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32611172', '32611377', '.', '+', '.', 'gene_id \"chr19.g5600\"; transcript_id \"chr19.g5600.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '33492173', '33492411', '.', '+', '.', 'gene_id \"chr19.g5651\"; transcript_id \"chr19.g5651.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '34015981', '34016165', '.', '+', '.', 'gene_id \"chr19.g5743\"; transcript_id \"chr19.g5743.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '34016250', '34016552', '.', '+', '.', 'gene_id \"chr19.g5744\"; transcript_id \"chr19.g5744.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '34736811', '34736962', '.', '+', '.', 'gene_id \"chr19.g5817\"; transcript_id \"chr19.g5817.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35249796', '35249984', '.', '+', '.', 'gene_id \"chr19.g5971\"; transcript_id \"chr19.g5971.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35474346', '35474655', '.', '+', '.', 'gene_id \"chr19.g6071\"; transcript_id \"chr19.g6071.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35654106', '35654276', '.', '-', '.', 'gene_id \"chr19.g6105\"; transcript_id \"chr19.g6105.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '22929938', '22930173', '.', '+', '.', 'gene_id \"chr16.g9951\"; transcript_id \"chr16.g9951.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '22946313', '22946353', '.', '-', '.', 'gene_id \"chr16.g9962\"; transcript_id \"chr16.g9962.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35744602', '35744775', '.', '+', '.', 'gene_id \"chr19.g6180\"; transcript_id \"chr19.g6180.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35750053', '35750544', '.', '-', '.', 'gene_id \"chr19.g6191\"; transcript_id \"chr19.g6191.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35868655', '35868813', '.', '-', '.', 'gene_id \"chr19.g6268\"; transcript_id \"chr19.g6268.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35878643', '35878837', '.', '+', '.', 'gene_id \"chr19.g6273\"; transcript_id \"chr19.g6273.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35987158', '35987317', '.', '-', '.', 'gene_id \"chr19.g6296\"; transcript_id \"chr19.g6296.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '36013116', '36016392', '.', '+', '.', 'gene_id \"chr19.g6298\"; transcript_id \"chr19.g6298.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '36107345', '36107511', '.', '-', '.', 'gene_id \"chr19.g6306\"; transcript_id \"chr19.g6306.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '23439787', '23439981', '.', '+', '.', 'gene_id \"chr16.g10312\"; transcript_id \"chr16.g10312.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '36993220', '36993252', '.', '-', '.', 'gene_id \"chr19.g6469\"; transcript_id \"chr19.g6469.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37096465', '37096630', '.', '-', '.', 'gene_id \"chr19.g6482\"; transcript_id \"chr19.g6482.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37107265', '37107463', '.', '-', '.', 'gene_id \"chr19.g6493\"; transcript_id \"chr19.g6493.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37115232', '37115421', '.', '+', '.', 'gene_id \"chr19.g6497\"; transcript_id \"chr19.g6497.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37132062', '37132258', '.', '+', '.', 'gene_id \"chr19.g6504\"; transcript_id \"chr19.g6504.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37177701', '37177946', '.', '-', '.', 'gene_id \"chr19.g6522\"; transcript_id \"chr19.g6522.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37177982', '37178158', '.', '+', '.', 'gene_id \"chr19.g6523\"; transcript_id \"chr19.g6523.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37674239', '37674415', '.', '+', '.', 'gene_id \"chr19.g6724\"; transcript_id \"chr19.g6724.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37676184', '37676482', '.', '+', '.', 'gene_id \"chr19.g6726\"; transcript_id \"chr19.g6726.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37830256', '37830302', '.', '+', '.', 'gene_id \"chr19.g6766\"; transcript_id \"chr19.g6766.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37749750', '37749905', '.', '+', '.', 'gene_id \"chr19.g6769\"; transcript_id \"chr19.g6769.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38077363', '38077579', '.', '+', '.', 'gene_id \"chr19.g7060\"; transcript_id \"chr19.g7060.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38473958', '38474175', '.', '-', '.', 'gene_id \"chr19.g7238\"; transcript_id \"chr19.g7238.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38514798', '38515006', '.', '-', '.', 'gene_id \"chr19.g7267\"; transcript_id \"chr19.g7267.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '25721015', '25721318', '.', '-', '.', 'gene_id \"chr16.g11180\"; transcript_id \"chr16.g11180.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38702952', '38703002', '.', '-', '.', 'gene_id \"chr19.g7418\"; transcript_id \"chr19.g7418.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38728557', '38728725', '.', '+', '.', 'gene_id \"chr19.g7422\"; transcript_id \"chr19.g7422.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38958790', '38958999', '.', '+', '.', 'gene_id \"chr19.g7516\"; transcript_id \"chr19.g7516.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26437467', '26437767', '.', '-', '.', 'gene_id \"chr16.g11371\"; transcript_id \"chr16.g11371.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26531565', '26531764', '.', '+', '.', 'gene_id \"chr16.g11387\"; transcript_id \"chr16.g11387.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26536241', '26536534', '.', '+', '.', 'gene_id \"chr16.g11391\"; transcript_id \"chr16.g11391.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26537051', '26537377', '.', '+', '.', 'gene_id \"chr16.g11392\"; transcript_id \"chr16.g11392.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '39921825', '39921865', '.', '+', '.', 'gene_id \"chr19.g7595\"; transcript_id \"chr19.g7595.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '40510838', '40511043', '.', '-', '.', 'gene_id \"chr19.g7646\"; transcript_id \"chr19.g7646.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '40780473', '40780890', '.', '-', '.', 'gene_id \"chr19.g7698\"; transcript_id \"chr19.g7698.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26809667', '26809908', '.', '+', '.', 'gene_id \"chr16.g11580\"; transcript_id \"chr16.g11580.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41258698', '41258890', '.', '-', '.', 'gene_id \"chr19.g7815\"; transcript_id \"chr19.g7815.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26923569', '26925470', '.', '+', '.', 'gene_id \"chr16.g11667\"; transcript_id \"chr16.g11667.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41266127', '41266441', '.', '-', '.', 'gene_id \"chr19.g7826\"; transcript_id \"chr19.g7826.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41266448', '41266666', '.', '-', '.', 'gene_id \"chr19.g7827\"; transcript_id \"chr19.g7827.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41282111', '41282363', '.', '+', '.', 'gene_id \"chr19.g7847\"; transcript_id \"chr19.g7847.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41283530', '41283706', '.', '-', '.', 'gene_id \"chr19.g7850\"; transcript_id \"chr19.g7850.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41455239', '41455271', '.', '+', '.', 'gene_id \"chr19.g7904\"; transcript_id \"chr19.g7904.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41480141', '41480186', '.', '+', '.', 'gene_id \"chr19.g7908\"; transcript_id \"chr19.g7908.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42425782', '42425996', '.', '-', '.', 'gene_id \"chr19.g7966\"; transcript_id \"chr19.g7966.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42589634', '42589859', '.', '-', '.', 'gene_id \"chr19.g7978\"; transcript_id \"chr19.g7978.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42697007', '42697208', '.', '-', '.', 'gene_id \"chr19.g7992\"; transcript_id \"chr19.g7992.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42700254', '42700440', '.', '-', '.', 'gene_id \"chr19.g7993\"; transcript_id \"chr19.g7993.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42746607', '42746720', '.', '-', '.', 'gene_id \"chr19.g7999\"; transcript_id \"chr19.g7999.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42769363', '42769519', '.', '+', '.', 'gene_id \"chr19.g8000\"; transcript_id \"chr19.g8000.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42770203', '42770364', '.', '+', '.', 'gene_id \"chr19.g8002\"; transcript_id \"chr19.g8002.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42772883', '42773040', '.', '+', '.', 'gene_id \"chr19.g8003\"; transcript_id \"chr19.g8003.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43130337', '43130515', '.', '+', '.', 'gene_id \"chr19.g8014\"; transcript_id \"chr19.g8014.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43158602', '43158754', '.', '-', '.', 'gene_id \"chr19.g8018\"; transcript_id \"chr19.g8018.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43160724', '43160984', '.', '-', '.', 'gene_id \"chr19.g8019\"; transcript_id \"chr19.g8019.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43162368', '43162535', '.', '+', '.', 'gene_id \"chr19.g8020\"; transcript_id \"chr19.g8020.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43711804', '43711836', '.', '+', '.', 'gene_id \"chr19.g8041\"; transcript_id \"chr19.g8041.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43856393', '43856589', '.', '+', '.', 'gene_id \"chr19.g8047\"; transcript_id \"chr19.g8047.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43875905', '43876083', '.', '+', '.', 'gene_id \"chr19.g8048\"; transcript_id \"chr19.g8048.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '44816323', '44816442', '.', '-', '.', 'gene_id \"chr19.g8119\"; transcript_id \"chr19.g8119.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45358495', '45358731', '.', '-', '.', 'gene_id \"chr19.g8193\"; transcript_id \"chr19.g8193.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45373291', '45374098', '.', '-', '.', 'gene_id \"chr19.g8204\"; transcript_id \"chr19.g8204.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45374714', '45374952', '.', '+', '.', 'gene_id \"chr19.g8205\"; transcript_id \"chr19.g8205.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45375140', '45375350', '.', '-', '.', 'gene_id \"chr19.g8206\"; transcript_id \"chr19.g8206.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45415447', '45415607', '.', '+', '.', 'gene_id \"chr19.g8213\"; transcript_id \"chr19.g8213.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45458056', '45458404', '.', '+', '.', 'gene_id \"chr19.g8221\"; transcript_id \"chr19.g8221.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45458410', '45459971', '.', '+', '.', 'gene_id \"chr19.g8223\"; transcript_id \"chr19.g8223.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45465190', '45465781', '.', '+', '.', 'gene_id \"chr19.g8224\"; transcript_id \"chr19.g8224.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45467837', '45468052', '.', '-', '.', 'gene_id \"chr19.g8225\"; transcript_id \"chr19.g8225.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45475254', '45475432', '.', '+', '.', 'gene_id \"chr19.g8226\"; transcript_id \"chr19.g8226.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45476410', '45476567', '.', '+', '.', 'gene_id \"chr19.g8229\"; transcript_id \"chr19.g8229.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45478274', '45478430', '.', '-', '.', 'gene_id \"chr19.g8230\"; transcript_id \"chr19.g8230.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45494160', '45494426', '.', '-', '.', 'gene_id \"chr19.g8240\"; transcript_id \"chr19.g8240.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45511750', '45511968', '.', '-', '.', 'gene_id \"chr19.g8245\"; transcript_id \"chr19.g8245.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45552220', '45552405', '.', '+', '.', 'gene_id \"chr19.g8251\"; transcript_id \"chr19.g8251.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45558079', '45558973', '.', '+', '.', 'gene_id \"chr19.g8254\"; transcript_id \"chr19.g8254.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45591586', '45591798', '.', '-', '.', 'gene_id \"chr19.g8261\"; transcript_id \"chr19.g8261.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45650141', '45651062', '.', '+', '.', 'gene_id \"chr19.g8263\"; transcript_id \"chr19.g8263.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45660656', '45660809', '.', '-', '.', 'gene_id \"chr19.g8268\"; transcript_id \"chr19.g8268.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45709252', '45709402', '.', '-', '.', 'gene_id \"chr19.g8272\"; transcript_id \"chr19.g8272.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45701877', '45702032', '.', '+', '.', 'gene_id \"chr19.g8273\"; transcript_id \"chr19.g8273.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45712652', '45712978', '.', '-', '.', 'gene_id \"chr19.g8276\"; transcript_id \"chr19.g8276.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45736686', '45737236', '.', '+', '.', 'gene_id \"chr19.g8288\"; transcript_id \"chr19.g8288.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45749644', '45749816', '.', '+', '.', 'gene_id \"chr19.g8291\"; transcript_id \"chr19.g8291.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45749988', '45750214', '.', '+', '.', 'gene_id \"chr19.g8292\"; transcript_id \"chr19.g8292.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45764021', '45764264', '.', '-', '.', 'gene_id \"chr19.g8293\"; transcript_id \"chr19.g8293.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45766087', '45766538', '.', '+', '.', 'gene_id \"chr19.g8294\"; transcript_id \"chr19.g8294.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45766556', '45767178', '.', '-', '.', 'gene_id \"chr19.g8296\"; transcript_id \"chr19.g8296.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45780007', '45780257', '.', '-', '.', 'gene_id \"chr19.g8299\"; transcript_id \"chr19.g8299.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45803736', '45804025', '.', '-', '.', 'gene_id \"chr19.g8303\"; transcript_id \"chr19.g8303.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45812562', '45812723', '.', '+', '.', 'gene_id \"chr19.g8311\"; transcript_id \"chr19.g8311.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45812881', '45813244', '.', '-', '.', 'gene_id \"chr19.g8312\"; transcript_id \"chr19.g8312.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45816444', '45816766', '.', '+', '.', 'gene_id \"chr19.g8317\"; transcript_id \"chr19.g8317.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45818351', '45819888', '.', '-', '.', 'gene_id \"chr19.g8319\"; transcript_id \"chr19.g8319.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45839996', '45840952', '.', '+', '.', 'gene_id \"chr19.g8322\"; transcript_id \"chr19.g8322.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45844624', '45846574', '.', '-', '.', 'gene_id \"chr19.g8325\"; transcript_id \"chr19.g8325.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45850555', '45850750', '.', '+', '.', 'gene_id \"chr19.g8329\"; transcript_id \"chr19.g8329.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45852528', '45853645', '.', '-', '.', 'gene_id \"chr19.g8333\"; transcript_id \"chr19.g8333.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45853672', '45854730', '.', '+', '.', 'gene_id \"chr19.g8334\"; transcript_id \"chr19.g8334.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45857468', '45857706', '.', '+', '.', 'gene_id \"chr19.g8338\"; transcript_id \"chr19.g8338.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45862019', '45862343', '.', '+', '.', 'gene_id \"chr19.g8342\"; transcript_id \"chr19.g8342.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45901800', '45901956', '.', '+', '.', 'gene_id \"chr19.g8349\"; transcript_id \"chr19.g8349.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45916157', '45916353', '.', '-', '.', 'gene_id \"chr19.g8351\"; transcript_id \"chr19.g8351.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45928226', '45928404', '.', '-', '.', 'gene_id \"chr19.g8354\"; transcript_id \"chr19.g8354.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45934780', '45934938', '.', '+', '.', 'gene_id \"chr19.g8356\"; transcript_id \"chr19.g8356.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46056449', '46057126', '.', '+', '.', 'gene_id \"chr19.g8368\"; transcript_id \"chr19.g8368.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46070957', '46071126', '.', '+', '.', 'gene_id \"chr19.g8377\"; transcript_id \"chr19.g8377.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46078957', '46079482', '.', '-', '.', 'gene_id \"chr19.g8380\"; transcript_id \"chr19.g8380.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46085437', '46085775', '.', '-', '.', 'gene_id \"chr19.g8383\"; transcript_id \"chr19.g8383.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46085958', '46086119', '.', '+', '.', 'gene_id \"chr19.g8384\"; transcript_id \"chr19.g8384.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46090455', '46090863', '.', '-', '.', 'gene_id \"chr19.g8393\"; transcript_id \"chr19.g8393.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46384933', '46385000', '.', '+', '.', 'gene_id \"chr19.g8397\"; transcript_id \"chr19.g8397.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46358773', '46358976', '.', '+', '.', 'gene_id \"chr19.g8400\"; transcript_id \"chr19.g8400.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46100458', '46101383', '.', '-', '.', 'gene_id \"chr19.g8406\"; transcript_id \"chr19.g8406.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46105354', '46105583', '.', '-', '.', 'gene_id \"chr19.g8410\"; transcript_id \"chr19.g8410.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46202595', '46202833', '.', '-', '.', 'gene_id \"chr19.g8423\"; transcript_id \"chr19.g8423.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46215752', '46215968', '.', '+', '.', 'gene_id \"chr19.g8425\"; transcript_id \"chr19.g8425.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46243978', '46244177', '.', '+', '.', 'gene_id \"chr19.g8429\"; transcript_id \"chr19.g8429.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46265253', '46265493', '.', '-', '.', 'gene_id \"chr19.g8432\"; transcript_id \"chr19.g8432.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46265557', '46266168', '.', '+', '.', 'gene_id \"chr19.g8433\"; transcript_id \"chr19.g8433.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46266215', '46266413', '.', '+', '.', 'gene_id \"chr19.g8435\"; transcript_id \"chr19.g8435.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46284062', '46284288', '.', '+', '.', 'gene_id \"chr19.g8438\"; transcript_id \"chr19.g8438.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46287130', '46287282', '.', '+', '.', 'gene_id \"chr19.g8439\"; transcript_id \"chr19.g8439.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46468803', '46470691', '.', '-', '.', 'gene_id \"chr19.g8451\"; transcript_id \"chr19.g8451.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46476596', '46476834', '.', '+', '.', 'gene_id \"chr19.g8463\"; transcript_id \"chr19.g8463.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46478903', '46479060', '.', '-', '.', 'gene_id \"chr19.g8466\"; transcript_id \"chr19.g8466.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46488559', '46488722', '.', '+', '.', 'gene_id \"chr19.g8476\"; transcript_id \"chr19.g8476.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46490822', '46491319', '.', '+', '.', 'gene_id \"chr19.g8478\"; transcript_id \"chr19.g8478.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46491405', '46491781', '.', '-', '.', 'gene_id \"chr19.g8479\"; transcript_id \"chr19.g8479.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46494243', '46495117', '.', '-', '.', 'gene_id \"chr19.g8480\"; transcript_id \"chr19.g8480.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46497135', '46497505', '.', '+', '.', 'gene_id \"chr19.g8481\"; transcript_id \"chr19.g8481.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46501606', '46501758', '.', '+', '.', 'gene_id \"chr19.g8486\"; transcript_id \"chr19.g8486.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46536998', '46537398', '.', '-', '.', 'gene_id \"chr19.g8502\"; transcript_id \"chr19.g8502.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46550098', '46551069', '.', '-', '.', 'gene_id \"chr19.g8506\"; transcript_id \"chr19.g8506.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46561457', '46561647', '.', '+', '.', 'gene_id \"chr19.g8512\"; transcript_id \"chr19.g8512.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46577417', '46577576', '.', '+', '.', 'gene_id \"chr19.g8523\"; transcript_id \"chr19.g8523.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46587002', '46587153', '.', '-', '.', 'gene_id \"chr19.g8530\"; transcript_id \"chr19.g8530.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46596425', '46596645', '.', '-', '.', 'gene_id \"chr19.g8532\"; transcript_id \"chr19.g8532.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46608124', '46608446', '.', '-', '.', 'gene_id \"chr19.g8537\"; transcript_id \"chr19.g8537.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46615731', '46616073', '.', '-', '.', 'gene_id \"chr19.g8546\"; transcript_id \"chr19.g8546.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46616730', '46616881', '.', '-', '.', 'gene_id \"chr19.g8549\"; transcript_id \"chr19.g8549.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46617138', '46617305', '.', '+', '.', 'gene_id \"chr19.g8550\"; transcript_id \"chr19.g8550.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46622024', '46622595', '.', '-', '.', 'gene_id \"chr19.g8555\"; transcript_id \"chr19.g8555.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '28997262', '28997302', '.', '-', '.', 'gene_id \"chr16.g12481\"; transcript_id \"chr16.g12481.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46627377', '46628202', '.', '-', '.', 'gene_id \"chr19.g8560\"; transcript_id \"chr19.g8560.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46629871', '46630118', '.', '-', '.', 'gene_id \"chr19.g8563\"; transcript_id \"chr19.g8563.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46632947', '46633316', '.', '+', '.', 'gene_id \"chr19.g8565\"; transcript_id \"chr19.g8565.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46638633', '46638927', '.', '+', '.', 'gene_id \"chr19.g8572\"; transcript_id \"chr19.g8572.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46638905', '46639200', '.', '-', '.', 'gene_id \"chr19.g8573\"; transcript_id \"chr19.g8573.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46641274', '46641496', '.', '-', '.', 'gene_id \"chr19.g8575\"; transcript_id \"chr19.g8575.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46643251', '46643603', '.', '-', '.', 'gene_id \"chr19.g8576\"; transcript_id \"chr19.g8576.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46643668', '46643929', '.', '+', '.', 'gene_id \"chr19.g8578\"; transcript_id \"chr19.g8578.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46643930', '46645197', '.', '+', '.', 'gene_id \"chr19.g8579\"; transcript_id \"chr19.g8579.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46657376', '46657572', '.', '+', '.', 'gene_id \"chr19.g8588\"; transcript_id \"chr19.g8588.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46660748', '46661085', '.', '+', '.', 'gene_id \"chr19.g8592\"; transcript_id \"chr19.g8592.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46683593', '46683656', '.', '+', '.', 'gene_id \"chr19.g8607\"; transcript_id \"chr19.g8607.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46957229', '46957397', '.', '+', '.', 'gene_id \"chr19.g8618\"; transcript_id \"chr19.g8618.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46957755', '46957983', '.', '+', '.', 'gene_id \"chr19.g8620\"; transcript_id \"chr19.g8620.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47253563', '47253681', '.', '+', '.', 'gene_id \"chr19.g8637\"; transcript_id \"chr19.g8637.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47334838', '47335013', '.', '-', '.', 'gene_id \"chr19.g8645\"; transcript_id \"chr19.g8645.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47363976', '47364128', '.', '+', '.', 'gene_id \"chr19.g8648\"; transcript_id \"chr19.g8648.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47510470', '47510630', '.', '-', '.', 'gene_id \"chr19.g8657\"; transcript_id \"chr19.g8657.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47527172', '47527385', '.', '+', '.', 'gene_id \"chr19.g8661\"; transcript_id \"chr19.g8661.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47527635', '47527982', '.', '-', '.', 'gene_id \"chr19.g8662\"; transcript_id \"chr19.g8662.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47610886', '47611056', '.', '+', '.', 'gene_id \"chr19.g8665\"; transcript_id \"chr19.g8665.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47773792', '47773855', '.', '+', '.', 'gene_id \"chr19.g8674\"; transcript_id \"chr19.g8674.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47783865', '47784073', '.', '-', '.', 'gene_id \"chr19.g8676\"; transcript_id \"chr19.g8676.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47785289', '47785590', '.', '-', '.', 'gene_id \"chr19.g8676\"; transcript_id \"chr19.g8676.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47809687', '47809881', '.', '+', '.', 'gene_id \"chr19.g8679\"; transcript_id \"chr19.g8679.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '48243683', '48243767', '.', '-', '.', 'gene_id \"chr19.g8695\"; transcript_id \"chr19.g8695.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '48379808', '48379875', '.', '-', '.', 'gene_id \"chr19.g8699\"; transcript_id \"chr19.g8699.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '31017577', '31017624', '.', '-', '.', 'gene_id \"chr16.g12628\"; transcript_id \"chr16.g12628.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '48716866', '48717063', '.', '-', '.', 'gene_id \"chr19.g8711\"; transcript_id \"chr19.g8711.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '31112572', '31112716', '.', '+', '.', 'gene_id \"chr16.g12639\"; transcript_id \"chr16.g12639.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '49105802', '49106215', '.', '-', '.', 'gene_id \"chr19.g8728\"; transcript_id \"chr19.g8728.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '49139452', '49140419', '.', '+', '.', 'gene_id \"chr19.g8740\"; transcript_id \"chr19.g8740.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '49141524', '49141893', '.', '-', '.', 'gene_id \"chr19.g8745\"; transcript_id \"chr19.g8745.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '31544316', '31544691', '.', '-', '.', 'gene_id \"chr16.g12774\"; transcript_id \"chr16.g12774.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '31712020', '31712086', '.', '+', '.', 'gene_id \"chr16.g12789\"; transcript_id \"chr16.g12789.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '49980451', '49980716', '.', '-', '.', 'gene_id \"chr19.g8961\"; transcript_id \"chr19.g8961.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '32750491', '32750550', '.', '-', '.', 'gene_id \"chr16.g12970\"; transcript_id \"chr16.g12970.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr19', 'PASA', 'exon', '52031685', '52031952', '.', '+', '.', 'gene_id \"chr19.g9703\"; transcript_id \"chr19.g9703.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52036555', '52036728', '.', '+', '.', 'gene_id \"chr19.g9706\"; transcript_id \"chr19.g9706.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52039277', '52039443', '.', '+', '.', 'gene_id \"chr19.g9709\"; transcript_id \"chr19.g9709.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52042621', '52043081', '.', '+', '.', 'gene_id \"chr19.g9712\"; transcript_id \"chr19.g9712.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52043861', '52043913', '.', '+', '.', 'gene_id \"chr19.g9713\"; transcript_id \"chr19.g9713.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52049081', '52049254', '.', '+', '.', 'gene_id \"chr19.g9718\"; transcript_id \"chr19.g9718.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52051804', '52051969', '.', '+', '.', 'gene_id \"chr19.g9719\"; transcript_id \"chr19.g9719.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52051820', '52051969', '.', '+', '.', 'gene_id \"chr19.g9719\"; transcript_id \"chr19.g9719.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52055787', '52056028', '.', '+', '.', 'gene_id \"chr19.g9722\"; transcript_id \"chr19.g9722.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52060784', '52061039', '.', '+', '.', 'gene_id \"chr19.g9726\"; transcript_id \"chr19.g9726.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52061909', '52062517', '.', '+', '.', 'gene_id \"chr19.g9729\"; transcript_id \"chr19.g9729.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52061909', '52062517', '.', '+', '.', 'gene_id \"chr19.g9729\"; transcript_id \"chr19.g9729.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52117771', '52118032', '.', '-', '.', 'gene_id \"chr19.g9744\"; transcript_id \"chr19.g9744.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52252838', '52253002', '.', '+', '.', 'gene_id \"chr19.g9753\"; transcript_id \"chr19.g9753.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52257659', '52257873', '.', '+', '.', 'gene_id \"chr19.g9758\"; transcript_id \"chr19.g9758.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52260131', '52260289', '.', '+', '.', 'gene_id \"chr19.g9759\"; transcript_id \"chr19.g9759.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52264523', '52264698', '.', '-', '.', 'gene_id \"chr19.g9761\"; transcript_id \"chr19.g9761.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52271539', '52272114', '.', '-', '.', 'gene_id \"chr19.g9767\"; transcript_id \"chr19.g9767.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52272930', '52273148', '.', '+', '.', 'gene_id \"chr19.g9769\"; transcript_id \"chr19.g9769.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52274018', '52274407', '.', '+', '.', 'gene_id \"chr19.g9771\"; transcript_id \"chr19.g9771.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52276652', '52276913', '.', '+', '.', 'gene_id \"chr19.g9774\"; transcript_id \"chr19.g9774.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52304198', '52304408', '.', '-', '.', 'gene_id \"chr19.g9779\"; transcript_id \"chr19.g9779.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52361280', '52361440', '.', '+', '.', 'gene_id \"chr19.g9784\"; transcript_id \"chr19.g9784.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52361419', '52361834', '.', '-', '.', 'gene_id \"chr19.g9785\"; transcript_id \"chr19.g9785.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52362189', '52362393', '.', '-', '.', 'gene_id \"chr19.g9786\"; transcript_id \"chr19.g9786.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52362798', '52362974', '.', '-', '.', 'gene_id \"chr19.g9787\"; transcript_id \"chr19.g9787.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52530879', '52531103', '.', '-', '.', 'gene_id \"chr19.g9809\"; transcript_id \"chr19.g9809.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52585162', '52585860', '.', '-', '.', 'gene_id \"chr19.g9811\"; transcript_id \"chr19.g9811.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '34257310', '34257476', '.', '+', '.', 'gene_id \"chr16.g13740\"; transcript_id \"chr16.g13740.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52610647', '52610827', '.', '-', '.', 'gene_id \"chr19.g9814\"; transcript_id \"chr19.g9814.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52610647', '52610827', '.', '+', '.', 'gene_id \"chr19.g9815\"; transcript_id \"chr19.g9815.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52631015', '52631176', '.', '+', '.', 'gene_id \"chr19.g9828\"; transcript_id \"chr19.g9828.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52632815', '52632973', '.', '-', '.', 'gene_id \"chr19.g9830\"; transcript_id \"chr19.g9830.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52633634', '52633808', '.', '+', '.', 'gene_id \"chr19.g9831\"; transcript_id \"chr19.g9831.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52634088', '52634297', '.', '+', '.', 'gene_id \"chr19.g9833\"; transcript_id \"chr19.g9833.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52635641', '52635892', '.', '-', '.', 'gene_id \"chr19.g9836\"; transcript_id \"chr19.g9836.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52636224', '52636603', '.', '-', '.', 'gene_id \"chr19.g9837\"; transcript_id \"chr19.g9837.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52643944', '52644212', '.', '-', '.', 'gene_id \"chr19.g9838\"; transcript_id \"chr19.g9838.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52644998', '52645216', '.', '-', '.', 'gene_id \"chr19.g9839\"; transcript_id \"chr19.g9839.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52648887', '52649056', '.', '-', '.', 'gene_id \"chr19.g9842\"; transcript_id \"chr19.g9842.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52657665', '52657861', '.', '+', '.', 'gene_id \"chr19.g9847\"; transcript_id \"chr19.g9847.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52659434', '52659600', '.', '+', '.', 'gene_id \"chr19.g9849\"; transcript_id \"chr19.g9849.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52663932', '52664162', '.', '-', '.', 'gene_id \"chr19.g9851\"; transcript_id \"chr19.g9851.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52670257', '52670486', '.', '-', '.', 'gene_id \"chr19.g9857\"; transcript_id \"chr19.g9857.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52679062', '52679407', '.', '+', '.', 'gene_id \"chr19.g9860\"; transcript_id \"chr19.g9860.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52693339', '52693782', '.', '+', '.', 'gene_id \"chr19.g9866\"; transcript_id \"chr19.g9866.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52698436', '52698587', '.', '-', '.', 'gene_id \"chr19.g9869\"; transcript_id \"chr19.g9869.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52699398', '52699593', '.', '+', '.', 'gene_id \"chr19.g9871\"; transcript_id \"chr19.g9871.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52701621', '52702092', '.', '+', '.', 'gene_id \"chr19.g9872\"; transcript_id \"chr19.g9872.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52702119', '52702398', '.', '+', '.', 'gene_id \"chr19.g9876\"; transcript_id \"chr19.g9876.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52702385', '52702700', '.', '+', '.', 'gene_id \"chr19.g9877\"; transcript_id \"chr19.g9877.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52702707', '52703376', '.', '+', '.', 'gene_id \"chr19.g9878\"; transcript_id \"chr19.g9878.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52704861', '52705758', '.', '-', '.', 'gene_id \"chr19.g9879\"; transcript_id \"chr19.g9879.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '34367032', '34367176', '.', '+', '.', 'gene_id \"chr16.g13814\"; transcript_id \"chr16.g13814.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52707078', '52707247', '.', '+', '.', 'gene_id \"chr19.g9883\"; transcript_id \"chr19.g9883.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52720803', '52721414', '.', '+', '.', 'gene_id \"chr19.g9895\"; transcript_id \"chr19.g9895.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52721666', '52721856', '.', '-', '.', 'gene_id \"chr19.g9896\"; transcript_id \"chr19.g9896.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52722440', '52722475', '.', '+', '.', 'gene_id \"chr19.g9901\"; transcript_id \"chr19.g9901.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52722818', '52722856', '.', '+', '.', 'gene_id \"chr19.g9903\"; transcript_id \"chr19.g9903.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52817091', '52817173', '.', '+', '.', 'gene_id \"chr19.g9906\"; transcript_id \"chr19.g9906.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52836239', '52836399', '.', '+', '.', 'gene_id \"chr19.g9909\"; transcript_id \"chr19.g9909.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52877041', '52877198', '.', '-', '.', 'gene_id \"chr19.g9919\"; transcript_id \"chr19.g9919.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52878174', '52878323', '.', '+', '.', 'gene_id \"chr19.g9920\"; transcript_id \"chr19.g9920.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52928219', '52928762', '.', '+', '.', 'gene_id \"chr19.g9929\"; transcript_id \"chr19.g9929.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52886855', '52887043', '.', '+', '.', 'gene_id \"chr19.g9933\"; transcript_id \"chr19.g9933.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52894458', '52895114', '.', '+', '.', 'gene_id \"chr19.g9935\"; transcript_id \"chr19.g9935.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52896347', '52896625', '.', '+', '.', 'gene_id \"chr19.g9937\"; transcript_id \"chr19.g9937.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52896649', '52896832', '.', '+', '.', 'gene_id \"chr19.g9938\"; transcript_id \"chr19.g9938.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52897414', '52897585', '.', '-', '.', 'gene_id \"chr19.g9939\"; transcript_id \"chr19.g9939.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52898947', '52899113', '.', '+', '.', 'gene_id \"chr19.g9940\"; transcript_id \"chr19.g9940.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52900167', '52900325', '.', '-', '.', 'gene_id \"chr19.g9941\"; transcript_id \"chr19.g9941.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52900357', '52900509', '.', '-', '.', 'gene_id \"chr19.g9942\"; transcript_id \"chr19.g9942.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52907937', '52908253', '.', '-', '.', 'gene_id \"chr19.g9944\"; transcript_id \"chr19.g9944.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52927207', '52928016', '.', '+', '.', 'gene_id \"chr19.g9954\"; transcript_id \"chr19.g9954.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53055152', '53055293', '.', '-', '.', 'gene_id \"chr19.g9967\"; transcript_id \"chr19.g9967.i10\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53070256', '53070442', '.', '-', '.', 'gene_id \"chr19.g9967\"; transcript_id \"chr19.g9967.i11\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53096137', '53096287', '.', '-', '.', 'gene_id \"chr19.g9973\"; transcript_id \"chr19.g9973.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53015979', '53016237', '.', '+', '.', 'gene_id \"chr19.g9976\"; transcript_id \"chr19.g9976.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52994774', '52995006', '.', '-', '.', 'gene_id \"chr19.g9977\"; transcript_id \"chr19.g9977.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '34899621', '34899667', '.', '-', '.', 'gene_id \"chr16.g13885\"; transcript_id \"chr16.g13885.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52990379', '52990561', '.', '+', '.', 'gene_id \"chr19.g9982\"; transcript_id \"chr19.g9982.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52990695', '52991027', '.', '-', '.', 'gene_id \"chr19.g9983\"; transcript_id \"chr19.g9983.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53002266', '53002489', '.', '-', '.', 'gene_id \"chr19.g9986\"; transcript_id \"chr19.g9986.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53012987', '53013163', '.', '-', '.', 'gene_id \"chr19.g9988\"; transcript_id \"chr19.g9988.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53014476', '53014631', '.', '+', '.', 'gene_id \"chr19.g9990\"; transcript_id \"chr19.g9990.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53021270', '53021441', '.', '+', '.', 'gene_id \"chr19.g9991\"; transcript_id \"chr19.g9991.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53095521', '53095742', '.', '-', '.', 'gene_id \"chr19.g9997\"; transcript_id \"chr19.g9997.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53100165', '53100470', '.', '-', '.', 'gene_id \"chr19.g9999\"; transcript_id \"chr19.g9999.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53167751', '53168649', '.', '-', '.', 'gene_id \"chr19.g10009\"; transcript_id \"chr19.g10009.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53174122', '53174493', '.', '+', '.', 'gene_id \"chr19.g10015\"; transcript_id \"chr19.g10015.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53182015', '53182222', '.', '-', '.', 'gene_id \"chr19.g10017\"; transcript_id \"chr19.g10017.i4\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53175799', '53181836', '.', '+', '.', 'gene_id \"chr19.g10018\"; transcript_id \"chr19.g10018.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53188192', '53188720', '.', '+', '.', 'gene_id \"chr19.g10019\"; transcript_id \"chr19.g10019.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53200917', '53201337', '.', '-', '.', 'gene_id \"chr19.g10027\"; transcript_id \"chr19.g10027.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53205169', '53205468', '.', '-', '.', 'gene_id \"chr19.g10028\"; transcript_id \"chr19.g10028.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53205890', '53206107', '.', '-', '.', 'gene_id \"chr19.g10029\"; transcript_id \"chr19.g10029.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53210248', '53210459', '.', '-', '.', 'gene_id \"chr19.g10034\"; transcript_id \"chr19.g10034.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53219864', '53220171', '.', '+', '.', 'gene_id \"chr19.g10037\"; transcript_id \"chr19.g10037.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53221121', '53221357', '.', '+', '.', 'gene_id \"chr19.g10038\"; transcript_id \"chr19.g10038.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53225252', '53225490', '.', '+', '.', 'gene_id \"chr19.g10042\"; transcript_id \"chr19.g10042.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53230458', '53230688', '.', '+', '.', 'gene_id \"chr19.g10046\"; transcript_id \"chr19.g10046.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53248127', '53248293', '.', '-', '.', 'gene_id \"chr19.g10050\"; transcript_id \"chr19.g10050.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53235860', '53236162', '.', '-', '.', 'gene_id \"chr19.g10052\"; transcript_id \"chr19.g10052.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53244279', '53246652', '.', '-', '.', 'gene_id \"chr19.g10055\"; transcript_id \"chr19.g10055.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53246709', '53247219', '.', '-', '.', 'gene_id \"chr19.g10058\"; transcript_id \"chr19.g10058.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53255352', '53256127', '.', '-', '.', 'gene_id \"chr19.g10059\"; transcript_id \"chr19.g10059.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53358064', '53358260', '.', '-', '.', 'gene_id \"chr19.g10069\"; transcript_id \"chr19.g10069.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53411929', '53412454', '.', '+', '.', 'gene_id \"chr19.g10073\"; transcript_id \"chr19.g10073.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53429511', '53429712', '.', '+', '.', 'gene_id \"chr19.g10078\"; transcript_id \"chr19.g10078.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53439659', '53440281', '.', '-', '.', 'gene_id \"chr19.g10086\"; transcript_id \"chr19.g10086.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53464726', '53465650', '.', '-', '.', 'gene_id \"chr19.g10089\"; transcript_id \"chr19.g10089.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53463781', '53463938', '.', '+', '.', 'gene_id \"chr19.g10090\"; transcript_id \"chr19.g10090.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53443633', '53443818', '.', '-', '.', 'gene_id \"chr19.g10096\"; transcript_id \"chr19.g10096.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53444062', '53444259', '.', '-', '.', 'gene_id \"chr19.g10097\"; transcript_id \"chr19.g10097.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53449212', '53449672', '.', '-', '.', 'gene_id \"chr19.g10100\"; transcript_id \"chr19.g10100.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53450513', '53450692', '.', '+', '.', 'gene_id \"chr19.g10102\"; transcript_id \"chr19.g10102.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53458595', '53459072', '.', '+', '.', 'gene_id \"chr19.g10107\"; transcript_id \"chr19.g10107.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53463383', '53463617', '.', '+', '.', 'gene_id \"chr19.g10116\"; transcript_id \"chr19.g10116.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53467056', '53467246', '.', '+', '.', 'gene_id \"chr19.g10117\"; transcript_id \"chr19.g10117.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53468127', '53468285', '.', '-', '.', 'gene_id \"chr19.g10118\"; transcript_id \"chr19.g10118.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53473355', '53473526', '.', '-', '.', 'gene_id \"chr19.g10121\"; transcript_id \"chr19.g10121.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53474103', '53474436', '.', '+', '.', 'gene_id \"chr19.g10123\"; transcript_id \"chr19.g10123.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53474718', '53475030', '.', '-', '.', 'gene_id \"chr19.g10124\"; transcript_id \"chr19.g10124.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53476973', '53477310', '.', '-', '.', 'gene_id \"chr19.g10127\"; transcript_id \"chr19.g10127.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53488178', '53488686', '.', '-', '.', 'gene_id \"chr19.g10132\"; transcript_id \"chr19.g10132.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53490092', '53490360', '.', '+', '.', 'gene_id \"chr19.g10135\"; transcript_id \"chr19.g10135.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53491948', '53492169', '.', '+', '.', 'gene_id \"chr19.g10137\"; transcript_id \"chr19.g10137.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53496109', '53496268', '.', '+', '.', 'gene_id \"chr19.g10141\"; transcript_id \"chr19.g10141.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53570827', '53571052', '.', '+', '.', 'gene_id \"chr19.g10145\"; transcript_id \"chr19.g10145.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53754941', '53755022', '.', '-', '.', 'gene_id \"chr19.g10150\"; transcript_id \"chr19.g10150.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53612371', '53612572', '.', '-', '.', 'gene_id \"chr19.g10151\"; transcript_id \"chr19.g10151.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53613088', '53613403', '.', '-', '.', 'gene_id \"chr19.g10153\"; transcript_id \"chr19.g10153.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53613423', '53613609', '.', '+', '.', 'gene_id \"chr19.g10154\"; transcript_id \"chr19.g10154.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53614573', '53614964', '.', '+', '.', 'gene_id \"chr19.g10157\"; transcript_id \"chr19.g10157.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53619250', '53619412', '.', '+', '.', 'gene_id \"chr19.g10159\"; transcript_id \"chr19.g10159.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53622181', '53622341', '.', '+', '.', 'gene_id \"chr19.g10160\"; transcript_id \"chr19.g10160.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53623372', '53623576', '.', '+', '.', 'gene_id \"chr19.g10161\"; transcript_id \"chr19.g10161.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53628905', '53629132', '.', '-', '.', 'gene_id \"chr19.g10162\"; transcript_id \"chr19.g10162.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53629272', '53629536', '.', '-', '.', 'gene_id \"chr19.g10164\"; transcript_id \"chr19.g10164.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53642992', '53643181', '.', '-', '.', 'gene_id \"chr19.g10169\"; transcript_id \"chr19.g10169.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53647512', '53647680', '.', '-', '.', 'gene_id \"chr19.g10172\"; transcript_id \"chr19.g10172.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53654351', '53654756', '.', '-', '.', 'gene_id \"chr19.g10176\"; transcript_id \"chr19.g10176.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53655621', '53655863', '.', '-', '.', 'gene_id \"chr19.g10177\"; transcript_id \"chr19.g10177.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53673132', '53673380', '.', '+', '.', 'gene_id \"chr19.g10183\"; transcript_id \"chr19.g10183.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53676310', '53676540', '.', '+', '.', 'gene_id \"chr19.g10186\"; transcript_id \"chr19.g10186.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53677248', '53677634', '.', '-', '.', 'gene_id \"chr19.g10187\"; transcript_id \"chr19.g10187.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53694394', '53694821', '.', '+', '.', 'gene_id \"chr19.g10192\"; transcript_id \"chr19.g10192.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53706739', '53706934', '.', '-', '.', 'gene_id \"chr19.g10196\"; transcript_id \"chr19.g10196.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53722394', '53722740', '.', '-', '.', 'gene_id \"chr19.g10199\"; transcript_id \"chr19.g10199.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53723653', '53723901', '.', '-', '.', 'gene_id \"chr19.g10200\"; transcript_id \"chr19.g10200.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53729264', '53729419', '.', '+', '.', 'gene_id \"chr19.g10203\"; transcript_id \"chr19.g10203.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53729421', '53729589', '.', '-', '.', 'gene_id \"chr19.g10205\"; transcript_id \"chr19.g10205.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53734728', '53734925', '.', '+', '.', 'gene_id \"chr19.g10209\"; transcript_id \"chr19.g10209.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53736420', '53736571', '.', '-', '.', 'gene_id \"chr19.g10212\"; transcript_id \"chr19.g10212.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53744464', '53744621', '.', '-', '.', 'gene_id \"chr19.g10216\"; transcript_id \"chr19.g10216.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53763052', '53763261', '.', '-', '.', 'gene_id \"chr19.g10219\"; transcript_id \"chr19.g10219.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53792229', '53792487', '.', '-', '.', 'gene_id \"chr19.g10223\"; transcript_id \"chr19.g10223.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53792470', '53792645', '.', '+', '.', 'gene_id \"chr19.g10224\"; transcript_id \"chr19.g10224.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53796582', '53796937', '.', '-', '.', 'gene_id \"chr19.g10225\"; transcript_id \"chr19.g10225.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53797031', '53797212', '.', '+', '.', 'gene_id \"chr19.g10226\"; transcript_id \"chr19.g10226.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53806325', '53806590', '.', '-', '.', 'gene_id \"chr19.g10227\"; transcript_id \"chr19.g10227.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53806569', '53806739', '.', '+', '.', 'gene_id \"chr19.g10229\"; transcript_id \"chr19.g10229.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53817458', '53817634', '.', '-', '.', 'gene_id \"chr19.g10231\"; transcript_id \"chr19.g10231.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53819062', '53819349', '.', '-', '.', 'gene_id \"chr19.g10233\"; transcript_id \"chr19.g10233.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53820603', '53820753', '.', '-', '.', 'gene_id \"chr19.g10237\"; transcript_id \"chr19.g10237.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53821748', '53822186', '.', '+', '.', 'gene_id \"chr19.g10239\"; transcript_id \"chr19.g10239.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53835118', '53835284', '.', '-', '.', 'gene_id \"chr19.g10247\"; transcript_id \"chr19.g10247.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53835316', '53835497', '.', '-', '.', 'gene_id \"chr19.g10248\"; transcript_id \"chr19.g10248.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53835976', '53836384', '.', '-', '.', 'gene_id \"chr19.g10249\"; transcript_id \"chr19.g10249.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53835538', '53835994', '.', '-', '.', 'gene_id \"chr19.g10251\"; transcript_id \"chr19.g10251.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53837170', '53837360', '.', '+', '.', 'gene_id \"chr19.g10253\"; transcript_id \"chr19.g10253.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53841267', '53841432', '.', '+', '.', 'gene_id \"chr19.g10254\"; transcript_id \"chr19.g10254.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53856476', '53856668', '.', '-', '.', 'gene_id \"chr19.g10265\"; transcript_id \"chr19.g10265.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53859456', '53859612', '.', '+', '.', 'gene_id \"chr19.g10266\"; transcript_id \"chr19.g10266.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53861945', '53862499', '.', '+', '.', 'gene_id \"chr19.g10270\"; transcript_id \"chr19.g10270.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53863400', '53863597', '.', '-', '.', 'gene_id \"chr19.g10271\"; transcript_id \"chr19.g10271.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53875219', '53875475', '.', '-', '.', 'gene_id \"chr19.g10274\"; transcript_id \"chr19.g10274.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53875219', '53875475', '.', '-', '.', 'gene_id \"chr19.g10274\"; transcript_id \"chr19.g10274.i2\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53937412', '53937610', '.', '+', '.', 'gene_id \"chr19.g10277\"; transcript_id \"chr19.g10277.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53880462', '53880619', '.', '+', '.', 'gene_id \"chr19.g10284\"; transcript_id \"chr19.g10284.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53880605', '53880838', '.', '+', '.', 'gene_id \"chr19.g10285\"; transcript_id \"chr19.g10285.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53887739', '53888145', '.', '+', '.', 'gene_id \"chr19.g10286\"; transcript_id \"chr19.g10286.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53891515', '53891680', '.', '-', '.', 'gene_id \"chr19.g10290\"; transcript_id \"chr19.g10290.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53898790', '53899005', '.', '+', '.', 'gene_id \"chr19.g10291\"; transcript_id \"chr19.g10291.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53914745', '53914956', '.', '-', '.', 'gene_id \"chr19.g10294\"; transcript_id \"chr19.g10294.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53917623', '53917879', '.', '+', '.', 'gene_id \"chr19.g10295\"; transcript_id \"chr19.g10295.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53928153', '53928349', '.', '+', '.', 'gene_id \"chr19.g10297\"; transcript_id \"chr19.g10297.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '35801322', '35801451', '.', '-', '.', 'gene_id \"chr16.g14185\"; transcript_id \"chr16.g14185.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53929795', '53929985', '.', '+', '.', 'gene_id \"chr19.g10298\"; transcript_id \"chr19.g10298.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53931868', '53932163', '.', '-', '.', 'gene_id \"chr19.g10303\"; transcript_id \"chr19.g10303.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53933152', '53933387', '.', '+', '.', 'gene_id \"chr19.g10304\"; transcript_id \"chr19.g10304.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53942520', '53942937', '.', '-', '.', 'gene_id \"chr19.g10313\"; transcript_id \"chr19.g10313.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53943090', '53943242', '.', '-', '.', 'gene_id \"chr19.g10315\"; transcript_id \"chr19.g10315.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53945401', '53945551', '.', '-', '.', 'gene_id \"chr19.g10316\"; transcript_id \"chr19.g10316.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53946133', '53947360', '.', '+', '.', 'gene_id \"chr19.g10319\"; transcript_id \"chr19.g10319.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53950544', '53951477', '.', '-', '.', 'gene_id \"chr19.g10323\"; transcript_id \"chr19.g10323.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53955126', '53955287', '.', '-', '.', 'gene_id \"chr19.g10327\"; transcript_id \"chr19.g10327.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53955306', '53955495', '.', '+', '.', 'gene_id \"chr19.g10328\"; transcript_id \"chr19.g10328.i1\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53740754', '53740813', '100', '-', '.', 'gene_id \"chr19.g10346\"; transcript_id \"blat.proc18128.chain_24925\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1847628', '1847978', '100', '-', '.', 'gene_id \"chr19.g10399\"; transcript_id \"blat.proc18128.chain_12999\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37129931', '37130142', '100', '+', '.', 'gene_id \"chr19.g10406\"; transcript_id \"blat.proc18128.chain_26874\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30463705', '30463870', '100', '-', '.', 'gene_id \"chr19.g10410\"; transcript_id \"blat.proc18128.chain_11857\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45912765', '45912926', '100', '+', '.', 'gene_id \"chr19.g10427\"; transcript_id \"blat.proc18128.chain_22831\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30294844', '30295008', '100', '+', '.', 'gene_id \"chr19.g10446\"; transcript_id \"blat.proc18128.chain_11673\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53719929', '53720133', '100', '-', '.', 'gene_id \"chr19.g10453\"; transcript_id \"blat.proc18128.chain_24275\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2735921', '2736094', '100', '+', '.', 'gene_id \"chr19.g10486\"; transcript_id \"blat.proc18128.chain_12096\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22795504', '22795666', '100', '+', '.', 'gene_id \"chr19.g10489\"; transcript_id \"blat.proc18128.chain_4565\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38788543', '38788726', '100', '+', '.', 'gene_id \"chr19.g10490\"; transcript_id \"blat.proc18128.chain_26585\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7603970', '7604023', '100', '-', '.', 'gene_id \"chr19.g10506\"; transcript_id \"blat.proc28144.chain_16825\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30235146', '30235348', '100', '-', '.', 'gene_id \"chr19.g10519\"; transcript_id \"blat.proc18128.chain_11815\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3490156', '3490379', '100', '-', '.', 'gene_id \"chr19.g10541\"; transcript_id \"blat.proc18128.chain_14998\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35210160', '35210312', '100', '+', '.', 'gene_id \"chr19.g10543\"; transcript_id \"blat.proc18128.chain_26839\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2908729', '2908967', '100', '+', '.', 'gene_id \"chr19.g10548\"; transcript_id \"blat.proc4587.chain_1161\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45858720', '45858891', '100', '+', '.', 'gene_id \"chr19.g10565\"; transcript_id \"blat.proc18128.chain_22813\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13029852', '13030242', '100', '+', '.', 'gene_id \"chr19.g10596\"; transcript_id \"blat.proc18128.chain_17384\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20306424', '20306771', '100', '-', '.', 'gene_id \"chr19.g10606\"; transcript_id \"blat.proc18128.chain_5181\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23757039', '23757138', '100', '+', '.', 'gene_id \"chr19.g10623\"; transcript_id \"blat.proc4587.chain_13658\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30186261', '30186419', '100', '+', '.', 'gene_id \"chr19.g10625\"; transcript_id \"blat.proc18128.chain_11889\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31854776', '31854937', '100', '+', '.', 'gene_id \"chr19.g10632\"; transcript_id \"blat.proc4587.chain_10171\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13402307', '13402356', '100', '-', '.', 'gene_id \"chr19.g10639\"; transcript_id \"blat.proc18128.chain_5526\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1429803', '1430016', '100', '-', '.', 'gene_id \"chr19.g10642\"; transcript_id \"blat.proc18128.chain_15173\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30751845', '30752009', '100', '+', '.', 'gene_id \"chr19.g10645\"; transcript_id \"blat.proc18128.chain_10086\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20827970', '20828042', '100', '-', '.', 'gene_id \"chr19.g10650\"; transcript_id \"blat.proc18128.chain_2605\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46639221', '46639381', '100', '-', '.', 'gene_id \"chr19.g10660\"; transcript_id \"blat.proc18128.chain_23365\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3474003', '3474199', '100', '+', '.', 'gene_id \"chr19.g10692\"; transcript_id \"blat.proc18128.chain_12622\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23342623', '23342799', '100', '-', '.', 'gene_id \"chr19.g10695\"; transcript_id \"blat.proc18128.chain_7771\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3536254', '3536470', '100', '-', '.', 'gene_id \"chr19.g10735\"; transcript_id \"blat.proc18128.chain_15996\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22614677', '22614874', '100', '+', '.', 'gene_id \"chr19.g10739\"; transcript_id \"blat.proc18128.chain_499\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14525629', '14525890', '100', '+', '.', 'gene_id \"chr19.g10741\"; transcript_id \"blat.proc18128.chain_18333\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13051153', '13051407', '100', '+', '.', 'gene_id \"chr19.g10744\"; transcript_id \"blat.proc18128.chain_18906\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32087875', '32088058', '100', '+', '.', 'gene_id \"chr19.g10751\"; transcript_id \"blat.proc18128.chain_22341\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35582865', '35583032', '100', '+', '.', 'gene_id \"chr19.g10752\"; transcript_id \"blat.proc18128.chain_26809\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12820986', '12821179', '100', '+', '.', 'gene_id \"chr19.g10763\"; transcript_id \"blat.proc18128.chain_17114\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30433743', '30433798', '100', '-', '.', 'gene_id \"chr19.g10781\"; transcript_id \"blat.proc18128.chain_10839\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53919875', '53920059', '100', '-', '.', 'gene_id \"chr19.g10796\"; transcript_id \"blat.proc18128.chain_26417\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3307796', '3307981', '100', '+', '.', 'gene_id \"chr19.g10804\"; transcript_id \"blat.proc18128.chain_12235\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '40962359', '40962569', '100', '-', '.', 'gene_id \"chr19.g10816\"; transcript_id \"blat.proc18128.chain_26880\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2294475', '2294647', '100', '+', '.', 'gene_id \"chr19.g10841\"; transcript_id \"blat.proc18128.chain_16700\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17174627', '17174840', '100', '+', '.', 'gene_id \"chr19.g10861\"; transcript_id \"blat.proc18128.chain_8043\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28663737', '28663934', '100', '-', '.', 'gene_id \"chr19.g10869\"; transcript_id \"blat.proc18128.chain_9004\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22961946', '22962124', '100', '-', '.', 'gene_id \"chr19.g10924\"; transcript_id \"blat.proc18128.chain_7595\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12972493', '12972646', '100', '-', '.', 'gene_id \"chr19.g10931\"; transcript_id \"blat.proc18128.chain_18604\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53855287', '53855529', '100', '+', '.', 'gene_id \"chr19.g10947\"; transcript_id \"blat.proc18128.chain_26397\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23964207', '23964363', '100', '-', '.', 'gene_id \"chr19.g10956\"; transcript_id \"blat.proc18128.chain_8144\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23374793', '23374949', '100', '-', '.', 'gene_id \"chr19.g10970\"; transcript_id \"blat.proc18128.chain_6138\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3214247', '3214452', '100', '-', '.', 'gene_id \"chr19.g10994\"; transcript_id \"blat.proc18128.chain_11944\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '121320', '121491', '100', '-', '.', 'gene_id \"chr19.g10997\"; transcript_id \"blat.proc4587.chain_925\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32427950', '32428026', '100', '-', '.', 'gene_id \"chr19.g11014\"; transcript_id \"blat.proc18128.chain_22364\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35892401', '35892569', '100', '-', '.', 'gene_id \"chr19.g11026\"; transcript_id \"blat.proc18128.chain_26683\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1244367', '1244453', '100', '+', '.', 'gene_id \"chr19.g11029\"; transcript_id \"blat.proc18128.chain_14448\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11753944', '11754129', '100', '-', '.', 'gene_id \"chr19.g11041\"; transcript_id \"blat.proc18128.chain_17826\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30640973', '30641165', '100', '-', '.', 'gene_id \"chr19.g11049\"; transcript_id \"blat.proc18128.chain_10273\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13023964', '13024173', '100', '-', '.', 'gene_id \"chr19.g11076\"; transcript_id \"blat.proc18128.chain_17345\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24012802', '24012975', '100', '+', '.', 'gene_id \"chr19.g11099\"; transcript_id \"blat.proc18128.chain_1309\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2194993', '2195149', '100', '-', '.', 'gene_id \"chr19.g11113\"; transcript_id \"blat.proc18128.chain_15543\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28597277', '28597548', '100', '+', '.', 'gene_id \"chr19.g11129\"; transcript_id \"blat.proc18128.chain_10843\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '9112323', '9112487', '100', '+', '.', 'gene_id \"chr19.g11131\"; transcript_id \"blat.proc18128.chain_20780\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2966332', '2966541', '100', '-', '.', 'gene_id \"chr19.g11144\"; transcript_id \"blat.proc18128.chain_13257\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8699994', '8700151', '100', '+', '.', 'gene_id \"chr19.g11192\"; transcript_id \"blat.proc18128.chain_6263\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2896948', '2897213', '100', '-', '.', 'gene_id \"chr19.g11196\"; transcript_id \"blat.proc18128.chain_13425\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28576142', '28576316', '100', '-', '.', 'gene_id \"chr19.g11254\"; transcript_id \"blat.proc18128.chain_11875\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20445825', '20446050', '100', '-', '.', 'gene_id \"chr19.g11273\"; transcript_id \"blat.proc18128.chain_145\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3504328', '3504413', '100', '+', '.', 'gene_id \"chr19.g11315\"; transcript_id \"blat.proc18128.chain_15874\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46605028', '46605222', '100', '-', '.', 'gene_id \"chr19.g11320\"; transcript_id \"blat.proc18128.chain_22817\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22182330', '22182501', '100', '+', '.', 'gene_id \"chr19.g11333\"; transcript_id \"blat.proc18128.chain_8216\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19890591', '19890771', '100', '+', '.', 'gene_id \"chr19.g11350\"; transcript_id \"blat.proc18128.chain_7662\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3278293', '3278448', '100', '-', '.', 'gene_id \"chr19.g11358\"; transcript_id \"blat.proc18128.chain_13874\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47861626', '47861806', '100', '-', '.', 'gene_id \"chr19.g11384\"; transcript_id \"blat.proc18128.chain_21969\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18924384', '18924541', '100', '+', '.', 'gene_id \"chr19.g11389\"; transcript_id \"blat.proc18128.chain_8041\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17039517', '17039693', '100', '-', '.', 'gene_id \"chr19.g11393\"; transcript_id \"blat.proc18128.chain_1818\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4274852', '4274893', '100', '-', '.', 'gene_id \"chr19.g11422\"; transcript_id \"blat.proc18128.chain_8974\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20579133', '20579282', '100', '-', '.', 'gene_id \"chr19.g11443\"; transcript_id \"blat.proc18128.chain_325\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2805472', '2805636', '100', '-', '.', 'gene_id \"chr19.g11452\"; transcript_id \"blat.proc18128.chain_12208\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18626893', '18626941', '100', '+', '.', 'gene_id \"chr19.g11461\"; transcript_id \"blat.proc4005.chain_967\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18858213', '18858928', '100', '+', '.', 'gene_id \"chr19.g11487\"; transcript_id \"blat.proc18128.chain_3338\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1751228', '1751401', '100', '+', '.', 'gene_id \"chr19.g11501\"; transcript_id \"blat.proc18128.chain_12222\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53689399', '53689642', '100', '-', '.', 'gene_id \"chr19.g11509\"; transcript_id \"blat.proc18128.chain_24373\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37088105', '37088317', '100', '-', '.', 'gene_id \"chr19.g11510\"; transcript_id \"blat.proc18128.chain_26816\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31989421', '31989624', '100', '-', '.', 'gene_id \"chr19.g11511\"; transcript_id \"blat.proc18128.chain_20812\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1870452', '1870660', '100', '+', '.', 'gene_id \"chr19.g11530\"; transcript_id \"blat.proc18128.chain_12571\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19948952', '19949134', '100', '+', '.', 'gene_id \"chr19.g11542\"; transcript_id \"blat.proc18128.chain_7783\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '51457445', '51457723', '100', '+', '.', 'gene_id \"chr19.g11578\"; transcript_id \"blat.proc28144.chain_21182\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2148659', '2148861', '100', '-', '.', 'gene_id \"chr19.g11580\"; transcript_id \"blat.proc18128.chain_16749\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22423915', '22424106', '100', '-', '.', 'gene_id \"chr19.g11586\"; transcript_id \"blat.proc18128.chain_7791\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28865882', '28866214', '100', '+', '.', 'gene_id \"chr19.g11591\"; transcript_id \"blat.proc18128.chain_9411\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11841330', '11841613', '100', '+', '.', 'gene_id \"chr19.g11600\"; transcript_id \"blat.proc18128.chain_20211\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28637702', '28637878', '100', '+', '.', 'gene_id \"chr19.g11616\"; transcript_id \"blat.proc18128.chain_9038\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2284817', '2284910', '100', '+', '.', 'gene_id \"chr19.g11645\"; transcript_id \"blat.proc18128.chain_13643\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2469930', '2469967', '100', '+', '.', 'gene_id \"chr19.g11667\"; transcript_id \"blat.proc4587.chain_4968\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3618996', '3619188', '100', '-', '.', 'gene_id \"chr19.g11678\"; transcript_id \"blat.proc18128.chain_12504\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30072170', '30072365', '100', '-', '.', 'gene_id \"chr19.g11702\"; transcript_id \"blat.proc18128.chain_11796\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18888078', '18888396', '100', '+', '.', 'gene_id \"chr19.g11738\"; transcript_id \"blat.proc18128.chain_1709\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20708721', '20709023', '100', '+', '.', 'gene_id \"chr19.g11743\"; transcript_id \"blat.proc18128.chain_7710\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52931301', '52931686', '100', '+', '.', 'gene_id \"chr19.g11753\"; transcript_id \"blat.proc18128.chain_26125\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '44315', '44484', '100', '+', '.', 'gene_id \"chr19.g11816\"; transcript_id \"blat.proc18128.chain_14709\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30153012', '30153162', '100', '-', '.', 'gene_id \"chr19.g11827\"; transcript_id \"blat.proc18128.chain_11855\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12938503', '12938570', '100', '-', '.', 'gene_id \"chr19.g11841\"; transcript_id \"blat.proc18128.chain_18042\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28884935', '28885092', '100', '+', '.', 'gene_id \"chr19.g11848\"; transcript_id \"blat.proc18128.chain_11769\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23267699', '23267886', '100', '-', '.', 'gene_id \"chr19.g11856\"; transcript_id \"blat.proc18128.chain_7953\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30072609', '30072762', '100', '-', '.', 'gene_id \"chr19.g11866\"; transcript_id \"blat.proc18128.chain_11862\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12293321', '12293505', '100', '+', '.', 'gene_id \"chr19.g11868\"; transcript_id \"blat.proc18128.chain_20574\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22791321', '22791505', '100', '-', '.', 'gene_id \"chr19.g11871\"; transcript_id \"blat.proc18128.chain_308\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26298686', '26298854', '100', '+', '.', 'gene_id \"chr19.g11877\"; transcript_id \"blat.proc18128.chain_8482\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12042875', '12042990', '100', '+', '.', 'gene_id \"chr19.g11914\"; transcript_id \"blat.proc18128.chain_20514\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14584701', '14584757', '100', '+', '.', 'gene_id \"chr19.g11922\"; transcript_id \"blat.proc18128.chain_18907\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '887805', '887969', '100', '-', '.', 'gene_id \"chr19.g11927\"; transcript_id \"blat.proc18128.chain_16651\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2063477', '2063639', '100', '-', '.', 'gene_id \"chr19.g11939\"; transcript_id \"blat.proc18128.chain_15398\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15950029', '15950144', '100', '+', '.', 'gene_id \"chr19.g11952\"; transcript_id \"blat.proc18128.chain_17572\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3542410', '3542636', '100', '-', '.', 'gene_id \"chr19.g11955\"; transcript_id \"blat.proc18128.chain_12033\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38557560', '38557635', '100', '+', '.', 'gene_id \"chr19.g11975\"; transcript_id \"blat.proc18128.chain_26908\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2289777', '2289962', '100', '-', '.', 'gene_id \"chr19.g11979\"; transcript_id \"blat.proc18128.chain_13759\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23392325', '23392489', '100', '-', '.', 'gene_id \"chr19.g12018\"; transcript_id \"blat.proc18128.chain_80\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52973039', '52973207', '100', '-', '.', 'gene_id \"chr19.g12028\"; transcript_id \"blat.proc18128.chain_26483\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28873247', '28873411', '100', '-', '.', 'gene_id \"chr19.g12058\"; transcript_id \"blat.proc18128.chain_8906\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '986479', '986840', '100', '+', '.', 'gene_id \"chr19.g12061\"; transcript_id \"blat.proc18128.chain_12265\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2166957', '2167103', '100', '-', '.', 'gene_id \"chr19.g12076\"; transcript_id \"blat.proc18128.chain_15547\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20307025', '20307198', '100', '+', '.', 'gene_id \"chr19.g12128\"; transcript_id \"blat.proc18128.chain_7279\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12141502', '12141648', '100', '+', '.', 'gene_id \"chr19.g12140\"; transcript_id \"blat.proc18128.chain_20265\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3588207', '3588750', '100', '-', '.', 'gene_id \"chr19.g12149\"; transcript_id \"blat.proc18128.chain_14629\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28640469', '28640528', '100', '+', '.', 'gene_id \"chr19.g12156\"; transcript_id \"blat.proc18128.chain_9931\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29916786', '29916939', '100', '-', '.', 'gene_id \"chr19.g12188\"; transcript_id \"blat.proc18128.chain_10461\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3331740', '3331958', '100', '-', '.', 'gene_id \"chr19.g12269\"; transcript_id \"blat.proc18128.chain_12422\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2233918', '2234130', '100', '-', '.', 'gene_id \"chr19.g12278\"; transcript_id \"blat.proc18128.chain_12882\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '114705', '114948', '100', '+', '.', 'gene_id \"chr19.g12296\"; transcript_id \"blat.proc18128.chain_16554\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31297353', '31297502', '100', '-', '.', 'gene_id \"chr19.g12305\"; transcript_id \"blat.proc18128.chain_11590\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18374378', '18374592', '100', '-', '.', 'gene_id \"chr19.g12313\"; transcript_id \"blat.proc18128.chain_545\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1847330', '1847488', '100', '-', '.', 'gene_id \"chr19.g12334\"; transcript_id \"blat.proc18128.chain_12998\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23810846', '23811002', '100', '-', '.', 'gene_id \"chr19.g12337\"; transcript_id \"blat.proc18128.chain_8147\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '41329777', '41329974', '100', '+', '.', 'gene_id \"chr19.g12387\"; transcript_id \"blat.proc18128.chain_26875\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19955156', '19955312', '100', '-', '.', 'gene_id \"chr19.g12391\"; transcript_id \"blat.proc18128.chain_4789\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22871144', '22871368', '100', '+', '.', 'gene_id \"chr19.g12397\"; transcript_id \"blat.proc18128.chain_1317\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23143564', '23143777', '100', '-', '.', 'gene_id \"chr19.g12460\"; transcript_id \"blat.proc18128.chain_2486\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13025949', '13026331', '100', '-', '.', 'gene_id \"chr19.g12464\"; transcript_id \"blat.proc18128.chain_18975\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46795586', '46795618', '100', '+', '.', 'gene_id \"chr19.g12497\"; transcript_id \"blat.proc18128.chain_16613\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22871501', '22871707', '100', '+', '.', 'gene_id \"chr19.g12529\"; transcript_id \"blat.proc18128.chain_1319\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1958959', '1959215', '100', '+', '.', 'gene_id \"chr19.g12532\"; transcript_id \"blat.proc18128.chain_15321\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20856727', '20856974', '100', '+', '.', 'gene_id \"chr19.g12562\"; transcript_id \"blat.proc18128.chain_6036\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12767891', '12768147', '100', '+', '.', 'gene_id \"chr19.g12582\"; transcript_id \"blat.proc18128.chain_20391\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28682946', '28683183', '100', '-', '.', 'gene_id \"chr19.g12608\"; transcript_id \"blat.proc18128.chain_9339\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28542227', '28542421', '100', '+', '.', 'gene_id \"chr19.g12655\"; transcript_id \"blat.proc18128.chain_9463\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16922571', '16922735', '100', '+', '.', 'gene_id \"chr19.g12659\"; transcript_id \"blat.proc18128.chain_7858\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19284116', '19284520', '100', '-', '.', 'gene_id \"chr19.g12688\"; transcript_id \"blat.proc4587.chain_453\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2638972', '2639140', '100', '-', '.', 'gene_id \"chr19.g12746\"; transcript_id \"blat.proc28144.chain_6881\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28605298', '28605477', '100', '+', '.', 'gene_id \"chr19.g12764\"; transcript_id \"blat.proc18128.chain_10710\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17942688', '17942957', '100', '-', '.', 'gene_id \"chr19.g12787\"; transcript_id \"blat.proc18128.chain_4370\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13031003', '13031090', '100', '-', '.', 'gene_id \"chr19.g12821\"; transcript_id \"blat.proc18128.chain_18746\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2909116', '2909312', '100', '+', '.', 'gene_id \"chr19.g12836\"; transcript_id \"blat.proc4587.chain_1162\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16776804', '16776968', '100', '-', '.', 'gene_id \"chr19.g12844\"; transcript_id \"blat.proc18128.chain_212\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13263402', '13263579', '100', '+', '.', 'gene_id \"chr19.g12851\"; transcript_id \"blat.proc18128.chain_17932\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23420226', '23420532', '100', '-', '.', 'gene_id \"chr19.g12855\"; transcript_id \"blat.proc18128.chain_1981\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52845273', '52845504', '100', '-', '.', 'gene_id \"chr19.g12873\"; transcript_id \"blat.proc18128.chain_9096\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46074798', '46074969', '100', '-', '.', 'gene_id \"chr19.g12901\"; transcript_id \"blat.proc18128.chain_24112\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31791651', '31791898', '100', '-', '.', 'gene_id \"chr19.g12925\"; transcript_id \"blat.proc4587.chain_1775\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53928761', '53929071', '100', '-', '.', 'gene_id \"chr19.g12944\"; transcript_id \"blat.proc18128.chain_24142\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28485772', '28485894', '100', '+', '.', 'gene_id \"chr19.g12948\"; transcript_id \"blat.proc18128.chain_11694\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2777119', '2777182', '100', '+', '.', 'gene_id \"chr19.g13013\"; transcript_id \"blat.proc18128.chain_13655\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17623285', '17623573', '100', '-', '.', 'gene_id \"chr19.g13014\"; transcript_id \"blat.proc18128.chain_7888\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2686775', '2686983', '100', '-', '.', 'gene_id \"chr19.g13032\"; transcript_id \"blat.proc18128.chain_12106\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23785957', '23786190', '100', '-', '.', 'gene_id \"chr19.g13066\"; transcript_id \"blat.proc4587.chain_5389\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20461276', '20461597', '100', '+', '.', 'gene_id \"chr19.g13188\"; transcript_id \"blat.proc18128.chain_1678\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17296963', '17297130', '100', '-', '.', 'gene_id \"chr19.g13194\"; transcript_id \"blat.proc18128.chain_7572\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46949048', '46949290', '100', '+', '.', 'gene_id \"chr19.g13205\"; transcript_id \"blat.proc18128.chain_24009\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1923706', '1923894', '100', '+', '.', 'gene_id \"chr19.g13220\"; transcript_id \"blat.proc4587.chain_7573\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35676856', '35676896', '100', '+', '.', 'gene_id \"chr19.g13247\"; transcript_id \"blat.proc18128.chain_2841\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2991912', '2992115', '100', '+', '.', 'gene_id \"chr19.g13279\"; transcript_id \"blat.proc18128.chain_16672\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28617311', '28617381', '100', '-', '.', 'gene_id \"chr19.g13326\"; transcript_id \"blat.proc18128.chain_9848\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17292784', '17292942', '100', '-', '.', 'gene_id \"chr19.g13347\"; transcript_id \"blat.proc18128.chain_977\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16624362', '16624518', '100', '+', '.', 'gene_id \"chr19.g13350\"; transcript_id \"blat.proc18128.chain_7500\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1367728', '1367912', '100', '+', '.', 'gene_id \"chr19.g13357\"; transcript_id \"blat.proc18128.chain_16806\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30793150', '30793418', '100', '+', '.', 'gene_id \"chr19.g13363\"; transcript_id \"blat.proc18128.chain_8742\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14695875', '14696050', '100', '+', '.', 'gene_id \"chr19.g13399\"; transcript_id \"blat.proc18128.chain_19463\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23175428', '23175593', '100', '-', '.', 'gene_id \"chr19.g13410\"; transcript_id \"blat.proc18128.chain_442\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38137779', '38137973', '100', '+', '.', 'gene_id \"chr19.g13460\"; transcript_id \"blat.proc18128.chain_26899\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2216949', '2217171', '100', '-', '.', 'gene_id \"chr19.g13463\"; transcript_id \"blat.proc18128.chain_15908\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30550535', '30550661', '100', '+', '.', 'gene_id \"chr19.g13471\"; transcript_id \"blat.proc18128.chain_10430\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22901240', '22901992', '100', '-', '.', 'gene_id \"chr19.g13478\"; transcript_id \"blat.proc18128.chain_4125\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13038456', '13038644', '100', '+', '.', 'gene_id \"chr19.g13502\"; transcript_id \"blat.proc18128.chain_17188\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17149147', '17149313', '100', '-', '.', 'gene_id \"chr19.g13509\"; transcript_id \"blat.proc18128.chain_8171\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1123287', '1123577', '100', '-', '.', 'gene_id \"chr19.g13516\"; transcript_id \"blat.proc18128.chain_16755\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12112547', '12112768', '100', '-', '.', 'gene_id \"chr19.g13541\"; transcript_id \"blat.proc18128.chain_19700\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3542688', '3543072', '100', '+', '.', 'gene_id \"chr19.g13552\"; transcript_id \"blat.proc18128.chain_12034\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22463221', '22463378', '100', '+', '.', 'gene_id \"chr19.g13583\"; transcript_id \"blat.proc18128.chain_7831\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46062044', '46062206', '100', '+', '.', 'gene_id \"chr19.g13591\"; transcript_id \"blat.proc18128.chain_23063\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32007356', '32007554', '100', '+', '.', 'gene_id \"chr19.g13592\"; transcript_id \"blat.proc18128.chain_21630\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16795984', '16796232', '100', '+', '.', 'gene_id \"chr19.g13598\"; transcript_id \"blat.proc18128.chain_199\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23217963', '23218278', '100', '-', '.', 'gene_id \"chr19.g13607\"; transcript_id \"blat.proc18128.chain_1738\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '145912', '146082', '100', '-', '.', 'gene_id \"chr19.g13612\"; transcript_id \"blat.proc18128.chain_16973\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '96238', '96412', '100', '+', '.', 'gene_id \"chr19.g13635\"; transcript_id \"blat.proc18128.chain_16718\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20617771', '20618035', '100', '+', '.', 'gene_id \"chr19.g13661\"; transcript_id \"blat.proc18128.chain_633\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32069384', '32069574', '100', '+', '.', 'gene_id \"chr19.g13687\"; transcript_id \"blat.proc18128.chain_20842\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30830646', '30830841', '100', '+', '.', 'gene_id \"chr19.g13695\"; transcript_id \"blat.proc18128.chain_8610\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22620709', '22621009', '100', '+', '.', 'gene_id \"chr19.g13702\"; transcript_id \"blat.proc18128.chain_916\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1348157', '1348332', '100', '+', '.', 'gene_id \"chr19.g13721\"; transcript_id \"blat.proc18128.chain_12937\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3599378', '3599523', '100', '-', '.', 'gene_id \"chr19.g13759\"; transcript_id \"blat.proc18128.chain_13009\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16790700', '16790875', '100', '+', '.', 'gene_id \"chr19.g13765\"; transcript_id \"blat.proc18128.chain_7950\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13155978', '13156396', '100', '-', '.', 'gene_id \"chr19.g13773\"; transcript_id \"blat.proc18128.chain_20277\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17394961', '17395102', '100', '-', '.', 'gene_id \"chr19.g13786\"; transcript_id \"blat.proc18128.chain_7112\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12907994', '12908326', '100', '-', '.', 'gene_id \"chr19.g13793\"; transcript_id \"blat.proc18128.chain_18455\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52841195', '52841335', '100', '+', '.', 'gene_id \"chr19.g13820\"; transcript_id \"blat.proc18128.chain_26482\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1722383', '1722702', '100', '+', '.', 'gene_id \"chr19.g13821\"; transcript_id \"blat.proc18128.chain_16019\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29531755', '29531974', '100', '-', '.', 'gene_id \"chr19.g13836\"; transcript_id \"blat.proc18128.chain_8962\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45811410', '45811615', '100', '-', '.', 'gene_id \"chr19.g13851\"; transcript_id \"blat.proc18128.chain_23154\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31791987', '31792269', '100', '+', '.', 'gene_id \"chr19.g13852\"; transcript_id \"blat.proc4587.chain_1776\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '122311', '122592', '100', '+', '.', 'gene_id \"chr19.g13853\"; transcript_id \"blat.proc4587.chain_923\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30515052', '30515208', '100', '-', '.', 'gene_id \"chr19.g13868\"; transcript_id \"blat.proc18128.chain_8973\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2605073', '2605252', '100', '-', '.', 'gene_id \"chr19.g13885\"; transcript_id \"blat.proc18128.chain_12503\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20617587', '20617753', '100', '-', '.', 'gene_id \"chr19.g13913\"; transcript_id \"blat.proc18128.chain_636\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30633645', '30633809', '100', '+', '.', 'gene_id \"chr19.g13916\"; transcript_id \"blat.proc18128.chain_8637\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15963235', '15963405', '100', '+', '.', 'gene_id \"chr19.g13921\"; transcript_id \"blat.proc18128.chain_17655\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19946275', '19946432', '100', '-', '.', 'gene_id \"chr19.g13951\"; transcript_id \"blat.proc18128.chain_4788\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12092975', '12093243', '100', '+', '.', 'gene_id \"chr19.g14007\"; transcript_id \"blat.proc18128.chain_19148\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30248107', '30248265', '100', '-', '.', 'gene_id \"chr19.g14009\"; transcript_id \"blat.proc18128.chain_10573\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '24140200', '24140258', '100', '+', '.', 'gene_id \"chr19.g14020\"; transcript_id \"blat.proc28144.chain_10600\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7067522', '7067553', '100', '+', '.', 'gene_id \"chr19.g14040\"; transcript_id \"blat.proc18128.chain_7053\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28682728', '28682897', '100', '-', '.', 'gene_id \"chr19.g14059\"; transcript_id \"blat.proc18128.chain_9338\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11917690', '11917997', '100', '-', '.', 'gene_id \"chr19.g14079\"; transcript_id \"blat.proc18128.chain_20465\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42701714', '42701925', '100', '+', '.', 'gene_id \"chr19.g14091\"; transcript_id \"blat.proc18128.chain_22770\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17954627', '17954797', '100', '+', '.', 'gene_id \"chr19.g14105\"; transcript_id \"blat.proc18128.chain_243\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17848430', '17848618', '100', '-', '.', 'gene_id \"chr19.g14168\"; transcript_id \"blat.proc18128.chain_166\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23785733', '23785872', '100', '-', '.', 'gene_id \"chr19.g14184\"; transcript_id \"blat.proc4587.chain_5388\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46208130', '46208309', '100', '+', '.', 'gene_id \"chr19.g14216\"; transcript_id \"blat.proc18128.chain_23989\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53109486', '53109638', '100', '-', '.', 'gene_id \"chr19.g14233\"; transcript_id \"blat.proc18128.chain_26489\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52938740', '52938941', '100', '+', '.', 'gene_id \"chr19.g14240\"; transcript_id \"blat.proc18128.chain_25335\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23806285', '23806337', '100', '-', '.', 'gene_id \"chr19.g14249\"; transcript_id \"blat.proc18128.chain_209\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '88532', '88695', '100', '+', '.', 'gene_id \"chr19.g14258\"; transcript_id \"blat.proc18128.chain_12328\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22454350', '22454706', '100', '+', '.', 'gene_id \"chr19.g14264\"; transcript_id \"blat.proc18128.chain_211\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2233579', '2233911', '100', '-', '.', 'gene_id \"chr19.g14285\"; transcript_id \"blat.proc18128.chain_12881\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42496437', '42496513', '100', '-', '.', 'gene_id \"chr19.g14304\"; transcript_id \"blat.proc18128.chain_19320\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11275406', '11275445', '100', '-', '.', 'gene_id \"chr19.g14326\"; transcript_id \"blat.proc4587.chain_4932\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18542670', '18542871', '100', '+', '.', 'gene_id \"chr19.g14398\"; transcript_id \"blat.proc18128.chain_399\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7318311', '7318347', '100', '+', '.', 'gene_id \"chr19.g14408\"; transcript_id \"blat.proc28144.chain_21387\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11961720', '11961979', '100', '-', '.', 'gene_id \"chr19.g14414\"; transcript_id \"blat.proc18128.chain_18007\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30033558', '30033726', '100', '-', '.', 'gene_id \"chr19.g14436\"; transcript_id \"blat.proc18128.chain_9471\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31792494', '31792647', '100', '-', '.', 'gene_id \"chr19.g14439\"; transcript_id \"blat.proc4587.chain_1774\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45738387', '45738553', '100', '-', '.', 'gene_id \"chr19.g14451\"; transcript_id \"blat.proc18128.chain_24034\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3567518', '3567755', '100', '-', '.', 'gene_id \"chr19.g14456\"; transcript_id \"blat.proc18128.chain_14570\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28697979', '28698168', '100', '-', '.', 'gene_id \"chr19.g14459\"; transcript_id \"blat.proc18128.chain_9431\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13336749', '13336908', '100', '+', '.', 'gene_id \"chr19.g14471\"; transcript_id \"blat.proc18128.chain_17356\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19016601', '19016771', '100', '-', '.', 'gene_id \"chr19.g14475\"; transcript_id \"blat.proc18128.chain_8197\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1923024', '1923197', '100', '+', '.', 'gene_id \"chr19.g14479\"; transcript_id \"blat.proc4587.chain_7571\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53800251', '53800435', '100', '+', '.', 'gene_id \"chr19.g14496\"; transcript_id \"blat.proc18128.chain_26426\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3001268', '3001585', '100', '+', '.', 'gene_id \"chr19.g14497\"; transcript_id \"blat.proc18128.chain_15903\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19905766', '19905956', '100', '-', '.', 'gene_id \"chr19.g14498\"; transcript_id \"blat.proc18128.chain_2810\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29919542', '29919724', '100', '+', '.', 'gene_id \"chr19.g14509\"; transcript_id \"blat.proc18128.chain_11587\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35545730', '35545906', '100', '+', '.', 'gene_id \"chr19.g14511\"; transcript_id \"blat.proc18128.chain_26948\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2687112', '2687320', '100', '-', '.', 'gene_id \"chr19.g14542\"; transcript_id \"blat.proc18128.chain_12107\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37138894', '37139060', '100', '+', '.', 'gene_id \"chr19.g14562\"; transcript_id \"blat.proc18128.chain_26830\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '25696519', '25696700', '100', '+', '.', 'gene_id \"chr19.g14567\"; transcript_id \"blat.proc18128.chain_8595\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52916134', '52916360', '100', '-', '.', 'gene_id \"chr19.g14571\"; transcript_id \"blat.proc18128.chain_24744\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '14724308', '14724491', '100', '-', '.', 'gene_id \"chr19.g14577\"; transcript_id \"blat.proc18128.chain_17180\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1013983', '1014132', '100', '+', '.', 'gene_id \"chr19.g14598\"; transcript_id \"blat.proc18128.chain_16037\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52845561', '52845754', '100', '+', '.', 'gene_id \"chr19.g14603\"; transcript_id \"blat.proc18128.chain_24504\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2360674', '2360837', '100', '+', '.', 'gene_id \"chr19.g14612\"; transcript_id \"blat.proc18128.chain_14459\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17293923', '17294197', '100', '-', '.', 'gene_id \"chr19.g14638\"; transcript_id \"blat.proc4587.chain_10339\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '45877082', '45877259', '100', '+', '.', 'gene_id \"chr19.g14672\"; transcript_id \"blat.proc18128.chain_22847\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29229361', '29229523', '100', '-', '.', 'gene_id \"chr19.g14690\"; transcript_id \"blat.proc18128.chain_8872\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28654048', '28654336', '100', '+', '.', 'gene_id \"chr19.g14692\"; transcript_id \"blat.proc18128.chain_9090\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38568351', '38568531', '100', '-', '.', 'gene_id \"chr19.g14719\"; transcript_id \"blat.proc18128.chain_26814\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18397704', '18397951', '100', '+', '.', 'gene_id \"chr19.g14738\"; transcript_id \"blat.proc4587.chain_8413\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42469440', '42469607', '100', '-', '.', 'gene_id \"chr19.g14741\"; transcript_id \"blat.proc18128.chain_22769\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '921955', '922107', '100', '+', '.', 'gene_id \"chr19.g14775\"; transcript_id \"blat.proc18128.chain_12178\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3686072', '3686138', '100', '+', '.', 'gene_id \"chr19.g14836\"; transcript_id \"blat.proc18128.chain_3026\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3598867', '3599116', '100', '-', '.', 'gene_id \"chr19.g14864\"; transcript_id \"blat.proc18128.chain_12139\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3408670', '3408810', '100', '-', '.', 'gene_id \"chr19.g14905\"; transcript_id \"blat.proc18128.chain_13223\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29838925', '29839131', '100', '+', '.', 'gene_id \"chr19.g14918\"; transcript_id \"blat.proc18128.chain_11518\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '32375650', '32375865', '100', '-', '.', 'gene_id \"chr19.g14940\"; transcript_id \"blat.proc18128.chain_22322\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2908476', '2908719', '100', '+', '.', 'gene_id \"chr19.g14943\"; transcript_id \"blat.proc4587.chain_1165\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1870043', '1870300', '100', '+', '.', 'gene_id \"chr19.g14951\"; transcript_id \"blat.proc18128.chain_12573\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30792415', '30793135', '100', '+', '.', 'gene_id \"chr19.g14958\"; transcript_id \"blat.proc18128.chain_8741\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28883993', '28884157', '100', '+', '.', 'gene_id \"chr19.g14964\"; transcript_id \"blat.proc18128.chain_10921\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19902362', '19902479', '100', '+', '.', 'gene_id \"chr19.g14965\"; transcript_id \"blat.proc18128.chain_3194\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3065463', '3065579', '100', '-', '.', 'gene_id \"chr19.g14969\"; transcript_id \"blat.proc18128.chain_14347\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53857615', '53857821', '100', '-', '.', 'gene_id \"chr19.g14971\"; transcript_id \"blat.proc18128.chain_26464\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30171645', '30171868', '100', '+', '.', 'gene_id \"chr19.g14978\"; transcript_id \"blat.proc18128.chain_10311\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '40822521', '40822743', '100', '-', '.', 'gene_id \"chr19.g15007\"; transcript_id \"blat.proc4587.chain_1086\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '47185616', '47185832', '100', '+', '.', 'gene_id \"chr19.g15009\"; transcript_id \"blat.proc18128.chain_22978\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '20591899', '20592057', '100', '-', '.', 'gene_id \"chr19.g15013\"; transcript_id \"blat.proc18128.chain_40\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3437827', '3437981', '100', '+', '.', 'gene_id \"chr19.g15021\"; transcript_id \"blat.proc18128.chain_12224\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19026371', '19026530', '100', '+', '.', 'gene_id \"chr19.g15024\"; transcript_id \"blat.proc18128.chain_7857\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '895039', '895327', '100', '-', '.', 'gene_id \"chr19.g15027\"; transcript_id \"blat.proc18128.chain_12233\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30513973', '30514146', '100', '+', '.', 'gene_id \"chr19.g15028\"; transcript_id \"blat.proc18128.chain_8657\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12835144', '12835338', '100', '+', '.', 'gene_id \"chr19.g15045\"; transcript_id \"blat.proc18128.chain_17352\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3316152', '3316396', '100', '+', '.', 'gene_id \"chr19.g15050\"; transcript_id \"blat.proc18128.chain_16918\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3039407', '3039627', '100', '+', '.', 'gene_id \"chr19.g15054\"; transcript_id \"blat.proc18128.chain_12115\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2994264', '2994511', '100', '-', '.', 'gene_id \"chr19.g15071\"; transcript_id \"blat.proc18128.chain_12152\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '4970859', '4971308', '100', '+', '.', 'gene_id \"chr19.g15081\"; transcript_id \"blat.proc18128.chain_21781\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17686943', '17687099', '100', '-', '.', 'gene_id \"chr19.g15082\"; transcript_id \"blat.proc18128.chain_8239\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18541508', '18541695', '100', '+', '.', 'gene_id \"chr19.g15085\"; transcript_id \"blat.proc18128.chain_198\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16929525', '16929796', '100', '-', '.', 'gene_id \"chr19.g15089\"; transcript_id \"blat.proc18128.chain_1173\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19028306', '19028481', '100', '-', '.', 'gene_id \"chr19.g15090\"; transcript_id \"blat.proc18128.chain_155\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2049437', '2049688', '100', '-', '.', 'gene_id \"chr19.g15115\"; transcript_id \"blat.proc4587.chain_6842\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3324687', '3324871', '100', '+', '.', 'gene_id \"chr19.g15135\"; transcript_id \"blat.proc18128.chain_17028\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3502056', '3502389', '100', '-', '.', 'gene_id \"chr19.g15161\"; transcript_id \"blat.proc18128.chain_15778\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2649077', '2649302', '100', '+', '.', 'gene_id \"chr19.g15184\"; transcript_id \"blat.proc18128.chain_12212\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17489177', '17489644', '100', '-', '.', 'gene_id \"chr19.g15189\"; transcript_id \"blat.proc18128.chain_69\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '100239', '100480', '100', '-', '.', 'gene_id \"chr19.g15219\"; transcript_id \"blat.proc4587.chain_8947\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '123053', '123206', '100', '+', '.', 'gene_id \"chr19.g15230\"; transcript_id \"blat.proc18128.chain_13506\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16687942', '16688068', '100', '+', '.', 'gene_id \"chr19.g15231\"; transcript_id \"blat.proc18128.chain_1489\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35907715', '35907865', '100', '+', '.', 'gene_id \"chr19.g15240\"; transcript_id \"blat.proc18128.chain_26817\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18436042', '18436283', '100', '-', '.', 'gene_id \"chr19.g15243\"; transcript_id \"blat.proc18128.chain_367\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52610287', '52610449', '100', '+', '.', 'gene_id \"chr19.g15256\"; transcript_id \"blat.proc4587.chain_13111\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '43159462', '43159585', '100', '-', '.', 'gene_id \"chr19.g15278\"; transcript_id \"blat.proc18128.chain_22572\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '977302', '977466', '100', '+', '.', 'gene_id \"chr19.g15281\"; transcript_id \"blat.proc18128.chain_6961\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '34579092', '34579171', '100', '-', '.', 'gene_id \"chr19.g15294\"; transcript_id \"blat.proc4587.chain_7555\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23298366', '23298523', '100', '+', '.', 'gene_id \"chr19.g15300\"; transcript_id \"blat.proc18128.chain_222\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53703653', '53703805', '100', '+', '.', 'gene_id \"chr19.g15327\"; transcript_id \"blat.proc18128.chain_26394\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2986275', '2986524', '100', '-', '.', 'gene_id \"chr19.g15389\"; transcript_id \"blat.proc18128.chain_16963\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52627941', '52628176', '100', '+', '.', 'gene_id \"chr19.g15409\"; transcript_id \"blat.proc18128.chain_24363\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22920484', '22920709', '100', '+', '.', 'gene_id \"chr19.g15413\"; transcript_id \"blat.proc18128.chain_1316\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23310735', '23310908', '100', '-', '.', 'gene_id \"chr19.g15419\"; transcript_id \"blat.proc18128.chain_221\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15269778', '15269974', '100', '-', '.', 'gene_id \"chr19.g15441\"; transcript_id \"blat.proc18128.chain_20530\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22698125', '22698281', '100', '+', '.', 'gene_id \"chr19.g15444\"; transcript_id \"blat.proc18128.chain_2217\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19415913', '19416357', '100', '+', '.', 'gene_id \"chr19.g15501\"; transcript_id \"blat.proc18128.chain_7407\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53683911', '53684083', '100', '+', '.', 'gene_id \"chr19.g15512\"; transcript_id \"blat.proc18128.chain_25714\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '51940', '52118', '100', '-', '.', 'gene_id \"chr19.g15517\"; transcript_id \"blat.proc18128.chain_16538\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '28581452', '28581635', '100', '+', '.', 'gene_id \"chr19.g15555\"; transcript_id \"blat.proc18128.chain_11867\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53846936', '53847121', '100', '-', '.', 'gene_id \"chr19.g15591\"; transcript_id \"blat.proc18128.chain_26516\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22999758', '23000239', '100', '-', '.', 'gene_id \"chr19.g15593\"; transcript_id \"blat.proc18128.chain_4188\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '18378681', '18379037', '100', '+', '.', 'gene_id \"chr19.g15617\"; transcript_id \"blat.proc18128.chain_1146\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2329137', '2329353', '100', '+', '.', 'gene_id \"chr19.g15633\"; transcript_id \"blat.proc18128.chain_12270\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23959821', '23959984', '100', '-', '.', 'gene_id \"chr19.g15638\"; transcript_id \"blat.proc18128.chain_8078\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30634324', '30634493', '100', '-', '.', 'gene_id \"chr19.g15657\"; transcript_id \"blat.proc18128.chain_11570\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53199074', '53199264', '100', '+', '.', 'gene_id \"chr19.g15676\"; transcript_id \"blat.proc18128.chain_26463\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31661717', '31661877', '100', '+', '.', 'gene_id \"chr19.g15680\"; transcript_id \"blat.proc18128.chain_20837\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '38565665', '38565820', '100', '+', '.', 'gene_id \"chr19.g15707\"; transcript_id \"blat.proc18128.chain_26795\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23218331', '23218563', '100', '-', '.', 'gene_id \"chr19.g15713\"; transcript_id \"blat.proc18128.chain_1745\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '8725065', '8725248', '100', '+', '.', 'gene_id \"chr19.g15726\"; transcript_id \"blat.proc18128.chain_20731\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12908936', '12909535', '100', '+', '.', 'gene_id \"chr19.g15730\"; transcript_id \"blat.proc18128.chain_17656\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29885947', '29886149', '100', '-', '.', 'gene_id \"chr19.g15752\"; transcript_id \"blat.proc18128.chain_11668\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31603108', '31603263', '100', '+', '.', 'gene_id \"chr19.g15775\"; transcript_id \"blat.proc18128.chain_20847\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35320407', '35320681', '100', '+', '.', 'gene_id \"chr19.g15785\"; transcript_id \"blat.proc18128.chain_26736\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13178518', '13178670', '100', '+', '.', 'gene_id \"chr19.g15814\"; transcript_id \"blat.proc18128.chain_20321\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16913315', '16913495', '100', '+', '.', 'gene_id \"chr19.g15821\"; transcript_id \"blat.proc18128.chain_484\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19425081', '19425282', '100', '+', '.', 'gene_id \"chr19.g15840\"; transcript_id \"blat.proc18128.chain_498\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31972042', '31972330', '100', '+', '.', 'gene_id \"chr19.g15843\"; transcript_id \"blat.proc18128.chain_20983\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26181009', '26181315', '100', '+', '.', 'gene_id \"chr19.g15844\"; transcript_id \"blat.proc18128.chain_8516\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12066294', '12066417', '100', '+', '.', 'gene_id \"chr19.g15859\"; transcript_id \"blat.proc18128.chain_20360\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2803352', '2803505', '100', '-', '.', 'gene_id \"chr19.g15870\"; transcript_id \"blat.proc18128.chain_13176\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30295771', '30295912', '100', '-', '.', 'gene_id \"chr19.g15877\"; transcript_id \"blat.proc18128.chain_8935\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30295371', '30295528', '100', '+', '.', 'gene_id \"chr19.g15880\"; transcript_id \"blat.proc18128.chain_8936\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46557837', '46557990', '100', '-', '.', 'gene_id \"chr19.g15893\"; transcript_id \"blat.proc18128.chain_23964\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '53094615', '53094823', '100', '+', '.', 'gene_id \"chr19.g15912\"; transcript_id \"blat.proc18128.chain_26445\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '37991788', '37991944', '100', '-', '.', 'gene_id \"chr19.g15913\"; transcript_id \"blat.proc18128.chain_26877\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13170802', '13170960', '100', '-', '.', 'gene_id \"chr19.g15982\"; transcript_id \"blat.proc18128.chain_17908\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17622763', '17622991', '100', '+', '.', 'gene_id \"chr19.g16025\"; transcript_id \"blat.proc18128.chain_210\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30515372', '30515571', '100', '-', '.', 'gene_id \"chr19.g16046\"; transcript_id \"blat.proc18128.chain_11584\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22837622', '22837722', '100', '+', '.', 'gene_id \"chr19.g16052\"; transcript_id \"blat.proc18128.chain_5897\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35168487', '35168687', '100', '+', '.', 'gene_id \"chr19.g16074\"; transcript_id \"blat.proc18128.chain_26745\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2028223', '2028428', '100', '+', '.', 'gene_id \"chr19.g16086\"; transcript_id \"blat.proc18128.chain_15828\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29719870', '29720059', '100', '-', '.', 'gene_id \"chr19.g16110\"; transcript_id \"blat.proc4587.chain_9474\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12310233', '12310443', '100', '+', '.', 'gene_id \"chr19.g16128\"; transcript_id \"blat.proc18128.chain_17407\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '30620451', '30620610', '100', '+', '.', 'gene_id \"chr19.g16195\"; transcript_id \"blat.proc18128.chain_11881\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '3587570', '3588197', '100', '+', '.', 'gene_id \"chr19.g16238\"; transcript_id \"blat.proc18128.chain_14547\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22900627', '22901047', '100', '-', '.', 'gene_id \"chr19.g16239\"; transcript_id \"blat.proc18128.chain_4003\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2749131', '2749300', '100', '+', '.', 'gene_id \"chr19.g16242\"; transcript_id \"blat.proc18128.chain_12127\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52715787', '52715991', '100', '-', '.', 'gene_id \"chr19.g16250\"; transcript_id \"blat.proc18128.chain_26400\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12976479', '12976699', '100', '+', '.', 'gene_id \"chr19.g16310\"; transcript_id \"blat.proc18128.chain_17514\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23525658', '23525889', '100', '+', '.', 'gene_id \"chr19.g16386\"; transcript_id \"blat.proc18128.chain_507\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26040577', '26040707', '100', '+', '.', 'gene_id \"chr19.g16436\"; transcript_id \"blat.proc18128.chain_8588\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52696875', '52697170', '100', '-', '.', 'gene_id \"chr19.g16443\"; transcript_id \"blat.proc18128.chain_26474\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '13570894', '13571244', '100', '+', '.', 'gene_id \"chr19.g16459\"; transcript_id \"blat.proc18128.chain_12825\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2034894', '2035049', '100', '+', '.', 'gene_id \"chr19.g16467\"; transcript_id \"blat.proc18128.chain_12138\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12927284', '12927466', '100', '-', '.', 'gene_id \"chr19.g16501\"; transcript_id \"blat.proc18128.chain_20221\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '29891369', '29891541', '100', '+', '.', 'gene_id \"chr19.g16502\"; transcript_id \"blat.proc18128.chain_11733\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2206476', '2206644', '100', '+', '.', 'gene_id \"chr19.g16517\"; transcript_id \"blat.proc18128.chain_12090\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '121538', '122120', '100', '-', '.', 'gene_id \"chr19.g16526\"; transcript_id \"blat.proc4587.chain_924\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '35685970', '35686138', '100', '+', '.', 'gene_id \"chr19.g16538\"; transcript_id \"blat.proc18128.chain_26532\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2654003', '2654282', '100', '+', '.', 'gene_id \"chr19.g16576\"; transcript_id \"blat.proc18128.chain_15323\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '7470747', '7470809', '100', '+', '.', 'gene_id \"chr19.g16581\"; transcript_id \"blat.proc4005.chain_19001\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2006164', '2006381', '100', '-', '.', 'gene_id \"chr19.g16614\"; transcript_id \"blat.proc18128.chain_16907\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22829488', '22829714', '100', '+', '.', 'gene_id \"chr19.g16620\"; transcript_id \"blat.proc18128.chain_1973\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2109145', '2109212', '100', '-', '.', 'gene_id \"chr19.g16743\"; transcript_id \"blat.proc18128.chain_5935\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52028262', '52028430', '100', '-', '.', 'gene_id \"chr19.g16854\"; transcript_id \"blat.proc18128.chain_26407\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2967586', '2967658', '100', '+', '.', 'gene_id \"chr19.g16884\"; transcript_id \"blat.proc18128.chain_12279\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1153627', '1153816', '100', '-', '.', 'gene_id \"chr19.g16893\"; transcript_id \"blat.proc18128.chain_12945\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '15337710', '15337833', '100', '-', '.', 'gene_id \"chr19.g16939\"; transcript_id \"blat.proc18128.chain_17364\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '42456258', '42456368', '100', '-', '.', 'gene_id \"chr19.g16972\"; transcript_id \"blat.proc18128.chain_22553\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23257926', '23258056', '100', '+', '.', 'gene_id \"chr19.g16985\"; transcript_id \"blat.proc18128.chain_3288\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '17387011', '17387182', '100', '-', '.', 'gene_id \"chr19.g17008\"; transcript_id \"blat.proc18128.chain_7679\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22748599', '22748780', '100', '+', '.', 'gene_id \"chr19.g17018\"; transcript_id \"blat.proc18128.chain_292\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12165929', '12166044', '100', '-', '.', 'gene_id \"chr19.g17092\"; transcript_id \"blat.proc18128.chain_17530\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '2159026', '2159274', '100', '+', '.', 'gene_id \"chr19.g17109\"; transcript_id \"blat.proc18128.chain_12075\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '26373991', '26374158', '100', '+', '.', 'gene_id \"chr19.g17126\"; transcript_id \"blat.proc18128.chain_8578\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '46558435', '46558656', '100', '+', '.', 'gene_id \"chr19.g17136\"; transcript_id \"blat.proc18128.chain_23314\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16901986', '16902170', '100', '+', '.', 'gene_id \"chr19.g17162\"; transcript_id \"blat.proc18128.chain_1152\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12835429', '12835592', '100', '+', '.', 'gene_id \"chr19.g17183\"; transcript_id \"blat.proc18128.chain_17353\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '16497811', '16498021', '100', '-', '.', 'gene_id \"chr19.g17224\"; transcript_id \"blat.proc18128.chain_7790\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23302827', '23302877', '100', '-', '.', 'gene_id \"chr19.g17256\"; transcript_id \"blat.proc18128.chain_2231\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '31987492', '31987660', '100', '-', '.', 'gene_id \"chr19.g17276\"; transcript_id \"blat.proc18128.chain_20883\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '22774610', '22774781', '100', '+', '.', 'gene_id \"chr19.g17335\"; transcript_id \"blat.proc18128.chain_7527\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '23277251', '23277406', '100', '+', '.', 'gene_id \"chr19.g17417\"; transcript_id \"blat.proc18128.chain_4203\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12778451', '12778621', '100', '+', '.', 'gene_id \"chr19.g17448\"; transcript_id \"blat.proc18128.chain_17990\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '19069650', '19069749', '100', '+', '.', 'gene_id \"chr19.g17456\"; transcript_id \"blat.proc18128.chain_7899\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1923342', '1923594', '100', '+', '.', 'gene_id \"chr19.g17468\"; transcript_id \"blat.proc4587.chain_7574\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '12827090', '12827264', '100', '+', '.', 'gene_id \"chr19.g17490\"; transcript_id \"blat.proc18128.chain_20508\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '1930339', '1930519', '100', '+', '.', 'gene_id \"chr19.g17527\"; transcript_id \"blat.proc18128.chain_16808\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '52627604', '52627759', '100', '+', '.', 'gene_id \"chr19.g17529\"; transcript_id \"blat.proc18128.chain_24362\";']\n",
      "negative:  ['chr19', 'PASA', 'exon', '11852309', '11852503', '100', '-', '.', 'gene_id \"chr19.g17530\"; transcript_id \"blat.proc18128.chain_18368\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '35809654', '35809829', '.', '-', '.', 'gene_id \"chr16.g14262\"; transcript_id \"chr16.g14262.i1\";']\n",
      "negative:  ['scaffold_1684', 'PASA', 'exon', '46097', '46126', '.', '-', '.', 'gene_id \"scaffold_1684.g7\"; transcript_id \"scaffold_1684.g7.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '36556561', '36556886', '.', '-', '.', 'gene_id \"chr16.g14476\"; transcript_id \"chr16.g14476.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '36789957', '36790479', '.', '+', '.', 'gene_id \"chr16.g14677\"; transcript_id \"chr16.g14677.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '37032262', '37032701', '.', '+', '.', 'gene_id \"chr16.g14848\"; transcript_id \"chr16.g14848.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '1563951', '1564082', '.', '+', '.', 'gene_id \"chr24.g413\"; transcript_id \"chr24.g413.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '37525585', '37525624', '.', '+', '.', 'gene_id \"chr16.g14924\"; transcript_id \"chr16.g14924.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '38990412', '38990544', '.', '-', '.', 'gene_id \"chr16.g15064\"; transcript_id \"chr16.g15064.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '39021079', '39021403', '.', '-', '.', 'gene_id \"chr16.g15084\"; transcript_id \"chr16.g15084.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '2784790', '2784829', '.', '+', '.', 'gene_id \"chr24.g851\"; transcript_id \"chr24.g851.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr24', 'PASA', 'exon', '3604917', '3605116', '.', '+', '.', 'gene_id \"chr24.g1380\"; transcript_id \"chr24.g1380.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '41135877', '41136057', '.', '-', '.', 'gene_id \"chr16.g15849\"; transcript_id \"chr16.g15849.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '41136452', '41136627', '.', '+', '.', 'gene_id \"chr16.g15850\"; transcript_id \"chr16.g15850.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '41160058', '41162461', '.', '+', '.', 'gene_id \"chr16.g15889\"; transcript_id \"chr16.g15889.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '3683664', '3683900', '.', '-', '.', 'gene_id \"chr24.g1451\"; transcript_id \"chr24.g1451.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '3683877', '3684170', '.', '-', '.', 'gene_id \"chr24.g1451\"; transcript_id \"chr24.g1451.i3\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '3947845', '3947881', '.', '-', '.', 'gene_id \"chr24.g1555\"; transcript_id \"chr24.g1555.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4274450', '4274514', '.', '+', '.', 'gene_id \"chr24.g1614\"; transcript_id \"chr24.g1614.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4282308', '4282467', '.', '+', '.', 'gene_id \"chr24.g1616\"; transcript_id \"chr24.g1616.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4650842', '4650878', '.', '-', '.', 'gene_id \"chr24.g1801\"; transcript_id \"chr24.g1801.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4733656', '4733764', '.', '-', '.', 'gene_id \"chr24.g1865\"; transcript_id \"chr24.g1865.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '44302927', '44303320', '.', '+', '.', 'gene_id \"chr16.g16449\"; transcript_id \"chr16.g16449.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4800678', '4800939', '.', '+', '.', 'gene_id \"chr24.g1920\"; transcript_id \"chr24.g1920.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4802728', '4803011', '.', '-', '.', 'gene_id \"chr24.g1925\"; transcript_id \"chr24.g1925.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4803016', '4803223', '.', '-', '.', 'gene_id \"chr24.g1926\"; transcript_id \"chr24.g1926.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '44740374', '44740574', '.', '-', '.', 'gene_id \"chr16.g16590\"; transcript_id \"chr16.g16590.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '44738896', '44739584', '.', '-', '.', 'gene_id \"chr16.g16602\"; transcript_id \"chr16.g16602.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '45079673', '45079826', '.', '+', '.', 'gene_id \"chr16.g16762\"; transcript_id \"chr16.g16762.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '45103135', '45103325', '.', '+', '.', 'gene_id \"chr16.g16793\"; transcript_id \"chr16.g16793.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '45125162', '45125576', '.', '-', '.', 'gene_id \"chr16.g16831\"; transcript_id \"chr16.g16831.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '5575546', '5575854', '.', '-', '.', 'gene_id \"chr24.g2456\"; transcript_id \"chr24.g2456.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '6099900', '6099947', '.', '+', '.', 'gene_id \"chr24.g2513\"; transcript_id \"chr24.g2513.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '6452246', '6452415', '.', '+', '.', 'gene_id \"chr24.g2622\"; transcript_id \"chr24.g2622.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8664849', '8664903', '.', '-', '.', 'gene_id \"chr24.g3105\"; transcript_id \"chr24.g3105.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8686785', '8687018', '.', '-', '.', 'gene_id \"chr24.g3115\"; transcript_id \"chr24.g3115.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8687059', '8687374', '.', '-', '.', 'gene_id \"chr24.g3116\"; transcript_id \"chr24.g3116.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '46639497', '46639673', '.', '-', '.', 'gene_id \"chr16.g17778\"; transcript_id \"chr16.g17778.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '46639659', '46639811', '.', '+', '.', 'gene_id \"chr16.g17779\"; transcript_id \"chr16.g17779.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '46640119', '46640317', '.', '+', '.', 'gene_id \"chr16.g17780\"; transcript_id \"chr16.g17780.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '46640721', '46640883', '.', '-', '.', 'gene_id \"chr16.g17782\"; transcript_id \"chr16.g17782.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8862617', '8862788', '.', '+', '.', 'gene_id \"chr24.g3246\"; transcript_id \"chr24.g3246.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8862832', '8863027', '.', '-', '.', 'gene_id \"chr24.g3247\"; transcript_id \"chr24.g3247.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9015159', '9015206', '.', '+', '.', 'gene_id \"chr24.g3322\"; transcript_id \"chr24.g3322.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '46851889', '46852396', '.', '-', '.', 'gene_id \"chr16.g17948\"; transcript_id \"chr16.g17948.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9131347', '9131389', '.', '+', '.', 'gene_id \"chr24.g3386\"; transcript_id \"chr24.g3386.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9522505', '9523211', '.', '+', '.', 'gene_id \"chr24.g3557\"; transcript_id \"chr24.g3557.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9529333', '9529581', '.', '-', '.', 'gene_id \"chr24.g3561\"; transcript_id \"chr24.g3561.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '10190025', '10190062', '.', '+', '.', 'gene_id \"chr24.g3630\"; transcript_id \"chr24.g3630.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '11564251', '11564503', '.', '+', '.', 'gene_id \"chr24.g3788\"; transcript_id \"chr24.g3788.i2\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '49563443', '49563493', '.', '+', '.', 'gene_id \"chr16.g18441\"; transcript_id \"chr16.g18441.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '50659928', '50659983', '.', '+', '.', 'gene_id \"chr16.g18488\"; transcript_id \"chr16.g18488.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '11762892', '11762924', '.', '-', '.', 'gene_id \"chr24.g3872\"; transcript_id \"chr24.g3872.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '12513046', '12514243', '.', '+', '.', 'gene_id \"chr24.g4099\"; transcript_id \"chr24.g4099.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '12513586', '12514160', '.', '+', '.', 'gene_id \"chr24.g4099\"; transcript_id \"chr24.g4099.i3\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '12609857', '12610031', '.', '-', '.', 'gene_id \"chr24.g4111\"; transcript_id \"chr24.g4111.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '53523520', '53524454', '.', '+', '.', 'gene_id \"chr16.g18860\"; transcript_id \"chr16.g18860.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '13895148', '13895332', '.', '+', '.', 'gene_id \"chr24.g4587\"; transcript_id \"chr24.g4587.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '54680709', '54680915', '.', '+', '.', 'gene_id \"chr16.g19286\"; transcript_id \"chr16.g19286.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '56131925', '56132030', '.', '+', '.', 'gene_id \"chr16.g19393\"; transcript_id \"chr16.g19393.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '14131640', '14131685', '.', '+', '.', 'gene_id \"chr24.g4711\"; transcript_id \"chr24.g4711.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '57875939', '57875996', '.', '+', '.', 'gene_id \"chr16.g19552\"; transcript_id \"chr16.g19552.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '15612583', '15612620', '.', '-', '.', 'gene_id \"chr24.g4918\"; transcript_id \"chr24.g4918.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '58573880', '58574061', '.', '+', '.', 'gene_id \"chr16.g19643\"; transcript_id \"chr16.g19643.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '58574377', '58574560', '.', '+', '.', 'gene_id \"chr16.g19644\"; transcript_id \"chr16.g19644.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '15883935', '15884117', '.', '+', '.', 'gene_id \"chr24.g5023\"; transcript_id \"chr24.g5023.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16116504', '16116545', '.', '+', '.', 'gene_id \"chr24.g5206\"; transcript_id \"chr24.g5206.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16186323', '16186368', '.', '+', '.', 'gene_id \"chr24.g5251\"; transcript_id \"chr24.g5251.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16239549', '16239878', '.', '-', '.', 'gene_id \"chr24.g5302\"; transcript_id \"chr24.g5302.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '59129445', '59129666', '.', '+', '.', 'gene_id \"chr16.g19966\"; transcript_id \"chr16.g19966.i1\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '22827235', '22827284', '100', '-', '.', 'gene_id \"chr16.g20209\"; transcript_id \"blat.proc4587.chain_9067\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '2664729', '2664764', '100', '+', '.', 'gene_id \"chr16.g20404\"; transcript_id \"blat.proc4587.chain_1628\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '16346867', '16346906', '100', '-', '.', 'gene_id \"chr16.g20449\"; transcript_id \"blat.proc28144.chain_7736\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '36582305', '36582337', '100', '-', '.', 'gene_id \"chr16.g20522\"; transcript_id \"blat.proc18128.chain_416\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '6347679', '6347921', '100', '-', '.', 'gene_id \"chr16.g21055\"; transcript_id \"blat.proc4587.chain_12732\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '38581599', '38581631', '100', '+', '.', 'gene_id \"chr16.g21346\"; transcript_id \"blat.proc4587.chain_13659\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '497209', '497273', '100', '+', '.', 'gene_id \"chr16.g21598\"; transcript_id \"blat.proc18128.chain_14442\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '13947769', '13947802', '100', '-', '.', 'gene_id \"chr16.g21708\"; transcript_id \"blat.proc4587.chain_6643\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '39903986', '39904033', '100', '-', '.', 'gene_id \"chr16.g22086\"; transcript_id \"blat.proc18128.chain_3559\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '50830468', '50830557', '100', '-', '.', 'gene_id \"chr16.g22122\"; transcript_id \"blat.proc18128.chain_19269\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '42334830', '42334864', '100', '+', '.', 'gene_id \"chr16.g22234\"; transcript_id \"blat.proc18128.chain_16777\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '18245999', '18246039', '100', '+', '.', 'gene_id \"chr16.g22650\"; transcript_id \"blat.proc28144.chain_4500\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '30594068', '30594102', '100', '-', '.', 'gene_id \"chr16.g22672\"; transcript_id \"blat.proc4587.chain_7790\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '53336567', '53336604', '100', '+', '.', 'gene_id \"chr16.g22920\"; transcript_id \"blat.proc4587.chain_1149\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '38319344', '38319376', '100', '+', '.', 'gene_id \"chr16.g23398\"; transcript_id \"blat.proc18128.chain_7324\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '15596982', '15597066', '100', '+', '.', 'gene_id \"chr16.g23456\"; transcript_id \"blat.proc18128.chain_15039\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '8101685', '8101762', '100', '+', '.', 'gene_id \"chr16.g23510\"; transcript_id \"blat.proc4587.chain_12582\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '3768425', '3768570', '100', '+', '.', 'gene_id \"chr16.g23805\"; transcript_id \"blat.proc18128.chain_1529\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '23470869', '23471041', '100', '+', '.', 'gene_id \"chr16.g24044\"; transcript_id \"blat.proc18128.chain_25545\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '11919526', '11919557', '100', '+', '.', 'gene_id \"chr16.g24221\"; transcript_id \"blat.proc4587.chain_6140\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '41317750', '41317893', '100', '-', '.', 'gene_id \"chr16.g24250\"; transcript_id \"blat.proc4587.chain_12094\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '30993702', '30993739', '100', '+', '.', 'gene_id \"chr16.g24439\"; transcript_id \"blat.proc4587.chain_2109\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '26060287', '26060415', '100', '+', '.', 'gene_id \"chr16.g24787\"; transcript_id \"blat.proc4587.chain_1750\";']\n",
      "negative:  ['chr16', 'PASA', 'exon', '10767777', '10767814', '100', '-', '.', 'gene_id \"chr16.g25154\"; transcript_id \"blat.proc4005.chain_21373\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16593905', '16594192', '.', '-', '.', 'gene_id \"chr24.g5518\"; transcript_id \"chr24.g5518.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16593705', '16593915', '.', '+', '.', 'gene_id \"chr24.g5519\"; transcript_id \"chr24.g5519.i1\";']\n",
      "negative:  ['scaffold_1672', 'PASA', 'exon', '49886', '49915', '.', '-', '.', 'gene_id \"scaffold_1672.g5\"; transcript_id \"scaffold_1672.g5.i1\";']\n",
      "negative:  ['scaffold_1588', 'PASA', 'exon', '26753', '26905', '100', '+', '.', 'gene_id \"scaffold_1588.g8\"; transcript_id \"blat.proc4587.chain_7342\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16842785', '16842964', '.', '+', '.', 'gene_id \"chr24.g5701\"; transcript_id \"chr24.g5701.i1\";']\n",
      "negative:  ['scaffold_1259', 'PASA', 'exon', '7534', '7793', '100', '+', '.', 'gene_id \"scaffold_1259.g19\"; transcript_id \"blat.proc18128.chain_21878\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '17480564', '17480900', '.', '+', '.', 'gene_id \"chr24.g5973\"; transcript_id \"chr24.g5973.i1\";']\n",
      "negative:  ['scaffold_620', 'PASA', 'exon', '3717', '3762', '.', '+', '.', 'gene_id \"scaffold_620.g1\"; transcript_id \"scaffold_620.g1.i1\";']\n",
      "negative:  ['scaffold_1653', 'PASA', 'exon', '36748', '36964', '100', '+', '.', 'gene_id \"scaffold_1653.g10\"; transcript_id \"blat.proc4005.chain_5648\";']\n",
      "negative:  ['scaffold_1669', 'PASA', 'exon', '57484', '57557', '.', '-', '.', 'gene_id \"scaffold_1669.g27\"; transcript_id \"scaffold_1669.g27.i1\";']\n",
      "negative:  ['scaffold_1376', 'PASA', 'exon', '18988', '19288', '.', '-', '.', 'gene_id \"scaffold_1376.g14\"; transcript_id \"scaffold_1376.g14.i1\";']\n",
      "negative:  ['scaffold_1376', 'PASA', 'exon', '19973', '20125', '.', '-', '.', 'gene_id \"scaffold_1376.g15\"; transcript_id \"scaffold_1376.g15.i1\";']\n",
      "negative:  ['scaffold_1376', 'PASA', 'exon', '21266', '21345', '.', '+', '.', 'gene_id \"scaffold_1376.g16\"; transcript_id \"scaffold_1376.g16.i1\";']\n",
      "negative:  ['scaffold_1376', 'PASA', 'exon', '15288', '15580', '100', '+', '.', 'gene_id \"scaffold_1376.g17\"; transcript_id \"blat.proc4587.chain_9280\";']\n",
      "negative:  ['scaffold_1376', 'PASA', 'exon', '15800', '15997', '100', '+', '.', 'gene_id \"scaffold_1376.g22\"; transcript_id \"blat.proc4587.chain_9278\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '1739', '2073', '.', '+', '.', 'gene_id \"scaffold_1659.g2\"; transcript_id \"scaffold_1659.g2.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '6801', '7366', '.', '-', '.', 'gene_id \"scaffold_1659.g5\"; transcript_id \"scaffold_1659.g5.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '16112', '16315', '.', '-', '.', 'gene_id \"scaffold_1659.g20\"; transcript_id \"scaffold_1659.g20.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '22052', '22242', '.', '+', '.', 'gene_id \"scaffold_1659.g27\"; transcript_id \"scaffold_1659.g27.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '28841', '29002', '.', '-', '.', 'gene_id \"scaffold_1659.g32\"; transcript_id \"scaffold_1659.g32.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '29321', '29547', '.', '-', '.', 'gene_id \"scaffold_1659.g33\"; transcript_id \"scaffold_1659.g33.i1\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '55779', '55872', '.', '-', '.', 'gene_id \"scaffold_1659.g67\"; transcript_id \"scaffold_1659.g67.i2\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '19957', '20259', '100', '+', '.', 'gene_id \"scaffold_1659.g80\"; transcript_id \"blat.proc18128.chain_12812\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '2921', '3103', '100', '-', '.', 'gene_id \"scaffold_1659.g93\"; transcript_id \"blat.proc4005.chain_2842\";']\n",
      "negative:  ['scaffold_1659', 'PASA', 'exon', '1458', '1612', '100', '-', '.', 'gene_id \"scaffold_1659.g95\"; transcript_id \"blat.proc4005.chain_1525\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '17948797', '17949010', '.', '+', '.', 'gene_id \"chr24.g6392\"; transcript_id \"chr24.g6392.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '17993339', '17993682', '.', '+', '.', 'gene_id \"chr24.g6448\"; transcript_id \"chr24.g6448.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18121502', '18123008', '.', '-', '.', 'gene_id \"chr24.g6578\"; transcript_id \"chr24.g6578.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18123112', '18123429', '.', '+', '.', 'gene_id \"chr24.g6579\"; transcript_id \"chr24.g6579.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18247919', '18248277', '.', '+', '.', 'gene_id \"chr24.g6684\"; transcript_id \"chr24.g6684.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18247774', '18247946', '.', '+', '.', 'gene_id \"chr24.g6684\"; transcript_id \"chr24.g6684.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '394979', '395021', '.', '-', '.', 'gene_id \"chr33.g68\"; transcript_id \"chr33.g68.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '433641', '433797', '.', '-', '.', 'gene_id \"chr33.g80\"; transcript_id \"chr33.g80.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '463664', '464300', '.', '-', '.', 'gene_id \"chr33.g88\"; transcript_id \"chr33.g88.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18385820', '18385975', '.', '+', '.', 'gene_id \"chr24.g6790\"; transcript_id \"chr24.g6790.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18440953', '18442357', '.', '+', '.', 'gene_id \"chr24.g6838\"; transcript_id \"chr24.g6838.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18447335', '18447376', '.', '+', '.', 'gene_id \"chr24.g6842\"; transcript_id \"chr24.g6842.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18491207', '18491415', '.', '+', '.', 'gene_id \"chr24.g6878\"; transcript_id \"chr24.g6878.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18634518', '18634550', '.', '-', '.', 'gene_id \"chr24.g6907\"; transcript_id \"chr24.g6907.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '1638931', '1639081', '.', '-', '.', 'gene_id \"chr33.g292\"; transcript_id \"chr33.g292.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '1639062', '1639328', '.', '+', '.', 'gene_id \"chr33.g293\"; transcript_id \"chr33.g293.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '1689789', '1690187', '.', '+', '.', 'gene_id \"chr33.g340\"; transcript_id \"chr33.g340.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19090125', '19090158', '.', '-', '.', 'gene_id \"chr24.g7120\"; transcript_id \"chr24.g7120.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '1827502', '1827815', '.', '-', '.', 'gene_id \"chr33.g464\"; transcript_id \"chr33.g464.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19158728', '19158976', '.', '+', '.', 'gene_id \"chr24.g7192\"; transcript_id \"chr24.g7192.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19316973', '19317194', '.', '-', '.', 'gene_id \"chr24.g7251\"; transcript_id \"chr24.g7251.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '2377846', '2377880', '.', '-', '.', 'gene_id \"chr33.g567\"; transcript_id \"chr33.g567.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '3197244', '3197448', '.', '-', '.', 'gene_id \"chr33.g607\"; transcript_id \"chr33.g607.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '3443348', '3443396', '.', '-', '.', 'gene_id \"chr33.g621\"; transcript_id \"chr33.g621.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4261694', '4261845', '.', '+', '.', 'gene_id \"chr33.g701\"; transcript_id \"chr33.g701.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4410181', '4410294', '.', '+', '.', 'gene_id \"chr33.g720\"; transcript_id \"chr33.g720.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19663458', '19663691', '.', '+', '.', 'gene_id \"chr24.g7442\"; transcript_id \"chr24.g7442.i4\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19663764', '19664379', '.', '+', '.', 'gene_id \"chr24.g7442\"; transcript_id \"chr24.g7442.i6\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19646154', '19646312', '.', '+', '.', 'gene_id \"chr24.g7443\"; transcript_id \"chr24.g7443.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19651597', '19652499', '.', '+', '.', 'gene_id \"chr24.g7445\"; transcript_id \"chr24.g7445.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19653714', '19653885', '.', '-', '.', 'gene_id \"chr24.g7446\"; transcript_id \"chr24.g7446.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19661474', '19661662', '.', '+', '.', 'gene_id \"chr24.g7449\"; transcript_id \"chr24.g7449.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4495903', '4495967', '.', '-', '.', 'gene_id \"chr33.g733\"; transcript_id \"chr33.g733.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19674159', '19674312', '.', '+', '.', 'gene_id \"chr24.g7459\"; transcript_id \"chr24.g7459.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4900777', '4900987', '.', '+', '.', 'gene_id \"chr33.g915\"; transcript_id \"chr33.g915.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4901026', '4901345', '.', '+', '.', 'gene_id \"chr33.g916\"; transcript_id \"chr33.g916.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '20247313', '20247545', '.', '-', '.', 'gene_id \"chr24.g7761\"; transcript_id \"chr24.g7761.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '5494135', '5494167', '.', '-', '.', 'gene_id \"chr33.g1096\"; transcript_id \"chr33.g1096.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '20319138', '20319282', '.', '-', '.', 'gene_id \"chr24.g7811\"; transcript_id \"chr24.g7811.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '20321964', '20322058', '.', '-', '.', 'gene_id \"chr24.g7813\"; transcript_id \"chr24.g7813.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '20322128', '20322369', '.', '+', '.', 'gene_id \"chr24.g7815\"; transcript_id \"chr24.g7815.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '5722595', '5722629', '.', '+', '.', 'gene_id \"chr33.g1242\"; transcript_id \"chr33.g1242.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6108053', '6108319', '.', '-', '.', 'gene_id \"chr33.g1293\"; transcript_id \"chr33.g1293.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6344071', '6344339', '.', '-', '.', 'gene_id \"chr33.g1383\"; transcript_id \"chr33.g1383.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6409475', '6409646', '.', '+', '.', 'gene_id \"chr33.g1414\"; transcript_id \"chr33.g1414.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6409695', '6409941', '.', '+', '.', 'gene_id \"chr33.g1415\"; transcript_id \"chr33.g1415.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6410130', '6410285', '.', '+', '.', 'gene_id \"chr33.g1416\"; transcript_id \"chr33.g1416.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6438271', '6438429', '.', '-', '.', 'gene_id \"chr33.g1422\"; transcript_id \"chr33.g1422.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6438271', '6438374', '.', '-', '.', 'gene_id \"chr33.g1422\"; transcript_id \"chr33.g1422.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21028761', '21028801', '.', '+', '.', 'gene_id \"chr24.g8140\"; transcript_id \"chr24.g8140.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21067479', '21067531', '.', '+', '.', 'gene_id \"chr24.g8160\"; transcript_id \"chr24.g8160.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6480772', '6480817', '.', '+', '.', 'gene_id \"chr33.g1463\"; transcript_id \"chr33.g1463.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21319217', '21319615', '.', '-', '.', 'gene_id \"chr24.g8334\"; transcript_id \"chr24.g8334.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6657577', '6658033', '.', '-', '.', 'gene_id \"chr33.g1641\"; transcript_id \"chr33.g1641.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21368845', '21368887', '.', '+', '.', 'gene_id \"chr24.g8377\"; transcript_id \"chr24.g8377.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21441131', '21441179', '.', '+', '.', 'gene_id \"chr24.g8413\"; transcript_id \"chr24.g8413.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6713826', '6714264', '.', '-', '.', 'gene_id \"chr33.g1689\"; transcript_id \"chr33.g1689.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6905549', '6905771', '.', '+', '.', 'gene_id \"chr33.g1802\"; transcript_id \"chr33.g1802.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6905939', '6906240', '.', '-', '.', 'gene_id \"chr33.g1803\"; transcript_id \"chr33.g1803.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '6933304', '6933341', '.', '-', '.', 'gene_id \"chr33.g1827\"; transcript_id \"chr33.g1827.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr24', 'PASA', 'exon', '21826946', '21826977', '.', '-', '.', 'gene_id \"chr24.g8767\"; transcript_id \"chr24.g8767.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7288467', '7288742', '.', '-', '.', 'gene_id \"chr33.g2071\"; transcript_id \"chr33.g2071.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7288812', '7289005', '.', '-', '.', 'gene_id \"chr33.g2072\"; transcript_id \"chr33.g2072.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21989188', '21989367', '.', '-', '.', 'gene_id \"chr24.g8835\"; transcript_id \"chr24.g8835.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21989834', '21990404', '.', '-', '.', 'gene_id \"chr24.g8838\"; transcript_id \"chr24.g8838.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21990930', '21991099', '.', '-', '.', 'gene_id \"chr24.g8839\"; transcript_id \"chr24.g8839.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21990658', '21990931', '.', '-', '.', 'gene_id \"chr24.g8840\"; transcript_id \"chr24.g8840.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '22057455', '22057754', '.', '+', '.', 'gene_id \"chr24.g8915\"; transcript_id \"chr24.g8915.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7780757', '7780791', '.', '+', '.', 'gene_id \"chr33.g2204\"; transcript_id \"chr33.g2204.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7967874', '7967984', '.', '-', '.', 'gene_id \"chr33.g2344\"; transcript_id \"chr33.g2344.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '22438130', '22438176', '.', '-', '.', 'gene_id \"chr24.g9100\"; transcript_id \"chr24.g9100.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '22598987', '22599016', '.', '+', '.', 'gene_id \"chr24.g9222\"; transcript_id \"chr24.g9222.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '8312943', '8313004', '.', '-', '.', 'gene_id \"chr33.g2587\"; transcript_id \"chr33.g2587.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '22796245', '22796435', '.', '-', '.', 'gene_id \"chr24.g9344\"; transcript_id \"chr24.g9344.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '8708951', '8709000', '.', '-', '.', 'gene_id \"chr33.g2628\"; transcript_id \"chr33.g2628.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '9787674', '9787787', '.', '-', '.', 'gene_id \"chr33.g2686\"; transcript_id \"chr33.g2686.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '10229897', '10229935', '.', '-', '.', 'gene_id \"chr33.g2708\"; transcript_id \"chr33.g2708.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '10991802', '10991834', '.', '+', '.', 'gene_id \"chr33.g2746\"; transcript_id \"chr33.g2746.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23001239', '23001480', '.', '+', '.', 'gene_id \"chr24.g9566\"; transcript_id \"chr24.g9566.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '11094334', '11094518', '.', '-', '.', 'gene_id \"chr33.g2823\"; transcript_id \"chr33.g2823.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23034011', '23034056', '.', '+', '.', 'gene_id \"chr24.g9592\"; transcript_id \"chr24.g9592.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '11170775', '11170919', '.', '-', '.', 'gene_id \"chr33.g2850\"; transcript_id \"chr33.g2850.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23152315', '23152687', '.', '+', '.', 'gene_id \"chr24.g9640\"; transcript_id \"chr24.g9640.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23264668', '23265502', '.', '+', '.', 'gene_id \"chr24.g9754\"; transcript_id \"chr24.g9754.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '11402751', '11403175', '.', '+', '.', 'gene_id \"chr33.g2973\"; transcript_id \"chr33.g2973.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '12015254', '12015284', '.', '-', '.', 'gene_id \"chr33.g3080\"; transcript_id \"chr33.g3080.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23458229', '23458476', '.', '+', '.', 'gene_id \"chr24.g9894\"; transcript_id \"chr24.g9894.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '12348445', '12348634', '.', '-', '.', 'gene_id \"chr33.g3126\"; transcript_id \"chr33.g3126.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '12626227', '12626258', '.', '+', '.', 'gene_id \"chr33.g3158\"; transcript_id \"chr33.g3158.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23767953', '23768280', '.', '-', '.', 'gene_id \"chr24.g10213\"; transcript_id \"chr24.g10213.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13852782', '13852882', '.', '-', '.', 'gene_id \"chr33.g3607\"; transcript_id \"chr33.g3607.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24033001', '24033184', '.', '-', '.', 'gene_id \"chr24.g10488\"; transcript_id \"chr24.g10488.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24036642', '24036913', '.', '+', '.', 'gene_id \"chr24.g10491\"; transcript_id \"chr24.g10491.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13879369', '13880318', '.', '-', '.', 'gene_id \"chr33.g3625\"; transcript_id \"chr33.g3625.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13880316', '13880540', '.', '+', '.', 'gene_id \"chr33.g3626\"; transcript_id \"chr33.g3626.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13884526', '13884771', '.', '-', '.', 'gene_id \"chr33.g3632\"; transcript_id \"chr33.g3632.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13884866', '13885129', '.', '+', '.', 'gene_id \"chr33.g3633\"; transcript_id \"chr33.g3633.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13940283', '13940353', '.', '+', '.', 'gene_id \"chr33.g3680\"; transcript_id \"chr33.g3680.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13940513', '13940790', '.', '-', '.', 'gene_id \"chr33.g3682\"; transcript_id \"chr33.g3682.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '13940513', '13940790', '.', '-', '.', 'gene_id \"chr33.g3682\"; transcript_id \"chr33.g3682.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24147113', '24147153', '.', '-', '.', 'gene_id \"chr24.g10603\"; transcript_id \"chr24.g10603.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24243591', '24243795', '.', '+', '.', 'gene_id \"chr24.g10676\"; transcript_id \"chr24.g10676.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '14825569', '14825820', '.', '+', '.', 'gene_id \"chr33.g3860\"; transcript_id \"chr33.g3860.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '15763345', '15763388', '.', '-', '.', 'gene_id \"chr33.g3932\"; transcript_id \"chr33.g3932.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16390250', '16390345', '.', '-', '.', 'gene_id \"chr33.g4067\"; transcript_id \"chr33.g4067.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16391572', '16391603', '.', '+', '.', 'gene_id \"chr33.g4068\"; transcript_id \"chr33.g4068.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24790687', '24790961', '.', '+', '.', 'gene_id \"chr24.g11015\"; transcript_id \"chr24.g11015.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '25077426', '25077467', '.', '-', '.', 'gene_id \"chr24.g11173\"; transcript_id \"chr24.g11173.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16687473', '16687661', '.', '+', '.', 'gene_id \"chr33.g4294\"; transcript_id \"chr33.g4294.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16789010', '16789194', '.', '+', '.', 'gene_id \"chr33.g4334\"; transcript_id \"chr33.g4334.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16855717', '16856200', '.', '-', '.', 'gene_id \"chr33.g4350\"; transcript_id \"chr33.g4350.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16856430', '16856703', '.', '+', '.', 'gene_id \"chr33.g4351\"; transcript_id \"chr33.g4351.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '16949934', '16950088', '.', '+', '.', 'gene_id \"chr33.g4390\"; transcript_id \"chr33.g4390.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17092208', '17092847', '.', '+', '.', 'gene_id \"chr33.g4425\"; transcript_id \"chr33.g4425.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17124381', '17124440', '.', '-', '.', 'gene_id \"chr33.g4478\"; transcript_id \"chr33.g4478.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17186283', '17186318', '.', '-', '.', 'gene_id \"chr33.g4521\"; transcript_id \"chr33.g4521.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17412120', '17412169', '.', '+', '.', 'gene_id \"chr33.g4580\"; transcript_id \"chr33.g4580.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17462348', '17462645', '.', '+', '.', 'gene_id \"chr33.g4637\"; transcript_id \"chr33.g4637.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '25514533', '25514720', '.', '-', '.', 'gene_id \"chr24.g11570\"; transcript_id \"chr24.g11570.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17500188', '17500340', '.', '+', '.', 'gene_id \"chr33.g4675\"; transcript_id \"chr33.g4675.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17681295', '17681455', '.', '-', '.', 'gene_id \"chr33.g4742\"; transcript_id \"chr33.g4742.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17698399', '17698563', '.', '+', '.', 'gene_id \"chr33.g4749\"; transcript_id \"chr33.g4749.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17712988', '17713020', '.', '+', '.', 'gene_id \"chr33.g4750\"; transcript_id \"chr33.g4750.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17883837', '17884201', '.', '-', '.', 'gene_id \"chr33.g4846\"; transcript_id \"chr33.g4846.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17914942', '17915489', '.', '-', '.', 'gene_id \"chr33.g4863\"; transcript_id \"chr33.g4863.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17941262', '17941316', '.', '-', '.', 'gene_id \"chr33.g4872\"; transcript_id \"chr33.g4872.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17973255', '17973648', '.', '-', '.', 'gene_id \"chr33.g4888\"; transcript_id \"chr33.g4888.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17977261', '17978839', '.', '-', '.', 'gene_id \"chr33.g4889\"; transcript_id \"chr33.g4889.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '17973653', '17974237', '.', '-', '.', 'gene_id \"chr33.g4892\"; transcript_id \"chr33.g4892.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18128591', '18128742', '.', '+', '.', 'gene_id \"chr33.g5028\"; transcript_id \"chr33.g5028.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18281276', '18281568', '.', '+', '.', 'gene_id \"chr33.g5143\"; transcript_id \"chr33.g5143.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18309363', '18309403', '.', '-', '.', 'gene_id \"chr33.g5163\"; transcript_id \"chr33.g5163.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '26459152', '26459305', '.', '-', '.', 'gene_id \"chr24.g12161\"; transcript_id \"chr24.g12161.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18587498', '18587697', '.', '-', '.', 'gene_id \"chr33.g5268\"; transcript_id \"chr33.g5268.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18592168', '18593143', '.', '-', '.', 'gene_id \"chr33.g5275\"; transcript_id \"chr33.g5275.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18593289', '18593479', '.', '-', '.', 'gene_id \"chr33.g5277\"; transcript_id \"chr33.g5277.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '26548545', '26548718', '.', '-', '.', 'gene_id \"chr24.g12214\"; transcript_id \"chr24.g12214.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18617326', '18618223', '.', '-', '.', 'gene_id \"chr33.g5319\"; transcript_id \"chr33.g5319.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18618889', '18619709', '.', '+', '.', 'gene_id \"chr33.g5321\"; transcript_id \"chr33.g5321.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18645984', '18646230', '.', '+', '.', 'gene_id \"chr33.g5354\"; transcript_id \"chr33.g5354.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18646264', '18646433', '.', '-', '.', 'gene_id \"chr33.g5355\"; transcript_id \"chr33.g5355.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18704281', '18704492', '.', '-', '.', 'gene_id \"chr33.g5407\"; transcript_id \"chr33.g5407.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '26936449', '26936496', '.', '-', '.', 'gene_id \"chr24.g12450\"; transcript_id \"chr24.g12450.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '26951912', '26952102', '.', '-', '.', 'gene_id \"chr24.g12463\"; transcript_id \"chr24.g12463.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '26991263', '26992904', '.', '+', '.', 'gene_id \"chr24.g12499\"; transcript_id \"chr24.g12499.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18869307', '18869594', '.', '-', '.', 'gene_id \"chr33.g5593\"; transcript_id \"chr33.g5593.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '27156603', '27156712', '.', '-', '.', 'gene_id \"chr24.g12536\"; transcript_id \"chr24.g12536.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '27140276', '27140450', '.', '-', '.', 'gene_id \"chr24.g12548\"; transcript_id \"chr24.g12548.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18881081', '18881459', '.', '-', '.', 'gene_id \"chr33.g5613\"; transcript_id \"chr33.g5613.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18929206', '18929890', '.', '-', '.', 'gene_id \"chr33.g5683\"; transcript_id \"chr33.g5683.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '18934678', '18934906', '.', '-', '.', 'gene_id \"chr33.g5688\"; transcript_id \"chr33.g5688.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '19007948', '19008242', '.', '+', '.', 'gene_id \"chr33.g5761\"; transcript_id \"chr33.g5761.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '28165417', '28165454', '.', '-', '.', 'gene_id \"chr24.g12769\"; transcript_id \"chr24.g12769.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '28175895', '28175957', '.', '+', '.', 'gene_id \"chr24.g12770\"; transcript_id \"chr24.g12770.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '19097863', '19098105', '.', '-', '.', 'gene_id \"chr33.g5852\"; transcript_id \"chr33.g5852.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '28443358', '28443395', '.', '+', '.', 'gene_id \"chr24.g12785\"; transcript_id \"chr24.g12785.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '28940909', '28941162', '.', '+', '.', 'gene_id \"chr24.g12851\"; transcript_id \"chr24.g12851.i3\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29056621', '29057922', '.', '-', '.', 'gene_id \"chr24.g12899\"; transcript_id \"chr24.g12899.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '19249581', '19249612', '.', '+', '.', 'gene_id \"chr33.g5965\"; transcript_id \"chr33.g5965.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '19729286', '19729328', '.', '+', '.', 'gene_id \"chr33.g6076\"; transcript_id \"chr33.g6076.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '20009478', '20009740', '.', '+', '.', 'gene_id \"chr33.g6161\"; transcript_id \"chr33.g6161.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '20528110', '20528322', '.', '-', '.', 'gene_id \"chr33.g6261\"; transcript_id \"chr33.g6261.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29358628', '29358686', '.', '-', '.', 'gene_id \"chr24.g13237\"; transcript_id \"chr24.g13237.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '20729607', '20729731', '.', '-', '.', 'gene_id \"chr33.g6289\"; transcript_id \"chr33.g6289.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '20729649', '20729731', '.', '+', '.', 'gene_id \"chr33.g6290\"; transcript_id \"chr33.g6290.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29546762', '29546792', '.', '+', '.', 'gene_id \"chr24.g13373\"; transcript_id \"chr24.g13373.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22319486', '22319550', '.', '+', '.', 'gene_id \"chr33.g6450\"; transcript_id \"chr33.g6450.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29735311', '29735616', '.', '+', '.', 'gene_id \"chr24.g13428\"; transcript_id \"chr24.g13428.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22340605', '22340784', '.', '-', '.', 'gene_id \"chr33.g6452\"; transcript_id \"chr33.g6452.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29739250', '29740130', '.', '+', '.', 'gene_id \"chr24.g13432\"; transcript_id \"chr24.g13432.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '29889282', '29889434', '.', '+', '.', 'gene_id \"chr24.g13446\"; transcript_id \"chr24.g13446.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22796547', '22796579', '.', '-', '.', 'gene_id \"chr33.g6565\"; transcript_id \"chr33.g6565.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22833510', '22833563', '.', '+', '.', 'gene_id \"chr33.g6583\"; transcript_id \"chr33.g6583.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31307801', '31308566', '.', '-', '.', 'gene_id \"chr24.g13674\"; transcript_id \"chr24.g13674.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31332397', '31332560', '.', '+', '.', 'gene_id \"chr24.g13687\"; transcript_id \"chr24.g13687.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31512492', '31512525', '.', '+', '.', 'gene_id \"chr24.g13742\"; transcript_id \"chr24.g13742.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31529874', '31530178', '.', '-', '.', 'gene_id \"chr24.g13749\"; transcript_id \"chr24.g13749.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31605370', '31605409', '.', '+', '.', 'gene_id \"chr24.g13768\"; transcript_id \"chr24.g13768.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23200850', '23201581', '.', '-', '.', 'gene_id \"chr33.g6863\"; transcript_id \"chr33.g6863.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23438414', '23438451', '.', '+', '.', 'gene_id \"chr33.g7069\"; transcript_id \"chr33.g7069.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '32074469', '32074768', '.', '-', '.', 'gene_id \"chr24.g14155\"; transcript_id \"chr24.g14155.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23483699', '23484009', '.', '+', '.', 'gene_id \"chr33.g7129\"; transcript_id \"chr33.g7129.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23484418', '23484599', '.', '+', '.', 'gene_id \"chr33.g7133\"; transcript_id \"chr33.g7133.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '32278063', '32278254', '.', '-', '.', 'gene_id \"chr24.g14225\"; transcript_id \"chr24.g14225.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23549687', '23549965', '.', '+', '.', 'gene_id \"chr33.g7209\"; transcript_id \"chr33.g7209.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '32333933', '32333989', '.', '-', '.', 'gene_id \"chr24.g14284\"; transcript_id \"chr24.g14284.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23774583', '23774756', '.', '-', '.', 'gene_id \"chr33.g7383\"; transcript_id \"chr33.g7383.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23774959', '23775254', '.', '-', '.', 'gene_id \"chr33.g7385\"; transcript_id \"chr33.g7385.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23871512', '23871829', '.', '-', '.', 'gene_id \"chr33.g7425\"; transcript_id \"chr33.g7425.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23957514', '23957882', '.', '+', '.', 'gene_id \"chr33.g7520\"; transcript_id \"chr33.g7520.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33139107', '33139564', '.', '+', '.', 'gene_id \"chr24.g14711\"; transcript_id \"chr24.g14711.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33352318', '33352545', '.', '-', '.', 'gene_id \"chr24.g14838\"; transcript_id \"chr24.g14838.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '24724758', '24725167', '.', '+', '.', 'gene_id \"chr33.g7846\"; transcript_id \"chr33.g7846.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33632099', '33632139', '.', '+', '.', 'gene_id \"chr24.g14965\"; transcript_id \"chr24.g14965.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '24750691', '24751231', '.', '+', '.', 'gene_id \"chr33.g7868\"; transcript_id \"chr33.g7868.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '24756117', '24756294', '.', '+', '.', 'gene_id \"chr33.g7875\"; transcript_id \"chr33.g7875.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '24756692', '24756906', '.', '+', '.', 'gene_id \"chr33.g7877\"; transcript_id \"chr33.g7877.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33798870', '33799075', '.', '+', '.', 'gene_id \"chr24.g15022\"; transcript_id \"chr24.g15022.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33845826', '33845990', '.', '-', '.', 'gene_id \"chr24.g15098\"; transcript_id \"chr24.g15098.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25142311', '25142463', '.', '-', '.', 'gene_id \"chr33.g8072\"; transcript_id \"chr33.g8072.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25380365', '25380400', '.', '-', '.', 'gene_id \"chr33.g8158\"; transcript_id \"chr33.g8158.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '34545188', '34545373', '.', '-', '.', 'gene_id \"chr24.g15372\"; transcript_id \"chr24.g15372.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25557186', '25557399', '.', '+', '.', 'gene_id \"chr33.g8272\"; transcript_id \"chr33.g8272.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25704523', '25704685', '.', '+', '.', 'gene_id \"chr33.g8439\"; transcript_id \"chr33.g8439.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25861213', '25861264', '.', '-', '.', 'gene_id \"chr33.g8579\"; transcript_id \"chr33.g8579.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25876089', '25876549', '.', '+', '.', 'gene_id \"chr33.g8593\"; transcript_id \"chr33.g8593.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '35632195', '35632319', '.', '-', '.', 'gene_id \"chr24.g15820\"; transcript_id \"chr24.g15820.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '35810999', '35811288', '.', '-', '.', 'gene_id \"chr24.g15974\"; transcript_id \"chr24.g15974.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26375584', '26375775', '.', '-', '.', 'gene_id \"chr33.g8906\"; transcript_id \"chr33.g8906.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '35859280', '35859628', '.', '-', '.', 'gene_id \"chr24.g16034\"; transcript_id \"chr24.g16034.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '35986606', '35986655', '.', '+', '.', 'gene_id \"chr24.g16084\"; transcript_id \"chr24.g16084.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '36001086', '36001123', '.', '+', '.', 'gene_id \"chr24.g16089\"; transcript_id \"chr24.g16089.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26471167', '26471336', '.', '+', '.', 'gene_id \"chr33.g8970\"; transcript_id \"chr33.g8970.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26562409', '26562507', '.', '+', '.', 'gene_id \"chr33.g9032\"; transcript_id \"chr33.g9032.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26656432', '26656466', '.', '-', '.', 'gene_id \"chr33.g9100\"; transcript_id \"chr33.g9100.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr33', 'PASA', 'exon', '26757844', '26758135', '.', '+', '.', 'gene_id \"chr33.g9221\"; transcript_id \"chr33.g9221.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26762574', '26762958', '.', '-', '.', 'gene_id \"chr33.g9230\"; transcript_id \"chr33.g9230.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26762955', '26763470', '.', '+', '.', 'gene_id \"chr33.g9231\"; transcript_id \"chr33.g9231.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27052172', '27052213', '.', '+', '.', 'gene_id \"chr33.g9378\"; transcript_id \"chr33.g9378.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '36712295', '36712336', '.', '+', '.', 'gene_id \"chr24.g16544\"; transcript_id \"chr24.g16544.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27460307', '27460795', '.', '+', '.', 'gene_id \"chr33.g9515\"; transcript_id \"chr33.g9515.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27461050', '27461314', '.', '+', '.', 'gene_id \"chr33.g9516\"; transcript_id \"chr33.g9516.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27460839', '27461065', '.', '-', '.', 'gene_id \"chr33.g9517\"; transcript_id \"chr33.g9517.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27461464', '27461610', '.', '+', '.', 'gene_id \"chr33.g9519\"; transcript_id \"chr33.g9519.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27461577', '27461610', '.', '+', '.', 'gene_id \"chr33.g9519\"; transcript_id \"chr33.g9519.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27564099', '27564275', '.', '+', '.', 'gene_id \"chr33.g9594\"; transcript_id \"chr33.g9594.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27583162', '27583397', '.', '+', '.', 'gene_id \"chr33.g9612\"; transcript_id \"chr33.g9612.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27611965', '27612006', '.', '-', '.', 'gene_id \"chr33.g9630\"; transcript_id \"chr33.g9630.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37177216', '37177368', '.', '+', '.', 'gene_id \"chr24.g16919\"; transcript_id \"chr24.g16919.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37395367', '37395412', '.', '-', '.', 'gene_id \"chr24.g17009\"; transcript_id \"chr24.g17009.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37425755', '37426184', '.', '+', '.', 'gene_id \"chr24.g17011\"; transcript_id \"chr24.g17011.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37798919', '37799158', '.', '+', '.', 'gene_id \"chr24.g17189\"; transcript_id \"chr24.g17189.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28046552', '28046591', '.', '-', '.', 'gene_id \"chr33.g10071\"; transcript_id \"chr33.g10071.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37852623', '37852663', '.', '+', '.', 'gene_id \"chr24.g17221\"; transcript_id \"chr24.g17221.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '38087950', '38087985', '.', '-', '.', 'gene_id \"chr24.g17307\"; transcript_id \"chr24.g17307.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28272066', '28272315', '.', '-', '.', 'gene_id \"chr33.g10165\"; transcript_id \"chr33.g10165.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28454795', '28454955', '.', '+', '.', 'gene_id \"chr33.g10286\"; transcript_id \"chr33.g10286.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28445944', '28446263', '.', '+', '.', 'gene_id \"chr33.g10299\"; transcript_id \"chr33.g10299.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28499434', '28499614', '.', '-', '.', 'gene_id \"chr33.g10350\"; transcript_id \"chr33.g10350.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28537814', '28537975', '.', '+', '.', 'gene_id \"chr33.g10397\"; transcript_id \"chr33.g10397.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28607073', '28607348', '.', '-', '.', 'gene_id \"chr33.g10437\"; transcript_id \"chr33.g10437.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28608442', '28608616', '.', '+', '.', 'gene_id \"chr33.g10439\"; transcript_id \"chr33.g10439.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '38918401', '38918436', '.', '+', '.', 'gene_id \"chr24.g17577\"; transcript_id \"chr24.g17577.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28842875', '28843057', '.', '+', '.', 'gene_id \"chr33.g10622\"; transcript_id \"chr33.g10622.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28843887', '28844106', '.', '-', '.', 'gene_id \"chr33.g10624\"; transcript_id \"chr33.g10624.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '39160505', '39161138', '.', '+', '.', 'gene_id \"chr24.g17743\"; transcript_id \"chr24.g17743.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '39639215', '39639246', '.', '+', '.', 'gene_id \"chr24.g17882\"; transcript_id \"chr24.g17882.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '39639215', '39639244', '.', '+', '.', 'gene_id \"chr24.g17882\"; transcript_id \"chr24.g17882.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29202435', '29202476', '.', '+', '.', 'gene_id \"chr33.g10839\"; transcript_id \"chr33.g10839.i2\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29322176', '29322471', '.', '+', '.', 'gene_id \"chr33.g10913\"; transcript_id \"chr33.g10913.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '39909397', '39909444', '.', '-', '.', 'gene_id \"chr24.g17981\"; transcript_id \"chr24.g17981.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29402132', '29402290', '.', '+', '.', 'gene_id \"chr33.g10986\"; transcript_id \"chr33.g10986.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29416627', '29417033', '.', '+', '.', 'gene_id \"chr33.g11005\"; transcript_id \"chr33.g11005.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29424089', '29424159', '.', '-', '.', 'gene_id \"chr33.g11013\"; transcript_id \"chr33.g11013.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '41371101', '41371314', '.', '+', '.', 'gene_id \"chr24.g18175\"; transcript_id \"chr24.g18175.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '41389810', '41389841', '.', '-', '.', 'gene_id \"chr24.g18180\"; transcript_id \"chr24.g18180.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '41541587', '41541776', '.', '-', '.', 'gene_id \"chr24.g18225\"; transcript_id \"chr24.g18225.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29634432', '29634463', '.', '-', '.', 'gene_id \"chr33.g11215\"; transcript_id \"chr33.g11215.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29655520', '29655794', '.', '-', '.', 'gene_id \"chr33.g11259\"; transcript_id \"chr33.g11259.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '41696337', '41696386', '.', '+', '.', 'gene_id \"chr24.g18342\"; transcript_id \"chr24.g18342.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29818440', '29818828', '.', '-', '.', 'gene_id \"chr33.g11436\"; transcript_id \"chr33.g11436.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42221548', '42221615', '.', '-', '.', 'gene_id \"chr24.g18462\"; transcript_id \"chr24.g18462.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29868611', '29868666', '.', '-', '.', 'gene_id \"chr33.g11509\"; transcript_id \"chr33.g11509.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42479712', '42479883', '.', '+', '.', 'gene_id \"chr24.g18552\"; transcript_id \"chr24.g18552.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29915079', '29915119', '.', '+', '.', 'gene_id \"chr33.g11540\"; transcript_id \"chr33.g11540.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42532958', '42533168', '.', '+', '.', 'gene_id \"chr24.g18563\"; transcript_id \"chr24.g18563.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42533980', '42534172', '.', '+', '.', 'gene_id \"chr24.g18564\"; transcript_id \"chr24.g18564.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42536613', '42536873', '.', '+', '.', 'gene_id \"chr24.g18566\"; transcript_id \"chr24.g18566.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42537245', '42537681', '.', '-', '.', 'gene_id \"chr24.g18567\"; transcript_id \"chr24.g18567.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '42537812', '42539111', '.', '-', '.', 'gene_id \"chr24.g18568\"; transcript_id \"chr24.g18568.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29925338', '29925450', '.', '+', '.', 'gene_id \"chr33.g11559\"; transcript_id \"chr33.g11559.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '29960320', '29960497', '.', '+', '.', 'gene_id \"chr33.g11584\"; transcript_id \"chr33.g11584.i6\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43053594', '43053639', '.', '+', '.', 'gene_id \"chr24.g18744\"; transcript_id \"chr24.g18744.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43179143', '43180004', '.', '-', '.', 'gene_id \"chr24.g18868\"; transcript_id \"chr24.g18868.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43180012', '43180362', '.', '-', '.', 'gene_id \"chr24.g18869\"; transcript_id \"chr24.g18869.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43352099', '43352128', '.', '-', '.', 'gene_id \"chr24.g18927\"; transcript_id \"chr24.g18927.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43439928', '43440083', '.', '-', '.', 'gene_id \"chr24.g18962\"; transcript_id \"chr24.g18962.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '30747342', '30747785', '.', '+', '.', 'gene_id \"chr33.g12138\"; transcript_id \"chr33.g12138.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '44243452', '44243658', '.', '+', '.', 'gene_id \"chr24.g19205\"; transcript_id \"chr24.g19205.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '31146860', '31147107', '.', '-', '.', 'gene_id \"chr33.g12414\"; transcript_id \"chr33.g12414.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46044557', '46044863', '.', '+', '.', 'gene_id \"chr24.g19496\"; transcript_id \"chr24.g19496.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46087599', '46087786', '.', '-', '.', 'gene_id \"chr24.g19547\"; transcript_id \"chr24.g19547.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46088754', '46088912', '.', '+', '.', 'gene_id \"chr24.g19549\"; transcript_id \"chr24.g19549.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '31366368', '31366415', '.', '-', '.', 'gene_id \"chr33.g12568\"; transcript_id \"chr33.g12568.i1\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26173939', '26173987', '100', '+', '.', 'gene_id \"chr33.g12718\"; transcript_id \"blat.proc4587.chain_181\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7109626', '7109656', '100', '-', '.', 'gene_id \"chr33.g12747\"; transcript_id \"blat.proc18128.chain_20274\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23854154', '23854218', '100', '+', '.', 'gene_id \"chr33.g12759\"; transcript_id \"blat.proc18128.chain_1664\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22172204', '22172326', '100', '+', '.', 'gene_id \"chr33.g12878\"; transcript_id \"blat.proc4005.chain_22690\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '3444288', '3444348', '100', '+', '.', 'gene_id \"chr33.g12895\"; transcript_id \"blat.proc18128.chain_12180\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '27785787', '27785831', '100', '+', '.', 'gene_id \"chr33.g12988\"; transcript_id \"blat.proc18128.chain_12036\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '14432382', '14432413', '100', '-', '.', 'gene_id \"chr33.g13196\"; transcript_id \"blat.proc28144.chain_13035\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '5217154', '5217190', '100', '+', '.', 'gene_id \"chr33.g13290\"; transcript_id \"blat.proc4005.chain_19896\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '26401231', '26401276', '100', '-', '.', 'gene_id \"chr33.g13393\"; transcript_id \"blat.proc4005.chain_2448\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7785326', '7785361', '100', '-', '.', 'gene_id \"chr33.g13768\"; transcript_id \"blat.proc4005.chain_2487\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '4166925', '4167162', '100', '+', '.', 'gene_id \"chr33.g13787\"; transcript_id \"blat.proc4587.chain_6864\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22581705', '22581737', '100', '+', '.', 'gene_id \"chr33.g13842\"; transcript_id \"blat.proc18128.chain_16207\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '11130157', '11130189', '100', '-', '.', 'gene_id \"chr33.g13906\"; transcript_id \"blat.proc18128.chain_7329\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '12508602', '12509137', '100', '+', '.', 'gene_id \"chr33.g14084\"; transcript_id \"blat.proc28144.chain_4340\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '11258174', '11258225', '100', '+', '.', 'gene_id \"chr33.g14115\"; transcript_id \"blat.proc18128.chain_16834\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '5508278', '5508525', '100', '+', '.', 'gene_id \"chr33.g14200\"; transcript_id \"blat.proc28144.chain_8377\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '23394998', '23395076', '100', '+', '.', 'gene_id \"chr33.g14260\"; transcript_id \"blat.proc28144.chain_23199\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '14939241', '14939396', '100', '+', '.', 'gene_id \"chr33.g14262\"; transcript_id \"blat.proc28144.chain_6221\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '2713558', '2713592', '100', '+', '.', 'gene_id \"chr33.g14513\"; transcript_id \"blat.proc4587.chain_8859\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '5238531', '5238562', '100', '+', '.', 'gene_id \"chr33.g14554\"; transcript_id \"blat.proc18128.chain_8982\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '3331472', '3331706', '100', '-', '.', 'gene_id \"chr33.g14658\"; transcript_id \"blat.proc18128.chain_20723\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '9160801', '9160838', '100', '+', '.', 'gene_id \"chr33.g14918\"; transcript_id \"blat.proc4005.chain_15585\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '22973933', '22973966', '100', '-', '.', 'gene_id \"chr33.g14947\"; transcript_id \"blat.proc18128.chain_10524\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '21436030', '21436098', '100', '-', '.', 'gene_id \"chr33.g15131\"; transcript_id \"blat.proc4587.chain_8418\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '7221644', '7221703', '100', '-', '.', 'gene_id \"chr33.g15376\"; transcript_id \"blat.proc18128.chain_10778\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '3630059', '3630102', '100', '-', '.', 'gene_id \"chr33.g15459\"; transcript_id \"blat.proc28144.chain_21581\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '9730374', '9730435', '100', '-', '.', 'gene_id \"chr33.g15686\"; transcript_id \"blat.proc28144.chain_15762\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '20380231', '20380287', '100', '-', '.', 'gene_id \"chr33.g15720\"; transcript_id \"blat.proc4587.chain_9739\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '25235585', '25235617', '100', '-', '.', 'gene_id \"chr33.g15729\"; transcript_id \"blat.proc4005.chain_8925\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '2078538', '2078576', '100', '+', '.', 'gene_id \"chr33.g15777\"; transcript_id \"blat.proc18128.chain_2577\";']\n",
      "negative:  ['chr33', 'PASA', 'exon', '28668894', '28669026', '100', '+', '.', 'gene_id \"chr33.g15902\"; transcript_id \"blat.proc18128.chain_17070\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46241149', '46241220', '.', '+', '.', 'gene_id \"chr24.g19697\"; transcript_id \"chr24.g19697.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46263465', '46263514', '.', '+', '.', 'gene_id \"chr24.g19720\"; transcript_id \"chr24.g19720.i2\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46331639', '46332093', '.', '-', '.', 'gene_id \"chr24.g19764\"; transcript_id \"chr24.g19764.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46483478', '46483683', '.', '-', '.', 'gene_id \"chr24.g19852\"; transcript_id \"chr24.g19852.i1\";']\n",
      "negative:  ['scaffold_1677', 'PASA', 'exon', '66114', '66269', '100', '-', '.', 'gene_id \"scaffold_1677.g20\"; transcript_id \"blat.proc4587.chain_12782\";']\n",
      "negative:  ['scaffold_1677', 'PASA', 'exon', '66275', '66726', '100', '+', '.', 'gene_id \"scaffold_1677.g25\"; transcript_id \"blat.proc4587.chain_12783\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46917678', '46918147', '.', '+', '.', 'gene_id \"chr24.g20066\"; transcript_id \"chr24.g20066.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46918428', '46918614', '.', '+', '.', 'gene_id \"chr24.g20068\"; transcript_id \"chr24.g20068.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '46919623', '46919901', '.', '-', '.', 'gene_id \"chr24.g20069\"; transcript_id \"chr24.g20069.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '47257451', '47258043', '.', '+', '.', 'gene_id \"chr24.g20277\"; transcript_id \"chr24.g20277.i7\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '47426664', '47426850', '.', '+', '.', 'gene_id \"chr24.g20429\"; transcript_id \"chr24.g20429.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '47426971', '47427203', '.', '-', '.', 'gene_id \"chr24.g20430\"; transcript_id \"chr24.g20430.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '47509757', '47510981', '.', '-', '.', 'gene_id \"chr24.g20502\"; transcript_id \"chr24.g20502.i1\";']\n",
      "negative:  ['scaffold_1635', 'PASA', 'exon', '14629', '15404', '.', '+', '.', 'gene_id \"scaffold_1635.g12\"; transcript_id \"scaffold_1635.g12.i1\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '43116908', '43117073', '100', '-', '.', 'gene_id \"chr24.g20594\"; transcript_id \"blat.proc4587.chain_6884\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23275729', '23275773', '100', '+', '.', 'gene_id \"chr24.g20813\"; transcript_id \"blat.proc4005.chain_19361\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '363272', '363387', '100', '-', '.', 'gene_id \"chr24.g20845\"; transcript_id \"blat.proc28144.chain_9292\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16545593', '16545640', '100', '-', '.', 'gene_id \"chr24.g20872\"; transcript_id \"blat.proc28144.chain_3778\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4547253', '4547376', '100', '-', '.', 'gene_id \"chr24.g21273\"; transcript_id \"blat.proc28144.chain_8217\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21050516', '21050547', '100', '+', '.', 'gene_id \"chr24.g21490\"; transcript_id \"blat.proc18128.chain_20991\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '31408399', '31408455', '100', '-', '.', 'gene_id \"chr24.g21528\"; transcript_id \"blat.proc18128.chain_21515\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '13050301', '13050333', '100', '+', '.', 'gene_id \"chr24.g21614\"; transcript_id \"blat.proc28144.chain_13515\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21093637', '21093685', '100', '+', '.', 'gene_id \"chr24.g21652\"; transcript_id \"blat.proc4005.chain_14663\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '40259005', '40259038', '100', '-', '.', 'gene_id \"chr24.g21976\"; transcript_id \"blat.proc28144.chain_4102\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '22768183', '22768223', '100', '-', '.', 'gene_id \"chr24.g22022\"; transcript_id \"blat.proc18128.chain_19628\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16764251', '16764291', '100', '-', '.', 'gene_id \"chr24.g22030\"; transcript_id \"blat.proc18128.chain_11461\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '33221352', '33221398', '100', '+', '.', 'gene_id \"chr24.g22215\"; transcript_id \"blat.proc18128.chain_7260\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37433721', '37433753', '100', '+', '.', 'gene_id \"chr24.g22281\"; transcript_id \"blat.proc18128.chain_2168\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19046486', '19046517', '100', '-', '.', 'gene_id \"chr24.g22282\"; transcript_id \"blat.proc28144.chain_15657\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '6674855', '6674920', '100', '-', '.', 'gene_id \"chr24.g22757\"; transcript_id \"blat.proc28144.chain_19969\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '36718212', '36718244', '100', '+', '.', 'gene_id \"chr24.g22939\"; transcript_id \"blat.proc28144.chain_15717\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9076690', '9076865', '100', '-', '.', 'gene_id \"chr24.g23069\"; transcript_id \"blat.proc4587.chain_11618\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23174060', '23174228', '100', '-', '.', 'gene_id \"chr24.g23097\"; transcript_id \"blat.proc4587.chain_3352\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '4123669', '4123706', '100', '-', '.', 'gene_id \"chr24.g23169\"; transcript_id \"blat.proc18128.chain_23028\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '35235670', '35235712', '100', '+', '.', 'gene_id \"chr24.g23465\"; transcript_id \"blat.proc4005.chain_8346\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '2254884', '2254922', '100', '-', '.', 'gene_id \"chr24.g23472\"; transcript_id \"blat.proc28144.chain_11861\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '17048250', '17048286', '100', '+', '.', 'gene_id \"chr24.g23641\"; transcript_id \"blat.proc4005.chain_21126\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '23104653', '23104692', '100', '+', '.', 'gene_id \"chr24.g23672\"; transcript_id \"blat.proc28144.chain_3957\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '3860558', '3860708', '100', '+', '.', 'gene_id \"chr24.g23933\"; transcript_id \"blat.proc18128.chain_25385\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18751637', '18751728', '100', '-', '.', 'gene_id \"chr24.g24082\"; transcript_id \"blat.proc4587.chain_10625\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '10077371', '10077425', '100', '-', '.', 'gene_id \"chr24.g24217\"; transcript_id \"blat.proc18128.chain_13077\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '11496650', '11496765', '100', '+', '.', 'gene_id \"chr24.g24511\"; transcript_id \"blat.proc28144.chain_1996\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '36716', '36799', '100', '-', '.', 'gene_id \"chr24.g24560\"; transcript_id \"blat.proc4005.chain_16939\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '3188483', '3188513', '100', '+', '.', 'gene_id \"chr24.g24619\"; transcript_id \"blat.proc28144.chain_10253\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '9662', '9739', '100', '-', '.', 'gene_id \"chr24.g24866\"; transcript_id \"blat.proc28144.chain_19228\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '39415730', '39415795', '100', '-', '.', 'gene_id \"chr24.g24959\"; transcript_id \"blat.proc18128.chain_13739\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '20546903', '20547130', '100', '+', '.', 'gene_id \"chr24.g25010\"; transcript_id \"blat.proc18128.chain_4102\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '18219862', '18219913', '100', '+', '.', 'gene_id \"chr24.g25052\"; transcript_id \"blat.proc4587.chain_12360\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '21962757', '21962792', '100', '+', '.', 'gene_id \"chr24.g25126\"; transcript_id \"blat.proc4587.chain_2529\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '16559566', '16559600', '100', '+', '.', 'gene_id \"chr24.g25215\"; transcript_id \"blat.proc4587.chain_2107\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24172240', '24172276', '100', '-', '.', 'gene_id \"chr24.g25251\"; transcript_id \"blat.proc4005.chain_21047\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '8617419', '8617530', '100', '+', '.', 'gene_id \"chr24.g25311\"; transcript_id \"blat.proc18128.chain_7239\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '24742610', '24742667', '100', '+', '.', 'gene_id \"chr24.g25318\"; transcript_id \"blat.proc28144.chain_18538\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '37906319', '37906348', '100', '-', '.', 'gene_id \"chr24.g25401\"; transcript_id \"blat.proc4587.chain_5253\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '14325604', '14325644', '100', '+', '.', 'gene_id \"chr24.g25425\"; transcript_id \"blat.proc28144.chain_5108\";']\n",
      "negative:  ['chr24', 'PASA', 'exon', '19046383', '19046469', '100', '-', '.', 'gene_id \"chr24.g25436\"; transcript_id \"blat.proc28144.chain_15656\";']\n",
      "negative:  ['scaffold_1499', 'PASA', 'exon', '14902', '14964', '100', '+', '.', 'gene_id \"scaffold_1499.g13\"; transcript_id \"blat.proc28144.chain_963\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49508', '50378', '.', '+', '.', 'gene_id \"chr22.g34\"; transcript_id \"chr22.g34.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '327478', '327921', '.', '-', '.', 'gene_id \"chr22.g309\"; transcript_id \"chr22.g309.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '336983', '337323', '.', '+', '.', 'gene_id \"chr22.g314\"; transcript_id \"chr22.g314.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '337346', '337541', '.', '-', '.', 'gene_id \"chr22.g315\"; transcript_id \"chr22.g315.i1\";']\n",
      "negative:  ['scaffold_1128', 'PASA', 'exon', '1256', '1340', '.', '+', '.', 'gene_id \"scaffold_1128.g1\"; transcript_id \"scaffold_1128.g1.i2\";']\n",
      "negative:  ['scaffold_1128', 'PASA', 'exon', '683', '953', '100', '+', '.', 'gene_id \"scaffold_1128.g5\"; transcript_id \"blat.proc28144.chain_2305\";']\n",
      "negative:  ['scaffold_1602', 'PASA', 'exon', '29593', '29657', '.', '-', '.', 'gene_id \"scaffold_1602.g21\"; transcript_id \"scaffold_1602.g21.i1\";']\n",
      "negative:  ['scaffold_1633', 'PASA', 'exon', '22657', '22916', '.', '+', '.', 'gene_id \"scaffold_1633.g2\"; transcript_id \"scaffold_1633.g2.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '1068217', '1068258', '.', '-', '.', 'gene_id \"chr22.g683\"; transcript_id \"chr22.g683.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '1401038', '1401079', '.', '+', '.', 'gene_id \"chr22.g721\"; transcript_id \"chr22.g721.i1\";']\n",
      "negative:  ['scaffold_1580', 'PASA', 'exon', '17854', '18071', '.', '-', '.', 'gene_id \"scaffold_1580.g18\"; transcript_id \"scaffold_1580.g18.i1\";']\n",
      "negative:  ['scaffold_1580', 'PASA', 'exon', '8624', '8802', '100', '-', '.', 'gene_id \"scaffold_1580.g35\"; transcript_id \"blat.proc18128.chain_423\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '1772948', '1774120', '.', '-', '.', 'gene_id \"chr22.g896\"; transcript_id \"chr22.g896.i1\";']\n",
      "negative:  ['scaffold_1537', 'PASA', 'exon', '15851', '15911', '.', '+', '.', 'gene_id \"scaffold_1537.g1\"; transcript_id \"scaffold_1537.g1.i2\";']\n",
      "negative:  ['scaffold_1537', 'PASA', 'exon', '8349', '8463', '100', '-', '.', 'gene_id \"scaffold_1537.g10\"; transcript_id \"blat.proc4587.chain_7148\";']\n",
      "negative:  ['scaffold_1632', 'PASA', 'exon', '7646', '7808', '.', '+', '.', 'gene_id \"scaffold_1632.g4\"; transcript_id \"scaffold_1632.g4.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2020537', '2020735', '.', '-', '.', 'gene_id \"chr22.g1044\"; transcript_id \"chr22.g1044.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2020776', '2021039', '.', '-', '.', 'gene_id \"chr22.g1045\"; transcript_id \"chr22.g1045.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2183077', '2183347', '.', '-', '.', 'gene_id \"chr22.g1216\"; transcript_id \"chr22.g1216.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2330210', '2330434', '.', '-', '.', 'gene_id \"chr22.g1346\"; transcript_id \"chr22.g1346.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr22', 'PASA', 'exon', '2330580', '2330929', '.', '-', '.', 'gene_id \"chr22.g1348\"; transcript_id \"chr22.g1348.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2364333', '2364435', '.', '-', '.', 'gene_id \"chr22.g1384\"; transcript_id \"chr22.g1384.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2388759', '2388920', '.', '+', '.', 'gene_id \"chr22.g1404\"; transcript_id \"chr22.g1404.i3\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2388759', '2389114', '.', '+', '.', 'gene_id \"chr22.g1404\"; transcript_id \"chr22.g1404.i5\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2389555', '2389621', '.', '+', '.', 'gene_id \"chr22.g1409\"; transcript_id \"chr22.g1409.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2404910', '2404956', '.', '+', '.', 'gene_id \"chr22.g1420\"; transcript_id \"chr22.g1420.i3\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2431092', '2431617', '.', '+', '.', 'gene_id \"chr22.g1457\"; transcript_id \"chr22.g1457.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2525031', '2525277', '.', '+', '.', 'gene_id \"chr22.g1513\"; transcript_id \"chr22.g1513.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2533932', '2534154', '.', '-', '.', 'gene_id \"chr22.g1520\"; transcript_id \"chr22.g1520.i1\";']\n",
      "negative:  ['scaffold_1080', 'PASA', 'exon', '8604', '8636', '.', '+', '.', 'gene_id \"scaffold_1080.g4\"; transcript_id \"scaffold_1080.g4.i1\";']\n",
      "negative:  ['scaffold_1685', 'PASA', 'exon', '13961', '14412', '100', '-', '.', 'gene_id \"scaffold_1685.g14\"; transcript_id \"blat.proc4587.chain_12784\";']\n",
      "negative:  ['scaffold_1685', 'PASA', 'exon', '14418', '14573', '100', '+', '.', 'gene_id \"scaffold_1685.g25\"; transcript_id \"blat.proc4587.chain_12780\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2898203', '2898259', '.', '+', '.', 'gene_id \"chr22.g1666\"; transcript_id \"chr22.g1666.i1\";']\n",
      "negative:  ['scaffold_1654', 'PASA', 'exon', '5456', '5551', '.', '-', '.', 'gene_id \"scaffold_1654.g2\"; transcript_id \"scaffold_1654.g2.i1\";']\n",
      "negative:  ['scaffold_1654', 'PASA', 'exon', '5698', '5851', '.', '-', '.', 'gene_id \"scaffold_1654.g3\"; transcript_id \"scaffold_1654.g3.i1\";']\n",
      "negative:  ['scaffold_1110', 'PASA', 'exon', '7696', '7737', '.', '+', '.', 'gene_id \"scaffold_1110.g4\"; transcript_id \"scaffold_1110.g4.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '3324783', '3324820', '.', '+', '.', 'gene_id \"chr22.g1998\"; transcript_id \"chr22.g1998.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '565', '728', '.', '+', '.', 'gene_id \"scaffold_1349.g1\"; transcript_id \"scaffold_1349.g1.i2\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '2417', '2582', '.', '+', '.', 'gene_id \"scaffold_1349.g1\"; transcript_id \"scaffold_1349.g1.i3\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '1', '140', '.', '+', '.', 'gene_id \"scaffold_1349.g2\"; transcript_id \"scaffold_1349.g2.i2\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '2588', '2845', '.', '-', '.', 'gene_id \"scaffold_1349.g3\"; transcript_id \"scaffold_1349.g3.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '132', '299', '.', '-', '.', 'gene_id \"scaffold_1349.g4\"; transcript_id \"scaffold_1349.g4.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '286', '485', '.', '-', '.', 'gene_id \"scaffold_1349.g5\"; transcript_id \"scaffold_1349.g5.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '964', '1529', '.', '-', '.', 'gene_id \"scaffold_1349.g6\"; transcript_id \"scaffold_1349.g6.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '1542', '2058', '.', '-', '.', 'gene_id \"scaffold_1349.g7\"; transcript_id \"scaffold_1349.g7.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '2092', '2357', '.', '-', '.', 'gene_id \"scaffold_1349.g8\"; transcript_id \"scaffold_1349.g8.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '7990', '8328', '.', '+', '.', 'gene_id \"scaffold_1349.g15\"; transcript_id \"scaffold_1349.g15.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '11097', '11260', '.', '-', '.', 'gene_id \"scaffold_1349.g20\"; transcript_id \"scaffold_1349.g20.i2\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '13755', '13980', '.', '+', '.', 'gene_id \"scaffold_1349.g24\"; transcript_id \"scaffold_1349.g24.i1\";']\n",
      "negative:  ['scaffold_1349', 'PASA', 'exon', '17042', '17256', '.', '-', '.', 'gene_id \"scaffold_1349.g27\"; transcript_id \"scaffold_1349.g27.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '3854133', '3854178', '.', '+', '.', 'gene_id \"chr22.g2081\"; transcript_id \"chr22.g2081.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4441022', '4441193', '.', '-', '.', 'gene_id \"chr22.g2126\"; transcript_id \"chr22.g2126.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4447654', '4448249', '.', '-', '.', 'gene_id \"chr22.g2133\"; transcript_id \"chr22.g2133.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4448591', '4448792', '.', '-', '.', 'gene_id \"chr22.g2134\"; transcript_id \"chr22.g2134.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4586650', '4587260', '.', '-', '.', 'gene_id \"chr22.g2250\"; transcript_id \"chr22.g2250.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4634208', '4634480', '.', '+', '.', 'gene_id \"chr22.g2292\"; transcript_id \"chr22.g2292.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4814187', '4814400', '.', '+', '.', 'gene_id \"chr22.g2425\"; transcript_id \"chr22.g2425.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4829727', '4829922', '.', '-', '.', 'gene_id \"chr22.g2439\"; transcript_id \"chr22.g2439.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '1679898', '1679931', '.', '+', '.', 'gene_id \"chr18.g257\"; transcript_id \"chr18.g257.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4941697', '4941872', '.', '-', '.', 'gene_id \"chr22.g2554\"; transcript_id \"chr22.g2554.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5047499', '5047809', '.', '-', '.', 'gene_id \"chr22.g2598\"; transcript_id \"chr22.g2598.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5206064', '5206094', '.', '-', '.', 'gene_id \"chr22.g2647\"; transcript_id \"chr22.g2647.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5224206', '5224238', '.', '+', '.', 'gene_id \"chr22.g2652\"; transcript_id \"chr22.g2652.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5399557', '5399629', '.', '+', '.', 'gene_id \"chr22.g2712\"; transcript_id \"chr22.g2712.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5411994', '5412052', '.', '-', '.', 'gene_id \"chr22.g2715\"; transcript_id \"chr22.g2715.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5473821', '5473854', '.', '-', '.', 'gene_id \"chr22.g2775\"; transcript_id \"chr22.g2775.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5544075', '5544513', '.', '+', '.', 'gene_id \"chr22.g2824\"; transcript_id \"chr22.g2824.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5656770', '5656811', '.', '-', '.', 'gene_id \"chr22.g2962\"; transcript_id \"chr22.g2962.i3\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5926226', '5926378', '.', '-', '.', 'gene_id \"chr22.g3109\"; transcript_id \"chr22.g3109.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5925519', '5925672', '.', '+', '.', 'gene_id \"chr22.g3110\"; transcript_id \"chr22.g3110.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '2471996', '2472177', '.', '-', '.', 'gene_id \"chr18.g840\"; transcript_id \"chr18.g840.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '3009596', '3010525', '.', '+', '.', 'gene_id \"chr18.g897\"; transcript_id \"chr18.g897.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '5810614', '5810794', '.', '-', '.', 'gene_id \"chr18.g1163\"; transcript_id \"chr18.g1163.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '5884555', '5884755', '.', '+', '.', 'gene_id \"chr18.g1191\"; transcript_id \"chr18.g1191.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '5948700', '5948983', '.', '+', '.', 'gene_id \"chr18.g1287\"; transcript_id \"chr18.g1287.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '5949078', '5949268', '.', '+', '.', 'gene_id \"chr18.g1289\"; transcript_id \"chr18.g1289.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '5950558', '5950840', '.', '+', '.', 'gene_id \"chr18.g1291\"; transcript_id \"chr18.g1291.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7237874', '7238043', '.', '-', '.', 'gene_id \"chr22.g3600\"; transcript_id \"chr22.g3600.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7238080', '7238270', '.', '+', '.', 'gene_id \"chr22.g3601\"; transcript_id \"chr22.g3601.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7624356', '7624866', '.', '-', '.', 'gene_id \"chr22.g3753\"; transcript_id \"chr22.g3753.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7680724', '7680762', '.', '+', '.', 'gene_id \"chr22.g3786\"; transcript_id \"chr22.g3786.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7851475', '7851821', '.', '+', '.', 'gene_id \"chr22.g3858\"; transcript_id \"chr22.g3858.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '7853103', '7853318', '.', '+', '.', 'gene_id \"chr22.g3862\"; transcript_id \"chr22.g3862.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '8730459', '8730504', '.', '+', '.', 'gene_id \"chr22.g4134\"; transcript_id \"chr22.g4134.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '7472944', '7473392', '.', '-', '.', 'gene_id \"chr18.g1893\"; transcript_id \"chr18.g1893.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '7786437', '7786660', '.', '-', '.', 'gene_id \"chr18.g1917\"; transcript_id \"chr18.g1917.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '7786443', '7786660', '.', '-', '.', 'gene_id \"chr18.g1917\"; transcript_id \"chr18.g1917.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '8039235', '8039389', '.', '+', '.', 'gene_id \"chr18.g2002\"; transcript_id \"chr18.g2002.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '8992373', '8992588', '.', '+', '.', 'gene_id \"chr22.g4280\"; transcript_id \"chr22.g4280.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '8173162', '8173205', '.', '-', '.', 'gene_id \"chr18.g2052\"; transcript_id \"chr18.g2052.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9047740', '9047937', '.', '-', '.', 'gene_id \"chr22.g4331\"; transcript_id \"chr22.g4331.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9138824', '9138968', '.', '-', '.', 'gene_id \"chr22.g4399\"; transcript_id \"chr22.g4399.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9267342', '9267613', '.', '+', '.', 'gene_id \"chr18.g2257\"; transcript_id \"chr18.g2257.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9312970', '9313677', '.', '+', '.', 'gene_id \"chr22.g4547\"; transcript_id \"chr22.g4547.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9399177', '9399390', '.', '-', '.', 'gene_id \"chr22.g4570\"; transcript_id \"chr22.g4570.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9398922', '9399191', '.', '-', '.', 'gene_id \"chr22.g4579\"; transcript_id \"chr22.g4579.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9398996', '9399314', '.', '+', '.', 'gene_id \"chr18.g2330\"; transcript_id \"chr18.g2330.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9399436', '9399593', '.', '-', '.', 'gene_id \"chr18.g2331\"; transcript_id \"chr18.g2331.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9399586', '9399804', '.', '+', '.', 'gene_id \"chr18.g2332\"; transcript_id \"chr18.g2332.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9400101', '9400305', '.', '-', '.', 'gene_id \"chr18.g2334\"; transcript_id \"chr18.g2334.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9597444', '9597713', '.', '-', '.', 'gene_id \"chr18.g2375\"; transcript_id \"chr18.g2375.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9620226', '9620393', '.', '-', '.', 'gene_id \"chr22.g4774\"; transcript_id \"chr22.g4774.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9865334', '9865697', '.', '-', '.', 'gene_id \"chr18.g2496\"; transcript_id \"chr18.g2496.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '10208671', '10208825', '.', '-', '.', 'gene_id \"chr22.g4878\"; transcript_id \"chr22.g4878.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '10250751', '10250923', '.', '-', '.', 'gene_id \"chr22.g4897\"; transcript_id \"chr22.g4897.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '12370530', '12371008', '.', '-', '.', 'gene_id \"chr22.g5021\"; transcript_id \"chr22.g5021.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '12894615', '12894652', '.', '+', '.', 'gene_id \"chr22.g5040\"; transcript_id \"chr22.g5040.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '13190793', '13191168', '.', '-', '.', 'gene_id \"chr22.g5061\"; transcript_id \"chr22.g5061.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '15551295', '15551689', '.', '-', '.', 'gene_id \"chr22.g5261\"; transcript_id \"chr22.g5261.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '11824762', '11825568', '.', '+', '.', 'gene_id \"chr18.g3075\"; transcript_id \"chr18.g3075.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '16891783', '16891823', '.', '+', '.', 'gene_id \"chr22.g5500\"; transcript_id \"chr22.g5500.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '18676357', '18676422', '.', '+', '.', 'gene_id \"chr22.g5619\"; transcript_id \"chr22.g5619.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '18956096', '18956135', '.', '-', '.', 'gene_id \"chr22.g5651\"; transcript_id \"chr22.g5651.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '19575099', '19575188', '.', '+', '.', 'gene_id \"chr22.g5680\"; transcript_id \"chr22.g5680.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '12743418', '12743664', '.', '+', '.', 'gene_id \"chr18.g3433\"; transcript_id \"chr18.g3433.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '12991194', '12991794', '.', '+', '.', 'gene_id \"chr18.g3603\"; transcript_id \"chr18.g3603.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '22688600', '22688647', '.', '+', '.', 'gene_id \"chr22.g5930\"; transcript_id \"chr22.g5930.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '23219493', '23219528', '.', '+', '.', 'gene_id \"chr22.g5967\"; transcript_id \"chr22.g5967.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '25103849', '25103928', '.', '-', '.', 'gene_id \"chr22.g6115\"; transcript_id \"chr22.g6115.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '13536053', '13536095', '.', '+', '.', 'gene_id \"chr18.g4012\"; transcript_id \"chr18.g4012.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '25893627', '25893779', '.', '+', '.', 'gene_id \"chr22.g6209\"; transcript_id \"chr22.g6209.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '14258735', '14258780', '.', '+', '.', 'gene_id \"chr18.g4198\"; transcript_id \"chr18.g4198.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26858817', '26858984', '.', '-', '.', 'gene_id \"chr22.g6430\"; transcript_id \"chr22.g6430.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26859015', '26859199', '.', '-', '.', 'gene_id \"chr22.g6431\"; transcript_id \"chr22.g6431.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26916162', '26916462', '.', '+', '.', 'gene_id \"chr22.g6449\"; transcript_id \"chr22.g6449.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26909222', '26909465', '.', '+', '.', 'gene_id \"chr22.g6478\"; transcript_id \"chr22.g6478.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26943493', '26943665', '.', '-', '.', 'gene_id \"chr22.g6492\"; transcript_id \"chr22.g6492.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26948618', '26948774', '.', '-', '.', 'gene_id \"chr22.g6498\"; transcript_id \"chr22.g6498.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27041225', '27041494', '.', '+', '.', 'gene_id \"chr22.g6515\"; transcript_id \"chr22.g6515.i5\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27041432', '27041494', '.', '+', '.', 'gene_id \"chr22.g6515\"; transcript_id \"chr22.g6515.i7\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26968950', '26969130', '.', '-', '.', 'gene_id \"chr22.g6520\"; transcript_id \"chr22.g6520.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26973404', '26973644', '.', '-', '.', 'gene_id \"chr22.g6521\"; transcript_id \"chr22.g6521.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26970281', '26970435', '.', '-', '.', 'gene_id \"chr22.g6523\"; transcript_id \"chr22.g6523.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27046164', '27046745', '.', '+', '.', 'gene_id \"chr22.g6560\"; transcript_id \"chr22.g6560.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27045719', '27045905', '.', '-', '.', 'gene_id \"chr22.g6561\"; transcript_id \"chr22.g6561.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27047000', '27047212', '.', '+', '.', 'gene_id \"chr22.g6563\"; transcript_id \"chr22.g6563.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27047200', '27047782', '.', '-', '.', 'gene_id \"chr22.g6564\"; transcript_id \"chr22.g6564.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27099693', '27100187', '.', '+', '.', 'gene_id \"chr22.g6585\"; transcript_id \"chr22.g6585.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27116193', '27116369', '.', '+', '.', 'gene_id \"chr22.g6599\"; transcript_id \"chr22.g6599.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27838061', '27838174', '.', '-', '.', 'gene_id \"chr22.g6784\"; transcript_id \"chr22.g6784.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27838154', '27838316', '.', '+', '.', 'gene_id \"chr22.g6785\"; transcript_id \"chr22.g6785.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27965438', '27965689', '.', '-', '.', 'gene_id \"chr22.g6846\"; transcript_id \"chr22.g6846.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '28174947', '28175007', '.', '-', '.', 'gene_id \"chr22.g6919\"; transcript_id \"chr22.g6919.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15303358', '15303566', '.', '+', '.', 'gene_id \"chr18.g4798\"; transcript_id \"chr18.g4798.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15303358', '15303566', '.', '+', '.', 'gene_id \"chr18.g4798\"; transcript_id \"chr18.g4798.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15157465', '15157756', '.', '+', '.', 'gene_id \"chr18.g4886\"; transcript_id \"chr18.g4886.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '29107294', '29107804', '.', '-', '.', 'gene_id \"chr22.g7053\"; transcript_id \"chr22.g7053.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15401947', '15402395', '.', '-', '.', 'gene_id \"chr18.g5088\"; transcript_id \"chr18.g5088.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '29436465', '29436549', '.', '-', '.', 'gene_id \"chr22.g7286\"; transcript_id \"chr22.g7286.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15436278', '15436455', '.', '-', '.', 'gene_id \"chr18.g5141\"; transcript_id \"chr18.g5141.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15539466', '15539709', '.', '-', '.', 'gene_id \"chr18.g5195\"; transcript_id \"chr18.g5195.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '29872650', '29872709', '.', '-', '.', 'gene_id \"chr22.g7379\"; transcript_id \"chr22.g7379.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15912991', '15913501', '.', '-', '.', 'gene_id \"chr18.g5255\"; transcript_id \"chr18.g5255.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30045605', '30045642', '.', '-', '.', 'gene_id \"chr22.g7409\"; transcript_id \"chr22.g7409.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30158684', '30158722', '.', '+', '.', 'gene_id \"chr22.g7422\"; transcript_id \"chr22.g7422.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30189500', '30189677', '.', '-', '.', 'gene_id \"chr22.g7424\"; transcript_id \"chr22.g7424.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '16267514', '16267713', '.', '+', '.', 'gene_id \"chr18.g5402\"; transcript_id \"chr18.g5402.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '16326392', '16326509', '.', '-', '.', 'gene_id \"chr18.g5433\"; transcript_id \"chr18.g5433.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30664151', '30664395', '.', '-', '.', 'gene_id \"chr22.g7705\"; transcript_id \"chr22.g7705.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30681271', '30681377', '.', '+', '.', 'gene_id \"chr22.g7721\"; transcript_id \"chr22.g7721.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '17082309', '17082625', '.', '-', '.', 'gene_id \"chr18.g5700\"; transcript_id \"chr18.g5700.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '17249585', '17249778', '.', '+', '.', 'gene_id \"chr18.g5776\"; transcript_id \"chr18.g5776.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '31259608', '31259649', '.', '-', '.', 'gene_id \"chr22.g8019\"; transcript_id \"chr22.g8019.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '32279167', '32279228', '.', '-', '.', 'gene_id \"chr22.g8146\"; transcript_id \"chr22.g8146.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '17716445', '17716614', '.', '-', '.', 'gene_id \"chr18.g6056\"; transcript_id \"chr18.g6056.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '32759585', '32759630', '.', '+', '.', 'gene_id \"chr22.g8311\"; transcript_id \"chr22.g8311.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '33054354', '33054393', '.', '-', '.', 'gene_id \"chr22.g8349\"; transcript_id \"chr22.g8349.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '33246734', '33246775', '.', '+', '.', 'gene_id \"chr22.g8357\"; transcript_id \"chr22.g8357.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '34305445', '34305503', '.', '+', '.', 'gene_id \"chr22.g8478\"; transcript_id \"chr22.g8478.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '35134776', '35134809', '.', '+', '.', 'gene_id \"chr22.g8552\"; transcript_id \"chr22.g8552.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '35139559', '35139613', '.', '-', '.', 'gene_id \"chr22.g8555\"; transcript_id \"chr22.g8555.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '35165228', '35165267', '.', '-', '.', 'gene_id \"chr22.g8559\"; transcript_id \"chr22.g8559.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18438673', '18439496', '.', '-', '.', 'gene_id \"chr18.g6505\"; transcript_id \"chr18.g6505.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18458481', '18458682', '.', '-', '.', 'gene_id \"chr18.g6529\"; transcript_id \"chr18.g6529.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18461708', '18462126', '.', '+', '.', 'gene_id \"chr18.g6531\"; transcript_id \"chr18.g6531.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18461332', '18461734', '.', '+', '.', 'gene_id \"chr18.g6533\"; transcript_id \"chr18.g6533.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18471127', '18471341', '.', '+', '.', 'gene_id \"chr18.g6546\"; transcript_id \"chr18.g6546.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '36947709', '36947763', '.', '+', '.', 'gene_id \"chr22.g8677\"; transcript_id \"chr22.g8677.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18807526', '18807736', '.', '+', '.', 'gene_id \"chr18.g6660\"; transcript_id \"chr18.g6660.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18819391', '18819665', '.', '-', '.', 'gene_id \"chr18.g6667\"; transcript_id \"chr18.g6667.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '39512723', '39512758', '.', '+', '.', 'gene_id \"chr22.g8861\"; transcript_id \"chr22.g8861.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '19015012', '19015238', '.', '+', '.', 'gene_id \"chr18.g6809\"; transcript_id \"chr18.g6809.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '40060558', '40060642', '.', '+', '.', 'gene_id \"chr22.g8891\"; transcript_id \"chr22.g8891.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '19170885', '19171190', '.', '+', '.', 'gene_id \"chr18.g6878\"; transcript_id \"chr18.g6878.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '19257916', '19258102', '.', '+', '.', 'gene_id \"chr18.g6922\"; transcript_id \"chr18.g6922.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '42139908', '42140070', '.', '-', '.', 'gene_id \"chr22.g9035\"; transcript_id \"chr22.g9035.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '42404501', '42404905', '.', '+', '.', 'gene_id \"chr22.g9071\"; transcript_id \"chr22.g9071.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '42712996', '42714039', '.', '-', '.', 'gene_id \"chr22.g9168\"; transcript_id \"chr22.g9168.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '42784388', '42784423', '.', '-', '.', 'gene_id \"chr22.g9176\"; transcript_id \"chr22.g9176.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '43673463', '43673495', '.', '+', '.', 'gene_id \"chr22.g9239\"; transcript_id \"chr22.g9239.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '44652417', '44652468', '.', '+', '.', 'gene_id \"chr22.g9334\"; transcript_id \"chr22.g9334.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '45102311', '45102350', '.', '-', '.', 'gene_id \"chr22.g9400\"; transcript_id \"chr22.g9400.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46134813', '46135075', '.', '+', '.', 'gene_id \"chr22.g9619\"; transcript_id \"chr22.g9619.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46151163', '46151670', '.', '+', '.', 'gene_id \"chr22.g9625\"; transcript_id \"chr22.g9625.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46208714', '46208798', '.', '-', '.', 'gene_id \"chr22.g9657\"; transcript_id \"chr22.g9657.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46304524', '46304947', '.', '-', '.', 'gene_id \"chr22.g9724\"; transcript_id \"chr22.g9724.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '23693407', '23693439', '.', '+', '.', 'gene_id \"chr18.g7793\"; transcript_id \"chr18.g7793.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '23976958', '23977022', '.', '-', '.', 'gene_id \"chr18.g7808\"; transcript_id \"chr18.g7808.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46722574', '46723389', '.', '-', '.', 'gene_id \"chr22.g10018\"; transcript_id \"chr22.g10018.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46723086', '46723316', '.', '-', '.', 'gene_id \"chr22.g10018\"; transcript_id \"chr22.g10018.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '25397993', '25398323', '.', '-', '.', 'gene_id \"chr18.g7938\"; transcript_id \"chr18.g7938.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '47219501', '47219545', '.', '-', '.', 'gene_id \"chr22.g10086\"; transcript_id \"chr22.g10086.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '47352838', '47354629', '.', '+', '.', 'gene_id \"chr22.g10094\"; transcript_id \"chr22.g10094.i3\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '47347790', '47347972', '.', '-', '.', 'gene_id \"chr22.g10095\"; transcript_id \"chr22.g10095.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr18', 'PASA', 'exon', '25883534', '25884027', '.', '-', '.', 'gene_id \"chr18.g8262\"; transcript_id \"chr18.g8262.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '26006839', '26009973', '.', '+', '.', 'gene_id \"chr18.g8333\"; transcript_id \"chr18.g8333.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48197881', '48197977', '.', '+', '.', 'gene_id \"chr22.g10440\"; transcript_id \"chr22.g10440.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48197909', '48197975', '.', '+', '.', 'gene_id \"chr22.g10440\"; transcript_id \"chr22.g10440.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48293571', '48293661', '.', '+', '.', 'gene_id \"chr22.g10499\"; transcript_id \"chr22.g10499.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48548972', '48549623', '.', '+', '.', 'gene_id \"chr22.g10656\"; transcript_id \"chr22.g10656.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48620539', '48620573', '.', '+', '.', 'gene_id \"chr22.g10685\"; transcript_id \"chr22.g10685.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '28222032', '28222187', '.', '+', '.', 'gene_id \"chr18.g8776\"; transcript_id \"chr18.g8776.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48881339', '48881378', '.', '-', '.', 'gene_id \"chr22.g10847\"; transcript_id \"chr22.g10847.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48983349', '48983555', '.', '-', '.', 'gene_id \"chr22.g10903\"; transcript_id \"chr22.g10903.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '48983557', '48984156', '.', '+', '.', 'gene_id \"chr22.g10906\"; transcript_id \"chr22.g10906.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '31454610', '31454650', '.', '-', '.', 'gene_id \"chr18.g8964\"; transcript_id \"chr18.g8964.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49402693', '49402725', '.', '+', '.', 'gene_id \"chr22.g11174\"; transcript_id \"chr22.g11174.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49409363', '49409412', '.', '+', '.', 'gene_id \"chr22.g11180\"; transcript_id \"chr22.g11180.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49732858', '49732903', '.', '-', '.', 'gene_id \"chr22.g11398\"; transcript_id \"chr22.g11398.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '32369520', '32369569', '.', '-', '.', 'gene_id \"chr18.g9359\"; transcript_id \"chr18.g9359.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49906567', '49906788', '.', '-', '.', 'gene_id \"chr22.g11534\"; transcript_id \"chr22.g11534.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49937252', '49937485', '.', '+', '.', 'gene_id \"chr22.g11548\"; transcript_id \"chr22.g11548.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49937706', '49938155', '.', '-', '.', 'gene_id \"chr22.g11549\"; transcript_id \"chr22.g11549.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50188475', '50188534', '.', '-', '.', 'gene_id \"chr22.g11664\"; transcript_id \"chr22.g11664.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50199941', '50200321', '.', '+', '.', 'gene_id \"chr22.g11683\"; transcript_id \"chr22.g11683.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50471075', '50471414', '.', '-', '.', 'gene_id \"chr22.g11815\"; transcript_id \"chr22.g11815.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50471443', '50471714', '.', '-', '.', 'gene_id \"chr22.g11816\"; transcript_id \"chr22.g11816.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '32869509', '32869980', '.', '-', '.', 'gene_id \"chr18.g9727\"; transcript_id \"chr18.g9727.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50542458', '50542618', '.', '+', '.', 'gene_id \"chr22.g11883\"; transcript_id \"chr22.g11883.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50571676', '50571890', '.', '+', '.', 'gene_id \"chr22.g11920\"; transcript_id \"chr22.g11920.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '51967103', '51967260', '.', '+', '.', 'gene_id \"chr22.g12167\"; transcript_id \"chr22.g12167.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '33567675', '33567877', '.', '-', '.', 'gene_id \"chr18.g10130\"; transcript_id \"chr18.g10130.i4\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '33567231', '33567693', '.', '-', '.', 'gene_id \"chr18.g10141\"; transcript_id \"chr18.g10141.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52220638', '52220840', '.', '+', '.', 'gene_id \"chr22.g12306\"; transcript_id \"chr22.g12306.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '33877619', '33878746', '.', '+', '.', 'gene_id \"chr18.g10397\"; transcript_id \"chr18.g10397.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '33877213', '33877487', '.', '+', '.', 'gene_id \"chr18.g10403\"; transcript_id \"chr18.g10403.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52475302', '52476036', '.', '+', '.', 'gene_id \"chr22.g12489\"; transcript_id \"chr22.g12489.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '33924711', '33925434', '.', '+', '.', 'gene_id \"chr18.g10469\"; transcript_id \"chr18.g10469.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52493378', '52493623', '.', '+', '.', 'gene_id \"chr22.g12511\"; transcript_id \"chr22.g12511.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52494787', '52495932', '.', '-', '.', 'gene_id \"chr22.g12512\"; transcript_id \"chr22.g12512.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52926419', '52926864', '.', '+', '.', 'gene_id \"chr22.g12588\"; transcript_id \"chr22.g12588.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '34200501', '34200679', '.', '+', '.', 'gene_id \"chr18.g10620\"; transcript_id \"chr18.g10620.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '54947281', '54947319', '.', '+', '.', 'gene_id \"chr22.g12682\"; transcript_id \"chr22.g12682.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '34467548', '34467587', '.', '+', '.', 'gene_id \"chr18.g10663\"; transcript_id \"chr18.g10663.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '55892269', '55892303', '.', '+', '.', 'gene_id \"chr22.g12903\"; transcript_id \"chr22.g12903.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '56435577', '56435628', '.', '+', '.', 'gene_id \"chr22.g12964\"; transcript_id \"chr22.g12964.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '35344984', '35345149', '.', '-', '.', 'gene_id \"chr18.g11140\"; transcript_id \"chr18.g11140.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '35578041', '35578139', '.', '+', '.', 'gene_id \"chr18.g11205\"; transcript_id \"chr18.g11205.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '57801272', '57801328', '.', '+', '.', 'gene_id \"chr22.g13210\"; transcript_id \"chr22.g13210.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '35854388', '35854871', '.', '-', '.', 'gene_id \"chr18.g11244\"; transcript_id \"chr18.g11244.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '58130228', '58130378', '.', '+', '.', 'gene_id \"chr22.g13269\"; transcript_id \"chr22.g13269.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '58759271', '58759525', '.', '+', '.', 'gene_id \"chr22.g13510\"; transcript_id \"chr22.g13510.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '36194838', '36195284', '.', '-', '.', 'gene_id \"chr18.g11532\"; transcript_id \"chr18.g11532.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '58829416', '58829776', '.', '+', '.', 'gene_id \"chr22.g13608\"; transcript_id \"chr22.g13608.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '58830153', '58830464', '.', '+', '.', 'gene_id \"chr22.g13610\"; transcript_id \"chr22.g13610.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '59055298', '59055913', '.', '+', '.', 'gene_id \"chr22.g13785\"; transcript_id \"chr22.g13785.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '59011436', '59011701', '.', '+', '.', 'gene_id \"chr22.g13798\"; transcript_id \"chr22.g13798.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '59318716', '59319021', '.', '-', '.', 'gene_id \"chr22.g13972\"; transcript_id \"chr22.g13972.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '59318716', '59318958', '.', '-', '.', 'gene_id \"chr22.g13972\"; transcript_id \"chr22.g13972.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '59549731', '59549777', '.', '-', '.', 'gene_id \"chr22.g14051\"; transcript_id \"chr22.g14051.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '37857814', '37857940', '.', '-', '.', 'gene_id \"chr18.g12027\"; transcript_id \"chr18.g12027.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '37862430', '37862760', '.', '-', '.', 'gene_id \"chr18.g12031\"; transcript_id \"chr18.g12031.i2\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '60068932', '60070347', '.', '-', '.', 'gene_id \"chr22.g14101\"; transcript_id \"chr22.g14101.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '60428306', '60428354', '.', '-', '.', 'gene_id \"chr22.g14218\"; transcript_id \"chr22.g14218.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '60553440', '60555405', '.', '-', '.', 'gene_id \"chr22.g14230\"; transcript_id \"chr22.g14230.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '38102631', '38102812', '.', '-', '.', 'gene_id \"chr18.g12196\"; transcript_id \"chr18.g12196.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '38101787', '38101946', '.', '-', '.', 'gene_id \"chr18.g12197\"; transcript_id \"chr18.g12197.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '38748615', '38748791', '.', '-', '.', 'gene_id \"chr18.g12362\"; transcript_id \"chr18.g12362.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '38969438', '38969469', '.', '+', '.', 'gene_id \"chr18.g12552\"; transcript_id \"chr18.g12552.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '39824962', '39824998', '.', '+', '.', 'gene_id \"chr18.g12685\"; transcript_id \"chr18.g12685.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61147760', '61148237', '.', '-', '.', 'gene_id \"chr22.g14860\"; transcript_id \"chr22.g14860.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61216927', '61217104', '.', '-', '.', 'gene_id \"chr22.g14929\"; transcript_id \"chr22.g14929.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61251030', '61251354', '.', '+', '.', 'gene_id \"chr22.g14954\"; transcript_id \"chr22.g14954.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '41776517', '41776955', '.', '-', '.', 'gene_id \"chr18.g12864\"; transcript_id \"chr18.g12864.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61341488', '61341657', '.', '-', '.', 'gene_id \"chr22.g15011\"; transcript_id \"chr22.g15011.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61347502', '61347677', '.', '+', '.', 'gene_id \"chr22.g15016\"; transcript_id \"chr22.g15016.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61347931', '61348233', '.', '-', '.', 'gene_id \"chr22.g15017\"; transcript_id \"chr22.g15017.i1\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '19799484', '19799523', '100', '+', '.', 'gene_id \"chr22.g15165\"; transcript_id \"blat.proc4587.chain_183\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46287048', '46287101', '100', '-', '.', 'gene_id \"chr22.g15264\"; transcript_id \"blat.proc18128.chain_7565\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '46385447', '46385502', '100', '-', '.', 'gene_id \"chr22.g15396\"; transcript_id \"blat.proc18128.chain_15534\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '47605728', '47605921', '100', '+', '.', 'gene_id \"chr22.g15496\"; transcript_id \"blat.proc28144.chain_18726\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '19842934', '19842969', '100', '+', '.', 'gene_id \"chr22.g15604\"; transcript_id \"blat.proc18128.chain_17785\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '42409077', '42409134', '100', '+', '.', 'gene_id \"chr22.g15671\"; transcript_id \"blat.proc4587.chain_5521\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '39270302', '39270461', '100', '+', '.', 'gene_id \"chr22.g15860\"; transcript_id \"blat.proc4005.chain_22907\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61356478', '61356544', '100', '-', '.', 'gene_id \"chr22.g15864\"; transcript_id \"blat.proc28144.chain_8843\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5981607', '5981685', '100', '+', '.', 'gene_id \"chr22.g15943\"; transcript_id \"blat.proc18128.chain_19588\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '61184617', '61184651', '100', '-', '.', 'gene_id \"chr22.g15993\"; transcript_id \"blat.proc18128.chain_2348\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '33415000', '33415398', '100', '+', '.', 'gene_id \"chr22.g16445\"; transcript_id \"blat.proc18128.chain_4035\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '5739528', '5739611', '100', '-', '.', 'gene_id \"chr22.g16502\"; transcript_id \"blat.proc4587.chain_6463\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '26769171', '26769230', '100', '-', '.', 'gene_id \"chr22.g16531\"; transcript_id \"blat.proc4587.chain_9768\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '18764965', '18765004', '100', '-', '.', 'gene_id \"chr22.g16867\"; transcript_id \"blat.proc4587.chain_8839\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '11047415', '11047603', '100', '-', '.', 'gene_id \"chr22.g16976\"; transcript_id \"blat.proc4005.chain_23066\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '18037882', '18037924', '100', '-', '.', 'gene_id \"chr22.g17034\"; transcript_id \"blat.proc18128.chain_1178\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '52093272', '52093360', '100', '+', '.', 'gene_id \"chr22.g17070\"; transcript_id \"blat.proc18128.chain_6432\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '9197636', '9197753', '100', '-', '.', 'gene_id \"chr22.g17298\"; transcript_id \"blat.proc4587.chain_237\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '6839856', '6839890', '100', '-', '.', 'gene_id \"chr22.g17350\"; transcript_id \"blat.proc18128.chain_12715\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '38393423', '38393464', '100', '+', '.', 'gene_id \"chr22.g17376\"; transcript_id \"blat.proc18128.chain_21689\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '34847524', '34847622', '100', '-', '.', 'gene_id \"chr22.g17430\"; transcript_id \"blat.proc18128.chain_25638\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '20011510', '20011705', '100', '+', '.', 'gene_id \"chr22.g18032\"; transcript_id \"blat.proc28144.chain_5668\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '17229765', '17229806', '100', '-', '.', 'gene_id \"chr22.g18035\"; transcript_id \"blat.proc28144.chain_6743\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27318315', '27318353', '100', '-', '.', 'gene_id \"chr22.g18068\"; transcript_id \"blat.proc18128.chain_17987\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '55117145', '55117189', '100', '-', '.', 'gene_id \"chr22.g18079\"; transcript_id \"blat.proc28144.chain_6817\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '43356759', '43356899', '100', '+', '.', 'gene_id \"chr22.g18220\"; transcript_id \"blat.proc28144.chain_15456\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '28011492', '28011546', '100', '+', '.', 'gene_id \"chr22.g18327\"; transcript_id \"blat.proc4005.chain_22832\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '30037035', '30037068', '100', '+', '.', 'gene_id \"chr22.g18351\"; transcript_id \"blat.proc4587.chain_9777\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '49288380', '49288414', '100', '-', '.', 'gene_id \"chr22.g18352\"; transcript_id \"blat.proc18128.chain_15610\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '2481904', '2482002', '100', '+', '.', 'gene_id \"chr22.g18634\"; transcript_id \"blat.proc18128.chain_13132\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '6054815', '6054862', '100', '+', '.', 'gene_id \"chr22.g18675\"; transcript_id \"blat.proc4587.chain_9939\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '27376996', '27377043', '100', '+', '.', 'gene_id \"chr22.g18758\"; transcript_id \"blat.proc28144.chain_7877\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '50126541', '50126643', '100', '-', '.', 'gene_id \"chr22.g18976\"; transcript_id \"blat.proc18128.chain_25584\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '4870740', '4870776', '100', '-', '.', 'gene_id \"chr22.g19071\"; transcript_id \"blat.proc18128.chain_12714\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '29460516', '29460576', '100', '-', '.', 'gene_id \"chr22.g19126\"; transcript_id \"blat.proc4005.chain_16494\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '56501819', '56501878', '100', '+', '.', 'gene_id \"chr22.g19349\"; transcript_id \"blat.proc4587.chain_4663\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '23367621', '23367660', '100', '-', '.', 'gene_id \"chr22.g19468\"; transcript_id \"blat.proc28144.chain_6028\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '17487423', '17487507', '100', '+', '.', 'gene_id \"chr22.g19557\"; transcript_id \"blat.proc28144.chain_17355\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '56348269', '56348326', '100', '-', '.', 'gene_id \"chr22.g19563\"; transcript_id \"blat.proc4587.chain_6543\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '29857341', '29857566', '100', '+', '.', 'gene_id \"chr22.g19862\"; transcript_id \"blat.proc18128.chain_8332\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '647866', '647913', '100', '+', '.', 'gene_id \"chr22.g19912\"; transcript_id \"blat.proc28144.chain_18949\";']\n",
      "negative:  ['chr22', 'PASA', 'exon', '43118001', '43118033', '100', '-', '.', 'gene_id \"chr22.g19918\"; transcript_id \"blat.proc28144.chain_11591\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '42204683', '42204995', '.', '+', '.', 'gene_id \"chr18.g13134\"; transcript_id \"chr18.g13134.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '55680', '55709', '.', '+', '.', 'gene_id \"chr36.g6\"; transcript_id \"chr36.g6.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '42300466', '42300700', '.', '-', '.', 'gene_id \"chr18.g13229\"; transcript_id \"chr18.g13229.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '409332', '409373', '.', '-', '.', 'gene_id \"chr36.g48\"; transcript_id \"chr36.g48.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '870915', '870978', '.', '+', '.', 'gene_id \"chr36.g122\"; transcript_id \"chr36.g122.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '1037531', '1037565', '.', '-', '.', 'gene_id \"chr36.g173\"; transcript_id \"chr36.g173.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '1942436', '1942475', '.', '-', '.', 'gene_id \"chr36.g303\"; transcript_id \"chr36.g303.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '1965414', '1965452', '.', '+', '.', 'gene_id \"chr36.g304\"; transcript_id \"chr36.g304.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2383404', '2383450', '.', '-', '.', 'gene_id \"chr36.g364\"; transcript_id \"chr36.g364.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2662228', '2662275', '.', '+', '.', 'gene_id \"chr36.g388\"; transcript_id \"chr36.g388.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2826115', '2826291', '.', '-', '.', 'gene_id \"chr36.g451\"; transcript_id \"chr36.g451.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2826421', '2826896', '.', '-', '.', 'gene_id \"chr36.g452\"; transcript_id \"chr36.g452.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2868954', '2869266', '.', '+', '.', 'gene_id \"chr36.g504\"; transcript_id \"chr36.g504.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '3391503', '3391544', '.', '-', '.', 'gene_id \"chr36.g571\"; transcript_id \"chr36.g571.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '3863725', '3864040', '.', '+', '.', 'gene_id \"chr36.g673\"; transcript_id \"chr36.g673.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '3863725', '3864118', '.', '+', '.', 'gene_id \"chr36.g673\"; transcript_id \"chr36.g673.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '43433278', '43433325', '.', '-', '.', 'gene_id \"chr18.g14154\"; transcript_id \"chr18.g14154.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '43624846', '43624875', '.', '-', '.', 'gene_id \"chr18.g14195\"; transcript_id \"chr18.g14195.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4196811', '4197110', '.', '+', '.', 'gene_id \"chr36.g888\"; transcript_id \"chr36.g888.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4281509', '4281672', '.', '+', '.', 'gene_id \"chr36.g962\"; transcript_id \"chr36.g962.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4281711', '4282020', '.', '-', '.', 'gene_id \"chr36.g963\"; transcript_id \"chr36.g963.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4450314', '4450371', '.', '+', '.', 'gene_id \"chr36.g996\"; transcript_id \"chr36.g996.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4473669', '4473701', '.', '-', '.', 'gene_id \"chr36.g1033\"; transcript_id \"chr36.g1033.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4525286', '4525335', '.', '-', '.', 'gene_id \"chr36.g1061\"; transcript_id \"chr36.g1061.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '43827897', '43828106', '.', '-', '.', 'gene_id \"chr18.g14453\"; transcript_id \"chr18.g14453.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '44255719', '44255898', '.', '-', '.', 'gene_id \"chr18.g14569\"; transcript_id \"chr18.g14569.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr36', 'PASA', 'exon', '5080367', '5080523', '.', '+', '.', 'gene_id \"chr36.g1283\"; transcript_id \"chr36.g1283.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '44534891', '44535087', '.', '+', '.', 'gene_id \"chr18.g14702\"; transcript_id \"chr18.g14702.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5151444', '5151509', '.', '-', '.', 'gene_id \"chr36.g1356\"; transcript_id \"chr36.g1356.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '44816567', '44816771', '.', '+', '.', 'gene_id \"chr18.g14782\"; transcript_id \"chr18.g14782.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '44841803', '44841972', '.', '+', '.', 'gene_id \"chr18.g14803\"; transcript_id \"chr18.g14803.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '44841607', '44841814', '.', '-', '.', 'gene_id \"chr18.g14804\"; transcript_id \"chr18.g14804.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5441973', '5442117', '.', '-', '.', 'gene_id \"chr36.g1618\"; transcript_id \"chr36.g1618.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '45334356', '45334510', '.', '+', '.', 'gene_id \"chr18.g15019\"; transcript_id \"chr18.g15019.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '45334551', '45334758', '.', '-', '.', 'gene_id \"chr18.g15020\"; transcript_id \"chr18.g15020.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5490782', '5490815', '.', '-', '.', 'gene_id \"chr36.g1641\"; transcript_id \"chr36.g1641.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5534764', '5535462', '.', '-', '.', 'gene_id \"chr36.g1688\"; transcript_id \"chr36.g1688.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '45409775', '45409931', '.', '+', '.', 'gene_id \"chr18.g15114\"; transcript_id \"chr18.g15114.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '45441139', '45441369', '.', '+', '.', 'gene_id \"chr18.g15147\"; transcript_id \"chr18.g15147.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '45469156', '45469620', '.', '+', '.', 'gene_id \"chr18.g15185\"; transcript_id \"chr18.g15185.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5630193', '5630538', '.', '+', '.', 'gene_id \"chr36.g1826\"; transcript_id \"chr36.g1826.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5630681', '5630879', '.', '-', '.', 'gene_id \"chr36.g1827\"; transcript_id \"chr36.g1827.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5832207', '5832239', '.', '-', '.', 'gene_id \"chr36.g1928\"; transcript_id \"chr36.g1928.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5992123', '5992471', '.', '+', '.', 'gene_id \"chr36.g1975\"; transcript_id \"chr36.g1975.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6035305', '6035353', '.', '+', '.', 'gene_id \"chr36.g2042\"; transcript_id \"chr36.g2042.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6145101', '6145139', '.', '+', '.', 'gene_id \"chr36.g2094\"; transcript_id \"chr36.g2094.i2\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '46160667', '46160840', '.', '-', '.', 'gene_id \"chr18.g15524\"; transcript_id \"chr18.g15524.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6180680', '6180906', '.', '-', '.', 'gene_id \"chr36.g2139\"; transcript_id \"chr36.g2139.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6180884', '6181190', '.', '+', '.', 'gene_id \"chr36.g2140\"; transcript_id \"chr36.g2140.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6230281', '6230459', '.', '-', '.', 'gene_id \"chr36.g2200\"; transcript_id \"chr36.g2200.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6307706', '6308026', '.', '-', '.', 'gene_id \"chr36.g2252\"; transcript_id \"chr36.g2252.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6429366', '6429931', '.', '+', '.', 'gene_id \"chr36.g2294\"; transcript_id \"chr36.g2294.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6588444', '6588484', '.', '+', '.', 'gene_id \"chr36.g2310\"; transcript_id \"chr36.g2310.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '46780333', '46780595', '.', '-', '.', 'gene_id \"chr18.g15775\"; transcript_id \"chr18.g15775.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6897330', '6897364', '.', '-', '.', 'gene_id \"chr36.g2360\"; transcript_id \"chr36.g2360.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6917221', '6917263', '.', '+', '.', 'gene_id \"chr36.g2387\"; transcript_id \"chr36.g2387.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '46965887', '46966104', '.', '-', '.', 'gene_id \"chr18.g15877\"; transcript_id \"chr18.g15877.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '46966105', '46966676', '.', '-', '.', 'gene_id \"chr18.g15878\"; transcript_id \"chr18.g15878.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '7073526', '7073563', '.', '+', '.', 'gene_id \"chr36.g2501\"; transcript_id \"chr36.g2501.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '47215380', '47215576', '.', '-', '.', 'gene_id \"chr18.g16068\"; transcript_id \"chr18.g16068.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '7691371', '7691429', '.', '+', '.', 'gene_id \"chr36.g2667\"; transcript_id \"chr36.g2667.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '7782017', '7782068', '.', '+', '.', 'gene_id \"chr36.g2720\"; transcript_id \"chr36.g2720.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '47488072', '47488596', '.', '+', '.', 'gene_id \"chr18.g16192\"; transcript_id \"chr18.g16192.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '8615228', '8615308', '.', '+', '.', 'gene_id \"chr36.g2836\"; transcript_id \"chr36.g2836.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '9549271', '9549338', '.', '+', '.', 'gene_id \"chr36.g3007\"; transcript_id \"chr36.g3007.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10053285', '10053779', '.', '-', '.', 'gene_id \"chr36.g3179\"; transcript_id \"chr36.g3179.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10053893', '10054063', '.', '-', '.', 'gene_id \"chr36.g3180\"; transcript_id \"chr36.g3180.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10060510', '10060663', '.', '-', '.', 'gene_id \"chr36.g3189\"; transcript_id \"chr36.g3189.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10130957', '10131158', '.', '+', '.', 'gene_id \"chr36.g3245\"; transcript_id \"chr36.g3245.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10331380', '10331536', '.', '-', '.', 'gene_id \"chr36.g3401\"; transcript_id \"chr36.g3401.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '49362483', '49362708', '.', '-', '.', 'gene_id \"chr18.g16919\"; transcript_id \"chr18.g16919.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '49363759', '49363949', '.', '+', '.', 'gene_id \"chr18.g16948\"; transcript_id \"chr18.g16948.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10808500', '10808531', '.', '+', '.', 'gene_id \"chr36.g3479\"; transcript_id \"chr36.g3479.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10812625', '10812889', '.', '+', '.', 'gene_id \"chr36.g3505\"; transcript_id \"chr36.g3505.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '11264556', '11264714', '.', '-', '.', 'gene_id \"chr36.g3652\"; transcript_id \"chr36.g3652.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '11366488', '11366519', '.', '-', '.', 'gene_id \"chr36.g3692\"; transcript_id \"chr36.g3692.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '49768791', '49768954', '.', '-', '.', 'gene_id \"chr18.g17211\"; transcript_id \"chr18.g17211.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '11827962', '11828972', '.', '-', '.', 'gene_id \"chr36.g3824\"; transcript_id \"chr36.g3824.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '12280904', '12280939', '.', '+', '.', 'gene_id \"chr36.g3865\"; transcript_id \"chr36.g3865.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '49979730', '49979765', '.', '+', '.', 'gene_id \"chr18.g17406\"; transcript_id \"chr18.g17406.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '12639782', '12639814', '.', '+', '.', 'gene_id \"chr36.g3906\"; transcript_id \"chr36.g3906.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '13010893', '13010927', '.', '-', '.', 'gene_id \"chr36.g3944\"; transcript_id \"chr36.g3944.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '50354977', '50355180', '.', '-', '.', 'gene_id \"chr18.g17752\"; transcript_id \"chr18.g17752.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '50355356', '50355622', '.', '-', '.', 'gene_id \"chr18.g17753\"; transcript_id \"chr18.g17753.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '50401471', '50401505', '.', '-', '.', 'gene_id \"chr18.g17789\"; transcript_id \"chr18.g17789.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14317271', '14317308', '.', '+', '.', 'gene_id \"chr36.g4529\"; transcript_id \"chr36.g4529.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14408318', '14408508', '.', '-', '.', 'gene_id \"chr36.g4558\"; transcript_id \"chr36.g4558.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14720817', '14721123', '.', '-', '.', 'gene_id \"chr36.g4857\"; transcript_id \"chr36.g4857.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14876460', '14876645', '.', '-', '.', 'gene_id \"chr36.g4960\"; transcript_id \"chr36.g4960.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14896880', '14897083', '.', '+', '.', 'gene_id \"chr36.g4975\"; transcript_id \"chr36.g4975.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14952326', '14952384', '.', '-', '.', 'gene_id \"chr36.g5032\"; transcript_id \"chr36.g5032.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '51738388', '51739092', '.', '+', '.', 'gene_id \"chr18.g18791\"; transcript_id \"chr18.g18791.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '15289133', '15289165', '.', '+', '.', 'gene_id \"chr36.g5078\"; transcript_id \"chr36.g5078.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '51777237', '51778185', '.', '+', '.', 'gene_id \"chr18.g18862\"; transcript_id \"chr18.g18862.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '15733325', '15733480', '.', '-', '.', 'gene_id \"chr36.g5224\"; transcript_id \"chr36.g5224.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '51970098', '51970166', '.', '-', '.', 'gene_id \"chr18.g19050\"; transcript_id \"chr18.g19050.i13\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '15888535', '15888820', '.', '+', '.', 'gene_id \"chr36.g5369\"; transcript_id \"chr36.g5369.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '16185918', '16186041', '.', '+', '.', 'gene_id \"chr36.g5586\"; transcript_id \"chr36.g5586.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '16430237', '16430287', '.', '+', '.', 'gene_id \"chr36.g5761\"; transcript_id \"chr36.g5761.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '16436691', '16436721', '.', '+', '.', 'gene_id \"chr36.g5774\"; transcript_id \"chr36.g5774.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '16601084', '16601236', '.', '+', '.', 'gene_id \"chr36.g5874\"; transcript_id \"chr36.g5874.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '17154785', '17155062', '.', '+', '.', 'gene_id \"chr36.g6106\"; transcript_id \"chr36.g6106.i6\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '17155119', '17155294', '.', '-', '.', 'gene_id \"chr36.g6117\"; transcript_id \"chr36.g6117.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53220781', '53220963', '.', '-', '.', 'gene_id \"chr18.g20101\"; transcript_id \"chr18.g20101.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '17521886', '17522229', '.', '+', '.', 'gene_id \"chr36.g6339\"; transcript_id \"chr36.g6339.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53384522', '53384553', '.', '+', '.', 'gene_id \"chr18.g20233\"; transcript_id \"chr18.g20233.i6\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53384514', '53384545', '.', '+', '.', 'gene_id \"chr18.g20233\"; transcript_id \"chr18.g20233.i7\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53462577', '53462723', '.', '-', '.', 'gene_id \"chr18.g20352\"; transcript_id \"chr18.g20352.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53638252', '53638303', '.', '+', '.', 'gene_id \"chr18.g20484\"; transcript_id \"chr18.g20484.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53742805', '53743351', '.', '-', '.', 'gene_id \"chr18.g20557\"; transcript_id \"chr18.g20557.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18366814', '18367020', '.', '-', '.', 'gene_id \"chr36.g6713\"; transcript_id \"chr36.g6713.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '53986790', '53986828', '.', '+', '.', 'gene_id \"chr18.g20703\"; transcript_id \"chr18.g20703.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '54028984', '54029027', '.', '+', '.', 'gene_id \"chr18.g20757\"; transcript_id \"chr18.g20757.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18569731', '18569765', '.', '-', '.', 'gene_id \"chr36.g6791\"; transcript_id \"chr36.g6791.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '54085237', '54085422', '.', '-', '.', 'gene_id \"chr18.g20799\"; transcript_id \"chr18.g20799.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '54222342', '54222589', '.', '+', '.', 'gene_id \"chr18.g20915\"; transcript_id \"chr18.g20915.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18896246', '18896530', '.', '+', '.', 'gene_id \"chr36.g7068\"; transcript_id \"chr36.g7068.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18926600', '18926819', '.', '-', '.', 'gene_id \"chr36.g7105\"; transcript_id \"chr36.g7105.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19054309', '19054343', '.', '+', '.', 'gene_id \"chr36.g7239\"; transcript_id \"chr36.g7239.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19170394', '19170447', '.', '+', '.', 'gene_id \"chr36.g7257\"; transcript_id \"chr36.g7257.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '55191092', '55191806', '.', '+', '.', 'gene_id \"chr18.g21580\"; transcript_id \"chr18.g21580.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr36', 'PASA', 'exon', '19680706', '19680895', '.', '-', '.', 'gene_id \"chr36.g7466\"; transcript_id \"chr36.g7466.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19680916', '19681160', '.', '-', '.', 'gene_id \"chr36.g7467\"; transcript_id \"chr36.g7467.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19917477', '19917762', '.', '+', '.', 'gene_id \"chr36.g7490\"; transcript_id \"chr36.g7490.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19963699', '19963739', '.', '-', '.', 'gene_id \"chr36.g7515\"; transcript_id \"chr36.g7515.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '20280049', '20280079', '.', '-', '.', 'gene_id \"chr36.g7665\"; transcript_id \"chr36.g7665.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '20302419', '20302462', '.', '-', '.', 'gene_id \"chr36.g7684\"; transcript_id \"chr36.g7684.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '55827206', '55827632', '.', '+', '.', 'gene_id \"chr18.g21907\"; transcript_id \"chr18.g21907.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '20531235', '20531275', '.', '+', '.', 'gene_id \"chr36.g7724\"; transcript_id \"chr36.g7724.i1\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '23211912', '23211971', '100', '+', '.', 'gene_id \"chr18.g22009\"; transcript_id \"blat.proc4587.chain_6553\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '41435597', '41435663', '100', '+', '.', 'gene_id \"chr18.g22103\"; transcript_id \"blat.proc4587.chain_11491\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '21212188', '21212425', '100', '-', '.', 'gene_id \"chr18.g22449\"; transcript_id \"blat.proc18128.chain_4109\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '47542977', '47543014', '100', '+', '.', 'gene_id \"chr18.g22778\"; transcript_id \"blat.proc4587.chain_14103\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '47155135', '47155180', '100', '-', '.', 'gene_id \"chr18.g22921\"; transcript_id \"blat.proc4005.chain_22440\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18466085', '18466378', '100', '-', '.', 'gene_id \"chr18.g22969\"; transcript_id \"blat.proc4587.chain_6120\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '13745065', '13745221', '100', '-', '.', 'gene_id \"chr18.g23664\"; transcript_id \"blat.proc4005.chain_21019\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '22186411', '22186452', '100', '-', '.', 'gene_id \"chr18.g23700\"; transcript_id \"blat.proc28144.chain_6745\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '17890772', '17890811', '100', '-', '.', 'gene_id \"chr18.g24014\"; transcript_id \"blat.proc4005.chain_7712\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '11819132', '11819174', '100', '-', '.', 'gene_id \"chr18.g24017\"; transcript_id \"blat.proc4587.chain_9888\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '7193547', '7193600', '100', '-', '.', 'gene_id \"chr18.g24032\"; transcript_id \"blat.proc4587.chain_9177\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18831807', '18831842', '100', '-', '.', 'gene_id \"chr18.g24277\"; transcript_id \"blat.proc4005.chain_10676\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '31693893', '31693926', '100', '-', '.', 'gene_id \"chr18.g24281\"; transcript_id \"blat.proc4587.chain_11780\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '13856584', '13856695', '100', '+', '.', 'gene_id \"chr18.g24320\"; transcript_id \"blat.proc18128.chain_7241\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '22438221', '22438260', '100', '-', '.', 'gene_id \"chr18.g24410\"; transcript_id \"blat.proc18128.chain_13778\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '34827502', '34827541', '100', '+', '.', 'gene_id \"chr18.g24795\"; transcript_id \"blat.proc18128.chain_18456\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '50679382', '50679439', '100', '-', '.', 'gene_id \"chr18.g25169\"; transcript_id \"blat.proc4587.chain_1585\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18287659', '18287698', '100', '+', '.', 'gene_id \"chr18.g25348\"; transcript_id \"blat.proc18128.chain_18375\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '18269560', '18269592', '100', '+', '.', 'gene_id \"chr18.g25532\"; transcript_id \"blat.proc18128.chain_21182\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15069255', '15069333', '100', '-', '.', 'gene_id \"chr18.g25631\"; transcript_id \"blat.proc4587.chain_10752\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '19090134', '19090181', '100', '-', '.', 'gene_id \"chr18.g25690\"; transcript_id \"blat.proc28144.chain_7554\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '43177401', '43177449', '100', '-', '.', 'gene_id \"chr18.g25920\"; transcript_id \"blat.proc28144.chain_12434\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '48415454', '48415502', '100', '+', '.', 'gene_id \"chr18.g26163\"; transcript_id \"blat.proc4587.chain_5600\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '10216588', '10216635', '100', '+', '.', 'gene_id \"chr18.g26506\"; transcript_id \"blat.proc4005.chain_6172\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '9671932', '9671961', '100', '-', '.', 'gene_id \"chr18.g26562\"; transcript_id \"blat.proc28144.chain_23254\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '37135991', '37136160', '100', '+', '.', 'gene_id \"chr18.g26804\"; transcript_id \"blat.proc4587.chain_4964\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '42925154', '42925241', '100', '-', '.', 'gene_id \"chr18.g26844\"; transcript_id \"blat.proc28144.chain_20188\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '52291911', '52292105', '100', '-', '.', 'gene_id \"chr18.g26948\"; transcript_id \"blat.proc18128.chain_14520\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '24894165', '24894201', '100', '-', '.', 'gene_id \"chr18.g27006\"; transcript_id \"blat.proc18128.chain_15285\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '51858181', '51858223', '100', '+', '.', 'gene_id \"chr18.g27025\"; transcript_id \"blat.proc4587.chain_6559\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '49539620', '49539717', '100', '-', '.', 'gene_id \"chr18.g27072\"; transcript_id \"blat.proc4587.chain_3277\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '15850452', '15850492', '100', '+', '.', 'gene_id \"chr18.g27077\"; transcript_id \"blat.proc4005.chain_2709\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '6392097', '6392172', '100', '-', '.', 'gene_id \"chr18.g27079\"; transcript_id \"blat.proc4587.chain_11254\";']\n",
      "negative:  ['chr18', 'PASA', 'exon', '21293107', '21293152', '100', '+', '.', 'gene_id \"chr18.g27306\"; transcript_id \"blat.proc4587.chain_6467\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21005217', '21005256', '.', '+', '.', 'gene_id \"chr36.g7787\"; transcript_id \"chr36.g7787.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21220801', '21221424', '.', '-', '.', 'gene_id \"chr36.g7887\"; transcript_id \"chr36.g7887.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21221760', '21222134', '.', '+', '.', 'gene_id \"chr36.g7888\"; transcript_id \"chr36.g7888.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21279452', '21279786', '.', '-', '.', 'gene_id \"chr36.g7948\"; transcript_id \"chr36.g7948.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '888', '938', '.', '-', '.', 'gene_id \"chr32.g4\"; transcript_id \"chr32.g4.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '137496', '137531', '.', '+', '.', 'gene_id \"chr32.g103\"; transcript_id \"chr32.g103.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21491814', '21491967', '.', '-', '.', 'gene_id \"chr36.g8087\"; transcript_id \"chr36.g8087.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '259426', '259515', '.', '+', '.', 'gene_id \"chr32.g195\"; transcript_id \"chr32.g195.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '280814', '280843', '.', '-', '.', 'gene_id \"chr32.g230\"; transcript_id \"chr32.g230.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '536479', '536548', '.', '-', '.', 'gene_id \"chr32.g409\"; transcript_id \"chr32.g409.i6\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '548677', '549192', '.', '-', '.', 'gene_id \"chr32.g420\"; transcript_id \"chr32.g420.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '538782', '538898', '.', '+', '.', 'gene_id \"chr32.g425\"; transcript_id \"chr32.g425.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '544500', '544671', '.', '-', '.', 'gene_id \"chr32.g432\"; transcript_id \"chr32.g432.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '549249', '549591', '.', '+', '.', 'gene_id \"chr32.g436\"; transcript_id \"chr32.g436.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22307630', '22307881', '.', '+', '.', 'gene_id \"chr36.g8423\"; transcript_id \"chr36.g8423.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22387655', '22387868', '.', '+', '.', 'gene_id \"chr36.g8500\"; transcript_id \"chr36.g8500.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22387655', '22387868', '.', '+', '.', 'gene_id \"chr36.g8500\"; transcript_id \"chr36.g8500.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '876079', '876189', '.', '+', '.', 'gene_id \"chr32.g620\"; transcript_id \"chr32.g620.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1041116', '1041389', '.', '+', '.', 'gene_id \"chr32.g688\"; transcript_id \"chr32.g688.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22714513', '22714693', '.', '-', '.', 'gene_id \"chr36.g8704\"; transcript_id \"chr36.g8704.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1246099', '1246281', '.', '+', '.', 'gene_id \"chr32.g793\"; transcript_id \"chr32.g793.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22920377', '22920453', '.', '+', '.', 'gene_id \"chr36.g8765\"; transcript_id \"chr36.g8765.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '23195082', '23195232', '.', '-', '.', 'gene_id \"chr36.g8863\"; transcript_id \"chr36.g8863.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '23281495', '23281707', '.', '+', '.', 'gene_id \"chr36.g8915\"; transcript_id \"chr36.g8915.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1564882', '1565042', '.', '+', '.', 'gene_id \"chr32.g1047\"; transcript_id \"chr32.g1047.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '23666547', '23666694', '.', '+', '.', 'gene_id \"chr36.g9030\"; transcript_id \"chr36.g9030.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1965639', '1965689', '.', '+', '.', 'gene_id \"chr32.g1166\"; transcript_id \"chr32.g1166.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '23974174', '23974312', '.', '-', '.', 'gene_id \"chr36.g9068\"; transcript_id \"chr36.g9068.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2002267', '2002574', '.', '-', '.', 'gene_id \"chr32.g1169\"; transcript_id \"chr32.g1169.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '24537015', '24538427', '.', '+', '.', 'gene_id \"chr36.g9101\"; transcript_id \"chr36.g9101.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2167210', '2167271', '.', '+', '.', 'gene_id \"chr32.g1193\"; transcript_id \"chr32.g1193.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2211062', '2211211', '.', '-', '.', 'gene_id \"chr32.g1203\"; transcript_id \"chr32.g1203.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2228142', '2228315', '.', '-', '.', 'gene_id \"chr32.g1210\"; transcript_id \"chr32.g1210.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '24533816', '24533981', '.', '+', '.', 'gene_id \"chr36.g9139\"; transcript_id \"chr36.g9139.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '24534063', '24534177', '.', '-', '.', 'gene_id \"chr36.g9140\"; transcript_id \"chr36.g9140.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2306423', '2306463', '.', '-', '.', 'gene_id \"chr32.g1254\"; transcript_id \"chr32.g1254.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2334405', '2334601', '.', '-', '.', 'gene_id \"chr32.g1291\"; transcript_id \"chr32.g1291.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2335110', '2336140', '.', '+', '.', 'gene_id \"chr32.g1292\"; transcript_id \"chr32.g1292.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2345543', '2345859', '.', '-', '.', 'gene_id \"chr32.g1307\"; transcript_id \"chr32.g1307.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2424739', '2425001', '.', '+', '.', 'gene_id \"chr32.g1355\"; transcript_id \"chr32.g1355.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2426237', '2426395', '.', '+', '.', 'gene_id \"chr32.g1357\"; transcript_id \"chr32.g1357.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '25419519', '25419559', '.', '-', '.', 'gene_id \"chr36.g9333\"; transcript_id \"chr36.g9333.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2513374', '2513417', '.', '-', '.', 'gene_id \"chr32.g1437\"; transcript_id \"chr32.g1437.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '25582085', '25582280', '.', '+', '.', 'gene_id \"chr36.g9409\"; transcript_id \"chr36.g9409.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2627881', '2628019', '.', '+', '.', 'gene_id \"chr32.g1463\"; transcript_id \"chr32.g1463.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '25677139', '25677290', '.', '+', '.', 'gene_id \"chr36.g9432\"; transcript_id \"chr36.g9432.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '25758673', '25758718', '.', '+', '.', 'gene_id \"chr36.g9461\"; transcript_id \"chr36.g9461.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2913757', '2914282', '.', '-', '.', 'gene_id \"chr32.g1540\"; transcript_id \"chr32.g1540.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3324829', '3325247', '.', '+', '.', 'gene_id \"chr32.g1758\"; transcript_id \"chr32.g1758.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3324470', '3324853', '.', '-', '.', 'gene_id \"chr32.g1759\"; transcript_id \"chr32.g1759.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3325237', '3325403', '.', '-', '.', 'gene_id \"chr32.g1760\"; transcript_id \"chr32.g1760.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3378550', '3378743', '.', '+', '.', 'gene_id \"chr32.g1781\"; transcript_id \"chr32.g1781.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3410525', '3410680', '.', '-', '.', 'gene_id \"chr32.g1782\"; transcript_id \"chr32.g1782.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '26426708', '26426898', '.', '-', '.', 'gene_id \"chr36.g9795\"; transcript_id \"chr36.g9795.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3868501', '3868712', '.', '+', '.', 'gene_id \"chr32.g1845\"; transcript_id \"chr32.g1845.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3935702', '3935781', '.', '-', '.', 'gene_id \"chr32.g1856\"; transcript_id \"chr32.g1856.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4277477', '4277665', '.', '+', '.', 'gene_id \"chr32.g1942\"; transcript_id \"chr32.g1942.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '27617190', '27617532', '.', '+', '.', 'gene_id \"chr36.g9941\"; transcript_id \"chr36.g9941.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4479976', '4480150', '.', '+', '.', 'gene_id \"chr32.g2041\"; transcript_id \"chr32.g2041.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4494210', '4494503', '.', '-', '.', 'gene_id \"chr32.g2053\"; transcript_id \"chr32.g2053.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4495557', '4495840', '.', '-', '.', 'gene_id \"chr32.g2053\"; transcript_id \"chr32.g2053.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4496128', '4496303', '.', '+', '.', 'gene_id \"chr32.g2055\"; transcript_id \"chr32.g2055.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '28212427', '28212461', '.', '+', '.', 'gene_id \"chr36.g10019\"; transcript_id \"chr36.g10019.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4534169', '4534235', '.', '-', '.', 'gene_id \"chr32.g2081\"; transcript_id \"chr32.g2081.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '28761559', '28761781', '.', '-', '.', 'gene_id \"chr36.g10048\"; transcript_id \"chr36.g10048.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '28806377', '28806438', '.', '+', '.', 'gene_id \"chr36.g10054\"; transcript_id \"chr36.g10054.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '4729598', '4729638', '.', '-', '.', 'gene_id \"chr32.g2123\"; transcript_id \"chr32.g2123.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5250981', '5251205', '.', '+', '.', 'gene_id \"chr32.g2189\"; transcript_id \"chr32.g2189.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5251208', '5251360', '.', '-', '.', 'gene_id \"chr32.g2190\"; transcript_id \"chr32.g2190.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5596762', '5597312', '.', '+', '.', 'gene_id \"chr32.g2244\"; transcript_id \"chr32.g2244.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5771907', '5772060', '.', '-', '.', 'gene_id \"chr32.g2323\"; transcript_id \"chr32.g2323.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5856002', '5856194', '.', '-', '.', 'gene_id \"chr32.g2338\"; transcript_id \"chr32.g2338.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5884430', '5884595', '.', '+', '.', 'gene_id \"chr32.g2345\"; transcript_id \"chr32.g2345.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5885716', '5886140', '.', '+', '.', 'gene_id \"chr32.g2346\"; transcript_id \"chr32.g2346.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5911231', '5911387', '.', '+', '.', 'gene_id \"chr32.g2350\"; transcript_id \"chr32.g2350.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '29884600', '29884643', '.', '-', '.', 'gene_id \"chr36.g10432\"; transcript_id \"chr36.g10432.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30321140', '30321293', '.', '+', '.', 'gene_id \"chr36.g10468\"; transcript_id \"chr36.g10468.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30321761', '30322152', '.', '+', '.', 'gene_id \"chr36.g10469\"; transcript_id \"chr36.g10469.i3\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6439211', '6439254', '.', '-', '.', 'gene_id \"chr32.g2502\"; transcript_id \"chr32.g2502.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6472499', '6472561', '.', '+', '.', 'gene_id \"chr32.g2521\"; transcript_id \"chr32.g2521.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30503053', '30503106', '.', '-', '.', 'gene_id \"chr36.g10555\"; transcript_id \"chr36.g10555.i2\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30501997', '30503245', '.', '+', '.', 'gene_id \"chr36.g10556\"; transcript_id \"chr36.g10556.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6574073', '6574226', '.', '-', '.', 'gene_id \"chr32.g2606\"; transcript_id \"chr32.g2606.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6574237', '6574448', '.', '+', '.', 'gene_id \"chr32.g2607\"; transcript_id \"chr32.g2607.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6574449', '6574629', '.', '-', '.', 'gene_id \"chr32.g2608\"; transcript_id \"chr32.g2608.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30620904', '30620943', '.', '-', '.', 'gene_id \"chr36.g10586\"; transcript_id \"chr36.g10586.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30811969', '30812124', '.', '+', '.', 'gene_id \"chr36.g10635\"; transcript_id \"chr36.g10635.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30898665', '30898978', '.', '-', '.', 'gene_id \"chr36.g10710\"; transcript_id \"chr36.g10710.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30899038', '30899355', '.', '-', '.', 'gene_id \"chr36.g10711\"; transcript_id \"chr36.g10711.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30917314', '30917714', '.', '-', '.', 'gene_id \"chr36.g10726\"; transcript_id \"chr36.g10726.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30917940', '30918541', '.', '+', '.', 'gene_id \"chr36.g10727\"; transcript_id \"chr36.g10727.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '30944786', '30944828', '.', '+', '.', 'gene_id \"chr36.g10740\"; transcript_id \"chr36.g10740.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6808880', '6808949', '.', '+', '.', 'gene_id \"chr32.g2726\"; transcript_id \"chr32.g2726.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '31011362', '31011507', '.', '-', '.', 'gene_id \"chr36.g10777\"; transcript_id \"chr36.g10777.i1\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '17623272', '17623341', '100', '+', '.', 'gene_id \"chr36.g10879\"; transcript_id \"blat.proc4587.chain_11201\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21982770', '21983224', '100', '+', '.', 'gene_id \"chr36.g10923\"; transcript_id \"blat.proc4587.chain_12893\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18102371', '18102410', '100', '-', '.', 'gene_id \"chr36.g10938\"; transcript_id \"blat.proc4587.chain_216\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14276208', '14276264', '100', '+', '.', 'gene_id \"chr36.g11026\"; transcript_id \"blat.proc28144.chain_20401\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19567221', '19567268', '100', '+', '.', 'gene_id \"chr36.g11037\"; transcript_id \"blat.proc4587.chain_12152\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '4710418', '4710580', '100', '+', '.', 'gene_id \"chr36.g11084\"; transcript_id \"blat.proc18128.chain_22160\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '26062561', '26062606', '100', '+', '.', 'gene_id \"chr36.g11165\"; transcript_id \"blat.proc18128.chain_23238\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21103492', '21103538', '100', '-', '.', 'gene_id \"chr36.g11365\"; transcript_id \"blat.proc18128.chain_6711\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '5606217', '5606301', '100', '-', '.', 'gene_id \"chr36.g11458\"; transcript_id \"blat.proc18128.chain_6443\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '2078242', '2078286', '100', '+', '.', 'gene_id \"chr36.g11531\"; transcript_id \"blat.proc18128.chain_25059\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '13306924', '13306976', '100', '+', '.', 'gene_id \"chr36.g11645\"; transcript_id \"blat.proc18128.chain_1036\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10255139', '10255285', '100', '+', '.', 'gene_id \"chr36.g11707\"; transcript_id \"blat.proc18128.chain_20018\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '15303687', '15303938', '100', '-', '.', 'gene_id \"chr36.g11774\"; transcript_id \"blat.proc4587.chain_948\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18482319', '18482409', '100', '-', '.', 'gene_id \"chr36.g11796\"; transcript_id \"blat.proc18128.chain_10080\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '9002706', '9002821', '100', '+', '.', 'gene_id \"chr36.g11813\"; transcript_id \"blat.proc28144.chain_1992\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6230617', '6231184', '100', '+', '.', 'gene_id \"chr36.g12003\"; transcript_id \"blat.proc4587.chain_5142\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '29427199', '29427251', '100', '-', '.', 'gene_id \"chr36.g12020\"; transcript_id \"blat.proc4587.chain_2949\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '20868337', '20868382', '100', '-', '.', 'gene_id \"chr36.g12055\"; transcript_id \"blat.proc4587.chain_2181\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '22046330', '22046365', '100', '-', '.', 'gene_id \"chr36.g12116\"; transcript_id \"blat.proc4005.chain_576\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6080294', '6080327', '100', '+', '.', 'gene_id \"chr36.g12120\"; transcript_id \"blat.proc18128.chain_8586\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '23454333', '23454388', '100', '+', '.', 'gene_id \"chr36.g12237\"; transcript_id \"blat.proc4587.chain_6706\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '19471684', '19471727', '100', '-', '.', 'gene_id \"chr36.g12311\"; transcript_id \"blat.proc4005.chain_10225\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6073584', '6073628', '100', '-', '.', 'gene_id \"chr36.g12354\"; transcript_id \"blat.proc18128.chain_25122\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10414974', '10415044', '100', '-', '.', 'gene_id \"chr36.g12376\"; transcript_id \"blat.proc4005.chain_19418\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '10213788', '10214012', '100', '+', '.', 'gene_id \"chr36.g12484\"; transcript_id \"blat.proc4587.chain_6530\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18551984', '18552064', '100', '+', '.', 'gene_id \"chr36.g12543\"; transcript_id \"blat.proc28144.chain_2942\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6224549', '6224590', '100', '+', '.', 'gene_id \"chr36.g12551\"; transcript_id \"blat.proc4587.chain_3151\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14637017', '14637060', '100', '-', '.', 'gene_id \"chr36.g12636\"; transcript_id \"blat.proc4005.chain_20889\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '9635683', '9635716', '100', '-', '.', 'gene_id \"chr36.g12725\"; transcript_id \"blat.proc4005.chain_9828\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '8721397', '8721620', '100', '+', '.', 'gene_id \"chr36.g12851\"; transcript_id \"blat.proc18128.chain_19918\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '21993962', '21994111', '100', '+', '.', 'gene_id \"chr36.g12871\"; transcript_id \"blat.proc4587.chain_7345\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14951074', '14951107', '100', '+', '.', 'gene_id \"chr36.g13038\"; transcript_id \"blat.proc4005.chain_15927\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '15906504', '15906543', '100', '-', '.', 'gene_id \"chr36.g13114\"; transcript_id \"blat.proc18128.chain_20160\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18172532', '18172586', '100', '-', '.', 'gene_id \"chr36.g13242\"; transcript_id \"blat.proc28144.chain_18292\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '6369016', '6369058', '100', '+', '.', 'gene_id \"chr36.g13246\"; transcript_id \"blat.proc28144.chain_22427\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '8533253', '8533296', '100', '+', '.', 'gene_id \"chr36.g13483\"; transcript_id \"blat.proc28144.chain_20606\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '14602269', '14602316', '100', '+', '.', 'gene_id \"chr36.g13689\"; transcript_id \"blat.proc4005.chain_4084\";']\n",
      "negative:  ['chr36', 'PASA', 'exon', '18585528', '18585560', '100', '+', '.', 'gene_id \"chr36.g13755\"; transcript_id \"blat.proc18128.chain_21778\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6897856', '6898076', '.', '-', '.', 'gene_id \"chr32.g2818\"; transcript_id \"chr32.g2818.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '7080301', '7080516', '.', '-', '.', 'gene_id \"chr32.g2952\"; transcript_id \"chr32.g2952.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '7087917', '7088083', '.', '-', '.', 'gene_id \"chr32.g2960\"; transcript_id \"chr32.g2960.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24371', '24729', '.', '-', '.', 'gene_id \"chr7.g34\"; transcript_id \"chr7.g34.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26683', '26948', '.', '+', '.', 'gene_id \"chr7.g36\"; transcript_id \"chr7.g36.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38395', '38751', '.', '-', '.', 'gene_id \"chr7.g41\"; transcript_id \"chr7.g41.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '7313951', '7313990', '.', '+', '.', 'gene_id \"chr32.g3111\"; transcript_id \"chr32.g3111.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '7795066', '7795108', '.', '-', '.', 'gene_id \"chr32.g3245\"; transcript_id \"chr32.g3245.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '334761', '334806', '.', '-', '.', 'gene_id \"chr7.g302\"; transcript_id \"chr7.g302.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8005159', '8005733', '.', '-', '.', 'gene_id \"chr32.g3256\"; transcript_id \"chr32.g3256.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8005203', '8005733', '.', '-', '.', 'gene_id \"chr32.g3256\"; transcript_id \"chr32.g3256.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '351025', '351085', '.', '+', '.', 'gene_id \"chr7.g338\"; transcript_id \"chr7.g338.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8382995', '8383863', '.', '+', '.', 'gene_id \"chr32.g3298\"; transcript_id \"chr32.g3298.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8382995', '8383858', '.', '+', '.', 'gene_id \"chr32.g3298\"; transcript_id \"chr32.g3298.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8405885', '8406181', '.', '-', '.', 'gene_id \"chr32.g3303\"; transcript_id \"chr32.g3303.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '393155', '393917', '.', '+', '.', 'gene_id \"chr7.g374\"; transcript_id \"chr7.g374.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8489994', '8490329', '.', '+', '.', 'gene_id \"chr32.g3323\"; transcript_id \"chr32.g3323.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '416275', '416546', '.', '+', '.', 'gene_id \"chr7.g390\"; transcript_id \"chr7.g390.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '504954', '505087', '.', '-', '.', 'gene_id \"chr7.g424\"; transcript_id \"chr7.g424.i4\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '621652', '621913', '.', '-', '.', 'gene_id \"chr7.g579\"; transcript_id \"chr7.g579.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '778618', '778789', '.', '-', '.', 'gene_id \"chr7.g683\"; transcript_id \"chr7.g683.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '829227', '829389', '.', '-', '.', 'gene_id \"chr7.g724\"; transcript_id \"chr7.g724.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '829451', '830056', '.', '-', '.', 'gene_id \"chr7.g726\"; transcript_id \"chr7.g726.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '852412', '852702', '.', '+', '.', 'gene_id \"chr7.g741\"; transcript_id \"chr7.g741.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9454334', '9454694', '.', '-', '.', 'gene_id \"chr32.g3669\"; transcript_id \"chr32.g3669.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '905402', '905893', '.', '+', '.', 'gene_id \"chr7.g775\"; transcript_id \"chr7.g775.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '978212', '978356', '.', '-', '.', 'gene_id \"chr7.g802\"; transcript_id \"chr7.g802.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9561239', '9561484', '.', '+', '.', 'gene_id \"chr32.g3744\"; transcript_id \"chr32.g3744.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9612380', '9612675', '.', '+', '.', 'gene_id \"chr32.g3770\"; transcript_id \"chr32.g3770.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9612718', '9612895', '.', '-', '.', 'gene_id \"chr32.g3771\"; transcript_id \"chr32.g3771.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9612913', '9613180', '.', '+', '.', 'gene_id \"chr32.g3772\"; transcript_id \"chr32.g3772.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9904588', '9904632', '.', '+', '.', 'gene_id \"chr32.g3891\"; transcript_id \"chr32.g3891.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10040785', '10040825', '.', '+', '.', 'gene_id \"chr32.g3906\"; transcript_id \"chr32.g3906.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1301685', '1301882', '.', '-', '.', 'gene_id \"chr7.g1099\"; transcript_id \"chr7.g1099.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1318350', '1318588', '.', '-', '.', 'gene_id \"chr7.g1122\"; transcript_id \"chr7.g1122.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10403282', '10403436', '.', '-', '.', 'gene_id \"chr32.g4049\"; transcript_id \"chr32.g4049.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10403491', '10403802', '.', '+', '.', 'gene_id \"chr32.g4050\"; transcript_id \"chr32.g4050.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10403856', '10404235', '.', '+', '.', 'gene_id \"chr32.g4051\"; transcript_id \"chr32.g4051.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1456841', '1458199', '.', '+', '.', 'gene_id \"chr7.g1214\"; transcript_id \"chr7.g1214.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1455959', '1456132', '.', '-', '.', 'gene_id \"chr7.g1216\"; transcript_id \"chr7.g1216.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1456057', '1456727', '.', '-', '.', 'gene_id \"chr7.g1216\"; transcript_id \"chr7.g1216.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10604322', '10604703', '.', '+', '.', 'gene_id \"chr32.g4223\"; transcript_id \"chr32.g4223.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10605659', '10606017', '.', '+', '.', 'gene_id \"chr32.g4226\"; transcript_id \"chr32.g4226.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1911968', '1912140', '.', '-', '.', 'gene_id \"chr7.g1479\"; transcript_id \"chr7.g1479.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10737619', '10737664', '.', '+', '.', 'gene_id \"chr32.g4412\"; transcript_id \"chr32.g4412.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2141306', '2141519', '.', '-', '.', 'gene_id \"chr7.g1630\"; transcript_id \"chr7.g1630.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2150835', '2151061', '.', '-', '.', 'gene_id \"chr7.g1640\"; transcript_id \"chr7.g1640.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2151512', '2151661', '.', '+', '.', 'gene_id \"chr7.g1642\"; transcript_id \"chr7.g1642.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3079293', '3079322', '.', '+', '.', 'gene_id \"chr7.g1907\"; transcript_id \"chr7.g1907.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11472445', '11472511', '.', '+', '.', 'gene_id \"chr32.g4746\"; transcript_id \"chr32.g4746.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3790316', '3790466', '.', '-', '.', 'gene_id \"chr7.g2049\"; transcript_id \"chr7.g2049.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3825116', '3825340', '.', '-', '.', 'gene_id \"chr7.g2069\"; transcript_id \"chr7.g2069.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3955883', '3956307', '.', '+', '.', 'gene_id \"chr7.g2116\"; transcript_id \"chr7.g2116.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4001856', '4001909', '.', '-', '.', 'gene_id \"chr7.g2122\"; transcript_id \"chr7.g2122.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4019903', '4020477', '.', '+', '.', 'gene_id \"chr7.g2130\"; transcript_id \"chr7.g2130.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4020681', '4020839', '.', '+', '.', 'gene_id \"chr7.g2131\"; transcript_id \"chr7.g2131.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11877413', '11877670', '.', '+', '.', 'gene_id \"chr32.g5047\"; transcript_id \"chr32.g5047.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4273196', '4273453', '.', '-', '.', 'gene_id \"chr7.g2210\"; transcript_id \"chr7.g2210.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11944438', '11944587', '.', '+', '.', 'gene_id \"chr32.g5111\"; transcript_id \"chr32.g5111.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4570109', '4570148', '.', '-', '.', 'gene_id \"chr7.g2306\"; transcript_id \"chr7.g2306.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4580202', '4580430', '.', '+', '.', 'gene_id \"chr7.g2308\"; transcript_id \"chr7.g2308.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5206476', '5206749', '.', '+', '.', 'gene_id \"chr7.g2547\"; transcript_id \"chr7.g2547.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '12489102', '12489359', '.', '-', '.', 'gene_id \"chr32.g5468\"; transcript_id \"chr32.g5468.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '12489614', '12490385', '.', '-', '.', 'gene_id \"chr32.g5470\"; transcript_id \"chr32.g5470.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '12548874', '12549042', '.', '+', '.', 'gene_id \"chr32.g5485\"; transcript_id \"chr32.g5485.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5440999', '5441029', '.', '-', '.', 'gene_id \"chr7.g2626\"; transcript_id \"chr7.g2626.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '12599217', '12599299', '.', '-', '.', 'gene_id \"chr32.g5495\"; transcript_id \"chr32.g5495.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '12800491', '12800537', '.', '+', '.', 'gene_id \"chr32.g5567\"; transcript_id \"chr32.g5567.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5700322', '5701021', '.', '-', '.', 'gene_id \"chr7.g2759\"; transcript_id \"chr7.g2759.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5723569', '5723750', '.', '+', '.', 'gene_id \"chr7.g2782\"; transcript_id \"chr7.g2782.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '13455344', '13455396', '.', '+', '.', 'gene_id \"chr32.g5689\"; transcript_id \"chr32.g5689.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5897348', '5897388', '.', '+', '.', 'gene_id \"chr7.g2886\"; transcript_id \"chr7.g2886.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6014233', '6014320', '.', '+', '.', 'gene_id \"chr7.g2932\"; transcript_id \"chr7.g2932.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6088201', '6088241', '.', '-', '.', 'gene_id \"chr7.g2945\"; transcript_id \"chr7.g2945.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6117086', '6117367', '.', '+', '.', 'gene_id \"chr7.g2989\"; transcript_id \"chr7.g2989.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15211878', '15211909', '.', '-', '.', 'gene_id \"chr32.g5914\"; transcript_id \"chr32.g5914.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15294137', '15294169', '.', '-', '.', 'gene_id \"chr32.g5927\"; transcript_id \"chr32.g5927.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15305162', '15305191', '.', '-', '.', 'gene_id \"chr32.g5930\"; transcript_id \"chr32.g5930.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15355409', '15356843', '.', '-', '.', 'gene_id \"chr32.g5934\"; transcript_id \"chr32.g5934.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6336463', '6336785', '.', '-', '.', 'gene_id \"chr7.g3168\"; transcript_id \"chr7.g3168.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6350553', '6354820', '.', '-', '.', 'gene_id \"chr7.g3177\"; transcript_id \"chr7.g3177.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6350553', '6354803', '.', '-', '.', 'gene_id \"chr7.g3177\"; transcript_id \"chr7.g3177.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15611600', '15611768', '.', '+', '.', 'gene_id \"chr32.g5969\"; transcript_id \"chr32.g5969.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15932572', '15933371', '.', '-', '.', 'gene_id \"chr32.g6032\"; transcript_id \"chr32.g6032.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '16773840', '16773902', '.', '-', '.', 'gene_id \"chr32.g6122\"; transcript_id \"chr32.g6122.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17204437', '17204479', '.', '+', '.', 'gene_id \"chr32.g6202\"; transcript_id \"chr32.g6202.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6887508', '6887589', '.', '-', '.', 'gene_id \"chr7.g3432\"; transcript_id \"chr7.g3432.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6947951', '6947990', '.', '+', '.', 'gene_id \"chr7.g3438\"; transcript_id \"chr7.g3438.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17315238', '17315306', '.', '-', '.', 'gene_id \"chr32.g6239\"; transcript_id \"chr32.g6239.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '7231069', '7231101', '.', '-', '.', 'gene_id \"chr7.g3458\"; transcript_id \"chr7.g3458.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17288794', '17288996', '.', '+', '.', 'gene_id \"chr32.g6251\"; transcript_id \"chr32.g6251.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '7699801', '7699839', '.', '-', '.', 'gene_id \"chr7.g3487\"; transcript_id \"chr7.g3487.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '7702786', '7702912', '.', '-', '.', 'gene_id \"chr7.g3487\"; transcript_id \"chr7.g3487.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17501558', '17502195', '.', '-', '.', 'gene_id \"chr32.g6341\"; transcript_id \"chr32.g6341.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17595792', '17596507', '.', '+', '.', 'gene_id \"chr32.g6406\"; transcript_id \"chr32.g6406.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17596637', '17596802', '.', '+', '.', 'gene_id \"chr32.g6407\"; transcript_id \"chr32.g6407.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17596804', '17596958', '.', '+', '.', 'gene_id \"chr32.g6408\"; transcript_id \"chr32.g6408.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17597095', '17598091', '.', '-', '.', 'gene_id \"chr32.g6409\"; transcript_id \"chr32.g6409.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17641740', '17642179', '.', '-', '.', 'gene_id \"chr32.g6447\"; transcript_id \"chr32.g6447.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17642169', '17642869', '.', '-', '.', 'gene_id \"chr32.g6448\"; transcript_id \"chr32.g6448.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8025795', '8026034', '.', '-', '.', 'gene_id \"chr7.g3690\"; transcript_id \"chr7.g3690.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8253253', '8253282', '.', '-', '.', 'gene_id \"chr7.g3802\"; transcript_id \"chr7.g3802.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8478577', '8478956', '.', '+', '.', 'gene_id \"chr7.g3841\"; transcript_id \"chr7.g3841.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8628007', '8628077', '.', '-', '.', 'gene_id \"chr7.g3905\"; transcript_id \"chr7.g3905.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8684842', '8684875', '.', '+', '.', 'gene_id \"chr7.g3915\"; transcript_id \"chr7.g3915.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '18665692', '18665734', '.', '+', '.', 'gene_id \"chr32.g6702\"; transcript_id \"chr32.g6702.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8976169', '8976382', '.', '-', '.', 'gene_id \"chr7.g3982\"; transcript_id \"chr7.g3982.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9223725', '9223784', '.', '-', '.', 'gene_id \"chr7.g4005\"; transcript_id \"chr7.g4005.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '20078631', '20078663', '.', '+', '.', 'gene_id \"chr32.g6762\"; transcript_id \"chr32.g6762.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9401802', '9402221', '.', '+', '.', 'gene_id \"chr7.g4083\"; transcript_id \"chr7.g4083.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9573864', '9573985', '.', '+', '.', 'gene_id \"chr7.g4181\"; transcript_id \"chr7.g4181.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21127755', '21127788', '.', '-', '.', 'gene_id \"chr32.g7031\"; transcript_id \"chr32.g7031.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9825649', '9825844', '.', '-', '.', 'gene_id \"chr7.g4356\"; transcript_id \"chr7.g4356.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21233169', '21233374', '.', '+', '.', 'gene_id \"chr32.g7100\"; transcript_id \"chr32.g7100.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21233523', '21233969', '.', '-', '.', 'gene_id \"chr32.g7102\"; transcript_id \"chr32.g7102.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21237949', '21238120', '.', '+', '.', 'gene_id \"chr32.g7104\"; transcript_id \"chr32.g7104.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21359192', '21359256', '.', '-', '.', 'gene_id \"chr32.g7212\"; transcript_id \"chr32.g7212.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21389473', '21389516', '.', '-', '.', 'gene_id \"chr32.g7238\"; transcript_id \"chr32.g7238.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21396398', '21396562', '.', '+', '.', 'gene_id \"chr32.g7245\"; transcript_id \"chr32.g7245.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21397052', '21397282', '.', '+', '.', 'gene_id \"chr32.g7248\"; transcript_id \"chr32.g7248.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21397384', '21397660', '.', '-', '.', 'gene_id \"chr32.g7250\"; transcript_id \"chr32.g7250.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21877600', '21877947', '.', '-', '.', 'gene_id \"chr32.g7415\"; transcript_id \"chr32.g7415.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10317324', '10317541', '.', '+', '.', 'gene_id \"chr7.g4691\"; transcript_id \"chr7.g4691.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '22184627', '22184847', '.', '+', '.', 'gene_id \"chr32.g7563\"; transcript_id \"chr32.g7563.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '22184848', '22185079', '.', '+', '.', 'gene_id \"chr32.g7564\"; transcript_id \"chr32.g7564.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10570764', '10570880', '.', '+', '.', 'gene_id \"chr7.g4854\"; transcript_id \"chr7.g4854.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '22506667', '22507059', '.', '-', '.', 'gene_id \"chr32.g7708\"; transcript_id \"chr32.g7708.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10835035', '10835236', '.', '+', '.', 'gene_id \"chr7.g5054\"; transcript_id \"chr7.g5054.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '22908689', '22908848', '.', '+', '.', 'gene_id \"chr32.g7774\"; transcript_id \"chr32.g7774.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10860987', '10861186', '.', '-', '.', 'gene_id \"chr7.g5084\"; transcript_id \"chr7.g5084.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10861253', '10861623', '.', '+', '.', 'gene_id \"chr7.g5086\"; transcript_id \"chr7.g5086.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10883701', '10884047', '.', '+', '.', 'gene_id \"chr7.g5112\"; transcript_id \"chr7.g5112.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10945593', '10945748', '.', '-', '.', 'gene_id \"chr7.g5182\"; transcript_id \"chr7.g5182.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10953931', '10954487', '.', '-', '.', 'gene_id \"chr7.g5209\"; transcript_id \"chr7.g5209.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '11396756', '11396837', '.', '+', '.', 'gene_id \"chr7.g5236\"; transcript_id \"chr7.g5236.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '23265651', '23265683', '.', '-', '.', 'gene_id \"chr32.g8031\"; transcript_id \"chr32.g8031.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '23484467', '23484734', '.', '+', '.', 'gene_id \"chr32.g8047\"; transcript_id \"chr32.g8047.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '11946250', '11946371', '.', '+', '.', 'gene_id \"chr7.g5450\"; transcript_id \"chr7.g5450.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '11985217', '11985442', '.', '-', '.', 'gene_id \"chr7.g5485\"; transcript_id \"chr7.g5485.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '11985654', '11985825', '.', '+', '.', 'gene_id \"chr7.g5486\"; transcript_id \"chr7.g5486.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12085140', '12085172', '.', '+', '.', 'gene_id \"chr7.g5570\"; transcript_id \"chr7.g5570.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24171856', '24171890', '.', '-', '.', 'gene_id \"chr32.g8265\"; transcript_id \"chr32.g8265.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12200453', '12200531', '.', '+', '.', 'gene_id \"chr7.g5647\"; transcript_id \"chr7.g5647.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12415223', '12415288', '.', '-', '.', 'gene_id \"chr7.g5665\"; transcript_id \"chr7.g5665.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12652129', '12652170', '.', '-', '.', 'gene_id \"chr7.g5692\"; transcript_id \"chr7.g5692.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24312266', '24312298', '.', '-', '.', 'gene_id \"chr32.g8385\"; transcript_id \"chr32.g8385.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12957783', '12957981', '.', '+', '.', 'gene_id \"chr7.g5725\"; transcript_id \"chr7.g5725.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13073894', '13074174', '.', '+', '.', 'gene_id \"chr7.g5757\"; transcript_id \"chr7.g5757.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24449734', '24450361', '.', '-', '.', 'gene_id \"chr32.g8445\"; transcript_id \"chr32.g8445.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24473588', '24473669', '.', '-', '.', 'gene_id \"chr32.g8469\"; transcript_id \"chr32.g8469.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13117562', '13117868', '.', '+', '.', 'gene_id \"chr7.g5804\"; transcript_id \"chr7.g5804.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24562009', '24562457', '.', '-', '.', 'gene_id \"chr32.g8564\"; transcript_id \"chr32.g8564.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13265878', '13266051', '.', '+', '.', 'gene_id \"chr7.g5902\"; transcript_id \"chr7.g5902.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13266041', '13266228', '.', '-', '.', 'gene_id \"chr7.g5903\"; transcript_id \"chr7.g5903.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13296069', '13296112', '.', '+', '.', 'gene_id \"chr7.g5928\"; transcript_id \"chr7.g5928.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24817259', '24817320', '.', '-', '.', 'gene_id \"chr32.g8660\"; transcript_id \"chr32.g8660.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24823857', '24823931', '.', '+', '.', 'gene_id \"chr32.g8663\"; transcript_id \"chr32.g8663.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24836028', '24836557', '.', '+', '.', 'gene_id \"chr32.g8664\"; transcript_id \"chr32.g8664.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13486389', '13486597', '.', '-', '.', 'gene_id \"chr7.g6037\"; transcript_id \"chr7.g6037.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '25426438', '25426499', '.', '-', '.', 'gene_id \"chr32.g8733\"; transcript_id \"chr32.g8733.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '25711394', '25711653', '.', '-', '.', 'gene_id \"chr32.g8771\"; transcript_id \"chr32.g8771.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13537762', '13537819', '.', '+', '.', 'gene_id \"chr7.g6112\"; transcript_id \"chr7.g6112.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '25769052', '25769090', '.', '+', '.', 'gene_id \"chr32.g8802\"; transcript_id \"chr32.g8802.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13570174', '13570333', '.', '-', '.', 'gene_id \"chr7.g6142\"; transcript_id \"chr7.g6142.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26047511', '26047573', '.', '-', '.', 'gene_id \"chr32.g8832\"; transcript_id \"chr32.g8832.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26085468', '26085631', '.', '-', '.', 'gene_id \"chr32.g8853\"; transcript_id \"chr32.g8853.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26127595', '26127895', '.', '-', '.', 'gene_id \"chr32.g8898\"; transcript_id \"chr32.g8898.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13762236', '13762286', '.', '+', '.', 'gene_id \"chr7.g6229\"; transcript_id \"chr7.g6229.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26150654', '26151310', '.', '-', '.', 'gene_id \"chr32.g8939\"; transcript_id \"chr32.g8939.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14055261', '14055461', '.', '+', '.', 'gene_id \"chr7.g6342\"; transcript_id \"chr7.g6342.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14055816', '14056491', '.', '+', '.', 'gene_id \"chr7.g6344\"; transcript_id \"chr7.g6344.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14120385', '14120414', '.', '+', '.', 'gene_id \"chr7.g6411\"; transcript_id \"chr7.g6411.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26354925', '26355076', '.', '-', '.', 'gene_id \"chr32.g9099\"; transcript_id \"chr32.g9099.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14181450', '14181480', '.', '+', '.', 'gene_id \"chr7.g6430\"; transcript_id \"chr7.g6430.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26365656', '26366003', '.', '+', '.', 'gene_id \"chr32.g9122\"; transcript_id \"chr32.g9122.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14254678', '14254718', '.', '-', '.', 'gene_id \"chr7.g6459\"; transcript_id \"chr7.g6459.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14278128', '14278191', '.', '+', '.', 'gene_id \"chr7.g6464\"; transcript_id \"chr7.g6464.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14360104', '14360171', '.', '+', '.', 'gene_id \"chr7.g6473\"; transcript_id \"chr7.g6473.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26479301', '26479413', '.', '-', '.', 'gene_id \"chr32.g9168\"; transcript_id \"chr32.g9168.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14525015', '14525070', '.', '+', '.', 'gene_id \"chr7.g6488\"; transcript_id \"chr7.g6488.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26488041', '26488302', '.', '+', '.', 'gene_id \"chr32.g9176\"; transcript_id \"chr32.g9176.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14729628', '14729662', '.', '-', '.', 'gene_id \"chr7.g6520\"; transcript_id \"chr7.g6520.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26602945', '26603122', '.', '+', '.', 'gene_id \"chr32.g9223\"; transcript_id \"chr32.g9223.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15024108', '15024157', '.', '-', '.', 'gene_id \"chr7.g6579\"; transcript_id \"chr7.g6579.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15103871', '15104654', '.', '+', '.', 'gene_id \"chr7.g6587\"; transcript_id \"chr7.g6587.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15181002', '15181144', '.', '+', '.', 'gene_id \"chr7.g6595\"; transcript_id \"chr7.g6595.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15337859', '15337908', '.', '+', '.', 'gene_id \"chr7.g6609\"; transcript_id \"chr7.g6609.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15349296', '15349336', '.', '+', '.', 'gene_id \"chr7.g6611\"; transcript_id \"chr7.g6611.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15382584', '15382774', '.', '-', '.', 'gene_id \"chr7.g6633\"; transcript_id \"chr7.g6633.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15590616', '15590985', '.', '+', '.', 'gene_id \"chr7.g6676\"; transcript_id \"chr7.g6676.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15590634', '15590863', '.', '-', '.', 'gene_id \"chr7.g6677\"; transcript_id \"chr7.g6677.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15622892', '15622976', '.', '+', '.', 'gene_id \"chr7.g6700\"; transcript_id \"chr7.g6700.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26925972', '26926013', '.', '-', '.', 'gene_id \"chr32.g9369\"; transcript_id \"chr32.g9369.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27082480', '27082783', '.', '+', '.', 'gene_id \"chr32.g9483\"; transcript_id \"chr32.g9483.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '15943482', '15943663', '.', '-', '.', 'gene_id \"chr7.g6859\"; transcript_id \"chr7.g6859.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27137600', '27137781', '.', '-', '.', 'gene_id \"chr32.g9531\"; transcript_id \"chr32.g9531.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27138564', '27138770', '.', '-', '.', 'gene_id \"chr32.g9534\"; transcript_id \"chr32.g9534.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27335524', '27335754', '.', '+', '.', 'gene_id \"chr32.g9634\"; transcript_id \"chr32.g9634.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27815442', '27815634', '.', '+', '.', 'gene_id \"chr32.g9689\"; transcript_id \"chr32.g9689.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16461831', '16462058', '.', '-', '.', 'gene_id \"chr7.g7104\"; transcript_id \"chr7.g7104.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16541951', '16542468', '.', '+', '.', 'gene_id \"chr7.g7145\"; transcript_id \"chr7.g7145.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28358735', '28358782', '.', '-', '.', 'gene_id \"chr32.g9755\"; transcript_id \"chr32.g9755.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28359005', '28359327', '.', '+', '.', 'gene_id \"chr32.g9756\"; transcript_id \"chr32.g9756.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16756450', '16756631', '.', '+', '.', 'gene_id \"chr7.g7212\"; transcript_id \"chr7.g7212.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28556683', '28556956', '.', '-', '.', 'gene_id \"chr32.g9875\"; transcript_id \"chr32.g9875.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28564754', '28564791', '.', '+', '.', 'gene_id \"chr32.g9886\"; transcript_id \"chr32.g9886.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16947942', '16948116', '.', '+', '.', 'gene_id \"chr7.g7339\"; transcript_id \"chr7.g7339.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28602663', '28602977', '.', '+', '.', 'gene_id \"chr32.g9923\"; transcript_id \"chr32.g9923.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28602978', '28603134', '.', '+', '.', 'gene_id \"chr32.g9924\"; transcript_id \"chr32.g9924.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16998244', '16998472', '.', '-', '.', 'gene_id \"chr7.g7355\"; transcript_id \"chr7.g7355.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17000131', '17000162', '.', '+', '.', 'gene_id \"chr7.g7364\"; transcript_id \"chr7.g7364.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17089345', '17089600', '.', '-', '.', 'gene_id \"chr7.g7408\"; transcript_id \"chr7.g7408.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17089750', '17089984', '.', '-', '.', 'gene_id \"chr7.g7410\"; transcript_id \"chr7.g7410.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17118933', '17119027', '.', '-', '.', 'gene_id \"chr7.g7452\"; transcript_id \"chr7.g7452.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17173640', '17173675', '.', '+', '.', 'gene_id \"chr7.g7487\"; transcript_id \"chr7.g7487.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17218437', '17218685', '.', '+', '.', 'gene_id \"chr7.g7506\"; transcript_id \"chr7.g7506.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28845007', '28845142', '.', '+', '.', 'gene_id \"chr32.g10129\"; transcript_id \"chr32.g10129.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17318305', '17318760', '.', '-', '.', 'gene_id \"chr7.g7543\"; transcript_id \"chr7.g7543.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29049333', '29049372', '.', '-', '.', 'gene_id \"chr32.g10162\"; transcript_id \"chr32.g10162.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29129199', '29129360', '.', '+', '.', 'gene_id \"chr32.g10167\"; transcript_id \"chr32.g10167.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17472646', '17472902', '.', '-', '.', 'gene_id \"chr7.g7648\"; transcript_id \"chr7.g7648.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29929521', '29929758', '.', '+', '.', 'gene_id \"chr32.g10387\"; transcript_id \"chr32.g10387.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29983169', '29983221', '.', '+', '.', 'gene_id \"chr32.g10437\"; transcript_id \"chr32.g10437.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17788577', '17788633', '.', '+', '.', 'gene_id \"chr7.g7863\"; transcript_id \"chr7.g7863.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '30067569', '30067602', '.', '-', '.', 'gene_id \"chr32.g10484\"; transcript_id \"chr32.g10484.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17806757', '17806802', '.', '+', '.', 'gene_id \"chr7.g7891\"; transcript_id \"chr7.g7891.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17806832', '17808222', '.', '-', '.', 'gene_id \"chr7.g7893\"; transcript_id \"chr7.g7893.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '30178238', '30178799', '.', '+', '.', 'gene_id \"chr32.g10558\"; transcript_id \"chr32.g10558.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17913684', '17913927', '.', '+', '.', 'gene_id \"chr7.g7957\"; transcript_id \"chr7.g7957.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '18004932', '18004973', '.', '+', '.', 'gene_id \"chr7.g8048\"; transcript_id \"chr7.g8048.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18344610', '18344640', '.', '+', '.', 'gene_id \"chr7.g8185\"; transcript_id \"chr7.g8185.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18414934', '18415121', '.', '+', '.', 'gene_id \"chr7.g8189\"; transcript_id \"chr7.g8189.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18453447', '18453486', '.', '+', '.', 'gene_id \"chr7.g8205\"; transcript_id \"chr7.g8205.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18764632', '18764689', '.', '-', '.', 'gene_id \"chr7.g8255\"; transcript_id \"chr7.g8255.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18893551', '18893738', '.', '+', '.', 'gene_id \"chr7.g8285\"; transcript_id \"chr7.g8285.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18972716', '18972860', '.', '-', '.', 'gene_id \"chr7.g8292\"; transcript_id \"chr7.g8292.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19029522', '19029643', '.', '+', '.', 'gene_id \"chr7.g8325\"; transcript_id \"chr7.g8325.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19216427', '19216507', '.', '+', '.', 'gene_id \"chr7.g8405\"; transcript_id \"chr7.g8405.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19218628', '19218685', '.', '+', '.', 'gene_id \"chr7.g8409\"; transcript_id \"chr7.g8409.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '31986296', '31986485', '.', '-', '.', 'gene_id \"chr32.g11026\"; transcript_id \"chr32.g11026.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19284931', '19285075', '.', '+', '.', 'gene_id \"chr7.g8421\"; transcript_id \"chr7.g8421.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19541464', '19541620', '.', '-', '.', 'gene_id \"chr7.g8496\"; transcript_id \"chr7.g8496.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32223768', '32223994', '.', '-', '.', 'gene_id \"chr32.g11114\"; transcript_id \"chr32.g11114.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32224058', '32224669', '.', '+', '.', 'gene_id \"chr32.g11115\"; transcript_id \"chr32.g11115.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '19612702', '19613268', '.', '-', '.', 'gene_id \"chr7.g8514\"; transcript_id \"chr7.g8514.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32283104', '32283262', '.', '+', '.', 'gene_id \"chr32.g11168\"; transcript_id \"chr32.g11168.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32283623', '32284234', '.', '-', '.', 'gene_id \"chr32.g11169\"; transcript_id \"chr32.g11169.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32285113', '32285327', '.', '+', '.', 'gene_id \"chr32.g11171\"; transcript_id \"chr32.g11171.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32319014', '32319238', '.', '-', '.', 'gene_id \"chr32.g11181\"; transcript_id \"chr32.g11181.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20200321', '20200511', '.', '+', '.', 'gene_id \"chr7.g8711\"; transcript_id \"chr7.g8711.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32647581', '32647868', '.', '-', '.', 'gene_id \"chr32.g11406\"; transcript_id \"chr32.g11406.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20538088', '20538196', '.', '-', '.', 'gene_id \"chr7.g8893\"; transcript_id \"chr7.g8893.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '33018756', '33018811', '.', '+', '.', 'gene_id \"chr32.g11580\"; transcript_id \"chr32.g11580.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '33074629', '33074935', '.', '+', '.', 'gene_id \"chr32.g11606\"; transcript_id \"chr32.g11606.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20939293', '20939336', '.', '-', '.', 'gene_id \"chr7.g9098\"; transcript_id \"chr7.g9098.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20946520', '20946567', '.', '-', '.', 'gene_id \"chr7.g9110\"; transcript_id \"chr7.g9110.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20955930', '20956159', '.', '-', '.', 'gene_id \"chr7.g9119\"; transcript_id \"chr7.g9119.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20956260', '20956413', '.', '-', '.', 'gene_id \"chr7.g9120\"; transcript_id \"chr7.g9120.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21214141', '21214203', '.', '+', '.', 'gene_id \"chr7.g9283\"; transcript_id \"chr7.g9283.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21242054', '21242337', '.', '-', '.', 'gene_id \"chr7.g9314\"; transcript_id \"chr7.g9314.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21242326', '21242551', '.', '+', '.', 'gene_id \"chr7.g9315\"; transcript_id \"chr7.g9315.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21242555', '21242806', '.', '-', '.', 'gene_id \"chr7.g9316\"; transcript_id \"chr7.g9316.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21327415', '21327462', '.', '-', '.', 'gene_id \"chr7.g9340\"; transcript_id \"chr7.g9340.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21438297', '21438397', '.', '-', '.', 'gene_id \"chr7.g9372\"; transcript_id \"chr7.g9372.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21438297', '21438357', '.', '-', '.', 'gene_id \"chr7.g9372\"; transcript_id \"chr7.g9372.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21452832', '21452905', '.', '-', '.', 'gene_id \"chr7.g9377\"; transcript_id \"chr7.g9377.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21452830', '21452905', '.', '-', '.', 'gene_id \"chr7.g9377\"; transcript_id \"chr7.g9377.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21496816', '21497002', '.', '-', '.', 'gene_id \"chr7.g9385\"; transcript_id \"chr7.g9385.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21538186', '21538338', '.', '-', '.', 'gene_id \"chr7.g9390\"; transcript_id \"chr7.g9390.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21572350', '21572470', '.', '-', '.', 'gene_id \"chr7.g9398\"; transcript_id \"chr7.g9398.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21615487', '21615546', '.', '+', '.', 'gene_id \"chr7.g9410\"; transcript_id \"chr7.g9410.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21979692', '21980627', '.', '-', '.', 'gene_id \"chr7.g9456\"; transcript_id \"chr7.g9456.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21981483', '21981853', '.', '-', '.', 'gene_id \"chr7.g9457\"; transcript_id \"chr7.g9457.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22074182', '22074238', '.', '+', '.', 'gene_id \"chr7.g9486\"; transcript_id \"chr7.g9486.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '35214620', '35214649', '.', '-', '.', 'gene_id \"chr32.g12163\"; transcript_id \"chr32.g12163.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22578244', '22578289', '.', '-', '.', 'gene_id \"chr7.g9561\"; transcript_id \"chr7.g9561.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22816077', '22816273', '.', '+', '.', 'gene_id \"chr7.g9616\"; transcript_id \"chr7.g9616.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22967891', '22967933', '.', '-', '.', 'gene_id \"chr7.g9652\"; transcript_id \"chr7.g9652.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23046696', '23046898', '.', '-', '.', 'gene_id \"chr7.g9697\"; transcript_id \"chr7.g9697.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23046880', '23047075', '.', '+', '.', 'gene_id \"chr7.g9698\"; transcript_id \"chr7.g9698.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '36887668', '36887845', '.', '+', '.', 'gene_id \"chr32.g12308\"; transcript_id \"chr32.g12308.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23151898', '23152074', '.', '+', '.', 'gene_id \"chr7.g9770\"; transcript_id \"chr7.g9770.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '37586659', '37586692', '.', '-', '.', 'gene_id \"chr32.g12372\"; transcript_id \"chr32.g12372.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '37684772', '37684939', '.', '+', '.', 'gene_id \"chr32.g12402\"; transcript_id \"chr32.g12402.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23732274', '23732305', '.', '-', '.', 'gene_id \"chr7.g9906\"; transcript_id \"chr7.g9906.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23880941', '23881092', '.', '+', '.', 'gene_id \"chr7.g9934\"; transcript_id \"chr7.g9934.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24000725', '24000768', '.', '-', '.', 'gene_id \"chr7.g10001\"; transcript_id \"chr7.g10001.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38039694', '38039750', '.', '+', '.', 'gene_id \"chr32.g12658\"; transcript_id \"chr32.g12658.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24093321', '24093381', '.', '+', '.', 'gene_id \"chr7.g10101\"; transcript_id \"chr7.g10101.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38061123', '38061189', '.', '+', '.', 'gene_id \"chr32.g12672\"; transcript_id \"chr32.g12672.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38076420', '38076617', '.', '+', '.', 'gene_id \"chr32.g12681\"; transcript_id \"chr32.g12681.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24178715', '24179120', '.', '+', '.', 'gene_id \"chr7.g10173\"; transcript_id \"chr7.g10173.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38356115', '38356267', '.', '-', '.', 'gene_id \"chr32.g12791\"; transcript_id \"chr32.g12791.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38356294', '38356458', '.', '+', '.', 'gene_id \"chr32.g12792\"; transcript_id \"chr32.g12792.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24310513', '24310693', '.', '-', '.', 'gene_id \"chr7.g10276\"; transcript_id \"chr7.g10276.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38412744', '38412899', '.', '-', '.', 'gene_id \"chr32.g12866\"; transcript_id \"chr32.g12866.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24709932', '24710209', '.', '+', '.', 'gene_id \"chr7.g10525\"; transcript_id \"chr7.g10525.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24718425', '24718939', '.', '+', '.', 'gene_id \"chr7.g10527\"; transcript_id \"chr7.g10527.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38724318', '38724654', '.', '+', '.', 'gene_id \"chr32.g13089\"; transcript_id \"chr32.g13089.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38724409', '38724604', '.', '+', '.', 'gene_id \"chr32.g13089\"; transcript_id \"chr32.g13089.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38875625', '38875808', '.', '-', '.', 'gene_id \"chr32.g13115\"; transcript_id \"chr32.g13115.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38944814', '38945009', '.', '-', '.', 'gene_id \"chr32.g13156\"; transcript_id \"chr32.g13156.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38945013', '38945439', '.', '+', '.', 'gene_id \"chr32.g13157\"; transcript_id \"chr32.g13157.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38950593', '38950847', '.', '+', '.', 'gene_id \"chr32.g13160\"; transcript_id \"chr32.g13160.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38957869', '38958183', '.', '+', '.', 'gene_id \"chr32.g13168\"; transcript_id \"chr32.g13168.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38962333', '38962643', '.', '-', '.', 'gene_id \"chr32.g13170\"; transcript_id \"chr32.g13170.i2\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38962638', '38962797', '.', '+', '.', 'gene_id \"chr32.g13171\"; transcript_id \"chr32.g13171.i1\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '22333091', '22333225', '100', '+', '.', 'gene_id \"chr32.g13279\"; transcript_id \"blat.proc4587.chain_1112\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '2992043', '2992072', '100', '-', '.', 'gene_id \"chr32.g13348\"; transcript_id \"blat.proc18128.chain_14584\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32653910', '32653972', '100', '+', '.', 'gene_id \"chr32.g13356\"; transcript_id \"blat.proc4005.chain_78\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '34321492', '34321535', '100', '+', '.', 'gene_id \"chr32.g13447\"; transcript_id \"blat.proc18128.chain_2286\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38902282', '38902519', '100', '+', '.', 'gene_id \"chr32.g13518\"; transcript_id \"blat.proc4587.chain_6866\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '38957462', '38957670', '100', '+', '.', 'gene_id \"chr32.g13547\"; transcript_id \"blat.proc18128.chain_8114\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32946918', '32947082', '100', '+', '.', 'gene_id \"chr32.g13580\"; transcript_id \"blat.proc18128.chain_10920\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '19353969', '19354048', '100', '+', '.', 'gene_id \"chr32.g13589\"; transcript_id \"blat.proc18128.chain_90\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8965878', '8965956', '100', '+', '.', 'gene_id \"chr32.g13702\"; transcript_id \"blat.proc28144.chain_10477\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10406058', '10406216', '100', '+', '.', 'gene_id \"chr32.g13741\"; transcript_id \"blat.proc4587.chain_701\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '5048253', '5048295', '100', '+', '.', 'gene_id \"chr32.g13847\"; transcript_id \"blat.proc4587.chain_5947\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3368201', '3368367', '100', '-', '.', 'gene_id \"chr32.g13892\"; transcript_id \"blat.proc4587.chain_8129\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '14028285', '14028488', '100', '+', '.', 'gene_id \"chr32.g13961\"; transcript_id \"blat.proc28144.chain_3603\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21155777', '21155819', '100', '-', '.', 'gene_id \"chr32.g13990\"; transcript_id \"blat.proc18128.chain_3739\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32894554', '32894624', '100', '-', '.', 'gene_id \"chr32.g14022\"; transcript_id \"blat.proc18128.chain_18683\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '13121972', '13122021', '100', '-', '.', 'gene_id \"chr32.g14152\"; transcript_id \"blat.proc28144.chain_1440\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26958596', '26958717', '100', '-', '.', 'gene_id \"chr32.g14166\"; transcript_id \"blat.proc18128.chain_9719\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '35308419', '35308451', '100', '-', '.', 'gene_id \"chr32.g14260\"; transcript_id \"blat.proc4587.chain_2585\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '20197250', '20197285', '100', '-', '.', 'gene_id \"chr32.g14330\"; transcript_id \"blat.proc28144.chain_2199\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '17736722', '17736892', '100', '-', '.', 'gene_id \"chr32.g14440\"; transcript_id \"blat.proc18128.chain_18458\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '9851129', '9851196', '100', '-', '.', 'gene_id \"chr32.g14453\"; transcript_id \"blat.proc18128.chain_20206\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3334871', '3335033', '100', '+', '.', 'gene_id \"chr32.g14469\"; transcript_id \"blat.proc4587.chain_6278\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24562793', '24563030', '100', '+', '.', 'gene_id \"chr32.g14517\"; transcript_id \"blat.proc4587.chain_6865\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29070410', '29070684', '100', '+', '.', 'gene_id \"chr32.g14686\"; transcript_id \"blat.proc28144.chain_16908\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32508434', '32508466', '100', '+', '.', 'gene_id \"chr32.g14798\"; transcript_id \"blat.proc18128.chain_2006\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3367148', '3367335', '100', '+', '.', 'gene_id \"chr32.g14819\"; transcript_id \"blat.proc4587.chain_8123\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1558173', '1558222', '100', '-', '.', 'gene_id \"chr32.g14980\"; transcript_id \"blat.proc28144.chain_14078\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '25132294', '25132327', '100', '-', '.', 'gene_id \"chr32.g15144\"; transcript_id \"blat.proc28144.chain_11979\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32962587', '32962621', '100', '-', '.', 'gene_id \"chr32.g15172\"; transcript_id \"blat.proc4587.chain_11779\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29036525', '29036558', '100', '-', '.', 'gene_id \"chr32.g15492\"; transcript_id \"blat.proc4005.chain_15570\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32892485', '32892516', '100', '-', '.', 'gene_id \"chr32.g15495\"; transcript_id \"blat.proc4005.chain_19843\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '40953', '41094', '100', '+', '.', 'gene_id \"chr32.g15560\"; transcript_id \"blat.proc18128.chain_9514\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '15515529', '15515577', '100', '-', '.', 'gene_id \"chr32.g15578\"; transcript_id \"blat.proc28144.chain_11859\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10614469', '10614523', '100', '+', '.', 'gene_id \"chr32.g15595\"; transcript_id \"blat.proc4005.chain_20415\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26143350', '26143384', '100', '+', '.', 'gene_id \"chr32.g15689\"; transcript_id \"blat.proc28144.chain_990\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11984776', '11984810', '100', '+', '.', 'gene_id \"chr32.g15773\"; transcript_id \"blat.proc4005.chain_19341\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '6433686', '6433746', '100', '-', '.', 'gene_id \"chr32.g15790\"; transcript_id \"blat.proc18128.chain_24207\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24406916', '24406956', '100', '+', '.', 'gene_id \"chr32.g15833\"; transcript_id \"blat.proc4587.chain_2137\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '13955542', '13955578', '100', '-', '.', 'gene_id \"chr32.g15852\"; transcript_id \"blat.proc4005.chain_2449\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '29259219', '29259378', '100', '+', '.', 'gene_id \"chr32.g16394\"; transcript_id \"blat.proc18128.chain_17193\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3324183', '3324370', '100', '+', '.', 'gene_id \"chr32.g16468\"; transcript_id \"blat.proc4587.chain_8124\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '10848117', '10848158', '100', '+', '.', 'gene_id \"chr32.g16602\"; transcript_id \"blat.proc4005.chain_14707\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '27741572', '27741601', '100', '-', '.', 'gene_id \"chr32.g16704\"; transcript_id \"blat.proc4005.chain_1606\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11199941', '11200277', '100', '-', '.', 'gene_id \"chr32.g16767\"; transcript_id \"blat.proc18128.chain_17473\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24982290', '24982327', '100', '-', '.', 'gene_id \"chr32.g16878\"; transcript_id \"blat.proc28144.chain_5568\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '19599371', '19599870', '100', '-', '.', 'gene_id \"chr32.g16903\"; transcript_id \"blat.proc28144.chain_2839\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '13536986', '13537094', '100', '-', '.', 'gene_id \"chr32.g16951\"; transcript_id \"blat.proc18128.chain_23198\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3371716', '3371937', '100', '+', '.', 'gene_id \"chr32.g17012\"; transcript_id \"blat.proc18128.chain_11084\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3378220', '3378382', '100', '+', '.', 'gene_id \"chr32.g17096\"; transcript_id \"blat.proc4587.chain_6277\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26978426', '26978486', '100', '-', '.', 'gene_id \"chr32.g17143\"; transcript_id \"blat.proc4587.chain_8419\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26953321', '26953366', '100', '+', '.', 'gene_id \"chr32.g17163\"; transcript_id \"blat.proc18128.chain_15396\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '21767051', '21767199', '100', '-', '.', 'gene_id \"chr32.g17194\"; transcript_id \"blat.proc18128.chain_12284\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '1925110', '1925156', '100', '+', '.', 'gene_id \"chr32.g17229\"; transcript_id \"blat.proc18128.chain_22415\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '30661150', '30661193', '100', '+', '.', 'gene_id \"chr32.g17238\"; transcript_id \"blat.proc18128.chain_17372\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '20130262', '20130291', '100', '+', '.', 'gene_id \"chr32.g17294\"; transcript_id \"blat.proc28144.chain_566\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '3367435', '3367818', '100', '-', '.', 'gene_id \"chr32.g17302\"; transcript_id \"blat.proc4587.chain_8127\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '8476994', '8477091', '100', '-', '.', 'gene_id \"chr32.g17318\"; transcript_id \"blat.proc18128.chain_15666\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '35628448', '35628485', '100', '+', '.', 'gene_id \"chr32.g17347\"; transcript_id \"blat.proc18128.chain_26733\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '24383630', '24383675', '100', '+', '.', 'gene_id \"chr32.g17350\"; transcript_id \"blat.proc18128.chain_14824\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '28812565', '28812608', '100', '+', '.', 'gene_id \"chr32.g17376\"; transcript_id \"blat.proc18128.chain_19434\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '11938992', '11939042', '100', '-', '.', 'gene_id \"chr32.g17384\"; transcript_id \"blat.proc28144.chain_3209\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '26812105', '26812153', '100', '-', '.', 'gene_id \"chr32.g17453\"; transcript_id \"blat.proc18128.chain_14098\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '20345534', '20345568', '100', '+', '.', 'gene_id \"chr32.g17457\"; transcript_id \"blat.proc18128.chain_20766\";']\n",
      "negative:  ['chr32', 'PASA', 'exon', '32675819', '32675852', '100', '-', '.', 'gene_id \"chr32.g17471\"; transcript_id \"blat.proc18128.chain_16730\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25017520', '25017836', '.', '+', '.', 'gene_id \"chr7.g10679\"; transcript_id \"chr7.g10679.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25030402', '25030734', '.', '-', '.', 'gene_id \"chr7.g10683\"; transcript_id \"chr7.g10683.i9\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25031896', '25032465', '.', '-', '.', 'gene_id \"chr7.g10683\"; transcript_id \"chr7.g10683.i9\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25019933', '25020223', '.', '+', '.', 'gene_id \"chr7.g10684\"; transcript_id \"chr7.g10684.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25029590', '25029746', '.', '+', '.', 'gene_id \"chr7.g10685\"; transcript_id \"chr7.g10685.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25029201', '25029460', '.', '+', '.', 'gene_id \"chr7.g10686\"; transcript_id \"chr7.g10686.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25027830', '25027990', '.', '+', '.', 'gene_id \"chr7.g10691\"; transcript_id \"chr7.g10691.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25046930', '25047779', '.', '+', '.', 'gene_id \"chr7.g10704\"; transcript_id \"chr7.g10704.i2\";']\n",
      "negative:  ['scaffold_956', 'PASA', 'exon', '4989', '5187', '.', '-', '.', 'gene_id \"scaffold_956.g4\"; transcript_id \"scaffold_956.g4.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25254502', '25254581', '.', '+', '.', 'gene_id \"chr7.g10868\"; transcript_id \"chr7.g10868.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25254503', '25254581', '.', '+', '.', 'gene_id \"chr7.g10868\"; transcript_id \"chr7.g10868.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25349767', '25349810', '.', '+', '.', 'gene_id \"chr7.g10901\"; transcript_id \"chr7.g10901.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25361160', '25361730', '.', '-', '.', 'gene_id \"chr7.g10907\"; transcript_id \"chr7.g10907.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25452563', '25453766', '.', '+', '.', 'gene_id \"chr7.g10924\"; transcript_id \"chr7.g10924.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '130619', '130781', '.', '-', '.', 'gene_id \"chr6.g72\"; transcript_id \"chr6.g72.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25662699', '25662864', '.', '+', '.', 'gene_id \"chr7.g10955\"; transcript_id \"chr7.g10955.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25662856', '25663077', '.', '-', '.', 'gene_id \"chr7.g10956\"; transcript_id \"chr7.g10956.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '136582', '136646', '.', '+', '.', 'gene_id \"chr6.g73\"; transcript_id \"chr6.g73.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25981660', '25981711', '.', '-', '.', 'gene_id \"chr7.g10997\"; transcript_id \"chr7.g10997.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26098338', '26098509', '.', '+', '.', 'gene_id \"chr7.g11059\"; transcript_id \"chr7.g11059.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26098490', '26099208', '.', '-', '.', 'gene_id \"chr7.g11060\"; transcript_id \"chr7.g11060.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '426352', '426514', '.', '-', '.', 'gene_id \"chr6.g336\"; transcript_id \"chr6.g336.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '428763', '428816', '.', '+', '.', 'gene_id \"chr6.g342\"; transcript_id \"chr6.g342.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '430104', '430155', '.', '+', '.', 'gene_id \"chr6.g345\"; transcript_id \"chr6.g345.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26401328', '26401359', '.', '-', '.', 'gene_id \"chr7.g11265\"; transcript_id \"chr7.g11265.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26428400', '26428452', '.', '-', '.', 'gene_id \"chr7.g11268\"; transcript_id \"chr7.g11268.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26626630', '26626787', '.', '-', '.', 'gene_id \"chr7.g11313\"; transcript_id \"chr7.g11313.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '596076', '596146', '.', '-', '.', 'gene_id \"chr6.g543\"; transcript_id \"chr6.g543.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26918593', '26919652', '.', '+', '.', 'gene_id \"chr7.g11408\"; transcript_id \"chr7.g11408.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27027174', '27028629', '.', '-', '.', 'gene_id \"chr7.g11466\"; transcript_id \"chr7.g11466.i48\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27084719', '27085002', '.', '-', '.', 'gene_id \"chr7.g11467\"; transcript_id \"chr7.g11467.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27084988', '27085251', '.', '+', '.', 'gene_id \"chr7.g11471\"; transcript_id \"chr7.g11471.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '717708', '717741', '.', '-', '.', 'gene_id \"chr6.g689\"; transcript_id \"chr6.g689.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '721127', '721331', '.', '+', '.', 'gene_id \"chr6.g692\"; transcript_id \"chr6.g692.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '781133', '781304', '.', '+', '.', 'gene_id \"chr6.g776\"; transcript_id \"chr6.g776.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '780267', '781141', '.', '-', '.', 'gene_id \"chr6.g777\"; transcript_id \"chr6.g777.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27406501', '27406553', '.', '-', '.', 'gene_id \"chr7.g11665\"; transcript_id \"chr7.g11665.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '804279', '804312', '.', '+', '.', 'gene_id \"chr6.g815\"; transcript_id \"chr6.g815.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '819221', '819270', '.', '+', '.', 'gene_id \"chr6.g833\"; transcript_id \"chr6.g833.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27881369', '27882034', '.', '+', '.', 'gene_id \"chr7.g11777\"; transcript_id \"chr7.g11777.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27882166', '27882341', '.', '+', '.', 'gene_id \"chr7.g11778\"; transcript_id \"chr7.g11778.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27882696', '27882992', '.', '+', '.', 'gene_id \"chr7.g11780\"; transcript_id \"chr7.g11780.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '910178', '910210', '.', '+', '.', 'gene_id \"chr6.g930\"; transcript_id \"chr6.g930.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28300209', '28300994', '.', '-', '.', 'gene_id \"chr7.g11878\"; transcript_id \"chr7.g11878.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28301036', '28301213', '.', '+', '.', 'gene_id \"chr7.g11879\"; transcript_id \"chr7.g11879.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1041777', '1041810', '.', '+', '.', 'gene_id \"chr6.g1030\"; transcript_id \"chr6.g1030.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1552279', '1552904', '.', '-', '.', 'gene_id \"chr6.g1174\"; transcript_id \"chr6.g1174.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28566646', '28566880', '.', '-', '.', 'gene_id \"chr7.g12091\"; transcript_id \"chr7.g12091.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1856702', '1856757', '.', '+', '.', 'gene_id \"chr6.g1232\"; transcript_id \"chr6.g1232.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1887014', '1887255', '.', '-', '.', 'gene_id \"chr6.g1266\"; transcript_id \"chr6.g1266.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1887946', '1888104', '.', '+', '.', 'gene_id \"chr6.g1267\"; transcript_id \"chr6.g1267.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1919418', '1919468', '.', '+', '.', 'gene_id \"chr6.g1292\"; transcript_id \"chr6.g1292.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28819674', '28819909', '.', '+', '.', 'gene_id \"chr7.g12203\"; transcript_id \"chr7.g12203.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2043226', '2043313', '.', '-', '.', 'gene_id \"chr6.g1317\"; transcript_id \"chr6.g1317.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28896639', '28896670', '.', '-', '.', 'gene_id \"chr7.g12263\"; transcript_id \"chr7.g12263.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28995851', '28996006', '.', '-', '.', 'gene_id \"chr7.g12279\"; transcript_id \"chr7.g12279.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28998177', '28998244', '.', '-', '.', 'gene_id \"chr7.g12280\"; transcript_id \"chr7.g12280.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '28998197', '28998244', '.', '-', '.', 'gene_id \"chr7.g12280\"; transcript_id \"chr7.g12280.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2460043', '2460085', '.', '+', '.', 'gene_id \"chr6.g1388\"; transcript_id \"chr6.g1388.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2460043', '2460085', '.', '-', '.', 'gene_id \"chr6.g1389\"; transcript_id \"chr6.g1389.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29324138', '29325098', '.', '-', '.', 'gene_id \"chr7.g12387\"; transcript_id \"chr7.g12387.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29324138', '29325103', '.', '-', '.', 'gene_id \"chr7.g12387\"; transcript_id \"chr7.g12387.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2578792', '2579017', '.', '-', '.', 'gene_id \"chr6.g1527\"; transcript_id \"chr6.g1527.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2579014', '2579195', '.', '+', '.', 'gene_id \"chr6.g1528\"; transcript_id \"chr6.g1528.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29528660', '29528743', '.', '-', '.', 'gene_id \"chr7.g12463\"; transcript_id \"chr7.g12463.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2641192', '2641380', '.', '-', '.', 'gene_id \"chr6.g1576\"; transcript_id \"chr6.g1576.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29794766', '29794801', '.', '-', '.', 'gene_id \"chr7.g12515\"; transcript_id \"chr7.g12515.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29837532', '29837561', '.', '+', '.', 'gene_id \"chr7.g12519\"; transcript_id \"chr7.g12519.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2766332', '2766985', '.', '+', '.', 'gene_id \"chr6.g1666\"; transcript_id \"chr6.g1666.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30024687', '30024902', '.', '+', '.', 'gene_id \"chr7.g12633\"; transcript_id \"chr7.g12633.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30033903', '30034094', '.', '-', '.', 'gene_id \"chr7.g12643\"; transcript_id \"chr7.g12643.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30063584', '30069607', '.', '+', '.', 'gene_id \"chr7.g12656\"; transcript_id \"chr7.g12656.i4\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30061048', '30061496', '.', '+', '.', 'gene_id \"chr7.g12691\"; transcript_id \"chr7.g12691.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30090555', '30091252', '.', '+', '.', 'gene_id \"chr7.g12708\"; transcript_id \"chr7.g12708.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30090689', '30091252', '.', '+', '.', 'gene_id \"chr7.g12708\"; transcript_id \"chr7.g12708.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30101357', '30101868', '.', '+', '.', 'gene_id \"chr7.g12726\"; transcript_id \"chr7.g12726.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30124728', '30124974', '.', '+', '.', 'gene_id \"chr7.g12744\"; transcript_id \"chr7.g12744.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30149631', '30149948', '.', '+', '.', 'gene_id \"chr7.g12763\"; transcript_id \"chr7.g12763.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3052969', '3053015', '.', '+', '.', 'gene_id \"chr6.g1819\"; transcript_id \"chr6.g1819.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30173220', '30173412', '.', '+', '.', 'gene_id \"chr7.g12776\"; transcript_id \"chr7.g12776.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30173427', '30173588', '.', '-', '.', 'gene_id \"chr7.g12777\"; transcript_id \"chr7.g12777.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3106241', '3106466', '.', '-', '.', 'gene_id \"chr6.g1851\"; transcript_id \"chr6.g1851.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3145203', '3145235', '.', '+', '.', 'gene_id \"chr6.g1862\"; transcript_id \"chr6.g1862.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30309483', '30309533', '.', '+', '.', 'gene_id \"chr7.g12856\"; transcript_id \"chr7.g12856.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3559464', '3559629', '.', '-', '.', 'gene_id \"chr6.g2067\"; transcript_id \"chr6.g2067.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3559793', '3560072', '.', '-', '.', 'gene_id \"chr6.g2069\"; transcript_id \"chr6.g2069.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3568107', '3568547', '.', '-', '.', 'gene_id \"chr6.g2080\"; transcript_id \"chr6.g2080.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3568107', '3568328', '.', '-', '.', 'gene_id \"chr6.g2080\"; transcript_id \"chr6.g2080.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3571712', '3571753', '.', '+', '.', 'gene_id \"chr6.g2089\"; transcript_id \"chr6.g2089.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3822146', '3822192', '.', '-', '.', 'gene_id \"chr6.g2171\"; transcript_id \"chr6.g2171.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30795512', '30795667', '.', '+', '.', 'gene_id \"chr7.g13193\"; transcript_id \"chr7.g13193.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '4724875', '4724941', '.', '+', '.', 'gene_id \"chr6.g2263\"; transcript_id \"chr6.g2263.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5241104', '5241133', '.', '+', '.', 'gene_id \"chr6.g2301\"; transcript_id \"chr6.g2301.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '31120487', '31120905', '.', '+', '.', 'gene_id \"chr7.g13306\"; transcript_id \"chr7.g13306.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5314591', '5314809', '.', '+', '.', 'gene_id \"chr6.g2327\"; transcript_id \"chr6.g2327.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '31157731', '31157774', '.', '+', '.', 'gene_id \"chr7.g13402\"; transcript_id \"chr7.g13402.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '31663910', '31664097', '.', '+', '.', 'gene_id \"chr7.g13514\"; transcript_id \"chr7.g13514.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5556956', '5557241', '.', '+', '.', 'gene_id \"chr6.g2545\"; transcript_id \"chr6.g2545.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5603187', '5603405', '.', '-', '.', 'gene_id \"chr6.g2581\"; transcript_id \"chr6.g2581.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5681774', '5681806', '.', '+', '.', 'gene_id \"chr6.g2660\"; transcript_id \"chr6.g2660.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5684387', '5684520', '.', '+', '.', 'gene_id \"chr6.g2662\"; transcript_id \"chr6.g2662.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32069406', '32069557', '.', '-', '.', 'gene_id \"chr7.g13681\"; transcript_id \"chr7.g13681.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32141746', '32141847', '.', '+', '.', 'gene_id \"chr7.g13692\"; transcript_id \"chr7.g13692.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '6008359', '6008584', '.', '+', '.', 'gene_id \"chr6.g2925\"; transcript_id \"chr6.g2925.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32788866', '32788901', '.', '+', '.', 'gene_id \"chr7.g13962\"; transcript_id \"chr7.g13962.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '33486715', '33486750', '.', '-', '.', 'gene_id \"chr7.g14075\"; transcript_id \"chr7.g14075.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '33716738', '33716771', '.', '-', '.', 'gene_id \"chr7.g14093\"; transcript_id \"chr7.g14093.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '6573029', '6574228', '.', '-', '.', 'gene_id \"chr6.g3265\"; transcript_id \"chr6.g3265.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34044868', '34044908', '.', '+', '.', 'gene_id \"chr7.g14249\"; transcript_id \"chr7.g14249.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34056353', '34056658', '.', '+', '.', 'gene_id \"chr7.g14263\"; transcript_id \"chr7.g14263.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '6711611', '6711851', '.', '-', '.', 'gene_id \"chr6.g3345\"; transcript_id \"chr6.g3345.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34300523', '34301177', '.', '-', '.', 'gene_id \"chr7.g14272\"; transcript_id \"chr7.g14272.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34365956', '34366311', '.', '+', '.', 'gene_id \"chr7.g14289\"; transcript_id \"chr7.g14289.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34366395', '34366750', '.', '+', '.', 'gene_id \"chr7.g14290\"; transcript_id \"chr7.g14290.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '6780664', '6780771', '.', '+', '.', 'gene_id \"chr6.g3405\"; transcript_id \"chr6.g3405.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34177848', '34178067', '.', '+', '.', 'gene_id \"chr7.g14334\"; transcript_id \"chr7.g14334.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34367988', '34368193', '.', '-', '.', 'gene_id \"chr7.g14467\"; transcript_id \"chr7.g14467.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7058579', '7058743', '.', '-', '.', 'gene_id \"chr6.g3700\"; transcript_id \"chr6.g3700.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7147176', '7147467', '.', '+', '.', 'gene_id \"chr6.g3776\"; transcript_id \"chr6.g3776.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7151287', '7151567', '.', '-', '.', 'gene_id \"chr6.g3781\"; transcript_id \"chr6.g3781.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7168182', '7168274', '.', '+', '.', 'gene_id \"chr6.g3807\"; transcript_id \"chr6.g3807.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7283935', '7284011', '.', '+', '.', 'gene_id \"chr6.g3877\"; transcript_id \"chr6.g3877.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7373344', '7373399', '.', '+', '.', 'gene_id \"chr6.g3950\"; transcript_id \"chr6.g3950.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35310527', '35311232', '.', '-', '.', 'gene_id \"chr7.g14842\"; transcript_id \"chr7.g14842.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35310767', '35311103', '.', '-', '.', 'gene_id \"chr7.g14842\"; transcript_id \"chr7.g14842.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35312950', '35313105', '.', '-', '.', 'gene_id \"chr7.g14845\"; transcript_id \"chr7.g14845.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7676090', '7676356', '.', '-', '.', 'gene_id \"chr6.g4156\"; transcript_id \"chr6.g4156.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7861828', '7862036', '.', '+', '.', 'gene_id \"chr6.g4327\"; transcript_id \"chr6.g4327.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36178141', '36178421', '.', '-', '.', 'gene_id \"chr7.g15202\"; transcript_id \"chr7.g15202.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36178465', '36178622', '.', '-', '.', 'gene_id \"chr7.g15203\"; transcript_id \"chr7.g15203.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8074487', '8074528', '.', '+', '.', 'gene_id \"chr6.g4452\"; transcript_id \"chr6.g4452.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8136811', '8136857', '.', '-', '.', 'gene_id \"chr6.g4467\"; transcript_id \"chr6.g4467.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8403376', '8403767', '.', '-', '.', 'gene_id \"chr6.g4554\"; transcript_id \"chr6.g4554.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8485745', '8486205', '.', '+', '.', 'gene_id \"chr6.g4623\"; transcript_id \"chr6.g4623.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8486189', '8486674', '.', '-', '.', 'gene_id \"chr6.g4629\"; transcript_id \"chr6.g4629.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8554119', '8554296', '.', '+', '.', 'gene_id \"chr6.g4658\"; transcript_id \"chr6.g4658.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8554399', '8554550', '.', '+', '.', 'gene_id \"chr6.g4658\"; transcript_id \"chr6.g4658.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36894067', '36894107', '.', '+', '.', 'gene_id \"chr7.g15474\"; transcript_id \"chr7.g15474.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36908225', '36908268', '.', '-', '.', 'gene_id \"chr7.g15480\"; transcript_id \"chr7.g15480.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36945132', '36945338', '.', '-', '.', 'gene_id \"chr7.g15531\"; transcript_id \"chr7.g15531.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36945718', '36945829', '.', '-', '.', 'gene_id \"chr7.g15532\"; transcript_id \"chr7.g15532.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36946303', '36946453', '.', '-', '.', 'gene_id \"chr7.g15535\"; transcript_id \"chr7.g15535.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37136282', '37136408', '.', '+', '.', 'gene_id \"chr7.g15677\"; transcript_id \"chr7.g15677.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9016008', '9016053', '.', '+', '.', 'gene_id \"chr6.g4949\"; transcript_id \"chr6.g4949.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37188995', '37189045', '.', '-', '.', 'gene_id \"chr7.g15705\"; transcript_id \"chr7.g15705.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37213748', '37214096', '.', '-', '.', 'gene_id \"chr7.g15709\"; transcript_id \"chr7.g15709.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9042559', '9042687', '.', '+', '.', 'gene_id \"chr6.g4981\"; transcript_id \"chr6.g4981.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37293289', '37293466', '.', '+', '.', 'gene_id \"chr7.g15746\"; transcript_id \"chr7.g15746.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9088862', '9089428', '.', '+', '.', 'gene_id \"chr6.g4994\"; transcript_id \"chr6.g4994.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37260322', '37260362', '.', '-', '.', 'gene_id \"chr7.g15772\"; transcript_id \"chr7.g15772.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9221502', '9221605', '.', '+', '.', 'gene_id \"chr6.g5050\"; transcript_id \"chr6.g5050.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37340933', '37340988', '.', '-', '.', 'gene_id \"chr7.g15812\"; transcript_id \"chr7.g15812.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37504774', '37504807', '.', '-', '.', 'gene_id \"chr7.g15860\"; transcript_id \"chr7.g15860.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9396133', '9396421', '.', '-', '.', 'gene_id \"chr6.g5179\"; transcript_id \"chr6.g5179.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9399365', '9399424', '.', '+', '.', 'gene_id \"chr6.g5184\"; transcript_id \"chr6.g5184.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37741928', '37742206', '.', '-', '.', 'gene_id \"chr7.g16002\"; transcript_id \"chr7.g16002.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9586903', '9587668', '.', '+', '.', 'gene_id \"chr6.g5297\"; transcript_id \"chr6.g5297.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37910052', '37910234', '.', '+', '.', 'gene_id \"chr7.g16102\"; transcript_id \"chr7.g16102.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9881330', '9881400', '.', '-', '.', 'gene_id \"chr6.g5452\"; transcript_id \"chr6.g5452.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9938182', '9938234', '.', '-', '.', 'gene_id \"chr6.g5510\"; transcript_id \"chr6.g5510.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '9961704', '9961962', '.', '-', '.', 'gene_id \"chr6.g5532\"; transcript_id \"chr6.g5532.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10045222', '10045265', '.', '-', '.', 'gene_id \"chr6.g5618\"; transcript_id \"chr6.g5618.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38204252', '38204507', '.', '+', '.', 'gene_id \"chr7.g16349\"; transcript_id \"chr7.g16349.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38286675', '38286735', '.', '-', '.', 'gene_id \"chr7.g16380\"; transcript_id \"chr7.g16380.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38483644', '38483679', '.', '+', '.', 'gene_id \"chr7.g16489\"; transcript_id \"chr7.g16489.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38530400', '38530451', '.', '+', '.', 'gene_id \"chr7.g16503\"; transcript_id \"chr7.g16503.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10323078', '10323401', '.', '-', '.', 'gene_id \"chr6.g5861\"; transcript_id \"chr6.g5861.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10326592', '10326786', '.', '+', '.', 'gene_id \"chr6.g5864\"; transcript_id \"chr6.g5864.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38666103', '38666610', '.', '+', '.', 'gene_id \"chr7.g16609\"; transcript_id \"chr7.g16609.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38687538', '38687758', '.', '-', '.', 'gene_id \"chr7.g16621\"; transcript_id \"chr7.g16621.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38712049', '38712102', '.', '+', '.', 'gene_id \"chr7.g16636\"; transcript_id \"chr7.g16636.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38860735', '38860935', '.', '-', '.', 'gene_id \"chr7.g16727\"; transcript_id \"chr7.g16727.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10531365', '10531399', '.', '-', '.', 'gene_id \"chr6.g6055\"; transcript_id \"chr6.g6055.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10575753', '10575792', '.', '-', '.', 'gene_id \"chr6.g6059\"; transcript_id \"chr6.g6059.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10649509', '10649549', '.', '+', '.', 'gene_id \"chr6.g6067\"; transcript_id \"chr6.g6067.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10649509', '10649549', '.', '-', '.', 'gene_id \"chr6.g6068\"; transcript_id \"chr6.g6068.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38912464', '38912870', '.', '+', '.', 'gene_id \"chr7.g16762\"; transcript_id \"chr7.g16762.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10778500', '10778594', '.', '-', '.', 'gene_id \"chr6.g6120\"; transcript_id \"chr6.g6120.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10779510', '10779976', '.', '-', '.', 'gene_id \"chr6.g6124\"; transcript_id \"chr6.g6124.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10788436', '10788479', '.', '-', '.', 'gene_id \"chr6.g6131\"; transcript_id \"chr6.g6131.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39249832', '39249864', '.', '+', '.', 'gene_id \"chr7.g16858\"; transcript_id \"chr7.g16858.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10912289', '10912549', '.', '-', '.', 'gene_id \"chr6.g6212\"; transcript_id \"chr6.g6212.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10944016', '10944249', '.', '-', '.', 'gene_id \"chr6.g6233\"; transcript_id \"chr6.g6233.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10945068', '10945380', '.', '-', '.', 'gene_id \"chr6.g6234\"; transcript_id \"chr6.g6234.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10945563', '10945991', '.', '-', '.', 'gene_id \"chr6.g6236\"; transcript_id \"chr6.g6236.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i5\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i6\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i7\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i8\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i9\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i10\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i11\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946232', '10946396', '.', '-', '.', 'gene_id \"chr6.g6237\"; transcript_id \"chr6.g6237.i12\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10946035', '10946252', '.', '+', '.', 'gene_id \"chr6.g6238\"; transcript_id \"chr6.g6238.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39595721', '39595957', '.', '+', '.', 'gene_id \"chr7.g16940\"; transcript_id \"chr7.g16940.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11189338', '11189718', '.', '+', '.', 'gene_id \"chr6.g6376\"; transcript_id \"chr6.g6376.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39826399', '39826449', '.', '+', '.', 'gene_id \"chr7.g17129\"; transcript_id \"chr7.g17129.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39967593', '39967816', '.', '-', '.', 'gene_id \"chr7.g17261\"; transcript_id \"chr7.g17261.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11564623', '11564906', '.', '-', '.', 'gene_id \"chr6.g6658\"; transcript_id \"chr6.g6658.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11564980', '11565160', '.', '-', '.', 'gene_id \"chr6.g6659\"; transcript_id \"chr6.g6659.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '40231370', '40231411', '.', '+', '.', 'gene_id \"chr7.g17415\"; transcript_id \"chr7.g17415.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11662123', '11662534', '.', '+', '.', 'gene_id \"chr6.g6749\"; transcript_id \"chr6.g6749.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11761107', '11761140', '.', '-', '.', 'gene_id \"chr6.g6827\"; transcript_id \"chr6.g6827.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '40684718', '40684911', '.', '+', '.', 'gene_id \"chr7.g17576\"; transcript_id \"chr7.g17576.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41033874', '41033947', '.', '+', '.', 'gene_id \"chr7.g17830\"; transcript_id \"chr7.g17830.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41112213', '41112726', '.', '-', '.', 'gene_id \"chr7.g17877\"; transcript_id \"chr7.g17877.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41112062', '41112225', '.', '+', '.', 'gene_id \"chr7.g17879\"; transcript_id \"chr7.g17879.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41120914', '41121323', '.', '+', '.', 'gene_id \"chr7.g17880\"; transcript_id \"chr7.g17880.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41119119', '41119275', '.', '-', '.', 'gene_id \"chr7.g17883\"; transcript_id \"chr7.g17883.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41122063', '41122415', '.', '-', '.', 'gene_id \"chr7.g17884\"; transcript_id \"chr7.g17884.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41121303', '41121977', '.', '-', '.', 'gene_id \"chr7.g17885\"; transcript_id \"chr7.g17885.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12368853', '12369025', '.', '+', '.', 'gene_id \"chr6.g7262\"; transcript_id \"chr6.g7262.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12440483', '12440529', '.', '+', '.', 'gene_id \"chr6.g7313\"; transcript_id \"chr6.g7313.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41338207', '41338367', '.', '+', '.', 'gene_id \"chr7.g18072\"; transcript_id \"chr7.g18072.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41338406', '41338664', '.', '-', '.', 'gene_id \"chr7.g18073\"; transcript_id \"chr7.g18073.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41338742', '41339004', '.', '-', '.', 'gene_id \"chr7.g18074\"; transcript_id \"chr7.g18074.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41338989', '41339145', '.', '+', '.', 'gene_id \"chr7.g18075\"; transcript_id \"chr7.g18075.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12594606', '12594783', '.', '+', '.', 'gene_id \"chr6.g7457\"; transcript_id \"chr6.g7457.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41530076', '41530296', '.', '+', '.', 'gene_id \"chr7.g18260\"; transcript_id \"chr7.g18260.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41532321', '41532640', '.', '+', '.', 'gene_id \"chr7.g18267\"; transcript_id \"chr7.g18267.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41534608', '41534798', '.', '-', '.', 'gene_id \"chr7.g18269\"; transcript_id \"chr7.g18269.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41527408', '41527817', '.', '+', '.', 'gene_id \"chr7.g18288\"; transcript_id \"chr7.g18288.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12665485', '12665817', '.', '-', '.', 'gene_id \"chr6.g7510\"; transcript_id \"chr6.g7510.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12706989', '12708211', '.', '-', '.', 'gene_id \"chr6.g7512\"; transcript_id \"chr6.g7512.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12664970', '12665139', '.', '-', '.', 'gene_id \"chr6.g7514\"; transcript_id \"chr6.g7514.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12667149', '12667383', '.', '-', '.', 'gene_id \"chr6.g7521\"; transcript_id \"chr6.g7521.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12667368', '12667524', '.', '+', '.', 'gene_id \"chr6.g7522\"; transcript_id \"chr6.g7522.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41587834', '41587897', '.', '+', '.', 'gene_id \"chr7.g18339\"; transcript_id \"chr7.g18339.i27\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12687852', '12688053', '.', '+', '.', 'gene_id \"chr6.g7547\"; transcript_id \"chr6.g7547.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12689537', '12690014', '.', '+', '.', 'gene_id \"chr6.g7550\"; transcript_id \"chr6.g7550.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12696671', '12697209', '.', '-', '.', 'gene_id \"chr6.g7553\"; transcript_id \"chr6.g7553.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12702263', '12702414', '.', '-', '.', 'gene_id \"chr6.g7561\"; transcript_id \"chr6.g7561.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41597451', '41597638', '.', '+', '.', 'gene_id \"chr7.g18377\"; transcript_id \"chr7.g18377.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41609063', '41609105', '.', '-', '.', 'gene_id \"chr7.g18385\"; transcript_id \"chr7.g18385.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41659786', '41659975', '.', '-', '.', 'gene_id \"chr7.g18419\"; transcript_id \"chr7.g18419.i13\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12847958', '12848223', '.', '-', '.', 'gene_id \"chr6.g7648\"; transcript_id \"chr6.g7648.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13483215', '13483282', '.', '+', '.', 'gene_id \"chr6.g7785\"; transcript_id \"chr6.g7785.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41828468', '41828502', '.', '+', '.', 'gene_id \"chr7.g18639\"; transcript_id \"chr7.g18639.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13656134', '13656187', '.', '+', '.', 'gene_id \"chr6.g7837\"; transcript_id \"chr6.g7837.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13663102', '13663758', '.', '+', '.', 'gene_id \"chr6.g7842\"; transcript_id \"chr6.g7842.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13681556', '13681698', '.', '+', '.', 'gene_id \"chr6.g7847\"; transcript_id \"chr6.g7847.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41919321', '41919386', '.', '-', '.', 'gene_id \"chr7.g18704\"; transcript_id \"chr7.g18704.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13839140', '13839348', '.', '-', '.', 'gene_id \"chr6.g7884\"; transcript_id \"chr6.g7884.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13839364', '13839570', '.', '+', '.', 'gene_id \"chr6.g7885\"; transcript_id \"chr6.g7885.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13854173', '13854332', '.', '+', '.', 'gene_id \"chr6.g7892\"; transcript_id \"chr6.g7892.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42103520', '42103690', '.', '+', '.', 'gene_id \"chr7.g18826\"; transcript_id \"chr7.g18826.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42103732', '42103916', '.', '-', '.', 'gene_id \"chr7.g18827\"; transcript_id \"chr7.g18827.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '42154364', '42154570', '.', '-', '.', 'gene_id \"chr7.g18864\"; transcript_id \"chr7.g18864.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42281861', '42281893', '.', '-', '.', 'gene_id \"chr7.g18944\"; transcript_id \"chr7.g18944.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '14534320', '14534742', '.', '+', '.', 'gene_id \"chr6.g8207\"; transcript_id \"chr6.g8207.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '14535685', '14536248', '.', '-', '.', 'gene_id \"chr6.g8210\"; transcript_id \"chr6.g8210.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42562375', '42562436', '.', '-', '.', 'gene_id \"chr7.g19161\"; transcript_id \"chr7.g19161.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42572028', '42572069', '.', '+', '.', 'gene_id \"chr7.g19170\"; transcript_id \"chr7.g19170.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42644088', '42644296', '.', '-', '.', 'gene_id \"chr7.g19243\"; transcript_id \"chr7.g19243.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '14919166', '14919399', '.', '+', '.', 'gene_id \"chr6.g8439\"; transcript_id \"chr6.g8439.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42877949', '42878283', '.', '+', '.', 'gene_id \"chr7.g19444\"; transcript_id \"chr7.g19444.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42878234', '42878299', '.', '+', '.', 'gene_id \"chr7.g19444\"; transcript_id \"chr7.g19444.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15336519', '15336814', '.', '+', '.', 'gene_id \"chr6.g8602\"; transcript_id \"chr6.g8602.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i5\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i6\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15338518', '15338851', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i7\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15345818', '15346600', '.', '+', '.', 'gene_id \"chr6.g8603\"; transcript_id \"chr6.g8603.i14\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15336865', '15338150', '.', '+', '.', 'gene_id \"chr6.g8605\"; transcript_id \"chr6.g8605.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15560087', '15560126', '.', '-', '.', 'gene_id \"chr6.g8673\"; transcript_id \"chr6.g8673.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '43119578', '43119608', '.', '+', '.', 'gene_id \"chr7.g19717\"; transcript_id \"chr7.g19717.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '43383715', '43383874', '.', '+', '.', 'gene_id \"chr7.g19809\"; transcript_id \"chr7.g19809.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '43383997', '43384278', '.', '-', '.', 'gene_id \"chr7.g19810\"; transcript_id \"chr7.g19810.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '15800601', '15800764', '.', '+', '.', 'gene_id \"chr6.g8844\"; transcript_id \"chr6.g8844.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '43531868', '43532215', '.', '+', '.', 'gene_id \"chr7.g19885\"; transcript_id \"chr7.g19885.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '43805287', '43805326', '.', '-', '.', 'gene_id \"chr7.g19909\"; transcript_id \"chr7.g19909.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16135680', '16135850', '.', '+', '.', 'gene_id \"chr6.g9124\"; transcript_id \"chr6.g9124.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16135703', '16135854', '.', '+', '.', 'gene_id \"chr6.g9124\"; transcript_id \"chr6.g9124.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16135748', '16135861', '.', '+', '.', 'gene_id \"chr6.g9124\"; transcript_id \"chr6.g9124.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44285525', '44285585', '.', '-', '.', 'gene_id \"chr7.g20145\"; transcript_id \"chr7.g20145.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44610438', '44610500', '.', '+', '.', 'gene_id \"chr7.g20214\"; transcript_id \"chr7.g20214.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44658251', '44658638', '.', '+', '.', 'gene_id \"chr7.g20245\"; transcript_id \"chr7.g20245.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44679687', '44680010', '.', '-', '.', 'gene_id \"chr7.g20260\"; transcript_id \"chr7.g20260.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44680328', '44680713', '.', '+', '.', 'gene_id \"chr7.g20261\"; transcript_id \"chr7.g20261.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44681580', '44681889', '.', '+', '.', 'gene_id \"chr7.g20262\"; transcript_id \"chr7.g20262.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16550421', '16550541', '.', '-', '.', 'gene_id \"chr6.g9335\"; transcript_id \"chr6.g9335.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44845233', '44845714', '.', '+', '.', 'gene_id \"chr7.g20329\"; transcript_id \"chr7.g20329.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44849078', '44849649', '.', '+', '.', 'gene_id \"chr7.g20332\"; transcript_id \"chr7.g20332.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44813174', '44813299', '.', '+', '.', 'gene_id \"chr7.g20343\"; transcript_id \"chr7.g20343.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '44813175', '44813258', '.', '-', '.', 'gene_id \"chr7.g20344\"; transcript_id \"chr7.g20344.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16919921', '16919961', '.', '-', '.', 'gene_id \"chr6.g9490\"; transcript_id \"chr6.g9490.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16946738', '16946839', '.', '+', '.', 'gene_id \"chr6.g9497\"; transcript_id \"chr6.g9497.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16966986', '16967027', '.', '-', '.', 'gene_id \"chr6.g9502\"; transcript_id \"chr6.g9502.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16967105', '16967265', '.', '+', '.', 'gene_id \"chr6.g9510\"; transcript_id \"chr6.g9510.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '45458162', '45458552', '.', '-', '.', 'gene_id \"chr7.g20518\"; transcript_id \"chr7.g20518.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '45565095', '45565830', '.', '-', '.', 'gene_id \"chr7.g20527\"; transcript_id \"chr7.g20527.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '45694885', '45695304', '.', '-', '.', 'gene_id \"chr7.g20582\"; transcript_id \"chr7.g20582.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17235084', '17235643', '.', '-', '.', 'gene_id \"chr6.g9697\"; transcript_id \"chr6.g9697.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17235084', '17235643', '.', '-', '.', 'gene_id \"chr6.g9697\"; transcript_id \"chr6.g9697.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17239946', '17240101', '.', '-', '.', 'gene_id \"chr6.g9700\"; transcript_id \"chr6.g9700.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17242960', '17243196', '.', '+', '.', 'gene_id \"chr6.g9711\"; transcript_id \"chr6.g9711.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17243199', '17243368', '.', '+', '.', 'gene_id \"chr6.g9712\"; transcript_id \"chr6.g9712.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17244847', '17244917', '.', '+', '.', 'gene_id \"chr6.g9716\"; transcript_id \"chr6.g9716.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17261189', '17261384', '.', '+', '.', 'gene_id \"chr6.g9745\"; transcript_id \"chr6.g9745.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17261391', '17261803', '.', '+', '.', 'gene_id \"chr6.g9746\"; transcript_id \"chr6.g9746.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17261804', '17262178', '.', '-', '.', 'gene_id \"chr6.g9747\"; transcript_id \"chr6.g9747.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17261996', '17262178', '.', '-', '.', 'gene_id \"chr6.g9747\"; transcript_id \"chr6.g9747.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '45979019', '45979209', '.', '-', '.', 'gene_id \"chr7.g20731\"; transcript_id \"chr7.g20731.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17471807', '17471938', '.', '-', '.', 'gene_id \"chr6.g9896\"; transcript_id \"chr6.g9896.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17470799', '17471461', '.', '-', '.', 'gene_id \"chr6.g9898\"; transcript_id \"chr6.g9898.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17482761', '17484070', '.', '-', '.', 'gene_id \"chr6.g9916\"; transcript_id \"chr6.g9916.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '46102811', '46102963', '.', '-', '.', 'gene_id \"chr7.g20787\"; transcript_id \"chr7.g20787.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '46126198', '46126361', '.', '+', '.', 'gene_id \"chr7.g20792\"; transcript_id \"chr7.g20792.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17498252', '17500050', '.', '-', '.', 'gene_id \"chr6.g9927\"; transcript_id \"chr6.g9927.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17508374', '17509376', '.', '-', '.', 'gene_id \"chr6.g9942\"; transcript_id \"chr6.g9942.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17508926', '17509057', '.', '-', '.', 'gene_id \"chr6.g9942\"; transcript_id \"chr6.g9942.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17509957', '17510137', '.', '-', '.', 'gene_id \"chr6.g9945\"; transcript_id \"chr6.g9945.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '46524321', '46524360', '.', '+', '.', 'gene_id \"chr7.g20813\"; transcript_id \"chr7.g20813.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17520348', '17520493', '.', '-', '.', 'gene_id \"chr6.g9960\"; transcript_id \"chr6.g9960.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17520360', '17520493', '.', '-', '.', 'gene_id \"chr6.g9960\"; transcript_id \"chr6.g9960.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17520349', '17520484', '.', '-', '.', 'gene_id \"chr6.g9960\"; transcript_id \"chr6.g9960.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17520348', '17520494', '.', '-', '.', 'gene_id \"chr6.g9960\"; transcript_id \"chr6.g9960.i4\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '46748159', '46748211', '.', '+', '.', 'gene_id \"chr7.g20834\"; transcript_id \"chr7.g20834.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '47809528', '47809591', '.', '+', '.', 'gene_id \"chr7.g20945\"; transcript_id \"chr7.g20945.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48223735', '48223785', '.', '+', '.', 'gene_id \"chr7.g20977\"; transcript_id \"chr7.g20977.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48299884', '48300253', '.', '-', '.', 'gene_id \"chr7.g20982\"; transcript_id \"chr7.g20982.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48302101', '48302259', '.', '-', '.', 'gene_id \"chr7.g20984\"; transcript_id \"chr7.g20984.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17832665', '17832816', '.', '+', '.', 'gene_id \"chr6.g10163\"; transcript_id \"chr6.g10163.i9\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17829360', '17829640', '.', '-', '.', 'gene_id \"chr6.g10165\"; transcript_id \"chr6.g10165.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17826484', '17826677', '.', '-', '.', 'gene_id \"chr6.g10166\"; transcript_id \"chr6.g10166.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17827109', '17827379', '.', '+', '.', 'gene_id \"chr6.g10172\"; transcript_id \"chr6.g10172.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17826826', '17827052', '.', '+', '.', 'gene_id \"chr6.g10178\"; transcript_id \"chr6.g10178.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17832001', '17832158', '.', '+', '.', 'gene_id \"chr6.g10182\"; transcript_id \"chr6.g10182.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17824799', '17824949', '.', '-', '.', 'gene_id \"chr6.g10201\"; transcript_id \"chr6.g10201.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17824955', '17825308', '.', '+', '.', 'gene_id \"chr6.g10202\"; transcript_id \"chr6.g10202.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48352888', '48353119', '.', '-', '.', 'gene_id \"chr7.g21044\"; transcript_id \"chr7.g21044.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48408440', '48408710', '.', '+', '.', 'gene_id \"chr7.g21095\"; transcript_id \"chr7.g21095.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48428296', '48428403', '.', '-', '.', 'gene_id \"chr7.g21122\"; transcript_id \"chr7.g21122.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48756512', '48756598', '.', '+', '.', 'gene_id \"chr7.g21178\"; transcript_id \"chr7.g21178.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '49361605', '49361869', '.', '-', '.', 'gene_id \"chr7.g21218\"; transcript_id \"chr7.g21218.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '49752045', '49752252', '.', '-', '.', 'gene_id \"chr7.g21256\"; transcript_id \"chr7.g21256.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '18138237', '18138275', '.', '-', '.', 'gene_id \"chr6.g10492\"; transcript_id \"chr6.g10492.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '18130013', '18130273', '.', '-', '.', 'gene_id \"chr6.g10493\"; transcript_id \"chr6.g10493.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '50112417', '50112479', '.', '-', '.', 'gene_id \"chr7.g21283\"; transcript_id \"chr7.g21283.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '18249129', '18249204', '.', '+', '.', 'gene_id \"chr6.g10569\"; transcript_id \"chr6.g10569.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52157785', '52157836', '.', '-', '.', 'gene_id \"chr7.g21402\"; transcript_id \"chr7.g21402.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52237233', '52237278', '.', '+', '.', 'gene_id \"chr7.g21413\"; transcript_id \"chr7.g21413.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52678779', '52679050', '.', '+', '.', 'gene_id \"chr7.g21510\"; transcript_id \"chr7.g21510.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52825113', '52825385', '.', '+', '.', 'gene_id \"chr7.g21599\"; transcript_id \"chr7.g21599.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52888294', '52888683', '.', '+', '.', 'gene_id \"chr7.g21648\"; transcript_id \"chr7.g21648.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52888666', '52889091', '.', '+', '.', 'gene_id \"chr7.g21650\"; transcript_id \"chr7.g21650.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52893245', '52893545', '.', '-', '.', 'gene_id \"chr7.g21655\"; transcript_id \"chr7.g21655.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19046986', '19047021', '.', '-', '.', 'gene_id \"chr6.g11106\"; transcript_id \"chr6.g11106.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53007676', '53008046', '.', '+', '.', 'gene_id \"chr7.g21767\"; transcript_id \"chr7.g21767.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53008618', '53009310', '.', '-', '.', 'gene_id \"chr7.g21770\"; transcript_id \"chr7.g21770.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19063174', '19063274', '.', '+', '.', 'gene_id \"chr6.g11110\"; transcript_id \"chr6.g11110.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19063251', '19063334', '.', '+', '.', 'gene_id \"chr6.g11110\"; transcript_id \"chr6.g11110.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53019881', '53020521', '.', '-', '.', 'gene_id \"chr7.g21780\"; transcript_id \"chr7.g21780.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53020344', '53020417', '.', '+', '.', 'gene_id \"chr7.g21781\"; transcript_id \"chr7.g21781.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53184198', '53184237', '.', '+', '.', 'gene_id \"chr7.g21888\"; transcript_id \"chr7.g21888.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19327251', '19327525', '.', '+', '.', 'gene_id \"chr6.g11279\"; transcript_id \"chr6.g11279.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19327518', '19327689', '.', '-', '.', 'gene_id \"chr6.g11280\"; transcript_id \"chr6.g11280.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19327732', '19328039', '.', '+', '.', 'gene_id \"chr6.g11281\"; transcript_id \"chr6.g11281.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53613331', '53613500', '.', '+', '.', 'gene_id \"chr7.g22012\"; transcript_id \"chr7.g22012.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19603187', '19603216', '.', '+', '.', 'gene_id \"chr6.g11335\"; transcript_id \"chr6.g11335.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53976397', '53976477', '.', '-', '.', 'gene_id \"chr7.g22202\"; transcript_id \"chr7.g22202.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54097365', '54097650', '.', '-', '.', 'gene_id \"chr7.g22266\"; transcript_id \"chr7.g22266.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54155292', '54155482', '.', '-', '.', 'gene_id \"chr7.g22316\"; transcript_id \"chr7.g22316.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54239094', '54239847', '.', '+', '.', 'gene_id \"chr7.g22367\"; transcript_id \"chr7.g22367.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54311761', '54311980', '.', '-', '.', 'gene_id \"chr7.g22394\"; transcript_id \"chr7.g22394.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21146851', '21146925', '.', '-', '.', 'gene_id \"chr6.g11742\"; transcript_id \"chr6.g11742.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21196816', '21196970', '.', '-', '.', 'gene_id \"chr6.g11769\"; transcript_id \"chr6.g11769.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21203168', '21203406', '.', '-', '.', 'gene_id \"chr6.g11780\"; transcript_id \"chr6.g11780.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21206436', '21206709', '.', '-', '.', 'gene_id \"chr6.g11786\"; transcript_id \"chr6.g11786.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21245051', '21245308', '.', '-', '.', 'gene_id \"chr6.g11816\"; transcript_id \"chr6.g11816.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21245288', '21245608', '.', '+', '.', 'gene_id \"chr6.g11817\"; transcript_id \"chr6.g11817.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54563250', '54563378', '.', '-', '.', 'gene_id \"chr7.g22522\"; transcript_id \"chr7.g22522.i7\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54593184', '54593475', '.', '-', '.', 'gene_id \"chr7.g22522\"; transcript_id \"chr7.g22522.i8\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54659903', '54660200', '.', '+', '.', 'gene_id \"chr7.g22526\"; transcript_id \"chr7.g22526.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54567757', '54567947', '.', '-', '.', 'gene_id \"chr7.g22586\"; transcript_id \"chr7.g22586.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21370359', '21370447', '.', '-', '.', 'gene_id \"chr6.g11929\"; transcript_id \"chr6.g11929.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21370359', '21370445', '.', '-', '.', 'gene_id \"chr6.g11929\"; transcript_id \"chr6.g11929.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54590156', '54590371', '.', '-', '.', 'gene_id \"chr7.g22610\"; transcript_id \"chr7.g22610.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54592032', '54592207', '.', '-', '.', 'gene_id \"chr7.g22614\"; transcript_id \"chr7.g22614.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54651708', '54651859', '.', '+', '.', 'gene_id \"chr7.g22636\"; transcript_id \"chr7.g22636.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54660604', '54660775', '.', '-', '.', 'gene_id \"chr7.g22642\"; transcript_id \"chr7.g22642.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21575391', '21575577', '.', '-', '.', 'gene_id \"chr6.g11990\"; transcript_id \"chr6.g11990.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21579901', '21580178', '.', '-', '.', 'gene_id \"chr6.g11994\"; transcript_id \"chr6.g11994.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55004252', '55004469', '.', '-', '.', 'gene_id \"chr7.g22793\"; transcript_id \"chr7.g22793.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55163256', '55163318', '.', '-', '.', 'gene_id \"chr7.g22826\"; transcript_id \"chr7.g22826.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55191516', '55191667', '.', '+', '.', 'gene_id \"chr7.g22828\"; transcript_id \"chr7.g22828.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22041370', '22041402', '.', '-', '.', 'gene_id \"chr6.g12153\"; transcript_id \"chr6.g12153.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55234957', '55235040', '.', '+', '.', 'gene_id \"chr7.g22841\"; transcript_id \"chr7.g22841.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55236577', '55236805', '.', '+', '.', 'gene_id \"chr7.g22842\"; transcript_id \"chr7.g22842.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55487393', '55487620', '.', '-', '.', 'gene_id \"chr7.g22896\"; transcript_id \"chr7.g22896.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55516489', '55516527', '.', '-', '.', 'gene_id \"chr7.g22898\"; transcript_id \"chr7.g22898.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22200337', '22200375', '.', '+', '.', 'gene_id \"chr6.g12246\"; transcript_id \"chr6.g12246.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55644200', '55644385', '.', '+', '.', 'gene_id \"chr7.g22909\"; transcript_id \"chr7.g22909.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55644592', '55644787', '.', '+', '.', 'gene_id \"chr7.g22910\"; transcript_id \"chr7.g22910.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '56148099', '56148137', '.', '-', '.', 'gene_id \"chr7.g22975\"; transcript_id \"chr7.g22975.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '56381030', '56381081', '.', '+', '.', 'gene_id \"chr7.g22995\"; transcript_id \"chr7.g22995.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '56437139', '56437282', '.', '-', '.', 'gene_id \"chr7.g22998\"; transcript_id \"chr7.g22998.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22573978', '22574193', '.', '-', '.', 'gene_id \"chr6.g12570\"; transcript_id \"chr6.g12570.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22642143', '22642647', '.', '+', '.', 'gene_id \"chr6.g12595\"; transcript_id \"chr6.g12595.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22666907', '22666977', '.', '+', '.', 'gene_id \"chr6.g12616\"; transcript_id \"chr6.g12616.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57193804', '57194047', '.', '-', '.', 'gene_id \"chr7.g23225\"; transcript_id \"chr7.g23225.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57239348', '57239586', '.', '+', '.', 'gene_id \"chr7.g23248\"; transcript_id \"chr7.g23248.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57239661', '57239866', '.', '+', '.', 'gene_id \"chr7.g23249\"; transcript_id \"chr7.g23249.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57240922', '57241145', '.', '+', '.', 'gene_id \"chr7.g23252\"; transcript_id \"chr7.g23252.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23008791', '23008841', '.', '-', '.', 'gene_id \"chr6.g12672\"; transcript_id \"chr6.g12672.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57274629', '57274658', '.', '-', '.', 'gene_id \"chr7.g23277\"; transcript_id \"chr7.g23277.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23118269', '23118308', '.', '+', '.', 'gene_id \"chr6.g12692\"; transcript_id \"chr6.g12692.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23159093', '23159213', '.', '-', '.', 'gene_id \"chr6.g12731\"; transcript_id \"chr6.g12731.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57454548', '57454588', '.', '-', '.', 'gene_id \"chr7.g23407\"; transcript_id \"chr7.g23407.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23268038', '23268090', '.', '-', '.', 'gene_id \"chr6.g12820\"; transcript_id \"chr6.g12820.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57484203', '57484844', '.', '+', '.', 'gene_id \"chr7.g23444\"; transcript_id \"chr7.g23444.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57562177', '57562967', '.', '-', '.', 'gene_id \"chr7.g23514\"; transcript_id \"chr7.g23514.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57562997', '57563309', '.', '+', '.', 'gene_id \"chr7.g23515\"; transcript_id \"chr7.g23515.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23414105', '23414221', '.', '+', '.', 'gene_id \"chr6.g12948\"; transcript_id \"chr6.g12948.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23449290', '23449616', '.', '-', '.', 'gene_id \"chr6.g12961\"; transcript_id \"chr6.g12961.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23501499', '23501543', '.', '+', '.', 'gene_id \"chr6.g12981\"; transcript_id \"chr6.g12981.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23515280', '23515431', '.', '-', '.', 'gene_id \"chr6.g12998\"; transcript_id \"chr6.g12998.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57701015', '57701057', '.', '-', '.', 'gene_id \"chr7.g23600\"; transcript_id \"chr7.g23600.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57864382', '57864551', '.', '-', '.', 'gene_id \"chr7.g23674\"; transcript_id \"chr7.g23674.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23735445', '23735481', '.', '+', '.', 'gene_id \"chr6.g13120\"; transcript_id \"chr6.g13120.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23865040', '23865072', '.', '+', '.', 'gene_id \"chr6.g13133\"; transcript_id \"chr6.g13133.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23945715', '23945786', '.', '-', '.', 'gene_id \"chr6.g13148\"; transcript_id \"chr6.g13148.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24129331', '24129552', '.', '+', '.', 'gene_id \"chr6.g13189\"; transcript_id \"chr6.g13189.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24129382', '24129552', '.', '+', '.', 'gene_id \"chr6.g13189\"; transcript_id \"chr6.g13189.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '58687896', '58688073', '.', '+', '.', 'gene_id \"chr7.g23807\"; transcript_id \"chr7.g23807.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '59273276', '59273455', '.', '+', '.', 'gene_id \"chr7.g23841\"; transcript_id \"chr7.g23841.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24224718', '24224762', '.', '-', '.', 'gene_id \"chr6.g13285\"; transcript_id \"chr6.g13285.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '59600992', '59601059', '.', '+', '.', 'gene_id \"chr7.g23864\"; transcript_id \"chr7.g23864.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '59600992', '59601039', '.', '+', '.', 'gene_id \"chr7.g23864\"; transcript_id \"chr7.g23864.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24310315', '24310363', '.', '+', '.', 'gene_id \"chr6.g13324\"; transcript_id \"chr6.g13324.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60427655', '60427992', '.', '-', '.', 'gene_id \"chr7.g23884\"; transcript_id \"chr7.g23884.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60427979', '60428116', '.', '+', '.', 'gene_id \"chr7.g23885\"; transcript_id \"chr7.g23885.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60428139', '60428548', '.', '-', '.', 'gene_id \"chr7.g23886\"; transcript_id \"chr7.g23886.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60460504', '60460617', '.', '+', '.', 'gene_id \"chr7.g23904\"; transcript_id \"chr7.g23904.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60632188', '60632353', '.', '+', '.', 'gene_id \"chr7.g23904\"; transcript_id \"chr7.g23904.i4\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60642798', '60644198', '.', '+', '.', 'gene_id \"chr7.g23904\"; transcript_id \"chr7.g23904.i5\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60642415', '60644198', '.', '+', '.', 'gene_id \"chr7.g23904\"; transcript_id \"chr7.g23904.i6\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60642414', '60644198', '.', '+', '.', 'gene_id \"chr7.g23904\"; transcript_id \"chr7.g23904.i7\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24412486', '24415023', '.', '-', '.', 'gene_id \"chr6.g13358\"; transcript_id \"chr6.g13358.i12\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24412486', '24414379', '.', '-', '.', 'gene_id \"chr6.g13358\"; transcript_id \"chr6.g13358.i13\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24420647', '24420827', '.', '-', '.', 'gene_id \"chr6.g13396\"; transcript_id \"chr6.g13396.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60666372', '60666887', '.', '+', '.', 'gene_id \"chr7.g23957\"; transcript_id \"chr7.g23957.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60996016', '60996297', '.', '-', '.', 'gene_id \"chr7.g23973\"; transcript_id \"chr7.g23973.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24507961', '24508121', '.', '+', '.', 'gene_id \"chr6.g13445\"; transcript_id \"chr6.g13445.i11\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24527200', '24527880', '.', '+', '.', 'gene_id \"chr6.g13448\"; transcript_id \"chr6.g13448.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24468345', '24468566', '.', '-', '.', 'gene_id \"chr6.g13451\"; transcript_id \"chr6.g13451.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24530245', '24530561', '.', '+', '.', 'gene_id \"chr6.g13459\"; transcript_id \"chr6.g13459.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24466579', '24466889', '.', '-', '.', 'gene_id \"chr6.g13470\"; transcript_id \"chr6.g13470.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '61333473', '61333698', '.', '-', '.', 'gene_id \"chr7.g24018\"; transcript_id \"chr7.g24018.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24526605', '24526814', '.', '-', '.', 'gene_id \"chr6.g13511\"; transcript_id \"chr6.g13511.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24531174', '24532148', '.', '+', '.', 'gene_id \"chr6.g13519\"; transcript_id \"chr6.g13519.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24557802', '24558104', '.', '+', '.', 'gene_id \"chr6.g13549\"; transcript_id \"chr6.g13549.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24558141', '24558299', '.', '-', '.', 'gene_id \"chr6.g13550\"; transcript_id \"chr6.g13550.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24808100', '24808357', '.', '+', '.', 'gene_id \"chr6.g13632\"; transcript_id \"chr6.g13632.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24808588', '24808850', '.', '+', '.', 'gene_id \"chr6.g13633\"; transcript_id \"chr6.g13633.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24821842', '24821895', '.', '+', '.', 'gene_id \"chr6.g13640\"; transcript_id \"chr6.g13640.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24886409', '24886653', '.', '+', '.', 'gene_id \"chr6.g13670\"; transcript_id \"chr6.g13670.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24881219', '24882330', '.', '+', '.', 'gene_id \"chr6.g13673\"; transcript_id \"chr6.g13673.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24877443', '24877621', '.', '-', '.', 'gene_id \"chr6.g13675\"; transcript_id \"chr6.g13675.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24886105', '24886345', '.', '-', '.', 'gene_id \"chr6.g13688\"; transcript_id \"chr6.g13688.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24900351', '24900543', '.', '+', '.', 'gene_id \"chr6.g13693\"; transcript_id \"chr6.g13693.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24897983', '24898265', '.', '-', '.', 'gene_id \"chr6.g13695\"; transcript_id \"chr6.g13695.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '24924526', '24924709', '.', '-', '.', 'gene_id \"chr6.g13696\"; transcript_id \"chr6.g13696.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '61881024', '61881182', '.', '+', '.', 'gene_id \"chr7.g24227\"; transcript_id \"chr7.g24227.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25455028', '25455062', '.', '-', '.', 'gene_id \"chr6.g13859\"; transcript_id \"chr6.g13859.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '62619305', '62619347', '.', '-', '.', 'gene_id \"chr7.g24486\"; transcript_id \"chr7.g24486.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '62921835', '62922045', '.', '+', '.', 'gene_id \"chr7.g24531\"; transcript_id \"chr7.g24531.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25722245', '25722480', '.', '-', '.', 'gene_id \"chr6.g14084\"; transcript_id \"chr6.g14084.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25722467', '25722627', '.', '+', '.', 'gene_id \"chr6.g14086\"; transcript_id \"chr6.g14086.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '62922424', '62922681', '.', '+', '.', 'gene_id \"chr7.g24533\"; transcript_id \"chr7.g24533.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25935845', '25936223', '.', '+', '.', 'gene_id \"chr6.g14180\"; transcript_id \"chr6.g14180.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63680679', '63680920', '.', '+', '.', 'gene_id \"chr7.g24636\"; transcript_id \"chr7.g24636.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63680947', '63681166', '.', '-', '.', 'gene_id \"chr7.g24637\"; transcript_id \"chr7.g24637.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63769140', '63769300', '.', '-', '.', 'gene_id \"chr7.g24718\"; transcript_id \"chr7.g24718.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63807972', '63808143', '.', '-', '.', 'gene_id \"chr7.g24749\"; transcript_id \"chr7.g24749.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26185320', '26185368', '.', '-', '.', 'gene_id \"chr6.g14359\"; transcript_id \"chr6.g14359.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63894787', '63894987', '.', '-', '.', 'gene_id \"chr7.g24812\"; transcript_id \"chr7.g24812.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63966448', '63966568', '.', '+', '.', 'gene_id \"chr7.g24861\"; transcript_id \"chr7.g24861.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63966385', '63966568', '.', '+', '.', 'gene_id \"chr7.g24861\"; transcript_id \"chr7.g24861.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63951857', '63953845', '.', '+', '.', 'gene_id \"chr7.g24863\"; transcript_id \"chr7.g24863.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63965547', '63966402', '.', '+', '.', 'gene_id \"chr7.g24867\"; transcript_id \"chr7.g24867.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63980825', '63981028', '.', '+', '.', 'gene_id \"chr7.g24894\"; transcript_id \"chr7.g24894.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64053854', '64054075', '.', '+', '.', 'gene_id \"chr7.g24932\"; transcript_id \"chr7.g24932.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64058492', '64058547', '.', '+', '.', 'gene_id \"chr7.g24937\"; transcript_id \"chr7.g24937.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26723285', '26723824', '.', '+', '.', 'gene_id \"chr6.g14501\"; transcript_id \"chr6.g14501.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26723837', '26724082', '.', '+', '.', 'gene_id \"chr6.g14503\"; transcript_id \"chr6.g14503.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26724514', '26725021', '.', '+', '.', 'gene_id \"chr6.g14504\"; transcript_id \"chr6.g14504.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26725575', '26726249', '.', '-', '.', 'gene_id \"chr6.g14507\"; transcript_id \"chr6.g14507.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64136256', '64136296', '.', '-', '.', 'gene_id \"chr7.g24982\"; transcript_id \"chr7.g24982.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27003020', '27003053', '.', '-', '.', 'gene_id \"chr6.g14609\"; transcript_id \"chr6.g14609.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27034716', '27035027', '.', '-', '.', 'gene_id \"chr6.g14618\"; transcript_id \"chr6.g14618.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27279053', '27279218', '.', '+', '.', 'gene_id \"chr6.g14645\"; transcript_id \"chr6.g14645.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27429954', '27430063', '.', '-', '.', 'gene_id \"chr6.g14660\"; transcript_id \"chr6.g14660.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27571411', '27571445', '.', '-', '.', 'gene_id \"chr6.g14666\"; transcript_id \"chr6.g14666.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27725916', '27726194', '.', '+', '.', 'gene_id \"chr6.g14717\"; transcript_id \"chr6.g14717.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64584641', '64584868', '.', '+', '.', 'gene_id \"chr7.g25201\"; transcript_id \"chr7.g25201.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64592852', '64593040', '.', '+', '.', 'gene_id \"chr7.g25209\"; transcript_id \"chr7.g25209.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64567346', '64568969', '.', '+', '.', 'gene_id \"chr7.g25215\"; transcript_id \"chr7.g25215.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64581776', '64582062', '.', '+', '.', 'gene_id \"chr7.g25229\"; transcript_id \"chr7.g25229.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27804842', '27805130', '.', '-', '.', 'gene_id \"chr6.g14767\"; transcript_id \"chr6.g14767.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64599754', '64599837', '.', '+', '.', 'gene_id \"chr7.g25247\"; transcript_id \"chr7.g25247.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27860434', '27860482', '.', '-', '.', 'gene_id \"chr6.g14806\"; transcript_id \"chr6.g14806.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27874808', '27874837', '.', '-', '.', 'gene_id \"chr6.g14820\"; transcript_id \"chr6.g14820.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64755883', '64756259', '.', '-', '.', 'gene_id \"chr7.g25377\"; transcript_id \"chr7.g25377.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64931558', '64931775', '.', '-', '.', 'gene_id \"chr7.g25455\"; transcript_id \"chr7.g25455.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28363963', '28363995', '.', '-', '.', 'gene_id \"chr6.g15108\"; transcript_id \"chr6.g15108.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65456465', '65456505', '.', '+', '.', 'gene_id \"chr7.g25613\"; transcript_id \"chr7.g25613.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28450658', '28450723', '.', '-', '.', 'gene_id \"chr6.g15210\"; transcript_id \"chr6.g15210.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65530454', '65530930', '.', '-', '.', 'gene_id \"chr7.g25622\"; transcript_id \"chr7.g25622.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28469178', '28469218', '.', '-', '.', 'gene_id \"chr6.g15226\"; transcript_id \"chr6.g15226.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65655766', '65656113', '.', '-', '.', 'gene_id \"chr7.g25638\"; transcript_id \"chr7.g25638.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28560110', '28562730', '.', '-', '.', 'gene_id \"chr6.g15253\"; transcript_id \"chr6.g15253.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28563775', '28564067', '.', '+', '.', 'gene_id \"chr6.g15257\"; transcript_id \"chr6.g15257.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28589756', '28589940', '.', '-', '.', 'gene_id \"chr6.g15260\"; transcript_id \"chr6.g15260.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28563042', '28563256', '.', '+', '.', 'gene_id \"chr6.g15263\"; transcript_id \"chr6.g15263.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28590522', '28590566', '.', '+', '.', 'gene_id \"chr6.g15292\"; transcript_id \"chr6.g15292.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28602235', '28602354', '.', '-', '.', 'gene_id \"chr6.g15298\"; transcript_id \"chr6.g15298.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28642145', '28642297', '.', '-', '.', 'gene_id \"chr6.g15333\"; transcript_id \"chr6.g15333.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65795211', '65795247', '.', '-', '.', 'gene_id \"chr7.g25725\"; transcript_id \"chr7.g25725.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28715324', '28715364', '.', '-', '.', 'gene_id \"chr6.g15354\"; transcript_id \"chr6.g15354.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65917498', '65917536', '.', '+', '.', 'gene_id \"chr7.g25752\"; transcript_id \"chr7.g25752.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65933891', '65933931', '.', '-', '.', 'gene_id \"chr7.g25762\"; transcript_id \"chr7.g25762.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28886470', '28886775', '.', '-', '.', 'gene_id \"chr6.g15441\"; transcript_id \"chr6.g15441.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28887323', '28887475', '.', '+', '.', 'gene_id \"chr6.g15442\"; transcript_id \"chr6.g15442.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28887559', '28887734', '.', '+', '.', 'gene_id \"chr6.g15443\"; transcript_id \"chr6.g15443.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66053373', '66053427', '.', '+', '.', 'gene_id \"chr7.g25859\"; transcript_id \"chr7.g25859.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28932219', '28932256', '.', '-', '.', 'gene_id \"chr6.g15497\"; transcript_id \"chr6.g15497.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28932838', '28933159', '.', '+', '.', 'gene_id \"chr6.g15498\"; transcript_id \"chr6.g15498.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28933143', '28933558', '.', '-', '.', 'gene_id \"chr6.g15499\"; transcript_id \"chr6.g15499.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66134426', '66134777', '.', '+', '.', 'gene_id \"chr7.g25932\"; transcript_id \"chr7.g25932.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66176373', '66176644', '.', '-', '.', 'gene_id \"chr7.g25994\"; transcript_id \"chr7.g25994.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29000553', '29000789', '.', '+', '.', 'gene_id \"chr6.g15602\"; transcript_id \"chr6.g15602.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29007513', '29007607', '.', '-', '.', 'gene_id \"chr6.g15607\"; transcript_id \"chr6.g15607.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29007584', '29007672', '.', '-', '.', 'gene_id \"chr6.g15607\"; transcript_id \"chr6.g15607.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29340750', '29340789', '.', '-', '.', 'gene_id \"chr6.g15744\"; transcript_id \"chr6.g15744.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29435546', '29435680', '.', '+', '.', 'gene_id \"chr6.g15752\"; transcript_id \"chr6.g15752.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29900070', '29900101', '.', '+', '.', 'gene_id \"chr6.g15796\"; transcript_id \"chr6.g15796.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29788970', '29789014', '.', '-', '.', 'gene_id \"chr6.g15805\"; transcript_id \"chr6.g15805.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29898606', '29898638', '.', '+', '.', 'gene_id \"chr6.g15850\"; transcript_id \"chr6.g15850.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66734543', '66734701', '.', '-', '.', 'gene_id \"chr7.g26315\"; transcript_id \"chr7.g26315.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66734542', '66734701', '.', '-', '.', 'gene_id \"chr7.g26315\"; transcript_id \"chr7.g26315.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66747318', '66747369', '.', '-', '.', 'gene_id \"chr7.g26322\"; transcript_id \"chr7.g26322.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30134182', '30134614', '.', '+', '.', 'gene_id \"chr6.g15916\"; transcript_id \"chr6.g15916.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30134638', '30134796', '.', '+', '.', 'gene_id \"chr6.g15917\"; transcript_id \"chr6.g15917.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66852898', '66852929', '.', '+', '.', 'gene_id \"chr7.g26409\"; transcript_id \"chr7.g26409.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66871790', '66872016', '.', '-', '.', 'gene_id \"chr7.g26418\"; transcript_id \"chr7.g26418.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66872037', '66872233', '.', '+', '.', 'gene_id \"chr7.g26419\"; transcript_id \"chr7.g26419.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30233565', '30233853', '.', '+', '.', 'gene_id \"chr6.g16001\"; transcript_id \"chr6.g16001.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30233888', '30234266', '.', '+', '.', 'gene_id \"chr6.g16002\"; transcript_id \"chr6.g16002.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30288296', '30288487', '.', '-', '.', 'gene_id \"chr6.g16024\"; transcript_id \"chr6.g16024.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67039533', '67039588', '.', '-', '.', 'gene_id \"chr7.g26483\"; transcript_id \"chr7.g26483.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30324136', '30324274', '.', '-', '.', 'gene_id \"chr6.g16068\"; transcript_id \"chr6.g16068.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67285582', '67285775', '.', '+', '.', 'gene_id \"chr7.g26597\"; transcript_id \"chr7.g26597.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30573444', '30573854', '.', '+', '.', 'gene_id \"chr6.g16173\"; transcript_id \"chr6.g16173.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67464823', '67464886', '.', '+', '.', 'gene_id \"chr7.g26682\"; transcript_id \"chr7.g26682.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67584992', '67585045', '.', '+', '.', 'gene_id \"chr7.g26696\"; transcript_id \"chr7.g26696.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30875498', '30875563', '.', '+', '.', 'gene_id \"chr6.g16257\"; transcript_id \"chr6.g16257.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68003805', '68003855', '.', '-', '.', 'gene_id \"chr7.g26749\"; transcript_id \"chr7.g26749.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68532945', '68532988', '.', '-', '.', 'gene_id \"chr7.g26805\"; transcript_id \"chr7.g26805.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68605489', '68605540', '.', '-', '.', 'gene_id \"chr7.g26815\"; transcript_id \"chr7.g26815.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68633716', '68633818', '.', '-', '.', 'gene_id \"chr7.g26820\"; transcript_id \"chr7.g26820.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68650714', '68651290', '.', '+', '.', 'gene_id \"chr7.g26825\"; transcript_id \"chr7.g26825.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31064836', '31065043', '.', '-', '.', 'gene_id \"chr6.g16400\"; transcript_id \"chr6.g16400.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68911052', '68911365', '.', '-', '.', 'gene_id \"chr7.g26863\"; transcript_id \"chr7.g26863.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31096966', '31097178', '.', '-', '.', 'gene_id \"chr6.g16456\"; transcript_id \"chr6.g16456.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31212502', '31212679', '.', '+', '.', 'gene_id \"chr6.g16487\"; transcript_id \"chr6.g16487.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69059459', '69059627', '.', '-', '.', 'gene_id \"chr7.g26931\"; transcript_id \"chr7.g26931.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31232442', '31232484', '.', '+', '.', 'gene_id \"chr6.g16513\"; transcript_id \"chr6.g16513.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69135997', '69136053', '.', '+', '.', 'gene_id \"chr7.g26977\"; transcript_id \"chr7.g26977.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69179933', '69180010', '.', '+', '.', 'gene_id \"chr7.g26995\"; transcript_id \"chr7.g26995.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69356363', '69356521', '.', '+', '.', 'gene_id \"chr7.g27102\"; transcript_id \"chr7.g27102.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69416005', '69416046', '.', '-', '.', 'gene_id \"chr7.g27157\"; transcript_id \"chr7.g27157.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31600855', '31601325', '.', '+', '.', 'gene_id \"chr6.g16739\"; transcript_id \"chr6.g16739.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31601359', '31601552', '.', '+', '.', 'gene_id \"chr6.g16740\"; transcript_id \"chr6.g16740.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31712173', '31712430', '.', '-', '.', 'gene_id \"chr6.g16817\"; transcript_id \"chr6.g16817.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69612432', '69612806', '.', '-', '.', 'gene_id \"chr7.g27278\"; transcript_id \"chr7.g27278.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69625468', '69625924', '.', '+', '.', 'gene_id \"chr7.g27306\"; transcript_id \"chr7.g27306.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69827981', '69828012', '.', '-', '.', 'gene_id \"chr7.g27365\"; transcript_id \"chr7.g27365.i12\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32026439', '32026721', '.', '+', '.', 'gene_id \"chr6.g17000\"; transcript_id \"chr6.g17000.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32027747', '32027985', '.', '+', '.', 'gene_id \"chr6.g17002\"; transcript_id \"chr6.g17002.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32060041', '32060120', '.', '+', '.', 'gene_id \"chr6.g17042\"; transcript_id \"chr6.g17042.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32061427', '32061576', '.', '+', '.', 'gene_id \"chr6.g17047\"; transcript_id \"chr6.g17047.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69989495', '69989687', '.', '-', '.', 'gene_id \"chr7.g27487\"; transcript_id \"chr7.g27487.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69989755', '69990251', '.', '-', '.', 'gene_id \"chr7.g27488\"; transcript_id \"chr7.g27488.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '70250856', '70251055', '.', '-', '.', 'gene_id \"chr7.g27556\"; transcript_id \"chr7.g27556.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32305207', '32305479', '.', '+', '.', 'gene_id \"chr6.g17138\"; transcript_id \"chr6.g17138.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32855296', '32855331', '.', '+', '.', 'gene_id \"chr6.g17208\"; transcript_id \"chr6.g17208.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32971495', '32971706', '.', '+', '.', 'gene_id \"chr6.g17212\"; transcript_id \"chr6.g17212.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33126911', '33127108', '.', '+', '.', 'gene_id \"chr6.g17259\"; transcript_id \"chr6.g17259.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33154294', '33154495', '.', '-', '.', 'gene_id \"chr6.g17291\"; transcript_id \"chr6.g17291.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33154813', '33155068', '.', '+', '.', 'gene_id \"chr6.g17292\"; transcript_id \"chr6.g17292.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33168054', '33168109', '.', '+', '.', 'gene_id \"chr6.g17300\"; transcript_id \"chr6.g17300.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33258977', '33259186', '.', '-', '.', 'gene_id \"chr6.g17358\"; transcript_id \"chr6.g17358.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71186644', '71186681', '.', '+', '.', 'gene_id \"chr7.g27755\"; transcript_id \"chr7.g27755.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33259227', '33259405', '.', '-', '.', 'gene_id \"chr6.g17359\"; transcript_id \"chr6.g17359.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33259462', '33259636', '.', '+', '.', 'gene_id \"chr6.g17360\"; transcript_id \"chr6.g17360.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71335526', '71335560', '.', '-', '.', 'gene_id \"chr7.g27779\"; transcript_id \"chr7.g27779.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71537254', '71537302', '.', '+', '.', 'gene_id \"chr7.g27846\"; transcript_id \"chr7.g27846.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33429295', '33430140', '.', '-', '.', 'gene_id \"chr6.g17498\"; transcript_id \"chr6.g17498.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33452506', '33452535', '.', '+', '.', 'gene_id \"chr6.g17514\"; transcript_id \"chr6.g17514.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '34518829', '34519018', '.', '-', '.', 'gene_id \"chr6.g17710\"; transcript_id \"chr6.g17710.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '72481412', '72481467', '.', '+', '.', 'gene_id \"chr7.g28136\"; transcript_id \"chr7.g28136.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '35013583', '35013678', '.', '-', '.', 'gene_id \"chr6.g17806\"; transcript_id \"chr6.g17806.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73001872', '73002038', '.', '-', '.', 'gene_id \"chr7.g28235\"; transcript_id \"chr7.g28235.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '35710004', '35710048', '.', '+', '.', 'gene_id \"chr6.g17871\"; transcript_id \"chr6.g17871.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '35784503', '35784585', '.', '+', '.', 'gene_id \"chr6.g17878\"; transcript_id \"chr6.g17878.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73268212', '73268248', '.', '+', '.', 'gene_id \"chr7.g28339\"; transcript_id \"chr7.g28339.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73321369', '73321560', '.', '-', '.', 'gene_id \"chr7.g28359\"; transcript_id \"chr7.g28359.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '36436453', '36436488', '.', '+', '.', 'gene_id \"chr6.g17983\"; transcript_id \"chr6.g17983.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73437101', '73437135', '.', '-', '.', 'gene_id \"chr7.g28386\"; transcript_id \"chr7.g28386.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '36482865', '36482898', '.', '-', '.', 'gene_id \"chr6.g17998\"; transcript_id \"chr6.g17998.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73664539', '73664697', '.', '-', '.', 'gene_id \"chr7.g28555\"; transcript_id \"chr7.g28555.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73665123', '73665319', '.', '-', '.', 'gene_id \"chr7.g28557\"; transcript_id \"chr7.g28557.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '36757449', '36757606', '.', '-', '.', 'gene_id \"chr6.g18283\"; transcript_id \"chr6.g18283.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73845108', '73845288', '.', '+', '.', 'gene_id \"chr7.g28647\"; transcript_id \"chr7.g28647.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '36840333', '36840538', '.', '+', '.', 'gene_id \"chr6.g18335\"; transcript_id \"chr6.g18335.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73949976', '73950016', '.', '-', '.', 'gene_id \"chr7.g28692\"; transcript_id \"chr7.g28692.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74044519', '74044549', '.', '-', '.', 'gene_id \"chr7.g28773\"; transcript_id \"chr7.g28773.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74207970', '74208338', '.', '-', '.', 'gene_id \"chr7.g28848\"; transcript_id \"chr7.g28848.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37056111', '37056282', '.', '-', '.', 'gene_id \"chr6.g18565\"; transcript_id \"chr6.g18565.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74252307', '74252512', '.', '+', '.', 'gene_id \"chr7.g28869\"; transcript_id \"chr7.g28869.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37074540', '37074569', '.', '+', '.', 'gene_id \"chr6.g18591\"; transcript_id \"chr6.g18591.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37099216', '37099409', '.', '+', '.', 'gene_id \"chr6.g18629\"; transcript_id \"chr6.g18629.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74397393', '74397563', '.', '+', '.', 'gene_id \"chr7.g28980\"; transcript_id \"chr7.g28980.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74397416', '74397567', '.', '+', '.', 'gene_id \"chr7.g28980\"; transcript_id \"chr7.g28980.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74540278', '74540323', '.', '+', '.', 'gene_id \"chr7.g28991\"; transcript_id \"chr7.g28991.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37167935', '37168137', '.', '-', '.', 'gene_id \"chr6.g18710\"; transcript_id \"chr6.g18710.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37222071', '37222412', '.', '+', '.', 'gene_id \"chr6.g18752\"; transcript_id \"chr6.g18752.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74748058', '74748135', '.', '-', '.', 'gene_id \"chr7.g29123\"; transcript_id \"chr7.g29123.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74914911', '74915117', '.', '+', '.', 'gene_id \"chr7.g29186\"; transcript_id \"chr7.g29186.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74918868', '74919062', '.', '+', '.', 'gene_id \"chr7.g29195\"; transcript_id \"chr7.g29195.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37584243', '37584313', '.', '-', '.', 'gene_id \"chr6.g19040\"; transcript_id \"chr6.g19040.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37606480', '37606547', '.', '+', '.', 'gene_id \"chr6.g19051\"; transcript_id \"chr6.g19051.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75161530', '75161572', '.', '-', '.', 'gene_id \"chr7.g29370\"; transcript_id \"chr7.g29370.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37767260', '37767445', '.', '+', '.', 'gene_id \"chr6.g19175\"; transcript_id \"chr6.g19175.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37766753', '37766985', '.', '-', '.', 'gene_id \"chr6.g19176\"; transcript_id \"chr6.g19176.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37759925', '37760512', '.', '+', '.', 'gene_id \"chr6.g19181\"; transcript_id \"chr6.g19181.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37762455', '37762968', '.', '+', '.', 'gene_id \"chr6.g19182\"; transcript_id \"chr6.g19182.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75319597', '75319630', '.', '-', '.', 'gene_id \"chr7.g29533\"; transcript_id \"chr7.g29533.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75372335', '75373629', '.', '-', '.', 'gene_id \"chr7.g29572\"; transcript_id \"chr7.g29572.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38033640', '38033698', '.', '+', '.', 'gene_id \"chr6.g19374\"; transcript_id \"chr6.g19374.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38100269', '38100328', '.', '-', '.', 'gene_id \"chr6.g19420\"; transcript_id \"chr6.g19420.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38133161', '38133193', '.', '-', '.', 'gene_id \"chr6.g19430\"; transcript_id \"chr6.g19430.i6\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38195038', '38195081', '.', '+', '.', 'gene_id \"chr6.g19466\"; transcript_id \"chr6.g19466.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75653774', '75653928', '.', '+', '.', 'gene_id \"chr7.g29736\"; transcript_id \"chr7.g29736.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75663600', '75663974', '.', '-', '.', 'gene_id \"chr7.g29753\"; transcript_id \"chr7.g29753.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38430857', '38432355', '.', '-', '.', 'gene_id \"chr6.g19612\"; transcript_id \"chr6.g19612.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38433431', '38433845', '.', '-', '.', 'gene_id \"chr6.g19615\"; transcript_id \"chr6.g19615.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38433846', '38434913', '.', '-', '.', 'gene_id \"chr6.g19616\"; transcript_id \"chr6.g19616.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38434899', '38435153', '.', '+', '.', 'gene_id \"chr6.g19617\"; transcript_id \"chr6.g19617.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76226965', '76227021', '.', '-', '.', 'gene_id \"chr7.g29955\"; transcript_id \"chr7.g29955.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38524460', '38524646', '.', '+', '.', 'gene_id \"chr6.g19729\"; transcript_id \"chr6.g19729.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38527806', '38528388', '.', '-', '.', 'gene_id \"chr6.g19731\"; transcript_id \"chr6.g19731.i14\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38555082', '38555287', '.', '+', '.', 'gene_id \"chr6.g19765\"; transcript_id \"chr6.g19765.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76397163', '76397229', '.', '-', '.', 'gene_id \"chr7.g30009\"; transcript_id \"chr7.g30009.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76563430', '76563468', '.', '-', '.', 'gene_id \"chr7.g30022\"; transcript_id \"chr7.g30022.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76563430', '76563494', '.', '-', '.', 'gene_id \"chr7.g30022\"; transcript_id \"chr7.g30022.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '76563430', '76563473', '.', '-', '.', 'gene_id \"chr7.g30022\"; transcript_id \"chr7.g30022.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76721488', '76721638', '.', '+', '.', 'gene_id \"chr7.g30037\"; transcript_id \"chr7.g30037.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76779659', '76780166', '.', '-', '.', 'gene_id \"chr7.g30041\"; transcript_id \"chr7.g30041.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76796204', '76797602', '.', '+', '.', 'gene_id \"chr7.g30045\"; transcript_id \"chr7.g30045.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38687067', '38687164', '.', '-', '.', 'gene_id \"chr6.g19834\"; transcript_id \"chr6.g19834.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76942311', '76942795', '.', '+', '.', 'gene_id \"chr7.g30053\"; transcript_id \"chr7.g30053.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77120534', '77120606', '.', '-', '.', 'gene_id \"chr7.g30135\"; transcript_id \"chr7.g30135.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77120485', '77120604', '.', '-', '.', 'gene_id \"chr7.g30135\"; transcript_id \"chr7.g30135.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '38802226', '38802260', '.', '+', '.', 'gene_id \"chr6.g19949\"; transcript_id \"chr6.g19949.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77137957', '77138646', '.', '-', '.', 'gene_id \"chr7.g30167\"; transcript_id \"chr7.g30167.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77160467', '77160627', '.', '+', '.', 'gene_id \"chr7.g30197\"; transcript_id \"chr7.g30197.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77206862', '77207047', '.', '+', '.', 'gene_id \"chr7.g30211\"; transcript_id \"chr7.g30211.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39271776', '39271935', '.', '-', '.', 'gene_id \"chr6.g20340\"; transcript_id \"chr6.g20340.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77595571', '77595769', '.', '-', '.', 'gene_id \"chr7.g30553\"; transcript_id \"chr7.g30553.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77595818', '77596714', '.', '-', '.', 'gene_id \"chr7.g30554\"; transcript_id \"chr7.g30554.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39360553', '39360854', '.', '-', '.', 'gene_id \"chr6.g20414\"; transcript_id \"chr6.g20414.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77651475', '77651535', '.', '+', '.', 'gene_id \"chr7.g30616\"; transcript_id \"chr7.g30616.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77672165', '77672199', '.', '-', '.', 'gene_id \"chr7.g30622\"; transcript_id \"chr7.g30622.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77857344', '77857521', '.', '-', '.', 'gene_id \"chr7.g30683\"; transcript_id \"chr7.g30683.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39592520', '39592794', '.', '-', '.', 'gene_id \"chr6.g20541\"; transcript_id \"chr6.g20541.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39593103', '39593289', '.', '+', '.', 'gene_id \"chr6.g20542\"; transcript_id \"chr6.g20542.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39593651', '39593933', '.', '+', '.', 'gene_id \"chr6.g20543\"; transcript_id \"chr6.g20543.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39593945', '39594127', '.', '+', '.', 'gene_id \"chr6.g20544\"; transcript_id \"chr6.g20544.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77986567', '77986779', '.', '-', '.', 'gene_id \"chr7.g30749\"; transcript_id \"chr7.g30749.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39746937', '39747105', '.', '+', '.', 'gene_id \"chr6.g20623\"; transcript_id \"chr6.g20623.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78112148', '78112310', '.', '-', '.', 'gene_id \"chr7.g30802\"; transcript_id \"chr7.g30802.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78137184', '78137220', '.', '+', '.', 'gene_id \"chr7.g30819\"; transcript_id \"chr7.g30819.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78139750', '78139944', '.', '+', '.', 'gene_id \"chr7.g30821\"; transcript_id \"chr7.g30821.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39919295', '39919759', '.', '+', '.', 'gene_id \"chr6.g20748\"; transcript_id \"chr6.g20748.i12\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39921392', '39921550', '.', '-', '.', 'gene_id \"chr6.g20750\"; transcript_id \"chr6.g20750.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39921117', '39921412', '.', '-', '.', 'gene_id \"chr6.g20750\"; transcript_id \"chr6.g20750.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '39926765', '39927207', '.', '+', '.', 'gene_id \"chr6.g20752\"; transcript_id \"chr6.g20752.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78420906', '78421054', '.', '+', '.', 'gene_id \"chr7.g31011\"; transcript_id \"chr7.g31011.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78421176', '78421430', '.', '+', '.', 'gene_id \"chr7.g31012\"; transcript_id \"chr7.g31012.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78739821', '78739989', '.', '-', '.', 'gene_id \"chr7.g31110\"; transcript_id \"chr7.g31110.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40118064', '40118108', '.', '-', '.', 'gene_id \"chr6.g20946\"; transcript_id \"chr6.g20946.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78812062', '78812109', '.', '+', '.', 'gene_id \"chr7.g31176\"; transcript_id \"chr7.g31176.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78834608', '78834789', '.', '+', '.', 'gene_id \"chr7.g31198\"; transcript_id \"chr7.g31198.i3\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78838496', '78838976', '.', '+', '.', 'gene_id \"chr7.g31202\"; transcript_id \"chr7.g31202.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78995089', '78995362', '.', '-', '.', 'gene_id \"chr7.g31248\"; transcript_id \"chr7.g31248.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '79013268', '79013330', '.', '+', '.', 'gene_id \"chr7.g31254\"; transcript_id \"chr7.g31254.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40355596', '40355632', '.', '-', '.', 'gene_id \"chr6.g21201\"; transcript_id \"chr6.g21201.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40359319', '40359355', '.', '-', '.', 'gene_id \"chr6.g21201\"; transcript_id \"chr6.g21201.i5\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '79374740', '79374971', '.', '+', '.', 'gene_id \"chr7.g31469\"; transcript_id \"chr7.g31469.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40603050', '40603478', '.', '+', '.', 'gene_id \"chr6.g21381\"; transcript_id \"chr6.g21381.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '79495933', '79496288', '.', '+', '.', 'gene_id \"chr7.g31555\"; transcript_id \"chr7.g31555.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40935133', '40935182', '.', '+', '.', 'gene_id \"chr6.g21529\"; transcript_id \"chr6.g21529.i21\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41073915', '41075137', '.', '+', '.', 'gene_id \"chr6.g21588\"; transcript_id \"chr6.g21588.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41075133', '41075402', '.', '+', '.', 'gene_id \"chr6.g21589\"; transcript_id \"chr6.g21589.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41106284', '41106517', '.', '+', '.', 'gene_id \"chr6.g21634\"; transcript_id \"chr6.g21634.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '79710862', '79710911', '.', '-', '.', 'gene_id \"chr7.g31714\"; transcript_id \"chr7.g31714.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41171806', '41171942', '.', '-', '.', 'gene_id \"chr6.g21664\"; transcript_id \"chr6.g21664.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41647297', '41647658', '.', '-', '.', 'gene_id \"chr6.g21741\"; transcript_id \"chr6.g21741.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41746558', '41746722', '.', '-', '.', 'gene_id \"chr6.g21856\"; transcript_id \"chr6.g21856.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41746733', '41746942', '.', '-', '.', 'gene_id \"chr6.g21857\"; transcript_id \"chr6.g21857.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41758092', '41758635', '.', '+', '.', 'gene_id \"chr6.g21865\"; transcript_id \"chr6.g21865.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41759625', '41759878', '.', '+', '.', 'gene_id \"chr6.g21867\"; transcript_id \"chr6.g21867.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80174903', '80175016', '.', '+', '.', 'gene_id \"chr7.g31930\"; transcript_id \"chr7.g31930.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80194598', '80194774', '.', '+', '.', 'gene_id \"chr7.g31936\"; transcript_id \"chr7.g31936.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80194814', '80195043', '.', '-', '.', 'gene_id \"chr7.g31937\"; transcript_id \"chr7.g31937.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80378573', '80378626', '.', '-', '.', 'gene_id \"chr7.g32027\"; transcript_id \"chr7.g32027.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41948513', '41948689', '.', '+', '.', 'gene_id \"chr6.g22021\"; transcript_id \"chr6.g22021.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41991630', '41991833', '.', '-', '.', 'gene_id \"chr6.g22070\"; transcript_id \"chr6.g22070.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42121566', '42121629', '.', '+', '.', 'gene_id \"chr6.g22169\"; transcript_id \"chr6.g22169.i1\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80741888', '80742413', '.', '+', '.', 'gene_id \"chr7.g32137\"; transcript_id \"chr7.g32137.i2\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80406971', '80407112', '100', '+', '.', 'gene_id \"chr7.g32173\"; transcript_id \"blat.proc28144.chain_15453\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17739703', '17739837', '100', '+', '.', 'gene_id \"chr7.g32174\"; transcript_id \"blat.proc4587.chain_647\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52898446', '52898533', '100', '+', '.', 'gene_id \"chr7.g32207\"; transcript_id \"blat.proc18128.chain_18590\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67888777', '67888823', '100', '-', '.', 'gene_id \"chr7.g32236\"; transcript_id \"blat.proc4005.chain_2537\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '621317', '621441', '100', '-', '.', 'gene_id \"chr7.g32240\"; transcript_id \"blat.proc18128.chain_24577\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42372940', '42373159', '100', '-', '.', 'gene_id \"chr7.g32255\"; transcript_id \"blat.proc18128.chain_3635\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14324849', '14324908', '100', '-', '.', 'gene_id \"chr7.g32284\"; transcript_id \"blat.proc18128.chain_15132\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34647217', '34647249', '100', '-', '.', 'gene_id \"chr7.g32321\"; transcript_id \"blat.proc4005.chain_18736\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26381862', '26381897', '100', '-', '.', 'gene_id \"chr7.g32337\"; transcript_id \"blat.proc28144.chain_4158\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25398246', '25398290', '100', '+', '.', 'gene_id \"chr7.g32423\"; transcript_id \"blat.proc4005.chain_19087\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37441199', '37441255', '100', '-', '.', 'gene_id \"chr7.g32512\"; transcript_id \"blat.proc4587.chain_4513\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66941758', '66941910', '100', '-', '.', 'gene_id \"chr7.g32543\"; transcript_id \"blat.proc18128.chain_23574\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '58127034', '58127098', '100', '-', '.', 'gene_id \"chr7.g32632\"; transcript_id \"blat.proc28144.chain_10096\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55955443', '55955473', '100', '+', '.', 'gene_id \"chr7.g32893\"; transcript_id \"blat.proc4005.chain_15727\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57701908', '57701941', '100', '-', '.', 'gene_id \"chr7.g32908\"; transcript_id \"blat.proc4005.chain_14615\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21684297', '21684353', '100', '+', '.', 'gene_id \"chr7.g32983\"; transcript_id \"blat.proc4587.chain_11793\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '10359255', '10359288', '100', '+', '.', 'gene_id \"chr7.g33046\"; transcript_id \"blat.proc4005.chain_4984\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27990642', '27990688', '100', '+', '.', 'gene_id \"chr7.g33140\"; transcript_id \"blat.proc4005.chain_6169\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24223403', '24223442', '100', '-', '.', 'gene_id \"chr7.g33141\"; transcript_id \"blat.proc18128.chain_7327\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29880036', '29880072', '100', '+', '.', 'gene_id \"chr7.g33250\"; transcript_id \"blat.proc18128.chain_1441\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38630972', '38631030', '100', '-', '.', 'gene_id \"chr7.g33291\"; transcript_id \"blat.proc18128.chain_2533\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38828055', '38828116', '100', '+', '.', 'gene_id \"chr7.g33318\"; transcript_id \"blat.proc4005.chain_19439\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77002931', '77002996', '100', '+', '.', 'gene_id \"chr7.g33329\"; transcript_id \"blat.proc4005.chain_5737\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48755584', '48755624', '100', '+', '.', 'gene_id \"chr7.g33406\"; transcript_id \"blat.proc28144.chain_14897\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '55529957', '55529986', '100', '-', '.', 'gene_id \"chr7.g33417\"; transcript_id \"blat.proc4005.chain_1605\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23033151', '23033220', '100', '+', '.', 'gene_id \"chr7.g33453\"; transcript_id \"blat.proc4587.chain_7461\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12984691', '12984831', '100', '+', '.', 'gene_id \"chr7.g33477\"; transcript_id \"blat.proc18128.chain_8484\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9188162', '9188209', '100', '+', '.', 'gene_id \"chr7.g33598\"; transcript_id \"blat.proc4005.chain_2027\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30553911', '30553956', '100', '+', '.', 'gene_id \"chr7.g33609\"; transcript_id \"blat.proc4005.chain_19894\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2934666', '2934770', '100', '-', '.', 'gene_id \"chr7.g33622\"; transcript_id \"blat.proc4587.chain_629\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74566262', '74566327', '100', '-', '.', 'gene_id \"chr7.g33731\"; transcript_id \"blat.proc4005.chain_21624\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42908550', '42908591', '100', '-', '.', 'gene_id \"chr7.g33828\"; transcript_id \"blat.proc18128.chain_5188\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '77655147', '77655201', '100', '+', '.', 'gene_id \"chr7.g33877\"; transcript_id \"blat.proc18128.chain_12874\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '6059975', '6060032', '100', '-', '.', 'gene_id \"chr7.g33891\"; transcript_id \"blat.proc4587.chain_5526\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37459169', '37459221', '100', '+', '.', 'gene_id \"chr7.g33915\"; transcript_id \"blat.proc4587.chain_1673\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9094264', '9094543', '100', '+', '.', 'gene_id \"chr7.g34099\"; transcript_id \"blat.proc28144.chain_6968\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '12192929', '12192974', '100', '-', '.', 'gene_id \"chr7.g34132\"; transcript_id \"blat.proc4587.chain_12048\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '42403112', '42403146', '100', '+', '.', 'gene_id \"chr7.g34142\"; transcript_id \"blat.proc4005.chain_19138\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63568679', '63568726', '100', '-', '.', 'gene_id \"chr7.g34155\"; transcript_id \"blat.proc4587.chain_10363\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22585804', '22585839', '100', '+', '.', 'gene_id \"chr7.g34205\"; transcript_id \"blat.proc18128.chain_18957\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48707662', '48707694', '100', '-', '.', 'gene_id \"chr7.g34254\"; transcript_id \"blat.proc28144.chain_15902\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73609954', '73609992', '100', '+', '.', 'gene_id \"chr7.g34255\"; transcript_id \"blat.proc18128.chain_7908\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '27606683', '27606909', '100', '+', '.', 'gene_id \"chr7.g34258\"; transcript_id \"blat.proc18128.chain_21894\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '686111', '686179', '100', '-', '.', 'gene_id \"chr7.g34288\"; transcript_id \"blat.proc28144.chain_15307\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '63806315', '63806348', '100', '-', '.', 'gene_id \"chr7.g34295\"; transcript_id \"blat.proc18128.chain_24674\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69086968', '69087010', '100', '+', '.', 'gene_id \"chr7.g34358\"; transcript_id \"blat.proc4005.chain_3764\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75774348', '75774386', '100', '-', '.', 'gene_id \"chr7.g34460\"; transcript_id \"blat.proc4005.chain_20987\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71894508', '71894572', '100', '+', '.', 'gene_id \"chr7.g34478\"; transcript_id \"blat.proc28144.chain_16239\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '31011915', '31011958', '100', '-', '.', 'gene_id \"chr7.g34624\"; transcript_id \"blat.proc4005.chain_4266\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73865221', '73865356', '100', '-', '.', 'gene_id \"chr7.g34640\"; transcript_id \"blat.proc28144.chain_8587\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2319038', '2319094', '100', '-', '.', 'gene_id \"chr7.g34642\"; transcript_id \"blat.proc4005.chain_16625\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25269123', '25269171', '100', '+', '.', 'gene_id \"chr7.g34683\"; transcript_id \"blat.proc28144.chain_6044\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '73639885', '73639919', '100', '+', '.', 'gene_id \"chr7.g34759\"; transcript_id \"blat.proc18128.chain_6977\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9757476', '9757615', '100', '+', '.', 'gene_id \"chr7.g34872\"; transcript_id \"blat.proc18128.chain_18574\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74904427', '74904502', '100', '+', '.', 'gene_id \"chr7.g34898\"; transcript_id \"blat.proc4587.chain_244\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '80511794', '80511838', '100', '+', '.', 'gene_id \"chr7.g34933\"; transcript_id \"blat.proc4587.chain_4070\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3825399', '3825533', '100', '-', '.', 'gene_id \"chr7.g34961\"; transcript_id \"blat.proc4587.chain_4906\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64677891', '64677925', '100', '-', '.', 'gene_id \"chr7.g35014\"; transcript_id \"blat.proc28144.chain_21130\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '62168404', '62168479', '100', '+', '.', 'gene_id \"chr7.g35061\"; transcript_id \"blat.proc4587.chain_1297\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75344061', '75344103', '100', '+', '.', 'gene_id \"chr7.g35080\"; transcript_id \"blat.proc4005.chain_3296\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '59682667', '59682712', '100', '-', '.', 'gene_id \"chr7.g35095\"; transcript_id \"blat.proc28144.chain_5375\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65768130', '65768188', '100', '+', '.', 'gene_id \"chr7.g35120\"; transcript_id \"blat.proc4587.chain_6915\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '23053096', '23053142', '100', '+', '.', 'gene_id \"chr7.g35139\"; transcript_id \"blat.proc28144.chain_18160\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68197051', '68197082', '100', '+', '.', 'gene_id \"chr7.g35141\"; transcript_id \"blat.proc28144.chain_20339\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '45408298', '45408342', '100', '+', '.', 'gene_id \"chr7.g35163\"; transcript_id \"blat.proc4005.chain_18339\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '25799224', '25799267', '100', '+', '.', 'gene_id \"chr7.g35226\"; transcript_id \"blat.proc4005.chain_10414\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '50756637', '50756783', '100', '+', '.', 'gene_id \"chr7.g35252\"; transcript_id \"blat.proc28144.chain_15652\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71927711', '71927753', '100', '+', '.', 'gene_id \"chr7.g35623\"; transcript_id \"blat.proc4587.chain_7772\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41565780', '41565830', '100', '+', '.', 'gene_id \"chr7.g35715\"; transcript_id \"blat.proc28144.chain_15541\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '612982', '613073', '100', '+', '.', 'gene_id \"chr7.g35761\"; transcript_id \"blat.proc18128.chain_8998\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '3303994', '3304034', '100', '+', '.', 'gene_id \"chr7.g35791\"; transcript_id \"blat.proc28144.chain_4496\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53347185', '53347296', '100', '-', '.', 'gene_id \"chr7.g35871\"; transcript_id \"blat.proc28144.chain_4621\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69857394', '69857465', '100', '+', '.', 'gene_id \"chr7.g35966\"; transcript_id \"blat.proc4587.chain_4176\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21390622', '21390655', '100', '-', '.', 'gene_id \"chr7.g35991\"; transcript_id \"blat.proc18128.chain_8263\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4608774', '4608813', '100', '+', '.', 'gene_id \"chr7.g35993\"; transcript_id \"blat.proc18128.chain_7722\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '14265764', '14265938', '100', '-', '.', 'gene_id \"chr7.g36125\"; transcript_id \"blat.proc28144.chain_10237\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '59341867', '59341932', '100', '+', '.', 'gene_id \"chr7.g36134\"; transcript_id \"blat.proc4005.chain_17162\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '72584132', '72584164', '100', '-', '.', 'gene_id \"chr7.g36253\"; transcript_id \"blat.proc4587.chain_10232\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8858750', '8858789', '100', '-', '.', 'gene_id \"chr7.g36293\"; transcript_id \"blat.proc4587.chain_9566\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '72153670', '72153735', '100', '+', '.', 'gene_id \"chr7.g36310\"; transcript_id \"blat.proc4005.chain_5736\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17240020', '17240049', '100', '+', '.', 'gene_id \"chr7.g36385\"; transcript_id \"blat.proc4005.chain_22402\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '30855784', '30855818', '100', '+', '.', 'gene_id \"chr7.g36406\"; transcript_id \"blat.proc28144.chain_11670\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '60978351', '60978389', '100', '+', '.', 'gene_id \"chr7.g36431\"; transcript_id \"blat.proc4005.chain_16921\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41864496', '41864549', '100', '+', '.', 'gene_id \"chr7.g36432\"; transcript_id \"blat.proc28144.chain_15540\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5113419', '5113557', '100', '+', '.', 'gene_id \"chr7.g36439\"; transcript_id \"blat.proc28144.chain_16072\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '47050912', '47050955', '100', '-', '.', 'gene_id \"chr7.g36500\"; transcript_id \"blat.proc28144.chain_8820\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '76394145', '76394186', '100', '-', '.', 'gene_id \"chr7.g36517\"; transcript_id \"blat.proc18128.chain_3011\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '7944769', '7945315', '100', '-', '.', 'gene_id \"chr7.g36541\"; transcript_id \"blat.proc18128.chain_6346\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53284903', '53284943', '100', '-', '.', 'gene_id \"chr7.g36568\"; transcript_id \"blat.proc28144.chain_5566\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53023756', '53023822', '100', '+', '.', 'gene_id \"chr7.g36570\"; transcript_id \"blat.proc4587.chain_11488\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '22961046', '22961095', '100', '-', '.', 'gene_id \"chr7.g36575\"; transcript_id \"blat.proc28144.chain_8445\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '70372046', '70372088', '100', '+', '.', 'gene_id \"chr7.g36589\"; transcript_id \"blat.proc4587.chain_1380\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64594072', '64594102', '100', '+', '.', 'gene_id \"chr7.g36602\"; transcript_id \"blat.proc18128.chain_13805\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '51420810', '51420841', '100', '+', '.', 'gene_id \"chr7.g36668\"; transcript_id \"blat.proc4587.chain_4035\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '46555266', '46555317', '100', '-', '.', 'gene_id \"chr7.g36698\"; transcript_id \"blat.proc18128.chain_10486\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75540779', '75540823', '100', '-', '.', 'gene_id \"chr7.g36739\"; transcript_id \"blat.proc28144.chain_23141\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '29943246', '29943287', '100', '-', '.', 'gene_id \"chr7.g36983\"; transcript_id \"blat.proc4587.chain_2434\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '36407168', '36407321', '100', '+', '.', 'gene_id \"chr7.g37044\"; transcript_id \"blat.proc28144.chain_21237\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '16034854', '16034888', '100', '+', '.', 'gene_id \"chr7.g37104\"; transcript_id \"blat.proc28144.chain_11671\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '69877860', '69878011', '100', '-', '.', 'gene_id \"chr7.g37149\"; transcript_id \"blat.proc18128.chain_8025\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '38854893', '38854934', '100', '-', '.', 'gene_id \"chr7.g37207\"; transcript_id \"blat.proc18128.chain_13566\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54322524', '54322564', '100', '-', '.', 'gene_id \"chr7.g37306\"; transcript_id \"blat.proc4005.chain_16926\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52174529', '52174580', '100', '+', '.', 'gene_id \"chr7.g37365\"; transcript_id \"blat.proc28144.chain_13414\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '37604018', '37604056', '100', '+', '.', 'gene_id \"chr7.g37372\"; transcript_id \"blat.proc18128.chain_1577\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64734262', '64734320', '100', '-', '.', 'gene_id \"chr7.g37418\"; transcript_id \"blat.proc4587.chain_543\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '49079903', '49079932', '100', '-', '.', 'gene_id \"chr7.g37477\"; transcript_id \"blat.proc18128.chain_11771\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24595094', '24595146', '100', '+', '.', 'gene_id \"chr7.g37559\"; transcript_id \"blat.proc28144.chain_4080\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '192670', '192741', '100', '-', '.', 'gene_id \"chr7.g37581\"; transcript_id \"blat.proc4587.chain_7219\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '52783672', '52783731', '100', '-', '.', 'gene_id \"chr7.g37842\"; transcript_id \"blat.proc4587.chain_3091\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39191973', '39192025', '100', '+', '.', 'gene_id \"chr7.g37865\"; transcript_id \"blat.proc28144.chain_4604\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67050261', '67050375', '100', '-', '.', 'gene_id \"chr7.g37934\"; transcript_id \"blat.proc4587.chain_13145\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '61794411', '61794503', '100', '+', '.', 'gene_id \"chr7.g37971\"; transcript_id \"blat.proc4587.chain_10252\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26722479', '26722526', '100', '-', '.', 'gene_id \"chr7.g38001\"; transcript_id \"blat.proc18128.chain_5861\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '70725365', '70725408', '100', '+', '.', 'gene_id \"chr7.g38030\"; transcript_id \"blat.proc4005.chain_8018\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67380577', '67380617', '100', '+', '.', 'gene_id \"chr7.g38032\"; transcript_id \"blat.proc28144.chain_5563\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '2003952', '2003982', '100', '+', '.', 'gene_id \"chr7.g38133\"; transcript_id \"blat.proc28144.chain_18849\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '26554011', '26554290', '100', '-', '.', 'gene_id \"chr7.g38183\"; transcript_id \"blat.proc28144.chain_21116\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53873760', '53873789', '100', '-', '.', 'gene_id \"chr7.g38311\"; transcript_id \"blat.proc28144.chain_19925\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '21733509', '21733557', '100', '+', '.', 'gene_id \"chr7.g38531\"; transcript_id \"blat.proc18128.chain_20302\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '56398155', '56398212', '100', '+', '.', 'gene_id \"chr7.g38569\"; transcript_id \"blat.proc18128.chain_17060\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '40309313', '40309347', '100', '+', '.', 'gene_id \"chr7.g38581\"; transcript_id \"blat.proc18128.chain_2842\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65921530', '65921596', '100', '+', '.', 'gene_id \"chr7.g38595\"; transcript_id \"blat.proc4005.chain_4697\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65740986', '65741029', '100', '+', '.', 'gene_id \"chr7.g38642\"; transcript_id \"blat.proc4005.chain_21679\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34331403', '34331444', '100', '-', '.', 'gene_id \"chr7.g38828\"; transcript_id \"blat.proc28144.chain_9135\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '39628261', '39628380', '100', '+', '.', 'gene_id \"chr7.g38931\"; transcript_id \"blat.proc18128.chain_24129\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '78454344', '78454377', '100', '+', '.', 'gene_id \"chr7.g38971\"; transcript_id \"blat.proc18128.chain_18229\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '68910001', '68910040', '100', '+', '.', 'gene_id \"chr7.g39034\"; transcript_id \"blat.proc4005.chain_5402\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64099761', '64099862', '100', '-', '.', 'gene_id \"chr7.g39041\"; transcript_id \"blat.proc28144.chain_11453\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '8925223', '8925268', '100', '-', '.', 'gene_id \"chr7.g39082\"; transcript_id \"blat.proc28144.chain_17136\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '34266180', '34266223', '100', '-', '.', 'gene_id \"chr7.g39224\"; transcript_id \"blat.proc18128.chain_21160\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32965064', '32965110', '100', '+', '.', 'gene_id \"chr7.g39258\"; transcript_id \"blat.proc4587.chain_7356\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35633716', '35633802', '100', '-', '.', 'gene_id \"chr7.g39316\"; transcript_id \"blat.proc18128.chain_7563\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '75206110', '75206209', '100', '-', '.', 'gene_id \"chr7.g39362\"; transcript_id \"blat.proc28144.chain_13534\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32750146', '32750180', '100', '-', '.', 'gene_id \"chr7.g39372\"; transcript_id \"blat.proc28144.chain_19685\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '32003196', '32003363', '100', '-', '.', 'gene_id \"chr7.g39380\"; transcript_id \"blat.proc4587.chain_8489\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35343866', '35343906', '100', '+', '.', 'gene_id \"chr7.g39385\"; transcript_id \"blat.proc18128.chain_1556\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '17979882', '17979933', '100', '+', '.', 'gene_id \"chr7.g39501\"; transcript_id \"blat.proc18128.chain_23539\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66827129', '66827178', '100', '+', '.', 'gene_id \"chr7.g39515\"; transcript_id \"blat.proc18128.chain_24279\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '62394883', '62394998', '100', '-', '.', 'gene_id \"chr7.g39592\"; transcript_id \"blat.proc4587.chain_235\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '57548027', '57548056', '100', '-', '.', 'gene_id \"chr7.g39646\"; transcript_id \"blat.proc28144.chain_18809\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13004276', '13004310', '100', '-', '.', 'gene_id \"chr7.g39662\"; transcript_id \"blat.proc4005.chain_3776\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '18196927', '18196970', '100', '+', '.', 'gene_id \"chr7.g39754\"; transcript_id \"blat.proc28144.chain_16897\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '53610043', '53610073', '100', '+', '.', 'gene_id \"chr7.g39763\"; transcript_id \"blat.proc18128.chain_20415\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '35252594', '35252750', '100', '+', '.', 'gene_id \"chr7.g39804\"; transcript_id \"blat.proc4005.chain_21010\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '20612945', '20613012', '100', '-', '.', 'gene_id \"chr7.g39858\"; transcript_id \"blat.proc28144.chain_5042\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '4861275', '4861329', '100', '-', '.', 'gene_id \"chr7.g40006\"; transcript_id \"blat.proc28144.chain_22362\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9263513', '9263555', '100', '-', '.', 'gene_id \"chr7.g40079\"; transcript_id \"blat.proc18128.chain_3738\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41623532', '41623626', '100', '-', '.', 'gene_id \"chr7.g40093\"; transcript_id \"blat.proc18128.chain_16197\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr7', 'PASA', 'exon', '16151600', '16151654', '100', '+', '.', 'gene_id \"chr7.g40143\"; transcript_id \"blat.proc18128.chain_10281\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '67360732', '67360783', '100', '-', '.', 'gene_id \"chr7.g40152\"; transcript_id \"blat.proc4005.chain_60\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '48329865', '48329898', '100', '+', '.', 'gene_id \"chr7.g40179\"; transcript_id \"blat.proc28144.chain_7691\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '70898748', '70899267', '100', '+', '.', 'gene_id \"chr7.g40246\"; transcript_id \"blat.proc18128.chain_6159\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '9777482', '9777521', '100', '+', '.', 'gene_id \"chr7.g40254\"; transcript_id \"blat.proc18128.chain_5886\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '13631353', '13631399', '100', '+', '.', 'gene_id \"chr7.g40354\"; transcript_id \"blat.proc18128.chain_11763\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '65570714', '65570784', '100', '+', '.', 'gene_id \"chr7.g40366\"; transcript_id \"blat.proc4005.chain_20388\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '1775916', '1775992', '100', '+', '.', 'gene_id \"chr7.g40404\"; transcript_id \"blat.proc4005.chain_13968\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '5990986', '5991023', '100', '-', '.', 'gene_id \"chr7.g40416\"; transcript_id \"blat.proc18128.chain_4507\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '64098755', '64098788', '100', '-', '.', 'gene_id \"chr7.g40429\"; transcript_id \"blat.proc18128.chain_2341\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '66445806', '66445857', '100', '-', '.', 'gene_id \"chr7.g40448\"; transcript_id \"blat.proc4587.chain_11756\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '74711868', '74711912', '100', '+', '.', 'gene_id \"chr7.g40581\"; transcript_id \"blat.proc28144.chain_4555\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '54364985', '54365024', '100', '+', '.', 'gene_id \"chr7.g40644\"; transcript_id \"blat.proc4005.chain_14221\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '71150123', '71150262', '100', '+', '.', 'gene_id \"chr7.g40706\"; transcript_id \"blat.proc18128.chain_25934\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '24256437', '24256475', '100', '+', '.', 'gene_id \"chr7.g40747\"; transcript_id \"blat.proc28144.chain_12819\";']\n",
      "negative:  ['chr7', 'PASA', 'exon', '41336970', '41337097', '100', '-', '.', 'gene_id \"chr7.g40790\"; transcript_id \"blat.proc4587.chain_2370\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42514780', '42514944', '.', '+', '.', 'gene_id \"chr6.g22444\"; transcript_id \"chr6.g22444.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42535937', '42536125', '.', '+', '.', 'gene_id \"chr6.g22492\"; transcript_id \"chr6.g22492.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42613503', '42613560', '.', '-', '.', 'gene_id \"chr6.g22579\"; transcript_id \"chr6.g22579.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42663787', '42663819', '.', '-', '.', 'gene_id \"chr6.g22617\"; transcript_id \"chr6.g22617.i1\";']\n",
      "negative:  ['scaffold_1474', 'PASA', 'exon', '8715', '8869', '100', '+', '.', 'gene_id \"scaffold_1474.g13\"; transcript_id \"blat.proc28144.chain_18766\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42694806', '42695023', '.', '+', '.', 'gene_id \"chr6.g22651\"; transcript_id \"chr6.g22651.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42705731', '42705795', '.', '+', '.', 'gene_id \"chr6.g22655\"; transcript_id \"chr6.g22655.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42802451', '42802679', '.', '-', '.', 'gene_id \"chr6.g22753\"; transcript_id \"chr6.g22753.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42802732', '42802902', '.', '-', '.', 'gene_id \"chr6.g22755\"; transcript_id \"chr6.g22755.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42895178', '42895551', '.', '+', '.', 'gene_id \"chr6.g22823\"; transcript_id \"chr6.g22823.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42912984', '42913018', '.', '+', '.', 'gene_id \"chr6.g22838\"; transcript_id \"chr6.g22838.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42914193', '42914225', '.', '+', '.', 'gene_id \"chr6.g22841\"; transcript_id \"chr6.g22841.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43004274', '43004321', '.', '-', '.', 'gene_id \"chr6.g22880\"; transcript_id \"chr6.g22880.i1\";']\n",
      "negative:  ['scaffold_789', 'PASA', 'exon', '4380', '4547', '100', '+', '.', 'gene_id \"scaffold_789.g4\"; transcript_id \"blat.proc18128.chain_25499\";']\n",
      "negative:  ['scaffold_227', 'PASA', 'exon', '40557', '40600', '100', '+', '.', 'gene_id \"scaffold_227.g55\"; transcript_id \"blat.proc28144.chain_6557\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43550268', '43550311', '.', '-', '.', 'gene_id \"chr6.g23169\"; transcript_id \"chr6.g23169.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43766598', '43766893', '.', '+', '.', 'gene_id \"chr6.g23212\"; transcript_id \"chr6.g23212.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43922462', '43922809', '.', '+', '.', 'gene_id \"chr6.g23273\"; transcript_id \"chr6.g23273.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43923493', '43923746', '.', '+', '.', 'gene_id \"chr6.g23276\"; transcript_id \"chr6.g23276.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43923734', '43924103', '.', '-', '.', 'gene_id \"chr6.g23277\"; transcript_id \"chr6.g23277.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '44066401', '44066551', '.', '+', '.', 'gene_id \"chr6.g23380\"; transcript_id \"chr6.g23380.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '588859', '588901', '.', '+', '.', 'gene_id \"chr30.g53\"; transcript_id \"chr30.g53.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '799873', '800108', '.', '-', '.', 'gene_id \"chr30.g94\"; transcript_id \"chr30.g94.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '800263', '800519', '.', '-', '.', 'gene_id \"chr30.g98\"; transcript_id \"chr30.g98.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '44242415', '44242596', '.', '+', '.', 'gene_id \"chr6.g23462\"; transcript_id \"chr6.g23462.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '44241178', '44241390', '.', '+', '.', 'gene_id \"chr6.g23465\"; transcript_id \"chr6.g23465.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '889296', '889380', '.', '+', '.', 'gene_id \"chr30.g198\"; transcript_id \"chr30.g198.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '916432', '916752', '.', '-', '.', 'gene_id \"chr30.g219\"; transcript_id \"chr30.g219.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '935165', '935347', '.', '+', '.', 'gene_id \"chr30.g249\"; transcript_id \"chr30.g249.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '935538', '935694', '.', '-', '.', 'gene_id \"chr30.g251\"; transcript_id \"chr30.g251.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '959409', '959717', '.', '-', '.', 'gene_id \"chr30.g288\"; transcript_id \"chr30.g288.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45023325', '45023383', '.', '+', '.', 'gene_id \"chr6.g23637\"; transcript_id \"chr6.g23637.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1043879', '1044056', '.', '-', '.', 'gene_id \"chr30.g373\"; transcript_id \"chr30.g373.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1044040', '1044431', '.', '+', '.', 'gene_id \"chr30.g374\"; transcript_id \"chr30.g374.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45554925', '45554957', '.', '-', '.', 'gene_id \"chr6.g23679\"; transcript_id \"chr6.g23679.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1053152', '1053196', '.', '-', '.', 'gene_id \"chr30.g408\"; transcript_id \"chr30.g408.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45959849', '45959882', '.', '-', '.', 'gene_id \"chr6.g23701\"; transcript_id \"chr6.g23701.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '46364420', '46364485', '.', '-', '.', 'gene_id \"chr6.g23726\"; transcript_id \"chr6.g23726.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '46956837', '46956992', '.', '-', '.', 'gene_id \"chr6.g23753\"; transcript_id \"chr6.g23753.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '47007966', '47008481', '.', '+', '.', 'gene_id \"chr6.g23771\"; transcript_id \"chr6.g23771.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '47038837', '47038881', '.', '-', '.', 'gene_id \"chr6.g23797\"; transcript_id \"chr6.g23797.i9\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1365562', '1365602', '.', '-', '.', 'gene_id \"chr30.g568\"; transcript_id \"chr30.g568.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '47478637', '47478685', '.', '-', '.', 'gene_id \"chr6.g23922\"; transcript_id \"chr6.g23922.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '47575299', '47575455', '.', '-', '.', 'gene_id \"chr6.g23942\"; transcript_id \"chr6.g23942.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '2060856', '2061016', '.', '+', '.', 'gene_id \"chr30.g752\"; transcript_id \"chr30.g752.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '48971965', '48972240', '.', '-', '.', 'gene_id \"chr6.g24089\"; transcript_id \"chr6.g24089.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '49174253', '49174283', '.', '+', '.', 'gene_id \"chr6.g24260\"; transcript_id \"chr6.g24260.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '49227876', '49228131', '.', '-', '.', 'gene_id \"chr6.g24318\"; transcript_id \"chr6.g24318.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '2839046', '2839101', '.', '-', '.', 'gene_id \"chr30.g1071\"; transcript_id \"chr30.g1071.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '2979525', '2979572', '.', '+', '.', 'gene_id \"chr30.g1120\"; transcript_id \"chr30.g1120.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '49729808', '49729935', '.', '+', '.', 'gene_id \"chr6.g24524\"; transcript_id \"chr6.g24524.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '3264141', '3264186', '.', '-', '.', 'gene_id \"chr30.g1254\"; transcript_id \"chr30.g1254.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '49875378', '49876785', '.', '+', '.', 'gene_id \"chr6.g24631\"; transcript_id \"chr6.g24631.i5\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '4238852', '4238909', '.', '-', '.', 'gene_id \"chr30.g1351\"; transcript_id \"chr30.g1351.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '4345359', '4345563', '.', '-', '.', 'gene_id \"chr30.g1417\"; transcript_id \"chr30.g1417.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '4345623', '4346016', '.', '-', '.', 'gene_id \"chr30.g1418\"; transcript_id \"chr30.g1418.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50019683', '50019716', '.', '-', '.', 'gene_id \"chr6.g24766\"; transcript_id \"chr6.g24766.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50104430', '50104814', '.', '+', '.', 'gene_id \"chr6.g24831\"; transcript_id \"chr6.g24831.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50104843', '50105072', '.', '-', '.', 'gene_id \"chr6.g24833\"; transcript_id \"chr6.g24833.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50143821', '50144146', '.', '+', '.', 'gene_id \"chr6.g24869\"; transcript_id \"chr6.g24869.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '4745551', '4745583', '.', '-', '.', 'gene_id \"chr30.g1635\"; transcript_id \"chr30.g1635.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50303502', '50303842', '.', '-', '.', 'gene_id \"chr6.g24988\"; transcript_id \"chr6.g24988.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50303911', '50304062', '.', '+', '.', 'gene_id \"chr6.g24989\"; transcript_id \"chr6.g24989.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '5122360', '5122409', '.', '-', '.', 'gene_id \"chr30.g1672\"; transcript_id \"chr30.g1672.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '5430022', '5430297', '.', '+', '.', 'gene_id \"chr30.g1716\"; transcript_id \"chr30.g1716.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '50637980', '50638151', '.', '+', '.', 'gene_id \"chr6.g25101\"; transcript_id \"chr6.g25101.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '5795241', '5795453', '.', '+', '.', 'gene_id \"chr30.g1861\"; transcript_id \"chr30.g1861.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51779383', '51779897', '.', '-', '.', 'gene_id \"chr6.g25312\"; transcript_id \"chr6.g25312.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51779023', '51779272', '.', '-', '.', 'gene_id \"chr6.g25315\"; transcript_id \"chr6.g25315.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51804466', '51804535', '.', '-', '.', 'gene_id \"chr6.g25345\"; transcript_id \"chr6.g25345.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51829196', '51829399', '.', '-', '.', 'gene_id \"chr6.g25399\"; transcript_id \"chr6.g25399.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6648204', '6648433', '.', '-', '.', 'gene_id \"chr30.g2097\"; transcript_id \"chr30.g2097.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6826639', '6826957', '.', '+', '.', 'gene_id \"chr30.g2138\"; transcript_id \"chr30.g2138.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6825545', '6826514', '.', '+', '.', 'gene_id \"chr30.g2140\"; transcript_id \"chr30.g2140.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6820966', '6821156', '.', '+', '.', 'gene_id \"chr30.g2145\"; transcript_id \"chr30.g2145.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6821726', '6822218', '.', '-', '.', 'gene_id \"chr30.g2147\"; transcript_id \"chr30.g2147.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51953263', '51953596', '.', '-', '.', 'gene_id \"chr6.g25499\"; transcript_id \"chr6.g25499.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51981239', '51981473', '.', '-', '.', 'gene_id \"chr6.g25509\"; transcript_id \"chr6.g25509.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52109609', '52109682', '.', '+', '.', 'gene_id \"chr6.g25582\"; transcript_id \"chr6.g25582.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52113982', '52114318', '.', '-', '.', 'gene_id \"chr6.g25591\"; transcript_id \"chr6.g25591.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52276790', '52278689', '.', '-', '.', 'gene_id \"chr6.g25661\"; transcript_id \"chr6.g25661.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7197271', '7197499', '.', '+', '.', 'gene_id \"chr30.g2383\"; transcript_id \"chr30.g2383.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7197483', '7197911', '.', '-', '.', 'gene_id \"chr30.g2384\"; transcript_id \"chr30.g2384.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52331761', '52332040', '.', '+', '.', 'gene_id \"chr6.g25704\"; transcript_id \"chr6.g25704.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52523719', '52523954', '.', '-', '.', 'gene_id \"chr6.g25826\"; transcript_id \"chr6.g25826.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52592697', '52592940', '.', '-', '.', 'gene_id \"chr6.g25882\"; transcript_id \"chr6.g25882.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52607335', '52607573', '.', '+', '.', 'gene_id \"chr6.g25910\"; transcript_id \"chr6.g25910.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52625123', '52625176', '.', '-', '.', 'gene_id \"chr6.g25918\"; transcript_id \"chr6.g25918.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7561409', '7561754', '.', '+', '.', 'gene_id \"chr30.g2627\"; transcript_id \"chr30.g2627.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52723405', '52723454', '.', '-', '.', 'gene_id \"chr6.g25925\"; transcript_id \"chr6.g25925.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52933982', '52934387', '.', '+', '.', 'gene_id \"chr6.g25960\"; transcript_id \"chr6.g25960.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52933982', '52934445', '.', '+', '.', 'gene_id \"chr6.g25960\"; transcript_id \"chr6.g25960.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '53675469', '53675512', '.', '-', '.', 'gene_id \"chr6.g26054\"; transcript_id \"chr6.g26054.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '53792106', '53792882', '.', '-', '.', 'gene_id \"chr6.g26060\"; transcript_id \"chr6.g26060.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '53793226', '53793464', '.', '+', '.', 'gene_id \"chr6.g26061\"; transcript_id \"chr6.g26061.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7795028', '7795692', '.', '-', '.', 'gene_id \"chr30.g2804\"; transcript_id \"chr30.g2804.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54156558', '54156653', '.', '-', '.', 'gene_id \"chr6.g26154\"; transcript_id \"chr6.g26154.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54156558', '54156656', '.', '-', '.', 'gene_id \"chr6.g26154\"; transcript_id \"chr6.g26154.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7949956', '7950127', '.', '-', '.', 'gene_id \"chr30.g2880\"; transcript_id \"chr30.g2880.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7984721', '7984796', '.', '+', '.', 'gene_id \"chr30.g2894\"; transcript_id \"chr30.g2894.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54200360', '54200401', '.', '+', '.', 'gene_id \"chr6.g26189\"; transcript_id \"chr6.g26189.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '8248182', '8248435', '.', '-', '.', 'gene_id \"chr30.g3093\"; transcript_id \"chr30.g3093.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '8371800', '8371841', '.', '+', '.', 'gene_id \"chr30.g3192\"; transcript_id \"chr30.g3192.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54686948', '54687123', '.', '-', '.', 'gene_id \"chr6.g26462\"; transcript_id \"chr6.g26462.i5\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54721965', '54722006', '.', '-', '.', 'gene_id \"chr6.g26489\"; transcript_id \"chr6.g26489.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54792879', '54793184', '.', '+', '.', 'gene_id \"chr6.g26541\"; transcript_id \"chr6.g26541.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55053196', '55053407', '.', '-', '.', 'gene_id \"chr6.g26720\"; transcript_id \"chr6.g26720.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '8884015', '8884054', '.', '-', '.', 'gene_id \"chr30.g3632\"; transcript_id \"chr30.g3632.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55243265', '55243490', '.', '+', '.', 'gene_id \"chr6.g26842\"; transcript_id \"chr6.g26842.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55301206', '55301237', '.', '-', '.', 'gene_id \"chr6.g26890\"; transcript_id \"chr6.g26890.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55314408', '55314680', '.', '-', '.', 'gene_id \"chr6.g26912\"; transcript_id \"chr6.g26912.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55315087', '55315316', '.', '-', '.', 'gene_id \"chr6.g26913\"; transcript_id \"chr6.g26913.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '9041187', '9041230', '.', '+', '.', 'gene_id \"chr30.g3731\"; transcript_id \"chr30.g3731.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55394471', '55394500', '.', '+', '.', 'gene_id \"chr6.g26943\"; transcript_id \"chr6.g26943.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55543066', '55543280', '.', '-', '.', 'gene_id \"chr6.g27029\"; transcript_id \"chr6.g27029.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55750085', '55750359', '.', '-', '.', 'gene_id \"chr6.g27200\"; transcript_id \"chr6.g27200.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55754760', '55756009', '.', '+', '.', 'gene_id \"chr6.g27201\"; transcript_id \"chr6.g27201.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '9443653', '9443818', '.', '-', '.', 'gene_id \"chr30.g4034\"; transcript_id \"chr30.g4034.i7\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '9526465', '9526508', '.', '+', '.', 'gene_id \"chr30.g4104\"; transcript_id \"chr30.g4104.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '9568222', '9568251', '.', '+', '.', 'gene_id \"chr30.g4124\"; transcript_id \"chr30.g4124.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55991444', '55991600', '.', '-', '.', 'gene_id \"chr6.g27307\"; transcript_id \"chr6.g27307.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55991444', '55991585', '.', '-', '.', 'gene_id \"chr6.g27307\"; transcript_id \"chr6.g27307.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55991242', '55991280', '.', '-', '.', 'gene_id \"chr6.g27319\"; transcript_id \"chr6.g27319.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56190383', '56190434', '.', '-', '.', 'gene_id \"chr6.g27466\"; transcript_id \"chr6.g27466.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '9963068', '9963248', '.', '+', '.', 'gene_id \"chr30.g4346\"; transcript_id \"chr30.g4346.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56403218', '56403441', '.', '-', '.', 'gene_id \"chr6.g27571\"; transcript_id \"chr6.g27571.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56403496', '56403829', '.', '+', '.', 'gene_id \"chr6.g27573\"; transcript_id \"chr6.g27573.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56432973', '56433085', '.', '+', '.', 'gene_id \"chr6.g27598\"; transcript_id \"chr6.g27598.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10132601', '10132657', '.', '+', '.', 'gene_id \"chr30.g4471\"; transcript_id \"chr30.g4471.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10262207', '10262243', '.', '-', '.', 'gene_id \"chr30.g4530\"; transcript_id \"chr30.g4530.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56621008', '56621343', '.', '+', '.', 'gene_id \"chr6.g27693\"; transcript_id \"chr6.g27693.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56695364', '56695395', '.', '-', '.', 'gene_id \"chr6.g27767\"; transcript_id \"chr6.g27767.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56697138', '56697251', '.', '-', '.', 'gene_id \"chr6.g27771\"; transcript_id \"chr6.g27771.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10531939', '10532105', '.', '-', '.', 'gene_id \"chr30.g4688\"; transcript_id \"chr30.g4688.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10525030', '10525911', '.', '+', '.', 'gene_id \"chr30.g4689\"; transcript_id \"chr30.g4689.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10519626', '10519903', '.', '+', '.', 'gene_id \"chr30.g4694\"; transcript_id \"chr30.g4694.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10533379', '10533437', '.', '+', '.', 'gene_id \"chr30.g4713\"; transcript_id \"chr30.g4713.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10557666', '10557858', '.', '-', '.', 'gene_id \"chr30.g4714\"; transcript_id \"chr30.g4714.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10556506', '10556674', '.', '+', '.', 'gene_id \"chr30.g4715\"; transcript_id \"chr30.g4715.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10555280', '10555723', '.', '+', '.', 'gene_id \"chr30.g4728\"; transcript_id \"chr30.g4728.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10543172', '10543616', '.', '+', '.', 'gene_id \"chr30.g4744\"; transcript_id \"chr30.g4744.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10565231', '10565421', '.', '-', '.', 'gene_id \"chr30.g4768\"; transcript_id \"chr30.g4768.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10701377', '10701439', '.', '-', '.', 'gene_id \"chr30.g4858\"; transcript_id \"chr30.g4858.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10918721', '10918781', '.', '+', '.', 'gene_id \"chr30.g4893\"; transcript_id \"chr30.g4893.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '10966473', '10966726', '.', '+', '.', 'gene_id \"chr30.g4904\"; transcript_id \"chr30.g4904.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11050486', '11051012', '.', '-', '.', 'gene_id \"chr30.g4926\"; transcript_id \"chr30.g4926.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57262304', '57262926', '.', '+', '.', 'gene_id \"chr6.g28066\"; transcript_id \"chr6.g28066.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11118867', '11119015', '.', '+', '.', 'gene_id \"chr30.g4960\"; transcript_id \"chr30.g4960.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57293514', '57293575', '.', '+', '.', 'gene_id \"chr6.g28093\"; transcript_id \"chr6.g28093.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11149629', '11149978', '.', '-', '.', 'gene_id \"chr30.g4997\"; transcript_id \"chr30.g4997.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57393378', '57393586', '.', '+', '.', 'gene_id \"chr6.g28115\"; transcript_id \"chr6.g28115.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11258064', '11258244', '.', '-', '.', 'gene_id \"chr30.g5068\"; transcript_id \"chr30.g5068.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57611330', '57611430', '.', '-', '.', 'gene_id \"chr6.g28167\"; transcript_id \"chr6.g28167.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57787258', '57787301', '.', '-', '.', 'gene_id \"chr6.g28245\"; transcript_id \"chr6.g28245.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57815904', '57815947', '.', '+', '.', 'gene_id \"chr6.g28264\"; transcript_id \"chr6.g28264.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11536264', '11536521', '.', '-', '.', 'gene_id \"chr30.g5260\"; transcript_id \"chr30.g5260.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr30', 'PASA', 'exon', '11638145', '11638302', '.', '+', '.', 'gene_id \"chr30.g5313\"; transcript_id \"chr30.g5313.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '58328014', '58328053', '.', '-', '.', 'gene_id \"chr6.g28393\"; transcript_id \"chr6.g28393.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11654727', '11655109', '.', '+', '.', 'gene_id \"chr30.g5324\"; transcript_id \"chr30.g5324.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11656799', '11657213', '.', '-', '.', 'gene_id \"chr30.g5328\"; transcript_id \"chr30.g5328.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11657228', '11658470', '.', '+', '.', 'gene_id \"chr30.g5329\"; transcript_id \"chr30.g5329.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11659326', '11659900', '.', '-', '.', 'gene_id \"chr30.g5331\"; transcript_id \"chr30.g5331.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11659543', '11659900', '.', '-', '.', 'gene_id \"chr30.g5331\"; transcript_id \"chr30.g5331.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11660513', '11660760', '.', '-', '.', 'gene_id \"chr30.g5336\"; transcript_id \"chr30.g5336.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11689044', '11689231', '.', '-', '.', 'gene_id \"chr30.g5356\"; transcript_id \"chr30.g5356.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11698765', '11698940', '.', '-', '.', 'gene_id \"chr30.g5363\"; transcript_id \"chr30.g5363.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11782965', '11785796', '.', '-', '.', 'gene_id \"chr30.g5421\"; transcript_id \"chr30.g5421.i7\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11781288', '11782600', '.', '-', '.', 'gene_id \"chr30.g5423\"; transcript_id \"chr30.g5423.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '58784659', '58784832', '.', '-', '.', 'gene_id \"chr6.g28514\"; transcript_id \"chr6.g28514.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '58788364', '58788579', '.', '-', '.', 'gene_id \"chr6.g28515\"; transcript_id \"chr6.g28515.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11978285', '11978324', '.', '+', '.', 'gene_id \"chr30.g5573\"; transcript_id \"chr30.g5573.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11978283', '11978319', '.', '-', '.', 'gene_id \"chr30.g5574\"; transcript_id \"chr30.g5574.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59050189', '59050312', '.', '-', '.', 'gene_id \"chr6.g28691\"; transcript_id \"chr6.g28691.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12053083', '12053257', '.', '-', '.', 'gene_id \"chr30.g5667\"; transcript_id \"chr30.g5667.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12053924', '12054076', '.', '-', '.', 'gene_id \"chr30.g5668\"; transcript_id \"chr30.g5668.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12095747', '12095781', '.', '-', '.', 'gene_id \"chr30.g5683\"; transcript_id \"chr30.g5683.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12171598', '12171754', '.', '-', '.', 'gene_id \"chr30.g5702\"; transcript_id \"chr30.g5702.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59232299', '59232459', '.', '+', '.', 'gene_id \"chr6.g28826\"; transcript_id \"chr6.g28826.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '13714926', '13714959', '.', '-', '.', 'gene_id \"chr30.g5835\"; transcript_id \"chr30.g5835.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59600834', '59600891', '.', '-', '.', 'gene_id \"chr6.g28994\"; transcript_id \"chr6.g28994.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59885891', '59885923', '.', '-', '.', 'gene_id \"chr6.g29157\"; transcript_id \"chr6.g29157.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '14921345', '14921397', '.', '-', '.', 'gene_id \"chr30.g6230\"; transcript_id \"chr30.g6230.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15311291', '15311336', '.', '-', '.', 'gene_id \"chr30.g6375\"; transcript_id \"chr30.g6375.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15331557', '15331951', '.', '-', '.', 'gene_id \"chr30.g6394\"; transcript_id \"chr30.g6394.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61418448', '61418488', '.', '-', '.', 'gene_id \"chr6.g29443\"; transcript_id \"chr6.g29443.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61404416', '61404464', '.', '+', '.', 'gene_id \"chr6.g29458\"; transcript_id \"chr6.g29458.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61422400', '61422595', '.', '-', '.', 'gene_id \"chr6.g29476\"; transcript_id \"chr6.g29476.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61422445', '61422601', '.', '-', '.', 'gene_id \"chr6.g29476\"; transcript_id \"chr6.g29476.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61422356', '61422528', '.', '+', '.', 'gene_id \"chr6.g29477\"; transcript_id \"chr6.g29477.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15464936', '15465421', '.', '-', '.', 'gene_id \"chr30.g6482\"; transcript_id \"chr30.g6482.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15475842', '15475882', '.', '+', '.', 'gene_id \"chr30.g6496\"; transcript_id \"chr30.g6496.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61577927', '61578169', '.', '-', '.', 'gene_id \"chr6.g29574\"; transcript_id \"chr6.g29574.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15642786', '15643287', '.', '+', '.', 'gene_id \"chr30.g6593\"; transcript_id \"chr30.g6593.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15643370', '15643645', '.', '-', '.', 'gene_id \"chr30.g6594\"; transcript_id \"chr30.g6594.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15643652', '15643831', '.', '-', '.', 'gene_id \"chr30.g6595\"; transcript_id \"chr30.g6595.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15670897', '15671218', '.', '-', '.', 'gene_id \"chr30.g6608\"; transcript_id \"chr30.g6608.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61679493', '61679738', '.', '-', '.', 'gene_id \"chr6.g29652\"; transcript_id \"chr6.g29652.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61690139', '61690310', '.', '-', '.', 'gene_id \"chr6.g29655\"; transcript_id \"chr6.g29655.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61690478', '61690629', '.', '-', '.', 'gene_id \"chr6.g29656\"; transcript_id \"chr6.g29656.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15709868', '15710071', '.', '+', '.', 'gene_id \"chr30.g6622\"; transcript_id \"chr30.g6622.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15710280', '15710542', '.', '+', '.', 'gene_id \"chr30.g6623\"; transcript_id \"chr30.g6623.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15710534', '15710710', '.', '-', '.', 'gene_id \"chr30.g6624\"; transcript_id \"chr30.g6624.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61790489', '61792041', '.', '+', '.', 'gene_id \"chr6.g29741\"; transcript_id \"chr6.g29741.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15961007', '15961422', '.', '+', '.', 'gene_id \"chr30.g6763\"; transcript_id \"chr30.g6763.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15961763', '15962489', '.', '+', '.', 'gene_id \"chr30.g6765\"; transcript_id \"chr30.g6765.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16054022', '16054472', '.', '-', '.', 'gene_id \"chr30.g6815\"; transcript_id \"chr30.g6815.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62284582', '62284729', '.', '+', '.', 'gene_id \"chr6.g29853\"; transcript_id \"chr6.g29853.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62285988', '62287308', '.', '+', '.', 'gene_id \"chr6.g29854\"; transcript_id \"chr6.g29854.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62465997', '62466197', '.', '+', '.', 'gene_id \"chr6.g29904\"; transcript_id \"chr6.g29904.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62466408', '62466621', '.', '-', '.', 'gene_id \"chr6.g29905\"; transcript_id \"chr6.g29905.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16312124', '16312413', '.', '-', '.', 'gene_id \"chr30.g6913\"; transcript_id \"chr30.g6913.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16307533', '16307689', '.', '+', '.', 'gene_id \"chr30.g6917\"; transcript_id \"chr30.g6917.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16309925', '16310164', '.', '+', '.', 'gene_id \"chr30.g6919\"; transcript_id \"chr30.g6919.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16310302', '16310502', '.', '+', '.', 'gene_id \"chr30.g6920\"; transcript_id \"chr30.g6920.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62611823', '62612025', '.', '+', '.', 'gene_id \"chr6.g29970\"; transcript_id \"chr6.g29970.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16369943', '16370418', '.', '+', '.', 'gene_id \"chr30.g6969\"; transcript_id \"chr30.g6969.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63087407', '63087447', '.', '-', '.', 'gene_id \"chr6.g30179\"; transcript_id \"chr6.g30179.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63127432', '63128187', '.', '+', '.', 'gene_id \"chr6.g30181\"; transcript_id \"chr6.g30181.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63116178', '63116330', '.', '-', '.', 'gene_id \"chr6.g30186\"; transcript_id \"chr6.g30186.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63337511', '63337563', '.', '-', '.', 'gene_id \"chr6.g30278\"; transcript_id \"chr6.g30278.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63464374', '63464715', '.', '-', '.', 'gene_id \"chr6.g30347\"; transcript_id \"chr6.g30347.i8\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63802692', '63802759', '.', '-', '.', 'gene_id \"chr6.g30653\"; transcript_id \"chr6.g30653.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '63858376', '63858471', '.', '+', '.', 'gene_id \"chr6.g30669\"; transcript_id \"chr6.g30669.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '17398664', '17398827', '.', '-', '.', 'gene_id \"chr30.g7655\"; transcript_id \"chr30.g7655.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '17514193', '17514348', '.', '+', '.', 'gene_id \"chr30.g7745\"; transcript_id \"chr30.g7745.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65097236', '65097778', '.', '-', '.', 'gene_id \"chr6.g30795\"; transcript_id \"chr6.g30795.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65640653', '65641068', '.', '+', '.', 'gene_id \"chr6.g30828\"; transcript_id \"chr6.g30828.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65640653', '65640917', '.', '+', '.', 'gene_id \"chr6.g30828\"; transcript_id \"chr6.g30828.i4\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65640312', '65640493', '.', '-', '.', 'gene_id \"chr6.g30829\"; transcript_id \"chr6.g30829.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65713164', '65713686', '.', '-', '.', 'gene_id \"chr6.g30876\"; transcript_id \"chr6.g30876.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65713824', '65713994', '.', '+', '.', 'gene_id \"chr6.g30877\"; transcript_id \"chr6.g30877.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65774521', '65774560', '.', '-', '.', 'gene_id \"chr6.g30907\"; transcript_id \"chr6.g30907.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65923455', '65923487', '.', '-', '.', 'gene_id \"chr6.g30954\"; transcript_id \"chr6.g30954.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '17801501', '17801754', '.', '+', '.', 'gene_id \"chr30.g7931\"; transcript_id \"chr30.g7931.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '67193809', '67193862', '.', '+', '.', 'gene_id \"chr6.g31035\"; transcript_id \"chr6.g31035.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '67593481', '67593770', '.', '-', '.', 'gene_id \"chr6.g31058\"; transcript_id \"chr6.g31058.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '67609775', '67610818', '.', '-', '.', 'gene_id \"chr6.g31062\"; transcript_id \"chr6.g31062.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '67610132', '67610656', '.', '+', '.', 'gene_id \"chr6.g31063\"; transcript_id \"chr6.g31063.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68113533', '68114086', '.', '+', '.', 'gene_id \"chr6.g31095\"; transcript_id \"chr6.g31095.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68144964', '68146161', '.', '+', '.', 'gene_id \"chr6.g31101\"; transcript_id \"chr6.g31101.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18239282', '18239410', '.', '-', '.', 'gene_id \"chr30.g8122\"; transcript_id \"chr30.g8122.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18271440', '18271639', '.', '+', '.', 'gene_id \"chr30.g8151\"; transcript_id \"chr30.g8151.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18347635', '18347694', '.', '+', '.', 'gene_id \"chr30.g8230\"; transcript_id \"chr30.g8230.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18347635', '18347696', '.', '+', '.', 'gene_id \"chr30.g8230\"; transcript_id \"chr30.g8230.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68546391', '68546984', '.', '-', '.', 'gene_id \"chr6.g31250\"; transcript_id \"chr6.g31250.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18363841', '18364007', '.', '-', '.', 'gene_id \"chr30.g8258\"; transcript_id \"chr30.g8258.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18374898', '18374936', '.', '-', '.', 'gene_id \"chr30.g8268\"; transcript_id \"chr30.g8268.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18885311', '18885613', '.', '-', '.', 'gene_id \"chr30.g8365\"; transcript_id \"chr30.g8365.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68984041', '68984460', '.', '-', '.', 'gene_id \"chr6.g31438\"; transcript_id \"chr6.g31438.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '19238043', '19238093', '.', '-', '.', 'gene_id \"chr30.g8465\"; transcript_id \"chr30.g8465.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69044947', '69045013', '.', '+', '.', 'gene_id \"chr6.g31505\"; transcript_id \"chr6.g31505.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69134413', '69134619', '.', '+', '.', 'gene_id \"chr6.g31583\"; transcript_id \"chr6.g31583.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69134433', '69134615', '.', '+', '.', 'gene_id \"chr6.g31583\"; transcript_id \"chr6.g31583.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69260584', '69260806', '.', '-', '.', 'gene_id \"chr6.g31624\"; transcript_id \"chr6.g31624.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69257952', '69258387', '.', '+', '.', 'gene_id \"chr6.g31633\"; transcript_id \"chr6.g31633.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '20686782', '20686885', '.', '-', '.', 'gene_id \"chr30.g8584\"; transcript_id \"chr30.g8584.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69336027', '69336185', '.', '-', '.', 'gene_id \"chr6.g31744\"; transcript_id \"chr6.g31744.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69356333', '69356512', '.', '+', '.', 'gene_id \"chr6.g31771\"; transcript_id \"chr6.g31771.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69358620', '69358668', '.', '+', '.', 'gene_id \"chr6.g31773\"; transcript_id \"chr6.g31773.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '20894870', '20894954', '.', '+', '.', 'gene_id \"chr30.g8738\"; transcript_id \"chr30.g8738.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '20954341', '20954388', '.', '+', '.', 'gene_id \"chr30.g8782\"; transcript_id \"chr30.g8782.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21039948', '21040419', '.', '-', '.', 'gene_id \"chr30.g8842\"; transcript_id \"chr30.g8842.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21229883', '21230069', '.', '-', '.', 'gene_id \"chr30.g8892\"; transcript_id \"chr30.g8892.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69766015', '69766202', '.', '-', '.', 'gene_id \"chr6.g31950\"; transcript_id \"chr6.g31950.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69766357', '69766532', '.', '-', '.', 'gene_id \"chr6.g31951\"; transcript_id \"chr6.g31951.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69766552', '69766765', '.', '+', '.', 'gene_id \"chr6.g31952\"; transcript_id \"chr6.g31952.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70175233', '70175397', '.', '-', '.', 'gene_id \"chr6.g32059\"; transcript_id \"chr6.g32059.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21505336', '21505659', '.', '-', '.', 'gene_id \"chr30.g9068\"; transcript_id \"chr30.g9068.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21635572', '21635619', '.', '-', '.', 'gene_id \"chr30.g9105\"; transcript_id \"chr30.g9105.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70649146', '70649302', '.', '+', '.', 'gene_id \"chr6.g32175\"; transcript_id \"chr6.g32175.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70695917', '70695960', '.', '+', '.', 'gene_id \"chr6.g32189\"; transcript_id \"chr6.g32189.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21858729', '21859212', '.', '+', '.', 'gene_id \"chr30.g9174\"; transcript_id \"chr30.g9174.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70881896', '70881936', '.', '+', '.', 'gene_id \"chr6.g32215\"; transcript_id \"chr6.g32215.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70939819', '70939907', '.', '+', '.', 'gene_id \"chr6.g32238\"; transcript_id \"chr6.g32238.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '71027422', '71027470', '.', '-', '.', 'gene_id \"chr6.g32298\"; transcript_id \"chr6.g32298.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21935030', '21935193', '.', '+', '.', 'gene_id \"chr30.g9264\"; transcript_id \"chr30.g9264.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '21935214', '21936349', '.', '+', '.', 'gene_id \"chr30.g9265\"; transcript_id \"chr30.g9265.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '71416801', '71417018', '.', '+', '.', 'gene_id \"chr6.g32352\"; transcript_id \"chr6.g32352.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '71507630', '71507671', '.', '-', '.', 'gene_id \"chr6.g32359\"; transcript_id \"chr6.g32359.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22028738', '22029366', '.', '-', '.', 'gene_id \"chr30.g9335\"; transcript_id \"chr30.g9335.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22029603', '22029873', '.', '-', '.', 'gene_id \"chr30.g9337\"; transcript_id \"chr30.g9337.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '72163992', '72164042', '.', '+', '.', 'gene_id \"chr6.g32516\"; transcript_id \"chr6.g32516.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22290991', '22291024', '.', '+', '.', 'gene_id \"chr30.g9485\"; transcript_id \"chr30.g9485.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '72444505', '72444535', '.', '+', '.', 'gene_id \"chr6.g32571\"; transcript_id \"chr6.g32571.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22392129', '22392371', '.', '-', '.', 'gene_id \"chr30.g9532\"; transcript_id \"chr30.g9532.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22394346', '22394498', '.', '-', '.', 'gene_id \"chr30.g9537\"; transcript_id \"chr30.g9537.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22394515', '22395406', '.', '+', '.', 'gene_id \"chr30.g9538\"; transcript_id \"chr30.g9538.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22395454', '22395655', '.', '+', '.', 'gene_id \"chr30.g9539\"; transcript_id \"chr30.g9539.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '72743844', '72743904', '.', '+', '.', 'gene_id \"chr6.g32593\"; transcript_id \"chr6.g32593.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22415808', '22416027', '.', '-', '.', 'gene_id \"chr30.g9560\"; transcript_id \"chr30.g9560.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '73018591', '73018656', '.', '+', '.', 'gene_id \"chr6.g32619\"; transcript_id \"chr6.g32619.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22441294', '22441463', '.', '+', '.', 'gene_id \"chr30.g9597\"; transcript_id \"chr30.g9597.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '73758248', '73758476', '.', '-', '.', 'gene_id \"chr6.g32654\"; transcript_id \"chr6.g32654.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '73963923', '73964246', '.', '-', '.', 'gene_id \"chr6.g32674\"; transcript_id \"chr6.g32674.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74774505', '74776551', '.', '+', '.', 'gene_id \"chr6.g32770\"; transcript_id \"chr6.g32770.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74773583', '74774529', '.', '-', '.', 'gene_id \"chr6.g32771\"; transcript_id \"chr6.g32771.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74588028', '74588184', '.', '-', '.', 'gene_id \"chr6.g32772\"; transcript_id \"chr6.g32772.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74588064', '74588184', '.', '-', '.', 'gene_id \"chr6.g32772\"; transcript_id \"chr6.g32772.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74604311', '74604462', '.', '-', '.', 'gene_id \"chr6.g32773\"; transcript_id \"chr6.g32773.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74437147', '74437272', '.', '-', '.', 'gene_id \"chr6.g32775\"; transcript_id \"chr6.g32775.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74504329', '74504462', '.', '-', '.', 'gene_id \"chr6.g32776\"; transcript_id \"chr6.g32776.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22770060', '22770101', '.', '-', '.', 'gene_id \"chr30.g9821\"; transcript_id \"chr30.g9821.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22784734', '22784924', '.', '+', '.', 'gene_id \"chr30.g9838\"; transcript_id \"chr30.g9838.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22785017', '22785207', '.', '-', '.', 'gene_id \"chr30.g9839\"; transcript_id \"chr30.g9839.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22818899', '22819155', '.', '+', '.', 'gene_id \"chr30.g9885\"; transcript_id \"chr30.g9885.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22851430', '22851581', '.', '-', '.', 'gene_id \"chr30.g9909\"; transcript_id \"chr30.g9909.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75389452', '75389488', '.', '+', '.', 'gene_id \"chr6.g33043\"; transcript_id \"chr6.g33043.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75466773', '75466854', '.', '-', '.', 'gene_id \"chr6.g33050\"; transcript_id \"chr6.g33050.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75466787', '75466834', '.', '+', '.', 'gene_id \"chr6.g33051\"; transcript_id \"chr6.g33051.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23009729', '23009804', '.', '+', '.', 'gene_id \"chr30.g10027\"; transcript_id \"chr30.g10027.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75555056', '75555096', '.', '+', '.', 'gene_id \"chr6.g33080\"; transcript_id \"chr6.g33080.i3\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75582358', '75583242', '.', '+', '.', 'gene_id \"chr6.g33081\"; transcript_id \"chr6.g33081.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23229106', '23229142', '.', '+', '.', 'gene_id \"chr30.g10054\"; transcript_id \"chr30.g10054.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75592067', '75592220', '.', '+', '.', 'gene_id \"chr6.g33131\"; transcript_id \"chr6.g33131.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75612931', '75614959', '.', '+', '.', 'gene_id \"chr6.g33164\"; transcript_id \"chr6.g33164.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '76229841', '76229890', '.', '-', '.', 'gene_id \"chr6.g33333\"; transcript_id \"chr6.g33333.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23791680', '23791717', '.', '-', '.', 'gene_id \"chr30.g10303\"; transcript_id \"chr30.g10303.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23808438', '23809135', '.', '+', '.', 'gene_id \"chr30.g10320\"; transcript_id \"chr30.g10320.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23987276', '23987682', '.', '+', '.', 'gene_id \"chr30.g10492\"; transcript_id \"chr30.g10492.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77365674', '77366111', '.', '+', '.', 'gene_id \"chr6.g33569\"; transcript_id \"chr6.g33569.i2\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77480100', '77480585', '.', '+', '.', 'gene_id \"chr6.g33622\"; transcript_id \"chr6.g33622.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77481001', '77481215', '.', '+', '.', 'gene_id \"chr6.g33623\"; transcript_id \"chr6.g33623.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24106936', '24107293', '.', '-', '.', 'gene_id \"chr30.g10581\"; transcript_id \"chr30.g10581.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77638100', '77638328', '.', '+', '.', 'gene_id \"chr6.g33698\"; transcript_id \"chr6.g33698.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77638683', '77638864', '.', '+', '.', 'gene_id \"chr6.g33700\"; transcript_id \"chr6.g33700.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77639212', '77639491', '.', '+', '.', 'gene_id \"chr6.g33701\"; transcript_id \"chr6.g33701.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24196569', '24196706', '.', '+', '.', 'gene_id \"chr30.g10696\"; transcript_id \"chr30.g10696.i1\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21203532', '21203688', '100', '+', '.', 'gene_id \"chr6.g33743\"; transcript_id \"blat.proc4587.chain_389\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '5679948', '5680004', '100', '+', '.', 'gene_id \"chr6.g33763\"; transcript_id \"blat.proc28144.chain_12913\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '66452966', '66452995', '100', '-', '.', 'gene_id \"chr6.g33767\"; transcript_id \"blat.proc4005.chain_12086\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '18126087', '18126118', '100', '+', '.', 'gene_id \"chr6.g33780\"; transcript_id \"blat.proc18128.chain_23618\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54911876', '54911922', '100', '+', '.', 'gene_id \"chr6.g33815\"; transcript_id \"blat.proc28144.chain_8761\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12449473', '12449521', '100', '-', '.', 'gene_id \"chr6.g33874\"; transcript_id \"blat.proc4005.chain_8425\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '77304203', '77304249', '100', '+', '.', 'gene_id \"chr6.g33937\"; transcript_id \"blat.proc4587.chain_10437\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33279260', '33279292', '100', '+', '.', 'gene_id \"chr6.g33956\"; transcript_id \"blat.proc18128.chain_11063\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '18739737', '18739778', '100', '-', '.', 'gene_id \"chr6.g33985\"; transcript_id \"blat.proc18128.chain_7443\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '46665245', '46665288', '100', '-', '.', 'gene_id \"chr6.g34033\"; transcript_id \"blat.proc28144.chain_14741\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26343099', '26343130', '100', '-', '.', 'gene_id \"chr6.g34234\"; transcript_id \"blat.proc18128.chain_1792\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43955245', '43955275', '100', '+', '.', 'gene_id \"chr6.g34372\"; transcript_id \"blat.proc18128.chain_24243\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '75985941', '75985978', '100', '+', '.', 'gene_id \"chr6.g34429\"; transcript_id \"blat.proc18128.chain_15339\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27631897', '27631934', '100', '+', '.', 'gene_id \"chr6.g34478\"; transcript_id \"blat.proc28144.chain_18526\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11893905', '11894030', '100', '+', '.', 'gene_id \"chr6.g34482\"; transcript_id \"blat.proc18128.chain_17557\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23751268', '23751545', '100', '+', '.', 'gene_id \"chr6.g34558\"; transcript_id \"blat.proc18128.chain_7057\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '62567935', '62567978', '100', '-', '.', 'gene_id \"chr6.g34580\"; transcript_id \"blat.proc18128.chain_20476\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51960165', '51960329', '100', '-', '.', 'gene_id \"chr6.g34712\"; transcript_id \"blat.proc4005.chain_10100\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59353000', '59353047', '100', '-', '.', 'gene_id \"chr6.g34785\"; transcript_id \"blat.proc28144.chain_22603\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '47231100', '47231152', '100', '+', '.', 'gene_id \"chr6.g34845\"; transcript_id \"blat.proc18128.chain_17286\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41232153', '41232422', '100', '-', '.', 'gene_id \"chr6.g34992\"; transcript_id \"blat.proc4587.chain_2249\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12575949', '12575992', '100', '+', '.', 'gene_id \"chr6.g35508\"; transcript_id \"blat.proc18128.chain_13771\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '13016298', '13016475', '100', '+', '.', 'gene_id \"chr6.g35517\"; transcript_id \"blat.proc4005.chain_13985\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68937148', '68937239', '100', '-', '.', 'gene_id \"chr6.g35551\"; transcript_id \"blat.proc4587.chain_6729\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '66486697', '66486766', '100', '-', '.', 'gene_id \"chr6.g35586\"; transcript_id \"blat.proc18128.chain_16480\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '20572456', '20572488', '100', '+', '.', 'gene_id \"chr6.g35699\"; transcript_id \"blat.proc4005.chain_1371\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69188372', '69188407', '100', '+', '.', 'gene_id \"chr6.g35712\"; transcript_id \"blat.proc4587.chain_10440\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '71000069', '71000112', '100', '-', '.', 'gene_id \"chr6.g35775\"; transcript_id \"blat.proc28144.chain_11872\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59174411', '59174450', '100', '+', '.', 'gene_id \"chr6.g35800\"; transcript_id \"blat.proc18128.chain_467\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17268746', '17268788', '100', '+', '.', 'gene_id \"chr6.g35837\"; transcript_id \"blat.proc28144.chain_11535\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32050321', '32050488', '100', '-', '.', 'gene_id \"chr6.g35871\"; transcript_id \"blat.proc18128.chain_18421\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '16785002', '16785117', '100', '-', '.', 'gene_id \"chr6.g35881\"; transcript_id \"blat.proc28144.chain_21297\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30870037', '30870362', '100', '+', '.', 'gene_id \"chr6.g35940\"; transcript_id \"blat.proc18128.chain_8486\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30620804', '30620900', '100', '-', '.', 'gene_id \"chr6.g36024\"; transcript_id \"blat.proc4587.chain_13146\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1251752', '1251791', '100', '+', '.', 'gene_id \"chr6.g36050\"; transcript_id \"blat.proc4587.chain_8457\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22020634', '22020680', '100', '-', '.', 'gene_id \"chr6.g36101\"; transcript_id \"blat.proc4005.chain_1710\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37089590', '37089695', '100', '+', '.', 'gene_id \"chr6.g36158\"; transcript_id \"blat.proc4005.chain_1908\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68575064', '68575094', '100', '-', '.', 'gene_id \"chr6.g36315\"; transcript_id \"blat.proc18128.chain_19679\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '6934130', '6934162', '100', '-', '.', 'gene_id \"chr6.g36420\"; transcript_id \"blat.proc18128.chain_24736\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12658995', '12659056', '100', '-', '.', 'gene_id \"chr6.g36465\"; transcript_id \"blat.proc28144.chain_20622\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23609618', '23609667', '100', '-', '.', 'gene_id \"chr6.g36480\"; transcript_id \"blat.proc4005.chain_12693\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '57048961', '57049003', '100', '+', '.', 'gene_id \"chr6.g36497\"; transcript_id \"blat.proc28144.chain_10335\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45459818', '45459883', '100', '-', '.', 'gene_id \"chr6.g36512\"; transcript_id \"blat.proc18128.chain_5899\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37778753', '37778910', '100', '-', '.', 'gene_id \"chr6.g36556\"; transcript_id \"blat.proc18128.chain_13920\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56180138', '56180187', '100', '+', '.', 'gene_id \"chr6.g36631\"; transcript_id \"blat.proc4587.chain_6627\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1208380', '1208424', '100', '+', '.', 'gene_id \"chr6.g36708\"; transcript_id \"blat.proc4587.chain_10122\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28017116', '28017202', '100', '-', '.', 'gene_id \"chr6.g36749\"; transcript_id \"blat.proc4005.chain_9420\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '10579547', '10579580', '100', '-', '.', 'gene_id \"chr6.g36790\"; transcript_id \"blat.proc4587.chain_1632\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52871681', '52871729', '100', '-', '.', 'gene_id \"chr6.g36882\"; transcript_id \"blat.proc18128.chain_12093\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56288849', '56288885', '100', '+', '.', 'gene_id \"chr6.g36906\"; transcript_id \"blat.proc18128.chain_3948\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '34872606', '34872655', '100', '-', '.', 'gene_id \"chr6.g36987\"; transcript_id \"blat.proc4587.chain_13272\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54281451', '54281496', '100', '-', '.', 'gene_id \"chr6.g37080\"; transcript_id \"blat.proc28144.chain_11377\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '61764719', '61764763', '100', '-', '.', 'gene_id \"chr6.g37086\"; transcript_id \"blat.proc18128.chain_6680\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '71079350', '71079389', '100', '+', '.', 'gene_id \"chr6.g37121\"; transcript_id \"blat.proc18128.chain_26534\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59524616', '59524696', '100', '+', '.', 'gene_id \"chr6.g37126\"; transcript_id \"blat.proc4587.chain_11935\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41330342', '41330382', '100', '+', '.', 'gene_id \"chr6.g37205\"; transcript_id \"blat.proc28144.chain_3954\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '2335572', '2335605', '100', '-', '.', 'gene_id \"chr6.g37238\"; transcript_id \"blat.proc28144.chain_18533\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19147719', '19147755', '100', '+', '.', 'gene_id \"chr6.g37498\"; transcript_id \"blat.proc28144.chain_20121\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '70993029', '70993076', '100', '-', '.', 'gene_id \"chr6.g37626\"; transcript_id \"blat.proc28144.chain_12677\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '23347670', '23347712', '100', '-', '.', 'gene_id \"chr6.g37666\"; transcript_id \"blat.proc28144.chain_23282\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '51716117', '51716149', '100', '+', '.', 'gene_id \"chr6.g37836\"; transcript_id \"blat.proc28144.chain_19142\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '20073102', '20073143', '100', '+', '.', 'gene_id \"chr6.g37993\"; transcript_id \"blat.proc4005.chain_1927\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25903031', '25903107', '100', '-', '.', 'gene_id \"chr6.g38117\"; transcript_id \"blat.proc28144.chain_22442\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8743930', '8744008', '100', '+', '.', 'gene_id \"chr6.g38222\"; transcript_id \"blat.proc18128.chain_89\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43686664', '43686862', '100', '+', '.', 'gene_id \"chr6.g38382\"; transcript_id \"blat.proc28144.chain_11040\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55664812', '55664847', '100', '-', '.', 'gene_id \"chr6.g38436\"; transcript_id \"blat.proc18128.chain_21552\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '49342937', '49343118', '100', '-', '.', 'gene_id \"chr6.g38481\"; transcript_id \"blat.proc28144.chain_4386\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33212199', '33212260', '100', '-', '.', 'gene_id \"chr6.g38499\"; transcript_id \"blat.proc28144.chain_3597\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr6', 'PASA', 'exon', '18083132', '18083164', '100', '+', '.', 'gene_id \"chr6.g38538\"; transcript_id \"blat.proc4587.chain_8541\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8774745', '8774791', '100', '-', '.', 'gene_id \"chr6.g38653\"; transcript_id \"blat.proc28144.chain_8765\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30172392', '30172436', '100', '-', '.', 'gene_id \"chr6.g38693\"; transcript_id \"blat.proc28144.chain_9064\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55940021', '55940071', '100', '+', '.', 'gene_id \"chr6.g38712\"; transcript_id \"blat.proc18128.chain_20784\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '7160217', '7160298', '100', '-', '.', 'gene_id \"chr6.g38732\"; transcript_id \"blat.proc18128.chain_20532\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3515623', '3515782', '100', '-', '.', 'gene_id \"chr6.g38760\"; transcript_id \"blat.proc28144.chain_16133\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1834786', '1834866', '100', '-', '.', 'gene_id \"chr6.g38761\"; transcript_id \"blat.proc18128.chain_4543\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21675609', '21675638', '100', '-', '.', 'gene_id \"chr6.g38807\"; transcript_id \"blat.proc28144.chain_15630\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45983621', '45983676', '100', '+', '.', 'gene_id \"chr6.g38864\"; transcript_id \"blat.proc4587.chain_5612\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '1191298', '1191345', '100', '+', '.', 'gene_id \"chr6.g38875\"; transcript_id \"blat.proc28144.chain_7873\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '31213219', '31213391', '100', '+', '.', 'gene_id \"chr6.g38882\"; transcript_id \"blat.proc18128.chain_20629\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25349655', '25349688', '100', '+', '.', 'gene_id \"chr6.g38943\"; transcript_id \"blat.proc18128.chain_23016\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21343551', '21343591', '100', '-', '.', 'gene_id \"chr6.g38946\"; transcript_id \"blat.proc4005.chain_16380\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59357112', '59357144', '100', '-', '.', 'gene_id \"chr6.g38997\"; transcript_id \"blat.proc28144.chain_21757\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30141682', '30141738', '100', '+', '.', 'gene_id \"chr6.g39023\"; transcript_id \"blat.proc18128.chain_18410\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17827468', '17827660', '100', '+', '.', 'gene_id \"chr6.g39026\"; transcript_id \"blat.proc18128.chain_14833\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17750577', '17750616', '100', '+', '.', 'gene_id \"chr6.g39143\"; transcript_id \"blat.proc18128.chain_923\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65239345', '65239385', '100', '+', '.', 'gene_id \"chr6.g39248\"; transcript_id \"blat.proc28144.chain_11697\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65657184', '65657221', '100', '-', '.', 'gene_id \"chr6.g39329\"; transcript_id \"blat.proc4587.chain_13479\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '65099510', '65099918', '100', '+', '.', 'gene_id \"chr6.g39635\"; transcript_id \"blat.proc18128.chain_19714\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56323981', '56324071', '100', '-', '.', 'gene_id \"chr6.g39684\"; transcript_id \"blat.proc4005.chain_19801\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '3902996', '3903212', '100', '-', '.', 'gene_id \"chr6.g39719\"; transcript_id \"blat.proc4005.chain_5656\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42773503', '42773558', '100', '-', '.', 'gene_id \"chr6.g39742\"; transcript_id \"blat.proc4005.chain_19874\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '739916', '739992', '100', '-', '.', 'gene_id \"chr6.g39824\"; transcript_id \"blat.proc18128.chain_18693\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54725623', '54725748', '100', '+', '.', 'gene_id \"chr6.g39857\"; transcript_id \"blat.proc18128.chain_14256\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11894102', '11894241', '100', '-', '.', 'gene_id \"chr6.g39869\"; transcript_id \"blat.proc18128.chain_17551\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '12687598', '12687754', '100', '-', '.', 'gene_id \"chr6.g39884\"; transcript_id \"blat.proc4587.chain_10682\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '48959948', '48959989', '100', '-', '.', 'gene_id \"chr6.g39987\"; transcript_id \"blat.proc18128.chain_23372\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40537780', '40537978', '100', '+', '.', 'gene_id \"chr6.g40063\"; transcript_id \"blat.proc28144.chain_11038\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '54654261', '54654295', '100', '-', '.', 'gene_id \"chr6.g40077\"; transcript_id \"blat.proc18128.chain_20284\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '36195375', '36195429', '100', '+', '.', 'gene_id \"chr6.g40109\"; transcript_id \"blat.proc18128.chain_12875\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74058238', '74058369', '100', '+', '.', 'gene_id \"chr6.g40177\"; transcript_id \"blat.proc18128.chain_19312\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '41277353', '41277410', '100', '+', '.', 'gene_id \"chr6.g40220\"; transcript_id \"blat.proc18128.chain_6901\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '37763555', '37763782', '100', '+', '.', 'gene_id \"chr6.g40221\"; transcript_id \"blat.proc18128.chain_13943\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '64787671', '64787750', '100', '-', '.', 'gene_id \"chr6.g40329\"; transcript_id \"blat.proc4005.chain_23511\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '33224918', '33224966', '100', '-', '.', 'gene_id \"chr6.g40332\"; transcript_id \"blat.proc28144.chain_13132\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '22651748', '22651798', '100', '-', '.', 'gene_id \"chr6.g40336\"; transcript_id \"blat.proc28144.chain_16040\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '64734006', '64734056', '100', '+', '.', 'gene_id \"chr6.g40548\"; transcript_id \"blat.proc28144.chain_20348\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '28575756', '28576147', '100', '-', '.', 'gene_id \"chr6.g40622\"; transcript_id \"blat.proc18128.chain_17921\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '32418570', '32418606', '100', '-', '.', 'gene_id \"chr6.g40633\"; transcript_id \"blat.proc4587.chain_11924\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '45968031', '45968092', '100', '+', '.', 'gene_id \"chr6.g40693\"; transcript_id \"blat.proc28144.chain_23410\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '8948243', '8948290', '100', '-', '.', 'gene_id \"chr6.g40700\"; transcript_id \"blat.proc4005.chain_2524\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '30857207', '30857266', '100', '-', '.', 'gene_id \"chr6.g40746\"; transcript_id \"blat.proc28144.chain_2775\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '29005218', '29005319', '100', '+', '.', 'gene_id \"chr6.g40760\"; transcript_id \"blat.proc18128.chain_14102\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '14286909', '14286943', '100', '-', '.', 'gene_id \"chr6.g40820\"; transcript_id \"blat.proc28144.chain_21752\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '43535303', '43535333', '100', '-', '.', 'gene_id \"chr6.g40932\"; transcript_id \"blat.proc4005.chain_13912\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '42455966', '42456007', '100', '+', '.', 'gene_id \"chr6.g40933\"; transcript_id \"blat.proc18128.chain_12903\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '52613821', '52613861', '100', '-', '.', 'gene_id \"chr6.g40973\"; transcript_id \"blat.proc28144.chain_14904\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '19842952', '19842994', '100', '+', '.', 'gene_id \"chr6.g41001\"; transcript_id \"blat.proc18128.chain_13840\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55219330', '55219439', '100', '-', '.', 'gene_id \"chr6.g41022\"; transcript_id \"blat.proc18128.chain_14717\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '27317524', '27317699', '100', '-', '.', 'gene_id \"chr6.g41072\"; transcript_id \"blat.proc28144.chain_5191\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '11399514', '11399582', '100', '+', '.', 'gene_id \"chr6.g41147\"; transcript_id \"blat.proc28144.chain_4452\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '25935725', '25935839', '100', '+', '.', 'gene_id \"chr6.g41199\"; transcript_id \"blat.proc4587.chain_2204\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '59026072', '59026125', '100', '-', '.', 'gene_id \"chr6.g41251\"; transcript_id \"blat.proc4005.chain_20256\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '55998039', '55998149', '100', '+', '.', 'gene_id \"chr6.g41286\"; transcript_id \"blat.proc18128.chain_13532\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '66866960', '66866999', '100', '-', '.', 'gene_id \"chr6.g41310\"; transcript_id \"blat.proc28144.chain_18281\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '66187737', '66187770', '100', '-', '.', 'gene_id \"chr6.g41375\"; transcript_id \"blat.proc18128.chain_6347\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '26012007', '26012337', '100', '+', '.', 'gene_id \"chr6.g41407\"; transcript_id \"blat.proc4587.chain_5531\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '34626898', '34626931', '100', '+', '.', 'gene_id \"chr6.g41425\"; transcript_id \"blat.proc18128.chain_19968\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40714333', '40714512', '100', '+', '.', 'gene_id \"chr6.g41435\"; transcript_id \"blat.proc28144.chain_20978\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '67398764', '67398853', '100', '-', '.', 'gene_id \"chr6.g41436\"; transcript_id \"blat.proc4005.chain_12100\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '21305470', '21305519', '100', '+', '.', 'gene_id \"chr6.g41437\"; transcript_id \"blat.proc4005.chain_12642\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '68193086', '68193119', '100', '-', '.', 'gene_id \"chr6.g41458\"; transcript_id \"blat.proc28144.chain_4100\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '74729281', '74729321', '100', '-', '.', 'gene_id \"chr6.g41459\"; transcript_id \"blat.proc4587.chain_13649\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '60406524', '60406601', '100', '+', '.', 'gene_id \"chr6.g41473\"; transcript_id \"blat.proc4587.chain_13891\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '56565192', '56565234', '100', '+', '.', 'gene_id \"chr6.g41491\"; transcript_id \"blat.proc18128.chain_8134\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '69631759', '69631789', '100', '+', '.', 'gene_id \"chr6.g41678\"; transcript_id \"blat.proc4005.chain_7154\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '40333901', '40333948', '100', '-', '.', 'gene_id \"chr6.g41783\"; transcript_id \"blat.proc4005.chain_12819\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '48486455', '48486685', '100', '+', '.', 'gene_id \"chr6.g41872\"; transcript_id \"blat.proc18128.chain_6874\";']\n",
      "negative:  ['chr6', 'PASA', 'exon', '17298146', '17298183', '100', '+', '.', 'gene_id \"chr6.g41888\"; transcript_id \"blat.proc18128.chain_16491\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24244227', '24244661', '.', '-', '.', 'gene_id \"chr30.g10730\"; transcript_id \"chr30.g10730.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24267312', '24267577', '.', '+', '.', 'gene_id \"chr30.g10742\"; transcript_id \"chr30.g10742.i1\";']\n",
      "negative:  ['scaffold_1737', 'PASA', 'exon', '16350', '16392', '.', '-', '.', 'gene_id \"scaffold_1737.g6\"; transcript_id \"scaffold_1737.g6.i1\";']\n",
      "negative:  ['scaffold_1737', 'PASA', 'exon', '24920', '25075', '.', '-', '.', 'gene_id \"scaffold_1737.g17\"; transcript_id \"scaffold_1737.g17.i1\";']\n",
      "negative:  ['scaffold_1737', 'PASA', 'exon', '129396', '129451', '.', '+', '.', 'gene_id \"scaffold_1737.g65\"; transcript_id \"scaffold_1737.g65.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24542461', '24542494', '.', '-', '.', 'gene_id \"chr30.g10900\"; transcript_id \"chr30.g10900.i1\";']\n",
      "negative:  ['scaffold_1737', 'PASA', 'exon', '328139', '328212', '.', '-', '.', 'gene_id \"scaffold_1737.g178\"; transcript_id \"scaffold_1737.g178.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24725802', '24726493', '.', '+', '.', 'gene_id \"chr30.g10982\"; transcript_id \"chr30.g10982.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '42421', '42479', '.', '-', '.', 'gene_id \"chr26.g20\"; transcript_id \"chr26.g20.i3\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '42455', '42823', '.', '-', '.', 'gene_id \"chr26.g20\"; transcript_id \"chr26.g20.i4\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '42421', '42823', '.', '-', '.', 'gene_id \"chr26.g20\"; transcript_id \"chr26.g20.i5\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '153486', '153681', '.', '-', '.', 'gene_id \"chr26.g99\"; transcript_id \"chr26.g99.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25454974', '25455205', '.', '+', '.', 'gene_id \"chr30.g11230\"; transcript_id \"chr30.g11230.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25455186', '25455520', '.', '-', '.', 'gene_id \"chr30.g11231\"; transcript_id \"chr30.g11231.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25455527', '25456017', '.', '+', '.', 'gene_id \"chr30.g11233\"; transcript_id \"chr30.g11233.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '522982', '523177', '.', '+', '.', 'gene_id \"chr26.g430\"; transcript_id \"chr26.g430.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '545405', '545873', '.', '+', '.', 'gene_id \"chr26.g443\"; transcript_id \"chr26.g443.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '543140', '543793', '.', '-', '.', 'gene_id \"chr26.g450\"; transcript_id \"chr26.g450.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '546092', '546296', '.', '-', '.', 'gene_id \"chr26.g451\"; transcript_id \"chr26.g451.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '546312', '546500', '.', '-', '.', 'gene_id \"chr26.g452\"; transcript_id \"chr26.g452.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '546642', '546901', '.', '+', '.', 'gene_id \"chr26.g453\"; transcript_id \"chr26.g453.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '561826', '561856', '.', '-', '.', 'gene_id \"chr26.g481\"; transcript_id \"chr26.g481.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25701550', '25701589', '.', '+', '.', 'gene_id \"chr30.g11438\"; transcript_id \"chr30.g11438.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '579639', '579890', '.', '-', '.', 'gene_id \"chr26.g501\"; transcript_id \"chr26.g501.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '588582', '588925', '.', '+', '.', 'gene_id \"chr26.g517\"; transcript_id \"chr26.g517.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '609515', '610198', '.', '-', '.', 'gene_id \"chr26.g546\"; transcript_id \"chr26.g546.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '609515', '610113', '.', '-', '.', 'gene_id \"chr26.g546\"; transcript_id \"chr26.g546.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25918224', '25918408', '.', '-', '.', 'gene_id \"chr30.g11614\"; transcript_id \"chr30.g11614.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '26029434', '26029466', '.', '-', '.', 'gene_id \"chr30.g11701\"; transcript_id \"chr30.g11701.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '1045581', '1045740', '.', '+', '.', 'gene_id \"chr26.g836\"; transcript_id \"chr26.g836.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '26082228', '26082564', '.', '-', '.', 'gene_id \"chr30.g11751\"; transcript_id \"chr30.g11751.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '1111053', '1111101', '.', '+', '.', 'gene_id \"chr26.g914\"; transcript_id \"chr26.g914.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '26724161', '26724757', '.', '-', '.', 'gene_id \"chr30.g11874\"; transcript_id \"chr30.g11874.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '1305123', '1305350', '.', '-', '.', 'gene_id \"chr26.g1017\"; transcript_id \"chr26.g1017.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27011938', '27012379', '.', '-', '.', 'gene_id \"chr30.g11999\"; transcript_id \"chr30.g11999.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27012736', '27012932', '.', '+', '.', 'gene_id \"chr30.g12001\"; transcript_id \"chr30.g12001.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27014186', '27014460', '.', '-', '.', 'gene_id \"chr30.g12003\"; transcript_id \"chr30.g12003.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '1526253', '1526441', '.', '-', '.', 'gene_id \"chr26.g1140\"; transcript_id \"chr26.g1140.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27427556', '27427624', '.', '+', '.', 'gene_id \"chr30.g12136\"; transcript_id \"chr30.g12136.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '2423531', '2423587', '.', '+', '.', 'gene_id \"chr26.g1341\"; transcript_id \"chr26.g1341.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27613609', '27613649', '.', '-', '.', 'gene_id \"chr30.g12239\"; transcript_id \"chr30.g12239.i4\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27623995', '27624191', '.', '+', '.', 'gene_id \"chr30.g12243\"; transcript_id \"chr30.g12243.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27968941', '27969431', '.', '+', '.', 'gene_id \"chr30.g12429\"; transcript_id \"chr30.g12429.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27970034', '27970452', '.', '-', '.', 'gene_id \"chr30.g12430\"; transcript_id \"chr30.g12430.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28044939', '28044968', '.', '-', '.', 'gene_id \"chr30.g12519\"; transcript_id \"chr30.g12519.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28089639', '28089712', '.', '-', '.', 'gene_id \"chr30.g12544\"; transcript_id \"chr30.g12544.i3\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28253137', '28253358', '.', '-', '.', 'gene_id \"chr30.g12676\"; transcript_id \"chr30.g12676.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '4652220', '4652251', '.', '+', '.', 'gene_id \"chr26.g1799\"; transcript_id \"chr26.g1799.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28588696', '28588762', '.', '+', '.', 'gene_id \"chr30.g12895\"; transcript_id \"chr30.g12895.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28624716', '28624776', '.', '-', '.', 'gene_id \"chr30.g12910\"; transcript_id \"chr30.g12910.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28763700', '28768275', '.', '+', '.', 'gene_id \"chr30.g13013\"; transcript_id \"chr30.g13013.i7\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28763462', '28763668', '.', '+', '.', 'gene_id \"chr30.g13018\"; transcript_id \"chr30.g13018.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '5189175', '5189525', '.', '-', '.', 'gene_id \"chr26.g2094\"; transcript_id \"chr26.g2094.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '5189175', '5189525', '.', '-', '.', 'gene_id \"chr26.g2094\"; transcript_id \"chr26.g2094.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '5474487', '5474544', '.', '+', '.', 'gene_id \"chr26.g2309\"; transcript_id \"chr26.g2309.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29064952', '29065051', '.', '+', '.', 'gene_id \"chr30.g13299\"; transcript_id \"chr30.g13299.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29064962', '29065051', '.', '+', '.', 'gene_id \"chr30.g13299\"; transcript_id \"chr30.g13299.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29150255', '29150466', '.', '+', '.', 'gene_id \"chr30.g13404\"; transcript_id \"chr30.g13404.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29187149', '29187311', '.', '+', '.', 'gene_id \"chr30.g13433\"; transcript_id \"chr30.g13433.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29187969', '29188339', '.', '-', '.', 'gene_id \"chr30.g13434\"; transcript_id \"chr30.g13434.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29225404', '29225747', '.', '+', '.', 'gene_id \"chr30.g13489\"; transcript_id \"chr30.g13489.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29297103', '29298426', '.', '-', '.', 'gene_id \"chr30.g13559\"; transcript_id \"chr30.g13559.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29330889', '29331241', '.', '+', '.', 'gene_id \"chr30.g13596\"; transcript_id \"chr30.g13596.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29330940', '29331241', '.', '+', '.', 'gene_id \"chr30.g13596\"; transcript_id \"chr30.g13596.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29350161', '29350313', '.', '-', '.', 'gene_id \"chr30.g13636\"; transcript_id \"chr30.g13636.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29386254', '29386524', '.', '-', '.', 'gene_id \"chr30.g13672\"; transcript_id \"chr30.g13672.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6158842', '6158874', '.', '+', '.', 'gene_id \"chr26.g2785\"; transcript_id \"chr26.g2785.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29558573', '29558798', '.', '+', '.', 'gene_id \"chr30.g13817\"; transcript_id \"chr30.g13817.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29654079', '29654454', '.', '-', '.', 'gene_id \"chr30.g13919\"; transcript_id \"chr30.g13919.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29678164', '29678445', '.', '+', '.', 'gene_id \"chr30.g13939\"; transcript_id \"chr30.g13939.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29684631', '29684976', '.', '+', '.', 'gene_id \"chr30.g13948\"; transcript_id \"chr30.g13948.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6391583', '6391737', '.', '-', '.', 'gene_id \"chr26.g3032\"; transcript_id \"chr26.g3032.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29914259', '29914324', '.', '-', '.', 'gene_id \"chr30.g14106\"; transcript_id \"chr30.g14106.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6647884', '6648089', '.', '+', '.', 'gene_id \"chr26.g3271\"; transcript_id \"chr26.g3271.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30034243', '30034824', '.', '+', '.', 'gene_id \"chr30.g14209\"; transcript_id \"chr30.g14209.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6738726', '6738767', '.', '+', '.', 'gene_id \"chr26.g3373\"; transcript_id \"chr26.g3373.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30222005', '30222047', '.', '-', '.', 'gene_id \"chr30.g14346\"; transcript_id \"chr30.g14346.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6817821', '6818093', '.', '+', '.', 'gene_id \"chr26.g3450\"; transcript_id \"chr26.g3450.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6818188', '6818351', '.', '-', '.', 'gene_id \"chr26.g3451\"; transcript_id \"chr26.g3451.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '6874363', '6874407', '.', '-', '.', 'gene_id \"chr26.g3512\"; transcript_id \"chr26.g3512.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30545617', '30545649', '.', '-', '.', 'gene_id \"chr30.g14452\"; transcript_id \"chr30.g14452.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30657231', '30657297', '.', '+', '.', 'gene_id \"chr30.g14459\"; transcript_id \"chr30.g14459.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30730970', '30731164', '.', '-', '.', 'gene_id \"chr30.g14484\"; transcript_id \"chr30.g14484.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30731200', '30731706', '.', '+', '.', 'gene_id \"chr30.g14486\"; transcript_id \"chr30.g14486.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30803640', '30803674', '.', '-', '.', 'gene_id \"chr30.g14552\"; transcript_id \"chr30.g14552.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '7170562', '7170805', '.', '+', '.', 'gene_id \"chr26.g3720\"; transcript_id \"chr26.g3720.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '31084602', '31084988', '.', '+', '.', 'gene_id \"chr30.g14726\"; transcript_id \"chr30.g14726.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '31106182', '31106433', '.', '+', '.', 'gene_id \"chr30.g14767\"; transcript_id \"chr30.g14767.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '31107056', '31107282', '.', '+', '.', 'gene_id \"chr30.g14769\"; transcript_id \"chr30.g14769.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '7664254', '7664658', '.', '-', '.', 'gene_id \"chr26.g4145\"; transcript_id \"chr26.g4145.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '7864883', '7865055', '.', '+', '.', 'gene_id \"chr26.g4311\"; transcript_id \"chr26.g4311.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '31846257', '31846545', '.', '-', '.', 'gene_id \"chr30.g15176\"; transcript_id \"chr30.g15176.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8168028', '8168843', '.', '+', '.', 'gene_id \"chr26.g4563\"; transcript_id \"chr26.g4563.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32178022', '32178095', '.', '-', '.', 'gene_id \"chr30.g15410\"; transcript_id \"chr30.g15410.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32245977', '32246281', '.', '-', '.', 'gene_id \"chr30.g15461\"; transcript_id \"chr30.g15461.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32256879', '32257038', '.', '+', '.', 'gene_id \"chr30.g15478\"; transcript_id \"chr30.g15478.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8417866', '8418377', '.', '-', '.', 'gene_id \"chr26.g4674\"; transcript_id \"chr26.g4674.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8417697', '8417879', '.', '-', '.', 'gene_id \"chr26.g4674\"; transcript_id \"chr26.g4674.i3\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32305073', '32305362', '.', '+', '.', 'gene_id \"chr30.g15525\"; transcript_id \"chr30.g15525.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8488712', '8488764', '.', '-', '.', 'gene_id \"chr26.g4803\"; transcript_id \"chr26.g4803.i7\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32424587', '32424813', '.', '+', '.', 'gene_id \"chr30.g15616\"; transcript_id \"chr30.g15616.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32622986', '32623055', '.', '-', '.', 'gene_id \"chr30.g15693\"; transcript_id \"chr30.g15693.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8627690', '8628101', '.', '-', '.', 'gene_id \"chr26.g4953\"; transcript_id \"chr26.g4953.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32707946', '32708114', '.', '-', '.', 'gene_id \"chr30.g15728\"; transcript_id \"chr30.g15728.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8677407', '8677916', '.', '-', '.', 'gene_id \"chr26.g5000\"; transcript_id \"chr26.g5000.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8714032', '8714240', '.', '-', '.', 'gene_id \"chr26.g5019\"; transcript_id \"chr26.g5019.i4\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32758243', '32758275', '.', '-', '.', 'gene_id \"chr30.g15784\"; transcript_id \"chr30.g15784.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32846661', '32846859', '.', '-', '.', 'gene_id \"chr30.g15858\"; transcript_id \"chr30.g15858.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32909841', '32910013', '.', '-', '.', 'gene_id \"chr30.g15930\"; transcript_id \"chr30.g15930.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9267395', '9267664', '.', '+', '.', 'gene_id \"chr26.g5322\"; transcript_id \"chr26.g5322.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9267395', '9267664', '.', '+', '.', 'gene_id \"chr26.g5322\"; transcript_id \"chr26.g5322.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '33364132', '33364295', '.', '+', '.', 'gene_id \"chr30.g16118\"; transcript_id \"chr30.g16118.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9333686', '9333960', '.', '-', '.', 'gene_id \"chr26.g5395\"; transcript_id \"chr26.g5395.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9450352', '9450696', '.', '-', '.', 'gene_id \"chr26.g5484\"; transcript_id \"chr26.g5484.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '33747968', '33748014', '.', '+', '.', 'gene_id \"chr30.g16257\"; transcript_id \"chr30.g16257.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '33968990', '33969287', '.', '-', '.', 'gene_id \"chr30.g16280\"; transcript_id \"chr30.g16280.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr26', 'PASA', 'exon', '9674689', '9674727', '.', '+', '.', 'gene_id \"chr26.g5716\"; transcript_id \"chr26.g5716.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9717132', '9717568', '.', '+', '.', 'gene_id \"chr26.g5749\"; transcript_id \"chr26.g5749.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9782100', '9782139', '.', '-', '.', 'gene_id \"chr26.g5761\"; transcript_id \"chr26.g5761.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34675048', '34675156', '.', '+', '.', 'gene_id \"chr30.g16518\"; transcript_id \"chr30.g16518.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34726092', '34726310', '.', '+', '.', 'gene_id \"chr30.g16559\"; transcript_id \"chr30.g16559.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34726242', '34726460', '.', '+', '.', 'gene_id \"chr30.g16559\"; transcript_id \"chr30.g16559.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34726640', '34726857', '.', '+', '.', 'gene_id \"chr30.g16561\"; transcript_id \"chr30.g16561.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34726983', '34727199', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34727131', '34727349', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34727281', '34727499', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i3\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34727381', '34727599', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i4\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34727331', '34727549', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i5\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34727181', '34727399', '.', '+', '.', 'gene_id \"chr30.g16563\"; transcript_id \"chr30.g16563.i6\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34742482', '34742688', '.', '-', '.', 'gene_id \"chr30.g16571\"; transcript_id \"chr30.g16571.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35059935', '35060046', '.', '+', '.', 'gene_id \"chr30.g16664\"; transcript_id \"chr30.g16664.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '10159480', '10159513', '.', '+', '.', 'gene_id \"chr26.g6006\"; transcript_id \"chr26.g6006.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35388098', '35388133', '.', '+', '.', 'gene_id \"chr30.g16776\"; transcript_id \"chr30.g16776.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '10449576', '10449731', '.', '-', '.', 'gene_id \"chr26.g6107\"; transcript_id \"chr26.g6107.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35670580', '35670639', '.', '-', '.', 'gene_id \"chr30.g16907\"; transcript_id \"chr30.g16907.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35669853', '35670314', '.', '+', '.', 'gene_id \"chr30.g16908\"; transcript_id \"chr30.g16908.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '10755702', '10755745', '.', '+', '.', 'gene_id \"chr26.g6287\"; transcript_id \"chr26.g6287.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35795197', '35795349', '.', '+', '.', 'gene_id \"chr30.g16992\"; transcript_id \"chr30.g16992.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '10881972', '10883054', '.', '-', '.', 'gene_id \"chr26.g6400\"; transcript_id \"chr26.g6400.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '10935018', '10935080', '.', '-', '.', 'gene_id \"chr26.g6465\"; transcript_id \"chr26.g6465.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '11970478', '11970654', '.', '-', '.', 'gene_id \"chr26.g6686\"; transcript_id \"chr26.g6686.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '36570145', '36570304', '.', '+', '.', 'gene_id \"chr30.g17512\"; transcript_id \"chr30.g17512.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '36711927', '36711960', '.', '-', '.', 'gene_id \"chr30.g17583\"; transcript_id \"chr30.g17583.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37061248', '37061401', '.', '+', '.', 'gene_id \"chr30.g17701\"; transcript_id \"chr30.g17701.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37069353', '37069402', '.', '+', '.', 'gene_id \"chr30.g17708\"; transcript_id \"chr30.g17708.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13244468', '13244621', '.', '+', '.', 'gene_id \"chr26.g7062\"; transcript_id \"chr26.g7062.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13247923', '13248104', '.', '-', '.', 'gene_id \"chr26.g7067\"; transcript_id \"chr26.g7067.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13248694', '13248979', '.', '-', '.', 'gene_id \"chr26.g7070\"; transcript_id \"chr26.g7070.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13257556', '13257695', '.', '+', '.', 'gene_id \"chr26.g7083\"; transcript_id \"chr26.g7083.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37171872', '37172084', '.', '+', '.', 'gene_id \"chr30.g17775\"; transcript_id \"chr30.g17775.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13576749', '13577117', '.', '+', '.', 'gene_id \"chr26.g7183\"; transcript_id \"chr26.g7183.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37622316', '37622432', '.', '-', '.', 'gene_id \"chr30.g18014\"; transcript_id \"chr30.g18014.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37769300', '37769340', '.', '+', '.', 'gene_id \"chr30.g18100\"; transcript_id \"chr30.g18100.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37786865', '37787056', '.', '+', '.', 'gene_id \"chr30.g18119\"; transcript_id \"chr30.g18119.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '38095859', '38095963', '.', '-', '.', 'gene_id \"chr30.g18354\"; transcript_id \"chr30.g18354.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '38332918', '38337885', '.', '+', '.', 'gene_id \"chr30.g18522\"; transcript_id \"chr30.g18522.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '14915033', '14915400', '.', '+', '.', 'gene_id \"chr26.g7820\"; transcript_id \"chr26.g7820.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '14954353', '14954650', '.', '-', '.', 'gene_id \"chr26.g7871\"; transcript_id \"chr26.g7871.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '14988986', '14989166', '.', '+', '.', 'gene_id \"chr26.g7913\"; transcript_id \"chr26.g7913.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '14989157', '14989318', '.', '+', '.', 'gene_id \"chr26.g7915\"; transcript_id \"chr26.g7915.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '15005547', '15005965', '.', '-', '.', 'gene_id \"chr26.g7924\"; transcript_id \"chr26.g7924.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '15055262', '15055371', '.', '-', '.', 'gene_id \"chr26.g7930\"; transcript_id \"chr26.g7930.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '15215783', '15216044', '.', '-', '.', 'gene_id \"chr26.g8012\"; transcript_id \"chr26.g8012.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '39007328', '39007506', '.', '-', '.', 'gene_id \"chr30.g18990\"; transcript_id \"chr30.g18990.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '39085801', '39085851', '.', '-', '.', 'gene_id \"chr30.g19086\"; transcript_id \"chr30.g19086.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '39270870', '39270899', '.', '+', '.', 'gene_id \"chr30.g19238\"; transcript_id \"chr30.g19238.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '39350304', '39350983', '.', '+', '.', 'gene_id \"chr30.g19295\"; transcript_id \"chr30.g19295.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '16682194', '16682354', '.', '-', '.', 'gene_id \"chr26.g8763\"; transcript_id \"chr26.g8763.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '16671505', '16671748', '.', '+', '.', 'gene_id \"chr26.g8778\"; transcript_id \"chr26.g8778.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '16681784', '16682136', '.', '+', '.', 'gene_id \"chr26.g8785\"; transcript_id \"chr26.g8785.i2\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '39999129', '39999412', '.', '+', '.', 'gene_id \"chr30.g19613\"; transcript_id \"chr30.g19613.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '40058747', '40058966', '.', '-', '.', 'gene_id \"chr30.g19694\"; transcript_id \"chr30.g19694.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '40060013', '40060223', '.', '-', '.', 'gene_id \"chr30.g19695\"; transcript_id \"chr30.g19695.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17038524', '17039132', '.', '-', '.', 'gene_id \"chr26.g9056\"; transcript_id \"chr26.g9056.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17315381', '17315668', '.', '+', '.', 'gene_id \"chr26.g9146\"; transcript_id \"chr26.g9146.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17316381', '17316701', '.', '+', '.', 'gene_id \"chr26.g9149\"; transcript_id \"chr26.g9149.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17316896', '17317015', '.', '+', '.', 'gene_id \"chr26.g9150\"; transcript_id \"chr26.g9150.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17379249', '17380178', '.', '+', '.', 'gene_id \"chr26.g9153\"; transcript_id \"chr26.g9153.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17379250', '17380178', '.', '+', '.', 'gene_id \"chr26.g9153\"; transcript_id \"chr26.g9153.i3\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17379248', '17380178', '.', '+', '.', 'gene_id \"chr26.g9153\"; transcript_id \"chr26.g9153.i4\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17379247', '17380178', '.', '+', '.', 'gene_id \"chr26.g9153\"; transcript_id \"chr26.g9153.i5\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17318599', '17318669', '.', '+', '.', 'gene_id \"chr26.g9153\"; transcript_id \"chr26.g9153.i6\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17437525', '17438176', '.', '+', '.', 'gene_id \"chr26.g9157\"; transcript_id \"chr26.g9157.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17331883', '17332064', '.', '-', '.', 'gene_id \"chr26.g9163\"; transcript_id \"chr26.g9163.i1\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11225444', '11225494', '100', '+', '.', 'gene_id \"chr30.g19949\"; transcript_id \"blat.proc4005.chain_7805\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '13299747', '13299992', '100', '+', '.', 'gene_id \"chr30.g19980\"; transcript_id \"blat.proc18128.chain_17303\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6407704', '6407915', '100', '+', '.', 'gene_id \"chr30.g19986\"; transcript_id \"blat.proc4587.chain_4367\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '33761171', '33761214', '100', '+', '.', 'gene_id \"chr30.g20014\"; transcript_id \"blat.proc28144.chain_12523\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27388459', '27388488', '100', '+', '.', 'gene_id \"chr30.g20051\"; transcript_id \"blat.proc4587.chain_1015\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32991453', '32991519', '100', '+', '.', 'gene_id \"chr30.g20167\"; transcript_id \"blat.proc4587.chain_8416\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29887429', '29887520', '100', '+', '.', 'gene_id \"chr30.g20194\"; transcript_id \"blat.proc4587.chain_46\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11016215', '11016249', '100', '+', '.', 'gene_id \"chr30.g20228\"; transcript_id \"blat.proc18128.chain_10581\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7950358', '7950687', '100', '-', '.', 'gene_id \"chr30.g20417\"; transcript_id \"blat.proc4587.chain_11785\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '595540', '595612', '100', '-', '.', 'gene_id \"chr30.g20431\"; transcript_id \"blat.proc18128.chain_6298\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '17819812', '17819876', '100', '+', '.', 'gene_id \"chr30.g20489\"; transcript_id \"blat.proc18128.chain_10053\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '5589518', '5589563', '100', '+', '.', 'gene_id \"chr30.g20517\"; transcript_id \"blat.proc18128.chain_7259\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29355569', '29355603', '100', '+', '.', 'gene_id \"chr30.g20573\"; transcript_id \"blat.proc18128.chain_12023\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16826649', '16826687', '100', '+', '.', 'gene_id \"chr30.g20601\"; transcript_id \"blat.proc28144.chain_16007\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '36567319', '36567365', '100', '+', '.', 'gene_id \"chr30.g20729\"; transcript_id \"blat.proc4005.chain_5157\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1136373', '1136415', '100', '+', '.', 'gene_id \"chr30.g20846\"; transcript_id \"blat.proc4005.chain_13713\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '3212781', '3212815', '100', '+', '.', 'gene_id \"chr30.g20852\"; transcript_id \"blat.proc18128.chain_18105\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '37101323', '37101356', '100', '-', '.', 'gene_id \"chr30.g20860\"; transcript_id \"blat.proc4587.chain_1361\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11829562', '11829608', '100', '+', '.', 'gene_id \"chr30.g21034\"; transcript_id \"blat.proc28144.chain_17697\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25236904', '25237089', '100', '+', '.', 'gene_id \"chr30.g21102\"; transcript_id \"blat.proc18128.chain_4073\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22969695', '22969727', '100', '+', '.', 'gene_id \"chr30.g21128\"; transcript_id \"blat.proc4005.chain_20097\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23462902', '23462937', '100', '-', '.', 'gene_id \"chr30.g21153\"; transcript_id \"blat.proc18128.chain_24763\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '32227929', '32227970', '100', '+', '.', 'gene_id \"chr30.g21177\"; transcript_id \"blat.proc28144.chain_9579\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28762888', '28763113', '100', '-', '.', 'gene_id \"chr30.g21196\"; transcript_id \"blat.proc18128.chain_21920\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30223815', '30224041', '100', '-', '.', 'gene_id \"chr30.g21284\"; transcript_id \"blat.proc18128.chain_25597\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '28472698', '28472757', '100', '+', '.', 'gene_id \"chr30.g21404\"; transcript_id \"blat.proc4587.chain_12828\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7974435', '7974466', '100', '-', '.', 'gene_id \"chr30.g21460\"; transcript_id \"blat.proc4005.chain_21046\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18087742', '18087839', '100', '-', '.', 'gene_id \"chr30.g21473\"; transcript_id \"blat.proc18128.chain_20259\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '18342726', '18342788', '100', '+', '.', 'gene_id \"chr30.g21526\"; transcript_id \"blat.proc18128.chain_18759\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '2596724', '2596774', '100', '-', '.', 'gene_id \"chr30.g21550\"; transcript_id \"blat.proc28144.chain_20550\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7323893', '7323925', '100', '+', '.', 'gene_id \"chr30.g21593\"; transcript_id \"blat.proc18128.chain_13483\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '3253317', '3253379', '100', '+', '.', 'gene_id \"chr30.g21642\"; transcript_id \"blat.proc18128.chain_7496\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11670812', '11670967', '100', '-', '.', 'gene_id \"chr30.g21660\"; transcript_id \"blat.proc18128.chain_9878\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11157662', '11157701', '100', '+', '.', 'gene_id \"chr30.g21666\"; transcript_id \"blat.proc4587.chain_3896\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '3646974', '3647019', '100', '+', '.', 'gene_id \"chr30.g21710\"; transcript_id \"blat.proc28144.chain_19680\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '22851261', '22851391', '100', '+', '.', 'gene_id \"chr30.g21762\"; transcript_id \"blat.proc4587.chain_2395\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '36821883', '36821920', '100', '+', '.', 'gene_id \"chr30.g21841\"; transcript_id \"blat.proc4587.chain_12325\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '13009933', '13010003', '100', '-', '.', 'gene_id \"chr30.g21860\"; transcript_id \"blat.proc18128.chain_21667\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '23467462', '23467548', '100', '+', '.', 'gene_id \"chr30.g21970\"; transcript_id \"blat.proc18128.chain_5912\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29443068', '29443187', '100', '-', '.', 'gene_id \"chr30.g22072\"; transcript_id \"blat.proc28144.chain_7371\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '31498698', '31498729', '100', '-', '.', 'gene_id \"chr30.g22179\"; transcript_id \"blat.proc28144.chain_5481\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '5674723', '5674778', '100', '+', '.', 'gene_id \"chr30.g22286\"; transcript_id \"blat.proc4587.chain_2716\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '7919734', '7919830', '100', '+', '.', 'gene_id \"chr30.g22367\"; transcript_id \"blat.proc4587.chain_5378\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16134087', '16134135', '100', '+', '.', 'gene_id \"chr30.g22380\"; transcript_id \"blat.proc4005.chain_20741\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '36570305', '36570490', '100', '-', '.', 'gene_id \"chr30.g22394\"; transcript_id \"blat.proc4587.chain_3949\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25768369', '25768419', '100', '+', '.', 'gene_id \"chr30.g22806\"; transcript_id \"blat.proc4005.chain_15963\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12593516', '12593564', '100', '-', '.', 'gene_id \"chr30.g22976\"; transcript_id \"blat.proc4005.chain_19178\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '30649235', '30649273', '100', '+', '.', 'gene_id \"chr30.g22990\"; transcript_id \"blat.proc18128.chain_21247\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16059861', '16060047', '100', '+', '.', 'gene_id \"chr30.g23035\"; transcript_id \"blat.proc18128.chain_4072\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11685870', '11686023', '100', '-', '.', 'gene_id \"chr30.g23190\"; transcript_id \"blat.proc18128.chain_9879\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '40152758', '40152815', '100', '-', '.', 'gene_id \"chr30.g23216\"; transcript_id \"blat.proc28144.chain_19967\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12631006', '12631283', '100', '+', '.', 'gene_id \"chr30.g23280\"; transcript_id \"blat.proc18128.chain_7061\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '34937462', '34937522', '100', '+', '.', 'gene_id \"chr30.g23356\"; transcript_id \"blat.proc18128.chain_8052\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '35695459', '35695519', '100', '-', '.', 'gene_id \"chr30.g23375\"; transcript_id \"blat.proc18128.chain_13496\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '15235023', '15235102', '100', '+', '.', 'gene_id \"chr30.g23436\"; transcript_id \"blat.proc28144.chain_19225\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27634068', '27634112', '100', '+', '.', 'gene_id \"chr30.g23564\"; transcript_id \"blat.proc4005.chain_10273\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '13395778', '13395810', '100', '-', '.', 'gene_id \"chr30.g23747\"; transcript_id \"blat.proc18128.chain_24251\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12226028', '12226078', '100', '-', '.', 'gene_id \"chr30.g23769\"; transcript_id \"blat.proc28144.chain_19466\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '25614568', '25614615', '100', '+', '.', 'gene_id \"chr30.g23844\"; transcript_id \"blat.proc28144.chain_20037\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11253686', '11253715', '100', '+', '.', 'gene_id \"chr30.g23845\"; transcript_id \"blat.proc28144.chain_2183\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '8695097', '8695126', '100', '-', '.', 'gene_id \"chr30.g24010\"; transcript_id \"blat.proc4587.chain_2038\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '29259236', '29259320', '100', '+', '.', 'gene_id \"chr30.g24074\"; transcript_id \"blat.proc18128.chain_25603\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '27949300', '27949351', '100', '+', '.', 'gene_id \"chr30.g24107\"; transcript_id \"blat.proc28144.chain_1083\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6604090', '6604125', '100', '+', '.', 'gene_id \"chr30.g24151\"; transcript_id \"blat.proc28144.chain_21990\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16370461', '16370636', '100', '-', '.', 'gene_id \"chr30.g24156\"; transcript_id \"blat.proc18128.chain_20710\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12608810', '12608843', '100', '+', '.', 'gene_id \"chr30.g24251\"; transcript_id \"blat.proc28144.chain_275\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '17630111', '17630304', '100', '+', '.', 'gene_id \"chr30.g24266\"; transcript_id \"blat.proc28144.chain_19848\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '12407006', '12407046', '100', '-', '.', 'gene_id \"chr30.g24297\"; transcript_id \"blat.proc28144.chain_19784\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '6870952', '6871026', '100', '+', '.', 'gene_id \"chr30.g24783\"; transcript_id \"blat.proc4587.chain_5951\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '24456314', '24456381', '100', '+', '.', 'gene_id \"chr30.g24789\"; transcript_id \"blat.proc18128.chain_10950\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '8710871', '8710922', '100', '-', '.', 'gene_id \"chr30.g24808\"; transcript_id \"blat.proc18128.chain_10543\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '11058257', '11058296', '100', '+', '.', 'gene_id \"chr30.g24826\"; transcript_id \"blat.proc18128.chain_20566\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '16599632', '16599697', '100', '+', '.', 'gene_id \"chr30.g24833\"; transcript_id \"blat.proc18128.chain_14953\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '20663950', '20663998', '100', '-', '.', 'gene_id \"chr30.g24894\"; transcript_id \"blat.proc18128.chain_12094\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '1032409', '1032682', '100', '-', '.', 'gene_id \"chr30.g24914\"; transcript_id \"blat.proc4587.chain_3738\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '14843024', '14843092', '100', '-', '.', 'gene_id \"chr30.g24921\"; transcript_id \"blat.proc28144.chain_19589\";']\n",
      "negative:  ['chr30', 'PASA', 'exon', '13312935', '13313718', '100', '-', '.', 'gene_id \"chr30.g24965\"; transcript_id \"blat.proc28144.chain_7238\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '17418000', '17418198', '.', '+', '.', 'gene_id \"chr26.g9192\"; transcript_id \"chr26.g9192.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '47909', '48123', '.', '-', '.', 'gene_id \"chr12.g10\"; transcript_id \"chr12.g10.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18055605', '18055645', '.', '-', '.', 'gene_id \"chr26.g9593\"; transcript_id \"chr26.g9593.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '436775', '437046', '.', '-', '.', 'gene_id \"chr12.g273\"; transcript_id \"chr12.g273.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18515552', '18515727', '.', '-', '.', 'gene_id \"chr26.g9905\"; transcript_id \"chr26.g9905.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18819156', '18819321', '.', '+', '.', 'gene_id \"chr26.g10118\"; transcript_id \"chr26.g10118.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18840498', '18840533', '.', '-', '.', 'gene_id \"chr26.g10136\"; transcript_id \"chr26.g10136.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18964033', '18964856', '.', '+', '.', 'gene_id \"chr26.g10201\"; transcript_id \"chr26.g10201.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr26', 'PASA', 'exon', '19092270', '19092304', '.', '-', '.', 'gene_id \"chr26.g10265\"; transcript_id \"chr26.g10265.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19324288', '19324717', '.', '+', '.', 'gene_id \"chr26.g10401\"; transcript_id \"chr26.g10401.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19333955', '19334169', '.', '-', '.', 'gene_id \"chr26.g10410\"; transcript_id \"chr26.g10410.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19334893', '19335379', '.', '+', '.', 'gene_id \"chr26.g10412\"; transcript_id \"chr26.g10412.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19335372', '19335585', '.', '-', '.', 'gene_id \"chr26.g10413\"; transcript_id \"chr26.g10413.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19748280', '19748396', '.', '-', '.', 'gene_id \"chr26.g10538\"; transcript_id \"chr26.g10538.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '19748321', '19748399', '.', '-', '.', 'gene_id \"chr26.g10538\"; transcript_id \"chr26.g10538.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '20427705', '20427739', '.', '-', '.', 'gene_id \"chr26.g10778\"; transcript_id \"chr26.g10778.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '2644956', '2645108', '.', '-', '.', 'gene_id \"chr12.g1554\"; transcript_id \"chr12.g1554.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '2830945', '2831102', '.', '-', '.', 'gene_id \"chr12.g1676\"; transcript_id \"chr12.g1676.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21565291', '21565344', '.', '+', '.', 'gene_id \"chr26.g11040\"; transcript_id \"chr26.g11040.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '3110760', '3110923', '.', '+', '.', 'gene_id \"chr12.g1892\"; transcript_id \"chr12.g1892.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '3110987', '3111189', '.', '-', '.', 'gene_id \"chr12.g1893\"; transcript_id \"chr12.g1893.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21600514', '21600704', '.', '+', '.', 'gene_id \"chr26.g11080\"; transcript_id \"chr26.g11080.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21601251', '21601628', '.', '-', '.', 'gene_id \"chr26.g11083\"; transcript_id \"chr26.g11083.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21672476', '21672628', '.', '+', '.', 'gene_id \"chr26.g11126\"; transcript_id \"chr26.g11126.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21791780', '21792029', '.', '+', '.', 'gene_id \"chr26.g11174\"; transcript_id \"chr26.g11174.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '21867055', '21867115', '.', '+', '.', 'gene_id \"chr26.g11211\"; transcript_id \"chr26.g11211.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '3588091', '3588163', '.', '+', '.', 'gene_id \"chr12.g2124\"; transcript_id \"chr12.g2124.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '3590209', '3590284', '.', '+', '.', 'gene_id \"chr12.g2128\"; transcript_id \"chr12.g2128.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22034675', '22034871', '.', '+', '.', 'gene_id \"chr26.g11317\"; transcript_id \"chr26.g11317.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22036587', '22036627', '.', '+', '.', 'gene_id \"chr26.g11322\"; transcript_id \"chr26.g11322.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22038628', '22038923', '.', '+', '.', 'gene_id \"chr26.g11325\"; transcript_id \"chr26.g11325.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22337593', '22337629', '.', '+', '.', 'gene_id \"chr26.g11546\"; transcript_id \"chr26.g11546.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22627822', '22628305', '.', '-', '.', 'gene_id \"chr26.g11723\"; transcript_id \"chr26.g11723.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22644551', '22647578', '.', '+', '.', 'gene_id \"chr26.g11744\"; transcript_id \"chr26.g11744.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22652803', '22655247', '.', '+', '.', 'gene_id \"chr26.g11745\"; transcript_id \"chr26.g11745.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22866563', '22866725', '.', '-', '.', 'gene_id \"chr26.g11903\"; transcript_id \"chr26.g11903.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22901519', '22901929', '.', '+', '.', 'gene_id \"chr26.g11950\"; transcript_id \"chr26.g11950.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22902139', '22902449', '.', '+', '.', 'gene_id \"chr26.g11951\"; transcript_id \"chr26.g11951.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22902579', '22902811', '.', '-', '.', 'gene_id \"chr26.g11952\"; transcript_id \"chr26.g11952.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22912166', '22912638', '.', '+', '.', 'gene_id \"chr26.g11959\"; transcript_id \"chr26.g11959.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '4497539', '4497825', '.', '-', '.', 'gene_id \"chr12.g2865\"; transcript_id \"chr12.g2865.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '4607240', '4607394', '.', '+', '.', 'gene_id \"chr12.g2986\"; transcript_id \"chr12.g2986.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '24243208', '24243253', '.', '-', '.', 'gene_id \"chr26.g12787\"; transcript_id \"chr26.g12787.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '5533531', '5533708', '.', '+', '.', 'gene_id \"chr12.g3667\"; transcript_id \"chr12.g3667.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '5620339', '5620567', '.', '-', '.', 'gene_id \"chr12.g3724\"; transcript_id \"chr12.g3724.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '5620107', '5620292', '.', '-', '.', 'gene_id \"chr12.g3741\"; transcript_id \"chr12.g3741.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '5924812', '5925209', '.', '+', '.', 'gene_id \"chr12.g3883\"; transcript_id \"chr12.g3883.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '26118026', '26118113', '.', '-', '.', 'gene_id \"chr26.g13444\"; transcript_id \"chr26.g13444.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '26553522', '26553648', '.', '+', '.', 'gene_id \"chr26.g13486\"; transcript_id \"chr26.g13486.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '26563127', '26563159', '.', '-', '.', 'gene_id \"chr26.g13490\"; transcript_id \"chr26.g13490.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '26649028', '26649226', '.', '-', '.', 'gene_id \"chr26.g13506\"; transcript_id \"chr26.g13506.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '6883544', '6883776', '.', '+', '.', 'gene_id \"chr12.g4511\"; transcript_id \"chr12.g4511.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27348191', '27348332', '.', '-', '.', 'gene_id \"chr26.g13766\"; transcript_id \"chr26.g13766.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '6932987', '6933273', '.', '-', '.', 'gene_id \"chr12.g4558\"; transcript_id \"chr12.g4558.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27429828', '27430074', '.', '+', '.', 'gene_id \"chr26.g13796\"; transcript_id \"chr26.g13796.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27631707', '27632130', '.', '+', '.', 'gene_id \"chr26.g13959\"; transcript_id \"chr26.g13959.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27632133', '27632510', '.', '-', '.', 'gene_id \"chr26.g13960\"; transcript_id \"chr26.g13960.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27632651', '27632838', '.', '-', '.', 'gene_id \"chr26.g13961\"; transcript_id \"chr26.g13961.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27633673', '27633926', '.', '+', '.', 'gene_id \"chr26.g13963\"; transcript_id \"chr26.g13963.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27713503', '27713549', '.', '+', '.', 'gene_id \"chr26.g14020\"; transcript_id \"chr26.g14020.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '7153446', '7153613', '.', '-', '.', 'gene_id \"chr12.g4783\"; transcript_id \"chr12.g4783.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27917034', '27917349', '.', '-', '.', 'gene_id \"chr26.g14193\"; transcript_id \"chr26.g14193.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '28019173', '28019494', '.', '-', '.', 'gene_id \"chr26.g14237\"; transcript_id \"chr26.g14237.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '28208240', '28208603', '.', '-', '.', 'gene_id \"chr26.g14443\"; transcript_id \"chr26.g14443.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '28268112', '28268195', '.', '-', '.', 'gene_id \"chr26.g14482\"; transcript_id \"chr26.g14482.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '28863209', '28863504', '.', '-', '.', 'gene_id \"chr26.g14872\"; transcript_id \"chr26.g14872.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '29178891', '29179119', '.', '-', '.', 'gene_id \"chr26.g15066\"; transcript_id \"chr26.g15066.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '9987872', '9988128', '.', '+', '.', 'gene_id \"chr12.g5855\"; transcript_id \"chr12.g5855.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10066683', '10066972', '.', '+', '.', 'gene_id \"chr12.g5878\"; transcript_id \"chr12.g5878.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '29494565', '29494610', '.', '-', '.', 'gene_id \"chr26.g15312\"; transcript_id \"chr26.g15312.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10343080', '10343265', '.', '-', '.', 'gene_id \"chr12.g5963\"; transcript_id \"chr12.g5963.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10343522', '10343727', '.', '-', '.', 'gene_id \"chr12.g5964\"; transcript_id \"chr12.g5964.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10430223', '10430599', '.', '+', '.', 'gene_id \"chr12.g5972\"; transcript_id \"chr12.g5972.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10381748', '10381962', '.', '-', '.', 'gene_id \"chr12.g5973\"; transcript_id \"chr12.g5973.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10382039', '10382202', '.', '-', '.', 'gene_id \"chr12.g5979\"; transcript_id \"chr12.g5979.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10421865', '10422044', '.', '-', '.', 'gene_id \"chr12.g6037\"; transcript_id \"chr12.g6037.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10461044', '10461209', '.', '-', '.', 'gene_id \"chr12.g6067\"; transcript_id \"chr12.g6067.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '10462282', '10462458', '.', '-', '.', 'gene_id \"chr12.g6070\"; transcript_id \"chr12.g6070.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '29831408', '29831570', '.', '+', '.', 'gene_id \"chr26.g15650\"; transcript_id \"chr26.g15650.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30362296', '30362669', '.', '-', '.', 'gene_id \"chr26.g16083\"; transcript_id \"chr26.g16083.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30539469', '30539543', '.', '-', '.', 'gene_id \"chr26.g16278\"; transcript_id \"chr26.g16278.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30539330', '30539543', '.', '-', '.', 'gene_id \"chr26.g16278\"; transcript_id \"chr26.g16278.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30567051', '30567245', '.', '+', '.', 'gene_id \"chr26.g16317\"; transcript_id \"chr26.g16317.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '31410177', '31410239', '.', '-', '.', 'gene_id \"chr26.g16524\"; transcript_id \"chr26.g16524.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '31734988', '31735024', '.', '+', '.', 'gene_id \"chr26.g16540\"; transcript_id \"chr26.g16540.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '31863072', '31863332', '.', '+', '.', 'gene_id \"chr26.g16546\"; transcript_id \"chr26.g16546.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '11744809', '11744872', '.', '-', '.', 'gene_id \"chr12.g7203\"; transcript_id \"chr12.g7203.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '32428939', '32428980', '.', '-', '.', 'gene_id \"chr26.g16648\"; transcript_id \"chr26.g16648.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '32554928', '32555312', '.', '+', '.', 'gene_id \"chr26.g16659\"; transcript_id \"chr26.g16659.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '32775743', '32775788', '.', '-', '.', 'gene_id \"chr26.g16690\"; transcript_id \"chr26.g16690.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '33865188', '33865235', '.', '-', '.', 'gene_id \"chr26.g16763\"; transcript_id \"chr26.g16763.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr26', 'PASA', 'exon', '35741040', '35741334', '.', '-', '.', 'gene_id \"chr26.g17148\"; transcript_id \"chr26.g17148.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '35849459', '35849680', '.', '+', '.', 'gene_id \"chr26.g17218\"; transcript_id \"chr26.g17218.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36018094', '36018307', '.', '-', '.', 'gene_id \"chr26.g17357\"; transcript_id \"chr26.g17357.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36205379', '36205414', '.', '+', '.', 'gene_id \"chr26.g17500\"; transcript_id \"chr26.g17500.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36474407', '36474606', '.', '-', '.', 'gene_id \"chr26.g17608\"; transcript_id \"chr26.g17608.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36542506', '36542754', '.', '+', '.', 'gene_id \"chr26.g17668\"; transcript_id \"chr26.g17668.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36542758', '36543006', '.', '-', '.', 'gene_id \"chr26.g17669\"; transcript_id \"chr26.g17669.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '14460348', '14460556', '.', '-', '.', 'gene_id \"chr12.g8557\"; transcript_id \"chr12.g8557.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36827089', '36827237', '.', '-', '.', 'gene_id \"chr26.g17881\"; transcript_id \"chr26.g17881.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36862828', '36862905', '.', '-', '.', 'gene_id \"chr26.g17930\"; transcript_id \"chr26.g17930.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36935869', '36936132', '.', '+', '.', 'gene_id \"chr26.g17995\"; transcript_id \"chr26.g17995.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37068653', '37068810', '.', '+', '.', 'gene_id \"chr26.g18063\"; transcript_id \"chr26.g18063.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37089426', '37089483', '.', '-', '.', 'gene_id \"chr26.g18066\"; transcript_id \"chr26.g18066.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37141614', '37141781', '.', '+', '.', 'gene_id \"chr26.g18083\"; transcript_id \"chr26.g18083.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '15151555', '15151600', '.', '+', '.', 'gene_id \"chr12.g8899\"; transcript_id \"chr12.g8899.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37334844', '37335733', '.', '-', '.', 'gene_id \"chr26.g18295\"; transcript_id \"chr26.g18295.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37335718', '37336027', '.', '+', '.', 'gene_id \"chr26.g18296\"; transcript_id \"chr26.g18296.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37837265', '37837532', '.', '+', '.', 'gene_id \"chr26.g18375\"; transcript_id \"chr26.g18375.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37837178', '37837532', '.', '+', '.', 'gene_id \"chr26.g18375\"; transcript_id \"chr26.g18375.i2\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37711005', '37711358', '.', '+', '.', 'gene_id \"chr26.g18491\"; transcript_id \"chr26.g18491.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '17268372', '17268420', '.', '+', '.', 'gene_id \"chr12.g9325\"; transcript_id \"chr12.g9325.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '38205636', '38205684', '.', '-', '.', 'gene_id \"chr26.g18860\"; transcript_id \"chr26.g18860.i1\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '15714908', '15715129', '100', '+', '.', 'gene_id \"chr26.g19098\"; transcript_id \"blat.proc28144.chain_22410\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '24410924', '24410987', '100', '-', '.', 'gene_id \"chr26.g19116\"; transcript_id \"blat.proc18128.chain_12684\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13295346', '13295383', '100', '-', '.', 'gene_id \"chr26.g19166\"; transcript_id \"blat.proc4587.chain_12330\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '23768364', '23768397', '100', '-', '.', 'gene_id \"chr26.g19199\"; transcript_id \"blat.proc4005.chain_22179\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '3395009', '3395053', '100', '+', '.', 'gene_id \"chr26.g19234\"; transcript_id \"blat.proc4005.chain_18285\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '4466089', '4466162', '100', '+', '.', 'gene_id \"chr26.g19383\"; transcript_id \"blat.proc4587.chain_5121\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '23412094', '23412146', '100', '+', '.', 'gene_id \"chr26.g19424\"; transcript_id \"blat.proc28144.chain_1376\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18592804', '18592845', '100', '+', '.', 'gene_id \"chr26.g19689\"; transcript_id \"blat.proc4587.chain_10021\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '5473864', '5473898', '100', '+', '.', 'gene_id \"chr26.g19721\"; transcript_id \"blat.proc28144.chain_9013\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22713910', '22713955', '100', '-', '.', 'gene_id \"chr26.g19836\"; transcript_id \"blat.proc18128.chain_14215\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '16036472', '16036504', '100', '+', '.', 'gene_id \"chr26.g20002\"; transcript_id \"blat.proc18128.chain_23055\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '8315979', '8316023', '100', '+', '.', 'gene_id \"chr26.g20082\"; transcript_id \"blat.proc4587.chain_5075\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30765818', '30765895', '100', '-', '.', 'gene_id \"chr26.g20193\"; transcript_id \"blat.proc18128.chain_11205\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30451668', '30452087', '100', '+', '.', 'gene_id \"chr26.g20223\"; transcript_id \"blat.proc28144.chain_11077\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '25450988', '25451114', '100', '+', '.', 'gene_id \"chr26.g20228\"; transcript_id \"blat.proc4587.chain_13990\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '37054145', '37054179', '100', '+', '.', 'gene_id \"chr26.g20236\"; transcript_id \"blat.proc4587.chain_8532\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '24935324', '24935485', '100', '+', '.', 'gene_id \"chr26.g20364\"; transcript_id \"blat.proc18128.chain_20228\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27086537', '27086764', '100', '+', '.', 'gene_id \"chr26.g20643\"; transcript_id \"blat.proc18128.chain_21895\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30768692', '30768878', '100', '+', '.', 'gene_id \"chr26.g20669\"; transcript_id \"blat.proc18128.chain_26484\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '842905', '843079', '100', '+', '.', 'gene_id \"chr26.g20688\"; transcript_id \"blat.proc4587.chain_13065\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '20429989', '20430030', '100', '+', '.', 'gene_id \"chr26.g20720\"; transcript_id \"blat.proc18128.chain_20989\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '18953139', '18953168', '100', '+', '.', 'gene_id \"chr26.g20885\"; transcript_id \"blat.proc18128.chain_9430\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '29283279', '29283325', '100', '+', '.', 'gene_id \"chr26.g21012\"; transcript_id \"blat.proc18128.chain_21599\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27242', '27331', '100', '-', '.', 'gene_id \"chr26.g21068\"; transcript_id \"blat.proc18128.chain_6384\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '14914487', '14914535', '100', '-', '.', 'gene_id \"chr26.g21286\"; transcript_id \"blat.proc18128.chain_11684\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '23706051', '23706092', '100', '+', '.', 'gene_id \"chr26.g21305\"; transcript_id \"blat.proc28144.chain_11571\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9474082', '9474122', '100', '-', '.', 'gene_id \"chr26.g21591\"; transcript_id \"blat.proc18128.chain_13776\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36048140', '36048254', '100', '-', '.', 'gene_id \"chr26.g21657\"; transcript_id \"blat.proc28144.chain_7372\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '34770769', '34770824', '100', '+', '.', 'gene_id \"chr26.g21673\"; transcript_id \"blat.proc4587.chain_3745\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '33543674', '33543733', '100', '+', '.', 'gene_id \"chr26.g21699\"; transcript_id \"blat.proc28144.chain_10618\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '27149199', '27149243', '100', '-', '.', 'gene_id \"chr26.g21856\"; transcript_id \"blat.proc4005.chain_10226\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '15364759', '15364925', '100', '+', '.', 'gene_id \"chr26.g21939\"; transcript_id \"blat.proc18128.chain_7109\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '5871343', '5871382', '100', '+', '.', 'gene_id \"chr26.g22030\"; transcript_id \"blat.proc18128.chain_24833\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '4991811', '4991843', '100', '-', '.', 'gene_id \"chr26.g22133\"; transcript_id \"blat.proc18128.chain_1793\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '12268073', '12268172', '100', '-', '.', 'gene_id \"chr26.g22193\"; transcript_id \"blat.proc28144.chain_16559\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '30938290', '30938321', '100', '+', '.', 'gene_id \"chr26.g22667\"; transcript_id \"blat.proc4587.chain_8014\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '7772688', '7772731', '100', '+', '.', 'gene_id \"chr26.g22766\"; transcript_id \"blat.proc4587.chain_7572\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '22901344', '22901507', '100', '+', '.', 'gene_id \"chr26.g22822\"; transcript_id \"blat.proc4587.chain_7787\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '9137223', '9137262', '100', '-', '.', 'gene_id \"chr26.g22881\"; transcript_id \"blat.proc4005.chain_5405\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '23106567', '23106621', '100', '+', '.', 'gene_id \"chr26.g23101\"; transcript_id \"blat.proc18128.chain_924\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '36655680', '36655722', '100', '+', '.', 'gene_id \"chr26.g23157\"; transcript_id \"blat.proc4005.chain_4260\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '32080378', '32080411', '100', '-', '.', 'gene_id \"chr26.g23320\"; transcript_id \"blat.proc18128.chain_18020\";']\n",
      "negative:  ['chr26', 'PASA', 'exon', '13498528', '13498567', '100', '-', '.', 'gene_id \"chr26.g23488\"; transcript_id \"blat.proc28144.chain_21709\";']\n",
      "negative:  ['scaffold_1041', 'PASA', 'exon', '4045', '6031', '.', '-', '.', 'gene_id \"scaffold_1041.g5\"; transcript_id \"scaffold_1041.g5.i1\";']\n",
      "negative:  ['scaffold_1041', 'PASA', 'exon', '6027', '6706', '.', '+', '.', 'gene_id \"scaffold_1041.g6\"; transcript_id \"scaffold_1041.g6.i1\";']\n",
      "negative:  ['scaffold_1041', 'PASA', 'exon', '6684', '7622', '.', '+', '.', 'gene_id \"scaffold_1041.g7\"; transcript_id \"scaffold_1041.g7.i1\";']\n",
      "negative:  ['scaffold_1704', 'PASA', 'exon', '14122', '14549', '.', '-', '.', 'gene_id \"scaffold_1704.g9\"; transcript_id \"scaffold_1704.g9.i1\";']\n",
      "negative:  ['scaffold_1704', 'PASA', 'exon', '103203', '103271', '.', '+', '.', 'gene_id \"scaffold_1704.g53\"; transcript_id \"scaffold_1704.g53.i2\";']\n",
      "negative:  ['scaffold_1233', 'PASA', 'exon', '1054', '1102', '100', '+', '.', 'gene_id \"scaffold_1233.g4\"; transcript_id \"blat.proc4587.chain_4554\";']\n",
      "negative:  ['scaffold_1273', 'PASA', 'exon', '14861', '15040', '.', '+', '.', 'gene_id \"scaffold_1273.g8\"; transcript_id \"scaffold_1273.g8.i1\";']\n",
      "negative:  ['scaffold_1273', 'PASA', 'exon', '15315', '15432', '.', '-', '.', 'gene_id \"scaffold_1273.g12\"; transcript_id \"scaffold_1273.g12.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '501316', '501838', '.', '-', '.', 'gene_id \"chr29.g256\"; transcript_id \"chr29.g256.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '555541', '555584', '.', '+', '.', 'gene_id \"chr29.g259\"; transcript_id \"chr29.g259.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '591830', '592050', '.', '+', '.', 'gene_id \"chr29.g263\"; transcript_id \"chr29.g263.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '749150', '749320', '.', '-', '.', 'gene_id \"chr29.g281\"; transcript_id \"chr29.g281.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '825139', '825376', '.', '+', '.', 'gene_id \"chr29.g302\"; transcript_id \"chr29.g302.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '1412366', '1412410', '.', '+', '.', 'gene_id \"chr29.g402\"; transcript_id \"chr29.g402.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '1432375', '1432433', '.', '+', '.', 'gene_id \"chr29.g405\"; transcript_id \"chr29.g405.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '22820827', '22820998', '.', '-', '.', 'gene_id \"chr12.g11047\"; transcript_id \"chr12.g11047.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '3670528', '3670829', '.', '+', '.', 'gene_id \"chr29.g698\"; transcript_id \"chr29.g698.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '23776709', '23777061', '.', '-', '.', 'gene_id \"chr12.g11219\"; transcript_id \"chr12.g11219.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '3717863', '3718658', '.', '+', '.', 'gene_id \"chr29.g745\"; transcript_id \"chr29.g745.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '3762834', '3762924', '.', '+', '.', 'gene_id \"chr29.g780\"; transcript_id \"chr29.g780.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '4023391', '4023422', '.', '-', '.', 'gene_id \"chr29.g833\"; transcript_id \"chr29.g833.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '24203724', '24203975', '.', '+', '.', 'gene_id \"chr12.g11402\"; transcript_id \"chr12.g11402.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '4570162', '4570336', '.', '-', '.', 'gene_id \"chr29.g1081\"; transcript_id \"chr29.g1081.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '4805289', '4805352', '.', '+', '.', 'gene_id \"chr29.g1138\"; transcript_id \"chr29.g1138.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '5299049', '5299139', '.', '-', '.', 'gene_id \"chr29.g1210\"; transcript_id \"chr29.g1210.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '5369026', '5369189', '.', '+', '.', 'gene_id \"chr29.g1217\"; transcript_id \"chr29.g1217.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '24826039', '24826207', '.', '-', '.', 'gene_id \"chr12.g11795\"; transcript_id \"chr12.g11795.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6048490', '6048529', '.', '-', '.', 'gene_id \"chr29.g1503\"; transcript_id \"chr29.g1503.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '26648598', '26648851', '.', '-', '.', 'gene_id \"chr12.g12010\"; transcript_id \"chr12.g12010.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6154789', '6154935', '.', '-', '.', 'gene_id \"chr29.g1529\"; transcript_id \"chr29.g1529.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6695126', '6695745', '.', '+', '.', 'gene_id \"chr29.g1604\"; transcript_id \"chr29.g1604.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6767232', '6767299', '.', '-', '.', 'gene_id \"chr29.g1617\"; transcript_id \"chr29.g1617.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6964217', '6964416', '.', '+', '.', 'gene_id \"chr29.g1646\"; transcript_id \"chr29.g1646.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7118961', '7122245', '.', '+', '.', 'gene_id \"chr29.g1665\"; transcript_id \"chr29.g1665.i3\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7118961', '7122599', '.', '+', '.', 'gene_id \"chr29.g1665\"; transcript_id \"chr29.g1665.i4\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7135346', '7135392', '.', '+', '.', 'gene_id \"chr29.g1665\"; transcript_id \"chr29.g1665.i5\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7176377', '7176419', '.', '+', '.', 'gene_id \"chr29.g1675\"; transcript_id \"chr29.g1675.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '27535680', '27535985', '.', '-', '.', 'gene_id \"chr12.g12259\"; transcript_id \"chr12.g12259.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '29221448', '29221967', '.', '-', '.', 'gene_id \"chr12.g12352\"; transcript_id \"chr12.g12352.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7748389', '7748473', '.', '+', '.', 'gene_id \"chr29.g1982\"; transcript_id \"chr29.g1982.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '31403350', '31403564', '.', '-', '.', 'gene_id \"chr12.g12497\"; transcript_id \"chr12.g12497.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7904520', '7905049', '.', '-', '.', 'gene_id \"chr29.g2017\"; transcript_id \"chr29.g2017.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '8222240', '8222381', '.', '+', '.', 'gene_id \"chr29.g2047\"; transcript_id \"chr29.g2047.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '8277153', '8277603', '.', '-', '.', 'gene_id \"chr29.g2064\"; transcript_id \"chr29.g2064.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '8547306', '8547455', '.', '+', '.', 'gene_id \"chr29.g2138\"; transcript_id \"chr29.g2138.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '9315931', '9316136', '.', '-', '.', 'gene_id \"chr29.g2318\"; transcript_id \"chr29.g2318.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '9326587', '9326783', '.', '+', '.', 'gene_id \"chr29.g2324\"; transcript_id \"chr29.g2324.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '9930440', '9930780', '.', '-', '.', 'gene_id \"chr29.g2525\"; transcript_id \"chr29.g2525.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '10404822', '10404856', '.', '+', '.', 'gene_id \"chr29.g2560\"; transcript_id \"chr29.g2560.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '10931514', '10931582', '.', '+', '.', 'gene_id \"chr29.g2684\"; transcript_id \"chr29.g2684.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '10989100', '10989161', '.', '+', '.', 'gene_id \"chr29.g2690\"; transcript_id \"chr29.g2690.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '11101210', '11101321', '.', '+', '.', 'gene_id \"chr29.g2711\"; transcript_id \"chr29.g2711.i3\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '11098794', '11098949', '.', '-', '.', 'gene_id \"chr29.g2763\"; transcript_id \"chr29.g2763.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '33753096', '33753157', '.', '-', '.', 'gene_id \"chr12.g13255\"; transcript_id \"chr12.g13255.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '33870161', '33870487', '.', '+', '.', 'gene_id \"chr12.g13418\"; transcript_id \"chr12.g13418.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '33870610', '33871000', '.', '-', '.', 'gene_id \"chr12.g13419\"; transcript_id \"chr12.g13419.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '34892517', '34892713', '.', '-', '.', 'gene_id \"chr12.g13582\"; transcript_id \"chr12.g13582.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '34892822', '34892973', '.', '-', '.', 'gene_id \"chr12.g13583\"; transcript_id \"chr12.g13583.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '35201357', '35201761', '.', '-', '.', 'gene_id \"chr12.g13745\"; transcript_id \"chr12.g13745.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '35408562', '35408833', '.', '-', '.', 'gene_id \"chr12.g13813\"; transcript_id \"chr12.g13813.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '12607449', '12607604', '.', '-', '.', 'gene_id \"chr29.g3339\"; transcript_id \"chr29.g3339.i4\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '12607247', '12607453', '.', '+', '.', 'gene_id \"chr29.g3343\"; transcript_id \"chr29.g3343.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13206535', '13207061', '.', '-', '.', 'gene_id \"chr29.g3416\"; transcript_id \"chr29.g3416.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13423173', '13423343', '.', '-', '.', 'gene_id \"chr29.g3539\"; transcript_id \"chr29.g3539.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13776854', '13777556', '.', '+', '.', 'gene_id \"chr29.g3562\"; transcript_id \"chr29.g3562.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13794154', '13794397', '.', '-', '.', 'gene_id \"chr29.g3563\"; transcript_id \"chr29.g3563.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '14970761', '14971604', '.', '+', '.', 'gene_id \"chr29.g3838\"; transcript_id \"chr29.g3838.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '14971723', '14971887', '.', '-', '.', 'gene_id \"chr29.g3841\"; transcript_id \"chr29.g3841.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15324306', '15324353', '.', '+', '.', 'gene_id \"chr29.g3890\"; transcript_id \"chr29.g3890.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15334408', '15334457', '.', '+', '.', 'gene_id \"chr29.g3893\"; transcript_id \"chr29.g3893.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '37476728', '37476896', '.', '+', '.', 'gene_id \"chr12.g14451\"; transcript_id \"chr12.g14451.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15499177', '15499355', '.', '-', '.', 'gene_id \"chr29.g3983\"; transcript_id \"chr29.g3983.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15615340', '15615719', '.', '-', '.', 'gene_id \"chr29.g4072\"; transcript_id \"chr29.g4072.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15810563', '15811107', '.', '+', '.', 'gene_id \"chr29.g4114\"; transcript_id \"chr29.g4114.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15866140', '15866180', '.', '-', '.', 'gene_id \"chr29.g4126\"; transcript_id \"chr29.g4126.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15972309', '15972637', '.', '-', '.', 'gene_id \"chr29.g4191\"; transcript_id \"chr29.g4191.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15972822', '15972979', '.', '-', '.', 'gene_id \"chr29.g4193\"; transcript_id \"chr29.g4193.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '16130434', '16130648', '.', '+', '.', 'gene_id \"chr29.g4239\"; transcript_id \"chr29.g4239.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '16130633', '16131743', '.', '-', '.', 'gene_id \"chr29.g4247\"; transcript_id \"chr29.g4247.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '40034122', '40034286', '.', '+', '.', 'gene_id \"chr12.g14770\"; transcript_id \"chr12.g14770.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '40072613', '40072854', '.', '+', '.', 'gene_id \"chr12.g14809\"; transcript_id \"chr12.g14809.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '16313509', '16313550', '.', '-', '.', 'gene_id \"chr29.g4349\"; transcript_id \"chr29.g4349.i3\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '40199771', '40200003', '.', '+', '.', 'gene_id \"chr12.g14895\"; transcript_id \"chr12.g14895.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '16368385', '16368436', '.', '+', '.', 'gene_id \"chr29.g4411\"; transcript_id \"chr29.g4411.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr12', 'PASA', 'exon', '40722262', '40722419', '.', '-', '.', 'gene_id \"chr12.g15175\"; transcript_id \"chr12.g15175.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '41049108', '41049271', '.', '-', '.', 'gene_id \"chr12.g15296\"; transcript_id \"chr12.g15296.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17279703', '17280017', '.', '+', '.', 'gene_id \"chr29.g5001\"; transcript_id \"chr29.g5001.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17279321', '17279440', '.', '+', '.', 'gene_id \"chr29.g5003\"; transcript_id \"chr29.g5003.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17341660', '17341698', '.', '-', '.', 'gene_id \"chr29.g5007\"; transcript_id \"chr29.g5007.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17728795', '17728976', '.', '-', '.', 'gene_id \"chr29.g5064\"; transcript_id \"chr29.g5064.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '42179698', '42179872', '.', '+', '.', 'gene_id \"chr12.g15560\"; transcript_id \"chr12.g15560.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '42507854', '42507885', '.', '+', '.', 'gene_id \"chr12.g15641\"; transcript_id \"chr12.g15641.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '18496341', '18496381', '.', '-', '.', 'gene_id \"chr29.g5230\"; transcript_id \"chr29.g5230.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '18631018', '18631060', '.', '+', '.', 'gene_id \"chr29.g5244\"; transcript_id \"chr29.g5244.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '19282186', '19282230', '.', '+', '.', 'gene_id \"chr29.g5414\"; transcript_id \"chr29.g5414.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '43515949', '43516111', '.', '-', '.', 'gene_id \"chr12.g15925\"; transcript_id \"chr12.g15925.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '43548121', '43548374', '.', '+', '.', 'gene_id \"chr12.g15943\"; transcript_id \"chr12.g15943.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '43548493', '43548771', '.', '-', '.', 'gene_id \"chr12.g15944\"; transcript_id \"chr12.g15944.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '43821993', '43822151', '.', '+', '.', 'gene_id \"chr12.g16153\"; transcript_id \"chr12.g16153.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '44001168', '44001399', '.', '+', '.', 'gene_id \"chr12.g16211\"; transcript_id \"chr12.g16211.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '44001732', '44001924', '.', '+', '.', 'gene_id \"chr12.g16213\"; transcript_id \"chr12.g16213.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '19846698', '19847056', '.', '-', '.', 'gene_id \"chr29.g5816\"; transcript_id \"chr29.g5816.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '19847035', '19847929', '.', '-', '.', 'gene_id \"chr29.g5817\"; transcript_id \"chr29.g5817.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '44272660', '44272853', '.', '+', '.', 'gene_id \"chr12.g16324\"; transcript_id \"chr12.g16324.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '44272860', '44273076', '.', '+', '.', 'gene_id \"chr12.g16325\"; transcript_id \"chr12.g16325.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '20932958', '20932989', '.', '-', '.', 'gene_id \"chr29.g6005\"; transcript_id \"chr29.g6005.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '21023282', '21023321', '.', '+', '.', 'gene_id \"chr29.g6023\"; transcript_id \"chr29.g6023.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '45193296', '45193502', '.', '+', '.', 'gene_id \"chr12.g16552\"; transcript_id \"chr12.g16552.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '45193518', '45193821', '.', '+', '.', 'gene_id \"chr12.g16553\"; transcript_id \"chr12.g16553.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '21522813', '21522924', '.', '+', '.', 'gene_id \"chr29.g6107\"; transcript_id \"chr29.g6107.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '21538669', '21538832', '.', '-', '.', 'gene_id \"chr29.g6115\"; transcript_id \"chr29.g6115.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '21991767', '21992088', '.', '+', '.', 'gene_id \"chr29.g6272\"; transcript_id \"chr29.g6272.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22168597', '22168770', '.', '-', '.', 'gene_id \"chr29.g6359\"; transcript_id \"chr29.g6359.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22403773', '22403969', '.', '-', '.', 'gene_id \"chr29.g6470\"; transcript_id \"chr29.g6470.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '46791944', '46792146', '.', '+', '.', 'gene_id \"chr12.g17020\"; transcript_id \"chr12.g17020.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22571624', '22571723', '.', '+', '.', 'gene_id \"chr29.g6603\"; transcript_id \"chr29.g6603.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22585078', '22585311', '.', '+', '.', 'gene_id \"chr29.g6618\"; transcript_id \"chr29.g6618.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22624980', '22625244', '.', '+', '.', 'gene_id \"chr29.g6637\"; transcript_id \"chr29.g6637.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22677950', '22678121', '.', '+', '.', 'gene_id \"chr29.g6697\"; transcript_id \"chr29.g6697.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22678198', '22678549', '.', '-', '.', 'gene_id \"chr29.g6698\"; transcript_id \"chr29.g6698.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22722596', '22722640', '.', '+', '.', 'gene_id \"chr29.g6722\"; transcript_id \"chr29.g6722.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '47233762', '47233941', '.', '-', '.', 'gene_id \"chr12.g17294\"; transcript_id \"chr12.g17294.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '47558005', '47558260', '.', '+', '.', 'gene_id \"chr12.g17328\"; transcript_id \"chr12.g17328.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '23361585', '23361651', '.', '+', '.', 'gene_id \"chr29.g6852\"; transcript_id \"chr29.g6852.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '47966369', '47967027', '.', '-', '.', 'gene_id \"chr12.g17405\"; transcript_id \"chr12.g17405.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27362798', '27362844', '.', '-', '.', 'gene_id \"chr29.g7493\"; transcript_id \"chr29.g7493.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27477158', '27477518', '.', '-', '.', 'gene_id \"chr29.g7539\"; transcript_id \"chr29.g7539.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '49482993', '49483214', '.', '-', '.', 'gene_id \"chr12.g18131\"; transcript_id \"chr12.g18131.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '49777986', '49778180', '.', '-', '.', 'gene_id \"chr12.g18218\"; transcript_id \"chr12.g18218.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '49778171', '49778332', '.', '+', '.', 'gene_id \"chr12.g18219\"; transcript_id \"chr12.g18219.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '49778444', '49778636', '.', '-', '.', 'gene_id \"chr12.g18220\"; transcript_id \"chr12.g18220.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27876478', '27876717', '.', '-', '.', 'gene_id \"chr29.g7782\"; transcript_id \"chr29.g7782.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27905838', '27905997', '.', '-', '.', 'gene_id \"chr29.g7817\"; transcript_id \"chr29.g7817.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27910602', '27910956', '.', '+', '.', 'gene_id \"chr29.g7820\"; transcript_id \"chr29.g7820.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '51333382', '51333414', '.', '-', '.', 'gene_id \"chr12.g18422\"; transcript_id \"chr12.g18422.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '53456996', '53460270', '.', '+', '.', 'gene_id \"chr12.g18535\"; transcript_id \"chr12.g18535.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28199056', '28199439', '.', '+', '.', 'gene_id \"chr29.g8111\"; transcript_id \"chr29.g8111.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28205371', '28205701', '.', '+', '.', 'gene_id \"chr29.g8115\"; transcript_id \"chr29.g8115.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28206141', '28206359', '.', '+', '.', 'gene_id \"chr29.g8116\"; transcript_id \"chr29.g8116.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28796925', '28796973', '.', '-', '.', 'gene_id \"chr29.g8409\"; transcript_id \"chr29.g8409.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28976538', '28976891', '.', '+', '.', 'gene_id \"chr29.g8432\"; transcript_id \"chr29.g8432.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '28976501', '28976891', '.', '+', '.', 'gene_id \"chr29.g8432\"; transcript_id \"chr29.g8432.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '29056663', '29056905', '.', '-', '.', 'gene_id \"chr29.g8504\"; transcript_id \"chr29.g8504.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '29057026', '29057178', '.', '-', '.', 'gene_id \"chr29.g8505\"; transcript_id \"chr29.g8505.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '29062560', '29062744', '.', '-', '.', 'gene_id \"chr29.g8514\"; transcript_id \"chr29.g8514.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '30725673', '30726155', '.', '-', '.', 'gene_id \"chr29.g8714\"; transcript_id \"chr29.g8714.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '31044645', '31044799', '.', '-', '.', 'gene_id \"chr29.g8755\"; transcript_id \"chr29.g8755.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '57888320', '57888349', '.', '-', '.', 'gene_id \"chr12.g19348\"; transcript_id \"chr12.g19348.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '31424506', '31424903', '.', '+', '.', 'gene_id \"chr29.g8848\"; transcript_id \"chr29.g8848.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '32576568', '32576779', '.', '-', '.', 'gene_id \"chr29.g9191\"; transcript_id \"chr29.g9191.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '32576399', '32576588', '.', '+', '.', 'gene_id \"chr29.g9192\"; transcript_id \"chr29.g9192.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '32740286', '32740334', '.', '-', '.', 'gene_id \"chr29.g9279\"; transcript_id \"chr29.g9279.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '62521184', '62521744', '.', '+', '.', 'gene_id \"chr12.g20013\"; transcript_id \"chr12.g20013.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '62521769', '62522052', '.', '-', '.', 'gene_id \"chr12.g20014\"; transcript_id \"chr12.g20014.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '62760509', '62760723', '.', '+', '.', 'gene_id \"chr12.g20064\"; transcript_id \"chr12.g20064.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '34375249', '34375447', '.', '-', '.', 'gene_id \"chr29.g9584\"; transcript_id \"chr29.g9584.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '63777792', '63778059', '.', '-', '.', 'gene_id \"chr12.g20317\"; transcript_id \"chr12.g20317.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '35747662', '35747883', '.', '+', '.', 'gene_id \"chr29.g9956\"; transcript_id \"chr29.g9956.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '36005514', '36005550', '.', '-', '.', 'gene_id \"chr29.g9966\"; transcript_id \"chr29.g9966.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '36685635', '36685864', '.', '-', '.', 'gene_id \"chr29.g10187\"; transcript_id \"chr29.g10187.i2\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '64625785', '64625829', '.', '-', '.', 'gene_id \"chr12.g20797\"; transcript_id \"chr12.g20797.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '37380058', '37380208', '.', '+', '.', 'gene_id \"chr29.g10288\"; transcript_id \"chr29.g10288.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '37380220', '37380700', '.', '+', '.', 'gene_id \"chr29.g10289\"; transcript_id \"chr29.g10289.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '37420508', '37420690', '.', '-', '.', 'gene_id \"chr29.g10326\"; transcript_id \"chr29.g10326.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '64799843', '64800055', '.', '-', '.', 'gene_id \"chr12.g20901\"; transcript_id \"chr12.g20901.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '38849505', '38849664', '.', '-', '.', 'gene_id \"chr29.g10583\"; transcript_id \"chr29.g10583.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39118610', '39118663', '.', '-', '.', 'gene_id \"chr29.g10682\"; transcript_id \"chr29.g10682.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '65811850', '65812046', '.', '+', '.', 'gene_id \"chr12.g21578\"; transcript_id \"chr12.g21578.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39917190', '39917232', '.', '-', '.', 'gene_id \"chr29.g11104\"; transcript_id \"chr29.g11104.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39976017', '39976750', '.', '+', '.', 'gene_id \"chr29.g11142\"; transcript_id \"chr29.g11142.i2\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39975734', '39976062', '.', '-', '.', 'gene_id \"chr29.g11143\"; transcript_id \"chr29.g11143.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39979319', '39979622', '.', '+', '.', 'gene_id \"chr29.g11149\"; transcript_id \"chr29.g11149.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '66197752', '66197994', '.', '+', '.', 'gene_id \"chr12.g21763\"; transcript_id \"chr12.g21763.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '41737817', '41737857', '.', '+', '.', 'gene_id \"chr29.g11612\"; transcript_id \"chr29.g11612.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '41838676', '41838721', '.', '-', '.', 'gene_id \"chr29.g11658\"; transcript_id \"chr29.g11658.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '41928052', '41928346', '.', '+', '.', 'gene_id \"chr29.g11734\"; transcript_id \"chr29.g11734.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '41928357', '41928509', '.', '-', '.', 'gene_id \"chr29.g11735\"; transcript_id \"chr29.g11735.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '41961768', '41961946', '.', '-', '.', 'gene_id \"chr29.g11774\"; transcript_id \"chr29.g11774.i1\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15962907', '15962950', '100', '-', '.', 'gene_id \"chr29.g11876\"; transcript_id \"blat.proc18128.chain_7032\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '29480726', '29480803', '100', '+', '.', 'gene_id \"chr29.g11899\"; transcript_id \"blat.proc4005.chain_6779\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '36828463', '36828496', '100', '-', '.', 'gene_id \"chr29.g11927\"; transcript_id \"blat.proc18128.chain_24699\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '8717611', '8718009', '100', '+', '.', 'gene_id \"chr29.g11949\"; transcript_id \"blat.proc18128.chain_4033\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7568302', '7568341', '100', '-', '.', 'gene_id \"chr29.g11967\"; transcript_id \"blat.proc28144.chain_22280\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '32803721', '32804230', '100', '-', '.', 'gene_id \"chr29.g11973\"; transcript_id \"blat.proc18128.chain_6867\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '21022524', '21022576', '100', '+', '.', 'gene_id \"chr29.g11990\"; transcript_id \"blat.proc18128.chain_10054\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6204309', '6204348', '100', '-', '.', 'gene_id \"chr29.g11993\"; transcript_id \"blat.proc18128.chain_1111\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6592982', '6593097', '100', '+', '.', 'gene_id \"chr29.g12015\"; transcript_id \"blat.proc28144.chain_1994\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27720500', '27720539', '100', '+', '.', 'gene_id \"chr29.g12125\"; transcript_id \"blat.proc18128.chain_8645\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '11289360', '11289389', '100', '-', '.', 'gene_id \"chr29.g12156\"; transcript_id \"blat.proc4005.chain_21743\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39856040', '39856069', '100', '+', '.', 'gene_id \"chr29.g12163\"; transcript_id \"blat.proc28144.chain_2184\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15330879', '15331132', '100', '+', '.', 'gene_id \"chr29.g12175\"; transcript_id \"blat.proc28144.chain_18981\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '20473487', '20473530', '100', '-', '.', 'gene_id \"chr29.g12527\"; transcript_id \"blat.proc4587.chain_13858\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '27736936', '27736967', '100', '-', '.', 'gene_id \"chr29.g12576\"; transcript_id \"blat.proc18128.chain_17446\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '5705665', '5705835', '100', '-', '.', 'gene_id \"chr29.g12644\"; transcript_id \"blat.proc28144.chain_20067\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '24266997', '24267031', '100', '-', '.', 'gene_id \"chr29.g12676\"; transcript_id \"blat.proc4005.chain_20904\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '29992116', '29992159', '100', '+', '.', 'gene_id \"chr29.g12798\"; transcript_id \"blat.proc28144.chain_4499\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39791202', '39791257', '100', '+', '.', 'gene_id \"chr29.g12823\"; transcript_id \"blat.proc18128.chain_6995\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '12093204', '12093258', '100', '+', '.', 'gene_id \"chr29.g12828\"; transcript_id \"blat.proc4005.chain_22831\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '20035592', '20035628', '100', '+', '.', 'gene_id \"chr29.g12975\"; transcript_id \"blat.proc28144.chain_5564\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '7246200', '7246229', '100', '-', '.', 'gene_id \"chr29.g13046\"; transcript_id \"blat.proc4005.chain_23392\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '36393281', '36393320', '100', '-', '.', 'gene_id \"chr29.g13058\"; transcript_id \"blat.proc18128.chain_9214\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '35801773', '35801805', '100', '-', '.', 'gene_id \"chr29.g13142\"; transcript_id \"blat.proc18128.chain_21889\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '5435569', '5435651', '100', '-', '.', 'gene_id \"chr29.g13157\"; transcript_id \"blat.proc4587.chain_12980\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '12560477', '12560517', '100', '+', '.', 'gene_id \"chr29.g13233\"; transcript_id \"blat.proc18128.chain_21598\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '22585622', '22585793', '100', '+', '.', 'gene_id \"chr29.g13278\"; transcript_id \"blat.proc4587.chain_5345\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '16058285', '16058329', '100', '-', '.', 'gene_id \"chr29.g13317\"; transcript_id \"blat.proc18128.chain_17379\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '24770451', '24770493', '100', '+', '.', 'gene_id \"chr29.g13385\"; transcript_id \"blat.proc18128.chain_572\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '2803805', '2803878', '100', '+', '.', 'gene_id \"chr29.g13405\"; transcript_id \"blat.proc4587.chain_3159\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '24576705', '24576739', '100', '-', '.', 'gene_id \"chr29.g13437\"; transcript_id \"blat.proc18128.chain_1809\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '38184363', '38184399', '100', '+', '.', 'gene_id \"chr29.g13573\"; transcript_id \"blat.proc18128.chain_10792\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '6135479', '6135548', '100', '-', '.', 'gene_id \"chr29.g13574\"; transcript_id \"blat.proc18128.chain_25234\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '23823292', '23823323', '100', '-', '.', 'gene_id \"chr29.g13668\"; transcript_id \"blat.proc28144.chain_6458\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '529644', '529779', '100', '+', '.', 'gene_id \"chr29.g13679\"; transcript_id \"blat.proc28144.chain_8583\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '12338388', '12338462', '100', '+', '.', 'gene_id \"chr29.g13729\"; transcript_id \"blat.proc18128.chain_22219\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '15769205', '15769379', '100', '+', '.', 'gene_id \"chr29.g13920\"; transcript_id \"blat.proc4587.chain_13320\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '4054687', '4054733', '100', '+', '.', 'gene_id \"chr29.g14169\"; transcript_id \"blat.proc18128.chain_2819\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39492345', '39492413', '100', '-', '.', 'gene_id \"chr29.g14384\"; transcript_id \"blat.proc18128.chain_9858\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '14837016', '14837074', '100', '-', '.', 'gene_id \"chr29.g14742\"; transcript_id \"blat.proc4587.chain_4424\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '9115166', '9115215', '100', '+', '.', 'gene_id \"chr29.g14929\"; transcript_id \"blat.proc18128.chain_4993\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13956199', '13956259', '100', '-', '.', 'gene_id \"chr29.g14992\"; transcript_id \"blat.proc18128.chain_19974\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '25442493', '25442529', '100', '+', '.', 'gene_id \"chr29.g15095\"; transcript_id \"blat.proc4587.chain_5379\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17735035', '17735178', '100', '+', '.', 'gene_id \"chr29.g15142\"; transcript_id \"blat.proc4005.chain_3242\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '19555010', '19555046', '100', '-', '.', 'gene_id \"chr29.g15209\"; transcript_id \"blat.proc18128.chain_8206\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '17619647', '17619691', '100', '+', '.', 'gene_id \"chr29.g15245\"; transcript_id \"blat.proc18128.chain_22172\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '33723255', '33723292', '100', '-', '.', 'gene_id \"chr29.g15343\"; transcript_id \"blat.proc28144.chain_20228\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '13476493', '13476522', '100', '+', '.', 'gene_id \"chr29.g15351\"; transcript_id \"blat.proc18128.chain_11687\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '1905347', '1905388', '100', '-', '.', 'gene_id \"chr29.g15359\"; transcript_id \"blat.proc18128.chain_2846\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '18210155', '18210219', '100', '+', '.', 'gene_id \"chr29.g15428\"; transcript_id \"blat.proc18128.chain_15914\";']\n",
      "negative:  ['chr29', 'PASA', 'exon', '39646320', '39646363', '100', '-', '.', 'gene_id \"chr29.g15481\"; transcript_id \"blat.proc18128.chain_21989\";']\n",
      "negative:  ['scaffold_1595', 'PASA', 'exon', '13898', '14066', '.', '+', '.', 'gene_id \"scaffold_1595.g1\"; transcript_id \"scaffold_1595.g1.i2\";']\n",
      "negative:  ['scaffold_1595', 'PASA', 'exon', '13898', '14172', '.', '-', '.', 'gene_id \"scaffold_1595.g2\"; transcript_id \"scaffold_1595.g2.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '67257085', '67257241', '.', '+', '.', 'gene_id \"chr12.g22445\"; transcript_id \"chr12.g22445.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '67504383', '67504424', '.', '+', '.', 'gene_id \"chr12.g22630\"; transcript_id \"chr12.g22630.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '67620493', '67621257', '.', '+', '.', 'gene_id \"chr12.g22696\"; transcript_id \"chr12.g22696.i1\";']\n",
      "negative:  ['scaffold_1341', 'PASA', 'exon', '9533', '11516', '.', '+', '.', 'gene_id \"scaffold_1341.g4\"; transcript_id \"scaffold_1341.g4.i1\";']\n",
      "negative:  ['scaffold_1341', 'PASA', 'exon', '8860', '9537', '.', '-', '.', 'gene_id \"scaffold_1341.g5\"; transcript_id \"scaffold_1341.g5.i1\";']\n",
      "negative:  ['scaffold_1341', 'PASA', 'exon', '7944', '8879', '.', '-', '.', 'gene_id \"scaffold_1341.g6\"; transcript_id \"scaffold_1341.g6.i1\";']\n",
      "negative:  ['scaffold_1292', 'PASA', 'exon', '10453', '10560', '100', '+', '.', 'gene_id \"scaffold_1292.g12\"; transcript_id \"blat.proc18128.chain_22261\";']\n",
      "negative:  ['scaffold_686', 'PASA', 'exon', '489', '666', '100', '+', '.', 'gene_id \"scaffold_686.g3\"; transcript_id \"blat.proc28144.chain_12989\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '68028389', '68028676', '.', '-', '.', 'gene_id \"chr12.g23059\"; transcript_id \"chr12.g23059.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '68103940', '68104300', '.', '+', '.', 'gene_id \"chr12.g23120\"; transcript_id \"chr12.g23120.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '68104322', '68104532', '.', '+', '.', 'gene_id \"chr12.g23121\"; transcript_id \"chr12.g23121.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['scaffold_1493', 'PASA', 'exon', '5459', '5503', '100', '-', '.', 'gene_id \"scaffold_1493.g16\"; transcript_id \"blat.proc18128.chain_17377\";']\n",
      "negative:  ['scaffold_1493', 'PASA', 'exon', '9799', '9977', '100', '-', '.', 'gene_id \"scaffold_1493.g19\"; transcript_id \"blat.proc18128.chain_6218\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '68853297', '68853467', '.', '-', '.', 'gene_id \"chr12.g23731\"; transcript_id \"chr12.g23731.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '70262334', '70262365', '.', '-', '.', 'gene_id \"chr12.g23886\"; transcript_id \"chr12.g23886.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '106954', '106987', '.', '+', '.', 'gene_id \"scaffold_1712.g6\"; transcript_id \"scaffold_1712.g6.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '109084', '110502', '.', '+', '.', 'gene_id \"scaffold_1712.g7\"; transcript_id \"scaffold_1712.g7.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '112780', '114566', '.', '+', '.', 'gene_id \"scaffold_1712.g9\"; transcript_id \"scaffold_1712.g9.i2\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '115098', '115320', '.', '-', '.', 'gene_id \"scaffold_1712.g10\"; transcript_id \"scaffold_1712.g10.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '114853', '115077', '.', '-', '.', 'gene_id \"scaffold_1712.g12\"; transcript_id \"scaffold_1712.g12.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '116142', '116766', '.', '+', '.', 'gene_id \"scaffold_1712.g13\"; transcript_id \"scaffold_1712.g13.i1\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '116744', '116944', '.', '+', '.', 'gene_id \"scaffold_1712.g13\"; transcript_id \"scaffold_1712.g13.i2\";']\n",
      "negative:  ['scaffold_1712', 'PASA', 'exon', '117655', '117892', '.', '+', '.', 'gene_id \"scaffold_1712.g16\"; transcript_id \"scaffold_1712.g16.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '70603496', '70603684', '.', '+', '.', 'gene_id \"chr12.g24024\"; transcript_id \"chr12.g24024.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30112', '30310', '.', '-', '.', 'gene_id \"chr34.g23\"; transcript_id \"chr34.g23.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39008', '39206', '.', '-', '.', 'gene_id \"chr34.g27\"; transcript_id \"chr34.g27.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '48592', '49269', '.', '+', '.', 'gene_id \"chr34.g32\"; transcript_id \"chr34.g32.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '48428', '48594', '.', '+', '.', 'gene_id \"chr34.g36\"; transcript_id \"chr34.g36.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '46797', '46980', '.', '+', '.', 'gene_id \"chr34.g38\"; transcript_id \"chr34.g38.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '72199989', '72200172', '.', '-', '.', 'gene_id \"chr12.g24316\"; transcript_id \"chr12.g24316.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '113671', '114129', '.', '-', '.', 'gene_id \"chr34.g112\"; transcript_id \"chr34.g112.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '72370863', '72371074', '.', '-', '.', 'gene_id \"chr12.g24393\"; transcript_id \"chr12.g24393.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '151138', '151168', '.', '+', '.', 'gene_id \"chr34.g168\"; transcript_id \"chr34.g168.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '244082', '244310', '.', '-', '.', 'gene_id \"chr34.g249\"; transcript_id \"chr34.g249.i1\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '22017579', '22017608', '100', '+', '.', 'gene_id \"chr12.g24658\"; transcript_id \"blat.proc4587.chain_7359\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '11338831', '11338913', '100', '-', '.', 'gene_id \"chr12.g24762\"; transcript_id \"blat.proc28144.chain_19438\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '14282586', '14282655', '100', '+', '.', 'gene_id \"chr12.g24845\"; transcript_id \"blat.proc4587.chain_4534\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '9612910', '9613015', '100', '+', '.', 'gene_id \"chr12.g24931\"; transcript_id \"blat.proc4005.chain_3314\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '26168330', '26168383', '100', '+', '.', 'gene_id \"chr12.g25134\"; transcript_id \"blat.proc4587.chain_1058\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '39431381', '39431414', '100', '-', '.', 'gene_id \"chr12.g25186\"; transcript_id \"blat.proc4587.chain_8502\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '49999656', '49999687', '100', '-', '.', 'gene_id \"chr12.g25422\"; transcript_id \"blat.proc4005.chain_10782\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '6746840', '6746874', '100', '-', '.', 'gene_id \"chr12.g25714\"; transcript_id \"blat.proc18128.chain_6783\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '52740296', '52740336', '100', '+', '.', 'gene_id \"chr12.g26381\"; transcript_id \"blat.proc4005.chain_17353\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '36131580', '36131626', '100', '-', '.', 'gene_id \"chr12.g26463\"; transcript_id \"blat.proc28144.chain_5558\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '72081204', '72081866', '100', '+', '.', 'gene_id \"chr12.g26755\"; transcript_id \"blat.proc4587.chain_12457\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '36735048', '36735091', '100', '+', '.', 'gene_id \"chr12.g27531\"; transcript_id \"blat.proc4005.chain_20704\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '7153764', '7154161', '100', '-', '.', 'gene_id \"chr12.g27936\"; transcript_id \"blat.proc4587.chain_2951\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '37520451', '37520515', '100', '-', '.', 'gene_id \"chr12.g28190\"; transcript_id \"blat.proc4587.chain_2788\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '24110945', '24110984', '100', '+', '.', 'gene_id \"chr12.g28347\"; transcript_id \"blat.proc28144.chain_8424\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '1675172', '1675401', '100', '+', '.', 'gene_id \"chr12.g28770\"; transcript_id \"blat.proc18128.chain_21193\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '39366225', '39366281', '100', '-', '.', 'gene_id \"chr12.g28815\"; transcript_id \"blat.proc18128.chain_9963\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '68319435', '68319468', '100', '+', '.', 'gene_id \"chr12.g29913\"; transcript_id \"blat.proc4005.chain_18731\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '25354892', '25354923', '100', '+', '.', 'gene_id \"chr12.g30030\"; transcript_id \"blat.proc4587.chain_7216\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '35641040', '35641072', '100', '+', '.', 'gene_id \"chr12.g30435\"; transcript_id \"blat.proc18128.chain_5018\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '24217910', '24218018', '100', '+', '.', 'gene_id \"chr12.g30664\"; transcript_id \"blat.proc18128.chain_25504\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '39799567', '39799633', '100', '-', '.', 'gene_id \"chr12.g31093\"; transcript_id \"blat.proc4005.chain_21760\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '13242234', '13242302', '100', '+', '.', 'gene_id \"chr12.g31293\"; transcript_id \"blat.proc18128.chain_22766\";']\n",
      "negative:  ['chr12', 'PASA', 'exon', '15892624', '15892659', '100', '+', '.', 'gene_id \"chr12.g31632\"; transcript_id \"blat.proc4005.chain_18029\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '487456', '487782', '.', '-', '.', 'gene_id \"chr34.g456\"; transcript_id \"chr34.g456.i1\";']\n",
      "negative:  ['scaffold_1772', 'PASA', 'exon', '2570', '2681', '.', '+', '.', 'gene_id \"scaffold_1772.g3\"; transcript_id \"scaffold_1772.g3.i1\";']\n",
      "negative:  ['scaffold_1772', 'PASA', 'exon', '18653', '18685', '.', '-', '.', 'gene_id \"scaffold_1772.g5\"; transcript_id \"scaffold_1772.g5.i1\";']\n",
      "negative:  ['chrM', 'PASA', 'exon', '1', '223', '.', '-', '.', 'gene_id \"chrM.g1\"; transcript_id \"chrM.g1.i2\";']\n",
      "negative:  ['chrM', 'PASA', 'exon', '484', '700', '.', '-', '.', 'gene_id \"chrM.g1\"; transcript_id \"chrM.g1.i3\";']\n",
      "negative:  ['chrM', 'PASA', 'exon', '304', '488', '.', '-', '.', 'gene_id \"chrM.g1\"; transcript_id \"chrM.g1.i4\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1038782', '1039087', '.', '+', '.', 'gene_id \"chr34.g620\"; transcript_id \"chr34.g620.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1174332', '1174770', '.', '-', '.', 'gene_id \"chr34.g629\"; transcript_id \"chr34.g629.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1223473', '1223599', '.', '+', '.', 'gene_id \"chr34.g635\"; transcript_id \"chr34.g635.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1223837', '1224009', '.', '-', '.', 'gene_id \"chr34.g636\"; transcript_id \"chr34.g636.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1652505', '1652680', '.', '+', '.', 'gene_id \"chr34.g659\"; transcript_id \"chr34.g659.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '2140694', '2140754', '.', '+', '.', 'gene_id \"chr34.g696\"; transcript_id \"chr34.g696.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '2976778', '2976988', '.', '+', '.', 'gene_id \"chr34.g822\"; transcript_id \"chr34.g822.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3337436', '3337597', '.', '-', '.', 'gene_id \"chr34.g900\"; transcript_id \"chr34.g900.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3419383', '3419431', '.', '-', '.', 'gene_id \"chr34.g937\"; transcript_id \"chr34.g937.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3449882', '3449921', '.', '+', '.', 'gene_id \"chr34.g943\"; transcript_id \"chr34.g943.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3539465', '3539590', '.', '-', '.', 'gene_id \"chr34.g952\"; transcript_id \"chr34.g952.i1\";']\n",
      "negative:  ['scaffold_1442', 'PASA', 'exon', '23413', '23443', '.', '-', '.', 'gene_id \"scaffold_1442.g2\"; transcript_id \"scaffold_1442.g2.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3731053', '3731089', '.', '+', '.', 'gene_id \"chr34.g1134\"; transcript_id \"chr34.g1134.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3873589', '3873641', '.', '-', '.', 'gene_id \"chr34.g1194\"; transcript_id \"chr34.g1194.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '3873590', '3873641', '.', '-', '.', 'gene_id \"chr34.g1194\"; transcript_id \"chr34.g1194.i4\";']\n",
      "negative:  ['scaffold_1692', 'PASA', 'exon', '80402', '80554', '.', '-', '.', 'gene_id \"scaffold_1692.g11\"; transcript_id \"scaffold_1692.g11.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '4192271', '4192372', '.', '+', '.', 'gene_id \"chr34.g1461\"; transcript_id \"chr34.g1461.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '4227007', '4227449', '.', '+', '.', 'gene_id \"chr34.g1482\"; transcript_id \"chr34.g1482.i1\";']\n",
      "negative:  ['scaffold_1500', 'PASA', 'exon', '22474', '22577', '.', '-', '.', 'gene_id \"scaffold_1500.g15\"; transcript_id \"scaffold_1500.g15.i1\";']\n",
      "negative:  ['scaffold_1473', 'PASA', 'exon', '120', '367', '.', '-', '.', 'gene_id \"scaffold_1473.g1\"; transcript_id \"scaffold_1473.g1.i1\";']\n",
      "negative:  ['scaffold_1473', 'PASA', 'exon', '359', '523', '.', '+', '.', 'gene_id \"scaffold_1473.g2\"; transcript_id \"scaffold_1473.g2.i1\";']\n",
      "negative:  ['scaffold_1473', 'PASA', 'exon', '12834', '13043', '.', '+', '.', 'gene_id \"scaffold_1473.g14\"; transcript_id \"scaffold_1473.g14.i2\";']\n",
      "negative:  ['scaffold_747', 'PASA', 'exon', '2741', '2885', '.', '+', '.', 'gene_id \"scaffold_747.g3\"; transcript_id \"scaffold_747.g3.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5160992', '5161403', '.', '+', '.', 'gene_id \"chr34.g1733\"; transcript_id \"chr34.g1733.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5325299', '5325333', '.', '+', '.', 'gene_id \"chr34.g1748\"; transcript_id \"chr34.g1748.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5576841', '5577124', '.', '-', '.', 'gene_id \"chr34.g1766\"; transcript_id \"chr34.g1766.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '6076965', '6077218', '.', '+', '.', 'gene_id \"chr34.g1867\"; transcript_id \"chr34.g1867.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '6085600', '6085750', '.', '+', '.', 'gene_id \"chr34.g1887\"; transcript_id \"chr34.g1887.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '747409', '748062', '.', '-', '.', 'gene_id \"chr27.g157\"; transcript_id \"chr27.g157.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '748409', '748612', '.', '-', '.', 'gene_id \"chr27.g159\"; transcript_id \"chr27.g159.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '748766', '748947', '.', '-', '.', 'gene_id \"chr27.g160\"; transcript_id \"chr27.g160.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '6609972', '6610002', '.', '-', '.', 'gene_id \"chr34.g2029\"; transcript_id \"chr34.g2029.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7002564', '7002716', '.', '-', '.', 'gene_id \"chr34.g2080\"; transcript_id \"chr34.g2080.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7033283', '7033426', '.', '+', '.', 'gene_id \"chr34.g2125\"; transcript_id \"chr34.g2125.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7068844', '7070008', '.', '-', '.', 'gene_id \"chr34.g2166\"; transcript_id \"chr34.g2166.i5\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7077800', '7077986', '.', '+', '.', 'gene_id \"chr34.g2180\"; transcript_id \"chr34.g2180.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7078473', '7078699', '.', '+', '.', 'gene_id \"chr34.g2182\"; transcript_id \"chr34.g2182.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '1254432', '1254494', '.', '+', '.', 'gene_id \"chr27.g567\"; transcript_id \"chr27.g567.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '1293211', '1293398', '.', '+', '.', 'gene_id \"chr27.g597\"; transcript_id \"chr27.g597.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '7398529', '7398570', '.', '-', '.', 'gene_id \"chr34.g2354\"; transcript_id \"chr34.g2354.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '9035721', '9036232', '.', '-', '.', 'gene_id \"chr34.g2599\"; transcript_id \"chr34.g2599.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '9366995', '9367067', '.', '+', '.', 'gene_id \"chr34.g2614\"; transcript_id \"chr34.g2614.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '1872465', '1872883', '.', '-', '.', 'gene_id \"chr27.g999\"; transcript_id \"chr27.g999.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '10628443', '10628732', '.', '-', '.', 'gene_id \"chr34.g2720\"; transcript_id \"chr34.g2720.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '10693066', '10693096', '.', '+', '.', 'gene_id \"chr34.g2724\"; transcript_id \"chr34.g2724.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '10841195', '10841407', '.', '-', '.', 'gene_id \"chr34.g2745\"; transcript_id \"chr34.g2745.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '10841432', '10841590', '.', '-', '.', 'gene_id \"chr34.g2746\"; transcript_id \"chr34.g2746.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '11050932', '11051087', '.', '-', '.', 'gene_id \"chr34.g2813\"; transcript_id \"chr34.g2813.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2172275', '2172836', '.', '+', '.', 'gene_id \"chr27.g1266\"; transcript_id \"chr27.g1266.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2239295', '2239453', '.', '+', '.', 'gene_id \"chr27.g1309\"; transcript_id \"chr27.g1309.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2242965', '2243335', '.', '+', '.', 'gene_id \"chr27.g1320\"; transcript_id \"chr27.g1320.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '11488278', '11488327', '.', '-', '.', 'gene_id \"chr34.g3032\"; transcript_id \"chr34.g3032.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2519152', '2519320', '.', '-', '.', 'gene_id \"chr27.g1402\"; transcript_id \"chr27.g1402.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2519535', '2519749', '.', '-', '.', 'gene_id \"chr27.g1403\"; transcript_id \"chr27.g1403.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '11573698', '11573737', '.', '+', '.', 'gene_id \"chr34.g3102\"; transcript_id \"chr34.g3102.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '2849266', '2849454', '.', '+', '.', 'gene_id \"chr27.g1597\"; transcript_id \"chr27.g1597.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3052566', '3052607', '.', '-', '.', 'gene_id \"chr27.g1722\"; transcript_id \"chr27.g1722.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '12309970', '12310017', '.', '+', '.', 'gene_id \"chr34.g3384\"; transcript_id \"chr34.g3384.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '12334084', '12334356', '.', '+', '.', 'gene_id \"chr34.g3389\"; transcript_id \"chr34.g3389.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3190291', '3192311', '.', '-', '.', 'gene_id \"chr27.g1811\"; transcript_id \"chr27.g1811.i4\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3190291', '3191964', '.', '-', '.', 'gene_id \"chr27.g1811\"; transcript_id \"chr27.g1811.i5\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3195422', '3195794', '.', '-', '.', 'gene_id \"chr27.g1811\"; transcript_id \"chr27.g1811.i5\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3192513', '3193325', '.', '+', '.', 'gene_id \"chr27.g1812\"; transcript_id \"chr27.g1812.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3192302', '3192517', '.', '-', '.', 'gene_id \"chr27.g1814\"; transcript_id \"chr27.g1814.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3194573', '3194891', '.', '+', '.', 'gene_id \"chr27.g1817\"; transcript_id \"chr27.g1817.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3193400', '3193633', '.', '-', '.', 'gene_id \"chr27.g1819\"; transcript_id \"chr27.g1819.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '12398258', '12398509', '.', '-', '.', 'gene_id \"chr34.g3401\"; transcript_id \"chr34.g3401.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3197391', '3197987', '.', '+', '.', 'gene_id \"chr27.g1820\"; transcript_id \"chr27.g1820.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '12567233', '12567397', '.', '-', '.', 'gene_id \"chr34.g3459\"; transcript_id \"chr34.g3459.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '13016103', '13016135', '.', '-', '.', 'gene_id \"chr34.g3768\"; transcript_id \"chr34.g3768.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '13028970', '13029066', '.', '-', '.', 'gene_id \"chr34.g3771\"; transcript_id \"chr34.g3771.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3831985', '3832263', '.', '-', '.', 'gene_id \"chr27.g2219\"; transcript_id \"chr27.g2219.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3856107', '3856445', '.', '+', '.', 'gene_id \"chr27.g2276\"; transcript_id \"chr27.g2276.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '13444018', '13445061', '.', '-', '.', 'gene_id \"chr34.g3874\"; transcript_id \"chr34.g3874.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '13886161', '13886221', '.', '-', '.', 'gene_id \"chr34.g3933\"; transcript_id \"chr34.g3933.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '14185474', '14185514', '.', '+', '.', 'gene_id \"chr34.g3994\"; transcript_id \"chr34.g3994.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3981651', '3982182', '.', '+', '.', 'gene_id \"chr27.g2429\"; transcript_id \"chr27.g2429.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '14362371', '14362404', '.', '+', '.', 'gene_id \"chr34.g4014\"; transcript_id \"chr34.g4014.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '14707399', '14707610', '.', '+', '.', 'gene_id \"chr34.g4039\"; transcript_id \"chr34.g4039.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '15238345', '15238500', '.', '+', '.', 'gene_id \"chr34.g4147\"; transcript_id \"chr34.g4147.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '15745665', '15745817', '.', '+', '.', 'gene_id \"chr34.g4219\"; transcript_id \"chr34.g4219.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4264148', '4264416', '.', '+', '.', 'gene_id \"chr27.g2656\"; transcript_id \"chr27.g2656.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '15993150', '15993192', '.', '+', '.', 'gene_id \"chr34.g4389\"; transcript_id \"chr34.g4389.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4463066', '4463115', '.', '+', '.', 'gene_id \"chr27.g2811\"; transcript_id \"chr27.g2811.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16001827', '16002000', '.', '+', '.', 'gene_id \"chr34.g4419\"; transcript_id \"chr34.g4419.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16008902', '16009002', '.', '+', '.', 'gene_id \"chr34.g4426\"; transcript_id \"chr34.g4426.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16130994', '16131038', '.', '-', '.', 'gene_id \"chr34.g4509\"; transcript_id \"chr34.g4509.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4722525', '4722666', '.', '-', '.', 'gene_id \"chr27.g3038\"; transcript_id \"chr27.g3038.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16556480', '16556650', '.', '-', '.', 'gene_id \"chr34.g4654\"; transcript_id \"chr34.g4654.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16556633', '16556803', '.', '+', '.', 'gene_id \"chr34.g4655\"; transcript_id \"chr34.g4655.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16556251', '16556488', '.', '-', '.', 'gene_id \"chr34.g4663\"; transcript_id \"chr34.g4663.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16651937', '16652008', '.', '-', '.', 'gene_id \"chr34.g4747\"; transcript_id \"chr34.g4747.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4942685', '4942939', '.', '-', '.', 'gene_id \"chr27.g3198\"; transcript_id \"chr27.g3198.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4948180', '4948339', '.', '-', '.', 'gene_id \"chr27.g3206\"; transcript_id \"chr27.g3206.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4948472', '4948737', '.', '+', '.', 'gene_id \"chr27.g3207\"; transcript_id \"chr27.g3207.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr27', 'PASA', 'exon', '5063505', '5063554', '.', '-', '.', 'gene_id \"chr27.g3321\"; transcript_id \"chr27.g3321.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5193501', '5193768', '.', '-', '.', 'gene_id \"chr27.g3369\"; transcript_id \"chr27.g3369.i4\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5126310', '5130412', '.', '-', '.', 'gene_id \"chr27.g3369\"; transcript_id \"chr27.g3369.i5\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5144085', '5144278', '.', '-', '.', 'gene_id \"chr27.g3392\"; transcript_id \"chr27.g3392.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5159899', '5160082', '.', '+', '.', 'gene_id \"chr27.g3404\"; transcript_id \"chr27.g3404.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5184945', '5185185', '.', '-', '.', 'gene_id \"chr27.g3412\"; transcript_id \"chr27.g3412.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5325456', '5325498', '.', '+', '.', 'gene_id \"chr27.g3473\"; transcript_id \"chr27.g3473.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5359002', '5359403', '.', '-', '.', 'gene_id \"chr27.g3490\"; transcript_id \"chr27.g3490.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '17516379', '17516421', '.', '+', '.', 'gene_id \"chr34.g5215\"; transcript_id \"chr34.g5215.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '17581749', '17582019', '.', '+', '.', 'gene_id \"chr34.g5259\"; transcript_id \"chr34.g5259.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '17618407', '17618616', '.', '+', '.', 'gene_id \"chr34.g5283\"; transcript_id \"chr34.g5283.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5800024', '5800700', '.', '-', '.', 'gene_id \"chr27.g3849\"; transcript_id \"chr27.g3849.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5936902', '5936987', '.', '+', '.', 'gene_id \"chr27.g3927\"; transcript_id \"chr27.g3927.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6159831', '6159959', '.', '+', '.', 'gene_id \"chr27.g3958\"; transcript_id \"chr27.g3958.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18044432', '18044880', '.', '+', '.', 'gene_id \"chr34.g5528\"; transcript_id \"chr34.g5528.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18051875', '18052053', '.', '-', '.', 'gene_id \"chr34.g5538\"; transcript_id \"chr34.g5538.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18055910', '18056061', '.', '+', '.', 'gene_id \"chr34.g5544\"; transcript_id \"chr34.g5544.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18150035', '18150720', '.', '+', '.', 'gene_id \"chr34.g5544\"; transcript_id \"chr34.g5544.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18177295', '18177580', '.', '+', '.', 'gene_id \"chr34.g5545\"; transcript_id \"chr34.g5545.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18099781', '18100070', '.', '-', '.', 'gene_id \"chr34.g5588\"; transcript_id \"chr34.g5588.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18100103', '18100358', '.', '+', '.', 'gene_id \"chr34.g5590\"; transcript_id \"chr34.g5590.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18155506', '18155887', '.', '-', '.', 'gene_id \"chr34.g5622\"; transcript_id \"chr34.g5622.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18155898', '18156658', '.', '+', '.', 'gene_id \"chr34.g5624\"; transcript_id \"chr34.g5624.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18157675', '18157909', '.', '+', '.', 'gene_id \"chr34.g5628\"; transcript_id \"chr34.g5628.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6658057', '6658091', '.', '+', '.', 'gene_id \"chr27.g4138\"; transcript_id \"chr27.g4138.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18235167', '18235388', '.', '+', '.', 'gene_id \"chr34.g5679\"; transcript_id \"chr34.g5679.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18240382', '18240598', '.', '+', '.', 'gene_id \"chr34.g5697\"; transcript_id \"chr34.g5697.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18261461', '18261633', '.', '-', '.', 'gene_id \"chr34.g5712\"; transcript_id \"chr34.g5712.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6697750', '6698092', '.', '-', '.', 'gene_id \"chr27.g4191\"; transcript_id \"chr27.g4191.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18273271', '18273443', '.', '-', '.', 'gene_id \"chr34.g5734\"; transcript_id \"chr34.g5734.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6765867', '6765937', '.', '-', '.', 'gene_id \"chr27.g4247\"; transcript_id \"chr27.g4247.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6797186', '6797221', '.', '-', '.', 'gene_id \"chr27.g4257\"; transcript_id \"chr27.g4257.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18481408', '18481468', '.', '-', '.', 'gene_id \"chr34.g5818\"; transcript_id \"chr34.g5818.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6938653', '6939004', '.', '+', '.', 'gene_id \"chr27.g4349\"; transcript_id \"chr27.g4349.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6942973', '6943491', '.', '-', '.', 'gene_id \"chr27.g4353\"; transcript_id \"chr27.g4353.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18708442', '18708616', '.', '-', '.', 'gene_id \"chr34.g5909\"; transcript_id \"chr34.g5909.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6960056', '6960294', '.', '-', '.', 'gene_id \"chr27.g4374\"; transcript_id \"chr27.g4374.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6960512', '6960719', '.', '-', '.', 'gene_id \"chr27.g4375\"; transcript_id \"chr27.g4375.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18787228', '18787262', '.', '-', '.', 'gene_id \"chr34.g5964\"; transcript_id \"chr34.g5964.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '18825637', '18825895', '.', '-', '.', 'gene_id \"chr34.g6018\"; transcript_id \"chr34.g6018.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7395918', '7395960', '.', '-', '.', 'gene_id \"chr27.g4524\"; transcript_id \"chr27.g4524.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7417014', '7418047', '.', '-', '.', 'gene_id \"chr27.g4551\"; transcript_id \"chr27.g4551.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7507321', '7507514', '.', '+', '.', 'gene_id \"chr27.g4587\"; transcript_id \"chr27.g4587.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7685196', '7685229', '.', '-', '.', 'gene_id \"chr27.g4615\"; transcript_id \"chr27.g4615.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '19449535', '19449721', '.', '-', '.', 'gene_id \"chr34.g6445\"; transcript_id \"chr34.g6445.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8129941', '8130100', '.', '-', '.', 'gene_id \"chr27.g4853\"; transcript_id \"chr27.g4853.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8212042', '8212085', '.', '+', '.', 'gene_id \"chr27.g4889\"; transcript_id \"chr27.g4889.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8337484', '8338129', '.', '-', '.', 'gene_id \"chr27.g4930\"; transcript_id \"chr27.g4930.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8376078', '8376112', '.', '+', '.', 'gene_id \"chr27.g4957\"; transcript_id \"chr27.g4957.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8591981', '8592228', '.', '-', '.', 'gene_id \"chr27.g5126\"; transcript_id \"chr27.g5126.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8732705', '8732771', '.', '-', '.', 'gene_id \"chr27.g5172\"; transcript_id \"chr27.g5172.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '20245861', '20246038', '.', '-', '.', 'gene_id \"chr34.g6821\"; transcript_id \"chr34.g6821.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '20421559', '20421772', '.', '-', '.', 'gene_id \"chr34.g6944\"; transcript_id \"chr34.g6944.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '20953848', '20954061', '.', '+', '.', 'gene_id \"chr34.g7147\"; transcript_id \"chr34.g7147.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '21007318', '21007558', '.', '-', '.', 'gene_id \"chr34.g7189\"; transcript_id \"chr34.g7189.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '21397916', '21398068', '.', '-', '.', 'gene_id \"chr34.g7299\"; transcript_id \"chr34.g7299.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '21693410', '21693458', '.', '+', '.', 'gene_id \"chr34.g7348\"; transcript_id \"chr34.g7348.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '21736756', '21736825', '.', '+', '.', 'gene_id \"chr34.g7353\"; transcript_id \"chr34.g7353.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11100210', '11100251', '.', '+', '.', 'gene_id \"chr27.g5791\"; transcript_id \"chr27.g5791.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11404162', '11404420', '.', '+', '.', 'gene_id \"chr27.g5890\"; transcript_id \"chr27.g5890.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11404579', '11404758', '.', '-', '.', 'gene_id \"chr27.g5891\"; transcript_id \"chr27.g5891.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '22519141', '22519301', '.', '-', '.', 'gene_id \"chr34.g7700\"; transcript_id \"chr34.g7700.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '22536337', '22536533', '.', '+', '.', 'gene_id \"chr34.g7704\"; transcript_id \"chr34.g7704.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '22722998', '22723218', '.', '+', '.', 'gene_id \"chr34.g7768\"; transcript_id \"chr34.g7768.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11731993', '11732605', '.', '+', '.', 'gene_id \"chr27.g6176\"; transcript_id \"chr27.g6176.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23065284', '23065455', '.', '-', '.', 'gene_id \"chr34.g7799\"; transcript_id \"chr34.g7799.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23099465', '23099561', '.', '+', '.', 'gene_id \"chr34.g7803\"; transcript_id \"chr34.g7803.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23122119', '23122202', '.', '+', '.', 'gene_id \"chr34.g7808\"; transcript_id \"chr34.g7808.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '12307289', '12307330', '.', '+', '.', 'gene_id \"chr27.g6259\"; transcript_id \"chr27.g6259.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '12405263', '12405492', '.', '+', '.', 'gene_id \"chr27.g6272\"; transcript_id \"chr27.g6272.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23617546', '23618193', '.', '-', '.', 'gene_id \"chr34.g7908\"; transcript_id \"chr34.g7908.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23618206', '23618363', '.', '-', '.', 'gene_id \"chr34.g7908\"; transcript_id \"chr34.g7908.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23618357', '23618542', '.', '-', '.', 'gene_id \"chr34.g7908\"; transcript_id \"chr34.g7908.i4\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23617672', '23617850', '.', '+', '.', 'gene_id \"chr34.g7910\"; transcript_id \"chr34.g7910.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23793047', '23794448', '.', '-', '.', 'gene_id \"chr34.g7917\"; transcript_id \"chr34.g7917.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23814094', '23814296', '.', '-', '.', 'gene_id \"chr34.g7922\"; transcript_id \"chr34.g7922.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23928840', '23929053', '.', '+', '.', 'gene_id \"chr34.g7927\"; transcript_id \"chr34.g7927.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23945583', '23945870', '.', '-', '.', 'gene_id \"chr34.g7931\"; transcript_id \"chr34.g7931.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23952524', '23953119', '.', '-', '.', 'gene_id \"chr34.g7932\"; transcript_id \"chr34.g7932.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23953780', '23953945', '.', '-', '.', 'gene_id \"chr34.g7933\"; transcript_id \"chr34.g7933.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23955081', '23955248', '.', '-', '.', 'gene_id \"chr34.g7934\"; transcript_id \"chr34.g7934.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23955344', '23955586', '.', '-', '.', 'gene_id \"chr34.g7935\"; transcript_id \"chr34.g7935.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23957532', '23958193', '.', '-', '.', 'gene_id \"chr34.g7939\"; transcript_id \"chr34.g7939.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23978282', '23978386', '.', '-', '.', 'gene_id \"chr34.g7949\"; transcript_id \"chr34.g7949.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24005233', '24005343', '.', '-', '.', 'gene_id \"chr34.g7952\"; transcript_id \"chr34.g7952.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24046465', '24047507', '.', '-', '.', 'gene_id \"chr34.g7957\"; transcript_id \"chr34.g7957.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24047603', '24048192', '.', '+', '.', 'gene_id \"chr34.g7960\"; transcript_id \"chr34.g7960.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24051567', '24051934', '.', '-', '.', 'gene_id \"chr34.g7962\"; transcript_id \"chr34.g7962.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24051358', '24051578', '.', '+', '.', 'gene_id \"chr34.g7963\"; transcript_id \"chr34.g7963.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24058687', '24058930', '.', '-', '.', 'gene_id \"chr34.g7964\"; transcript_id \"chr34.g7964.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24085813', '24085982', '.', '+', '.', 'gene_id \"chr34.g7965\"; transcript_id \"chr34.g7965.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24155485', '24155643', '.', '+', '.', 'gene_id \"chr34.g7971\"; transcript_id \"chr34.g7971.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24338590', '24338752', '.', '-', '.', 'gene_id \"chr34.g7987\"; transcript_id \"chr34.g7987.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24393096', '24393319', '.', '-', '.', 'gene_id \"chr34.g7991\"; transcript_id \"chr34.g7991.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24500945', '24501222', '.', '+', '.', 'gene_id \"chr34.g7992\"; transcript_id \"chr34.g7992.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24412097', '24412323', '.', '+', '.', 'gene_id \"chr34.g7996\"; transcript_id \"chr34.g7996.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24412609', '24412936', '.', '+', '.', 'gene_id \"chr34.g7997\"; transcript_id \"chr34.g7997.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24424408', '24424565', '.', '+', '.', 'gene_id \"chr34.g7998\"; transcript_id \"chr34.g7998.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24424592', '24424901', '.', '-', '.', 'gene_id \"chr34.g7999\"; transcript_id \"chr34.g7999.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24424902', '24425289', '.', '-', '.', 'gene_id \"chr34.g8000\"; transcript_id \"chr34.g8000.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24459142', '24459317', '.', '-', '.', 'gene_id \"chr34.g8006\"; transcript_id \"chr34.g8006.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24471275', '24471780', '.', '+', '.', 'gene_id \"chr34.g8008\"; transcript_id \"chr34.g8008.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24478511', '24478729', '.', '-', '.', 'gene_id \"chr34.g8010\"; transcript_id \"chr34.g8010.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24478897', '24479090', '.', '-', '.', 'gene_id \"chr34.g8011\"; transcript_id \"chr34.g8011.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24481137', '24481293', '.', '+', '.', 'gene_id \"chr34.g8012\"; transcript_id \"chr34.g8012.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24486172', '24486419', '.', '-', '.', 'gene_id \"chr34.g8015\"; transcript_id \"chr34.g8015.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24489591', '24489782', '.', '+', '.', 'gene_id \"chr34.g8016\"; transcript_id \"chr34.g8016.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24491432', '24491658', '.', '-', '.', 'gene_id \"chr34.g8017\"; transcript_id \"chr34.g8017.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24496346', '24496509', '.', '+', '.', 'gene_id \"chr34.g8020\"; transcript_id \"chr34.g8020.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24496574', '24496808', '.', '+', '.', 'gene_id \"chr34.g8021\"; transcript_id \"chr34.g8021.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24498743', '24499244', '.', '+', '.', 'gene_id \"chr34.g8024\"; transcript_id \"chr34.g8024.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '13225972', '13226181', '.', '-', '.', 'gene_id \"chr27.g6406\"; transcript_id \"chr27.g6406.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '13233014', '13233090', '.', '+', '.', 'gene_id \"chr27.g6417\"; transcript_id \"chr27.g6417.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24751001', '24751186', '.', '-', '.', 'gene_id \"chr34.g8049\"; transcript_id \"chr34.g8049.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24740133', '24740520', '.', '-', '.', 'gene_id \"chr34.g8050\"; transcript_id \"chr34.g8050.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24732837', '24732993', '.', '-', '.', 'gene_id \"chr34.g8052\"; transcript_id \"chr34.g8052.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24746133', '24746297', '.', '-', '.', 'gene_id \"chr34.g8056\"; transcript_id \"chr34.g8056.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24754275', '24754463', '.', '+', '.', 'gene_id \"chr34.g8059\"; transcript_id \"chr34.g8059.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24760813', '24761009', '.', '+', '.', 'gene_id \"chr34.g8061\"; transcript_id \"chr34.g8061.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24768742', '24768893', '.', '-', '.', 'gene_id \"chr34.g8062\"; transcript_id \"chr34.g8062.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24896753', '24897013', '.', '-', '.', 'gene_id \"chr34.g8077\"; transcript_id \"chr34.g8077.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24921133', '24921298', '.', '+', '.', 'gene_id \"chr34.g8080\"; transcript_id \"chr34.g8080.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24892824', '24893041', '.', '+', '.', 'gene_id \"chr34.g8081\"; transcript_id \"chr34.g8081.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24934656', '24934864', '.', '-', '.', 'gene_id \"chr34.g8096\"; transcript_id \"chr34.g8096.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24966376', '24966549', '.', '-', '.', 'gene_id \"chr34.g8099\"; transcript_id \"chr34.g8099.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24995914', '24996281', '.', '-', '.', 'gene_id \"chr34.g8100\"; transcript_id \"chr34.g8100.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24999599', '24999915', '.', '-', '.', 'gene_id \"chr34.g8102\"; transcript_id \"chr34.g8102.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25038601', '25038772', '.', '+', '.', 'gene_id \"chr34.g8110\"; transcript_id \"chr34.g8110.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25038601', '25038780', '.', '-', '.', 'gene_id \"chr34.g8111\"; transcript_id \"chr34.g8111.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25039720', '25039983', '.', '-', '.', 'gene_id \"chr34.g8112\"; transcript_id \"chr34.g8112.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25041242', '25041608', '.', '+', '.', 'gene_id \"chr34.g8114\"; transcript_id \"chr34.g8114.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25042566', '25042782', '.', '+', '.', 'gene_id \"chr34.g8116\"; transcript_id \"chr34.g8116.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25042899', '25042982', '.', '+', '.', 'gene_id \"chr34.g8117\"; transcript_id \"chr34.g8117.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25043023', '25043245', '.', '-', '.', 'gene_id \"chr34.g8118\"; transcript_id \"chr34.g8118.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25043537', '25043691', '.', '-', '.', 'gene_id \"chr34.g8120\"; transcript_id \"chr34.g8120.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25047035', '25047224', '.', '+', '.', 'gene_id \"chr34.g8122\"; transcript_id \"chr34.g8122.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25048838', '25049033', '.', '-', '.', 'gene_id \"chr34.g8125\"; transcript_id \"chr34.g8125.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25059731', '25061768', '.', '-', '.', 'gene_id \"chr34.g8130\"; transcript_id \"chr34.g8130.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25067414', '25067634', '.', '-', '.', 'gene_id \"chr34.g8132\"; transcript_id \"chr34.g8132.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25050879', '25051171', '.', '-', '.', 'gene_id \"chr34.g8134\"; transcript_id \"chr34.g8134.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25051152', '25051829', '.', '+', '.', 'gene_id \"chr34.g8135\"; transcript_id \"chr34.g8135.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25056237', '25056478', '.', '+', '.', 'gene_id \"chr34.g8139\"; transcript_id \"chr34.g8139.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25056485', '25056794', '.', '+', '.', 'gene_id \"chr34.g8140\"; transcript_id \"chr34.g8140.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25059239', '25059725', '.', '+', '.', 'gene_id \"chr34.g8144\"; transcript_id \"chr34.g8144.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25061899', '25063328', '.', '+', '.', 'gene_id \"chr34.g8145\"; transcript_id \"chr34.g8145.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25063304', '25063833', '.', '-', '.', 'gene_id \"chr34.g8146\"; transcript_id \"chr34.g8146.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25067041', '25067239', '.', '-', '.', 'gene_id \"chr34.g8148\"; transcript_id \"chr34.g8148.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25068912', '25069089', '.', '+', '.', 'gene_id \"chr34.g8150\"; transcript_id \"chr34.g8150.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25072142', '25072369', '.', '-', '.', 'gene_id \"chr34.g8151\"; transcript_id \"chr34.g8151.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25078789', '25078988', '.', '-', '.', 'gene_id \"chr34.g8157\"; transcript_id \"chr34.g8157.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25079594', '25079748', '.', '-', '.', 'gene_id \"chr34.g8159\"; transcript_id \"chr34.g8159.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25080852', '25081065', '.', '+', '.', 'gene_id \"chr34.g8160\"; transcript_id \"chr34.g8160.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25178138', '25178342', '.', '-', '.', 'gene_id \"chr34.g8166\"; transcript_id \"chr34.g8166.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25311577', '25311741', '.', '-', '.', 'gene_id \"chr34.g8182\"; transcript_id \"chr34.g8182.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '13433971', '13434014', '.', '-', '.', 'gene_id \"chr27.g6575\"; transcript_id \"chr27.g6575.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25517541', '25517774', '.', '-', '.', 'gene_id \"chr34.g8194\"; transcript_id \"chr34.g8194.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25556803', '25557022', '.', '+', '.', 'gene_id \"chr34.g8198\"; transcript_id \"chr34.g8198.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25599789', '25599954', '.', '+', '.', 'gene_id \"chr34.g8210\"; transcript_id \"chr34.g8210.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25600978', '25601186', '.', '-', '.', 'gene_id \"chr34.g8211\"; transcript_id \"chr34.g8211.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25643705', '25643928', '.', '-', '.', 'gene_id \"chr34.g8212\"; transcript_id \"chr34.g8212.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25653707', '25654147', '.', '-', '.', 'gene_id \"chr34.g8214\"; transcript_id \"chr34.g8214.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25655642', '25655814', '.', '-', '.', 'gene_id \"chr34.g8217\"; transcript_id \"chr34.g8217.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25655823', '25656085', '.', '-', '.', 'gene_id \"chr34.g8219\"; transcript_id \"chr34.g8219.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25656797', '25656834', '.', '-', '.', 'gene_id \"chr34.g8221\"; transcript_id \"chr34.g8221.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25716691', '25716897', '.', '+', '.', 'gene_id \"chr34.g8226\"; transcript_id \"chr34.g8226.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25763026', '25763266', '.', '-', '.', 'gene_id \"chr34.g8232\"; transcript_id \"chr34.g8232.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25763547', '25763699', '.', '-', '.', 'gene_id \"chr34.g8234\"; transcript_id \"chr34.g8234.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25879782', '25879936', '.', '-', '.', 'gene_id \"chr34.g8241\"; transcript_id \"chr34.g8241.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25880623', '25880859', '.', '-', '.', 'gene_id \"chr34.g8242\"; transcript_id \"chr34.g8242.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25880968', '25881147', '.', '+', '.', 'gene_id \"chr34.g8243\"; transcript_id \"chr34.g8243.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25881316', '25881529', '.', '-', '.', 'gene_id \"chr34.g8244\"; transcript_id \"chr34.g8244.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25885582', '25885956', '.', '-', '.', 'gene_id \"chr34.g8248\"; transcript_id \"chr34.g8248.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25892829', '25892982', '.', '-', '.', 'gene_id \"chr34.g8252\"; transcript_id \"chr34.g8252.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25894307', '25894594', '.', '-', '.', 'gene_id \"chr34.g8253\"; transcript_id \"chr34.g8253.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25894912', '25895066', '.', '-', '.', 'gene_id \"chr34.g8255\"; transcript_id \"chr34.g8255.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25899194', '25899448', '.', '+', '.', 'gene_id \"chr34.g8260\"; transcript_id \"chr34.g8260.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25900101', '25901093', '.', '+', '.', 'gene_id \"chr34.g8262\"; transcript_id \"chr34.g8262.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25902299', '25902747', '.', '+', '.', 'gene_id \"chr34.g8265\"; transcript_id \"chr34.g8265.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25903025', '25903220', '.', '+', '.', 'gene_id \"chr34.g8267\"; transcript_id \"chr34.g8267.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25903647', '25903896', '.', '+', '.', 'gene_id \"chr34.g8268\"; transcript_id \"chr34.g8268.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25908232', '25908264', '.', '+', '.', 'gene_id \"chr34.g8273\"; transcript_id \"chr34.g8273.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25975946', '25976211', '.', '+', '.', 'gene_id \"chr34.g8274\"; transcript_id \"chr34.g8274.i8\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26020968', '26021678', '.', '-', '.', 'gene_id \"chr34.g8277\"; transcript_id \"chr34.g8277.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26023527', '26023999', '.', '+', '.', 'gene_id \"chr34.g8278\"; transcript_id \"chr34.g8278.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25975803', '25975963', '.', '-', '.', 'gene_id \"chr34.g8279\"; transcript_id \"chr34.g8279.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25909887', '25910069', '.', '+', '.', 'gene_id \"chr34.g8280\"; transcript_id \"chr34.g8280.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25913835', '25914118', '.', '-', '.', 'gene_id \"chr34.g8284\"; transcript_id \"chr34.g8284.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25914608', '25914948', '.', '-', '.', 'gene_id \"chr34.g8286\"; transcript_id \"chr34.g8286.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25914967', '25915131', '.', '+', '.', 'gene_id \"chr34.g8287\"; transcript_id \"chr34.g8287.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25916326', '25916493', '.', '-', '.', 'gene_id \"chr34.g8288\"; transcript_id \"chr34.g8288.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25922405', '25922581', '.', '+', '.', 'gene_id \"chr34.g8289\"; transcript_id \"chr34.g8289.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25923597', '25923977', '.', '+', '.', 'gene_id \"chr34.g8292\"; transcript_id \"chr34.g8292.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25925084', '25925287', '.', '-', '.', 'gene_id \"chr34.g8293\"; transcript_id \"chr34.g8293.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25925283', '25925405', '.', '+', '.', 'gene_id \"chr34.g8294\"; transcript_id \"chr34.g8294.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25926136', '25926431', '.', '+', '.', 'gene_id \"chr34.g8295\"; transcript_id \"chr34.g8295.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25926951', '25927235', '.', '+', '.', 'gene_id \"chr34.g8297\"; transcript_id \"chr34.g8297.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25929145', '25929460', '.', '+', '.', 'gene_id \"chr34.g8300\"; transcript_id \"chr34.g8300.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25929930', '25930156', '.', '+', '.', 'gene_id \"chr34.g8301\"; transcript_id \"chr34.g8301.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25931120', '25931308', '.', '-', '.', 'gene_id \"chr34.g8302\"; transcript_id \"chr34.g8302.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25932080', '25932259', '.', '-', '.', 'gene_id \"chr34.g8304\"; transcript_id \"chr34.g8304.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25932552', '25932824', '.', '-', '.', 'gene_id \"chr34.g8305\"; transcript_id \"chr34.g8305.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25936075', '25936574', '.', '+', '.', 'gene_id \"chr34.g8309\"; transcript_id \"chr34.g8309.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25937143', '25937335', '.', '+', '.', 'gene_id \"chr34.g8310\"; transcript_id \"chr34.g8310.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25940885', '25941205', '.', '+', '.', 'gene_id \"chr34.g8312\"; transcript_id \"chr34.g8312.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25941200', '25941410', '.', '+', '.', 'gene_id \"chr34.g8313\"; transcript_id \"chr34.g8313.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25942293', '25942531', '.', '-', '.', 'gene_id \"chr34.g8314\"; transcript_id \"chr34.g8314.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25943413', '25943590', '.', '-', '.', 'gene_id \"chr34.g8317\"; transcript_id \"chr34.g8317.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25947925', '25948142', '.', '+', '.', 'gene_id \"chr34.g8321\"; transcript_id \"chr34.g8321.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25948967', '25949185', '.', '+', '.', 'gene_id \"chr34.g8323\"; transcript_id \"chr34.g8323.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25959413', '25959565', '.', '+', '.', 'gene_id \"chr34.g8329\"; transcript_id \"chr34.g8329.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25960871', '25961040', '.', '+', '.', 'gene_id \"chr34.g8330\"; transcript_id \"chr34.g8330.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25964211', '25964442', '.', '-', '.', 'gene_id \"chr34.g8331\"; transcript_id \"chr34.g8331.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25972172', '25972334', '.', '-', '.', 'gene_id \"chr34.g8335\"; transcript_id \"chr34.g8335.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25974658', '25974823', '.', '+', '.', 'gene_id \"chr34.g8337\"; transcript_id \"chr34.g8337.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25974842', '25975113', '.', '+', '.', 'gene_id \"chr34.g8338\"; transcript_id \"chr34.g8338.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25978442', '25978601', '.', '+', '.', 'gene_id \"chr34.g8340\"; transcript_id \"chr34.g8340.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25978880', '25979116', '.', '-', '.', 'gene_id \"chr34.g8341\"; transcript_id \"chr34.g8341.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25986038', '25986246', '.', '+', '.', 'gene_id \"chr34.g8345\"; transcript_id \"chr34.g8345.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25985866', '25986046', '.', '-', '.', 'gene_id \"chr34.g8346\"; transcript_id \"chr34.g8346.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25988881', '25989234', '.', '+', '.', 'gene_id \"chr34.g8349\"; transcript_id \"chr34.g8349.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25989338', '25989661', '.', '+', '.', 'gene_id \"chr34.g8350\"; transcript_id \"chr34.g8350.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25989668', '25990048', '.', '+', '.', 'gene_id \"chr34.g8351\"; transcript_id \"chr34.g8351.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25990050', '25990392', '.', '-', '.', 'gene_id \"chr34.g8352\"; transcript_id \"chr34.g8352.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25990457', '25990654', '.', '+', '.', 'gene_id \"chr34.g8353\"; transcript_id \"chr34.g8353.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25993544', '25993799', '.', '+', '.', 'gene_id \"chr34.g8358\"; transcript_id \"chr34.g8358.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25994131', '25994563', '.', '-', '.', 'gene_id \"chr34.g8359\"; transcript_id \"chr34.g8359.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25994725', '25994893', '.', '+', '.', 'gene_id \"chr34.g8360\"; transcript_id \"chr34.g8360.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25995449', '25995701', '.', '-', '.', 'gene_id \"chr34.g8362\"; transcript_id \"chr34.g8362.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25996933', '25997136', '.', '-', '.', 'gene_id \"chr34.g8364\"; transcript_id \"chr34.g8364.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25997475', '25997932', '.', '+', '.', 'gene_id \"chr34.g8365\"; transcript_id \"chr34.g8365.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25998005', '25998177', '.', '-', '.', 'gene_id \"chr34.g8366\"; transcript_id \"chr34.g8366.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '13963818', '13963854', '.', '+', '.', 'gene_id \"chr27.g6747\"; transcript_id \"chr27.g6747.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25998291', '25998453', '.', '+', '.', 'gene_id \"chr34.g8368\"; transcript_id \"chr34.g8368.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26006744', '26006901', '.', '-', '.', 'gene_id \"chr34.g8371\"; transcript_id \"chr34.g8371.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26008207', '26008401', '.', '-', '.', 'gene_id \"chr34.g8373\"; transcript_id \"chr34.g8373.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26008675', '26008847', '.', '+', '.', 'gene_id \"chr34.g8374\"; transcript_id \"chr34.g8374.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26010841', '26011040', '.', '-', '.', 'gene_id \"chr34.g8377\"; transcript_id \"chr34.g8377.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26014955', '26015157', '.', '+', '.', 'gene_id \"chr34.g8380\"; transcript_id \"chr34.g8380.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26020275', '26020488', '.', '+', '.', 'gene_id \"chr34.g8382\"; transcript_id \"chr34.g8382.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26020469', '26020709', '.', '+', '.', 'gene_id \"chr34.g8383\"; transcript_id \"chr34.g8383.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26023039', '26023208', '.', '+', '.', 'gene_id \"chr34.g8385\"; transcript_id \"chr34.g8385.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26067484', '26067541', '.', '+', '.', 'gene_id \"chr34.g8387\"; transcript_id \"chr34.g8387.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26113706', '26113871', '.', '-', '.', 'gene_id \"chr34.g8392\"; transcript_id \"chr34.g8392.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26118637', '26118798', '.', '-', '.', 'gene_id \"chr34.g8397\"; transcript_id \"chr34.g8397.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26122115', '26122295', '.', '-', '.', 'gene_id \"chr34.g8401\"; transcript_id \"chr34.g8401.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26133269', '26133581', '.', '+', '.', 'gene_id \"chr34.g8404\"; transcript_id \"chr34.g8404.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26142401', '26142572', '.', '+', '.', 'gene_id \"chr34.g8407\"; transcript_id \"chr34.g8407.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26130118', '26130849', '.', '-', '.', 'gene_id \"chr34.g8411\"; transcript_id \"chr34.g8411.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26131916', '26132435', '.', '-', '.', 'gene_id \"chr34.g8414\"; transcript_id \"chr34.g8414.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26140506', '26140614', '.', '+', '.', 'gene_id \"chr34.g8420\"; transcript_id \"chr34.g8420.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26142624', '26142780', '.', '-', '.', 'gene_id \"chr34.g8421\"; transcript_id \"chr34.g8421.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26144496', '26144945', '.', '-', '.', 'gene_id \"chr34.g8425\"; transcript_id \"chr34.g8425.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26145140', '26145563', '.', '-', '.', 'gene_id \"chr34.g8426\"; transcript_id \"chr34.g8426.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26149152', '26149931', '.', '-', '.', 'gene_id \"chr34.g8427\"; transcript_id \"chr34.g8427.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26150882', '26151214', '.', '-', '.', 'gene_id \"chr34.g8429\"; transcript_id \"chr34.g8429.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26151315', '26151559', '.', '-', '.', 'gene_id \"chr34.g8430\"; transcript_id \"chr34.g8430.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26154677', '26154901', '.', '+', '.', 'gene_id \"chr34.g8433\"; transcript_id \"chr34.g8433.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26154937', '26155132', '.', '-', '.', 'gene_id \"chr34.g8434\"; transcript_id \"chr34.g8434.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26155204', '26155447', '.', '+', '.', 'gene_id \"chr34.g8435\"; transcript_id \"chr34.g8435.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26155452', '26155930', '.', '-', '.', 'gene_id \"chr34.g8436\"; transcript_id \"chr34.g8436.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26155943', '26156473', '.', '+', '.', 'gene_id \"chr34.g8437\"; transcript_id \"chr34.g8437.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26164885', '26165097', '.', '-', '.', 'gene_id \"chr34.g8444\"; transcript_id \"chr34.g8444.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26164715', '26164904', '.', '+', '.', 'gene_id \"chr34.g8445\"; transcript_id \"chr34.g8445.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26165582', '26165792', '.', '+', '.', 'gene_id \"chr34.g8446\"; transcript_id \"chr34.g8446.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26168380', '26168550', '.', '+', '.', 'gene_id \"chr34.g8451\"; transcript_id \"chr34.g8451.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26168550', '26168727', '.', '-', '.', 'gene_id \"chr34.g8452\"; transcript_id \"chr34.g8452.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26172751', '26172987', '.', '+', '.', 'gene_id \"chr34.g8455\"; transcript_id \"chr34.g8455.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26173196', '26173457', '.', '-', '.', 'gene_id \"chr34.g8456\"; transcript_id \"chr34.g8456.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26173467', '26173655', '.', '+', '.', 'gene_id \"chr34.g8457\"; transcript_id \"chr34.g8457.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26176545', '26176672', '.', '+', '.', 'gene_id \"chr34.g8458\"; transcript_id \"chr34.g8458.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26178712', '26178920', '.', '-', '.', 'gene_id \"chr34.g8461\"; transcript_id \"chr34.g8461.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26179281', '26179451', '.', '+', '.', 'gene_id \"chr34.g8462\"; transcript_id \"chr34.g8462.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26187067', '26187340', '.', '-', '.', 'gene_id \"chr34.g8464\"; transcript_id \"chr34.g8464.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26190237', '26190780', '.', '-', '.', 'gene_id \"chr34.g8468\"; transcript_id \"chr34.g8468.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26190237', '26190755', '.', '-', '.', 'gene_id \"chr34.g8468\"; transcript_id \"chr34.g8468.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26190732', '26190896', '.', '-', '.', 'gene_id \"chr34.g8468\"; transcript_id \"chr34.g8468.i3\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '14214163', '14214235', '.', '+', '.', 'gene_id \"chr27.g6854\"; transcript_id \"chr27.g6854.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26196291', '26196504', '.', '+', '.', 'gene_id \"chr34.g8472\"; transcript_id \"chr34.g8472.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26229740', '26229910', '.', '+', '.', 'gene_id \"chr34.g8483\"; transcript_id \"chr34.g8483.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26230312', '26230554', '.', '+', '.', 'gene_id \"chr34.g8484\"; transcript_id \"chr34.g8484.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26230649', '26230864', '.', '+', '.', 'gene_id \"chr34.g8485\"; transcript_id \"chr34.g8485.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26232028', '26232194', '.', '-', '.', 'gene_id \"chr34.g8488\"; transcript_id \"chr34.g8488.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26232271', '26232840', '.', '+', '.', 'gene_id \"chr34.g8489\"; transcript_id \"chr34.g8489.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26232845', '26233178', '.', '+', '.', 'gene_id \"chr34.g8490\"; transcript_id \"chr34.g8490.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26233427', '26233680', '.', '+', '.', 'gene_id \"chr34.g8491\"; transcript_id \"chr34.g8491.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26233955', '26234106', '.', '+', '.', 'gene_id \"chr34.g8492\"; transcript_id \"chr34.g8492.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26234136', '26234346', '.', '+', '.', 'gene_id \"chr34.g8493\"; transcript_id \"chr34.g8493.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26235280', '26235467', '.', '+', '.', 'gene_id \"chr34.g8494\"; transcript_id \"chr34.g8494.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26236235', '26236411', '.', '+', '.', 'gene_id \"chr34.g8495\"; transcript_id \"chr34.g8495.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26237701', '26238409', '.', '-', '.', 'gene_id \"chr34.g8498\"; transcript_id \"chr34.g8498.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26239687', '26240106', '.', '-', '.', 'gene_id \"chr34.g8499\"; transcript_id \"chr34.g8499.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26242428', '26242634', '.', '-', '.', 'gene_id \"chr34.g8500\"; transcript_id \"chr34.g8500.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26243316', '26243575', '.', '+', '.', 'gene_id \"chr34.g8501\"; transcript_id \"chr34.g8501.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26246291', '26246618', '.', '+', '.', 'gene_id \"chr34.g8502\"; transcript_id \"chr34.g8502.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26249317', '26250084', '.', '+', '.', 'gene_id \"chr34.g8503\"; transcript_id \"chr34.g8503.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26249321', '26250084', '.', '+', '.', 'gene_id \"chr34.g8503\"; transcript_id \"chr34.g8503.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '14285582', '14285785', '.', '+', '.', 'gene_id \"chr27.g6895\"; transcript_id \"chr27.g6895.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26293066', '26293805', '.', '+', '.', 'gene_id \"chr34.g8511\"; transcript_id \"chr34.g8511.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26290585', '26290760', '.', '-', '.', 'gene_id \"chr34.g8513\"; transcript_id \"chr34.g8513.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26294194', '26294839', '.', '+', '.', 'gene_id \"chr34.g8519\"; transcript_id \"chr34.g8519.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26299330', '26299575', '.', '+', '.', 'gene_id \"chr34.g8523\"; transcript_id \"chr34.g8523.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26300035', '26300216', '.', '+', '.', 'gene_id \"chr34.g8525\"; transcript_id \"chr34.g8525.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26300196', '26300410', '.', '-', '.', 'gene_id \"chr34.g8526\"; transcript_id \"chr34.g8526.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26301950', '26302173', '.', '-', '.', 'gene_id \"chr34.g8528\"; transcript_id \"chr34.g8528.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26304439', '26304621', '.', '+', '.', 'gene_id \"chr34.g8531\"; transcript_id \"chr34.g8531.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26307903', '26308072', '.', '+', '.', 'gene_id \"chr34.g8534\"; transcript_id \"chr34.g8534.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26308370', '26308535', '.', '-', '.', 'gene_id \"chr34.g8536\"; transcript_id \"chr34.g8536.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26313341', '26313500', '.', '-', '.', 'gene_id \"chr34.g8542\"; transcript_id \"chr34.g8542.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26313539', '26313709', '.', '-', '.', 'gene_id \"chr34.g8543\"; transcript_id \"chr34.g8543.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26313819', '26314023', '.', '+', '.', 'gene_id \"chr34.g8544\"; transcript_id \"chr34.g8544.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26314569', '26314723', '.', '+', '.', 'gene_id \"chr34.g8546\"; transcript_id \"chr34.g8546.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26315591', '26315852', '.', '+', '.', 'gene_id \"chr34.g8547\"; transcript_id \"chr34.g8547.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26441068', '26441630', '.', '-', '.', 'gene_id \"chr34.g8549\"; transcript_id \"chr34.g8549.i4\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26319052', '26319350', '.', '-', '.', 'gene_id \"chr34.g8550\"; transcript_id \"chr34.g8550.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26368724', '26368885', '.', '+', '.', 'gene_id \"chr34.g8551\"; transcript_id \"chr34.g8551.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26319699', '26319975', '.', '-', '.', 'gene_id \"chr34.g8552\"; transcript_id \"chr34.g8552.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26317458', '26317721', '.', '-', '.', 'gene_id \"chr34.g8555\"; transcript_id \"chr34.g8555.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26320648', '26321212', '.', '-', '.', 'gene_id \"chr34.g8560\"; transcript_id \"chr34.g8560.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26321706', '26322384', '.', '-', '.', 'gene_id \"chr34.g8562\"; transcript_id \"chr34.g8562.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26325846', '26326083', '.', '-', '.', 'gene_id \"chr34.g8565\"; transcript_id \"chr34.g8565.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26326061', '26326266', '.', '+', '.', 'gene_id \"chr34.g8566\"; transcript_id \"chr34.g8566.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26328900', '26329123', '.', '-', '.', 'gene_id \"chr34.g8570\"; transcript_id \"chr34.g8570.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26329717', '26330073', '.', '+', '.', 'gene_id \"chr34.g8572\"; transcript_id \"chr34.g8572.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26330240', '26330452', '.', '-', '.', 'gene_id \"chr34.g8573\"; transcript_id \"chr34.g8573.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26333308', '26333529', '.', '-', '.', 'gene_id \"chr34.g8575\"; transcript_id \"chr34.g8575.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26340195', '26340366', '.', '+', '.', 'gene_id \"chr34.g8576\"; transcript_id \"chr34.g8576.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26342189', '26342343', '.', '+', '.', 'gene_id \"chr34.g8579\"; transcript_id \"chr34.g8579.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26361271', '26361426', '.', '-', '.', 'gene_id \"chr34.g8582\"; transcript_id \"chr34.g8582.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26372184', '26372433', '.', '+', '.', 'gene_id \"chr34.g8587\"; transcript_id \"chr34.g8587.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26373451', '26373606', '.', '-', '.', 'gene_id \"chr34.g8588\"; transcript_id \"chr34.g8588.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26374582', '26374733', '.', '-', '.', 'gene_id \"chr34.g8589\"; transcript_id \"chr34.g8589.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26376505', '26376698', '.', '-', '.', 'gene_id \"chr34.g8591\"; transcript_id \"chr34.g8591.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26392382', '26392538', '.', '-', '.', 'gene_id \"chr34.g8596\"; transcript_id \"chr34.g8596.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26393564', '26394011', '.', '+', '.', 'gene_id \"chr34.g8598\"; transcript_id \"chr34.g8598.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26418765', '26419127', '.', '+', '.', 'gene_id \"chr34.g8608\"; transcript_id \"chr34.g8608.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26423560', '26423763', '.', '+', '.', 'gene_id \"chr34.g8610\"; transcript_id \"chr34.g8610.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26439203', '26439366', '.', '+', '.', 'gene_id \"chr34.g8614\"; transcript_id \"chr34.g8614.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26440557', '26440885', '.', '-', '.', 'gene_id \"chr34.g8616\"; transcript_id \"chr34.g8616.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26442109', '26442345', '.', '-', '.', 'gene_id \"chr34.g8617\"; transcript_id \"chr34.g8617.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26457791', '26458107', '.', '+', '.', 'gene_id \"chr34.g8618\"; transcript_id \"chr34.g8618.i4\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26457791', '26458057', '.', '+', '.', 'gene_id \"chr34.g8618\"; transcript_id \"chr34.g8618.i5\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26457791', '26458442', '.', '+', '.', 'gene_id \"chr34.g8618\"; transcript_id \"chr34.g8618.i6\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26458865', '26459129', '.', '+', '.', 'gene_id \"chr34.g8619\"; transcript_id \"chr34.g8619.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26477071', '26477225', '.', '+', '.', 'gene_id \"chr34.g8620\"; transcript_id \"chr34.g8620.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26459823', '26460035', '.', '+', '.', 'gene_id \"chr34.g8621\"; transcript_id \"chr34.g8621.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26465577', '26465980', '.', '-', '.', 'gene_id \"chr34.g8622\"; transcript_id \"chr34.g8622.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26477573', '26477763', '.', '-', '.', 'gene_id \"chr34.g8624\"; transcript_id \"chr34.g8624.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26447238', '26447412', '.', '-', '.', 'gene_id \"chr34.g8626\"; transcript_id \"chr34.g8626.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26449548', '26449737', '.', '-', '.', 'gene_id \"chr34.g8630\"; transcript_id \"chr34.g8630.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26449835', '26450015', '.', '-', '.', 'gene_id \"chr34.g8631\"; transcript_id \"chr34.g8631.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26451359', '26451514', '.', '-', '.', 'gene_id \"chr34.g8634\"; transcript_id \"chr34.g8634.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26452100', '26452335', '.', '+', '.', 'gene_id \"chr34.g8637\"; transcript_id \"chr34.g8637.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26452618', '26452880', '.', '-', '.', 'gene_id \"chr34.g8638\"; transcript_id \"chr34.g8638.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26453194', '26453347', '.', '+', '.', 'gene_id \"chr34.g8639\"; transcript_id \"chr34.g8639.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26453334', '26453732', '.', '+', '.', 'gene_id \"chr34.g8640\"; transcript_id \"chr34.g8640.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26455796', '26455980', '.', '-', '.', 'gene_id \"chr34.g8641\"; transcript_id \"chr34.g8641.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26471231', '26471483', '.', '-', '.', 'gene_id \"chr34.g8649\"; transcript_id \"chr34.g8649.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26473469', '26473686', '.', '-', '.', 'gene_id \"chr34.g8651\"; transcript_id \"chr34.g8651.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26479201', '26479506', '.', '-', '.', 'gene_id \"chr34.g8657\"; transcript_id \"chr34.g8657.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26480022', '26480411', '.', '+', '.', 'gene_id \"chr34.g8658\"; transcript_id \"chr34.g8658.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26481965', '26485165', '.', '+', '.', 'gene_id \"chr34.g8660\"; transcript_id \"chr34.g8660.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26495656', '26495901', '.', '+', '.', 'gene_id \"chr34.g8661\"; transcript_id \"chr34.g8661.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26499148', '26500326', '.', '+', '.', 'gene_id \"chr34.g8663\"; transcript_id \"chr34.g8663.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26495983', '26497237', '.', '+', '.', 'gene_id \"chr34.g8664\"; transcript_id \"chr34.g8664.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26497568', '26497930', '.', '-', '.', 'gene_id \"chr34.g8665\"; transcript_id \"chr34.g8665.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26504296', '26504572', '.', '+', '.', 'gene_id \"chr34.g8667\"; transcript_id \"chr34.g8667.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '14910405', '14910440', '.', '+', '.', 'gene_id \"chr27.g7042\"; transcript_id \"chr27.g7042.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26504677', '26504853', '.', '-', '.', 'gene_id \"chr34.g8668\"; transcript_id \"chr34.g8668.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26510620', '26510852', '.', '-', '.', 'gene_id \"chr34.g8674\"; transcript_id \"chr34.g8674.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26520300', '26520465', '.', '-', '.', 'gene_id \"chr34.g8678\"; transcript_id \"chr34.g8678.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15038297', '15038342', '.', '-', '.', 'gene_id \"chr27.g7056\"; transcript_id \"chr27.g7056.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26546675', '26546919', '.', '+', '.', 'gene_id \"chr34.g8687\"; transcript_id \"chr34.g8687.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26546091', '26546919', '.', '+', '.', 'gene_id \"chr34.g8687\"; transcript_id \"chr34.g8687.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26571574', '26572274', '.', '+', '.', 'gene_id \"chr34.g8691\"; transcript_id \"chr34.g8691.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26547307', '26547475', '.', '-', '.', 'gene_id \"chr34.g8692\"; transcript_id \"chr34.g8692.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26546902', '26547169', '.', '+', '.', 'gene_id \"chr34.g8698\"; transcript_id \"chr34.g8698.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26542225', '26542842', '.', '-', '.', 'gene_id \"chr34.g8701\"; transcript_id \"chr34.g8701.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26543259', '26543449', '.', '-', '.', 'gene_id \"chr34.g8703\"; transcript_id \"chr34.g8703.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26549346', '26549825', '.', '-', '.', 'gene_id \"chr34.g8707\"; transcript_id \"chr34.g8707.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26553782', '26554093', '.', '-', '.', 'gene_id \"chr34.g8711\"; transcript_id \"chr34.g8711.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26562532', '26562715', '.', '+', '.', 'gene_id \"chr34.g8718\"; transcript_id \"chr34.g8718.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26572542', '26572948', '.', '-', '.', 'gene_id \"chr34.g8721\"; transcript_id \"chr34.g8721.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26573318', '26573521', '.', '+', '.', 'gene_id \"chr34.g8723\"; transcript_id \"chr34.g8723.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26576295', '26576494', '.', '-', '.', 'gene_id \"chr34.g8725\"; transcript_id \"chr34.g8725.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26578592', '26578859', '.', '-', '.', 'gene_id \"chr34.g8726\"; transcript_id \"chr34.g8726.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26589510', '26589732', '.', '+', '.', 'gene_id \"chr34.g8730\"; transcript_id \"chr34.g8730.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26592479', '26592705', '.', '-', '.', 'gene_id \"chr34.g8733\"; transcript_id \"chr34.g8733.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26598248', '26598398', '.', '-', '.', 'gene_id \"chr34.g8734\"; transcript_id \"chr34.g8734.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15240431', '15240471', '.', '-', '.', 'gene_id \"chr27.g7100\"; transcript_id \"chr27.g7100.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26599786', '26600096', '.', '-', '.', 'gene_id \"chr34.g8735\"; transcript_id \"chr34.g8735.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26605483', '26605883', '.', '-', '.', 'gene_id \"chr34.g8740\"; transcript_id \"chr34.g8740.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26609127', '26609451', '.', '+', '.', 'gene_id \"chr34.g8743\"; transcript_id \"chr34.g8743.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26609499', '26609653', '.', '+', '.', 'gene_id \"chr34.g8744\"; transcript_id \"chr34.g8744.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26610018', '26610634', '.', '-', '.', 'gene_id \"chr34.g8745\"; transcript_id \"chr34.g8745.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26610973', '26611150', '.', '-', '.', 'gene_id \"chr34.g8747\"; transcript_id \"chr34.g8747.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26612479', '26612822', '.', '-', '.', 'gene_id \"chr34.g8749\"; transcript_id \"chr34.g8749.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26613200', '26613565', '.', '-', '.', 'gene_id \"chr34.g8751\"; transcript_id \"chr34.g8751.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26614163', '26614450', '.', '-', '.', 'gene_id \"chr34.g8753\"; transcript_id \"chr34.g8753.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26615465', '26615916', '.', '-', '.', 'gene_id \"chr34.g8755\"; transcript_id \"chr34.g8755.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26616070', '26616364', '.', '-', '.', 'gene_id \"chr34.g8756\"; transcript_id \"chr34.g8756.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26617299', '26617480', '.', '-', '.', 'gene_id \"chr34.g8758\"; transcript_id \"chr34.g8758.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26619588', '26619786', '.', '+', '.', 'gene_id \"chr34.g8760\"; transcript_id \"chr34.g8760.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26620668', '26620911', '.', '-', '.', 'gene_id \"chr34.g8761\"; transcript_id \"chr34.g8761.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15385737', '15385983', '.', '-', '.', 'gene_id \"chr27.g7134\"; transcript_id \"chr27.g7134.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26634632', '26635088', '.', '+', '.', 'gene_id \"chr34.g8769\"; transcript_id \"chr34.g8769.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26635425', '26635703', '.', '+', '.', 'gene_id \"chr34.g8770\"; transcript_id \"chr34.g8770.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26635781', '26635934', '.', '+', '.', 'gene_id \"chr34.g8771\"; transcript_id \"chr34.g8771.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26635984', '26636235', '.', '-', '.', 'gene_id \"chr34.g8772\"; transcript_id \"chr34.g8772.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26636910', '26637134', '.', '+', '.', 'gene_id \"chr34.g8773\"; transcript_id \"chr34.g8773.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26637323', '26638043', '.', '-', '.', 'gene_id \"chr34.g8774\"; transcript_id \"chr34.g8774.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26639137', '26639358', '.', '-', '.', 'gene_id \"chr34.g8777\"; transcript_id \"chr34.g8777.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26639593', '26640876', '.', '-', '.', 'gene_id \"chr34.g8778\"; transcript_id \"chr34.g8778.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26639427', '26639597', '.', '-', '.', 'gene_id \"chr34.g8779\"; transcript_id \"chr34.g8779.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26641957', '26642142', '.', '-', '.', 'gene_id \"chr34.g8783\"; transcript_id \"chr34.g8783.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15557006', '15557158', '.', '-', '.', 'gene_id \"chr27.g7150\"; transcript_id \"chr27.g7150.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26643693', '26643930', '.', '+', '.', 'gene_id \"chr34.g8787\"; transcript_id \"chr34.g8787.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26645725', '26645899', '.', '+', '.', 'gene_id \"chr34.g8792\"; transcript_id \"chr34.g8792.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26648660', '26648858', '.', '-', '.', 'gene_id \"chr34.g8795\"; transcript_id \"chr34.g8795.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26648912', '26649116', '.', '+', '.', 'gene_id \"chr34.g8796\"; transcript_id \"chr34.g8796.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26649437', '26649817', '.', '+', '.', 'gene_id \"chr34.g8797\"; transcript_id \"chr34.g8797.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26649934', '26650216', '.', '-', '.', 'gene_id \"chr34.g8799\"; transcript_id \"chr34.g8799.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26650735', '26651096', '.', '+', '.', 'gene_id \"chr34.g8802\"; transcript_id \"chr34.g8802.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26651099', '26651299', '.', '-', '.', 'gene_id \"chr34.g8804\"; transcript_id \"chr34.g8804.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26654315', '26654506', '.', '+', '.', 'gene_id \"chr34.g8806\"; transcript_id \"chr34.g8806.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26656974', '26657282', '.', '-', '.', 'gene_id \"chr34.g8807\"; transcript_id \"chr34.g8807.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26659044', '26659582', '.', '-', '.', 'gene_id \"chr34.g8810\"; transcript_id \"chr34.g8810.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26659603', '26660050', '.', '-', '.', 'gene_id \"chr34.g8811\"; transcript_id \"chr34.g8811.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26660173', '26660502', '.', '+', '.', 'gene_id \"chr34.g8812\"; transcript_id \"chr34.g8812.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26664744', '26664953', '.', '-', '.', 'gene_id \"chr34.g8816\"; transcript_id \"chr34.g8816.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26665181', '26665372', '.', '-', '.', 'gene_id \"chr34.g8817\"; transcript_id \"chr34.g8817.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15841932', '15841973', '.', '-', '.', 'gene_id \"chr27.g7187\"; transcript_id \"chr27.g7187.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26668513', '26668666', '.', '+', '.', 'gene_id \"chr34.g8820\"; transcript_id \"chr34.g8820.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26669852', '26670051', '.', '+', '.', 'gene_id \"chr34.g8823\"; transcript_id \"chr34.g8823.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26670721', '26671036', '.', '+', '.', 'gene_id \"chr34.g8824\"; transcript_id \"chr34.g8824.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26671459', '26671681', '.', '+', '.', 'gene_id \"chr34.g8825\"; transcript_id \"chr34.g8825.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26671934', '26672703', '.', '-', '.', 'gene_id \"chr34.g8826\"; transcript_id \"chr34.g8826.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26671934', '26672683', '.', '-', '.', 'gene_id \"chr34.g8826\"; transcript_id \"chr34.g8826.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26673019', '26673637', '.', '-', '.', 'gene_id \"chr34.g8828\"; transcript_id \"chr34.g8828.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26674927', '26675273', '.', '+', '.', 'gene_id \"chr34.g8832\"; transcript_id \"chr34.g8832.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26679085', '26680027', '.', '-', '.', 'gene_id \"chr34.g8837\"; transcript_id \"chr34.g8837.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16006716', '16006803', '.', '-', '.', 'gene_id \"chr27.g7206\"; transcript_id \"chr27.g7206.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26685849', '26686026', '.', '-', '.', 'gene_id \"chr34.g8841\"; transcript_id \"chr34.g8841.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26688052', '26688287', '.', '+', '.', 'gene_id \"chr34.g8844\"; transcript_id \"chr34.g8844.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26688389', '26688597', '.', '-', '.', 'gene_id \"chr34.g8845\"; transcript_id \"chr34.g8845.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26755455', '26755728', '.', '+', '.', 'gene_id \"chr34.g8853\"; transcript_id \"chr34.g8853.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26755731', '26756033', '.', '+', '.', 'gene_id \"chr34.g8854\"; transcript_id \"chr34.g8854.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27051523', '27055836', '.', '-', '.', 'gene_id \"chr34.g8856\"; transcript_id \"chr34.g8856.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27052540', '27055836', '.', '-', '.', 'gene_id \"chr34.g8856\"; transcript_id \"chr34.g8856.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27045641', '27050158', '.', '-', '.', 'gene_id \"chr34.g8857\"; transcript_id \"chr34.g8857.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27045641', '27050185', '.', '-', '.', 'gene_id \"chr34.g8857\"; transcript_id \"chr34.g8857.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26758696', '26758915', '.', '-', '.', 'gene_id \"chr34.g8863\"; transcript_id \"chr34.g8863.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26764180', '26764347', '.', '+', '.', 'gene_id \"chr34.g8865\"; transcript_id \"chr34.g8865.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26764671', '26764862', '.', '-', '.', 'gene_id \"chr34.g8867\"; transcript_id \"chr34.g8867.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26775377', '26775664', '.', '-', '.', 'gene_id \"chr34.g8869\"; transcript_id \"chr34.g8869.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26776877', '26777020', '.', '-', '.', 'gene_id \"chr34.g8870\"; transcript_id \"chr34.g8870.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26781457', '26781666', '.', '-', '.', 'gene_id \"chr34.g8872\"; transcript_id \"chr34.g8872.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26781256', '26781469', '.', '+', '.', 'gene_id \"chr34.g8873\"; transcript_id \"chr34.g8873.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26782780', '26782980', '.', '-', '.', 'gene_id \"chr34.g8877\"; transcript_id \"chr34.g8877.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26783089', '26783297', '.', '-', '.', 'gene_id \"chr34.g8878\"; transcript_id \"chr34.g8878.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26784256', '26784831', '.', '-', '.', 'gene_id \"chr34.g8881\"; transcript_id \"chr34.g8881.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26786170', '26786357', '.', '-', '.', 'gene_id \"chr34.g8882\"; transcript_id \"chr34.g8882.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26787121', '26787374', '.', '-', '.', 'gene_id \"chr34.g8883\"; transcript_id \"chr34.g8883.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26787572', '26788015', '.', '+', '.', 'gene_id \"chr34.g8884\"; transcript_id \"chr34.g8884.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26791864', '26792077', '.', '+', '.', 'gene_id \"chr34.g8885\"; transcript_id \"chr34.g8885.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26792094', '26792289', '.', '+', '.', 'gene_id \"chr34.g8886\"; transcript_id \"chr34.g8886.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26796177', '26796444', '.', '+', '.', 'gene_id \"chr34.g8887\"; transcript_id \"chr34.g8887.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26796518', '26796748', '.', '+', '.', 'gene_id \"chr34.g8889\"; transcript_id \"chr34.g8889.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26798060', '26798760', '.', '+', '.', 'gene_id \"chr34.g8892\"; transcript_id \"chr34.g8892.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26798126', '26798347', '.', '-', '.', 'gene_id \"chr34.g8893\"; transcript_id \"chr34.g8893.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26800625', '26800783', '.', '+', '.', 'gene_id \"chr34.g8894\"; transcript_id \"chr34.g8894.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26806143', '26806338', '.', '+', '.', 'gene_id \"chr34.g8897\"; transcript_id \"chr34.g8897.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26806511', '26806682', '.', '+', '.', 'gene_id \"chr34.g8898\"; transcript_id \"chr34.g8898.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26810418', '26810665', '.', '-', '.', 'gene_id \"chr34.g8899\"; transcript_id \"chr34.g8899.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26824832', '26825079', '.', '+', '.', 'gene_id \"chr34.g8902\"; transcript_id \"chr34.g8902.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26827096', '26827269', '.', '-', '.', 'gene_id \"chr34.g8903\"; transcript_id \"chr34.g8903.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26873070', '26873314', '.', '+', '.', 'gene_id \"chr34.g8914\"; transcript_id \"chr34.g8914.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26877152', '26877321', '.', '+', '.', 'gene_id \"chr34.g8915\"; transcript_id \"chr34.g8915.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26919276', '26919592', '.', '+', '.', 'gene_id \"chr34.g8924\"; transcript_id \"chr34.g8924.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26947446', '26947739', '.', '+', '.', 'gene_id \"chr34.g8929\"; transcript_id \"chr34.g8929.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26953523', '26953588', '.', '-', '.', 'gene_id \"chr34.g8930\"; transcript_id \"chr34.g8930.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26959037', '26959187', '.', '+', '.', 'gene_id \"chr34.g8931\"; transcript_id \"chr34.g8931.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26989915', '26990098', '.', '-', '.', 'gene_id \"chr34.g8934\"; transcript_id \"chr34.g8934.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26994385', '26994541', '.', '+', '.', 'gene_id \"chr34.g8935\"; transcript_id \"chr34.g8935.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27004706', '27004860', '.', '+', '.', 'gene_id \"chr34.g8939\"; transcript_id \"chr34.g8939.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27030767', '27030943', '.', '+', '.', 'gene_id \"chr34.g8944\"; transcript_id \"chr34.g8944.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27032331', '27032506', '.', '+', '.', 'gene_id \"chr34.g8945\"; transcript_id \"chr34.g8945.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27036170', '27036430', '.', '-', '.', 'gene_id \"chr34.g8947\"; transcript_id \"chr34.g8947.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27036496', '27036883', '.', '-', '.', 'gene_id \"chr34.g8948\"; transcript_id \"chr34.g8948.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27040757', '27040928', '.', '+', '.', 'gene_id \"chr34.g8950\"; transcript_id \"chr34.g8950.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27068428', '27068803', '.', '+', '.', 'gene_id \"chr34.g8954\"; transcript_id \"chr34.g8954.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27070812', '27071217', '.', '-', '.', 'gene_id \"chr34.g8956\"; transcript_id \"chr34.g8956.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27071437', '27071606', '.', '-', '.', 'gene_id \"chr34.g8957\"; transcript_id \"chr34.g8957.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27071972', '27072226', '.', '+', '.', 'gene_id \"chr34.g8958\"; transcript_id \"chr34.g8958.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27094823', '27095082', '.', '-', '.', 'gene_id \"chr34.g8963\"; transcript_id \"chr34.g8963.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27096285', '27096493', '.', '-', '.', 'gene_id \"chr34.g8965\"; transcript_id \"chr34.g8965.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27098427', '27098689', '.', '-', '.', 'gene_id \"chr34.g8966\"; transcript_id \"chr34.g8966.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27098718', '27099097', '.', '+', '.', 'gene_id \"chr34.g8968\"; transcript_id \"chr34.g8968.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27099155', '27099341', '.', '+', '.', 'gene_id \"chr34.g8969\"; transcript_id \"chr34.g8969.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27100514', '27101422', '.', '+', '.', 'gene_id \"chr34.g8972\"; transcript_id \"chr34.g8972.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27103519', '27103671', '.', '+', '.', 'gene_id \"chr34.g8973\"; transcript_id \"chr34.g8973.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27105444', '27105655', '.', '-', '.', 'gene_id \"chr34.g8976\"; transcript_id \"chr34.g8976.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27105651', '27105944', '.', '+', '.', 'gene_id \"chr34.g8977\"; transcript_id \"chr34.g8977.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27106178', '27106437', '.', '-', '.', 'gene_id \"chr34.g8979\"; transcript_id \"chr34.g8979.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27106407', '27106575', '.', '+', '.', 'gene_id \"chr34.g8980\"; transcript_id \"chr34.g8980.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27112267', '27112491', '.', '-', '.', 'gene_id \"chr34.g8985\"; transcript_id \"chr34.g8985.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27112489', '27112723', '.', '+', '.', 'gene_id \"chr34.g8986\"; transcript_id \"chr34.g8986.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27113334', '27113764', '.', '+', '.', 'gene_id \"chr34.g8989\"; transcript_id \"chr34.g8989.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27115011', '27115193', '.', '-', '.', 'gene_id \"chr34.g8993\"; transcript_id \"chr34.g8993.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27118768', '27119093', '.', '+', '.', 'gene_id \"chr34.g8997\"; transcript_id \"chr34.g8997.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27120543', '27120965', '.', '-', '.', 'gene_id \"chr34.g9000\"; transcript_id \"chr34.g9000.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27121841', '27122105', '.', '+', '.', 'gene_id \"chr34.g9001\"; transcript_id \"chr34.g9001.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27123170', '27123340', '.', '-', '.', 'gene_id \"chr34.g9003\"; transcript_id \"chr34.g9003.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27125175', '27125393', '.', '-', '.', 'gene_id \"chr34.g9009\"; transcript_id \"chr34.g9009.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27125691', '27125868', '.', '+', '.', 'gene_id \"chr34.g9011\"; transcript_id \"chr34.g9011.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27128393', '27128845', '.', '-', '.', 'gene_id \"chr34.g9015\"; transcript_id \"chr34.g9015.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27130438', '27130940', '.', '+', '.', 'gene_id \"chr34.g9018\"; transcript_id \"chr34.g9018.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27131804', '27132031', '.', '-', '.', 'gene_id \"chr34.g9020\"; transcript_id \"chr34.g9020.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27133823', '27134098', '.', '-', '.', 'gene_id \"chr34.g9021\"; transcript_id \"chr34.g9021.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27134453', '27134635', '.', '-', '.', 'gene_id \"chr34.g9022\"; transcript_id \"chr34.g9022.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27136677', '27136839', '.', '-', '.', 'gene_id \"chr34.g9027\"; transcript_id \"chr34.g9027.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27137817', '27138064', '.', '-', '.', 'gene_id \"chr34.g9028\"; transcript_id \"chr34.g9028.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27139369', '27139535', '.', '-', '.', 'gene_id \"chr34.g9029\"; transcript_id \"chr34.g9029.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27152217', '27152411', '.', '-', '.', 'gene_id \"chr34.g9034\"; transcript_id \"chr34.g9034.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27164156', '27164363', '.', '+', '.', 'gene_id \"chr34.g9038\"; transcript_id \"chr34.g9038.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27168971', '27169210', '.', '+', '.', 'gene_id \"chr34.g9040\"; transcript_id \"chr34.g9040.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27169787', '27169961', '.', '+', '.', 'gene_id \"chr34.g9045\"; transcript_id \"chr34.g9045.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27170077', '27170303', '.', '-', '.', 'gene_id \"chr34.g9046\"; transcript_id \"chr34.g9046.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27171483', '27171530', '.', '-', '.', 'gene_id \"chr34.g9049\"; transcript_id \"chr34.g9049.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27183472', '27183624', '.', '+', '.', 'gene_id \"chr34.g9053\"; transcript_id \"chr34.g9053.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27183826', '27184036', '.', '+', '.', 'gene_id \"chr34.g9054\"; transcript_id \"chr34.g9054.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27218681', '27218895', '.', '-', '.', 'gene_id \"chr34.g9058\"; transcript_id \"chr34.g9058.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27193979', '27194068', '.', '-', '.', 'gene_id \"chr34.g9060\"; transcript_id \"chr34.g9060.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27216962', '27217165', '.', '-', '.', 'gene_id \"chr34.g9061\"; transcript_id \"chr34.g9061.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27221299', '27221474', '.', '+', '.', 'gene_id \"chr34.g9062\"; transcript_id \"chr34.g9062.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27212805', '27212966', '.', '-', '.', 'gene_id \"chr34.g9071\"; transcript_id \"chr34.g9071.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27213868', '27214103', '.', '-', '.', 'gene_id \"chr34.g9073\"; transcript_id \"chr34.g9073.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27216209', '27216423', '.', '+', '.', 'gene_id \"chr34.g9076\"; transcript_id \"chr34.g9076.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27216452', '27216604', '.', '+', '.', 'gene_id \"chr34.g9078\"; transcript_id \"chr34.g9078.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27217465', '27217772', '.', '+', '.', 'gene_id \"chr34.g9079\"; transcript_id \"chr34.g9079.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27219361', '27220218', '.', '-', '.', 'gene_id \"chr34.g9081\"; transcript_id \"chr34.g9081.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27220298', '27220511', '.', '+', '.', 'gene_id \"chr34.g9082\"; transcript_id \"chr34.g9082.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27220830', '27221296', '.', '+', '.', 'gene_id \"chr34.g9083\"; transcript_id \"chr34.g9083.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27222712', '27222867', '.', '-', '.', 'gene_id \"chr34.g9087\"; transcript_id \"chr34.g9087.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27223878', '27224129', '.', '-', '.', 'gene_id \"chr34.g9089\"; transcript_id \"chr34.g9089.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27231080', '27231255', '.', '-', '.', 'gene_id \"chr34.g9096\"; transcript_id \"chr34.g9096.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27231637', '27232032', '.', '-', '.', 'gene_id \"chr34.g9097\"; transcript_id \"chr34.g9097.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27235295', '27235484', '.', '-', '.', 'gene_id \"chr34.g9099\"; transcript_id \"chr34.g9099.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27235824', '27235977', '.', '-', '.', 'gene_id \"chr34.g9101\"; transcript_id \"chr34.g9101.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27238298', '27238460', '.', '-', '.', 'gene_id \"chr34.g9103\"; transcript_id \"chr34.g9103.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27238760', '27238943', '.', '-', '.', 'gene_id \"chr34.g9105\"; transcript_id \"chr34.g9105.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27239911', '27240085', '.', '-', '.', 'gene_id \"chr34.g9106\"; transcript_id \"chr34.g9106.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27240073', '27240353', '.', '-', '.', 'gene_id \"chr34.g9107\"; transcript_id \"chr34.g9107.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27240555', '27240993', '.', '-', '.', 'gene_id \"chr34.g9108\"; transcript_id \"chr34.g9108.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27251565', '27251755', '.', '+', '.', 'gene_id \"chr34.g9111\"; transcript_id \"chr34.g9111.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27252357', '27252507', '.', '+', '.', 'gene_id \"chr34.g9112\"; transcript_id \"chr34.g9112.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27254916', '27255071', '.', '+', '.', 'gene_id \"chr34.g9113\"; transcript_id \"chr34.g9113.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27255167', '27255325', '.', '+', '.', 'gene_id \"chr34.g9114\"; transcript_id \"chr34.g9114.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27275531', '27275695', '.', '+', '.', 'gene_id \"chr34.g9118\"; transcript_id \"chr34.g9118.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27300960', '27300997', '.', '-', '.', 'gene_id \"chr34.g9120\"; transcript_id \"chr34.g9120.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27300960', '27300997', '.', '+', '.', 'gene_id \"chr34.g9121\"; transcript_id \"chr34.g9121.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27307652', '27309208', '.', '+', '.', 'gene_id \"chr34.g9123\"; transcript_id \"chr34.g9123.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27312304', '27312509', '.', '-', '.', 'gene_id \"chr34.g9126\"; transcript_id \"chr34.g9126.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27319629', '27319801', '.', '+', '.', 'gene_id \"chr34.g9127\"; transcript_id \"chr34.g9127.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27329973', '27330229', '.', '+', '.', 'gene_id \"chr34.g9129\"; transcript_id \"chr34.g9129.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27330514', '27330730', '.', '+', '.', 'gene_id \"chr34.g9130\"; transcript_id \"chr34.g9130.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27331180', '27331471', '.', '-', '.', 'gene_id \"chr34.g9131\"; transcript_id \"chr34.g9131.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27332353', '27332541', '.', '+', '.', 'gene_id \"chr34.g9133\"; transcript_id \"chr34.g9133.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27334317', '27334771', '.', '+', '.', 'gene_id \"chr34.g9134\"; transcript_id \"chr34.g9134.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27336946', '27337185', '.', '+', '.', 'gene_id \"chr34.g9136\"; transcript_id \"chr34.g9136.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27337376', '27337581', '.', '+', '.', 'gene_id \"chr34.g9137\"; transcript_id \"chr34.g9137.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16491537', '16491707', '.', '-', '.', 'gene_id \"chr27.g7508\"; transcript_id \"chr27.g7508.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27440680', '27440873', '.', '+', '.', 'gene_id \"chr34.g9147\"; transcript_id \"chr34.g9147.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16557925', '16557978', '.', '-', '.', 'gene_id \"chr27.g7544\"; transcript_id \"chr27.g7544.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28179580', '28179753', '.', '+', '.', 'gene_id \"chr34.g9188\"; transcript_id \"chr34.g9188.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28179584', '28179753', '.', '+', '.', 'gene_id \"chr34.g9188\"; transcript_id \"chr34.g9188.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28362476', '28362869', '.', '+', '.', 'gene_id \"chr34.g9201\"; transcript_id \"chr34.g9201.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28362424', '28362822', '.', '+', '.', 'gene_id \"chr34.g9201\"; transcript_id \"chr34.g9201.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28426180', '28426235', '.', '+', '.', 'gene_id \"chr34.g9206\"; transcript_id \"chr34.g9206.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28829200', '28829442', '.', '-', '.', 'gene_id \"chr34.g9236\"; transcript_id \"chr34.g9236.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28829609', '28829829', '.', '+', '.', 'gene_id \"chr34.g9237\"; transcript_id \"chr34.g9237.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28829807', '28829963', '.', '-', '.', 'gene_id \"chr34.g9238\"; transcript_id \"chr34.g9238.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28892501', '28892682', '.', '-', '.', 'gene_id \"chr34.g9248\"; transcript_id \"chr34.g9248.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16833826', '16834250', '.', '-', '.', 'gene_id \"chr27.g7613\"; transcript_id \"chr27.g7613.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16837118', '16837916', '.', '+', '.', 'gene_id \"chr27.g7626\"; transcript_id \"chr27.g7626.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16839075', '16839392', '.', '-', '.', 'gene_id \"chr27.g7627\"; transcript_id \"chr27.g7627.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29356965', '29357143', '.', '-', '.', 'gene_id \"chr34.g9275\"; transcript_id \"chr34.g9275.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16846679', '16846833', '.', '+', '.', 'gene_id \"chr27.g7641\"; transcript_id \"chr27.g7641.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16861234', '16861294', '.', '-', '.', 'gene_id \"chr27.g7647\"; transcript_id \"chr27.g7647.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29683505', '29683705', '.', '-', '.', 'gene_id \"chr34.g9293\"; transcript_id \"chr34.g9293.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29856980', '29857208', '.', '+', '.', 'gene_id \"chr34.g9304\"; transcript_id \"chr34.g9304.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29914920', '29915108', '.', '-', '.', 'gene_id \"chr34.g9309\"; transcript_id \"chr34.g9309.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30411060', '30411122', '.', '+', '.', 'gene_id \"chr34.g9322\"; transcript_id \"chr34.g9322.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30411060', '30411179', '.', '+', '.', 'gene_id \"chr34.g9322\"; transcript_id \"chr34.g9322.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30443374', '30443732', '.', '+', '.', 'gene_id \"chr34.g9330\"; transcript_id \"chr34.g9330.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30443730', '30443880', '.', '+', '.', 'gene_id \"chr34.g9331\"; transcript_id \"chr34.g9331.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30443858', '30444025', '.', '-', '.', 'gene_id \"chr34.g9332\"; transcript_id \"chr34.g9332.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30448736', '30449162', '.', '+', '.', 'gene_id \"chr34.g9335\"; transcript_id \"chr34.g9335.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30450682', '30452430', '.', '-', '.', 'gene_id \"chr34.g9337\"; transcript_id \"chr34.g9337.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30452433', '30455909', '.', '+', '.', 'gene_id \"chr34.g9340\"; transcript_id \"chr34.g9340.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30456134', '30456317', '.', '+', '.', 'gene_id \"chr34.g9341\"; transcript_id \"chr34.g9341.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30456375', '30456849', '.', '+', '.', 'gene_id \"chr34.g9342\"; transcript_id \"chr34.g9342.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17092791', '17092837', '.', '-', '.', 'gene_id \"chr27.g7711\"; transcript_id \"chr27.g7711.i6\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30460176', '30460322', '.', '-', '.', 'gene_id \"chr34.g9344\"; transcript_id \"chr34.g9344.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30473912', '30474123', '.', '+', '.', 'gene_id \"chr34.g9345\"; transcript_id \"chr34.g9345.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30478475', '30478708', '.', '+', '.', 'gene_id \"chr34.g9346\"; transcript_id \"chr34.g9346.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30578684', '30578927', '.', '-', '.', 'gene_id \"chr34.g9357\"; transcript_id \"chr34.g9357.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30580653', '30580884', '.', '-', '.', 'gene_id \"chr34.g9358\"; transcript_id \"chr34.g9358.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30581493', '30581799', '.', '-', '.', 'gene_id \"chr34.g9360\"; transcript_id \"chr34.g9360.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30583258', '30583460', '.', '-', '.', 'gene_id \"chr34.g9363\"; transcript_id \"chr34.g9363.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30590513', '30590701', '.', '+', '.', 'gene_id \"chr34.g9365\"; transcript_id \"chr34.g9365.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30590786', '30590984', '.', '-', '.', 'gene_id \"chr34.g9366\"; transcript_id \"chr34.g9366.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30591048', '30591289', '.', '-', '.', 'gene_id \"chr34.g9367\"; transcript_id \"chr34.g9367.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30592126', '30592323', '.', '-', '.', 'gene_id \"chr34.g9368\"; transcript_id \"chr34.g9368.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30594337', '30594511', '.', '+', '.', 'gene_id \"chr34.g9369\"; transcript_id \"chr34.g9369.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30594988', '30595316', '.', '+', '.', 'gene_id \"chr34.g9371\"; transcript_id \"chr34.g9371.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30595297', '30595668', '.', '-', '.', 'gene_id \"chr34.g9372\"; transcript_id \"chr34.g9372.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30596852', '30597133', '.', '+', '.', 'gene_id \"chr34.g9374\"; transcript_id \"chr34.g9374.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30638615', '30638775', '.', '-', '.', 'gene_id \"chr34.g9384\"; transcript_id \"chr34.g9384.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30638776', '30639134', '.', '-', '.', 'gene_id \"chr34.g9385\"; transcript_id \"chr34.g9385.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30657901', '30658089', '.', '-', '.', 'gene_id \"chr34.g9387\"; transcript_id \"chr34.g9387.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30737845', '30738079', '.', '+', '.', 'gene_id \"chr34.g9396\"; transcript_id \"chr34.g9396.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30773705', '30773785', '.', '+', '.', 'gene_id \"chr34.g9399\"; transcript_id \"chr34.g9399.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30910840', '30911042', '.', '+', '.', 'gene_id \"chr34.g9406\"; transcript_id \"chr34.g9406.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30916139', '30916388', '.', '-', '.', 'gene_id \"chr34.g9409\"; transcript_id \"chr34.g9409.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30916724', '30917118', '.', '-', '.', 'gene_id \"chr34.g9411\"; transcript_id \"chr34.g9411.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30931308', '30931347', '.', '+', '.', 'gene_id \"chr34.g9414\"; transcript_id \"chr34.g9414.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30948926', '30949081', '.', '+', '.', 'gene_id \"chr34.g9417\"; transcript_id \"chr34.g9417.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30956602', '30956814', '.', '-', '.', 'gene_id \"chr34.g9419\"; transcript_id \"chr34.g9419.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17187233', '17187428', '.', '-', '.', 'gene_id \"chr27.g7829\"; transcript_id \"chr27.g7829.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '31444803', '31444960', '.', '+', '.', 'gene_id \"chr34.g9456\"; transcript_id \"chr34.g9456.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '31496975', '31497195', '.', '-', '.', 'gene_id \"chr34.g9468\"; transcript_id \"chr34.g9468.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17290593', '17290917', '.', '-', '.', 'gene_id \"chr27.g7871\"; transcript_id \"chr27.g7871.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '31597819', '31597970', '.', '+', '.', 'gene_id \"chr34.g9482\"; transcript_id \"chr34.g9482.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '31639152', '31639324', '.', '+', '.', 'gene_id \"chr34.g9483\"; transcript_id \"chr34.g9483.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32006405', '32006946', '.', '+', '.', 'gene_id \"chr34.g9502\"; transcript_id \"chr34.g9502.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32190980', '32191224', '.', '+', '.', 'gene_id \"chr34.g9521\"; transcript_id \"chr34.g9521.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32193041', '32193226', '.', '+', '.', 'gene_id \"chr34.g9524\"; transcript_id \"chr34.g9524.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32197734', '32197931', '.', '+', '.', 'gene_id \"chr34.g9528\"; transcript_id \"chr34.g9528.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32200564', '32200721', '.', '-', '.', 'gene_id \"chr34.g9532\"; transcript_id \"chr34.g9532.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32203183', '32203360', '.', '-', '.', 'gene_id \"chr34.g9533\"; transcript_id \"chr34.g9533.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32207968', '32208121', '.', '+', '.', 'gene_id \"chr34.g9534\"; transcript_id \"chr34.g9534.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32212002', '32212186', '.', '+', '.', 'gene_id \"chr34.g9538\"; transcript_id \"chr34.g9538.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32293631', '32293994', '.', '-', '.', 'gene_id \"chr34.g9549\"; transcript_id \"chr34.g9549.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32294459', '32294614', '.', '-', '.', 'gene_id \"chr34.g9552\"; transcript_id \"chr34.g9552.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32294608', '32294762', '.', '+', '.', 'gene_id \"chr34.g9553\"; transcript_id \"chr34.g9553.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32374052', '32374221', '.', '-', '.', 'gene_id \"chr34.g9569\"; transcript_id \"chr34.g9569.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32425381', '32425535', '.', '-', '.', 'gene_id \"chr34.g9581\"; transcript_id \"chr34.g9581.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32428620', '32428782', '.', '+', '.', 'gene_id \"chr34.g9584\"; transcript_id \"chr34.g9584.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32477190', '32478033', '.', '-', '.', 'gene_id \"chr34.g9589\"; transcript_id \"chr34.g9589.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32539593', '32539807', '.', '-', '.', 'gene_id \"chr34.g9608\"; transcript_id \"chr34.g9608.i4\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32536864', '32537331', '.', '+', '.', 'gene_id \"chr34.g9611\"; transcript_id \"chr34.g9611.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32537550', '32537785', '.', '-', '.', 'gene_id \"chr34.g9613\"; transcript_id \"chr34.g9613.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32539227', '32539489', '.', '-', '.', 'gene_id \"chr34.g9616\"; transcript_id \"chr34.g9616.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr34', 'PASA', 'exon', '32541722', '32541938', '.', '-', '.', 'gene_id \"chr34.g9620\"; transcript_id \"chr34.g9620.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32542415', '32542595', '.', '+', '.', 'gene_id \"chr34.g9622\"; transcript_id \"chr34.g9622.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32542672', '32542857', '.', '-', '.', 'gene_id \"chr34.g9623\"; transcript_id \"chr34.g9623.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32543931', '32544119', '.', '-', '.', 'gene_id \"chr34.g9625\"; transcript_id \"chr34.g9625.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32544102', '32544343', '.', '+', '.', 'gene_id \"chr34.g9626\"; transcript_id \"chr34.g9626.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32563171', '32563373', '.', '-', '.', 'gene_id \"chr34.g9642\"; transcript_id \"chr34.g9642.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32564904', '32565294', '.', '-', '.', 'gene_id \"chr34.g9646\"; transcript_id \"chr34.g9646.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32579601', '32579744', '.', '-', '.', 'gene_id \"chr34.g9664\"; transcript_id \"chr34.g9664.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32582957', '32583512', '.', '+', '.', 'gene_id \"chr34.g9671\"; transcript_id \"chr34.g9671.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32584371', '32584878', '.', '+', '.', 'gene_id \"chr34.g9672\"; transcript_id \"chr34.g9672.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17749117', '17749157', '.', '-', '.', 'gene_id \"chr27.g8083\"; transcript_id \"chr27.g8083.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32588788', '32588933', '.', '+', '.', 'gene_id \"chr34.g9681\"; transcript_id \"chr34.g9681.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32629042', '32629196', '.', '-', '.', 'gene_id \"chr34.g9692\"; transcript_id \"chr34.g9692.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32635335', '32635488', '.', '-', '.', 'gene_id \"chr34.g9693\"; transcript_id \"chr34.g9693.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32644998', '32645178', '.', '-', '.', 'gene_id \"chr34.g9700\"; transcript_id \"chr34.g9700.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32653833', '32654259', '.', '-', '.', 'gene_id \"chr34.g9704\"; transcript_id \"chr34.g9704.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32656548', '32656770', '.', '-', '.', 'gene_id \"chr34.g9706\"; transcript_id \"chr34.g9706.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32657138', '32657395', '.', '-', '.', 'gene_id \"chr34.g9707\"; transcript_id \"chr34.g9707.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32658037', '32658257', '.', '-', '.', 'gene_id \"chr34.g9708\"; transcript_id \"chr34.g9708.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32658289', '32658468', '.', '+', '.', 'gene_id \"chr34.g9710\"; transcript_id \"chr34.g9710.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32659068', '32659763', '.', '-', '.', 'gene_id \"chr34.g9713\"; transcript_id \"chr34.g9713.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32754726', '32754760', '.', '-', '.', 'gene_id \"chr34.g9723\"; transcript_id \"chr34.g9723.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32818326', '32818735', '.', '+', '.', 'gene_id \"chr34.g9732\"; transcript_id \"chr34.g9732.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32827234', '32827640', '.', '-', '.', 'gene_id \"chr34.g9752\"; transcript_id \"chr34.g9752.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32839535', '32839719', '.', '+', '.', 'gene_id \"chr34.g9766\"; transcript_id \"chr34.g9766.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32839750', '32840203', '.', '-', '.', 'gene_id \"chr34.g9768\"; transcript_id \"chr34.g9768.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32849439', '32849602', '.', '-', '.', 'gene_id \"chr34.g9774\"; transcript_id \"chr34.g9774.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32849850', '32850049', '.', '-', '.', 'gene_id \"chr34.g9776\"; transcript_id \"chr34.g9776.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32852693', '32853217', '.', '-', '.', 'gene_id \"chr34.g9782\"; transcript_id \"chr34.g9782.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32877661', '32877842', '.', '+', '.', 'gene_id \"chr34.g9801\"; transcript_id \"chr34.g9801.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32881624', '32881823', '.', '-', '.', 'gene_id \"chr34.g9806\"; transcript_id \"chr34.g9806.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32895889', '32896218', '.', '+', '.', 'gene_id \"chr34.g9816\"; transcript_id \"chr34.g9816.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32898148', '32898451', '.', '+', '.', 'gene_id \"chr34.g9819\"; transcript_id \"chr34.g9819.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32904757', '32904969', '.', '+', '.', 'gene_id \"chr34.g9824\"; transcript_id \"chr34.g9824.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32907315', '32907507', '.', '+', '.', 'gene_id \"chr34.g9825\"; transcript_id \"chr34.g9825.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32909720', '32910043', '.', '+', '.', 'gene_id \"chr34.g9828\"; transcript_id \"chr34.g9828.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33023916', '33024074', '.', '-', '.', 'gene_id \"chr34.g9856\"; transcript_id \"chr34.g9856.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33034028', '33034273', '.', '-', '.', 'gene_id \"chr34.g9858\"; transcript_id \"chr34.g9858.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33077162', '33077326', '.', '-', '.', 'gene_id \"chr34.g9866\"; transcript_id \"chr34.g9866.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33085629', '33085739', '.', '+', '.', 'gene_id \"chr34.g9870\"; transcript_id \"chr34.g9870.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33087145', '33087240', '.', '+', '.', 'gene_id \"chr34.g9870\"; transcript_id \"chr34.g9870.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33113917', '33114362', '.', '+', '.', 'gene_id \"chr34.g9872\"; transcript_id \"chr34.g9872.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33114360', '33114552', '.', '+', '.', 'gene_id \"chr34.g9873\"; transcript_id \"chr34.g9873.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33153196', '33153416', '.', '-', '.', 'gene_id \"chr34.g9876\"; transcript_id \"chr34.g9876.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33170339', '33170508', '.', '-', '.', 'gene_id \"chr34.g9880\"; transcript_id \"chr34.g9880.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33182126', '33182451', '.', '+', '.', 'gene_id \"chr34.g9881\"; transcript_id \"chr34.g9881.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33187153', '33187312', '.', '-', '.', 'gene_id \"chr34.g9883\"; transcript_id \"chr34.g9883.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33188983', '33189262', '.', '+', '.', 'gene_id \"chr34.g9885\"; transcript_id \"chr34.g9885.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33206010', '33206178', '.', '-', '.', 'gene_id \"chr34.g9887\"; transcript_id \"chr34.g9887.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33240566', '33240626', '.', '-', '.', 'gene_id \"chr34.g9888\"; transcript_id \"chr34.g9888.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33260036', '33260222', '.', '-', '.', 'gene_id \"chr34.g9889\"; transcript_id \"chr34.g9889.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33338885', '33339075', '.', '+', '.', 'gene_id \"chr34.g9898\"; transcript_id \"chr34.g9898.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33377620', '33377652', '.', '+', '.', 'gene_id \"chr34.g9900\"; transcript_id \"chr34.g9900.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33486378', '33486540', '.', '-', '.', 'gene_id \"chr34.g9917\"; transcript_id \"chr34.g9917.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33503858', '33504023', '.', '+', '.', 'gene_id \"chr34.g9923\"; transcript_id \"chr34.g9923.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33505537', '33505655', '.', '-', '.', 'gene_id \"chr34.g9926\"; transcript_id \"chr34.g9926.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33523826', '33525104', '.', '+', '.', 'gene_id \"chr34.g9928\"; transcript_id \"chr34.g9928.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33528231', '33528562', '.', '+', '.', 'gene_id \"chr34.g9934\"; transcript_id \"chr34.g9934.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33531641', '33531849', '.', '+', '.', 'gene_id \"chr34.g9937\"; transcript_id \"chr34.g9937.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33602894', '33603163', '.', '-', '.', 'gene_id \"chr34.g9951\"; transcript_id \"chr34.g9951.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33603142', '33603307', '.', '+', '.', 'gene_id \"chr34.g9952\"; transcript_id \"chr34.g9952.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33614377', '33614755', '.', '-', '.', 'gene_id \"chr34.g9957\"; transcript_id \"chr34.g9957.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33626753', '33627033', '.', '-', '.', 'gene_id \"chr34.g9977\"; transcript_id \"chr34.g9977.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33636572', '33636791', '.', '+', '.', 'gene_id \"chr34.g9984\"; transcript_id \"chr34.g9984.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33644547', '33644871', '.', '-', '.', 'gene_id \"chr34.g9996\"; transcript_id \"chr34.g9996.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33656848', '33657275', '.', '-', '.', 'gene_id \"chr34.g10011\"; transcript_id \"chr34.g10011.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33787670', '33788186', '.', '-', '.', 'gene_id \"chr34.g10041\"; transcript_id \"chr34.g10041.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33792989', '33793204', '.', '+', '.', 'gene_id \"chr34.g10047\"; transcript_id \"chr34.g10047.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33800938', '33801111', '.', '-', '.', 'gene_id \"chr34.g10052\"; transcript_id \"chr34.g10052.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33805763', '33806069', '.', '-', '.', 'gene_id \"chr34.g10054\"; transcript_id \"chr34.g10054.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33806260', '33806450', '.', '+', '.', 'gene_id \"chr34.g10055\"; transcript_id \"chr34.g10055.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33813630', '33813858', '.', '+', '.', 'gene_id \"chr34.g10060\"; transcript_id \"chr34.g10060.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33813873', '33814030', '.', '-', '.', 'gene_id \"chr34.g10062\"; transcript_id \"chr34.g10062.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33816088', '33816272', '.', '-', '.', 'gene_id \"chr34.g10063\"; transcript_id \"chr34.g10063.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33822659', '33822863', '.', '-', '.', 'gene_id \"chr34.g10067\"; transcript_id \"chr34.g10067.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33839536', '33839711', '.', '-', '.', 'gene_id \"chr34.g10074\"; transcript_id \"chr34.g10074.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33926183', '33926355', '.', '-', '.', 'gene_id \"chr34.g10090\"; transcript_id \"chr34.g10090.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33943888', '33944059', '.', '+', '.', 'gene_id \"chr34.g10105\"; transcript_id \"chr34.g10105.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33944551', '33944781', '.', '+', '.', 'gene_id \"chr34.g10106\"; transcript_id \"chr34.g10106.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33945644', '33945859', '.', '+', '.', 'gene_id \"chr34.g10109\"; transcript_id \"chr34.g10109.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33973677', '33973829', '.', '+', '.', 'gene_id \"chr34.g10122\"; transcript_id \"chr34.g10122.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33976014', '33976177', '.', '+', '.', 'gene_id \"chr34.g10124\"; transcript_id \"chr34.g10124.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33976393', '33976547', '.', '+', '.', 'gene_id \"chr34.g10125\"; transcript_id \"chr34.g10125.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33976527', '33976842', '.', '-', '.', 'gene_id \"chr34.g10126\"; transcript_id \"chr34.g10126.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33976829', '33977001', '.', '-', '.', 'gene_id \"chr34.g10127\"; transcript_id \"chr34.g10127.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34027844', '34028625', '.', '-', '.', 'gene_id \"chr34.g10139\"; transcript_id \"chr34.g10139.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34030942', '34031144', '.', '+', '.', 'gene_id \"chr34.g10142\"; transcript_id \"chr34.g10142.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34034785', '34034977', '.', '+', '.', 'gene_id \"chr34.g10144\"; transcript_id \"chr34.g10144.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34039709', '34039901', '.', '-', '.', 'gene_id \"chr34.g10146\"; transcript_id \"chr34.g10146.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34039216', '34039733', '.', '+', '.', 'gene_id \"chr34.g10147\"; transcript_id \"chr34.g10147.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34047077', '34047335', '.', '+', '.', 'gene_id \"chr34.g10148\"; transcript_id \"chr34.g10148.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34096318', '34096382', '.', '+', '.', 'gene_id \"chr34.g10155\"; transcript_id \"chr34.g10155.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34125226', '34125504', '.', '+', '.', 'gene_id \"chr34.g10157\"; transcript_id \"chr34.g10157.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34136919', '34137090', '.', '+', '.', 'gene_id \"chr34.g10165\"; transcript_id \"chr34.g10165.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34144397', '34144633', '.', '+', '.', 'gene_id \"chr34.g10168\"; transcript_id \"chr34.g10168.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34185435', '34185600', '.', '+', '.', 'gene_id \"chr34.g10188\"; transcript_id \"chr34.g10188.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34193906', '34194348', '.', '-', '.', 'gene_id \"chr34.g10195\"; transcript_id \"chr34.g10195.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34221817', '34221993', '.', '+', '.', 'gene_id \"chr34.g10200\"; transcript_id \"chr34.g10200.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34241160', '34241355', '.', '+', '.', 'gene_id \"chr34.g10205\"; transcript_id \"chr34.g10205.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34242890', '34243204', '.', '-', '.', 'gene_id \"chr34.g10207\"; transcript_id \"chr34.g10207.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34252612', '34252791', '.', '-', '.', 'gene_id \"chr34.g10215\"; transcript_id \"chr34.g10215.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34252777', '34252978', '.', '+', '.', 'gene_id \"chr34.g10216\"; transcript_id \"chr34.g10216.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34254850', '34255003', '.', '+', '.', 'gene_id \"chr34.g10217\"; transcript_id \"chr34.g10217.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34256272', '34256480', '.', '-', '.', 'gene_id \"chr34.g10221\"; transcript_id \"chr34.g10221.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34265501', '34265732', '.', '-', '.', 'gene_id \"chr34.g10223\"; transcript_id \"chr34.g10223.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34265721', '34266022', '.', '-', '.', 'gene_id \"chr34.g10224\"; transcript_id \"chr34.g10224.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34274981', '34275161', '.', '-', '.', 'gene_id \"chr34.g10234\"; transcript_id \"chr34.g10234.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34277390', '34277651', '.', '+', '.', 'gene_id \"chr34.g10238\"; transcript_id \"chr34.g10238.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34278791', '34278978', '.', '-', '.', 'gene_id \"chr34.g10240\"; transcript_id \"chr34.g10240.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34280787', '34280976', '.', '+', '.', 'gene_id \"chr34.g10241\"; transcript_id \"chr34.g10241.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '19784296', '19784335', '.', '+', '.', 'gene_id \"chr27.g8646\"; transcript_id \"chr27.g8646.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34293171', '34293386', '.', '+', '.', 'gene_id \"chr34.g10248\"; transcript_id \"chr34.g10248.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34298881', '34299042', '.', '+', '.', 'gene_id \"chr34.g10253\"; transcript_id \"chr34.g10253.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34302861', '34303217', '.', '+', '.', 'gene_id \"chr34.g10255\"; transcript_id \"chr34.g10255.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34319094', '34319271', '.', '+', '.', 'gene_id \"chr34.g10262\"; transcript_id \"chr34.g10262.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34355879', '34356076', '.', '+', '.', 'gene_id \"chr34.g10293\"; transcript_id \"chr34.g10293.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34445250', '34445493', '.', '+', '.', 'gene_id \"chr34.g10348\"; transcript_id \"chr34.g10348.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20089709', '20089757', '.', '+', '.', 'gene_id \"chr27.g8744\"; transcript_id \"chr27.g8744.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20062115', '20062333', '.', '+', '.', 'gene_id \"chr27.g8753\"; transcript_id \"chr27.g8753.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34464965', '34465132', '.', '-', '.', 'gene_id \"chr34.g10353\"; transcript_id \"chr34.g10353.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34473335', '34473561', '.', '-', '.', 'gene_id \"chr34.g10358\"; transcript_id \"chr34.g10358.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '19992563', '19992681', '.', '+', '.', 'gene_id \"chr27.g8784\"; transcript_id \"chr27.g8784.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '19992658', '19993404', '.', '+', '.', 'gene_id \"chr27.g8784\"; transcript_id \"chr27.g8784.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34579077', '34579285', '.', '-', '.', 'gene_id \"chr34.g10388\"; transcript_id \"chr34.g10388.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34607246', '34607677', '.', '+', '.', 'gene_id \"chr34.g10389\"; transcript_id \"chr34.g10389.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34598230', '34598418', '.', '+', '.', 'gene_id \"chr34.g10394\"; transcript_id \"chr34.g10394.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34579413', '34580172', '.', '-', '.', 'gene_id \"chr34.g10398\"; transcript_id \"chr34.g10398.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34605015', '34605260', '.', '+', '.', 'gene_id \"chr34.g10424\"; transcript_id \"chr34.g10424.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34606795', '34607220', '.', '-', '.', 'gene_id \"chr34.g10427\"; transcript_id \"chr34.g10427.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34611334', '34611609', '.', '+', '.', 'gene_id \"chr34.g10430\"; transcript_id \"chr34.g10430.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34659596', '34659965', '.', '+', '.', 'gene_id \"chr34.g10453\"; transcript_id \"chr34.g10453.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34695670', '34695845', '.', '+', '.', 'gene_id \"chr34.g10461\"; transcript_id \"chr34.g10461.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34702151', '34702418', '.', '-', '.', 'gene_id \"chr34.g10466\"; transcript_id \"chr34.g10466.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34702933', '34703237', '.', '-', '.', 'gene_id \"chr34.g10467\"; transcript_id \"chr34.g10467.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34704754', '34705212', '.', '-', '.', 'gene_id \"chr34.g10469\"; transcript_id \"chr34.g10469.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34706430', '34706744', '.', '+', '.', 'gene_id \"chr34.g10473\"; transcript_id \"chr34.g10473.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34706794', '34707007', '.', '-', '.', 'gene_id \"chr34.g10474\"; transcript_id \"chr34.g10474.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20123995', '20124689', '.', '+', '.', 'gene_id \"chr27.g8886\"; transcript_id \"chr27.g8886.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34720285', '34720327', '.', '-', '.', 'gene_id \"chr34.g10486\"; transcript_id \"chr34.g10486.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20127161', '20127315', '.', '-', '.', 'gene_id \"chr27.g8888\"; transcript_id \"chr27.g8888.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34723341', '34723533', '.', '+', '.', 'gene_id \"chr34.g10487\"; transcript_id \"chr34.g10487.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34728729', '34729047', '.', '-', '.', 'gene_id \"chr34.g10490\"; transcript_id \"chr34.g10490.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34729070', '34729364', '.', '+', '.', 'gene_id \"chr34.g10491\"; transcript_id \"chr34.g10491.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34729736', '34730067', '.', '+', '.', 'gene_id \"chr34.g10493\"; transcript_id \"chr34.g10493.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34732913', '34733115', '.', '+', '.', 'gene_id \"chr34.g10499\"; transcript_id \"chr34.g10499.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34735824', '34736035', '.', '+', '.', 'gene_id \"chr34.g10502\"; transcript_id \"chr34.g10502.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34736395', '34736672', '.', '-', '.', 'gene_id \"chr34.g10504\"; transcript_id \"chr34.g10504.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34741351', '34741509', '.', '-', '.', 'gene_id \"chr34.g10511\"; transcript_id \"chr34.g10511.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34753968', '34754119', '.', '-', '.', 'gene_id \"chr34.g10529\"; transcript_id \"chr34.g10529.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34759889', '34760116', '.', '-', '.', 'gene_id \"chr34.g10537\"; transcript_id \"chr34.g10537.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34760094', '34760314', '.', '-', '.', 'gene_id \"chr34.g10537\"; transcript_id \"chr34.g10537.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34784125', '34784352', '.', '-', '.', 'gene_id \"chr34.g10542\"; transcript_id \"chr34.g10542.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34788968', '34789205', '.', '-', '.', 'gene_id \"chr34.g10544\"; transcript_id \"chr34.g10544.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34789289', '34789479', '.', '-', '.', 'gene_id \"chr34.g10545\"; transcript_id \"chr34.g10545.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34790251', '34790427', '.', '-', '.', 'gene_id \"chr34.g10546\"; transcript_id \"chr34.g10546.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34790456', '34790741', '.', '+', '.', 'gene_id \"chr34.g10547\"; transcript_id \"chr34.g10547.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34849473', '34849774', '.', '-', '.', 'gene_id \"chr34.g10554\"; transcript_id \"chr34.g10554.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34855053', '34855596', '.', '+', '.', 'gene_id \"chr34.g10555\"; transcript_id \"chr34.g10555.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34850254', '34850483', '.', '+', '.', 'gene_id \"chr34.g10574\"; transcript_id \"chr34.g10574.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34850605', '34850867', '.', '-', '.', 'gene_id \"chr34.g10577\"; transcript_id \"chr34.g10577.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34852962', '34853131', '.', '-', '.', 'gene_id \"chr34.g10581\"; transcript_id \"chr34.g10581.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34858579', '34859107', '.', '-', '.', 'gene_id \"chr34.g10584\"; transcript_id \"chr34.g10584.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34860361', '34860517', '.', '+', '.', 'gene_id \"chr34.g10587\"; transcript_id \"chr34.g10587.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34860900', '34861073', '.', '-', '.', 'gene_id \"chr34.g10588\"; transcript_id \"chr34.g10588.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34861217', '34861605', '.', '+', '.', 'gene_id \"chr34.g10589\"; transcript_id \"chr34.g10589.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34869228', '34869377', '.', '+', '.', 'gene_id \"chr34.g10591\"; transcript_id \"chr34.g10591.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34886605', '34886780', '.', '-', '.', 'gene_id \"chr34.g10594\"; transcript_id \"chr34.g10594.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34904485', '34904948', '.', '+', '.', 'gene_id \"chr34.g10606\"; transcript_id \"chr34.g10606.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34904961', '34905183', '.', '+', '.', 'gene_id \"chr34.g10607\"; transcript_id \"chr34.g10607.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34905185', '34905344', '.', '+', '.', 'gene_id \"chr34.g10608\"; transcript_id \"chr34.g10608.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20435272', '20435306', '.', '+', '.', 'gene_id \"chr27.g9014\"; transcript_id \"chr27.g9014.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34931959', '34932523', '.', '-', '.', 'gene_id \"chr34.g10621\"; transcript_id \"chr34.g10621.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34941014', '34941225', '.', '-', '.', 'gene_id \"chr34.g10635\"; transcript_id \"chr34.g10635.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34941557', '34941837', '.', '+', '.', 'gene_id \"chr34.g10637\"; transcript_id \"chr34.g10637.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20451738', '20452094', '.', '-', '.', 'gene_id \"chr27.g9043\"; transcript_id \"chr27.g9043.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34992503', '34992709', '.', '+', '.', 'gene_id \"chr34.g10672\"; transcript_id \"chr34.g10672.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35151089', '35151125', '.', '-', '.', 'gene_id \"chr34.g10696\"; transcript_id \"chr34.g10696.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35319965', '35320223', '.', '+', '.', 'gene_id \"chr34.g10718\"; transcript_id \"chr34.g10718.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35332294', '35332463', '.', '-', '.', 'gene_id \"chr34.g10736\"; transcript_id \"chr34.g10736.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35332487', '35332637', '.', '-', '.', 'gene_id \"chr34.g10737\"; transcript_id \"chr34.g10737.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35345294', '35345473', '.', '-', '.', 'gene_id \"chr34.g10741\"; transcript_id \"chr34.g10741.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35342845', '35343024', '.', '-', '.', 'gene_id \"chr34.g10749\"; transcript_id \"chr34.g10749.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35344487', '35344660', '.', '-', '.', 'gene_id \"chr34.g10755\"; transcript_id \"chr34.g10755.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35345019', '35345228', '.', '+', '.', 'gene_id \"chr34.g10757\"; transcript_id \"chr34.g10757.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35348524', '35348670', '.', '-', '.', 'gene_id \"chr34.g10761\"; transcript_id \"chr34.g10761.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35356358', '35356562', '.', '+', '.', 'gene_id \"chr34.g10764\"; transcript_id \"chr34.g10764.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35358296', '35358461', '.', '+', '.', 'gene_id \"chr34.g10766\"; transcript_id \"chr34.g10766.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35405427', '35405624', '.', '-', '.', 'gene_id \"chr34.g10803\"; transcript_id \"chr34.g10803.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35405638', '35405955', '.', '+', '.', 'gene_id \"chr34.g10804\"; transcript_id \"chr34.g10804.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35406447', '35406675', '.', '-', '.', 'gene_id \"chr34.g10806\"; transcript_id \"chr34.g10806.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35407391', '35408126', '.', '+', '.', 'gene_id \"chr34.g10808\"; transcript_id \"chr34.g10808.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35410682', '35410851', '.', '-', '.', 'gene_id \"chr34.g10812\"; transcript_id \"chr34.g10812.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '20676873', '20677217', '.', '+', '.', 'gene_id \"chr27.g9194\"; transcript_id \"chr27.g9194.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35411410', '35412570', '.', '+', '.', 'gene_id \"chr34.g10813\"; transcript_id \"chr34.g10813.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35462795', '35462975', '.', '+', '.', 'gene_id \"chr34.g10844\"; transcript_id \"chr34.g10844.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35463869', '35464073', '.', '+', '.', 'gene_id \"chr34.g10847\"; transcript_id \"chr34.g10847.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35488441', '35488594', '.', '+', '.', 'gene_id \"chr34.g10867\"; transcript_id \"chr34.g10867.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35500214', '35500734', '.', '-', '.', 'gene_id \"chr34.g10874\"; transcript_id \"chr34.g10874.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35500735', '35501193', '.', '+', '.', 'gene_id \"chr34.g10875\"; transcript_id \"chr34.g10875.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35513971', '35514156', '.', '+', '.', 'gene_id \"chr34.g10890\"; transcript_id \"chr34.g10890.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35526572', '35526740', '.', '+', '.', 'gene_id \"chr34.g10899\"; transcript_id \"chr34.g10899.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35529982', '35530158', '.', '-', '.', 'gene_id \"chr34.g10900\"; transcript_id \"chr34.g10900.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35539695', '35539928', '.', '+', '.', 'gene_id \"chr34.g10907\"; transcript_id \"chr34.g10907.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35543567', '35543981', '.', '+', '.', 'gene_id \"chr34.g10909\"; transcript_id \"chr34.g10909.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35555233', '35555453', '.', '-', '.', 'gene_id \"chr34.g10918\"; transcript_id \"chr34.g10918.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35557197', '35557376', '.', '+', '.', 'gene_id \"chr34.g10920\"; transcript_id \"chr34.g10920.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35558866', '35559190', '.', '-', '.', 'gene_id \"chr34.g10924\"; transcript_id \"chr34.g10924.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35560370', '35560594', '.', '+', '.', 'gene_id \"chr34.g10927\"; transcript_id \"chr34.g10927.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35609926', '35610133', '.', '+', '.', 'gene_id \"chr34.g10934\"; transcript_id \"chr34.g10934.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35632445', '35632652', '.', '+', '.', 'gene_id \"chr34.g10936\"; transcript_id \"chr34.g10936.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35664543', '35664690', '.', '+', '.', 'gene_id \"chr34.g10943\"; transcript_id \"chr34.g10943.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35676410', '35676612', '.', '-', '.', 'gene_id \"chr34.g10948\"; transcript_id \"chr34.g10948.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35676673', '35676926', '.', '-', '.', 'gene_id \"chr34.g10949\"; transcript_id \"chr34.g10949.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35676979', '35677145', '.', '-', '.', 'gene_id \"chr34.g10950\"; transcript_id \"chr34.g10950.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35677877', '35678062', '.', '-', '.', 'gene_id \"chr34.g10952\"; transcript_id \"chr34.g10952.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35702314', '35702543', '.', '-', '.', 'gene_id \"chr34.g10957\"; transcript_id \"chr34.g10957.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35709029', '35709228', '.', '-', '.', 'gene_id \"chr34.g10958\"; transcript_id \"chr34.g10958.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35723604', '35723809', '.', '-', '.', 'gene_id \"chr34.g10970\"; transcript_id \"chr34.g10970.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35739791', '35739978', '.', '+', '.', 'gene_id \"chr34.g10981\"; transcript_id \"chr34.g10981.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35757207', '35757376', '.', '+', '.', 'gene_id \"chr34.g10989\"; transcript_id \"chr34.g10989.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35805677', '35805864', '.', '+', '.', 'gene_id \"chr34.g10997\"; transcript_id \"chr34.g10997.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '21027074', '21027111', '.', '+', '.', 'gene_id \"chr27.g9384\"; transcript_id \"chr27.g9384.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35818296', '35818451', '.', '-', '.', 'gene_id \"chr34.g11007\"; transcript_id \"chr34.g11007.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35821447', '35821653', '.', '-', '.', 'gene_id \"chr34.g11009\"; transcript_id \"chr34.g11009.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35829742', '35829972', '.', '+', '.', 'gene_id \"chr34.g11012\"; transcript_id \"chr34.g11012.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36005332', '36005606', '.', '-', '.', 'gene_id \"chr34.g11025\"; transcript_id \"chr34.g11025.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36062146', '36062481', '.', '+', '.', 'gene_id \"chr34.g11029\"; transcript_id \"chr34.g11029.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35950535', '35951061', '.', '-', '.', 'gene_id \"chr34.g11037\"; transcript_id \"chr34.g11037.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35959992', '35960525', '.', '-', '.', 'gene_id \"chr34.g11058\"; transcript_id \"chr34.g11058.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35960570', '35961133', '.', '-', '.', 'gene_id \"chr34.g11059\"; transcript_id \"chr34.g11059.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35962817', '35963212', '.', '+', '.', 'gene_id \"chr34.g11060\"; transcript_id \"chr34.g11060.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35967165', '35967392', '.', '-', '.', 'gene_id \"chr34.g11065\"; transcript_id \"chr34.g11065.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35970326', '35970663', '.', '-', '.', 'gene_id \"chr34.g11070\"; transcript_id \"chr34.g11070.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35970687', '35970860', '.', '-', '.', 'gene_id \"chr34.g11071\"; transcript_id \"chr34.g11071.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35974267', '35974655', '.', '-', '.', 'gene_id \"chr34.g11080\"; transcript_id \"chr34.g11080.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35975567', '35976143', '.', '-', '.', 'gene_id \"chr34.g11081\"; transcript_id \"chr34.g11081.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35978884', '35979142', '.', '-', '.', 'gene_id \"chr34.g11085\"; transcript_id \"chr34.g11085.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35985179', '35985452', '.', '+', '.', 'gene_id \"chr34.g11091\"; transcript_id \"chr34.g11091.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35986088', '35986141', '.', '+', '.', 'gene_id \"chr34.g11095\"; transcript_id \"chr34.g11095.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36009939', '36010372', '.', '+', '.', 'gene_id \"chr34.g11123\"; transcript_id \"chr34.g11123.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36009451', '36009941', '.', '-', '.', 'gene_id \"chr34.g11124\"; transcript_id \"chr34.g11124.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '21231847', '21231889', '.', '-', '.', 'gene_id \"chr27.g9504\"; transcript_id \"chr27.g9504.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36048982', '36049452', '.', '-', '.', 'gene_id \"chr34.g11142\"; transcript_id \"chr34.g11142.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36061171', '36061328', '.', '-', '.', 'gene_id \"chr34.g11155\"; transcript_id \"chr34.g11155.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36061484', '36061636', '.', '+', '.', 'gene_id \"chr34.g11156\"; transcript_id \"chr34.g11156.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36071994', '36072143', '.', '+', '.', 'gene_id \"chr34.g11163\"; transcript_id \"chr34.g11163.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36076918', '36077114', '.', '-', '.', 'gene_id \"chr34.g11166\"; transcript_id \"chr34.g11166.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36112310', '36112624', '.', '+', '.', 'gene_id \"chr34.g11179\"; transcript_id \"chr34.g11179.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36125046', '36125246', '.', '+', '.', 'gene_id \"chr34.g11186\"; transcript_id \"chr34.g11186.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36125442', '36125692', '.', '+', '.', 'gene_id \"chr34.g11187\"; transcript_id \"chr34.g11187.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36126937', '36127187', '.', '+', '.', 'gene_id \"chr34.g11188\"; transcript_id \"chr34.g11188.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36132450', '36132636', '.', '+', '.', 'gene_id \"chr34.g11196\"; transcript_id \"chr34.g11196.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36137429', '36137662', '.', '-', '.', 'gene_id \"chr34.g11201\"; transcript_id \"chr34.g11201.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36139007', '36139307', '.', '-', '.', 'gene_id \"chr34.g11203\"; transcript_id \"chr34.g11203.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36138852', '36139018', '.', '+', '.', 'gene_id \"chr34.g11204\"; transcript_id \"chr34.g11204.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36140962', '36141125', '.', '-', '.', 'gene_id \"chr34.g11207\"; transcript_id \"chr34.g11207.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36141124', '36141314', '.', '-', '.', 'gene_id \"chr34.g11208\"; transcript_id \"chr34.g11208.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36141714', '36141746', '.', '-', '.', 'gene_id \"chr34.g11210\"; transcript_id \"chr34.g11210.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36143483', '36143699', '.', '-', '.', 'gene_id \"chr34.g11212\"; transcript_id \"chr34.g11212.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36143739', '36143948', '.', '+', '.', 'gene_id \"chr34.g11213\"; transcript_id \"chr34.g11213.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36181441', '36181604', '.', '-', '.', 'gene_id \"chr34.g11219\"; transcript_id \"chr34.g11219.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36183795', '36184117', '.', '-', '.', 'gene_id \"chr34.g11222\"; transcript_id \"chr34.g11222.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36184722', '36184979', '.', '+', '.', 'gene_id \"chr34.g11223\"; transcript_id \"chr34.g11223.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36200784', '36200983', '.', '+', '.', 'gene_id \"chr34.g11230\"; transcript_id \"chr34.g11230.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36280097', '36280421', '.', '+', '.', 'gene_id \"chr34.g11239\"; transcript_id \"chr34.g11239.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36605486', '36605748', '.', '+', '.', 'gene_id \"chr34.g11250\"; transcript_id \"chr34.g11250.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36604882', '36605168', '.', '+', '.', 'gene_id \"chr34.g11254\"; transcript_id \"chr34.g11254.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36528030', '36528602', '.', '-', '.', 'gene_id \"chr34.g11258\"; transcript_id \"chr34.g11258.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36576219', '36576721', '.', '-', '.', 'gene_id \"chr34.g11263\"; transcript_id \"chr34.g11263.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36328931', '36329174', '.', '+', '.', 'gene_id \"chr34.g11269\"; transcript_id \"chr34.g11269.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36329163', '36329376', '.', '+', '.', 'gene_id \"chr34.g11270\"; transcript_id \"chr34.g11270.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36332830', '36333010', '.', '-', '.', 'gene_id \"chr34.g11271\"; transcript_id \"chr34.g11271.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36333008', '36333258', '.', '+', '.', 'gene_id \"chr34.g11272\"; transcript_id \"chr34.g11272.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36336015', '36336170', '.', '+', '.', 'gene_id \"chr34.g11279\"; transcript_id \"chr34.g11279.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36336465', '36336840', '.', '+', '.', 'gene_id \"chr34.g11280\"; transcript_id \"chr34.g11280.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36336871', '36337135', '.', '+', '.', 'gene_id \"chr34.g11281\"; transcript_id \"chr34.g11281.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36341229', '36341464', '.', '-', '.', 'gene_id \"chr34.g11283\"; transcript_id \"chr34.g11283.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36344235', '36345038', '.', '+', '.', 'gene_id \"chr34.g11285\"; transcript_id \"chr34.g11285.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36344015', '36344186', '.', '-', '.', 'gene_id \"chr34.g11286\"; transcript_id \"chr34.g11286.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36345354', '36345598', '.', '+', '.', 'gene_id \"chr34.g11288\"; transcript_id \"chr34.g11288.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36352749', '36353131', '.', '+', '.', 'gene_id \"chr34.g11292\"; transcript_id \"chr34.g11292.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36359568', '36360037', '.', '-', '.', 'gene_id \"chr34.g11296\"; transcript_id \"chr34.g11296.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36373822', '36373851', '.', '+', '.', 'gene_id \"chr34.g11304\"; transcript_id \"chr34.g11304.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36383949', '36384152', '.', '-', '.', 'gene_id \"chr34.g11317\"; transcript_id \"chr34.g11317.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36385986', '36386135', '.', '-', '.', 'gene_id \"chr34.g11319\"; transcript_id \"chr34.g11319.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36434342', '36434562', '.', '-', '.', 'gene_id \"chr34.g11336\"; transcript_id \"chr34.g11336.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36434687', '36434948', '.', '-', '.', 'gene_id \"chr34.g11337\"; transcript_id \"chr34.g11337.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36435558', '36435790', '.', '-', '.', 'gene_id \"chr34.g11338\"; transcript_id \"chr34.g11338.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36435981', '36436169', '.', '+', '.', 'gene_id \"chr34.g11339\"; transcript_id \"chr34.g11339.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36438547', '36438863', '.', '+', '.', 'gene_id \"chr34.g11340\"; transcript_id \"chr34.g11340.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '21587338', '21587374', '.', '+', '.', 'gene_id \"chr27.g9733\"; transcript_id \"chr27.g9733.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36447518', '36447684', '.', '-', '.', 'gene_id \"chr34.g11352\"; transcript_id \"chr34.g11352.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36461682', '36461924', '.', '-', '.', 'gene_id \"chr34.g11366\"; transcript_id \"chr34.g11366.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36467476', '36467805', '.', '-', '.', 'gene_id \"chr34.g11371\"; transcript_id \"chr34.g11371.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '21611348', '21611476', '.', '-', '.', 'gene_id \"chr27.g9756\"; transcript_id \"chr27.g9756.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36480669', '36480941', '.', '+', '.', 'gene_id \"chr34.g11377\"; transcript_id \"chr34.g11377.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36489097', '36489249', '.', '-', '.', 'gene_id \"chr34.g11378\"; transcript_id \"chr34.g11378.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36489274', '36489434', '.', '-', '.', 'gene_id \"chr34.g11379\"; transcript_id \"chr34.g11379.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36502475', '36502663', '.', '-', '.', 'gene_id \"chr34.g11381\"; transcript_id \"chr34.g11381.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36516315', '36516538', '.', '+', '.', 'gene_id \"chr34.g11384\"; transcript_id \"chr34.g11384.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36522862', '36523180', '.', '-', '.', 'gene_id \"chr34.g11388\"; transcript_id \"chr34.g11388.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36529487', '36529933', '.', '-', '.', 'gene_id \"chr34.g11391\"; transcript_id \"chr34.g11391.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36533068', '36533238', '.', '+', '.', 'gene_id \"chr34.g11398\"; transcript_id \"chr34.g11398.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36534159', '36534396', '.', '+', '.', 'gene_id \"chr34.g11400\"; transcript_id \"chr34.g11400.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36534378', '36534658', '.', '+', '.', 'gene_id \"chr34.g11401\"; transcript_id \"chr34.g11401.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36534638', '36534977', '.', '+', '.', 'gene_id \"chr34.g11402\"; transcript_id \"chr34.g11402.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36538453', '36538630', '.', '-', '.', 'gene_id \"chr34.g11409\"; transcript_id \"chr34.g11409.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36546721', '36547127', '.', '-', '.', 'gene_id \"chr34.g11414\"; transcript_id \"chr34.g11414.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36547131', '36547348', '.', '+', '.', 'gene_id \"chr34.g11415\"; transcript_id \"chr34.g11415.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36577908', '36578081', '.', '-', '.', 'gene_id \"chr34.g11426\"; transcript_id \"chr34.g11426.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36583953', '36584167', '.', '-', '.', 'gene_id \"chr34.g11429\"; transcript_id \"chr34.g11429.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36586069', '36586339', '.', '+', '.', 'gene_id \"chr34.g11432\"; transcript_id \"chr34.g11432.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36593016', '36593502', '.', '-', '.', 'gene_id \"chr34.g11437\"; transcript_id \"chr34.g11437.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36599067', '36599221', '.', '+', '.', 'gene_id \"chr34.g11445\"; transcript_id \"chr34.g11445.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36621797', '36622084', '.', '+', '.', 'gene_id \"chr34.g11461\"; transcript_id \"chr34.g11461.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36626883', '36627165', '.', '+', '.', 'gene_id \"chr34.g11466\"; transcript_id \"chr34.g11466.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36629990', '36630245', '.', '-', '.', 'gene_id \"chr34.g11469\"; transcript_id \"chr34.g11469.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36636996', '36637449', '.', '+', '.', 'gene_id \"chr34.g11473\"; transcript_id \"chr34.g11473.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22024120', '22024424', '.', '-', '.', 'gene_id \"chr27.g9858\"; transcript_id \"chr27.g9858.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36644494', '36644687', '.', '-', '.', 'gene_id \"chr34.g11477\"; transcript_id \"chr34.g11477.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36644693', '36644954', '.', '-', '.', 'gene_id \"chr34.g11478\"; transcript_id \"chr34.g11478.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36649802', '36649983', '.', '+', '.', 'gene_id \"chr34.g11481\"; transcript_id \"chr34.g11481.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36666286', '36666449', '.', '-', '.', 'gene_id \"chr34.g11499\"; transcript_id \"chr34.g11499.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36667018', '36667872', '.', '+', '.', 'gene_id \"chr34.g11501\"; transcript_id \"chr34.g11501.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36713813', '36714018', '.', '+', '.', 'gene_id \"chr34.g11507\"; transcript_id \"chr34.g11507.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36735696', '36736505', '.', '-', '.', 'gene_id \"chr34.g11510\"; transcript_id \"chr34.g11510.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36735995', '36736505', '.', '-', '.', 'gene_id \"chr34.g11510\"; transcript_id \"chr34.g11510.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36736497', '36736729', '.', '+', '.', 'gene_id \"chr34.g11511\"; transcript_id \"chr34.g11511.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36744004', '36744785', '.', '-', '.', 'gene_id \"chr34.g11529\"; transcript_id \"chr34.g11529.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36743118', '36743964', '.', '-', '.', 'gene_id \"chr34.g11530\"; transcript_id \"chr34.g11530.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36759029', '36759246', '.', '-', '.', 'gene_id \"chr34.g11537\"; transcript_id \"chr34.g11537.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36781653', '36781993', '.', '+', '.', 'gene_id \"chr34.g11542\"; transcript_id \"chr34.g11542.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22302335', '22302507', '.', '+', '.', 'gene_id \"chr27.g9912\"; transcript_id \"chr27.g9912.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36845055', '36845265', '.', '+', '.', 'gene_id \"chr34.g11557\"; transcript_id \"chr34.g11557.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36846753', '36846981', '.', '+', '.', 'gene_id \"chr34.g11561\"; transcript_id \"chr34.g11561.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22320889', '22320925', '.', '+', '.', 'gene_id \"chr27.g9933\"; transcript_id \"chr27.g9933.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36848894', '36849203', '.', '-', '.', 'gene_id \"chr34.g11562\"; transcript_id \"chr34.g11562.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36850792', '36850968', '.', '+', '.', 'gene_id \"chr34.g11563\"; transcript_id \"chr34.g11563.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36854294', '36854742', '.', '-', '.', 'gene_id \"chr34.g11570\"; transcript_id \"chr34.g11570.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36855106', '36855266', '.', '-', '.', 'gene_id \"chr34.g11572\"; transcript_id \"chr34.g11572.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36864039', '36864272', '.', '+', '.', 'gene_id \"chr34.g11581\"; transcript_id \"chr34.g11581.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36867737', '36867930', '.', '-', '.', 'gene_id \"chr34.g11582\"; transcript_id \"chr34.g11582.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36869556', '36869747', '.', '+', '.', 'gene_id \"chr34.g11583\"; transcript_id \"chr34.g11583.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36869560', '36869747', '.', '+', '.', 'gene_id \"chr34.g11583\"; transcript_id \"chr34.g11583.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36869747', '36869921', '.', '+', '.', 'gene_id \"chr34.g11584\"; transcript_id \"chr34.g11584.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36880029', '36880336', '.', '+', '.', 'gene_id \"chr34.g11589\"; transcript_id \"chr34.g11589.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36887497', '36887784', '.', '+', '.', 'gene_id \"chr34.g11595\"; transcript_id \"chr34.g11595.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36889130', '36889408', '.', '-', '.', 'gene_id \"chr34.g11601\"; transcript_id \"chr34.g11601.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36890271', '36890464', '.', '-', '.', 'gene_id \"chr34.g11604\"; transcript_id \"chr34.g11604.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36891854', '36892020', '.', '-', '.', 'gene_id \"chr34.g11607\"; transcript_id \"chr34.g11607.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22394319', '22394522', '.', '+', '.', 'gene_id \"chr27.g9976\"; transcript_id \"chr27.g9976.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22394617', '22394976', '.', '-', '.', 'gene_id \"chr27.g9977\"; transcript_id \"chr27.g9977.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22394979', '22395256', '.', '-', '.', 'gene_id \"chr27.g9978\"; transcript_id \"chr27.g9978.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36894577', '36894921', '.', '+', '.', 'gene_id \"chr34.g11611\"; transcript_id \"chr34.g11611.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36894901', '36895181', '.', '+', '.', 'gene_id \"chr34.g11612\"; transcript_id \"chr34.g11612.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36895182', '36895347', '.', '-', '.', 'gene_id \"chr34.g11613\"; transcript_id \"chr34.g11613.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36895617', '36895833', '.', '+', '.', 'gene_id \"chr34.g11615\"; transcript_id \"chr34.g11615.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37048542', '37050394', '.', '-', '.', 'gene_id \"chr34.g11638\"; transcript_id \"chr34.g11638.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37055348', '37055521', '.', '-', '.', 'gene_id \"chr34.g11640\"; transcript_id \"chr34.g11640.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22558091', '22558275', '.', '-', '.', 'gene_id \"chr27.g10013\"; transcript_id \"chr27.g10013.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37071622', '37071804', '.', '+', '.', 'gene_id \"chr34.g11647\"; transcript_id \"chr34.g11647.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37072084', '37072324', '.', '+', '.', 'gene_id \"chr34.g11648\"; transcript_id \"chr34.g11648.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37082226', '37082443', '.', '-', '.', 'gene_id \"chr34.g11650\"; transcript_id \"chr34.g11650.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37114108', '37114318', '.', '+', '.', 'gene_id \"chr34.g11655\"; transcript_id \"chr34.g11655.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37307942', '37307982', '.', '+', '.', 'gene_id \"chr34.g11676\"; transcript_id \"chr34.g11676.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22679527', '22679573', '.', '-', '.', 'gene_id \"chr27.g10047\"; transcript_id \"chr27.g10047.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22831710', '22831757', '.', '+', '.', 'gene_id \"chr27.g10062\"; transcript_id \"chr27.g10062.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22853987', '22854048', '.', '-', '.', 'gene_id \"chr27.g10069\"; transcript_id \"chr27.g10069.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22870047', '22870312', '.', '+', '.', 'gene_id \"chr27.g10076\"; transcript_id \"chr27.g10076.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37606236', '37606757', '.', '-', '.', 'gene_id \"chr34.g11709\"; transcript_id \"chr34.g11709.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37645284', '37645403', '.', '+', '.', 'gene_id \"chr34.g11717\"; transcript_id \"chr34.g11717.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37645284', '37645405', '.', '+', '.', 'gene_id \"chr34.g11717\"; transcript_id \"chr34.g11717.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37671665', '37671838', '.', '-', '.', 'gene_id \"chr34.g11722\"; transcript_id \"chr34.g11722.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37678627', '37678778', '.', '-', '.', 'gene_id \"chr34.g11723\"; transcript_id \"chr34.g11723.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37697090', '37697220', '.', '-', '.', 'gene_id \"chr34.g11726\"; transcript_id \"chr34.g11726.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37697229', '37697513', '.', '+', '.', 'gene_id \"chr34.g11727\"; transcript_id \"chr34.g11727.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37800909', '37801128', '.', '+', '.', 'gene_id \"chr34.g11734\"; transcript_id \"chr34.g11734.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37846837', '37847051', '.', '-', '.', 'gene_id \"chr34.g11746\"; transcript_id \"chr34.g11746.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37884072', '37884242', '.', '+', '.', 'gene_id \"chr34.g11751\"; transcript_id \"chr34.g11751.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37887545', '37887732', '.', '+', '.', 'gene_id \"chr34.g11753\"; transcript_id \"chr34.g11753.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37892672', '37892839', '.', '-', '.', 'gene_id \"chr34.g11756\"; transcript_id \"chr34.g11756.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22966430', '22966684', '.', '-', '.', 'gene_id \"chr27.g10134\"; transcript_id \"chr27.g10134.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37900052', '37900213', '.', '-', '.', 'gene_id \"chr34.g11767\"; transcript_id \"chr34.g11767.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37900429', '37900600', '.', '-', '.', 'gene_id \"chr34.g11768\"; transcript_id \"chr34.g11768.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37900244', '37900467', '.', '-', '.', 'gene_id \"chr34.g11768\"; transcript_id \"chr34.g11768.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37900905', '37901148', '.', '+', '.', 'gene_id \"chr34.g11770\"; transcript_id \"chr34.g11770.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37901211', '37901539', '.', '+', '.', 'gene_id \"chr34.g11771\"; transcript_id \"chr34.g11771.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37901553', '37901757', '.', '-', '.', 'gene_id \"chr34.g11773\"; transcript_id \"chr34.g11773.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37919238', '37919538', '.', '+', '.', 'gene_id \"chr34.g11782\"; transcript_id \"chr34.g11782.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37924242', '37924397', '.', '+', '.', 'gene_id \"chr34.g11785\"; transcript_id \"chr34.g11785.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37950570', '37950758', '.', '+', '.', 'gene_id \"chr34.g11796\"; transcript_id \"chr34.g11796.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37954763', '37954962', '.', '+', '.', 'gene_id \"chr34.g11799\"; transcript_id \"chr34.g11799.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37958501', '37958744', '.', '-', '.', 'gene_id \"chr34.g11806\"; transcript_id \"chr34.g11806.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37961193', '37961398', '.', '+', '.', 'gene_id \"chr34.g11808\"; transcript_id \"chr34.g11808.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37962059', '37962354', '.', '-', '.', 'gene_id \"chr34.g11810\"; transcript_id \"chr34.g11810.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37980963', '37981122', '.', '+', '.', 'gene_id \"chr34.g11813\"; transcript_id \"chr34.g11813.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38021253', '38021429', '.', '+', '.', 'gene_id \"chr34.g11829\"; transcript_id \"chr34.g11829.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38057979', '38058233', '.', '+', '.', 'gene_id \"chr34.g11835\"; transcript_id \"chr34.g11835.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38059845', '38060078', '.', '+', '.', 'gene_id \"chr34.g11837\"; transcript_id \"chr34.g11837.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38061005', '38061150', '.', '-', '.', 'gene_id \"chr34.g11838\"; transcript_id \"chr34.g11838.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '23215506', '23215760', '.', '+', '.', 'gene_id \"chr27.g10214\"; transcript_id \"chr27.g10214.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38159990', '38160179', '.', '-', '.', 'gene_id \"chr34.g11851\"; transcript_id \"chr34.g11851.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38164532', '38164720', '.', '+', '.', 'gene_id \"chr34.g11852\"; transcript_id \"chr34.g11852.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38292352', '38292561', '.', '+', '.', 'gene_id \"chr34.g11883\"; transcript_id \"chr34.g11883.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38292607', '38292824', '.', '-', '.', 'gene_id \"chr34.g11884\"; transcript_id \"chr34.g11884.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38300329', '38300364', '.', '+', '.', 'gene_id \"chr34.g11886\"; transcript_id \"chr34.g11886.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '23291349', '23291671', '.', '+', '.', 'gene_id \"chr27.g10267\"; transcript_id \"chr27.g10267.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38492364', '38492672', '.', '+', '.', 'gene_id \"chr34.g11917\"; transcript_id \"chr34.g11917.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38509380', '38509556', '.', '+', '.', 'gene_id \"chr34.g11930\"; transcript_id \"chr34.g11930.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38560534', '38560726', '.', '-', '.', 'gene_id \"chr34.g11945\"; transcript_id \"chr34.g11945.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38661096', '38661261', '.', '-', '.', 'gene_id \"chr34.g11979\"; transcript_id \"chr34.g11979.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38679506', '38679729', '.', '+', '.', 'gene_id \"chr34.g11980\"; transcript_id \"chr34.g11980.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38679869', '38680098', '.', '-', '.', 'gene_id \"chr34.g11981\"; transcript_id \"chr34.g11981.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '23537531', '23537737', '.', '-', '.', 'gene_id \"chr27.g10389\"; transcript_id \"chr27.g10389.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38724899', '38725298', '.', '+', '.', 'gene_id \"chr34.g12015\"; transcript_id \"chr34.g12015.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38743699', '38743848', '.', '+', '.', 'gene_id \"chr34.g12019\"; transcript_id \"chr34.g12019.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38749280', '38749497', '.', '-', '.', 'gene_id \"chr34.g12021\"; transcript_id \"chr34.g12021.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38770755', '38770985', '.', '-', '.', 'gene_id \"chr34.g12041\"; transcript_id \"chr34.g12041.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38772481', '38772715', '.', '+', '.', 'gene_id \"chr34.g12042\"; transcript_id \"chr34.g12042.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38773247', '38773919', '.', '-', '.', 'gene_id \"chr34.g12043\"; transcript_id \"chr34.g12043.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38774035', '38774258', '.', '-', '.', 'gene_id \"chr34.g12044\"; transcript_id \"chr34.g12044.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38775106', '38775307', '.', '+', '.', 'gene_id \"chr34.g12046\"; transcript_id \"chr34.g12046.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38785424', '38785667', '.', '-', '.', 'gene_id \"chr34.g12048\"; transcript_id \"chr34.g12048.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38785752', '38785933', '.', '-', '.', 'gene_id \"chr34.g12049\"; transcript_id \"chr34.g12049.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38794433', '38794583', '.', '+', '.', 'gene_id \"chr34.g12059\"; transcript_id \"chr34.g12059.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38795331', '38795765', '.', '+', '.', 'gene_id \"chr34.g12060\"; transcript_id \"chr34.g12060.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38796110', '38796299', '.', '-', '.', 'gene_id \"chr34.g12061\"; transcript_id \"chr34.g12061.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38799330', '38799517', '.', '-', '.', 'gene_id \"chr34.g12065\"; transcript_id \"chr34.g12065.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38807913', '38808657', '.', '+', '.', 'gene_id \"chr34.g12068\"; transcript_id \"chr34.g12068.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38808678', '38809170', '.', '-', '.', 'gene_id \"chr34.g12070\"; transcript_id \"chr34.g12070.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38844394', '38844563', '.', '+', '.', 'gene_id \"chr34.g12084\"; transcript_id \"chr34.g12084.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '23705485', '23705691', '.', '+', '.', 'gene_id \"chr27.g10474\"; transcript_id \"chr27.g10474.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '23706078', '23706312', '.', '+', '.', 'gene_id \"chr27.g10475\"; transcript_id \"chr27.g10475.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38880762', '38880964', '.', '+', '.', 'gene_id \"chr34.g12101\"; transcript_id \"chr34.g12101.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38896876', '38897335', '.', '-', '.', 'gene_id \"chr34.g12106\"; transcript_id \"chr34.g12106.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38896776', '38896900', '.', '+', '.', 'gene_id \"chr34.g12108\"; transcript_id \"chr34.g12108.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38907120', '38907374', '.', '+', '.', 'gene_id \"chr34.g12112\"; transcript_id \"chr34.g12112.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38916942', '38917842', '.', '+', '.', 'gene_id \"chr34.g12114\"; transcript_id \"chr34.g12114.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38930263', '38930421', '.', '-', '.', 'gene_id \"chr34.g12127\"; transcript_id \"chr34.g12127.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38950311', '38950503', '.', '-', '.', 'gene_id \"chr34.g12148\"; transcript_id \"chr34.g12148.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38952974', '38953230', '.', '-', '.', 'gene_id \"chr34.g12153\"; transcript_id \"chr34.g12153.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38955409', '38955667', '.', '+', '.', 'gene_id \"chr34.g12156\"; transcript_id \"chr34.g12156.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38963531', '38963728', '.', '+', '.', 'gene_id \"chr34.g12160\"; transcript_id \"chr34.g12160.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38985282', '38985517', '.', '+', '.', 'gene_id \"chr34.g12164\"; transcript_id \"chr34.g12164.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39009954', '39010156', '.', '+', '.', 'gene_id \"chr34.g12177\"; transcript_id \"chr34.g12177.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39014377', '39014787', '.', '+', '.', 'gene_id \"chr34.g12178\"; transcript_id \"chr34.g12178.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39015561', '39015787', '.', '-', '.', 'gene_id \"chr34.g12180\"; transcript_id \"chr34.g12180.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39015387', '39015580', '.', '+', '.', 'gene_id \"chr34.g12181\"; transcript_id \"chr34.g12181.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39042654', '39042862', '.', '-', '.', 'gene_id \"chr34.g12191\"; transcript_id \"chr34.g12191.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39049020', '39049174', '.', '+', '.', 'gene_id \"chr34.g12193\"; transcript_id \"chr34.g12193.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39051263', '39051558', '.', '-', '.', 'gene_id \"chr34.g12195\"; transcript_id \"chr34.g12195.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39051543', '39051695', '.', '+', '.', 'gene_id \"chr34.g12196\"; transcript_id \"chr34.g12196.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39053559', '39053771', '.', '+', '.', 'gene_id \"chr34.g12197\"; transcript_id \"chr34.g12197.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39053758', '39053920', '.', '-', '.', 'gene_id \"chr34.g12198\"; transcript_id \"chr34.g12198.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39059207', '39059423', '.', '-', '.', 'gene_id \"chr34.g12207\"; transcript_id \"chr34.g12207.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39080138', '39080439', '.', '-', '.', 'gene_id \"chr34.g12213\"; transcript_id \"chr34.g12213.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39083206', '39083422', '.', '-', '.', 'gene_id \"chr34.g12214\"; transcript_id \"chr34.g12214.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39109891', '39110226', '.', '-', '.', 'gene_id \"chr34.g12219\"; transcript_id \"chr34.g12219.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39177792', '39178000', '.', '-', '.', 'gene_id \"chr34.g12231\"; transcript_id \"chr34.g12231.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39247092', '39247318', '.', '+', '.', 'gene_id \"chr34.g12254\"; transcript_id \"chr34.g12254.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39249561', '39249738', '.', '+', '.', 'gene_id \"chr34.g12255\"; transcript_id \"chr34.g12255.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39267150', '39267300', '.', '-', '.', 'gene_id \"chr34.g12272\"; transcript_id \"chr34.g12272.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39274381', '39274575', '.', '-', '.', 'gene_id \"chr34.g12278\"; transcript_id \"chr34.g12278.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39276152', '39276335', '.', '-', '.', 'gene_id \"chr34.g12279\"; transcript_id \"chr34.g12279.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39297023', '39297251', '.', '+', '.', 'gene_id \"chr34.g12287\"; transcript_id \"chr34.g12287.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39312715', '39312933', '.', '-', '.', 'gene_id \"chr34.g12288\"; transcript_id \"chr34.g12288.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39320497', '39320664', '.', '-', '.', 'gene_id \"chr34.g12290\"; transcript_id \"chr34.g12290.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24671028', '24671585', '.', '+', '.', 'gene_id \"chr27.g10669\"; transcript_id \"chr27.g10669.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39370044', '39370207', '.', '-', '.', 'gene_id \"chr34.g12307\"; transcript_id \"chr34.g12307.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39413374', '39413633', '.', '+', '.', 'gene_id \"chr34.g12319\"; transcript_id \"chr34.g12319.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39417462', '39417841', '.', '+', '.', 'gene_id \"chr34.g12322\"; transcript_id \"chr34.g12322.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39417210', '39417484', '.', '-', '.', 'gene_id \"chr34.g12323\"; transcript_id \"chr34.g12323.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39544392', '39544546', '.', '-', '.', 'gene_id \"chr34.g12338\"; transcript_id \"chr34.g12338.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39568662', '39568897', '.', '-', '.', 'gene_id \"chr34.g12345\"; transcript_id \"chr34.g12345.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39573013', '39573334', '.', '-', '.', 'gene_id \"chr34.g12350\"; transcript_id \"chr34.g12350.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40106481', '40106526', '.', '+', '.', 'gene_id \"chr34.g12391\"; transcript_id \"chr34.g12391.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40340803', '40340848', '.', '+', '.', 'gene_id \"chr34.g12402\"; transcript_id \"chr34.g12402.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24845274', '24845355', '.', '-', '.', 'gene_id \"chr27.g10818\"; transcript_id \"chr27.g10818.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25191819', '25191856', '.', '+', '.', 'gene_id \"chr27.g10901\"; transcript_id \"chr27.g10901.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25191820', '25191860', '.', '-', '.', 'gene_id \"chr27.g10902\"; transcript_id \"chr27.g10902.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40721879', '40722047', '.', '-', '.', 'gene_id \"chr34.g12541\"; transcript_id \"chr34.g12541.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25454606', '25454667', '.', '-', '.', 'gene_id \"chr27.g10969\"; transcript_id \"chr27.g10969.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25524247', '25524284', '.', '-', '.', 'gene_id \"chr27.g11005\"; transcript_id \"chr27.g11005.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40827888', '40827935', '.', '-', '.', 'gene_id \"chr34.g12614\"; transcript_id \"chr34.g12614.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40855463', '40855705', '.', '-', '.', 'gene_id \"chr34.g12624\"; transcript_id \"chr34.g12624.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40855804', '40855971', '.', '+', '.', 'gene_id \"chr34.g12625\"; transcript_id \"chr34.g12625.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '41057967', '41058002', '.', '-', '.', 'gene_id \"chr34.g12671\"; transcript_id \"chr34.g12671.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25921843', '25922248', '.', '-', '.', 'gene_id \"chr27.g11149\"; transcript_id \"chr27.g11149.i2\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '41838498', '41838663', '.', '+', '.', 'gene_id \"chr34.g12785\"; transcript_id \"chr34.g12785.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26459984', '26460264', '.', '+', '.', 'gene_id \"chr27.g11248\"; transcript_id \"chr27.g11248.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26504066', '26505109', '.', '+', '.', 'gene_id \"chr27.g11263\"; transcript_id \"chr27.g11263.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26504198', '26504503', '.', '+', '.', 'gene_id \"chr27.g11263\"; transcript_id \"chr27.g11263.i3\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '41992827', '41992949', '.', '+', '.', 'gene_id \"chr34.g12823\"; transcript_id \"chr34.g12823.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '42004771', '42005003', '.', '+', '.', 'gene_id \"chr34.g12829\"; transcript_id \"chr34.g12829.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26567503', '26567679', '.', '-', '.', 'gene_id \"chr27.g11292\"; transcript_id \"chr27.g11292.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26567821', '26568003', '.', '-', '.', 'gene_id \"chr27.g11293\"; transcript_id \"chr27.g11293.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26627014', '26627046', '.', '+', '.', 'gene_id \"chr27.g11332\"; transcript_id \"chr27.g11332.i1\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27187776', '27187973', '100', '-', '.', 'gene_id \"chr34.g12888\"; transcript_id \"blat.proc28144.chain_22145\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34611641', '34611807', '100', '-', '.', 'gene_id \"chr34.g12898\"; transcript_id \"blat.proc28144.chain_23264\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23924480', '23924662', '100', '+', '.', 'gene_id \"chr34.g12906\"; transcript_id \"blat.proc28144.chain_18908\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30444301', '30444515', '100', '+', '.', 'gene_id \"chr34.g12922\"; transcript_id \"blat.proc28144.chain_22403\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5992050', '5992381', '100', '+', '.', 'gene_id \"chr34.g12962\"; transcript_id \"blat.proc4587.chain_12922\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39418258', '39418410', '100', '-', '.', 'gene_id \"chr34.g12965\"; transcript_id \"blat.proc28144.chain_23894\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33791108', '33791282', '100', '-', '.', 'gene_id \"chr34.g12975\"; transcript_id \"blat.proc28144.chain_22521\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34719469', '34719646', '100', '-', '.', 'gene_id \"chr34.g12984\"; transcript_id \"blat.proc28144.chain_22968\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27095527', '27095674', '100', '-', '.', 'gene_id \"chr34.g12989\"; transcript_id \"blat.proc28144.chain_19974\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39321384', '39321756', '100', '+', '.', 'gene_id \"chr34.g12993\"; transcript_id \"blat.proc28144.chain_22600\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34908657', '34908824', '100', '+', '.', 'gene_id \"chr34.g12998\"; transcript_id \"blat.proc28144.chain_22504\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35462229', '35462399', '100', '-', '.', 'gene_id \"chr34.g13012\"; transcript_id \"blat.proc28144.chain_23167\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25987222', '25987500', '100', '-', '.', 'gene_id \"chr34.g13043\"; transcript_id \"blat.proc28144.chain_22126\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38940269', '38940426', '100', '+', '.', 'gene_id \"chr34.g13048\"; transcript_id \"blat.proc28144.chain_22876\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33188098', '33188228', '100', '+', '.', 'gene_id \"chr34.g13056\"; transcript_id \"blat.proc28144.chain_23411\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26012049', '26012136', '100', '+', '.', 'gene_id \"chr34.g13066\"; transcript_id \"blat.proc28144.chain_19205\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27431863', '27432053', '100', '-', '.', 'gene_id \"chr34.g13076\"; transcript_id \"blat.proc28144.chain_19199\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26324323', '26324577', '100', '-', '.', 'gene_id \"chr34.g13077\"; transcript_id \"blat.proc28144.chain_20799\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30640125', '30640337', '100', '-', '.', 'gene_id \"chr34.g13092\"; transcript_id \"blat.proc28144.chain_22207\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26641011', '26641146', '100', '-', '.', 'gene_id \"chr34.g13098\"; transcript_id \"blat.proc28144.chain_20764\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34830235', '34830435', '100', '-', '.', 'gene_id \"chr34.g13105\"; transcript_id \"blat.proc28144.chain_23338\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26020008', '26020267', '100', '-', '.', 'gene_id \"chr34.g13127\"; transcript_id \"blat.proc28144.chain_19354\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36634830', '36634952', '100', '-', '.', 'gene_id \"chr34.g13161\"; transcript_id \"blat.proc4587.chain_9546\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25847047', '25847246', '100', '+', '.', 'gene_id \"chr34.g13162\"; transcript_id \"blat.proc28144.chain_18933\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25224065', '25224239', '100', '-', '.', 'gene_id \"chr34.g13171\"; transcript_id \"blat.proc28144.chain_20054\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '2140921', '2141108', '100', '+', '.', 'gene_id \"chr34.g13174\"; transcript_id \"blat.proc4005.chain_21804\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25849275', '25849527', '100', '-', '.', 'gene_id \"chr34.g13185\"; transcript_id \"blat.proc28144.chain_22043\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26566924', '26567130', '100', '+', '.', 'gene_id \"chr34.g13188\"; transcript_id \"blat.proc28144.chain_21873\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26663378', '26663861', '100', '-', '.', 'gene_id \"chr34.g13189\"; transcript_id \"blat.proc28144.chain_19009\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38775740', '38776144', '100', '-', '.', 'gene_id \"chr34.g13206\"; transcript_id \"blat.proc28144.chain_22835\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26196543', '26196728', '100', '-', '.', 'gene_id \"chr34.g13207\"; transcript_id \"blat.proc28144.chain_19183\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35545610', '35545826', '100', '-', '.', 'gene_id \"chr34.g13216\"; transcript_id \"blat.proc28144.chain_22716\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34677571', '34677723', '100', '-', '.', 'gene_id \"chr34.g13225\"; transcript_id \"blat.proc28144.chain_22677\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35429122', '35429294', '100', '+', '.', 'gene_id \"chr34.g13228\"; transcript_id \"blat.proc28144.chain_22911\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26675883', '26676050', '100', '+', '.', 'gene_id \"chr34.g13244\"; transcript_id \"blat.proc28144.chain_21799\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38796672', '38796954', '100', '+', '.', 'gene_id \"chr34.g13250\"; transcript_id \"blat.proc28144.chain_23500\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34908417', '34908588', '100', '-', '.', 'gene_id \"chr34.g13253\"; transcript_id \"blat.proc28144.chain_22503\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32557397', '32557592', '100', '-', '.', 'gene_id \"chr34.g13256\"; transcript_id \"blat.proc28144.chain_22969\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24835074', '24835266', '100', '+', '.', 'gene_id \"chr34.g13271\"; transcript_id \"blat.proc28144.chain_18941\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39613294', '39613491', '100', '+', '.', 'gene_id \"chr34.g13273\"; transcript_id \"blat.proc28144.chain_22868\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36113120', '36113219', '100', '+', '.', 'gene_id \"chr34.g13275\"; transcript_id \"blat.proc28144.chain_23321\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35961866', '35962021', '100', '-', '.', 'gene_id \"chr34.g13307\"; transcript_id \"blat.proc28144.chain_22663\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26551962', '26552222', '100', '+', '.', 'gene_id \"chr34.g13311\"; transcript_id \"blat.proc28144.chain_21944\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '1466046', '1466077', '100', '+', '.', 'gene_id \"chr34.g13321\"; transcript_id \"blat.proc18128.chain_22469\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26688758', '26689091', '100', '-', '.', 'gene_id \"chr34.g13323\"; transcript_id \"blat.proc28144.chain_19928\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27118508', '27118668', '100', '-', '.', 'gene_id \"chr34.g13324\"; transcript_id \"blat.proc28144.chain_22027\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27131376', '27131585', '100', '-', '.', 'gene_id \"chr34.g13330\"; transcript_id \"blat.proc28144.chain_19324\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16103353', '16103393', '100', '+', '.', 'gene_id \"chr34.g13336\"; transcript_id \"blat.proc18128.chain_7632\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36044617', '36044781', '100', '-', '.', 'gene_id \"chr34.g13359\"; transcript_id \"blat.proc28144.chain_22492\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38918371', '38918534', '100', '+', '.', 'gene_id \"chr34.g13375\"; transcript_id \"blat.proc28144.chain_23148\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32589084', '32589238', '100', '-', '.', 'gene_id \"chr34.g13410\"; transcript_id \"blat.proc28144.chain_22943\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38095841', '38095988', '100', '+', '.', 'gene_id \"chr34.g13433\"; transcript_id \"blat.proc28144.chain_22676\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26340630', '26340866', '100', '+', '.', 'gene_id \"chr34.g13451\"; transcript_id \"blat.proc28144.chain_19180\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32091485', '32091635', '100', '+', '.', 'gene_id \"chr34.g13488\"; transcript_id \"blat.proc28144.chain_23439\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26305528', '26305697', '100', '+', '.', 'gene_id \"chr34.g13489\"; transcript_id \"blat.proc28144.chain_18946\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26478041', '26478312', '100', '+', '.', 'gene_id \"chr34.g13494\"; transcript_id \"blat.proc28144.chain_21625\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38890589', '38890866', '100', '-', '.', 'gene_id \"chr34.g13511\"; transcript_id \"blat.proc28144.chain_23251\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26928130', '26928285', '100', '-', '.', 'gene_id \"chr34.g13520\"; transcript_id \"blat.proc28144.chain_21920\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26191720', '26191795', '100', '+', '.', 'gene_id \"chr34.g13522\"; transcript_id \"blat.proc28144.chain_20703\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32299497', '32299622', '100', '-', '.', 'gene_id \"chr34.g13533\"; transcript_id \"blat.proc28144.chain_22566\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26629970', '26630343', '100', '-', '.', 'gene_id \"chr34.g13540\"; transcript_id \"blat.proc28144.chain_22090\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26759297', '26759430', '100', '-', '.', 'gene_id \"chr34.g13549\"; transcript_id \"blat.proc28144.chain_22180\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26582142', '26582308', '100', '-', '.', 'gene_id \"chr34.g13563\"; transcript_id \"blat.proc28144.chain_18962\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38955808', '38956069', '100', '+', '.', 'gene_id \"chr34.g13570\"; transcript_id \"blat.proc4587.chain_358\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27214800', '27214971', '100', '+', '.', 'gene_id \"chr34.g13575\"; transcript_id \"blat.proc28144.chain_22173\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26398442', '26398693', '100', '-', '.', 'gene_id \"chr34.g13576\"; transcript_id \"blat.proc28144.chain_18985\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26247444', '26247605', '100', '+', '.', 'gene_id \"chr34.g13581\"; transcript_id \"blat.proc28144.chain_22011\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5391996', '5392037', '100', '-', '.', 'gene_id \"chr34.g13582\"; transcript_id \"blat.proc28144.chain_12540\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36711373', '36711559', '100', '-', '.', 'gene_id \"chr34.g13610\"; transcript_id \"blat.proc28144.chain_23573\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27325587', '27325711', '100', '+', '.', 'gene_id \"chr34.g13612\"; transcript_id \"blat.proc28144.chain_19104\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27121164', '27121258', '100', '-', '.', 'gene_id \"chr34.g13618\"; transcript_id \"blat.proc28144.chain_20493\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24441231', '24441458', '100', '-', '.', 'gene_id \"chr34.g13623\"; transcript_id \"blat.proc28144.chain_22424\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34214735', '34214958', '100', '+', '.', 'gene_id \"chr34.g13641\"; transcript_id \"blat.proc28144.chain_22861\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36426663', '36426826', '100', '+', '.', 'gene_id \"chr34.g13672\"; transcript_id \"blat.proc28144.chain_23006\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26544395', '26544572', '100', '+', '.', 'gene_id \"chr34.g13703\"; transcript_id \"blat.proc28144.chain_21953\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39473058', '39473233', '100', '-', '.', 'gene_id \"chr34.g13706\"; transcript_id \"blat.proc28144.chain_23329\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32730715', '32730759', '100', '-', '.', 'gene_id \"chr34.g13723\"; transcript_id \"blat.proc28144.chain_5426\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27107752', '27107930', '100', '-', '.', 'gene_id \"chr34.g13726\"; transcript_id \"blat.proc28144.chain_19133\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25977081', '25977286', '100', '-', '.', 'gene_id \"chr34.g13729\"; transcript_id \"blat.proc28144.chain_19179\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33270180', '33270219', '100', '+', '.', 'gene_id \"chr34.g13751\"; transcript_id \"blat.proc28144.chain_11569\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33646534', '33646764', '100', '+', '.', 'gene_id \"chr34.g13760\"; transcript_id \"blat.proc28144.chain_23234\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34848018', '34848191', '100', '-', '.', 'gene_id \"chr34.g13786\"; transcript_id \"blat.proc28144.chain_22931\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29467528', '29467575', '100', '-', '.', 'gene_id \"chr34.g13798\"; transcript_id \"blat.proc18128.chain_26608\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26171447', '26171627', '100', '+', '.', 'gene_id \"chr34.g13799\"; transcript_id \"blat.proc28144.chain_19158\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27037831', '27038135', '100', '+', '.', 'gene_id \"chr34.g13802\"; transcript_id \"blat.proc28144.chain_19362\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34027088', '34027275', '100', '-', '.', 'gene_id \"chr34.g13810\"; transcript_id \"blat.proc28144.chain_23967\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26753260', '26753433', '100', '-', '.', 'gene_id \"chr34.g13822\"; transcript_id \"blat.proc28144.chain_19040\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34045317', '34045466', '100', '-', '.', 'gene_id \"chr34.g13837\"; transcript_id \"blat.proc28144.chain_22675\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27230166', '27230430', '100', '+', '.', 'gene_id \"chr34.g13855\"; transcript_id \"blat.proc28144.chain_19504\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24070825', '24070990', '100', '+', '.', 'gene_id \"chr34.g13863\"; transcript_id \"blat.proc28144.chain_18802\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '41879127', '41879194', '100', '-', '.', 'gene_id \"chr34.g13890\"; transcript_id \"blat.proc28144.chain_17343\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26137953', '26138172', '100', '+', '.', 'gene_id \"chr34.g13898\"; transcript_id \"blat.proc28144.chain_20294\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26337460', '26337676', '100', '-', '.', 'gene_id \"chr34.g13914\"; transcript_id \"blat.proc28144.chain_19099\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38880370', '38880654', '100', '-', '.', 'gene_id \"chr34.g13946\"; transcript_id \"blat.proc28144.chain_23906\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26839897', '26840001', '100', '+', '.', 'gene_id \"chr34.g13949\"; transcript_id \"blat.proc28144.chain_22091\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26272094', '26272252', '100', '-', '.', 'gene_id \"chr34.g13971\"; transcript_id \"blat.proc28144.chain_22151\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30413738', '30413894', '100', '+', '.', 'gene_id \"chr34.g13985\"; transcript_id \"blat.proc28144.chain_22328\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26586983', '26587176', '100', '-', '.', 'gene_id \"chr34.g13998\"; transcript_id \"blat.proc28144.chain_22171\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26367823', '26367994', '100', '+', '.', 'gene_id \"chr34.g14025\"; transcript_id \"blat.proc28144.chain_22162\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27328825', '27328990', '100', '-', '.', 'gene_id \"chr34.g14046\"; transcript_id \"blat.proc28144.chain_22183\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30425159', '30425323', '100', '-', '.', 'gene_id \"chr34.g14049\"; transcript_id \"blat.proc18128.chain_13604\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32947898', '32948143', '100', '-', '.', 'gene_id \"chr34.g14066\"; transcript_id \"blat.proc28144.chain_23731\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35991079', '35991284', '100', '-', '.', 'gene_id \"chr34.g14076\"; transcript_id \"blat.proc28144.chain_22505\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34269918', '34270088', '100', '-', '.', 'gene_id \"chr34.g14084\"; transcript_id \"blat.proc28144.chain_23485\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35805223', '35805523', '100', '+', '.', 'gene_id \"chr34.g14094\"; transcript_id \"blat.proc18128.chain_25866\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33253931', '33254137', '100', '-', '.', 'gene_id \"chr34.g14096\"; transcript_id \"blat.proc28144.chain_22536\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '8578596', '8578847', '100', '-', '.', 'gene_id \"chr34.g14101\"; transcript_id \"blat.proc4587.chain_951\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '22362241', '22362275', '100', '-', '.', 'gene_id \"chr34.g14103\"; transcript_id \"blat.proc4587.chain_1810\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27188146', '27188389', '100', '+', '.', 'gene_id \"chr34.g14113\"; transcript_id \"blat.proc28144.chain_22080\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26904329', '26904542', '100', '-', '.', 'gene_id \"chr34.g14125\"; transcript_id \"blat.proc28144.chain_19194\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27315568', '27315871', '100', '+', '.', 'gene_id \"chr34.g14146\"; transcript_id \"blat.proc28144.chain_19650\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27284152', '27284197', '100', '+', '.', 'gene_id \"chr34.g14153\"; transcript_id \"blat.proc28144.chain_21319\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33845508', '33845672', '100', '-', '.', 'gene_id \"chr34.g14161\"; transcript_id \"blat.proc28144.chain_22488\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35313897', '35314085', '100', '+', '.', 'gene_id \"chr34.g14171\"; transcript_id \"blat.proc28144.chain_22748\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26233705', '26233890', '100', '+', '.', 'gene_id \"chr34.g14177\"; transcript_id \"blat.proc28144.chain_20952\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37876305', '37876477', '100', '-', '.', 'gene_id \"chr34.g14193\"; transcript_id \"blat.proc28144.chain_22799\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35502187', '35502386', '100', '-', '.', 'gene_id \"chr34.g14239\"; transcript_id \"blat.proc28144.chain_22996\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25924467', '25924678', '100', '-', '.', 'gene_id \"chr34.g14240\"; transcript_id \"blat.proc28144.chain_22020\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26582592', '26582743', '100', '+', '.', 'gene_id \"chr34.g14255\"; transcript_id \"blat.proc28144.chain_22066\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36848645', '36848802', '100', '-', '.', 'gene_id \"chr34.g14257\"; transcript_id \"blat.proc28144.chain_23709\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27660126', '27660377', '100', '+', '.', 'gene_id \"chr34.g14266\"; transcript_id \"blat.proc28144.chain_18979\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26607844', '26608051', '100', '-', '.', 'gene_id \"chr34.g14287\"; transcript_id \"blat.proc28144.chain_20624\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26686317', '26686435', '100', '+', '.', 'gene_id \"chr34.g14316\"; transcript_id \"blat.proc28144.chain_22088\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34243588', '34243990', '100', '+', '.', 'gene_id \"chr34.g14330\"; transcript_id \"blat.proc28144.chain_22916\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26153483', '26153660', '100', '+', '.', 'gene_id \"chr34.g14360\"; transcript_id \"blat.proc28144.chain_19189\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27014753', '27014934', '100', '-', '.', 'gene_id \"chr34.g14401\"; transcript_id \"blat.proc28144.chain_22157\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33021859', '33021889', '100', '-', '.', 'gene_id \"chr34.g14404\"; transcript_id \"blat.proc18128.chain_21356\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34727351', '34727733', '100', '-', '.', 'gene_id \"chr34.g14418\"; transcript_id \"blat.proc28144.chain_22683\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25783177', '25783266', '100', '+', '.', 'gene_id \"chr34.g14433\"; transcript_id \"blat.proc28144.chain_22002\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25944360', '25944616', '100', '+', '.', 'gene_id \"chr34.g14451\"; transcript_id \"blat.proc28144.chain_19811\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26605895', '26606025', '100', '+', '.', 'gene_id \"chr34.g14458\"; transcript_id \"blat.proc28144.chain_19263\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33815551', '33815755', '100', '-', '.', 'gene_id \"chr34.g14466\"; transcript_id \"blat.proc28144.chain_22697\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40576077', '40576108', '100', '-', '.', 'gene_id \"chr34.g14480\"; transcript_id \"blat.proc4587.chain_9302\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39568064', '39568228', '100', '+', '.', 'gene_id \"chr34.g14502\"; transcript_id \"blat.proc28144.chain_22843\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36446581', '36446761', '100', '-', '.', 'gene_id \"chr34.g14504\"; transcript_id \"blat.proc28144.chain_22761\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26127821', '26128006', '100', '+', '.', 'gene_id \"chr34.g14520\"; transcript_id \"blat.proc28144.chain_21977\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37151434', '37151489', '100', '-', '.', 'gene_id \"chr34.g14524\"; transcript_id \"blat.proc4587.chain_1061\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32647741', '32648098', '100', '-', '.', 'gene_id \"chr34.g14550\"; transcript_id \"blat.proc28144.chain_23686\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26400204', '26400559', '100', '-', '.', 'gene_id \"chr34.g14554\"; transcript_id \"blat.proc28144.chain_21842\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24735099', '24735305', '100', '+', '.', 'gene_id \"chr34.g14559\"; transcript_id \"blat.proc28144.chain_21032\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32299638', '32299803', '100', '+', '.', 'gene_id \"chr34.g14600\"; transcript_id \"blat.proc28144.chain_22565\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34716528', '34716722', '100', '-', '.', 'gene_id \"chr34.g14622\"; transcript_id \"blat.proc28144.chain_23746\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27315880', '27316189', '100', '+', '.', 'gene_id \"chr34.g14624\"; transcript_id \"blat.proc28144.chain_19651\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36142594', '36142860', '100', '-', '.', 'gene_id \"chr34.g14664\"; transcript_id \"blat.proc28144.chain_23451\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33308264', '33308410', '100', '-', '.', 'gene_id \"chr34.g14666\"; transcript_id \"blat.proc18128.chain_21420\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29718866', '29718898', '100', '+', '.', 'gene_id \"chr34.g14675\"; transcript_id \"blat.proc28144.chain_22460\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33260559', '33260767', '100', '+', '.', 'gene_id \"chr34.g14684\"; transcript_id \"blat.proc28144.chain_22790\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39414052', '39414251', '100', '+', '.', 'gene_id \"chr34.g14689\"; transcript_id \"blat.proc28144.chain_22950\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34269455', '34269806', '100', '+', '.', 'gene_id \"chr34.g14698\"; transcript_id \"blat.proc28144.chain_23486\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26849626', '26849745', '100', '-', '.', 'gene_id \"chr34.g14704\"; transcript_id \"blat.proc28144.chain_22089\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36547363', '36547607', '100', '-', '.', 'gene_id \"chr34.g14734\"; transcript_id \"blat.proc28144.chain_22952\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36143088', '36143271', '100', '-', '.', 'gene_id \"chr34.g14752\"; transcript_id \"blat.proc28144.chain_23450\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34187105', '34187335', '100', '+', '.', 'gene_id \"chr34.g14753\"; transcript_id \"blat.proc28144.chain_22691\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32444951', '32444999', '100', '+', '.', 'gene_id \"chr34.g14769\"; transcript_id \"blat.proc18128.chain_609\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24999989', '25000541', '100', '+', '.', 'gene_id \"chr34.g14774\"; transcript_id \"blat.proc28144.chain_20539\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30416921', '30417074', '100', '-', '.', 'gene_id \"chr34.g14792\"; transcript_id \"blat.proc28144.chain_22417\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36329964', '36330163', '100', '+', '.', 'gene_id \"chr34.g14794\"; transcript_id \"blat.proc28144.chain_22937\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25034219', '25034376', '100', '+', '.', 'gene_id \"chr34.g14805\"; transcript_id \"blat.proc28144.chain_21983\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26555413', '26555583', '100', '+', '.', 'gene_id \"chr34.g14813\"; transcript_id \"blat.proc28144.chain_19344\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36134385', '36134553', '100', '+', '.', 'gene_id \"chr34.g14819\"; transcript_id \"blat.proc28144.chain_22507\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38796991', '38797162', '100', '+', '.', 'gene_id \"chr34.g14824\"; transcript_id \"blat.proc28144.chain_23499\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29230573', '29230611', '100', '+', '.', 'gene_id \"chr34.g14830\"; transcript_id \"blat.proc4005.chain_19281\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34944371', '34944535', '100', '+', '.', 'gene_id \"chr34.g14845\"; transcript_id \"blat.proc28144.chain_23680\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36462899', '36463117', '100', '-', '.', 'gene_id \"chr34.g14856\"; transcript_id \"blat.proc28144.chain_22924\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24501297', '24501335', '100', '-', '.', 'gene_id \"chr34.g14863\"; transcript_id \"blat.proc18128.chain_14362\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26300527', '26300667', '100', '+', '.', 'gene_id \"chr34.g14867\"; transcript_id \"blat.proc28144.chain_19656\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33815265', '33815515', '100', '-', '.', 'gene_id \"chr34.g14871\"; transcript_id \"blat.proc28144.chain_22698\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26503280', '26503453', '100', '-', '.', 'gene_id \"chr34.g14884\"; transcript_id \"blat.proc28144.chain_21940\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32376323', '32376546', '100', '-', '.', 'gene_id \"chr34.g14893\"; transcript_id \"blat.proc28144.chain_22807\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36428544', '36428734', '100', '-', '.', 'gene_id \"chr34.g14895\"; transcript_id \"blat.proc28144.chain_22801\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26502785', '26502947', '100', '-', '.', 'gene_id \"chr34.g14898\"; transcript_id \"blat.proc28144.chain_22085\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34243331', '34243523', '100', '+', '.', 'gene_id \"chr34.g14906\"; transcript_id \"blat.proc28144.chain_22915\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38869804', '38869997', '100', '-', '.', 'gene_id \"chr34.g14954\"; transcript_id \"blat.proc28144.chain_23455\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39318785', '39318950', '100', '-', '.', 'gene_id \"chr34.g14970\"; transcript_id \"blat.proc28144.chain_22792\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39194492', '39194799', '100', '+', '.', 'gene_id \"chr34.g14978\"; transcript_id \"blat.proc28144.chain_23104\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36058908', '36059277', '100', '+', '.', 'gene_id \"chr34.g14985\"; transcript_id \"blat.proc28144.chain_22754\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40722059', '40722169', '100', '+', '.', 'gene_id \"chr34.g15004\"; transcript_id \"blat.proc4587.chain_9774\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25075114', '25075269', '100', '-', '.', 'gene_id \"chr34.g15035\"; transcript_id \"blat.proc28144.chain_19436\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35506308', '35506559', '100', '-', '.', 'gene_id \"chr34.g15046\"; transcript_id \"blat.proc28144.chain_24053\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36563990', '36564150', '100', '-', '.', 'gene_id \"chr34.g15053\"; transcript_id \"blat.proc28144.chain_22811\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35666048', '35666220', '100', '+', '.', 'gene_id \"chr34.g15058\"; transcript_id \"blat.proc28144.chain_22695\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25811552', '25811599', '100', '+', '.', 'gene_id \"chr34.g15096\"; transcript_id \"blat.proc4005.chain_19446\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24751226', '24751381', '100', '+', '.', 'gene_id \"chr34.g15125\"; transcript_id \"blat.proc28144.chain_21954\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26372666', '26372787', '100', '-', '.', 'gene_id \"chr34.g15135\"; transcript_id \"blat.proc28144.chain_19211\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36563705', '36563949', '100', '+', '.', 'gene_id \"chr34.g15157\"; transcript_id \"blat.proc28144.chain_22812\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33646800', '33646965', '100', '-', '.', 'gene_id \"chr34.g15159\"; transcript_id \"blat.proc28144.chain_23233\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '808347', '808381', '100', '+', '.', 'gene_id \"chr34.g15182\"; transcript_id \"blat.proc28144.chain_19533\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27249315', '27249470', '100', '-', '.', 'gene_id \"chr34.g15193\"; transcript_id \"blat.proc28144.chain_19408\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26165113', '26165302', '100', '+', '.', 'gene_id \"chr34.g15202\"; transcript_id \"blat.proc28144.chain_20080\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26699783', '26699844', '100', '-', '.', 'gene_id \"chr34.g15226\"; transcript_id \"blat.proc18128.chain_25893\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16980575', '16980648', '100', '+', '.', 'gene_id \"chr34.g15232\"; transcript_id \"blat.proc18128.chain_21127\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26186342', '26186523', '100', '+', '.', 'gene_id \"chr34.g15235\"; transcript_id \"blat.proc28144.chain_19060\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35653876', '35654284', '100', '-', '.', 'gene_id \"chr34.g15241\"; transcript_id \"blat.proc28144.chain_22793\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25035340', '25035491', '100', '-', '.', 'gene_id \"chr34.g15259\"; transcript_id \"blat.proc28144.chain_21068\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24412380', '24412600', '100', '-', '.', 'gene_id \"chr34.g15283\"; transcript_id \"blat.proc4587.chain_7109\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24397543', '24397760', '100', '-', '.', 'gene_id \"chr34.g15288\"; transcript_id \"blat.proc28144.chain_22418\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26515157', '26515256', '100', '+', '.', 'gene_id \"chr34.g15292\"; transcript_id \"blat.proc28144.chain_20955\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '10363428', '10363465', '100', '-', '.', 'gene_id \"chr34.g15330\"; transcript_id \"blat.proc18128.chain_16660\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27134137', '27134421', '100', '+', '.', 'gene_id \"chr34.g15332\"; transcript_id \"blat.proc28144.chain_20490\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36449518', '36449667', '100', '-', '.', 'gene_id \"chr34.g15350\"; transcript_id \"blat.proc28144.chain_23290\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36122465', '36122660', '100', '+', '.', 'gene_id \"chr34.g15375\"; transcript_id \"blat.proc28144.chain_22855\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32149177', '32149213', '100', '+', '.', 'gene_id \"chr34.g15378\"; transcript_id \"blat.proc4005.chain_16793\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37800148', '37800329', '100', '+', '.', 'gene_id \"chr34.g15414\"; transcript_id \"blat.proc28144.chain_22696\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32544519', '32544690', '100', '-', '.', 'gene_id \"chr34.g15426\"; transcript_id \"blat.proc28144.chain_23674\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24895928', '24896265', '100', '+', '.', 'gene_id \"chr34.g15430\"; transcript_id \"blat.proc28144.chain_19404\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35582044', '35582167', '100', '-', '.', 'gene_id \"chr34.g15434\"; transcript_id \"blat.proc28144.chain_23880\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33173329', '33173543', '100', '-', '.', 'gene_id \"chr34.g15437\"; transcript_id \"blat.proc28144.chain_22960\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23943093', '23943241', '100', '-', '.', 'gene_id \"chr34.g15440\"; transcript_id \"blat.proc28144.chain_18876\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26134105', '26134256', '100', '-', '.', 'gene_id \"chr34.g15444\"; transcript_id \"blat.proc28144.chain_19261\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26548831', '26549240', '100', '-', '.', 'gene_id \"chr34.g15448\"; transcript_id \"blat.proc28144.chain_19420\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25925763', '25925929', '100', '-', '.', 'gene_id \"chr34.g15480\"; transcript_id \"blat.proc28144.chain_21857\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '41607631', '41607679', '100', '+', '.', 'gene_id \"chr34.g15495\"; transcript_id \"blat.proc28144.chain_4317\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25969412', '25969675', '100', '-', '.', 'gene_id \"chr34.g15498\"; transcript_id \"blat.proc28144.chain_19052\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36381447', '36381515', '100', '+', '.', 'gene_id \"chr34.g15505\"; transcript_id \"blat.proc28144.chain_23225\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25071001', '25071139', '100', '+', '.', 'gene_id \"chr34.g15526\"; transcript_id \"blat.proc28144.chain_21917\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25893658', '25893817', '100', '+', '.', 'gene_id \"chr34.g15536\"; transcript_id \"blat.proc28144.chain_21342\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24750625', '24750725', '100', '+', '.', 'gene_id \"chr34.g15539\"; transcript_id \"blat.proc28144.chain_20727\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '14020105', '14020165', '100', '+', '.', 'gene_id \"chr34.g15542\"; transcript_id \"blat.proc4005.chain_18064\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26455031', '26455238', '100', '-', '.', 'gene_id \"chr34.g15570\"; transcript_id \"blat.proc28144.chain_21910\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '12370314', '12370378', '100', '-', '.', 'gene_id \"chr34.g15585\"; transcript_id \"blat.proc18128.chain_15917\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38779954', '38780199', '100', '-', '.', 'gene_id \"chr34.g15587\"; transcript_id \"blat.proc28144.chain_22502\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34674376', '34674538', '100', '+', '.', 'gene_id \"chr34.g15625\"; transcript_id \"blat.proc28144.chain_22917\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36636827', '36636974', '100', '-', '.', 'gene_id \"chr34.g15628\"; transcript_id \"blat.proc28144.chain_23795\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26234717', '26235037', '100', '-', '.', 'gene_id \"chr34.g15642\"; transcript_id \"blat.proc28144.chain_19816\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24397268', '24397450', '100', '+', '.', 'gene_id \"chr34.g15648\"; transcript_id \"blat.proc28144.chain_22419\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26620130', '26620186', '100', '-', '.', 'gene_id \"chr34.g15658\"; transcript_id \"blat.proc28144.chain_21245\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '13469203', '13469243', '100', '+', '.', 'gene_id \"chr34.g15666\"; transcript_id \"blat.proc4005.chain_5083\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36358406', '36358570', '100', '+', '.', 'gene_id \"chr34.g15674\"; transcript_id \"blat.proc28144.chain_23013\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32911516', '32911706', '100', '-', '.', 'gene_id \"chr34.g15678\"; transcript_id \"blat.proc28144.chain_23065\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34165502', '34165662', '100', '-', '.', 'gene_id \"chr34.g15684\"; transcript_id \"blat.proc28144.chain_22649\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26142837', '26142993', '100', '-', '.', 'gene_id \"chr34.g15689\"; transcript_id \"blat.proc28144.chain_21974\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26518882', '26519063', '100', '+', '.', 'gene_id \"chr34.g15694\"; transcript_id \"blat.proc28144.chain_22087\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38096058', '38096155', '100', '+', '.', 'gene_id \"chr34.g15697\"; transcript_id \"blat.proc28144.chain_23947\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24740983', '24741114', '100', '-', '.', 'gene_id \"chr34.g15713\"; transcript_id \"blat.proc28144.chain_21797\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26526957', '26527062', '100', '-', '.', 'gene_id \"chr34.g15732\"; transcript_id \"blat.proc28144.chain_19592\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24499323', '24499452', '100', '-', '.', 'gene_id \"chr34.g15733\"; transcript_id \"blat.proc28144.chain_22456\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25799872', '25800040', '100', '+', '.', 'gene_id \"chr34.g15749\"; transcript_id \"blat.proc28144.chain_18928\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35376034', '35376079', '100', '-', '.', 'gene_id \"chr34.g15765\"; transcript_id \"blat.proc28144.chain_19876\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28828328', '28828499', '100', '-', '.', 'gene_id \"chr34.g15770\"; transcript_id \"blat.proc28144.chain_22270\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38301063', '38301093', '100', '+', '.', 'gene_id \"chr34.g15772\"; transcript_id \"blat.proc18128.chain_24244\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26612860', '26613042', '100', '-', '.', 'gene_id \"chr34.g15780\"; transcript_id \"blat.proc28144.chain_19909\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24460546', '24460702', '100', '-', '.', 'gene_id \"chr34.g15785\"; transcript_id \"blat.proc28144.chain_22426\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39413835', '39414024', '100', '+', '.', 'gene_id \"chr34.g15789\"; transcript_id \"blat.proc28144.chain_22949\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27068143', '27068414', '100', '-', '.', 'gene_id \"chr34.g15812\"; transcript_id \"blat.proc28144.chain_19235\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34049335', '34049494', '100', '+', '.', 'gene_id \"chr34.g15826\"; transcript_id \"blat.proc28144.chain_22871\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36014409', '36014588', '100', '-', '.', 'gene_id \"chr34.g15829\"; transcript_id \"blat.proc28144.chain_23194\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29141973', '29142088', '100', '+', '.', 'gene_id \"chr34.g15832\"; transcript_id \"blat.proc28144.chain_1993\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26528509', '26528744', '100', '+', '.', 'gene_id \"chr34.g15871\"; transcript_id \"blat.proc28144.chain_19184\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26797166', '26797331', '100', '+', '.', 'gene_id \"chr34.g15899\"; transcript_id \"blat.proc28144.chain_21948\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27203926', '27204153', '100', '+', '.', 'gene_id \"chr34.g15917\"; transcript_id \"blat.proc28144.chain_21952\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27246664', '27246849', '100', '-', '.', 'gene_id \"chr34.g15940\"; transcript_id \"blat.proc28144.chain_19051\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26338969', '26339083', '100', '+', '.', 'gene_id \"chr34.g15947\"; transcript_id \"blat.proc4587.chain_12250\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33617103', '33617508', '100', '-', '.', 'gene_id \"chr34.g15974\"; transcript_id \"blat.proc28144.chain_22580\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24460774', '24460967', '100', '+', '.', 'gene_id \"chr34.g15987\"; transcript_id \"blat.proc28144.chain_22421\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '14249632', '14249675', '100', '-', '.', 'gene_id \"chr34.g15988\"; transcript_id \"blat.proc28144.chain_5762\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34848833', '34849066', '100', '-', '.', 'gene_id \"chr34.g15996\"; transcript_id \"blat.proc28144.chain_22564\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '23922857', '23923099', '100', '+', '.', 'gene_id \"chr34.g16002\"; transcript_id \"blat.proc28144.chain_18816\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '16523204', '16523303', '100', '+', '.', 'gene_id \"chr34.g16003\"; transcript_id \"blat.proc28144.chain_11345\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36358737', '36359107', '100', '+', '.', 'gene_id \"chr34.g16013\"; transcript_id \"blat.proc28144.chain_23021\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26379128', '26379333', '100', '-', '.', 'gene_id \"chr34.g16024\"; transcript_id \"blat.proc28144.chain_19125\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24890863', '24891245', '100', '+', '.', 'gene_id \"chr34.g16036\"; transcript_id \"blat.proc28144.chain_19392\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25917822', '25918038', '100', '-', '.', 'gene_id \"chr34.g16037\"; transcript_id \"blat.proc28144.chain_22144\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '40424723', '40424752', '100', '+', '.', 'gene_id \"chr34.g16042\"; transcript_id \"blat.proc4005.chain_17089\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '37047396', '37047493', '100', '+', '.', 'gene_id \"chr34.g16047\"; transcript_id \"blat.proc28144.chain_23157\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34798288', '34798485', '100', '+', '.', 'gene_id \"chr34.g16065\"; transcript_id \"blat.proc28144.chain_23360\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25988371', '25988631', '100', '-', '.', 'gene_id \"chr34.g16085\"; transcript_id \"blat.proc28144.chain_19097\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36627250', '36627528', '100', '-', '.', 'gene_id \"chr34.g16086\"; transcript_id \"blat.proc28144.chain_22869\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '15472059', '15472295', '100', '+', '.', 'gene_id \"chr34.g16088\"; transcript_id \"blat.proc28144.chain_18801\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35342523', '35342791', '100', '+', '.', 'gene_id \"chr34.g16095\"; transcript_id \"blat.proc28144.chain_23362\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38779724', '38779891', '100', '+', '.', 'gene_id \"chr34.g16129\"; transcript_id \"blat.proc28144.chain_22501\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '5351391', '5351423', '100', '+', '.', 'gene_id \"chr34.g16135\"; transcript_id \"blat.proc4587.chain_4320\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25077139', '25077324', '100', '-', '.', 'gene_id \"chr34.g16175\"; transcript_id \"blat.proc28144.chain_19770\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '24488408', '24488568', '100', '-', '.', 'gene_id \"chr34.g16184\"; transcript_id \"blat.proc4587.chain_4762\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27208257', '27208473', '100', '+', '.', 'gene_id \"chr34.g16186\"; transcript_id \"blat.proc28144.chain_21938\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26855447', '26855629', '100', '-', '.', 'gene_id \"chr34.g16205\"; transcript_id \"blat.proc28144.chain_22109\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27104713', '27105072', '100', '+', '.', 'gene_id \"chr34.g16206\"; transcript_id \"blat.proc28144.chain_20078\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '32091265', '32091459', '100', '+', '.', 'gene_id \"chr34.g16230\"; transcript_id \"blat.proc28144.chain_23440\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26002482', '26002644', '100', '-', '.', 'gene_id \"chr34.g16242\"; transcript_id \"blat.proc28144.chain_21872\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38058292', '38058449', '100', '+', '.', 'gene_id \"chr34.g16244\"; transcript_id \"blat.proc28144.chain_23821\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34847392', '34847431', '100', '+', '.', 'gene_id \"chr34.g16253\"; transcript_id \"blat.proc28144.chain_11673\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30036962', '30037024', '100', '+', '.', 'gene_id \"chr34.g16257\"; transcript_id \"blat.proc18128.chain_13357\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28811494', '28811538', '100', '+', '.', 'gene_id \"chr34.g16258\"; transcript_id \"blat.proc4005.chain_7888\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33781113', '33781311', '100', '-', '.', 'gene_id \"chr34.g16277\"; transcript_id \"blat.proc28144.chain_23064\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28888865', '28889015', '100', '-', '.', 'gene_id \"chr34.g16279\"; transcript_id \"blat.proc28144.chain_22399\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25065515', '25065712', '100', '+', '.', 'gene_id \"chr34.g16284\"; transcript_id \"blat.proc28144.chain_21847\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '34660872', '34661091', '100', '+', '.', 'gene_id \"chr34.g16286\"; transcript_id \"blat.proc28144.chain_23634\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27075250', '27075410', '100', '-', '.', 'gene_id \"chr34.g16292\"; transcript_id \"blat.proc28144.chain_19655\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '36463174', '36463375', '100', '+', '.', 'gene_id \"chr34.g16304\"; transcript_id \"blat.proc28144.chain_22923\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26239084', '26239608', '100', '-', '.', 'gene_id \"chr34.g16352\"; transcript_id \"blat.proc28144.chain_19417\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26229235', '26229362', '100', '-', '.', 'gene_id \"chr34.g16361\"; transcript_id \"blat.proc28144.chain_22005\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '28829965', '28830236', '100', '-', '.', 'gene_id \"chr34.g16363\"; transcript_id \"blat.proc28144.chain_22220\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '33616852', '33617078', '100', '+', '.', 'gene_id \"chr34.g16392\"; transcript_id \"blat.proc28144.chain_22581\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '39469454', '39469628', '100', '+', '.', 'gene_id \"chr34.g16406\"; transcript_id \"blat.proc28144.chain_22821\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '25847299', '25847568', '100', '+', '.', 'gene_id \"chr34.g16410\"; transcript_id \"blat.proc28144.chain_18932\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26574015', '26574200', '100', '+', '.', 'gene_id \"chr34.g16411\"; transcript_id \"blat.proc28144.chain_22084\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '29136843', '29136888', '100', '-', '.', 'gene_id \"chr34.g16437\"; transcript_id \"blat.proc28144.chain_16905\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '30415410', '30415592', '100', '+', '.', 'gene_id \"chr34.g16464\"; transcript_id \"blat.proc28144.chain_22253\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '26785873', '26786057', '100', '+', '.', 'gene_id \"chr34.g16495\"; transcript_id \"blat.proc28144.chain_20104\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '38916437', '38916935', '100', '-', '.', 'gene_id \"chr34.g16513\"; transcript_id \"blat.proc28144.chain_23583\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '27037503', '27037715', '100', '-', '.', 'gene_id \"chr34.g16514\"; transcript_id \"blat.proc28144.chain_19372\";']\n",
      "negative:  ['chr34', 'PASA', 'exon', '35526891', '35527014', '100', '-', '.', 'gene_id \"chr34.g16534\"; transcript_id \"blat.proc28144.chain_22941\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '26870366', '26870440', '.', '-', '.', 'gene_id \"chr27.g11453\"; transcript_id \"chr27.g11453.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '27186498', '27186868', '.', '-', '.', 'gene_id \"chr27.g11590\"; transcript_id \"chr27.g11590.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '27674964', '27675046', '.', '+', '.', 'gene_id \"chr27.g11627\"; transcript_id \"chr27.g11627.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr13', 'PASA', 'exon', '647471', '647925', '.', '+', '.', 'gene_id \"chr13.g410\"; transcript_id \"chr13.g410.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28536140', '28536194', '.', '-', '.', 'gene_id \"chr27.g12091\"; transcript_id \"chr27.g12091.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1138593', '1138633', '.', '-', '.', 'gene_id \"chr13.g716\"; transcript_id \"chr13.g716.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28621395', '28621599', '.', '+', '.', 'gene_id \"chr27.g12111\"; transcript_id \"chr27.g12111.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28653664', '28654830', '.', '-', '.', 'gene_id \"chr27.g12126\"; transcript_id \"chr27.g12126.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1363752', '1363962', '.', '+', '.', 'gene_id \"chr13.g802\"; transcript_id \"chr13.g802.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28986425', '28986636', '.', '+', '.', 'gene_id \"chr27.g12198\"; transcript_id \"chr27.g12198.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '29144096', '29144239', '.', '-', '.', 'gene_id \"chr27.g12219\"; transcript_id \"chr27.g12219.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1479265', '1479483', '.', '-', '.', 'gene_id \"chr13.g834\"; transcript_id \"chr13.g834.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1479466', '1479619', '.', '+', '.', 'gene_id \"chr13.g835\"; transcript_id \"chr13.g835.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '30521160', '30521353', '.', '+', '.', 'gene_id \"chr27.g12345\"; transcript_id \"chr27.g12345.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1800394', '1800614', '.', '-', '.', 'gene_id \"chr13.g1052\"; transcript_id \"chr13.g1052.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1800683', '1800840', '.', '-', '.', 'gene_id \"chr13.g1053\"; transcript_id \"chr13.g1053.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '30915988', '30916358', '.', '+', '.', 'gene_id \"chr27.g12466\"; transcript_id \"chr27.g12466.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '30966071', '30966130', '.', '-', '.', 'gene_id \"chr27.g12515\"; transcript_id \"chr27.g12515.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '31128725', '31128905', '.', '-', '.', 'gene_id \"chr27.g12634\"; transcript_id \"chr27.g12634.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '31172064', '31172097', '.', '-', '.', 'gene_id \"chr27.g12663\"; transcript_id \"chr27.g12663.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '31198738', '31198773', '.', '+', '.', 'gene_id \"chr27.g12680\"; transcript_id \"chr27.g12680.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '2371822', '2372043', '.', '-', '.', 'gene_id \"chr13.g1507\"; transcript_id \"chr13.g1507.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '2372036', '2372189', '.', '+', '.', 'gene_id \"chr13.g1508\"; transcript_id \"chr13.g1508.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32322837', '32323201', '.', '-', '.', 'gene_id \"chr27.g13202\"; transcript_id \"chr27.g13202.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32387625', '32387857', '.', '-', '.', 'gene_id \"chr27.g13248\"; transcript_id \"chr27.g13248.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34165319', '34165363', '.', '+', '.', 'gene_id \"chr27.g13869\"; transcript_id \"chr27.g13869.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34599665', '34599857', '.', '+', '.', 'gene_id \"chr27.g14086\"; transcript_id \"chr27.g14086.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34665695', '34665976', '.', '+', '.', 'gene_id \"chr27.g14143\"; transcript_id \"chr27.g14143.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34683943', '34684234', '.', '-', '.', 'gene_id \"chr27.g14166\"; transcript_id \"chr27.g14166.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '5670010', '5670136', '.', '-', '.', 'gene_id \"chr13.g2938\"; transcript_id \"chr13.g2938.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '5763151', '5763321', '.', '-', '.', 'gene_id \"chr13.g3003\"; transcript_id \"chr13.g3003.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34974893', '34975173', '.', '-', '.', 'gene_id \"chr27.g14351\"; transcript_id \"chr27.g14351.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34977743', '34977903', '.', '+', '.', 'gene_id \"chr27.g14361\"; transcript_id \"chr27.g14361.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '35298237', '35298544', '.', '-', '.', 'gene_id \"chr27.g14452\"; transcript_id \"chr27.g14452.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '35298933', '35298968', '.', '-', '.', 'gene_id \"chr27.g14453\"; transcript_id \"chr27.g14453.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '35361247', '35361410', '.', '-', '.', 'gene_id \"chr27.g14475\"; transcript_id \"chr27.g14475.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '35476683', '35476814', '.', '-', '.', 'gene_id \"chr27.g14493\"; transcript_id \"chr27.g14493.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '35887978', '35888141', '.', '+', '.', 'gene_id \"chr27.g14597\"; transcript_id \"chr27.g14597.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36030652', '36030682', '.', '-', '.', 'gene_id \"chr27.g14660\"; transcript_id \"chr27.g14660.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36118052', '36119118', '.', '-', '.', 'gene_id \"chr27.g14681\"; transcript_id \"chr27.g14681.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '6710317', '6710600', '.', '+', '.', 'gene_id \"chr13.g3320\"; transcript_id \"chr13.g3320.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36298474', '36299252', '.', '+', '.', 'gene_id \"chr27.g14751\"; transcript_id \"chr27.g14751.i8\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36301385', '36301666', '.', '+', '.', 'gene_id \"chr27.g14751\"; transcript_id \"chr27.g14751.i8\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '7123050', '7123087', '.', '+', '.', 'gene_id \"chr13.g3383\"; transcript_id \"chr13.g3383.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36378095', '36378273', '.', '-', '.', 'gene_id \"chr27.g14760\"; transcript_id \"chr27.g14760.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36378095', '36378252', '.', '-', '.', 'gene_id \"chr27.g14760\"; transcript_id \"chr27.g14760.i2\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '7507185', '7507218', '.', '-', '.', 'gene_id \"chr13.g3508\"; transcript_id \"chr13.g3508.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36787240', '36787449', '.', '-', '.', 'gene_id \"chr27.g14896\"; transcript_id \"chr27.g14896.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '36903962', '36904145', '.', '+', '.', 'gene_id \"chr27.g14910\"; transcript_id \"chr27.g14910.i6\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '7531591', '7531741', '.', '+', '.', 'gene_id \"chr13.g3537\"; transcript_id \"chr13.g3537.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '7783560', '7784106', '.', '-', '.', 'gene_id \"chr13.g3606\"; transcript_id \"chr13.g3606.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '8093146', '8093618', '.', '-', '.', 'gene_id \"chr13.g3656\"; transcript_id \"chr13.g3656.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '37415732', '37415801', '.', '-', '.', 'gene_id \"chr27.g15247\"; transcript_id \"chr27.g15247.i2\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '9375879', '9376075', '.', '-', '.', 'gene_id \"chr13.g3960\"; transcript_id \"chr13.g3960.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '9698021', '9705368', '.', '-', '.', 'gene_id \"chr13.g4032\"; transcript_id \"chr13.g4032.i2\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '9840758', '9840966', '.', '-', '.', 'gene_id \"chr13.g4056\"; transcript_id \"chr13.g4056.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '10911243', '10911382', '.', '-', '.', 'gene_id \"chr13.g4323\"; transcript_id \"chr13.g4323.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '38454446', '38454648', '.', '+', '.', 'gene_id \"chr27.g15902\"; transcript_id \"chr27.g15902.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '11374172', '11374335', '.', '+', '.', 'gene_id \"chr13.g4349\"; transcript_id \"chr13.g4349.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '38468640', '38468690', '.', '-', '.', 'gene_id \"chr27.g15927\"; transcript_id \"chr27.g15927.i3\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '13462957', '13463251', '.', '+', '.', 'gene_id \"chr13.g4499\"; transcript_id \"chr13.g4499.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '39166386', '39166598', '.', '-', '.', 'gene_id \"chr27.g16467\"; transcript_id \"chr27.g16467.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '39166744', '39166941', '.', '+', '.', 'gene_id \"chr27.g16468\"; transcript_id \"chr27.g16468.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '39521524', '39521830', '.', '+', '.', 'gene_id \"chr27.g16556\"; transcript_id \"chr27.g16556.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '16130088', '16130530', '.', '+', '.', 'gene_id \"chr13.g4884\"; transcript_id \"chr13.g4884.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '39684593', '39684858', '.', '-', '.', 'gene_id \"chr27.g16599\"; transcript_id \"chr27.g16599.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '39709275', '39709337', '.', '-', '.', 'gene_id \"chr27.g16604\"; transcript_id \"chr27.g16604.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40091063', '40091179', '.', '+', '.', 'gene_id \"chr27.g16693\"; transcript_id \"chr27.g16693.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40200174', '40200216', '.', '-', '.', 'gene_id \"chr27.g16723\"; transcript_id \"chr27.g16723.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '17284073', '17284223', '.', '+', '.', 'gene_id \"chr13.g5148\"; transcript_id \"chr13.g5148.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40398887', '40398932', '.', '-', '.', 'gene_id \"chr27.g16843\"; transcript_id \"chr27.g16843.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40438720', '40439178', '.', '-', '.', 'gene_id \"chr27.g16875\"; transcript_id \"chr27.g16875.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40635366', '40635572', '.', '-', '.', 'gene_id \"chr27.g17027\"; transcript_id \"chr27.g17027.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40635602', '40635771', '.', '+', '.', 'gene_id \"chr27.g17028\"; transcript_id \"chr27.g17028.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '41380648', '41381012', '.', '-', '.', 'gene_id \"chr27.g17290\"; transcript_id \"chr27.g17290.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '41527593', '41527777', '.', '-', '.', 'gene_id \"chr27.g17343\"; transcript_id \"chr27.g17343.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '18369972', '18370194', '.', '-', '.', 'gene_id \"chr13.g5667\"; transcript_id \"chr13.g5667.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '41682407', '41682633', '.', '+', '.', 'gene_id \"chr27.g17468\"; transcript_id \"chr27.g17468.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '41735872', '41736083', '.', '+', '.', 'gene_id \"chr27.g17508\"; transcript_id \"chr27.g17508.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '19151816', '19151891', '.', '-', '.', 'gene_id \"chr13.g6112\"; transcript_id \"chr13.g6112.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42044057', '42044109', '.', '+', '.', 'gene_id \"chr27.g17835\"; transcript_id \"chr27.g17835.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42107337', '42107495', '.', '-', '.', 'gene_id \"chr27.g17903\"; transcript_id \"chr27.g17903.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42107613', '42107809', '.', '-', '.', 'gene_id \"chr27.g17904\"; transcript_id \"chr27.g17904.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42248405', '42249101', '.', '-', '.', 'gene_id \"chr27.g18035\"; transcript_id \"chr27.g18035.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42283040', '42283076', '.', '+', '.', 'gene_id \"chr27.g18069\"; transcript_id \"chr27.g18069.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42327478', '42327618', '.', '+', '.', 'gene_id \"chr27.g18100\"; transcript_id \"chr27.g18100.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42359868', '42359921', '.', '-', '.', 'gene_id \"chr27.g18140\"; transcript_id \"chr27.g18140.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42386016', '42386237', '.', '-', '.', 'gene_id \"chr27.g18191\"; transcript_id \"chr27.g18191.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42386335', '42386501', '.', '+', '.', 'gene_id \"chr27.g18192\"; transcript_id \"chr27.g18192.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42715140', '42715264', '.', '-', '.', 'gene_id \"chr27.g18358\"; transcript_id \"chr27.g18358.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '21343595', '21343875', '.', '-', '.', 'gene_id \"chr13.g6608\"; transcript_id \"chr13.g6608.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42751135', '42751371', '.', '+', '.', 'gene_id \"chr27.g18387\"; transcript_id \"chr27.g18387.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42751394', '42751652', '.', '-', '.', 'gene_id \"chr27.g18388\"; transcript_id \"chr27.g18388.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42777314', '42777530', '.', '+', '.', 'gene_id \"chr27.g18396\"; transcript_id \"chr27.g18396.i4\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '21421707', '21421903', '.', '-', '.', 'gene_id \"chr13.g6688\"; transcript_id \"chr13.g6688.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42951323', '42951412', '.', '+', '.', 'gene_id \"chr27.g18479\"; transcript_id \"chr27.g18479.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43047716', '43047794', '.', '-', '.', 'gene_id \"chr27.g18510\"; transcript_id \"chr27.g18510.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43301834', '43301870', '.', '-', '.', 'gene_id \"chr27.g18679\"; transcript_id \"chr27.g18679.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '21726938', '21726990', '.', '-', '.', 'gene_id \"chr13.g6925\"; transcript_id \"chr13.g6925.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43412799', '43412925', '.', '+', '.', 'gene_id \"chr27.g18738\"; transcript_id \"chr27.g18738.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43715686', '43716800', '.', '-', '.', 'gene_id \"chr27.g18938\"; transcript_id \"chr27.g18938.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43966879', '43967179', '.', '+', '.', 'gene_id \"chr27.g19112\"; transcript_id \"chr27.g19112.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '22515329', '22515610', '.', '+', '.', 'gene_id \"chr13.g7392\"; transcript_id \"chr13.g7392.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '44123050', '44123285', '.', '+', '.', 'gene_id \"chr27.g19165\"; transcript_id \"chr27.g19165.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '44088337', '44088380', '.', '+', '.', 'gene_id \"chr27.g19213\"; transcript_id \"chr27.g19213.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '44122769', '44122924', '.', '-', '.', 'gene_id \"chr27.g19244\"; transcript_id \"chr27.g19244.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '44320681', '44320754', '.', '+', '.', 'gene_id \"chr27.g19372\"; transcript_id \"chr27.g19372.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '44366001', '44367155', '.', '+', '.', 'gene_id \"chr27.g19413\"; transcript_id \"chr27.g19413.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '23311871', '23312038', '.', '+', '.', 'gene_id \"chr13.g8016\"; transcript_id \"chr13.g8016.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45401102', '45401570', '.', '-', '.', 'gene_id \"chr27.g19921\"; transcript_id \"chr27.g19921.i2\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45606683', '45606743', '.', '-', '.', 'gene_id \"chr27.g20077\"; transcript_id \"chr27.g20077.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45844693', '45844736', '.', '-', '.', 'gene_id \"chr27.g20272\"; transcript_id \"chr27.g20272.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45911350', '45911394', '.', '+', '.', 'gene_id \"chr27.g20314\"; transcript_id \"chr27.g20314.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45939923', '45939963', '.', '-', '.', 'gene_id \"chr27.g20320\"; transcript_id \"chr27.g20320.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '46002699', '46002750', '.', '+', '.', 'gene_id \"chr27.g20367\"; transcript_id \"chr27.g20367.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '46089147', '46089181', '.', '+', '.', 'gene_id \"chr27.g20429\"; transcript_id \"chr27.g20429.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '27039506', '27040062', '.', '+', '.', 'gene_id \"chr13.g8683\"; transcript_id \"chr13.g8683.i1\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '14976720', '14976879', '100', '-', '.', 'gene_id \"chr27.g20471\"; transcript_id \"blat.proc28144.chain_6684\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '29974960', '29975006', '100', '+', '.', 'gene_id \"chr27.g20551\"; transcript_id \"blat.proc4005.chain_2534\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '22482019', '22482063', '100', '+', '.', 'gene_id \"chr27.g20622\"; transcript_id \"blat.proc28144.chain_16899\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '1540554', '1540614', '100', '-', '.', 'gene_id \"chr27.g20667\"; transcript_id \"blat.proc28144.chain_20860\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '10073683', '10073719', '100', '-', '.', 'gene_id \"chr27.g20766\"; transcript_id \"blat.proc28144.chain_15258\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3245708', '3245800', '100', '+', '.', 'gene_id \"chr27.g20831\"; transcript_id \"blat.proc18128.chain_14258\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '21019487', '21019520', '100', '-', '.', 'gene_id \"chr27.g20842\"; transcript_id \"blat.proc18128.chain_9729\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr27', 'PASA', 'exon', '26769114', '26769152', '100', '+', '.', 'gene_id \"chr27.g21009\"; transcript_id \"blat.proc18128.chain_17471\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '4998544', '4998578', '100', '+', '.', 'gene_id \"chr27.g21063\"; transcript_id \"blat.proc18128.chain_18509\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '37452624', '37452655', '100', '+', '.', 'gene_id \"chr27.g21261\"; transcript_id \"blat.proc4005.chain_10188\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5285250', '5285282', '100', '-', '.', 'gene_id \"chr27.g21298\"; transcript_id \"blat.proc28144.chain_7743\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24546523', '24546567', '100', '+', '.', 'gene_id \"chr27.g21317\"; transcript_id \"blat.proc28144.chain_4642\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5990572', '5990721', '100', '+', '.', 'gene_id \"chr27.g21335\"; transcript_id \"blat.proc4587.chain_11372\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24041128', '24041171', '100', '+', '.', 'gene_id \"chr27.g21430\"; transcript_id \"blat.proc28144.chain_7669\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8786988', '8787062', '100', '+', '.', 'gene_id \"chr27.g21614\"; transcript_id \"blat.proc18128.chain_17151\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11307945', '11307984', '100', '+', '.', 'gene_id \"chr27.g21629\"; transcript_id \"blat.proc18128.chain_2338\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '25997156', '25997360', '100', '+', '.', 'gene_id \"chr27.g21643\"; transcript_id \"blat.proc4587.chain_4370\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17609571', '17609605', '100', '+', '.', 'gene_id \"chr27.g21729\"; transcript_id \"blat.proc4587.chain_4062\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7195841', '7195873', '100', '+', '.', 'gene_id \"chr27.g21804\"; transcript_id \"blat.proc18128.chain_3007\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '34173689', '34173755', '100', '+', '.', 'gene_id \"chr27.g21996\"; transcript_id \"blat.proc18128.chain_8763\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17162175', '17162206', '100', '-', '.', 'gene_id \"chr27.g22026\"; transcript_id \"blat.proc18128.chain_8977\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '13422559', '13422623', '100', '-', '.', 'gene_id \"chr27.g22043\"; transcript_id \"blat.proc18128.chain_22692\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '17425056', '17425495', '100', '-', '.', 'gene_id \"chr27.g22179\"; transcript_id \"blat.proc4587.chain_513\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8814333', '8814369', '100', '+', '.', 'gene_id \"chr27.g22198\"; transcript_id \"blat.proc4005.chain_3869\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '9323353', '9323411', '100', '+', '.', 'gene_id \"chr27.g22301\"; transcript_id \"blat.proc18128.chain_14150\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32525681', '32525722', '100', '+', '.', 'gene_id \"chr27.g22351\"; transcript_id \"blat.proc4587.chain_14102\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40771005', '40771447', '100', '-', '.', 'gene_id \"chr27.g22397\"; transcript_id \"blat.proc4587.chain_4890\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '6179027', '6179061', '100', '+', '.', 'gene_id \"chr27.g22420\"; transcript_id \"blat.proc18128.chain_15990\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32121881', '32121922', '100', '-', '.', 'gene_id \"chr27.g22422\"; transcript_id \"blat.proc4005.chain_11012\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '19561156', '19561215', '100', '-', '.', 'gene_id \"chr27.g22425\"; transcript_id \"blat.proc18128.chain_17067\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '16107183', '16107222', '100', '+', '.', 'gene_id \"chr27.g22580\"; transcript_id \"blat.proc18128.chain_25095\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11665111', '11665221', '100', '-', '.', 'gene_id \"chr27.g22587\"; transcript_id \"blat.proc18128.chain_19431\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28204238', '28204275', '100', '+', '.', 'gene_id \"chr27.g22598\"; transcript_id \"blat.proc4005.chain_19841\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '45230307', '45230340', '100', '+', '.', 'gene_id \"chr27.g22669\"; transcript_id \"blat.proc4005.chain_2844\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '43374992', '43375032', '100', '+', '.', 'gene_id \"chr27.g22801\"; transcript_id \"blat.proc4005.chain_15222\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '19560660', '19560692', '100', '+', '.', 'gene_id \"chr27.g22802\"; transcript_id \"blat.proc4587.chain_4945\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32822911', '32822969', '100', '+', '.', 'gene_id \"chr27.g22816\"; transcript_id \"blat.proc18128.chain_13335\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '42932702', '42932782', '100', '+', '.', 'gene_id \"chr27.g22958\"; transcript_id \"blat.proc4587.chain_11999\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3696537', '3696567', '100', '-', '.', 'gene_id \"chr27.g23232\"; transcript_id \"blat.proc18128.chain_11329\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32515617', '32515665', '100', '-', '.', 'gene_id \"chr27.g23233\"; transcript_id \"blat.proc28144.chain_10410\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '41538350', '41538544', '100', '-', '.', 'gene_id \"chr27.g23281\"; transcript_id \"blat.proc18128.chain_25577\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '28397774', '28397806', '100', '+', '.', 'gene_id \"chr27.g23284\"; transcript_id \"blat.proc28144.chain_21939\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '5230830', '5230872', '100', '-', '.', 'gene_id \"chr27.g23537\"; transcript_id \"blat.proc18128.chain_8958\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32899580', '32899634', '100', '+', '.', 'gene_id \"chr27.g23605\"; transcript_id \"blat.proc4587.chain_11944\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '9030418', '9030508', '100', '-', '.', 'gene_id \"chr27.g23992\"; transcript_id \"blat.proc4587.chain_5892\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '38942448', '38942508', '100', '-', '.', 'gene_id \"chr27.g24006\"; transcript_id \"blat.proc18128.chain_24922\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '10686165', '10686315', '100', '-', '.', 'gene_id \"chr27.g24099\"; transcript_id \"blat.proc18128.chain_8028\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '14119774', '14119809', '100', '-', '.', 'gene_id \"chr27.g24165\"; transcript_id \"blat.proc4587.chain_4715\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '40588819', '40588871', '100', '-', '.', 'gene_id \"chr27.g24242\"; transcript_id \"blat.proc18128.chain_10473\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3060890', '3061013', '100', '+', '.', 'gene_id \"chr27.g24360\"; transcript_id \"blat.proc18128.chain_12698\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24338340', '24338460', '100', '-', '.', 'gene_id \"chr27.g24371\"; transcript_id \"blat.proc18128.chain_20105\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '11878645', '11878687', '100', '-', '.', 'gene_id \"chr27.g24401\"; transcript_id \"blat.proc28144.chain_1490\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7091854', '7091898', '100', '-', '.', 'gene_id \"chr27.g24461\"; transcript_id \"blat.proc28144.chain_5996\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '9922270', '9922308', '100', '+', '.', 'gene_id \"chr27.g24483\"; transcript_id \"blat.proc28144.chain_19607\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '8223015', '8223078', '100', '-', '.', 'gene_id \"chr27.g24584\"; transcript_id \"blat.proc18128.chain_19842\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24636352', '24636403', '100', '-', '.', 'gene_id \"chr27.g24653\"; transcript_id \"blat.proc18128.chain_14967\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '7298751', '7298848', '100', '-', '.', 'gene_id \"chr27.g24826\"; transcript_id \"blat.proc4587.chain_2048\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '3977182', '3977226', '100', '+', '.', 'gene_id \"chr27.g24839\"; transcript_id \"blat.proc18128.chain_26613\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '27090845', '27090885', '100', '-', '.', 'gene_id \"chr27.g24936\"; transcript_id \"blat.proc4587.chain_3495\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '10029393', '10029472', '100', '+', '.', 'gene_id \"chr27.g24948\"; transcript_id \"blat.proc18128.chain_17615\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '37098315', '37098358', '100', '-', '.', 'gene_id \"chr27.g25117\"; transcript_id \"blat.proc18128.chain_10907\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '24710625', '24710719', '100', '+', '.', 'gene_id \"chr27.g25325\"; transcript_id \"blat.proc18128.chain_11706\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '32998846', '32998970', '100', '+', '.', 'gene_id \"chr27.g25475\"; transcript_id \"blat.proc28144.chain_723\";']\n",
      "negative:  ['chr27', 'PASA', 'exon', '15564416', '15564448', '100', '+', '.', 'gene_id \"chr27.g25570\"; transcript_id \"blat.proc18128.chain_18413\";']\n",
      "negative:  ['scaffold_1680', 'PASA', 'exon', '6045', '6090', '100', '+', '.', 'gene_id \"scaffold_1680.g7\"; transcript_id \"blat.proc4005.chain_23413\";']\n",
      "negative:  ['scaffold_1232', 'PASA', 'exon', '10886', '11079', '.', '-', '.', 'gene_id \"scaffold_1232.g7\"; transcript_id \"scaffold_1232.g7.i1\";']\n",
      "negative:  ['scaffold_1232', 'PASA', 'exon', '15741', '15947', '.', '-', '.', 'gene_id \"scaffold_1232.g7\"; transcript_id \"scaffold_1232.g7.i4\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '27300528', '27300829', '.', '-', '.', 'gene_id \"chr13.g8961\"; transcript_id \"chr13.g8961.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '27399220', '27399393', '.', '+', '.', 'gene_id \"chr13.g9030\"; transcript_id \"chr13.g9030.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '27399408', '27399630', '.', '-', '.', 'gene_id \"chr13.g9031\"; transcript_id \"chr13.g9031.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '27478833', '27479075', '.', '+', '.', 'gene_id \"chr13.g9089\"; transcript_id \"chr13.g9089.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '29810617', '29811175', '.', '+', '.', 'gene_id \"chr13.g9710\"; transcript_id \"chr13.g9710.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '29811182', '29811375', '.', '+', '.', 'gene_id \"chr13.g9711\"; transcript_id \"chr13.g9711.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '225500', '225547', '.', '-', '.', 'gene_id \"chr31.g129\"; transcript_id \"chr31.g129.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '234365', '234404', '.', '+', '.', 'gene_id \"chr31.g138\"; transcript_id \"chr31.g138.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '274897', '274939', '.', '+', '.', 'gene_id \"chr31.g150\"; transcript_id \"chr31.g150.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '318318', '318364', '.', '-', '.', 'gene_id \"chr31.g160\"; transcript_id \"chr31.g160.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '318318', '318363', '.', '-', '.', 'gene_id \"chr31.g160\"; transcript_id \"chr31.g160.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '746708', '746896', '.', '-', '.', 'gene_id \"chr31.g266\"; transcript_id \"chr31.g266.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '2454607', '2454648', '.', '+', '.', 'gene_id \"chr31.g501\"; transcript_id \"chr31.g501.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '2824713', '2825084', '.', '-', '.', 'gene_id \"chr31.g577\"; transcript_id \"chr31.g577.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '2871354', '2871417', '.', '-', '.', 'gene_id \"chr31.g593\"; transcript_id \"chr31.g593.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '4550226', '4550322', '.', '+', '.', 'gene_id \"chr31.g738\"; transcript_id \"chr31.g738.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5111752', '5111807', '.', '+', '.', 'gene_id \"chr31.g771\"; transcript_id \"chr31.g771.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5138442', '5138489', '.', '+', '.', 'gene_id \"chr31.g776\"; transcript_id \"chr31.g776.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5308193', '5308230', '.', '-', '.', 'gene_id \"chr31.g793\"; transcript_id \"chr31.g793.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5704729', '5704776', '.', '+', '.', 'gene_id \"chr31.g811\"; transcript_id \"chr31.g811.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5737214', '5737561', '.', '-', '.', 'gene_id \"chr31.g832\"; transcript_id \"chr31.g832.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '33453123', '33453168', '.', '-', '.', 'gene_id \"chr13.g10502\"; transcript_id \"chr13.g10502.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5799567', '5799931', '.', '-', '.', 'gene_id \"chr31.g857\"; transcript_id \"chr31.g857.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '6519550', '6519581', '.', '+', '.', 'gene_id \"chr31.g1068\"; transcript_id \"chr31.g1068.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '7606011', '7606167', '.', '+', '.', 'gene_id \"chr31.g1161\"; transcript_id \"chr31.g1161.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '7656451', '7656798', '.', '+', '.', 'gene_id \"chr31.g1208\"; transcript_id \"chr31.g1208.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '7656805', '7657011', '.', '+', '.', 'gene_id \"chr31.g1209\"; transcript_id \"chr31.g1209.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '35463131', '35463294', '.', '+', '.', 'gene_id \"chr13.g10922\"; transcript_id \"chr13.g10922.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '35646550', '35646708', '.', '+', '.', 'gene_id \"chr13.g11049\"; transcript_id \"chr13.g11049.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '35770934', '35771084', '.', '-', '.', 'gene_id \"chr13.g11135\"; transcript_id \"chr13.g11135.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '8656727', '8656773', '.', '-', '.', 'gene_id \"chr31.g1478\"; transcript_id \"chr31.g1478.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '9143064', '9143100', '.', '+', '.', 'gene_id \"chr31.g1510\"; transcript_id \"chr31.g1510.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '36633476', '36633810', '.', '+', '.', 'gene_id \"chr13.g11421\"; transcript_id \"chr13.g11421.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '11040481', '11040535', '.', '+', '.', 'gene_id \"chr31.g1808\"; transcript_id \"chr31.g1808.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '11397485', '11397536', '.', '+', '.', 'gene_id \"chr31.g1906\"; transcript_id \"chr31.g1906.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37108554', '37108765', '.', '-', '.', 'gene_id \"chr13.g11645\"; transcript_id \"chr13.g11645.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37108778', '37108955', '.', '+', '.', 'gene_id \"chr13.g11646\"; transcript_id \"chr13.g11646.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37123020', '37123175', '.', '+', '.', 'gene_id \"chr13.g11660\"; transcript_id \"chr13.g11660.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37123273', '37123498', '.', '+', '.', 'gene_id \"chr13.g11661\"; transcript_id \"chr13.g11661.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '11858858', '11858913', '.', '-', '.', 'gene_id \"chr31.g1988\"; transcript_id \"chr31.g1988.i2\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37283387', '37283542', '.', '-', '.', 'gene_id \"chr13.g11787\"; transcript_id \"chr13.g11787.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37284012', '37284328', '.', '-', '.', 'gene_id \"chr13.g11788\"; transcript_id \"chr13.g11788.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '12617170', '12617346', '.', '+', '.', 'gene_id \"chr31.g2190\"; transcript_id \"chr31.g2190.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37591997', '37592041', '.', '+', '.', 'gene_id \"chr13.g11967\"; transcript_id \"chr13.g11967.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '12914560', '12914643', '.', '-', '.', 'gene_id \"chr31.g2295\"; transcript_id \"chr31.g2295.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37931974', '37932226', '.', '-', '.', 'gene_id \"chr13.g12253\"; transcript_id \"chr13.g12253.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '37946286', '37946326', '.', '-', '.', 'gene_id \"chr13.g12271\"; transcript_id \"chr13.g12271.i19\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '13384513', '13384668', '.', '+', '.', 'gene_id \"chr31.g2566\"; transcript_id \"chr31.g2566.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '13384770', '13384976', '.', '-', '.', 'gene_id \"chr31.g2568\"; transcript_id \"chr31.g2568.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '38163466', '38163622', '.', '+', '.', 'gene_id \"chr13.g12403\"; transcript_id \"chr13.g12403.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '13445399', '13445428', '.', '+', '.', 'gene_id \"chr31.g2583\"; transcript_id \"chr31.g2583.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '13966472', '13966633', '.', '+', '.', 'gene_id \"chr31.g2703\"; transcript_id \"chr31.g2703.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '38385571', '38385977', '.', '-', '.', 'gene_id \"chr13.g12597\"; transcript_id \"chr13.g12597.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '14116885', '14117048', '.', '-', '.', 'gene_id \"chr31.g2796\"; transcript_id \"chr31.g2796.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '15062036', '15062104', '.', '-', '.', 'gene_id \"chr31.g3045\"; transcript_id \"chr31.g3045.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '15267185', '15267634', '.', '-', '.', 'gene_id \"chr31.g3061\"; transcript_id \"chr31.g3061.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '17197647', '17197975', '.', '-', '.', 'gene_id \"chr31.g3172\"; transcript_id \"chr31.g3172.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '18584247', '18584339', '.', '-', '.', 'gene_id \"chr31.g3270\"; transcript_id \"chr31.g3270.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '18602758', '18602794', '.', '+', '.', 'gene_id \"chr31.g3272\"; transcript_id \"chr31.g3272.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '18786619', '18786649', '.', '+', '.', 'gene_id \"chr31.g3287\"; transcript_id \"chr31.g3287.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '19303125', '19303167', '.', '+', '.', 'gene_id \"chr31.g3319\"; transcript_id \"chr31.g3319.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '20135909', '20135954', '.', '-', '.', 'gene_id \"chr31.g3388\"; transcript_id \"chr31.g3388.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21264600', '21264645', '.', '+', '.', 'gene_id \"chr31.g3514\"; transcript_id \"chr31.g3514.i3\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21255941', '21256356', '.', '-', '.', 'gene_id \"chr31.g3584\"; transcript_id \"chr31.g3584.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21514340', '21514667', '.', '+', '.', 'gene_id \"chr31.g3759\"; transcript_id \"chr31.g3759.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '42477530', '42477724', '.', '-', '.', 'gene_id \"chr13.g13725\"; transcript_id \"chr13.g13725.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21631014', '21631198', '.', '-', '.', 'gene_id \"chr31.g3852\"; transcript_id \"chr31.g3852.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '42874936', '42874970', '.', '-', '.', 'gene_id \"chr13.g13803\"; transcript_id \"chr13.g13803.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21916314', '21916346', '.', '+', '.', 'gene_id \"chr31.g3946\"; transcript_id \"chr31.g3946.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21967902', '21968232', '.', '-', '.', 'gene_id \"chr31.g3968\"; transcript_id \"chr31.g3968.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '43455487', '43456024', '.', '+', '.', 'gene_id \"chr13.g13934\"; transcript_id \"chr13.g13934.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '22124342', '22124637', '.', '-', '.', 'gene_id \"chr31.g4080\"; transcript_id \"chr31.g4080.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '22175445', '22175597', '.', '-', '.', 'gene_id \"chr31.g4093\"; transcript_id \"chr31.g4093.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '22860362', '22860409', '.', '-', '.', 'gene_id \"chr31.g4207\"; transcript_id \"chr31.g4207.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24344661', '24344886', '.', '+', '.', 'gene_id \"chr31.g4470\"; transcript_id \"chr31.g4470.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24367195', '24367691', '.', '+', '.', 'gene_id \"chr31.g4486\"; transcript_id \"chr31.g4486.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24598054', '24598086', '.', '-', '.', 'gene_id \"chr31.g4532\"; transcript_id \"chr31.g4532.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24718057', '24718092', '.', '+', '.', 'gene_id \"chr31.g4543\"; transcript_id \"chr31.g4543.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24831646', '24831718', '.', '-', '.', 'gene_id \"chr31.g4549\"; transcript_id \"chr31.g4549.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '24850075', '24850115', '.', '+', '.', 'gene_id \"chr31.g4550\"; transcript_id \"chr31.g4550.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25766470', '25766603', '.', '-', '.', 'gene_id \"chr31.g4697\"; transcript_id \"chr31.g4697.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25766476', '25766603', '.', '-', '.', 'gene_id \"chr31.g4697\"; transcript_id \"chr31.g4697.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25914377', '25914555', '.', '+', '.', 'gene_id \"chr31.g4728\"; transcript_id \"chr31.g4728.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25914324', '25914555', '.', '-', '.', 'gene_id \"chr31.g4729\"; transcript_id \"chr31.g4729.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '45067966', '45068029', '.', '-', '.', 'gene_id \"chr13.g14662\"; transcript_id \"chr13.g14662.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '26253776', '26253835', '.', '+', '.', 'gene_id \"chr31.g4832\"; transcript_id \"chr31.g4832.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '26298064', '26298201', '.', '-', '.', 'gene_id \"chr31.g4850\"; transcript_id \"chr31.g4850.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '26920780', '26920959', '.', '+', '.', 'gene_id \"chr31.g5119\"; transcript_id \"chr31.g5119.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '26920974', '26921652', '.', '-', '.', 'gene_id \"chr31.g5120\"; transcript_id \"chr31.g5120.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '45977292', '45977458', '.', '+', '.', 'gene_id \"chr13.g15084\"; transcript_id \"chr13.g15084.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27288440', '27288888', '.', '+', '.', 'gene_id \"chr31.g5253\"; transcript_id \"chr31.g5253.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27296083', '27296282', '.', '+', '.', 'gene_id \"chr31.g5263\"; transcript_id \"chr31.g5263.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27399429', '27399460', '.', '+', '.', 'gene_id \"chr31.g5344\"; transcript_id \"chr31.g5344.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27455425', '27455702', '.', '+', '.', 'gene_id \"chr31.g5402\"; transcript_id \"chr31.g5402.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27455915', '27456081', '.', '-', '.', 'gene_id \"chr31.g5404\"; transcript_id \"chr31.g5404.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27629632', '27629925', '.', '+', '.', 'gene_id \"chr31.g5492\"; transcript_id \"chr31.g5492.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr31', 'PASA', 'exon', '27971106', '27972396', '.', '-', '.', 'gene_id \"chr31.g5562\"; transcript_id \"chr31.g5562.i3\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27969756', '27970033', '.', '+', '.', 'gene_id \"chr31.g5571\"; transcript_id \"chr31.g5571.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27994188', '27994442', '.', '+', '.', 'gene_id \"chr31.g5596\"; transcript_id \"chr31.g5596.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27994784', '27995276', '.', '+', '.', 'gene_id \"chr31.g5597\"; transcript_id \"chr31.g5597.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28315278', '28315547', '.', '-', '.', 'gene_id \"chr31.g5901\"; transcript_id \"chr31.g5901.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28463902', '28464374', '.', '+', '.', 'gene_id \"chr31.g5912\"; transcript_id \"chr31.g5912.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28331669', '28331844', '.', '+', '.', 'gene_id \"chr31.g5922\"; transcript_id \"chr31.g5922.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28332105', '28332369', '.', '-', '.', 'gene_id \"chr31.g5923\"; transcript_id \"chr31.g5923.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28447703', '28448045', '.', '+', '.', 'gene_id \"chr31.g5969\"; transcript_id \"chr31.g5969.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28466545', '28466784', '.', '+', '.', 'gene_id \"chr31.g5990\"; transcript_id \"chr31.g5990.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28467131', '28467410', '.', '+', '.', 'gene_id \"chr31.g5992\"; transcript_id \"chr31.g5992.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28467410', '28467630', '.', '-', '.', 'gene_id \"chr31.g5993\"; transcript_id \"chr31.g5993.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28478172', '28478444', '.', '+', '.', 'gene_id \"chr31.g6005\"; transcript_id \"chr31.g6005.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28480438', '28480671', '.', '-', '.', 'gene_id \"chr31.g6010\"; transcript_id \"chr31.g6010.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28753247', '28755334', '.', '+', '.', 'gene_id \"chr31.g6142\"; transcript_id \"chr31.g6142.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '48834792', '48835055', '.', '+', '.', 'gene_id \"chr13.g16059\"; transcript_id \"chr13.g16059.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '48835178', '48835328', '.', '+', '.', 'gene_id \"chr13.g16060\"; transcript_id \"chr13.g16060.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28795438', '28795653', '.', '+', '.', 'gene_id \"chr31.g6191\"; transcript_id \"chr31.g6191.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '48937392', '48937660', '.', '+', '.', 'gene_id \"chr13.g16148\"; transcript_id \"chr13.g16148.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '49000877', '49000911', '.', '-', '.', 'gene_id \"chr13.g16208\"; transcript_id \"chr13.g16208.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '29084353', '29084703', '.', '-', '.', 'gene_id \"chr31.g6323\"; transcript_id \"chr31.g6323.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '49115001', '49115276', '.', '-', '.', 'gene_id \"chr13.g16243\"; transcript_id \"chr13.g16243.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '29118784', '29119010', '.', '+', '.', 'gene_id \"chr31.g6356\"; transcript_id \"chr31.g6356.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '49275023', '49275063', '.', '+', '.', 'gene_id \"chr13.g16308\"; transcript_id \"chr13.g16308.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '29365342', '29365540', '.', '-', '.', 'gene_id \"chr31.g6495\"; transcript_id \"chr31.g6495.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '49488109', '49488398', '.', '-', '.', 'gene_id \"chr13.g16503\"; transcript_id \"chr13.g16503.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '29576645', '29576770', '.', '-', '.', 'gene_id \"chr31.g6648\"; transcript_id \"chr31.g6648.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '29642561', '29642620', '.', '+', '.', 'gene_id \"chr31.g6657\"; transcript_id \"chr31.g6657.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '30076995', '30077066', '.', '+', '.', 'gene_id \"chr31.g6676\"; transcript_id \"chr31.g6676.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '30234602', '30234826', '.', '-', '.', 'gene_id \"chr31.g6691\"; transcript_id \"chr31.g6691.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '30589407', '30589583', '.', '+', '.', 'gene_id \"chr31.g6860\"; transcript_id \"chr31.g6860.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '53398861', '53398891', '.', '+', '.', 'gene_id \"chr13.g16858\"; transcript_id \"chr13.g16858.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '30737463', '30737909', '.', '+', '.', 'gene_id \"chr31.g6961\"; transcript_id \"chr31.g6961.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '53597711', '53597959', '.', '+', '.', 'gene_id \"chr13.g16912\"; transcript_id \"chr13.g16912.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '54328460', '54328505', '.', '+', '.', 'gene_id \"chr13.g17045\"; transcript_id \"chr13.g17045.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31427416', '31427617', '.', '+', '.', 'gene_id \"chr31.g7223\"; transcript_id \"chr31.g7223.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31425019', '31426292', '.', '-', '.', 'gene_id \"chr31.g7230\"; transcript_id \"chr31.g7230.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31501528', '31502462', '.', '-', '.', 'gene_id \"chr31.g7233\"; transcript_id \"chr31.g7233.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '55579147', '55579184', '.', '-', '.', 'gene_id \"chr13.g17131\"; transcript_id \"chr13.g17131.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '55581193', '55581778', '.', '-', '.', 'gene_id \"chr13.g17137\"; transcript_id \"chr13.g17137.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31403303', '31403437', '.', '+', '.', 'gene_id \"chr31.g7258\"; transcript_id \"chr31.g7258.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31523854', '31524154', '.', '+', '.', 'gene_id \"chr31.g7367\"; transcript_id \"chr31.g7367.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31554309', '31554349', '.', '-', '.', 'gene_id \"chr31.g7404\"; transcript_id \"chr31.g7404.i5\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '56757536', '56757784', '.', '-', '.', 'gene_id \"chr13.g17284\"; transcript_id \"chr13.g17284.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '56757837', '56758142', '.', '+', '.', 'gene_id \"chr13.g17285\"; transcript_id \"chr13.g17285.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '57731509', '57732010', '.', '+', '.', 'gene_id \"chr13.g17339\"; transcript_id \"chr13.g17339.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31890642', '31890674', '.', '+', '.', 'gene_id \"chr31.g7677\"; transcript_id \"chr31.g7677.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '32916291', '32916324', '.', '+', '.', 'gene_id \"chr31.g7967\"; transcript_id \"chr31.g7967.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '32996980', '32997015', '.', '-', '.', 'gene_id \"chr31.g7984\"; transcript_id \"chr31.g7984.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33360437', '33360637', '.', '-', '.', 'gene_id \"chr31.g8096\"; transcript_id \"chr31.g8096.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33401508', '33401579', '.', '-', '.', 'gene_id \"chr31.g8132\"; transcript_id \"chr31.g8132.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33464866', '33465100', '.', '-', '.', 'gene_id \"chr31.g8207\"; transcript_id \"chr31.g8207.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33493959', '33494223', '.', '-', '.', 'gene_id \"chr31.g8240\"; transcript_id \"chr31.g8240.i9\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33548300', '33548530', '.', '-', '.', 'gene_id \"chr31.g8291\"; transcript_id \"chr31.g8291.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33766821', '33766974', '.', '+', '.', 'gene_id \"chr31.g8389\"; transcript_id \"chr31.g8389.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33767174', '33767349', '.', '-', '.', 'gene_id \"chr31.g8390\"; transcript_id \"chr31.g8390.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '34004817', '34004849', '.', '-', '.', 'gene_id \"chr31.g8446\"; transcript_id \"chr31.g8446.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '34055618', '34055917', '.', '+', '.', 'gene_id \"chr31.g8455\"; transcript_id \"chr31.g8455.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '34056019', '34056276', '.', '+', '.', 'gene_id \"chr31.g8457\"; transcript_id \"chr31.g8457.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '34307048', '34307201', '.', '-', '.', 'gene_id \"chr31.g8498\"; transcript_id \"chr31.g8498.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35358762', '35358865', '.', '-', '.', 'gene_id \"chr31.g8761\"; transcript_id \"chr31.g8761.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35358798', '35358878', '.', '-', '.', 'gene_id \"chr31.g8761\"; transcript_id \"chr31.g8761.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35373933', '35374148', '.', '+', '.', 'gene_id \"chr31.g8772\"; transcript_id \"chr31.g8772.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35417231', '35417277', '.', '+', '.', 'gene_id \"chr31.g8782\"; transcript_id \"chr31.g8782.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35712483', '35712765', '.', '+', '.', 'gene_id \"chr31.g8890\"; transcript_id \"chr31.g8890.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '35799032', '35799209', '.', '+', '.', 'gene_id \"chr31.g8959\"; transcript_id \"chr31.g8959.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36018560', '36018932', '.', '+', '.', 'gene_id \"chr31.g9033\"; transcript_id \"chr31.g9033.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36018943', '36019115', '.', '-', '.', 'gene_id \"chr31.g9034\"; transcript_id \"chr31.g9034.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36196689', '36196720', '.', '-', '.', 'gene_id \"chr31.g9091\"; transcript_id \"chr31.g9091.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36394588', '36394620', '.', '-', '.', 'gene_id \"chr31.g9223\"; transcript_id \"chr31.g9223.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36476375', '36476434', '.', '+', '.', 'gene_id \"chr31.g9284\"; transcript_id \"chr31.g9284.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36547580', '36547797', '.', '-', '.', 'gene_id \"chr31.g9347\"; transcript_id \"chr31.g9347.i1\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '38163704', '38163899', '100', '-', '.', 'gene_id \"chr13.g19956\"; transcript_id \"blat.proc4587.chain_7893\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '12509231', '12509278', '100', '-', '.', 'gene_id \"chr13.g20483\"; transcript_id \"blat.proc28144.chain_12409\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '26000767', '26000819', '100', '+', '.', 'gene_id \"chr13.g20511\"; transcript_id \"blat.proc4587.chain_12364\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '53010325', '53010367', '100', '+', '.', 'gene_id \"chr13.g21046\"; transcript_id \"blat.proc4587.chain_12615\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1914308', '1914412', '100', '-', '.', 'gene_id \"chr13.g21645\"; transcript_id \"blat.proc4587.chain_4064\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '22933969', '22934004', '100', '+', '.', 'gene_id \"chr13.g21807\"; transcript_id \"blat.proc4587.chain_12328\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '41378515', '41378673', '100', '-', '.', 'gene_id \"chr13.g22305\"; transcript_id \"blat.proc28144.chain_10904\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '15806124', '15806616', '100', '+', '.', 'gene_id \"chr13.g22374\"; transcript_id \"blat.proc28144.chain_2818\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '28376961', '28377009', '100', '+', '.', 'gene_id \"chr13.g22760\"; transcript_id \"blat.proc28144.chain_13901\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '52941558', '52941809', '100', '+', '.', 'gene_id \"chr13.g22967\"; transcript_id \"blat.proc28144.chain_11565\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '9186483', '9186527', '100', '-', '.', 'gene_id \"chr13.g22980\"; transcript_id \"blat.proc18128.chain_26892\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '43036317', '43036369', '100', '-', '.', 'gene_id \"chr13.g22984\"; transcript_id \"blat.proc18128.chain_21154\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '62196732', '62196783', '100', '+', '.', 'gene_id \"chr13.g23014\"; transcript_id \"blat.proc4587.chain_7647\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '1014305', '1014362', '100', '+', '.', 'gene_id \"chr13.g23079\"; transcript_id \"blat.proc18128.chain_12163\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '4136370', '4136412', '100', '-', '.', 'gene_id \"chr13.g23607\"; transcript_id \"blat.proc18128.chain_13690\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '9808627', '9808677', '100', '-', '.', 'gene_id \"chr13.g24075\"; transcript_id \"blat.proc4587.chain_8920\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '52854837', '52854890', '100', '-', '.', 'gene_id \"chr13.g24165\"; transcript_id \"blat.proc28144.chain_16826\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '13710899', '13710942', '100', '+', '.', 'gene_id \"chr13.g24542\"; transcript_id \"blat.proc4587.chain_739\";']\n",
      "negative:  ['chr13', 'PASA', 'exon', '34126825', '34126865', '100', '+', '.', 'gene_id \"chr13.g24653\"; transcript_id \"blat.proc18128.chain_16829\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36725539', '36726508', '.', '-', '.', 'gene_id \"chr31.g9468\"; transcript_id \"chr31.g9468.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37048960', '37050115', '.', '+', '.', 'gene_id \"chr31.g9663\"; transcript_id \"chr31.g9663.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37145509', '37145697', '.', '+', '.', 'gene_id \"chr31.g9770\"; transcript_id \"chr31.g9770.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37145789', '37146146', '.', '+', '.', 'gene_id \"chr31.g9774\"; transcript_id \"chr31.g9774.i1\";']\n",
      "negative:  ['scaffold_1622', 'PASA', 'exon', '6148', '6264', '.', '-', '.', 'gene_id \"scaffold_1622.g3\"; transcript_id \"scaffold_1622.g3.i1\";']\n",
      "negative:  ['scaffold_1622', 'PASA', 'exon', '6253', '6475', '.', '-', '.', 'gene_id \"scaffold_1622.g3\"; transcript_id \"scaffold_1622.g3.i2\";']\n",
      "negative:  ['scaffold_1622', 'PASA', 'exon', '6531', '6798', '100', '-', '.', 'gene_id \"scaffold_1622.g38\"; transcript_id \"blat.proc18128.chain_13224\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37723408', '37723603', '.', '+', '.', 'gene_id \"chr31.g10139\"; transcript_id \"chr31.g10139.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37731928', '37732123', '.', '+', '.', 'gene_id \"chr31.g10148\"; transcript_id \"chr31.g10148.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37798704', '37798769', '.', '+', '.', 'gene_id \"chr31.g10187\"; transcript_id \"chr31.g10187.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37847119', '37847314', '.', '-', '.', 'gene_id \"chr31.g10211\"; transcript_id \"chr31.g10211.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '163590', '163636', '.', '+', '.', 'gene_id \"chr15.g29\"; transcript_id \"chr15.g29.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '222938', '223090', '.', '-', '.', 'gene_id \"chr15.g47\"; transcript_id \"chr15.g47.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38048975', '38049217', '.', '-', '.', 'gene_id \"chr31.g10372\"; transcript_id \"chr31.g10372.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38049200', '38049425', '.', '+', '.', 'gene_id \"chr31.g10373\"; transcript_id \"chr31.g10373.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '508604', '508794', '.', '-', '.', 'gene_id \"chr15.g141\"; transcript_id \"chr15.g141.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '509210', '509362', '.', '-', '.', 'gene_id \"chr15.g143\"; transcript_id \"chr15.g143.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '751294', '751418', '.', '-', '.', 'gene_id \"chr15.g318\"; transcript_id \"chr15.g318.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '869288', '869795', '.', '-', '.', 'gene_id \"chr15.g382\"; transcript_id \"chr15.g382.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38477389', '38477541', '.', '-', '.', 'gene_id \"chr31.g10646\"; transcript_id \"chr31.g10646.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38477536', '38477774', '.', '+', '.', 'gene_id \"chr31.g10647\"; transcript_id \"chr31.g10647.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38712170', '38712271', '.', '+', '.', 'gene_id \"chr31.g10685\"; transcript_id \"chr31.g10685.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '39009665', '39009847', '.', '-', '.', 'gene_id \"chr31.g10793\"; transcript_id \"chr31.g10793.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '1208228', '1208393', '.', '-', '.', 'gene_id \"chr15.g559\"; transcript_id \"chr15.g559.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '39078205', '39078279', '.', '-', '.', 'gene_id \"chr31.g10844\"; transcript_id \"chr31.g10844.i2\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '39320745', '39320908', '.', '-', '.', 'gene_id \"chr31.g11069\"; transcript_id \"chr31.g11069.i1\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '18395979', '18396021', '100', '+', '.', 'gene_id \"chr31.g11416\"; transcript_id \"blat.proc18128.chain_23856\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '11275526', '11275562', '100', '-', '.', 'gene_id \"chr31.g11461\"; transcript_id \"blat.proc28144.chain_16182\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25046552', '25046607', '100', '+', '.', 'gene_id \"chr31.g11511\"; transcript_id \"blat.proc18128.chain_22957\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '36475340', '36475460', '100', '+', '.', 'gene_id \"chr31.g11528\"; transcript_id \"blat.proc28144.chain_21502\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '32007207', '32007290', '100', '+', '.', 'gene_id \"chr31.g11555\"; transcript_id \"blat.proc28144.chain_17354\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '14139090', '14139122', '100', '+', '.', 'gene_id \"chr31.g11715\"; transcript_id \"blat.proc28144.chain_2322\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '22016651', '22016724', '100', '+', '.', 'gene_id \"chr31.g11732\"; transcript_id \"blat.proc4587.chain_10943\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '19589833', '19589868', '100', '-', '.', 'gene_id \"chr31.g11745\"; transcript_id \"blat.proc18128.chain_11822\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '7300622', '7300656', '100', '-', '.', 'gene_id \"chr31.g11746\"; transcript_id \"blat.proc28144.chain_20376\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '451988', '452100', '100', '-', '.', 'gene_id \"chr31.g11747\"; transcript_id \"blat.proc4587.chain_11047\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '19433143', '19433182', '100', '-', '.', 'gene_id \"chr31.g11932\"; transcript_id \"blat.proc28144.chain_1638\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28285358', '28285526', '100', '-', '.', 'gene_id \"chr31.g11950\"; transcript_id \"blat.proc18128.chain_4050\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '9691133', '9691315', '100', '-', '.', 'gene_id \"chr31.g12075\"; transcript_id \"blat.proc18128.chain_19910\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37925379', '37925483', '100', '-', '.', 'gene_id \"chr31.g12225\"; transcript_id \"blat.proc18128.chain_7939\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '6504658', '6504744', '100', '+', '.', 'gene_id \"chr31.g12257\"; transcript_id \"blat.proc28144.chain_10147\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '30975228', '30975268', '100', '-', '.', 'gene_id \"chr31.g12351\"; transcript_id \"blat.proc18128.chain_22691\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '27639718', '27639766', '100', '-', '.', 'gene_id \"chr31.g12405\"; transcript_id \"blat.proc4587.chain_3737\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '25439728', '25439950', '100', '+', '.', 'gene_id \"chr31.g12458\"; transcript_id \"blat.proc18128.chain_6872\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '17641348', '17641391', '100', '-', '.', 'gene_id \"chr31.g12691\"; transcript_id \"blat.proc18128.chain_17378\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '10482467', '10482507', '100', '+', '.', 'gene_id \"chr31.g12746\"; transcript_id \"blat.proc28144.chain_20608\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '38855967', '38856000', '100', '-', '.', 'gene_id \"chr31.g12749\"; transcript_id \"blat.proc28144.chain_19108\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '19813473', '19813586', '100', '-', '.', 'gene_id \"chr31.g12819\"; transcript_id \"blat.proc4005.chain_4277\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '7874007', '7874050', '100', '+', '.', 'gene_id \"chr31.g12827\"; transcript_id \"blat.proc4587.chain_9065\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '34055431', '34055554', '100', '-', '.', 'gene_id \"chr31.g12903\"; transcript_id \"blat.proc4587.chain_9975\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '16534901', '16534936', '100', '+', '.', 'gene_id \"chr31.g13132\"; transcript_id \"blat.proc18128.chain_25747\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '20658900', '20658942', '100', '+', '.', 'gene_id \"chr31.g13163\"; transcript_id \"blat.proc4005.chain_13684\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '31232954', '31233111', '100', '-', '.', 'gene_id \"chr31.g13337\"; transcript_id \"blat.proc18128.chain_6790\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '28506544', '28506573', '100', '-', '.', 'gene_id \"chr31.g13341\"; transcript_id \"blat.proc4005.chain_18054\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '21642380', '21642436', '100', '+', '.', 'gene_id \"chr31.g13389\"; transcript_id \"blat.proc4587.chain_7977\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '1006132', '1006195', '100', '-', '.', 'gene_id \"chr31.g13479\"; transcript_id \"blat.proc4587.chain_5096\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '12477557', '12477685', '100', '+', '.', 'gene_id \"chr31.g13507\"; transcript_id \"blat.proc28144.chain_16506\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '2248862', '2248939', '100', '-', '.', 'gene_id \"chr31.g13743\"; transcript_id \"blat.proc18128.chain_11204\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '23735183', '23735229', '100', '+', '.', 'gene_id \"chr31.g13820\"; transcript_id \"blat.proc4005.chain_2532\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr31', 'PASA', 'exon', '21556246', '21556329', '100', '+', '.', 'gene_id \"chr31.g13902\"; transcript_id \"blat.proc4587.chain_10718\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '472264', '472293', '100', '+', '.', 'gene_id \"chr31.g13932\"; transcript_id \"blat.proc18128.chain_17784\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '13752358', '13752515', '100', '-', '.', 'gene_id \"chr31.g13978\"; transcript_id \"blat.proc18128.chain_8387\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '12028792', '12028838', '100', '-', '.', 'gene_id \"chr31.g13989\"; transcript_id \"blat.proc4587.chain_7752\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '37002730', '37002904', '100', '+', '.', 'gene_id \"chr31.g14036\"; transcript_id \"blat.proc4587.chain_6592\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '2844313', '2844370', '100', '+', '.', 'gene_id \"chr31.g14099\"; transcript_id \"blat.proc18128.chain_6902\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '15142940', '15142993', '100', '+', '.', 'gene_id \"chr31.g14149\"; transcript_id \"blat.proc4587.chain_738\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '8069698', '8069731', '100', '+', '.', 'gene_id \"chr31.g14190\"; transcript_id \"blat.proc18128.chain_14257\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '18408554', '18408588', '100', '+', '.', 'gene_id \"chr31.g14273\"; transcript_id \"blat.proc18128.chain_10905\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '33359860', '33360366', '100', '-', '.', 'gene_id \"chr31.g14303\"; transcript_id \"blat.proc18128.chain_797\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '6665625', '6665679', '100', '-', '.', 'gene_id \"chr31.g14379\"; transcript_id \"blat.proc4587.chain_6556\";']\n",
      "negative:  ['chr31', 'PASA', 'exon', '5440048', '5440090', '100', '+', '.', 'gene_id \"chr31.g14509\"; transcript_id \"blat.proc4587.chain_2801\";']\n",
      "negative:  ['scaffold_1642', 'PASA', 'exon', '25338', '25427', '.', '+', '.', 'gene_id \"scaffold_1642.g16\"; transcript_id \"scaffold_1642.g16.i1\";']\n",
      "negative:  ['scaffold_1019', 'PASA', 'exon', '5861', '5925', '.', '+', '.', 'gene_id \"scaffold_1019.g4\"; transcript_id \"scaffold_1019.g4.i1\";']\n",
      "negative:  ['scaffold_1019', 'PASA', 'exon', '4850', '4908', '100', '+', '.', 'gene_id \"scaffold_1019.g18\"; transcript_id \"blat.proc28144.chain_21241\";']\n",
      "negative:  ['scaffold_693', 'PASA', 'exon', '146', '410', '.', '-', '.', 'gene_id \"scaffold_693.g1\"; transcript_id \"scaffold_693.g1.i1\";']\n",
      "negative:  ['scaffold_693', 'PASA', 'exon', '1', '153', '.', '-', '.', 'gene_id \"scaffold_693.g2\"; transcript_id \"scaffold_693.g2.i1\";']\n",
      "negative:  ['scaffold_1617', 'PASA', 'exon', '15879', '15980', '100', '-', '.', 'gene_id \"scaffold_1617.g43\"; transcript_id \"blat.proc18128.chain_8980\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '27610', '27772', '.', '+', '.', 'gene_id \"chr20.g33\"; transcript_id \"chr20.g33.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '2954181', '2954363', '.', '-', '.', 'gene_id \"chr15.g1702\"; transcript_id \"chr15.g1702.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '80450', '80642', '.', '-', '.', 'gene_id \"chr20.g69\"; transcript_id \"chr20.g69.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '80978', '81207', '.', '+', '.', 'gene_id \"chr20.g71\"; transcript_id \"chr20.g71.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3160433', '3160467', '.', '-', '.', 'gene_id \"chr15.g1871\"; transcript_id \"chr15.g1871.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '381015', '381239', '.', '-', '.', 'gene_id \"chr20.g258\"; transcript_id \"chr20.g258.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '381240', '381491', '.', '-', '.', 'gene_id \"chr20.g259\"; transcript_id \"chr20.g259.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3243422', '3243836', '.', '+', '.', 'gene_id \"chr15.g1908\"; transcript_id \"chr15.g1908.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '402401', '402479', '.', '+', '.', 'gene_id \"chr20.g275\"; transcript_id \"chr20.g275.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3341653', '3341883', '.', '-', '.', 'gene_id \"chr15.g1980\"; transcript_id \"chr15.g1980.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3397744', '3397949', '.', '+', '.', 'gene_id \"chr15.g2062\"; transcript_id \"chr15.g2062.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3397976', '3398136', '.', '+', '.', 'gene_id \"chr15.g2063\"; transcript_id \"chr15.g2063.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '643831', '643879', '.', '+', '.', 'gene_id \"chr20.g450\"; transcript_id \"chr20.g450.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3605655', '3605829', '.', '+', '.', 'gene_id \"chr15.g2258\"; transcript_id \"chr15.g2258.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1128096', '1128169', '.', '+', '.', 'gene_id \"chr20.g688\"; transcript_id \"chr20.g688.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1471084', '1471283', '.', '-', '.', 'gene_id \"chr20.g786\"; transcript_id \"chr20.g786.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1496198', '1497115', '.', '+', '.', 'gene_id \"chr20.g820\"; transcript_id \"chr20.g820.i5\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1502297', '1502330', '.', '+', '.', 'gene_id \"chr20.g820\"; transcript_id \"chr20.g820.i8\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1497093', '1497414', '.', '+', '.', 'gene_id \"chr20.g820\"; transcript_id \"chr20.g820.i15\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1497517', '1497597', '.', '+', '.', 'gene_id \"chr20.g820\"; transcript_id \"chr20.g820.i15\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1497351', '1497514', '.', '+', '.', 'gene_id \"chr20.g820\"; transcript_id \"chr20.g820.i16\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1544272', '1545398', '.', '+', '.', 'gene_id \"chr20.g862\"; transcript_id \"chr20.g862.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1591257', '1591568', '.', '-', '.', 'gene_id \"chr20.g896\"; transcript_id \"chr20.g896.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1592397', '1592712', '.', '-', '.', 'gene_id \"chr20.g897\"; transcript_id \"chr20.g897.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1600513', '1600692', '.', '+', '.', 'gene_id \"chr20.g907\"; transcript_id \"chr20.g907.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1600696', '1600919', '.', '+', '.', 'gene_id \"chr20.g908\"; transcript_id \"chr20.g908.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '1676059', '1676091', '.', '+', '.', 'gene_id \"chr20.g959\"; transcript_id \"chr20.g959.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2058007', '2058220', '.', '-', '.', 'gene_id \"chr20.g1200\"; transcript_id \"chr20.g1200.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '4827828', '4828005', '.', '-', '.', 'gene_id \"chr15.g2918\"; transcript_id \"chr15.g2918.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '4828300', '4828463', '.', '+', '.', 'gene_id \"chr15.g2920\"; transcript_id \"chr15.g2920.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2242711', '2244037', '.', '-', '.', 'gene_id \"chr20.g1364\"; transcript_id \"chr20.g1364.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '5888194', '5888445', '.', '+', '.', 'gene_id \"chr15.g3168\"; transcript_id \"chr15.g3168.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '5888487', '5888801', '.', '-', '.', 'gene_id \"chr15.g3169\"; transcript_id \"chr15.g3169.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6023612', '6023651', '.', '+', '.', 'gene_id \"chr15.g3275\"; transcript_id \"chr15.g3275.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6154378', '6154451', '.', '+', '.', 'gene_id \"chr15.g3376\"; transcript_id \"chr15.g3376.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2573118', '2573659', '.', '-', '.', 'gene_id \"chr20.g1651\"; transcript_id \"chr20.g1651.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2643118', '2643422', '.', '-', '.', 'gene_id \"chr20.g1707\"; transcript_id \"chr20.g1707.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2685920', '2685967', '.', '+', '.', 'gene_id \"chr20.g1746\"; transcript_id \"chr20.g1746.i4\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2705256', '2705332', '.', '-', '.', 'gene_id \"chr20.g1790\"; transcript_id \"chr20.g1790.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6368635', '6368682', '.', '+', '.', 'gene_id \"chr15.g3526\"; transcript_id \"chr15.g3526.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2727059', '2727172', '.', '+', '.', 'gene_id \"chr20.g1806\"; transcript_id \"chr20.g1806.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6651316', '6651493', '.', '-', '.', 'gene_id \"chr15.g3620\"; transcript_id \"chr15.g3620.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6640644', '6640811', '.', '-', '.', 'gene_id \"chr15.g3621\"; transcript_id \"chr15.g3621.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '6983071', '6983101', '.', '-', '.', 'gene_id \"chr15.g3868\"; transcript_id \"chr15.g3868.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '3056073', '3056258', '.', '+', '.', 'gene_id \"chr20.g2120\"; transcript_id \"chr20.g2120.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '3056802', '3056834', '.', '+', '.', 'gene_id \"chr20.g2122\"; transcript_id \"chr20.g2122.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '3146119', '3146495', '.', '-', '.', 'gene_id \"chr20.g2193\"; transcript_id \"chr20.g2193.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '3861492', '3861531', '.', '+', '.', 'gene_id \"chr20.g2449\"; transcript_id \"chr20.g2449.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '8323183', '8323360', '.', '+', '.', 'gene_id \"chr15.g4269\"; transcript_id \"chr15.g4269.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '4212675', '4212706', '.', '+', '.', 'gene_id \"chr20.g2575\"; transcript_id \"chr20.g2575.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '8972740', '8973133', '.', '-', '.', 'gene_id \"chr15.g4736\"; transcript_id \"chr15.g4736.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5026381', '5026417', '.', '-', '.', 'gene_id \"chr20.g3025\"; transcript_id \"chr20.g3025.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '9162187', '9162350', '.', '-', '.', 'gene_id \"chr15.g4853\"; transcript_id \"chr15.g4853.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5159812', '5159963', '.', '+', '.', 'gene_id \"chr20.g3140\"; transcript_id \"chr20.g3140.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5200908', '5201216', '.', '+', '.', 'gene_id \"chr20.g3175\"; transcript_id \"chr20.g3175.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5283304', '5283492', '.', '-', '.', 'gene_id \"chr20.g3244\"; transcript_id \"chr20.g3244.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5283513', '5283781', '.', '-', '.', 'gene_id \"chr20.g3245\"; transcript_id \"chr20.g3245.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5526895', '5526925', '.', '-', '.', 'gene_id \"chr20.g3444\"; transcript_id \"chr20.g3444.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5773327', '5773611', '.', '+', '.', 'gene_id \"chr20.g3611\"; transcript_id \"chr20.g3611.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5773634', '5773838', '.', '-', '.', 'gene_id \"chr20.g3612\"; transcript_id \"chr20.g3612.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5891670', '5891956', '.', '+', '.', 'gene_id \"chr20.g3697\"; transcript_id \"chr20.g3697.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5891959', '5892320', '.', '+', '.', 'gene_id \"chr20.g3698\"; transcript_id \"chr20.g3698.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5907695', '5907737', '.', '+', '.', 'gene_id \"chr20.g3719\"; transcript_id \"chr20.g3719.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '5923139', '5923245', '.', '-', '.', 'gene_id \"chr20.g3733\"; transcript_id \"chr20.g3733.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '6103099', '6103152', '.', '-', '.', 'gene_id \"chr20.g3780\"; transcript_id \"chr20.g3780.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '6411797', '6412124', '.', '-', '.', 'gene_id \"chr20.g3919\"; transcript_id \"chr20.g3919.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '6617488', '6617638', '.', '-', '.', 'gene_id \"chr20.g4083\"; transcript_id \"chr20.g4083.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '6636829', '6637041', '.', '+', '.', 'gene_id \"chr20.g4105\"; transcript_id \"chr20.g4105.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '11663235', '11664097', '.', '-', '.', 'gene_id \"chr15.g5989\"; transcript_id \"chr15.g5989.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '6781490', '6781527', '.', '+', '.', 'gene_id \"chr20.g4254\"; transcript_id \"chr20.g4254.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '12356822', '12357058', '.', '-', '.', 'gene_id \"chr15.g6204\"; transcript_id \"chr15.g6204.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '7268932', '7268969', '.', '-', '.', 'gene_id \"chr20.g4424\"; transcript_id \"chr20.g4424.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '12753895', '12754188', '.', '+', '.', 'gene_id \"chr15.g6353\"; transcript_id \"chr15.g6353.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '7874299', '7874918', '.', '+', '.', 'gene_id \"chr20.g4671\"; transcript_id \"chr20.g4671.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '7887936', '7888036', '.', '-', '.', 'gene_id \"chr20.g4681\"; transcript_id \"chr20.g4681.i4\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '7906721', '7906767', '.', '+', '.', 'gene_id \"chr20.g4705\"; transcript_id \"chr20.g4705.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '8363723', '8363786', '.', '-', '.', 'gene_id \"chr20.g5105\"; transcript_id \"chr20.g5105.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '8470871', '8471185', '.', '-', '.', 'gene_id \"chr20.g5191\"; transcript_id \"chr20.g5191.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '8820636', '8820671', '.', '-', '.', 'gene_id \"chr20.g5292\"; transcript_id \"chr20.g5292.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '9297870', '9298020', '.', '+', '.', 'gene_id \"chr20.g5541\"; transcript_id \"chr20.g5541.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '11316194', '11316264', '.', '+', '.', 'gene_id \"chr20.g5759\"; transcript_id \"chr20.g5759.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '11433873', '11433994', '.', '-', '.', 'gene_id \"chr20.g5768\"; transcript_id \"chr20.g5768.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '12095044', '12095343', '.', '+', '.', 'gene_id \"chr20.g5845\"; transcript_id \"chr20.g5845.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '12200897', '12201089', '.', '-', '.', 'gene_id \"chr20.g5865\"; transcript_id \"chr20.g5865.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '15598166', '15598199', '.', '-', '.', 'gene_id \"chr15.g8106\"; transcript_id \"chr15.g8106.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '16300877', '16301068', '.', '+', '.', 'gene_id \"chr15.g8464\"; transcript_id \"chr15.g8464.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '16303091', '16303468', '.', '-', '.', 'gene_id \"chr15.g8471\"; transcript_id \"chr15.g8471.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr15', 'PASA', 'exon', '16481223', '16481268', '.', '-', '.', 'gene_id \"chr15.g8608\"; transcript_id \"chr15.g8608.i7\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '15995446', '15995493', '.', '+', '.', 'gene_id \"chr20.g6779\"; transcript_id \"chr20.g6779.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '17403350', '17404547', '.', '+', '.', 'gene_id \"chr20.g6883\"; transcript_id \"chr20.g6883.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '18453414', '18453851', '.', '-', '.', 'gene_id \"chr20.g7066\"; transcript_id \"chr20.g7066.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '18377823', '18377939', '.', '+', '.', 'gene_id \"chr15.g9641\"; transcript_id \"chr15.g9641.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '20206226', '20207059', '.', '-', '.', 'gene_id \"chr20.g7640\"; transcript_id \"chr20.g7640.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '20671576', '20671731', '.', '+', '.', 'gene_id \"chr20.g7949\"; transcript_id \"chr20.g7949.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '20672112', '20672333', '.', '-', '.', 'gene_id \"chr20.g7950\"; transcript_id \"chr20.g7950.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21556073', '21556239', '.', '-', '.', 'gene_id \"chr20.g8120\"; transcript_id \"chr20.g8120.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21556421', '21556738', '.', '-', '.', 'gene_id \"chr20.g8121\"; transcript_id \"chr20.g8121.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21613107', '21613378', '.', '-', '.', 'gene_id \"chr20.g8178\"; transcript_id \"chr20.g8178.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21769095', '21769252', '.', '-', '.', 'gene_id \"chr20.g8250\"; transcript_id \"chr20.g8250.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22255501', '22255694', '.', '-', '.', 'gene_id \"chr20.g8382\"; transcript_id \"chr20.g8382.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22392892', '22393263', '.', '+', '.', 'gene_id \"chr20.g8487\"; transcript_id \"chr20.g8487.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22392615', '22392768', '.', '-', '.', 'gene_id \"chr20.g8488\"; transcript_id \"chr20.g8488.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22426318', '22426539', '.', '-', '.', 'gene_id \"chr20.g8541\"; transcript_id \"chr20.g8541.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22693501', '22693852', '.', '-', '.', 'gene_id \"chr20.g8632\"; transcript_id \"chr20.g8632.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22852198', '22852456', '.', '-', '.', 'gene_id \"chr20.g8685\"; transcript_id \"chr20.g8685.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '21661984', '21662143', '.', '+', '.', 'gene_id \"chr15.g10758\"; transcript_id \"chr15.g10758.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '21662139', '21663071', '.', '-', '.', 'gene_id \"chr15.g10759\"; transcript_id \"chr15.g10759.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '21663062', '21663399', '.', '+', '.', 'gene_id \"chr15.g10760\"; transcript_id \"chr15.g10760.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '22902690', '22902719', '.', '+', '.', 'gene_id \"chr20.g8693\"; transcript_id \"chr20.g8693.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '21700844', '21701035', '.', '+', '.', 'gene_id \"chr15.g10805\"; transcript_id \"chr15.g10805.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '24142667', '24142729', '.', '+', '.', 'gene_id \"chr20.g8956\"; transcript_id \"chr20.g8956.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '22133748', '22133993', '.', '+', '.', 'gene_id \"chr15.g11028\"; transcript_id \"chr15.g11028.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '24147335', '24147544', '.', '-', '.', 'gene_id \"chr20.g8960\"; transcript_id \"chr20.g8960.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '22145518', '22145906', '.', '-', '.', 'gene_id \"chr15.g11040\"; transcript_id \"chr15.g11040.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '22145925', '22146136', '.', '-', '.', 'gene_id \"chr15.g11041\"; transcript_id \"chr15.g11041.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '24267812', '24268097', '.', '+', '.', 'gene_id \"chr20.g9058\"; transcript_id \"chr20.g9058.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '24457114', '24457158', '.', '+', '.', 'gene_id \"chr20.g9106\"; transcript_id \"chr20.g9106.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '22915350', '22915508', '.', '-', '.', 'gene_id \"chr15.g11314\"; transcript_id \"chr15.g11314.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '24804351', '24804385', '.', '+', '.', 'gene_id \"chr20.g9293\"; transcript_id \"chr20.g9293.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '25166147', '25166358', '.', '+', '.', 'gene_id \"chr20.g9492\"; transcript_id \"chr20.g9492.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '25211929', '25212160', '.', '-', '.', 'gene_id \"chr20.g9561\"; transcript_id \"chr20.g9561.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '23976691', '23976735', '.', '+', '.', 'gene_id \"chr15.g11700\"; transcript_id \"chr15.g11700.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '25359342', '25359525', '.', '+', '.', 'gene_id \"chr20.g9691\"; transcript_id \"chr20.g9691.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '25455495', '25455664', '.', '-', '.', 'gene_id \"chr20.g9780\"; transcript_id \"chr20.g9780.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '25499905', '25499977', '.', '+', '.', 'gene_id \"chr20.g9829\"; transcript_id \"chr20.g9829.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '24814417', '24814614', '.', '-', '.', 'gene_id \"chr15.g12020\"; transcript_id \"chr15.g12020.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26010308', '26010802', '.', '+', '.', 'gene_id \"chr20.g10089\"; transcript_id \"chr20.g10089.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '26683077', '26683111', '.', '-', '.', 'gene_id \"chr15.g12264\"; transcript_id \"chr15.g12264.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '27039233', '27039437', '.', '+', '.', 'gene_id \"chr15.g12309\"; transcript_id \"chr15.g12309.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26251115', '26251444', '.', '-', '.', 'gene_id \"chr20.g10290\"; transcript_id \"chr20.g10290.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '27427143', '27427339', '.', '-', '.', 'gene_id \"chr15.g12408\"; transcript_id \"chr15.g12408.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '27427340', '27427615', '.', '+', '.', 'gene_id \"chr15.g12409\"; transcript_id \"chr15.g12409.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26339279', '26339444', '.', '+', '.', 'gene_id \"chr20.g10369\"; transcript_id \"chr20.g10369.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26367550', '26367848', '.', '-', '.', 'gene_id \"chr20.g10389\"; transcript_id \"chr20.g10389.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '27915711', '27915742', '.', '+', '.', 'gene_id \"chr15.g12493\"; transcript_id \"chr15.g12493.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '28444263', '28444549', '.', '-', '.', 'gene_id \"chr15.g12530\"; transcript_id \"chr15.g12530.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26478712', '26478925', '.', '-', '.', 'gene_id \"chr20.g10473\"; transcript_id \"chr20.g10473.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26479115', '26479339', '.', '+', '.', 'gene_id \"chr20.g10475\"; transcript_id \"chr20.g10475.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26801228', '26801443', '.', '+', '.', 'gene_id \"chr20.g10613\"; transcript_id \"chr20.g10613.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '26825544', '26825803', '.', '-', '.', 'gene_id \"chr20.g10633\"; transcript_id \"chr20.g10633.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '27256673', '27256988', '.', '-', '.', 'gene_id \"chr20.g10904\"; transcript_id \"chr20.g10904.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '30438634', '30438929', '.', '+', '.', 'gene_id \"chr15.g13005\"; transcript_id \"chr15.g13005.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '28013295', '28013342', '.', '+', '.', 'gene_id \"chr20.g10976\"; transcript_id \"chr20.g10976.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '28348054', '28348100', '.', '+', '.', 'gene_id \"chr20.g11072\"; transcript_id \"chr20.g11072.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '31181811', '31181851', '.', '-', '.', 'gene_id \"chr15.g13180\"; transcript_id \"chr15.g13180.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '28635672', '28635828', '.', '-', '.', 'gene_id \"chr20.g11243\"; transcript_id \"chr20.g11243.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '28660343', '28660581', '.', '+', '.', 'gene_id \"chr20.g11260\"; transcript_id \"chr20.g11260.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '32700109', '32700265', '.', '-', '.', 'gene_id \"chr15.g13468\"; transcript_id \"chr15.g13468.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '29137218', '29137268', '.', '-', '.', 'gene_id \"chr20.g11513\"; transcript_id \"chr20.g11513.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '29374907', '29374952', '.', '-', '.', 'gene_id \"chr20.g11584\"; transcript_id \"chr20.g11584.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '29540016', '29540062', '.', '+', '.', 'gene_id \"chr20.g11609\"; transcript_id \"chr20.g11609.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '33795855', '33796076', '.', '+', '.', 'gene_id \"chr15.g13842\"; transcript_id \"chr15.g13842.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30374647', '30374804', '.', '-', '.', 'gene_id \"chr20.g11836\"; transcript_id \"chr20.g11836.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30485052', '30485185', '.', '-', '.', 'gene_id \"chr20.g11862\"; transcript_id \"chr20.g11862.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30485059', '30485107', '.', '-', '.', 'gene_id \"chr20.g11862\"; transcript_id \"chr20.g11862.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '31586573', '31587090', '.', '+', '.', 'gene_id \"chr20.g12051\"; transcript_id \"chr20.g12051.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '31815937', '31816143', '.', '+', '.', 'gene_id \"chr20.g12190\"; transcript_id \"chr20.g12190.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '31877303', '31877451', '.', '-', '.', 'gene_id \"chr20.g12237\"; transcript_id \"chr20.g12237.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '31877511', '31877787', '.', '-', '.', 'gene_id \"chr20.g12238\"; transcript_id \"chr20.g12238.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '34582172', '34582325', '.', '+', '.', 'gene_id \"chr15.g14368\"; transcript_id \"chr15.g14368.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32112941', '32113241', '.', '-', '.', 'gene_id \"chr20.g12441\"; transcript_id \"chr20.g12441.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32113257', '32113381', '.', '-', '.', 'gene_id \"chr20.g12441\"; transcript_id \"chr20.g12441.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32261146', '32261387', '.', '-', '.', 'gene_id \"chr20.g12445\"; transcript_id \"chr20.g12445.i5\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32384460', '32384711', '.', '+', '.', 'gene_id \"chr20.g12645\"; transcript_id \"chr20.g12645.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32384831', '32385338', '.', '+', '.', 'gene_id \"chr20.g12647\"; transcript_id \"chr20.g12647.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32898831', '32898874', '.', '+', '.', 'gene_id \"chr20.g12946\"; transcript_id \"chr20.g12946.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '35787381', '35787994', '.', '-', '.', 'gene_id \"chr15.g15155\"; transcript_id \"chr15.g15155.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '33215195', '33215381', '.', '+', '.', 'gene_id \"chr20.g13043\"; transcript_id \"chr20.g13043.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '35850998', '35851152', '.', '+', '.', 'gene_id \"chr15.g15189\"; transcript_id \"chr15.g15189.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '33232798', '33232963', '.', '+', '.', 'gene_id \"chr20.g13070\"; transcript_id \"chr20.g13070.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '33390558', '33390843', '.', '+', '.', 'gene_id \"chr20.g13241\"; transcript_id \"chr20.g13241.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '35395843', '35395882', '.', '-', '.', 'gene_id \"chr20.g13633\"; transcript_id \"chr20.g13633.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '38119359', '38119695', '.', '-', '.', 'gene_id \"chr15.g15868\"; transcript_id \"chr15.g15868.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '35842300', '35842453', '.', '-', '.', 'gene_id \"chr20.g13781\"; transcript_id \"chr20.g13781.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36363112', '36363377', '.', '-', '.', 'gene_id \"chr20.g14015\"; transcript_id \"chr20.g14015.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36364088', '36364434', '.', '-', '.', 'gene_id \"chr20.g14017\"; transcript_id \"chr20.g14017.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36365359', '36365575', '.', '-', '.', 'gene_id \"chr20.g14018\"; transcript_id \"chr20.g14018.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36365575', '36365803', '.', '+', '.', 'gene_id \"chr20.g14019\"; transcript_id \"chr20.g14019.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36593642', '36593837', '.', '-', '.', 'gene_id \"chr20.g14232\"; transcript_id \"chr20.g14232.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36628190', '36628474', '.', '+', '.', 'gene_id \"chr20.g14265\"; transcript_id \"chr20.g14265.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36628628', '36629204', '.', '+', '.', 'gene_id \"chr20.g14267\"; transcript_id \"chr20.g14267.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36727012', '36727273', '.', '+', '.', 'gene_id \"chr20.g14364\"; transcript_id \"chr20.g14364.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36727081', '36727273', '.', '+', '.', 'gene_id \"chr20.g14364\"; transcript_id \"chr20.g14364.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '40530403', '40531025', '.', '+', '.', 'gene_id \"chr15.g16643\"; transcript_id \"chr15.g16643.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '37157535', '37157750', '.', '-', '.', 'gene_id \"chr20.g14699\"; transcript_id \"chr20.g14699.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '37157781', '37157963', '.', '+', '.', 'gene_id \"chr20.g14700\"; transcript_id \"chr20.g14700.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '40630206', '40630428', '.', '-', '.', 'gene_id \"chr15.g16790\"; transcript_id \"chr15.g16790.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr15', 'PASA', 'exon', '40730448', '40730762', '.', '+', '.', 'gene_id \"chr15.g16874\"; transcript_id \"chr15.g16874.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '40731032', '40731429', '.', '-', '.', 'gene_id \"chr15.g16875\"; transcript_id \"chr15.g16875.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '40756352', '40756821', '.', '+', '.', 'gene_id \"chr15.g16902\"; transcript_id \"chr15.g16902.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '40847225', '40847386', '.', '+', '.', 'gene_id \"chr15.g16989\"; transcript_id \"chr15.g16989.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '37748738', '37748918', '.', '+', '.', 'gene_id \"chr20.g15106\"; transcript_id \"chr20.g15106.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38188224', '38188378', '.', '-', '.', 'gene_id \"chr20.g15374\"; transcript_id \"chr20.g15374.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38338223', '38338442', '.', '+', '.', 'gene_id \"chr20.g15434\"; transcript_id \"chr20.g15434.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38339064', '38339237', '.', '+', '.', 'gene_id \"chr20.g15436\"; transcript_id \"chr20.g15436.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38461792', '38462312', '.', '-', '.', 'gene_id \"chr20.g15485\"; transcript_id \"chr20.g15485.i6\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38479433', '38479966', '.', '-', '.', 'gene_id \"chr20.g15485\"; transcript_id \"chr20.g15485.i8\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38456662', '38457175', '.', '+', '.', 'gene_id \"chr20.g15486\"; transcript_id \"chr20.g15486.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38469449', '38469849', '.', '+', '.', 'gene_id \"chr20.g15490\"; transcript_id \"chr20.g15490.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38469299', '38469467', '.', '-', '.', 'gene_id \"chr20.g15491\"; transcript_id \"chr20.g15491.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38480180', '38480335', '.', '+', '.', 'gene_id \"chr20.g15493\"; transcript_id \"chr20.g15493.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38457468', '38458746', '.', '-', '.', 'gene_id \"chr20.g15494\"; transcript_id \"chr20.g15494.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38468183', '38468349', '.', '-', '.', 'gene_id \"chr20.g15509\"; transcript_id \"chr20.g15509.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38468523', '38468812', '.', '-', '.', 'gene_id \"chr20.g15510\"; transcript_id \"chr20.g15510.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38469095', '38469246', '.', '-', '.', 'gene_id \"chr20.g15511\"; transcript_id \"chr20.g15511.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38806535', '38806921', '.', '-', '.', 'gene_id \"chr20.g15678\"; transcript_id \"chr20.g15678.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38807451', '38808790', '.', '-', '.', 'gene_id \"chr20.g15679\"; transcript_id \"chr20.g15679.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38808894', '38809300', '.', '-', '.', 'gene_id \"chr20.g15689\"; transcript_id \"chr20.g15689.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38809311', '38809603', '.', '-', '.', 'gene_id \"chr20.g15690\"; transcript_id \"chr20.g15690.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38809622', '38810802', '.', '-', '.', 'gene_id \"chr20.g15691\"; transcript_id \"chr20.g15691.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38812063', '38812291', '.', '-', '.', 'gene_id \"chr20.g15694\"; transcript_id \"chr20.g15694.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '39636514', '39636713', '.', '-', '.', 'gene_id \"chr20.g16382\"; transcript_id \"chr20.g16382.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '39729384', '39730294', '.', '+', '.', 'gene_id \"chr20.g16454\"; transcript_id \"chr20.g16454.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '44637301', '44637487', '.', '+', '.', 'gene_id \"chr15.g18423\"; transcript_id \"chr15.g18423.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '45439650', '45439819', '.', '+', '.', 'gene_id \"chr15.g18641\"; transcript_id \"chr15.g18641.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40043148', '40043318', '.', '-', '.', 'gene_id \"chr20.g16767\"; transcript_id \"chr20.g16767.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '45749615', '45749785', '.', '+', '.', 'gene_id \"chr15.g18673\"; transcript_id \"chr15.g18673.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40160556', '40160649', '.', '+', '.', 'gene_id \"chr20.g16839\"; transcript_id \"chr20.g16839.i7\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '45928847', '45929227', '.', '-', '.', 'gene_id \"chr15.g18841\"; transcript_id \"chr15.g18841.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40418040', '40418354', '.', '+', '.', 'gene_id \"chr20.g17099\"; transcript_id \"chr20.g17099.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40421305', '40421347', '.', '+', '.', 'gene_id \"chr20.g17102\"; transcript_id \"chr20.g17102.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '46113377', '46113617', '.', '+', '.', 'gene_id \"chr15.g18995\"; transcript_id \"chr15.g18995.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40825785', '40827279', '.', '-', '.', 'gene_id \"chr20.g17211\"; transcript_id \"chr20.g17211.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '46358894', '46359085', '.', '+', '.', 'gene_id \"chr15.g19129\"; transcript_id \"chr15.g19129.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40808408', '40808744', '.', '-', '.', 'gene_id \"chr20.g17382\"; transcript_id \"chr20.g17382.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '46544712', '46544748', '.', '+', '.', 'gene_id \"chr15.g19229\"; transcript_id \"chr15.g19229.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '46576934', '46576971', '.', '-', '.', 'gene_id \"chr15.g19260\"; transcript_id \"chr15.g19260.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '47959973', '47960015', '.', '+', '.', 'gene_id \"chr15.g19415\"; transcript_id \"chr15.g19415.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '48603067', '48603107', '.', '+', '.', 'gene_id \"chr15.g19624\"; transcript_id \"chr15.g19624.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '41752283', '41752440', '.', '+', '.', 'gene_id \"chr20.g18163\"; transcript_id \"chr20.g18163.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '41782366', '41782556', '.', '+', '.', 'gene_id \"chr20.g18178\"; transcript_id \"chr20.g18178.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '49531122', '49531587', '.', '-', '.', 'gene_id \"chr15.g19902\"; transcript_id \"chr15.g19902.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '41889047', '41889445', '.', '+', '.', 'gene_id \"chr20.g18240\"; transcript_id \"chr20.g18240.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42311284', '42311472', '.', '-', '.', 'gene_id \"chr20.g18398\"; transcript_id \"chr20.g18398.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42311639', '42311903', '.', '-', '.', 'gene_id \"chr20.g18399\"; transcript_id \"chr20.g18399.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '50344640', '50345663', '.', '-', '.', 'gene_id \"chr15.g20195\"; transcript_id \"chr15.g20195.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42473447', '42473643', '.', '-', '.', 'gene_id \"chr20.g18553\"; transcript_id \"chr20.g18553.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42477719', '42477898', '.', '-', '.', 'gene_id \"chr20.g18559\"; transcript_id \"chr20.g18559.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42477719', '42477869', '.', '-', '.', 'gene_id \"chr20.g18559\"; transcript_id \"chr20.g18559.i2\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '50432698', '50433216', '.', '-', '.', 'gene_id \"chr15.g20246\"; transcript_id \"chr15.g20246.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42595516', '42595577', '.', '+', '.', 'gene_id \"chr20.g18651\"; transcript_id \"chr20.g18651.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '42661774', '42662002', '.', '-', '.', 'gene_id \"chr20.g18731\"; transcript_id \"chr20.g18731.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43152569', '43152641', '.', '+', '.', 'gene_id \"chr20.g19046\"; transcript_id \"chr20.g19046.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43257334', '43257588', '.', '+', '.', 'gene_id \"chr20.g19101\"; transcript_id \"chr20.g19101.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43306611', '43306792', '.', '+', '.', 'gene_id \"chr20.g19147\"; transcript_id \"chr20.g19147.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43390393', '43390639', '.', '-', '.', 'gene_id \"chr20.g19221\"; transcript_id \"chr20.g19221.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '51586291', '51586337', '.', '-', '.', 'gene_id \"chr15.g21024\"; transcript_id \"chr15.g21024.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '51623089', '51623150', '.', '-', '.', 'gene_id \"chr15.g21030\"; transcript_id \"chr15.g21030.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43724111', '43724277', '.', '+', '.', 'gene_id \"chr20.g19512\"; transcript_id \"chr20.g19512.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr15', 'PASA', 'exon', '53476208', '53476293', '.', '+', '.', 'gene_id \"chr15.g21488\"; transcript_id \"chr15.g21488.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '44264486', '44264709', '.', '+', '.', 'gene_id \"chr20.g19959\"; transcript_id \"chr20.g19959.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '54212644', '54212805', '.', '-', '.', 'gene_id \"chr15.g21580\"; transcript_id \"chr15.g21580.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '54453726', '54453896', '.', '-', '.', 'gene_id \"chr15.g21693\"; transcript_id \"chr15.g21693.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '44656248', '44656293', '.', '+', '.', 'gene_id \"chr20.g20256\"; transcript_id \"chr20.g20256.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '44655873', '44656251', '.', '+', '.', 'gene_id \"chr20.g20262\"; transcript_id \"chr20.g20262.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45177547', '45177758', '.', '+', '.', 'gene_id \"chr20.g20616\"; transcript_id \"chr20.g20616.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45177984', '45178168', '.', '+', '.', 'gene_id \"chr20.g20619\"; transcript_id \"chr20.g20619.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45347319', '45347564', '.', '+', '.', 'gene_id \"chr20.g20786\"; transcript_id \"chr20.g20786.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45386625', '45386826', '.', '-', '.', 'gene_id \"chr20.g20806\"; transcript_id \"chr20.g20806.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '56108539', '56108706', '.', '-', '.', 'gene_id \"chr15.g22291\"; transcript_id \"chr15.g22291.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '56221758', '56221986', '.', '+', '.', 'gene_id \"chr15.g22390\"; transcript_id \"chr15.g22390.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45614662', '45614858', '.', '-', '.', 'gene_id \"chr20.g20955\"; transcript_id \"chr20.g20955.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45625045', '45625315', '.', '+', '.', 'gene_id \"chr20.g20972\"; transcript_id \"chr20.g20972.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '56479313', '56479358', '.', '+', '.', 'gene_id \"chr15.g22529\"; transcript_id \"chr15.g22529.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45801095', '45801299', '.', '-', '.', 'gene_id \"chr20.g21127\"; transcript_id \"chr20.g21127.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '46177390', '46177963', '.', '-', '.', 'gene_id \"chr20.g21360\"; transcript_id \"chr20.g21360.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '46499873', '46500052', '.', '-', '.', 'gene_id \"chr20.g21518\"; transcript_id \"chr20.g21518.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '60638338', '60638369', '.', '+', '.', 'gene_id \"chr15.g22961\"; transcript_id \"chr15.g22961.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '60717009', '60717052', '.', '-', '.', 'gene_id \"chr15.g22970\"; transcript_id \"chr15.g22970.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '46589395', '46589428', '.', '-', '.', 'gene_id \"chr20.g21610\"; transcript_id \"chr20.g21610.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '46691263', '46691433', '.', '-', '.', 'gene_id \"chr20.g21682\"; transcript_id \"chr20.g21682.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '61067430', '61067516', '.', '+', '.', 'gene_id \"chr15.g23065\"; transcript_id \"chr15.g23065.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '61284384', '61284455', '.', '-', '.', 'gene_id \"chr15.g23168\"; transcript_id \"chr15.g23168.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '46914147', '46914398', '.', '-', '.', 'gene_id \"chr20.g21796\"; transcript_id \"chr20.g21796.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47229178', '47229345', '.', '+', '.', 'gene_id \"chr20.g21815\"; transcript_id \"chr20.g21815.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47752878', '47752961', '.', '-', '.', 'gene_id \"chr20.g21937\"; transcript_id \"chr20.g21937.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47797014', '47797482', '.', '+', '.', 'gene_id \"chr20.g21970\"; transcript_id \"chr20.g21970.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47833767', '47834022', '.', '+', '.', 'gene_id \"chr20.g22002\"; transcript_id \"chr20.g22002.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47986938', '47987048', '.', '+', '.', 'gene_id \"chr20.g22139\"; transcript_id \"chr20.g22139.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48074585', '48074964', '.', '+', '.', 'gene_id \"chr20.g22175\"; transcript_id \"chr20.g22175.i1\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '39807639', '39807704', '100', '-', '.', 'gene_id \"chr15.g24148\"; transcript_id \"blat.proc18128.chain_15219\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '10008675', '10008719', '100', '-', '.', 'gene_id \"chr15.g24288\"; transcript_id \"blat.proc28144.chain_5635\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '30649474', '30649517', '100', '-', '.', 'gene_id \"chr15.g24595\"; transcript_id \"blat.proc4587.chain_10014\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '53273934', '53273994', '100', '-', '.', 'gene_id \"chr15.g25115\"; transcript_id \"blat.proc4587.chain_6026\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '43059236', '43059296', '100', '-', '.', 'gene_id \"chr15.g25208\"; transcript_id \"blat.proc4587.chain_11098\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '33703471', '33703506', '100', '+', '.', 'gene_id \"chr15.g25314\"; transcript_id \"blat.proc18128.chain_13486\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '23368250', '23368335', '100', '-', '.', 'gene_id \"chr15.g25388\"; transcript_id \"blat.proc18128.chain_10346\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '25353283', '25353602', '100', '-', '.', 'gene_id \"chr15.g25468\"; transcript_id \"blat.proc18128.chain_8493\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '34898148', '34898181', '100', '-', '.', 'gene_id \"chr15.g25568\"; transcript_id \"blat.proc4005.chain_11577\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '49394234', '49394283', '100', '+', '.', 'gene_id \"chr15.g26365\"; transcript_id \"blat.proc18128.chain_15169\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '13717690', '13717730', '100', '+', '.', 'gene_id \"chr15.g26460\"; transcript_id \"blat.proc4587.chain_7488\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '10043044', '10043081', '100', '-', '.', 'gene_id \"chr15.g26529\"; transcript_id \"blat.proc4587.chain_6475\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '8480305', '8480377', '100', '+', '.', 'gene_id \"chr15.g26562\"; transcript_id \"blat.proc4005.chain_7194\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '33846153', '33846198', '100', '-', '.', 'gene_id \"chr15.g26827\"; transcript_id \"blat.proc18128.chain_1943\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '21114169', '21114218', '100', '-', '.', 'gene_id \"chr15.g26866\"; transcript_id \"blat.proc4587.chain_13983\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3695724', '3695773', '100', '-', '.', 'gene_id \"chr15.g27047\"; transcript_id \"blat.proc28144.chain_3696\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '3874285', '3874346', '100', '+', '.', 'gene_id \"chr15.g27230\"; transcript_id \"blat.proc28144.chain_5571\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '2489612', '2489679', '100', '+', '.', 'gene_id \"chr15.g27767\"; transcript_id \"blat.proc4587.chain_10308\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '27387831', '27387878', '100', '-', '.', 'gene_id \"chr15.g27768\"; transcript_id \"blat.proc18128.chain_3560\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '17396903', '17397062', '100', '-', '.', 'gene_id \"chr15.g27961\"; transcript_id \"blat.proc18128.chain_6955\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '16409493', '16409543', '100', '-', '.', 'gene_id \"chr15.g28499\"; transcript_id \"blat.proc18128.chain_11772\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '53023082', '53023309', '100', '+', '.', 'gene_id \"chr15.g29803\"; transcript_id \"blat.proc18128.chain_21901\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '22606138', '22606169', '100', '-', '.', 'gene_id \"chr15.g29894\"; transcript_id \"blat.proc18128.chain_16007\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '19808019', '19808177', '100', '-', '.', 'gene_id \"chr15.g29955\"; transcript_id \"blat.proc4587.chain_3201\";']\n",
      "negative:  ['chr15', 'PASA', 'exon', '61140554', '61140587', '100', '+', '.', 'gene_id \"chr15.g29970\"; transcript_id \"blat.proc4587.chain_12035\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48194813', '48195000', '.', '-', '.', 'gene_id \"chr20.g22286\"; transcript_id \"chr20.g22286.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48201885', '48202100', '.', '-', '.', 'gene_id \"chr20.g22294\"; transcript_id \"chr20.g22294.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48390697', '48390865', '.', '-', '.', 'gene_id \"chr20.g22463\"; transcript_id \"chr20.g22463.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '92', '316', '.', '+', '.', 'gene_id \"scaffold_1626.g2\"; transcript_id \"scaffold_1626.g2.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '7181', '7352', '.', '+', '.', 'gene_id \"scaffold_1626.g6\"; transcript_id \"scaffold_1626.g6.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '7713', '8068', '.', '-', '.', 'gene_id \"scaffold_1626.g7\"; transcript_id \"scaffold_1626.g7.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '7742', '8011', '.', '-', '.', 'gene_id \"scaffold_1626.g7\"; transcript_id \"scaffold_1626.g7.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '7425', '7634', '.', '-', '.', 'gene_id \"scaffold_1626.g9\"; transcript_id \"scaffold_1626.g9.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '11995', '12189', '.', '-', '.', 'gene_id \"scaffold_1626.g12\"; transcript_id \"scaffold_1626.g12.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '12088', '12873', '.', '-', '.', 'gene_id \"scaffold_1626.g12\"; transcript_id \"scaffold_1626.g12.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '13711', '13903', '.', '+', '.', 'gene_id \"scaffold_1626.g13\"; transcript_id \"scaffold_1626.g13.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '11009', '11622', '.', '-', '.', 'gene_id \"scaffold_1626.g16\"; transcript_id \"scaffold_1626.g16.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '11720', '11878', '.', '-', '.', 'gene_id \"scaffold_1626.g18\"; transcript_id \"scaffold_1626.g18.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '12855', '13053', '.', '+', '.', 'gene_id \"scaffold_1626.g19\"; transcript_id \"scaffold_1626.g19.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '13498', '13689', '.', '+', '.', 'gene_id \"scaffold_1626.g21\"; transcript_id \"scaffold_1626.g21.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '15300', '15550', '.', '+', '.', 'gene_id \"scaffold_1626.g24\"; transcript_id \"scaffold_1626.g24.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '17130', '17287', '.', '+', '.', 'gene_id \"scaffold_1626.g26\"; transcript_id \"scaffold_1626.g26.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '18627', '18806', '.', '-', '.', 'gene_id \"scaffold_1626.g29\"; transcript_id \"scaffold_1626.g29.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '19240', '19403', '.', '+', '.', 'gene_id \"scaffold_1626.g30\"; transcript_id \"scaffold_1626.g30.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '18258', '18408', '.', '+', '.', 'gene_id \"scaffold_1626.g32\"; transcript_id \"scaffold_1626.g32.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '19524', '19763', '.', '-', '.', 'gene_id \"scaffold_1626.g33\"; transcript_id \"scaffold_1626.g33.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '20027', '20185', '.', '+', '.', 'gene_id \"scaffold_1626.g34\"; transcript_id \"scaffold_1626.g34.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '24000', '24321', '.', '+', '.', 'gene_id \"scaffold_1626.g40\"; transcript_id \"scaffold_1626.g40.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '24440', '24573', '.', '+', '.', 'gene_id \"scaffold_1626.g41\"; transcript_id \"scaffold_1626.g41.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '24568', '24761', '.', '+', '.', 'gene_id \"scaffold_1626.g42\"; transcript_id \"scaffold_1626.g42.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '27241', '28236', '.', '+', '.', 'gene_id \"scaffold_1626.g44\"; transcript_id \"scaffold_1626.g44.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '26163', '26600', '.', '-', '.', 'gene_id \"scaffold_1626.g45\"; transcript_id \"scaffold_1626.g45.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '26758', '26908', '.', '-', '.', 'gene_id \"scaffold_1626.g45\"; transcript_id \"scaffold_1626.g45.i3\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '25971', '26170', '.', '+', '.', 'gene_id \"scaffold_1626.g46\"; transcript_id \"scaffold_1626.g46.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '27645', '28020', '.', '-', '.', 'gene_id \"scaffold_1626.g48\"; transcript_id \"scaffold_1626.g48.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '28348', '30464', '.', '+', '.', 'gene_id \"scaffold_1626.g49\"; transcript_id \"scaffold_1626.g49.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '29695', '30464', '.', '+', '.', 'gene_id \"scaffold_1626.g49\"; transcript_id \"scaffold_1626.g49.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '28237', '28986', '.', '+', '.', 'gene_id \"scaffold_1626.g49\"; transcript_id \"scaffold_1626.g49.i3\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '28348', '28551', '.', '+', '.', 'gene_id \"scaffold_1626.g49\"; transcript_id \"scaffold_1626.g49.i4\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '28975', '29375', '.', '-', '.', 'gene_id \"scaffold_1626.g50\"; transcript_id \"scaffold_1626.g50.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '32498', '32662', '.', '+', '.', 'gene_id \"scaffold_1626.g51\"; transcript_id \"scaffold_1626.g51.i2\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '34356', '34521', '.', '+', '.', 'gene_id \"scaffold_1626.g51\"; transcript_id \"scaffold_1626.g51.i3\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '30466', '31488', '.', '+', '.', 'gene_id \"scaffold_1626.g52\"; transcript_id \"scaffold_1626.g52.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '31718', '32071', '.', '+', '.', 'gene_id \"scaffold_1626.g52\"; transcript_id \"scaffold_1626.g52.i3\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '31467', '31739', '.', '+', '.', 'gene_id \"scaffold_1626.g52\"; transcript_id \"scaffold_1626.g52.i4\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '34527', '34784', '.', '-', '.', 'gene_id \"scaffold_1626.g53\"; transcript_id \"scaffold_1626.g53.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '34031', '34296', '.', '-', '.', 'gene_id \"scaffold_1626.g54\"; transcript_id \"scaffold_1626.g54.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '33479', '33997', '.', '-', '.', 'gene_id \"scaffold_1626.g55\"; transcript_id \"scaffold_1626.g55.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '32898', '33466', '.', '-', '.', 'gene_id \"scaffold_1626.g56\"; transcript_id \"scaffold_1626.g56.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '32218', '32418', '.', '-', '.', 'gene_id \"scaffold_1626.g57\"; transcript_id \"scaffold_1626.g57.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '32063', '32231', '.', '-', '.', 'gene_id \"scaffold_1626.g58\"; transcript_id \"scaffold_1626.g58.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '39925', '40264', '.', '+', '.', 'gene_id \"scaffold_1626.g65\"; transcript_id \"scaffold_1626.g65.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '43030', '43193', '.', '-', '.', 'gene_id \"scaffold_1626.g69\"; transcript_id \"scaffold_1626.g69.i1\";']\n",
      "negative:  ['scaffold_1626', 'PASA', 'exon', '44445', '44510', '.', '-', '.', 'gene_id \"scaffold_1626.g70\"; transcript_id \"scaffold_1626.g70.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48928549', '48928804', '.', '+', '.', 'gene_id \"chr20.g22732\"; transcript_id \"chr20.g22732.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22707', '22753', '.', '+', '.', 'gene_id \"chrX.g11\"; transcript_id \"chrX.g11.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '126377', '126568', '.', '+', '.', 'gene_id \"chrX.g92\"; transcript_id \"chrX.g92.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '695327', '695376', '.', '-', '.', 'gene_id \"chrX.g143\"; transcript_id \"chrX.g143.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '752619', '753100', '.', '+', '.', 'gene_id \"chrX.g147\"; transcript_id \"chrX.g147.i14\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '806463', '806649', '.', '-', '.', 'gene_id \"chrX.g181\"; transcript_id \"chrX.g181.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1105791', '1105963', '.', '-', '.', 'gene_id \"chrX.g335\"; transcript_id \"chrX.g335.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1106130', '1106334', '.', '+', '.', 'gene_id \"chrX.g336\"; transcript_id \"chrX.g336.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1143935', '1144381', '.', '+', '.', 'gene_id \"chrX.g387\"; transcript_id \"chrX.g387.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49861214', '49861452', '.', '+', '.', 'gene_id \"chr20.g23467\"; transcript_id \"chr20.g23467.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49922221', '49922616', '.', '-', '.', 'gene_id \"chr20.g23518\"; transcript_id \"chr20.g23518.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49968807', '49969059', '.', '-', '.', 'gene_id \"chr20.g23539\"; transcript_id \"chr20.g23539.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1415238', '1415273', '.', '+', '.', 'gene_id \"chrX.g594\"; transcript_id \"chrX.g594.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1438327', '1438526', '.', '-', '.', 'gene_id \"chrX.g625\"; transcript_id \"chrX.g625.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1843343', '1843499', '.', '-', '.', 'gene_id \"chrX.g750\"; transcript_id \"chrX.g750.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1843556', '1843963', '.', '-', '.', 'gene_id \"chrX.g751\"; transcript_id \"chrX.g751.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '1844664', '1844996', '.', '+', '.', 'gene_id \"chrX.g754\"; transcript_id \"chrX.g754.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50331347', '50331606', '.', '-', '.', 'gene_id \"chr20.g23835\"; transcript_id \"chr20.g23835.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50417969', '50418227', '.', '+', '.', 'gene_id \"chr20.g23893\"; transcript_id \"chr20.g23893.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50418318', '50418595', '.', '-', '.', 'gene_id \"chr20.g23911\"; transcript_id \"chr20.g23911.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '2512510', '2512569', '.', '+', '.', 'gene_id \"chrX.g915\"; transcript_id \"chrX.g915.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '2556688', '2556739', '.', '-', '.', 'gene_id \"chrX.g921\"; transcript_id \"chrX.g921.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50477053', '50477243', '.', '-', '.', 'gene_id \"chr20.g23984\"; transcript_id \"chr20.g23984.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50477428', '50477550', '.', '+', '.', 'gene_id \"chr20.g23993\"; transcript_id \"chr20.g23993.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '2977095', '2977248', '.', '+', '.', 'gene_id \"chrX.g960\"; transcript_id \"chrX.g960.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3022596', '3022751', '.', '+', '.', 'gene_id \"chrX.g973\"; transcript_id \"chrX.g973.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3022861', '3023410', '.', '+', '.', 'gene_id \"chrX.g974\"; transcript_id \"chrX.g974.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3086824', '3086974', '.', '+', '.', 'gene_id \"chrX.g980\"; transcript_id \"chrX.g980.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3110974', '3111014', '.', '+', '.', 'gene_id \"chrX.g985\"; transcript_id \"chrX.g985.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3110974', '3111013', '.', '+', '.', 'gene_id \"chrX.g985\"; transcript_id \"chrX.g985.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3755656', '3755689', '.', '+', '.', 'gene_id \"chrX.g1096\"; transcript_id \"chrX.g1096.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50734554', '50735434', '.', '+', '.', 'gene_id \"chr20.g24196\"; transcript_id \"chr20.g24196.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '50789992', '50790031', '.', '-', '.', 'gene_id \"chr20.g24200\"; transcript_id \"chr20.g24200.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3986948', '3987262', '.', '-', '.', 'gene_id \"chrX.g1110\"; transcript_id \"chrX.g1110.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4190842', '4191044', '.', '+', '.', 'gene_id \"chrX.g1181\"; transcript_id \"chrX.g1181.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4412100', '4412278', '.', '+', '.', 'gene_id \"chrX.g1317\"; transcript_id \"chrX.g1317.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chrX', 'PASA', 'exon', '4692487', '4692679', '.', '+', '.', 'gene_id \"chrX.g1363\"; transcript_id \"chrX.g1363.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4712945', '4712974', '.', '-', '.', 'gene_id \"chrX.g1367\"; transcript_id \"chrX.g1367.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51615338', '51615516', '.', '+', '.', 'gene_id \"chr20.g24585\"; transcript_id \"chr20.g24585.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4897180', '4897211', '.', '-', '.', 'gene_id \"chrX.g1473\"; transcript_id \"chrX.g1473.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51681714', '51681774', '.', '-', '.', 'gene_id \"chr20.g24642\"; transcript_id \"chr20.g24642.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51726828', '51726959', '.', '-', '.', 'gene_id \"chr20.g24651\"; transcript_id \"chr20.g24651.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51794739', '51794928', '.', '-', '.', 'gene_id \"chr20.g24710\"; transcript_id \"chr20.g24710.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51816339', '51816520', '.', '+', '.', 'gene_id \"chr20.g24731\"; transcript_id \"chr20.g24731.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51816547', '51816754', '.', '-', '.', 'gene_id \"chr20.g24732\"; transcript_id \"chr20.g24732.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51819270', '51819457', '.', '-', '.', 'gene_id \"chr20.g24734\"; transcript_id \"chr20.g24734.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51819706', '51819946', '.', '-', '.', 'gene_id \"chr20.g24735\"; transcript_id \"chr20.g24735.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '5295053', '5295100', '.', '-', '.', 'gene_id \"chrX.g1575\"; transcript_id \"chrX.g1575.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '5367308', '5367573', '.', '+', '.', 'gene_id \"chrX.g1609\"; transcript_id \"chrX.g1609.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51963142', '51963314', '.', '-', '.', 'gene_id \"chr20.g24828\"; transcript_id \"chr20.g24828.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51963809', '51964059', '.', '+', '.', 'gene_id \"chr20.g24830\"; transcript_id \"chr20.g24830.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51963651', '51963826', '.', '-', '.', 'gene_id \"chr20.g24831\"; transcript_id \"chr20.g24831.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '5491553', '5491588', '.', '-', '.', 'gene_id \"chrX.g1663\"; transcript_id \"chrX.g1663.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '5781030', '5781086', '.', '-', '.', 'gene_id \"chrX.g1711\"; transcript_id \"chrX.g1711.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6115392', '6115432', '.', '-', '.', 'gene_id \"chrX.g1770\"; transcript_id \"chrX.g1770.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6115393', '6115433', '.', '-', '.', 'gene_id \"chrX.g1770\"; transcript_id \"chrX.g1770.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6116952', '6117091', '.', '-', '.', 'gene_id \"chrX.g1772\"; transcript_id \"chrX.g1772.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52152388', '52152888', '.', '-', '.', 'gene_id \"chr20.g24997\"; transcript_id \"chr20.g24997.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52146589', '52146782', '.', '-', '.', 'gene_id \"chr20.g25004\"; transcript_id \"chr20.g25004.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52151514', '52151905', '.', '+', '.', 'gene_id \"chr20.g25007\"; transcript_id \"chr20.g25007.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6171770', '6171857', '.', '-', '.', 'gene_id \"chrX.g1821\"; transcript_id \"chrX.g1821.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6183561', '6183712', '.', '-', '.', 'gene_id \"chrX.g1839\"; transcript_id \"chrX.g1839.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6247960', '6248111', '.', '+', '.', 'gene_id \"chrX.g1889\"; transcript_id \"chrX.g1889.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6248095', '6248300', '.', '-', '.', 'gene_id \"chrX.g1890\"; transcript_id \"chrX.g1890.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6288728', '6288951', '.', '-', '.', 'gene_id \"chrX.g1928\"; transcript_id \"chrX.g1928.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52405814', '52406036', '.', '-', '.', 'gene_id \"chr20.g25185\"; transcript_id \"chr20.g25185.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6376391', '6376436', '.', '-', '.', 'gene_id \"chrX.g2036\"; transcript_id \"chrX.g2036.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6407788', '6408035', '.', '-', '.', 'gene_id \"chrX.g2080\"; transcript_id \"chrX.g2080.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6418676', '6418726', '.', '+', '.', 'gene_id \"chrX.g2086\"; transcript_id \"chrX.g2086.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52930577', '52931331', '.', '-', '.', 'gene_id \"chr20.g25563\"; transcript_id \"chr20.g25563.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6810727', '6810768', '.', '-', '.', 'gene_id \"chrX.g2336\"; transcript_id \"chrX.g2336.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6816537', '6816661', '.', '+', '.', 'gene_id \"chrX.g2341\"; transcript_id \"chrX.g2341.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '53333939', '53333975', '.', '+', '.', 'gene_id \"chr20.g25742\"; transcript_id \"chr20.g25742.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '53347511', '53347580', '.', '-', '.', 'gene_id \"chr20.g25756\"; transcript_id \"chr20.g25756.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '53484740', '53485632', '.', '-', '.', 'gene_id \"chr20.g25825\"; transcript_id \"chr20.g25825.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7489225', '7489265', '.', '+', '.', 'gene_id \"chrX.g2497\"; transcript_id \"chrX.g2497.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7584529', '7584704', '.', '+', '.', 'gene_id \"chrX.g2510\"; transcript_id \"chrX.g2510.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7587921', '7588053', '.', '+', '.', 'gene_id \"chrX.g2512\"; transcript_id \"chrX.g2512.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7673951', '7673996', '.', '-', '.', 'gene_id \"chrX.g2568\"; transcript_id \"chrX.g2568.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7686701', '7686812', '.', '-', '.', 'gene_id \"chrX.g2576\"; transcript_id \"chrX.g2576.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7723530', '7723559', '.', '-', '.', 'gene_id \"chrX.g2584\"; transcript_id \"chrX.g2584.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7825967', '7826126', '.', '-', '.', 'gene_id \"chrX.g2604\"; transcript_id \"chrX.g2604.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7864593', '7864765', '.', '+', '.', 'gene_id \"chrX.g2618\"; transcript_id \"chrX.g2618.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8130151', '8130421', '.', '+', '.', 'gene_id \"chrX.g2682\"; transcript_id \"chrX.g2682.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8250156', '8250202', '.', '-', '.', 'gene_id \"chrX.g2733\"; transcript_id \"chrX.g2733.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8301535', '8301569', '.', '+', '.', 'gene_id \"chrX.g2749\"; transcript_id \"chrX.g2749.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8552006', '8552144', '.', '-', '.', 'gene_id \"chrX.g2773\"; transcript_id \"chrX.g2773.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8716862', '8716899', '.', '-', '.', 'gene_id \"chrX.g2811\"; transcript_id \"chrX.g2811.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9135809', '9135842', '.', '-', '.', 'gene_id \"chrX.g2865\"; transcript_id \"chrX.g2865.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9195340', '9195390', '.', '+', '.', 'gene_id \"chrX.g2881\"; transcript_id \"chrX.g2881.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9255064', '9255244', '.', '-', '.', 'gene_id \"chrX.g2892\"; transcript_id \"chrX.g2892.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54074791', '54074960', '.', '+', '.', 'gene_id \"chr20.g26311\"; transcript_id \"chr20.g26311.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9259213', '9259657', '.', '-', '.', 'gene_id \"chrX.g2921\"; transcript_id \"chrX.g2921.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9310794', '9310847', '.', '+', '.', 'gene_id \"chrX.g2945\"; transcript_id \"chrX.g2945.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9608292', '9608333', '.', '-', '.', 'gene_id \"chrX.g2989\"; transcript_id \"chrX.g2989.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9615925', '9616046', '.', '-', '.', 'gene_id \"chrX.g2999\"; transcript_id \"chrX.g2999.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9689057', '9689215', '.', '-', '.', 'gene_id \"chrX.g3034\"; transcript_id \"chrX.g3034.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9759530', '9759577', '.', '+', '.', 'gene_id \"chrX.g3048\"; transcript_id \"chrX.g3048.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54489640', '54489841', '.', '-', '.', 'gene_id \"chr20.g26526\"; transcript_id \"chr20.g26526.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10132099', '10132142', '.', '-', '.', 'gene_id \"chrX.g3150\"; transcript_id \"chrX.g3150.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10176877', '10177030', '.', '+', '.', 'gene_id \"chrX.g3206\"; transcript_id \"chrX.g3206.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10223570', '10223683', '.', '-', '.', 'gene_id \"chrX.g3240\"; transcript_id \"chrX.g3240.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10467135', '10467333', '.', '-', '.', 'gene_id \"chrX.g3291\"; transcript_id \"chrX.g3291.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10592718', '10592751', '.', '-', '.', 'gene_id \"chrX.g3350\"; transcript_id \"chrX.g3350.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54881363', '54882061', '.', '+', '.', 'gene_id \"chr20.g26819\"; transcript_id \"chr20.g26819.i2\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54888598', '54888833', '.', '-', '.', 'gene_id \"chr20.g26820\"; transcript_id \"chr20.g26820.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10873151', '10874656', '.', '+', '.', 'gene_id \"chrX.g3390\"; transcript_id \"chrX.g3390.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10873151', '10873994', '.', '-', '.', 'gene_id \"chrX.g3391\"; transcript_id \"chrX.g3391.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10877670', '10878182', '.', '-', '.', 'gene_id \"chrX.g3393\"; transcript_id \"chrX.g3393.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54977615', '54977659', '.', '-', '.', 'gene_id \"chr20.g26913\"; transcript_id \"chr20.g26913.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55002003', '55002644', '.', '+', '.', 'gene_id \"chr20.g26927\"; transcript_id \"chr20.g26927.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55002989', '55003239', '.', '+', '.', 'gene_id \"chr20.g26929\"; transcript_id \"chr20.g26929.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55005186', '55005821', '.', '+', '.', 'gene_id \"chr20.g26934\"; transcript_id \"chr20.g26934.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55126244', '55126466', '.', '-', '.', 'gene_id \"chr20.g27025\"; transcript_id \"chr20.g27025.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55276735', '55276978', '.', '-', '.', 'gene_id \"chr20.g27137\"; transcript_id \"chr20.g27137.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55276179', '55276756', '.', '-', '.', 'gene_id \"chr20.g27138\"; transcript_id \"chr20.g27138.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55336573', '55337825', '.', '+', '.', 'gene_id \"chr20.g27193\"; transcript_id \"chr20.g27193.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11694093', '11694177', '.', '+', '.', 'gene_id \"chrX.g3696\"; transcript_id \"chrX.g3696.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55366600', '55367350', '.', '+', '.', 'gene_id \"chr20.g27218\"; transcript_id \"chr20.g27218.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11730771', '11730800', '.', '-', '.', 'gene_id \"chrX.g3715\"; transcript_id \"chrX.g3715.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11958734', '11958774', '.', '+', '.', 'gene_id \"chrX.g3828\"; transcript_id \"chrX.g3828.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11988287', '11988330', '.', '+', '.', 'gene_id \"chrX.g3849\"; transcript_id \"chrX.g3849.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12014147', '12014358', '.', '-', '.', 'gene_id \"chrX.g3858\"; transcript_id \"chrX.g3858.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12047610', '12047796', '.', '-', '.', 'gene_id \"chrX.g3879\"; transcript_id \"chrX.g3879.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12226266', '12226397', '.', '+', '.', 'gene_id \"chrX.g3987\"; transcript_id \"chrX.g3987.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12252122', '12252179', '.', '-', '.', 'gene_id \"chrX.g3990\"; transcript_id \"chrX.g3990.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55720984', '55721028', '.', '-', '.', 'gene_id \"chr20.g27525\"; transcript_id \"chr20.g27525.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12318934', '12319001', '.', '+', '.', 'gene_id \"chrX.g4010\"; transcript_id \"chrX.g4010.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12331889', '12331920', '.', '-', '.', 'gene_id \"chrX.g4014\"; transcript_id \"chrX.g4014.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12475087', '12475143', '.', '+', '.', 'gene_id \"chrX.g4056\"; transcript_id \"chrX.g4056.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12569902', '12569954', '.', '+', '.', 'gene_id \"chrX.g4089\"; transcript_id \"chrX.g4089.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55865367', '55865717', '.', '+', '.', 'gene_id \"chr20.g27639\"; transcript_id \"chr20.g27639.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55922920', '55923094', '.', '-', '.', 'gene_id \"chr20.g27673\"; transcript_id \"chr20.g27673.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '55923156', '55923311', '.', '+', '.', 'gene_id \"chr20.g27674\"; transcript_id \"chr20.g27674.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12891207', '12891248', '.', '+', '.', 'gene_id \"chrX.g4176\"; transcript_id \"chrX.g4176.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '56156242', '56157132', '.', '-', '.', 'gene_id \"chr20.g27820\"; transcript_id \"chr20.g27820.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13016913', '13017499', '.', '+', '.', 'gene_id \"chrX.g4272\"; transcript_id \"chrX.g4272.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13245305', '13245481', '.', '-', '.', 'gene_id \"chrX.g4344\"; transcript_id \"chrX.g4344.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13411750', '13411798', '.', '-', '.', 'gene_id \"chrX.g4403\"; transcript_id \"chrX.g4403.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13600372', '13600415', '.', '-', '.', 'gene_id \"chrX.g4469\"; transcript_id \"chrX.g4469.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13664342', '13664566', '.', '-', '.', 'gene_id \"chrX.g4493\"; transcript_id \"chrX.g4493.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13664572', '13665012', '.', '-', '.', 'gene_id \"chrX.g4494\"; transcript_id \"chrX.g4494.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14212341', '14212375', '.', '-', '.', 'gene_id \"chrX.g4658\"; transcript_id \"chrX.g4658.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14435954', '14436161', '.', '+', '.', 'gene_id \"chrX.g4718\"; transcript_id \"chrX.g4718.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '56827687', '56827894', '.', '-', '.', 'gene_id \"chr20.g28386\"; transcript_id \"chr20.g28386.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '56828016', '56828206', '.', '-', '.', 'gene_id \"chr20.g28387\"; transcript_id \"chr20.g28387.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14484682', '14484805', '.', '+', '.', 'gene_id \"chrX.g4733\"; transcript_id \"chrX.g4733.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14535366', '14535415', '.', '-', '.', 'gene_id \"chrX.g4741\"; transcript_id \"chrX.g4741.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14712369', '14712413', '.', '+', '.', 'gene_id \"chrX.g4793\"; transcript_id \"chrX.g4793.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14929774', '14930187', '.', '-', '.', 'gene_id \"chrX.g4883\"; transcript_id \"chrX.g4883.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14930174', '14930580', '.', '+', '.', 'gene_id \"chrX.g4884\"; transcript_id \"chrX.g4884.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '15183793', '15183859', '.', '+', '.', 'gene_id \"chrX.g4964\"; transcript_id \"chrX.g4964.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '15202734', '15202782', '.', '-', '.', 'gene_id \"chrX.g4972\"; transcript_id \"chrX.g4972.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '15900823', '15900888', '.', '-', '.', 'gene_id \"chrX.g5251\"; transcript_id \"chrX.g5251.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '15941886', '15942136', '.', '-', '.', 'gene_id \"chrX.g5266\"; transcript_id \"chrX.g5266.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16013351', '16013499', '.', '+', '.', 'gene_id \"chrX.g5282\"; transcript_id \"chrX.g5282.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '57683147', '57683371', '.', '-', '.', 'gene_id \"chr20.g29058\"; transcript_id \"chr20.g29058.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16059333', '16059487', '.', '-', '.', 'gene_id \"chrX.g5310\"; transcript_id \"chrX.g5310.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16059675', '16060250', '.', '+', '.', 'gene_id \"chrX.g5311\"; transcript_id \"chrX.g5311.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16102366', '16102578', '.', '-', '.', 'gene_id \"chrX.g5357\"; transcript_id \"chrX.g5357.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '57790288', '57790326', '.', '+', '.', 'gene_id \"chr20.g29130\"; transcript_id \"chr20.g29130.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16182783', '16182823', '.', '-', '.', 'gene_id \"chrX.g5434\"; transcript_id \"chrX.g5434.i1\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '36230895', '36230932', '100', '+', '.', 'gene_id \"chr20.g29316\"; transcript_id \"blat.proc28144.chain_18529\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '47308443', '47308500', '100', '+', '.', 'gene_id \"chr20.g29542\"; transcript_id \"blat.proc4587.chain_5522\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48259366', '48259421', '100', '-', '.', 'gene_id \"chr20.g29568\"; transcript_id \"blat.proc4587.chain_19\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30581236', '30581299', '100', '-', '.', 'gene_id \"chr20.g29641\"; transcript_id \"blat.proc18128.chain_24503\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '29072008', '29072045', '100', '+', '.', 'gene_id \"chr20.g29658\"; transcript_id \"blat.proc4587.chain_11305\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51773241', '51773361', '100', '+', '.', 'gene_id \"chr20.g29713\"; transcript_id \"blat.proc18128.chain_6971\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '16810991', '16811047', '100', '-', '.', 'gene_id \"chr20.g29879\"; transcript_id \"blat.proc18128.chain_5312\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '32910735', '32910765', '100', '-', '.', 'gene_id \"chr20.g29999\"; transcript_id \"blat.proc18128.chain_79\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '4101140', '4101174', '100', '-', '.', 'gene_id \"chr20.g30082\"; transcript_id \"blat.proc4005.chain_12484\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '16425273', '16425304', '100', '+', '.', 'gene_id \"chr20.g30239\"; transcript_id \"blat.proc28144.chain_18656\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '53709245', '53709289', '100', '+', '.', 'gene_id \"chr20.g30503\"; transcript_id \"blat.proc18128.chain_13841\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '2280786', '2280893', '100', '-', '.', 'gene_id \"chr20.g30549\"; transcript_id \"blat.proc4587.chain_8518\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54882185', '54882350', '100', '-', '.', 'gene_id \"chr20.g30650\"; transcript_id \"blat.proc18128.chain_9215\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43160378', '43160438', '100', '+', '.', 'gene_id \"chr20.g30663\"; transcript_id \"blat.proc18128.chain_21361\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '11479289', '11479347', '100', '+', '.', 'gene_id \"chr20.g30835\"; transcript_id \"blat.proc28144.chain_16601\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '44014488', '44014521', '100', '-', '.', 'gene_id \"chr20.g30941\"; transcript_id \"blat.proc18128.chain_11491\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '51184120', '51184166', '100', '-', '.', 'gene_id \"chr20.g30954\"; transcript_id \"blat.proc28144.chain_3560\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54247465', '54247521', '100', '+', '.', 'gene_id \"chr20.g31100\"; transcript_id \"blat.proc18128.chain_26487\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43205304', '43205370', '100', '+', '.', 'gene_id \"chr20.g31266\"; transcript_id \"blat.proc4005.chain_19655\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '4527440', '4527491', '100', '-', '.', 'gene_id \"chr20.g31307\"; transcript_id \"blat.proc18128.chain_20617\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49733292', '49733349', '100', '+', '.', 'gene_id \"chr20.g31423\"; transcript_id \"blat.proc18128.chain_10450\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '48059230', '48059266', '100', '-', '.', 'gene_id \"chr20.g31473\"; transcript_id \"blat.proc28144.chain_966\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '4455718', '4455752', '100', '+', '.', 'gene_id \"chr20.g31577\"; transcript_id \"blat.proc4005.chain_7736\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '41248784', '41248823', '100', '+', '.', 'gene_id \"chr20.g31588\"; transcript_id \"blat.proc28144.chain_16473\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '39709317', '39709382', '100', '+', '.', 'gene_id \"chr20.g31927\"; transcript_id \"blat.proc18128.chain_13259\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '15286471', '15286515', '100', '-', '.', 'gene_id \"chr20.g32015\"; transcript_id \"blat.proc18128.chain_1811\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '33870260', '33870364', '100', '-', '.', 'gene_id \"chr20.g32084\"; transcript_id \"blat.proc4587.chain_3691\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '13808517', '13808694', '100', '-', '.', 'gene_id \"chr20.g32149\"; transcript_id \"blat.proc4587.chain_11619\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43058260', '43058302', '100', '+', '.', 'gene_id \"chr20.g32159\"; transcript_id \"blat.proc4005.chain_9314\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52852808', '52852885', '100', '-', '.', 'gene_id \"chr20.g32409\"; transcript_id \"blat.proc4005.chain_9422\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr20', 'PASA', 'exon', '41118744', '41118791', '100', '+', '.', 'gene_id \"chr20.g32437\"; transcript_id \"blat.proc4005.chain_5394\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '16165010', '16165231', '100', '-', '.', 'gene_id \"chr20.g32472\"; transcript_id \"blat.proc18128.chain_11089\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49537360', '49537392', '100', '+', '.', 'gene_id \"chr20.g32701\"; transcript_id \"blat.proc18128.chain_1789\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21249691', '21249730', '100', '-', '.', 'gene_id \"chr20.g32795\"; transcript_id \"blat.proc18128.chain_10867\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '49346891', '49346946', '100', '+', '.', 'gene_id \"chr20.g32912\"; transcript_id \"blat.proc4587.chain_1057\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40611811', '40611890', '100', '-', '.', 'gene_id \"chr20.g33150\"; transcript_id \"blat.proc4587.chain_1662\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '21906190', '21906227', '100', '+', '.', 'gene_id \"chr20.g33228\"; transcript_id \"blat.proc28144.chain_2406\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40058700', '40058798', '100', '+', '.', 'gene_id \"chr20.g33525\"; transcript_id \"blat.proc18128.chain_21432\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '18920765', '18920804', '100', '+', '.', 'gene_id \"chr20.g33546\"; transcript_id \"blat.proc18128.chain_23066\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '10450613', '10450667', '100', '+', '.', 'gene_id \"chr20.g33637\"; transcript_id \"blat.proc18128.chain_12880\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43385530', '43385577', '100', '-', '.', 'gene_id \"chr20.g33786\"; transcript_id \"blat.proc28144.chain_20627\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30562445', '30562494', '100', '-', '.', 'gene_id \"chr20.g33805\"; transcript_id \"blat.proc18128.chain_17328\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '52582206', '52582639', '100', '-', '.', 'gene_id \"chr20.g33950\"; transcript_id \"blat.proc18128.chain_24209\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '30013776', '30013886', '100', '-', '.', 'gene_id \"chr20.g33986\"; transcript_id \"blat.proc18128.chain_23175\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '13964788', '13964847', '100', '+', '.', 'gene_id \"chr20.g34085\"; transcript_id \"blat.proc18128.chain_24205\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '7036183', '7036227', '100', '-', '.', 'gene_id \"chr20.g34135\"; transcript_id \"blat.proc18128.chain_8661\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '9434292', '9434336', '100', '-', '.', 'gene_id \"chr20.g34238\"; transcript_id \"blat.proc18128.chain_12084\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '45800859', '45800961', '100', '-', '.', 'gene_id \"chr20.g34496\"; transcript_id \"blat.proc4587.chain_5318\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '38311460', '38311499', '100', '+', '.', 'gene_id \"chr20.g34578\"; transcript_id \"blat.proc18128.chain_21313\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '27185377', '27185408', '100', '+', '.', 'gene_id \"chr20.g34934\"; transcript_id \"blat.proc28144.chain_5395\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '54989672', '54989717', '100', '-', '.', 'gene_id \"chr20.g35448\"; transcript_id \"blat.proc4587.chain_3229\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '18792763', '18792812', '100', '+', '.', 'gene_id \"chr20.g35561\"; transcript_id \"blat.proc18128.chain_25223\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '43707033', '43707070', '100', '+', '.', 'gene_id \"chr20.g35685\"; transcript_id \"blat.proc4005.chain_19923\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '20671457', '20671561', '100', '-', '.', 'gene_id \"chr20.g35763\"; transcript_id \"blat.proc4587.chain_4343\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '40854361', '40854406', '100', '-', '.', 'gene_id \"chr20.g35911\"; transcript_id \"blat.proc18128.chain_26052\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '18752699', '18752731', '100', '-', '.', 'gene_id \"chr20.g35951\"; transcript_id \"blat.proc4587.chain_13663\";']\n",
      "negative:  ['chr20', 'PASA', 'exon', '14114286', '14114362', '100', '-', '.', 'gene_id \"chr20.g36010\"; transcript_id \"blat.proc18128.chain_26301\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16612507', '16612558', '.', '-', '.', 'gene_id \"chrX.g5469\"; transcript_id \"chrX.g5469.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16717954', '16718549', '.', '+', '.', 'gene_id \"chrX.g5488\"; transcript_id \"chrX.g5488.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16744712', '16745619', '.', '-', '.', 'gene_id \"chrX.g5493\"; transcript_id \"chrX.g5493.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16744922', '16745516', '.', '+', '.', 'gene_id \"chrX.g5494\"; transcript_id \"chrX.g5494.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16856109', '16856224', '.', '+', '.', 'gene_id \"chrX.g5517\"; transcript_id \"chrX.g5517.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16912782', '16912830', '.', '-', '.', 'gene_id \"chrX.g5522\"; transcript_id \"chrX.g5522.i1\";']\n",
      "negative:  ['scaffold_1263', 'PASA', 'exon', '1', '1810', '.', '+', '.', 'gene_id \"scaffold_1263.g1\"; transcript_id \"scaffold_1263.g1.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16951703', '16951885', '.', '-', '.', 'gene_id \"chrX.g5533\"; transcript_id \"chrX.g5533.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17035108', '17035387', '.', '+', '.', 'gene_id \"chrX.g5543\"; transcript_id \"chrX.g5543.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17035132', '17035311', '.', '+', '.', 'gene_id \"chrX.g5543\"; transcript_id \"chrX.g5543.i4\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17035076', '17035670', '.', '-', '.', 'gene_id \"chrX.g5544\"; transcript_id \"chrX.g5544.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17069561', '17070470', '.', '+', '.', 'gene_id \"chrX.g5546\"; transcript_id \"chrX.g5546.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17207713', '17207779', '.', '+', '.', 'gene_id \"chrX.g5564\"; transcript_id \"chrX.g5564.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17304561', '17304779', '.', '-', '.', 'gene_id \"chrX.g5581\"; transcript_id \"chrX.g5581.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17615613', '17615697', '.', '+', '.', 'gene_id \"chrX.g5707\"; transcript_id \"chrX.g5707.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17652774', '17653131', '.', '-', '.', 'gene_id \"chrX.g5722\"; transcript_id \"chrX.g5722.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17766885', '17767007', '.', '+', '.', 'gene_id \"chrX.g5805\"; transcript_id \"chrX.g5805.i5\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17750945', '17750977', '.', '+', '.', 'gene_id \"chrX.g5824\"; transcript_id \"chrX.g5824.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17806125', '17806162', '.', '+', '.', 'gene_id \"chrX.g5856\"; transcript_id \"chrX.g5856.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '95042', '95459', '.', '+', '.', 'gene_id \"chr5.g28\"; transcript_id \"chr5.g28.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '95476', '95643', '.', '+', '.', 'gene_id \"chr5.g29\"; transcript_id \"chr5.g29.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '95712', '95888', '.', '-', '.', 'gene_id \"chr5.g30\"; transcript_id \"chr5.g30.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17953947', '17954124', '.', '-', '.', 'gene_id \"chrX.g5868\"; transcript_id \"chrX.g5868.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17957433', '17957467', '.', '-', '.', 'gene_id \"chrX.g5870\"; transcript_id \"chrX.g5870.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '277941', '278123', '.', '+', '.', 'gene_id \"chr5.g44\"; transcript_id \"chr5.g44.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '277338', '277769', '.', '+', '.', 'gene_id \"chr5.g44\"; transcript_id \"chr5.g44.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '265255', '265457', '.', '+', '.', 'gene_id \"chr5.g46\"; transcript_id \"chr5.g46.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '270539', '270903', '.', '+', '.', 'gene_id \"chr5.g47\"; transcript_id \"chr5.g47.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '273529', '273682', '.', '+', '.', 'gene_id \"chr5.g49\"; transcript_id \"chr5.g49.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '278546', '278746', '.', '+', '.', 'gene_id \"chr5.g50\"; transcript_id \"chr5.g50.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '333946', '333989', '.', '-', '.', 'gene_id \"chr5.g55\"; transcript_id \"chr5.g55.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18077503', '18077546', '.', '+', '.', 'gene_id \"chrX.g5897\"; transcript_id \"chrX.g5897.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18167247', '18167632', '.', '-', '.', 'gene_id \"chrX.g5899\"; transcript_id \"chrX.g5899.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '497018', '497274', '.', '-', '.', 'gene_id \"chr5.g65\"; transcript_id \"chr5.g65.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '498154', '498878', '.', '+', '.', 'gene_id \"chr5.g67\"; transcript_id \"chr5.g67.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '499996', '500589', '.', '-', '.', 'gene_id \"chr5.g71\"; transcript_id \"chr5.g71.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '504582', '504771', '.', '-', '.', 'gene_id \"chr5.g75\"; transcript_id \"chr5.g75.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '514832', '515039', '.', '+', '.', 'gene_id \"chr5.g79\"; transcript_id \"chr5.g79.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '515584', '515749', '.', '+', '.', 'gene_id \"chr5.g80\"; transcript_id \"chr5.g80.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18281313', '18281350', '.', '+', '.', 'gene_id \"chrX.g5916\"; transcript_id \"chrX.g5916.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '521830', '522005', '.', '+', '.', 'gene_id \"chr5.g85\"; transcript_id \"chr5.g85.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '522178', '522403', '.', '+', '.', 'gene_id \"chr5.g86\"; transcript_id \"chr5.g86.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '522771', '523148', '.', '-', '.', 'gene_id \"chr5.g87\"; transcript_id \"chr5.g87.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '524072', '524272', '.', '+', '.', 'gene_id \"chr5.g88\"; transcript_id \"chr5.g88.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '526127', '526355', '.', '-', '.', 'gene_id \"chr5.g91\"; transcript_id \"chr5.g91.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18339879', '18340068', '.', '+', '.', 'gene_id \"chrX.g5927\"; transcript_id \"chrX.g5927.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '590113', '590294', '.', '-', '.', 'gene_id \"chr5.g99\"; transcript_id \"chr5.g99.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '648171', '648500', '.', '-', '.', 'gene_id \"chr5.g101\"; transcript_id \"chr5.g101.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '675701', '675901', '.', '+', '.', 'gene_id \"chr5.g108\"; transcript_id \"chr5.g108.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '682709', '682874', '.', '-', '.', 'gene_id \"chr5.g111\"; transcript_id \"chr5.g111.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18608029', '18608221', '.', '+', '.', 'gene_id \"chrX.g5958\"; transcript_id \"chrX.g5958.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '735168', '735894', '.', '+', '.', 'gene_id \"chr5.g122\"; transcript_id \"chr5.g122.i12\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '750275', '750903', '.', '+', '.', 'gene_id \"chr5.g124\"; transcript_id \"chr5.g124.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '828803', '829516', '.', '-', '.', 'gene_id \"chr5.g129\"; transcript_id \"chr5.g129.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '779887', '780055', '.', '+', '.', 'gene_id \"chr5.g133\"; transcript_id \"chr5.g133.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '745868', '746045', '.', '-', '.', 'gene_id \"chr5.g139\"; transcript_id \"chr5.g139.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '746245', '746464', '.', '+', '.', 'gene_id \"chr5.g141\"; transcript_id \"chr5.g141.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '746576', '746748', '.', '-', '.', 'gene_id \"chr5.g143\"; transcript_id \"chr5.g143.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '753117', '753292', '.', '-', '.', 'gene_id \"chr5.g145\"; transcript_id \"chr5.g145.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '754040', '754313', '.', '-', '.', 'gene_id \"chr5.g146\"; transcript_id \"chr5.g146.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '757652', '757953', '.', '+', '.', 'gene_id \"chr5.g150\"; transcript_id \"chr5.g150.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '787164', '787453', '.', '-', '.', 'gene_id \"chr5.g152\"; transcript_id \"chr5.g152.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '826731', '826901', '.', '+', '.', 'gene_id \"chr5.g158\"; transcript_id \"chr5.g158.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '849173', '850053', '.', '+', '.', 'gene_id \"chr5.g163\"; transcript_id \"chr5.g163.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '864352', '864729', '.', '+', '.', 'gene_id \"chr5.g169\"; transcript_id \"chr5.g169.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '865077', '865681', '.', '+', '.', 'gene_id \"chr5.g172\"; transcript_id \"chr5.g172.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18935368', '18935482', '.', '+', '.', 'gene_id \"chrX.g5996\"; transcript_id \"chrX.g5996.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '882890', '883149', '.', '-', '.', 'gene_id \"chr5.g178\"; transcript_id \"chr5.g178.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '886234', '886425', '.', '+', '.', 'gene_id \"chr5.g180\"; transcript_id \"chr5.g180.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '892656', '892812', '.', '+', '.', 'gene_id \"chr5.g183\"; transcript_id \"chr5.g183.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '894576', '894752', '.', '-', '.', 'gene_id \"chr5.g186\"; transcript_id \"chr5.g186.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '895841', '896134', '.', '+', '.', 'gene_id \"chr5.g188\"; transcript_id \"chr5.g188.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '912527', '912686', '.', '-', '.', 'gene_id \"chr5.g199\"; transcript_id \"chr5.g199.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '913428', '913628', '.', '+', '.', 'gene_id \"chr5.g201\"; transcript_id \"chr5.g201.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '913901', '914053', '.', '+', '.', 'gene_id \"chr5.g202\"; transcript_id \"chr5.g202.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '915803', '916109', '.', '-', '.', 'gene_id \"chr5.g203\"; transcript_id \"chr5.g203.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19135946', '19135978', '.', '+', '.', 'gene_id \"chrX.g6025\"; transcript_id \"chrX.g6025.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '920180', '920270', '.', '+', '.', 'gene_id \"chr5.g209\"; transcript_id \"chr5.g209.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19141329', '19146370', '.', '+', '.', 'gene_id \"chrX.g6026\"; transcript_id \"chrX.g6026.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19141329', '19144861', '.', '+', '.', 'gene_id \"chrX.g6026\"; transcript_id \"chrX.g6026.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19142417', '19146370', '.', '+', '.', 'gene_id \"chrX.g6026\"; transcript_id \"chrX.g6026.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19142419', '19146370', '.', '+', '.', 'gene_id \"chrX.g6026\"; transcript_id \"chrX.g6026.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '920757', '920849', '.', '+', '.', 'gene_id \"chr5.g213\"; transcript_id \"chr5.g213.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19146388', '19146694', '.', '-', '.', 'gene_id \"chrX.g6028\"; transcript_id \"chrX.g6028.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '975823', '976238', '.', '+', '.', 'gene_id \"chr5.g231\"; transcript_id \"chr5.g231.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '975823', '976070', '.', '+', '.', 'gene_id \"chr5.g231\"; transcript_id \"chr5.g231.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '958877', '962915', '.', '+', '.', 'gene_id \"chr5.g234\"; transcript_id \"chr5.g234.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '982941', '983211', '.', '+', '.', 'gene_id \"chr5.g238\"; transcript_id \"chr5.g238.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '975513', '975725', '.', '+', '.', 'gene_id \"chr5.g239\"; transcript_id \"chr5.g239.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '991298', '991668', '.', '+', '.', 'gene_id \"chr5.g248\"; transcript_id \"chr5.g248.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '973814', '974155', '.', '-', '.', 'gene_id \"chr5.g252\"; transcript_id \"chr5.g252.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '973148', '973531', '.', '+', '.', 'gene_id \"chr5.g253\"; transcript_id \"chr5.g253.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '968448', '969494', '.', '-', '.', 'gene_id \"chr5.g255\"; transcript_id \"chr5.g255.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '971145', '971401', '.', '+', '.', 'gene_id \"chr5.g257\"; transcript_id \"chr5.g257.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19268462', '19268503', '.', '-', '.', 'gene_id \"chrX.g6044\"; transcript_id \"chrX.g6044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1002694', '1002957', '.', '+', '.', 'gene_id \"chr5.g271\"; transcript_id \"chr5.g271.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1004678', '1005002', '.', '-', '.', 'gene_id \"chr5.g275\"; transcript_id \"chr5.g275.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1006018', '1006286', '.', '+', '.', 'gene_id \"chr5.g276\"; transcript_id \"chr5.g276.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1008553', '1008793', '.', '+', '.', 'gene_id \"chr5.g280\"; transcript_id \"chr5.g280.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1042981', '1043145', '.', '-', '.', 'gene_id \"chr5.g289\"; transcript_id \"chr5.g289.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1062520', '1062768', '.', '+', '.', 'gene_id \"chr5.g292\"; transcript_id \"chr5.g292.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1080402', '1080669', '.', '-', '.', 'gene_id \"chr5.g297\"; transcript_id \"chr5.g297.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1083713', '1083877', '.', '+', '.', 'gene_id \"chr5.g301\"; transcript_id \"chr5.g301.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1084455', '1085145', '.', '+', '.', 'gene_id \"chr5.g302\"; transcript_id \"chr5.g302.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19385914', '19386100', '.', '-', '.', 'gene_id \"chrX.g6092\"; transcript_id \"chrX.g6092.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19385943', '19386100', '.', '-', '.', 'gene_id \"chrX.g6092\"; transcript_id \"chrX.g6092.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1086898', '1087080', '.', '+', '.', 'gene_id \"chr5.g305\"; transcript_id \"chr5.g305.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1088426', '1088666', '.', '-', '.', 'gene_id \"chr5.g307\"; transcript_id \"chr5.g307.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1089732', '1089906', '.', '-', '.', 'gene_id \"chr5.g308\"; transcript_id \"chr5.g308.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1111805', '1112030', '.', '-', '.', 'gene_id \"chr5.g321\"; transcript_id \"chr5.g321.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1112465', '1112619', '.', '+', '.', 'gene_id \"chr5.g322\"; transcript_id \"chr5.g322.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1127601', '1127772', '.', '+', '.', 'gene_id \"chr5.g325\"; transcript_id \"chr5.g325.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1132196', '1132566', '.', '-', '.', 'gene_id \"chr5.g329\"; transcript_id \"chr5.g329.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1134451', '1134812', '.', '+', '.', 'gene_id \"chr5.g332\"; transcript_id \"chr5.g332.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1135628', '1135931', '.', '-', '.', 'gene_id \"chr5.g335\"; transcript_id \"chr5.g335.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1135933', '1136244', '.', '-', '.', 'gene_id \"chr5.g336\"; transcript_id \"chr5.g336.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1136287', '1136500', '.', '+', '.', 'gene_id \"chr5.g337\"; transcript_id \"chr5.g337.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19455140', '19455173', '.', '-', '.', 'gene_id \"chrX.g6137\"; transcript_id \"chrX.g6137.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1141222', '1141386', '.', '+', '.', 'gene_id \"chr5.g339\"; transcript_id \"chr5.g339.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1145280', '1145450', '.', '+', '.', 'gene_id \"chr5.g340\"; transcript_id \"chr5.g340.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1152127', '1152330', '.', '+', '.', 'gene_id \"chr5.g342\"; transcript_id \"chr5.g342.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1166884', '1166935', '.', '+', '.', 'gene_id \"chr5.g344\"; transcript_id \"chr5.g344.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1168823', '1168874', '.', '-', '.', 'gene_id \"chr5.g345\"; transcript_id \"chr5.g345.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1177962', '1178525', '.', '-', '.', 'gene_id \"chr5.g350\"; transcript_id \"chr5.g350.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1180332', '1184832', '.', '+', '.', 'gene_id \"chr5.g352\"; transcript_id \"chr5.g352.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1222366', '1222517', '.', '-', '.', 'gene_id \"chr5.g361\"; transcript_id \"chr5.g361.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1262623', '1262903', '.', '+', '.', 'gene_id \"chr5.g364\"; transcript_id \"chr5.g364.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1270045', '1270291', '.', '-', '.', 'gene_id \"chr5.g367\"; transcript_id \"chr5.g367.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1306366', '1306560', '.', '-', '.', 'gene_id \"chr5.g373\"; transcript_id \"chr5.g373.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1307536', '1307866', '.', '+', '.', 'gene_id \"chr5.g374\"; transcript_id \"chr5.g374.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1307961', '1308194', '.', '-', '.', 'gene_id \"chr5.g375\"; transcript_id \"chr5.g375.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1348105', '1348314', '.', '-', '.', 'gene_id \"chr5.g378\"; transcript_id \"chr5.g378.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1686549', '1686719', '.', '-', '.', 'gene_id \"chr5.g402\"; transcript_id \"chr5.g402.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1731154', '1731336', '.', '-', '.', 'gene_id \"chr5.g403\"; transcript_id \"chr5.g403.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1737944', '1737981', '.', '+', '.', 'gene_id \"chr5.g404\"; transcript_id \"chr5.g404.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1803049', '1804447', '.', '+', '.', 'gene_id \"chr5.g408\"; transcript_id \"chr5.g408.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1872364', '1872527', '.', '-', '.', 'gene_id \"chr5.g413\"; transcript_id \"chr5.g413.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1891543', '1891849', '.', '-', '.', 'gene_id \"chr5.g415\"; transcript_id \"chr5.g415.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '2119472', '2119521', '.', '+', '.', 'gene_id \"chr5.g426\"; transcript_id \"chr5.g426.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '2119472', '2119521', '.', '-', '.', 'gene_id \"chr5.g427\"; transcript_id \"chr5.g427.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19711861', '19711905', '.', '+', '.', 'gene_id \"chrX.g6309\"; transcript_id \"chrX.g6309.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3077119', '3077343', '.', '+', '.', 'gene_id \"chr5.g542\"; transcript_id \"chr5.g542.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3110116', '3110298', '.', '+', '.', 'gene_id \"chr5.g544\"; transcript_id \"chr5.g544.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3144296', '3144472', '.', '+', '.', 'gene_id \"chr5.g548\"; transcript_id \"chr5.g548.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3144851', '3145004', '.', '-', '.', 'gene_id \"chr5.g549\"; transcript_id \"chr5.g549.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3235768', '3235876', '.', '-', '.', 'gene_id \"chr5.g552\"; transcript_id \"chr5.g552.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3310332', '3310574', '.', '+', '.', 'gene_id \"chr5.g557\"; transcript_id \"chr5.g557.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3321309', '3321505', '.', '-', '.', 'gene_id \"chr5.g558\"; transcript_id \"chr5.g558.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3322421', '3322575', '.', '-', '.', 'gene_id \"chr5.g560\"; transcript_id \"chr5.g560.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3407484', '3407669', '.', '+', '.', 'gene_id \"chr5.g565\"; transcript_id \"chr5.g565.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3471749', '3471922', '.', '-', '.', 'gene_id \"chr5.g570\"; transcript_id \"chr5.g570.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3517582', '3517750', '.', '+', '.', 'gene_id \"chr5.g571\"; transcript_id \"chr5.g571.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3606191', '3606406', '.', '+', '.', 'gene_id \"chr5.g579\"; transcript_id \"chr5.g579.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19828587', '19828695', '.', '+', '.', 'gene_id \"chrX.g6388\"; transcript_id \"chrX.g6388.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19828606', '19828695', '.', '+', '.', 'gene_id \"chrX.g6388\"; transcript_id \"chrX.g6388.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3651772', '3651937', '.', '-', '.', 'gene_id \"chr5.g583\"; transcript_id \"chr5.g583.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3655329', '3655762', '.', '-', '.', 'gene_id \"chr5.g584\"; transcript_id \"chr5.g584.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3757087', '3757530', '.', '-', '.', 'gene_id \"chr5.g591\"; transcript_id \"chr5.g591.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3757087', '3757527', '.', '-', '.', 'gene_id \"chr5.g591\"; transcript_id \"chr5.g591.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19935132', '19935172', '.', '+', '.', 'gene_id \"chrX.g6412\"; transcript_id \"chrX.g6412.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '20108325', '20108364', '.', '-', '.', 'gene_id \"chrX.g6500\"; transcript_id \"chrX.g6500.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '20273548', '20273706', '.', '+', '.', 'gene_id \"chrX.g6546\"; transcript_id \"chrX.g6546.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4444774', '4444962', '.', '+', '.', 'gene_id \"chr5.g750\"; transcript_id \"chr5.g750.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4448352', '4448728', '.', '-', '.', 'gene_id \"chr5.g755\"; transcript_id \"chr5.g755.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4448815', '4448975', '.', '+', '.', 'gene_id \"chr5.g756\"; transcript_id \"chr5.g756.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4471555', '4471734', '.', '-', '.', 'gene_id \"chr5.g788\"; transcript_id \"chr5.g788.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4476099', '4476347', '.', '-', '.', 'gene_id \"chr5.g792\"; transcript_id \"chr5.g792.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4496748', '4496934', '.', '+', '.', 'gene_id \"chr5.g806\"; transcript_id \"chr5.g806.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4503376', '4503754', '.', '+', '.', 'gene_id \"chr5.g808\"; transcript_id \"chr5.g808.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4573418', '4573569', '.', '-', '.', 'gene_id \"chr5.g828\"; transcript_id \"chr5.g828.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4586052', '4586209', '.', '-', '.', 'gene_id \"chr5.g845\"; transcript_id \"chr5.g845.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4593943', '4594635', '.', '+', '.', 'gene_id \"chr5.g850\"; transcript_id \"chr5.g850.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '20594556', '20594599', '.', '+', '.', 'gene_id \"chrX.g6637\"; transcript_id \"chrX.g6637.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4620367', '4620580', '.', '+', '.', 'gene_id \"chr5.g878\"; transcript_id \"chr5.g878.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4626542', '4627224', '.', '-', '.', 'gene_id \"chr5.g880\"; transcript_id \"chr5.g880.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4626564', '4627224', '.', '-', '.', 'gene_id \"chr5.g880\"; transcript_id \"chr5.g880.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4627525', '4627821', '.', '+', '.', 'gene_id \"chr5.g881\"; transcript_id \"chr5.g881.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4627222', '4627526', '.', '+', '.', 'gene_id \"chr5.g882\"; transcript_id \"chr5.g882.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4637379', '4637796', '.', '-', '.', 'gene_id \"chr5.g885\"; transcript_id \"chr5.g885.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4638800', '4639050', '.', '-', '.', 'gene_id \"chr5.g887\"; transcript_id \"chr5.g887.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '20751308', '20751545', '.', '+', '.', 'gene_id \"chrX.g6662\"; transcript_id \"chrX.g6662.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4668232', '4668430', '.', '+', '.', 'gene_id \"chr5.g904\"; transcript_id \"chr5.g904.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4693434', '4693883', '.', '-', '.', 'gene_id \"chr5.g917\"; transcript_id \"chr5.g917.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4695360', '4695558', '.', '+', '.', 'gene_id \"chr5.g922\"; transcript_id \"chr5.g922.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4703504', '4703752', '.', '+', '.', 'gene_id \"chr5.g928\"; transcript_id \"chr5.g928.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4761954', '4762106', '.', '+', '.', 'gene_id \"chr5.g943\"; transcript_id \"chr5.g943.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4742528', '4742709', '.', '+', '.', 'gene_id \"chr5.g956\"; transcript_id \"chr5.g956.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4742788', '4742986', '.', '+', '.', 'gene_id \"chr5.g957\"; transcript_id \"chr5.g957.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21314959', '21315369', '.', '+', '.', 'gene_id \"chrX.g6719\"; transcript_id \"chrX.g6719.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4775098', '4775433', '.', '-', '.', 'gene_id \"chr5.g963\"; transcript_id \"chr5.g963.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21382268', '21382601', '.', '-', '.', 'gene_id \"chrX.g6726\"; transcript_id \"chrX.g6726.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4776922', '4777165', '.', '+', '.', 'gene_id \"chr5.g967\"; transcript_id \"chr5.g967.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4787795', '4787835', '.', '+', '.', 'gene_id \"chr5.g980\"; transcript_id \"chr5.g980.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4789965', '4790169', '.', '-', '.', 'gene_id \"chr5.g981\"; transcript_id \"chr5.g981.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4804987', '4805243', '.', '+', '.', 'gene_id \"chr5.g989\"; transcript_id \"chr5.g989.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4864771', '4864929', '.', '-', '.', 'gene_id \"chr5.g1001\"; transcript_id \"chr5.g1001.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4865033', '4865385', '.', '-', '.', 'gene_id \"chr5.g1002\"; transcript_id \"chr5.g1002.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4868906', '4869085', '.', '-', '.', 'gene_id \"chr5.g1009\"; transcript_id \"chr5.g1009.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4869182', '4869342', '.', '+', '.', 'gene_id \"chr5.g1010\"; transcript_id \"chr5.g1010.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21619158', '21619316', '.', '-', '.', 'gene_id \"chrX.g6773\"; transcript_id \"chrX.g6773.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4913018', '4913272', '.', '+', '.', 'gene_id \"chr5.g1021\"; transcript_id \"chr5.g1021.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21655838', '21655939', '.', '+', '.', 'gene_id \"chrX.g6785\"; transcript_id \"chrX.g6785.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4953693', '4953857', '.', '+', '.', 'gene_id \"chr5.g1043\"; transcript_id \"chr5.g1043.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4956041', '4956452', '.', '-', '.', 'gene_id \"chr5.g1044\"; transcript_id \"chr5.g1044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4956441', '4956680', '.', '-', '.', 'gene_id \"chr5.g1045\"; transcript_id \"chr5.g1045.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22082679', '22083170', '.', '-', '.', 'gene_id \"chrX.g6812\"; transcript_id \"chrX.g6812.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4976115', '4976309', '.', '+', '.', 'gene_id \"chr5.g1073\"; transcript_id \"chr5.g1073.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4976308', '4976495', '.', '-', '.', 'gene_id \"chr5.g1074\"; transcript_id \"chr5.g1074.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22196019', '22196118', '.', '+', '.', 'gene_id \"chrX.g6824\"; transcript_id \"chrX.g6824.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22216918', '22216999', '.', '-', '.', 'gene_id \"chrX.g6830\"; transcript_id \"chrX.g6830.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5023556', '5023709', '.', '-', '.', 'gene_id \"chr5.g1088\"; transcript_id \"chr5.g1088.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5145220', '5145383', '.', '-', '.', 'gene_id \"chr5.g1102\"; transcript_id \"chr5.g1102.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5189832', '5190063', '.', '-', '.', 'gene_id \"chr5.g1116\"; transcript_id \"chr5.g1116.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5299648', '5299977', '.', '+', '.', 'gene_id \"chr5.g1150\"; transcript_id \"chr5.g1150.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22795137', '22795314', '.', '+', '.', 'gene_id \"chrX.g6899\"; transcript_id \"chrX.g6899.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22832211', '22832785', '.', '+', '.', 'gene_id \"chrX.g6901\"; transcript_id \"chrX.g6901.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5417762', '5418016', '.', '+', '.', 'gene_id \"chr5.g1181\"; transcript_id \"chr5.g1181.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5694961', '5695147', '.', '+', '.', 'gene_id \"chr5.g1188\"; transcript_id \"chr5.g1188.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23078538', '23078647', '.', '-', '.', 'gene_id \"chrX.g6933\"; transcript_id \"chrX.g6933.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5420368', '5420520', '.', '-', '.', 'gene_id \"chr5.g1193\"; transcript_id \"chr5.g1193.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5423604', '5423814', '.', '+', '.', 'gene_id \"chr5.g1197\"; transcript_id \"chr5.g1197.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5423833', '5424287', '.', '-', '.', 'gene_id \"chr5.g1198\"; transcript_id \"chr5.g1198.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5426949', '5427121', '.', '-', '.', 'gene_id \"chr5.g1203\"; transcript_id \"chr5.g1203.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23220037', '23220067', '.', '-', '.', 'gene_id \"chrX.g6945\"; transcript_id \"chrX.g6945.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5428995', '5429148', '.', '+', '.', 'gene_id \"chr5.g1204\"; transcript_id \"chr5.g1204.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5439925', '5440115', '.', '-', '.', 'gene_id \"chr5.g1210\"; transcript_id \"chr5.g1210.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5457285', '5457450', '.', '+', '.', 'gene_id \"chr5.g1218\"; transcript_id \"chr5.g1218.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23354987', '23355169', '.', '-', '.', 'gene_id \"chrX.g6969\"; transcript_id \"chrX.g6969.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5471053', '5471371', '.', '+', '.', 'gene_id \"chr5.g1231\"; transcript_id \"chr5.g1231.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5471425', '5471698', '.', '+', '.', 'gene_id \"chr5.g1232\"; transcript_id \"chr5.g1232.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23499445', '23499855', '.', '+', '.', 'gene_id \"chrX.g6987\"; transcript_id \"chrX.g6987.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23499950', '23500018', '.', '+', '.', 'gene_id \"chrX.g6989\"; transcript_id \"chrX.g6989.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5494209', '5494379', '.', '-', '.', 'gene_id \"chr5.g1252\"; transcript_id \"chr5.g1252.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5494239', '5494357', '.', '-', '.', 'gene_id \"chr5.g1252\"; transcript_id \"chr5.g1252.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23554774', '23555181', '.', '-', '.', 'gene_id \"chrX.g6997\"; transcript_id \"chrX.g6997.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5513201', '5513576', '.', '+', '.', 'gene_id \"chr5.g1259\"; transcript_id \"chr5.g1259.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23580320', '23581163', '.', '-', '.', 'gene_id \"chrX.g7003\"; transcript_id \"chrX.g7003.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5522244', '5522804', '.', '-', '.', 'gene_id \"chr5.g1265\"; transcript_id \"chr5.g1265.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23671152', '23671192', '.', '+', '.', 'gene_id \"chrX.g7011\"; transcript_id \"chrX.g7011.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5527204', '5527265', '.', '+', '.', 'gene_id \"chr5.g1271\"; transcript_id \"chr5.g1271.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5528766', '5528911', '.', '-', '.', 'gene_id \"chr5.g1276\"; transcript_id \"chr5.g1276.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23732451', '23732501', '.', '-', '.', 'gene_id \"chrX.g7020\"; transcript_id \"chrX.g7020.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5538189', '5538447', '.', '-', '.', 'gene_id \"chr5.g1283\"; transcript_id \"chr5.g1283.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23776720', '23777021', '.', '-', '.', 'gene_id \"chrX.g7030\"; transcript_id \"chrX.g7030.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5583291', '5583519', '.', '+', '.', 'gene_id \"chr5.g1293\"; transcript_id \"chr5.g1293.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23839048', '23839213', '.', '-', '.', 'gene_id \"chrX.g7045\"; transcript_id \"chrX.g7045.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23839481', '23839871', '.', '+', '.', 'gene_id \"chrX.g7046\"; transcript_id \"chrX.g7046.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23945219', '23945264', '.', '-', '.', 'gene_id \"chrX.g7065\"; transcript_id \"chrX.g7065.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5710279', '5710503', '.', '+', '.', 'gene_id \"chr5.g1328\"; transcript_id \"chr5.g1328.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5710493', '5710693', '.', '-', '.', 'gene_id \"chr5.g1329\"; transcript_id \"chr5.g1329.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23985702', '23985746', '.', '+', '.', 'gene_id \"chrX.g7072\"; transcript_id \"chrX.g7072.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5733235', '5733428', '.', '+', '.', 'gene_id \"chr5.g1336\"; transcript_id \"chr5.g1336.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5802675', '5802946', '.', '-', '.', 'gene_id \"chr5.g1349\"; transcript_id \"chr5.g1349.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5831126', '5837404', '.', '-', '.', 'gene_id \"chr5.g1351\"; transcript_id \"chr5.g1351.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '24114897', '24115142', '.', '+', '.', 'gene_id \"chrX.g7093\"; transcript_id \"chrX.g7093.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5841980', '5842157', '.', '+', '.', 'gene_id \"chr5.g1359\"; transcript_id \"chr5.g1359.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5843333', '5843498', '.', '-', '.', 'gene_id \"chr5.g1360\"; transcript_id \"chr5.g1360.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5879636', '5879842', '.', '+', '.', 'gene_id \"chr5.g1394\"; transcript_id \"chr5.g1394.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5882597', '5882750', '.', '+', '.', 'gene_id \"chr5.g1398\"; transcript_id \"chr5.g1398.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5883671', '5883826', '.', '-', '.', 'gene_id \"chr5.g1400\"; transcript_id \"chr5.g1400.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5892085', '5892273', '.', '+', '.', 'gene_id \"chr5.g1409\"; transcript_id \"chr5.g1409.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5893291', '5893467', '.', '+', '.', 'gene_id \"chr5.g1412\"; transcript_id \"chr5.g1412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5893977', '5894348', '.', '-', '.', 'gene_id \"chr5.g1413\"; transcript_id \"chr5.g1413.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5894421', '5894609', '.', '-', '.', 'gene_id \"chr5.g1414\"; transcript_id \"chr5.g1414.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5894867', '5895035', '.', '+', '.', 'gene_id \"chr5.g1415\"; transcript_id \"chr5.g1415.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5895787', '5895949', '.', '-', '.', 'gene_id \"chr5.g1416\"; transcript_id \"chr5.g1416.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5899137', '5899337', '.', '-', '.', 'gene_id \"chr5.g1418\"; transcript_id \"chr5.g1418.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5903438', '5903729', '.', '-', '.', 'gene_id \"chr5.g1421\"; transcript_id \"chr5.g1421.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5906504', '5907122', '.', '+', '.', 'gene_id \"chr5.g1423\"; transcript_id \"chr5.g1423.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '24674948', '24675015', '.', '+', '.', 'gene_id \"chrX.g7155\"; transcript_id \"chrX.g7155.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5913428', '5913653', '.', '-', '.', 'gene_id \"chr5.g1429\"; transcript_id \"chr5.g1429.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5922929', '5923169', '.', '+', '.', 'gene_id \"chr5.g1437\"; transcript_id \"chr5.g1437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5925523', '5925679', '.', '+', '.', 'gene_id \"chr5.g1440\"; transcript_id \"chr5.g1440.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5928996', '5929622', '.', '+', '.', 'gene_id \"chr5.g1445\"; transcript_id \"chr5.g1445.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5929607', '5929782', '.', '-', '.', 'gene_id \"chr5.g1446\"; transcript_id \"chr5.g1446.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5935215', '5935704', '.', '-', '.', 'gene_id \"chr5.g1452\"; transcript_id \"chr5.g1452.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5939848', '5940011', '.', '+', '.', 'gene_id \"chr5.g1456\"; transcript_id \"chr5.g1456.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5943641', '5944186', '.', '-', '.', 'gene_id \"chr5.g1460\"; transcript_id \"chr5.g1460.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5944615', '5945199', '.', '-', '.', 'gene_id \"chr5.g1461\"; transcript_id \"chr5.g1461.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5945206', '5945646', '.', '+', '.', 'gene_id \"chr5.g1462\"; transcript_id \"chr5.g1462.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5946002', '5946204', '.', '+', '.', 'gene_id \"chr5.g1463\"; transcript_id \"chr5.g1463.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5961076', '5961555', '.', '+', '.', 'gene_id \"chr5.g1483\"; transcript_id \"chr5.g1483.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5966447', '5966622', '.', '-', '.', 'gene_id \"chr5.g1489\"; transcript_id \"chr5.g1489.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5966617', '5967140', '.', '-', '.', 'gene_id \"chr5.g1490\"; transcript_id \"chr5.g1490.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5968106', '5968280', '.', '-', '.', 'gene_id \"chr5.g1493\"; transcript_id \"chr5.g1493.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25304098', '25304413', '.', '+', '.', 'gene_id \"chrX.g7233\"; transcript_id \"chrX.g7233.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5995002', '5995306', '.', '-', '.', 'gene_id \"chr5.g1505\"; transcript_id \"chr5.g1505.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6002023', '6002311', '.', '-', '.', 'gene_id \"chr5.g1511\"; transcript_id \"chr5.g1511.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6004676', '6004862', '.', '-', '.', 'gene_id \"chr5.g1515\"; transcript_id \"chr5.g1515.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6010436', '6010678', '.', '+', '.', 'gene_id \"chr5.g1516\"; transcript_id \"chr5.g1516.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6010657', '6010811', '.', '+', '.', 'gene_id \"chr5.g1516\"; transcript_id \"chr5.g1516.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6014774', '6015020', '.', '+', '.', 'gene_id \"chr5.g1520\"; transcript_id \"chr5.g1520.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25500395', '25501233', '.', '-', '.', 'gene_id \"chrX.g7262\"; transcript_id \"chrX.g7262.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6024424', '6024489', '.', '+', '.', 'gene_id \"chr5.g1535\"; transcript_id \"chr5.g1535.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25565104', '25565192', '.', '+', '.', 'gene_id \"chrX.g7275\"; transcript_id \"chrX.g7275.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6036964', '6037145', '.', '+', '.', 'gene_id \"chr5.g1540\"; transcript_id \"chr5.g1540.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6145576', '6145735', '.', '-', '.', 'gene_id \"chr5.g1569\"; transcript_id \"chr5.g1569.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6156125', '6156362', '.', '-', '.', 'gene_id \"chr5.g1578\"; transcript_id \"chr5.g1578.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6169002', '6169278', '.', '+', '.', 'gene_id \"chr5.g1583\"; transcript_id \"chr5.g1583.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6174732', '6175140', '.', '+', '.', 'gene_id \"chr5.g1587\"; transcript_id \"chr5.g1587.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6174732', '6174975', '.', '+', '.', 'gene_id \"chr5.g1587\"; transcript_id \"chr5.g1587.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6174954', '6175112', '.', '+', '.', 'gene_id \"chr5.g1587\"; transcript_id \"chr5.g1587.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6174498', '6174736', '.', '-', '.', 'gene_id \"chr5.g1588\"; transcript_id \"chr5.g1588.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6189422', '6189627', '.', '-', '.', 'gene_id \"chr5.g1598\"; transcript_id \"chr5.g1598.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6190082', '6190859', '.', '-', '.', 'gene_id \"chr5.g1599\"; transcript_id \"chr5.g1599.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25828818', '25828848', '.', '+', '.', 'gene_id \"chrX.g7328\"; transcript_id \"chrX.g7328.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25830015', '25830173', '.', '-', '.', 'gene_id \"chrX.g7329\"; transcript_id \"chrX.g7329.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6193797', '6194070', '.', '+', '.', 'gene_id \"chr5.g1605\"; transcript_id \"chr5.g1605.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6193901', '6194070', '.', '-', '.', 'gene_id \"chr5.g1606\"; transcript_id \"chr5.g1606.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6195696', '6196061', '.', '+', '.', 'gene_id \"chr5.g1609\"; transcript_id \"chr5.g1609.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6414959', '6415003', '.', '+', '.', 'gene_id \"chr5.g1629\"; transcript_id \"chr5.g1629.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6416636', '6416669', '.', '-', '.', 'gene_id \"chr5.g1632\"; transcript_id \"chr5.g1632.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25989631', '25989678', '.', '-', '.', 'gene_id \"chrX.g7417\"; transcript_id \"chrX.g7417.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25996541', '25996596', '.', '+', '.', 'gene_id \"chrX.g7421\"; transcript_id \"chrX.g7421.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '7279210', '7279352', '.', '-', '.', 'gene_id \"chr5.g1690\"; transcript_id \"chr5.g1690.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26080247', '26080468', '.', '+', '.', 'gene_id \"chrX.g7468\"; transcript_id \"chrX.g7468.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26079915', '26080262', '.', '+', '.', 'gene_id \"chrX.g7469\"; transcript_id \"chrX.g7469.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '8176876', '8176929', '.', '-', '.', 'gene_id \"chr5.g1851\"; transcript_id \"chr5.g1851.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26993306', '26993348', '.', '+', '.', 'gene_id \"chrX.g7661\"; transcript_id \"chrX.g7661.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '27242610', '27242675', '.', '-', '.', 'gene_id \"chrX.g7681\"; transcript_id \"chrX.g7681.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '8503227', '8503397', '.', '-', '.', 'gene_id \"chr5.g2037\"; transcript_id \"chr5.g2037.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '27774327', '27774369', '.', '+', '.', 'gene_id \"chrX.g7774\"; transcript_id \"chrX.g7774.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28011237', '28011271', '.', '+', '.', 'gene_id \"chrX.g7816\"; transcript_id \"chrX.g7816.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '8870205', '8870282', '.', '-', '.', 'gene_id \"chr5.g2185\"; transcript_id \"chr5.g2185.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28359846', '28360038', '.', '-', '.', 'gene_id \"chrX.g7901\"; transcript_id \"chrX.g7901.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28779813', '28779851', '.', '-', '.', 'gene_id \"chrX.g7987\"; transcript_id \"chrX.g7987.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28836811', '28836843', '.', '+', '.', 'gene_id \"chrX.g7994\"; transcript_id \"chrX.g7994.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28836812', '28836843', '.', '+', '.', 'gene_id \"chrX.g7994\"; transcript_id \"chrX.g7994.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28905415', '28905545', '.', '-', '.', 'gene_id \"chrX.g8004\"; transcript_id \"chrX.g8004.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29159751', '29159905', '.', '+', '.', 'gene_id \"chrX.g8027\"; transcript_id \"chrX.g8027.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29622473', '29622685', '.', '+', '.', 'gene_id \"chrX.g8082\"; transcript_id \"chrX.g8082.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29738561', '29738594', '.', '+', '.', 'gene_id \"chrX.g8091\"; transcript_id \"chrX.g8091.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '9324005', '9324040', '.', '-', '.', 'gene_id \"chr5.g2432\"; transcript_id \"chr5.g2432.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '9422865', '9422897', '.', '-', '.', 'gene_id \"chr5.g2481\"; transcript_id \"chr5.g2481.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30096778', '30096839', '.', '+', '.', 'gene_id \"chrX.g8166\"; transcript_id \"chrX.g8166.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30106027', '30106238', '.', '+', '.', 'gene_id \"chrX.g8167\"; transcript_id \"chrX.g8167.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30389226', '30389268', '.', '+', '.', 'gene_id \"chrX.g8192\"; transcript_id \"chrX.g8192.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30596986', '30597028', '.', '+', '.', 'gene_id \"chrX.g8219\"; transcript_id \"chrX.g8219.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30626694', '30626801', '.', '-', '.', 'gene_id \"chrX.g8223\"; transcript_id \"chrX.g8223.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30626697', '30626800', '.', '-', '.', 'gene_id \"chrX.g8223\"; transcript_id \"chrX.g8223.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '9637481', '9638128', '.', '-', '.', 'gene_id \"chr5.g2585\"; transcript_id \"chr5.g2585.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30801197', '30801235', '.', '-', '.', 'gene_id \"chrX.g8250\"; transcript_id \"chrX.g8250.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30840370', '30840417', '.', '+', '.', 'gene_id \"chrX.g8259\"; transcript_id \"chrX.g8259.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30872559', '30872948', '.', '+', '.', 'gene_id \"chrX.g8266\"; transcript_id \"chrX.g8266.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30872564', '30872948', '.', '+', '.', 'gene_id \"chrX.g8266\"; transcript_id \"chrX.g8266.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31066876', '31066911', '.', '-', '.', 'gene_id \"chrX.g8295\"; transcript_id \"chrX.g8295.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31164504', '31164648', '.', '+', '.', 'gene_id \"chrX.g8333\"; transcript_id \"chrX.g8333.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31597771', '31597820', '.', '+', '.', 'gene_id \"chrX.g8412\"; transcript_id \"chrX.g8412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10357329', '10357480', '.', '+', '.', 'gene_id \"chr5.g2802\"; transcript_id \"chr5.g2802.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31757434', '31757465', '.', '+', '.', 'gene_id \"chrX.g8435\"; transcript_id \"chrX.g8435.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31939268', '31939692', '.', '-', '.', 'gene_id \"chrX.g8455\"; transcript_id \"chrX.g8455.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32092809', '32092841', '.', '+', '.', 'gene_id \"chrX.g8507\"; transcript_id \"chrX.g8507.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32107053', '32107127', '.', '-', '.', 'gene_id \"chrX.g8510\"; transcript_id \"chrX.g8510.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32109131', '32109178', '.', '+', '.', 'gene_id \"chrX.g8512\"; transcript_id \"chrX.g8512.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10863294', '10863402', '.', '-', '.', 'gene_id \"chr5.g2924\"; transcript_id \"chr5.g2924.i14\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32140041', '32140160', '.', '+', '.', 'gene_id \"chrX.g8534\"; transcript_id \"chrX.g8534.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32140041', '32140161', '.', '+', '.', 'gene_id \"chrX.g8534\"; transcript_id \"chrX.g8534.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32140041', '32140150', '.', '+', '.', 'gene_id \"chrX.g8534\"; transcript_id \"chrX.g8534.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10881516', '10881711', '.', '+', '.', 'gene_id \"chr5.g2947\"; transcript_id \"chr5.g2947.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32330188', '32330239', '.', '+', '.', 'gene_id \"chrX.g8585\"; transcript_id \"chrX.g8585.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32346323', '32346363', '.', '+', '.', 'gene_id \"chrX.g8588\"; transcript_id \"chrX.g8588.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32636169', '32636209', '.', '+', '.', 'gene_id \"chrX.g8650\"; transcript_id \"chrX.g8650.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32878176', '32878255', '.', '+', '.', 'gene_id \"chrX.g8703\"; transcript_id \"chrX.g8703.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32970018', '32970160', '.', '-', '.', 'gene_id \"chrX.g8717\"; transcript_id \"chrX.g8717.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33008054', '33008422', '.', '+', '.', 'gene_id \"chrX.g8721\"; transcript_id \"chrX.g8721.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11153663', '11154345', '.', '-', '.', 'gene_id \"chr5.g3152\"; transcript_id \"chr5.g3152.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33088595', '33088775', '.', '-', '.', 'gene_id \"chrX.g8773\"; transcript_id \"chrX.g8773.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33089119', '33089479', '.', '-', '.', 'gene_id \"chrX.g8776\"; transcript_id \"chrX.g8776.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11272717', '11272756', '.', '-', '.', 'gene_id \"chr5.g3228\"; transcript_id \"chr5.g3228.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33181869', '33181918', '.', '+', '.', 'gene_id \"chrX.g8837\"; transcript_id \"chrX.g8837.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11281391', '11281638', '.', '+', '.', 'gene_id \"chr5.g3238\"; transcript_id \"chr5.g3238.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33260064', '33260339', '.', '-', '.', 'gene_id \"chrX.g8886\"; transcript_id \"chrX.g8886.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33290592', '33290680', '.', '-', '.', 'gene_id \"chrX.g8888\"; transcript_id \"chrX.g8888.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11356521', '11356567', '.', '-', '.', 'gene_id \"chr5.g3284\"; transcript_id \"chr5.g3284.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11356690', '11356742', '.', '+', '.', 'gene_id \"chr5.g3286\"; transcript_id \"chr5.g3286.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33507551', '33507618', '.', '+', '.', 'gene_id \"chrX.g8970\"; transcript_id \"chrX.g8970.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '33679624', '33680107', '.', '+', '.', 'gene_id \"chrX.g9021\"; transcript_id \"chrX.g9021.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11923846', '11923875', '.', '+', '.', 'gene_id \"chr5.g3497\"; transcript_id \"chr5.g3497.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '11977439', '11977592', '.', '+', '.', 'gene_id \"chr5.g3518\"; transcript_id \"chr5.g3518.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '12117041', '12117356', '.', '+', '.', 'gene_id \"chr5.g3578\"; transcript_id \"chr5.g3578.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '34811248', '34811283', '.', '+', '.', 'gene_id \"chrX.g9207\"; transcript_id \"chrX.g9207.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '34979899', '34980058', '.', '+', '.', 'gene_id \"chrX.g9236\"; transcript_id \"chrX.g9236.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35077424', '35077471', '.', '-', '.', 'gene_id \"chrX.g9249\"; transcript_id \"chrX.g9249.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35170227', '35170266', '.', '+', '.', 'gene_id \"chrX.g9260\"; transcript_id \"chrX.g9260.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35285649', '35285680', '.', '-', '.', 'gene_id \"chrX.g9357\"; transcript_id \"chrX.g9357.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35285649', '35285679', '.', '-', '.', 'gene_id \"chrX.g9357\"; transcript_id \"chrX.g9357.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35442051', '35442216', '.', '+', '.', 'gene_id \"chrX.g9422\"; transcript_id \"chrX.g9422.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35464324', '35464354', '.', '+', '.', 'gene_id \"chrX.g9439\"; transcript_id \"chrX.g9439.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35600575', '35600665', '.', '-', '.', 'gene_id \"chrX.g9492\"; transcript_id \"chrX.g9492.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35614612', '35614652', '.', '-', '.', 'gene_id \"chrX.g9498\"; transcript_id \"chrX.g9498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '13628137', '13628209', '.', '+', '.', 'gene_id \"chr5.g4046\"; transcript_id \"chr5.g4046.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35891709', '35891966', '.', '+', '.', 'gene_id \"chrX.g9672\"; transcript_id \"chrX.g9672.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '13648831', '13648874', '.', '+', '.', 'gene_id \"chr5.g4096\"; transcript_id \"chr5.g4096.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35965120', '35965164', '.', '+', '.', 'gene_id \"chrX.g9699\"; transcript_id \"chrX.g9699.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36297188', '36297347', '.', '-', '.', 'gene_id \"chrX.g9881\"; transcript_id \"chrX.g9881.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36756449', '36756481', '.', '+', '.', 'gene_id \"chrX.g9971\"; transcript_id \"chrX.g9971.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36806352', '36806399', '.', '+', '.', 'gene_id \"chrX.g9978\"; transcript_id \"chrX.g9978.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14350329', '14350480', '.', '+', '.', 'gene_id \"chr5.g4402\"; transcript_id \"chr5.g4402.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36984836', '36985008', '.', '-', '.', 'gene_id \"chrX.g10002\"; transcript_id \"chrX.g10002.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36994875', '36994917', '.', '-', '.', 'gene_id \"chrX.g10003\"; transcript_id \"chrX.g10003.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37138713', '37139016', '.', '-', '.', 'gene_id \"chrX.g10024\"; transcript_id \"chrX.g10024.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14460472', '14460504', '.', '-', '.', 'gene_id \"chr5.g4430\"; transcript_id \"chr5.g4430.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37375372', '37375594', '.', '+', '.', 'gene_id \"chrX.g10055\"; transcript_id \"chrX.g10055.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14566298', '14566348', '.', '+', '.', 'gene_id \"chr5.g4466\"; transcript_id \"chr5.g4466.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37535218', '37535262', '.', '+', '.', 'gene_id \"chrX.g10090\"; transcript_id \"chrX.g10090.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37771008', '37771039', '.', '-', '.', 'gene_id \"chrX.g10122\"; transcript_id \"chrX.g10122.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14693570', '14693617', '.', '+', '.', 'gene_id \"chr5.g4551\"; transcript_id \"chr5.g4551.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37920535', '37921439', '.', '+', '.', 'gene_id \"chrX.g10232\"; transcript_id \"chrX.g10232.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14905154', '14905967', '.', '-', '.', 'gene_id \"chr5.g4728\"; transcript_id \"chr5.g4728.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14905959', '14906347', '.', '+', '.', 'gene_id \"chr5.g4737\"; transcript_id \"chr5.g4737.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '14899658', '14900075', '.', '+', '.', 'gene_id \"chr5.g4746\"; transcript_id \"chr5.g4746.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38129046', '38129170', '.', '-', '.', 'gene_id \"chrX.g10317\"; transcript_id \"chrX.g10317.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38192504', '38192720', '.', '-', '.', 'gene_id \"chrX.g10323\"; transcript_id \"chrX.g10323.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38240585', '38240638', '.', '+', '.', 'gene_id \"chrX.g10329\"; transcript_id \"chrX.g10329.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38334993', '38335090', '.', '+', '.', 'gene_id \"chrX.g10343\"; transcript_id \"chrX.g10343.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38431595', '38431644', '.', '-', '.', 'gene_id \"chrX.g10371\"; transcript_id \"chrX.g10371.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38622211', '38622415', '.', '+', '.', 'gene_id \"chrX.g10410\"; transcript_id \"chrX.g10410.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38717422', '38718247', '.', '-', '.', 'gene_id \"chrX.g10429\"; transcript_id \"chrX.g10429.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '15273379', '15273412', '.', '+', '.', 'gene_id \"chr5.g4970\"; transcript_id \"chr5.g4970.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39293037', '39293068', '.', '-', '.', 'gene_id \"chrX.g10720\"; transcript_id \"chrX.g10720.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39347489', '39347544', '.', '+', '.', 'gene_id \"chrX.g10730\"; transcript_id \"chrX.g10730.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39366170', '39366201', '.', '+', '.', 'gene_id \"chrX.g10731\"; transcript_id \"chrX.g10731.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39387792', '39388040', '.', '+', '.', 'gene_id \"chrX.g10743\"; transcript_id \"chrX.g10743.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '15716471', '15717156', '.', '-', '.', 'gene_id \"chr5.g5374\"; transcript_id \"chr5.g5374.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39673419', '39673460', '.', '-', '.', 'gene_id \"chrX.g10803\"; transcript_id \"chrX.g10803.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39677312', '39677372', '.', '-', '.', 'gene_id \"chrX.g10811\"; transcript_id \"chrX.g10811.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39769621', '39769664', '.', '-', '.', 'gene_id \"chrX.g10853\"; transcript_id \"chrX.g10853.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40078185', '40078427', '.', '+', '.', 'gene_id \"chrX.g10916\"; transcript_id \"chrX.g10916.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40079773', '40079938', '.', '-', '.', 'gene_id \"chrX.g10918\"; transcript_id \"chrX.g10918.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16290951', '16291209', '.', '-', '.', 'gene_id \"chr5.g5553\"; transcript_id \"chr5.g5553.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16302269', '16302415', '.', '-', '.', 'gene_id \"chr5.g5562\"; transcript_id \"chr5.g5562.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40255524', '40256054', '.', '+', '.', 'gene_id \"chrX.g10967\"; transcript_id \"chrX.g10967.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40261788', '40261967', '.', '-', '.', 'gene_id \"chrX.g10974\"; transcript_id \"chrX.g10974.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40292673', '40293204', '.', '-', '.', 'gene_id \"chrX.g10999\"; transcript_id \"chrX.g10999.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16415620', '16415847', '.', '-', '.', 'gene_id \"chr5.g5641\"; transcript_id \"chr5.g5641.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40353242', '40353312', '.', '+', '.', 'gene_id \"chrX.g11054\"; transcript_id \"chrX.g11054.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40354951', '40355120', '.', '+', '.', 'gene_id \"chrX.g11060\"; transcript_id \"chrX.g11060.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40355156', '40355408', '.', '-', '.', 'gene_id \"chrX.g11062\"; transcript_id \"chrX.g11062.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16577624', '16577660', '.', '-', '.', 'gene_id \"chr5.g5741\"; transcript_id \"chr5.g5741.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40376817', '40378568', '.', '-', '.', 'gene_id \"chrX.g11087\"; transcript_id \"chrX.g11087.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40376817', '40378498', '.', '-', '.', 'gene_id \"chrX.g11087\"; transcript_id \"chrX.g11087.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40376817', '40378514', '.', '-', '.', 'gene_id \"chrX.g11087\"; transcript_id \"chrX.g11087.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40376817', '40377842', '.', '-', '.', 'gene_id \"chrX.g11087\"; transcript_id \"chrX.g11087.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16586012', '16586233', '.', '+', '.', 'gene_id \"chr5.g5759\"; transcript_id \"chr5.g5759.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40379436', '40379476', '.', '-', '.', 'gene_id \"chrX.g11089\"; transcript_id \"chrX.g11089.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40379424', '40379480', '.', '-', '.', 'gene_id \"chrX.g11089\"; transcript_id \"chrX.g11089.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40523557', '40523592', '.', '+', '.', 'gene_id \"chrX.g11128\"; transcript_id \"chrX.g11128.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40526377', '40526429', '.', '-', '.', 'gene_id \"chrX.g11130\"; transcript_id \"chrX.g11130.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40579891', '40580154', '.', '+', '.', 'gene_id \"chrX.g11135\"; transcript_id \"chrX.g11135.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40636993', '40637053', '.', '-', '.', 'gene_id \"chrX.g11160\"; transcript_id \"chrX.g11160.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40719731', '40719768', '.', '-', '.', 'gene_id \"chrX.g11176\"; transcript_id \"chrX.g11176.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40738182', '40738214', '.', '-', '.', 'gene_id \"chrX.g11180\"; transcript_id \"chrX.g11180.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40796646', '40796865', '.', '-', '.', 'gene_id \"chrX.g11205\"; transcript_id \"chrX.g11205.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40839465', '40839571', '.', '+', '.', 'gene_id \"chrX.g11244\"; transcript_id \"chrX.g11244.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40880220', '40880266', '.', '-', '.', 'gene_id \"chrX.g11275\"; transcript_id \"chrX.g11275.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16737630', '16737800', '.', '+', '.', 'gene_id \"chr5.g5955\"; transcript_id \"chr5.g5955.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16738370', '16738522', '.', '-', '.', 'gene_id \"chr5.g5957\"; transcript_id \"chr5.g5957.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41058875', '41058991', '.', '-', '.', 'gene_id \"chrX.g11370\"; transcript_id \"chrX.g11370.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41306013', '41306068', '.', '-', '.', 'gene_id \"chrX.g11484\"; transcript_id \"chrX.g11484.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41306438', '41306655', '.', '-', '.', 'gene_id \"chrX.g11486\"; transcript_id \"chrX.g11486.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41392786', '41392986', '.', '-', '.', 'gene_id \"chrX.g11551\"; transcript_id \"chrX.g11551.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41393078', '41393248', '.', '+', '.', 'gene_id \"chrX.g11552\"; transcript_id \"chrX.g11552.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41468175', '41468214', '.', '+', '.', 'gene_id \"chrX.g11583\"; transcript_id \"chrX.g11583.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '18092375', '18092718', '.', '+', '.', 'gene_id \"chr5.g6304\"; transcript_id \"chr5.g6304.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41775849', '41776275', '.', '-', '.', 'gene_id \"chrX.g11720\"; transcript_id \"chrX.g11720.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41825791', '41826059', '.', '+', '.', 'gene_id \"chrX.g11735\"; transcript_id \"chrX.g11735.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '18239613', '18239782', '.', '-', '.', 'gene_id \"chr5.g6368\"; transcript_id \"chr5.g6368.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '18239941', '18240197', '.', '-', '.', 'gene_id \"chr5.g6369\"; transcript_id \"chr5.g6369.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '18885233', '18885294', '.', '-', '.', 'gene_id \"chr5.g6498\"; transcript_id \"chr5.g6498.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42137965', '42138213', '.', '-', '.', 'gene_id \"chrX.g11944\"; transcript_id \"chrX.g11944.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42137801', '42137980', '.', '+', '.', 'gene_id \"chrX.g11945\"; transcript_id \"chrX.g11945.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42150809', '42150856', '.', '+', '.', 'gene_id \"chrX.g11951\"; transcript_id \"chrX.g11951.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '19037650', '19037894', '.', '-', '.', 'gene_id \"chr5.g6557\"; transcript_id \"chr5.g6557.i4\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42155256', '42155482', '.', '+', '.', 'gene_id \"chrX.g11956\"; transcript_id \"chrX.g11956.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42168207', '42168362', '.', '+', '.', 'gene_id \"chrX.g11962\"; transcript_id \"chrX.g11962.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '19105675', '19106355', '.', '+', '.', 'gene_id \"chr5.g6568\"; transcript_id \"chr5.g6568.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42648679', '42649544', '.', '-', '.', 'gene_id \"chrX.g12274\"; transcript_id \"chrX.g12274.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42648677', '42649102', '.', '-', '.', 'gene_id \"chrX.g12274\"; transcript_id \"chrX.g12274.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '19446249', '19446287', '.', '-', '.', 'gene_id \"chr5.g6905\"; transcript_id \"chr5.g6905.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43053304', '43054526', '.', '+', '.', 'gene_id \"chrX.g12355\"; transcript_id \"chrX.g12355.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43373116', '43373344', '.', '+', '.', 'gene_id \"chrX.g12414\"; transcript_id \"chrX.g12414.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43391719', '43392599', '.', '-', '.', 'gene_id \"chrX.g12420\"; transcript_id \"chrX.g12420.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43393963', '43394083', '.', '+', '.', 'gene_id \"chrX.g12421\"; transcript_id \"chrX.g12421.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43630460', '43630580', '.', '+', '.', 'gene_id \"chrX.g12485\"; transcript_id \"chrX.g12485.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43742586', '43743069', '.', '-', '.', 'gene_id \"chrX.g12503\"; transcript_id \"chrX.g12503.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43968089', '43968372', '.', '-', '.', 'gene_id \"chrX.g12546\"; transcript_id \"chrX.g12546.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '20083794', '20083823', '.', '+', '.', 'gene_id \"chr5.g7167\"; transcript_id \"chr5.g7167.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44187339', '44187493', '.', '+', '.', 'gene_id \"chrX.g12589\"; transcript_id \"chrX.g12589.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '20365567', '20365607', '.', '+', '.', 'gene_id \"chr5.g7188\"; transcript_id \"chr5.g7188.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44390235', '44390479', '.', '-', '.', 'gene_id \"chrX.g12660\"; transcript_id \"chrX.g12660.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21003077', '21003483', '.', '+', '.', 'gene_id \"chr5.g7307\"; transcript_id \"chr5.g7307.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21057448', '21057646', '.', '-', '.', 'gene_id \"chr5.g7329\"; transcript_id \"chr5.g7329.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21056841', '21057259', '.', '-', '.', 'gene_id \"chr5.g7336\"; transcript_id \"chr5.g7336.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44629754', '44629799', '.', '+', '.', 'gene_id \"chrX.g12751\"; transcript_id \"chrX.g12751.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44709247', '44709285', '.', '+', '.', 'gene_id \"chrX.g12804\"; transcript_id \"chrX.g12804.i10\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21236560', '21236841', '.', '+', '.', 'gene_id \"chr5.g7530\"; transcript_id \"chr5.g7530.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44906947', '44906982', '.', '-', '.', 'gene_id \"chrX.g12931\"; transcript_id \"chrX.g12931.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44906949', '44906986', '.', '+', '.', 'gene_id \"chrX.g12932\"; transcript_id \"chrX.g12932.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21280356', '21280763', '.', '-', '.', 'gene_id \"chr5.g7596\"; transcript_id \"chr5.g7596.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44995630', '44995665', '.', '+', '.', 'gene_id \"chrX.g12992\"; transcript_id \"chrX.g12992.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45197190', '45197226', '.', '-', '.', 'gene_id \"chrX.g13066\"; transcript_id \"chrX.g13066.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21402868', '21403127', '.', '+', '.', 'gene_id \"chr5.g7680\"; transcript_id \"chr5.g7680.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45213279', '45213702', '.', '+', '.', 'gene_id \"chrX.g13075\"; transcript_id \"chrX.g13075.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45213292', '45213702', '.', '+', '.', 'gene_id \"chrX.g13075\"; transcript_id \"chrX.g13075.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45366704', '45366743', '.', '+', '.', 'gene_id \"chrX.g13103\"; transcript_id \"chrX.g13103.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21432513', '21432696', '.', '-', '.', 'gene_id \"chr5.g7733\"; transcript_id \"chr5.g7733.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21463803', '21464008', '.', '+', '.', 'gene_id \"chr5.g7757\"; transcript_id \"chr5.g7757.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45531688', '45531888', '.', '+', '.', 'gene_id \"chrX.g13194\"; transcript_id \"chrX.g13194.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45534291', '45534466', '.', '+', '.', 'gene_id \"chrX.g13196\"; transcript_id \"chrX.g13196.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45534324', '45534561', '.', '+', '.', 'gene_id \"chrX.g13196\"; transcript_id \"chrX.g13196.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45879759', '45879801', '.', '-', '.', 'gene_id \"chrX.g13323\"; transcript_id \"chrX.g13323.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45909565', '45909829', '.', '-', '.', 'gene_id \"chrX.g13332\"; transcript_id \"chrX.g13332.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '21966615', '21966648', '.', '+', '.', 'gene_id \"chr5.g7969\"; transcript_id \"chr5.g7969.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45914618', '45914651', '.', '-', '.', 'gene_id \"chrX.g13333\"; transcript_id \"chrX.g13333.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45926032', '45926061', '.', '+', '.', 'gene_id \"chrX.g13338\"; transcript_id \"chrX.g13338.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22175237', '22175289', '.', '+', '.', 'gene_id \"chr5.g7988\"; transcript_id \"chr5.g7988.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46020034', '46020398', '.', '-', '.', 'gene_id \"chrX.g13370\"; transcript_id \"chrX.g13370.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22349437', '22349478', '.', '+', '.', 'gene_id \"chr5.g8025\"; transcript_id \"chr5.g8025.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22367744', '22367844', '.', '+', '.', 'gene_id \"chr5.g8038\"; transcript_id \"chr5.g8038.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22521317', '22521573', '.', '-', '.', 'gene_id \"chr5.g8083\"; transcript_id \"chr5.g8083.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46398850', '46399103', '.', '+', '.', 'gene_id \"chrX.g13498\"; transcript_id \"chrX.g13498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22617865', '22618226', '.', '-', '.', 'gene_id \"chr5.g8210\"; transcript_id \"chr5.g8210.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22618275', '22618484', '.', '+', '.', 'gene_id \"chr5.g8211\"; transcript_id \"chr5.g8211.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46529087', '46529128', '.', '-', '.', 'gene_id \"chrX.g13586\"; transcript_id \"chrX.g13586.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46959079', '46959591', '.', '+', '.', 'gene_id \"chrX.g13669\"; transcript_id \"chrX.g13669.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47185441', '47185708', '.', '+', '.', 'gene_id \"chrX.g13695\"; transcript_id \"chrX.g13695.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47194488', '47194534', '.', '+', '.', 'gene_id \"chrX.g13697\"; transcript_id \"chrX.g13697.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47419879', '47420276', '.', '+', '.', 'gene_id \"chrX.g13759\"; transcript_id \"chrX.g13759.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22821044', '22821195', '.', '-', '.', 'gene_id \"chr5.g8429\"; transcript_id \"chr5.g8429.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47421034', '47421183', '.', '+', '.', 'gene_id \"chrX.g13761\"; transcript_id \"chrX.g13761.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47427141', '47427353', '.', '+', '.', 'gene_id \"chrX.g13764\"; transcript_id \"chrX.g13764.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22885888', '22885939', '.', '+', '.', 'gene_id \"chr5.g8444\"; transcript_id \"chr5.g8444.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22871116', '22871229', '.', '+', '.', 'gene_id \"chr5.g8455\"; transcript_id \"chr5.g8455.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47560411', '47560926', '.', '-', '.', 'gene_id \"chrX.g13842\"; transcript_id \"chrX.g13842.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47613355', '47613505', '.', '+', '.', 'gene_id \"chrX.g13865\"; transcript_id \"chrX.g13865.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47820089', '47820254', '.', '-', '.', 'gene_id \"chrX.g13889\"; transcript_id \"chrX.g13889.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47860769', '47860809', '.', '-', '.', 'gene_id \"chrX.g13896\"; transcript_id \"chrX.g13896.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47957156', '47957193', '.', '+', '.', 'gene_id \"chrX.g13913\"; transcript_id \"chrX.g13913.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '23800637', '23800841', '.', '-', '.', 'gene_id \"chr5.g8578\"; transcript_id \"chr5.g8578.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '23917812', '23917843', '.', '-', '.', 'gene_id \"chr5.g8598\"; transcript_id \"chr5.g8598.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48166124', '48166221', '.', '+', '.', 'gene_id \"chrX.g13967\"; transcript_id \"chrX.g13967.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48268051', '48268259', '.', '-', '.', 'gene_id \"chrX.g13992\"; transcript_id \"chrX.g13992.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48531855', '48532122', '.', '+', '.', 'gene_id \"chrX.g14059\"; transcript_id \"chrX.g14059.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48621517', '48621568', '.', '+', '.', 'gene_id \"chrX.g14073\"; transcript_id \"chrX.g14073.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48637840', '48637878', '.', '+', '.', 'gene_id \"chrX.g14075\"; transcript_id \"chrX.g14075.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24191375', '24191501', '.', '+', '.', 'gene_id \"chr5.g8761\"; transcript_id \"chr5.g8761.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48875540', '48875618', '.', '-', '.', 'gene_id \"chrX.g14116\"; transcript_id \"chrX.g14116.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24369766', '24369798', '.', '-', '.', 'gene_id \"chr5.g8831\"; transcript_id \"chr5.g8831.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49197182', '49197403', '.', '+', '.', 'gene_id \"chrX.g14179\"; transcript_id \"chrX.g14179.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49200572', '49200623', '.', '+', '.', 'gene_id \"chrX.g14180\"; transcript_id \"chrX.g14180.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49548003', '49548315', '.', '+', '.', 'gene_id \"chrX.g14245\"; transcript_id \"chrX.g14245.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24506272', '24506312', '.', '+', '.', 'gene_id \"chr5.g8956\"; transcript_id \"chr5.g8956.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24666678', '24666745', '.', '+', '.', 'gene_id \"chr5.g9028\"; transcript_id \"chr5.g9028.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49803364', '49803477', '.', '+', '.', 'gene_id \"chrX.g14351\"; transcript_id \"chrX.g14351.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49902523', '49902574', '.', '-', '.', 'gene_id \"chrX.g14359\"; transcript_id \"chrX.g14359.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49937508', '49937545', '.', '-', '.', 'gene_id \"chrX.g14362\"; transcript_id \"chrX.g14362.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50303832', '50304060', '.', '+', '.', 'gene_id \"chrX.g14428\"; transcript_id \"chrX.g14428.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50360245', '50360369', '.', '+', '.', 'gene_id \"chrX.g14440\"; transcript_id \"chrX.g14440.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50905095', '50905559', '.', '+', '.', 'gene_id \"chrX.g14480\"; transcript_id \"chrX.g14480.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50985496', '50985651', '.', '+', '.', 'gene_id \"chrX.g14504\"; transcript_id \"chrX.g14504.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50985502', '50985696', '.', '+', '.', 'gene_id \"chrX.g14504\"; transcript_id \"chrX.g14504.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50985569', '50985740', '.', '-', '.', 'gene_id \"chrX.g14505\"; transcript_id \"chrX.g14505.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51143346', '51143693', '.', '+', '.', 'gene_id \"chrX.g14539\"; transcript_id \"chrX.g14539.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51247177', '51247303', '.', '+', '.', 'gene_id \"chrX.g14548\"; transcript_id \"chrX.g14548.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51264936', '51265062', '.', '+', '.', 'gene_id \"chrX.g14551\"; transcript_id \"chrX.g14551.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51466922', '51466958', '.', '-', '.', 'gene_id \"chrX.g14610\"; transcript_id \"chrX.g14610.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51576781', '51577165', '.', '-', '.', 'gene_id \"chrX.g14621\"; transcript_id \"chrX.g14621.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51592568', '51592724', '.', '+', '.', 'gene_id \"chrX.g14623\"; transcript_id \"chrX.g14623.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51604551', '51604589', '.', '+', '.', 'gene_id \"chrX.g14625\"; transcript_id \"chrX.g14625.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25105670', '25105893', '.', '-', '.', 'gene_id \"chr5.g9339\"; transcript_id \"chr5.g9339.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51780093', '51780365', '.', '+', '.', 'gene_id \"chrX.g14655\"; transcript_id \"chrX.g14655.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25114588', '25114637', '.', '+', '.', 'gene_id \"chr5.g9347\"; transcript_id \"chr5.g9347.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51783050', '51784931', '.', '+', '.', 'gene_id \"chrX.g14657\"; transcript_id \"chrX.g14657.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25121030', '25121218', '.', '+', '.', 'gene_id \"chr5.g9352\"; transcript_id \"chr5.g9352.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51849771', '51849976', '.', '+', '.', 'gene_id \"chrX.g14669\"; transcript_id \"chrX.g14669.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25317060', '25317108', '.', '-', '.', 'gene_id \"chr5.g9388\"; transcript_id \"chr5.g9388.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52113291', '52113325', '.', '+', '.', 'gene_id \"chrX.g14739\"; transcript_id \"chrX.g14739.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25912425', '25913694', '.', '+', '.', 'gene_id \"chr5.g9484\"; transcript_id \"chr5.g9484.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52278744', '52278776', '.', '+', '.', 'gene_id \"chrX.g14808\"; transcript_id \"chrX.g14808.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '26283380', '26283443', '.', '-', '.', 'gene_id \"chr5.g9549\"; transcript_id \"chr5.g9549.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52731752', '52731854', '.', '+', '.', 'gene_id \"chrX.g14856\"; transcript_id \"chrX.g14856.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52914359', '52914533', '.', '+', '.', 'gene_id \"chrX.g14878\"; transcript_id \"chrX.g14878.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52914695', '52914881', '.', '+', '.', 'gene_id \"chrX.g14879\"; transcript_id \"chrX.g14879.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53048059', '53048209', '.', '-', '.', 'gene_id \"chrX.g14912\"; transcript_id \"chrX.g14912.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53066294', '53066361', '.', '+', '.', 'gene_id \"chrX.g14924\"; transcript_id \"chrX.g14924.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53114099', '53114738', '.', '-', '.', 'gene_id \"chrX.g14942\"; transcript_id \"chrX.g14942.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53200759', '53201016', '.', '+', '.', 'gene_id \"chrX.g14975\"; transcript_id \"chrX.g14975.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '26618560', '26618598', '.', '+', '.', 'gene_id \"chr5.g9696\"; transcript_id \"chr5.g9696.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53230623', '53230844', '.', '-', '.', 'gene_id \"chrX.g14989\"; transcript_id \"chrX.g14989.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53722475', '53722563', '.', '+', '.', 'gene_id \"chrX.g15184\"; transcript_id \"chrX.g15184.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53760001', '53760037', '.', '+', '.', 'gene_id \"chrX.g15196\"; transcript_id \"chrX.g15196.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53859730', '53859771', '.', '+', '.', 'gene_id \"chrX.g15249\"; transcript_id \"chrX.g15249.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54233060', '54233349', '.', '+', '.', 'gene_id \"chrX.g15373\"; transcript_id \"chrX.g15373.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '28015130', '28015164', '.', '-', '.', 'gene_id \"chr5.g10101\"; transcript_id \"chr5.g10101.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54260288', '54260329', '.', '+', '.', 'gene_id \"chrX.g15390\"; transcript_id \"chrX.g15390.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54276936', '54276965', '.', '+', '.', 'gene_id \"chrX.g15399\"; transcript_id \"chrX.g15399.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54305300', '54305614', '.', '-', '.', 'gene_id \"chrX.g15410\"; transcript_id \"chrX.g15410.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54307781', '54307820', '.', '+', '.', 'gene_id \"chrX.g15413\"; transcript_id \"chrX.g15413.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54313864', '54313906', '.', '+', '.', 'gene_id \"chrX.g15419\"; transcript_id \"chrX.g15419.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54335766', '54336197', '.', '-', '.', 'gene_id \"chrX.g15430\"; transcript_id \"chrX.g15430.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54335786', '54336197', '.', '-', '.', 'gene_id \"chrX.g15430\"; transcript_id \"chrX.g15430.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54444724', '54444781', '.', '-', '.', 'gene_id \"chrX.g15467\"; transcript_id \"chrX.g15467.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54445174', '54445216', '.', '-', '.', 'gene_id \"chrX.g15468\"; transcript_id \"chrX.g15468.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '28491109', '28491360', '.', '-', '.', 'gene_id \"chr5.g10309\"; transcript_id \"chr5.g10309.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54564253', '54564494', '.', '-', '.', 'gene_id \"chrX.g15589\"; transcript_id \"chrX.g15589.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54563951', '54564269', '.', '-', '.', 'gene_id \"chrX.g15591\"; transcript_id \"chrX.g15591.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '28641574', '28641811', '.', '+', '.', 'gene_id \"chr5.g10336\"; transcript_id \"chr5.g10336.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54598156', '54598370', '.', '-', '.', 'gene_id \"chrX.g15617\"; transcript_id \"chrX.g15617.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54860765', '54860797', '.', '-', '.', 'gene_id \"chrX.g15797\"; transcript_id \"chrX.g15797.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54905720', '54905936', '.', '-', '.', 'gene_id \"chrX.g15814\"; transcript_id \"chrX.g15814.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55088230', '55088500', '.', '-', '.', 'gene_id \"chrX.g15921\"; transcript_id \"chrX.g15921.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55126042', '55126149', '.', '+', '.', 'gene_id \"chrX.g15955\"; transcript_id \"chrX.g15955.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29512561', '29512779', '.', '-', '.', 'gene_id \"chr5.g10716\"; transcript_id \"chr5.g10716.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29513343', '29513648', '.', '+', '.', 'gene_id \"chr5.g10717\"; transcript_id \"chr5.g10717.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55196774', '55196872', '.', '+', '.', 'gene_id \"chrX.g15987\"; transcript_id \"chrX.g15987.i7\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55213772', '55213969', '.', '+', '.', 'gene_id \"chrX.g16008\"; transcript_id \"chrX.g16008.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55253599', '55255786', '.', '-', '.', 'gene_id \"chrX.g16020\"; transcript_id \"chrX.g16020.i7\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55256567', '55256844', '.', '+', '.', 'gene_id \"chrX.g16021\"; transcript_id \"chrX.g16021.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55258229', '55258428', '.', '-', '.', 'gene_id \"chrX.g16025\"; transcript_id \"chrX.g16025.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55309088', '55309304', '.', '-', '.', 'gene_id \"chrX.g16043\"; transcript_id \"chrX.g16043.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55335097', '55335254', '.', '+', '.', 'gene_id \"chrX.g16065\"; transcript_id \"chrX.g16065.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55335686', '55335919', '.', '+', '.', 'gene_id \"chrX.g16066\"; transcript_id \"chrX.g16066.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55335931', '55336389', '.', '+', '.', 'gene_id \"chrX.g16067\"; transcript_id \"chrX.g16067.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55337805', '55337857', '.', '+', '.', 'gene_id \"chrX.g16069\"; transcript_id \"chrX.g16069.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29630462', '29630519', '.', '-', '.', 'gene_id \"chr5.g10820\"; transcript_id \"chr5.g10820.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55408017', '55408169', '.', '+', '.', 'gene_id \"chrX.g16129\"; transcript_id \"chrX.g16129.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29685079', '29685391', '.', '+', '.', 'gene_id \"chr5.g10863\"; transcript_id \"chr5.g10863.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29735394', '29735568', '.', '+', '.', 'gene_id \"chr5.g10907\"; transcript_id \"chr5.g10907.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55492212', '55492275', '.', '+', '.', 'gene_id \"chrX.g16199\"; transcript_id \"chrX.g16199.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29779816', '29779849', '.', '+', '.', 'gene_id \"chr5.g10938\"; transcript_id \"chr5.g10938.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29808348', '29808758', '.', '+', '.', 'gene_id \"chr5.g10950\"; transcript_id \"chr5.g10950.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30075289', '30075350', '.', '-', '.', 'gene_id \"chr5.g11009\"; transcript_id \"chr5.g11009.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30197416', '30197536', '.', '+', '.', 'gene_id \"chr5.g11043\"; transcript_id \"chr5.g11043.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55646440', '55646617', '.', '+', '.', 'gene_id \"chrX.g16350\"; transcript_id \"chrX.g16350.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55755182', '55755639', '.', '-', '.', 'gene_id \"chrX.g16421\"; transcript_id \"chrX.g16421.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55852454', '55852506', '.', '+', '.', 'gene_id \"chrX.g16443\"; transcript_id \"chrX.g16443.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55893320', '55893366', '.', '-', '.', 'gene_id \"chrX.g16458\"; transcript_id \"chrX.g16458.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55900807', '55901921', '.', '-', '.', 'gene_id \"chrX.g16461\"; transcript_id \"chrX.g16461.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30352630', '30352819', '.', '+', '.', 'gene_id \"chr5.g11224\"; transcript_id \"chr5.g11224.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56230726', '56230814', '.', '+', '.', 'gene_id \"chrX.g16522\"; transcript_id \"chrX.g16522.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56416013', '56416174', '.', '+', '.', 'gene_id \"chrX.g16595\"; transcript_id \"chrX.g16595.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30463119', '30463467', '.', '+', '.', 'gene_id \"chr5.g11349\"; transcript_id \"chr5.g11349.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30478723', '30479035', '.', '+', '.', 'gene_id \"chr5.g11365\"; transcript_id \"chr5.g11365.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30490885', '30491136', '.', '+', '.', 'gene_id \"chr5.g11380\"; transcript_id \"chr5.g11380.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30491182', '30491333', '.', '-', '.', 'gene_id \"chr5.g11381\"; transcript_id \"chr5.g11381.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30491398', '30491584', '.', '-', '.', 'gene_id \"chr5.g11382\"; transcript_id \"chr5.g11382.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30491796', '30491954', '.', '-', '.', 'gene_id \"chr5.g11383\"; transcript_id \"chr5.g11383.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30492007', '30492170', '.', '-', '.', 'gene_id \"chr5.g11384\"; transcript_id \"chr5.g11384.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56475164', '56475206', '.', '+', '.', 'gene_id \"chrX.g16670\"; transcript_id \"chrX.g16670.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56536932', '56537196', '.', '-', '.', 'gene_id \"chrX.g16738\"; transcript_id \"chrX.g16738.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30770398', '30770433', '.', '-', '.', 'gene_id \"chr5.g11546\"; transcript_id \"chr5.g11546.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56620057', '56620697', '.', '-', '.', 'gene_id \"chrX.g16786\"; transcript_id \"chrX.g16786.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56697233', '56697327', '.', '-', '.', 'gene_id \"chrX.g16801\"; transcript_id \"chrX.g16801.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56721367', '56721746', '.', '+', '.', 'gene_id \"chrX.g16810\"; transcript_id \"chrX.g16810.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30801102', '30801605', '.', '+', '.', 'gene_id \"chr5.g11596\"; transcript_id \"chr5.g11596.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30800824', '30801116', '.', '+', '.', 'gene_id \"chr5.g11597\"; transcript_id \"chr5.g11597.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30862686', '30862979', '.', '-', '.', 'gene_id \"chr5.g11614\"; transcript_id \"chr5.g11614.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56792808', '56793019', '.', '+', '.', 'gene_id \"chrX.g16826\"; transcript_id \"chrX.g16826.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31017176', '31017326', '.', '-', '.', 'gene_id \"chr5.g11679\"; transcript_id \"chr5.g11679.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31017361', '31017536', '.', '+', '.', 'gene_id \"chr5.g11680\"; transcript_id \"chr5.g11680.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31053573', '31053777', '.', '-', '.', 'gene_id \"chr5.g11696\"; transcript_id \"chr5.g11696.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56886928', '56886957', '.', '-', '.', 'gene_id \"chrX.g16910\"; transcript_id \"chrX.g16910.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31174777', '31175027', '.', '+', '.', 'gene_id \"chr5.g11712\"; transcript_id \"chr5.g11712.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57177664', '57177892', '.', '-', '.', 'gene_id \"chrX.g16938\"; transcript_id \"chrX.g16938.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57481235', '57481279', '.', '-', '.', 'gene_id \"chrX.g17030\"; transcript_id \"chrX.g17030.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57482293', '57482527', '.', '-', '.', 'gene_id \"chrX.g17032\"; transcript_id \"chrX.g17032.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57553427', '57553458', '.', '+', '.', 'gene_id \"chrX.g17045\"; transcript_id \"chrX.g17045.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57640990', '57641058', '.', '+', '.', 'gene_id \"chrX.g17068\"; transcript_id \"chrX.g17068.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57755093', '57755382', '.', '+', '.', 'gene_id \"chrX.g17085\"; transcript_id \"chrX.g17085.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31600058', '31600306', '.', '+', '.', 'gene_id \"chr5.g11965\"; transcript_id \"chr5.g11965.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57983691', '57984332', '.', '+', '.', 'gene_id \"chrX.g17135\"; transcript_id \"chrX.g17135.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58084230', '58084496', '.', '+', '.', 'gene_id \"chrX.g17159\"; transcript_id \"chrX.g17159.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31631022', '31631412', '.', '+', '.', 'gene_id \"chr5.g12014\"; transcript_id \"chr5.g12014.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58378295', '58378508', '.', '+', '.', 'gene_id \"chrX.g17208\"; transcript_id \"chrX.g17208.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58379506', '58379677', '.', '-', '.', 'gene_id \"chrX.g17211\"; transcript_id \"chrX.g17211.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58429744', '58429823', '.', '-', '.', 'gene_id \"chrX.g17225\"; transcript_id \"chrX.g17225.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31759628', '31759885', '.', '-', '.', 'gene_id \"chr5.g12131\"; transcript_id \"chr5.g12131.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31759870', '31760093', '.', '+', '.', 'gene_id \"chr5.g12132\"; transcript_id \"chr5.g12132.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31762361', '31762537', '.', '-', '.', 'gene_id \"chr5.g12138\"; transcript_id \"chr5.g12138.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58919514', '58919829', '.', '-', '.', 'gene_id \"chrX.g17351\"; transcript_id \"chrX.g17351.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59038909', '59038963', '.', '-', '.', 'gene_id \"chrX.g17380\"; transcript_id \"chrX.g17380.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31924092', '31924262', '.', '+', '.', 'gene_id \"chr5.g12309\"; transcript_id \"chr5.g12309.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31924283', '31924482', '.', '+', '.', 'gene_id \"chr5.g12310\"; transcript_id \"chr5.g12310.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59130696', '59130917', '.', '+', '.', 'gene_id \"chrX.g17399\"; transcript_id \"chrX.g17399.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59075983', '59076021', '.', '-', '.', 'gene_id \"chrX.g17406\"; transcript_id \"chrX.g17406.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59620479', '59620514', '.', '-', '.', 'gene_id \"chrX.g17530\"; transcript_id \"chrX.g17530.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59961850', '59961886', '.', '+', '.', 'gene_id \"chrX.g17604\"; transcript_id \"chrX.g17604.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '32458997', '32459037', '.', '+', '.', 'gene_id \"chr5.g12699\"; transcript_id \"chr5.g12699.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '32514094', '32514179', '.', '-', '.', 'gene_id \"chr5.g12756\"; transcript_id \"chr5.g12756.i7\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '61421658', '61421771', '.', '+', '.', 'gene_id \"chrX.g17803\"; transcript_id \"chrX.g17803.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '61776454', '61776623', '.', '-', '.', 'gene_id \"chrX.g17887\"; transcript_id \"chrX.g17887.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '61852435', '61852494', '.', '+', '.', 'gene_id \"chrX.g17893\"; transcript_id \"chrX.g17893.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '62188324', '62188356', '.', '-', '.', 'gene_id \"chrX.g17919\"; transcript_id \"chrX.g17919.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '32897458', '32897520', '.', '+', '.', 'gene_id \"chr5.g13064\"; transcript_id \"chr5.g13064.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63306705', '63306800', '.', '+', '.', 'gene_id \"chrX.g17996\"; transcript_id \"chrX.g17996.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63606124', '63606298', '.', '+', '.', 'gene_id \"chrX.g18028\"; transcript_id \"chrX.g18028.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63734296', '63734363', '.', '+', '.', 'gene_id \"chrX.g18040\"; transcript_id \"chrX.g18040.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63754764', '63754793', '.', '-', '.', 'gene_id \"chrX.g18043\"; transcript_id \"chrX.g18043.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63761179', '63761246', '.', '+', '.', 'gene_id \"chrX.g18044\"; transcript_id \"chrX.g18044.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63970663', '63970694', '.', '-', '.', 'gene_id \"chrX.g18068\"; transcript_id \"chrX.g18068.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64088750', '64089226', '.', '+', '.', 'gene_id \"chrX.g18085\"; transcript_id \"chrX.g18085.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64037257', '64037315', '.', '+', '.', 'gene_id \"chrX.g18087\"; transcript_id \"chrX.g18087.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64280100', '64280141', '.', '+', '.', 'gene_id \"chrX.g18135\"; transcript_id \"chrX.g18135.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64451699', '64451891', '.', '+', '.', 'gene_id \"chrX.g18154\"; transcript_id \"chrX.g18154.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64451455', '64452297', '.', '-', '.', 'gene_id \"chrX.g18155\"; transcript_id \"chrX.g18155.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64821445', '64821640', '.', '-', '.', 'gene_id \"chrX.g18198\"; transcript_id \"chrX.g18198.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33376587', '33376910', '.', '+', '.', 'gene_id \"chr5.g13410\"; transcript_id \"chr5.g13410.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33551780', '33552038', '.', '+', '.', 'gene_id \"chr5.g13471\"; transcript_id \"chr5.g13471.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65470249', '65470295', '.', '+', '.', 'gene_id \"chrX.g18335\"; transcript_id \"chrX.g18335.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33710562', '33710782', '.', '-', '.', 'gene_id \"chr5.g13574\"; transcript_id \"chr5.g13574.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33710375', '33710579', '.', '+', '.', 'gene_id \"chr5.g13575\"; transcript_id \"chr5.g13575.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33716114', '33716271', '.', '-', '.', 'gene_id \"chr5.g13583\"; transcript_id \"chr5.g13583.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65924549', '65924762', '.', '+', '.', 'gene_id \"chrX.g18431\"; transcript_id \"chrX.g18431.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66216722', '66216766', '.', '+', '.', 'gene_id \"chrX.g18460\"; transcript_id \"chrX.g18460.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66332160', '66332222', '.', '-', '.', 'gene_id \"chrX.g18466\"; transcript_id \"chrX.g18466.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66447401', '66447451', '.', '+', '.', 'gene_id \"chrX.g18472\"; transcript_id \"chrX.g18472.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33831736', '33831792', '.', '+', '.', 'gene_id \"chr5.g13667\"; transcript_id \"chr5.g13667.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33865679', '33865710', '.', '-', '.', 'gene_id \"chr5.g13682\"; transcript_id \"chr5.g13682.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33883937', '33884184', '.', '-', '.', 'gene_id \"chr5.g13690\"; transcript_id \"chr5.g13690.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66641718', '66641895', '.', '+', '.', 'gene_id \"chrX.g18502\"; transcript_id \"chrX.g18502.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66833799', '66834490', '.', '+', '.', 'gene_id \"chrX.g18523\"; transcript_id \"chrX.g18523.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33908636', '33908785', '.', '-', '.', 'gene_id \"chr5.g13736\"; transcript_id \"chr5.g13736.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67342635', '67342668', '.', '-', '.', 'gene_id \"chrX.g18592\"; transcript_id \"chrX.g18592.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67599558', '67599707', '.', '-', '.', 'gene_id \"chrX.g18637\"; transcript_id \"chrX.g18637.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67690962', '67691148', '.', '-', '.', 'gene_id \"chrX.g18659\"; transcript_id \"chrX.g18659.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '34058570', '34058625', '.', '-', '.', 'gene_id \"chr5.g13884\"; transcript_id \"chr5.g13884.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67942805', '67942942', '.', '-', '.', 'gene_id \"chrX.g18697\"; transcript_id \"chrX.g18697.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67942805', '67942997', '.', '-', '.', 'gene_id \"chrX.g18697\"; transcript_id \"chrX.g18697.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67998924', '67999443', '.', '-', '.', 'gene_id \"chrX.g18709\"; transcript_id \"chrX.g18709.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68069483', '68069528', '.', '+', '.', 'gene_id \"chrX.g18714\"; transcript_id \"chrX.g18714.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68094797', '68094844', '.', '-', '.', 'gene_id \"chrX.g18717\"; transcript_id \"chrX.g18717.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68336950', '68337110', '.', '+', '.', 'gene_id \"chrX.g18752\"; transcript_id \"chrX.g18752.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68337097', '68337891', '.', '+', '.', 'gene_id \"chrX.g18753\"; transcript_id \"chrX.g18753.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68379369', '68379435', '.', '+', '.', 'gene_id \"chrX.g18763\"; transcript_id \"chrX.g18763.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '34286811', '34286905', '.', '+', '.', 'gene_id \"chr5.g13990\"; transcript_id \"chr5.g13990.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68846909', '68847024', '.', '+', '.', 'gene_id \"chrX.g18798\"; transcript_id \"chrX.g18798.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68857227', '68857294', '.', '+', '.', 'gene_id \"chrX.g18803\"; transcript_id \"chrX.g18803.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69134926', '69135081', '.', '+', '.', 'gene_id \"chrX.g18874\"; transcript_id \"chrX.g18874.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69134932', '69135126', '.', '+', '.', 'gene_id \"chrX.g18874\"; transcript_id \"chrX.g18874.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '34538921', '34539156', '.', '+', '.', 'gene_id \"chr5.g14107\"; transcript_id \"chr5.g14107.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '34539264', '34539445', '.', '+', '.', 'gene_id \"chr5.g14108\"; transcript_id \"chr5.g14108.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '34618521', '34618594', '.', '-', '.', 'gene_id \"chr5.g14153\"; transcript_id \"chr5.g14153.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69499675', '69499721', '.', '+', '.', 'gene_id \"chrX.g18998\"; transcript_id \"chrX.g18998.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69654597', '69654758', '.', '+', '.', 'gene_id \"chrX.g19051\"; transcript_id \"chrX.g19051.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '35549206', '35549249', '.', '-', '.', 'gene_id \"chr5.g14402\"; transcript_id \"chr5.g14402.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '35853573', '35853631', '.', '-', '.', 'gene_id \"chr5.g14440\"; transcript_id \"chr5.g14440.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69948968', '69949018', '.', '+', '.', 'gene_id \"chrX.g19186\"; transcript_id \"chrX.g19186.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69965572', '69966141', '.', '+', '.', 'gene_id \"chrX.g19193\"; transcript_id \"chrX.g19193.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '36023750', '36023799', '.', '-', '.', 'gene_id \"chr5.g14460\"; transcript_id \"chr5.g14460.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70476883', '70477145', '.', '+', '.', 'gene_id \"chrX.g19306\"; transcript_id \"chrX.g19306.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70491279', '70491483', '.', '+', '.', 'gene_id \"chrX.g19310\"; transcript_id \"chrX.g19310.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '36207798', '36207962', '.', '-', '.', 'gene_id \"chr5.g14563\"; transcript_id \"chr5.g14563.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70604190', '70604302', '.', '-', '.', 'gene_id \"chrX.g19327\"; transcript_id \"chrX.g19327.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70743342', '70743731', '.', '+', '.', 'gene_id \"chrX.g19340\"; transcript_id \"chrX.g19340.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71046662', '71046724', '.', '-', '.', 'gene_id \"chrX.g19363\"; transcript_id \"chrX.g19363.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71244963', '71245001', '.', '-', '.', 'gene_id \"chrX.g19387\"; transcript_id \"chrX.g19387.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71376564', '71376742', '.', '-', '.', 'gene_id \"chrX.g19399\"; transcript_id \"chrX.g19399.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71612877', '71613144', '.', '+', '.', 'gene_id \"chrX.g19413\"; transcript_id \"chrX.g19413.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71810330', '71810378', '.', '+', '.', 'gene_id \"chrX.g19431\"; transcript_id \"chrX.g19431.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '36369507', '36369554', '.', '-', '.', 'gene_id \"chr5.g14707\"; transcript_id \"chr5.g14707.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '72487697', '72487967', '.', '+', '.', 'gene_id \"chrX.g19495\"; transcript_id \"chrX.g19495.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '72562425', '72562456', '.', '+', '.', 'gene_id \"chrX.g19506\"; transcript_id \"chrX.g19506.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '72648087', '72648125', '.', '-', '.', 'gene_id \"chrX.g19510\"; transcript_id \"chrX.g19510.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '72798083', '72798207', '.', '-', '.', 'gene_id \"chrX.g19525\"; transcript_id \"chrX.g19525.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73110434', '73110668', '.', '-', '.', 'gene_id \"chrX.g19559\"; transcript_id \"chrX.g19559.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '37006832', '37007301', '.', '+', '.', 'gene_id \"chr5.g14849\"; transcript_id \"chr5.g14849.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '37007670', '37007934', '.', '-', '.', 'gene_id \"chr5.g14850\"; transcript_id \"chr5.g14850.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73498887', '73498952', '.', '+', '.', 'gene_id \"chrX.g19642\"; transcript_id \"chrX.g19642.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73517518', '73517835', '.', '-', '.', 'gene_id \"chrX.g19659\"; transcript_id \"chrX.g19659.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73658349', '73658547', '.', '+', '.', 'gene_id \"chrX.g19712\"; transcript_id \"chrX.g19712.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73677909', '73677941', '.', '+', '.', 'gene_id \"chrX.g19722\"; transcript_id \"chrX.g19722.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73829801', '73830032', '.', '+', '.', 'gene_id \"chrX.g19791\"; transcript_id \"chrX.g19791.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73878152', '73878184', '.', '+', '.', 'gene_id \"chrX.g19827\"; transcript_id \"chrX.g19827.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74077054', '74077354', '.', '+', '.', 'gene_id \"chrX.g19895\"; transcript_id \"chrX.g19895.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74091425', '74091464', '.', '+', '.', 'gene_id \"chrX.g19901\"; transcript_id \"chrX.g19901.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '38292340', '38292406', '.', '+', '.', 'gene_id \"chr5.g15169\"; transcript_id \"chr5.g15169.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '38663607', '38663636', '.', '-', '.', 'gene_id \"chr5.g15202\"; transcript_id \"chr5.g15202.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74232901', '74233075', '.', '+', '.', 'gene_id \"chrX.g19950\"; transcript_id \"chrX.g19950.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '38840956', '38840997', '.', '-', '.', 'gene_id \"chr5.g15219\"; transcript_id \"chr5.g15219.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '38900199', '38900519', '.', '+', '.', 'gene_id \"chr5.g15244\"; transcript_id \"chr5.g15244.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74668696', '74668850', '.', '+', '.', 'gene_id \"chrX.g20119\"; transcript_id \"chrX.g20119.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74675142', '74675302', '.', '+', '.', 'gene_id \"chrX.g20123\"; transcript_id \"chrX.g20123.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39208099', '39208139', '.', '+', '.', 'gene_id \"chr5.g15403\"; transcript_id \"chr5.g15403.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74801341', '74801499', '.', '-', '.', 'gene_id \"chrX.g20146\"; transcript_id \"chrX.g20146.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74905802', '74905861', '.', '+', '.', 'gene_id \"chrX.g20177\"; transcript_id \"chrX.g20177.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75011539', '75011697', '.', '+', '.', 'gene_id \"chrX.g20192\"; transcript_id \"chrX.g20192.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39283771', '39285032', '.', '-', '.', 'gene_id \"chr5.g15466\"; transcript_id \"chr5.g15466.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39489921', '39490194', '.', '-', '.', 'gene_id \"chr5.g15586\"; transcript_id \"chr5.g15586.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75357787', '75358106', '.', '+', '.', 'gene_id \"chrX.g20337\"; transcript_id \"chrX.g20337.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75366475', '75366742', '.', '-', '.', 'gene_id \"chrX.g20339\"; transcript_id \"chrX.g20339.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75538928', '75538970', '.', '-', '.', 'gene_id \"chrX.g20391\"; transcript_id \"chrX.g20391.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39613636', '39613711', '.', '-', '.', 'gene_id \"chr5.g15675\"; transcript_id \"chr5.g15675.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75770902', '75771074', '.', '+', '.', 'gene_id \"chrX.g20435\"; transcript_id \"chrX.g20435.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75901131', '75901198', '.', '+', '.', 'gene_id \"chrX.g20485\"; transcript_id \"chrX.g20485.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39739678', '39740013', '.', '-', '.', 'gene_id \"chr5.g15773\"; transcript_id \"chr5.g15773.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76321059', '76321138', '.', '+', '.', 'gene_id \"chrX.g20615\"; transcript_id \"chrX.g20615.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76469933', '76470255', '.', '+', '.', 'gene_id \"chrX.g20639\"; transcript_id \"chrX.g20639.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76720018', '76720077', '.', '+', '.', 'gene_id \"chrX.g20699\"; transcript_id \"chrX.g20699.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76720018', '76720103', '.', '+', '.', 'gene_id \"chrX.g20699\"; transcript_id \"chrX.g20699.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76734593', '76735125', '.', '+', '.', 'gene_id \"chrX.g20702\"; transcript_id \"chrX.g20702.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76734689', '76734987', '.', '-', '.', 'gene_id \"chrX.g20703\"; transcript_id \"chrX.g20703.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40118709', '40119549', '.', '-', '.', 'gene_id \"chr5.g16016\"; transcript_id \"chr5.g16016.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76776316', '76776586', '.', '+', '.', 'gene_id \"chrX.g20716\"; transcript_id \"chrX.g20716.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40119562', '40119864', '.', '-', '.', 'gene_id \"chr5.g16018\"; transcript_id \"chr5.g16018.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76908374', '76908596', '.', '-', '.', 'gene_id \"chrX.g20730\"; transcript_id \"chrX.g20730.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77219887', '77220421', '.', '+', '.', 'gene_id \"chrX.g20762\"; transcript_id \"chrX.g20762.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77505088', '77505155', '.', '+', '.', 'gene_id \"chrX.g20798\"; transcript_id \"chrX.g20798.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77626291', '77626786', '.', '-', '.', 'gene_id \"chrX.g20815\"; transcript_id \"chrX.g20815.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77663240', '77663279', '.', '+', '.', 'gene_id \"chrX.g20819\"; transcript_id \"chrX.g20819.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78001955', '78002122', '.', '-', '.', 'gene_id \"chrX.g20852\"; transcript_id \"chrX.g20852.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78098087', '78098116', '.', '+', '.', 'gene_id \"chrX.g20857\"; transcript_id \"chrX.g20857.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40508692', '40508897', '.', '+', '.', 'gene_id \"chr5.g16207\"; transcript_id \"chr5.g16207.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40441509', '40441735', '.', '+', '.', 'gene_id \"chr5.g16230\"; transcript_id \"chr5.g16230.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78556324', '78556375', '.', '+', '.', 'gene_id \"chrX.g20921\"; transcript_id \"chrX.g20921.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40464810', '40464858', '.', '+', '.', 'gene_id \"chr5.g16248\"; transcript_id \"chr5.g16248.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40791856', '40791968', '.', '+', '.', 'gene_id \"chr5.g16385\"; transcript_id \"chr5.g16385.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79592047', '79593249', '.', '-', '.', 'gene_id \"chrX.g21109\"; transcript_id \"chrX.g21109.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79592047', '79592945', '.', '-', '.', 'gene_id \"chrX.g21109\"; transcript_id \"chrX.g21109.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79593237', '79593372', '.', '+', '.', 'gene_id \"chrX.g21110\"; transcript_id \"chrX.g21110.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79604832', '79605704', '.', '-', '.', 'gene_id \"chrX.g21117\"; transcript_id \"chrX.g21117.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79604832', '79605704', '.', '-', '.', 'gene_id \"chrX.g21117\"; transcript_id \"chrX.g21117.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79603390', '79604082', '.', '+', '.', 'gene_id \"chrX.g21121\"; transcript_id \"chrX.g21121.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79610692', '79610952', '.', '+', '.', 'gene_id \"chrX.g21128\"; transcript_id \"chrX.g21128.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40996001', '40996108', '.', '-', '.', 'gene_id \"chr5.g16522\"; transcript_id \"chr5.g16522.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41022575', '41023088', '.', '-', '.', 'gene_id \"chr5.g16538\"; transcript_id \"chr5.g16538.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41023061', '41023445', '.', '+', '.', 'gene_id \"chr5.g16539\"; transcript_id \"chr5.g16539.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79764801', '79764909', '.', '+', '.', 'gene_id \"chrX.g21196\"; transcript_id \"chrX.g21196.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79963667', '79963697', '.', '-', '.', 'gene_id \"chrX.g21275\"; transcript_id \"chrX.g21275.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80038468', '80038639', '.', '-', '.', 'gene_id \"chrX.g21295\"; transcript_id \"chrX.g21295.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80040185', '80040359', '.', '+', '.', 'gene_id \"chrX.g21296\"; transcript_id \"chrX.g21296.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80040382', '80040559', '.', '+', '.', 'gene_id \"chrX.g21296\"; transcript_id \"chrX.g21296.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80325070', '80325512', '.', '-', '.', 'gene_id \"chrX.g21361\"; transcript_id \"chrX.g21361.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80338301', '80338335', '.', '-', '.', 'gene_id \"chrX.g21362\"; transcript_id \"chrX.g21362.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80439598', '80439676', '.', '-', '.', 'gene_id \"chrX.g21459\"; transcript_id \"chrX.g21459.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41588231', '41588496', '.', '-', '.', 'gene_id \"chr5.g16970\"; transcript_id \"chr5.g16970.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41613810', '41613839', '.', '+', '.', 'gene_id \"chr5.g16994\"; transcript_id \"chr5.g16994.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41650193', '41650540', '.', '-', '.', 'gene_id \"chr5.g17023\"; transcript_id \"chr5.g17023.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80722834', '80723014', '.', '-', '.', 'gene_id \"chrX.g21620\"; transcript_id \"chrX.g21620.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80883935', '80883986', '.', '-', '.', 'gene_id \"chrX.g21677\"; transcript_id \"chrX.g21677.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80930923', '80931075', '.', '+', '.', 'gene_id \"chrX.g21685\"; transcript_id \"chrX.g21685.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80950601', '80950766', '.', '-', '.', 'gene_id \"chrX.g21697\"; transcript_id \"chrX.g21697.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81006079', '81006146', '.', '-', '.', 'gene_id \"chrX.g21700\"; transcript_id \"chrX.g21700.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41744130', '41744439', '.', '+', '.', 'gene_id \"chr5.g17128\"; transcript_id \"chr5.g17128.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81129798', '81129830', '.', '+', '.', 'gene_id \"chrX.g21734\"; transcript_id \"chrX.g21734.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81316112', '81316296', '.', '+', '.', 'gene_id \"chrX.g21788\"; transcript_id \"chrX.g21788.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81320225', '81320294', '.', '+', '.', 'gene_id \"chrX.g21789\"; transcript_id \"chrX.g21789.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81955085', '81955122', '.', '+', '.', 'gene_id \"chrX.g21919\"; transcript_id \"chrX.g21919.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82005201', '82005406', '.', '+', '.', 'gene_id \"chrX.g21940\"; transcript_id \"chrX.g21940.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82006480', '82006822', '.', '-', '.', 'gene_id \"chrX.g21944\"; transcript_id \"chrX.g21944.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42044968', '42045137', '.', '-', '.', 'gene_id \"chr5.g17372\"; transcript_id \"chr5.g17372.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42162628', '42164107', '.', '-', '.', 'gene_id \"chr5.g17421\"; transcript_id \"chr5.g17421.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82116308', '82116375', '.', '+', '.', 'gene_id \"chrX.g22028\"; transcript_id \"chrX.g22028.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42333304', '42333790', '.', '+', '.', 'gene_id \"chr5.g17527\"; transcript_id \"chr5.g17527.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82366682', '82366961', '.', '+', '.', 'gene_id \"chrX.g22112\"; transcript_id \"chrX.g22112.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82388456', '82388848', '.', '-', '.', 'gene_id \"chrX.g22144\"; transcript_id \"chrX.g22144.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82491204', '82491967', '.', '+', '.', 'gene_id \"chrX.g22198\"; transcript_id \"chrX.g22198.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82491458', '82491878', '.', '+', '.', 'gene_id \"chrX.g22198\"; transcript_id \"chrX.g22198.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82538803', '82539465', '.', '-', '.', 'gene_id \"chrX.g22234\"; transcript_id \"chrX.g22234.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82539471', '82539793', '.', '+', '.', 'gene_id \"chrX.g22235\"; transcript_id \"chrX.g22235.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82611777', '82611996', '.', '+', '.', 'gene_id \"chrX.g22304\"; transcript_id \"chrX.g22304.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82617347', '82617545', '.', '-', '.', 'gene_id \"chrX.g22315\"; transcript_id \"chrX.g22315.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82617960', '82618240', '.', '+', '.', 'gene_id \"chrX.g22316\"; transcript_id \"chrX.g22316.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42582208', '42583219', '.', '+', '.', 'gene_id \"chr5.g17778\"; transcript_id \"chr5.g17778.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82777054', '82777433', '.', '-', '.', 'gene_id \"chrX.g22365\"; transcript_id \"chrX.g22365.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82797383', '82797440', '.', '-', '.', 'gene_id \"chrX.g22367\"; transcript_id \"chrX.g22367.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42642872', '42642931', '.', '+', '.', 'gene_id \"chr5.g17831\"; transcript_id \"chr5.g17831.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42674903', '42674940', '.', '-', '.', 'gene_id \"chr5.g17834\"; transcript_id \"chr5.g17834.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82906616', '82906779', '.', '+', '.', 'gene_id \"chrX.g22386\"; transcript_id \"chrX.g22386.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83091609', '83091665', '.', '-', '.', 'gene_id \"chrX.g22444\"; transcript_id \"chrX.g22444.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83239057', '83239088', '.', '+', '.', 'gene_id \"chrX.g22463\"; transcript_id \"chrX.g22463.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83315066', '83315132', '.', '+', '.', 'gene_id \"chrX.g22479\"; transcript_id \"chrX.g22479.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83315066', '83315120', '.', '+', '.', 'gene_id \"chrX.g22479\"; transcript_id \"chrX.g22479.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83315066', '83315132', '.', '-', '.', 'gene_id \"chrX.g22480\"; transcript_id \"chrX.g22480.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42810705', '42811137', '.', '+', '.', 'gene_id \"chr5.g17965\"; transcript_id \"chr5.g17965.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42840055', '42840278', '.', '-', '.', 'gene_id \"chr5.g18009\"; transcript_id \"chr5.g18009.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42840295', '42840591', '.', '-', '.', 'gene_id \"chr5.g18010\"; transcript_id \"chr5.g18010.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83431800', '83432930', '.', '-', '.', 'gene_id \"chrX.g22539\"; transcript_id \"chrX.g22539.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83442756', '83442789', '.', '+', '.', 'gene_id \"chrX.g22544\"; transcript_id \"chrX.g22544.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83550465', '83550506', '.', '+', '.', 'gene_id \"chrX.g22589\"; transcript_id \"chrX.g22589.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83720309', '83720407', '.', '-', '.', 'gene_id \"chrX.g22625\"; transcript_id \"chrX.g22625.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43117218', '43119012', '.', '-', '.', 'gene_id \"chr5.g18189\"; transcript_id \"chr5.g18189.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43172318', '43172848', '.', '-', '.', 'gene_id \"chr5.g18198\"; transcript_id \"chr5.g18198.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43228028', '43228183', '.', '-', '.', 'gene_id \"chr5.g18204\"; transcript_id \"chr5.g18204.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83958598', '83958786', '.', '-', '.', 'gene_id \"chrX.g22710\"; transcript_id \"chrX.g22710.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43286040', '43286086', '.', '+', '.', 'gene_id \"chr5.g18220\"; transcript_id \"chr5.g18220.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84208986', '84209045', '.', '+', '.', 'gene_id \"chrX.g22791\"; transcript_id \"chrX.g22791.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43429618', '43429651', '.', '+', '.', 'gene_id \"chr5.g18305\"; transcript_id \"chr5.g18305.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84599218', '84599249', '.', '-', '.', 'gene_id \"chrX.g22867\"; transcript_id \"chrX.g22867.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84717511', '84717548', '.', '+', '.', 'gene_id \"chrX.g22889\"; transcript_id \"chrX.g22889.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84729751', '84730177', '.', '+', '.', 'gene_id \"chrX.g22891\"; transcript_id \"chrX.g22891.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43665733', '43665978', '.', '+', '.', 'gene_id \"chr5.g18410\"; transcript_id \"chr5.g18410.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43666075', '43667001', '.', '+', '.', 'gene_id \"chr5.g18411\"; transcript_id \"chr5.g18411.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43667003', '43667750', '.', '-', '.', 'gene_id \"chr5.g18412\"; transcript_id \"chr5.g18412.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84888384', '84888547', '.', '-', '.', 'gene_id \"chrX.g22923\"; transcript_id \"chrX.g22923.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43746693', '43747252', '.', '-', '.', 'gene_id \"chr5.g18494\"; transcript_id \"chr5.g18494.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43757617', '43757771', '.', '+', '.', 'gene_id \"chr5.g18504\"; transcript_id \"chr5.g18504.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43757820', '43757997', '.', '+', '.', 'gene_id \"chr5.g18505\"; transcript_id \"chr5.g18505.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85221881', '85222054', '.', '+', '.', 'gene_id \"chrX.g22978\"; transcript_id \"chrX.g22978.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85282417', '85282580', '.', '-', '.', 'gene_id \"chrX.g22982\"; transcript_id \"chrX.g22982.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85406552', '85406610', '.', '-', '.', 'gene_id \"chrX.g23006\"; transcript_id \"chrX.g23006.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85723398', '85723533', '.', '-', '.', 'gene_id \"chrX.g23041\"; transcript_id \"chrX.g23041.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85771739', '85771775', '.', '-', '.', 'gene_id \"chrX.g23044\"; transcript_id \"chrX.g23044.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86494985', '86495029', '.', '+', '.', 'gene_id \"chrX.g23143\"; transcript_id \"chrX.g23143.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86739814', '86740055', '.', '-', '.', 'gene_id \"chrX.g23177\"; transcript_id \"chrX.g23177.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86771699', '86771751', '.', '+', '.', 'gene_id \"chrX.g23182\"; transcript_id \"chrX.g23182.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86776505', '86776544', '.', '-', '.', 'gene_id \"chrX.g23184\"; transcript_id \"chrX.g23184.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86865735', '86865931', '.', '-', '.', 'gene_id \"chrX.g23209\"; transcript_id \"chrX.g23209.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86886375', '86886542', '.', '-', '.', 'gene_id \"chrX.g23213\"; transcript_id \"chrX.g23213.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '44243387', '44243419', '.', '-', '.', 'gene_id \"chr5.g18770\"; transcript_id \"chr5.g18770.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86951663', '86951738', '.', '+', '.', 'gene_id \"chrX.g23249\"; transcript_id \"chrX.g23249.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86977263', '86977293', '.', '-', '.', 'gene_id \"chrX.g23258\"; transcript_id \"chrX.g23258.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87129040', '87129072', '.', '+', '.', 'gene_id \"chrX.g23317\"; transcript_id \"chrX.g23317.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87196227', '87196262', '.', '-', '.', 'gene_id \"chrX.g23330\"; transcript_id \"chrX.g23330.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87293888', '87294141', '.', '+', '.', 'gene_id \"chrX.g23361\"; transcript_id \"chrX.g23361.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87388345', '87388396', '.', '+', '.', 'gene_id \"chrX.g23376\"; transcript_id \"chrX.g23376.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '44924374', '44924642', '.', '-', '.', 'gene_id \"chr5.g18935\"; transcript_id \"chr5.g18935.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '44968314', '44968363', '.', '-', '.', 'gene_id \"chr5.g18951\"; transcript_id \"chr5.g18951.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87493997', '87494198', '.', '-', '.', 'gene_id \"chrX.g23450\"; transcript_id \"chrX.g23450.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87494318', '87494580', '.', '+', '.', 'gene_id \"chrX.g23453\"; transcript_id \"chrX.g23453.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45027624', '45027660', '.', '-', '.', 'gene_id \"chr5.g18973\"; transcript_id \"chr5.g18973.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87546545', '87546915', '.', '-', '.', 'gene_id \"chrX.g23511\"; transcript_id \"chrX.g23511.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45349076', '45349108', '.', '-', '.', 'gene_id \"chr5.g19079\"; transcript_id \"chr5.g19079.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87672759', '87672844', '.', '-', '.', 'gene_id \"chrX.g23585\"; transcript_id \"chrX.g23585.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87672759', '87672911', '.', '-', '.', 'gene_id \"chrX.g23585\"; transcript_id \"chrX.g23585.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45466584', '45466760', '.', '+', '.', 'gene_id \"chr5.g19179\"; transcript_id \"chr5.g19179.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45474253', '45474455', '.', '-', '.', 'gene_id \"chr5.g19190\"; transcript_id \"chr5.g19190.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45474701', '45474867', '.', '-', '.', 'gene_id \"chr5.g19191\"; transcript_id \"chr5.g19191.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87812707', '87812758', '.', '-', '.', 'gene_id \"chrX.g23642\"; transcript_id \"chrX.g23642.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87908031', '87908119', '.', '-', '.', 'gene_id \"chrX.g23666\"; transcript_id \"chrX.g23666.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45502351', '45502514', '.', '+', '.', 'gene_id \"chr5.g19222\"; transcript_id \"chr5.g19222.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87981408', '87981563', '.', '+', '.', 'gene_id \"chrX.g23677\"; transcript_id \"chrX.g23677.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45634196', '45634239', '.', '-', '.', 'gene_id \"chr5.g19287\"; transcript_id \"chr5.g19287.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '88409785', '88409847', '.', '+', '.', 'gene_id \"chrX.g23751\"; transcript_id \"chrX.g23751.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '88521800', '88521942', '.', '+', '.', 'gene_id \"chrX.g23766\"; transcript_id \"chrX.g23766.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45717567', '45718695', '.', '-', '.', 'gene_id \"chr5.g19335\"; transcript_id \"chr5.g19335.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45718728', '45719293', '.', '+', '.', 'gene_id \"chr5.g19336\"; transcript_id \"chr5.g19336.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45719728', '45719886', '.', '+', '.', 'gene_id \"chr5.g19337\"; transcript_id \"chr5.g19337.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '88678846', '88678909', '.', '-', '.', 'gene_id \"chrX.g23788\"; transcript_id \"chrX.g23788.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89090011', '89090046', '.', '+', '.', 'gene_id \"chrX.g23824\"; transcript_id \"chrX.g23824.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '45882840', '45882895', '.', '-', '.', 'gene_id \"chr5.g19382\"; transcript_id \"chr5.g19382.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89166517', '89166614', '.', '+', '.', 'gene_id \"chrX.g23837\"; transcript_id \"chrX.g23837.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89279396', '89279440', '.', '-', '.', 'gene_id \"chrX.g23858\"; transcript_id \"chrX.g23858.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46034582', '46034620', '.', '-', '.', 'gene_id \"chr5.g19411\"; transcript_id \"chr5.g19411.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46042337', '46042370', '.', '+', '.', 'gene_id \"chr5.g19412\"; transcript_id \"chr5.g19412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46045683', '46045734', '.', '+', '.', 'gene_id \"chr5.g19414\"; transcript_id \"chr5.g19414.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46119044', '46119092', '.', '-', '.', 'gene_id \"chr5.g19452\"; transcript_id \"chr5.g19452.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89401248', '89401332', '.', '-', '.', 'gene_id \"chrX.g23902\"; transcript_id \"chrX.g23902.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46228524', '46228770', '.', '+', '.', 'gene_id \"chr5.g19488\"; transcript_id \"chr5.g19488.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89707250', '89707390', '.', '+', '.', 'gene_id \"chrX.g23946\"; transcript_id \"chrX.g23946.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89742785', '89742847', '.', '-', '.', 'gene_id \"chrX.g23954\"; transcript_id \"chrX.g23954.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90079223', '90079557', '.', '+', '.', 'gene_id \"chrX.g24039\"; transcript_id \"chrX.g24039.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46497892', '46497921', '.', '-', '.', 'gene_id \"chr5.g19598\"; transcript_id \"chr5.g19598.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90022965', '90023120', '.', '-', '.', 'gene_id \"chrX.g24066\"; transcript_id \"chrX.g24066.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90089581', '90089619', '.', '-', '.', 'gene_id \"chrX.g24098\"; transcript_id \"chrX.g24098.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90158621', '90158947', '.', '+', '.', 'gene_id \"chrX.g24162\"; transcript_id \"chrX.g24162.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46705180', '46705367', '.', '-', '.', 'gene_id \"chr5.g19838\"; transcript_id \"chr5.g19838.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90502064', '90502131', '.', '-', '.', 'gene_id \"chrX.g24268\"; transcript_id \"chrX.g24268.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90570461', '90570520', '.', '+', '.', 'gene_id \"chrX.g24316\"; transcript_id \"chrX.g24316.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46828015', '46828319', '.', '+', '.', 'gene_id \"chr5.g19914\"; transcript_id \"chr5.g19914.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46828321', '46828557', '.', '-', '.', 'gene_id \"chr5.g19916\"; transcript_id \"chr5.g19916.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90647199', '90647253', '.', '-', '.', 'gene_id \"chrX.g24338\"; transcript_id \"chrX.g24338.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46884651', '46885216', '.', '+', '.', 'gene_id \"chr5.g19999\"; transcript_id \"chr5.g19999.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90850704', '90850940', '.', '-', '.', 'gene_id \"chrX.g24478\"; transcript_id \"chrX.g24478.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46920481', '46920586', '.', '-', '.', 'gene_id \"chr5.g20059\"; transcript_id \"chr5.g20059.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90904254', '90904428', '.', '+', '.', 'gene_id \"chrX.g24519\"; transcript_id \"chrX.g24519.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90904467', '90904656', '.', '-', '.', 'gene_id \"chrX.g24520\"; transcript_id \"chrX.g24520.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47343997', '47344064', '.', '+', '.', 'gene_id \"chr5.g20135\"; transcript_id \"chr5.g20135.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47436027', '47436180', '.', '-', '.', 'gene_id \"chr5.g20250\"; transcript_id \"chr5.g20250.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47436745', '47436941', '.', '+', '.', 'gene_id \"chr5.g20252\"; transcript_id \"chr5.g20252.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47439809', '47439965', '.', '+', '.', 'gene_id \"chr5.g20253\"; transcript_id \"chr5.g20253.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47496105', '47496298', '.', '-', '.', 'gene_id \"chr5.g20272\"; transcript_id \"chr5.g20272.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47562393', '47562434', '.', '+', '.', 'gene_id \"chr5.g20282\"; transcript_id \"chr5.g20282.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47728151', '47728716', '.', '-', '.', 'gene_id \"chr5.g20390\"; transcript_id \"chr5.g20390.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48174724', '48174921', '.', '+', '.', 'gene_id \"chr5.g20525\"; transcript_id \"chr5.g20525.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48076980', '48077216', '.', '-', '.', 'gene_id \"chr5.g20570\"; transcript_id \"chr5.g20570.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48077248', '48077483', '.', '-', '.', 'gene_id \"chr5.g20571\"; transcript_id \"chr5.g20571.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91801546', '91802337', '.', '-', '.', 'gene_id \"chrX.g25038\"; transcript_id \"chrX.g25038.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48173987', '48174199', '.', '+', '.', 'gene_id \"chr5.g20626\"; transcript_id \"chr5.g20626.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91914986', '91915015', '.', '-', '.', 'gene_id \"chrX.g25079\"; transcript_id \"chrX.g25079.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91927902', '91927937', '.', '-', '.', 'gene_id \"chrX.g25082\"; transcript_id \"chrX.g25082.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48298908', '48298967', '.', '+', '.', 'gene_id \"chr5.g20659\"; transcript_id \"chr5.g20659.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92146375', '92146488', '.', '-', '.', 'gene_id \"chrX.g25142\"; transcript_id \"chrX.g25142.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92557388', '92557424', '.', '-', '.', 'gene_id \"chrX.g25205\"; transcript_id \"chrX.g25205.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92740037', '92740337', '.', '+', '.', 'gene_id \"chrX.g25225\"; transcript_id \"chrX.g25225.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92747641', '92747781', '.', '+', '.', 'gene_id \"chrX.g25227\"; transcript_id \"chrX.g25227.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92808804', '92808861', '.', '-', '.', 'gene_id \"chrX.g25231\"; transcript_id \"chrX.g25231.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92839404', '92839893', '.', '-', '.', 'gene_id \"chrX.g25233\"; transcript_id \"chrX.g25233.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92941552', '92941739', '.', '-', '.', 'gene_id \"chrX.g25247\"; transcript_id \"chrX.g25247.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92974854', '92974901', '.', '+', '.', 'gene_id \"chrX.g25249\"; transcript_id \"chrX.g25249.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92974858', '92974901', '.', '+', '.', 'gene_id \"chrX.g25249\"; transcript_id \"chrX.g25249.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48697236', '48697568', '.', '+', '.', 'gene_id \"chr5.g20835\"; transcript_id \"chr5.g20835.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48697767', '48697981', '.', '-', '.', 'gene_id \"chr5.g20836\"; transcript_id \"chr5.g20836.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93258214', '93258611', '.', '+', '.', 'gene_id \"chrX.g25293\"; transcript_id \"chrX.g25293.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93258291', '93258930', '.', '-', '.', 'gene_id \"chrX.g25294\"; transcript_id \"chrX.g25294.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93259579', '93259611', '.', '+', '.', 'gene_id \"chrX.g25296\"; transcript_id \"chrX.g25296.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93320857', '93321251', '.', '-', '.', 'gene_id \"chrX.g25299\"; transcript_id \"chrX.g25299.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93339451', '93339484', '.', '-', '.', 'gene_id \"chrX.g25304\"; transcript_id \"chrX.g25304.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93516687', '93516732', '.', '-', '.', 'gene_id \"chrX.g25323\"; transcript_id \"chrX.g25323.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48869592', '48869756', '.', '+', '.', 'gene_id \"chr5.g20966\"; transcript_id \"chr5.g20966.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48876326', '48876654', '.', '+', '.', 'gene_id \"chr5.g20976\"; transcript_id \"chr5.g20976.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48927532', '48927699', '.', '+', '.', 'gene_id \"chr5.g21066\"; transcript_id \"chr5.g21066.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48931165', '48931428', '.', '-', '.', 'gene_id \"chr5.g21070\"; transcript_id \"chr5.g21070.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48931512', '48931775', '.', '+', '.', 'gene_id \"chr5.g21071\"; transcript_id \"chr5.g21071.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48932183', '48932466', '.', '-', '.', 'gene_id \"chr5.g21072\"; transcript_id \"chr5.g21072.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94506487', '94506548', '.', '+', '.', 'gene_id \"chrX.g25558\"; transcript_id \"chrX.g25558.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94520346', '94520394', '.', '-', '.', 'gene_id \"chrX.g25588\"; transcript_id \"chrX.g25588.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48976179', '48976364', '.', '+', '.', 'gene_id \"chr5.g21139\"; transcript_id \"chr5.g21139.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48976453', '48976611', '.', '-', '.', 'gene_id \"chr5.g21140\"; transcript_id \"chr5.g21140.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49601437', '49601475', '.', '+', '.', 'gene_id \"chr5.g21261\"; transcript_id \"chr5.g21261.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49625668', '49625810', '.', '-', '.', 'gene_id \"chr5.g21264\"; transcript_id \"chr5.g21264.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49625668', '49625870', '.', '-', '.', 'gene_id \"chr5.g21264\"; transcript_id \"chr5.g21264.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49691285', '49691326', '.', '+', '.', 'gene_id \"chr5.g21272\"; transcript_id \"chr5.g21272.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94930900', '94930964', '.', '-', '.', 'gene_id \"chrX.g25776\"; transcript_id \"chrX.g25776.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50021414', '50021608', '.', '+', '.', 'gene_id \"chr5.g21314\"; transcript_id \"chr5.g21314.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50051841', '50052120', '.', '+', '.', 'gene_id \"chr5.g21320\"; transcript_id \"chr5.g21320.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50101043', '50101279', '.', '+', '.', 'gene_id \"chr5.g21370\"; transcript_id \"chr5.g21370.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '95846843', '95847100', '.', '+', '.', 'gene_id \"chrX.g25892\"; transcript_id \"chrX.g25892.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '95956856', '95956916', '.', '+', '.', 'gene_id \"chrX.g25917\"; transcript_id \"chrX.g25917.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '95956856', '95956916', '.', '-', '.', 'gene_id \"chrX.g25918\"; transcript_id \"chrX.g25918.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96030113', '96030206', '.', '+', '.', 'gene_id \"chrX.g25925\"; transcript_id \"chrX.g25925.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96051642', '96051695', '.', '+', '.', 'gene_id \"chrX.g25929\"; transcript_id \"chrX.g25929.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96726497', '96726544', '.', '+', '.', 'gene_id \"chrX.g26003\"; transcript_id \"chrX.g26003.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96856225', '96856496', '.', '-', '.', 'gene_id \"chrX.g26023\"; transcript_id \"chrX.g26023.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96989823', '96990023', '.', '+', '.', 'gene_id \"chrX.g26035\"; transcript_id \"chrX.g26035.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97118970', '97119112', '.', '+', '.', 'gene_id \"chrX.g26049\"; transcript_id \"chrX.g26049.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97247445', '97247596', '.', '+', '.', 'gene_id \"chrX.g26063\"; transcript_id \"chrX.g26063.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97247490', '97247603', '.', '+', '.', 'gene_id \"chrX.g26063\"; transcript_id \"chrX.g26063.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97494158', '97495272', '.', '-', '.', 'gene_id \"chrX.g26084\"; transcript_id \"chrX.g26084.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97495000', '97495251', '.', '+', '.', 'gene_id \"chrX.g26085\"; transcript_id \"chrX.g26085.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50619523', '50619602', '.', '+', '.', 'gene_id \"chr5.g21641\"; transcript_id \"chr5.g21641.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50686480', '50686526', '.', '+', '.', 'gene_id \"chr5.g21668\"; transcript_id \"chr5.g21668.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '98721926', '98722012', '.', '-', '.', 'gene_id \"chrX.g26187\"; transcript_id \"chrX.g26187.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '51092710', '51093010', '.', '-', '.', 'gene_id \"chr5.g21716\"; transcript_id \"chr5.g21716.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '51093046', '51093478', '.', '+', '.', 'gene_id \"chr5.g21717\"; transcript_id \"chr5.g21717.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '51094451', '51094666', '.', '-', '.', 'gene_id \"chr5.g21719\"; transcript_id \"chr5.g21719.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99323901', '99324104', '.', '-', '.', 'gene_id \"chrX.g26259\"; transcript_id \"chrX.g26259.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99915547', '99915592', '.', '+', '.', 'gene_id \"chrX.g26399\"; transcript_id \"chrX.g26399.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chrX', 'PASA', 'exon', '99949572', '99949759', '.', '-', '.', 'gene_id \"chrX.g26421\"; transcript_id \"chrX.g26421.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99984333', '99985250', '.', '-', '.', 'gene_id \"chrX.g26442\"; transcript_id \"chrX.g26442.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99985412', '99985635', '.', '-', '.', 'gene_id \"chrX.g26444\"; transcript_id \"chrX.g26444.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52493825', '52493882', '.', '-', '.', 'gene_id \"chr5.g22003\"; transcript_id \"chr5.g22003.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100234242', '100234273', '.', '-', '.', 'gene_id \"chrX.g26513\"; transcript_id \"chrX.g26513.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100307297', '100307608', '.', '-', '.', 'gene_id \"chrX.g26575\"; transcript_id \"chrX.g26575.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100390055', '100390088', '.', '-', '.', 'gene_id \"chrX.g26611\"; transcript_id \"chrX.g26611.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52860682', '52860844', '.', '-', '.', 'gene_id \"chr5.g22151\"; transcript_id \"chr5.g22151.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52927234', '52928028', '.', '-', '.', 'gene_id \"chr5.g22185\"; transcript_id \"chr5.g22185.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100478009', '100478360', '.', '+', '.', 'gene_id \"chrX.g26659\"; transcript_id \"chrX.g26659.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100478339', '100478527', '.', '-', '.', 'gene_id \"chrX.g26660\"; transcript_id \"chrX.g26660.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100519219', '100519273', '.', '+', '.', 'gene_id \"chrX.g26678\"; transcript_id \"chrX.g26678.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53135886', '53136194', '.', '-', '.', 'gene_id \"chr5.g22231\"; transcript_id \"chr5.g22231.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53135932', '53136194', '.', '-', '.', 'gene_id \"chr5.g22231\"; transcript_id \"chr5.g22231.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53066265', '53066492', '.', '+', '.', 'gene_id \"chr5.g22252\"; transcript_id \"chr5.g22252.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53072836', '53073377', '.', '-', '.', 'gene_id \"chr5.g22262\"; transcript_id \"chr5.g22262.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53117255', '53117849', '.', '-', '.', 'gene_id \"chr5.g22319\"; transcript_id \"chr5.g22319.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53117998', '53118261', '.', '+', '.', 'gene_id \"chr5.g22321\"; transcript_id \"chr5.g22321.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100639547', '100639585', '.', '+', '.', 'gene_id \"chrX.g26803\"; transcript_id \"chrX.g26803.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53318134', '53318181', '.', '+', '.', 'gene_id \"chr5.g22364\"; transcript_id \"chr5.g22364.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53620184', '53620385', '.', '-', '.', 'gene_id \"chr5.g22383\"; transcript_id \"chr5.g22383.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '53870485', '53871365', '.', '-', '.', 'gene_id \"chr5.g22404\"; transcript_id \"chr5.g22404.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100944059', '100944179', '.', '+', '.', 'gene_id \"chrX.g26954\"; transcript_id \"chrX.g26954.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100949953', '100950129', '.', '+', '.', 'gene_id \"chrX.g26956\"; transcript_id \"chrX.g26956.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101078439', '101078608', '.', '-', '.', 'gene_id \"chrX.g27025\"; transcript_id \"chrX.g27025.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101078768', '101078919', '.', '+', '.', 'gene_id \"chrX.g27026\"; transcript_id \"chrX.g27026.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101091174', '101091425', '.', '+', '.', 'gene_id \"chrX.g27037\"; transcript_id \"chrX.g27037.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101208144', '101208391', '.', '+', '.', 'gene_id \"chrX.g27064\"; transcript_id \"chrX.g27064.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101234765', '101234942', '.', '+', '.', 'gene_id \"chrX.g27071\"; transcript_id \"chrX.g27071.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54386377', '54386418', '.', '-', '.', 'gene_id \"chr5.g22626\"; transcript_id \"chr5.g22626.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101293722', '101293933', '.', '+', '.', 'gene_id \"chrX.g27082\"; transcript_id \"chrX.g27082.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101704458', '101704529', '.', '+', '.', 'gene_id \"chrX.g27136\"; transcript_id \"chrX.g27136.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101951901', '101952661', '.', '+', '.', 'gene_id \"chrX.g27174\"; transcript_id \"chrX.g27174.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101955387', '101955608', '.', '+', '.', 'gene_id \"chrX.g27178\"; transcript_id \"chrX.g27178.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102015582', '102015618', '.', '-', '.', 'gene_id \"chrX.g27190\"; transcript_id \"chrX.g27190.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102231199', '102231388', '.', '+', '.', 'gene_id \"chrX.g27253\"; transcript_id \"chrX.g27253.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54814695', '54814748', '.', '-', '.', 'gene_id \"chr5.g22860\"; transcript_id \"chr5.g22860.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102298936', '102298989', '.', '+', '.', 'gene_id \"chrX.g27281\"; transcript_id \"chrX.g27281.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102405715', '102408574', '.', '-', '.', 'gene_id \"chrX.g27296\"; transcript_id \"chrX.g27296.i5\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102421206', '102421878', '.', '-', '.', 'gene_id \"chrX.g27296\"; transcript_id \"chrX.g27296.i9\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102416070', '102416560', '.', '+', '.', 'gene_id \"chrX.g27297\"; transcript_id \"chrX.g27297.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102413971', '102414338', '.', '+', '.', 'gene_id \"chrX.g27304\"; transcript_id \"chrX.g27304.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102421981', '102422169', '.', '-', '.', 'gene_id \"chrX.g27311\"; transcript_id \"chrX.g27311.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102654346', '102654391', '.', '-', '.', 'gene_id \"chrX.g27369\"; transcript_id \"chrX.g27369.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102799832', '102802936', '.', '+', '.', 'gene_id \"chrX.g27387\"; transcript_id \"chrX.g27387.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102885481', '102885515', '.', '+', '.', 'gene_id \"chrX.g27388\"; transcript_id \"chrX.g27388.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102813854', '102813894', '.', '-', '.', 'gene_id \"chrX.g27394\"; transcript_id \"chrX.g27394.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102847541', '102847628', '.', '+', '.', 'gene_id \"chrX.g27402\"; transcript_id \"chrX.g27402.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '103444649', '103444796', '.', '+', '.', 'gene_id \"chrX.g27514\"; transcript_id \"chrX.g27514.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '103467017', '103467839', '.', '+', '.', 'gene_id \"chrX.g27517\"; transcript_id \"chrX.g27517.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55238573', '55238749', '.', '+', '.', 'gene_id \"chr5.g23207\"; transcript_id \"chr5.g23207.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55238727', '55239016', '.', '-', '.', 'gene_id \"chr5.g23208\"; transcript_id \"chr5.g23208.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55239065', '55240218', '.', '+', '.', 'gene_id \"chr5.g23209\"; transcript_id \"chr5.g23209.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104104252', '104104516', '.', '-', '.', 'gene_id \"chrX.g27607\"; transcript_id \"chrX.g27607.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104345209', '104345468', '.', '-', '.', 'gene_id \"chrX.g27661\"; transcript_id \"chrX.g27661.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55322937', '55323148', '.', '+', '.', 'gene_id \"chr5.g23300\"; transcript_id \"chr5.g23300.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55323436', '55323766', '.', '+', '.', 'gene_id \"chr5.g23302\"; transcript_id \"chr5.g23302.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104419347', '104419381', '.', '-', '.', 'gene_id \"chrX.g27709\"; transcript_id \"chrX.g27709.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55366106', '55366592', '.', '+', '.', 'gene_id \"chr5.g23345\"; transcript_id \"chr5.g23345.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55383195', '55383425', '.', '+', '.', 'gene_id \"chr5.g23349\"; transcript_id \"chr5.g23349.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55489268', '55489474', '.', '-', '.', 'gene_id \"chr5.g23412\"; transcript_id \"chr5.g23412.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104923140', '104923171', '.', '-', '.', 'gene_id \"chrX.g27901\"; transcript_id \"chrX.g27901.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105022061', '105022237', '.', '-', '.', 'gene_id \"chrX.g27963\"; transcript_id \"chrX.g27963.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55918270', '55918309', '.', '-', '.', 'gene_id \"chr5.g23614\"; transcript_id \"chr5.g23614.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105116634', '105117038', '.', '+', '.', 'gene_id \"chrX.g28033\"; transcript_id \"chrX.g28033.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55977304', '55977340', '.', '+', '.', 'gene_id \"chr5.g23656\"; transcript_id \"chr5.g23656.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105234911', '105235083', '.', '+', '.', 'gene_id \"chrX.g28113\"; transcript_id \"chrX.g28113.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105278891', '105278925', '.', '-', '.', 'gene_id \"chrX.g28119\"; transcript_id \"chrX.g28119.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105321668', '105321700', '.', '+', '.', 'gene_id \"chrX.g28170\"; transcript_id \"chrX.g28170.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105348992', '105349180', '.', '+', '.', 'gene_id \"chrX.g28190\"; transcript_id \"chrX.g28190.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105474457', '105474547', '.', '+', '.', 'gene_id \"chrX.g28227\"; transcript_id \"chrX.g28227.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105511505', '105511544', '.', '+', '.', 'gene_id \"chrX.g28249\"; transcript_id \"chrX.g28249.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105550466', '105550650', '.', '+', '.', 'gene_id \"chrX.g28282\"; transcript_id \"chrX.g28282.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105606477', '105606514', '.', '+', '.', 'gene_id \"chrX.g28315\"; transcript_id \"chrX.g28315.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105627714', '105627758', '.', '-', '.', 'gene_id \"chrX.g28318\"; transcript_id \"chrX.g28318.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105677394', '105677641', '.', '-', '.', 'gene_id \"chrX.g28365\"; transcript_id \"chrX.g28365.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105677733', '105677908', '.', '+', '.', 'gene_id \"chrX.g28366\"; transcript_id \"chrX.g28366.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56525252', '56525484', '.', '-', '.', 'gene_id \"chr5.g24043\"; transcript_id \"chr5.g24043.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56525489', '56526259', '.', '+', '.', 'gene_id \"chr5.g24044\"; transcript_id \"chr5.g24044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56526331', '56526954', '.', '-', '.', 'gene_id \"chr5.g24045\"; transcript_id \"chr5.g24045.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56526969', '56527139', '.', '+', '.', 'gene_id \"chr5.g24046\"; transcript_id \"chr5.g24046.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105867484', '105867527', '.', '+', '.', 'gene_id \"chrX.g28455\"; transcript_id \"chrX.g28455.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105994004', '105994067', '.', '-', '.', 'gene_id \"chrX.g28495\"; transcript_id \"chrX.g28495.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106071229', '106071311', '.', '+', '.', 'gene_id \"chrX.g28509\"; transcript_id \"chrX.g28509.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56736681', '56737142', '.', '-', '.', 'gene_id \"chr5.g24215\"; transcript_id \"chr5.g24215.i13\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56742665', '56742885', '.', '+', '.', 'gene_id \"chr5.g24216\"; transcript_id \"chr5.g24216.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56867855', '56867987', '.', '-', '.', 'gene_id \"chr5.g24318\"; transcript_id \"chr5.g24318.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56867855', '56868002', '.', '-', '.', 'gene_id \"chr5.g24318\"; transcript_id \"chr5.g24318.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106454785', '106454883', '.', '+', '.', 'gene_id \"chrX.g28677\"; transcript_id \"chrX.g28677.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106713528', '106713688', '.', '+', '.', 'gene_id \"chrX.g28701\"; transcript_id \"chrX.g28701.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106723102', '106723145', '.', '+', '.', 'gene_id \"chrX.g28707\"; transcript_id \"chrX.g28707.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107053429', '107053461', '.', '-', '.', 'gene_id \"chrX.g28764\"; transcript_id \"chrX.g28764.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107233303', '107233338', '.', '+', '.', 'gene_id \"chrX.g28792\"; transcript_id \"chrX.g28792.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107373747', '107373794', '.', '-', '.', 'gene_id \"chrX.g28811\"; transcript_id \"chrX.g28811.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57051266', '57051726', '.', '-', '.', 'gene_id \"chr5.g24560\"; transcript_id \"chr5.g24560.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107392827', '107393127', '.', '-', '.', 'gene_id \"chrX.g28817\"; transcript_id \"chrX.g28817.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107391412', '107392411', '.', '-', '.', 'gene_id \"chrX.g28818\"; transcript_id \"chrX.g28818.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57051734', '57052184', '.', '-', '.', 'gene_id \"chr5.g24561\"; transcript_id \"chr5.g24561.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57059009', '57059077', '.', '-', '.', 'gene_id \"chr5.g24569\"; transcript_id \"chr5.g24569.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57085748', '57085777', '.', '+', '.', 'gene_id \"chr5.g24586\"; transcript_id \"chr5.g24586.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107698819', '107698858', '.', '+', '.', 'gene_id \"chrX.g28861\"; transcript_id \"chrX.g28861.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107846449', '107847386', '.', '+', '.', 'gene_id \"chrX.g28879\"; transcript_id \"chrX.g28879.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107846035', '107846425', '.', '-', '.', 'gene_id \"chrX.g28886\"; transcript_id \"chrX.g28886.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108000419', '108000546', '.', '-', '.', 'gene_id \"chrX.g28902\"; transcript_id \"chrX.g28902.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108121561', '108121627', '.', '-', '.', 'gene_id \"chrX.g28924\"; transcript_id \"chrX.g28924.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108210420', '108210455', '.', '+', '.', 'gene_id \"chrX.g28936\"; transcript_id \"chrX.g28936.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108339657', '108339936', '.', '+', '.', 'gene_id \"chrX.g28949\"; transcript_id \"chrX.g28949.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108537227', '108537380', '.', '+', '.', 'gene_id \"chrX.g28981\"; transcript_id \"chrX.g28981.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108550951', '108550992', '.', '-', '.', 'gene_id \"chrX.g28986\"; transcript_id \"chrX.g28986.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108669548', '108669598', '.', '-', '.', 'gene_id \"chrX.g29028\"; transcript_id \"chrX.g29028.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108717745', '108717883', '.', '+', '.', 'gene_id \"chrX.g29039\"; transcript_id \"chrX.g29039.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108728409', '108728445', '.', '-', '.', 'gene_id \"chrX.g29044\"; transcript_id \"chrX.g29044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57447241', '57447392', '.', '+', '.', 'gene_id \"chr5.g24862\"; transcript_id \"chr5.g24862.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108995125', '108995275', '.', '+', '.', 'gene_id \"chrX.g29139\"; transcript_id \"chrX.g29139.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109048257', '109048315', '.', '-', '.', 'gene_id \"chrX.g29159\"; transcript_id \"chrX.g29159.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109060410', '109060460', '.', '+', '.', 'gene_id \"chrX.g29163\"; transcript_id \"chrX.g29163.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109098322', '109098359', '.', '+', '.', 'gene_id \"chrX.g29173\"; transcript_id \"chrX.g29173.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109280468', '109280693', '.', '-', '.', 'gene_id \"chrX.g29227\"; transcript_id \"chrX.g29227.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57882693', '57882901', '.', '+', '.', 'gene_id \"chr5.g25020\"; transcript_id \"chr5.g25020.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '57958402', '57958559', '.', '+', '.', 'gene_id \"chr5.g25032\"; transcript_id \"chr5.g25032.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109394963', '109395039', '.', '+', '.', 'gene_id \"chrX.g29250\"; transcript_id \"chrX.g29250.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109452169', '109452248', '.', '-', '.', 'gene_id \"chrX.g29254\"; transcript_id \"chrX.g29254.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109478964', '109478997', '.', '+', '.', 'gene_id \"chrX.g29258\"; transcript_id \"chrX.g29258.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '58178793', '58178962', '.', '-', '.', 'gene_id \"chr5.g25060\"; transcript_id \"chr5.g25060.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109657202', '109658476', '.', '-', '.', 'gene_id \"chrX.g29289\"; transcript_id \"chrX.g29289.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109835293', '109835418', '.', '+', '.', 'gene_id \"chrX.g29344\"; transcript_id \"chrX.g29344.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109901061', '109901181', '.', '+', '.', 'gene_id \"chrX.g29357\"; transcript_id \"chrX.g29357.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110049707', '110049754', '.', '+', '.', 'gene_id \"chrX.g29387\"; transcript_id \"chrX.g29387.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110117364', '110117584', '.', '-', '.', 'gene_id \"chrX.g29407\"; transcript_id \"chrX.g29407.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110242968', '110243030', '.', '-', '.', 'gene_id \"chrX.g29458\"; transcript_id \"chrX.g29458.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110301233', '110301398', '.', '-', '.', 'gene_id \"chrX.g29470\"; transcript_id \"chrX.g29470.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110564690', '110564740', '.', '-', '.', 'gene_id \"chrX.g29508\"; transcript_id \"chrX.g29508.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110668185', '110668357', '.', '+', '.', 'gene_id \"chrX.g29513\"; transcript_id \"chrX.g29513.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59298542', '59298831', '.', '+', '.', 'gene_id \"chr5.g25355\"; transcript_id \"chr5.g25355.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59934013', '59934205', '.', '-', '.', 'gene_id \"chr5.g25372\"; transcript_id \"chr5.g25372.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59940907', '59941163', '.', '-', '.', 'gene_id \"chr5.g25373\"; transcript_id \"chr5.g25373.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110873985', '110874188', '.', '+', '.', 'gene_id \"chrX.g29535\"; transcript_id \"chrX.g29535.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59963682', '59963883', '.', '-', '.', 'gene_id \"chr5.g25376\"; transcript_id \"chr5.g25376.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59968970', '59969145', '.', '+', '.', 'gene_id \"chr5.g25377\"; transcript_id \"chr5.g25377.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110914745', '110914922', '.', '+', '.', 'gene_id \"chrX.g29539\"; transcript_id \"chrX.g29539.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59977417', '59977592', '.', '-', '.', 'gene_id \"chr5.g25381\"; transcript_id \"chr5.g25381.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59980048', '59980199', '.', '-', '.', 'gene_id \"chr5.g25383\"; transcript_id \"chr5.g25383.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59980199', '59980543', '.', '+', '.', 'gene_id \"chr5.g25384\"; transcript_id \"chr5.g25384.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59983041', '59983356', '.', '+', '.', 'gene_id \"chr5.g25386\"; transcript_id \"chr5.g25386.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59985504', '59985682', '.', '+', '.', 'gene_id \"chr5.g25387\"; transcript_id \"chr5.g25387.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60075182', '60075370', '.', '-', '.', 'gene_id \"chr5.g25393\"; transcript_id \"chr5.g25393.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60030577', '60030817', '.', '+', '.', 'gene_id \"chr5.g25398\"; transcript_id \"chr5.g25398.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60030852', '60031005', '.', '+', '.', 'gene_id \"chr5.g25399\"; transcript_id \"chr5.g25399.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60032365', '60032635', '.', '-', '.', 'gene_id \"chr5.g25400\"; transcript_id \"chr5.g25400.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60034311', '60034579', '.', '-', '.', 'gene_id \"chr5.g25402\"; transcript_id \"chr5.g25402.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60039170', '60039419', '.', '+', '.', 'gene_id \"chr5.g25404\"; transcript_id \"chr5.g25404.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60040875', '60041034', '.', '+', '.', 'gene_id \"chr5.g25405\"; transcript_id \"chr5.g25405.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60042567', '60042821', '.', '-', '.', 'gene_id \"chr5.g25407\"; transcript_id \"chr5.g25407.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60045762', '60045947', '.', '-', '.', 'gene_id \"chr5.g25411\"; transcript_id \"chr5.g25411.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60047348', '60047517', '.', '+', '.', 'gene_id \"chr5.g25415\"; transcript_id \"chr5.g25415.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60055016', '60055240', '.', '+', '.', 'gene_id \"chr5.g25417\"; transcript_id \"chr5.g25417.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60139859', '60140034', '.', '-', '.', 'gene_id \"chr5.g25427\"; transcript_id \"chr5.g25427.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60140067', '60140270', '.', '-', '.', 'gene_id \"chr5.g25428\"; transcript_id \"chr5.g25428.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60230185', '60230561', '.', '+', '.', 'gene_id \"chr5.g25432\"; transcript_id \"chr5.g25432.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60230020', '60230194', '.', '-', '.', 'gene_id \"chr5.g25436\"; transcript_id \"chr5.g25436.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60185593', '60185751', '.', '+', '.', 'gene_id \"chr5.g25437\"; transcript_id \"chr5.g25437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60184693', '60184934', '.', '+', '.', 'gene_id \"chr5.g25441\"; transcript_id \"chr5.g25441.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60187556', '60187768', '.', '-', '.', 'gene_id \"chr5.g25443\"; transcript_id \"chr5.g25443.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60198664', '60199103', '.', '+', '.', 'gene_id \"chr5.g25449\"; transcript_id \"chr5.g25449.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60218938', '60219137', '.', '+', '.', 'gene_id \"chr5.g25459\"; transcript_id \"chr5.g25459.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111699389', '111699429', '.', '-', '.', 'gene_id \"chrX.g29613\"; transcript_id \"chrX.g29613.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60229483', '60229636', '.', '-', '.', 'gene_id \"chr5.g25462\"; transcript_id \"chr5.g25462.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60245494', '60245694', '.', '+', '.', 'gene_id \"chr5.g25466\"; transcript_id \"chr5.g25466.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60247020', '60247178', '.', '+', '.', 'gene_id \"chr5.g25467\"; transcript_id \"chr5.g25467.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60262828', '60263002', '.', '-', '.', 'gene_id \"chr5.g25469\"; transcript_id \"chr5.g25469.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60291097', '60291255', '.', '-', '.', 'gene_id \"chr5.g25476\"; transcript_id \"chr5.g25476.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111842509', '111842759', '.', '+', '.', 'gene_id \"chrX.g29633\"; transcript_id \"chrX.g29633.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60304607', '60304825', '.', '+', '.', 'gene_id \"chr5.g25481\"; transcript_id \"chr5.g25481.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60307144', '60307722', '.', '-', '.', 'gene_id \"chr5.g25483\"; transcript_id \"chr5.g25483.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60307144', '60307710', '.', '-', '.', 'gene_id \"chr5.g25483\"; transcript_id \"chr5.g25483.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60313146', '60314047', '.', '+', '.', 'gene_id \"chr5.g25485\"; transcript_id \"chr5.g25485.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60320275', '60322190', '.', '-', '.', 'gene_id \"chr5.g25491\"; transcript_id \"chr5.g25491.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60329443', '60329581', '.', '+', '.', 'gene_id \"chr5.g25496\"; transcript_id \"chr5.g25496.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60330867', '60331256', '.', '+', '.', 'gene_id \"chr5.g25498\"; transcript_id \"chr5.g25498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60341870', '60342295', '.', '+', '.', 'gene_id \"chr5.g25502\"; transcript_id \"chr5.g25502.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60358974', '60360077', '.', '+', '.', 'gene_id \"chr5.g25506\"; transcript_id \"chr5.g25506.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60361378', '60361567', '.', '+', '.', 'gene_id \"chr5.g25507\"; transcript_id \"chr5.g25507.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60361993', '60362153', '.', '-', '.', 'gene_id \"chr5.g25508\"; transcript_id \"chr5.g25508.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60368685', '60369075', '.', '-', '.', 'gene_id \"chr5.g25512\"; transcript_id \"chr5.g25512.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60392848', '60393125', '.', '+', '.', 'gene_id \"chr5.g25515\"; transcript_id \"chr5.g25515.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60420930', '60421160', '.', '-', '.', 'gene_id \"chr5.g25517\"; transcript_id \"chr5.g25517.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60438095', '60438269', '.', '+', '.', 'gene_id \"chr5.g25518\"; transcript_id \"chr5.g25518.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60382052', '60382270', '.', '-', '.', 'gene_id \"chr5.g25519\"; transcript_id \"chr5.g25519.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60373414', '60373594', '.', '+', '.', 'gene_id \"chr5.g25526\"; transcript_id \"chr5.g25526.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60375348', '60375851', '.', '-', '.', 'gene_id \"chr5.g25527\"; transcript_id \"chr5.g25527.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111982715', '111982982', '.', '+', '.', 'gene_id \"chrX.g29667\"; transcript_id \"chrX.g29667.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60376836', '60377465', '.', '-', '.', 'gene_id \"chr5.g25530\"; transcript_id \"chr5.g25530.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60385419', '60385594', '.', '+', '.', 'gene_id \"chr5.g25537\"; transcript_id \"chr5.g25537.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60386045', '60386270', '.', '-', '.', 'gene_id \"chr5.g25538\"; transcript_id \"chr5.g25538.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60386170', '60386270', '.', '-', '.', 'gene_id \"chr5.g25538\"; transcript_id \"chr5.g25538.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60416732', '60416874', '.', '-', '.', 'gene_id \"chr5.g25543\"; transcript_id \"chr5.g25543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60419486', '60419644', '.', '+', '.', 'gene_id \"chr5.g25546\"; transcript_id \"chr5.g25546.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60471796', '60472191', '.', '+', '.', 'gene_id \"chr5.g25549\"; transcript_id \"chr5.g25549.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60522653', '60523533', '.', '-', '.', 'gene_id \"chr5.g25565\"; transcript_id \"chr5.g25565.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60520174', '60520331', '.', '-', '.', 'gene_id \"chr5.g25566\"; transcript_id \"chr5.g25566.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60537948', '60538297', '.', '-', '.', 'gene_id \"chr5.g25571\"; transcript_id \"chr5.g25571.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60532671', '60532823', '.', '-', '.', 'gene_id \"chr5.g25572\"; transcript_id \"chr5.g25572.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60532391', '60532578', '.', '-', '.', 'gene_id \"chr5.g25574\"; transcript_id \"chr5.g25574.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60556780', '60557036', '.', '-', '.', 'gene_id \"chr5.g25578\"; transcript_id \"chr5.g25578.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60608577', '60608760', '.', '+', '.', 'gene_id \"chr5.g25590\"; transcript_id \"chr5.g25590.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60591180', '60591586', '.', '+', '.', 'gene_id \"chr5.g25593\"; transcript_id \"chr5.g25593.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60593694', '60593908', '.', '+', '.', 'gene_id \"chr5.g25597\"; transcript_id \"chr5.g25597.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60593886', '60594032', '.', '-', '.', 'gene_id \"chr5.g25598\"; transcript_id \"chr5.g25598.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60594525', '60594923', '.', '+', '.', 'gene_id \"chr5.g25599\"; transcript_id \"chr5.g25599.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60597684', '60597867', '.', '+', '.', 'gene_id \"chr5.g25601\"; transcript_id \"chr5.g25601.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60600420', '60600837', '.', '+', '.', 'gene_id \"chr5.g25604\"; transcript_id \"chr5.g25604.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112574323', '112574383', '.', '+', '.', 'gene_id \"chrX.g29730\"; transcript_id \"chrX.g29730.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60601127', '60601366', '.', '+', '.', 'gene_id \"chr5.g25605\"; transcript_id \"chr5.g25605.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60601164', '60601366', '.', '+', '.', 'gene_id \"chr5.g25605\"; transcript_id \"chr5.g25605.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112615089', '112615129', '.', '-', '.', 'gene_id \"chrX.g29734\"; transcript_id \"chrX.g29734.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60606792', '60607471', '.', '-', '.', 'gene_id \"chr5.g25609\"; transcript_id \"chr5.g25609.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60613105', '60613273', '.', '-', '.', 'gene_id \"chr5.g25611\"; transcript_id \"chr5.g25611.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60628718', '60628878', '.', '+', '.', 'gene_id \"chr5.g25620\"; transcript_id \"chr5.g25620.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60636219', '60636500', '.', '+', '.', 'gene_id \"chr5.g25625\"; transcript_id \"chr5.g25625.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60640303', '60640477', '.', '+', '.', 'gene_id \"chr5.g25626\"; transcript_id \"chr5.g25626.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60643272', '60643461', '.', '-', '.', 'gene_id \"chr5.g25628\"; transcript_id \"chr5.g25628.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112758376', '112758406', '.', '-', '.', 'gene_id \"chrX.g29756\"; transcript_id \"chrX.g29756.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60669513', '60670594', '.', '-', '.', 'gene_id \"chr5.g25639\"; transcript_id \"chr5.g25639.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60668962', '60669504', '.', '-', '.', 'gene_id \"chr5.g25639\"; transcript_id \"chr5.g25639.i5\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112796919', '112796995', '.', '+', '.', 'gene_id \"chrX.g29764\"; transcript_id \"chrX.g29764.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60674071', '60674271', '.', '+', '.', 'gene_id \"chr5.g25647\"; transcript_id \"chr5.g25647.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60674269', '60674780', '.', '+', '.', 'gene_id \"chr5.g25648\"; transcript_id \"chr5.g25648.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60685781', '60686454', '.', '-', '.', 'gene_id \"chr5.g25655\"; transcript_id \"chr5.g25655.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60687047', '60687449', '.', '-', '.', 'gene_id \"chr5.g25659\"; transcript_id \"chr5.g25659.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60687533', '60687898', '.', '-', '.', 'gene_id \"chr5.g25660\"; transcript_id \"chr5.g25660.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60687533', '60687898', '.', '-', '.', 'gene_id \"chr5.g25660\"; transcript_id \"chr5.g25660.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60688485', '60688804', '.', '+', '.', 'gene_id \"chr5.g25661\"; transcript_id \"chr5.g25661.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60692989', '60693154', '.', '+', '.', 'gene_id \"chr5.g25662\"; transcript_id \"chr5.g25662.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60693150', '60693313', '.', '-', '.', 'gene_id \"chr5.g25663\"; transcript_id \"chr5.g25663.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60694163', '60694315', '.', '+', '.', 'gene_id \"chr5.g25664\"; transcript_id \"chr5.g25664.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60695211', '60695394', '.', '-', '.', 'gene_id \"chr5.g25665\"; transcript_id \"chr5.g25665.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60695211', '60695436', '.', '-', '.', 'gene_id \"chr5.g25665\"; transcript_id \"chr5.g25665.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112982383', '112982436', '.', '-', '.', 'gene_id \"chrX.g29784\"; transcript_id \"chrX.g29784.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60702169', '60702325', '.', '-', '.', 'gene_id \"chr5.g25670\"; transcript_id \"chr5.g25670.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60706072', '60706316', '.', '-', '.', 'gene_id \"chr5.g25672\"; transcript_id \"chr5.g25672.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60706940', '60707564', '.', '-', '.', 'gene_id \"chr5.g25674\"; transcript_id \"chr5.g25674.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60711130', '60711343', '.', '-', '.', 'gene_id \"chr5.g25675\"; transcript_id \"chr5.g25675.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60728198', '60728380', '.', '+', '.', 'gene_id \"chr5.g25677\"; transcript_id \"chr5.g25677.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60730637', '60730823', '.', '-', '.', 'gene_id \"chr5.g25678\"; transcript_id \"chr5.g25678.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60756761', '60756971', '.', '-', '.', 'gene_id \"chr5.g25685\"; transcript_id \"chr5.g25685.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60812236', '60812268', '.', '-', '.', 'gene_id \"chr5.g25696\"; transcript_id \"chr5.g25696.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60822401', '60822566', '.', '-', '.', 'gene_id \"chr5.g25697\"; transcript_id \"chr5.g25697.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60824563', '60824821', '.', '+', '.', 'gene_id \"chr5.g25699\"; transcript_id \"chr5.g25699.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60833597', '60833791', '.', '-', '.', 'gene_id \"chr5.g25701\"; transcript_id \"chr5.g25701.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60837742', '60837991', '.', '-', '.', 'gene_id \"chr5.g25706\"; transcript_id \"chr5.g25706.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60856746', '60856983', '.', '+', '.', 'gene_id \"chr5.g25715\"; transcript_id \"chr5.g25715.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60856944', '60857166', '.', '-', '.', 'gene_id \"chr5.g25716\"; transcript_id \"chr5.g25716.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60857352', '60857874', '.', '-', '.', 'gene_id \"chr5.g25717\"; transcript_id \"chr5.g25717.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60861994', '60862438', '.', '+', '.', 'gene_id \"chr5.g25719\"; transcript_id \"chr5.g25719.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113494562', '113494603', '.', '-', '.', 'gene_id \"chrX.g29837\"; transcript_id \"chrX.g29837.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60869657', '60869865', '.', '+', '.', 'gene_id \"chr5.g25723\"; transcript_id \"chr5.g25723.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60879443', '60879723', '.', '-', '.', 'gene_id \"chr5.g25727\"; transcript_id \"chr5.g25727.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60881183', '60881336', '.', '-', '.', 'gene_id \"chr5.g25728\"; transcript_id \"chr5.g25728.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60882228', '60882420', '.', '+', '.', 'gene_id \"chr5.g25729\"; transcript_id \"chr5.g25729.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60882433', '60882809', '.', '-', '.', 'gene_id \"chr5.g25730\"; transcript_id \"chr5.g25730.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113615616', '113615646', '.', '-', '.', 'gene_id \"chrX.g29849\"; transcript_id \"chrX.g29849.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60883440', '60883705', '.', '+', '.', 'gene_id \"chr5.g25731\"; transcript_id \"chr5.g25731.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60883740', '60883952', '.', '+', '.', 'gene_id \"chr5.g25732\"; transcript_id \"chr5.g25732.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113625461', '113625536', '.', '-', '.', 'gene_id \"chrX.g29851\"; transcript_id \"chrX.g29851.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113625461', '113625535', '.', '-', '.', 'gene_id \"chrX.g29851\"; transcript_id \"chrX.g29851.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60895354', '60895841', '.', '-', '.', 'gene_id \"chr5.g25738\"; transcript_id \"chr5.g25738.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60896020', '60896701', '.', '+', '.', 'gene_id \"chr5.g25740\"; transcript_id \"chr5.g25740.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60897023', '60897231', '.', '+', '.', 'gene_id \"chr5.g25741\"; transcript_id \"chr5.g25741.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60897399', '60897667', '.', '-', '.', 'gene_id \"chr5.g25742\"; transcript_id \"chr5.g25742.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113680553', '113680606', '.', '+', '.', 'gene_id \"chrX.g29861\"; transcript_id \"chrX.g29861.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60898312', '60898744', '.', '-', '.', 'gene_id \"chr5.g25745\"; transcript_id \"chr5.g25745.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60900348', '60900858', '.', '-', '.', 'gene_id \"chr5.g25747\"; transcript_id \"chr5.g25747.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60902475', '60903884', '.', '-', '.', 'gene_id \"chr5.g25749\"; transcript_id \"chr5.g25749.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60903890', '60904136', '.', '-', '.', 'gene_id \"chr5.g25750\"; transcript_id \"chr5.g25750.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61077671', '61077860', '.', '-', '.', 'gene_id \"chr5.g25757\"; transcript_id \"chr5.g25757.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61640586', '61642357', '.', '+', '.', 'gene_id \"chr5.g25760\"; transcript_id \"chr5.g25760.i17\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61224897', '61225049', '.', '+', '.', 'gene_id \"chr5.g25768\"; transcript_id \"chr5.g25768.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113964075', '113964115', '.', '-', '.', 'gene_id \"chrX.g29889\"; transcript_id \"chrX.g29889.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61313183', '61313408', '.', '-', '.', 'gene_id \"chr5.g25775\"; transcript_id \"chr5.g25775.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61494550', '61494704', '.', '+', '.', 'gene_id \"chr5.g25783\"; transcript_id \"chr5.g25783.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61611618', '61611802', '.', '+', '.', 'gene_id \"chr5.g25794\"; transcript_id \"chr5.g25794.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61614023', '61614300', '.', '-', '.', 'gene_id \"chr5.g25796\"; transcript_id \"chr5.g25796.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61621744', '61621895', '.', '+', '.', 'gene_id \"chr5.g25797\"; transcript_id \"chr5.g25797.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61631496', '61631915', '.', '+', '.', 'gene_id \"chr5.g25800\"; transcript_id \"chr5.g25800.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61635378', '61635743', '.', '+', '.', 'gene_id \"chr5.g25805\"; transcript_id \"chr5.g25805.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114323643', '114323683', '.', '-', '.', 'gene_id \"chrX.g29924\"; transcript_id \"chrX.g29924.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61649138', '61649306', '.', '-', '.', 'gene_id \"chr5.g25811\"; transcript_id \"chr5.g25811.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61650265', '61650423', '.', '-', '.', 'gene_id \"chr5.g25817\"; transcript_id \"chr5.g25817.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61651584', '61651748', '.', '-', '.', 'gene_id \"chr5.g25818\"; transcript_id \"chr5.g25818.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114349291', '114349336', '.', '-', '.', 'gene_id \"chrX.g29929\"; transcript_id \"chrX.g29929.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61657275', '61657433', '.', '+', '.', 'gene_id \"chr5.g25819\"; transcript_id \"chr5.g25819.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61658341', '61658586', '.', '-', '.', 'gene_id \"chr5.g25821\"; transcript_id \"chr5.g25821.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61658902', '61659118', '.', '-', '.', 'gene_id \"chr5.g25823\"; transcript_id \"chr5.g25823.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61680897', '61681425', '.', '-', '.', 'gene_id \"chr5.g25828\"; transcript_id \"chr5.g25828.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61692715', '61693528', '.', '-', '.', 'gene_id \"chr5.g25830\"; transcript_id \"chr5.g25830.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61679739', '61680713', '.', '-', '.', 'gene_id \"chr5.g25832\"; transcript_id \"chr5.g25832.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61676924', '61677219', '.', '-', '.', 'gene_id \"chr5.g25834\"; transcript_id \"chr5.g25834.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61665501', '61665670', '.', '+', '.', 'gene_id \"chr5.g25835\"; transcript_id \"chr5.g25835.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61694328', '61695107', '.', '+', '.', 'gene_id \"chr5.g25840\"; transcript_id \"chr5.g25840.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61667020', '61667338', '.', '+', '.', 'gene_id \"chr5.g25842\"; transcript_id \"chr5.g25842.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114439340', '114439495', '.', '+', '.', 'gene_id \"chrX.g29941\"; transcript_id \"chrX.g29941.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61669172', '61669369', '.', '-', '.', 'gene_id \"chr5.g25845\"; transcript_id \"chr5.g25845.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61669637', '61669872', '.', '-', '.', 'gene_id \"chr5.g25846\"; transcript_id \"chr5.g25846.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61681528', '61681685', '.', '+', '.', 'gene_id \"chr5.g25847\"; transcript_id \"chr5.g25847.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61683984', '61684190', '.', '-', '.', 'gene_id \"chr5.g25849\"; transcript_id \"chr5.g25849.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61684215', '61684371', '.', '+', '.', 'gene_id \"chr5.g25850\"; transcript_id \"chr5.g25850.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114585267', '114585309', '.', '+', '.', 'gene_id \"chrX.g29950\"; transcript_id \"chrX.g29950.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61708945', '61709201', '.', '+', '.', 'gene_id \"chr5.g25859\"; transcript_id \"chr5.g25859.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61709466', '61709908', '.', '+', '.', 'gene_id \"chr5.g25860\"; transcript_id \"chr5.g25860.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61712279', '61712541', '.', '+', '.', 'gene_id \"chr5.g25861\"; transcript_id \"chr5.g25861.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61718394', '61718584', '.', '-', '.', 'gene_id \"chr5.g25862\"; transcript_id \"chr5.g25862.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61719836', '61720038', '.', '+', '.', 'gene_id \"chr5.g25864\"; transcript_id \"chr5.g25864.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61736654', '61736993', '.', '+', '.', 'gene_id \"chr5.g25871\"; transcript_id \"chr5.g25871.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61739680', '61740561', '.', '-', '.', 'gene_id \"chr5.g25874\"; transcript_id \"chr5.g25874.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61751894', '61752237', '.', '-', '.', 'gene_id \"chr5.g25879\"; transcript_id \"chr5.g25879.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61755542', '61755693', '.', '-', '.', 'gene_id \"chr5.g25880\"; transcript_id \"chr5.g25880.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61761003', '61761074', '.', '-', '.', 'gene_id \"chr5.g25883\"; transcript_id \"chr5.g25883.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61761004', '61761074', '.', '-', '.', 'gene_id \"chr5.g25883\"; transcript_id \"chr5.g25883.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114924700', '114924938', '.', '+', '.', 'gene_id \"chrX.g29984\"; transcript_id \"chrX.g29984.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61762979', '61763294', '.', '+', '.', 'gene_id \"chr5.g25887\"; transcript_id \"chr5.g25887.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61764794', '61764992', '.', '+', '.', 'gene_id \"chr5.g25888\"; transcript_id \"chr5.g25888.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61765047', '61765330', '.', '-', '.', 'gene_id \"chr5.g25889\"; transcript_id \"chr5.g25889.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61771658', '61771862', '.', '-', '.', 'gene_id \"chr5.g25893\"; transcript_id \"chr5.g25893.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61779607', '61779974', '.', '-', '.', 'gene_id \"chr5.g25900\"; transcript_id \"chr5.g25900.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61780444', '61780682', '.', '+', '.', 'gene_id \"chr5.g25901\"; transcript_id \"chr5.g25901.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61780697', '61780883', '.', '-', '.', 'gene_id \"chr5.g25902\"; transcript_id \"chr5.g25902.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61783187', '61785301', '.', '+', '.', 'gene_id \"chr5.g25905\"; transcript_id \"chr5.g25905.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61785452', '61786392', '.', '+', '.', 'gene_id \"chr5.g25905\"; transcript_id \"chr5.g25905.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61791423', '61791582', '.', '+', '.', 'gene_id \"chr5.g25906\"; transcript_id \"chr5.g25906.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61800999', '61801435', '.', '+', '.', 'gene_id \"chr5.g25909\"; transcript_id \"chr5.g25909.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61796624', '61797481', '.', '-', '.', 'gene_id \"chr5.g25914\"; transcript_id \"chr5.g25914.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61797873', '61800953', '.', '-', '.', 'gene_id \"chr5.g25915\"; transcript_id \"chr5.g25915.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61843405', '61844457', '.', '+', '.', 'gene_id \"chr5.g25931\"; transcript_id \"chr5.g25931.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61853406', '61853613', '.', '-', '.', 'gene_id \"chr5.g25933\"; transcript_id \"chr5.g25933.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61857776', '61857936', '.', '+', '.', 'gene_id \"chr5.g25937\"; transcript_id \"chr5.g25937.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61857520', '61857788', '.', '-', '.', 'gene_id \"chr5.g25938\"; transcript_id \"chr5.g25938.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61857916', '61858212', '.', '-', '.', 'gene_id \"chr5.g25939\"; transcript_id \"chr5.g25939.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61881040', '61881083', '.', '+', '.', 'gene_id \"chr5.g25941\"; transcript_id \"chr5.g25941.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61890298', '61890331', '.', '+', '.', 'gene_id \"chr5.g25942\"; transcript_id \"chr5.g25942.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61948746', '61949020', '.', '+', '.', 'gene_id \"chr5.g25952\"; transcript_id \"chr5.g25952.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61969319', '61969570', '.', '+', '.', 'gene_id \"chr5.g25955\"; transcript_id \"chr5.g25955.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61973862', '61975979', '.', '+', '.', 'gene_id \"chr5.g25956\"; transcript_id \"chr5.g25956.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61971188', '61973774', '.', '+', '.', 'gene_id \"chr5.g25956\"; transcript_id \"chr5.g25956.i3\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115343724', '115343798', '.', '-', '.', 'gene_id \"chrX.g30051\"; transcript_id \"chrX.g30051.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62001081', '62004936', '.', '-', '.', 'gene_id \"chr5.g25966\"; transcript_id \"chr5.g25966.i9\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62001081', '62003572', '.', '-', '.', 'gene_id \"chr5.g25966\"; transcript_id \"chr5.g25966.i10\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62007591', '62011709', '.', '-', '.', 'gene_id \"chr5.g25966\"; transcript_id \"chr5.g25966.i12\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61992158', '61993456', '.', '-', '.', 'gene_id \"chr5.g25966\"; transcript_id \"chr5.g25966.i13\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115379500', '115379543', '.', '+', '.', 'gene_id \"chrX.g30055\"; transcript_id \"chrX.g30055.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62014002', '62014346', '.', '+', '.', 'gene_id \"chr5.g25974\"; transcript_id \"chr5.g25974.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62021236', '62021696', '.', '-', '.', 'gene_id \"chr5.g25978\"; transcript_id \"chr5.g25978.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62021692', '62021856', '.', '+', '.', 'gene_id \"chr5.g25979\"; transcript_id \"chr5.g25979.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62031699', '62031853', '.', '+', '.', 'gene_id \"chr5.g25982\"; transcript_id \"chr5.g25982.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62035874', '62036145', '.', '-', '.', 'gene_id \"chr5.g25985\"; transcript_id \"chr5.g25985.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62045440', '62045675', '.', '+', '.', 'gene_id \"chr5.g25990\"; transcript_id \"chr5.g25990.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62045834', '62046001', '.', '+', '.', 'gene_id \"chr5.g25992\"; transcript_id \"chr5.g25992.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62046750', '62046964', '.', '+', '.', 'gene_id \"chr5.g25995\"; transcript_id \"chr5.g25995.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62047785', '62048095', '.', '-', '.', 'gene_id \"chr5.g25998\"; transcript_id \"chr5.g25998.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62048959', '62049284', '.', '+', '.', 'gene_id \"chr5.g26001\"; transcript_id \"chr5.g26001.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62060316', '62060611', '.', '+', '.', 'gene_id \"chr5.g26006\"; transcript_id \"chr5.g26006.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115675042', '115675848', '.', '-', '.', 'gene_id \"chrX.g30098\"; transcript_id \"chrX.g30098.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62069379', '62069845', '.', '-', '.', 'gene_id \"chr5.g26010\"; transcript_id \"chr5.g26010.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62075176', '62075410', '.', '-', '.', 'gene_id \"chr5.g26014\"; transcript_id \"chr5.g26014.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62078809', '62078991', '.', '-', '.', 'gene_id \"chr5.g26019\"; transcript_id \"chr5.g26019.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62139889', '62139970', '.', '+', '.', 'gene_id \"chr5.g26022\"; transcript_id \"chr5.g26022.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62235612', '62235717', '.', '+', '.', 'gene_id \"chr5.g26023\"; transcript_id \"chr5.g26023.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62493512', '62494034', '.', '+', '.', 'gene_id \"chr5.g26027\"; transcript_id \"chr5.g26027.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62092426', '62092589', '.', '+', '.', 'gene_id \"chr5.g26032\"; transcript_id \"chr5.g26032.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62310619', '62310694', '.', '+', '.', 'gene_id \"chr5.g26035\"; transcript_id \"chr5.g26035.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62096478', '62096939', '.', '-', '.', 'gene_id \"chr5.g26036\"; transcript_id \"chr5.g26036.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115979055', '115979106', '.', '-', '.', 'gene_id \"chrX.g30118\"; transcript_id \"chrX.g30118.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62105565', '62106386', '.', '-', '.', 'gene_id \"chr5.g26044\"; transcript_id \"chr5.g26044.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115994346', '115994792', '.', '+', '.', 'gene_id \"chrX.g30120\"; transcript_id \"chrX.g30120.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62111803', '62112174', '.', '-', '.', 'gene_id \"chr5.g26047\"; transcript_id \"chr5.g26047.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62113963', '62114207', '.', '+', '.', 'gene_id \"chr5.g26048\"; transcript_id \"chr5.g26048.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62114205', '62114455', '.', '+', '.', 'gene_id \"chr5.g26049\"; transcript_id \"chr5.g26049.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62114535', '62114741', '.', '+', '.', 'gene_id \"chr5.g26050\"; transcript_id \"chr5.g26050.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62114849', '62115034', '.', '-', '.', 'gene_id \"chr5.g26051\"; transcript_id \"chr5.g26051.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62117759', '62118382', '.', '-', '.', 'gene_id \"chr5.g26057\"; transcript_id \"chr5.g26057.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62120530', '62120865', '.', '+', '.', 'gene_id \"chr5.g26060\"; transcript_id \"chr5.g26060.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62121826', '62122239', '.', '+', '.', 'gene_id \"chr5.g26061\"; transcript_id \"chr5.g26061.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62124868', '62125110', '.', '-', '.', 'gene_id \"chr5.g26065\"; transcript_id \"chr5.g26065.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62125439', '62125663', '.', '-', '.', 'gene_id \"chr5.g26066\"; transcript_id \"chr5.g26066.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '116182941', '116182970', '.', '-', '.', 'gene_id \"chrX.g30141\"; transcript_id \"chrX.g30141.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62127980', '62128264', '.', '-', '.', 'gene_id \"chr5.g26070\"; transcript_id \"chr5.g26070.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62128563', '62128835', '.', '+', '.', 'gene_id \"chr5.g26071\"; transcript_id \"chr5.g26071.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62128907', '62129251', '.', '-', '.', 'gene_id \"chr5.g26073\"; transcript_id \"chr5.g26073.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62133959', '62134313', '.', '-', '.', 'gene_id \"chr5.g26075\"; transcript_id \"chr5.g26075.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62134752', '62135005', '.', '-', '.', 'gene_id \"chr5.g26076\"; transcript_id \"chr5.g26076.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62134991', '62135183', '.', '+', '.', 'gene_id \"chr5.g26077\"; transcript_id \"chr5.g26077.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62139101', '62139288', '.', '-', '.', 'gene_id \"chr5.g26081\"; transcript_id \"chr5.g26081.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62141676', '62141892', '.', '-', '.', 'gene_id \"chr5.g26082\"; transcript_id \"chr5.g26082.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62157815', '62158167', '.', '-', '.', 'gene_id \"chr5.g26089\"; transcript_id \"chr5.g26089.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62162346', '62162535', '.', '-', '.', 'gene_id \"chr5.g26092\"; transcript_id \"chr5.g26092.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62166265', '62166500', '.', '+', '.', 'gene_id \"chr5.g26093\"; transcript_id \"chr5.g26093.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62167888', '62168155', '.', '+', '.', 'gene_id \"chr5.g26095\"; transcript_id \"chr5.g26095.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62172480', '62172769', '.', '-', '.', 'gene_id \"chr5.g26097\"; transcript_id \"chr5.g26097.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62173496', '62173857', '.', '-', '.', 'gene_id \"chr5.g26098\"; transcript_id \"chr5.g26098.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62182129', '62182567', '.', '-', '.', 'gene_id \"chr5.g26102\"; transcript_id \"chr5.g26102.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62182960', '62183205', '.', '-', '.', 'gene_id \"chr5.g26104\"; transcript_id \"chr5.g26104.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62194896', '62195225', '.', '-', '.', 'gene_id \"chr5.g26106\"; transcript_id \"chr5.g26106.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62195306', '62195383', '.', '-', '.', 'gene_id \"chr5.g26107\"; transcript_id \"chr5.g26107.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62195685', '62195857', '.', '-', '.', 'gene_id \"chr5.g26108\"; transcript_id \"chr5.g26108.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62205634', '62206170', '.', '+', '.', 'gene_id \"chr5.g26112\"; transcript_id \"chr5.g26112.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62206945', '62207138', '.', '-', '.', 'gene_id \"chr5.g26113\"; transcript_id \"chr5.g26113.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62211379', '62211689', '.', '-', '.', 'gene_id \"chr5.g26117\"; transcript_id \"chr5.g26117.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62252607', '62252762', '.', '-', '.', 'gene_id \"chr5.g26123\"; transcript_id \"chr5.g26123.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62255354', '62255552', '.', '-', '.', 'gene_id \"chr5.g26124\"; transcript_id \"chr5.g26124.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62259033', '62259269', '.', '+', '.', 'gene_id \"chr5.g26125\"; transcript_id \"chr5.g26125.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62259393', '62259589', '.', '+', '.', 'gene_id \"chr5.g26126\"; transcript_id \"chr5.g26126.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62261944', '62262004', '.', '-', '.', 'gene_id \"chr5.g26128\"; transcript_id \"chr5.g26128.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62265648', '62266071', '.', '-', '.', 'gene_id \"chr5.g26130\"; transcript_id \"chr5.g26130.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62286949', '62287634', '.', '+', '.', 'gene_id \"chr5.g26135\"; transcript_id \"chr5.g26135.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62291959', '62292156', '.', '-', '.', 'gene_id \"chr5.g26137\"; transcript_id \"chr5.g26137.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62293185', '62293365', '.', '+', '.', 'gene_id \"chr5.g26138\"; transcript_id \"chr5.g26138.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62293841', '62293968', '.', '+', '.', 'gene_id \"chr5.g26139\"; transcript_id \"chr5.g26139.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62297370', '62297641', '.', '+', '.', 'gene_id \"chr5.g26143\"; transcript_id \"chr5.g26143.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62306157', '62306368', '.', '+', '.', 'gene_id \"chr5.g26148\"; transcript_id \"chr5.g26148.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62306836', '62307033', '.', '-', '.', 'gene_id \"chr5.g26149\"; transcript_id \"chr5.g26149.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62319332', '62319537', '.', '+', '.', 'gene_id \"chr5.g26152\"; transcript_id \"chr5.g26152.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62320941', '62321179', '.', '-', '.', 'gene_id \"chr5.g26153\"; transcript_id \"chr5.g26153.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62333669', '62333951', '.', '+', '.', 'gene_id \"chr5.g26157\"; transcript_id \"chr5.g26157.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62334398', '62334668', '.', '+', '.', 'gene_id \"chr5.g26158\"; transcript_id \"chr5.g26158.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62338244', '62338415', '.', '+', '.', 'gene_id \"chr5.g26160\"; transcript_id \"chr5.g26160.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62350726', '62350979', '.', '+', '.', 'gene_id \"chr5.g26163\"; transcript_id \"chr5.g26163.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62355120', '62355434', '.', '+', '.', 'gene_id \"chr5.g26165\"; transcript_id \"chr5.g26165.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62371635', '62372033', '.', '+', '.', 'gene_id \"chr5.g26172\"; transcript_id \"chr5.g26172.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62394466', '62394657', '.', '+', '.', 'gene_id \"chr5.g26177\"; transcript_id \"chr5.g26177.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62396536', '62396738', '.', '+', '.', 'gene_id \"chr5.g26183\"; transcript_id \"chr5.g26183.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62396731', '62396884', '.', '-', '.', 'gene_id \"chr5.g26184\"; transcript_id \"chr5.g26184.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62396961', '62397115', '.', '+', '.', 'gene_id \"chr5.g26185\"; transcript_id \"chr5.g26185.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62403879', '62404091', '.', '-', '.', 'gene_id \"chr5.g26187\"; transcript_id \"chr5.g26187.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62404159', '62404342', '.', '+', '.', 'gene_id \"chr5.g26189\"; transcript_id \"chr5.g26189.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62404418', '62404777', '.', '-', '.', 'gene_id \"chr5.g26190\"; transcript_id \"chr5.g26190.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62409493', '62410106', '.', '+', '.', 'gene_id \"chr5.g26192\"; transcript_id \"chr5.g26192.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62411053', '62411315', '.', '-', '.', 'gene_id \"chr5.g26196\"; transcript_id \"chr5.g26196.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62416825', '62416985', '.', '+', '.', 'gene_id \"chr5.g26201\"; transcript_id \"chr5.g26201.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62428243', '62428413', '.', '+', '.', 'gene_id \"chr5.g26206\"; transcript_id \"chr5.g26206.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62428793', '62429208', '.', '-', '.', 'gene_id \"chr5.g26208\"; transcript_id \"chr5.g26208.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62430064', '62430280', '.', '-', '.', 'gene_id \"chr5.g26209\"; transcript_id \"chr5.g26209.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62430635', '62430798', '.', '-', '.', 'gene_id \"chr5.g26210\"; transcript_id \"chr5.g26210.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62444089', '62444400', '.', '-', '.', 'gene_id \"chr5.g26215\"; transcript_id \"chr5.g26215.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62446794', '62447095', '.', '-', '.', 'gene_id \"chr5.g26216\"; transcript_id \"chr5.g26216.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62451003', '62451202', '.', '-', '.', 'gene_id \"chr5.g26219\"; transcript_id \"chr5.g26219.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62451613', '62451878', '.', '+', '.', 'gene_id \"chr5.g26220\"; transcript_id \"chr5.g26220.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62455127', '62455323', '.', '-', '.', 'gene_id \"chr5.g26223\"; transcript_id \"chr5.g26223.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '116882088', '116882119', '.', '+', '.', 'gene_id \"chrX.g30320\"; transcript_id \"chrX.g30320.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62460670', '62461053', '.', '+', '.', 'gene_id \"chr5.g26228\"; transcript_id \"chr5.g26228.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62461881', '62462055', '.', '+', '.', 'gene_id \"chr5.g26229\"; transcript_id \"chr5.g26229.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62463673', '62463731', '.', '-', '.', 'gene_id \"chr5.g26231\"; transcript_id \"chr5.g26231.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62463708', '62463867', '.', '-', '.', 'gene_id \"chr5.g26231\"; transcript_id \"chr5.g26231.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62463445', '62463700', '.', '+', '.', 'gene_id \"chr5.g26232\"; transcript_id \"chr5.g26232.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62467684', '62467815', '.', '-', '.', 'gene_id \"chr5.g26234\"; transcript_id \"chr5.g26234.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '116927321', '116927366', '.', '+', '.', 'gene_id \"chrX.g30328\"; transcript_id \"chrX.g30328.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62471455', '62471755', '.', '+', '.', 'gene_id \"chr5.g26238\"; transcript_id \"chr5.g26238.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62479134', '62479307', '.', '-', '.', 'gene_id \"chr5.g26241\"; transcript_id \"chr5.g26241.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62482476', '62482651', '.', '+', '.', 'gene_id \"chr5.g26244\"; transcript_id \"chr5.g26244.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62483857', '62484131', '.', '-', '.', 'gene_id \"chr5.g26245\"; transcript_id \"chr5.g26245.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62490881', '62491097', '.', '-', '.', 'gene_id \"chr5.g26247\"; transcript_id \"chr5.g26247.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62508182', '62508666', '.', '+', '.', 'gene_id \"chr5.g26254\"; transcript_id \"chr5.g26254.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62528207', '62528917', '.', '+', '.', 'gene_id \"chr5.g26264\"; transcript_id \"chr5.g26264.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62527554', '62528044', '.', '+', '.', 'gene_id \"chr5.g26269\"; transcript_id \"chr5.g26269.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62591268', '62591423', '.', '-', '.', 'gene_id \"chr5.g26281\"; transcript_id \"chr5.g26281.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62603896', '62604104', '.', '+', '.', 'gene_id \"chr5.g26283\"; transcript_id \"chr5.g26283.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '62643326', '62643837', '.', '-', '.', 'gene_id \"chr5.g26286\"; transcript_id \"chr5.g26286.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62650043', '62650343', '.', '-', '.', 'gene_id \"chr5.g26294\"; transcript_id \"chr5.g26294.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62659085', '62659253', '.', '-', '.', 'gene_id \"chr5.g26295\"; transcript_id \"chr5.g26295.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62678318', '62681203', '.', '+', '.', 'gene_id \"chr5.g26299\"; transcript_id \"chr5.g26299.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62681764', '62682123', '.', '-', '.', 'gene_id \"chr5.g26303\"; transcript_id \"chr5.g26303.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62689403', '62689711', '.', '-', '.', 'gene_id \"chr5.g26312\"; transcript_id \"chr5.g26312.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62696595', '62696989', '.', '-', '.', 'gene_id \"chr5.g26317\"; transcript_id \"chr5.g26317.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62699674', '62700306', '.', '+', '.', 'gene_id \"chr5.g26320\"; transcript_id \"chr5.g26320.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62700289', '62700498', '.', '-', '.', 'gene_id \"chr5.g26321\"; transcript_id \"chr5.g26321.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62701531', '62701797', '.', '+', '.', 'gene_id \"chr5.g26324\"; transcript_id \"chr5.g26324.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62703764', '62704221', '.', '-', '.', 'gene_id \"chr5.g26326\"; transcript_id \"chr5.g26326.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62704359', '62704928', '.', '+', '.', 'gene_id \"chr5.g26328\"; transcript_id \"chr5.g26328.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62705009', '62705306', '.', '+', '.', 'gene_id \"chr5.g26329\"; transcript_id \"chr5.g26329.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62706290', '62706619', '.', '-', '.', 'gene_id \"chr5.g26330\"; transcript_id \"chr5.g26330.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62709347', '62710049', '.', '+', '.', 'gene_id \"chr5.g26333\"; transcript_id \"chr5.g26333.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62713047', '62713280', '.', '+', '.', 'gene_id \"chr5.g26334\"; transcript_id \"chr5.g26334.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62733593', '62733833', '.', '+', '.', 'gene_id \"chr5.g26340\"; transcript_id \"chr5.g26340.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62733896', '62734072', '.', '+', '.', 'gene_id \"chr5.g26341\"; transcript_id \"chr5.g26341.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62736518', '62736748', '.', '+', '.', 'gene_id \"chr5.g26344\"; transcript_id \"chr5.g26344.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62737438', '62737693', '.', '+', '.', 'gene_id \"chr5.g26345\"; transcript_id \"chr5.g26345.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62767673', '62767898', '.', '-', '.', 'gene_id \"chr5.g26351\"; transcript_id \"chr5.g26351.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62771183', '62771441', '.', '+', '.', 'gene_id \"chr5.g26361\"; transcript_id \"chr5.g26361.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62783803', '62783992', '.', '-', '.', 'gene_id \"chr5.g26363\"; transcript_id \"chr5.g26363.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62788472', '62788689', '.', '-', '.', 'gene_id \"chr5.g26366\"; transcript_id \"chr5.g26366.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62793578', '62793656', '.', '+', '.', 'gene_id \"chr5.g26368\"; transcript_id \"chr5.g26368.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62823217', '62823452', '.', '+', '.', 'gene_id \"chr5.g26374\"; transcript_id \"chr5.g26374.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62824606', '62824708', '.', '+', '.', 'gene_id \"chr5.g26374\"; transcript_id \"chr5.g26374.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62890180', '62892430', '.', '-', '.', 'gene_id \"chr5.g26376\"; transcript_id \"chr5.g26376.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62825753', '62825943', '.', '+', '.', 'gene_id \"chr5.g26381\"; transcript_id \"chr5.g26381.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62848338', '62848602', '.', '+', '.', 'gene_id \"chr5.g26384\"; transcript_id \"chr5.g26384.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62894699', '62894976', '.', '-', '.', 'gene_id \"chr5.g26390\"; transcript_id \"chr5.g26390.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62897168', '62897337', '.', '-', '.', 'gene_id \"chr5.g26392\"; transcript_id \"chr5.g26392.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62901703', '62901883', '.', '+', '.', 'gene_id \"chr5.g26397\"; transcript_id \"chr5.g26397.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62903565', '62903792', '.', '+', '.', 'gene_id \"chr5.g26401\"; transcript_id \"chr5.g26401.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62911057', '62911102', '.', '-', '.', 'gene_id \"chr5.g26403\"; transcript_id \"chr5.g26403.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62931313', '62931541', '.', '-', '.', 'gene_id \"chr5.g26404\"; transcript_id \"chr5.g26404.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62936432', '62936938', '.', '-', '.', 'gene_id \"chr5.g26407\"; transcript_id \"chr5.g26407.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62935722', '62935913', '.', '+', '.', 'gene_id \"chr5.g26412\"; transcript_id \"chr5.g26412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62974046', '62974205', '.', '+', '.', 'gene_id \"chr5.g26418\"; transcript_id \"chr5.g26418.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63063474', '63063758', '.', '+', '.', 'gene_id \"chr5.g26427\"; transcript_id \"chr5.g26427.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63071992', '63072653', '.', '-', '.', 'gene_id \"chr5.g26429\"; transcript_id \"chr5.g26429.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63068358', '63069702', '.', '-', '.', 'gene_id \"chr5.g26432\"; transcript_id \"chr5.g26432.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63040210', '63040376', '.', '+', '.', 'gene_id \"chr5.g26434\"; transcript_id \"chr5.g26434.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63043849', '63044087', '.', '-', '.', 'gene_id \"chr5.g26437\"; transcript_id \"chr5.g26437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63044371', '63044609', '.', '+', '.', 'gene_id \"chr5.g26438\"; transcript_id \"chr5.g26438.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63057824', '63058083', '.', '+', '.', 'gene_id \"chr5.g26443\"; transcript_id \"chr5.g26443.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63059891', '63060307', '.', '-', '.', 'gene_id \"chr5.g26445\"; transcript_id \"chr5.g26445.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117347282', '117347696', '.', '-', '.', 'gene_id \"chrX.g30513\"; transcript_id \"chrX.g30513.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63061020', '63061489', '.', '-', '.', 'gene_id \"chr5.g26450\"; transcript_id \"chr5.g26450.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63073238', '63073545', '.', '-', '.', 'gene_id \"chr5.g26454\"; transcript_id \"chr5.g26454.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117367081', '117367381', '.', '+', '.', 'gene_id \"chrX.g30527\"; transcript_id \"chrX.g30527.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63102245', '63102416', '.', '+', '.', 'gene_id \"chr5.g26470\"; transcript_id \"chr5.g26470.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63105354', '63105526', '.', '+', '.', 'gene_id \"chr5.g26471\"; transcript_id \"chr5.g26471.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63187646', '63187962', '.', '+', '.', 'gene_id \"chr5.g26482\"; transcript_id \"chr5.g26482.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63182619', '63182844', '.', '-', '.', 'gene_id \"chr5.g26484\"; transcript_id \"chr5.g26484.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63182935', '63183193', '.', '-', '.', 'gene_id \"chr5.g26485\"; transcript_id \"chr5.g26485.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63189333', '63189559', '.', '+', '.', 'gene_id \"chr5.g26487\"; transcript_id \"chr5.g26487.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63190046', '63190419', '.', '-', '.', 'gene_id \"chr5.g26488\"; transcript_id \"chr5.g26488.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63201901', '63202172', '.', '-', '.', 'gene_id \"chr5.g26489\"; transcript_id \"chr5.g26489.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63132948', '63133056', '.', '-', '.', 'gene_id \"chr5.g26490\"; transcript_id \"chr5.g26490.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63133862', '63134013', '.', '+', '.', 'gene_id \"chr5.g26491\"; transcript_id \"chr5.g26491.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63138872', '63139050', '.', '+', '.', 'gene_id \"chr5.g26494\"; transcript_id \"chr5.g26494.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63139112', '63139300', '.', '+', '.', 'gene_id \"chr5.g26496\"; transcript_id \"chr5.g26496.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63158067', '63158299', '.', '-', '.', 'gene_id \"chr5.g26505\"; transcript_id \"chr5.g26505.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63160849', '63161077', '.', '+', '.', 'gene_id \"chr5.g26507\"; transcript_id \"chr5.g26507.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63165653', '63165973', '.', '+', '.', 'gene_id \"chr5.g26509\"; transcript_id \"chr5.g26509.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63166239', '63166513', '.', '-', '.', 'gene_id \"chr5.g26510\"; transcript_id \"chr5.g26510.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63175448', '63175767', '.', '+', '.', 'gene_id \"chr5.g26513\"; transcript_id \"chr5.g26513.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63191475', '63191886', '.', '-', '.', 'gene_id \"chr5.g26516\"; transcript_id \"chr5.g26516.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63197090', '63197285', '.', '-', '.', 'gene_id \"chr5.g26517\"; transcript_id \"chr5.g26517.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63197259', '63197591', '.', '-', '.', 'gene_id \"chr5.g26517\"; transcript_id \"chr5.g26517.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63198489', '63198887', '.', '+', '.', 'gene_id \"chr5.g26519\"; transcript_id \"chr5.g26519.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63199447', '63199691', '.', '-', '.', 'gene_id \"chr5.g26521\"; transcript_id \"chr5.g26521.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63202814', '63203106', '.', '+', '.', 'gene_id \"chr5.g26523\"; transcript_id \"chr5.g26523.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117424560', '117424600', '.', '+', '.', 'gene_id \"chrX.g30575\"; transcript_id \"chrX.g30575.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63210899', '63211075', '.', '-', '.', 'gene_id \"chr5.g26531\"; transcript_id \"chr5.g26531.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63212457', '63212619', '.', '-', '.', 'gene_id \"chr5.g26534\"; transcript_id \"chr5.g26534.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63213115', '63213369', '.', '+', '.', 'gene_id \"chr5.g26535\"; transcript_id \"chr5.g26535.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63217163', '63217352', '.', '-', '.', 'gene_id \"chr5.g26542\"; transcript_id \"chr5.g26542.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63220833', '63220991', '.', '+', '.', 'gene_id \"chr5.g26543\"; transcript_id \"chr5.g26543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63222143', '63222295', '.', '-', '.', 'gene_id \"chr5.g26545\"; transcript_id \"chr5.g26545.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63225135', '63225313', '.', '+', '.', 'gene_id \"chr5.g26547\"; transcript_id \"chr5.g26547.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63226189', '63226519', '.', '+', '.', 'gene_id \"chr5.g26549\"; transcript_id \"chr5.g26549.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117440858', '117441277', '.', '+', '.', 'gene_id \"chrX.g30600\"; transcript_id \"chrX.g30600.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63227560', '63228193', '.', '+', '.', 'gene_id \"chr5.g26551\"; transcript_id \"chr5.g26551.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63230413', '63231761', '.', '+', '.', 'gene_id \"chr5.g26557\"; transcript_id \"chr5.g26557.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63232146', '63233280', '.', '-', '.', 'gene_id \"chr5.g26558\"; transcript_id \"chr5.g26558.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63235185', '63235888', '.', '+', '.', 'gene_id \"chr5.g26560\"; transcript_id \"chr5.g26560.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63243794', '63243987', '.', '+', '.', 'gene_id \"chr5.g26568\"; transcript_id \"chr5.g26568.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63252865', '63253098', '.', '-', '.', 'gene_id \"chr5.g26570\"; transcript_id \"chr5.g26570.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63259924', '63260179', '.', '-', '.', 'gene_id \"chr5.g26576\"; transcript_id \"chr5.g26576.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63262360', '63262626', '.', '+', '.', 'gene_id \"chr5.g26579\"; transcript_id \"chr5.g26579.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63265748', '63266085', '.', '+', '.', 'gene_id \"chr5.g26581\"; transcript_id \"chr5.g26581.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63268218', '63268564', '.', '-', '.', 'gene_id \"chr5.g26584\"; transcript_id \"chr5.g26584.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63286593', '63286784', '.', '-', '.', 'gene_id \"chr5.g26587\"; transcript_id \"chr5.g26587.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63288161', '63288333', '.', '-', '.', 'gene_id \"chr5.g26592\"; transcript_id \"chr5.g26592.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63333151', '63333658', '.', '+', '.', 'gene_id \"chr5.g26595\"; transcript_id \"chr5.g26595.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63333151', '63333266', '.', '+', '.', 'gene_id \"chr5.g26595\"; transcript_id \"chr5.g26595.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63296356', '63296567', '.', '-', '.', 'gene_id \"chr5.g26600\"; transcript_id \"chr5.g26600.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117592496', '117592530', '.', '-', '.', 'gene_id \"chrX.g30654\"; transcript_id \"chrX.g30654.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63306035', '63306220', '.', '-', '.', 'gene_id \"chr5.g26614\"; transcript_id \"chr5.g26614.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63306809', '63307102', '.', '+', '.', 'gene_id \"chr5.g26617\"; transcript_id \"chr5.g26617.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63309788', '63310074', '.', '+', '.', 'gene_id \"chr5.g26620\"; transcript_id \"chr5.g26620.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117612807', '117612888', '.', '-', '.', 'gene_id \"chrX.g30664\"; transcript_id \"chrX.g30664.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63320725', '63320756', '.', '-', '.', 'gene_id \"chr5.g26624\"; transcript_id \"chr5.g26624.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63324573', '63324770', '.', '+', '.', 'gene_id \"chr5.g26626\"; transcript_id \"chr5.g26626.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117643674', '117643862', '.', '-', '.', 'gene_id \"chrX.g30670\"; transcript_id \"chrX.g30670.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63325183', '63325365', '.', '+', '.', 'gene_id \"chr5.g26627\"; transcript_id \"chr5.g26627.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63325370', '63325538', '.', '-', '.', 'gene_id \"chr5.g26628\"; transcript_id \"chr5.g26628.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63335440', '63335650', '.', '-', '.', 'gene_id \"chr5.g26633\"; transcript_id \"chr5.g26633.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63336385', '63336926', '.', '+', '.', 'gene_id \"chr5.g26634\"; transcript_id \"chr5.g26634.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63337263', '63337574', '.', '-', '.', 'gene_id \"chr5.g26635\"; transcript_id \"chr5.g26635.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63394921', '63395121', '.', '-', '.', 'gene_id \"chr5.g26642\"; transcript_id \"chr5.g26642.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63394899', '63394933', '.', '-', '.', 'gene_id \"chr5.g26642\"; transcript_id \"chr5.g26642.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63362567', '63362783', '.', '-', '.', 'gene_id \"chr5.g26643\"; transcript_id \"chr5.g26643.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63399720', '63399974', '.', '-', '.', 'gene_id \"chr5.g26648\"; transcript_id \"chr5.g26648.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63363937', '63364160', '.', '-', '.', 'gene_id \"chr5.g26655\"; transcript_id \"chr5.g26655.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63364162', '63364363', '.', '+', '.', 'gene_id \"chr5.g26656\"; transcript_id \"chr5.g26656.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63383994', '63384579', '.', '+', '.', 'gene_id \"chr5.g26665\"; transcript_id \"chr5.g26665.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63378934', '63379494', '.', '+', '.', 'gene_id \"chr5.g26666\"; transcript_id \"chr5.g26666.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63385438', '63385940', '.', '-', '.', 'gene_id \"chr5.g26667\"; transcript_id \"chr5.g26667.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63377824', '63377898', '.', '-', '.', 'gene_id \"chr5.g26669\"; transcript_id \"chr5.g26669.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63379692', '63379908', '.', '+', '.', 'gene_id \"chr5.g26674\"; transcript_id \"chr5.g26674.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63396433', '63396665', '.', '+', '.', 'gene_id \"chr5.g26682\"; transcript_id \"chr5.g26682.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63396947', '63397468', '.', '-', '.', 'gene_id \"chr5.g26685\"; transcript_id \"chr5.g26685.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63404286', '63404487', '.', '-', '.', 'gene_id \"chr5.g26689\"; transcript_id \"chr5.g26689.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63404689', '63404851', '.', '-', '.', 'gene_id \"chr5.g26690\"; transcript_id \"chr5.g26690.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63425891', '63425956', '.', '-', '.', 'gene_id \"chr5.g26695\"; transcript_id \"chr5.g26695.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63421200', '63421360', '.', '+', '.', 'gene_id \"chr5.g26697\"; transcript_id \"chr5.g26697.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63423003', '63423367', '.', '-', '.', 'gene_id \"chr5.g26699\"; transcript_id \"chr5.g26699.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63423003', '63423143', '.', '-', '.', 'gene_id \"chr5.g26699\"; transcript_id \"chr5.g26699.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63454986', '63455155', '.', '+', '.', 'gene_id \"chr5.g26709\"; transcript_id \"chr5.g26709.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63456389', '63456548', '.', '+', '.', 'gene_id \"chr5.g26711\"; transcript_id \"chr5.g26711.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63457228', '63457384', '.', '+', '.', 'gene_id \"chr5.g26713\"; transcript_id \"chr5.g26713.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63457838', '63458007', '.', '-', '.', 'gene_id \"chr5.g26714\"; transcript_id \"chr5.g26714.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63555117', '63555477', '.', '+', '.', 'gene_id \"chr5.g26725\"; transcript_id \"chr5.g26725.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63538811', '63538992', '.', '+', '.', 'gene_id \"chr5.g26728\"; transcript_id \"chr5.g26728.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63542379', '63542540', '.', '+', '.', 'gene_id \"chr5.g26729\"; transcript_id \"chr5.g26729.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63554822', '63555124', '.', '-', '.', 'gene_id \"chr5.g26732\"; transcript_id \"chr5.g26732.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63586233', '63586743', '.', '-', '.', 'gene_id \"chr5.g26736\"; transcript_id \"chr5.g26736.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63571287', '63571499', '.', '+', '.', 'gene_id \"chr5.g26737\"; transcript_id \"chr5.g26737.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63474507', '63474999', '.', '-', '.', 'gene_id \"chr5.g26739\"; transcript_id \"chr5.g26739.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63482916', '63483119', '.', '+', '.', 'gene_id \"chr5.g26741\"; transcript_id \"chr5.g26741.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63494715', '63494886', '.', '+', '.', 'gene_id \"chr5.g26743\"; transcript_id \"chr5.g26743.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63501408', '63501580', '.', '+', '.', 'gene_id \"chr5.g26746\"; transcript_id \"chr5.g26746.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63507084', '63507271', '.', '+', '.', 'gene_id \"chr5.g26748\"; transcript_id \"chr5.g26748.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63507263', '63507708', '.', '-', '.', 'gene_id \"chr5.g26749\"; transcript_id \"chr5.g26749.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63507816', '63508160', '.', '+', '.', 'gene_id \"chr5.g26751\"; transcript_id \"chr5.g26751.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63510377', '63510587', '.', '-', '.', 'gene_id \"chr5.g26753\"; transcript_id \"chr5.g26753.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63516923', '63517081', '.', '-', '.', 'gene_id \"chr5.g26755\"; transcript_id \"chr5.g26755.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63522442', '63522730', '.', '-', '.', 'gene_id \"chr5.g26759\"; transcript_id \"chr5.g26759.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63525471', '63526040', '.', '-', '.', 'gene_id \"chr5.g26763\"; transcript_id \"chr5.g26763.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63530452', '63530678', '.', '+', '.', 'gene_id \"chr5.g26766\"; transcript_id \"chr5.g26766.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63535133', '63535306', '.', '-', '.', 'gene_id \"chr5.g26767\"; transcript_id \"chr5.g26767.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63535809', '63536169', '.', '+', '.', 'gene_id \"chr5.g26768\"; transcript_id \"chr5.g26768.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63541418', '63541843', '.', '-', '.', 'gene_id \"chr5.g26772\"; transcript_id \"chr5.g26772.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63543584', '63544268', '.', '+', '.', 'gene_id \"chr5.g26774\"; transcript_id \"chr5.g26774.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63546344', '63546628', '.', '+', '.', 'gene_id \"chr5.g26776\"; transcript_id \"chr5.g26776.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63556768', '63557066', '.', '+', '.', 'gene_id \"chr5.g26782\"; transcript_id \"chr5.g26782.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63559430', '63559664', '.', '-', '.', 'gene_id \"chr5.g26785\"; transcript_id \"chr5.g26785.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63559772', '63560027', '.', '-', '.', 'gene_id \"chr5.g26786\"; transcript_id \"chr5.g26786.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63564326', '63564719', '.', '-', '.', 'gene_id \"chr5.g26787\"; transcript_id \"chr5.g26787.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63566131', '63566708', '.', '+', '.', 'gene_id \"chr5.g26790\"; transcript_id \"chr5.g26790.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63570466', '63570629', '.', '+', '.', 'gene_id \"chr5.g26793\"; transcript_id \"chr5.g26793.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63572676', '63572944', '.', '-', '.', 'gene_id \"chr5.g26795\"; transcript_id \"chr5.g26795.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63574429', '63574594', '.', '-', '.', 'gene_id \"chr5.g26796\"; transcript_id \"chr5.g26796.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63577675', '63577871', '.', '-', '.', 'gene_id \"chr5.g26798\"; transcript_id \"chr5.g26798.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63577853', '63578043', '.', '+', '.', 'gene_id \"chr5.g26799\"; transcript_id \"chr5.g26799.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63578414', '63578812', '.', '+', '.', 'gene_id \"chr5.g26800\"; transcript_id \"chr5.g26800.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63580193', '63580344', '.', '+', '.', 'gene_id \"chr5.g26801\"; transcript_id \"chr5.g26801.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63580497', '63580814', '.', '-', '.', 'gene_id \"chr5.g26802\"; transcript_id \"chr5.g26802.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63584564', '63585265', '.', '-', '.', 'gene_id \"chr5.g26805\"; transcript_id \"chr5.g26805.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63615676', '63616046', '.', '+', '.', 'gene_id \"chr5.g26812\"; transcript_id \"chr5.g26812.i12\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63677976', '63678783', '.', '-', '.', 'gene_id \"chr5.g26816\"; transcript_id \"chr5.g26816.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63684837', '63685034', '.', '+', '.', 'gene_id \"chr5.g26820\"; transcript_id \"chr5.g26820.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63741001', '63741299', '.', '+', '.', 'gene_id \"chr5.g26821\"; transcript_id \"chr5.g26821.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63718971', '63719399', '.', '+', '.', 'gene_id \"chr5.g26822\"; transcript_id \"chr5.g26822.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63729278', '63729617', '.', '+', '.', 'gene_id \"chr5.g26823\"; transcript_id \"chr5.g26823.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63736797', '63736968', '.', '-', '.', 'gene_id \"chr5.g26824\"; transcript_id \"chr5.g26824.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63659418', '63659623', '.', '+', '.', 'gene_id \"chr5.g26825\"; transcript_id \"chr5.g26825.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63735468', '63735982', '.', '+', '.', 'gene_id \"chr5.g26829\"; transcript_id \"chr5.g26829.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63757089', '63757280', '.', '-', '.', 'gene_id \"chr5.g26832\"; transcript_id \"chr5.g26832.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63619558', '63619817', '.', '-', '.', 'gene_id \"chr5.g26833\"; transcript_id \"chr5.g26833.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63624676', '63624879', '.', '-', '.', 'gene_id \"chr5.g26836\"; transcript_id \"chr5.g26836.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63624972', '63625199', '.', '+', '.', 'gene_id \"chr5.g26837\"; transcript_id \"chr5.g26837.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63626479', '63626683', '.', '-', '.', 'gene_id \"chr5.g26838\"; transcript_id \"chr5.g26838.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63637872', '63638055', '.', '+', '.', 'gene_id \"chr5.g26841\"; transcript_id \"chr5.g26841.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63675878', '63676028', '.', '-', '.', 'gene_id \"chr5.g26847\"; transcript_id \"chr5.g26847.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63679919', '63680458', '.', '-', '.', 'gene_id \"chr5.g26850\"; transcript_id \"chr5.g26850.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63687866', '63688278', '.', '-', '.', 'gene_id \"chr5.g26851\"; transcript_id \"chr5.g26851.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63695467', '63695655', '.', '+', '.', 'gene_id \"chr5.g26854\"; transcript_id \"chr5.g26854.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63700520', '63700762', '.', '+', '.', 'gene_id \"chr5.g26857\"; transcript_id \"chr5.g26857.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117902723', '117902915', '.', '-', '.', 'gene_id \"chrX.g30883\"; transcript_id \"chrX.g30883.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63710716', '63710896', '.', '-', '.', 'gene_id \"chr5.g26864\"; transcript_id \"chr5.g26864.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63711433', '63711880', '.', '+', '.', 'gene_id \"chr5.g26865\"; transcript_id \"chr5.g26865.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63711174', '63711451', '.', '+', '.', 'gene_id \"chr5.g26866\"; transcript_id \"chr5.g26866.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117903310', '117903473', '.', '+', '.', 'gene_id \"chrX.g30885\"; transcript_id \"chrX.g30885.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63714127', '63714564', '.', '+', '.', 'gene_id \"chr5.g26868\"; transcript_id \"chr5.g26868.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63723474', '63723620', '.', '-', '.', 'gene_id \"chr5.g26876\"; transcript_id \"chr5.g26876.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63725773', '63725991', '.', '-', '.', 'gene_id \"chr5.g26877\"; transcript_id \"chr5.g26877.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63728900', '63729136', '.', '+', '.', 'gene_id \"chr5.g26878\"; transcript_id \"chr5.g26878.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63737339', '63737517', '.', '-', '.', 'gene_id \"chr5.g26879\"; transcript_id \"chr5.g26879.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63747815', '63748241', '.', '-', '.', 'gene_id \"chr5.g26884\"; transcript_id \"chr5.g26884.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63754345', '63754599', '.', '-', '.', 'gene_id \"chr5.g26887\"; transcript_id \"chr5.g26887.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63768150', '63768724', '.', '-', '.', 'gene_id \"chr5.g26891\"; transcript_id \"chr5.g26891.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63772307', '63772787', '.', '+', '.', 'gene_id \"chr5.g26894\"; transcript_id \"chr5.g26894.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63773834', '63773994', '.', '+', '.', 'gene_id \"chr5.g26896\"; transcript_id \"chr5.g26896.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63773986', '63774241', '.', '+', '.', 'gene_id \"chr5.g26897\"; transcript_id \"chr5.g26897.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63776468', '63776654', '.', '-', '.', 'gene_id \"chr5.g26900\"; transcript_id \"chr5.g26900.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63792168', '63792728', '.', '+', '.', 'gene_id \"chr5.g26904\"; transcript_id \"chr5.g26904.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63847655', '63847888', '.', '+', '.', 'gene_id \"chr5.g26909\"; transcript_id \"chr5.g26909.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63840839', '63841368', '.', '-', '.', 'gene_id \"chr5.g26911\"; transcript_id \"chr5.g26911.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63790778', '63791080', '.', '+', '.', 'gene_id \"chr5.g26913\"; transcript_id \"chr5.g26913.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63793356', '63793929', '.', '-', '.', 'gene_id \"chr5.g26915\"; transcript_id \"chr5.g26915.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63797382', '63797658', '.', '+', '.', 'gene_id \"chr5.g26922\"; transcript_id \"chr5.g26922.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63812316', '63812657', '.', '+', '.', 'gene_id \"chr5.g26930\"; transcript_id \"chr5.g26930.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63814392', '63814575', '.', '-', '.', 'gene_id \"chr5.g26931\"; transcript_id \"chr5.g26931.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63885500', '63886258', '.', '+', '.', 'gene_id \"chr5.g26948\"; transcript_id \"chr5.g26948.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63885337', '63885507', '.', '-', '.', 'gene_id \"chr5.g26955\"; transcript_id \"chr5.g26955.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118128057', '118128198', '.', '-', '.', 'gene_id \"chrX.g30953\"; transcript_id \"chrX.g30953.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63876350', '63876566', '.', '-', '.', 'gene_id \"chr5.g26962\"; transcript_id \"chr5.g26962.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63876539', '63876688', '.', '+', '.', 'gene_id \"chr5.g26963\"; transcript_id \"chr5.g26963.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63884495', '63884645', '.', '-', '.', 'gene_id \"chr5.g26968\"; transcript_id \"chr5.g26968.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63887317', '63887922', '.', '-', '.', 'gene_id \"chr5.g26969\"; transcript_id \"chr5.g26969.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63895427', '63895634', '.', '-', '.', 'gene_id \"chr5.g26972\"; transcript_id \"chr5.g26972.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63900229', '63900871', '.', '-', '.', 'gene_id \"chr5.g26973\"; transcript_id \"chr5.g26973.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118247598', '118247735', '.', '-', '.', 'gene_id \"chrX.g30965\"; transcript_id \"chrX.g30965.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63903073', '63903345', '.', '-', '.', 'gene_id \"chr5.g26977\"; transcript_id \"chr5.g26977.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63918855', '63919281', '.', '+', '.', 'gene_id \"chr5.g26981\"; transcript_id \"chr5.g26981.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118304715', '118304779', '.', '+', '.', 'gene_id \"chrX.g30973\"; transcript_id \"chrX.g30973.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63934642', '63934929', '.', '+', '.', 'gene_id \"chr5.g26985\"; transcript_id \"chr5.g26985.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63935571', '63935740', '.', '-', '.', 'gene_id \"chr5.g26986\"; transcript_id \"chr5.g26986.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63941033', '63941369', '.', '+', '.', 'gene_id \"chr5.g26989\"; transcript_id \"chr5.g26989.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63977761', '63978705', '.', '-', '.', 'gene_id \"chr5.g26992\"; transcript_id \"chr5.g26992.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63954900', '63955055', '.', '-', '.', 'gene_id \"chr5.g26992\"; transcript_id \"chr5.g26992.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63963104', '63963460', '.', '+', '.', 'gene_id \"chr5.g26998\"; transcript_id \"chr5.g26998.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63962371', '63962895', '.', '+', '.', 'gene_id \"chr5.g26999\"; transcript_id \"chr5.g26999.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63954649', '63954836', '.', '-', '.', 'gene_id \"chr5.g27003\"; transcript_id \"chr5.g27003.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63948988', '63949365', '.', '-', '.', 'gene_id \"chr5.g27004\"; transcript_id \"chr5.g27004.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63960388', '63960979', '.', '-', '.', 'gene_id \"chr5.g27007\"; transcript_id \"chr5.g27007.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63967167', '63967200', '.', '-', '.', 'gene_id \"chr5.g27009\"; transcript_id \"chr5.g27009.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63982556', '63982998', '.', '-', '.', 'gene_id \"chr5.g27021\"; transcript_id \"chr5.g27021.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63983636', '63983874', '.', '-', '.', 'gene_id \"chr5.g27025\"; transcript_id \"chr5.g27025.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63984563', '63984766', '.', '-', '.', 'gene_id \"chr5.g27027\"; transcript_id \"chr5.g27027.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63986429', '63986697', '.', '+', '.', 'gene_id \"chr5.g27031\"; transcript_id \"chr5.g27031.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64005489', '64005751', '.', '-', '.', 'gene_id \"chr5.g27044\"; transcript_id \"chr5.g27044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64006114', '64006488', '.', '+', '.', 'gene_id \"chr5.g27045\"; transcript_id \"chr5.g27045.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64006780', '64007038', '.', '+', '.', 'gene_id \"chr5.g27046\"; transcript_id \"chr5.g27046.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64013019', '64013196', '.', '+', '.', 'gene_id \"chr5.g27047\"; transcript_id \"chr5.g27047.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64079369', '64079815', '.', '+', '.', 'gene_id \"chr5.g27056\"; transcript_id \"chr5.g27056.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64074894', '64076873', '.', '+', '.', 'gene_id \"chr5.g27058\"; transcript_id \"chr5.g27058.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64090421', '64091417', '.', '-', '.', 'gene_id \"chr5.g27060\"; transcript_id \"chr5.g27060.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64114530', '64114780', '.', '+', '.', 'gene_id \"chr5.g27061\"; transcript_id \"chr5.g27061.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64118767', '64119204', '.', '-', '.', 'gene_id \"chr5.g27062\"; transcript_id \"chr5.g27062.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64100643', '64100877', '.', '-', '.', 'gene_id \"chr5.g27064\"; transcript_id \"chr5.g27064.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64107506', '64107952', '.', '-', '.', 'gene_id \"chr5.g27065\"; transcript_id \"chr5.g27065.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64111836', '64112062', '.', '-', '.', 'gene_id \"chr5.g27076\"; transcript_id \"chr5.g27076.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64071115', '64071268', '.', '+', '.', 'gene_id \"chr5.g27078\"; transcript_id \"chr5.g27078.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64038142', '64038331', '.', '+', '.', 'gene_id \"chr5.g27079\"; transcript_id \"chr5.g27079.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64037926', '64038156', '.', '-', '.', 'gene_id \"chr5.g27080\"; transcript_id \"chr5.g27080.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64067070', '64067560', '.', '-', '.', 'gene_id \"chr5.g27085\"; transcript_id \"chr5.g27085.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64093786', '64094114', '.', '+', '.', 'gene_id \"chr5.g27090\"; transcript_id \"chr5.g27090.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64094600', '64094894', '.', '+', '.', 'gene_id \"chr5.g27091\"; transcript_id \"chr5.g27091.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64094643', '64095054', '.', '+', '.', 'gene_id \"chr5.g27091\"; transcript_id \"chr5.g27091.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64104039', '64104189', '.', '+', '.', 'gene_id \"chr5.g27094\"; transcript_id \"chr5.g27094.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64108434', '64108740', '.', '+', '.', 'gene_id \"chr5.g27095\"; transcript_id \"chr5.g27095.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64108721', '64109006', '.', '-', '.', 'gene_id \"chr5.g27096\"; transcript_id \"chr5.g27096.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64129991', '64130173', '.', '-', '.', 'gene_id \"chr5.g27100\"; transcript_id \"chr5.g27100.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64123213', '64124345', '.', '+', '.', 'gene_id \"chr5.g27102\"; transcript_id \"chr5.g27102.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64128038', '64128338', '.', '-', '.', 'gene_id \"chr5.g27107\"; transcript_id \"chr5.g27107.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64147367', '64147620', '.', '-', '.', 'gene_id \"chr5.g27109\"; transcript_id \"chr5.g27109.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64136338', '64136594', '.', '-', '.', 'gene_id \"chr5.g27112\"; transcript_id \"chr5.g27112.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64135677', '64136348', '.', '-', '.', 'gene_id \"chr5.g27115\"; transcript_id \"chr5.g27115.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64139536', '64139834', '.', '+', '.', 'gene_id \"chr5.g27121\"; transcript_id \"chr5.g27121.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64140958', '64141325', '.', '+', '.', 'gene_id \"chr5.g27124\"; transcript_id \"chr5.g27124.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64145864', '64146152', '.', '-', '.', 'gene_id \"chr5.g27129\"; transcript_id \"chr5.g27129.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64149254', '64149571', '.', '-', '.', 'gene_id \"chr5.g27133\"; transcript_id \"chr5.g27133.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64159189', '64159490', '.', '+', '.', 'gene_id \"chr5.g27139\"; transcript_id \"chr5.g27139.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64207508', '64207732', '.', '-', '.', 'gene_id \"chr5.g27140\"; transcript_id \"chr5.g27140.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64203864', '64204024', '.', '-', '.', 'gene_id \"chr5.g27143\"; transcript_id \"chr5.g27143.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64204122', '64204283', '.', '+', '.', 'gene_id \"chr5.g27144\"; transcript_id \"chr5.g27144.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64223677', '64223900', '.', '+', '.', 'gene_id \"chr5.g27151\"; transcript_id \"chr5.g27151.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64241585', '64242400', '.', '-', '.', 'gene_id \"chr5.g27153\"; transcript_id \"chr5.g27153.i10\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64240005', '64240645', '.', '-', '.', 'gene_id \"chr5.g27153\"; transcript_id \"chr5.g27153.i12\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64249653', '64249999', '.', '+', '.', 'gene_id \"chr5.g27158\"; transcript_id \"chr5.g27158.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64237019', '64237975', '.', '+', '.', 'gene_id \"chr5.g27162\"; transcript_id \"chr5.g27162.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64267957', '64268185', '.', '+', '.', 'gene_id \"chr5.g27170\"; transcript_id \"chr5.g27170.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64272104', '64272396', '.', '-', '.', 'gene_id \"chr5.g27172\"; transcript_id \"chr5.g27172.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118770940', '118770977', '.', '+', '.', 'gene_id \"chrX.g31101\"; transcript_id \"chrX.g31101.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64280469', '64280798', '.', '-', '.', 'gene_id \"chr5.g27177\"; transcript_id \"chr5.g27177.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64281828', '64282037', '.', '-', '.', 'gene_id \"chr5.g27178\"; transcript_id \"chr5.g27178.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64472135', '64472589', '.', '-', '.', 'gene_id \"chr5.g27186\"; transcript_id \"chr5.g27186.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64291633', '64291972', '.', '+', '.', 'gene_id \"chr5.g27194\"; transcript_id \"chr5.g27194.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64291968', '64292249', '.', '+', '.', 'gene_id \"chr5.g27195\"; transcript_id \"chr5.g27195.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64295710', '64296520', '.', '+', '.', 'gene_id \"chr5.g27200\"; transcript_id \"chr5.g27200.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64302119', '64302346', '.', '-', '.', 'gene_id \"chr5.g27206\"; transcript_id \"chr5.g27206.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64307151', '64307624', '.', '-', '.', 'gene_id \"chr5.g27211\"; transcript_id \"chr5.g27211.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118921093', '118921251', '.', '+', '.', 'gene_id \"chrX.g31141\"; transcript_id \"chrX.g31141.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64310038', '64310427', '.', '-', '.', 'gene_id \"chr5.g27218\"; transcript_id \"chr5.g27218.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64310419', '64310730', '.', '+', '.', 'gene_id \"chr5.g27220\"; transcript_id \"chr5.g27220.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64311083', '64311514', '.', '-', '.', 'gene_id \"chr5.g27222\"; transcript_id \"chr5.g27222.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64314988', '64315146', '.', '+', '.', 'gene_id \"chr5.g27225\"; transcript_id \"chr5.g27225.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64317578', '64317967', '.', '+', '.', 'gene_id \"chr5.g27226\"; transcript_id \"chr5.g27226.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64319784', '64320132', '.', '+', '.', 'gene_id \"chr5.g27232\"; transcript_id \"chr5.g27232.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64321352', '64321522', '.', '-', '.', 'gene_id \"chr5.g27233\"; transcript_id \"chr5.g27233.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64330853', '64331019', '.', '+', '.', 'gene_id \"chr5.g27236\"; transcript_id \"chr5.g27236.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64338894', '64339112', '.', '+', '.', 'gene_id \"chr5.g27242\"; transcript_id \"chr5.g27242.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119029180', '119029541', '.', '+', '.', 'gene_id \"chrX.g31164\"; transcript_id \"chrX.g31164.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64349573', '64349809', '.', '+', '.', 'gene_id \"chr5.g27244\"; transcript_id \"chr5.g27244.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64350066', '64350220', '.', '+', '.', 'gene_id \"chr5.g27246\"; transcript_id \"chr5.g27246.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64350918', '64351072', '.', '+', '.', 'gene_id \"chr5.g27247\"; transcript_id \"chr5.g27247.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64374839', '64375083', '.', '+', '.', 'gene_id \"chr5.g27260\"; transcript_id \"chr5.g27260.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64377219', '64377487', '.', '-', '.', 'gene_id \"chr5.g27262\"; transcript_id \"chr5.g27262.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64380517', '64380773', '.', '+', '.', 'gene_id \"chr5.g27268\"; transcript_id \"chr5.g27268.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64387630', '64387783', '.', '-', '.', 'gene_id \"chr5.g27269\"; transcript_id \"chr5.g27269.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64389952', '64390109', '.', '-', '.', 'gene_id \"chr5.g27270\"; transcript_id \"chr5.g27270.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64391790', '64391995', '.', '+', '.', 'gene_id \"chr5.g27271\"; transcript_id \"chr5.g27271.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64394283', '64394448', '.', '-', '.', 'gene_id \"chr5.g27272\"; transcript_id \"chr5.g27272.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64404198', '64404411', '.', '+', '.', 'gene_id \"chr5.g27280\"; transcript_id \"chr5.g27280.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64405346', '64405835', '.', '+', '.', 'gene_id \"chr5.g27282\"; transcript_id \"chr5.g27282.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64405840', '64406089', '.', '-', '.', 'gene_id \"chr5.g27283\"; transcript_id \"chr5.g27283.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64406959', '64407240', '.', '+', '.', 'gene_id \"chr5.g27284\"; transcript_id \"chr5.g27284.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119224787', '119224828', '.', '-', '.', 'gene_id \"chrX.g31218\"; transcript_id \"chrX.g31218.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64421044', '64421205', '.', '-', '.', 'gene_id \"chr5.g27300\"; transcript_id \"chr5.g27300.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64429580', '64429914', '.', '-', '.', 'gene_id \"chr5.g27307\"; transcript_id \"chr5.g27307.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64435766', '64435957', '.', '+', '.', 'gene_id \"chr5.g27309\"; transcript_id \"chr5.g27309.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64436914', '64437111', '.', '-', '.', 'gene_id \"chr5.g27312\"; transcript_id \"chr5.g27312.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64459274', '64459815', '.', '-', '.', 'gene_id \"chr5.g27318\"; transcript_id \"chr5.g27318.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64461006', '64461244', '.', '+', '.', 'gene_id \"chr5.g27319\"; transcript_id \"chr5.g27319.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64470176', '64470355', '.', '-', '.', 'gene_id \"chr5.g27323\"; transcript_id \"chr5.g27323.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64477545', '64477797', '.', '-', '.', 'gene_id \"chr5.g27326\"; transcript_id \"chr5.g27326.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64506823', '64507202', '.', '+', '.', 'gene_id \"chr5.g27337\"; transcript_id \"chr5.g27337.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64512468', '64512815', '.', '-', '.', 'gene_id \"chr5.g27345\"; transcript_id \"chr5.g27345.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64512821', '64513258', '.', '-', '.', 'gene_id \"chr5.g27346\"; transcript_id \"chr5.g27346.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64536997', '64537299', '.', '+', '.', 'gene_id \"chr5.g27352\"; transcript_id \"chr5.g27352.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64583440', '64583586', '.', '+', '.', 'gene_id \"chr5.g27357\"; transcript_id \"chr5.g27357.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64582136', '64582784', '.', '+', '.', 'gene_id \"chr5.g27358\"; transcript_id \"chr5.g27358.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64559073', '64559463', '.', '+', '.', 'gene_id \"chr5.g27370\"; transcript_id \"chr5.g27370.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64563324', '64563484', '.', '+', '.', 'gene_id \"chr5.g27375\"; transcript_id \"chr5.g27375.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64568589', '64568975', '.', '-', '.', 'gene_id \"chr5.g27378\"; transcript_id \"chr5.g27378.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119621118', '119621560', '.', '-', '.', 'gene_id \"chrX.g31293\"; transcript_id \"chrX.g31293.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64585172', '64585455', '.', '-', '.', 'gene_id \"chr5.g27387\"; transcript_id \"chr5.g27387.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64585494', '64586246', '.', '-', '.', 'gene_id \"chr5.g27389\"; transcript_id \"chr5.g27389.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64586572', '64586798', '.', '+', '.', 'gene_id \"chr5.g27390\"; transcript_id \"chr5.g27390.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64613616', '64613799', '.', '+', '.', 'gene_id \"chr5.g27398\"; transcript_id \"chr5.g27398.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64619921', '64620083', '.', '-', '.', 'gene_id \"chr5.g27401\"; transcript_id \"chr5.g27401.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119690672', '119691332', '.', '-', '.', 'gene_id \"chrX.g31322\"; transcript_id \"chrX.g31322.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119688527', '119689197', '.', '+', '.', 'gene_id \"chrX.g31323\"; transcript_id \"chrX.g31323.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64621786', '64621972', '.', '+', '.', 'gene_id \"chr5.g27403\"; transcript_id \"chr5.g27403.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64622523', '64622693', '.', '-', '.', 'gene_id \"chr5.g27405\"; transcript_id \"chr5.g27405.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64624545', '64624770', '.', '+', '.', 'gene_id \"chr5.g27407\"; transcript_id \"chr5.g27407.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64631003', '64631217', '.', '-', '.', 'gene_id \"chr5.g27411\"; transcript_id \"chr5.g27411.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64733000', '64733390', '.', '+', '.', 'gene_id \"chr5.g27428\"; transcript_id \"chr5.g27428.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64755503', '64755736', '.', '-', '.', 'gene_id \"chr5.g27430\"; transcript_id \"chr5.g27430.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64725981', '64726180', '.', '+', '.', 'gene_id \"chr5.g27436\"; transcript_id \"chr5.g27436.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64727656', '64727810', '.', '-', '.', 'gene_id \"chr5.g27437\"; transcript_id \"chr5.g27437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64730712', '64730966', '.', '-', '.', 'gene_id \"chr5.g27439\"; transcript_id \"chr5.g27439.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64736816', '64737083', '.', '-', '.', 'gene_id \"chr5.g27443\"; transcript_id \"chr5.g27443.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64738175', '64738481', '.', '+', '.', 'gene_id \"chr5.g27445\"; transcript_id \"chr5.g27445.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64740691', '64741184', '.', '-', '.', 'gene_id \"chr5.g27449\"; transcript_id \"chr5.g27449.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64741415', '64741569', '.', '-', '.', 'gene_id \"chr5.g27450\"; transcript_id \"chr5.g27450.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64757908', '64758195', '.', '+', '.', 'gene_id \"chr5.g27455\"; transcript_id \"chr5.g27455.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64758558', '64758730', '.', '+', '.', 'gene_id \"chr5.g27458\"; transcript_id \"chr5.g27458.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64761731', '64761963', '.', '-', '.', 'gene_id \"chr5.g27459\"; transcript_id \"chr5.g27459.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64770143', '64770305', '.', '+', '.', 'gene_id \"chr5.g27460\"; transcript_id \"chr5.g27460.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64772056', '64772380', '.', '+', '.', 'gene_id \"chr5.g27462\"; transcript_id \"chr5.g27462.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64774788', '64775284', '.', '+', '.', 'gene_id \"chr5.g27464\"; transcript_id \"chr5.g27464.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64778188', '64778358', '.', '+', '.', 'gene_id \"chr5.g27465\"; transcript_id \"chr5.g27465.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64794274', '64794517', '.', '+', '.', 'gene_id \"chr5.g27469\"; transcript_id \"chr5.g27469.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64804258', '64804451', '.', '+', '.', 'gene_id \"chr5.g27484\"; transcript_id \"chr5.g27484.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64816028', '64816063', '.', '-', '.', 'gene_id \"chr5.g27488\"; transcript_id \"chr5.g27488.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64825436', '64826877', '.', '+', '.', 'gene_id \"chr5.g27489\"; transcript_id \"chr5.g27489.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64885438', '64886252', '.', '-', '.', 'gene_id \"chr5.g27492\"; transcript_id \"chr5.g27492.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64830610', '64830946', '.', '-', '.', 'gene_id \"chr5.g27493\"; transcript_id \"chr5.g27493.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64899765', '64899998', '.', '+', '.', 'gene_id \"chr5.g27498\"; transcript_id \"chr5.g27498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64900372', '64900561', '.', '-', '.', 'gene_id \"chr5.g27499\"; transcript_id \"chr5.g27499.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64838878', '64838981', '.', '+', '.', 'gene_id \"chr5.g27505\"; transcript_id \"chr5.g27505.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64839376', '64839551', '.', '-', '.', 'gene_id \"chr5.g27506\"; transcript_id \"chr5.g27506.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64845474', '64845680', '.', '+', '.', 'gene_id \"chr5.g27510\"; transcript_id \"chr5.g27510.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64845708', '64845868', '.', '+', '.', 'gene_id \"chr5.g27511\"; transcript_id \"chr5.g27511.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64847132', '64847294', '.', '+', '.', 'gene_id \"chr5.g27512\"; transcript_id \"chr5.g27512.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119820035', '119820191', '.', '+', '.', 'gene_id \"chrX.g31422\"; transcript_id \"chrX.g31422.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64855848', '64856100', '.', '-', '.', 'gene_id \"chr5.g27518\"; transcript_id \"chr5.g27518.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64873386', '64873791', '.', '-', '.', 'gene_id \"chr5.g27521\"; transcript_id \"chr5.g27521.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64883763', '64884090', '.', '+', '.', 'gene_id \"chr5.g27526\"; transcript_id \"chr5.g27526.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119872751', '119872949', '.', '+', '.', 'gene_id \"chrX.g31434\"; transcript_id \"chrX.g31434.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64890003', '64890235', '.', '+', '.', 'gene_id \"chr5.g27527\"; transcript_id \"chr5.g27527.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64900564', '64900800', '.', '-', '.', 'gene_id \"chr5.g27532\"; transcript_id \"chr5.g27532.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64900788', '64901026', '.', '-', '.', 'gene_id \"chr5.g27533\"; transcript_id \"chr5.g27533.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64905523', '64905674', '.', '-', '.', 'gene_id \"chr5.g27535\"; transcript_id \"chr5.g27535.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64920282', '64921737', '.', '-', '.', 'gene_id \"chr5.g27542\"; transcript_id \"chr5.g27542.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64922232', '64922521', '.', '-', '.', 'gene_id \"chr5.g27543\"; transcript_id \"chr5.g27543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64926241', '64926400', '.', '+', '.', 'gene_id \"chr5.g27544\"; transcript_id \"chr5.g27544.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64926374', '64926820', '.', '+', '.', 'gene_id \"chr5.g27544\"; transcript_id \"chr5.g27544.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64927811', '64927999', '.', '+', '.', 'gene_id \"chr5.g27547\"; transcript_id \"chr5.g27547.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64934560', '64934734', '.', '+', '.', 'gene_id \"chr5.g27549\"; transcript_id \"chr5.g27549.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64932089', '64932467', '.', '-', '.', 'gene_id \"chr5.g27550\"; transcript_id \"chr5.g27550.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64935377', '64935722', '.', '-', '.', 'gene_id \"chr5.g27552\"; transcript_id \"chr5.g27552.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64934144', '64934443', '.', '+', '.', 'gene_id \"chr5.g27555\"; transcript_id \"chr5.g27555.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64934862', '64935031', '.', '-', '.', 'gene_id \"chr5.g27557\"; transcript_id \"chr5.g27557.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64944715', '64945596', '.', '-', '.', 'gene_id \"chr5.g27561\"; transcript_id \"chr5.g27561.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119935307', '119935492', '.', '-', '.', 'gene_id \"chrX.g31468\"; transcript_id \"chrX.g31468.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119935468', '119935653', '.', '+', '.', 'gene_id \"chrX.g31469\"; transcript_id \"chrX.g31469.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64954898', '64955056', '.', '+', '.', 'gene_id \"chr5.g27572\"; transcript_id \"chr5.g27572.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64967672', '64969086', '.', '-', '.', 'gene_id \"chr5.g27580\"; transcript_id \"chr5.g27580.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64969404', '64969697', '.', '-', '.', 'gene_id \"chr5.g27582\"; transcript_id \"chr5.g27582.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119951125', '119951169', '.', '+', '.', 'gene_id \"chrX.g31481\"; transcript_id \"chrX.g31481.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64978557', '64978957', '.', '+', '.', 'gene_id \"chr5.g27593\"; transcript_id \"chr5.g27593.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119990224', '119991532', '.', '-', '.', 'gene_id \"chrX.g31496\"; transcript_id \"chrX.g31496.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64992292', '64993053', '.', '+', '.', 'gene_id \"chr5.g27597\"; transcript_id \"chr5.g27597.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64992295', '64993060', '.', '-', '.', 'gene_id \"chr5.g27598\"; transcript_id \"chr5.g27598.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119991553', '119991738', '.', '-', '.', 'gene_id \"chrX.g31497\"; transcript_id \"chrX.g31497.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119991714', '119991899', '.', '+', '.', 'gene_id \"chrX.g31498\"; transcript_id \"chrX.g31498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64995526', '64996749', '.', '-', '.', 'gene_id \"chr5.g27600\"; transcript_id \"chr5.g27600.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65000774', '65000954', '.', '-', '.', 'gene_id \"chr5.g27603\"; transcript_id \"chr5.g27603.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65043577', '65043749', '.', '+', '.', 'gene_id \"chr5.g27609\"; transcript_id \"chr5.g27609.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65055340', '65057453', '.', '+', '.', 'gene_id \"chr5.g27611\"; transcript_id \"chr5.g27611.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65064606', '65064849', '.', '+', '.', 'gene_id \"chr5.g27615\"; transcript_id \"chr5.g27615.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65058618', '65058769', '.', '-', '.', 'gene_id \"chr5.g27617\"; transcript_id \"chr5.g27617.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65057645', '65057836', '.', '+', '.', 'gene_id \"chr5.g27618\"; transcript_id \"chr5.g27618.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65060965', '65061230', '.', '+', '.', 'gene_id \"chr5.g27621\"; transcript_id \"chr5.g27621.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65061208', '65061847', '.', '-', '.', 'gene_id \"chr5.g27622\"; transcript_id \"chr5.g27622.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65061837', '65062021', '.', '+', '.', 'gene_id \"chr5.g27623\"; transcript_id \"chr5.g27623.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120051674', '120051778', '.', '+', '.', 'gene_id \"chrX.g31523\"; transcript_id \"chrX.g31523.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65068742', '65069674', '.', '+', '.', 'gene_id \"chr5.g27629\"; transcript_id \"chr5.g27629.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65069665', '65069943', '.', '-', '.', 'gene_id \"chr5.g27630\"; transcript_id \"chr5.g27630.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65071250', '65071433', '.', '-', '.', 'gene_id \"chr5.g27633\"; transcript_id \"chr5.g27633.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65082776', '65082940', '.', '-', '.', 'gene_id \"chr5.g27638\"; transcript_id \"chr5.g27638.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65124013', '65124106', '.', '+', '.', 'gene_id \"chr5.g27644\"; transcript_id \"chr5.g27644.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65124013', '65124106', '.', '-', '.', 'gene_id \"chr5.g27645\"; transcript_id \"chr5.g27645.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65124174', '65124420', '.', '+', '.', 'gene_id \"chr5.g27646\"; transcript_id \"chr5.g27646.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120082631', '120082850', '.', '-', '.', 'gene_id \"chrX.g31548\"; transcript_id \"chrX.g31548.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120082896', '120083157', '.', '-', '.', 'gene_id \"chrX.g31549\"; transcript_id \"chrX.g31549.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120083146', '120083319', '.', '-', '.', 'gene_id \"chrX.g31550\"; transcript_id \"chrX.g31550.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65135127', '65135481', '.', '-', '.', 'gene_id \"chr5.g27649\"; transcript_id \"chr5.g27649.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65138819', '65138999', '.', '-', '.', 'gene_id \"chr5.g27655\"; transcript_id \"chr5.g27655.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65140529', '65140872', '.', '+', '.', 'gene_id \"chr5.g27656\"; transcript_id \"chr5.g27656.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65143286', '65143472', '.', '+', '.', 'gene_id \"chr5.g27660\"; transcript_id \"chr5.g27660.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120141106', '120141153', '.', '+', '.', 'gene_id \"chrX.g31563\"; transcript_id \"chrX.g31563.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65238315', '65238597', '.', '+', '.', 'gene_id \"chr5.g27666\"; transcript_id \"chr5.g27666.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65245260', '65245429', '.', '-', '.', 'gene_id \"chr5.g27668\"; transcript_id \"chr5.g27668.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120168102', '120168136', '.', '+', '.', 'gene_id \"chrX.g31576\"; transcript_id \"chrX.g31576.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65323036', '65323200', '.', '+', '.', 'gene_id \"chr5.g27673\"; transcript_id \"chr5.g27673.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65471064', '65471284', '.', '+', '.', 'gene_id \"chr5.g27680\"; transcript_id \"chr5.g27680.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65403469', '65403633', '.', '+', '.', 'gene_id \"chr5.g27692\"; transcript_id \"chr5.g27692.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65407580', '65407781', '.', '-', '.', 'gene_id \"chr5.g27696\"; transcript_id \"chr5.g27696.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65407764', '65408055', '.', '+', '.', 'gene_id \"chr5.g27697\"; transcript_id \"chr5.g27697.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65409656', '65410410', '.', '-', '.', 'gene_id \"chr5.g27700\"; transcript_id \"chr5.g27700.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65410388', '65410623', '.', '+', '.', 'gene_id \"chr5.g27701\"; transcript_id \"chr5.g27701.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65415249', '65415660', '.', '-', '.', 'gene_id \"chr5.g27709\"; transcript_id \"chr5.g27709.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65415821', '65416158', '.', '-', '.', 'gene_id \"chr5.g27711\"; transcript_id \"chr5.g27711.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65416142', '65416339', '.', '+', '.', 'gene_id \"chr5.g27712\"; transcript_id \"chr5.g27712.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65417361', '65417543', '.', '-', '.', 'gene_id \"chr5.g27713\"; transcript_id \"chr5.g27713.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65419234', '65419427', '.', '+', '.', 'gene_id \"chr5.g27716\"; transcript_id \"chr5.g27716.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65419446', '65419716', '.', '+', '.', 'gene_id \"chr5.g27719\"; transcript_id \"chr5.g27719.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65421118', '65421282', '.', '-', '.', 'gene_id \"chr5.g27722\"; transcript_id \"chr5.g27722.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65421262', '65421465', '.', '+', '.', 'gene_id \"chr5.g27723\"; transcript_id \"chr5.g27723.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65427944', '65427987', '.', '-', '.', 'gene_id \"chr5.g27729\"; transcript_id \"chr5.g27729.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65429235', '65429438', '.', '-', '.', 'gene_id \"chr5.g27733\"; transcript_id \"chr5.g27733.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65491179', '65491425', '.', '-', '.', 'gene_id \"chr5.g27742\"; transcript_id \"chr5.g27742.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65492630', '65493134', '.', '-', '.', 'gene_id \"chr5.g27743\"; transcript_id \"chr5.g27743.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65494227', '65494418', '.', '+', '.', 'gene_id \"chr5.g27744\"; transcript_id \"chr5.g27744.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65495218', '65495481', '.', '+', '.', 'gene_id \"chr5.g27746\"; transcript_id \"chr5.g27746.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65515186', '65515409', '.', '+', '.', 'gene_id \"chr5.g27747\"; transcript_id \"chr5.g27747.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65520162', '65520868', '.', '+', '.', 'gene_id \"chr5.g27747\"; transcript_id \"chr5.g27747.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65520934', '65522653', '.', '+', '.', 'gene_id \"chr5.g27748\"; transcript_id \"chr5.g27748.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65533588', '65533789', '.', '-', '.', 'gene_id \"chr5.g27750\"; transcript_id \"chr5.g27750.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65548130', '65548304', '.', '+', '.', 'gene_id \"chr5.g27752\"; transcript_id \"chr5.g27752.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65550820', '65551398', '.', '-', '.', 'gene_id \"chr5.g27753\"; transcript_id \"chr5.g27753.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65550820', '65551398', '.', '-', '.', 'gene_id \"chr5.g27753\"; transcript_id \"chr5.g27753.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65555916', '65556149', '.', '+', '.', 'gene_id \"chr5.g27754\"; transcript_id \"chr5.g27754.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65558332', '65558816', '.', '+', '.', 'gene_id \"chr5.g27756\"; transcript_id \"chr5.g27756.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65570779', '65570903', '.', '+', '.', 'gene_id \"chr5.g27756\"; transcript_id \"chr5.g27756.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65584613', '65585143', '.', '+', '.', 'gene_id \"chr5.g27756\"; transcript_id \"chr5.g27756.i10\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65587433', '65588300', '.', '+', '.', 'gene_id \"chr5.g27756\"; transcript_id \"chr5.g27756.i14\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65580891', '65581207', '.', '-', '.', 'gene_id \"chr5.g27757\"; transcript_id \"chr5.g27757.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65560530', '65560681', '.', '+', '.', 'gene_id \"chr5.g27764\"; transcript_id \"chr5.g27764.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65563409', '65563620', '.', '+', '.', 'gene_id \"chr5.g27767\"; transcript_id \"chr5.g27767.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65564551', '65564740', '.', '+', '.', 'gene_id \"chr5.g27770\"; transcript_id \"chr5.g27770.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65564929', '65565119', '.', '+', '.', 'gene_id \"chr5.g27771\"; transcript_id \"chr5.g27771.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65570579', '65570744', '.', '+', '.', 'gene_id \"chr5.g27779\"; transcript_id \"chr5.g27779.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65575283', '65575536', '.', '+', '.', 'gene_id \"chr5.g27784\"; transcript_id \"chr5.g27784.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65576524', '65577000', '.', '-', '.', 'gene_id \"chr5.g27785\"; transcript_id \"chr5.g27785.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65577878', '65578101', '.', '-', '.', 'gene_id \"chr5.g27786\"; transcript_id \"chr5.g27786.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65578982', '65579214', '.', '+', '.', 'gene_id \"chr5.g27788\"; transcript_id \"chr5.g27788.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65581889', '65582086', '.', '+', '.', 'gene_id \"chr5.g27789\"; transcript_id \"chr5.g27789.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120374251', '120374414', '.', '+', '.', 'gene_id \"chrX.g31677\"; transcript_id \"chrX.g31677.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65625277', '65625465', '.', '+', '.', 'gene_id \"chr5.g27791\"; transcript_id \"chr5.g27791.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65625451', '65626220', '.', '-', '.', 'gene_id \"chr5.g27792\"; transcript_id \"chr5.g27792.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65626400', '65626616', '.', '+', '.', 'gene_id \"chr5.g27795\"; transcript_id \"chr5.g27795.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65627255', '65629273', '.', '-', '.', 'gene_id \"chr5.g27796\"; transcript_id \"chr5.g27796.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65626740', '65627235', '.', '-', '.', 'gene_id \"chr5.g27799\"; transcript_id \"chr5.g27799.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65631338', '65631495', '.', '+', '.', 'gene_id \"chr5.g27802\"; transcript_id \"chr5.g27802.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65650283', '65650424', '.', '-', '.', 'gene_id \"chr5.g27804\"; transcript_id \"chr5.g27804.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65638622', '65638969', '.', '+', '.', 'gene_id \"chr5.g27811\"; transcript_id \"chr5.g27811.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65651351', '65651550', '.', '-', '.', 'gene_id \"chr5.g27812\"; transcript_id \"chr5.g27812.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65652810', '65652998', '.', '+', '.', 'gene_id \"chr5.g27813\"; transcript_id \"chr5.g27813.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65680629', '65680834', '.', '+', '.', 'gene_id \"chr5.g27824\"; transcript_id \"chr5.g27824.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65681350', '65681509', '.', '+', '.', 'gene_id \"chr5.g27826\"; transcript_id \"chr5.g27826.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65682074', '65682253', '.', '+', '.', 'gene_id \"chr5.g27828\"; transcript_id \"chr5.g27828.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65689467', '65689714', '.', '+', '.', 'gene_id \"chr5.g27830\"; transcript_id \"chr5.g27830.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65690447', '65690753', '.', '-', '.', 'gene_id \"chr5.g27831\"; transcript_id \"chr5.g27831.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65690869', '65691146', '.', '+', '.', 'gene_id \"chr5.g27832\"; transcript_id \"chr5.g27832.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65701764', '65701991', '.', '-', '.', 'gene_id \"chr5.g27835\"; transcript_id \"chr5.g27835.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65704885', '65705064', '.', '-', '.', 'gene_id \"chr5.g27838\"; transcript_id \"chr5.g27838.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65705046', '65705231', '.', '+', '.', 'gene_id \"chr5.g27839\"; transcript_id \"chr5.g27839.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65714418', '65714743', '.', '+', '.', 'gene_id \"chr5.g27849\"; transcript_id \"chr5.g27849.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65744118', '65744895', '.', '+', '.', 'gene_id \"chr5.g27853\"; transcript_id \"chr5.g27853.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65752336', '65752528', '.', '+', '.', 'gene_id \"chr5.g27856\"; transcript_id \"chr5.g27856.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65825463', '65825518', '.', '+', '.', 'gene_id \"chr5.g27869\"; transcript_id \"chr5.g27869.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65845680', '65846023', '.', '-', '.', 'gene_id \"chr5.g27871\"; transcript_id \"chr5.g27871.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65846604', '65846775', '.', '+', '.', 'gene_id \"chr5.g27872\"; transcript_id \"chr5.g27872.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65846187', '65846339', '.', '+', '.', 'gene_id \"chr5.g27873\"; transcript_id \"chr5.g27873.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65854205', '65854700', '.', '+', '.', 'gene_id \"chr5.g27876\"; transcript_id \"chr5.g27876.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65855192', '65855737', '.', '+', '.', 'gene_id \"chr5.g27877\"; transcript_id \"chr5.g27877.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65847993', '65848164', '.', '-', '.', 'gene_id \"chr5.g27881\"; transcript_id \"chr5.g27881.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65848425', '65848954', '.', '-', '.', 'gene_id \"chr5.g27882\"; transcript_id \"chr5.g27882.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65851083', '65851432', '.', '-', '.', 'gene_id \"chr5.g27883\"; transcript_id \"chr5.g27883.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65851581', '65851800', '.', '+', '.', 'gene_id \"chr5.g27884\"; transcript_id \"chr5.g27884.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65856107', '65856291', '.', '-', '.', 'gene_id \"chr5.g27886\"; transcript_id \"chr5.g27886.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65899089', '65899301', '.', '-', '.', 'gene_id \"chr5.g27892\"; transcript_id \"chr5.g27892.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65901473', '65901861', '.', '-', '.', 'gene_id \"chr5.g27895\"; transcript_id \"chr5.g27895.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65902754', '65904409', '.', '-', '.', 'gene_id \"chr5.g27898\"; transcript_id \"chr5.g27898.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65887632', '65887928', '.', '-', '.', 'gene_id \"chr5.g27904\"; transcript_id \"chr5.g27904.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65887632', '65888000', '.', '-', '.', 'gene_id \"chr5.g27904\"; transcript_id \"chr5.g27904.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65888105', '65888293', '.', '-', '.', 'gene_id \"chr5.g27907\"; transcript_id \"chr5.g27907.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65893715', '65894230', '.', '-', '.', 'gene_id \"chr5.g27913\"; transcript_id \"chr5.g27913.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65895751', '65896001', '.', '-', '.', 'gene_id \"chr5.g27916\"; transcript_id \"chr5.g27916.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65896241', '65896842', '.', '-', '.', 'gene_id \"chr5.g27917\"; transcript_id \"chr5.g27917.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65896831', '65896990', '.', '+', '.', 'gene_id \"chr5.g27918\"; transcript_id \"chr5.g27918.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65930009', '65930197', '.', '-', '.', 'gene_id \"chr5.g27924\"; transcript_id \"chr5.g27924.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65918506', '65918689', '.', '+', '.', 'gene_id \"chr5.g27929\"; transcript_id \"chr5.g27929.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65925318', '65925612', '.', '-', '.', 'gene_id \"chr5.g27934\"; transcript_id \"chr5.g27934.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65935227', '65935412', '.', '-', '.', 'gene_id \"chr5.g27937\"; transcript_id \"chr5.g27937.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65979249', '65980668', '.', '-', '.', 'gene_id \"chr5.g27939\"; transcript_id \"chr5.g27939.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65973046', '65973363', '.', '-', '.', 'gene_id \"chr5.g27941\"; transcript_id \"chr5.g27941.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65949304', '65949548', '.', '-', '.', 'gene_id \"chr5.g27944\"; transcript_id \"chr5.g27944.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65944754', '65945011', '.', '-', '.', 'gene_id \"chr5.g27945\"; transcript_id \"chr5.g27945.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65946932', '65947478', '.', '+', '.', 'gene_id \"chr5.g27947\"; transcript_id \"chr5.g27947.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65950547', '65950713', '.', '+', '.', 'gene_id \"chr5.g27948\"; transcript_id \"chr5.g27948.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65954383', '65954635', '.', '+', '.', 'gene_id \"chr5.g27953\"; transcript_id \"chr5.g27953.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65955451', '65955628', '.', '+', '.', 'gene_id \"chr5.g27955\"; transcript_id \"chr5.g27955.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65960650', '65960867', '.', '-', '.', 'gene_id \"chr5.g27957\"; transcript_id \"chr5.g27957.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65962396', '65962703', '.', '+', '.', 'gene_id \"chr5.g27960\"; transcript_id \"chr5.g27960.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65968073', '65968338', '.', '-', '.', 'gene_id \"chr5.g27965\"; transcript_id \"chr5.g27965.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65973842', '65974621', '.', '-', '.', 'gene_id \"chr5.g27967\"; transcript_id \"chr5.g27967.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65981287', '65981693', '.', '+', '.', 'gene_id \"chr5.g27971\"; transcript_id \"chr5.g27971.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66002030', '66002184', '.', '-', '.', 'gene_id \"chr5.g27972\"; transcript_id \"chr5.g27972.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65984032', '65984444', '.', '-', '.', 'gene_id \"chr5.g27976\"; transcript_id \"chr5.g27976.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65996312', '65996929', '.', '+', '.', 'gene_id \"chr5.g27981\"; transcript_id \"chr5.g27981.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65982428', '65982655', '.', '+', '.', 'gene_id \"chr5.g27987\"; transcript_id \"chr5.g27987.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65984582', '65984752', '.', '-', '.', 'gene_id \"chr5.g27990\"; transcript_id \"chr5.g27990.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66004581', '66004796', '.', '-', '.', 'gene_id \"chr5.g27994\"; transcript_id \"chr5.g27994.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66004819', '66005077', '.', '+', '.', 'gene_id \"chr5.g27995\"; transcript_id \"chr5.g27995.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66006972', '66007608', '.', '+', '.', 'gene_id \"chr5.g27998\"; transcript_id \"chr5.g27998.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66007987', '66008378', '.', '-', '.', 'gene_id \"chr5.g28000\"; transcript_id \"chr5.g28000.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66017778', '66018037', '.', '-', '.', 'gene_id \"chr5.g28009\"; transcript_id \"chr5.g28009.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66021647', '66021918', '.', '+', '.', 'gene_id \"chr5.g28013\"; transcript_id \"chr5.g28013.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66022425', '66023228', '.', '-', '.', 'gene_id \"chr5.g28014\"; transcript_id \"chr5.g28014.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66023196', '66024216', '.', '+', '.', 'gene_id \"chr5.g28016\"; transcript_id \"chr5.g28016.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66028264', '66028478', '.', '-', '.', 'gene_id \"chr5.g28023\"; transcript_id \"chr5.g28023.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66034035', '66034223', '.', '+', '.', 'gene_id \"chr5.g28025\"; transcript_id \"chr5.g28025.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66038914', '66040558', '.', '+', '.', 'gene_id \"chr5.g28032\"; transcript_id \"chr5.g28032.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66040499', '66041114', '.', '-', '.', 'gene_id \"chr5.g28034\"; transcript_id \"chr5.g28034.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66038709', '66038936', '.', '-', '.', 'gene_id \"chr5.g28035\"; transcript_id \"chr5.g28035.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66041922', '66042266', '.', '+', '.', 'gene_id \"chr5.g28037\"; transcript_id \"chr5.g28037.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66042450', '66042714', '.', '+', '.', 'gene_id \"chr5.g28038\"; transcript_id \"chr5.g28038.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66043201', '66043481', '.', '-', '.', 'gene_id \"chr5.g28040\"; transcript_id \"chr5.g28040.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66053172', '66053810', '.', '-', '.', 'gene_id \"chr5.g28044\"; transcript_id \"chr5.g28044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66054748', '66055234', '.', '-', '.', 'gene_id \"chr5.g28045\"; transcript_id \"chr5.g28045.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66059095', '66059246', '.', '+', '.', 'gene_id \"chr5.g28055\"; transcript_id \"chr5.g28055.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66075845', '66075997', '.', '+', '.', 'gene_id \"chr5.g28057\"; transcript_id \"chr5.g28057.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66085219', '66085404', '.', '-', '.', 'gene_id \"chr5.g28059\"; transcript_id \"chr5.g28059.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66168689', '66168976', '.', '-', '.', 'gene_id \"chr5.g28069\"; transcript_id \"chr5.g28069.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66171843', '66172031', '.', '+', '.', 'gene_id \"chr5.g28070\"; transcript_id \"chr5.g28070.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66179238', '66182653', '.', '-', '.', 'gene_id \"chr5.g28073\"; transcript_id \"chr5.g28073.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66230522', '66230726', '.', '+', '.', 'gene_id \"chr5.g28076\"; transcript_id \"chr5.g28076.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66245292', '66245630', '.', '+', '.', 'gene_id \"chr5.g28077\"; transcript_id \"chr5.g28077.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66267737', '66267801', '.', '+', '.', 'gene_id \"chr5.g28079\"; transcript_id \"chr5.g28079.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66433812', '66434089', '.', '+', '.', 'gene_id \"chr5.g28090\"; transcript_id \"chr5.g28090.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66463912', '66464167', '.', '+', '.', 'gene_id \"chr5.g28094\"; transcript_id \"chr5.g28094.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66464338', '66464678', '.', '+', '.', 'gene_id \"chr5.g28095\"; transcript_id \"chr5.g28095.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66467757', '66468024', '.', '+', '.', 'gene_id \"chr5.g28096\"; transcript_id \"chr5.g28096.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66475211', '66475471', '.', '-', '.', 'gene_id \"chr5.g28097\"; transcript_id \"chr5.g28097.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66529075', '66529344', '.', '+', '.', 'gene_id \"chr5.g28109\"; transcript_id \"chr5.g28109.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66538380', '66538890', '.', '+', '.', 'gene_id \"chr5.g28113\"; transcript_id \"chr5.g28113.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66557648', '66559828', '.', '-', '.', 'gene_id \"chr5.g28116\"; transcript_id \"chr5.g28116.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66557030', '66557350', '.', '-', '.', 'gene_id \"chr5.g28116\"; transcript_id \"chr5.g28116.i10\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66562722', '66562977', '.', '-', '.', 'gene_id \"chr5.g28116\"; transcript_id \"chr5.g28116.i14\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66573369', '66573600', '.', '-', '.', 'gene_id \"chr5.g28129\"; transcript_id \"chr5.g28129.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66572687', '66572930', '.', '+', '.', 'gene_id \"chr5.g28132\"; transcript_id \"chr5.g28132.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66587804', '66589799', '.', '-', '.', 'gene_id \"chr5.g28136\"; transcript_id \"chr5.g28136.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66589955', '66590205', '.', '+', '.', 'gene_id \"chr5.g28139\"; transcript_id \"chr5.g28139.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66590377', '66590913', '.', '+', '.', 'gene_id \"chr5.g28140\"; transcript_id \"chr5.g28140.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66617323', '66617588', '.', '-', '.', 'gene_id \"chr5.g28152\"; transcript_id \"chr5.g28152.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120825485', '120825638', '.', '-', '.', 'gene_id \"chrX.g32006\"; transcript_id \"chrX.g32006.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66619092', '66619555', '.', '+', '.', 'gene_id \"chr5.g28153\"; transcript_id \"chr5.g28153.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66619773', '66620133', '.', '+', '.', 'gene_id \"chr5.g28154\"; transcript_id \"chr5.g28154.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66621231', '66621452', '.', '+', '.', 'gene_id \"chr5.g28158\"; transcript_id \"chr5.g28158.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66621826', '66622046', '.', '+', '.', 'gene_id \"chr5.g28160\"; transcript_id \"chr5.g28160.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66627514', '66627776', '.', '+', '.', 'gene_id \"chr5.g28165\"; transcript_id \"chr5.g28165.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66627982', '66628209', '.', '-', '.', 'gene_id \"chr5.g28166\"; transcript_id \"chr5.g28166.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66629756', '66630056', '.', '-', '.', 'gene_id \"chr5.g28167\"; transcript_id \"chr5.g28167.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66632115', '66632586', '.', '-', '.', 'gene_id \"chr5.g28169\"; transcript_id \"chr5.g28169.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66632944', '66633115', '.', '-', '.', 'gene_id \"chr5.g28170\"; transcript_id \"chr5.g28170.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66633112', '66633271', '.', '+', '.', 'gene_id \"chr5.g28171\"; transcript_id \"chr5.g28171.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66638444', '66638608', '.', '-', '.', 'gene_id \"chr5.g28176\"; transcript_id \"chr5.g28176.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66647058', '66647226', '.', '-', '.', 'gene_id \"chr5.g28180\"; transcript_id \"chr5.g28180.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66654597', '66654958', '.', '-', '.', 'gene_id \"chr5.g28188\"; transcript_id \"chr5.g28188.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66669291', '66669505', '.', '-', '.', 'gene_id \"chr5.g28189\"; transcript_id \"chr5.g28189.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66670568', '66670718', '.', '+', '.', 'gene_id \"chr5.g28190\"; transcript_id \"chr5.g28190.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66747114', '66747278', '.', '+', '.', 'gene_id \"chr5.g28194\"; transcript_id \"chr5.g28194.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66824807', '66825108', '.', '+', '.', 'gene_id \"chr5.g28200\"; transcript_id \"chr5.g28200.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66831973', '66832977', '.', '-', '.', 'gene_id \"chr5.g28201\"; transcript_id \"chr5.g28201.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66839340', '66839495', '.', '-', '.', 'gene_id \"chr5.g28204\"; transcript_id \"chr5.g28204.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66842185', '66842367', '.', '+', '.', 'gene_id \"chr5.g28205\"; transcript_id \"chr5.g28205.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66853707', '66853934', '.', '+', '.', 'gene_id \"chr5.g28208\"; transcript_id \"chr5.g28208.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66857082', '66857349', '.', '-', '.', 'gene_id \"chr5.g28210\"; transcript_id \"chr5.g28210.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66864815', '66864970', '.', '+', '.', 'gene_id \"chr5.g28211\"; transcript_id \"chr5.g28211.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66877657', '66877788', '.', '-', '.', 'gene_id \"chr5.g28212\"; transcript_id \"chr5.g28212.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66947933', '66948084', '.', '+', '.', 'gene_id \"chr5.g28217\"; transcript_id \"chr5.g28217.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67014662', '67014775', '.', '+', '.', 'gene_id \"chr5.g28221\"; transcript_id \"chr5.g28221.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67017861', '67018149', '.', '-', '.', 'gene_id \"chr5.g28226\"; transcript_id \"chr5.g28226.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67037543', '67037743', '.', '-', '.', 'gene_id \"chr5.g28227\"; transcript_id \"chr5.g28227.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67048055', '67048239', '.', '+', '.', 'gene_id \"chr5.g28229\"; transcript_id \"chr5.g28229.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67043569', '67044111', '.', '-', '.', 'gene_id \"chr5.g28231\"; transcript_id \"chr5.g28231.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121016663', '121016715', '.', '+', '.', 'gene_id \"chrX.g32095\"; transcript_id \"chrX.g32095.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67046289', '67046453', '.', '-', '.', 'gene_id \"chr5.g28236\"; transcript_id \"chr5.g28236.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67046489', '67046765', '.', '-', '.', 'gene_id \"chr5.g28237\"; transcript_id \"chr5.g28237.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67047289', '67047521', '.', '-', '.', 'gene_id \"chr5.g28238\"; transcript_id \"chr5.g28238.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67048256', '67048445', '.', '+', '.', 'gene_id \"chr5.g28239\"; transcript_id \"chr5.g28239.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67048523', '67048848', '.', '+', '.', 'gene_id \"chr5.g28240\"; transcript_id \"chr5.g28240.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121031968', '121032308', '.', '+', '.', 'gene_id \"chrX.g32112\"; transcript_id \"chrX.g32112.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67066683', '67066836', '.', '-', '.', 'gene_id \"chr5.g28246\"; transcript_id \"chr5.g28246.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67102830', '67103016', '.', '+', '.', 'gene_id \"chr5.g28251\"; transcript_id \"chr5.g28251.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67136902', '67137017', '.', '-', '.', 'gene_id \"chr5.g28252\"; transcript_id \"chr5.g28252.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67140077', '67140342', '.', '+', '.', 'gene_id \"chr5.g28253\"; transcript_id \"chr5.g28253.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67135085', '67135765', '.', '+', '.', 'gene_id \"chr5.g28255\"; transcript_id \"chr5.g28255.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67137575', '67140087', '.', '+', '.', 'gene_id \"chr5.g28256\"; transcript_id \"chr5.g28256.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67141162', '67141335', '.', '-', '.', 'gene_id \"chr5.g28257\"; transcript_id \"chr5.g28257.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67165511', '67166523', '.', '+', '.', 'gene_id \"chr5.g28263\"; transcript_id \"chr5.g28263.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67152612', '67152950', '.', '+', '.', 'gene_id \"chr5.g28267\"; transcript_id \"chr5.g28267.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67153054', '67153260', '.', '+', '.', 'gene_id \"chr5.g28268\"; transcript_id \"chr5.g28268.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67156630', '67156906', '.', '+', '.', 'gene_id \"chr5.g28269\"; transcript_id \"chr5.g28269.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67170431', '67170603', '.', '+', '.', 'gene_id \"chr5.g28273\"; transcript_id \"chr5.g28273.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67192117', '67192293', '.', '-', '.', 'gene_id \"chr5.g28279\"; transcript_id \"chr5.g28279.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67192338', '67192586', '.', '-', '.', 'gene_id \"chr5.g28280\"; transcript_id \"chr5.g28280.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67192956', '67193117', '.', '-', '.', 'gene_id \"chr5.g28281\"; transcript_id \"chr5.g28281.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67193177', '67193550', '.', '-', '.', 'gene_id \"chr5.g28282\"; transcript_id \"chr5.g28282.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67193908', '67194202', '.', '-', '.', 'gene_id \"chr5.g28283\"; transcript_id \"chr5.g28283.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67194424', '67194815', '.', '+', '.', 'gene_id \"chr5.g28284\"; transcript_id \"chr5.g28284.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67194207', '67194424', '.', '+', '.', 'gene_id \"chr5.g28285\"; transcript_id \"chr5.g28285.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67197675', '67197942', '.', '-', '.', 'gene_id \"chr5.g28286\"; transcript_id \"chr5.g28286.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67201325', '67201903', '.', '-', '.', 'gene_id \"chr5.g28290\"; transcript_id \"chr5.g28290.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67204548', '67204609', '.', '+', '.', 'gene_id \"chr5.g28294\"; transcript_id \"chr5.g28294.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67204478', '67205153', '.', '-', '.', 'gene_id \"chr5.g28295\"; transcript_id \"chr5.g28295.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67205797', '67206588', '.', '+', '.', 'gene_id \"chr5.g28298\"; transcript_id \"chr5.g28298.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67207567', '67208680', '.', '+', '.', 'gene_id \"chr5.g28299\"; transcript_id \"chr5.g28299.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67210042', '67210697', '.', '-', '.', 'gene_id \"chr5.g28302\"; transcript_id \"chr5.g28302.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67284928', '67285416', '.', '+', '.', 'gene_id \"chr5.g28308\"; transcript_id \"chr5.g28308.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67354502', '67354746', '.', '+', '.', 'gene_id \"chr5.g28309\"; transcript_id \"chr5.g28309.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67354502', '67354585', '.', '+', '.', 'gene_id \"chr5.g28309\"; transcript_id \"chr5.g28309.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67354586', '67354746', '.', '+', '.', 'gene_id \"chr5.g28309\"; transcript_id \"chr5.g28309.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67301608', '67301842', '.', '+', '.', 'gene_id \"chr5.g28312\"; transcript_id \"chr5.g28312.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67245931', '67246727', '.', '-', '.', 'gene_id \"chr5.g28314\"; transcript_id \"chr5.g28314.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67257418', '67257864', '.', '+', '.', 'gene_id \"chr5.g28317\"; transcript_id \"chr5.g28317.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67282686', '67282824', '.', '+', '.', 'gene_id \"chr5.g28319\"; transcript_id \"chr5.g28319.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67245137', '67245946', '.', '-', '.', 'gene_id \"chr5.g28321\"; transcript_id \"chr5.g28321.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67354748', '67355601', '.', '+', '.', 'gene_id \"chr5.g28323\"; transcript_id \"chr5.g28323.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67241556', '67241869', '.', '+', '.', 'gene_id \"chr5.g28324\"; transcript_id \"chr5.g28324.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67253545', '67253630', '.', '+', '.', 'gene_id \"chr5.g28326\"; transcript_id \"chr5.g28326.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67253839', '67254469', '.', '+', '.', 'gene_id \"chr5.g28328\"; transcript_id \"chr5.g28328.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67264716', '67265047', '.', '+', '.', 'gene_id \"chr5.g28334\"; transcript_id \"chr5.g28334.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67267435', '67267643', '.', '+', '.', 'gene_id \"chr5.g28335\"; transcript_id \"chr5.g28335.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67268928', '67269099', '.', '-', '.', 'gene_id \"chr5.g28336\"; transcript_id \"chr5.g28336.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67275373', '67275523', '.', '+', '.', 'gene_id \"chr5.g28337\"; transcript_id \"chr5.g28337.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67275672', '67275831', '.', '+', '.', 'gene_id \"chr5.g28338\"; transcript_id \"chr5.g28338.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67285704', '67285902', '.', '+', '.', 'gene_id \"chr5.g28341\"; transcript_id \"chr5.g28341.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67305633', '67305848', '.', '-', '.', 'gene_id \"chr5.g28343\"; transcript_id \"chr5.g28343.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67333686', '67333916', '.', '+', '.', 'gene_id \"chr5.g28351\"; transcript_id \"chr5.g28351.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67334869', '67335071', '.', '+', '.', 'gene_id \"chr5.g28353\"; transcript_id \"chr5.g28353.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67342893', '67343043', '.', '+', '.', 'gene_id \"chr5.g28360\"; transcript_id \"chr5.g28360.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67350561', '67350800', '.', '-', '.', 'gene_id \"chr5.g28363\"; transcript_id \"chr5.g28363.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67353253', '67353455', '.', '-', '.', 'gene_id \"chr5.g28364\"; transcript_id \"chr5.g28364.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67356366', '67356721', '.', '-', '.', 'gene_id \"chr5.g28365\"; transcript_id \"chr5.g28365.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67370463', '67370713', '.', '+', '.', 'gene_id \"chr5.g28372\"; transcript_id \"chr5.g28372.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67398859', '67399108', '.', '+', '.', 'gene_id \"chr5.g28375\"; transcript_id \"chr5.g28375.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67403001', '67403181', '.', '+', '.', 'gene_id \"chr5.g28379\"; transcript_id \"chr5.g28379.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67405631', '67406366', '.', '+', '.', 'gene_id \"chr5.g28380\"; transcript_id \"chr5.g28380.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121264153', '121264191', '.', '+', '.', 'gene_id \"chrX.g32280\"; transcript_id \"chrX.g32280.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67444244', '67444423', '.', '-', '.', 'gene_id \"chr5.g28389\"; transcript_id \"chr5.g28389.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67468226', '67468258', '.', '+', '.', 'gene_id \"chr5.g28392\"; transcript_id \"chr5.g28392.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67478914', '67480733', '.', '-', '.', 'gene_id \"chr5.g28396\"; transcript_id \"chr5.g28396.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67481329', '67481729', '.', '-', '.', 'gene_id \"chr5.g28399\"; transcript_id \"chr5.g28399.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67481927', '67482052', '.', '-', '.', 'gene_id \"chr5.g28400\"; transcript_id \"chr5.g28400.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67508302', '67508481', '.', '+', '.', 'gene_id \"chr5.g28403\"; transcript_id \"chr5.g28403.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67546242', '67546354', '.', '+', '.', 'gene_id \"chr5.g28405\"; transcript_id \"chr5.g28405.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67568055', '67568247', '.', '+', '.', 'gene_id \"chr5.g28411\"; transcript_id \"chr5.g28411.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67569459', '67569649', '.', '-', '.', 'gene_id \"chr5.g28412\"; transcript_id \"chr5.g28412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67571526', '67571729', '.', '-', '.', 'gene_id \"chr5.g28413\"; transcript_id \"chr5.g28413.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67571811', '67572084', '.', '-', '.', 'gene_id \"chr5.g28414\"; transcript_id \"chr5.g28414.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67585591', '67585758', '.', '+', '.', 'gene_id \"chr5.g28416\"; transcript_id \"chr5.g28416.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121455521', '121455558', '.', '+', '.', 'gene_id \"chrX.g32304\"; transcript_id \"chrX.g32304.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67585964', '67586149', '.', '-', '.', 'gene_id \"chr5.g28417\"; transcript_id \"chr5.g28417.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67590582', '67590735', '.', '+', '.', 'gene_id \"chr5.g28419\"; transcript_id \"chr5.g28419.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67602801', '67602981', '.', '-', '.', 'gene_id \"chr5.g28421\"; transcript_id \"chr5.g28421.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67606559', '67606732', '.', '-', '.', 'gene_id \"chr5.g28423\"; transcript_id \"chr5.g28423.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67620201', '67620448', '.', '+', '.', 'gene_id \"chr5.g28425\"; transcript_id \"chr5.g28425.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67619419', '67619753', '.', '-', '.', 'gene_id \"chr5.g28427\"; transcript_id \"chr5.g28427.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67623579', '67623785', '.', '+', '.', 'gene_id \"chr5.g28436\"; transcript_id \"chr5.g28436.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67627260', '67627437', '.', '-', '.', 'gene_id \"chr5.g28437\"; transcript_id \"chr5.g28437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67644894', '67645091', '.', '+', '.', 'gene_id \"chr5.g28445\"; transcript_id \"chr5.g28445.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121566354', '121566397', '.', '+', '.', 'gene_id \"chrX.g32331\"; transcript_id \"chrX.g32331.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67656085', '67656287', '.', '-', '.', 'gene_id \"chr5.g28450\"; transcript_id \"chr5.g28450.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67685822', '67686029', '.', '+', '.', 'gene_id \"chr5.g28458\"; transcript_id \"chr5.g28458.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121602269', '121602773', '.', '-', '.', 'gene_id \"chrX.g32347\"; transcript_id \"chrX.g32347.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67722070', '67722245', '.', '-', '.', 'gene_id \"chr5.g28465\"; transcript_id \"chr5.g28465.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67727612', '67727855', '.', '+', '.', 'gene_id \"chr5.g28467\"; transcript_id \"chr5.g28467.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67728100', '67728468', '.', '-', '.', 'gene_id \"chr5.g28468\"; transcript_id \"chr5.g28468.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67791446', '67791627', '.', '+', '.', 'gene_id \"chr5.g28474\"; transcript_id \"chr5.g28474.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67750486', '67751875', '.', '+', '.', 'gene_id \"chr5.g28476\"; transcript_id \"chr5.g28476.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67752323', '67752632', '.', '+', '.', 'gene_id \"chr5.g28477\"; transcript_id \"chr5.g28477.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67747877', '67748209', '.', '+', '.', 'gene_id \"chr5.g28484\"; transcript_id \"chr5.g28484.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67749174', '67749340', '.', '-', '.', 'gene_id \"chr5.g28487\"; transcript_id \"chr5.g28487.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67778879', '67779586', '.', '-', '.', 'gene_id \"chr5.g28492\"; transcript_id \"chr5.g28492.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67779588', '67779804', '.', '-', '.', 'gene_id \"chr5.g28493\"; transcript_id \"chr5.g28493.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67782577', '67782740', '.', '-', '.', 'gene_id \"chr5.g28496\"; transcript_id \"chr5.g28496.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67782886', '67783157', '.', '+', '.', 'gene_id \"chr5.g28497\"; transcript_id \"chr5.g28497.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67783738', '67783936', '.', '-', '.', 'gene_id \"chr5.g28498\"; transcript_id \"chr5.g28498.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67785342', '67785742', '.', '-', '.', 'gene_id \"chr5.g28499\"; transcript_id \"chr5.g28499.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67789967', '67790147', '.', '+', '.', 'gene_id \"chr5.g28504\"; transcript_id \"chr5.g28504.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67794045', '67794294', '.', '+', '.', 'gene_id \"chr5.g28507\"; transcript_id \"chr5.g28507.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67824390', '67825171', '.', '-', '.', 'gene_id \"chr5.g28517\"; transcript_id \"chr5.g28517.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67823830', '67824353', '.', '-', '.', 'gene_id \"chr5.g28518\"; transcript_id \"chr5.g28518.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67803245', '67803911', '.', '-', '.', 'gene_id \"chr5.g28520\"; transcript_id \"chr5.g28520.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67805991', '67806214', '.', '+', '.', 'gene_id \"chr5.g28521\"; transcript_id \"chr5.g28521.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67812433', '67812833', '.', '+', '.', 'gene_id \"chr5.g28523\"; transcript_id \"chr5.g28523.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67816317', '67816505', '.', '-', '.', 'gene_id \"chr5.g28528\"; transcript_id \"chr5.g28528.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67820637', '67821943', '.', '+', '.', 'gene_id \"chr5.g28531\"; transcript_id \"chr5.g28531.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67822686', '67822969', '.', '-', '.', 'gene_id \"chr5.g28533\"; transcript_id \"chr5.g28533.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67829564', '67829971', '.', '+', '.', 'gene_id \"chr5.g28534\"; transcript_id \"chr5.g28534.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67832157', '67832586', '.', '-', '.', 'gene_id \"chr5.g28537\"; transcript_id \"chr5.g28537.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67871947', '67872104', '.', '-', '.', 'gene_id \"chr5.g28543\"; transcript_id \"chr5.g28543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67887823', '67888002', '.', '-', '.', 'gene_id \"chr5.g28545\"; transcript_id \"chr5.g28545.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67930052', '67930266', '.', '-', '.', 'gene_id \"chr5.g28547\"; transcript_id \"chr5.g28547.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67915583', '67916543', '.', '+', '.', 'gene_id \"chr5.g28549\"; transcript_id \"chr5.g28549.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67931459', '67931619', '.', '+', '.', 'gene_id \"chr5.g28550\"; transcript_id \"chr5.g28550.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67893978', '67894171', '.', '-', '.', 'gene_id \"chr5.g28559\"; transcript_id \"chr5.g28559.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67895547', '67895752', '.', '+', '.', 'gene_id \"chr5.g28561\"; transcript_id \"chr5.g28561.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67895875', '67896503', '.', '-', '.', 'gene_id \"chr5.g28562\"; transcript_id \"chr5.g28562.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67896858', '67897194', '.', '+', '.', 'gene_id \"chr5.g28563\"; transcript_id \"chr5.g28563.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67897714', '67897985', '.', '+', '.', 'gene_id \"chr5.g28565\"; transcript_id \"chr5.g28565.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67898585', '67898847', '.', '+', '.', 'gene_id \"chr5.g28567\"; transcript_id \"chr5.g28567.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67899254', '67899541', '.', '-', '.', 'gene_id \"chr5.g28568\"; transcript_id \"chr5.g28568.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67898972', '67899271', '.', '-', '.', 'gene_id \"chr5.g28570\"; transcript_id \"chr5.g28570.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67906721', '67907422', '.', '+', '.', 'gene_id \"chr5.g28576\"; transcript_id \"chr5.g28576.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67914806', '67915158', '.', '+', '.', 'gene_id \"chr5.g28584\"; transcript_id \"chr5.g28584.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67915286', '67915455', '.', '-', '.', 'gene_id \"chr5.g28585\"; transcript_id \"chr5.g28585.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67928316', '67928633', '.', '-', '.', 'gene_id \"chr5.g28586\"; transcript_id \"chr5.g28586.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67935904', '67936133', '.', '+', '.', 'gene_id \"chr5.g28588\"; transcript_id \"chr5.g28588.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67939393', '67939547', '.', '-', '.', 'gene_id \"chr5.g28589\"; transcript_id \"chr5.g28589.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67948950', '67949156', '.', '-', '.', 'gene_id \"chr5.g28592\"; transcript_id \"chr5.g28592.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67981228', '67981691', '.', '-', '.', 'gene_id \"chr5.g28593\"; transcript_id \"chr5.g28593.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67981921', '67982079', '.', '+', '.', 'gene_id \"chr5.g28594\"; transcript_id \"chr5.g28594.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67982760', '67983216', '.', '-', '.', 'gene_id \"chr5.g28595\"; transcript_id \"chr5.g28595.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67983332', '67983573', '.', '+', '.', 'gene_id \"chr5.g28596\"; transcript_id \"chr5.g28596.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67999626', '67999990', '.', '+', '.', 'gene_id \"chr5.g28603\"; transcript_id \"chr5.g28603.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68000337', '68000646', '.', '+', '.', 'gene_id \"chr5.g28614\"; transcript_id \"chr5.g28614.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68003819', '68004038', '.', '-', '.', 'gene_id \"chr5.g28617\"; transcript_id \"chr5.g28617.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68004967', '68005185', '.', '+', '.', 'gene_id \"chr5.g28619\"; transcript_id \"chr5.g28619.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68007531', '68007737', '.', '+', '.', 'gene_id \"chr5.g28620\"; transcript_id \"chr5.g28620.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68009510', '68009683', '.', '-', '.', 'gene_id \"chr5.g28621\"; transcript_id \"chr5.g28621.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68016710', '68017016', '.', '+', '.', 'gene_id \"chr5.g28624\"; transcript_id \"chr5.g28624.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68026984', '68027239', '.', '-', '.', 'gene_id \"chr5.g28628\"; transcript_id \"chr5.g28628.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68028662', '68028818', '.', '-', '.', 'gene_id \"chr5.g28629\"; transcript_id \"chr5.g28629.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68028925', '68029285', '.', '-', '.', 'gene_id \"chr5.g28630\"; transcript_id \"chr5.g28630.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68031333', '68031554', '.', '-', '.', 'gene_id \"chr5.g28634\"; transcript_id \"chr5.g28634.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68034757', '68035113', '.', '+', '.', 'gene_id \"chr5.g28636\"; transcript_id \"chr5.g28636.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68047246', '68047480', '.', '-', '.', 'gene_id \"chr5.g28647\"; transcript_id \"chr5.g28647.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68047500', '68047959', '.', '-', '.', 'gene_id \"chr5.g28648\"; transcript_id \"chr5.g28648.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68048805', '68048971', '.', '-', '.', 'gene_id \"chr5.g28649\"; transcript_id \"chr5.g28649.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68049020', '68049287', '.', '+', '.', 'gene_id \"chr5.g28650\"; transcript_id \"chr5.g28650.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68050019', '68050278', '.', '-', '.', 'gene_id \"chr5.g28651\"; transcript_id \"chr5.g28651.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121935453', '121935773', '.', '-', '.', 'gene_id \"chrX.g32526\"; transcript_id \"chrX.g32526.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121946599', '121946749', '.', '+', '.', 'gene_id \"chrX.g32540\"; transcript_id \"chrX.g32540.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68118263', '68118544', '.', '+', '.', 'gene_id \"chr5.g28668\"; transcript_id \"chr5.g28668.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68139707', '68139863', '.', '+', '.', 'gene_id \"chr5.g28668\"; transcript_id \"chr5.g28668.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68120544', '68121072', '.', '+', '.', 'gene_id \"chr5.g28673\"; transcript_id \"chr5.g28673.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68122689', '68122722', '.', '+', '.', 'gene_id \"chr5.g28674\"; transcript_id \"chr5.g28674.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68126609', '68127913', '.', '-', '.', 'gene_id \"chr5.g28675\"; transcript_id \"chr5.g28675.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68129208', '68129512', '.', '+', '.', 'gene_id \"chr5.g28680\"; transcript_id \"chr5.g28680.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68129524', '68130472', '.', '+', '.', 'gene_id \"chr5.g28683\"; transcript_id \"chr5.g28683.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68134345', '68134976', '.', '-', '.', 'gene_id \"chr5.g28688\"; transcript_id \"chr5.g28688.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68134026', '68134358', '.', '+', '.', 'gene_id \"chr5.g28689\"; transcript_id \"chr5.g28689.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68135038', '68136099', '.', '+', '.', 'gene_id \"chr5.g28690\"; transcript_id \"chr5.g28690.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68137045', '68137469', '.', '+', '.', 'gene_id \"chr5.g28692\"; transcript_id \"chr5.g28692.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68138757', '68139190', '.', '+', '.', 'gene_id \"chr5.g28694\"; transcript_id \"chr5.g28694.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68139881', '68141364', '.', '+', '.', 'gene_id \"chr5.g28695\"; transcript_id \"chr5.g28695.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68143710', '68143937', '.', '-', '.', 'gene_id \"chr5.g28700\"; transcript_id \"chr5.g28700.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68146315', '68148104', '.', '+', '.', 'gene_id \"chr5.g28702\"; transcript_id \"chr5.g28702.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68169948', '68170121', '.', '-', '.', 'gene_id \"chr5.g28708\"; transcript_id \"chr5.g28708.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68171194', '68171378', '.', '-', '.', 'gene_id \"chr5.g28709\"; transcript_id \"chr5.g28709.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68187670', '68187921', '.', '-', '.', 'gene_id \"chr5.g28717\"; transcript_id \"chr5.g28717.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68200256', '68200430', '.', '+', '.', 'gene_id \"chr5.g28718\"; transcript_id \"chr5.g28718.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68212592', '68213035', '.', '-', '.', 'gene_id \"chr5.g28722\"; transcript_id \"chr5.g28722.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68210790', '68210955', '.', '-', '.', 'gene_id \"chr5.g28725\"; transcript_id \"chr5.g28725.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68226088', '68227605', '.', '-', '.', 'gene_id \"chr5.g28729\"; transcript_id \"chr5.g28729.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68231552', '68231803', '.', '-', '.', 'gene_id \"chr5.g28730\"; transcript_id \"chr5.g28730.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68234603', '68234769', '.', '-', '.', 'gene_id \"chr5.g28733\"; transcript_id \"chr5.g28733.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68233796', '68234005', '.', '-', '.', 'gene_id \"chr5.g28736\"; transcript_id \"chr5.g28736.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68249737', '68249889', '.', '-', '.', 'gene_id \"chr5.g28741\"; transcript_id \"chr5.g28741.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68251092', '68251257', '.', '-', '.', 'gene_id \"chr5.g28742\"; transcript_id \"chr5.g28742.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68254603', '68255062', '.', '+', '.', 'gene_id \"chr5.g28745\"; transcript_id \"chr5.g28745.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122118282', '122118465', '.', '-', '.', 'gene_id \"chrX.g32639\"; transcript_id \"chrX.g32639.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68319271', '68319748', '.', '-', '.', 'gene_id \"chr5.g28762\"; transcript_id \"chr5.g28762.i14\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68323358', '68324890', '.', '+', '.', 'gene_id \"chr5.g28763\"; transcript_id \"chr5.g28763.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68341535', '68341696', '.', '+', '.', 'gene_id \"chr5.g28765\"; transcript_id \"chr5.g28765.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68341318', '68341556', '.', '+', '.', 'gene_id \"chr5.g28765\"; transcript_id \"chr5.g28765.i2\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122135412', '122135454', '.', '-', '.', 'gene_id \"chrX.g32645\"; transcript_id \"chrX.g32645.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68328757', '68331421', '.', '+', '.', 'gene_id \"chr5.g28769\"; transcript_id \"chr5.g28769.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68397504', '68398588', '.', '-', '.', 'gene_id \"chr5.g28779\"; transcript_id \"chr5.g28779.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68401183', '68401659', '.', '+', '.', 'gene_id \"chr5.g28784\"; transcript_id \"chr5.g28784.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122151315', '122151357', '.', '-', '.', 'gene_id \"chrX.g32666\"; transcript_id \"chrX.g32666.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122155873', '122156505', '.', '+', '.', 'gene_id \"chrX.g32669\"; transcript_id \"chrX.g32669.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68414075', '68414266', '.', '-', '.', 'gene_id \"chr5.g28791\"; transcript_id \"chr5.g28791.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68431607', '68431829', '.', '-', '.', 'gene_id \"chr5.g28794\"; transcript_id \"chr5.g28794.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68431403', '68431629', '.', '-', '.', 'gene_id \"chr5.g28794\"; transcript_id \"chr5.g28794.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68430404', '68430553', '.', '-', '.', 'gene_id \"chr5.g28798\"; transcript_id \"chr5.g28798.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122174877', '122175113', '.', '+', '.', 'gene_id \"chrX.g32673\"; transcript_id \"chrX.g32673.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68430894', '68431047', '.', '+', '.', 'gene_id \"chr5.g28805\"; transcript_id \"chr5.g28805.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68430767', '68430905', '.', '+', '.', 'gene_id \"chr5.g28806\"; transcript_id \"chr5.g28806.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68464467', '68464617', '.', '+', '.', 'gene_id \"chr5.g28810\"; transcript_id \"chr5.g28810.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68469711', '68470064', '.', '+', '.', 'gene_id \"chr5.g28811\"; transcript_id \"chr5.g28811.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68464842', '68465059', '.', '+', '.', 'gene_id \"chr5.g28817\"; transcript_id \"chr5.g28817.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68466749', '68467035', '.', '-', '.', 'gene_id \"chr5.g28819\"; transcript_id \"chr5.g28819.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68474060', '68474211', '.', '-', '.', 'gene_id \"chr5.g28825\"; transcript_id \"chr5.g28825.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68520424', '68520609', '.', '-', '.', 'gene_id \"chr5.g28829\"; transcript_id \"chr5.g28829.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68507582', '68507771', '.', '-', '.', 'gene_id \"chr5.g28830\"; transcript_id \"chr5.g28830.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68512044', '68512298', '.', '+', '.', 'gene_id \"chr5.g28832\"; transcript_id \"chr5.g28832.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68507307', '68507456', '.', '+', '.', 'gene_id \"chr5.g28834\"; transcript_id \"chr5.g28834.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68498771', '68498943', '.', '+', '.', 'gene_id \"chr5.g28836\"; transcript_id \"chr5.g28836.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68476477', '68476643', '.', '-', '.', 'gene_id \"chr5.g28842\"; transcript_id \"chr5.g28842.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68478201', '68478484', '.', '-', '.', 'gene_id \"chr5.g28846\"; transcript_id \"chr5.g28846.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68484350', '68484519', '.', '-', '.', 'gene_id \"chr5.g28853\"; transcript_id \"chr5.g28853.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68485711', '68486161', '.', '-', '.', 'gene_id \"chr5.g28856\"; transcript_id \"chr5.g28856.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68486834', '68487069', '.', '-', '.', 'gene_id \"chr5.g28859\"; transcript_id \"chr5.g28859.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68494257', '68494417', '.', '-', '.', 'gene_id \"chr5.g28862\"; transcript_id \"chr5.g28862.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68505045', '68505269', '.', '-', '.', 'gene_id \"chr5.g28868\"; transcript_id \"chr5.g28868.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68509452', '68510176', '.', '+', '.', 'gene_id \"chr5.g28872\"; transcript_id \"chr5.g28872.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68510176', '68510674', '.', '-', '.', 'gene_id \"chr5.g28873\"; transcript_id \"chr5.g28873.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68510818', '68510970', '.', '+', '.', 'gene_id \"chr5.g28874\"; transcript_id \"chr5.g28874.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68510936', '68511127', '.', '+', '.', 'gene_id \"chr5.g28874\"; transcript_id \"chr5.g28874.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68512331', '68512831', '.', '-', '.', 'gene_id \"chr5.g28875\"; transcript_id \"chr5.g28875.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68515031', '68515451', '.', '+', '.', 'gene_id \"chr5.g28876\"; transcript_id \"chr5.g28876.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68516989', '68517174', '.', '+', '.', 'gene_id \"chr5.g28877\"; transcript_id \"chr5.g28877.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68517387', '68517597', '.', '+', '.', 'gene_id \"chr5.g28880\"; transcript_id \"chr5.g28880.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68525143', '68525296', '.', '-', '.', 'gene_id \"chr5.g28881\"; transcript_id \"chr5.g28881.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68623802', '68625300', '.', '+', '.', 'gene_id \"chr5.g28883\"; transcript_id \"chr5.g28883.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68569765', '68569963', '.', '+', '.', 'gene_id \"chr5.g28889\"; transcript_id \"chr5.g28889.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68561511', '68561710', '.', '+', '.', 'gene_id \"chr5.g28891\"; transcript_id \"chr5.g28891.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68573059', '68573393', '.', '+', '.', 'gene_id \"chr5.g28897\"; transcript_id \"chr5.g28897.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68573395', '68574951', '.', '-', '.', 'gene_id \"chr5.g28899\"; transcript_id \"chr5.g28899.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68574956', '68575300', '.', '-', '.', 'gene_id \"chr5.g28900\"; transcript_id \"chr5.g28900.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68577759', '68577971', '.', '+', '.', 'gene_id \"chr5.g28902\"; transcript_id \"chr5.g28902.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68602942', '68603166', '.', '+', '.', 'gene_id \"chr5.g28908\"; transcript_id \"chr5.g28908.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68605558', '68605791', '.', '-', '.', 'gene_id \"chr5.g28909\"; transcript_id \"chr5.g28909.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68607047', '68607234', '.', '-', '.', 'gene_id \"chr5.g28910\"; transcript_id \"chr5.g28910.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68627440', '68627628', '.', '+', '.', 'gene_id \"chr5.g28916\"; transcript_id \"chr5.g28916.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68627703', '68627857', '.', '-', '.', 'gene_id \"chr5.g28917\"; transcript_id \"chr5.g28917.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68627703', '68627857', '.', '-', '.', 'gene_id \"chr5.g28917\"; transcript_id \"chr5.g28917.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68634023', '68634212', '.', '-', '.', 'gene_id \"chr5.g28920\"; transcript_id \"chr5.g28920.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68720716', '68720870', '.', '-', '.', 'gene_id \"chr5.g28933\"; transcript_id \"chr5.g28933.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68727929', '68728232', '.', '-', '.', 'gene_id \"chr5.g28937\"; transcript_id \"chr5.g28937.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68728601', '68728847', '.', '+', '.', 'gene_id \"chr5.g28939\"; transcript_id \"chr5.g28939.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68731074', '68731242', '.', '-', '.', 'gene_id \"chr5.g28940\"; transcript_id \"chr5.g28940.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68734136', '68734536', '.', '-', '.', 'gene_id \"chr5.g28943\"; transcript_id \"chr5.g28943.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69218227', '69218376', '.', '-', '.', 'gene_id \"chr5.g28946\"; transcript_id \"chr5.g28946.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69118572', '69118717', '.', '-', '.', 'gene_id \"chr5.g28946\"; transcript_id \"chr5.g28946.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68770607', '68770748', '.', '-', '.', 'gene_id \"chr5.g28946\"; transcript_id \"chr5.g28946.i9\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69464497', '69464705', '.', '+', '.', 'gene_id \"chr5.g28947\"; transcript_id \"chr5.g28947.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68738043', '68738951', '.', '+', '.', 'gene_id \"chr5.g28950\"; transcript_id \"chr5.g28950.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68738949', '68739278', '.', '+', '.', 'gene_id \"chr5.g28952\"; transcript_id \"chr5.g28952.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68741399', '68741572', '.', '-', '.', 'gene_id \"chr5.g28954\"; transcript_id \"chr5.g28954.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122444629', '122444688', '.', '-', '.', 'gene_id \"chrX.g32795\"; transcript_id \"chrX.g32795.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68743043', '68743234', '.', '-', '.', 'gene_id \"chr5.g28956\"; transcript_id \"chr5.g28956.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68748459', '68748686', '.', '-', '.', 'gene_id \"chr5.g28961\"; transcript_id \"chr5.g28961.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68749247', '68749414', '.', '+', '.', 'gene_id \"chr5.g28962\"; transcript_id \"chr5.g28962.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68761888', '68762051', '.', '+', '.', 'gene_id \"chr5.g28963\"; transcript_id \"chr5.g28963.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68765263', '68765485', '.', '+', '.', 'gene_id \"chr5.g28964\"; transcript_id \"chr5.g28964.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68809316', '68809818', '.', '-', '.', 'gene_id \"chr5.g28971\"; transcript_id \"chr5.g28971.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68832214', '68832364', '.', '+', '.', 'gene_id \"chr5.g28972\"; transcript_id \"chr5.g28972.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68847073', '68847285', '.', '+', '.', 'gene_id \"chr5.g28974\"; transcript_id \"chr5.g28974.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122592487', '122592652', '.', '-', '.', 'gene_id \"chrX.g32820\"; transcript_id \"chrX.g32820.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68919817', '68919972', '.', '-', '.', 'gene_id \"chr5.g28979\"; transcript_id \"chr5.g28979.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122618649', '122618727', '.', '+', '.', 'gene_id \"chrX.g32827\"; transcript_id \"chrX.g32827.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68953210', '68953486', '.', '-', '.', 'gene_id \"chr5.g28987\"; transcript_id \"chr5.g28987.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68953888', '68954104', '.', '+', '.', 'gene_id \"chr5.g28989\"; transcript_id \"chr5.g28989.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68954246', '68954396', '.', '-', '.', 'gene_id \"chr5.g28990\"; transcript_id \"chr5.g28990.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69001191', '69001741', '.', '+', '.', 'gene_id \"chr5.g28996\"; transcript_id \"chr5.g28996.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69013813', '69014024', '.', '+', '.', 'gene_id \"chr5.g28999\"; transcript_id \"chr5.g28999.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122634568', '122634610', '.', '+', '.', 'gene_id \"chrX.g32842\"; transcript_id \"chrX.g32842.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69014359', '69014534', '.', '+', '.', 'gene_id \"chr5.g29000\"; transcript_id \"chr5.g29000.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69015588', '69015872', '.', '+', '.', 'gene_id \"chr5.g29002\"; transcript_id \"chr5.g29002.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69016479', '69017201', '.', '-', '.', 'gene_id \"chr5.g29003\"; transcript_id \"chr5.g29003.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69040421', '69040662', '.', '-', '.', 'gene_id \"chr5.g29007\"; transcript_id \"chr5.g29007.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122708420', '122708464', '.', '-', '.', 'gene_id \"chrX.g32859\"; transcript_id \"chrX.g32859.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93795510', '93795555', '100', '+', '.', 'gene_id \"chrX.g32910\"; transcript_id \"blat.proc4587.chain_4851\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90930956', '90930991', '100', '+', '.', 'gene_id \"chrX.g32947\"; transcript_id \"blat.proc4005.chain_475\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73675578', '73675611', '100', '-', '.', 'gene_id \"chrX.g32956\"; transcript_id \"blat.proc4005.chain_15928\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54338504', '54338561', '100', '-', '.', 'gene_id \"chrX.g33077\"; transcript_id \"blat.proc18128.chain_7699\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77153095', '77153265', '100', '-', '.', 'gene_id \"chrX.g33124\"; transcript_id \"blat.proc28144.chain_11912\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59534357', '59534525', '100', '-', '.', 'gene_id \"chrX.g33151\"; transcript_id \"blat.proc4587.chain_13669\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30158795', '30158840', '100', '+', '.', 'gene_id \"chrX.g33187\"; transcript_id \"blat.proc18128.chain_9715\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113815197', '113815234', '100', '+', '.', 'gene_id \"chrX.g33203\"; transcript_id \"blat.proc18128.chain_21635\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86032175', '86032217', '100', '+', '.', 'gene_id \"chrX.g33220\"; transcript_id \"blat.proc4587.chain_11594\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37218831', '37218952', '100', '+', '.', 'gene_id \"chrX.g33221\"; transcript_id \"blat.proc28144.chain_7351\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101774980', '101775208', '100', '-', '.', 'gene_id \"chrX.g33262\"; transcript_id \"blat.proc28144.chain_18972\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104475514', '104475557', '100', '-', '.', 'gene_id \"chrX.g33303\"; transcript_id \"blat.proc18128.chain_2340\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74097867', '74097953', '100', '+', '.', 'gene_id \"chrX.g33318\"; transcript_id \"blat.proc28144.chain_6174\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28938736', '28938840', '100', '+', '.', 'gene_id \"chrX.g33362\"; transcript_id \"blat.proc18128.chain_6356\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35667099', '35667151', '100', '+', '.', 'gene_id \"chrX.g33385\"; transcript_id \"blat.proc4005.chain_915\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112023639', '112023703', '100', '-', '.', 'gene_id \"chrX.g33409\"; transcript_id \"blat.proc18128.chain_12688\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90359685', '90359724', '100', '-', '.', 'gene_id \"chrX.g33413\"; transcript_id \"blat.proc4005.chain_21531\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21363135', '21363185', '100', '+', '.', 'gene_id \"chrX.g33437\"; transcript_id \"blat.proc18128.chain_17371\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78296733', '78296876', '100', '+', '.', 'gene_id \"chrX.g33452\"; transcript_id \"blat.proc4005.chain_6070\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66637820', '66638012', '100', '+', '.', 'gene_id \"chrX.g33453\"; transcript_id \"blat.proc18128.chain_13628\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53516750', '53516806', '100', '+', '.', 'gene_id \"chrX.g33470\"; transcript_id \"blat.proc4587.chain_13764\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113251364', '113251478', '100', '+', '.', 'gene_id \"chrX.g33471\"; transcript_id \"blat.proc18128.chain_15304\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19566498', '19566540', '100', '+', '.', 'gene_id \"chrX.g33528\"; transcript_id \"blat.proc18128.chain_10834\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35638384', '35638427', '100', '+', '.', 'gene_id \"chrX.g33570\"; transcript_id \"blat.proc4005.chain_13855\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '24681283', '24681320', '100', '+', '.', 'gene_id \"chrX.g33598\"; transcript_id \"blat.proc4587.chain_9508\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12952292', '12952432', '100', '+', '.', 'gene_id \"chrX.g33645\"; transcript_id \"blat.proc28144.chain_10686\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46214019', '46214241', '100', '+', '.', 'gene_id \"chrX.g33775\"; transcript_id \"blat.proc18128.chain_22418\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120149096', '120149135', '100', '-', '.', 'gene_id \"chrX.g33776\"; transcript_id \"blat.proc4005.chain_7969\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97194209', '97194305', '100', '+', '.', 'gene_id \"chrX.g33799\"; transcript_id \"blat.proc28144.chain_8771\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31519992', '31520025', '100', '+', '.', 'gene_id \"chrX.g33823\"; transcript_id \"blat.proc28144.chain_2153\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55714580', '55714617', '100', '-', '.', 'gene_id \"chrX.g33888\"; transcript_id \"blat.proc4005.chain_14349\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90145627', '90145788', '100', '-', '.', 'gene_id \"chrX.g33948\"; transcript_id \"blat.proc18128.chain_1868\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '113602328', '113602369', '100', '+', '.', 'gene_id \"chrX.g33952\"; transcript_id \"blat.proc28144.chain_125\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75062259', '75062482', '100', '+', '.', 'gene_id \"chrX.g34107\"; transcript_id \"blat.proc18128.chain_19915\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58769128', '58769173', '100', '-', '.', 'gene_id \"chrX.g34117\"; transcript_id \"blat.proc18128.chain_1630\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6446771', '6446892', '100', '+', '.', 'gene_id \"chrX.g34136\"; transcript_id \"blat.proc28144.chain_6573\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99717122', '99717184', '100', '+', '.', 'gene_id \"chrX.g34189\"; transcript_id \"blat.proc4005.chain_4295\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30388785', '30388939', '100', '+', '.', 'gene_id \"chrX.g34200\"; transcript_id \"blat.proc4587.chain_13006\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120449145', '120449253', '100', '+', '.', 'gene_id \"chrX.g34211\"; transcript_id \"blat.proc4587.chain_4875\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '2514104', '2514163', '100', '+', '.', 'gene_id \"chrX.g34216\"; transcript_id \"blat.proc4005.chain_9485\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45532568', '45532787', '100', '+', '.', 'gene_id \"chrX.g34259\"; transcript_id \"blat.proc18128.chain_3629\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115365557', '115365603', '100', '+', '.', 'gene_id \"chrX.g34266\"; transcript_id \"blat.proc18128.chain_7258\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '24966337', '24966373', '100', '-', '.', 'gene_id \"chrX.g34296\"; transcript_id \"blat.proc28144.chain_18755\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13508082', '13508261', '100', '-', '.', 'gene_id \"chrX.g34314\"; transcript_id \"blat.proc28144.chain_10235\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56922140', '56922187', '100', '+', '.', 'gene_id \"chrX.g34363\"; transcript_id \"blat.proc18128.chain_16635\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107525213', '107525251', '100', '-', '.', 'gene_id \"chrX.g34489\"; transcript_id \"blat.proc4005.chain_18463\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '5856462', '5856527', '100', '+', '.', 'gene_id \"chrX.g34537\"; transcript_id \"blat.proc28144.chain_7977\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14438310', '14438358', '100', '-', '.', 'gene_id \"chrX.g34580\"; transcript_id \"blat.proc18128.chain_25051\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18434300', '18434444', '100', '-', '.', 'gene_id \"chrX.g34655\"; transcript_id \"blat.proc28144.chain_9491\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71455322', '71455373', '100', '+', '.', 'gene_id \"chrX.g34670\"; transcript_id \"blat.proc4587.chain_2468\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86838685', '86838746', '100', '+', '.', 'gene_id \"chrX.g34718\"; transcript_id \"blat.proc4587.chain_10568\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chrX', 'PASA', 'exon', '68197621', '68197669', '100', '+', '.', 'gene_id \"chrX.g34774\"; transcript_id \"blat.proc18128.chain_8653\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4834269', '4834299', '100', '-', '.', 'gene_id \"chrX.g34831\"; transcript_id \"blat.proc28144.chain_10705\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53585083', '53585186', '100', '-', '.', 'gene_id \"chrX.g34847\"; transcript_id \"blat.proc4587.chain_13148\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101247371', '101247417', '100', '-', '.', 'gene_id \"chrX.g34930\"; transcript_id \"blat.proc18128.chain_1629\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90268057', '90268096', '100', '+', '.', 'gene_id \"chrX.g34955\"; transcript_id \"blat.proc28144.chain_6731\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79917510', '79917678', '100', '+', '.', 'gene_id \"chrX.g34957\"; transcript_id \"blat.proc4587.chain_13660\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73666108', '73666154', '100', '-', '.', 'gene_id \"chrX.g34967\"; transcript_id \"blat.proc28144.chain_18951\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16327854', '16327894', '100', '+', '.', 'gene_id \"chrX.g34971\"; transcript_id \"blat.proc4005.chain_10879\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104291084', '104291125', '100', '+', '.', 'gene_id \"chrX.g35031\"; transcript_id \"blat.proc4005.chain_15317\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56174734', '56174794', '100', '+', '.', 'gene_id \"chrX.g35048\"; transcript_id \"blat.proc28144.chain_14797\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122076245', '122076384', '100', '-', '.', 'gene_id \"chrX.g35059\"; transcript_id \"blat.proc28144.chain_16445\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94127626', '94127659', '100', '+', '.', 'gene_id \"chrX.g35065\"; transcript_id \"blat.proc28144.chain_1164\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43972327', '43972546', '100', '-', '.', 'gene_id \"chrX.g35072\"; transcript_id \"blat.proc18128.chain_3923\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60812095', '60812307', '100', '+', '.', 'gene_id \"chrX.g35075\"; transcript_id \"blat.proc28144.chain_4538\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106801234', '106801278', '100', '+', '.', 'gene_id \"chrX.g35169\"; transcript_id \"blat.proc4587.chain_6336\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118208519', '118208548', '100', '+', '.', 'gene_id \"chrX.g35172\"; transcript_id \"blat.proc4005.chain_7661\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77259118', '77259809', '100', '+', '.', 'gene_id \"chrX.g35176\"; transcript_id \"blat.proc18128.chain_8404\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18300794', '18300823', '100', '-', '.', 'gene_id \"chrX.g35196\"; transcript_id \"blat.proc4005.chain_8414\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38418165', '38418216', '100', '+', '.', 'gene_id \"chrX.g35231\"; transcript_id \"blat.proc28144.chain_8814\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45647867', '45647990', '100', '-', '.', 'gene_id \"chrX.g35268\"; transcript_id \"blat.proc18128.chain_11383\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74497796', '74497836', '100', '-', '.', 'gene_id \"chrX.g35273\"; transcript_id \"blat.proc18128.chain_25427\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114133818', '114133882', '100', '+', '.', 'gene_id \"chrX.g35297\"; transcript_id \"blat.proc18128.chain_15911\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69093040', '69093271', '.', '-', '.', 'gene_id \"chr5.g29017\"; transcript_id \"chr5.g29017.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91337288', '91337323', '100', '-', '.', 'gene_id \"chrX.g35338\"; transcript_id \"blat.proc18128.chain_6122\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76365389', '76365424', '100', '-', '.', 'gene_id \"chrX.g35345\"; transcript_id \"blat.proc4587.chain_6716\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '95654860', '95654900', '100', '+', '.', 'gene_id \"chrX.g35374\"; transcript_id \"blat.proc28144.chain_5107\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30098241', '30098319', '100', '-', '.', 'gene_id \"chrX.g35399\"; transcript_id \"blat.proc4587.chain_2291\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91145032', '91145120', '100', '+', '.', 'gene_id \"chrX.g35408\"; transcript_id \"blat.proc18128.chain_23646\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79917126', '79917282', '100', '-', '.', 'gene_id \"chrX.g35478\"; transcript_id \"blat.proc4005.chain_21015\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117376773', '117376817', '100', '+', '.', 'gene_id \"chrX.g35548\"; transcript_id \"blat.proc4005.chain_21928\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55502921', '55502955', '100', '-', '.', 'gene_id \"chrX.g35629\"; transcript_id \"blat.proc28144.chain_8754\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6620667', '6620718', '100', '-', '.', 'gene_id \"chrX.g35633\"; transcript_id \"blat.proc18128.chain_11770\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52882193', '52882374', '100', '+', '.', 'gene_id \"chrX.g35678\"; transcript_id \"blat.proc28144.chain_4380\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40293215', '40293399', '100', '-', '.', 'gene_id \"chrX.g35684\"; transcript_id \"blat.proc4587.chain_15\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105483599', '105483752', '100', '-', '.', 'gene_id \"chrX.g35740\"; transcript_id \"blat.proc18128.chain_20665\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38313356', '38313396', '100', '-', '.', 'gene_id \"chrX.g35751\"; transcript_id \"blat.proc4005.chain_327\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78626558', '78626779', '100', '+', '.', 'gene_id \"chrX.g35765\"; transcript_id \"blat.proc18128.chain_22419\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56678751', '56678809', '100', '-', '.', 'gene_id \"chrX.g35782\"; transcript_id \"blat.proc18128.chain_18440\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89040766', '89040816', '100', '-', '.', 'gene_id \"chrX.g35783\"; transcript_id \"blat.proc18128.chain_9505\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16387616', '16387660', '100', '+', '.', 'gene_id \"chrX.g35870\"; transcript_id \"blat.proc28144.chain_11586\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52875400', '52875558', '100', '+', '.', 'gene_id \"chrX.g35918\"; transcript_id \"blat.proc18128.chain_9654\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '103591931', '103591976', '100', '+', '.', 'gene_id \"chrX.g35944\"; transcript_id \"blat.proc18128.chain_8641\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60003967', '60004195', '100', '-', '.', 'gene_id \"chrX.g35969\"; transcript_id \"blat.proc28144.chain_18971\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29176310', '29176357', '100', '-', '.', 'gene_id \"chrX.g35986\"; transcript_id \"blat.proc18128.chain_15032\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50553818', '50553997', '100', '+', '.', 'gene_id \"chrX.g35988\"; transcript_id \"blat.proc28144.chain_20974\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54712804', '54712847', '100', '-', '.', 'gene_id \"chrX.g35998\"; transcript_id \"blat.proc28144.chain_16901\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26642998', '26643081', '100', '-', '.', 'gene_id \"chrX.g36026\"; transcript_id \"blat.proc28144.chain_17356\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65874997', '65875043', '100', '+', '.', 'gene_id \"chrX.g36048\"; transcript_id \"blat.proc4005.chain_9070\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97870243', '97870418', '100', '+', '.', 'gene_id \"chrX.g36068\"; transcript_id \"blat.proc28144.chain_5186\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42693971', '42694021', '100', '-', '.', 'gene_id \"chrX.g36077\"; transcript_id \"blat.proc18128.chain_5969\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12389526', '12389566', '100', '-', '.', 'gene_id \"chrX.g36078\"; transcript_id \"blat.proc18128.chain_10923\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32955781', '32955819', '100', '-', '.', 'gene_id \"chrX.g36115\"; transcript_id \"blat.proc28144.chain_9645\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8270062', '8270110', '100', '+', '.', 'gene_id \"chrX.g36120\"; transcript_id \"blat.proc18128.chain_10725\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '62701153', '62701203', '100', '+', '.', 'gene_id \"chrX.g36123\"; transcript_id \"blat.proc28144.chain_2776\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '98481470', '98481508', '100', '-', '.', 'gene_id \"chrX.g36137\"; transcript_id \"blat.proc4005.chain_13857\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57981541', '57981841', '100', '-', '.', 'gene_id \"chrX.g36156\"; transcript_id \"blat.proc28144.chain_4236\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12960832', '12960953', '100', '+', '.', 'gene_id \"chrX.g36204\"; transcript_id \"blat.proc4005.chain_10932\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78518604', '78518647', '100', '+', '.', 'gene_id \"chrX.g36222\"; transcript_id \"blat.proc18128.chain_15541\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52822317', '52822355', '100', '-', '.', 'gene_id \"chrX.g36231\"; transcript_id \"blat.proc28144.chain_21951\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121732566', '121732598', '100', '-', '.', 'gene_id \"chrX.g36298\"; transcript_id \"blat.proc18128.chain_23330\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38430535', '38430580', '100', '-', '.', 'gene_id \"chrX.g36319\"; transcript_id \"blat.proc4005.chain_23418\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13140392', '13140433', '100', '+', '.', 'gene_id \"chrX.g36344\"; transcript_id \"blat.proc18128.chain_6674\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12628331', '12628370', '100', '-', '.', 'gene_id \"chrX.g36363\"; transcript_id \"blat.proc28144.chain_5628\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91271020', '91271296', '100', '+', '.', 'gene_id \"chrX.g36385\"; transcript_id \"blat.proc4587.chain_5018\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12435240', '12435305', '100', '-', '.', 'gene_id \"chrX.g36445\"; transcript_id \"blat.proc4587.chain_10668\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73493516', '73493567', '100', '+', '.', 'gene_id \"chrX.g36468\"; transcript_id \"blat.proc18128.chain_23537\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '95504889', '95504932', '100', '-', '.', 'gene_id \"chrX.g36496\"; transcript_id \"blat.proc4005.chain_21228\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57323956', '57323985', '100', '+', '.', 'gene_id \"chrX.g36528\"; transcript_id \"blat.proc4005.chain_9973\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81366246', '81366288', '100', '+', '.', 'gene_id \"chrX.g36568\"; transcript_id \"blat.proc4005.chain_6968\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74362960', '74362994', '100', '+', '.', 'gene_id \"chrX.g36634\"; transcript_id \"blat.proc4005.chain_13870\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '122118056', '122118274', '100', '-', '.', 'gene_id \"chrX.g36697\"; transcript_id \"blat.proc4587.chain_2766\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45443947', '45443999', '100', '+', '.', 'gene_id \"chrX.g36776\"; transcript_id \"blat.proc28144.chain_4601\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108950617', '108950661', '100', '+', '.', 'gene_id \"chrX.g36777\"; transcript_id \"blat.proc18128.chain_4642\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70743754', '70744048', '100', '+', '.', 'gene_id \"chrX.g36784\"; transcript_id \"blat.proc18128.chain_712\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58231116', '58231253', '100', '-', '.', 'gene_id \"chrX.g36801\"; transcript_id \"blat.proc4587.chain_29\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111415216', '111415250', '100', '-', '.', 'gene_id \"chrX.g36837\"; transcript_id \"blat.proc18128.chain_864\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '23242435', '23242469', '100', '+', '.', 'gene_id \"chrX.g36895\"; transcript_id \"blat.proc18128.chain_18589\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107067873', '107067907', '100', '+', '.', 'gene_id \"chrX.g36976\"; transcript_id \"blat.proc28144.chain_23278\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97704213', '97704246', '100', '-', '.', 'gene_id \"chrX.g37008\"; transcript_id \"blat.proc4587.chain_961\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106285920', '106286192', '100', '+', '.', 'gene_id \"chrX.g37061\"; transcript_id \"blat.proc18128.chain_16483\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65093753', '65093923', '100', '-', '.', 'gene_id \"chrX.g37108\"; transcript_id \"blat.proc28144.chain_11910\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44993453', '44993487', '100', '+', '.', 'gene_id \"chrX.g37113\"; transcript_id \"blat.proc28144.chain_21189\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37674407', '37674446', '100', '-', '.', 'gene_id \"chrX.g37120\"; transcript_id \"blat.proc4005.chain_12099\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84333723', '84333882', '100', '+', '.', 'gene_id \"chrX.g37183\"; transcript_id \"blat.proc28144.chain_10895\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39232381', '39232414', '100', '-', '.', 'gene_id \"chrX.g37258\"; transcript_id \"blat.proc18128.chain_24150\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57431277', '57431469', '100', '+', '.', 'gene_id \"chrX.g37282\"; transcript_id \"blat.proc18128.chain_13626\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '62485254', '62485524', '100', '+', '.', 'gene_id \"chrX.g37302\"; transcript_id \"blat.proc28144.chain_2313\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '66064543', '66064699', '100', '-', '.', 'gene_id \"chrX.g37303\"; transcript_id \"blat.proc4587.chain_10867\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108937025', '108937072', '100', '+', '.', 'gene_id \"chrX.g37309\"; transcript_id \"blat.proc18128.chain_7257\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14638639', '14638688', '100', '-', '.', 'gene_id \"chrX.g37337\"; transcript_id \"blat.proc4587.chain_11161\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109997484', '109997519', '100', '+', '.', 'gene_id \"chrX.g37352\"; transcript_id \"blat.proc28144.chain_18861\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25974592', '25974625', '100', '-', '.', 'gene_id \"chrX.g37394\"; transcript_id \"blat.proc28144.chain_22433\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119279007', '119279040', '100', '+', '.', 'gene_id \"chrX.g37444\"; transcript_id \"blat.proc28144.chain_6485\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58706473', '58706512', '100', '-', '.', 'gene_id \"chrX.g37501\"; transcript_id \"blat.proc28144.chain_11609\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57433540', '57433715', '100', '+', '.', 'gene_id \"chrX.g37502\"; transcript_id \"blat.proc28144.chain_20048\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '78370401', '78370560', '100', '+', '.', 'gene_id \"chrX.g37533\"; transcript_id \"blat.proc18128.chain_6950\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21559979', '21560025', '100', '-', '.', 'gene_id \"chrX.g37547\"; transcript_id \"blat.proc4005.chain_6174\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31643946', '31644009', '100', '-', '.', 'gene_id \"chrX.g37596\"; transcript_id \"blat.proc18128.chain_13308\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121582233', '121582517', '100', '-', '.', 'gene_id \"chrX.g37603\"; transcript_id \"blat.proc4587.chain_13074\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10435775', '10435817', '100', '+', '.', 'gene_id \"chrX.g37622\"; transcript_id \"blat.proc18128.chain_1842\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53330814', '53330962', '100', '-', '.', 'gene_id \"chrX.g37640\"; transcript_id \"blat.proc18128.chain_5589\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118385671', '118385719', '100', '-', '.', 'gene_id \"chrX.g37653\"; transcript_id \"blat.proc18128.chain_12091\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104394477', '104394516', '100', '-', '.', 'gene_id \"chrX.g37740\"; transcript_id \"blat.proc18128.chain_17763\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106322360', '106322627', '100', '+', '.', 'gene_id \"chrX.g37749\"; transcript_id \"blat.proc28144.chain_6959\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '61210378', '61210410', '100', '-', '.', 'gene_id \"chrX.g37764\"; transcript_id \"blat.proc18128.chain_20155\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68727594', '68727634', '100', '+', '.', 'gene_id \"chrX.g37788\"; transcript_id \"blat.proc18128.chain_2837\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18528217', '18528268', '100', '+', '.', 'gene_id \"chrX.g37799\"; transcript_id \"blat.proc4587.chain_2465\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81902513', '81902544', '100', '+', '.', 'gene_id \"chrX.g37829\"; transcript_id \"blat.proc28144.chain_4463\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '20336461', '20336528', '100', '+', '.', 'gene_id \"chrX.g37918\"; transcript_id \"blat.proc4005.chain_2920\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41183472', '41183519', '100', '+', '.', 'gene_id \"chrX.g37921\"; transcript_id \"blat.proc18128.chain_8464\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108723142', '108723186', '100', '-', '.', 'gene_id \"chrX.g37946\"; transcript_id \"blat.proc28144.chain_23726\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92747113', '92747266', '100', '+', '.', 'gene_id \"chrX.g37970\"; transcript_id \"blat.proc28144.chain_21232\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112900580', '112900622', '100', '+', '.', 'gene_id \"chrX.g37980\"; transcript_id \"blat.proc18128.chain_17981\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46669819', '46669861', '100', '+', '.', 'gene_id \"chrX.g38008\"; transcript_id \"blat.proc18128.chain_8951\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35105383', '35105432', '100', '-', '.', 'gene_id \"chrX.g38095\"; transcript_id \"blat.proc28144.chain_22322\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108584133', '108584169', '100', '+', '.', 'gene_id \"chrX.g38103\"; transcript_id \"blat.proc18128.chain_22573\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77467013', '77467225', '100', '+', '.', 'gene_id \"chrX.g38111\"; transcript_id \"blat.proc28144.chain_4539\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60554113', '60554383', '100', '+', '.', 'gene_id \"chrX.g38131\"; transcript_id \"blat.proc28144.chain_2312\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117485218', '117485258', '100', '+', '.', 'gene_id \"chrX.g38143\"; transcript_id \"blat.proc4587.chain_9660\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60231567', '60231610', '100', '+', '.', 'gene_id \"chrX.g38245\"; transcript_id \"blat.proc4005.chain_10413\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10852034', '10852096', '100', '-', '.', 'gene_id \"chrX.g38263\"; transcript_id \"blat.proc4587.chain_9608\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42321425', '42321456', '100', '-', '.', 'gene_id \"chrX.g38288\"; transcript_id \"blat.proc18128.chain_24223\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '50604734', '50604790', '100', '+', '.', 'gene_id \"chrX.g38309\"; transcript_id \"blat.proc4005.chain_21739\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38773377', '38773479', '100', '+', '.', 'gene_id \"chrX.g38315\"; transcript_id \"blat.proc4587.chain_10924\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37540767', '37540831', '100', '-', '.', 'gene_id \"chrX.g38393\"; transcript_id \"blat.proc4587.chain_4091\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109899528', '109899559', '100', '-', '.', 'gene_id \"chrX.g38440\"; transcript_id \"blat.proc4587.chain_4039\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90791089', '90791136', '100', '-', '.', 'gene_id \"chrX.g38449\"; transcript_id \"blat.proc4005.chain_3747\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36668765', '36668803', '100', '-', '.', 'gene_id \"chrX.g38526\"; transcript_id \"blat.proc4587.chain_3492\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25971196', '25971265', '100', '-', '.', 'gene_id \"chrX.g38577\"; transcript_id \"blat.proc4587.chain_2892\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19963612', '19963655', '100', '+', '.', 'gene_id \"chrX.g38579\"; transcript_id \"blat.proc4005.chain_13522\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112581261', '112581293', '100', '+', '.', 'gene_id \"chrX.g38597\"; transcript_id \"blat.proc28144.chain_21056\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10446755', '10446786', '100', '+', '.', 'gene_id \"chrX.g38605\"; transcript_id \"blat.proc28144.chain_22824\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85170845', '85170917', '100', '+', '.', 'gene_id \"chrX.g38713\"; transcript_id \"blat.proc18128.chain_6294\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18643550', '18643610', '100', '+', '.', 'gene_id \"chrX.g38718\"; transcript_id \"blat.proc18128.chain_23605\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96533549', '96533582', '100', '-', '.', 'gene_id \"chrX.g38744\"; transcript_id \"blat.proc4005.chain_2847\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108755177', '108755456', '100', '-', '.', 'gene_id \"chrX.g38768\"; transcript_id \"blat.proc28144.chain_21115\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53705382', '53705652', '100', '+', '.', 'gene_id \"chrX.g38842\"; transcript_id \"blat.proc28144.chain_2310\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '16752913', '16752970', '100', '+', '.', 'gene_id \"chrX.g38920\"; transcript_id \"blat.proc28144.chain_1723\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12370381', '12370413', '100', '+', '.', 'gene_id \"chrX.g38924\"; transcript_id \"blat.proc4587.chain_1163\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '51001751', '51001799', '100', '-', '.', 'gene_id \"chrX.g38973\"; transcript_id \"blat.proc4005.chain_10088\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67829301', '67829478', '100', '+', '.', 'gene_id \"chrX.g39126\"; transcript_id \"blat.proc28144.chain_12993\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '102106979', '102107046', '100', '+', '.', 'gene_id \"chrX.g39155\"; transcript_id \"blat.proc18128.chain_6025\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35438778', '35438810', '100', '+', '.', 'gene_id \"chrX.g39163\"; transcript_id \"blat.proc4005.chain_20895\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7638071', '7638118', '100', '-', '.', 'gene_id \"chrX.g39244\"; transcript_id \"blat.proc18128.chain_24760\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '9625252', '9625300', '100', '-', '.', 'gene_id \"chrX.g39247\"; transcript_id \"blat.proc4005.chain_314\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56250190', '56250301', '100', '+', '.', 'gene_id \"chrX.g39257\"; transcript_id \"blat.proc4005.chain_22699\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49235564', '49235607', '100', '+', '.', 'gene_id \"chrX.g39260\"; transcript_id \"blat.proc18128.chain_3235\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106542224', '106542262', '100', '-', '.', 'gene_id \"chrX.g39271\"; transcript_id \"blat.proc4005.chain_18462\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46183260', '46183344', '100', '+', '.', 'gene_id \"chrX.g39278\"; transcript_id \"blat.proc18128.chain_10141\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93902465', '93902512', '100', '-', '.', 'gene_id \"chrX.g39288\"; transcript_id \"blat.proc18128.chain_3850\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12621806', '12621851', '100', '-', '.', 'gene_id \"chrX.g39302\"; transcript_id \"blat.proc4005.chain_23417\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121380348', '121380409', '100', '+', '.', 'gene_id \"chrX.g39318\"; transcript_id \"blat.proc18128.chain_19751\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '88435380', '88435412', '100', '+', '.', 'gene_id \"chrX.g39332\"; transcript_id \"blat.proc4005.chain_3193\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17557038', '17557080', '100', '-', '.', 'gene_id \"chrX.g39380\"; transcript_id \"blat.proc18128.chain_11030\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104208646', '104208691', '100', '+', '.', 'gene_id \"chrX.g39400\"; transcript_id \"blat.proc18128.chain_14822\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120908297', '120908362', '100', '+', '.', 'gene_id \"chrX.g39405\"; transcript_id \"blat.proc4005.chain_5733\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74444562', '74444608', '100', '-', '.', 'gene_id \"chrX.g39431\"; transcript_id \"blat.proc28144.chain_3557\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69093586', '69094025', '.', '-', '.', 'gene_id \"chr5.g29018\"; transcript_id \"chr5.g29018.i1\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28343961', '28344035', '100', '-', '.', 'gene_id \"chrX.g39446\"; transcript_id \"blat.proc4587.chain_5954\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49727547', '49727579', '100', '-', '.', 'gene_id \"chrX.g39450\"; transcript_id \"blat.proc4587.chain_7654\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101705954', '101706067', '100', '+', '.', 'gene_id \"chrX.g39462\"; transcript_id \"blat.proc28144.chain_21291\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40292409', '40292603', '100', '-', '.', 'gene_id \"chrX.g39481\"; transcript_id \"blat.proc4587.chain_17\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106804037', '106804071', '100', '+', '.', 'gene_id \"chrX.g39511\"; transcript_id \"blat.proc28144.chain_18753\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64070252', '64070438', '100', '+', '.', 'gene_id \"chrX.g39517\"; transcript_id \"blat.proc4005.chain_22029\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109781936', '109781990', '100', '-', '.', 'gene_id \"chrX.g39546\"; transcript_id \"blat.proc4587.chain_2632\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71373587', '71373880', '100', '-', '.', 'gene_id \"chrX.g39592\"; transcript_id \"blat.proc18128.chain_717\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92625460', '92625502', '100', '-', '.', 'gene_id \"chrX.g39622\"; transcript_id \"blat.proc4005.chain_13394\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18831549', '18831583', '100', '-', '.', 'gene_id \"chrX.g39635\"; transcript_id \"blat.proc4005.chain_14578\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82187390', '82187593', '100', '+', '.', 'gene_id \"chrX.g39703\"; transcript_id \"blat.proc28144.chain_3602\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '116871195', '116871256', '100', '-', '.', 'gene_id \"chrX.g39741\"; transcript_id \"blat.proc4587.chain_4465\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '92033851', '92033884', '100', '+', '.', 'gene_id \"chrX.g39752\"; transcript_id \"blat.proc28144.chain_11595\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '62181024', '62181192', '100', '+', '.', 'gene_id \"chrX.g39785\"; transcript_id \"blat.proc28144.chain_15227\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '13488594', '13488633', '100', '+', '.', 'gene_id \"chrX.g39828\"; transcript_id \"blat.proc4005.chain_20750\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82849885', '82850152', '100', '+', '.', 'gene_id \"chrX.g39859\"; transcript_id \"blat.proc28144.chain_6957\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121382464', '121382498', '100', '+', '.', 'gene_id \"chrX.g39890\"; transcript_id \"blat.proc18128.chain_7966\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60923559', '60923610', '100', '+', '.', 'gene_id \"chrX.g39969\"; transcript_id \"blat.proc4587.chain_2467\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '96714176', '96714276', '100', '-', '.', 'gene_id \"chrX.g39986\"; transcript_id \"blat.proc28144.chain_11349\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12536366', '12536407', '100', '+', '.', 'gene_id \"chrX.g39992\"; transcript_id \"blat.proc28144.chain_7484\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58920647', '58922152', '100', '+', '.', 'gene_id \"chrX.g39997\"; transcript_id \"blat.proc4005.chain_23921\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '38335160', '38335245', '100', '+', '.', 'gene_id \"chrX.g40049\"; transcript_id \"blat.proc4587.chain_10617\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54704492', '54704533', '100', '-', '.', 'gene_id \"chrX.g40099\"; transcript_id \"blat.proc4005.chain_1478\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45359962', '45360232', '100', '+', '.', 'gene_id \"chrX.g40211\"; transcript_id \"blat.proc28144.chain_2308\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73928683', '73928855', '100', '+', '.', 'gene_id \"chrX.g40282\"; transcript_id \"blat.proc18128.chain_20627\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '18976349', '18976431', '100', '-', '.', 'gene_id \"chrX.g40307\"; transcript_id \"blat.proc4005.chain_21282\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91303683', '91303958', '100', '-', '.', 'gene_id \"chrX.g40318\"; transcript_id \"blat.proc4587.chain_5020\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14482068', '14482106', '100', '+', '.', 'gene_id \"chrX.g40358\"; transcript_id \"blat.proc28144.chain_21357\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '68285280', '68285324', '100', '+', '.', 'gene_id \"chrX.g40368\"; transcript_id \"blat.proc18128.chain_21695\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19276247', '19276308', '100', '-', '.', 'gene_id \"chrX.g40399\"; transcript_id \"blat.proc18128.chain_24605\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '101788764', '101788807', '100', '-', '.', 'gene_id \"chrX.g40422\"; transcript_id \"blat.proc4005.chain_17954\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44009200', '44009284', '100', '+', '.', 'gene_id \"chrX.g40427\"; transcript_id \"blat.proc18128.chain_10140\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87167860', '87167914', '100', '+', '.', 'gene_id \"chrX.g40443\"; transcript_id \"blat.proc4587.chain_2795\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58794385', '58794559', '100', '+', '.', 'gene_id \"chrX.g40464\"; transcript_id \"blat.proc18128.chain_6206\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83484747', '83484924', '100', '+', '.', 'gene_id \"chrX.g40489\"; transcript_id \"blat.proc28144.chain_12994\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '62109158', '62109332', '100', '+', '.', 'gene_id \"chrX.g40493\"; transcript_id \"blat.proc28144.chain_16401\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73926183', '73926318', '100', '+', '.', 'gene_id \"chrX.g40501\"; transcript_id \"blat.proc18128.chain_1525\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109073647', '109073682', '100', '+', '.', 'gene_id \"chrX.g40513\"; transcript_id \"blat.proc4005.chain_14342\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73771146', '73771202', '100', '-', '.', 'gene_id \"chrX.g40517\"; transcript_id \"blat.proc28144.chain_5346\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37237473', '37237513', '100', '+', '.', 'gene_id \"chrX.g40531\"; transcript_id \"blat.proc28144.chain_23275\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111000777', '111000935', '100', '+', '.', 'gene_id \"chrX.g40575\"; transcript_id \"blat.proc4005.chain_22904\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '83997088', '83997153', '100', '+', '.', 'gene_id \"chrX.g40582\"; transcript_id \"blat.proc4587.chain_2170\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '93084880', '93085055', '100', '-', '.', 'gene_id \"chrX.g40598\"; transcript_id \"blat.proc4005.chain_20932\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44669394', '44669427', '100', '+', '.', 'gene_id \"chrX.g40630\"; transcript_id \"blat.proc18128.chain_10708\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119072124', '119072268', '100', '+', '.', 'gene_id \"chrX.g40715\"; transcript_id \"blat.proc18128.chain_3585\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108319640', '108319712', '100', '+', '.', 'gene_id \"chrX.g40731\"; transcript_id \"blat.proc28144.chain_13645\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '22911526', '22911582', '100', '+', '.', 'gene_id \"chrX.g40764\"; transcript_id \"blat.proc4005.chain_17336\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91744882', '91744953', '100', '+', '.', 'gene_id \"chrX.g40776\"; transcript_id \"blat.proc4005.chain_11291\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81597007', '81597093', '100', '+', '.', 'gene_id \"chrX.g40805\"; transcript_id \"blat.proc28144.chain_6175\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106040261', '106040308', '100', '-', '.', 'gene_id \"chrX.g40814\"; transcript_id \"blat.proc4005.chain_5197\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91420526', '91420557', '100', '+', '.', 'gene_id \"chrX.g40825\"; transcript_id \"blat.proc4005.chain_21841\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12873736', '12873793', '100', '+', '.', 'gene_id \"chrX.g40877\"; transcript_id \"blat.proc4005.chain_19157\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12728104', '12728161', '100', '+', '.', 'gene_id \"chrX.g40914\"; transcript_id \"blat.proc4005.chain_7291\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '116782669', '116782711', '100', '-', '.', 'gene_id \"chrX.g40999\"; transcript_id \"blat.proc28144.chain_3722\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21431312', '21431373', '100', '+', '.', 'gene_id \"chrX.g41016\"; transcript_id \"blat.proc18128.chain_19505\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14229155', '14229185', '100', '-', '.', 'gene_id \"chrX.g41047\"; transcript_id \"blat.proc4587.chain_9301\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48453505', '48453775', '100', '+', '.', 'gene_id \"chrX.g41066\"; transcript_id \"blat.proc28144.chain_2309\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7409156', '7409270', '100', '-', '.', 'gene_id \"chrX.g41145\"; transcript_id \"blat.proc18128.chain_23171\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39266946', '39267004', '100', '-', '.', 'gene_id \"chrX.g41169\"; transcript_id \"blat.proc18128.chain_7043\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17583837', '17583874', '100', '-', '.', 'gene_id \"chrX.g41350\"; transcript_id \"blat.proc4587.chain_11814\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28872988', '28873133', '100', '-', '.', 'gene_id \"chrX.g41388\"; transcript_id \"blat.proc18128.chain_25974\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77366451', '77366498', '100', '+', '.', 'gene_id \"chrX.g41399\"; transcript_id \"blat.proc28144.chain_7871\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79698405', '79698529', '100', '-', '.', 'gene_id \"chrX.g41526\"; transcript_id \"blat.proc28144.chain_726\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67565018', '67565081', '100', '-', '.', 'gene_id \"chrX.g41528\"; transcript_id \"blat.proc4005.chain_6009\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111008453', '111008500', '100', '-', '.', 'gene_id \"chrX.g41572\"; transcript_id \"blat.proc4005.chain_4065\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40255328', '40255512', '100', '+', '.', 'gene_id \"chrX.g41598\"; transcript_id \"blat.proc4587.chain_14\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '70740887', '70741065', '100', '+', '.', 'gene_id \"chrX.g41649\"; transcript_id \"blat.proc18128.chain_420\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '109116016', '109116055', '100', '-', '.', 'gene_id \"chrX.g41654\"; transcript_id \"blat.proc18128.chain_12866\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115094710', '115094771', '100', '+', '.', 'gene_id \"chrX.g41696\"; transcript_id \"blat.proc28144.chain_7232\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '90252093', '90252136', '100', '+', '.', 'gene_id \"chrX.g41743\"; transcript_id \"blat.proc28144.chain_16003\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32132150', '32132207', '100', '-', '.', 'gene_id \"chrX.g41770\"; transcript_id \"blat.proc28144.chain_21183\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106219024', '106219066', '100', '-', '.', 'gene_id \"chrX.g41846\"; transcript_id \"blat.proc4005.chain_4740\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57749218', '57749488', '100', '+', '.', 'gene_id \"chrX.g41849\"; transcript_id \"blat.proc28144.chain_2311\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29571983', '29572155', '100', '+', '.', 'gene_id \"chrX.g41856\"; transcript_id \"blat.proc18128.chain_20626\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8807674', '8807733', '100', '-', '.', 'gene_id \"chrX.g41868\"; transcript_id \"blat.proc4587.chain_12832\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14801142', '14801205', '100', '-', '.', 'gene_id \"chrX.g41872\"; transcript_id \"blat.proc4005.chain_21798\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '29417338', '29417385', '100', '-', '.', 'gene_id \"chrX.g41892\"; transcript_id \"blat.proc28144.chain_603\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17937729', '17937795', '100', '+', '.', 'gene_id \"chrX.g41907\"; transcript_id \"blat.proc18128.chain_6024\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59268619', '59268705', '100', '+', '.', 'gene_id \"chrX.g41913\"; transcript_id \"blat.proc28144.chain_6173\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63626438', '63626492', '100', '+', '.', 'gene_id \"chrX.g41932\"; transcript_id \"blat.proc18128.chain_10280\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73178930', '73178985', '100', '+', '.', 'gene_id \"chrX.g41964\"; transcript_id \"blat.proc18128.chain_6989\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74458284', '74458330', '100', '+', '.', 'gene_id \"chrX.g42056\"; transcript_id \"blat.proc28144.chain_3552\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73663487', '73663721', '100', '+', '.', 'gene_id \"chrX.g42064\"; transcript_id \"blat.proc18128.chain_20718\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40632055', '40632127', '100', '+', '.', 'gene_id \"chrX.g42075\"; transcript_id \"blat.proc18128.chain_7069\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85204995', '85205068', '100', '+', '.', 'gene_id \"chrX.g42098\"; transcript_id \"blat.proc4587.chain_3156\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53519393', '53519492', '100', '+', '.', 'gene_id \"chrX.g42120\"; transcript_id \"blat.proc18128.chain_6583\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32376839', '32376903', '100', '+', '.', 'gene_id \"chrX.g42123\"; transcript_id \"blat.proc28144.chain_10323\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57525796', '57525875', '100', '+', '.', 'gene_id \"chrX.g42140\"; transcript_id \"blat.proc4005.chain_14769\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28649173', '28649323', '100', '+', '.', 'gene_id \"chrX.g42189\"; transcript_id \"blat.proc4587.chain_1894\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59026525', '59026643', '100', '-', '.', 'gene_id \"chrX.g42200\"; transcript_id \"blat.proc4005.chain_1487\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '106012422', '106012492', '100', '-', '.', 'gene_id \"chrX.g42261\"; transcript_id \"blat.proc28144.chain_15016\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '107448134', '107448449', '100', '+', '.', 'gene_id \"chrX.g42271\"; transcript_id \"blat.proc4005.chain_10448\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '73192037', '73192089', '100', '-', '.', 'gene_id \"chrX.g42371\"; transcript_id \"blat.proc18128.chain_9710\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114304379', '114304443', '100', '+', '.', 'gene_id \"chrX.g42386\"; transcript_id \"blat.proc18128.chain_15912\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32117070', '32117115', '100', '-', '.', 'gene_id \"chrX.g42423\"; transcript_id \"blat.proc4587.chain_2560\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '120073961', '120074017', '100', '-', '.', 'gene_id \"chrX.g42460\"; transcript_id \"blat.proc4587.chain_1900\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89891055', '89891098', '100', '+', '.', 'gene_id \"chrX.g42463\"; transcript_id \"blat.proc28144.chain_22327\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99866794', '99866958', '100', '-', '.', 'gene_id \"chrX.g42501\"; transcript_id \"blat.proc28144.chain_9301\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '111692422', '111692516', '100', '-', '.', 'gene_id \"chrX.g42556\"; transcript_id \"blat.proc4005.chain_10079\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '58309971', '58310058', '100', '+', '.', 'gene_id \"chrX.g42592\"; transcript_id \"blat.proc18128.chain_24020\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '6713303', '6713355', '100', '-', '.', 'gene_id \"chrX.g42606\"; transcript_id \"blat.proc4005.chain_13222\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '8312139', '8312173', '100', '+', '.', 'gene_id \"chrX.g42638\"; transcript_id \"blat.proc18128.chain_17893\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45737008', '45737076', '100', '+', '.', 'gene_id \"chrX.g42686\"; transcript_id \"blat.proc4005.chain_18977\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47659573', '47659624', '100', '+', '.', 'gene_id \"chrX.g42746\"; transcript_id \"blat.proc4587.chain_2466\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85742431', '85742559', '100', '+', '.', 'gene_id \"chrX.g42842\"; transcript_id \"blat.proc18128.chain_20078\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54285133', '54285170', '100', '-', '.', 'gene_id \"chrX.g42879\"; transcript_id \"blat.proc4587.chain_13534\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91125605', '91125646', '100', '+', '.', 'gene_id \"chrX.g42939\"; transcript_id \"blat.proc18128.chain_10918\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12550934', '12550990', '100', '-', '.', 'gene_id \"chrX.g43087\"; transcript_id \"blat.proc4005.chain_19386\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '36729798', '36729957', '100', '+', '.', 'gene_id \"chrX.g43154\"; transcript_id \"blat.proc18128.chain_6947\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44216470', '44216514', '100', '-', '.', 'gene_id \"chrX.g43177\"; transcript_id \"blat.proc28144.chain_18241\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '80460353', '80460411', '100', '-', '.', 'gene_id \"chrX.g43206\"; transcript_id \"blat.proc18128.chain_5967\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69999295', '69999328', '100', '-', '.', 'gene_id \"chrX.g43225\"; transcript_id \"blat.proc18128.chain_10523\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42793293', '42793331', '100', '-', '.', 'gene_id \"chrX.g43239\"; transcript_id \"blat.proc18128.chain_8647\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43810813', '43810855', '100', '-', '.', 'gene_id \"chrX.g43246\"; transcript_id \"blat.proc4587.chain_3491\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '104474919', '104474976', '100', '-', '.', 'gene_id \"chrX.g43288\"; transcript_id \"blat.proc4587.chain_2256\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87074030', '87074084', '100', '+', '.', 'gene_id \"chrX.g43292\"; transcript_id \"blat.proc4587.chain_2794\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '72375309', '72375387', '100', '+', '.', 'gene_id \"chrX.g43351\"; transcript_id \"blat.proc18128.chain_7659\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '54208113', '54208295', '100', '+', '.', 'gene_id \"chrX.g43359\"; transcript_id \"blat.proc28144.chain_12776\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '110287516', '110287557', '100', '+', '.', 'gene_id \"chrX.g43405\"; transcript_id \"blat.proc18128.chain_13845\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76875150', '76875182', '100', '+', '.', 'gene_id \"chrX.g43440\"; transcript_id \"blat.proc18128.chain_12931\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37991933', '37992003', '100', '+', '.', 'gene_id \"chrX.g43460\"; transcript_id \"blat.proc18128.chain_3615\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41432428', '41432525', '100', '+', '.', 'gene_id \"chrX.g43480\"; transcript_id \"blat.proc4005.chain_8732\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12680710', '12680767', '100', '-', '.', 'gene_id \"chrX.g43511\"; transcript_id \"blat.proc4587.chain_5525\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81639329', '81639372', '100', '+', '.', 'gene_id \"chrX.g43553\"; transcript_id \"blat.proc28144.chain_6559\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94298967', '94299071', '100', '+', '.', 'gene_id \"chrX.g43558\"; transcript_id \"blat.proc4587.chain_2860\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '53724360', '53724411', '100', '-', '.', 'gene_id \"chrX.g43559\"; transcript_id \"blat.proc28144.chain_10869\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112314266', '112314312', '100', '+', '.', 'gene_id \"chrX.g43583\"; transcript_id \"blat.proc4005.chain_6166\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65414196', '65414366', '100', '-', '.', 'gene_id \"chrX.g43592\"; transcript_id \"blat.proc28144.chain_11911\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65763387', '65763452', '100', '+', '.', 'gene_id \"chrX.g43642\"; transcript_id \"blat.proc4005.chain_19652\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '12096864', '12097358', '100', '-', '.', 'gene_id \"chrX.g43647\"; transcript_id \"blat.proc18128.chain_18681\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '97080197', '97080341', '100', '+', '.', 'gene_id \"chrX.g43682\"; transcript_id \"blat.proc4005.chain_6071\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '28263710', '28263765', '100', '+', '.', 'gene_id \"chrX.g43763\"; transcript_id \"blat.proc18128.chain_8744\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21534541', '21534645', '100', '+', '.', 'gene_id \"chrX.g43819\"; transcript_id \"blat.proc18128.chain_6355\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42634886', '42634933', '100', '+', '.', 'gene_id \"chrX.g43846\"; transcript_id \"blat.proc18128.chain_11720\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '21469445', '21469622', '100', '+', '.', 'gene_id \"chrX.g43849\"; transcript_id \"blat.proc28144.chain_12990\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74554360', '74554738', '100', '-', '.', 'gene_id \"chrX.g43898\"; transcript_id \"blat.proc4005.chain_5863\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115665929', '115665960', '100', '-', '.', 'gene_id \"chrX.g43917\"; transcript_id \"blat.proc4587.chain_10109\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '60803817', '60803853', '100', '+', '.', 'gene_id \"chrX.g43964\"; transcript_id \"blat.proc4005.chain_9006\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44341755', '44341789', '100', '+', '.', 'gene_id \"chrX.g43965\"; transcript_id \"blat.proc4005.chain_10478\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '112940671', '112940762', '100', '-', '.', 'gene_id \"chrX.g44019\"; transcript_id \"blat.proc4587.chain_1629\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '56250702', '56250876', '100', '-', '.', 'gene_id \"chrX.g44020\"; transcript_id \"blat.proc28144.chain_16408\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '4263051', '4263095', '100', '+', '.', 'gene_id \"chrX.g44110\"; transcript_id \"blat.proc18128.chain_25118\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '39879164', '39879203', '100', '-', '.', 'gene_id \"chrX.g44159\"; transcript_id \"blat.proc18128.chain_25686\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '43610413', '43610456', '100', '+', '.', 'gene_id \"chrX.g44172\"; transcript_id \"blat.proc18128.chain_16051\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '67114301', '67114354', '100', '-', '.', 'gene_id \"chrX.g44340\"; transcript_id \"blat.proc18128.chain_22049\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '10917886', '10917929', '100', '+', '.', 'gene_id \"chrX.g44408\"; transcript_id \"blat.proc18128.chain_22574\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86931080', '86931129', '100', '+', '.', 'gene_id \"chrX.g44419\"; transcript_id \"blat.proc4587.chain_13860\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '81216037', '81216196', '100', '-', '.', 'gene_id \"chrX.g44442\"; transcript_id \"blat.proc18128.chain_20232\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100713496', '100713796', '100', '-', '.', 'gene_id \"chrX.g44499\"; transcript_id \"blat.proc28144.chain_4237\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '994313', '994351', '100', '-', '.', 'gene_id \"chrX.g44508\"; transcript_id \"blat.proc18128.chain_9648\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19980133', '19980170', '100', '+', '.', 'gene_id \"chrX.g44555\"; transcript_id \"blat.proc28144.chain_9639\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87310797', '87310903', '100', '+', '.', 'gene_id \"chrX.g44561\"; transcript_id \"blat.proc28144.chain_7289\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '65936663', '65936709', '100', '-', '.', 'gene_id \"chrX.g44589\"; transcript_id \"blat.proc4005.chain_13604\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46885491', '46885521', '100', '-', '.', 'gene_id \"chrX.g44635\"; transcript_id \"blat.proc28144.chain_16662\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '45777044', '45777073', '100', '+', '.', 'gene_id \"chrX.g44684\"; transcript_id \"blat.proc18128.chain_4423\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25928352', '25928392', '100', '-', '.', 'gene_id \"chrX.g44689\"; transcript_id \"blat.proc28144.chain_3647\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '49326347', '49326394', '100', '+', '.', 'gene_id \"chrX.g44752\"; transcript_id \"blat.proc18128.chain_3848\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '74101238', '74101284', '100', '+', '.', 'gene_id \"chrX.g44792\"; transcript_id \"blat.proc18128.chain_6628\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '71619619', '71619716', '100', '+', '.', 'gene_id \"chrX.g44797\"; transcript_id \"blat.proc18128.chain_15660\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89717644', '89717688', '100', '-', '.', 'gene_id \"chrX.g44839\"; transcript_id \"blat.proc18128.chain_10986\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17471274', '17471320', '100', '+', '.', 'gene_id \"chrX.g44842\"; transcript_id \"blat.proc28144.chain_9396\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '103074113', '103074167', '100', '+', '.', 'gene_id \"chrX.g45054\"; transcript_id \"blat.proc4587.chain_2796\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '63371712', '63371836', '100', '-', '.', 'gene_id \"chrX.g45142\"; transcript_id \"blat.proc4587.chain_11691\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14473069', '14473107', '100', '+', '.', 'gene_id \"chrX.g45159\"; transcript_id \"blat.proc18128.chain_7400\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '86313287', '86313451', '100', '-', '.', 'gene_id \"chrX.g45182\"; transcript_id \"blat.proc28144.chain_8933\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37216214', '37216267', '100', '+', '.', 'gene_id \"chrX.g45293\"; transcript_id \"blat.proc18128.chain_8745\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7096903', '7096932', '100', '+', '.', 'gene_id \"chrX.g45294\"; transcript_id \"blat.proc4005.chain_22400\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40256124', '40256318', '100', '+', '.', 'gene_id \"chrX.g45335\"; transcript_id \"blat.proc4587.chain_16\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '41414362', '41414393', '100', '+', '.', 'gene_id \"chrX.g45350\"; transcript_id \"blat.proc18128.chain_642\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7665533', '7665583', '100', '+', '.', 'gene_id \"chrX.g45400\"; transcript_id \"blat.proc18128.chain_7399\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75783087', '75783150', '100', '-', '.', 'gene_id \"chrX.g45497\"; transcript_id \"blat.proc28144.chain_7295\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '94850406', '94850506', '100', '-', '.', 'gene_id \"chrX.g45598\"; transcript_id \"blat.proc4005.chain_15340\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79764976', '79765023', '100', '-', '.', 'gene_id \"chrX.g45638\"; transcript_id \"blat.proc28144.chain_22601\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '64950815', '64950846', '100', '+', '.', 'gene_id \"chrX.g45645\"; transcript_id \"blat.proc4587.chain_10406\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11907252', '11907375', '100', '-', '.', 'gene_id \"chrX.g45666\"; transcript_id \"blat.proc28144.chain_9234\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37696865', '37696940', '100', '-', '.', 'gene_id \"chrX.g45668\"; transcript_id \"blat.proc28144.chain_13380\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117420866', '117420946', '100', '+', '.', 'gene_id \"chrX.g45746\"; transcript_id \"blat.proc18128.chain_5577\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121507473', '121507518', '100', '-', '.', 'gene_id \"chrX.g45866\"; transcript_id \"blat.proc4005.chain_5396\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '42856511', '42856540', '100', '+', '.', 'gene_id \"chrX.g45929\"; transcript_id \"blat.proc28144.chain_15904\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87571773', '87571872', '100', '-', '.', 'gene_id \"chrX.g45993\"; transcript_id \"blat.proc18128.chain_2876\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '76625205', '76625266', '100', '+', '.', 'gene_id \"chrX.g46032\"; transcript_id \"blat.proc18128.chain_15844\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114002123', '114002277', '100', '+', '.', 'gene_id \"chrX.g46035\"; transcript_id \"blat.proc4587.chain_13005\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11693520', '11693790', '100', '+', '.', 'gene_id \"chrX.g46070\"; transcript_id \"blat.proc28144.chain_2306\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '108484694', '108484739', '100', '-', '.', 'gene_id \"chrX.g46115\"; transcript_id \"blat.proc4587.chain_8333\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105930533', '105930577', '100', '+', '.', 'gene_id \"chrX.g46117\"; transcript_id \"blat.proc18128.chain_18984\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '32482069', '32482111', '100', '+', '.', 'gene_id \"chrX.g46157\"; transcript_id \"blat.proc4005.chain_20712\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '31950471', '31950522', '100', '-', '.', 'gene_id \"chrX.g46199\"; transcript_id \"blat.proc4587.chain_9833\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '100968208', '100968255', '100', '-', '.', 'gene_id \"chrX.g46232\"; transcript_id \"blat.proc18128.chain_16702\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '40490646', '40490736', '100', '+', '.', 'gene_id \"chrX.g46241\"; transcript_id \"blat.proc18128.chain_2953\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '79696276', '79696791', '100', '-', '.', 'gene_id \"chrX.g46254\"; transcript_id \"blat.proc4587.chain_13466\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '699572', '699667', '100', '-', '.', 'gene_id \"chrX.g46256\"; transcript_id \"blat.proc4587.chain_2577\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '25096869', '25096898', '100', '-', '.', 'gene_id \"chrX.g46264\"; transcript_id \"blat.proc18128.chain_7075\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '121602846', '121603129', '100', '+', '.', 'gene_id \"chrX.g46336\"; transcript_id \"blat.proc4587.chain_13070\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '44588528', '44588566', '100', '+', '.', 'gene_id \"chrX.g46362\"; transcript_id \"blat.proc4587.chain_12840\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '34005610', '34005640', '100', '-', '.', 'gene_id \"chrX.g46472\"; transcript_id \"blat.proc18128.chain_19830\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47599711', '47599797', '100', '+', '.', 'gene_id \"chrX.g46497\"; transcript_id \"blat.proc28144.chain_6172\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114925770', '114926022', '100', '+', '.', 'gene_id \"chrX.g46622\"; transcript_id \"blat.proc4005.chain_13243\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '52853239', '52853274', '100', '+', '.', 'gene_id \"chrX.g46668\"; transcript_id \"blat.proc4005.chain_11229\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '89307688', '89307731', '100', '+', '.', 'gene_id \"chrX.g46719\"; transcript_id \"blat.proc4005.chain_7125\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '17950935', '17950977', '100', '-', '.', 'gene_id \"chrX.g46747\"; transcript_id \"blat.proc18128.chain_24629\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '117617654', '117617688', '100', '-', '.', 'gene_id \"chrX.g46794\"; transcript_id \"blat.proc18128.chain_8591\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '115403618', '115403667', '100', '-', '.', 'gene_id \"chrX.g46798\"; transcript_id \"blat.proc4005.chain_6974\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '30223037', '30223119', '100', '+', '.', 'gene_id \"chrX.g46815\"; transcript_id \"blat.proc18128.chain_16653\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '37457952', '37457996', '100', '+', '.', 'gene_id \"chrX.g46862\"; transcript_id \"blat.proc18128.chain_22446\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '87619869', '87620412', '100', '-', '.', 'gene_id \"chrX.g46876\"; transcript_id \"blat.proc18128.chain_20475\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '91971303', '91971364', '100', '+', '.', 'gene_id \"chrX.g46931\"; transcript_id \"blat.proc4005.chain_6847\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '47537015', '47537178', '100', '-', '.', 'gene_id \"chrX.g46983\"; transcript_id \"blat.proc4587.chain_11286\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '99757361', '99757400', '100', '-', '.', 'gene_id \"chrX.g46985\"; transcript_id \"blat.proc28144.chain_22330\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '114030840', '114030873', '100', '-', '.', 'gene_id \"chrX.g47006\"; transcript_id \"blat.proc28144.chain_18509\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '75698223', '75698327', '100', '+', '.', 'gene_id \"chrX.g47008\"; transcript_id \"blat.proc4005.chain_9440\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77885170', '77885247', '100', '-', '.', 'gene_id \"chrX.g47026\"; transcript_id \"blat.proc28144.chain_8104\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '85180526', '85180578', '100', '-', '.', 'gene_id \"chrX.g47074\"; transcript_id \"blat.proc18128.chain_10058\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '14914362', '14914404', '100', '+', '.', 'gene_id \"chrX.g47087\"; transcript_id \"blat.proc18128.chain_16050\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '15206632', '15206672', '100', '+', '.', 'gene_id \"chrX.g47106\"; transcript_id \"blat.proc18128.chain_2565\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '3102296', '3102348', '100', '+', '.', 'gene_id \"chrX.g47112\"; transcript_id \"blat.proc28144.chain_4600\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '48859058', '48859273', '100', '-', '.', 'gene_id \"chrX.g47123\"; transcript_id \"blat.proc18128.chain_25628\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '7739672', '7739705', '100', '-', '.', 'gene_id \"chrX.g47130\"; transcript_id \"blat.proc18128.chain_19756\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '82049653', '82049732', '100', '+', '.', 'gene_id \"chrX.g47134\"; transcript_id \"blat.proc28144.chain_19222\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26273110', '26273168', '100', '-', '.', 'gene_id \"chrX.g47185\"; transcript_id \"blat.proc4005.chain_4049\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '46082781', '46082996', '100', '+', '.', 'gene_id \"chrX.g47240\"; transcript_id \"blat.proc18128.chain_25486\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '84450132', '84450299', '100', '-', '.', 'gene_id \"chrX.g47274\"; transcript_id \"blat.proc4005.chain_10097\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '57435853', '57436069', '100', '+', '.', 'gene_id \"chrX.g47313\"; transcript_id \"blat.proc4005.chain_5650\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '77193884', '77193996', '100', '-', '.', 'gene_id \"chrX.g47425\"; transcript_id \"blat.proc28144.chain_21294\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '105161410', '105161441', '100', '+', '.', 'gene_id \"chrX.g47467\"; transcript_id \"blat.proc18128.chain_319\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '19206053', '19206312', '100', '+', '.', 'gene_id \"chrX.g47609\"; transcript_id \"blat.proc18128.chain_21881\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '11289457', '11289558', '100', '+', '.', 'gene_id \"chrX.g47677\"; transcript_id \"blat.proc18128.chain_15824\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '119071275', '119071555', '100', '-', '.', 'gene_id \"chrX.g47701\"; transcript_id \"blat.proc28144.chain_21185\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '59755698', '59755872', '100', '+', '.', 'gene_id \"chrX.g47747\"; transcript_id \"blat.proc18128.chain_24182\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '69871360', '69871395', '100', '+', '.', 'gene_id \"chrX.g47801\"; transcript_id \"blat.proc28144.chain_7588\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '118349986', '118350034', '100', '+', '.', 'gene_id \"chrX.g47822\"; transcript_id \"blat.proc4587.chain_8659\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '35823319', '35823382', '100', '-', '.', 'gene_id \"chrX.g47832\"; transcript_id \"blat.proc28144.chain_968\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '26000836', '26000877', '100', '-', '.', 'gene_id \"chrX.g47852\"; transcript_id \"blat.proc18128.chain_21603\";']\n",
      "negative:  ['chrX', 'PASA', 'exon', '55804951', '55805020', '100', '-', '.', 'gene_id \"chrX.g47863\"; transcript_id \"blat.proc4587.chain_11335\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69112520', '69112936', '.', '+', '.', 'gene_id \"chr5.g29022\"; transcript_id \"chr5.g29022.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69172811', '69173452', '.', '-', '.', 'gene_id \"chr5.g29029\"; transcript_id \"chr5.g29029.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69177695', '69177959', '.', '+', '.', 'gene_id \"chr5.g29031\"; transcript_id \"chr5.g29031.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69177940', '69178201', '.', '-', '.', 'gene_id \"chr5.g29032\"; transcript_id \"chr5.g29032.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69216345', '69216577', '.', '-', '.', 'gene_id \"chr5.g29039\"; transcript_id \"chr5.g29039.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69229017', '69229145', '.', '-', '.', 'gene_id \"chr5.g29043\"; transcript_id \"chr5.g29043.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69349693', '69350141', '.', '-', '.', 'gene_id \"chr5.g29053\"; transcript_id \"chr5.g29053.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69410132', '69410285', '.', '-', '.', 'gene_id \"chr5.g29056\"; transcript_id \"chr5.g29056.i1\";']\n",
      "negative:  ['scaffold_1188', 'PASA', 'exon', '602', '825', '.', '+', '.', 'gene_id \"scaffold_1188.g2\"; transcript_id \"scaffold_1188.g2.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69413593', '69414010', '.', '-', '.', 'gene_id \"chr5.g29059\"; transcript_id \"chr5.g29059.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69464711', '69465153', '.', '-', '.', 'gene_id \"chr5.g29064\"; transcript_id \"chr5.g29064.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69470396', '69470662', '.', '+', '.', 'gene_id \"chr5.g29067\"; transcript_id \"chr5.g29067.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69473544', '69474215', '.', '+', '.', 'gene_id \"chr5.g29069\"; transcript_id \"chr5.g29069.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69477899', '69478205', '.', '+', '.', 'gene_id \"chr5.g29070\"; transcript_id \"chr5.g29070.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69508536', '69508669', '.', '-', '.', 'gene_id \"chr5.g29077\"; transcript_id \"chr5.g29077.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69531765', '69532008', '.', '-', '.', 'gene_id \"chr5.g29082\"; transcript_id \"chr5.g29082.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69631203', '69631554', '.', '+', '.', 'gene_id \"chr5.g29093\"; transcript_id \"chr5.g29093.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69654744', '69655061', '.', '+', '.', 'gene_id \"chr5.g29099\"; transcript_id \"chr5.g29099.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69675016', '69675176', '.', '+', '.', 'gene_id \"chr5.g29103\"; transcript_id \"chr5.g29103.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69706969', '69707382', '.', '-', '.', 'gene_id \"chr5.g29104\"; transcript_id \"chr5.g29104.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69714221', '69714413', '.', '-', '.', 'gene_id \"chr5.g29105\"; transcript_id \"chr5.g29105.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69716702', '69717620', '.', '-', '.', 'gene_id \"chr5.g29108\"; transcript_id \"chr5.g29108.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69731411', '69731749', '.', '-', '.', 'gene_id \"chr5.g29114\"; transcript_id \"chr5.g29114.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69915550', '69915581', '.', '+', '.', 'gene_id \"chr5.g29129\"; transcript_id \"chr5.g29129.i1\";']\n",
      "negative:  ['scaffold_938', 'PASA', 'exon', '5991', '6266', '100', '+', '.', 'gene_id \"scaffold_938.g17\"; transcript_id \"blat.proc4587.chain_5016\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70025550', '70025799', '.', '+', '.', 'gene_id \"chr5.g29144\"; transcript_id \"chr5.g29144.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70041581', '70042609', '.', '-', '.', 'gene_id \"chr5.g29148\"; transcript_id \"chr5.g29148.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70041584', '70041788', '.', '+', '.', 'gene_id \"chr5.g29149\"; transcript_id \"chr5.g29149.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70098787', '70098913', '.', '+', '.', 'gene_id \"chr5.g29154\"; transcript_id \"chr5.g29154.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70112846', '70113022', '.', '+', '.', 'gene_id \"chr5.g29157\"; transcript_id \"chr5.g29157.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70114758', '70114915', '.', '-', '.', 'gene_id \"chr5.g29158\"; transcript_id \"chr5.g29158.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70155299', '70155581', '.', '+', '.', 'gene_id \"chr5.g29162\"; transcript_id \"chr5.g29162.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70155185', '70155357', '.', '+', '.', 'gene_id \"chr5.g29162\"; transcript_id \"chr5.g29162.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70156766', '70157114', '.', '-', '.', 'gene_id \"chr5.g29164\"; transcript_id \"chr5.g29164.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70132430', '70132724', '.', '-', '.', 'gene_id \"chr5.g29172\"; transcript_id \"chr5.g29172.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70136810', '70137227', '.', '+', '.', 'gene_id \"chr5.g29174\"; transcript_id \"chr5.g29174.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70144416', '70144583', '.', '-', '.', 'gene_id \"chr5.g29177\"; transcript_id \"chr5.g29177.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70145439', '70145639', '.', '-', '.', 'gene_id \"chr5.g29179\"; transcript_id \"chr5.g29179.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70148196', '70148542', '.', '+', '.', 'gene_id \"chr5.g29180\"; transcript_id \"chr5.g29180.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70154564', '70154728', '.', '+', '.', 'gene_id \"chr5.g29184\"; transcript_id \"chr5.g29184.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70169266', '70169694', '.', '-', '.', 'gene_id \"chr5.g29186\"; transcript_id \"chr5.g29186.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70172007', '70172929', '.', '-', '.', 'gene_id \"chr5.g29189\"; transcript_id \"chr5.g29189.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70171086', '70171272', '.', '+', '.', 'gene_id \"chr5.g29191\"; transcript_id \"chr5.g29191.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70171263', '70171992', '.', '-', '.', 'gene_id \"chr5.g29192\"; transcript_id \"chr5.g29192.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70174941', '70174995', '.', '-', '.', 'gene_id \"chr5.g29193\"; transcript_id \"chr5.g29193.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70174386', '70174593', '.', '-', '.', 'gene_id \"chr5.g29194\"; transcript_id \"chr5.g29194.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70256264', '70256471', '.', '+', '.', 'gene_id \"chr5.g29197\"; transcript_id \"chr5.g29197.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70178269', '70178814', '.', '-', '.', 'gene_id \"chr5.g29201\"; transcript_id \"chr5.g29201.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70178804', '70179032', '.', '+', '.', 'gene_id \"chr5.g29202\"; transcript_id \"chr5.g29202.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70179805', '70180272', '.', '-', '.', 'gene_id \"chr5.g29205\"; transcript_id \"chr5.g29205.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70181163', '70181386', '.', '+', '.', 'gene_id \"chr5.g29206\"; transcript_id \"chr5.g29206.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70188001', '70188186', '.', '+', '.', 'gene_id \"chr5.g29210\"; transcript_id \"chr5.g29210.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70190338', '70190635', '.', '-', '.', 'gene_id \"chr5.g29211\"; transcript_id \"chr5.g29211.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70204916', '70205075', '.', '+', '.', 'gene_id \"chr5.g29215\"; transcript_id \"chr5.g29215.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70247842', '70248106', '.', '-', '.', 'gene_id \"chr5.g29223\"; transcript_id \"chr5.g29223.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70261082', '70261235', '.', '-', '.', 'gene_id \"chr5.g29225\"; transcript_id \"chr5.g29225.i1\";']\n",
      "negative:  ['scaffold_1636', 'PASA', 'exon', '4619', '4774', '100', '+', '.', 'gene_id \"scaffold_1636.g11\"; transcript_id \"blat.proc4587.chain_6478\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70289373', '70289544', '.', '-', '.', 'gene_id \"chr5.g29233\"; transcript_id \"chr5.g29233.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70302307', '70305599', '.', '-', '.', 'gene_id \"chr5.g29235\"; transcript_id \"chr5.g29235.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70308034', '70308189', '.', '+', '.', 'gene_id \"chr5.g29241\"; transcript_id \"chr5.g29241.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70308182', '70308375', '.', '+', '.', 'gene_id \"chr5.g29242\"; transcript_id \"chr5.g29242.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70311362', '70311749', '.', '+', '.', 'gene_id \"chr5.g29247\"; transcript_id \"chr5.g29247.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70315966', '70316158', '.', '-', '.', 'gene_id \"chr5.g29250\"; transcript_id \"chr5.g29250.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70316362', '70316518', '.', '+', '.', 'gene_id \"chr5.g29251\"; transcript_id \"chr5.g29251.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70319648', '70320304', '.', '-', '.', 'gene_id \"chr5.g29256\"; transcript_id \"chr5.g29256.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70320324', '70320539', '.', '-', '.', 'gene_id \"chr5.g29258\"; transcript_id \"chr5.g29258.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70325749', '70325948', '.', '-', '.', 'gene_id \"chr5.g29261\"; transcript_id \"chr5.g29261.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70341581', '70341871', '.', '+', '.', 'gene_id \"chr5.g29264\"; transcript_id \"chr5.g29264.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70347293', '70347563', '.', '-', '.', 'gene_id \"chr5.g29269\"; transcript_id \"chr5.g29269.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70408755', '70409064', '.', '+', '.', 'gene_id \"chr5.g29279\"; transcript_id \"chr5.g29279.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70465934', '70466085', '.', '-', '.', 'gene_id \"chr5.g29288\"; transcript_id \"chr5.g29288.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70481373', '70481564', '.', '+', '.', 'gene_id \"chr5.g29292\"; transcript_id \"chr5.g29292.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70485019', '70485720', '.', '+', '.', 'gene_id \"chr5.g29292\"; transcript_id \"chr5.g29292.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70483089', '70485011', '.', '+', '.', 'gene_id \"chr5.g29292\"; transcript_id \"chr5.g29292.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70496097', '70496309', '.', '+', '.', 'gene_id \"chr5.g29305\"; transcript_id \"chr5.g29305.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70496294', '70497695', '.', '-', '.', 'gene_id \"chr5.g29306\"; transcript_id \"chr5.g29306.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70502412', '70503333', '.', '-', '.', 'gene_id \"chr5.g29316\"; transcript_id \"chr5.g29316.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '70519162', '70519418', '.', '+', '.', 'gene_id \"chr5.g29329\"; transcript_id \"chr5.g29329.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70521759', '70522060', '.', '+', '.', 'gene_id \"chr5.g29331\"; transcript_id \"chr5.g29331.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70532441', '70532677', '.', '+', '.', 'gene_id \"chr5.g29339\"; transcript_id \"chr5.g29339.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70537927', '70538635', '.', '-', '.', 'gene_id \"chr5.g29341\"; transcript_id \"chr5.g29341.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70540972', '70541344', '.', '-', '.', 'gene_id \"chr5.g29345\"; transcript_id \"chr5.g29345.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70542746', '70543021', '.', '-', '.', 'gene_id \"chr5.g29348\"; transcript_id \"chr5.g29348.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70548079', '70548311', '.', '+', '.', 'gene_id \"chr5.g29353\"; transcript_id \"chr5.g29353.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70548295', '70548463', '.', '-', '.', 'gene_id \"chr5.g29354\"; transcript_id \"chr5.g29354.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70573726', '70573879', '.', '-', '.', 'gene_id \"chr5.g29364\"; transcript_id \"chr5.g29364.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70574822', '70574973', '.', '+', '.', 'gene_id \"chr5.g29365\"; transcript_id \"chr5.g29365.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70575239', '70575519', '.', '-', '.', 'gene_id \"chr5.g29366\"; transcript_id \"chr5.g29366.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70578160', '70578378', '.', '+', '.', 'gene_id \"chr5.g29367\"; transcript_id \"chr5.g29367.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70589785', '70590199', '.', '+', '.', 'gene_id \"chr5.g29371\"; transcript_id \"chr5.g29371.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70602946', '70603178', '.', '-', '.', 'gene_id \"chr5.g29372\"; transcript_id \"chr5.g29372.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70608017', '70608194', '.', '+', '.', 'gene_id \"chr5.g29373\"; transcript_id \"chr5.g29373.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70619975', '70620245', '.', '+', '.', 'gene_id \"chr5.g29376\"; transcript_id \"chr5.g29376.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70626127', '70626418', '.', '-', '.', 'gene_id \"chr5.g29379\"; transcript_id \"chr5.g29379.i1\";']\n",
      "negative:  ['scaffold_1149', 'PASA', 'exon', '8849', '9019', '100', '+', '.', 'gene_id \"scaffold_1149.g2\"; transcript_id \"blat.proc28144.chain_20065\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70635635', '70635804', '.', '+', '.', 'gene_id \"chr5.g29385\"; transcript_id \"chr5.g29385.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70635959', '70636122', '.', '+', '.', 'gene_id \"chr5.g29386\"; transcript_id \"chr5.g29386.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70636479', '70636810', '.', '+', '.', 'gene_id \"chr5.g29388\"; transcript_id \"chr5.g29388.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70637034', '70637198', '.', '+', '.', 'gene_id \"chr5.g29389\"; transcript_id \"chr5.g29389.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70639619', '70639785', '.', '-', '.', 'gene_id \"chr5.g29391\"; transcript_id \"chr5.g29391.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70646357', '70646516', '.', '+', '.', 'gene_id \"chr5.g29395\"; transcript_id \"chr5.g29395.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70647646', '70648077', '.', '-', '.', 'gene_id \"chr5.g29397\"; transcript_id \"chr5.g29397.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70656066', '70656236', '.', '-', '.', 'gene_id \"chr5.g29401\"; transcript_id \"chr5.g29401.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70658525', '70659116', '.', '-', '.', 'gene_id \"chr5.g29404\"; transcript_id \"chr5.g29404.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70660536', '70660701', '.', '+', '.', 'gene_id \"chr5.g29407\"; transcript_id \"chr5.g29407.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70670261', '70670433', '.', '-', '.', 'gene_id \"chr5.g29415\"; transcript_id \"chr5.g29415.i1\";']\n",
      "negative:  ['scaffold_1647', 'PASA', 'exon', '6903', '7175', '.', '-', '.', 'gene_id \"scaffold_1647.g2\"; transcript_id \"scaffold_1647.g2.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70670772', '70671745', '.', '-', '.', 'gene_id \"chr5.g29417\"; transcript_id \"chr5.g29417.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70672427', '70672649', '.', '-', '.', 'gene_id \"chr5.g29420\"; transcript_id \"chr5.g29420.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70673448', '70673733', '.', '-', '.', 'gene_id \"chr5.g29422\"; transcript_id \"chr5.g29422.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70673802', '70674062', '.', '-', '.', 'gene_id \"chr5.g29423\"; transcript_id \"chr5.g29423.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70683837', '70684172', '.', '-', '.', 'gene_id \"chr5.g29437\"; transcript_id \"chr5.g29437.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70687828', '70688061', '.', '-', '.', 'gene_id \"chr5.g29438\"; transcript_id \"chr5.g29438.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70687332', '70687850', '.', '-', '.', 'gene_id \"chr5.g29440\"; transcript_id \"chr5.g29440.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70691073', '70691275', '.', '-', '.', 'gene_id \"chr5.g29442\"; transcript_id \"chr5.g29442.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70702612', '70702767', '.', '+', '.', 'gene_id \"chr5.g29445\"; transcript_id \"chr5.g29445.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70703378', '70703543', '.', '-', '.', 'gene_id \"chr5.g29447\"; transcript_id \"chr5.g29447.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70704330', '70704548', '.', '-', '.', 'gene_id \"chr5.g29448\"; transcript_id \"chr5.g29448.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70709342', '70709563', '.', '+', '.', 'gene_id \"chr5.g29455\"; transcript_id \"chr5.g29455.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70709879', '70710035', '.', '-', '.', 'gene_id \"chr5.g29456\"; transcript_id \"chr5.g29456.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70710242', '70710592', '.', '-', '.', 'gene_id \"chr5.g29457\"; transcript_id \"chr5.g29457.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70714297', '70714596', '.', '+', '.', 'gene_id \"chr5.g29462\"; transcript_id \"chr5.g29462.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70722410', '70722648', '.', '-', '.', 'gene_id \"chr5.g29470\"; transcript_id \"chr5.g29470.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70723274', '70723483', '.', '+', '.', 'gene_id \"chr5.g29472\"; transcript_id \"chr5.g29472.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70728540', '70728578', '.', '-', '.', 'gene_id \"chr5.g29477\"; transcript_id \"chr5.g29477.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70731111', '70731273', '.', '-', '.', 'gene_id \"chr5.g29479\"; transcript_id \"chr5.g29479.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70826608', '70826952', '.', '+', '.', 'gene_id \"chr5.g29485\"; transcript_id \"chr5.g29485.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70766106', '70774215', '.', '-', '.', 'gene_id \"chr5.g29486\"; transcript_id \"chr5.g29486.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70844787', '70845395', '.', '-', '.', 'gene_id \"chr5.g29489\"; transcript_id \"chr5.g29489.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70833647', '70834206', '.', '-', '.', 'gene_id \"chr5.g29490\"; transcript_id \"chr5.g29490.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70782815', '70783136', '.', '+', '.', 'gene_id \"chr5.g29499\"; transcript_id \"chr5.g29499.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70784620', '70784808', '.', '-', '.', 'gene_id \"chr5.g29502\"; transcript_id \"chr5.g29502.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70786424', '70786662', '.', '-', '.', 'gene_id \"chr5.g29503\"; transcript_id \"chr5.g29503.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70788854', '70789095', '.', '-', '.', 'gene_id \"chr5.g29507\"; transcript_id \"chr5.g29507.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70803405', '70803606', '.', '+', '.', 'gene_id \"chr5.g29511\"; transcript_id \"chr5.g29511.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70807197', '70807373', '.', '+', '.', 'gene_id \"chr5.g29512\"; transcript_id \"chr5.g29512.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70808704', '70809142', '.', '-', '.', 'gene_id \"chr5.g29513\"; transcript_id \"chr5.g29513.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70810156', '70810349', '.', '-', '.', 'gene_id \"chr5.g29514\"; transcript_id \"chr5.g29514.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70813833', '70814303', '.', '-', '.', 'gene_id \"chr5.g29516\"; transcript_id \"chr5.g29516.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70813853', '70814126', '.', '-', '.', 'gene_id \"chr5.g29516\"; transcript_id \"chr5.g29516.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70815662', '70815839', '.', '+', '.', 'gene_id \"chr5.g29517\"; transcript_id \"chr5.g29517.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70818069', '70818281', '.', '-', '.', 'gene_id \"chr5.g29518\"; transcript_id \"chr5.g29518.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70819711', '70819883', '.', '+', '.', 'gene_id \"chr5.g29519\"; transcript_id \"chr5.g29519.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70823278', '70823437', '.', '+', '.', 'gene_id \"chr5.g29520\"; transcript_id \"chr5.g29520.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70825077', '70826235', '.', '+', '.', 'gene_id \"chr5.g29522\"; transcript_id \"chr5.g29522.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70829382', '70830248', '.', '+', '.', 'gene_id \"chr5.g29524\"; transcript_id \"chr5.g29524.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70836918', '70837084', '.', '-', '.', 'gene_id \"chr5.g29530\"; transcript_id \"chr5.g29530.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70837126', '70837279', '.', '+', '.', 'gene_id \"chr5.g29531\"; transcript_id \"chr5.g29531.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70850132', '70850312', '.', '-', '.', 'gene_id \"chr5.g29539\"; transcript_id \"chr5.g29539.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70863905', '70864016', '.', '+', '.', 'gene_id \"chr5.g29543\"; transcript_id \"chr5.g29543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70874168', '70874520', '.', '-', '.', 'gene_id \"chr5.g29550\"; transcript_id \"chr5.g29550.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70875089', '70875380', '.', '-', '.', 'gene_id \"chr5.g29551\"; transcript_id \"chr5.g29551.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70879431', '70879762', '.', '+', '.', 'gene_id \"chr5.g29553\"; transcript_id \"chr5.g29553.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70880870', '70881013', '.', '+', '.', 'gene_id \"chr5.g29556\"; transcript_id \"chr5.g29556.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70947151', '70947286', '.', '-', '.', 'gene_id \"chr5.g29564\"; transcript_id \"chr5.g29564.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71004652', '71005104', '.', '-', '.', 'gene_id \"chr5.g29569\"; transcript_id \"chr5.g29569.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70997314', '70997553', '.', '-', '.', 'gene_id \"chr5.g29573\"; transcript_id \"chr5.g29573.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71000251', '71001327', '.', '-', '.', 'gene_id \"chr5.g29582\"; transcript_id \"chr5.g29582.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71001785', '71002214', '.', '-', '.', 'gene_id \"chr5.g29585\"; transcript_id \"chr5.g29585.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71011120', '71011542', '.', '+', '.', 'gene_id \"chr5.g29596\"; transcript_id \"chr5.g29596.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71011615', '71011801', '.', '-', '.', 'gene_id \"chr5.g29598\"; transcript_id \"chr5.g29598.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71017830', '71018118', '.', '-', '.', 'gene_id \"chr5.g29602\"; transcript_id \"chr5.g29602.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71018544', '71018809', '.', '+', '.', 'gene_id \"chr5.g29604\"; transcript_id \"chr5.g29604.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71023147', '71023517', '.', '+', '.', 'gene_id \"chr5.g29605\"; transcript_id \"chr5.g29605.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71038836', '71040036', '.', '+', '.', 'gene_id \"chr5.g29613\"; transcript_id \"chr5.g29613.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71040687', '71041228', '.', '-', '.', 'gene_id \"chr5.g29615\"; transcript_id \"chr5.g29615.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71040713', '71041228', '.', '-', '.', 'gene_id \"chr5.g29615\"; transcript_id \"chr5.g29615.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71074430', '71075176', '.', '+', '.', 'gene_id \"chr5.g29619\"; transcript_id \"chr5.g29619.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71069041', '71069435', '.', '+', '.', 'gene_id \"chr5.g29623\"; transcript_id \"chr5.g29623.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71075419', '71075600', '.', '-', '.', 'gene_id \"chr5.g29625\"; transcript_id \"chr5.g29625.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71060784', '71061216', '.', '+', '.', 'gene_id \"chr5.g29627\"; transcript_id \"chr5.g29627.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71061681', '71061974', '.', '-', '.', 'gene_id \"chr5.g29629\"; transcript_id \"chr5.g29629.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71070709', '71073767', '.', '+', '.', 'gene_id \"chr5.g29632\"; transcript_id \"chr5.g29632.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71082251', '71082800', '.', '+', '.', 'gene_id \"chr5.g29635\"; transcript_id \"chr5.g29635.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71082801', '71083033', '.', '-', '.', 'gene_id \"chr5.g29636\"; transcript_id \"chr5.g29636.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71127292', '71127484', '.', '-', '.', 'gene_id \"chr5.g29640\"; transcript_id \"chr5.g29640.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71139522', '71139759', '.', '-', '.', 'gene_id \"chr5.g29645\"; transcript_id \"chr5.g29645.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71146116', '71146285', '.', '-', '.', 'gene_id \"chr5.g29647\"; transcript_id \"chr5.g29647.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71222711', '71222947', '.', '+', '.', 'gene_id \"chr5.g29657\"; transcript_id \"chr5.g29657.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71356667', '71356892', '.', '-', '.', 'gene_id \"chr5.g29666\"; transcript_id \"chr5.g29666.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71357161', '71357363', '.', '-', '.', 'gene_id \"chr5.g29667\"; transcript_id \"chr5.g29667.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71358213', '71358579', '.', '+', '.', 'gene_id \"chr5.g29670\"; transcript_id \"chr5.g29670.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71358678', '71358934', '.', '+', '.', 'gene_id \"chr5.g29671\"; transcript_id \"chr5.g29671.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71359889', '71360081', '.', '-', '.', 'gene_id \"chr5.g29672\"; transcript_id \"chr5.g29672.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71368265', '71368988', '.', '+', '.', 'gene_id \"chr5.g29674\"; transcript_id \"chr5.g29674.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71369874', '71370051', '.', '+', '.', 'gene_id \"chr5.g29676\"; transcript_id \"chr5.g29676.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71423637', '71424006', '.', '-', '.', 'gene_id \"chr5.g29685\"; transcript_id \"chr5.g29685.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71446140', '71446300', '.', '+', '.', 'gene_id \"chr5.g29692\"; transcript_id \"chr5.g29692.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71451094', '71451265', '.', '-', '.', 'gene_id \"chr5.g29694\"; transcript_id \"chr5.g29694.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71528782', '71528976', '.', '-', '.', 'gene_id \"chr5.g29698\"; transcript_id \"chr5.g29698.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71556998', '71557168', '.', '+', '.', 'gene_id \"chr5.g29700\"; transcript_id \"chr5.g29700.i1\";']\n",
      "negative:  ['scaffold_540', 'PASA', 'exon', '225', '502', '.', '+', '.', 'gene_id \"scaffold_540.g2\"; transcript_id \"scaffold_540.g2.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71740863', '71741098', '.', '+', '.', 'gene_id \"chr5.g29720\"; transcript_id \"chr5.g29720.i1\";']\n",
      "negative:  ['scaffold_540', 'PASA', 'exon', '642', '814', '100', '-', '.', 'gene_id \"scaffold_540.g8\"; transcript_id \"blat.proc4005.chain_2077\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71848821', '71848878', '.', '+', '.', 'gene_id \"chr5.g29725\"; transcript_id \"chr5.g29725.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71861481', '71861773', '.', '+', '.', 'gene_id \"chr5.g29727\"; transcript_id \"chr5.g29727.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71864083', '71864333', '.', '+', '.', 'gene_id \"chr5.g29730\"; transcript_id \"chr5.g29730.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71917772', '71918128', '.', '+', '.', 'gene_id \"chr5.g29731\"; transcript_id \"chr5.g29731.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71918725', '71918963', '.', '+', '.', 'gene_id \"chr5.g29732\"; transcript_id \"chr5.g29732.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71918981', '71919133', '.', '+', '.', 'gene_id \"chr5.g29733\"; transcript_id \"chr5.g29733.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71969015', '71969314', '.', '-', '.', 'gene_id \"chr5.g29739\"; transcript_id \"chr5.g29739.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71981350', '71981435', '.', '-', '.', 'gene_id \"chr5.g29743\"; transcript_id \"chr5.g29743.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72032508', '72032770', '.', '+', '.', 'gene_id \"chr5.g29745\"; transcript_id \"chr5.g29745.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72032887', '72033079', '.', '+', '.', 'gene_id \"chr5.g29746\"; transcript_id \"chr5.g29746.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72033338', '72033541', '.', '-', '.', 'gene_id \"chr5.g29748\"; transcript_id \"chr5.g29748.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72067908', '72068080', '.', '-', '.', 'gene_id \"chr5.g29753\"; transcript_id \"chr5.g29753.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72067477', '72067711', '.', '+', '.', 'gene_id \"chr5.g29754\"; transcript_id \"chr5.g29754.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72068280', '72068457', '.', '-', '.', 'gene_id \"chr5.g29756\"; transcript_id \"chr5.g29756.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72072020', '72072189', '.', '-', '.', 'gene_id \"chr5.g29759\"; transcript_id \"chr5.g29759.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72082664', '72082851', '.', '+', '.', 'gene_id \"chr5.g29765\"; transcript_id \"chr5.g29765.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72087645', '72087686', '.', '+', '.', 'gene_id \"chr5.g29770\"; transcript_id \"chr5.g29770.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72089167', '72089324', '.', '+', '.', 'gene_id \"chr5.g29771\"; transcript_id \"chr5.g29771.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72111112', '72111215', '.', '-', '.', 'gene_id \"chr5.g29772\"; transcript_id \"chr5.g29772.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72111121', '72111215', '.', '-', '.', 'gene_id \"chr5.g29772\"; transcript_id \"chr5.g29772.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72115497', '72115545', '.', '-', '.', 'gene_id \"chr5.g29775\"; transcript_id \"chr5.g29775.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72156926', '72157080', '.', '-', '.', 'gene_id \"chr5.g29779\"; transcript_id \"chr5.g29779.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72209139', '72209438', '.', '-', '.', 'gene_id \"chr5.g29785\"; transcript_id \"chr5.g29785.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72205275', '72205494', '.', '+', '.', 'gene_id \"chr5.g29793\"; transcript_id \"chr5.g29793.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72206963', '72207889', '.', '-', '.', 'gene_id \"chr5.g29794\"; transcript_id \"chr5.g29794.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72206963', '72207871', '.', '-', '.', 'gene_id \"chr5.g29794\"; transcript_id \"chr5.g29794.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72213416', '72213591', '.', '+', '.', 'gene_id \"chr5.g29797\"; transcript_id \"chr5.g29797.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72213503', '72213591', '.', '+', '.', 'gene_id \"chr5.g29797\"; transcript_id \"chr5.g29797.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72212922', '72213001', '.', '+', '.', 'gene_id \"chr5.g29797\"; transcript_id \"chr5.g29797.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72212922', '72212998', '.', '+', '.', 'gene_id \"chr5.g29797\"; transcript_id \"chr5.g29797.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72225290', '72225348', '.', '-', '.', 'gene_id \"chr5.g29803\"; transcript_id \"chr5.g29803.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72214511', '72215880', '.', '-', '.', 'gene_id \"chr5.g29805\"; transcript_id \"chr5.g29805.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72219801', '72220036', '.', '+', '.', 'gene_id \"chr5.g29810\"; transcript_id \"chr5.g29810.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72224119', '72224609', '.', '+', '.', 'gene_id \"chr5.g29814\"; transcript_id \"chr5.g29814.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72234924', '72235184', '.', '+', '.', 'gene_id \"chr5.g29820\"; transcript_id \"chr5.g29820.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72235221', '72235613', '.', '+', '.', 'gene_id \"chr5.g29821\"; transcript_id \"chr5.g29821.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72237438', '72237757', '.', '-', '.', 'gene_id \"chr5.g29823\"; transcript_id \"chr5.g29823.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72237819', '72237992', '.', '+', '.', 'gene_id \"chr5.g29824\"; transcript_id \"chr5.g29824.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72238564', '72238793', '.', '-', '.', 'gene_id \"chr5.g29825\"; transcript_id \"chr5.g29825.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72242134', '72242292', '.', '-', '.', 'gene_id \"chr5.g29827\"; transcript_id \"chr5.g29827.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72248100', '72248323', '.', '-', '.', 'gene_id \"chr5.g29828\"; transcript_id \"chr5.g29828.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72257527', '72257772', '.', '-', '.', 'gene_id \"chr5.g29833\"; transcript_id \"chr5.g29833.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72267451', '72267720', '.', '+', '.', 'gene_id \"chr5.g29836\"; transcript_id \"chr5.g29836.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72268974', '72269238', '.', '-', '.', 'gene_id \"chr5.g29837\"; transcript_id \"chr5.g29837.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72275010', '72275329', '.', '-', '.', 'gene_id \"chr5.g29843\"; transcript_id \"chr5.g29843.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72278423', '72278627', '.', '-', '.', 'gene_id \"chr5.g29844\"; transcript_id \"chr5.g29844.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72288473', '72288634', '.', '-', '.', 'gene_id \"chr5.g29846\"; transcript_id \"chr5.g29846.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72289214', '72289425', '.', '+', '.', 'gene_id \"chr5.g29847\"; transcript_id \"chr5.g29847.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72310069', '72310250', '.', '-', '.', 'gene_id \"chr5.g29851\"; transcript_id \"chr5.g29851.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72315834', '72316005', '.', '+', '.', 'gene_id \"chr5.g29858\"; transcript_id \"chr5.g29858.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72323539', '72323700', '.', '+', '.', 'gene_id \"chr5.g29861\"; transcript_id \"chr5.g29861.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72326177', '72326386', '.', '-', '.', 'gene_id \"chr5.g29865\"; transcript_id \"chr5.g29865.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72328534', '72328982', '.', '+', '.', 'gene_id \"chr5.g29868\"; transcript_id \"chr5.g29868.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72330277', '72330960', '.', '-', '.', 'gene_id \"chr5.g29871\"; transcript_id \"chr5.g29871.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72330277', '72330960', '.', '-', '.', 'gene_id \"chr5.g29871\"; transcript_id \"chr5.g29871.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72333754', '72334347', '.', '+', '.', 'gene_id \"chr5.g29877\"; transcript_id \"chr5.g29877.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72334394', '72334733', '.', '-', '.', 'gene_id \"chr5.g29878\"; transcript_id \"chr5.g29878.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72336125', '72336371', '.', '+', '.', 'gene_id \"chr5.g29880\"; transcript_id \"chr5.g29880.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72336557', '72337775', '.', '+', '.', 'gene_id \"chr5.g29881\"; transcript_id \"chr5.g29881.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72340631', '72341376', '.', '-', '.', 'gene_id \"chr5.g29885\"; transcript_id \"chr5.g29885.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72343224', '72343645', '.', '-', '.', 'gene_id \"chr5.g29888\"; transcript_id \"chr5.g29888.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72348366', '72348528', '.', '+', '.', 'gene_id \"chr5.g29889\"; transcript_id \"chr5.g29889.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72351882', '72352158', '.', '-', '.', 'gene_id \"chr5.g29892\"; transcript_id \"chr5.g29892.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72355316', '72355491', '.', '+', '.', 'gene_id \"chr5.g29894\"; transcript_id \"chr5.g29894.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72356717', '72356940', '.', '-', '.', 'gene_id \"chr5.g29897\"; transcript_id \"chr5.g29897.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72358327', '72358655', '.', '-', '.', 'gene_id \"chr5.g29899\"; transcript_id \"chr5.g29899.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72411879', '72412117', '.', '+', '.', 'gene_id \"chr5.g29902\"; transcript_id \"chr5.g29902.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72426108', '72426348', '.', '+', '.', 'gene_id \"chr5.g29905\"; transcript_id \"chr5.g29905.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72473277', '72473538', '.', '-', '.', 'gene_id \"chr5.g29908\"; transcript_id \"chr5.g29908.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72525513', '72525696', '.', '-', '.', 'gene_id \"chr5.g29913\"; transcript_id \"chr5.g29913.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72532040', '72532198', '.', '+', '.', 'gene_id \"chr5.g29914\"; transcript_id \"chr5.g29914.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72535380', '72535569', '.', '+', '.', 'gene_id \"chr5.g29918\"; transcript_id \"chr5.g29918.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72535610', '72535837', '.', '-', '.', 'gene_id \"chr5.g29919\"; transcript_id \"chr5.g29919.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72544740', '72546063', '.', '-', '.', 'gene_id \"chr5.g29924\"; transcript_id \"chr5.g29924.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72556063', '72556219', '.', '-', '.', 'gene_id \"chr5.g29927\"; transcript_id \"chr5.g29927.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72580557', '72580821', '.', '-', '.', 'gene_id \"chr5.g29929\"; transcript_id \"chr5.g29929.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72633725', '72633962', '.', '+', '.', 'gene_id \"chr5.g29934\"; transcript_id \"chr5.g29934.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72635841', '72636000', '.', '-', '.', 'gene_id \"chr5.g29936\"; transcript_id \"chr5.g29936.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72636909', '72637169', '.', '+', '.', 'gene_id \"chr5.g29937\"; transcript_id \"chr5.g29937.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72645743', '72646040', '.', '-', '.', 'gene_id \"chr5.g29938\"; transcript_id \"chr5.g29938.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72672139', '72672299', '.', '+', '.', 'gene_id \"chr5.g29942\"; transcript_id \"chr5.g29942.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72689489', '72689815', '.', '-', '.', 'gene_id \"chr5.g29945\"; transcript_id \"chr5.g29945.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72725967', '72726379', '.', '-', '.', 'gene_id \"chr5.g29954\"; transcript_id \"chr5.g29954.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72734694', '72735119', '.', '+', '.', 'gene_id \"chr5.g29955\"; transcript_id \"chr5.g29955.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72737923', '72738962', '.', '-', '.', 'gene_id \"chr5.g29958\"; transcript_id \"chr5.g29958.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72763999', '72764034', '.', '-', '.', 'gene_id \"chr5.g29964\"; transcript_id \"chr5.g29964.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72792530', '72792707', '.', '-', '.', 'gene_id \"chr5.g29965\"; transcript_id \"chr5.g29965.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72803538', '72803701', '.', '-', '.', 'gene_id \"chr5.g29971\"; transcript_id \"chr5.g29971.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72812047', '72812448', '.', '-', '.', 'gene_id \"chr5.g29977\"; transcript_id \"chr5.g29977.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72812076', '72812448', '.', '-', '.', 'gene_id \"chr5.g29977\"; transcript_id \"chr5.g29977.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72813797', '72813982', '.', '+', '.', 'gene_id \"chr5.g29980\"; transcript_id \"chr5.g29980.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72826061', '72826406', '.', '+', '.', 'gene_id \"chr5.g29985\"; transcript_id \"chr5.g29985.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72842490', '72842645', '.', '+', '.', 'gene_id \"chr5.g29989\"; transcript_id \"chr5.g29989.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72842221', '72842497', '.', '-', '.', 'gene_id \"chr5.g29990\"; transcript_id \"chr5.g29990.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72842491', '72843432', '.', '-', '.', 'gene_id \"chr5.g29991\"; transcript_id \"chr5.g29991.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72845941', '72846122', '.', '+', '.', 'gene_id \"chr5.g29992\"; transcript_id \"chr5.g29992.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72850295', '72850557', '.', '-', '.', 'gene_id \"chr5.g29993\"; transcript_id \"chr5.g29993.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72852714', '72852950', '.', '+', '.', 'gene_id \"chr5.g29994\"; transcript_id \"chr5.g29994.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72859699', '72859855', '.', '+', '.', 'gene_id \"chr5.g29998\"; transcript_id \"chr5.g29998.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72864117', '72864515', '.', '+', '.', 'gene_id \"chr5.g30007\"; transcript_id \"chr5.g30007.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72892935', '72893123', '.', '-', '.', 'gene_id \"chr5.g30013\"; transcript_id \"chr5.g30013.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72893289', '72893449', '.', '+', '.', 'gene_id \"chr5.g30014\"; transcript_id \"chr5.g30014.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72900056', '72900261', '.', '+', '.', 'gene_id \"chr5.g30017\"; transcript_id \"chr5.g30017.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72908825', '72909473', '.', '-', '.', 'gene_id \"chr5.g30023\"; transcript_id \"chr5.g30023.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72912003', '72912235', '.', '-', '.', 'gene_id \"chr5.g30024\"; transcript_id \"chr5.g30024.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72919990', '72920236', '.', '+', '.', 'gene_id \"chr5.g30025\"; transcript_id \"chr5.g30025.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72920598', '72920873', '.', '+', '.', 'gene_id \"chr5.g30026\"; transcript_id \"chr5.g30026.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72921319', '72921547', '.', '-', '.', 'gene_id \"chr5.g30027\"; transcript_id \"chr5.g30027.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72923440', '72923779', '.', '+', '.', 'gene_id \"chr5.g30030\"; transcript_id \"chr5.g30030.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72925907', '72926062', '.', '-', '.', 'gene_id \"chr5.g30033\"; transcript_id \"chr5.g30033.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72931674', '72931840', '.', '-', '.', 'gene_id \"chr5.g30034\"; transcript_id \"chr5.g30034.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72940344', '72941215', '.', '-', '.', 'gene_id \"chr5.g30037\"; transcript_id \"chr5.g30037.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72941709', '72941975', '.', '+', '.', 'gene_id \"chr5.g30040\"; transcript_id \"chr5.g30040.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72943228', '72943560', '.', '-', '.', 'gene_id \"chr5.g30043\"; transcript_id \"chr5.g30043.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72943564', '72943746', '.', '-', '.', 'gene_id \"chr5.g30044\"; transcript_id \"chr5.g30044.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72944642', '72945056', '.', '-', '.', 'gene_id \"chr5.g30047\"; transcript_id \"chr5.g30047.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72968900', '72970003', '.', '-', '.', 'gene_id \"chr5.g30051\"; transcript_id \"chr5.g30051.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72979886', '72980278', '.', '-', '.', 'gene_id \"chr5.g30056\"; transcript_id \"chr5.g30056.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72998856', '72999234', '.', '-', '.', 'gene_id \"chr5.g30065\"; transcript_id \"chr5.g30065.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73003585', '73003737', '.', '+', '.', 'gene_id \"chr5.g30067\"; transcript_id \"chr5.g30067.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73005729', '73006286', '.', '+', '.', 'gene_id \"chr5.g30070\"; transcript_id \"chr5.g30070.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73008857', '73009358', '.', '-', '.', 'gene_id \"chr5.g30071\"; transcript_id \"chr5.g30071.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73009392', '73009552', '.', '-', '.', 'gene_id \"chr5.g30073\"; transcript_id \"chr5.g30073.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73009728', '73010029', '.', '-', '.', 'gene_id \"chr5.g30074\"; transcript_id \"chr5.g30074.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73010693', '73010873', '.', '-', '.', 'gene_id \"chr5.g30076\"; transcript_id \"chr5.g30076.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73025308', '73025531', '.', '-', '.', 'gene_id \"chr5.g30079\"; transcript_id \"chr5.g30079.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73027461', '73027643', '.', '+', '.', 'gene_id \"chr5.g30082\"; transcript_id \"chr5.g30082.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73037346', '73037762', '.', '-', '.', 'gene_id \"chr5.g30089\"; transcript_id \"chr5.g30089.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73049250', '73049432', '.', '+', '.', 'gene_id \"chr5.g30093\"; transcript_id \"chr5.g30093.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73072478', '73072796', '.', '+', '.', 'gene_id \"chr5.g30098\"; transcript_id \"chr5.g30098.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73096465', '73097294', '.', '-', '.', 'gene_id \"chr5.g30104\"; transcript_id \"chr5.g30104.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73095502', '73096312', '.', '-', '.', 'gene_id \"chr5.g30105\"; transcript_id \"chr5.g30105.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73103709', '73103897', '.', '+', '.', 'gene_id \"chr5.g30111\"; transcript_id \"chr5.g30111.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73103883', '73104134', '.', '+', '.', 'gene_id \"chr5.g30112\"; transcript_id \"chr5.g30112.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73115374', '73115731', '.', '+', '.', 'gene_id \"chr5.g30119\"; transcript_id \"chr5.g30119.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73129559', '73129982', '.', '-', '.', 'gene_id \"chr5.g30127\"; transcript_id \"chr5.g30127.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73128341', '73128630', '.', '+', '.', 'gene_id \"chr5.g30130\"; transcript_id \"chr5.g30130.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73132824', '73133074', '.', '-', '.', 'gene_id \"chr5.g30135\"; transcript_id \"chr5.g30135.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73137180', '73137656', '.', '-', '.', 'gene_id \"chr5.g30140\"; transcript_id \"chr5.g30140.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73141601', '73141855', '.', '-', '.', 'gene_id \"chr5.g30142\"; transcript_id \"chr5.g30142.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73143908', '73144624', '.', '-', '.', 'gene_id \"chr5.g30144\"; transcript_id \"chr5.g30144.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73175273', '73175452', '.', '-', '.', 'gene_id \"chr5.g30152\"; transcript_id \"chr5.g30152.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73195655', '73196034', '.', '+', '.', 'gene_id \"chr5.g30157\"; transcript_id \"chr5.g30157.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73197173', '73197703', '.', '-', '.', 'gene_id \"chr5.g30160\"; transcript_id \"chr5.g30160.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73197857', '73198149', '.', '-', '.', 'gene_id \"chr5.g30162\"; transcript_id \"chr5.g30162.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73208936', '73209051', '.', '+', '.', 'gene_id \"chr5.g30164\"; transcript_id \"chr5.g30164.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73475373', '73477565', '.', '-', '.', 'gene_id \"chr5.g30165\"; transcript_id \"chr5.g30165.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73475939', '73477531', '.', '-', '.', 'gene_id \"chr5.g30165\"; transcript_id \"chr5.g30165.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73211071', '73211114', '.', '-', '.', 'gene_id \"chr5.g30165\"; transcript_id \"chr5.g30165.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73496522', '73496675', '.', '+', '.', 'gene_id \"chr5.g30167\"; transcript_id \"chr5.g30167.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73475373', '73477450', '.', '+', '.', 'gene_id \"chr5.g30172\"; transcript_id \"chr5.g30172.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73223658', '73223827', '.', '+', '.', 'gene_id \"chr5.g30176\"; transcript_id \"chr5.g30176.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73326482', '73326947', '.', '-', '.', 'gene_id \"chr5.g30183\"; transcript_id \"chr5.g30183.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73386945', '73387268', '.', '-', '.', 'gene_id \"chr5.g30192\"; transcript_id \"chr5.g30192.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73388783', '73389001', '.', '-', '.', 'gene_id \"chr5.g30194\"; transcript_id \"chr5.g30194.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73391072', '73391243', '.', '+', '.', 'gene_id \"chr5.g30195\"; transcript_id \"chr5.g30195.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73403767', '73404034', '.', '-', '.', 'gene_id \"chr5.g30199\"; transcript_id \"chr5.g30199.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73407157', '73407213', '.', '+', '.', 'gene_id \"chr5.g30202\"; transcript_id \"chr5.g30202.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73429717', '73429898', '.', '+', '.', 'gene_id \"chr5.g30204\"; transcript_id \"chr5.g30204.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73439843', '73440014', '.', '+', '.', 'gene_id \"chr5.g30205\"; transcript_id \"chr5.g30205.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73439630', '73439861', '.', '+', '.', 'gene_id \"chr5.g30205\"; transcript_id \"chr5.g30205.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73440027', '73440180', '.', '-', '.', 'gene_id \"chr5.g30206\"; transcript_id \"chr5.g30206.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73440733', '73441741', '.', '-', '.', 'gene_id \"chr5.g30208\"; transcript_id \"chr5.g30208.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73451627', '73451796', '.', '-', '.', 'gene_id \"chr5.g30214\"; transcript_id \"chr5.g30214.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73458546', '73458757', '.', '+', '.', 'gene_id \"chr5.g30215\"; transcript_id \"chr5.g30215.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73460723', '73460946', '.', '-', '.', 'gene_id \"chr5.g30216\"; transcript_id \"chr5.g30216.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73485436', '73485653', '.', '-', '.', 'gene_id \"chr5.g30223\"; transcript_id \"chr5.g30223.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73560972', '73561004', '.', '+', '.', 'gene_id \"chr5.g30230\"; transcript_id \"chr5.g30230.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73587691', '73587847', '.', '+', '.', 'gene_id \"chr5.g30233\"; transcript_id \"chr5.g30233.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73702308', '73702459', '.', '+', '.', 'gene_id \"chr5.g30244\"; transcript_id \"chr5.g30244.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73768672', '73769117', '.', '+', '.', 'gene_id \"chr5.g30251\"; transcript_id \"chr5.g30251.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73770903', '73771223', '.', '-', '.', 'gene_id \"chr5.g30254\"; transcript_id \"chr5.g30254.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73772109', '73772304', '.', '+', '.', 'gene_id \"chr5.g30255\"; transcript_id \"chr5.g30255.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73771961', '73772125', '.', '+', '.', 'gene_id \"chr5.g30255\"; transcript_id \"chr5.g30255.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73810129', '73810286', '.', '-', '.', 'gene_id \"chr5.g30259\"; transcript_id \"chr5.g30259.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74024032', '74024344', '.', '+', '.', 'gene_id \"chr5.g30274\"; transcript_id \"chr5.g30274.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74099400', '74099654', '.', '+', '.', 'gene_id \"chr5.g30279\"; transcript_id \"chr5.g30279.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74099694', '74100153', '.', '+', '.', 'gene_id \"chr5.g30280\"; transcript_id \"chr5.g30280.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74100917', '74102113', '.', '+', '.', 'gene_id \"chr5.g30281\"; transcript_id \"chr5.g30281.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74127980', '74128183', '.', '+', '.', 'gene_id \"chr5.g30290\"; transcript_id \"chr5.g30290.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74138801', '74139102', '.', '+', '.', 'gene_id \"chr5.g30298\"; transcript_id \"chr5.g30298.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74144343', '74144584', '.', '-', '.', 'gene_id \"chr5.g30301\"; transcript_id \"chr5.g30301.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74145683', '74145850', '.', '-', '.', 'gene_id \"chr5.g30302\"; transcript_id \"chr5.g30302.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74145872', '74146090', '.', '-', '.', 'gene_id \"chr5.g30303\"; transcript_id \"chr5.g30303.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74231278', '74232310', '.', '-', '.', 'gene_id \"chr5.g30324\"; transcript_id \"chr5.g30324.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74233400', '74233675', '.', '+', '.', 'gene_id \"chr5.g30325\"; transcript_id \"chr5.g30325.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74259153', '74259341', '.', '+', '.', 'gene_id \"chr5.g30329\"; transcript_id \"chr5.g30329.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74259528', '74259596', '.', '+', '.', 'gene_id \"chr5.g30330\"; transcript_id \"chr5.g30330.i1\";']\n",
      "negative:  ['scaffold_1464', 'PASA', 'exon', '17642', '17686', '.', '+', '.', 'gene_id \"scaffold_1464.g8\"; transcript_id \"scaffold_1464.g8.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74468165', '74468434', '.', '+', '.', 'gene_id \"chr5.g30344\"; transcript_id \"chr5.g30344.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74498541', '74498754', '.', '+', '.', 'gene_id \"chr5.g30349\"; transcript_id \"chr5.g30349.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74574775', '74575214', '.', '-', '.', 'gene_id \"chr5.g30353\"; transcript_id \"chr5.g30353.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74660301', '74660471', '.', '-', '.', 'gene_id \"chr5.g30361\"; transcript_id \"chr5.g30361.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74661138', '74661406', '.', '-', '.', 'gene_id \"chr5.g30362\"; transcript_id \"chr5.g30362.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74763383', '74763554', '.', '+', '.', 'gene_id \"chr5.g30395\"; transcript_id \"chr5.g30395.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74764275', '74764431', '.', '-', '.', 'gene_id \"chr5.g30396\"; transcript_id \"chr5.g30396.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74766429', '74766663', '.', '-', '.', 'gene_id \"chr5.g30397\"; transcript_id \"chr5.g30397.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74769747', '74769901', '.', '+', '.', 'gene_id \"chr5.g30399\"; transcript_id \"chr5.g30399.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74811116', '74811309', '.', '-', '.', 'gene_id \"chr5.g30408\"; transcript_id \"chr5.g30408.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75222991', '75223154', '.', '+', '.', 'gene_id \"chr5.g30448\"; transcript_id \"chr5.g30448.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75486772', '75486822', '.', '+', '.', 'gene_id \"chr5.g30498\"; transcript_id \"chr5.g30498.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75448417', '75448723', '.', '+', '.', 'gene_id \"chr5.g30498\"; transcript_id \"chr5.g30498.i20\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75443745', '75443981', '.', '+', '.', 'gene_id \"chr5.g30508\"; transcript_id \"chr5.g30508.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75498327', '75498484', '.', '+', '.', 'gene_id \"chr5.g30567\"; transcript_id \"chr5.g30567.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75513229', '75513441', '.', '+', '.', 'gene_id \"chr5.g30579\"; transcript_id \"chr5.g30579.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75522811', '75523270', '.', '+', '.', 'gene_id \"chr5.g30591\"; transcript_id \"chr5.g30591.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '288728', '288789', '.', '-', '.', 'gene_id \"chr38.g97\"; transcript_id \"chr38.g97.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75532835', '75533053', '.', '-', '.', 'gene_id \"chr5.g30599\"; transcript_id \"chr5.g30599.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75545997', '75546360', '.', '-', '.', 'gene_id \"chr5.g30619\"; transcript_id \"chr5.g30619.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75546361', '75546623', '.', '+', '.', 'gene_id \"chr5.g30621\"; transcript_id \"chr5.g30621.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75557687', '75557902', '.', '+', '.', 'gene_id \"chr5.g30626\"; transcript_id \"chr5.g30626.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75576124', '75576354', '.', '+', '.', 'gene_id \"chr5.g30642\"; transcript_id \"chr5.g30642.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '334937', '335092', '.', '+', '.', 'gene_id \"chr38.g145\"; transcript_id \"chr38.g145.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75593204', '75593392', '.', '+', '.', 'gene_id \"chr5.g30655\"; transcript_id \"chr5.g30655.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75605280', '75605562', '.', '-', '.', 'gene_id \"chr5.g30665\"; transcript_id \"chr5.g30665.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '350154', '350320', '.', '+', '.', 'gene_id \"chr38.g160\"; transcript_id \"chr38.g160.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '350258', '350320', '.', '+', '.', 'gene_id \"chr38.g160\"; transcript_id \"chr38.g160.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75608782', '75609004', '.', '-', '.', 'gene_id \"chr5.g30668\"; transcript_id \"chr5.g30668.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75624199', '75624452', '.', '+', '.', 'gene_id \"chr5.g30685\"; transcript_id \"chr5.g30685.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75630677', '75630969', '.', '-', '.', 'gene_id \"chr5.g30688\"; transcript_id \"chr5.g30688.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75647464', '75647631', '.', '+', '.', 'gene_id \"chr5.g30698\"; transcript_id \"chr5.g30698.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75652121', '75652439', '.', '+', '.', 'gene_id \"chr5.g30705\"; transcript_id \"chr5.g30705.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75653858', '75654032', '.', '-', '.', 'gene_id \"chr5.g30706\"; transcript_id \"chr5.g30706.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75656373', '75656541', '.', '-', '.', 'gene_id \"chr5.g30711\"; transcript_id \"chr5.g30711.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75663469', '75663637', '.', '-', '.', 'gene_id \"chr5.g30718\"; transcript_id \"chr5.g30718.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75677767', '75678007', '.', '+', '.', 'gene_id \"chr5.g30737\"; transcript_id \"chr5.g30737.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '464344', '464381', '.', '-', '.', 'gene_id \"chr38.g265\"; transcript_id \"chr38.g265.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75727565', '75727846', '.', '+', '.', 'gene_id \"chr5.g30786\"; transcript_id \"chr5.g30786.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '501205', '501250', '.', '+', '.', 'gene_id \"chr38.g304\"; transcript_id \"chr38.g304.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75778482', '75778654', '.', '+', '.', 'gene_id \"chr5.g30825\"; transcript_id \"chr5.g30825.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75828646', '75828956', '.', '+', '.', 'gene_id \"chr5.g30877\"; transcript_id \"chr5.g30877.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75869537', '75869630', '.', '-', '.', 'gene_id \"chr5.g30885\"; transcript_id \"chr5.g30885.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75850826', '75851264', '.', '+', '.', 'gene_id \"chr5.g30895\"; transcript_id \"chr5.g30895.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75848780', '75849058', '.', '+', '.', 'gene_id \"chr5.g30896\"; transcript_id \"chr5.g30896.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75883318', '75883537', '.', '-', '.', 'gene_id \"chr5.g30915\"; transcript_id \"chr5.g30915.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75888754', '75888904', '.', '-', '.', 'gene_id \"chr5.g30917\"; transcript_id \"chr5.g30917.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75942461', '75942669', '.', '+', '.', 'gene_id \"chr5.g30928\"; transcript_id \"chr5.g30928.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75960124', '75960779', '.', '+', '.', 'gene_id \"chr5.g30936\"; transcript_id \"chr5.g30936.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76069873', '76071142', '.', '+', '.', 'gene_id \"chr5.g30940\"; transcript_id \"chr5.g30940.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76008897', '76009060', '.', '-', '.', 'gene_id \"chr5.g30943\"; transcript_id \"chr5.g30943.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76024300', '76024741', '.', '-', '.', 'gene_id \"chr5.g30944\"; transcript_id \"chr5.g30944.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75999043', '75999212', '.', '-', '.', 'gene_id \"chr5.g30958\"; transcript_id \"chr5.g30958.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76009316', '76009485', '.', '+', '.', 'gene_id \"chr5.g30966\"; transcript_id \"chr5.g30966.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76017057', '76017211', '.', '-', '.', 'gene_id \"chr5.g30967\"; transcript_id \"chr5.g30967.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76019769', '76020024', '.', '+', '.', 'gene_id \"chr5.g30970\"; transcript_id \"chr5.g30970.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76029902', '76030093', '.', '-', '.', 'gene_id \"chr5.g30974\"; transcript_id \"chr5.g30974.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76035879', '76036034', '.', '-', '.', 'gene_id \"chr5.g30976\"; transcript_id \"chr5.g30976.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76056204', '76056385', '.', '+', '.', 'gene_id \"chr5.g30992\"; transcript_id \"chr5.g30992.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76063410', '76065336', '.', '+', '.', 'gene_id \"chr5.g30998\"; transcript_id \"chr5.g30998.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76075752', '76076030', '.', '+', '.', 'gene_id \"chr5.g31012\"; transcript_id \"chr5.g31012.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76077041', '76077281', '.', '-', '.', 'gene_id \"chr5.g31014\"; transcript_id \"chr5.g31014.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76143046', '76143085', '.', '-', '.', 'gene_id \"chr5.g31036\"; transcript_id \"chr5.g31036.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76146421', '76146466', '.', '+', '.', 'gene_id \"chr5.g31037\"; transcript_id \"chr5.g31037.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76234886', '76235066', '.', '-', '.', 'gene_id \"chr5.g31076\"; transcript_id \"chr5.g31076.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76280091', '76280564', '.', '-', '.', 'gene_id \"chr5.g31108\"; transcript_id \"chr5.g31108.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76371539', '76371573', '.', '-', '.', 'gene_id \"chr5.g31109\"; transcript_id \"chr5.g31109.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76410895', '76411705', '.', '+', '.', 'gene_id \"chr5.g31113\"; transcript_id \"chr5.g31113.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76419248', '76419422', '.', '+', '.', 'gene_id \"chr5.g31114\"; transcript_id \"chr5.g31114.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76409306', '76409702', '.', '-', '.', 'gene_id \"chr5.g31116\"; transcript_id \"chr5.g31116.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76404720', '76404982', '.', '+', '.', 'gene_id \"chr5.g31117\"; transcript_id \"chr5.g31117.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76418255', '76418418', '.', '+', '.', 'gene_id \"chr5.g31118\"; transcript_id \"chr5.g31118.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76419532', '76419764', '.', '-', '.', 'gene_id \"chr5.g31119\"; transcript_id \"chr5.g31119.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76283957', '76284341', '.', '-', '.', 'gene_id \"chr5.g31127\"; transcript_id \"chr5.g31127.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76287292', '76287489', '.', '+', '.', 'gene_id \"chr5.g31132\"; transcript_id \"chr5.g31132.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76289311', '76289942', '.', '-', '.', 'gene_id \"chr5.g31135\"; transcript_id \"chr5.g31135.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76291061', '76291343', '.', '-', '.', 'gene_id \"chr5.g31139\"; transcript_id \"chr5.g31139.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '968238', '968403', '.', '+', '.', 'gene_id \"chr38.g688\"; transcript_id \"chr38.g688.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76318137', '76318176', '.', '+', '.', 'gene_id \"chr5.g31174\"; transcript_id \"chr5.g31174.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76318130', '76318616', '.', '-', '.', 'gene_id \"chr5.g31175\"; transcript_id \"chr5.g31175.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76347202', '76347449', '.', '+', '.', 'gene_id \"chr5.g31213\"; transcript_id \"chr5.g31213.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76350286', '76350469', '.', '+', '.', 'gene_id \"chr5.g31215\"; transcript_id \"chr5.g31215.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76367039', '76367449', '.', '+', '.', 'gene_id \"chr5.g31222\"; transcript_id \"chr5.g31222.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76383696', '76383859', '.', '+', '.', 'gene_id \"chr5.g31231\"; transcript_id \"chr5.g31231.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1050397', '1050839', '.', '-', '.', 'gene_id \"chr38.g762\"; transcript_id \"chr38.g762.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76400305', '76400510', '.', '+', '.', 'gene_id \"chr5.g31241\"; transcript_id \"chr5.g31241.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76405187', '76405376', '.', '+', '.', 'gene_id \"chr5.g31243\"; transcript_id \"chr5.g31243.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76441792', '76442012', '.', '-', '.', 'gene_id \"chr5.g31261\"; transcript_id \"chr5.g31261.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76448918', '76448966', '.', '+', '.', 'gene_id \"chr5.g31264\"; transcript_id \"chr5.g31264.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76450427', '76450721', '.', '+', '.', 'gene_id \"chr5.g31266\"; transcript_id \"chr5.g31266.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76450746', '76450914', '.', '-', '.', 'gene_id \"chr5.g31267\"; transcript_id \"chr5.g31267.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76454082', '76454481', '.', '+', '.', 'gene_id \"chr5.g31271\"; transcript_id \"chr5.g31271.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76454724', '76456604', '.', '-', '.', 'gene_id \"chr5.g31276\"; transcript_id \"chr5.g31276.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76505327', '76507213', '.', '+', '.', 'gene_id \"chr5.g31277\"; transcript_id \"chr5.g31277.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76477165', '76478051', '.', '+', '.', 'gene_id \"chr5.g31292\"; transcript_id \"chr5.g31292.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76511024', '76511192', '.', '+', '.', 'gene_id \"chr5.g31305\"; transcript_id \"chr5.g31305.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76512221', '76512458', '.', '-', '.', 'gene_id \"chr5.g31306\"; transcript_id \"chr5.g31306.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76540529', '76540817', '.', '+', '.', 'gene_id \"chr5.g31312\"; transcript_id \"chr5.g31312.i4\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76542494', '76542714', '.', '+', '.', 'gene_id \"chr5.g31318\"; transcript_id \"chr5.g31318.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76587888', '76588278', '.', '+', '.', 'gene_id \"chr5.g31336\"; transcript_id \"chr5.g31336.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76559896', '76560064', '.', '-', '.', 'gene_id \"chr5.g31340\"; transcript_id \"chr5.g31340.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76621768', '76621933', '.', '-', '.', 'gene_id \"chr5.g31341\"; transcript_id \"chr5.g31341.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76617640', '76617855', '.', '+', '.', 'gene_id \"chr5.g31342\"; transcript_id \"chr5.g31342.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76615750', '76616091', '.', '-', '.', 'gene_id \"chr5.g31343\"; transcript_id \"chr5.g31343.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76612558', '76613013', '.', '+', '.', 'gene_id \"chr5.g31346\"; transcript_id \"chr5.g31346.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76609005', '76609222', '.', '-', '.', 'gene_id \"chr5.g31347\"; transcript_id \"chr5.g31347.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1111602', '1111850', '.', '-', '.', 'gene_id \"chr38.g834\"; transcript_id \"chr38.g834.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1116761', '1116970', '.', '+', '.', 'gene_id \"chr38.g842\"; transcript_id \"chr38.g842.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76610458', '76612388', '.', '-', '.', 'gene_id \"chr5.g31375\"; transcript_id \"chr5.g31375.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76613451', '76613788', '.', '-', '.', 'gene_id \"chr5.g31377\"; transcript_id \"chr5.g31377.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76624448', '76624689', '.', '+', '.', 'gene_id \"chr5.g31386\"; transcript_id \"chr5.g31386.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76629616', '76629814', '.', '+', '.', 'gene_id \"chr5.g31388\"; transcript_id \"chr5.g31388.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76643065', '76643331', '.', '+', '.', 'gene_id \"chr5.g31403\"; transcript_id \"chr5.g31403.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76647312', '76647484', '.', '-', '.', 'gene_id \"chr5.g31412\"; transcript_id \"chr5.g31412.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76666682', '76666777', '.', '-', '.', 'gene_id \"chr5.g31433\"; transcript_id \"chr5.g31433.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76667262', '76667725', '.', '-', '.', 'gene_id \"chr5.g31435\"; transcript_id \"chr5.g31435.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76668682', '76668952', '.', '-', '.', 'gene_id \"chr5.g31438\"; transcript_id \"chr5.g31438.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76684150', '76684315', '.', '-', '.', 'gene_id \"chr5.g31462\"; transcript_id \"chr5.g31462.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76701232', '76701444', '.', '+', '.', 'gene_id \"chr5.g31475\"; transcript_id \"chr5.g31475.i21\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76724766', '76724960', '.', '+', '.', 'gene_id \"chr5.g31487\"; transcript_id \"chr5.g31487.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76740838', '76741015', '.', '+', '.', 'gene_id \"chr5.g31504\"; transcript_id \"chr5.g31504.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76758613', '76759429', '.', '+', '.', 'gene_id \"chr5.g31512\"; transcript_id \"chr5.g31512.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76759593', '76759802', '.', '+', '.', 'gene_id \"chr5.g31513\"; transcript_id \"chr5.g31513.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76731441', '76731950', '.', '+', '.', 'gene_id \"chr5.g31520\"; transcript_id \"chr5.g31520.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76730685', '76731311', '.', '-', '.', 'gene_id \"chr5.g31524\"; transcript_id \"chr5.g31524.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1277463', '1277498', '.', '-', '.', 'gene_id \"chr38.g950\"; transcript_id \"chr38.g950.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76747261', '76747464', '.', '+', '.', 'gene_id \"chr5.g31531\"; transcript_id \"chr5.g31531.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76747510', '76747662', '.', '+', '.', 'gene_id \"chr5.g31532\"; transcript_id \"chr5.g31532.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76750599', '76750758', '.', '-', '.', 'gene_id \"chr5.g31536\"; transcript_id \"chr5.g31536.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76752684', '76752734', '.', '-', '.', 'gene_id \"chr5.g31540\"; transcript_id \"chr5.g31540.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1477278', '1477712', '.', '+', '.', 'gene_id \"chr38.g997\"; transcript_id \"chr38.g997.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76801290', '76801457', '.', '+', '.', 'gene_id \"chr5.g31575\"; transcript_id \"chr5.g31575.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76870120', '76870358', '.', '+', '.', 'gene_id \"chr5.g31586\"; transcript_id \"chr5.g31586.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76827786', '76828484', '.', '+', '.', 'gene_id \"chr5.g31602\"; transcript_id \"chr5.g31602.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76837484', '76837812', '.', '-', '.', 'gene_id \"chr5.g31615\"; transcript_id \"chr5.g31615.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76856337', '76856389', '.', '-', '.', 'gene_id \"chr5.g31631\"; transcript_id \"chr5.g31631.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76863606', '76863806', '.', '+', '.', 'gene_id \"chr5.g31642\"; transcript_id \"chr5.g31642.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76887362', '76887787', '.', '+', '.', 'gene_id \"chr5.g31651\"; transcript_id \"chr5.g31651.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76871133', '76871296', '.', '+', '.', 'gene_id \"chr5.g31652\"; transcript_id \"chr5.g31652.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76882673', '76882857', '.', '+', '.', 'gene_id \"chr5.g31653\"; transcript_id \"chr5.g31653.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76883570', '76883801', '.', '+', '.', 'gene_id \"chr5.g31656\"; transcript_id \"chr5.g31656.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76906468', '76907126', '.', '-', '.', 'gene_id \"chr5.g31680\"; transcript_id \"chr5.g31680.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76936204', '76936407', '.', '-', '.', 'gene_id \"chr5.g31712\"; transcript_id \"chr5.g31712.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76963095', '76963296', '.', '-', '.', 'gene_id \"chr5.g31726\"; transcript_id \"chr5.g31726.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76968152', '76968330', '.', '+', '.', 'gene_id \"chr5.g31728\"; transcript_id \"chr5.g31728.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76968873', '76969058', '.', '+', '.', 'gene_id \"chr5.g31729\"; transcript_id \"chr5.g31729.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77003516', '77003741', '.', '-', '.', 'gene_id \"chr5.g31743\"; transcript_id \"chr5.g31743.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77007819', '77008107', '.', '+', '.', 'gene_id \"chr5.g31746\"; transcript_id \"chr5.g31746.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1760397', '1760429', '.', '+', '.', 'gene_id \"chr38.g1185\"; transcript_id \"chr38.g1185.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77034357', '77034680', '.', '+', '.', 'gene_id \"chr5.g31755\"; transcript_id \"chr5.g31755.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1764914', '1765159', '.', '+', '.', 'gene_id \"chr38.g1191\"; transcript_id \"chr38.g1191.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77057869', '77058028', '.', '+', '.', 'gene_id \"chr5.g31759\"; transcript_id \"chr5.g31759.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77066538', '77066604', '.', '-', '.', 'gene_id \"chr5.g31761\"; transcript_id \"chr5.g31761.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77090507', '77090686', '.', '-', '.', 'gene_id \"chr5.g31769\"; transcript_id \"chr5.g31769.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77204827', '77205057', '.', '+', '.', 'gene_id \"chr5.g31779\"; transcript_id \"chr5.g31779.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77298786', '77298996', '.', '+', '.', 'gene_id \"chr5.g31790\"; transcript_id \"chr5.g31790.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77331059', '77331216', '.', '-', '.', 'gene_id \"chr5.g31792\"; transcript_id \"chr5.g31792.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77373054', '77373314', '.', '-', '.', 'gene_id \"chr5.g31803\"; transcript_id \"chr5.g31803.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1895487', '1895769', '.', '-', '.', 'gene_id \"chr38.g1248\"; transcript_id \"chr38.g1248.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77380644', '77380860', '.', '-', '.', 'gene_id \"chr5.g31806\"; transcript_id \"chr5.g31806.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77413212', '77413405', '.', '+', '.', 'gene_id \"chr5.g31810\"; transcript_id \"chr5.g31810.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77448216', '77448264', '.', '-', '.', 'gene_id \"chr5.g31814\"; transcript_id \"chr5.g31814.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77479320', '77479486', '.', '+', '.', 'gene_id \"chr5.g31819\"; transcript_id \"chr5.g31819.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77478988', '77479167', '.', '-', '.', 'gene_id \"chr5.g31824\"; transcript_id \"chr5.g31824.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77507902', '77508055', '.', '+', '.', 'gene_id \"chr5.g31845\"; transcript_id \"chr5.g31845.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77535619', '77535837', '.', '-', '.', 'gene_id \"chr5.g31860\"; transcript_id \"chr5.g31860.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77539962', '77540214', '.', '+', '.', 'gene_id \"chr5.g31862\"; transcript_id \"chr5.g31862.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77539133', '77539313', '.', '+', '.', 'gene_id \"chr5.g31863\"; transcript_id \"chr5.g31863.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77536580', '77536721', '.', '+', '.', 'gene_id \"chr5.g31864\"; transcript_id \"chr5.g31864.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77551077', '77551306', '.', '+', '.', 'gene_id \"chr5.g31875\"; transcript_id \"chr5.g31875.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77579609', '77579685', '.', '+', '.', 'gene_id \"chr5.g31881\"; transcript_id \"chr5.g31881.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77601910', '77602097', '.', '-', '.', 'gene_id \"chr5.g31893\"; transcript_id \"chr5.g31893.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77632341', '77632693', '.', '+', '.', 'gene_id \"chr5.g31904\"; transcript_id \"chr5.g31904.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77681758', '77681853', '.', '-', '.', 'gene_id \"chr5.g31925\"; transcript_id \"chr5.g31925.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77714947', '77715304', '.', '-', '.', 'gene_id \"chr5.g31958\"; transcript_id \"chr5.g31958.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77720400', '77720595', '.', '-', '.', 'gene_id \"chr5.g31959\"; transcript_id \"chr5.g31959.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77758342', '77758598', '.', '-', '.', 'gene_id \"chr5.g31977\"; transcript_id \"chr5.g31977.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77758583', '77758822', '.', '+', '.', 'gene_id \"chr5.g31978\"; transcript_id \"chr5.g31978.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77782732', '77782962', '.', '+', '.', 'gene_id \"chr5.g32004\"; transcript_id \"chr5.g32004.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77781581', '77781879', '.', '-', '.', 'gene_id \"chr5.g32022\"; transcript_id \"chr5.g32022.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77794215', '77794840', '.', '-', '.', 'gene_id \"chr5.g32037\"; transcript_id \"chr5.g32037.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77809078', '77809284', '.', '-', '.', 'gene_id \"chr5.g32053\"; transcript_id \"chr5.g32053.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77822272', '77822509', '.', '+', '.', 'gene_id \"chr5.g32055\"; transcript_id \"chr5.g32055.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77837169', '77837334', '.', '+', '.', 'gene_id \"chr5.g32060\"; transcript_id \"chr5.g32060.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77838584', '77838845', '.', '-', '.', 'gene_id \"chr5.g32061\"; transcript_id \"chr5.g32061.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77853678', '77853882', '.', '+', '.', 'gene_id \"chr5.g32074\"; transcript_id \"chr5.g32074.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77895259', '77896129', '.', '+', '.', 'gene_id \"chr5.g32083\"; transcript_id \"chr5.g32083.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77891765', '77892351', '.', '+', '.', 'gene_id \"chr5.g32085\"; transcript_id \"chr5.g32085.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2242242', '2242462', '.', '+', '.', 'gene_id \"chr38.g1543\"; transcript_id \"chr38.g1543.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77955050', '77955857', '.', '-', '.', 'gene_id \"chr5.g32127\"; transcript_id \"chr5.g32127.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2267852', '2268219', '.', '+', '.', 'gene_id \"chr38.g1551\"; transcript_id \"chr38.g1551.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77959980', '77960164', '.', '-', '.', 'gene_id \"chr5.g32132\"; transcript_id \"chr5.g32132.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77976913', '77977067', '.', '-', '.', 'gene_id \"chr5.g32142\"; transcript_id \"chr5.g32142.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77987275', '77987436', '.', '-', '.', 'gene_id \"chr5.g32145\"; transcript_id \"chr5.g32145.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77988172', '77988336', '.', '+', '.', 'gene_id \"chr5.g32146\"; transcript_id \"chr5.g32146.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78009279', '78009474', '.', '-', '.', 'gene_id \"chr5.g32167\"; transcript_id \"chr5.g32167.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '78040665', '78040829', '.', '-', '.', 'gene_id \"chr5.g32208\"; transcript_id \"chr5.g32208.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78040796', '78040946', '.', '-', '.', 'gene_id \"chr5.g32208\"; transcript_id \"chr5.g32208.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2705093', '2705366', '.', '-', '.', 'gene_id \"chr38.g1622\"; transcript_id \"chr38.g1622.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2712473', '2718742', '.', '-', '.', 'gene_id \"chr38.g1631\"; transcript_id \"chr38.g1631.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2721773', '2722152', '.', '-', '.', 'gene_id \"chr38.g1632\"; transcript_id \"chr38.g1632.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2703760', '2703933', '.', '+', '.', 'gene_id \"chr38.g1634\"; transcript_id \"chr38.g1634.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78048352', '78048625', '.', '+', '.', 'gene_id \"chr5.g32214\"; transcript_id \"chr5.g32214.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78048921', '78049163', '.', '+', '.', 'gene_id \"chr5.g32217\"; transcript_id \"chr5.g32217.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78051916', '78052421', '.', '+', '.', 'gene_id \"chr5.g32219\"; transcript_id \"chr5.g32219.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78066353', '78066942', '.', '+', '.', 'gene_id \"chr5.g32236\"; transcript_id \"chr5.g32236.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78082967', '78083191', '.', '+', '.', 'gene_id \"chr5.g32242\"; transcript_id \"chr5.g32242.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78087719', '78088134', '.', '-', '.', 'gene_id \"chr5.g32244\"; transcript_id \"chr5.g32244.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78078278', '78078585', '.', '-', '.', 'gene_id \"chr5.g32245\"; transcript_id \"chr5.g32245.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78081820', '78081991', '.', '+', '.', 'gene_id \"chr5.g32249\"; transcript_id \"chr5.g32249.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78077285', '78077438', '.', '-', '.', 'gene_id \"chr5.g32251\"; transcript_id \"chr5.g32251.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78074770', '78075326', '.', '-', '.', 'gene_id \"chr5.g32252\"; transcript_id \"chr5.g32252.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78073769', '78073965', '.', '-', '.', 'gene_id \"chr5.g32254\"; transcript_id \"chr5.g32254.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78071373', '78072096', '.', '+', '.', 'gene_id \"chr5.g32255\"; transcript_id \"chr5.g32255.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78093061', '78093211', '.', '-', '.', 'gene_id \"chr5.g32263\"; transcript_id \"chr5.g32263.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78094631', '78094947', '.', '-', '.', 'gene_id \"chr5.g32264\"; transcript_id \"chr5.g32264.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78411846', '78412199', '.', '-', '.', 'gene_id \"chr5.g32314\"; transcript_id \"chr5.g32314.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78412178', '78412450', '.', '-', '.', 'gene_id \"chr5.g32314\"; transcript_id \"chr5.g32314.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78411499', '78411699', '.', '+', '.', 'gene_id \"chr5.g32315\"; transcript_id \"chr5.g32315.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78411686', '78411854', '.', '+', '.', 'gene_id \"chr5.g32316\"; transcript_id \"chr5.g32316.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78415518', '78416516', '.', '-', '.', 'gene_id \"chr5.g32317\"; transcript_id \"chr5.g32317.i6\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78414766', '78415517', '.', '-', '.', 'gene_id \"chr5.g32317\"; transcript_id \"chr5.g32317.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78417587', '78417786', '.', '-', '.', 'gene_id \"chr5.g32317\"; transcript_id \"chr5.g32317.i8\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78416702', '78416909', '.', '-', '.', 'gene_id \"chr5.g32317\"; transcript_id \"chr5.g32317.i9\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78416847', '78416999', '.', '+', '.', 'gene_id \"chr5.g32318\"; transcript_id \"chr5.g32318.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78417157', '78417594', '.', '+', '.', 'gene_id \"chr5.g32318\"; transcript_id \"chr5.g32318.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78418995', '78419188', '.', '-', '.', 'gene_id \"chr5.g32322\"; transcript_id \"chr5.g32322.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78419435', '78419757', '.', '-', '.', 'gene_id \"chr5.g32323\"; transcript_id \"chr5.g32323.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78421086', '78421374', '.', '-', '.', 'gene_id \"chr5.g32325\"; transcript_id \"chr5.g32325.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78423035', '78423276', '.', '-', '.', 'gene_id \"chr5.g32328\"; transcript_id \"chr5.g32328.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78423566', '78423724', '.', '-', '.', 'gene_id \"chr5.g32329\"; transcript_id \"chr5.g32329.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78424347', '78424510', '.', '-', '.', 'gene_id \"chr5.g32330\"; transcript_id \"chr5.g32330.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78425113', '78425276', '.', '-', '.', 'gene_id \"chr5.g32331\"; transcript_id \"chr5.g32331.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78423988', '78424227', '.', '+', '.', 'gene_id \"chr5.g32333\"; transcript_id \"chr5.g32333.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78430618', '78430793', '.', '-', '.', 'gene_id \"chr5.g32338\"; transcript_id \"chr5.g32338.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78434991', '78435624', '.', '-', '.', 'gene_id \"chr5.g32341\"; transcript_id \"chr5.g32341.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78444869', '78445101', '.', '-', '.', 'gene_id \"chr5.g32349\"; transcript_id \"chr5.g32349.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78445101', '78445258', '.', '-', '.', 'gene_id \"chr5.g32349\"; transcript_id \"chr5.g32349.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78446228', '78446732', '.', '+', '.', 'gene_id \"chr5.g32350\"; transcript_id \"chr5.g32350.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78446837', '78447088', '.', '-', '.', 'gene_id \"chr5.g32351\"; transcript_id \"chr5.g32351.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78449523', '78450309', '.', '+', '.', 'gene_id \"chr5.g32353\"; transcript_id \"chr5.g32353.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78450207', '78450402', '.', '+', '.', 'gene_id \"chr5.g32353\"; transcript_id \"chr5.g32353.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78451372', '78451663', '.', '+', '.', 'gene_id \"chr5.g32357\"; transcript_id \"chr5.g32357.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78449343', '78449541', '.', '-', '.', 'gene_id \"chr5.g32358\"; transcript_id \"chr5.g32358.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78450519', '78450677', '.', '+', '.', 'gene_id \"chr5.g32359\"; transcript_id \"chr5.g32359.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78451669', '78451829', '.', '+', '.', 'gene_id \"chr5.g32361\"; transcript_id \"chr5.g32361.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78451891', '78452447', '.', '-', '.', 'gene_id \"chr5.g32362\"; transcript_id \"chr5.g32362.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78455053', '78455224', '.', '-', '.', 'gene_id \"chr5.g32364\"; transcript_id \"chr5.g32364.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78454335', '78454690', '.', '+', '.', 'gene_id \"chr5.g32365\"; transcript_id \"chr5.g32365.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78462086', '78462311', '.', '-', '.', 'gene_id \"chr5.g32373\"; transcript_id \"chr5.g32373.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78487961', '78488472', '.', '+', '.', 'gene_id \"chr5.g32396\"; transcript_id \"chr5.g32396.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78487961', '78488472', '.', '+', '.', 'gene_id \"chr5.g32396\"; transcript_id \"chr5.g32396.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78487315', '78487808', '.', '+', '.', 'gene_id \"chr5.g32397\"; transcript_id \"chr5.g32397.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2704652', '2705048', '.', '+', '.', 'gene_id \"chr38.g1791\"; transcript_id \"chr38.g1791.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2719046', '2719220', '.', '-', '.', 'gene_id \"chr38.g1802\"; transcript_id \"chr38.g1802.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2719782', '2719969', '.', '-', '.', 'gene_id \"chr38.g1804\"; transcript_id \"chr38.g1804.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78511468', '78511682', '.', '+', '.', 'gene_id \"chr5.g32432\"; transcript_id \"chr5.g32432.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78513977', '78514866', '.', '-', '.', 'gene_id \"chr5.g32434\"; transcript_id \"chr5.g32434.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78514086', '78514866', '.', '-', '.', 'gene_id \"chr5.g32434\"; transcript_id \"chr5.g32434.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78518118', '78518391', '.', '+', '.', 'gene_id \"chr5.g32442\"; transcript_id \"chr5.g32442.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78584259', '78584352', '.', '+', '.', 'gene_id \"chr5.g32451\"; transcript_id \"chr5.g32451.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78600806', '78600966', '.', '+', '.', 'gene_id \"chr5.g32469\"; transcript_id \"chr5.g32469.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78602260', '78602468', '.', '-', '.', 'gene_id \"chr5.g32470\"; transcript_id \"chr5.g32470.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78615343', '78615481', '.', '-', '.', 'gene_id \"chr5.g32480\"; transcript_id \"chr5.g32480.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78615343', '78615522', '.', '-', '.', 'gene_id \"chr5.g32480\"; transcript_id \"chr5.g32480.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78615502', '78615804', '.', '-', '.', 'gene_id \"chr5.g32480\"; transcript_id \"chr5.g32480.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78619443', '78619646', '.', '+', '.', 'gene_id \"chr5.g32487\"; transcript_id \"chr5.g32487.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78651448', '78652172', '.', '-', '.', 'gene_id \"chr5.g32508\"; transcript_id \"chr5.g32508.i3\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78658501', '78658690', '.', '-', '.', 'gene_id \"chr5.g32533\"; transcript_id \"chr5.g32533.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78731896', '78732131', '.', '+', '.', 'gene_id \"chr5.g32550\"; transcript_id \"chr5.g32550.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78734416', '78734706', '.', '-', '.', 'gene_id \"chr5.g32551\"; transcript_id \"chr5.g32551.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78773910', '78773939', '.', '-', '.', 'gene_id \"chr5.g32561\"; transcript_id \"chr5.g32561.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78792436', '78792662', '.', '+', '.', 'gene_id \"chr5.g32563\"; transcript_id \"chr5.g32563.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78826522', '78826677', '.', '-', '.', 'gene_id \"chr5.g32577\"; transcript_id \"chr5.g32577.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78830150', '78830371', '.', '+', '.', 'gene_id \"chr5.g32579\"; transcript_id \"chr5.g32579.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78831409', '78831562', '.', '+', '.', 'gene_id \"chr5.g32584\"; transcript_id \"chr5.g32584.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78848205', '78848468', '.', '-', '.', 'gene_id \"chr5.g32586\"; transcript_id \"chr5.g32586.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78860352', '78860558', '.', '+', '.', 'gene_id \"chr5.g32588\"; transcript_id \"chr5.g32588.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78915530', '78915798', '.', '-', '.', 'gene_id \"chr5.g32608\"; transcript_id \"chr5.g32608.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78965471', '78965649', '.', '-', '.', 'gene_id \"chr5.g32656\"; transcript_id \"chr5.g32656.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78965925', '78966321', '.', '-', '.', 'gene_id \"chr5.g32658\"; transcript_id \"chr5.g32658.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79001887', '79002138', '.', '-', '.', 'gene_id \"chr5.g32670\"; transcript_id \"chr5.g32670.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79004893', '79005051', '.', '-', '.', 'gene_id \"chr5.g32672\"; transcript_id \"chr5.g32672.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79055360', '79055518', '.', '-', '.', 'gene_id \"chr5.g32686\"; transcript_id \"chr5.g32686.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79197393', '79197437', '.', '-', '.', 'gene_id \"chr5.g32704\"; transcript_id \"chr5.g32704.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '3355692', '3356093', '.', '+', '.', 'gene_id \"chr38.g2151\"; transcript_id \"chr38.g2151.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80010426', '80010481', '.', '+', '.', 'gene_id \"chr5.g32754\"; transcript_id \"chr5.g32754.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80043334', '80043543', '.', '-', '.', 'gene_id \"chr5.g32761\"; transcript_id \"chr5.g32761.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80142237', '80142376', '.', '-', '.', 'gene_id \"chr5.g32864\"; transcript_id \"chr5.g32864.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '5033885', '5034116', '.', '-', '.', 'gene_id \"chr38.g2310\"; transcript_id \"chr38.g2310.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '5235154', '5235186', '.', '-', '.', 'gene_id \"chr38.g2318\"; transcript_id \"chr38.g2318.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80233676', '80233717', '.', '+', '.', 'gene_id \"chr5.g32953\"; transcript_id \"chr5.g32953.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80243490', '80243522', '.', '-', '.', 'gene_id \"chr5.g32963\"; transcript_id \"chr5.g32963.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80272456', '80272716', '.', '-', '.', 'gene_id \"chr5.g32991\"; transcript_id \"chr5.g32991.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80275960', '80276462', '.', '-', '.', 'gene_id \"chr5.g32997\"; transcript_id \"chr5.g32997.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '6011936', '6012890', '.', '-', '.', 'gene_id \"chr38.g2505\"; transcript_id \"chr38.g2505.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '6027109', '6027148', '.', '-', '.', 'gene_id \"chr38.g2530\"; transcript_id \"chr38.g2530.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80570468', '80570521', '.', '-', '.', 'gene_id \"chr5.g33163\"; transcript_id \"chr5.g33163.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '6407446', '6407481', '.', '-', '.', 'gene_id \"chr38.g2639\"; transcript_id \"chr38.g2639.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80711728', '80711992', '.', '+', '.', 'gene_id \"chr5.g33289\"; transcript_id \"chr5.g33289.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80713258', '80713509', '.', '-', '.', 'gene_id \"chr5.g33294\"; transcript_id \"chr5.g33294.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80771099', '80771402', '.', '+', '.', 'gene_id \"chr5.g33307\"; transcript_id \"chr5.g33307.i5\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80894714', '80894743', '.', '+', '.', 'gene_id \"chr5.g33429\"; transcript_id \"chr5.g33429.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '8197422', '8197541', '.', '-', '.', 'gene_id \"chr38.g2792\"; transcript_id \"chr38.g2792.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '8227352', '8227398', '.', '-', '.', 'gene_id \"chr38.g2805\"; transcript_id \"chr38.g2805.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '8595595', '8595794', '.', '+', '.', 'gene_id \"chr38.g2833\"; transcript_id \"chr38.g2833.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '9065338', '9065748', '.', '+', '.', 'gene_id \"chr38.g2848\"; transcript_id \"chr38.g2848.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '9384494', '9384523', '.', '-', '.', 'gene_id \"chr38.g2865\"; transcript_id \"chr38.g2865.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81030893', '81031066', '.', '+', '.', 'gene_id \"chr5.g33527\"; transcript_id \"chr5.g33527.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81031232', '81031410', '.', '-', '.', 'gene_id \"chr5.g33528\"; transcript_id \"chr5.g33528.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '10028779', '10028823', '.', '-', '.', 'gene_id \"chr38.g2918\"; transcript_id \"chr38.g2918.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81090869', '81091081', '.', '+', '.', 'gene_id \"chr5.g33576\"; transcript_id \"chr5.g33576.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '10554086', '10554128', '.', '-', '.', 'gene_id \"chr38.g2949\"; transcript_id \"chr38.g2949.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81268383', '81269091', '.', '+', '.', 'gene_id \"chr5.g33701\"; transcript_id \"chr5.g33701.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81268398', '81269091', '.', '+', '.', 'gene_id \"chr5.g33701\"; transcript_id \"chr5.g33701.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '11165422', '11165648', '.', '+', '.', 'gene_id \"chr38.g3055\"; transcript_id \"chr38.g3055.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '11166821', '11167015', '.', '-', '.', 'gene_id \"chr38.g3058\"; transcript_id \"chr38.g3058.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81323177', '81323569', '.', '-', '.', 'gene_id \"chr5.g33734\"; transcript_id \"chr5.g33734.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12085038', '12085107', '.', '-', '.', 'gene_id \"chr38.g3144\"; transcript_id \"chr38.g3144.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81436779', '81436858', '.', '-', '.', 'gene_id \"chr5.g33819\"; transcript_id \"chr5.g33819.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81483038', '81483073', '.', '+', '.', 'gene_id \"chr5.g33855\"; transcript_id \"chr5.g33855.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12491519', '12491780', '.', '+', '.', 'gene_id \"chr38.g3342\"; transcript_id \"chr38.g3342.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12560913', '12561087', '.', '+', '.', 'gene_id \"chr38.g3344\"; transcript_id \"chr38.g3344.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12620202', '12620465', '.', '+', '.', 'gene_id \"chr38.g3351\"; transcript_id \"chr38.g3351.i2\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12744687', '12744947', '.', '-', '.', 'gene_id \"chr38.g3396\"; transcript_id \"chr38.g3396.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81849561', '81849767', '.', '-', '.', 'gene_id \"chr5.g34182\"; transcript_id \"chr5.g34182.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81849967', '81850210', '.', '+', '.', 'gene_id \"chr5.g34183\"; transcript_id \"chr5.g34183.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81850346', '81850504', '.', '-', '.', 'gene_id \"chr5.g34184\"; transcript_id \"chr5.g34184.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82041137', '82041196', '.', '-', '.', 'gene_id \"chr5.g34384\"; transcript_id \"chr5.g34384.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '13473322', '13473976', '.', '+', '.', 'gene_id \"chr38.g3630\"; transcript_id \"chr38.g3630.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82162160', '82162190', '.', '+', '.', 'gene_id \"chr5.g34516\"; transcript_id \"chr5.g34516.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '14238734', '14238768', '.', '+', '.', 'gene_id \"chr38.g3842\"; transcript_id \"chr38.g3842.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '14522259', '14522669', '.', '+', '.', 'gene_id \"chr38.g3875\"; transcript_id \"chr38.g3875.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82463504', '82463753', '.', '+', '.', 'gene_id \"chr5.g34812\"; transcript_id \"chr5.g34812.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '14978528', '14979530', '.', '+', '.', 'gene_id \"chr38.g4055\"; transcript_id \"chr38.g4055.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '15043944', '15044138', '.', '+', '.', 'gene_id \"chr38.g4140\"; transcript_id \"chr38.g4140.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '15282586', '15282681', '.', '+', '.', 'gene_id \"chr38.g4222\"; transcript_id \"chr38.g4222.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82849058', '82849460', '.', '-', '.', 'gene_id \"chr5.g35069\"; transcript_id \"chr5.g35069.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82849469', '82849759', '.', '-', '.', 'gene_id \"chr5.g35070\"; transcript_id \"chr5.g35070.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '15314951', '15315135', '.', '+', '.', 'gene_id \"chr38.g4229\"; transcript_id \"chr38.g4229.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82969552', '82970499', '.', '-', '.', 'gene_id \"chr5.g35142\"; transcript_id \"chr5.g35142.i7\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82976047', '82976330', '.', '+', '.', 'gene_id \"chr5.g35144\"; transcript_id \"chr5.g35144.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82986081', '82986355', '.', '+', '.', 'gene_id \"chr5.g35145\"; transcript_id \"chr5.g35145.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82971203', '82971258', '.', '-', '.', 'gene_id \"chr5.g35146\"; transcript_id \"chr5.g35146.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82975759', '82975910', '.', '+', '.', 'gene_id \"chr5.g35157\"; transcript_id \"chr5.g35157.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82982060', '82982290', '.', '+', '.', 'gene_id \"chr5.g35167\"; transcript_id \"chr5.g35167.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82984011', '82984124', '.', '-', '.', 'gene_id \"chr5.g35172\"; transcript_id \"chr5.g35172.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83004470', '83005040', '.', '-', '.', 'gene_id \"chr5.g35200\"; transcript_id \"chr5.g35200.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83005040', '83005366', '.', '+', '.', 'gene_id \"chr5.g35201\"; transcript_id \"chr5.g35201.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83067302', '83067350', '.', '+', '.', 'gene_id \"chr5.g35230\"; transcript_id \"chr5.g35230.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83109267', '83109301', '.', '-', '.', 'gene_id \"chr5.g35304\"; transcript_id \"chr5.g35304.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83294126', '83294416', '.', '+', '.', 'gene_id \"chr5.g35390\"; transcript_id \"chr5.g35390.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83393322', '83393354', '.', '+', '.', 'gene_id \"chr5.g35402\"; transcript_id \"chr5.g35402.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83608759', '83608862', '.', '+', '.', 'gene_id \"chr5.g35409\"; transcript_id \"chr5.g35409.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '16693393', '16693426', '.', '-', '.', 'gene_id \"chr38.g4600\"; transcript_id \"chr38.g4600.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '16787397', '16787506', '.', '-', '.', 'gene_id \"chr38.g4610\"; transcript_id \"chr38.g4610.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83999830', '83999872', '.', '-', '.', 'gene_id \"chr5.g35442\"; transcript_id \"chr5.g35442.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17009060', '17009277', '.', '+', '.', 'gene_id \"chr38.g4740\"; transcript_id \"chr38.g4740.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '85135115', '85135160', '.', '+', '.', 'gene_id \"chr5.g35569\"; transcript_id \"chr5.g35569.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '86105686', '86105954', '.', '+', '.', 'gene_id \"chr5.g35599\"; transcript_id \"chr5.g35599.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17320273', '17320386', '.', '+', '.', 'gene_id \"chr38.g4835\"; transcript_id \"chr38.g4835.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87019515', '87019556', '.', '-', '.', 'gene_id \"chr5.g35680\"; transcript_id \"chr5.g35680.i2\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87454267', '87454308', '.', '+', '.', 'gene_id \"chr5.g35703\"; transcript_id \"chr5.g35703.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87506209', '87506270', '.', '+', '.', 'gene_id \"chr5.g35707\"; transcript_id \"chr5.g35707.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '88048209', '88048272', '.', '-', '.', 'gene_id \"chr5.g35728\"; transcript_id \"chr5.g35728.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17585101', '17585150', '.', '+', '.', 'gene_id \"chr38.g4972\"; transcript_id \"chr38.g4972.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '89205324', '89205496', '.', '+', '.', 'gene_id \"chr5.g35825\"; transcript_id \"chr5.g35825.i1\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1875616', '1875776', '100', '+', '.', 'gene_id \"chr5.g35863\"; transcript_id \"blat.proc4005.chain_5739\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75286367', '75286400', '100', '-', '.', 'gene_id \"chr5.g35881\"; transcript_id \"blat.proc4587.chain_1118\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6192594', '6192839', '100', '+', '.', 'gene_id \"chr5.g35893\"; transcript_id \"blat.proc4005.chain_23128\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76968640', '76968815', '100', '-', '.', 'gene_id \"chr5.g35918\"; transcript_id \"blat.proc4005.chain_2924\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '38718061', '38718259', '100', '+', '.', 'gene_id \"chr5.g35921\"; transcript_id \"blat.proc18128.chain_25570\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67316816', '67316891', '100', '+', '.', 'gene_id \"chr5.g35941\"; transcript_id \"blat.proc4005.chain_8721\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1039828', '1039879', '100', '-', '.', 'gene_id \"chr5.g35957\"; transcript_id \"blat.proc28144.chain_18607\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54238967', '54239125', '100', '-', '.', 'gene_id \"chr5.g35959\"; transcript_id \"blat.proc4587.chain_4645\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10047553', '10047598', '100', '-', '.', 'gene_id \"chr5.g35971\"; transcript_id \"blat.proc28144.chain_16423\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60632', '60792', '100', '+', '.', 'gene_id \"chr5.g35979\"; transcript_id \"blat.proc18128.chain_20226\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68049290', '68049457', '100', '+', '.', 'gene_id \"chr5.g35981\"; transcript_id \"blat.proc4005.chain_9058\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70325358', '70325713', '100', '-', '.', 'gene_id \"chr5.g35995\"; transcript_id \"blat.proc4005.chain_6573\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1071452', '1071693', '100', '-', '.', 'gene_id \"chr5.g36004\"; transcript_id \"blat.proc4005.chain_4352\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76936877', '76937133', '100', '-', '.', 'gene_id \"chr5.g36010\"; transcript_id \"blat.proc4005.chain_2870\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71422217', '71422560', '100', '-', '.', 'gene_id \"chr5.g36022\"; transcript_id \"blat.proc4005.chain_7858\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70154082', '70154239', '100', '+', '.', 'gene_id \"chr5.g36039\"; transcript_id \"blat.proc4005.chain_8809\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67396505', '67396675', '100', '+', '.', 'gene_id \"chr5.g36067\"; transcript_id \"blat.proc4005.chain_6485\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68477667', '68478172', '100', '-', '.', 'gene_id \"chr5.g36089\"; transcript_id \"blat.proc4005.chain_6429\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1607115', '1607144', '100', '+', '.', 'gene_id \"chr5.g36094\"; transcript_id \"blat.proc28144.chain_4527\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68087851', '68088058', '100', '-', '.', 'gene_id \"chr5.g36095\"; transcript_id \"blat.proc4005.chain_5930\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63222768', '63223013', '100', '-', '.', 'gene_id \"chr5.g36096\"; transcript_id \"blat.proc4005.chain_11952\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6018902', '6018960', '100', '-', '.', 'gene_id \"chr5.g36113\"; transcript_id \"blat.proc4005.chain_23690\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73773889', '73774063', '100', '+', '.', 'gene_id \"chr5.g36136\"; transcript_id \"blat.proc4005.chain_3368\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29849412', '29849469', '100', '-', '.', 'gene_id \"chr5.g36144\"; transcript_id \"blat.proc4587.chain_5527\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62323410', '62323580', '100', '+', '.', 'gene_id \"chr5.g36165\"; transcript_id \"blat.proc4005.chain_12079\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80234879', '80234916', '100', '-', '.', 'gene_id \"chr5.g36167\"; transcript_id \"blat.proc28144.chain_9433\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50583972', '50584010', '100', '+', '.', 'gene_id \"chr5.g36208\"; transcript_id \"blat.proc28144.chain_21027\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67191856', '67192025', '100', '-', '.', 'gene_id \"chr5.g36214\"; transcript_id \"blat.proc4587.chain_5357\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70589444', '70589597', '100', '-', '.', 'gene_id \"chr5.g36215\"; transcript_id \"blat.proc4005.chain_11615\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66856591', '66856920', '100', '+', '.', 'gene_id \"chr5.g36223\"; transcript_id \"blat.proc4005.chain_8354\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67783436', '67783624', '100', '-', '.', 'gene_id \"chr5.g36225\"; transcript_id \"blat.proc4005.chain_7010\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72282462', '72282748', '100', '-', '.', 'gene_id \"chr5.g36244\"; transcript_id \"blat.proc4587.chain_14006\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33446747', '33446777', '100', '+', '.', 'gene_id \"chr5.g36280\"; transcript_id \"blat.proc4005.chain_8040\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76705867', '76706045', '100', '-', '.', 'gene_id \"chr5.g36286\"; transcript_id \"blat.proc4005.chain_2558\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77012530', '77012590', '100', '-', '.', 'gene_id \"chr5.g36302\"; transcript_id \"blat.proc4005.chain_3191\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72754593', '72754701', '100', '+', '.', 'gene_id \"chr5.g36314\"; transcript_id \"blat.proc4005.chain_3302\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76075251', '76075429', '100', '+', '.', 'gene_id \"chr5.g36334\"; transcript_id \"blat.proc4005.chain_2719\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76412394', '76412591', '100', '-', '.', 'gene_id \"chr5.g36377\"; transcript_id \"blat.proc4005.chain_2747\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25741212', '25741273', '100', '-', '.', 'gene_id \"chr5.g36385\"; transcript_id \"blat.proc18128.chain_23974\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65982201', '65982385', '100', '-', '.', 'gene_id \"chr5.g36394\"; transcript_id \"blat.proc4005.chain_22064\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67054427', '67054516', '100', '-', '.', 'gene_id \"chr5.g36449\"; transcript_id \"blat.proc4005.chain_7825\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68301928', '68302131', '100', '+', '.', 'gene_id \"chr5.g36454\"; transcript_id \"blat.proc4587.chain_1105\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75419794', '75420002', '100', '+', '.', 'gene_id \"chr5.g36474\"; transcript_id \"blat.proc4005.chain_1039\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62407306', '62407586', '100', '-', '.', 'gene_id \"chr5.g36475\"; transcript_id \"blat.proc4005.chain_13545\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77666382', '77666587', '100', '-', '.', 'gene_id \"chr5.g36481\"; transcript_id \"blat.proc4005.chain_2612\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62792708', '62792875', '100', '+', '.', 'gene_id \"chr5.g36500\"; transcript_id \"blat.proc4005.chain_13745\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75956354', '75956573', '100', '+', '.', 'gene_id \"chr5.g36503\"; transcript_id \"blat.proc4005.chain_2916\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60483307', '60483483', '100', '+', '.', 'gene_id \"chr5.g36507\"; transcript_id \"blat.proc4005.chain_21332\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63437272', '63437425', '100', '+', '.', 'gene_id \"chr5.g36514\"; transcript_id \"blat.proc4005.chain_21665\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '747435', '748007', '100', '+', '.', 'gene_id \"chr5.g36533\"; transcript_id \"blat.proc4005.chain_5194\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61612615', '61612847', '100', '+', '.', 'gene_id \"chr5.g36566\"; transcript_id \"blat.proc4005.chain_11851\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64419138', '64419500', '100', '-', '.', 'gene_id \"chr5.g36570\"; transcript_id \"blat.proc4005.chain_11884\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75949051', '75949189', '100', '-', '.', 'gene_id \"chr5.g36579\"; transcript_id \"blat.proc4005.chain_2650\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76005988', '76006143', '100', '+', '.', 'gene_id \"chr5.g36611\"; transcript_id \"blat.proc4005.chain_56\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65706020', '65706224', '100', '+', '.', 'gene_id \"chr5.g36619\"; transcript_id \"blat.proc18128.chain_7429\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76527769', '76527959', '100', '-', '.', 'gene_id \"chr5.g36630\"; transcript_id \"blat.proc4005.chain_2633\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68020835', '68021033', '100', '-', '.', 'gene_id \"chr5.g36635\"; transcript_id \"blat.proc4005.chain_6497\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64221619', '64221815', '100', '+', '.', 'gene_id \"chr5.g36668\"; transcript_id \"blat.proc4005.chain_21444\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41189591', '41189763', '100', '-', '.', 'gene_id \"chr5.g36672\"; transcript_id \"blat.proc4587.chain_5588\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66016957', '66017297', '100', '+', '.', 'gene_id \"chr5.g36674\"; transcript_id \"blat.proc4005.chain_22535\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67619214', '67619413', '100', '-', '.', 'gene_id \"chr5.g36685\"; transcript_id \"blat.proc4005.chain_7960\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22591603', '22591635', '100', '-', '.', 'gene_id \"chr5.g36688\"; transcript_id \"blat.proc18128.chain_7272\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73419342', '73419414', '100', '-', '.', 'gene_id \"chr5.g36712\"; transcript_id \"blat.proc18128.chain_7614\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75594903', '75595058', '100', '+', '.', 'gene_id \"chr5.g36726\"; transcript_id \"blat.proc4005.chain_2270\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70327507', '70327731', '100', '-', '.', 'gene_id \"chr5.g36738\"; transcript_id \"blat.proc4005.chain_6535\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '35367289', '35367376', '100', '+', '.', 'gene_id \"chr5.g36762\"; transcript_id \"blat.proc28144.chain_12511\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '35800568', '35800600', '100', '-', '.', 'gene_id \"chr5.g36786\"; transcript_id \"blat.proc18128.chain_14286\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61604755', '61604941', '100', '-', '.', 'gene_id \"chr5.g36804\"; transcript_id \"blat.proc4005.chain_12204\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60073125', '60073210', '100', '-', '.', 'gene_id \"chr5.g36815\"; transcript_id \"blat.proc4587.chain_11417\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72361351', '72361595', '100', '+', '.', 'gene_id \"chr5.g36821\"; transcript_id \"blat.proc4005.chain_6276\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75936532', '75936762', '100', '+', '.', 'gene_id \"chr5.g36832\"; transcript_id \"blat.proc18128.chain_6839\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62530166', '62530319', '100', '-', '.', 'gene_id \"chr5.g36853\"; transcript_id \"blat.proc4005.chain_21232\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '800204', '800360', '100', '+', '.', 'gene_id \"chr5.g36865\"; transcript_id \"blat.proc4005.chain_5019\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70583819', '70583981', '100', '+', '.', 'gene_id \"chr5.g36895\"; transcript_id \"blat.proc4005.chain_5768\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63774256', '63774436', '100', '-', '.', 'gene_id \"chr5.g36903\"; transcript_id \"blat.proc4005.chain_20837\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68503411', '68503518', '100', '+', '.', 'gene_id \"chr5.g36917\"; transcript_id \"blat.proc4005.chain_10865\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63338718', '63338873', '100', '-', '.', 'gene_id \"chr5.g36932\"; transcript_id \"blat.proc4005.chain_12419\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3121908', '3122083', '100', '-', '.', 'gene_id \"chr5.g36937\"; transcript_id \"blat.proc4005.chain_22989\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41167004', '41167045', '100', '-', '.', 'gene_id \"chr5.g36947\"; transcript_id \"blat.proc4587.chain_13481\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73318817', '73318980', '100', '+', '.', 'gene_id \"chr5.g36955\"; transcript_id \"blat.proc4005.chain_3048\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42460004', '42460056', '100', '+', '.', 'gene_id \"chr5.g36957\"; transcript_id \"blat.proc28144.chain_1435\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65092394', '65092558', '100', '+', '.', 'gene_id \"chr5.g36982\"; transcript_id \"blat.proc4005.chain_21622\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63547709', '63547879', '100', '-', '.', 'gene_id \"chr5.g37067\"; transcript_id \"blat.proc4005.chain_21268\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75989052', '75989128', '100', '-', '.', 'gene_id \"chr5.g37096\"; transcript_id \"blat.proc4005.chain_2167\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71069677', '71069833', '100', '-', '.', 'gene_id \"chr5.g37101\"; transcript_id \"blat.proc4005.chain_7963\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65721152', '65721388', '100', '-', '.', 'gene_id \"chr5.g37111\"; transcript_id \"blat.proc4005.chain_21717\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63059296', '63059450', '100', '-', '.', 'gene_id \"chr5.g37112\"; transcript_id \"blat.proc4005.chain_21602\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60838823', '60838973', '100', '-', '.', 'gene_id \"chr5.g37127\"; transcript_id \"blat.proc4005.chain_12095\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62448404', '62448574', '100', '-', '.', 'gene_id \"chr5.g37128\"; transcript_id \"blat.proc4005.chain_21597\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76696106', '76696351', '100', '+', '.', 'gene_id \"chr5.g37147\"; transcript_id \"blat.proc4005.chain_2785\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70817888', '70818068', '100', '-', '.', 'gene_id \"chr5.g37168\"; transcript_id \"blat.proc4005.chain_6648\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '38526312', '38526362', '100', '+', '.', 'gene_id \"chr5.g37188\"; transcript_id \"blat.proc28144.chain_21273\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10182889', '10182942', '100', '+', '.', 'gene_id \"chr5.g37195\"; transcript_id \"blat.proc4005.chain_1096\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67430285', '67430505', '100', '-', '.', 'gene_id \"chr5.g37198\"; transcript_id \"blat.proc4005.chain_11489\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5583015', '5583253', '100', '-', '.', 'gene_id \"chr5.g37228\"; transcript_id \"blat.proc4005.chain_23450\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73314029', '73314342', '100', '-', '.', 'gene_id \"chr5.g37240\"; transcript_id \"blat.proc4005.chain_3122\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82737064', '82737136', '100', '+', '.', 'gene_id \"chr5.g37248\"; transcript_id \"blat.proc18128.chain_2599\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6195277', '6195432', '100', '-', '.', 'gene_id \"chr5.g37255\"; transcript_id \"blat.proc4005.chain_23694\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68568672', '68568826', '100', '-', '.', 'gene_id \"chr5.g37293\"; transcript_id \"blat.proc4005.chain_6534\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55116101', '55116135', '100', '-', '.', 'gene_id \"chr5.g37335\"; transcript_id \"blat.proc18128.chain_2342\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40220608', '40220698', '100', '+', '.', 'gene_id \"chr5.g37387\"; transcript_id \"blat.proc4587.chain_6311\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81086757', '81086799', '100', '-', '.', 'gene_id \"chr5.g37413\"; transcript_id \"blat.proc18128.chain_10250\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70513468', '70513793', '100', '+', '.', 'gene_id \"chr5.g37421\"; transcript_id \"blat.proc4005.chain_7041\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67902192', '67902394', '100', '-', '.', 'gene_id \"chr5.g37442\"; transcript_id \"blat.proc4005.chain_7274\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77535076', '77535227', '100', '-', '.', 'gene_id \"chr5.g37451\"; transcript_id \"blat.proc4005.chain_2911\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64534546', '64534659', '100', '-', '.', 'gene_id \"chr5.g37466\"; transcript_id \"blat.proc4005.chain_19688\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60691749', '60691965', '100', '+', '.', 'gene_id \"chr5.g37478\"; transcript_id \"blat.proc4005.chain_19352\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70322602', '70322794', '100', '-', '.', 'gene_id \"chr5.g37479\"; transcript_id \"blat.proc4587.chain_9752\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1710243', '1710290', '100', '-', '.', 'gene_id \"chr5.g37501\"; transcript_id \"blat.proc4587.chain_11904\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78972557', '78972714', '100', '+', '.', 'gene_id \"chr5.g37505\"; transcript_id \"blat.proc4005.chain_2840\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75330726', '75330768', '100', '+', '.', 'gene_id \"chr5.g37507\"; transcript_id \"blat.proc18128.chain_2838\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63207229', '63207388', '100', '-', '.', 'gene_id \"chr5.g37545\"; transcript_id \"blat.proc4005.chain_19693\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62462155', '62462401', '100', '-', '.', 'gene_id \"chr5.g37547\"; transcript_id \"blat.proc4005.chain_21088\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68527380', '68527654', '100', '+', '.', 'gene_id \"chr5.g37584\"; transcript_id \"blat.proc4005.chain_7852\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62828213', '62828644', '100', '+', '.', 'gene_id \"chr5.g37614\"; transcript_id \"blat.proc4005.chain_20715\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67422094', '67422127', '100', '+', '.', 'gene_id \"chr5.g37617\"; transcript_id \"blat.proc4005.chain_6601\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66465696', '66465972', '100', '+', '.', 'gene_id \"chr5.g37673\"; transcript_id \"blat.proc4587.chain_6944\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76443472', '76443626', '100', '-', '.', 'gene_id \"chr5.g37710\"; transcript_id \"blat.proc4005.chain_2693\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5584056', '5584221', '100', '+', '.', 'gene_id \"chr5.g37719\"; transcript_id \"blat.proc4005.chain_23780\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5593192', '5593351', '100', '-', '.', 'gene_id \"chr5.g37733\"; transcript_id \"blat.proc4005.chain_23126\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71551028', '71551458', '100', '+', '.', 'gene_id \"chr5.g37747\"; transcript_id \"blat.proc4005.chain_6375\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68739741', '68739803', '100', '+', '.', 'gene_id \"chr5.g37771\"; transcript_id \"blat.proc28144.chain_965\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78411255', '78411419', '100', '-', '.', 'gene_id \"chr5.g37794\"; transcript_id \"blat.proc28144.chain_18628\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63634818', '63634944', '100', '+', '.', 'gene_id \"chr5.g37817\"; transcript_id \"blat.proc4005.chain_21073\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67925156', '67925321', '100', '+', '.', 'gene_id \"chr5.g37842\"; transcript_id \"blat.proc4005.chain_6360\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72197637', '72197829', '100', '+', '.', 'gene_id \"chr5.g37851\"; transcript_id \"blat.proc4005.chain_6295\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52706930', '52707245', '100', '+', '.', 'gene_id \"chr5.g37860\"; transcript_id \"blat.proc4005.chain_10449\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60147006', '60147163', '100', '+', '.', 'gene_id \"chr5.g37865\"; transcript_id \"blat.proc4005.chain_21630\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72066260', '72066426', '100', '-', '.', 'gene_id \"chr5.g37894\"; transcript_id \"blat.proc4005.chain_8520\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60606334', '60606573', '100', '+', '.', 'gene_id \"chr5.g37911\"; transcript_id \"blat.proc4005.chain_20856\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '22430156', '22430317', '100', '+', '.', 'gene_id \"chr5.g37914\"; transcript_id \"blat.proc18128.chain_5196\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '884469', '884706', '100', '+', '.', 'gene_id \"chr5.g37922\"; transcript_id \"blat.proc4005.chain_4259\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68456138', '68456190', '100', '+', '.', 'gene_id \"chr5.g37929\"; transcript_id \"blat.proc18128.chain_8307\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '37620283', '37620362', '100', '-', '.', 'gene_id \"chr5.g37942\"; transcript_id \"blat.proc18128.chain_6113\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72881477', '72881666', '100', '+', '.', 'gene_id \"chr5.g37959\"; transcript_id \"blat.proc4005.chain_3134\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '48044117', '48044197', '100', '+', '.', 'gene_id \"chr5.g37965\"; transcript_id \"blat.proc4587.chain_11856\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '27454340', '27454377', '100', '-', '.', 'gene_id \"chr5.g37980\"; transcript_id \"blat.proc18128.chain_9924\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5448414', '5448640', '100', '+', '.', 'gene_id \"chr5.g37999\"; transcript_id \"blat.proc4005.chain_23024\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '13735409', '13735461', '100', '+', '.', 'gene_id \"chr5.g38032\"; transcript_id \"blat.proc28144.chain_20036\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67169494', '67169643', '100', '+', '.', 'gene_id \"chr5.g38041\"; transcript_id \"blat.proc4005.chain_6032\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70310228', '70310506', '100', '-', '.', 'gene_id \"chr5.g38046\"; transcript_id \"blat.proc4005.chain_11559\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60312170', '60312470', '100', '+', '.', 'gene_id \"chr5.g38081\"; transcript_id \"blat.proc4005.chain_20770\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54290206', '54290364', '100', '-', '.', 'gene_id \"chr5.g38089\"; transcript_id \"blat.proc4587.chain_4644\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5806542', '5806767', '100', '+', '.', 'gene_id \"chr5.g38095\"; transcript_id \"blat.proc4005.chain_23379\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63571979', '63572186', '100', '-', '.', 'gene_id \"chr5.g38096\"; transcript_id \"blat.proc4587.chain_7596\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '894216', '894373', '100', '-', '.', 'gene_id \"chr5.g38100\"; transcript_id \"blat.proc4005.chain_5536\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5898365', '5898539', '100', '-', '.', 'gene_id \"chr5.g38123\"; transcript_id \"blat.proc4587.chain_13713\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67049820', '67050015', '100', '+', '.', 'gene_id \"chr5.g38124\"; transcript_id \"blat.proc4005.chain_11511\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '55453670', '55453712', '100', '+', '.', 'gene_id \"chr5.g38138\"; transcript_id \"blat.proc4587.chain_11897\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68570208', '68570458', '100', '-', '.', 'gene_id \"chr5.g38139\"; transcript_id \"blat.proc4005.chain_7970\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87623632', '87623663', '100', '-', '.', 'gene_id \"chr5.g38143\"; transcript_id \"blat.proc4005.chain_6217\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '28808573', '28808617', '100', '-', '.', 'gene_id \"chr5.g38179\"; transcript_id \"blat.proc28144.chain_9743\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6841968', '6842025', '100', '-', '.', 'gene_id \"chr5.g38207\"; transcript_id \"blat.proc18128.chain_24546\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62112991', '62113333', '100', '+', '.', 'gene_id \"chr5.g38232\"; transcript_id \"blat.proc4005.chain_13570\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68004062', '68004145', '100', '+', '.', 'gene_id \"chr5.g38237\"; transcript_id \"blat.proc4005.chain_10727\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72800333', '72800514', '100', '-', '.', 'gene_id \"chr5.g38251\"; transcript_id \"blat.proc4005.chain_3703\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '58738374', '58738571', '100', '+', '.', 'gene_id \"chr5.g38269\"; transcript_id \"blat.proc4005.chain_12422\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67153970', '67154226', '100', '+', '.', 'gene_id \"chr5.g38282\"; transcript_id \"blat.proc4005.chain_11148\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67310075', '67310236', '100', '+', '.', 'gene_id \"chr5.g38287\"; transcript_id \"blat.proc4005.chain_5812\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60099006', '60099169', '100', '+', '.', 'gene_id \"chr5.g38317\"; transcript_id \"blat.proc4005.chain_21297\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63181845', '63182011', '100', '-', '.', 'gene_id \"chr5.g38338\"; transcript_id \"blat.proc4005.chain_12170\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '30719277', '30719348', '100', '+', '.', 'gene_id \"chr5.g38340\"; transcript_id \"blat.proc18128.chain_11795\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64340730', '64340935', '100', '+', '.', 'gene_id \"chr5.g38349\"; transcript_id \"blat.proc4005.chain_12916\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '82796659', '82796719', '100', '+', '.', 'gene_id \"chr5.g38357\"; transcript_id \"blat.proc28144.chain_15649\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61647920', '61648113', '100', '+', '.', 'gene_id \"chr5.g38380\"; transcript_id \"blat.proc4005.chain_12458\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63106622', '63106782', '100', '-', '.', 'gene_id \"chr5.g38394\"; transcript_id \"blat.proc4005.chain_21117\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73056417', '73056679', '100', '+', '.', 'gene_id \"chr5.g38406\"; transcript_id \"blat.proc4005.chain_3817\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24034676', '24034717', '100', '+', '.', 'gene_id \"chr5.g38411\"; transcript_id \"blat.proc28144.chain_19604\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40068144', '40068184', '100', '+', '.', 'gene_id \"chr5.g38428\"; transcript_id \"blat.proc28144.chain_3279\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5468974', '5469394', '100', '+', '.', 'gene_id \"chr5.g38494\"; transcript_id \"blat.proc4005.chain_23108\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65561111', '65561241', '100', '+', '.', 'gene_id \"chr5.g38523\"; transcript_id \"blat.proc4005.chain_21892\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77941254', '77941434', '100', '-', '.', 'gene_id \"chr5.g38544\"; transcript_id \"blat.proc4005.chain_2864\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6774598', '6774628', '100', '-', '.', 'gene_id \"chr5.g38549\"; transcript_id \"blat.proc4005.chain_13075\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78484029', '78484200', '100', '+', '.', 'gene_id \"chr5.g38550\"; transcript_id \"blat.proc4005.chain_2076\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65014856', '65015127', '100', '+', '.', 'gene_id \"chr5.g38576\"; transcript_id \"blat.proc4005.chain_21009\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64150143', '64150274', '100', '+', '.', 'gene_id \"chr5.g38614\"; transcript_id \"blat.proc4005.chain_15056\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76390105', '76390308', '100', '+', '.', 'gene_id \"chr5.g38621\"; transcript_id \"blat.proc4005.chain_1519\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49806164', '49806223', '100', '-', '.', 'gene_id \"chr5.g38622\"; transcript_id \"blat.proc4587.chain_3256\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '27683265', '27683314', '100', '+', '.', 'gene_id \"chr5.g38631\"; transcript_id \"blat.proc4005.chain_21251\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29389540', '29389651', '100', '-', '.', 'gene_id \"chr5.g38637\"; transcript_id \"blat.proc18128.chain_16742\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67805449', '67805601', '100', '+', '.', 'gene_id \"chr5.g38654\"; transcript_id \"blat.proc4005.chain_11647\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71132166', '71132343', '100', '+', '.', 'gene_id \"chr5.g38658\"; transcript_id \"blat.proc4005.chain_6422\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65059724', '65059924', '100', '-', '.', 'gene_id \"chr5.g38675\"; transcript_id \"blat.proc18128.chain_13046\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67817009', '67817259', '100', '+', '.', 'gene_id \"chr5.g38741\"; transcript_id \"blat.proc4005.chain_5979\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70263779', '70264007', '100', '-', '.', 'gene_id \"chr5.g38751\"; transcript_id \"blat.proc4005.chain_10291\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66096016', '66096213', '100', '-', '.', 'gene_id \"chr5.g38848\"; transcript_id \"blat.proc4005.chain_22911\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50868297', '50868353', '100', '-', '.', 'gene_id \"chr5.g38882\"; transcript_id \"blat.proc4005.chain_23547\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1216094', '1216283', '100', '+', '.', 'gene_id \"chr5.g38903\"; transcript_id \"blat.proc4005.chain_4494\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76388746', '76388938', '100', '+', '.', 'gene_id \"chr5.g38915\"; transcript_id \"blat.proc4005.chain_1911\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65089907', '65090171', '100', '-', '.', 'gene_id \"chr5.g38920\"; transcript_id \"blat.proc18128.chain_13039\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75975037', '75975083', '100', '+', '.', 'gene_id \"chr5.g38952\"; transcript_id \"blat.proc28144.chain_1436\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '26034075', '26034256', '100', '+', '.', 'gene_id \"chr5.g38958\"; transcript_id \"blat.proc28144.chain_4382\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61767658', '61767742', '100', '+', '.', 'gene_id \"chr5.g39036\"; transcript_id \"blat.proc4587.chain_6118\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75716130', '75716282', '100', '-', '.', 'gene_id \"chr5.g39068\"; transcript_id \"blat.proc4005.chain_2599\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67149763', '67149833', '100', '-', '.', 'gene_id \"chr5.g39070\"; transcript_id \"blat.proc4005.chain_10339\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54238786', '54238940', '100', '+', '.', 'gene_id \"chr5.g39071\"; transcript_id \"blat.proc4587.chain_4647\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6005186', '6005385', '100', '+', '.', 'gene_id \"chr5.g39085\"; transcript_id \"blat.proc4005.chain_23621\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73531376', '73531531', '100', '+', '.', 'gene_id \"chr5.g39095\"; transcript_id \"blat.proc28144.chain_6219\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63083575', '63083717', '100', '-', '.', 'gene_id \"chr5.g39118\"; transcript_id \"blat.proc4005.chain_12417\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5495445', '5495617', '100', '-', '.', 'gene_id \"chr5.g39125\"; transcript_id \"blat.proc4005.chain_23419\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61060569', '61060670', '100', '-', '.', 'gene_id \"chr5.g39130\"; transcript_id \"blat.proc4005.chain_21025\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67781536', '67781738', '100', '-', '.', 'gene_id \"chr5.g39159\"; transcript_id \"blat.proc4005.chain_11403\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63876083', '63876235', '100', '+', '.', 'gene_id \"chr5.g39172\"; transcript_id \"blat.proc4005.chain_18263\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72439398', '72439559', '100', '-', '.', 'gene_id \"chr5.g39173\"; transcript_id \"blat.proc4005.chain_6383\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76831665', '76831885', '100', '+', '.', 'gene_id \"chr5.g39179\"; transcript_id \"blat.proc4005.chain_2587\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '51808879', '51808925', '100', '-', '.', 'gene_id \"chr5.g39197\"; transcript_id \"blat.proc28144.chain_14748\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67792733', '67792945', '100', '+', '.', 'gene_id \"chr5.g39207\"; transcript_id \"blat.proc4005.chain_6261\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63175791', '63175946', '100', '-', '.', 'gene_id \"chr5.g39213\"; transcript_id \"blat.proc4005.chain_12367\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62863034', '62863214', '100', '-', '.', 'gene_id \"chr5.g39233\"; transcript_id \"blat.proc4005.chain_21543\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80185640', '80185694', '100', '-', '.', 'gene_id \"chr5.g39248\"; transcript_id \"blat.proc18128.chain_9959\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64544581', '64544731', '100', '-', '.', 'gene_id \"chr5.g39285\"; transcript_id \"blat.proc4005.chain_13997\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60659431', '60659640', '100', '-', '.', 'gene_id \"chr5.g39287\"; transcript_id \"blat.proc4005.chain_20797\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '49838482', '49838526', '100', '+', '.', 'gene_id \"chr5.g39295\"; transcript_id \"blat.proc28144.chain_11371\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74156730', '74156961', '100', '-', '.', 'gene_id \"chr5.g39308\"; transcript_id \"blat.proc4005.chain_3126\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61720641', '61720815', '100', '-', '.', 'gene_id \"chr5.g39319\"; transcript_id \"blat.proc4005.chain_12122\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4767776', '4767964', '100', '-', '.', 'gene_id \"chr5.g39352\"; transcript_id \"blat.proc4005.chain_23127\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69166227', '69166317', '100', '-', '.', 'gene_id \"chr5.g39353\"; transcript_id \"blat.proc4587.chain_4719\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77418707', '77418803', '100', '+', '.', 'gene_id \"chr5.g39393\"; transcript_id \"blat.proc4005.chain_2690\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75500054', '75500405', '100', '+', '.', 'gene_id \"chr5.g39420\"; transcript_id \"blat.proc4005.chain_2655\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61770102', '61770257', '100', '-', '.', 'gene_id \"chr5.g39454\"; transcript_id \"blat.proc4005.chain_12347\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60241776', '60242030', '100', '+', '.', 'gene_id \"chr5.g39481\"; transcript_id \"blat.proc4005.chain_20956\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4801946', '4802042', '100', '+', '.', 'gene_id \"chr5.g39524\"; transcript_id \"blat.proc4005.chain_23279\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5984240', '5984514', '100', '-', '.', 'gene_id \"chr5.g39544\"; transcript_id \"blat.proc4005.chain_23026\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50834994', '50835031', '100', '-', '.', 'gene_id \"chr5.g39564\"; transcript_id \"blat.proc28144.chain_11978\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29098266', '29098363', '100', '-', '.', 'gene_id \"chr5.g39566\"; transcript_id \"blat.proc18128.chain_15664\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5278855', '5278917', '100', '-', '.', 'gene_id \"chr5.g39595\"; transcript_id \"blat.proc4587.chain_9321\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63686375', '63686551', '100', '+', '.', 'gene_id \"chr5.g39604\"; transcript_id \"blat.proc4005.chain_19705\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64877526', '64877711', '100', '+', '.', 'gene_id \"chr5.g39611\"; transcript_id \"blat.proc4005.chain_20996\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71943069', '71943237', '100', '+', '.', 'gene_id \"chr5.g39613\"; transcript_id \"blat.proc4005.chain_6280\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63460637', '63460723', '100', '-', '.', 'gene_id \"chr5.g39698\"; transcript_id \"blat.proc4005.chain_20483\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39542400', '39542433', '100', '-', '.', 'gene_id \"chr5.g39706\"; transcript_id \"blat.proc4587.chain_12122\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63572237', '63572492', '100', '-', '.', 'gene_id \"chr5.g39719\"; transcript_id \"blat.proc4005.chain_13050\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68252581', '68252762', '100', '+', '.', 'gene_id \"chr5.g39743\"; transcript_id \"blat.proc4587.chain_6887\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr5', 'PASA', 'exon', '76303282', '76303449', '100', '-', '.', 'gene_id \"chr5.g39771\"; transcript_id \"blat.proc4005.chain_2319\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76567371', '76567523', '100', '+', '.', 'gene_id \"chr5.g39805\"; transcript_id \"blat.proc4005.chain_2326\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70869537', '70869587', '100', '+', '.', 'gene_id \"chr5.g39806\"; transcript_id \"blat.proc28144.chain_8367\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68230878', '68231037', '100', '+', '.', 'gene_id \"chr5.g39807\"; transcript_id \"blat.proc4005.chain_5810\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61781198', '61781380', '100', '+', '.', 'gene_id \"chr5.g39817\"; transcript_id \"blat.proc4005.chain_21213\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5898804', '5898977', '100', '-', '.', 'gene_id \"chr5.g39845\"; transcript_id \"blat.proc4587.chain_13714\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61709212', '61709461', '100', '+', '.', 'gene_id \"chr5.g39850\"; transcript_id \"blat.proc4587.chain_1240\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '50034517', '50034550', '100', '+', '.', 'gene_id \"chr5.g39852\"; transcript_id \"blat.proc28144.chain_22473\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66766105', '66766343', '100', '+', '.', 'gene_id \"chr5.g39878\"; transcript_id \"blat.proc4005.chain_11191\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65698339', '65698523', '100', '-', '.', 'gene_id \"chr5.g39907\"; transcript_id \"blat.proc4005.chain_21697\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76027839', '76028054', '100', '-', '.', 'gene_id \"chr5.g39932\"; transcript_id \"blat.proc4005.chain_2286\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62382180', '62382382', '100', '+', '.', 'gene_id \"chr5.g39936\"; transcript_id \"blat.proc4005.chain_21118\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '26127285', '26127513', '100', '-', '.', 'gene_id \"chr5.g40064\"; transcript_id \"blat.proc18128.chain_4106\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63732421', '63732571', '100', '-', '.', 'gene_id \"chr5.g40141\"; transcript_id \"blat.proc4005.chain_12618\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64892980', '64893146', '100', '+', '.', 'gene_id \"chr5.g40167\"; transcript_id \"blat.proc4005.chain_20939\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75685366', '75685531', '100', '-', '.', 'gene_id \"chr5.g40173\"; transcript_id \"blat.proc4005.chain_2630\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '80973040', '80973082', '100', '+', '.', 'gene_id \"chr5.g40212\"; transcript_id \"blat.proc4005.chain_20486\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63216446', '63216685', '100', '+', '.', 'gene_id \"chr5.g40312\"; transcript_id \"blat.proc4005.chain_12002\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4934853', '4935059', '100', '-', '.', 'gene_id \"chr5.g40346\"; transcript_id \"blat.proc4005.chain_23103\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60587099', '60587131', '100', '+', '.', 'gene_id \"chr5.g40352\"; transcript_id \"blat.proc4005.chain_10185\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52922375', '52922494', '100', '+', '.', 'gene_id \"chr5.g40376\"; transcript_id \"blat.proc18128.chain_17150\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '35258909', '35258950', '100', '+', '.', 'gene_id \"chr5.g40393\"; transcript_id \"blat.proc18128.chain_19515\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74642150', '74642349', '100', '+', '.', 'gene_id \"chr5.g40406\"; transcript_id \"blat.proc18128.chain_14946\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76585749', '76585903', '100', '+', '.', 'gene_id \"chr5.g40415\"; transcript_id \"blat.proc4005.chain_2440\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63793164', '63793351', '100', '-', '.', 'gene_id \"chr5.g40432\"; transcript_id \"blat.proc4005.chain_13829\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63760173', '63760335', '100', '+', '.', 'gene_id \"chr5.g40433\"; transcript_id \"blat.proc4005.chain_12087\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60483546', '60483658', '100', '+', '.', 'gene_id \"chr5.g40505\"; transcript_id \"blat.proc4005.chain_20967\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64348915', '64349090', '100', '+', '.', 'gene_id \"chr5.g40506\"; transcript_id \"blat.proc4005.chain_12473\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5156806', '5156978', '100', '-', '.', 'gene_id \"chr5.g40513\"; transcript_id \"blat.proc18128.chain_21788\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62652197', '62652388', '100', '-', '.', 'gene_id \"chr5.g40519\"; transcript_id \"blat.proc4587.chain_12469\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81169824', '81169867', '100', '+', '.', 'gene_id \"chr5.g40533\"; transcript_id \"blat.proc18128.chain_10653\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39689754', '39689784', '100', '-', '.', 'gene_id \"chr5.g40566\"; transcript_id \"blat.proc28144.chain_18027\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70309399', '70309628', '100', '+', '.', 'gene_id \"chr5.g40604\"; transcript_id \"blat.proc4005.chain_11424\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70878901', '70878989', '100', '+', '.', 'gene_id \"chr5.g40625\"; transcript_id \"blat.proc28144.chain_7635\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '4593540', '4593723', '100', '-', '.', 'gene_id \"chr5.g40664\"; transcript_id \"blat.proc4005.chain_23220\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63979773', '63979947', '100', '+', '.', 'gene_id \"chr5.g40679\"; transcript_id \"blat.proc4005.chain_11921\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75452696', '75452783', '100', '-', '.', 'gene_id \"chr5.g40703\"; transcript_id \"blat.proc18128.chain_24143\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64580969', '64581119', '100', '-', '.', 'gene_id \"chr5.g40707\"; transcript_id \"blat.proc4005.chain_21211\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '17469565', '17469716', '100', '-', '.', 'gene_id \"chr5.g40708\"; transcript_id \"blat.proc18128.chain_5137\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73314343', '73314554', '100', '-', '.', 'gene_id \"chr5.g40714\"; transcript_id \"blat.proc4005.chain_3121\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63448931', '63449315', '100', '-', '.', 'gene_id \"chr5.g40715\"; transcript_id \"blat.proc4005.chain_21384\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64012077', '64012279', '100', '-', '.', 'gene_id \"chr5.g40718\"; transcript_id \"blat.proc4005.chain_12167\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63197668', '63197840', '100', '+', '.', 'gene_id \"chr5.g40728\"; transcript_id \"blat.proc4587.chain_8570\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63735055', '63735245', '100', '+', '.', 'gene_id \"chr5.g40736\"; transcript_id \"blat.proc4005.chain_12093\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '59983914', '59984177', '100', '+', '.', 'gene_id \"chr5.g40751\"; transcript_id \"blat.proc4005.chain_12322\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63234977', '63235162', '100', '-', '.', 'gene_id \"chr5.g40754\"; transcript_id \"blat.proc4005.chain_14202\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75814120', '75814274', '100', '+', '.', 'gene_id \"chr5.g40755\"; transcript_id \"blat.proc4005.chain_2944\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68015901', '68016068', '100', '+', '.', 'gene_id \"chr5.g40772\"; transcript_id \"blat.proc4005.chain_6484\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75701791', '75701863', '100', '-', '.', 'gene_id \"chr5.g40773\"; transcript_id \"blat.proc28144.chain_11667\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66634019', '66634330', '100', '+', '.', 'gene_id \"chr5.g40785\"; transcript_id \"blat.proc4005.chain_11343\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67647456', '67647605', '100', '-', '.', 'gene_id \"chr5.g40786\"; transcript_id \"blat.proc4005.chain_11417\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75572953', '75573203', '100', '+', '.', 'gene_id \"chr5.g40803\"; transcript_id \"blat.proc4005.chain_2468\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67893792', '67893967', '100', '-', '.', 'gene_id \"chr5.g40822\"; transcript_id \"blat.proc4005.chain_6102\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '33908953', '33909099', '100', '+', '.', 'gene_id \"chr5.g40837\"; transcript_id \"blat.proc4587.chain_5797\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72794613', '72794780', '100', '-', '.', 'gene_id \"chr5.g40847\"; transcript_id \"blat.proc4005.chain_3047\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61854050', '61854248', '100', '+', '.', 'gene_id \"chr5.g40882\"; transcript_id \"blat.proc4005.chain_12768\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '653777', '653929', '100', '+', '.', 'gene_id \"chr5.g40910\"; transcript_id \"blat.proc4005.chain_5680\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24797123', '24797162', '100', '-', '.', 'gene_id \"chr5.g40913\"; transcript_id \"blat.proc28144.chain_23758\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '31176435', '31176478', '100', '+', '.', 'gene_id \"chr5.g40922\"; transcript_id \"blat.proc28144.chain_6562\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62415541', '62415573', '100', '+', '.', 'gene_id \"chr5.g40951\"; transcript_id \"blat.proc4005.chain_12459\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79048400', '79048464', '100', '+', '.', 'gene_id \"chr5.g41017\"; transcript_id \"blat.proc4005.chain_1272\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64847783', '64847933', '100', '+', '.', 'gene_id \"chr5.g41077\"; transcript_id \"blat.proc4005.chain_12405\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64349193', '64349403', '100', '-', '.', 'gene_id \"chr5.g41112\"; transcript_id \"blat.proc4005.chain_12474\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71212981', '71213122', '100', '-', '.', 'gene_id \"chr5.g41137\"; transcript_id \"blat.proc4005.chain_10661\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '259063', '259221', '100', '+', '.', 'gene_id \"chr5.g41151\"; transcript_id \"blat.proc18128.chain_25783\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63760522', '63760676', '100', '+', '.', 'gene_id \"chr5.g41167\"; transcript_id \"blat.proc4005.chain_12088\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70634540', '70634764', '100', '+', '.', 'gene_id \"chr5.g41193\"; transcript_id \"blat.proc4005.chain_5843\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54290025', '54290179', '100', '+', '.', 'gene_id \"chr5.g41233\"; transcript_id \"blat.proc4587.chain_4646\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67777789', '67778091', '100', '-', '.', 'gene_id \"chr5.g41236\"; transcript_id \"blat.proc4005.chain_9799\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63434966', '63435165', '100', '-', '.', 'gene_id \"chr5.g41278\"; transcript_id \"blat.proc4587.chain_1817\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78410452', '78411020', '100', '+', '.', 'gene_id \"chr5.g41353\"; transcript_id \"blat.proc28144.chain_18623\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63481924', '63482054', '100', '+', '.', 'gene_id \"chr5.g41358\"; transcript_id \"blat.proc4005.chain_17176\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64514328', '64514530', '100', '-', '.', 'gene_id \"chr5.g41410\"; transcript_id \"blat.proc4005.chain_13465\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60497150', '60497203', '100', '+', '.', 'gene_id \"chr5.g41421\"; transcript_id \"blat.proc28144.chain_1071\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64787611', '64787801', '100', '+', '.', 'gene_id \"chr5.g41422\"; transcript_id \"blat.proc4005.chain_14492\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71116474', '71116655', '100', '+', '.', 'gene_id \"chr5.g41476\"; transcript_id \"blat.proc4005.chain_5894\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66889024', '66889201', '100', '+', '.', 'gene_id \"chr5.g41487\"; transcript_id \"blat.proc4005.chain_6317\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68457343', '68457598', '100', '-', '.', 'gene_id \"chr5.g41493\"; transcript_id \"blat.proc4005.chain_6591\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78000074', '78000459', '100', '+', '.', 'gene_id \"chr5.g41520\"; transcript_id \"blat.proc4005.chain_1197\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47504889', '47504930', '100', '+', '.', 'gene_id \"chr5.g41532\"; transcript_id \"blat.proc4005.chain_21635\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65924748', '65924917', '100', '+', '.', 'gene_id \"chr5.g41542\"; transcript_id \"blat.proc4005.chain_22590\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65658549', '65658709', '100', '-', '.', 'gene_id \"chr5.g41556\"; transcript_id \"blat.proc4005.chain_22962\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61515895', '61516046', '100', '+', '.', 'gene_id \"chr5.g41560\"; transcript_id \"blat.proc18128.chain_6617\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66582328', '66582700', '100', '+', '.', 'gene_id \"chr5.g41593\"; transcript_id \"blat.proc4005.chain_5926\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '23315378', '23315418', '100', '+', '.', 'gene_id \"chr5.g41607\"; transcript_id \"blat.proc18128.chain_13846\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66465338', '66465555', '100', '+', '.', 'gene_id \"chr5.g41613\"; transcript_id \"blat.proc4587.chain_6943\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79017961', '79018229', '100', '+', '.', 'gene_id \"chr5.g41635\"; transcript_id \"blat.proc18128.chain_6177\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5977088', '5977311', '100', '-', '.', 'gene_id \"chr5.g41654\"; transcript_id \"blat.proc4005.chain_23302\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62434656', '62434812', '100', '+', '.', 'gene_id \"chr5.g41656\"; transcript_id \"blat.proc4005.chain_21504\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63514962', '63515133', '100', '-', '.', 'gene_id \"chr5.g41660\"; transcript_id \"blat.proc4005.chain_21529\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '988072', '988677', '100', '+', '.', 'gene_id \"chr5.g41672\"; transcript_id \"blat.proc4005.chain_4801\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5913692', '5913869', '100', '-', '.', 'gene_id \"chr5.g41695\"; transcript_id \"blat.proc4005.chain_23705\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71549745', '71550170', '100', '+', '.', 'gene_id \"chr5.g41714\"; transcript_id \"blat.proc4587.chain_11195\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70513907', '70514204', '100', '-', '.', 'gene_id \"chr5.g41775\"; transcript_id \"blat.proc4005.chain_7040\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77788875', '77789218', '100', '-', '.', 'gene_id \"chr5.g41910\"; transcript_id \"blat.proc4005.chain_2301\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65882577', '65882797', '100', '+', '.', 'gene_id \"chr5.g41937\"; transcript_id \"blat.proc4005.chain_21727\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73471672', '73471856', '100', '-', '.', 'gene_id \"chr5.g41951\"; transcript_id \"blat.proc18128.chain_6461\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '27523316', '27523393', '100', '-', '.', 'gene_id \"chr5.g41958\"; transcript_id \"blat.proc18128.chain_11203\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '54956039', '54956083', '100', '+', '.', 'gene_id \"chr5.g41977\"; transcript_id \"blat.proc18128.chain_9826\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66392881', '66393124', '100', '+', '.', 'gene_id \"chr5.g41984\"; transcript_id \"blat.proc4005.chain_11745\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70257402', '70257624', '100', '-', '.', 'gene_id \"chr5.g41999\"; transcript_id \"blat.proc4005.chain_11395\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76482739', '76482897', '100', '+', '.', 'gene_id \"chr5.g42002\"; transcript_id \"blat.proc4005.chain_2797\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1338258', '1338540', '100', '-', '.', 'gene_id \"chr5.g42046\"; transcript_id \"blat.proc4005.chain_5669\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76811476', '76811817', '100', '+', '.', 'gene_id \"chr5.g42087\"; transcript_id \"blat.proc4005.chain_2748\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76145222', '76145400', '100', '+', '.', 'gene_id \"chr5.g42102\"; transcript_id \"blat.proc4005.chain_2541\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62544768', '62544802', '100', '-', '.', 'gene_id \"chr5.g42103\"; transcript_id \"blat.proc28144.chain_3193\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71424599', '71424911', '100', '-', '.', 'gene_id \"chr5.g42106\"; transcript_id \"blat.proc4005.chain_6382\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67193631', '67193826', '100', '+', '.', 'gene_id \"chr5.g42107\"; transcript_id \"blat.proc4587.chain_8651\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78036218', '78036414', '100', '+', '.', 'gene_id \"chr5.g42119\"; transcript_id \"blat.proc4005.chain_2476\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '29399753', '29399864', '100', '-', '.', 'gene_id \"chr5.g42145\"; transcript_id \"blat.proc18128.chain_16743\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62389822', '62390037', '100', '+', '.', 'gene_id \"chr5.g42190\"; transcript_id \"blat.proc4005.chain_21673\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70287166', '70287379', '100', '-', '.', 'gene_id \"chr5.g42191\"; transcript_id \"blat.proc4005.chain_11472\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '23353887', '23354106', '100', '+', '.', 'gene_id \"chr5.g42287\"; transcript_id \"blat.proc18128.chain_25591\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '26485118', '26485160', '100', '+', '.', 'gene_id \"chr5.g42318\"; transcript_id \"blat.proc18128.chain_20712\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62252328', '62252513', '100', '-', '.', 'gene_id \"chr5.g42324\"; transcript_id \"blat.proc4005.chain_12158\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67641113', '67641267', '100', '+', '.', 'gene_id \"chr5.g42350\"; transcript_id \"blat.proc4005.chain_11505\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '36973363', '36973398', '100', '+', '.', 'gene_id \"chr5.g42405\"; transcript_id \"blat.proc4005.chain_6006\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '32300325', '32300367', '100', '+', '.', 'gene_id \"chr5.g42411\"; transcript_id \"blat.proc28144.chain_10615\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5014683', '5014861', '100', '+', '.', 'gene_id \"chr5.g42416\"; transcript_id \"blat.proc4005.chain_23117\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '46790285', '46790325', '100', '-', '.', 'gene_id \"chr5.g42446\"; transcript_id \"blat.proc4587.chain_2640\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '47588277', '47588325', '100', '+', '.', 'gene_id \"chr5.g42448\"; transcript_id \"blat.proc28144.chain_4721\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71154205', '71154550', '100', '-', '.', 'gene_id \"chr5.g42495\"; transcript_id \"blat.proc4005.chain_6520\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70310933', '70311111', '100', '-', '.', 'gene_id \"chr5.g42522\"; transcript_id \"blat.proc4005.chain_11299\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1119538', '1119744', '100', '+', '.', 'gene_id \"chr5.g42559\"; transcript_id \"blat.proc4005.chain_5717\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63138540', '63138692', '100', '-', '.', 'gene_id \"chr5.g42566\"; transcript_id \"blat.proc4005.chain_12668\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74330679', '74330718', '100', '+', '.', 'gene_id \"chr5.g42575\"; transcript_id \"blat.proc18128.chain_18992\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '89196371', '89196528', '100', '-', '.', 'gene_id \"chr5.g42576\"; transcript_id \"blat.proc28144.chain_12279\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77844171', '77844325', '100', '-', '.', 'gene_id \"chr5.g42582\"; transcript_id \"blat.proc4005.chain_2849\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '19801674', '19801739', '100', '+', '.', 'gene_id \"chr5.g42613\"; transcript_id \"blat.proc18128.chain_14963\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62426262', '62426492', '100', '-', '.', 'gene_id \"chr5.g42639\"; transcript_id \"blat.proc4005.chain_21546\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5267048', '5267319', '100', '-', '.', 'gene_id \"chr5.g42643\"; transcript_id \"blat.proc4005.chain_23709\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75525963', '75526122', '100', '+', '.', 'gene_id \"chr5.g42645\"; transcript_id \"blat.proc4005.chain_536\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '40897275', '40897331', '100', '-', '.', 'gene_id \"chr5.g42661\"; transcript_id \"blat.proc28144.chain_23404\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62728139', '62728183', '100', '-', '.', 'gene_id \"chr5.g42697\"; transcript_id \"blat.proc18128.chain_25106\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75625860', '75626129', '100', '+', '.', 'gene_id \"chr5.g42706\"; transcript_id \"blat.proc4005.chain_2732\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24645470', '24645504', '100', '-', '.', 'gene_id \"chr5.g42774\"; transcript_id \"blat.proc4005.chain_12542\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1059095', '1059269', '100', '+', '.', 'gene_id \"chr5.g42777\"; transcript_id \"blat.proc4005.chain_5225\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63659803', '63660005', '100', '+', '.', 'gene_id \"chr5.g42787\"; transcript_id \"blat.proc4005.chain_20766\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73219890', '73220040', '100', '+', '.', 'gene_id \"chr5.g42799\"; transcript_id \"blat.proc4005.chain_3359\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63734683', '63734935', '100', '-', '.', 'gene_id \"chr5.g42819\"; transcript_id \"blat.proc4005.chain_12094\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62301735', '62301931', '100', '+', '.', 'gene_id \"chr5.g42824\"; transcript_id \"blat.proc4005.chain_21209\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87735', '88027', '100', '+', '.', 'gene_id \"chr5.g42865\"; transcript_id \"blat.proc4005.chain_5694\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70764555', '70764739', '100', '-', '.', 'gene_id \"chr5.g42913\"; transcript_id \"blat.proc4005.chain_6044\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72932039', '72932318', '100', '+', '.', 'gene_id \"chr5.g42930\"; transcript_id \"blat.proc4005.chain_3107\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60677611', '60677872', '100', '-', '.', 'gene_id \"chr5.g42938\"; transcript_id \"blat.proc4587.chain_12343\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74217577', '74217788', '100', '+', '.', 'gene_id \"chr5.g42970\"; transcript_id \"blat.proc4005.chain_3932\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70321905', '70322077', '100', '-', '.', 'gene_id \"chr5.g43050\"; transcript_id \"blat.proc4005.chain_6828\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '3121327', '3121833', '100', '-', '.', 'gene_id \"chr5.g43057\"; transcript_id \"blat.proc4005.chain_22988\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63062364', '63062594', '100', '+', '.', 'gene_id \"chr5.g43087\"; transcript_id \"blat.proc4005.chain_12024\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '43062764', '43062959', '100', '+', '.', 'gene_id \"chr5.g43092\"; transcript_id \"blat.proc28144.chain_5663\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66050460', '66050616', '100', '-', '.', 'gene_id \"chr5.g43102\"; transcript_id \"blat.proc4005.chain_22557\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5035455', '5035611', '100', '+', '.', 'gene_id \"chr5.g43118\"; transcript_id \"blat.proc4005.chain_23034\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64944485', '64944652', '100', '-', '.', 'gene_id \"chr5.g43145\"; transcript_id \"blat.proc4005.chain_12399\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6836541', '6836834', '100', '+', '.', 'gene_id \"chr5.g43157\"; transcript_id \"blat.proc4587.chain_13496\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60280117', '60280286', '100', '+', '.', 'gene_id \"chr5.g43203\"; transcript_id \"blat.proc4005.chain_12494\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '89196130', '89196306', '100', '+', '.', 'gene_id \"chr5.g43221\"; transcript_id \"blat.proc4587.chain_3110\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62176292', '62176377', '100', '-', '.', 'gene_id \"chr5.g43271\"; transcript_id \"blat.proc4005.chain_14203\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '39424671', '39424702', '100', '+', '.', 'gene_id \"chr5.g43285\"; transcript_id \"blat.proc4005.chain_2275\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67902490', '67902673', '100', '+', '.', 'gene_id \"chr5.g43339\"; transcript_id \"blat.proc4005.chain_7275\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70512496', '70512660', '100', '+', '.', 'gene_id \"chr5.g43446\"; transcript_id \"blat.proc4005.chain_6253\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '16233432', '16233471', '100', '-', '.', 'gene_id \"chr5.g43452\"; transcript_id \"blat.proc4005.chain_11832\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74404326', '74404366', '100', '+', '.', 'gene_id \"chr5.g43471\"; transcript_id \"blat.proc4587.chain_13530\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71095848', '71096010', '100', '+', '.', 'gene_id \"chr5.g43506\"; transcript_id \"blat.proc4005.chain_5884\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63968876', '63969034', '100', '+', '.', 'gene_id \"chr5.g43528\"; transcript_id \"blat.proc4005.chain_21299\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64980508', '64980658', '100', '-', '.', 'gene_id \"chr5.g43546\"; transcript_id \"blat.proc4005.chain_20132\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '79636120', '79636324', '100', '-', '.', 'gene_id \"chr5.g43570\"; transcript_id \"blat.proc4005.chain_5992\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '17659383', '17659422', '100', '+', '.', 'gene_id \"chr5.g43606\"; transcript_id \"blat.proc4005.chain_19280\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60262263', '60262484', '100', '-', '.', 'gene_id \"chr5.g43645\"; transcript_id \"blat.proc4005.chain_21391\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6195473', '6195593', '100', '+', '.', 'gene_id \"chr5.g43692\"; transcript_id \"blat.proc4005.chain_23686\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68568977', '68569210', '100', '+', '.', 'gene_id \"chr5.g43706\"; transcript_id \"blat.proc4005.chain_6533\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76830558', '76830731', '100', '+', '.', 'gene_id \"chr5.g43722\"; transcript_id \"blat.proc4005.chain_2629\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62454335', '62454518', '100', '-', '.', 'gene_id \"chr5.g43776\"; transcript_id \"blat.proc4005.chain_21555\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70392607', '70392764', '100', '-', '.', 'gene_id \"chr5.g43795\"; transcript_id \"blat.proc4005.chain_11578\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63488344', '63488531', '100', '-', '.', 'gene_id \"chr5.g43839\"; transcript_id \"blat.proc4005.chain_21574\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '41276010', '41276039', '100', '+', '.', 'gene_id \"chr5.g43894\"; transcript_id \"blat.proc28144.chain_12250\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '65682855', '65683073', '100', '-', '.', 'gene_id \"chr5.g43906\"; transcript_id \"blat.proc4005.chain_21749\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67262108', '67262636', '100', '+', '.', 'gene_id \"chr5.g43917\"; transcript_id \"blat.proc4005.chain_10984\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63976898', '63977555', '100', '+', '.', 'gene_id \"chr5.g43921\"; transcript_id \"blat.proc4005.chain_20710\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70319035', '70319084', '100', '-', '.', 'gene_id \"chr5.g43938\"; transcript_id \"blat.proc4005.chain_11439\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63988796', '63988975', '100', '-', '.', 'gene_id \"chr5.g43959\"; transcript_id \"blat.proc4005.chain_12061\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69013561', '69013717', '100', '-', '.', 'gene_id \"chr5.g43960\"; transcript_id \"blat.proc4005.chain_11396\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63848682', '63848780', '100', '-', '.', 'gene_id \"chr5.g44012\"; transcript_id \"blat.proc4005.chain_20332\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68746778', '68746958', '100', '+', '.', 'gene_id \"chr5.g44069\"; transcript_id \"blat.proc4005.chain_6969\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72387773', '72387883', '100', '-', '.', 'gene_id \"chr5.g44076\"; transcript_id \"blat.proc4005.chain_11170\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60672616', '60672913', '100', '-', '.', 'gene_id \"chr5.g44088\"; transcript_id \"blat.proc4005.chain_20419\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62205035', '62205199', '100', '-', '.', 'gene_id \"chr5.g44112\"; transcript_id \"blat.proc4005.chain_12272\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70137973', '70138143', '100', '+', '.', 'gene_id \"chr5.g44145\"; transcript_id \"blat.proc4005.chain_6388\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81956686', '81956734', '100', '+', '.', 'gene_id \"chr5.g44158\"; transcript_id \"blat.proc18128.chain_6406\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63935757', '63935911', '100', '+', '.', 'gene_id \"chr5.g44214\"; transcript_id \"blat.proc4005.chain_11962\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67169225', '67169375', '100', '-', '.', 'gene_id \"chr5.g44240\"; transcript_id \"blat.proc4005.chain_6033\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77140801', '77140985', '100', '-', '.', 'gene_id \"chr5.g44244\"; transcript_id \"blat.proc4005.chain_2716\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70230858', '70231033', '100', '-', '.', 'gene_id \"chr5.g44245\"; transcript_id \"blat.proc4005.chain_11408\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74804538', '74804705', '100', '+', '.', 'gene_id \"chr5.g44282\"; transcript_id \"blat.proc4005.chain_2517\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63579796', '63580000', '100', '+', '.', 'gene_id \"chr5.g44345\"; transcript_id \"blat.proc4005.chain_13404\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74822870', '74823042', '100', '+', '.', 'gene_id \"chr5.g44403\"; transcript_id \"blat.proc4005.chain_2263\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5640867', '5641077', '100', '+', '.', 'gene_id \"chr5.g44404\"; transcript_id \"blat.proc4005.chain_23383\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60487913', '60487966', '100', '-', '.', 'gene_id \"chr5.g44405\"; transcript_id \"blat.proc4005.chain_20875\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63362086', '63362291', '100', '+', '.', 'gene_id \"chr5.g44427\"; transcript_id \"blat.proc4005.chain_19070\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5883907', '5884410', '100', '-', '.', 'gene_id \"chr5.g44432\"; transcript_id \"blat.proc4005.chain_23664\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76935959', '76936111', '100', '-', '.', 'gene_id \"chr5.g44462\"; transcript_id \"blat.proc4005.chain_2474\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52782153', '52782225', '100', '+', '.', 'gene_id \"chr5.g44467\"; transcript_id \"blat.proc4005.chain_4422\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62485764', '62485915', '100', '+', '.', 'gene_id \"chr5.g44475\"; transcript_id \"blat.proc4005.chain_12336\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63640563', '63640729', '100', '+', '.', 'gene_id \"chr5.g44484\"; transcript_id \"blat.proc4005.chain_21165\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63467129', '63467380', '100', '+', '.', 'gene_id \"chr5.g44510\"; transcript_id \"blat.proc4005.chain_12082\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '42068092', '42068126', '100', '+', '.', 'gene_id \"chr5.g44516\"; transcript_id \"blat.proc4005.chain_21837\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63191221', '63191386', '100', '-', '.', 'gene_id \"chr5.g44526\"; transcript_id \"blat.proc4005.chain_13946\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1089104', '1089411', '100', '-', '.', 'gene_id \"chr5.g44534\"; transcript_id \"blat.proc4005.chain_4292\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6948181', '6948423', '100', '+', '.', 'gene_id \"chr5.g44569\"; transcript_id \"blat.proc28144.chain_6228\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1134057', '1134249', '100', '-', '.', 'gene_id \"chr5.g44570\"; transcript_id \"blat.proc4005.chain_4849\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '56254407', '56254456', '100', '+', '.', 'gene_id \"chr5.g44577\"; transcript_id \"blat.proc28144.chain_14274\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64980257', '64980301', '100', '+', '.', 'gene_id \"chr5.g44609\"; transcript_id \"blat.proc4005.chain_20040\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67564541', '67564696', '100', '-', '.', 'gene_id \"chr5.g44648\"; transcript_id \"blat.proc18128.chain_12256\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '1004283', '1004451', '100', '+', '.', 'gene_id \"chr5.g44662\"; transcript_id \"blat.proc4005.chain_5056\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77840406', '77840558', '100', '+', '.', 'gene_id \"chr5.g44671\"; transcript_id \"blat.proc4005.chain_2523\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25242613', '25242649', '100', '+', '.', 'gene_id \"chr5.g44710\"; transcript_id \"blat.proc18128.chain_21746\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '28953463', '28953527', '100', '+', '.', 'gene_id \"chr5.g44719\"; transcript_id \"blat.proc4005.chain_23769\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63378481', '63378657', '100', '-', '.', 'gene_id \"chr5.g44769\"; transcript_id \"blat.proc4005.chain_20783\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63271880', '63272052', '100', '+', '.', 'gene_id \"chr5.g44778\"; transcript_id \"blat.proc4005.chain_20961\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66889314', '66889478', '100', '-', '.', 'gene_id \"chr5.g44786\"; transcript_id \"blat.proc4005.chain_6316\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61230136', '61230349', '100', '+', '.', 'gene_id \"chr5.g44823\"; transcript_id \"blat.proc4005.chain_11909\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71824371', '71824413', '100', '+', '.', 'gene_id \"chr5.g44828\"; transcript_id \"blat.proc4587.chain_10464\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '24011625', '24011670', '100', '+', '.', 'gene_id \"chr5.g44852\"; transcript_id \"blat.proc4587.chain_9439\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '17584423', '17584462', '100', '+', '.', 'gene_id \"chr5.g44870\"; transcript_id \"blat.proc18128.chain_10520\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67483585', '67483760', '100', '+', '.', 'gene_id \"chr5.g44875\"; transcript_id \"blat.proc4005.chain_6866\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75879039', '75879207', '100', '-', '.', 'gene_id \"chr5.g44892\"; transcript_id \"blat.proc4005.chain_2606\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75245394', '75245594', '100', '-', '.', 'gene_id \"chr5.g44929\"; transcript_id \"blat.proc4005.chain_2762\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64892578', '64892778', '100', '+', '.', 'gene_id \"chr5.g44936\"; transcript_id \"blat.proc4005.chain_20760\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60891777', '60891974', '100', '-', '.', 'gene_id \"chr5.g44964\"; transcript_id \"blat.proc4005.chain_12425\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62436180', '62436790', '100', '+', '.', 'gene_id \"chr5.g44989\"; transcript_id \"blat.proc4005.chain_12621\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75631086', '75631256', '100', '+', '.', 'gene_id \"chr5.g44991\"; transcript_id \"blat.proc4005.chain_1769\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61647444', '61647800', '100', '-', '.', 'gene_id \"chr5.g45025\"; transcript_id \"blat.proc4005.chain_12467\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '66094098', '66094226', '100', '+', '.', 'gene_id \"chr5.g45055\"; transcript_id \"blat.proc4005.chain_22958\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5423417', '5423589', '100', '-', '.', 'gene_id \"chr5.g45092\"; transcript_id \"blat.proc4005.chain_23733\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60329136', '60329401', '100', '-', '.', 'gene_id \"chr5.g45108\"; transcript_id \"blat.proc4005.chain_13378\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76022056', '76022221', '100', '+', '.', 'gene_id \"chr5.g45137\"; transcript_id \"blat.proc4005.chain_273\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64428943', '64429108', '100', '+', '.', 'gene_id \"chr5.g45139\"; transcript_id \"blat.proc4005.chain_18295\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81609662', '81609813', '100', '+', '.', 'gene_id \"chr5.g45149\"; transcript_id \"blat.proc18128.chain_16217\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '748091', '748341', '100', '+', '.', 'gene_id \"chr5.g45172\"; transcript_id \"blat.proc4005.chain_5214\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70230003', '70230282', '100', '+', '.', 'gene_id \"chr5.g45227\"; transcript_id \"blat.proc4005.chain_10693\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '970353', '970588', '100', '-', '.', 'gene_id \"chr5.g45237\"; transcript_id \"blat.proc4005.chain_4350\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70802744', '70802935', '100', '+', '.', 'gene_id \"chr5.g45271\"; transcript_id \"blat.proc4005.chain_6531\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75534768', '75534892', '100', '+', '.', 'gene_id \"chr5.g45303\"; transcript_id \"blat.proc4005.chain_843\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63258864', '63258998', '100', '-', '.', 'gene_id \"chr5.g45311\"; transcript_id \"blat.proc4005.chain_19093\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '449730', '449792', '100', '-', '.', 'gene_id \"chr5.g45414\"; transcript_id \"blat.proc28144.chain_9240\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68087324', '68087479', '100', '+', '.', 'gene_id \"chr5.g45423\"; transcript_id \"blat.proc4005.chain_11307\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76620094', '76620252', '100', '-', '.', 'gene_id \"chr5.g45472\"; transcript_id \"blat.proc4005.chain_2754\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63414658', '63414793', '100', '+', '.', 'gene_id \"chr5.g45507\"; transcript_id \"blat.proc4005.chain_16263\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '71134425', '71134576', '100', '+', '.', 'gene_id \"chr5.g45516\"; transcript_id \"blat.proc4005.chain_11719\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60372171', '60372292', '100', '+', '.', 'gene_id \"chr5.g45596\"; transcript_id \"blat.proc4005.chain_21355\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '61781044', '61781122', '100', '-', '.', 'gene_id \"chr5.g45617\"; transcript_id \"blat.proc4005.chain_20973\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6992361', '6992407', '100', '-', '.', 'gene_id \"chr5.g45625\"; transcript_id \"blat.proc18128.chain_3816\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67965079', '67965121', '100', '+', '.', 'gene_id \"chr5.g45661\"; transcript_id \"blat.proc18128.chain_25879\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64646404', '64646660', '100', '+', '.', 'gene_id \"chr5.g45662\"; transcript_id \"blat.proc4005.chain_12414\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64418858', '64419058', '100', '-', '.', 'gene_id \"chr5.g45763\"; transcript_id \"blat.proc4005.chain_11883\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63151648', '63151873', '100', '+', '.', 'gene_id \"chr5.g45765\"; transcript_id \"blat.proc4005.chain_21442\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64245289', '64245447', '100', '+', '.', 'gene_id \"chr5.g45767\"; transcript_id \"blat.proc4005.chain_21170\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67390564', '67390716', '100', '+', '.', 'gene_id \"chr5.g45787\"; transcript_id \"blat.proc4005.chain_11342\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74465426', '74465480', '100', '+', '.', 'gene_id \"chr5.g45819\"; transcript_id \"blat.proc28144.chain_8836\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63817221', '63817383', '100', '+', '.', 'gene_id \"chr5.g45907\"; transcript_id \"blat.proc4005.chain_21055\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68369313', '68369346', '100', '-', '.', 'gene_id \"chr5.g45908\"; transcript_id \"blat.proc4587.chain_9050\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '78979469', '78979623', '100', '+', '.', 'gene_id \"chr5.g45916\"; transcript_id \"blat.proc4005.chain_2767\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76852927', '76853089', '100', '-', '.', 'gene_id \"chr5.g45951\"; transcript_id \"blat.proc4005.chain_1446\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60484034', '60484185', '100', '+', '.', 'gene_id \"chr5.g45965\"; transcript_id \"blat.proc4005.chain_11946\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '258620', '258778', '100', '-', '.', 'gene_id \"chr5.g46001\"; transcript_id \"blat.proc4005.chain_4312\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '60146801', '60146945', '100', '+', '.', 'gene_id \"chr5.g46005\"; transcript_id \"blat.proc4005.chain_19444\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '10031431', '10032070', '100', '-', '.', 'gene_id \"chr5.g46007\"; transcript_id \"blat.proc28144.chain_8212\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '87625732', '87625889', '100', '+', '.', 'gene_id \"chr5.g46047\"; transcript_id \"blat.proc28144.chain_16760\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72918273', '72918431', '100', '+', '.', 'gene_id \"chr5.g46056\"; transcript_id \"blat.proc4005.chain_2986\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62651668', '62651998', '100', '+', '.', 'gene_id \"chr5.g46082\"; transcript_id \"blat.proc4587.chain_12470\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63712362', '63712564', '100', '+', '.', 'gene_id \"chr5.g46088\"; transcript_id \"blat.proc4005.chain_20997\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69618450', '69618517', '100', '-', '.', 'gene_id \"chr5.g46143\"; transcript_id \"blat.proc4587.chain_7025\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '77701019', '77701262', '100', '+', '.', 'gene_id \"chr5.g46199\"; transcript_id \"blat.proc4005.chain_2527\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70542347', '70542579', '100', '-', '.', 'gene_id \"chr5.g46237\"; transcript_id \"blat.proc4587.chain_142\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '69693921', '69694072', '100', '+', '.', 'gene_id \"chr5.g46261\"; transcript_id \"blat.proc4005.chain_6254\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '67782076', '67782235', '100', '+', '.', 'gene_id \"chr5.g46264\"; transcript_id \"blat.proc4005.chain_11597\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76800640', '76800861', '100', '-', '.', 'gene_id \"chr5.g46289\"; transcript_id \"blat.proc4005.chain_2889\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68002465', '68002672', '100', '+', '.', 'gene_id \"chr5.g46292\"; transcript_id \"blat.proc4005.chain_7643\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '72282829', '72283471', '100', '-', '.', 'gene_id \"chr5.g46294\"; transcript_id \"blat.proc4587.chain_14005\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '63639526', '63639696', '100', '-', '.', 'gene_id \"chr5.g46299\"; transcript_id \"blat.proc4005.chain_21138\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '76752466', '76752502', '100', '+', '.', 'gene_id \"chr5.g46332\"; transcript_id \"blat.proc4587.chain_11051\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73011864', '73012042', '100', '+', '.', 'gene_id \"chr5.g46368\"; transcript_id \"blat.proc4005.chain_4249\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '19083560', '19083592', '100', '-', '.', 'gene_id \"chr5.g46376\"; transcript_id \"blat.proc4005.chain_9192\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '551799', '551877', '100', '-', '.', 'gene_id \"chr5.g46385\"; transcript_id \"blat.proc4587.chain_607\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '68437628', '68437870', '100', '+', '.', 'gene_id \"chr5.g46427\"; transcript_id \"blat.proc4005.chain_6268\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '13921825', '13921876', '100', '-', '.', 'gene_id \"chr5.g46486\"; transcript_id \"blat.proc4005.chain_23785\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '52685843', '52685889', '100', '-', '.', 'gene_id \"chr5.g46538\"; transcript_id \"blat.proc4005.chain_3644\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '74015717', '74015750', '100', '+', '.', 'gene_id \"chr5.g46564\"; transcript_id \"blat.proc4005.chain_17238\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '64383330', '64383501', '100', '+', '.', 'gene_id \"chr5.g46585\"; transcript_id \"blat.proc4005.chain_12003\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '70290031', '70290188', '100', '+', '.', 'gene_id \"chr5.g46667\"; transcript_id \"blat.proc4005.chain_11260\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '62902529', '62902774', '100', '+', '.', 'gene_id \"chr5.g46671\"; transcript_id \"blat.proc4005.chain_12531\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '25787341', '25787578', '100', '+', '.', 'gene_id \"chr5.g46719\"; transcript_id \"blat.proc4587.chain_6861\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '81656284', '81656324', '100', '+', '.', 'gene_id \"chr5.g46737\"; transcript_id \"blat.proc28144.chain_20574\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '73112621', '73112811', '100', '+', '.', 'gene_id \"chr5.g46742\"; transcript_id \"blat.proc4005.chain_3396\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '5640668', '5640841', '100', '-', '.', 'gene_id \"chr5.g46801\"; transcript_id \"blat.proc4005.chain_23382\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '83992396', '83992439', '100', '-', '.', 'gene_id \"chr5.g46818\"; transcript_id \"blat.proc18128.chain_7348\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '75840096', '75840265', '100', '-', '.', 'gene_id \"chr5.g46849\"; transcript_id \"blat.proc4005.chain_2478\";']\n",
      "negative:  ['chr5', 'PASA', 'exon', '6796774', '6797147', '100', '-', '.', 'gene_id \"chr5.g46856\"; transcript_id \"blat.proc4005.chain_23510\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17646003', '17646063', '.', '-', '.', 'gene_id \"chr38.g5063\"; transcript_id \"chr38.g5063.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17768937', '17768973', '.', '+', '.', 'gene_id \"chr38.g5162\"; transcript_id \"chr38.g5162.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17768943', '17769005', '.', '+', '.', 'gene_id \"chr38.g5162\"; transcript_id \"chr38.g5162.i2\";']\n",
      "negative:  ['scaffold_224', 'PASA', 'exon', '98862', '99121', '.', '+', '.', 'gene_id \"scaffold_224.g75\"; transcript_id \"scaffold_224.g75.i1\";']\n",
      "negative:  ['scaffold_224', 'PASA', 'exon', '123952', '124003', '.', '-', '.', 'gene_id \"scaffold_224.g97\"; transcript_id \"scaffold_224.g97.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '18710564', '18710963', '.', '+', '.', 'gene_id \"chr38.g5524\"; transcript_id \"chr38.g5524.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '18716191', '18716467', '.', '+', '.', 'gene_id \"chr38.g5530\"; transcript_id \"chr38.g5530.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '18716465', '18716761', '.', '-', '.', 'gene_id \"chr38.g5531\"; transcript_id \"chr38.g5531.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '19833285', '19833441', '.', '-', '.', 'gene_id \"chr38.g5653\"; transcript_id \"chr38.g5653.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '19835621', '19835773', '.', '-', '.', 'gene_id \"chr38.g5660\"; transcript_id \"chr38.g5660.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '19835767', '19836377', '.', '+', '.', 'gene_id \"chr38.g5661\"; transcript_id \"chr38.g5661.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '19899151', '19899201', '.', '-', '.', 'gene_id \"chr38.g5665\"; transcript_id \"chr38.g5665.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20035373', '20035547', '.', '-', '.', 'gene_id \"chr38.g5684\"; transcript_id \"chr38.g5684.i5\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20111257', '20111589', '.', '-', '.', 'gene_id \"chr38.g5758\"; transcript_id \"chr38.g5758.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '370763', '370957', '.', '+', '.', 'gene_id \"chr21.g25\"; transcript_id \"chr21.g25.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '372646', '372894', '.', '+', '.', 'gene_id \"chr21.g27\"; transcript_id \"chr21.g27.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '380709', '380893', '.', '-', '.', 'gene_id \"chr21.g33\"; transcript_id \"chr21.g33.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20287074', '20287252', '.', '+', '.', 'gene_id \"chr38.g5857\"; transcript_id \"chr38.g5857.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20287328', '20287476', '.', '-', '.', 'gene_id \"chr38.g5859\"; transcript_id \"chr38.g5859.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '432091', '432207', '.', '+', '.', 'gene_id \"chr21.g41\"; transcript_id \"chr21.g41.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '436317', '436353', '.', '+', '.', 'gene_id \"chr21.g43\"; transcript_id \"chr21.g43.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '458414', '458570', '.', '-', '.', 'gene_id \"chr21.g47\"; transcript_id \"chr21.g47.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '469071', '469865', '.', '+', '.', 'gene_id \"chr21.g49\"; transcript_id \"chr21.g49.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '473006', '473816', '.', '+', '.', 'gene_id \"chr21.g50\"; transcript_id \"chr21.g50.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '475988', '476138', '.', '-', '.', 'gene_id \"chr21.g52\"; transcript_id \"chr21.g52.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '569870', '570214', '.', '+', '.', 'gene_id \"chr21.g67\"; transcript_id \"chr21.g67.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '581545', '581814', '.', '+', '.', 'gene_id \"chr21.g76\"; transcript_id \"chr21.g76.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '550440', '550698', '.', '-', '.', 'gene_id \"chr21.g80\"; transcript_id \"chr21.g80.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '580215', '580388', '.', '-', '.', 'gene_id \"chr21.g89\"; transcript_id \"chr21.g89.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '580388', '580567', '.', '-', '.', 'gene_id \"chr21.g90\"; transcript_id \"chr21.g90.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '593240', '593428', '.', '-', '.', 'gene_id \"chr21.g93\"; transcript_id \"chr21.g93.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '610026', '610201', '.', '-', '.', 'gene_id \"chr21.g94\"; transcript_id \"chr21.g94.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '679427', '679590', '.', '-', '.', 'gene_id \"chr21.g108\"; transcript_id \"chr21.g108.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '690674', '690886', '.', '+', '.', 'gene_id \"chr21.g111\"; transcript_id \"chr21.g111.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '700438', '700654', '.', '+', '.', 'gene_id \"chr21.g114\"; transcript_id \"chr21.g114.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '710189', '710349', '.', '-', '.', 'gene_id \"chr21.g115\"; transcript_id \"chr21.g115.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '751812', '752041', '.', '-', '.', 'gene_id \"chr21.g125\"; transcript_id \"chr21.g125.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '759612', '759985', '.', '-', '.', 'gene_id \"chr21.g126\"; transcript_id \"chr21.g126.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '765334', '765591', '.', '+', '.', 'gene_id \"chr21.g128\"; transcript_id \"chr21.g128.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '826161', '826316', '.', '+', '.', 'gene_id \"chr21.g135\"; transcript_id \"chr21.g135.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1185357', '1185893', '.', '-', '.', 'gene_id \"chr21.g177\"; transcript_id \"chr21.g177.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1327905', '1328122', '.', '-', '.', 'gene_id \"chr21.g184\"; transcript_id \"chr21.g184.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1328193', '1328518', '.', '+', '.', 'gene_id \"chr21.g185\"; transcript_id \"chr21.g185.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1329681', '1329827', '.', '-', '.', 'gene_id \"chr21.g186\"; transcript_id \"chr21.g186.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1371119', '1371300', '.', '-', '.', 'gene_id \"chr21.g191\"; transcript_id \"chr21.g191.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1627050', '1627079', '.', '+', '.', 'gene_id \"chr21.g206\"; transcript_id \"chr21.g206.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1796948', '1797157', '.', '-', '.', 'gene_id \"chr21.g220\"; transcript_id \"chr21.g220.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1803286', '1803524', '.', '+', '.', 'gene_id \"chr21.g221\"; transcript_id \"chr21.g221.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1808407', '1808533', '.', '-', '.', 'gene_id \"chr21.g225\"; transcript_id \"chr21.g225.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1883790', '1883861', '.', '-', '.', 'gene_id \"chr21.g233\"; transcript_id \"chr21.g233.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20596333', '20596383', '.', '+', '.', 'gene_id \"chr38.g6076\"; transcript_id \"chr38.g6076.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2276051', '2276224', '.', '+', '.', 'gene_id \"chr21.g277\"; transcript_id \"chr21.g277.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2314524', '2314706', '.', '+', '.', 'gene_id \"chr21.g279\"; transcript_id \"chr21.g279.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2346482', '2346725', '.', '+', '.', 'gene_id \"chr21.g282\"; transcript_id \"chr21.g282.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '3079836', '3079878', '.', '-', '.', 'gene_id \"chr21.g328\"; transcript_id \"chr21.g328.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '3385864', '3386022', '.', '-', '.', 'gene_id \"chr21.g345\"; transcript_id \"chr21.g345.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20814777', '20814816', '.', '+', '.', 'gene_id \"chr38.g6191\"; transcript_id \"chr38.g6191.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4459363', '4459411', '.', '-', '.', 'gene_id \"chr21.g403\"; transcript_id \"chr21.g403.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4685620', '4685814', '.', '-', '.', 'gene_id \"chr21.g413\"; transcript_id \"chr21.g413.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4690014', '4690237', '.', '+', '.', 'gene_id \"chr21.g415\"; transcript_id \"chr21.g415.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4734201', '4734456', '.', '+', '.', 'gene_id \"chr21.g422\"; transcript_id \"chr21.g422.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4734823', '4735066', '.', '-', '.', 'gene_id \"chr21.g423\"; transcript_id \"chr21.g423.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4751541', '4751847', '.', '-', '.', 'gene_id \"chr21.g433\"; transcript_id \"chr21.g433.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4793619', '4793792', '.', '+', '.', 'gene_id \"chr21.g454\"; transcript_id \"chr21.g454.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4795333', '4796062', '.', '-', '.', 'gene_id \"chr21.g457\"; transcript_id \"chr21.g457.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4803475', '4803685', '.', '+', '.', 'gene_id \"chr21.g462\"; transcript_id \"chr21.g462.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4803675', '4803852', '.', '+', '.', 'gene_id \"chr21.g463\"; transcript_id \"chr21.g463.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4808709', '4808997', '.', '-', '.', 'gene_id \"chr21.g467\"; transcript_id \"chr21.g467.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4812717', '4813252', '.', '-', '.', 'gene_id \"chr21.g471\"; transcript_id \"chr21.g471.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4813260', '4813588', '.', '-', '.', 'gene_id \"chr21.g472\"; transcript_id \"chr21.g472.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4826573', '4826947', '.', '-', '.', 'gene_id \"chr21.g480\"; transcript_id \"chr21.g480.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4854493', '4854662', '.', '+', '.', 'gene_id \"chr21.g506\"; transcript_id \"chr21.g506.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4857620', '4857826', '.', '-', '.', 'gene_id \"chr21.g507\"; transcript_id \"chr21.g507.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4862500', '4863045', '.', '-', '.', 'gene_id \"chr21.g511\"; transcript_id \"chr21.g511.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4864383', '4866251', '.', '+', '.', 'gene_id \"chr21.g516\"; transcript_id \"chr21.g516.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4869612', '4869872', '.', '-', '.', 'gene_id \"chr21.g520\"; transcript_id \"chr21.g520.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4871328', '4871708', '.', '+', '.', 'gene_id \"chr21.g522\"; transcript_id \"chr21.g522.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4872151', '4872331', '.', '-', '.', 'gene_id \"chr21.g524\"; transcript_id \"chr21.g524.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4903135', '4903288', '.', '-', '.', 'gene_id \"chr21.g557\"; transcript_id \"chr21.g557.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4909998', '4910265', '.', '-', '.', 'gene_id \"chr21.g561\"; transcript_id \"chr21.g561.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4912128', '4912284', '.', '-', '.', 'gene_id \"chr21.g565\"; transcript_id \"chr21.g565.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4929159', '4929416', '.', '-', '.', 'gene_id \"chr21.g583\"; transcript_id \"chr21.g583.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4944186', '4944413', '.', '+', '.', 'gene_id \"chr21.g591\"; transcript_id \"chr21.g591.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4955669', '4955861', '.', '+', '.', 'gene_id \"chr21.g597\"; transcript_id \"chr21.g597.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4959018', '4959292', '.', '+', '.', 'gene_id \"chr21.g601\"; transcript_id \"chr21.g601.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4963425', '4963619', '.', '-', '.', 'gene_id \"chr21.g603\"; transcript_id \"chr21.g603.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4964205', '4964398', '.', '-', '.', 'gene_id \"chr21.g604\"; transcript_id \"chr21.g604.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4964390', '4966024', '.', '+', '.', 'gene_id \"chr21.g605\"; transcript_id \"chr21.g605.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4968722', '4968954', '.', '-', '.', 'gene_id \"chr21.g606\"; transcript_id \"chr21.g606.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4978245', '4978487', '.', '-', '.', 'gene_id \"chr21.g615\"; transcript_id \"chr21.g615.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4978598', '4978749', '.', '-', '.', 'gene_id \"chr21.g616\"; transcript_id \"chr21.g616.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4980244', '4980399', '.', '+', '.', 'gene_id \"chr21.g617\"; transcript_id \"chr21.g617.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4983703', '4983869', '.', '+', '.', 'gene_id \"chr21.g620\"; transcript_id \"chr21.g620.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5010138', '5010305', '.', '+', '.', 'gene_id \"chr21.g631\"; transcript_id \"chr21.g631.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5010486', '5010668', '.', '-', '.', 'gene_id \"chr21.g632\"; transcript_id \"chr21.g632.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5015007', '5015272', '.', '-', '.', 'gene_id \"chr21.g637\"; transcript_id \"chr21.g637.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5017464', '5017864', '.', '-', '.', 'gene_id \"chr21.g639\"; transcript_id \"chr21.g639.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5019101', '5019409', '.', '-', '.', 'gene_id \"chr21.g641\"; transcript_id \"chr21.g641.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '21330225', '21330526', '.', '+', '.', 'gene_id \"chr38.g6482\"; transcript_id \"chr38.g6482.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5044045', '5044389', '.', '-', '.', 'gene_id \"chr21.g662\"; transcript_id \"chr21.g662.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5045996', '5046150', '.', '-', '.', 'gene_id \"chr21.g665\"; transcript_id \"chr21.g665.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5046173', '5046407', '.', '-', '.', 'gene_id \"chr21.g666\"; transcript_id \"chr21.g666.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5051077', '5051306', '.', '-', '.', 'gene_id \"chr21.g675\"; transcript_id \"chr21.g675.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5057061', '5057578', '.', '-', '.', 'gene_id \"chr21.g681\"; transcript_id \"chr21.g681.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5075323', '5075518', '.', '-', '.', 'gene_id \"chr21.g701\"; transcript_id \"chr21.g701.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5084691', '5084866', '.', '-', '.', 'gene_id \"chr21.g712\"; transcript_id \"chr21.g712.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5090298', '5090489', '.', '+', '.', 'gene_id \"chr21.g716\"; transcript_id \"chr21.g716.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5096837', '5097215', '.', '-', '.', 'gene_id \"chr21.g721\"; transcript_id \"chr21.g721.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5102975', '5103135', '.', '-', '.', 'gene_id \"chr21.g727\"; transcript_id \"chr21.g727.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5107226', '5107836', '.', '-', '.', 'gene_id \"chr21.g731\"; transcript_id \"chr21.g731.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5107226', '5107732', '.', '-', '.', 'gene_id \"chr21.g731\"; transcript_id \"chr21.g731.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5110129', '5110553', '.', '-', '.', 'gene_id \"chr21.g732\"; transcript_id \"chr21.g732.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5113807', '5114180', '.', '-', '.', 'gene_id \"chr21.g737\"; transcript_id \"chr21.g737.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5114939', '5115837', '.', '-', '.', 'gene_id \"chr21.g738\"; transcript_id \"chr21.g738.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5119395', '5119724', '.', '-', '.', 'gene_id \"chr21.g745\"; transcript_id \"chr21.g745.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5124159', '5124368', '.', '-', '.', 'gene_id \"chr21.g750\"; transcript_id \"chr21.g750.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5137839', '5138077', '.', '-', '.', 'gene_id \"chr21.g759\"; transcript_id \"chr21.g759.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5157329', '5157485', '.', '-', '.', 'gene_id \"chr21.g778\"; transcript_id \"chr21.g778.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5158663', '5158904', '.', '-', '.', 'gene_id \"chr21.g779\"; transcript_id \"chr21.g779.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5211992', '5212096', '.', '-', '.', 'gene_id \"chr21.g792\"; transcript_id \"chr21.g792.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5233448', '5233657', '.', '+', '.', 'gene_id \"chr21.g805\"; transcript_id \"chr21.g805.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5233669', '5233850', '.', '+', '.', 'gene_id \"chr21.g806\"; transcript_id \"chr21.g806.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5273389', '5273728', '.', '-', '.', 'gene_id \"chr21.g810\"; transcript_id \"chr21.g810.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5277878', '5278174', '.', '+', '.', 'gene_id \"chr21.g813\"; transcript_id \"chr21.g813.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5280682', '5280790', '.', '-', '.', 'gene_id \"chr21.g814\"; transcript_id \"chr21.g814.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5244452', '5244644', '.', '-', '.', 'gene_id \"chr21.g816\"; transcript_id \"chr21.g816.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5303568', '5304932', '.', '+', '.', 'gene_id \"chr21.g842\"; transcript_id \"chr21.g842.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5284216', '5284401', '.', '+', '.', 'gene_id \"chr21.g852\"; transcript_id \"chr21.g852.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5297505', '5297793', '.', '-', '.', 'gene_id \"chr21.g861\"; transcript_id \"chr21.g861.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5301694', '5302177', '.', '+', '.', 'gene_id \"chr21.g865\"; transcript_id \"chr21.g865.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '21563022', '21563057', '.', '+', '.', 'gene_id \"chr38.g6711\"; transcript_id \"chr38.g6711.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5323456', '5323636', '.', '+', '.', 'gene_id \"chr21.g878\"; transcript_id \"chr21.g878.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5379473', '5379810', '.', '-', '.', 'gene_id \"chr21.g887\"; transcript_id \"chr21.g887.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5385945', '5386173', '.', '-', '.', 'gene_id \"chr21.g890\"; transcript_id \"chr21.g890.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5505397', '5506071', '.', '-', '.', 'gene_id \"chr21.g901\"; transcript_id \"chr21.g901.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5630638', '5630818', '.', '-', '.', 'gene_id \"chr21.g913\"; transcript_id \"chr21.g913.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5683888', '5684082', '.', '+', '.', 'gene_id \"chr21.g927\"; transcript_id \"chr21.g927.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5700823', '5701476', '.', '+', '.', 'gene_id \"chr21.g934\"; transcript_id \"chr21.g934.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5724060', '5724104', '.', '+', '.', 'gene_id \"chr21.g938\"; transcript_id \"chr21.g938.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5782408', '5782725', '.', '-', '.', 'gene_id \"chr21.g942\"; transcript_id \"chr21.g942.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5783016', '5783548', '.', '+', '.', 'gene_id \"chr21.g943\"; transcript_id \"chr21.g943.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5790032', '5790191', '.', '+', '.', 'gene_id \"chr21.g946\"; transcript_id \"chr21.g946.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5790197', '5790611', '.', '+', '.', 'gene_id \"chr21.g947\"; transcript_id \"chr21.g947.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5793315', '5793506', '.', '-', '.', 'gene_id \"chr21.g948\"; transcript_id \"chr21.g948.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5819868', '5820086', '.', '-', '.', 'gene_id \"chr21.g967\"; transcript_id \"chr21.g967.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5825789', '5825992', '.', '+', '.', 'gene_id \"chr21.g971\"; transcript_id \"chr21.g971.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5832345', '5832521', '.', '-', '.', 'gene_id \"chr21.g979\"; transcript_id \"chr21.g979.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5836440', '5837168', '.', '-', '.', 'gene_id \"chr21.g987\"; transcript_id \"chr21.g987.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5841285', '5841699', '.', '+', '.', 'gene_id \"chr21.g988\"; transcript_id \"chr21.g988.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5868710', '5868889', '.', '+', '.', 'gene_id \"chr21.g993\"; transcript_id \"chr21.g993.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5850836', '5850991', '.', '+', '.', 'gene_id \"chr21.g998\"; transcript_id \"chr21.g998.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5854093', '5854736', '.', '+', '.', 'gene_id \"chr21.g1004\"; transcript_id \"chr21.g1004.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '21916165', '21916832', '.', '-', '.', 'gene_id \"chr38.g6873\"; transcript_id \"chr38.g6873.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5870789', '5870977', '.', '-', '.', 'gene_id \"chr21.g1009\"; transcript_id \"chr21.g1009.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5874941', '5875195', '.', '+', '.', 'gene_id \"chr21.g1012\"; transcript_id \"chr21.g1012.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5881601', '5881787', '.', '+', '.', 'gene_id \"chr21.g1018\"; transcript_id \"chr21.g1018.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5904622', '5908786', '.', '-', '.', 'gene_id \"chr21.g1027\"; transcript_id \"chr21.g1027.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5913124', '5913283', '.', '+', '.', 'gene_id \"chr21.g1033\"; transcript_id \"chr21.g1033.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5915112', '5915307', '.', '+', '.', 'gene_id \"chr21.g1036\"; transcript_id \"chr21.g1036.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5932350', '5933023', '.', '+', '.', 'gene_id \"chr21.g1040\"; transcript_id \"chr21.g1040.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6007461', '6007976', '.', '-', '.', 'gene_id \"chr21.g1053\"; transcript_id \"chr21.g1053.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6054305', '6054478', '.', '+', '.', 'gene_id \"chr21.g1064\"; transcript_id \"chr21.g1064.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6060491', '6060712', '.', '+', '.', 'gene_id \"chr21.g1067\"; transcript_id \"chr21.g1067.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6069739', '6069951', '.', '-', '.', 'gene_id \"chr21.g1068\"; transcript_id \"chr21.g1068.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6069957', '6070179', '.', '-', '.', 'gene_id \"chr21.g1069\"; transcript_id \"chr21.g1069.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6088499', '6090085', '.', '-', '.', 'gene_id \"chr21.g1074\"; transcript_id \"chr21.g1074.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6086981', '6087583', '.', '-', '.', 'gene_id \"chr21.g1074\"; transcript_id \"chr21.g1074.i4\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6095056', '6096284', '.', '+', '.', 'gene_id \"chr21.g1083\"; transcript_id \"chr21.g1083.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6115683', '6116058', '.', '+', '.', 'gene_id \"chr21.g1090\"; transcript_id \"chr21.g1090.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6131967', '6132119', '.', '-', '.', 'gene_id \"chr21.g1100\"; transcript_id \"chr21.g1100.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr21', 'PASA', 'exon', '6141458', '6141609', '.', '-', '.', 'gene_id \"chr21.g1104\"; transcript_id \"chr21.g1104.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6148216', '6148447', '.', '-', '.', 'gene_id \"chr21.g1106\"; transcript_id \"chr21.g1106.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6148660', '6148862', '.', '-', '.', 'gene_id \"chr21.g1107\"; transcript_id \"chr21.g1107.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6154992', '6155245', '.', '+', '.', 'gene_id \"chr21.g1110\"; transcript_id \"chr21.g1110.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6204315', '6204448', '.', '+', '.', 'gene_id \"chr21.g1121\"; transcript_id \"chr21.g1121.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6234371', '6234479', '.', '+', '.', 'gene_id \"chr21.g1128\"; transcript_id \"chr21.g1128.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6247745', '6247904', '.', '+', '.', 'gene_id \"chr21.g1132\"; transcript_id \"chr21.g1132.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6403258', '6403417', '.', '-', '.', 'gene_id \"chr21.g1165\"; transcript_id \"chr21.g1165.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6403399', '6403597', '.', '-', '.', 'gene_id \"chr21.g1165\"; transcript_id \"chr21.g1165.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6403572', '6403968', '.', '+', '.', 'gene_id \"chr21.g1166\"; transcript_id \"chr21.g1166.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6412591', '6412970', '.', '+', '.', 'gene_id \"chr21.g1173\"; transcript_id \"chr21.g1173.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6476434', '6476620', '.', '-', '.', 'gene_id \"chr21.g1179\"; transcript_id \"chr21.g1179.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6421234', '6421450', '.', '-', '.', 'gene_id \"chr21.g1180\"; transcript_id \"chr21.g1180.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6421860', '6422142', '.', '-', '.', 'gene_id \"chr21.g1181\"; transcript_id \"chr21.g1181.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6472491', '6472671', '.', '-', '.', 'gene_id \"chr21.g1190\"; transcript_id \"chr21.g1190.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6484520', '6485235', '.', '-', '.', 'gene_id \"chr21.g1195\"; transcript_id \"chr21.g1195.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6499140', '6499695', '.', '+', '.', 'gene_id \"chr21.g1203\"; transcript_id \"chr21.g1203.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6500934', '6501715', '.', '+', '.', 'gene_id \"chr21.g1207\"; transcript_id \"chr21.g1207.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6647385', '6647623', '.', '+', '.', 'gene_id \"chr21.g1232\"; transcript_id \"chr21.g1232.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6669890', '6670239', '.', '-', '.', 'gene_id \"chr21.g1238\"; transcript_id \"chr21.g1238.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6671629', '6672061', '.', '-', '.', 'gene_id \"chr21.g1239\"; transcript_id \"chr21.g1239.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6679548', '6679700', '.', '-', '.', 'gene_id \"chr21.g1246\"; transcript_id \"chr21.g1246.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6685626', '6685665', '.', '-', '.', 'gene_id \"chr21.g1252\"; transcript_id \"chr21.g1252.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6685634', '6685679', '.', '-', '.', 'gene_id \"chr21.g1252\"; transcript_id \"chr21.g1252.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6701064', '6701268', '.', '-', '.', 'gene_id \"chr21.g1258\"; transcript_id \"chr21.g1258.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6701248', '6701450', '.', '+', '.', 'gene_id \"chr21.g1259\"; transcript_id \"chr21.g1259.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6724809', '6725182', '.', '-', '.', 'gene_id \"chr21.g1264\"; transcript_id \"chr21.g1264.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6725166', '6725685', '.', '-', '.', 'gene_id \"chr21.g1265\"; transcript_id \"chr21.g1265.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6741632', '6741930', '.', '+', '.', 'gene_id \"chr21.g1270\"; transcript_id \"chr21.g1270.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6735047', '6735253', '.', '+', '.', 'gene_id \"chr21.g1271\"; transcript_id \"chr21.g1271.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6908246', '6908466', '.', '+', '.', 'gene_id \"chr21.g1293\"; transcript_id \"chr21.g1293.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6936893', '6937762', '.', '-', '.', 'gene_id \"chr21.g1304\"; transcript_id \"chr21.g1304.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6945534', '6945795', '.', '+', '.', 'gene_id \"chr21.g1309\"; transcript_id \"chr21.g1309.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6940455', '6940736', '.', '-', '.', 'gene_id \"chr21.g1314\"; transcript_id \"chr21.g1314.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '22696652', '22696705', '.', '+', '.', 'gene_id \"chr38.g7212\"; transcript_id \"chr38.g7212.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6942020', '6942346', '.', '+', '.', 'gene_id \"chr21.g1317\"; transcript_id \"chr21.g1317.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6943642', '6943812', '.', '+', '.', 'gene_id \"chr21.g1318\"; transcript_id \"chr21.g1318.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6959926', '6960129', '.', '+', '.', 'gene_id \"chr21.g1324\"; transcript_id \"chr21.g1324.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6971838', '6972101', '.', '-', '.', 'gene_id \"chr21.g1330\"; transcript_id \"chr21.g1330.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6977388', '6977647', '.', '+', '.', 'gene_id \"chr21.g1333\"; transcript_id \"chr21.g1333.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7010054', '7010206', '.', '+', '.', 'gene_id \"chr21.g1337\"; transcript_id \"chr21.g1337.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6994106', '6994343', '.', '-', '.', 'gene_id \"chr21.g1348\"; transcript_id \"chr21.g1348.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7013818', '7014745', '.', '-', '.', 'gene_id \"chr21.g1351\"; transcript_id \"chr21.g1351.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '22785591', '22785621', '.', '-', '.', 'gene_id \"chr38.g7251\"; transcript_id \"chr38.g7251.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '22818588', '22818802', '.', '+', '.', 'gene_id \"chr38.g7265\"; transcript_id \"chr38.g7265.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7178421', '7178706', '.', '+', '.', 'gene_id \"chr21.g1372\"; transcript_id \"chr21.g1372.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7179761', '7180094', '.', '+', '.', 'gene_id \"chr21.g1375\"; transcript_id \"chr21.g1375.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7190215', '7190402', '.', '+', '.', 'gene_id \"chr21.g1389\"; transcript_id \"chr21.g1389.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7197801', '7197990', '.', '-', '.', 'gene_id \"chr21.g1393\"; transcript_id \"chr21.g1393.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7223449', '7223667', '.', '-', '.', 'gene_id \"chr21.g1401\"; transcript_id \"chr21.g1401.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7421627', '7421795', '.', '-', '.', 'gene_id \"chr21.g1419\"; transcript_id \"chr21.g1419.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7436035', '7436255', '.', '-', '.', 'gene_id \"chr21.g1422\"; transcript_id \"chr21.g1422.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7454329', '7454564', '.', '-', '.', 'gene_id \"chr21.g1426\"; transcript_id \"chr21.g1426.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7460476', '7460719', '.', '+', '.', 'gene_id \"chr21.g1429\"; transcript_id \"chr21.g1429.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7466407', '7466655', '.', '-', '.', 'gene_id \"chr21.g1439\"; transcript_id \"chr21.g1439.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23287435', '23287465', '.', '-', '.', 'gene_id \"chr38.g7350\"; transcript_id \"chr38.g7350.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7678002', '7678378', '.', '+', '.', 'gene_id \"chr21.g1459\"; transcript_id \"chr21.g1459.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7836989', '7837142', '.', '-', '.', 'gene_id \"chr21.g1473\"; transcript_id \"chr21.g1473.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7914059', '7914190', '.', '+', '.', 'gene_id \"chr21.g1481\"; transcript_id \"chr21.g1481.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23409897', '23410050', '.', '+', '.', 'gene_id \"chr38.g7397\"; transcript_id \"chr38.g7397.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23410056', '23410332', '.', '+', '.', 'gene_id \"chr38.g7398\"; transcript_id \"chr38.g7398.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '8278586', '8278748', '.', '-', '.', 'gene_id \"chr21.g1519\"; transcript_id \"chr21.g1519.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9598193', '9598630', '.', '-', '.', 'gene_id \"chr21.g1602\"; transcript_id \"chr21.g1602.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9839999', '9840209', '.', '+', '.', 'gene_id \"chr21.g1613\"; transcript_id \"chr21.g1613.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9919529', '9920820', '.', '-', '.', 'gene_id \"chr21.g1619\"; transcript_id \"chr21.g1619.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10068133', '10068365', '.', '-', '.', 'gene_id \"chr21.g1635\"; transcript_id \"chr21.g1635.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10124623', '10124955', '.', '+', '.', 'gene_id \"chr21.g1647\"; transcript_id \"chr21.g1647.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10110399', '10110696', '.', '-', '.', 'gene_id \"chr21.g1649\"; transcript_id \"chr21.g1649.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10118504', '10119716', '.', '-', '.', 'gene_id \"chr21.g1651\"; transcript_id \"chr21.g1651.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10130909', '10131107', '.', '+', '.', 'gene_id \"chr21.g1654\"; transcript_id \"chr21.g1654.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10133019', '10133842', '.', '+', '.', 'gene_id \"chr21.g1655\"; transcript_id \"chr21.g1655.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10135528', '10136302', '.', '-', '.', 'gene_id \"chr21.g1656\"; transcript_id \"chr21.g1656.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10161504', '10162847', '.', '-', '.', 'gene_id \"chr21.g1659\"; transcript_id \"chr21.g1659.i15\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10177151', '10177423', '.', '+', '.', 'gene_id \"chr21.g1662\"; transcript_id \"chr21.g1662.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10168836', '10169036', '.', '+', '.', 'gene_id \"chr21.g1664\"; transcript_id \"chr21.g1664.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10142363', '10142440', '.', '-', '.', 'gene_id \"chr21.g1672\"; transcript_id \"chr21.g1672.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10196300', '10196416', '.', '+', '.', 'gene_id \"chr21.g1685\"; transcript_id \"chr21.g1685.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10234514', '10234681', '.', '-', '.', 'gene_id \"chr21.g1694\"; transcript_id \"chr21.g1694.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23721120', '23721175', '.', '-', '.', 'gene_id \"chr38.g7611\"; transcript_id \"chr38.g7611.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10375476', '10375794', '.', '-', '.', 'gene_id \"chr21.g1713\"; transcript_id \"chr21.g1713.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10385649', '10386011', '.', '-', '.', 'gene_id \"chr21.g1724\"; transcript_id \"chr21.g1724.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10585777', '10586018', '.', '-', '.', 'gene_id \"chr21.g1749\"; transcript_id \"chr21.g1749.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10593235', '10593430', '.', '-', '.', 'gene_id \"chr21.g1752\"; transcript_id \"chr21.g1752.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10598469', '10598679', '.', '+', '.', 'gene_id \"chr21.g1755\"; transcript_id \"chr21.g1755.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23764540', '23764731', '.', '+', '.', 'gene_id \"chr38.g7653\"; transcript_id \"chr38.g7653.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10602014', '10602365', '.', '+', '.', 'gene_id \"chr21.g1759\"; transcript_id \"chr21.g1759.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10606951', '10607319', '.', '-', '.', 'gene_id \"chr21.g1765\"; transcript_id \"chr21.g1765.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10615908', '10616143', '.', '-', '.', 'gene_id \"chr21.g1772\"; transcript_id \"chr21.g1772.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10620669', '10620829', '.', '+', '.', 'gene_id \"chr21.g1778\"; transcript_id \"chr21.g1778.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10630342', '10630494', '.', '+', '.', 'gene_id \"chr21.g1781\"; transcript_id \"chr21.g1781.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '23078220', '23078301', '100', '-', '.', 'gene_id \"chr38.g7735\"; transcript_id \"blat.proc18128.chain_15784\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '4314327', '4314370', '100', '+', '.', 'gene_id \"chr38.g7807\"; transcript_id \"blat.proc4587.chain_4926\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1579394', '1579441', '100', '+', '.', 'gene_id \"chr38.g7810\"; transcript_id \"blat.proc18128.chain_24202\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '4987532', '4987597', '100', '+', '.', 'gene_id \"chr38.g7814\"; transcript_id \"blat.proc28144.chain_15949\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20323600', '20323647', '100', '-', '.', 'gene_id \"chr38.g7828\"; transcript_id \"blat.proc18128.chain_10360\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '13727512', '13727557', '100', '-', '.', 'gene_id \"chr38.g7866\"; transcript_id \"blat.proc4587.chain_12717\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '21593630', '21593695', '100', '-', '.', 'gene_id \"chr38.g8012\"; transcript_id \"blat.proc18128.chain_16455\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20200353', '20200405', '100', '+', '.', 'gene_id \"chr38.g8042\"; transcript_id \"blat.proc28144.chain_4606\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '1947497', '1947552', '100', '-', '.', 'gene_id \"chr38.g8122\"; transcript_id \"blat.proc28144.chain_1626\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '14998336', '14998368', '100', '+', '.', 'gene_id \"chr38.g8161\"; transcript_id \"blat.proc4005.chain_13683\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '5967793', '5967842', '100', '+', '.', 'gene_id \"chr38.g8165\"; transcript_id \"blat.proc4587.chain_12548\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '2895250', '2895319', '100', '-', '.', 'gene_id \"chr38.g8178\"; transcript_id \"blat.proc4587.chain_2269\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '10220694', '10220868', '100', '-', '.', 'gene_id \"chr38.g8222\"; transcript_id \"blat.proc28144.chain_20053\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20876878', '20876921', '100', '+', '.', 'gene_id \"chr38.g8258\"; transcript_id \"blat.proc18128.chain_3137\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '17762784', '17762822', '100', '-', '.', 'gene_id \"chr38.g8345\"; transcript_id \"blat.proc28144.chain_22870\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '21601069', '21601098', '100', '-', '.', 'gene_id \"chr38.g8354\"; transcript_id \"blat.proc4005.chain_5288\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '8030452', '8030487', '100', '-', '.', 'gene_id \"chr38.g8509\"; transcript_id \"blat.proc18128.chain_5469\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '16817285', '16817325', '100', '+', '.', 'gene_id \"chr38.g8553\"; transcript_id \"blat.proc18128.chain_20764\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '19672699', '19672730', '100', '+', '.', 'gene_id \"chr38.g8572\"; transcript_id \"blat.proc28144.chain_12146\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '3588605', '3588640', '100', '+', '.', 'gene_id \"chr38.g8614\"; transcript_id \"blat.proc18128.chain_25825\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '12175146', '12175200', '100', '+', '.', 'gene_id \"chr38.g8744\"; transcript_id \"blat.proc4005.chain_15292\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '9745593', '9745655', '100', '-', '.', 'gene_id \"chr38.g8817\"; transcript_id \"blat.proc28144.chain_5598\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10633254', '10633294', '.', '+', '.', 'gene_id \"chr21.g1782\"; transcript_id \"chr21.g1782.i1\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '5166511', '5166663', '100', '-', '.', 'gene_id \"chr38.g8861\"; transcript_id \"blat.proc4587.chain_7349\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '20522374', '20522431', '100', '+', '.', 'gene_id \"chr38.g9112\"; transcript_id \"blat.proc28144.chain_11405\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '10677975', '10678016', '100', '-', '.', 'gene_id \"chr38.g9161\"; transcript_id \"blat.proc18128.chain_19201\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '16814326', '16814358', '100', '+', '.', 'gene_id \"chr38.g9200\"; transcript_id \"blat.proc4005.chain_18164\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '7426312', '7426358', '100', '-', '.', 'gene_id \"chr38.g9370\"; transcript_id \"blat.proc4005.chain_19888\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '5017558', '5017600', '100', '-', '.', 'gene_id \"chr38.g9621\"; transcript_id \"blat.proc28144.chain_13467\";']\n",
      "negative:  ['chr38', 'PASA', 'exon', '8968690', '8968763', '100', '+', '.', 'gene_id \"chr38.g9637\"; transcript_id \"blat.proc28144.chain_3274\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10633789', '10634630', '.', '+', '.', 'gene_id \"chr21.g1783\"; transcript_id \"chr21.g1783.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10658130', '10658307', '.', '-', '.', 'gene_id \"chr21.g1794\"; transcript_id \"chr21.g1794.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10684350', '10684664', '.', '-', '.', 'gene_id \"chr21.g1799\"; transcript_id \"chr21.g1799.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10743306', '10743576', '.', '+', '.', 'gene_id \"chr21.g1817\"; transcript_id \"chr21.g1817.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11231994', '11232166', '.', '+', '.', 'gene_id \"chr21.g1848\"; transcript_id \"chr21.g1848.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11445606', '11445933', '.', '+', '.', 'gene_id \"chr21.g1873\"; transcript_id \"chr21.g1873.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11445912', '11446064', '.', '-', '.', 'gene_id \"chr21.g1874\"; transcript_id \"chr21.g1874.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11446066', '11446299', '.', '-', '.', 'gene_id \"chr21.g1875\"; transcript_id \"chr21.g1875.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11459522', '11460732', '.', '+', '.', 'gene_id \"chr21.g1878\"; transcript_id \"chr21.g1878.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11456239', '11456464', '.', '+', '.', 'gene_id \"chr21.g1882\"; transcript_id \"chr21.g1882.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11475869', '11476094', '.', '-', '.', 'gene_id \"chr21.g1888\"; transcript_id \"chr21.g1888.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11476079', '11476238', '.', '+', '.', 'gene_id \"chr21.g1889\"; transcript_id \"chr21.g1889.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11485305', '11485464', '.', '-', '.', 'gene_id \"chr21.g1892\"; transcript_id \"chr21.g1892.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11546041', '11547279', '.', '-', '.', 'gene_id \"chr21.g1900\"; transcript_id \"chr21.g1900.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11661663', '11662281', '.', '+', '.', 'gene_id \"chr21.g1913\"; transcript_id \"chr21.g1913.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11634274', '11634501', '.', '-', '.', 'gene_id \"chr21.g1917\"; transcript_id \"chr21.g1917.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11627556', '11627753', '.', '-', '.', 'gene_id \"chr21.g1919\"; transcript_id \"chr21.g1919.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11642871', '11643070', '.', '+', '.', 'gene_id \"chr21.g1924\"; transcript_id \"chr21.g1924.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11651447', '11652035', '.', '+', '.', 'gene_id \"chr21.g1929\"; transcript_id \"chr21.g1929.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11654175', '11654520', '.', '-', '.', 'gene_id \"chr21.g1931\"; transcript_id \"chr21.g1931.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11666728', '11666955', '.', '-', '.', 'gene_id \"chr21.g1937\"; transcript_id \"chr21.g1937.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11667898', '11668434', '.', '-', '.', 'gene_id \"chr21.g1939\"; transcript_id \"chr21.g1939.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11673019', '11673889', '.', '-', '.', 'gene_id \"chr21.g1943\"; transcript_id \"chr21.g1943.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11679854', '11680080', '.', '-', '.', 'gene_id \"chr21.g1949\"; transcript_id \"chr21.g1949.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11681675', '11681982', '.', '+', '.', 'gene_id \"chr21.g1951\"; transcript_id \"chr21.g1951.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11687084', '11687367', '.', '+', '.', 'gene_id \"chr21.g1954\"; transcript_id \"chr21.g1954.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11713065', '11713279', '.', '-', '.', 'gene_id \"chr21.g1969\"; transcript_id \"chr21.g1969.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11715490', '11715703', '.', '+', '.', 'gene_id \"chr21.g1972\"; transcript_id \"chr21.g1972.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11790409', '11790589', '.', '+', '.', 'gene_id \"chr21.g1991\"; transcript_id \"chr21.g1991.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11795335', '11795548', '.', '-', '.', 'gene_id \"chr21.g1997\"; transcript_id \"chr21.g1997.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12397499', '12397768', '.', '+', '.', 'gene_id \"chr21.g2044\"; transcript_id \"chr21.g2044.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12383668', '12383937', '.', '-', '.', 'gene_id \"chr21.g2046\"; transcript_id \"chr21.g2046.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12386962', '12387192', '.', '-', '.', 'gene_id \"chr21.g2047\"; transcript_id \"chr21.g2047.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12403119', '12403450', '.', '+', '.', 'gene_id \"chr21.g2058\"; transcript_id \"chr21.g2058.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12406747', '12406931', '.', '+', '.', 'gene_id \"chr21.g2065\"; transcript_id \"chr21.g2065.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12406913', '12407069', '.', '-', '.', 'gene_id \"chr21.g2066\"; transcript_id \"chr21.g2066.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12482618', '12482813', '.', '-', '.', 'gene_id \"chr21.g2090\"; transcript_id \"chr21.g2090.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12500400', '12500550', '.', '-', '.', 'gene_id \"chr21.g2092\"; transcript_id \"chr21.g2092.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12508018', '12508175', '.', '-', '.', 'gene_id \"chr21.g2096\"; transcript_id \"chr21.g2096.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12539944', '12540115', '.', '-', '.', 'gene_id \"chr21.g2109\"; transcript_id \"chr21.g2109.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12608473', '12608657', '.', '+', '.', 'gene_id \"chr21.g2126\"; transcript_id \"chr21.g2126.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12608710', '12610278', '.', '+', '.', 'gene_id \"chr21.g2127\"; transcript_id \"chr21.g2127.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12611711', '12612362', '.', '-', '.', 'gene_id \"chr21.g2128\"; transcript_id \"chr21.g2128.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12621168', '12621358', '.', '-', '.', 'gene_id \"chr21.g2128\"; transcript_id \"chr21.g2128.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12616184', '12616578', '.', '+', '.', 'gene_id \"chr21.g2133\"; transcript_id \"chr21.g2133.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12633266', '12633514', '.', '-', '.', 'gene_id \"chr21.g2137\"; transcript_id \"chr21.g2137.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12633529', '12634166', '.', '+', '.', 'gene_id \"chr21.g2138\"; transcript_id \"chr21.g2138.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12637072', '12637253', '.', '+', '.', 'gene_id \"chr21.g2139\"; transcript_id \"chr21.g2139.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12637485', '12637641', '.', '-', '.', 'gene_id \"chr21.g2140\"; transcript_id \"chr21.g2140.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12638021', '12638252', '.', '+', '.', 'gene_id \"chr21.g2141\"; transcript_id \"chr21.g2141.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12659625', '12659837', '.', '+', '.', 'gene_id \"chr21.g2157\"; transcript_id \"chr21.g2157.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12701570', '12701892', '.', '+', '.', 'gene_id \"chr21.g2164\"; transcript_id \"chr21.g2164.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12730862', '12731055', '.', '+', '.', 'gene_id \"chr21.g2173\"; transcript_id \"chr21.g2173.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12739882', '12740067', '.', '-', '.', 'gene_id \"chr21.g2177\"; transcript_id \"chr21.g2177.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12752669', '12752838', '.', '-', '.', 'gene_id \"chr21.g2179\"; transcript_id \"chr21.g2179.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12758108', '12758383', '.', '-', '.', 'gene_id \"chr21.g2188\"; transcript_id \"chr21.g2188.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12760074', '12760233', '.', '+', '.', 'gene_id \"chr21.g2189\"; transcript_id \"chr21.g2189.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12770267', '12770601', '.', '+', '.', 'gene_id \"chr21.g2193\"; transcript_id \"chr21.g2193.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12768061', '12768485', '.', '+', '.', 'gene_id \"chr21.g2196\"; transcript_id \"chr21.g2196.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12777415', '12777633', '.', '-', '.', 'gene_id \"chr21.g2203\"; transcript_id \"chr21.g2203.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12888023', '12888491', '.', '+', '.', 'gene_id \"chr21.g2214\"; transcript_id \"chr21.g2214.i5\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12888657', '12890335', '.', '+', '.', 'gene_id \"chr21.g2214\"; transcript_id \"chr21.g2214.i6\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13067413', '13067795', '.', '-', '.', 'gene_id \"chr21.g2216\"; transcript_id \"chr21.g2216.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13072583', '13072832', '.', '-', '.', 'gene_id \"chr21.g2219\"; transcript_id \"chr21.g2219.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13068268', '13068527', '.', '+', '.', 'gene_id \"chr21.g2223\"; transcript_id \"chr21.g2223.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12903963', '12904146', '.', '-', '.', 'gene_id \"chr21.g2226\"; transcript_id \"chr21.g2226.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12904428', '12904651', '.', '+', '.', 'gene_id \"chr21.g2227\"; transcript_id \"chr21.g2227.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12916993', '12917184', '.', '-', '.', 'gene_id \"chr21.g2232\"; transcript_id \"chr21.g2232.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12921487', '12921650', '.', '-', '.', 'gene_id \"chr21.g2233\"; transcript_id \"chr21.g2233.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12940595', '12941541', '.', '+', '.', 'gene_id \"chr21.g2239\"; transcript_id \"chr21.g2239.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12978117', '12978362', '.', '-', '.', 'gene_id \"chr21.g2250\"; transcript_id \"chr21.g2250.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12983955', '12984109', '.', '-', '.', 'gene_id \"chr21.g2255\"; transcript_id \"chr21.g2255.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12985416', '12985763', '.', '+', '.', 'gene_id \"chr21.g2256\"; transcript_id \"chr21.g2256.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12997106', '12997642', '.', '+', '.', 'gene_id \"chr21.g2258\"; transcript_id \"chr21.g2258.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13025449', '13025961', '.', '-', '.', 'gene_id \"chr21.g2271\"; transcript_id \"chr21.g2271.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13026156', '13026367', '.', '+', '.', 'gene_id \"chr21.g2272\"; transcript_id \"chr21.g2272.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13033931', '13034096', '.', '+', '.', 'gene_id \"chr21.g2278\"; transcript_id \"chr21.g2278.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13043534', '13043696', '.', '-', '.', 'gene_id \"chr21.g2284\"; transcript_id \"chr21.g2284.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13055178', '13055598', '.', '-', '.', 'gene_id \"chr21.g2297\"; transcript_id \"chr21.g2297.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13062842', '13063181', '.', '+', '.', 'gene_id \"chr21.g2306\"; transcript_id \"chr21.g2306.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13121743', '13121926', '.', '-', '.', 'gene_id \"chr21.g2318\"; transcript_id \"chr21.g2318.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13177739', '13178429', '.', '-', '.', 'gene_id \"chr21.g2329\"; transcript_id \"chr21.g2329.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13195840', '13197197', '.', '+', '.', 'gene_id \"chr21.g2342\"; transcript_id \"chr21.g2342.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13206018', '13206384', '.', '-', '.', 'gene_id \"chr21.g2350\"; transcript_id \"chr21.g2350.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13220343', '13220624', '.', '-', '.', 'gene_id \"chr21.g2358\"; transcript_id \"chr21.g2358.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13242458', '13243059', '.', '+', '.', 'gene_id \"chr21.g2365\"; transcript_id \"chr21.g2365.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13232516', '13234189', '.', '+', '.', 'gene_id \"chr21.g2369\"; transcript_id \"chr21.g2369.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13238006', '13238282', '.', '-', '.', 'gene_id \"chr21.g2372\"; transcript_id \"chr21.g2372.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13248267', '13248956', '.', '-', '.', 'gene_id \"chr21.g2377\"; transcript_id \"chr21.g2377.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13253036', '13253755', '.', '-', '.', 'gene_id \"chr21.g2379\"; transcript_id \"chr21.g2379.i1\";']\n",
      "negative:  ['scaffold_1099', 'PASA', 'exon', '8958', '8998', '.', '+', '.', 'gene_id \"scaffold_1099.g14\"; transcript_id \"scaffold_1099.g14.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13273265', '13273506', '.', '-', '.', 'gene_id \"chr21.g2396\"; transcript_id \"chr21.g2396.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13278216', '13278361', '.', '+', '.', 'gene_id \"chr21.g2399\"; transcript_id \"chr21.g2399.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13278440', '13278629', '.', '+', '.', 'gene_id \"chr21.g2400\"; transcript_id \"chr21.g2400.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13279101', '13279451', '.', '-', '.', 'gene_id \"chr21.g2402\"; transcript_id \"chr21.g2402.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13337279', '13337459', '.', '-', '.', 'gene_id \"chr21.g2412\"; transcript_id \"chr21.g2412.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13338380', '13338565', '.', '+', '.', 'gene_id \"chr21.g2414\"; transcript_id \"chr21.g2414.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13353543', '13353785', '.', '+', '.', 'gene_id \"chr21.g2415\"; transcript_id \"chr21.g2415.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13373347', '13373522', '.', '-', '.', 'gene_id \"chr21.g2418\"; transcript_id \"chr21.g2418.i1\";']\n",
      "negative:  ['scaffold_1352', 'PASA', 'exon', '8020', '8089', '.', '-', '.', 'gene_id \"scaffold_1352.g5\"; transcript_id \"scaffold_1352.g5.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13395676', '13395818', '.', '-', '.', 'gene_id \"chr21.g2433\"; transcript_id \"chr21.g2433.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13461645', '13462737', '.', '+', '.', 'gene_id \"chr21.g2447\"; transcript_id \"chr21.g2447.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13504258', '13504678', '.', '-', '.', 'gene_id \"chr21.g2452\"; transcript_id \"chr21.g2452.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13491856', '13492051', '.', '-', '.', 'gene_id \"chr21.g2453\"; transcript_id \"chr21.g2453.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13407845', '13408156', '.', '+', '.', 'gene_id \"chr21.g2457\"; transcript_id \"chr21.g2457.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13415621', '13415781', '.', '-', '.', 'gene_id \"chr21.g2464\"; transcript_id \"chr21.g2464.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13417311', '13417540', '.', '-', '.', 'gene_id \"chr21.g2468\"; transcript_id \"chr21.g2468.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13417311', '13417540', '.', '-', '.', 'gene_id \"chr21.g2468\"; transcript_id \"chr21.g2468.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13423822', '13424002', '.', '+', '.', 'gene_id \"chr21.g2472\"; transcript_id \"chr21.g2472.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13427845', '13428121', '.', '-', '.', 'gene_id \"chr21.g2476\"; transcript_id \"chr21.g2476.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13428179', '13428368', '.', '+', '.', 'gene_id \"chr21.g2478\"; transcript_id \"chr21.g2478.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13430568', '13430887', '.', '+', '.', 'gene_id \"chr21.g2480\"; transcript_id \"chr21.g2480.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13434081', '13434406', '.', '-', '.', 'gene_id \"chr21.g2482\"; transcript_id \"chr21.g2482.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13434482', '13434648', '.', '-', '.', 'gene_id \"chr21.g2485\"; transcript_id \"chr21.g2485.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13436790', '13436985', '.', '+', '.', 'gene_id \"chr21.g2487\"; transcript_id \"chr21.g2487.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13437080', '13437410', '.', '-', '.', 'gene_id \"chr21.g2488\"; transcript_id \"chr21.g2488.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13440227', '13440529', '.', '+', '.', 'gene_id \"chr21.g2494\"; transcript_id \"chr21.g2494.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13480075', '13480522', '.', '-', '.', 'gene_id \"chr21.g2519\"; transcript_id \"chr21.g2519.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13492744', '13492848', '.', '-', '.', 'gene_id \"chr21.g2523\"; transcript_id \"chr21.g2523.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13502250', '13502408', '.', '-', '.', 'gene_id \"chr21.g2529\"; transcript_id \"chr21.g2529.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13529950', '13530240', '.', '+', '.', 'gene_id \"chr21.g2536\"; transcript_id \"chr21.g2536.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13556229', '13556394', '.', '+', '.', 'gene_id \"chr21.g2537\"; transcript_id \"chr21.g2537.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13578299', '13578672', '.', '+', '.', 'gene_id \"chr21.g2545\"; transcript_id \"chr21.g2545.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13599825', '13599988', '.', '+', '.', 'gene_id \"chr21.g2551\"; transcript_id \"chr21.g2551.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13608472', '13608549', '.', '+', '.', 'gene_id \"chr21.g2553\"; transcript_id \"chr21.g2553.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13608472', '13608549', '.', '+', '.', 'gene_id \"chr21.g2553\"; transcript_id \"chr21.g2553.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13657606', '13658104', '.', '+', '.', 'gene_id \"chr21.g2553\"; transcript_id \"chr21.g2553.i4\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13707665', '13708678', '.', '+', '.', 'gene_id \"chr21.g2553\"; transcript_id \"chr21.g2553.i15\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13722467', '13723694', '.', '+', '.', 'gene_id \"chr21.g2553\"; transcript_id \"chr21.g2553.i17\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13713215', '13713464', '.', '+', '.', 'gene_id \"chr21.g2557\"; transcript_id \"chr21.g2557.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13666879', '13667047', '.', '-', '.', 'gene_id \"chr21.g2568\"; transcript_id \"chr21.g2568.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13679207', '13679399', '.', '+', '.', 'gene_id \"chr21.g2573\"; transcript_id \"chr21.g2573.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13711116', '13711348', '.', '-', '.', 'gene_id \"chr21.g2583\"; transcript_id \"chr21.g2583.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13720156', '13720604', '.', '+', '.', 'gene_id \"chr21.g2588\"; transcript_id \"chr21.g2588.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13720765', '13721024', '.', '+', '.', 'gene_id \"chr21.g2589\"; transcript_id \"chr21.g2589.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13727023', '13727169', '.', '+', '.', 'gene_id \"chr21.g2592\"; transcript_id \"chr21.g2592.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13727360', '13727658', '.', '-', '.', 'gene_id \"chr21.g2594\"; transcript_id \"chr21.g2594.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13757686', '13758789', '.', '-', '.', 'gene_id \"chr21.g2607\"; transcript_id \"chr21.g2607.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13761623', '13762578', '.', '+', '.', 'gene_id \"chr21.g2609\"; transcript_id \"chr21.g2609.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13777068', '13777262', '.', '-', '.', 'gene_id \"chr21.g2617\"; transcript_id \"chr21.g2617.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13786368', '13786525', '.', '+', '.', 'gene_id \"chr21.g2620\"; transcript_id \"chr21.g2620.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13787725', '13787926', '.', '-', '.', 'gene_id \"chr21.g2623\"; transcript_id \"chr21.g2623.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14203538', '14204239', '.', '+', '.', 'gene_id \"chr21.g2660\"; transcript_id \"chr21.g2660.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14246615', '14246787', '.', '-', '.', 'gene_id \"chr21.g2664\"; transcript_id \"chr21.g2664.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14367419', '14367577', '.', '-', '.', 'gene_id \"chr21.g2676\"; transcript_id \"chr21.g2676.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14441530', '14441788', '.', '+', '.', 'gene_id \"chr21.g2681\"; transcript_id \"chr21.g2681.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14444750', '14445107', '.', '+', '.', 'gene_id \"chr21.g2682\"; transcript_id \"chr21.g2682.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14448290', '14448491', '.', '-', '.', 'gene_id \"chr21.g2684\"; transcript_id \"chr21.g2684.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14543627', '14543860', '.', '+', '.', 'gene_id \"chr21.g2699\"; transcript_id \"chr21.g2699.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14544519', '14545038', '.', '-', '.', 'gene_id \"chr21.g2701\"; transcript_id \"chr21.g2701.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14552473', '14552808', '.', '+', '.', 'gene_id \"chr21.g2704\"; transcript_id \"chr21.g2704.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14679308', '14679634', '.', '-', '.', 'gene_id \"chr21.g2710\"; transcript_id \"chr21.g2710.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14806708', '14806995', '.', '-', '.', 'gene_id \"chr21.g2725\"; transcript_id \"chr21.g2725.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14807931', '14808597', '.', '-', '.', 'gene_id \"chr21.g2726\"; transcript_id \"chr21.g2726.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14925550', '14925714', '.', '-', '.', 'gene_id \"chr21.g2737\"; transcript_id \"chr21.g2737.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14925556', '14925714', '.', '-', '.', 'gene_id \"chr21.g2737\"; transcript_id \"chr21.g2737.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14966917', '14967926', '.', '+', '.', 'gene_id \"chr21.g2740\"; transcript_id \"chr21.g2740.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15007303', '15007440', '.', '+', '.', 'gene_id \"chr21.g2742\"; transcript_id \"chr21.g2742.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15301337', '15301526', '.', '+', '.', 'gene_id \"chr21.g2774\"; transcript_id \"chr21.g2774.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15379953', '15380110', '.', '+', '.', 'gene_id \"chr21.g2782\"; transcript_id \"chr21.g2782.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15442369', '15442564', '.', '+', '.', 'gene_id \"chr21.g2788\"; transcript_id \"chr21.g2788.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15455017', '15455366', '.', '-', '.', 'gene_id \"chr21.g2790\"; transcript_id \"chr21.g2790.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15472999', '15473239', '.', '-', '.', 'gene_id \"chr21.g2794\"; transcript_id \"chr21.g2794.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15534678', '15534938', '.', '+', '.', 'gene_id \"chr21.g2800\"; transcript_id \"chr21.g2800.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15597689', '15597958', '.', '-', '.', 'gene_id \"chr21.g2806\"; transcript_id \"chr21.g2806.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15617680', '15618058', '.', '+', '.', 'gene_id \"chr21.g2808\"; transcript_id \"chr21.g2808.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15618066', '15618470', '.', '+', '.', 'gene_id \"chr21.g2810\"; transcript_id \"chr21.g2810.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15637741', '15637787', '.', '+', '.', 'gene_id \"chr21.g2816\"; transcript_id \"chr21.g2816.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15691546', '15691994', '.', '+', '.', 'gene_id \"chr21.g2823\"; transcript_id \"chr21.g2823.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15696061', '15696350', '.', '-', '.', 'gene_id \"chr21.g2826\"; transcript_id \"chr21.g2826.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15698294', '15698482', '.', '-', '.', 'gene_id \"chr21.g2828\"; transcript_id \"chr21.g2828.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15709098', '15709144', '.', '+', '.', 'gene_id \"chr21.g2834\"; transcript_id \"chr21.g2834.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15901573', '15902232', '.', '-', '.', 'gene_id \"chr21.g2857\"; transcript_id \"chr21.g2857.i4\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15876178', '15876370', '.', '-', '.', 'gene_id \"chr21.g2859\"; transcript_id \"chr21.g2859.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15905814', '15906080', '.', '+', '.', 'gene_id \"chr21.g2861\"; transcript_id \"chr21.g2861.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15906422', '15906636', '.', '+', '.', 'gene_id \"chr21.g2862\"; transcript_id \"chr21.g2862.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '230510', '230683', '.', '+', '.', 'gene_id \"chr4.g83\"; transcript_id \"chr4.g83.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '251459', '251953', '.', '-', '.', 'gene_id \"chr4.g86\"; transcript_id \"chr4.g86.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15874764', '15875018', '.', '-', '.', 'gene_id \"chr21.g2884\"; transcript_id \"chr21.g2884.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15875724', '15875882', '.', '+', '.', 'gene_id \"chr21.g2886\"; transcript_id \"chr21.g2886.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15904299', '15904492', '.', '-', '.', 'gene_id \"chr21.g2909\"; transcript_id \"chr21.g2909.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15904299', '15904487', '.', '+', '.', 'gene_id \"chr21.g2910\"; transcript_id \"chr21.g2910.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15904618', '15904831', '.', '-', '.', 'gene_id \"chr21.g2911\"; transcript_id \"chr21.g2911.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15906941', '15907168', '.', '-', '.', 'gene_id \"chr21.g2912\"; transcript_id \"chr21.g2912.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15907658', '15907854', '.', '+', '.', 'gene_id \"chr21.g2914\"; transcript_id \"chr21.g2914.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15930283', '15930483', '.', '-', '.', 'gene_id \"chr21.g2919\"; transcript_id \"chr21.g2919.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15968158', '15969907', '.', '-', '.', 'gene_id \"chr21.g2932\"; transcript_id \"chr21.g2932.i8\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15970548', '15970806', '.', '-', '.', 'gene_id \"chr21.g2935\"; transcript_id \"chr21.g2935.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15973328', '15973569', '.', '+', '.', 'gene_id \"chr21.g2943\"; transcript_id \"chr21.g2943.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15992872', '15993562', '.', '-', '.', 'gene_id \"chr21.g2948\"; transcript_id \"chr21.g2948.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16031755', '16031928', '.', '-', '.', 'gene_id \"chr21.g2955\"; transcript_id \"chr21.g2955.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16041736', '16042089', '.', '+', '.', 'gene_id \"chr21.g2961\"; transcript_id \"chr21.g2961.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16051029', '16051363', '.', '+', '.', 'gene_id \"chr21.g2966\"; transcript_id \"chr21.g2966.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16062038', '16062278', '.', '-', '.', 'gene_id \"chr21.g2972\"; transcript_id \"chr21.g2972.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16064246', '16064410', '.', '-', '.', 'gene_id \"chr21.g2973\"; transcript_id \"chr21.g2973.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16065359', '16065559', '.', '+', '.', 'gene_id \"chr21.g2974\"; transcript_id \"chr21.g2974.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16065861', '16066012', '.', '-', '.', 'gene_id \"chr21.g2975\"; transcript_id \"chr21.g2975.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16066214', '16066369', '.', '-', '.', 'gene_id \"chr21.g2977\"; transcript_id \"chr21.g2977.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16066344', '16066559', '.', '+', '.', 'gene_id \"chr21.g2978\"; transcript_id \"chr21.g2978.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16099663', '16099836', '.', '+', '.', 'gene_id \"chr21.g2990\"; transcript_id \"chr21.g2990.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16116472', '16116627', '.', '-', '.', 'gene_id \"chr21.g2998\"; transcript_id \"chr21.g2998.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16116972', '16117158', '.', '-', '.', 'gene_id \"chr21.g2999\"; transcript_id \"chr21.g2999.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16130474', '16130629', '.', '-', '.', 'gene_id \"chr21.g3006\"; transcript_id \"chr21.g3006.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16163568', '16165240', '.', '-', '.', 'gene_id \"chr21.g3014\"; transcript_id \"chr21.g3014.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16227072', '16227345', '.', '-', '.', 'gene_id \"chr21.g3022\"; transcript_id \"chr21.g3022.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16183643', '16184606', '.', '-', '.', 'gene_id \"chr21.g3024\"; transcript_id \"chr21.g3024.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16221669', '16222032', '.', '+', '.', 'gene_id \"chr21.g3032\"; transcript_id \"chr21.g3032.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16221447', '16221671', '.', '-', '.', 'gene_id \"chr21.g3034\"; transcript_id \"chr21.g3034.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16238792', '16239083', '.', '+', '.', 'gene_id \"chr21.g3040\"; transcript_id \"chr21.g3040.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16240284', '16240607', '.', '-', '.', 'gene_id \"chr21.g3041\"; transcript_id \"chr21.g3041.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16241746', '16242007', '.', '-', '.', 'gene_id \"chr21.g3042\"; transcript_id \"chr21.g3042.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16243867', '16244637', '.', '-', '.', 'gene_id \"chr21.g3050\"; transcript_id \"chr21.g3050.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16253442', '16253504', '.', '-', '.', 'gene_id \"chr21.g3053\"; transcript_id \"chr21.g3053.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16324411', '16325219', '.', '-', '.', 'gene_id \"chr21.g3065\"; transcript_id \"chr21.g3065.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16326667', '16326798', '.', '-', '.', 'gene_id \"chr21.g3067\"; transcript_id \"chr21.g3067.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17280712', '17280750', '.', '-', '.', 'gene_id \"chr21.g3137\"; transcript_id \"chr21.g3137.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17925947', '17926191', '.', '+', '.', 'gene_id \"chr21.g3186\"; transcript_id \"chr21.g3186.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17990864', '17990903', '.', '-', '.', 'gene_id \"chr21.g3191\"; transcript_id \"chr21.g3191.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '18042089', '18042198', '.', '-', '.', 'gene_id \"chr21.g3196\"; transcript_id \"chr21.g3196.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '18135115', '18135438', '.', '-', '.', 'gene_id \"chr21.g3202\"; transcript_id \"chr21.g3202.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '18268988', '18269196', '.', '+', '.', 'gene_id \"chr21.g3208\"; transcript_id \"chr21.g3208.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '659935', '660144', '.', '-', '.', 'gene_id \"chr4.g428\"; transcript_id \"chr4.g428.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '18269471', '18269618', '.', '-', '.', 'gene_id \"chr21.g3209\"; transcript_id \"chr21.g3209.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '681801', '681959', '.', '-', '.', 'gene_id \"chr4.g430\"; transcript_id \"chr4.g430.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1023387', '1023418', '.', '-', '.', 'gene_id \"chr4.g567\"; transcript_id \"chr4.g567.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '19787242', '19787415', '.', '-', '.', 'gene_id \"chr21.g3381\"; transcript_id \"chr21.g3381.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1578418', '1578483', '.', '+', '.', 'gene_id \"chr4.g631\"; transcript_id \"chr4.g631.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '19885318', '19885475', '.', '-', '.', 'gene_id \"chr21.g3432\"; transcript_id \"chr21.g3432.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1796181', '1796415', '.', '-', '.', 'gene_id \"chr4.g659\"; transcript_id \"chr4.g659.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '2533938', '2534086', '.', '+', '.', 'gene_id \"chr4.g840\"; transcript_id \"chr4.g840.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20503892', '20504069', '.', '+', '.', 'gene_id \"chr21.g3797\"; transcript_id \"chr21.g3797.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20570549', '20570711', '.', '+', '.', 'gene_id \"chr21.g3853\"; transcript_id \"chr21.g3853.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20582378', '20582740', '.', '+', '.', 'gene_id \"chr21.g3864\"; transcript_id \"chr21.g3864.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20582753', '20582951', '.', '-', '.', 'gene_id \"chr21.g3865\"; transcript_id \"chr21.g3865.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '3468742', '3469378', '.', '+', '.', 'gene_id \"chr4.g1188\"; transcript_id \"chr4.g1188.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20864624', '20864782', '.', '+', '.', 'gene_id \"chr21.g4001\"; transcript_id \"chr21.g4001.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20889742', '20889800', '.', '-', '.', 'gene_id \"chr21.g4019\"; transcript_id \"chr21.g4019.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20918218', '20918386', '.', '+', '.', 'gene_id \"chr21.g4048\"; transcript_id \"chr21.g4048.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '20918369', '20918515', '.', '+', '.', 'gene_id \"chr21.g4048\"; transcript_id \"chr21.g4048.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '21008278', '21008496', '.', '+', '.', 'gene_id \"chr21.g4140\"; transcript_id \"chr21.g4140.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '21057114', '21057366', '.', '-', '.', 'gene_id \"chr21.g4179\"; transcript_id \"chr21.g4179.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '21282415', '21282460', '.', '-', '.', 'gene_id \"chr21.g4353\"; transcript_id \"chr21.g4353.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4311596', '4311741', '.', '+', '.', 'gene_id \"chr4.g1654\"; transcript_id \"chr4.g1654.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '21561299', '21561356', '.', '-', '.', 'gene_id \"chr21.g4445\"; transcript_id \"chr21.g4445.i16\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4520500', '4520681', '.', '+', '.', 'gene_id \"chr4.g1710\"; transcript_id \"chr4.g1710.i4\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4518757', '4518932', '.', '+', '.', 'gene_id \"chr4.g1719\"; transcript_id \"chr4.g1719.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4468600', '4468795', '.', '-', '.', 'gene_id \"chr4.g1724\"; transcript_id \"chr4.g1724.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4468889', '4469061', '.', '+', '.', 'gene_id \"chr4.g1725\"; transcript_id \"chr4.g1725.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4649896', '4650201', '.', '-', '.', 'gene_id \"chr4.g1854\"; transcript_id \"chr4.g1854.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22030329', '22030538', '.', '+', '.', 'gene_id \"chr21.g4716\"; transcript_id \"chr21.g4716.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22041198', '22041410', '.', '+', '.', 'gene_id \"chr21.g4722\"; transcript_id \"chr21.g4722.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22234843', '22235260', '.', '+', '.', 'gene_id \"chr21.g4811\"; transcript_id \"chr21.g4811.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22236524', '22236854', '.', '-', '.', 'gene_id \"chr21.g4818\"; transcript_id \"chr21.g4818.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22247326', '22247656', '.', '+', '.', 'gene_id \"chr21.g4832\"; transcript_id \"chr21.g4832.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22249963', '22250289', '.', '+', '.', 'gene_id \"chr21.g4835\"; transcript_id \"chr21.g4835.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22249963', '22250232', '.', '+', '.', 'gene_id \"chr21.g4835\"; transcript_id \"chr21.g4835.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22256197', '22256444', '.', '-', '.', 'gene_id \"chr21.g4840\"; transcript_id \"chr21.g4840.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22258107', '22258452', '.', '+', '.', 'gene_id \"chr21.g4843\"; transcript_id \"chr21.g4843.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22281847', '22282040', '.', '+', '.', 'gene_id \"chr21.g4863\"; transcript_id \"chr21.g4863.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22319237', '22319440', '.', '+', '.', 'gene_id \"chr21.g4887\"; transcript_id \"chr21.g4887.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22424777', '22424952', '.', '-', '.', 'gene_id \"chr21.g4943\"; transcript_id \"chr21.g4943.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22425024', '22425356', '.', '-', '.', 'gene_id \"chr21.g4944\"; transcript_id \"chr21.g4944.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5287158', '5287201', '.', '+', '.', 'gene_id \"chr4.g2202\"; transcript_id \"chr4.g2202.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5457082', '5457123', '.', '-', '.', 'gene_id \"chr4.g2253\"; transcript_id \"chr4.g2253.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5391584', '5391621', '.', '-', '.', 'gene_id \"chr4.g2262\"; transcript_id \"chr4.g2262.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5382946', '5383056', '.', '-', '.', 'gene_id \"chr4.g2282\"; transcript_id \"chr4.g2282.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22536869', '22537033', '.', '-', '.', 'gene_id \"chr21.g5026\"; transcript_id \"chr21.g5026.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22541676', '22542156', '.', '-', '.', 'gene_id \"chr21.g5027\"; transcript_id \"chr21.g5027.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22542160', '22542350', '.', '-', '.', 'gene_id \"chr21.g5028\"; transcript_id \"chr21.g5028.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22585450', '22585681', '.', '+', '.', 'gene_id \"chr21.g5054\"; transcript_id \"chr21.g5054.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22589895', '22590075', '.', '-', '.', 'gene_id \"chr21.g5062\"; transcript_id \"chr21.g5062.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22597786', '22597946', '.', '-', '.', 'gene_id \"chr21.g5077\"; transcript_id \"chr21.g5077.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22598236', '22598409', '.', '-', '.', 'gene_id \"chr21.g5078\"; transcript_id \"chr21.g5078.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22606106', '22606957', '.', '-', '.', 'gene_id \"chr21.g5082\"; transcript_id \"chr21.g5082.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22619945', '22620104', '.', '+', '.', 'gene_id \"chr21.g5089\"; transcript_id \"chr21.g5089.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22644078', '22644141', '.', '+', '.', 'gene_id \"chr21.g5098\"; transcript_id \"chr21.g5098.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22774357', '22774908', '.', '+', '.', 'gene_id \"chr21.g5127\"; transcript_id \"chr21.g5127.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22852897', '22853416', '.', '-', '.', 'gene_id \"chr21.g5144\"; transcript_id \"chr21.g5144.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22851618', '22851849', '.', '-', '.', 'gene_id \"chr21.g5160\"; transcript_id \"chr21.g5160.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22855496', '22855686', '.', '-', '.', 'gene_id \"chr21.g5165\"; transcript_id \"chr21.g5165.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22856970', '22857135', '.', '-', '.', 'gene_id \"chr21.g5169\"; transcript_id \"chr21.g5169.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22861560', '22861975', '.', '+', '.', 'gene_id \"chr21.g5176\"; transcript_id \"chr21.g5176.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23030403', '23030604', '.', '-', '.', 'gene_id \"chr21.g5212\"; transcript_id \"chr21.g5212.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5947419', '5947935', '.', '-', '.', 'gene_id \"chr4.g2467\"; transcript_id \"chr4.g2467.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23116455', '23116604', '.', '-', '.', 'gene_id \"chr21.g5249\"; transcript_id \"chr21.g5249.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23173733', '23173898', '.', '-', '.', 'gene_id \"chr21.g5267\"; transcript_id \"chr21.g5267.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr4', 'PASA', 'exon', '6052275', '6052442', '.', '+', '.', 'gene_id \"chr4.g2487\"; transcript_id \"chr4.g2487.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23195741', '23195924', '.', '-', '.', 'gene_id \"chr21.g5289\"; transcript_id \"chr21.g5289.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23200465', '23200645', '.', '-', '.', 'gene_id \"chr21.g5291\"; transcript_id \"chr21.g5291.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23205946', '23206122', '.', '+', '.', 'gene_id \"chr21.g5293\"; transcript_id \"chr21.g5293.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6186177', '6186429', '.', '-', '.', 'gene_id \"chr4.g2521\"; transcript_id \"chr4.g2521.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23253861', '23254105', '.', '-', '.', 'gene_id \"chr21.g5331\"; transcript_id \"chr21.g5331.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23258637', '23258791', '.', '-', '.', 'gene_id \"chr21.g5337\"; transcript_id \"chr21.g5337.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23391726', '23391966', '.', '-', '.', 'gene_id \"chr21.g5402\"; transcript_id \"chr21.g5402.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6412039', '6412234', '.', '+', '.', 'gene_id \"chr4.g2599\"; transcript_id \"chr4.g2599.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6412466', '6412985', '.', '-', '.', 'gene_id \"chr4.g2600\"; transcript_id \"chr4.g2600.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23471461', '23471664', '.', '-', '.', 'gene_id \"chr21.g5416\"; transcript_id \"chr21.g5416.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23472974', '23473216', '.', '+', '.', 'gene_id \"chr21.g5417\"; transcript_id \"chr21.g5417.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23476316', '23476484', '.', '+', '.', 'gene_id \"chr21.g5420\"; transcript_id \"chr21.g5420.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23488655', '23488798', '.', '+', '.', 'gene_id \"chr21.g5427\"; transcript_id \"chr21.g5427.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23488843', '23489565', '.', '-', '.', 'gene_id \"chr21.g5428\"; transcript_id \"chr21.g5428.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23489723', '23489895', '.', '+', '.', 'gene_id \"chr21.g5429\"; transcript_id \"chr21.g5429.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23497166', '23497425', '.', '-', '.', 'gene_id \"chr21.g5439\"; transcript_id \"chr21.g5439.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23506145', '23506331', '.', '-', '.', 'gene_id \"chr21.g5443\"; transcript_id \"chr21.g5443.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23509366', '23509586', '.', '-', '.', 'gene_id \"chr21.g5444\"; transcript_id \"chr21.g5444.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23509568', '23509720', '.', '+', '.', 'gene_id \"chr21.g5445\"; transcript_id \"chr21.g5445.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23539205', '23539406', '.', '-', '.', 'gene_id \"chr21.g5453\"; transcript_id \"chr21.g5453.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23542359', '23542595', '.', '-', '.', 'gene_id \"chr21.g5455\"; transcript_id \"chr21.g5455.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23542590', '23542839', '.', '-', '.', 'gene_id \"chr21.g5456\"; transcript_id \"chr21.g5456.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23544472', '23544696', '.', '-', '.', 'gene_id \"chr21.g5457\"; transcript_id \"chr21.g5457.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6760335', '6760424', '.', '+', '.', 'gene_id \"chr4.g2702\"; transcript_id \"chr4.g2702.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6745248', '6745329', '.', '+', '.', 'gene_id \"chr4.g2704\"; transcript_id \"chr4.g2704.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6809834', '6810007', '.', '+', '.', 'gene_id \"chr4.g2722\"; transcript_id \"chr4.g2722.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23631525', '23631724', '.', '+', '.', 'gene_id \"chr21.g5539\"; transcript_id \"chr21.g5539.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23631778', '23632269', '.', '-', '.', 'gene_id \"chr21.g5540\"; transcript_id \"chr21.g5540.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23636960', '23637202', '.', '+', '.', 'gene_id \"chr21.g5544\"; transcript_id \"chr21.g5544.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23646118', '23646478', '.', '-', '.', 'gene_id \"chr21.g5562\"; transcript_id \"chr21.g5562.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23654267', '23654473', '.', '-', '.', 'gene_id \"chr21.g5571\"; transcript_id \"chr21.g5571.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23765433', '23765864', '.', '-', '.', 'gene_id \"chr21.g5649\"; transcript_id \"chr21.g5649.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23765850', '23766055', '.', '+', '.', 'gene_id \"chr21.g5650\"; transcript_id \"chr21.g5650.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23766894', '23767060', '.', '-', '.', 'gene_id \"chr21.g5652\"; transcript_id \"chr21.g5652.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23774946', '23775146', '.', '+', '.', 'gene_id \"chr21.g5657\"; transcript_id \"chr21.g5657.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '7299113', '7299145', '.', '-', '.', 'gene_id \"chr4.g2875\"; transcript_id \"chr4.g2875.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23909293', '23909332', '.', '+', '.', 'gene_id \"chr21.g5699\"; transcript_id \"chr21.g5699.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23914690', '23914845', '.', '+', '.', 'gene_id \"chr21.g5704\"; transcript_id \"chr21.g5704.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23945345', '23945413', '.', '-', '.', 'gene_id \"chr21.g5725\"; transcript_id \"chr21.g5725.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24024753', '24025004', '.', '+', '.', 'gene_id \"chr21.g5775\"; transcript_id \"chr21.g5775.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24057112', '24057673', '.', '-', '.', 'gene_id \"chr21.g5813\"; transcript_id \"chr21.g5813.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24078160', '24078324', '.', '+', '.', 'gene_id \"chr21.g5826\"; transcript_id \"chr21.g5826.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '7948572', '7948877', '.', '+', '.', 'gene_id \"chr4.g3020\"; transcript_id \"chr4.g3020.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24137869', '24138573', '.', '-', '.', 'gene_id \"chr21.g5855\"; transcript_id \"chr21.g5855.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24220354', '24220507', '.', '+', '.', 'gene_id \"chr21.g5883\"; transcript_id \"chr21.g5883.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24222838', '24223147', '.', '-', '.', 'gene_id \"chr21.g5887\"; transcript_id \"chr21.g5887.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24223981', '24224219', '.', '-', '.', 'gene_id \"chr21.g5889\"; transcript_id \"chr21.g5889.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24226538', '24226867', '.', '+', '.', 'gene_id \"chr21.g5892\"; transcript_id \"chr21.g5892.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8002739', '8002770', '.', '-', '.', 'gene_id \"chr4.g3074\"; transcript_id \"chr4.g3074.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24245889', '24246060', '.', '+', '.', 'gene_id \"chr21.g5908\"; transcript_id \"chr21.g5908.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24277723', '24278235', '.', '-', '.', 'gene_id \"chr21.g5939\"; transcript_id \"chr21.g5939.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24293512', '24293560', '.', '-', '.', 'gene_id \"chr21.g5960\"; transcript_id \"chr21.g5960.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24326893', '24327478', '.', '-', '.', 'gene_id \"chr21.g5975\"; transcript_id \"chr21.g5975.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24327469', '24327649', '.', '-', '.', 'gene_id \"chr21.g5976\"; transcript_id \"chr21.g5976.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8122104', '8122151', '.', '-', '.', 'gene_id \"chr4.g3149\"; transcript_id \"chr4.g3149.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24353069', '24353641', '.', '+', '.', 'gene_id \"chr21.g6006\"; transcript_id \"chr21.g6006.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24364263', '24364477', '.', '-', '.', 'gene_id \"chr21.g6013\"; transcript_id \"chr21.g6013.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24399220', '24399697', '.', '+', '.', 'gene_id \"chr21.g6047\"; transcript_id \"chr21.g6047.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24440733', '24440923', '.', '-', '.', 'gene_id \"chr21.g6075\"; transcript_id \"chr21.g6075.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24452944', '24452990', '.', '+', '.', 'gene_id \"chr21.g6081\"; transcript_id \"chr21.g6081.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24456397', '24456606', '.', '-', '.', 'gene_id \"chr21.g6085\"; transcript_id \"chr21.g6085.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24462485', '24462584', '.', '-', '.', 'gene_id \"chr21.g6090\"; transcript_id \"chr21.g6090.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24492567', '24492767', '.', '-', '.', 'gene_id \"chr21.g6098\"; transcript_id \"chr21.g6098.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8374014', '8374046', '.', '+', '.', 'gene_id \"chr4.g3304\"; transcript_id \"chr4.g3304.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8375174', '8375216', '.', '+', '.', 'gene_id \"chr4.g3305\"; transcript_id \"chr4.g3305.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24561248', '24561572', '.', '+', '.', 'gene_id \"chr21.g6136\"; transcript_id \"chr21.g6136.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24810342', '24811120', '.', '+', '.', 'gene_id \"chr21.g6192\"; transcript_id \"chr21.g6192.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24715422', '24715574', '.', '+', '.', 'gene_id \"chr21.g6236\"; transcript_id \"chr21.g6236.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24744924', '24745249', '.', '+', '.', 'gene_id \"chr21.g6245\"; transcript_id \"chr21.g6245.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24758498', '24758829', '.', '-', '.', 'gene_id \"chr21.g6253\"; transcript_id \"chr21.g6253.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24833208', '24833387', '.', '+', '.', 'gene_id \"chr21.g6267\"; transcript_id \"chr21.g6267.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24877015', '24877301', '.', '+', '.', 'gene_id \"chr21.g6309\"; transcript_id \"chr21.g6309.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24877297', '24877529', '.', '-', '.', 'gene_id \"chr21.g6310\"; transcript_id \"chr21.g6310.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24879779', '24880008', '.', '-', '.', 'gene_id \"chr21.g6311\"; transcript_id \"chr21.g6311.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24880524', '24880685', '.', '-', '.', 'gene_id \"chr21.g6313\"; transcript_id \"chr21.g6313.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24904837', '24905032', '.', '+', '.', 'gene_id \"chr21.g6331\"; transcript_id \"chr21.g6331.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24907442', '24907672', '.', '-', '.', 'gene_id \"chr21.g6335\"; transcript_id \"chr21.g6335.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24996507', '24996706', '.', '+', '.', 'gene_id \"chr21.g6373\"; transcript_id \"chr21.g6373.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25026186', '25026337', '.', '+', '.', 'gene_id \"chr21.g6387\"; transcript_id \"chr21.g6387.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25075532', '25075786', '.', '+', '.', 'gene_id \"chr21.g6397\"; transcript_id \"chr21.g6397.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8830459', '8830508', '.', '+', '.', 'gene_id \"chr4.g3600\"; transcript_id \"chr4.g3600.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '8982860', '8982907', '.', '-', '.', 'gene_id \"chr4.g3618\"; transcript_id \"chr4.g3618.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25221798', '25222037', '.', '+', '.', 'gene_id \"chr21.g6445\"; transcript_id \"chr21.g6445.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25236791', '25236965', '.', '+', '.', 'gene_id \"chr21.g6473\"; transcript_id \"chr21.g6473.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25267670', '25267709', '.', '-', '.', 'gene_id \"chr21.g6485\"; transcript_id \"chr21.g6485.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25356811', '25356986', '.', '-', '.', 'gene_id \"chr21.g6529\"; transcript_id \"chr21.g6529.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25382397', '25382703', '.', '-', '.', 'gene_id \"chr21.g6550\"; transcript_id \"chr21.g6550.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '9310259', '9310290', '.', '-', '.', 'gene_id \"chr4.g3787\"; transcript_id \"chr4.g3787.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25484066', '25484286', '.', '-', '.', 'gene_id \"chr21.g6633\"; transcript_id \"chr21.g6633.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25488110', '25488286', '.', '+', '.', 'gene_id \"chr21.g6635\"; transcript_id \"chr21.g6635.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25535389', '25535741', '.', '+', '.', 'gene_id \"chr21.g6649\"; transcript_id \"chr21.g6649.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25536731', '25536968', '.', '-', '.', 'gene_id \"chr21.g6651\"; transcript_id \"chr21.g6651.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '9549860', '9550283', '.', '+', '.', 'gene_id \"chr4.g3830\"; transcript_id \"chr4.g3830.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25538043', '25538233', '.', '+', '.', 'gene_id \"chr21.g6653\"; transcript_id \"chr21.g6653.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25557557', '25557780', '.', '-', '.', 'gene_id \"chr21.g6670\"; transcript_id \"chr21.g6670.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '9645844', '9646080', '.', '+', '.', 'gene_id \"chr4.g3851\"; transcript_id \"chr4.g3851.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25568206', '25569395', '.', '+', '.', 'gene_id \"chr21.g6690\"; transcript_id \"chr21.g6690.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25569379', '25569610', '.', '+', '.', 'gene_id \"chr21.g6691\"; transcript_id \"chr21.g6691.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25609680', '25609846', '.', '+', '.', 'gene_id \"chr21.g6711\"; transcript_id \"chr21.g6711.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25625304', '25625584', '.', '+', '.', 'gene_id \"chr21.g6717\"; transcript_id \"chr21.g6717.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '9692751', '9693007', '.', '-', '.', 'gene_id \"chr4.g3902\"; transcript_id \"chr4.g3902.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25761801', '25762004', '.', '-', '.', 'gene_id \"chr21.g6738\"; transcript_id \"chr21.g6738.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25792630', '25792818', '.', '+', '.', 'gene_id \"chr21.g6749\"; transcript_id \"chr21.g6749.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25792807', '25793007', '.', '-', '.', 'gene_id \"chr21.g6750\"; transcript_id \"chr21.g6750.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25802268', '25802674', '.', '+', '.', 'gene_id \"chr21.g6752\"; transcript_id \"chr21.g6752.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25858997', '25859318', '.', '+', '.', 'gene_id \"chr21.g6760\"; transcript_id \"chr21.g6760.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25981697', '25982134', '.', '-', '.', 'gene_id \"chr21.g6839\"; transcript_id \"chr21.g6839.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25999197', '26000270', '.', '+', '.', 'gene_id \"chr21.g6849\"; transcript_id \"chr21.g6849.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26005975', '26006208', '.', '-', '.', 'gene_id \"chr21.g6854\"; transcript_id \"chr21.g6854.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26038438', '26038622', '.', '+', '.', 'gene_id \"chr21.g6888\"; transcript_id \"chr21.g6888.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26038991', '26039438', '.', '+', '.', 'gene_id \"chr21.g6889\"; transcript_id \"chr21.g6889.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26142551', '26142765', '.', '-', '.', 'gene_id \"chr21.g6944\"; transcript_id \"chr21.g6944.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26176930', '26177196', '.', '-', '.', 'gene_id \"chr21.g6975\"; transcript_id \"chr21.g6975.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26199292', '26199493', '.', '+', '.', 'gene_id \"chr21.g6988\"; transcript_id \"chr21.g6988.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26199295', '26199493', '.', '-', '.', 'gene_id \"chr21.g6989\"; transcript_id \"chr21.g6989.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26235072', '26235539', '.', '+', '.', 'gene_id \"chr21.g7046\"; transcript_id \"chr21.g7046.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26277936', '26278103', '.', '+', '.', 'gene_id \"chr21.g7070\"; transcript_id \"chr21.g7070.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26278237', '26278394', '.', '-', '.', 'gene_id \"chr21.g7071\"; transcript_id \"chr21.g7071.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26305461', '26305677', '.', '-', '.', 'gene_id \"chr21.g7082\"; transcript_id \"chr21.g7082.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26344502', '26344693', '.', '-', '.', 'gene_id \"chr21.g7101\"; transcript_id \"chr21.g7101.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26439636', '26439786', '.', '+', '.', 'gene_id \"chr21.g7177\"; transcript_id \"chr21.g7177.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26443815', '26444062', '.', '+', '.', 'gene_id \"chr21.g7179\"; transcript_id \"chr21.g7179.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26450405', '26450751', '.', '+', '.', 'gene_id \"chr21.g7184\"; transcript_id \"chr21.g7184.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26479253', '26479526', '.', '-', '.', 'gene_id \"chr21.g7194\"; transcript_id \"chr21.g7194.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '10847027', '10847065', '.', '+', '.', 'gene_id \"chr4.g4364\"; transcript_id \"chr4.g4364.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26558746', '26558899', '.', '+', '.', 'gene_id \"chr21.g7219\"; transcript_id \"chr21.g7219.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26596116', '26597115', '.', '-', '.', 'gene_id \"chr21.g7234\"; transcript_id \"chr21.g7234.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26623812', '26623965', '.', '-', '.', 'gene_id \"chr21.g7247\"; transcript_id \"chr21.g7247.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26773991', '26774022', '.', '+', '.', 'gene_id \"chr21.g7261\"; transcript_id \"chr21.g7261.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26977578', '26977794', '.', '+', '.', 'gene_id \"chr21.g7282\"; transcript_id \"chr21.g7282.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '27028774', '27029139', '.', '+', '.', 'gene_id \"chr21.g7300\"; transcript_id \"chr21.g7300.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '27205165', '27205402', '.', '+', '.', 'gene_id \"chr21.g7323\"; transcript_id \"chr21.g7323.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28177785', '28178241', '.', '+', '.', 'gene_id \"chr21.g7383\"; transcript_id \"chr21.g7383.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '11371137', '11371201', '.', '+', '.', 'gene_id \"chr4.g4532\"; transcript_id \"chr4.g4532.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28738317', '28738500', '.', '+', '.', 'gene_id \"chr21.g7446\"; transcript_id \"chr21.g7446.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28738563', '28738799', '.', '-', '.', 'gene_id \"chr21.g7447\"; transcript_id \"chr21.g7447.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28820201', '28820381', '.', '+', '.', 'gene_id \"chr21.g7478\"; transcript_id \"chr21.g7478.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28850825', '28851233', '.', '+', '.', 'gene_id \"chr21.g7502\"; transcript_id \"chr21.g7502.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29641361', '29641599', '.', '+', '.', 'gene_id \"chr21.g7595\"; transcript_id \"chr21.g7595.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29683792', '29683949', '.', '+', '.', 'gene_id \"chr21.g7602\"; transcript_id \"chr21.g7602.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29695439', '29696036', '.', '-', '.', 'gene_id \"chr21.g7611\"; transcript_id \"chr21.g7611.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29745464', '29745776', '.', '+', '.', 'gene_id \"chr21.g7644\"; transcript_id \"chr21.g7644.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29797652', '29797812', '.', '-', '.', 'gene_id \"chr21.g7669\"; transcript_id \"chr21.g7669.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29817751', '29818330', '.', '-', '.', 'gene_id \"chr21.g7685\"; transcript_id \"chr21.g7685.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29862572', '29862766', '.', '+', '.', 'gene_id \"chr21.g7700\"; transcript_id \"chr21.g7700.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29896556', '29896764', '.', '+', '.', 'gene_id \"chr21.g7711\"; transcript_id \"chr21.g7711.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29910680', '29910979', '.', '-', '.', 'gene_id \"chr21.g7720\"; transcript_id \"chr21.g7720.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29912082', '29912444', '.', '+', '.', 'gene_id \"chr21.g7722\"; transcript_id \"chr21.g7722.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29939917', '29940126', '.', '+', '.', 'gene_id \"chr21.g7732\"; transcript_id \"chr21.g7732.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29944328', '29944545', '.', '-', '.', 'gene_id \"chr21.g7737\"; transcript_id \"chr21.g7737.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '12417064', '12417258', '.', '+', '.', 'gene_id \"chr4.g4935\"; transcript_id \"chr4.g4935.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30019965', '30020115', '.', '+', '.', 'gene_id \"chr21.g7762\"; transcript_id \"chr21.g7762.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30020805', '30021050', '.', '+', '.', 'gene_id \"chr21.g7764\"; transcript_id \"chr21.g7764.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30069852', '30070328', '.', '-', '.', 'gene_id \"chr21.g7793\"; transcript_id \"chr21.g7793.i5\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30070356', '30070619', '.', '-', '.', 'gene_id \"chr21.g7793\"; transcript_id \"chr21.g7793.i6\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30103811', '30104305', '.', '+', '.', 'gene_id \"chr21.g7809\"; transcript_id \"chr21.g7809.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '12515153', '12515324', '.', '+', '.', 'gene_id \"chr4.g5000\"; transcript_id \"chr4.g5000.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30185759', '30185913', '.', '+', '.', 'gene_id \"chr21.g7849\"; transcript_id \"chr21.g7849.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30185800', '30185955', '.', '+', '.', 'gene_id \"chr21.g7849\"; transcript_id \"chr21.g7849.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30230845', '30231032', '.', '-', '.', 'gene_id \"chr21.g7851\"; transcript_id \"chr21.g7851.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30785442', '30785644', '.', '-', '.', 'gene_id \"chr21.g7913\"; transcript_id \"chr21.g7913.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30789814', '30790075', '.', '-', '.', 'gene_id \"chr21.g7916\"; transcript_id \"chr21.g7916.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30796288', '30796519', '.', '-', '.', 'gene_id \"chr21.g7924\"; transcript_id \"chr21.g7924.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30809925', '30810197', '.', '-', '.', 'gene_id \"chr21.g7938\"; transcript_id \"chr21.g7938.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30832670', '30832944', '.', '-', '.', 'gene_id \"chr21.g7953\"; transcript_id \"chr21.g7953.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '12767056', '12767248', '.', '-', '.', 'gene_id \"chr4.g5143\"; transcript_id \"chr4.g5143.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31122679', '31122829', '.', '-', '.', 'gene_id \"chr21.g8005\"; transcript_id \"chr21.g8005.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31261364', '31261624', '.', '+', '.', 'gene_id \"chr21.g8027\"; transcript_id \"chr21.g8027.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31282402', '31282637', '.', '+', '.', 'gene_id \"chr21.g8042\"; transcript_id \"chr21.g8042.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31311433', '31311591', '.', '+', '.', 'gene_id \"chr21.g8055\"; transcript_id \"chr21.g8055.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31353939', '31353973', '.', '-', '.', 'gene_id \"chr21.g8063\"; transcript_id \"chr21.g8063.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '13083008', '13083040', '.', '+', '.', 'gene_id \"chr4.g5233\"; transcript_id \"chr4.g5233.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31385374', '31385550', '.', '+', '.', 'gene_id \"chr21.g8091\"; transcript_id \"chr21.g8091.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31399580', '31399752', '.', '+', '.', 'gene_id \"chr21.g8098\"; transcript_id \"chr21.g8098.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31419729', '31420081', '.', '-', '.', 'gene_id \"chr21.g8113\"; transcript_id \"chr21.g8113.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31439819', '31440019', '.', '+', '.', 'gene_id \"chr21.g8125\"; transcript_id \"chr21.g8125.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31744127', '31744286', '.', '+', '.', 'gene_id \"chr21.g8191\"; transcript_id \"chr21.g8191.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31761440', '31761696', '.', '+', '.', 'gene_id \"chr21.g8206\"; transcript_id \"chr21.g8206.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31779087', '31779288', '.', '-', '.', 'gene_id \"chr21.g8212\"; transcript_id \"chr21.g8212.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31883317', '31883589', '.', '-', '.', 'gene_id \"chr21.g8246\"; transcript_id \"chr21.g8246.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31900032', '31900241', '.', '-', '.', 'gene_id \"chr21.g8252\"; transcript_id \"chr21.g8252.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31954619', '31954770', '.', '-', '.', 'gene_id \"chr21.g8272\"; transcript_id \"chr21.g8272.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31982051', '31982082', '.', '+', '.', 'gene_id \"chr21.g8283\"; transcript_id \"chr21.g8283.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32023758', '32023943', '.', '+', '.', 'gene_id \"chr21.g8295\"; transcript_id \"chr21.g8295.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32202148', '32202315', '.', '+', '.', 'gene_id \"chr21.g8330\"; transcript_id \"chr21.g8330.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32242612', '32243056', '.', '-', '.', 'gene_id \"chr21.g8342\"; transcript_id \"chr21.g8342.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32260734', '32261292', '.', '-', '.', 'gene_id \"chr21.g8356\"; transcript_id \"chr21.g8356.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32261275', '32261629', '.', '-', '.', 'gene_id \"chr21.g8357\"; transcript_id \"chr21.g8357.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32269319', '32269722', '.', '+', '.', 'gene_id \"chr21.g8361\"; transcript_id \"chr21.g8361.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '13947195', '13948021', '.', '+', '.', 'gene_id \"chr4.g5512\"; transcript_id \"chr4.g5512.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '13948020', '13948221', '.', '-', '.', 'gene_id \"chr4.g5513\"; transcript_id \"chr4.g5513.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32276871', '32277024', '.', '+', '.', 'gene_id \"chr21.g8392\"; transcript_id \"chr21.g8392.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32337868', '32338045', '.', '-', '.', 'gene_id \"chr21.g8414\"; transcript_id \"chr21.g8414.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32452026', '32452451', '.', '+', '.', 'gene_id \"chr21.g8457\"; transcript_id \"chr21.g8457.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32452452', '32452750', '.', '+', '.', 'gene_id \"chr21.g8459\"; transcript_id \"chr21.g8459.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32454077', '32455129', '.', '+', '.', 'gene_id \"chr21.g8461\"; transcript_id \"chr21.g8461.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '14092686', '14092880', '.', '+', '.', 'gene_id \"chr4.g5596\"; transcript_id \"chr4.g5596.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32538886', '32539038', '.', '-', '.', 'gene_id \"chr21.g8510\"; transcript_id \"chr21.g8510.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32539190', '32539525', '.', '-', '.', 'gene_id \"chr21.g8511\"; transcript_id \"chr21.g8511.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32556929', '32557120', '.', '-', '.', 'gene_id \"chr21.g8519\"; transcript_id \"chr21.g8519.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32654560', '32654713', '.', '+', '.', 'gene_id \"chr21.g8564\"; transcript_id \"chr21.g8564.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32675861', '32676100', '.', '+', '.', 'gene_id \"chr21.g8574\"; transcript_id \"chr21.g8574.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32697146', '32697381', '.', '-', '.', 'gene_id \"chr21.g8583\"; transcript_id \"chr21.g8583.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32697012', '32697163', '.', '-', '.', 'gene_id \"chr21.g8583\"; transcript_id \"chr21.g8583.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32696853', '32697031', '.', '+', '.', 'gene_id \"chr21.g8584\"; transcript_id \"chr21.g8584.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32723494', '32724146', '.', '+', '.', 'gene_id \"chr21.g8609\"; transcript_id \"chr21.g8609.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32879852', '32880009', '.', '+', '.', 'gene_id \"chr21.g8671\"; transcript_id \"chr21.g8671.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32863368', '32863423', '.', '+', '.', 'gene_id \"chr21.g8678\"; transcript_id \"chr21.g8678.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32865734', '32865954', '.', '-', '.', 'gene_id \"chr21.g8681\"; transcript_id \"chr21.g8681.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32951013', '32951062', '.', '-', '.', 'gene_id \"chr21.g8731\"; transcript_id \"chr21.g8731.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32953014', '32953231', '.', '+', '.', 'gene_id \"chr21.g8734\"; transcript_id \"chr21.g8734.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '14457207', '14457257', '.', '-', '.', 'gene_id \"chr4.g5847\"; transcript_id \"chr4.g5847.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33087550', '33087703', '.', '+', '.', 'gene_id \"chr21.g8797\"; transcript_id \"chr21.g8797.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33107686', '33107866', '.', '+', '.', 'gene_id \"chr21.g8818\"; transcript_id \"chr21.g8818.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '14561391', '14561653', '.', '-', '.', 'gene_id \"chr4.g5922\"; transcript_id \"chr4.g5922.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33118681', '33118905', '.', '-', '.', 'gene_id \"chr21.g8824\"; transcript_id \"chr21.g8824.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '14562046', '14562296', '.', '+', '.', 'gene_id \"chr4.g5923\"; transcript_id \"chr4.g5923.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33174710', '33174864', '.', '+', '.', 'gene_id \"chr21.g8862\"; transcript_id \"chr21.g8862.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33198650', '33198833', '.', '-', '.', 'gene_id \"chr21.g8911\"; transcript_id \"chr21.g8911.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33207150', '33207339', '.', '+', '.', 'gene_id \"chr21.g8920\"; transcript_id \"chr21.g8920.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33256017', '33256178', '.', '-', '.', 'gene_id \"chr21.g8937\"; transcript_id \"chr21.g8937.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33325134', '33325289', '.', '-', '.', 'gene_id \"chr21.g8967\"; transcript_id \"chr21.g8967.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33330767', '33331200', '.', '+', '.', 'gene_id \"chr21.g8968\"; transcript_id \"chr21.g8968.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33342170', '33342372', '.', '-', '.', 'gene_id \"chr21.g8974\"; transcript_id \"chr21.g8974.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33365645', '33365829', '.', '-', '.', 'gene_id \"chr21.g8983\"; transcript_id \"chr21.g8983.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33397323', '33397551', '.', '-', '.', 'gene_id \"chr21.g8995\"; transcript_id \"chr21.g8995.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33463190', '33463481', '.', '+', '.', 'gene_id \"chr21.g9019\"; transcript_id \"chr21.g9019.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33483161', '33483364', '.', '+', '.', 'gene_id \"chr21.g9021\"; transcript_id \"chr21.g9021.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33489896', '33490194', '.', '-', '.', 'gene_id \"chr21.g9029\"; transcript_id \"chr21.g9029.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33583602', '33583758', '.', '+', '.', 'gene_id \"chr21.g9064\"; transcript_id \"chr21.g9064.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15213103', '15213335', '.', '+', '.', 'gene_id \"chr4.g6145\"; transcript_id \"chr4.g6145.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33597966', '33598260', '.', '+', '.', 'gene_id \"chr21.g9071\"; transcript_id \"chr21.g9071.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15243234', '15243283', '.', '+', '.', 'gene_id \"chr4.g6184\"; transcript_id \"chr4.g6184.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33744872', '33745069', '.', '-', '.', 'gene_id \"chr21.g9107\"; transcript_id \"chr21.g9107.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33746796', '33746966', '.', '-', '.', 'gene_id \"chr21.g9109\"; transcript_id \"chr21.g9109.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33778742', '33779169', '.', '+', '.', 'gene_id \"chr21.g9126\"; transcript_id \"chr21.g9126.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33819504', '33819886', '.', '-', '.', 'gene_id \"chr21.g9157\"; transcript_id \"chr21.g9157.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33826168', '33826501', '.', '-', '.', 'gene_id \"chr21.g9163\"; transcript_id \"chr21.g9163.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33834056', '33834409', '.', '-', '.', 'gene_id \"chr21.g9173\"; transcript_id \"chr21.g9173.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33874612', '33875637', '.', '-', '.', 'gene_id \"chr21.g9195\"; transcript_id \"chr21.g9195.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33901310', '33901814', '.', '+', '.', 'gene_id \"chr21.g9196\"; transcript_id \"chr21.g9196.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33891621', '33893927', '.', '+', '.', 'gene_id \"chr21.g9200\"; transcript_id \"chr21.g9200.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33888541', '33888716', '.', '+', '.', 'gene_id \"chr21.g9207\"; transcript_id \"chr21.g9207.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33904774', '33905268', '.', '-', '.', 'gene_id \"chr21.g9211\"; transcript_id \"chr21.g9211.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33939246', '33939411', '.', '+', '.', 'gene_id \"chr21.g9218\"; transcript_id \"chr21.g9218.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33999467', '33999613', '.', '+', '.', 'gene_id \"chr21.g9232\"; transcript_id \"chr21.g9232.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34031102', '34031395', '.', '+', '.', 'gene_id \"chr21.g9235\"; transcript_id \"chr21.g9235.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34124437', '34124821', '.', '+', '.', 'gene_id \"chr21.g9268\"; transcript_id \"chr21.g9268.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34134398', '34135549', '.', '-', '.', 'gene_id \"chr21.g9274\"; transcript_id \"chr21.g9274.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34136269', '34136419', '.', '-', '.', 'gene_id \"chr21.g9276\"; transcript_id \"chr21.g9276.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15378771', '15378817', '.', '-', '.', 'gene_id \"chr4.g6359\"; transcript_id \"chr4.g6359.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34433619', '34433656', '.', '-', '.', 'gene_id \"chr21.g9361\"; transcript_id \"chr21.g9361.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34525891', '34526075', '.', '+', '.', 'gene_id \"chr21.g9383\"; transcript_id \"chr21.g9383.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34529388', '34529580', '.', '+', '.', 'gene_id \"chr21.g9385\"; transcript_id \"chr21.g9385.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34548354', '34548615', '.', '+', '.', 'gene_id \"chr21.g9394\"; transcript_id \"chr21.g9394.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34548603', '34549059', '.', '-', '.', 'gene_id \"chr21.g9395\"; transcript_id \"chr21.g9395.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34550358', '34550822', '.', '-', '.', 'gene_id \"chr21.g9396\"; transcript_id \"chr21.g9396.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34560015', '34560176', '.', '+', '.', 'gene_id \"chr21.g9397\"; transcript_id \"chr21.g9397.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34751695', '34751895', '.', '+', '.', 'gene_id \"chr21.g9433\"; transcript_id \"chr21.g9433.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34757400', '34757618', '.', '-', '.', 'gene_id \"chr21.g9436\"; transcript_id \"chr21.g9436.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34793520', '34793707', '.', '+', '.', 'gene_id \"chr21.g9453\"; transcript_id \"chr21.g9453.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15569367', '15569398', '.', '-', '.', 'gene_id \"chr4.g6536\"; transcript_id \"chr4.g6536.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34798453', '34798641', '.', '-', '.', 'gene_id \"chr21.g9458\"; transcript_id \"chr21.g9458.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34838407', '34838718', '.', '-', '.', 'gene_id \"chr21.g9477\"; transcript_id \"chr21.g9477.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34917636', '34917686', '.', '+', '.', 'gene_id \"chr21.g9487\"; transcript_id \"chr21.g9487.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35051279', '35051476', '.', '+', '.', 'gene_id \"chr21.g9510\"; transcript_id \"chr21.g9510.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35085399', '35085533', '.', '+', '.', 'gene_id \"chr21.g9519\"; transcript_id \"chr21.g9519.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15607043', '15607088', '.', '+', '.', 'gene_id \"chr4.g6609\"; transcript_id \"chr4.g6609.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35126305', '35126458', '.', '+', '.', 'gene_id \"chr21.g9551\"; transcript_id \"chr21.g9551.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35128350', '35128979', '.', '-', '.', 'gene_id \"chr21.g9555\"; transcript_id \"chr21.g9555.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35171021', '35171634', '.', '-', '.', 'gene_id \"chr21.g9572\"; transcript_id \"chr21.g9572.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35172543', '35172741', '.', '+', '.', 'gene_id \"chr21.g9573\"; transcript_id \"chr21.g9573.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35193708', '35193861', '.', '-', '.', 'gene_id \"chr21.g9609\"; transcript_id \"chr21.g9609.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35199022', '35199269', '.', '-', '.', 'gene_id \"chr21.g9615\"; transcript_id \"chr21.g9615.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35200572', '35200874', '.', '-', '.', 'gene_id \"chr21.g9616\"; transcript_id \"chr21.g9616.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35209899', '35210146', '.', '+', '.', 'gene_id \"chr21.g9625\"; transcript_id \"chr21.g9625.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15987170', '15987806', '.', '+', '.', 'gene_id \"chr4.g6699\"; transcript_id \"chr4.g6699.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35301130', '35301300', '.', '+', '.', 'gene_id \"chr21.g9655\"; transcript_id \"chr21.g9655.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16376521', '16376571', '.', '+', '.', 'gene_id \"chr4.g6718\"; transcript_id \"chr4.g6718.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35308731', '35309005', '.', '-', '.', 'gene_id \"chr21.g9661\"; transcript_id \"chr21.g9661.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35381921', '35382234', '.', '+', '.', 'gene_id \"chr21.g9671\"; transcript_id \"chr21.g9671.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16562587', '16562908', '.', '+', '.', 'gene_id \"chr4.g6731\"; transcript_id \"chr4.g6731.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16820686', '16820829', '.', '+', '.', 'gene_id \"chr4.g6752\"; transcript_id \"chr4.g6752.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16941964', '16942005', '.', '-', '.', 'gene_id \"chr4.g6768\"; transcript_id \"chr4.g6768.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35491784', '35491934', '.', '+', '.', 'gene_id \"chr21.g9711\"; transcript_id \"chr21.g9711.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35494101', '35494420', '.', '-', '.', 'gene_id \"chr21.g9714\"; transcript_id \"chr21.g9714.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17252434', '17252605', '.', '+', '.', 'gene_id \"chr4.g6793\"; transcript_id \"chr4.g6793.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17266578', '17266855', '.', '+', '.', 'gene_id \"chr4.g6797\"; transcript_id \"chr4.g6797.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35523476', '35524000', '.', '-', '.', 'gene_id \"chr21.g9749\"; transcript_id \"chr21.g9749.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35524641', '35524923', '.', '-', '.', 'gene_id \"chr21.g9751\"; transcript_id \"chr21.g9751.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35528580', '35528945', '.', '+', '.', 'gene_id \"chr21.g9755\"; transcript_id \"chr21.g9755.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35541861', '35542132', '.', '+', '.', 'gene_id \"chr21.g9769\"; transcript_id \"chr21.g9769.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35542127', '35542318', '.', '-', '.', 'gene_id \"chr21.g9770\"; transcript_id \"chr21.g9770.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35543418', '35543581', '.', '+', '.', 'gene_id \"chr21.g9772\"; transcript_id \"chr21.g9772.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17732486', '17732520', '.', '+', '.', 'gene_id \"chr4.g6828\"; transcript_id \"chr4.g6828.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17766449', '17766485', '.', '-', '.', 'gene_id \"chr4.g6833\"; transcript_id \"chr4.g6833.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35581565', '35581720', '.', '+', '.', 'gene_id \"chr21.g9786\"; transcript_id \"chr21.g9786.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35587022', '35587209', '.', '+', '.', 'gene_id \"chr21.g9790\"; transcript_id \"chr21.g9790.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35627737', '35627934', '.', '+', '.', 'gene_id \"chr21.g9804\"; transcript_id \"chr21.g9804.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18117815', '18117880', '.', '-', '.', 'gene_id \"chr4.g6879\"; transcript_id \"chr4.g6879.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18273166', '18273204', '.', '-', '.', 'gene_id \"chr4.g6903\"; transcript_id \"chr4.g6903.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35819177', '35819610', '.', '-', '.', 'gene_id \"chr21.g9891\"; transcript_id \"chr21.g9891.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18548278', '18548455', '.', '-', '.', 'gene_id \"chr4.g6958\"; transcript_id \"chr4.g6958.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35905499', '35906175', '.', '+', '.', 'gene_id \"chr21.g9920\"; transcript_id \"chr21.g9920.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18597806', '18598054', '.', '-', '.', 'gene_id \"chr4.g6988\"; transcript_id \"chr4.g6988.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35954349', '35954591', '.', '-', '.', 'gene_id \"chr21.g9961\"; transcript_id \"chr21.g9961.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36021205', '36021427', '.', '-', '.', 'gene_id \"chr21.g9993\"; transcript_id \"chr21.g9993.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36021680', '36021994', '.', '+', '.', 'gene_id \"chr21.g9994\"; transcript_id \"chr21.g9994.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36021680', '36021843', '.', '+', '.', 'gene_id \"chr21.g9994\"; transcript_id \"chr21.g9994.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18719049', '18719313', '.', '+', '.', 'gene_id \"chr4.g7034\"; transcript_id \"chr4.g7034.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36030594', '36030814', '.', '+', '.', 'gene_id \"chr21.g10002\"; transcript_id \"chr21.g10002.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36052564', '36052721', '.', '+', '.', 'gene_id \"chr21.g10004\"; transcript_id \"chr21.g10004.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36079266', '36079501', '.', '-', '.', 'gene_id \"chr21.g10013\"; transcript_id \"chr21.g10013.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19014499', '19014737', '.', '+', '.', 'gene_id \"chr4.g7070\"; transcript_id \"chr4.g7070.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36366021', '36366257', '.', '-', '.', 'gene_id \"chr21.g10047\"; transcript_id \"chr21.g10047.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36399720', '36399941', '.', '+', '.', 'gene_id \"chr21.g10049\"; transcript_id \"chr21.g10049.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36405849', '36406064', '.', '+', '.', 'gene_id \"chr21.g10056\"; transcript_id \"chr21.g10056.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36422429', '36422814', '.', '-', '.', 'gene_id \"chr21.g10063\"; transcript_id \"chr21.g10063.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36422795', '36423030', '.', '+', '.', 'gene_id \"chr21.g10064\"; transcript_id \"chr21.g10064.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36436910', '36437288', '.', '-', '.', 'gene_id \"chr21.g10071\"; transcript_id \"chr21.g10071.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36432822', '36433230', '.', '-', '.', 'gene_id \"chr21.g10076\"; transcript_id \"chr21.g10076.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36434833', '36435097', '.', '+', '.', 'gene_id \"chr21.g10079\"; transcript_id \"chr21.g10079.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36460971', '36461042', '.', '-', '.', 'gene_id \"chr21.g10088\"; transcript_id \"chr21.g10088.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36467523', '36467807', '.', '+', '.', 'gene_id \"chr21.g10093\"; transcript_id \"chr21.g10093.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36520334', '36520488', '.', '+', '.', 'gene_id \"chr21.g10097\"; transcript_id \"chr21.g10097.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36469791', '36470029', '.', '+', '.', 'gene_id \"chr21.g10098\"; transcript_id \"chr21.g10098.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36470369', '36470589', '.', '+', '.', 'gene_id \"chr21.g10099\"; transcript_id \"chr21.g10099.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36491318', '36491474', '.', '-', '.', 'gene_id \"chr21.g10109\"; transcript_id \"chr21.g10109.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36528666', '36528937', '.', '-', '.', 'gene_id \"chr21.g10117\"; transcript_id \"chr21.g10117.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36530429', '36531090', '.', '+', '.', 'gene_id \"chr21.g10121\"; transcript_id \"chr21.g10121.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36535344', '36535693', '.', '+', '.', 'gene_id \"chr21.g10124\"; transcript_id \"chr21.g10124.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36539630', '36539789', '.', '+', '.', 'gene_id \"chr21.g10125\"; transcript_id \"chr21.g10125.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36561711', '36562017', '.', '+', '.', 'gene_id \"chr21.g10130\"; transcript_id \"chr21.g10130.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36564153', '36564387', '.', '-', '.', 'gene_id \"chr21.g10134\"; transcript_id \"chr21.g10134.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36585772', '36586357', '.', '+', '.', 'gene_id \"chr21.g10143\"; transcript_id \"chr21.g10143.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36626264', '36626456', '.', '-', '.', 'gene_id \"chr21.g10147\"; transcript_id \"chr21.g10147.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36661541', '36661717', '.', '-', '.', 'gene_id \"chr21.g10159\"; transcript_id \"chr21.g10159.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36661854', '36662167', '.', '+', '.', 'gene_id \"chr21.g10160\"; transcript_id \"chr21.g10160.i2\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr21', 'PASA', 'exon', '36670360', '36670771', '.', '+', '.', 'gene_id \"chr21.g10165\"; transcript_id \"chr21.g10165.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36692746', '36692965', '.', '+', '.', 'gene_id \"chr21.g10167\"; transcript_id \"chr21.g10167.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36735524', '36735932', '.', '-', '.', 'gene_id \"chr21.g10174\"; transcript_id \"chr21.g10174.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36741470', '36741626', '.', '-', '.', 'gene_id \"chr21.g10179\"; transcript_id \"chr21.g10179.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36764168', '36764335', '.', '-', '.', 'gene_id \"chr21.g10184\"; transcript_id \"chr21.g10184.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36766128', '36766316', '.', '+', '.', 'gene_id \"chr21.g10185\"; transcript_id \"chr21.g10185.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36986951', '36987120', '.', '-', '.', 'gene_id \"chr21.g10204\"; transcript_id \"chr21.g10204.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37061612', '37061896', '.', '-', '.', 'gene_id \"chr21.g10210\"; transcript_id \"chr21.g10210.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37065209', '37065604', '.', '-', '.', 'gene_id \"chr21.g10213\"; transcript_id \"chr21.g10213.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37096435', '37096588', '.', '-', '.', 'gene_id \"chr21.g10216\"; transcript_id \"chr21.g10216.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37119093', '37119258', '.', '+', '.', 'gene_id \"chr21.g10221\"; transcript_id \"chr21.g10221.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37134050', '37134220', '.', '-', '.', 'gene_id \"chr21.g10222\"; transcript_id \"chr21.g10222.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37159990', '37160321', '.', '-', '.', 'gene_id \"chr21.g10226\"; transcript_id \"chr21.g10226.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37160951', '37161199', '.', '-', '.', 'gene_id \"chr21.g10227\"; transcript_id \"chr21.g10227.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37161316', '37161484', '.', '-', '.', 'gene_id \"chr21.g10228\"; transcript_id \"chr21.g10228.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37162049', '37162288', '.', '+', '.', 'gene_id \"chr21.g10230\"; transcript_id \"chr21.g10230.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37197566', '37197845', '.', '-', '.', 'gene_id \"chr21.g10247\"; transcript_id \"chr21.g10247.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37199923', '37200472', '.', '-', '.', 'gene_id \"chr21.g10250\"; transcript_id \"chr21.g10250.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37200559', '37200834', '.', '-', '.', 'gene_id \"chr21.g10252\"; transcript_id \"chr21.g10252.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37205638', '37205901', '.', '-', '.', 'gene_id \"chr21.g10256\"; transcript_id \"chr21.g10256.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37226008', '37226282', '.', '-', '.', 'gene_id \"chr21.g10270\"; transcript_id \"chr21.g10270.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37234524', '37234870', '.', '-', '.', 'gene_id \"chr21.g10275\"; transcript_id \"chr21.g10275.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37235396', '37235770', '.', '-', '.', 'gene_id \"chr21.g10277\"; transcript_id \"chr21.g10277.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37244963', '37245188', '.', '-', '.', 'gene_id \"chr21.g10282\"; transcript_id \"chr21.g10282.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37248248', '37248454', '.', '-', '.', 'gene_id \"chr21.g10285\"; transcript_id \"chr21.g10285.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37266457', '37266608', '.', '+', '.', 'gene_id \"chr21.g10288\"; transcript_id \"chr21.g10288.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37320111', '37320303', '.', '-', '.', 'gene_id \"chr21.g10295\"; transcript_id \"chr21.g10295.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37339913', '37340152', '.', '-', '.', 'gene_id \"chr21.g10312\"; transcript_id \"chr21.g10312.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37341563', '37341732', '.', '-', '.', 'gene_id \"chr21.g10314\"; transcript_id \"chr21.g10314.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37370455', '37370871', '.', '-', '.', 'gene_id \"chr21.g10320\"; transcript_id \"chr21.g10320.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37371921', '37372199', '.', '+', '.', 'gene_id \"chr21.g10322\"; transcript_id \"chr21.g10322.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37375060', '37375436', '.', '+', '.', 'gene_id \"chr21.g10328\"; transcript_id \"chr21.g10328.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19574333', '19574382', '.', '+', '.', 'gene_id \"chr4.g7370\"; transcript_id \"chr4.g7370.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19576068', '19576169', '.', '+', '.', 'gene_id \"chr4.g7372\"; transcript_id \"chr4.g7372.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37392884', '37393160', '.', '-', '.', 'gene_id \"chr21.g10334\"; transcript_id \"chr21.g10334.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37491532', '37491566', '.', '+', '.', 'gene_id \"chr21.g10351\"; transcript_id \"chr21.g10351.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37499162', '37499347', '.', '+', '.', 'gene_id \"chr21.g10353\"; transcript_id \"chr21.g10353.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37614199', '37614693', '.', '+', '.', 'gene_id \"chr21.g10359\"; transcript_id \"chr21.g10359.i4\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37501904', '37502137', '.', '+', '.', 'gene_id \"chr21.g10362\"; transcript_id \"chr21.g10362.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37508180', '37508421', '.', '+', '.', 'gene_id \"chr21.g10364\"; transcript_id \"chr21.g10364.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37534137', '37534296', '.', '-', '.', 'gene_id \"chr21.g10372\"; transcript_id \"chr21.g10372.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37546801', '37546973', '.', '+', '.', 'gene_id \"chr21.g10379\"; transcript_id \"chr21.g10379.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37551605', '37551814', '.', '+', '.', 'gene_id \"chr21.g10380\"; transcript_id \"chr21.g10380.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37571681', '37571901', '.', '-', '.', 'gene_id \"chr21.g10387\"; transcript_id \"chr21.g10387.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37583894', '37584198', '.', '+', '.', 'gene_id \"chr21.g10391\"; transcript_id \"chr21.g10391.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37584877', '37585042', '.', '-', '.', 'gene_id \"chr21.g10392\"; transcript_id \"chr21.g10392.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37595704', '37596066', '.', '+', '.', 'gene_id \"chr21.g10397\"; transcript_id \"chr21.g10397.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37604804', '37605298', '.', '+', '.', 'gene_id \"chr21.g10399\"; transcript_id \"chr21.g10399.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37605571', '37605779', '.', '+', '.', 'gene_id \"chr21.g10402\"; transcript_id \"chr21.g10402.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37615582', '37615757', '.', '+', '.', 'gene_id \"chr21.g10407\"; transcript_id \"chr21.g10407.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37621063', '37621367', '.', '+', '.', 'gene_id \"chr21.g10411\"; transcript_id \"chr21.g10411.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37621358', '37621579', '.', '-', '.', 'gene_id \"chr21.g10412\"; transcript_id \"chr21.g10412.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37623454', '37624794', '.', '-', '.', 'gene_id \"chr21.g10414\"; transcript_id \"chr21.g10414.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37632611', '37632821', '.', '+', '.', 'gene_id \"chr21.g10422\"; transcript_id \"chr21.g10422.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37642399', '37642622', '.', '-', '.', 'gene_id \"chr21.g10427\"; transcript_id \"chr21.g10427.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37645095', '37645312', '.', '+', '.', 'gene_id \"chr21.g10430\"; transcript_id \"chr21.g10430.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37684522', '37684697', '.', '-', '.', 'gene_id \"chr21.g10448\"; transcript_id \"chr21.g10448.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19758243', '19758787', '.', '+', '.', 'gene_id \"chr4.g7511\"; transcript_id \"chr4.g7511.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37866962', '37866995', '.', '+', '.', 'gene_id \"chr21.g10479\"; transcript_id \"chr21.g10479.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38090042', '38090076', '.', '-', '.', 'gene_id \"chr21.g10499\"; transcript_id \"chr21.g10499.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38554943', '38555191', '.', '+', '.', 'gene_id \"chr21.g10522\"; transcript_id \"chr21.g10522.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38560285', '38560640', '.', '-', '.', 'gene_id \"chr21.g10526\"; transcript_id \"chr21.g10526.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38561882', '38562003', '.', '-', '.', 'gene_id \"chr21.g10526\"; transcript_id \"chr21.g10526.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38603929', '38604106', '.', '-', '.', 'gene_id \"chr21.g10534\"; transcript_id \"chr21.g10534.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38604683', '38604858', '.', '+', '.', 'gene_id \"chr21.g10535\"; transcript_id \"chr21.g10535.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38604837', '38605101', '.', '-', '.', 'gene_id \"chr21.g10536\"; transcript_id \"chr21.g10536.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38666880', '38667132', '.', '-', '.', 'gene_id \"chr21.g10552\"; transcript_id \"chr21.g10552.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38669066', '38669439', '.', '+', '.', 'gene_id \"chr21.g10555\"; transcript_id \"chr21.g10555.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38809081', '38809288', '.', '+', '.', 'gene_id \"chr21.g10576\"; transcript_id \"chr21.g10576.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38809994', '38810292', '.', '-', '.', 'gene_id \"chr21.g10578\"; transcript_id \"chr21.g10578.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20095837', '20096007', '.', '-', '.', 'gene_id \"chr4.g7665\"; transcript_id \"chr4.g7665.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38854101', '38854462', '.', '-', '.', 'gene_id \"chr21.g10585\"; transcript_id \"chr21.g10585.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20115841', '20116049', '.', '-', '.', 'gene_id \"chr4.g7685\"; transcript_id \"chr4.g7685.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38989172', '38989407', '.', '+', '.', 'gene_id \"chr21.g10602\"; transcript_id \"chr21.g10602.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39034150', '39034301', '.', '-', '.', 'gene_id \"chr21.g10608\"; transcript_id \"chr21.g10608.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20139510', '20139671', '.', '-', '.', 'gene_id \"chr4.g7696\"; transcript_id \"chr4.g7696.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39267050', '39267429', '.', '-', '.', 'gene_id \"chr21.g10623\"; transcript_id \"chr21.g10623.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39288623', '39288857', '.', '-', '.', 'gene_id \"chr21.g10628\"; transcript_id \"chr21.g10628.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39298171', '39298467', '.', '+', '.', 'gene_id \"chr21.g10631\"; transcript_id \"chr21.g10631.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39328364', '39328433', '.', '-', '.', 'gene_id \"chr21.g10636\"; transcript_id \"chr21.g10636.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39372838', '39373257', '.', '-', '.', 'gene_id \"chr21.g10639\"; transcript_id \"chr21.g10639.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39387380', '39387582', '.', '-', '.', 'gene_id \"chr21.g10643\"; transcript_id \"chr21.g10643.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39406389', '39406806', '.', '-', '.', 'gene_id \"chr21.g10644\"; transcript_id \"chr21.g10644.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39435300', '39435815', '.', '-', '.', 'gene_id \"chr21.g10651\"; transcript_id \"chr21.g10651.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39484552', '39484764', '.', '-', '.', 'gene_id \"chr21.g10658\"; transcript_id \"chr21.g10658.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39538693', '39538949', '.', '-', '.', 'gene_id \"chr21.g10671\"; transcript_id \"chr21.g10671.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39540859', '39541432', '.', '-', '.', 'gene_id \"chr21.g10674\"; transcript_id \"chr21.g10674.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39542036', '39542348', '.', '-', '.', 'gene_id \"chr21.g10677\"; transcript_id \"chr21.g10677.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39543723', '39543885', '.', '-', '.', 'gene_id \"chr21.g10678\"; transcript_id \"chr21.g10678.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39634643', '39634727', '.', '-', '.', 'gene_id \"chr21.g10679\"; transcript_id \"chr21.g10679.i4\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39543946', '39544318', '.', '+', '.', 'gene_id \"chr21.g10682\"; transcript_id \"chr21.g10682.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39552568', '39552849', '.', '-', '.', 'gene_id \"chr21.g10688\"; transcript_id \"chr21.g10688.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39553082', '39553285', '.', '-', '.', 'gene_id \"chr21.g10690\"; transcript_id \"chr21.g10690.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39573584', '39573795', '.', '+', '.', 'gene_id \"chr21.g10694\"; transcript_id \"chr21.g10694.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39623370', '39623636', '.', '+', '.', 'gene_id \"chr21.g10704\"; transcript_id \"chr21.g10704.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39642727', '39643041', '.', '+', '.', 'gene_id \"chr21.g10707\"; transcript_id \"chr21.g10707.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39643051', '39643293', '.', '+', '.', 'gene_id \"chr21.g10708\"; transcript_id \"chr21.g10708.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39717414', '39717571', '.', '-', '.', 'gene_id \"chr21.g10717\"; transcript_id \"chr21.g10717.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39799270', '39799481', '.', '-', '.', 'gene_id \"chr21.g10737\"; transcript_id \"chr21.g10737.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39802074', '39802422', '.', '-', '.', 'gene_id \"chr21.g10742\"; transcript_id \"chr21.g10742.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39803642', '39804281', '.', '+', '.', 'gene_id \"chr21.g10746\"; transcript_id \"chr21.g10746.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39843336', '39843697', '.', '-', '.', 'gene_id \"chr21.g10752\"; transcript_id \"chr21.g10752.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39831119', '39831293', '.', '+', '.', 'gene_id \"chr21.g10768\"; transcript_id \"chr21.g10768.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39831600', '39831945', '.', '+', '.', 'gene_id \"chr21.g10769\"; transcript_id \"chr21.g10769.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39846633', '39846804', '.', '-', '.', 'gene_id \"chr21.g10774\"; transcript_id \"chr21.g10774.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39884391', '39884575', '.', '+', '.', 'gene_id \"chr21.g10778\"; transcript_id \"chr21.g10778.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39895196', '39895830', '.', '+', '.', 'gene_id \"chr21.g10781\"; transcript_id \"chr21.g10781.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39917377', '39917592', '.', '+', '.', 'gene_id \"chr21.g10791\"; transcript_id \"chr21.g10791.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39931118', '39931356', '.', '+', '.', 'gene_id \"chr21.g10793\"; transcript_id \"chr21.g10793.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39935296', '39935777', '.', '-', '.', 'gene_id \"chr21.g10799\"; transcript_id \"chr21.g10799.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39936897', '39937142', '.', '+', '.', 'gene_id \"chr21.g10802\"; transcript_id \"chr21.g10802.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39938826', '39938877', '.', '+', '.', 'gene_id \"chr21.g10803\"; transcript_id \"chr21.g10803.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39941232', '39941418', '.', '+', '.', 'gene_id \"chr21.g10805\"; transcript_id \"chr21.g10805.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39943150', '39943352', '.', '-', '.', 'gene_id \"chr21.g10809\"; transcript_id \"chr21.g10809.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39946871', '39947119', '.', '-', '.', 'gene_id \"chr21.g10810\"; transcript_id \"chr21.g10810.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39972401', '39972567', '.', '-', '.', 'gene_id \"chr21.g10816\"; transcript_id \"chr21.g10816.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39980909', '39981095', '.', '-', '.', 'gene_id \"chr21.g10819\"; transcript_id \"chr21.g10819.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40040363', '40040750', '.', '+', '.', 'gene_id \"chr21.g10836\"; transcript_id \"chr21.g10836.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40041062', '40041379', '.', '-', '.', 'gene_id \"chr21.g10841\"; transcript_id \"chr21.g10841.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40069255', '40069531', '.', '-', '.', 'gene_id \"chr21.g10852\"; transcript_id \"chr21.g10852.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40078889', '40079257', '.', '-', '.', 'gene_id \"chr21.g10856\"; transcript_id \"chr21.g10856.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40078729', '40078894', '.', '-', '.', 'gene_id \"chr21.g10857\"; transcript_id \"chr21.g10857.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40082557', '40082746', '.', '-', '.', 'gene_id \"chr21.g10863\"; transcript_id \"chr21.g10863.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40101284', '40101510', '.', '+', '.', 'gene_id \"chr21.g10865\"; transcript_id \"chr21.g10865.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40123959', '40124111', '.', '+', '.', 'gene_id \"chr21.g10866\"; transcript_id \"chr21.g10866.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40171698', '40171832', '.', '-', '.', 'gene_id \"chr21.g10871\"; transcript_id \"chr21.g10871.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40190102', '40190372', '.', '-', '.', 'gene_id \"chr21.g10886\"; transcript_id \"chr21.g10886.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40193926', '40194281', '.', '-', '.', 'gene_id \"chr21.g10890\"; transcript_id \"chr21.g10890.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40195875', '40196032', '.', '-', '.', 'gene_id \"chr21.g10892\"; transcript_id \"chr21.g10892.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40198180', '40198428', '.', '-', '.', 'gene_id \"chr21.g10894\"; transcript_id \"chr21.g10894.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40239400', '40239579', '.', '+', '.', 'gene_id \"chr21.g10898\"; transcript_id \"chr21.g10898.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40279975', '40280239', '.', '+', '.', 'gene_id \"chr21.g10908\"; transcript_id \"chr21.g10908.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40373125', '40373305', '.', '-', '.', 'gene_id \"chr21.g10921\"; transcript_id \"chr21.g10921.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40441572', '40441958', '.', '-', '.', 'gene_id \"chr21.g10937\"; transcript_id \"chr21.g10937.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40445928', '40446380', '.', '+', '.', 'gene_id \"chr21.g10942\"; transcript_id \"chr21.g10942.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40452214', '40452432', '.', '+', '.', 'gene_id \"chr21.g10952\"; transcript_id \"chr21.g10952.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20566288', '20566333', '.', '-', '.', 'gene_id \"chr4.g8042\"; transcript_id \"chr4.g8042.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40454381', '40454660', '.', '+', '.', 'gene_id \"chr21.g10955\"; transcript_id \"chr21.g10955.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40457580', '40457886', '.', '+', '.', 'gene_id \"chr21.g10960\"; transcript_id \"chr21.g10960.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20583749', '20583819', '.', '+', '.', 'gene_id \"chr4.g8068\"; transcript_id \"chr4.g8068.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40458927', '40459109', '.', '+', '.', 'gene_id \"chr21.g10961\"; transcript_id \"chr21.g10961.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40462798', '40463123', '.', '+', '.', 'gene_id \"chr21.g10965\"; transcript_id \"chr21.g10965.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40470399', '40470565', '.', '+', '.', 'gene_id \"chr21.g10972\"; transcript_id \"chr21.g10972.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40471834', '40471999', '.', '+', '.', 'gene_id \"chr21.g10974\"; transcript_id \"chr21.g10974.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40597701', '40597888', '.', '-', '.', 'gene_id \"chr21.g11002\"; transcript_id \"chr21.g11002.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40597905', '40598939', '.', '-', '.', 'gene_id \"chr21.g11003\"; transcript_id \"chr21.g11003.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40625253', '40625416', '.', '+', '.', 'gene_id \"chr21.g11017\"; transcript_id \"chr21.g11017.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40632063', '40632557', '.', '+', '.', 'gene_id \"chr21.g11019\"; transcript_id \"chr21.g11019.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40653997', '40654159', '.', '+', '.', 'gene_id \"chr21.g11022\"; transcript_id \"chr21.g11022.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40713225', '40713432', '.', '+', '.', 'gene_id \"chr21.g11037\"; transcript_id \"chr21.g11037.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40743527', '40744733', '.', '-', '.', 'gene_id \"chr21.g11049\"; transcript_id \"chr21.g11049.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40745264', '40745364', '.', '+', '.', 'gene_id \"chr21.g11050\"; transcript_id \"chr21.g11050.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40782170', '40782400', '.', '-', '.', 'gene_id \"chr21.g11056\"; transcript_id \"chr21.g11056.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40800270', '40800376', '.', '-', '.', 'gene_id \"chr21.g11057\"; transcript_id \"chr21.g11057.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40800851', '40801081', '.', '-', '.', 'gene_id \"chr21.g11058\"; transcript_id \"chr21.g11058.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40788319', '40788463', '.', '+', '.', 'gene_id \"chr21.g11062\"; transcript_id \"chr21.g11062.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40792324', '40792520', '.', '-', '.', 'gene_id \"chr21.g11065\"; transcript_id \"chr21.g11065.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40807564', '40807708', '.', '+', '.', 'gene_id \"chr21.g11068\"; transcript_id \"chr21.g11068.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40811570', '40811766', '.', '-', '.', 'gene_id \"chr21.g11071\"; transcript_id \"chr21.g11071.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40819503', '40819609', '.', '-', '.', 'gene_id \"chr21.g11072\"; transcript_id \"chr21.g11072.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40851533', '40851698', '.', '-', '.', 'gene_id \"chr21.g11091\"; transcript_id \"chr21.g11091.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40903605', '40904677', '.', '+', '.', 'gene_id \"chr21.g11096\"; transcript_id \"chr21.g11096.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40918563', '40918973', '.', '+', '.', 'gene_id \"chr21.g11103\"; transcript_id \"chr21.g11103.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40952453', '40953287', '.', '-', '.', 'gene_id \"chr21.g11116\"; transcript_id \"chr21.g11116.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40948275', '40948582', '.', '-', '.', 'gene_id \"chr21.g11120\"; transcript_id \"chr21.g11120.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41030873', '41031841', '.', '-', '.', 'gene_id \"chr21.g11139\"; transcript_id \"chr21.g11139.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41041297', '41042032', '.', '-', '.', 'gene_id \"chr21.g11140\"; transcript_id \"chr21.g11140.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41022935', '41023238', '.', '-', '.', 'gene_id \"chr21.g11142\"; transcript_id \"chr21.g11142.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41009653', '41009875', '.', '+', '.', 'gene_id \"chr21.g11148\"; transcript_id \"chr21.g11148.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41009861', '41010013', '.', '-', '.', 'gene_id \"chr21.g11149\"; transcript_id \"chr21.g11149.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41018027', '41018329', '.', '-', '.', 'gene_id \"chr21.g11152\"; transcript_id \"chr21.g11152.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41038888', '41039008', '.', '+', '.', 'gene_id \"chr21.g11162\"; transcript_id \"chr21.g11162.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41039561', '41041278', '.', '-', '.', 'gene_id \"chr21.g11164\"; transcript_id \"chr21.g11164.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41046176', '41046749', '.', '-', '.', 'gene_id \"chr21.g11166\"; transcript_id \"chr21.g11166.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41060524', '41060942', '.', '+', '.', 'gene_id \"chr21.g11175\"; transcript_id \"chr21.g11175.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41077354', '41077503', '.', '-', '.', 'gene_id \"chr21.g11183\"; transcript_id \"chr21.g11183.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41104386', '41105257', '.', '-', '.', 'gene_id \"chr21.g11187\"; transcript_id \"chr21.g11187.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41088526', '41088815', '.', '+', '.', 'gene_id \"chr21.g11193\"; transcript_id \"chr21.g11193.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41094724', '41095049', '.', '+', '.', 'gene_id \"chr21.g11198\"; transcript_id \"chr21.g11198.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41100490', '41100706', '.', '+', '.', 'gene_id \"chr21.g11203\"; transcript_id \"chr21.g11203.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41107272', '41107446', '.', '+', '.', 'gene_id \"chr21.g11207\"; transcript_id \"chr21.g11207.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41114347', '41114567', '.', '-', '.', 'gene_id \"chr21.g11211\"; transcript_id \"chr21.g11211.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41120674', '41121477', '.', '+', '.', 'gene_id \"chr21.g11214\"; transcript_id \"chr21.g11214.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41125389', '41125588', '.', '+', '.', 'gene_id \"chr21.g11218\"; transcript_id \"chr21.g11218.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41132903', '41133115', '.', '+', '.', 'gene_id \"chr21.g11231\"; transcript_id \"chr21.g11231.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41134270', '41135379', '.', '-', '.', 'gene_id \"chr21.g11235\"; transcript_id \"chr21.g11235.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41136241', '41136404', '.', '+', '.', 'gene_id \"chr21.g11238\"; transcript_id \"chr21.g11238.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41149760', '41149922', '.', '+', '.', 'gene_id \"chr21.g11249\"; transcript_id \"chr21.g11249.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41164777', '41164957', '.', '+', '.', 'gene_id \"chr21.g11259\"; transcript_id \"chr21.g11259.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41164982', '41165543', '.', '-', '.', 'gene_id \"chr21.g11260\"; transcript_id \"chr21.g11260.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41167182', '41167375', '.', '-', '.', 'gene_id \"chr21.g11261\"; transcript_id \"chr21.g11261.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41167716', '41168359', '.', '-', '.', 'gene_id \"chr21.g11263\"; transcript_id \"chr21.g11263.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41245013', '41245228', '.', '+', '.', 'gene_id \"chr21.g11281\"; transcript_id \"chr21.g11281.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41279859', '41280031', '.', '-', '.', 'gene_id \"chr21.g11298\"; transcript_id \"chr21.g11298.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41286877', '41287045', '.', '+', '.', 'gene_id \"chr21.g11300\"; transcript_id \"chr21.g11300.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41321548', '41321703', '.', '-', '.', 'gene_id \"chr21.g11304\"; transcript_id \"chr21.g11304.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41383211', '41383280', '.', '+', '.', 'gene_id \"chr21.g11314\"; transcript_id \"chr21.g11314.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41394907', '41395131', '.', '+', '.', 'gene_id \"chr21.g11316\"; transcript_id \"chr21.g11316.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41403702', '41403909', '.', '-', '.', 'gene_id \"chr21.g11319\"; transcript_id \"chr21.g11319.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41410702', '41410870', '.', '-', '.', 'gene_id \"chr21.g11323\"; transcript_id \"chr21.g11323.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41416897', '41416951', '.', '+', '.', 'gene_id \"chr21.g11328\"; transcript_id \"chr21.g11328.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41439240', '41439504', '.', '+', '.', 'gene_id \"chr21.g11329\"; transcript_id \"chr21.g11329.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41424529', '41425901', '.', '-', '.', 'gene_id \"chr21.g11330\"; transcript_id \"chr21.g11330.i3\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41426184', '41427730', '.', '+', '.', 'gene_id \"chr21.g11331\"; transcript_id \"chr21.g11331.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41446294', '41446850', '.', '+', '.', 'gene_id \"chr21.g11334\"; transcript_id \"chr21.g11334.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41430876', '41431079', '.', '+', '.', 'gene_id \"chr21.g11337\"; transcript_id \"chr21.g11337.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41431084', '41431533', '.', '+', '.', 'gene_id \"chr21.g11338\"; transcript_id \"chr21.g11338.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41482802', '41483023', '.', '+', '.', 'gene_id \"chr21.g11350\"; transcript_id \"chr21.g11350.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '21416388', '21416744', '.', '+', '.', 'gene_id \"chr4.g8446\"; transcript_id \"chr4.g8446.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41629514', '41629852', '.', '+', '.', 'gene_id \"chr21.g11361\"; transcript_id \"chr21.g11361.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41716141', '41716457', '.', '+', '.', 'gene_id \"chr21.g11363\"; transcript_id \"chr21.g11363.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '42032771', '42032804', '.', '+', '.', 'gene_id \"chr21.g11417\"; transcript_id \"chr21.g11417.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '21589826', '21590057', '.', '+', '.', 'gene_id \"chr4.g8548\"; transcript_id \"chr4.g8548.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22239149', '22239420', '.', '+', '.', 'gene_id \"chr4.g8705\"; transcript_id \"chr4.g8705.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22278299', '22278464', '.', '+', '.', 'gene_id \"chr4.g8715\"; transcript_id \"chr4.g8715.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22289702', '22289842', '.', '+', '.', 'gene_id \"chr4.g8719\"; transcript_id \"chr4.g8719.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '42422851', '42422900', '.', '-', '.', 'gene_id \"chr21.g11626\"; transcript_id \"chr21.g11626.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22472098', '22472130', '.', '-', '.', 'gene_id \"chr4.g8757\"; transcript_id \"chr4.g8757.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22671387', '22671450', '.', '-', '.', 'gene_id \"chr4.g8788\"; transcript_id \"chr4.g8788.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22701603', '22704058', '.', '+', '.', 'gene_id \"chr4.g8806\"; transcript_id \"chr4.g8806.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22704108', '22704454', '.', '+', '.', 'gene_id \"chr4.g8807\"; transcript_id \"chr4.g8807.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '42897096', '42897264', '.', '-', '.', 'gene_id \"chr21.g11786\"; transcript_id \"chr21.g11786.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '43562955', '43563148', '.', '-', '.', 'gene_id \"chr21.g11806\"; transcript_id \"chr21.g11806.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '43589861', '43590020', '.', '+', '.', 'gene_id \"chr21.g11811\"; transcript_id \"chr21.g11811.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '43590086', '43590246', '.', '+', '.', 'gene_id \"chr21.g11812\"; transcript_id \"chr21.g11812.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '43775358', '43775860', '.', '+', '.', 'gene_id \"chr21.g11825\"; transcript_id \"chr21.g11825.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '43775865', '43776371', '.', '+', '.', 'gene_id \"chr21.g11825\"; transcript_id \"chr21.g11825.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23005685', '23005911', '.', '+', '.', 'gene_id \"chr4.g9038\"; transcript_id \"chr4.g9038.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23005933', '23006190', '.', '+', '.', 'gene_id \"chr4.g9039\"; transcript_id \"chr4.g9039.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23104685', '23104756', '.', '-', '.', 'gene_id \"chr4.g9102\"; transcript_id \"chr4.g9102.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '44469914', '44469953', '.', '-', '.', 'gene_id \"chr21.g11996\"; transcript_id \"chr21.g11996.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23288918', '23289176', '.', '-', '.', 'gene_id \"chr4.g9275\"; transcript_id \"chr4.g9275.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23289219', '23289563', '.', '+', '.', 'gene_id \"chr4.g9278\"; transcript_id \"chr4.g9278.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '45172801', '45172850', '.', '+', '.', 'gene_id \"chr21.g12109\"; transcript_id \"chr21.g12109.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23456879', '23456932', '.', '+', '.', 'gene_id \"chr4.g9376\"; transcript_id \"chr4.g9376.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23569915', '23570131', '.', '+', '.', 'gene_id \"chr4.g9446\"; transcript_id \"chr4.g9446.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23698060', '23698225', '.', '+', '.', 'gene_id \"chr4.g9532\"; transcript_id \"chr4.g9532.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '47486007', '47486217', '.', '+', '.', 'gene_id \"chr21.g12354\"; transcript_id \"chr21.g12354.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23803178', '23803217', '.', '+', '.', 'gene_id \"chr4.g9594\"; transcript_id \"chr4.g9594.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24264572', '24264723', '.', '-', '.', 'gene_id \"chr4.g10037\"; transcript_id \"chr4.g10037.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24264728', '24264858', '.', '-', '.', 'gene_id \"chr4.g10038\"; transcript_id \"chr4.g10038.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '49396676', '49396754', '.', '-', '.', 'gene_id \"chr21.g12902\"; transcript_id \"chr21.g12902.i1\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '50717473', '50717678', '.', '+', '.', 'gene_id \"chr21.g13033\"; transcript_id \"chr21.g13033.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24628616', '24628681', '.', '-', '.', 'gene_id \"chr4.g10312\"; transcript_id \"chr4.g10312.i2\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33164040', '33164237', '100', '+', '.', 'gene_id \"chr21.g13126\"; transcript_id \"blat.proc28144.chain_19682\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10395242', '10395394', '100', '-', '.', 'gene_id \"chr21.g13132\"; transcript_id \"blat.proc28144.chain_9304\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25457863', '25457923', '100', '-', '.', 'gene_id \"chr21.g13179\"; transcript_id \"blat.proc18128.chain_20568\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39947549', '39947687', '100', '-', '.', 'gene_id \"chr21.g13197\"; transcript_id \"blat.proc28144.chain_18198\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39605106', '39605382', '100', '-', '.', 'gene_id \"chr21.g13208\"; transcript_id \"blat.proc28144.chain_17970\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31389084', '31389133', '100', '-', '.', 'gene_id \"chr21.g13216\"; transcript_id \"blat.proc28144.chain_20356\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12971391', '12971561', '100', '-', '.', 'gene_id \"chr21.g13246\"; transcript_id \"blat.proc28144.chain_8923\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10599388', '10599666', '100', '+', '.', 'gene_id \"chr21.g13267\"; transcript_id \"blat.proc4587.chain_2979\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39822191', '39822316', '100', '+', '.', 'gene_id \"chr21.g13284\"; transcript_id \"blat.proc28144.chain_15374\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4949335', '4949413', '100', '+', '.', 'gene_id \"chr21.g13323\"; transcript_id \"blat.proc28144.chain_6992\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5482944', '5482984', '100', '-', '.', 'gene_id \"chr21.g13347\"; transcript_id \"blat.proc18128.chain_473\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13464774', '13464952', '100', '+', '.', 'gene_id \"chr21.g13357\"; transcript_id \"blat.proc28144.chain_9455\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39859436', '39859653', '100', '-', '.', 'gene_id \"chr21.g13365\"; transcript_id \"blat.proc28144.chain_18673\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24591122', '24591399', '100', '+', '.', 'gene_id \"chr21.g13370\"; transcript_id \"blat.proc28144.chain_19454\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26306613', '26306867', '100', '-', '.', 'gene_id \"chr21.g13432\"; transcript_id \"blat.proc28144.chain_19676\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39996835', '39996999', '100', '-', '.', 'gene_id \"chr21.g13536\"; transcript_id \"blat.proc28144.chain_13843\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '27849867', '27849899', '100', '+', '.', 'gene_id \"chr21.g13546\"; transcript_id \"blat.proc18128.chain_26408\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36591276', '36591458', '100', '-', '.', 'gene_id \"chr21.g13557\"; transcript_id \"blat.proc28144.chain_18711\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37632198', '37632440', '100', '+', '.', 'gene_id \"chr21.g13562\"; transcript_id \"blat.proc28144.chain_15514\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15802282', '15802323', '100', '+', '.', 'gene_id \"chr21.g13597\"; transcript_id \"blat.proc18128.chain_4506\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14307208', '14307508', '100', '-', '.', 'gene_id \"chr21.g13610\"; transcript_id \"blat.proc28144.chain_588\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24562622', '24562948', '100', '-', '.', 'gene_id \"chr21.g13617\"; transcript_id \"blat.proc28144.chain_19105\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10633409', '10633571', '100', '+', '.', 'gene_id \"chr21.g13651\"; transcript_id \"blat.proc28144.chain_9399\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35527142', '35527296', '100', '+', '.', 'gene_id \"chr21.g13717\"; transcript_id \"blat.proc28144.chain_19365\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '640147', '640297', '100', '+', '.', 'gene_id \"chr21.g13720\"; transcript_id \"blat.proc28144.chain_10306\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2027569', '2027760', '100', '-', '.', 'gene_id \"chr21.g13723\"; transcript_id \"blat.proc28144.chain_11320\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24879633', '24879714', '100', '+', '.', 'gene_id \"chr21.g13740\"; transcript_id \"blat.proc4587.chain_1416\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25308263', '25308460', '100', '+', '.', 'gene_id \"chr21.g13765\"; transcript_id \"blat.proc28144.chain_20086\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10168511', '10168732', '100', '-', '.', 'gene_id \"chr21.g13768\"; transcript_id \"blat.proc28144.chain_2178\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29101161', '29101435', '100', '+', '.', 'gene_id \"chr21.g13794\"; transcript_id \"blat.proc28144.chain_20749\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17671759', '17671804', '100', '-', '.', 'gene_id \"chr21.g13800\"; transcript_id \"blat.proc4587.chain_2182\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37687390', '37687545', '100', '-', '.', 'gene_id \"chr21.g13815\"; transcript_id \"blat.proc4587.chain_6233\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '27828635', '27828736', '100', '+', '.', 'gene_id \"chr21.g13890\"; transcript_id \"blat.proc4005.chain_215\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26523164', '26523357', '100', '-', '.', 'gene_id \"chr21.g13895\"; transcript_id \"blat.proc28144.chain_19786\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39767996', '39768057', '100', '-', '.', 'gene_id \"chr21.g13904\"; transcript_id \"blat.proc28144.chain_15523\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '44202670', '44202712', '100', '+', '.', 'gene_id \"chr21.g13916\"; transcript_id \"blat.proc4587.chain_1250\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9996712', '9996750', '100', '+', '.', 'gene_id \"chr21.g13918\"; transcript_id \"blat.proc4587.chain_7980\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35730011', '35730092', '100', '+', '.', 'gene_id \"chr21.g13926\"; transcript_id \"blat.proc28144.chain_19963\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6059566', '6059866', '100', '-', '.', 'gene_id \"chr21.g13940\"; transcript_id \"blat.proc28144.chain_2513\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29931279', '29931520', '100', '-', '.', 'gene_id \"chr21.g13973\"; transcript_id \"blat.proc4587.chain_5450\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23200935', '23201149', '100', '-', '.', 'gene_id \"chr21.g13983\"; transcript_id \"blat.proc28144.chain_19445\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32631425', '32631810', '100', '-', '.', 'gene_id \"chr21.g13998\"; transcript_id \"blat.proc4587.chain_12097\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36678266', '36678573', '100', '-', '.', 'gene_id \"chr21.g14001\"; transcript_id \"blat.proc18128.chain_7118\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15035597', '15035771', '100', '+', '.', 'gene_id \"chr21.g14014\"; transcript_id \"blat.proc28144.chain_9790\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '49084009', '49084082', '100', '+', '.', 'gene_id \"chr21.g14057\"; transcript_id \"blat.proc18128.chain_18999\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10327220', '10327429', '100', '+', '.', 'gene_id \"chr21.g14069\"; transcript_id \"blat.proc4587.chain_7979\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40728251', '40728421', '100', '+', '.', 'gene_id \"chr21.g14114\"; transcript_id \"blat.proc4587.chain_4748\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41129249', '41129422', '100', '+', '.', 'gene_id \"chr21.g14123\"; transcript_id \"blat.proc28144.chain_17642\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26446569', '26446763', '100', '-', '.', 'gene_id \"chr21.g14131\"; transcript_id \"blat.proc28144.chain_19671\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '912742', '912987', '100', '-', '.', 'gene_id \"chr21.g14135\"; transcript_id \"blat.proc28144.chain_11422\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5042264', '5042348', '100', '+', '.', 'gene_id \"chr21.g14139\"; transcript_id \"blat.proc28144.chain_3215\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37027606', '37027766', '100', '+', '.', 'gene_id \"chr21.g14143\"; transcript_id \"blat.proc28144.chain_18578\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40946172', '40946345', '100', '+', '.', 'gene_id \"chr21.g14168\"; transcript_id \"blat.proc28144.chain_18213\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37368210', '37368497', '100', '-', '.', 'gene_id \"chr21.g14195\"; transcript_id \"blat.proc28144.chain_12360\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4763714', '4763895', '100', '-', '.', 'gene_id \"chr21.g14212\"; transcript_id \"blat.proc28144.chain_519\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24808956', '24809109', '100', '+', '.', 'gene_id \"chr21.g14224\"; transcript_id \"blat.proc28144.chain_19910\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25481705', '25482002', '100', '+', '.', 'gene_id \"chr21.g14233\"; transcript_id \"blat.proc28144.chain_21606\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15216964', '15217063', '100', '+', '.', 'gene_id \"chr21.g14273\"; transcript_id \"blat.proc28144.chain_2545\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37173076', '37173264', '100', '+', '.', 'gene_id \"chr21.g14284\"; transcript_id \"blat.proc28144.chain_18651\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11739992', '11740149', '100', '-', '.', 'gene_id \"chr21.g14345\"; transcript_id \"blat.proc28144.chain_9808\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35838103', '35838277', '100', '-', '.', 'gene_id \"chr21.g14374\"; transcript_id \"blat.proc28144.chain_18892\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '641214', '641675', '100', '+', '.', 'gene_id \"chr21.g14418\"; transcript_id \"blat.proc18128.chain_2905\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38834970', '38835032', '100', '+', '.', 'gene_id \"chr21.g14439\"; transcript_id \"blat.proc18128.chain_25267\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41013808', '41014000', '100', '+', '.', 'gene_id \"chr21.g14463\"; transcript_id \"blat.proc28144.chain_11748\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22228846', '22229067', '100', '+', '.', 'gene_id \"chr21.g14493\"; transcript_id \"blat.proc28144.chain_20991\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25357388', '25357554', '100', '-', '.', 'gene_id \"chr21.g14589\"; transcript_id \"blat.proc28144.chain_22914\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '3232235', '3232264', '100', '+', '.', 'gene_id \"chr21.g14610\"; transcript_id \"blat.proc18128.chain_26595\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13189539', '13189695', '100', '-', '.', 'gene_id \"chr21.g14613\"; transcript_id \"blat.proc18128.chain_17616\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13473186', '13474767', '100', '+', '.', 'gene_id \"chr21.g14640\"; transcript_id \"blat.proc28144.chain_5804\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23759044', '23759329', '100', '+', '.', 'gene_id \"chr21.g14704\"; transcript_id \"blat.proc28144.chain_19283\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10598932', '10599137', '100', '-', '.', 'gene_id \"chr21.g14748\"; transcript_id \"blat.proc4587.chain_2978\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38630638', '38630839', '100', '+', '.', 'gene_id \"chr21.g14769\"; transcript_id \"blat.proc28144.chain_12176\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5298087', '5298188', '100', '+', '.', 'gene_id \"chr21.g14796\"; transcript_id \"blat.proc28144.chain_4935\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15446293', '15446332', '100', '-', '.', 'gene_id \"chr21.g14832\"; transcript_id \"blat.proc4587.chain_13693\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15627438', '15627931', '100', '+', '.', 'gene_id \"chr21.g14840\"; transcript_id \"blat.proc28144.chain_9201\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39851981', '39852281', '100', '+', '.', 'gene_id \"chr21.g14879\"; transcript_id \"blat.proc18128.chain_25860\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7449915', '7450129', '100', '+', '.', 'gene_id \"chr21.g14881\"; transcript_id \"blat.proc28144.chain_628\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33361730', '33361885', '100', '-', '.', 'gene_id \"chr21.g14994\"; transcript_id \"blat.proc28144.chain_21700\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37153764', '37153955', '100', '+', '.', 'gene_id \"chr21.g15000\"; transcript_id \"blat.proc4587.chain_6843\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41055065', '41055318', '100', '+', '.', 'gene_id \"chr21.g15013\"; transcript_id \"blat.proc28144.chain_12106\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37192741', '37192819', '100', '-', '.', 'gene_id \"chr21.g15025\"; transcript_id \"blat.proc28144.chain_18093\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35915260', '35915412', '100', '+', '.', 'gene_id \"chr21.g15060\"; transcript_id \"blat.proc18128.chain_6855\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '48008036', '48008262', '100', '+', '.', 'gene_id \"chr21.g15070\"; transcript_id \"blat.proc18128.chain_4103\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7177589', '7177774', '100', '+', '.', 'gene_id \"chr21.g15101\"; transcript_id \"blat.proc28144.chain_9746\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25198806', '25198976', '100', '+', '.', 'gene_id \"chr21.g15112\"; transcript_id \"blat.proc4587.chain_4519\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29677244', '29677708', '100', '-', '.', 'gene_id \"chr21.g15138\"; transcript_id \"blat.proc28144.chain_19098\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4803158', '4803361', '100', '-', '.', 'gene_id \"chr21.g15151\"; transcript_id \"blat.proc28144.chain_1334\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5188753', '5188823', '100', '-', '.', 'gene_id \"chr21.g15158\"; transcript_id \"blat.proc28144.chain_9571\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17561527', '17561739', '100', '-', '.', 'gene_id \"chr21.g15169\"; transcript_id \"blat.proc28144.chain_11550\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23772119', '23772282', '100', '-', '.', 'gene_id \"chr21.g15185\"; transcript_id \"blat.proc28144.chain_19552\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '8215808', '8215876', '100', '+', '.', 'gene_id \"chr21.g15226\"; transcript_id \"blat.proc28144.chain_9247\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10649799', '10649984', '100', '+', '.', 'gene_id \"chr21.g15236\"; transcript_id \"blat.proc28144.chain_9199\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13429391', '13429523', '100', '+', '.', 'gene_id \"chr21.g15250\"; transcript_id \"blat.proc28144.chain_5008\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '8455516', '8455610', '100', '-', '.', 'gene_id \"chr21.g15251\"; transcript_id \"blat.proc28144.chain_11332\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33172462', '33172571', '100', '-', '.', 'gene_id \"chr21.g15284\"; transcript_id \"blat.proc18128.chain_15319\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10624388', '10624505', '100', '-', '.', 'gene_id \"chr21.g15303\"; transcript_id \"blat.proc28144.chain_7599\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13669960', '13670214', '100', '+', '.', 'gene_id \"chr21.g15356\"; transcript_id \"blat.proc28144.chain_4421\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35386781', '35387083', '100', '-', '.', 'gene_id \"chr21.g15412\"; transcript_id \"blat.proc28144.chain_19312\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37184775', '37184883', '100', '-', '.', 'gene_id \"chr21.g15414\"; transcript_id \"blat.proc28144.chain_13457\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33903804', '33903849', '100', '-', '.', 'gene_id \"chr21.g15463\"; transcript_id \"blat.proc28144.chain_6801\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13275555', '13275711', '100', '+', '.', 'gene_id \"chr21.g15526\"; transcript_id \"blat.proc28144.chain_7954\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5854747', '5855020', '100', '+', '.', 'gene_id \"chr21.g15546\"; transcript_id \"blat.proc28144.chain_2433\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7457517', '7457683', '100', '+', '.', 'gene_id \"chr21.g15585\"; transcript_id \"blat.proc28144.chain_164\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5028375', '5028618', '100', '-', '.', 'gene_id \"chr21.g15590\"; transcript_id \"blat.proc28144.chain_2427\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41056464', '41056792', '100', '+', '.', 'gene_id \"chr21.g15609\"; transcript_id \"blat.proc28144.chain_12525\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39925625', '39925786', '100', '+', '.', 'gene_id \"chr21.g15611\"; transcript_id \"blat.proc28144.chain_18177\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25046978', '25047018', '100', '-', '.', 'gene_id \"chr21.g15613\"; transcript_id \"blat.proc18128.chain_8326\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4812487', '4812700', '100', '+', '.', 'gene_id \"chr21.g15656\"; transcript_id \"blat.proc4587.chain_3660\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6660858', '6661034', '100', '+', '.', 'gene_id \"chr21.g15697\"; transcript_id \"blat.proc28144.chain_8944\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11433062', '11433245', '100', '+', '.', 'gene_id \"chr21.g15702\"; transcript_id \"blat.proc28144.chain_9928\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13492256', '13492480', '100', '+', '.', 'gene_id \"chr21.g15717\"; transcript_id \"blat.proc28144.chain_8763\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9472680', '9472738', '100', '-', '.', 'gene_id \"chr21.g15725\"; transcript_id \"blat.proc18128.chain_14445\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41158432', '41158673', '100', '+', '.', 'gene_id \"chr21.g15733\"; transcript_id \"blat.proc28144.chain_17097\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13495326', '13495488', '100', '-', '.', 'gene_id \"chr21.g15810\"; transcript_id \"blat.proc28144.chain_885\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12773378', '12773563', '100', '-', '.', 'gene_id \"chr21.g15817\"; transcript_id \"blat.proc28144.chain_9816\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41159223', '41159435', '100', '-', '.', 'gene_id \"chr21.g15827\"; transcript_id \"blat.proc28144.chain_12826\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25357575', '25357856', '100', '+', '.', 'gene_id \"chr21.g15878\"; transcript_id \"blat.proc4587.chain_2004\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31390732', '31390897', '100', '-', '.', 'gene_id \"chr21.g15935\"; transcript_id \"blat.proc28144.chain_19245\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '764746', '764949', '100', '+', '.', 'gene_id \"chr21.g16013\"; transcript_id \"blat.proc28144.chain_10374\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7464012', '7464114', '100', '-', '.', 'gene_id \"chr21.g16061\"; transcript_id \"blat.proc28144.chain_4676\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37232328', '37232588', '100', '+', '.', 'gene_id \"chr21.g16089\"; transcript_id \"blat.proc28144.chain_18436\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15928544', '15928914', '100', '+', '.', 'gene_id \"chr21.g16098\"; transcript_id \"blat.proc4587.chain_11020\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40050130', '40050292', '100', '+', '.', 'gene_id \"chr21.g16111\"; transcript_id \"blat.proc28144.chain_11950\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25981499', '25981668', '100', '-', '.', 'gene_id \"chr21.g16149\"; transcript_id \"blat.proc28144.chain_20616\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10113337', '10113537', '100', '-', '.', 'gene_id \"chr21.g16186\"; transcript_id \"blat.proc28144.chain_9405\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23246662', '23246863', '100', '-', '.', 'gene_id \"chr21.g16239\"; transcript_id \"blat.proc28144.chain_21058\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7283877', '7284054', '100', '-', '.', 'gene_id \"chr21.g16265\"; transcript_id \"blat.proc28144.chain_9927\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6200158', '6200342', '100', '+', '.', 'gene_id \"chr21.g16282\"; transcript_id \"blat.proc28144.chain_4976\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25884477', '25884932', '100', '+', '.', 'gene_id \"chr21.g16344\"; transcript_id \"blat.proc28144.chain_20737\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26199560', '26199981', '100', '-', '.', 'gene_id \"chr21.g16384\"; transcript_id \"blat.proc28144.chain_23018\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34550845', '34551078', '100', '-', '.', 'gene_id \"chr21.g16394\"; transcript_id \"blat.proc4587.chain_6200\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41165617', '41165838', '100', '+', '.', 'gene_id \"chr21.g16404\"; transcript_id \"blat.proc28144.chain_13154\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '27775418', '27775457', '100', '-', '.', 'gene_id \"chr21.g16429\"; transcript_id \"blat.proc28144.chain_16623\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '21076163', '21076195', '100', '-', '.', 'gene_id \"chr21.g16557\"; transcript_id \"blat.proc4005.chain_16757\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37328347', '37328519', '100', '+', '.', 'gene_id \"chr21.g16615\"; transcript_id \"blat.proc28144.chain_17013\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7479758', '7479991', '100', '-', '.', 'gene_id \"chr21.g16624\"; transcript_id \"blat.proc28144.chain_3547\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39718845', '39719371', '100', '-', '.', 'gene_id \"chr21.g16632\"; transcript_id \"blat.proc28144.chain_13712\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7038634', '7038676', '100', '-', '.', 'gene_id \"chr21.g16633\"; transcript_id \"blat.proc18128.chain_4495\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38812990', '38813041', '100', '+', '.', 'gene_id \"chr21.g16651\"; transcript_id \"blat.proc28144.chain_18431\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1775016', '1775140', '100', '-', '.', 'gene_id \"chr21.g16658\"; transcript_id \"blat.proc28144.chain_11423\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '14248421', '14248742', '100', '-', '.', 'gene_id \"chr21.g16695\"; transcript_id \"blat.proc28144.chain_9792\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '47067701', '47067735', '100', '+', '.', 'gene_id \"chr21.g16716\"; transcript_id \"blat.proc4587.chain_10392\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24451034', '24451206', '100', '+', '.', 'gene_id \"chr21.g16766\"; transcript_id \"blat.proc28144.chain_19936\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24878538', '24878720', '100', '-', '.', 'gene_id \"chr21.g16786\"; transcript_id \"blat.proc4587.chain_1429\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32276223', '32276530', '100', '-', '.', 'gene_id \"chr21.g16814\"; transcript_id \"blat.proc28144.chain_19709\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr21', 'PASA', 'exon', '25622357', '25622572', '100', '+', '.', 'gene_id \"chr21.g16853\"; transcript_id \"blat.proc18128.chain_20997\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4763920', '4764158', '100', '+', '.', 'gene_id \"chr21.g16880\"; transcript_id \"blat.proc28144.chain_520\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32954142', '32954174', '100', '-', '.', 'gene_id \"chr21.g16892\"; transcript_id \"blat.proc18128.chain_25090\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5127591', '5127750', '100', '-', '.', 'gene_id \"chr21.g16911\"; transcript_id \"blat.proc28144.chain_9687\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6057164', '6057464', '100', '-', '.', 'gene_id \"chr21.g16926\"; transcript_id \"blat.proc28144.chain_1621\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7679500', '7679755', '100', '-', '.', 'gene_id \"chr21.g16968\"; transcript_id \"blat.proc28144.chain_7266\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41403247', '41403415', '100', '+', '.', 'gene_id \"chr21.g17007\"; transcript_id \"blat.proc28144.chain_13033\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '412904', '413177', '100', '+', '.', 'gene_id \"chr21.g17008\"; transcript_id \"blat.proc28144.chain_11426\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39928488', '39928711', '100', '+', '.', 'gene_id \"chr21.g17015\"; transcript_id \"blat.proc28144.chain_17211\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26416370', '26416468', '100', '-', '.', 'gene_id \"chr21.g17016\"; transcript_id \"blat.proc4587.chain_2630\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '3777262', '3777432', '100', '+', '.', 'gene_id \"chr21.g17044\"; transcript_id \"blat.proc28144.chain_10031\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16218713', '16218904', '100', '-', '.', 'gene_id \"chr21.g17046\"; transcript_id \"blat.proc28144.chain_5073\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25589288', '25589471', '100', '+', '.', 'gene_id \"chr21.g17075\"; transcript_id \"blat.proc28144.chain_22613\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31777732', '31777956', '100', '+', '.', 'gene_id \"chr21.g17091\"; transcript_id \"blat.proc28144.chain_19126\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13484282', '13484469', '100', '+', '.', 'gene_id \"chr21.g17097\"; transcript_id \"blat.proc28144.chain_1426\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '34142472', '34142679', '100', '-', '.', 'gene_id \"chr21.g17112\"; transcript_id \"blat.proc28144.chain_20481\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25078175', '25078357', '100', '-', '.', 'gene_id \"chr21.g17120\"; transcript_id \"blat.proc4587.chain_11375\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23543045', '23543603', '100', '-', '.', 'gene_id \"chr21.g17128\"; transcript_id \"blat.proc4587.chain_3815\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25567402', '25568155', '100', '-', '.', 'gene_id \"chr21.g17163\"; transcript_id \"blat.proc4587.chain_1663\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12516260', '12516538', '100', '-', '.', 'gene_id \"chr21.g17228\"; transcript_id \"blat.proc28144.chain_34\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10361965', '10362006', '100', '+', '.', 'gene_id \"chr21.g17294\"; transcript_id \"blat.proc18128.chain_6408\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15903900', '15904111', '100', '+', '.', 'gene_id \"chr21.g17316\"; transcript_id \"blat.proc4587.chain_2188\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '721260', '721525', '100', '+', '.', 'gene_id \"chr21.g17317\"; transcript_id \"blat.proc28144.chain_10164\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13924369', '13924428', '100', '+', '.', 'gene_id \"chr21.g17361\"; transcript_id \"blat.proc28144.chain_9166\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15610073', '15610240', '100', '+', '.', 'gene_id \"chr21.g17384\"; transcript_id \"blat.proc28144.chain_4073\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35838379', '35838606', '100', '+', '.', 'gene_id \"chr21.g17388\"; transcript_id \"blat.proc28144.chain_18891\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23495066', '23495237', '100', '+', '.', 'gene_id \"chr21.g17419\"; transcript_id \"blat.proc28144.chain_22321\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37794011', '37794186', '100', '+', '.', 'gene_id \"chr21.g17436\"; transcript_id \"blat.proc28144.chain_18305\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '297328', '297409', '100', '-', '.', 'gene_id \"chr21.g17439\"; transcript_id \"blat.proc4587.chain_9017\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12447732', '12447903', '100', '-', '.', 'gene_id \"chr21.g17442\"; transcript_id \"blat.proc28144.chain_154\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25588946', '25589184', '100', '-', '.', 'gene_id \"chr21.g17450\"; transcript_id \"blat.proc28144.chain_22612\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31882576', '31882728', '100', '+', '.', 'gene_id \"chr21.g17463\"; transcript_id \"blat.proc28144.chain_18841\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10683031', '10683283', '100', '-', '.', 'gene_id \"chr21.g17468\"; transcript_id \"blat.proc28144.chain_9254\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5302301', '5302539', '100', '+', '.', 'gene_id \"chr21.g17540\"; transcript_id \"blat.proc28144.chain_8844\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4813611', '4814124', '100', '+', '.', 'gene_id \"chr21.g17561\"; transcript_id \"blat.proc28144.chain_1833\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25322680', '25322888', '100', '-', '.', 'gene_id \"chr21.g17563\"; transcript_id \"blat.proc28144.chain_20038\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23195334', '23195730', '100', '+', '.', 'gene_id \"chr21.g17581\"; transcript_id \"blat.proc4587.chain_1332\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24596434', '24596616', '100', '+', '.', 'gene_id \"chr21.g17590\"; transcript_id \"blat.proc28144.chain_20557\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26439830', '26440032', '100', '-', '.', 'gene_id \"chr21.g17592\"; transcript_id \"blat.proc4587.chain_6853\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33386932', '33387087', '100', '-', '.', 'gene_id \"chr21.g17608\"; transcript_id \"blat.proc28144.chain_20768\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '22249429', '22249610', '100', '+', '.', 'gene_id \"chr21.g17615\"; transcript_id \"blat.proc4587.chain_5152\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25143474', '25143632', '100', '+', '.', 'gene_id \"chr21.g17640\"; transcript_id \"blat.proc28144.chain_19094\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15213684', '15213931', '100', '-', '.', 'gene_id \"chr21.g17667\"; transcript_id \"blat.proc28144.chain_183\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11652247', '11652413', '100', '+', '.', 'gene_id \"chr21.g17668\"; transcript_id \"blat.proc28144.chain_3934\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '374359', '374492', '100', '+', '.', 'gene_id \"chr21.g17678\"; transcript_id \"blat.proc18128.chain_25562\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6934870', '6935022', '100', '+', '.', 'gene_id \"chr21.g17700\"; transcript_id \"blat.proc28144.chain_9511\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10701500', '10701652', '100', '+', '.', 'gene_id \"chr21.g17708\"; transcript_id \"blat.proc28144.chain_40\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9561060', '9561118', '100', '-', '.', 'gene_id \"chr21.g17780\"; transcript_id \"blat.proc4587.chain_8753\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31807618', '31807970', '100', '+', '.', 'gene_id \"chr21.g17810\"; transcript_id \"blat.proc4587.chain_4506\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40453221', '40453512', '100', '-', '.', 'gene_id \"chr21.g17816\"; transcript_id \"blat.proc18128.chain_6417\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38263429', '38263605', '100', '-', '.', 'gene_id \"chr21.g17829\"; transcript_id \"blat.proc4587.chain_3115\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10713300', '10713542', '100', '+', '.', 'gene_id \"chr21.g17893\"; transcript_id \"blat.proc4587.chain_7902\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5057679', '5057792', '100', '-', '.', 'gene_id \"chr21.g17912\"; transcript_id \"blat.proc28144.chain_6625\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4961801', '4961856', '100', '-', '.', 'gene_id \"chr21.g17918\"; transcript_id \"blat.proc28144.chain_3144\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '44003285', '44003324', '100', '+', '.', 'gene_id \"chr21.g17983\"; transcript_id \"blat.proc18128.chain_16493\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12653591', '12653843', '100', '-', '.', 'gene_id \"chr21.g18040\"; transcript_id \"blat.proc28144.chain_492\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13449947', '13450177', '100', '-', '.', 'gene_id \"chr21.g18041\"; transcript_id \"blat.proc28144.chain_79\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39980302', '39980887', '100', '-', '.', 'gene_id \"chr21.g18069\"; transcript_id \"blat.proc28144.chain_14558\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '31361374', '31361648', '100', '-', '.', 'gene_id \"chr21.g18094\"; transcript_id \"blat.proc28144.chain_19415\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2529972', '2530026', '100', '-', '.', 'gene_id \"chr21.g18095\"; transcript_id \"blat.proc18128.chain_10287\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36583223', '36583339', '100', '-', '.', 'gene_id \"chr21.g18182\"; transcript_id \"blat.proc28144.chain_14031\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10631923', '10632172', '100', '+', '.', 'gene_id \"chr21.g18205\"; transcript_id \"blat.proc18128.chain_11133\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5078529', '5079226', '100', '-', '.', 'gene_id \"chr21.g18208\"; transcript_id \"blat.proc28144.chain_7256\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23771910', '23772075', '100', '-', '.', 'gene_id \"chr21.g18213\"; transcript_id \"blat.proc28144.chain_19551\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35161018', '35161490', '100', '-', '.', 'gene_id \"chr21.g18221\"; transcript_id \"blat.proc28144.chain_22284\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5845693', '5845880', '100', '+', '.', 'gene_id \"chr21.g18224\"; transcript_id \"blat.proc28144.chain_210\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15411316', '15411514', '100', '+', '.', 'gene_id \"chr21.g18287\"; transcript_id \"blat.proc28144.chain_9600\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '8126117', '8126151', '100', '-', '.', 'gene_id \"chr21.g18304\"; transcript_id \"blat.proc18128.chain_8188\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '709561', '709925', '100', '-', '.', 'gene_id \"chr21.g18307\"; transcript_id \"blat.proc28144.chain_11147\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12515830', '12516252', '100', '+', '.', 'gene_id \"chr21.g18310\"; transcript_id \"blat.proc28144.chain_33\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25446041', '25446246', '100', '+', '.', 'gene_id \"chr21.g18314\"; transcript_id \"blat.proc28144.chain_18886\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25377289', '25377446', '100', '-', '.', 'gene_id \"chr21.g18331\"; transcript_id \"blat.proc28144.chain_19763\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15603205', '15603486', '100', '+', '.', 'gene_id \"chr21.g18363\"; transcript_id \"blat.proc28144.chain_3958\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39266111', '39266264', '100', '+', '.', 'gene_id \"chr21.g18365\"; transcript_id \"blat.proc28144.chain_18545\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5852095', '5852322', '100', '-', '.', 'gene_id \"chr21.g18388\"; transcript_id \"blat.proc28144.chain_9456\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25234396', '25234870', '100', '+', '.', 'gene_id \"chr21.g18393\"; transcript_id \"blat.proc28144.chain_19145\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24878769', '24879047', '100', '-', '.', 'gene_id \"chr21.g18420\"; transcript_id \"blat.proc4587.chain_1424\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39731296', '39731387', '100', '-', '.', 'gene_id \"chr21.g18432\"; transcript_id \"blat.proc28144.chain_17414\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '9084197', '9084245', '100', '-', '.', 'gene_id \"chr21.g18445\"; transcript_id \"blat.proc18128.chain_23312\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '846447', '846735', '100', '+', '.', 'gene_id \"chr21.g18460\"; transcript_id \"blat.proc28144.chain_10283\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5815837', '5816199', '100', '-', '.', 'gene_id \"chr21.g18471\"; transcript_id \"blat.proc4587.chain_207\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4955058', '4955214', '100', '-', '.', 'gene_id \"chr21.g18527\"; transcript_id \"blat.proc28144.chain_8803\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39820957', '39821111', '100', '-', '.', 'gene_id \"chr21.g18552\"; transcript_id \"blat.proc28144.chain_15978\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33019249', '33019410', '100', '+', '.', 'gene_id \"chr21.g18554\"; transcript_id \"blat.proc28144.chain_19478\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32940124', '32940307', '100', '+', '.', 'gene_id \"chr21.g18562\"; transcript_id \"blat.proc28144.chain_18771\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16115782', '16115957', '100', '+', '.', 'gene_id \"chr21.g18566\"; transcript_id \"blat.proc28144.chain_531\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12478530', '12478683', '100', '+', '.', 'gene_id \"chr21.g18568\"; transcript_id \"blat.proc28144.chain_7901\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12539573', '12539901', '100', '+', '.', 'gene_id \"chr21.g18607\"; transcript_id \"blat.proc4587.chain_11842\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12560841', '12561052', '100', '-', '.', 'gene_id \"chr21.g18608\"; transcript_id \"blat.proc28144.chain_8802\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36571552', '36571745', '100', '-', '.', 'gene_id \"chr21.g18639\"; transcript_id \"blat.proc28144.chain_12372\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4844318', '4844647', '100', '+', '.', 'gene_id \"chr21.g18654\"; transcript_id \"blat.proc18128.chain_15617\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30920586', '30920742', '100', '+', '.', 'gene_id \"chr21.g18671\"; transcript_id \"blat.proc28144.chain_21565\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15247067', '15247103', '100', '-', '.', 'gene_id \"chr21.g18752\"; transcript_id \"blat.proc18128.chain_23029\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40445345', '40445536', '100', '+', '.', 'gene_id \"chr21.g18837\"; transcript_id \"blat.proc28144.chain_11933\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26670618', '26670908', '100', '-', '.', 'gene_id \"chr21.g18851\"; transcript_id \"blat.proc28144.chain_20377\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26522987', '26523154', '100', '+', '.', 'gene_id \"chr21.g18861\"; transcript_id \"blat.proc28144.chain_19787\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41000291', '41000466', '100', '-', '.', 'gene_id \"chr21.g18869\"; transcript_id \"blat.proc28144.chain_18414\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40896071', '40896212', '100', '+', '.', 'gene_id \"chr21.g18875\"; transcript_id \"blat.proc28144.chain_12693\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39030761', '39030914', '100', '-', '.', 'gene_id \"chr21.g18921\"; transcript_id \"blat.proc28144.chain_12174\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24643876', '24644080', '100', '-', '.', 'gene_id \"chr21.g18944\"; transcript_id \"blat.proc28144.chain_19587\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '612350', '612615', '100', '+', '.', 'gene_id \"chr21.g18954\"; transcript_id \"blat.proc28144.chain_9978\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39762267', '39762464', '100', '-', '.', 'gene_id \"chr21.g18962\"; transcript_id \"blat.proc28144.chain_18727\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32334457', '32334763', '100', '-', '.', 'gene_id \"chr21.g18972\"; transcript_id \"blat.proc4587.chain_2017\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40923286', '40923475', '100', '+', '.', 'gene_id \"chr21.g18986\"; transcript_id \"blat.proc28144.chain_18455\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40714171', '40714305', '100', '-', '.', 'gene_id \"chr21.g18989\"; transcript_id \"blat.proc28144.chain_18432\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25494510', '25494791', '100', '-', '.', 'gene_id \"chr21.g19033\"; transcript_id \"blat.proc28144.chain_19146\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '47125402', '47125501', '100', '+', '.', 'gene_id \"chr21.g19054\"; transcript_id \"blat.proc18128.chain_15826\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24879385', '24879454', '100', '-', '.', 'gene_id \"chr21.g19087\"; transcript_id \"blat.proc4587.chain_1427\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2498549', '2498680', '100', '-', '.', 'gene_id \"chr21.g19106\"; transcript_id \"blat.proc28144.chain_10640\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16078391', '16078569', '100', '+', '.', 'gene_id \"chr21.g19129\"; transcript_id \"blat.proc28144.chain_9569\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25305888', '25305951', '100', '+', '.', 'gene_id \"chr21.g19138\"; transcript_id \"blat.proc4005.chain_10719\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13033136', '13033306', '100', '-', '.', 'gene_id \"chr21.g19142\"; transcript_id \"blat.proc28144.chain_4975\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39801082', '39801264', '100', '+', '.', 'gene_id \"chr21.g19144\"; transcript_id \"blat.proc28144.chain_16576\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4917629', '4917864', '100', '+', '.', 'gene_id \"chr21.g19210\"; transcript_id \"blat.proc28144.chain_7592\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15856211', '15856465', '100', '-', '.', 'gene_id \"chr21.g19241\"; transcript_id \"blat.proc28144.chain_1311\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13485360', '13485475', '100', '-', '.', 'gene_id \"chr21.g19250\"; transcript_id \"blat.proc28144.chain_2018\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10712967', '10713223', '100', '-', '.', 'gene_id \"chr21.g19253\"; transcript_id \"blat.proc4587.chain_7901\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36045651', '36045874', '100', '+', '.', 'gene_id \"chr21.g19261\"; transcript_id \"blat.proc28144.chain_20011\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10716161', '10716216', '100', '+', '.', 'gene_id \"chr21.g19281\"; transcript_id \"blat.proc28144.chain_8817\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11540533', '11540699', '100', '+', '.', 'gene_id \"chr21.g19307\"; transcript_id \"blat.proc28144.chain_9651\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30111916', '30112085', '100', '+', '.', 'gene_id \"chr21.g19311\"; transcript_id \"blat.proc28144.chain_19873\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10112286', '10112334', '100', '+', '.', 'gene_id \"chr21.g19327\"; transcript_id \"blat.proc28144.chain_2744\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23195105', '23195322', '100', '+', '.', 'gene_id \"chr21.g19356\"; transcript_id \"blat.proc4587.chain_1330\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13472754', '13473137', '100', '+', '.', 'gene_id \"chr21.g19373\"; transcript_id \"blat.proc4587.chain_5941\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13450180', '13450568', '100', '+', '.', 'gene_id \"chr21.g19383\"; transcript_id \"blat.proc28144.chain_80\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '16316113', '16316282', '100', '+', '.', 'gene_id \"chr21.g19384\"; transcript_id \"blat.proc28144.chain_8919\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '24381572', '24382477', '100', '+', '.', 'gene_id \"chr21.g19401\"; transcript_id \"blat.proc28144.chain_20840\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12896951', '12897164', '100', '+', '.', 'gene_id \"chr21.g19404\"; transcript_id \"blat.proc4587.chain_4444\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5847484', '5847688', '100', '+', '.', 'gene_id \"chr21.g19564\"; transcript_id \"blat.proc28144.chain_3766\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36442463', '36442767', '100', '-', '.', 'gene_id \"chr21.g19601\"; transcript_id \"blat.proc4587.chain_11821\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '40862799', '40863129', '100', '-', '.', 'gene_id \"chr21.g19626\"; transcript_id \"blat.proc28144.chain_11996\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33164538', '33164808', '100', '-', '.', 'gene_id \"chr21.g19662\"; transcript_id \"blat.proc28144.chain_19677\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41786789', '41786962', '100', '+', '.', 'gene_id \"chr21.g19676\"; transcript_id \"blat.proc4587.chain_2688\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '38158854', '38158929', '100', '-', '.', 'gene_id \"chr21.g19694\"; transcript_id \"blat.proc28144.chain_14600\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41842116', '41842230', '100', '+', '.', 'gene_id \"chr21.g19695\"; transcript_id \"blat.proc28144.chain_18115\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28083366', '28083592', '100', '+', '.', 'gene_id \"chr21.g19720\"; transcript_id \"blat.proc18128.chain_21899\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25887307', '25887547', '100', '+', '.', 'gene_id \"chr21.g19749\"; transcript_id \"blat.proc28144.chain_19456\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39302096', '39302639', '100', '-', '.', 'gene_id \"chr21.g19777\"; transcript_id \"blat.proc28144.chain_15381\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11671318', '11671475', '100', '+', '.', 'gene_id \"chr21.g19780\"; transcript_id \"blat.proc28144.chain_1130\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29575253', '29575286', '100', '+', '.', 'gene_id \"chr21.g19786\"; transcript_id \"blat.proc18128.chain_573\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13725723', '13725914', '100', '+', '.', 'gene_id \"chr21.g19826\"; transcript_id \"blat.proc28144.chain_7797\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15894195', '15894346', '100', '+', '.', 'gene_id \"chr21.g19844\"; transcript_id \"blat.proc28144.chain_9133\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '15928985', '15929138', '100', '-', '.', 'gene_id \"chr21.g19917\"; transcript_id \"blat.proc28144.chain_4924\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35101940', '35102130', '100', '-', '.', 'gene_id \"chr21.g19950\"; transcript_id \"blat.proc4587.chain_873\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13413996', '13414127', '100', '-', '.', 'gene_id \"chr21.g19997\"; transcript_id \"blat.proc28144.chain_1215\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39383285', '39383573', '100', '+', '.', 'gene_id \"chr21.g20015\"; transcript_id \"blat.proc28144.chain_14956\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '41786963', '41787231', '100', '+', '.', 'gene_id \"chr21.g20112\"; transcript_id \"blat.proc4587.chain_2687\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12396773', '12396973', '100', '-', '.', 'gene_id \"chr21.g20142\"; transcript_id \"blat.proc28144.chain_19\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '17561091', '17561370', '100', '+', '.', 'gene_id \"chr21.g20159\"; transcript_id \"blat.proc28144.chain_11551\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6129705', '6129833', '100', '+', '.', 'gene_id \"chr21.g20179\"; transcript_id \"blat.proc28144.chain_9226\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1452484', '1452543', '100', '+', '.', 'gene_id \"chr21.g20212\"; transcript_id \"blat.proc18128.chain_6903\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '33808518', '33808724', '100', '+', '.', 'gene_id \"chr21.g20264\"; transcript_id \"blat.proc28144.chain_19931\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13484472', '13484835', '100', '+', '.', 'gene_id \"chr21.g20266\"; transcript_id \"blat.proc28144.chain_1425\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25544392', '25544568', '100', '+', '.', 'gene_id \"chr21.g20304\"; transcript_id \"blat.proc28144.chain_21517\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12935881', '12936055', '100', '+', '.', 'gene_id \"chr21.g20330\"; transcript_id \"blat.proc28144.chain_9880\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30785411', '30785440', '100', '-', '.', 'gene_id \"chr21.g20360\"; transcript_id \"blat.proc18128.chain_7330\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32024757', '32025004', '100', '+', '.', 'gene_id \"chr21.g20437\"; transcript_id \"blat.proc28144.chain_19313\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '42826625', '42826897', '100', '+', '.', 'gene_id \"chr21.g20461\"; transcript_id \"blat.proc28144.chain_11607\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '32631210', '32631368', '100', '+', '.', 'gene_id \"chr21.g20513\"; transcript_id \"blat.proc4587.chain_12098\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13475413', '13475597', '100', '-', '.', 'gene_id \"chr21.g20517\"; transcript_id \"blat.proc28144.chain_3097\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6060751', '6060810', '100', '-', '.', 'gene_id \"chr21.g20597\"; transcript_id \"blat.proc18128.chain_19087\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '4844068', '4844189', '100', '+', '.', 'gene_id \"chr21.g20611\"; transcript_id \"blat.proc18128.chain_15597\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13119716', '13120375', '100', '+', '.', 'gene_id \"chr21.g20682\"; transcript_id \"blat.proc28144.chain_468\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '2463592', '2463671', '100', '+', '.', 'gene_id \"chr21.g20734\"; transcript_id \"blat.proc4587.chain_12054\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35391850', '35392008', '100', '+', '.', 'gene_id \"chr21.g20757\"; transcript_id \"blat.proc28144.chain_20933\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '6655787', '6656040', '100', '+', '.', 'gene_id \"chr21.g20761\"; transcript_id \"blat.proc4587.chain_9309\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25892384', '25892543', '100', '+', '.', 'gene_id \"chr21.g20768\"; transcript_id \"blat.proc28144.chain_20838\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35326871', '35327109', '100', '+', '.', 'gene_id \"chr21.g20773\"; transcript_id \"blat.proc28144.chain_18950\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7928147', '7928301', '100', '-', '.', 'gene_id \"chr21.g20796\"; transcript_id \"blat.proc18128.chain_17183\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12652260', '12652629', '100', '-', '.', 'gene_id \"chr21.g20864\"; transcript_id \"blat.proc4587.chain_10117\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5815539', '5815696', '100', '+', '.', 'gene_id \"chr21.g20874\"; transcript_id \"blat.proc28144.chain_622\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '5374915', '5375097', '100', '+', '.', 'gene_id \"chr21.g20936\"; transcript_id \"blat.proc28144.chain_9015\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25622069', '25622310', '100', '+', '.', 'gene_id \"chr21.g20942\"; transcript_id \"blat.proc18128.chain_20992\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25482159', '25482564', '100', '-', '.', 'gene_id \"chr21.g20950\"; transcript_id \"blat.proc4587.chain_9420\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '12912671', '12912772', '100', '+', '.', 'gene_id \"chr21.g20963\"; transcript_id \"blat.proc28144.chain_9482\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '10166082', '10166248', '100', '+', '.', 'gene_id \"chr21.g20982\"; transcript_id \"blat.proc18128.chain_23378\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '26480662', '26480959', '100', '+', '.', 'gene_id \"chr21.g21024\"; transcript_id \"blat.proc18128.chain_23388\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '23104932', '23105119', '100', '+', '.', 'gene_id \"chr21.g21026\"; transcript_id \"blat.proc28144.chain_19370\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '39563882', '39564209', '100', '+', '.', 'gene_id \"chr21.g21047\"; transcript_id \"blat.proc28144.chain_11988\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '35351491', '35351532', '100', '-', '.', 'gene_id \"chr21.g21049\"; transcript_id \"blat.proc4587.chain_2686\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25358130', '25358293', '100', '+', '.', 'gene_id \"chr21.g21074\"; transcript_id \"blat.proc4587.chain_2003\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '36997240', '36997269', '100', '-', '.', 'gene_id \"chr21.g21091\"; transcript_id \"blat.proc4005.chain_3197\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '13495575', '13496098', '100', '-', '.', 'gene_id \"chr21.g21104\"; transcript_id \"blat.proc28144.chain_886\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '7199143', '7199360', '100', '-', '.', 'gene_id \"chr21.g21110\"; transcript_id \"blat.proc28144.chain_222\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25441850', '25441992', '100', '-', '.', 'gene_id \"chr21.g21118\"; transcript_id \"blat.proc28144.chain_21402\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '30636186', '30636232', '100', '-', '.', 'gene_id \"chr21.g21145\"; transcript_id \"blat.proc18128.chain_24844\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '28849859', '28850332', '100', '-', '.', 'gene_id \"chr21.g21160\"; transcript_id \"blat.proc28144.chain_19172\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '19670881', '19670915', '100', '-', '.', 'gene_id \"chr21.g21236\"; transcript_id \"blat.proc4587.chain_4060\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '1799538', '1799752', '100', '-', '.', 'gene_id \"chr21.g21244\"; transcript_id \"blat.proc28144.chain_11356\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '25221601', '25221710', '100', '+', '.', 'gene_id \"chr21.g21341\"; transcript_id \"blat.proc28144.chain_18894\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '47012697', '47012736', '100', '+', '.', 'gene_id \"chr21.g21365\"; transcript_id \"blat.proc4587.chain_8248\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '11680571', '11680725', '100', '+', '.', 'gene_id \"chr21.g21401\"; transcript_id \"blat.proc28144.chain_9960\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '29652567', '29652732', '100', '-', '.', 'gene_id \"chr21.g21404\"; transcript_id \"blat.proc28144.chain_20085\";']\n",
      "negative:  ['chr21', 'PASA', 'exon', '37241658', '37241753', '100', '+', '.', 'gene_id \"chr21.g21416\"; transcript_id \"blat.proc28144.chain_18116\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24742535', '24742575', '.', '+', '.', 'gene_id \"chr4.g10396\"; transcript_id \"chr4.g10396.i1\";']\n",
      "negative:  ['scaffold_1187', 'PASA', 'exon', '1', '140', '.', '-', '.', 'gene_id \"scaffold_1187.g2\"; transcript_id \"scaffold_1187.g2.i1\";']\n",
      "negative:  ['scaffold_1187', 'PASA', 'exon', '7004', '7053', '.', '+', '.', 'gene_id \"scaffold_1187.g8\"; transcript_id \"scaffold_1187.g8.i1\";']\n",
      "negative:  ['scaffold_1187', 'PASA', 'exon', '9001', '9077', '.', '+', '.', 'gene_id \"scaffold_1187.g11\"; transcript_id \"scaffold_1187.g11.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25007946', '25008219', '.', '-', '.', 'gene_id \"chr4.g10547\"; transcript_id \"chr4.g10547.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25008665', '25008709', '.', '-', '.', 'gene_id \"chr4.g10549\"; transcript_id \"chr4.g10549.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25066406', '25066571', '.', '+', '.', 'gene_id \"chr4.g10575\"; transcript_id \"chr4.g10575.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25258740', '25259370', '.', '+', '.', 'gene_id \"chr4.g10736\"; transcript_id \"chr4.g10736.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25301293', '25301471', '.', '-', '.', 'gene_id \"chr4.g10765\"; transcript_id \"chr4.g10765.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25355839', '25356019', '.', '-', '.', 'gene_id \"chr4.g10820\"; transcript_id \"chr4.g10820.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25367291', '25367330', '.', '-', '.', 'gene_id \"chr4.g10833\"; transcript_id \"chr4.g10833.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25458877', '25459044', '.', '-', '.', 'gene_id \"chr4.g10885\"; transcript_id \"chr4.g10885.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25458618', '25458882', '.', '+', '.', 'gene_id \"chr4.g10886\"; transcript_id \"chr4.g10886.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25459052', '25459290', '.', '+', '.', 'gene_id \"chr4.g10887\"; transcript_id \"chr4.g10887.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25459444', '25459665', '.', '-', '.', 'gene_id \"chr4.g10888\"; transcript_id \"chr4.g10888.i1\";']\n",
      "negative:  ['scaffold_810', 'PASA', 'exon', '5390', '5860', '.', '+', '.', 'gene_id \"scaffold_810.g6\"; transcript_id \"scaffold_810.g6.i1\";']\n",
      "negative:  ['scaffold_810', 'PASA', 'exon', '5841', '6175', '.', '-', '.', 'gene_id \"scaffold_810.g7\"; transcript_id \"scaffold_810.g7.i1\";']\n",
      "negative:  ['scaffold_810', 'PASA', 'exon', '3461', '4282', '100', '-', '.', 'gene_id \"scaffold_810.g9\"; transcript_id \"blat.proc4587.chain_9506\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25484271', '25484569', '.', '-', '.', 'gene_id \"chr4.g10912\"; transcript_id \"chr4.g10912.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25484859', '25485000', '.', '+', '.', 'gene_id \"chr4.g10913\"; transcript_id \"chr4.g10913.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25484659', '25484877', '.', '-', '.', 'gene_id \"chr4.g10914\"; transcript_id \"chr4.g10914.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25494299', '25494335', '.', '+', '.', 'gene_id \"chr4.g10927\"; transcript_id \"chr4.g10927.i1\";']\n",
      "negative:  ['scaffold_510', 'PASA', 'exon', '13787', '13898', '100', '-', '.', 'gene_id \"scaffold_510.g8\"; transcript_id \"blat.proc18128.chain_7242\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25655013', '25655272', '.', '-', '.', 'gene_id \"chr4.g11060\"; transcript_id \"chr4.g11060.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25663280', '25663538', '.', '-', '.', 'gene_id \"chr4.g11074\"; transcript_id \"chr4.g11074.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25707198', '25707353', '.', '-', '.', 'gene_id \"chr4.g11140\"; transcript_id \"chr4.g11140.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26009510', '26009677', '.', '+', '.', 'gene_id \"chr4.g11286\"; transcript_id \"chr4.g11286.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '45321', '45708', '.', '+', '.', 'gene_id \"chr28.g18\"; transcript_id \"chr28.g18.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26521322', '26521572', '.', '-', '.', 'gene_id \"chr4.g11402\"; transcript_id \"chr4.g11402.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26647616', '26647659', '.', '+', '.', 'gene_id \"chr4.g11438\"; transcript_id \"chr4.g11438.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26915583', '26915777', '.', '-', '.', 'gene_id \"chr4.g11489\"; transcript_id \"chr4.g11489.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26915825', '26915951', '.', '-', '.', 'gene_id \"chr4.g11490\"; transcript_id \"chr4.g11490.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '380047', '380238', '.', '-', '.', 'gene_id \"chr28.g152\"; transcript_id \"chr28.g152.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '27247425', '27247730', '.', '-', '.', 'gene_id \"chr4.g11565\"; transcript_id \"chr4.g11565.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '815918', '816203', '.', '-', '.', 'gene_id \"chr28.g383\"; transcript_id \"chr28.g383.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '816848', '817119', '.', '-', '.', 'gene_id \"chr28.g384\"; transcript_id \"chr28.g384.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '840844', '841137', '.', '-', '.', 'gene_id \"chr28.g415\"; transcript_id \"chr28.g415.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '842972', '843795', '.', '-', '.', 'gene_id \"chr28.g420\"; transcript_id \"chr28.g420.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '869411', '869589', '.', '+', '.', 'gene_id \"chr28.g444\"; transcript_id \"chr28.g444.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '887903', '888055', '.', '-', '.', 'gene_id \"chr28.g465\"; transcript_id \"chr28.g465.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '971428', '971581', '.', '-', '.', 'gene_id \"chr28.g539\"; transcript_id \"chr28.g539.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '971647', '972196', '.', '-', '.', 'gene_id \"chr28.g540\"; transcript_id \"chr28.g540.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '28566856', '28566890', '.', '-', '.', 'gene_id \"chr4.g11974\"; transcript_id \"chr4.g11974.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '28770707', '28770741', '.', '+', '.', 'gene_id \"chr4.g11999\"; transcript_id \"chr4.g11999.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '28893778', '28893824', '.', '-', '.', 'gene_id \"chr4.g12030\"; transcript_id \"chr4.g12030.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '28979932', '28979975', '.', '-', '.', 'gene_id \"chr4.g12062\"; transcript_id \"chr4.g12062.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '1233710', '1233869', '.', '-', '.', 'gene_id \"chr28.g661\"; transcript_id \"chr28.g661.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29059477', '29059631', '.', '+', '.', 'gene_id \"chr4.g12138\"; transcript_id \"chr4.g12138.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29082195', '29082360', '.', '-', '.', 'gene_id \"chr4.g12169\"; transcript_id \"chr4.g12169.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29082442', '29082598', '.', '+', '.', 'gene_id \"chr4.g12170\"; transcript_id \"chr4.g12170.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29302326', '29302587', '.', '-', '.', 'gene_id \"chr4.g12350\"; transcript_id \"chr4.g12350.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29317278', '29317313', '.', '-', '.', 'gene_id \"chr4.g12361\"; transcript_id \"chr4.g12361.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29403448', '29403670', '.', '-', '.', 'gene_id \"chr4.g12416\"; transcript_id \"chr4.g12416.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29417909', '29418083', '.', '+', '.', 'gene_id \"chr4.g12442\"; transcript_id \"chr4.g12442.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29573653', '29573702', '.', '+', '.', 'gene_id \"chr4.g12517\"; transcript_id \"chr4.g12517.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29773545', '29774543', '.', '-', '.', 'gene_id \"chr4.g12633\"; transcript_id \"chr4.g12633.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '2184164', '2184395', '.', '-', '.', 'gene_id \"chr28.g1244\"; transcript_id \"chr28.g1244.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '30198435', '30198487', '.', '+', '.', 'gene_id \"chr4.g12706\"; transcript_id \"chr4.g12706.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '30762875', '30762979', '.', '+', '.', 'gene_id \"chr4.g12748\"; transcript_id \"chr4.g12748.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '30794948', '30795139', '.', '+', '.', 'gene_id \"chr4.g12754\"; transcript_id \"chr4.g12754.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '2926218', '2926284', '.', '+', '.', 'gene_id \"chr28.g1439\"; transcript_id \"chr28.g1439.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '2934029', '2934074', '.', '+', '.', 'gene_id \"chr28.g1445\"; transcript_id \"chr28.g1445.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32692562', '32692607', '.', '+', '.', 'gene_id \"chr4.g13050\"; transcript_id \"chr4.g13050.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32795144', '32795354', '.', '+', '.', 'gene_id \"chr4.g13090\"; transcript_id \"chr4.g13090.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32796704', '32796811', '.', '+', '.', 'gene_id \"chr4.g13093\"; transcript_id \"chr4.g13093.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '33363330', '33363501', '.', '+', '.', 'gene_id \"chr4.g13215\"; transcript_id \"chr4.g13215.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '33546871', '33546922', '.', '-', '.', 'gene_id \"chr4.g13223\"; transcript_id \"chr4.g13223.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '33597948', '33597980', '.', '-', '.', 'gene_id \"chr4.g13227\"; transcript_id \"chr4.g13227.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '33632257', '33632312', '.', '+', '.', 'gene_id \"chr4.g13231\"; transcript_id \"chr4.g13231.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34104781', '34105012', '.', '-', '.', 'gene_id \"chr4.g13321\"; transcript_id \"chr4.g13321.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34868276', '34868537', '.', '+', '.', 'gene_id \"chr4.g13705\"; transcript_id \"chr4.g13705.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34875565', '34875598', '.', '+', '.', 'gene_id \"chr4.g13784\"; transcript_id \"chr4.g13784.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '4771986', '4772279', '.', '-', '.', 'gene_id \"chr28.g2301\"; transcript_id \"chr28.g2301.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '4775075', '4775289', '.', '+', '.', 'gene_id \"chr28.g2306\"; transcript_id \"chr28.g2306.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34897555', '34897751', '.', '-', '.', 'gene_id \"chr4.g13809\"; transcript_id \"chr4.g13809.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34918707', '34918946', '.', '+', '.', 'gene_id \"chr4.g13833\"; transcript_id \"chr4.g13833.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34919008', '34919329', '.', '-', '.', 'gene_id \"chr4.g13834\"; transcript_id \"chr4.g13834.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '5256288', '5256355', '.', '+', '.', 'gene_id \"chr28.g2350\"; transcript_id \"chr28.g2350.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34923026', '34923265', '.', '+', '.', 'gene_id \"chr4.g13840\"; transcript_id \"chr4.g13840.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34924153', '34924383', '.', '-', '.', 'gene_id \"chr4.g13842\"; transcript_id \"chr4.g13842.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35146703', '35146773', '.', '+', '.', 'gene_id \"chr4.g13957\"; transcript_id \"chr4.g13957.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35160860', '35161019', '.', '+', '.', 'gene_id \"chr4.g13972\"; transcript_id \"chr4.g13972.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '5857106', '5857258', '.', '-', '.', 'gene_id \"chr28.g2664\"; transcript_id \"chr28.g2664.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '5930447', '5930498', '.', '+', '.', 'gene_id \"chr28.g2686\"; transcript_id \"chr28.g2686.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6045148', '6045542', '.', '+', '.', 'gene_id \"chr28.g2736\"; transcript_id \"chr28.g2736.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35871863', '35872722', '.', '-', '.', 'gene_id \"chr4.g14277\"; transcript_id \"chr4.g14277.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35898375', '35899203', '.', '+', '.', 'gene_id \"chr4.g14282\"; transcript_id \"chr4.g14282.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35899067', '35899168', '.', '+', '.', 'gene_id \"chr4.g14282\"; transcript_id \"chr4.g14282.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35898891', '35899168', '.', '+', '.', 'gene_id \"chr4.g14282\"; transcript_id \"chr4.g14282.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35919361', '35919426', '.', '+', '.', 'gene_id \"chr4.g14286\"; transcript_id \"chr4.g14286.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6531811', '6532357', '.', '-', '.', 'gene_id \"chr28.g3038\"; transcript_id \"chr28.g3038.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6532764', '6533173', '.', '-', '.', 'gene_id \"chr28.g3040\"; transcript_id \"chr28.g3040.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6533286', '6533599', '.', '+', '.', 'gene_id \"chr28.g3041\"; transcript_id \"chr28.g3041.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36331810', '36332027', '.', '-', '.', 'gene_id \"chr4.g14573\"; transcript_id \"chr4.g14573.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6544905', '6545094', '.', '-', '.', 'gene_id \"chr28.g3059\"; transcript_id \"chr28.g3059.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6795875', '6795946', '.', '+', '.', 'gene_id \"chr28.g3228\"; transcript_id \"chr28.g3228.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36597518', '36597550', '.', '-', '.', 'gene_id \"chr4.g14789\"; transcript_id \"chr4.g14789.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36644705', '36645682', '.', '-', '.', 'gene_id \"chr4.g14842\"; transcript_id \"chr4.g14842.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7019583', '7019889', '.', '-', '.', 'gene_id \"chr28.g3354\"; transcript_id \"chr28.g3354.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36850188', '36850230', '.', '-', '.', 'gene_id \"chr4.g14919\"; transcript_id \"chr4.g14919.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7124807', '7124936', '.', '-', '.', 'gene_id \"chr28.g3393\"; transcript_id \"chr28.g3393.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36927447', '36927671', '.', '-', '.', 'gene_id \"chr4.g14992\"; transcript_id \"chr4.g14992.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '36995475', '36995505', '.', '-', '.', 'gene_id \"chr4.g15044\"; transcript_id \"chr4.g15044.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37037377', '37037556', '.', '-', '.', 'gene_id \"chr4.g15087\"; transcript_id \"chr4.g15087.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37055177', '37055259', '.', '-', '.', 'gene_id \"chr4.g15104\"; transcript_id \"chr4.g15104.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr28', 'PASA', 'exon', '7622672', '7622853', '.', '+', '.', 'gene_id \"chr28.g3652\"; transcript_id \"chr28.g3652.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37186976', '37187166', '.', '+', '.', 'gene_id \"chr4.g15245\"; transcript_id \"chr4.g15245.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37187148', '37187427', '.', '-', '.', 'gene_id \"chr4.g15246\"; transcript_id \"chr4.g15246.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7681695', '7681899', '.', '-', '.', 'gene_id \"chr28.g3684\"; transcript_id \"chr28.g3684.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7710666', '7710709', '.', '-', '.', 'gene_id \"chr28.g3698\"; transcript_id \"chr28.g3698.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7848537', '7848743', '.', '-', '.', 'gene_id \"chr28.g3750\"; transcript_id \"chr28.g3750.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37413529', '37413667', '.', '+', '.', 'gene_id \"chr4.g15382\"; transcript_id \"chr4.g15382.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37413693', '37413863', '.', '+', '.', 'gene_id \"chr4.g15383\"; transcript_id \"chr4.g15383.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37620409', '37620536', '.', '-', '.', 'gene_id \"chr4.g15434\"; transcript_id \"chr4.g15434.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37892111', '37892319', '.', '-', '.', 'gene_id \"chr4.g15495\"; transcript_id \"chr4.g15495.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38172345', '38172377', '.', '+', '.', 'gene_id \"chr4.g15514\"; transcript_id \"chr4.g15514.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38485581', '38485614', '.', '+', '.', 'gene_id \"chr4.g15547\"; transcript_id \"chr4.g15547.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8430380', '8430629', '.', '+', '.', 'gene_id \"chr28.g3999\"; transcript_id \"chr28.g3999.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8459803', '8460211', '.', '-', '.', 'gene_id \"chr28.g4014\"; transcript_id \"chr28.g4014.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8505739', '8506053', '.', '-', '.', 'gene_id \"chr28.g4049\"; transcript_id \"chr28.g4049.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38850587', '38850769', '.', '-', '.', 'gene_id \"chr4.g15715\"; transcript_id \"chr4.g15715.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38877053', '38877272', '.', '-', '.', 'gene_id \"chr4.g15722\"; transcript_id \"chr4.g15722.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8655261', '8655322', '.', '+', '.', 'gene_id \"chr28.g4163\"; transcript_id \"chr28.g4163.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8722892', '8722947', '.', '+', '.', 'gene_id \"chr28.g4190\"; transcript_id \"chr28.g4190.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39012565', '39012746', '.', '+', '.', 'gene_id \"chr4.g15783\"; transcript_id \"chr4.g15783.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8763721', '8763760', '.', '+', '.', 'gene_id \"chr28.g4242\"; transcript_id \"chr28.g4242.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8763706', '8763956', '.', '-', '.', 'gene_id \"chr28.g4243\"; transcript_id \"chr28.g4243.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8841950', '8842174', '.', '-', '.', 'gene_id \"chr28.g4332\"; transcript_id \"chr28.g4332.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39211879', '39211963', '.', '+', '.', 'gene_id \"chr4.g15902\"; transcript_id \"chr4.g15902.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8894656', '8894816', '.', '+', '.', 'gene_id \"chr28.g4393\"; transcript_id \"chr28.g4393.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8904038', '8904257', '.', '+', '.', 'gene_id \"chr28.g4405\"; transcript_id \"chr28.g4405.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8909568', '8909946', '.', '-', '.', 'gene_id \"chr28.g4414\"; transcript_id \"chr28.g4414.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8927773', '8927982', '.', '+', '.', 'gene_id \"chr28.g4433\"; transcript_id \"chr28.g4433.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8973162', '8973198', '.', '-', '.', 'gene_id \"chr28.g4445\"; transcript_id \"chr28.g4445.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39466942', '39467097', '.', '+', '.', 'gene_id \"chr4.g16007\"; transcript_id \"chr4.g16007.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39467253', '39467568', '.', '+', '.', 'gene_id \"chr4.g16008\"; transcript_id \"chr4.g16008.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39468686', '39469501', '.', '+', '.', 'gene_id \"chr4.g16009\"; transcript_id \"chr4.g16009.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9042443', '9042474', '.', '+', '.', 'gene_id \"chr28.g4494\"; transcript_id \"chr28.g4494.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39526716', '39526770', '.', '-', '.', 'gene_id \"chr4.g16062\"; transcript_id \"chr4.g16062.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39600725', '39600916', '.', '-', '.', 'gene_id \"chr4.g16150\"; transcript_id \"chr4.g16150.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39600941', '39601149', '.', '+', '.', 'gene_id \"chr4.g16151\"; transcript_id \"chr4.g16151.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9348903', '9349053', '.', '-', '.', 'gene_id \"chr28.g4734\"; transcript_id \"chr28.g4734.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9516863', '9517021', '.', '+', '.', 'gene_id \"chr28.g4829\"; transcript_id \"chr28.g4829.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9564171', '9564214', '.', '+', '.', 'gene_id \"chr28.g4857\"; transcript_id \"chr28.g4857.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39989658', '39989691', '.', '+', '.', 'gene_id \"chr4.g16408\"; transcript_id \"chr4.g16408.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9637410', '9637588', '.', '+', '.', 'gene_id \"chr28.g4947\"; transcript_id \"chr28.g4947.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '9637846', '9638079', '.', '-', '.', 'gene_id \"chr28.g4948\"; transcript_id \"chr28.g4948.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '40165090', '40165132', '.', '-', '.', 'gene_id \"chr4.g16519\"; transcript_id \"chr4.g16519.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '40288852', '40288909', '.', '+', '.', 'gene_id \"chr4.g16638\"; transcript_id \"chr4.g16638.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10291726', '10299945', '.', '-', '.', 'gene_id \"chr28.g5271\"; transcript_id \"chr28.g5271.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10280752', '10280962', '.', '-', '.', 'gene_id \"chr28.g5273\"; transcript_id \"chr28.g5273.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10284744', '10284953', '.', '-', '.', 'gene_id \"chr28.g5281\"; transcript_id \"chr28.g5281.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10314611', '10320862', '.', '-', '.', 'gene_id \"chr28.g5300\"; transcript_id \"chr28.g5300.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '40821586', '40821617', '.', '+', '.', 'gene_id \"chr4.g16864\"; transcript_id \"chr4.g16864.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41039469', '41039733', '.', '+', '.', 'gene_id \"chr4.g16948\"; transcript_id \"chr4.g16948.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41068110', '41068269', '.', '-', '.', 'gene_id \"chr4.g16979\"; transcript_id \"chr4.g16979.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41199309', '41199503', '.', '+', '.', 'gene_id \"chr4.g17048\"; transcript_id \"chr4.g17048.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41199588', '41199765', '.', '+', '.', 'gene_id \"chr4.g17049\"; transcript_id \"chr4.g17049.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41278310', '41278595', '.', '+', '.', 'gene_id \"chr4.g17099\"; transcript_id \"chr4.g17099.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10919489', '10919528', '.', '-', '.', 'gene_id \"chr28.g5760\"; transcript_id \"chr28.g5760.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41908427', '41908473', '.', '-', '.', 'gene_id \"chr4.g17260\"; transcript_id \"chr4.g17260.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '41972592', '41972763', '.', '-', '.', 'gene_id \"chr4.g17269\"; transcript_id \"chr4.g17269.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11018256', '11018407', '.', '-', '.', 'gene_id \"chr28.g5837\"; transcript_id \"chr28.g5837.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11288954', '11289017', '.', '-', '.', 'gene_id \"chr28.g5908\"; transcript_id \"chr28.g5908.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11383531', '11383562', '.', '+', '.', 'gene_id \"chr28.g5992\"; transcript_id \"chr28.g5992.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '42743097', '42743561', '.', '-', '.', 'gene_id \"chr4.g17478\"; transcript_id \"chr4.g17478.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '42815143', '42815388', '.', '-', '.', 'gene_id \"chr4.g17505\"; transcript_id \"chr4.g17505.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '42841366', '42841531', '.', '-', '.', 'gene_id \"chr4.g17530\"; transcript_id \"chr4.g17530.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11626797', '11626856', '.', '-', '.', 'gene_id \"chr28.g6107\"; transcript_id \"chr28.g6107.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11936915', '11937116', '.', '-', '.', 'gene_id \"chr28.g6132\"; transcript_id \"chr28.g6132.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12323934', '12323973', '.', '+', '.', 'gene_id \"chr28.g6229\"; transcript_id \"chr28.g6229.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12393068', '12393105', '.', '+', '.', 'gene_id \"chr28.g6294\"; transcript_id \"chr28.g6294.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43262441', '43262744', '.', '+', '.', 'gene_id \"chr4.g17779\"; transcript_id \"chr4.g17779.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12624771', '12624926', '.', '-', '.', 'gene_id \"chr28.g6352\"; transcript_id \"chr28.g6352.i5\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12624801', '12624926', '.', '-', '.', 'gene_id \"chr28.g6352\"; transcript_id \"chr28.g6352.i6\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12849824', '12850428', '.', '+', '.', 'gene_id \"chr28.g6560\"; transcript_id \"chr28.g6560.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12849659', '12849848', '.', '-', '.', 'gene_id \"chr28.g6562\"; transcript_id \"chr28.g6562.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12878063', '12878102', '.', '-', '.', 'gene_id \"chr28.g6598\"; transcript_id \"chr28.g6598.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43655838', '43656089', '.', '+', '.', 'gene_id \"chr4.g18131\"; transcript_id \"chr4.g18131.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43696928', '43697116', '.', '+', '.', 'gene_id \"chr4.g18155\"; transcript_id \"chr4.g18155.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43697308', '43697930', '.', '+', '.', 'gene_id \"chr4.g18157\"; transcript_id \"chr4.g18157.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43716212', '43716246', '.', '+', '.', 'gene_id \"chr4.g18175\"; transcript_id \"chr4.g18175.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43881637', '43881860', '.', '-', '.', 'gene_id \"chr4.g18209\"; transcript_id \"chr4.g18209.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43933625', '43933841', '.', '-', '.', 'gene_id \"chr4.g18232\"; transcript_id \"chr4.g18232.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '43933842', '43934022', '.', '-', '.', 'gene_id \"chr4.g18234\"; transcript_id \"chr4.g18234.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13151691', '13151855', '.', '+', '.', 'gene_id \"chr28.g6812\"; transcript_id \"chr28.g6812.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13202895', '13203118', '.', '-', '.', 'gene_id \"chr28.g6822\"; transcript_id \"chr28.g6822.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13225967', '13226235', '.', '-', '.', 'gene_id \"chr28.g6836\"; transcript_id \"chr28.g6836.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13225967', '13226235', '.', '-', '.', 'gene_id \"chr28.g6836\"; transcript_id \"chr28.g6836.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13225967', '13226235', '.', '-', '.', 'gene_id \"chr28.g6836\"; transcript_id \"chr28.g6836.i3\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13225967', '13226235', '.', '-', '.', 'gene_id \"chr28.g6836\"; transcript_id \"chr28.g6836.i4\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13225967', '13226235', '.', '-', '.', 'gene_id \"chr28.g6836\"; transcript_id \"chr28.g6836.i5\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13243944', '13244138', '.', '-', '.', 'gene_id \"chr28.g6841\"; transcript_id \"chr28.g6841.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '44124365', '44124399', '.', '+', '.', 'gene_id \"chr4.g18278\"; transcript_id \"chr4.g18278.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '44309089', '44309123', '.', '+', '.', 'gene_id \"chr4.g18299\"; transcript_id \"chr4.g18299.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13341110', '13341415', '.', '+', '.', 'gene_id \"chr28.g6916\"; transcript_id \"chr28.g6916.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '45695156', '45695205', '.', '-', '.', 'gene_id \"chr4.g18412\"; transcript_id \"chr4.g18412.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '46370050', '46370079', '.', '-', '.', 'gene_id \"chr4.g18450\"; transcript_id \"chr4.g18450.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '47576821', '47576893', '.', '+', '.', 'gene_id \"chr4.g18522\"; transcript_id \"chr4.g18522.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '47715440', '47715470', '.', '+', '.', 'gene_id \"chr4.g18533\"; transcript_id \"chr4.g18533.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13979095', '13979261', '.', '-', '.', 'gene_id \"chr28.g7152\"; transcript_id \"chr28.g7152.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14022586', '14022742', '.', '-', '.', 'gene_id \"chr28.g7179\"; transcript_id \"chr28.g7179.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '47926568', '47926733', '.', '+', '.', 'gene_id \"chr4.g18577\"; transcript_id \"chr4.g18577.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '48181559', '48181933', '.', '+', '.', 'gene_id \"chr4.g18672\"; transcript_id \"chr4.g18672.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '48253989', '48254159', '.', '-', '.', 'gene_id \"chr4.g18686\"; transcript_id \"chr4.g18686.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '48555145', '48555178', '.', '+', '.', 'gene_id \"chr4.g18712\"; transcript_id \"chr4.g18712.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49161570', '49162024', '.', '+', '.', 'gene_id \"chr4.g18761\"; transcript_id \"chr4.g18761.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49161668', '49161757', '.', '+', '.', 'gene_id \"chr4.g18761\"; transcript_id \"chr4.g18761.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49908349', '49908746', '.', '-', '.', 'gene_id \"chr4.g18847\"; transcript_id \"chr4.g18847.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49908542', '49909025', '.', '-', '.', 'gene_id \"chr4.g18847\"; transcript_id \"chr4.g18847.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49956344', '49956392', '.', '-', '.', 'gene_id \"chr4.g18852\"; transcript_id \"chr4.g18852.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50197671', '50197898', '.', '+', '.', 'gene_id \"chr4.g18896\"; transcript_id \"chr4.g18896.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14481996', '14483469', '.', '+', '.', 'gene_id \"chr28.g7536\"; transcript_id \"chr28.g7536.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50419042', '50419396', '.', '-', '.', 'gene_id \"chr4.g18956\"; transcript_id \"chr4.g18956.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50427492', '50427907', '.', '+', '.', 'gene_id \"chr4.g18965\"; transcript_id \"chr4.g18965.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50652180', '50652218', '.', '+', '.', 'gene_id \"chr4.g19065\"; transcript_id \"chr4.g19065.i8\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50661102', '50661257', '.', '-', '.', 'gene_id \"chr4.g19083\"; transcript_id \"chr4.g19083.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14637469', '14637668', '.', '-', '.', 'gene_id \"chr28.g7686\"; transcript_id \"chr28.g7686.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14637683', '14637882', '.', '+', '.', 'gene_id \"chr28.g7687\"; transcript_id \"chr28.g7687.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50680284', '50680334', '.', '+', '.', 'gene_id \"chr4.g19098\"; transcript_id \"chr4.g19098.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14670225', '14670398', '.', '-', '.', 'gene_id \"chr28.g7703\"; transcript_id \"chr28.g7703.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14670483', '14670854', '.', '+', '.', 'gene_id \"chr28.g7704\"; transcript_id \"chr28.g7704.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14827461', '14827494', '.', '+', '.', 'gene_id \"chr28.g7824\"; transcript_id \"chr28.g7824.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50881912', '50882096', '.', '+', '.', 'gene_id \"chr4.g19255\"; transcript_id \"chr4.g19255.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14864544', '14864785', '.', '+', '.', 'gene_id \"chr28.g7866\"; transcript_id \"chr28.g7866.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14897879', '14897924', '.', '+', '.', 'gene_id \"chr28.g7902\"; transcript_id \"chr28.g7902.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50941883', '50941946', '.', '-', '.', 'gene_id \"chr4.g19292\"; transcript_id \"chr4.g19292.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14978822', '14978858', '.', '+', '.', 'gene_id \"chr28.g7989\"; transcript_id \"chr28.g7989.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51256094', '51256124', '.', '-', '.', 'gene_id \"chr4.g19354\"; transcript_id \"chr4.g19354.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51295897', '51295929', '.', '+', '.', 'gene_id \"chr4.g19360\"; transcript_id \"chr4.g19360.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51527065', '51527484', '.', '+', '.', 'gene_id \"chr4.g19474\"; transcript_id \"chr4.g19474.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15165247', '15165278', '.', '+', '.', 'gene_id \"chr28.g8175\"; transcript_id \"chr28.g8175.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51608267', '51608356', '.', '+', '.', 'gene_id \"chr4.g19583\"; transcript_id \"chr4.g19583.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51639400', '51639839', '.', '-', '.', 'gene_id \"chr4.g19590\"; transcript_id \"chr4.g19590.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51639854', '51640012', '.', '+', '.', 'gene_id \"chr4.g19591\"; transcript_id \"chr4.g19591.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15272764', '15272901', '.', '+', '.', 'gene_id \"chr28.g8292\"; transcript_id \"chr28.g8292.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51775860', '51776053', '.', '+', '.', 'gene_id \"chr4.g19672\"; transcript_id \"chr4.g19672.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51940891', '51941322', '.', '-', '.', 'gene_id \"chr4.g19729\"; transcript_id \"chr4.g19729.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51941323', '51941520', '.', '+', '.', 'gene_id \"chr4.g19730\"; transcript_id \"chr4.g19730.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51952489', '51952532', '.', '-', '.', 'gene_id \"chr4.g19734\"; transcript_id \"chr4.g19734.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15369467', '15369667', '.', '+', '.', 'gene_id \"chr28.g8386\"; transcript_id \"chr28.g8386.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52307385', '52307433', '.', '-', '.', 'gene_id \"chr4.g19779\"; transcript_id \"chr4.g19779.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52711270', '52711329', '.', '-', '.', 'gene_id \"chr4.g19812\"; transcript_id \"chr4.g19812.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52725018', '52725095', '.', '-', '.', 'gene_id \"chr4.g19829\"; transcript_id \"chr4.g19829.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52725949', '52726174', '.', '-', '.', 'gene_id \"chr4.g19832\"; transcript_id \"chr4.g19832.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52726157', '52726349', '.', '+', '.', 'gene_id \"chr4.g19833\"; transcript_id \"chr4.g19833.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15607022', '15607250', '.', '-', '.', 'gene_id \"chr28.g8578\"; transcript_id \"chr28.g8578.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15607254', '15607521', '.', '+', '.', 'gene_id \"chr28.g8579\"; transcript_id \"chr28.g8579.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52886681', '52886876', '.', '+', '.', 'gene_id \"chr4.g19929\"; transcript_id \"chr4.g19929.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52887030', '52887535', '.', '-', '.', 'gene_id \"chr4.g19930\"; transcript_id \"chr4.g19930.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15738329', '15738389', '.', '+', '.', 'gene_id \"chr28.g8653\"; transcript_id \"chr28.g8653.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53273713', '53273902', '.', '+', '.', 'gene_id \"chr4.g20198\"; transcript_id \"chr4.g20198.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53274333', '53274531', '.', '+', '.', 'gene_id \"chr4.g20200\"; transcript_id \"chr4.g20200.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53387460', '53387723', '.', '+', '.', 'gene_id \"chr4.g20293\"; transcript_id \"chr4.g20293.i3\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16350339', '16350539', '.', '-', '.', 'gene_id \"chr28.g9040\"; transcript_id \"chr28.g9040.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53595965', '53596435', '.', '-', '.', 'gene_id \"chr4.g20363\"; transcript_id \"chr4.g20363.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53596813', '53597130', '.', '+', '.', 'gene_id \"chr4.g20366\"; transcript_id \"chr4.g20366.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53734805', '53735058', '.', '+', '.', 'gene_id \"chr4.g20393\"; transcript_id \"chr4.g20393.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53764146', '53764391', '.', '+', '.', 'gene_id \"chr4.g20397\"; transcript_id \"chr4.g20397.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16467483', '16467663', '.', '+', '.', 'gene_id \"chr28.g9106\"; transcript_id \"chr28.g9106.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16467771', '16468054', '.', '+', '.', 'gene_id \"chr28.g9107\"; transcript_id \"chr28.g9107.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16468079', '16468298', '.', '-', '.', 'gene_id \"chr28.g9108\"; transcript_id \"chr28.g9108.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16468278', '16468865', '.', '+', '.', 'gene_id \"chr28.g9109\"; transcript_id \"chr28.g9109.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53942340', '53942377', '.', '+', '.', 'gene_id \"chr4.g20420\"; transcript_id \"chr4.g20420.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54553693', '54553882', '.', '+', '.', 'gene_id \"chr4.g20492\"; transcript_id \"chr4.g20492.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54622047', '54622281', '.', '-', '.', 'gene_id \"chr4.g20500\"; transcript_id \"chr4.g20500.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54731754', '54731790', '.', '+', '.', 'gene_id \"chr4.g20509\"; transcript_id \"chr4.g20509.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55256182', '55256372', '.', '+', '.', 'gene_id \"chr4.g20676\"; transcript_id \"chr4.g20676.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '18042664', '18042799', '.', '+', '.', 'gene_id \"chr28.g9340\"; transcript_id \"chr28.g9340.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '18531767', '18531796', '.', '-', '.', 'gene_id \"chr28.g9363\"; transcript_id \"chr28.g9363.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '18555043', '18556086', '.', '+', '.', 'gene_id \"chr28.g9368\"; transcript_id \"chr28.g9368.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '18557088', '18557823', '.', '+', '.', 'gene_id \"chr28.g9369\"; transcript_id \"chr28.g9369.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55319308', '55319551', '.', '-', '.', 'gene_id \"chr4.g20779\"; transcript_id \"chr4.g20779.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '20078397', '20078429', '.', '-', '.', 'gene_id \"chr28.g9560\"; transcript_id \"chr28.g9560.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55698334', '55698403', '.', '-', '.', 'gene_id \"chr4.g21037\"; transcript_id \"chr4.g21037.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '21395947', '21396108', '.', '-', '.', 'gene_id \"chr28.g9711\"; transcript_id \"chr28.g9711.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '21479244', '21479283', '.', '-', '.', 'gene_id \"chr28.g9730\"; transcript_id \"chr28.g9730.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '21676285', '21676667', '.', '-', '.', 'gene_id \"chr28.g9879\"; transcript_id \"chr28.g9879.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '21722452', '21722857', '.', '+', '.', 'gene_id \"chr28.g9908\"; transcript_id \"chr28.g9908.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56272200', '56272288', '.', '-', '.', 'gene_id \"chr4.g21290\"; transcript_id \"chr4.g21290.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56876393', '56876514', '.', '-', '.', 'gene_id \"chr4.g21329\"; transcript_id \"chr4.g21329.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57324245', '57324450', '.', '+', '.', 'gene_id \"chr4.g21383\"; transcript_id \"chr4.g21383.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '21928370', '21928583', '.', '-', '.', 'gene_id \"chr28.g10042\"; transcript_id \"chr28.g10042.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57663666', '57663697', '.', '-', '.', 'gene_id \"chr4.g21413\"; transcript_id \"chr4.g21413.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57763101', '57763437', '.', '-', '.', 'gene_id \"chr4.g21422\"; transcript_id \"chr4.g21422.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '22198629', '22198666', '.', '+', '.', 'gene_id \"chr28.g10132\"; transcript_id \"chr28.g10132.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57872310', '57872792', '.', '-', '.', 'gene_id \"chr4.g21546\"; transcript_id \"chr4.g21546.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57904499', '57904788', '.', '+', '.', 'gene_id \"chr4.g21554\"; transcript_id \"chr4.g21554.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57946204', '57946347', '.', '+', '.', 'gene_id \"chr4.g21587\"; transcript_id \"chr4.g21587.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '22384374', '22384421', '.', '-', '.', 'gene_id \"chr28.g10262\"; transcript_id \"chr28.g10262.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58104065', '58104318', '.', '-', '.', 'gene_id \"chr4.g21655\"; transcript_id \"chr4.g21655.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '22727091', '22727255', '.', '+', '.', 'gene_id \"chr28.g10333\"; transcript_id \"chr28.g10333.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58292857', '58293024', '.', '-', '.', 'gene_id \"chr4.g21712\"; transcript_id \"chr4.g21712.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58323924', '58324132', '.', '-', '.', 'gene_id \"chr4.g21732\"; transcript_id \"chr4.g21732.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58348042', '58348249', '.', '-', '.', 'gene_id \"chr4.g21760\"; transcript_id \"chr4.g21760.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58348392', '58348549', '.', '-', '.', 'gene_id \"chr4.g21761\"; transcript_id \"chr4.g21761.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58371698', '58371884', '.', '-', '.', 'gene_id \"chr4.g21781\"; transcript_id \"chr4.g21781.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58382540', '58382820', '.', '+', '.', 'gene_id \"chr4.g21785\"; transcript_id \"chr4.g21785.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '23511258', '23511307', '.', '+', '.', 'gene_id \"chr28.g10490\"; transcript_id \"chr28.g10490.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '23626149', '23626340', '.', '+', '.', 'gene_id \"chr28.g10569\"; transcript_id \"chr28.g10569.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '23847443', '23847604', '.', '-', '.', 'gene_id \"chr28.g10639\"; transcript_id \"chr28.g10639.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58671984', '58672210', '.', '+', '.', 'gene_id \"chr4.g22018\"; transcript_id \"chr4.g22018.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58672714', '58672943', '.', '+', '.', 'gene_id \"chr4.g22020\"; transcript_id \"chr4.g22020.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24057944', '24058221', '.', '+', '.', 'gene_id \"chr28.g10709\"; transcript_id \"chr28.g10709.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58734104', '58734143', '.', '-', '.', 'gene_id \"chr4.g22106\"; transcript_id \"chr4.g22106.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24058226', '24058672', '.', '+', '.', 'gene_id \"chr28.g10710\"; transcript_id \"chr28.g10710.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24058717', '24059592', '.', '-', '.', 'gene_id \"chr28.g10711\"; transcript_id \"chr28.g10711.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24059630', '24059905', '.', '+', '.', 'gene_id \"chr28.g10712\"; transcript_id \"chr28.g10712.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24067089', '24067461', '.', '+', '.', 'gene_id \"chr28.g10723\"; transcript_id \"chr28.g10723.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr28', 'PASA', 'exon', '24146937', '24147117', '.', '-', '.', 'gene_id \"chr28.g10801\"; transcript_id \"chr28.g10801.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24691037', '24691479', '.', '+', '.', 'gene_id \"chr28.g10985\"; transcript_id \"chr28.g10985.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24691733', '24691938', '.', '-', '.', 'gene_id \"chr28.g10986\"; transcript_id \"chr28.g10986.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24691919', '24692376', '.', '+', '.', 'gene_id \"chr28.g10987\"; transcript_id \"chr28.g10987.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59349024', '59349358', '.', '+', '.', 'gene_id \"chr4.g22484\"; transcript_id \"chr4.g22484.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59594928', '59594960', '.', '+', '.', 'gene_id \"chr4.g22625\"; transcript_id \"chr4.g22625.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59682093', '59682163', '.', '-', '.', 'gene_id \"chr4.g22721\"; transcript_id \"chr4.g22721.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59706231', '59706955', '.', '-', '.', 'gene_id \"chr4.g22740\"; transcript_id \"chr4.g22740.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59706998', '59707254', '.', '+', '.', 'gene_id \"chr4.g22744\"; transcript_id \"chr4.g22744.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59729578', '59729946', '.', '-', '.', 'gene_id \"chr4.g22785\"; transcript_id \"chr4.g22785.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59759873', '59760134', '.', '+', '.', 'gene_id \"chr4.g22812\"; transcript_id \"chr4.g22812.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59787432', '59787481', '.', '+', '.', 'gene_id \"chr4.g22828\"; transcript_id \"chr4.g22828.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59858783', '59858831', '.', '+', '.', 'gene_id \"chr4.g22878\"; transcript_id \"chr4.g22878.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '25420836', '25421076', '.', '-', '.', 'gene_id \"chr28.g11536\"; transcript_id \"chr28.g11536.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60020389', '60020480', '.', '+', '.', 'gene_id \"chr4.g23036\"; transcript_id \"chr4.g23036.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60020457', '60020619', '.', '+', '.', 'gene_id \"chr4.g23036\"; transcript_id \"chr4.g23036.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60181761', '60181796', '.', '+', '.', 'gene_id \"chr4.g23147\"; transcript_id \"chr4.g23147.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '25620891', '25621130', '.', '-', '.', 'gene_id \"chr28.g11687\"; transcript_id \"chr28.g11687.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60302549', '60303268', '.', '+', '.', 'gene_id \"chr4.g23230\"; transcript_id \"chr4.g23230.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '26204926', '26204961', '.', '+', '.', 'gene_id \"chr28.g11881\"; transcript_id \"chr28.g11881.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60678707', '60679085', '.', '-', '.', 'gene_id \"chr4.g23358\"; transcript_id \"chr4.g23358.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61031512', '61031544', '.', '+', '.', 'gene_id \"chr4.g23589\"; transcript_id \"chr4.g23589.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61396762', '61397485', '.', '-', '.', 'gene_id \"chr4.g23775\"; transcript_id \"chr4.g23775.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61446771', '61446933', '.', '-', '.', 'gene_id \"chr4.g23797\"; transcript_id \"chr4.g23797.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61589909', '61590065', '.', '+', '.', 'gene_id \"chr4.g23862\"; transcript_id \"chr4.g23862.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61590526', '61590729', '.', '-', '.', 'gene_id \"chr4.g23864\"; transcript_id \"chr4.g23864.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '27948117', '27948670', '.', '+', '.', 'gene_id \"chr28.g12435\"; transcript_id \"chr28.g12435.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28081404', '28083844', '.', '-', '.', 'gene_id \"chr28.g12461\"; transcript_id \"chr28.g12461.i3\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28083500', '28083570', '.', '-', '.', 'gene_id \"chr28.g12461\"; transcript_id \"chr28.g12461.i7\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28084619', '28084876', '.', '-', '.', 'gene_id \"chr28.g12471\"; transcript_id \"chr28.g12471.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61662896', '61663158', '.', '-', '.', 'gene_id \"chr4.g23942\"; transcript_id \"chr4.g23942.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28179754', '28179817', '.', '-', '.', 'gene_id \"chr28.g12531\"; transcript_id \"chr28.g12531.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28271170', '28271480', '.', '-', '.', 'gene_id \"chr28.g12545\"; transcript_id \"chr28.g12545.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28271489', '28271704', '.', '+', '.', 'gene_id \"chr28.g12546\"; transcript_id \"chr28.g12546.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28272003', '28272234', '.', '+', '.', 'gene_id \"chr28.g12548\"; transcript_id \"chr28.g12548.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28272708', '28273158', '.', '+', '.', 'gene_id \"chr28.g12551\"; transcript_id \"chr28.g12551.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28406790', '28407428', '.', '+', '.', 'gene_id \"chr28.g12609\"; transcript_id \"chr28.g12609.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '28549360', '28549553', '.', '-', '.', 'gene_id \"chr28.g12675\"; transcript_id \"chr28.g12675.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62064212', '62064290', '.', '+', '.', 'gene_id \"chr4.g24164\"; transcript_id \"chr4.g24164.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62076709', '62076773', '.', '+', '.', 'gene_id \"chr4.g24167\"; transcript_id \"chr4.g24167.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62172750', '62172787', '.', '+', '.', 'gene_id \"chr4.g24174\"; transcript_id \"chr4.g24174.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62426638', '62426788', '.', '+', '.', 'gene_id \"chr4.g24193\"; transcript_id \"chr4.g24193.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29109412', '29109590', '.', '+', '.', 'gene_id \"chr28.g12794\"; transcript_id \"chr28.g12794.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29109616', '29109891', '.', '+', '.', 'gene_id \"chr28.g12795\"; transcript_id \"chr28.g12795.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29109870', '29110165', '.', '-', '.', 'gene_id \"chr28.g12796\"; transcript_id \"chr28.g12796.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29168208', '29168555', '.', '-', '.', 'gene_id \"chr28.g12851\"; transcript_id \"chr28.g12851.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29240417', '29240463', '.', '+', '.', 'gene_id \"chr28.g12862\"; transcript_id \"chr28.g12862.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62688963', '62689008', '.', '+', '.', 'gene_id \"chr4.g24365\"; transcript_id \"chr4.g24365.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29450585', '29450821', '.', '+', '.', 'gene_id \"chr28.g12977\"; transcript_id \"chr28.g12977.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29450799', '29450969', '.', '-', '.', 'gene_id \"chr28.g12978\"; transcript_id \"chr28.g12978.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29509536', '29509571', '.', '-', '.', 'gene_id \"chr28.g13018\"; transcript_id \"chr28.g13018.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '64413106', '64413312', '.', '+', '.', 'gene_id \"chr4.g24564\"; transcript_id \"chr4.g24564.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '64729500', '64729895', '.', '+', '.', 'gene_id \"chr4.g24718\"; transcript_id \"chr4.g24718.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '64875223', '64875262', '.', '+', '.', 'gene_id \"chr4.g24730\"; transcript_id \"chr4.g24730.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29845446', '29845660', '.', '-', '.', 'gene_id \"chr28.g13359\"; transcript_id \"chr28.g13359.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29933350', '29933401', '.', '+', '.', 'gene_id \"chr28.g13385\"; transcript_id \"chr28.g13385.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '65791499', '65791809', '.', '+', '.', 'gene_id \"chr4.g24855\"; transcript_id \"chr4.g24855.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '30633714', '30633773', '.', '-', '.', 'gene_id \"chr28.g13549\"; transcript_id \"chr28.g13549.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '30897992', '30898389', '.', '+', '.', 'gene_id \"chr28.g13651\"; transcript_id \"chr28.g13651.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31174210', '31174245', '.', '+', '.', 'gene_id \"chr28.g13666\"; transcript_id \"chr28.g13666.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31318044', '31318080', '.', '+', '.', 'gene_id \"chr28.g13673\"; transcript_id \"chr28.g13673.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31346254', '31347553', '.', '+', '.', 'gene_id \"chr28.g13682\"; transcript_id \"chr28.g13682.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '66876388', '66876417', '.', '-', '.', 'gene_id \"chr4.g25133\"; transcript_id \"chr4.g25133.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31619306', '31620195', '.', '-', '.', 'gene_id \"chr28.g13819\"; transcript_id \"chr28.g13819.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31631130', '31631436', '.', '-', '.', 'gene_id \"chr28.g13831\"; transcript_id \"chr28.g13831.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31632185', '31632402', '.', '-', '.', 'gene_id \"chr28.g13832\"; transcript_id \"chr28.g13832.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31751659', '31751825', '.', '+', '.', 'gene_id \"chr28.g13879\"; transcript_id \"chr28.g13879.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67240984', '67241690', '.', '+', '.', 'gene_id \"chr4.g25356\"; transcript_id \"chr4.g25356.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67303518', '67303746', '.', '+', '.', 'gene_id \"chr4.g25434\"; transcript_id \"chr4.g25434.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31885870', '31885914', '.', '+', '.', 'gene_id \"chr28.g13985\"; transcript_id \"chr28.g13985.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67317870', '67317951', '.', '+', '.', 'gene_id \"chr4.g25454\"; transcript_id \"chr4.g25454.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67341542', '67341594', '.', '-', '.', 'gene_id \"chr4.g25479\"; transcript_id \"chr4.g25479.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67345601', '67346128', '.', '-', '.', 'gene_id \"chr4.g25489\"; transcript_id \"chr4.g25489.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67346135', '67347129', '.', '+', '.', 'gene_id \"chr4.g25490\"; transcript_id \"chr4.g25490.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67348375', '67349555', '.', '+', '.', 'gene_id \"chr4.g25491\"; transcript_id \"chr4.g25491.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67348375', '67348465', '.', '+', '.', 'gene_id \"chr4.g25491\"; transcript_id \"chr4.g25491.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31992784', '31993409', '.', '+', '.', 'gene_id \"chr28.g14104\"; transcript_id \"chr28.g14104.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67418616', '67418821', '.', '+', '.', 'gene_id \"chr4.g25600\"; transcript_id \"chr4.g25600.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '32099426', '32099514', '.', '+', '.', 'gene_id \"chr28.g14177\"; transcript_id \"chr28.g14177.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '32102979', '32103186', '.', '+', '.', 'gene_id \"chr28.g14184\"; transcript_id \"chr28.g14184.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67496796', '67496826', '.', '+', '.', 'gene_id \"chr4.g25709\"; transcript_id \"chr4.g25709.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67506409', '67507815', '.', '+', '.', 'gene_id \"chr4.g25727\"; transcript_id \"chr4.g25727.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '32227276', '32227365', '.', '+', '.', 'gene_id \"chr28.g14293\"; transcript_id \"chr28.g14293.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67579569', '67579643', '.', '+', '.', 'gene_id \"chr4.g25784\"; transcript_id \"chr4.g25784.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67740014', '67740055', '.', '-', '.', 'gene_id \"chr4.g25801\"; transcript_id \"chr4.g25801.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67825805', '67825860', '.', '-', '.', 'gene_id \"chr4.g25821\"; transcript_id \"chr4.g25821.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67935042', '67935584', '.', '+', '.', 'gene_id \"chr4.g25869\"; transcript_id \"chr4.g25869.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67989193', '67989439', '.', '+', '.', 'gene_id \"chr4.g25919\"; transcript_id \"chr4.g25919.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68076707', '68076742', '.', '+', '.', 'gene_id \"chr4.g25934\"; transcript_id \"chr4.g25934.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68114270', '68115147', '.', '-', '.', 'gene_id \"chr4.g25967\"; transcript_id \"chr4.g25967.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '32975844', '32975877', '.', '-', '.', 'gene_id \"chr28.g14636\"; transcript_id \"chr28.g14636.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68589691', '68590490', '.', '+', '.', 'gene_id \"chr4.g26028\"; transcript_id \"chr4.g26028.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68590477', '68590648', '.', '-', '.', 'gene_id \"chr4.g26033\"; transcript_id \"chr4.g26033.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68592518', '68592935', '.', '-', '.', 'gene_id \"chr4.g26072\"; transcript_id \"chr4.g26072.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68592995', '68593319', '.', '-', '.', 'gene_id \"chr4.g26073\"; transcript_id \"chr4.g26073.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68593323', '68593682', '.', '-', '.', 'gene_id \"chr4.g26074\"; transcript_id \"chr4.g26074.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68593683', '68594052', '.', '-', '.', 'gene_id \"chr4.g26075\"; transcript_id \"chr4.g26075.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68631248', '68631429', '.', '-', '.', 'gene_id \"chr4.g26088\"; transcript_id \"chr4.g26088.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68838707', '68838928', '.', '-', '.', 'gene_id \"chr4.g26198\"; transcript_id \"chr4.g26198.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69017338', '69017389', '.', '+', '.', 'gene_id \"chr4.g26316\"; transcript_id \"chr4.g26316.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69303009', '69303278', '.', '+', '.', 'gene_id \"chr4.g26347\"; transcript_id \"chr4.g26347.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69303009', '69303277', '.', '+', '.', 'gene_id \"chr4.g26347\"; transcript_id \"chr4.g26347.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69303711', '69304089', '.', '-', '.', 'gene_id \"chr4.g26348\"; transcript_id \"chr4.g26348.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69303049', '69303215', '.', '-', '.', 'gene_id \"chr4.g26350\"; transcript_id \"chr4.g26350.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '33915045', '33915105', '.', '+', '.', 'gene_id \"chr28.g14970\"; transcript_id \"chr28.g14970.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '33922959', '33923322', '.', '-', '.', 'gene_id \"chr28.g14974\"; transcript_id \"chr28.g14974.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69872393', '69872463', '.', '-', '.', 'gene_id \"chr4.g26469\"; transcript_id \"chr4.g26469.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34110161', '34110450', '.', '-', '.', 'gene_id \"chr28.g15087\"; transcript_id \"chr28.g15087.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34110521', '34110770', '.', '+', '.', 'gene_id \"chr28.g15088\"; transcript_id \"chr28.g15088.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34115164', '34115471', '.', '-', '.', 'gene_id \"chr28.g15092\"; transcript_id \"chr28.g15092.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34133106', '34133256', '.', '+', '.', 'gene_id \"chr28.g15124\"; transcript_id \"chr28.g15124.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34133322', '34133589', '.', '-', '.', 'gene_id \"chr28.g15125\"; transcript_id \"chr28.g15125.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34133595', '34133900', '.', '-', '.', 'gene_id \"chr28.g15126\"; transcript_id \"chr28.g15126.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70087686', '70088125', '.', '+', '.', 'gene_id \"chr4.g26577\"; transcript_id \"chr4.g26577.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70097799', '70098044', '.', '-', '.', 'gene_id \"chr4.g26585\"; transcript_id \"chr4.g26585.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70157215', '70157261', '.', '+', '.', 'gene_id \"chr4.g26610\"; transcript_id \"chr4.g26610.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70227672', '70227837', '.', '-', '.', 'gene_id \"chr4.g26665\"; transcript_id \"chr4.g26665.i3\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70311882', '70311978', '.', '-', '.', 'gene_id \"chr4.g26727\"; transcript_id \"chr4.g26727.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70311882', '70312127', '.', '-', '.', 'gene_id \"chr4.g26727\"; transcript_id \"chr4.g26727.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70561677', '70562437', '.', '+', '.', 'gene_id \"chr4.g26819\"; transcript_id \"chr4.g26819.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70599535', '70599777', '.', '+', '.', 'gene_id \"chr4.g26870\"; transcript_id \"chr4.g26870.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34497004', '34497577', '.', '-', '.', 'gene_id \"chr28.g15444\"; transcript_id \"chr28.g15444.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34508045', '34508218', '.', '-', '.', 'gene_id \"chr28.g15456\"; transcript_id \"chr28.g15456.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34508382', '34508581', '.', '-', '.', 'gene_id \"chr28.g15458\"; transcript_id \"chr28.g15458.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35029897', '35029944', '.', '+', '.', 'gene_id \"chr28.g15560\"; transcript_id \"chr28.g15560.i15\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35066829', '35067627', '.', '-', '.', 'gene_id \"chr28.g15630\"; transcript_id \"chr28.g15630.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35132265', '35132468', '.', '-', '.', 'gene_id \"chr28.g15666\"; transcript_id \"chr28.g15666.i2\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35197829', '35197865', '.', '-', '.', 'gene_id \"chr28.g15759\"; transcript_id \"chr28.g15759.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '71585454', '71585509', '.', '-', '.', 'gene_id \"chr4.g27223\"; transcript_id \"chr4.g27223.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35534606', '35534653', '.', '+', '.', 'gene_id \"chr28.g15845\"; transcript_id \"chr28.g15845.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35678670', '35678702', '.', '+', '.', 'gene_id \"chr28.g15884\"; transcript_id \"chr28.g15884.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35834476', '35834538', '.', '+', '.', 'gene_id \"chr28.g15919\"; transcript_id \"chr28.g15919.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '35931067', '35931185', '.', '+', '.', 'gene_id \"chr28.g15936\"; transcript_id \"chr28.g15936.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36115563', '36115727', '.', '+', '.', 'gene_id \"chr28.g16017\"; transcript_id \"chr28.g16017.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '71871244', '71871464', '.', '-', '.', 'gene_id \"chr4.g27451\"; transcript_id \"chr4.g27451.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36269992', '36270041', '.', '-', '.', 'gene_id \"chr28.g16126\"; transcript_id \"chr28.g16126.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36830057', '36831030', '.', '-', '.', 'gene_id \"chr28.g16261\"; transcript_id \"chr28.g16261.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72526897', '72527046', '.', '-', '.', 'gene_id \"chr4.g27745\"; transcript_id \"chr4.g27745.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36924849', '36925000', '.', '+', '.', 'gene_id \"chr28.g16312\"; transcript_id \"chr28.g16312.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36924991', '36925216', '.', '-', '.', 'gene_id \"chr28.g16313\"; transcript_id \"chr28.g16313.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72560178', '72560787', '.', '+', '.', 'gene_id \"chr4.g27778\"; transcript_id \"chr4.g27778.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72674942', '72675186', '.', '+', '.', 'gene_id \"chr4.g27872\"; transcript_id \"chr4.g27872.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72675700', '72676252', '.', '+', '.', 'gene_id \"chr4.g27874\"; transcript_id \"chr4.g27874.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72715457', '72715506', '.', '+', '.', 'gene_id \"chr4.g27884\"; transcript_id \"chr4.g27884.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73152749', '73152789', '.', '-', '.', 'gene_id \"chr4.g28066\"; transcript_id \"chr4.g28066.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73264329', '73264381', '.', '+', '.', 'gene_id \"chr4.g28116\"; transcript_id \"chr4.g28116.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73327826', '73328016', '.', '-', '.', 'gene_id \"chr4.g28134\"; transcript_id \"chr4.g28134.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73496237', '73496806', '.', '-', '.', 'gene_id \"chr4.g28294\"; transcript_id \"chr4.g28294.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73496704', '73496888', '.', '+', '.', 'gene_id \"chr4.g28295\"; transcript_id \"chr4.g28295.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '40378914', '40379029', '.', '-', '.', 'gene_id \"chr28.g16894\"; transcript_id \"chr28.g16894.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73621570', '73621845', '.', '-', '.', 'gene_id \"chr4.g28407\"; transcript_id \"chr4.g28407.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73621892', '73622096', '.', '-', '.', 'gene_id \"chr4.g28408\"; transcript_id \"chr4.g28408.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73663995', '73664155', '.', '-', '.', 'gene_id \"chr4.g28414\"; transcript_id \"chr4.g28414.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '74096503', '74096689', '.', '+', '.', 'gene_id \"chr4.g28696\"; transcript_id \"chr4.g28696.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '74097006', '74097310', '.', '-', '.', 'gene_id \"chr4.g28698\"; transcript_id \"chr4.g28698.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '74209771', '74210195', '.', '+', '.', 'gene_id \"chr4.g28744\"; transcript_id \"chr4.g28744.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '74553402', '74553523', '.', '-', '.', 'gene_id \"chr4.g28859\"; transcript_id \"chr4.g28859.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '41028115', '41028647', '.', '-', '.', 'gene_id \"chr28.g17341\"; transcript_id \"chr28.g17341.i1\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24497935', '24497974', '100', '-', '.', 'gene_id \"chr28.g17614\"; transcript_id \"blat.proc4587.chain_8357\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '16753896', '16754378', '100', '-', '.', 'gene_id \"chr28.g17761\"; transcript_id \"blat.proc18128.chain_4187\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6761017', '6761057', '100', '-', '.', 'gene_id \"chr28.g17813\"; transcript_id \"blat.proc18128.chain_3012\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '6829827', '6829907', '100', '-', '.', 'gene_id \"chr28.g17896\"; transcript_id \"blat.proc18128.chain_14458\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12099198', '12099245', '100', '+', '.', 'gene_id \"chr28.g17972\"; transcript_id \"blat.proc4587.chain_4856\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '2653894', '2653979', '100', '+', '.', 'gene_id \"chr28.g18054\"; transcript_id \"blat.proc28144.chain_22212\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '15620764', '15620807', '100', '+', '.', 'gene_id \"chr28.g18080\"; transcript_id \"blat.proc28144.chain_22439\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '116888', '116930', '100', '+', '.', 'gene_id \"chr28.g18102\"; transcript_id \"blat.proc18128.chain_22558\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14461263', '14461306', '100', '+', '.', 'gene_id \"chr28.g18189\"; transcript_id \"blat.proc18128.chain_5635\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '17538412', '17538624', '100', '+', '.', 'gene_id \"chr28.g18241\"; transcript_id \"blat.proc18128.chain_25623\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr28', 'PASA', 'exon', '17881830', '17881859', '100', '+', '.', 'gene_id \"chr28.g18303\"; transcript_id \"blat.proc4005.chain_16552\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '17073263', '17073309', '100', '+', '.', 'gene_id \"chr28.g18328\"; transcript_id \"blat.proc4005.chain_2533\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '22367434', '22367478', '100', '-', '.', 'gene_id \"chr28.g18444\"; transcript_id \"blat.proc28144.chain_4954\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11987505', '11987578', '100', '+', '.', 'gene_id \"chr28.g18556\"; transcript_id \"blat.proc28144.chain_5540\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '40734242', '40734274', '100', '+', '.', 'gene_id \"chr28.g18749\"; transcript_id \"blat.proc18128.chain_7653\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13657004', '13657036', '100', '-', '.', 'gene_id \"chr28.g18833\"; transcript_id \"blat.proc4005.chain_19146\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24124089', '24124202', '100', '+', '.', 'gene_id \"chr28.g19055\"; transcript_id \"blat.proc18128.chain_18596\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '5778013', '5778289', '100', '-', '.', 'gene_id \"chr28.g19179\"; transcript_id \"blat.proc18128.chain_7132\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '4652972', '4653016', '100', '-', '.', 'gene_id \"chr28.g19236\"; transcript_id \"blat.proc4587.chain_12589\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '36841480', '36841636', '100', '-', '.', 'gene_id \"chr28.g19290\"; transcript_id \"blat.proc4587.chain_1007\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '49688', '49742', '100', '+', '.', 'gene_id \"chr28.g19393\"; transcript_id \"blat.proc4587.chain_11943\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '1597698', '1597741', '100', '-', '.', 'gene_id \"chr28.g19533\"; transcript_id \"blat.proc18128.chain_9262\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '26196254', '26196291', '100', '-', '.', 'gene_id \"chr28.g19604\"; transcript_id \"blat.proc28144.chain_19785\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11827969', '11828000', '100', '-', '.', 'gene_id \"chr28.g19623\"; transcript_id \"blat.proc28144.chain_18693\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '5334979', '5335014', '100', '+', '.', 'gene_id \"chr28.g19867\"; transcript_id \"blat.proc4587.chain_13531\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '11315598', '11315683', '100', '+', '.', 'gene_id \"chr28.g19956\"; transcript_id \"blat.proc4587.chain_13522\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7649015', '7649057', '100', '+', '.', 'gene_id \"chr28.g20099\"; transcript_id \"blat.proc28144.chain_14400\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '31331750', '31331781', '100', '-', '.', 'gene_id \"chr28.g20184\"; transcript_id \"blat.proc18128.chain_24644\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '4150433', '4150495', '100', '+', '.', 'gene_id \"chr28.g20191\"; transcript_id \"blat.proc18128.chain_18439\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '33109062', '33109120', '100', '+', '.', 'gene_id \"chr28.g20193\"; transcript_id \"blat.proc4587.chain_4672\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '22466211', '22466490', '100', '-', '.', 'gene_id \"chr28.g20445\"; transcript_id \"blat.proc28144.chain_21118\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '30742865', '30742902', '100', '-', '.', 'gene_id \"chr28.g20680\"; transcript_id \"blat.proc4587.chain_13221\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '24445058', '24445090', '100', '+', '.', 'gene_id \"chr28.g20734\"; transcript_id \"blat.proc18128.chain_13505\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '8548610', '8548660', '100', '-', '.', 'gene_id \"chr28.g20744\"; transcript_id \"blat.proc18128.chain_8943\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '7679929', '7680059', '100', '-', '.', 'gene_id \"chr28.g20923\"; transcript_id \"blat.proc4587.chain_4668\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '29425517', '29425553', '100', '+', '.', 'gene_id \"chr28.g20935\"; transcript_id \"blat.proc28144.chain_15681\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '12608389', '12608426', '100', '-', '.', 'gene_id \"chr28.g20946\"; transcript_id \"blat.proc28144.chain_7146\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '3656573', '3656630', '100', '+', '.', 'gene_id \"chr28.g21116\"; transcript_id \"blat.proc4587.chain_9119\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '14862321', '14862409', '100', '+', '.', 'gene_id \"chr28.g21215\"; transcript_id \"blat.proc18128.chain_20303\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '13356952', '13357000', '100', '-', '.', 'gene_id \"chr28.g21411\"; transcript_id \"blat.proc4005.chain_2665\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '10222354', '10222420', '100', '+', '.', 'gene_id \"chr28.g21442\"; transcript_id \"blat.proc4587.chain_11807\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '34609514', '34609846', '100', '+', '.', 'gene_id \"chr28.g21564\"; transcript_id \"blat.proc18128.chain_7322\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '20721565', '20721660', '100', '+', '.', 'gene_id \"chr28.g21704\"; transcript_id \"blat.proc18128.chain_1233\";']\n",
      "negative:  ['chr28', 'PASA', 'exon', '1518607', '1518638', '100', '+', '.', 'gene_id \"chr28.g21729\"; transcript_id \"blat.proc18128.chain_4426\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75732787', '75732852', '.', '+', '.', 'gene_id \"chr4.g29384\"; transcript_id \"chr4.g29384.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75803172', '75803211', '.', '+', '.', 'gene_id \"chr4.g29415\"; transcript_id \"chr4.g29415.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75899721', '75899784', '.', '+', '.', 'gene_id \"chr4.g29445\"; transcript_id \"chr4.g29445.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75980970', '75981131', '.', '-', '.', 'gene_id \"chr4.g29473\"; transcript_id \"chr4.g29473.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '76587869', '76589765', '.', '+', '.', 'gene_id \"chr4.g29676\"; transcript_id \"chr4.g29676.i2\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '76592962', '76593102', '.', '-', '.', 'gene_id \"chr4.g29678\"; transcript_id \"chr4.g29678.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '77565368', '77565401', '.', '+', '.', 'gene_id \"chr4.g29733\"; transcript_id \"chr4.g29733.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '441963', '442003', '.', '+', '.', 'gene_id \"chr35.g226\"; transcript_id \"chr35.g226.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '77803275', '77803319', '.', '+', '.', 'gene_id \"chr4.g29757\"; transcript_id \"chr4.g29757.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '78314533', '78314562', '.', '-', '.', 'gene_id \"chr4.g29774\"; transcript_id \"chr4.g29774.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '587096', '587139', '.', '-', '.', 'gene_id \"chr35.g250\"; transcript_id \"chr35.g250.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '78543114', '78543166', '.', '+', '.', 'gene_id \"chr4.g29788\"; transcript_id \"chr4.g29788.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79134368', '79134525', '.', '+', '.', 'gene_id \"chr4.g29817\"; transcript_id \"chr4.g29817.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79630156', '79630193', '.', '-', '.', 'gene_id \"chr4.g29850\"; transcript_id \"chr4.g29850.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79783483', '79784680', '.', '+', '.', 'gene_id \"chr4.g29859\"; transcript_id \"chr4.g29859.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79980572', '79980725', '.', '+', '.', 'gene_id \"chr4.g29877\"; transcript_id \"chr4.g29877.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '80741429', '80741470', '.', '-', '.', 'gene_id \"chr4.g29929\"; transcript_id \"chr4.g29929.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '81895192', '81895468', '.', '-', '.', 'gene_id \"chr4.g29995\"; transcript_id \"chr4.g29995.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '928386', '928629', '.', '-', '.', 'gene_id \"chr35.g482\"; transcript_id \"chr35.g482.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '928600', '928792', '.', '+', '.', 'gene_id \"chr35.g483\"; transcript_id \"chr35.g483.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '1011721', '1011780', '.', '-', '.', 'gene_id \"chr35.g526\"; transcript_id \"chr35.g526.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '82639465', '82639736', '.', '+', '.', 'gene_id \"chr4.g30078\"; transcript_id \"chr4.g30078.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '1370047', '1370127', '.', '-', '.', 'gene_id \"chr35.g560\"; transcript_id \"chr35.g560.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '1471376', '1471405', '.', '-', '.', 'gene_id \"chr35.g575\"; transcript_id \"chr35.g575.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '83110332', '83110403', '.', '-', '.', 'gene_id \"chr4.g30119\"; transcript_id \"chr4.g30119.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2013137', '2013402', '.', '+', '.', 'gene_id \"chr35.g678\"; transcript_id \"chr35.g678.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '84804324', '84804374', '.', '+', '.', 'gene_id \"chr4.g30221\"; transcript_id \"chr4.g30221.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2023328', '2023616', '.', '-', '.', 'gene_id \"chr35.g698\"; transcript_id \"chr35.g698.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2063183', '2063355', '.', '+', '.', 'gene_id \"chr35.g751\"; transcript_id \"chr35.g751.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2063516', '2063807', '.', '+', '.', 'gene_id \"chr35.g752\"; transcript_id \"chr35.g752.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2099670', '2099885', '.', '+', '.', 'gene_id \"chr35.g777\"; transcript_id \"chr35.g777.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '86897341', '86897492', '.', '-', '.', 'gene_id \"chr4.g30504\"; transcript_id \"chr4.g30504.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '86914868', '86915406', '.', '-', '.', 'gene_id \"chr4.g30513\"; transcript_id \"chr4.g30513.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '86914483', '86914889', '.', '+', '.', 'gene_id \"chr4.g30514\"; transcript_id \"chr4.g30514.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2768492', '2768650', '.', '-', '.', 'gene_id \"chr35.g993\"; transcript_id \"chr35.g993.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87096792', '87098240', '.', '+', '.', 'gene_id \"chr4.g30687\"; transcript_id \"chr4.g30687.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87543799', '87544166', '.', '-', '.', 'gene_id \"chr4.g30745\"; transcript_id \"chr4.g30745.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87555077', '87555149', '.', '-', '.', 'gene_id \"chr4.g30751\"; transcript_id \"chr4.g30751.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87607630', '87607861', '.', '-', '.', 'gene_id \"chr4.g30789\"; transcript_id \"chr4.g30789.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87641754', '87641946', '.', '-', '.', 'gene_id \"chr4.g30811\"; transcript_id \"chr4.g30811.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3367033', '3367073', '.', '+', '.', 'gene_id \"chr35.g1284\"; transcript_id \"chr35.g1284.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3369175', '3369323', '.', '+', '.', 'gene_id \"chr35.g1290\"; transcript_id \"chr35.g1290.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3369338', '3369548', '.', '-', '.', 'gene_id \"chr35.g1291\"; transcript_id \"chr35.g1291.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3723650', '3723710', '.', '+', '.', 'gene_id \"chr35.g1578\"; transcript_id \"chr35.g1578.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '88407704', '88407745', '.', '-', '.', 'gene_id \"chr4.g31124\"; transcript_id \"chr4.g31124.i1\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '66971269', '66971351', '100', '-', '.', 'gene_id \"chr4.g31197\"; transcript_id \"blat.proc18128.chain_21126\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '817933', '817973', '100', '+', '.', 'gene_id \"chr4.g31232\"; transcript_id \"blat.proc28144.chain_3643\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60540781', '60540818', '100', '-', '.', 'gene_id \"chr4.g31239\"; transcript_id \"blat.proc18128.chain_1584\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '889819', '889863', '100', '-', '.', 'gene_id \"chr4.g31261\"; transcript_id \"blat.proc28144.chain_16902\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '27669024', '27669109', '100', '-', '.', 'gene_id \"chr4.g31277\"; transcript_id \"blat.proc18128.chain_6981\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '65668035', '65668065', '100', '+', '.', 'gene_id \"chr4.g31328\"; transcript_id \"blat.proc4005.chain_19415\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '60556161', '60556252', '100', '-', '.', 'gene_id \"chr4.g31370\"; transcript_id \"blat.proc4587.chain_2695\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38594246', '38594413', '100', '+', '.', 'gene_id \"chr4.g31480\"; transcript_id \"blat.proc18128.chain_22763\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20028610', '20028652', '100', '-', '.', 'gene_id \"chr4.g31506\"; transcript_id \"blat.proc28144.chain_7489\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79309872', '79309903', '100', '+', '.', 'gene_id \"chr4.g31586\"; transcript_id \"blat.proc18128.chain_7832\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62248404', '62248445', '100', '-', '.', 'gene_id \"chr4.g31659\"; transcript_id \"blat.proc18128.chain_22460\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1694662', '1694730', '100', '+', '.', 'gene_id \"chr4.g31767\"; transcript_id \"blat.proc28144.chain_1247\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59371855', '59371902', '100', '-', '.', 'gene_id \"chr4.g31776\"; transcript_id \"blat.proc28144.chain_14393\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35909761', '35909810', '100', '-', '.', 'gene_id \"chr4.g31856\"; transcript_id \"blat.proc28144.chain_1489\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '45515099', '45515139', '100', '+', '.', 'gene_id \"chr4.g31912\"; transcript_id \"blat.proc18128.chain_7404\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5946165', '5946200', '100', '+', '.', 'gene_id \"chr4.g31916\"; transcript_id \"blat.proc18128.chain_13575\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '2844710', '2844909', '100', '+', '.', 'gene_id \"chr4.g31931\"; transcript_id \"blat.proc18128.chain_25571\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '40539634', '40539665', '100', '-', '.', 'gene_id \"chr4.g31936\"; transcript_id \"blat.proc4587.chain_13166\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '53413741', '53413807', '100', '-', '.', 'gene_id \"chr4.g31946\"; transcript_id \"blat.proc4587.chain_520\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '45344674', '45344736', '100', '-', '.', 'gene_id \"chr4.g31951\"; transcript_id \"blat.proc28144.chain_15966\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52204857', '52204890', '100', '+', '.', 'gene_id \"chr4.g32088\"; transcript_id \"blat.proc18128.chain_777\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55790032', '55790067', '100', '+', '.', 'gene_id \"chr4.g32097\"; transcript_id \"blat.proc18128.chain_8642\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '65956031', '65956070', '100', '+', '.', 'gene_id \"chr4.g32099\"; transcript_id \"blat.proc18128.chain_16104\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87678043', '87678115', '100', '-', '.', 'gene_id \"chr4.g32137\"; transcript_id \"blat.proc18128.chain_7076\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '63867408', '63867457', '100', '+', '.', 'gene_id \"chr4.g32148\"; transcript_id \"blat.proc18128.chain_3152\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19292173', '19292202', '100', '+', '.', 'gene_id \"chr4.g32158\"; transcript_id \"blat.proc28144.chain_22104\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51485270', '51485422', '100', '+', '.', 'gene_id \"chr4.g32214\"; transcript_id \"blat.proc4587.chain_12138\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73972525', '73972561', '100', '-', '.', 'gene_id \"chr4.g32247\"; transcript_id \"blat.proc18128.chain_2309\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '66257569', '66258081', '100', '-', '.', 'gene_id \"chr4.g32331\"; transcript_id \"blat.proc18128.chain_8419\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '22069490', '22069529', '100', '+', '.', 'gene_id \"chr4.g32367\"; transcript_id \"blat.proc18128.chain_1375\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '4890339', '4890387', '100', '+', '.', 'gene_id \"chr4.g32406\"; transcript_id \"blat.proc4005.chain_2794\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '10153963', '10153992', '100', '-', '.', 'gene_id \"chr4.g32468\"; transcript_id \"blat.proc18128.chain_9052\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '39761324', '39761393', '100', '+', '.', 'gene_id \"chr4.g32470\"; transcript_id \"blat.proc18128.chain_17783\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25666793', '25666828', '100', '+', '.', 'gene_id \"chr4.g32485\"; transcript_id \"blat.proc28144.chain_16201\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '78170927', '78170962', '100', '+', '.', 'gene_id \"chr4.g32487\"; transcript_id \"blat.proc18128.chain_2336\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1268366', '1268397', '100', '+', '.', 'gene_id \"chr4.g32525\"; transcript_id \"blat.proc4005.chain_15236\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '391325', '391372', '100', '+', '.', 'gene_id \"chr4.g32578\"; transcript_id \"blat.proc28144.chain_7183\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56037936', '56037967', '100', '-', '.', 'gene_id \"chr4.g32606\"; transcript_id \"blat.proc4005.chain_14846\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18008670', '18008709', '100', '+', '.', 'gene_id \"chr4.g32618\"; transcript_id \"blat.proc4587.chain_10225\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '44247354', '44247401', '100', '-', '.', 'gene_id \"chr4.g32753\"; transcript_id \"blat.proc4005.chain_10740\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56703393', '56703462', '100', '-', '.', 'gene_id \"chr4.g32765\"; transcript_id \"blat.proc4005.chain_9371\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '2887866', '2888065', '100', '-', '.', 'gene_id \"chr4.g32902\"; transcript_id \"blat.proc18128.chain_25576\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '25847977', '25848019', '100', '-', '.', 'gene_id \"chr4.g32947\"; transcript_id \"blat.proc18128.chain_6123\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79102735', '79103217', '100', '-', '.', 'gene_id \"chr4.g32989\"; transcript_id \"blat.proc18128.chain_4184\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75888187', '75888245', '100', '-', '.', 'gene_id \"chr4.g33093\"; transcript_id \"blat.proc28144.chain_15160\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '81759015', '81759086', '100', '-', '.', 'gene_id \"chr4.g33182\"; transcript_id \"blat.proc28144.chain_1254\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6098070', '6098104', '100', '-', '.', 'gene_id \"chr4.g33196\"; transcript_id \"blat.proc18128.chain_16698\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '47813404', '47813472', '100', '+', '.', 'gene_id \"chr4.g33218\"; transcript_id \"blat.proc18128.chain_10185\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '33347564', '33347606', '100', '+', '.', 'gene_id \"chr4.g33240\"; transcript_id \"blat.proc4005.chain_16845\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '14792201', '14792257', '100', '+', '.', 'gene_id \"chr4.g33268\"; transcript_id \"blat.proc18128.chain_13911\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70367578', '70367609', '100', '+', '.', 'gene_id \"chr4.g33359\"; transcript_id \"blat.proc4005.chain_796\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51160608', '51160687', '100', '-', '.', 'gene_id \"chr4.g33410\"; transcript_id \"blat.proc4587.chain_608\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55068686', '55068858', '100', '+', '.', 'gene_id \"chr4.g33424\"; transcript_id \"blat.proc28144.chain_20050\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '6574602', '6574633', '100', '+', '.', 'gene_id \"chr4.g33488\"; transcript_id \"blat.proc4587.chain_2658\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '20274482', '20274738', '100', '-', '.', 'gene_id \"chr4.g33546\"; transcript_id \"blat.proc18128.chain_21480\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19080690', '19080746', '100', '+', '.', 'gene_id \"chr4.g33570\"; transcript_id \"blat.proc4005.chain_1775\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16367556', '16367610', '100', '+', '.', 'gene_id \"chr4.g33599\"; transcript_id \"blat.proc28144.chain_22300\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '37484692', '37484805', '100', '-', '.', 'gene_id \"chr4.g33673\"; transcript_id \"blat.proc4005.chain_19074\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17023729', '17023788', '100', '-', '.', 'gene_id \"chr4.g33721\"; transcript_id \"blat.proc4005.chain_6649\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69121929', '69122049', '100', '-', '.', 'gene_id \"chr4.g33865\"; transcript_id \"blat.proc4587.chain_3112\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '9318648', '9318682', '100', '-', '.', 'gene_id \"chr4.g34019\"; transcript_id \"blat.proc18128.chain_2343\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '813129', '813351', '100', '-', '.', 'gene_id \"chr4.g34081\"; transcript_id \"blat.proc18128.chain_6876\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '62116266', '62116310', '100', '-', '.', 'gene_id \"chr4.g34185\"; transcript_id \"blat.proc28144.chain_8707\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57992425', '57992463', '100', '-', '.', 'gene_id \"chr4.g34190\"; transcript_id \"blat.proc18128.chain_19023\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '34673946', '34673994', '100', '+', '.', 'gene_id \"chr4.g34242\"; transcript_id \"blat.proc4005.chain_20739\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24549007', '24549091', '100', '-', '.', 'gene_id \"chr4.g34325\"; transcript_id \"blat.proc18128.chain_1986\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5296361', '5296429', '100', '+', '.', 'gene_id \"chr4.g34368\"; transcript_id \"blat.proc18128.chain_4425\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54589051', '54589166', '100', '+', '.', 'gene_id \"chr4.g34489\"; transcript_id \"blat.proc4005.chain_22866\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '59420242', '59420354', '100', '+', '.', 'gene_id \"chr4.g34564\"; transcript_id \"blat.proc18128.chain_14377\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19915680', '19915738', '100', '-', '.', 'gene_id \"chr4.g34575\"; transcript_id \"blat.proc18128.chain_6632\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '85134595', '85134667', '100', '+', '.', 'gene_id \"chr4.g34664\"; transcript_id \"blat.proc18128.chain_19147\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '82806125', '82806441', '100', '-', '.', 'gene_id \"chr4.g34701\"; transcript_id \"blat.proc18128.chain_8491\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56915667', '56915718', '100', '+', '.', 'gene_id \"chr4.g34822\"; transcript_id \"blat.proc18128.chain_26825\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '82252901', '82252962', '100', '-', '.', 'gene_id \"chr4.g34845\"; transcript_id \"blat.proc28144.chain_6899\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35827349', '35827382', '100', '+', '.', 'gene_id \"chr4.g34862\"; transcript_id \"blat.proc18128.chain_16484\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38424704', '38424754', '100', '-', '.', 'gene_id \"chr4.g34867\"; transcript_id \"blat.proc28144.chain_22369\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '23736494', '23736539', '100', '+', '.', 'gene_id \"chr4.g34945\"; transcript_id \"blat.proc4005.chain_18590\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '11196242', '11196315', '100', '+', '.', 'gene_id \"chr4.g35024\"; transcript_id \"blat.proc4587.chain_3158\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1267200', '1267246', '100', '-', '.', 'gene_id \"chr4.g35060\"; transcript_id \"blat.proc28144.chain_5555\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '27471928', '27472028', '100', '+', '.', 'gene_id \"chr4.g35064\"; transcript_id \"blat.proc18128.chain_16176\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75587281', '75587316', '100', '+', '.', 'gene_id \"chr4.g35203\"; transcript_id \"blat.proc4587.chain_10319\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '76568866', '76568902', '100', '-', '.', 'gene_id \"chr4.g35219\"; transcript_id \"blat.proc18128.chain_26082\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '80811004', '80811038', '100', '-', '.', 'gene_id \"chr4.g35225\"; transcript_id \"blat.proc18128.chain_20940\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '26287721', '26287756', '100', '-', '.', 'gene_id \"chr4.g35258\"; transcript_id \"blat.proc4005.chain_12464\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '72789439', '72789476', '100', '-', '.', 'gene_id \"chr4.g35304\"; transcript_id \"blat.proc18128.chain_1885\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '17857383', '17857651', '100', '+', '.', 'gene_id \"chr4.g35356\"; transcript_id \"blat.proc4587.chain_13359\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54472028', '54472099', '100', '-', '.', 'gene_id \"chr4.g35431\"; transcript_id \"blat.proc18128.chain_14872\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79611367', '79611400', '100', '-', '.', 'gene_id \"chr4.g35443\"; transcript_id \"blat.proc4005.chain_365\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '68657717', '68657763', '100', '-', '.', 'gene_id \"chr4.g35627\"; transcript_id \"blat.proc28144.chain_18541\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '467169', '467209', '100', '+', '.', 'gene_id \"chr4.g35776\"; transcript_id \"blat.proc28144.chain_11754\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '2737535', '2737581', '100', '+', '.', 'gene_id \"chr4.g35779\"; transcript_id \"blat.proc28144.chain_20043\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32040485', '32040525', '100', '+', '.', 'gene_id \"chr4.g35847\"; transcript_id \"blat.proc4005.chain_21178\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29598295', '29598343', '100', '-', '.', 'gene_id \"chr4.g36098\"; transcript_id \"blat.proc4587.chain_9942\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '7662193', '7662222', '100', '+', '.', 'gene_id \"chr4.g36101\"; transcript_id \"blat.proc4005.chain_4642\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '15213975', '15214006', '100', '+', '.', 'gene_id \"chr4.g36168\"; transcript_id \"blat.proc28144.chain_3220\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '12664336', '12664402', '100', '-', '.', 'gene_id \"chr4.g36175\"; transcript_id \"blat.proc4005.chain_19658\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '75555530', '75555597', '100', '+', '.', 'gene_id \"chr4.g36383\"; transcript_id \"blat.proc4005.chain_9360\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '67026417', '67026455', '100', '-', '.', 'gene_id \"chr4.g36569\"; transcript_id \"blat.proc4587.chain_4267\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55587692', '55587727', '100', '+', '.', 'gene_id \"chr4.g36597\"; transcript_id \"blat.proc28144.chain_18525\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '46306235', '46306283', '100', '+', '.', 'gene_id \"chr4.g36750\"; transcript_id \"blat.proc4005.chain_13862\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '70562901', '70563592', '100', '+', '.', 'gene_id \"chr4.g36840\"; transcript_id \"blat.proc18128.chain_8405\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '19740307', '19740538', '100', '+', '.', 'gene_id \"chr4.g36870\"; transcript_id \"blat.proc4587.chain_9989\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '49474354', '49474459', '100', '-', '.', 'gene_id \"chr4.g36911\"; transcript_id \"blat.proc18128.chain_15217\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '5627556', '5627712', '100', '+', '.', 'gene_id \"chr4.g37036\"; transcript_id \"blat.proc28144.chain_10896\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54572838', '54572910', '100', '-', '.', 'gene_id \"chr4.g37072\"; transcript_id \"blat.proc18128.chain_22549\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '13082135', '13082188', '100', '-', '.', 'gene_id \"chr4.g37078\"; transcript_id \"blat.proc28144.chain_3030\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '87769892', '87769972', '100', '+', '.', 'gene_id \"chr4.g37142\"; transcript_id \"blat.proc4005.chain_4655\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '21298787', '21298851', '100', '-', '.', 'gene_id \"chr4.g37165\"; transcript_id \"blat.proc18128.chain_9712\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '29363499', '29363565', '100', '+', '.', 'gene_id \"chr4.g37229\"; transcript_id \"blat.proc28144.chain_15960\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '42217024', '42217481', '100', '-', '.', 'gene_id \"chr4.g37238\"; transcript_id \"blat.proc4587.chain_12899\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '54989355', '54989506', '100', '+', '.', 'gene_id \"chr4.g37261\"; transcript_id \"blat.proc4587.chain_8253\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '57873306', '57873557', '100', '+', '.', 'gene_id \"chr4.g37340\"; transcript_id \"blat.proc28144.chain_11561\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '16115022', '16115067', '100', '+', '.', 'gene_id \"chr4.g37376\"; transcript_id \"blat.proc4587.chain_6256\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '28448554', '28448590', '100', '+', '.', 'gene_id \"chr4.g37460\"; transcript_id \"blat.proc18128.chain_9725\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32317037', '32317074', '100', '-', '.', 'gene_id \"chr4.g37494\"; transcript_id \"blat.proc18128.chain_3800\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '56965955', '56965987', '100', '+', '.', 'gene_id \"chr4.g37517\"; transcript_id \"blat.proc4005.chain_2902\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18966248', '18966333', '100', '+', '.', 'gene_id \"chr4.g37580\"; transcript_id \"blat.proc4005.chain_10357\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '18842023', '18842057', '100', '+', '.', 'gene_id \"chr4.g37635\"; transcript_id \"blat.proc4587.chain_10226\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '32980063', '32980164', '100', '+', '.', 'gene_id \"chr4.g37760\"; transcript_id \"blat.proc18128.chain_6914\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52782214', '52782247', '100', '+', '.', 'gene_id \"chr4.g37773\"; transcript_id \"blat.proc4005.chain_12677\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '38179738', '38179769', '100', '-', '.', 'gene_id \"chr4.g37811\"; transcript_id \"blat.proc4005.chain_9336\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '69706568', '69706631', '100', '-', '.', 'gene_id \"chr4.g37865\"; transcript_id \"blat.proc18128.chain_17066\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '48318034', '48318186', '100', '+', '.', 'gene_id \"chr4.g37871\"; transcript_id \"blat.proc4587.chain_7344\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '10475099', '10475132', '100', '-', '.', 'gene_id \"chr4.g37918\"; transcript_id \"blat.proc18128.chain_20370\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '51912751', '51912840', '100', '+', '.', 'gene_id \"chr4.g38220\"; transcript_id \"blat.proc4587.chain_12622\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55420129', '55420205', '100', '+', '.', 'gene_id \"chr4.g38264\"; transcript_id \"blat.proc28144.chain_15002\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79691606', '79691644', '100', '-', '.', 'gene_id \"chr4.g38381\"; transcript_id \"blat.proc18128.chain_12281\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '24269475', '24269520', '100', '-', '.', 'gene_id \"chr4.g38438\"; transcript_id \"blat.proc18128.chain_2474\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '73630752', '73630802', '100', '+', '.', 'gene_id \"chr4.g38498\"; transcript_id \"blat.proc18128.chain_8212\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '55696439', '55696475', '100', '-', '.', 'gene_id \"chr4.g38624\"; transcript_id \"blat.proc4587.chain_8231\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '35919484', '35919625', '100', '-', '.', 'gene_id \"chr4.g38638\"; transcript_id \"blat.proc18128.chain_14053\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '61529065', '61529128', '100', '-', '.', 'gene_id \"chr4.g38665\"; transcript_id \"blat.proc28144.chain_6340\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52892517', '52892559', '100', '+', '.', 'gene_id \"chr4.g38828\"; transcript_id \"blat.proc4005.chain_13682\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '30871413', '30871446', '100', '+', '.', 'gene_id \"chr4.g38960\"; transcript_id \"blat.proc4005.chain_20096\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '1372422', '1372698', '100', '-', '.', 'gene_id \"chr4.g38967\"; transcript_id \"blat.proc4005.chain_3189\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '79328202', '79328261', '100', '+', '.', 'gene_id \"chr4.g39048\"; transcript_id \"blat.proc18128.chain_15537\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '52618416', '52618457', '100', '-', '.', 'gene_id \"chr4.g39162\"; transcript_id \"blat.proc28144.chain_5377\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '58949705', '58949748', '100', '-', '.', 'gene_id \"chr4.g39172\"; transcript_id \"blat.proc28144.chain_10340\";']\n",
      "negative:  ['chr4', 'PASA', 'exon', '50176463', '50176521', '100', '-', '.', 'gene_id \"chr4.g39276\"; transcript_id \"blat.proc4005.chain_17262\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3932856', '3933072', '.', '-', '.', 'gene_id \"chr35.g1638\"; transcript_id \"chr35.g1638.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '4189833', '4190125', '.', '+', '.', 'gene_id \"chr35.g1760\"; transcript_id \"chr35.g1760.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '4237113', '4237162', '.', '-', '.', 'gene_id \"chr35.g1798\"; transcript_id \"chr35.g1798.i1\";']\n",
      "negative:  ['scaffold_1512', 'PASA', 'exon', '25023', '25178', '100', '+', '.', 'gene_id \"scaffold_1512.g4\"; transcript_id \"blat.proc4587.chain_6480\";']\n",
      "negative:  ['scaffold_1512', 'PASA', 'exon', '20081', '20114', '100', '+', '.', 'gene_id \"scaffold_1512.g6\"; transcript_id \"blat.proc4005.chain_4982\";']\n",
      "negative:  ['scaffold_553', 'PASA', 'exon', '140', '371', '100', '+', '.', 'gene_id \"scaffold_553.g2\"; transcript_id \"blat.proc4587.chain_1749\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5217092', '5217426', '.', '-', '.', 'gene_id \"chr35.g2130\"; transcript_id \"chr35.g2130.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5350434', '5350500', '.', '-', '.', 'gene_id \"chr35.g2272\"; transcript_id \"chr35.g2272.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5701713', '5702024', '.', '-', '.', 'gene_id \"chr35.g2404\"; transcript_id \"chr35.g2404.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5810577', '5810766', '.', '+', '.', 'gene_id \"chr35.g2482\"; transcript_id \"chr35.g2482.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5824493', '5824685', '.', '-', '.', 'gene_id \"chr35.g2507\"; transcript_id \"chr35.g2507.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5825095', '5825491', '.', '-', '.', 'gene_id \"chr35.g2509\"; transcript_id \"chr35.g2509.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6470117', '6470390', '.', '+', '.', 'gene_id \"chr35.g2702\"; transcript_id \"chr35.g2702.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6618874', '6619172', '.', '-', '.', 'gene_id \"chr35.g2751\"; transcript_id \"chr35.g2751.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6694603', '6694634', '.', '-', '.', 'gene_id \"chr35.g2783\"; transcript_id \"chr35.g2783.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6852175', '6852514', '.', '+', '.', 'gene_id \"chr35.g2833\"; transcript_id \"chr35.g2833.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '7055016', '7055171', '.', '+', '.', 'gene_id \"chr35.g2971\"; transcript_id \"chr35.g2971.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '7734033', '7734383', '.', '-', '.', 'gene_id \"chr35.g3356\"; transcript_id \"chr35.g3356.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '8216197', '8216229', '.', '-', '.', 'gene_id \"chr35.g3621\"; transcript_id \"chr35.g3621.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '9177267', '9177313', '.', '+', '.', 'gene_id \"chr35.g3702\"; transcript_id \"chr35.g3702.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '9865006', '9865035', '.', '-', '.', 'gene_id \"chr35.g3750\"; transcript_id \"chr35.g3750.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10092847', '10093079', '.', '+', '.', 'gene_id \"chr35.g3773\"; transcript_id \"chr35.g3773.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10211080', '10211111', '.', '-', '.', 'gene_id \"chr35.g3803\"; transcript_id \"chr35.g3803.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10257926', '10258216', '.', '+', '.', 'gene_id \"chr35.g3843\"; transcript_id \"chr35.g3843.i1\";']\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "negative:  ['chr35', 'PASA', 'exon', '10423646', '10423994', '.', '+', '.', 'gene_id \"chr35.g4010\"; transcript_id \"chr35.g4010.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10419359', '10419565', '.', '+', '.', 'gene_id \"chr35.g4011\"; transcript_id \"chr35.g4011.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10409357', '10409509', '.', '-', '.', 'gene_id \"chr35.g4012\"; transcript_id \"chr35.g4012.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10414702', '10415069', '.', '-', '.', 'gene_id \"chr35.g4017\"; transcript_id \"chr35.g4017.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10415050', '10415203', '.', '+', '.', 'gene_id \"chr35.g4018\"; transcript_id \"chr35.g4018.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10415742', '10416108', '.', '-', '.', 'gene_id \"chr35.g4019\"; transcript_id \"chr35.g4019.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10416822', '10417206', '.', '-', '.', 'gene_id \"chr35.g4022\"; transcript_id \"chr35.g4022.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10511731', '10511992', '.', '-', '.', 'gene_id \"chr35.g4066\"; transcript_id \"chr35.g4066.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '11131659', '11131851', '.', '+', '.', 'gene_id \"chr35.g4473\"; transcript_id \"chr35.g4473.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '11135789', '11135829', '.', '+', '.', 'gene_id \"chr35.g4478\"; transcript_id \"chr35.g4478.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '11490897', '11490941', '.', '-', '.', 'gene_id \"chr35.g4535\"; transcript_id \"chr35.g4535.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '12747328', '12747492', '.', '+', '.', 'gene_id \"chr35.g5038\"; transcript_id \"chr35.g5038.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '12747500', '12747655', '.', '+', '.', 'gene_id \"chr35.g5039\"; transcript_id \"chr35.g5039.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '12765526', '12765731', '.', '+', '.', 'gene_id \"chr35.g5063\"; transcript_id \"chr35.g5063.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '12831362', '12832008', '.', '-', '.', 'gene_id \"chr35.g5147\"; transcript_id \"chr35.g5147.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '13066078', '13066410', '.', '-', '.', 'gene_id \"chr35.g5290\"; transcript_id \"chr35.g5290.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '13136135', '13136177', '.', '-', '.', 'gene_id \"chr35.g5343\"; transcript_id \"chr35.g5343.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14189524', '14189897', '.', '+', '.', 'gene_id \"chr35.g5510\"; transcript_id \"chr35.g5510.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14401843', '14401906', '.', '+', '.', 'gene_id \"chr35.g5552\"; transcript_id \"chr35.g5552.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14430570', '14430810', '.', '+', '.', 'gene_id \"chr35.g5583\"; transcript_id \"chr35.g5583.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14437282', '14437324', '.', '+', '.', 'gene_id \"chr35.g5592\"; transcript_id \"chr35.g5592.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14477590', '14477782', '.', '-', '.', 'gene_id \"chr35.g5636\"; transcript_id \"chr35.g5636.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14477776', '14477928', '.', '+', '.', 'gene_id \"chr35.g5637\"; transcript_id \"chr35.g5637.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14608847', '14608908', '.', '+', '.', 'gene_id \"chr35.g5715\"; transcript_id \"chr35.g5715.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14629858', '14629895', '.', '+', '.', 'gene_id \"chr35.g5731\"; transcript_id \"chr35.g5731.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14771406', '14771451', '.', '+', '.', 'gene_id \"chr35.g5849\"; transcript_id \"chr35.g5849.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14977966', '14978159', '.', '+', '.', 'gene_id \"chr35.g5879\"; transcript_id \"chr35.g5879.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '15196417', '15196787', '.', '-', '.', 'gene_id \"chr35.g5893\"; transcript_id \"chr35.g5893.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '15370515', '15370921', '.', '-', '.', 'gene_id \"chr35.g5938\"; transcript_id \"chr35.g5938.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '15370591', '15370649', '.', '-', '.', 'gene_id \"chr35.g5938\"; transcript_id \"chr35.g5938.i3\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '15870104', '15870159', '.', '-', '.', 'gene_id \"chr35.g6419\"; transcript_id \"chr35.g6419.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '16099809', '16099999', '.', '-', '.', 'gene_id \"chr35.g6460\"; transcript_id \"chr35.g6460.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '16099809', '16099990', '.', '-', '.', 'gene_id \"chr35.g6460\"; transcript_id \"chr35.g6460.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '16296120', '16296189', '.', '-', '.', 'gene_id \"chr35.g6556\"; transcript_id \"chr35.g6556.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17123316', '17123742', '.', '+', '.', 'gene_id \"chr35.g7055\"; transcript_id \"chr35.g7055.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17136113', '17136676', '.', '-', '.', 'gene_id \"chr35.g7079\"; transcript_id \"chr35.g7079.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17138161', '17138441', '.', '-', '.', 'gene_id \"chr35.g7082\"; transcript_id \"chr35.g7082.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17137709', '17138171', '.', '+', '.', 'gene_id \"chr35.g7083\"; transcript_id \"chr35.g7083.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17499455', '17499485', '.', '-', '.', 'gene_id \"chr35.g7120\"; transcript_id \"chr35.g7120.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18253506', '18253760', '.', '+', '.', 'gene_id \"chr35.g7237\"; transcript_id \"chr35.g7237.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18519303', '18519373', '.', '+', '.', 'gene_id \"chr35.g7291\"; transcript_id \"chr35.g7291.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18717557', '18718438', '.', '+', '.', 'gene_id \"chr35.g7392\"; transcript_id \"chr35.g7392.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18718449', '18718641', '.', '+', '.', 'gene_id \"chr35.g7393\"; transcript_id \"chr35.g7393.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18739773', '18739954', '.', '+', '.', 'gene_id \"chr35.g7420\"; transcript_id \"chr35.g7420.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19003983', '19005689', '.', '+', '.', 'gene_id \"chr35.g7508\"; transcript_id \"chr35.g7508.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19003983', '19005231', '.', '+', '.', 'gene_id \"chr35.g7508\"; transcript_id \"chr35.g7508.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19056394', '19056669', '.', '+', '.', 'gene_id \"chr35.g7523\"; transcript_id \"chr35.g7523.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19518093', '19518252', '.', '-', '.', 'gene_id \"chr35.g7675\"; transcript_id \"chr35.g7675.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19522769', '19522882', '.', '-', '.', 'gene_id \"chr35.g7678\"; transcript_id \"chr35.g7678.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19553865', '19554214', '.', '-', '.', 'gene_id \"chr35.g7690\"; transcript_id \"chr35.g7690.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '19803099', '19803135', '.', '+', '.', 'gene_id \"chr35.g7776\"; transcript_id \"chr35.g7776.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '20517660', '20517705', '.', '-', '.', 'gene_id \"chr35.g7977\"; transcript_id \"chr35.g7977.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '21279062', '21279116', '.', '+', '.', 'gene_id \"chr35.g8089\"; transcript_id \"chr35.g8089.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22068543', '22068639', '.', '+', '.', 'gene_id \"chr35.g8147\"; transcript_id \"chr35.g8147.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22850056', '22850095', '.', '-', '.', 'gene_id \"chr35.g8625\"; transcript_id \"chr35.g8625.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23030468', '23030618', '.', '-', '.', 'gene_id \"chr35.g8726\"; transcript_id \"chr35.g8726.i3\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23145386', '23145791', '.', '-', '.', 'gene_id \"chr35.g8780\"; transcript_id \"chr35.g8780.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23295531', '23296003', '.', '+', '.', 'gene_id \"chr35.g8857\"; transcript_id \"chr35.g8857.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23527184', '23527470', '.', '-', '.', 'gene_id \"chr35.g8955\"; transcript_id \"chr35.g8955.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23754663', '23754712', '.', '-', '.', 'gene_id \"chr35.g9088\"; transcript_id \"chr35.g9088.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23944181', '23944363', '.', '-', '.', 'gene_id \"chr35.g9254\"; transcript_id \"chr35.g9254.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24114432', '24114613', '.', '-', '.', 'gene_id \"chr35.g9408\"; transcript_id \"chr35.g9408.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24176888', '24177016', '.', '-', '.', 'gene_id \"chr35.g9469\"; transcript_id \"chr35.g9469.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24199838', '24199883', '.', '+', '.', 'gene_id \"chr35.g9474\"; transcript_id \"chr35.g9474.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24528810', '24528898', '.', '-', '.', 'gene_id \"chr35.g9625\"; transcript_id \"chr35.g9625.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24711100', '24711872', '.', '+', '.', 'gene_id \"chr35.g9662\"; transcript_id \"chr35.g9662.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24720205', '24720501', '.', '+', '.', 'gene_id \"chr35.g9679\"; transcript_id \"chr35.g9679.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24720679', '24720847', '.', '+', '.', 'gene_id \"chr35.g9680\"; transcript_id \"chr35.g9680.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '24912324', '24912356', '.', '-', '.', 'gene_id \"chr35.g9779\"; transcript_id \"chr35.g9779.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25112849', '25113107', '.', '-', '.', 'gene_id \"chr35.g9903\"; transcript_id \"chr35.g9903.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25342101', '25342628', '.', '-', '.', 'gene_id \"chr35.g10033\"; transcript_id \"chr35.g10033.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25432612', '25432682', '.', '-', '.', 'gene_id \"chr35.g10136\"; transcript_id \"chr35.g10136.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25471901', '25472238', '.', '-', '.', 'gene_id \"chr35.g10155\"; transcript_id \"chr35.g10155.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25498791', '25498960', '.', '-', '.', 'gene_id \"chr35.g10179\"; transcript_id \"chr35.g10179.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25500032', '25500779', '.', '-', '.', 'gene_id \"chr35.g10182\"; transcript_id \"chr35.g10182.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25821468', '25821984', '.', '-', '.', 'gene_id \"chr35.g10250\"; transcript_id \"chr35.g10250.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26294089', '26294808', '.', '-', '.', 'gene_id \"chr35.g10454\"; transcript_id \"chr35.g10454.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26288055', '26288237', '.', '+', '.', 'gene_id \"chr35.g10455\"; transcript_id \"chr35.g10455.i2\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26356215', '26356634', '.', '-', '.', 'gene_id \"chr35.g10485\"; transcript_id \"chr35.g10485.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26359217', '26359579', '.', '+', '.', 'gene_id \"chr35.g10487\"; transcript_id \"chr35.g10487.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26359634', '26360218', '.', '-', '.', 'gene_id \"chr35.g10488\"; transcript_id \"chr35.g10488.i1\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25296124', '25296187', '100', '-', '.', 'gene_id \"chr35.g10747\"; transcript_id \"blat.proc18128.chain_13494\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14296342', '14296373', '100', '-', '.', 'gene_id \"chr35.g10810\"; transcript_id \"blat.proc4587.chain_7536\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '11064307', '11064348', '100', '-', '.', 'gene_id \"chr35.g10878\"; transcript_id \"blat.proc4005.chain_16383\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10381278', '10381311', '100', '-', '.', 'gene_id \"chr35.g10892\"; transcript_id \"blat.proc18128.chain_19644\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '2212690', '2213076', '100', '+', '.', 'gene_id \"chr35.g11043\"; transcript_id \"blat.proc18128.chain_22555\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '17263053', '17263142', '100', '+', '.', 'gene_id \"chr35.g11106\"; transcript_id \"blat.proc18128.chain_5911\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23529114', '23529221', '100', '+', '.', 'gene_id \"chr35.g11115\"; transcript_id \"blat.proc18128.chain_6127\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '15974168', '15974207', '100', '-', '.', 'gene_id \"chr35.g11161\"; transcript_id \"blat.proc4005.chain_20212\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22411234', '22411297', '100', '-', '.', 'gene_id \"chr35.g11374\"; transcript_id \"blat.proc28144.chain_5995\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3156479', '3156519', '100', '+', '.', 'gene_id \"chr35.g11447\"; transcript_id \"blat.proc28144.chain_8703\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '9434546', '9434602', '100', '-', '.', 'gene_id \"chr35.g11481\"; transcript_id \"blat.proc4587.chain_3899\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '7615433', '7615465', '100', '+', '.', 'gene_id \"chr35.g11502\"; transcript_id \"blat.proc4005.chain_11160\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6363599', '6363628', '100', '-', '.', 'gene_id \"chr35.g11565\"; transcript_id \"blat.proc28144.chain_18810\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23391867', '23391907', '100', '-', '.', 'gene_id \"chr35.g11579\"; transcript_id \"blat.proc4005.chain_20703\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '16901003', '16901049', '100', '+', '.', 'gene_id \"chr35.g11770\"; transcript_id \"blat.proc4587.chain_8370\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '20579110', '20579174', '100', '+', '.', 'gene_id \"chr35.g11794\"; transcript_id \"blat.proc28144.chain_4456\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '16155851', '16155911', '100', '+', '.', 'gene_id \"chr35.g11810\"; transcript_id \"blat.proc4587.chain_535\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '26321647', '26321681', '100', '-', '.', 'gene_id \"chr35.g11823\"; transcript_id \"blat.proc4587.chain_12006\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22396331', '22396405', '100', '+', '.', 'gene_id \"chr35.g11831\"; transcript_id \"blat.proc18128.chain_17174\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '14937767', '14937813', '100', '-', '.', 'gene_id \"chr35.g11972\"; transcript_id \"blat.proc28144.chain_18850\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '20378903', '20379154', '100', '-', '.', 'gene_id \"chr35.g12035\"; transcript_id \"blat.proc4587.chain_949\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '21875281', '21875532', '100', '-', '.', 'gene_id \"chr35.g12079\"; transcript_id \"blat.proc4587.chain_950\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23453606', '23453639', '100', '-', '.', 'gene_id \"chr35.g12197\"; transcript_id \"blat.proc18128.chain_16074\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '8787208', '8787243', '100', '+', '.', 'gene_id \"chr35.g12247\"; transcript_id \"blat.proc28144.chain_17830\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22905222', '22905270', '100', '+', '.', 'gene_id \"chr35.g12311\"; transcript_id \"blat.proc18128.chain_7725\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22458517', '22458573', '100', '-', '.', 'gene_id \"chr35.g12346\"; transcript_id \"blat.proc4005.chain_11211\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '11338433', '11338485', '100', '+', '.', 'gene_id \"chr35.g12351\"; transcript_id \"blat.proc4587.chain_11460\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '13375405', '13375454', '100', '+', '.', 'gene_id \"chr35.g12414\"; transcript_id \"blat.proc4587.chain_7317\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22996499', '22996557', '100', '+', '.', 'gene_id \"chr35.g12508\"; transcript_id \"blat.proc18128.chain_3285\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23143281', '23143325', '100', '-', '.', 'gene_id \"chr35.g12600\"; transcript_id \"blat.proc4587.chain_4560\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22408729', '22408769', '100', '+', '.', 'gene_id \"chr35.g12625\"; transcript_id \"blat.proc28144.chain_4498\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10187222', '10187255', '100', '+', '.', 'gene_id \"chr35.g12712\"; transcript_id \"blat.proc28144.chain_5139\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '18802881', '18802932', '100', '+', '.', 'gene_id \"chr35.g12723\"; transcript_id \"blat.proc4005.chain_2727\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '452', '497', '100', '-', '.', 'gene_id \"chr35.g12736\"; transcript_id \"blat.proc4587.chain_7182\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '13335688', '13335743', '100', '+', '.', 'gene_id \"chr35.g12737\"; transcript_id \"blat.proc4587.chain_1828\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '6361313', '6361433', '100', '+', '.', 'gene_id \"chr35.g12816\"; transcript_id \"blat.proc28144.chain_21494\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '23023848', '23023891', '100', '-', '.', 'gene_id \"chr35.g12839\"; transcript_id \"blat.proc18128.chain_2290\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '13735328', '13735362', '100', '-', '.', 'gene_id \"chr35.g12865\"; transcript_id \"blat.proc4587.chain_5894\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '316698', '316738', '100', '-', '.', 'gene_id \"chr35.g12943\"; transcript_id \"blat.proc18128.chain_13853\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '5474156', '5474197', '100', '+', '.', 'gene_id \"chr35.g12962\"; transcript_id \"blat.proc18128.chain_23308\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '4651130', '4651170', '100', '-', '.', 'gene_id \"chr35.g13063\"; transcript_id \"blat.proc28144.chain_20551\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '10716322', '10716354', '100', '+', '.', 'gene_id \"chr35.g13064\"; transcript_id \"blat.proc28144.chain_13936\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '3732894', '3732926', '100', '+', '.', 'gene_id \"chr35.g13128\"; transcript_id \"blat.proc28144.chain_22438\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '22552156', '22552208', '100', '-', '.', 'gene_id \"chr35.g13130\"; transcript_id \"blat.proc18128.chain_13700\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '25691981', '25692035', '100', '+', '.', 'gene_id \"chr35.g13162\"; transcript_id \"blat.proc18128.chain_12878\";']\n",
      "negative:  ['chr35', 'PASA', 'exon', '12929048', '12929081', '100', '-', '.', 'gene_id \"chr35.g13170\"; transcript_id \"blat.proc28144.chain_6113\";']\n",
      "Stored transcript, exon, and intron stats for 1217471 transcripts\n"
     ]
    }
   ],
   "source": [
    "length1File = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/rna-seq/PASA_Processing/pasa-lite_assemblyAssembler_results/pasa-lite_ZoeyTrinity.pasa_assembled_alignments.gtf'\n",
    "\n",
    "#Keeping track of exon counts/lengths\n",
    "length1ExonLengths = []\n",
    "length1ExonCounts = []\n",
    "exon1Dict = {}\n",
    "#Keeping track of intron stats\n",
    "length1IntronLengths = []\n",
    "intron1Dict = {}\n",
    "\n",
    "#Keeping track of full leng\n",
    "length1TranscriptLengths = []\n",
    "\n",
    "lineCount,transcriptCount = 0,0 \n",
    "exonCount = 0 # for first call, set to zero\n",
    "\n",
    "for line in open(length1File,'r'):\n",
    "    lineCount += 1        \n",
    "    line=line.rstrip().split('\\t')\n",
    "    if '#' in line[0] or len(line) == 1: #skip header\n",
    "        continue\n",
    "    \n",
    "    #calculate start/end coordinates\n",
    "    start,end = int(line[3]) - 1, int(line[4])\n",
    "    length = end - start\n",
    "    \n",
    "    if 'transcript' in line[2]:\n",
    "        transcriptCount += 1\n",
    "        length1TranscriptLengths.append(length)\n",
    "        if lineCount != 2:\n",
    "            length1ExonCounts.append(exonCount) #add the count from the previous gene\n",
    "        exonCount = 0 #reset the count to zero with each new transcript\n",
    "        exonCoords, e = [], -1 # to keep track of the start, end coordinates of each exon\n",
    "        i = 0 #intron index\n",
    "        \n",
    "    if 'exon' in line[2]:\n",
    "        #Add exon length to array:\n",
    "        length1ExonLengths.append(length)\n",
    "        exonCount += 1 #Keep track of exon count\n",
    "        e += 1 #for dict/array index\n",
    "        exonCoords.append([start,end]) #start/end coordinates per exon\n",
    "        \n",
    "        #Storing exon length by exon number:\n",
    "        if e+1 not in exon1Dict.keys():\n",
    "            exon1Dict[e+1] = []\n",
    "        exon1Dict[e+1].append(length)    \n",
    "        \n",
    "    # Get intron lengths, means that the gene must have more than one exon\n",
    "    if exonCount > 1:\n",
    "        i += 1 #intron index = intron number\n",
    "        \n",
    "        #Intron size depends on coordinates of the exons A and B surrounding it:\n",
    "        startB, endA = exonCoords[e-1][1],exonCoords[e][0]\n",
    "        intronLength = int(endA) - int(startB)\n",
    "        length1IntronLengths.append(intronLength)\n",
    "        \n",
    "        if intronLength > 1000000:\n",
    "            print('long: ',line)\n",
    "        if intronLength < 0:\n",
    "            print('negative: ',line) \n",
    "            \n",
    "        #Storing intron length by intron number:\n",
    "        if i not in intron1Dict.keys():\n",
    "            intron1Dict[i] = []\n",
    "        intron1Dict[i].append(intronLength)\n",
    "\n",
    "        \n",
    "print('Stored transcript, exon, and intron stats for %i transcripts' % transcriptCount)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAxAAAAGoCAYAAADW/wPMAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAAIABJREFUeJzs3Xd8HPWd//H3R8bghBKc4HAcJoHjnATILyHgAwIJocnYlFCOHOUCpuQIHBA6MskRekAJvQYDDqYEMA7FgMF2wAZCKDbY2MjGTbg3uRfZliV9fn/M7HpWu5JG8lbp9Xw89qHdz3znO5+ZHe3OZ6eZuwsAAAAA4igrdAIAAAAASgcFBAAAAIDYKCAAAAAAxEYBAQAAACA2CggAAAAAsVFAAAAAAIiNAgIAkGRms8zMzeyJdozr4eOG7GeGLWFmY8L3ZkyhcwFQ+iggAJSUyIZQpsfZhc4vKpLrrELnEmVmh0WW2WGFzieuSHGT6XFYofOLapJr/0g8uuyPK2SOANBeWxU6AQBopzpJ45vEagqRCPJujaTJTWKrC5FITDea2bPuXlfoRHLJzLbu6PMIIMAeCAClaqG7H9Tk8bqZbWVmH4W/8L5jm70Vxj41s60lycy+b2YvmtlSM6szsy/N7A4z2y4xkeihH2Z2UfjL8hoze83M/mVLZ8LMtjaz68xsqpltNLNlZvZXM+sZaXND5Ffrw8N5WB/+PahJf782szlmVhvm+Mvor/Th4UWjI6OMbuaQJTOz35rZAjNbYWZPm9n2MWdrazO718yWm9lKM3sgssw/CKf3XJOJzQ7jt8Xo/9MM7/2nZrZD+B66mT0V9ruNmU0OY69FpvcTMxthZqvC5T7VzH5nZl0jbRJ7EZ40sxvNbGE7loUkfVvShc0NbG6PUCR2Q4Z255rZ6HA9+MjM9jKzcjP7PFw/hze3fprZ+eG8rTez16PrWjj8NDP70MzWhY+3zeyQZvL9nzCPDZL+tw3LBEApc3cePHjwKJmHpDGSXNKsFtr8u6S1YbuLFWzYuKRaSd8L2+yl4JdsD9tOltQQvn5fUlmT6dVJWi9pWvjaJT2ThVxfDds0SJooaXn4erak7mGbGyLT3CDpC0mbEn1L2ipsd0yk3TJJ1ZHl4JIOk/SrcF4TscmSPpR0XdjHrMiyWh32kWh7ayvzm2i3VsHeoC8jsT+GbX4ZmY+vh7EDIu2+10L/idzGtNDmEEn1YbvjJP0xfL5I0jfDNodFlt8KSVMj0382w/Tq2rEsEuN+FuazRNJ24bQTfRwXySf5HmVYnjdkaLchzHtj+HqmpHWSpkhqVJP1U5vXxbVhu8mRdh9F2l0ZmcYMSXMiy+DHGfLYGL7XkyX9ptCfDzx48MjPgz0QAErVty39OPgdJcndZ0i6NGx3m4KNSEm60t2/CJ8PULBBVytpH3ffW9JF4bCDJR3bZHpdFGxAfUfSS2HsyC2ZATM7VMFGriT1c/cfSPo3SUslfUuZf9G92t2/p2BDTwp+3f73yDxJ0lxJe7r7v0kaFh3Z3R9r0u//evAL/s1NplOvoMj6d0njwljc+Z0nafdwXhLL6hIz+6qkFxRscG6joJiQpP8M/34UeX9a8rMm7/vKxAB3f1/Bey5JgyRdET4/x92XhM9vVHAI71xJ/+bu35VUGQ47zcz+X5PpbVD7l8V0SU9I6qHN71k2PBXm/afw9b9J+oO77yXpmRZy3EbS/uH6nlg2B4R7tr6qYNlI0m3u/u8K3seRkrpKuilDf/+UtFvY34NbOE8ASgQFBIBSVSfpoyaP+sRAd39c0ssKioRtJb3u7g9Hxv+P8O/77j47fP7XyPDeTaY3yd0nhM8Tx99/cwvn4cDI8xFmlvhFfKcwdlD6KHqqSQ6StHP49/vh3zfdPbFRnXKoUBu87e7z3b1RwS/d0em05nV3X+furqBgkKRuCoqajZIeC2Pnhn8TBcTgmP2vUer7PrbJ8BvDWA8Fhd9D7v5GZHjivX/T3VeEz1t677dkWUjBHqQNCgqInVpuGtur4d9ZGWLV4d9M6+fESJH2fCT+fUn7KPhfkaRrw/WxQVKfMJZpfXzE3TdIkrs3xM4eQEnjJGoApWqhu2faoJEkmdlWknaNhHYzs23CDdgojzm9lZHniULFYo7bnOj4H2fIZU7TESKFQX0k3DSPuPPUkmzNb6Zx/izpGkk/NLPzJO2p4FCYuMXOp+5+WAvDt1XqxvOeZmZhQROVl/fe3eeZ2YMKCohrMzWJPO8iSWb2tVa6TZw0Xp8hluivtRybDo++/kLSqibDMy2vRa1MA0AHxB4IAB3VDQp+aV4UPn4g6Q+R4YlfrQ8xs2+Hz8+IDB+n7DEz69bksZWCoiHhrvBQooMk/VjBBvYjbZzOpPBvn8hJvqdlaFcbeb5thuFb6hgz+6qZmaSTw9gGBcfpy93nSHo9jN8b/h0W2RuwpR5UcGjXNAXzerSCc2ESEu99PzPrHj7P1Xuf8AcFG+T7ZRi2JPJ8z/DvSTnIQZJ+YGbfDZ+fEol/Hj4S68bbkg6OrJNnS7o+RzkBKDEUEABK1S7hlWKij19JwRV2tPl8gF8rOHFYki43syPC57crOKF0W0lVZlalzcdw/1ObN3Cz4VsKTsCOPv7P3cdIShxa85yZTTOzSQo2NN9R5o3NliSO499d0pdmVi3phAztZio4iViSngyX3SkZ2rXXbgoOranW5sOTHnD3aOGSWNaJAuaJNvS/X4b3/lgpuIKQpP9W8Mv86QoKMUn6o5ntHT6/Phy+m6RqM5sqqSIc9py7T1KWuftySXc0M3i6Nu9tusvMRkt6uJm2W2qjpE/D9f2eMDZOwYnptdp8DsT/SppvZuPNbImCk7PPSOsNQKdEAQGgVG2t4ByC6KOnme2g4DyBLgpONB3m7q9LelzBIRqDzay7u09R8Ev/Swo2qr6j4KTaOyUdHR7vng8nKdig/ULBr+Y9FWx436ngyjmxuftwSRcomI9tFRyvf1Wkyfqw3TJJvwnbdVew7Lb4krQR90l6VtLXFBxW87Ck/2vSZpSCDWcp2EM0og39b6/0976Hme2mzRvet7n7p5IeCqfVTdIzFtyrYIykwxWcHFwmaQ8Feyuuk3RWG/Joq7slLW4adPd6SacquK9JF0lf1+bCK9vGSbpcwblBdZLelHRy4vAud/+jggLsQ0k7KPi/WKng/JTHMnUIoPOx9ENCAQClKLyHQU93/zISe1zBycp1knZy9zWFyq8pMxsm6XhJf3L3a1prDwAoDpxEDQAdx7aSZpjZJ5IWKPj1eK9w2G3FUjyY2e8k/VTBuQkbJN1f2IwAAG1BAQEAHccGSa8pOHl83/D1+woutflUSyPmWbmkQxWci3GFu88tcD4AgDbgECYAAAAAsXESNQAAAIDY8nIIk5l1k/SupG3CaQ519+vN7AlJP9Pmm9Wc7e4TwmuH3yvpGAXXpD47vJqGzKy/Nl/N4xZ3HxzG91dwGcCvSBou6dIMNw1KsdNOO/nuu++erdkEAAAASsYnn3yy1N17tHW8fJ0DsVHSEe6+NrxKyD/MLHHt86vdfWiT9v0k9QofByq4LN+BZvZ1BZc77K3gjpifmFni5kMPSzpfwaXnhkvqq83XV89o991317hxubhfEAAAAFDczGx2e8bLyyFMHlgbvuwaPlraO3CCpCfD8T6UtKOZ7aLgih2j3H15WDSMktQ3HLaDu38Q7nV4UtKJOZshAAAAoJPK2zkQZtbFzCZIWqKgCPgoHHSrmU00s7vNbJswtquCGxwlzAtjLcXnZYhnyuN8MxtnZuNqamq2eL4AAACAziRvBYS7N7j7vgrusnqAmX1f0rWSvqfgkoNfl1QRNrdMXbQjnimPge7e29179+jR5kO+AAAAgE4t71dhcveVksZI6uvuC8PDlDZK+oukA8Jm8yTtFhmtp4KbIrUU75khDgAAACCL8lJAmFkPM9sxfP4VSUdJ+iI8d0HhVZdOlPR5OMowSWdZ4CBJq9x9oaQRkvqYWXcz6y6pj6QR4bA1ZnZQ2NdZkl7Jx7wBAAAAnUm+rsK0i6TBZtZFQdEyxN1fM7O3zayHgkOQJki6IGw/XMElXGcouIzrOZLk7svN7GZJY8N2N7n78vD5hdp8Gdc31MoVmAAAAAC0Xae+E3Xv3r2dy7gCAACgMzKzT9y9d1vH407UAAAAAGKjgAAAAAAQGwUEAAAAgNgoIAAAAADERgEBAAAAIDYKCAAAAACxUUAAAAAAiI0CQtL8+fPVme+HAQAAAMRFASHpnHPO0axZswqdBgAAAFD0KCBCGzduLHQKAAAAQNGjgAAAAAAQGwUEAAAAgNgoIAAAAADERgEBAAAAIDYKCAAAAACxUUAAAAAAiI0CAgAAAEBsFBAAAAAAYqOAAAAAABAbBQQAAACA2CggAAAAAMRGAQEAAAAgNgoIAAAAALFRQAAAAACIjQICAAAAQGwUEAAAAABio4AAAAAAEBsFBAAAAIDYKCAAAAAAxEYBAQAAACA2CggAAAAAsVFAAAAAAIiNAgIAAABAbBQQAAAAAGKjgAAAAAAQGwUEAAAAgNgoIAAAAADERgEBAAAAIDYKCAAAAACx5aWAMLNuZvaxmX1mZlVmdmMY38PMPjKz6Wb2vJltHca3CV/PCIfvHunr2jA+1cyOjsT7hrEZZjYgH/MFAAAAdDb52gOxUdIR7v5DSftK6mtmB0mqlHS3u/eStELSeWH78yStcPd/l3R32E5mtrek0yTtI6mvpIfMrIuZdZH0oKR+kvaWdHrYFgAAAEAW5aWA8MDa8GXX8OGSjpA0NIwPlnRi+PyE8LXC4UeamYXx59x9o7t/KWmGpAPCxwx3r3b3OknPhW0BAAAAZFHezoEI9xRMkLRE0ihJMyWtdPf6sMk8SbuGz3eVNFeSwuGrJH0jGm8yTnPxTHmcb2bjzGxcTU1NNmYNAAAA6DTyVkC4e4O77yupp4I9Bntlahb+tWaGtTWeKY+B7t7b3Xv36NGj9cQBAAAAJOX9KkzuvlLSGEkHSdrRzLYKB/WUtCB8Pk/SbpIUDv+apOXReJNxmosDAAAAyKJ8XYWph5ntGD7/iqSjJE2RNFrSKWGz/pJeCZ8PC18rHP62u3sYPy28StMeknpJ+ljSWEm9wqs6ba3gROthuZ8zAAAAoHPZqvUmWbGLpMHh1ZLKJA1x99fMbLKk58zsFknjJT0etn9c0lNmNkPBnofTJMndq8xsiKTJkuolXeTuDZJkZhdLGiGpi6RB7l6Vp3kDAAAAOo28FBDuPlHSjzLEqxWcD9E0vkHSL5rp61ZJt2aID5c0fIuTBQAAANAs7kQNAAAAIDYKCAAAAACxUUAAAAAAiI0CAgAAAEBsFBAAAAAAYqOAAAAAABAbBQQAAACA2CggAAAAAMRGAQEAAAAgNgoIAAAAALFRQAAAAACIjQICAAAAQGwUEAAAAABio4AAAAAAEBsFBAAAAIDYKCAAAAAAxEYBAQAAACA2CggAAAAAsVFAAAAAAIiNAgIAAABAbBQQAAAAAGKjgAAAAAAQGwUEAAAAgNgoIAAAAADERgEBAAAAIDYKCAAAAACxUUAAAAAAiI0CAgAAAEBsFBAAAAAAYqOAAAAAABAbBQQAAACA2CggAAAAAMRGAQEAAAAgNgoIAAAAALFRQAAAAACIjQICAAAAQGwUEAAAAABio4AAAAAAEBsFBAAAAIDY8lJAmNluZjbazKaYWZWZXRrGbzCz+WY2IXwcExnnWjObYWZTzezoSLxvGJthZgMi8T3M7CMzm25mz5vZ1vmYNwAAAKAzydceiHpJV7r7XpIOknSRme0dDrvb3fcNH8MlKRx2mqR9JPWV9JCZdTGzLpIelNRP0t6STo/0Uxn21UvSCknn5WneAAAAgE4jLwWEuy9090/D52skTZG0awujnCDpOXff6O5fSpoh6YDwMcPdq929TtJzkk4wM5N0hKSh4fiDJZ2Ym7kBAAAAOq+8nwNhZrtL+pGkj8LQxWY20cwGmVn3MLarpLmR0eaFsebi35C00t3rm8QzTf98MxtnZuNqamqyMEcAAABA55HXAsLMtpP0N0mXuftqSQ9L2lPSvpIWSroz0TTD6N6OeHrQfaC793b33j169GjjHAAAAACd21b5mpCZdVVQPDzj7i9Kkrsvjgx/VNJr4ct5knaLjN5T0oLweab4Ukk7mtlW4V6IaHsAAAAAWZKvqzCZpMclTXH3uyLxXSLNTpL0efh8mKTTzGwbM9tDUi9JH0saK6lXeMWlrRWcaD3M3V3SaEmnhOP3l/RKLucJAAAA6IzytQfiEElnSppkZhPC2G8VXEVpXwWHG82S9GtJcvcqMxsiabKCKzhd5O4NkmRmF0saIamLpEHuXhX2VyHpOTO7RdJ4BQULAAAAgCzKSwHh7v9Q5vMUhrcwzq2Sbs0QH55pPHevVnCVJgAAAAA5wp2oAQAAAMRGAQEAAAAgNgoIAAAAALFRQAAAAACIjQICAAAAQGwUEAAAAABio4AAAAAAEBsFBAAAAIDYKCAAAAAAxEYBAQAAACA2CggAAAAAsVFAAAAAAIiNAgIAAABAbBQQAAAAAGKjgAAAAAAQGwUEAAAAgNgoIAAAAADERgEBAAAAIDYKCAAAAACxbdVaAzPrLqlc0o8kdZe0UtJ4SaPcfXlu0wMAAABQTJotIMxsT0k3SDpF0taSLDLYJdWZ2QuSbnD36lwmCQAAAKA4tLQHYko4fLqkd8LXqyXtIGkvST+T9EtJ/yWpW27TBAAAAFAMWioghki6y90/ba6Bme0v6bKsZwUAAACgKDVbQLj7L1sb2d0/kXRmVjMCAAAAULRaPYk6wcx6KDgfoqekeZL+5u5LcpUYAAAAgOITq4AwsyMkvSLpq5FwpZmd5O5v5SQzAAAAAEUn7n0g7pe0raTPJD0f/t1O0r05ygsAAABAEYp7CNO3JY1w936JgJm9KemQnGQFAAAAoCjF3QPxSjPxodlKBAAAAEDxi7sHoqukcjMbJ+kLSd+T9ENJL5jZoLCNu/t5OcgRAAAAQJGIW0CcEv7dL3wknBZ57pIoIAAAAIAOLG4BcWNOswAAAABQEmIVEO5OAQEAAAAg3knUFrjIzD4ys/lm9mH42nKdIAAAAIDiEfcQptslXSUpUTDsIuk/JH1LUkUO8gIAAABQhOJexvUcSeskXSapr6TLJdWGcQAAAACdRNw9EBslDXP3+8LXI83sB5LKc5MWAAAAgGIUdw/EAEk/M7Pvm9lWZvb/JB0axgEAAAB0Es3ugTCzhgzhz6JNJD0p6a/ZTgoAAABAcWppD4S18mht/M0dme1mZqPNbIqZVZnZpWH862Y2ysymh3+7h3Ezs/vMbIaZTTSz/SJ99Q/bTzez/pH4/mY2KRznPq4QBQAAAGRfS+dA7JHF6dRLutLdPzWz7SV9YmajJJ0t6S13v93MBig4JKpCUj9JvcLHgZIelnSgmX1d0vWSeiu48/UnZjbM3VeEbc6X9KGk4QpO9n4ji/MAAAAAdHrNFhDuPjtbE3H3hZIWhs/XmNkUSbtKOkHSYWGzwZLGKCggTpD0pLu7pA/NbEcz2yVsO8rdl0tSWIT0NbMxknZw9w/C+JOSThQFBAAAAJBVzR6CZGaDzeyHLY1sZvuGG+uxmdnukn4k6SNJO4fFRaLI+GbYbFdJcyOjzQtjLcXnZYhnmv75ZjbOzMbV1NS0JXUAAACg02vpEKYzJP3SzKYr2DMwRdIaSdtL2kvB3oBeCg5POivOxMxsO0l/k3SZu69u4TSFTAO8HfH0oPtASQMlqXfv3hnbAAAAAMispQJiH0k3SjpZ0neUukFukjZJel7BOQmtMrOuCoqHZ9z9xTC82Mx2cfeF4SFKS8L4PEm7RUbvKWlBGD+sSXxMGO+ZoT0AAACALGrpHIhpkk43sx6S+kj6gaTuklZKmihppLsvaW78qPCKSI9LmuLud0UGDZPUX9Lt4d9XIvGLzew5BSdRrwqLjBGS/pC4WlOY17XuvtzM1pjZQQoOjTpL0v1xcgMAAAAQX6t3onb3GknPhI/2OkTSmZImmdmEMPZbBYXDEDM7T9IcSb8Ihw2XdIykGZJqJZ0T5rLczG6WNDZsd1PihGpJF0p6QtJXFJw8zQnUAAAAQJa1WkBkg7v/Q5nPU5CkIzO0d0kXNdPXIEmDMsTHSfr+FqQJAAAAoBWxbgQHAAAAABIFBAAAAIA2oIAAAAAAEFvscyDCezh8R9J20bi7v5vtpErZhg0btPXWW6usjNoMAAAAHU+sAsLMzpD0Z0nbNhnkcfvoLC6++GL97Gc/05lnnlnoVAAAAICsi7vx/0cFex4aJK1TM3d5hjRnzhxNmTKl0GkAAAAAORG3gPiqpJGSjnP3+hzmAwAAAKCIxT1Q/xFJO6r5ezkAAAAA6ASa3QNhZm9HXpZJ6i1pgZlNkZTYC+HunnYjuFLn7qqurtaee+5Z6FQAAACAotLSIUyHZYh9Q9JPIq875LkQCxYs0IUXXqiRI0cWOhUAAACgqLRUQJyTtyyKTH09p3kAAAAAmTRbQLj74MRzMztL0mR3HxeJ7Spp59ymBwAAAKCYxD2J+glJpzeJXSVpbFazAQAAAFDUWryMq5kdGnm5W+R14qTqxlwlBgAAAKD4tHYfiDEKTpR2Sf8ZPqJm5CAnAAAAAEWqtQJijoLi4duS1khaHsYbJM2VdEPOMgMAAABQdFosINx9d0kys1mSHnf3m/OQEwAAAIAi1doeCEmbCwkAAAAAnVusAsLMljczaKOkqZLuc/cXs5YVAAAAgKIU9zKuOzbz2FnSoZJeMLOTc5IhgFiuu+46zZkzp9BpAACADi5uAXGlgpOor5DUV9LlklZL+q2k3ynYE3FVLhIEkG7SpElasmRJSuyjjz7StGnTCpQRAADoLOIWEGdLGuru97j7SHe/V9LfJJ3m7rcpuNHc93KTIoCmrrzySj3yyCOFTgMAAHRCsc6BkLS7pO5m9hNJX0jqJekoBYcxSdJSSVtnPTsAzVqzZk2hUwAAAJ1Q3ALiTUm/kPROJGaShoTP+yk4mRpAnphZoVMAAACdUNwC4n8kbZB0qoI9DXWSnpV0qZltJ+kWBTedAwAAANCBxb0PxGpJ/c3sHEk7SVrq7o2RJq/kIjkAAAAAxSXuHgiZ2Z6Sfihpu/C1JMndn8xJZgAAAACKTtwbyV0l6XYF5z001SkKiIaGBq1evVrdu3dPxmpra1VfX68ddtihxXEXLlyonXfeWWVlcS96BQAAABSnuFu0V4dt50qaJGli5NEpvPHGGzr11FNTYnfccYd+97vftTpu//799emnn+YqNQAAACBvYh/CJOlFdz8lZ5kUuYULF6bFqqqqtGLFiljjt+eSm+PHj9ecOXN0wgkntHlcAAAAIBfi7oG4R9L3zKxHLpNBqsGDB+vBBx8sdBoAAABAUtw9EOdL+pak+Wa2SFJ9GHd33zMnmUGNjY2tNwIAAADyKG4B8e1I+56RuGc3HQAAAADFLG4BcXhOswAAAABQEuLeSO4dSTKzbpL+zd0n5zQrAAAAAEUp1knUZvYVM3tc0lpJE81sdzObaWaX5DY9AAAAAMUk7lWYbpZ0Ttje3H2WpBWS/jtHeQEAAAAoQnELiFMkfSLp4UjsM0nfyXpGAAAAAIpW3AKiu6TJkuoisa9IsqxnhBYtWbJEq1evLnQaAAAA6KTiFhBfSOoraS9JMrOLJJ0oaUqckc1skJktMbPPI7EbzGy+mU0IH8dEhl1rZjPMbKqZHR2J9w1jM8xsQCS+h5l9ZGbTzex5M9s65nyVnMsuu0z33HNPodMAAABAJxW3gLhVUg9J5eHr+yR1k3R7zPGfUFCANHW3u+8bPoZLkpntLek0SfuE4zxkZl3MrIukByX1k7S3pNPDtpJUGfbVS8G5GefFzKvkLF26VLNmzSp0GgAAAOikYhUQ7j5MwYb7Gwr2Orwh6ZgwHmf8dyUtj5nTCZKec/eN7v6lpBmSDggfM9y92t3rJD0n6QQzM0lHSBoajj9Ywd4RAAAAAFkWdw+E3H2Eux/n7t939+MkrTWzn2/h9C82s4nhIU7dw9iukuZG2swLY83FvyFppbvXN4kDAAAAyLLYBUQGf5L00haM/7CkPSXtK2mhpDvDeKYTs70d8YzM7HwzG2dm42pqatqWMQAAANDJbUkBsUXcfbG7N7h7o6RHFRyiJAV7EHaLNO0paUEL8aWSdjSzrZrEm5vuQHfv7e69e/TokZ2ZAQAAADqJghUQZrZL5OVJkhJXaBom6TQz28bM9pDUS9LHksZK6hVecWlrBSdaD3N3lzRawb0qJKm/pFfyMQ8AAABAZ7NVSwNbOcehewvDmvbzrKTDJO1kZvMkXS/pMDPbV8HhRrMk/VqS3L3KzIYouO9EvaSL3L0h7OdiSSMkdZE0yN2rwklUSHrOzG6RNF7S43Fz6yyWLFmiO++8U5WVlYVOBQAAACWsxQJC0stq/nwCa2FYCnc/PUO42Y18d79VwaVjm8aHSxqeIV6tzYdAIYPZs2dr/PjxKbFFixZp5syZOuSQQwqUFQAAAEpNa4cwzWnhMTv8izZqaGjQddddlxKbNm2arrjiirzm8dRTT+nGG2/M6zQBAABQ2lrcA+Huu+cpj06loaFBH330UUps2rRp+vzzz5sZo+2GDh2qk046SV26dGm2TV1dXdamBwAAgM6hYCdRI7cGDhyoVatWFToNAAAAdDAUEAAAAABio4AAAAAAEBsFBEpWXV2dqqqqWm8IAACArKGAQMl67733dPnllxc6DQAAgE6ltftASJLMbDdJt0j6kaTtIoPc3ffMRWJAazZs2FDoFAAAADqdWAWEpGcl/VjBzeOiYt1IDgAAAEDHELeA+JGkuZJukLRSFA4AAABApxS3gBgnaY67P5HDXAAAAAAUubgFxPuSrjazFZImSKpPDHD3J3ORGAAAAIDiE7eAGKDgsKWLMgyjgAAAAAA6ibgFxLvivAcAAACg04tVQLj7YTnOAwAAAEAJiLsHQma2h6TzJH1b0ixJg9z9yxzlBQAAAKAIxb2R3AGS3pL0VW2+F8TlZnaEu3+cq+RQvFasWKHu3bvBWndPAAAgAElEQVQXOg0AAADkWVnMdrdL2lbSK+HzlxUUE7flKC8UuVNPPVWzZs0qdBoAAADIs7iHMO0r6W/u/otEwMxekHRkTrJCSVi3bl2hUwAAAECexd0DsUHSTk1iO4VxAAAAAJ1EW24kd7KZTZc0VdJ3JO0p6W+5SgwAAABA8YlbQFwlaX8FRcOeYWyWpGtykBMAAACAIhX3PhCzzez7ko6VtLuC4uF1d6/NXWoAAAAAik3cy7j+wt1fkPRCk/jV7v6nnGQGAAAAoOjEPYn6eTN70sx2kCQz293M3lVwSVeg5NTW1srdU2L19fUFygYAAKB0xC0gpkj6paRJZna9pM8k/UTS27lKDMilE088UWPGjEmJHXPMMVqzZk1hEgIAACgRcQuIfSXdKqmnpN9L6irpQncvz1Vi6JgWLVqk2bNnp8RmzpyplStX5j2XBQsWpMU2bODKxAAAAC2JW0D8q6QDJVn42EbSj83sa7lKDKVnzpw5rbaprKzUVVddlRK78MIL9eijj+YqLQAAAGRR3ALic0lHSXpT0hEKDmE6S9LkHOWFErN48WL96le/arXdwoULtWrVqrT4smXLspLHpk2btHbt2qz0BQAAgHRxCwiT9L/ufoy7j5F0gIITqL+Zq8QQ37vvvqvVq1fnrP/GxsZW22zatCln02+Lxx57TJdeemmh0wAAAOiw4hYQP3L3PydeuHu9u/9W0qG5SQttccstt2jEiBE56//Xv/61Ro4c2ebxPvroI11yySU5yKh5U6dO1dy5c/M6TQAAgM6kxQLCzH5gZru4+/QMw/aR1CNnmaFNGhoactb37NmzVVVV1ebxqqqqNHXq1BxkBAAAgEJpbQ/EeElXSZKZ3W1m0Qvl/0rSS7lKDAAAAEDxaa2ASFx1KfoaAAAAQCcV9xwIAAAAANBWMdrsaWY/l7SnJJnZ8Qr2ROyZy8QAAAAAFJ84BcRx4SPh5RzlAgAAAKDItVZAzJHk+UgExWHw4MHq2rWrzjjjjEKnAgAAgCLUYgHh7rvnKQ8UiWeeeabDFRA1NTXq1q2btt9++0KnUtLWr1+vbt26yazjXEthxYoV2nHHHTvUPAEAkGucRI0O78orr9Q999xT6DRK3kknnaQxY8YUOo2sOvXUUzVu3LhCpwEAQEnJSwFhZoPMbImZfR6Jfd3MRpnZ9PBv9zBuZnafmc0ws4lmtl9knP5h++lm1j8S39/MJoXj3Gf8nIiIRYsWqbq6utBplLzGxkbNnz+/0Glk3YoVKwqdAgAAJSVfeyCekNS3SWyApLfcvZekt8LXktRPUq/wcb6kh6Wg4JB0vaQDJR0g6fpE0RG2OT8yXtNpAQAAAMiCvBQQ7v6upOVNwidIGhw+HyzpxEj8SQ98KGlHM9tF0tGSRrn7cndfIWmUpL7hsB3c/QN3d0lPRvoCAAAAkEWFPAdiZ3dfKEnh32+G8V0lzY20mxfGWorPyxDPyMzON7NxZjaupqZmi2cCAAAA6EyK8STqTOcveDviGbn7QHfv7e69e/To0c4UAQAAgM6pkAXE4vDwI4V/l4TxeZJ2i7TrKWlBK/GeGeIAAAAAsqyQBcQwSYkrKfWX9EokflZ4NaaDJK0KD3EaIamPmXUPT57uI2lEOGyNmR0UXn3prEhfAAAAALKotTtRZ4WZPSvpMEk7mdk8BVdTul3SEDM7T8Edr38RNh8u6RhJMyTVSjpHktx9uZndLGls2O4md0+cmH2hgis9fUXSG+EDAAAAQJblpYBw99ObGXRkhrYu6aJm+hkkaVCG+DhJ39+SHAEAAAC0rhhPogYAAABQpCggAAAAAMRGAQEAAAAgNgoIAAAAALFRQAAAAACIjQICAAAAQGwUEChK1dXVamhoKHQayIKGhgatXr261Xbr1q3jPQcAoARQQGyB4JYVyIULLrhAH3/8caHT6BSGDRumW2+9NWf9v/DCCzrrrLNabXfWWWfppZdeylkeAAAgOyggULTWr19f6BQ6hTFjxuidd97JWf9z5sxRbW1tq+3WrFmj2bNn5ywPAACQHRQQAAAAAGKjgEBJcHe9/PLLhU4jqxoaGjR8+PBW2y1YsEAzZ87MQ0YAAACto4BASaivr9dDDz1U6DQyGjt2rD755JM2j7do0SLdc889rbarrKzUlVde2Z7UAAAAsm6rQicAlLqbbrpJXbt21YsvvpiT/mtqamKdQwAAAJAP7IEAttDGjRuLdgO/pqYm5bW7a8WKFQXKBgAAdAQUEEAzampqVFFRUeg02q2xsVH//d//nRIbO3asTj311AJlBAAAOgIKCKAZc+fO1fjx4wudRlYtX7680CkAAIASRwEBAAAAIDYKCAAAAACxUUAAAAAAiI0CAgAAAEBsFBAAAAAAYqOAQKdlZoVOoeTNmzev0CkAAIA8o4AA0G7nnnsul4YFAKCToYAAsEXq6+sLnQIAAMgjCggAAAAAsVFAAAAAAIiNAgJog9WrV2vq1KmFTgMAAKBgKCDQabl7m8d55plndMkll7TabtSoUVqwYEF70gIAAChqFBBAG6xatSpWuz/96U8aMmRIjrMBAADIPwoIIEcaGxsLnQIAAEDWUUAAAAAAiI0CAkBWrV27ttApAACAHKKAAJA1M2fO1Mknn1zoNAAAQA5RQADImtWrVxc6BQAAkGMUEAAAAABio4AAAAAAEBsFBAC00Zdffqn33nuv0Gmorq6uXTdEBABgS1BAAEAbPfroo7r55psLnYbOO+88vfzyyymxu+66SytXrixQRgCAzoACAp2WmRU6BZSourq6QqcgSVq8eLGmT5+eEnvzzTf15ZdfFigjAEBnUPACwsxmmdkkM5tgZuPC2NfNbJSZTQ//dg/jZmb3mdkMM5toZvtF+ukftp9uZv0LNT8oThzmAQAAkB0FLyBCh7v7vu7eO3w9QNJb7t5L0lvha0nqJ6lX+Dhf0sNSUHBIul7SgZIOkHR9ougA0HZPPvmk3njjjUKnAQAAilCxFBBNnSBpcPh8sKQTI/EnPfChpB3NbBdJR0sa5e7L3X2FpFGS+uY7aRSvYjhcadKkSWpsbGyxTWNjY1HcS+Hpp5/Ws88+W+g0EDF27Fh9/PHHhU4DAICiKCBc0kgz+8TMzg9jO7v7QkkK/34zjO8qaW5k3HlhrLl4GjM738zGmdm4mpqaLM4G0LIrr7xS06ZNa7HNqFGjdMopp+QpI5SSW2+9Vbfcckuh0wAAQFsVOgFJh7j7AjP7pqRRZvZFC20z/YzsLcTTg+4DJQ2UpN69e3NgPPKqvr6+xeELFizIUyYoNbW1tYVOAQAASUWwB8LdF4R/l0h6ScE5DIvDQ5MU/l0SNp8nabfI6D0lLWghDjSLE6tRSorhMLyO6sUXX9T69esLnQYAlIyCFhBmtq2ZbZ94LqmPpM8lDZOUuJJSf0mvhM+HSTorvBrTQZJWhYc4jZDUx8y6hydP9wljQLM6+gZZMRdIM2fOVHV1dV6nuXjxYn7FR0Z//vOfNXny5EKnAQAlo9CHMO0s6aVwQ24rSX919zfNbKykIWZ2nqQ5kn4Rth8u6RhJMyTVSjpHktx9uZndLGls2O4md1+ev9no+Dr6xjby67rrrlOXLl301FNP5W2aF1xwgX72s5/psssuy9s0AQDoiApaQLh7taQfZogvk3RkhrhLuqiZvgZJGpTtHAFk39KlS/M+zXXr1mnevHl5ny5KAz+SAEB8BT8HAgASivmwKwAAEKCAADqoYv9FtdjzAwAAmVFAACgaFBUAABQ/CgjEwqElAAAAkCggYuOXUSD/hg0bpo0bNxY6DQAAEEEBgVgooFAIDzzwgKZOnVroNAAAQAQFBNBBcdgZAADIBQqITqwz7VVgYzrAcgAAAFuKAgIAAABAbBQQaLeOuAejI84TAABANlFAACWKw5EAAEAhUEAAJarU95ZQAAEAUJooIAAURKYCqBiKildffTXldV1dncaPH5+1/j///POimE8AANqLAgJAQRTrRvT999+f8vr9999XRUVFu/q6/PLL9cknn6TErrjiCtXU1LS5r2JdXgCAzocCAkDRKMbDsjZt2tTucauqqlRVVZUWb2ho2JKUAAAoKAqIDHKxEVOMG0ZIx6+8+ZPL/4np06en/fJfrDZt2qTRo0fndZqLFy/W3/72t7xOEyiUBQsWaNmyZSmxyZMna926dQXKCCh9FBBABIVe6Zk/f76ee+65lNgdd9yha6+9tkAZtc2UKVN02223pcRmzpypoUOHpsSyuW6OGjVKjzzySNb6A4pZRUVF2v/YZZddpldeeaVAGQGljwICWcEv9yiUt956S4MGDUqJ1dbWFiibtsv0v/PSSy9p4MCBBcgG6HgWL16sOXPmpMXZAwG0HwUE0IxSL4oy/WJdTHtYSn35bon2vA+deXnV1dUVOoW82LhxY8rrZcuWaerUqa2OV1tbm7Z+rF27NuV1XV2dhg0blhJbt26dZs6c2a5cP/vss6ydy7Ny5cq0ec+HYvo8BEoNBUQBdJYNgc4yn2ifQnx5s8GQG3369FFjY2PO+j/uuOO0YsWKnPVfDCZOnKjjjz8+Jfboo4/qkksuaXXc//zP/9SoUaNSYieffLKWLl2afD19+nQ98MADKW2GDBmiCy+8sF35Xn311Zo4cWK7xm3qkksu6RSH1E2YMCGl6Kqrq9Nll11WwIyA9qOAAEoUBVrzmls2nXGZuXtWTyhv7pfi1pbtZ599lnZex1NPPaXhw4fHmm4pHZbWHpku7Ru3aGpoaNDcuXPT4uvXr29xvJUrV8ZLrhn19fVbNH7C4sWLNWPGjKz01ZxiuO/MNddco88++yz5et26dZo8eXJecwCyhQIi1BE3LKLzxC+vbccyy7+O+H+YSWvzmc11r6amJu2E8rjL+c677tIbb7yREjv++OP16aefprVtLee//vWvaed1PPXUU3r66adj5dLRbel7XuyHLCKwJZeFBooJBUSR6CwbTp0d73Nx6yzvT9wNyxFvvpl2aIwU/GLcVp1l2WYTBUDHw3uKjoICAiUr1x/EudjgiebMF0nxYmM3+1jfS8OqVavSDm2K+/+Q6aTqXJ4b0xaZ5oF1Emg/CogixkbMZsXyQZ/N9yTX72+m/jvLOpWLw0FaU1tbG/tqQcWyPrdXqeeP5t1yyy36/e+vT4ldfvnlaYeyNbVu3Tr169cvJfbxxx/r9DPOyHqO7dGZPw+BXKCAALIgW8cfs2FWHNqzYfHb3/5W9913X8Gmn4++0PFVVVXpiy+mpMQmT56c8byXqA0bNqTFZs6cqRXLl2c1v/bK1mf0qlWrNH78+GykBJQ0CogOgo2E4sMvXtnX0NCg119/vdV2hVjOkydPTrnCCtDZdcQfRIYMGaKKiopCpwEUHAVEBmzkFYdSeh/i5traF2opzXMhLFmyRPfee2/O+k+8P6V4fk1zOuJGHIpPZ1nPWrs07paaP3++nnjiiZxOA8gGCohQZ/nwA1Ca2vsZxZ6w0pXpfZoyZYpmz55dgGyQD++//77++te/FjoNoFUUEDEV+xduqRdArR2fWuzLHyglpf55Edf8+fM1f/78QqfRbpnep6uvvlq33357AbIpbblc56dNm6aqqqqc9Q8Uo60KnQACubgJUCltJJR6gVBKyzqujjhPCJT6/1tcN954ozbV1+svgwYVOpV2yfQ+1dXVxb5DNfLj5ptv1rp16/Tiiy8WOhWUAHdP+35tbGxUWVnqb/oNDQ3q0qVL8nVVVZXq6ur0ox/9SJJ07LHHJtvEOTcw29gDEWrPxlJn+RJG6+KuC/m8AzGKV2vvczbXg858h+I5c+Zo/rx5hU4jL7J1qFpnWTek7H2Hr1ixQmvXrs1KX+j4jj76aNXU1CRfL1myRH379k1p8/e//11n9++fErv99ttTTuBvaGhQZWVlxvuv5AMFRKgQ1+TPp1wcB83GMJA/7f1/LfRnDzJbunRpyutieZ+KJY98yNZ3FN91aKvVq1cnn69atSpt+NSpU7V4yZKU2OLFiyVJffr0kSR16dJFFRUVKXsp8okCIqYt/RWvo3/AFGL+CvFFl+vDyjrTl3dHlK09UejY3F1nnHGG6uvrs9Znob9jWKeB+LKxF3rTpk1qbGzUpk2bspVWm1BAhKIffoW4akkpXimlI57k3FHmoy2am+dS3iu3pVcs6ozrwZZimcWXWD9zvcwKXVQUkzjLuk+fPpxfgqJQCv+7FBChbJ4DketfmePmWugv9EJPPyruMiulPQSl8AFTKO3934zbrpjWg9ZkmpdSv9P1+vXri+JSl6+99poGPvpoSmz69OlatmxZSmzSpEmqq6trc//8j+df9NASAM2jgMhgSz+0M32hZvOLoD03LSulDZ6EUt/IKRacq7JZttaDUlpmuVj3W+tz2bJl+stf/pL16SbMmDEj6zfbam2exo8fr4suuiglNnToUA194YWU2EUXXaSHH344JXbllVfqvffey06i2GKJY8iHDBlS4EyA0kUBEcrml2yuf/HLpNQ3kNvzi28u9uS0dihbMWktv2LJv1jyiKOxsVGSUq5qUUr5F0Km/6NPP/1Uzz77bLv6u//++zVt2rQ2TzOuhQsXasOGDW0eb/LkyZo+fXqsPJqeIC2pXdNsy7pX6PW0lIrqYnHqqadK2lzQSNJjjz2WFkPx69OnT/LRWXT6AiLxZl966aUtttvSD/Loh2suNlIy7W1of5/5/yJKLJPmJOYlm5cry7R8ml6HOa5i/PJMLNPWlu2WYg9H9hViY7C6ulrjx49v83iJ/8lsFbSvvvqq3n///Tb3tXDhQr311lut9t+/f/9Wi5tFixapT58+Wr9+fav9ZUtlZaWk1A3HxPuRiCX+Ll++PNkmEXv++efTYueee26L03zjjTfSphlHov3vfve7ZOy0005L6yvueQeS9MUXX7Qph4T2FGZb4rXXXpOUOp8bN25Mi/Xp00cTJ05MG7/QhR5SDXl+iJZErnbU0NCgPn36pBxyuHTp0rT/kb+PGtWpioVMOlQBYWZ9zWyqmc0wswFxxunatavKysrUtWvXZOx//ud/JKV+GAwdOjQtlrj0VpyVKLoRl/jAa/8HX8u/xifOyG/vmfmZPt/iFkWtFwLt25hNfJG3d5m1tgcj8R6efvrpOek/UyzuMotTACTy/+Mf/5iMJb7Uosvs7rvvTmkvSZ9//nlarLn8oxIbjtH8EteobmkjItP/SyI2cuTItNhVV12VjPUPr4sd7ePpp59OiyW+EJpueEW/0BOxO+64Ixnr169fyl9JuvPOO9P6T/TTtP/oF1EilsgvGusfub731VdfndZ/Yjm09NmSGPb3v/89LRa9VviZZ56Z1tdTTz2VFrvgggtUUVGRNk+J9SMaS6xHUubPm0yfPRMmTEjpI/E3+mt9IhbdwE9cqSj640Gi/2js2WefTW6EJyTWzabLMbqHIzHs2muvTcYSPyR069YtGctUTCTuct10niZPnpzW/7333psWO/bYY9P6zJXLL788ZdpS5u+/xPJuOk/vvPNOi/1n6itxGFvTDevoL7WZxsv0a26m2Pjx4/Xzn/88JY+FCxemFMKJ9itXrmwx10zGjRuns88+RwsXLmxx3KaxY489VmVlZbr22muT62gidtNNN7U4zaZ9rVmzRscee6z69u2bXF+iy2LdunXNLp+VK1eqT58+aZ9LcZbt/Pnzk5cNTZg4cWJK8bpp0yb94x//0Jo1a5rta/Xq1Wn5L1mypMV5SnxnNDY26uOPP045L6Vp/3V1dWl9LVu2LFb/kvTY44+lrNuJz/7jjjsuGevfv7/KyspS/l/vvueetFhb5KqQbLp8oq+zvYekwxQQZtZF0oOS+knaW9LpZrZ3a+NluhFHnA+ITLGBAweqrKws5cMmERs9enQyNmjQIJWVlWngwIHJ2AcffJAWW7FieUos0de6dbVp/X/yySfJ2MMPP6yysjI9+eSTydibb76psrIy/fnPf07GZs+enbH/6DWFE7HorvuHHnpIZWVlKXfdfPbZZ1VWVqYHH3wwGZs0aVLaPG3aVJ9xmtEPqkceeURlZWUaM2ZMMvbYY4+prKxMjzzySDL2/vvvp/W/dOnSjP2vX5++zD799NNkLNP7O3z4cJWVlemhhx5KxmbN+jJj/1tttVVa/9Fl9uCDD6qsrEwvv/xyMvb000+rrKxM999/fzI2YcKEtPmsq6vLOM3ol0Km/BPLMfqet2U9jm4wZVpmDzzwgMrKyjR48OAW+6+urk7Jv73/X1sSK/X+o7F8Lce44913330qKyvTfffdl4y9++67KisrSzkXoL39Jz5voidNv/rqq2n/J2+99VbaF3rT/hL/E61tECY+o6Pz9Pnnn6dNs9TXszjff9nsa8OGDWmx9n4Hu3vyM+i2226Tu2vWrFm67LLLdO2116qyslJ1dXWxc7333ntTvjfdXffee68WLVqou+++p8Vxm8airxM3mIs7n03bLViwIC0WHW/q1KlpscSv54l5in7HZFqOTWMPPPCAzjvvPJ177rmqra2Vu+u+++7TNddco0suuST549Qf/vAH3XLLLSnFd9O+vvjii7T8//n+P1ucp8Qy+9Of/qTf//73uuKKK5rtf/bs2Wl9ffjhhy32n/hRLfG9PHr06OQGfdz1paX+E5oWMYlYWVmZfvOb3yRjiXOrohv3r732WrsKlKZ5jBw5Mu2RrYKiwxQQkg6QNMPdq929TtJzkk5obaRMN+KIs7Jkir3yyiuqrKzUokWLkm0SsXHjxiVjb7/9tiorKzVq1KhkbMKECaqsrNQrr7ySjC1duiwllugretORRKyqqioZGzFihCorK1M2wD/88ENVVlZq2LBhydjcuXMz9h/99TAR+/LLL5Ox1157TZWVlSmHGbz77ruqrKxM7hKXghuhNJ2n2trajNNcHHOZRX9tzbzMlmbsf/XqNWn9R38lzPT+JpZZYpe1JM2Zk3mZRa/nnojNnj07GXv99ddVWVmpf/7zn8nYe++9p8rKSo0YMSIZmzZtWvxlFim6MuWfWGbRwzrash5H76yaiE2ZMiUZS6xn7777bov9z5kzJyX/9v5/bUms1PuPxvK1HOOON3r06LTPm8T/ZvTzpr39v/nmm6qsrEz5lTDT/2Z7P6MzjffOO++k/W9m+jwr9fUszvdfNvsys7RYe7+D6+rqNH/+/OS6MX/+fJ1//vlatWqVKisrNXr0aI0aNSp2ron1LPoZXVNTo8rKSk2aNLHFcZvGoq+/9rWvtWk+m7b77ne/mxaLjrfffvulxZYuXap169bpgw8+UGVlpcaOHasVK1aouro6pZ27a+7cuSmxGTNmJL/jGxoadOKJJ+qxxx7T8OHDVVlZqWXLlun444/XpEmTkv3PmDFDo0ePVl1dXdo8HXDAAWn5H1V+VIvz1LVrV61fvz752TJv3jx98MEHGZdZr1690vo69thjW+x/4cKFWrt2rV599VVVVlaqurpaL4QXQ4i7vrTUf6b3N7GnKFO79hbkiT3VP/7xj1scr6mmBUV7daQCYldJcyOv54WxFGZ2vpmNM7NxNTU1GW/EEWdlyRQ74YQTVFFRoX/5l39JtknEevfunYwdccQRqqio0FFHHZWM7bvvvqqoqNAJJ2yueXb6xjdSYom+Eh9I0dg+++yTjPXp00cVFRU69NBDk7GDDjpIFRUVOv7445Ox3XbbLWP/0ZU4Edtjjz2SsWOPPVYVFRUpK+1PfvITVVRUpNyO/bvf/W7aPH31q1/NOM2dI8vs5z//uSoqKrT//vsnY4cffrgqKip0xBFHtLzMdtopY/877LB92jztvffmHVSZ3t8DDjhAFRUVKb8ANLfMonsgErFvf/vbyVi/fv1UUVGhAw88MBk75JBDUg4ZkaTvfKdX2jxtu20zy2znnVvM/7DDDlNFRYUOP/zwFts1tx5vt922afO01157JWOJ9eynP/1pi/1/61vfSsm/vf9fWxIr9f6jsXwtx7jjlZeXp32e/fCHP1RFRYWOPvroLe7/qKOOUkVFhQ4++OBkbL/99lNFRYWOPPLINs3T8ccfr4qKCvX6zndaHO/ggw9O+7zZe5990j4PSn09i/P9l82+ttlmm7RYe7+Dt9lmGx144IGqqKjQcccdp549e2rkyJHq2bOnKioqdNJJJ2XckGxumk3XMzNLfu9E3/NM4zaNxWnTluXYNNba8vnXf/1XdevWTXvttZcGDBig733ve9p+++3VvXv3lHZmpl133TUltueee+qnP/2pBgwYoP3220933XWXzjnnHB122GEaMGCA9t13Xw0dOlT77LOPDj/8cA0YMECHHnqoDjnkEG299dax5mm77bZrcZ66deumr3zlKzryyCM1YMAAHXzwwTrggAOytsx69uyp7bbbTuXl5RowYID+4z/+QyeeeGJO35PEYViZ2rW3IM8k03g54+4d4iHpF5Iei7w+U9L9LY2z//77u7t7eXm5T5kyxROqq6u9vLzcGxoakrHHH3/cy8vLPers/v29T58+KbHy8nK/4YYb0mJvv/128nVNTY2Xl5f7qlWrkrFXX301rf+rr746LVZeXu4XXXRRWmzIkCHJ17W1tV5eXu7z589PxsaOHZvW15133pkWO/200/ycs89O63/gwIHJ142NjV5eXu6TJ09Oxr788ksvLy/3+vr6ZOzpp59O6/+Siy/2k046Ma3/ysrKtNh7772XfL18+XIvLy/31atXJ2NvvPFGWv//93//lxY7+eST/Nprr03r/8UXX0y+3rhxo5eXl/uiRYuSsaqqqrS+Bg8e7P369U2J/e53v/Ubbrg+JXbmmb/0V199Nfk6scyqq6uTsfnz56cts48//tgvvfQ3KX3deOONfv75v0rL/5prrkmLjRo1Kvl67dq1Xl5e7rW1tcnYpEmT/L/+6xcp473++uteWXlbSuymm2/0v/zlL2n9jyaNmgoAACAASURBVBw5Mvl606ZNXl5e7osXL07Gpv3/9s48zM6i2P+fzgJkIYAQrmKUgCyiiGyCKCAYQORqMAIiwg9EcEMELouKCAYREZRNEKJCIAIisgVkSRRCwhLCnpDJnpDJZJJJJutkZjL79O+PbzVvn5Mgw3JvSKjv85znzNunurq6uqq6ut9+35k5cw2d/eMf/4jHHHN0SdmtI26JI/56a0nZsd88tmRMYozxuuuuK/GTjo6OEruIMcYVK1bEp59+uqRswoQJ8eFHHi4pu+HGG+LYsWNLyoYMGRIrKipev25vb4833nhjyZg0NjbGZ555pqTeSy+9FO+86+8lZfPmzYu1tbVl/RwR582bV8L/0EMPjUuWLHm9rLOzs8RXY5S9v/rqqyVl99x3b7zn3ntLyk797nfjQw89VFL2l5tuKtFZe3t7fOKJJ0po6urq4vPPP19S1t7eHltaWkrKKioqYk1NTUnZ7XfcEZcuXVpSNnny5DV4zZ8/v6Sstra2RNcxyl7K6aqqqtaQY/To0bGtre31646OjpL4E6PsM30SVqxYEZcvX15C19DQUBLbY4zxW8cfH6urq1+/7uzsjBMnTiyh6ezsXINXS0tLbGhoKClbtmxZic/FGOPixYtLbCpG2VW5XPk8EWOM48ePj3/6819Kyqqrq0vsJ0bFjRUrVpSU/fW222Jzc/Pr1wsXLoxfHTy4hKaiomIN/mPHjYuPPfZ4SdnLL7+8Rpu/vvTSEntva2uLI/761xKaVatWxeeee66krK2tLTY1NZWUPfnkk3HWrFnx7aC1tbXkuqOjYw37+fOfb4p33313SdlZZ/1PnDBhQklZ+RitDQ0NDXHGjBlvS9Z77rkn1tfXv36d5rXctmfMmBF/85vL1lbd8Q7R1tYWOzs735SunKazs7MkpsYou1u5cuXr12uLP10tO+KII+KXvvSleMQRR7xluqOPOqpkrl5bvTcD8GJ8O3n326n0XvwA+wGjs+vzgfP/U520gLj//vtLAm1ra2scOXJkiYKXLFkSH3+8NKhOnTo1vvjiiyVl48aNi5WVlSVlDzzwwBoT+p133llipCtXroyjRo0qqVdZWbnGJP/cc8/FmTNnlpQ99thja0xsI0eOLAmGzc3NayQRixcvji+88EJJ2Zw5c+LcuXNLyiZNmrTG5DRhwoSSoNfe3h6ffPLJEpq6urr48ssvl5RVV1fH2bNnl5TNnj17DflnzJhRMjF0dnauIWtDQ0McN25cSdnChQvXSGaqqqrWSHimTp26xiSfJ/epzTw5jlEJw7Jly9YoK5/E1jYRlbeX5H0zNDY2lgSqGGP84Wk/iLfddltJWfkCIsZYMkbvFA0NDWsE1vKkItHl6OjoWCNhWBtaWlq6FNzXd6zNDt4u3g/6cjg2ZJQnpg7HW0FdXd07nuff7gIiqO76jxBCD2AmMAhYALwAfCvGOOWN6uy9994xP2fvcKwv6OjooFu3biVvwjrssMM4//zzS44sORwOh8PhcLwRQggvxRj3fnPKUvR4c5L1AzHG9hDC6cBooDsw/D8tHhyO9RlrO984dOhQ9thjj3UgjcPhcDgcjvcTNpgFBECM8RHgkXUth8OxLpA/ZOpwOBwOh8Pxv4UN6S1MDofD4XA4HA6H438ZvoBwOBwOh8PhcDgcXYYvIBwOh8PhcDgcDkeX4QsIh8PhcDgcDofD0WX4AsLhcDgcDofD4XB0Gb6AcDgcDofD4XA4HF2GLyAcDofD4XA4HA5Hl+ELCIfD4XA4HA6Hw9Fl+ALC4XA4HA6Hw+FwdBm+gHA4HA6Hw+FwOBxdhi8gHA6Hw+FwOBwOR5fhCwiHw+FwOBwOh8PRZfgCwuFwOBwOh8PhcHQZvoBwOBwOh8PhcDgcXUaIMa5rGdYZQghLgHnAVsDSsp/fbtm7yWtD5L8u2lzf+a+LNtd3/uuiTe/Te4//umhzfee/Ltpc3/mvizbXd/7ros31nf//Vpvbxhj781YRY3zff4AX362yd5PXhsh/Q+yT6+y9x9/75Pw31D65zt57/DfEPrnO3nv8/y/afCsfP8LkcDgcDofD4XA4ugxfQDgcDofD4XA4HI4uwxcQwp/fxbJ3k9eGyH9dtLm+818Xba7v/NdFm96n9x7/ddHm+s5/XbS5vvNfF22u7/zXRZvrO///iza7jPf1Q9QOh8PhcDgcDofjrcHvQDgcDofD4XA4HI4uwxcQDofD4XA4HA6Ho+t4J69wWt8/wHCgFqiw648ATwDTgCnAmcAmwPPAJCu7OKvfHXgFeMiuK4HJwETs9VjA5sA9wHTje6z9nj6rgPuNdwVwp7V5JrACaAdqsjZvB1qBFuDfwBbAGKOLwN5GN9nKmo3/7UCjXU8E/mVt1QI1Vncr608H0GR0R5ie6q3NKcAVwGtAm9FVAsuA5cDq1H/gx0CdtVkP/MR0/BTQYPymA7ta31tMjj/ZOCyzsgXAKKDKeDVZvb2z8VpkdXcD5mayVQMnGt0i47fYeK3KaOab7E32qQL2Nz0mvc0A+lv/Go1XBbCp6SDJf4WN4VKrl3T+gvWp2WS4z/Q0ycpbrd4S49VkMu+DbHCxlS8BzjadJnnrkb3Myeo9DZwAvGxl84EewHbAcybHQmAj4HRgtsk/2mzob0azCtnAcJO1CdlM38wPlgDtdn2rjcFqNP67AwH4DYUtnIHsINEsNHlftrLlwA7Wj9THpSZ/FYU91iEfqDXeEZhqciTbazLdbA6szPjVmU6Sz863+nUU9rMa+UClyZj4XYFsIPFqMR4LKeyo0cZu/2zc64HBwAP2e6uN7WnWbjTd7Qf8wXSf6h1q7a60sgbgqxQxptbqHwVMtT40AzOBY5Bvt9jndmR/iVer0R2QydoE/Bz5Wuprm/VxssnUgWJBlf0eTa5LszFJ9a4sG8924FqKWLjA6l9tdK1G12FjPxH5akcZ7yagE/nHjEzWDuAq4Hc2Js3Wxi+sr6utD3ejGJn8917khy9ZWTPyq77ABAqfrgK2BP4HxcVl1s80ZyT5V1ifppjcq0ynSzP5VwMPAjdkZUuATwDX23Wz8e0BnE8RQ2cAP6OIj50oNj6eyf8qspPRlMbQbdBcU0ERQ7cCHqXwgQXAMKOpsT5MKRvL5TY+V1DMRa3I/j9tfem0MRmDfDbNZR3W/ynAI1bWaZ8Lja7druut/RUUfldv7bdRzItfNPokSz2y2UnGv83KpiG7yfmvsDqNpuN6+7vDyp8wfdZl/OfbOEzMylbb2I3J6s62cU++H61efUazN/DRTKaY8V6Q8X/VdNaQ1Z2B7DBS2PNsCrtoRv/7agqFj7QZj1ko9rRbeTs6Iz/Zrjutzyk+JPtemfFK+cauWZ1oNJOtbrvJtsz035TRJrqYyVBnn1bjv4xi7mu373kme86rE/iV6S2V1aN5eHpWtgrZT/KxicjPkwwtpqMKG9NUtsr03VDGa7rVTXqsN7qnkB+keWqp9aXCfovI9pYbXRrD5ynmpGjX95uuUtlE4CFK87k0Xkkv0XR7EXBudj0FxbcUhytM7g/4a1z/M24FDs+u24FzYoy7AJ8FfgRsD3wxxvhplAgdHkL4rNGfiRwgx8Exxt1jjHvb9bXAqBjjx1EgHW2/7w7shYxpL5T474qSsXOA7wLfQInEpiGEHY1fNxTMZ6EJ4mfALShBWF3Wt/1Q8JiJErjDgdnW9kNAb+AkNDFWWb2JKHGZY3I+goLuVKv7SeD3wLeBfZHh3QuMRE5Sa/wvQgnjOTHGTYCf2mc7lHD8Gk1UW6LJ8CKU/D8N/DcKvMcB/4WC7TLgKeO1NfABNLmcAxyGjL4d2BYtjs43utUo+bvTaLZBDj80xtgvo3nG2vi6lXVDyUcrSiA3BXoCl5v8p5je+psc3wJ2RoFsELKV7wG9rO4A69t2FAuOZpQ0nIIm7O5WbzRwPNDP+HVDyftoq/sasrsPxRh7GV0bcCNy/H2s3rNoMTYaBZwmG+/Lbazus3qnWP/vNF0nNBrNk9aPycCIjNfpRvc702mOccgmnooxTkT2cqjxGwf83fiMRMHzWeBjaOxGUiR6WwEHWT9vMPk3A66OMW4MXIZ8oA34gvE+0WRoQv8gpxeaCM9HAfUjVnYhSqoORon4VAo/uCTG2CvG2Nt8YBPkc5tZ3d+jIJ94XW992wI42cqOQQnVSOBms92zkV9sDFyCbOgR4CyjHYfsfhrwceBcqzcM+cWArOxCYAiKMc+hBUIVssXZ1t9+yB5OtTb7IRsaiia2xOuPaBFyF3CblR2LbL/evvsiX7kH2etvTAf/sHHb3eRvNn2fYHX6oomwN7CT6Sbx2tLixVeRf3YiGxuGFi99rY29UKI+3a4/A3w049WEEs15wNFWVmc2cQZwpPWp1vQyGfiO9WMbs4HdrH534AfAh80+eiHfvwX4ILCF8WpCcfYM5OuPIPwA2AX4tsmXFvLXoo2QUda/RjRJ90JJwyoU9/ezsirr0/dRrO2L/OH3yI4uB/og2/wJcCSwp/X788iftrJ6/YC/oA2cD5gNBOAma3MISkibgUNM3xcZXRVKyM9FPjIZOA8lO1saTUrQf2j19kGx6DprYxLykZTc/AzFtUfNDn6FNg/2sfGfC4y1cemGfGs6WpimWL/K+n6pyVKPfHSMjVUN2qCpN5pHULJ0i8l2PYq/z6DYMtXoJqHFUSfyi0ttDFdbuzNjjLtZ+Wyj+waaXz+E4k8zcA3CLaavJuCbaM5KC9AnUaw/3+RMc/jVaAF7r+nrePOTnyG/7AQus/n4XIr5/3jjv5giEf0m8vdFFMn4Qyh2VlqfhqF8YlM01vPsei9k4yeiPGIYitVN1vdN0RzzkvFN+ca9Nk63IN//k+n/N8g2I9pEedD0uJ/p/08oQV4O/BLNK8OQnSyzNh4ELrAxmWl9bUTz3PlobpuLFpKHms5OQfZzAYoBAAdam79AeU+tyb8X8pFaq7vQeD6N8oODTf4r0WLyFBSPEq+5yIaGWT8uMJ19Go13QGP/C5Q/bY5ifRXKi2pRzFmE7O8F4GLTY5W1uzNwMoUdnQF8iiKfa7Ex+TWKZ/VGd7CNyZfRnHmC2dB+WV56PjAuxricN8H7egERY3wSDXC6rokxvmx/p92JD8cYU3LU0z4xhDAATeg3vRH/EEI/ZKQ3G8/WGOPKjGQQctQA9Aoh9ECT7CbAhBjjv1EAaEQBHmTct9nfI4CvxRhvQ8lD3rcrrS5oNQ3aJUnogxzjx8gB09P08yhdiIAc/g+JJsZYW6a7bwC/Rcl2sqnNrB832/U/0U73xmhxNsJ0/BJwYIzxvhjjDBTY5wCrYoz/Mn1NQxND6ls9Wr13s/G6GgXNNmujJqObZm0eAvw2xphW/PPLaL6Igmk/K1uEgteOyOl7onE4GI3pPVa20uQfH2OsRGPZQ6zjfVFb4T2Ro28eY1xFYUcbm85+hyZdKHYuyOgiSnZ+Zbx7Asszu9wCJQj/NtotjKYPxS7gTaazo0wXA6xsPvA1ZCufsz5j9r09hX0/jxY/yea7IT/4KFocnEeB3iiRyX3jDBufVLZRxqu76b8Z+cRN1s+V1p85VuffJn9vlOiC+YDxTnQJzcieQD4wgMLOMf2k66tREvZGb5XYFLg2xtgC8oH0QwghIB+4M6MF+UAtmox+amVPAzshXY6IMbYCdwAD0eQKxc7jjijpAxiPkpP9KHxqE6M7ECWYP7HyvVEyhvHvRD58Royxxcpqrd7NJv/RaCGwOZrskn6q0GT1JBqbmch+jkQT5RyUGB9g/rsFUB1jnGf+22715iPfWmW8B6H4UW/XV6MFZRu6y0BGNyfGOA8lp/9O19kYDEL2MgyNXz8rW2Ly9wResvjagPzuIGuvN0o8NqaIl71QTGyzv9ME3MP+TrF646z8t2jiDVa3G7CR0fUyub6PkqXeGU2vEMLmaNNijLXfP5sL2oDVMcZXTZdjULLYE7jJyl4BesUYX4kxTkV+8yXgsRhjvdGMR3PH+BjjaiubhxYZE1AyeR7yma/bGLRldCtRDLoMJfbHAc/EGNOO7ObI7uuQTSxBsWcjZD8fQfNWg/39NZOnw3iPQHGqH1osNCEb3tXofkexuPyItbfC4usItGNfZ/rriXyjL0rkoIgTe1l7q9Cc9jXgk5TOqXvamDZbn0YgX1xm4/s8QIzxCrTB1WG6mosS0XRHrNXobrM+pdjyVxv7lqzsYRtr0KL3NZQ0LyaD8drU6s2ysmGmq4Sd0PzUbu10R3Gk2XTzODAgxvgr+y1toG2K5uMtrWwGust8pf3ejmwl6TmafUxAc0uKtelOU9powmgGGK9dTJZ/WllKmhPdvsiX60z+503Opkz+PijZX4FiziST5xrkl21WJ8YYRyGb7bR6i6yvC6zNPhS7+aCcaJrxTnNszutAZD+tVnYXRS7UB905/yPKF7pZWQ/r4wEUdxfuQfazNdoUSLY8gOIUQjuKs4dQ3CF8Ec0t36XI3XrbmP3YdNto+mlA4zmN0rlugtEvgNL5DPn2nXQF8T1wlGhdftDEXfEG5VUooHVHgaEBuNx+vwcFo4MojjDNRY78EtqR2h0Z/60oyN8E9MnaGI52cc803ktQMrELmqi3RCvT1cB1VmdlLjMKokneRuwIU9435Kgn2HUtmswrrGy4/V2JdquGosmj2X7bwvp+jckxDvhMxv81iuNag5BTzUeG+RLa+QPttnaaPleW6XhFJvME5Cj9ymgeNXkvRQufVpSEDUYJzEDkbAOtD5XIaRrQ5DIRreJfsb4dlPFfbOO2i7W10Hh9Ek28rxmfx+17NoU93GD6SzbSCfwhFkd7Et0iNDFMRIGqEe1enGe6agBard4IiuT3BRQQl6GFUToysmPGvxntrnRHk0pEAW6qjdkJyE7nWlkDhe0+ZvIne16EEqrcvh82/YxFuz/L0YTR29q/0ujSEaZK+8wxWTdGwfCP2F0qtAub+L9i7Y1F9r0EJWb9bBymI1t6nOJ2eu5nKyh8rx64dC3+OMn0MNf02Gp6n2dyLjZelSZDC8XEuoVd19i4zUQ7Ton/DKDS2qw23q0m15dNpsetn6+a/K0UMeGz1s9bjXakleWxYx6y63ScstXG4csmz1Sja6S4Fd1iNP+kOFrYaP04OeM/03T+WZOvwfg3ovgzHi0YhqOdxXrT0XDg9LI4VANcURZLh9u4nWDXl1obC9EdvOTDw5GdpzhUafI/SxGHXrHyPA49DMyzv5MPN5iM21Lcwl9CcSypkyLefsTkTkcR7jFeeVxejib/M5H/diJf6W06SUcL26zucxRHaKahOwbpWMwyFM8uMf6rgCqrdy3F0Yk6tBhsQ7G1t8m5wn7fksIHO7PrOrTbOzMrW45seSa6s1qNbOJu5Ad/MrpmK1tmekzHTtLRkkvR+C9GNrkl2uVtQHdLKo13jcl0K7KfBhRrayiOCa1EfvFasiHra7obNdRkLJ/zVlMcHx1CcWQk/T3OeLWiZKkd+UwbxXHMZSjetNuYDDR+vSmOokxFMTrdNTnZeN5hcuxDsVBIR4dvRgvdRpP/+vL5GS3Wn7b2XkLxpwHFn0bkg31TfWs79esAtJHUhvziZbRxkPOfQ3FXPh33akQL3AXI705E82cLiuUpR0jyVyBf+9dacolrM/n3tfFsNvmnIFs7JZM/HaWZlsm/0mSbm8mf+C9GC8cdKGJxM6V3aSrQ/JF8+TWgv8k63/RTC/zSyq6nOIb0SxR3r6c4TrUC2U86dpyOkz9qNO1W9qj1o8N0Osl0mfhPMV4fQD7VSXFE7FdGvxT5yaEUR8j2t7LjbYxm2/UVFMeHU552EbrjOxz5YJu1eR66s99IceriRopjZ60oVtyG8rlVKCa8BJxoekqx4k2PL0U/wrR2hBD6oknhrBjjqhhjR9StnQHAPiGE09BRnZfKqn4+xrgnmtR/hJKMPYEbY4x7oIH9mbWxEZo4/4Um5+3QrfQ+yDEuR7ttKZls5+2hv9W9w65rY4wfQbuNv0UBNseNaIU9GwXOK9HqeTPkpOcB/7BdS6w8rVZPABYZ//9BjvmjEMIrKKA0RtuBzHWcGrayTwJXxhhXZTRPmQ7uQDtgNSb/qej24OVGtxw56o3oOMJqlKgOtT5sjRz6IuCWjP8rxvuHKEjVIAe7Fu26zbbP9tguS2YPuwMbZTZSDewZQtg1K7sXOfkyu+6PkreVaBdhW+PVLYSwK7KRXmjHeYCNwcZol2JLZEd/z/g/ZWO8C5rkD0FBYToKxt83nbSjANFRZrt9KbXn/mXXn0JBeWqMcTCaABtMP/9FsbtOCOEraGLeDh3l6IkmpW7Ifr6PAtpuGf8Po8Tmo2jn9Bhr7yoU1Jegnb9dTQ8NZX7Wg8L3XgWOCSEcmJWNpliIfj7GmPTaDU3CS9Hdlx8Z/8Gmp+3QJHY7mujutjLQ7ffE/1m023ygtXWc8a+zsUvHbDqtH92AliwmnIoWfzfauDVZ2Z5Wdg+ynwFW9r0Y40ZoIrsMLSZPN34R2f751ofbrF4vbEcRJSLXZvwfN16nmo5/bfzHoUT0O2ij40SUnLSaDgabToDXY9pWaFGalx1r45li0MXIn29B/n8BSqYHUxyhuxEtXjpRMn4lsqWPo6QlxaGN0N2r4VbvhxQ76eeh+LkYbUxUWf+7owk2xdtvoR3J7UzO3iGE71HE5QfQBL2LlW1j+lxtcg5CPryNuhy+Z+P1Xyg+9kU+nZLKp6z9Hxv/8UCN1TsOHYXZxGT5HUoiHkALloWUnvUfZX3ryK4bTd40h0xBtvi8lR2EfHEyso0W0+ko470UHX1YanzmGc8d0N3GO5DN9KaYo15Ffjob2UcVxR3b7yC/+iey/WQ/IH8oxxvdBSSEcDDFhgQxxvujjgeD7kJ/iCwemRwzo44gN6Px7o7G779RHNuEwq+/SjHX9kUJ1zYo2fqV/bZpCGEiGr9lb0V+NIdcjo7ngRLiY1GucBq6Y/O3WNxdbqDYBDobPZeW7or9AiWeQ1D8As2fq1E87W2yHG79S8cYVxuva9CY9aI0R4hoDug0Pgn9jfYrmfyHIH/fw+QfiGwg+WMDsqMx1ockf1+0OfbJTP7+yA9rY4wV6MjPxihp/6jJX4MS8w+YvOORLfyL4khtFbLZV9GdV9DcOQPF6tORjfwezZFD0abNOSjGHESxkfug6eASNI9fh+arFjSOo9FR0nuN/ySTPz0TtBzlG39FC6Wz0dhtg+bH9CwJyBfOpvSkSA8UX1Ketgnw/1AecDWynwWmm1+gjZZFRtsdjX0Nik81yP4/Z3w3RvHmS8CFIYSdkP0/E7twfAnwOxCU3YEwAxoNnP0G9L9EwbMaTQKLkEPeXkY3FE2UlVnZAcDD9veRyOiPQeejE82JwA1l8i0BTrPrGSjYVKBgOSOjK78DcY7J1ru8r2iHop1ix7QdOd4HKXYD0vcodI4y1Z2DnP1jVm+Ala/KaIJdJ31eBjyf9eEJ5CwfsutENxvtoqTrvyGn752PDXKIWWiXIT3El/owIKNLfRiNJtCzsz6MQedHF1MkfKleQMeo8jH9g/FfCvSwsr+kMbDrShRYzs3sZSSyh3PL7Gg4CiCV9ukElpa1eQvaCZsODMzqNtnfW6JJ7BKUMM3JaJ5BCUElstNm62NHWVk9hT2nHZG0i1FPsftSXVavg+JBt/QQX8ta6FZRHE1ZRPHgVyXFLdg5xifVa8LuVGW6uA0F5Rno+Q/QJFCb0YxF5++T/k9C9nNprn/77Wprv5bizkOHjfEHjeYaFHhHUdy1GopsoD8KxIvRYudc02/I6JopjQE/sHGegWz/AHQ8qC2T/1Q0KVdm8qeJOud1NMXDnZUU55k7M/kPQLuFnZn8B6C4UJXJ/3Xj35nJf0AmV4pXOyE/WgCMtd+SDx+J/DCPQdeYXntnZYlX7sOLKR6KTHEo0Q1E9vcy8ELGZ47pp4UiBtVl9YL18+YyG8ofyD8RJeDVmf+ehuLAzRT+e1Iqy3hdSnGnKum/08Ykp7sP+dF068uJxquVwn9PsbJVWb2z0FHF3GbvRIvM3AeuBRZnNPPQcU0o7OcKbA7J6NIzCcn+k/zLMfsxuj+ihcsolFz9xnQ0x8ZpMUqs0osAAsW8leJUmrdmog2bGRTPYc2nsKFWtDFQgfl2VneWyfiatT09s790RzHdAU93iZabPhL/dIyyNitbiRZ4FWjBvJDirsABGd14ZDvpbkOgeLg3lY1GCfHa7kCsNlk+T2HTY7O6zyJfXkgRj9IOfKIZa2PZlpVdaGPSiBLfnxv/9JByonvOeOW+mB4+753Jf63J+r1M/nOs77My+atN5lQ3PQ8yL5M/PSzeO5P/YWwzK5M/3SW9Dvi5lT+dy4s2DdOdum0p7uDUocVcnp9MR/G9JtP/LJNtZp4PZbzmm6yV9nd6ecO8RJPFmCVZvZuR7adcab7RjUG2muJcK/KVlG+lEwUdxi/No232aad4gL+aYt6sNbmq18Kr3WgixYPkaV5tz2hW2m8p1t6MctH7gW91NX/2OxAZbFf9ZmBajPEqK+tvZ1QJIfRCE/nVMcYBMcaBKLEeA3w/hLCp0fVBCfqzwPwQws7WxCCUDEJxzqwK+GwIobe1PwiYFkLY2ui2QUc50i7/gyhxAE0OD7xBXw5Hycq8qHOqIIdJ2BlNjPujoF6NdiTzHaEhyClGorPX2Cp1I2TA+6Od1HRmuRatcEG7gnOTPtGqfZj1sR3oaTpOfUh0idfNKKDsjnYmm1AASWMzGK32RwF/ijF+IOvD1Rld6sNmQGeM8SrrwwfRDkUFCjbpDTD1Vu+LwNwQwg7W795op+hRFNiONnv4CvBMshHT34HAwhDCj9Hq/mS0E1AdQtghs6Otgb+YHe1i7Z+R7MXoDkPB51Hgi1Z2lPHfHDn9o+h2aBWwWQjhU8b/RfTQ/kC0a9Fg+rgPTZjfRAHtp5k9LwHGZVzYXAAAC15JREFUxBj7oF3IeSgZ+xRKQAdavQXID7rHGNMuSUfUg82fyegWomNeV6KdmG+aXC8azR02bjujoHaY0cwExoYQtjNdbIF2R9IDvCeZnx0DPJ18D+1w7gvMCiEMQTtN30Q7obNCCLsbvz7o/POcGOPWaCdsKvacSIxxkdEcWab/Psj2IsUDpzPR8Z9Z2FG1TLb5wKIQws4hhG5oh3wl2oU+yfQWE431YR+UZDWgyXUw8rVXgCUZ3XdMd+OBL5k+03G5HYwmHStsNHlAz2uks+3fRva/G8XxvhOM7lRgmcWi49Dt8V+g88mrKJ4ZSD58HMVZ5hSDTgYuSjHIXgaRYt9gYJLp/wk0CedxKNElH07JVR6HTgbmZjFoITpmdCfFw6Cft/jaDdlBC1q0HW362QTd/ettPA5Eu/OHo2T2OOM1GTgwi9VHohgyF70taTuT8UZg/4xuD+vPQ8ZnEMUxpG9ZeWqzdwhhN6t3DFAVQviE9XkH09nFaLF3kj2DdJzpH7vuD4wOIRyL7P+HVu/OEMK+Gd0xaDG4q9nOgSbTZ9CdpER3NFq8jUQ+83WrtwmKz3NR/LkTxZMvoHlrM2CG2U+at/qjBC89QJuenzjJeC5DixSM9xSjOxntQj+CYsBjwJamp5OQD25q339Ec8BUlDTXG81zFG+j6W0807MHO6LYcRB6vuoxux5kdf+N7h5EtPAE+cdzaKe3p8WqHSmeu8jRD+34/jHG+IyV9cTm2xDCtsa/GRhs43EN2kBqN5rtjf+DJtvG9qzMFyieETwE+SnWr5S7HGn6rMnaTHcpO4AOk3839MKEedjOuPnxaVbvZyb/F9BYDokxrg4hnIgS5dkxxm1N/odM33OMZnt0h/MTWZ96IHva09ocAvzd2tzZ6jdZPN0HaMticHqofQWy8ekhhKMpNr12RvFyR2SPH0V3Cl61sm1NT4NRzNwELS4GokVaDYq/TUYzPYRwlI1RL+QLs9Aic7aNyQHAi8Z/IbrTMpji5Q9HW51qihdszEdzYrXpfgpasFajWPJLFLPTsclPxBi3QsedUu52PEXcn2K6vAvdVZ+NYlM1imkLUT7Xgnx9FZoz59u4rjWnXCu6utLYED8o4NVQ7I5eRvF6tHSO+DSKs8sVaDLMeRyEHGV7FDjS614vsN93N+N41YxlC+S0y9AbXUATwnTjfxsKNE9RvLor3Sk4heK8bUpg0usw0w5wh/HJXwuXVr5N2fVEk6cma+NMilegRqM/2wwxr/s70116gK4aBdKlGc1r6O5B2pVebG2eS/FavRb7vjCrl1bMqV56VVxK8JspXsF5Sdl4tSJnS7InuospXpGWXtWW6i1HjpPkyl/B+VvrW4t9nkIBdgql53D3otgNiMgh09szWijOX6cz9M2mj4eRvSTbajH+6RV1zfb7vkaTXuk5HwWilGRWokkznQ9vRsFkLNqNn4YWF1PM3ran2EVeiOztDIpXZDajXbp2o6kz/aYz0XONtl+ZH6RnIMaU0fVFE9nDyC5WAJ822onAc/b3EKs3G/nHFyhe19li9bensO8WlIzsYW0l+2lACWJuPzVosbIy0+1sG8vXfdbkzWmmZfpfQfHq2+Ot3nK7vsBkm03xytD5yDauyPqQXpX6ivWhAR0VuiCTvxnd5azK5K9Dd6weo7DtGjQZ5zFmNVrULsvqPYomjvT6xZVoUtudYicrxaZvU7wGdbmN67nILmYjn0jnZMdSvA3uFIrXGy5GO5lzrCz55zBrJ93R+ifaWHg9HlI8i/W3jO5B0+0yFIvS3YgvW3/OyuxwkNWbjGLSXmiToZXCX39tsq2m8NXRFHFvtek6nW9PuriY4lWpzciWt6I0frchf5pbRnc5Gvt0/n+C6SPdvUhx/y6KGLcIHdlKMagZJaCgzalGK3/ZrtMOaor3aTczvTFnGMUZ7yar82EU16ZSvH1oK6uf6MabvFMpfd7oVWRL1cAgk2sSpbugi9FcUZ+VpXkrn8s6kR+NoYijaSzSPJPoVmdjlfgtz+ol/vPK6s1FNlve5qSsrAPtaqe3MCV/bKR4JWlqY2nGP8X6irKy5BN53XSmvvy6M6tTYfLn9ZpQnMr5pziTl600feQyxDJejZTeOV6brpNsnWV1c93n83Zndp30nNerXwv/9LxVLuNYSvOX1Gb5mKeHjJOcSyl25PM+5brtpLhTlbfZQGEDKT8qt7sFZW12UjyTlGTrQHdr5q+FbjxFjpTykXRHvoHilbVLM37piFOux2oUC1dlZXPRIjfP59IRxI6s7EXgc7G4+zQD2dpZKPb//a3k0OlWtcPhcDgcDofD4XC8KfwIk8PhcDgcDofD4egyfAHhcDgcDofD4XA4ugxfQDgcDofD4XA4HI4uwxcQDofD4XA4HA6Ho8vwBYTD4XA4HA6Hw+HoMnwB4XA4HI4NEiGEyhBCDCEctK5lcTgcjg0JvoBwOByODQRZwlz+2f09INNC+0eIhBB2T7KtK7kcDofD8fbR481JHA6Hw7Ge4SH0z9ISlrwR4f8hPoT+Y+xV61qQd4IQQs8YY9u6lsPhcDjWJfwOhMPhcGx4uDnGeFb2WRBC2CmE0BhCaAghbBdC2CiEMMXuBBwLEEIYGEK4O4RQE0JYEUJ4IoSwb2IaQhhr9JeFEJ4MIawOITwTQti2CzJF4KchhN5r+zG7WzLQrofa9a12/W27nhRCuMr6MTWEsEcI4ZIQQl0I4bUQwmFrYb9nCGFiCKE+hHB/CGHLrN39rV8r7C7J8PS76SPJ9YMQwkLgX10aAYfD4diA4QsIh8Ph2PBwSgjhmvQBiDHOBM4D+gB/Bi4CPgH8LcZ4VwihDzAGOBqYaX8fBIwJIXysjP9PgPnAUuBzwK+7INPdwNbAj95h3z4F7AtMA3YBnjCZJwDbAcPXUmco8LLJ+zXUf0IIuwKPA3sBo1C/TwbuDiGEMh6XAo8C49+h/A6Hw7Hew48wORwOx4aHr5RdnwUQY7whhPBV4HBgEFBNkdD/N0rAXwMOijHGEML9KOE+Bfh5xm9YjPFHIYSTUcK+RxdkugvYFS1inn1bvRIagUPQIuIJYDO0iJkPrAI+HELoH2PMj21dGGO8NoTwaWAi8PUQQl90pGoj4BVgsX0+CxwM7Aw0ZzyOiTGOeQdyOxwOxwYDvwPhcDgcGx6GxBhD+pT9doV9B2B4jHGlXQ+07xkxxvRw83T7Lj+i9Ip9p7p9uyBTJ3Ax0B84vQv03d+gvDLG2JS1DZK5PrvuU1Znmn1Pz8o+TNHnfYEz7bOxle1QxuOZLsjscDgc7wv4AsLhcDjeJwgh9ASutMsW4OwQwvZ2XWnfO2XHd3a273llrNrt+62+ReluYDLwjbX8ttq++9n3rm/Ao6O8IMa4RlkZdrHvj2dlCyj6fFXZgmv7GONDZW20vEkbDofD8b6BH2FyOByODQ+nlP3vg5tjjJPRHYA9gNvRWf4bgBEhhC8AD6OE+mPAEyGEpcAQoIm1P1fwlmHHoi4G7lnLz68AnweuDyHMAI58N9o0XGLHlw626/tjjA0hhD8D3wXOtIXUUrTY+By+weZwOBxvCF9AOBwOx4aH8mcgxoYQNkUPPy9GR3VWAEehZyHOizFeHkIYhI447Q9sAowDzo8xzn4XZbsPPYdQ/r8pfgyMsPJG4BbgO+9Sm0OBE9HxqQeB7wHEGCeFEA6x3w9Ez0O8Bvz2XWrX4XA4NkiE4qirw+FwOBwOh8PhcPxn+C1ah8PhcDgcDofD0WX4AsLhcDgcDofD4XB0Gb6AcDgcDofD4XA4HF2GLyAcDofD4XA4HA5Hl+ELCIfD4XA4HA6Hw9Fl+ALC4XA4HA6Hw+FwdBm+gHA4HA6Hw+FwOBxdhi8gHA6Hw+FwOBwOR5fx/wFaFkAwUsq4cwAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 792x432 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#plotting exon size distributions by exon number\n",
    "df_exon_length1_1 = pd.DataFrame.from_dict(exonDict, orient='index')\n",
    "df_exon_length1_2 = df_exon_length1_1.transpose()\n",
    "df_exon_length1_2.head()\n",
    "\n",
    "#PLOTTING\n",
    "f, ax = plt.subplots(figsize=(11, 6))\n",
    "sns.violinplot(data=df_exon_length1_2, bw=.2, cut=1, linewidth=1)\n",
    "plt.xlabel('Exon Number', fontweight='bold',fontsize=12)\n",
    "plt.ylabel('Exon Length (bp)', fontweight='bold',fontsize=12)\n",
    "plt.title('Exon Length by Exon Number', fontweight='bold',fontsize=14)\n",
    "plt.tight_layout()\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#plotting exon size distributions by intron number\n",
    "df_intron_length1_1 = pd.DataFrame.from_dict(intronDict, orient='index')\n",
    "df_intron_length1_2 = df_intron_length1_1.transpose()\n",
    "df_intron_length1_2.head()\n",
    "\n",
    "#PLOTTING\n",
    "f, ax = plt.subplots(figsize=(11, 6))\n",
    "sns.violinplot(data=df_intron_length1_2, bw=.2, cut=1, linewidth=1)\n",
    "plt.xlabel('Intron Number', fontweight='bold',fontsize=12)\n",
    "plt.ylabel('Intron Length (bp)', fontweight='bold',fontsize=12)\n",
    "plt.title('Intron Length by Intron Number', fontweight='bold',fontsize=14)\n",
    "plt.tight_layout()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PASA OUTPUTS\n",
    "## Calculating Stats using 2 thresholds of maximum intron lengths during PASA processing: \n",
    "## Length 1 = 10,000 bp, Length 2 = 100,000 bp (default for PASA)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Stored transcript, exon, and intron stats for 1227181 transcripts\n"
     ]
    }
   ],
   "source": [
    "length2File = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/rna-seq/PASA_Processing/version_2/pasa-lite_assemblyAssembler_results/pasa-lite_ZoeyTrinity.pasa_assembled_alignments.gtf'\n",
    "\n",
    "#Keeping track of exon counts/lengths\n",
    "length2ExonLengths = []\n",
    "length2ExonCounts = []\n",
    "exon2Dict = {}\n",
    "#Keeping track of intron stats\n",
    "length2IntronLengths = []\n",
    "intron2Dict = {}\n",
    "\n",
    "#Keeping track of full leng\n",
    "length2TranscriptLengths = []\n",
    "\n",
    "lineCount,transcriptCount = 0,0 \n",
    "exonCount = 0 # for first call, set to zero\n",
    "\n",
    "for line in open(length2File,'r'):\n",
    "    lineCount += 1        \n",
    "    line=line.rstrip().split('\\t')\n",
    "    if '#' in line[0] or len(line) == 1: #skip header\n",
    "        continue\n",
    "    \n",
    "    #calculate start/end coordinates\n",
    "    start,end = int(line[3]) - 1, int(line[4])\n",
    "    length = end - start\n",
    "    \n",
    "    if 'transcript' in line[2]:\n",
    "        transcriptCount += 1\n",
    "        length2TranscriptLengths.append(length)\n",
    "        if lineCount != 2:\n",
    "            length2ExonCounts.append(exonCount) #add the count from the previous gene\n",
    "        exonCount = 0\n",
    "        exonCoords, e = [], -1 # to keep track of the start, end coordinates of each exon\n",
    "        i = 0 #intron index\n",
    "        \n",
    "    if 'exon' in line[2]:\n",
    "        #Add exon length to array:\n",
    "        length2ExonLengths.append(length)\n",
    "        exonCount += 1 #Keep track of exon count\n",
    "        e += 1 #for dict/array index\n",
    "        exonCoords.append([start,end]) #start/end coordinates per exon\n",
    "        \n",
    "        #Storing exon length by exon number:\n",
    "        if e+1 not in exon2Dict.keys():\n",
    "            exon2Dict[e+1] = []\n",
    "        exon2Dict[e+1].append(length)    \n",
    "        \n",
    "    # Get intron lengths, means that the gene must have more than one exon\n",
    "    if exonCount > 1:\n",
    "        i += 1 #intron index = intron number\n",
    "        \n",
    "        #Intron size depends on coordinates of the exons A and B surrounding it:\n",
    "        startB, endA = exonCoords[e-1][1],exonCoords[e][0]\n",
    "        intronLength = int(endA) - int(startB)\n",
    "        length2IntronLengths.append(intronLength)\n",
    "        if intronLength > 1000000:\n",
    "            print('long: ',line)\n",
    "        if intronLength < 0:\n",
    "            print('negative: ',line)        \n",
    "        #Storing intron length by intron number:\n",
    "        if i not in intron2Dict.keys():\n",
    "            intron2Dict[i] = []\n",
    "        intron2Dict[i].append(intronLength)\n",
    "\n",
    "        \n",
    "print('Stored transcript, exon, and intron stats for %i transcripts' % transcriptCount)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###  Comparative stats\n",
    "### Intron 10,000 vs Intron 100,000"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#### INTRON LENGTH STATS ####\n",
      "## 10,000 length max intron\n",
      "min length of intron =  -8220\n",
      "max length of intron =  2028351\n",
      "mean length of intron =  2615.58283005\n",
      "median length of intron =  853.0\n",
      "## 100,000 length max intron\n",
      "min length of intron =  10\n",
      "max length of intron =  199790\n",
      "mean length of intron =  2683.12057097\n",
      "median length of intron =  898.0\n",
      "#### EXON LENGTH STATS ####\n",
      "## 10,000 length max intron\n",
      "min length of intron =  2\n",
      "max length of intron =  33430\n",
      "mean length of intron =  331.267270136\n",
      "median length of intron =  168.0\n",
      "## 100,000 length max intron\n",
      "min length of exon =  2\n",
      "max length of exon =  33430\n",
      "mean length of exon =  332.823287791\n",
      "median length of exon =  168.0\n"
     ]
    }
   ],
   "source": [
    "print('#### INTRON LENGTH STATS ####')\n",
    "print('## 10,000 length max intron')\n",
    "print('min length of intron = ',min(length1IntronLengths))\n",
    "print('max length of intron = ',max(length1IntronLengths))\n",
    "print('mean length of intron = ',np.mean(length1IntronLengths))\n",
    "print('median length of intron = ',np.median(length1IntronLengths))\n",
    "print('## 100,000 length max intron')\n",
    "print('min length of intron = ',min(length2IntronLengths))\n",
    "print('max length of intron = ',max(length2IntronLengths))\n",
    "print('mean length of intron = ',np.mean(length2IntronLengths))\n",
    "print('median length of intron = ',np.median(length2IntronLengths))\n",
    "\n",
    "print('#### EXON LENGTH STATS ####')\n",
    "print('## 10,000 length max intron')\n",
    "print('min length of intron = ',min(length1ExonLengths))\n",
    "print('max length of intron = ',max(length1ExonLengths))\n",
    "print('mean length of intron = ',np.mean(length1ExonLengths))\n",
    "print('median length of intron = ',np.median(length1ExonLengths))\n",
    "print('## 100,000 length max intron')\n",
    "print('min length of exon = ',min(length2ExonLengths))\n",
    "print('max length of exon = ',max(length2ExonLengths))\n",
    "print('mean length of exon = ',np.mean(length2ExonLengths))\n",
    "print('median length of exon = ',np.median(length2ExonLengths))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/matplotlib/axes/_axes.py:6462: UserWarning: The 'normed' kwarg is deprecated, and has been replaced by the 'density' kwarg.\n",
      "  warnings.warn(\"The 'normed' kwarg is deprecated, and has been \"\n",
      "/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/matplotlib/axes/_axes.py:6462: UserWarning: The 'normed' kwarg is deprecated, and has been replaced by the 'density' kwarg.\n",
      "  warnings.warn(\"The 'normed' kwarg is deprecated, and has been \"\n",
      "/sw/lsa/centos7/python-anaconda-arc-connect/created-20170421/lib/python3.5/site-packages/matplotlib/axes/_axes.py:6462: UserWarning: The 'normed' kwarg is deprecated, and has been replaced by the 'density' kwarg.\n",
      "  warnings.warn(\"The 'normed' kwarg is deprecated, and has been \"\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(0, 300000)"
      ]
     },
     "execution_count": 67,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAaYAAAD8CAYAAAA8EwAmAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDIuMi4yLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvhp/UCwAAHDxJREFUeJzt3X+MVed95/H3Z4dAEjsFDI5NAWdwPfVm0KaOM8KOXDUhTvhh7XpS1VKH3W1IioXUBXl3q92E2WjjLImU0GoXKa0dmxYqF6UZKInraWWHkGJvKjcGhppggzPxGMh6ireOM5g0P4R3yHf/OM+Ey+X+ODNhZp5hPi/pinO/5znf53nuHc8358yTcxQRmJmZ5eJfTPYAzMzMKrkwmZlZVlyYzMwsKy5MZmaWFRcmMzPLiguTmZllxYXJzMyy4sJkZmZZcWEyM7OszJjsAUyW+fPnR2tr62QPw8xsyjh8+PBrEXHtePczbQtTa2srfX19kz0MM7MpQ9L3JqIfX8ozM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZllxYTIzs6xM2zs/8M//F5783IX3y7snbyxmZvZzPmMyM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZlkpVZgkrZLUL2lA0qYa+2dJ2pX2H5DUWrGvO8X7Ja1sllPSl1L8eUk7JL0pxSXpC6n9UUm3VhyzVtKL6bV2bB+FmZnloGlhktQCPACsBtqBNZLaq5qtA85ExE3AVmBLOrYd6AKWAquAByW1NMn5JeBfAv8KeAtwb4qvBtrSaz3wxdTHNcD9wG3AMuB+SXNH9zGYmVkuypwxLQMGIuJERLwB9ACdVW06gUfS9h7gTklK8Z6IOBcRJ4GBlK9uzoh4PBLgILCooo8/T7ueAeZIWgCsBPZFxFBEnAH2URRBMzObgsoUpoXAyxXvB1OsZpuIGAbOAvMaHNs0Z7qE9zvA15qMo8z4zMxsiihTmFQjFiXbjDZe6UHgmxHxd2Ps4xKS1kvqk9T3/bM/rtXEzMwmWZnCNAgsrni/CDhdr42kGcBsYKjBsQ1zSrofuBb4/RLjKDM+ACJiW0R0RETHtbOvqtXEzMwmWZnCdAhok7RE0kyKxQy9VW16gZHVcPcA+9PfiHqBrrRqbwnFwoWDjXJKupfi70ZrIuJnVX18JK3Oux04GxGvAHuBFZLmpkUPK1LMzMymoKbPY4qIYUkbKX7ZtwA7IuKYpM1AX0T0AtuBnZIGKM6UutKxxyTtBo4Dw8CGiDgPUCtn6vIh4HvAt4r1E3w1IjYDjwN3USyg+AnwsdTHkKTPUBQ7gM0RMfSLfChmZjZ5VJzYTD8dNy+Kvoc2XAj4QYFmZg1JOhwRHePdj+/8YGZmWXFhMjOzrLgwmZlZVlyYzMwsKy5MZmaWFRcmMzPLiguTmZllxYXJzMyy4sJkZmZZcWEyM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWWlVGGStEpSv6QBSZtq7J8laVfaf0BSa8W+7hTvl7SyWU5JG1MsJM2viP9XSUfS63lJ5yVdk/adkvRc2tc3to/CzMxy0LQwSWoBHgBWA+3AGkntVc3WAWci4iZgK7AlHdtO8TTbpcAq4EFJLU1yPg18kOIptj8XEX8YEbdExC1AN/C/q55UuzztH/eHWJmZ2fgpc8a0DBiIiBMR8QbQA3RWtekEHknbe4A7VTwXvRPoiYhzEXGS4rHoyxrljIhnI+JUkzGtAb5cYuxmZjbFlClMC4GXK94PpljNNhExDJwF5jU4tkzOmiS9leLs6ysV4QC+LumwpPVl8piZWZ5mlGijGrEo2aZevFZBrM5Zz78Bnq66jHdHRJyW9HZgn6TvRMQ3qw9MRWs9wA3XzSnZnZmZTaQyZ0yDwOKK94uA0/XaSJoBzAaGGhxbJmc9XVRdxouI0+nfV4FHKS4VXiIitkVER0R0XDv7qpLdmZnZRCpTmA4BbZKWSJpJURh6q9r0AmvT9j3A/oiIFO9Kq/aWAG3AwZI5LyFpNvA+4LGK2FWS3jayDawAni8xLzMzy1DTS3kRMSxpI7AXaAF2RMQxSZuBvojoBbYDOyUNUJwpdaVjj0naDRwHhoENEXEeimXh1TlT/D7g48D1wFFJj0fEvWk4vwl8PSJ+XDHE64BHi7UWzAD+IiK+NvaPxMzMJpOKE5vpp+PmRdH30IYLgeXdkzcYM7MpQNLhifi/5PjOD2ZmlhUXJjMzy4oLk5mZZcWFyczMsuLCZGZmWXFhMjOzrLgwmZlZVlyYzMwsKy5MZmaWFRcmMzPLiguTmZllxYXJzMyy4sJkZmZZcWEyM7OsuDCZmVlWXJjMzCwrpQqTpFWS+iUNSNpUY/8sSbvS/gOSWiv2dad4v6SVzXJK2phiIWl+Rfz9ks5KOpJenyo7PjMzmzqaPlpdUgvwAPAhYBA4JKk3Io5XNFsHnImImyR1AVuA35bUTvGY9aXALwPfkPSr6Zh6OZ8G/gZ4qsZw/i4i/vUYxmdmZlNEmTOmZcBARJyIiDeAHqCzqk0n8Eja3gPcKUkp3hMR5yLiJDCQ8tXNGRHPRsSpUcyhzPjMzGyKKFOYFgIvV7wfTLGabSJiGDgLzGtwbJmctbxX0rclPSFp6SjGB4Ck9ZL6JPV9/+yPS3RnZmYTrUxhUo1YlGwz2ngj/wC8IyJ+Dfgj4K9GMb4iGLEtIjoiouPa2Vc16c7MzCZDmcI0CCyueL8IOF2vjaQZwGxgqMGxZXJeJCJ+GBE/StuPA29KiyNGncvMzPJVpjAdAtokLZE0k2IxQ29Vm15gbdq+B9gfEZHiXWnV3hKgDThYMudFJF2f/m6FpGVp7D8YSy4zM8tX01V5ETEsaSOwF2gBdkTEMUmbgb6I6AW2AzslDVCcKXWlY49J2g0cB4aBDRFxHopl4dU5U/w+4OPA9cBRSY9HxL0UBe/3JA0DPwW6UvGrOb7L8umYmdmEU/G7ffrpuHlR9D204UJgeffkDcbMbAqQdDgiOsa7H9/5wczMsuLCZGZmWXFhMjOzrLgwmZlZVlyYzMwsKy5MZmaWFRcmMzPLiguTmZllxYXJzMyy4sJkZmZZcWEyM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8tKqcIkaZWkfkkDkjbV2D9L0q60/4Ck1op93SneL2lls5ySNqZYpEenj8T/naSj6fX3kn6tYt8pSc9JOiKpb/Qfg5mZ5aLpE2wltQAPAB8CBoFDknoj4nhFs3XAmYi4SVIXsAX4bUntFE+zXQr8MvANSb+ajqmX82ngb4CnqoZyEnhfRJyRtBrYBtxWsX95RLw2irmbmVmGypwxLQMGIuJERLwB9ACdVW06gUfS9h7gTklK8Z6IOBcRJ4GBlK9uzoh4NiJOVQ8iIv4+Is6kt88Ai0YxTzMzmyLKFKaFwMsV7wdTrGabiBgGzgLzGhxbJmcj64AnKt4H8HVJhyWtH0UeMzPLTNNLeYBqxKJkm3rxWgWxOmftwUjLKQrTr1eE74iI05LeDuyT9J2I+GaNY9cD6wFuuG5Ome7MzGyClTljGgQWV7xfBJyu10bSDGA2MNTg2DI5LyHpXcCfAp0R8YOReEScTv++CjxKcanwEhGxLSI6IqLj2tlXNevOzMwmQZnCdAhok7RE0kyKxQy9VW16gbVp+x5gf0REinelVXtLgDbgYMmcF5F0A/BV4Hci4rsV8askvW1kG1gBPF9iXmZmlqGml/IiYljSRmAv0ALsiIhjkjYDfRHRC2wHdkoaoDhT6krHHpO0GzgODAMbIuI8FMvCq3Om+H3Ax4HrgaOSHo+Ie4FPUfzd6sFiXQXDEdEBXAc8mmIzgL+IiK9dhs/GzMwmgYoTm+mn4+ZF0ffQhguB5d2TNxgzsylA0uF0QjCufOcHMzPLiguTmZllxYXJzMyy4sJkZmZZcWEyM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZllxYTIzs6y4MJmZWVZcmMzMLCulCpOkVZL6JQ1I2lRj/yxJu9L+A5JaK/Z1p3i/pJXNckramGIhaX5FXJK+kPYdlXRrxb61kl5Mr5En6ZqZ2RTUtDBJagEeAFYD7cAaSe1VzdYBZyLiJmArsCUd207xNNulwCqKp8+2NMn5NPBB4HtVfaymeDR7G7Ae+GLq4xrgfuA2YBlwv6S5ZT8AMzPLS5kzpmXAQESciIg3gB6gs6pNJ/BI2t4D3KniWeedQE9EnIuIk8BAylc3Z0Q8GxGnaoyjE/jzKDwDzJG0AFgJ7IuIoYg4A+yjKIJmZjYFlSlMC4GXK94PpljNNhExDJwF5jU4tkzOsuMYSy4zM8tUmcKkGrEo2Wa08bGMo3QuSesl9Unq+/7ZHzfpzszMJkOZwjQILK54vwg4Xa+NpBnAbGCowbFlcpYdR+lcEbEtIjoiouPa2Vc16c7MzCZDmcJ0CGiTtETSTIrFDL1VbXqBkdVw9wD7IyJSvCut2ltCsXDhYMmc1XqBj6TVebcDZyPiFWAvsELS3LToYUWKmZnZFDSjWYOIGJa0keKXfQuwIyKOSdoM9EVEL7Ad2ClpgOJMqSsde0zSbuA4MAxsiIjzUCwLr86Z4vcBHweuB45Kejwi7gUeB+6iWEDxE+BjqY8hSZ+hKHYAmyNi6Bf9YMzMbHKoOLGZfjpuXhR9D224EFjePXmDMTObAiQdjoiO8e7Hd34wM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZllxYTIzs6y4MJmZWVZcmMzMLCsuTGZmlhUXJjMzy4oLk5mZZcWFyczMslKqMElaJalf0oCkTTX2z5K0K+0/IKm1Yl93ivdLWtksZ3qq7QFJL6acM1N8q6Qj6fVdSa9XHHO+Yl+zJ+GamVnGmhYmSS3AA8BqoB1YI6m9qtk64ExE3ARsBbakY9spnma7FFgFPCippUnOLcDWiGgDzqTcRMR/johbIuIW4I+Ar1b0/9ORfRFx96g/BTMzy0aZM6ZlwEBEnIiIN4AeoLOqTSfwSNreA9wpSSneExHnIuIkxWPRl9XLmY75QMpByvnhGmNaA3y57CTNzGzqKFOYFgIvV7wfTLGabSJiGDgLzGtwbL34POD1lKNmX5LeASwB9leE3yypT9IzkmoVMjMzmyJmlGijGrEo2aZevFZBbNS+UhewJyLOV8RuiIjTkm4E9kt6LiJeqk4kaT2wHuCG6+bU6MrMzCZbmTOmQWBxxftFwOl6bSTNAGYDQw2OrRd/DZiTctTrq4uqy3gRcTr9ewJ4Cnh3rYlExLaI6IiIjmtnX1V7tmZmNqnKFKZDQFtaLTeTojBUr3zrBdam7XuA/RERKd6VVu0tAdqAg/VypmOeTDlIOR8b6UTSzcBc4FsVsbmSZqXt+cAdwPGyH4CZmeWl6aW8iBiWtBHYC7QAOyLimKTNQF9E9ALbgZ2SBijOlLrSscck7aYoFMPAhpFLcLVypi4/AfRI+izwbMo9Yg3FYorKy3vvBB6W9DOKQvv5iHBhMjObonTx7/jpo+PmRdH30IYLgeXdkzcYM7MpQNLhiOgY736m7Z0f/vHcj+g+dWCyh2FmZlWmbWEyM7M8uTCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZllxYTIzs6y4MJmZWVZcmMzMLCsuTGZmlhUXJjMzy4oLk5mZZcWFyczMslKqMElaJalf0oCkTTX2z5K0K+0/IKm1Yl93ivdLWtksZ3rc+gFJL6acM1P8o5K+L+lIet1bccza1P5FSSOPeDczsymoaWGS1AI8AKwG2oE1ktqrmq0DzkTETcBWYEs6tp3iMetLgVXAg5JamuTcAmyNiDbgTMo9YldE3JJef5r6uAa4H7gNWAbcL2nuKD8HMzPLRJkzpmXAQESciIg3gB6gs6pNJ/BI2t4D3ClJKd4TEeci4iQwkPLVzJmO+UDKQcr54SbjWwnsi4ihiDgD7KMogmZmNgWVKUwLgZcr3g+mWM02ETEMnAXmNTi2Xnwe8HrKUauv35J0VNIeSYtHMT4AJK2X1Cep7yc/eqP+jM3MbNKUKUyqEYuSbS5XHOCvgdaIeBfwDS6coZUZXxGM2BYRHRHR8darZ9ZqYmZmk6xMYRoEFle8XwScrtdG0gxgNjDU4Nh68deAOSnHRX1FxA8i4lyK/wnwnlGMz8zMpogyhekQ0JZWy82kWMzQW9WmFxhZDXcPsD8iIsW70qq9JUAbcLBeznTMkykHKedjAJIWVPR3N/BC2t4LrJA0Ny16WJFiZmY2Bc1o1iAihiVtpPhl3wLsiIhjkjYDfRHRC2wHdkoaoDhT6krHHpO0GzgODAMbIuI8QK2cqctPAD2SPgs8m3ID3Cfp7pRnCPho6mNI0mcoih3A5ogYGvMnYmZmk0rFScr0s6B1Tnz0/vfzudbbisDy7skdkJlZ5iQdjoiO8e7Hd34wM7OsuDCZmVlWXJjMzCwrLkxmZpYVFyYzM8uKC5OZmWXFhcnMzLLiwmRmZllxYTIzs6y4MJmZWVZcmMzMLCsuTGZmlhUXJjMzy4oLk5mZZcWFyczMslKqMElaJalf0oCkTTX2z5K0K+0/IKm1Yl93ivdLWtksZ3qq7QFJL6acM1P89yUdl3RU0t9KekfFMeclHUmv6qfrmpnZFNK0MElqAR4AVgPtwBpJ7VXN1gFnIuImYCuwJR3bTvE026XAKuBBSS1Ncm4BtkZEG3Am5YbiabYdEfEuYA/wBxX9/zQibkmvu0f1CZiZWVbKnDEtAwYi4kREvAH0AJ1VbTqBR9L2HuBOSUrxnog4FxEngYGUr2bOdMwHUg5Szg8DRMSTEfGTFH8GWDT66ZqZWe5mlGizEHi54v0gcFu9NhExLOksMC/Fn6k6dmHarpVzHvB6RAzXaF9pHfBExfs3S+oDhoHPR8RflZgXAN2nDhQdfeUzTdvu/K3/XjatmZmNUZnCpBqxKNmmXrzWmVqj9hc6kv490AG8ryJ8Q0SclnQjsF/ScxHxUnUiSeuB9QC/NO8tNboyM7PJVuZS3iCwuOL9IuB0vTaSZgCzgaEGx9aLvwbMSTku6UvSB4FPAndHxLmReEScTv+eAJ4C3l1rIhGxLSI6IqLjrVfPbDZvMzObBGUK0yGgLa2Wm0mxmKF65VsvsDZt3wPsj4hI8a60am8J0AYcrJczHfNkykHK+RiApHcDD1MUpVdHOpY0V9KstD0fuAM4PpoPAWDRDw9f8jIzs4nX9FJe+pvRRmAv0ALsiIhjkjYDfRHRC2wHdkoaoDhT6krHHpO0m6JQDAMbIuI8QK2cqctPAD2SPkuxEm97iv8hcDXwl8UaCf5PWoH3TuBhST+jKLSfj4hRFyYzM8uDipOU6WdB65z46P3vb9hm8Jfec9F7L34ws+lM0uGI6BjvfnznBzMzy4oLk5mZZcWFyczMsuLCZGZmWXFhMjOzrLgwmZlZVlyYzMwsKy5MZmaWFRcmMzPLiguTmZllxYXJzMyy4sJkZmZZKfOgwGnLj74wM5t4PmMyM7OsuDCZmVlWShUmSask9UsakLSpxv5Zknal/QcktVbs607xfkkrm+VMT7U9IOnFlHPmWPswM7Opp2lhktQCPACsBtqBNZLaq5qtA85ExE3AVmBLOrad4mm2S4FVwIOSWprk3AJsjYg24EzKPeo+RvtBmJlZHsosflgGDETECQBJPUAnxePSR3QCn07be4A/VvH8806gJyLOASfTo9eXpXaX5JT0AvAB4N+mNo+kvF8cQx/fKvkZlNb9Zx++6H31E24r+Wm3ZmZjU6YwLQRerng/CNxWr01EDEs6C8xL8Weqjl2YtmvlnAe8HhHDNdqPpY9x1WjVXnURa+ZzrVUf6fLusQzJzGzKK1OYVCMWJdvUi9e6hNio/Vj6uISk9cD69Pbc53/3sedrtZsMn+exqsh/+0XSzQde+0USZM7zm9o8v6nr5onopExhGgQWV7xfBJyu02ZQ0gxgNjDU5Nha8deAOZJmpLOmyvZj6eMiEbEN2AYgqS8iOhrOfIq6kucGnt9U5/lNXZL6JqKfMqvyDgFtabXcTIqFBr1VbXqBtWn7HmB/RESKd6UVdUuANuBgvZzpmCdTDlLOx8bYh5mZTUFNz5jS33M2AnuBFmBHRByTtBnoi4heYDuwMy08GKIoNKR2uykWSgwDGyLiPECtnKnLTwA9kj4LPJtyM5Y+zMxs6lFx0jH9SFqfLu1dca7kuYHnN9V5flPXRM1t2hYmMzPLk29JZGZmWZl2hanZ7ZVyIumUpOckHRlZDSPpGkn70i2b9kmam+KS9IU0r6OSbq3Isza1f1HS2or4e1L+gXRsraX3l3M+OyS9Kun5iti4z6deHxM0v09L+sf0HR6RdFfFvnG/Xddlnt9iSU9KekHSMUn/McWn/HfYYG5XxPcn6c2SDkr6dprf/xjrmC7XvBuKiGnzolho8RJwIzAT+DbQPtnjajDeU8D8qtgfAJvS9iZgS9q+C3iC4v/XdTtwIMWvAU6kf+em7blp30HgvemYJ4DV4zyf3wBuBZ6fyPnU62OC5vdp4L/UaNuefv5mAUvSz2VLo59RYDfQlbYfAn4vbf8H4KG03QXsGqf5LQBuTdtvA76b5jHlv8MGc7sivr/0eV6dtt8EHEjfyajGdDnn3XC84/EDnOsr/cDvrXjfDXRP9rgajPcUlxamfmBB2l4A9Kfth4E11e2ANcDDFfGHU2wB8J2K+EXtxnFOrVz8i3vc51Ovjwma36ep/Yvtop89ihWq7633M5p+sbwGzKj+WR45Nm3PSO00Ad/lY8CHrrTvsGpuV9z3B7wV+AeKu+2MakyXc96NXtPtUl6t2ytNyO2LxiiAr0s6rOKuFQDXRcQrAOnft6d4vbk1ig/WiE+0iZhPvT4mysZ0KWtHxSWo0c6v9O26gJHbdY2bdGnn3RT/y/uK+g6r5gZXyPen4gbaR4BXgX0UZzijHdPlnHdd060wlb59USbuiIhbKe7CvkHSbzRoO9pbNuX+WVwp8/ki8CvALcArwP9M8cs5vwmdu6Srga8A/ykiftioaY1Y1t9hjbldMd9fRJyPiFso7o6zDHjnGMY0Id/pdCtMpW9flIOIOJ3+fRV4lOKH6Z8kLQBI/76amtebW6P4ohrxiTYR86nXx7iLiH9KvxB+BvwJF+6uP9r5/fx2XVXxi3Lp4tt1XXaS3kTxi/tLEfHVFL4ivsNac7vSvj+AiHgdeIrib0yjHdPlnHdd060wlbm9UhYkXSXpbSPbwArgeS6+NVP1LZs+klZC3Q6cTZc89gIrJM1NlyFWUFzjfQX4Z0m3p5VPH6nINZEmYj71+hh3I79Mk9+k+A5HxjTet+u63HMRxR1YXoiI/1Wxa8p/h/XmdqV8f5KulTQnbb8F+CDwwhjGdDnnXd94/gExxxfFSqHvUlxf/eRkj6fBOG+kWNnybeDYyFgprtn+LfBi+veaFBfFwxdfAp4DOipy/S4wkF4fq4h3UPyH9hLwx4zzH8yBL1NcDvl/FP8La91EzKdeHxM0v51p/EfTf9QLKtp/Mo21n4oVkfV+RtPPxME0778EZqX4m9P7gbT/xnGa369TXIY5ChxJr7uuhO+wwdyuiO8PeBfFLd6Ops/3U2Md0+Wad6OX7/xgZmZZmW6X8szMLHMuTGZmlhUXJjMzy4oLk5mZZcWFyczMsuLCZGZmWXFhMjOzrLgwmZlZVv4/mIjuHcfAWeQAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "#Plotting intron size distribution of LENGTH 1 (10,000) vs LENGTH 2 (100,000)\n",
    "_ = plt.hist(length1IntronLengths,bins=200,normed=True,alpha=0.5)\n",
    "_ = plt.hist(length2IntronLengths,bins=50,normed=True,alpha=0.5)\n",
    "plt.xlim(0,300000)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
