{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2018-10-10\n",
    "# A. Pendleton\n",
    "# Generation of miropeats images for any bed file as input"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import gzip\n",
    "import fileinput\n",
    "import glob\n",
    "from scipy import stats\n",
    "import re\n",
    "from matplotlib_venn import venn3, venn3_circles\n",
    "from collections import OrderedDict\n",
    "\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)\n",
    "# TO REMOVE TOP AND RIGHT AXIS OF PLOTS\n",
    "def simpleaxis(ax):\n",
    "    ax.spines['top'].set_visible(False)\n",
    "    ax.spines['right'].set_visible(False)\n",
    "    ax.get_xaxis().tick_bottom()\n",
    "    ax.get_yaxis().tick_left()\n",
    "#Write PBS Script\n",
    "def write_pbs_file(wkDir,cmdsFile,jobName, mem, jobCount):\n",
    "    #write PBS file\n",
    "    if '/scripts/' in wkDir:\n",
    "        wkDir = wkDir.replace('/scripts/','')\n",
    "    pbsFile = open(cmdsFile.replace('.cmds','.pbs'),'w')\n",
    "    print('Writing pbs file: %s'% pbsFile)\n",
    "    pbsFile.write('#!/bin/bash\\n')\n",
    "    pbsFile.write('#PBS -S /bin/bash\\n')\n",
    "    pbsFile.write('#PBS -V\\n')\n",
    "    pbsFile.write('#PBS -M ampend@med.umich.edu\\n')\n",
    "    pbsFile.write('#PBS -j oe\\n')\n",
    "    pbsFile.write('#PBS -N %s\\n' % jobName)\n",
    "    pbsFile.write('#PBS -o %s\\n' % (wkDir + 'logs/'))\n",
    "    pbsFile.write('#PBS -l pmem=%iG\\n' % mem)\n",
    "    pbsFile.write('#PBS -l nodes=1:ppn=1,qos=flux,walltime=100:00:00\\n')\n",
    "    pbsFile.write('##PBS -A medbsm_flux\\n')\n",
    "    pbsFile.write('##PBS -q flux\\n')\n",
    "    pbsFile.write('#PBS -A jmkidd_fluxod\\n')\n",
    "    pbsFile.write('#PBS -q fluxod\\n')\n",
    "    pbsFile.write('#PBS -t 1-%s\\n' % jobCount)\n",
    "    pbsFile.write('cd %s\\n' % (wkDir))\n",
    "    pbsFile.write('/home/ampend/links/kidd-lab/jmkidd-projects/scripts/perlUtils/run-by-id-log.pl %s %sBLAT_commands.logs $PBS_ARRAYID' % (cmdsFile,cmdsFile.replace('/scripts/','/logs/')))\n",
    "    pbsFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "###############################################################################\n",
    "#changes based on PS_template_April2010 value\n",
    "def display_edit_setup(line):\n",
    "    if '%%DocumentFonts: Helvetica-Bold Helvetica' in line:\n",
    "        line = '%%DocumentFonts: Arial-Bold Arial\\n'\n",
    "\n",
    "    if '%%BoundingBox:' in line:\n",
    "        line = '%%BoundingBox: 0 0 612 792\\n'\n",
    "    if '/repwidth' in line:\n",
    "        line = '/repwidth 0.25 cm def\\n'\n",
    "    if '/linkwidth' in line:\n",
    "        line = '/linkwidth 0 def\\n'\n",
    "    if '/gapwidth' in line:\n",
    "        line = '/gapwidth 0 def\\n'\n",
    "    if '/leftmargin' in line:\n",
    "        line = '/leftmargin 0.5 cm def\\n'\n",
    "    if '/bottommargin' in line:\n",
    "        line = '/bottommargin 2 cm def\\n'\n",
    "    if '/pageheight' in line:\n",
    "        line = '/pageheight 26 cm def\\n'\n",
    "\n",
    "    if '/Helvetica-Bold findfont' in line:\n",
    "        line = '/Arial-Bold findfont 7 scalefont setfont\\n\\n'\n",
    "\n",
    "    if '/Helvetica findfont' in line:\n",
    "        line = '/Arial findfont 7 scalefont setfont\\n'\n",
    "    if '0 pageheight 2 cm sub moveto' in line:\n",
    "        line = '0 pageheight 1.5 cm sub moveto\\n'\n",
    "    if '/graphicmargin 17.5' in line:\n",
    "        line = '/graphicmargin 17.5 cm def\\n'\n",
    "\n",
    "    if '(Longest Sequence ' in line:\n",
    "        line = '\\n'    \n",
    "    if '( Threshold Score ' in line:\n",
    "        line = '\\n'    \n",
    "\n",
    "    if 'curveto' in line:\n",
    "        line= line.replace('curveto','lineto')\n",
    "    return line\n",
    "###############################################################################\n",
    "def read_rm_file(rmFileName):\n",
    "    rmLines = []\n",
    "    inFile = open(rmFileName,'r')\n",
    "    for line in inFile:\n",
    "        if line == '\\n':\n",
    "            continue\n",
    "        line = line.rstrip()\n",
    "        line = line.split()\n",
    "        if line[0] == 'There':\n",
    "            return []\n",
    "        if line[0] == 'SW':\n",
    "            continue\n",
    "        if line[0] == 'score':\n",
    "            continue\n",
    "        rmLines.append(line)\n",
    "    inFile.close()\n",
    "    return rmLines\n",
    "###############################################################################\n",
    "def repeat_class_to_name(r):\n",
    "    if 'SINE' in r:\n",
    "        return 'SINE'\n",
    "    if 'ARTEFACT' in r:\n",
    "        return 'ARTEFACT'\n",
    "    if 'DNA' in r:\n",
    "        return 'DNA'\n",
    "    if 'LINE' in r:\n",
    "        return 'LINE'\n",
    "    if 'Low_complexity' in r:\n",
    "        return 'Low_complexity'\n",
    "    if 'LTR' in r:\n",
    "        return 'LTR'\n",
    "    if 'Other' in r:\n",
    "        return 'Other'\n",
    "    if 'rRNA' in r:\n",
    "        return 'rRNA'\n",
    "    if 'scRNA' in r:\n",
    "        return 'scRNA'\n",
    "    if 'snRNA' in r:\n",
    "        return 'snRNA'\n",
    "    if 'srpRNA' in r:\n",
    "        return 'srpRNA'\n",
    "    if 'tRNA' in r:\n",
    "        return 'tRNA'\n",
    "    if 'RNA' in r:\n",
    "        return 'RNA'\n",
    "    if 'Satellite' in r:\n",
    "        return 'Satellite'\n",
    "    if 'Simple_repeat' in r:\n",
    "        return 'Simple_repeat'\n",
    "    if 'Unknown' in r:\n",
    "        return 'Unknown'\n",
    "    if 'Retroposon' in r:\n",
    "        return 'SINE'    \n",
    "    print ('repeat class unknown for',r)\n",
    "    return 'Unknown'\n",
    "###############################################################################\n",
    "def sort_and_merge_repeats(repeats):\n",
    "    repeats.sort()\n",
    "    newRep = []\n",
    "    for r in repeats:\n",
    "        s = r[0]\n",
    "        e = r[1]\n",
    "        orientation = r[2]\n",
    "        repClass = r[3]\n",
    "        if len(newRep) == 0:\n",
    "            newRep.append(r)\n",
    "        else:\n",
    "            lr = newRep[-1]\n",
    "            ls = lr[0]\n",
    "            le = lr[1]\n",
    "            lorient = lr[2]\n",
    "            lrepClass = lr[3]\n",
    "            # overlap, need to extend\n",
    "            if le > s and lrepClass == repClass and lorient == orientation:\n",
    "                n = [ls,e,orientation,repClass]\n",
    "                newRep[-1] = n\n",
    "            else:\n",
    "                newRep.append(r)\n",
    "    return newRep    \n",
    "###############################################################################\n",
    "def process_repeat_file(fn):\n",
    "    repeatLines = read_rm_file(fn)\n",
    "    repeats = []\n",
    "    for R in repeatLines:\n",
    "        s = int(R[5])\n",
    "        e = int(R[6])\n",
    "        orientation = R[8]\n",
    "\n",
    "        repClass = R[10]\n",
    "        if orientation == 'C':\n",
    "            orientation = '-'\n",
    "        repClass = repeat_class_to_name(repClass)\n",
    "        reps = [s,e,orientation,repClass]\n",
    "        repeats.append(reps)\n",
    "    repeats = sort_and_merge_repeats(repeats)\n",
    "    return repeats\n",
    "###############################################################################\n",
    "def process_gap_file(fn):\n",
    "    gapLines = []\n",
    "    inFile = open(fn,'r')\n",
    "    for line in inFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split()\n",
    "        s = int(line[1])\n",
    "        e = int(line[2])\n",
    "        gapLines.append([s,e])\n",
    "    inFile.close()\n",
    "    return gapLines\n",
    "###############################################################################\n",
    "def process_exon_file(fn):\n",
    "    exonLines = []\n",
    "    inFile = open(fn,'r')\n",
    "    for line in inFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split()\n",
    "        s = int(line[1])\n",
    "        e = int(line[2])\n",
    "        name = line[3]\n",
    "        exonLines.append([name,s,e])\n",
    "        exonLines.sort()\n",
    "    inFile.close()\n",
    "    return exonLines  \n",
    "###############################################################################\n",
    "def process_primer_file(fn):\n",
    "    primerLines = []\n",
    "    inFile = open(fn,'r')\n",
    "    for line in inFile:\n",
    "        line = line.rstrip()\n",
    "        line = line.split()\n",
    "        s = int(line[1])\n",
    "        e = int(line[2])\n",
    "        dir = line[3]\n",
    "        primerLines.append([s,e,dir])\n",
    "    inFile.close()\n",
    "    return primerLines\n",
    "###############################################################################"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "###############################################################################\n",
    "USAGE = \"\"\"\n",
    "python annotate-miropeats-3seqs-AP.py    \n",
    "--miroin <in miropeat ps file>  \n",
    "--topRM <RM of top> \n",
    "--bottomRM <RM of bottom>\n",
    "--topName <name of top> \n",
    "--bottomName <name of bottom>\n",
    "--blat <blat output of canFam vs fosmid, blast9 output format>\n",
    "\"\"\"\n",
    "\n",
    "parser = OptionParser(USAGE)\n",
    "parser.add_option('--miroin',dest='miropeatsInput',help='input file of miropeats ps file')\n",
    "parser.add_option('--topRM',dest='topRM',help='repeat mask out file for top sequence')\n",
    "parser.add_option('--bottomRM',dest='bottomRM',help='repeat mask out file for bottom sequence')\n",
    "parser.add_option('--topName',dest='topName',help='name of top seqence')\n",
    "parser.add_option('--bottomName',dest='bottomName',help='name of bottom seqence')\n",
    "parser.add_option('--blat',dest='blat',help='blat of canFam versus fosmid results, blast9 output format')\n",
    "\n",
    "(options,args)=parser.parse_args()\n",
    "if options.miropeatsInput is None:\n",
    "    parser.error('miropeats input file name not given')\n",
    "if options.topRM is None:\n",
    "    parser.error('top RM input file name not given')\n",
    "if options.bottomRM is None:\n",
    "    parser.error('bottom RM input file name not given')\n",
    "if options.topName is None:\n",
    "    parser.error('top seq name not given')\n",
    "if options.bottomName is None:\n",
    "    parser.error('bottom seq name not given')\n",
    "if options.blat is None:\n",
    "    parser.error('blat output not given, blast9 output format')\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "##### Info files #####\n",
    "PSTemplate = '/home/ampend/links/kidd-lab/jmkidd-projects/people-projects/jwilds-projects/RetroSeq-HGDP/align/scripts/PS_template_July2008_withlines.txt'\n",
    "\n",
    "#### setup the repeat to color dictionary#####\n",
    "color = {}\n",
    "color['Other'] = 'Black'\n",
    "color['Simple Repeat'] = 'DkGray'\n",
    "color['Low Complexity'] = 'LtGray'\n",
    "color['DNA'] = 'Pink'\n",
    "color['LTR'] = 'Orange'\n",
    "color['LINE'] = 'Green'\n",
    "color['SINE'] = 'Purple'\n",
    "color['GAP'] = 'Black' \n",
    "color['EXON'] = 'Blue'\n",
    "\n",
    "#height of canfam and zoey sequence, in \"Y\" coordinates\n",
    "#zoey is what is on the bottom\n",
    "#ref_line=1.2\n",
    "#fos_line=0.4\n",
    "#####bottomLine = 0.3\n",
    "bottomLine = .6 #0.3\n",
    "#middleLine = 0.8 # ONLY FOR WHEN YOU HAVE A THREE-WAY ALIGNMENT\n",
    "####topLine = 1.0\n",
    "topLine = 1.3\n",
    "# for direction...\n",
    "#arrowEndBp = 1000\n",
    "arrowEndBp = 100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# these are for coloring breakpoints... we will skip this for now...\n",
    "chrm_breaks = {}\n",
    "chrm_breaks['not do'] = 1\n",
    "clone_breaks = {}\n",
    "clone_breaks['not do'] = 1\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "############################################################################################################\n",
    "############################################################################################################\n",
    "############################################################################################################\n",
    "############################################################################################################"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Generate FASTA files for the regions of interest based off of bed coordinate file"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Get the coordinates of the Zoey ROIs"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1 Zoey ROIs read into array\n"
     ]
    }
   ],
   "source": [
    "miropeatsDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/'\n",
    "\n",
    "zoeyROIBedfile = miropeatsDir + 'input/' + 'Zoey_ROI.bed'\n",
    "\n",
    "zoeyCoords = []\n",
    "\n",
    "for line in open(zoeyROIBedfile,'r'):\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,start,end,ID = line[0],int(line[1]),int(line[2]),line[3]\n",
    "    zoeyCoords.append([chrom,start,end,ID])\n",
    "\n",
    "print('%i Zoey ROIs read into array' % len(zoeyCoords))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Generate FASTA files for each ROI in its own directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa\n"
     ]
    }
   ],
   "source": [
    "#Conditionally create scripts directory\n",
    "cmd = 'mkdir -p %s' % (miropeatsDir + 'scripts/')\n",
    "runCMD(cmd)\n",
    "\n",
    "#Cannot run repeatmasker within ipython notebooks so you'll have to run these yourself later\n",
    "repeatMaskerCommandfile = miropeatsDir + 'scripts/' + 'RepeatMask_ZoeySeqs.cmds'\n",
    "repeatMaskerCommandFile = open(repeatMaskerCommandfile,'w')\n",
    "\n",
    "\n",
    "for roi in range(0,len(zoeyCoords)):\n",
    "    chrom,start,end,ID = zoeyCoords[roi][0],int(zoeyCoords[roi][1]),int(zoeyCoords[roi][2]),zoeyCoords[roi][3]\n",
    "    \n",
    "    #EXTEND OUT 5KB FOR VIEWING\n",
    "    start,end = start-5000, end+5000\n",
    "    #Change in the array\n",
    "    zoeyCoords[roi][1],zoeyCoords[roi][2] = start,end\n",
    "    \n",
    "    #conditionally make directory for each ROI, make it only if it doesn't already exist\n",
    "    newDir = '%sinput/%s/' % (miropeatsDir,ID)\n",
    "    cmd = 'mkdir -p %s ' % (newDir)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    ##EXTRACT FASTA \n",
    "    zoeyGenomeFastaFile = '/home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa'\n",
    "    print(zoeyGenomeFastaFile)\n",
    "    \n",
    "    outFasta = '%s%s.fa' % (newDir,ID)\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (zoeyGenomeFastaFile,chrom,start,end,outFasta)\n",
    "    #print(cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    ##REPEAT MASK FASTA\n",
    "    cmd = 'RepeatMasker --species dog %s' % outFasta\n",
    "    #print(cmd)\n",
    "    repeatMaskerCommandFile.write('%s\\n' % cmd)\n",
    "    \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## ***** Run RepeatMasker Commands now *****\n",
    "\n",
    "You'll need to do this on Flux\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## BLAT Zoey Fasta against the CanFam3.1 genome"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#### CRABP1_Locus\n",
      "/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.2bit\n",
      "blat -out=psl -ooc=/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/11.ooc -tileSize=11 -noHead /home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.2bit /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CRABP1_Locus/CRABP1_Locus.fa /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CRABP1_Locus/BLAT_to_Canfam_CRABP1_Locus.psl\n",
      "Best Hit: chr1:87290321-87290504 (0.941241%)\n"
     ]
    }
   ],
   "source": [
    "blatCommands = ''\n",
    "for roi in range(0,len(zoeyCoords)):\n",
    "    chrom,start,end,ID = zoeyCoords[roi][0],int(zoeyCoords[roi][1]),int(zoeyCoords[roi][2]),zoeyCoords[roi][3]\n",
    "    print('\\n#### %s' % ID)\n",
    "    #conditionally make directory for each ROI, make it only if it doesn't already exist\n",
    "    newDir = '%sinput/%s/' % (miropeatsDir,ID)\n",
    "    \n",
    "    ##Run BLAT against CanFam3.1 \n",
    "    CanFamGenome2bitFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.2bit'\n",
    "    oocFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/11.ooc'\n",
    "    print(CanFamGenome2bitFile)\n",
    "    \n",
    "    zoeyFasta = '%s%s.fa' % (newDir,ID)\n",
    "    blatOutFile = newDir + 'BLAT_to_Canfam_' + ID + '.psl' \n",
    "    cmd = 'blat -out=psl -ooc=%s -tileSize=11 -noHead %s %s %s' % (oocFile,CanFamGenome2bitFile,zoeyFasta,blatOutFile)\n",
    "    print(cmd)\n",
    "    #runCMD(cmd)\n",
    "\n",
    "    #PARSE BLAT HIT TO FIND BEST HIT\n",
    "    bestHit,bestHitInfo = 0, [] \n",
    "    for line in open(blatOutFile,'r'):\n",
    "        line=line.rstrip().split('\\t')\n",
    "        hitLength,zoeyLength = int(line[0]),int(line[10])\n",
    "        propHit = hitLength/float(zoeyLength)\n",
    "        if propHit > bestHit:\n",
    "            bestHit = propHit\n",
    "            bestHitInfo = line\n",
    "    CFchrom,CFstart,CFend = line[13],int(line[15]),int(line[16])\n",
    "    print('Best Hit: %s:%i-%i (%f%%)' % (CFchrom,CFstart,CFend,bestHit))\n",
    "    \n",
    "    \n",
    "    #EXTEND OUT FOR ALIGNMENTS\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "bottomName = options.bottomName\n",
    "miropeatsInFile = options.miropeatsInput\n",
    "topRepeatMaskFile = options.topRM\n",
    "bottomRepeatMaskFile = options.bottomRM\n",
    "miropeatsOutFile = miropeatsInFile + '.annotated.ps'\n",
    "\n",
    "inFile = open(miropeatsInFile,'r')\n",
    "outFile = open(miropeatsOutFile,'w')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
