{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2018-10-10\n",
    "#\n",
    "# Updated 2019-04-25\n",
    "# A. Pendleton\n",
    "# Generation of miropeats images for any bed file as input\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "#Remaining required \n",
    "import matplotlib.pyplot as plt\n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import gzip\n",
    "import fileinput\n",
    "import glob\n",
    "\n",
    "\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "#Facilitates running commands on flux from within the ipython enviroment.\n",
    "# Requires import of: \n",
    "#   - sys\n",
    "#   - subprocess\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)\n",
    "#Write PBS Script header and standard run-by-id perl script to run tasks \n",
    "def write_pbs_file(wkDir,cmdsFile,jobName, mem, jobCount):\n",
    "    #write PBS file\n",
    "    if '/scripts/' in wkDir:\n",
    "        wkDir = wkDir.replace('/scripts/','')\n",
    "    pbsFile = open(cmdsFile.replace('.cmds','.pbs'),'w')\n",
    "    print('Writing pbs file: %s'% pbsFile)\n",
    "    pbsFile.write('#!/bin/bash\\n')\n",
    "    pbsFile.write('#PBS -S /bin/bash\\n')\n",
    "    pbsFile.write('#PBS -V\\n')\n",
    "    pbsFile.write('#PBS -M ampend@med.umich.edu\\n')\n",
    "    pbsFile.write('#PBS -j oe\\n')\n",
    "    pbsFile.write('#PBS -N %s\\n' % jobName)\n",
    "    pbsFile.write('#PBS -o %s\\n' % (wkDir + 'logs/'))\n",
    "    pbsFile.write('#PBS -l pmem=%iG\\n' % mem)\n",
    "    pbsFile.write('#PBS -l nodes=1:ppn=1,qos=flux,walltime=100:00:00\\n')\n",
    "    pbsFile.write('##PBS -A medbsm_flux\\n')\n",
    "    pbsFile.write('##PBS -q flux\\n')\n",
    "    pbsFile.write('#PBS -A jmkidd_fluxod\\n')\n",
    "    pbsFile.write('#PBS -q fluxod\\n')\n",
    "    pbsFile.write('#PBS -t 1-%s\\n' % jobCount)\n",
    "    pbsFile.write('cd %s\\n' % (wkDir))\n",
    "    pbsFile.write('/home/ampend/links/kidd-lab/jmkidd-projects/scripts/perlUtils/run-by-id-log.pl %s %sBLAT_commands.logs $PBS_ARRAYID' % (cmdsFile,cmdsFile.replace('/scripts/','/logs/')))\n",
    "    pbsFile.close()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "______________\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Miropeats pipeline processing script\n",
    "### Goal \n",
    "The goal of this notebook is to facilitate an easier processing of regions of interest in the Zoey PacBio genome to which make miropeats images against the current dog genome reference assembly (CanFam3.1). \n",
    "\n",
    "__Warning__: I have hard-coded regions of this script, of which I will try to denote. \n",
    "\n",
    "### Required inputs\n",
    "1. Input BED file\n",
    "2. Root directory path that you wish all resulting directories and files to be created.\n",
    "3. Path to the genome reference FASTA file(s). \n",
    "\n",
    "### Input files\n",
    "The input file for this script is a bed file of coordinates of interested for which you would like to generate a miropeats image. \n",
    "\n",
    "__BED file__: The input file should be a tab-delimited BED coordinate file that has chromosome, start (0-based), end, and sequence identifier. \n",
    "\n",
    "- The BED input file can contain only one sequence, for practice. This script writes a command file for EACH sequence that you wish to process.\n",
    "\n",
    "__Genome fasta files__: The full path will be needed for the two genome fasta files that you want to extract the sequence from for alignment and visualization with miropeats. \n",
    "\n",
    "You will need a genome FASTA sequence for ___sequence 'A'___ (matches the sequence with coordinates of your input bed file). For example, genome A == Zoey. You will also need to have a genome B for ___sequence 'B'___, against which you are visualizing alignment. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "______________\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "## HARD CODED -- Change the below to fit your needs."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Root directory that has input files, and to which all results will be written\n",
    "rootDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/example/'\n",
    "\n",
    "#Example input file can be found here:\n",
    "inputBedfile = rootDir + 'input/' + 'example.bed'\n",
    "\n",
    "\n",
    "#Provide the name of the samples or genome names that you'd like to process.\n",
    "# Each will need a fasta file (e.g. genome) associated with it.\n",
    "seqAID = 'Zoey'\n",
    "seqBID = 'CanFam'\n",
    "\n",
    "#Here, provide full path of the fasta file that will be used to extract sequences \n",
    "#   for alignment and visualization with miropeats\n",
    "#Example below: Zoey and CanFam3 \n",
    "seqAGenome = '/home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa'\n",
    "seqBGenome = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.fa'\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Read in what coordinates you wish to plot. They will be stored in roiCoords array"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1 ROIs read into array\n"
     ]
    }
   ],
   "source": [
    "\n",
    "roiCoords = [] #coordinates chrom,start,end,ID for regions of interest\n",
    "\n",
    "for line in open(inputBedfile,'r'):\n",
    "    line=line.rstrip().split('\\t')\n",
    "    chrom,start,end,ID = line[0],int(line[1]),int(line[2]),line[3]\n",
    "    roiCoords.append([chrom,start,end,ID])\n",
    "\n",
    "print('%i ROIs read into array' % len(roiCoords))\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "______________\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Generate FASTA files for the regions of interest based off of bed coordinate file in step above, stored in roiCoord array"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Creates a directory named input within your defined input directory (== rootDir)\n",
    "def create_directory(rootDir,ID):\n",
    "    inDir = rootDir + 'input/' + ID + '/'\n",
    "    cmd = 'mkdir -p %s' % inDir \n",
    "    runCMD(cmd)\n",
    "    \n",
    "    return inDir\n",
    "############################################################################################################\n",
    "#This generates a commands file named MiropeatsCommands for each \n",
    "def create_commands_file(inDir,ID):\n",
    "    #CREATE COMMANDS FILE \n",
    "    cmdsfile = '%sMiropeatsCommands_%s.cmds' % (inDir,ID)\n",
    "    cmdsFile = open(cmdsfile,'w')\n",
    "    print('All commands for processing of ROI %s written to: %s\\n' % (ID,cmdsfile))\n",
    "\n",
    "    cmd = 'chmod 777 %s' % cmdsfile #make the commands file executable\n",
    "    runCMD(cmd)\n",
    "    return cmdsFile\n",
    "\n",
    "############################################################################################################\n",
    "# To process each region of interest (ROI) through the miropeats pipeline, it will require a BED file \n",
    "# to be generated for the sequences to be compared: sequence A (e.g.,  Zoey) and sequence B (e.g.,  CanFam) \n",
    "def make_bedfile(inDir,ROI_ID,seqACoords,seqBCoords,seqAID,seqBID):\n",
    "    #SeqA (e.g. Zoey == bottom of miropeats image)\n",
    "    bedfile = inDir + '%s_%s.bed' % (ROI_ID, seqAID)\n",
    "    bedFile = open(bedfile,'w')\n",
    "    chrom = seqACoords.split(':')[0]\n",
    "    start,end = int(seqACoords.split(':')[1].split('-')[0]),int(seqACoords.split(':')[1].split('-')[1])\n",
    "    bedFile.write('%s\\t%i\\t%i\\t%s_%s\\n' % (chrom,start,end,seqAID,ROI_ID))\n",
    "    #SeqB (e.g. CanFam == top of miropeats image)\n",
    "    bedfile = inDir + '%s_%s.bed' % (ROI_ID, seqBID)\n",
    "    bedFile = open(bedfile,'w')\n",
    "    chrom = seqBCoords.split(':')[0]\n",
    "    start,end = int(seqBCoords.split(':')[1].split('-')[0]),int(seqBCoords.split(':')[1].split('-')[1])\n",
    "    bedFile.write('%s\\t%i\\t%i\\t%s_%s\\n' % (chrom,start,end,seqBID,ROI_ID))\n",
    "    \n",
    "############################################################################################################\n",
    "#This reads through the bedfile that was created in the function above, and extends the coordinates \n",
    "#  out for viewing in the miropeats image. This can be altered with the provided window size\n",
    "def read_bedfile(inDir,ROI_ID,name,windowSize):\n",
    "    #WINDOW SIZE - extension to view the entire region\n",
    "    windowSize = int(windowSize) #Just in case it wasn't read in as an integer. \n",
    "    \n",
    "    #name == either seqA or seqB identifier\n",
    "    \n",
    "    ##READ BED FILE FOR SEQA (BOTTOM) or SEQB (TOP)\n",
    "    bedfile = '%s%s_%s.bed' % (inDir,name,ROI_ID)\n",
    "    for line in open(bedfile, 'r'):\n",
    "        line=line.rstrip().split('\\t')\n",
    "        chrom,start,end,ID = line[0], int(line[1]),int(line[2]),line[3]\n",
    "        break\n",
    "    print('Coordinates for ROI %s: %s:%i-%i' % (ROI_ID,chrom,start,end))\n",
    "    \n",
    "    #EXTEND OUT 1KB FOR VIEWING\n",
    "    start,end = start-1000, end+1000\n",
    "    if start < 0:\n",
    "        start = 0\n",
    "    print('Extended coordinates are %s:%i-%i' % (chrom,start,end))\n",
    "    return chrom,start,end\n",
    "############################################################################################################\n",
    "# Here, it will require \n",
    "def extract_fasta(inDir,name,ID,chrom,start,end,cmdsFile):\n",
    "    if 'Zoey' in name: \n",
    "        GenomeFastaFile = '/home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa'\n",
    "    else:\n",
    "        GenomeFastaFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.fa'\n",
    "\n",
    "    #Name of output file\n",
    "    outFasta = '%s%s_%s.fa' % (inDir,name,ID)\n",
    "    #Samtools index of new \n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (GenomeFastaFile,chrom,start,end,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "\n",
    "    #RENAME FASTA ID\n",
    "    cmd = 'sed -i \\'s/%s:%i-%i/%s_%s/g\\' %s' % (chrom,start,end,name,ID,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    return outFasta\n",
    "############################################################################################################\n",
    "def write_repeat_commands(outFasta,cmdsFile):\n",
    "    cmd = 'module load RepeatMasker'\n",
    "    cmdsFile.write('%s\\n' % cmd)    \n",
    "\n",
    "    cmd = 'RepeatMasker --species dog %s' % outFasta\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "############################################################################################################\n",
    "def do_exon_intersects(inDir,ID,name,cmdsFile):    \n",
    "    #INTERSECT WITH EXONS\n",
    "    if 'Zoey' in name:\n",
    "        exonBedFile = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/FINAL_GENES/Zoey_FinalGenes_exons.bed'\n",
    "    else:\n",
    "        exonBedFile = ' ~/links/kidd-lab/genomes/canFam3.1/annotations/ensemble-79/ensGene.exons.bed'\n",
    "    \n",
    "    cmd = 'bedtools intersect -wo -a %s%s_%s.bed -b %s> %sexons_%s.exons' % (inDir,name,ID,exonBedFile,inDir,name)\n",
    "    runCMD(cmd)\n",
    "    print(cmd)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "############################################################################################################    \n",
    "def run_miropeats_script(cmdsFile):\n",
    "    #WRITE MIROPEATS COMMAND\n",
    "    ## INPUTS == MASKED FASTA FILES\n",
    "    zoeyFasta = '%sZoey_%s.fa' % (inDir,ID)\n",
    "    canfamFasta = '%sCanFam_%s.fa' % (inDir,ID)\n",
    "    \n",
    "    sList = [50,100,200,250]\n",
    "    for s in sList:\n",
    "        miropeatsoutput = inDir + 'miropeats_s' + str(s) + '_' + ID + '_onlyinter.ps' \n",
    "        #Remove any previously generated *ps files\n",
    "        cmd = 'rm %s' % miropeatsoutput\n",
    "        cmdsFile.write('%s\\n' % cmd)        \n",
    "        \n",
    "        #Write command to change directories\n",
    "        cmd = 'cd %s' % inDir\n",
    "        cmdsFile.write('%s\\n' % cmd)\n",
    "        \n",
    "        #Write miropeats commands\n",
    "        cmd = 'miropeats -s %i -onlyinter -o %s -seq %s %s' % (s,miropeatsoutput,canfamFasta.replace(inDir,'').replace('.masked',''),zoeyFasta.replace(inDir,'').replace('.masked',''))\n",
    "        cmdsFile.write('%s\\n' % cmd)\n",
    "        \n",
    "        \n",
    "    # BLAT COMMAND\n",
    "    cmd = 'blat -fine -minMatch=1 -minScore=10 -out=blast9 %s %s %sBLAT_CanFam_vs_Zoey.blat' % (canfamFasta,zoeyFasta,inDir)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "############################################################################################################    \n",
    "def call_gaps(cmdsFile):\n",
    "    names = ['Zoey','CanFam']\n",
    "    zoeyFasta = '%sZoey_%s.fa' % (inDir,ID)\n",
    "    canfamFasta = '%sCanFam_%s.fa' % (inDir,ID)\n",
    "    \n",
    "    for name in names:\n",
    "        fastaFile = '%s%s_%s.fa' % (inDir,name,ID)\n",
    "        cmd = 'perl ~/links/kidd-lab/jmkidd-projects/scripts/perlUtils/get_gaps.pl %s > %sgaps_%s.gaps' % (fastaFile, inDir, name)\n",
    "        cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "############################################################################################################    \n",
    "def write_miropeats_annotate(miropeatsDir,inDir,ID,chrom,start,end):\n",
    "    #ANNOTATING MIROPEATS IMAGE\n",
    "    #miropeats script:\n",
    "    scriptPath = miropeatsDir + 'scripts/annotate-miropeats-2seqs-ForZoeyGenomeComparisons.py'\n",
    "    \n",
    "    #Miropeats command\n",
    "    sList = [50,100,200,250]\n",
    "    for s in sList:\n",
    "        cmd = 'python %s ' % (scriptPath)\n",
    "        cmd += '--miroin %s ' % (inDir + 'miropeats_s' + str(s) + '_' + ID + '_onlyinter.ps')\n",
    "        cmd += '--topRM %s%s_%s.fa.out ' % (inDir,'CanFam',ID)\n",
    "        cmd += '--bottomRM %s%s_%s.fa.out ' % (inDir,'Zoey',ID)\n",
    "        cmd += '--topName CanFam_%s ' % (ID)\n",
    "        cmd += '--bottomName Zoey_%s ' % (ID)\n",
    "        cmd += '--blat %sBLAT_CanFam_vs_Zoey.blat ' % (inDir)\n",
    "        cmd += '--coord %s:%i-%i ' % (chrom,start,end)\n",
    "        print(cmd)\n",
    "        cmdsFile.write('%s\\n' % cmd)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Get the coordinates of the Zoey ROIs\n",
    "### Generate FASTA files for each ROI in its own directory"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#### CRABP1_Locus\n",
      "Coordinates for ROI CRABP1_Locus: chr3:58404873-58416852\n",
      "Expanded new coord:  (58403873, 58417852)\n",
      "All commands for processing of ROI CRABP1_Locus written to: /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/example/input/CRABP1_Locus/MiropeatsCommands_CRABP1_Locus.cmds\n",
      "samtools faidx /home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa chr3:58403873-58417852 > /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/example/input/CRABP1_Locus/Zoey_CRABP1_Locus.fa\n",
      "/home/jmkidd/links/kidd-lab/pkg/RepeatMasker/4.0.7/RepeatMasker/RepeatMasker --species dog /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/example/input/CRABP1_Locus/Zoey_CRABP1_Locus.fa\n"
     ]
    }
   ],
   "source": [
    "#Reading through each of the regions of interest (ROI)\n",
    "for roi in range(0,len(roiCoords)):\n",
    "    chrom,start,end,ID = roiCoords[roi][0],int(roiCoords[roi][1]),int(roiCoords[roi][2]),roiCoords[roi][3]\n",
    "    print('\\n#### %s' % ID)\n",
    "    \n",
    "    \n",
    "    print('Coordinates for ROI %s: %s:%i-%i' % (ID,chrom,start,end))\n",
    "    #EXTEND OUT 1KB FOR VIEWING\n",
    "    start,end = start-1000, end+1000\n",
    "    if start < 0:\n",
    "        start = 0\n",
    "        \n",
    "    #Overwrite and change in the array\n",
    "    roiCoords[roi][1],roiCoords[roi][2] = start,end\n",
    "    print('Expanded new coord: ', (start,end))\n",
    "    \n",
    "    #MAKE DIRECTORY FOR OUTPUTS\n",
    "    newDir = '%sinput/%s/' % (rootDir,ID)\n",
    "    cmd = 'mkdir -p %s ' % (newDir)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    \n",
    "    #CREATE COMMANDS FILE \n",
    "    cmdsfile = '%sMiropeatsCommands_%s.cmds' % (newDir,ID)\n",
    "    cmdsFile = open(cmdsfile,'w')\n",
    "    print('All commands for processing of ROI %s written to: %s' % (ID,cmdsfile))\n",
    "    \n",
    "    cmd = 'chmod 777 %s' % cmdsfile #make the commands file executable\n",
    "    runCMD(cmd) \n",
    "    \n",
    "    \n",
    "    ##WRITE BED FILE FOR ZOEY\n",
    "    seqAbedfile = '%s%s_%s.bed' % (newDir,seqAID,ID)\n",
    "    bedFile = open(seqAbedfile, 'w')\n",
    "    bedFile.write('%s\\t%i\\t%i\\t%s_%s\\n' % (chrom,start,end,seqAID,ID))\n",
    "    bedFile.close()\n",
    "    \n",
    "    ##EXTRACT FASTA \n",
    "    outFasta = '%s%s_%s.fa' % (newDir,seqAID,ID)\n",
    "    #faidx extracts from genome in between extended roi coordinates\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (seqAGenome,chrom,start,end,outFasta) \n",
    "    print(cmd) #print out command\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    #RENAME FASTA ID\n",
    "    cmd = 'sed -i \\'s/%s:%i-%i/%s_%s/g\\' %s' % (chrom,start,end,seqAID,ID,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    ##REPEAT MASK FASTA\n",
    "    #************ HARD CODED PATH TO REPEATMASKER************\n",
    "    cmd = '/home/jmkidd/links/kidd-lab/pkg/RepeatMasker/4.0.7/RepeatMasker/RepeatMasker --species dog %s' % outFasta\n",
    "    print(cmd)\n",
    "    runCMD(cmd)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    \"\"\"#INTERSECT WITH EXONS\n",
    "    exonBedFile = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/FINAL_GENES/Zoey_FinalGenes_exons.bed'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s> %sexons_Zoey.exons' % (zoeybedfile,exonBedFile,newDir)\n",
    "    runCMD(cmd)\n",
    "    cmdsFile.write('%s\\n' % cmd)\"\"\"\n",
    "    break\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#### CRABP1_Locus\n",
      "Coordinates for ROI CRABP1_Locus: chr3:58403873-58417852\n"
     ]
    }
   ],
   "source": [
    "\n",
    "#Reading through each of the regions of interest (ROI)\n",
    "for roi in range(0,len(roiCoords)):\n",
    "    chrom,start,end,ID = roiCoords[roi][0],int(roiCoords[roi][1]),int(roiCoords[roi][2]),roiCoords[roi][3]\n",
    "    print('\\n#### %s' % ID)\n",
    "    \n",
    "    print('Coordinates for ROI %s: %s:%i-%i' % (ID,chrom,start,end))\n",
    "\n",
    "    ############################################\n",
    "    # Find the equivalent region in the sequence B fasta that matches sequence A\n",
    "    # Example: ROI in zoey (seqA) but need to find corresponding region in the canfam (seqB) genome\n",
    "    ############################################\n",
    "    \n",
    "    ## BLAT SEQA vs SEQB \n",
    "    seqB_Genome2bitFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.2bit'\n",
    "    seqB_oocFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/11.ooc'\n",
    "    \n",
    "    zoeyFasta = '%s%s_%s.fa' % (newDir,seqAID,ID)\n",
    "    blatOutFile = newDir + 'BLAT_to_%s_' % (seqBID) + ID + '.psl' \n",
    "    cmd = 'blat -out=psl -ooc=%s -tileSize=11 -noHead %s %s %s' % (seqB_oocFile,seqB_Genome2bitFile,zoeyFasta,blatOutFile)\n",
    "    print(cmd)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    break\n",
    "    \n",
    "    #PARSE BLAT FILE TO FIND BEST HIT\n",
    "    bestHit,bestHitInfo = 0, [] \n",
    "    for line in open(blatOutFile,'r'):\n",
    "        line=line.rstrip().split('\\t')\n",
    "        hitLength,zoeyLength = int(line[0]),int(line[10])\n",
    "        propHit = hitLength/float(zoeyLength)\n",
    "        if propHit > bestHit:\n",
    "            bestHit = propHit\n",
    "            bestHitInfo = line\n",
    "    CFchrom,CFstart,CFend = bestHitInfo[13],int(bestHitInfo[15]),int(bestHitInfo[16])\n",
    "    print('Best Hit: %s:%i-%i (%f%%)' % (CFchrom,CFstart,CFend,bestHit))\n",
    "    \n",
    "    \n",
    "    #EXTEND OUT FOR ALIGNMENTS\n",
    "    CFstart,CFend = CFstart - 1000, CFend + 1000\n",
    "    if CFstart < 0:\n",
    "        CFstart = 0\n",
    "    \n",
    "    #WRITE BED FILE FOR CANFAM\n",
    "    canfamBedfile = '%sCanFam_%s.bed' % (newDir,ID)\n",
    "    bedFile = open(canfamBedfile, 'w')\n",
    "    bedFile.write('%s\\t%i\\t%i\\tCanFam_%s\\n' % (CFchrom,CFstart,CFend,ID))\n",
    "    bedFile.close()\n",
    "    \n",
    "    \n",
    "    #GET FASTA\n",
    "    CanFamGenomeFastaFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.fa'\n",
    "    outFasta = '%sCanFam_%s.fa' % (newDir,ID)\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (CanFamGenomeFastaFile,CFchrom,CFstart,CFend,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    \n",
    "    #RENAME FASTA ID\n",
    "    cmd = 'sed -i \\'s/%s:%i-%i/CanFam_%s/g\\' %s' % (CFchrom,CFstart,CFend,ID,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    \n",
    "    \n",
    "    ##REPEAT MASK FASTA\n",
    "    cmd = 'RepeatMasker --species dog %s' % outFasta\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    \n",
    "    #INTERSECT WITH EXONS\n",
    "    exonBedFile = ' ~/links/kidd-lab/genomes/canFam3.1/annotations/ensemble-79/ensGene.exons.bed'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %sexons_CanFam.exons' % (canfamBedfile,exonBedFile,newDir)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    #WRITE MIROPEATS COMMAND\n",
    "    ## INPUTS == MASKED FASTA FILES\n",
    "    zoeyFasta = '%sZoey_%s.fa' % (newDir,ID)\n",
    "    canfamFasta = '%sCanFam_%s.fa' % (newDir,ID)\n",
    "\n",
    "    \n",
    "    #################\n",
    "    ### MIROPEATS ###\n",
    "    #################\n",
    "    print('\\n#MIROPEATS')\n",
    "\n",
    "    #Run first miropeats + blat\n",
    "    run_miropeats_script(cmdsFile)\n",
    "\n",
    "    #Call gaps\n",
    "    call_gaps(cmdsFile)\n",
    "\n",
    "    #Run miropeats annotate script\n",
    "    write_miropeats_annotate(miropeatsDir,newDir,ID,chrom,start,end)\n",
    "\n",
    "    \n",
    "cmdsFile.close()\n",
    "print('Done')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\"\n",
    "#Reading through each of the \n",
    "for roi in range(0,len(roiCoords)):\n",
    "    chrom,start,end,ID = roiCoords[roi][0],int(roiCoords[roi][1]),int(roiCoords[roi][2]),roiCoords[roi][3]\n",
    "    print('\\n#### %s' % ID)\n",
    "    \n",
    "    \n",
    "    print('Coordinates for ROI %s: %s:%i-%i' % (ID,chrom,start,end))\n",
    "    #EXTEND OUT 1KB FOR VIEWING\n",
    "    start,end = start-1000, end+1000\n",
    "    if start < 0:\n",
    "        start = 0\n",
    "        \n",
    "    #Overwrite and change in the array\n",
    "    roiCoords[roi][1],roiCoords[roi][2] = start,end\n",
    "    print('Expanded new coord: ', (start,end))\n",
    "    \n",
    "    #MAKE DIRECTORY FOR OUTPUTS\n",
    "    newDir = '%sinput/%s/' % (miropeatsDir,ID)\n",
    "    cmd = 'mkdir -p %s ' % (newDir)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    \n",
    "    #CREATE COMMANDS FILE \n",
    "    cmdsfile = '%sMiropeatsCommands_%s.cmds' % (newDir,ID)\n",
    "    cmdsFile = open(cmdsfile,'w')\n",
    "    print('All commands for processing of ROI %s written to: %s' % (ID,cmdsfile))\n",
    "    \n",
    "    cmd = 'chmod 777 %s' % cmdsfile #make the commands file executable\n",
    "    runCMD(cmd) \n",
    "    \n",
    "    \n",
    "    ##WRITE BED FILE FOR ZOEY\n",
    "    seqAbedfile = '%s%s_%s.bed' % (seqAID,newDir,ID)\n",
    "    bedFile = open(seqAbedfile, 'w')\n",
    "    bedFile.write('%s\\t%i\\t%i\\tZoey_%s\\n' % (chrom,start,end,ID))\n",
    "    bedFile.close()\n",
    "    \n",
    "    ##EXTRACT FASTA \n",
    "    seqAGenome = '/home/ampend/links/kidd-lab/genomes/zoey/assemblies/2.3/ref/zoey.2.3.fa'\n",
    "    outFasta = '%sZoey_%s.fa' % (newDir,ID)\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (zoeyGenomeFastaFile,chrom,start,end,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    #RENAME FASTA ID\n",
    "    cmd = 'sed -i \\'s/%s:%i-%i/Zoey_%s/g\\' %s' % (chrom,start,end,ID,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    ##REPEAT MASK FASTA\n",
    "    cmd = 'module load RepeatMasker'\n",
    "    cmdsFile.write('%s\\n' % cmd)    \n",
    "    \n",
    "    cmd = 'RepeatMasker --species dog %s' % outFasta\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    #INTERSECT WITH EXONS\n",
    "    exonBedFile = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/FINAL_GENES/Zoey_FinalGenes_exons.bed'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s> %sexons_Zoey.exons' % (zoeybedfile,exonBedFile,newDir)\n",
    "    runCMD(cmd)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    ############################################\n",
    "    #GET CANFAM EQUIVALENT PROCESSED\n",
    "    ############################################\n",
    "    \n",
    "    ## BLAT ZOEY SEQ AGAINST CANFAM3.1\n",
    "    CanFamGenome2bitFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.2bit'\n",
    "    oocFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/11.ooc'\n",
    "    \n",
    "    zoeyFasta = '%sZoey_%s.fa' % (newDir,ID)\n",
    "    blatOutFile = newDir + 'BLAT_to_Canfam_' + ID + '.psl' \n",
    "    cmd = 'blat -out=psl -ooc=%s -tileSize=11 -noHead %s %s %s' % (oocFile,CanFamGenome2bitFile,zoeyFasta,blatOutFile)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    runCMD(cmd)\n",
    "    \n",
    "    \n",
    "    #PARSE BLAT FILE TO FIND BEST HIT\n",
    "    bestHit,bestHitInfo = 0, [] \n",
    "    for line in open(blatOutFile,'r'):\n",
    "        line=line.rstrip().split('\\t')\n",
    "        hitLength,zoeyLength = int(line[0]),int(line[10])\n",
    "        propHit = hitLength/float(zoeyLength)\n",
    "        if propHit > bestHit:\n",
    "            bestHit = propHit\n",
    "            bestHitInfo = line\n",
    "    CFchrom,CFstart,CFend = bestHitInfo[13],int(bestHitInfo[15]),int(bestHitInfo[16])\n",
    "    print('Best Hit: %s:%i-%i (%f%%)' % (CFchrom,CFstart,CFend,bestHit))\n",
    "    \n",
    "    \n",
    "    #EXTEND OUT FOR ALIGNMENTS\n",
    "    CFstart,CFend = CFstart - 1000, CFend + 1000\n",
    "    if CFstart < 0:\n",
    "        CFstart = 0\n",
    "    \n",
    "    #WRITE BED FILE FOR CANFAM\n",
    "    canfamBedfile = '%sCanFam_%s.bed' % (newDir,ID)\n",
    "    bedFile = open(canfamBedfile, 'w')\n",
    "    bedFile.write('%s\\t%i\\t%i\\tCanFam_%s\\n' % (CFchrom,CFstart,CFend,ID))\n",
    "    bedFile.close()\n",
    "    \n",
    "    \n",
    "    #GET FASTA\n",
    "    CanFamGenomeFastaFile = '/home/ampend/links/kidd-lab/genomes/canFam3.1/canFam3.1-cat/canFam3.1.fa'\n",
    "    outFasta = '%sCanFam_%s.fa' % (newDir,ID)\n",
    "    cmd = 'samtools faidx %s %s:%i-%i > %s' % (CanFamGenomeFastaFile,CFchrom,CFstart,CFend,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    \n",
    "    #RENAME FASTA ID\n",
    "    cmd = 'sed -i \\'s/%s:%i-%i/CanFam_%s/g\\' %s' % (CFchrom,CFstart,CFend,ID,outFasta)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "    \n",
    "    \n",
    "    ##REPEAT MASK FASTA\n",
    "    cmd = 'RepeatMasker --species dog %s' % outFasta\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    \n",
    "    #INTERSECT WITH EXONS\n",
    "    exonBedFile = ' ~/links/kidd-lab/genomes/canFam3.1/annotations/ensemble-79/ensGene.exons.bed'\n",
    "    cmd = 'bedtools intersect -wo -a %s -b %s > %sexons_CanFam.exons' % (canfamBedfile,exonBedFile,newDir)\n",
    "    cmdsFile.write('%s\\n' % cmd)\n",
    "\n",
    "    #WRITE MIROPEATS COMMAND\n",
    "    ## INPUTS == MASKED FASTA FILES\n",
    "    zoeyFasta = '%sZoey_%s.fa' % (newDir,ID)\n",
    "    canfamFasta = '%sCanFam_%s.fa' % (newDir,ID)\n",
    "\n",
    "    \n",
    "    #################\n",
    "    ### MIROPEATS ###\n",
    "    #################\n",
    "    print('\\n#MIROPEATS')\n",
    "\n",
    "    #Run first miropeats + blat\n",
    "    run_miropeats_script(cmdsFile)\n",
    "\n",
    "    #Call gaps\n",
    "    call_gaps(cmdsFile)\n",
    "\n",
    "    #Run miropeats annotate script\n",
    "    write_miropeats_annotate(miropeatsDir,newDir,ID,chrom,start,end)\n",
    "\n",
    "    \n",
    "cmdsFile.close()\n",
    "print('Done')\"\"\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "############################################################################################################\n",
    "############################################################################################################\n",
    "############################################################################################################\n",
    "############################################################################################################"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# This section generates miropeats images from specified coordinates in Zoey AND CanFam. \n",
    "### Some regions may not have good BLAT alignments with corresponding CanFam so you may need to do a custom alignment of two Zoey/CanFam regions."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Find all gene symbols for printing onto miropeats image for each exon/gene model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Added 42911 gene and gene symbol pairings\n"
     ]
    }
   ],
   "source": [
    "finalGeneDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/FINAL_GENES/'\n",
    "symbolDict = {}\n",
    "for line in open(finalGeneDir + 'FinalDeNovoGenes_ZoeyCoord_CanfamCoord_BLAST2GO.txt', 'r'):\n",
    "    if 'ShortGene' in line: #skip header\n",
    "        continue\n",
    "    line=line.rstrip().split('\\t')\n",
    "    longGeneID = line[1]\n",
    "    geneSymbol = line[14]\n",
    "    symbolDict[longGeneID] = geneSymbol\n",
    "\n",
    "print('Added %i gene and gene symbol pairings' % len(symbolDict.keys()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_gene_symbols(cmdsFile,inDir,ID,name):\n",
    "    for line in open('%sexons_Zoey.exons' % inDir):\n",
    "        line=line.rstrip().split('\\t')\n",
    "        exon = line[7]\n",
    "        longGeneID = exon.rsplit('.',1)[0]\n",
    "        if longGeneID not in symbolDict.keys():\n",
    "            symbol = longGeneID\n",
    "        else:\n",
    "            if ',' in symbolDict[longGeneID]:\n",
    "                symbol = symbolDict[longGeneID].split(', ')[0]\n",
    "            else:\n",
    "                symbol = symbolDict[longGeneID]\n",
    "        cmdsFile.write('sed -i \\'s/%s/%s/g\\' %sexons_Zoey.exons\\n' % (line[7],symbol,inDir))\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "#############################################\n",
    "ID = 'OCA2_Locus_5prime'\n",
    "zoeyCoords = 'chr3:32147080-32222336'\n",
    "canFamCoords = 'chr3:31657254-32343345'\n",
    "#############################################\n",
    "ID = 'IMPK_Locus'\n",
    "zoeyCoords = 'chr4:10542321-10753952'\n",
    "canFamCoords = 'chr4:10500414-10512850'\n",
    "#############################################\n",
    "ID = 'PAX6_Locus'\n",
    "zoeyCoords = 'chr18:35804273-35862527'\n",
    "canFamCoords = 'chr18:35615647-35648660'\n",
    "#############################################\n",
    "ID = 'CHP2_Locus'\n",
    "zoeyCoords = 'chr6:22,071,627-22,078,595'.replace(',','')\n",
    "canFamCoords = 'chr6:22125564-22132356'\n",
    "#############################################\n",
    "ID = 'CD151_POLR2L_Locus'\n",
    "zoeyCoords = 'chr18:45320491-45333588'\n",
    "canFamCoords = 'chr18:45185291-45205702'\n",
    "#############################################\n",
    "ID = 'CAMK2N1_Locus'\n",
    "zoeyCoords = 'chr2:75797821-75804153'\n",
    "canFamCoords = 'chr2:78402536-78420364'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "78402536"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "78362536+40000"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "#### CD151_POLR2L_Locus\n",
      "All commands for processing of ROI CD151_POLR2L_Locus written to: /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/MiropeatsCommands_CD151_POLR2L_Locus.cmds\n",
      "\n",
      "\n",
      "#Zoey\n",
      "Coordinates for ROI Zoey_CD151_POLR2L_Locus: chr18:45315491-45333588\n",
      "Extended coordinates are chr18:45314491-45334588\n",
      "bedtools intersect -wo -a /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/Zoey_CD151_POLR2L_Locus.bed -b /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/FINAL_GENES/Zoey_FinalGenes_exons.bed> /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/exons_Zoey.exons\n",
      "\n",
      "#CanFam\n",
      "Coordinates for ROI CanFam_CD151_POLR2L_Locus: chr18:45185291-45205702\n",
      "Extended coordinates are chr18:45184291-45206702\n",
      "bedtools intersect -wo -a /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/CanFam_CD151_POLR2L_Locus.bed -b  ~/links/kidd-lab/genomes/canFam3.1/annotations/ensemble-79/ensGene.exons.bed> /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/exons_CanFam.exons\n",
      "\n",
      "#MIROPEATS\n",
      "python /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/scripts/annotate-miropeats-2seqs-ForZoeyGenomeComparisons.py --miroin /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/miropeats_s50_CD151_POLR2L_Locus_onlyinter.ps --topRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/CanFam_CD151_POLR2L_Locus.fa.out --bottomRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/Zoey_CD151_POLR2L_Locus.fa.out --topName CanFam_CD151_POLR2L_Locus --bottomName Zoey_CD151_POLR2L_Locus --blat /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/BLAT_CanFam_vs_Zoey.blat --coord chr18:45314491-45334588 \n",
      "python /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/scripts/annotate-miropeats-2seqs-ForZoeyGenomeComparisons.py --miroin /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/miropeats_s100_CD151_POLR2L_Locus_onlyinter.ps --topRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/CanFam_CD151_POLR2L_Locus.fa.out --bottomRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/Zoey_CD151_POLR2L_Locus.fa.out --topName CanFam_CD151_POLR2L_Locus --bottomName Zoey_CD151_POLR2L_Locus --blat /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/BLAT_CanFam_vs_Zoey.blat --coord chr18:45314491-45334588 \n",
      "python /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/scripts/annotate-miropeats-2seqs-ForZoeyGenomeComparisons.py --miroin /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/miropeats_s200_CD151_POLR2L_Locus_onlyinter.ps --topRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/CanFam_CD151_POLR2L_Locus.fa.out --bottomRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/Zoey_CD151_POLR2L_Locus.fa.out --topName CanFam_CD151_POLR2L_Locus --bottomName Zoey_CD151_POLR2L_Locus --blat /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/BLAT_CanFam_vs_Zoey.blat --coord chr18:45314491-45334588 \n",
      "python /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/scripts/annotate-miropeats-2seqs-ForZoeyGenomeComparisons.py --miroin /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/miropeats_s250_CD151_POLR2L_Locus_onlyinter.ps --topRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/CanFam_CD151_POLR2L_Locus.fa.out --bottomRM /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/Zoey_CD151_POLR2L_Locus.fa.out --topName CanFam_CD151_POLR2L_Locus --bottomName Zoey_CD151_POLR2L_Locus --blat /home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/input/CD151_POLR2L_Locus/BLAT_CanFam_vs_Zoey.blat --coord chr18:45314491-45334588 \n"
     ]
    }
   ],
   "source": [
    "miropeatsDir = '/home/ampend/links/kidd-lab/ampend-projects/Zoey_Genome_Project/miropeats/'\n",
    "\n",
    "#***** CHANGE *****\n",
    "ID = 'CD151_POLR2L_Locus'\n",
    "zoeyCoords = 'chr18:45315491-45333588'\n",
    "canFamCoords = 'chr18:45185291-45205702'\n",
    "\n",
    "\"\"\"ID = 'CAMK2N1_Locus'\n",
    "zoeyCoords = 'chr2:75797821-75804153'\n",
    "canFamCoords = 'chr2:78402536-78420364'\"\"\"\n",
    "\n",
    "print('\\n#### %s' % ID)\n",
    "\n",
    "#Create directory and commands file\n",
    "inDir = create_directory(miropeatsDir,ID)\n",
    "cmdsFile = create_commands_file(inDir,ID)\n",
    "\n",
    "\n",
    "#MAKE BEDFILES FOR EACH\n",
    "make_bedfile(inDir,ID,zoeyCoords,canFamCoords)\n",
    "\n",
    "##############\n",
    "###  ZOEY  ###\n",
    "##############\n",
    "name = 'Zoey'\n",
    "print('\\n#%s' % name)\n",
    "\n",
    "#Get coordinates from zoey bedfile\n",
    "chrom,start,end = read_bedfile(inDir,ID,name)\n",
    "\n",
    "##EXTRACT FASTA \n",
    "outFasta = extract_fasta(inDir,name,ID,chrom,start,end,cmdsFile)\n",
    "\n",
    "##REPEAT MASK FASTA\n",
    "write_repeat_commands(outFasta,cmdsFile)\n",
    "\n",
    "## INTERSECT WITH EXONS\n",
    "do_exon_intersects(inDir,ID,name,cmdsFile)\n",
    "get_gene_symbols(cmdsFile,inDir,ID,name)\n",
    "\n",
    "\n",
    "##############\n",
    "### CANFAM ###\n",
    "##############\n",
    "name = 'CanFam'\n",
    "print('\\n#%s' % name)\n",
    "\n",
    "#Get coordinates from CANFAM bedfile\n",
    "CFchrom,CFstart,CFend = read_bedfile(inDir,ID,name)\n",
    "\n",
    "##EXTRACT FASTA \n",
    "outFasta = extract_fasta(inDir,name,ID,CFchrom,CFstart,CFend,cmdsFile)\n",
    "\n",
    "##REPEAT MASK FASTA\n",
    "write_repeat_commands(outFasta,cmdsFile)\n",
    "\n",
    "## INTERSECT WITH EXONS\n",
    "do_exon_intersects(inDir,ID,name,cmdsFile)\n",
    "\n",
    "\n",
    "#################\n",
    "### MIROPEATS ###\n",
    "#################\n",
    "print('\\n#MIROPEATS')\n",
    "\n",
    "#Run first miropeats + blat\n",
    "run_miropeats_script(cmdsFile)\n",
    "\n",
    "#Call gaps\n",
    "call_gaps(cmdsFile)\n",
    "\n",
    "#Run miropeats annotate script\n",
    "write_miropeats_annotate(miropeatsDir,inDir,ID,chrom,start,end)\n",
    "\n",
    "cmdsFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
