{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2018-02-26\n",
    "# A. Pendleton\n",
    "# Processing the de novo transcriptome assembly generated by Trinity"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "metadata": {},
   "outputs": [],
   "source": [
    "#this uses iPython magic to make plots appear inline\n",
    "%matplotlib inline\n",
    "import matplotlib.pyplot as plt\n",
    "import pandas as pd\n",
    "import subprocess\n",
    "import sys\n",
    "import numpy as np\n",
    "import matplotlib.patches as patches\n",
    "import gzip\n",
    "import fileinput\n",
    "import glob\n",
    "from scipy import stats\n",
    "import re\n",
    "from matplotlib_venn import venn3, venn3_circles\n",
    "\n",
    "def count_lines(f):\n",
    "    lineCount = 0\n",
    "    with open(f, 'r') as f:\n",
    "        for line in f:\n",
    "            lineCount += 1\n",
    "        return lineCount\n",
    "def runCMD(cmd):\n",
    "    val = subprocess.Popen(cmd, shell=True).wait()\n",
    "    if val == 0:\n",
    "        pass\n",
    "    else:\n",
    "        print ('command failed')\n",
    "        print (cmd)\n",
    "        sys.exit(1)\n",
    "def write_pbs_lines(scriptsDir,sample,cmdFile,wkDir,name,mem):\n",
    "    #WRITE PBS SCRIPT\n",
    "    cmdFile.write('#!/bin/bash\\n')\n",
    "    cmdFile.write('#PBS -S /bin/bash\\n')\n",
    "    cmdFile.write('#PBS -V\\n')\n",
    "    cmdFile.write('#PBS -M ampend@med.umich.edu\\n')\n",
    "    cmdFile.write('#PBS -j oe\\n')\n",
    "    cmdFile.write('#PBS -N %s_%s\\n' % (sample,name))\n",
    "    cmdFile.write('#PBS -o /home/ampend/links/kidd-lab-scratch/ampend-projects/Zoey_Genome_Project/rna-seq/logs/\\n')\n",
    "    cmdFile.write('#PBS -l pmem=%sG\\n' % mem)\n",
    "    cmdFile.write('#PBS -l nodes=1:ppn=1,qos=flux,walltime=100:00:00\\n')\n",
    "    cmdFile.write('##PBS -A medbsm_flux\\n')\n",
    "    cmdFile.write('##PBS -q flux\\n')\t\n",
    "    cmdFile.write('#PBS -A jmkidd_fluxod\\n')\n",
    "    cmdFile.write('#PBS -q fluxod\\n')\n",
    "    cmdFile.write('cd %s\\n' % wkDir)\n",
    "    cmdFile.write('\\n\\n')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "#2. What sample IDs are you processing? Read in README to dictionary\n",
    "def read_in_samples():\n",
    "    sampleInfo = '/home/ampend/links/kidd-lab/ampend-projects/Novel_Sequence_Analysis/rna-seq/SampleInfoTable.txt'\n",
    "    samples = []\n",
    "    sampleDict = {}\n",
    "\n",
    "    for line in open(sampleInfo, 'r'):\n",
    "        if 'Bio' in line: #skips header\n",
    "            continue\n",
    "        line=line.rstrip().split()\n",
    "        ID = line[4]\n",
    "        tissue = line[5]\n",
    "        samples.append(ID)\n",
    "        if tissue not in sampleDict.keys():\n",
    "            sampleDict[tissue] = []\n",
    "        sampleDict[tissue].append(ID)\n",
    "\n",
    "    print('%i samples added to a sample array...\\n' % len(samples))\n",
    "\n",
    "    return samples, sampleDict\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Save important paths \n",
    "scriptsDir = '/home/ampend/links/kidd-lab-scratch/ampend-projects/Zoey_Genome_Project/rna-seq/scripts/'\n",
    "fastaDir = '/home/ampend/links/kidd-lab-scratch/ampend-projects/Zoey_Genome_Project/rna-seq/input/Fastool_Trimmomatic_Fixed_Rcorrected_FASTQ/'\n",
    "\n",
    "trinityDir = '/home/ampend/links/kidd-lab-scratch/ampend-projects/Zoey_Genome_Project/rna-seq/Trinity_RNA-Seq_Analysis/trinity_alignments/no-genome-guided/AllLibraries_trinity/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "42 samples added to a sample array...\n",
      "\n"
     ]
    }
   ],
   "source": [
    "#What sample IDs are you wanting to align to the trinity assembly? Read in README to dictionary\n",
    "samples, sampleDict = read_in_samples()\n",
    "\n",
    "#Make a special directory to store scripts/pbs files for processing the trinity results:\n",
    "cmd = 'mkdir -p %s/trinity-processing/' % scriptsDir\n",
    "runCMD(cmd)\n",
    "\n",
    "trin_scriptsDir = scriptsDir + 'trinity-processing/'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Make a GSNAP genome for the trinity de novo transcripts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['>TRINITY_DN69561_c0_g1_i1', 'len=222', 'path=[200:0-221]', '[-1,', '200,', '-2]']\n"
     ]
    }
   ],
   "source": [
    "#First you need to generate independent \n",
    "splitDir = trinityDir + 'split_fastas/'\n",
    "transcriptFasta = trinityDir + 'Trinity.fasta'\n",
    "\n",
    "\n",
    "count = 0\n",
    "\n",
    "for line in open(transcriptFasta,'r'):\n",
    "    if '>' in line:\n",
    "        line = line.rstrip().split(' ')\n",
    "        fastaID = line[0]\n",
    "        fastaFile = open()\n",
    "    break\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Write commands to align RNA-Seq reads (per sample) to the new GSNAP transcriptome \n",
    "## genome in order to save time (instead of running with separate files)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/ampend/links/kidd-lab-scratch/ampend-projects/Zoey_Genome_Project/rna-seq/scripts/trinity-processing/\n"
     ]
    }
   ],
   "source": [
    "commandFile = open(trin_scriptsDir + 'trinity-gsnap-test-assembly.cmds','w')\n",
    "\n",
    "for sample in samples:\n",
    "    #Where are the left and right reads for the sample\n",
    "    fastafile1 = fastaDir + 'SRR543735_Rcorrected_filtered_formatted_1P.fq'\n",
    "    fastafile2 = fastaDir + 'SRR543735_Rcorrected_filtered_formatted_2P.fq'\n",
    "    cmd = 'gunzip %s && gunzip %s ' % (fastafile1, fastafile2)\n",
    "    \n",
    "    #now add to the command that you want to align to the genome\n",
    "    \n",
    "    \n",
    "    print(sample)\n",
    "\n",
    "\n",
    "\n",
    "commandFile.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
